{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "0691c309",
   "metadata": {},
   "source": [
    "# Design CTP-14 human genome 5kb library \n",
    "\n",
    "for neurexin1\n",
    "\n",
    "by Pu Zheng\n",
    "\n",
    "2022.10.14\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "137597b3",
   "metadata": {},
   "source": [
    "# Table of contents\n",
    "\n",
    "\n",
    "> 0. [Minimum required packages and settings](#0)\n",
    ">>\n",
    ">> 0.1: [import required packages](#0.1)\n",
    ">\n",
    "> 1. [Extract region sequences](#1)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3b459614",
   "metadata": {},
   "source": [
    "<a id='0'></a>\n",
    "# 0 Minimum required packages and settings"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4846ea43",
   "metadata": {},
   "source": [
    "<a id='0.1'></a>\n",
    "## 0.1 load required packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "id": "99a843b8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "6372\n"
     ]
    }
   ],
   "source": [
    "%run \"..\\..\\Startup_py3.py\"\n",
    "sys.path.append(r\"..\\..\\..\\..\\Documents\")\n",
    "\n",
    "import ImageAnalysis3 as ia\n",
    "%matplotlib notebook\n",
    "\n",
    "from ImageAnalysis3 import *\n",
    "print(os.getpid())\n",
    "\n",
    "# library design specific tools\n",
    "from ImageAnalysis3.library_tools import LibraryDesigner as ld\n",
    "from ImageAnalysis3.library_tools import LibraryTools as lt\n",
    "# biopython imports\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "from Bio.Blast.Applications import NcbiblastnCommandline\n",
    "from Bio.Blast import NCBIXML"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bd4cf4d8",
   "metadata": {},
   "source": [
    "<a id='1'></a>\n",
    "# 1 Extract region sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "d1426dad",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Use pool_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\n"
     ]
    }
   ],
   "source": [
    "## Some folders\n",
    "# human genome\n",
    "reference_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38'\n",
    "genome_folder = os.path.join(reference_folder, 'Genome')\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain'\n",
    "if not os.path.exists(pool_folder):\n",
    "    print(f\"Create pool_folder: {pool_folder}\")\n",
    "    os.makedirs(pool_folder)\n",
    "else:\n",
    "    print(f\"Use pool_folder: {pool_folder}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "714f9640",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- library_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\n",
      "-- sequence_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\n",
      "-- report_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\reports\n"
     ]
    }
   ],
   "source": [
    "# folder for sub-pool\n",
    "library_folder = os.path.join(pool_folder, f'human_NRNX1_5kb')\n",
    "if not os.path.exists(library_folder):\n",
    "    print(f\"create library folder: {library_folder}\")\n",
    "    os.makedirs(library_folder)\n",
    "# folder for fasta sequences\n",
    "sequence_folder = os.path.join(library_folder, 'sequences')\n",
    "if not os.path.exists(sequence_folder):\n",
    "    print(f\"create sequence folder: {sequence_folder}\")\n",
    "    os.makedirs(sequence_folder)\n",
    "# folder to save result probes\n",
    "report_folder = os.path.join(library_folder, 'reports')\n",
    "if not os.path.exists(report_folder):\n",
    "    print(f\"create report folder: {report_folder}\")\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "print(f\"-- library_folder: {library_folder}\")\n",
    "print(f\"-- sequence_folder: {sequence_folder}\")\n",
    "print(f\"-- report_folder: {report_folder}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "217df8b4",
   "metadata": {},
   "source": [
    "<a id='1.1'></a>\n",
    "## 1.1 select regions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "980fa9c6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- load sequence: 1, size=248956422\n",
      "-- load sequence: 10, size=133797422\n",
      "-- load sequence: 11, size=135086622\n",
      "-- load sequence: 12, size=133275309\n",
      "-- load sequence: 13, size=114364328\n",
      "-- load sequence: 14, size=107043718\n",
      "-- load sequence: 15, size=101991189\n",
      "-- load sequence: 16, size=90338345\n",
      "-- load sequence: 17, size=83257441\n",
      "-- load sequence: 18, size=80373285\n",
      "-- load sequence: 19, size=58617616\n",
      "-- load sequence: 2, size=242193529\n",
      "-- load sequence: 20, size=64444167\n",
      "-- load sequence: 21, size=46709983\n",
      "-- load sequence: 22, size=50818468\n",
      "-- load sequence: 3, size=198295559\n",
      "-- load sequence: 4, size=190214555\n",
      "-- load sequence: 5, size=181538259\n",
      "-- load sequence: 6, size=170805979\n",
      "-- load sequence: 7, size=159345973\n",
      "-- load sequence: 8, size=145138636\n",
      "-- load sequence: 9, size=138394717\n",
      "-- load sequence: MT, size=16569\n",
      "-- load sequence: X, size=156040895\n",
      "-- load sequence: Y, size=57227415\n",
      "-- load sequence: KI270728.1, size=1872759\n",
      "-- load sequence: KI270727.1, size=448248\n",
      "-- load sequence: KI270442.1, size=392061\n",
      "-- load sequence: KI270729.1, size=280839\n",
      "-- load sequence: GL000225.1, size=211173\n",
      "-- load sequence: KI270743.1, size=210658\n",
      "-- load sequence: GL000008.2, size=209709\n",
      "-- load sequence: GL000009.2, size=201709\n",
      "-- load sequence: KI270747.1, size=198735\n",
      "-- load sequence: KI270722.1, size=194050\n",
      "-- load sequence: GL000194.1, size=191469\n",
      "-- load sequence: KI270742.1, size=186739\n",
      "-- load sequence: GL000205.2, size=185591\n",
      "-- load sequence: GL000195.1, size=182896\n",
      "-- load sequence: KI270736.1, size=181920\n",
      "-- load sequence: KI270733.1, size=179772\n",
      "-- load sequence: GL000224.1, size=179693\n",
      "-- load sequence: GL000219.1, size=179198\n",
      "-- load sequence: KI270719.1, size=176845\n",
      "-- load sequence: GL000216.2, size=176608\n",
      "-- load sequence: KI270712.1, size=176043\n",
      "-- load sequence: KI270706.1, size=175055\n",
      "-- load sequence: KI270725.1, size=172810\n",
      "-- load sequence: KI270744.1, size=168472\n",
      "-- load sequence: KI270734.1, size=165050\n",
      "-- load sequence: GL000213.1, size=164239\n",
      "-- load sequence: GL000220.1, size=161802\n",
      "-- load sequence: KI270715.1, size=161471\n",
      "-- load sequence: GL000218.1, size=161147\n",
      "-- load sequence: KI270749.1, size=158759\n",
      "-- load sequence: KI270741.1, size=157432\n",
      "-- load sequence: GL000221.1, size=155397\n",
      "-- load sequence: KI270716.1, size=153799\n",
      "-- load sequence: KI270731.1, size=150754\n",
      "-- load sequence: KI270751.1, size=150742\n",
      "-- load sequence: KI270750.1, size=148850\n",
      "-- load sequence: KI270519.1, size=138126\n",
      "-- load sequence: GL000214.1, size=137718\n",
      "-- load sequence: KI270708.1, size=127682\n",
      "-- load sequence: KI270730.1, size=112551\n",
      "-- load sequence: KI270438.1, size=112505\n",
      "-- load sequence: KI270737.1, size=103838\n",
      "-- load sequence: KI270721.1, size=100316\n",
      "-- load sequence: KI270738.1, size=99375\n",
      "-- load sequence: KI270748.1, size=93321\n",
      "-- load sequence: KI270435.1, size=92983\n",
      "-- load sequence: GL000208.1, size=92689\n",
      "-- load sequence: KI270538.1, size=91309\n",
      "-- load sequence: KI270756.1, size=79590\n",
      "-- load sequence: KI270739.1, size=73985\n",
      "-- load sequence: KI270757.1, size=71251\n",
      "-- load sequence: KI270709.1, size=66860\n",
      "-- load sequence: KI270746.1, size=66486\n",
      "-- load sequence: KI270753.1, size=62944\n",
      "-- load sequence: KI270589.1, size=44474\n",
      "-- load sequence: KI270726.1, size=43739\n",
      "-- load sequence: KI270735.1, size=42811\n",
      "-- load sequence: KI270711.1, size=42210\n",
      "-- load sequence: KI270745.1, size=41891\n",
      "-- load sequence: KI270714.1, size=41717\n",
      "-- load sequence: KI270732.1, size=41543\n",
      "-- load sequence: KI270713.1, size=40745\n",
      "-- load sequence: KI270754.1, size=40191\n",
      "-- load sequence: KI270710.1, size=40176\n",
      "-- load sequence: KI270717.1, size=40062\n",
      "-- load sequence: KI270724.1, size=39555\n",
      "-- load sequence: KI270720.1, size=39050\n",
      "-- load sequence: KI270723.1, size=38115\n",
      "-- load sequence: KI270718.1, size=38054\n",
      "-- load sequence: KI270317.1, size=37690\n",
      "-- load sequence: KI270740.1, size=37240\n",
      "-- load sequence: KI270755.1, size=36723\n",
      "-- load sequence: KI270707.1, size=32032\n",
      "-- load sequence: KI270579.1, size=31033\n",
      "-- load sequence: KI270752.1, size=27745\n",
      "-- load sequence: KI270512.1, size=22689\n",
      "-- load sequence: KI270322.1, size=21476\n",
      "-- load sequence: GL000226.1, size=15008\n",
      "-- load sequence: KI270311.1, size=12399\n",
      "-- load sequence: KI270366.1, size=8320\n",
      "-- load sequence: KI270511.1, size=8127\n",
      "-- load sequence: KI270448.1, size=7992\n",
      "-- load sequence: KI270521.1, size=7642\n",
      "-- load sequence: KI270581.1, size=7046\n",
      "-- load sequence: KI270582.1, size=6504\n",
      "-- load sequence: KI270515.1, size=6361\n",
      "-- load sequence: KI270588.1, size=6158\n",
      "-- load sequence: KI270591.1, size=5796\n",
      "-- load sequence: KI270522.1, size=5674\n",
      "-- load sequence: KI270507.1, size=5353\n",
      "-- load sequence: KI270590.1, size=4685\n",
      "-- load sequence: KI270584.1, size=4513\n",
      "-- load sequence: KI270320.1, size=4416\n",
      "-- load sequence: KI270382.1, size=4215\n",
      "-- load sequence: KI270468.1, size=4055\n",
      "-- load sequence: KI270467.1, size=3920\n",
      "-- load sequence: KI270362.1, size=3530\n",
      "-- load sequence: KI270517.1, size=3253\n",
      "-- load sequence: KI270593.1, size=3041\n",
      "-- load sequence: KI270528.1, size=2983\n",
      "-- load sequence: KI270587.1, size=2969\n",
      "-- load sequence: KI270364.1, size=2855\n",
      "-- load sequence: KI270371.1, size=2805\n",
      "-- load sequence: KI270333.1, size=2699\n",
      "-- load sequence: KI270374.1, size=2656\n",
      "-- load sequence: KI270411.1, size=2646\n",
      "-- load sequence: KI270414.1, size=2489\n",
      "-- load sequence: KI270510.1, size=2415\n",
      "-- load sequence: KI270390.1, size=2387\n",
      "-- load sequence: KI270375.1, size=2378\n",
      "-- load sequence: KI270420.1, size=2321\n",
      "-- load sequence: KI270509.1, size=2318\n",
      "-- load sequence: KI270315.1, size=2276\n",
      "-- load sequence: KI270302.1, size=2274\n",
      "-- load sequence: KI270518.1, size=2186\n",
      "-- load sequence: KI270530.1, size=2168\n",
      "-- load sequence: KI270304.1, size=2165\n",
      "-- load sequence: KI270418.1, size=2145\n",
      "-- load sequence: KI270424.1, size=2140\n",
      "-- load sequence: KI270417.1, size=2043\n",
      "-- load sequence: KI270508.1, size=1951\n",
      "-- load sequence: KI270303.1, size=1942\n",
      "-- load sequence: KI270381.1, size=1930\n",
      "-- load sequence: KI270529.1, size=1899\n",
      "-- load sequence: KI270425.1, size=1884\n",
      "-- load sequence: KI270396.1, size=1880\n",
      "-- load sequence: KI270363.1, size=1803\n",
      "-- load sequence: KI270386.1, size=1788\n",
      "-- load sequence: KI270465.1, size=1774\n",
      "-- load sequence: KI270383.1, size=1750\n",
      "-- load sequence: KI270384.1, size=1658\n",
      "-- load sequence: KI270330.1, size=1652\n",
      "-- load sequence: KI270372.1, size=1650\n",
      "-- load sequence: KI270548.1, size=1599\n",
      "-- load sequence: KI270580.1, size=1553\n",
      "-- load sequence: KI270387.1, size=1537\n",
      "-- load sequence: KI270391.1, size=1484\n",
      "-- load sequence: KI270305.1, size=1472\n",
      "-- load sequence: KI270373.1, size=1451\n",
      "-- load sequence: KI270422.1, size=1445\n",
      "-- load sequence: KI270316.1, size=1444\n",
      "-- load sequence: KI270340.1, size=1428\n",
      "-- load sequence: KI270338.1, size=1428\n",
      "-- load sequence: KI270583.1, size=1400\n",
      "-- load sequence: KI270334.1, size=1368\n",
      "-- load sequence: KI270429.1, size=1361\n",
      "-- load sequence: KI270393.1, size=1308\n",
      "-- load sequence: KI270516.1, size=1300\n",
      "-- load sequence: KI270389.1, size=1298\n",
      "-- load sequence: KI270466.1, size=1233\n",
      "-- load sequence: KI270388.1, size=1216\n",
      "-- load sequence: KI270544.1, size=1202\n",
      "-- load sequence: KI270310.1, size=1201\n",
      "-- load sequence: KI270412.1, size=1179\n",
      "-- load sequence: KI270395.1, size=1143\n",
      "-- load sequence: KI270376.1, size=1136\n",
      "-- load sequence: KI270337.1, size=1121\n",
      "-- load sequence: KI270335.1, size=1048\n",
      "-- load sequence: KI270378.1, size=1048\n",
      "-- load sequence: KI270379.1, size=1045\n",
      "-- load sequence: KI270329.1, size=1040\n",
      "-- load sequence: KI270419.1, size=1029\n",
      "-- load sequence: KI270336.1, size=1026\n",
      "-- load sequence: KI270312.1, size=998\n",
      "-- load sequence: KI270539.1, size=993\n",
      "-- load sequence: KI270385.1, size=990\n",
      "-- load sequence: KI270423.1, size=981\n",
      "-- load sequence: KI270392.1, size=971\n",
      "-- load sequence: KI270394.1, size=970\n",
      "- 194 sequences loaded.\n"
     ]
    }
   ],
   "source": [
    "# prepare sequence reader\n",
    "reload(ia.library_tools.sequences)\n",
    "resolution = 5000\n",
    "flanking = 0\n",
    "\n",
    "seq_rd = ia.library_tools.sequences.sequence_reader(genome_folder, resolution=resolution, flanking=flanking)\n",
    "seq_rd.load_ref_sequences() # load genomic sequences"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1e1e46c8",
   "metadata": {},
   "source": [
    "<a id='1.2'></a>\n",
    "## 1.2 manually input region information"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "d29b4013",
   "metadata": {},
   "outputs": [],
   "source": [
    "region_dicts = [\n",
    "    {\n",
    "        'Chr': '2',\n",
    "        'Start':'48000001',\n",
    "        'End':'51500000',\n",
    "        'Region':f\"2:48000001-51500000\",\n",
    "        'Strand':'+',\n",
    "        'Gene': f\"NRNX1\",\n",
    "    }\n",
    "]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "0ab776c8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n"
     ]
    }
   ],
   "source": [
    "seqs = seq_rd.find_sequence_for_region(region_dicts)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "2970dec8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- saving sequences into folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-2.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-3.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-4.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-5.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-6.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-7.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-8.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-9.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-10.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-11.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-12.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-13.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-14.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-15.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-16.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-17.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-18.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-19.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-20.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-21.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-22.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-23.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-24.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-25.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-26.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-27.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-28.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-29.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-30.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-31.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-32.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-33.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-34.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-35.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-36.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-37.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-38.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-39.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-40.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-41.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-42.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-43.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-44.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-45.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-46.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-47.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-48.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-49.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-50.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-51.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-52.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-53.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-54.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-55.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-56.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-57.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-58.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-59.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-60.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-61.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-62.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-63.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-64.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-65.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-66.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-67.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-68.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-69.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-70.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-71.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-72.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-73.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-74.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-75.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-76.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-77.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-78.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-79.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-80.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-81.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-82.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-83.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-84.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-85.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-86.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-87.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-88.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-89.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-90.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-91.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-92.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-93.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-94.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-95.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-96.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-97.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-98.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-99.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-100.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-101.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-102.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-103.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-104.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-105.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-106.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-107.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-108.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-109.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-110.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-111.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-112.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-113.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-114.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-115.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-116.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-117.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-118.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-119.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-120.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-121.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-122.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-123.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-124.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-125.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-126.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-127.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-128.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-129.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-130.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-131.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-132.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-133.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-134.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-135.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-136.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-137.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-138.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-139.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-140.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-141.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-142.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-143.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-144.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-145.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-146.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-147.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-148.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-149.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-150.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-151.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-152.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-153.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-154.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-155.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-156.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-157.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-158.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-159.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-160.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-161.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-162.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-163.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-164.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-165.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-166.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-167.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-168.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-169.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-170.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-171.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-172.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-173.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-174.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-175.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-176.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-177.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-178.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-179.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-180.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-181.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-182.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-183.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-184.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-185.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-186.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-187.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-188.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-189.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-190.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-191.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-192.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-193.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-194.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-195.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-196.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-197.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-198.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-199.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-200.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-201.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-202.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-203.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-204.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-205.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-206.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-207.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-208.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-209.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-210.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-211.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-212.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-213.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-214.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-215.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-216.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-217.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-218.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-219.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-220.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-221.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-222.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-223.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-224.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-225.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-226.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-227.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-228.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-229.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-230.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-231.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-232.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-233.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-234.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-235.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-236.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-237.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-238.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-239.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-240.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-241.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-242.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-243.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-244.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-245.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-246.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-247.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-248.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-249.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-250.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-251.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-252.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-253.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-254.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-255.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-256.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-257.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-258.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-259.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-260.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-261.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-262.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-263.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-264.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-265.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-266.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-267.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-268.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-269.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-270.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-271.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-272.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-273.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-274.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-275.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-276.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-277.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-278.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-279.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-280.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-281.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-282.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-283.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-284.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-285.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-286.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-287.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-288.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-289.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-290.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-291.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-292.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-293.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-294.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-295.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-296.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-297.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-298.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-299.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-300.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-301.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-302.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-303.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-304.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-305.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-306.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-307.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-308.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-309.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-310.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-311.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-312.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-313.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-314.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-315.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-316.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-317.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-318.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-319.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-320.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-321.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-322.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-323.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-324.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-325.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-326.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-327.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-328.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-329.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-330.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-331.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-332.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-333.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-334.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-335.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-336.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-337.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-338.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-339.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-340.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-341.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-342.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-343.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-344.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-345.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-346.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-347.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-348.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-349.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-350.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-351.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-352.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-353.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-354.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-355.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-356.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-357.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-358.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-359.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-360.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-361.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-362.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-363.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-364.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-365.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-366.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-367.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-368.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-369.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-370.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-371.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-372.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-373.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-374.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-375.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-376.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-377.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-378.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-379.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-380.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-381.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-382.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-383.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-384.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-385.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-386.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-387.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-388.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-389.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-390.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-391.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-392.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-393.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-394.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-395.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-396.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-397.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-398.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-399.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-400.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-401.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-402.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-403.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-404.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-405.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-406.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-407.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-408.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-409.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-410.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-411.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-412.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-413.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-414.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-415.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-416.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-417.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-418.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-419.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-420.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-421.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-422.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-423.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-424.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-425.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-426.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-427.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-428.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-429.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-430.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-431.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-432.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-433.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-434.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-435.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-436.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-437.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-438.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-439.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-440.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-441.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-442.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-443.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-444.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-445.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-446.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-447.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-448.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-449.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-450.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-451.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-452.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-453.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-454.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-455.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-456.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-457.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-458.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-459.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-460.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-461.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-462.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-463.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-464.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-465.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-466.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-467.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-468.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-469.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-470.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-471.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-472.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-473.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-474.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-475.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-476.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-477.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-478.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-479.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-480.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-481.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-482.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-483.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-484.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-485.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-486.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-487.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-488.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-489.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-490.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-491.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-492.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-493.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-494.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-495.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-496.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-497.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-498.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-499.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-500.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-501.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-502.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-503.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-504.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-505.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-506.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-507.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-508.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-509.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-510.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-511.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-512.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-513.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-514.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-515.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-516.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-517.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-518.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-519.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-520.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-521.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-522.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-523.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-524.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-525.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-526.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-527.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-528.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-529.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-530.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-531.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-532.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-533.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-534.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-535.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-536.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-537.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-538.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-539.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-540.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-541.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-542.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-543.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-544.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-545.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-546.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-547.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-548.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-549.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-550.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-551.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-552.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-553.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-554.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-555.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-556.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-557.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-558.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-559.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-560.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-561.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-562.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-563.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-564.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-565.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-566.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-567.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-568.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-569.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-570.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-571.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-572.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-573.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-574.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-575.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-576.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-577.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-578.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-579.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-580.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-581.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-582.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-583.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-584.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-585.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-586.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-587.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-588.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-589.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-590.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-591.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-592.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-593.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-594.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-595.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-596.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-597.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-598.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-599.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-600.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-601.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-602.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-603.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-604.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-605.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-606.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-607.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-608.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-609.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-610.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-611.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-612.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-613.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-614.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-615.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-616.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-617.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-618.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-619.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-620.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-621.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-622.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-623.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-624.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-625.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-626.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-627.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-628.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-629.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-630.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-631.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-632.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-633.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-634.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-635.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-636.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-637.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-638.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-639.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-640.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-641.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-642.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-643.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-644.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-645.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-646.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-647.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-648.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-649.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-650.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-651.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-652.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-653.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-654.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-655.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-656.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-657.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-658.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-659.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-660.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-661.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-662.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-663.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-664.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-665.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-666.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-667.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-668.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-669.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-670.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-671.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-672.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-673.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-674.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-675.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-676.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-677.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-678.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-679.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-680.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-681.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-682.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-683.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-684.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-685.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-686.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-687.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-688.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-689.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-690.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-691.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-692.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-693.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-694.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-695.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-696.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-697.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-698.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-699.fasta\n"
     ]
    }
   ],
   "source": [
    "seq_rd.save_sequences(save_folder=sequence_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fb106a4a",
   "metadata": {},
   "source": [
    "<a id='2'></a>\n",
    "# 2. Design probe targeting sequences by probe_designer\n",
    "\n",
    "run probe_designer, remeber to clear the memory usage because each table should take ~32GB"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "cfad443e",
   "metadata": {},
   "source": [
    "<a id='2.1'></a>\n",
    "## 2.1 Construct count table with all the 17-mers in the genome\n",
    "\n",
    "Only do this if you don't have pre-built 17-mer\n",
    "\n",
    "However you can do almost the same thing for your own library during quality check\n",
    "\n",
    "This library requires hg38 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "414635fc",
   "metadata": {},
   "outputs": [],
   "source": [
    "overwrite_table = False"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "85dda5fb",
   "metadata": {},
   "source": [
    "### construct map for whole genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "35c7e4f5",
   "metadata": {},
   "outputs": [],
   "source": [
    "from ImageAnalysis3 import library_tools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "id": "dc089b78",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\hg38_genome_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.design)\n",
    " \n",
    "genome_table_file = os.path.join(reference_folder, 'hg38_genome_17w.npy')\n",
    "\n",
    "if not os.path.exists(genome_table_file) or overwrite_table:\n",
    "    # genome\n",
    "    _genome_filenames = [os.path.join(genome_folder, _fl) \n",
    "         for _fl in os.listdir(genome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_genome_filenames))\n",
    "\n",
    "    ct = library_tools.design.countTable(word=17,save_file=genome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_genome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "\n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {genome_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "29731c46",
   "metadata": {},
   "source": [
    "### construct map for transcriptome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "88940c0a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\hg38_transcriptome_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "from tqdm import tqdm\n",
    "# transcriptome\n",
    "transcriptome_folder = os.path.join(reference_folder, 'Transcriptome')\n",
    "\n",
    "transcriptome_table_file = os.path.join(reference_folder, 'hg38_transcriptome_17w.npy')\n",
    "\n",
    "if not os.path.exists(transcriptome_table_file) or overwrite_table:\n",
    "    # transcriptome\n",
    "    _transcriptome_filenames = [os.path.join(transcriptome_folder, _fl) \n",
    "         for _fl in os.listdir(transcriptome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_transcriptome_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=transcriptome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_transcriptome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {transcriptome_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0b477fab",
   "metadata": {},
   "source": [
    "### construct map for repeats from RepBase"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "dac1a780",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\Repbase_v2603_repeat_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "from tqdm import tqdm\n",
    "# repeat\n",
    "repeat_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\Repbase'\n",
    "\n",
    "repeat_table_file = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "\n",
    "if not os.path.exists(repeat_table_file) or overwrite_table:\n",
    "    # repeat\n",
    "    _repeat_filenames = [os.path.join(repeat_folder, _fl) \n",
    "         for _fl in os.listdir(repeat_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_repeat_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=repeat_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_repeat_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {repeat_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "688a066f",
   "metadata": {},
   "source": [
    "<a id='2.2'></a>\n",
    "## 2.2 Design probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "272a8424",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'\\\\\\\\10.245.74.212\\\\Chromatin_NAS_2\\\\Chromatin_Libraries\\\\CTP-14_human_brain\\\\mouse_long_5kb\\\\sequences'"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sequence_folder"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "e7c4541c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "700 regions loaded to design probes.\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\reports\\NRNX1_5kb.pbr\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "# requires pre_defined genome_folder and library_folder\n",
    "# Indices\n",
    "genome_index = os.path.join(reference_folder, 'hg38_genome_17w.npy')\n",
    "transcriptome_index = os.path.join(reference_folder, 'hg38_transcriptome_17w.npy') \n",
    "repeat_index = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "#ref_merfish_index = os.path.join(reference_folder, 'M1_meng_MERFISH_17w.npy') # merfish designed by Meng\n",
    "# get input files \n",
    "input_files = glob.glob(os.path.join(sequence_folder, '*.fasta'))\n",
    "\n",
    "print(f\"{len(input_files)} regions loaded to design probes.\")\n",
    "\n",
    "if not os.path.exists(report_folder):\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "# filename to save probe reports\n",
    "probe_savefile = os.path.join(report_folder, f'NRNX1_5kb.pbr')\n",
    "print(probe_savefile)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "722e6b4b",
   "metadata": {},
   "source": [
    "### create pb_designer class"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "f3c242de",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Probe designer derived from Bogdan Bintu:\n",
      "https://github.com/BogdanBintu/ChromatinImaging/blob/master/LibraryDesign/LibraryDesigner.py\n",
      "by Pu Zheng, 2020.11\n",
      "\n",
      "Major changes:\n",
      "    1. allow design of two strands\n",
      "    2. separate reverse_complement (rev_com) and from two strands (two_stranded) as \n",
      "    two different inputs for map_dic and sequence_dic\n",
      "    3. replace 'local_genome' with 'self_sequences' to be more explicit, and only \n",
      "    exclude the counts for the corresponding self_sequence within each input. \n",
      "\n",
      "Key information:\n",
      "    - number of input_sequence(s): 700\n",
      "    - save_file location: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\reports\\NRNX1_5kb.pbr\n",
      "\n",
      "- Fail to load from savefile: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\reports\\NRNX1_5kb.pbr, file doesn't exist.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "False"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "pb_designer = library_tools.design.pb_reports_class(\n",
    "    sequence_dic={'file':input_files,\n",
    "                  'rev_com':True, # design two strands\n",
    "                  'two_stranded':True},\n",
    "    map_dic={'genome':{'file':genome_index,'rev_com':False,'two_stranded':True},\n",
    "             #'transcriptome':{'file':transcriptome_index,'rev_com':True,'two_stranded':False},\n",
    "             'rep_genome':{'file':repeat_index,'rev_com':False,'two_stranded':True},\n",
    "             'self_sequences':{'file':input_files,'force_list':True,'rev_com':False,'two_stranded':True},\n",
    "             #'ref_merfish':{'file':ref_merfish_index,'rev_com':False,'two_stranded':True},\n",
    "             },\n",
    "        save_file=probe_savefile,\n",
    "    params_dic={'word_size':17,'pb_len':42,'buffer_len':0,'max_count':2**16-1,\n",
    "                'check_on_go': False, # whether automatically check probes\n",
    "                'auto': False, # whether automatically convert reference maps\n",
    "               },\n",
    "    check_dic={('genome','self_sequences'): 50, # 25\n",
    "               'rep_genome': 0,\n",
    "               #'transcriptome': 25, # 14\n",
    "               #'ref_merfish': 14,\n",
    "               'gc':[0.25,0.85],'tm': 37+0.62*50+5,\n",
    "               }\n",
    "    )\n",
    "print(pb_designer)\n",
    "pb_designer.load_from_file(load_probes_only=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b84dae63",
   "metadata": {},
   "source": [
    "### calculate probe reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "686689a5",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- setting attribute: map_genome\n",
      "--- finish map_genome in 74.490s.\n",
      "-- setting attribute: map_rep_genome\n",
      "--- finish map_rep_genome in 83.684s.\n",
      "Time(s): 158.17513370513916\n",
      "- Designing targeting sequence for 700 regions\n",
      "-- designing region: 2:48000000-48005000_strand_+_gene_NRNX1-seg-1 -- region: 0, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.383s.\n",
      "-- designing region: 2:48005000-48010000_strand_+_gene_NRNX1-seg-2 -- region: 1, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.199s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 2:48010000-48015000_strand_+_gene_NRNX1-seg-3 -- region: 2, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-2.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.866s.\n",
      "-- designing region: 2:48015000-48020000_strand_+_gene_NRNX1-seg-4 -- region: 3, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-3.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 2:48020000-48025000_strand_+_gene_NRNX1-seg-5 -- region: 4, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-4.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 3.594s.\n",
      "-- designing region: 2:48025000-48030000_strand_+_gene_NRNX1-seg-6 -- region: 5, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-5.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.214s.\n",
      "- Designed 9918 candidate probes in 3.631s.\n",
      "-- designing region: 2:48030000-48035000_strand_+_gene_NRNX1-seg-7 -- region: 6, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-6.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.200s.\n",
      "- Designed 9918 candidate probes in 3.628s.\n",
      "-- designing region: 2:48035000-48040000_strand_+_gene_NRNX1-seg-8 -- region: 7, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-7.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.197s.\n",
      "- Designed 9918 candidate probes in 3.610s.\n",
      "-- designing region: 2:48040000-48045000_strand_+_gene_NRNX1-seg-9 -- region: 8, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-8.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.215s.\n",
      "- Designed 9918 candidate probes in 3.645s.\n",
      "-- designing region: 2:48045000-48050000_strand_+_gene_NRNX1-seg-10 -- region: 9, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-9.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.210s.\n",
      "- Designed 9918 candidate probes in 3.648s.\n",
      "-- designing region: 2:48050000-48055000_strand_+_gene_NRNX1-seg-11 -- region: 10, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-10.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.209s.\n",
      "- Designed 9918 candidate probes in 3.643s.\n",
      "-- designing region: 2:48055000-48060000_strand_+_gene_NRNX1-seg-12 -- region: 11, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-11.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.207s.\n",
      "- Designed 9918 candidate probes in 3.795s.\n",
      "-- designing region: 2:48060000-48065000_strand_+_gene_NRNX1-seg-13 -- region: 12, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-12.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.206s.\n",
      "- Designed 9918 candidate probes in 3.643s.\n",
      "-- designing region: 2:48065000-48070000_strand_+_gene_NRNX1-seg-14 -- region: 13, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-13.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.207s.\n",
      "- Designed 9918 candidate probes in 3.658s.\n",
      "-- designing region: 2:48070000-48075000_strand_+_gene_NRNX1-seg-15 -- region: 14, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-14.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.204s.\n",
      "- Designed 9918 candidate probes in 3.624s.\n",
      "-- designing region: 2:48075000-48080000_strand_+_gene_NRNX1-seg-16 -- region: 15, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-15.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.199s.\n",
      "- Designed 9918 candidate probes in 3.508s.\n",
      "-- designing region: 2:48080000-48085000_strand_+_gene_NRNX1-seg-17 -- region: 16, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-16.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.195s.\n",
      "- Designed 9918 candidate probes in 2.814s.\n",
      "-- designing region: 2:48085000-48090000_strand_+_gene_NRNX1-seg-18 -- region: 17, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-17.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 2:48090000-48095000_strand_+_gene_NRNX1-seg-19 -- region: 18, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-18.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:48095000-48100000_strand_+_gene_NRNX1-seg-20 -- region: 19, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-19.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 2:48100000-48105000_strand_+_gene_NRNX1-seg-21 -- region: 20, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-20.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.917s.\n",
      "-- designing region: 2:48105000-48110000_strand_+_gene_NRNX1-seg-22 -- region: 21, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-21.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.752s.\n",
      "-- designing region: 2:48110000-48115000_strand_+_gene_NRNX1-seg-23 -- region: 22, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-22.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.177s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.753s.\n",
      "-- designing region: 2:48115000-48120000_strand_+_gene_NRNX1-seg-24 -- region: 23, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-23.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:48120000-48125000_strand_+_gene_NRNX1-seg-25 -- region: 24, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-24.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:48125000-48130000_strand_+_gene_NRNX1-seg-26 -- region: 25, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-25.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 2:48130000-48135000_strand_+_gene_NRNX1-seg-27 -- region: 26, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-26.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.178s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 2:48135000-48140000_strand_+_gene_NRNX1-seg-28 -- region: 27, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-27.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.178s.\n",
      "- Designed 9918 candidate probes in 2.751s.\n",
      "-- designing region: 2:48140000-48145000_strand_+_gene_NRNX1-seg-29 -- region: 28, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-28.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.753s.\n",
      "-- designing region: 2:48145000-48150000_strand_+_gene_NRNX1-seg-30 -- region: 29, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-29.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.748s.\n",
      "-- designing region: 2:48150000-48155000_strand_+_gene_NRNX1-seg-31 -- region: 30, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-30.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.753s.\n",
      "-- designing region: 2:48155000-48160000_strand_+_gene_NRNX1-seg-32 -- region: 31, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-31.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.972s.\n",
      "-- designing region: 2:48160000-48165000_strand_+_gene_NRNX1-seg-33 -- region: 32, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-32.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:48165000-48170000_strand_+_gene_NRNX1-seg-34 -- region: 33, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-33.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:48170000-48175000_strand_+_gene_NRNX1-seg-35 -- region: 34, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-34.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:48175000-48180000_strand_+_gene_NRNX1-seg-36 -- region: 35, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-35.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 2:48180000-48185000_strand_+_gene_NRNX1-seg-37 -- region: 36, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-36.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:48185000-48190000_strand_+_gene_NRNX1-seg-38 -- region: 37, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-37.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:48190000-48195000_strand_+_gene_NRNX1-seg-39 -- region: 38, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-38.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.751s.\n",
      "-- designing region: 2:48195000-48200000_strand_+_gene_NRNX1-seg-40 -- region: 39, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-39.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:48200000-48205000_strand_+_gene_NRNX1-seg-41 -- region: 40, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-40.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.754s.\n",
      "-- designing region: 2:48205000-48210000_strand_+_gene_NRNX1-seg-42 -- region: 41, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-41.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:48210000-48215000_strand_+_gene_NRNX1-seg-43 -- region: 42, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-42.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:48215000-48220000_strand_+_gene_NRNX1-seg-44 -- region: 43, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-43.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:48220000-48225000_strand_+_gene_NRNX1-seg-45 -- region: 44, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-44.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:48225000-48230000_strand_+_gene_NRNX1-seg-46 -- region: 45, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-45.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 3.022s.\n",
      "-- designing region: 2:48230000-48235000_strand_+_gene_NRNX1-seg-47 -- region: 46, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-46.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 2:48235000-48240000_strand_+_gene_NRNX1-seg-48 -- region: 47, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-47.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:48240000-48245000_strand_+_gene_NRNX1-seg-49 -- region: 48, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-48.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:48245000-48250000_strand_+_gene_NRNX1-seg-50 -- region: 49, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-49.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:48250000-48255000_strand_+_gene_NRNX1-seg-51 -- region: 50, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-50.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:48255000-48260000_strand_+_gene_NRNX1-seg-52 -- region: 51, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-51.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:48260000-48265000_strand_+_gene_NRNX1-seg-53 -- region: 52, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-52.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:48265000-48270000_strand_+_gene_NRNX1-seg-54 -- region: 53, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-53.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:48270000-48275000_strand_+_gene_NRNX1-seg-55 -- region: 54, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-54.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:48275000-48280000_strand_+_gene_NRNX1-seg-56 -- region: 55, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-55.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:48280000-48285000_strand_+_gene_NRNX1-seg-57 -- region: 56, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-56.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:48285000-48290000_strand_+_gene_NRNX1-seg-58 -- region: 57, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-57.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 2:48290000-48295000_strand_+_gene_NRNX1-seg-59 -- region: 58, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-58.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:48295000-48300000_strand_+_gene_NRNX1-seg-60 -- region: 59, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-59.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.178s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:48300000-48305000_strand_+_gene_NRNX1-seg-61 -- region: 60, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-60.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.178s.\n",
      "- Designed 9918 candidate probes in 2.753s.\n",
      "-- designing region: 2:48305000-48310000_strand_+_gene_NRNX1-seg-62 -- region: 61, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-61.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 3.100s.\n",
      "-- designing region: 2:48310000-48315000_strand_+_gene_NRNX1-seg-63 -- region: 62, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-62.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.753s.\n",
      "-- designing region: 2:48315000-48320000_strand_+_gene_NRNX1-seg-64 -- region: 63, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-63.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:48320000-48325000_strand_+_gene_NRNX1-seg-65 -- region: 64, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-64.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:48325000-48330000_strand_+_gene_NRNX1-seg-66 -- region: 65, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-65.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:48330000-48335000_strand_+_gene_NRNX1-seg-67 -- region: 66, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-66.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:48335000-48340000_strand_+_gene_NRNX1-seg-68 -- region: 67, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-67.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:48340000-48345000_strand_+_gene_NRNX1-seg-69 -- region: 68, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-68.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.753s.\n",
      "-- designing region: 2:48345000-48350000_strand_+_gene_NRNX1-seg-70 -- region: 69, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-69.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.157s.\n",
      "- Designed 9918 candidate probes in 2.734s.\n",
      "-- designing region: 2:48350000-48355000_strand_+_gene_NRNX1-seg-71 -- region: 70, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-70.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.809s.\n",
      "-- designing region: 2:48355000-48360000_strand_+_gene_NRNX1-seg-72 -- region: 71, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-71.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:48360000-48365000_strand_+_gene_NRNX1-seg-73 -- region: 72, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-72.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.754s.\n",
      "-- designing region: 2:48365000-48370000_strand_+_gene_NRNX1-seg-74 -- region: 73, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-73.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:48370000-48375000_strand_+_gene_NRNX1-seg-75 -- region: 74, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-74.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.749s.\n",
      "-- designing region: 2:48375000-48380000_strand_+_gene_NRNX1-seg-76 -- region: 75, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-75.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 2:48380000-48385000_strand_+_gene_NRNX1-seg-77 -- region: 76, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-76.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 2:48385000-48390000_strand_+_gene_NRNX1-seg-78 -- region: 77, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-77.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:48390000-48395000_strand_+_gene_NRNX1-seg-79 -- region: 78, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-78.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:48395000-48400000_strand_+_gene_NRNX1-seg-80 -- region: 79, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-79.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:48400000-48405000_strand_+_gene_NRNX1-seg-81 -- region: 80, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-80.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.754s.\n",
      "-- designing region: 2:48405000-48410000_strand_+_gene_NRNX1-seg-82 -- region: 81, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-81.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:48410000-48415000_strand_+_gene_NRNX1-seg-83 -- region: 82, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-82.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 3.202s.\n",
      "-- designing region: 2:48415000-48420000_strand_+_gene_NRNX1-seg-84 -- region: 83, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-83.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:48420000-48425000_strand_+_gene_NRNX1-seg-85 -- region: 84, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-84.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.754s.\n",
      "-- designing region: 2:48425000-48430000_strand_+_gene_NRNX1-seg-86 -- region: 85, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-85.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.177s.\n",
      "- Designed 9918 candidate probes in 2.753s.\n",
      "-- designing region: 2:48430000-48435000_strand_+_gene_NRNX1-seg-87 -- region: 86, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-86.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.753s.\n",
      "-- designing region: 2:48435000-48440000_strand_+_gene_NRNX1-seg-88 -- region: 87, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-87.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 2:48440000-48445000_strand_+_gene_NRNX1-seg-89 -- region: 88, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-88.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 2:48445000-48450000_strand_+_gene_NRNX1-seg-90 -- region: 89, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-89.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:48450000-48455000_strand_+_gene_NRNX1-seg-91 -- region: 90, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-90.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 2:48455000-48460000_strand_+_gene_NRNX1-seg-92 -- region: 91, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-91.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:48460000-48465000_strand_+_gene_NRNX1-seg-93 -- region: 92, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-92.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 2:48465000-48470000_strand_+_gene_NRNX1-seg-94 -- region: 93, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-93.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.753s.\n",
      "-- designing region: 2:48470000-48475000_strand_+_gene_NRNX1-seg-95 -- region: 94, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-94.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:48475000-48480000_strand_+_gene_NRNX1-seg-96 -- region: 95, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-95.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 2:48480000-48485000_strand_+_gene_NRNX1-seg-97 -- region: 96, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-96.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:48485000-48490000_strand_+_gene_NRNX1-seg-98 -- region: 97, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-97.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 2:48490000-48495000_strand_+_gene_NRNX1-seg-99 -- region: 98, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-98.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.751s.\n",
      "-- designing region: 2:48495000-48500000_strand_+_gene_NRNX1-seg-100 -- region: 99, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-99.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 2:48500000-48505000_strand_+_gene_NRNX1-seg-101 -- region: 100, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-100.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:48505000-48510000_strand_+_gene_NRNX1-seg-102 -- region: 101, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-101.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:48510000-48515000_strand_+_gene_NRNX1-seg-103 -- region: 102, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-102.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.753s.\n",
      "-- designing region: 2:48515000-48520000_strand_+_gene_NRNX1-seg-104 -- region: 103, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-103.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.177s.\n",
      "- Designed 9918 candidate probes in 2.754s.\n",
      "-- designing region: 2:48520000-48525000_strand_+_gene_NRNX1-seg-105 -- region: 104, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-104.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:48525000-48530000_strand_+_gene_NRNX1-seg-106 -- region: 105, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-105.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:48530000-48535000_strand_+_gene_NRNX1-seg-107 -- region: 106, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-106.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:48535000-48540000_strand_+_gene_NRNX1-seg-108 -- region: 107, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-107.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 2:48540000-48545000_strand_+_gene_NRNX1-seg-109 -- region: 108, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-108.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 2:48545000-48550000_strand_+_gene_NRNX1-seg-110 -- region: 109, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-109.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 3.325s.\n",
      "-- designing region: 2:48550000-48555000_strand_+_gene_NRNX1-seg-111 -- region: 110, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-110.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:48555000-48560000_strand_+_gene_NRNX1-seg-112 -- region: 111, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-111.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.744s.\n",
      "-- designing region: 2:48560000-48565000_strand_+_gene_NRNX1-seg-113 -- region: 112, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-112.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.737s.\n",
      "-- designing region: 2:48565000-48570000_strand_+_gene_NRNX1-seg-114 -- region: 113, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-113.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:48570000-48575000_strand_+_gene_NRNX1-seg-115 -- region: 114, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-114.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 2:48575000-48580000_strand_+_gene_NRNX1-seg-116 -- region: 115, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-115.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:48580000-48585000_strand_+_gene_NRNX1-seg-117 -- region: 116, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-116.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 2:48585000-48590000_strand_+_gene_NRNX1-seg-118 -- region: 117, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-117.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:48590000-48595000_strand_+_gene_NRNX1-seg-119 -- region: 118, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-118.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:48595000-48600000_strand_+_gene_NRNX1-seg-120 -- region: 119, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-119.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:48600000-48605000_strand_+_gene_NRNX1-seg-121 -- region: 120, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-120.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:48605000-48610000_strand_+_gene_NRNX1-seg-122 -- region: 121, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-121.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:48610000-48615000_strand_+_gene_NRNX1-seg-123 -- region: 122, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-122.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 2:48615000-48620000_strand_+_gene_NRNX1-seg-124 -- region: 123, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-123.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.178s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 2:48620000-48625000_strand_+_gene_NRNX1-seg-125 -- region: 124, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-124.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:48625000-48630000_strand_+_gene_NRNX1-seg-126 -- region: 125, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-125.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 2:48630000-48635000_strand_+_gene_NRNX1-seg-127 -- region: 126, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-126.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:48635000-48640000_strand_+_gene_NRNX1-seg-128 -- region: 127, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-127.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 2:48640000-48645000_strand_+_gene_NRNX1-seg-129 -- region: 128, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-128.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 2:48645000-48650000_strand_+_gene_NRNX1-seg-130 -- region: 129, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-129.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:48650000-48655000_strand_+_gene_NRNX1-seg-131 -- region: 130, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-130.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 2:48655000-48660000_strand_+_gene_NRNX1-seg-132 -- region: 131, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-131.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 2:48660000-48665000_strand_+_gene_NRNX1-seg-133 -- region: 132, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-132.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:48665000-48670000_strand_+_gene_NRNX1-seg-134 -- region: 133, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-133.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:48670000-48675000_strand_+_gene_NRNX1-seg-135 -- region: 134, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-134.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:48675000-48680000_strand_+_gene_NRNX1-seg-136 -- region: 135, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-135.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:48680000-48685000_strand_+_gene_NRNX1-seg-137 -- region: 136, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-136.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:48685000-48690000_strand_+_gene_NRNX1-seg-138 -- region: 137, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-137.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:48690000-48695000_strand_+_gene_NRNX1-seg-139 -- region: 138, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-138.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:48695000-48700000_strand_+_gene_NRNX1-seg-140 -- region: 139, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-139.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 2:48700000-48705000_strand_+_gene_NRNX1-seg-141 -- region: 140, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-140.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:48705000-48710000_strand_+_gene_NRNX1-seg-142 -- region: 141, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-141.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.858s.\n",
      "-- designing region: 2:48710000-48715000_strand_+_gene_NRNX1-seg-143 -- region: 142, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-142.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 3.463s.\n",
      "-- designing region: 2:48715000-48720000_strand_+_gene_NRNX1-seg-144 -- region: 143, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-143.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:48720000-48725000_strand_+_gene_NRNX1-seg-145 -- region: 144, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-144.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 2:48725000-48730000_strand_+_gene_NRNX1-seg-146 -- region: 145, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-145.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:48730000-48735000_strand_+_gene_NRNX1-seg-147 -- region: 146, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-146.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:48735000-48740000_strand_+_gene_NRNX1-seg-148 -- region: 147, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-147.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.754s.\n",
      "-- designing region: 2:48740000-48745000_strand_+_gene_NRNX1-seg-149 -- region: 148, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-148.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:48745000-48750000_strand_+_gene_NRNX1-seg-150 -- region: 149, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-149.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 2:48750000-48755000_strand_+_gene_NRNX1-seg-151 -- region: 150, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-150.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:48755000-48760000_strand_+_gene_NRNX1-seg-152 -- region: 151, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-151.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:48760000-48765000_strand_+_gene_NRNX1-seg-153 -- region: 152, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-152.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:48765000-48770000_strand_+_gene_NRNX1-seg-154 -- region: 153, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-153.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "- Designed 9918 candidate probes in 2.714s.\n",
      "-- designing region: 2:48770000-48775000_strand_+_gene_NRNX1-seg-155 -- region: 154, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-154.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:48775000-48780000_strand_+_gene_NRNX1-seg-156 -- region: 155, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-155.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:48780000-48785000_strand_+_gene_NRNX1-seg-157 -- region: 156, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-156.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:48785000-48790000_strand_+_gene_NRNX1-seg-158 -- region: 157, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-157.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:48790000-48795000_strand_+_gene_NRNX1-seg-159 -- region: 158, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-158.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:48795000-48800000_strand_+_gene_NRNX1-seg-160 -- region: 159, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-159.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.176s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:48800000-48805000_strand_+_gene_NRNX1-seg-161 -- region: 160, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-160.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:48805000-48810000_strand_+_gene_NRNX1-seg-162 -- region: 161, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-161.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:48810000-48815000_strand_+_gene_NRNX1-seg-163 -- region: 162, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-162.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:48815000-48820000_strand_+_gene_NRNX1-seg-164 -- region: 163, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-163.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:48820000-48825000_strand_+_gene_NRNX1-seg-165 -- region: 164, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-164.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 2:48825000-48830000_strand_+_gene_NRNX1-seg-166 -- region: 165, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-165.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 2:48830000-48835000_strand_+_gene_NRNX1-seg-167 -- region: 166, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-166.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:48835000-48840000_strand_+_gene_NRNX1-seg-168 -- region: 167, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-167.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:48840000-48845000_strand_+_gene_NRNX1-seg-169 -- region: 168, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-168.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:48845000-48850000_strand_+_gene_NRNX1-seg-170 -- region: 169, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-169.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:48850000-48855000_strand_+_gene_NRNX1-seg-171 -- region: 170, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-170.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:48855000-48860000_strand_+_gene_NRNX1-seg-172 -- region: 171, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-171.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:48860000-48865000_strand_+_gene_NRNX1-seg-173 -- region: 172, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-172.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:48865000-48870000_strand_+_gene_NRNX1-seg-174 -- region: 173, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-173.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.176s.\n",
      "- Designed 9918 candidate probes in 2.751s.\n",
      "-- designing region: 2:48870000-48875000_strand_+_gene_NRNX1-seg-175 -- region: 174, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-174.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:48875000-48880000_strand_+_gene_NRNX1-seg-176 -- region: 175, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-175.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:48880000-48885000_strand_+_gene_NRNX1-seg-177 -- region: 176, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-176.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:48885000-48890000_strand_+_gene_NRNX1-seg-178 -- region: 177, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-177.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.612s.\n",
      "-- designing region: 2:48890000-48895000_strand_+_gene_NRNX1-seg-179 -- region: 178, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-178.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:48895000-48900000_strand_+_gene_NRNX1-seg-180 -- region: 179, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-179.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:48900000-48905000_strand_+_gene_NRNX1-seg-181 -- region: 180, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-180.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:48905000-48910000_strand_+_gene_NRNX1-seg-182 -- region: 181, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-181.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 2:48910000-48915000_strand_+_gene_NRNX1-seg-183 -- region: 182, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-182.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 2:48915000-48920000_strand_+_gene_NRNX1-seg-184 -- region: 183, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-183.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 3.648s.\n",
      "-- designing region: 2:48920000-48925000_strand_+_gene_NRNX1-seg-185 -- region: 184, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-184.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:48925000-48930000_strand_+_gene_NRNX1-seg-186 -- region: 185, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-185.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:48930000-48935000_strand_+_gene_NRNX1-seg-187 -- region: 186, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-186.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 2:48935000-48940000_strand_+_gene_NRNX1-seg-188 -- region: 187, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-187.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:48940000-48945000_strand_+_gene_NRNX1-seg-189 -- region: 188, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-188.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.176s.\n",
      "- Designed 9918 candidate probes in 2.751s.\n",
      "-- designing region: 2:48945000-48950000_strand_+_gene_NRNX1-seg-190 -- region: 189, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-189.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:48950000-48955000_strand_+_gene_NRNX1-seg-191 -- region: 190, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-190.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:48955000-48960000_strand_+_gene_NRNX1-seg-192 -- region: 191, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-191.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:48960000-48965000_strand_+_gene_NRNX1-seg-193 -- region: 192, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-192.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 2:48965000-48970000_strand_+_gene_NRNX1-seg-194 -- region: 193, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-193.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:48970000-48975000_strand_+_gene_NRNX1-seg-195 -- region: 194, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-194.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:48975000-48980000_strand_+_gene_NRNX1-seg-196 -- region: 195, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-195.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:48980000-48985000_strand_+_gene_NRNX1-seg-197 -- region: 196, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-196.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 2:48985000-48990000_strand_+_gene_NRNX1-seg-198 -- region: 197, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-197.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:48990000-48995000_strand_+_gene_NRNX1-seg-199 -- region: 198, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-198.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:48995000-49000000_strand_+_gene_NRNX1-seg-200 -- region: 199, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-199.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:49000000-49005000_strand_+_gene_NRNX1-seg-201 -- region: 200, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-200.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:49005000-49010000_strand_+_gene_NRNX1-seg-202 -- region: 201, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-201.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:49010000-49015000_strand_+_gene_NRNX1-seg-203 -- region: 202, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-202.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 2:49015000-49020000_strand_+_gene_NRNX1-seg-204 -- region: 203, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-203.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:49020000-49025000_strand_+_gene_NRNX1-seg-205 -- region: 204, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-204.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:49025000-49030000_strand_+_gene_NRNX1-seg-206 -- region: 205, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-205.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 2:49030000-49035000_strand_+_gene_NRNX1-seg-207 -- region: 206, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-206.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:49035000-49040000_strand_+_gene_NRNX1-seg-208 -- region: 207, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-207.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.752s.\n",
      "-- designing region: 2:49040000-49045000_strand_+_gene_NRNX1-seg-209 -- region: 208, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-208.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:49045000-49050000_strand_+_gene_NRNX1-seg-210 -- region: 209, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-209.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:49050000-49055000_strand_+_gene_NRNX1-seg-211 -- region: 210, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-210.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:49055000-49060000_strand_+_gene_NRNX1-seg-212 -- region: 211, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-211.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:49060000-49065000_strand_+_gene_NRNX1-seg-213 -- region: 212, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-212.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:49065000-49070000_strand_+_gene_NRNX1-seg-214 -- region: 213, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-213.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:49070000-49075000_strand_+_gene_NRNX1-seg-215 -- region: 214, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-214.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:49075000-49080000_strand_+_gene_NRNX1-seg-216 -- region: 215, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-215.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 2:49080000-49085000_strand_+_gene_NRNX1-seg-217 -- region: 216, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-216.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:49085000-49090000_strand_+_gene_NRNX1-seg-218 -- region: 217, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-217.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:49090000-49095000_strand_+_gene_NRNX1-seg-219 -- region: 218, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-218.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:49095000-49100000_strand_+_gene_NRNX1-seg-220 -- region: 219, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-219.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.749s.\n",
      "-- designing region: 2:49100000-49105000_strand_+_gene_NRNX1-seg-221 -- region: 220, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-220.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:49105000-49110000_strand_+_gene_NRNX1-seg-222 -- region: 221, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-221.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:49110000-49115000_strand_+_gene_NRNX1-seg-223 -- region: 222, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-222.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 2:49115000-49120000_strand_+_gene_NRNX1-seg-224 -- region: 223, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-223.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:49120000-49125000_strand_+_gene_NRNX1-seg-225 -- region: 224, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-224.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:49125000-49130000_strand_+_gene_NRNX1-seg-226 -- region: 225, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-225.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:49130000-49135000_strand_+_gene_NRNX1-seg-227 -- region: 226, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-226.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:49135000-49140000_strand_+_gene_NRNX1-seg-228 -- region: 227, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-227.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:49140000-49145000_strand_+_gene_NRNX1-seg-229 -- region: 228, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-228.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:49145000-49150000_strand_+_gene_NRNX1-seg-230 -- region: 229, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-229.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:49150000-49155000_strand_+_gene_NRNX1-seg-231 -- region: 230, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-230.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:49155000-49160000_strand_+_gene_NRNX1-seg-232 -- region: 231, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-231.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:49160000-49165000_strand_+_gene_NRNX1-seg-233 -- region: 232, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-232.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.754s.\n",
      "-- designing region: 2:49165000-49170000_strand_+_gene_NRNX1-seg-234 -- region: 233, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-233.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 2:49170000-49175000_strand_+_gene_NRNX1-seg-235 -- region: 234, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-234.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 3.878s.\n",
      "-- designing region: 2:49175000-49180000_strand_+_gene_NRNX1-seg-236 -- region: 235, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-235.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:49180000-49185000_strand_+_gene_NRNX1-seg-237 -- region: 236, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-236.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:49185000-49190000_strand_+_gene_NRNX1-seg-238 -- region: 237, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-237.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:49190000-49195000_strand_+_gene_NRNX1-seg-239 -- region: 238, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-238.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 2:49195000-49200000_strand_+_gene_NRNX1-seg-240 -- region: 239, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-239.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:49200000-49205000_strand_+_gene_NRNX1-seg-241 -- region: 240, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-240.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:49205000-49210000_strand_+_gene_NRNX1-seg-242 -- region: 241, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-241.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.754s.\n",
      "-- designing region: 2:49210000-49215000_strand_+_gene_NRNX1-seg-243 -- region: 242, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-242.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 2:49215000-49220000_strand_+_gene_NRNX1-seg-244 -- region: 243, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-243.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.754s.\n",
      "-- designing region: 2:49220000-49225000_strand_+_gene_NRNX1-seg-245 -- region: 244, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-244.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 2:49225000-49230000_strand_+_gene_NRNX1-seg-246 -- region: 245, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-245.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:49230000-49235000_strand_+_gene_NRNX1-seg-247 -- region: 246, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-246.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:49235000-49240000_strand_+_gene_NRNX1-seg-248 -- region: 247, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-247.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:49240000-49245000_strand_+_gene_NRNX1-seg-249 -- region: 248, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-248.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:49245000-49250000_strand_+_gene_NRNX1-seg-250 -- region: 249, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-249.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 2:49250000-49255000_strand_+_gene_NRNX1-seg-251 -- region: 250, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-250.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 2:49255000-49260000_strand_+_gene_NRNX1-seg-252 -- region: 251, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-251.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:49260000-49265000_strand_+_gene_NRNX1-seg-253 -- region: 252, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-252.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:49265000-49270000_strand_+_gene_NRNX1-seg-254 -- region: 253, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-253.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 2:49270000-49275000_strand_+_gene_NRNX1-seg-255 -- region: 254, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-254.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:49275000-49280000_strand_+_gene_NRNX1-seg-256 -- region: 255, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-255.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:49280000-49285000_strand_+_gene_NRNX1-seg-257 -- region: 256, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-256.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:49285000-49290000_strand_+_gene_NRNX1-seg-258 -- region: 257, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-257.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:49290000-49295000_strand_+_gene_NRNX1-seg-259 -- region: 258, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-258.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:49295000-49300000_strand_+_gene_NRNX1-seg-260 -- region: 259, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-259.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:49300000-49305000_strand_+_gene_NRNX1-seg-261 -- region: 260, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-260.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:49305000-49310000_strand_+_gene_NRNX1-seg-262 -- region: 261, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-261.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.105s.\n",
      "- Designed 9918 candidate probes in 2.682s.\n",
      "-- designing region: 2:49310000-49315000_strand_+_gene_NRNX1-seg-263 -- region: 262, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-262.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:49315000-49320000_strand_+_gene_NRNX1-seg-264 -- region: 263, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-263.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:49320000-49325000_strand_+_gene_NRNX1-seg-265 -- region: 264, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-264.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.749s.\n",
      "-- designing region: 2:49325000-49330000_strand_+_gene_NRNX1-seg-266 -- region: 265, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-265.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.754s.\n",
      "-- designing region: 2:49330000-49335000_strand_+_gene_NRNX1-seg-267 -- region: 266, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-266.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.197s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:49335000-49340000_strand_+_gene_NRNX1-seg-268 -- region: 267, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-267.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:49340000-49345000_strand_+_gene_NRNX1-seg-269 -- region: 268, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-268.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.749s.\n",
      "-- designing region: 2:49345000-49350000_strand_+_gene_NRNX1-seg-270 -- region: 269, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-269.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 2:49350000-49355000_strand_+_gene_NRNX1-seg-271 -- region: 270, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-270.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:49355000-49360000_strand_+_gene_NRNX1-seg-272 -- region: 271, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-271.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.177s.\n",
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 2:49360000-49365000_strand_+_gene_NRNX1-seg-273 -- region: 272, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-272.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:49365000-49370000_strand_+_gene_NRNX1-seg-274 -- region: 273, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-273.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:49370000-49375000_strand_+_gene_NRNX1-seg-275 -- region: 274, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-274.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:49375000-49380000_strand_+_gene_NRNX1-seg-276 -- region: 275, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-275.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:49380000-49385000_strand_+_gene_NRNX1-seg-277 -- region: 276, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-276.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 2:49385000-49390000_strand_+_gene_NRNX1-seg-278 -- region: 277, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-277.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 2:49390000-49395000_strand_+_gene_NRNX1-seg-279 -- region: 278, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-278.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.753s.\n",
      "-- designing region: 2:49395000-49400000_strand_+_gene_NRNX1-seg-280 -- region: 279, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-279.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 2:49400000-49405000_strand_+_gene_NRNX1-seg-281 -- region: 280, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-280.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:49405000-49410000_strand_+_gene_NRNX1-seg-282 -- region: 281, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-281.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:49410000-49415000_strand_+_gene_NRNX1-seg-283 -- region: 282, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-282.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.752s.\n",
      "-- designing region: 2:49415000-49420000_strand_+_gene_NRNX1-seg-284 -- region: 283, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-283.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:49420000-49425000_strand_+_gene_NRNX1-seg-285 -- region: 284, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-284.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:49425000-49430000_strand_+_gene_NRNX1-seg-286 -- region: 285, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-285.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.832s.\n",
      "-- designing region: 2:49430000-49435000_strand_+_gene_NRNX1-seg-287 -- region: 286, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-286.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.744s.\n",
      "-- designing region: 2:49435000-49440000_strand_+_gene_NRNX1-seg-288 -- region: 287, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-287.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:49440000-49445000_strand_+_gene_NRNX1-seg-289 -- region: 288, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-288.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.754s.\n",
      "-- designing region: 2:49445000-49450000_strand_+_gene_NRNX1-seg-290 -- region: 289, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-289.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.751s.\n",
      "-- designing region: 2:49450000-49455000_strand_+_gene_NRNX1-seg-291 -- region: 290, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-290.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:49455000-49460000_strand_+_gene_NRNX1-seg-292 -- region: 291, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-291.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:49460000-49465000_strand_+_gene_NRNX1-seg-293 -- region: 292, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-292.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 2:49465000-49470000_strand_+_gene_NRNX1-seg-294 -- region: 293, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-293.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:49470000-49475000_strand_+_gene_NRNX1-seg-295 -- region: 294, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-294.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 2:49475000-49480000_strand_+_gene_NRNX1-seg-296 -- region: 295, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-295.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:49480000-49485000_strand_+_gene_NRNX1-seg-297 -- region: 296, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-296.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 2:49485000-49490000_strand_+_gene_NRNX1-seg-298 -- region: 297, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-297.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.751s.\n",
      "-- designing region: 2:49490000-49495000_strand_+_gene_NRNX1-seg-299 -- region: 298, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-298.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 4.147s.\n",
      "-- designing region: 2:49495000-49500000_strand_+_gene_NRNX1-seg-300 -- region: 299, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-299.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.748s.\n",
      "-- designing region: 2:49500000-49505000_strand_+_gene_NRNX1-seg-301 -- region: 300, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-300.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 2:49505000-49510000_strand_+_gene_NRNX1-seg-302 -- region: 301, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-301.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.752s.\n",
      "-- designing region: 2:49510000-49515000_strand_+_gene_NRNX1-seg-303 -- region: 302, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-302.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:49515000-49520000_strand_+_gene_NRNX1-seg-304 -- region: 303, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-303.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:49520000-49525000_strand_+_gene_NRNX1-seg-305 -- region: 304, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-304.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.140s.\n",
      "- Designed 9918 candidate probes in 2.714s.\n",
      "-- designing region: 2:49525000-49530000_strand_+_gene_NRNX1-seg-306 -- region: 305, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-305.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.750s.\n",
      "-- designing region: 2:49530000-49535000_strand_+_gene_NRNX1-seg-307 -- region: 306, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-306.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:49535000-49540000_strand_+_gene_NRNX1-seg-308 -- region: 307, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-307.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:49540000-49545000_strand_+_gene_NRNX1-seg-309 -- region: 308, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-308.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:49545000-49550000_strand_+_gene_NRNX1-seg-310 -- region: 309, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-309.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:49550000-49555000_strand_+_gene_NRNX1-seg-311 -- region: 310, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-310.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.753s.\n",
      "-- designing region: 2:49555000-49560000_strand_+_gene_NRNX1-seg-312 -- region: 311, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-311.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.757s.\n",
      "-- designing region: 2:49560000-49565000_strand_+_gene_NRNX1-seg-313 -- region: 312, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-312.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 7.429s.\n",
      "- Designed 9918 candidate probes in 23.157s.\n",
      "-- designing region: 2:49565000-49570000_strand_+_gene_NRNX1-seg-314 -- region: 313, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-313.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 2:49570000-49575000_strand_+_gene_NRNX1-seg-315 -- region: 314, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-314.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 3.151s.\n",
      "-- designing region: 2:49575000-49580000_strand_+_gene_NRNX1-seg-316 -- region: 315, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-315.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.328s.\n",
      "- Designed 9918 candidate probes in 5.797s.\n",
      "-- designing region: 2:49580000-49585000_strand_+_gene_NRNX1-seg-317 -- region: 316, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-316.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.349s.\n",
      "- Designed 9918 candidate probes in 4.279s.\n",
      "-- designing region: 2:49585000-49590000_strand_+_gene_NRNX1-seg-318 -- region: 317, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-317.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:49590000-49595000_strand_+_gene_NRNX1-seg-319 -- region: 318, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-318.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 2:49595000-49600000_strand_+_gene_NRNX1-seg-320 -- region: 319, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-319.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 5.456s.\n",
      "-- designing region: 2:49600000-49605000_strand_+_gene_NRNX1-seg-321 -- region: 320, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-320.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.360s.\n",
      "- Designed 9918 candidate probes in 5.337s.\n",
      "-- designing region: 2:49605000-49610000_strand_+_gene_NRNX1-seg-322 -- region: 321, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-321.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:49610000-49615000_strand_+_gene_NRNX1-seg-323 -- region: 322, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-322.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 2:49615000-49620000_strand_+_gene_NRNX1-seg-324 -- region: 323, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-323.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 2:49620000-49625000_strand_+_gene_NRNX1-seg-325 -- region: 324, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-324.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:49625000-49630000_strand_+_gene_NRNX1-seg-326 -- region: 325, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-325.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 2:49630000-49635000_strand_+_gene_NRNX1-seg-327 -- region: 326, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-326.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 2:49635000-49640000_strand_+_gene_NRNX1-seg-328 -- region: 327, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-327.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 2:49640000-49645000_strand_+_gene_NRNX1-seg-329 -- region: 328, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-328.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 2:49645000-49650000_strand_+_gene_NRNX1-seg-330 -- region: 329, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-329.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:49650000-49655000_strand_+_gene_NRNX1-seg-331 -- region: 330, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-330.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.178s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 2:49655000-49660000_strand_+_gene_NRNX1-seg-332 -- region: 331, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-331.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 2:49660000-49665000_strand_+_gene_NRNX1-seg-333 -- region: 332, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-332.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 2:49665000-49670000_strand_+_gene_NRNX1-seg-334 -- region: 333, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-333.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 2:49670000-49675000_strand_+_gene_NRNX1-seg-335 -- region: 334, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-334.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 2:49675000-49680000_strand_+_gene_NRNX1-seg-336 -- region: 335, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-335.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 2:49680000-49685000_strand_+_gene_NRNX1-seg-337 -- region: 336, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-336.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.749s.\n",
      "-- designing region: 2:49685000-49690000_strand_+_gene_NRNX1-seg-338 -- region: 337, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-337.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 2:49690000-49695000_strand_+_gene_NRNX1-seg-339 -- region: 338, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-338.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 2:49695000-49700000_strand_+_gene_NRNX1-seg-340 -- region: 339, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-339.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:49700000-49705000_strand_+_gene_NRNX1-seg-341 -- region: 340, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-340.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 2:49705000-49710000_strand_+_gene_NRNX1-seg-342 -- region: 341, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-341.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:49710000-49715000_strand_+_gene_NRNX1-seg-343 -- region: 342, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-342.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 2:49715000-49720000_strand_+_gene_NRNX1-seg-344 -- region: 343, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-343.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:49720000-49725000_strand_+_gene_NRNX1-seg-345 -- region: 344, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-344.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:49725000-49730000_strand_+_gene_NRNX1-seg-346 -- region: 345, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-345.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 2:49730000-49735000_strand_+_gene_NRNX1-seg-347 -- region: 346, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-346.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 2:49735000-49740000_strand_+_gene_NRNX1-seg-348 -- region: 347, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-347.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:49740000-49745000_strand_+_gene_NRNX1-seg-349 -- region: 348, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-348.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:49745000-49750000_strand_+_gene_NRNX1-seg-350 -- region: 349, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-349.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 2:49750000-49755000_strand_+_gene_NRNX1-seg-351 -- region: 350, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-350.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.194s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 2:49755000-49760000_strand_+_gene_NRNX1-seg-352 -- region: 351, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-351.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.177s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:49760000-49765000_strand_+_gene_NRNX1-seg-353 -- region: 352, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-352.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 2:49765000-49770000_strand_+_gene_NRNX1-seg-354 -- region: 353, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-353.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:49770000-49775000_strand_+_gene_NRNX1-seg-355 -- region: 354, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-354.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 2:49775000-49780000_strand_+_gene_NRNX1-seg-356 -- region: 355, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-355.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 2:49780000-49785000_strand_+_gene_NRNX1-seg-357 -- region: 356, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-356.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 2:49785000-49790000_strand_+_gene_NRNX1-seg-358 -- region: 357, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-357.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 2:49790000-49795000_strand_+_gene_NRNX1-seg-359 -- region: 358, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-358.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.109s.\n",
      "- Designed 9918 candidate probes in 2.693s.\n",
      "-- designing region: 2:49795000-49800000_strand_+_gene_NRNX1-seg-360 -- region: 359, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-359.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 2:49800000-49805000_strand_+_gene_NRNX1-seg-361 -- region: 360, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-360.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 2:49805000-49810000_strand_+_gene_NRNX1-seg-362 -- region: 361, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-361.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 2:49810000-49815000_strand_+_gene_NRNX1-seg-363 -- region: 362, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-362.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 2:49815000-49820000_strand_+_gene_NRNX1-seg-364 -- region: 363, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-363.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 2:49820000-49825000_strand_+_gene_NRNX1-seg-365 -- region: 364, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-364.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 2:49825000-49830000_strand_+_gene_NRNX1-seg-366 -- region: 365, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-365.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 2:49830000-49835000_strand_+_gene_NRNX1-seg-367 -- region: 366, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-366.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 2:49835000-49840000_strand_+_gene_NRNX1-seg-368 -- region: 367, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-367.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:49840000-49845000_strand_+_gene_NRNX1-seg-369 -- region: 368, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-368.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:49845000-49850000_strand_+_gene_NRNX1-seg-370 -- region: 369, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-369.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:49850000-49855000_strand_+_gene_NRNX1-seg-371 -- region: 370, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-370.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:49855000-49860000_strand_+_gene_NRNX1-seg-372 -- region: 371, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-371.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:49860000-49865000_strand_+_gene_NRNX1-seg-373 -- region: 372, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-372.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 2:49865000-49870000_strand_+_gene_NRNX1-seg-374 -- region: 373, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-373.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:49870000-49875000_strand_+_gene_NRNX1-seg-375 -- region: 374, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-374.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:49875000-49880000_strand_+_gene_NRNX1-seg-376 -- region: 375, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-375.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:49880000-49885000_strand_+_gene_NRNX1-seg-377 -- region: 376, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-376.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 2:49885000-49890000_strand_+_gene_NRNX1-seg-378 -- region: 377, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-377.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:49890000-49895000_strand_+_gene_NRNX1-seg-379 -- region: 378, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-378.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 4.518s.\n",
      "-- designing region: 2:49895000-49900000_strand_+_gene_NRNX1-seg-380 -- region: 379, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-379.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.632s.\n",
      "-- designing region: 2:49900000-49905000_strand_+_gene_NRNX1-seg-381 -- region: 380, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-380.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 2:49905000-49910000_strand_+_gene_NRNX1-seg-382 -- region: 381, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-381.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:49910000-49915000_strand_+_gene_NRNX1-seg-383 -- region: 382, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-382.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:49915000-49920000_strand_+_gene_NRNX1-seg-384 -- region: 383, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-383.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.759s.\n",
      "-- designing region: 2:49920000-49925000_strand_+_gene_NRNX1-seg-385 -- region: 384, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-384.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:49925000-49930000_strand_+_gene_NRNX1-seg-386 -- region: 385, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-385.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:49930000-49935000_strand_+_gene_NRNX1-seg-387 -- region: 386, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-386.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.178s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:49935000-49940000_strand_+_gene_NRNX1-seg-388 -- region: 387, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-387.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:49940000-49945000_strand_+_gene_NRNX1-seg-389 -- region: 388, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-388.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:49945000-49950000_strand_+_gene_NRNX1-seg-390 -- region: 389, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-389.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.176s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 2:49950000-49955000_strand_+_gene_NRNX1-seg-391 -- region: 390, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-390.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.761s.\n",
      "-- designing region: 2:49955000-49960000_strand_+_gene_NRNX1-seg-392 -- region: 391, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-391.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:49960000-49965000_strand_+_gene_NRNX1-seg-393 -- region: 392, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-392.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:49965000-49970000_strand_+_gene_NRNX1-seg-394 -- region: 393, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-393.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:49970000-49975000_strand_+_gene_NRNX1-seg-395 -- region: 394, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-394.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.760s.\n",
      "-- designing region: 2:49975000-49980000_strand_+_gene_NRNX1-seg-396 -- region: 395, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-395.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:49980000-49985000_strand_+_gene_NRNX1-seg-397 -- region: 396, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-396.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:49985000-49990000_strand_+_gene_NRNX1-seg-398 -- region: 397, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-397.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.758s.\n",
      "-- designing region: 2:49990000-49995000_strand_+_gene_NRNX1-seg-399 -- region: 398, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-398.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 2:49995000-50000000_strand_+_gene_NRNX1-seg-400 -- region: 399, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-399.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 2:50000000-50005000_strand_+_gene_NRNX1-seg-401 -- region: 400, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-400.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:50005000-50010000_strand_+_gene_NRNX1-seg-402 -- region: 401, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-401.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:50010000-50015000_strand_+_gene_NRNX1-seg-403 -- region: 402, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-402.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 2:50015000-50020000_strand_+_gene_NRNX1-seg-404 -- region: 403, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-403.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.693s.\n",
      "-- designing region: 2:50020000-50025000_strand_+_gene_NRNX1-seg-405 -- region: 404, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-404.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.762s.\n",
      "-- designing region: 2:50025000-50030000_strand_+_gene_NRNX1-seg-406 -- region: 405, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-405.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:50030000-50035000_strand_+_gene_NRNX1-seg-407 -- region: 406, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-406.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.756s.\n",
      "-- designing region: 2:50035000-50040000_strand_+_gene_NRNX1-seg-408 -- region: 407, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-407.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:50040000-50045000_strand_+_gene_NRNX1-seg-409 -- region: 408, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-408.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 2:50045000-50050000_strand_+_gene_NRNX1-seg-410 -- region: 409, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-409.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:50050000-50055000_strand_+_gene_NRNX1-seg-411 -- region: 410, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-410.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 2:50055000-50060000_strand_+_gene_NRNX1-seg-412 -- region: 411, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-411.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 2:50060000-50065000_strand_+_gene_NRNX1-seg-413 -- region: 412, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-412.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 2:50065000-50070000_strand_+_gene_NRNX1-seg-414 -- region: 413, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-413.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 2:50070000-50075000_strand_+_gene_NRNX1-seg-415 -- region: 414, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-414.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:50075000-50080000_strand_+_gene_NRNX1-seg-416 -- region: 415, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-415.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 2:50080000-50085000_strand_+_gene_NRNX1-seg-417 -- region: 416, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-416.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:50085000-50090000_strand_+_gene_NRNX1-seg-418 -- region: 417, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-417.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 2:50090000-50095000_strand_+_gene_NRNX1-seg-419 -- region: 418, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-418.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 2:50095000-50100000_strand_+_gene_NRNX1-seg-420 -- region: 419, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-419.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 2:50100000-50105000_strand_+_gene_NRNX1-seg-421 -- region: 420, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-420.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.805s.\n",
      "-- designing region: 2:50105000-50110000_strand_+_gene_NRNX1-seg-422 -- region: 421, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-421.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:50110000-50115000_strand_+_gene_NRNX1-seg-423 -- region: 422, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-422.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 2:50115000-50120000_strand_+_gene_NRNX1-seg-424 -- region: 423, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-423.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 2:50120000-50125000_strand_+_gene_NRNX1-seg-425 -- region: 424, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-424.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:50125000-50130000_strand_+_gene_NRNX1-seg-426 -- region: 425, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-425.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.800s.\n",
      "-- designing region: 2:50130000-50135000_strand_+_gene_NRNX1-seg-427 -- region: 426, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-426.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.799s.\n",
      "-- designing region: 2:50135000-50140000_strand_+_gene_NRNX1-seg-428 -- region: 427, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-427.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 2:50140000-50145000_strand_+_gene_NRNX1-seg-429 -- region: 428, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-428.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.799s.\n",
      "-- designing region: 2:50145000-50150000_strand_+_gene_NRNX1-seg-430 -- region: 429, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-429.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 2:50150000-50155000_strand_+_gene_NRNX1-seg-431 -- region: 430, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-430.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 2:50155000-50160000_strand_+_gene_NRNX1-seg-432 -- region: 431, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-431.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 2:50160000-50165000_strand_+_gene_NRNX1-seg-433 -- region: 432, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-432.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 2:50165000-50170000_strand_+_gene_NRNX1-seg-434 -- region: 433, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-433.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.800s.\n",
      "-- designing region: 2:50170000-50175000_strand_+_gene_NRNX1-seg-435 -- region: 434, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-434.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 2:50175000-50180000_strand_+_gene_NRNX1-seg-436 -- region: 435, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-435.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:50180000-50185000_strand_+_gene_NRNX1-seg-437 -- region: 436, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-436.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:50185000-50190000_strand_+_gene_NRNX1-seg-438 -- region: 437, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-437.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:50190000-50195000_strand_+_gene_NRNX1-seg-439 -- region: 438, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-438.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 2:50195000-50200000_strand_+_gene_NRNX1-seg-440 -- region: 439, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-439.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 2:50200000-50205000_strand_+_gene_NRNX1-seg-441 -- region: 440, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-440.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50205000-50210000_strand_+_gene_NRNX1-seg-442 -- region: 441, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-441.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 2:50210000-50215000_strand_+_gene_NRNX1-seg-443 -- region: 442, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-442.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 2:50215000-50220000_strand_+_gene_NRNX1-seg-444 -- region: 443, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-443.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 2:50220000-50225000_strand_+_gene_NRNX1-seg-445 -- region: 444, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-444.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 2:50225000-50230000_strand_+_gene_NRNX1-seg-446 -- region: 445, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-445.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 2:50230000-50235000_strand_+_gene_NRNX1-seg-447 -- region: 446, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-446.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 2:50235000-50240000_strand_+_gene_NRNX1-seg-448 -- region: 447, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-447.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50240000-50245000_strand_+_gene_NRNX1-seg-449 -- region: 448, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-448.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50245000-50250000_strand_+_gene_NRNX1-seg-450 -- region: 449, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-449.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50250000-50255000_strand_+_gene_NRNX1-seg-451 -- region: 450, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-450.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 2:50255000-50260000_strand_+_gene_NRNX1-seg-452 -- region: 451, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-451.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 2:50260000-50265000_strand_+_gene_NRNX1-seg-453 -- region: 452, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-452.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.796s.\n",
      "-- designing region: 2:50265000-50270000_strand_+_gene_NRNX1-seg-454 -- region: 453, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-453.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 2:50270000-50275000_strand_+_gene_NRNX1-seg-455 -- region: 454, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-454.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:50275000-50280000_strand_+_gene_NRNX1-seg-456 -- region: 455, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-455.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 2:50280000-50285000_strand_+_gene_NRNX1-seg-457 -- region: 456, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-456.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50285000-50290000_strand_+_gene_NRNX1-seg-458 -- region: 457, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-457.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 2:50290000-50295000_strand_+_gene_NRNX1-seg-459 -- region: 458, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-458.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 2:50295000-50300000_strand_+_gene_NRNX1-seg-460 -- region: 459, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-459.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 2:50300000-50305000_strand_+_gene_NRNX1-seg-461 -- region: 460, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-460.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 2:50305000-50310000_strand_+_gene_NRNX1-seg-462 -- region: 461, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-461.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 2:50310000-50315000_strand_+_gene_NRNX1-seg-463 -- region: 462, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-462.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 2:50315000-50320000_strand_+_gene_NRNX1-seg-464 -- region: 463, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-463.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.801s.\n",
      "-- designing region: 2:50320000-50325000_strand_+_gene_NRNX1-seg-465 -- region: 464, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-464.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 2:50325000-50330000_strand_+_gene_NRNX1-seg-466 -- region: 465, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-465.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 2:50330000-50335000_strand_+_gene_NRNX1-seg-467 -- region: 466, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-466.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.796s.\n",
      "-- designing region: 2:50335000-50340000_strand_+_gene_NRNX1-seg-468 -- region: 467, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-467.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 2:50340000-50345000_strand_+_gene_NRNX1-seg-469 -- region: 468, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-468.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.804s.\n",
      "-- designing region: 2:50345000-50350000_strand_+_gene_NRNX1-seg-470 -- region: 469, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-469.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 2:50350000-50355000_strand_+_gene_NRNX1-seg-471 -- region: 470, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-470.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 2:50355000-50360000_strand_+_gene_NRNX1-seg-472 -- region: 471, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-471.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.801s.\n",
      "-- designing region: 2:50360000-50365000_strand_+_gene_NRNX1-seg-473 -- region: 472, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-472.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 2:50365000-50370000_strand_+_gene_NRNX1-seg-474 -- region: 473, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-473.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 2:50370000-50375000_strand_+_gene_NRNX1-seg-475 -- region: 474, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-474.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50375000-50380000_strand_+_gene_NRNX1-seg-476 -- region: 475, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-475.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 2:50380000-50385000_strand_+_gene_NRNX1-seg-477 -- region: 476, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-476.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.801s.\n",
      "-- designing region: 2:50385000-50390000_strand_+_gene_NRNX1-seg-478 -- region: 477, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-477.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 4.980s.\n",
      "-- designing region: 2:50390000-50395000_strand_+_gene_NRNX1-seg-479 -- region: 478, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-478.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 2:50395000-50400000_strand_+_gene_NRNX1-seg-480 -- region: 479, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-479.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.198s.\n",
      "- Designed 9918 candidate probes in 2.804s.\n",
      "-- designing region: 2:50400000-50405000_strand_+_gene_NRNX1-seg-481 -- region: 480, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-480.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 2:50405000-50410000_strand_+_gene_NRNX1-seg-482 -- region: 481, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-481.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.802s.\n",
      "-- designing region: 2:50410000-50415000_strand_+_gene_NRNX1-seg-483 -- region: 482, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-482.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 2:50415000-50420000_strand_+_gene_NRNX1-seg-484 -- region: 483, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-483.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:50420000-50425000_strand_+_gene_NRNX1-seg-485 -- region: 484, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-484.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 2:50425000-50430000_strand_+_gene_NRNX1-seg-486 -- region: 485, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-485.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50430000-50435000_strand_+_gene_NRNX1-seg-487 -- region: 486, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-486.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.670s.\n",
      "-- designing region: 2:50435000-50440000_strand_+_gene_NRNX1-seg-488 -- region: 487, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-487.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50440000-50445000_strand_+_gene_NRNX1-seg-489 -- region: 488, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-488.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 2:50445000-50450000_strand_+_gene_NRNX1-seg-490 -- region: 489, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-489.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.796s.\n",
      "-- designing region: 2:50450000-50455000_strand_+_gene_NRNX1-seg-491 -- region: 490, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-490.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 2:50455000-50460000_strand_+_gene_NRNX1-seg-492 -- region: 491, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-491.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 2:50460000-50465000_strand_+_gene_NRNX1-seg-493 -- region: 492, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-492.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 2:50465000-50470000_strand_+_gene_NRNX1-seg-494 -- region: 493, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-493.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50470000-50475000_strand_+_gene_NRNX1-seg-495 -- region: 494, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-494.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50475000-50480000_strand_+_gene_NRNX1-seg-496 -- region: 495, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-495.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50480000-50485000_strand_+_gene_NRNX1-seg-497 -- region: 496, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-496.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.801s.\n",
      "-- designing region: 2:50485000-50490000_strand_+_gene_NRNX1-seg-498 -- region: 497, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-497.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.801s.\n",
      "-- designing region: 2:50490000-50495000_strand_+_gene_NRNX1-seg-499 -- region: 498, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-498.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 2:50495000-50500000_strand_+_gene_NRNX1-seg-500 -- region: 499, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-499.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 2:50500000-50505000_strand_+_gene_NRNX1-seg-501 -- region: 500, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-500.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.804s.\n",
      "-- designing region: 2:50505000-50510000_strand_+_gene_NRNX1-seg-502 -- region: 501, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-501.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 2:50510000-50515000_strand_+_gene_NRNX1-seg-503 -- region: 502, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-502.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 2:50515000-50520000_strand_+_gene_NRNX1-seg-504 -- region: 503, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-503.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 2:50520000-50525000_strand_+_gene_NRNX1-seg-505 -- region: 504, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-504.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 2:50525000-50530000_strand_+_gene_NRNX1-seg-506 -- region: 505, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-505.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 2:50530000-50535000_strand_+_gene_NRNX1-seg-507 -- region: 506, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-506.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 2:50535000-50540000_strand_+_gene_NRNX1-seg-508 -- region: 507, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-507.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.799s.\n",
      "-- designing region: 2:50540000-50545000_strand_+_gene_NRNX1-seg-509 -- region: 508, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-508.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50545000-50550000_strand_+_gene_NRNX1-seg-510 -- region: 509, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-509.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 2:50550000-50555000_strand_+_gene_NRNX1-seg-511 -- region: 510, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-510.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 2:50555000-50560000_strand_+_gene_NRNX1-seg-512 -- region: 511, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-511.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 2:50560000-50565000_strand_+_gene_NRNX1-seg-513 -- region: 512, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-512.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50565000-50570000_strand_+_gene_NRNX1-seg-514 -- region: 513, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-513.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50570000-50575000_strand_+_gene_NRNX1-seg-515 -- region: 514, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-514.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:50575000-50580000_strand_+_gene_NRNX1-seg-516 -- region: 515, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-515.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 2:50580000-50585000_strand_+_gene_NRNX1-seg-517 -- region: 516, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-516.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50585000-50590000_strand_+_gene_NRNX1-seg-518 -- region: 517, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-517.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 2:50590000-50595000_strand_+_gene_NRNX1-seg-519 -- region: 518, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-518.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 2:50595000-50600000_strand_+_gene_NRNX1-seg-520 -- region: 519, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-519.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 2:50600000-50605000_strand_+_gene_NRNX1-seg-521 -- region: 520, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-520.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.799s.\n",
      "-- designing region: 2:50605000-50610000_strand_+_gene_NRNX1-seg-522 -- region: 521, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-521.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.799s.\n",
      "-- designing region: 2:50610000-50615000_strand_+_gene_NRNX1-seg-523 -- region: 522, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-522.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 2:50615000-50620000_strand_+_gene_NRNX1-seg-524 -- region: 523, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-523.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.796s.\n",
      "-- designing region: 2:50620000-50625000_strand_+_gene_NRNX1-seg-525 -- region: 524, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-524.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 2:50625000-50630000_strand_+_gene_NRNX1-seg-526 -- region: 525, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-525.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 2:50630000-50635000_strand_+_gene_NRNX1-seg-527 -- region: 526, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-526.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 2:50635000-50640000_strand_+_gene_NRNX1-seg-528 -- region: 527, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-527.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 2:50640000-50645000_strand_+_gene_NRNX1-seg-529 -- region: 528, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-528.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.195s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 2:50645000-50650000_strand_+_gene_NRNX1-seg-530 -- region: 529, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-529.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.677s.\n",
      "-- designing region: 2:50650000-50655000_strand_+_gene_NRNX1-seg-531 -- region: 530, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-530.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 2:50655000-50660000_strand_+_gene_NRNX1-seg-532 -- region: 531, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-531.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50660000-50665000_strand_+_gene_NRNX1-seg-533 -- region: 532, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-532.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.798s.\n",
      "-- designing region: 2:50665000-50670000_strand_+_gene_NRNX1-seg-534 -- region: 533, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-533.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50670000-50675000_strand_+_gene_NRNX1-seg-535 -- region: 534, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-534.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 2:50675000-50680000_strand_+_gene_NRNX1-seg-536 -- region: 535, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-535.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 2:50680000-50685000_strand_+_gene_NRNX1-seg-537 -- region: 536, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-536.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 2:50685000-50690000_strand_+_gene_NRNX1-seg-538 -- region: 537, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-537.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 2:50690000-50695000_strand_+_gene_NRNX1-seg-539 -- region: 538, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-538.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.803s.\n",
      "-- designing region: 2:50695000-50700000_strand_+_gene_NRNX1-seg-540 -- region: 539, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-539.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 2:50700000-50705000_strand_+_gene_NRNX1-seg-541 -- region: 540, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-540.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 2:50705000-50710000_strand_+_gene_NRNX1-seg-542 -- region: 541, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-541.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 2:50710000-50715000_strand_+_gene_NRNX1-seg-543 -- region: 542, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-542.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 2:50715000-50720000_strand_+_gene_NRNX1-seg-544 -- region: 543, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-543.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 2:50720000-50725000_strand_+_gene_NRNX1-seg-545 -- region: 544, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-544.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 2:50725000-50730000_strand_+_gene_NRNX1-seg-546 -- region: 545, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-545.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 2:50730000-50735000_strand_+_gene_NRNX1-seg-547 -- region: 546, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-546.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.790s.\n",
      "-- designing region: 2:50735000-50740000_strand_+_gene_NRNX1-seg-548 -- region: 547, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-547.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 2:50740000-50745000_strand_+_gene_NRNX1-seg-549 -- region: 548, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-548.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 2:50745000-50750000_strand_+_gene_NRNX1-seg-550 -- region: 549, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-549.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 2:50750000-50755000_strand_+_gene_NRNX1-seg-551 -- region: 550, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-550.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 2:50755000-50760000_strand_+_gene_NRNX1-seg-552 -- region: 551, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-551.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50760000-50765000_strand_+_gene_NRNX1-seg-553 -- region: 552, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-552.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 2:50765000-50770000_strand_+_gene_NRNX1-seg-554 -- region: 553, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-553.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 2:50770000-50775000_strand_+_gene_NRNX1-seg-555 -- region: 554, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-554.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 2:50775000-50780000_strand_+_gene_NRNX1-seg-556 -- region: 555, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-555.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50780000-50785000_strand_+_gene_NRNX1-seg-557 -- region: 556, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-556.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 2:50785000-50790000_strand_+_gene_NRNX1-seg-558 -- region: 557, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-557.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:50790000-50795000_strand_+_gene_NRNX1-seg-559 -- region: 558, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-558.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 2:50795000-50800000_strand_+_gene_NRNX1-seg-560 -- region: 559, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-559.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.796s.\n",
      "-- designing region: 2:50800000-50805000_strand_+_gene_NRNX1-seg-561 -- region: 560, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-560.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:50805000-50810000_strand_+_gene_NRNX1-seg-562 -- region: 561, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-561.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.194s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 2:50810000-50815000_strand_+_gene_NRNX1-seg-563 -- region: 562, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-562.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.796s.\n",
      "-- designing region: 2:50815000-50820000_strand_+_gene_NRNX1-seg-564 -- region: 563, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-563.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 2:50820000-50825000_strand_+_gene_NRNX1-seg-565 -- region: 564, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-564.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:50825000-50830000_strand_+_gene_NRNX1-seg-566 -- region: 565, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-565.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:50830000-50835000_strand_+_gene_NRNX1-seg-567 -- region: 566, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-566.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 2:50835000-50840000_strand_+_gene_NRNX1-seg-568 -- region: 567, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-567.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:50840000-50845000_strand_+_gene_NRNX1-seg-569 -- region: 568, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-568.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.796s.\n",
      "-- designing region: 2:50845000-50850000_strand_+_gene_NRNX1-seg-570 -- region: 569, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-569.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 3.237s.\n",
      "-- designing region: 2:50850000-50855000_strand_+_gene_NRNX1-seg-571 -- region: 570, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-570.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 2:50855000-50860000_strand_+_gene_NRNX1-seg-572 -- region: 571, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-571.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:50860000-50865000_strand_+_gene_NRNX1-seg-573 -- region: 572, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-572.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 2:50865000-50870000_strand_+_gene_NRNX1-seg-574 -- region: 573, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-573.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 2:50870000-50875000_strand_+_gene_NRNX1-seg-575 -- region: 574, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-574.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 2:50875000-50880000_strand_+_gene_NRNX1-seg-576 -- region: 575, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-575.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:50880000-50885000_strand_+_gene_NRNX1-seg-577 -- region: 576, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-576.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 2:50885000-50890000_strand_+_gene_NRNX1-seg-578 -- region: 577, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-577.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 2:50890000-50895000_strand_+_gene_NRNX1-seg-579 -- region: 578, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-578.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:50895000-50900000_strand_+_gene_NRNX1-seg-580 -- region: 579, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-579.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 2:50900000-50905000_strand_+_gene_NRNX1-seg-581 -- region: 580, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-580.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 2:50905000-50910000_strand_+_gene_NRNX1-seg-582 -- region: 581, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-581.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 2:50910000-50915000_strand_+_gene_NRNX1-seg-583 -- region: 582, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-582.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.799s.\n",
      "-- designing region: 2:50915000-50920000_strand_+_gene_NRNX1-seg-584 -- region: 583, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-583.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 2:50920000-50925000_strand_+_gene_NRNX1-seg-585 -- region: 584, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-584.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:50925000-50930000_strand_+_gene_NRNX1-seg-586 -- region: 585, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-585.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 2:50930000-50935000_strand_+_gene_NRNX1-seg-587 -- region: 586, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-586.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 2:50935000-50940000_strand_+_gene_NRNX1-seg-588 -- region: 587, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-587.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 2:50940000-50945000_strand_+_gene_NRNX1-seg-589 -- region: 588, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-588.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 2:50945000-50950000_strand_+_gene_NRNX1-seg-590 -- region: 589, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-589.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 2:50950000-50955000_strand_+_gene_NRNX1-seg-591 -- region: 590, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-590.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 2:50955000-50960000_strand_+_gene_NRNX1-seg-592 -- region: 591, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-591.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 2:50960000-50965000_strand_+_gene_NRNX1-seg-593 -- region: 592, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-592.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 2:50965000-50970000_strand_+_gene_NRNX1-seg-594 -- region: 593, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-593.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 2:50970000-50975000_strand_+_gene_NRNX1-seg-595 -- region: 594, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-594.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 2:50975000-50980000_strand_+_gene_NRNX1-seg-596 -- region: 595, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-595.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 2:50980000-50985000_strand_+_gene_NRNX1-seg-597 -- region: 596, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-596.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 2:50985000-50990000_strand_+_gene_NRNX1-seg-598 -- region: 597, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-597.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 2:50990000-50995000_strand_+_gene_NRNX1-seg-599 -- region: 598, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-598.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 2:50995000-51000000_strand_+_gene_NRNX1-seg-600 -- region: 599, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-599.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 2:51000000-51005000_strand_+_gene_NRNX1-seg-601 -- region: 600, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-600.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 5.527s.\n",
      "-- designing region: 2:51005000-51010000_strand_+_gene_NRNX1-seg-602 -- region: 601, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-601.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 2:51010000-51015000_strand_+_gene_NRNX1-seg-603 -- region: 602, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-602.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 2:51015000-51020000_strand_+_gene_NRNX1-seg-604 -- region: 603, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-603.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 2:51020000-51025000_strand_+_gene_NRNX1-seg-605 -- region: 604, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-604.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 2:51025000-51030000_strand_+_gene_NRNX1-seg-606 -- region: 605, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-605.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:51030000-51035000_strand_+_gene_NRNX1-seg-607 -- region: 606, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-606.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 2:51035000-51040000_strand_+_gene_NRNX1-seg-608 -- region: 607, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-607.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 2:51040000-51045000_strand_+_gene_NRNX1-seg-609 -- region: 608, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-608.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 2:51045000-51050000_strand_+_gene_NRNX1-seg-610 -- region: 609, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-609.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.782s.\n",
      "-- designing region: 2:51050000-51055000_strand_+_gene_NRNX1-seg-611 -- region: 610, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-610.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 2:51055000-51060000_strand_+_gene_NRNX1-seg-612 -- region: 611, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-611.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.781s.\n",
      "-- designing region: 2:51060000-51065000_strand_+_gene_NRNX1-seg-613 -- region: 612, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-612.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.179s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 2:51065000-51070000_strand_+_gene_NRNX1-seg-614 -- region: 613, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-613.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 2:51070000-51075000_strand_+_gene_NRNX1-seg-615 -- region: 614, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-614.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.683s.\n",
      "-- designing region: 2:51075000-51080000_strand_+_gene_NRNX1-seg-616 -- region: 615, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-615.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 2:51080000-51085000_strand_+_gene_NRNX1-seg-617 -- region: 616, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-616.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.788s.\n",
      "-- designing region: 2:51085000-51090000_strand_+_gene_NRNX1-seg-618 -- region: 617, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-617.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 2:51090000-51095000_strand_+_gene_NRNX1-seg-619 -- region: 618, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-618.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.789s.\n",
      "-- designing region: 2:51095000-51100000_strand_+_gene_NRNX1-seg-620 -- region: 619, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-619.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 2:51100000-51105000_strand_+_gene_NRNX1-seg-621 -- region: 620, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-620.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.792s.\n",
      "-- designing region: 2:51105000-51110000_strand_+_gene_NRNX1-seg-622 -- region: 621, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-621.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 2:51110000-51115000_strand_+_gene_NRNX1-seg-623 -- region: 622, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-622.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.795s.\n",
      "-- designing region: 2:51115000-51120000_strand_+_gene_NRNX1-seg-624 -- region: 623, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-623.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 2:51120000-51125000_strand_+_gene_NRNX1-seg-625 -- region: 624, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-624.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.794s.\n",
      "-- designing region: 2:51125000-51130000_strand_+_gene_NRNX1-seg-626 -- region: 625, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-625.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 2:51130000-51135000_strand_+_gene_NRNX1-seg-627 -- region: 626, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-626.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.793s.\n",
      "-- designing region: 2:51135000-51140000_strand_+_gene_NRNX1-seg-628 -- region: 627, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-627.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 2:51140000-51145000_strand_+_gene_NRNX1-seg-629 -- region: 628, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-628.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 2:51145000-51150000_strand_+_gene_NRNX1-seg-630 -- region: 629, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-629.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 2:51150000-51155000_strand_+_gene_NRNX1-seg-631 -- region: 630, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-630.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.791s.\n",
      "-- designing region: 2:51155000-51160000_strand_+_gene_NRNX1-seg-632 -- region: 631, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-631.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 2:51160000-51165000_strand_+_gene_NRNX1-seg-633 -- region: 632, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-632.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.193s.\n",
      "- Designed 9918 candidate probes in 2.803s.\n",
      "-- designing region: 2:51165000-51170000_strand_+_gene_NRNX1-seg-634 -- region: 633, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-633.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.787s.\n",
      "-- designing region: 2:51170000-51175000_strand_+_gene_NRNX1-seg-635 -- region: 634, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-634.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.786s.\n",
      "-- designing region: 2:51175000-51180000_strand_+_gene_NRNX1-seg-636 -- region: 635, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-635.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.814s.\n",
      "-- designing region: 2:51180000-51185000_strand_+_gene_NRNX1-seg-637 -- region: 636, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-636.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.797s.\n",
      "-- designing region: 2:51185000-51190000_strand_+_gene_NRNX1-seg-638 -- region: 637, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-637.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.785s.\n",
      "-- designing region: 2:51190000-51195000_strand_+_gene_NRNX1-seg-639 -- region: 638, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-638.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 5.110s.\n",
      "- Designed 9918 candidate probes in 7.856s.\n",
      "-- designing region: 2:51195000-51200000_strand_+_gene_NRNX1-seg-640 -- region: 639, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-639.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.783s.\n",
      "-- designing region: 2:51200000-51205000_strand_+_gene_NRNX1-seg-641 -- region: 640, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-640.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 2:51205000-51210000_strand_+_gene_NRNX1-seg-642 -- region: 641, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-641.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 2:51210000-51215000_strand_+_gene_NRNX1-seg-643 -- region: 642, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-642.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 2:51215000-51220000_strand_+_gene_NRNX1-seg-644 -- region: 643, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-643.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 2:51220000-51225000_strand_+_gene_NRNX1-seg-645 -- region: 644, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-644.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 2:51225000-51230000_strand_+_gene_NRNX1-seg-646 -- region: 645, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-645.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 2:51230000-51235000_strand_+_gene_NRNX1-seg-647 -- region: 646, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-646.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 2:51235000-51240000_strand_+_gene_NRNX1-seg-648 -- region: 647, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-647.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 2:51240000-51245000_strand_+_gene_NRNX1-seg-649 -- region: 648, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-648.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 2:51245000-51250000_strand_+_gene_NRNX1-seg-650 -- region: 649, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-649.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 2:51250000-51255000_strand_+_gene_NRNX1-seg-651 -- region: 650, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-650.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 2:51255000-51260000_strand_+_gene_NRNX1-seg-652 -- region: 651, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-651.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 2:51260000-51265000_strand_+_gene_NRNX1-seg-653 -- region: 652, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-652.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:51265000-51270000_strand_+_gene_NRNX1-seg-654 -- region: 653, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-653.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 2:51270000-51275000_strand_+_gene_NRNX1-seg-655 -- region: 654, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-654.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 2:51275000-51280000_strand_+_gene_NRNX1-seg-656 -- region: 655, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-655.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 2:51280000-51285000_strand_+_gene_NRNX1-seg-657 -- region: 656, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-656.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 2:51285000-51290000_strand_+_gene_NRNX1-seg-658 -- region: 657, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-657.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.775s.\n",
      "-- designing region: 2:51290000-51295000_strand_+_gene_NRNX1-seg-659 -- region: 658, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-658.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.777s.\n",
      "-- designing region: 2:51295000-51300000_strand_+_gene_NRNX1-seg-660 -- region: 659, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-659.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.178s.\n",
      "- Designed 9918 candidate probes in 2.763s.\n",
      "-- designing region: 2:51300000-51305000_strand_+_gene_NRNX1-seg-661 -- region: 660, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-660.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.180s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 2:51305000-51310000_strand_+_gene_NRNX1-seg-662 -- region: 661, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-661.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:51310000-51315000_strand_+_gene_NRNX1-seg-663 -- region: 662, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-662.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 2:51315000-51320000_strand_+_gene_NRNX1-seg-664 -- region: 663, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-663.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.784s.\n",
      "-- designing region: 2:51320000-51325000_strand_+_gene_NRNX1-seg-665 -- region: 664, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-664.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:51325000-51330000_strand_+_gene_NRNX1-seg-666 -- region: 665, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-665.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 2:51330000-51335000_strand_+_gene_NRNX1-seg-667 -- region: 666, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-666.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 2:51335000-51340000_strand_+_gene_NRNX1-seg-668 -- region: 667, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-667.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 2:51340000-51345000_strand_+_gene_NRNX1-seg-669 -- region: 668, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-668.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:51345000-51350000_strand_+_gene_NRNX1-seg-670 -- region: 669, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-669.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.765s.\n",
      "-- designing region: 2:51350000-51355000_strand_+_gene_NRNX1-seg-671 -- region: 670, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-670.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 2:51355000-51360000_strand_+_gene_NRNX1-seg-672 -- region: 671, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-671.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 2:51360000-51365000_strand_+_gene_NRNX1-seg-673 -- region: 672, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-672.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 2:51365000-51370000_strand_+_gene_NRNX1-seg-674 -- region: 673, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-673.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 2:51370000-51375000_strand_+_gene_NRNX1-seg-675 -- region: 674, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-674.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:51375000-51380000_strand_+_gene_NRNX1-seg-676 -- region: 675, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-675.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.779s.\n",
      "-- designing region: 2:51380000-51385000_strand_+_gene_NRNX1-seg-677 -- region: 676, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-676.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.191s.\n",
      "- Designed 9918 candidate probes in 2.772s.\n",
      "-- designing region: 2:51385000-51390000_strand_+_gene_NRNX1-seg-678 -- region: 677, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-677.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.634s.\n",
      "-- designing region: 2:51390000-51395000_strand_+_gene_NRNX1-seg-679 -- region: 678, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-678.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 2:51395000-51400000_strand_+_gene_NRNX1-seg-680 -- region: 679, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-679.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 2:51400000-51405000_strand_+_gene_NRNX1-seg-681 -- region: 680, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-680.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.776s.\n",
      "-- designing region: 2:51405000-51410000_strand_+_gene_NRNX1-seg-682 -- region: 681, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-681.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:51410000-51415000_strand_+_gene_NRNX1-seg-683 -- region: 682, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-682.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:51415000-51420000_strand_+_gene_NRNX1-seg-684 -- region: 683, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-683.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "- Designed 9918 candidate probes in 2.773s.\n",
      "-- designing region: 2:51420000-51425000_strand_+_gene_NRNX1-seg-685 -- region: 684, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-684.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 2:51425000-51430000_strand_+_gene_NRNX1-seg-686 -- region: 685, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-685.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.182s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:51430000-51435000_strand_+_gene_NRNX1-seg-687 -- region: 686, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-686.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.778s.\n",
      "-- designing region: 2:51435000-51440000_strand_+_gene_NRNX1-seg-688 -- region: 687, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-687.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "- Designed 9918 candidate probes in 2.767s.\n",
      "-- designing region: 2:51440000-51445000_strand_+_gene_NRNX1-seg-689 -- region: 688, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-688.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "-- designing region: 2:51445000-51450000_strand_+_gene_NRNX1-seg-690 -- region: 689, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-689.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 9918 candidate probes in 2.768s.\n",
      "-- designing region: 2:51450000-51455000_strand_+_gene_NRNX1-seg-691 -- region: 690, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-690.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "- Designed 9918 candidate probes in 2.780s.\n",
      "-- designing region: 2:51455000-51460000_strand_+_gene_NRNX1-seg-692 -- region: 691, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-691.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.771s.\n",
      "-- designing region: 2:51460000-51465000_strand_+_gene_NRNX1-seg-693 -- region: 692, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-692.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "- Designed 9918 candidate probes in 2.764s.\n",
      "-- designing region: 2:51465000-51470000_strand_+_gene_NRNX1-seg-694 -- region: 693, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-693.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.184s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 2:51470000-51475000_strand_+_gene_NRNX1-seg-695 -- region: 694, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-694.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 2:51475000-51480000_strand_+_gene_NRNX1-seg-696 -- region: 695, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-695.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.769s.\n",
      "-- designing region: 2:51480000-51485000_strand_+_gene_NRNX1-seg-697 -- region: 696, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-696.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.766s.\n",
      "-- designing region: 2:51485000-51490000_strand_+_gene_NRNX1-seg-698 -- region: 697, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-697.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "- Designed 9918 candidate probes in 2.770s.\n",
      "-- designing region: 2:51490000-51495000_strand_+_gene_NRNX1-seg-699 -- region: 698, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-698.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.185s.\n",
      "- Designed 9918 candidate probes in 2.755s.\n",
      "-- designing region: 2:51495000-51500000_strand_+_gene_NRNX1-seg-700 -- region: 699, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\sequences\\NRNX1-seg-699.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "- Designed 9918 candidate probes in 2.774s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\reports\\NRNX1_5kb.pbr\n",
      "Wall time: 37min 34s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "pb_designer.computeOTmaps() # load the tables \n",
    "pb_designer.compute_pb_report() # design candidate probes"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e23bacce",
   "metadata": {},
   "source": [
    "### check probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "8541e32f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:0 2:48000000-48005000_strand_+_gene_NRNX1-seg-1, 9722 candidate probes\n",
      "--- 1938 probes passed check_dic selection.\n",
      "finish in 0.309s, 36 probes kept.\n",
      "-- check region:1 2:48005000-48010000_strand_+_gene_NRNX1-seg-2, 9914 candidate probes\n",
      "--- 1998 probes passed check_dic selection.\n",
      "finish in 0.314s, 40 probes kept.\n",
      "-- check region:2 2:48010000-48015000_strand_+_gene_NRNX1-seg-3, 9898 candidate probes\n",
      "--- 1936 probes passed check_dic selection.\n",
      "finish in 0.220s, 40 probes kept.\n",
      "-- check region:3 2:48015000-48020000_strand_+_gene_NRNX1-seg-4, 9902 candidate probes\n",
      "--- 3906 probes passed check_dic selection.\n",
      "finish in 0.289s, 53 probes kept.\n",
      "-- check region:4 2:48020000-48025000_strand_+_gene_NRNX1-seg-5, 9890 candidate probes\n",
      "--- 4756 probes passed check_dic selection.\n",
      "finish in 0.321s, 64 probes kept.\n",
      "-- check region:5 2:48025000-48030000_strand_+_gene_NRNX1-seg-6, 9900 candidate probes\n",
      "--- 3742 probes passed check_dic selection.\n",
      "finish in 0.280s, 53 probes kept.\n",
      "-- check region:6 2:48030000-48035000_strand_+_gene_NRNX1-seg-7, 9640 candidate probes\n",
      "--- 2040 probes passed check_dic selection.\n",
      "finish in 0.220s, 34 probes kept.\n",
      "-- check region:7 2:48035000-48040000_strand_+_gene_NRNX1-seg-8, 9738 candidate probes\n",
      "--- 726 probes passed check_dic selection.\n",
      "finish in 0.174s, 19 probes kept.\n",
      "-- check region:8 2:48040000-48045000_strand_+_gene_NRNX1-seg-9, 9800 candidate probes\n",
      "--- 3456 probes passed check_dic selection.\n",
      "finish in 0.275s, 54 probes kept.\n",
      "-- check region:9 2:48045000-48050000_strand_+_gene_NRNX1-seg-10, 9918 candidate probes\n",
      "--- 4298 probes passed check_dic selection.\n",
      "finish in 0.301s, 66 probes kept.\n",
      "-- check region:10 2:48050000-48055000_strand_+_gene_NRNX1-seg-11, 8254 candidate probes\n",
      "--- 748 probes passed check_dic selection.\n",
      "finish in 0.157s, 20 probes kept.\n",
      "-- check region:11 2:48055000-48060000_strand_+_gene_NRNX1-seg-12, 8170 candidate probes\n",
      "--- 36 probes passed check_dic selection.\n",
      "finish in 0.131s, 4 probes kept.\n",
      "-- check region:12 2:48060000-48065000_strand_+_gene_NRNX1-seg-13, 9918 candidate probes\n",
      "--- 3122 probes passed check_dic selection.\n",
      "finish in 0.263s, 55 probes kept.\n",
      "-- check region:13 2:48065000-48070000_strand_+_gene_NRNX1-seg-14, 9896 candidate probes\n",
      "--- 5490 probes passed check_dic selection.\n",
      "finish in 0.346s, 67 probes kept.\n",
      "-- check region:14 2:48070000-48075000_strand_+_gene_NRNX1-seg-15, 9918 candidate probes\n",
      "--- 6386 probes passed check_dic selection.\n",
      "finish in 0.375s, 78 probes kept.\n",
      "-- check region:15 2:48075000-48080000_strand_+_gene_NRNX1-seg-16, 9916 candidate probes\n",
      "--- 6076 probes passed check_dic selection.\n",
      "finish in 0.367s, 70 probes kept.\n",
      "-- check region:16 2:48080000-48085000_strand_+_gene_NRNX1-seg-17, 9812 candidate probes\n",
      "--- 3482 probes passed check_dic selection.\n",
      "finish in 0.272s, 54 probes kept.\n",
      "-- check region:17 2:48085000-48090000_strand_+_gene_NRNX1-seg-18, 9870 candidate probes\n",
      "--- 1848 probes passed check_dic selection.\n",
      "finish in 0.216s, 36 probes kept.\n",
      "-- check region:18 2:48090000-48095000_strand_+_gene_NRNX1-seg-19, 9918 candidate probes\n",
      "--- 5350 probes passed check_dic selection.\n",
      "finish in 0.338s, 72 probes kept.\n",
      "-- check region:19 2:48095000-48100000_strand_+_gene_NRNX1-seg-20, 9902 candidate probes\n",
      "--- 2718 probes passed check_dic selection.\n",
      "finish in 0.245s, 45 probes kept.\n",
      "-- check region:20 2:48100000-48105000_strand_+_gene_NRNX1-seg-21, 9796 candidate probes\n",
      "--- 1922 probes passed check_dic selection.\n",
      "finish in 0.214s, 42 probes kept.\n",
      "-- check region:21 2:48105000-48110000_strand_+_gene_NRNX1-seg-22, 9818 candidate probes\n",
      "--- 5042 probes passed check_dic selection.\n",
      "finish in 0.327s, 65 probes kept.\n",
      "-- check region:22 2:48110000-48115000_strand_+_gene_NRNX1-seg-23, 9882 candidate probes\n",
      "--- 5582 probes passed check_dic selection.\n",
      "finish in 0.346s, 70 probes kept.\n",
      "-- check region:23 2:48115000-48120000_strand_+_gene_NRNX1-seg-24, 9874 candidate probes\n",
      "--- 4332 probes passed check_dic selection.\n",
      "finish in 0.304s, 57 probes kept.\n",
      "-- check region:24 2:48120000-48125000_strand_+_gene_NRNX1-seg-25, 9754 candidate probes\n",
      "--- 3076 probes passed check_dic selection.\n",
      "finish in 0.258s, 45 probes kept.\n",
      "-- check region:25 2:48125000-48130000_strand_+_gene_NRNX1-seg-26, 9772 candidate probes\n",
      "--- 3924 probes passed check_dic selection.\n",
      "finish in 0.289s, 55 probes kept.\n",
      "-- check region:26 2:48130000-48135000_strand_+_gene_NRNX1-seg-27, 9832 candidate probes\n",
      "--- 3278 probes passed check_dic selection.\n",
      "finish in 0.265s, 50 probes kept.\n",
      "-- check region:27 2:48135000-48140000_strand_+_gene_NRNX1-seg-28, 9918 candidate probes\n",
      "--- 3328 probes passed check_dic selection.\n",
      "finish in 0.267s, 50 probes kept.\n",
      "-- check region:28 2:48140000-48145000_strand_+_gene_NRNX1-seg-29, 9888 candidate probes\n",
      "--- 290 probes passed check_dic selection.\n",
      "finish in 0.164s, 11 probes kept.\n",
      "-- check region:29 2:48145000-48150000_strand_+_gene_NRNX1-seg-30, 9912 candidate probes\n",
      "--- 3112 probes passed check_dic selection.\n",
      "finish in 0.258s, 47 probes kept.\n",
      "-- check region:30 2:48150000-48155000_strand_+_gene_NRNX1-seg-31, 9754 candidate probes\n",
      "--- 2246 probes passed check_dic selection.\n",
      "finish in 0.224s, 41 probes kept.\n",
      "-- check region:31 2:48155000-48160000_strand_+_gene_NRNX1-seg-32, 9918 candidate probes\n",
      "--- 3084 probes passed check_dic selection.\n",
      "finish in 0.259s, 49 probes kept.\n",
      "-- check region:32 2:48160000-48165000_strand_+_gene_NRNX1-seg-33, 9918 candidate probes\n",
      "--- 1656 probes passed check_dic selection.\n",
      "finish in 0.208s, 34 probes kept.\n",
      "-- check region:33 2:48165000-48170000_strand_+_gene_NRNX1-seg-34, 9846 candidate probes\n",
      "--- 2692 probes passed check_dic selection.\n",
      "finish in 0.248s, 43 probes kept.\n",
      "-- check region:34 2:48170000-48175000_strand_+_gene_NRNX1-seg-35, 9880 candidate probes\n",
      "--- 2646 probes passed check_dic selection.\n",
      "finish in 0.246s, 42 probes kept.\n",
      "-- check region:35 2:48175000-48180000_strand_+_gene_NRNX1-seg-36, 9854 candidate probes\n",
      "--- 2762 probes passed check_dic selection.\n",
      "finish in 0.252s, 53 probes kept.\n",
      "-- check region:36 2:48180000-48185000_strand_+_gene_NRNX1-seg-37, 9876 candidate probes\n",
      "--- 3188 probes passed check_dic selection.\n",
      "finish in 0.266s, 44 probes kept.\n",
      "-- check region:37 2:48185000-48190000_strand_+_gene_NRNX1-seg-38, 9918 candidate probes\n",
      "--- 2684 probes passed check_dic selection.\n",
      "finish in 0.246s, 48 probes kept.\n",
      "-- check region:38 2:48190000-48195000_strand_+_gene_NRNX1-seg-39, 9826 candidate probes\n",
      "--- 2958 probes passed check_dic selection.\n",
      "finish in 0.255s, 47 probes kept.\n",
      "-- check region:39 2:48195000-48200000_strand_+_gene_NRNX1-seg-40, 9900 candidate probes\n",
      "--- 5848 probes passed check_dic selection.\n",
      "finish in 0.357s, 70 probes kept.\n",
      "-- check region:40 2:48200000-48205000_strand_+_gene_NRNX1-seg-41, 9918 candidate probes\n",
      "--- 4014 probes passed check_dic selection.\n",
      "finish in 0.294s, 55 probes kept.\n",
      "-- check region:41 2:48205000-48210000_strand_+_gene_NRNX1-seg-42, 9868 candidate probes\n",
      "--- 4950 probes passed check_dic selection.\n",
      "finish in 0.325s, 61 probes kept.\n",
      "-- check region:42 2:48210000-48215000_strand_+_gene_NRNX1-seg-43, 9904 candidate probes\n",
      "--- 5212 probes passed check_dic selection.\n",
      "finish in 0.335s, 69 probes kept.\n",
      "-- check region:43 2:48215000-48220000_strand_+_gene_NRNX1-seg-44, 9838 candidate probes\n",
      "--- 3486 probes passed check_dic selection.\n",
      "finish in 0.273s, 54 probes kept.\n",
      "-- check region:44 2:48220000-48225000_strand_+_gene_NRNX1-seg-45, 9806 candidate probes\n",
      "--- 3828 probes passed check_dic selection.\n",
      "finish in 0.285s, 50 probes kept.\n",
      "-- check region:45 2:48225000-48230000_strand_+_gene_NRNX1-seg-46, 9918 candidate probes\n",
      "--- 2612 probes passed check_dic selection.\n",
      "finish in 0.240s, 42 probes kept.\n",
      "-- check region:46 2:48230000-48235000_strand_+_gene_NRNX1-seg-47, 9372 candidate probes\n",
      "--- 2814 probes passed check_dic selection.\n",
      "finish in 0.245s, 41 probes kept.\n",
      "-- check region:47 2:48235000-48240000_strand_+_gene_NRNX1-seg-48, 9770 candidate probes\n",
      "--- 2642 probes passed check_dic selection.\n",
      "finish in 0.241s, 40 probes kept.\n",
      "-- check region:48 2:48240000-48245000_strand_+_gene_NRNX1-seg-49, 9906 candidate probes\n",
      "--- 3666 probes passed check_dic selection.\n",
      "finish in 0.278s, 58 probes kept.\n",
      "-- check region:49 2:48245000-48250000_strand_+_gene_NRNX1-seg-50, 9916 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 3634 probes passed check_dic selection.\n",
      "finish in 0.281s, 50 probes kept.\n",
      "-- check region:50 2:48250000-48255000_strand_+_gene_NRNX1-seg-51, 9826 candidate probes\n",
      "--- 3104 probes passed check_dic selection.\n",
      "finish in 0.260s, 52 probes kept.\n",
      "-- check region:51 2:48255000-48260000_strand_+_gene_NRNX1-seg-52, 9890 candidate probes\n",
      "--- 4320 probes passed check_dic selection.\n",
      "finish in 0.303s, 59 probes kept.\n",
      "-- check region:52 2:48260000-48265000_strand_+_gene_NRNX1-seg-53, 9820 candidate probes\n",
      "--- 3716 probes passed check_dic selection.\n",
      "finish in 0.279s, 53 probes kept.\n",
      "-- check region:53 2:48265000-48270000_strand_+_gene_NRNX1-seg-54, 9816 candidate probes\n",
      "--- 4878 probes passed check_dic selection.\n",
      "finish in 0.322s, 66 probes kept.\n",
      "-- check region:54 2:48270000-48275000_strand_+_gene_NRNX1-seg-55, 9752 candidate probes\n",
      "--- 3782 probes passed check_dic selection.\n",
      "finish in 0.278s, 57 probes kept.\n",
      "-- check region:55 2:48275000-48280000_strand_+_gene_NRNX1-seg-56, 9846 candidate probes\n",
      "--- 2852 probes passed check_dic selection.\n",
      "finish in 0.248s, 44 probes kept.\n",
      "-- check region:56 2:48280000-48285000_strand_+_gene_NRNX1-seg-57, 9876 candidate probes\n",
      "--- 3828 probes passed check_dic selection.\n",
      "finish in 0.282s, 54 probes kept.\n",
      "-- check region:57 2:48285000-48290000_strand_+_gene_NRNX1-seg-58, 9894 candidate probes\n",
      "--- 2860 probes passed check_dic selection.\n",
      "finish in 0.252s, 48 probes kept.\n",
      "-- check region:58 2:48290000-48295000_strand_+_gene_NRNX1-seg-59, 9778 candidate probes\n",
      "--- 3214 probes passed check_dic selection.\n",
      "finish in 0.263s, 48 probes kept.\n",
      "-- check region:59 2:48295000-48300000_strand_+_gene_NRNX1-seg-60, 9434 candidate probes\n",
      "--- 1482 probes passed check_dic selection.\n",
      "finish in 0.199s, 23 probes kept.\n",
      "-- check region:60 2:48300000-48305000_strand_+_gene_NRNX1-seg-61, 9646 candidate probes\n",
      "--- 814 probes passed check_dic selection.\n",
      "finish in 0.183s, 17 probes kept.\n",
      "-- check region:61 2:48305000-48310000_strand_+_gene_NRNX1-seg-62, 9648 candidate probes\n",
      "--- 3174 probes passed check_dic selection.\n",
      "finish in 0.261s, 43 probes kept.\n",
      "-- check region:62 2:48310000-48315000_strand_+_gene_NRNX1-seg-63, 9918 candidate probes\n",
      "--- 4082 probes passed check_dic selection.\n",
      "finish in 0.291s, 57 probes kept.\n",
      "-- check region:63 2:48315000-48320000_strand_+_gene_NRNX1-seg-64, 9914 candidate probes\n",
      "--- 3284 probes passed check_dic selection.\n",
      "finish in 0.265s, 44 probes kept.\n",
      "-- check region:64 2:48320000-48325000_strand_+_gene_NRNX1-seg-65, 9738 candidate probes\n",
      "--- 1182 probes passed check_dic selection.\n",
      "finish in 0.189s, 24 probes kept.\n",
      "-- check region:65 2:48325000-48330000_strand_+_gene_NRNX1-seg-66, 9882 candidate probes\n",
      "--- 2480 probes passed check_dic selection.\n",
      "finish in 0.236s, 45 probes kept.\n",
      "-- check region:66 2:48330000-48335000_strand_+_gene_NRNX1-seg-67, 9918 candidate probes\n",
      "--- 2782 probes passed check_dic selection.\n",
      "finish in 0.245s, 51 probes kept.\n",
      "-- check region:67 2:48335000-48340000_strand_+_gene_NRNX1-seg-68, 9796 candidate probes\n",
      "--- 700 probes passed check_dic selection.\n",
      "finish in 0.176s, 24 probes kept.\n",
      "-- check region:68 2:48340000-48345000_strand_+_gene_NRNX1-seg-69, 9918 candidate probes\n",
      "--- 2448 probes passed check_dic selection.\n",
      "finish in 0.233s, 45 probes kept.\n",
      "-- check region:69 2:48345000-48350000_strand_+_gene_NRNX1-seg-70, 9766 candidate probes\n",
      "--- 2738 probes passed check_dic selection.\n",
      "finish in 0.243s, 42 probes kept.\n",
      "-- check region:70 2:48350000-48355000_strand_+_gene_NRNX1-seg-71, 9768 candidate probes\n",
      "--- 3288 probes passed check_dic selection.\n",
      "finish in 0.265s, 54 probes kept.\n",
      "-- check region:71 2:48355000-48360000_strand_+_gene_NRNX1-seg-72, 9906 candidate probes\n",
      "--- 2198 probes passed check_dic selection.\n",
      "finish in 0.227s, 44 probes kept.\n",
      "-- check region:72 2:48360000-48365000_strand_+_gene_NRNX1-seg-73, 9918 candidate probes\n",
      "--- 1170 probes passed check_dic selection.\n",
      "finish in 0.192s, 24 probes kept.\n",
      "-- check region:73 2:48365000-48370000_strand_+_gene_NRNX1-seg-74, 9734 candidate probes\n",
      "--- 2020 probes passed check_dic selection.\n",
      "finish in 0.220s, 36 probes kept.\n",
      "-- check region:74 2:48370000-48375000_strand_+_gene_NRNX1-seg-75, 9916 candidate probes\n",
      "--- 1388 probes passed check_dic selection.\n",
      "finish in 0.197s, 28 probes kept.\n",
      "-- check region:75 2:48375000-48380000_strand_+_gene_NRNX1-seg-76, 9918 candidate probes\n",
      "--- 2688 probes passed check_dic selection.\n",
      "finish in 0.241s, 46 probes kept.\n",
      "-- check region:76 2:48380000-48385000_strand_+_gene_NRNX1-seg-77, 9800 candidate probes\n",
      "--- 1406 probes passed check_dic selection.\n",
      "finish in 0.200s, 34 probes kept.\n",
      "-- check region:77 2:48385000-48390000_strand_+_gene_NRNX1-seg-78, 9904 candidate probes\n",
      "--- 214 probes passed check_dic selection.\n",
      "finish in 0.166s, 13 probes kept.\n",
      "-- check region:78 2:48390000-48395000_strand_+_gene_NRNX1-seg-79, 9842 candidate probes\n",
      "--- 1286 probes passed check_dic selection.\n",
      "finish in 0.195s, 32 probes kept.\n",
      "-- check region:79 2:48395000-48400000_strand_+_gene_NRNX1-seg-80, 9918 candidate probes\n",
      "--- 3642 probes passed check_dic selection.\n",
      "finish in 0.278s, 51 probes kept.\n",
      "-- check region:80 2:48400000-48405000_strand_+_gene_NRNX1-seg-81, 9798 candidate probes\n",
      "--- 1946 probes passed check_dic selection.\n",
      "finish in 0.219s, 37 probes kept.\n",
      "-- check region:81 2:48405000-48410000_strand_+_gene_NRNX1-seg-82, 9798 candidate probes\n",
      "--- 3464 probes passed check_dic selection.\n",
      "finish in 0.271s, 48 probes kept.\n",
      "-- check region:82 2:48410000-48415000_strand_+_gene_NRNX1-seg-83, 9894 candidate probes\n",
      "--- 4984 probes passed check_dic selection.\n",
      "finish in 0.325s, 63 probes kept.\n",
      "-- check region:83 2:48415000-48420000_strand_+_gene_NRNX1-seg-84, 9872 candidate probes\n",
      "--- 5998 probes passed check_dic selection.\n",
      "finish in 0.363s, 69 probes kept.\n",
      "-- check region:84 2:48420000-48425000_strand_+_gene_NRNX1-seg-85, 9900 candidate probes\n",
      "--- 3702 probes passed check_dic selection.\n",
      "finish in 0.284s, 55 probes kept.\n",
      "-- check region:85 2:48425000-48430000_strand_+_gene_NRNX1-seg-86, 9736 candidate probes\n",
      "--- 3604 probes passed check_dic selection.\n",
      "finish in 0.278s, 46 probes kept.\n",
      "-- check region:86 2:48430000-48435000_strand_+_gene_NRNX1-seg-87, 9918 candidate probes\n",
      "--- 2624 probes passed check_dic selection.\n",
      "finish in 0.243s, 50 probes kept.\n",
      "-- check region:87 2:48435000-48440000_strand_+_gene_NRNX1-seg-88, 9858 candidate probes\n",
      "--- 3294 probes passed check_dic selection.\n",
      "finish in 0.265s, 47 probes kept.\n",
      "-- check region:88 2:48440000-48445000_strand_+_gene_NRNX1-seg-89, 9918 candidate probes\n",
      "--- 6222 probes passed check_dic selection.\n",
      "finish in 0.368s, 77 probes kept.\n",
      "-- check region:89 2:48445000-48450000_strand_+_gene_NRNX1-seg-90, 9850 candidate probes\n",
      "--- 3472 probes passed check_dic selection.\n",
      "finish in 0.270s, 52 probes kept.\n",
      "-- check region:90 2:48450000-48455000_strand_+_gene_NRNX1-seg-91, 9914 candidate probes\n",
      "--- 3340 probes passed check_dic selection.\n",
      "finish in 0.267s, 50 probes kept.\n",
      "-- check region:91 2:48455000-48460000_strand_+_gene_NRNX1-seg-92, 9628 candidate probes\n",
      "--- 2492 probes passed check_dic selection.\n",
      "finish in 0.233s, 38 probes kept.\n",
      "-- check region:92 2:48460000-48465000_strand_+_gene_NRNX1-seg-93, 9918 candidate probes\n",
      "--- 5794 probes passed check_dic selection.\n",
      "finish in 0.354s, 73 probes kept.\n",
      "-- check region:93 2:48465000-48470000_strand_+_gene_NRNX1-seg-94, 9792 candidate probes\n",
      "--- 3142 probes passed check_dic selection.\n",
      "finish in 0.259s, 45 probes kept.\n",
      "-- check region:94 2:48470000-48475000_strand_+_gene_NRNX1-seg-95, 9772 candidate probes\n",
      "--- 1758 probes passed check_dic selection.\n",
      "finish in 0.211s, 29 probes kept.\n",
      "-- check region:95 2:48475000-48480000_strand_+_gene_NRNX1-seg-96, 9878 candidate probes\n",
      "--- 2682 probes passed check_dic selection.\n",
      "finish in 0.247s, 44 probes kept.\n",
      "-- check region:96 2:48480000-48485000_strand_+_gene_NRNX1-seg-97, 9858 candidate probes\n",
      "--- 2726 probes passed check_dic selection.\n",
      "finish in 0.250s, 44 probes kept.\n",
      "-- check region:97 2:48485000-48490000_strand_+_gene_NRNX1-seg-98, 9846 candidate probes\n",
      "--- 1854 probes passed check_dic selection.\n",
      "finish in 0.216s, 37 probes kept.\n",
      "-- check region:98 2:48490000-48495000_strand_+_gene_NRNX1-seg-99, 9900 candidate probes\n",
      "--- 1602 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 0.208s, 32 probes kept.\n",
      "-- check region:99 2:48495000-48500000_strand_+_gene_NRNX1-seg-100, 9824 candidate probes\n",
      "--- 2100 probes passed check_dic selection.\n",
      "finish in 0.226s, 44 probes kept.\n",
      "-- check region:100 2:48500000-48505000_strand_+_gene_NRNX1-seg-101, 9802 candidate probes\n",
      "--- 2574 probes passed check_dic selection.\n",
      "finish in 0.240s, 42 probes kept.\n",
      "-- check region:101 2:48505000-48510000_strand_+_gene_NRNX1-seg-102, 9722 candidate probes\n",
      "--- 3534 probes passed check_dic selection.\n",
      "finish in 0.272s, 49 probes kept.\n",
      "-- check region:102 2:48510000-48515000_strand_+_gene_NRNX1-seg-103, 9868 candidate probes\n",
      "--- 3238 probes passed check_dic selection.\n",
      "finish in 0.264s, 52 probes kept.\n",
      "-- check region:103 2:48515000-48520000_strand_+_gene_NRNX1-seg-104, 9856 candidate probes\n",
      "--- 4978 probes passed check_dic selection.\n",
      "finish in 0.327s, 58 probes kept.\n",
      "-- check region:104 2:48520000-48525000_strand_+_gene_NRNX1-seg-105, 9238 candidate probes\n",
      "--- 1242 probes passed check_dic selection.\n",
      "finish in 0.191s, 23 probes kept.\n",
      "-- check region:105 2:48525000-48530000_strand_+_gene_NRNX1-seg-106, 9872 candidate probes\n",
      "--- 4998 probes passed check_dic selection.\n",
      "finish in 0.326s, 69 probes kept.\n",
      "-- check region:106 2:48530000-48535000_strand_+_gene_NRNX1-seg-107, 9842 candidate probes\n",
      "--- 3624 probes passed check_dic selection.\n",
      "finish in 0.278s, 46 probes kept.\n",
      "-- check region:107 2:48535000-48540000_strand_+_gene_NRNX1-seg-108, 9880 candidate probes\n",
      "--- 1874 probes passed check_dic selection.\n",
      "finish in 0.218s, 32 probes kept.\n",
      "-- check region:108 2:48540000-48545000_strand_+_gene_NRNX1-seg-109, 9806 candidate probes\n",
      "--- 2632 probes passed check_dic selection.\n",
      "finish in 0.245s, 42 probes kept.\n",
      "-- check region:109 2:48545000-48550000_strand_+_gene_NRNX1-seg-110, 9874 candidate probes\n",
      "--- 5636 probes passed check_dic selection.\n",
      "finish in 0.349s, 66 probes kept.\n",
      "-- check region:110 2:48550000-48555000_strand_+_gene_NRNX1-seg-111, 9590 candidate probes\n",
      "--- 3938 probes passed check_dic selection.\n",
      "finish in 0.287s, 47 probes kept.\n",
      "-- check region:111 2:48555000-48560000_strand_+_gene_NRNX1-seg-112, 7992 candidate probes\n",
      "--- 1744 probes passed check_dic selection.\n",
      "finish in 0.184s, 25 probes kept.\n",
      "-- check region:112 2:48560000-48565000_strand_+_gene_NRNX1-seg-113, 9634 candidate probes\n",
      "--- 5224 probes passed check_dic selection.\n",
      "finish in 0.333s, 62 probes kept.\n",
      "-- check region:113 2:48565000-48570000_strand_+_gene_NRNX1-seg-114, 9750 candidate probes\n",
      "--- 4092 probes passed check_dic selection.\n",
      "finish in 0.294s, 57 probes kept.\n",
      "-- check region:114 2:48570000-48575000_strand_+_gene_NRNX1-seg-115, 9764 candidate probes\n",
      "--- 3734 probes passed check_dic selection.\n",
      "finish in 0.283s, 50 probes kept.\n",
      "-- check region:115 2:48575000-48580000_strand_+_gene_NRNX1-seg-116, 9682 candidate probes\n",
      "--- 214 probes passed check_dic selection.\n",
      "finish in 0.162s, 8 probes kept.\n",
      "-- check region:116 2:48580000-48585000_strand_+_gene_NRNX1-seg-117, 9918 candidate probes\n",
      "--- 4268 probes passed check_dic selection.\n",
      "finish in 0.300s, 57 probes kept.\n",
      "-- check region:117 2:48585000-48590000_strand_+_gene_NRNX1-seg-118, 9898 candidate probes\n",
      "--- 4908 probes passed check_dic selection.\n",
      "finish in 0.323s, 58 probes kept.\n",
      "-- check region:118 2:48590000-48595000_strand_+_gene_NRNX1-seg-119, 9862 candidate probes\n",
      "--- 3262 probes passed check_dic selection.\n",
      "finish in 0.264s, 53 probes kept.\n",
      "-- check region:119 2:48595000-48600000_strand_+_gene_NRNX1-seg-120, 9852 candidate probes\n",
      "--- 4150 probes passed check_dic selection.\n",
      "finish in 0.294s, 56 probes kept.\n",
      "-- check region:120 2:48600000-48605000_strand_+_gene_NRNX1-seg-121, 9918 candidate probes\n",
      "--- 3982 probes passed check_dic selection.\n",
      "finish in 0.292s, 65 probes kept.\n",
      "-- check region:121 2:48605000-48610000_strand_+_gene_NRNX1-seg-122, 9872 candidate probes\n",
      "--- 3400 probes passed check_dic selection.\n",
      "finish in 0.272s, 49 probes kept.\n",
      "-- check region:122 2:48610000-48615000_strand_+_gene_NRNX1-seg-123, 9676 candidate probes\n",
      "--- 2082 probes passed check_dic selection.\n",
      "finish in 0.223s, 37 probes kept.\n",
      "-- check region:123 2:48615000-48620000_strand_+_gene_NRNX1-seg-124, 9764 candidate probes\n",
      "--- 4152 probes passed check_dic selection.\n",
      "finish in 0.295s, 59 probes kept.\n",
      "-- check region:124 2:48620000-48625000_strand_+_gene_NRNX1-seg-125, 9868 candidate probes\n",
      "--- 1750 probes passed check_dic selection.\n",
      "finish in 0.210s, 36 probes kept.\n",
      "-- check region:125 2:48625000-48630000_strand_+_gene_NRNX1-seg-126, 9876 candidate probes\n",
      "--- 884 probes passed check_dic selection.\n",
      "finish in 0.185s, 27 probes kept.\n",
      "-- check region:126 2:48630000-48635000_strand_+_gene_NRNX1-seg-127, 9850 candidate probes\n",
      "--- 1514 probes passed check_dic selection.\n",
      "finish in 0.204s, 38 probes kept.\n",
      "-- check region:127 2:48635000-48640000_strand_+_gene_NRNX1-seg-128, 9900 candidate probes\n",
      "--- 1618 probes passed check_dic selection.\n",
      "finish in 0.209s, 36 probes kept.\n",
      "-- check region:128 2:48640000-48645000_strand_+_gene_NRNX1-seg-129, 9890 candidate probes\n",
      "--- 1502 probes passed check_dic selection.\n",
      "finish in 0.202s, 30 probes kept.\n",
      "-- check region:129 2:48645000-48650000_strand_+_gene_NRNX1-seg-130, 9774 candidate probes\n",
      "--- 3132 probes passed check_dic selection.\n",
      "finish in 0.255s, 46 probes kept.\n",
      "-- check region:130 2:48650000-48655000_strand_+_gene_NRNX1-seg-131, 9840 candidate probes\n",
      "--- 2036 probes passed check_dic selection.\n",
      "finish in 0.220s, 40 probes kept.\n",
      "-- check region:131 2:48655000-48660000_strand_+_gene_NRNX1-seg-132, 9918 candidate probes\n",
      "--- 518 probes passed check_dic selection.\n",
      "finish in 0.167s, 19 probes kept.\n",
      "-- check region:132 2:48660000-48665000_strand_+_gene_NRNX1-seg-133, 9916 candidate probes\n",
      "--- 700 probes passed check_dic selection.\n",
      "finish in 0.175s, 24 probes kept.\n",
      "-- check region:133 2:48665000-48670000_strand_+_gene_NRNX1-seg-134, 9876 candidate probes\n",
      "--- 1650 probes passed check_dic selection.\n",
      "finish in 0.206s, 28 probes kept.\n",
      "-- check region:134 2:48670000-48675000_strand_+_gene_NRNX1-seg-135, 9860 candidate probes\n",
      "--- 1996 probes passed check_dic selection.\n",
      "finish in 0.221s, 42 probes kept.\n",
      "-- check region:135 2:48675000-48680000_strand_+_gene_NRNX1-seg-136, 9918 candidate probes\n",
      "--- 2924 probes passed check_dic selection.\n",
      "finish in 0.249s, 51 probes kept.\n",
      "-- check region:136 2:48680000-48685000_strand_+_gene_NRNX1-seg-137, 9918 candidate probes\n",
      "--- 3000 probes passed check_dic selection.\n",
      "finish in 0.256s, 50 probes kept.\n",
      "-- check region:137 2:48685000-48690000_strand_+_gene_NRNX1-seg-138, 9904 candidate probes\n",
      "--- 3418 probes passed check_dic selection.\n",
      "finish in 0.267s, 51 probes kept.\n",
      "-- check region:138 2:48690000-48695000_strand_+_gene_NRNX1-seg-139, 9900 candidate probes\n",
      "--- 4348 probes passed check_dic selection.\n",
      "finish in 0.303s, 67 probes kept.\n",
      "-- check region:139 2:48695000-48700000_strand_+_gene_NRNX1-seg-140, 9886 candidate probes\n",
      "--- 3924 probes passed check_dic selection.\n",
      "finish in 0.289s, 61 probes kept.\n",
      "-- check region:140 2:48700000-48705000_strand_+_gene_NRNX1-seg-141, 8966 candidate probes\n",
      "--- 3218 probes passed check_dic selection.\n",
      "finish in 0.252s, 39 probes kept.\n",
      "-- check region:141 2:48705000-48710000_strand_+_gene_NRNX1-seg-142, 9560 candidate probes\n",
      "--- 2422 probes passed check_dic selection.\n",
      "finish in 0.238s, 31 probes kept.\n",
      "-- check region:142 2:48710000-48715000_strand_+_gene_NRNX1-seg-143, 9918 candidate probes\n",
      "--- 5156 probes passed check_dic selection.\n",
      "finish in 0.331s, 71 probes kept.\n",
      "-- check region:143 2:48715000-48720000_strand_+_gene_NRNX1-seg-144, 9890 candidate probes\n",
      "--- 5000 probes passed check_dic selection.\n",
      "finish in 0.325s, 69 probes kept.\n",
      "-- check region:144 2:48720000-48725000_strand_+_gene_NRNX1-seg-145, 9880 candidate probes\n",
      "--- 4772 probes passed check_dic selection.\n",
      "finish in 0.316s, 69 probes kept.\n",
      "-- check region:145 2:48725000-48730000_strand_+_gene_NRNX1-seg-146, 9918 candidate probes\n",
      "--- 5458 probes passed check_dic selection.\n",
      "finish in 0.343s, 69 probes kept.\n",
      "-- check region:146 2:48730000-48735000_strand_+_gene_NRNX1-seg-147, 9918 candidate probes\n",
      "--- 5392 probes passed check_dic selection.\n",
      "finish in 0.338s, 68 probes kept.\n",
      "-- check region:147 2:48735000-48740000_strand_+_gene_NRNX1-seg-148, 9184 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 4166 probes passed check_dic selection.\n",
      "finish in 0.285s, 57 probes kept.\n",
      "-- check region:148 2:48740000-48745000_strand_+_gene_NRNX1-seg-149, 5164 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.084s, 0 probes kept.\n",
      "-- check region:149 2:48745000-48750000_strand_+_gene_NRNX1-seg-150, 9144 candidate probes\n",
      "--- 4686 probes passed check_dic selection.\n",
      "finish in 0.304s, 59 probes kept.\n",
      "-- check region:150 2:48750000-48755000_strand_+_gene_NRNX1-seg-151, 9918 candidate probes\n",
      "--- 4826 probes passed check_dic selection.\n",
      "finish in 0.320s, 69 probes kept.\n",
      "-- check region:151 2:48755000-48760000_strand_+_gene_NRNX1-seg-152, 9866 candidate probes\n",
      "--- 4514 probes passed check_dic selection.\n",
      "finish in 0.306s, 66 probes kept.\n",
      "-- check region:152 2:48760000-48765000_strand_+_gene_NRNX1-seg-153, 9160 candidate probes\n",
      "--- 1022 probes passed check_dic selection.\n",
      "finish in 0.179s, 17 probes kept.\n",
      "-- check region:153 2:48765000-48770000_strand_+_gene_NRNX1-seg-154, 9600 candidate probes\n",
      "--- 2558 probes passed check_dic selection.\n",
      "finish in 0.241s, 35 probes kept.\n",
      "-- check region:154 2:48770000-48775000_strand_+_gene_NRNX1-seg-155, 9872 candidate probes\n",
      "--- 4156 probes passed check_dic selection.\n",
      "finish in 0.295s, 59 probes kept.\n",
      "-- check region:155 2:48775000-48780000_strand_+_gene_NRNX1-seg-156, 9718 candidate probes\n",
      "--- 5212 probes passed check_dic selection.\n",
      "finish in 0.329s, 66 probes kept.\n",
      "-- check region:156 2:48780000-48785000_strand_+_gene_NRNX1-seg-157, 9918 candidate probes\n",
      "--- 3146 probes passed check_dic selection.\n",
      "finish in 0.263s, 49 probes kept.\n",
      "-- check region:157 2:48785000-48790000_strand_+_gene_NRNX1-seg-158, 9918 candidate probes\n",
      "--- 5094 probes passed check_dic selection.\n",
      "finish in 0.328s, 67 probes kept.\n",
      "-- check region:158 2:48790000-48795000_strand_+_gene_NRNX1-seg-159, 9888 candidate probes\n",
      "--- 4206 probes passed check_dic selection.\n",
      "finish in 0.302s, 58 probes kept.\n",
      "-- check region:159 2:48795000-48800000_strand_+_gene_NRNX1-seg-160, 9918 candidate probes\n",
      "--- 4256 probes passed check_dic selection.\n",
      "finish in 0.298s, 64 probes kept.\n",
      "-- check region:160 2:48800000-48805000_strand_+_gene_NRNX1-seg-161, 9902 candidate probes\n",
      "--- 5486 probes passed check_dic selection.\n",
      "finish in 0.344s, 75 probes kept.\n",
      "-- check region:161 2:48805000-48810000_strand_+_gene_NRNX1-seg-162, 9918 candidate probes\n",
      "--- 4364 probes passed check_dic selection.\n",
      "finish in 0.305s, 66 probes kept.\n",
      "-- check region:162 2:48810000-48815000_strand_+_gene_NRNX1-seg-163, 9870 candidate probes\n",
      "--- 1794 probes passed check_dic selection.\n",
      "finish in 0.218s, 34 probes kept.\n",
      "-- check region:163 2:48815000-48820000_strand_+_gene_NRNX1-seg-164, 9918 candidate probes\n",
      "--- 3738 probes passed check_dic selection.\n",
      "finish in 0.280s, 56 probes kept.\n",
      "-- check region:164 2:48820000-48825000_strand_+_gene_NRNX1-seg-165, 9918 candidate probes\n",
      "--- 6126 probes passed check_dic selection.\n",
      "finish in 0.367s, 83 probes kept.\n",
      "-- check region:165 2:48825000-48830000_strand_+_gene_NRNX1-seg-166, 9918 candidate probes\n",
      "--- 2294 probes passed check_dic selection.\n",
      "finish in 0.237s, 51 probes kept.\n",
      "-- check region:166 2:48830000-48835000_strand_+_gene_NRNX1-seg-167, 9884 candidate probes\n",
      "--- 72 probes passed check_dic selection.\n",
      "finish in 0.161s, 6 probes kept.\n",
      "-- check region:167 2:48835000-48840000_strand_+_gene_NRNX1-seg-168, 9918 candidate probes\n",
      "--- 178 probes passed check_dic selection.\n",
      "finish in 0.158s, 8 probes kept.\n",
      "-- check region:168 2:48840000-48845000_strand_+_gene_NRNX1-seg-169, 9918 candidate probes\n",
      "--- 4246 probes passed check_dic selection.\n",
      "finish in 0.305s, 62 probes kept.\n",
      "-- check region:169 2:48845000-48850000_strand_+_gene_NRNX1-seg-170, 9914 candidate probes\n",
      "--- 5980 probes passed check_dic selection.\n",
      "finish in 0.362s, 71 probes kept.\n",
      "-- check region:170 2:48850000-48855000_strand_+_gene_NRNX1-seg-171, 9754 candidate probes\n",
      "--- 4786 probes passed check_dic selection.\n",
      "finish in 0.319s, 63 probes kept.\n",
      "-- check region:171 2:48855000-48860000_strand_+_gene_NRNX1-seg-172, 9918 candidate probes\n",
      "--- 5552 probes passed check_dic selection.\n",
      "finish in 0.348s, 76 probes kept.\n",
      "-- check region:172 2:48860000-48865000_strand_+_gene_NRNX1-seg-173, 9918 candidate probes\n",
      "--- 5326 probes passed check_dic selection.\n",
      "finish in 0.339s, 68 probes kept.\n",
      "-- check region:173 2:48865000-48870000_strand_+_gene_NRNX1-seg-174, 9918 candidate probes\n",
      "--- 4018 probes passed check_dic selection.\n",
      "finish in 0.291s, 58 probes kept.\n",
      "-- check region:174 2:48870000-48875000_strand_+_gene_NRNX1-seg-175, 9918 candidate probes\n",
      "--- 5980 probes passed check_dic selection.\n",
      "finish in 0.362s, 71 probes kept.\n",
      "-- check region:175 2:48875000-48880000_strand_+_gene_NRNX1-seg-176, 9840 candidate probes\n",
      "--- 5594 probes passed check_dic selection.\n",
      "finish in 0.345s, 70 probes kept.\n",
      "-- check region:176 2:48880000-48885000_strand_+_gene_NRNX1-seg-177, 9874 candidate probes\n",
      "--- 4982 probes passed check_dic selection.\n",
      "finish in 0.326s, 64 probes kept.\n",
      "-- check region:177 2:48885000-48890000_strand_+_gene_NRNX1-seg-178, 9918 candidate probes\n",
      "--- 5362 probes passed check_dic selection.\n",
      "finish in 0.339s, 66 probes kept.\n",
      "-- check region:178 2:48890000-48895000_strand_+_gene_NRNX1-seg-179, 9896 candidate probes\n",
      "--- 5354 probes passed check_dic selection.\n",
      "finish in 0.335s, 71 probes kept.\n",
      "-- check region:179 2:48895000-48900000_strand_+_gene_NRNX1-seg-180, 9756 candidate probes\n",
      "--- 5326 probes passed check_dic selection.\n",
      "finish in 0.336s, 69 probes kept.\n",
      "-- check region:180 2:48900000-48905000_strand_+_gene_NRNX1-seg-181, 9888 candidate probes\n",
      "--- 6184 probes passed check_dic selection.\n",
      "finish in 0.371s, 71 probes kept.\n",
      "-- check region:181 2:48905000-48910000_strand_+_gene_NRNX1-seg-182, 9896 candidate probes\n",
      "--- 4942 probes passed check_dic selection.\n",
      "finish in 0.324s, 71 probes kept.\n",
      "-- check region:182 2:48910000-48915000_strand_+_gene_NRNX1-seg-183, 9918 candidate probes\n",
      "--- 3098 probes passed check_dic selection.\n",
      "finish in 0.259s, 56 probes kept.\n",
      "-- check region:183 2:48915000-48920000_strand_+_gene_NRNX1-seg-184, 8950 candidate probes\n",
      "--- 3468 probes passed check_dic selection.\n",
      "finish in 0.262s, 48 probes kept.\n",
      "-- check region:184 2:48920000-48925000_strand_+_gene_NRNX1-seg-185, 9012 candidate probes\n",
      "--- 2734 probes passed check_dic selection.\n",
      "finish in 0.236s, 43 probes kept.\n",
      "-- check region:185 2:48925000-48930000_strand_+_gene_NRNX1-seg-186, 9554 candidate probes\n",
      "--- 4362 probes passed check_dic selection.\n",
      "finish in 0.300s, 54 probes kept.\n",
      "-- check region:186 2:48930000-48935000_strand_+_gene_NRNX1-seg-187, 9738 candidate probes\n",
      "--- 4604 probes passed check_dic selection.\n",
      "finish in 0.311s, 65 probes kept.\n",
      "-- check region:187 2:48935000-48940000_strand_+_gene_NRNX1-seg-188, 9840 candidate probes\n",
      "--- 4650 probes passed check_dic selection.\n",
      "finish in 0.315s, 59 probes kept.\n",
      "-- check region:188 2:48940000-48945000_strand_+_gene_NRNX1-seg-189, 9918 candidate probes\n",
      "--- 4564 probes passed check_dic selection.\n",
      "finish in 0.312s, 63 probes kept.\n",
      "-- check region:189 2:48945000-48950000_strand_+_gene_NRNX1-seg-190, 9918 candidate probes\n",
      "--- 4418 probes passed check_dic selection.\n",
      "finish in 0.306s, 64 probes kept.\n",
      "-- check region:190 2:48950000-48955000_strand_+_gene_NRNX1-seg-191, 9868 candidate probes\n",
      "--- 3766 probes passed check_dic selection.\n",
      "finish in 0.283s, 52 probes kept.\n",
      "-- check region:191 2:48955000-48960000_strand_+_gene_NRNX1-seg-192, 9918 candidate probes\n",
      "--- 6420 probes passed check_dic selection.\n",
      "finish in 0.376s, 80 probes kept.\n",
      "-- check region:192 2:48960000-48965000_strand_+_gene_NRNX1-seg-193, 9918 candidate probes\n",
      "--- 5904 probes passed check_dic selection.\n",
      "finish in 0.359s, 71 probes kept.\n",
      "-- check region:193 2:48965000-48970000_strand_+_gene_NRNX1-seg-194, 9918 candidate probes\n",
      "--- 5802 probes passed check_dic selection.\n",
      "finish in 0.355s, 67 probes kept.\n",
      "-- check region:194 2:48970000-48975000_strand_+_gene_NRNX1-seg-195, 9918 candidate probes\n",
      "--- 4970 probes passed check_dic selection.\n",
      "finish in 0.323s, 72 probes kept.\n",
      "-- check region:195 2:48975000-48980000_strand_+_gene_NRNX1-seg-196, 9632 candidate probes\n",
      "--- 3804 probes passed check_dic selection.\n",
      "finish in 0.283s, 49 probes kept.\n",
      "-- check region:196 2:48980000-48985000_strand_+_gene_NRNX1-seg-197, 9918 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 5916 probes passed check_dic selection.\n",
      "finish in 0.357s, 74 probes kept.\n",
      "-- check region:197 2:48985000-48990000_strand_+_gene_NRNX1-seg-198, 9608 candidate probes\n",
      "--- 3320 probes passed check_dic selection.\n",
      "finish in 0.262s, 48 probes kept.\n",
      "-- check region:198 2:48990000-48995000_strand_+_gene_NRNX1-seg-199, 9918 candidate probes\n",
      "--- 4776 probes passed check_dic selection.\n",
      "finish in 0.320s, 68 probes kept.\n",
      "-- check region:199 2:48995000-49000000_strand_+_gene_NRNX1-seg-200, 9918 candidate probes\n",
      "--- 5118 probes passed check_dic selection.\n",
      "finish in 0.328s, 70 probes kept.\n",
      "-- check region:200 2:49000000-49005000_strand_+_gene_NRNX1-seg-201, 9918 candidate probes\n",
      "--- 5588 probes passed check_dic selection.\n",
      "finish in 0.347s, 77 probes kept.\n",
      "-- check region:201 2:49005000-49010000_strand_+_gene_NRNX1-seg-202, 7898 candidate probes\n",
      "--- 2860 probes passed check_dic selection.\n",
      "finish in 0.221s, 41 probes kept.\n",
      "-- check region:202 2:49010000-49015000_strand_+_gene_NRNX1-seg-203, 9004 candidate probes\n",
      "--- 3124 probes passed check_dic selection.\n",
      "finish in 0.247s, 45 probes kept.\n",
      "-- check region:203 2:49015000-49020000_strand_+_gene_NRNX1-seg-204, 9918 candidate probes\n",
      "--- 5098 probes passed check_dic selection.\n",
      "finish in 0.329s, 65 probes kept.\n",
      "-- check region:204 2:49020000-49025000_strand_+_gene_NRNX1-seg-205, 9821 candidate probes\n",
      "--- 5164 probes passed check_dic selection.\n",
      "finish in 0.331s, 65 probes kept.\n",
      "-- check region:205 2:49025000-49030000_strand_+_gene_NRNX1-seg-206, 9918 candidate probes\n",
      "--- 5418 probes passed check_dic selection.\n",
      "finish in 0.339s, 71 probes kept.\n",
      "-- check region:206 2:49030000-49035000_strand_+_gene_NRNX1-seg-207, 9908 candidate probes\n",
      "--- 5374 probes passed check_dic selection.\n",
      "finish in 0.339s, 70 probes kept.\n",
      "-- check region:207 2:49035000-49040000_strand_+_gene_NRNX1-seg-208, 9752 candidate probes\n",
      "--- 3286 probes passed check_dic selection.\n",
      "finish in 0.262s, 51 probes kept.\n",
      "-- check region:208 2:49040000-49045000_strand_+_gene_NRNX1-seg-209, 9918 candidate probes\n",
      "--- 5386 probes passed check_dic selection.\n",
      "finish in 0.340s, 75 probes kept.\n",
      "-- check region:209 2:49045000-49050000_strand_+_gene_NRNX1-seg-210, 9918 candidate probes\n",
      "--- 4188 probes passed check_dic selection.\n",
      "finish in 0.297s, 68 probes kept.\n",
      "-- check region:210 2:49050000-49055000_strand_+_gene_NRNX1-seg-211, 9918 candidate probes\n",
      "--- 4310 probes passed check_dic selection.\n",
      "finish in 0.299s, 62 probes kept.\n",
      "-- check region:211 2:49055000-49060000_strand_+_gene_NRNX1-seg-212, 9914 candidate probes\n",
      "--- 500 probes passed check_dic selection.\n",
      "finish in 0.168s, 23 probes kept.\n",
      "-- check region:212 2:49060000-49065000_strand_+_gene_NRNX1-seg-213, 9918 candidate probes\n",
      "--- 2936 probes passed check_dic selection.\n",
      "finish in 0.254s, 49 probes kept.\n",
      "-- check region:213 2:49065000-49070000_strand_+_gene_NRNX1-seg-214, 9918 candidate probes\n",
      "--- 5598 probes passed check_dic selection.\n",
      "finish in 0.345s, 73 probes kept.\n",
      "-- check region:214 2:49070000-49075000_strand_+_gene_NRNX1-seg-215, 9918 candidate probes\n",
      "--- 1326 probes passed check_dic selection.\n",
      "finish in 0.195s, 30 probes kept.\n",
      "-- check region:215 2:49075000-49080000_strand_+_gene_NRNX1-seg-216, 9894 candidate probes\n",
      "--- 826 probes passed check_dic selection.\n",
      "finish in 0.183s, 27 probes kept.\n",
      "-- check region:216 2:49080000-49085000_strand_+_gene_NRNX1-seg-217, 8532 candidate probes\n",
      "--- 1392 probes passed check_dic selection.\n",
      "finish in 0.182s, 24 probes kept.\n",
      "-- check region:217 2:49085000-49090000_strand_+_gene_NRNX1-seg-218, 8924 candidate probes\n",
      "--- 2780 probes passed check_dic selection.\n",
      "finish in 0.231s, 44 probes kept.\n",
      "-- check region:218 2:49090000-49095000_strand_+_gene_NRNX1-seg-219, 9912 candidate probes\n",
      "--- 836 probes passed check_dic selection.\n",
      "finish in 0.178s, 23 probes kept.\n",
      "-- check region:219 2:49095000-49100000_strand_+_gene_NRNX1-seg-220, 9884 candidate probes\n",
      "--- 1470 probes passed check_dic selection.\n",
      "finish in 0.202s, 35 probes kept.\n",
      "-- check region:220 2:49100000-49105000_strand_+_gene_NRNX1-seg-221, 9918 candidate probes\n",
      "--- 5252 probes passed check_dic selection.\n",
      "finish in 0.335s, 73 probes kept.\n",
      "-- check region:221 2:49105000-49110000_strand_+_gene_NRNX1-seg-222, 9918 candidate probes\n",
      "--- 5662 probes passed check_dic selection.\n",
      "finish in 0.350s, 71 probes kept.\n",
      "-- check region:222 2:49110000-49115000_strand_+_gene_NRNX1-seg-223, 9918 candidate probes\n",
      "--- 5700 probes passed check_dic selection.\n",
      "finish in 0.351s, 75 probes kept.\n",
      "-- check region:223 2:49115000-49120000_strand_+_gene_NRNX1-seg-224, 9918 candidate probes\n",
      "--- 5380 probes passed check_dic selection.\n",
      "finish in 0.339s, 74 probes kept.\n",
      "-- check region:224 2:49120000-49125000_strand_+_gene_NRNX1-seg-225, 9908 candidate probes\n",
      "--- 3824 probes passed check_dic selection.\n",
      "finish in 0.287s, 60 probes kept.\n",
      "-- check region:225 2:49125000-49130000_strand_+_gene_NRNX1-seg-226, 9744 candidate probes\n",
      "--- 3930 probes passed check_dic selection.\n",
      "finish in 0.284s, 54 probes kept.\n",
      "-- check region:226 2:49130000-49135000_strand_+_gene_NRNX1-seg-227, 9364 candidate probes\n",
      "--- 3180 probes passed check_dic selection.\n",
      "finish in 0.253s, 41 probes kept.\n",
      "-- check region:227 2:49135000-49140000_strand_+_gene_NRNX1-seg-228, 9888 candidate probes\n",
      "--- 1334 probes passed check_dic selection.\n",
      "finish in 0.195s, 29 probes kept.\n",
      "-- check region:228 2:49140000-49145000_strand_+_gene_NRNX1-seg-229, 9918 candidate probes\n",
      "--- 4048 probes passed check_dic selection.\n",
      "finish in 0.295s, 62 probes kept.\n",
      "-- check region:229 2:49145000-49150000_strand_+_gene_NRNX1-seg-230, 9918 candidate probes\n",
      "--- 4492 probes passed check_dic selection.\n",
      "finish in 0.306s, 70 probes kept.\n",
      "-- check region:230 2:49150000-49155000_strand_+_gene_NRNX1-seg-231, 9918 candidate probes\n",
      "--- 5454 probes passed check_dic selection.\n",
      "finish in 0.343s, 70 probes kept.\n",
      "-- check region:231 2:49155000-49160000_strand_+_gene_NRNX1-seg-232, 9860 candidate probes\n",
      "--- 1542 probes passed check_dic selection.\n",
      "finish in 0.203s, 29 probes kept.\n",
      "-- check region:232 2:49160000-49165000_strand_+_gene_NRNX1-seg-233, 9780 candidate probes\n",
      "--- 4894 probes passed check_dic selection.\n",
      "finish in 0.318s, 65 probes kept.\n",
      "-- check region:233 2:49165000-49170000_strand_+_gene_NRNX1-seg-234, 9904 candidate probes\n",
      "--- 3474 probes passed check_dic selection.\n",
      "finish in 0.275s, 54 probes kept.\n",
      "-- check region:234 2:49170000-49175000_strand_+_gene_NRNX1-seg-235, 9904 candidate probes\n",
      "--- 4410 probes passed check_dic selection.\n",
      "finish in 0.307s, 64 probes kept.\n",
      "-- check region:235 2:49175000-49180000_strand_+_gene_NRNX1-seg-236, 9918 candidate probes\n",
      "--- 3656 probes passed check_dic selection.\n",
      "finish in 0.280s, 54 probes kept.\n",
      "-- check region:236 2:49180000-49185000_strand_+_gene_NRNX1-seg-237, 9248 candidate probes\n",
      "--- 126 probes passed check_dic selection.\n",
      "finish in 0.151s, 5 probes kept.\n",
      "-- check region:237 2:49185000-49190000_strand_+_gene_NRNX1-seg-238, 9798 candidate probes\n",
      "--- 3444 probes passed check_dic selection.\n",
      "finish in 0.271s, 53 probes kept.\n",
      "-- check region:238 2:49190000-49195000_strand_+_gene_NRNX1-seg-239, 9918 candidate probes\n",
      "--- 5358 probes passed check_dic selection.\n",
      "finish in 0.338s, 73 probes kept.\n",
      "-- check region:239 2:49195000-49200000_strand_+_gene_NRNX1-seg-240, 9902 candidate probes\n",
      "--- 4734 probes passed check_dic selection.\n",
      "finish in 0.318s, 66 probes kept.\n",
      "-- check region:240 2:49200000-49205000_strand_+_gene_NRNX1-seg-241, 9846 candidate probes\n",
      "--- 3372 probes passed check_dic selection.\n",
      "finish in 0.266s, 49 probes kept.\n",
      "-- check region:241 2:49205000-49210000_strand_+_gene_NRNX1-seg-242, 9918 candidate probes\n",
      "--- 3068 probes passed check_dic selection.\n",
      "finish in 0.259s, 53 probes kept.\n",
      "-- check region:242 2:49210000-49215000_strand_+_gene_NRNX1-seg-243, 9002 candidate probes\n",
      "--- 84 probes passed check_dic selection.\n",
      "finish in 0.148s, 7 probes kept.\n",
      "-- check region:243 2:49215000-49220000_strand_+_gene_NRNX1-seg-244, 9368 candidate probes\n",
      "--- 3268 probes passed check_dic selection.\n",
      "finish in 0.258s, 54 probes kept.\n",
      "-- check region:244 2:49220000-49225000_strand_+_gene_NRNX1-seg-245, 9918 candidate probes\n",
      "--- 4542 probes passed check_dic selection.\n",
      "finish in 0.309s, 64 probes kept.\n",
      "-- check region:245 2:49225000-49230000_strand_+_gene_NRNX1-seg-246, 8890 candidate probes\n",
      "--- 1860 probes passed check_dic selection.\n",
      "finish in 0.203s, 38 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:246 2:49230000-49235000_strand_+_gene_NRNX1-seg-247, 9626 candidate probes\n",
      "--- 402 probes passed check_dic selection.\n",
      "finish in 0.166s, 9 probes kept.\n",
      "-- check region:247 2:49235000-49240000_strand_+_gene_NRNX1-seg-248, 9888 candidate probes\n",
      "--- 4640 probes passed check_dic selection.\n",
      "finish in 0.311s, 68 probes kept.\n",
      "-- check region:248 2:49240000-49245000_strand_+_gene_NRNX1-seg-249, 9860 candidate probes\n",
      "--- 2906 probes passed check_dic selection.\n",
      "finish in 0.254s, 41 probes kept.\n",
      "-- check region:249 2:49245000-49250000_strand_+_gene_NRNX1-seg-250, 9470 candidate probes\n",
      "--- 1100 probes passed check_dic selection.\n",
      "finish in 0.186s, 17 probes kept.\n",
      "-- check region:250 2:49250000-49255000_strand_+_gene_NRNX1-seg-251, 9918 candidate probes\n",
      "--- 4310 probes passed check_dic selection.\n",
      "finish in 0.301s, 68 probes kept.\n",
      "-- check region:251 2:49255000-49260000_strand_+_gene_NRNX1-seg-252, 9908 candidate probes\n",
      "--- 3218 probes passed check_dic selection.\n",
      "finish in 0.261s, 53 probes kept.\n",
      "-- check region:252 2:49260000-49265000_strand_+_gene_NRNX1-seg-253, 9918 candidate probes\n",
      "--- 5232 probes passed check_dic selection.\n",
      "finish in 0.334s, 68 probes kept.\n",
      "-- check region:253 2:49265000-49270000_strand_+_gene_NRNX1-seg-254, 9878 candidate probes\n",
      "--- 3538 probes passed check_dic selection.\n",
      "finish in 0.275s, 50 probes kept.\n",
      "-- check region:254 2:49270000-49275000_strand_+_gene_NRNX1-seg-255, 9852 candidate probes\n",
      "--- 4540 probes passed check_dic selection.\n",
      "finish in 0.309s, 64 probes kept.\n",
      "-- check region:255 2:49275000-49280000_strand_+_gene_NRNX1-seg-256, 9888 candidate probes\n",
      "--- 4358 probes passed check_dic selection.\n",
      "finish in 0.304s, 63 probes kept.\n",
      "-- check region:256 2:49280000-49285000_strand_+_gene_NRNX1-seg-257, 8932 candidate probes\n",
      "--- 3234 probes passed check_dic selection.\n",
      "finish in 0.249s, 52 probes kept.\n",
      "-- check region:257 2:49285000-49290000_strand_+_gene_NRNX1-seg-258, 9918 candidate probes\n",
      "--- 3818 probes passed check_dic selection.\n",
      "finish in 0.281s, 63 probes kept.\n",
      "-- check region:258 2:49290000-49295000_strand_+_gene_NRNX1-seg-259, 9918 candidate probes\n",
      "--- 3372 probes passed check_dic selection.\n",
      "finish in 0.271s, 54 probes kept.\n",
      "-- check region:259 2:49295000-49300000_strand_+_gene_NRNX1-seg-260, 9918 candidate probes\n",
      "--- 3418 probes passed check_dic selection.\n",
      "finish in 0.274s, 50 probes kept.\n",
      "-- check region:260 2:49300000-49305000_strand_+_gene_NRNX1-seg-261, 8710 candidate probes\n",
      "--- 272 probes passed check_dic selection.\n",
      "finish in 0.149s, 3 probes kept.\n",
      "-- check region:261 2:49305000-49310000_strand_+_gene_NRNX1-seg-262, 8620 candidate probes\n",
      "--- 2776 probes passed check_dic selection.\n",
      "finish in 0.227s, 42 probes kept.\n",
      "-- check region:262 2:49310000-49315000_strand_+_gene_NRNX1-seg-263, 9106 candidate probes\n",
      "--- 1840 probes passed check_dic selection.\n",
      "finish in 0.206s, 25 probes kept.\n",
      "-- check region:263 2:49315000-49320000_strand_+_gene_NRNX1-seg-264, 8150 candidate probes\n",
      "--- 2362 probes passed check_dic selection.\n",
      "finish in 0.209s, 35 probes kept.\n",
      "-- check region:264 2:49320000-49325000_strand_+_gene_NRNX1-seg-265, 9918 candidate probes\n",
      "--- 1722 probes passed check_dic selection.\n",
      "finish in 0.209s, 32 probes kept.\n",
      "-- check region:265 2:49325000-49330000_strand_+_gene_NRNX1-seg-266, 9918 candidate probes\n",
      "--- 2014 probes passed check_dic selection.\n",
      "finish in 0.218s, 33 probes kept.\n",
      "-- check region:266 2:49330000-49335000_strand_+_gene_NRNX1-seg-267, 9918 candidate probes\n",
      "--- 3928 probes passed check_dic selection.\n",
      "finish in 0.286s, 55 probes kept.\n",
      "-- check region:267 2:49335000-49340000_strand_+_gene_NRNX1-seg-268, 9918 candidate probes\n",
      "--- 4984 probes passed check_dic selection.\n",
      "finish in 0.328s, 70 probes kept.\n",
      "-- check region:268 2:49340000-49345000_strand_+_gene_NRNX1-seg-269, 9860 candidate probes\n",
      "--- 5220 probes passed check_dic selection.\n",
      "finish in 0.331s, 63 probes kept.\n",
      "-- check region:269 2:49345000-49350000_strand_+_gene_NRNX1-seg-270, 9918 candidate probes\n",
      "--- 3136 probes passed check_dic selection.\n",
      "finish in 0.259s, 50 probes kept.\n",
      "-- check region:270 2:49350000-49355000_strand_+_gene_NRNX1-seg-271, 9918 candidate probes\n",
      "--- 4000 probes passed check_dic selection.\n",
      "finish in 0.289s, 59 probes kept.\n",
      "-- check region:271 2:49355000-49360000_strand_+_gene_NRNX1-seg-272, 9918 candidate probes\n",
      "--- 5052 probes passed check_dic selection.\n",
      "finish in 0.329s, 64 probes kept.\n",
      "-- check region:272 2:49360000-49365000_strand_+_gene_NRNX1-seg-273, 9918 candidate probes\n",
      "--- 2144 probes passed check_dic selection.\n",
      "finish in 0.228s, 37 probes kept.\n",
      "-- check region:273 2:49365000-49370000_strand_+_gene_NRNX1-seg-274, 9918 candidate probes\n",
      "--- 4602 probes passed check_dic selection.\n",
      "finish in 0.310s, 65 probes kept.\n",
      "-- check region:274 2:49370000-49375000_strand_+_gene_NRNX1-seg-275, 9772 candidate probes\n",
      "--- 3762 probes passed check_dic selection.\n",
      "finish in 0.278s, 56 probes kept.\n",
      "-- check region:275 2:49375000-49380000_strand_+_gene_NRNX1-seg-276, 8782 candidate probes\n",
      "--- 4414 probes passed check_dic selection.\n",
      "finish in 0.289s, 57 probes kept.\n",
      "-- check region:276 2:49380000-49385000_strand_+_gene_NRNX1-seg-277, 9698 candidate probes\n",
      "--- 1922 probes passed check_dic selection.\n",
      "finish in 0.212s, 46 probes kept.\n",
      "-- check region:277 2:49385000-49390000_strand_+_gene_NRNX1-seg-278, 9444 candidate probes\n",
      "--- 942 probes passed check_dic selection.\n",
      "finish in 0.177s, 24 probes kept.\n",
      "-- check region:278 2:49390000-49395000_strand_+_gene_NRNX1-seg-279, 9878 candidate probes\n",
      "--- 1418 probes passed check_dic selection.\n",
      "finish in 0.201s, 28 probes kept.\n",
      "-- check region:279 2:49395000-49400000_strand_+_gene_NRNX1-seg-280, 9892 candidate probes\n",
      "--- 2828 probes passed check_dic selection.\n",
      "finish in 0.247s, 50 probes kept.\n",
      "-- check region:280 2:49400000-49405000_strand_+_gene_NRNX1-seg-281, 9918 candidate probes\n",
      "--- 4362 probes passed check_dic selection.\n",
      "finish in 0.302s, 63 probes kept.\n",
      "-- check region:281 2:49405000-49410000_strand_+_gene_NRNX1-seg-282, 9860 candidate probes\n",
      "--- 2432 probes passed check_dic selection.\n",
      "finish in 0.236s, 46 probes kept.\n",
      "-- check region:282 2:49410000-49415000_strand_+_gene_NRNX1-seg-283, 9842 candidate probes\n",
      "--- 3942 probes passed check_dic selection.\n",
      "finish in 0.286s, 57 probes kept.\n",
      "-- check region:283 2:49415000-49420000_strand_+_gene_NRNX1-seg-284, 9890 candidate probes\n",
      "--- 3796 probes passed check_dic selection.\n",
      "finish in 0.280s, 61 probes kept.\n",
      "-- check region:284 2:49420000-49425000_strand_+_gene_NRNX1-seg-285, 9892 candidate probes\n",
      "--- 2604 probes passed check_dic selection.\n",
      "finish in 0.242s, 51 probes kept.\n",
      "-- check region:285 2:49425000-49430000_strand_+_gene_NRNX1-seg-286, 9800 candidate probes\n",
      "--- 2390 probes passed check_dic selection.\n",
      "finish in 0.231s, 39 probes kept.\n",
      "-- check region:286 2:49430000-49435000_strand_+_gene_NRNX1-seg-287, 9680 candidate probes\n",
      "--- 1086 probes passed check_dic selection.\n",
      "finish in 0.187s, 26 probes kept.\n",
      "-- check region:287 2:49435000-49440000_strand_+_gene_NRNX1-seg-288, 9836 candidate probes\n",
      "--- 1720 probes passed check_dic selection.\n",
      "finish in 0.211s, 29 probes kept.\n",
      "-- check region:288 2:49440000-49445000_strand_+_gene_NRNX1-seg-289, 9916 candidate probes\n",
      "--- 2176 probes passed check_dic selection.\n",
      "finish in 0.225s, 45 probes kept.\n",
      "-- check region:289 2:49445000-49450000_strand_+_gene_NRNX1-seg-290, 9898 candidate probes\n",
      "--- 2468 probes passed check_dic selection.\n",
      "finish in 0.234s, 36 probes kept.\n",
      "-- check region:290 2:49450000-49455000_strand_+_gene_NRNX1-seg-291, 9896 candidate probes\n",
      "--- 1812 probes passed check_dic selection.\n",
      "finish in 0.209s, 39 probes kept.\n",
      "-- check region:291 2:49455000-49460000_strand_+_gene_NRNX1-seg-292, 9918 candidate probes\n",
      "--- 2918 probes passed check_dic selection.\n",
      "finish in 0.249s, 57 probes kept.\n",
      "-- check region:292 2:49460000-49465000_strand_+_gene_NRNX1-seg-293, 9918 candidate probes\n",
      "--- 1040 probes passed check_dic selection.\n",
      "finish in 0.186s, 27 probes kept.\n",
      "-- check region:293 2:49465000-49470000_strand_+_gene_NRNX1-seg-294, 9892 candidate probes\n",
      "--- 1460 probes passed check_dic selection.\n",
      "finish in 0.206s, 31 probes kept.\n",
      "-- check region:294 2:49470000-49475000_strand_+_gene_NRNX1-seg-295, 9908 candidate probes\n",
      "--- 1886 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 0.216s, 44 probes kept.\n",
      "-- check region:295 2:49475000-49480000_strand_+_gene_NRNX1-seg-296, 9752 candidate probes\n",
      "--- 996 probes passed check_dic selection.\n",
      "finish in 0.185s, 17 probes kept.\n",
      "-- check region:296 2:49480000-49485000_strand_+_gene_NRNX1-seg-297, 8792 candidate probes\n",
      "--- 504 probes passed check_dic selection.\n",
      "finish in 0.156s, 8 probes kept.\n",
      "-- check region:297 2:49485000-49490000_strand_+_gene_NRNX1-seg-298, 9196 candidate probes\n",
      "--- 2346 probes passed check_dic selection.\n",
      "finish in 0.223s, 42 probes kept.\n",
      "-- check region:298 2:49490000-49495000_strand_+_gene_NRNX1-seg-299, 9426 candidate probes\n",
      "--- 2806 probes passed check_dic selection.\n",
      "finish in 0.240s, 39 probes kept.\n",
      "-- check region:299 2:49495000-49500000_strand_+_gene_NRNX1-seg-300, 9802 candidate probes\n",
      "--- 1980 probes passed check_dic selection.\n",
      "finish in 0.216s, 36 probes kept.\n",
      "-- check region:300 2:49500000-49505000_strand_+_gene_NRNX1-seg-301, 9918 candidate probes\n",
      "--- 3318 probes passed check_dic selection.\n",
      "finish in 0.263s, 54 probes kept.\n",
      "-- check region:301 2:49505000-49510000_strand_+_gene_NRNX1-seg-302, 9878 candidate probes\n",
      "--- 1508 probes passed check_dic selection.\n",
      "finish in 0.201s, 31 probes kept.\n",
      "-- check region:302 2:49510000-49515000_strand_+_gene_NRNX1-seg-303, 9908 candidate probes\n",
      "--- 3064 probes passed check_dic selection.\n",
      "finish in 0.256s, 50 probes kept.\n",
      "-- check region:303 2:49515000-49520000_strand_+_gene_NRNX1-seg-304, 9772 candidate probes\n",
      "--- 3736 probes passed check_dic selection.\n",
      "finish in 0.284s, 58 probes kept.\n",
      "-- check region:304 2:49520000-49525000_strand_+_gene_NRNX1-seg-305, 9822 candidate probes\n",
      "--- 2650 probes passed check_dic selection.\n",
      "finish in 0.241s, 46 probes kept.\n",
      "-- check region:305 2:49525000-49530000_strand_+_gene_NRNX1-seg-306, 9878 candidate probes\n",
      "--- 2470 probes passed check_dic selection.\n",
      "finish in 0.236s, 49 probes kept.\n",
      "-- check region:306 2:49530000-49535000_strand_+_gene_NRNX1-seg-307, 9918 candidate probes\n",
      "--- 712 probes passed check_dic selection.\n",
      "finish in 0.175s, 22 probes kept.\n",
      "-- check region:307 2:49535000-49540000_strand_+_gene_NRNX1-seg-308, 9916 candidate probes\n",
      "--- 1060 probes passed check_dic selection.\n",
      "finish in 0.188s, 24 probes kept.\n",
      "-- check region:308 2:49540000-49545000_strand_+_gene_NRNX1-seg-309, 9918 candidate probes\n",
      "--- 392 probes passed check_dic selection.\n",
      "finish in 0.170s, 19 probes kept.\n",
      "-- check region:309 2:49545000-49550000_strand_+_gene_NRNX1-seg-310, 9442 candidate probes\n",
      "--- 256 probes passed check_dic selection.\n",
      "finish in 0.160s, 13 probes kept.\n",
      "-- check region:310 2:49550000-49555000_strand_+_gene_NRNX1-seg-311, 9918 candidate probes\n",
      "--- 3178 probes passed check_dic selection.\n",
      "finish in 0.258s, 58 probes kept.\n",
      "-- check region:311 2:49555000-49560000_strand_+_gene_NRNX1-seg-312, 9918 candidate probes\n",
      "--- 3188 probes passed check_dic selection.\n",
      "finish in 0.259s, 57 probes kept.\n",
      "-- check region:312 2:49560000-49565000_strand_+_gene_NRNX1-seg-313, 9918 candidate probes\n",
      "--- 5666 probes passed check_dic selection.\n",
      "finish in 0.348s, 76 probes kept.\n",
      "-- check region:313 2:49565000-49570000_strand_+_gene_NRNX1-seg-314, 9918 candidate probes\n",
      "--- 3456 probes passed check_dic selection.\n",
      "finish in 0.270s, 54 probes kept.\n",
      "-- check region:314 2:49570000-49575000_strand_+_gene_NRNX1-seg-315, 9474 candidate probes\n",
      "--- 1820 probes passed check_dic selection.\n",
      "finish in 0.206s, 42 probes kept.\n",
      "-- check region:315 2:49575000-49580000_strand_+_gene_NRNX1-seg-316, 9916 candidate probes\n",
      "--- 2866 probes passed check_dic selection.\n",
      "finish in 0.251s, 48 probes kept.\n",
      "-- check region:316 2:49580000-49585000_strand_+_gene_NRNX1-seg-317, 9918 candidate probes\n",
      "--- 2444 probes passed check_dic selection.\n",
      "finish in 0.237s, 44 probes kept.\n",
      "-- check region:317 2:49585000-49590000_strand_+_gene_NRNX1-seg-318, 9738 candidate probes\n",
      "--- 2362 probes passed check_dic selection.\n",
      "finish in 0.232s, 34 probes kept.\n",
      "-- check region:318 2:49590000-49595000_strand_+_gene_NRNX1-seg-319, 9918 candidate probes\n",
      "--- 1318 probes passed check_dic selection.\n",
      "finish in 0.198s, 35 probes kept.\n",
      "-- check region:319 2:49595000-49600000_strand_+_gene_NRNX1-seg-320, 9916 candidate probes\n",
      "--- 2774 probes passed check_dic selection.\n",
      "finish in 0.247s, 50 probes kept.\n",
      "-- check region:320 2:49600000-49605000_strand_+_gene_NRNX1-seg-321, 9918 candidate probes\n",
      "--- 4960 probes passed check_dic selection.\n",
      "finish in 0.324s, 69 probes kept.\n",
      "-- check region:321 2:49605000-49610000_strand_+_gene_NRNX1-seg-322, 9904 candidate probes\n",
      "--- 3494 probes passed check_dic selection.\n",
      "finish in 0.270s, 55 probes kept.\n",
      "-- check region:322 2:49610000-49615000_strand_+_gene_NRNX1-seg-323, 9910 candidate probes\n",
      "--- 2428 probes passed check_dic selection.\n",
      "finish in 0.236s, 41 probes kept.\n",
      "-- check region:323 2:49615000-49620000_strand_+_gene_NRNX1-seg-324, 9896 candidate probes\n",
      "--- 3550 probes passed check_dic selection.\n",
      "finish in 0.273s, 57 probes kept.\n",
      "-- check region:324 2:49620000-49625000_strand_+_gene_NRNX1-seg-325, 9894 candidate probes\n",
      "--- 3666 probes passed check_dic selection.\n",
      "finish in 0.277s, 59 probes kept.\n",
      "-- check region:325 2:49625000-49630000_strand_+_gene_NRNX1-seg-326, 9852 candidate probes\n",
      "--- 3810 probes passed check_dic selection.\n",
      "finish in 0.280s, 64 probes kept.\n",
      "-- check region:326 2:49630000-49635000_strand_+_gene_NRNX1-seg-327, 9918 candidate probes\n",
      "--- 4088 probes passed check_dic selection.\n",
      "finish in 0.291s, 64 probes kept.\n",
      "-- check region:327 2:49635000-49640000_strand_+_gene_NRNX1-seg-328, 8814 candidate probes\n",
      "--- 1924 probes passed check_dic selection.\n",
      "finish in 0.202s, 34 probes kept.\n",
      "-- check region:328 2:49640000-49645000_strand_+_gene_NRNX1-seg-329, 9918 candidate probes\n",
      "--- 4004 probes passed check_dic selection.\n",
      "finish in 0.287s, 61 probes kept.\n",
      "-- check region:329 2:49645000-49650000_strand_+_gene_NRNX1-seg-330, 9918 candidate probes\n",
      "--- 1820 probes passed check_dic selection.\n",
      "finish in 0.213s, 31 probes kept.\n",
      "-- check region:330 2:49650000-49655000_strand_+_gene_NRNX1-seg-331, 9910 candidate probes\n",
      "--- 2766 probes passed check_dic selection.\n",
      "finish in 0.245s, 53 probes kept.\n",
      "-- check region:331 2:49655000-49660000_strand_+_gene_NRNX1-seg-332, 9918 candidate probes\n",
      "--- 3398 probes passed check_dic selection.\n",
      "finish in 0.267s, 60 probes kept.\n",
      "-- check region:332 2:49660000-49665000_strand_+_gene_NRNX1-seg-333, 9918 candidate probes\n",
      "--- 2602 probes passed check_dic selection.\n",
      "finish in 0.240s, 52 probes kept.\n",
      "-- check region:333 2:49665000-49670000_strand_+_gene_NRNX1-seg-334, 9918 candidate probes\n",
      "--- 3742 probes passed check_dic selection.\n",
      "finish in 0.287s, 65 probes kept.\n",
      "-- check region:334 2:49670000-49675000_strand_+_gene_NRNX1-seg-335, 9622 candidate probes\n",
      "--- 3060 probes passed check_dic selection.\n",
      "finish in 0.253s, 52 probes kept.\n",
      "-- check region:335 2:49675000-49680000_strand_+_gene_NRNX1-seg-336, 9918 candidate probes\n",
      "--- 3730 probes passed check_dic selection.\n",
      "finish in 0.280s, 61 probes kept.\n",
      "-- check region:336 2:49680000-49685000_strand_+_gene_NRNX1-seg-337, 9842 candidate probes\n",
      "--- 1990 probes passed check_dic selection.\n",
      "finish in 0.220s, 35 probes kept.\n",
      "-- check region:337 2:49685000-49690000_strand_+_gene_NRNX1-seg-338, 9918 candidate probes\n",
      "--- 2294 probes passed check_dic selection.\n",
      "finish in 0.231s, 49 probes kept.\n",
      "-- check region:338 2:49690000-49695000_strand_+_gene_NRNX1-seg-339, 9918 candidate probes\n",
      "--- 3020 probes passed check_dic selection.\n",
      "finish in 0.255s, 50 probes kept.\n",
      "-- check region:339 2:49695000-49700000_strand_+_gene_NRNX1-seg-340, 9880 candidate probes\n",
      "--- 2408 probes passed check_dic selection.\n",
      "finish in 0.231s, 46 probes kept.\n",
      "-- check region:340 2:49700000-49705000_strand_+_gene_NRNX1-seg-341, 9916 candidate probes\n",
      "--- 3264 probes passed check_dic selection.\n",
      "finish in 0.263s, 49 probes kept.\n",
      "-- check region:341 2:49705000-49710000_strand_+_gene_NRNX1-seg-342, 9918 candidate probes\n",
      "--- 3746 probes passed check_dic selection.\n",
      "finish in 0.279s, 65 probes kept.\n",
      "-- check region:342 2:49710000-49715000_strand_+_gene_NRNX1-seg-343, 9916 candidate probes\n",
      "--- 3006 probes passed check_dic selection.\n",
      "finish in 0.255s, 50 probes kept.\n",
      "-- check region:343 2:49715000-49720000_strand_+_gene_NRNX1-seg-344, 9918 candidate probes\n",
      "--- 3464 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 0.268s, 56 probes kept.\n",
      "-- check region:344 2:49720000-49725000_strand_+_gene_NRNX1-seg-345, 9918 candidate probes\n",
      "--- 1324 probes passed check_dic selection.\n",
      "finish in 0.193s, 38 probes kept.\n",
      "-- check region:345 2:49725000-49730000_strand_+_gene_NRNX1-seg-346, 9644 candidate probes\n",
      "--- 1894 probes passed check_dic selection.\n",
      "finish in 0.212s, 42 probes kept.\n",
      "-- check region:346 2:49730000-49735000_strand_+_gene_NRNX1-seg-347, 9914 candidate probes\n",
      "--- 986 probes passed check_dic selection.\n",
      "finish in 0.190s, 25 probes kept.\n",
      "-- check region:347 2:49735000-49740000_strand_+_gene_NRNX1-seg-348, 9918 candidate probes\n",
      "--- 2598 probes passed check_dic selection.\n",
      "finish in 0.241s, 44 probes kept.\n",
      "-- check region:348 2:49740000-49745000_strand_+_gene_NRNX1-seg-349, 9808 candidate probes\n",
      "--- 1944 probes passed check_dic selection.\n",
      "finish in 0.213s, 35 probes kept.\n",
      "-- check region:349 2:49745000-49750000_strand_+_gene_NRNX1-seg-350, 9918 candidate probes\n",
      "--- 1760 probes passed check_dic selection.\n",
      "finish in 0.210s, 37 probes kept.\n",
      "-- check region:350 2:49750000-49755000_strand_+_gene_NRNX1-seg-351, 9894 candidate probes\n",
      "--- 3344 probes passed check_dic selection.\n",
      "finish in 0.265s, 53 probes kept.\n",
      "-- check region:351 2:49755000-49760000_strand_+_gene_NRNX1-seg-352, 9878 candidate probes\n",
      "--- 1794 probes passed check_dic selection.\n",
      "finish in 0.213s, 36 probes kept.\n",
      "-- check region:352 2:49760000-49765000_strand_+_gene_NRNX1-seg-353, 9918 candidate probes\n",
      "--- 3660 probes passed check_dic selection.\n",
      "finish in 0.277s, 59 probes kept.\n",
      "-- check region:353 2:49765000-49770000_strand_+_gene_NRNX1-seg-354, 9918 candidate probes\n",
      "--- 3228 probes passed check_dic selection.\n",
      "finish in 0.261s, 50 probes kept.\n",
      "-- check region:354 2:49770000-49775000_strand_+_gene_NRNX1-seg-355, 9918 candidate probes\n",
      "--- 3622 probes passed check_dic selection.\n",
      "finish in 0.279s, 60 probes kept.\n",
      "-- check region:355 2:49775000-49780000_strand_+_gene_NRNX1-seg-356, 9918 candidate probes\n",
      "--- 2014 probes passed check_dic selection.\n",
      "finish in 0.221s, 40 probes kept.\n",
      "-- check region:356 2:49780000-49785000_strand_+_gene_NRNX1-seg-357, 9918 candidate probes\n",
      "--- 4914 probes passed check_dic selection.\n",
      "finish in 0.320s, 73 probes kept.\n",
      "-- check region:357 2:49785000-49790000_strand_+_gene_NRNX1-seg-358, 9906 candidate probes\n",
      "--- 3128 probes passed check_dic selection.\n",
      "finish in 0.262s, 47 probes kept.\n",
      "-- check region:358 2:49790000-49795000_strand_+_gene_NRNX1-seg-359, 9880 candidate probes\n",
      "--- 2876 probes passed check_dic selection.\n",
      "finish in 0.252s, 52 probes kept.\n",
      "-- check region:359 2:49795000-49800000_strand_+_gene_NRNX1-seg-360, 9904 candidate probes\n",
      "--- 4004 probes passed check_dic selection.\n",
      "finish in 0.293s, 56 probes kept.\n",
      "-- check region:360 2:49800000-49805000_strand_+_gene_NRNX1-seg-361, 9904 candidate probes\n",
      "--- 3772 probes passed check_dic selection.\n",
      "finish in 0.284s, 61 probes kept.\n",
      "-- check region:361 2:49805000-49810000_strand_+_gene_NRNX1-seg-362, 9898 candidate probes\n",
      "--- 2028 probes passed check_dic selection.\n",
      "finish in 0.218s, 40 probes kept.\n",
      "-- check region:362 2:49810000-49815000_strand_+_gene_NRNX1-seg-363, 9550 candidate probes\n",
      "--- 2954 probes passed check_dic selection.\n",
      "finish in 0.247s, 55 probes kept.\n",
      "-- check region:363 2:49815000-49820000_strand_+_gene_NRNX1-seg-364, 9744 candidate probes\n",
      "--- 3290 probes passed check_dic selection.\n",
      "finish in 0.263s, 57 probes kept.\n",
      "-- check region:364 2:49820000-49825000_strand_+_gene_NRNX1-seg-365, 9682 candidate probes\n",
      "--- 2668 probes passed check_dic selection.\n",
      "finish in 0.242s, 37 probes kept.\n",
      "-- check region:365 2:49825000-49830000_strand_+_gene_NRNX1-seg-366, 9902 candidate probes\n",
      "--- 2878 probes passed check_dic selection.\n",
      "finish in 0.254s, 42 probes kept.\n",
      "-- check region:366 2:49830000-49835000_strand_+_gene_NRNX1-seg-367, 9914 candidate probes\n",
      "--- 4544 probes passed check_dic selection.\n",
      "finish in 0.309s, 64 probes kept.\n",
      "-- check region:367 2:49835000-49840000_strand_+_gene_NRNX1-seg-368, 9844 candidate probes\n",
      "--- 2886 probes passed check_dic selection.\n",
      "finish in 0.249s, 47 probes kept.\n",
      "-- check region:368 2:49840000-49845000_strand_+_gene_NRNX1-seg-369, 9918 candidate probes\n",
      "--- 3424 probes passed check_dic selection.\n",
      "finish in 0.270s, 56 probes kept.\n",
      "-- check region:369 2:49845000-49850000_strand_+_gene_NRNX1-seg-370, 9912 candidate probes\n",
      "--- 2744 probes passed check_dic selection.\n",
      "finish in 0.244s, 51 probes kept.\n",
      "-- check region:370 2:49850000-49855000_strand_+_gene_NRNX1-seg-371, 9904 candidate probes\n",
      "--- 2868 probes passed check_dic selection.\n",
      "finish in 0.249s, 47 probes kept.\n",
      "-- check region:371 2:49855000-49860000_strand_+_gene_NRNX1-seg-372, 9918 candidate probes\n",
      "--- 4624 probes passed check_dic selection.\n",
      "finish in 0.312s, 65 probes kept.\n",
      "-- check region:372 2:49860000-49865000_strand_+_gene_NRNX1-seg-373, 9918 candidate probes\n",
      "--- 3688 probes passed check_dic selection.\n",
      "finish in 0.279s, 55 probes kept.\n",
      "-- check region:373 2:49865000-49870000_strand_+_gene_NRNX1-seg-374, 9622 candidate probes\n",
      "--- 1214 probes passed check_dic selection.\n",
      "finish in 0.188s, 31 probes kept.\n",
      "-- check region:374 2:49870000-49875000_strand_+_gene_NRNX1-seg-375, 9854 candidate probes\n",
      "--- 912 probes passed check_dic selection.\n",
      "finish in 0.180s, 21 probes kept.\n",
      "-- check region:375 2:49875000-49880000_strand_+_gene_NRNX1-seg-376, 8554 candidate probes\n",
      "--- 666 probes passed check_dic selection.\n",
      "finish in 0.159s, 25 probes kept.\n",
      "-- check region:376 2:49880000-49885000_strand_+_gene_NRNX1-seg-377, 9834 candidate probes\n",
      "--- 1020 probes passed check_dic selection.\n",
      "finish in 0.189s, 19 probes kept.\n",
      "-- check region:377 2:49885000-49890000_strand_+_gene_NRNX1-seg-378, 9578 candidate probes\n",
      "--- 722 probes passed check_dic selection.\n",
      "finish in 0.174s, 15 probes kept.\n",
      "-- check region:378 2:49890000-49895000_strand_+_gene_NRNX1-seg-379, 9918 candidate probes\n",
      "--- 1130 probes passed check_dic selection.\n",
      "finish in 0.187s, 30 probes kept.\n",
      "-- check region:379 2:49895000-49900000_strand_+_gene_NRNX1-seg-380, 9910 candidate probes\n",
      "--- 2970 probes passed check_dic selection.\n",
      "finish in 0.253s, 49 probes kept.\n",
      "-- check region:380 2:49900000-49905000_strand_+_gene_NRNX1-seg-381, 9918 candidate probes\n",
      "--- 3548 probes passed check_dic selection.\n",
      "finish in 0.275s, 60 probes kept.\n",
      "-- check region:381 2:49905000-49910000_strand_+_gene_NRNX1-seg-382, 9918 candidate probes\n",
      "--- 2504 probes passed check_dic selection.\n",
      "finish in 0.234s, 43 probes kept.\n",
      "-- check region:382 2:49910000-49915000_strand_+_gene_NRNX1-seg-383, 9918 candidate probes\n",
      "--- 3316 probes passed check_dic selection.\n",
      "finish in 0.264s, 52 probes kept.\n",
      "-- check region:383 2:49915000-49920000_strand_+_gene_NRNX1-seg-384, 9918 candidate probes\n",
      "--- 2298 probes passed check_dic selection.\n",
      "finish in 0.226s, 41 probes kept.\n",
      "-- check region:384 2:49920000-49925000_strand_+_gene_NRNX1-seg-385, 9902 candidate probes\n",
      "--- 3148 probes passed check_dic selection.\n",
      "finish in 0.257s, 55 probes kept.\n",
      "-- check region:385 2:49925000-49930000_strand_+_gene_NRNX1-seg-386, 9912 candidate probes\n",
      "--- 3742 probes passed check_dic selection.\n",
      "finish in 0.279s, 58 probes kept.\n",
      "-- check region:386 2:49930000-49935000_strand_+_gene_NRNX1-seg-387, 9900 candidate probes\n",
      "--- 3106 probes passed check_dic selection.\n",
      "finish in 0.257s, 45 probes kept.\n",
      "-- check region:387 2:49935000-49940000_strand_+_gene_NRNX1-seg-388, 9918 candidate probes\n",
      "--- 4172 probes passed check_dic selection.\n",
      "finish in 0.294s, 62 probes kept.\n",
      "-- check region:388 2:49940000-49945000_strand_+_gene_NRNX1-seg-389, 9908 candidate probes\n",
      "--- 3776 probes passed check_dic selection.\n",
      "finish in 0.282s, 60 probes kept.\n",
      "-- check region:389 2:49945000-49950000_strand_+_gene_NRNX1-seg-390, 9720 candidate probes\n",
      "--- 1970 probes passed check_dic selection.\n",
      "finish in 0.216s, 32 probes kept.\n",
      "-- check region:390 2:49950000-49955000_strand_+_gene_NRNX1-seg-391, 9918 candidate probes\n",
      "--- 3578 probes passed check_dic selection.\n",
      "finish in 0.277s, 61 probes kept.\n",
      "-- check region:391 2:49955000-49960000_strand_+_gene_NRNX1-seg-392, 9918 candidate probes\n",
      "--- 4356 probes passed check_dic selection.\n",
      "finish in 0.301s, 67 probes kept.\n",
      "-- check region:392 2:49960000-49965000_strand_+_gene_NRNX1-seg-393, 9906 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 3234 probes passed check_dic selection.\n",
      "finish in 0.264s, 49 probes kept.\n",
      "-- check region:393 2:49965000-49970000_strand_+_gene_NRNX1-seg-394, 9854 candidate probes\n",
      "--- 2852 probes passed check_dic selection.\n",
      "finish in 0.247s, 53 probes kept.\n",
      "-- check region:394 2:49970000-49975000_strand_+_gene_NRNX1-seg-395, 9918 candidate probes\n",
      "--- 3492 probes passed check_dic selection.\n",
      "finish in 0.270s, 56 probes kept.\n",
      "-- check region:395 2:49975000-49980000_strand_+_gene_NRNX1-seg-396, 9890 candidate probes\n",
      "--- 2412 probes passed check_dic selection.\n",
      "finish in 0.234s, 43 probes kept.\n",
      "-- check region:396 2:49980000-49985000_strand_+_gene_NRNX1-seg-397, 9918 candidate probes\n",
      "--- 3196 probes passed check_dic selection.\n",
      "finish in 0.260s, 52 probes kept.\n",
      "-- check region:397 2:49985000-49990000_strand_+_gene_NRNX1-seg-398, 9918 candidate probes\n",
      "--- 2432 probes passed check_dic selection.\n",
      "finish in 0.232s, 47 probes kept.\n",
      "-- check region:398 2:49990000-49995000_strand_+_gene_NRNX1-seg-399, 9914 candidate probes\n",
      "--- 2770 probes passed check_dic selection.\n",
      "finish in 0.248s, 48 probes kept.\n",
      "-- check region:399 2:49995000-50000000_strand_+_gene_NRNX1-seg-400, 9876 candidate probes\n",
      "--- 3074 probes passed check_dic selection.\n",
      "finish in 0.256s, 55 probes kept.\n",
      "-- check region:400 2:50000000-50005000_strand_+_gene_NRNX1-seg-401, 9918 candidate probes\n",
      "--- 3960 probes passed check_dic selection.\n",
      "finish in 0.285s, 61 probes kept.\n",
      "-- check region:401 2:50005000-50010000_strand_+_gene_NRNX1-seg-402, 9760 candidate probes\n",
      "--- 3572 probes passed check_dic selection.\n",
      "finish in 0.273s, 55 probes kept.\n",
      "-- check region:402 2:50010000-50015000_strand_+_gene_NRNX1-seg-403, 9918 candidate probes\n",
      "--- 3520 probes passed check_dic selection.\n",
      "finish in 0.271s, 64 probes kept.\n",
      "-- check region:403 2:50015000-50020000_strand_+_gene_NRNX1-seg-404, 9880 candidate probes\n",
      "--- 3386 probes passed check_dic selection.\n",
      "finish in 0.270s, 58 probes kept.\n",
      "-- check region:404 2:50020000-50025000_strand_+_gene_NRNX1-seg-405, 9848 candidate probes\n",
      "--- 2854 probes passed check_dic selection.\n",
      "finish in 0.247s, 47 probes kept.\n",
      "-- check region:405 2:50025000-50030000_strand_+_gene_NRNX1-seg-406, 9896 candidate probes\n",
      "--- 3476 probes passed check_dic selection.\n",
      "finish in 0.273s, 54 probes kept.\n",
      "-- check region:406 2:50030000-50035000_strand_+_gene_NRNX1-seg-407, 9914 candidate probes\n",
      "--- 1724 probes passed check_dic selection.\n",
      "finish in 0.210s, 39 probes kept.\n",
      "-- check region:407 2:50035000-50040000_strand_+_gene_NRNX1-seg-408, 9910 candidate probes\n",
      "--- 2714 probes passed check_dic selection.\n",
      "finish in 0.245s, 45 probes kept.\n",
      "-- check region:408 2:50040000-50045000_strand_+_gene_NRNX1-seg-409, 9814 candidate probes\n",
      "--- 2780 probes passed check_dic selection.\n",
      "finish in 0.244s, 44 probes kept.\n",
      "-- check region:409 2:50045000-50050000_strand_+_gene_NRNX1-seg-410, 9918 candidate probes\n",
      "--- 3382 probes passed check_dic selection.\n",
      "finish in 0.266s, 51 probes kept.\n",
      "-- check region:410 2:50050000-50055000_strand_+_gene_NRNX1-seg-411, 9918 candidate probes\n",
      "--- 2308 probes passed check_dic selection.\n",
      "finish in 0.226s, 46 probes kept.\n",
      "-- check region:411 2:50055000-50060000_strand_+_gene_NRNX1-seg-412, 9918 candidate probes\n",
      "--- 2014 probes passed check_dic selection.\n",
      "finish in 0.219s, 37 probes kept.\n",
      "-- check region:412 2:50060000-50065000_strand_+_gene_NRNX1-seg-413, 9908 candidate probes\n",
      "--- 2018 probes passed check_dic selection.\n",
      "finish in 0.219s, 41 probes kept.\n",
      "-- check region:413 2:50065000-50070000_strand_+_gene_NRNX1-seg-414, 9912 candidate probes\n",
      "--- 3692 probes passed check_dic selection.\n",
      "finish in 0.277s, 58 probes kept.\n",
      "-- check region:414 2:50070000-50075000_strand_+_gene_NRNX1-seg-415, 9606 candidate probes\n",
      "--- 2582 probes passed check_dic selection.\n",
      "finish in 0.234s, 49 probes kept.\n",
      "-- check region:415 2:50075000-50080000_strand_+_gene_NRNX1-seg-416, 9918 candidate probes\n",
      "--- 3020 probes passed check_dic selection.\n",
      "finish in 0.257s, 52 probes kept.\n",
      "-- check region:416 2:50080000-50085000_strand_+_gene_NRNX1-seg-417, 9900 candidate probes\n",
      "--- 2010 probes passed check_dic selection.\n",
      "finish in 0.223s, 45 probes kept.\n",
      "-- check region:417 2:50085000-50090000_strand_+_gene_NRNX1-seg-418, 9914 candidate probes\n",
      "--- 2328 probes passed check_dic selection.\n",
      "finish in 0.228s, 41 probes kept.\n",
      "-- check region:418 2:50090000-50095000_strand_+_gene_NRNX1-seg-419, 9918 candidate probes\n",
      "--- 3702 probes passed check_dic selection.\n",
      "finish in 0.278s, 54 probes kept.\n",
      "-- check region:419 2:50095000-50100000_strand_+_gene_NRNX1-seg-420, 9824 candidate probes\n",
      "--- 3940 probes passed check_dic selection.\n",
      "finish in 0.285s, 55 probes kept.\n",
      "-- check region:420 2:50100000-50105000_strand_+_gene_NRNX1-seg-421, 9918 candidate probes\n",
      "--- 3798 probes passed check_dic selection.\n",
      "finish in 0.282s, 62 probes kept.\n",
      "-- check region:421 2:50105000-50110000_strand_+_gene_NRNX1-seg-422, 9918 candidate probes\n",
      "--- 2754 probes passed check_dic selection.\n",
      "finish in 0.242s, 48 probes kept.\n",
      "-- check region:422 2:50110000-50115000_strand_+_gene_NRNX1-seg-423, 9914 candidate probes\n",
      "--- 3232 probes passed check_dic selection.\n",
      "finish in 0.262s, 54 probes kept.\n",
      "-- check region:423 2:50115000-50120000_strand_+_gene_NRNX1-seg-424, 9918 candidate probes\n",
      "--- 3376 probes passed check_dic selection.\n",
      "finish in 0.266s, 59 probes kept.\n",
      "-- check region:424 2:50120000-50125000_strand_+_gene_NRNX1-seg-425, 9902 candidate probes\n",
      "--- 2866 probes passed check_dic selection.\n",
      "finish in 0.249s, 49 probes kept.\n",
      "-- check region:425 2:50125000-50130000_strand_+_gene_NRNX1-seg-426, 9918 candidate probes\n",
      "--- 2746 probes passed check_dic selection.\n",
      "finish in 0.243s, 53 probes kept.\n",
      "-- check region:426 2:50130000-50135000_strand_+_gene_NRNX1-seg-427, 9890 candidate probes\n",
      "--- 4718 probes passed check_dic selection.\n",
      "finish in 0.313s, 67 probes kept.\n",
      "-- check region:427 2:50135000-50140000_strand_+_gene_NRNX1-seg-428, 9908 candidate probes\n",
      "--- 2904 probes passed check_dic selection.\n",
      "finish in 0.252s, 46 probes kept.\n",
      "-- check region:428 2:50140000-50145000_strand_+_gene_NRNX1-seg-429, 9918 candidate probes\n",
      "--- 2904 probes passed check_dic selection.\n",
      "finish in 0.249s, 46 probes kept.\n",
      "-- check region:429 2:50145000-50150000_strand_+_gene_NRNX1-seg-430, 9888 candidate probes\n",
      "--- 3492 probes passed check_dic selection.\n",
      "finish in 0.268s, 58 probes kept.\n",
      "-- check region:430 2:50150000-50155000_strand_+_gene_NRNX1-seg-431, 9918 candidate probes\n",
      "--- 2534 probes passed check_dic selection.\n",
      "finish in 0.236s, 41 probes kept.\n",
      "-- check region:431 2:50155000-50160000_strand_+_gene_NRNX1-seg-432, 9910 candidate probes\n",
      "--- 3354 probes passed check_dic selection.\n",
      "finish in 0.263s, 55 probes kept.\n",
      "-- check region:432 2:50160000-50165000_strand_+_gene_NRNX1-seg-433, 9918 candidate probes\n",
      "--- 3034 probes passed check_dic selection.\n",
      "finish in 0.255s, 54 probes kept.\n",
      "-- check region:433 2:50165000-50170000_strand_+_gene_NRNX1-seg-434, 9898 candidate probes\n",
      "--- 3414 probes passed check_dic selection.\n",
      "finish in 0.271s, 52 probes kept.\n",
      "-- check region:434 2:50170000-50175000_strand_+_gene_NRNX1-seg-435, 9896 candidate probes\n",
      "--- 2234 probes passed check_dic selection.\n",
      "finish in 0.227s, 47 probes kept.\n",
      "-- check region:435 2:50175000-50180000_strand_+_gene_NRNX1-seg-436, 9918 candidate probes\n",
      "--- 3302 probes passed check_dic selection.\n",
      "finish in 0.263s, 52 probes kept.\n",
      "-- check region:436 2:50180000-50185000_strand_+_gene_NRNX1-seg-437, 9918 candidate probes\n",
      "--- 2272 probes passed check_dic selection.\n",
      "finish in 0.228s, 41 probes kept.\n",
      "-- check region:437 2:50185000-50190000_strand_+_gene_NRNX1-seg-438, 9760 candidate probes\n",
      "--- 2014 probes passed check_dic selection.\n",
      "finish in 0.217s, 34 probes kept.\n",
      "-- check region:438 2:50190000-50195000_strand_+_gene_NRNX1-seg-439, 9918 candidate probes\n",
      "--- 1344 probes passed check_dic selection.\n",
      "finish in 0.195s, 38 probes kept.\n",
      "-- check region:439 2:50195000-50200000_strand_+_gene_NRNX1-seg-440, 9918 candidate probes\n",
      "--- 3392 probes passed check_dic selection.\n",
      "finish in 0.268s, 59 probes kept.\n",
      "-- check region:440 2:50200000-50205000_strand_+_gene_NRNX1-seg-441, 9890 candidate probes\n",
      "--- 3182 probes passed check_dic selection.\n",
      "finish in 0.258s, 57 probes kept.\n",
      "-- check region:441 2:50205000-50210000_strand_+_gene_NRNX1-seg-442, 9918 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 3286 probes passed check_dic selection.\n",
      "finish in 0.263s, 50 probes kept.\n",
      "-- check region:442 2:50210000-50215000_strand_+_gene_NRNX1-seg-443, 9918 candidate probes\n",
      "--- 2350 probes passed check_dic selection.\n",
      "finish in 0.228s, 39 probes kept.\n",
      "-- check region:443 2:50215000-50220000_strand_+_gene_NRNX1-seg-444, 9911 candidate probes\n",
      "--- 2782 probes passed check_dic selection.\n",
      "finish in 0.245s, 46 probes kept.\n",
      "-- check region:444 2:50220000-50225000_strand_+_gene_NRNX1-seg-445, 9908 candidate probes\n",
      "--- 3928 probes passed check_dic selection.\n",
      "finish in 0.286s, 62 probes kept.\n",
      "-- check region:445 2:50225000-50230000_strand_+_gene_NRNX1-seg-446, 9908 candidate probes\n",
      "--- 4394 probes passed check_dic selection.\n",
      "finish in 0.302s, 61 probes kept.\n",
      "-- check region:446 2:50230000-50235000_strand_+_gene_NRNX1-seg-447, 9916 candidate probes\n",
      "--- 2226 probes passed check_dic selection.\n",
      "finish in 0.226s, 41 probes kept.\n",
      "-- check region:447 2:50235000-50240000_strand_+_gene_NRNX1-seg-448, 9888 candidate probes\n",
      "--- 2882 probes passed check_dic selection.\n",
      "finish in 0.247s, 53 probes kept.\n",
      "-- check region:448 2:50240000-50245000_strand_+_gene_NRNX1-seg-449, 9918 candidate probes\n",
      "--- 2110 probes passed check_dic selection.\n",
      "finish in 0.220s, 43 probes kept.\n",
      "-- check region:449 2:50245000-50250000_strand_+_gene_NRNX1-seg-450, 9918 candidate probes\n",
      "--- 2762 probes passed check_dic selection.\n",
      "finish in 0.245s, 51 probes kept.\n",
      "-- check region:450 2:50250000-50255000_strand_+_gene_NRNX1-seg-451, 9854 candidate probes\n",
      "--- 480 probes passed check_dic selection.\n",
      "finish in 0.169s, 16 probes kept.\n",
      "-- check region:451 2:50255000-50260000_strand_+_gene_NRNX1-seg-452, 9918 candidate probes\n",
      "--- 2456 probes passed check_dic selection.\n",
      "finish in 0.234s, 50 probes kept.\n",
      "-- check region:452 2:50260000-50265000_strand_+_gene_NRNX1-seg-453, 9910 candidate probes\n",
      "--- 3800 probes passed check_dic selection.\n",
      "finish in 0.280s, 65 probes kept.\n",
      "-- check region:453 2:50265000-50270000_strand_+_gene_NRNX1-seg-454, 9910 candidate probes\n",
      "--- 3512 probes passed check_dic selection.\n",
      "finish in 0.271s, 48 probes kept.\n",
      "-- check region:454 2:50270000-50275000_strand_+_gene_NRNX1-seg-455, 9878 candidate probes\n",
      "--- 2324 probes passed check_dic selection.\n",
      "finish in 0.231s, 43 probes kept.\n",
      "-- check region:455 2:50275000-50280000_strand_+_gene_NRNX1-seg-456, 9911 candidate probes\n",
      "--- 2684 probes passed check_dic selection.\n",
      "finish in 0.242s, 49 probes kept.\n",
      "-- check region:456 2:50280000-50285000_strand_+_gene_NRNX1-seg-457, 9798 candidate probes\n",
      "--- 2766 probes passed check_dic selection.\n",
      "finish in 0.245s, 47 probes kept.\n",
      "-- check region:457 2:50285000-50290000_strand_+_gene_NRNX1-seg-458, 9918 candidate probes\n",
      "--- 2374 probes passed check_dic selection.\n",
      "finish in 0.232s, 44 probes kept.\n",
      "-- check region:458 2:50290000-50295000_strand_+_gene_NRNX1-seg-459, 9918 candidate probes\n",
      "--- 5010 probes passed check_dic selection.\n",
      "finish in 0.325s, 68 probes kept.\n",
      "-- check region:459 2:50295000-50300000_strand_+_gene_NRNX1-seg-460, 9904 candidate probes\n",
      "--- 2504 probes passed check_dic selection.\n",
      "finish in 0.237s, 49 probes kept.\n",
      "-- check region:460 2:50300000-50305000_strand_+_gene_NRNX1-seg-461, 9918 candidate probes\n",
      "--- 2780 probes passed check_dic selection.\n",
      "finish in 0.247s, 52 probes kept.\n",
      "-- check region:461 2:50305000-50310000_strand_+_gene_NRNX1-seg-462, 9912 candidate probes\n",
      "--- 2964 probes passed check_dic selection.\n",
      "finish in 0.254s, 49 probes kept.\n",
      "-- check region:462 2:50310000-50315000_strand_+_gene_NRNX1-seg-463, 9918 candidate probes\n",
      "--- 3442 probes passed check_dic selection.\n",
      "finish in 0.269s, 59 probes kept.\n",
      "-- check region:463 2:50315000-50320000_strand_+_gene_NRNX1-seg-464, 9918 candidate probes\n",
      "--- 3334 probes passed check_dic selection.\n",
      "finish in 0.263s, 53 probes kept.\n",
      "-- check region:464 2:50320000-50325000_strand_+_gene_NRNX1-seg-465, 9864 candidate probes\n",
      "--- 4198 probes passed check_dic selection.\n",
      "finish in 0.294s, 60 probes kept.\n",
      "-- check region:465 2:50325000-50330000_strand_+_gene_NRNX1-seg-466, 9848 candidate probes\n",
      "--- 2228 probes passed check_dic selection.\n",
      "finish in 0.225s, 34 probes kept.\n",
      "-- check region:466 2:50330000-50335000_strand_+_gene_NRNX1-seg-467, 9918 candidate probes\n",
      "--- 4078 probes passed check_dic selection.\n",
      "finish in 0.289s, 61 probes kept.\n",
      "-- check region:467 2:50335000-50340000_strand_+_gene_NRNX1-seg-468, 9918 candidate probes\n",
      "--- 2914 probes passed check_dic selection.\n",
      "finish in 0.250s, 51 probes kept.\n",
      "-- check region:468 2:50340000-50345000_strand_+_gene_NRNX1-seg-469, 9918 candidate probes\n",
      "--- 4582 probes passed check_dic selection.\n",
      "finish in 0.310s, 68 probes kept.\n",
      "-- check region:469 2:50345000-50350000_strand_+_gene_NRNX1-seg-470, 9918 candidate probes\n",
      "--- 6620 probes passed check_dic selection.\n",
      "finish in 0.381s, 79 probes kept.\n",
      "-- check region:470 2:50350000-50355000_strand_+_gene_NRNX1-seg-471, 9918 candidate probes\n",
      "--- 4182 probes passed check_dic selection.\n",
      "finish in 0.297s, 62 probes kept.\n",
      "-- check region:471 2:50355000-50360000_strand_+_gene_NRNX1-seg-472, 9872 candidate probes\n",
      "--- 1022 probes passed check_dic selection.\n",
      "finish in 0.190s, 26 probes kept.\n",
      "-- check region:472 2:50360000-50365000_strand_+_gene_NRNX1-seg-473, 9790 candidate probes\n",
      "--- 558 probes passed check_dic selection.\n",
      "finish in 0.173s, 14 probes kept.\n",
      "-- check region:473 2:50365000-50370000_strand_+_gene_NRNX1-seg-474, 9918 candidate probes\n",
      "--- 3938 probes passed check_dic selection.\n",
      "finish in 0.284s, 61 probes kept.\n",
      "-- check region:474 2:50370000-50375000_strand_+_gene_NRNX1-seg-475, 9882 candidate probes\n",
      "--- 3606 probes passed check_dic selection.\n",
      "finish in 0.274s, 60 probes kept.\n",
      "-- check region:475 2:50375000-50380000_strand_+_gene_NRNX1-seg-476, 9918 candidate probes\n",
      "--- 2962 probes passed check_dic selection.\n",
      "finish in 0.252s, 51 probes kept.\n",
      "-- check region:476 2:50380000-50385000_strand_+_gene_NRNX1-seg-477, 9890 candidate probes\n",
      "--- 4124 probes passed check_dic selection.\n",
      "finish in 0.294s, 58 probes kept.\n",
      "-- check region:477 2:50385000-50390000_strand_+_gene_NRNX1-seg-478, 9918 candidate probes\n",
      "--- 3674 probes passed check_dic selection.\n",
      "finish in 0.278s, 52 probes kept.\n",
      "-- check region:478 2:50390000-50395000_strand_+_gene_NRNX1-seg-479, 9918 candidate probes\n",
      "--- 3268 probes passed check_dic selection.\n",
      "finish in 0.262s, 58 probes kept.\n",
      "-- check region:479 2:50395000-50400000_strand_+_gene_NRNX1-seg-480, 9876 candidate probes\n",
      "--- 4378 probes passed check_dic selection.\n",
      "finish in 0.301s, 58 probes kept.\n",
      "-- check region:480 2:50400000-50405000_strand_+_gene_NRNX1-seg-481, 9918 candidate probes\n",
      "--- 4758 probes passed check_dic selection.\n",
      "finish in 0.314s, 66 probes kept.\n",
      "-- check region:481 2:50405000-50410000_strand_+_gene_NRNX1-seg-482, 9918 candidate probes\n",
      "--- 3534 probes passed check_dic selection.\n",
      "finish in 0.273s, 60 probes kept.\n",
      "-- check region:482 2:50410000-50415000_strand_+_gene_NRNX1-seg-483, 9838 candidate probes\n",
      "--- 1160 probes passed check_dic selection.\n",
      "finish in 0.191s, 25 probes kept.\n",
      "-- check region:483 2:50415000-50420000_strand_+_gene_NRNX1-seg-484, 9918 candidate probes\n",
      "--- 3364 probes passed check_dic selection.\n",
      "finish in 0.267s, 59 probes kept.\n",
      "-- check region:484 2:50420000-50425000_strand_+_gene_NRNX1-seg-485, 9876 candidate probes\n",
      "--- 3640 probes passed check_dic selection.\n",
      "finish in 0.277s, 63 probes kept.\n",
      "-- check region:485 2:50425000-50430000_strand_+_gene_NRNX1-seg-486, 9918 candidate probes\n",
      "--- 3194 probes passed check_dic selection.\n",
      "finish in 0.261s, 50 probes kept.\n",
      "-- check region:486 2:50430000-50435000_strand_+_gene_NRNX1-seg-487, 9908 candidate probes\n",
      "--- 3836 probes passed check_dic selection.\n",
      "finish in 0.284s, 64 probes kept.\n",
      "-- check region:487 2:50435000-50440000_strand_+_gene_NRNX1-seg-488, 9884 candidate probes\n",
      "--- 3296 probes passed check_dic selection.\n",
      "finish in 0.266s, 49 probes kept.\n",
      "-- check region:488 2:50440000-50445000_strand_+_gene_NRNX1-seg-489, 9918 candidate probes\n",
      "--- 3536 probes passed check_dic selection.\n",
      "finish in 0.272s, 57 probes kept.\n",
      "-- check region:489 2:50445000-50450000_strand_+_gene_NRNX1-seg-490, 9898 candidate probes\n",
      "--- 3640 probes passed check_dic selection.\n",
      "finish in 0.276s, 58 probes kept.\n",
      "-- check region:490 2:50450000-50455000_strand_+_gene_NRNX1-seg-491, 9918 candidate probes\n",
      "--- 2568 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 0.239s, 50 probes kept.\n",
      "-- check region:491 2:50455000-50460000_strand_+_gene_NRNX1-seg-492, 9910 candidate probes\n",
      "--- 2816 probes passed check_dic selection.\n",
      "finish in 0.247s, 47 probes kept.\n",
      "-- check region:492 2:50460000-50465000_strand_+_gene_NRNX1-seg-493, 9918 candidate probes\n",
      "--- 3250 probes passed check_dic selection.\n",
      "finish in 0.261s, 54 probes kept.\n",
      "-- check region:493 2:50465000-50470000_strand_+_gene_NRNX1-seg-494, 9918 candidate probes\n",
      "--- 2940 probes passed check_dic selection.\n",
      "finish in 0.249s, 49 probes kept.\n",
      "-- check region:494 2:50470000-50475000_strand_+_gene_NRNX1-seg-495, 9918 candidate probes\n",
      "--- 3188 probes passed check_dic selection.\n",
      "finish in 0.259s, 59 probes kept.\n",
      "-- check region:495 2:50475000-50480000_strand_+_gene_NRNX1-seg-496, 9886 candidate probes\n",
      "--- 4178 probes passed check_dic selection.\n",
      "finish in 0.295s, 65 probes kept.\n",
      "-- check region:496 2:50480000-50485000_strand_+_gene_NRNX1-seg-497, 9552 candidate probes\n",
      "--- 3704 probes passed check_dic selection.\n",
      "finish in 0.276s, 52 probes kept.\n",
      "-- check region:497 2:50485000-50490000_strand_+_gene_NRNX1-seg-498, 9918 candidate probes\n",
      "--- 6400 probes passed check_dic selection.\n",
      "finish in 0.373s, 81 probes kept.\n",
      "-- check region:498 2:50490000-50495000_strand_+_gene_NRNX1-seg-499, 9898 candidate probes\n",
      "--- 5146 probes passed check_dic selection.\n",
      "finish in 0.331s, 66 probes kept.\n",
      "-- check region:499 2:50495000-50500000_strand_+_gene_NRNX1-seg-500, 9880 candidate probes\n",
      "--- 3948 probes passed check_dic selection.\n",
      "finish in 0.287s, 67 probes kept.\n",
      "-- check region:500 2:50500000-50505000_strand_+_gene_NRNX1-seg-501, 9904 candidate probes\n",
      "--- 3474 probes passed check_dic selection.\n",
      "finish in 0.270s, 49 probes kept.\n",
      "-- check region:501 2:50505000-50510000_strand_+_gene_NRNX1-seg-502, 9918 candidate probes\n",
      "--- 4176 probes passed check_dic selection.\n",
      "finish in 0.293s, 65 probes kept.\n",
      "-- check region:502 2:50510000-50515000_strand_+_gene_NRNX1-seg-503, 9916 candidate probes\n",
      "--- 3056 probes passed check_dic selection.\n",
      "finish in 0.255s, 49 probes kept.\n",
      "-- check region:503 2:50515000-50520000_strand_+_gene_NRNX1-seg-504, 9916 candidate probes\n",
      "--- 2432 probes passed check_dic selection.\n",
      "finish in 0.232s, 47 probes kept.\n",
      "-- check region:504 2:50520000-50525000_strand_+_gene_NRNX1-seg-505, 9902 candidate probes\n",
      "--- 2336 probes passed check_dic selection.\n",
      "finish in 0.228s, 44 probes kept.\n",
      "-- check region:505 2:50525000-50530000_strand_+_gene_NRNX1-seg-506, 9918 candidate probes\n",
      "--- 3686 probes passed check_dic selection.\n",
      "finish in 0.276s, 60 probes kept.\n",
      "-- check region:506 2:50530000-50535000_strand_+_gene_NRNX1-seg-507, 9918 candidate probes\n",
      "--- 3988 probes passed check_dic selection.\n",
      "finish in 0.289s, 58 probes kept.\n",
      "-- check region:507 2:50535000-50540000_strand_+_gene_NRNX1-seg-508, 9918 candidate probes\n",
      "--- 4374 probes passed check_dic selection.\n",
      "finish in 0.300s, 62 probes kept.\n",
      "-- check region:508 2:50540000-50545000_strand_+_gene_NRNX1-seg-509, 9908 candidate probes\n",
      "--- 2132 probes passed check_dic selection.\n",
      "finish in 0.222s, 45 probes kept.\n",
      "-- check region:509 2:50545000-50550000_strand_+_gene_NRNX1-seg-510, 9918 candidate probes\n",
      "--- 3584 probes passed check_dic selection.\n",
      "finish in 0.275s, 55 probes kept.\n",
      "-- check region:510 2:50550000-50555000_strand_+_gene_NRNX1-seg-511, 9904 candidate probes\n",
      "--- 3378 probes passed check_dic selection.\n",
      "finish in 0.269s, 52 probes kept.\n",
      "-- check region:511 2:50555000-50560000_strand_+_gene_NRNX1-seg-512, 9900 candidate probes\n",
      "--- 3008 probes passed check_dic selection.\n",
      "finish in 0.254s, 56 probes kept.\n",
      "-- check region:512 2:50560000-50565000_strand_+_gene_NRNX1-seg-513, 9918 candidate probes\n",
      "--- 2866 probes passed check_dic selection.\n",
      "finish in 0.248s, 52 probes kept.\n",
      "-- check region:513 2:50565000-50570000_strand_+_gene_NRNX1-seg-514, 9918 candidate probes\n",
      "--- 3150 probes passed check_dic selection.\n",
      "finish in 0.260s, 51 probes kept.\n",
      "-- check region:514 2:50570000-50575000_strand_+_gene_NRNX1-seg-515, 9918 candidate probes\n",
      "--- 4122 probes passed check_dic selection.\n",
      "finish in 0.294s, 63 probes kept.\n",
      "-- check region:515 2:50575000-50580000_strand_+_gene_NRNX1-seg-516, 9918 candidate probes\n",
      "--- 3210 probes passed check_dic selection.\n",
      "finish in 0.259s, 52 probes kept.\n",
      "-- check region:516 2:50580000-50585000_strand_+_gene_NRNX1-seg-517, 9918 candidate probes\n",
      "--- 3716 probes passed check_dic selection.\n",
      "finish in 0.280s, 59 probes kept.\n",
      "-- check region:517 2:50585000-50590000_strand_+_gene_NRNX1-seg-518, 9810 candidate probes\n",
      "--- 3724 probes passed check_dic selection.\n",
      "finish in 0.278s, 55 probes kept.\n",
      "-- check region:518 2:50590000-50595000_strand_+_gene_NRNX1-seg-519, 9880 candidate probes\n",
      "--- 4078 probes passed check_dic selection.\n",
      "finish in 0.292s, 62 probes kept.\n",
      "-- check region:519 2:50595000-50600000_strand_+_gene_NRNX1-seg-520, 9896 candidate probes\n",
      "--- 4080 probes passed check_dic selection.\n",
      "finish in 0.291s, 51 probes kept.\n",
      "-- check region:520 2:50600000-50605000_strand_+_gene_NRNX1-seg-521, 9910 candidate probes\n",
      "--- 3536 probes passed check_dic selection.\n",
      "finish in 0.271s, 61 probes kept.\n",
      "-- check region:521 2:50605000-50610000_strand_+_gene_NRNX1-seg-522, 9918 candidate probes\n",
      "--- 2828 probes passed check_dic selection.\n",
      "finish in 0.248s, 50 probes kept.\n",
      "-- check region:522 2:50610000-50615000_strand_+_gene_NRNX1-seg-523, 9888 candidate probes\n",
      "--- 3720 probes passed check_dic selection.\n",
      "finish in 0.281s, 59 probes kept.\n",
      "-- check region:523 2:50615000-50620000_strand_+_gene_NRNX1-seg-524, 9918 candidate probes\n",
      "--- 3250 probes passed check_dic selection.\n",
      "finish in 0.261s, 55 probes kept.\n",
      "-- check region:524 2:50620000-50625000_strand_+_gene_NRNX1-seg-525, 9678 candidate probes\n",
      "--- 5060 probes passed check_dic selection.\n",
      "finish in 0.325s, 58 probes kept.\n",
      "-- check region:525 2:50625000-50630000_strand_+_gene_NRNX1-seg-526, 9914 candidate probes\n",
      "--- 2380 probes passed check_dic selection.\n",
      "finish in 0.229s, 44 probes kept.\n",
      "-- check region:526 2:50630000-50635000_strand_+_gene_NRNX1-seg-527, 9918 candidate probes\n",
      "--- 3292 probes passed check_dic selection.\n",
      "finish in 0.261s, 57 probes kept.\n",
      "-- check region:527 2:50635000-50640000_strand_+_gene_NRNX1-seg-528, 9868 candidate probes\n",
      "--- 3530 probes passed check_dic selection.\n",
      "finish in 0.272s, 47 probes kept.\n",
      "-- check region:528 2:50640000-50645000_strand_+_gene_NRNX1-seg-529, 9918 candidate probes\n",
      "--- 2484 probes passed check_dic selection.\n",
      "finish in 0.233s, 50 probes kept.\n",
      "-- check region:529 2:50645000-50650000_strand_+_gene_NRNX1-seg-530, 9888 candidate probes\n",
      "--- 4160 probes passed check_dic selection.\n",
      "finish in 0.293s, 61 probes kept.\n",
      "-- check region:530 2:50650000-50655000_strand_+_gene_NRNX1-seg-531, 9624 candidate probes\n",
      "--- 2756 probes passed check_dic selection.\n",
      "finish in 0.244s, 48 probes kept.\n",
      "-- check region:531 2:50655000-50660000_strand_+_gene_NRNX1-seg-532, 9918 candidate probes\n",
      "--- 4264 probes passed check_dic selection.\n",
      "finish in 0.297s, 61 probes kept.\n",
      "-- check region:532 2:50660000-50665000_strand_+_gene_NRNX1-seg-533, 9884 candidate probes\n",
      "--- 4108 probes passed check_dic selection.\n",
      "finish in 0.293s, 67 probes kept.\n",
      "-- check region:533 2:50665000-50670000_strand_+_gene_NRNX1-seg-534, 9918 candidate probes\n",
      "--- 2802 probes passed check_dic selection.\n",
      "finish in 0.247s, 52 probes kept.\n",
      "-- check region:534 2:50670000-50675000_strand_+_gene_NRNX1-seg-535, 9918 candidate probes\n",
      "--- 1952 probes passed check_dic selection.\n",
      "finish in 0.216s, 39 probes kept.\n",
      "-- check region:535 2:50675000-50680000_strand_+_gene_NRNX1-seg-536, 9918 candidate probes\n",
      "--- 3392 probes passed check_dic selection.\n",
      "finish in 0.268s, 59 probes kept.\n",
      "-- check region:536 2:50680000-50685000_strand_+_gene_NRNX1-seg-537, 9818 candidate probes\n",
      "--- 3140 probes passed check_dic selection.\n",
      "finish in 0.259s, 51 probes kept.\n",
      "-- check region:537 2:50685000-50690000_strand_+_gene_NRNX1-seg-538, 9898 candidate probes\n",
      "--- 4194 probes passed check_dic selection.\n",
      "finish in 0.296s, 65 probes kept.\n",
      "-- check region:538 2:50690000-50695000_strand_+_gene_NRNX1-seg-539, 9918 candidate probes\n",
      "--- 3260 probes passed check_dic selection.\n",
      "finish in 0.261s, 55 probes kept.\n",
      "-- check region:539 2:50695000-50700000_strand_+_gene_NRNX1-seg-540, 9918 candidate probes\n",
      "--- 3556 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 0.274s, 54 probes kept.\n",
      "-- check region:540 2:50700000-50705000_strand_+_gene_NRNX1-seg-541, 9886 candidate probes\n",
      "--- 2132 probes passed check_dic selection.\n",
      "finish in 0.221s, 45 probes kept.\n",
      "-- check region:541 2:50705000-50710000_strand_+_gene_NRNX1-seg-542, 9918 candidate probes\n",
      "--- 4724 probes passed check_dic selection.\n",
      "finish in 0.314s, 65 probes kept.\n",
      "-- check region:542 2:50710000-50715000_strand_+_gene_NRNX1-seg-543, 9914 candidate probes\n",
      "--- 3416 probes passed check_dic selection.\n",
      "finish in 0.268s, 52 probes kept.\n",
      "-- check region:543 2:50715000-50720000_strand_+_gene_NRNX1-seg-544, 9898 candidate probes\n",
      "--- 2880 probes passed check_dic selection.\n",
      "finish in 0.248s, 47 probes kept.\n",
      "-- check region:544 2:50720000-50725000_strand_+_gene_NRNX1-seg-545, 9918 candidate probes\n",
      "--- 4056 probes passed check_dic selection.\n",
      "finish in 0.292s, 60 probes kept.\n",
      "-- check region:545 2:50725000-50730000_strand_+_gene_NRNX1-seg-546, 9918 candidate probes\n",
      "--- 2820 probes passed check_dic selection.\n",
      "finish in 0.246s, 52 probes kept.\n",
      "-- check region:546 2:50730000-50735000_strand_+_gene_NRNX1-seg-547, 9856 candidate probes\n",
      "--- 2754 probes passed check_dic selection.\n",
      "finish in 0.243s, 46 probes kept.\n",
      "-- check region:547 2:50735000-50740000_strand_+_gene_NRNX1-seg-548, 9918 candidate probes\n",
      "--- 3100 probes passed check_dic selection.\n",
      "finish in 0.257s, 47 probes kept.\n",
      "-- check region:548 2:50740000-50745000_strand_+_gene_NRNX1-seg-549, 9918 candidate probes\n",
      "--- 2622 probes passed check_dic selection.\n",
      "finish in 0.239s, 46 probes kept.\n",
      "-- check region:549 2:50745000-50750000_strand_+_gene_NRNX1-seg-550, 9918 candidate probes\n",
      "--- 4252 probes passed check_dic selection.\n",
      "finish in 0.301s, 56 probes kept.\n",
      "-- check region:550 2:50750000-50755000_strand_+_gene_NRNX1-seg-551, 9918 candidate probes\n",
      "--- 3458 probes passed check_dic selection.\n",
      "finish in 0.268s, 48 probes kept.\n",
      "-- check region:551 2:50755000-50760000_strand_+_gene_NRNX1-seg-552, 9906 candidate probes\n",
      "--- 4078 probes passed check_dic selection.\n",
      "finish in 0.291s, 58 probes kept.\n",
      "-- check region:552 2:50760000-50765000_strand_+_gene_NRNX1-seg-553, 9918 candidate probes\n",
      "--- 3784 probes passed check_dic selection.\n",
      "finish in 0.286s, 52 probes kept.\n",
      "-- check region:553 2:50765000-50770000_strand_+_gene_NRNX1-seg-554, 9918 candidate probes\n",
      "--- 4480 probes passed check_dic selection.\n",
      "finish in 0.306s, 65 probes kept.\n",
      "-- check region:554 2:50770000-50775000_strand_+_gene_NRNX1-seg-555, 9918 candidate probes\n",
      "--- 4150 probes passed check_dic selection.\n",
      "finish in 0.294s, 59 probes kept.\n",
      "-- check region:555 2:50775000-50780000_strand_+_gene_NRNX1-seg-556, 9880 candidate probes\n",
      "--- 3294 probes passed check_dic selection.\n",
      "finish in 0.267s, 53 probes kept.\n",
      "-- check region:556 2:50780000-50785000_strand_+_gene_NRNX1-seg-557, 9900 candidate probes\n",
      "--- 3600 probes passed check_dic selection.\n",
      "finish in 0.276s, 57 probes kept.\n",
      "-- check region:557 2:50785000-50790000_strand_+_gene_NRNX1-seg-558, 9890 candidate probes\n",
      "--- 3854 probes passed check_dic selection.\n",
      "finish in 0.288s, 59 probes kept.\n",
      "-- check region:558 2:50790000-50795000_strand_+_gene_NRNX1-seg-559, 9918 candidate probes\n",
      "--- 3456 probes passed check_dic selection.\n",
      "finish in 0.268s, 52 probes kept.\n",
      "-- check region:559 2:50795000-50800000_strand_+_gene_NRNX1-seg-560, 9918 candidate probes\n",
      "--- 3590 probes passed check_dic selection.\n",
      "finish in 0.274s, 61 probes kept.\n",
      "-- check region:560 2:50800000-50805000_strand_+_gene_NRNX1-seg-561, 9862 candidate probes\n",
      "--- 2084 probes passed check_dic selection.\n",
      "finish in 0.231s, 40 probes kept.\n",
      "-- check region:561 2:50805000-50810000_strand_+_gene_NRNX1-seg-562, 9918 candidate probes\n",
      "--- 2692 probes passed check_dic selection.\n",
      "finish in 0.243s, 49 probes kept.\n",
      "-- check region:562 2:50810000-50815000_strand_+_gene_NRNX1-seg-563, 9918 candidate probes\n",
      "--- 1932 probes passed check_dic selection.\n",
      "finish in 0.215s, 43 probes kept.\n",
      "-- check region:563 2:50815000-50820000_strand_+_gene_NRNX1-seg-564, 9918 candidate probes\n",
      "--- 2174 probes passed check_dic selection.\n",
      "finish in 0.226s, 45 probes kept.\n",
      "-- check region:564 2:50820000-50825000_strand_+_gene_NRNX1-seg-565, 9918 candidate probes\n",
      "--- 3224 probes passed check_dic selection.\n",
      "finish in 0.261s, 49 probes kept.\n",
      "-- check region:565 2:50825000-50830000_strand_+_gene_NRNX1-seg-566, 9908 candidate probes\n",
      "--- 2864 probes passed check_dic selection.\n",
      "finish in 0.255s, 41 probes kept.\n",
      "-- check region:566 2:50830000-50835000_strand_+_gene_NRNX1-seg-567, 9918 candidate probes\n",
      "--- 3010 probes passed check_dic selection.\n",
      "finish in 0.253s, 50 probes kept.\n",
      "-- check region:567 2:50835000-50840000_strand_+_gene_NRNX1-seg-568, 9918 candidate probes\n",
      "--- 2434 probes passed check_dic selection.\n",
      "finish in 0.323s, 46 probes kept.\n",
      "-- check region:568 2:50840000-50845000_strand_+_gene_NRNX1-seg-569, 9918 candidate probes\n",
      "--- 4296 probes passed check_dic selection.\n",
      "finish in 0.300s, 64 probes kept.\n",
      "-- check region:569 2:50845000-50850000_strand_+_gene_NRNX1-seg-570, 9918 candidate probes\n",
      "--- 3996 probes passed check_dic selection.\n",
      "finish in 0.292s, 61 probes kept.\n",
      "-- check region:570 2:50850000-50855000_strand_+_gene_NRNX1-seg-571, 9918 candidate probes\n",
      "--- 3210 probes passed check_dic selection.\n",
      "finish in 0.260s, 55 probes kept.\n",
      "-- check region:571 2:50855000-50860000_strand_+_gene_NRNX1-seg-572, 9918 candidate probes\n",
      "--- 3866 probes passed check_dic selection.\n",
      "finish in 0.283s, 58 probes kept.\n",
      "-- check region:572 2:50860000-50865000_strand_+_gene_NRNX1-seg-573, 9918 candidate probes\n",
      "--- 4198 probes passed check_dic selection.\n",
      "finish in 0.298s, 61 probes kept.\n",
      "-- check region:573 2:50865000-50870000_strand_+_gene_NRNX1-seg-574, 9908 candidate probes\n",
      "--- 2614 probes passed check_dic selection.\n",
      "finish in 0.239s, 43 probes kept.\n",
      "-- check region:574 2:50870000-50875000_strand_+_gene_NRNX1-seg-575, 9918 candidate probes\n",
      "--- 2666 probes passed check_dic selection.\n",
      "finish in 0.241s, 47 probes kept.\n",
      "-- check region:575 2:50875000-50880000_strand_+_gene_NRNX1-seg-576, 9918 candidate probes\n",
      "--- 4036 probes passed check_dic selection.\n",
      "finish in 0.289s, 61 probes kept.\n",
      "-- check region:576 2:50880000-50885000_strand_+_gene_NRNX1-seg-577, 9812 candidate probes\n",
      "--- 2448 probes passed check_dic selection.\n",
      "finish in 0.230s, 45 probes kept.\n",
      "-- check region:577 2:50885000-50890000_strand_+_gene_NRNX1-seg-578, 9918 candidate probes\n",
      "--- 1846 probes passed check_dic selection.\n",
      "finish in 0.208s, 39 probes kept.\n",
      "-- check region:578 2:50890000-50895000_strand_+_gene_NRNX1-seg-579, 9798 candidate probes\n",
      "--- 2276 probes passed check_dic selection.\n",
      "finish in 0.224s, 41 probes kept.\n",
      "-- check region:579 2:50895000-50900000_strand_+_gene_NRNX1-seg-580, 9916 candidate probes\n",
      "--- 2800 probes passed check_dic selection.\n",
      "finish in 0.248s, 52 probes kept.\n",
      "-- check region:580 2:50900000-50905000_strand_+_gene_NRNX1-seg-581, 9876 candidate probes\n",
      "--- 3414 probes passed check_dic selection.\n",
      "finish in 0.266s, 52 probes kept.\n",
      "-- check region:581 2:50905000-50910000_strand_+_gene_NRNX1-seg-582, 9918 candidate probes\n",
      "--- 3422 probes passed check_dic selection.\n",
      "finish in 0.268s, 59 probes kept.\n",
      "-- check region:582 2:50910000-50915000_strand_+_gene_NRNX1-seg-583, 9914 candidate probes\n",
      "--- 4232 probes passed check_dic selection.\n",
      "finish in 0.296s, 65 probes kept.\n",
      "-- check region:583 2:50915000-50920000_strand_+_gene_NRNX1-seg-584, 9918 candidate probes\n",
      "--- 2544 probes passed check_dic selection.\n",
      "finish in 0.236s, 39 probes kept.\n",
      "-- check region:584 2:50920000-50925000_strand_+_gene_NRNX1-seg-585, 9918 candidate probes\n",
      "--- 3110 probes passed check_dic selection.\n",
      "finish in 0.255s, 51 probes kept.\n",
      "-- check region:585 2:50925000-50930000_strand_+_gene_NRNX1-seg-586, 9918 candidate probes\n",
      "--- 3378 probes passed check_dic selection.\n",
      "finish in 0.266s, 54 probes kept.\n",
      "-- check region:586 2:50930000-50935000_strand_+_gene_NRNX1-seg-587, 9910 candidate probes\n",
      "--- 1822 probes passed check_dic selection.\n",
      "finish in 0.211s, 35 probes kept.\n",
      "-- check region:587 2:50935000-50940000_strand_+_gene_NRNX1-seg-588, 9870 candidate probes\n",
      "--- 1728 probes passed check_dic selection.\n",
      "finish in 0.210s, 38 probes kept.\n",
      "-- check region:588 2:50940000-50945000_strand_+_gene_NRNX1-seg-589, 9918 candidate probes\n",
      "--- 1680 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 0.213s, 41 probes kept.\n",
      "-- check region:589 2:50945000-50950000_strand_+_gene_NRNX1-seg-590, 9918 candidate probes\n",
      "--- 2924 probes passed check_dic selection.\n",
      "finish in 0.249s, 52 probes kept.\n",
      "-- check region:590 2:50950000-50955000_strand_+_gene_NRNX1-seg-591, 9828 candidate probes\n",
      "--- 3694 probes passed check_dic selection.\n",
      "finish in 0.278s, 49 probes kept.\n",
      "-- check region:591 2:50955000-50960000_strand_+_gene_NRNX1-seg-592, 9918 candidate probes\n",
      "--- 2542 probes passed check_dic selection.\n",
      "finish in 0.238s, 50 probes kept.\n",
      "-- check region:592 2:50960000-50965000_strand_+_gene_NRNX1-seg-593, 9918 candidate probes\n",
      "--- 2870 probes passed check_dic selection.\n",
      "finish in 0.247s, 55 probes kept.\n",
      "-- check region:593 2:50965000-50970000_strand_+_gene_NRNX1-seg-594, 9918 candidate probes\n",
      "--- 2486 probes passed check_dic selection.\n",
      "finish in 0.234s, 45 probes kept.\n",
      "-- check region:594 2:50970000-50975000_strand_+_gene_NRNX1-seg-595, 9884 candidate probes\n",
      "--- 2628 probes passed check_dic selection.\n",
      "finish in 0.241s, 43 probes kept.\n",
      "-- check region:595 2:50975000-50980000_strand_+_gene_NRNX1-seg-596, 9918 candidate probes\n",
      "--- 2856 probes passed check_dic selection.\n",
      "finish in 0.245s, 45 probes kept.\n",
      "-- check region:596 2:50980000-50985000_strand_+_gene_NRNX1-seg-597, 9918 candidate probes\n",
      "--- 1986 probes passed check_dic selection.\n",
      "finish in 0.217s, 39 probes kept.\n",
      "-- check region:597 2:50985000-50990000_strand_+_gene_NRNX1-seg-598, 9918 candidate probes\n",
      "--- 2462 probes passed check_dic selection.\n",
      "finish in 0.232s, 46 probes kept.\n",
      "-- check region:598 2:50990000-50995000_strand_+_gene_NRNX1-seg-599, 9918 candidate probes\n",
      "--- 3086 probes passed check_dic selection.\n",
      "finish in 0.256s, 59 probes kept.\n",
      "-- check region:599 2:50995000-51000000_strand_+_gene_NRNX1-seg-600, 9918 candidate probes\n",
      "--- 3736 probes passed check_dic selection.\n",
      "finish in 0.282s, 54 probes kept.\n",
      "-- check region:600 2:51000000-51005000_strand_+_gene_NRNX1-seg-601, 9918 candidate probes\n",
      "--- 3086 probes passed check_dic selection.\n",
      "finish in 0.257s, 53 probes kept.\n",
      "-- check region:601 2:51005000-51010000_strand_+_gene_NRNX1-seg-602, 9838 candidate probes\n",
      "--- 2294 probes passed check_dic selection.\n",
      "finish in 0.227s, 42 probes kept.\n",
      "-- check region:602 2:51010000-51015000_strand_+_gene_NRNX1-seg-603, 9918 candidate probes\n",
      "--- 4138 probes passed check_dic selection.\n",
      "finish in 0.293s, 60 probes kept.\n",
      "-- check region:603 2:51015000-51020000_strand_+_gene_NRNX1-seg-604, 9880 candidate probes\n",
      "--- 2440 probes passed check_dic selection.\n",
      "finish in 0.235s, 38 probes kept.\n",
      "-- check region:604 2:51020000-51025000_strand_+_gene_NRNX1-seg-605, 9918 candidate probes\n",
      "--- 3168 probes passed check_dic selection.\n",
      "finish in 0.257s, 55 probes kept.\n",
      "-- check region:605 2:51025000-51030000_strand_+_gene_NRNX1-seg-606, 9918 candidate probes\n",
      "--- 5802 probes passed check_dic selection.\n",
      "finish in 0.352s, 67 probes kept.\n",
      "-- check region:606 2:51030000-51035000_strand_+_gene_NRNX1-seg-607, 9912 candidate probes\n",
      "--- 5394 probes passed check_dic selection.\n",
      "finish in 0.337s, 68 probes kept.\n",
      "-- check region:607 2:51035000-51040000_strand_+_gene_NRNX1-seg-608, 9910 candidate probes\n",
      "--- 3242 probes passed check_dic selection.\n",
      "finish in 0.261s, 56 probes kept.\n",
      "-- check region:608 2:51040000-51045000_strand_+_gene_NRNX1-seg-609, 9794 candidate probes\n",
      "--- 1390 probes passed check_dic selection.\n",
      "finish in 0.193s, 32 probes kept.\n",
      "-- check region:609 2:51045000-51050000_strand_+_gene_NRNX1-seg-610, 9886 candidate probes\n",
      "--- 1018 probes passed check_dic selection.\n",
      "finish in 0.185s, 27 probes kept.\n",
      "-- check region:610 2:51050000-51055000_strand_+_gene_NRNX1-seg-611, 9918 candidate probes\n",
      "--- 2714 probes passed check_dic selection.\n",
      "finish in 0.243s, 47 probes kept.\n",
      "-- check region:611 2:51055000-51060000_strand_+_gene_NRNX1-seg-612, 9908 candidate probes\n",
      "--- 1776 probes passed check_dic selection.\n",
      "finish in 0.210s, 30 probes kept.\n",
      "-- check region:612 2:51060000-51065000_strand_+_gene_NRNX1-seg-613, 9906 candidate probes\n",
      "--- 1102 probes passed check_dic selection.\n",
      "finish in 0.186s, 28 probes kept.\n",
      "-- check region:613 2:51065000-51070000_strand_+_gene_NRNX1-seg-614, 9918 candidate probes\n",
      "--- 3394 probes passed check_dic selection.\n",
      "finish in 0.265s, 49 probes kept.\n",
      "-- check region:614 2:51070000-51075000_strand_+_gene_NRNX1-seg-615, 9918 candidate probes\n",
      "--- 1818 probes passed check_dic selection.\n",
      "finish in 0.210s, 33 probes kept.\n",
      "-- check region:615 2:51075000-51080000_strand_+_gene_NRNX1-seg-616, 9890 candidate probes\n",
      "--- 1920 probes passed check_dic selection.\n",
      "finish in 0.221s, 42 probes kept.\n",
      "-- check region:616 2:51080000-51085000_strand_+_gene_NRNX1-seg-617, 9916 candidate probes\n",
      "--- 3168 probes passed check_dic selection.\n",
      "finish in 0.259s, 48 probes kept.\n",
      "-- check region:617 2:51085000-51090000_strand_+_gene_NRNX1-seg-618, 9918 candidate probes\n",
      "--- 2998 probes passed check_dic selection.\n",
      "finish in 0.255s, 60 probes kept.\n",
      "-- check region:618 2:51090000-51095000_strand_+_gene_NRNX1-seg-619, 9918 candidate probes\n",
      "--- 5934 probes passed check_dic selection.\n",
      "finish in 0.359s, 73 probes kept.\n",
      "-- check region:619 2:51095000-51100000_strand_+_gene_NRNX1-seg-620, 9918 candidate probes\n",
      "--- 6166 probes passed check_dic selection.\n",
      "finish in 0.366s, 75 probes kept.\n",
      "-- check region:620 2:51100000-51105000_strand_+_gene_NRNX1-seg-621, 9918 candidate probes\n",
      "--- 4262 probes passed check_dic selection.\n",
      "finish in 0.299s, 56 probes kept.\n",
      "-- check region:621 2:51105000-51110000_strand_+_gene_NRNX1-seg-622, 9918 candidate probes\n",
      "--- 4264 probes passed check_dic selection.\n",
      "finish in 0.300s, 65 probes kept.\n",
      "-- check region:622 2:51110000-51115000_strand_+_gene_NRNX1-seg-623, 9918 candidate probes\n",
      "--- 3122 probes passed check_dic selection.\n",
      "finish in 0.256s, 50 probes kept.\n",
      "-- check region:623 2:51115000-51120000_strand_+_gene_NRNX1-seg-624, 9918 candidate probes\n",
      "--- 4316 probes passed check_dic selection.\n",
      "finish in 0.300s, 58 probes kept.\n",
      "-- check region:624 2:51120000-51125000_strand_+_gene_NRNX1-seg-625, 9786 candidate probes\n",
      "--- 1652 probes passed check_dic selection.\n",
      "finish in 0.209s, 39 probes kept.\n",
      "-- check region:625 2:51125000-51130000_strand_+_gene_NRNX1-seg-626, 9918 candidate probes\n",
      "--- 2726 probes passed check_dic selection.\n",
      "finish in 0.241s, 48 probes kept.\n",
      "-- check region:626 2:51130000-51135000_strand_+_gene_NRNX1-seg-627, 9888 candidate probes\n",
      "--- 2734 probes passed check_dic selection.\n",
      "finish in 0.247s, 44 probes kept.\n",
      "-- check region:627 2:51135000-51140000_strand_+_gene_NRNX1-seg-628, 9918 candidate probes\n",
      "--- 1988 probes passed check_dic selection.\n",
      "finish in 0.219s, 37 probes kept.\n",
      "-- check region:628 2:51140000-51145000_strand_+_gene_NRNX1-seg-629, 9910 candidate probes\n",
      "--- 2062 probes passed check_dic selection.\n",
      "finish in 0.219s, 40 probes kept.\n",
      "-- check region:629 2:51145000-51150000_strand_+_gene_NRNX1-seg-630, 9918 candidate probes\n",
      "--- 1704 probes passed check_dic selection.\n",
      "finish in 0.207s, 34 probes kept.\n",
      "-- check region:630 2:51150000-51155000_strand_+_gene_NRNX1-seg-631, 9872 candidate probes\n",
      "--- 3718 probes passed check_dic selection.\n",
      "finish in 0.282s, 55 probes kept.\n",
      "-- check region:631 2:51155000-51160000_strand_+_gene_NRNX1-seg-632, 9918 candidate probes\n",
      "--- 3758 probes passed check_dic selection.\n",
      "finish in 0.277s, 57 probes kept.\n",
      "-- check region:632 2:51160000-51165000_strand_+_gene_NRNX1-seg-633, 9918 candidate probes\n",
      "--- 3200 probes passed check_dic selection.\n",
      "finish in 0.259s, 51 probes kept.\n",
      "-- check region:633 2:51165000-51170000_strand_+_gene_NRNX1-seg-634, 9918 candidate probes\n",
      "--- 3578 probes passed check_dic selection.\n",
      "finish in 0.280s, 64 probes kept.\n",
      "-- check region:634 2:51170000-51175000_strand_+_gene_NRNX1-seg-635, 9918 candidate probes\n",
      "--- 2578 probes passed check_dic selection.\n",
      "finish in 0.237s, 45 probes kept.\n",
      "-- check region:635 2:51175000-51180000_strand_+_gene_NRNX1-seg-636, 9918 candidate probes\n",
      "--- 1382 probes passed check_dic selection.\n",
      "finish in 0.203s, 32 probes kept.\n",
      "-- check region:636 2:51180000-51185000_strand_+_gene_NRNX1-seg-637, 9918 candidate probes\n",
      "--- 696 probes passed check_dic selection.\n",
      "finish in 0.174s, 22 probes kept.\n",
      "-- check region:637 2:51185000-51190000_strand_+_gene_NRNX1-seg-638, 9882 candidate probes\n",
      "--- 812 probes passed check_dic selection.\n",
      "finish in 0.178s, 18 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:638 2:51190000-51195000_strand_+_gene_NRNX1-seg-639, 9918 candidate probes\n",
      "--- 1676 probes passed check_dic selection.\n",
      "finish in 0.215s, 32 probes kept.\n",
      "-- check region:639 2:51195000-51200000_strand_+_gene_NRNX1-seg-640, 9918 candidate probes\n",
      "--- 3334 probes passed check_dic selection.\n",
      "finish in 0.265s, 54 probes kept.\n",
      "-- check region:640 2:51200000-51205000_strand_+_gene_NRNX1-seg-641, 9918 candidate probes\n",
      "--- 3104 probes passed check_dic selection.\n",
      "finish in 0.258s, 46 probes kept.\n",
      "-- check region:641 2:51205000-51210000_strand_+_gene_NRNX1-seg-642, 9914 candidate probes\n",
      "--- 3390 probes passed check_dic selection.\n",
      "finish in 0.269s, 45 probes kept.\n",
      "-- check region:642 2:51210000-51215000_strand_+_gene_NRNX1-seg-643, 9918 candidate probes\n",
      "--- 3810 probes passed check_dic selection.\n",
      "finish in 0.283s, 57 probes kept.\n",
      "-- check region:643 2:51215000-51220000_strand_+_gene_NRNX1-seg-644, 9906 candidate probes\n",
      "--- 3346 probes passed check_dic selection.\n",
      "finish in 0.267s, 58 probes kept.\n",
      "-- check region:644 2:51220000-51225000_strand_+_gene_NRNX1-seg-645, 9918 candidate probes\n",
      "--- 2864 probes passed check_dic selection.\n",
      "finish in 0.250s, 49 probes kept.\n",
      "-- check region:645 2:51225000-51230000_strand_+_gene_NRNX1-seg-646, 9882 candidate probes\n",
      "--- 5368 probes passed check_dic selection.\n",
      "finish in 0.337s, 69 probes kept.\n",
      "-- check region:646 2:51230000-51235000_strand_+_gene_NRNX1-seg-647, 9756 candidate probes\n",
      "--- 2072 probes passed check_dic selection.\n",
      "finish in 0.223s, 33 probes kept.\n",
      "-- check region:647 2:51235000-51240000_strand_+_gene_NRNX1-seg-648, 9918 candidate probes\n",
      "--- 1966 probes passed check_dic selection.\n",
      "finish in 0.218s, 41 probes kept.\n",
      "-- check region:648 2:51240000-51245000_strand_+_gene_NRNX1-seg-649, 9918 candidate probes\n",
      "--- 2582 probes passed check_dic selection.\n",
      "finish in 0.237s, 45 probes kept.\n",
      "-- check region:649 2:51245000-51250000_strand_+_gene_NRNX1-seg-650, 9918 candidate probes\n",
      "--- 2800 probes passed check_dic selection.\n",
      "finish in 0.244s, 44 probes kept.\n",
      "-- check region:650 2:51250000-51255000_strand_+_gene_NRNX1-seg-651, 9918 candidate probes\n",
      "--- 3082 probes passed check_dic selection.\n",
      "finish in 0.256s, 47 probes kept.\n",
      "-- check region:651 2:51255000-51260000_strand_+_gene_NRNX1-seg-652, 9918 candidate probes\n",
      "--- 1658 probes passed check_dic selection.\n",
      "finish in 0.206s, 35 probes kept.\n",
      "-- check region:652 2:51260000-51265000_strand_+_gene_NRNX1-seg-653, 9892 candidate probes\n",
      "--- 2800 probes passed check_dic selection.\n",
      "finish in 0.245s, 49 probes kept.\n",
      "-- check region:653 2:51265000-51270000_strand_+_gene_NRNX1-seg-654, 9918 candidate probes\n",
      "--- 3034 probes passed check_dic selection.\n",
      "finish in 0.256s, 57 probes kept.\n",
      "-- check region:654 2:51270000-51275000_strand_+_gene_NRNX1-seg-655, 9918 candidate probes\n",
      "--- 3398 probes passed check_dic selection.\n",
      "finish in 0.267s, 55 probes kept.\n",
      "-- check region:655 2:51275000-51280000_strand_+_gene_NRNX1-seg-656, 9894 candidate probes\n",
      "--- 2408 probes passed check_dic selection.\n",
      "finish in 0.233s, 50 probes kept.\n",
      "-- check region:656 2:51280000-51285000_strand_+_gene_NRNX1-seg-657, 9856 candidate probes\n",
      "--- 2600 probes passed check_dic selection.\n",
      "finish in 0.238s, 49 probes kept.\n",
      "-- check region:657 2:51285000-51290000_strand_+_gene_NRNX1-seg-658, 9918 candidate probes\n",
      "--- 3888 probes passed check_dic selection.\n",
      "finish in 0.283s, 60 probes kept.\n",
      "-- check region:658 2:51290000-51295000_strand_+_gene_NRNX1-seg-659, 9918 candidate probes\n",
      "--- 2922 probes passed check_dic selection.\n",
      "finish in 0.250s, 50 probes kept.\n",
      "-- check region:659 2:51295000-51300000_strand_+_gene_NRNX1-seg-660, 9910 candidate probes\n",
      "--- 3074 probes passed check_dic selection.\n",
      "finish in 0.253s, 50 probes kept.\n",
      "-- check region:660 2:51300000-51305000_strand_+_gene_NRNX1-seg-661, 9918 candidate probes\n",
      "--- 2632 probes passed check_dic selection.\n",
      "finish in 0.241s, 51 probes kept.\n",
      "-- check region:661 2:51305000-51310000_strand_+_gene_NRNX1-seg-662, 9918 candidate probes\n",
      "--- 2556 probes passed check_dic selection.\n",
      "finish in 0.238s, 46 probes kept.\n",
      "-- check region:662 2:51310000-51315000_strand_+_gene_NRNX1-seg-663, 8790 candidate probes\n",
      "--- 800 probes passed check_dic selection.\n",
      "finish in 0.163s, 16 probes kept.\n",
      "-- check region:663 2:51315000-51320000_strand_+_gene_NRNX1-seg-664, 9396 candidate probes\n",
      "--- 876 probes passed check_dic selection.\n",
      "finish in 0.179s, 15 probes kept.\n",
      "-- check region:664 2:51320000-51325000_strand_+_gene_NRNX1-seg-665, 9918 candidate probes\n",
      "--- 2908 probes passed check_dic selection.\n",
      "finish in 0.252s, 53 probes kept.\n",
      "-- check region:665 2:51325000-51330000_strand_+_gene_NRNX1-seg-666, 9918 candidate probes\n",
      "--- 3928 probes passed check_dic selection.\n",
      "finish in 0.291s, 68 probes kept.\n",
      "-- check region:666 2:51330000-51335000_strand_+_gene_NRNX1-seg-667, 9916 candidate probes\n",
      "--- 2836 probes passed check_dic selection.\n",
      "finish in 0.247s, 50 probes kept.\n",
      "-- check region:667 2:51335000-51340000_strand_+_gene_NRNX1-seg-668, 9918 candidate probes\n",
      "--- 2644 probes passed check_dic selection.\n",
      "finish in 0.241s, 49 probes kept.\n",
      "-- check region:668 2:51340000-51345000_strand_+_gene_NRNX1-seg-669, 9902 candidate probes\n",
      "--- 3022 probes passed check_dic selection.\n",
      "finish in 0.253s, 45 probes kept.\n",
      "-- check region:669 2:51345000-51350000_strand_+_gene_NRNX1-seg-670, 9882 candidate probes\n",
      "--- 1454 probes passed check_dic selection.\n",
      "finish in 0.197s, 33 probes kept.\n",
      "-- check region:670 2:51350000-51355000_strand_+_gene_NRNX1-seg-671, 9906 candidate probes\n",
      "--- 2768 probes passed check_dic selection.\n",
      "finish in 0.244s, 54 probes kept.\n",
      "-- check region:671 2:51355000-51360000_strand_+_gene_NRNX1-seg-672, 9918 candidate probes\n",
      "--- 2072 probes passed check_dic selection.\n",
      "finish in 0.220s, 38 probes kept.\n",
      "-- check region:672 2:51360000-51365000_strand_+_gene_NRNX1-seg-673, 9142 candidate probes\n",
      "--- 8 probes passed check_dic selection.\n",
      "finish in 0.145s, 1 probes kept.\n",
      "-- check region:673 2:51365000-51370000_strand_+_gene_NRNX1-seg-674, 9918 candidate probes\n",
      "--- 2646 probes passed check_dic selection.\n",
      "finish in 0.239s, 47 probes kept.\n",
      "-- check region:674 2:51370000-51375000_strand_+_gene_NRNX1-seg-675, 9910 candidate probes\n",
      "--- 2402 probes passed check_dic selection.\n",
      "finish in 0.233s, 43 probes kept.\n",
      "-- check region:675 2:51375000-51380000_strand_+_gene_NRNX1-seg-676, 9918 candidate probes\n",
      "--- 1784 probes passed check_dic selection.\n",
      "finish in 0.215s, 40 probes kept.\n",
      "-- check region:676 2:51380000-51385000_strand_+_gene_NRNX1-seg-677, 8852 candidate probes\n",
      "--- 444 probes passed check_dic selection.\n",
      "finish in 0.153s, 11 probes kept.\n",
      "-- check region:677 2:51385000-51390000_strand_+_gene_NRNX1-seg-678, 9918 candidate probes\n",
      "--- 2128 probes passed check_dic selection.\n",
      "finish in 0.226s, 44 probes kept.\n",
      "-- check region:678 2:51390000-51395000_strand_+_gene_NRNX1-seg-679, 9862 candidate probes\n",
      "--- 1158 probes passed check_dic selection.\n",
      "finish in 0.199s, 22 probes kept.\n",
      "-- check region:679 2:51395000-51400000_strand_+_gene_NRNX1-seg-680, 9912 candidate probes\n",
      "--- 228 probes passed check_dic selection.\n",
      "finish in 0.162s, 9 probes kept.\n",
      "-- check region:680 2:51400000-51405000_strand_+_gene_NRNX1-seg-681, 9908 candidate probes\n",
      "--- 1588 probes passed check_dic selection.\n",
      "finish in 0.205s, 31 probes kept.\n",
      "-- check region:681 2:51405000-51410000_strand_+_gene_NRNX1-seg-682, 9918 candidate probes\n",
      "--- 2032 probes passed check_dic selection.\n",
      "finish in 0.219s, 39 probes kept.\n",
      "-- check region:682 2:51410000-51415000_strand_+_gene_NRNX1-seg-683, 9918 candidate probes\n",
      "--- 3098 probes passed check_dic selection.\n",
      "finish in 0.256s, 53 probes kept.\n",
      "-- check region:683 2:51415000-51420000_strand_+_gene_NRNX1-seg-684, 9720 candidate probes\n",
      "--- 1152 probes passed check_dic selection.\n",
      "finish in 0.188s, 23 probes kept.\n",
      "-- check region:684 2:51420000-51425000_strand_+_gene_NRNX1-seg-685, 9918 candidate probes\n",
      "--- 2040 probes passed check_dic selection.\n",
      "finish in 0.220s, 37 probes kept.\n",
      "-- check region:685 2:51425000-51430000_strand_+_gene_NRNX1-seg-686, 9918 candidate probes\n",
      "--- 1514 probes passed check_dic selection.\n",
      "finish in 0.206s, 28 probes kept.\n",
      "-- check region:686 2:51430000-51435000_strand_+_gene_NRNX1-seg-687, 9918 candidate probes\n",
      "--- 1134 probes passed check_dic selection.\n",
      "finish in 0.192s, 27 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:687 2:51435000-51440000_strand_+_gene_NRNX1-seg-688, 9854 candidate probes\n",
      "--- 1020 probes passed check_dic selection.\n",
      "finish in 0.183s, 28 probes kept.\n",
      "-- check region:688 2:51440000-51445000_strand_+_gene_NRNX1-seg-689, 9890 candidate probes\n",
      "--- 2070 probes passed check_dic selection.\n",
      "finish in 0.219s, 45 probes kept.\n",
      "-- check region:689 2:51445000-51450000_strand_+_gene_NRNX1-seg-690, 9902 candidate probes\n",
      "--- 2346 probes passed check_dic selection.\n",
      "finish in 0.230s, 43 probes kept.\n",
      "-- check region:690 2:51450000-51455000_strand_+_gene_NRNX1-seg-691, 9918 candidate probes\n",
      "--- 1530 probes passed check_dic selection.\n",
      "finish in 0.208s, 36 probes kept.\n",
      "-- check region:691 2:51455000-51460000_strand_+_gene_NRNX1-seg-692, 9898 candidate probes\n",
      "--- 2266 probes passed check_dic selection.\n",
      "finish in 0.228s, 47 probes kept.\n",
      "-- check region:692 2:51460000-51465000_strand_+_gene_NRNX1-seg-693, 9918 candidate probes\n",
      "--- 3518 probes passed check_dic selection.\n",
      "finish in 0.270s, 54 probes kept.\n",
      "-- check region:693 2:51465000-51470000_strand_+_gene_NRNX1-seg-694, 9918 candidate probes\n",
      "--- 2660 probes passed check_dic selection.\n",
      "finish in 0.240s, 49 probes kept.\n",
      "-- check region:694 2:51470000-51475000_strand_+_gene_NRNX1-seg-695, 9918 candidate probes\n",
      "--- 3238 probes passed check_dic selection.\n",
      "finish in 0.263s, 57 probes kept.\n",
      "-- check region:695 2:51475000-51480000_strand_+_gene_NRNX1-seg-696, 9918 candidate probes\n",
      "--- 2048 probes passed check_dic selection.\n",
      "finish in 0.219s, 40 probes kept.\n",
      "-- check region:696 2:51480000-51485000_strand_+_gene_NRNX1-seg-697, 9918 candidate probes\n",
      "--- 2264 probes passed check_dic selection.\n",
      "finish in 0.230s, 42 probes kept.\n",
      "-- check region:697 2:51485000-51490000_strand_+_gene_NRNX1-seg-698, 9918 candidate probes\n",
      "--- 3700 probes passed check_dic selection.\n",
      "finish in 0.278s, 61 probes kept.\n",
      "-- check region:698 2:51490000-51495000_strand_+_gene_NRNX1-seg-699, 9918 candidate probes\n",
      "--- 2954 probes passed check_dic selection.\n",
      "finish in 0.249s, 46 probes kept.\n",
      "-- check region:699 2:51495000-51500000_strand_+_gene_NRNX1-seg-700, 9918 candidate probes\n",
      "--- 404 probes passed check_dic selection.\n",
      "finish in 0.172s, 13 probes kept.\n",
      "Wall time: 13min 23s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "pbs, pb_scores = pb_designer.check_probes()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ae777bea",
   "metadata": {},
   "source": [
    "### save probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "4b217c7c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\reports\\NRNX1_5kb.pbr\n",
      "-- number of probes kept: 34459\n"
     ]
    }
   ],
   "source": [
    "overwrite_savefile = True \n",
    "if not os.path.exists(pb_designer.save_file) or overwrite_savefile:\n",
    "    #pb_designer.plots()\n",
    "    #pb_designer.save_csv()\n",
    "    pb_designer.save_to_file()\n",
    "print(f\"-- number of probes kept: {len(pb_designer.kept_probes)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "id": "f342e259",
   "metadata": {},
   "outputs": [],
   "source": [
    "del(pb_designer)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e728e379",
   "metadata": {},
   "source": [
    "## Reload from here:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "842303ee",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pb_designer already exists in RAM, skip.\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "if 'probe_savefile' not in locals():\n",
    "    probe_savefile = os.path.join(report_folder, f'mouse_5Mb_5kb.pbr')\n",
    "    print(probe_savefile)\n",
    "\n",
    "# Parse segemented list\n",
    "if 'pb_designer' not in locals():\n",
    "    pb_designer = library_tools.design.pb_reports_class()\n",
    "    pb_designer.load_from_file(probe_savefile)\n",
    "else:\n",
    "    print(\"pb_designer already exists in RAM, skip.\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "id": "7a63afbb",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Sort genes\n",
      "- 699 genes\n",
      "-- appending region:NRNX1-seg-1 with 36 probes\n",
      "-- appending region:NRNX1-seg-2 with 40 probes\n",
      "-- appending region:NRNX1-seg-3 with 40 probes\n",
      "-- appending region:NRNX1-seg-4 with 53 probes\n",
      "-- appending region:NRNX1-seg-5 with 64 probes\n",
      "-- appending region:NRNX1-seg-6 with 53 probes\n",
      "-- appending region:NRNX1-seg-7 with 34 probes\n",
      "-- appending region:NRNX1-seg-8 with 19 probes\n",
      "-- appending region:NRNX1-seg-9 with 54 probes\n",
      "-- appending region:NRNX1-seg-10 with 66 probes\n",
      "-- appending region:NRNX1-seg-11 with 20 probes\n",
      "-- appending region:NRNX1-seg-12 with 4 probes\n",
      "-- appending region:NRNX1-seg-13 with 55 probes\n",
      "-- appending region:NRNX1-seg-14 with 67 probes\n",
      "-- appending region:NRNX1-seg-15 with 78 probes\n",
      "-- appending region:NRNX1-seg-16 with 70 probes\n",
      "-- appending region:NRNX1-seg-17 with 54 probes\n",
      "-- appending region:NRNX1-seg-18 with 36 probes\n",
      "-- appending region:NRNX1-seg-19 with 72 probes\n",
      "-- appending region:NRNX1-seg-20 with 45 probes\n",
      "-- appending region:NRNX1-seg-21 with 42 probes\n",
      "-- appending region:NRNX1-seg-22 with 65 probes\n",
      "-- appending region:NRNX1-seg-23 with 70 probes\n",
      "-- appending region:NRNX1-seg-24 with 57 probes\n",
      "-- appending region:NRNX1-seg-25 with 45 probes\n",
      "-- appending region:NRNX1-seg-26 with 55 probes\n",
      "-- appending region:NRNX1-seg-27 with 50 probes\n",
      "-- appending region:NRNX1-seg-28 with 50 probes\n",
      "-- appending region:NRNX1-seg-29 with 11 probes\n",
      "-- appending region:NRNX1-seg-30 with 47 probes\n",
      "-- appending region:NRNX1-seg-31 with 41 probes\n",
      "-- appending region:NRNX1-seg-32 with 49 probes\n",
      "-- appending region:NRNX1-seg-33 with 34 probes\n",
      "-- appending region:NRNX1-seg-34 with 43 probes\n",
      "-- appending region:NRNX1-seg-35 with 42 probes\n",
      "-- appending region:NRNX1-seg-36 with 53 probes\n",
      "-- appending region:NRNX1-seg-37 with 44 probes\n",
      "-- appending region:NRNX1-seg-38 with 48 probes\n",
      "-- appending region:NRNX1-seg-39 with 47 probes\n",
      "-- appending region:NRNX1-seg-40 with 70 probes\n",
      "-- appending region:NRNX1-seg-41 with 55 probes\n",
      "-- appending region:NRNX1-seg-42 with 61 probes\n",
      "-- appending region:NRNX1-seg-43 with 69 probes\n",
      "-- appending region:NRNX1-seg-44 with 54 probes\n",
      "-- appending region:NRNX1-seg-45 with 50 probes\n",
      "-- appending region:NRNX1-seg-46 with 42 probes\n",
      "-- appending region:NRNX1-seg-47 with 41 probes\n",
      "-- appending region:NRNX1-seg-48 with 40 probes\n",
      "-- appending region:NRNX1-seg-49 with 58 probes\n",
      "-- appending region:NRNX1-seg-50 with 50 probes\n",
      "-- appending region:NRNX1-seg-51 with 52 probes\n",
      "-- appending region:NRNX1-seg-52 with 59 probes\n",
      "-- appending region:NRNX1-seg-53 with 53 probes\n",
      "-- appending region:NRNX1-seg-54 with 66 probes\n",
      "-- appending region:NRNX1-seg-55 with 57 probes\n",
      "-- appending region:NRNX1-seg-56 with 44 probes\n",
      "-- appending region:NRNX1-seg-57 with 54 probes\n",
      "-- appending region:NRNX1-seg-58 with 48 probes\n",
      "-- appending region:NRNX1-seg-59 with 48 probes\n",
      "-- appending region:NRNX1-seg-60 with 23 probes\n",
      "-- appending region:NRNX1-seg-61 with 17 probes\n",
      "-- appending region:NRNX1-seg-62 with 43 probes\n",
      "-- appending region:NRNX1-seg-63 with 57 probes\n",
      "-- appending region:NRNX1-seg-64 with 44 probes\n",
      "-- appending region:NRNX1-seg-65 with 24 probes\n",
      "-- appending region:NRNX1-seg-66 with 45 probes\n",
      "-- appending region:NRNX1-seg-67 with 51 probes\n",
      "-- appending region:NRNX1-seg-68 with 24 probes\n",
      "-- appending region:NRNX1-seg-69 with 45 probes\n",
      "-- appending region:NRNX1-seg-70 with 42 probes\n",
      "-- appending region:NRNX1-seg-71 with 54 probes\n",
      "-- appending region:NRNX1-seg-72 with 44 probes\n",
      "-- appending region:NRNX1-seg-73 with 24 probes\n",
      "-- appending region:NRNX1-seg-74 with 36 probes\n",
      "-- appending region:NRNX1-seg-75 with 28 probes\n",
      "-- appending region:NRNX1-seg-76 with 46 probes\n",
      "-- appending region:NRNX1-seg-77 with 34 probes\n",
      "-- appending region:NRNX1-seg-78 with 13 probes\n",
      "-- appending region:NRNX1-seg-79 with 32 probes\n",
      "-- appending region:NRNX1-seg-80 with 51 probes\n",
      "-- appending region:NRNX1-seg-81 with 37 probes\n",
      "-- appending region:NRNX1-seg-82 with 48 probes\n",
      "-- appending region:NRNX1-seg-83 with 63 probes\n",
      "-- appending region:NRNX1-seg-84 with 69 probes\n",
      "-- appending region:NRNX1-seg-85 with 55 probes\n",
      "-- appending region:NRNX1-seg-86 with 46 probes\n",
      "-- appending region:NRNX1-seg-87 with 50 probes\n",
      "-- appending region:NRNX1-seg-88 with 47 probes\n",
      "-- appending region:NRNX1-seg-89 with 77 probes\n",
      "-- appending region:NRNX1-seg-90 with 52 probes\n",
      "-- appending region:NRNX1-seg-91 with 50 probes\n",
      "-- appending region:NRNX1-seg-92 with 38 probes\n",
      "-- appending region:NRNX1-seg-93 with 73 probes\n",
      "-- appending region:NRNX1-seg-94 with 45 probes\n",
      "-- appending region:NRNX1-seg-95 with 29 probes\n",
      "-- appending region:NRNX1-seg-96 with 44 probes\n",
      "-- appending region:NRNX1-seg-97 with 44 probes\n",
      "-- appending region:NRNX1-seg-98 with 37 probes\n",
      "-- appending region:NRNX1-seg-99 with 32 probes\n",
      "-- appending region:NRNX1-seg-100 with 44 probes\n",
      "-- appending region:NRNX1-seg-101 with 42 probes\n",
      "-- appending region:NRNX1-seg-102 with 49 probes\n",
      "-- appending region:NRNX1-seg-103 with 52 probes\n",
      "-- appending region:NRNX1-seg-104 with 58 probes\n",
      "-- appending region:NRNX1-seg-105 with 23 probes\n",
      "-- appending region:NRNX1-seg-106 with 69 probes\n",
      "-- appending region:NRNX1-seg-107 with 46 probes\n",
      "-- appending region:NRNX1-seg-108 with 32 probes\n",
      "-- appending region:NRNX1-seg-109 with 42 probes\n",
      "-- appending region:NRNX1-seg-110 with 66 probes\n",
      "-- appending region:NRNX1-seg-111 with 47 probes\n",
      "-- appending region:NRNX1-seg-112 with 25 probes\n",
      "-- appending region:NRNX1-seg-113 with 62 probes\n",
      "-- appending region:NRNX1-seg-114 with 57 probes\n",
      "-- appending region:NRNX1-seg-115 with 50 probes\n",
      "-- appending region:NRNX1-seg-116 with 8 probes\n",
      "-- appending region:NRNX1-seg-117 with 57 probes\n",
      "-- appending region:NRNX1-seg-118 with 58 probes\n",
      "-- appending region:NRNX1-seg-119 with 53 probes\n",
      "-- appending region:NRNX1-seg-120 with 56 probes\n",
      "-- appending region:NRNX1-seg-121 with 65 probes\n",
      "-- appending region:NRNX1-seg-122 with 49 probes\n",
      "-- appending region:NRNX1-seg-123 with 37 probes\n",
      "-- appending region:NRNX1-seg-124 with 59 probes\n",
      "-- appending region:NRNX1-seg-125 with 36 probes\n",
      "-- appending region:NRNX1-seg-126 with 27 probes\n",
      "-- appending region:NRNX1-seg-127 with 38 probes\n",
      "-- appending region:NRNX1-seg-128 with 36 probes\n",
      "-- appending region:NRNX1-seg-129 with 30 probes\n",
      "-- appending region:NRNX1-seg-130 with 46 probes\n",
      "-- appending region:NRNX1-seg-131 with 40 probes\n",
      "-- appending region:NRNX1-seg-132 with 19 probes\n",
      "-- appending region:NRNX1-seg-133 with 24 probes\n",
      "-- appending region:NRNX1-seg-134 with 28 probes\n",
      "-- appending region:NRNX1-seg-135 with 42 probes\n",
      "-- appending region:NRNX1-seg-136 with 51 probes\n",
      "-- appending region:NRNX1-seg-137 with 50 probes\n",
      "-- appending region:NRNX1-seg-138 with 51 probes\n",
      "-- appending region:NRNX1-seg-139 with 67 probes\n",
      "-- appending region:NRNX1-seg-140 with 61 probes\n",
      "-- appending region:NRNX1-seg-141 with 39 probes\n",
      "-- appending region:NRNX1-seg-142 with 31 probes\n",
      "-- appending region:NRNX1-seg-143 with 71 probes\n",
      "-- appending region:NRNX1-seg-144 with 69 probes\n",
      "-- appending region:NRNX1-seg-145 with 69 probes\n",
      "-- appending region:NRNX1-seg-146 with 69 probes\n",
      "-- appending region:NRNX1-seg-147 with 68 probes\n",
      "-- appending region:NRNX1-seg-148 with 57 probes\n",
      "-- appending region:NRNX1-seg-150 with 59 probes\n",
      "-- appending region:NRNX1-seg-151 with 69 probes\n",
      "-- appending region:NRNX1-seg-152 with 66 probes\n",
      "-- appending region:NRNX1-seg-153 with 17 probes\n",
      "-- appending region:NRNX1-seg-154 with 35 probes\n",
      "-- appending region:NRNX1-seg-155 with 59 probes\n",
      "-- appending region:NRNX1-seg-156 with 66 probes\n",
      "-- appending region:NRNX1-seg-157 with 49 probes\n",
      "-- appending region:NRNX1-seg-158 with 67 probes\n",
      "-- appending region:NRNX1-seg-159 with 58 probes\n",
      "-- appending region:NRNX1-seg-160 with 64 probes\n",
      "-- appending region:NRNX1-seg-161 with 75 probes\n",
      "-- appending region:NRNX1-seg-162 with 66 probes\n",
      "-- appending region:NRNX1-seg-163 with 34 probes\n",
      "-- appending region:NRNX1-seg-164 with 56 probes\n",
      "-- appending region:NRNX1-seg-165 with 83 probes\n",
      "-- appending region:NRNX1-seg-166 with 51 probes\n",
      "-- appending region:NRNX1-seg-167 with 6 probes\n",
      "-- appending region:NRNX1-seg-168 with 8 probes\n",
      "-- appending region:NRNX1-seg-169 with 62 probes\n",
      "-- appending region:NRNX1-seg-170 with 71 probes\n",
      "-- appending region:NRNX1-seg-171 with 63 probes\n",
      "-- appending region:NRNX1-seg-172 with 76 probes\n",
      "-- appending region:NRNX1-seg-173 with 68 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:NRNX1-seg-174 with 58 probes\n",
      "-- appending region:NRNX1-seg-175 with 71 probes\n",
      "-- appending region:NRNX1-seg-176 with 70 probes\n",
      "-- appending region:NRNX1-seg-177 with 64 probes\n",
      "-- appending region:NRNX1-seg-178 with 66 probes\n",
      "-- appending region:NRNX1-seg-179 with 71 probes\n",
      "-- appending region:NRNX1-seg-180 with 69 probes\n",
      "-- appending region:NRNX1-seg-181 with 71 probes\n",
      "-- appending region:NRNX1-seg-182 with 71 probes\n",
      "-- appending region:NRNX1-seg-183 with 56 probes\n",
      "-- appending region:NRNX1-seg-184 with 48 probes\n",
      "-- appending region:NRNX1-seg-185 with 43 probes\n",
      "-- appending region:NRNX1-seg-186 with 54 probes\n",
      "-- appending region:NRNX1-seg-187 with 65 probes\n",
      "-- appending region:NRNX1-seg-188 with 59 probes\n",
      "-- appending region:NRNX1-seg-189 with 63 probes\n",
      "-- appending region:NRNX1-seg-190 with 64 probes\n",
      "-- appending region:NRNX1-seg-191 with 52 probes\n",
      "-- appending region:NRNX1-seg-192 with 80 probes\n",
      "-- appending region:NRNX1-seg-193 with 71 probes\n",
      "-- appending region:NRNX1-seg-194 with 67 probes\n",
      "-- appending region:NRNX1-seg-195 with 72 probes\n",
      "-- appending region:NRNX1-seg-196 with 49 probes\n",
      "-- appending region:NRNX1-seg-197 with 74 probes\n",
      "-- appending region:NRNX1-seg-198 with 48 probes\n",
      "-- appending region:NRNX1-seg-199 with 68 probes\n",
      "-- appending region:NRNX1-seg-200 with 70 probes\n",
      "-- appending region:NRNX1-seg-201 with 77 probes\n",
      "-- appending region:NRNX1-seg-202 with 41 probes\n",
      "-- appending region:NRNX1-seg-203 with 45 probes\n",
      "-- appending region:NRNX1-seg-204 with 65 probes\n",
      "-- appending region:NRNX1-seg-205 with 65 probes\n",
      "-- appending region:NRNX1-seg-206 with 71 probes\n",
      "-- appending region:NRNX1-seg-207 with 70 probes\n",
      "-- appending region:NRNX1-seg-208 with 51 probes\n",
      "-- appending region:NRNX1-seg-209 with 75 probes\n",
      "-- appending region:NRNX1-seg-210 with 68 probes\n",
      "-- appending region:NRNX1-seg-211 with 62 probes\n",
      "-- appending region:NRNX1-seg-212 with 23 probes\n",
      "-- appending region:NRNX1-seg-213 with 49 probes\n",
      "-- appending region:NRNX1-seg-214 with 73 probes\n",
      "-- appending region:NRNX1-seg-215 with 30 probes\n",
      "-- appending region:NRNX1-seg-216 with 27 probes\n",
      "-- appending region:NRNX1-seg-217 with 24 probes\n",
      "-- appending region:NRNX1-seg-218 with 44 probes\n",
      "-- appending region:NRNX1-seg-219 with 23 probes\n",
      "-- appending region:NRNX1-seg-220 with 35 probes\n",
      "-- appending region:NRNX1-seg-221 with 73 probes\n",
      "-- appending region:NRNX1-seg-222 with 71 probes\n",
      "-- appending region:NRNX1-seg-223 with 75 probes\n",
      "-- appending region:NRNX1-seg-224 with 74 probes\n",
      "-- appending region:NRNX1-seg-225 with 60 probes\n",
      "-- appending region:NRNX1-seg-226 with 54 probes\n",
      "-- appending region:NRNX1-seg-227 with 41 probes\n",
      "-- appending region:NRNX1-seg-228 with 29 probes\n",
      "-- appending region:NRNX1-seg-229 with 62 probes\n",
      "-- appending region:NRNX1-seg-230 with 70 probes\n",
      "-- appending region:NRNX1-seg-231 with 70 probes\n",
      "-- appending region:NRNX1-seg-232 with 29 probes\n",
      "-- appending region:NRNX1-seg-233 with 65 probes\n",
      "-- appending region:NRNX1-seg-234 with 54 probes\n",
      "-- appending region:NRNX1-seg-235 with 64 probes\n",
      "-- appending region:NRNX1-seg-236 with 54 probes\n",
      "-- appending region:NRNX1-seg-237 with 5 probes\n",
      "-- appending region:NRNX1-seg-238 with 53 probes\n",
      "-- appending region:NRNX1-seg-239 with 73 probes\n",
      "-- appending region:NRNX1-seg-240 with 66 probes\n",
      "-- appending region:NRNX1-seg-241 with 49 probes\n",
      "-- appending region:NRNX1-seg-242 with 53 probes\n",
      "-- appending region:NRNX1-seg-243 with 7 probes\n",
      "-- appending region:NRNX1-seg-244 with 54 probes\n",
      "-- appending region:NRNX1-seg-245 with 64 probes\n",
      "-- appending region:NRNX1-seg-246 with 38 probes\n",
      "-- appending region:NRNX1-seg-247 with 9 probes\n",
      "-- appending region:NRNX1-seg-248 with 68 probes\n",
      "-- appending region:NRNX1-seg-249 with 41 probes\n",
      "-- appending region:NRNX1-seg-250 with 17 probes\n",
      "-- appending region:NRNX1-seg-251 with 68 probes\n",
      "-- appending region:NRNX1-seg-252 with 53 probes\n",
      "-- appending region:NRNX1-seg-253 with 68 probes\n",
      "-- appending region:NRNX1-seg-254 with 50 probes\n",
      "-- appending region:NRNX1-seg-255 with 64 probes\n",
      "-- appending region:NRNX1-seg-256 with 63 probes\n",
      "-- appending region:NRNX1-seg-257 with 52 probes\n",
      "-- appending region:NRNX1-seg-258 with 63 probes\n",
      "-- appending region:NRNX1-seg-259 with 54 probes\n",
      "-- appending region:NRNX1-seg-260 with 50 probes\n",
      "-- appending region:NRNX1-seg-261 with 3 probes\n",
      "-- appending region:NRNX1-seg-262 with 42 probes\n",
      "-- appending region:NRNX1-seg-263 with 25 probes\n",
      "-- appending region:NRNX1-seg-264 with 35 probes\n",
      "-- appending region:NRNX1-seg-265 with 32 probes\n",
      "-- appending region:NRNX1-seg-266 with 33 probes\n",
      "-- appending region:NRNX1-seg-267 with 55 probes\n",
      "-- appending region:NRNX1-seg-268 with 70 probes\n",
      "-- appending region:NRNX1-seg-269 with 63 probes\n",
      "-- appending region:NRNX1-seg-270 with 50 probes\n",
      "-- appending region:NRNX1-seg-271 with 59 probes\n",
      "-- appending region:NRNX1-seg-272 with 64 probes\n",
      "-- appending region:NRNX1-seg-273 with 37 probes\n",
      "-- appending region:NRNX1-seg-274 with 65 probes\n",
      "-- appending region:NRNX1-seg-275 with 56 probes\n",
      "-- appending region:NRNX1-seg-276 with 57 probes\n",
      "-- appending region:NRNX1-seg-277 with 46 probes\n",
      "-- appending region:NRNX1-seg-278 with 24 probes\n",
      "-- appending region:NRNX1-seg-279 with 28 probes\n",
      "-- appending region:NRNX1-seg-280 with 50 probes\n",
      "-- appending region:NRNX1-seg-281 with 63 probes\n",
      "-- appending region:NRNX1-seg-282 with 46 probes\n",
      "-- appending region:NRNX1-seg-283 with 57 probes\n",
      "-- appending region:NRNX1-seg-284 with 61 probes\n",
      "-- appending region:NRNX1-seg-285 with 51 probes\n",
      "-- appending region:NRNX1-seg-286 with 39 probes\n",
      "-- appending region:NRNX1-seg-287 with 26 probes\n",
      "-- appending region:NRNX1-seg-288 with 29 probes\n",
      "-- appending region:NRNX1-seg-289 with 45 probes\n",
      "-- appending region:NRNX1-seg-290 with 36 probes\n",
      "-- appending region:NRNX1-seg-291 with 39 probes\n",
      "-- appending region:NRNX1-seg-292 with 57 probes\n",
      "-- appending region:NRNX1-seg-293 with 27 probes\n",
      "-- appending region:NRNX1-seg-294 with 31 probes\n",
      "-- appending region:NRNX1-seg-295 with 44 probes\n",
      "-- appending region:NRNX1-seg-296 with 17 probes\n",
      "-- appending region:NRNX1-seg-297 with 8 probes\n",
      "-- appending region:NRNX1-seg-298 with 42 probes\n",
      "-- appending region:NRNX1-seg-299 with 39 probes\n",
      "-- appending region:NRNX1-seg-300 with 36 probes\n",
      "-- appending region:NRNX1-seg-301 with 54 probes\n",
      "-- appending region:NRNX1-seg-302 with 31 probes\n",
      "-- appending region:NRNX1-seg-303 with 50 probes\n",
      "-- appending region:NRNX1-seg-304 with 58 probes\n",
      "-- appending region:NRNX1-seg-305 with 46 probes\n",
      "-- appending region:NRNX1-seg-306 with 49 probes\n",
      "-- appending region:NRNX1-seg-307 with 22 probes\n",
      "-- appending region:NRNX1-seg-308 with 24 probes\n",
      "-- appending region:NRNX1-seg-309 with 19 probes\n",
      "-- appending region:NRNX1-seg-310 with 13 probes\n",
      "-- appending region:NRNX1-seg-311 with 58 probes\n",
      "-- appending region:NRNX1-seg-312 with 57 probes\n",
      "-- appending region:NRNX1-seg-313 with 76 probes\n",
      "-- appending region:NRNX1-seg-314 with 54 probes\n",
      "-- appending region:NRNX1-seg-315 with 42 probes\n",
      "-- appending region:NRNX1-seg-316 with 48 probes\n",
      "-- appending region:NRNX1-seg-317 with 44 probes\n",
      "-- appending region:NRNX1-seg-318 with 34 probes\n",
      "-- appending region:NRNX1-seg-319 with 35 probes\n",
      "-- appending region:NRNX1-seg-320 with 50 probes\n",
      "-- appending region:NRNX1-seg-321 with 69 probes\n",
      "-- appending region:NRNX1-seg-322 with 55 probes\n",
      "-- appending region:NRNX1-seg-323 with 41 probes\n",
      "-- appending region:NRNX1-seg-324 with 57 probes\n",
      "-- appending region:NRNX1-seg-325 with 59 probes\n",
      "-- appending region:NRNX1-seg-326 with 64 probes\n",
      "-- appending region:NRNX1-seg-327 with 64 probes\n",
      "-- appending region:NRNX1-seg-328 with 34 probes\n",
      "-- appending region:NRNX1-seg-329 with 61 probes\n",
      "-- appending region:NRNX1-seg-330 with 31 probes\n",
      "-- appending region:NRNX1-seg-331 with 53 probes\n",
      "-- appending region:NRNX1-seg-332 with 60 probes\n",
      "-- appending region:NRNX1-seg-333 with 52 probes\n",
      "-- appending region:NRNX1-seg-334 with 65 probes\n",
      "-- appending region:NRNX1-seg-335 with 52 probes\n",
      "-- appending region:NRNX1-seg-336 with 61 probes\n",
      "-- appending region:NRNX1-seg-337 with 35 probes\n",
      "-- appending region:NRNX1-seg-338 with 49 probes\n",
      "-- appending region:NRNX1-seg-339 with 50 probes\n",
      "-- appending region:NRNX1-seg-340 with 46 probes\n",
      "-- appending region:NRNX1-seg-341 with 49 probes\n",
      "-- appending region:NRNX1-seg-342 with 65 probes\n",
      "-- appending region:NRNX1-seg-343 with 50 probes\n",
      "-- appending region:NRNX1-seg-344 with 56 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:NRNX1-seg-345 with 38 probes\n",
      "-- appending region:NRNX1-seg-346 with 42 probes\n",
      "-- appending region:NRNX1-seg-347 with 25 probes\n",
      "-- appending region:NRNX1-seg-348 with 44 probes\n",
      "-- appending region:NRNX1-seg-349 with 35 probes\n",
      "-- appending region:NRNX1-seg-350 with 37 probes\n",
      "-- appending region:NRNX1-seg-351 with 53 probes\n",
      "-- appending region:NRNX1-seg-352 with 36 probes\n",
      "-- appending region:NRNX1-seg-353 with 59 probes\n",
      "-- appending region:NRNX1-seg-354 with 50 probes\n",
      "-- appending region:NRNX1-seg-355 with 60 probes\n",
      "-- appending region:NRNX1-seg-356 with 40 probes\n",
      "-- appending region:NRNX1-seg-357 with 73 probes\n",
      "-- appending region:NRNX1-seg-358 with 47 probes\n",
      "-- appending region:NRNX1-seg-359 with 52 probes\n",
      "-- appending region:NRNX1-seg-360 with 56 probes\n",
      "-- appending region:NRNX1-seg-361 with 61 probes\n",
      "-- appending region:NRNX1-seg-362 with 40 probes\n",
      "-- appending region:NRNX1-seg-363 with 55 probes\n",
      "-- appending region:NRNX1-seg-364 with 57 probes\n",
      "-- appending region:NRNX1-seg-365 with 37 probes\n",
      "-- appending region:NRNX1-seg-366 with 42 probes\n",
      "-- appending region:NRNX1-seg-367 with 64 probes\n",
      "-- appending region:NRNX1-seg-368 with 47 probes\n",
      "-- appending region:NRNX1-seg-369 with 56 probes\n",
      "-- appending region:NRNX1-seg-370 with 51 probes\n",
      "-- appending region:NRNX1-seg-371 with 47 probes\n",
      "-- appending region:NRNX1-seg-372 with 65 probes\n",
      "-- appending region:NRNX1-seg-373 with 55 probes\n",
      "-- appending region:NRNX1-seg-374 with 31 probes\n",
      "-- appending region:NRNX1-seg-375 with 21 probes\n",
      "-- appending region:NRNX1-seg-376 with 25 probes\n",
      "-- appending region:NRNX1-seg-377 with 19 probes\n",
      "-- appending region:NRNX1-seg-378 with 15 probes\n",
      "-- appending region:NRNX1-seg-379 with 30 probes\n",
      "-- appending region:NRNX1-seg-380 with 49 probes\n",
      "-- appending region:NRNX1-seg-381 with 60 probes\n",
      "-- appending region:NRNX1-seg-382 with 43 probes\n",
      "-- appending region:NRNX1-seg-383 with 52 probes\n",
      "-- appending region:NRNX1-seg-384 with 41 probes\n",
      "-- appending region:NRNX1-seg-385 with 55 probes\n",
      "-- appending region:NRNX1-seg-386 with 58 probes\n",
      "-- appending region:NRNX1-seg-387 with 45 probes\n",
      "-- appending region:NRNX1-seg-388 with 62 probes\n",
      "-- appending region:NRNX1-seg-389 with 60 probes\n",
      "-- appending region:NRNX1-seg-390 with 32 probes\n",
      "-- appending region:NRNX1-seg-391 with 61 probes\n",
      "-- appending region:NRNX1-seg-392 with 67 probes\n",
      "-- appending region:NRNX1-seg-393 with 49 probes\n",
      "-- appending region:NRNX1-seg-394 with 53 probes\n",
      "-- appending region:NRNX1-seg-395 with 56 probes\n",
      "-- appending region:NRNX1-seg-396 with 43 probes\n",
      "-- appending region:NRNX1-seg-397 with 52 probes\n",
      "-- appending region:NRNX1-seg-398 with 47 probes\n",
      "-- appending region:NRNX1-seg-399 with 48 probes\n",
      "-- appending region:NRNX1-seg-400 with 55 probes\n",
      "-- appending region:NRNX1-seg-401 with 61 probes\n",
      "-- appending region:NRNX1-seg-402 with 55 probes\n",
      "-- appending region:NRNX1-seg-403 with 64 probes\n",
      "-- appending region:NRNX1-seg-404 with 58 probes\n",
      "-- appending region:NRNX1-seg-405 with 47 probes\n",
      "-- appending region:NRNX1-seg-406 with 54 probes\n",
      "-- appending region:NRNX1-seg-407 with 39 probes\n",
      "-- appending region:NRNX1-seg-408 with 45 probes\n",
      "-- appending region:NRNX1-seg-409 with 44 probes\n",
      "-- appending region:NRNX1-seg-410 with 51 probes\n",
      "-- appending region:NRNX1-seg-411 with 46 probes\n",
      "-- appending region:NRNX1-seg-412 with 37 probes\n",
      "-- appending region:NRNX1-seg-413 with 41 probes\n",
      "-- appending region:NRNX1-seg-414 with 58 probes\n",
      "-- appending region:NRNX1-seg-415 with 49 probes\n",
      "-- appending region:NRNX1-seg-416 with 52 probes\n",
      "-- appending region:NRNX1-seg-417 with 45 probes\n",
      "-- appending region:NRNX1-seg-418 with 41 probes\n",
      "-- appending region:NRNX1-seg-419 with 54 probes\n",
      "-- appending region:NRNX1-seg-420 with 55 probes\n",
      "-- appending region:NRNX1-seg-421 with 62 probes\n",
      "-- appending region:NRNX1-seg-422 with 48 probes\n",
      "-- appending region:NRNX1-seg-423 with 54 probes\n",
      "-- appending region:NRNX1-seg-424 with 59 probes\n",
      "-- appending region:NRNX1-seg-425 with 49 probes\n",
      "-- appending region:NRNX1-seg-426 with 53 probes\n",
      "-- appending region:NRNX1-seg-427 with 67 probes\n",
      "-- appending region:NRNX1-seg-428 with 46 probes\n",
      "-- appending region:NRNX1-seg-429 with 46 probes\n",
      "-- appending region:NRNX1-seg-430 with 58 probes\n",
      "-- appending region:NRNX1-seg-431 with 41 probes\n",
      "-- appending region:NRNX1-seg-432 with 55 probes\n",
      "-- appending region:NRNX1-seg-433 with 54 probes\n",
      "-- appending region:NRNX1-seg-434 with 52 probes\n",
      "-- appending region:NRNX1-seg-435 with 47 probes\n",
      "-- appending region:NRNX1-seg-436 with 52 probes\n",
      "-- appending region:NRNX1-seg-437 with 41 probes\n",
      "-- appending region:NRNX1-seg-438 with 34 probes\n",
      "-- appending region:NRNX1-seg-439 with 38 probes\n",
      "-- appending region:NRNX1-seg-440 with 59 probes\n",
      "-- appending region:NRNX1-seg-441 with 57 probes\n",
      "-- appending region:NRNX1-seg-442 with 50 probes\n",
      "-- appending region:NRNX1-seg-443 with 39 probes\n",
      "-- appending region:NRNX1-seg-444 with 46 probes\n",
      "-- appending region:NRNX1-seg-445 with 62 probes\n",
      "-- appending region:NRNX1-seg-446 with 61 probes\n",
      "-- appending region:NRNX1-seg-447 with 41 probes\n",
      "-- appending region:NRNX1-seg-448 with 53 probes\n",
      "-- appending region:NRNX1-seg-449 with 43 probes\n",
      "-- appending region:NRNX1-seg-450 with 51 probes\n",
      "-- appending region:NRNX1-seg-451 with 16 probes\n",
      "-- appending region:NRNX1-seg-452 with 50 probes\n",
      "-- appending region:NRNX1-seg-453 with 65 probes\n",
      "-- appending region:NRNX1-seg-454 with 48 probes\n",
      "-- appending region:NRNX1-seg-455 with 43 probes\n",
      "-- appending region:NRNX1-seg-456 with 49 probes\n",
      "-- appending region:NRNX1-seg-457 with 47 probes\n",
      "-- appending region:NRNX1-seg-458 with 44 probes\n",
      "-- appending region:NRNX1-seg-459 with 68 probes\n",
      "-- appending region:NRNX1-seg-460 with 49 probes\n",
      "-- appending region:NRNX1-seg-461 with 52 probes\n",
      "-- appending region:NRNX1-seg-462 with 49 probes\n",
      "-- appending region:NRNX1-seg-463 with 59 probes\n",
      "-- appending region:NRNX1-seg-464 with 53 probes\n",
      "-- appending region:NRNX1-seg-465 with 60 probes\n",
      "-- appending region:NRNX1-seg-466 with 34 probes\n",
      "-- appending region:NRNX1-seg-467 with 61 probes\n",
      "-- appending region:NRNX1-seg-468 with 51 probes\n",
      "-- appending region:NRNX1-seg-469 with 68 probes\n",
      "-- appending region:NRNX1-seg-470 with 79 probes\n",
      "-- appending region:NRNX1-seg-471 with 62 probes\n",
      "-- appending region:NRNX1-seg-472 with 26 probes\n",
      "-- appending region:NRNX1-seg-473 with 14 probes\n",
      "-- appending region:NRNX1-seg-474 with 61 probes\n",
      "-- appending region:NRNX1-seg-475 with 60 probes\n",
      "-- appending region:NRNX1-seg-476 with 51 probes\n",
      "-- appending region:NRNX1-seg-477 with 58 probes\n",
      "-- appending region:NRNX1-seg-478 with 52 probes\n",
      "-- appending region:NRNX1-seg-479 with 58 probes\n",
      "-- appending region:NRNX1-seg-480 with 58 probes\n",
      "-- appending region:NRNX1-seg-481 with 66 probes\n",
      "-- appending region:NRNX1-seg-482 with 60 probes\n",
      "-- appending region:NRNX1-seg-483 with 25 probes\n",
      "-- appending region:NRNX1-seg-484 with 59 probes\n",
      "-- appending region:NRNX1-seg-485 with 63 probes\n",
      "-- appending region:NRNX1-seg-486 with 50 probes\n",
      "-- appending region:NRNX1-seg-487 with 64 probes\n",
      "-- appending region:NRNX1-seg-488 with 49 probes\n",
      "-- appending region:NRNX1-seg-489 with 57 probes\n",
      "-- appending region:NRNX1-seg-490 with 58 probes\n",
      "-- appending region:NRNX1-seg-491 with 50 probes\n",
      "-- appending region:NRNX1-seg-492 with 47 probes\n",
      "-- appending region:NRNX1-seg-493 with 54 probes\n",
      "-- appending region:NRNX1-seg-494 with 49 probes\n",
      "-- appending region:NRNX1-seg-495 with 59 probes\n",
      "-- appending region:NRNX1-seg-496 with 65 probes\n",
      "-- appending region:NRNX1-seg-497 with 52 probes\n",
      "-- appending region:NRNX1-seg-498 with 81 probes\n",
      "-- appending region:NRNX1-seg-499 with 66 probes\n",
      "-- appending region:NRNX1-seg-500 with 67 probes\n",
      "-- appending region:NRNX1-seg-501 with 49 probes\n",
      "-- appending region:NRNX1-seg-502 with 65 probes\n",
      "-- appending region:NRNX1-seg-503 with 49 probes\n",
      "-- appending region:NRNX1-seg-504 with 47 probes\n",
      "-- appending region:NRNX1-seg-505 with 44 probes\n",
      "-- appending region:NRNX1-seg-506 with 60 probes\n",
      "-- appending region:NRNX1-seg-507 with 58 probes\n",
      "-- appending region:NRNX1-seg-508 with 62 probes\n",
      "-- appending region:NRNX1-seg-509 with 45 probes\n",
      "-- appending region:NRNX1-seg-510 with 55 probes\n",
      "-- appending region:NRNX1-seg-511 with 52 probes\n",
      "-- appending region:NRNX1-seg-512 with 56 probes\n",
      "-- appending region:NRNX1-seg-513 with 52 probes\n",
      "-- appending region:NRNX1-seg-514 with 51 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:NRNX1-seg-515 with 63 probes\n",
      "-- appending region:NRNX1-seg-516 with 52 probes\n",
      "-- appending region:NRNX1-seg-517 with 59 probes\n",
      "-- appending region:NRNX1-seg-518 with 55 probes\n",
      "-- appending region:NRNX1-seg-519 with 62 probes\n",
      "-- appending region:NRNX1-seg-520 with 51 probes\n",
      "-- appending region:NRNX1-seg-521 with 61 probes\n",
      "-- appending region:NRNX1-seg-522 with 50 probes\n",
      "-- appending region:NRNX1-seg-523 with 59 probes\n",
      "-- appending region:NRNX1-seg-524 with 55 probes\n",
      "-- appending region:NRNX1-seg-525 with 58 probes\n",
      "-- appending region:NRNX1-seg-526 with 44 probes\n",
      "-- appending region:NRNX1-seg-527 with 57 probes\n",
      "-- appending region:NRNX1-seg-528 with 47 probes\n",
      "-- appending region:NRNX1-seg-529 with 50 probes\n",
      "-- appending region:NRNX1-seg-530 with 61 probes\n",
      "-- appending region:NRNX1-seg-531 with 48 probes\n",
      "-- appending region:NRNX1-seg-532 with 61 probes\n",
      "-- appending region:NRNX1-seg-533 with 67 probes\n",
      "-- appending region:NRNX1-seg-534 with 52 probes\n",
      "-- appending region:NRNX1-seg-535 with 39 probes\n",
      "-- appending region:NRNX1-seg-536 with 59 probes\n",
      "-- appending region:NRNX1-seg-537 with 51 probes\n",
      "-- appending region:NRNX1-seg-538 with 65 probes\n",
      "-- appending region:NRNX1-seg-539 with 55 probes\n",
      "-- appending region:NRNX1-seg-540 with 54 probes\n",
      "-- appending region:NRNX1-seg-541 with 45 probes\n",
      "-- appending region:NRNX1-seg-542 with 65 probes\n",
      "-- appending region:NRNX1-seg-543 with 52 probes\n",
      "-- appending region:NRNX1-seg-544 with 47 probes\n",
      "-- appending region:NRNX1-seg-545 with 60 probes\n",
      "-- appending region:NRNX1-seg-546 with 52 probes\n",
      "-- appending region:NRNX1-seg-547 with 46 probes\n",
      "-- appending region:NRNX1-seg-548 with 47 probes\n",
      "-- appending region:NRNX1-seg-549 with 46 probes\n",
      "-- appending region:NRNX1-seg-550 with 56 probes\n",
      "-- appending region:NRNX1-seg-551 with 48 probes\n",
      "-- appending region:NRNX1-seg-552 with 58 probes\n",
      "-- appending region:NRNX1-seg-553 with 52 probes\n",
      "-- appending region:NRNX1-seg-554 with 65 probes\n",
      "-- appending region:NRNX1-seg-555 with 59 probes\n",
      "-- appending region:NRNX1-seg-556 with 53 probes\n",
      "-- appending region:NRNX1-seg-557 with 57 probes\n",
      "-- appending region:NRNX1-seg-558 with 59 probes\n",
      "-- appending region:NRNX1-seg-559 with 52 probes\n",
      "-- appending region:NRNX1-seg-560 with 61 probes\n",
      "-- appending region:NRNX1-seg-561 with 40 probes\n",
      "-- appending region:NRNX1-seg-562 with 49 probes\n",
      "-- appending region:NRNX1-seg-563 with 43 probes\n",
      "-- appending region:NRNX1-seg-564 with 45 probes\n",
      "-- appending region:NRNX1-seg-565 with 49 probes\n",
      "-- appending region:NRNX1-seg-566 with 41 probes\n",
      "-- appending region:NRNX1-seg-567 with 50 probes\n",
      "-- appending region:NRNX1-seg-568 with 46 probes\n",
      "-- appending region:NRNX1-seg-569 with 64 probes\n",
      "-- appending region:NRNX1-seg-570 with 61 probes\n",
      "-- appending region:NRNX1-seg-571 with 55 probes\n",
      "-- appending region:NRNX1-seg-572 with 58 probes\n",
      "-- appending region:NRNX1-seg-573 with 61 probes\n",
      "-- appending region:NRNX1-seg-574 with 43 probes\n",
      "-- appending region:NRNX1-seg-575 with 47 probes\n",
      "-- appending region:NRNX1-seg-576 with 61 probes\n",
      "-- appending region:NRNX1-seg-577 with 45 probes\n",
      "-- appending region:NRNX1-seg-578 with 39 probes\n",
      "-- appending region:NRNX1-seg-579 with 41 probes\n",
      "-- appending region:NRNX1-seg-580 with 52 probes\n",
      "-- appending region:NRNX1-seg-581 with 52 probes\n",
      "-- appending region:NRNX1-seg-582 with 59 probes\n",
      "-- appending region:NRNX1-seg-583 with 65 probes\n",
      "-- appending region:NRNX1-seg-584 with 39 probes\n",
      "-- appending region:NRNX1-seg-585 with 51 probes\n",
      "-- appending region:NRNX1-seg-586 with 54 probes\n",
      "-- appending region:NRNX1-seg-587 with 35 probes\n",
      "-- appending region:NRNX1-seg-588 with 38 probes\n",
      "-- appending region:NRNX1-seg-589 with 41 probes\n",
      "-- appending region:NRNX1-seg-590 with 52 probes\n",
      "-- appending region:NRNX1-seg-591 with 49 probes\n",
      "-- appending region:NRNX1-seg-592 with 50 probes\n",
      "-- appending region:NRNX1-seg-593 with 55 probes\n",
      "-- appending region:NRNX1-seg-594 with 45 probes\n",
      "-- appending region:NRNX1-seg-595 with 43 probes\n",
      "-- appending region:NRNX1-seg-596 with 45 probes\n",
      "-- appending region:NRNX1-seg-597 with 39 probes\n",
      "-- appending region:NRNX1-seg-598 with 46 probes\n",
      "-- appending region:NRNX1-seg-599 with 59 probes\n",
      "-- appending region:NRNX1-seg-600 with 54 probes\n",
      "-- appending region:NRNX1-seg-601 with 53 probes\n",
      "-- appending region:NRNX1-seg-602 with 42 probes\n",
      "-- appending region:NRNX1-seg-603 with 60 probes\n",
      "-- appending region:NRNX1-seg-604 with 38 probes\n",
      "-- appending region:NRNX1-seg-605 with 55 probes\n",
      "-- appending region:NRNX1-seg-606 with 67 probes\n",
      "-- appending region:NRNX1-seg-607 with 68 probes\n",
      "-- appending region:NRNX1-seg-608 with 56 probes\n",
      "-- appending region:NRNX1-seg-609 with 32 probes\n",
      "-- appending region:NRNX1-seg-610 with 27 probes\n",
      "-- appending region:NRNX1-seg-611 with 47 probes\n",
      "-- appending region:NRNX1-seg-612 with 30 probes\n",
      "-- appending region:NRNX1-seg-613 with 28 probes\n",
      "-- appending region:NRNX1-seg-614 with 49 probes\n",
      "-- appending region:NRNX1-seg-615 with 33 probes\n",
      "-- appending region:NRNX1-seg-616 with 42 probes\n",
      "-- appending region:NRNX1-seg-617 with 48 probes\n",
      "-- appending region:NRNX1-seg-618 with 60 probes\n",
      "-- appending region:NRNX1-seg-619 with 73 probes\n",
      "-- appending region:NRNX1-seg-620 with 75 probes\n",
      "-- appending region:NRNX1-seg-621 with 56 probes\n",
      "-- appending region:NRNX1-seg-622 with 65 probes\n",
      "-- appending region:NRNX1-seg-623 with 50 probes\n",
      "-- appending region:NRNX1-seg-624 with 58 probes\n",
      "-- appending region:NRNX1-seg-625 with 39 probes\n",
      "-- appending region:NRNX1-seg-626 with 48 probes\n",
      "-- appending region:NRNX1-seg-627 with 44 probes\n",
      "-- appending region:NRNX1-seg-628 with 37 probes\n",
      "-- appending region:NRNX1-seg-629 with 40 probes\n",
      "-- appending region:NRNX1-seg-630 with 34 probes\n",
      "-- appending region:NRNX1-seg-631 with 55 probes\n",
      "-- appending region:NRNX1-seg-632 with 57 probes\n",
      "-- appending region:NRNX1-seg-633 with 51 probes\n",
      "-- appending region:NRNX1-seg-634 with 64 probes\n",
      "-- appending region:NRNX1-seg-635 with 45 probes\n",
      "-- appending region:NRNX1-seg-636 with 32 probes\n",
      "-- appending region:NRNX1-seg-637 with 22 probes\n",
      "-- appending region:NRNX1-seg-638 with 18 probes\n",
      "-- appending region:NRNX1-seg-639 with 32 probes\n",
      "-- appending region:NRNX1-seg-640 with 54 probes\n",
      "-- appending region:NRNX1-seg-641 with 46 probes\n",
      "-- appending region:NRNX1-seg-642 with 45 probes\n",
      "-- appending region:NRNX1-seg-643 with 57 probes\n",
      "-- appending region:NRNX1-seg-644 with 58 probes\n",
      "-- appending region:NRNX1-seg-645 with 49 probes\n",
      "-- appending region:NRNX1-seg-646 with 69 probes\n",
      "-- appending region:NRNX1-seg-647 with 33 probes\n",
      "-- appending region:NRNX1-seg-648 with 41 probes\n",
      "-- appending region:NRNX1-seg-649 with 45 probes\n",
      "-- appending region:NRNX1-seg-650 with 44 probes\n",
      "-- appending region:NRNX1-seg-651 with 47 probes\n",
      "-- appending region:NRNX1-seg-652 with 35 probes\n",
      "-- appending region:NRNX1-seg-653 with 49 probes\n",
      "-- appending region:NRNX1-seg-654 with 57 probes\n",
      "-- appending region:NRNX1-seg-655 with 55 probes\n",
      "-- appending region:NRNX1-seg-656 with 50 probes\n",
      "-- appending region:NRNX1-seg-657 with 49 probes\n",
      "-- appending region:NRNX1-seg-658 with 60 probes\n",
      "-- appending region:NRNX1-seg-659 with 50 probes\n",
      "-- appending region:NRNX1-seg-660 with 50 probes\n",
      "-- appending region:NRNX1-seg-661 with 51 probes\n",
      "-- appending region:NRNX1-seg-662 with 46 probes\n",
      "-- appending region:NRNX1-seg-663 with 16 probes\n",
      "-- appending region:NRNX1-seg-664 with 15 probes\n",
      "-- appending region:NRNX1-seg-665 with 53 probes\n",
      "-- appending region:NRNX1-seg-666 with 68 probes\n",
      "-- appending region:NRNX1-seg-667 with 50 probes\n",
      "-- appending region:NRNX1-seg-668 with 49 probes\n",
      "-- appending region:NRNX1-seg-669 with 45 probes\n",
      "-- appending region:NRNX1-seg-670 with 33 probes\n",
      "-- appending region:NRNX1-seg-671 with 54 probes\n",
      "-- appending region:NRNX1-seg-672 with 38 probes\n",
      "-- appending region:NRNX1-seg-673 with 1 probes\n",
      "-- appending region:NRNX1-seg-674 with 47 probes\n",
      "-- appending region:NRNX1-seg-675 with 43 probes\n",
      "-- appending region:NRNX1-seg-676 with 40 probes\n",
      "-- appending region:NRNX1-seg-677 with 11 probes\n",
      "-- appending region:NRNX1-seg-678 with 44 probes\n",
      "-- appending region:NRNX1-seg-679 with 22 probes\n",
      "-- appending region:NRNX1-seg-680 with 9 probes\n",
      "-- appending region:NRNX1-seg-681 with 31 probes\n",
      "-- appending region:NRNX1-seg-682 with 39 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:NRNX1-seg-683 with 53 probes\n",
      "-- appending region:NRNX1-seg-684 with 23 probes\n",
      "-- appending region:NRNX1-seg-685 with 37 probes\n",
      "-- appending region:NRNX1-seg-686 with 28 probes\n",
      "-- appending region:NRNX1-seg-687 with 27 probes\n",
      "-- appending region:NRNX1-seg-688 with 28 probes\n",
      "-- appending region:NRNX1-seg-689 with 45 probes\n",
      "-- appending region:NRNX1-seg-690 with 43 probes\n",
      "-- appending region:NRNX1-seg-691 with 36 probes\n",
      "-- appending region:NRNX1-seg-692 with 47 probes\n",
      "-- appending region:NRNX1-seg-693 with 54 probes\n",
      "-- appending region:NRNX1-seg-694 with 49 probes\n",
      "-- appending region:NRNX1-seg-695 with 57 probes\n",
      "-- appending region:NRNX1-seg-696 with 40 probes\n",
      "-- appending region:NRNX1-seg-697 with 42 probes\n",
      "-- appending region:NRNX1-seg-698 with 61 probes\n",
      "-- appending region:NRNX1-seg-699 with 46 probes\n",
      "-- appending region:NRNX1-seg-700 with 13 probes\n",
      "Saving pb_dict into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\reports\\pb_dict.pkl\n"
     ]
    }
   ],
   "source": [
    "overwrite_pb_dict = True\n",
    "verbose = True\n",
    "pb_dict_filename = os.path.join(report_folder, 'pb_dict.pkl')\n",
    "\n",
    "cand_prob_nums = []\n",
    "pb_dict = {}\n",
    "\n",
    "if not os.path.exists(pb_dict_filename) or overwrite_pb_dict:\n",
    "    # get genes\n",
    "    print(\"Sort genes\")\n",
    "    gene_names = np.unique([_v['reg_name'].split('gene_')[1] for _s, _v in pb_designer.kept_probes.items()])\n",
    "    #region_inds = np.unique()\n",
    "    print(f\"- {len(gene_names)} genes\")\n",
    "    for _gene_name in sorted(gene_names, key=lambda v:int(v.split('seg-')[1])):\n",
    "        _sel_pbs = {_s:_v for _s, _v in pb_designer.kept_probes.items() if _v['reg_name'].split('gene_')[1]==_gene_name}\n",
    "\n",
    "        if len(_sel_pbs) > 0:\n",
    "            cand_prob_nums.append(len(_sel_pbs))\n",
    "            if verbose:\n",
    "                print(f\"-- appending region:{_gene_name} with {len(_sel_pbs)} probes\")\n",
    "            _splitted_pbde = library_tools.design.pb_reports_class()\n",
    "            _splitted_pbde.kept_probes = _sel_pbs\n",
    "            pb_dict[_gene_name] = _splitted_pbde\n",
    "    # sazve\n",
    "    print(f\"Saving pb_dict into file: {pb_dict_filename}\")\n",
    "    pickle.dump(pb_dict, open(pb_dict_filename, 'wb'))\n",
    "else:\n",
    "    print(f\"Loading pb_dict from file: {pb_dict_filename}\")\n",
    "    pb_dict = pickle.load(open(pb_dict_filename, 'rb'))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "a78ce7a4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7snQf4FtWx/4cmKCKIHVFElKAgWBB71Kix5GrQ2GJDozdXiS02/igWLIloNJqokMQaiB2IaKImEkWMUgUViUGRrmIBK0UR/s9s7nL3t7/dPXPK7p593+8+T54k7DlzZj4z+/7mPe/snCZr1qxZQ7hAAARAAARAAARAAARAoE4INEECXCeehpkgAAIgAAIgAAIgAAIBASTACAQQAAEQAAEQAAEQAIG6IoAEuK7cDWNBAARAAARAAARAAASQACMGQAAEQAAEQAAEQAAE6ooAEuC6cjeMBQEQAAEQAAEQAAEQQAKMGAABEAABEAABEAABEKgrAkiA68rdMBYEQAAEQAAEQAAEQAAJMGIABEAABEAABEAABECgrgggAa4rd8NYEAABEAABEAABEAABJMCIARAAARAAARAAARAAgboigAS4rtwNY0EABEAABEAABEAABJAAIwZAAARAAARAAARAAATqigAS4LpyN4wFARAAARAAARAAARBAAowYAAEQAAEQAAEQAAEQqCsCSIDryt0wFgRAAARAAARAAARAAAkwYgAEQAAEQAAEQAAEQKCuCCABrit3w1gQAAEQAAEQAAEQAAEkwIgBEAABEAABEAABEACBuiKABLiu3A1jQQAEQAAEQAAEQAAEkAAjBkAABEAABEAABEAABOqKABLgunI3jAUBEAABEAABEAABEEACjBgAARAAARAAARAAARCoKwJIgOvK3TAWBEAABEAABEAABEAACTBiAARAAARAAARAAARAoK4IIAGuK3fDWBAAARAAARAAARAAASTAiAEQAAEQAAEQAAEQAIG6IoAEuK7cDWNBAARAAARAAARAAASQACMGQAAEQAAEQAAEQAAE6ooAEuC6cjeMBQEQAAEQAAEQAAEQQAKMGAABEAABEAABEAABEKgrAkiA68rdMBYEQAAEQAAEQAAEQAAJMGIABEAABEAABEAABECgrgggAa4rd8NYEAABEAABEAABEAABJMCIARAAARAAARAAARAAgboigAS4rtwNY0EABEAABEAABEAABJAAIwZAAARAAARAAARAAATqigAS4LpyN4wFARAAARAAARAAARBAAowYAAEQAAEQAAEQAAEQqCsCSIDryt0wFgRAAARAAARAAARAAAkwYgAEaoTA6aefTg888EADa5o0aULrrbcebb/99nTaaafRBRdcQE2bNnVi8dy5c6lz5840YMAAuvHGG53IrIKQAw44gN566y364IMPSlGXfXrCCSfQww8/7Gz9O+64I/Dhxx9/TH379nUq20TJNWvW0JVXXkm/+93v6Msvv6TzzjuPPvzwwyC+ly9fTq1ataIw3sP/z+usXr2a5syZQ126dDFZNnHONttsQ5tvvjlNmDDBmUwIAgEQKJ8AEuDyfQANQMAJgTAh+PWvf00bb7xxIJMTCU4gRo8eTX//+9+DROI3v/mNk/WQANdGAjxjxgzaaaedaJdddqFzzjknSB6/973vOYkRUyFPPfUUHXnkkXTQQQfRiSeeSL169aJVq1bR7Nmz6aSTTgq+xMUT4M8//5wOPvjgQHeXX8j+/Oc/07rrrkuHHnqoqTmYBwIg4CEBJMAeOgUqgYAJgTAh4B0w3rWKXt9++y3tvffeNGXKFFq4cCFtscUWJks0mIMEuDYS4AcffJBOPvlkeuSRR+j444+3jgsXAn7xi1/QFVdcQRMnTqQ+ffokiownwPUajy54QwYI1CMBJMD16HXYXJMEshJgNvjWW2+liy++mEaNGkVHH320NYN6TThqrQTi/vvvpzPOOIOefvppOuyww6zjwoWAa665hgYPHkz/+te/qFu3bkiAXUCFDBAAgQYEkAAjIECgRgioEuDbbruNfv7zn6/d6ePxL730Ev2///f/gjpe/on5rrvuCnYDP/30U7rqqquC0onFixdTx44dg7pTrsvkmmK+wgSYk+oWLVrQvffeG5Rb7LfffsFP0DvvvHMDspxg3XDDDTRt2jRq1qwZ7bvvvnT99dfTrrvumumBUM/HHnuMLrnkkqAWk2tAf/jDH9KvfvUrat++fTA/LWkaNmxY8NP+888/T5y8hnqzvvyTOv8327vHHnsENafrrLMOnX/++fSPf/yD2rVrF9ROX3fddYHOfIUJMNejXnrppfTvf/+btt1226C8pH///g1sWbRoEQ0aNIj++te/BmtwLTbr8rOf/WztuDABHTlyZOAH3qH/yU9+QnfeeWcil7AGePfddyf2KdftMkPWMV66wKyYy8svv0z8K0Dv3r0Dv3JpQWjLuHHjGqwT/oLAerC/Q93ZRk6U2d8hixdeeIEOPPBAuu+++4LSmpkzZ9IhhxxCTz75ZCBz+PDhgY787xw3vO4vf/nLzBpd/vVi3rx5DXTiUp74jm/0/7OdrEf0Cu1YuXJlEHe8071gwQLabLPN6Ljjjgu4tGnTpkEs85dE1v2f//wn7bjjjjR16tTAt9EaYPZ/8+bNaeDAgcEu9WuvvRbEyY9//GPinWuOzfCaNWtW4FNmzPXJXF99zDHHBLEbxmONfPzADBCoHAEkwJVzGRQGgWQCqgT4Bz/4QZDMvP3227TddtsFCcWjjz5KrVu3DpJgTtDOOussatu2Le2zzz5BYvff//3f1LNnzyDp/OMf/xj8OyeGnCSGiSTP5zmcAHISzQkPJx2TJk2iHXbYIVCWkzxO6jjpPfbYY4NE+e6776b333+fxo4dG5RnpF2s5xNPPEGc+HECseeeewaJOydXnMiwDSYJMCf1G264If3P//xPkBhxMv2d73yHvvrqqyCZ2muvvehPf/oTvfjii/T73/8+YBEmja+88kqgT79+/YL6WdZh/PjxQRLKCS9f7733HnGS+s033wRJLydezz77LI0ZMyZIsG+//fa1bDix5GSMddlyyy0DPQ4//PDUBJiZcxLGLzXyPE6WOWH929/+tjYR5Jrv//qv/wp2UDmJDxPSN954I/gSxH7gMY8//nhgHyfz7Gv+deCTTz4JvhBwXS0n9ZyU8lj2Q5R5mACzPlyry4n4BhtsQKecckqwg8tJJtfy8s4yv8TGX7A4EefSBo7BpItrbjnW+MvXTTfdFJTrsLysBPizzz6jhx56KPiCxzbzlzW2g5NRXpsTUI5tto/tZ3u55pl91rJlywaxzAk8z2G/nXvuuYHt8QSYE9svvvgiiOkePXoQ68zPFjNknfmaP38+7bbbbsEzEfrpD3/4A3300Ue0ZMkSJMD4IAeBkgkgAS7ZAVgeBFwRCBOEV199lbbaaqtALCcbnNzxLufQoUPpRz/6UZDw8BWO53ucgIXX5ZdfHuzSjRgxItgNDq9bbrkl2IHlBJf/oEcTYP6pOlyTEwze/eUEhNfiJIrv8c4ZJ1DhxUkLJ4+bbLJJsNOWdoV68q4y76aFFyepnAizHN5d1N0B3nTTTemdd95ZuwvIiR3rG31RkGXzOE6qeIeWL7aDEyrmcdFFFwX/xknOd7/7XWL2vOu70UYbBXw5ieMdwmhNNrPj3VL+d07Iwh1gTlLjXTySmHDizf/hLxi8o8sX79Lz7nLXrl2DOm/ebeT/zwk+J+u8Q88XfzHhHXqOCd5l5S8ySSUQ/KIZJ5ScIPKXlvBiNtwxghM+3sUME2C2PbqT/O677wbrcwIZJvosg5NCThg5yQx5JtmY5MusBJgT3aSSnNC2eNkP+4V3YtkPbFM4l7uacExEO6UkJcBsK3fh4ESbr5D3ihUrAv/zxckx+3Py5Mlrf+XgeGL7+csKdoBdffJBDgiYEUACbMYNs0DAOwJJbdBCJXm3kJMaTl7Cn33D8Zys8B/+8OKffvkPOZcHcKIVXl9//XWQrPLOGSc+YdLw05/+NCgdiF7f//73g5/d+Q8+J0u828jJwBFHHNFgHP8cz4k5JwS885l0pe1sc/LJHS84qeIEWzcB5l1F3kUOL9655Z/K44kJy+b/sD18cQLMiS6XHnACGV7hy2ScOPLLZFyawbvI0TV4LH9Z4ISR1+IvG2GSFv8ikhZg7BPmy7vJ0Yv9wDuMvKvOSRgnx7wjffbZZzcYxzvv/PM9/8zPO+/xBJi/NPFP+jyfWUQvTrR5N5R3vnlemACzH3nHN7zCL0v8awF/yYle/GWLd/35ixHHZdLlKgHm3WdOVjnG4xd/WWAbedc8jOWkLyFJCTDHAv9SEH6xYNmcUPPuPn8Z4pIN7sTCO+K8cx69hgwZEvziggQ4LcLx7yBQDAEkwMVwxiogkDuBMFHknVv+uZ0v3sni8gT+GZx/po5e4XguR4je45ZPnORxzW784uR36dKlQcIQJg1cN8k/PUcv/tmcE1tOxjjx5T/4WRcnFJwsJl2hnpxwhPXHPI5lcjIR1nrqJsBcy8plD+EVzud61bB0g+8lJUDMgHdwoxfvyHLZAO+ec5LHiWLWFX5xCBNQbv3FZSqqixPgaAlFOJ5/eucdct7x5d1dLknIujhh57rVeAIcJrmcOLMP4xfvKnNbMk5+wwSYv1hF65q55INrr7MuLhFJ60biKgFmP3LP5rSLk2Au9QljmX/huPnmmxsMT/I/xwiXdEQv5s2lJZz8cgkJJ8D8HMRruflXEC7lQQKsinTcB4F8CSABzpcvpINAYQRUNcBxRZIOEuAx/HMylxckJcCc+PDBA1wDGSYNYUlEVD4nT7wrzPWOvCvJO52cCHDCkXRxrSQnVklXmp7SBJiTM/6ZO/4SXPwAj7QEOikB4p1tfpkvenGdNCfxnEBx6UiHDh3oqKOOCtZOuvg+77brdmHgBPjCCy8Mdr+jF5eI8M4u/+TOdd68488vsfFuc9LVvXv3IAGNr88HfPC/pyXA/IWKE33eOQ0TYE6UozvNXMvMdbZcUsLjky4urYi+MBYd4yoB5lpqLvvgXe+ki7/scV17VkeTJP8nHYQSTYBDhvzFkL8gRi/euecaYyTAhX00YiEQSCSABBiBAQI1QsBVAsy7ZlwCwT8bx0sgeFeLX+zin7DDpIFrWjkJjl6cQE+fPj142YdrJcOa0viuJCeN/FM4J2lpyZA0Aeaf+/mneF6XE/Xw4iSQu024TIC5zpl3AMNuCLwW73Szrlxvyj+9c+LH9bbPPPNMAzb8pYBfrOOf3zt16mSUAHP9LZeWRC/edWYduDSDdzW5vIF3o+O776w7l41wAso6ppVA8JcSTnCjF+/oc+LO9a333HNPagLM6/KXHt6N5pcWoxfHDscVx0g0vvJIgPnwCv51gXfs4+UW/OIi1ynzrxquE2CuCeYyEo5r3tmPXvEvZDXy8QMzQKByBJAAV85lUBgEkgm4SoDDndX4S3BcLsBvufPOI+9AhkkD/9TPda38B58v3oHkHUKup+Tkin8O5hpa3u3kulN+654vTo75JTD+yZh/sk+rB5UmwLzTzGUF3JKL5/DFdctch/nmm286TYC5rpRrfcOEntfhRI9foOJ6Zu6EwHXA/KIXJ/n8pSG8wvKQv/zlL0FNtMkOMH9Z4J/hw9ptTmi5zIUPjeCklTsY8M4ll8DwSW/hLizvhrKevIPJevLLeknrh7uZ8Zfg+KU23snnl8j4Z/y0HWDmzS978Q44jw1fKuNyFf533n3mkpG0y2QHmF/s23rrrRt0YuCuE1yawbHLJS/hxckvv8AWdm1wnQDzOlwnzSURXCrDO9F88RdLfjZef/117ADjgxwESiaABLhkB2B5EHBFwFUCzO3QOJHiHWBOKKNt0HjXMt46ipMorjnmuk/efeTdYE5yubMDtxrji9+2551ilnXqqacGyS7XiHIpBSeS4dv0SSykCXCYaPPanNhw0sfJMNc4846oyx1gTmr4ZSdOCPnlPf6ywK29uOyDmfHF/DjZ4ZIRTnq57Re/FMZJEdf68gtTnBiaJMDMnH++5y8irAfz5bIM7ooR9l/mxJM7W/CXD27hxjy4vRgnnlw7fdlllwV6Jq3PCSHHANdds+6caHPJA9ev8u4zy+bd27QEmOVywsk///MuOL8EuWzZsuAlTI4RfjGM/91lAsyc+QVPTja59IA7nnBt+/777x98CeGXHnnXm8tDOInnL27sM+7ykUcCzF9K+MsXf8Hjmu1wt51jnnVldqwbLhAAgXIIIAEuhztWBQHnBFwlwKwYJylXX3118DM7/2/eWeMXprjGlBMvvsKkgXfXeAeXk0DeeeQWV/xvfIBA9OKkif+dSxS4ewJ3B2B5af1uw7nSBJjHc2kB72BzlwauKeZdaO6YcPDBBztNgDmp5kMQWH/e1eSki7tIxI8S5ntclsHJIyeoXPLA9cGcoIccTRJg7oDBCRXvcHIJCZc7cO0x/5wfvXinmss/ONHjRIx3ifmLCCeD4ZW2PvuUbeISDu55ywk8lz7w/PhBGPEa4FA278rzPf6FYP311w/KPjiu4mUR8YfBZAeYZTADroXmBJP78vLLnJzEMwPe9eVdb/6yxjHKa4St+/JIgFkf3qXnF+v4SyMz4y8CHCv85SOpPMT5hwIEggAIpBJAAozgAAEQAAEQAAHHBLhGnNsGxuucwzZo3GYw/iXRsQoQBwIgkEEACTDCAwRAAARAAAQcE+DdZ65L5pKHcMeceyxzpxDeceZOEdEDNxwvD3EgAAIKAkiAESIgAAIgAAIg4JgAH2xy5plnBt0uuPSBO0NwWzguS5EeeuJYJYgDARCIEEACjHAAARAAARAAgRwI8AuP3DWFu25wAszt+bj+m7tj4AIBECiXQM0nwPzyw3PPPdegnyW/hMMvckyZMiVoA8Rv6PKLCuHFH1R8rCc3T+f+kfzmML9swi+B4AIBEAABEAABEAABEKg2gZpOgLkdE78tzc3Iw4bu3CqJ34TmHpZ8j9vjcJsfTnC5kTxfnPzy/+cWStziiN/Y5ZZG3NuS317HBQIgAAIgAAIgAAIgUF0CNZkAL1q0iM4666yg9Qy3ueG2N2ECzCcUcQ9IfgkhbLzPJxbx6U38MxU3tOfTrm666aa1R3tyX1Q+/YjrtuInWVXX9dAcBEAABEAABEAABOqTQE0mwE8++WTQXJ/LH6699tog2Q0TYD55iU+sevDBB9d6nEskuC/k4sWLg7HcvJ4b53ft2nXtGC6D4Cb+vDOMCwRAAARAAARAAARAoLoEajIBjrqDm+hHE2BOYrnxPvdiDC9uVs5Hc/IRrnx6D58gxKcWhY3qeRw3uOd/i5/rHl2LG9Lzf8KLj73kNjh8ilLaMa/VDR1oDgIgAAIgAAIg4DMBPinyo48+Cg4e4iPUcf0fgbpLgPlFtpNOOinYGQ4vru/t0qVLUDLByTIf1cr9GqM9GvlEqffeey94oS7tCk8vQoCBAAiAAAiAAAiAgC8E+Aj03Xff3Rd1vNCj7hJg/hbEZRBJO8B8fConw+G59fEd4JUrV9ITTzyR6rj4DjDv/vIRpRx4W2yxhRcOhxIgAAIgAAIgAAL1QeD999+nPn36BEe2b7PNNvVhtNDKukuAOfnl1mfDhw9fiyipBvidd94JdoXDi2uAuYcjv0AnvfjceX4JjxPhjh07SqdhHAiAAAiAAAiAAAhYE0Aeko6w7hLgG2+8kYYNG0Z8Dnt4POXAgQNp9OjRQRcI3uXddNNN6dZbbw1O8eHLtAsEAs/62YUAEAABEAABEAABQwLIQ5AAr+0C8eGHHwZ9gPkkHu7vy+UJ55xzTpAU9+vXLyB1xRVX0O9+97ug7Rn/ZMAn93Bt8BtvvKHVBxiBZ/jEYhoIgAAIgAAIgIA1AeQhSIAbnATH3R749Ldp06YFtbkXX3wxnXvuuWsp8Qtw3BuYD8JYvnx5cJAGlz7o1s8g8KyfXQgAARAAARAAARAwJIA8pI4TYMOYcTINgecEI4SAAAiAAAiAAAgYEEAeggTYIGzspyDw7BlCAgiAAAiAAAiAgBkB5CFIgM0ix3IWAs8SIKaDAAiAAAiAAAgYE0AeggTYOHhsJiLwbOhhLgiAAAiAAAiAgA0B5CFIgG3ix3guAs8YHSaCAAiAAAiAAAhYEkAeggTYMoTMpiPwzLhhFgiAAAiAAAiAgD0B5CFIgO2jyEACAs8AGqaAAAiAAAiAAAg4IYA8BAmwk0DSFYLA0yWG8SAAAiAAAiAAAq4IIA9BAuwqlrTkIPC0cGEwCIAACIAACICAQwLIQ5AAOwwnuSgEnpwVRoIACIAACIAACLglgDwECbDbiBJKQ+AJQWEYCIAACIAACICAcwLIQ5AAOw8qiUAEnoQSxoAACIAACIAACORBAHkIEuA84kopE4GnRIQBIAACIAACIAACORFAHoIEOKfQyhaLwCsFOxYFARAAARAAARAgIuQhSIBLeRAQeKVgx6IgAAIgAAIgAAJIgDNjoMmaNWvWIEryIYAEOB+ukAoCIAACIAACIKAmgDwEO8DqKMlhBAIvB6gQCQIgAAIgAAIgICKAPAQJsChQXA9C4LkmCnkgAAIgAAIgAAJSAshDkABLY8XpOASeU5wQBgIgAAIgAAIgoEEAeQgSYI1wcTcUgeeOJSSBAAiAAAiAAAjoEUAeggRYL2IcjUbgOQIJMSAAAiAAAiAAAtoEkIcgAdYOGhcTEHguKEIGCIAACIAACICACQHkIUiATeLGeg4CzxohBIAACIAACIAACBgSQB6CBNgwdOymIfDs+GE2CIAACIAACICAOQHkIUiAzaPHYiYCzwIepoIACIAACIAACFgRQB6CBNgqgEwnI/BMyWEeCIAACIAACICALQHkIUiAbWPIaD4CzwgbJoEACIAACIAACDgggDwECbCDMNIXgcDTZ4YZIAACIAACIAACbgggD0EC7CaSNKUg8DSBYTgIgAAIgAAIgIAzAshDkAA7CyYdQQg8HVoYCwIgAAIgAAIg4JIA8hAkwC7jSSwLgSdGhYEgAAIgAAIgAAKOCSAPQQLsOKRk4hB4Mk4YBQIgAAIgAAIg4J4A8hAkwO6jSiARgSeAhCEgAAIgAAIgAAK5EEAeggQ4l8BSCUXgqQjhPgiAAAiAAAiAQF4EkIcgAc4rtjLlIvBKwY5FQQAEQAAEQAAEiAh5CBLgUh4EBF4p2LEoCIAACIAACIAAEuDMGGiyZs2aNYiSfAggAc6HK6SCAAiAAAiAAAioCSAPwQ6wOkpyGIHAywEqRIIACIAACIAACIgIIA9BAiwKFNeDEHiuiUIeCIAACIAACICAlADyECTA0lhxOg6B5xQnhIEACIAACIAACGgQQB6CBFgjXNwNReC5YwlJIAACIAACIAACegSQhyAB1osYR6MReI5AQgwIgAAIgAAIgIA2AeQhSIC1g8bFBASeC4qQAQIgAAIgAAIgYEIAeQgSYJO4sZ6DwLNGCAEgAAIgAAIgAAKGBJCHIAE2DB27aQg8O36YDQIgAAIgAAIgYE4AeQgSYPPosZiJwLOAh6kgAAIgAAIgAAJWBJCHIAG2CiDTyQg8U3KYBwIgAAIgAAIgYEsAeQgSYNsYMpqPwDPChkkgAAIgAAIgAAIOCCAPQQLsIIz0RSDw9JlhBgiAAAiAAAiAgBsCyEOQALuJJE0pCDxNYBgOAiAAAiAAAiDgjADyECTAzoJJRxACT4cWxoIACIAACIAACLgkgDwECbDLeBLLQuCJUWEgCIAACIAACICAYwLIQ5AAOw4pmTgEnowTRoEACIAACIAACLgngDwECbD7qBJIROAJIGEICIAACIAACIBALgSQhyABziWwVEIReCpCuA8CIAACIAACIJAXAeQhSIDziq1MuQi8UrBjURAAARAAARAAASJCHoIEuJQHAYFXCnYsCgIgAAIgAAIggAQ4MwaarFmzZg2iJB8CSIDz4QqpIAACIAACIAACagLIQ7ADrI6SHEYg8HKACpEgAAIgAAIgAAIiAshDkACLAsX1IASea6KQBwIgAAIgAAIgICWAPAQJsDRWnI5D4DnFWfPCvl61mka+upDmL1lGndqvRz/arSO1aNa05u2GgSAAAiAAAvkQQB6CBDifyFJIReCVgr2Si3Lye9o9E2nCnCVr9d9z241o+Jl9kARX0qNQGgRAAATKJ4A8BAlwKVGIwCsFeyUXfWjSfBo46o1Gut94zE50Yp+tK2kTlAYBEAABECiXAPIQJMClRCACrxTslVx0yDNv0dAXZjfS/ZwDutCAw7pV0iYoDQIgAAIgUC4B5CFIgEuJQAReKdgruSh2gCvpNigNAiAAAl4TQB6CBLiUAEXglYK9kouiBriSboPSIAACIOA1AeQhSIBLCVAEXinYK7voN9+uppFTF9I8dIGorA+hOAiAAAj4RAB5CBLgUuIRgVcKdiwKAiAAAiAAAiCAo5AzYwBHIef4iCABzhEuRIMACIAACIAACGQSQB6CHeBSHhEEXinYsSgIgAAI5E4AB9fkjhgLOCCAPAQJcCMC33zzDV1zzTU0fPhwWrp0Ke288840ZMgQ2nvvvYOx06dPpwsuuICmTJlCG220EZ1//vl0ySWXaIUjAk8LFwaDAAiAQCUI4KXVSrgJSqIEIjMG6rYE4uqrr6bf//739MADD9C2224bJL+PPvoo/etf/6KWLVtSt27dqG/fvnTRRRfRhAkTqH///nTXXXfRGWecIX6okACLUWEgCIAACFSGANoWVsZVda8o8hDsADciwDu+Bx10EN1yyy3Bvc8//5zatm1Ljz/+OM2aNYvuvPNOmjt3LjVv3jy4f/nll9OoUaPorbfeEj9QCDwxKgwEARAAgcoQwME1lXFV3SuKPAQJcCMCnPxyYDz77LO01VZb0a9//WsaNGgQvfnmm3TeeedRu3bt6MEHH1w777nnnqNDDjmEFi9eTJtuuqnooULgiTBhUA4EUJ+YA1SIBIH/JYAdYIRCVQggD0EC3IjAjBkz6IQTTqCZM2dSs2bNqGnTpsHu71FHHUU9e/akww8/PCiLCC8e1717d5o8eTL17t1bFPsIPBEmDHJMAPWJjoFCHAjECOA6vwaTAAAgAElEQVQZQ0hUhQDyECTAjQg89thjdPvtt9Oll15KW265Jf3hD3+gRx55hF588UU65phj6KSTTqJrr7127bx3332XunTpQuPHj6d99903kSiXUfB/wuv999+nPn360IIFC6hjx45VeV6gZ8UJYHeq4g6E+pUggINrKuGmulcSCTAS4AYE5s2bR9tvvz2NHTuW9ttvv7X3+H9vvPHG9M4779ARRxyRuAP86quv0i677JJIlLtKDB48uNE9JMB1/xlUKADUJxaKG4uBAAiAgLcEkAAjAW5AgHd/jz/+eFq+fDm1atVq7T3eDf7rX/9KnTp1ClqfcYu08JLUAGMH2NvPgLpSDDvAdeVuGCskgLp4ISgMqykCSICRADcgwG3N9tprL5o4cWJQohBehx12WJAQ77nnnjRs2DCaPXt2UB/M18CBA2n06NHoAlFTHw21aQzqE2vTr7DKnACeCXN2mFltAkiAkQA3ILB69Wo64IAD6KOPPgp6+3J97h//+Ef65S9/SS+99FLQF5j7APMLcZdddhlNmjSJzjnnnCAp7tevn/hpQOCJUWGgYwKoT3QMFOIqTQC/ilTafVDeggDyECTAjQjw6W/c9uypp54KToLbaaed6Be/+AXtv//+wVju9sCnv02bNo222GILuvjii+ncc8/VCkMEnhYuDAYBEACBXAigLj4XrBBaAQLIQ5AAlxKmvgQeat9KcT8WBQEQ8IQAdoA9cQTUKJyAL3lI4YYLFqzbo5AFbKyH+BB4qH2zdiMEgAAIVJwAPgcr7kCob0zAhzzEWPmcJyIBzhGwD4GHnY8cHQzRIAAClSGAuvjKuAqKOiTgQx7i0BynopAAO8XZUJgPgYfatxwdDNEgAAIgAAIg4DEBH/IQX/EgAc7RMz4EXi3uAKOm2Txowc6cHWaCAAiAQNUI+JCH+MoMCXCOnvEh8Gqt9q3W7Mkx/BqJBrsiaWMtEAABECifgA95SPkUkjVAApyjZ3wJvFqqfavFHe0cQ7CBaLArijTWAQEQAAE/CPiSh/hBo6EWSIBz9AoCzz1c1DSbMwU7c3aYCQIgAAJVJIA8JN1rSIBzjGgEnnu49baL6bJmt97YuY8+Ipf+yEM/yAQBEACBKAHkIUiAS3kiEHjusddTHatrW13Lc+9dvyWCn9/+gXYgAAKNCSAPQQJcynOBwMsHey3VNGcRymPHtl7Y5RF5efgjDz0hEwRAAARCAshDkACX8jQg8ErBXjOLombXL1fCH375A9qAAAioCSAPQQKsjpIcRiDwcoBaYyKzakqL3HGU1rZKx9WYmwJzivRHLfKDTSAgJVDPnzNSRtJxyEOQAEtjxek4BJ5TnDUnTFVTqrrvCoh0Hek4V3r5Jqfe7ffNH9CnNgngOXPrV+QhSIDdRpRQGgJPCKpOh0l2FIuo2ZXogR3Q/wRpEf6o08cBZoMAfmnJIQaQhyABziGs1CIReGpG9TzCl5pSqR7ScfXsU9gOAiBgRwCfM3b84rORhyABdhtRQmkIPCGoOh0m3Xm1xaOqp5PqkTbuhr496OQ9O9mqifkgAAIggFp7xzGAPAQJsOOQkolD4Mk41euoImrdJGtIxrCPeNwpd0+gSXOXNnDZHp3b04iz9qAWzZrWqythNwiAgCMC0s8jR8vVvBjkIUiASwlyBF4p2Cu1aN41pdLdXakeIybMo0F/ntGI8Y3H7EQn9tm6UuyhLAiAgJ8EpJ9Hfmrvl1bIQ5AAlxKRCLxSsGPRCAHX9XSu5cFZIAACIAAC+RFAHoIEOL/oypCMwMsfu6q+NX8N/F5BugMctSKJ6Zo1RCNfXUjPzPiAxs36SLQDDN+4jQ3wdMsT0vwkgDh36xfkIUiA3UaUUBoCTwjKcBhqxdTgdBklje/TuX2w0KQ5SxIX3HPbjWj4mX0a1ADrrqu2pL5HgGd9+79erEecu/c08hAkwO6jSiARgSeAZDHEZHfTYrnKTtWpp0tjmmT8AV03ocN6bE4/2q1joxfg4Bu34QKebnlCmp8EEOfu/YI8BAmw+6gSSETgCSBZDEE9qgW8lKlpTJOGn3NAFxpwWLdESfCNW9+Ap1uekOYnAcS5e78gD0EC7D6qBBJ9DLxaqq/CboE6CHX9rbMDnNX5Ab5R+0ZnBHjq0MLYqhJAnLv3nI95iHsrzSQ2WbOGX2/BlQcB3wKv1uqras0e1zFowkdaA5xU9xvV32Rt1/bXkjzwrCVvwpY0Aohz97HhWx7i3kJziUiAzdkpZ/oWeLX47VqnvlXpsBobYOrvJKaMZuTUhTRvyTLq1H69xLrfOD74xm1AgadbnpDmJwHEuVu/+JaHuLXOThoSYDt+mbN9CzzUV+XobA9Fw98eOgUqgQAIgECBBHzLQwo0XbkUEmAlIvMBvgWe6Y6gOYH6m6lbc5tGKC7nyF4daMxr79H8Jctoy3ataNXqNTT2Xx/S4s9W0OZtW9Gh3Ten43ffqkE3huGvzKMrn8g+tU1HX52x9ed5WAwCIAAC/hHwLQ/xiRAS4By94Vvgob4qR2cTkSu+SXLatGpOX6xYlWnAHp3b04iz9giSYJZxyt0TaNLcpQ3mxMecds9EmhDp75tW2+vKtnw9AOkgAAIgAAJRAr7lIT55Bwlwjt7wMfBQX5Wfw13tsOt0YohbE3ZmSJNxQ98edPKenYJpOvrqjM2PMCSDAAiAAAjoEPAxD9HRP8+xSIBzpIvAyxGuh6Jd1dzq9OKNYwh780p0kYwJ5euM9dA1UAkEQAAE6pIA8pB0tyMBzvGR8CXwkmo3ufndyFcXBjWl0rf6UQOaHSyudknz3AE+ZpctabO2rQKfcx3xoD9n1wiHFmfZdsyuHemRKfPpuZkfUhMiOmTHzRrVI7t6zBCDrkhCDgiAQD0Q8CUP8ZE1EuAcveJD4KGva44Ojol2VSebJKdZE6JvFR27VfW98Tri3ttsSP/+4IsGtcUbtGpOEy4/iNZbp3kD69Jsu6dfbzrjvkmZtcauPOCKryt9IAcEQAAEfCfgQx7iKyMkwDl6xofA09lNxMle9sHgqsY6Koc7PYyatqiRcttv2pqaNGlCm2+Q3AVCIiPJ4rQ4SLLt8akLaeCoNxLBZcWTCWlXO+wma2MOCIAACFSRgA95iK/ckADn6BkfAk+nnjSsH01CghrQHANFIdoFe1dxEFc1S25WPJnQdMHBZF3MAQEQAIGqEvAhD/GVHRLgHD3jQ+DlvQPMNaVDju3ZoP9sHkhtaj9t5urawmtxPezf3ly8tkfvQTtsSs2aNqVFny4X11tH103z4QFdN6HDemwuOpXNVRzEeWTJrZcd4CLjSzceMR4E6pkAnk0iH/IQX2MQCXCOnvEh8PKsAQ7RpfWOdYXWpvbTZq6u/mm9d+NydHkl2RCVKZHnKg7itkj6DetyTBtfpC+lOvuok1R3jAOBWiaAZ/M/3vUhD/E1zpAA5+gZXwIvqXaTzR45dSHN0+gCwXIGPP56Yj2q690+yQ6oZM0i60bz2mVlFqEPn57xAY2b9VGjqJWwcBUH8cVZ7qOTF9DfZi7OvQuEqxprV499kfHlSmfIAYF6IIBnEwmwKs6RAKsIWdz3JQG2MKHR1DLqMG3WtJmryy2vOtuoHkXao2t/PY6HP+rR67C5CgTwbCIBVsWpNwnw6tWr6aGHHqKXXnqJvv76a1rDjWoj17333quyxbv7tZgAl/Gt2mZNm7m6AZXHDnC8hk2nd29cf1U9nOq+Lo+s8eFacz7+ij78bAUtWfY1NW3SJNcewi71D2UVGV956F9rMrNiuMj4rjWuLu0pyg+2z2ZRerpkmySrFvMQV8y8SYAvuugiuv3226lXr17Utm3bRvY9//zzrmwuTE4tBl4ZdVU2a9rM1Q0U1zXALut2VRxU93VZqJLf0+6ZSBPmLEkcFu1n7HLdPGQVyS0P/WtJZpYveD8lHnOS2vla4uODLUU+LzZr2cz1gXNUh1rMQ1wx9iYB3mSTTWjw4MHUv39/V7aVLqdWA6+MOkybNW3m6gZRWA9738tz6Z0Pv2w0XadrRtoOxg19e1Czpk206rdVuyGq+7ocssZLdsolNc0udbKRVWR82ehZ63OzYph/T0zqV12lOKsF/xX5OcO8TJ/NovXM07e1moe4YOZNAtymTRuaPn06denSxYVdXshA4HnhhlKUcFF/5kJGaLxKluq+S4iSWmnXPYRd6g9ZfhLIimHWeOgLsxspjjgr1pdFfs7YWFYVPSU2Ig9Jp+RNAnzsscfSgQceSD/72c8kPq3EGAReJdyUi5IudhBcyAiNU8lS3XcJqdZ2gF2ygSxzAtgBNmdX1MwiP2dsbKqKnhIbkYdUIAG+6aab6JprrqHDDjuMdthhB2rZsmUDra+66iqJr70a40Pg1Uohv1eOjSmTxFhVcyjxi20dWnSNLdu1ojGvvU+TInW30RpI27V0/KPqa1ylGmAdu6swVhKXtnbktUZZNcB52cOc85Rt60eT+V+uWEVH3vES8cuv4eVjLXaRn4cmHHXm+JCH6Ohb5FhvdoA7d+6caneTJk3o3XffLZKLk7XKDrxaeoidOCQHIVmMebmkXss6fjGtYUt7ge6HvTrQwpQT6UzXMsEarvUud4H4fAUt+aqaXSBMbPd1jk5cmtqQ9xpZMZxHfOdpT56yTf1nMy/Jnm03bk1Pnb8vrbdOcxvRuczNI15yUVQhtOw8pAybpWt6kwBLFa7SuLIDr5Z+xvHV7yaMTebo2l/EGro6YbzfBIqImSLWKJJynvbkKbtIRuFatWZPGQxN1iw7DzHRuag53iXAzz77LL3++uvUokUL6t69O33ve9+jZs2aFcXD6TplB14tFfI7dYxDYSaMTeboqlzEGro6YbzfBIqImSLWKJJynvbkKbtIRuFatWZPGQxN1iw7DzHRuag53iTAn376KX3/+9+nKVOmULt27YgPxvj8889pt912o7///e/Bv1XtKivwwrqxZyyOzTVl7XvNmmv9JLsa8VrcyXOX0hPT32uEWNIiTaq/RC9TH+vMk+qrIxNj8yFQRMwUsYYpHZNYHf7KPLryiRmNlnTRXs1nViaMa80eEwZlzCkrDynDVt01vUmAzzrrLJo4cWJwGlyPHj0CO1577TU65ZRTaN9996WhQ4fq2lb6+DICT/WCUZ4vHPhes5aHfiqZKn/EgzTLP6q1orJ0xub1oPigQ1621aLcIvxVxBomvjHRi+eccvcEmjR3aYMlXb3EaaKTie1Fzak1e4riZrtOGXmIrc5FzfcmAeaDMEaOHEnf/e53G9g+btw4OuGEE+iDDz4oiomzdcoIvLRv2Qd03YQO67E5/Wi3jtSiWVNnNkYF+f4NPy/9sl6WkLT8ijsjbfdIV/+yX+LQ1TeXoIRQLQJFxEwRa2gZTUQmsZo2hw+qOXnPTroqJI73kZWNYbVmjw2LouaWkYcUZZvtOt4kwFziMGHCBOrWrVsDm9566y3addddadmyZba2Fj6/jMArs86qzLUlzi1DP8mhD3Hd05rzl6G/hGvamKrpa2Mr5labgEmsmsypNiVoX0UCZeQhVeHkTQJ80EEHBaUPt99+ewN25513Hk2dOpVefvnlqjBdq2cZgSfdyTCpd1M5QLq2Sk5e9yX6ueZS1A7wMbt2pJGvLqT5S5ZRp/brKXf62c5Hpsyn52Z+SE2I6JAdN6Pjd9/K6teBOLtVq9fQoD/nUx8Zxohrf+UVeyw3SVfuF63jN6l+VeIitSk+zqWNks+G6Pq89oCRr9PoaYsaqS+p5XdhM/f2JmpCi1LaGrrkU8bzlof+ptyrPK+MPKQqvLxJgF955ZXgJLhevXrRPvvsQ9z7d/z48cHxyE8//TRxgly1q4zAk9RZScaYsM5LrokuSXNU+qnum+ihqgFu06o5fbFi1VrRJjXA9/TrTWfeP5kmpBxykZQ4uK5dTLKzT+f2wdJph2+Y8IwnIafdM1Fst+16NvOL5JNHHNvYnsdc1zbqyFM902xvHu9aqNbN+2AbHUa2Pi9yLVtdfZ9fRh7iO5NQP28S4OAP5aRJdMstt9CMGTNozZo1tNNOO9HPf/5z2nPPPavCs4GeZQWeqs5Kd7dDB75qbR1ZeYw1qde1faM7umbHdusGZoWHURy1cwcaM/09mifcuU3S//GpC2ngqDca4dKtJWYBprZm1UM2a9pEbJ+Oz/OMYx09JGN1fgkw9UGoR5W4SNgljcnDRulnl9SXtn6M2y1ZN1wzDz55yEzzf5FrmcZgVeaVlYdUgY9XCXAVgOno6GvgoXYt2YtV5aKrd1Zdclr9sSrudXVQyZPcL2NNiV5JY3RqwU19EK5bJS6uedqyk+gj9aVrXSTrhmvmEQN5yEzjXeRaEp9XeYyveYgPTEtNgH/yk58ENb9t2rQh/t9Z17333usDLy0digw8nXopfLtOdmPRXHR8xhqnjU/TO60WMWsnyXTXSvJGvMpe1f241/Lswar1oMcGJ9VXryFKrIdOWsfEB1F2iz9bQaMSalNN5Mb10/WRDcesuUU8q7rPW1xfF7yjMvPcAZb4tQjmob3StSR65xWDVZFbZB5SFSahnqUmwFzzO3r06OCQC/7fWdfzzz9fNbZUVODp1kvpjq8ceEOFi+Siu1bWeH6RKl4LGyJIqkVkWXnUAGfJTNLRpmYxDxsMw6bBtDS9dt9mw+C9hmg9tKsa6aTY0Kktl9qtG7NSuSbj8tZF93nLg3fSl4+055zHmj5PUpbScSb+lNga/ywrUh8XNpUlo6g8pCz7bNYtNQG2UbwKc4sKPOm35Sgzab1bFTi71LEoLro+U41nvQc8/rp454/HPzp5Af1t5mJnXSBGTJiX2vWBd0Cz6pRV9sV9LNlxdhkXUllZu3TcHzZeD81yR05daFUjnfULwGZtW4m6gkjs0/WRRKbNmDyfVZWt8bV1a/lN7c56nyDe413KR2VrWX83VPrr6G3KuxbmFZWHVJGVNwnwiy++mMiPd03WWWcd6tixI2255ZaVYlxU4KFeqlJhESir6zPJeMmYPEllrc/rDn1hdqPlTWsWy7Y1jWMe9dUqnxXFoqh1VPYWcR+2ErmuYXbtt3rykQ27ovIQGx3LmutNAtyiRQtavXp1wIE7QPDFyW/0OuCAA4LT4rhkogpXUYFX69+Ea7HOS+Iz3bpOicw8n5us9U13gJPqmJlLWg9WrrvU7Ynskkke9dUq/Yrye1HrqOwt4n4e9eW+fo5V1a9V1buI+I2uUVQeUrRdLtbzJgF+8MEH6fLLL6c77riD9ttvv8A2Phnu3HPPpXPOOSdohXbRRRfRzjvvTMOGDXNhe+4yigq8Wq6FqlXbVHaZ1HV+uWIV7XXj2AZ9hTdo1ZwmXH4QrbdO89zjXbduMlrTl6R7qLCqtjE6TrcnsmsoZdQmq2LJlY1FreNKX1M5efjQZ3Y+65blw6rqbRqXpvOKykNM9StznjcJ8HbbbUd33nknHXrooQ14jB07ls4++2x6++23iQ/LOOaYY+j9998vk5l47SIDT1UvJVbas4G1/C3fpCcx74im1XX6wCrLJhN7w3BU9TcNd4p1eyLnEe551Fer9Czq+S9qHZW9ed7Po77ch2czi1lV/VpVvfOM37jsIvOQIu1ysZY3CXDr1q2DI4+7devWwK633nqLdtllF1q+fDnNnz8/uL9s2TIXtucuA4Fnj7he67xM7DaZY+8hNxJUPU6ltcJVZuCGJKTYEsgjhvKQaWsn5tcHAeQh6X72JgHef//9adttt6W7776bmjVrFmj87bff0llnnUX/+te/gnKI4cOH04033khvvvlmJSIXgWfvprx2Tnytx2NiqhrXE/tsnQjWlpUJE5M5ScqrepyqdoCl9+0jsvYkuPJhGhkX8l3ICJ+tka8upPkZJy/aPkdJ66xavSaxQ8oBXTehw3psTvEODrUXZbCoLALIQyqQAPPu70EHHUQbbrgh7bbbbsELcfxvn332GT3zzDPB/+eX4H7729/S//zP/5QVS1rrIvC0cCUOzqPOKw+Z9pb+R0KSbqHspJ6+0XVt7DKZazInK0mS9DJWram678pPtSInb14u5LuQkfZspfXJjsei6tlTPYdJPZ+jc3Tk10rswY5iCCAPqUACzCpybS+/4DZt2jRq3rw59erVi/r370+bbLJJsAu8aNEiOvjgg4uJGgerIPAcQCQi13VeLnZ43FjWWEpWT9chx/akFs2aZi5tysqEicmcLOVD3d/9+Cv6+IuVtEmbltR549aNdsdUNqru5+W7Ksp17cM4AxfyXchgvXTk2MRQVg0x94F+esYHNG7WR43CxfXJcVWMR+jsngDykIokwKGaK1euDHr/xtuguQ+NfCUi8PLlayrd53q8snQzWddkjqnPMC8fAnn70IV8FzKYnis5Kk+o1lHdV8nHfRDQIYA8pCIJMO/+3nTTTcHLbrNmzaKbb76ZtthiC7rqqqt0/O3N2FoMPFe1eGU6SWcnSKqnKy556CaxwWRdkzkSXVyNceUTU32S1ucW56oaVMl6rmyT/uJgup6LGLGVEer+TMrOa1KvaYkP0sao9JUwZ50fmTKfnpv5obOTGrN8GL23ZbtW3IU/qJPO+iXGhlE9zzV9lkyZ1WIeYsoiPs+bl+C4DzCXO1x44YVBEjxjxgx66qmnaMCAATR48GC67LLLXNm8Vs4f//jH4KW6d999l7p06ULXXHMNHXfcccH96dOn0wUXXEBTpkyhjTbaiM4//3y65JJLtHSotcBzVYunBTGHwa7tcCnPpSwddCbrmszR0clmbNm6Ja2fVAdqUvvp0jZJzTkn7aY1sS50telvnWVfNL5M/JAWnyqbVcy5l/UZ902iSXOXNlhij87tacRZeyjLoJL0ytIpyb9JMlwysnm2qzxXFRt52FZreYhLRt4kwLvuumuQcPbr14/atGlDr732WtAV4oEHHqDrr78+6APs8hoxYgSdfvrpdMstt9APfvADeuihh4IE+KWXXqKuXbsG7db69u0bHL7BHSg4Ob/rrrvojDPOEKtRa4Gn2tkQg/FgoE2NX1x911xc6qaD2mRdkzk6OpmOde0TXT1UXS2i8nRrP13bxj4c8PjrNGraokZmsm6qU/xUbGxjxMbePP2QZbfK5izmvCOd5AteTzdWQh2zGKb5N8k+0/VVMVIv921i2ZRRreUhphyS5nmTAHMfYN717dy5c4MEeM6cObTjjjsGfYBdXXzUMifXxx57bFBmEV58CAd3muCLD+WYO3du8DIeX3xK3ahRo4j7EkuvWgs81K4lex5cpE9EcePK9omqr3GURNjjWEonD9uyZLJeQ1+Y3Ug9Xb2l9sXH2dibpx9M7Qnnpem2y9btaNr8TxPFmzI38W+SAqbr27Kqlfk2sWzKoNbyEFMOXifAXILAxyAffvjhDRJg3qm9+uqrafbsxh/ApiA4id1hhx3o1VdfDQ7ZiF9HHHEEtWvXjrgsI7yee+45OuSQQ2jx4sW06aabipautcAr49urCHTJg+qNS9E1bCbuLdsnrnceo8wXf7Yidbc2rUd0yDDNd8NfmUdXPjGjEWoXO8Am/ovOsfFlml1JOt3Qtwcd13sr6xpt6fORVQusswMsWa/sHWCJjqoYtY2jtPk6utnqYBPLpmvXWh5iysHrBJjrfocOHRqUJJx22mn02GOPBTuwgwYNCsoQrrjiCmd2jxkzhn74wx/S+PHj6brrrgvarvHOM6915JFHUs+ePYNEfMiQIWvXnDlzJnXv3p0mT55MvXv3FulSa4FXRv2SCHTJg+qJS1VsLVtPl7WnSbLatGpOX6xYtTbyJfWZaUxUNac2NcAuHk1TX/K8U+6e0KiWdvdtNgzKOqbEamz537nz0KQ5S7S4Rm3U0dXUH9FWiNL1yqwBlurIHHXGlhlbpmsXbR/rWWt5iCl7rxNgVo7LDG677TZasWJFoCuXH5x99tnBvzVtmt3/VAcK7yqfeuqpQdLLu8vcb3jkyJF0ww030N///vfgoI2TTjqJrr322rViwxflOGned999E5f7/PPPif8TXtzXuE+fPrRgwQLq2LGjjorejlXVtnmreM6K1QuXMnYwTF1Xtk+y6jx1TgDL2incrG0r6tR+PdFJYro7jrwjevKenQL8PrAcOXUhzcs4wS0eJ1n9eDkBHvTnxjveSbGmU/eq+3ykceV/f3TyAvrbzMWZXSB01svyYfRex3brBhiYtYsuEDo66ow1/VyIzit6vTKeJSTA6ZHiTQ3wuHHjaO+996ZvvvmGeLeVT37jF9E22GADF3HeQMajjz5KJ5xwQvBS2znnnLP2Hr8Mxxe3YeMyiKQd4LSyCZ7HL9Fxx4r4VUsJsHNnQGClCJRRw1YpQDFlXfByIYPV0q05rXq9Zxl1r658JY35oteT6hUdp6OjzlgTXeJzil7Phc66MpAAVyAB5tPenn32WeJuEHlfL7/8Mu2zzz5BCUR0N5dbrXHrtW222SZofTZ8+PC1qkhqgOthBzjLN0XWUuUdI65sics5slcHGvPae0GPTenOndRWVzpnrVfGjkmSPkXYKuUeHRfXa9XqNYk7jdJdRZY3YOTrNDqlQ4Oq5jeqm+4OsFRHKScdn+mMTYuPLG46nQ+iO+G8VpZukufD5DMhbU3VeuG8OYrTFSU+1PVJOD6t/3JSfKnskeipM6bo9XR0czUWCXAFEmCur+WX4A488EBXfk+Vwx0lNttss+CAjWhvX677bdGiRbATzYdy8It3zZo1C+QMHDiQRo8eXdddIFTJr2mv0NwdrrmAqzotV7WbEvVd6axaq6h1qhhrSWxsev9m1RFLan7jDLNqTs+8fzJNsKh9dRk3tjEm4ZZU18y+Wr1mTaPa4Gj/XZVuJvdV9dy69bthbGT199WNH5VdkliLjklbX3cdVdyp7he9nkqfPO4jAa5AAswvunHSyaUH22+/Pa277n/qkMLL9Wlw3FuYX7z73e9+FzH07M0AACAASURBVNTpPvzww0FCPHbs2KDtGpdfHHXUUcEBHJMmTQpKJVg/7lMsveop8Grpm7QrW6SdAFzstLnSWRLbZdeDFmmrhEc4JqvmtFnTJlr1qywza8d2yLE9jQ5EyKo51a2xdcEmj11AKbckFo9MXpC5Yy+Jvaznw+QzQbVm2nqqtXQ+d1Q6xGMhbbyk9r3oz5ei19N5blyMrac8RJeXNzXA/EJa2sVv5vJLaK6vW2+9Ndh1XrRoUdAWjet3uTsEX9ztgU9/4w4RfBzzxRdfTOeee66WCvUUeLVUS+XKFmkPUhe1lq501grwkgb7aqtrvVzLK8ldwbI6tuiMTbLJZr5qruq+irHJZ4Lpmqq1dD53dHXQHa/ihvvmBOopD9Gl5E0CrKt4FcbXS+C5rFN07Vdp3Vp0nE2f1aj+qh2YcCyf/JS1oyexQXeHxjXnIuXlbWsSb/45eeSrCzNrt13r5Vpeko8ksRWdpxqvW6vKsR/vZGFrt878UN93PvySXp23lD7+ciUtWNr40KXr+3anZk2bkrSeVZdD3DfRumMde0I5WZ/J4ZikHWBdvdN2kU10TvsMUcWc63lFfpYVsVa95CEmLJEAm1ATzqmHwJPU20X7VgrRORkmre/Kq1ZXIjc01LYmTmqrE7AlC8nTVps6Xtd6uZYXd5uufNV43VrVtNpX277DKj2jSWL8vYWk0E6q446Oiz+7uhzWb9mMvlz5bYOldeqOJX6Nj0n6vNHVO6uOWOoD1UeJqRzTeSp9qni/HvIQU78gATYlJ5hXD4EnrbcT4HI+RLoLkWWDTp/VJAPi9WVH7dyBBo2eIT7JS2oDr13rtWxRvnnZKt21Z12Sdr9c6+VaXpShTmzxPNV41f2oLapfWWztlsxX+brrZuvTT/bpTGmdPNLqWXU4cCeYlau+pavHzGz08RGNL4k9oYC09XfZqh1xMt9549aJfaN19f7Rbh0z69B1dE778Ffp5Hqe8z9CHgishzzEFDMSYFNygnn1EHg+13pJdZOOE7hcNERnPZ2xosUxKJOAqm4yOlmnhtJH7LqxpRqvuh9loDM2L3YqX++ydTsa3X8frfpl1lXXNt3xKh6m8kznqfSxuW+qk+k8G119nVsPeYgpe28S4DVr1gTHUNbS5XvgmdZWSXaRkmr7XKynEx+q3YNQn7++8T6Nf/vjzB0YnXXDsS7q6VQ2mOhVi3PirLP6LWfFoWpXMMou3h+W7xUd4zbr6caWarzqvuRzI7rraWObJMZVvubPsFtP2JmGvzKPrnyi8alxrupfXf+KZtqBQcd/Er4uxqTpFNZkp/VTV/ks79hKsr2MNVkP3/MQF3FiKsObBLh379507733Us+ePU1t8W6ez4HnqkZKUufKtWL39OtNefcajQeAbk1bdH60Bs8ksHTXtq0BNtGxVuZIY3D4mX1IVV+aVgOs6g8bJr9F9sK2fYZ156vGq+5H4001VnXfRewmrRHK3aBVc5pw+UHUvGlTOuXuCTRp7tIGS2Z9PujqnqWHbr/etDiMKl+lz5q055HtmZTSt5rnZPlM9RngIrZ0/hbl/Y6Mz3lIHqx1ZHqTAG+88cY0ceJE6tKli47+Xo/1OfBcftuX1PbxbsooB6dZ6TrctEdm0u6eztoqvjr1cTpjdXSslbGqnbzQTt6xSzsBTFVvqeoPy2uofO6at4v1dGNLNV51P8rApF+uTu9aCe9QB+4CMXXe0iA+tt24NV1/dA9ab53mqT5VfT7ocGA9efyAx1939hkZrv/0jA9o3KyPGqFI46irt4Sx7Zi4TqrTFdOei9BnLp4bXZvKWDPU0ec8RJej6/HeJMA333wz/eUvf6FLL72Utttuu0YHYWy99daubc9dns+Bl1eNVJpcrqebNv/TRszLqqNU1f/Z6pUX39yDtoILqHwZmsQ+5WvoC7O141DiT8kYl3iLXs+l7ipZvthWpB55rJWHTJXv8r6vssn2fh76q3TKY00kwGqq3iTATZs2XatttBY4rA3+9tuGbWLUppU/wucEOK9vpFn1bGXsAKdFgWrXUNWbVxVdunxN6sNM5qj0ruL9tHq/uC3SHeAkBhJ/Ssak8dXxZTg2rSdtUv29rl919NGVLRlvw1IiXzrGhR5SlnmspdotlXJwOU7Kw/QdChVH1X2XtoayylgTCbDak94kwOPGjcvUdv/991db49kInxNg3To1Kdo0uWXUAGfpnFV3F84zqb8L5+rw1RlrIl/quyqOS6v3c91jVuIjyZgkxjrzVHGbZrdOnaGOPnnFjA86sG22eujM1xkrjaOkHsY2n2u2/pbamDVOVcOrWkN139ZGqW+K8oPPeUgerHVkepMAR5VeuXIltWzZUscOL8f6Hnh51Xulyc1rPVPnh/q8+/FXNHnOEpq2oHGJhk3NodRek90BkzmmnHyel8bh2qO60zrNm9K8JcuIe61Ge5ZK/RK3WzJPMiYuV8eXWW/4t2+9jpMaUh198owNE5Z56GOjhy7LPNbi2tdmTZskPgt58MqSKeWhGqfiZHs/Dy4qnfJYk2X6nofkZbdErlcJ8LBhw2jIkCG0YMECmjVrFnFd8BZbbEFXXXWVxBbvxiDwvHNJqkJl1miZrG0ypzrekGtaCxx0bMgay9RM6pvjtHX0kXuqPkcWybLItUy9KdVROs5Uj3qahzwk3dveJMAPPvgg9e/fny688EK66aabaMaMGfTUU0/RgAEDaPDgwXTZZZdVLmbLDjzbWqs4cKm8PByV99qqHYc8bAplmqxtMidPG4qQnRQDj09dSANHvdFoeZudexNbbOJTx5dZNfa7dtqQBv25cb/atHp23RrLvr060NLl3xB3az9kx83o+N23yjwJjDmquKjum/jCpzlp9enxOm3+WX/kqwspra9taFMWL504SmJk6wvVfL4/YOTrNFrQDUhli2otSQy4kCFZp+wxZechZduftb43CfCuu+5KF1xwAfXr14/atGlDr732Gm277bb0wAMP0PXXX09vv/22zxwTdSsz8KR1Tq7H5eEkqY42axexRpp+JmubzLHhU/bcNHt9qC239YXO/KSxoW+S6j3De/F6Q90ay/VbNqMvVzZ8EVnVK1tll+p+2TFnuz7bl9SPNl6nLa3TVfFS3c+yx2Zu+EUnqwd2Vtwm1cLqxqduPa2tvbaxUeT8MvOQIu00WcubBLh169bBrm/nzp0bJMBz5syhHXfckZYvX25iX6lzygw81TfoEIzrcXkAl+pou3ZZNVqst8naJnNsGZU1PysGuL535NSFpdU4uohPHV9m9Yzlek/uZ6vquKLSOarP4s9WJMrjWMjaaVetobpfVqy5WjfNPqn8OFsJL504iuohkZ2lt2p+1i8XQ47tmfhLQpotqrUkfF3IkKzjw5gy8xAf7M/SwZsEmA/AuOOOO+jwww9vkACPGDGCrr76apo9u3HvTt/hlhl40hoq1+Py8IlUxzzWhkw/CPgcA2XoZlsLrKNzVp/lrH7ZqjVU9/2IPHMtpP2p01aIs82Tl61s1XzVfR3KLmS5kKGjc5ljy8xDyrRbsrY3CTDX/Q4dOpRuueUWOu200+ixxx6juXPn0qBBg+iiiy6iK664QmKPV2PKCLywrimtR+gBXTehw3psvvateOk3YZNv8NIaK9U4qY5eOb9gZVQMXauTth7/+yNT5tNzMz9sVCsq1TFpXBG1vqb6ldFrNeuZkJx2p/NMZe1kxj9PonGWVf/Ku35pPlWdsuY6lnXkSWOEZRa1A+yCl048sG1xDmnPwPV9u1Ozpk0p7e+RSa1+lq7H7NpRVEtta2+0q4wkfnTiRiJPZ0wZeYiOfmWO9SYBZgiXX3453XbbbbRixYqASfPmzenss88O/i16UEaZwHTWLjrwsuqs4nqHNVOqnorhPJc1XNGepJJaLMkYHb/U2tii+WTV455x3ySaNHdpA8RcK3rv6bvTmfdPpglzlqy9p1P7l3etr5Rh0jhpDafLuLOtkZTaGyY7SbWsUXuSaoyz5vB49mlavIw4aw/lC3YueUpk6TALucXrYpNiRRo/aTXFqlps17ZJn4GsmnTWSbduN+tvURhPks+YNN+k6aPr9zhv2/kS/2WNKToPsdW3yPleJcBs+LJly2jmzJm0evVq6tatG22wwQZF8nC6VtGBl/attuum69OsD79sZFv47VtaN6ZzXr30G7Z0nFRHpw6siDApQ1fmZP0akFR7yuvyW++qulQeV1atr5Rh2rgyeq1mPROS50UyJowZHvvo5AX0t5mL6f1Pl2d+nmT5MRqD/PlTxu656XMgjZGo/CTGfD9es570b0mHl4yYMC+x04fJTmqcgzQepM9Amm+zfjWQ+iZJV91fiWztlTI3iRspB8m4ovMQiU6+jPEqAf7iiy/o4YcfpjfeeIOaNWtGu+22Gx177LHUqlUrX3hp6VF04KXVNe2ydTuaNr/xIQ9Z9Xtphkprp1yP0wJfZ4OlrF1h0Y0zXlcag0XbEjKRrisd54q1j3IkDCT1r/z5w5eL3sVFcJLYnbceVdKhaF3zWs9Wru1825gqOg+x1bfI+d4kwG+99RYdeOCBxElw165dgx1gbn225ZZb0j/+8Q/q2LFjkVycrFVU4IV1l8Nfnpe4MyPdfUsyWlrrFX4bVtUgm7zZ7MQZAiFl1mkJ1EsdUvQOg8kO8FYbrksLljbu5BLWCIb9T8vaEZQylI6LOss0rkzn2cQSz1WtK2EgqX/lz6W03sVpu2sq3VxwT+MnsduWvWq+rg46vFRrh/elOkjHSddVjXO5XpRbWheUeIymsXapl4pB0v2i8hAT3cqe400CfMghh1CLFi3oT3/6E2244YYBl48//phOOukkWn/99WnUqFFls9Jev4jAS6sLC5XVrY2K/wGR1LBl1RNH5enUfA4/s0+hNYBl12lpB1dkQtG669YAp9lmUw9pwyvti15WH9Nwji5r3fGm67jiIdHXdEySjtL61zAxl/hId6yUncRuqSzTcTo66IzV0Ucq98sVq2ivG8fSFytWrRW/QavmNOHyg2i9dZrrLCkaK9VLJSxJTrx3s21fbdP6Z5XuSID1CHmTAHOSO3HiROrevXsDC/hAjP32248+//xzPcs8GF1EApy10xI9AUpa7xTFJq31Ct+ITRuvqvky0c21e8v+lm5rT9EM09YLa0UfeHlu4q8RoZ1Zu39l1NOyXlKG0nEs0zSuTOfZxpF0XQmD6JiO7dalyXOX0J+nv9dIRam/pbrZcFfxk9itkmF7X6qDDi9dnSQ65Ll+mr4SvVS2Zv3CtVnbVtSp/XpruyiFslS2utBLpXfa/SLyEFPdyp7nTQLMZQ+//e1v6dBDD23AZPz48XTyySfT/Pnzy2alvX4RgWfao1NijG7tku54iQ5Fjamy7kUx0llHVQNatfpPHdujY03jynSeqZ7hvDzXtZWtM19nrC0zX+eXzaDs9U39YqK3yRxT/XTnFZGH6Orky3hvEuAnn3ySLrzwQrr11ltp//33D8ohJk+eTP3796ezzjoreBkuvLbeemtf+GXqUUTgZe0AS99STTNC9a02Pk93vE9OrLLuPnFU7YiE9zk2Jf1qfbRNRyfTuDKdp6Nb0tg817WVrTNfZ6wtM1/nl82g7PVN/WKit8kcU/105xWRh+jq5Mt4bxLgaJ/fJk2arOWzhhvVEhH/G/9v/u9vv214Jr0vMON6FBF4PvWG5AMQhj4/m9777D99nPnKq9ZJ8nKHZEyop6v6MR9iUcfuvPRN4hmPiaQe1FwTelSvDrTo0+WJPzXmpW9ecpM4tFu3BV30/a704z5bp9a5Z9VZj3ntPQpfGtRtyB+N95GvLmwkJ2ndDu1a0c8O2I6O330rUV1+WvxlPWMcC2kHqOg+o7yOzmeR6fMimReOmfPxV/TxFyupfet1aMlXX9MmbVpS541bN/opPSkOs3iyD0PZUZkhz/jnMa/5/R03E68d6sP1vFc+MYPmfvIVdd6oNV1/dI+1dbwm/k5q8ZbXM6gr1+RvgckcXb1MxxeRh5jqVvY8bxLgcePGiVnwDnEVrqICL9qjk786HLLjZuI/ViqOktolF380VXpE70s+bCRj4mtKbNXRs4yxJnbnpWfI893//eOf9Ec/ynzLdq1ozGvv0yTFYRl56ZuX3K9WrqIf/GY8zf1kWYMlVAcYxOPxyF4dRIeJqOxQxQiv+8jk+XSXwZdZiex4D1xO1pIOzUjio3pGdT+LVPqmsZTMy/oSGP8ymJYQZn0Rih/6EMpMerFwi7YtqVWLZjTn4/+LQenmRNbLbM2bNqWsFxNV/lLFaln3TfQ2mVOEfUXlIUXY4noNbxJg14b5IK9eAq/on38k60nG+BAjrnWost1V1j3Lj67KlFzxkciRjEmy2WSeKz6sj+76uuNDmyXzsuyKsssqVct6ISvt0BmdzxRJmdzPH5lOo6ctaiSWX2TdvXN7GjjqjUb3JHJ19MRYcwL1koeYEEICbEJNOKdeAq/oFwAk60nGCN1YqWFVtrvKumcFiasXVV3xkciRjEmy2WSeKz6sj+76uuNDmyXzVC+ChrKyDiRKk5F2sIzuh5XkMKSj7/pn4kFKrAPvIlflIBNdNrUyvl7yEBN/IQE2oSacUy+BJ9kNiSPTqZ+L1ztK1pOMEboxdVhYa/jczA/JRemJhIlKZxO7Xayr0kty30R3lVwfbHO1wynhI7FXIkcyJmQfXVN6aEDUb7p8smxMkxU/cCWsnR7+yrygtjV+qXYwJXykO8BZbSLTZOyyVTuatqDx6Z6q50HXTh6ftgPcdbP1qUeHtqIjznX1koyXxLpETlIcJ7U605Hl09h6yUNMmCMBNqEmnFMvgSeph4sik4xXvTCjaogvWUPoxsRhLF9atyhZx5W+unJ0x0tsMR3jWhfX8mzschErKntU96N/6F09P0lrqg4NSPoyLOWjsjHpftphG/f0601n3DeJJs1d2kAlVW02D1bpkTYmK4akBwWFMuKcw393fcBMUg1w1A5df5s+R7p/Q3TWkfhTR55PY+slDzFhjgTYhJpwTj0Fns4LADa7J+HOjGQ9yRihKxsN0921Uq0jYaKSEd7XsdvlulL9ssbp6K5azyfbXL2omsVHx14JZ8mYrBrVtEMDkvwm5SOxMa532vHaaUfE88EcJ+/ZSRVeooNTQl3+8sb7NP7tj5Uyk3aeWcaAx19P3GllGzZu0zLoMBF9yZQXir9smPRv0m4My75eRYNGz6DJ85bQgiWNjzRnPXT8rQShGCCJA501XMvTWTvvsfWUh+iy9CoBfvDBB+m73/0udezYka6//np6+OGHaZ999qHbb7+dWrVqpWtb6eMReMkusKmfk9SsFeF4l3WLrK+ESR52lbVuHrbEZdaybUn8yrC36DVN1tOto83jM8a2HtjE7jyesVrVwxe78vAZ8pB0qt4kwJzw3nDDDTR27Nig1y8nvnwAxgsvvEBHHHEE3XbbbXnERq4yqxJ4JrWs8forbtEk7U8q+bYtqeXjtllETax6xprWkfm8A6wT1BJf6MgzHasTT1k+s61HZf11Y0J3fHwNVRxL5ZfhyzzXTLL78akLE7sORI99j8egbieFcBc2ur7KR6q4l9YD8+7zcb23oniP5jS7VbXKKr2ksRXKkXwuu6yfTdPPVdyF8p96/T365zufNMJV9M62yl8m96uSh5jYZjvHmwR42223pSFDhtBxxx1Hl156Kf3zn/+kl19+mfgo5BNPPJEWLWrchsXW+LznVyHwTGpZbev+JPVW0lq+qA+lfS3DORI90mKE6+L2/OVz9OXKhoey9NmmPf3pv/cQHRgQlW2ji00cl7Wuyva0usKkwzNCvyfd061P1OWhOz5MfuM1uGlxrCNfZ6xNzKh8p/scJumSZgvX7ab1v01bV0dWViy5/qxZv2WzRp8fu2+zYbABFO+FnWS3LWeTeJF+LtvqlvacZPlHd80kW6I+btOyOX2xctXaf9KV7+oZs5VThTzE1kbT+d4kwFzi8M477wTlDzvvvDMdffTRdPXVV9O8efNohx12oGXLGjaRNzW4yHlVCDyTnUzpbkbW7oSkvlBayxf1qc6OiM0uQtpcaf1gUhxKmOQRv2Wtq9pVituqOj457WhlnV0c3ZjQHc82SZ6fMI515ZfhyzzWzLKbOzik1cSmPf9pOqb9u46PpM9kfK2Vq76lq8fMFE1nu9jueF2vtIY3aRHd2AplSD+XdT6LTfSzjTuJj5M+g07ss7XIZ74MqkIeUhYrbxLg7bbbjn7/+99Tp06daPvttw92gPfaay8aMWIEXXfddfTvf/+7LEbG61Yh8ExqWW3r2UyBStbVqd+zqfuymWtqf63Ok/iVbWff8pXWdzTr3oDDuonw6fpVdzwrIbE3jGMT+SJDPR+kslt139Y8HR+ZriVZI5St87km1ccVQ1dy4nrnJTdcR4d/nn6Q+st0XBXyEFPbbOd5kwD/6le/ol/84he07rrr0iabbELTp0+nu+66KyiHGDx4MF1yySW2thY+vwqBl/UtOK0/pfSbs+0OQNxhknWja6rq22zq2Ux3T9gmlV6uA1V3Pd3xKn1V8iR+5TVMd4B14jCrXnTIsT0blbaYxIHEXtMd4CxfqPwQzpWOS1ora66OXBXXtPt9e3Wg3p3bE/cOj3dG0NktlfgorDvm0huu2Z2TceR3EivJGuE8nRiWPo9/TelMobuWaS/lqJ469d46+iXJDf31zIwPaNysj1S4Gty3+YVPJ/61lFIMrkIe4tJeHVneJMCs9JNPPknvvvsunXLKKbTRRhvRn/70J1q+fHnwMlwVryoEXloNcJR3vPbJtgbY1Jeqmq2onpL6Npt6Non8tARB1YPVlI+L9UztStNZIk8nnnRrgHXr9rJiTNqnVbWmqq9qtA+thJ8kXqRypON0Yy3Lb0mJqUoP1WdBXD+VT+LjJZ+LPCep524oS7WmzeePxOdSH0XHSXogxxNXaQ9n3c8I27pnKd8kvdq0bEZdN29DU+c1PGxEl08oWxXPpv6UzKtCHiKxI48xXiXAoYErV66kli1b5mFvoTKrEnjRHpzvf7qcZn34ZSNO8W/d8fqro3buQGOmv0fzliwjl28BxxWJrtux3brB7YWfLm+0pmoHKZRrU89mUoMm1ctVoOqupztepadUnk48ZXE38UlSjOnUmOquqdr5i+8y6cpP8onUD9Jxumuk1WfbvCeQ1R83ST+dnUOeP2LCPBr058anxKliPnpftWaSb3m+y1rfqD66saey1cW7EKp6b1MWKlvTbONT9vhF5lGvLkr0v8qnus9G3jXFVclDVLGWx32vEuBhw4YFnSAWLFhAs2bNoptvvpm22GILuuqqq/KwPXeZVQy8vOuucof+vwuY2mE6T2pX3vLjeuiupzteZbdrear1XN3PU29V7WGZ9Z42dmfNZb+k1W5L67OTfKtiGZ2jy1VHdlrc6a7pKn7T5Khs0tXXJl5CHV3IsI2NpDhxqZdLWboxUsU8RNdG0/HeJMB8CEb//v3pwgsvpJtuuolmzJhBTz31FA0YMCCoAb7ssstMbSxtXhUDL6sGUvekH/7Z55Ep8+m5mR9SEyI6ZMfN6Pjdt9JqEWZaN2W6k2U6TxpkruWr+Oju0LjWz7U8KWfbcSq9Vdyz1lftTJnsMKnsVdkTzrd5/tPmXt+3e/BT8uhpjVtZ2tqaVn+axEN3LZWfVMz5vu6aUZmmMZY1T2WTrr7SuDJ5HqK9nU1YqGxN0ymP+nsXnCTxljSminmIqa2687xJgHfddVe64IILqF+/ftSmTRt67bXXiHsDP/DAA8GpcG+//baubaWPr2Lg6dRjZr1UklZDp1NDZVM3ZTrXdJ402FzKl8jS9YNEptRWHudans7aNmOz9NatZ43roVtnbGNHOFfqB5vnX7fmUlUjq7JbWqfLckzWkrCwqQHOsk/qL0lsqd6NCGW4YqQrR/U8mNYC68ZjPE5MfZDkV5eyVM9F/H4V8xBdG03He5MAt27dOtj17dy5c4MEeM6cObTjjjsGL8NV7apq4EXr0hZ/tiLxDHrVTkHWt2/V3NDPtt+aTWsnTedJ49OVfCmftFpG3Z6pUvvi41zZa7q+6TzdHrHSuGZ9QtnvanYOMLUluqaqTt/m+ZfW1Ged3Ca1MS3+++7cgTbdoJVVF4hQB0ltOo/lOlWXvpQ+23FWknmuY8/F851Vz82xMsrw1wNVjXXWeyQ6z4wkZl1wkqyDBFhOyZsEuEuXLnTHHXfQ4Ycf3iAB5j7AfCDG7Nmz5VZ5MrKqCXAUn2ntUlatmbTOzHRtT9yfuxpSPtJxuStcIwvUE09bW23nZ4VMnrLLDlVT20znlW0vr5+m+y5bt6Np8xt2Y+Dx0r8jPthWpg61kIfkxc+bBJjrfocOHUq33HILnXbaafTYY4/R3LlzadCgQXTRRRfRFVdckReD3ORWLfCS6qzSzqBX9UPMcweYawqbNW0a9Po06ThhUk+WW5BYCJbs9rB46TgLVepqapk8s2I3vKfbjzbLeWk1ttLT9aSsTJ7JrHrlpJ7NtkEq0VEyRqUHyxgw8nVRzXR0vS3btaLJc5fSE9Pfa7SEzq8TPNmFHSo74+uk/dposwMc18HWLpP5JnMk7KRjqpaHSO1yMc6bBJiNufzyy+m2226jFStWBLY1b96czj777ODfmjZt6sLeQmVUKfDSapS4/uqM+ybRpLlLG7BT1fLq1p4mOUZaw6VTc1ZmLZbr4JPaIh3nWr9alVcWT9265JC/zvMR9VnaM9ymVXP6YsWqtUOz5EtYScZIPx9sbU6LWYmOkjGqZ0JVDzv8zD5rXyLOGhtdR9f/LuxQ2Rkmv/Ge6EmxZVoDnJT82vRgN+FiMkfCTmdMlfIQHbtcjPUqAWaDli1bRjNnzqTVq1dTt27daIMNNnBhZykyqhR4WTs1q1avMeqHGO0vbNoFQlpTKN3hkdRHcwAAIABJREFUkO5IlRIwBotK68qk4wxUqMspZfDMit20Pruhc6TPR9SZOm/R2/TztXkms2pHTWxOC2aJjpIxqodFZ1db4h+TWmsXdqjs5Ps6HUdcPG+2dpnMN5kjYaczpkp5iI5dLsZ6lQDzzi+3Q+OX4fggjJ122omOP/74YCe4ileVAq+MHp4mPrWtcbOdb6Iz5oCACwImz2i4rkm9pKpnbNQmE/nhfNtn0na+xDeSNSRjVGvpyJD4x8QvOjqo7Mm6X9Q6ZcZZ0TYm8a5SHmITTyZzvUmAudvDvvvuS59//jl17dqVvv322+AwjE6dOtHTTz9N22yzjYl9pc6pUuCZ7C653GEJHaWqcZTWxsUdH8pNO/9dZUvZdVylBnIFFo/758heHWjMa+95XSeuG1Mmz2joOlV8J7lYssMYzpPWBIfj+TjoK5+YQXM/+YpoDdG0BY1fcgp1VnEqYpdNsoZkjOpR0pEh8Y/E73G+pr/4RW1T+YzH6tiq4iZZW7Jelt6S+XE90+bYvseiw6NKeYiOXS7GepMAf//736dWrVrR8OHDqW3btoFtH330EZ144om03nrr0ZNPPunC3kJlVCnwdOsLdevKJOB1dQhlqnRR1cqZzFfNkdiLMW4IJPlXp0416cuSTa2gxCqT2sC8no80faU1+LqsOfnd68axDeqImzUh+pbrOP73Cp8vSc9lE5YSH8WTKlVMuNBDR4bqc439MvHyg2i9ddJ/QZX6WOfzTmqDdJyOr3SfEVWPZJ37SXrmwVeHB4+tUh6ia5vteG8SYE5yp0yZEvT8jV7Tp0+nffbZh7766itbWwufX7XAy6qzclGDpXKAyQ6XpMYtTe4BXTehw3psTj/arWPm6XQm3/xVtuK+OwKSnTBeTbIbxuOK8LfpGpJn1GU/WlUfVZM+4T9/ZHpih4NdtmpHe3bZqEFnFymnIj6fJGtIxqgiX0dGVg20JObT+HKXn2ZNm5CqZ3SSLVKf8VwdW1XcJM9u1noSvU30df0ei4RDdEzV8hBd+2zGe5MAc9nDXXfdRQcffHADe55//nk6/fTTad68eTZ2ljIXgaeH3aTGUVLjZluHZTtfjwJG6xKQ1EKyTEms8Lgi/F3EGrocTcab2HH0Xf9M7OvK/V5H99+ngRom8k3sqPKcrPhXxXwefPOQKfWPzdo2c6X6FfX5ggRY5pFSE+D58+ev1XLMmDH0q1/9KjgMg2uBmzVrRlOnTqWf/vSndOWVV9Kpp54qs8ijUT4mwK5rJU1whzrE+5Vm1Z6lveXOu3rH7NqRRr66MLXeU/LNPssO2/kmjPKeI6nRy1sHV/KlO8DSursi/F3EGhK+NnHAc9Nq8rNYp+0A8685t56wcwO1feEkYRkfY8NWtV5UdtouPMtI+9VD8k6E6nM1zd6/vvE+jX/740YmSH+BUdmexTmNBe9oH9d7K6O/E9LPDan/i45pH/MQXR/nNb7UBJh7+zZpwg2y/nOt4YIvokb/xmP4pbiqXb4FnutaSRN/ZNWt9encPhA5ac6StaJVtYCSHpG2tWa280045TmnHuyJ16VmxVaLZg17jBfBp4g1VDFko4Ppc8ysk2qAN2jVnCYk1Kva6KiyP8/7eeqdJHv9ls3oy5UN/0am9WpX1Q7zZ67kczXKTyVT1Tfe1BdSFrtvs2GQVyT9bQmff5f1uja1yKYs0ub5loe4ts9GXqkJ8Lhx48S677///uKxvgz0LfCkO2V5fVNnv6h0yKo9S6q/SjupLm6DSe1WNI5s5/sSk1k+yNPvedsf989RO3egMdPfW1vDqPtmexH+LmKNLO42O1Fpc3kXd9dOGyr7hi/7ehUNGj2D5nzyFXXeqDVdf3SP1Je1yuZkErs2bFXrpcnuu3MHWrrsG1L1XJe8EyH9XA11lXyun7xnJ5Vp2vdV66oEqv5O6H5uqHiE6xUZ077lISqfFHm/1AQ4zVDu/tCiRQtq165dkSycr+Vb4LmulTQBptJBVbMWX7Ooui0TW32dU4/M6tFmVfzZMMmrXl+lc1Xu27BV2WgrWzJfMiaqp+vPdRWD8L5qXZUc1d8bXQ4qvVTrqfQ1ue9bHmJiQ15zvEqA77zzTrrhhhto8eLFgb2bb745XXrppXThhRfmZX+ucn0LPOm35ei3YtM6trR5w1+ZF/T/TLskXR2ic9Ns2nmrtrRH542o88atlV0ecg0CD4Xr7E6Z+t83s7N2LDdr26pB5wGp7nmxyUtu3C6dOJDO5V9wmjZtQgNHvdEIY96/MBTFLSs+JLW1J/bZOlGEVH8bv/HCkvmSMZLP4XCMC98n8UnbqZY+wyq9sjhk1Ujr8pPqazLOtzzExIa85niTAN9999107rnnBv/57ne/G9T8vvjiizRs2DDixPgnP/lJXgxyk+tb4OnWAJvWsaXN47qyM+6bRJPmLs1krttz8pS7J2TK1JGXWzB4JFjqV+k4j0xLVUU39lU25cUmL7lJ9tisxXOTnjuu9bz39N3pzPsn04SEWv54vbWKs/S+jS3SNVTjVHWwWZ9DOvrrjDX1u+4aWba7+PzN+psSj7Wkev/e22xI//7giwa9p9PqzqPMdNa17Rmsii/T+77lIaZ25DHPmwS4W7dudMEFF9A555zTwE5OfjkJfuONxjsKeQBxKdPHwFPVSkZ74pp+i83abRs1bZEIseqbeVTIiAnzEmsOo2N05IkUrPggSQ2aqf99RRO12aR/bdSuvNjkJTfNJ5I4SJub9tzxs8afIyOnLjTqI2sSP0VzS9JRUlub9gVAV38bv7HukvmSMVEO4XiXfahD+Vl8kmKN50Xjz7SWN42VpEZal59J3Evm+JiHSPQuYow3CfC6665LM2bMoC5dujSwe/bs2dSjRw9avnx5ETycrlH1wHNd/8R9PqfNb3zkaRJ0nVopSR2YjjynQVBhYab+r4LJtrbZzk9jlJfcPHzik64+6GKjg83cPHzrm0xbPrbz4zxcy8uTd9XzkDzZeJMA8w7w4MGD6YQTTmhg70MPPUQDBw6kuXPn5skhF9lVDjz+6Setx6dqNzXt2zqf9DRtgSwBVq0h2Y2LjtGRl0sw5ChUWjuoq4LurpSu/DLH29pmOz/N9rzkRteTxotqXBG6xjkl6cTdM00/q1zGoA0Pm7m6Nqj8qrqftZ7N3Cy5tnxs58efH2m85cVDx+dVzkN07DQZ600C/Otf/zp4Ae66664Ljj7mnn3jx4+nq666is477zy6+uqrTewrdU5VA8+2nktSbxk6Rqc/a5oz0+oRw/EuatBKDaSMxXVr9XTsyFO2jh55jLW1zXZ+Viyfds/E3OpnpXpLxknGuPRd0npJnx9lPfc2PGzm6jBWraO6r0p+84pdG71YZ9v5od06fxtdranj36SxVc1DbO2WzPcmAV69ejVddNFFNHToUFq1alWgO58GxyfB/eY3vyE+NKNqV1UDL6uGd8ixPUnyIouk3jLs+MB+ta0X5PUenbyA/jZzcXCgSvvW69BmG7Sq+S4QLnc2kp4vX+rY8nj2bW2znZ9mU15yeT1pvEjH5alrnE+aTkkcdbvJuIovGx42c6X6q/yqup+1js1cif62fGznZz0/SfGWNw8JMx5T1TxEap/NOG8SYD4UY++996YVK1bQW2+9FSQxXBaxwQYb2NhX6tyqBp7r+ibX8kp1qmeLg61nDvFcHWm8SMcVaa6k1j/UBzX/yZ5R+VV1P8vfNnOLjCObtXRs1Blro5NqblXzEJVdLu57kwBvsskm9Oyzz9Kuu+7qwi4vZPgaeKq6pLRvrtzjM3qaj0pO6ATTXoo2TkzTTaqzam1XclTrqO77ssug0tP0fhGcddfQHS+13VSuzjxpvEjHSW1zMU5nB7iWa/5tWKr8qrqftXba3Ov7dqdmTZvS/CXLMvtt68SxDQObuTp8bHnY6Bmd62se4so+GzneJMDdu3enO+64gw488EAbe7ya62PgSeqS0mpqo+e5S+SEzkgbq3vevNS5ea+nY7tUZ9NxPuliakPavCJs011Dd7yUialc3XnS8dJxUvtcjEvSycU7BC50q4oMlV9V97PstPGPzbpFstfR04aHS5t8zENc2mcjy5sEmOt/ud/vEUccQdtvvz1xW7ToxS/DVe3yMfCk32CzenzySUZSOaHPkuqvJL0UTXyeVcOc1IdYd7dI13YTG3TmuKht01mvqLFFcNZdQ3e8lJWpXJN50niRjpPa6GJckk4s1/YdAhe6VUWGyq+q+1l2xudK+++axHFZvHX4mPJwaZuPeYhL+2xkeZMAd+7cOdUO7gjx7rvv2thZylwfA09al6Qap7ovAe5CRtI6aXLT+hDr1gvmpbeEWT2NKYKz7hq646X+MpVrOk+qF8aBgA0BaXxKx9no4sPcMuz0MQ/xwResgzcJsC9AXOrhY+CpvmmHdVjPzPiAxs36qBGOcLdUJUfC0YWMpHXqbQdYwjo6xlWtnY0cydy84iPKQneNvOrZdfUIbTCdJ+FvE1dbtmvFf15o0afLteo+j+zVgca89l5ivWgeOuvaWGvjJUyzxqjmS+NTOs6Uv0pPU7m68/K2M0kfH/MQXW55jfcqAeZWaGPHjg2OPeYWaLvtthvtu+++edmeu1wfAy+rhokbysf7OEYhRfvpfrliFe1141jts9XjiVgefSPrqQZYN4h1atiyZNvIkc6VjtNlYBODecWWqa0m80zmqBgnyUz77Aj/PWlOm1bNG3ymhJ85SZ9NtdzfW8XbxX1JHOj+vYj7RLIG2yIdZ2J3nrJ19SlDFx/zEF1ueY33JgF+//336dBDDw2OQ27Xrl3QBu2zzz4LDsV46qmnqG3btnkxoFmzZgXdJ/glvNNPPz1YZ/r06XTBBRfQlClTaKONNqLzzz+fLrnkEi0dfA28tBqmtG+nB3TdhA7rsTnxmethD2BX32R16ql04KfJdbWeKzk6NrkY68pvNnJ05hbBWXeNvOrZdfUI40F3ng5/acylyYzOj9faS+bwfJ63hogGjnqjkTq69ftSe+phnCQOssZIfSKNT+k4Xd9I7NSVaTM+LzvTdPI1D7Fh6GquNwnwMcccQwsWLKARI0bQd77zncC+N998k0477TTq2bMn3Xfffa5sbiDnm2++CfoPc6LLa3AC/MknnwQ9iPv27RsczjFhwgTq378/3XXXXXTGGWeI9aha4OnUJ+mMFQPDwNwJuPKbjRybubkDMlygSjbloaukR2+81l4yh93B8/ga+sLsRt7Rrd83dG9NTpPEQdaYqvhEYmdNOvh/japaHlKkL7xJgPnAi+effz4oe4heEydOpMMPP5yWLFmSC5fLL788SHB57TAB/uUvf0l33nknzZ07l5o3bx6sy+NGjRoVHNIhvYoIPC5FuPKJGTT3k6+o80at6fqje9B66/xHZ91L55ty2th4r2BdHXwe70sdmQ0jHR9nrWMjx2auje15zs3bpqTY47KAka8uVPZXjdudh66S3VzsAOcZgTLZ0Tha/NkKUnXFcbEDLNNMNsrkMziPeDfRgy00nSejkzyqiDzERr8y53qTAG+99db0yCOP0F577dWAx9SpU4PWaIsXL3bO6cUXX6Sjjz46KHfg9cMEmNfjMowHH3xw7ZrPPfccHXLIIYEem266qUiXvAPPRR1u1BCd+iQee8rdE2jS3KUNWER7BYsgVWSQDhufTXJlh40cm7m+ss3TpiTZNv1v89AVNcC+Rub/6aVTcx2WutnWALukYhq3pvPSdDeVZzrPlmHeeYitfmXO9yYBvv/+++mWW24JSiB69eoVMJkzZw6deuqpdNJJJwUlCC6vTz/9NFiHd3tZPrdaCxNgLrngXechQ4asXXLmzJnEh3VMnjyZevfunajK559/Tvyf8OK65j59+gSlHR07dnSpfiDr549Mp9HTFjWSy+eS33rCzkbr6dQnqXoFGyng6aQ8dhHKMlXHx1k62sixmVsWN9W6edkk2V0NdZPWxOaha1Rmx3b/6eO+UNEFIq7HUTt3oDHT36N5CaeG5aGzyqe1dD+rO85mbVulduvI4l6kT2w+g13qaaqH6TzbGEQCnE7QmwR4q622og8//JBWrVpFbdq0oRYtWgRlD/wyHCen0evbb7+1jQn68Y9/HMh++OGHA1nRBHi77bYLkuJrr7127Trch7hLly40fvz41M4U11xzDQ0ePLiRbnklwEff9U+aNv/TRutxv9vR/fexZqQSUE+1VfVkq8rvuF8sAWmtLGuFmthifVOl1ar+GeaL/qZ6mM6zjTEkwBVIgB944AGxn/v16ycemzRw+PDhNHDgwKDd2oYbbtgoAd5pp52CsoukHeBXX32Vdtlll8T1a2EHWAdsWd9odXR0NVZlaxm1Xa5sq3U5Ud+k9af12X957AC78HkRzIpYwwWLvGW44KD6DEuyIb5uVp/m6HwX+sblDRj5euIvntJfPVz5yHQn3YS/C52RAFcgAXbhaKmMAw88kF566SVq2bLl2ilfffVV8P95l7dTp05B6zNOlMOrHmqApfzCcWXVNOnq6WK8T7VwLuypFxmS2tR7+vWmM++fTBPm/N+Ltj71mHVdA+zC90U8+0Ws4YJF3jJccdCVY1IzzCx011Hxy3qGy3hOfeGi4hbeRwKMBLgBgUWLFtHy5csb/Nv2228f7PiecMIJ9NBDD9GwYcNo9uzZwYEcfPGO8ejRo73rArHs61U0aPQMmuOgC4T0gYqOc1lbZbJ+kXN0+ycXvTNRJIuqrCXZPeWaedXb8GXbmxR7rNPIqQsT62Xz1reI3awi1sibkwv5LjnofF5Lnh22T9rdw/TzMGvHdcixPdf2pnfBWiojylHSTSOUq8NfqotqHBJgJMCqGGlQA8y1yNwH+KijjqLLLruMJk2aROecc06QFOuUXyDwlNhrYkBZtV01AS9nIyT1s1wzn1RLj3radOcUEfNFrJFz+DkRXxYHybPDBkr7O5s+T2XZL3We7/ohD0ECrIzl6EtwPJi7PfDpb9OmTaMtttiCLr74Yjr33HOVcqIDEHhauLQHu64zS1JAsobLHRptCBWbIOHp0iTJLpYvO8BFs7HhnGfMhxyemfEBjZv1USM1TXcSbewtc26erLPskjw7PD9+UmjavOv7dqdmTZt60bfaxJ9pz2dZ/pHagDwECbA0VpyOQ+A5xdlAmOs6s7Tk97R7JiprQ4vQJT+SxUkug1NVaoDLYGPj+bz0lfhr+Jl9SvnZ24aXzdy8WKt0ktS6RmWE9bh8QEv8c9O3vtUq2+P3q/wOCPKQiiTA3Gt3xowZtGLFikYa85HIVbsQePl5rIhv3TprlFHblR/dfCTr8HSpgaQ/bdn+K4uNDec8mKVxiO8y2uhdxbl5sJZwSOvT/LRidz4+b9XqNTTozzMaLSndzS/L/lBh1fNZtn5ZvkQeUoEE+MYbbwyOG066uDzBRe9fyQPvcgwCzyXNhrKKqLsqYo38CPknGTzTfQI2/2EDDv49t0ka6fpJd7xvFKqsP/KQCiTAm2++OZ199tlBt4VoezLfHgQdfRB4OrSSx+rWXZnWmUVXRw2ivd+SJKh2UXRXrVLNrMo2m51PHzi40sF1jKi4474ZAV0/6Y5P08pVnOlaLdW/LP2wA6zr0f+M9+YkuPbt2wfdFvgUtlq5kADbeVK37sqmzizUFDWIdj7Lmu2yltGlrPwslks2jTsfOLjUwaUsOX2M1CWg6yfd8Un6uJCha2fW34V4D+Iy9UMCbOZZbxLgCy+8MNj5jZ6+ZmaSP7OQANv5QvWt23WdGWtrsxNnZ219zHZVK6eKjSrSDNmo6iujtvnAwbUOrmKkijFQJZ11/aQ7Ps7CdZzpslbpX7Z+afYgD0n3tDcJ8HvvvUc9e/ak1q1bU+fOnalp06YNtP7HP/6hG6+lj0fg2blAt+5Kd3ySdi5k2FmN2RICtewnHdt0xkq4mozxQQcTvTGnWgR8jzNf9UMeUoEE+IgjjqCJEyfSwQcfTOutt14jje+7775qPa1EhMCzc5nuN2rd8UnauZBhZzVmSwjUsp90bNMZK+FqMsYHHUz0xpxqEfA9znzVD3lIBRJgTnrHjh1Le+21V7Weygxt8w483YJ73fFFOyKu35G9OtCZ909W9uEN9XRRg5UkY4u2LWmPzhvRZhu0os4bt6Yf7daxrvqQFh0HkvVMfK2Kf9V9iV4uxiTZ1qFdK/rZAdvR8btv1SD2TDi40DEqwwcdXNsEef4R8D3OkvTj91KO6tWBFn26nDq1X6+Uvx155yH+RYpcI29KILbddlsaM2YM9ejRQ6695yPzDDzdDwPd8UWjTdPv3tN705jp79G8JctEHyCqOi2JXSzj/7d3JsBWFFcfPyyyqUhEMIRFQSAmAkqiYHAXNbiGgEH0ixKCUTEGIygRjIq4AZZbIggKblBqNIAoWVQsjSvigjFqqVEQWfxAZVGDoOj76ozfvbnvvbvMzO2Z7pn5TZWV8G5Pn9O/c3ru//ac6fnTC+/LtMfflTWb6u9JXffhBz990sY8gSCxrpT/lT437335HkvlYLHcC8IhqnG44ENUY6Nfdwi4nmeF/rVv1Uwe/OcHsmT5+jxAG98dUeoQdzIjnCfOCOD77rtPpk+fLlOnTpXu3btLo0aNwo3IobOiTLygt1uCto8bo2v+lfInx8XvBu5xc8RecQKV8qvS5za4uuiTDQ7YhEASCbgyf6PUIUmMS6HPzgjgbt26yfvvvy/btm0rypQXYdTGErTgPmj7uBPbNf9K+ZPjMvLQPeR3A/aMGxP2QhKolF+VPg9ptqrTXPSpqgFxMgQyRMCV+YsALp10zgjgO++8s+zUGDZsWOKmTpSJF/TXZdD2ccN2zb8oVoBdqTENE1ubvpuwXSm/Kn1eilkY3yqdk/v87xVeNxsmjpwDAVMEKuWxKTum+4nL77DXFNPjjVKHmPY17v6cEcBxDzwOe1EmXtCaxaDt4+BTaMM1/4r5k/M3TB2Xa+MLEl+bvpuyXamfSp8X4xXFOeXyTn0Ik3tBYk1bCPghECb3/fQbdZs4/Y7TVjluUeqQqOMVdf/OCOCJEyeWHesll1wSNQvj/UedeEEfCAja3jiQCh265l/On2Uf/Uc++nSrtNmxaehdIFxZDQgTU5u+m7RdKb8qfV6XXRjfKp1T6vNDu7eRAT2+beUp8jA5wznpJlApj10dfdx+B72mRMEtah0Shc9x9emMANaXXxQeX375paxbt06aNGki/fr1k0ceeSQuJsbskHjGUCa+I1fqwcKAtOm7TduVWIXxrdI5lT6v5BOfQyAOAknN06T6XU1M0SGl6TkjgIu5+Mknn4jW/h522GEyatSoanLAyrlxJ57J2iaTfQWFb9N2UF/9ti+38jDoBx1k7sur5H2fW735tWmqXdyrJoV+27Rdid/s51bIxQteq9dMdwgpFdNK46n0eSWf0vC56flvur80MK52DDbztJp42vS7WuZhz49bh4T108Z5TgtgBfLaa6/JscceKytWrLDBpyqbcSaeyXojk30FBWjTdlBfg7QvNa5Zw/YN9LKPIDZNtbUZE5u2y/FTv34+c7EseW9DrWZ9O+8st/1iv5IxrakROW3W8yVf7uLqeE3lUqV+TI/fdH+V/M/K57a4Vmu32vOTGN84dUjS+DgvgJcuXSoHHHCAbN68OWlsY30Vsslftib7Cho0m7aD+hq0fbF6sD+/tErGzftX0VXEoX06BTURWXubtWw2bZcCWipPrxzYQxo2bFA2ppXGU+nzyILsQMem57/p/hxA5IwLNvLURDxt+G0zaAjg0vSdEcB33XVXLS9rampk06ZNMmPGDGnXrp0sWrTIZg6Fsh1n4pmsbTLZV1BwNm0H9dVE+6yN1wQzF/ooFzf17+Yn3q3nJntHV46c6flgur/KI6BFlASIZ3C6ceqQ4N7ZPcMZAdywYcN6JLbbbjtv9VffEKdvh0vaEWfimfhlnONrsq+gMbNpO6ivJtpnbbxhmVVT9xfWZrnzysWtRiQRq/pRcKm2T9PzwXR/1Y6P84sT8Du/w8TTb9+lYlPt+bZjHqcOsT3WoPadEcBBHU9C+zgTz2Rtk8m+gsbJpu2gvppon7XxhmHmIqNyPlWq8w3DICvnmI616f6yEoc4xxkkRkHa6hiCtq877mrPj5NjKVtx6hAXxhvEBwRwEFoB28adeCZrm0z2FRCb2LQd1FcT7bM23qDMwqz6BLURpn25uBHTMES/Occ0O9P9hR8ZZxYjEHR+B4ln0L7r+lft+S5EPG4d4sKY/fpgVQAffvjhvvxs0KCBPPbYY77autSIxHMpGviSVALU/SU1cvgNgcoEopzf1fZd7fmVRx99C3RIacZWBfDw4cPLRv+pp56SZcuWScuWLWXjxo3RZ4phCyReeKBh667KnRe2z/Cj4EwTBNKwCmOCQ9R9MD+iJkz/OQKFubZ20xaZt3R1PTi6n7afnXByfS0v8sbOanfZScO1Bx3iqAAu5dann34qo0ePllmzZskRRxwhM2fOlE6d3NkSyu9ljMTzS6p2u7B1V9RlhuPt+lmfbdkmP5r0mHy6ZVve1ZbNGsvi8f2lRZPGrrufCP/CzrlEDA4nnSJQLNd2bNa41vzev0trmT2ij2zXqP7D8YWDKdZX7nPto9p91tMwL9AhCRLAjz76qPzqV7/yVnynTJkiZ5xxhlOTN4gzJF4QWv9tG/ZXN0/mh+Pt+llh88H1cbnkH4xdika6fSmVa4N6t5ddd2omu+3cQgb/sENF8auUSvWVI6iryNrX3JdWyYqQb9oMUnPsYuTQIQkQwJ999pm36qurvf379/dWf5O46luImsQLdzkIW3fF3qzheLt+Vth8cH1cLvkHY5eikW5fTOZaqb5yBNl7W2J9IVfSMtdqDXAOVm7Vd8OGDd6q75lnnpk0jkX9RQCHC2PY1SjbK8DUUIaLd6WzwuZDuX5Lxcp0DE33V4lV4ec5o5i5AAAgAElEQVRBbEfBOIivtM0OAZO55mcF2E8dcTX0g8yzauyEPRcd4ugKsK76jhkzptaqb8eOHcPG2bnzSLxwIQlbd2WzBjisz+EIZess02xL9VdtvWDdqJj2O0jUg9oO2j6IL7SFQN0fZqfNel4WL1+f/7Pfml8/cyzXJmyfQaKVhHmDDnFUAO++++6ycuVK6dKli/z85z8X3e6s1HHJJZcEyUsn2pJ44cMQtu7K1t6sJlc1wlNL75lh86EYkXI1iNU8jV7Xls2cCGPbJOP0ZiIjM0HAZK7l+lpWZBeISg/RVTuWMPOsWptBz0eHOCyAy4nenNvaRrdDS9pB4iUtYuH9NVnXFt4LzvRDoFSsendqJUvfr7/dYtg6Qps5YdO2nxjQBgJpIJCEeYYOcVQAp2EClBsDiZeeCFeq80rCSkCQaFQab5C+XGub5RXgME/auxY//EkPgaRfZ5Jw3UeHIICtXDFIPCvYjRv1U+flp41xxyLqME1jKYYoqzXAYfdajSjN6DbjBNJwnUnCGNAhCGArlxoSzwp240b9/so3WddmfBABOvQ73gBdOte0VKxMx9B0f0FAFtqu9m1bQezSFgJ+CKTlOmNzjvvhjA5BAPvJE+NtSDzjSK10mIQ6L5NgsjZek+xc7YuYuhqZ7PpFTsYTe3QIAjieTKtjhcSLD3uUtWRpWanwGw2/4y1k3r5VMxFpIKs3fh7oTU5+faq2XZT5Ua1vcZzvN6Zx+IINCCgBcjKePECHIIDjyTQEsBXOUddhRd2/FWhljPoZb7E2hV3GsQenX25+xuO3r6S2g0FSI5dev8nJeGKLAEYAx5NpCGArnONYSXC9zss0+ErjrfQGJvVn0qCeEvVbmPyMO4788OOH7TaVYmrbP+xnjwA5GX3MEcAI4OizrIgFEi8e7NSSxcO50Eop5oVtwu6fa3o05IdpovQHAQgkhQA6BAFsJVdJvGDYw9Zpxr3CF9bPYDTcbu3qCnCx2Pz5pVUybt6/6gF1ZYXa7UjjHQSiJZC266mO508vvi+L3lgn+m7bI7+/qwzZr6NE/Va6UlFChyCAo53BJXon8fxjr6YerJpz/Xv4Tcs4bQX1Lc72LtYAx7W/b5ycsQWBNBNI2/VUx/PzmYtlyXsbaoWtb+edZc7pfa2IYHQIAtjKNYTE84+92lXcuGrJqvXTPxH3WxYy79CquefwKou7QJSLzeAfdpC5L62SFes3O7lLhfvRxkMImCeQtutpuTtjtu44oUMQwOZnro8eSTwfkP6/SVLqNJPip3/y6WlJbNITS0aSDQJpm7Plno2w9UwEOgQBbOVqQuL5x+7qSkDd+rRtX9fI7x94rd7AbP269084eMuk1eYFyaGkjS149DgDAm4T0Dn4u7mvyvylq1NzPWUF2O2cq+tdg5qamppkuZwcbxHA/mPlYi1YMZ/6dN7ZG9SS5evzg3Npz1v/xMu3dDEelcbm12e/7SrZ43MIQCAcgXLPECT5eqrjOuXW5+TFFRtrgdmxWWN5fnx/adGkcThgVZyFDmEFuIr0CX8qiReMXVx1vH69KvVr/sqBPaRRwwapricNsprql2cc7fzkUFLHFgc/bEAgDgKl5uCg3u1l8om9rDwsZmrccxavcOouIToEAWwqtwP1Q+IFwuVc47TVpwUBnOaxp3lsQWJMWwjYIpDmOeja2NAhCGAr85zEs4LdmNEsrxSmaexZruM2NhlS1BH13/aDmabrS12aro0NHYIAtjLjSTwr2I0ZzXKtaFrGnuU6bmMTIUUdpSWvkx6SNMfBtbGhQxDAVq4XJJ4V7EaN+qkpNWrQoc7SMPYs13E7lErOuOLa6pwzYCw4kobrSylsLo0NHYIAtjC9RUg8K9gxCoE8Adfq8QiNXQLkg13+WI+fADoEARx/1ulbsVatko4dO8rKlSulQ4cOVnzAaHEC1AEmIzOqjVPSVvyqHW+xqEbRZzKyp76XfvIBXkmNLn4XI4AOQQBbmRkknhXsFY26VqNV0eGMNjARJxN9xIU/Cl+j6DMuHlHYqcSj0udR+ESfEIiSADoEARxlfpXsm8Szgr2iUT+rQBU7oUHkBEzFyaV6vHLQTI230EYUfUYe+IgNlMsHeEUMn+5jJ4AOQQDHnnRqkMSzgr2iUeoAKyJyokHW4hTFeKPo04nkiMgJeEUElm6tEUCHIICtJB+JVx97rr5u+Uf/kY8+3SptdmwqnXfZXo7f+zvy4D/XyPvrN8tuO7eQwT/sENnbgKpd5clSjaDNsQaNUxS+RtFnqYtR0PH6uahF0acfu0ltA6+kRg6/SxFAhyCArcwOEq829nLvf9d3pX+6ZVv+hCjfB19NnV8151pJwiqM2h5rEPtB2vpFEkWf5WxHYS+KPv3yS2I7eCUxavhcjgA6BAFsZYaQeLWxl1pdKRWcSYN6ytA+nSKJXdi60CytELkwVr9xisLXKPqslMx+x1upn8LPo+gziP2ktYVX0iKGvwjgcDnQoKampibcqZxViQACuDahUvV1pTiOPHQP+d2APSthjvXzLNUIJmmsUfgaRZ+xJivGIACBzBNAh7ACbGUSkHjfYM/VUf79tf+Vf7z9oe9YmFoBNlnHaWNV0DcwAw0LWa3dtEXmLV1dr9crB/aQ/9l/NwPWzHURRVyi6NPciOkJAhBwiYDJ7xmT40KHIIBN5pPvvki8b8TvabOel8XL15flFlUNsOmaPtP9+U6mGBoWG9sOTRvJZ1u/qmW9b+edZc7pfSN7SDHMUKOISxR9hhkb50AAAm4TcPlagQ5BAFuZPSSeSKlVtIO77SK77NA0vwvECft8Rx58ZY2sMLwLRBSreGmtEQxSo21qdd7kxIwiLlH0aXLM9AUBCNgnEMX3jKlRoUMQwKZyKVA/JJ6I7TpK2/YDJYzlxkFqtF2sz7aMD/MQgEBGCbj8PYMOQQBbmZYkXukV4LhWEF3+ZW4lKcsYTfoKcBQ8Xa3ri2Ks9AmBpBKwNU8rPd8S1/dcubihQxDAVuY1iVe8BjjKPX7rBtrl2iwrSVnGaDFWfTrv7J2xpKCGO8742WRE7tikj20I+CNga55Wer7FleskOgQB7G8mGW5F4n0D1HYdpW37htMq0u6KsVKDc19aZbw+O9KBGOicuwcGINIFBCImYGuelrJ7aPc2MqDHtyN9m2kQpOgQBHCQfDHWlsQzhpKOIBA7AZfr+mKHgUEIOErA1jy1ZTdoGNAhCOCgOWOkPYlnBCOdQMAKAVsrS1YGi1EIJJSArXlqy27QMKFDEMBBc8ZIexLPCEY6gYAVArZqC00O1tbDQSbHQF8QKEfA1jy1ZTdoNqBDEMD1CKxfv17Gjx8vCxculE8++UR69eolkyZNkgMPPNBr+8orr8i5554rL774orRu3VpGjRol559/fqDcI/EC4aIxBJwjkOT68aR8QTsXdBxKHAFb89SW3SABQocggOsROOqoo2TdunVy0003SZs2bWTq1Kkyc+ZMefnll71/77nnnjJw4EAZPXq0LF68WM4++2yZNm2aDB8+3HfukXi+UdEQAhAwTCApt2gND5vuIACBAgLoEARwLQLvvPOOdOvWTZ555hnp16+f91lNTY10795dTj75ZGnevLkniN977z1p3Lix97muFs+bN0/efPNN35OLxPONioYQgIBhAkl5SMfwsOkOAhBAAPvKgQY1qvwydmzcuFGeffZZ6d+/vzRt2jQ/+q5du8qAAQNk2bJl0qpVK7n77rvzny1atEiOPPJIWbt2rbRt29YXMZsCmNo/XyGiEQRSS4AV4NSGloFBwDcBmzrEt5OWGmZSABdjff/998uQIUPkwQcflIsuukiOPvpomTx5cr7pG2+8IXvttZe88MILsu+++/oKl63Eo/bPV3hoBIFUE+A6kOrwMjgI+CJgS4f4cs5yIwSwiFcKoSu/uiL8wAMPiK4En3LKKTJx4sR8eHRVeI899pCnnnoq/6Bc3djpw3T6X+744IMPpE+fPrJy5Urp0KFDbKFm5Sc21BiCgNMEkvCQjtMAcQ4CCSeAAC4dwMwL4AULFnhid//99/d2hND63549e8oxxxxTdAVYH5Lr3bt3UaITJkyQyy67rN5ncQtgav8SfsXCfQhAAAIQgIABAghgBHBRAroDhG51NmjQIJkzZ06+HljFr259Nnv27Px5fmqAXVgB1tuev5v7qsxfurremCcN6ilD+3QyMKWy2wW11dmLfdJjnnT/s5dxjBgC5ggggBHA9QjcfPPN3tZmur/v9ddfLw0bNsy30f2Ap0+fLu+++640atTI+/u4ceNk/vz5Tu8CUazmLzeo/bu0ltkj+sh2jf47TnNTLBs9UVOZjTgXjjLpMU+6/9nLOEYMAbMEEMAI4FoE3n77benRo4ccd9xx3t6+hYeWQGzdutXbB/iEE06QsWPHypIlS2TkyJGeKB42bJjv7Iw78UrV/g7q3V4mn9gL8es7csUbUltdJcAEnp70mCfd/wSmDC5DwCkCcesQpwZfwZlM1gBfddVV3k4PxQ4VuHfccYe324OuDi9dulTatWsnY8aMkXPOOSdQbONOPGp/A4UncGP4BkaW+BOSHvOk+5/4BGIAELBMIG4dYnm4gcxnUgAHIlRF47gTj9WeKoLl41T4+oCUsiZJj3nS/U9ZOjGcCAlQ614cbtw6JMIQG+8aAWwc6X87jDvxqPeLMJgiAt9o+brYe9JjnnT/XcwJfHKPAHleOiZx6xD3sqO0RwjgCKNlI/HY9zPCgIoIfKPl62LvSY950v13MSfwyS0C3OlAAIfJSARwGGo+z7EhgH26RjMIQAACEIBAKghQ644ADpPICOAw1HyegwD+BhS1WTAoNmVcyAsXfPB5OaEZBCBQggArwAjgMJMDARyGms9zEMDUzeZ+AJw263lZvHx9PnOyvi+zCzV7Lvjg81JCMwhAoAwB5jICOMwEQQCHoebzHASwCL/MYVBsuriQFy744PNSQjMIQKACAWrdiwNCh5ROHARwhJcVEk+E2iwYFJtiLuSFCz5EePmhawhAAAKCDkEAW5kGJB6rn5p4rDTWn34uMHHBBysXphQYpXY7BUF0dAhpyy10CALYylQj8agB1sSjPq3+9HOBiQs+WLkwJdwocUt4AB12P425hQ5BAFuZciTeN9ipzYJBsQnoQl644IOVi1OCjbJyn+DgOe56GnMLHYIAtjLtSDwr2DEKAQikmAC12ykOruWhpTG30CEIYCvTisSzgh2jEIBAHQJpqmtM4yodCesGgTTmFjoEAWxldpF4VrBjFAIQKCCQtrrGtI2HZHWHwGdbtsmPJj0mn27ZlneqZbPGsnh8f2nRpLE7jgbwBB2CAA6QLuaaknjmWNITBCAQjkAaV7Wo3Q6XC5xVnkAa5wo6BAFsZd6TeFawYxQCECggkMa6RgIMgSgIpHGuoEMQwFHMlYp9kngVEdEAAhCImEAaV7WqQZameuhqOHBufQJpnCvoEASwlblO4lnBjlEIQKCAADWz/4UBC6ZGOQJpzA90CALYyqwn8axgxygEIFCHADWz3wBJ4wofyW6WQNrmCjoEAWx2hvjsjcTzCYpmEIAABGIgkMYazxiwYSLBBNAhCGAr6UvihcdOnV54dpxpngD5aJ6pjR5ZAbZBHZvFCMR1TUGHIICtzEASLxz2NNZhhSPBWS4QIB9diIIZH4ilGY70Uh2BOPMQHYIAri5bQ55N4oUDxypNOG6cFQ0B8jEarrZ6TVuNpy2O2A1PIM5rCjoEARw+U6s4k8QLB486vXDcOCsaAuRjNFzpFQJZJRDnNQUdggC2Ms9IvHDY4/x1HM5DzsoSAfIxS9FmrBCInkCc1xR0CAI4+owuYoHEC4c9zvqocB5yVpYIkI9ZijZjhUD0BOK8pqBDEMDRZzQC2Chj6vSM4qSzKgmQj1UC5HQIQKAWgbiuKQhgBLCVqUfiWcGOUQhAAAIQgAAERAQdggC2MhGSlnhx7UtoJRgYhQAEIAABCBggkKTvyqTpEAPh8d1Fg5qamhrfrWkYiECSEi/OmqRAEGkMAQhAAAIQcIRA0r4rk6RD4g4xAjhC4klKvDifSo0QOV1DAAIQgAAEIiOQtO/KJOmQyIJWomMEcITEk5R4ce5LGCFyuoYABCAAAQhERiBp35VJ0iGRBQ0BHDfaZBWfJ+1XbfzRxCIEIAABCGSdQNK+KxHApTOWFeAIZ3OSEi9pdU0Rho2uIQABCEAAAkUJJO27Mkk6JO6UQwBHSDxpiRfXvoQRIqdrCEAAAhCAQKQEkvRdmTQdEmng6nSOAI6QNokXIVy6hgAEIAABCECgLAF0CCUQVqYIiWcFO0YdJ5CkPTQdR4l7EICAZQKuX8/QIQhgK1OExLOCHaMOE0ha/ZzDKHENAhCwTCAJ1zN0CALYyjQh8axgx6jDBJL2BLXDKHENAhCwTCAJ1zN0CALYyjQh8axgx6jDBJK2h6bDKHENAhCwTCAJ1zN0CALYyjQh8axgD2zUtRou1/wJDLTMCa6tmKSZtcm40RcEIFCfgGvXs2IxQocggK3MXRLPCvZARl2r4XLNn0AwfTR2aXwu+eIDHU0gAAHHCCThGoIOQQBbmTYknhXsgYy69gveNX8CwfTZ2JU9NLPA2mdIaAYBCIQk4Mr1rJT76BAEcMjUru40Eq86fnGc7VoNl2v+xBEDWzZgbYs8diEAgbgIoEMQwHHlWi07JJ4V7IGMurYK6Jo/gWCWaexirW1aWZuKGf1AAALJJ4AOQQBbyWISzwr2QEZdq+FyzZ9AMEs0dnVMrvplgjl9QAACEFAC6BAEsJWZQOJZwR7YqGs1XK75ExhonRNcXmlNG+tqY8X5EIBAugigQxDAVjKaxLOCHaOOEaDW1rGA4A4EIJAZAugQBLCVZCfxrGDHqGMEXF4BdgwV7kAAAhAwSgAdggA2mlB+OyPx/JKiXY6Aiw+LVRsdam2rJcj5EIAABMIRQIcggMNlTpVnkXhVAszY6WkWitTaZiyZGS4EIOAEAXQIAthKIpJ4VrAn1iilAokNHY5DAAIQcJIAOgQBbCUxSTwr2BNrlIfFEhs6HIcABCDgJAF0CALYSmKSeFawJ9ZomlaA01jLnNjEwnEIQCCzBNAhCGAryU/iWcGeWKNpqQFOyzgSm0g4DgEIQOD/CaBDEMBWJgOJZwV7oo2m4WGxNK1kJzqZcB4CEMg8AXQIAtjKJCDxrGDHqGUC1DJbDgDmIQABCLACXDEHGtTU1NRUbEWDUARsC2DqMEOFjZOqJMAKcJUAOR0CEICAIQK2dYihYUTSDQI4EqzfdGoz8ajDjDCwdF2WALlHgkAAAhBwg4BNHeIGgdJeIIAjjJDNxGMVLsLA0nVFAmmoZa44SBpAAAIQcJyATR3iOBpBAEcYIZuJRx1mhIGlawhAAAIQgEACCNjUIa7jQQBHGCGbiccKcISBpetYCFDDHgtmjEAAAikmYFOHuI4VARxhhGwmHnWYEQaWriMnQP5GjhgDEIBABgjY1CGu40UARxgh24lHHWaEwaXrSAlwByNSvHQOAQhkhIBtHeIyZgRwhNEh8SKES9epJkANe6rDy+AgAIGYCKBDSoNGAEeYhCRehHDpOtUEWAFOdXgZHAQgEBMBdAgCOKZUq22GxLOCHaMpIEANcAqCyBAgAAHrBNAhCGArSUjiWcGO0ZQQoIY9JYFkGBCAgDUC6BAEsJXkI/GsYMcoBCAAAQhAAAKW30jregCoAY4wQgjgCOHSNQQgAAEIQCBGAkncmxwdwgpw4Cny9ddfy2WXXSYzZ86UDRs2yIEHHijTpk2Trl27+u6LxPONioYQgAAEIAABZwkk9bkEdAgCOPCkUvGrgvf222+X9u3by9ixY2XZsmXy+uuvS5MmTXz1R+L5wkQjCEAAAhCAgNMEkrozDToEARxoYn3xxReyyy67yJQpU+Sss87yzt24caN85zvfkdtuu02GDh3qqz8SzxcmGkEAAhCAAAScJpDUvcnRIQjgQBNryZIl0rdvX3nrrbeke/fu+XO1DKJXr17eyrCfg8TzQ4k2EIAABCAAAbcJsALsdnzCeMdDcEWozZs3TwYPHiybN2+W5s2b51sMGTLE+9vChQt9sUYA+8JEIwhAAAIQgIDTBKgBdjo8oZxDABfBNmfOHDn11FPlq6++koYNG+ZbnHbaabJmzRpZtGhRUdiffPKJ6H+5Y+XKldKvXz/RFeV27dqFChAnQQACEIAABCBgn4DuTf7XVz+Q1Rs/l/atmssxvdrJdo3+qxHse1jfgw8++ED69Okjy5cvl913391FF635hAAugn7u3Lly4oknFl0B3rp1qyxYsKBowCZMmODtHMEBAQhAAAIQgAAEXCGgC3H77befK+444QcCuEgYcjXA77zzjuyxxx75FloDvPfee8vUqVN9rQBv2bJFdBW4c+fO0rhx40gCnvt1xypz6V++sIFNkMnHnCpNCzawCTKXcm3JG3t5s23bNvnwww+lZ8+e0qxZszDhS+05COAiodVV3rZt28p1110nI0aM8FqE2QUijqyhzrg0ZdjAJswcJG/IG/ImDAHyJgw1rjdhqJk5BwFcguNFF10kM2bM8LY907qZCy64QN577z3517/+5XsfYDMhKt8Lk4eLbpg8I2/IG/ImDAHyJgw1rjfkTZi8ifocBHAJwvoA3Pjx470XYXz++edy8MEHe6UPrhWRc2HhwhLmIkHekDfkTRgC5E0YalxvyJsweRP1OQjgqAlH3L/uOqGlGqNHj5aWLVtGbC1Z3cOmdLxgA5sws5m8IW/ImzAEyBuz1Mz0hgA2w5FeIAABCEAAAhCAAAQSQgABnJBA4SYEIAABCEAAAhCAgBkCCGAzHOkFAhCAAAQgAAEIQCAhBBDACQkUbkIAAhCAAAQgAAEImCGAADbDkV4gAAEIQAACEIAABBJCAAGckEDVdfPrr7/2Xrs8c+ZM2bBhg+hb6qZNmyZdu3ZN6IjCuX3FFVfIokWL5Iknnsh38Morr8i5554rL774orRu3VpGjRol559/fv7zNLNbv369t33fwoULRZ/Y79Wrl0yaNMnLDz2yzGbdunUyZswY+fvf/+5tbXjIIYfINddcI9///vczz6Zw9r399tvygx/8QG666Sb5xS9+kXk2K1asKLr95a233iqnn356pueUJsddd93lXWOWLVvmvTl1woQJ8rOf/SzTeaPfR4cddljRLzV9M6yyyvK1ONy3vfmzEMDmmcbSo4pfFby6T3H79u1l7Nix3qR6/fXXnXpRR5QwbrjhBm/7N92jOSeAP/74Y9lzzz1l4MCB3meLFy+Ws88+22M1fPhwz500szvqqKNEhZ6KlzZt2nh7V+uPpJdfftn7d5bZ7L///tKwYUP5wx/+INtvv71cfPHF8uyzz4q+8lwFcZbZ5Obpl19+Kf369fN+POq1RQVw1ufUQw89JEOGDPGurw0aNMhf0nbaaSfZvHlzpvNmzpw5Xo5ce+21cuyxx8o999zjCeCnn35aunfvnlk2X3zxhehiROGhL9E65phj5Oabb5af/vSnmWUTpSYI2jcCOCgxB9rr5Npll11kypQpctZZZ3keufqq5ihwrV692lt5eeqpp6Rjx46y66675gXw1Vdf7Yk+fWtf48aNPfO6Ijpv3jx58803Jc3sVMh169ZNnnnmGU/E6FFTU+N9EZ188snSvHnzzLJREXfOOefI73//e9lrr708Nv/85z9ln332kSVLlnh3EbKaN4VzVOeK/mh8/PHH8wI4y3NK2Vx11VVy3333eSt2dY8ss9FrS5cuXeTEE0/07qTkjh//+Mdy6KGHev9kTn1DRX9Y9u7dW3r06CH33nuvZDlvotAEYftEAIclZ/E8/cLu27evvPXWW564yR16m1tveetqZ5oPXZHRlQYtf5g4caIndnMrwPoLu1WrVnL33XfnEai4OfLII2Xt2rVe27Sy0x9BuqLZv39/adq0aX78WhYzYMAAbwUrq2zqzgfNBS2L0bzRH0Z6yzbrbJ588klvZUqFXqdOnfICOMtzSvNm6NCh3p2DwmtKLp+yzEbnzfe+9z3v7pKKu7pHltnUZaF3Ky+99FLvWtOuXTtvJTjr1xsXNAoC2IUoBPRBVzMHDx7s3X7TVb3cobfp9G9a/5mVQ2+/FQpg/QFw9NFHy+TJk/MI3njjDW/V74UXXpD3338/U+zuv/9+7/btgw8+KBdddBFsROSMM84Qrd/UHwnKRctGsp43+uNp77339lamTjnlFO9Wf64EIuts9NqhomXr1q2i9dF6l0XLZ3SlM8tsdO785Cc/8e7EXX755bJ06VLR+la9y3L88cdnmk3h9++WLVu8O5Vaiqfld3pkOW9c0iYIYJei4dMXrbs69dRT5auvvvJWJnLHaaedJmvWrPFu52blqCuAdbVTv8B1ZTh35B7O0Au1iuWssNNSCF351RXhBx54wHtAEjYi+oNIa361Fk9X9bReUX8kZJmNlsjoLW29PatHoQDOct5oyZTWi+sKp97m32GHHUSvvzfeeKM8+uijcuaZZ2Y2b3LfQyp6dXVTf0DNnTtXrrzyysyzKfz+1R/b5513nujDlPpQth5ZnlMuaRMEsEvR8OmLXmS07qrYCrCuUixYsMBnT8lvVlcA9+zZ07u9VGwFWG/VqRjOAjvNARV0+uCX3hHQOwWwqZ3vuhuIMtGSGL07kNW8mT17towbN070IZ1vfetb9QRw1vPmP//5j/c8QWFZkf6w1B8Jq1atymzeaF30SSed5JXcjRw5Mj+59GE4PfRuW1bnVOGV5qCDDvJqpe+88878n7M+p1xRHghgVyIRwI9cDbA+9KTbzuQOrQHWX+H64EFWjroCWC+4+itbv9RzR7Ea4DSz0x0gdBu4QYMGeatVuS/uLLPRnTEee+wxb6W3UaNG+dzQ2l/dLk7/ltW80e2adBW8UOCp6NN/6/VltxPtmnUAAAn6SURBVN12yyybUtfRCy64wNtOT29tZzVv9HmDAw44wCuByG2zqLx0RyL90b377rtnlk0ubz788EP59re/7fHQ0rzckeVrsUvaBAHsUjR8+qKrvG3btpXrrrtORowY4Z2VpV0gCjHVFcC6H+X06dPl3XffzQsdXd2aP3++9wBC2tnpbX2tNdO9j6+//vpaJTJZZqMPdultbBXBhx9+uJdC+mS2bn12wgkneDuJZDVvdFcVLQkpPLTOVe+i6AqfPnCaVTavvvqqt6OKit1Ckac/GjRndBeRrLLRnFEGl1xySa191lXobbfddh63rLLJzSW9E6fP6+iP7BYtWuSnWJavxT5lTizNEMCxYDZvRB9omjFjhtx2223eL21dkdD6Vr2N2aRJE/MGHe2xrgDWlb6cqNGVCF0t19tzeiEeNmyYN4q0stMHdHSbneOOO67eTiBaAqHiP6tstL5Vv5h1jmhNnt7q11rFhx9+2Nv1oFmzZpllU2xqF9YAZ3lOaZmMCjldEdcfl7r95C233OLNLy2bUQGY1TmleaM78eh2nPpd1KdPH6+GXAWx/tDUF8xkmY3y0WdR9C6cXpsLjyzPKZekBALYpWgE8EUfgNM9O/VJbf0lri+D0NIHFcNZOuoKYB27fjHpCqg+laxPb+vbv3QP2NyRVna6X6mK+2KHiv877rgjs2yUyaZNm7xaV30gUO+YaG2e3kXJ7Quc1bypJICzPKd07Hob+8ILL5S//vWvXt7oW/J0BU/zJ+tsdPw6h7TsSu8k6LZoutOB7g4BG/Huxun30HPPPVdvmnG9sa9UEMD2Y4AHEIAABCAAAQhAAAIxEkAAxwgbUxCAAAQgAAEIQAAC9gkggO3HAA8gAAEIQAACEIAABGIkgACOETamIAABCEAAAhCAAATsE0AA248BHkAAAhCAAAQgAAEIxEgAARwjbExBAAIQgAAEIAABCNgngAC2HwM8gAAEIAABCEAAAhCIkQACOEbYmIIABCAAAQhAAAIQsE8AAWw/BngAAQhAAAIQgAAEIBAjAQRwjLAxBQEIJIvAoYceKv/4xz9qOa2vGtc3DA4cOFCuvvpq0ddMmzr0TY76dsMJEyaY6rJeP3Vt6GuPC4+mTZtKhw4d5KSTTvLeLNiiRYvIfKFjCEAAArYIIIBtkccuBCDgPAEVwLvuuqvceOONeV8/++wzeeSRR+S3v/2tnHHGGd5rYE0d+tpdFdQ77LCDqS59CeAbbrjBE7x66PiWLFki559/vvdq24cfflgaN24cmT90DAEIQMAGAQSwDerYhAAEEkFABbCumN5xxx31/P3lL38pCxYskI8//jgRY8k5WWwF+Pbbb/dWnguPl156Sfbbbz+ZOXOm6Fg5IAABCKSJAAI4TdFkLBCAgFEC5QTw2WefLX/+859l3bp1ns0vvvhCLr74YpkzZ45s2rRJevToIRMnTpSjjjoq75OuHI8bN07eeOMN6dKli4wZM0ZGjBghy5cv94R2XXH6l7/8RS6//HJ57bXXpGXLlnLyySfLlVdeKc2aNfP61PKFW265Re677z55+umnZeedd5ZzzjnHs1Hq8CuA9fyDDz5YtORj0aJFRrnSGQQgAAHbBBDAtiOAfQhAwFkCxQTwtm3bvLKAoUOHeiUQ1157ref/KaecIq+//rpoOYHW0D700ENy4YUXyvz58+XYY4+VV155xVtRPe+88zzRq/8eOXKkbNiwoagAfuCBB2Tw4MFePfCQIUPk3//+t9e+T58+Mnfu3LwA3mmnnbwyDP377Nmz5YorrpAnn3xSDjrooKJcgwjgX//61/KnP/1JPvroI2djhGMQgAAEwhBAAIehxjkQgEAmCKgAfuaZZ0QfDMsdn3/+uey2225eycD48eO9+th33nlHunXrJi+88ILsu++++bbDhg2TFStWyBNPPCH6/99++2157rnn8p//8Y9/lFGjRhUVwH379pWOHTt6q8y5Q0X1CSec4K0ga32urgCfe+65nujOHboKfMEFF5RcBQ4igPUhuClTpsiXX36ZiXgzSAhAIDsEEMDZiTUjhQAEAhJQAdy6dWuZPHmyfP311/L88897K7iDBg2SadOm5R8Ou//++71V2u23376WBRWOrVq1krVr10rPnj3lyCOPlOuuuy7fRleBe/fuXVQA6+4LV111lfewXe7Q0grt79577/UeWlMBPH36dDnzzDPzbVTgDh8+XC699NKiow0igLWcQm2xAhwwcWgOAQg4TwAB7HyIcBACELBFoFgJxOOPPy5HHHGEJzpVBOuhZQJaEvHyyy/LjjvuWMvdRo0aSefOnT2he8ghh9RarS0ngHU3iEmTJnkrvLlj/fr1niDXEggV4SqA6z7AVmkrtSAC+Ec/+pE3Hq1d5oAABCCQJgII4DRFk7FAAAJGCZR6CG7s2LFyzTXXyN/+9jcZMGCAV/urD70tXLjQq/fNHVpC0LBhQ+9BNi2ZePPNN2Xx4sX5z2fMmCFnnXVW0RVgrent1KlTrRII3XVC9x8uLIGISgDrLhBazqE7YGj5BgcEIACBNBFAAKcpmowFAhAwSqCUAN6yZYv06tVLtm7d6olf3bf3+OOP9x5smzp1qieG582bJyqUZ82a5ZUk6E4O++yzj7e/rm4rpiJWH6LTvX/fe+89r664cHVWyyq0zOGyyy7z/lfrh/UhOO1Da4H1MLUCXGwfYH2Ar2vXrvLoo4+KrmJzQAACEEgTAQRwmqLJWCAAAaMEym2Dpm+IO+yww+Q3v/mN96KMzZs3e29O03IILVXQbc5Gjx4tp59+et4nFa66RZnu6PDd7343v8vDmjVrvLfL1S1PuOeee7y3zb311lvSpk0bb6cJFcS5t8+ZEsCF0HS7NS3ZUFv6gF5uyzWjYOkMAhCAgGUCCGDLAcA8BCCQDQK6Q4TuGKG1wLnj7rvv9laD9e1rvG0tG3nAKCEAATcIIIDdiANeQAACKSdw6623etuT3XXXXV4Zg26dpg/SHXDAAUXfNJdyHAwPAhCAgFUCCGCr+DEOAQhkhUBNTY33MJw+VLZ69Wpp27at92a3wpKGrLBgnBCAAARsE0AA244A9iEAAQhAAAIQgAAEYiWAAI4VN8YgAAEIQAACEIAABGwTQADbjgD2IQABCEAAAhCAAARiJYAAjhU3xiAAAQhAAAIQgAAEbBNAANuOAPYhAAEIQAACEIAABGIlgACOFTfGIAABCEAAAhCAAARsE0AA244A9iEAAQhAAAIQgAAEYiWAAI4VN8YgAAEIQAACEIAABGwTQADbjgD2IQABCEAAAhCAAARiJYAAjhU3xiAAAQhAAAIQgAAEbBNAANuOAPYhAAEIQAACEIAABGIlgACOFTfGIAABCEAAAhCAAARsE0AA244A9iEAAQhAAAIQgAAEYiXwf8TOeOW+8SRDAAAAAElFTkSuQmCC\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# number of probes histogram\n",
    "fig, ax = plt.subplots()\n",
    "ax.plot(cand_prob_nums, '.')\n",
    "ax.set_ylabel(\"Num probes per region\")\n",
    "ax.set_xlabel(\"Region ID\")\n",
    "ax.set_title(\"Probe number before filtering\")\n",
    "ax.set_ylim([0,100])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "7a4d0b43",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Screen probes by genome counts.\n",
      "-- filtering region:NRNX1-seg-1, directly append 36 probes\n",
      "-- filtering region:NRNX1-seg-2, directly append 40 probes\n",
      "-- filtering region:NRNX1-seg-3, directly append 40 probes\n",
      "-- filtering region:NRNX1-seg-4, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-5, directly append 64 probes\n",
      "-- filtering region:NRNX1-seg-6, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-7, directly append 34 probes\n",
      "-- filtering region:NRNX1-seg-8, directly append 19 probes\n",
      "-- filtering region:NRNX1-seg-9, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-10, directly append 66 probes\n",
      "-- filtering region:NRNX1-seg-11, directly append 20 probes\n",
      "-- filtering region:NRNX1-seg-12, directly append 4 probes\n",
      "-- filtering region:NRNX1-seg-13, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-14, directly append 67 probes\n",
      "-- filtering region:NRNX1-seg-15, directly append 78 probes\n",
      "-- filtering region:NRNX1-seg-16, directly append 70 probes\n",
      "-- filtering region:NRNX1-seg-17, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-18, directly append 36 probes\n",
      "-- filtering region:NRNX1-seg-19, directly append 72 probes\n",
      "-- filtering region:NRNX1-seg-20, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-21, directly append 42 probes\n",
      "-- filtering region:NRNX1-seg-22, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-23, directly append 70 probes\n",
      "-- filtering region:NRNX1-seg-24, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-25, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-26, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-27, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-28, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-29, directly append 11 probes\n",
      "-- filtering region:NRNX1-seg-30, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-31, directly append 41 probes\n",
      "-- filtering region:NRNX1-seg-32, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-33, directly append 34 probes\n",
      "-- filtering region:NRNX1-seg-34, directly append 43 probes\n",
      "-- filtering region:NRNX1-seg-35, directly append 42 probes\n",
      "-- filtering region:NRNX1-seg-36, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-37, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-38, directly append 48 probes\n",
      "-- filtering region:NRNX1-seg-39, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-40, directly append 70 probes\n",
      "-- filtering region:NRNX1-seg-41, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-42, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-43, directly append 69 probes\n",
      "-- filtering region:NRNX1-seg-44, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-45, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-46, directly append 42 probes\n",
      "-- filtering region:NRNX1-seg-47, directly append 41 probes\n",
      "-- filtering region:NRNX1-seg-48, directly append 40 probes\n",
      "-- filtering region:NRNX1-seg-49, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-50, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-51, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-52, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-53, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-54, directly append 66 probes\n",
      "-- filtering region:NRNX1-seg-55, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-56, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-57, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-58, directly append 48 probes\n",
      "-- filtering region:NRNX1-seg-59, directly append 48 probes\n",
      "-- filtering region:NRNX1-seg-60, directly append 23 probes\n",
      "-- filtering region:NRNX1-seg-61, directly append 17 probes\n",
      "-- filtering region:NRNX1-seg-62, directly append 43 probes\n",
      "-- filtering region:NRNX1-seg-63, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-64, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-65, directly append 24 probes\n",
      "-- filtering region:NRNX1-seg-66, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-67, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-68, directly append 24 probes\n",
      "-- filtering region:NRNX1-seg-69, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-70, directly append 42 probes\n",
      "-- filtering region:NRNX1-seg-71, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-72, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-73, directly append 24 probes\n",
      "-- filtering region:NRNX1-seg-74, directly append 36 probes\n",
      "-- filtering region:NRNX1-seg-75, directly append 28 probes\n",
      "-- filtering region:NRNX1-seg-76, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-77, directly append 34 probes\n",
      "-- filtering region:NRNX1-seg-78, directly append 13 probes\n",
      "-- filtering region:NRNX1-seg-79, directly append 32 probes\n",
      "-- filtering region:NRNX1-seg-80, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-81, directly append 37 probes\n",
      "-- filtering region:NRNX1-seg-82, directly append 48 probes\n",
      "-- filtering region:NRNX1-seg-83, directly append 63 probes\n",
      "-- filtering region:NRNX1-seg-84, directly append 69 probes\n",
      "-- filtering region:NRNX1-seg-85, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-86, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-87, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-88, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-89, directly append 77 probes\n",
      "-- filtering region:NRNX1-seg-90, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-91, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-92, directly append 38 probes\n",
      "-- filtering region:NRNX1-seg-93, directly append 73 probes\n",
      "-- filtering region:NRNX1-seg-94, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-95, directly append 29 probes\n",
      "-- filtering region:NRNX1-seg-96, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-97, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-98, directly append 37 probes\n",
      "-- filtering region:NRNX1-seg-99, directly append 32 probes\n",
      "-- filtering region:NRNX1-seg-100, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-101, directly append 42 probes\n",
      "-- filtering region:NRNX1-seg-102, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-103, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-104, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-105, directly append 23 probes\n",
      "-- filtering region:NRNX1-seg-106, directly append 69 probes\n",
      "-- filtering region:NRNX1-seg-107, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-108, directly append 32 probes\n",
      "-- filtering region:NRNX1-seg-109, directly append 42 probes\n",
      "-- filtering region:NRNX1-seg-110, directly append 66 probes\n",
      "-- filtering region:NRNX1-seg-111, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-112, directly append 25 probes\n",
      "-- filtering region:NRNX1-seg-113, directly append 62 probes\n",
      "-- filtering region:NRNX1-seg-114, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-115, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-116, directly append 8 probes\n",
      "-- filtering region:NRNX1-seg-117, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-118, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-119, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-120, directly append 56 probes\n",
      "-- filtering region:NRNX1-seg-121, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-122, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-123, directly append 37 probes\n",
      "-- filtering region:NRNX1-seg-124, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-125, directly append 36 probes\n",
      "-- filtering region:NRNX1-seg-126, directly append 27 probes\n",
      "-- filtering region:NRNX1-seg-127, directly append 38 probes\n",
      "-- filtering region:NRNX1-seg-128, directly append 36 probes\n",
      "-- filtering region:NRNX1-seg-129, directly append 30 probes\n",
      "-- filtering region:NRNX1-seg-130, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-131, directly append 40 probes\n",
      "-- filtering region:NRNX1-seg-132, directly append 19 probes\n",
      "-- filtering region:NRNX1-seg-133, directly append 24 probes\n",
      "-- filtering region:NRNX1-seg-134, directly append 28 probes\n",
      "-- filtering region:NRNX1-seg-135, directly append 42 probes\n",
      "-- filtering region:NRNX1-seg-136, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-137, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-138, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-139, directly append 67 probes\n",
      "-- filtering region:NRNX1-seg-140, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-141, directly append 39 probes\n",
      "-- filtering region:NRNX1-seg-142, directly append 31 probes\n",
      "-- filtering region:NRNX1-seg-143, directly append 71 probes\n",
      "-- filtering region:NRNX1-seg-144, directly append 69 probes\n",
      "-- filtering region:NRNX1-seg-145, directly append 69 probes\n",
      "-- filtering region:NRNX1-seg-146, directly append 69 probes\n",
      "-- filtering region:NRNX1-seg-147, directly append 68 probes\n",
      "-- filtering region:NRNX1-seg-148, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-150, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-151, directly append 69 probes\n",
      "-- filtering region:NRNX1-seg-152, directly append 66 probes\n",
      "-- filtering region:NRNX1-seg-153, directly append 17 probes\n",
      "-- filtering region:NRNX1-seg-154, directly append 35 probes\n",
      "-- filtering region:NRNX1-seg-155, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-156, directly append 66 probes\n",
      "-- filtering region:NRNX1-seg-157, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-158, directly append 67 probes\n",
      "-- filtering region:NRNX1-seg-159, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-160, directly append 64 probes\n",
      "-- filtering region:NRNX1-seg-161, directly append 75 probes\n",
      "-- filtering region:NRNX1-seg-162, directly append 66 probes\n",
      "-- filtering region:NRNX1-seg-163, directly append 34 probes\n",
      "-- filtering region:NRNX1-seg-164, directly append 56 probes\n",
      "-- filtering region:NRNX1-seg-165, screen genome to keep 80 probes\n",
      "-- filtering region:NRNX1-seg-166, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-167, directly append 6 probes\n",
      "-- filtering region:NRNX1-seg-168, directly append 8 probes\n",
      "-- filtering region:NRNX1-seg-169, directly append 62 probes\n",
      "-- filtering region:NRNX1-seg-170, directly append 71 probes\n",
      "-- filtering region:NRNX1-seg-171, directly append 63 probes\n",
      "-- filtering region:NRNX1-seg-172, directly append 76 probes\n",
      "-- filtering region:NRNX1-seg-173, directly append 68 probes\n",
      "-- filtering region:NRNX1-seg-174, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-175, directly append 71 probes\n",
      "-- filtering region:NRNX1-seg-176, directly append 70 probes\n",
      "-- filtering region:NRNX1-seg-177, directly append 64 probes\n",
      "-- filtering region:NRNX1-seg-178, directly append 66 probes\n",
      "-- filtering region:NRNX1-seg-179, directly append 71 probes\n",
      "-- filtering region:NRNX1-seg-180, directly append 69 probes\n",
      "-- filtering region:NRNX1-seg-181, directly append 71 probes\n",
      "-- filtering region:NRNX1-seg-182, directly append 71 probes\n",
      "-- filtering region:NRNX1-seg-183, directly append 56 probes\n",
      "-- filtering region:NRNX1-seg-184, directly append 48 probes\n",
      "-- filtering region:NRNX1-seg-185, directly append 43 probes\n",
      "-- filtering region:NRNX1-seg-186, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-187, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-188, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-189, directly append 63 probes\n",
      "-- filtering region:NRNX1-seg-190, directly append 64 probes\n",
      "-- filtering region:NRNX1-seg-191, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-192, directly append 80 probes\n",
      "-- filtering region:NRNX1-seg-193, directly append 71 probes\n",
      "-- filtering region:NRNX1-seg-194, directly append 67 probes\n",
      "-- filtering region:NRNX1-seg-195, directly append 72 probes\n",
      "-- filtering region:NRNX1-seg-196, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-197, directly append 74 probes\n",
      "-- filtering region:NRNX1-seg-198, directly append 48 probes\n",
      "-- filtering region:NRNX1-seg-199, directly append 68 probes\n",
      "-- filtering region:NRNX1-seg-200, directly append 70 probes\n",
      "-- filtering region:NRNX1-seg-201, directly append 77 probes\n",
      "-- filtering region:NRNX1-seg-202, directly append 41 probes\n",
      "-- filtering region:NRNX1-seg-203, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-204, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-205, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-206, directly append 71 probes\n",
      "-- filtering region:NRNX1-seg-207, directly append 70 probes\n",
      "-- filtering region:NRNX1-seg-208, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-209, directly append 75 probes\n",
      "-- filtering region:NRNX1-seg-210, directly append 68 probes\n",
      "-- filtering region:NRNX1-seg-211, directly append 62 probes\n",
      "-- filtering region:NRNX1-seg-212, directly append 23 probes\n",
      "-- filtering region:NRNX1-seg-213, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-214, directly append 73 probes\n",
      "-- filtering region:NRNX1-seg-215, directly append 30 probes\n",
      "-- filtering region:NRNX1-seg-216, directly append 27 probes\n",
      "-- filtering region:NRNX1-seg-217, directly append 24 probes\n",
      "-- filtering region:NRNX1-seg-218, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-219, directly append 23 probes\n",
      "-- filtering region:NRNX1-seg-220, directly append 35 probes\n",
      "-- filtering region:NRNX1-seg-221, directly append 73 probes\n",
      "-- filtering region:NRNX1-seg-222, directly append 71 probes\n",
      "-- filtering region:NRNX1-seg-223, directly append 75 probes\n",
      "-- filtering region:NRNX1-seg-224, directly append 74 probes\n",
      "-- filtering region:NRNX1-seg-225, directly append 60 probes\n",
      "-- filtering region:NRNX1-seg-226, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-227, directly append 41 probes\n",
      "-- filtering region:NRNX1-seg-228, directly append 29 probes\n",
      "-- filtering region:NRNX1-seg-229, directly append 62 probes\n",
      "-- filtering region:NRNX1-seg-230, directly append 70 probes\n",
      "-- filtering region:NRNX1-seg-231, directly append 70 probes\n",
      "-- filtering region:NRNX1-seg-232, directly append 29 probes\n",
      "-- filtering region:NRNX1-seg-233, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-234, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-235, directly append 64 probes\n",
      "-- filtering region:NRNX1-seg-236, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-237, directly append 5 probes\n",
      "-- filtering region:NRNX1-seg-238, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-239, directly append 73 probes\n",
      "-- filtering region:NRNX1-seg-240, directly append 66 probes\n",
      "-- filtering region:NRNX1-seg-241, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-242, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-243, directly append 7 probes\n",
      "-- filtering region:NRNX1-seg-244, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-245, directly append 64 probes\n",
      "-- filtering region:NRNX1-seg-246, directly append 38 probes\n",
      "-- filtering region:NRNX1-seg-247, directly append 9 probes\n",
      "-- filtering region:NRNX1-seg-248, directly append 68 probes\n",
      "-- filtering region:NRNX1-seg-249, directly append 41 probes\n",
      "-- filtering region:NRNX1-seg-250, directly append 17 probes\n",
      "-- filtering region:NRNX1-seg-251, directly append 68 probes\n",
      "-- filtering region:NRNX1-seg-252, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-253, directly append 68 probes\n",
      "-- filtering region:NRNX1-seg-254, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-255, directly append 64 probes\n",
      "-- filtering region:NRNX1-seg-256, directly append 63 probes\n",
      "-- filtering region:NRNX1-seg-257, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-258, directly append 63 probes\n",
      "-- filtering region:NRNX1-seg-259, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-260, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-261, directly append 3 probes\n",
      "-- filtering region:NRNX1-seg-262, directly append 42 probes\n",
      "-- filtering region:NRNX1-seg-263, directly append 25 probes\n",
      "-- filtering region:NRNX1-seg-264, directly append 35 probes\n",
      "-- filtering region:NRNX1-seg-265, directly append 32 probes\n",
      "-- filtering region:NRNX1-seg-266, directly append 33 probes\n",
      "-- filtering region:NRNX1-seg-267, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-268, directly append 70 probes\n",
      "-- filtering region:NRNX1-seg-269, directly append 63 probes\n",
      "-- filtering region:NRNX1-seg-270, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-271, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-272, directly append 64 probes\n",
      "-- filtering region:NRNX1-seg-273, directly append 37 probes\n",
      "-- filtering region:NRNX1-seg-274, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-275, directly append 56 probes\n",
      "-- filtering region:NRNX1-seg-276, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-277, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-278, directly append 24 probes\n",
      "-- filtering region:NRNX1-seg-279, directly append 28 probes\n",
      "-- filtering region:NRNX1-seg-280, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-281, directly append 63 probes\n",
      "-- filtering region:NRNX1-seg-282, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-283, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-284, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-285, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-286, directly append 39 probes\n",
      "-- filtering region:NRNX1-seg-287, directly append 26 probes\n",
      "-- filtering region:NRNX1-seg-288, directly append 29 probes\n",
      "-- filtering region:NRNX1-seg-289, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-290, directly append 36 probes\n",
      "-- filtering region:NRNX1-seg-291, directly append 39 probes\n",
      "-- filtering region:NRNX1-seg-292, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-293, directly append 27 probes\n",
      "-- filtering region:NRNX1-seg-294, directly append 31 probes\n",
      "-- filtering region:NRNX1-seg-295, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-296, directly append 17 probes\n",
      "-- filtering region:NRNX1-seg-297, directly append 8 probes\n",
      "-- filtering region:NRNX1-seg-298, directly append 42 probes\n",
      "-- filtering region:NRNX1-seg-299, directly append 39 probes\n",
      "-- filtering region:NRNX1-seg-300, directly append 36 probes\n",
      "-- filtering region:NRNX1-seg-301, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-302, directly append 31 probes\n",
      "-- filtering region:NRNX1-seg-303, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-304, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-305, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-306, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-307, directly append 22 probes\n",
      "-- filtering region:NRNX1-seg-308, directly append 24 probes\n",
      "-- filtering region:NRNX1-seg-309, directly append 19 probes\n",
      "-- filtering region:NRNX1-seg-310, directly append 13 probes\n",
      "-- filtering region:NRNX1-seg-311, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-312, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-313, directly append 76 probes\n",
      "-- filtering region:NRNX1-seg-314, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-315, directly append 42 probes\n",
      "-- filtering region:NRNX1-seg-316, directly append 48 probes\n",
      "-- filtering region:NRNX1-seg-317, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-318, directly append 34 probes\n",
      "-- filtering region:NRNX1-seg-319, directly append 35 probes\n",
      "-- filtering region:NRNX1-seg-320, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-321, directly append 69 probes\n",
      "-- filtering region:NRNX1-seg-322, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-323, directly append 41 probes\n",
      "-- filtering region:NRNX1-seg-324, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-325, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-326, directly append 64 probes\n",
      "-- filtering region:NRNX1-seg-327, directly append 64 probes\n",
      "-- filtering region:NRNX1-seg-328, directly append 34 probes\n",
      "-- filtering region:NRNX1-seg-329, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-330, directly append 31 probes\n",
      "-- filtering region:NRNX1-seg-331, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-332, directly append 60 probes\n",
      "-- filtering region:NRNX1-seg-333, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-334, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-335, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-336, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-337, directly append 35 probes\n",
      "-- filtering region:NRNX1-seg-338, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-339, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-340, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-341, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-342, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-343, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-344, directly append 56 probes\n",
      "-- filtering region:NRNX1-seg-345, directly append 38 probes\n",
      "-- filtering region:NRNX1-seg-346, directly append 42 probes\n",
      "-- filtering region:NRNX1-seg-347, directly append 25 probes\n",
      "-- filtering region:NRNX1-seg-348, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-349, directly append 35 probes\n",
      "-- filtering region:NRNX1-seg-350, directly append 37 probes\n",
      "-- filtering region:NRNX1-seg-351, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-352, directly append 36 probes\n",
      "-- filtering region:NRNX1-seg-353, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-354, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-355, directly append 60 probes\n",
      "-- filtering region:NRNX1-seg-356, directly append 40 probes\n",
      "-- filtering region:NRNX1-seg-357, directly append 73 probes\n",
      "-- filtering region:NRNX1-seg-358, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-359, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-360, directly append 56 probes\n",
      "-- filtering region:NRNX1-seg-361, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-362, directly append 40 probes\n",
      "-- filtering region:NRNX1-seg-363, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-364, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-365, directly append 37 probes\n",
      "-- filtering region:NRNX1-seg-366, directly append 42 probes\n",
      "-- filtering region:NRNX1-seg-367, directly append 64 probes\n",
      "-- filtering region:NRNX1-seg-368, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-369, directly append 56 probes\n",
      "-- filtering region:NRNX1-seg-370, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-371, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-372, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-373, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-374, directly append 31 probes\n",
      "-- filtering region:NRNX1-seg-375, directly append 21 probes\n",
      "-- filtering region:NRNX1-seg-376, directly append 25 probes\n",
      "-- filtering region:NRNX1-seg-377, directly append 19 probes\n",
      "-- filtering region:NRNX1-seg-378, directly append 15 probes\n",
      "-- filtering region:NRNX1-seg-379, directly append 30 probes\n",
      "-- filtering region:NRNX1-seg-380, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-381, directly append 60 probes\n",
      "-- filtering region:NRNX1-seg-382, directly append 43 probes\n",
      "-- filtering region:NRNX1-seg-383, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-384, directly append 41 probes\n",
      "-- filtering region:NRNX1-seg-385, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-386, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-387, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-388, directly append 62 probes\n",
      "-- filtering region:NRNX1-seg-389, directly append 60 probes\n",
      "-- filtering region:NRNX1-seg-390, directly append 32 probes\n",
      "-- filtering region:NRNX1-seg-391, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-392, directly append 67 probes\n",
      "-- filtering region:NRNX1-seg-393, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-394, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-395, directly append 56 probes\n",
      "-- filtering region:NRNX1-seg-396, directly append 43 probes\n",
      "-- filtering region:NRNX1-seg-397, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-398, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-399, directly append 48 probes\n",
      "-- filtering region:NRNX1-seg-400, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-401, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-402, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-403, directly append 64 probes\n",
      "-- filtering region:NRNX1-seg-404, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-405, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-406, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-407, directly append 39 probes\n",
      "-- filtering region:NRNX1-seg-408, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-409, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-410, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-411, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-412, directly append 37 probes\n",
      "-- filtering region:NRNX1-seg-413, directly append 41 probes\n",
      "-- filtering region:NRNX1-seg-414, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-415, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-416, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-417, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-418, directly append 41 probes\n",
      "-- filtering region:NRNX1-seg-419, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-420, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-421, directly append 62 probes\n",
      "-- filtering region:NRNX1-seg-422, directly append 48 probes\n",
      "-- filtering region:NRNX1-seg-423, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-424, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-425, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-426, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-427, directly append 67 probes\n",
      "-- filtering region:NRNX1-seg-428, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-429, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-430, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-431, directly append 41 probes\n",
      "-- filtering region:NRNX1-seg-432, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-433, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-434, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-435, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-436, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-437, directly append 41 probes\n",
      "-- filtering region:NRNX1-seg-438, directly append 34 probes\n",
      "-- filtering region:NRNX1-seg-439, directly append 38 probes\n",
      "-- filtering region:NRNX1-seg-440, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-441, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-442, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-443, directly append 39 probes\n",
      "-- filtering region:NRNX1-seg-444, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-445, directly append 62 probes\n",
      "-- filtering region:NRNX1-seg-446, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-447, directly append 41 probes\n",
      "-- filtering region:NRNX1-seg-448, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-449, directly append 43 probes\n",
      "-- filtering region:NRNX1-seg-450, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-451, directly append 16 probes\n",
      "-- filtering region:NRNX1-seg-452, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-453, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-454, directly append 48 probes\n",
      "-- filtering region:NRNX1-seg-455, directly append 43 probes\n",
      "-- filtering region:NRNX1-seg-456, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-457, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-458, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-459, directly append 68 probes\n",
      "-- filtering region:NRNX1-seg-460, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-461, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-462, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-463, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-464, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-465, directly append 60 probes\n",
      "-- filtering region:NRNX1-seg-466, directly append 34 probes\n",
      "-- filtering region:NRNX1-seg-467, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-468, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-469, directly append 68 probes\n",
      "-- filtering region:NRNX1-seg-470, directly append 79 probes\n",
      "-- filtering region:NRNX1-seg-471, directly append 62 probes\n",
      "-- filtering region:NRNX1-seg-472, directly append 26 probes\n",
      "-- filtering region:NRNX1-seg-473, directly append 14 probes\n",
      "-- filtering region:NRNX1-seg-474, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-475, directly append 60 probes\n",
      "-- filtering region:NRNX1-seg-476, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-477, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-478, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-479, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-480, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-481, directly append 66 probes\n",
      "-- filtering region:NRNX1-seg-482, directly append 60 probes\n",
      "-- filtering region:NRNX1-seg-483, directly append 25 probes\n",
      "-- filtering region:NRNX1-seg-484, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-485, directly append 63 probes\n",
      "-- filtering region:NRNX1-seg-486, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-487, directly append 64 probes\n",
      "-- filtering region:NRNX1-seg-488, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-489, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-490, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-491, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-492, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-493, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-494, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-495, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-496, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-497, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-498, screen genome to keep 80 probes\n",
      "-- filtering region:NRNX1-seg-499, directly append 66 probes\n",
      "-- filtering region:NRNX1-seg-500, directly append 67 probes\n",
      "-- filtering region:NRNX1-seg-501, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-502, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-503, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-504, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-505, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-506, directly append 60 probes\n",
      "-- filtering region:NRNX1-seg-507, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-508, directly append 62 probes\n",
      "-- filtering region:NRNX1-seg-509, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-510, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-511, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-512, directly append 56 probes\n",
      "-- filtering region:NRNX1-seg-513, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-514, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-515, directly append 63 probes\n",
      "-- filtering region:NRNX1-seg-516, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-517, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-518, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-519, directly append 62 probes\n",
      "-- filtering region:NRNX1-seg-520, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-521, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-522, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-523, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-524, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-525, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-526, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-527, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-528, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-529, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-530, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-531, directly append 48 probes\n",
      "-- filtering region:NRNX1-seg-532, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-533, directly append 67 probes\n",
      "-- filtering region:NRNX1-seg-534, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-535, directly append 39 probes\n",
      "-- filtering region:NRNX1-seg-536, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-537, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-538, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-539, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-540, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-541, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-542, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-543, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-544, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-545, directly append 60 probes\n",
      "-- filtering region:NRNX1-seg-546, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-547, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-548, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-549, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-550, directly append 56 probes\n",
      "-- filtering region:NRNX1-seg-551, directly append 48 probes\n",
      "-- filtering region:NRNX1-seg-552, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-553, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-554, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-555, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-556, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-557, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-558, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-559, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-560, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-561, directly append 40 probes\n",
      "-- filtering region:NRNX1-seg-562, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-563, directly append 43 probes\n",
      "-- filtering region:NRNX1-seg-564, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-565, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-566, directly append 41 probes\n",
      "-- filtering region:NRNX1-seg-567, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-568, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-569, directly append 64 probes\n",
      "-- filtering region:NRNX1-seg-570, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-571, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-572, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-573, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-574, directly append 43 probes\n",
      "-- filtering region:NRNX1-seg-575, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-576, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-577, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-578, directly append 39 probes\n",
      "-- filtering region:NRNX1-seg-579, directly append 41 probes\n",
      "-- filtering region:NRNX1-seg-580, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-581, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-582, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-583, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-584, directly append 39 probes\n",
      "-- filtering region:NRNX1-seg-585, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-586, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-587, directly append 35 probes\n",
      "-- filtering region:NRNX1-seg-588, directly append 38 probes\n",
      "-- filtering region:NRNX1-seg-589, directly append 41 probes\n",
      "-- filtering region:NRNX1-seg-590, directly append 52 probes\n",
      "-- filtering region:NRNX1-seg-591, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-592, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-593, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-594, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-595, directly append 43 probes\n",
      "-- filtering region:NRNX1-seg-596, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-597, directly append 39 probes\n",
      "-- filtering region:NRNX1-seg-598, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-599, directly append 59 probes\n",
      "-- filtering region:NRNX1-seg-600, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-601, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-602, directly append 42 probes\n",
      "-- filtering region:NRNX1-seg-603, directly append 60 probes\n",
      "-- filtering region:NRNX1-seg-604, directly append 38 probes\n",
      "-- filtering region:NRNX1-seg-605, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-606, directly append 67 probes\n",
      "-- filtering region:NRNX1-seg-607, directly append 68 probes\n",
      "-- filtering region:NRNX1-seg-608, directly append 56 probes\n",
      "-- filtering region:NRNX1-seg-609, directly append 32 probes\n",
      "-- filtering region:NRNX1-seg-610, directly append 27 probes\n",
      "-- filtering region:NRNX1-seg-611, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-612, directly append 30 probes\n",
      "-- filtering region:NRNX1-seg-613, directly append 28 probes\n",
      "-- filtering region:NRNX1-seg-614, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-615, directly append 33 probes\n",
      "-- filtering region:NRNX1-seg-616, directly append 42 probes\n",
      "-- filtering region:NRNX1-seg-617, directly append 48 probes\n",
      "-- filtering region:NRNX1-seg-618, directly append 60 probes\n",
      "-- filtering region:NRNX1-seg-619, directly append 73 probes\n",
      "-- filtering region:NRNX1-seg-620, directly append 75 probes\n",
      "-- filtering region:NRNX1-seg-621, directly append 56 probes\n",
      "-- filtering region:NRNX1-seg-622, directly append 65 probes\n",
      "-- filtering region:NRNX1-seg-623, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-624, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-625, directly append 39 probes\n",
      "-- filtering region:NRNX1-seg-626, directly append 48 probes\n",
      "-- filtering region:NRNX1-seg-627, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-628, directly append 37 probes\n",
      "-- filtering region:NRNX1-seg-629, directly append 40 probes\n",
      "-- filtering region:NRNX1-seg-630, directly append 34 probes\n",
      "-- filtering region:NRNX1-seg-631, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-632, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-633, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-634, directly append 64 probes\n",
      "-- filtering region:NRNX1-seg-635, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-636, directly append 32 probes\n",
      "-- filtering region:NRNX1-seg-637, directly append 22 probes\n",
      "-- filtering region:NRNX1-seg-638, directly append 18 probes\n",
      "-- filtering region:NRNX1-seg-639, directly append 32 probes\n",
      "-- filtering region:NRNX1-seg-640, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-641, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-642, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-643, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-644, directly append 58 probes\n",
      "-- filtering region:NRNX1-seg-645, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-646, directly append 69 probes\n",
      "-- filtering region:NRNX1-seg-647, directly append 33 probes\n",
      "-- filtering region:NRNX1-seg-648, directly append 41 probes\n",
      "-- filtering region:NRNX1-seg-649, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-650, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-651, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-652, directly append 35 probes\n",
      "-- filtering region:NRNX1-seg-653, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-654, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-655, directly append 55 probes\n",
      "-- filtering region:NRNX1-seg-656, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-657, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-658, directly append 60 probes\n",
      "-- filtering region:NRNX1-seg-659, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-660, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-661, directly append 51 probes\n",
      "-- filtering region:NRNX1-seg-662, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-663, directly append 16 probes\n",
      "-- filtering region:NRNX1-seg-664, directly append 15 probes\n",
      "-- filtering region:NRNX1-seg-665, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-666, directly append 68 probes\n",
      "-- filtering region:NRNX1-seg-667, directly append 50 probes\n",
      "-- filtering region:NRNX1-seg-668, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-669, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-670, directly append 33 probes\n",
      "-- filtering region:NRNX1-seg-671, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-672, directly append 38 probes\n",
      "-- filtering region:NRNX1-seg-673, directly append 1 probes\n",
      "-- filtering region:NRNX1-seg-674, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-675, directly append 43 probes\n",
      "-- filtering region:NRNX1-seg-676, directly append 40 probes\n",
      "-- filtering region:NRNX1-seg-677, directly append 11 probes\n",
      "-- filtering region:NRNX1-seg-678, directly append 44 probes\n",
      "-- filtering region:NRNX1-seg-679, directly append 22 probes\n",
      "-- filtering region:NRNX1-seg-680, directly append 9 probes\n",
      "-- filtering region:NRNX1-seg-681, directly append 31 probes\n",
      "-- filtering region:NRNX1-seg-682, directly append 39 probes\n",
      "-- filtering region:NRNX1-seg-683, directly append 53 probes\n",
      "-- filtering region:NRNX1-seg-684, directly append 23 probes\n",
      "-- filtering region:NRNX1-seg-685, directly append 37 probes\n",
      "-- filtering region:NRNX1-seg-686, directly append 28 probes\n",
      "-- filtering region:NRNX1-seg-687, directly append 27 probes\n",
      "-- filtering region:NRNX1-seg-688, directly append 28 probes\n",
      "-- filtering region:NRNX1-seg-689, directly append 45 probes\n",
      "-- filtering region:NRNX1-seg-690, directly append 43 probes\n",
      "-- filtering region:NRNX1-seg-691, directly append 36 probes\n",
      "-- filtering region:NRNX1-seg-692, directly append 47 probes\n",
      "-- filtering region:NRNX1-seg-693, directly append 54 probes\n",
      "-- filtering region:NRNX1-seg-694, directly append 49 probes\n",
      "-- filtering region:NRNX1-seg-695, directly append 57 probes\n",
      "-- filtering region:NRNX1-seg-696, directly append 40 probes\n",
      "-- filtering region:NRNX1-seg-697, directly append 42 probes\n",
      "-- filtering region:NRNX1-seg-698, directly append 61 probes\n",
      "-- filtering region:NRNX1-seg-699, directly append 46 probes\n",
      "-- filtering region:NRNX1-seg-700, directly append 13 probes\n",
      "region: NRNX1-seg-8 removed with 19 probes\n",
      "region: NRNX1-seg-11 removed with 20 probes\n",
      "region: NRNX1-seg-12 removed with 4 probes\n",
      "region: NRNX1-seg-29 removed with 11 probes\n",
      "region: NRNX1-seg-61 removed with 17 probes\n",
      "region: NRNX1-seg-78 removed with 13 probes\n",
      "region: NRNX1-seg-116 removed with 8 probes\n",
      "region: NRNX1-seg-132 removed with 19 probes\n",
      "region: NRNX1-seg-153 removed with 17 probes\n",
      "region: NRNX1-seg-167 removed with 6 probes\n",
      "region: NRNX1-seg-168 removed with 8 probes\n",
      "region: NRNX1-seg-237 removed with 5 probes\n",
      "region: NRNX1-seg-243 removed with 7 probes\n",
      "region: NRNX1-seg-247 removed with 9 probes\n",
      "region: NRNX1-seg-250 removed with 17 probes\n",
      "region: NRNX1-seg-261 removed with 3 probes\n",
      "region: NRNX1-seg-296 removed with 17 probes\n",
      "region: NRNX1-seg-297 removed with 8 probes\n",
      "region: NRNX1-seg-309 removed with 19 probes\n",
      "region: NRNX1-seg-310 removed with 13 probes\n",
      "region: NRNX1-seg-377 removed with 19 probes\n",
      "region: NRNX1-seg-378 removed with 15 probes\n",
      "region: NRNX1-seg-451 removed with 16 probes\n",
      "region: NRNX1-seg-473 removed with 14 probes\n",
      "region: NRNX1-seg-638 removed with 18 probes\n",
      "region: NRNX1-seg-663 removed with 16 probes\n",
      "region: NRNX1-seg-664 removed with 15 probes\n",
      "region: NRNX1-seg-673 removed with 1 probes\n",
      "region: NRNX1-seg-677 removed with 11 probes\n",
      "region: NRNX1-seg-680 removed with 9 probes\n",
      "region: NRNX1-seg-700 removed with 13 probes\n"
     ]
    }
   ],
   "source": [
    "# filter by genome counts\n",
    "reload(library_tools.quality_check)\n",
    "\n",
    "num_probes_kept = 80 # max num of probes kept for each region\n",
    "filtered_pb_dict = library_tools.quality_check.Screen_probe_by_hit(library_folder, pb_dict, num_probes_kept,\n",
    "                                                                  hit_type='genome')\n",
    "\n",
    "## filter by minimum number of probes in each region\n",
    "min_num_probes = 21\n",
    "removed_regions = []\n",
    "for _rid, _pb_class in filtered_pb_dict.items():\n",
    "    if len(_pb_class.kept_probes) < min_num_probes:\n",
    "        print(f\"region: {_rid} removed with {len(_pb_class.kept_probes)} probes\")\n",
    "        removed_regions.append(_rid)\n",
    "for _rid in removed_regions:\n",
    "    del(filtered_pb_dict[_rid])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "5571da09",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(699, 668)"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(pb_dict), len(filtered_pb_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "4b910471",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7snQm4FsWx94tNEAQRwRVBQAkKAqLiGperxiVXg8Ytbmj0Jkrc4sYHEnejaDSaqJDENRB3QNHEJRJFjGEVVCQGg+wqalBRASNwvqcmd86d856Z6eqe7pme9/3P8+RJcqa7uupXNUNNvzU1Terq6uoIBwiAAAiAAAiAAAiAAAjUCIEmSIBrxNMwEwRAAARAAARAAARAICCABBiBAAIgAAIgAAIgAAIgUFMEkADXlLthLAiAAAiAAAiAAAiAABJgxAAIgAAIgAAIgAAIgEBNEUACXFPuhrEgAAIgAAIgAAIgAAJIgBEDIAACIAACIAACIAACNUUACXBNuRvGggAIgAAIgAAIgAAIIAFGDIAACIAACIAACIAACNQUASTANeVuGAsCIAACIAACIAACIIAEGDEAAiAAAiAAAiAAAiBQUwSQANeUu2EsCIAACIAACIAACIAAEmDEAAiAAAiAAAiAAAiAQE0RQAJcU+6GsSAAAiAAAiAAAiAAAkiAEQMgAAIgAAIgAAIgAAI1RQAJcE25G8aCAAiAAAiAAAiAAAggAUYMgAAIgAAIgAAIgAAI1BQBJMA15W4YCwIgAAIgAAIgAAIggAQYMQACIAACIAACIAACIFBTBJAA15S7YSwIgAAIgAAIgAAIgAASYMQACIAACIAACIAACIBATRFAAlxT7oaxIAACIAACIAACIAACSIARAyAAAiAAAiAAAiAAAjVFAAlwTbkbxoIACIAACIAACIAACCABRgyAAAiAAAiAAAiAAAjUFAEkwDXlbhgLAiAAAiAAAiAAAiCABBgxAAIgAAIgAAIgAAIgUFMEkADXlLthLAiAAAiAAAiAAAiAABJgxAAIgAAIgAAIgAAIgEBNEUACXFPuhrEgAAIgAAIgAAIgAAJIgBEDIAACIAACIAACIAACNUUACXBNuRvGggAIgAAIgAAIgAAIIAFGDIAACIAACIAACIAACNQUASTANeVuGAsCIAACIAACIAACIIAEGDEAAiAAAiAAAiAAAiBQUwSQANeUu2EsCIAACIAACIAACIAAEmDEAAiAAAiAAAiAAAiAQE0RQAJcU+6GsSAAAiAAAiAAAiAAAkiAEQMgAAIgAAIgAAIgAAI1RQAJcE25G8aCAAiAAAiAAAiAAAggAUYMgEANEDjjjDPowQcfbGBpkyZNqHXr1rTjjjvS6aefThdeeCE1bdrUCo1FixZRt27daOjQoXTTTTdZkVkGIQceeCC988479OGHH3qt7r///W86//zz6dFHH6V169bRjTfeGPz/jz76iFq1akXt2rVzqv+yZcuIY/K1114L1vvzn/9Mu+++O5144on0yCOPBGtzfEb/P//Ntn4vv/wyHXTQQTRq1Cg655xznNoM4SAAAn4RQALslz+gDQg4IRAmwL/85S+pY8eOwRp1dXX05Zdf0oQJE4IEhBOgX/3qV1bWRwLsdwJ85513Bv4+4YQT6Dvf+Q7tu+++tHDhQjr55JPpb3/7G/Xq1ctKHCQJOe6442jcuHF0xRVXUJcuXYJ1n3zySdp+++1pv/32i02An332Wev6rVixIoj9PffcM3gQxAECIFA7BJAA146vYWkNEwgTYE5yOMmIHuvXr6d99tmHZs6cSbwzt/XWW2cmhQTY7wT4Rz/6Ef3ud78jTgC32GKLwN9XX301XXPNNfT3v//deQLcs2dPat68Oc2bNy8x1ip3gPPUL/MFAAEgAALeE0AC7L2LoCAIZCeQlgCz9Ntuu40uueQSGj9+PB1zzDGZF0QC7HcCHMbDmjVrghKEvBNgfgjbaqutaOrUqUiAM19tEAACIGBCAAmwCTXMAYGSEVAlwLfffjv99Kc/DWpC+WdxHv/qq6/S//t//y+o4+U60bvvvptOOeUU+uyzz+jKK68MSid4B7Fz585BrebPfvazoKaYjzAB5qS6RYsWdN999wXlFt/+9reDmuD+/fs3IMg/b99www00e/ZsatasWfAz+PXXX08DBgxIJR3q+fjjj9Oll14aJFSc0H3ve9+jX/ziF9ShQ4fU5G706NF07rnn0ksvvURcvxvqzfouWLAg0Jvt5Z/If/Ob39BGG21EF1xwAf3lL3+h9u3bB7XT1113XaAzH2ENMNdbX3bZZfSPf/yDunfvHpQbDBkypIEty5cvpxEjRtCf/vSnYA3+CZ51+clPflI/7oEHHqAzzzwzKBdgP/AO/Q9/+EO66667Yrkw45EjRwYPMu+9915Q0827rVzfzaz44J3V6NG1a9dA72iN+AEHHEBcH8sH1zSznmwzJ8y9e/emyy+/PIiT8ODdWfYXlzH8+Mc/DuzheOD4iR5hzW30b4MHDya2s3LHN/r/K2vYbem31157NagBDv1///330+LFiwP/c4x/61vfoquuuoqOPfbYBvY89NBDdPPNNwd+3nbbbYMY5F9SXnzxxSCWcIAACPhLAAmwv76BZiBgjYAqAf7ud78bJGLvvvsu7bDDDkGy9Nhjj1GbNm2CJIYTmrPPPps23XTToF6U/8H/n//5H+rbt2+QdP7+978P/s5JEieJYSLB83kOJ4CcRHOi/fXXX9P06dNpp512Cuzj5IeTOk56uTaUk7h77rmHPvjgA5o0aVJQnpF0sJ5PPfVUkDwNGjSIOKHhxH3MmDF0/PHHBzbwkfTzeVICzEn9ZpttFiRzS5cuDZJpToK++uqrIGHae++96Q9/+AO98sor9Nvf/jZgESbAXEPL+nBit8suuwQ6TJkyJUiUOZHk4/3336c99tiDvvnmmyDp3XLLLen555+niRMnBgn2HXfcUc+GE+C2bdsGunCSxXocccQRsUhYt2nTpgVJNNfx8st4rN+SJUuChJYTx7FjxwZ/Y504wWM7ef1bbrkleKjhhI4fUA499FB66623Ar/wgwSXTbA/ORlnu/lXA35oCvlee+21gSz+Gz8QHHLIIYGN0SOsueUxHBfslx49egQ80xJgZupCv9Cf4UtwYdzyQwGXaPCLcWwL28oPLG+++Sb16dMnMInn8EMNxxzXMPNcrq3eZJNNAn8hAbZ2+4IgEHBCAAmwE6wQCgJ+EQgT4Ndff5222267QDmu/eXkjpMg/sf8+9//Pj3xxBPBuXA8n+MELDyGDx8edAzgJIp3g8Pj1ltvDXa/OMHl3cZoAsw1peGanFBxcsVlFrzWqlWrgnO8A8mJbHh8/vnnQfLYqVMnmjVrViLMUE/eVeYd0vDgRJATYZbDu9K6CTDXxf7zn/8MEhk+OJlmfaMvCrJsHvff//3fQVLIB9sxefJkYh4XX3xx8DdO/Pfff39i9pxEbb755gFfTjbfeOONBjXZzI5fROS/88NFuAPMO82VXTwqocyYMYMGDhxI/KLjRRddVH+aZTHzqO7SEgi2h3eSOfHjHW8+NmzYEMTKc889FyTW7KOQL/8ywHXEqiOuBCItAWZ5cT7Mql9lF4gwbvmBgB/yOEnng33Ka3H88y8V/JAWPoz89a9/DX7l4IN33pkNJ9BIgFVRgPMgUCwBJMDF8sfqIJALgbg2aOHCvNPFO1i8exUmfOF4Tn64nVl47LzzzrR27dqgPCD6Uzq31eJEaNdddw12GsNEgncNuXQgenDXAW5/xQkk/2TOu76c3B155JENxnEyxYk5/+zPyUbckbSzzcknJ4KcoHGCrZsAn3rqqcEucnjwzi0nPmGpRPh3ls3/YXv44CSJE91PPvkk2AkPD/6pnB8YHn744aB0gHdUedczugaP5YcFTpZ5LU62wgS48kEkKWiYKZeAtGzZMhjCnT74AYJ3YvlBhuXwIUmA//WvfwUdQ9iHrE/04N1qZsS74Bw7IV/+BYAfPlSHjQTYhn5JCXBYlhHawQ9qnAzzLxX33ntv8LDGvzhUPgjyeP5lg0tFkACrogDnQaBYAkiAi+WP1UEgFwJhwsP/YPPuFh9cH8r/qPNP5fzTdvQIx/NOV/TcxhtvHCR5XLNbeXDy++mnnwb/8IcJcPRn8nA8/2zMiS2XOHDiW1knWimXk0tOFuOOUE/+KTusP+ZxLJNrYcOuF7oJMNcuc9lDeITzuWtBWLrB5yoTOWbDDHjXNXpwyQfXEfPuOSei/AJY2hE+OIQJ8DPPPENcpqI6Vq9eTVy/yokdl7PwLjaz4SOa1EkSYC6l4J/3046f//znNGzYsPoEeO7cuUGNsOqwkQDb0C8pAeb6bS4FCQ9+6OPYDxmGv3iwXyvLPPjXDa5lRwKsigKcB4FiCSABLpY/VgeBXAioaoArlYhLkHgM7y7yDl9cAtyvX79g52v+/Pn1CXBYEhGVz3WVvCv88ccfB624eKeTX+ril7Xijt122y2oLY07kvSUJsBhP9zKl+AqP+CRlEDHJcC8C8sJUPTgOmlO4rmOlXeCt9lmGzr66KODsoS4g8/zbnuYADPvww8/PDVWuE6b63U58T344IOJ/cGlD5ygcZ2tbgLMdbdcf83JOJeAxB38gh//R7dFmY0E2IZ+SQlwpf8rE2AuueHEPyxVibL5wQ9+EPRSRgKcy60Ni4CAMQEkwMboMBEEykPAVgLMu5+cDHBpRGUJBP9czskWv7gW7gBzTSsnwdGDE+g5c+bQypUrg69+8U/oXBpw0kknNUoa+adnLgkIW3VJE/XKBJhfQOOSCl6XE8Pw4E4F3L3AZgLMdc78xbKwMwSvxTvd7AOuET3qqKOCnXfuiMF1tNGDHwr4BTP+KhrXkeokwPwSGncq+OMf/9ignIR9ZZIA8w49J+L88h2/LBg9eGed64J5h5h/USgiAbahn2kCzPXg/FDA3Ue4hCd6sO+4BAYJcHnuj9C0NgkgAa5Nv8PqGiNgKwEOE8vK2kcuF+CfjcMXsMIEmH/q57rW8AUqflGLSwH4pS5O7riOk2toebeTXyYKa1c5OeaXwLiGldtRcZ1y3CHdAeadZt7J5PKAsB0Y1y1zm7W3337bagLML0xFE3pehxNFLkfgemb+zDDXAfOLc7wzHP0JPSwPCZNYnQQ4fIGu8mMnvMvMO92888x+4yOOW/iQwDzYH3zwS3Vc9sF/44ScD/YJv/jHXUPCkoc8EmAX+pkmwPxgxnXp7Fd+iAkfdjiGeRceL8HV2A0W5paSABLgUroNSoOAHgFbCTD/zM5JEe8qckIZbYPGO1/cWouT2DAB5o4HvEPIrb54V4x3g/k8v5jFrcb44K4HnLyxrNNOOy1IdnnHkUspOJHkHsNJhzQBDhNtXpsTdd6B5WSYa5z5bX+bO8D8szh3fjjvvPOCJImTTq5X5bIPZsYH8+MHAS4Z4aSXW8/xC2Tch5lrfbkdGtdo6yTAL7zwAh122GHBDnfYlo13nDnJ4wSNW6eFnTbiuHFrNN7tPeuss4LyDP4P68079lwHznryVwK5ewWvxT7l3tB85JEAu9DPNAFmm7lsh33M7f84Rrm1Hccyd8ngBz9+EMEBAiDgLwEkwP76BpqBgDUCthJgVogTWf6pnTs48P/u0qULcd0j10Tyi0J8hAkw7wzzDi4ngdzzlnvL8t+4bjR6cFLFf+cSBe6ewC3QWF5Sv9twrjQB5vFcWsA72NylgWuKeReaO1Jwv1qbCTAn1VdccUWgPydB3LeXu0hEPxzB+vA5LsvgZJLrhnnXkHdpOUEPOeokwCyTx3OdMSfYbCP3rOW1+W+8O8l9gZlvHDfWgXXkHWzu/ME793xwQs8JLv+dy1+4nIITbO41HO585pEAu9AvSwLMbLirBr8Qx7v7fB1wPTsn6vwiJH9ABAcIgIC/BJAA++sbaAYCIAACIOAhAf6YC3fciHs5k+vkuSUgP3DhAAEQ8JcAEmB/fQPNQAAEQAAEPCQQ/sJR+eEP3i3ndoDchzraRs9DE6ASCNQ8ASTANR8CAAACIAACIKBLgLuTcI0010JzyQ6/4Mj9rfngUh5Vr2fd9TAeBEDALgEkwHZ5QhoIgAAIgEANEOAXQvlDIPyiISe//HU/rinnbhXhp79rAANMBIHSEqj6BJh7fL744ovBm9DhwU/n/Nb5zJkzid9Sv+CCC+jSSy+tP89v8fL37O+5557gZQZua8NvO/Ob2jhAAARAAARAAARAAATKTaCqE2BuucS1WPxTVZgAczsk/vQrf8edz3EfTm7vwwkuf6KUD05++f9zmyRuY3T55ZcHb1VzL0x+gxoHCIAACIAACIAACIBAeQlUZQK8fPlyOvvss4OepPxTFPchDRPgG2+8MejfyC8xhM31uXUN/4zFbWu4aT1/0Yq/A8+fbOWDf+riLyJxy5vKr1WV1/XQHARAAARAAARAAARqk0BVJsBPP/100ECfyx/486Cc7IYJ8JFHHhl8leqhhx6q9ziXSHB/0hUrVgRjuUE9N8fv2bNn/Rgug+BG/WHj99oMF1gNAiAAAiAAAiAAAuUnUJUJcNQt3PA9mgBzEsvN9UeOHFk/jD/12bt3b+LPtC5ZsoS+//3vBz0ew2b0PJAbxPPfnnnmmUSv8+cx+T/hwU3jly5dGjSVT/qUa/lDCBaAAAiAAAiAAAj4SIC/Svnxxx8HnUpatWrlo4qF6VRzCTC/yHbyyScHO8PhwfW9/HUjLpngZJk/x7p+/frgU6ThwV+N4k9d8m5x0hF+Dakwb2JhEAABEAABEAABEKggMH36dNpjjz3AJUKg5hJgfgriMoi4HWD+RConw8cdd1zsDjB//eepp54S7wDz7u8+++xDHHhbb701Ag8EQAAEQAAEQAAEciPwwQcf0MCBA4NPr2+//fa5rVuGhWouAebkl1ufjRkzpt4/cTXA/G133hUOD64B7tevX/ACnfTg3pD8Eh4nwp07d5ZOwzgQAAEQAAEQAAEQyEwAeUgywppLgG+66SYaPXo0LViwgJo1axaQGTZsGE2YMCHoAsG7vFtssQXddtttdNZZZwXnTbtAIPAyX7sQAAIgAAIgAAIgYEgAeQgS4PouEB999FHQB/joo48O+vtyeQJ/ypKT4sGDBwekrrjiCvrNb34TtD3jnwwuu+yyoDb4rbfe0uoDjMAzvGIxDQRAAARAAARAIDMB5CFIgBt8CY67PfDX32bPnh3U5l5yySV03nnn1VPiF+C4NzB/CGPNmjXBhzS49EG3fgaBl/nahQAQAAEQAAEQAAFDAshDajgBNowZK9MQeFYwQggIgAAIgAAIgIABAeQhSIANwib7FARedoaQAAIgAAIgAAIgYEYAeQgSYLPIyTgLgZcRIKaDAAiAAAiAAAgYE0AeggTYOHiyTETgZaGHuSAAAiAAAiAAAlkIIA9BApwlfoznIvCM0WEiCIAACIAACIBARgLIQ5AAZwwhs+kIPDNumAUCIAACIAACIJCdAPIQJMDZo8hAAgLPABqmgAAIgAAIgAAIWCGAPAQJsJVA0hWCwNMlhvEgAAIgAAIgAAK2CCAPQQJsK5a05CDwtHBhMAiAAAiAAAiAgEUCyEOQAFsMJ7koBJ6cFUaCAAiAAAiAAAjYJYA8BAmw3YgSSkPgCUFhGAiAAAiAAAiAgHUCyEOQAFsPKolABJ6EEsaAAAiAAAiAAAi4IIA8BAmwi7hSykTgKRFhAAiAAAiAAAiAgCMCyEOQADsKrXSxCLxCsGNREAABEAABEAABIkIeggS4kAsBgVcIdiwKAiAAAiAAAiCABDg1BprU1dXVIUrcEEAC7IYrpIIACIAACIAACKgJIA/BDrA6ShyMQOA5gAqRIAACIAACIAACIgLIQ5AAiwLF9iAEnm2ikAcCIAACIAACICAlgDwECbA0VqyOQ+BZxQlhIAACIAACIAACGgSQhyAB1ggXe0MRePZYQhIIgAAIgAAIgIAeAeQhSID1IsbSaASeJZAQAwIgAAIgAAIgoE0AeQgSYO2gsTEBgWeDImSAAAiAAAiAAAiYEEAeggTYJG4yz0HgZUYIASAAAiAAAiAAAoYEkIcgATYMnWzTEHjZ+GE2CIAACIAACICAOQHkIUiAzaMnw0wEXgZ4mAoCIAACIAACIJCJAPIQJMCZAsh0MgLPlBzmgQAIgAAIgAAIZCWAPAQJcNYYMpqPwDPChkkgAAIgAAIgAAIWCCAPQQJsIYz0RSDw9JlhBgiAAAiAAAiAgB0CyEOQANuJJE0pCDxNYBgOAiAAAiAAAiBgjQDyECTA1oJJRxACT4cWxoIACIAACIAACNgkgDwECbDNeBLLQuCJUWEgCIAACIAACICAZQLIQ5AAWw4pmTgEnowTRoEACIAACIAACNgngDwECbD9qBJIROAJIGEICIAACIAACICAEwLIQ5AAOwkslVAEnooQzoMACIAACIAACLgigDwECbCr2EqVi8ArBDsWBQEQAAEQAAEQICLkIUiAC7kQEHiFYMeiIAACIAACIAACSIBTY6BJXV1dHaLEDQEkwG64QioIgAAIgAAIgICaAPIQ7ACro8TBCASeA6gQCQIgAAIgAAIgICKAPAQJsChQbA9C4NkmCnkgAAIgAAIgAAJSAshDkABLY8XqOASeVZwQBgIgAAIgAAIgoEEAeQgSYI1wsTcUgWePJSSBAAiAAAiAAAjoEUAeggRYL2IsjUbgWQIJMSAAAiAAAiAAAtoEkIcgAdYOGhsTEHg2KEIGCIAACIAACICACQHkIUiATeIm8xwEXmaEEAACIAACIAACIGBIAHkIEmDD0Mk2DYGXjR9mgwAIgAAIgAAImBNAHoIE2Dx6MsxE4GWAh6kgAAIgAAIgAAKZCCAPQQKcKYBMJyPwTMlhHgiAAAiAAAiAQFYCyEOQAGeNIaP5CDwjbJgEAiAAAiAAAiBggQDyECTAFsJIXwQCT58ZZoAACIAACIAACNghgDwECbCdSNKUgsDTBIbhIAACIAACIAAC1gggD0ECbC2YdAQh8HRoYSwIgAAIgAAIgIBNAshDkADbjCexLASeGBUGggAIgAAIgAAIWCaAPAQJsOWQkolD4Mk4YRQIgAAIgAAIgIB9AshDkADbjyqBRASeABKGgAAIgAAIgAAIOCGAPAQJsJPAUglF4KkI4TwIgAAIgAAIgIArAshDkAC7iq1UuQi8QrBjURAAARAAARAAASJCHoIEuJALAYFXCHYsCgIgAAIgAAIggAQ4NQaa1NXV1SFK3BBAAuyGK6SCAAiAAAiAAAioCSAPwQ6wOkocjEDgOYAKkSAAAiAAAiAAAiICyEOQAIsCxfYgBJ5topAHAiAAAiAAAiAgJYA8BAmwNFasjkPgWcUJYSAAAiAAAiAAAhoEkIcgAdYIF3tDEXj2WEISCIAACIAACICAHgHkIUiA9SLG0mgEniWQEAMCIAACIAACIKBNAHkIEmDtoLExAYFngyJkgAAIgAAIgAAImBBAHoIE2CRuMs9B4GVGCAEgAAIgAAIgAAKGBJCHIAE2DJ1s0xB42fhhNgiAAAiAAAiAgDkB5CFIgM2jJ8NMBF4GeJgKAiAAAiAAAiCQiQDyECTAmQLIdDICz5Qc5oEACIAACIAACGQlgDwECXDWGDKaj8AzwoZJIAACIAACIAACFgggD0ECbCGM9EUg8PSZYQYIgAAIgAAIgIAdAshDkAA3IvDNN9/Q1VdfTWPGjKFPP/2U+vfvTyNHjqR99tknGDtnzhy68MILaebMmbT55pvTBRdcQJdeeqlWRCLwtHBhMAiAAAiAAAiAgEUCyEOQADcicNVVV9Fvf/tbevDBB6l79+5B8vvYY4/R3//+d2rZsiX16tWLBg0aRBdffDFNnTqVhgwZQnfffTedeeaZ4tBE4IlRYSAIgAAIgAAIgIBlAshDkAA3IsA7vgcffDDdeuutwblVq1bRpptuSk888QTNnz+f7rrrLlq0aBE1b948OD98+HAaP348vfPOO+LwROCJUWEgCIAACIAACICAZQLIQ5AANyLAyS8HxvPPP0/bbbcd/fKXv6QRI0bQ22+/Teeffz61b9+eHnroofp5L774Ih166KG0YsUK2mKLLUQhisATYar5Qf9et4HGvb6MlqxcTV07tKbv79aZWjRrasTFpiwjBTAJBGqAAK6zGnBylZiIPAQJcCMCc+fOpRNPPJHmzZtHzZo1o6ZNmwa7v0cffTT17duXjjjiiKAsIjx4XO/evWnGjBm0++67iy4NBJ4IU00P4n9IT793Gk1duLKew17dN6cxZw3UToJtyqppp8B4EEghgOsM4VEmAshDkAA3IvD444/THXfcQZdddhltu+229Lvf/Y4effRReuWVV+jYY4+lk08+ma699tr6ee+99x716NGDpkyZQvvtt18sUS6j4P+ExwcffEADBw6kpUuXUufOnct0zUDXnAg8PH0JDRv/VqPVbjp2FzppYBctLWzK0loYg0GghgjgOqshZ1eBqUiAkQA3ILB48WLacccdadKkSfTtb3+7/hz/744dO9I///lPOvLII2N3gF9//XXaddddY4lyV4lrrrmm0TkkwFVwF3Fkwsjn3qFRLy9oJP3cA3vQ0MN7aa1qU5bWwhgMAjVEANdZDTm7CkxFAowEuAEB3v094YQTaM2aNdSqVav6c7wb/Kc//Ym6du0atD7jFmnhIakBxg5wFdwtcjbB5m6STVk5Y8ByIOCUgM2aXVxnTl0F4ZYJIAFGAtyAALc123vvvWnatGlBiUJ4HH744UFCvNdee9Ho0aNpwYIFQX0wH8OGDaMJEyagC4Tli7PWxdmsJ7Qpq9b9Avurh4Dt68K2vOohDUt8JIAEGAlwAwIbNmygAw88kD7++OOgty/X5/7+97+nG2+8kV599dWgLzD3AeYX4i6//HKaPn06nXvuuUFSPHjwYHGMI/DEqGp64DfrN9C4WctosYUuEDZl1bRTYHzVEHCxY4vrrGrCo+oNQR6CBLgRAf76G7c9e+aZZ4Ivwe2yyy7085//nA444IBgLHd74K+/zZ49m7beemu65JJL6LzzztO6WBB4WrgwGARAAASsE0DNrnWkEFgiAshDkAAXEq4vIMOnAAAgAElEQVRFB57NurdCAGJREAABEMhIwMUOcEaVMB0EciNQdB6Sm6EGCzWpq6urM5iHKQICRQYe6tQEDsIQEACBqieAe2HVuxgGphAoMg/x3TFIgB16qMjAw66HQ8dCNAiAQKkIoGa3VO6CshYJFJmHWDTDiSgkwE6w/kdokYGHujeHjoVoEAABEAABECgBgSLzEN/xIAF26KEiA69ad4BR12wesGBnzg4zQQAEQKCMBIrMQ3znhQTYoYeKDLxqrHurRpschl8D0WCXF2msAwIgAAL+ECgyD/GHQrwmSIAdeqjowKu2urdq3dV2GIL1osEuD8pYAwRAAAT8IlB0HuIXjYbaIAF26B0Enl24qGs25wl25uwwEwRAAATKSgB5SLLnkAA7jGoEnl24tbKL6aJWt1bY2Yw4F36wqR9kgQAIgICKAPIQJMCqGHFyHoFnF2st1LG6stGVXLse9kcaePnjC2gCAiBgTgB5CBJg8+jJMBOBlwFewtRqq2uuNNPlTm21s7MZbS79YFNPyAIBEACBNALIQ5AAF3KFIPAKwV7qRVGr64f74Ac//AAtQAAEshFAHoIEOFsEGc5G4BmCq4FpSfWlRew8SmtdpeOqwX1F+KEauMEGELBNoJbuO7bZsTzkIUiAXcSVUiYCT4moJgek1ZfW1RGdfu80mrpwZT2bvbpvTmPOGkgtmjW1zkta6yodZ13BggTWmr0FYcayIJBKANdh9gBBHoIEOHsUGUhA4BlAq4Epqt3FPGt1VbqE7pCOqyb35emHauIGW0DAFoFavO/YYhfKQR6CBNh2TInkIfBEmGpukE/1pVJdpONqzpkwGARAwBkB3Heyo0UeggQ4exQZSEDgGUCrgSl57mqo6uekuiSNu2FQHzplr6414DWYCAIgkDcB6f0pb73KtB7yECTAhcQrAq8Q7N4vmlddm2QdyRgGyuNOvWcqTV/0aQO+e3brQGPP3tNJfbL3joSCIAACTglI709OlSi5cOQhSIALCWEEXiHYS7FoHvWl0t0TqS5jpy6mEU/ObcT3pmN3oZMGdikFdygJAiBQLgLS+1O5rMpPW+QhSIDzi7bISgi8QrBj0f8lYLt+zrY8OAoEQAAEQMAtAeQhSIDdRliCdASeW+yq+la3q/svXboDHLUkjim3Zhv3+jJ6bu6HNHn+x6IdYPjGTnyAox2OkFJOAoj/7H5DHoIEOHsUGUhA4BlAE05BbZgalC6juPEDu3UIFpoe6UscXTmuR7HuumpLanMEONam32H1fwgg/u1EAvIQJMB2IklTCgJPE5jGcJPdTQ3xVTNUp34uiWkcjAN7dqLD+2xF39+tc6MX4OAbO+EDjnY4Qko5CSD+7fgNeQgSYDuRpCkFgacJTGM46lE1YAmHJjGNm37ugT1o6OG9YiXDN0LgimHgaIcjpJSTAOLfjt+QhyABthNJmlJ8Crxqq6XC7kB6MJr4W2cHOK3zA3yjeaNIGA6OdjhCSjkJIP7t+M2nPMSORfakNKmr41dccLgg4EvgVWMtVTXaZCsGTdlIa4Dj6n6jupuub8v+apEDjtXiSdhhQgDxb0Kt8Rxf8hA71tiVggTYLs8G0nwJvGp9ktapb3XoZu9EZ/F3HFM2cNysZbR45Wrq2qF1bN1vJQT4xk5YgKMdjpBSTgKI/+x+8yUPyW6JfQlIgO0zrZfoS+Chlsqhkz0UDX976BSoBAIgAAIFEPAlDynAdOWSSICViMwH+BJ4WXYEza2vrZkmNbdxhKJytm3fioia0PLP1lD4vxd+8hW9vvhT4rql7h3b0PXH9KHWGzVvIGrM3xbTz55K/2Kbrr6642vL+7AWBEAABPwk4Ese4iMdJMAOveJL4KGWyqGTLfarjPOTSvN2rZrT1OEH1yfBLOPUe6bS9EWfNpi6Z7cONPbsPYOWZbrxoDtepTPOgwAIgAAI5EPAlzwkH2v1VkECrMdLa7RPgYdaKi3XaQ22tcOu04UhquCxu25Lt53YP/hTkowbBvWhU/bqmjomqbODLfu0oGIwCIAACIBAZgI+5SGZjbEsAAmwZaBRcQg8h3A9Em2r5lanD2/U/F27tKcJQ/YN/iTRRTImKl93vEeugSogAAIgUNMEkIckux8JsMNLw4fAi6vd5MZ3415fRkuEb/Wj/jM9SGztkLrcAb5+UG9q1rRp4PMVn6+l8bOXNzIqukscPZlm37EDOtOjM5fQi/M+oiZEdOjOW9IJe2zX6OtwWS4zxF8WepgLAiBQywR8yEN85Y8E2KFnig48G31dUf+pDhBbjGzVAJ9+7zSaunBlveIDu3UI/vf0yN82admMvvx6fQPjonXC0RNJ9t07eHc68/7pqfXGanrpI2yxzaoH5oMACIBAGQkUnYf4zAwJsEPvFB14OjuKqP/MFgi2aqyjcj74bA09Oef9Roptt9nGtPkmLRO7QFTqsm5DHY14snFXiDiLk+Igzr4nZi2jYePfigWX9qU4HdK2dtd11sRYEAABEKgWAkXnIT5zRALs0DtFB55OTem5B/agoYf3akQD9Z8OA0Qh2hZ7G3EQp2qa3KR40qVpi4HuuhgPAiAAAtVAoOg8xGeGSIAdeqfowHO5A8ydB0Ye19dqrWecK7LWf2adrxMevBbXw77w9oqgznarTVvRwTttEdTeci9f6VfUwjWT/Hdgz050eJ+tRF9kY1k24iCOQ5rcat4BzjOmdOIPY0EABJIJ1Op1W3Qe4nNMIgF26J2iA89VDXCIbK/um9OYswY6S4Kz1n9mna8TGkn9dytl6DBT1QRLZdmIg6SHE1XPYR2GSWtU1jRL7c66dhn0cWEjZIJAtRHI898C39gVnYf4xiOqDxJgh97xIfDiajfZ5HGzltFiYRcIljH0iTdjOwfY2unT2WGUrpln/airXdbQf8/O/ZAmz/+4ESYpCxtxEOcjlvvYjKX0wrwVzrpA2KqvtnGp5xlTNvSFDBAAgeRfwaT3zzIz9CEP8ZUfEmCHnqmmwCuiFjPrmlnn64SGqzrbUIc8bdGxu9bGwg+15nHYWw0Eavm6raY8xHYsepMAb9iwgR5++GF69dVX6d///jfVcbPayHHffffZtt25vGoKvCJ2vrKumXW+ToDY3gGurFdL6uQg2cFQ1b6pzutwSBsbrrPwk6/oo8/X0srV/6amTZo46R1sS+dKOXnGlCsbqlFuWgznFd/VyNWlTXn6xfS6zVNHV6yrKQ+xzcibBPjiiy+mO+64g/r160ebbrppIztfeukl27Y7l1dNgVdEDVXWNbPO1wkQmzXANmt2VQxU53UYqJLfylre6PikHsS21rclJy9etvStBTlpPuF9FJ9qyGvBHxIb876OTNYzmSOxPe8x1ZSH2GbnTQLcqVMnuuaaa2jIkCG2bSxMXrUFXhG1mFnXzDpfJ3h4rcsefyO2d69O54ak3Qr+Uluzpk3Etdusu2rnQ3Vex/60sZIdcsluti19ssjJM6ay6Fkrc9NimH9HjOtVXZZYq1Yf5nXfifLTvW6L0NGFv6stD7HJyJsEuG3btjRnzhzq0aOHTfsKlYXAKxR/IYvbqDWzISM0XiVLdd4WREmNtK3ewbZ0hpxyEEiLYbZg1MsLGhmCWCvWt3ndd7JYWQYdJfYhD0mm5E0CfNxxx9FBBx1EP/nJTyQ+LcUYBF4p3GRVSRu7BjZkhEapZKnO24JTTTvAtphAjh0C2AG2wzFPKXndd7LYVAYdJfYhDylBAnzzzTfT1VdfTYcffjjttNNO1LJlywZaX3nllRJfezWmyMCrhuJ9r5wZo0wcY1XNocQvWWvPomts274VTXzjA5q+cGW9BdE+ulnXkvpI1dO4LDXAUnvLMk4SjzZscblOUTXALm0Kmeexhg3/6sr4cu06OurOV4lfiA2PIvt7x+mf171Rl53u+CLzEF1d8x7vzQ5wt27dEm1v0qQJvffee3mzybxeUYFXLRduZgc4FJDGmJeN67Os4xfderXoP5iVL/0M7NaBvtdvG1qW8DU607V08YbrvMddIFatpZVfla8LhK7NPo/XiccsduSxTloMu4jvPGzKY40sfjWdG2dX945t6JkL9qPWGzU3FetknovYcaJoitCi8pC87TRZz5sE2ER53+cUFXjV8tONz/41YWwyR5dBHmvo6oTxfhLIK1byWidPynnYlMcaeTIL16pWu4pgKVmzqDxEolvRY7xLgJ9//nl68803qUWLFtS7d2/6r//6L2rWrFnRnIzWLyrwqqV43wh6TpNMGJvM0TUnjzV0dcJ4PwnkFSt5rZMn5TxsymONPJmFa1WrXUWwlKxZVB4i0a3oMd4kwJ999hl95zvfoZkzZ1L79u2JP4yxatUq2m233ejPf/5z8LeyHUUF3pi/LaafPTW3ES7XrX98r1ezqZ90FyO65vufraan5nzQyC+qFmk6ekv1yuNa0tE7D32wRkMCecVKXuuY+tckTvOwKY81TJllmVetdmVh4nJuUXmIS5tsyfYmAT777LNp2rRpwdfg+vTpE9j3xhtv0Kmnnkr77bcfjRo1ypbNuckpIvD4Zn7qPVNp+qJPG9jp+iUj3+vVbOsnkad68asyEONeApGsE5WjO97VxeCLHq7sqwa5efkor3VMfGKqG7/EtfdNk+iLtevql23XqjlNHX6wtTpWU91MOOQ5p1rtypOhzlpF5CE6+hU51psEmD+EMW7cONp///0b8Jg8eTKdeOKJ9OGHHxbJyWjtIgIv7SMKp+zV1cgOySTfn+pd6Kd6QULS+quSbeUuvYneKr0k/sw6xkTvrGtivj6BvGIlr3V0CZjGqek8Xf185aZrR+X4arUrKxcX84vIQ1zY4UKmNwkwlzhMnTqVevXq1cDOd955hwYMGECrV692Yb9TmUUEXlH1VUWtK3VgEfpJPv5QqX9lg/4i9JYyTRtXVr1t2A4Z5SFgGqem88pDBppWC4Ei8pCysPMmAT744IOD0oc77rijAbvzzz+fZs2aRa+99lpZmNbrWUTgqXYmTOrdJOBV60pkuBwj0c82mzx2gE105jmPzlxCL877iJoQ0aE7b0kn7LEdtWjW1NgFlXqs21BHI550U4duYrOxYRkmJunpSn9XcjMgcDLVpp2S+0LUiHDt5+Z+SJPnf9zIvmN33ZZGHtc307WUBi1qO/f4JmpCyxPaG9rkFMdgycrV1LVDa/r+bp2d2Ruu68oWJwHqmdAi8hDPECSq400C/Le//S34Ely/fv1o3333Je79O2XKlODzyM8++yxxgly2o4jAS6uvUn2kIQtf3+u6VPqpzpuwUdUAt23VvEENoW4NsIk/XdSIx9nJvYf5SPoAhwlPnuPCT6a6qBKVyn7M7N97B+9OZz0wg6YmfJjEVJeycDG1L5oIxXEdc9ZAoyRMh5vqeg51dPVBB9X6eXzgRodXVl+78rktvcoip4g8pCxsvEmAg38sp0+nW2+9lebOnUt1dXW0yy670E9/+lPaa6+9ysKzgZ5FBV5SfZXubocudN/rutL0c8Umumbn9hsHSMMPUhzdfxuaOOd9WqzYSbHpz7RdadMuIWl1582aNlHapxNnrvyko4NkbJKevEM4fvbyRiJM2YeCysJFwi5tjAs7pfctnV90svozjoFk/XBdF5xYJ1dy8/Z51jgs0/yi8pAyMPIqAS4DMB0dfQs81K0le6+MbEx0TqtLrqw/lsa6iR5S2ZXj8lzLVEeel6Tnrl3a0+wlnzUSbco+FFQWLlmYpnHNyk+il05Nvwt9JOuH67qKB1dy0/gXsaYkHsoyxrc8xCduhSbAP/zhD4Oa37Zt2xL/77Tjvvvu84mbSJc8A09SI1XE07sIlAeD8mYj8VeIJWls2i5jUh1injvA0XpIW/WwRfW4TgvRuJrqOqLYGmibO8BRpis+X+tkZ5nt1olV15dyXtdpnM1PzFpGw8a/JTKxjDvAEj/nxT8KWbqmRH+R86psUJ55SNnQFZoAc83vhAkTgo9c8P9OO1566aWysaW8Ak9alyUdVzrQFhTOk43OWro13SGKpDrEvGqAo3ok1b3q1sO60D1r6CTptMf2mwXvMVTWQOvanKRfXFxIasp17dWJVV3ZJuPz0CdpjTjfuap1j2PjsgZYylU6zsS3OrFeeX8rQi+bNrqUlVce4tIGV7ILTYBdGeWL3LwCT/qEzFyk9W6+MMxTj7zY6PhLNZZ1HvrEm1q7fzznsRlL6YV5K6x1gUjTI2nXU3c3NK3W2GWP67QYTNtRv2FQH4qrgbYRZ2m7/1tu2sra2/mq+Mvz+gzXssHPxKe8q8sdD8bNWtagrp1lVf4tS0eVNN3S3imo7Magw0nHzzpybcWHak0d/W3pVBY5eeUhZeER1dObBPiVV16J5ce7KBtttBF17tyZtt1221IxzivwUCNVqrBIrA+NqxuU+FYyJg9CunWvuvWwvtgZZemiplriq7xY5LWOxOa8xsDm/yPtopbZhR9r0WdSjnnlIVJ9fBrnTQLcokUL2rBhQ8CGO0Dwwclv9DjwwAODr8VxyUQZjrwCr1qffqu1pivNX8cO6EzjXl9GYY9NST9dX/yv2/kgaQd4UP9t6Jbj+9W3tQrjIKn3Ku/MVXLLozcp34Nc1FRL7m15+TyvdSQ25zGGY23ouDdpguVOHb7fy8ru57Lr7zK288pDXNrgSrY3CfBDDz1Ew4cPpzvvvJO+/e1vB/byl+HOO+88Ovfcc4NWaBdffDH179+fRo8e7YqHVbl5BV411j9Vo01hcNmuMfxy7Tra+6ZJDXoKt2vVnKYOP5hab9TcakynCdOxK+yJe+b902n6ok8bid2zWwcae/aeFNfrODrYZW9dCbii6pLzuj7yWkfC2vWYtBrbLL19y8CwDDqa3HtM+0O7jrU85eeVh+Rpk621vEmAd9hhB7rrrrvosMMOa2DbpEmT6JxzzqF3332X+GMZxx57LH3wwQe27HcqJ8/AU9VIOTXUgfBqf6KP81fSW+ZJtaQhdp9YJcVh0t/HTl0c2y2BbeOdXf4tKO7N+wN7dqLD+2wV1GQmcXPxJn5cqLuoqZZcUnld83mtI7HZ5RiTrioSfXy6PtP0Lbufy66/JJZMxuSZh5joV+QcbxLgNm3aBJ887tWrVwMe77zzDu266660Zs0aWrJkSXB+9erVRTITr43AE6NqNLAWa7pMbTadZ+4dezNVNbS80qiXFzRaMFqbWGb77ZGEpKwEXMWRK7lZ7cX82iCAPCTZz94kwAcccAB1796d7rnnHmrWrFmg8fr16+nss8+mv//970E5xJgxY+imm26it99+uxSRi8Azd5PrXRMfa/JMe9zaYGXCw2ROZUSoamiTdoCju7s27DeP1PLNtOE3ldW21shTjq04qtQ5qY4/+iuGq64RKj/hfPUTQB5SggSYd38PPvhg2myzzWi33XYLXojjv33++ef03HPPBf+fX4L79a9/TT/+8Y9LEbUIPHM3uaxJcynb1OIstaRZ7TGZbzInjo3K7rgaYPQANY2y/3zU4vR7p9HUhSvrhWSpb03yqY01bOkqlSMdl0Y/TkZcr+CoDNv8zaMDM6uRAPKQEiTArCLX9vILbrNnz6bmzZtTv379aMiQIdSpU6dgF3j58uV0yCGHlCZGEXjZXOWqpsvWTk826xrOztrjNgsrEx4mc5J4qWpoJbZJxtj0V1ll2fRbEgNbaxQhJ2scpV3H3BP62bkf0uT5HzdCl1e9elnjFnqbE0AeUpIEOFTz66+/Dnr/VrZBMw+BYmYi8IrhrlrVx5q8InUyWdtkjsovOO+eQB5+s7WGb3Ik3lHprDovWQNjQECHAPKQkiTAvPt78803By+7zZ8/n2655Rbaeuut6corr9TxtzdjqynwbNXi+eAcWztLbIstLjZ10mVssrbJHF29TMfb8onu+nHrcglHtK9zlv7ENuxK8ltcParperZiI+uvIqH+Sf2juQ/1yOP61veb1vV33HiV7dJOE6z7ozOX0IvzPrL2tUbV/Srq723bt+JO/EE/8k+++Jo6tW1J3Tq2CbquoF45e6SYXlsmK1dTHmJif9ocb16C4z7AXO5w0UUXBUnw3Llz6ZlnnqGhQ4fSNddcQ5dffrlt2+n3v/998FLde++9Rz169KCrr76ajj/++GCdOXPm0IUXXkgzZ86kzTffnC644AK69NJLtXSolsCzURunBc7xYFv22JIT/sNko27SBJ2JHSZzTHTTnVOUXtLaT9N6T1t2xcmJMg71k9ReJ/nGpq6n3jO1UZ/osEd0WiKmsjPU3dQfpran6RVlb2p32vWS5hdVv21XvHSv72oYb+v6kLKoljxEaq/OOG8S4AEDBgQJ5+DBg6lt27b0xhtvBF0hHnzwQbr++uuDPsA2j7Fjx9IZZ5xBt956K333u9+lhx9+OEiAX331VerZs2fQbm3QoEHBxze4AwUn53fffTedeeaZYjWqJfBUuxpiIB4NzFrrx6bY5mJDJ1PEJmubzDHVTzrPtk+yrhs336Te06Zdod/S6lEl3TfS2NiKjaQ+0SqGad1FKvVWyZLGQDhOZTufH/rEmzQ+4WtzSexZfhZd02IobU3XvHT5ln28zWtZwqJa8hCJrbpjvEmAuQ8w7/p269atQQK8cOFC2nnnnYM+wLYO/tQyJ9fHHXdcUGYRHvwRDu40wQd/lGPRokXBy3h88Ffqxo8fT9yXWHpUS+Chbi3e4+AivRLyG1eUT9L6GVdaH+1hLCXjwq40mayXqv+yVPcs40ztdu2PLDbxXBP2PM8kdkJdTde0Eb9ZeVXTfNOYNmVQLXmIqf1p87xJgLkEgT+DfMQRRzRIgHmn9qqrrqIFCxo3wzcFwknsTjvtRK+//nrwkY3K48gjj6T27dsTl2WEx4svvkiHHnoorVixgrbYYgvR0tUSeHk/sYrgejCoFrnkWbtm4uKifGJ7x7GSc1IvWdWOYJK/+O9Dx71JEzR3IVXrmfgsbY6pP5N6asetxV9aPH737azUakuvD9Pd2Dj+rtesZJYlBqS68po6Y23FXR5rmsa0qY3VkoeY2l+KBJjrfkeNGhWUJJx++un0+OOPBzuwI0aMCMoQrrjiCmv2T5w4kb73ve/RlClT6LrrrgvarvHOM6911FFHUd++fYNEfOTIkfVrzps3j3r37k0zZsyg3XffXaRLtQRe3jVLIrgeDKo1LmWwtygdbdac2qonTmJx7+Dd6awHZjToBRxeTjZqgG1emib+5DlxNbR7bL9Z8GntmYs+baAi/507Dk3P2BtZR1dVPa60BtjmmpXvIMT5MUvNtC1dXb2Ep6NflhjPa51Qx2rJQ7IwT5rrzQ4wK8hlBrfffjutXbs20JfLD84555zgb02bNrVmP+8qn3baaUHSy7vL3G943LhxdMMNN9Cf//zn4EMbJ598Ml177bX1a4YvynHSvN9++8XqsmrVKuL/hAf3NR44cCAtXbqUOnfubE3/IgSp6tqK0MmHNWuJS947F6b+LconabWdOl/9UvWSXbxyNXXt0Fr5Rn5ax4G4+tPKrghFcaz0u64eafw4AR7x5FxRaOnudOpeH2l2qXpjhwbYXnPcrGXE8dW5/cbBEvy/bXWB0NFVZ6zImYJBea6pG9MC9ROHIAFOpudNAjx58mTaZ5996JtvviHebeUvv/GLaO3atcvi+9i5jz32GJ144onBS23nnntu/Rh+GY4PbsPGZRBxO8BJZRM8j1+i444VlUc1JMDWnQCBpSOQd+1a6QApajuHHt5LZJItzklydu3SnmYv+ayRLlnqS0WG5TSoqFpXW37TwVTEmjr6Rcfq6Koz1lSfynlFrGlL9zQ5SIBLkADz196ef/554m4Qro/XXnuN9t1336AEIrqby63WuPXa9ttvH7Q+GzNmTL0qkhrgat4BVvkkj9oplQ62ztuyJSon7Ku5/LM1ot07HVts6ataM88dkjhd8rJTxSF63latblRmUv2qrR1J3ulN6kBw0sAuOuYrx+r6THd8nAJp/HS6HXBt8Cl7dW2wRJp+kuujcv5R/bahiW+8H/TbTdvVT1pXtWY4b+EnX1nZyTXxTzgnqR9zXFyr7FIGnsGAItY0UFN7ChLgEiTAXF/LL8EddNBB2g7WncAdJbbccsvgAxvR3r5c99uiRYtgJ5o/ysEv3jVr1iwQP2zYMJowYUJNdoFQ8c27pkmlT5bztmxR1YRmqaWrTMDy6h9si42Jf4pcO0lfW7W6lf6U1n+qOOrUANuKxyyxacPHSfW/Ye/guH63A7t1oA11dY1qgyv7Dav0MznftlVz+mLtunpscX5Q1QsnXf9pvX1N/K2yL+mhNa22OEkPk7VU14PqfBFrqnSycR4JcAkSYH7RjZNOLj3YcccdaeON/1ODFB62vwbHvYX5xbvf/OY3QZ3uI488EiTEkyZNCtqucfnF0UcfHXyAY/r06UGpBOvHfYqlR60EXjU9OduyRdIVQHdHLy7ubOkrjek8a9eiOuVtp4SHrVpdiZ1xu5ESHZP8lYcfdX2mO17neojyi7P90RlLY2uDo9eoRL80rpJ7AttUeV9QrZu0pmo93fuPSg8df0hq4vOI0Uqdi1hTch1nGVMreYgJI29qgPmFtKSD39Dll9BsH7fddluw67x8+fKgLRrX73J3CD642wN//Y07RPDnmC+55BI677zztFSolcCrptopW7ZI+pDaqLm0pa9WYBcw2Ec7XejkQmYB7gqW1LVFd3ycXaYyJPMkY9JYS+4JPL/yvmC6rmo93fuPiR4mc4qK12pdt1byEBP/eZMAmyjv+5xaCDz+2Sitn6jtmkIdn0vr1aLjVny+1kp9pGr3he1Q7YJI9DfZldFh6MtY13aa1Ga60MmFzKgPJTFV6XPVHN361OsH9aZmTZs2qnu1YbtURqjzPz/6kl5f/Cl98uXXtPTTxh9biuoquTeY1AhX8q7sxiG1qdLPSfflcFzSDrCuP9N2kk10T7rnqFZ8RmkAACAASURBVOIw7V6VZa4v90BTPWohDzFlgwTYlJxgXrUHXlqdq0mNmQCpeIi0nitunKQuT6WIqgY4Ol+37i/aB1Nqp0pf38+7tNM0Blzo5EJm6FsT2ao5uvWpXG/LR1zP3biaVd37iEpfXlt6bcbpmnZvUK0tibPQV1G7VXLjHlhUPX1Nam9N/KOre1rya/qugy0dfL9HJulX7XlIFr8gAc5CTzG32gMv6em+cgfDIeJE0dKdhzQbtty0VaaODdF6srCv5gvzVtDk+R830lu37i8qoBrr1uIc68pOyW496xO30+VCJxcyWX/pNRFlr5qjOl9pi+qLdjZsV8lQ+bvnlpvQD/ftRkm68v0t7t6gYsFcK3U7uv82NGLCXOWvTiqbJD7bdbv2xEl9t45tEntIq2zQ0SPUyWRO5fWv0ivt35gsc4v4t8v2mtWeh2ThhQQ4C70aT4B9ru+S6iYdZytMpOtJx9nSq5blqGolQza6NZO+MTWJKdUc1flKBrrjXTBU+Zv7JE8Ysm9uNcy2mWSRl2WuC1+FMrPolWWuS5vyko0EOJm0NwlwXV1d8DnKajp8DjwbNVHS3VP+6Wzc68uUvS5t+l711B/a/6e3PqAp736i3JHV1S1rHZ1Kf119qnF8HOO0WNP1SSWzyh1gG9eQiV9M1zWJKdUc1XmTnTxT+6QsVTvAvMN724n9Sbcfsy6LUN+keaadP6T35bhPCpvaIGVvOi5Nr2MHdE7990XlR9fxFrU5z7XCdX3OQ0zjwdY8bxLg3Xffne677z7q27evLdsKl+Nr4NmqiZLUtKXV/Ln6pjs7Xrc2MRoslf0/dQNJd+0sNcC6ulXL+DjGpvWlcbWNzZoQreevJvzv0a5Vc5o6/GBqvVHz4C+2riFdf2RZ12Suao7qfKV9qvGq87q84san1QCHfm7etCnp9mM21Z3n6a6VxkFyXzapAXZ5v1b5NYntvYN3p7MemEFTF66sF1FZP53G1qSuWaVr0nnT+DBdDwmwmpw3CXDHjh1p2rRp1KNHD7XWJRnhawJs8yk/Wt+V9JZ0nLt0e1CauNy0P6bpzkuoo4qvtCZOOs6ETdnnqHbxovZxrCV9ASyMQ0kc6/aEdcFYFVuqNU1iSjVHdb5SJ5NeubbvF6EO3AVi1uJPg/jo3rENXX9Mn+Ahx3RXVpdFyGbs1MXKPsQq30bP68Zz0ty0r9Pp6GNjbBzbJ2Yto2Hj32okPowXlR+zXk86duW5VlQvX/MQHXauxnqTAN9yyy30xz/+kS677DLaYYcdGn0Io0sXu5/odAW0DIHnqiZKVVsXZVNkPaVKz6y6ueKbR8yWZQ2VDytjjf//qJcXNDIvztcS/0nGuGBZ1LoubImT6Yt9eevhcj2XsvOKi6R1VLZlPW/TPpUuNtcqQx7iyl4dud4kwE2bNq3XO1oLHNYGr1+/XscuL8b6+uTl6klUd1euqB7BKj2zdrHQ5atbF6Y73ouLwbISSXV9cctIdoCj8yT+k4xJM1nXh+H45+Z+GNtFJKkzgQ52XZ10ZEvHZuUqXUc1zpYeUqYu11PtkqpYuDwv4ZM2RsUt63mbtqt0sbkWEmAZTW8S4MmTJ6dqfMABB8gs8miUrwmwq1ok3brMomrK0moAw/DR7T8aDTsdvjpjeQ3d8R5dDtZUSaqZ3GP7zYIXabP2mJUwloxJMlh3ripeXfWtznINmDpbl43pOqp5NvTQkaEzVjeuVHWyKhauzktsVo1xfd6m7Spdba6FBFhG05sEOKru119/TS1btpRZ4PEoXxNgRmZaq6bCHSeX54ybtYwWr1ydqa+uam2d86Ge733yFc1YuJJmL/2s0fQsdYdSvrq7ArrjdZiUZWxaXd8Je2yXGGtSn0ivDx15Uba6Pkwaz18S7NBmI2UPWYlfdXWSyDQdY8rVdL2keVn10GXqcr3v79bZu3uwhI9kjIpb1vM240qli821Qlk+5yEu7NWR6VUCPHr0aBo5ciQtXbqU5s+fT1wXvPXWW9OVV16pY5M3YxF43rgiVZGiarNYKd21dceXwwN6Wpadga7+aeOZnLS2OY2yrk56HqvN0XkzzXu9rF6V6CsZk1WPap+PPCTZw94kwA899BANGTKELrroIrr55ptp7ty59Mwzz9DQoUPpmmuuocsvv7x0cVpU4EnqqhimdJzuWNuO0tHTZG3JLoOJXMkc3bV1x0t08HlMnO99qGnMEpO6Pkwaz3W/A7puFts9gHeHD++zVYMvfpnUUu6/Y0fquElL6tS2ZeoXxKIxJGEjGeNzXEp0S6pTv35Qb2rWtGl9X/Sj+m1DE994X9Qn3cSHOr9kZfWLZH44JqmeXbfbimRNlb9syFCtUdT5ovKQouzVWdebBHjAgAF04YUX0uDBg6lt27b0xhtvUPfu3enBBx+k66+/nt59910du7wYW0TgSeuMpOPC5Nf0O+xZHaGjp+laeayRpJvu2rrjTZn4MC/J1qJrGrP6QHd+Wg1wXO/jqO/COl5Vv9O0NeLkJdXvS2yTjPEh/rLowDbG9Z+Nq1OX1nCruKnOq+zJY74qzirrzlU6qc6rbC763zeJflnHFJGHZNU5r/neJMBt2rQJdn27devWIAFeuHAh7bzzzrRmzZq8mFhbp4jAk+4uSccxDJ2x1uD9r6C81i6iNitkpbu27njbPslLXprvi6xptBGTuj7k8UOfeDO23pf7Vjdr2oSeTegQIe2CEeqUJCf0e9qOooSNZExeMeZqnSQbB/Xfhp6c875y2TjGEm66cRVVRCI/TXHJ/LR69spfLCT3R8maKtg2ZKjWKPJ8EXlIkfbqrO1NAswfwLjzzjvpiCOOaJAAjx07lq666ipasKBxD08dQ4sYW0TgSWumpOOYm85Y25yLXNu2LZCnR8BX3xell2pdW7XCqh7LaX2yVToWfT/Ri0Dz0Ukcdu3SnmYvafzCbeVKpv2pzTXOfp8vwveSNVVMbMhQrVHk+SLykCLt1VnbmwSY635HjRpFt956K51++un0+OOP06JFi2jEiBF08cUX0xVXXKFjlxdjiwi8pLqzyppAnafetBrEkcf1pcqfQ6X1VJJxOnp64fQclZDwc6FO0rrh3xd+8hV98sXX9XWj0hrHSrnrNtRZ/TpWHAsVw7jzRdUgq66FtPOqL+FJdgLDMXH1xeE5yQ5fEr+sX2B0EeuhTFWcVK6dds8cP3u5UlWdHWBb3FTxVam0zvV67IDONO71ZSSp+1XCiQxI0zlcc4mi+5CO3bpxYBo/OgxUY4vIQ1Q6+XLemwSYgQwfPpxuv/12Wrt2bcCnefPmdM455wR/i34owxd4Kj3yDrykurOontKawOictLot3Zqt6A1BUldso8ZL5acyni+Ki05dbshVUuMYJzeuvtVmb1oVQx1bbeqVFI+m+o45ayCpaoCl13vcvST6APzl2nW0902T6Iu162LNYE5cw33m/dNp+qJPG4zZs1sHGnv2no0eqIu+PlXc4/TTiR3J9cFrJN3fbXHTsVPneo2r2VfFkdTnOpyTrlGp3dJxcQ8Kkn/rpDbrjss7D9HVr8jxXiXADGL16tU0b9482rBhA/Xq1YvatWtXJJ9Ma+cdeElPspVGhLsLOvViaTWIum/tsj46T906emZyWIkm6/CzaVbWna1Ql8odriS5YX2rix7SKoY+1iCrroW086q50TiJ9snm3fxPvvyaXnn3k0ahJPVjdCLPyWN331bcq+IkaZ0k3pV/P7r/NjRxzvuiPuljpy52+quINEZ0rtekHf+0XxJ0fBens+6vNBK7TePAdJ4Og7SxeechtvTOQ45XCfAXX3xBjzzyCL311lvUrFkz2m233ei4446jVq1a5cHC+hp5B56qdi80MK2GLw2CpFZKMobXkI6z7pQqEVgUv6y1jUkxWIQ9qjVV56sklERmSFlI7kF8/+HDRv9ikfIZB0ltz7iMaLovuujooTNWBEEwyMWapjJN5wnMFA3JOw8RKeXJIG8S4HfeeYcOOugg4iS4Z8+ewQ4wtz7bdttt6S9/+Qt17tzZE2RyNfIKvLAu6U9vfUBTYnZpKjWW9IXUrX3UrfEq+qk4ysS0rkseCfZHFsXP1g5wZS/UInYEVQxV55O8ahpPpvNsRJdqbSkLya9Qaf2Lk+5NKv1cXs9S2234QSXDRBcddqr1w/M6euiMla6vGmdrzSi7FZ+vTf3qYhJnW7qobE46n1ceYqpfkfO8SYAPPfRQatGiBf3hD3+gzTbbLGDyySef0Mknn0ybbLIJjR8/vkhORmvnEXhp9bmstLS+rPIfkLiapaT+qyY1Xqb1VEaOSJnkix66dhWlt07NXZJNcb1QXdf7xumiYqg6byIzLWkuqk5QYqdkDNumuh+F9uv4W7p20vpZ67N11te9jnXH6+qiO16qj47cuLrwdq2a09ThB1PrjZpLl9Qap6OfzjWZ9O9pWo29Tv29lpHCwXnkIUJVvBvmTQLMSe60adOod+/eDSDxBzG+/e1v06pVq7yDp1Ioj8BT7bhce3Rv2qh5U1F9WWiPbu2jaY2XpO5KxTjr+aKfzrPoXxS/tNrGpF61UTuTeqG6rPdN4qxiqDpfKdc0nkznZYkfyfV+0sAu9UtIWUTHdW6/Mc1YtDK2963U3zpsdMbqsJPariPTdKyOLq54sO5SPVzqkMZQql+SjLRfu7bctBV17dC6/kuLKhuz6mIaKzwvjzwki35FzvUmAeayh1//+td02GGHNeAxZcoUOuWUU2jJkiVFcjJaO4/AU9XcmdT76tYs6Y43guloUpl1d4Qkk1hVPLLwpF6oJrGaSVkHk03jyXSeDRNcr51Vvs58nbE22PkuwwcePuhg4icdvXXGmuiSZU4eeUgW/Yqc600C/PTTT9NFF11Et912Gx1wwAFBOcSMGTNoyJAhdPbZZwcvw4VHly7/tytRJDzV2nkEnmoHWFLvW2mH6mk263gVtzzP69qap25lXEsVj2wT14DG9UI1iVXfGJnGk+k8G/a7XjurfJ35OmNtsPNdhg88fNDBxE86euuMNdEly5w88pAs+hU515sEONrnt0mTJvVM6riAhoj4b/y/+b/Xr19fJDPx2nkEXlrNnWntm279FNd4HXXnq8QfQAgP07UlcCUvdUjG8Fq6tkr0K2qM1GaX+qlqQMM+sGc9MIOmLlxZr0q3jm3oOztvSfzf/Jnjyo+ruNTZpuw4+ztushEN2nVb2qHTJom2pcUh3wL5IwKqhv5pdqTFRtza27RvRT85cAc6YY/tRL7QlR/tR/7ozCX04ryPiO/6h+68ZaM1pdcoj2NZo15aQO9//p9e8nyk9X814Sq9zsJx4YdhOrTZiFZ+9e/6D8RI4zxpvUr5ndq2bHD9JPHQ9W3IMZQX5ytT//t8nUvjzvd/R/LIQ2zeQ/OU5U0CPHnyZLHdvENchiOvwKvs2Vl5IzRhJa1ZirtJdO/Yhp65YD8nLzhIbkqSMVEmUltNOOY1R9dml3pV1oDyWss+W9OgZi4as3+etyK3hyeXdoeyv/p6Hf33rxs+EKqSMT4fF4c2XqCRxAav/eiMJXS3MHmMcpTKHzdrWYN3Edi2U++ZKvoghuoa1U3iJTrHxYp0nupBMC0xl7BNe/E46SFzo2ZN6N/r/7OhJF0/mvwm+eq+M/agygfayocOlf/yuC5N1tDRW2esiS6mc/LKQ0z1K3KeNwlwkRBcrV0LgZf3Tz+S9SRjXPm8KLlltbmseqf5WVUGolPqYYOPVIZ0XKXttuexfB1GPF5XB93xoc3SeaoYCOWp7Ex7ESvtk8pJZUaVvlOtr7Kbz1dzSVNR93Ob69ZCHmLKCwmwKTnBvFoIvLyL/yXrScYI3FeqIWW1uax6pwWH6kVAnZf9bPCRypCOq7Td9jyWr8OIx+vqoDs+tFk6TxUDoTyVnUlykl4kDeWqzkvXV9nN56v5pdZS/SOQoGwt5CGmfkICbEpOMK8WAk+6IxLiktTPpY2RrCcZI3Bf4hBV7Z2JbAmXNLmmNmdd18TW6BxTvdPW9dWmUGfprhuPl/LJes3orMVjo+upPhCQ5Ku0XVLdD2Ikyar84ArX3XLpxdBxb9KE2csbqabyjdQf0h1g1eeAk+Tsul17mr30s8TLIM8d4CRdVCyz3jui821f87bl2bRVV1Yt5CG6TMLxSIBNyQnm1ULgSWviwn80Vc3+VfJU56XrCNwXOyStts/0xT+JTSp9TWSYzFHpoXvetg625enakxR/oRzdGJHYoxqjOh/qlmWc6Qd3pDXAqus6rlY67mMbcX/T8U0WRmmxlPaiXuU9M5RTyTxqR1yNsImPorER56skXXTj3OQ6041b6RpSH0vlFT2uFvIQU8ZIgE3JCebVSuBJi/8luyeSMZL1JGMELmw0RLWzY7LrIbFZoquuzbbWleiWNkZX7zRZvtn03idf0SdffK395n/URhUfic0qGeF6knFpdamVHwhQxQav99iMpfTCvBWJXSBCGSo7K3VP+rx2nE68YzryuL6ijhcSRrxGOO6PGT9Rz3KSPjDDends2zI2xir1PLr/NjRxzvtaH0WqjEP21YOvLaL5H33ZCCProut/VXxIzqviQiIjOsa2PN31bY+vlTzEhJtXCfBDDz1E+++/P3Xu3Jmuv/56euSRR2jfffelO+64g1q1amViX6FzEHgN8Uvq5yRjinSqqrZPVdMXp3tRNhe1rkv/VaNNKl5525z3eqH9uuuqrtUoV5PrVuUXld6V89N00LVdqpvJOJ90Yf1t62Nbngljm3OQhyTT9CYB5oT3hhtuoEmTJgW9fjnx5Q9gvPzyy3TkkUfS7bffbjMmcpFVhsDjn3tUfTgrYUXro7Ztzw8mTWh5RZurOMCSJ2tJLZ/OmmmONqnz8nkHWDeoJf7QlakzPo5/Us9bla/C88/N/ZAmz/+4kRppO/Mq2VFhOmOzXDc66+TtxzzWi7M/6ZPrSb5VXatR/0RlVK59VL9taOIb7xv3YZbqEe5Cx10DurZnvQ7T+vOm+f/YAZ0z96xO0j3pmrAVj6H8PyXs2KvqtXWY5zm2DHlInjyia3mTAHfv3p1GjhxJxx9/PF122WX017/+lV577TXiTyGfdNJJtHx54xcWioImXdf3wOMLXqcGj+1Oq4Hl82m1X5LaqrgxaXV7qjXTbqaqeuS4uUnMeOye3TrQ2LP3FP2MWplUmegijUPbDLKumxRHcX5O6mkajTPXMRnaK4lfHdbRsSp7sl5XNnxmg4NEjyTOo04ZQPvf8hJ9sXZdvZh2rZrT1OEHx/Ycl95LVOyz1M4mxXpS/a7pNSDhmnQ/0733JPknru7YVi1w2rXnqld2ElNbNpn6THee73mIrj02x3uTAHOJwz//+c+g/KF///50zDHH0FVXXUWLFy+mnXbaiVavXm3T7lxk+R54Jm9hS3Yz0nbbJPVzJrV8urW3WXYNxk5dTCOenNsohm4Y1IdO2aurUWxJuBgJVkwqal1JHIWqq/qMJsmS7NjoxIHO2ErsEnvDGDZZJ28/ulwvrcZY9xPacXqybyo/yhHueEr8xPN17zdx9bgjJsyN/SR43CXL63EHiyS9Te8NJrHGa8VxdblLrdIzazxK/R5y1vW/qX9szPM9D7Fho6kMbxLgHXbYgX77299S165daccddwx2gPfee28aO3YsXXfddfSPf/zD1MbC5vkeeGk1ckn1aJK6Otv1dC7WzFLnlWVuYcHo2cISn4Yqq/qMZvGHzlydsZW4JfaG102WdTxzs5E6ur1vbd5vJH5io2ysKV3L1npxzrAZazZlSa8fG37gtXR84dIfRheMYpLveYgLm6UyvUmAf/GLX9DPf/5z2njjjalTp040Z84cuvvuu4NyiGuuuYYuvfRSqU3ejPM98NKeepPeipY8Kdt+OtZdU1I/Kak17tqhdbDrUlkPp9qNSApAiV42g9dkPZM5aTrr1u3FyTLdAZbEYdpuY2VXAFO/s01j/raYfvZU418NovZm2QHOEnM2fJ4mQ1e+7g4w7/QfsvMWwbsIS1auztR1Q3KvYdaV9boLDbp9SNfi9SSxLL13RP3x/mer6ak5HzSaarJeUozryNK9X0hlJ8lVvTuQxNTk1z7d60DqT9U43/MQlf4uz3uTALORTz/9NL333nt06qmn0uabb05/+MMfaM2aNcHLcGU8fA+8tHpW5h1X65Sl3tLUhzprSus0TeoDQ/2la0TtNZljyovnmaxnMkeV/CbVF0r7tprWAEvr9NJiq1KGKR/VdVZ5rZmuU+kLiRzJGFUc2q7P1KkxVekmjYPotZ3W7za6nouewkk1wGPOGqj9bkEcG9W9lOeYvMuQFOM6smzHkep+HVezrIqn8LyOXab3Y6kuqnG+5yEq/V2e9yoBDg39+uuvqWXLli7tzkV2GQIv7MOZ1Nsx7gk7Wm/Vuf3GActlgi4QWaBL19TZpZPWGqsYJO0UR+3V0SsLp3CuyXomc9J0VcnTqc9U1fipzqfpmdZntdL3Juuodvrifm0xWafSRhV/Hi8Zo4rHNBl1RDRs/FuNRKh27pLsD//+bEK3jzhdVWtVzkmq8VdxqDwvWVfnGtBdP268KhZ5jsnuZpJcHVmqWDS9JnR/UYhy67nlJtRrq7Y08Y1su+Qq22z4NklGGfIQl/anyfYqAR49enTQCWLp0qU0f/58uuWWW2jrrbemK6+8sig+mdYtU+C5rN/KBFFzchY7ssxVqelSdtzaJuuZzEmz27Y8FeMs513qqqovtFXHWGm/xCbJGBXXNBk8d9TLCxqJyGqziml0Qd21dGSnsdFdV8XZxnmJbSZ6u46joYf3MjZft6a8MnZsxLANPqYAypSHmNpoOs+bBJg/gjFkyBC66KKL6Oabb6a5c+fSM888Q0OHDg1qgC+//HJTGwubV6bAS3tKNvm6D/+cpdtfuNJRJjVTWZ60s8xVBZlt2So2JjsytnW0LU/FOMv5NF2z9jZV7bpJdgpNbJPwz1ILH+qUJmPW4s9owuzGLSyz2iypqQ71011L5S+pL3TXjcpVXd9pOqTNldhmorck1lTc0v4Nitbj67LJsgPMLEx/xYjaa4OPil/S+TLlIaY2ms7zJgEeMGAAXXjhhTR48GBq27YtvfHGG8S9gR988MHgq3DvvvuuqY2FzStT4MXVX5n2v3RdD5bWpD1LTWOWuaogsylbIsvEBxK5Kjsr/xHX7TGqI9/mWJ26U5O60koOoe66snRslvgzSy18qItUhi2bJTXVWdaS2pO1BjjJlxK/mc5V1QCbxmMWndPiqNKPJj1/da7tKNeQhcmalf6xwUfn2o+OLVMeYmqj6TxvEuA2bdoEu77dunVrkAAvXLiQdt555+BluLIdZQu8aI3Vis/XxvaolOwOpO0ySOazn7M8MZvWivG6Weaq4tOWbCmbpFrGrH2aVXZGz9uyWWdN07Eue5uGst8z6BZgao80nrPUwoe6SWUkdZfRsTEp/gf134a2aNcqUxeIJHu4GwwflX14w7/Z9Kv0+o5jJpkrfZ9CxyfSWFPJVNXjm+7GqmrKF69cTWnvs9i4j9mQoeIXd75seYiJjaZzvEmAe/ToQXfeeScdccQRDRJg7gPMH8RYsKBxHZmp0XnNK3PgZalZSqszk9aXZVk/L/8WtY6UjXRcUXaUYd1aZGjDZhsykuLDpWwfYjKLfVnm+mA761BETbkvtrvQo8x5iAseUZneJMBc9ztq1Ci69dZb6fTTT6fHH3+cFi1aRCNGjKCLL76YrrjiCtcsrMsvS+DF1VQlfdVH8lavyx1g3kEyqUkOnatbP2Y9KCwIlOzy8DLScRZUqloRPjBMi9nwnEkf2iSnJdXX6lx7Um4m16O0VtRWUEp0lIyR6MNyho57U1w3HV132/ataMaiT+mpOe83Wkr6y1t0oi2bpHaPe31Z0Mc57ddH0x3gOB2y2Kc7V3e8hJl0TFnyEKk9Nsd5kwCzUcOHD6fbb7+d1q5dG9jYvHlzOuecc4K/NW3a1KbducgqQ+Cl1Uedef90mr7o0wasJP0PTepPKx0Sp5dpTXI0+S1LTWpagErryaTjcrkYSrpI0QzT1o+rTQwxZ6nljOuDq3vtSbhJxiQlLnnVVEt0lIyRhH9afa5JT/YssWDLJlO7k+LNRj0u65TFPt25uuMlzHTGlCEP0bHH5livEmA2bPXq1TRv3jzasGED9erVi9q1a2fT3lxllSHw0nZq1m2ooxFPNv56lWQ3Iewv/MK8FdSEiA7deUs6YY/ttJq526pJDp0u3ZXKNUgMF5PWk0nHGapRE9OKZJgWs0m7YaFTJNdppQMlXQKk8lXcslyPqlrRkwZ2sRKbEh0lYyTK6O5sS3xlWnNty6asdsf92qeKqyxrSq4ZXTa64yX664wpQx6iY4/NsV4lwLzzy+3Q+GU4/hDGLrvsQieccEKwE1zGowyBV5Z6Kxu1bTZklDEOoXN5CZhcn6G10nr7KB1Jn9gs8iVrSfXO43qWrCEZI4lAXTkSX0lZVuqnq4vEvqQxea4V6pBlTd25uuOzsIybW4Y8xLbNUnneJMDc7WG//fajVatWUc+ePWn9+vXBxzC6du1Kzz77LG2//fZSm7wZV4bAM9lhkjwlmzghrU4qy3fmQ7nPJXw9SmJPkTVcJixrYU6lT47qtw1NfOP9oI5Q8nW+SkaufWwi3+T6DO2SxHUlA8muYjjn+kG9qVnTpmLebH+0N3j71i3oyZR6VRWvPHbWJGtIxkiuR105El9J3tlg3SpZZ/n1L2qryoc8VtduCcs4m7iTR9hCU7Jmku6SuVEdk8brXj9SuyvHlSEPMbUt6zxvEuDvfOc71KpVKxozZgxtuummgV0ff/wxnXTSSdS6dWt6+umns9qa+/wyBJ5ujaFpbaEKvkqPuLpEaT1yUr0g6ySxp+gaLhW7Wjxvu0bctY9N5auuC9u1sHHrxfW7jftb2rXEcnVqiyW1PCbpJQAAIABJREFUnqZMda4XyRqSMZI1deWk1QyH65neI3X9G2ef1B7pOAnDcIxKZpbzktisfAiovE5t8JXyKEMeIrXF9jhvEmBOcmfOnBn0/I0ec+bMoX333Ze++uor27Y7l1eWwEurqbJRbyUBbbLTJdndSJJ7YM9OdHifrSi6K5Ckp+4Tv8RejMlGQLL7xStId0Fd+ziLfMn1abMPbdx6zDLaA1d3hzDNX3HdJaS88rg/SdaQjJFEvK6ctFrocD3VNZDEmu+vzZo2Ie6Ra/KLitSHrKeu3SqWkrXT1lTN19VX2idb5SuV3XHny5KHmNiWdY43CTCXPdx99910yCGHNLDppZdeojPOOIMWL16c1dbc5yPw5MhNah0l9W026q9syJCTwEgJAUn9I8uRxAiPc+1j1/IlzGyO0bUnzV9xPtKVb9O2sslSXQuqa8AVa1dyJf7JunbW+SodXcuPro88JNkbhSbAS5Ysqdds4sSJ9Itf/CL4GAbXAjdr1oxmzZpFP/rRj+hnP/sZnXbaaaqY8u68b4EXrWninpFETWj5Z2uMnu5NYKetn7ajlPS2u6SGSvUkL7HDhgzJOnmMkdTk5aFH1jWkO8CSGGFdXPvYtXwdnlliQFVPn9QrOM1fcbtePvHSYRuOzcJYsl5UflLf3FBO0q6iypfhPF1bwvF/eusDmvLuJ43McbHLWck96X0P3tU+fvftKOw5nLSznRR/Or2w07jlGd++5SGS+M5rTKEJMPf2bdKEm2T956jj4hpOyyr+xv+fX4or2+FT4KnqxSS1sFn4q9ZPq4mKq7mS1lCpar0kNtmQIVnH9ZhqsYM5SWqApTGSJM/mNeEL+yx6qK7htF7BSTXASTWqWfR0fR2p5LvWXRL7oY5JMazyZTjPRr1rlJekJlnFN+m8yiaet8f2mwX5xfSFK+vFSHss6/TCVsWA6rwpg7h5PuUhNu2yIavQBHjy5MliGw444ADxWF8G+hR4kh0zl0/mkvXTas6y1FDp1mvFxY8NGUXHZZ67DnnYWumTo/tvQxPnvF9fs6hbp+rax67lS5hniYG0evoObTai8bOXN1Ihek/R7Q3uAy8J08oxWRhL1kvbnezYtiV98sXX1KltS+rWsU3iOw7SdyN0bVHd5yXvbUgYxI1RrZ0mN+7fvix96CXc8opvn/IQU9+6mldoApxkFHd/aNGiBbVv396V3bnI9SnwVHViDERVK5YFmu3186yhymK3T3NrjVmt2SuJtSxMXNXpS/Qu05gsjCV22pAvlSEdF+qtus8X/W9MEl+VXrY4qNaR+F93jE95iK7ursd7lQDfdddddMMNN9CKFSsCu7faaiu67LLL6KKLLnLNwYl8nwJP8nRc+RSsW/sVQoyb98j0pfSzpxp/VS4KXuerRUn27LhFG+q9zaa0ZbtWqTsgThzuuVDJrkTUBFP/+4IhyV5pXXBeLPLkrBsDUQZp3QKaNm1Cw8a/1cj1Ln9VSrvfhP1e84xFaU1tkk7SOMjiw3BtqQzpOJXc8LyteIhj9cSsZbExKIkB1c607r1El5tER9MxPuUhpja4mudNAnzPPffQeeedF/xn//33D2p+X3nlFRo9ejRxYvzDH/7QFQNncn0KPFV9VGUdlGmNUtw8rsXcUFdHMxd9qmQtrbtU2RMuJJWnVKwKBuj4VGesr2iSYpH1VdUAVia/lX08bcVV3pyzrJdWx3vfGXvQWQ/MoKmK2krbsZLFHpu6qO5HqnjRsUNnbFqyLYlp3bXSOKgYSP2RpNO9g3dvFIOVdbtJ/xapapN17yW6tdNS203G+ZSHmOjvco43CXCvXr3owgsvpHPPPbeBvZz8chL81luNdxdcgrEh27fAi9YcdW6/cWDisoQuEKZPsJKd5pBt/+02pTlLP2+EWrpLIOmBycKl8mz43HcZ0rozU//7Zn+W2vHQFpcsXMpO8oU0BuLmj526mEY82fiXHL7GuKd2tFewpMd21ngpgl+cztKa2iR7de3I4sNQB6kM6bhKuTZ7U0e5pbGqjMHK9wL4/KMzlibG8EkDuySGpO69RJdb1mshab5veYgrO03kepMAb7zxxjR37lzq0aNHAzsWLFhAffr0oTVr1pjYV+icMgeebs1TCFpVAxZ1yK5d2tPsJZ818pFOnZRkPR15hQaMR4ub+t8jE2JVMbHLZI6Ug0vZUh10xvmmry/6ZNUj63wdH5Z9bFZWWeer/q3z7d+bMuchrmPVmwSYd4CvueYaOvHEExvY/PDDD9OwYcNo0aJFrllYl1/GwMtax6azA8w1v6o3x1VOkaxXrTvA0ppBFUOdHa2ys9TdaWM2JnOkzF3KDnWQxolkXB76JrGr1O+oftsEO3kTFN0npL7IMi4rl6zzdXVX+Vp1XrVe1vlp8rOyyjqfdWP7ho57Uxl7LjmofBCeL2MeIrUt6zhvEuBf/vKXwQtw1113XfDpY+7VN2XKFLryyivp/PPPp6uuuiqrrbnPL1vgZa1jC28MlbVlceC5HiyuZku3TiypLjFcU1de7kFiuKBubZ7uMq7l6+pja7yJXSZzpPq6lJ10PUr7nmYZJ7VfOi6OU2V9Z5HXfFY/Zp0v5SiJiay6ZJ2vsiWrfBfz42Iv6zoqDtLzZctDpHbZGOdNArxhwwa6+OKLadSoUbRu3brANv4aHH8J7le/+hXxRzPKdpQt8LLWsYX+CWufnp37IU2e/3Ejt0W7Pdiok4r2F+WPqXBP0mrvAmFjF0N1PdnwjWqNIs6b2GUyR2qbS9nSOJGOY5tc6pvETPJLD8/V6SQj9Y90XFYuWedL9VT5WnVetU7W+Sr5NmIwC+sk+ypjLw8OElZly0MkNtka400CzB/F2GeffWjt2rX0zjvvBF+F47KIdu3a2bI1dzllCzxbtVEhaNvycnegxwuCrcfO8Ug1aZxIxxVlmqTWn3Xzrf6yKF5p66p8rTqvsinrfJX8os9L7ZOOc21P2fIQ1zyi8r1JgDt16kTPP/88DRgwIE/7na7lW+Cp6pGSnlgreySq5IRQ056Ajx3QWfk9dlPnJOkn1TttXRsyTO2KzvNld8GGLVEZefHVWUdnrC6PLLIlc6VxIh2na5+t8dId4LLXqNvilSZH5WvVeZWOSfN1+m9LYlulh6vzUj42ONiwwbc8xIZNtmR4kwD37t2b7rzzTjrooINs2Va4HJ8CT1KPlFRPG+2RKJETgk8aa6P2N8m5LtfUsd118Pmkiy1b87JJZx2dsbocssiWzrU9TtdGW+MlNcDVWu9vi6HqvjzmrIHEHxCRxozOPZj77/Ih6b+ddX3bvCrlSfWLG6fDwZYdPuUhtmyyJcebBJjrf7nf75FHHkk77rgjcVu06MEvw5Xt8CnwpE+taX0+uUeiVE7oq7haq6Qv9tjYvUmrz3LVccKG3iaxnaWOzWQ913N0Y8tUH511dMbq6pNFts5caZxIx+naaWt8pX5xPV6L+AKcLfvylKPyteq8SlfdnrlReTqxrdLD1XkpnywcbOnuUx5iyyZbcrxJgLt165ZoE3eEeO+992zZnJscnwJPWo+kGqc6L4FrQ0bSOkmyXfYcRt2hxOvqMS7jIrq6zjo6Y9UWNhyRRXaWubp6YjwIZCWgE686Y7Pqlff8ImzzKQ/Jm7dqPW8SYJWiZTzvU+CpnqrDmqvnEjo3hLuc0jrhNH+pdMni61raAdblZLOuzlRW2jyXcWG6w+Syjj2LvSZzTX2WFmeVMrk378Q33qclK1dT1w6tg6/DVe7KxunBn44d9/qy2Hku9Na9dqp1vJRt2jiJDJ141Rlr4heJviZyJXNc2xang095iIRRnmO8SoC5FdqkSZOCzx5zC7TddtuN9ttvvzx5WF3Lp8BLq1uK+255FES0tk5SJ6yCKK2hUsmJO18rNcC6bGwyN5Wlmqc6r2tz0niddXyNJx0bmIPueAlrk7pc3brIuHsTan0l3lGPkcaE7r8dWftHS/VSW9h4hEvZEn2KWN+nPETCKM8x3iTAH3zwAR122GHB55Dbt28ftEH7/PPPg49iPPPMM7Tppps64zJ//vyg+wS/hHfGGWcE68yZM4cuvPBCmjlzJm2++eZ0wQUX0KWXXqqlg2+Bl1S3lPRUemDPTnR4n60a7eKo6oQlkKQ1VBJZlWOSZNtY04YME5uyzrG582AqSzIvL74667isY9fRQxrncbEiYa8bYyadGaRzWBf+1amOiIaNf6uRakXV3esy8nm8NCbSxun4RyfWdcbqMJbarCNTd6wr25L08C0P0eXlcrw3CfCxxx5LS5cupbFjx9K3vvWtwOa3336bTj/9dOrbty/df//9Tjh88803Qf9hTnR5DU6A//WvfwU9iAcNGhR8nGPq1Kk0ZMgQuvvuu+nMM88U61GWwNOtS9IdLwaGgc4I2PSZqSzTec6gZBBcNltc6GvSm1c6h13DtfV8jHp5QSNPoe4+Q/D+71RpTKSNK5t/pDZnp+uPhLLkIUUQ8yYB5g9evPTSS0HZQ/SYNm0aHXHEEbRy5UonfIYPHx4kuLx2mADfeOONdNddd9GiRYuoefPmwbo8bvz48cFHOqRHHoH35dp19LOn5tKif31F3TZvQ9cf04dab/QfnaWH7lOxjTpgqW6+jCuybswGA10fp61pKst0ng37bcvIwxbdWlkXPjORWTknuluLHWDbkagvL4wr1fseoWRbO8D6mqbPMLkn27xuTdZni0znmfLLIw8x1a3oed4kwF26dKFHH32U9t577wZMZs2aFbRGW7FihXVWr7zyCh1zzDFBuQOvHybAvB6XYTz00EP1a7744ot06KGHBnpsscUWIl1cBx4nv3vfNIm+WPufT0fz0a5Vc5o6/GCtJFi3LslGHbAIoCeDdPl4onYDNWzaYCrLdF6184yzT7dWVtX+ywV71AD7GJnqpPH0e6fR1IXxG0q69btF1WibxrPpvEqqpnJM52WJNNd5SBbdip7rTQL8wAMP0K233hqUQPTr1y/gsnDhQjrttNPo5JNPDkoQbB6fffZZsA7v9rJ8brUWJsBccsG7ziNHjqxfct68ecQf65gxYwbtvvvusaqsWrWK+D/hwXXNAwcODEo7OnfubFP9QNZPH51DE2YvbySXv0l+24n9tdbTrUuyUQespWCBg23uGhRoBun6OE1XU1mm84rklrS2S1t0d0q5R7fqcKGvSW/eOD1Y93GzltHimO4RLvRWsarW87rve4Qc0nxQhH+y3JNt6Gu6vum8LPGIBDiZnjcJ8HbbbUcfffQRrVu3jtq2bUstWrQIyh74ZThOTqPH+vXrs8RDMPcHP/hBIPuRRx4J/n80Ad5hhx2CpPjaa6+tX4f7EPfo0YOmTJmS2Jni6quvpmuuuaaRbq4S4GPu/ivNXvJZo/W45+2EIftmZpQmoJZqqWrJVqdBA+FiArq1skMP7yWWjYG1S6Ba7mVF22G6vum8LBGLBLgECfCDDz4o9vHgwYPFY+MGjhkzhoYNGxa0W9tss80aJcC77LJLUHYRtwP8+uuv06677hq7fpl3gHWBFvEkq6ujrfEqW/Ou6bJlVzXLqfRJZX9aSb/aIvm42AG2YU9esZ7XOjaYuJZhk4XqXhZni2ktuk29Q71065dd+cZ0J92Ef1YbkACXIAHO6mSd+QcddBC9+uqr1LJly/ppX331VfD/eZe3a9euQeszTpTDo5prgHXYRW9ElbVk1dqf00YfTBPGmGNGQFKb2rZV8wa1877Fru0aYDOSDWflVb+Y1zo2mLiWYZuFrjzTONRdR8IxTmZ0Xp7XsKkuLrio2CEBRgLcgMDy5ctpzZo1Df624447Bju+J554Ij388MM0evRoWrBgQfBBDj54x3jChAnedYFY/e91NGLCXFqYoQuE6gJKOm+jlsp07bzn6fZQRp/SvD30f+vp7J5GtfTNZ7q1sq6J57V7ldc6rnnZkO+Chc59W+daknT7yHKNme662vBDnIyQ47OKr6dWztXhb0N3JMBIgJVxFK0B5lpk7gN89NFH0+WXX07Tp0+nc889N0iKdcovEHhK7KUfUERNV+mhOTZAp342qgp6y6Y7Jq9Yz2sdx2FoRXzRLHSupej140JvFzJtOMlXvULbkIcgAVbGeTQB5sHc7YG//jZ79mzaeuut6ZJLLqHzzjtPKSc6AIGnhUtrsIv6skoFJGu42KHRAlGiwRKeNszR2bUqcgc4Lx42mLIM17HuS32nLV425LhmrtJR51qKfjn0iVnLEr/gd+yAzjTu9WW0JKbjR5o+RbMIdau8btdtqKMRT85tpHqW3W6VX3TOIw9BAqwTL9bGIvCsoWwgKI86Kuka0nFuSJRHap6cylADnCcPW1HiUmfTmkpbtvkqxyVzic3SGuCoLK7FvXfw7nTWAzMa9BtO+/uYswZSEb2sJQyiY6Q88qxHVtmAPKQkCTD32p07dy6tXbu2kcb8SeSyHQg8Nx7LYydAZ428a7rcUHUrVYenDU1U/WmP7r8NTZzzfmzfWRvrq2TkzUOlj/S8q1j3rb5TyiOPca6YS3VPq0VPq3/9/m6dG/V2TtsZLqqXtZQDj0uK0xsG9aFmTZsUdj9JswF5SAkS4Jtuuin43HDcweUJNnr/6gS6jbEIPBsUG8vIo+YqjzXc0PFTKng29At4gIefV6qeVrpxrDteTxv3o8uoP/KQEiTAW221FZ1zzjlBt4VoezL3Ie1uBQSeOdu0+sikp3D+At6Wm7airh1aE+8+qH5Si9MOdYjmPkubaWvHs2x1s0lMssawDxxs6mArPtxEL6TqxnFS/astP9uMPR3vqvQvSi/sAOt48f/GevMluA4dOgTdFvgrbNVyIAE286Sq7k1S42lSg4U6RDN/SWapfJqXDMk6eYzJEsM2WGa10bYOtuVltQ/zZQR0/aY7PmmToqge9Gn619URFaUXEmBZvFaO8iYBvuiii4Kd3+jX18xM8mcWEmAzX6iesllqtC5txedrafzs5Y0W030LF3WIZv6SzspayyiJC6kuPowzjWEfOLjQIWt8+ODTWtRB12+64yuZuog9Hb+VrSc88pBk73qTAL///vvUt29fatOmDXXr1o2aNm3aQOu//OUvOjHqxVgEnpkbdOusdMcnaWVLjpnVmKUiUM3+0bFNZ6yKqel5H3Qw1R3zyk3A19jzVS/kISVIgI888kiaNm0aHXLIIdS6detGGt9///2lu2oReGYu033C1x2fpJUtOWZWY5aKQDX7R8c2nbEqpqbnfdDBVHfMKzcBX2PPV72Qh5QgAeakd9KkSbT33nuX++qMaO868KQF99JxRYGv1O+oftvE9pBM6hVpo66MbY+Ts/WmLWnPbpvTlu1aUbeObYxfriuKbTWta+rnpPj36bqIs617xzb0zAX7UeuNmjdwoykHm7Hggw427YGs8hDwNfaS9OKeyBPfeF/7wx+2POI6D7GlZxFyvCmB6N69O02cOJH69OlTBAcna7oMPOlNQDrOCQCB0CT97jtjd60+rVnrykJVWc6jM5bQ3S8toPc/b9yP2uTlOgEGDBES0PVz2j9KcY36JQ35hapqD/vq63X0379+lRZ+8lX93KR40+WgrYxggg86CNTEkCok4GvsVeqlu5njwlUu8xAX+uYp05sE+LHHHqPRo0fTXXfdRT179qRmzZrlycHJWi4DT/pzi3ScEwACoT7qp/r8p+7LdQIMGOKIQFq7MRsvTtpU28drwaZ9kAUCtUbAh2vaZR5Sdn96kwDvuOOOtGTJElq3bl0sU3wIoyEWacG9dFxRgeyjfkk6hYzOPbAHDT28V1HIsK4GgSRf7tqlPc1e8lkjSUX61sdrQQM1hoIACFQQ8OGaRgKcHJbeJMAPPvhg6sUzePDg0l1cLgNP+mQpHVcUXB/1s70D7FOtqYmfi9Df1pq2d4BN9JLM4TFDx71JEyy08zPxMeaAgAkBSWybyM1rjmv9ffj3zWUekpefXK3jTQLsysAi5boMPGltr3RcUZx81C9Op5CPbg2wj/bp+LoI/W2uabMG2EQvyRyb8abjW4wFgSwEJLGdRb7ruXnon8caKk4u8xDV2r6f9yYBvvbaa1NZXXnllb6zbKSf68CTvgggHVcUYB/1C3V675Ov6JMvvqZObVsadYHwYQcgi1+L0N/2mknxpRt3JnpJ5qTtUo88rq/RJ72z+BxzQUBCQBLbEjlFjclLf937jG0ervMQ2/rmKc+bBJg/fhE9vvnmG/roo49oo402on322YdeeOGFPLlYWQuBZwVjqYX4UAOWBWAR+hexpoSRiV6SOZIxEv0wBgTyJFD2uC27/lJfIw9JJuVNAhyn4qpVq4hrfw866CC64IL/396ZQEtRnP375QICLkhAMCiLIBATASWfXgjua3AjBAyiUQnBqBACEYSIuABugMctURQFNcJRo7IpWYx4NK4ICsSoRwiyKOIfoyxqEFCZ/3nLzDD33rkz1d3Vy8w8fU7O98lUvVX11K/7/rr67erhtvOdmHJRC89FPpOLGEEmIO72g/Q9V91CqwxJH2+h/rvmpfHiaNNmHDNfXSdXz3+rRlHdFaTvD1vJ7KXra+z1aTMWmzI2/Sv2Mq7PBdfxip2v6/4nQbdB5jgJ/Xc9J7niRe1DohiTqzYSbYB1kG+99ZacccYZsm7dOldjjixOlMJzkWvkIkYQuHG3H6TvtdXNN6ZUSuTCGa/JojWbMtW95hiH0efsmHHMSRxtFuKofTp/+iJZvHZzlaLd2zWV+39xZK0fbrGZ4ySOtxAP17+7ZuA6nuvxlkK8uBkHbT9o/WKZwyh9SLEwSfcz8QZ42bJlctRRR8m2bduKja1EKTwXd7MuYgSZpLjbD9L3fHVrywErlvHGkcMWR5v55rC2ubqhT2epqKgjY+f8K+fK8IDKNmIzFpsyYekzCXFdnwuu4yWBURL7EKduXcxxnP2Paj6j9CFRjclVO4kxwA899FCVMaVSKdm6datMmzZNWrZsKQsXLnQ15sjiRCk8F/lMLmIEgRt3+0H67qduuY3XD6Ok1Mk3V9rHu59/r0ZX49xTOCncbPvh+lxwHc92HJSLjgBzbMc6Sh9i16PklEqMAa6oqKhBpX79+mb1V78Qp1+HK7YjSuG5uBt2ESPIHMXdfpC++6lbbuP1yihIfp/XtgqVzzdXKZG8K8CFYvO7+7xvzq3iVZXtee9njm1j56IXpG6csxGlD4lznH7aTowB9tP5pNeJUngu8plcxAgyJ3G3H6TvfuqW23i9MEoam2LP5fbCPo6yrufbdbw4mJRjm17mzUtZZem1fDb/IHXjnscofUjcY/XaPgbYKzEP5aMWnot8JhcxPCCqUTTu9oP03U/dchuvLSM/qzu2sf2WyzdXzKNfqrvruWboOl7wERKhEAGv572XOfYaO7uvQeoWGnPYv0ftQ8Iej8v4sRrgE0880WosderUkWeffdaqbJIKIbwkzQZ9KSYC5PcV02zRVwi4IRDmeR8kdpC6bsj4j4IPqZ1drAZ40KBBeWf1xRdflNWrV0vjxo1ly5Yt/hUQU02E5x18kDyrfHWDxPU+CmoEJZBvz13dWYEjOAHOieAMiRCMQHUNfr0rJVfNy73Xts15n463JscXPJ94Y73vXH1WgIPNc1Jrx2qAa4Py+eefy8iRI2XGjBly8skny/Tp06VNm+L7o4cB9ib7IHlW5Gh6Y53k0vn23J11UXc+Dexg8oKcaw6aJwQEcubkVrZrasgs9rE3ei5NpzHr/uozBh5R637d9evWfAk/e4qK+XzBhyR0BThXt5555hn51a9+ZVZ8p0yZIhdffHHRXioQnrepC3KXzVv63lgnuXS+PXd/3qNtkrteNH0Lcq4VzSDpaKIJ5DvP61bUkXWbtknbpntKv/9rZXXTW1u8NAT9YqPGmv3Ges+xNYaXfOMkgceHFIEB/uKLL8yqr672nnTSSWb1txhXfbNRIzxvl4EgeVbs0+qNdZJLB9FBkseVpL7BOEmzUZ59ca3B2uKl6Zbrvtz4kIQb4PSq7+bNm82q7yWXXFISVwSE520ag6xKxb0CTD6lt7nOVzqIDgr1Itc86eeKZy9dL+97XHHK1VYSdGDThzAZF5oDfoeAEnCtQZsVYJs8Yq+zY3O+eY3psjw+JKEGWFd9R40aVWXVt3Xr1i7nPtZYCM8b/iB5VnHmAAfptzdC5VE6LJ654gbJOaw+G2H128us2/bBtpyXtikLAS8EXGuwUA7wzMGVVqkUcY7BS9u2ZfEhCTXABx10kHzwwQfSvn17Of/880W3O6vtuOaaa2znOzHlEJ73qQiSZxXXPq2uVzK8Uyu9GkF0UBuNQitE2fU0X9DralESdOClD2EwLj0lMqIwCbjWYDre6hy7QBR60c3POL2cb37iu6iDD0mwAc5netPd1jK6HVqxHQiv2GbMX39d57L56wW1ChEolCOYXd9PvmASdJCEPhSaB36HQKkQKIbzDR+SUANcKidBbeNAeKUxw4VyvIphFcDLTBQar5dYSSpbzivAfbsdKJPP7ur8EXCS5pe+FDeBYrzuFMO1Hx+CAY7lyoDwYsHutFGbPDWbMk47FWKwUhpLdUzlmgOc5qB7oYaRBxmiHAldJgSK9bpTDP3Gh2CAY7mMILxYsDtt1PYO33Uum9NBeAhmO14PIRNVNNc8aQf97g1afXBJ0IH24XdPvClzln1Yg72f3OZETSCdKUkCxXzdScI5n08U+BAMcCwXDYQXC3anjRZDjpfLAZfbeF2yS1Is5jFJs0FfChFAr4UI+f8dH4IB9q+eADURXgB4llXDzhsr5pUJS4RVinkZbzb7A5s0FJE68uGWLz19vclPH73UCVsfXvoSZVkv8xhlv2gLArkIoNfwdIEPwQCHp648kRFeuNijyL+Koo1wKXmLbjvefHtuaotJyDe1HYs3QsVRupzHXhwzRC+zCaDX8PSAD8EAh6cuDHAsbLXRqFYNkp7xKzfGAAAgAElEQVTj5XoCbMZrs6NC3PmmUenDNX9X8Wzm0VVbxIFAUALoNSjB3PUxwBjgcJRVICrCCxc7eWPh8s0X3WZPXT976bocEfpwSZNYEIBAMRLAh2CAY9EtwrPH7idXM44VPj/9tKdQPCWTtAJc25zEoY/imUF6CoFkESiVa2t6HGsi+hpdoVnEh2CAC2kklN8Rnh1Wv/lffuvZ9apmqajb89vPKOolJQc435ykUiIXznhNFq3ZlEGShNzkKOaHNiBQTARK5dqa77oY17UHH4IBjuVagPDssAdZqYsybyxIP+1IFFepbPatmjQynV8f8S4QheYkSn0U1+zRWwgkh0Ch8zg5Pc3fk0JPxuJ4LwIfggGO5fxBeHbYiyVXs1j6aUe9NEoxJ6Uxj4yivAmUynlc6N2ION6LwIdggGO5uiA8O+xJvfuvnpP29a6UXDXvrRqDiuOu3o6s/1LFko/nRTvFMib/s0ZNCBQfAT0vfzf7TZlbAl8uZAW4uPRXJ5XSTDmOMAhggO2oJjH/K1efKts1NQNaXOI5pUmcj9qUZNtX23J2iqUUBCDggkASc2aDjEvHc/70RbJ47eYaYbq3ayqzLuou9etWBGnCc118CCvAnkXjogLCs6eYtFzN2u7kb+jTWepW1JF1m7Yl6otn9qQLl/Syqlo4WvglbLRTbGMKnxotQCB+ArWdl327HSiTz+4auVl0QWTWonU5nxTq346f92jroglPMfAhGGBPgnFVGOG5Ihl9nFLJSfNDrhTHXopj8jO31IFAkgiU4nmZtDHhQzDAsZzzCC8W7E4aLecVw1IZe3bO78at22VOCeQYOhF3mQYhBzx5E18q15psskkbEz4EAxzLmY/wYsHupNFyzhkthbHnGsM+DevJ59u/zugjrn05nQiUIJ4IlIKmPQ24SAqX4rwkbUz4EAxwLJcDhBcLdmeN2uSWOmssYYGKfez5cgv337dhyeZvJ0xGielO0lblEgMmAR0p9mtNLoRJGhM+BAMcy2mO8GLBTqMQkKTl4TEl8RJAD/Hyp/X4COBDMMCxqA/hxYK9YKPkAhZEFGsBF/NTLCt+Lsaaa7LCihurMAI0bqMHmAUATNXEEsCHYIBjESfCiwV73kaTlp+VPELx9sjV/LiKEyaNsPoYVtwwWYQduxCTQr+H3T/iQyAsAvgQDHBY2sobF+HFgj1vozYrQcnrdfn0yOX8JCkPL9cMuhxrdvyw4ha7CvPpAWbFPrv0vzYC+BAMcCxnB8KLBXveRskFTN6cZPeonOYnrLGGFTfZygnWO5gF40ft5BLAh2CAY1EnwquKPZ1jt+aT/8onn++Q5vs0kHb77SVnHXaAPPnPDfJ+BF9XC7LSU245gnGM1+v8hNHHMGKyAhzLJdi6Ua+6sw5MQQjETAAfggGORYIIbzf2fN98j3J/Vr+5fn7rxSI8B43GNV4v7Xopa4skjJi1tR1WW2HFtWVYjOVgVoyzRp9tCOBDMMA2OnFeBuHtRlrbCktt0Cf17SIDKts4nxMN6Cc3tNxWiOIcr+38hNHHMGLmE7HtWL2eCGHF9dqPYioPs2KaLfpqSwAfggG21YrTcghvN87acuxqAz7k+IPld70OcTofQYKVW45gMYw3jD6GETOI7qgLAQhAIAgBfAgGOIh+fNdFeCLpfMq/vfX/5B8r/2PN0tUKsKt8zqhXBq1BOSyYzWrj1u0yZ9mHNaLf0Kez/LxHW4et+g8VxpyEEdP/CKkJAQgUEwFXf29cjhkfggF2qSfrWOUuvHx5v9kQw8oBdpnX5zKWtYAiLJhrfHs3qCtf7PimSi+6t2sqsy7qLvXrVkTYu9xNhTEnYcSMHRQdgAAEQieQ1GtHufuQfBNfJ5VKpUJXRpk2UO7Cq2017diO+8l+ezfI7ALR+/AD5MnlG2Sd410gXK/mlXKOoJccbVer8y4uC2HMSRgxXYyVGBCAQHIJuP5742qk5e5DMMCulOQxTrkLL+58yrjb9yiXWIt7ydFOWn52rOBoHAIQgICIJPXvTbn7EAxwTKdnuQsv7jviuNuPSXa+mi3WFWBfg81TKYk5fK7HSDwIlDKBqM/hQu+5xP3ErNx9CAY4prO93IUXd05U3O3HJDtfzeZiVdmuqYm1eM2mTMwe7ZvJzMGVicgB9jXQAub3whmvyaIyGa9rfsSDQNwEor7mF3rPJQnXy3L3IRjgmM5KhOdvz12X00U+pz3NXKy09uw31jvPz7bvVXQleWIQHWtagkAYBKI+h2tr7/hOzaVX5+9Kv/9rFftiAT6kdqXxElwYZ+H/YiK8EOESGgKOCSQ1h8/xMAkHgZIlEPU5HHV7fiYOH4IB9qObwHUQXmCEBIBAZASiXj2KbGA0BIEyIRD1ORx1e36mER+CAfajm8B1EF5ghASAQGQEos4fDGtgUb8EFNY4iAsBrwSiPoejbs8rDy2PD8EA1yCwadMmufLKK2XBggXy2WefSdeuXWXSpEly9NFHm7LLly+XESNGyOuvvy7NmjWT4cOHy+WXX+5JfwjPEy4KQyB2AsWeM14Mf5Bjn2Q6UNIEoj6Ho27P6+ThQzDANQiceuqp8vHHH8udd94pzZs3l7vuukumT58uS5cuNf99yCGHSJ8+fWTkyJGyaNEiGTp0qEydOlUGDRpkrT+EZ42KghCAgAMCxfBI1sEwCQEBCFgSwIdggKsQWLVqlXTs2FFefvll6dmzp/lNP4jXqVMnOffcc6VRo0bGEK9du1bq1atnftfV4jlz5si7775rKTsePViDoiAEIOCEQDG8lONkoASBAASsCGCAMcBVCGzZskVeeeUVOemkk6RBgwaZ3zp06CC9evWS1atXS5MmTeThhx/O/LZw4UI55ZRTZOPGjdKiRQuEZ0WAQhCAQJQEWAGOkjZtQSD5BDDAGOCCKn388celf//+8uSTT8q4cePktNNOk8mTJ2fqvfPOO3LooYfKkiVL5IgjjigYTwsgPCtMFIIABBwRIAfYEUjCQKBECOBDMMB5paypELryqyvC8+bNE10JPu+882TixImZeroqfPDBB8uLL76YeVGuelB9mU7/lz4++ugjqayslA8++EBatWpVIqcTw4AABJJMIOkv5SSZHX2DQKkRwABjgGslMH/+fGN2e/ToYXaE0PzfLl26yOmnn55zBVhfkuvWrVvOeOPHj5cJEybU+A0DXGqXFMYDAQhAAAIQSD4BDDAGOCcB3QFCtzrr27evzJo1K5MPrOZXtz6bOXNmpp5NDjArwMm/GNBDCEAAAhCAQLkQwABjgGsQuPvuu83WZrq/72233SYVFRWZMrof8D333CPvvfee1K1b1/z72LFjZe7cuewCUS5XDcYJAQhAAAIQKHICGGAMcBUCK1eulM6dO8uZZ55p9vbNPjQFYseOHWYf4N69e8uYMWNk8eLFMmTIEGOKBw4caH06IDxrVBSEAAQgAAEIQMAxAXwIBrgKgRtvvNHs9JDrUIP74IMPmt0edHV42bJl0rJlSxk1apQMGzbMkzQRnidcFIYABCAAAQhAwCEBfAgG2KGc7EMhPHtWlIQABCAAAQhAwC0BfAgG2K2iLKMhPEtQFIMABCAAAQhAwDkBfAgG2LmobAIiPBtKlIEABCAAAQhAIAwC+BAMcBi6KhgT4RVERAEIQAACEIAABEIigA/BAIckrfxhEV4s2GkUAhCAAAQgAAERwYdggGM5ERBeLNhpFAIQgAAEIAABDHBeDdRJpVIpVBIOAQxwOFyJCgEIQAACEIBAYQL4EFaAC6skhBIILwSohIQABCAAAQhAwIoAPgQDbCUU14UQnmuixIMABCAAAQhAwJYAPgQDbKsVp+UQnlOcBIMABCAAAQhAwAMBfAgG2INc3BVFeO5YEgkCEIAABCAAAW8E8CEYYG+KcVQa4TkCSRgIQAACEIAABDwTwIdggD2LxkUFhOeCIjEgAAEIQAACEPBDAB+CAfajm8B1EF5ghASAAAQgAAEIQMAnAXwIBtindIJVQ3jB+FEbAhCAAAQgAAH/BPAhGGD/6glQE+EFgEdVCEAAAhCAAAQCEcCHYIADCchvZYTnlxz1IAABCEAAAhAISgAfggEOqiFf9RGeL2xUggAEIAABCEDAAQF8CAbYgYy8h0B43plRAwIQgAAEIAABNwTwIRhgN0ryGAXheQRGcQhAAAIQgAAEnBHAh2CAnYnJSyCE54UWZSEAAQhAAAIQcEkAH4IBdqkn61gIzxoVBSEAAQhAAAIQcEwAH4IBdiwpu3AIz44TpSAAAQhAAAIQcE8AH4IBdq8qi4gIzwISRSAAAQhAAAIQCIUAPgQDHIqwCgVFeIUI8TsEIAABCEAAAmERwIdggMPSVt64CC8W7DQKAQhAAAIQgICI4EMwwLGcCAgvFuw0CgEIQAACEIAABjivBuqkUqkUKgmHAAY4HK5EhQAEIAABCECgMAF8CCvAhVUSQgmEFwJUQkIAAhCAAAQgYEUAH4IBthKK60IIzzVR4kEAAhCAAAQgYEsAH4IBttWK03IIzylOgkEAAhCAAAQg4IEAPgQD7EEu7ooiPHcsiQQBCEAAAhCAgDcC+BAMsDfFOCqN8ByBJAwEIAABCEAAAp4J4EMwwJ5F46ICwnNBkRgQgAAEIAABCPghgA/BAPvRTeA6CC8wQgJAAAIQgAAEIOCTAD4EA+xTOsGqIbxg/KgNAQhAAAIQgIB/AvgQDLB/9QSoifACwKMqBCAAAQhAAAKBCOBDMMCBBOS3MsLzS456EIAABCAAAQgEJYAPwQAH1ZCv+gjPFzYqQQACEIAABCDggAA+BAPsQEbeQyA878yoAQEIQAACEICAGwL4EAywGyV5jILwPAKjOAQgAAEIQAACzgjgQzDAzsTkJRDC80KLshCAAAQgAAEIuCSAD8EAu9STdSyEZ42KghCAAAQgAAEIOCaAD8EAO5aUXTiEZ8eJUhCAAAQgAAEIuCeAD8EAu1eVRUSEZwGJIhCAAAQgAAEIhEIAH4IBDkVYhYIivEKE+B0CEIAABCAAgbAI4EMwwGFpK29chBcLdhqFAAQgAAEIQEBE8CEY4FhOBIQXC3YahQAEIAABCEAAA5xXA3VSqVQKlYRDAAMcDleiQgACEIAABCBQmAA+hBXgwioJoQTCCwEqISEAAQhAAAIQsCKAD8EAWwnFdSGE55oo8SAAAQhAAAIQsCWAD8EA22rFaTmE5xQnwSAAAQhAAAIQ8EAAH4IB9iAXd0URnjuWRIIABCAAAQhAwBsBfAgG2JtiHJVGeI5AEgYCEIAABCAAAc8E8CEYYM+icVEB4bmgSAwIQAACEIAABPwQwIdggP3oJnAdhBcYIQEgAAEIQAACEPBJAB+CAfYpnWDVEF4wftSGAAQgAAEIQMA/AXwIBti/egLURHgB4FEVAhCAAAQgAIFABPAhGOBAAvJbGeH5JUc9CEAAAhCAAASCEsCHYICDashXfYTnCxuVIAABCEAAAhBwQAAfggF2ICPvIRCed2bUgAAEIAABCEDADQF8CAbYjZI8RkF4HoFRHAIQgAAEIAABZwTwIRhgZ2LyEgjheaFFWQhAAAIQgAAEXBLAh2CAXerJOhbCs0ZFQQhAAAIQgAAEHBPAh2CAHUvKLhzCs+NEKQhAAAIQgAAE3BPAh2CA3avKIiLCs4BEEQhAAAIQgAAEQiGAD8EAhyKsQkERXiFC/A4BCEAAAhCAQFgE8CEY4LC0lTcuwosFO41CAAIQgAAEICAi+BAMcCwnAsKLBTuNQgACEIAABCCAAc6rgTqpVCqFSsIhgAEOhytRIQABCEAAAhAoTAAfwgpwYZVUK7Fr1y6ZMGGCTJ8+XTZv3ixHH320TJ06VTp06GAdC+FZo6IgBCAAAQhAAAKOCeBDMMCeJaXmVw3vAw88IAceeKCMGTNGVq9eLW+//bbsscceVvEQnhUmCkEAAhCAAAQgEAIBfAgG2JOsdu7cKfvtt59MmTJFLr30UlN3y5YtcsABB8j9998vAwYMsIqH8KwwUQgCEIAABCAAgRAI4EMwwJ5ktXjxYunevbusWLFCOnXqlKmraRBdu3Y1K8M2B8KzoUQZCEAAAhCAAATCIIAPwQB70tWcOXOkX79+sm3bNmnUqFGmbv/+/c2/LViwwCoewrPCRCEIQAACEIAABEIggA/BAHuS1axZs+SCCy6Qb775RioqKjJ1L7zwQtmwYYMsXLgwZ7zPPvtM9H/p44MPPpCePXuKrii3bNnSUx8oDAEIQAACEIAABIIQ+Oijj6SyslLWrFkjBx10UJBQJVeXbdByTOns2bPl7LPPzrkCvGPHDpk/f35OIYwfP97sHMEBAQhAAAIQgAAEkkJAF+KOPPLIpHQnEf3AAOeYhnQO8KpVq+Tggw/OlNAc4MMOO0zuuusuqxXg7du3i64Ct2vXTurVqxfKhKfv7lhlzo8XTvbyg5UdKzjByY6AfSk0ZccKTt44PfXUU3LyySdLw4YN7SqWSSkMcI6J1lXeFi1ayK233iqDBw82JfzsAhGFhsjvsaMMJztOWgpWdqzgBCc7Aval0JQdKzjByY5A/lIY4Fr4jBs3TqZNm2a2PdO8mdGjR8vatWvlX//6l/U+wC4mqFAMLgSFCH37O5zsOMEKTvYE7Epy7tlx4tyDkz0Bu5KcexhgO6VUK6UvwF155ZXmQxhffvmlHHvssSb1IWlJ5AjcbnrhZMeJP8JwsidgV5Jzz44T5x6c7AnYleTcwwDbKaVIS+muE5qqMXLkSGncuHGRjiL8bsPJnjGs7FjBCU52BOxLoSk7VnCCkx0BDLALTsSAAAQgAAEIQAACECgRAuQAl8hEMgwIQAACEIAABCAAATsCGGA7TpSCAAQgAAEIQAACECgRAhjgEplIhgEBCEAAAhCAAAQgYEcAA2zHiVIQgAAEIAABCEAAAiVCAANcpBO5a9cu89nl6dOny+bNm0W/Ujd16lTp0KFDkY4oeLevv/56WbhwoTz//POZYMuXL5cRI0bI66+/Ls2aNZPhw4fL5Zdfnvm9XDhu2rTJbOu3YMEC0Teou3btKpMmTTK60QNOu/X38ccfy6hRo+Rvf/ub2QLxuOOOk5tvvll+8IMfwKqW03TlypXywx/+UO688075xS9+AadqnNatW5dzC8377rtPLrroIs6/LF4PPfSQuTatXr3afIl1/Pjx8rOf/QxN/Y+R/n074YQTcp6J+tVZ5cb13M5PYIDtOCWulJpfNby6T/GBBx4oY8aMMcJ/++23E/WhjqjA3X777WYrON2vOW2AP/30UznkkEOkT58+5rdFixbJ0KFDDbdBgwaZrpULx1NPPVXU2KlBad68udnTWm+eli5dav4bTruV2qNHD6moqJDf//73stdee8nVV18tr7zyiuin0dUQw6rqWf3VV19Jz549zU2mXo/UAHPuVWWkn6Lt37+/uUbXqVMn8+O+++4r27ZtQ1P/IzJr1iyjn1tuuUXOOOMMeeSRR4wBfumll6RTp05wEpGdO3eKLmhkH/qBrtNPP13uvvtu+elPfwonS+OBAbYElaRiegLst99+MmXKFLn00ktN15L6qeawuX344YdmBeXFF1+U1q1by/77758xwDfddJMxevoFv3r16pmu6CronDlz5N133zUXknLgqMatY8eO8vLLLxujokcqlTJ/UM4991xp1KgRnP4nVDVuw4YNk6uuukoOPfRQ86///Oc/5fDDD5fFixebJwxoqupZreeU3lw+99xzGQPMuVeV0Y033iiPPfaYWZmrfsDqWyJ6TWrfvr2cffbZ5olL+vjxj38sxx9/vPlPzr2af1H1BrRbt27SuXNnefTRRwU92bsODLA9q8SU1D/E3bt3lxUrVhgTkz70cbY+2tYVznI5dGVFVwk0/WHixInG7KZXgPWOuEmTJvLwww9ncKiBOeWUU2Tjxo2mbDlw1JsjXcE86aSTpEGDBhkWmi7Tq1cvsyoFp9xnjOpEU2ZUU3rTpI9iYbWb1QsvvGBWnNTYtWnTJmOAOfeq6mnAgAHmqUL2tShdAlbfktDz6/vf/755KqWGrvoBp9zXKH36ee211xp+LVu2NCvBXKPsHBAG2I5TokrpCma/fv3MozNdvUsf+ohN/03zPMvx0Edn2QZYbwZOO+00mTx5cgbHO++8Y1b2lixZIu+//37Zcnz88cfNI9knn3xSxo0bB6ccJ8zFF18smqOpNw3KSdNI0NRuUHpjddhhh5kVp/POO8882k+nQMCpqqD0mqPmZMeOHaL50vpERlNrdHUTVt+y0nPsJz/5iXmad91118myZctEc1r1acxZZ50FpxzXqO3bt5snn5rap+l8eqAne/eDAbZnlZiSmid1wQUXyDfffGNWFdLHhRdeKBs2bDCPacvxqG6AdYVT/zDrynD6SL9YoRdZNcvlyFFTIXTlV1eE582bZ16chFPNM0ZvljTnV/PqdOVO8xD1pgFW37LS9Bl9bK2PXfXINsBoareeNNVKc8l1VVMf7e+9996i1/A77rhDnnnmGbnkkkvQlIhhotdjNb26oqk3V7Nnz5YbbrgBTrX8Qdcb9Msuu0z0JUt9yVsPzj1794MBtmeVmJJ6UdA8qVwrwLrCMH/+/MT0NcqOVDfAXbp0MY+Dcq0A62M2NcPlxlG1oQZOX/TSJwX6BAFO+VWqO4UoI02X0ScHaEpk5syZMnbsWNGXb77zne/UMMBoqqqm/vvf/5r3ELJTkPQmVG8a1q9fj6ZETI70OeecY1L4hgwZkgGoL8PpoU/sOPeq6uqYY44xedN//OMfMz9w7tm7DgywPavElEznAOvLTbpNTPrQHGC9a9YXBcrxqG6A9WKpd8X6xzp95MoBLheOugOEbgnXt29fs9qS/mMMp91ni+6U8eyzz5qV3rp162Z+0Nxf3T5O/w1NidmGSVfEsw2dmjz9b70mtW3bFk4FLsKjR482W+3pI2w0JeY9haOOOsqkQKS3Z1SEusOR3qwfdNBBcMrS1H/+8x/57ne/a9hoql/64Hpu734wwPasElNSV3lbtGght956qwwePNj0q1x3gcielOoGWPeSvOeee+S9997LmBldtZo7d655YaCcOOpjfM0T032Qb7vttiqpM3DarSJ9mUsfVasJPvHEE80P+pa1bn3Wu3dvs8sImhLR3Vc0PST70LxWfdqiq3j6YiqcvqXz5ptvmt1X1OxmGzu9iVA96Q4jsBKjJ+VxzTXXVNmrXc1d/fr1DUM47T7j9GmevgukN+Z77rln5geu5/ZWDQNszypRJfXFpWnTpsn9999v7ox1NUFzWvWR5B577JGovkbVmeoGWFfz0sZFVxF05VwfrelFdODAgaZb5cBRX7rRLXLOPPPMGjuEaAqE3gjA6VuVak6r/sHVc0nz6/TxvuYgPv3002ang4YNG8KqlhM6OweYc283JE2hUfOmK+R6I6pbL957773mXNSUGjV9nH/f8tLdfHR7T/3bVllZafLL1RDrDal+iAZOu3Wl77bokzy9vmcfnHv2jgMDbM8qUSX1BTjdf1PfutY7Z/0AhKY+qBku16O6AVYO+gdGVz31jWJ9C1u/8KX7vKaPcuCoe5Cq0c916I3Agw8+CKcsOFu3bjX5rfqCoD5Z0Tw7fdqS3hcYTeW+wmQbYM69qoz0cfUVV1whf/nLX4ym9Kt5ulKn2oJVVVZ6rmm6lj5l0G3RdHcD3R0CTlU56RM9/bv26quv1jghuUbZuSAMsB0nSkEAAhCAAAQgAAEIlAgBDHCJTCTDgAAEIAABCEAAAhCwI4ABtuNEKQhAAAIQgAAEIACBEiGAAS6RiWQYEIAABCAAAQhAAAJ2BDDAdpwoBQEIQAACEIAABCBQIgQwwCUykQwDAhCAAAQgAAEIQMCOAAbYjhOlIAABCEAAAhCAAARKhAAGuEQmkmFAAAIQgAAEIAABCNgRwADbcaIUBCAAAQhAAAIQgECJEMAAl8hEMgwIQMA9geOPP17+8Y9/VAmsnxrXrwr26dNHbrrpJtHPSbs69EuO+kXD8ePHuwpZI071NvQLbtlHgwYNpFWrVnLOOeeYLwjuueeeofWFwBCAAATiIoABjos87UIAAoknoAZ4//33lzvuuCPT1y+++EL+/ve/y29/+1u5+OKLzWdbXR36yVw11HvvvberkFYG+PbbbzeGVw8d3+LFi+Xyyy83n6J9+umnpV69eqH1h8AQgAAE4iCAAY6DOm1CAAJFQUANsK6YPvjggzX6+8tf/lLmz58vn376aVGMJd3JXCvADzzwgFl5zj7eeOMNOfLII2X69OmiY+WAAAQgUEoEMMClNJuMBQIQcEognwEeOnSoPPHEE/Lxxx+bNnfu3ClXX321zJo1S7Zu3SqdO3eWiRMnyqmnnprpk64cjx07Vt555x1p3769jBo1SgYPHixr1qwxRru6Of3zn/8s1113nbz11lvSuHFjOffcc+WGG26Qhg0bmpiavnDvvffKY489Ji+99JI0bdpUhg0bZtqo7bA1wFr/2GOPFU35WLhwoVOuBIMABCAQNwEMcNwzQPsQgEBiCeQywF9//bVJCxgwYIBJgbjllltM/8877zx5++23RdMJNIf2qaeekiuuuELmzp0rZ5xxhixfvtysqF522WXG9Op/DxkyRDZv3pzTAM+bN0/69etn8oH79+8v//73v035yspKmT17dsYA77vvviYNQ/995syZcv3118sLL7wgxxxzTE6uXgzwr3/9a/nTn/4kn3zySWLniI5BAAIQ8EMAA+yHGnUgAIGyIKAG+OWXXxZ9MSx9fPnll9K2bVuTMnDllVea/NhVq1ZJx44dZcmSJXLEEUdkyg4cOFDWrVsnzz//vOj/v3LlSnn11Vczv//hD3+Q4cOH5zTA3bt3l9atW5tV5vShprp3795mBVnzc3UFeMSIEcZ0pw9dBR49enStq8BeDLC+BDdlyhT56quvykGeGdUAAAQOSURBVGK+GSQEIFA+BDDA5TPXjBQCEPBIQA1ws2bNZPLkybJr1y557bXXzApu3759ZerUqZmXwx5//HGzSrvXXntVaUGNY5MmTWTjxo3SpUsXOeWUU+TWW2/NlNFV4G7duuU0wLr7wo033mhetksfmlqh8R599FHz0poa4HvuuUcuueSSTBk1uIMGDZJrr70252i9GGBNp9C2WAH2KByKQwACiSeAAU78FNFBCEAgLgK5UiCee+45Ofnkk43pVBOsh6YJaErE0qVLZZ999qnS3bp160q7du2M0T3uuOOqrNbmM8C6G8SkSZPMCm/62LRpkzHkmgKhJlwNcPUX2AptpebFAP/oRz8y49HcZQ4IQAACpUQAA1xKs8lYIAABpwRqewluzJgxcvPNN8tf//pX6dWrl8n91ZfeFixYYPJ904emEFRUVJgX2TRl4t1335VFixZlfp82bZpceumlOVeANae3TZs2VVIgdNcJ3X84OwUiLAOsu0BoOofugKHpGxwQgAAESokABriUZpOxQAACTgnUZoC3b98uXbt2lR07dhjzq/v2nnXWWebFtrvuusuY4Tlz5oga5RkzZpiUBN3J4fDDDzf76+q2Ympi9SU63ft37dq1Jq84e3VW0yo0zWHChAnm/2r+sL4EpzE0F1gPVyvAufYB1hf4OnToIM8884zoKjYHBCAAgVIigAEupdlkLBCAgFMC+bZB0y/EnXDCCfKb3/zGfChj27Zt5stpmg6hqQq6zdnIkSPloosuyvRJjatuUaY7Onzve9/L7PKwYcMG83W56ukJjzzyiPna3IoVK6R58+Zmpwk1xOmvz7kywNnQdLs1TdnQtvQFvfSWa07BEgwCEIBAzAQwwDFPAM1DAALlQUB3iNAdIzQXOH08/PDDZjVYv77G19bKQweMEgIQSAYBDHAy5oFeQAACJU7gvvvuM9uTPfTQQyaNQbdO0xfpjjrqqJxfmitxHAwPAhCAQKwEMMCx4qdxCECgXAikUinzMpy+VPbhhx9KixYtzJfdslMayoUF44QABCAQNwEMcNwzQPsQgAAEIAABCEAAApESwABHipvGIAABCEAAAhCAAATiJoABjnsGaB8CEIAABCAAAQhAIFICGOBIcdMYBCAAAQhAAAIQgEDcBDDAcc8A7UMAAhCAAAQgAAEIREoAAxwpbhqDAAQgAAEIQAACEIibAAY47hmgfQhAAAIQgAAEIACBSAlggCPFTWMQgAAEIAABCEAAAnETwADHPQO0DwEIQAACEIAABCAQKQEMcKS4aQwCEIAABCAAAQhAIG4CGOC4Z4D2IQABCEAAAhCAAAQiJYABjhQ3jUEAAhCAAAQgAAEIxE0AAxz3DNA+BCAAAQhAAAIQgECkBP4/SabpbTueAuQAAAAASUVORK5CYII=\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "pb_num_dict = {_k:len(_v.kept_probes) for _k,_v in filtered_pb_dict.items()}\n",
    "\n",
    "pb_nums = np.array(list(pb_num_dict.values()))\n",
    "# number of probes histogram\n",
    "fig, ax = plt.subplots()\n",
    "ax.plot(pb_nums, '.')\n",
    "ax.set_ylabel(\"Num probes per region\")\n",
    "ax.set_xlabel(\"Region ID\")\n",
    "ax.set_title(\"Probe number after filtering\")\n",
    "ax.set_ylim([0,100])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "d87753a7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\chr_2_final_region_num.pkl\n"
     ]
    }
   ],
   "source": [
    "chr_2_region_savefile = os.path.join(library_folder, 'chr_2_final_region_num.pkl')\n",
    "overwrite = True\n",
    "\n",
    "filtered_reg_names = {}\n",
    "for _reg, _pbde in filtered_pb_dict.items():\n",
    "    if len(_pbde.kept_probes) > 0:\n",
    "        filtered_reg_names[_reg] = list(_pbde.kept_probes.values())[0]['reg_name']\n",
    "\n",
    "if not os.path.exists(chr_2_region_savefile) or overwrite:\n",
    "    chr_2_region_num = {}\n",
    "    for _reg, _info in filtered_reg_names.items():\n",
    "        _chr = _info.split('_')[0].split(':')[0]\n",
    "        if _chr not in chr_2_region_num:\n",
    "            chr_2_region_num[_chr] = 1\n",
    "        else:\n",
    "            chr_2_region_num[_chr] += 1\n",
    "    print(f\"Saving into file: {chr_2_region_savefile}\")\n",
    "\n",
    "    pickle.dump(chr_2_region_num, open(chr_2_region_savefile, 'wb'))\n",
    "else:\n",
    "    print(f\"loading from file: {chr_2_region_savefile}\")\n",
    "    chr_2_region_num = pickle.load(open(chr_2_region_savefile, 'rb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "28386f9b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'2': 668}"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "chr_2_region_num"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "id": "f9b134b0",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving region names into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\chr_2_region_names.pkl\n"
     ]
    }
   ],
   "source": [
    "#\n",
    "reg_2_start = {}\n",
    "\n",
    "for _name, _info in filtered_reg_names.items():\n",
    "    _locus = _info.split('_')[0]\n",
    "    _start,_end = _locus.split(':')[1].split('-')\n",
    "    reg_2_start[_name] = int(_start)\n",
    "#    \n",
    "chr_2_region_names = {}\n",
    "\n",
    "for _reg, _info in filtered_reg_names.items():\n",
    "    _chr = _info.split('_')[0].split(':')[0]\n",
    "    if _chr not in chr_2_region_names:\n",
    "        chr_2_region_names[_chr] = [_reg]\n",
    "    else:\n",
    "        chr_2_region_names[_chr].append(_reg)\n",
    "# sort\n",
    "for _chr, _regs in chr_2_region_names.items():\n",
    "    _sorted_regs = sorted(_regs, key=lambda _r: reg_2_start[_r])\n",
    "    chr_2_region_names[_chr] = _sorted_regs\n",
    "    \n",
    "overwrite_region_names = True\n",
    "chr_2_region_name_savefile = os.path.join(library_folder, 'chr_2_region_names.pkl')\n",
    "\n",
    "if not os.path.exists(chr_2_region_name_savefile) or overwrite_region_names:\n",
    "    print(f\"Saving region names into file: {chr_2_region_name_savefile}\")\n",
    "    pickle.dump(chr_2_region_names, open(chr_2_region_name_savefile, 'wb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "id": "4a37d6da",
   "metadata": {},
   "outputs": [],
   "source": [
    "#\n",
    "reg_2_start = {}\n",
    "\n",
    "for _name, _info in filtered_reg_names.items():\n",
    "    _locus = _info.split('_')[0]\n",
    "    _start,_end = _locus.split(':')[1].split('-')\n",
    "    reg_2_start[_name] = int(_start)\n",
    "#    \n",
    "chr_2_region_names = {}\n",
    "\n",
    "for _reg, _info in filtered_reg_names.items():\n",
    "    _chr = _info.split('_')[0].split(':')[0]\n",
    "    if _chr not in chr_2_region_names:\n",
    "        chr_2_region_names[_chr] = [_reg]\n",
    "    else:\n",
    "        chr_2_region_names[_chr].append(_reg)\n",
    "# sort\n",
    "for _chr, _regs in chr_2_region_names.items():\n",
    "    _sorted_regs = sorted(_regs, key=lambda _r: reg_2_start[_r])\n",
    "    chr_2_region_names[_chr] = _sorted_regs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "id": "5469fa71",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\chr_2_region_names.pkl\n"
     ]
    }
   ],
   "source": [
    "overwrite_region_names = True\n",
    "chr_2_region_name_savefile = os.path.join(library_folder, 'chr_2_region_names.pkl')\n",
    "\n",
    "if not os.path.exists(chr_2_region_name_savefile) or overwrite_region_names:\n",
    "    print(chr_2_region_name_savefile)\n",
    "    pickle.dump(chr_2_region_names, open(chr_2_region_name_savefile, 'wb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "id": "79fb2c86",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'name': '2:48275000-48280000_strand_+_gene_NRNX1-seg-56_reg_55_pb_78',\n",
       " 'reg_index': 55,\n",
       " 'reg_name': '2:48275000-48280000_strand_+_gene_NRNX1-seg-56',\n",
       " 'pb_index': 78,\n",
       " 'strand': '+',\n",
       " 'gc': 0.35714285714285715,\n",
       " 'tm': 73.95448612289107,\n",
       " 'map_genome': 38,\n",
       " 'map_rep_genome': 0,\n",
       " 'map_self_sequences': 26}"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "list(list(filtered_pb_dict.values())[51].kept_probes.values())[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "id": "42c9910a",
   "metadata": {},
   "outputs": [],
   "source": [
    "# clear memory\n",
    "del(_pbde)\n",
    "del(_pb_class)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2c3d9549",
   "metadata": {},
   "source": [
    "<a id='3'></a>\n",
    "# 3. Load and asssemble probes"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d373567a",
   "metadata": {},
   "source": [
    "<a id='3.1'></a>\n",
    "## 3.1 design readout mapping dict (region_to_readout)\n",
    "\n",
    "load directly from http://localhost:8889/notebooks/Chromatin_Analysis_Scripts/Library_design/CTP-10_Aire/CTP-10_Aire_introns_DNA-MERFISH.ipynb#3.3-assemble-probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "136b756f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "668"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# summarize total readout usage\n",
    "region_2_readout_dict = pickle.load(open(os.path.join(library_folder, 'region_2_readout.pkl'), 'rb'))\n",
    "region_2_readout_dict = {str(_r):_bs for _r,_bs in region_2_readout_dict.items()}\n",
    "len(region_2_readout_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "55ba1861",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'NRNX1-seg-1': ['c6', 'c10', 'c23'],\n",
       " 'NRNX1-seg-2': ['c30', 'c34', 'c47'],\n",
       " 'NRNX1-seg-3': ['c54', 'c58', 'c71'],\n",
       " 'NRNX1-seg-4': ['c78', 'c82', 'c95'],\n",
       " 'NRNX1-seg-5': ['c102', 'c106', 'c119'],\n",
       " 'NRNX1-seg-6': ['c126', 'c130', 'c143'],\n",
       " 'NRNX1-seg-7': ['c150', 'c154', 'c167'],\n",
       " 'NRNX1-seg-9': ['c174', 'c178', 'c191'],\n",
       " 'NRNX1-seg-10': ['c200', 'c205', 'c209'],\n",
       " 'NRNX1-seg-13': ['c224', 'c229', 'c233'],\n",
       " 'NRNX1-seg-14': ['c0', 'c11', 'c15'],\n",
       " 'NRNX1-seg-15': ['c24', 'c35', 'c39'],\n",
       " 'NRNX1-seg-16': ['c48', 'c59', 'c63'],\n",
       " 'NRNX1-seg-17': ['c72', 'c83', 'c87'],\n",
       " 'NRNX1-seg-18': ['c96', 'c107', 'c111'],\n",
       " 'NRNX1-seg-19': ['c120', 'c131', 'c135'],\n",
       " 'NRNX1-seg-20': ['c144', 'c155', 'c159'],\n",
       " 'NRNX1-seg-21': ['c168', 'c179', 'c183'],\n",
       " 'NRNX1-seg-22': ['c207', 'c208', 'c215'],\n",
       " 'NRNX1-seg-23': ['c231', 'c232', 'c239'],\n",
       " 'NRNX1-seg-24': ['c2', 'c7', 'c14'],\n",
       " 'NRNX1-seg-25': ['c26', 'c31', 'c38'],\n",
       " 'NRNX1-seg-26': ['c50', 'c55', 'c62'],\n",
       " 'NRNX1-seg-27': ['c74', 'c79', 'c86'],\n",
       " 'NRNX1-seg-28': ['c98', 'c103', 'c110'],\n",
       " 'NRNX1-seg-30': ['c122', 'c127', 'c134'],\n",
       " 'NRNX1-seg-31': ['c146', 'c151', 'c158'],\n",
       " 'NRNX1-seg-32': ['c170', 'c175', 'c182'],\n",
       " 'NRNX1-seg-33': ['c192', 'c199', 'c201'],\n",
       " 'NRNX1-seg-34': ['c216', 'c223', 'c225'],\n",
       " 'NRNX1-seg-35': ['c3', 'c4', 'c16'],\n",
       " 'NRNX1-seg-36': ['c27', 'c28', 'c40'],\n",
       " 'NRNX1-seg-37': ['c51', 'c52', 'c64'],\n",
       " 'NRNX1-seg-38': ['c75', 'c76', 'c88'],\n",
       " 'NRNX1-seg-39': ['c99', 'c100', 'c112'],\n",
       " 'NRNX1-seg-40': ['c123', 'c124', 'c136'],\n",
       " 'NRNX1-seg-41': ['c147', 'c148', 'c160'],\n",
       " 'NRNX1-seg-42': ['c171', 'c172', 'c184'],\n",
       " 'NRNX1-seg-43': ['c193', 'c206', 'c212'],\n",
       " 'NRNX1-seg-44': ['c217', 'c230', 'c236'],\n",
       " 'NRNX1-seg-45': ['c1', 'c13', 'c22'],\n",
       " 'NRNX1-seg-46': ['c25', 'c37', 'c46'],\n",
       " 'NRNX1-seg-47': ['c49', 'c61', 'c70'],\n",
       " 'NRNX1-seg-48': ['c73', 'c85', 'c94'],\n",
       " 'NRNX1-seg-49': ['c97', 'c109', 'c118'],\n",
       " 'NRNX1-seg-50': ['c121', 'c133', 'c142'],\n",
       " 'NRNX1-seg-51': ['c145', 'c157', 'c166'],\n",
       " 'NRNX1-seg-52': ['c169', 'c181', 'c190'],\n",
       " 'NRNX1-seg-53': ['c195', 'c209', 'c213'],\n",
       " 'NRNX1-seg-54': ['c219', 'c233', 'c237'],\n",
       " 'NRNX1-seg-55': ['c10', 'c12', 'c19'],\n",
       " 'NRNX1-seg-56': ['c34', 'c36', 'c43'],\n",
       " 'NRNX1-seg-57': ['c58', 'c60', 'c67'],\n",
       " 'NRNX1-seg-58': ['c82', 'c84', 'c91'],\n",
       " 'NRNX1-seg-59': ['c106', 'c108', 'c115'],\n",
       " 'NRNX1-seg-60': ['c130', 'c132', 'c139'],\n",
       " 'NRNX1-seg-62': ['c154', 'c156', 'c163'],\n",
       " 'NRNX1-seg-63': ['c178', 'c180', 'c187'],\n",
       " 'NRNX1-seg-64': ['c202', 'c204', 'c211'],\n",
       " 'NRNX1-seg-65': ['c226', 'c228', 'c235'],\n",
       " 'NRNX1-seg-66': ['c5', 'c9', 'c17'],\n",
       " 'NRNX1-seg-67': ['c29', 'c33', 'c41'],\n",
       " 'NRNX1-seg-68': ['c53', 'c57', 'c65'],\n",
       " 'NRNX1-seg-69': ['c77', 'c81', 'c89'],\n",
       " 'NRNX1-seg-70': ['c101', 'c105', 'c113'],\n",
       " 'NRNX1-seg-71': ['c125', 'c129', 'c137'],\n",
       " 'NRNX1-seg-72': ['c149', 'c153', 'c161'],\n",
       " 'NRNX1-seg-73': ['c173', 'c177', 'c185'],\n",
       " 'NRNX1-seg-74': ['c197', 'c203', 'c214'],\n",
       " 'NRNX1-seg-75': ['c221', 'c227', 'c238'],\n",
       " 'NRNX1-seg-76': ['c2', 'c20', 'c21'],\n",
       " 'NRNX1-seg-77': ['c26', 'c44', 'c45'],\n",
       " 'NRNX1-seg-79': ['c50', 'c68', 'c69'],\n",
       " 'NRNX1-seg-80': ['c74', 'c92', 'c93'],\n",
       " 'NRNX1-seg-81': ['c98', 'c116', 'c117'],\n",
       " 'NRNX1-seg-82': ['c122', 'c140', 'c141'],\n",
       " 'NRNX1-seg-83': ['c146', 'c164', 'c165'],\n",
       " 'NRNX1-seg-84': ['c170', 'c188', 'c189'],\n",
       " 'NRNX1-seg-85': ['c201', 'c206', 'c215'],\n",
       " 'NRNX1-seg-86': ['c225', 'c230', 'c239'],\n",
       " 'NRNX1-seg-87': ['c1', 'c15', 'c18'],\n",
       " 'NRNX1-seg-88': ['c25', 'c39', 'c42'],\n",
       " 'NRNX1-seg-89': ['c49', 'c63', 'c66'],\n",
       " 'NRNX1-seg-90': ['c73', 'c87', 'c90'],\n",
       " 'NRNX1-seg-91': ['c97', 'c111', 'c114'],\n",
       " 'NRNX1-seg-92': ['c121', 'c135', 'c138'],\n",
       " 'NRNX1-seg-93': ['c145', 'c159', 'c162'],\n",
       " 'NRNX1-seg-94': ['c169', 'c183', 'c186'],\n",
       " 'NRNX1-seg-95': ['c194', 'c198', 'c205'],\n",
       " 'NRNX1-seg-96': ['c218', 'c222', 'c229'],\n",
       " 'NRNX1-seg-97': ['c6', 'c11', 'c12'],\n",
       " 'NRNX1-seg-98': ['c30', 'c35', 'c36'],\n",
       " 'NRNX1-seg-99': ['c54', 'c59', 'c60'],\n",
       " 'NRNX1-seg-100': ['c78', 'c83', 'c84'],\n",
       " 'NRNX1-seg-101': ['c102', 'c107', 'c108'],\n",
       " 'NRNX1-seg-102': ['c126', 'c131', 'c132'],\n",
       " 'NRNX1-seg-103': ['c150', 'c155', 'c156'],\n",
       " 'NRNX1-seg-104': ['c174', 'c179', 'c180'],\n",
       " 'NRNX1-seg-105': ['c195', 'c196', 'c208'],\n",
       " 'NRNX1-seg-106': ['c219', 'c220', 'c232'],\n",
       " 'NRNX1-seg-107': ['c0', 'c14', 'c17'],\n",
       " 'NRNX1-seg-108': ['c24', 'c38', 'c41'],\n",
       " 'NRNX1-seg-109': ['c48', 'c62', 'c65'],\n",
       " 'NRNX1-seg-110': ['c72', 'c86', 'c89'],\n",
       " 'NRNX1-seg-111': ['c96', 'c110', 'c113'],\n",
       " 'NRNX1-seg-112': ['c120', 'c134', 'c137'],\n",
       " 'NRNX1-seg-113': ['c144', 'c158', 'c161'],\n",
       " 'NRNX1-seg-114': ['c168', 'c182', 'c185'],\n",
       " 'NRNX1-seg-115': ['c193', 'c203', 'c213'],\n",
       " 'NRNX1-seg-117': ['c217', 'c227', 'c237'],\n",
       " 'NRNX1-seg-118': ['c9', 'c16', 'c19'],\n",
       " 'NRNX1-seg-119': ['c33', 'c40', 'c43'],\n",
       " 'NRNX1-seg-120': ['c57', 'c64', 'c67'],\n",
       " 'NRNX1-seg-121': ['c81', 'c88', 'c91'],\n",
       " 'NRNX1-seg-122': ['c105', 'c112', 'c115'],\n",
       " 'NRNX1-seg-123': ['c129', 'c136', 'c139'],\n",
       " 'NRNX1-seg-124': ['c153', 'c160', 'c163'],\n",
       " 'NRNX1-seg-125': ['c177', 'c184', 'c187'],\n",
       " 'NRNX1-seg-126': ['c200', 'c212', 'c215'],\n",
       " 'NRNX1-seg-127': ['c224', 'c236', 'c239'],\n",
       " 'NRNX1-seg-128': ['c7', 'c13', 'c23'],\n",
       " 'NRNX1-seg-129': ['c31', 'c37', 'c47'],\n",
       " 'NRNX1-seg-130': ['c55', 'c61', 'c71'],\n",
       " 'NRNX1-seg-131': ['c79', 'c85', 'c95'],\n",
       " 'NRNX1-seg-133': ['c103', 'c109', 'c119'],\n",
       " 'NRNX1-seg-134': ['c127', 'c133', 'c143'],\n",
       " 'NRNX1-seg-135': ['c151', 'c157', 'c167'],\n",
       " 'NRNX1-seg-136': ['c175', 'c181', 'c191'],\n",
       " 'NRNX1-seg-137': ['c192', 'c205', 'c211'],\n",
       " 'NRNX1-seg-138': ['c216', 'c229', 'c235'],\n",
       " 'NRNX1-seg-139': ['c3', 'c15', 'c20'],\n",
       " 'NRNX1-seg-140': ['c27', 'c39', 'c44'],\n",
       " 'NRNX1-seg-141': ['c51', 'c63', 'c68'],\n",
       " 'NRNX1-seg-142': ['c75', 'c87', 'c92'],\n",
       " 'NRNX1-seg-143': ['c99', 'c111', 'c116'],\n",
       " 'NRNX1-seg-144': ['c123', 'c135', 'c140'],\n",
       " 'NRNX1-seg-145': ['c147', 'c159', 'c164'],\n",
       " 'NRNX1-seg-146': ['c171', 'c183', 'c188'],\n",
       " 'NRNX1-seg-147': ['c204', 'c209', 'c214'],\n",
       " 'NRNX1-seg-148': ['c228', 'c233', 'c238'],\n",
       " 'NRNX1-seg-150': ['c14', 'c18', 'c22'],\n",
       " 'NRNX1-seg-151': ['c38', 'c42', 'c46'],\n",
       " 'NRNX1-seg-152': ['c62', 'c66', 'c70'],\n",
       " 'NRNX1-seg-154': ['c86', 'c90', 'c94'],\n",
       " 'NRNX1-seg-155': ['c110', 'c114', 'c118'],\n",
       " 'NRNX1-seg-156': ['c134', 'c138', 'c142'],\n",
       " 'NRNX1-seg-157': ['c158', 'c162', 'c166'],\n",
       " 'NRNX1-seg-158': ['c182', 'c186', 'c190'],\n",
       " 'NRNX1-seg-159': ['c202', 'c206', 'c207'],\n",
       " 'NRNX1-seg-160': ['c226', 'c230', 'c231'],\n",
       " 'NRNX1-seg-161': ['c5', 'c6', 'c19'],\n",
       " 'NRNX1-seg-162': ['c29', 'c30', 'c43'],\n",
       " 'NRNX1-seg-163': ['c53', 'c54', 'c67'],\n",
       " 'NRNX1-seg-164': ['c77', 'c78', 'c91'],\n",
       " 'NRNX1-seg-165': ['c101', 'c102', 'c115'],\n",
       " 'NRNX1-seg-166': ['c125', 'c126', 'c139'],\n",
       " 'NRNX1-seg-169': ['c149', 'c150', 'c163'],\n",
       " 'NRNX1-seg-170': ['c173', 'c174', 'c187'],\n",
       " 'NRNX1-seg-171': ['c195', 'c201', 'c210'],\n",
       " 'NRNX1-seg-172': ['c219', 'c225', 'c234'],\n",
       " 'NRNX1-seg-173': ['c1', 'c8', 'c10'],\n",
       " 'NRNX1-seg-174': ['c25', 'c32', 'c34'],\n",
       " 'NRNX1-seg-175': ['c49', 'c56', 'c58'],\n",
       " 'NRNX1-seg-176': ['c73', 'c80', 'c82'],\n",
       " 'NRNX1-seg-177': ['c97', 'c104', 'c106'],\n",
       " 'NRNX1-seg-178': ['c121', 'c128', 'c130'],\n",
       " 'NRNX1-seg-179': ['c145', 'c152', 'c154'],\n",
       " 'NRNX1-seg-180': ['c169', 'c176', 'c178'],\n",
       " 'NRNX1-seg-181': ['c193', 'c197', 'c199'],\n",
       " 'NRNX1-seg-182': ['c217', 'c221', 'c223'],\n",
       " 'NRNX1-seg-183': ['c2', 'c11', 'c16'],\n",
       " 'NRNX1-seg-184': ['c26', 'c35', 'c40'],\n",
       " 'NRNX1-seg-185': ['c50', 'c59', 'c64'],\n",
       " 'NRNX1-seg-186': ['c74', 'c83', 'c88'],\n",
       " 'NRNX1-seg-187': ['c98', 'c107', 'c112'],\n",
       " 'NRNX1-seg-188': ['c122', 'c131', 'c136'],\n",
       " 'NRNX1-seg-189': ['c146', 'c155', 'c160'],\n",
       " 'NRNX1-seg-190': ['c170', 'c179', 'c184'],\n",
       " 'NRNX1-seg-191': ['c196', 'c203', 'c215'],\n",
       " 'NRNX1-seg-192': ['c220', 'c227', 'c239'],\n",
       " 'NRNX1-seg-193': ['c9', 'c12', 'c13'],\n",
       " 'NRNX1-seg-194': ['c33', 'c36', 'c37'],\n",
       " 'NRNX1-seg-195': ['c57', 'c60', 'c61'],\n",
       " 'NRNX1-seg-196': ['c81', 'c84', 'c85'],\n",
       " 'NRNX1-seg-197': ['c105', 'c108', 'c109'],\n",
       " 'NRNX1-seg-198': ['c129', 'c132', 'c133'],\n",
       " 'NRNX1-seg-199': ['c153', 'c156', 'c157'],\n",
       " 'NRNX1-seg-200': ['c177', 'c180', 'c181'],\n",
       " 'NRNX1-seg-201': ['c194', 'c212', 'c213'],\n",
       " 'NRNX1-seg-202': ['c218', 'c236', 'c237'],\n",
       " 'NRNX1-seg-203': ['c3', 'c17', 'c21'],\n",
       " 'NRNX1-seg-204': ['c27', 'c41', 'c45'],\n",
       " 'NRNX1-seg-205': ['c51', 'c65', 'c69'],\n",
       " 'NRNX1-seg-206': ['c75', 'c89', 'c93'],\n",
       " 'NRNX1-seg-207': ['c99', 'c113', 'c117'],\n",
       " 'NRNX1-seg-208': ['c123', 'c137', 'c141'],\n",
       " 'NRNX1-seg-209': ['c147', 'c161', 'c165'],\n",
       " 'NRNX1-seg-210': ['c171', 'c185', 'c189'],\n",
       " 'NRNX1-seg-211': ['c198', 'c200', 'c201'],\n",
       " 'NRNX1-seg-212': ['c222', 'c224', 'c225'],\n",
       " 'NRNX1-seg-213': ['c8', 'c15', 'c19'],\n",
       " 'NRNX1-seg-214': ['c32', 'c39', 'c43'],\n",
       " 'NRNX1-seg-215': ['c56', 'c63', 'c67'],\n",
       " 'NRNX1-seg-216': ['c80', 'c87', 'c91'],\n",
       " 'NRNX1-seg-217': ['c104', 'c111', 'c115'],\n",
       " 'NRNX1-seg-218': ['c128', 'c135', 'c139'],\n",
       " 'NRNX1-seg-219': ['c152', 'c159', 'c163'],\n",
       " 'NRNX1-seg-220': ['c176', 'c183', 'c187'],\n",
       " 'NRNX1-seg-221': ['c206', 'c210', 'c214'],\n",
       " 'NRNX1-seg-222': ['c230', 'c234', 'c238'],\n",
       " 'NRNX1-seg-223': ['c7', 'c18', 'c20'],\n",
       " 'NRNX1-seg-224': ['c31', 'c42', 'c44'],\n",
       " 'NRNX1-seg-225': ['c55', 'c66', 'c68'],\n",
       " 'NRNX1-seg-226': ['c79', 'c90', 'c92'],\n",
       " 'NRNX1-seg-227': ['c103', 'c114', 'c116'],\n",
       " 'NRNX1-seg-228': ['c127', 'c138', 'c140'],\n",
       " 'NRNX1-seg-229': ['c151', 'c162', 'c164'],\n",
       " 'NRNX1-seg-230': ['c175', 'c186', 'c188'],\n",
       " 'NRNX1-seg-231': ['c195', 'c199', 'c204'],\n",
       " 'NRNX1-seg-232': ['c219', 'c223', 'c228'],\n",
       " 'NRNX1-seg-233': ['c5', 'c10', 'c13'],\n",
       " 'NRNX1-seg-234': ['c29', 'c34', 'c37'],\n",
       " 'NRNX1-seg-235': ['c53', 'c58', 'c61'],\n",
       " 'NRNX1-seg-236': ['c77', 'c82', 'c85'],\n",
       " 'NRNX1-seg-238': ['c101', 'c106', 'c109'],\n",
       " 'NRNX1-seg-239': ['c125', 'c130', 'c133'],\n",
       " 'NRNX1-seg-240': ['c149', 'c154', 'c157'],\n",
       " 'NRNX1-seg-241': ['c173', 'c178', 'c181'],\n",
       " 'NRNX1-seg-242': ['c196', 'c209', 'c211'],\n",
       " 'NRNX1-seg-244': ['c220', 'c233', 'c235'],\n",
       " 'NRNX1-seg-245': ['c1', 'c2', 'c4'],\n",
       " 'NRNX1-seg-246': ['c25', 'c26', 'c28'],\n",
       " 'NRNX1-seg-248': ['c49', 'c50', 'c52'],\n",
       " 'NRNX1-seg-249': ['c73', 'c74', 'c76'],\n",
       " 'NRNX1-seg-251': ['c97', 'c98', 'c100'],\n",
       " 'NRNX1-seg-252': ['c121', 'c122', 'c124'],\n",
       " 'NRNX1-seg-253': ['c145', 'c146', 'c148'],\n",
       " 'NRNX1-seg-254': ['c169', 'c170', 'c172'],\n",
       " 'NRNX1-seg-255': ['c192', 'c203', 'c207'],\n",
       " 'NRNX1-seg-256': ['c216', 'c227', 'c231'],\n",
       " 'NRNX1-seg-257': ['c0', 'c16', 'c22'],\n",
       " 'NRNX1-seg-258': ['c24', 'c40', 'c46'],\n",
       " 'NRNX1-seg-259': ['c48', 'c64', 'c70'],\n",
       " 'NRNX1-seg-260': ['c72', 'c88', 'c94'],\n",
       " 'NRNX1-seg-262': ['c96', 'c112', 'c118'],\n",
       " 'NRNX1-seg-263': ['c120', 'c136', 'c142'],\n",
       " 'NRNX1-seg-264': ['c144', 'c160', 'c166'],\n",
       " 'NRNX1-seg-265': ['c168', 'c184', 'c190'],\n",
       " 'NRNX1-seg-266': ['c205', 'c208', 'c212'],\n",
       " 'NRNX1-seg-267': ['c229', 'c232', 'c236'],\n",
       " 'NRNX1-seg-268': ['c6', 'c8', 'c9'],\n",
       " 'NRNX1-seg-269': ['c30', 'c32', 'c33'],\n",
       " 'NRNX1-seg-270': ['c54', 'c56', 'c57'],\n",
       " 'NRNX1-seg-271': ['c78', 'c80', 'c81'],\n",
       " 'NRNX1-seg-272': ['c102', 'c104', 'c105'],\n",
       " 'NRNX1-seg-273': ['c126', 'c128', 'c129'],\n",
       " 'NRNX1-seg-274': ['c150', 'c152', 'c153'],\n",
       " 'NRNX1-seg-275': ['c174', 'c176', 'c177'],\n",
       " 'NRNX1-seg-276': ['c201', 'c202', 'c213'],\n",
       " 'NRNX1-seg-277': ['c225', 'c226', 'c237'],\n",
       " 'NRNX1-seg-278': ['c10', 'c14', 'c15'],\n",
       " 'NRNX1-seg-279': ['c34', 'c38', 'c39'],\n",
       " 'NRNX1-seg-280': ['c58', 'c62', 'c63'],\n",
       " 'NRNX1-seg-281': ['c82', 'c86', 'c87'],\n",
       " 'NRNX1-seg-282': ['c106', 'c110', 'c111'],\n",
       " 'NRNX1-seg-283': ['c130', 'c134', 'c135'],\n",
       " 'NRNX1-seg-284': ['c154', 'c158', 'c159'],\n",
       " 'NRNX1-seg-285': ['c178', 'c182', 'c183'],\n",
       " 'NRNX1-seg-286': ['c197', 'c204', 'c215'],\n",
       " 'NRNX1-seg-287': ['c221', 'c228', 'c239'],\n",
       " 'NRNX1-seg-288': ['c5', 'c12', 'c23'],\n",
       " 'NRNX1-seg-289': ['c29', 'c36', 'c47'],\n",
       " 'NRNX1-seg-290': ['c53', 'c60', 'c71'],\n",
       " 'NRNX1-seg-291': ['c77', 'c84', 'c95'],\n",
       " 'NRNX1-seg-292': ['c101', 'c108', 'c119'],\n",
       " 'NRNX1-seg-293': ['c125', 'c132', 'c143'],\n",
       " 'NRNX1-seg-294': ['c149', 'c156', 'c167'],\n",
       " 'NRNX1-seg-295': ['c173', 'c180', 'c191'],\n",
       " 'NRNX1-seg-298': ['c194', 'c195', 'c211'],\n",
       " 'NRNX1-seg-299': ['c218', 'c219', 'c235'],\n",
       " 'NRNX1-seg-300': ['c7', 'c11', 'c17'],\n",
       " 'NRNX1-seg-301': ['c31', 'c35', 'c41'],\n",
       " 'NRNX1-seg-302': ['c55', 'c59', 'c65'],\n",
       " 'NRNX1-seg-303': ['c79', 'c83', 'c89'],\n",
       " 'NRNX1-seg-304': ['c103', 'c107', 'c113'],\n",
       " 'NRNX1-seg-305': ['c127', 'c131', 'c137'],\n",
       " 'NRNX1-seg-306': ['c151', 'c155', 'c161'],\n",
       " 'NRNX1-seg-307': ['c175', 'c179', 'c185'],\n",
       " 'NRNX1-seg-308': ['c193', 'c207', 'c210'],\n",
       " 'NRNX1-seg-311': ['c217', 'c231', 'c234'],\n",
       " 'NRNX1-seg-312': ['c2', 'c3', 'c19'],\n",
       " 'NRNX1-seg-313': ['c26', 'c27', 'c43'],\n",
       " 'NRNX1-seg-314': ['c50', 'c51', 'c67'],\n",
       " 'NRNX1-seg-315': ['c74', 'c75', 'c91'],\n",
       " 'NRNX1-seg-316': ['c98', 'c99', 'c115'],\n",
       " 'NRNX1-seg-317': ['c122', 'c123', 'c139'],\n",
       " 'NRNX1-seg-318': ['c146', 'c147', 'c163'],\n",
       " 'NRNX1-seg-319': ['c170', 'c171', 'c187'],\n",
       " 'NRNX1-seg-320': ['c199', 'c200', 'c214'],\n",
       " 'NRNX1-seg-321': ['c223', 'c224', 'c238'],\n",
       " 'NRNX1-seg-322': ['c4', 'c9', 'c22'],\n",
       " 'NRNX1-seg-323': ['c28', 'c33', 'c46'],\n",
       " 'NRNX1-seg-324': ['c52', 'c57', 'c70'],\n",
       " 'NRNX1-seg-325': ['c76', 'c81', 'c94'],\n",
       " 'NRNX1-seg-326': ['c100', 'c105', 'c118'],\n",
       " 'NRNX1-seg-327': ['c124', 'c129', 'c142'],\n",
       " 'NRNX1-seg-328': ['c148', 'c153', 'c166'],\n",
       " 'NRNX1-seg-329': ['c172', 'c177', 'c190'],\n",
       " 'NRNX1-seg-330': ['c192', 'c196', 'c198'],\n",
       " 'NRNX1-seg-331': ['c216', 'c220', 'c222'],\n",
       " 'NRNX1-seg-332': ['c6', 'c14', 'c21'],\n",
       " 'NRNX1-seg-333': ['c30', 'c38', 'c45'],\n",
       " 'NRNX1-seg-334': ['c54', 'c62', 'c69'],\n",
       " 'NRNX1-seg-335': ['c78', 'c86', 'c93'],\n",
       " 'NRNX1-seg-336': ['c102', 'c110', 'c117'],\n",
       " 'NRNX1-seg-337': ['c126', 'c134', 'c141'],\n",
       " 'NRNX1-seg-338': ['c150', 'c158', 'c165'],\n",
       " 'NRNX1-seg-339': ['c174', 'c182', 'c189'],\n",
       " 'NRNX1-seg-340': ['c197', 'c202', 'c205'],\n",
       " 'NRNX1-seg-341': ['c221', 'c226', 'c229'],\n",
       " 'NRNX1-seg-342': ['c11', 'c13', 'c18'],\n",
       " 'NRNX1-seg-343': ['c35', 'c37', 'c42'],\n",
       " 'NRNX1-seg-344': ['c59', 'c61', 'c66'],\n",
       " 'NRNX1-seg-345': ['c83', 'c85', 'c90'],\n",
       " 'NRNX1-seg-346': ['c107', 'c109', 'c114'],\n",
       " 'NRNX1-seg-347': ['c131', 'c133', 'c138'],\n",
       " 'NRNX1-seg-348': ['c155', 'c157', 'c162'],\n",
       " 'NRNX1-seg-349': ['c179', 'c181', 'c186'],\n",
       " 'NRNX1-seg-350': ['c210', 'c211', 'c215'],\n",
       " 'NRNX1-seg-351': ['c234', 'c235', 'c239'],\n",
       " 'NRNX1-seg-352': ['c1', 'c17', 'c23'],\n",
       " 'NRNX1-seg-353': ['c25', 'c41', 'c47'],\n",
       " 'NRNX1-seg-354': ['c49', 'c65', 'c71'],\n",
       " 'NRNX1-seg-355': ['c73', 'c89', 'c95'],\n",
       " 'NRNX1-seg-356': ['c97', 'c113', 'c119'],\n",
       " 'NRNX1-seg-357': ['c121', 'c137', 'c143'],\n",
       " 'NRNX1-seg-358': ['c145', 'c161', 'c167'],\n",
       " 'NRNX1-seg-359': ['c169', 'c185', 'c191'],\n",
       " 'NRNX1-seg-360': ['c200', 'c208', 'c213'],\n",
       " 'NRNX1-seg-361': ['c224', 'c232', 'c237'],\n",
       " 'NRNX1-seg-362': ['c0', 'c7', 'c9'],\n",
       " 'NRNX1-seg-363': ['c24', 'c31', 'c33'],\n",
       " 'NRNX1-seg-364': ['c48', 'c55', 'c57'],\n",
       " 'NRNX1-seg-365': ['c72', 'c79', 'c81'],\n",
       " 'NRNX1-seg-366': ['c96', 'c103', 'c105'],\n",
       " 'NRNX1-seg-367': ['c120', 'c127', 'c129'],\n",
       " 'NRNX1-seg-368': ['c144', 'c151', 'c153'],\n",
       " 'NRNX1-seg-369': ['c168', 'c175', 'c177'],\n",
       " 'NRNX1-seg-370': ['c198', 'c203', 'c204'],\n",
       " 'NRNX1-seg-371': ['c222', 'c227', 'c228'],\n",
       " 'NRNX1-seg-372': ['c4', 'c5', 'c15'],\n",
       " 'NRNX1-seg-373': ['c28', 'c29', 'c39'],\n",
       " 'NRNX1-seg-374': ['c52', 'c53', 'c63'],\n",
       " 'NRNX1-seg-375': ['c76', 'c77', 'c87'],\n",
       " 'NRNX1-seg-376': ['c100', 'c101', 'c111'],\n",
       " 'NRNX1-seg-379': ['c124', 'c125', 'c135'],\n",
       " 'NRNX1-seg-380': ['c148', 'c149', 'c159'],\n",
       " 'NRNX1-seg-381': ['c172', 'c173', 'c183'],\n",
       " 'NRNX1-seg-382': ['c193', 'c194', 'c196'],\n",
       " 'NRNX1-seg-383': ['c217', 'c218', 'c220'],\n",
       " 'NRNX1-seg-384': ['c8', 'c16', 'c21'],\n",
       " 'NRNX1-seg-385': ['c32', 'c40', 'c45'],\n",
       " 'NRNX1-seg-386': ['c56', 'c64', 'c69'],\n",
       " 'NRNX1-seg-387': ['c80', 'c88', 'c93'],\n",
       " 'NRNX1-seg-388': ['c104', 'c112', 'c117'],\n",
       " 'NRNX1-seg-389': ['c128', 'c136', 'c141'],\n",
       " 'NRNX1-seg-390': ['c152', 'c160', 'c165'],\n",
       " 'NRNX1-seg-391': ['c176', 'c184', 'c189'],\n",
       " 'NRNX1-seg-392': ['c197', 'c201', 'c209'],\n",
       " 'NRNX1-seg-393': ['c221', 'c225', 'c233'],\n",
       " 'NRNX1-seg-394': ['c2', 'c10', 'c22'],\n",
       " 'NRNX1-seg-395': ['c26', 'c34', 'c46'],\n",
       " 'NRNX1-seg-396': ['c50', 'c58', 'c70'],\n",
       " 'NRNX1-seg-397': ['c74', 'c82', 'c94'],\n",
       " 'NRNX1-seg-398': ['c98', 'c106', 'c118'],\n",
       " 'NRNX1-seg-399': ['c122', 'c130', 'c142'],\n",
       " 'NRNX1-seg-400': ['c146', 'c154', 'c166'],\n",
       " 'NRNX1-seg-401': ['c170', 'c178', 'c190'],\n",
       " 'NRNX1-seg-402': ['c195', 'c207', 'c212'],\n",
       " 'NRNX1-seg-403': ['c219', 'c231', 'c236'],\n",
       " 'NRNX1-seg-404': ['c18', 'c19', 'c23'],\n",
       " 'NRNX1-seg-405': ['c42', 'c43', 'c47'],\n",
       " 'NRNX1-seg-406': ['c66', 'c67', 'c71'],\n",
       " 'NRNX1-seg-407': ['c90', 'c91', 'c95'],\n",
       " 'NRNX1-seg-408': ['c114', 'c115', 'c119'],\n",
       " 'NRNX1-seg-409': ['c138', 'c139', 'c143'],\n",
       " 'NRNX1-seg-410': ['c162', 'c163', 'c167'],\n",
       " 'NRNX1-seg-411': ['c186', 'c187', 'c191'],\n",
       " 'NRNX1-seg-412': ['c198', 'c206', 'c213'],\n",
       " 'NRNX1-seg-413': ['c222', 'c230', 'c237'],\n",
       " 'NRNX1-seg-414': ['c3', 'c7', 'c12'],\n",
       " 'NRNX1-seg-415': ['c27', 'c31', 'c36'],\n",
       " 'NRNX1-seg-416': ['c51', 'c55', 'c60'],\n",
       " 'NRNX1-seg-417': ['c75', 'c79', 'c84'],\n",
       " 'NRNX1-seg-418': ['c99', 'c103', 'c108'],\n",
       " 'NRNX1-seg-419': ['c123', 'c127', 'c132'],\n",
       " 'NRNX1-seg-420': ['c147', 'c151', 'c156'],\n",
       " 'NRNX1-seg-421': ['c171', 'c175', 'c180'],\n",
       " 'NRNX1-seg-422': ['c192', 'c208', 'c214'],\n",
       " 'NRNX1-seg-423': ['c216', 'c232', 'c238'],\n",
       " 'NRNX1-seg-424': ['c0', 'c4', 'c6'],\n",
       " 'NRNX1-seg-425': ['c24', 'c28', 'c30'],\n",
       " 'NRNX1-seg-426': ['c48', 'c52', 'c54'],\n",
       " 'NRNX1-seg-427': ['c72', 'c76', 'c78'],\n",
       " 'NRNX1-seg-428': ['c96', 'c100', 'c102'],\n",
       " 'NRNX1-seg-429': ['c120', 'c124', 'c126'],\n",
       " 'NRNX1-seg-430': ['c144', 'c148', 'c150'],\n",
       " 'NRNX1-seg-431': ['c168', 'c172', 'c174'],\n",
       " 'NRNX1-seg-432': ['c203', 'c205', 'c210'],\n",
       " 'NRNX1-seg-433': ['c227', 'c229', 'c234'],\n",
       " 'NRNX1-seg-434': ['c13', 'c15', 'c21'],\n",
       " 'NRNX1-seg-435': ['c37', 'c39', 'c45'],\n",
       " 'NRNX1-seg-436': ['c61', 'c63', 'c69'],\n",
       " 'NRNX1-seg-437': ['c85', 'c87', 'c93'],\n",
       " 'NRNX1-seg-438': ['c109', 'c111', 'c117'],\n",
       " 'NRNX1-seg-439': ['c133', 'c135', 'c141'],\n",
       " 'NRNX1-seg-440': ['c157', 'c159', 'c165'],\n",
       " 'NRNX1-seg-441': ['c181', 'c183', 'c189'],\n",
       " 'NRNX1-seg-442': ['c193', 'c209', 'c215'],\n",
       " 'NRNX1-seg-443': ['c217', 'c233', 'c239'],\n",
       " 'NRNX1-seg-444': ['c1', 'c14', 'c20'],\n",
       " 'NRNX1-seg-445': ['c25', 'c38', 'c44'],\n",
       " 'NRNX1-seg-446': ['c49', 'c62', 'c68'],\n",
       " 'NRNX1-seg-447': ['c73', 'c86', 'c92'],\n",
       " 'NRNX1-seg-448': ['c97', 'c110', 'c116'],\n",
       " 'NRNX1-seg-449': ['c121', 'c134', 'c140'],\n",
       " 'NRNX1-seg-450': ['c145', 'c158', 'c164'],\n",
       " 'NRNX1-seg-452': ['c169', 'c182', 'c188'],\n",
       " 'NRNX1-seg-453': ['c194', 'c201', 'c207'],\n",
       " 'NRNX1-seg-454': ['c218', 'c225', 'c231'],\n",
       " 'NRNX1-seg-455': ['c10', 'c16', 'c17'],\n",
       " 'NRNX1-seg-456': ['c34', 'c40', 'c41'],\n",
       " 'NRNX1-seg-457': ['c58', 'c64', 'c65'],\n",
       " 'NRNX1-seg-458': ['c82', 'c88', 'c89'],\n",
       " 'NRNX1-seg-459': ['c106', 'c112', 'c113'],\n",
       " 'NRNX1-seg-460': ['c130', 'c136', 'c137'],\n",
       " 'NRNX1-seg-461': ['c154', 'c160', 'c161'],\n",
       " 'NRNX1-seg-462': ['c178', 'c184', 'c185'],\n",
       " 'NRNX1-seg-463': ['c195', 'c200', 'c206'],\n",
       " 'NRNX1-seg-464': ['c219', 'c224', 'c230'],\n",
       " 'NRNX1-seg-465': ['c5', 'c11', 'c22'],\n",
       " 'NRNX1-seg-466': ['c29', 'c35', 'c46'],\n",
       " 'NRNX1-seg-467': ['c53', 'c59', 'c70'],\n",
       " 'NRNX1-seg-468': ['c77', 'c83', 'c94'],\n",
       " 'NRNX1-seg-469': ['c101', 'c107', 'c118'],\n",
       " 'NRNX1-seg-470': ['c125', 'c131', 'c142'],\n",
       " 'NRNX1-seg-471': ['c149', 'c155', 'c166'],\n",
       " 'NRNX1-seg-472': ['c173', 'c179', 'c190'],\n",
       " 'NRNX1-seg-474': ['c197', 'c198', 'c211'],\n",
       " 'NRNX1-seg-475': ['c221', 'c222', 'c235'],\n",
       " 'NRNX1-seg-476': ['c3', 'c9', 'c18'],\n",
       " 'NRNX1-seg-477': ['c27', 'c33', 'c42'],\n",
       " 'NRNX1-seg-478': ['c51', 'c57', 'c66'],\n",
       " 'NRNX1-seg-479': ['c75', 'c81', 'c90'],\n",
       " 'NRNX1-seg-480': ['c99', 'c105', 'c114'],\n",
       " 'NRNX1-seg-481': ['c123', 'c129', 'c138'],\n",
       " 'NRNX1-seg-482': ['c147', 'c153', 'c162'],\n",
       " 'NRNX1-seg-483': ['c171', 'c177', 'c186'],\n",
       " 'NRNX1-seg-484': ['c202', 'c208', 'c209'],\n",
       " 'NRNX1-seg-485': ['c226', 'c232', 'c233'],\n",
       " 'NRNX1-seg-486': ['c8', 'c20', 'c23'],\n",
       " 'NRNX1-seg-487': ['c32', 'c44', 'c47'],\n",
       " 'NRNX1-seg-488': ['c56', 'c68', 'c71'],\n",
       " 'NRNX1-seg-489': ['c80', 'c92', 'c95'],\n",
       " 'NRNX1-seg-490': ['c104', 'c116', 'c119'],\n",
       " 'NRNX1-seg-491': ['c128', 'c140', 'c143'],\n",
       " 'NRNX1-seg-492': ['c152', 'c164', 'c167'],\n",
       " 'NRNX1-seg-493': ['c176', 'c188', 'c191'],\n",
       " 'NRNX1-seg-494': ['c199', 'c210', 'c212'],\n",
       " 'NRNX1-seg-495': ['c223', 'c234', 'c236'],\n",
       " 'NRNX1-seg-496': ['c0', 'c12', 'c21'],\n",
       " 'NRNX1-seg-497': ['c24', 'c36', 'c45'],\n",
       " 'NRNX1-seg-498': ['c48', 'c60', 'c69'],\n",
       " 'NRNX1-seg-499': ['c72', 'c84', 'c93'],\n",
       " 'NRNX1-seg-500': ['c96', 'c108', 'c117'],\n",
       " 'NRNX1-seg-501': ['c120', 'c132', 'c141'],\n",
       " 'NRNX1-seg-502': ['c144', 'c156', 'c165'],\n",
       " 'NRNX1-seg-503': ['c168', 'c180', 'c189'],\n",
       " 'NRNX1-seg-504': ['c193', 'c205', 'c214'],\n",
       " 'NRNX1-seg-505': ['c217', 'c229', 'c238'],\n",
       " 'NRNX1-seg-506': ['c5', 'c14', 'c16'],\n",
       " 'NRNX1-seg-507': ['c29', 'c38', 'c40'],\n",
       " 'NRNX1-seg-508': ['c53', 'c62', 'c64'],\n",
       " 'NRNX1-seg-509': ['c77', 'c86', 'c88'],\n",
       " 'NRNX1-seg-510': ['c101', 'c110', 'c112'],\n",
       " 'NRNX1-seg-511': ['c125', 'c134', 'c136'],\n",
       " 'NRNX1-seg-512': ['c149', 'c158', 'c160'],\n",
       " 'NRNX1-seg-513': ['c173', 'c182', 'c184'],\n",
       " 'NRNX1-seg-514': ['c194', 'c200', 'c204'],\n",
       " 'NRNX1-seg-515': ['c218', 'c224', 'c228'],\n",
       " 'NRNX1-seg-516': ['c4', 'c17', 'c19'],\n",
       " 'NRNX1-seg-517': ['c28', 'c41', 'c43'],\n",
       " 'NRNX1-seg-518': ['c52', 'c65', 'c67'],\n",
       " 'NRNX1-seg-519': ['c76', 'c89', 'c91'],\n",
       " 'NRNX1-seg-520': ['c100', 'c113', 'c115'],\n",
       " 'NRNX1-seg-521': ['c124', 'c137', 'c139'],\n",
       " 'NRNX1-seg-522': ['c148', 'c161', 'c163'],\n",
       " 'NRNX1-seg-523': ['c172', 'c185', 'c187'],\n",
       " 'NRNX1-seg-524': ['c196', 'c197', 'c207'],\n",
       " 'NRNX1-seg-525': ['c220', 'c221', 'c231'],\n",
       " 'NRNX1-seg-526': ['c6', 'c15', 'c22'],\n",
       " 'NRNX1-seg-527': ['c30', 'c39', 'c46'],\n",
       " 'NRNX1-seg-528': ['c54', 'c63', 'c70'],\n",
       " 'NRNX1-seg-529': ['c78', 'c87', 'c94'],\n",
       " 'NRNX1-seg-530': ['c102', 'c111', 'c118'],\n",
       " 'NRNX1-seg-531': ['c126', 'c135', 'c142'],\n",
       " 'NRNX1-seg-532': ['c150', 'c159', 'c166'],\n",
       " 'NRNX1-seg-533': ['c174', 'c183', 'c190'],\n",
       " 'NRNX1-seg-534': ['c192', 'c206', 'c209'],\n",
       " 'NRNX1-seg-535': ['c216', 'c230', 'c233'],\n",
       " 'NRNX1-seg-536': ['c3', 'c10', 'c11'],\n",
       " 'NRNX1-seg-537': ['c27', 'c34', 'c35'],\n",
       " 'NRNX1-seg-538': ['c51', 'c58', 'c59'],\n",
       " 'NRNX1-seg-539': ['c75', 'c82', 'c83'],\n",
       " 'NRNX1-seg-540': ['c99', 'c106', 'c107'],\n",
       " 'NRNX1-seg-541': ['c123', 'c130', 'c131'],\n",
       " 'NRNX1-seg-542': ['c147', 'c154', 'c155'],\n",
       " 'NRNX1-seg-543': ['c171', 'c178', 'c179'],\n",
       " 'NRNX1-seg-544': ['c199', 'c211', 'c213'],\n",
       " 'NRNX1-seg-545': ['c223', 'c235', 'c237'],\n",
       " 'NRNX1-seg-546': ['c0', 'c2', 'c23'],\n",
       " 'NRNX1-seg-547': ['c24', 'c26', 'c47'],\n",
       " 'NRNX1-seg-548': ['c48', 'c50', 'c71'],\n",
       " 'NRNX1-seg-549': ['c72', 'c74', 'c95'],\n",
       " 'NRNX1-seg-550': ['c96', 'c98', 'c119'],\n",
       " 'NRNX1-seg-551': ['c120', 'c122', 'c143'],\n",
       " 'NRNX1-seg-552': ['c144', 'c146', 'c167'],\n",
       " 'NRNX1-seg-553': ['c168', 'c170', 'c191'],\n",
       " 'NRNX1-seg-554': ['c198', 'c208', 'c210'],\n",
       " 'NRNX1-seg-555': ['c222', 'c232', 'c234'],\n",
       " 'NRNX1-seg-556': ['c13', 'c16', 'c20'],\n",
       " 'NRNX1-seg-557': ['c37', 'c40', 'c44'],\n",
       " 'NRNX1-seg-558': ['c61', 'c64', 'c68'],\n",
       " 'NRNX1-seg-559': ['c85', 'c88', 'c92'],\n",
       " 'NRNX1-seg-560': ['c109', 'c112', 'c116'],\n",
       " 'NRNX1-seg-561': ['c133', 'c136', 'c140'],\n",
       " 'NRNX1-seg-562': ['c157', 'c160', 'c164'],\n",
       " 'NRNX1-seg-563': ['c181', 'c184', 'c188'],\n",
       " 'NRNX1-seg-564': ['c194', 'c202', 'c214'],\n",
       " 'NRNX1-seg-565': ['c218', 'c226', 'c238'],\n",
       " 'NRNX1-seg-566': ['c7', 'c8', 'c22'],\n",
       " 'NRNX1-seg-567': ['c31', 'c32', 'c46'],\n",
       " 'NRNX1-seg-568': ['c55', 'c56', 'c70'],\n",
       " 'NRNX1-seg-569': ['c79', 'c80', 'c94'],\n",
       " 'NRNX1-seg-570': ['c103', 'c104', 'c118'],\n",
       " 'NRNX1-seg-571': ['c127', 'c128', 'c142'],\n",
       " 'NRNX1-seg-572': ['c151', 'c152', 'c166'],\n",
       " 'NRNX1-seg-573': ['c175', 'c176', 'c190'],\n",
       " 'NRNX1-seg-574': ['c196', 'c204', 'c212'],\n",
       " 'NRNX1-seg-575': ['c220', 'c228', 'c236'],\n",
       " 'NRNX1-seg-576': ['c5', 'c18', 'c21'],\n",
       " 'NRNX1-seg-577': ['c29', 'c42', 'c45'],\n",
       " 'NRNX1-seg-578': ['c53', 'c66', 'c69'],\n",
       " 'NRNX1-seg-579': ['c77', 'c90', 'c93'],\n",
       " 'NRNX1-seg-580': ['c101', 'c114', 'c117'],\n",
       " 'NRNX1-seg-581': ['c125', 'c138', 'c141'],\n",
       " 'NRNX1-seg-582': ['c149', 'c162', 'c165'],\n",
       " 'NRNX1-seg-583': ['c173', 'c186', 'c189'],\n",
       " 'NRNX1-seg-584': ['c199', 'c203', 'c209'],\n",
       " 'NRNX1-seg-585': ['c223', 'c227', 'c233'],\n",
       " 'NRNX1-seg-586': ['c0', 'c1', 'c3'],\n",
       " 'NRNX1-seg-587': ['c24', 'c25', 'c27'],\n",
       " 'NRNX1-seg-588': ['c48', 'c49', 'c51'],\n",
       " 'NRNX1-seg-589': ['c72', 'c73', 'c75'],\n",
       " 'NRNX1-seg-590': ['c96', 'c97', 'c99'],\n",
       " 'NRNX1-seg-591': ['c120', 'c121', 'c123'],\n",
       " 'NRNX1-seg-592': ['c144', 'c145', 'c147'],\n",
       " 'NRNX1-seg-593': ['c168', 'c169', 'c171'],\n",
       " 'NRNX1-seg-594': ['c192', 'c193', 'c195'],\n",
       " 'NRNX1-seg-595': ['c216', 'c217', 'c219'],\n",
       " 'NRNX1-seg-596': ['c4', 'c12', 'c20'],\n",
       " 'NRNX1-seg-597': ['c28', 'c36', 'c44'],\n",
       " 'NRNX1-seg-598': ['c52', 'c60', 'c68'],\n",
       " 'NRNX1-seg-599': ['c76', 'c84', 'c92'],\n",
       " 'NRNX1-seg-600': ['c100', 'c108', 'c116'],\n",
       " 'NRNX1-seg-601': ['c124', 'c132', 'c140'],\n",
       " 'NRNX1-seg-602': ['c148', 'c156', 'c164'],\n",
       " 'NRNX1-seg-603': ['c172', 'c180', 'c188'],\n",
       " 'NRNX1-seg-604': ['c197', 'c206', 'c208'],\n",
       " 'NRNX1-seg-605': ['c221', 'c230', 'c232'],\n",
       " 'NRNX1-seg-606': ['c2', 'c6', 'c13'],\n",
       " 'NRNX1-seg-607': ['c26', 'c30', 'c37'],\n",
       " 'NRNX1-seg-608': ['c50', 'c54', 'c61'],\n",
       " 'NRNX1-seg-609': ['c74', 'c78', 'c85'],\n",
       " 'NRNX1-seg-610': ['c98', 'c102', 'c109'],\n",
       " 'NRNX1-seg-611': ['c122', 'c126', 'c133'],\n",
       " 'NRNX1-seg-612': ['c146', 'c150', 'c157'],\n",
       " 'NRNX1-seg-613': ['c170', 'c174', 'c181'],\n",
       " 'NRNX1-seg-614': ['c201', 'c204', 'c205'],\n",
       " 'NRNX1-seg-615': ['c225', 'c228', 'c229'],\n",
       " 'NRNX1-seg-616': ['c11', 'c14', 'c19'],\n",
       " 'NRNX1-seg-617': ['c35', 'c38', 'c43'],\n",
       " 'NRNX1-seg-618': ['c59', 'c62', 'c67'],\n",
       " 'NRNX1-seg-619': ['c83', 'c86', 'c91'],\n",
       " 'NRNX1-seg-620': ['c107', 'c110', 'c115'],\n",
       " 'NRNX1-seg-621': ['c131', 'c134', 'c139'],\n",
       " 'NRNX1-seg-622': ['c155', 'c158', 'c163'],\n",
       " 'NRNX1-seg-623': ['c179', 'c182', 'c187'],\n",
       " 'NRNX1-seg-624': ['c213', 'c214', 'c215'],\n",
       " 'NRNX1-seg-625': ['c237', 'c238', 'c239'],\n",
       " 'NRNX1-seg-626': ['c0', 'c8', 'c18'],\n",
       " 'NRNX1-seg-627': ['c24', 'c32', 'c42'],\n",
       " 'NRNX1-seg-628': ['c48', 'c56', 'c66'],\n",
       " 'NRNX1-seg-629': ['c72', 'c80', 'c90'],\n",
       " 'NRNX1-seg-630': ['c96', 'c104', 'c114'],\n",
       " 'NRNX1-seg-631': ['c120', 'c128', 'c138'],\n",
       " 'NRNX1-seg-632': ['c144', 'c152', 'c162'],\n",
       " 'NRNX1-seg-633': ['c168', 'c176', 'c186'],\n",
       " 'NRNX1-seg-634': ['c196', 'c202', 'c210'],\n",
       " 'NRNX1-seg-635': ['c220', 'c226', 'c234'],\n",
       " 'NRNX1-seg-636': ['c9', 'c10', 'c21'],\n",
       " 'NRNX1-seg-637': ['c33', 'c34', 'c45'],\n",
       " 'NRNX1-seg-639': ['c57', 'c58', 'c69'],\n",
       " 'NRNX1-seg-640': ['c81', 'c82', 'c93'],\n",
       " 'NRNX1-seg-641': ['c105', 'c106', 'c117'],\n",
       " 'NRNX1-seg-642': ['c129', 'c130', 'c141'],\n",
       " 'NRNX1-seg-643': ['c153', 'c154', 'c165'],\n",
       " 'NRNX1-seg-644': ['c177', 'c178', 'c189'],\n",
       " 'NRNX1-seg-645': ['c200', 'c207', 'c211'],\n",
       " 'NRNX1-seg-646': ['c224', 'c231', 'c235'],\n",
       " 'NRNX1-seg-647': ['c1', 'c5', 'c7'],\n",
       " 'NRNX1-seg-648': ['c25', 'c29', 'c31'],\n",
       " 'NRNX1-seg-649': ['c49', 'c53', 'c55'],\n",
       " 'NRNX1-seg-650': ['c73', 'c77', 'c79'],\n",
       " 'NRNX1-seg-651': ['c97', 'c101', 'c103'],\n",
       " 'NRNX1-seg-652': ['c121', 'c125', 'c127'],\n",
       " 'NRNX1-seg-653': ['c145', 'c149', 'c151'],\n",
       " 'NRNX1-seg-654': ['c169', 'c173', 'c175'],\n",
       " 'NRNX1-seg-655': ['c201', 'c203', 'c212'],\n",
       " 'NRNX1-seg-656': ['c225', 'c227', 'c236'],\n",
       " 'NRNX1-seg-657': ['c6', 'c17', 'c20'],\n",
       " 'NRNX1-seg-658': ['c30', 'c41', 'c44'],\n",
       " 'NRNX1-seg-659': ['c54', 'c65', 'c68'],\n",
       " 'NRNX1-seg-660': ['c78', 'c89', 'c92'],\n",
       " 'NRNX1-seg-661': ['c102', 'c113', 'c116'],\n",
       " 'NRNX1-seg-662': ['c126', 'c137', 'c140'],\n",
       " 'NRNX1-seg-665': ['c150', 'c161', 'c164'],\n",
       " 'NRNX1-seg-666': ['c174', 'c185', 'c188'],\n",
       " 'NRNX1-seg-667': ['c194', 'c199', 'c206'],\n",
       " 'NRNX1-seg-668': ['c218', 'c223', 'c230'],\n",
       " 'NRNX1-seg-669': ['c2', 'c8', 'c12'],\n",
       " 'NRNX1-seg-670': ['c26', 'c32', 'c36'],\n",
       " 'NRNX1-seg-671': ['c50', 'c56', 'c60'],\n",
       " 'NRNX1-seg-672': ['c74', 'c80', 'c84'],\n",
       " 'NRNX1-seg-674': ['c98', 'c104', 'c108'],\n",
       " 'NRNX1-seg-675': ['c122', 'c128', 'c132'],\n",
       " 'NRNX1-seg-676': ['c146', 'c152', 'c156'],\n",
       " 'NRNX1-seg-678': ['c170', 'c176', 'c180'],\n",
       " 'NRNX1-seg-679': ['c198', 'c202', 'c215'],\n",
       " 'NRNX1-seg-681': ['c222', 'c226', 'c239'],\n",
       " 'NRNX1-seg-682': ['c4', 'c13', 'c14'],\n",
       " 'NRNX1-seg-683': ['c28', 'c37', 'c38'],\n",
       " 'NRNX1-seg-684': ['c52', 'c61', 'c62'],\n",
       " 'NRNX1-seg-685': ['c76', 'c85', 'c86'],\n",
       " 'NRNX1-seg-686': ['c100', 'c109', 'c110'],\n",
       " 'NRNX1-seg-687': ['c124', 'c133', 'c134'],\n",
       " 'NRNX1-seg-688': ['c148', 'c157', 'c158'],\n",
       " 'NRNX1-seg-689': ['c172', 'c181', 'c182'],\n",
       " 'NRNX1-seg-690': ['c192', 'c204', 'c213'],\n",
       " 'NRNX1-seg-691': ['c216', 'c228', 'c237'],\n",
       " 'NRNX1-seg-692': ['c15', 'c16', 'c23'],\n",
       " 'NRNX1-seg-693': ['c39', 'c40', 'c47'],\n",
       " 'NRNX1-seg-694': ['c63', 'c64', 'c71'],\n",
       " 'NRNX1-seg-695': ['c87', 'c88', 'c95'],\n",
       " 'NRNX1-seg-696': ['c111', 'c112', 'c119'],\n",
       " 'NRNX1-seg-697': ['c135', 'c136', 'c143'],\n",
       " 'NRNX1-seg-698': ['c159', 'c160', 'c167'],\n",
       " 'NRNX1-seg-699': ['c183', 'c184', 'c191']}"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "region_2_readout_dict"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a4dd2783",
   "metadata": {},
   "source": [
    "<a id='3.2'></a>\n",
    "## 3.2 loading components"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8e899029",
   "metadata": {},
   "source": [
    "### primers\n",
    "pick primer set 0,1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "a69492f8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Picked primer: ID: W1A01_primer_0\n",
      "Name: W1A01_primer_0\n",
      "Description: W1A01_primer_0\n",
      "Number of features: 0\n",
      "Seq('CGGCTCGCAGCGTGTAAACG')\n",
      "- Picked primer: ID: W1A02_primer_1\n",
      "Name: W1A02_primer_1\n",
      "Description: W1A02_primer_1\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGCATTTCAGGATCACCGGCGG')\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "primer_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Primers'\n",
    "primers = library_tools.assemble.load_primers([0,1], _primer_folder=primer_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3dfa7962",
   "metadata": {},
   "source": [
    "### readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "152a93c4",
   "metadata": {},
   "outputs": [],
   "source": [
    "# collect types from region_2_readout_dict\n",
    "readout_usage_by_type = {}\n",
    "for _region, _infos in region_2_readout_dict.items():\n",
    "    for _info in _infos:\n",
    "        if _info[0] not in readout_usage_by_type:\n",
    "            readout_usage_by_type[_info[0]] = []\n",
    "        if int(_info[1:]) not in readout_usage_by_type[_info[0]]:\n",
    "            readout_usage_by_type[_info[0]].append(int(_info[1:]))\n",
    "\n",
    "readout_usage_dict = {_type:len(_v) for _type,_v in readout_usage_by_type.items()}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "id": "54fa06dc",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\selected_NDB_sets\\highest_snr_750.fasta\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\selected_NDB_sets\\highest_snr_647.fasta\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\selected_NDB_sets\\highest_snr_561.fasta\n",
      "[80, 80, 80]\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3.classes import _allowed_kwds\n",
    "reload(ia.library_tools.assemble)\n",
    "readout_dict ={}\n",
    "overwrite_readout_usage = True\n",
    "readout_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\selected_NDB_sets'\n",
    "\n",
    "# combo\n",
    "readout_type = 'combo'\n",
    "# \n",
    "_type_char = _allowed_kwds[readout_type]\n",
    "_readout_start = 0\n",
    "_num_color = 3\n",
    "\n",
    "readout_dict[_type_char] = library_tools.assemble.load_readouts(readout_usage_dict[_type_char], \n",
    "                                                       _type='highest_snr', \n",
    "                                                       _readout_folder=readout_folder, \n",
    "                                                       _num_colors=_num_color, # 750,647,561\n",
    "                                                       _start_id=_readout_start,\n",
    "                                                       )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "dd5038ee",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['NDB_784',\n",
       " 'NDB_755',\n",
       " 'NDB_759',\n",
       " 'NDB_826',\n",
       " 'NDB_578',\n",
       " 'NDB_795',\n",
       " 'NDB_376',\n",
       " 'NDB_713',\n",
       " 'NDB_777',\n",
       " 'NDB_865',\n",
       " 'NDB_533',\n",
       " 'NDB_801',\n",
       " 'NDB_817',\n",
       " 'NDB_725',\n",
       " 'NDB_807',\n",
       " 'NDB_652',\n",
       " 'NDB_710',\n",
       " 'NDB_690',\n",
       " 'NDB_718',\n",
       " 'NDB_971',\n",
       " 'NDB_699',\n",
       " 'NDB_847',\n",
       " 'NDB_743',\n",
       " 'NDB_936',\n",
       " 'NDB_643',\n",
       " 'NDB_683',\n",
       " 'NDB_456',\n",
       " 'NDB_760',\n",
       " 'NDB_815',\n",
       " 'NDB_738',\n",
       " 'NDB_400',\n",
       " 'NDB_965',\n",
       " 'NDB_750',\n",
       " 'NDB_790',\n",
       " 'NDB_464',\n",
       " 'NDB_825',\n",
       " 'NDB_778',\n",
       " 'NDB_431',\n",
       " 'NDB_768',\n",
       " 'NDB_844',\n",
       " 'NDB_554',\n",
       " 'NDB_852',\n",
       " 'NDB_961',\n",
       " 'NDB_917',\n",
       " 'NDB_762',\n",
       " 'NDB_661',\n",
       " 'NDB_728',\n",
       " 'NDB_804',\n",
       " 'NDB_568',\n",
       " 'NDB_458',\n",
       " 'NDB_915',\n",
       " 'NDB_466',\n",
       " 'NDB_974',\n",
       " 'NDB_387',\n",
       " 'NDB_511',\n",
       " 'NDB_746',\n",
       " 'NDB_621',\n",
       " 'NDB_901',\n",
       " 'NDB_875',\n",
       " 'NDB_375',\n",
       " 'NDB_487',\n",
       " 'NDB_773',\n",
       " 'NDB_516',\n",
       " 'NDB_409',\n",
       " 'NDB_812',\n",
       " 'NDB_432',\n",
       " 'NDB_571',\n",
       " 'NDB_716',\n",
       " 'NDB_786',\n",
       " 'NDB_448',\n",
       " 'NDB_860',\n",
       " 'NDB_858',\n",
       " 'NDB_868',\n",
       " 'NDB_680',\n",
       " 'NDB_957',\n",
       " 'NDB_529',\n",
       " 'NDB_443',\n",
       " 'NDB_465',\n",
       " 'NDB_508',\n",
       " 'NDB_455',\n",
       " 'NDB_489',\n",
       " 'NDB_403',\n",
       " 'NDB_470',\n",
       " 'NDB_417',\n",
       " 'NDB_1027',\n",
       " 'NDB_737',\n",
       " 'NDB_657',\n",
       " 'NDB_754',\n",
       " 'NDB_698',\n",
       " 'NDB_1020',\n",
       " 'NDB_856',\n",
       " 'NDB_863',\n",
       " 'NDB_681',\n",
       " 'NDB_391',\n",
       " 'NDB_878',\n",
       " 'NDB_438',\n",
       " 'NDB_634',\n",
       " 'NDB_332',\n",
       " 'NDB_405',\n",
       " 'NDB_715',\n",
       " 'NDB_758',\n",
       " 'NDB_1035',\n",
       " 'NDB_883',\n",
       " 'NDB_536',\n",
       " 'NDB_663',\n",
       " 'NDB_1033',\n",
       " 'NDB_461',\n",
       " 'NDB_954',\n",
       " 'NDB_631',\n",
       " 'NDB_653',\n",
       " 'NDB_771',\n",
       " 'NDB_532',\n",
       " 'NDB_398',\n",
       " 'NDB_741',\n",
       " 'NDB_616',\n",
       " 'NDB_905',\n",
       " 'NDB_444',\n",
       " 'NDB_1075',\n",
       " 'NDB_467',\n",
       " 'NDB_789',\n",
       " 'NDB_388',\n",
       " 'NDB_335',\n",
       " 'NDB_390',\n",
       " 'NDB_1060',\n",
       " 'NDB_866',\n",
       " 'NDB_921',\n",
       " 'NDB_427',\n",
       " 'NDB_887',\n",
       " 'NDB_501',\n",
       " 'NDB_619',\n",
       " 'NDB_506',\n",
       " 'NDB_780',\n",
       " 'NDB_805',\n",
       " 'NDB_347',\n",
       " 'NDB_615',\n",
       " 'NDB_730',\n",
       " 'NDB_752',\n",
       " 'NDB_507',\n",
       " 'NDB_880',\n",
       " 'NDB_482',\n",
       " 'NDB_747',\n",
       " 'NDB_832',\n",
       " 'NDB_446',\n",
       " 'NDB_1047',\n",
       " 'NDB_835',\n",
       " 'NDB_473',\n",
       " 'NDB_846',\n",
       " 'NDB_433',\n",
       " 'NDB_539',\n",
       " 'NDB_864',\n",
       " 'NDB_397',\n",
       " 'NDB_896',\n",
       " 'NDB_813',\n",
       " 'NDB_946',\n",
       " 'NDB_989',\n",
       " 'NDB_630',\n",
       " 'NDB_721',\n",
       " 'NDB_641',\n",
       " 'NDB_453',\n",
       " 'NDB_853',\n",
       " 'NDB_899',\n",
       " 'NDB_816',\n",
       " 'NDB_505',\n",
       " 'NDB_941',\n",
       " 'NDB_1044',\n",
       " 'NDB_385',\n",
       " 'NDB_452',\n",
       " 'NDB_618',\n",
       " 'NDB_838',\n",
       " 'NDB_647',\n",
       " 'NDB_687',\n",
       " 'NDB_994',\n",
       " 'NDB_881',\n",
       " 'NDB_498',\n",
       " 'NDB_514',\n",
       " 'NDB_356',\n",
       " 'NDB_1023',\n",
       " 'NDB_1066',\n",
       " 'NDB_665',\n",
       " 'NDB_396',\n",
       " 'NDB_637',\n",
       " 'NDB_734',\n",
       " 'NDB_828',\n",
       " 'NDB_445',\n",
       " 'NDB_935',\n",
       " 'NDB_660',\n",
       " 'NDB_706',\n",
       " 'NDB_788',\n",
       " 'NDB_711',\n",
       " 'NDB_889',\n",
       " 'NDB_587',\n",
       " 'NDB_819',\n",
       " 'NDB_862',\n",
       " 'NDB_551',\n",
       " 'NDB_1014',\n",
       " 'NDB_628',\n",
       " 'NDB_557',\n",
       " 'NDB_459',\n",
       " 'NDB_421',\n",
       " 'NDB_491',\n",
       " 'NDB_951',\n",
       " 'NDB_694',\n",
       " 'NDB_779',\n",
       " 'NDB_879',\n",
       " 'NDB_751',\n",
       " 'NDB_401',\n",
       " 'NDB_705',\n",
       " 'NDB_379',\n",
       " 'NDB_977',\n",
       " 'NDB_594',\n",
       " 'NDB_742',\n",
       " 'NDB_611',\n",
       " 'NDB_1053',\n",
       " 'NDB_958',\n",
       " 'NDB_341',\n",
       " 'NDB_372',\n",
       " 'NDB_418',\n",
       " 'NDB_836',\n",
       " 'NDB_627',\n",
       " 'NDB_526',\n",
       " 'NDB_929',\n",
       " 'NDB_933',\n",
       " 'NDB_472',\n",
       " 'NDB_1028',\n",
       " 'NDB_735',\n",
       " 'NDB_622',\n",
       " 'NDB_386',\n",
       " 'NDB_840',\n",
       " 'NDB_517',\n",
       " 'NDB_824',\n",
       " 'NDB_519',\n",
       " 'NDB_544',\n",
       " 'NDB_392',\n",
       " 'NDB_903',\n",
       " 'NDB_925',\n",
       " 'NDB_614',\n",
       " 'NDB_450',\n",
       " 'NDB_712',\n",
       " 'NDB_395',\n",
       " 'NDB_513']"
      ]
     },
     "execution_count": 48,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[_r.id for _r in readout_dict[_type_char]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "id": "3f1ffff7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "save readout usage dict\n"
     ]
    }
   ],
   "source": [
    "# save\n",
    "readout_usage_file = os.path.join(library_folder, 'readout_usage.pkl')\n",
    "if not os.path.isfile(readout_usage_file) or overwrite_readout_usage:\n",
    "    print('save readout usage dict')\n",
    "    pickle.dump(readout_dict, open(readout_usage_file, 'wb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3d1eeb3b",
   "metadata": {},
   "source": [
    "<a id='3.3'></a>\n",
    "## 3.3 assemble probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "c37342ca",
   "metadata": {},
   "outputs": [],
   "source": [
    "reload(library_tools.assemble)\n",
    "from ImageAnalysis3.library_tools.assemble import Assemble_probes\n",
    "\n",
    "# params\n",
    "assemble_params = {\n",
    "    'primer_len':20,\n",
    "    'num_readout_per_probe':4, \n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "id": "ec61f5f6",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Assemble probes by given target sequences, readouts and primers.\n",
      "-- included readout types: ['c']\n",
      "--- assemblying 36 probes in region: NRNX1-seg-1\n",
      "--- assemblying 40 probes in region: NRNX1-seg-2\n",
      "--- assemblying 40 probes in region: NRNX1-seg-3\n",
      "--- assemblying 53 probes in region: NRNX1-seg-4\n",
      "--- assemblying 64 probes in region: NRNX1-seg-5\n",
      "--- assemblying 53 probes in region: NRNX1-seg-6\n",
      "--- assemblying 34 probes in region: NRNX1-seg-7\n",
      "--- assemblying 54 probes in region: NRNX1-seg-9\n",
      "--- assemblying 66 probes in region: NRNX1-seg-10\n",
      "--- assemblying 55 probes in region: NRNX1-seg-13\n",
      "--- assemblying 67 probes in region: NRNX1-seg-14\n",
      "--- assemblying 78 probes in region: NRNX1-seg-15\n",
      "--- assemblying 70 probes in region: NRNX1-seg-16\n",
      "--- assemblying 54 probes in region: NRNX1-seg-17\n",
      "--- assemblying 36 probes in region: NRNX1-seg-18\n",
      "--- assemblying 72 probes in region: NRNX1-seg-19\n",
      "--- assemblying 45 probes in region: NRNX1-seg-20\n",
      "--- assemblying 42 probes in region: NRNX1-seg-21\n",
      "--- assemblying 65 probes in region: NRNX1-seg-22\n",
      "--- assemblying 70 probes in region: NRNX1-seg-23\n",
      "--- assemblying 57 probes in region: NRNX1-seg-24\n",
      "--- assemblying 45 probes in region: NRNX1-seg-25\n",
      "--- assemblying 55 probes in region: NRNX1-seg-26\n",
      "--- assemblying 50 probes in region: NRNX1-seg-27\n",
      "--- assemblying 50 probes in region: NRNX1-seg-28\n",
      "--- assemblying 47 probes in region: NRNX1-seg-30\n",
      "--- assemblying 41 probes in region: NRNX1-seg-31\n",
      "--- assemblying 49 probes in region: NRNX1-seg-32\n",
      "--- assemblying 34 probes in region: NRNX1-seg-33\n",
      "--- assemblying 43 probes in region: NRNX1-seg-34\n",
      "--- assemblying 42 probes in region: NRNX1-seg-35\n",
      "--- assemblying 53 probes in region: NRNX1-seg-36\n",
      "--- assemblying 44 probes in region: NRNX1-seg-37\n",
      "--- assemblying 48 probes in region: NRNX1-seg-38\n",
      "--- assemblying 47 probes in region: NRNX1-seg-39\n",
      "--- assemblying 70 probes in region: NRNX1-seg-40\n",
      "--- assemblying 55 probes in region: NRNX1-seg-41\n",
      "--- assemblying 61 probes in region: NRNX1-seg-42\n",
      "--- assemblying 69 probes in region: NRNX1-seg-43\n",
      "--- assemblying 54 probes in region: NRNX1-seg-44\n",
      "--- assemblying 50 probes in region: NRNX1-seg-45\n",
      "--- assemblying 42 probes in region: NRNX1-seg-46\n",
      "--- assemblying 41 probes in region: NRNX1-seg-47\n",
      "--- assemblying 40 probes in region: NRNX1-seg-48\n",
      "--- assemblying 58 probes in region: NRNX1-seg-49\n",
      "--- assemblying 50 probes in region: NRNX1-seg-50\n",
      "--- assemblying 52 probes in region: NRNX1-seg-51\n",
      "--- assemblying 59 probes in region: NRNX1-seg-52\n",
      "--- assemblying 53 probes in region: NRNX1-seg-53\n",
      "--- assemblying 66 probes in region: NRNX1-seg-54\n",
      "--- assemblying 57 probes in region: NRNX1-seg-55\n",
      "--- assemblying 44 probes in region: NRNX1-seg-56\n",
      "--- assemblying 54 probes in region: NRNX1-seg-57\n",
      "--- assemblying 48 probes in region: NRNX1-seg-58\n",
      "--- assemblying 48 probes in region: NRNX1-seg-59\n",
      "--- assemblying 23 probes in region: NRNX1-seg-60\n",
      "--- assemblying 43 probes in region: NRNX1-seg-62\n",
      "--- assemblying 57 probes in region: NRNX1-seg-63\n",
      "--- assemblying 44 probes in region: NRNX1-seg-64\n",
      "--- assemblying 24 probes in region: NRNX1-seg-65\n",
      "--- assemblying 45 probes in region: NRNX1-seg-66\n",
      "--- assemblying 51 probes in region: NRNX1-seg-67\n",
      "--- assemblying 24 probes in region: NRNX1-seg-68\n",
      "--- assemblying 45 probes in region: NRNX1-seg-69\n",
      "--- assemblying 42 probes in region: NRNX1-seg-70\n",
      "--- assemblying 54 probes in region: NRNX1-seg-71\n",
      "--- assemblying 44 probes in region: NRNX1-seg-72\n",
      "--- assemblying 24 probes in region: NRNX1-seg-73\n",
      "--- assemblying 36 probes in region: NRNX1-seg-74\n",
      "--- assemblying 28 probes in region: NRNX1-seg-75\n",
      "--- assemblying 46 probes in region: NRNX1-seg-76\n",
      "--- assemblying 34 probes in region: NRNX1-seg-77\n",
      "--- assemblying 32 probes in region: NRNX1-seg-79\n",
      "--- assemblying 51 probes in region: NRNX1-seg-80\n",
      "--- assemblying 37 probes in region: NRNX1-seg-81\n",
      "--- assemblying 48 probes in region: NRNX1-seg-82\n",
      "--- assemblying 63 probes in region: NRNX1-seg-83\n",
      "--- assemblying 69 probes in region: NRNX1-seg-84\n",
      "--- assemblying 55 probes in region: NRNX1-seg-85\n",
      "--- assemblying 46 probes in region: NRNX1-seg-86\n",
      "--- assemblying 50 probes in region: NRNX1-seg-87\n",
      "--- assemblying 47 probes in region: NRNX1-seg-88\n",
      "--- assemblying 77 probes in region: NRNX1-seg-89\n",
      "--- assemblying 52 probes in region: NRNX1-seg-90\n",
      "--- assemblying 50 probes in region: NRNX1-seg-91\n",
      "--- assemblying 38 probes in region: NRNX1-seg-92\n",
      "--- assemblying 73 probes in region: NRNX1-seg-93\n",
      "--- assemblying 45 probes in region: NRNX1-seg-94\n",
      "--- assemblying 29 probes in region: NRNX1-seg-95\n",
      "--- assemblying 44 probes in region: NRNX1-seg-96\n",
      "--- assemblying 44 probes in region: NRNX1-seg-97\n",
      "--- assemblying 37 probes in region: NRNX1-seg-98\n",
      "--- assemblying 32 probes in region: NRNX1-seg-99\n",
      "--- assemblying 44 probes in region: NRNX1-seg-100\n",
      "--- assemblying 42 probes in region: NRNX1-seg-101\n",
      "--- assemblying 49 probes in region: NRNX1-seg-102\n",
      "--- assemblying 52 probes in region: NRNX1-seg-103\n",
      "--- assemblying 58 probes in region: NRNX1-seg-104\n",
      "--- assemblying 23 probes in region: NRNX1-seg-105\n",
      "--- assemblying 69 probes in region: NRNX1-seg-106\n",
      "--- assemblying 46 probes in region: NRNX1-seg-107\n",
      "--- assemblying 32 probes in region: NRNX1-seg-108\n",
      "--- assemblying 42 probes in region: NRNX1-seg-109\n",
      "--- assemblying 66 probes in region: NRNX1-seg-110\n",
      "--- assemblying 47 probes in region: NRNX1-seg-111\n",
      "--- assemblying 25 probes in region: NRNX1-seg-112\n",
      "--- assemblying 62 probes in region: NRNX1-seg-113\n",
      "--- assemblying 57 probes in region: NRNX1-seg-114\n",
      "--- assemblying 50 probes in region: NRNX1-seg-115\n",
      "--- assemblying 57 probes in region: NRNX1-seg-117\n",
      "--- assemblying 58 probes in region: NRNX1-seg-118\n",
      "--- assemblying 53 probes in region: NRNX1-seg-119\n",
      "--- assemblying 56 probes in region: NRNX1-seg-120\n",
      "--- assemblying 65 probes in region: NRNX1-seg-121\n",
      "--- assemblying 49 probes in region: NRNX1-seg-122\n",
      "--- assemblying 37 probes in region: NRNX1-seg-123\n",
      "--- assemblying 59 probes in region: NRNX1-seg-124\n",
      "--- assemblying 36 probes in region: NRNX1-seg-125\n",
      "--- assemblying 27 probes in region: NRNX1-seg-126\n",
      "--- assemblying 38 probes in region: NRNX1-seg-127\n",
      "--- assemblying 36 probes in region: NRNX1-seg-128\n",
      "--- assemblying 30 probes in region: NRNX1-seg-129\n",
      "--- assemblying 46 probes in region: NRNX1-seg-130\n",
      "--- assemblying 40 probes in region: NRNX1-seg-131\n",
      "--- assemblying 24 probes in region: NRNX1-seg-133\n",
      "--- assemblying 28 probes in region: NRNX1-seg-134\n",
      "--- assemblying 42 probes in region: NRNX1-seg-135\n",
      "--- assemblying 51 probes in region: NRNX1-seg-136\n",
      "--- assemblying 50 probes in region: NRNX1-seg-137\n",
      "--- assemblying 51 probes in region: NRNX1-seg-138\n",
      "--- assemblying 67 probes in region: NRNX1-seg-139\n",
      "--- assemblying 61 probes in region: NRNX1-seg-140\n",
      "--- assemblying 39 probes in region: NRNX1-seg-141\n",
      "--- assemblying 31 probes in region: NRNX1-seg-142\n",
      "--- assemblying 71 probes in region: NRNX1-seg-143\n",
      "--- assemblying 69 probes in region: NRNX1-seg-144\n",
      "--- assemblying 69 probes in region: NRNX1-seg-145\n",
      "--- assemblying 69 probes in region: NRNX1-seg-146\n",
      "--- assemblying 68 probes in region: NRNX1-seg-147\n",
      "--- assemblying 57 probes in region: NRNX1-seg-148\n",
      "--- assemblying 59 probes in region: NRNX1-seg-150\n",
      "--- assemblying 69 probes in region: NRNX1-seg-151\n",
      "--- assemblying 66 probes in region: NRNX1-seg-152\n",
      "--- assemblying 35 probes in region: NRNX1-seg-154\n",
      "--- assemblying 59 probes in region: NRNX1-seg-155\n",
      "--- assemblying 66 probes in region: NRNX1-seg-156\n",
      "--- assemblying 49 probes in region: NRNX1-seg-157\n",
      "--- assemblying 67 probes in region: NRNX1-seg-158\n",
      "--- assemblying 58 probes in region: NRNX1-seg-159\n",
      "--- assemblying 64 probes in region: NRNX1-seg-160\n",
      "--- assemblying 75 probes in region: NRNX1-seg-161\n",
      "--- assemblying 66 probes in region: NRNX1-seg-162\n",
      "--- assemblying 34 probes in region: NRNX1-seg-163\n",
      "--- assemblying 56 probes in region: NRNX1-seg-164\n",
      "--- assemblying 80 probes in region: NRNX1-seg-165\n",
      "--- assemblying 51 probes in region: NRNX1-seg-166\n",
      "--- assemblying 62 probes in region: NRNX1-seg-169\n",
      "--- assemblying 71 probes in region: NRNX1-seg-170\n",
      "--- assemblying 63 probes in region: NRNX1-seg-171\n",
      "--- assemblying 76 probes in region: NRNX1-seg-172\n",
      "--- assemblying 68 probes in region: NRNX1-seg-173\n",
      "--- assemblying 58 probes in region: NRNX1-seg-174\n",
      "--- assemblying 71 probes in region: NRNX1-seg-175\n",
      "--- assemblying 70 probes in region: NRNX1-seg-176\n",
      "--- assemblying 64 probes in region: NRNX1-seg-177\n",
      "--- assemblying 66 probes in region: NRNX1-seg-178\n",
      "--- assemblying 71 probes in region: NRNX1-seg-179\n",
      "--- assemblying 69 probes in region: NRNX1-seg-180\n",
      "--- assemblying 71 probes in region: NRNX1-seg-181\n",
      "--- assemblying 71 probes in region: NRNX1-seg-182\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 56 probes in region: NRNX1-seg-183\n",
      "--- assemblying 48 probes in region: NRNX1-seg-184\n",
      "--- assemblying 43 probes in region: NRNX1-seg-185\n",
      "--- assemblying 54 probes in region: NRNX1-seg-186\n",
      "--- assemblying 65 probes in region: NRNX1-seg-187\n",
      "--- assemblying 59 probes in region: NRNX1-seg-188\n",
      "--- assemblying 63 probes in region: NRNX1-seg-189\n",
      "--- assemblying 64 probes in region: NRNX1-seg-190\n",
      "--- assemblying 52 probes in region: NRNX1-seg-191\n",
      "--- assemblying 80 probes in region: NRNX1-seg-192\n",
      "--- assemblying 71 probes in region: NRNX1-seg-193\n",
      "--- assemblying 67 probes in region: NRNX1-seg-194\n",
      "--- assemblying 72 probes in region: NRNX1-seg-195\n",
      "--- assemblying 49 probes in region: NRNX1-seg-196\n",
      "--- assemblying 74 probes in region: NRNX1-seg-197\n",
      "--- assemblying 48 probes in region: NRNX1-seg-198\n",
      "--- assemblying 68 probes in region: NRNX1-seg-199\n",
      "--- assemblying 70 probes in region: NRNX1-seg-200\n",
      "--- assemblying 77 probes in region: NRNX1-seg-201\n",
      "--- assemblying 41 probes in region: NRNX1-seg-202\n",
      "--- assemblying 45 probes in region: NRNX1-seg-203\n",
      "--- assemblying 65 probes in region: NRNX1-seg-204\n",
      "--- assemblying 65 probes in region: NRNX1-seg-205\n",
      "--- assemblying 71 probes in region: NRNX1-seg-206\n",
      "--- assemblying 70 probes in region: NRNX1-seg-207\n",
      "--- assemblying 51 probes in region: NRNX1-seg-208\n",
      "--- assemblying 75 probes in region: NRNX1-seg-209\n",
      "--- assemblying 68 probes in region: NRNX1-seg-210\n",
      "--- assemblying 62 probes in region: NRNX1-seg-211\n",
      "--- assemblying 23 probes in region: NRNX1-seg-212\n",
      "--- assemblying 49 probes in region: NRNX1-seg-213\n",
      "--- assemblying 73 probes in region: NRNX1-seg-214\n",
      "--- assemblying 30 probes in region: NRNX1-seg-215\n",
      "--- assemblying 27 probes in region: NRNX1-seg-216\n",
      "--- assemblying 24 probes in region: NRNX1-seg-217\n",
      "--- assemblying 44 probes in region: NRNX1-seg-218\n",
      "--- assemblying 23 probes in region: NRNX1-seg-219\n",
      "--- assemblying 35 probes in region: NRNX1-seg-220\n",
      "--- assemblying 73 probes in region: NRNX1-seg-221\n",
      "--- assemblying 71 probes in region: NRNX1-seg-222\n",
      "--- assemblying 75 probes in region: NRNX1-seg-223\n",
      "--- assemblying 74 probes in region: NRNX1-seg-224\n",
      "--- assemblying 60 probes in region: NRNX1-seg-225\n",
      "--- assemblying 54 probes in region: NRNX1-seg-226\n",
      "--- assemblying 41 probes in region: NRNX1-seg-227\n",
      "--- assemblying 29 probes in region: NRNX1-seg-228\n",
      "--- assemblying 62 probes in region: NRNX1-seg-229\n",
      "--- assemblying 70 probes in region: NRNX1-seg-230\n",
      "--- assemblying 70 probes in region: NRNX1-seg-231\n",
      "--- assemblying 29 probes in region: NRNX1-seg-232\n",
      "--- assemblying 65 probes in region: NRNX1-seg-233\n",
      "--- assemblying 54 probes in region: NRNX1-seg-234\n",
      "--- assemblying 64 probes in region: NRNX1-seg-235\n",
      "--- assemblying 54 probes in region: NRNX1-seg-236\n",
      "--- assemblying 53 probes in region: NRNX1-seg-238\n",
      "--- assemblying 73 probes in region: NRNX1-seg-239\n",
      "--- assemblying 66 probes in region: NRNX1-seg-240\n",
      "--- assemblying 49 probes in region: NRNX1-seg-241\n",
      "--- assemblying 53 probes in region: NRNX1-seg-242\n",
      "--- assemblying 54 probes in region: NRNX1-seg-244\n",
      "--- assemblying 64 probes in region: NRNX1-seg-245\n",
      "--- assemblying 38 probes in region: NRNX1-seg-246\n",
      "--- assemblying 68 probes in region: NRNX1-seg-248\n",
      "--- assemblying 41 probes in region: NRNX1-seg-249\n",
      "--- assemblying 68 probes in region: NRNX1-seg-251\n",
      "--- assemblying 53 probes in region: NRNX1-seg-252\n",
      "--- assemblying 68 probes in region: NRNX1-seg-253\n",
      "--- assemblying 50 probes in region: NRNX1-seg-254\n",
      "--- assemblying 64 probes in region: NRNX1-seg-255\n",
      "--- assemblying 63 probes in region: NRNX1-seg-256\n",
      "--- assemblying 52 probes in region: NRNX1-seg-257\n",
      "--- assemblying 63 probes in region: NRNX1-seg-258\n",
      "--- assemblying 54 probes in region: NRNX1-seg-259\n",
      "--- assemblying 50 probes in region: NRNX1-seg-260\n",
      "--- assemblying 42 probes in region: NRNX1-seg-262\n",
      "--- assemblying 25 probes in region: NRNX1-seg-263\n",
      "--- assemblying 35 probes in region: NRNX1-seg-264\n",
      "--- assemblying 32 probes in region: NRNX1-seg-265\n",
      "--- assemblying 33 probes in region: NRNX1-seg-266\n",
      "--- assemblying 55 probes in region: NRNX1-seg-267\n",
      "--- assemblying 70 probes in region: NRNX1-seg-268\n",
      "--- assemblying 63 probes in region: NRNX1-seg-269\n",
      "--- assemblying 50 probes in region: NRNX1-seg-270\n",
      "--- assemblying 59 probes in region: NRNX1-seg-271\n",
      "--- assemblying 64 probes in region: NRNX1-seg-272\n",
      "--- assemblying 37 probes in region: NRNX1-seg-273\n",
      "--- assemblying 65 probes in region: NRNX1-seg-274\n",
      "--- assemblying 56 probes in region: NRNX1-seg-275\n",
      "--- assemblying 57 probes in region: NRNX1-seg-276\n",
      "--- assemblying 46 probes in region: NRNX1-seg-277\n",
      "--- assemblying 24 probes in region: NRNX1-seg-278\n",
      "--- assemblying 28 probes in region: NRNX1-seg-279\n",
      "--- assemblying 50 probes in region: NRNX1-seg-280\n",
      "--- assemblying 63 probes in region: NRNX1-seg-281\n",
      "--- assemblying 46 probes in region: NRNX1-seg-282\n",
      "--- assemblying 57 probes in region: NRNX1-seg-283\n",
      "--- assemblying 61 probes in region: NRNX1-seg-284\n",
      "--- assemblying 51 probes in region: NRNX1-seg-285\n",
      "--- assemblying 39 probes in region: NRNX1-seg-286\n",
      "--- assemblying 26 probes in region: NRNX1-seg-287\n",
      "--- assemblying 29 probes in region: NRNX1-seg-288\n",
      "--- assemblying 45 probes in region: NRNX1-seg-289\n",
      "--- assemblying 36 probes in region: NRNX1-seg-290\n",
      "--- assemblying 39 probes in region: NRNX1-seg-291\n",
      "--- assemblying 57 probes in region: NRNX1-seg-292\n",
      "--- assemblying 27 probes in region: NRNX1-seg-293\n",
      "--- assemblying 31 probes in region: NRNX1-seg-294\n",
      "--- assemblying 44 probes in region: NRNX1-seg-295\n",
      "--- assemblying 42 probes in region: NRNX1-seg-298\n",
      "--- assemblying 39 probes in region: NRNX1-seg-299\n",
      "--- assemblying 36 probes in region: NRNX1-seg-300\n",
      "--- assemblying 54 probes in region: NRNX1-seg-301\n",
      "--- assemblying 31 probes in region: NRNX1-seg-302\n",
      "--- assemblying 50 probes in region: NRNX1-seg-303\n",
      "--- assemblying 58 probes in region: NRNX1-seg-304\n",
      "--- assemblying 46 probes in region: NRNX1-seg-305\n",
      "--- assemblying 49 probes in region: NRNX1-seg-306\n",
      "--- assemblying 22 probes in region: NRNX1-seg-307\n",
      "--- assemblying 24 probes in region: NRNX1-seg-308\n",
      "--- assemblying 58 probes in region: NRNX1-seg-311\n",
      "--- assemblying 57 probes in region: NRNX1-seg-312\n",
      "--- assemblying 76 probes in region: NRNX1-seg-313\n",
      "--- assemblying 54 probes in region: NRNX1-seg-314\n",
      "--- assemblying 42 probes in region: NRNX1-seg-315\n",
      "--- assemblying 48 probes in region: NRNX1-seg-316\n",
      "--- assemblying 44 probes in region: NRNX1-seg-317\n",
      "--- assemblying 34 probes in region: NRNX1-seg-318\n",
      "--- assemblying 35 probes in region: NRNX1-seg-319\n",
      "--- assemblying 50 probes in region: NRNX1-seg-320\n",
      "--- assemblying 69 probes in region: NRNX1-seg-321\n",
      "--- assemblying 55 probes in region: NRNX1-seg-322\n",
      "--- assemblying 41 probes in region: NRNX1-seg-323\n",
      "--- assemblying 57 probes in region: NRNX1-seg-324\n",
      "--- assemblying 59 probes in region: NRNX1-seg-325\n",
      "--- assemblying 64 probes in region: NRNX1-seg-326\n",
      "--- assemblying 64 probes in region: NRNX1-seg-327\n",
      "--- assemblying 34 probes in region: NRNX1-seg-328\n",
      "--- assemblying 61 probes in region: NRNX1-seg-329\n",
      "--- assemblying 31 probes in region: NRNX1-seg-330\n",
      "--- assemblying 53 probes in region: NRNX1-seg-331\n",
      "--- assemblying 60 probes in region: NRNX1-seg-332\n",
      "--- assemblying 52 probes in region: NRNX1-seg-333\n",
      "--- assemblying 65 probes in region: NRNX1-seg-334\n",
      "--- assemblying 52 probes in region: NRNX1-seg-335\n",
      "--- assemblying 61 probes in region: NRNX1-seg-336\n",
      "--- assemblying 35 probes in region: NRNX1-seg-337\n",
      "--- assemblying 49 probes in region: NRNX1-seg-338\n",
      "--- assemblying 50 probes in region: NRNX1-seg-339\n",
      "--- assemblying 46 probes in region: NRNX1-seg-340\n",
      "--- assemblying 49 probes in region: NRNX1-seg-341\n",
      "--- assemblying 65 probes in region: NRNX1-seg-342\n",
      "--- assemblying 50 probes in region: NRNX1-seg-343\n",
      "--- assemblying 56 probes in region: NRNX1-seg-344\n",
      "--- assemblying 38 probes in region: NRNX1-seg-345\n",
      "--- assemblying 42 probes in region: NRNX1-seg-346\n",
      "--- assemblying 25 probes in region: NRNX1-seg-347\n",
      "--- assemblying 44 probes in region: NRNX1-seg-348\n",
      "--- assemblying 35 probes in region: NRNX1-seg-349\n",
      "--- assemblying 37 probes in region: NRNX1-seg-350\n",
      "--- assemblying 53 probes in region: NRNX1-seg-351\n",
      "--- assemblying 36 probes in region: NRNX1-seg-352\n",
      "--- assemblying 59 probes in region: NRNX1-seg-353\n",
      "--- assemblying 50 probes in region: NRNX1-seg-354\n",
      "--- assemblying 60 probes in region: NRNX1-seg-355\n",
      "--- assemblying 40 probes in region: NRNX1-seg-356\n",
      "--- assemblying 73 probes in region: NRNX1-seg-357\n",
      "--- assemblying 47 probes in region: NRNX1-seg-358\n",
      "--- assemblying 52 probes in region: NRNX1-seg-359\n",
      "--- assemblying 56 probes in region: NRNX1-seg-360\n",
      "--- assemblying 61 probes in region: NRNX1-seg-361\n",
      "--- assemblying 40 probes in region: NRNX1-seg-362\n",
      "--- assemblying 55 probes in region: NRNX1-seg-363\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 57 probes in region: NRNX1-seg-364\n",
      "--- assemblying 37 probes in region: NRNX1-seg-365\n",
      "--- assemblying 42 probes in region: NRNX1-seg-366\n",
      "--- assemblying 64 probes in region: NRNX1-seg-367\n",
      "--- assemblying 47 probes in region: NRNX1-seg-368\n",
      "--- assemblying 56 probes in region: NRNX1-seg-369\n",
      "--- assemblying 51 probes in region: NRNX1-seg-370\n",
      "--- assemblying 47 probes in region: NRNX1-seg-371\n",
      "--- assemblying 65 probes in region: NRNX1-seg-372\n",
      "--- assemblying 55 probes in region: NRNX1-seg-373\n",
      "--- assemblying 31 probes in region: NRNX1-seg-374\n",
      "--- assemblying 21 probes in region: NRNX1-seg-375\n",
      "--- assemblying 25 probes in region: NRNX1-seg-376\n",
      "--- assemblying 30 probes in region: NRNX1-seg-379\n",
      "--- assemblying 49 probes in region: NRNX1-seg-380\n",
      "--- assemblying 60 probes in region: NRNX1-seg-381\n",
      "--- assemblying 43 probes in region: NRNX1-seg-382\n",
      "--- assemblying 52 probes in region: NRNX1-seg-383\n",
      "--- assemblying 41 probes in region: NRNX1-seg-384\n",
      "--- assemblying 55 probes in region: NRNX1-seg-385\n",
      "--- assemblying 58 probes in region: NRNX1-seg-386\n",
      "--- assemblying 45 probes in region: NRNX1-seg-387\n",
      "--- assemblying 62 probes in region: NRNX1-seg-388\n",
      "--- assemblying 60 probes in region: NRNX1-seg-389\n",
      "--- assemblying 32 probes in region: NRNX1-seg-390\n",
      "--- assemblying 61 probes in region: NRNX1-seg-391\n",
      "--- assemblying 67 probes in region: NRNX1-seg-392\n",
      "--- assemblying 49 probes in region: NRNX1-seg-393\n",
      "--- assemblying 53 probes in region: NRNX1-seg-394\n",
      "--- assemblying 56 probes in region: NRNX1-seg-395\n",
      "--- assemblying 43 probes in region: NRNX1-seg-396\n",
      "--- assemblying 52 probes in region: NRNX1-seg-397\n",
      "--- assemblying 47 probes in region: NRNX1-seg-398\n",
      "--- assemblying 48 probes in region: NRNX1-seg-399\n",
      "--- assemblying 55 probes in region: NRNX1-seg-400\n",
      "--- assemblying 61 probes in region: NRNX1-seg-401\n",
      "--- assemblying 55 probes in region: NRNX1-seg-402\n",
      "--- assemblying 64 probes in region: NRNX1-seg-403\n",
      "--- assemblying 58 probes in region: NRNX1-seg-404\n",
      "--- assemblying 47 probes in region: NRNX1-seg-405\n",
      "--- assemblying 54 probes in region: NRNX1-seg-406\n",
      "--- assemblying 39 probes in region: NRNX1-seg-407\n",
      "--- assemblying 45 probes in region: NRNX1-seg-408\n",
      "--- assemblying 44 probes in region: NRNX1-seg-409\n",
      "--- assemblying 51 probes in region: NRNX1-seg-410\n",
      "--- assemblying 46 probes in region: NRNX1-seg-411\n",
      "--- assemblying 37 probes in region: NRNX1-seg-412\n",
      "--- assemblying 41 probes in region: NRNX1-seg-413\n",
      "--- assemblying 58 probes in region: NRNX1-seg-414\n",
      "--- assemblying 49 probes in region: NRNX1-seg-415\n",
      "--- assemblying 52 probes in region: NRNX1-seg-416\n",
      "--- assemblying 45 probes in region: NRNX1-seg-417\n",
      "--- assemblying 41 probes in region: NRNX1-seg-418\n",
      "--- assemblying 54 probes in region: NRNX1-seg-419\n",
      "--- assemblying 55 probes in region: NRNX1-seg-420\n",
      "--- assemblying 62 probes in region: NRNX1-seg-421\n",
      "--- assemblying 48 probes in region: NRNX1-seg-422\n",
      "--- assemblying 54 probes in region: NRNX1-seg-423\n",
      "--- assemblying 59 probes in region: NRNX1-seg-424\n",
      "--- assemblying 49 probes in region: NRNX1-seg-425\n",
      "--- assemblying 53 probes in region: NRNX1-seg-426\n",
      "--- assemblying 67 probes in region: NRNX1-seg-427\n",
      "--- assemblying 46 probes in region: NRNX1-seg-428\n",
      "--- assemblying 46 probes in region: NRNX1-seg-429\n",
      "--- assemblying 58 probes in region: NRNX1-seg-430\n",
      "--- assemblying 41 probes in region: NRNX1-seg-431\n",
      "--- assemblying 55 probes in region: NRNX1-seg-432\n",
      "--- assemblying 54 probes in region: NRNX1-seg-433\n",
      "--- assemblying 52 probes in region: NRNX1-seg-434\n",
      "--- assemblying 47 probes in region: NRNX1-seg-435\n",
      "--- assemblying 52 probes in region: NRNX1-seg-436\n",
      "--- assemblying 41 probes in region: NRNX1-seg-437\n",
      "--- assemblying 34 probes in region: NRNX1-seg-438\n",
      "--- assemblying 38 probes in region: NRNX1-seg-439\n",
      "--- assemblying 59 probes in region: NRNX1-seg-440\n",
      "--- assemblying 57 probes in region: NRNX1-seg-441\n",
      "--- assemblying 50 probes in region: NRNX1-seg-442\n",
      "--- assemblying 39 probes in region: NRNX1-seg-443\n",
      "--- assemblying 46 probes in region: NRNX1-seg-444\n",
      "--- assemblying 62 probes in region: NRNX1-seg-445\n",
      "--- assemblying 61 probes in region: NRNX1-seg-446\n",
      "--- assemblying 41 probes in region: NRNX1-seg-447\n",
      "--- assemblying 53 probes in region: NRNX1-seg-448\n",
      "--- assemblying 43 probes in region: NRNX1-seg-449\n",
      "--- assemblying 51 probes in region: NRNX1-seg-450\n",
      "--- assemblying 50 probes in region: NRNX1-seg-452\n",
      "--- assemblying 65 probes in region: NRNX1-seg-453\n",
      "--- assemblying 48 probes in region: NRNX1-seg-454\n",
      "--- assemblying 43 probes in region: NRNX1-seg-455\n",
      "--- assemblying 49 probes in region: NRNX1-seg-456\n",
      "--- assemblying 47 probes in region: NRNX1-seg-457\n",
      "--- assemblying 44 probes in region: NRNX1-seg-458\n",
      "--- assemblying 68 probes in region: NRNX1-seg-459\n",
      "--- assemblying 49 probes in region: NRNX1-seg-460\n",
      "--- assemblying 52 probes in region: NRNX1-seg-461\n",
      "--- assemblying 49 probes in region: NRNX1-seg-462\n",
      "--- assemblying 59 probes in region: NRNX1-seg-463\n",
      "--- assemblying 53 probes in region: NRNX1-seg-464\n",
      "--- assemblying 60 probes in region: NRNX1-seg-465\n",
      "--- assemblying 34 probes in region: NRNX1-seg-466\n",
      "--- assemblying 61 probes in region: NRNX1-seg-467\n",
      "--- assemblying 51 probes in region: NRNX1-seg-468\n",
      "--- assemblying 68 probes in region: NRNX1-seg-469\n",
      "--- assemblying 79 probes in region: NRNX1-seg-470\n",
      "--- assemblying 62 probes in region: NRNX1-seg-471\n",
      "--- assemblying 26 probes in region: NRNX1-seg-472\n",
      "--- assemblying 61 probes in region: NRNX1-seg-474\n",
      "--- assemblying 60 probes in region: NRNX1-seg-475\n",
      "--- assemblying 51 probes in region: NRNX1-seg-476\n",
      "--- assemblying 58 probes in region: NRNX1-seg-477\n",
      "--- assemblying 52 probes in region: NRNX1-seg-478\n",
      "--- assemblying 58 probes in region: NRNX1-seg-479\n",
      "--- assemblying 58 probes in region: NRNX1-seg-480\n",
      "--- assemblying 66 probes in region: NRNX1-seg-481\n",
      "--- assemblying 60 probes in region: NRNX1-seg-482\n",
      "--- assemblying 25 probes in region: NRNX1-seg-483\n",
      "--- assemblying 59 probes in region: NRNX1-seg-484\n",
      "--- assemblying 63 probes in region: NRNX1-seg-485\n",
      "--- assemblying 50 probes in region: NRNX1-seg-486\n",
      "--- assemblying 64 probes in region: NRNX1-seg-487\n",
      "--- assemblying 49 probes in region: NRNX1-seg-488\n",
      "--- assemblying 57 probes in region: NRNX1-seg-489\n",
      "--- assemblying 58 probes in region: NRNX1-seg-490\n",
      "--- assemblying 50 probes in region: NRNX1-seg-491\n",
      "--- assemblying 47 probes in region: NRNX1-seg-492\n",
      "--- assemblying 54 probes in region: NRNX1-seg-493\n",
      "--- assemblying 49 probes in region: NRNX1-seg-494\n",
      "--- assemblying 59 probes in region: NRNX1-seg-495\n",
      "--- assemblying 65 probes in region: NRNX1-seg-496\n",
      "--- assemblying 52 probes in region: NRNX1-seg-497\n",
      "--- assemblying 80 probes in region: NRNX1-seg-498\n",
      "--- assemblying 66 probes in region: NRNX1-seg-499\n",
      "--- assemblying 67 probes in region: NRNX1-seg-500\n",
      "--- assemblying 49 probes in region: NRNX1-seg-501\n",
      "--- assemblying 65 probes in region: NRNX1-seg-502\n",
      "--- assemblying 49 probes in region: NRNX1-seg-503\n",
      "--- assemblying 47 probes in region: NRNX1-seg-504\n",
      "--- assemblying 44 probes in region: NRNX1-seg-505\n",
      "--- assemblying 60 probes in region: NRNX1-seg-506\n",
      "--- assemblying 58 probes in region: NRNX1-seg-507\n",
      "--- assemblying 62 probes in region: NRNX1-seg-508\n",
      "--- assemblying 45 probes in region: NRNX1-seg-509\n",
      "--- assemblying 55 probes in region: NRNX1-seg-510\n",
      "--- assemblying 52 probes in region: NRNX1-seg-511\n",
      "--- assemblying 56 probes in region: NRNX1-seg-512\n",
      "--- assemblying 52 probes in region: NRNX1-seg-513\n",
      "--- assemblying 51 probes in region: NRNX1-seg-514\n",
      "--- assemblying 63 probes in region: NRNX1-seg-515\n",
      "--- assemblying 52 probes in region: NRNX1-seg-516\n",
      "--- assemblying 59 probes in region: NRNX1-seg-517\n",
      "--- assemblying 55 probes in region: NRNX1-seg-518\n",
      "--- assemblying 62 probes in region: NRNX1-seg-519\n",
      "--- assemblying 51 probes in region: NRNX1-seg-520\n",
      "--- assemblying 61 probes in region: NRNX1-seg-521\n",
      "--- assemblying 50 probes in region: NRNX1-seg-522\n",
      "--- assemblying 59 probes in region: NRNX1-seg-523\n",
      "--- assemblying 55 probes in region: NRNX1-seg-524\n",
      "--- assemblying 58 probes in region: NRNX1-seg-525\n",
      "--- assemblying 44 probes in region: NRNX1-seg-526\n",
      "--- assemblying 57 probes in region: NRNX1-seg-527\n",
      "--- assemblying 47 probes in region: NRNX1-seg-528\n",
      "--- assemblying 50 probes in region: NRNX1-seg-529\n",
      "--- assemblying 61 probes in region: NRNX1-seg-530\n",
      "--- assemblying 48 probes in region: NRNX1-seg-531\n",
      "--- assemblying 61 probes in region: NRNX1-seg-532\n",
      "--- assemblying 67 probes in region: NRNX1-seg-533\n",
      "--- assemblying 52 probes in region: NRNX1-seg-534\n",
      "--- assemblying 39 probes in region: NRNX1-seg-535\n",
      "--- assemblying 59 probes in region: NRNX1-seg-536\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 51 probes in region: NRNX1-seg-537\n",
      "--- assemblying 65 probes in region: NRNX1-seg-538\n",
      "--- assemblying 55 probes in region: NRNX1-seg-539\n",
      "--- assemblying 54 probes in region: NRNX1-seg-540\n",
      "--- assemblying 45 probes in region: NRNX1-seg-541\n",
      "--- assemblying 65 probes in region: NRNX1-seg-542\n",
      "--- assemblying 52 probes in region: NRNX1-seg-543\n",
      "--- assemblying 47 probes in region: NRNX1-seg-544\n",
      "--- assemblying 60 probes in region: NRNX1-seg-545\n",
      "--- assemblying 52 probes in region: NRNX1-seg-546\n",
      "--- assemblying 46 probes in region: NRNX1-seg-547\n",
      "--- assemblying 47 probes in region: NRNX1-seg-548\n",
      "--- assemblying 46 probes in region: NRNX1-seg-549\n",
      "--- assemblying 56 probes in region: NRNX1-seg-550\n",
      "--- assemblying 48 probes in region: NRNX1-seg-551\n",
      "--- assemblying 58 probes in region: NRNX1-seg-552\n",
      "--- assemblying 52 probes in region: NRNX1-seg-553\n",
      "--- assemblying 65 probes in region: NRNX1-seg-554\n",
      "--- assemblying 59 probes in region: NRNX1-seg-555\n",
      "--- assemblying 53 probes in region: NRNX1-seg-556\n",
      "--- assemblying 57 probes in region: NRNX1-seg-557\n",
      "--- assemblying 59 probes in region: NRNX1-seg-558\n",
      "--- assemblying 52 probes in region: NRNX1-seg-559\n",
      "--- assemblying 61 probes in region: NRNX1-seg-560\n",
      "--- assemblying 40 probes in region: NRNX1-seg-561\n",
      "--- assemblying 49 probes in region: NRNX1-seg-562\n",
      "--- assemblying 43 probes in region: NRNX1-seg-563\n",
      "--- assemblying 45 probes in region: NRNX1-seg-564\n",
      "--- assemblying 49 probes in region: NRNX1-seg-565\n",
      "--- assemblying 41 probes in region: NRNX1-seg-566\n",
      "--- assemblying 50 probes in region: NRNX1-seg-567\n",
      "--- assemblying 46 probes in region: NRNX1-seg-568\n",
      "--- assemblying 64 probes in region: NRNX1-seg-569\n",
      "--- assemblying 61 probes in region: NRNX1-seg-570\n",
      "--- assemblying 55 probes in region: NRNX1-seg-571\n",
      "--- assemblying 58 probes in region: NRNX1-seg-572\n",
      "--- assemblying 61 probes in region: NRNX1-seg-573\n",
      "--- assemblying 43 probes in region: NRNX1-seg-574\n",
      "--- assemblying 47 probes in region: NRNX1-seg-575\n",
      "--- assemblying 61 probes in region: NRNX1-seg-576\n",
      "--- assemblying 45 probes in region: NRNX1-seg-577\n",
      "--- assemblying 39 probes in region: NRNX1-seg-578\n",
      "--- assemblying 41 probes in region: NRNX1-seg-579\n",
      "--- assemblying 52 probes in region: NRNX1-seg-580\n",
      "--- assemblying 52 probes in region: NRNX1-seg-581\n",
      "--- assemblying 59 probes in region: NRNX1-seg-582\n",
      "--- assemblying 65 probes in region: NRNX1-seg-583\n",
      "--- assemblying 39 probes in region: NRNX1-seg-584\n",
      "--- assemblying 51 probes in region: NRNX1-seg-585\n",
      "--- assemblying 54 probes in region: NRNX1-seg-586\n",
      "--- assemblying 35 probes in region: NRNX1-seg-587\n",
      "--- assemblying 38 probes in region: NRNX1-seg-588\n",
      "--- assemblying 41 probes in region: NRNX1-seg-589\n",
      "--- assemblying 52 probes in region: NRNX1-seg-590\n",
      "--- assemblying 49 probes in region: NRNX1-seg-591\n",
      "--- assemblying 50 probes in region: NRNX1-seg-592\n",
      "--- assemblying 55 probes in region: NRNX1-seg-593\n",
      "--- assemblying 45 probes in region: NRNX1-seg-594\n",
      "--- assemblying 43 probes in region: NRNX1-seg-595\n",
      "--- assemblying 45 probes in region: NRNX1-seg-596\n",
      "--- assemblying 39 probes in region: NRNX1-seg-597\n",
      "--- assemblying 46 probes in region: NRNX1-seg-598\n",
      "--- assemblying 59 probes in region: NRNX1-seg-599\n",
      "--- assemblying 54 probes in region: NRNX1-seg-600\n",
      "--- assemblying 53 probes in region: NRNX1-seg-601\n",
      "--- assemblying 42 probes in region: NRNX1-seg-602\n",
      "--- assemblying 60 probes in region: NRNX1-seg-603\n",
      "--- assemblying 38 probes in region: NRNX1-seg-604\n",
      "--- assemblying 55 probes in region: NRNX1-seg-605\n",
      "--- assemblying 67 probes in region: NRNX1-seg-606\n",
      "--- assemblying 68 probes in region: NRNX1-seg-607\n",
      "--- assemblying 56 probes in region: NRNX1-seg-608\n",
      "--- assemblying 32 probes in region: NRNX1-seg-609\n",
      "--- assemblying 27 probes in region: NRNX1-seg-610\n",
      "--- assemblying 47 probes in region: NRNX1-seg-611\n",
      "--- assemblying 30 probes in region: NRNX1-seg-612\n",
      "--- assemblying 28 probes in region: NRNX1-seg-613\n",
      "--- assemblying 49 probes in region: NRNX1-seg-614\n",
      "--- assemblying 33 probes in region: NRNX1-seg-615\n",
      "--- assemblying 42 probes in region: NRNX1-seg-616\n",
      "--- assemblying 48 probes in region: NRNX1-seg-617\n",
      "--- assemblying 60 probes in region: NRNX1-seg-618\n",
      "--- assemblying 73 probes in region: NRNX1-seg-619\n",
      "--- assemblying 75 probes in region: NRNX1-seg-620\n",
      "--- assemblying 56 probes in region: NRNX1-seg-621\n",
      "--- assemblying 65 probes in region: NRNX1-seg-622\n",
      "--- assemblying 50 probes in region: NRNX1-seg-623\n",
      "--- assemblying 58 probes in region: NRNX1-seg-624\n",
      "--- assemblying 39 probes in region: NRNX1-seg-625\n",
      "--- assemblying 48 probes in region: NRNX1-seg-626\n",
      "--- assemblying 44 probes in region: NRNX1-seg-627\n",
      "--- assemblying 37 probes in region: NRNX1-seg-628\n",
      "--- assemblying 40 probes in region: NRNX1-seg-629\n",
      "--- assemblying 34 probes in region: NRNX1-seg-630\n",
      "--- assemblying 55 probes in region: NRNX1-seg-631\n",
      "--- assemblying 57 probes in region: NRNX1-seg-632\n",
      "--- assemblying 51 probes in region: NRNX1-seg-633\n",
      "--- assemblying 64 probes in region: NRNX1-seg-634\n",
      "--- assemblying 45 probes in region: NRNX1-seg-635\n",
      "--- assemblying 32 probes in region: NRNX1-seg-636\n",
      "--- assemblying 22 probes in region: NRNX1-seg-637\n",
      "--- assemblying 32 probes in region: NRNX1-seg-639\n",
      "--- assemblying 54 probes in region: NRNX1-seg-640\n",
      "--- assemblying 46 probes in region: NRNX1-seg-641\n",
      "--- assemblying 45 probes in region: NRNX1-seg-642\n",
      "--- assemblying 57 probes in region: NRNX1-seg-643\n",
      "--- assemblying 58 probes in region: NRNX1-seg-644\n",
      "--- assemblying 49 probes in region: NRNX1-seg-645\n",
      "--- assemblying 69 probes in region: NRNX1-seg-646\n",
      "--- assemblying 33 probes in region: NRNX1-seg-647\n",
      "--- assemblying 41 probes in region: NRNX1-seg-648\n",
      "--- assemblying 45 probes in region: NRNX1-seg-649\n",
      "--- assemblying 44 probes in region: NRNX1-seg-650\n",
      "--- assemblying 47 probes in region: NRNX1-seg-651\n",
      "--- assemblying 35 probes in region: NRNX1-seg-652\n",
      "--- assemblying 49 probes in region: NRNX1-seg-653\n",
      "--- assemblying 57 probes in region: NRNX1-seg-654\n",
      "--- assemblying 55 probes in region: NRNX1-seg-655\n",
      "--- assemblying 50 probes in region: NRNX1-seg-656\n",
      "--- assemblying 49 probes in region: NRNX1-seg-657\n",
      "--- assemblying 60 probes in region: NRNX1-seg-658\n",
      "--- assemblying 50 probes in region: NRNX1-seg-659\n",
      "--- assemblying 50 probes in region: NRNX1-seg-660\n",
      "--- assemblying 51 probes in region: NRNX1-seg-661\n",
      "--- assemblying 46 probes in region: NRNX1-seg-662\n",
      "--- assemblying 53 probes in region: NRNX1-seg-665\n",
      "--- assemblying 68 probes in region: NRNX1-seg-666\n",
      "--- assemblying 50 probes in region: NRNX1-seg-667\n",
      "--- assemblying 49 probes in region: NRNX1-seg-668\n",
      "--- assemblying 45 probes in region: NRNX1-seg-669\n",
      "--- assemblying 33 probes in region: NRNX1-seg-670\n",
      "--- assemblying 54 probes in region: NRNX1-seg-671\n",
      "--- assemblying 38 probes in region: NRNX1-seg-672\n",
      "--- assemblying 47 probes in region: NRNX1-seg-674\n",
      "--- assemblying 43 probes in region: NRNX1-seg-675\n",
      "--- assemblying 40 probes in region: NRNX1-seg-676\n",
      "--- assemblying 44 probes in region: NRNX1-seg-678\n",
      "--- assemblying 22 probes in region: NRNX1-seg-679\n",
      "--- assemblying 31 probes in region: NRNX1-seg-681\n",
      "--- assemblying 39 probes in region: NRNX1-seg-682\n",
      "--- assemblying 53 probes in region: NRNX1-seg-683\n",
      "--- assemblying 23 probes in region: NRNX1-seg-684\n",
      "--- assemblying 37 probes in region: NRNX1-seg-685\n",
      "--- assemblying 28 probes in region: NRNX1-seg-686\n",
      "--- assemblying 27 probes in region: NRNX1-seg-687\n",
      "--- assemblying 28 probes in region: NRNX1-seg-688\n",
      "--- assemblying 45 probes in region: NRNX1-seg-689\n",
      "--- assemblying 43 probes in region: NRNX1-seg-690\n",
      "--- assemblying 36 probes in region: NRNX1-seg-691\n",
      "--- assemblying 47 probes in region: NRNX1-seg-692\n",
      "--- assemblying 54 probes in region: NRNX1-seg-693\n",
      "--- assemblying 49 probes in region: NRNX1-seg-694\n",
      "--- assemblying 57 probes in region: NRNX1-seg-695\n",
      "--- assemblying 40 probes in region: NRNX1-seg-696\n",
      "--- assemblying 42 probes in region: NRNX1-seg-697\n",
      "--- assemblying 61 probes in region: NRNX1-seg-698\n",
      "--- assemblying 46 probes in region: NRNX1-seg-699\n",
      "-- 34068 probes assembled in total.\n",
      "-- saving 34068 probes into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\candidate_probes.fasta\n",
      "-- saving readout_summary into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\readout_summary.pkl\n"
     ]
    }
   ],
   "source": [
    "# run assembly\n",
    "cand_probes, readout_summary = Assemble_probes(library_folder, filtered_pb_dict, \n",
    "                                               region_2_readout_dict, readout_dict, primers, \n",
    "                                               primer_len=assemble_params['primer_len'], \n",
    "                                               num_readout_per_probe=assemble_params['num_readout_per_probe'], \n",
    "                                               unique_readout_per_probe=True,\n",
    "                                               rc_targets=False, overwrite=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9265b92d",
   "metadata": {},
   "source": [
    "# 4.Check quality"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8dca4823",
   "metadata": {},
   "source": [
    "\n",
    "## 4.1 Load probes\n",
    "\n",
    "Checking procedures could restart from here"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "a651e7eb",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total probe loaded: 34068\n"
     ]
    }
   ],
   "source": [
    "# folder for this library\n",
    "# biopython imports\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "\n",
    "# candidate full-length probe filename\n",
    "candidate_basename = 'candidate_probes.fasta' # old version\n",
    "# load full probes\n",
    "full_records = []\n",
    "with open(os.path.join(library_folder, candidate_basename), 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        full_records.append(record)\n",
    "\n",
    "print(f\"Total probe loaded: {len(full_records)}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fe532250",
   "metadata": {},
   "source": [
    "### Manually input the following info to double check!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "id": "cf1d4bf8",
   "metadata": {},
   "outputs": [],
   "source": [
    "num_readout_per_probe = 4\n",
    "primer_len=20\n",
    "target_len=42\n",
    "min_num_probes = 21\n",
    "# merfish related\n",
    "num_on_bits = 3"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2da410b1",
   "metadata": {},
   "source": [
    "### 4.1.1 check primer usage"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "id": "3cba3f0b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Checking primer usage, total probes: 34068\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "primer_check = check._check_primer_usage(full_records, primers[0], primers[1], primer_len=primer_len)\n",
    "print(primer_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "30ce1025",
   "metadata": {},
   "source": [
    "### 4.1.2 check region probe number"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "761720bd",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: NRNX1-seg-1 -> 36\n",
      "gene: NRNX1-seg-2 -> 40\n",
      "gene: NRNX1-seg-3 -> 40\n",
      "gene: NRNX1-seg-4 -> 53\n",
      "gene: NRNX1-seg-5 -> 64\n",
      "gene: NRNX1-seg-6 -> 53\n",
      "gene: NRNX1-seg-7 -> 34\n",
      "gene: NRNX1-seg-9 -> 54\n",
      "gene: NRNX1-seg-10 -> 66\n",
      "gene: NRNX1-seg-13 -> 55\n",
      "gene: NRNX1-seg-14 -> 67\n",
      "gene: NRNX1-seg-15 -> 78\n",
      "gene: NRNX1-seg-16 -> 70\n",
      "gene: NRNX1-seg-17 -> 54\n",
      "gene: NRNX1-seg-18 -> 36\n",
      "gene: NRNX1-seg-19 -> 72\n",
      "gene: NRNX1-seg-20 -> 45\n",
      "gene: NRNX1-seg-21 -> 42\n",
      "gene: NRNX1-seg-22 -> 65\n",
      "gene: NRNX1-seg-23 -> 70\n",
      "gene: NRNX1-seg-24 -> 57\n",
      "gene: NRNX1-seg-25 -> 45\n",
      "gene: NRNX1-seg-26 -> 55\n",
      "gene: NRNX1-seg-27 -> 50\n",
      "gene: NRNX1-seg-28 -> 50\n",
      "gene: NRNX1-seg-30 -> 47\n",
      "gene: NRNX1-seg-31 -> 41\n",
      "gene: NRNX1-seg-32 -> 49\n",
      "gene: NRNX1-seg-33 -> 34\n",
      "gene: NRNX1-seg-34 -> 43\n",
      "gene: NRNX1-seg-35 -> 42\n",
      "gene: NRNX1-seg-36 -> 53\n",
      "gene: NRNX1-seg-37 -> 44\n",
      "gene: NRNX1-seg-38 -> 48\n",
      "gene: NRNX1-seg-39 -> 47\n",
      "gene: NRNX1-seg-40 -> 70\n",
      "gene: NRNX1-seg-41 -> 55\n",
      "gene: NRNX1-seg-42 -> 61\n",
      "gene: NRNX1-seg-43 -> 69\n",
      "gene: NRNX1-seg-44 -> 54\n",
      "gene: NRNX1-seg-45 -> 50\n",
      "gene: NRNX1-seg-46 -> 42\n",
      "gene: NRNX1-seg-47 -> 41\n",
      "gene: NRNX1-seg-48 -> 40\n",
      "gene: NRNX1-seg-49 -> 58\n",
      "gene: NRNX1-seg-50 -> 50\n",
      "gene: NRNX1-seg-51 -> 52\n",
      "gene: NRNX1-seg-52 -> 59\n",
      "gene: NRNX1-seg-53 -> 53\n",
      "gene: NRNX1-seg-54 -> 66\n",
      "gene: NRNX1-seg-55 -> 57\n",
      "gene: NRNX1-seg-56 -> 44\n",
      "gene: NRNX1-seg-57 -> 54\n",
      "gene: NRNX1-seg-58 -> 48\n",
      "gene: NRNX1-seg-59 -> 48\n",
      "gene: NRNX1-seg-60 -> 23\n",
      "gene: NRNX1-seg-62 -> 43\n",
      "gene: NRNX1-seg-63 -> 57\n",
      "gene: NRNX1-seg-64 -> 44\n",
      "gene: NRNX1-seg-65 -> 24\n",
      "gene: NRNX1-seg-66 -> 45\n",
      "gene: NRNX1-seg-67 -> 51\n",
      "gene: NRNX1-seg-68 -> 24\n",
      "gene: NRNX1-seg-69 -> 45\n",
      "gene: NRNX1-seg-70 -> 42\n",
      "gene: NRNX1-seg-71 -> 54\n",
      "gene: NRNX1-seg-72 -> 44\n",
      "gene: NRNX1-seg-73 -> 24\n",
      "gene: NRNX1-seg-74 -> 36\n",
      "gene: NRNX1-seg-75 -> 28\n",
      "gene: NRNX1-seg-76 -> 46\n",
      "gene: NRNX1-seg-77 -> 34\n",
      "gene: NRNX1-seg-79 -> 32\n",
      "gene: NRNX1-seg-80 -> 51\n",
      "gene: NRNX1-seg-81 -> 37\n",
      "gene: NRNX1-seg-82 -> 48\n",
      "gene: NRNX1-seg-83 -> 63\n",
      "gene: NRNX1-seg-84 -> 69\n",
      "gene: NRNX1-seg-85 -> 55\n",
      "gene: NRNX1-seg-86 -> 46\n",
      "gene: NRNX1-seg-87 -> 50\n",
      "gene: NRNX1-seg-88 -> 47\n",
      "gene: NRNX1-seg-89 -> 77\n",
      "gene: NRNX1-seg-90 -> 52\n",
      "gene: NRNX1-seg-91 -> 50\n",
      "gene: NRNX1-seg-92 -> 38\n",
      "gene: NRNX1-seg-93 -> 73\n",
      "gene: NRNX1-seg-94 -> 45\n",
      "gene: NRNX1-seg-95 -> 29\n",
      "gene: NRNX1-seg-96 -> 44\n",
      "gene: NRNX1-seg-97 -> 44\n",
      "gene: NRNX1-seg-98 -> 37\n",
      "gene: NRNX1-seg-99 -> 32\n",
      "gene: NRNX1-seg-100 -> 44\n",
      "gene: NRNX1-seg-101 -> 42\n",
      "gene: NRNX1-seg-102 -> 49\n",
      "gene: NRNX1-seg-103 -> 52\n",
      "gene: NRNX1-seg-104 -> 58\n",
      "gene: NRNX1-seg-105 -> 23\n",
      "gene: NRNX1-seg-106 -> 69\n",
      "gene: NRNX1-seg-107 -> 46\n",
      "gene: NRNX1-seg-108 -> 32\n",
      "gene: NRNX1-seg-109 -> 42\n",
      "gene: NRNX1-seg-110 -> 66\n",
      "gene: NRNX1-seg-111 -> 47\n",
      "gene: NRNX1-seg-112 -> 25\n",
      "gene: NRNX1-seg-113 -> 62\n",
      "gene: NRNX1-seg-114 -> 57\n",
      "gene: NRNX1-seg-115 -> 50\n",
      "gene: NRNX1-seg-117 -> 57\n",
      "gene: NRNX1-seg-118 -> 58\n",
      "gene: NRNX1-seg-119 -> 53\n",
      "gene: NRNX1-seg-120 -> 56\n",
      "gene: NRNX1-seg-121 -> 65\n",
      "gene: NRNX1-seg-122 -> 49\n",
      "gene: NRNX1-seg-123 -> 37\n",
      "gene: NRNX1-seg-124 -> 59\n",
      "gene: NRNX1-seg-125 -> 36\n",
      "gene: NRNX1-seg-126 -> 27\n",
      "gene: NRNX1-seg-127 -> 38\n",
      "gene: NRNX1-seg-128 -> 36\n",
      "gene: NRNX1-seg-129 -> 30\n",
      "gene: NRNX1-seg-130 -> 46\n",
      "gene: NRNX1-seg-131 -> 40\n",
      "gene: NRNX1-seg-133 -> 24\n",
      "gene: NRNX1-seg-134 -> 28\n",
      "gene: NRNX1-seg-135 -> 42\n",
      "gene: NRNX1-seg-136 -> 51\n",
      "gene: NRNX1-seg-137 -> 50\n",
      "gene: NRNX1-seg-138 -> 51\n",
      "gene: NRNX1-seg-139 -> 67\n",
      "gene: NRNX1-seg-140 -> 61\n",
      "gene: NRNX1-seg-141 -> 39\n",
      "gene: NRNX1-seg-142 -> 31\n",
      "gene: NRNX1-seg-143 -> 71\n",
      "gene: NRNX1-seg-144 -> 69\n",
      "gene: NRNX1-seg-145 -> 69\n",
      "gene: NRNX1-seg-146 -> 69\n",
      "gene: NRNX1-seg-147 -> 68\n",
      "gene: NRNX1-seg-148 -> 57\n",
      "gene: NRNX1-seg-150 -> 59\n",
      "gene: NRNX1-seg-151 -> 69\n",
      "gene: NRNX1-seg-152 -> 66\n",
      "gene: NRNX1-seg-154 -> 35\n",
      "gene: NRNX1-seg-155 -> 59\n",
      "gene: NRNX1-seg-156 -> 66\n",
      "gene: NRNX1-seg-157 -> 49\n",
      "gene: NRNX1-seg-158 -> 67\n",
      "gene: NRNX1-seg-159 -> 58\n",
      "gene: NRNX1-seg-160 -> 64\n",
      "gene: NRNX1-seg-161 -> 75\n",
      "gene: NRNX1-seg-162 -> 66\n",
      "gene: NRNX1-seg-163 -> 34\n",
      "gene: NRNX1-seg-164 -> 56\n",
      "gene: NRNX1-seg-165 -> 80\n",
      "gene: NRNX1-seg-166 -> 51\n",
      "gene: NRNX1-seg-169 -> 62\n",
      "gene: NRNX1-seg-170 -> 71\n",
      "gene: NRNX1-seg-171 -> 63\n",
      "gene: NRNX1-seg-172 -> 76\n",
      "gene: NRNX1-seg-173 -> 68\n",
      "gene: NRNX1-seg-174 -> 58\n",
      "gene: NRNX1-seg-175 -> 71\n",
      "gene: NRNX1-seg-176 -> 70\n",
      "gene: NRNX1-seg-177 -> 64\n",
      "gene: NRNX1-seg-178 -> 66\n",
      "gene: NRNX1-seg-179 -> 71\n",
      "gene: NRNX1-seg-180 -> 69\n",
      "gene: NRNX1-seg-181 -> 71\n",
      "gene: NRNX1-seg-182 -> 71\n",
      "gene: NRNX1-seg-183 -> 56\n",
      "gene: NRNX1-seg-184 -> 48\n",
      "gene: NRNX1-seg-185 -> 43\n",
      "gene: NRNX1-seg-186 -> 54\n",
      "gene: NRNX1-seg-187 -> 65\n",
      "gene: NRNX1-seg-188 -> 59\n",
      "gene: NRNX1-seg-189 -> 63\n",
      "gene: NRNX1-seg-190 -> 64\n",
      "gene: NRNX1-seg-191 -> 52\n",
      "gene: NRNX1-seg-192 -> 80\n",
      "gene: NRNX1-seg-193 -> 71\n",
      "gene: NRNX1-seg-194 -> 67\n",
      "gene: NRNX1-seg-195 -> 72\n",
      "gene: NRNX1-seg-196 -> 49\n",
      "gene: NRNX1-seg-197 -> 74\n",
      "gene: NRNX1-seg-198 -> 48\n",
      "gene: NRNX1-seg-199 -> 68\n",
      "gene: NRNX1-seg-200 -> 70\n",
      "gene: NRNX1-seg-201 -> 77\n",
      "gene: NRNX1-seg-202 -> 41\n",
      "gene: NRNX1-seg-203 -> 45\n",
      "gene: NRNX1-seg-204 -> 65\n",
      "gene: NRNX1-seg-205 -> 65\n",
      "gene: NRNX1-seg-206 -> 71\n",
      "gene: NRNX1-seg-207 -> 70\n",
      "gene: NRNX1-seg-208 -> 51\n",
      "gene: NRNX1-seg-209 -> 75\n",
      "gene: NRNX1-seg-210 -> 68\n",
      "gene: NRNX1-seg-211 -> 62\n",
      "gene: NRNX1-seg-212 -> 23\n",
      "gene: NRNX1-seg-213 -> 49\n",
      "gene: NRNX1-seg-214 -> 73\n",
      "gene: NRNX1-seg-215 -> 30\n",
      "gene: NRNX1-seg-216 -> 27\n",
      "gene: NRNX1-seg-217 -> 24\n",
      "gene: NRNX1-seg-218 -> 44\n",
      "gene: NRNX1-seg-219 -> 23\n",
      "gene: NRNX1-seg-220 -> 35\n",
      "gene: NRNX1-seg-221 -> 73\n",
      "gene: NRNX1-seg-222 -> 71\n",
      "gene: NRNX1-seg-223 -> 75\n",
      "gene: NRNX1-seg-224 -> 74\n",
      "gene: NRNX1-seg-225 -> 60\n",
      "gene: NRNX1-seg-226 -> 54\n",
      "gene: NRNX1-seg-227 -> 41\n",
      "gene: NRNX1-seg-228 -> 29\n",
      "gene: NRNX1-seg-229 -> 62\n",
      "gene: NRNX1-seg-230 -> 70\n",
      "gene: NRNX1-seg-231 -> 70\n",
      "gene: NRNX1-seg-232 -> 29\n",
      "gene: NRNX1-seg-233 -> 65\n",
      "gene: NRNX1-seg-234 -> 54\n",
      "gene: NRNX1-seg-235 -> 64\n",
      "gene: NRNX1-seg-236 -> 54\n",
      "gene: NRNX1-seg-238 -> 53\n",
      "gene: NRNX1-seg-239 -> 73\n",
      "gene: NRNX1-seg-240 -> 66\n",
      "gene: NRNX1-seg-241 -> 49\n",
      "gene: NRNX1-seg-242 -> 53\n",
      "gene: NRNX1-seg-244 -> 54\n",
      "gene: NRNX1-seg-245 -> 64\n",
      "gene: NRNX1-seg-246 -> 38\n",
      "gene: NRNX1-seg-248 -> 68\n",
      "gene: NRNX1-seg-249 -> 41\n",
      "gene: NRNX1-seg-251 -> 68\n",
      "gene: NRNX1-seg-252 -> 53\n",
      "gene: NRNX1-seg-253 -> 68\n",
      "gene: NRNX1-seg-254 -> 50\n",
      "gene: NRNX1-seg-255 -> 64\n",
      "gene: NRNX1-seg-256 -> 63\n",
      "gene: NRNX1-seg-257 -> 52\n",
      "gene: NRNX1-seg-258 -> 63\n",
      "gene: NRNX1-seg-259 -> 54\n",
      "gene: NRNX1-seg-260 -> 50\n",
      "gene: NRNX1-seg-262 -> 42\n",
      "gene: NRNX1-seg-263 -> 25\n",
      "gene: NRNX1-seg-264 -> 35\n",
      "gene: NRNX1-seg-265 -> 32\n",
      "gene: NRNX1-seg-266 -> 33\n",
      "gene: NRNX1-seg-267 -> 55\n",
      "gene: NRNX1-seg-268 -> 70\n",
      "gene: NRNX1-seg-269 -> 63\n",
      "gene: NRNX1-seg-270 -> 50\n",
      "gene: NRNX1-seg-271 -> 59\n",
      "gene: NRNX1-seg-272 -> 64\n",
      "gene: NRNX1-seg-273 -> 37\n",
      "gene: NRNX1-seg-274 -> 65\n",
      "gene: NRNX1-seg-275 -> 56\n",
      "gene: NRNX1-seg-276 -> 57\n",
      "gene: NRNX1-seg-277 -> 46\n",
      "gene: NRNX1-seg-278 -> 24\n",
      "gene: NRNX1-seg-279 -> 28\n",
      "gene: NRNX1-seg-280 -> 50\n",
      "gene: NRNX1-seg-281 -> 63\n",
      "gene: NRNX1-seg-282 -> 46\n",
      "gene: NRNX1-seg-283 -> 57\n",
      "gene: NRNX1-seg-284 -> 61\n",
      "gene: NRNX1-seg-285 -> 51\n",
      "gene: NRNX1-seg-286 -> 39\n",
      "gene: NRNX1-seg-287 -> 26\n",
      "gene: NRNX1-seg-288 -> 29\n",
      "gene: NRNX1-seg-289 -> 45\n",
      "gene: NRNX1-seg-290 -> 36\n",
      "gene: NRNX1-seg-291 -> 39\n",
      "gene: NRNX1-seg-292 -> 57\n",
      "gene: NRNX1-seg-293 -> 27\n",
      "gene: NRNX1-seg-294 -> 31\n",
      "gene: NRNX1-seg-295 -> 44\n",
      "gene: NRNX1-seg-298 -> 42\n",
      "gene: NRNX1-seg-299 -> 39\n",
      "gene: NRNX1-seg-300 -> 36\n",
      "gene: NRNX1-seg-301 -> 54\n",
      "gene: NRNX1-seg-302 -> 31\n",
      "gene: NRNX1-seg-303 -> 50\n",
      "gene: NRNX1-seg-304 -> 58\n",
      "gene: NRNX1-seg-305 -> 46\n",
      "gene: NRNX1-seg-306 -> 49\n",
      "gene: NRNX1-seg-307 -> 22\n",
      "gene: NRNX1-seg-308 -> 24\n",
      "gene: NRNX1-seg-311 -> 58\n",
      "gene: NRNX1-seg-312 -> 57\n",
      "gene: NRNX1-seg-313 -> 76\n",
      "gene: NRNX1-seg-314 -> 54\n",
      "gene: NRNX1-seg-315 -> 42\n",
      "gene: NRNX1-seg-316 -> 48\n",
      "gene: NRNX1-seg-317 -> 44\n",
      "gene: NRNX1-seg-318 -> 34\n",
      "gene: NRNX1-seg-319 -> 35\n",
      "gene: NRNX1-seg-320 -> 50\n",
      "gene: NRNX1-seg-321 -> 69\n",
      "gene: NRNX1-seg-322 -> 55\n",
      "gene: NRNX1-seg-323 -> 41\n",
      "gene: NRNX1-seg-324 -> 57\n",
      "gene: NRNX1-seg-325 -> 59\n",
      "gene: NRNX1-seg-326 -> 64\n",
      "gene: NRNX1-seg-327 -> 64\n",
      "gene: NRNX1-seg-328 -> 34\n",
      "gene: NRNX1-seg-329 -> 61\n",
      "gene: NRNX1-seg-330 -> 31\n",
      "gene: NRNX1-seg-331 -> 53\n",
      "gene: NRNX1-seg-332 -> 60\n",
      "gene: NRNX1-seg-333 -> 52\n",
      "gene: NRNX1-seg-334 -> 65\n",
      "gene: NRNX1-seg-335 -> 52\n",
      "gene: NRNX1-seg-336 -> 61\n",
      "gene: NRNX1-seg-337 -> 35\n",
      "gene: NRNX1-seg-338 -> 49\n",
      "gene: NRNX1-seg-339 -> 50\n",
      "gene: NRNX1-seg-340 -> 46\n",
      "gene: NRNX1-seg-341 -> 49\n",
      "gene: NRNX1-seg-342 -> 65\n",
      "gene: NRNX1-seg-343 -> 50\n",
      "gene: NRNX1-seg-344 -> 56\n",
      "gene: NRNX1-seg-345 -> 38\n",
      "gene: NRNX1-seg-346 -> 42\n",
      "gene: NRNX1-seg-347 -> 25\n",
      "gene: NRNX1-seg-348 -> 44\n",
      "gene: NRNX1-seg-349 -> 35\n",
      "gene: NRNX1-seg-350 -> 37\n",
      "gene: NRNX1-seg-351 -> 53\n",
      "gene: NRNX1-seg-352 -> 36\n",
      "gene: NRNX1-seg-353 -> 59\n",
      "gene: NRNX1-seg-354 -> 50\n",
      "gene: NRNX1-seg-355 -> 60\n",
      "gene: NRNX1-seg-356 -> 40\n",
      "gene: NRNX1-seg-357 -> 73\n",
      "gene: NRNX1-seg-358 -> 47\n",
      "gene: NRNX1-seg-359 -> 52\n",
      "gene: NRNX1-seg-360 -> 56\n",
      "gene: NRNX1-seg-361 -> 61\n",
      "gene: NRNX1-seg-362 -> 40\n",
      "gene: NRNX1-seg-363 -> 55\n",
      "gene: NRNX1-seg-364 -> 57\n",
      "gene: NRNX1-seg-365 -> 37\n",
      "gene: NRNX1-seg-366 -> 42\n",
      "gene: NRNX1-seg-367 -> 64\n",
      "gene: NRNX1-seg-368 -> 47\n",
      "gene: NRNX1-seg-369 -> 56\n",
      "gene: NRNX1-seg-370 -> 51\n",
      "gene: NRNX1-seg-371 -> 47\n",
      "gene: NRNX1-seg-372 -> 65\n",
      "gene: NRNX1-seg-373 -> 55\n",
      "gene: NRNX1-seg-374 -> 31\n",
      "gene: NRNX1-seg-375 -> 21\n",
      "gene: NRNX1-seg-376 -> 25\n",
      "gene: NRNX1-seg-379 -> 30\n",
      "gene: NRNX1-seg-380 -> 49\n",
      "gene: NRNX1-seg-381 -> 60\n",
      "gene: NRNX1-seg-382 -> 43\n",
      "gene: NRNX1-seg-383 -> 52\n",
      "gene: NRNX1-seg-384 -> 41\n",
      "gene: NRNX1-seg-385 -> 55\n",
      "gene: NRNX1-seg-386 -> 58\n",
      "gene: NRNX1-seg-387 -> 45\n",
      "gene: NRNX1-seg-388 -> 62\n",
      "gene: NRNX1-seg-389 -> 60\n",
      "gene: NRNX1-seg-390 -> 32\n",
      "gene: NRNX1-seg-391 -> 61\n",
      "gene: NRNX1-seg-392 -> 67\n",
      "gene: NRNX1-seg-393 -> 49\n",
      "gene: NRNX1-seg-394 -> 53\n",
      "gene: NRNX1-seg-395 -> 56\n",
      "gene: NRNX1-seg-396 -> 43\n",
      "gene: NRNX1-seg-397 -> 52\n",
      "gene: NRNX1-seg-398 -> 47\n",
      "gene: NRNX1-seg-399 -> 48\n",
      "gene: NRNX1-seg-400 -> 55\n",
      "gene: NRNX1-seg-401 -> 61\n",
      "gene: NRNX1-seg-402 -> 55\n",
      "gene: NRNX1-seg-403 -> 64\n",
      "gene: NRNX1-seg-404 -> 58\n",
      "gene: NRNX1-seg-405 -> 47\n",
      "gene: NRNX1-seg-406 -> 54\n",
      "gene: NRNX1-seg-407 -> 39\n",
      "gene: NRNX1-seg-408 -> 45\n",
      "gene: NRNX1-seg-409 -> 44\n",
      "gene: NRNX1-seg-410 -> 51\n",
      "gene: NRNX1-seg-411 -> 46\n",
      "gene: NRNX1-seg-412 -> 37\n",
      "gene: NRNX1-seg-413 -> 41\n",
      "gene: NRNX1-seg-414 -> 58\n",
      "gene: NRNX1-seg-415 -> 49\n",
      "gene: NRNX1-seg-416 -> 52\n",
      "gene: NRNX1-seg-417 -> 45\n",
      "gene: NRNX1-seg-418 -> 41\n",
      "gene: NRNX1-seg-419 -> 54\n",
      "gene: NRNX1-seg-420 -> 55\n",
      "gene: NRNX1-seg-421 -> 62\n",
      "gene: NRNX1-seg-422 -> 48\n",
      "gene: NRNX1-seg-423 -> 54\n",
      "gene: NRNX1-seg-424 -> 59\n",
      "gene: NRNX1-seg-425 -> 49\n",
      "gene: NRNX1-seg-426 -> 53\n",
      "gene: NRNX1-seg-427 -> 67\n",
      "gene: NRNX1-seg-428 -> 46\n",
      "gene: NRNX1-seg-429 -> 46\n",
      "gene: NRNX1-seg-430 -> 58\n",
      "gene: NRNX1-seg-431 -> 41\n",
      "gene: NRNX1-seg-432 -> 55\n",
      "gene: NRNX1-seg-433 -> 54\n",
      "gene: NRNX1-seg-434 -> 52\n",
      "gene: NRNX1-seg-435 -> 47\n",
      "gene: NRNX1-seg-436 -> 52\n",
      "gene: NRNX1-seg-437 -> 41\n",
      "gene: NRNX1-seg-438 -> 34\n",
      "gene: NRNX1-seg-439 -> 38\n",
      "gene: NRNX1-seg-440 -> 59\n",
      "gene: NRNX1-seg-441 -> 57\n",
      "gene: NRNX1-seg-442 -> 50\n",
      "gene: NRNX1-seg-443 -> 39\n",
      "gene: NRNX1-seg-444 -> 46\n",
      "gene: NRNX1-seg-445 -> 62\n",
      "gene: NRNX1-seg-446 -> 61\n",
      "gene: NRNX1-seg-447 -> 41\n",
      "gene: NRNX1-seg-448 -> 53\n",
      "gene: NRNX1-seg-449 -> 43\n",
      "gene: NRNX1-seg-450 -> 51\n",
      "gene: NRNX1-seg-452 -> 50\n",
      "gene: NRNX1-seg-453 -> 65\n",
      "gene: NRNX1-seg-454 -> 48\n",
      "gene: NRNX1-seg-455 -> 43\n",
      "gene: NRNX1-seg-456 -> 49\n",
      "gene: NRNX1-seg-457 -> 47\n",
      "gene: NRNX1-seg-458 -> 44\n",
      "gene: NRNX1-seg-459 -> 68\n",
      "gene: NRNX1-seg-460 -> 49\n",
      "gene: NRNX1-seg-461 -> 52\n",
      "gene: NRNX1-seg-462 -> 49\n",
      "gene: NRNX1-seg-463 -> 59\n",
      "gene: NRNX1-seg-464 -> 53\n",
      "gene: NRNX1-seg-465 -> 60\n",
      "gene: NRNX1-seg-466 -> 34\n",
      "gene: NRNX1-seg-467 -> 61\n",
      "gene: NRNX1-seg-468 -> 51\n",
      "gene: NRNX1-seg-469 -> 68\n",
      "gene: NRNX1-seg-470 -> 79\n",
      "gene: NRNX1-seg-471 -> 62\n",
      "gene: NRNX1-seg-472 -> 26\n",
      "gene: NRNX1-seg-474 -> 61\n",
      "gene: NRNX1-seg-475 -> 60\n",
      "gene: NRNX1-seg-476 -> 51\n",
      "gene: NRNX1-seg-477 -> 58\n",
      "gene: NRNX1-seg-478 -> 52\n",
      "gene: NRNX1-seg-479 -> 58\n",
      "gene: NRNX1-seg-480 -> 58\n",
      "gene: NRNX1-seg-481 -> 66\n",
      "gene: NRNX1-seg-482 -> 60\n",
      "gene: NRNX1-seg-483 -> 25\n",
      "gene: NRNX1-seg-484 -> 59\n",
      "gene: NRNX1-seg-485 -> 63\n",
      "gene: NRNX1-seg-486 -> 50\n",
      "gene: NRNX1-seg-487 -> 64\n",
      "gene: NRNX1-seg-488 -> 49\n",
      "gene: NRNX1-seg-489 -> 57\n",
      "gene: NRNX1-seg-490 -> 58\n",
      "gene: NRNX1-seg-491 -> 50\n",
      "gene: NRNX1-seg-492 -> 47\n",
      "gene: NRNX1-seg-493 -> 54\n",
      "gene: NRNX1-seg-494 -> 49\n",
      "gene: NRNX1-seg-495 -> 59\n",
      "gene: NRNX1-seg-496 -> 65\n",
      "gene: NRNX1-seg-497 -> 52\n",
      "gene: NRNX1-seg-498 -> 80\n",
      "gene: NRNX1-seg-499 -> 66\n",
      "gene: NRNX1-seg-500 -> 67\n",
      "gene: NRNX1-seg-501 -> 49\n",
      "gene: NRNX1-seg-502 -> 65\n",
      "gene: NRNX1-seg-503 -> 49\n",
      "gene: NRNX1-seg-504 -> 47\n",
      "gene: NRNX1-seg-505 -> 44\n",
      "gene: NRNX1-seg-506 -> 60\n",
      "gene: NRNX1-seg-507 -> 58\n",
      "gene: NRNX1-seg-508 -> 62\n",
      "gene: NRNX1-seg-509 -> 45\n",
      "gene: NRNX1-seg-510 -> 55\n",
      "gene: NRNX1-seg-511 -> 52\n",
      "gene: NRNX1-seg-512 -> 56\n",
      "gene: NRNX1-seg-513 -> 52\n",
      "gene: NRNX1-seg-514 -> 51\n",
      "gene: NRNX1-seg-515 -> 63\n",
      "gene: NRNX1-seg-516 -> 52\n",
      "gene: NRNX1-seg-517 -> 59\n",
      "gene: NRNX1-seg-518 -> 55\n",
      "gene: NRNX1-seg-519 -> 62\n",
      "gene: NRNX1-seg-520 -> 51\n",
      "gene: NRNX1-seg-521 -> 61\n",
      "gene: NRNX1-seg-522 -> 50\n",
      "gene: NRNX1-seg-523 -> 59\n",
      "gene: NRNX1-seg-524 -> 55\n",
      "gene: NRNX1-seg-525 -> 58\n",
      "gene: NRNX1-seg-526 -> 44\n",
      "gene: NRNX1-seg-527 -> 57\n",
      "gene: NRNX1-seg-528 -> 47\n",
      "gene: NRNX1-seg-529 -> 50\n",
      "gene: NRNX1-seg-530 -> 61\n",
      "gene: NRNX1-seg-531 -> 48\n",
      "gene: NRNX1-seg-532 -> 61\n",
      "gene: NRNX1-seg-533 -> 67\n",
      "gene: NRNX1-seg-534 -> 52\n",
      "gene: NRNX1-seg-535 -> 39\n",
      "gene: NRNX1-seg-536 -> 59\n",
      "gene: NRNX1-seg-537 -> 51\n",
      "gene: NRNX1-seg-538 -> 65\n",
      "gene: NRNX1-seg-539 -> 55\n",
      "gene: NRNX1-seg-540 -> 54\n",
      "gene: NRNX1-seg-541 -> 45\n",
      "gene: NRNX1-seg-542 -> 65\n",
      "gene: NRNX1-seg-543 -> 52\n",
      "gene: NRNX1-seg-544 -> 47\n",
      "gene: NRNX1-seg-545 -> 60\n",
      "gene: NRNX1-seg-546 -> 52\n",
      "gene: NRNX1-seg-547 -> 46\n",
      "gene: NRNX1-seg-548 -> 47\n",
      "gene: NRNX1-seg-549 -> 46\n",
      "gene: NRNX1-seg-550 -> 56\n",
      "gene: NRNX1-seg-551 -> 48\n",
      "gene: NRNX1-seg-552 -> 58\n",
      "gene: NRNX1-seg-553 -> 52\n",
      "gene: NRNX1-seg-554 -> 65\n",
      "gene: NRNX1-seg-555 -> 59\n",
      "gene: NRNX1-seg-556 -> 53\n",
      "gene: NRNX1-seg-557 -> 57\n",
      "gene: NRNX1-seg-558 -> 59\n",
      "gene: NRNX1-seg-559 -> 52\n",
      "gene: NRNX1-seg-560 -> 61\n",
      "gene: NRNX1-seg-561 -> 40\n",
      "gene: NRNX1-seg-562 -> 49\n",
      "gene: NRNX1-seg-563 -> 43\n",
      "gene: NRNX1-seg-564 -> 45\n",
      "gene: NRNX1-seg-565 -> 49\n",
      "gene: NRNX1-seg-566 -> 41\n",
      "gene: NRNX1-seg-567 -> 50\n",
      "gene: NRNX1-seg-568 -> 46\n",
      "gene: NRNX1-seg-569 -> 64\n",
      "gene: NRNX1-seg-570 -> 61\n",
      "gene: NRNX1-seg-571 -> 55\n",
      "gene: NRNX1-seg-572 -> 58\n",
      "gene: NRNX1-seg-573 -> 61\n",
      "gene: NRNX1-seg-574 -> 43\n",
      "gene: NRNX1-seg-575 -> 47\n",
      "gene: NRNX1-seg-576 -> 61\n",
      "gene: NRNX1-seg-577 -> 45\n",
      "gene: NRNX1-seg-578 -> 39\n",
      "gene: NRNX1-seg-579 -> 41\n",
      "gene: NRNX1-seg-580 -> 52\n",
      "gene: NRNX1-seg-581 -> 52\n",
      "gene: NRNX1-seg-582 -> 59\n",
      "gene: NRNX1-seg-583 -> 65\n",
      "gene: NRNX1-seg-584 -> 39\n",
      "gene: NRNX1-seg-585 -> 51\n",
      "gene: NRNX1-seg-586 -> 54\n",
      "gene: NRNX1-seg-587 -> 35\n",
      "gene: NRNX1-seg-588 -> 38\n",
      "gene: NRNX1-seg-589 -> 41\n",
      "gene: NRNX1-seg-590 -> 52\n",
      "gene: NRNX1-seg-591 -> 49\n",
      "gene: NRNX1-seg-592 -> 50\n",
      "gene: NRNX1-seg-593 -> 55\n",
      "gene: NRNX1-seg-594 -> 45\n",
      "gene: NRNX1-seg-595 -> 43\n",
      "gene: NRNX1-seg-596 -> 45\n",
      "gene: NRNX1-seg-597 -> 39\n",
      "gene: NRNX1-seg-598 -> 46\n",
      "gene: NRNX1-seg-599 -> 59\n",
      "gene: NRNX1-seg-600 -> 54\n",
      "gene: NRNX1-seg-601 -> 53\n",
      "gene: NRNX1-seg-602 -> 42\n",
      "gene: NRNX1-seg-603 -> 60\n",
      "gene: NRNX1-seg-604 -> 38\n",
      "gene: NRNX1-seg-605 -> 55\n",
      "gene: NRNX1-seg-606 -> 67\n",
      "gene: NRNX1-seg-607 -> 68\n",
      "gene: NRNX1-seg-608 -> 56\n",
      "gene: NRNX1-seg-609 -> 32\n",
      "gene: NRNX1-seg-610 -> 27\n",
      "gene: NRNX1-seg-611 -> 47\n",
      "gene: NRNX1-seg-612 -> 30\n",
      "gene: NRNX1-seg-613 -> 28\n",
      "gene: NRNX1-seg-614 -> 49\n",
      "gene: NRNX1-seg-615 -> 33\n",
      "gene: NRNX1-seg-616 -> 42\n",
      "gene: NRNX1-seg-617 -> 48\n",
      "gene: NRNX1-seg-618 -> 60\n",
      "gene: NRNX1-seg-619 -> 73\n",
      "gene: NRNX1-seg-620 -> 75\n",
      "gene: NRNX1-seg-621 -> 56\n",
      "gene: NRNX1-seg-622 -> 65\n",
      "gene: NRNX1-seg-623 -> 50\n",
      "gene: NRNX1-seg-624 -> 58\n",
      "gene: NRNX1-seg-625 -> 39\n",
      "gene: NRNX1-seg-626 -> 48\n",
      "gene: NRNX1-seg-627 -> 44\n",
      "gene: NRNX1-seg-628 -> 37\n",
      "gene: NRNX1-seg-629 -> 40\n",
      "gene: NRNX1-seg-630 -> 34\n",
      "gene: NRNX1-seg-631 -> 55\n",
      "gene: NRNX1-seg-632 -> 57\n",
      "gene: NRNX1-seg-633 -> 51\n",
      "gene: NRNX1-seg-634 -> 64\n",
      "gene: NRNX1-seg-635 -> 45\n",
      "gene: NRNX1-seg-636 -> 32\n",
      "gene: NRNX1-seg-637 -> 22\n",
      "gene: NRNX1-seg-639 -> 32\n",
      "gene: NRNX1-seg-640 -> 54\n",
      "gene: NRNX1-seg-641 -> 46\n",
      "gene: NRNX1-seg-642 -> 45\n",
      "gene: NRNX1-seg-643 -> 57\n",
      "gene: NRNX1-seg-644 -> 58\n",
      "gene: NRNX1-seg-645 -> 49\n",
      "gene: NRNX1-seg-646 -> 69\n",
      "gene: NRNX1-seg-647 -> 33\n",
      "gene: NRNX1-seg-648 -> 41\n",
      "gene: NRNX1-seg-649 -> 45\n",
      "gene: NRNX1-seg-650 -> 44\n",
      "gene: NRNX1-seg-651 -> 47\n",
      "gene: NRNX1-seg-652 -> 35\n",
      "gene: NRNX1-seg-653 -> 49\n",
      "gene: NRNX1-seg-654 -> 57\n",
      "gene: NRNX1-seg-655 -> 55\n",
      "gene: NRNX1-seg-656 -> 50\n",
      "gene: NRNX1-seg-657 -> 49\n",
      "gene: NRNX1-seg-658 -> 60\n",
      "gene: NRNX1-seg-659 -> 50\n",
      "gene: NRNX1-seg-660 -> 50\n",
      "gene: NRNX1-seg-661 -> 51\n",
      "gene: NRNX1-seg-662 -> 46\n",
      "gene: NRNX1-seg-665 -> 53\n",
      "gene: NRNX1-seg-666 -> 68\n",
      "gene: NRNX1-seg-667 -> 50\n",
      "gene: NRNX1-seg-668 -> 49\n",
      "gene: NRNX1-seg-669 -> 45\n",
      "gene: NRNX1-seg-670 -> 33\n",
      "gene: NRNX1-seg-671 -> 54\n",
      "gene: NRNX1-seg-672 -> 38\n",
      "gene: NRNX1-seg-674 -> 47\n",
      "gene: NRNX1-seg-675 -> 43\n",
      "gene: NRNX1-seg-676 -> 40\n",
      "gene: NRNX1-seg-678 -> 44\n",
      "gene: NRNX1-seg-679 -> 22\n",
      "gene: NRNX1-seg-681 -> 31\n",
      "gene: NRNX1-seg-682 -> 39\n",
      "gene: NRNX1-seg-683 -> 53\n",
      "gene: NRNX1-seg-684 -> 23\n",
      "gene: NRNX1-seg-685 -> 37\n",
      "gene: NRNX1-seg-686 -> 28\n",
      "gene: NRNX1-seg-687 -> 27\n",
      "gene: NRNX1-seg-688 -> 28\n",
      "gene: NRNX1-seg-689 -> 45\n",
      "gene: NRNX1-seg-690 -> 43\n",
      "gene: NRNX1-seg-691 -> 36\n",
      "gene: NRNX1-seg-692 -> 47\n",
      "gene: NRNX1-seg-693 -> 54\n",
      "gene: NRNX1-seg-694 -> 49\n",
      "gene: NRNX1-seg-695 -> 57\n",
      "gene: NRNX1-seg-696 -> 40\n",
      "gene: NRNX1-seg-697 -> 42\n",
      "gene: NRNX1-seg-698 -> 61\n",
      "gene: NRNX1-seg-699 -> 46\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "reg_size_dic, len_check = check._check_region_size(full_records, \n",
    "                                                   min_size=min_num_probes)\n",
    "print(len_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "id": "95f60f31",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7t3QuwZWN6P/5HU+ghIaWRpjvpcQtxG8y0oZCfiNFkomSY1tOJW1Ez7ipu5RKl26U1KobUaNdgSruUuyEuYYyMzEQQRBdx7Wk6dKHQOnHrDP2vd+XfR1/OOb3eba+zeq/12VVKcd53v3t93mft8z3PWXudFRYsWLAgPAgQIECAAAECBAi0RGAFAbglO+0wCRAgQIAAAQIECgEBWCEQIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBWA0QIECAAAECBAi0SkAAbtV2O1gCBAgQIECAAAEBuMIa+PTTT2PGjBmx9tprx0orrVThSp6aAAECBAgQILC4wG9/+9t49913Y8stt4xVV10VzyICAnCF5fDkk0/G2LFjK1zBUxMgQIAAAQIEBhd44okn4lvf+hYmAXhoamDWrFnx9a9/PVLhjRw5cmgWtQoBAgQIECBAICLmzJlTNOJ+85vfxJgxY5gIwENTA//1X/8Vo0ePjtmzZ8eoUaOGZlGrECBAgAABAgQiQg4ZuAxcAlHhKaLwKsT11AQIECBAgMCgAnKIAFzLKaLwamG3KAECBAgQIKADPGgN6ABXeIoIwBXiemoCBAgQIEBAB7jDGhCAO4QrM00ALqNkDAECBAgQIFCFgBwysKoAXEXF/f/PqfAqxPXUBAgQIECAgA5whzUgAHcIV2aaAFxGyRgCBAgQIECgCgE5RAe4irpa5nMqvGUSGUCAAAECBAhUJCCHCMAVldbgT6vwamG3KAECBAgQIOAuEIPWgEsgKjxFBOAKcT01AQIECBAgMKiAHKIDXMspovBqYbcoAQIECBAgoAOsA1zXWSAA1yVvXQIECBAgQEAO0QGu5SxQeLWwW5QAAQIECBDQAdYBrussEIDrkrcuAQIECBAgIIfoANdyFii8WtgtSoAAAQIECOgA6wDXdRYIwHXJW5cAAQIECBCQQ3SAazkLFF4t7BYlQIAAAQIEdIB1gOs6CwTguuStS4AAAQIECMghOsC1nAUKr3r2Maf8Y/Yis6b+efYcEwgQIECAQK8JyCECcC01q/CqZxeAqze2AgECBAj0poAcIgDXUrkKr3p2Abh6YysQIECAQG8KyCECcC2Vq/CqZxeAqze2AgECBAj0poAcIgDXUrkKr3p2Abh6YysQIECAQG8KyCECcC2Vq/CqZxeAqze2AgECBAj0poAcIgDXUrkKr3p2Abh6YysQIECAQG8KyCECcC2Vq/CqZxeAqze2AgECBAj0poAcIgDXUrkKr3p2Abh6YysQIECAQG8KyCECcC2Vq/CqZxeAqze2AgECBAj0poAcIgDXUrkKr3p2Abh6YysQIECAQG8KyCECcC2Vq/CqZxeAqze2AgECBAj0poAcIgDXUrkKr3p2Abh6YysQIECAQG8KyCECcC2Vq/CqZxeAqze2AgECBAj0poAcIgDXUrkKr3p2Abh6YysQIECAQG8KyCECcC2Vq/CqZxeAqze2AgECBAj0poAc0rIA/P7778dpp50W9957b8ybNy+22mqrmDp1auy0006FxLPPPhvHHXdcPPXUU7HWWmvFscceGyeeeGKf0hdffBGTJ0+Oq6++Oj744INi3rRp02KjjTbKOgMUXhZXR4MF4I7YTCJAgACBFgjIIS0LwN/5znfinXfeiZ/85Cex9tprx6WXXlqE2aeffrr470033TT22WefOP744+Pxxx+PI488sgi4hxxySCGVwm/672uvvTbWX3/9OPnkk2PmzJnx/PPPx8orr1z6lFF4pak6HigAd0xnIgECBAg0XEAOaVEAfvXVV2PjjTeOX/3qV7HjjjsWR75gwYLYZJNN4gc/+EEMHz68CMSzZs2KlVZaqfh66hbfcccd8eKLL8b8+fNjxIgRccEFF8Thhx9efH3u3Lmx3nrrxTXXXBMTJkwofboovNJUHQ8UgDumM5EAAQIEGi4gh7QoAKew+utf/zp22223WGWVVfqOPF2+MG7cuKKTu+aaa8aNN97Y97WHH344dt9993j77beLYLz99tvHSy+9VITmhY90GUS6lCJ1hss+FF5Zqc7HCcCd25lJgAABAs0WkENaFID7O9Rbb701xo8fHz/72c/i9NNPjz333DPOP//8vqEvvPBCbL755vHkk0/GG2+8Efvuu298/PHHRbd44SPNT/8vXVdc9qHwykp1Pk4A7tzOTAIECBBotoAc0uIAnC6FSJ3f1BG+6667ig+yTZw4Mc4666w+ldQV3nDDDeOxxx4rOsAHHHBAfP755zFs2LC+MQceeGC89dZbkbrFAz3SB+7SPwsfc+bMibFjx8bs2bNj1KhRzT7Lajo6AbgmeMsSIECAwHIvIAC3NADffffdRdj99re/XXRuU0d3yy23jL322qvfDnD6kFwKw/vtt1+/HeDPPvss0nMO9Jg0aVLxAbolHwJwde8RAnB1tp6ZAAECBHpbQABuYQBOd4BItzr73ve+F9OnT++7HjiF33Trs+uvv75Ppb9rgNOH6VJXeOEjXQO89dZbFx+g0wFeft4QBODlZy+8EgIECBBYvgQE4JYF4Msuu6y4tVm6v++Pf/zjxS5lSPcDvvzyy+O1116LFVdcsZA59dRT48477yzuApG6vOuss05cdNFFceihhxZfdxeI5euEXvTVCMDL7954ZQQIECBQr4AA3KIA/PLLL8cWW2wR3/3ud5e6Y0O6BCIF3HQf4L333ru4v+8TTzwRRxxxRBGKDzrooEIqfVDuiiuuKG57NmbMmDjppJOKa4NnzJjhPsD1nstLrS4AL2cb4uUQIECAwHIjIAC3KABPmTKlCLD9PVLAve6664q7PaTu8DPPPBMjR46ME044IY4++ui+KekDcOnewOkPYXzyySexyy67FJc+pDCc81B4OVqdjRWAO3MziwABAgSaLyCHtCgAL0/lrPCq3w0BuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIQJwLZWr8KpnF4CrN7YCAQIECPSmgBwiANdSuQqvenYBuHpjKxAgQIBAbwrIIS0OwOecc048/PDD8eijjxYK/+///b/453/+535FfvrTn8aBBx4Yr7/+eowZM2apMVdddVUcdthhpc8ChVeaquOBAnDHdCYSIECAQMMF5JCWBuCLL744jj/++Nhll136AvD7778f8+fPX0wkhdrXXnstnnjiifid3/mduOeee2L8+PExc+bMWGGFFfrGrrHGGjF8+PDSp4vCK03V8UABuGM6EwkQIECg4QJySMsC8Jtvvll0ah977LEYPXp0rLvuun0BeEmKm266qej6Pv3007HlllsWX54yZUrccsst8eyzz36lU0PhfSW+UpMF4FJMBhEgQIBACwXkkJYF4NTBTcE2Xf5w1llnxaxZs/oNwB999FH80R/9UUycODEuuOCCPqUJEybEsGHD4sYbb/xKp4vCy+frJNDmrjJr6p/nTjGeAAECBAj0nIAc0rIAvOjhHnzwwQMG4L/7u78rAvIbb7wR6fKGhY/NN988Ro4cGZ999lm8/PLLsfHGG8cZZ5wRe+yxx6DFP2/evEj/LHzMmTMnxo4dG7Nnz45Ro0b13IlTxwsWgOtQtyYBAgQINFFAABaAl+oAf/7557HBBhsU3d/zzjuvTyhdH7zaaqvFNttsExdeeGGsvvrqMX369LjkkkvioYceit12221AzUmTJsXkyZOX+roAXP5tRQAub2UkAQIECBAYTEAAFoCXCsA///nP48/+7M+KD7+lILzoI10asdJKK8Uqq6zS97/HjRtXfCDu/vvvH1BTB/irvxEJwF/d0DMQIECAAIEkIAALwEsF4GOOOaa468O//du/lTpLTjrppHjggQdixowZpcYrvNJMiw0UgDtzM4sAAQIECCwpIAALwEsF4HSJQ+rqLnr5Q2J67rnnYscddyzC7k477dQnt+uuuxZ3k7j55ptLn2EKrzRV30ABON/MDAIECBAg0J+AHCIALxaA0/W/6X6+1113XXEN8KKPL774ogjA6TKIyy67LEaMGBFXXnllTJs2LZ588sm+W6WVOdUUXhmlxccIwPlmZhAgQIAAAQE4rwZWWLBgwYK8Kb01ur+7QLzzzjtFNzd1efu7s8O7774bp5xyStx3330xd+7c2HbbbWPq1Kmx8847Zx28AJzFVQwWgPPNzCBAgAABAgJwXg00PgDncXR3tACc7ykA55uZQYAAAQIEBOC8GhCA87yyRgvAWVw6wPlcZhAgQIAAgQEF5JCBi0MArvDEUXj5uDrA+WZmECBAgAABHeC8GhCA87yyRgvAWVw6wPlcZhAgQIAAAR3gDmpAAO4ArewUAbis1JfjdIDzzcwgQIAAAQI6wHk1IADneWWNFoCzuHSA87nMIECAAAECOsAd1IAA3AFa2SkCcFkpHeB8KTMIECBAgMDgAnLIwD4CcIVnj8LLx3UJRL6ZGQQIECBAoD8BOUQAruXMUHj57AJwvpkZBAgQIEBAAM6rAR3gPK+s0QJwFlcxWADONzODAAECBAgIwHk1IADneWWNFoCzuATgfC4zCBAgQIDAgAJyyMDFIQBXeOIovHxcHeB8MzMIECBAgIAOcF4NCMB5XlmjBeAsLh3gfC4zCBAgQICADnAHNSAAd4BWdooAXFbqy3E6wPlmZhAgQIAAAR3gvBoQgPO8skYLwFlcOsD5XGYQIECAAAEd4A5qQADuAK3sFAG4rJQOcL6UGQQIECBAYHABOWRgHwG4wrNH4eXjugQi38wMAgQIECDQn4AcIgDXcmYovHx2ATjfzAwCBAgQICAA59WADnCeV9ZoATiLqxgsAOebmUGAAAECBATgvBoQgPO8skYLwFlcAnA+lxkECBAgQGBAATlk4OIQgCs8cRRePq4OcL6ZGQQIECBAQAc4rwYE4DyvrNECcBaXDnA+lxkECBAgQEAHuIMaEIA7QCs7RQAuK/XlOB3gfDMzCBAgQICADnBeDQjAeV5ZowXgLC4d4HwuMwgQIECAgA5wBzUgAHeAVnaKAFxWSgc4X8oMAgQIECAwuIAcMrCPAFzh2aPw8nFdApFvZgYBAgQIEOhPQA4RgGs5MxRePrsAnG9mBgECBAgQEIDzakAHOM8ra7QAnMVVDBaA883MIECAAAECAnBeDQjAeV5ZowXgLC4BOJ/LDAIECBAgMKCAHDJwcQjAFZ44Ci8fVwc438wMAgQIECCgA5xXAwJwnlfWaAE4i0sHOJ/LDAIECBAgoAPcQQ0IwB2glZ0iAJeV+nKcDnC+mRkECBAgQEAHOK8GBOA8r6zRAnAWlw5wPpcZBAgQIEBAB7iDGhCAO0ArO0UALiulA5wvZQYBAgQIEBhcQA4Z2KfxAficc86Jhx9+OB599NE+hUMOOSSuu+66xVTWX3/9SIWSHl988UVMnjw5rr766vjggw9ip512imnTpsVGG22Uda4pvCwuHeB8LjMIECBAgIAOcAc10OgAfPHFF8fxxx8fu+yyy2IBeLvttotx48bFMccc00e24oorxtprr138dwq/KfBee+21kYLxySefHDNnzoznn38+Vl555dLMAnBpqr6BrgHONzODAAECBAj0JyCHtKwD/Oabb8Zhhx0Wjz32WIwePTrWXXfdvgD8+eefx+qrrx433XRT7LPPPkvJzJ8/P0aMGBEXXHBBHH744cXX586dG+utt15cc801MWHChNJnmcIrTSUA51OZQYAAAQIEBhWQQ1oWgO+5554i4KbLH84666yYNWtWXwB+8cUXY7PNNouXXnopNtlkk6Vknnjiidh+++2X+nq6DGKrrbYqOsNlHwqvrNSX43SA883MIECAAAECOsB5NdDoSyASxcEHH7xYAL711ltj//33L7q7999/fwwbNiz22muvIiyvscYacccdd8S+++4bH3/8cQwfPrxPc/z48cX/u/fee0sLC8ClqXSA86nMIECAAAECOsAd1kDrAvCZZ54Z5557bkyZMqUIvq+88kqceOKJxaUSjzzySNx4441xwAEHRLpUIoXjhY8DDzww3nrrreIDdQM95s2bF+mfhY85c+bE2LFjY/bs2TFq1KgOt6hd03SA27XfjpYAAQIEqhPQiBvYtnUBeMGCBfHhhx/Gmmuu2afy+OOPxw477BDp36lY9ttvv347wJ999lncfffdA2pOmjSp+ADdkg8BuPzJLQCXtzKSAAECBAgMJiAAC8CL3QViSY6PPvqo+GDcbbfdVnSC0zXAr776amy44YZ9Q9M1wFtvvXVceumlOsAVvt8IwBXiemoCBAgQaJWAACwA9wXgiRMnxnvvvRcPPvhgn0q6R/Cuu+4aL7zwQmywwQaxzjrrxEUXXRSHHnpoMcZdIIbu/UIAHjprKxEgQIBAswUEYAG4LwCnD7HtvffecfbZZxe3NHv55ZfjqKOOKi6BuOGGGwqp008/Pa644oritmdjxoyJk046qfgg3YwZM9wHuOL3CgG4YmBPT4AAAQKtERCABeDFLoG4/fbbiw/CpVuipWuBU1c43QVi1VVXLaTSB+BOO+204g9hfPLJJ8Uf0kiXPqQwnPNQeDla/zdWAM43M4MAAQIECPQnIIe0OADXeUoovHx9ATjfzAwCBAgQICAA59VA4+8CkcfR3dECcL6nAJxvZgYBAgQIEBCA82pAAM7zyhotAGdxFYMF4HwzMwgQIECAgACcVwMCcJ5X1mgBOItLAM7nMoMAAQIECAwoIIcMXBwCcIUnjsLLx9UBzjczgwABAgQI6ADn1YAAnOeVNVoAzuLSAc7nMoMAAQIECOgAd1ADAnAHaGWnCMBlpb4cpwOcb2YGAQIECBDQAc6rAQE4zytrtACcxaUDnM9lBgECBAgQ0AHuoAYE4A7Qyk4RgMtK6QDnS5lBgAABAgQGF5BDBvYRgCs8exRePq5LIPLNzCBAgAABAv0JyCECcC1nhsLLZxeA883MIECAAAECAnBeDegA53lljRaAs7iKwQJwvpkZBAgQIEBAAM6rAQE4zytrtACcxSUA53OZQYAAAQIEBhSQQwYuDgG4whNH4eXj6gDnm5lBgAABAgR0gPNqQADO88oaLQBncekA53OZQYAAAQIEdIA7qAEBuAO0slME4LJSX47TAc43M4MAAQIECOgA59WAAJznlTVaAM7i0gHO5zKDAAECBAjoAHdQAwJwB2hlpwjAZaV0gPOlzCBAgAABAoMLyCED+wjAFZ49Ci8f1yUQ+WZmECBAgACB/gTkEAG4ljND4eWzC8D5ZmYQIECAAAEBOK8GdIDzvLJGC8BZXMVgATjfzAwCBAgQICAA59WAAJznlTVaAM7iEoDzucwgQIAAAQIDCsghAxeHAFzhiaPw8nF1gPPNzCBAgAABAjrAeTUgAOd5ZY0WgLO4dIDzucwgQIAAAQI6wB3UgADcAVrZKQJwWakvx+kA55uZQYAAAQIEdIDzakAAzvPKGi0AZ3HpAOdzmUGAAAECBHSAO6gBAbgDtLJTBOCyUjrA+VJmECBAgACBwQXkkIF9BOAKzx6Fl4/rEoh8MzMIECBAgEB/AnKIAFzLmaHw8tkF4HwzMwgQIECAgACcVwM6wHleWaMF4CyuYrAAnG9mBgECBAgQEIDzakAAzvPKGi0AZ3EJwPlcZhAgQIAAgQEF5JCBi0MArvDEUXj5uDrA+WZmECBAgAABHeC8GhCA87yyRgvAWVw6wPlcZhAgQIAAAR3gDmpAAO4ArewUAbis1JfjdIDzzcwgQIAAAQI6wHk1IADneWWNFoCzuHSA87nMIECAAAECOsAd1EDjA/A555wTDz/8cDz66KN9PPfcc0+cddZZ8Z//+Z8xYsSI+P73v1/89/Dhw4sxr7/+eowZM2YpzquuuioOO+yw0swCcGmqvoE6wPlmZhAgQIAAAR3gvBpodAC++OKL4/jjj49ddtmlLwA/9thjseuuu0YKxvvtt1+8+uqr8cMf/jB22223uPbaawu9FJDHjx8fM2fOjBVWWKFPdI011ugLyWWYBeAySouPEYDzzcwgQIAAAQICcF4NNDIAv/nmm0WnNoXd0aNHx7rrrtsXgP/6r/863nnnnfinf/qnPqnp06fHIYccEv/zP/8Tq6yySkyZMiVuueWWePbZZ/M0lxgtAOfzCcD5ZmYQIECAAAEBOK8GGhmAUwf3pptuKrq86dKGWbNm9QXgZ555JoYNGxZbb731YgH4gAMOiHfffbe4JGLChAnFmBtvvDFPUwD+Sl5psgD8lQk9AQECBAgQKAQ04gYuhEYG4EUP9+CDD14sAC9JMX/+/Nh+++2LwPvv//7vxZc333zzGDlyZHz22Wfx8ssvx8YbbxxnnHFG7LHHHlmnlMLL4ioGC8D5ZmYQIECAAAEd4LwaaHUA/u1vfxt/9Vd/FXfddVf88pe/LIJwCsSrrbZabLPNNnHhhRfG6quvHukSiUsuuSQeeuih4lrhgR7z5s2L9M/Cx5w5c2Ls2LExe/bsGDVqVN7OtHS0ANzSjXfYBAgQINB1AY04HeDF7gKROP77v/+7+KDbL37xi7j11lvjL/7iL/qUPvroo1hppZWK64EXPsaNG1d8IO7+++8fUHPSpEkxefLkpb4uAJc/pwXg8lZGEiBAgACBwQQEYAF4sQCcOrN77rln/OY3vym6v+muEMt6nHTSSfHAAw/EjBkzdICXhfUVvj4UAbiTlzdr6p93Ms0cAgQIECBQm4AALAD3BeAPPvggdtxxx+JShRRot9xyy8V0nnvuueLr6Ws77bRT39dSSE53k7j55ptLF7LCK03VN1AAzjczgwABAgQI9CcghwjAfQE4fSgu3SEiBdzNNttsMZm11167uMwhBeB0GcRll11W3BXiyiuvjGnTpsWTTz65VGD2q4fuvukIwN319GwECBCRI9/fAAAgAElEQVQg0F4BAVgALgLwF198UXzA7dNPP+1XJF0Skf4CXLod2imnnBL33XdfzJ07N7bddtuYOnVq7LzzzllnkcLL4ioGC8D5ZmYQIECAAAEd4LwaaPxdIPI4ujtaAM73FIDzzcwgQIAAAQICcF4NCMB5XlmjBeAsLh3gfC4zCBAgQIDAgAJyyMDFIQBXeOIovHxcHeB8MzMIECBAgIAOcF4NCMB5XlmjBeAsLh3gfC4zCBAgQICADnAHNSAAd4BWdooAXFbqy3E6wPlmZhAgQIAAAR3gvBoQgPO8skYLwFlcOsD5XGYQIECAAAEd4A5qQADuAK3sFAG4rJQOcL6UGQQIECBAYHABOWRgHwG4wrNH4eXjugQi38wMAgQIECDQn4AcIgDXcmYovHx2ATjfzAwCBAgQICAA59WADnCeV9ZoATiLqxgsAOebmUGAAAECBATgvBoQgPO8skYLwFlcAnA+lxkECBAgQGBAATlk4OIQgCs8cRRePq4OcL6ZGQQIECBAQAc4rwYE4DyvrNECcBaXDnA+lxkECBAgQEAHuIMaEIA7QCs7RQAuK/XlOB3gfDMzCBAgQICADnBeDQjAeV5ZowXgLC4d4HwuMwgQIECAgA5wBzUgAHeAVnaKAFxWSgc4X8oMAgQIECAwuIAcMrCPAFzh2aPw8nFdApFvZgYBAgQIEOhPQA4RgGs5MxRePrsAnG9mBgECBAgQEIDzakAHOM8ra7QAnMVVDBaA883MIECAAAECAnBeDQjAeV5ZowXgLC4BOJ/LDAIECBAgMKCAHDJwcQjAFZ44Ci8fVwc438wMAgQIECCgA5xXAwJwnlfWaAE4i0sHOJ/LDAIECBAgoAPcQQ0IwB2glZ0iAJeV+nKcDnC+mRkECBAgQEAHOK8GBOA8r6zRAnAWlw5wPpcZBAgQIEBAB7iDGhCAO0ArO0UALiulA5wvZQYBAgQIEBhcQA4Z2EcArvDsUXj5uC6ByDczgwABAgQI9CcghwjAtZwZCi+fXQDONzODAAECBAgIwHk1oAOc55U1WgDO4ioGC8D5ZmYQIECAAAEBOK8GBOA8r6zRAnAWlwCcz2UGAQIECBAYUEAOGbg4BOAKTxyFl4+rA5xvZgYBAgQIENABzqsBATjPK2u0AJzFpQOcz2UGAQIECBDQAe6gBgTgDtDKThGAy0p9OU4HON/MDAIECBAgoAOcVwMCcJ5X1mgBOItLBzifywwCBAgQIKAD3EENCMAdoJWdIgCXldIBzpcygwABAgQIDC4ghwzsIwBXePYovHxcl0Dkm5lBgAABAgT6E5BDWhyAzznnnHj44Yfj0Ucf7VN49tln47jjjounnnoq1lprrTj22GPjxBNP7Pv6F198EZMnT46rr746Pvjgg9hpp51i2rRpsdFGG2WdYQovi6sYLADnm5lBgAABAgQE4LwaaHQH+OKLL47jjz8+dtlll74A/N5778Wmm24a++yzT/G1xx9/PI488sgi4B5yyCGFXgq/6b+vvfbaWH/99ePkk0+OmTNnxvPPPx8rr7xyaWEBuDRV30ABON/MDAIECBAgIADn1UAjA/Cbb74Zhx12WDz22GMxevToWHfddfsC8HnnnReXXnppzJo1K1ZaaaVC67TTTos77rgjXnzxxZg/f36MGDEiLrjggjj88MOLr8+dOzfWW2+9uOaaa2LChAmlhQXg0lQCcD6VGQQIECBAYFABOWRgnkYG4HvuuSduuummSJc/nHXWWUXYXXgJxF577RVrrrlm3HjjjX0q6RKJ3XffPd5+++1i7Pbbbx8vvfRSbLLJJn1j0mUQW221VdEZLvtQeGWlvhynA5xvZgYBAgQIENABzquBRgbgRQkOPvjgxQJwCrF77rlnnH/++X3DXnjhhdh8883jySefjDfeeCP23Xff+Pjjj2P48OF9Y8aPH1/8v3vvvXdA4Xnz5kX6Z+Fjzpw5MXbs2Jg9e3aMGjUqb2daOloAbunGO2wCBAgQ6LqARlzLOsCDBeD0QbaJEycWneGFj3R974YbblhcMpE6wAcccEB8/vnnMWzYsL4xBx54YLz11lvFB+oGekyaNKm4fnjJhwBc/pwWgMtbGUmAAAECBAYTEIAF4L5LILbccstIl0H01wF++umniw+77bfffv12gD/77LO4++67dYArfL8RgCvE9dQECBAg0CoBAVgAXuwa4HTrs+uvv75Ppb9rgF999dWiK7zwka4B3nrrrYsP0JV9KLyyUl+OE4DzzcwgQIAAAQL9CcghAnBfAJ46dWpcfvnl8dprr8WKK65YyJx66qlx5513FneBSF3eddZZJy666KI49NBDi6+7C8TQvbEIwENnbSUCBAgQaLaAACwA9wXgd955p7gP8N57713c3/eJJ56II444ogjFBx10UCF1+umnxxVXXFHc9mzMmDFx0kknFdcGz5gxw32AK36vEIArBvb0BAgQINAaAQFYAF7sL8Gluz2kv/72zDPPxMiRI+OEE06Io48+uk8pfQAu3Rs4/SGMTz75pPhDGunShxSGcx4KL0fr/8YKwPlmZhAgQIAAgf4E5JAWB+A6TwmFl68vAOebmUGAAAECBATgvBpo/H2A8zi6O1oAzvcUgPPNzCBAgAABAgJwXg0IwHleWaMF4CyuYrAAnG9mBgECBAgQEIDzakAAzvPKGi0AZ3EJwPlcZhAgQIAAgQEF5JCBi0MArvDEUXj5uDrA+WZmECBAgAABHeC8GhCA87yyRgvAWVw6wPlcZhAgQIAAAR3gDmpAAO4ArewUAbis1JfjdIDzzcwgQIAAAQI6wHk1IADneWWNFoCzuHSA87nMIECAAAECOsAd1IAA3AFa2SkCcFkpHeB8KTMIECBAgMDgAnLIwD4CcIVnj8LLx3UJRL6ZGQQIECBAoD8BOUQAruXMUHj57AJwvpkZBAgQIEBAAM6rAR3gPK+s0QJwFlcxWADONzODAAECBAgIwHk1IADneWWNFoCzuATgfC4zCBAgQIDAgAJyyMDFIQBXeOIovHxcHeB8MzMIECBAgIAOcF4NCMB5XlmjBeAsLh3gfC4zCBAgQICADnAHNSAAd4BWdooAXFbqy3E6wPlmZhAgQIAAAR3gvBoQgPO8skYLwFlcOsD5XGYQIECAAAEd4A5qQADuAK3sFAG4rJQOcL6UGQQIECBAYHABOWRgHwG4wrNH4eXjugQi38wMAgQIECDQn4AcIgDXcmYovHx2ATjfzAwCBAgQICAA59WADnCeV9ZoATiLqxgsAOebmUGAAAECBATgvBoQgPO8skYLwFlcAnA+lxkECBAgQGBAATlk4OIQgCs8cRRePq4OcL6ZGQQIECBAQAc4rwYE4DyvrNECcBaXDnA+lxkECBAgQEAHuIMaEIA7QCs7RQAuK/XlOB3gfDMzCBAgQICADnBeDQjAeV5ZowXgLC4d4HwuMwgQIECAgA5wBzUgAHeAVnaKAFxWSgc4X8oMAgQIECAwuIAcMrCPAFzh2aPw8nFdApFvZgYBAgQIEOhPQA4RgGs5MxRePrsAnG9mBgECBAgQEIDzakAHOM8ra7QAnMVVDBaA883MIECAAAECAnBeDQjAeV5ZowXgLC4BOJ/LDAIECBAgMKCAHDJwcQjAFZ44Ci8fVwc438wMAgQIECCgA5xXAwJwnlfWaAE4i0sHOJ/LDAIECBAgoAPcQQ0IwB2glZ0iAJeV+nKcDnC+mRkECBAgQEAHOK8GWhmAH3300dh11137lfr6178eM2fOjEMOOSSuu+66xcasv/76kUJt2YcAXFZKAM6XMoMAAQIECAwuIIcM7NPKADx//vx4//33F1OZMWNG7LXXXnHZZZfFYYcdFtttt12MGzcujjnmmL5xK664Yqy99tqlzzeFV5qqb6AOcL6ZGQQIECBAQAc4rwZaGYCXJPrf//3f2GabbWKLLbaIm2++OT7//PNYffXV46abbop99tknT3SR0W0PwMtrmO1kQ2dN/fNOpplDgAABAgRqE2h7DhkMXgCOiIsvvjjOPPPMePHFF2PkyJHFvzfbbLN46aWXYpNNNum4cNteeAJwx6VjIgECBAgQ+MoCbc8hAvAgAp9++mmMHj06jjzyyJg8eXIx8tZbb439998/Dj/88Lj//vtj2LBhxeUR55xzTqyxxhqlC7LthScAly4VAwkQIECAQNcF2p5DBOBBBK666qr4m7/5m3j99ddjrbXWKkambvC5554bU6ZMKYLvK6+8EieeeGIRlB955JEiEPf3mDdvXqR/Fj7mzJkTY8eOjdmzZ8eoUaO6XtjL+xMKwMv7Dnl9BAgQINBkAQF44N1t/SUQO++8c2ywwQbx05/+tE9pwYIF8eGHH8aaa67Z9/8ef/zx2GGHHSL9e/vtt+9XdNKkSX1d5EUHCMC9//biGuDe30NHQIAAgbYJCMACcL8C7777bvz+7/9+3HvvvbHnnnsOel589NFHxQfjbrvttth33311gEu8i+gAl0AyhAABAgQIVCQgAAvA/QrcfffdRZhNly187Wtf6xszceLEeO+99+LBBx/s+38L7x38wgsvFB+QK/Noe+EJwGWqxBgCBAgQIFCNQNtzyGCqrb4E4qyzzorp06fHyy+/vJhR6gjvvffecfbZZ8eECROKrx911FHFJRA33HBD6Spte+EJwKVLxUACBAgQINB1gbbnEAF4AIF054dnnnkm/vVf/3WpEbfffnvxQbh0S7R0LXDqCqe7QKy66qqlC7TthScAly4VAwkQIECAQNcF2p5DBOCul1S5J2x74QnA5erEKAIECBAgUIVA23OIAFxFVZV4zrYXngBcokgMIUCAAAECFQm0PYcIwBUV1rKetu2FJwAvq0J8nQABAgQIVCfQ9hwiAFdXW4M+c9sLTwCuqfAsS4AAAQIEIqLtOUQAruk0aHvhCcA1FZ5lCRAgQICAADxoDbT6NmhVnx0C8D9WTTxkz+8vwQ0ZtYUIECBAoEsCbc8hOsBdKqTcp2l74ekA51aM8QQIECBAoHsCbc8hAnD3ainrmdpeeAJwVrkYTIAAAQIEuirQ9hwiAHe1nMo/WdsLTwAuXytGEiBAgACBbgu0PYcIwN2uqJLP1/bCE4BLFophBAgQIECgAoG25xABuIKiKvOUbS88AbhMlRhDgAABAgSqEWh7DhGAq6mrZT5r2wtPAF5miRhAgAABAgQqE2h7DhGAKyutwZ+47YUnANdUeJYlQIAAAQLuAzxoDbgPcIWniADsPsAVlpenJkCAAAECgwq0PYfoANd0grS98HSAayo8yxIgQIAAAR1gHeC6zgIBWAe4rtqzLgECBAgQaHsO0QGu6Rxoe+HpANdUeJYlQIAAAQI6wDrAdZ0FArAOcF21Z10CBAgQIND2HKIDXNM50PbC0wGuqfAsS4AAAQIEdIB1gOs6CwRgHeC6as+6BAgQIECg7TlEB7imc6DthacDXFPhWZYAAQIECOgA6wDXdRYIwDrAddWedQkQIECAQNtziA5wTedA2wtPB7imwrMsAQIECBDQAdYBrussEIB1gOuqPesSIECAAIG25xAd4JrOgbYXng5wTYVnWQIECBAgoAOsA1zXWSAA6wDXVXvWJUCAAAECbc8hOsA1nQNtLzwd4JoKz7IECBAgQEAHWAe4rrNAANYBrqv2rEuAAAECBNqeQ3SAazoH2l54OsA1FZ5lCRAgQICADrAOcF1ngQCsA1xX7VmXAAECBAi0PYfoANd0DrS98HSAayo8yxIgQIAAAR1gHeC6zgIBWAe4rtqzLgECBAgQaHsO0QGu6Rxoe+HpANdUeJYlQIAAAQI6wDrAdZ0FArAOcF21Z10CBAgQIND2HKIDXNM50PbC0wGuqfAsS4AAAQIEdIB1gPsTeP3112PMmDFLfemqq66Kww47LJ599tk47rjj4qmnnoq11lorjj322DjxxBOzTigBWAc4q2AMJkCAAAECXRRoew7RAe5H4J577onx48fHzJkzY4UVVugbscYaa8THH38cm266aeyzzz5x/PHHx+OPPx5HHnlkTJs2LQ455JDSpdn2wtMBLl0qBhIgQIAAga4LtD2HCMD9CEyZMiVuueWWotO75OO8886LSy+9NGbNmhUrrbRS8eXTTjst7rjjjnjxxRdLF2jbC08ALl0qBhIgQIAAga4LtD2HCMD9CEyYMCGGDRsWN95441Jf3WuvvWLNNddc7GsPP/xw7L777vH222/HOuusU6pI2154AnCpMjGIAAECBAhUItD2HCIA9yOw+eabx8iRI+Ozzz6Ll19+OTbeeOM444wzYo899oitttoq9txzzzj//PP7Zr7wwguR5jz55JPxzW9+s1Shtr3wBOBSZWIQAQIECBCoRKDtOUQAXkJg/vz5sdpqq8U222wTF154Yay++uoxffr0uOSSS+Khhx6KH/3oRzFx4sQ466yz+mama4U33HDDeOyxx2KnnXbq13TevHmR/ln4mDNnTowdOzZmz54do0aNqqS4l+cnFYCX593x2ggQIECg6QIC8MA7vMKCBQsWNL0A+ju+jz76qLi+d5VVVun78rhx44oPxKWCSZdB9NcBfvrpp4vg3N9j0qRJMXny5KW+JAD3foXNmvrnvX8QjoAAAQIEWiUgAAvApQr+pJNOigceeCBGjx5d3Prs+uuv75tX5hpgHeDFmXWAS5WdQQQIECBAoBIBAVgAXkzgueeeix133LEIu4tezrDrrrvGuuuuG9/4xjfi8ssvj9deey1WXHHFYu6pp54ad955p7tAZJyiAnAGlqEECBAgQKDLAgKwALyYwBdffFEE4HQZxGWXXRYjRoyIK6+8srjPb/qQWwrB6T7Ae++9d5x88snxxBNPxBFHHFGE4oMOOqh0eba98ATg0qViIAECBAgQ6LpA23PIYKCtvQb43XffjVNOOSXuu+++mDt3bmy77bYxderU2HnnnQuvFITTX3975plnirtFnHDCCXH00UdnFWfbC08AzioXgwkQIECAQFcF2p5DBOCullP5J2t74QnA5WvFSAIECBAg0G2BtucQAbjbFVXy+dpeeAJwyUIxjAABAgQIVCDQ9hwiAFdQVGWesu2FJwCXqRJjCBAgQIBANQJtzyECcDV1tcxnbXvhCcDLLBEDCBAgQIBAZQJtzyECcGWlNfgTt73wBOCaCs+yBAgQIEAgovjDXulvG7T1D3IJwDWdBm0vPAG4psKzLAECBAgQEIAHrYHW3gZtKM4MAfgfh4J5SNbwp5CHhNkiBAgQINBFgbbnEB3gLhZTzlO1vfB0gHOqxVgCBAgQINBdgbbnEAG4u/VU+tnaXngCcOlSMZAAAQIECHRdoO05RADuekmVe8K2F54AXK5OjCJAgAABAlUItD2HCMBVVFWJ52x74QnAJYrEEAIECBAgUJFA23OIAFxRYS3radteeALwsirE1wkQIECAQHUCbc8hAnB1tTXoM7e98ATgmgrPsgQIECBAwG3QBq0Bt0Gr8BQRgN0GrcLy8tQECBAgQEAjrsMaEIA7hCszTQAWgMvUiTEECBAgQKAKgbbnEJdAVFFVJZ6z7YXnEogSRWIIAQIECBCoSKDtOUQArqiwlvW0bS88AXhZFeLrBAgQIECgOoG25xABuLraGvSZm1R4TQqzQ1UO/nzyUElbhwABAgT6E2hSDun2DrsGuNuiizxfkwpPAM4vFAE438wMAgQIEOieQJNySPdU/u+ZBOBuiwrAFYr21lMLwL21X14tAQIEmiYgAA+8owJwhdXepMLTAc4vFAE438wMAgQIEOieQJNySPdUdIC7bbnU8zWp8ATg/HIRgPPNzCBAgACB7gk0KYd0T0UA7ralAFy5aG8tIAD31n55tQQIEGiagAA88I66BKLCam9S4ekA5xeKAJxvZgYBAgQIdE+gSTmkeyo6wN221AGuXLS3FhCAe2u/vFoCBAg0TUAA1gGupaabVHg6wPklJADnm5lBgAABAt0TaFIO6Z6KDnC3LXWAKxftrQUE4N7aL6+WAAECTRMQgHWAa6npJhWeDnB+CQnA+WZmECBAgED3BJqUQ7qnogPcbUsd4MpFe2sBAbi39surJUCAQNMEBGAd4FpqukmFpwOcX0ICcL6ZGQQIECDQPYEm5ZDuqegAd9tSB7hy0d5aQADurf3yagkQINA0AQFYB7iWmm5S4ekA55eQAJxvZgYBAgQIdE+gSTmkeyo6wN221AGuXLS3FhCAe2u/vFoCBAg0TUAA1gGupaabVHg6wPklJADnm5lBgAABAt0TaFIO6Z6KDnC3LXWAKxftrQUE4N7aL6+WAAECTRMQgHWAlxJ4//3347TTTot777035s2bF1tttVVMnTo1dtppp2LsIYccEtddd91i89Zff/1IxVT20aTC0wEuu+tfjhOA883MIECAAIHuCTQph3RPpeUd4O985zvxzjvvxE9+8pNYe+2149JLL42rr746nn766dh0001ju+22i3HjxsUxxxzTZ77iiisWY8s+mlR4AnDZXReA86XMIECAAIEqBJqUQ7rts8KCBQsWdPtJl/fne/XVV2PjjTeOX/3qV7HjjjsWLzcxbLLJJvGDH/wgzjzzzFh99dXjpptuin322afjw2lS4QnA+WWgA5xvZgYBAgQIdE+gSTmkeyot7gDPnTs3fv3rX8duu+0Wq6yySp/pRhttVHR9jz766Nhss83ipZdeKkJxp48mFZ4AnF8FAnC+mRkECBAg0D2BJuWQ7qm0OAD3h3jrrbfG+PHj42c/+1l8+umnsf/++8fhhx8e999/fwwbNiz22muvOOecc2KNNdYYcA/StcTpn4WPOXPmxNixY2P27NkxatSobu/dkD6fAJzPLQDnm5lBgAABAt0TEIAHtmzlJRBLcqRLIVLnN3WE77rrruISiHPPPTemTJlSBN9XXnklTjzxxBg9enQ88sgjRSDu7zFp0qSYPHnyUl8SgLt3MvfSMwnAvbRbXisBAgSaJyAAC8ADCtx9990xceLE+Pa3v13cEWL48OHF9cAffvhhrLnmmn3zHn/88dhhhx0i/Xv77bfv9/l0gJv35vFVjkgA/ip65hIgQIDAVxUQgAXgfgXSHSCOO+64+N73vhfTp09f7HrgJSd89NFHxQfjbrvttth3331L1WSTCs8lEKW2fLFBAnC+mRkECBAg0D2BJuWQ7qn83zO19hKIyy67LI488sg49thj48c//vFilzWkjvB7770XDz74YJ/3o48+Grvuumu88MILxQfkyjyaVHgCcJkdX3yMAJxvZgYBAgQIdE+gSTmkeyotDsAvv/xybLHFFvHd7343pk2btphpugTisccei7333jvOPvvsmDBhQqTxRx11VHEJxA033FB6D5pUeAJw6W3vGygA55uZQYAAAQLdE2hSDumeSosDcPpw2+mnn96v5UEHHVT8Bbjbb7+9+CDciy++WFwLnLrC6S4Qq666auk9aFLhCcClt10AzqcygwABAgQqEGhSDuk2T2svgeg2ZH/P16TCE4DzK0YHON/MDAIECBDonkCTckj3VFrcAe424kDP16TCE4Dzq0YAzjczgwABAgS6J9CkHNI9FQG425ZLPV+TCk8Azi8XATjfzAwCBAgQ6J5Ak3JI91QE4G5bCsCVi/bWAgJwb+2XV0uAAIGmCQjAA++oa4ArrPYmFZ4OcH6hCMD5ZmYQIECAQPcEmpRDuqeiA9xtSx3gykV7awEBuLf2y6slQIBA0wQEYB3gWmq6SYWnA5xfQgJwvpkZBAgQINA9gSblkO6p6AB321IHuHLR3lpAAO6t/fJqCRAg0DQBAVgHuJaablLh6QDnl5AAnG9mBgECBAh0T6BJOaR7KjrA3bbUAa5ctLcWEIB7a7+8WgIECDRNQADWAa6lpptUeDrA+SUkAOebmUGAAAEC3RNoUg7pnooOcLctdYArF+2tBQTg3tovr5YAAQJNExCAdYBrqekmFZ4OcH4JCcD5ZmYQIECAQPcEmpRDuqeiA9xtSx3gykV7awEBuLf2y6slQIBA0wQEYB3gWmq6SYWnA5xfQgJwvpkZBAgQINA9gSblkO6p6AB321IHuHLR3lpAAO6t/fJqCRAg0DQBAVgHuJaablLh6QDnl5AAnG9mBgECBAh0T6BJOaR7KjrA3bbUAa5ctLcWEIB7a7+8WgIECDRNQADWAa6lpptUeDrA+SUkAOebmUGAAAEC3RNoUg7pnooOcLctdYArF+2tBQTg3tovr5YAAQJNExCAdYBrqekmFZ4OcH4JCcD5ZmYQIECAQPcEmpRDuqeiA9xtSx3gykV7awEBuLf2y6slQIBA0wQEYB3gWmq6SYWnA5xfQgJwvpkZBAgQINA9gSblkO6p6AB321IHuHLR3lpAAO6t/fJqCRAg0DQBAVgHuJaablLh6QDnl5AAnG9mBgECBAh0T6BJOaR7KjrA3bbUAa5ctLcWEIB7a7+8WgIECDRNQADWAa6lpptUeDrA+SUkAOebmUGAAAEC3RNoUg7pnooOcLctdYArF+2tBQTg3tovr5YAAQJNExCAdYBrqekmFZ4OcH4JCcD5ZmYQIECAQPcEmpRDuqeiA9xtSx3gykV7awEBuLf2y6slQIBA0wQEYB3gWmq6SYWnA5xfQgJwvpkZBAgQINA9gSblkO6p6AB321IHuHLR3lpAAO6t/fJqCRAg0DQBAVgHuJaablLh6QDnl5AAnG9mBgECBAh0T6BJOaR7KjrA3bbUAa5ctLcWEIB7a7+8WgIECDRNQADWAc6u6S+++CImT54cV199dXzwwQex0047xbRp02KjjTYq/VxNKjwd4NLb3vFAgbljOhMJECBAoB+BJuWQbm/wCgsWLFjQ7SdtwvOl8JsC77XXXhvrr79+nHzyyTFz5sx4/vnnY+WVVy51iE0qPAG41JZ/pUEC8FfiM5kAAQIElhBoUg7p9uYKwP2Izp8/P0aMGOcFlEEAAA6MSURBVBEXXHBBHH744cWIuXPnxnrrrRfXXHNNTJgwodQ+NKnwBOBSW/6VBgnAX4nPZAIECBAQgEvXgADcD9UTTzwR22+/fbz00kuxySab9I1Il0FstdVWRWe4zEMALqNkzEIBAVgtECBAgEA3BZqUQ7rpkp5LAO5H9I477oh99903Pv744xg+fHjfiPHjxxf/79577y21D00qPB3gUlv+lQYJwF+Jz2QCBAgQ0AEuXQMCcD9U06dPjwMOOCA+//zzGDZsWN+IAw88MN566614+OGH+wWeN29epH8WPmbPnh077rhjpI7yyJEjS2/K8jjw21N+vjy+rEa9psdP261Rx+NgCBAgQKBegTlz5sTYsWPjN7/5TYwZM6beF7OcrS4A97Mht99+e+y33379doA/++yzuPvuu/vdxkmTJhV3jvAgQIAAAQIECCwvAqkR961vfWt5eTnLxesQgPvZhoXXAL/66qux4YYb9o1I1wBvvfXWcemll5bqAH/66aeRusBf//rXY6WVVqpkwxf+dNeELnMuUJuPPVk5/v/rbKj93v7tUu553/bad96397zvpPZ/+9vfxrvvvhtbbrllrLrqqp2cbo2dIwD3s7Wpy7vOOuvERRddFIceemgxopO7QAxF1TTpOuNcrzYfe7Jy/P8Vo0ePLn7IHDVqVG759PR4e2/v21j33ve873fzjVsAHkDz9NNPjyuuuKK47Vm6buakk06KWbNmxYwZM0rfB7ibGzXQc7X5G2Gbj903gnZ/I1D7ArAA3L4ffL3vdzdVCcADeKYPwJ122mnFH8L45JNPYpdddikufVjeLiJv8zfCNh+7N0IBuK3d77bXvve99v7w0/ba7278dRu0bnsO+fOlu06kSzWOP/74+N3f/d0hX7/OBdt87Mnd8av9Np73ba995317z/u2136384YOcLdFPR8BAgQIECBAgMByLSAAL9fb48URIECAAAECBAh0W0AA7rao5yNAgAABAgQIEFiuBQTg5Xp7vDgCBAgQIECAAIFuCwjA3Rb1fAQIECBAgAABAsu1gAC8XG/PwC/uiy++KP7s8tVXXx0ffPBBpL9SN23atNhoo4169IgGf9mvv/56v7egu+qqq+Kwww6LZ599No477rh46qmnYq211opjjz02TjzxxJ63OOecc+Lhhx+ORx99tO9YlnWsTaqN/o7/kEMOieuuu26xvV1//fWLPwySHr18/O+//35x+8V77723uMvHVlttFVOnTi3O7/Ro8t4v69ibvO9pb99555044YQT4oEHHihuvfknf/InceGFF8Yf//EfN37vyxx/0/d/4Rvayy+/HNtuu2385Cc/iYMPPrgVe1/XN2oBuC75r7huCr8p8Kb7FKdv/ieffHLMnDkznn/++eXqD3V8xcPsm37PPffE+PHji2NcYYUV+v7/GmusER9//HFsuummsc8++xS3g3v88cfjyCOPLHzSm2avPi6++OLieNI9qBcG4Pfee2+Zx9qU2ujv+NNebrfddjFu3Lg45phj+rZ2xRVXjLXXXrv4714+/u985ztFEErf/NLxpHuPpx9yn3766eK/l1XnTT32dNxN3vdUt9/+9rdj2LBh8fd///ex2mqrxRlnnBG//vWv49VXXy0CcZP3flnH/7Wvfa3x+58M/vd//zd23HHHopGTvrenANym9/yh/l4tAA+1eBfWmz9/fowYMSIuuOCCOPzww4tnXF7/VHMXDrd4iilTpsQtt9xSdMCWfJx33nlFUEh/qW+llVYqvpy6aHfccUe8+OKL3XoJQ/Y8b775ZtHVfuyxx4o/9bvuuuv2BeBlHWsTamOw409/oGb11VePm266qfiBZ8lHLx9/Cjobb7xx/OpXvyq+CabHggULYpNNNokf/OAHMXz48EHrvMnHfuaZZzZ239M+p5Bz9NFHx9/+7d/G5ptvXuz9f/zHf8Q3vvGNeOKJJ4rfAg32HtfLe1/m+FNHtKnn/aLvYen7Vmrg/OIXv+gLwG14zx+yb65LLCQA1yX/FdZNb4jbb799vPTSS8U3x4WP9GvS9CvT1Pls2mPChAlFd+TGG29c6tD22muvWHPNNRf7WvqGsfvuu8fbb78d66yzTk9xpG53Cnjp1/9nnXVWEewXdoCXdaxpbK/XxmDHn36g2WyzzZaq/YUb3MvnRvohNnX8dtttt1hllVX6ajZd1pQ63um3H4PVeS/v/bKOPYXDpu57f29O6X0rXcKVzvtU89///vcbu/dljj/92eem7/8vf/nL+Mu//MuiyfMHf/AHfQG4De/5dX2DFoDrkv8K66bO5r777lv86j91hRY+0iUC6f+l6web9khdkZEjR8Znn30W6Rqp1ClLvyLcY489itC/5557xvnnn9932C+88ELRSXnyySfjm9/8Zs9ypF+BLRqAl3Wsb7zxRqNqY8njv/XWW2P//fcvfvNx//33Fz8UpW8Q6YeFdDlM086NdLzpvP7Zz34Wp59++qB13rS9X/TYP/3009bs+w9/+MNIn21IPwSlfU+XxbTpvO/v+Jt+3qcfALfeeutI3d6JEycWl/ktvASiTXs/1N+oBeChFu/CetOnT48DDjgg0q+DUwBY+DjwwAPjrbfeKn5d1qRH+vVeuiZum222KT4Ukn4VlgwuueSSeOihh+JHP/pR8aaRuqULH6lbtuGGGxaXESz8AFEvmiwZAFM3cLBjTWG5SbWx5PGnX4Wfe+65xSUxKfi+8sorRacsXSryyCOPFL8FaMrxp0shUuc3dYTvuuuu4gOubdn7JY+9TfuefnhP1/xedtllRT3/y7/8S/FDUFv2vr/jv/vuuxt93qdLnNLlTjfffHPxbWrRANym836ov0cLwEMt3oX1br/99thvv/367QCnDml6s2ja46OPPiqu7130V8MpHKQ3ivTp/xSG+usApw8PpeDcq48lA+CWW2456LGm4N+k2ljy+NM3iQ8//LD4dfDCR7pmbocddiiunUu10ITjT+dwCjzpg1HpNzrpNz1t2fv+jr0t+77o+1S6m0na83RJU/pN1mDvcU0775PDosf/D//wD40976+//vo49dRTY8aMGfF7v/d7SwXgtpz3dXyPFoDrUP+Kay68zjF9aCZ1ORc+Uqcz/RolfViiDY+TTjqpuGVQ6v6lW5+lN5KFj16+BnjRvVsyAKZvgoMd68LrQJtSG0sef391nX44Sr8VuO2224paSIGhl48/3QEi3dLve9/7XvGbjoU/9LVh7wc69qbve7rzx89//vOi05vuaLLwka79TbfDS/+vyef9so7/wQcfXKoEmnLe77rrrkWXf9HmTjq29N/p+/sf/uEfNnrv68wqAnCd+h2unbq86YNdF110URx66KHFszT5LhDPPfdc8an4FHYXvZwhvXGkOySkT0pffvnl8dprr/V980g/Ud955509eReIwQJwuifsYMfatNpYMgCnrmj6xPyi3xDTB4VSLaRfnW6wwQY9fW6kX3unW/il+1j/+Mc/XuwSp6bv/WDH3vR9Tx98Sr+pSiH4T//0T4u3gHRLrHTrs7333rt4n2vyeb+s408fCmzqeZ/uepMueVn0kT7jkn6jmT7vkD4Q3eS97zAGdWWaANwVxqF/kvSBmCuuuCKuueaa4g9EpG5o6v6lX6OsvPLKQ/+CKlwx/SosBeD0U3H6JpluAXfllVcWd7tIvxpM3xwWfqNI90NOHfIjjjiieNM46KCDKnxl1T/1kgEwdUqWdaxNqo0ljz9dDpACwdlnnx3pziDpA5FHHXVUcQnEDTfcUGxIrx5/OpYtttgivvvd7y51J5d0CUT64aape7+sY0/X8jd131PNpks80gd503t4+gBc+lV4utY9/aCXwuGqq67a2L0vc/ypCdLk/V/yO8mi1wC37T2/+u+qX64gAA+ldhfXSh+AS/cMTJ8UTT89pj+WkC59SGG4iY933303TjnllLjvvvuKbne6L2TqiO28887F4aYgnLpmzzzzTHG3iPQXldKtk3r90d8lAMs61ibVRn/Hn66BT+Eg3R4qXQucuoPpLhApJKRHrx5/+mBfCu/9PdIPcumv3zV178sce1P3feF+p2vb02+u0gce03tcem9Lv+VbeF/gpu592eNv+v4vet4vGoDLfH/r1fe8ur8/C8B174D1CRAgQIAAAQIEhlRAAB5SbosRIECAAAECBAjULSAA170D1idAgAABAgQIEBhSAQF4SLktRoAAAQIECBAgULeAAFz3DlifAAECBAgQIEBgSAUE4CHlthgBAgQIECBAgEDdAgJw3TtgfQIECBAgQIAAgSEVEICHlNtiBAgQIECAAAECdQsIwHXvgPUJECBAgAABAgSGVEAAHlJuixEgQIAAAQIECNQtIADXvQPWJ0CAAAECBAgQGFIBAXhIuS1GgAABAgQIECBQt4AAXPcOWJ8AAQIECBAgQGBIBQTgIeW2GAECBAgQIECAQN0CAnDdO2B9AgQIECBAgACBIRUQgIeU22IECBAgQIAAAQJ1CwjAde+A9QkQIECAAAECBIZUQAAeUm6LESBAgAABAgQI1C0gANe9A9YnQIAAAQIECBAYUgEBeEi5LUaAAAECBAgQIFC3gABc9w5YnwABAgQIECBAYEgFBOAh5bYYAQIECBAgQIBA3QICcN07YH0CBAgQIECAAIEhFRCAh5TbYgQIECBAgAABAnULCMB174D1CRAgQIAAAQIEhlRAAB5SbosRIECAAAECBAjULSAA170D1idAgAABAgQIEBhSAQF4SLktRoAAAQIECBAgULeAAFz3DlifAAECBAgQIEBgSAUE4CHlthgBAgQIECBAgEDdAgJw3TtgfQIECBAgQIAAgSEVEICHlNtiBAgQIECAAAECdQsIwHXvgPUJECBAgAABAgSGVEAAHlJuixEgQIAAAQIECNQtIADXvQPWJ0CAAAECBAgQGFIBAXhIuS1GgAABAgQIECBQt8D/B6DnU86sWDP/AAAAAElFTkSuQmCC\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "plt.figure()\n",
    "plt.hist(list(reg_size_dic.values()), bins=np.arange(0,400,10))\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2db97b4d",
   "metadata": {},
   "source": [
    "### 4.1.3 check readout usage for each gene"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "id": "8d0a6c3d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NRNX1-seg-1 -> ['NDB_376', 'NDB_533', 'NDB_936']\n",
      "NRNX1-seg-2 -> ['NDB_400', 'NDB_464', 'NDB_804']\n",
      "NRNX1-seg-3 -> ['NDB_511', 'NDB_875', 'NDB_858']\n",
      "NRNX1-seg-4 -> ['NDB_508', 'NDB_470', 'NDB_438']\n",
      "NRNX1-seg-5 -> ['NDB_883', 'NDB_461', 'NDB_789']\n",
      "NRNX1-seg-6 -> ['NDB_427', 'NDB_506', 'NDB_1047']\n",
      "NRNX1-seg-7 -> ['NDB_397', 'NDB_989', 'NDB_618']\n",
      "NRNX1-seg-9 -> ['NDB_514', 'NDB_665', 'NDB_819']\n",
      "NRNX1-seg-10 -> ['NDB_951', 'NDB_401', 'NDB_594']\n",
      "NRNX1-seg-13 -> ['NDB_735', 'NDB_824', 'NDB_903']\n",
      "NRNX1-seg-14 -> ['NDB_784', 'NDB_801', 'NDB_652']\n",
      "NRNX1-seg-15 -> ['NDB_643', 'NDB_825', 'NDB_844']\n",
      "NRNX1-seg-16 -> ['NDB_568', 'NDB_375', 'NDB_409']\n",
      "NRNX1-seg-17 -> ['NDB_868', 'NDB_417', 'NDB_754']\n",
      "NRNX1-seg-18 -> ['NDB_634', 'NDB_954', 'NDB_532']\n",
      "NRNX1-seg-19 -> ['NDB_388', 'NDB_780', 'NDB_730']\n",
      "NRNX1-seg-20 -> ['NDB_835', 'NDB_630', 'NDB_853']\n",
      "NRNX1-seg-21 -> ['NDB_838', 'NDB_396', 'NDB_445']\n",
      "NRNX1-seg-22 -> ['NDB_379', 'NDB_977', 'NDB_372']\n",
      "NRNX1-seg-23 -> ['NDB_544', 'NDB_392', 'NDB_513']\n",
      "NRNX1-seg-24 -> ['NDB_759', 'NDB_713', 'NDB_807']\n",
      "NRNX1-seg-25 -> ['NDB_456', 'NDB_965', 'NDB_768']\n",
      "NRNX1-seg-26 -> ['NDB_915', 'NDB_746', 'NDB_516']\n",
      "NRNX1-seg-27 -> ['NDB_957', 'NDB_455', 'NDB_657']\n",
      "NRNX1-seg-28 -> ['NDB_405', 'NDB_536', 'NDB_771']\n",
      "NRNX1-seg-30 -> ['NDB_390', 'NDB_887', 'NDB_615']\n",
      "NRNX1-seg-31 -> ['NDB_846', 'NDB_896', 'NDB_453']\n",
      "NRNX1-seg-32 -> ['NDB_687', 'NDB_356', 'NDB_828']\n",
      "NRNX1-seg-33 -> ['NDB_862', 'NDB_491', 'NDB_694']\n",
      "NRNX1-seg-34 -> ['NDB_418', 'NDB_1028', 'NDB_622']\n",
      "NRNX1-seg-35 -> ['NDB_826', 'NDB_578', 'NDB_710']\n",
      "NRNX1-seg-36 -> ['NDB_760', 'NDB_815', 'NDB_554']\n",
      "NRNX1-seg-37 -> ['NDB_466', 'NDB_974', 'NDB_812']\n",
      "NRNX1-seg-38 -> ['NDB_529', 'NDB_443', 'NDB_698']\n",
      "NRNX1-seg-39 -> ['NDB_715', 'NDB_758', 'NDB_398']\n",
      "NRNX1-seg-40 -> ['NDB_1060', 'NDB_866', 'NDB_752']\n",
      "NRNX1-seg-41 -> ['NDB_433', 'NDB_539', 'NDB_899']\n",
      "NRNX1-seg-42 -> ['NDB_994', 'NDB_881', 'NDB_935']\n",
      "NRNX1-seg-43 -> ['NDB_551', 'NDB_705', 'NDB_1053']\n",
      "NRNX1-seg-44 -> ['NDB_836', 'NDB_519', 'NDB_450']\n",
      "NRNX1-seg-45 -> ['NDB_755', 'NDB_725', 'NDB_743']\n",
      "NRNX1-seg-46 -> ['NDB_683', 'NDB_431', 'NDB_728']\n",
      "NRNX1-seg-47 -> ['NDB_458', 'NDB_773', 'NDB_860']\n",
      "NRNX1-seg-48 -> ['NDB_680', 'NDB_737', 'NDB_878']\n",
      "NRNX1-seg-49 -> ['NDB_332', 'NDB_653', 'NDB_467']\n",
      "NRNX1-seg-50 -> ['NDB_335', 'NDB_347', 'NDB_446']\n",
      "NRNX1-seg-51 -> ['NDB_473', 'NDB_641', 'NDB_452']\n",
      "NRNX1-seg-52 -> ['NDB_647', 'NDB_734', 'NDB_587']\n",
      "NRNX1-seg-53 -> ['NDB_628', 'NDB_594', 'NDB_958']\n",
      "NRNX1-seg-54 -> ['NDB_526', 'NDB_903', 'NDB_712']\n",
      "NRNX1-seg-55 -> ['NDB_533', 'NDB_817', 'NDB_971']\n",
      "NRNX1-seg-56 -> ['NDB_464', 'NDB_778', 'NDB_917']\n",
      "NRNX1-seg-57 -> ['NDB_875', 'NDB_487', 'NDB_716']\n",
      "NRNX1-seg-58 -> ['NDB_470', 'NDB_1027', 'NDB_863']\n",
      "NRNX1-seg-59 -> ['NDB_461', 'NDB_631', 'NDB_905']\n",
      "NRNX1-seg-60 -> ['NDB_506', 'NDB_805', 'NDB_482']\n",
      "NRNX1-seg-62 -> ['NDB_989', 'NDB_721', 'NDB_941']\n",
      "NRNX1-seg-63 -> ['NDB_665', 'NDB_637', 'NDB_788']\n",
      "NRNX1-seg-64 -> ['NDB_779', 'NDB_751', 'NDB_611']\n",
      "NRNX1-seg-65 -> ['NDB_386', 'NDB_517', 'NDB_614']\n",
      "NRNX1-seg-66 -> ['NDB_795', 'NDB_865', 'NDB_690']\n",
      "NRNX1-seg-67 -> ['NDB_738', 'NDB_790', 'NDB_852']\n",
      "NRNX1-seg-68 -> ['NDB_387', 'NDB_901', 'NDB_432']\n",
      "NRNX1-seg-69 -> ['NDB_465', 'NDB_403', 'NDB_1020']\n",
      "NRNX1-seg-70 -> ['NDB_1035', 'NDB_1033', 'NDB_741']\n",
      "NRNX1-seg-71 -> ['NDB_921', 'NDB_619', 'NDB_507']\n",
      "NRNX1-seg-72 -> ['NDB_864', 'NDB_946', 'NDB_816']\n",
      "NRNX1-seg-73 -> ['NDB_498', 'NDB_1066', 'NDB_660']\n",
      "NRNX1-seg-74 -> ['NDB_459', 'NDB_879', 'NDB_341']\n",
      "NRNX1-seg-75 -> ['NDB_933', 'NDB_840', 'NDB_395']\n",
      "NRNX1-seg-76 -> ['NDB_759', 'NDB_699', 'NDB_847']\n",
      "NRNX1-seg-77 -> ['NDB_456', 'NDB_762', 'NDB_661']\n",
      "NRNX1-seg-79 -> ['NDB_915', 'NDB_786', 'NDB_448']\n",
      "NRNX1-seg-80 -> ['NDB_957', 'NDB_681', 'NDB_391']\n",
      "NRNX1-seg-81 -> ['NDB_405', 'NDB_444', 'NDB_1075']\n",
      "NRNX1-seg-82 -> ['NDB_390', 'NDB_747', 'NDB_832']\n",
      "NRNX1-seg-83 -> ['NDB_846', 'NDB_1044', 'NDB_385']\n",
      "NRNX1-seg-84 -> ['NDB_687', 'NDB_711', 'NDB_889']\n",
      "NRNX1-seg-85 -> ['NDB_694', 'NDB_705', 'NDB_372']\n",
      "NRNX1-seg-86 -> ['NDB_622', 'NDB_519', 'NDB_513']\n",
      "NRNX1-seg-87 -> ['NDB_755', 'NDB_652', 'NDB_718']\n",
      "NRNX1-seg-88 -> ['NDB_683', 'NDB_844', 'NDB_961']\n",
      "NRNX1-seg-89 -> ['NDB_458', 'NDB_409', 'NDB_571']\n",
      "NRNX1-seg-90 -> ['NDB_680', 'NDB_754', 'NDB_856']\n",
      "NRNX1-seg-91 -> ['NDB_332', 'NDB_532', 'NDB_616']\n",
      "NRNX1-seg-92 -> ['NDB_335', 'NDB_730', 'NDB_880']\n",
      "NRNX1-seg-93 -> ['NDB_473', 'NDB_853', 'NDB_505']\n",
      "NRNX1-seg-94 -> ['NDB_647', 'NDB_445', 'NDB_706']\n",
      "NRNX1-seg-95 -> ['NDB_1014', 'NDB_421', 'NDB_401']\n",
      "NRNX1-seg-96 -> ['NDB_627', 'NDB_472', 'NDB_824']\n",
      "NRNX1-seg-97 -> ['NDB_376', 'NDB_801', 'NDB_817']\n",
      "NRNX1-seg-98 -> ['NDB_400', 'NDB_825', 'NDB_778']\n",
      "NRNX1-seg-99 -> ['NDB_511', 'NDB_375', 'NDB_487']\n",
      "NRNX1-seg-100 -> ['NDB_508', 'NDB_417', 'NDB_1027']\n",
      "NRNX1-seg-101 -> ['NDB_883', 'NDB_954', 'NDB_631']\n",
      "NRNX1-seg-102 -> ['NDB_427', 'NDB_780', 'NDB_805']\n",
      "NRNX1-seg-103 -> ['NDB_397', 'NDB_630', 'NDB_721']\n",
      "NRNX1-seg-104 -> ['NDB_514', 'NDB_396', 'NDB_637']\n",
      "NRNX1-seg-105 -> ['NDB_628', 'NDB_557', 'NDB_977']\n",
      "NRNX1-seg-106 -> ['NDB_526', 'NDB_929', 'NDB_392']\n",
      "NRNX1-seg-107 -> ['NDB_784', 'NDB_807', 'NDB_690']\n",
      "NRNX1-seg-108 -> ['NDB_643', 'NDB_768', 'NDB_852']\n",
      "NRNX1-seg-109 -> ['NDB_568', 'NDB_516', 'NDB_432']\n",
      "NRNX1-seg-110 -> ['NDB_868', 'NDB_657', 'NDB_1020']\n",
      "NRNX1-seg-111 -> ['NDB_634', 'NDB_771', 'NDB_741']\n",
      "NRNX1-seg-112 -> ['NDB_388', 'NDB_615', 'NDB_507']\n",
      "NRNX1-seg-113 -> ['NDB_835', 'NDB_453', 'NDB_816']\n",
      "NRNX1-seg-114 -> ['NDB_838', 'NDB_828', 'NDB_660']\n",
      "NRNX1-seg-115 -> ['NDB_551', 'NDB_879', 'NDB_958']\n",
      "NRNX1-seg-117 -> ['NDB_836', 'NDB_840', 'NDB_712']\n",
      "NRNX1-seg-118 -> ['NDB_865', 'NDB_710', 'NDB_971']\n",
      "NRNX1-seg-119 -> ['NDB_790', 'NDB_554', 'NDB_917']\n",
      "NRNX1-seg-120 -> ['NDB_901', 'NDB_812', 'NDB_716']\n",
      "NRNX1-seg-121 -> ['NDB_403', 'NDB_698', 'NDB_863']\n",
      "NRNX1-seg-122 -> ['NDB_1033', 'NDB_398', 'NDB_905']\n",
      "NRNX1-seg-123 -> ['NDB_619', 'NDB_752', 'NDB_482']\n",
      "NRNX1-seg-124 -> ['NDB_946', 'NDB_899', 'NDB_941']\n",
      "NRNX1-seg-125 -> ['NDB_1066', 'NDB_935', 'NDB_788']\n",
      "NRNX1-seg-126 -> ['NDB_951', 'NDB_1053', 'NDB_372']\n",
      "NRNX1-seg-127 -> ['NDB_735', 'NDB_450', 'NDB_513']\n",
      "NRNX1-seg-128 -> ['NDB_713', 'NDB_725', 'NDB_936']\n",
      "NRNX1-seg-129 -> ['NDB_965', 'NDB_431', 'NDB_804']\n",
      "NRNX1-seg-130 -> ['NDB_746', 'NDB_773', 'NDB_858']\n",
      "NRNX1-seg-131 -> ['NDB_455', 'NDB_737', 'NDB_438']\n",
      "NRNX1-seg-133 -> ['NDB_536', 'NDB_653', 'NDB_789']\n",
      "NRNX1-seg-134 -> ['NDB_887', 'NDB_347', 'NDB_1047']\n",
      "NRNX1-seg-135 -> ['NDB_896', 'NDB_641', 'NDB_618']\n",
      "NRNX1-seg-136 -> ['NDB_356', 'NDB_734', 'NDB_819']\n",
      "NRNX1-seg-137 -> ['NDB_862', 'NDB_401', 'NDB_611']\n",
      "NRNX1-seg-138 -> ['NDB_418', 'NDB_824', 'NDB_614']\n",
      "NRNX1-seg-139 -> ['NDB_826', 'NDB_652', 'NDB_699']\n",
      "NRNX1-seg-140 -> ['NDB_760', 'NDB_844', 'NDB_762']\n",
      "NRNX1-seg-141 -> ['NDB_466', 'NDB_409', 'NDB_786']\n",
      "NRNX1-seg-142 -> ['NDB_529', 'NDB_754', 'NDB_681']\n",
      "NRNX1-seg-143 -> ['NDB_715', 'NDB_532', 'NDB_444']\n",
      "NRNX1-seg-144 -> ['NDB_1060', 'NDB_730', 'NDB_747']\n",
      "NRNX1-seg-145 -> ['NDB_433', 'NDB_853', 'NDB_1044']\n",
      "NRNX1-seg-146 -> ['NDB_994', 'NDB_445', 'NDB_711']\n",
      "NRNX1-seg-147 -> ['NDB_751', 'NDB_594', 'NDB_341']\n",
      "NRNX1-seg-148 -> ['NDB_517', 'NDB_903', 'NDB_395']\n",
      "NRNX1-seg-150 -> ['NDB_807', 'NDB_718', 'NDB_743']\n",
      "NRNX1-seg-151 -> ['NDB_768', 'NDB_961', 'NDB_728']\n",
      "NRNX1-seg-152 -> ['NDB_516', 'NDB_571', 'NDB_860']\n",
      "NRNX1-seg-154 -> ['NDB_657', 'NDB_856', 'NDB_878']\n",
      "NRNX1-seg-155 -> ['NDB_771', 'NDB_616', 'NDB_467']\n",
      "NRNX1-seg-156 -> ['NDB_615', 'NDB_880', 'NDB_446']\n",
      "NRNX1-seg-157 -> ['NDB_453', 'NDB_505', 'NDB_452']\n",
      "NRNX1-seg-158 -> ['NDB_828', 'NDB_706', 'NDB_587']\n",
      "NRNX1-seg-159 -> ['NDB_779', 'NDB_705', 'NDB_379']\n",
      "NRNX1-seg-160 -> ['NDB_386', 'NDB_519', 'NDB_544']\n",
      "NRNX1-seg-161 -> ['NDB_795', 'NDB_376', 'NDB_971']\n",
      "NRNX1-seg-162 -> ['NDB_738', 'NDB_400', 'NDB_917']\n",
      "NRNX1-seg-163 -> ['NDB_387', 'NDB_511', 'NDB_716']\n",
      "NRNX1-seg-164 -> ['NDB_465', 'NDB_508', 'NDB_863']\n",
      "NRNX1-seg-165 -> ['NDB_1035', 'NDB_883', 'NDB_905']\n",
      "NRNX1-seg-166 -> ['NDB_921', 'NDB_427', 'NDB_482']\n",
      "NRNX1-seg-169 -> ['NDB_864', 'NDB_397', 'NDB_941']\n",
      "NRNX1-seg-170 -> ['NDB_498', 'NDB_514', 'NDB_788']\n",
      "NRNX1-seg-171 -> ['NDB_628', 'NDB_694', 'NDB_742']\n",
      "NRNX1-seg-172 -> ['NDB_526', 'NDB_622', 'NDB_925']\n",
      "NRNX1-seg-173 -> ['NDB_755', 'NDB_777', 'NDB_533']\n",
      "NRNX1-seg-174 -> ['NDB_683', 'NDB_750', 'NDB_464']\n",
      "NRNX1-seg-175 -> ['NDB_458', 'NDB_621', 'NDB_875']\n",
      "NRNX1-seg-176 -> ['NDB_680', 'NDB_489', 'NDB_470']\n",
      "NRNX1-seg-177 -> ['NDB_332', 'NDB_663', 'NDB_461']\n",
      "NRNX1-seg-178 -> ['NDB_335', 'NDB_501', 'NDB_506']\n",
      "NRNX1-seg-179 -> ['NDB_473', 'NDB_813', 'NDB_989']\n",
      "NRNX1-seg-180 -> ['NDB_647', 'NDB_1023', 'NDB_665']\n",
      "NRNX1-seg-181 -> ['NDB_551', 'NDB_459', 'NDB_491']\n",
      "NRNX1-seg-182 -> ['NDB_836', 'NDB_933', 'NDB_1028']\n",
      "NRNX1-seg-183 -> ['NDB_759', 'NDB_801', 'NDB_710']\n",
      "NRNX1-seg-184 -> ['NDB_456', 'NDB_825', 'NDB_554']\n",
      "NRNX1-seg-185 -> ['NDB_915', 'NDB_375', 'NDB_812']\n",
      "NRNX1-seg-186 -> ['NDB_957', 'NDB_417', 'NDB_698']\n",
      "NRNX1-seg-187 -> ['NDB_405', 'NDB_954', 'NDB_398']\n",
      "NRNX1-seg-188 -> ['NDB_390', 'NDB_780', 'NDB_752']\n",
      "NRNX1-seg-189 -> ['NDB_846', 'NDB_630', 'NDB_899']\n",
      "NRNX1-seg-190 -> ['NDB_687', 'NDB_396', 'NDB_935']\n",
      "NRNX1-seg-191 -> ['NDB_557', 'NDB_879', 'NDB_372']\n",
      "NRNX1-seg-192 -> ['NDB_929', 'NDB_840', 'NDB_513']\n",
      "NRNX1-seg-193 -> ['NDB_865', 'NDB_817', 'NDB_725']\n",
      "NRNX1-seg-194 -> ['NDB_790', 'NDB_778', 'NDB_431']\n",
      "NRNX1-seg-195 -> ['NDB_901', 'NDB_487', 'NDB_773']\n",
      "NRNX1-seg-196 -> ['NDB_403', 'NDB_1027', 'NDB_737']\n",
      "NRNX1-seg-197 -> ['NDB_1033', 'NDB_631', 'NDB_653']\n",
      "NRNX1-seg-198 -> ['NDB_619', 'NDB_805', 'NDB_347']\n",
      "NRNX1-seg-199 -> ['NDB_946', 'NDB_721', 'NDB_641']\n",
      "NRNX1-seg-200 -> ['NDB_1066', 'NDB_637', 'NDB_734']\n",
      "NRNX1-seg-201 -> ['NDB_1014', 'NDB_1053', 'NDB_958']\n",
      "NRNX1-seg-202 -> ['NDB_627', 'NDB_450', 'NDB_712']\n",
      "NRNX1-seg-203 -> ['NDB_826', 'NDB_690', 'NDB_847']\n",
      "NRNX1-seg-204 -> ['NDB_760', 'NDB_852', 'NDB_661']\n",
      "NRNX1-seg-205 -> ['NDB_466', 'NDB_432', 'NDB_448']\n",
      "NRNX1-seg-206 -> ['NDB_529', 'NDB_1020', 'NDB_391']\n",
      "NRNX1-seg-207 -> ['NDB_715', 'NDB_741', 'NDB_1075']\n",
      "NRNX1-seg-208 -> ['NDB_1060', 'NDB_507', 'NDB_832']\n",
      "NRNX1-seg-209 -> ['NDB_433', 'NDB_816', 'NDB_385']\n",
      "NRNX1-seg-210 -> ['NDB_994', 'NDB_660', 'NDB_889']\n",
      "NRNX1-seg-211 -> ['NDB_421', 'NDB_951', 'NDB_694']\n",
      "NRNX1-seg-212 -> ['NDB_472', 'NDB_735', 'NDB_622']\n",
      "NRNX1-seg-213 -> ['NDB_777', 'NDB_652', 'NDB_971']\n",
      "NRNX1-seg-214 -> ['NDB_750', 'NDB_844', 'NDB_917']\n",
      "NRNX1-seg-215 -> ['NDB_621', 'NDB_409', 'NDB_716']\n",
      "NRNX1-seg-216 -> ['NDB_489', 'NDB_754', 'NDB_863']\n",
      "NRNX1-seg-217 -> ['NDB_663', 'NDB_532', 'NDB_905']\n",
      "NRNX1-seg-218 -> ['NDB_501', 'NDB_730', 'NDB_482']\n",
      "NRNX1-seg-219 -> ['NDB_813', 'NDB_853', 'NDB_941']\n",
      "NRNX1-seg-220 -> ['NDB_1023', 'NDB_445', 'NDB_788']\n",
      "NRNX1-seg-221 -> ['NDB_705', 'NDB_742', 'NDB_341']\n",
      "NRNX1-seg-222 -> ['NDB_519', 'NDB_925', 'NDB_395']\n",
      "NRNX1-seg-223 -> ['NDB_713', 'NDB_718', 'NDB_699']\n",
      "NRNX1-seg-224 -> ['NDB_965', 'NDB_961', 'NDB_762']\n",
      "NRNX1-seg-225 -> ['NDB_746', 'NDB_571', 'NDB_786']\n",
      "NRNX1-seg-226 -> ['NDB_455', 'NDB_856', 'NDB_681']\n",
      "NRNX1-seg-227 -> ['NDB_536', 'NDB_616', 'NDB_444']\n",
      "NRNX1-seg-228 -> ['NDB_887', 'NDB_880', 'NDB_747']\n",
      "NRNX1-seg-229 -> ['NDB_896', 'NDB_505', 'NDB_1044']\n",
      "NRNX1-seg-230 -> ['NDB_356', 'NDB_706', 'NDB_711']\n",
      "NRNX1-seg-231 -> ['NDB_628', 'NDB_491', 'NDB_751']\n",
      "NRNX1-seg-232 -> ['NDB_526', 'NDB_1028', 'NDB_517']\n",
      "NRNX1-seg-233 -> ['NDB_795', 'NDB_533', 'NDB_725']\n",
      "NRNX1-seg-234 -> ['NDB_738', 'NDB_464', 'NDB_431']\n",
      "NRNX1-seg-235 -> ['NDB_387', 'NDB_875', 'NDB_773']\n",
      "NRNX1-seg-236 -> ['NDB_465', 'NDB_470', 'NDB_737']\n",
      "NRNX1-seg-238 -> ['NDB_1035', 'NDB_461', 'NDB_653']\n",
      "NRNX1-seg-239 -> ['NDB_921', 'NDB_506', 'NDB_347']\n",
      "NRNX1-seg-240 -> ['NDB_864', 'NDB_989', 'NDB_641']\n",
      "NRNX1-seg-241 -> ['NDB_498', 'NDB_665', 'NDB_734']\n",
      "NRNX1-seg-242 -> ['NDB_557', 'NDB_594', 'NDB_611']\n",
      "NRNX1-seg-244 -> ['NDB_929', 'NDB_903', 'NDB_614']\n",
      "NRNX1-seg-245 -> ['NDB_755', 'NDB_759', 'NDB_578']\n",
      "NRNX1-seg-246 -> ['NDB_683', 'NDB_456', 'NDB_815']\n",
      "NRNX1-seg-248 -> ['NDB_458', 'NDB_915', 'NDB_974']\n",
      "NRNX1-seg-249 -> ['NDB_680', 'NDB_957', 'NDB_443']\n",
      "NRNX1-seg-251 -> ['NDB_332', 'NDB_405', 'NDB_758']\n",
      "NRNX1-seg-252 -> ['NDB_335', 'NDB_390', 'NDB_866']\n",
      "NRNX1-seg-253 -> ['NDB_473', 'NDB_846', 'NDB_539']\n",
      "NRNX1-seg-254 -> ['NDB_647', 'NDB_687', 'NDB_881']\n",
      "NRNX1-seg-255 -> ['NDB_862', 'NDB_879', 'NDB_379']\n",
      "NRNX1-seg-256 -> ['NDB_418', 'NDB_840', 'NDB_544']\n",
      "NRNX1-seg-257 -> ['NDB_784', 'NDB_710', 'NDB_743']\n",
      "NRNX1-seg-258 -> ['NDB_643', 'NDB_554', 'NDB_728']\n",
      "NRNX1-seg-259 -> ['NDB_568', 'NDB_812', 'NDB_860']\n",
      "NRNX1-seg-260 -> ['NDB_868', 'NDB_698', 'NDB_878']\n",
      "NRNX1-seg-262 -> ['NDB_634', 'NDB_398', 'NDB_467']\n",
      "NRNX1-seg-263 -> ['NDB_388', 'NDB_752', 'NDB_446']\n",
      "NRNX1-seg-264 -> ['NDB_835', 'NDB_899', 'NDB_452']\n",
      "NRNX1-seg-265 -> ['NDB_838', 'NDB_935', 'NDB_587']\n",
      "NRNX1-seg-266 -> ['NDB_401', 'NDB_977', 'NDB_1053']\n",
      "NRNX1-seg-267 -> ['NDB_824', 'NDB_392', 'NDB_450']\n",
      "NRNX1-seg-268 -> ['NDB_376', 'NDB_777', 'NDB_865']\n",
      "NRNX1-seg-269 -> ['NDB_400', 'NDB_750', 'NDB_790']\n",
      "NRNX1-seg-270 -> ['NDB_511', 'NDB_621', 'NDB_901']\n",
      "NRNX1-seg-271 -> ['NDB_508', 'NDB_489', 'NDB_403']\n",
      "NRNX1-seg-272 -> ['NDB_883', 'NDB_663', 'NDB_1033']\n",
      "NRNX1-seg-273 -> ['NDB_427', 'NDB_501', 'NDB_619']\n",
      "NRNX1-seg-274 -> ['NDB_397', 'NDB_813', 'NDB_946']\n",
      "NRNX1-seg-275 -> ['NDB_514', 'NDB_1023', 'NDB_1066']\n",
      "NRNX1-seg-276 -> ['NDB_694', 'NDB_779', 'NDB_958']\n",
      "NRNX1-seg-277 -> ['NDB_622', 'NDB_386', 'NDB_712']\n",
      "NRNX1-seg-278 -> ['NDB_533', 'NDB_807', 'NDB_652']\n",
      "NRNX1-seg-279 -> ['NDB_464', 'NDB_768', 'NDB_844']\n",
      "NRNX1-seg-280 -> ['NDB_875', 'NDB_516', 'NDB_409']\n",
      "NRNX1-seg-281 -> ['NDB_470', 'NDB_657', 'NDB_754']\n",
      "NRNX1-seg-282 -> ['NDB_461', 'NDB_771', 'NDB_532']\n",
      "NRNX1-seg-283 -> ['NDB_506', 'NDB_615', 'NDB_730']\n",
      "NRNX1-seg-284 -> ['NDB_989', 'NDB_453', 'NDB_853']\n",
      "NRNX1-seg-285 -> ['NDB_665', 'NDB_828', 'NDB_445']\n",
      "NRNX1-seg-286 -> ['NDB_459', 'NDB_751', 'NDB_372']\n",
      "NRNX1-seg-287 -> ['NDB_933', 'NDB_517', 'NDB_513']\n",
      "NRNX1-seg-288 -> ['NDB_795', 'NDB_817', 'NDB_936']\n",
      "NRNX1-seg-289 -> ['NDB_738', 'NDB_778', 'NDB_804']\n",
      "NRNX1-seg-290 -> ['NDB_387', 'NDB_487', 'NDB_858']\n",
      "NRNX1-seg-291 -> ['NDB_465', 'NDB_1027', 'NDB_438']\n",
      "NRNX1-seg-292 -> ['NDB_1035', 'NDB_631', 'NDB_789']\n",
      "NRNX1-seg-293 -> ['NDB_921', 'NDB_805', 'NDB_1047']\n",
      "NRNX1-seg-294 -> ['NDB_864', 'NDB_721', 'NDB_618']\n",
      "NRNX1-seg-295 -> ['NDB_498', 'NDB_637', 'NDB_819']\n",
      "NRNX1-seg-298 -> ['NDB_1014', 'NDB_628', 'NDB_611']\n",
      "NRNX1-seg-299 -> ['NDB_627', 'NDB_526', 'NDB_614']\n",
      "NRNX1-seg-300 -> ['NDB_713', 'NDB_801', 'NDB_690']\n",
      "NRNX1-seg-301 -> ['NDB_965', 'NDB_825', 'NDB_852']\n",
      "NRNX1-seg-302 -> ['NDB_746', 'NDB_375', 'NDB_432']\n",
      "NRNX1-seg-303 -> ['NDB_455', 'NDB_417', 'NDB_1020']\n",
      "NRNX1-seg-304 -> ['NDB_536', 'NDB_954', 'NDB_741']\n",
      "NRNX1-seg-305 -> ['NDB_887', 'NDB_780', 'NDB_507']\n",
      "NRNX1-seg-306 -> ['NDB_896', 'NDB_630', 'NDB_816']\n",
      "NRNX1-seg-307 -> ['NDB_356', 'NDB_396', 'NDB_660']\n",
      "NRNX1-seg-308 -> ['NDB_551', 'NDB_379', 'NDB_742']\n",
      "NRNX1-seg-311 -> ['NDB_836', 'NDB_544', 'NDB_925']\n",
      "NRNX1-seg-312 -> ['NDB_759', 'NDB_826', 'NDB_971']\n",
      "NRNX1-seg-313 -> ['NDB_456', 'NDB_760', 'NDB_917']\n",
      "NRNX1-seg-314 -> ['NDB_915', 'NDB_466', 'NDB_716']\n",
      "NRNX1-seg-315 -> ['NDB_957', 'NDB_529', 'NDB_863']\n",
      "NRNX1-seg-316 -> ['NDB_405', 'NDB_715', 'NDB_905']\n",
      "NRNX1-seg-317 -> ['NDB_390', 'NDB_1060', 'NDB_482']\n",
      "NRNX1-seg-318 -> ['NDB_846', 'NDB_433', 'NDB_941']\n",
      "NRNX1-seg-319 -> ['NDB_687', 'NDB_994', 'NDB_788']\n",
      "NRNX1-seg-320 -> ['NDB_491', 'NDB_951', 'NDB_341']\n",
      "NRNX1-seg-321 -> ['NDB_1028', 'NDB_735', 'NDB_395']\n",
      "NRNX1-seg-322 -> ['NDB_578', 'NDB_865', 'NDB_743']\n",
      "NRNX1-seg-323 -> ['NDB_815', 'NDB_790', 'NDB_728']\n",
      "NRNX1-seg-324 -> ['NDB_974', 'NDB_901', 'NDB_860']\n",
      "NRNX1-seg-325 -> ['NDB_443', 'NDB_403', 'NDB_878']\n",
      "NRNX1-seg-326 -> ['NDB_758', 'NDB_1033', 'NDB_467']\n",
      "NRNX1-seg-327 -> ['NDB_866', 'NDB_619', 'NDB_446']\n",
      "NRNX1-seg-328 -> ['NDB_539', 'NDB_946', 'NDB_452']\n",
      "NRNX1-seg-329 -> ['NDB_881', 'NDB_1066', 'NDB_587']\n",
      "NRNX1-seg-330 -> ['NDB_862', 'NDB_557', 'NDB_421']\n",
      "NRNX1-seg-331 -> ['NDB_418', 'NDB_929', 'NDB_472']\n",
      "NRNX1-seg-332 -> ['NDB_376', 'NDB_807', 'NDB_847']\n",
      "NRNX1-seg-333 -> ['NDB_400', 'NDB_768', 'NDB_661']\n",
      "NRNX1-seg-334 -> ['NDB_511', 'NDB_516', 'NDB_448']\n",
      "NRNX1-seg-335 -> ['NDB_508', 'NDB_657', 'NDB_391']\n",
      "NRNX1-seg-336 -> ['NDB_883', 'NDB_771', 'NDB_1075']\n",
      "NRNX1-seg-337 -> ['NDB_427', 'NDB_615', 'NDB_832']\n",
      "NRNX1-seg-338 -> ['NDB_397', 'NDB_453', 'NDB_385']\n",
      "NRNX1-seg-339 -> ['NDB_514', 'NDB_828', 'NDB_889']\n",
      "NRNX1-seg-340 -> ['NDB_459', 'NDB_779', 'NDB_401']\n",
      "NRNX1-seg-341 -> ['NDB_933', 'NDB_386', 'NDB_824']\n",
      "NRNX1-seg-342 -> ['NDB_801', 'NDB_725', 'NDB_718']\n",
      "NRNX1-seg-343 -> ['NDB_825', 'NDB_431', 'NDB_961']\n",
      "NRNX1-seg-344 -> ['NDB_375', 'NDB_773', 'NDB_571']\n",
      "NRNX1-seg-345 -> ['NDB_417', 'NDB_737', 'NDB_856']\n",
      "NRNX1-seg-346 -> ['NDB_954', 'NDB_653', 'NDB_616']\n",
      "NRNX1-seg-347 -> ['NDB_780', 'NDB_347', 'NDB_880']\n",
      "NRNX1-seg-348 -> ['NDB_630', 'NDB_641', 'NDB_505']\n",
      "NRNX1-seg-349 -> ['NDB_396', 'NDB_734', 'NDB_706']\n",
      "NRNX1-seg-350 -> ['NDB_742', 'NDB_611', 'NDB_372']\n",
      "NRNX1-seg-351 -> ['NDB_925', 'NDB_614', 'NDB_513']\n",
      "NRNX1-seg-352 -> ['NDB_755', 'NDB_690', 'NDB_936']\n",
      "NRNX1-seg-353 -> ['NDB_683', 'NDB_852', 'NDB_804']\n",
      "NRNX1-seg-354 -> ['NDB_458', 'NDB_432', 'NDB_858']\n",
      "NRNX1-seg-355 -> ['NDB_680', 'NDB_1020', 'NDB_438']\n",
      "NRNX1-seg-356 -> ['NDB_332', 'NDB_741', 'NDB_789']\n",
      "NRNX1-seg-357 -> ['NDB_335', 'NDB_507', 'NDB_1047']\n",
      "NRNX1-seg-358 -> ['NDB_473', 'NDB_816', 'NDB_618']\n",
      "NRNX1-seg-359 -> ['NDB_647', 'NDB_660', 'NDB_819']\n",
      "NRNX1-seg-360 -> ['NDB_951', 'NDB_977', 'NDB_958']\n",
      "NRNX1-seg-361 -> ['NDB_735', 'NDB_392', 'NDB_712']\n",
      "NRNX1-seg-362 -> ['NDB_784', 'NDB_713', 'NDB_865']\n",
      "NRNX1-seg-363 -> ['NDB_643', 'NDB_965', 'NDB_790']\n",
      "NRNX1-seg-364 -> ['NDB_568', 'NDB_746', 'NDB_901']\n",
      "NRNX1-seg-365 -> ['NDB_868', 'NDB_455', 'NDB_403']\n",
      "NRNX1-seg-366 -> ['NDB_634', 'NDB_536', 'NDB_1033']\n",
      "NRNX1-seg-367 -> ['NDB_388', 'NDB_887', 'NDB_619']\n",
      "NRNX1-seg-368 -> ['NDB_835', 'NDB_896', 'NDB_946']\n",
      "NRNX1-seg-369 -> ['NDB_838', 'NDB_356', 'NDB_1066']\n",
      "NRNX1-seg-370 -> ['NDB_421', 'NDB_879', 'NDB_751']\n",
      "NRNX1-seg-371 -> ['NDB_472', 'NDB_840', 'NDB_517']\n",
      "NRNX1-seg-372 -> ['NDB_578', 'NDB_795', 'NDB_652']\n",
      "NRNX1-seg-373 -> ['NDB_815', 'NDB_738', 'NDB_844']\n",
      "NRNX1-seg-374 -> ['NDB_974', 'NDB_387', 'NDB_409']\n",
      "NRNX1-seg-375 -> ['NDB_443', 'NDB_465', 'NDB_754']\n",
      "NRNX1-seg-376 -> ['NDB_758', 'NDB_1035', 'NDB_532']\n",
      "NRNX1-seg-379 -> ['NDB_866', 'NDB_921', 'NDB_730']\n",
      "NRNX1-seg-380 -> ['NDB_539', 'NDB_864', 'NDB_853']\n",
      "NRNX1-seg-381 -> ['NDB_881', 'NDB_498', 'NDB_445']\n",
      "NRNX1-seg-382 -> ['NDB_551', 'NDB_1014', 'NDB_557']\n",
      "NRNX1-seg-383 -> ['NDB_836', 'NDB_627', 'NDB_929']\n",
      "NRNX1-seg-384 -> ['NDB_777', 'NDB_710', 'NDB_847']\n",
      "NRNX1-seg-385 -> ['NDB_750', 'NDB_554', 'NDB_661']\n",
      "NRNX1-seg-386 -> ['NDB_621', 'NDB_812', 'NDB_448']\n",
      "NRNX1-seg-387 -> ['NDB_489', 'NDB_698', 'NDB_391']\n",
      "NRNX1-seg-388 -> ['NDB_663', 'NDB_398', 'NDB_1075']\n",
      "NRNX1-seg-389 -> ['NDB_501', 'NDB_752', 'NDB_832']\n",
      "NRNX1-seg-390 -> ['NDB_813', 'NDB_899', 'NDB_385']\n",
      "NRNX1-seg-391 -> ['NDB_1023', 'NDB_935', 'NDB_889']\n",
      "NRNX1-seg-392 -> ['NDB_459', 'NDB_694', 'NDB_594']\n",
      "NRNX1-seg-393 -> ['NDB_933', 'NDB_622', 'NDB_903']\n",
      "NRNX1-seg-394 -> ['NDB_759', 'NDB_533', 'NDB_743']\n",
      "NRNX1-seg-395 -> ['NDB_456', 'NDB_464', 'NDB_728']\n",
      "NRNX1-seg-396 -> ['NDB_915', 'NDB_875', 'NDB_860']\n",
      "NRNX1-seg-397 -> ['NDB_957', 'NDB_470', 'NDB_878']\n",
      "NRNX1-seg-398 -> ['NDB_405', 'NDB_461', 'NDB_467']\n",
      "NRNX1-seg-399 -> ['NDB_390', 'NDB_506', 'NDB_446']\n",
      "NRNX1-seg-400 -> ['NDB_846', 'NDB_989', 'NDB_452']\n",
      "NRNX1-seg-401 -> ['NDB_687', 'NDB_665', 'NDB_587']\n",
      "NRNX1-seg-402 -> ['NDB_628', 'NDB_379', 'NDB_1053']\n",
      "NRNX1-seg-403 -> ['NDB_526', 'NDB_544', 'NDB_450']\n",
      "NRNX1-seg-404 -> ['NDB_718', 'NDB_971', 'NDB_936']\n",
      "NRNX1-seg-405 -> ['NDB_961', 'NDB_917', 'NDB_804']\n",
      "NRNX1-seg-406 -> ['NDB_571', 'NDB_716', 'NDB_858']\n",
      "NRNX1-seg-407 -> ['NDB_856', 'NDB_863', 'NDB_438']\n",
      "NRNX1-seg-408 -> ['NDB_616', 'NDB_905', 'NDB_789']\n",
      "NRNX1-seg-409 -> ['NDB_880', 'NDB_482', 'NDB_1047']\n",
      "NRNX1-seg-410 -> ['NDB_505', 'NDB_941', 'NDB_618']\n",
      "NRNX1-seg-411 -> ['NDB_706', 'NDB_788', 'NDB_819']\n",
      "NRNX1-seg-412 -> ['NDB_421', 'NDB_705', 'NDB_958']\n",
      "NRNX1-seg-413 -> ['NDB_472', 'NDB_519', 'NDB_712']\n",
      "NRNX1-seg-414 -> ['NDB_826', 'NDB_713', 'NDB_817']\n",
      "NRNX1-seg-415 -> ['NDB_760', 'NDB_965', 'NDB_778']\n",
      "NRNX1-seg-416 -> ['NDB_466', 'NDB_746', 'NDB_487']\n",
      "NRNX1-seg-417 -> ['NDB_529', 'NDB_455', 'NDB_1027']\n",
      "NRNX1-seg-418 -> ['NDB_715', 'NDB_536', 'NDB_631']\n",
      "NRNX1-seg-419 -> ['NDB_1060', 'NDB_887', 'NDB_805']\n",
      "NRNX1-seg-420 -> ['NDB_433', 'NDB_896', 'NDB_721']\n",
      "NRNX1-seg-421 -> ['NDB_994', 'NDB_356', 'NDB_637']\n",
      "NRNX1-seg-422 -> ['NDB_862', 'NDB_977', 'NDB_341']\n",
      "NRNX1-seg-423 -> ['NDB_418', 'NDB_392', 'NDB_395']\n",
      "NRNX1-seg-424 -> ['NDB_784', 'NDB_578', 'NDB_376']\n",
      "NRNX1-seg-425 -> ['NDB_643', 'NDB_815', 'NDB_400']\n",
      "NRNX1-seg-426 -> ['NDB_568', 'NDB_974', 'NDB_511']\n",
      "NRNX1-seg-427 -> ['NDB_868', 'NDB_443', 'NDB_508']\n",
      "NRNX1-seg-428 -> ['NDB_634', 'NDB_758', 'NDB_883']\n",
      "NRNX1-seg-429 -> ['NDB_388', 'NDB_866', 'NDB_427']\n",
      "NRNX1-seg-430 -> ['NDB_835', 'NDB_539', 'NDB_397']\n",
      "NRNX1-seg-431 -> ['NDB_838', 'NDB_881', 'NDB_514']\n",
      "NRNX1-seg-432 -> ['NDB_879', 'NDB_401', 'NDB_742']\n",
      "NRNX1-seg-433 -> ['NDB_840', 'NDB_824', 'NDB_925']\n",
      "NRNX1-seg-434 -> ['NDB_725', 'NDB_652', 'NDB_847']\n",
      "NRNX1-seg-435 -> ['NDB_431', 'NDB_844', 'NDB_661']\n",
      "NRNX1-seg-436 -> ['NDB_773', 'NDB_409', 'NDB_448']\n",
      "NRNX1-seg-437 -> ['NDB_737', 'NDB_754', 'NDB_391']\n",
      "NRNX1-seg-438 -> ['NDB_653', 'NDB_532', 'NDB_1075']\n",
      "NRNX1-seg-439 -> ['NDB_347', 'NDB_730', 'NDB_832']\n",
      "NRNX1-seg-440 -> ['NDB_641', 'NDB_853', 'NDB_385']\n",
      "NRNX1-seg-441 -> ['NDB_734', 'NDB_445', 'NDB_889']\n",
      "NRNX1-seg-442 -> ['NDB_551', 'NDB_594', 'NDB_372']\n",
      "NRNX1-seg-443 -> ['NDB_836', 'NDB_903', 'NDB_513']\n",
      "NRNX1-seg-444 -> ['NDB_755', 'NDB_807', 'NDB_699']\n",
      "NRNX1-seg-445 -> ['NDB_683', 'NDB_768', 'NDB_762']\n",
      "NRNX1-seg-446 -> ['NDB_458', 'NDB_516', 'NDB_786']\n",
      "NRNX1-seg-447 -> ['NDB_680', 'NDB_657', 'NDB_681']\n",
      "NRNX1-seg-448 -> ['NDB_332', 'NDB_771', 'NDB_444']\n",
      "NRNX1-seg-449 -> ['NDB_335', 'NDB_615', 'NDB_747']\n",
      "NRNX1-seg-450 -> ['NDB_473', 'NDB_453', 'NDB_1044']\n",
      "NRNX1-seg-452 -> ['NDB_647', 'NDB_828', 'NDB_711']\n",
      "NRNX1-seg-453 -> ['NDB_1014', 'NDB_694', 'NDB_379']\n",
      "NRNX1-seg-454 -> ['NDB_627', 'NDB_622', 'NDB_544']\n",
      "NRNX1-seg-455 -> ['NDB_533', 'NDB_710', 'NDB_690']\n",
      "NRNX1-seg-456 -> ['NDB_464', 'NDB_554', 'NDB_852']\n",
      "NRNX1-seg-457 -> ['NDB_875', 'NDB_812', 'NDB_432']\n",
      "NRNX1-seg-458 -> ['NDB_470', 'NDB_698', 'NDB_1020']\n",
      "NRNX1-seg-459 -> ['NDB_461', 'NDB_398', 'NDB_741']\n",
      "NRNX1-seg-460 -> ['NDB_506', 'NDB_752', 'NDB_507']\n",
      "NRNX1-seg-461 -> ['NDB_989', 'NDB_899', 'NDB_816']\n",
      "NRNX1-seg-462 -> ['NDB_665', 'NDB_935', 'NDB_660']\n",
      "NRNX1-seg-463 -> ['NDB_628', 'NDB_951', 'NDB_705']\n",
      "NRNX1-seg-464 -> ['NDB_526', 'NDB_735', 'NDB_519']\n",
      "NRNX1-seg-465 -> ['NDB_795', 'NDB_801', 'NDB_743']\n",
      "NRNX1-seg-466 -> ['NDB_738', 'NDB_825', 'NDB_728']\n",
      "NRNX1-seg-467 -> ['NDB_387', 'NDB_375', 'NDB_860']\n",
      "NRNX1-seg-468 -> ['NDB_465', 'NDB_417', 'NDB_878']\n",
      "NRNX1-seg-469 -> ['NDB_1035', 'NDB_954', 'NDB_467']\n",
      "NRNX1-seg-470 -> ['NDB_921', 'NDB_780', 'NDB_446']\n",
      "NRNX1-seg-471 -> ['NDB_864', 'NDB_630', 'NDB_452']\n",
      "NRNX1-seg-472 -> ['NDB_498', 'NDB_396', 'NDB_587']\n",
      "NRNX1-seg-474 -> ['NDB_459', 'NDB_421', 'NDB_611']\n",
      "NRNX1-seg-475 -> ['NDB_933', 'NDB_472', 'NDB_614']\n",
      "NRNX1-seg-476 -> ['NDB_826', 'NDB_865', 'NDB_718']\n",
      "NRNX1-seg-477 -> ['NDB_760', 'NDB_790', 'NDB_961']\n",
      "NRNX1-seg-478 -> ['NDB_466', 'NDB_901', 'NDB_571']\n",
      "NRNX1-seg-479 -> ['NDB_529', 'NDB_403', 'NDB_856']\n",
      "NRNX1-seg-480 -> ['NDB_715', 'NDB_1033', 'NDB_616']\n",
      "NRNX1-seg-481 -> ['NDB_1060', 'NDB_619', 'NDB_880']\n",
      "NRNX1-seg-482 -> ['NDB_433', 'NDB_946', 'NDB_505']\n",
      "NRNX1-seg-483 -> ['NDB_994', 'NDB_1066', 'NDB_706']\n",
      "NRNX1-seg-484 -> ['NDB_779', 'NDB_977', 'NDB_594']\n",
      "NRNX1-seg-485 -> ['NDB_386', 'NDB_392', 'NDB_903']\n",
      "NRNX1-seg-486 -> ['NDB_777', 'NDB_699', 'NDB_936']\n",
      "NRNX1-seg-487 -> ['NDB_750', 'NDB_762', 'NDB_804']\n",
      "NRNX1-seg-488 -> ['NDB_621', 'NDB_786', 'NDB_858']\n",
      "NRNX1-seg-489 -> ['NDB_489', 'NDB_681', 'NDB_438']\n",
      "NRNX1-seg-490 -> ['NDB_663', 'NDB_444', 'NDB_789']\n",
      "NRNX1-seg-491 -> ['NDB_501', 'NDB_747', 'NDB_1047']\n",
      "NRNX1-seg-492 -> ['NDB_813', 'NDB_1044', 'NDB_618']\n",
      "NRNX1-seg-493 -> ['NDB_1023', 'NDB_711', 'NDB_819']\n",
      "NRNX1-seg-494 -> ['NDB_491', 'NDB_742', 'NDB_1053']\n",
      "NRNX1-seg-495 -> ['NDB_1028', 'NDB_925', 'NDB_450']\n",
      "NRNX1-seg-496 -> ['NDB_784', 'NDB_817', 'NDB_847']\n",
      "NRNX1-seg-497 -> ['NDB_643', 'NDB_778', 'NDB_661']\n",
      "NRNX1-seg-498 -> ['NDB_568', 'NDB_487', 'NDB_448']\n",
      "NRNX1-seg-499 -> ['NDB_868', 'NDB_1027', 'NDB_391']\n",
      "NRNX1-seg-500 -> ['NDB_634', 'NDB_631', 'NDB_1075']\n",
      "NRNX1-seg-501 -> ['NDB_388', 'NDB_805', 'NDB_832']\n",
      "NRNX1-seg-502 -> ['NDB_835', 'NDB_721', 'NDB_385']\n",
      "NRNX1-seg-503 -> ['NDB_838', 'NDB_637', 'NDB_889']\n",
      "NRNX1-seg-504 -> ['NDB_551', 'NDB_401', 'NDB_341']\n",
      "NRNX1-seg-505 -> ['NDB_836', 'NDB_824', 'NDB_395']\n",
      "NRNX1-seg-506 -> ['NDB_795', 'NDB_807', 'NDB_710']\n",
      "NRNX1-seg-507 -> ['NDB_738', 'NDB_768', 'NDB_554']\n",
      "NRNX1-seg-508 -> ['NDB_387', 'NDB_516', 'NDB_812']\n",
      "NRNX1-seg-509 -> ['NDB_465', 'NDB_657', 'NDB_698']\n",
      "NRNX1-seg-510 -> ['NDB_1035', 'NDB_771', 'NDB_398']\n",
      "NRNX1-seg-511 -> ['NDB_921', 'NDB_615', 'NDB_752']\n",
      "NRNX1-seg-512 -> ['NDB_864', 'NDB_453', 'NDB_899']\n",
      "NRNX1-seg-513 -> ['NDB_498', 'NDB_828', 'NDB_935']\n",
      "NRNX1-seg-514 -> ['NDB_1014', 'NDB_951', 'NDB_751']\n",
      "NRNX1-seg-515 -> ['NDB_627', 'NDB_735', 'NDB_517']\n",
      "NRNX1-seg-516 -> ['NDB_578', 'NDB_690', 'NDB_971']\n",
      "NRNX1-seg-517 -> ['NDB_815', 'NDB_852', 'NDB_917']\n",
      "NRNX1-seg-518 -> ['NDB_974', 'NDB_432', 'NDB_716']\n",
      "NRNX1-seg-519 -> ['NDB_443', 'NDB_1020', 'NDB_863']\n",
      "NRNX1-seg-520 -> ['NDB_758', 'NDB_741', 'NDB_905']\n",
      "NRNX1-seg-521 -> ['NDB_866', 'NDB_507', 'NDB_482']\n",
      "NRNX1-seg-522 -> ['NDB_539', 'NDB_816', 'NDB_941']\n",
      "NRNX1-seg-523 -> ['NDB_881', 'NDB_660', 'NDB_788']\n",
      "NRNX1-seg-524 -> ['NDB_557', 'NDB_459', 'NDB_379']\n",
      "NRNX1-seg-525 -> ['NDB_929', 'NDB_933', 'NDB_544']\n",
      "NRNX1-seg-526 -> ['NDB_376', 'NDB_652', 'NDB_743']\n",
      "NRNX1-seg-527 -> ['NDB_400', 'NDB_844', 'NDB_728']\n",
      "NRNX1-seg-528 -> ['NDB_511', 'NDB_409', 'NDB_860']\n",
      "NRNX1-seg-529 -> ['NDB_508', 'NDB_754', 'NDB_878']\n",
      "NRNX1-seg-530 -> ['NDB_883', 'NDB_532', 'NDB_467']\n",
      "NRNX1-seg-531 -> ['NDB_427', 'NDB_730', 'NDB_446']\n",
      "NRNX1-seg-532 -> ['NDB_397', 'NDB_853', 'NDB_452']\n",
      "NRNX1-seg-533 -> ['NDB_514', 'NDB_445', 'NDB_587']\n",
      "NRNX1-seg-534 -> ['NDB_862', 'NDB_705', 'NDB_594']\n",
      "NRNX1-seg-535 -> ['NDB_418', 'NDB_519', 'NDB_903']\n",
      "NRNX1-seg-536 -> ['NDB_826', 'NDB_533', 'NDB_801']\n",
      "NRNX1-seg-537 -> ['NDB_760', 'NDB_464', 'NDB_825']\n",
      "NRNX1-seg-538 -> ['NDB_466', 'NDB_875', 'NDB_375']\n",
      "NRNX1-seg-539 -> ['NDB_529', 'NDB_470', 'NDB_417']\n",
      "NRNX1-seg-540 -> ['NDB_715', 'NDB_461', 'NDB_954']\n",
      "NRNX1-seg-541 -> ['NDB_1060', 'NDB_506', 'NDB_780']\n",
      "NRNX1-seg-542 -> ['NDB_433', 'NDB_989', 'NDB_630']\n",
      "NRNX1-seg-543 -> ['NDB_994', 'NDB_665', 'NDB_396']\n",
      "NRNX1-seg-544 -> ['NDB_491', 'NDB_611', 'NDB_958']\n",
      "NRNX1-seg-545 -> ['NDB_1028', 'NDB_614', 'NDB_712']\n",
      "NRNX1-seg-546 -> ['NDB_784', 'NDB_759', 'NDB_936']\n",
      "NRNX1-seg-547 -> ['NDB_643', 'NDB_456', 'NDB_804']\n",
      "NRNX1-seg-548 -> ['NDB_568', 'NDB_915', 'NDB_858']\n",
      "NRNX1-seg-549 -> ['NDB_868', 'NDB_957', 'NDB_438']\n",
      "NRNX1-seg-550 -> ['NDB_634', 'NDB_405', 'NDB_789']\n",
      "NRNX1-seg-551 -> ['NDB_388', 'NDB_390', 'NDB_1047']\n",
      "NRNX1-seg-552 -> ['NDB_835', 'NDB_846', 'NDB_618']\n",
      "NRNX1-seg-553 -> ['NDB_838', 'NDB_687', 'NDB_819']\n",
      "NRNX1-seg-554 -> ['NDB_421', 'NDB_977', 'NDB_742']\n",
      "NRNX1-seg-555 -> ['NDB_472', 'NDB_392', 'NDB_925']\n",
      "NRNX1-seg-556 -> ['NDB_725', 'NDB_710', 'NDB_699']\n",
      "NRNX1-seg-557 -> ['NDB_431', 'NDB_554', 'NDB_762']\n",
      "NRNX1-seg-558 -> ['NDB_773', 'NDB_812', 'NDB_786']\n",
      "NRNX1-seg-559 -> ['NDB_737', 'NDB_698', 'NDB_681']\n",
      "NRNX1-seg-560 -> ['NDB_653', 'NDB_398', 'NDB_444']\n",
      "NRNX1-seg-561 -> ['NDB_347', 'NDB_752', 'NDB_747']\n",
      "NRNX1-seg-562 -> ['NDB_641', 'NDB_899', 'NDB_1044']\n",
      "NRNX1-seg-563 -> ['NDB_734', 'NDB_935', 'NDB_711']\n",
      "NRNX1-seg-564 -> ['NDB_1014', 'NDB_779', 'NDB_341']\n",
      "NRNX1-seg-565 -> ['NDB_627', 'NDB_386', 'NDB_395']\n",
      "NRNX1-seg-566 -> ['NDB_713', 'NDB_777', 'NDB_743']\n",
      "NRNX1-seg-567 -> ['NDB_965', 'NDB_750', 'NDB_728']\n",
      "NRNX1-seg-568 -> ['NDB_746', 'NDB_621', 'NDB_860']\n",
      "NRNX1-seg-569 -> ['NDB_455', 'NDB_489', 'NDB_878']\n",
      "NRNX1-seg-570 -> ['NDB_536', 'NDB_663', 'NDB_467']\n",
      "NRNX1-seg-571 -> ['NDB_887', 'NDB_501', 'NDB_446']\n",
      "NRNX1-seg-572 -> ['NDB_896', 'NDB_813', 'NDB_452']\n",
      "NRNX1-seg-573 -> ['NDB_356', 'NDB_1023', 'NDB_587']\n",
      "NRNX1-seg-574 -> ['NDB_557', 'NDB_751', 'NDB_1053']\n",
      "NRNX1-seg-575 -> ['NDB_929', 'NDB_517', 'NDB_450']\n",
      "NRNX1-seg-576 -> ['NDB_795', 'NDB_718', 'NDB_847']\n",
      "NRNX1-seg-577 -> ['NDB_738', 'NDB_961', 'NDB_661']\n",
      "NRNX1-seg-578 -> ['NDB_387', 'NDB_571', 'NDB_448']\n",
      "NRNX1-seg-579 -> ['NDB_465', 'NDB_856', 'NDB_391']\n",
      "NRNX1-seg-580 -> ['NDB_1035', 'NDB_616', 'NDB_1075']\n",
      "NRNX1-seg-581 -> ['NDB_921', 'NDB_880', 'NDB_832']\n",
      "NRNX1-seg-582 -> ['NDB_864', 'NDB_505', 'NDB_385']\n",
      "NRNX1-seg-583 -> ['NDB_498', 'NDB_706', 'NDB_889']\n",
      "NRNX1-seg-584 -> ['NDB_491', 'NDB_879', 'NDB_594']\n",
      "NRNX1-seg-585 -> ['NDB_1028', 'NDB_840', 'NDB_903']\n",
      "NRNX1-seg-586 -> ['NDB_784', 'NDB_755', 'NDB_826']\n",
      "NRNX1-seg-587 -> ['NDB_643', 'NDB_683', 'NDB_760']\n",
      "NRNX1-seg-588 -> ['NDB_568', 'NDB_458', 'NDB_466']\n",
      "NRNX1-seg-589 -> ['NDB_868', 'NDB_680', 'NDB_529']\n",
      "NRNX1-seg-590 -> ['NDB_634', 'NDB_332', 'NDB_715']\n",
      "NRNX1-seg-591 -> ['NDB_388', 'NDB_335', 'NDB_1060']\n",
      "NRNX1-seg-592 -> ['NDB_835', 'NDB_473', 'NDB_433']\n",
      "NRNX1-seg-593 -> ['NDB_838', 'NDB_647', 'NDB_994']\n",
      "NRNX1-seg-594 -> ['NDB_862', 'NDB_551', 'NDB_628']\n",
      "NRNX1-seg-595 -> ['NDB_418', 'NDB_836', 'NDB_526']\n",
      "NRNX1-seg-596 -> ['NDB_578', 'NDB_817', 'NDB_699']\n",
      "NRNX1-seg-597 -> ['NDB_815', 'NDB_778', 'NDB_762']\n",
      "NRNX1-seg-598 -> ['NDB_974', 'NDB_487', 'NDB_786']\n",
      "NRNX1-seg-599 -> ['NDB_443', 'NDB_1027', 'NDB_681']\n",
      "NRNX1-seg-600 -> ['NDB_758', 'NDB_631', 'NDB_444']\n",
      "NRNX1-seg-601 -> ['NDB_866', 'NDB_805', 'NDB_747']\n",
      "NRNX1-seg-602 -> ['NDB_539', 'NDB_721', 'NDB_1044']\n",
      "NRNX1-seg-603 -> ['NDB_881', 'NDB_637', 'NDB_711']\n",
      "NRNX1-seg-604 -> ['NDB_459', 'NDB_705', 'NDB_977']\n",
      "NRNX1-seg-605 -> ['NDB_933', 'NDB_519', 'NDB_392']\n",
      "NRNX1-seg-606 -> ['NDB_759', 'NDB_376', 'NDB_725']\n",
      "NRNX1-seg-607 -> ['NDB_456', 'NDB_400', 'NDB_431']\n",
      "NRNX1-seg-608 -> ['NDB_915', 'NDB_511', 'NDB_773']\n",
      "NRNX1-seg-609 -> ['NDB_957', 'NDB_508', 'NDB_737']\n",
      "NRNX1-seg-610 -> ['NDB_405', 'NDB_883', 'NDB_653']\n",
      "NRNX1-seg-611 -> ['NDB_390', 'NDB_427', 'NDB_347']\n",
      "NRNX1-seg-612 -> ['NDB_846', 'NDB_397', 'NDB_641']\n",
      "NRNX1-seg-613 -> ['NDB_687', 'NDB_514', 'NDB_734']\n",
      "NRNX1-seg-614 -> ['NDB_694', 'NDB_751', 'NDB_401']\n",
      "NRNX1-seg-615 -> ['NDB_622', 'NDB_517', 'NDB_824']\n",
      "NRNX1-seg-616 -> ['NDB_801', 'NDB_807', 'NDB_971']\n",
      "NRNX1-seg-617 -> ['NDB_825', 'NDB_768', 'NDB_917']\n",
      "NRNX1-seg-618 -> ['NDB_375', 'NDB_516', 'NDB_716']\n",
      "NRNX1-seg-619 -> ['NDB_417', 'NDB_657', 'NDB_863']\n",
      "NRNX1-seg-620 -> ['NDB_954', 'NDB_771', 'NDB_905']\n",
      "NRNX1-seg-621 -> ['NDB_780', 'NDB_615', 'NDB_482']\n",
      "NRNX1-seg-622 -> ['NDB_630', 'NDB_453', 'NDB_941']\n",
      "NRNX1-seg-623 -> ['NDB_396', 'NDB_828', 'NDB_788']\n",
      "NRNX1-seg-624 -> ['NDB_958', 'NDB_341', 'NDB_372']\n",
      "NRNX1-seg-625 -> ['NDB_712', 'NDB_395', 'NDB_513']\n",
      "NRNX1-seg-626 -> ['NDB_784', 'NDB_777', 'NDB_718']\n",
      "NRNX1-seg-627 -> ['NDB_643', 'NDB_750', 'NDB_961']\n",
      "NRNX1-seg-628 -> ['NDB_568', 'NDB_621', 'NDB_571']\n",
      "NRNX1-seg-629 -> ['NDB_868', 'NDB_489', 'NDB_856']\n",
      "NRNX1-seg-630 -> ['NDB_634', 'NDB_663', 'NDB_616']\n",
      "NRNX1-seg-631 -> ['NDB_388', 'NDB_501', 'NDB_880']\n",
      "NRNX1-seg-632 -> ['NDB_835', 'NDB_813', 'NDB_505']\n",
      "NRNX1-seg-633 -> ['NDB_838', 'NDB_1023', 'NDB_706']\n",
      "NRNX1-seg-634 -> ['NDB_557', 'NDB_779', 'NDB_742']\n",
      "NRNX1-seg-635 -> ['NDB_929', 'NDB_386', 'NDB_925']\n",
      "NRNX1-seg-636 -> ['NDB_865', 'NDB_533', 'NDB_847']\n",
      "NRNX1-seg-637 -> ['NDB_790', 'NDB_464', 'NDB_661']\n",
      "NRNX1-seg-639 -> ['NDB_901', 'NDB_875', 'NDB_448']\n",
      "NRNX1-seg-640 -> ['NDB_403', 'NDB_470', 'NDB_391']\n",
      "NRNX1-seg-641 -> ['NDB_1033', 'NDB_461', 'NDB_1075']\n",
      "NRNX1-seg-642 -> ['NDB_619', 'NDB_506', 'NDB_832']\n",
      "NRNX1-seg-643 -> ['NDB_946', 'NDB_989', 'NDB_385']\n",
      "NRNX1-seg-644 -> ['NDB_1066', 'NDB_665', 'NDB_889']\n",
      "NRNX1-seg-645 -> ['NDB_951', 'NDB_379', 'NDB_611']\n",
      "NRNX1-seg-646 -> ['NDB_735', 'NDB_544', 'NDB_614']\n",
      "NRNX1-seg-647 -> ['NDB_755', 'NDB_795', 'NDB_713']\n",
      "NRNX1-seg-648 -> ['NDB_683', 'NDB_738', 'NDB_965']\n",
      "NRNX1-seg-649 -> ['NDB_458', 'NDB_387', 'NDB_746']\n",
      "NRNX1-seg-650 -> ['NDB_680', 'NDB_465', 'NDB_455']\n",
      "NRNX1-seg-651 -> ['NDB_332', 'NDB_1035', 'NDB_536']\n",
      "NRNX1-seg-652 -> ['NDB_335', 'NDB_921', 'NDB_887']\n",
      "NRNX1-seg-653 -> ['NDB_473', 'NDB_864', 'NDB_896']\n",
      "NRNX1-seg-654 -> ['NDB_647', 'NDB_498', 'NDB_356']\n",
      "NRNX1-seg-655 -> ['NDB_694', 'NDB_879', 'NDB_1053']\n",
      "NRNX1-seg-656 -> ['NDB_622', 'NDB_840', 'NDB_450']\n",
      "NRNX1-seg-657 -> ['NDB_376', 'NDB_690', 'NDB_699']\n",
      "NRNX1-seg-658 -> ['NDB_400', 'NDB_852', 'NDB_762']\n",
      "NRNX1-seg-659 -> ['NDB_511', 'NDB_432', 'NDB_786']\n",
      "NRNX1-seg-660 -> ['NDB_508', 'NDB_1020', 'NDB_681']\n",
      "NRNX1-seg-661 -> ['NDB_883', 'NDB_741', 'NDB_444']\n",
      "NRNX1-seg-662 -> ['NDB_427', 'NDB_507', 'NDB_747']\n",
      "NRNX1-seg-665 -> ['NDB_397', 'NDB_816', 'NDB_1044']\n",
      "NRNX1-seg-666 -> ['NDB_514', 'NDB_660', 'NDB_711']\n",
      "NRNX1-seg-667 -> ['NDB_1014', 'NDB_491', 'NDB_705']\n",
      "NRNX1-seg-668 -> ['NDB_627', 'NDB_1028', 'NDB_519']\n",
      "NRNX1-seg-669 -> ['NDB_759', 'NDB_777', 'NDB_817']\n",
      "NRNX1-seg-670 -> ['NDB_456', 'NDB_750', 'NDB_778']\n",
      "NRNX1-seg-671 -> ['NDB_915', 'NDB_621', 'NDB_487']\n",
      "NRNX1-seg-672 -> ['NDB_957', 'NDB_489', 'NDB_1027']\n",
      "NRNX1-seg-674 -> ['NDB_405', 'NDB_663', 'NDB_631']\n",
      "NRNX1-seg-675 -> ['NDB_390', 'NDB_501', 'NDB_805']\n",
      "NRNX1-seg-676 -> ['NDB_846', 'NDB_813', 'NDB_721']\n",
      "NRNX1-seg-678 -> ['NDB_687', 'NDB_1023', 'NDB_637']\n",
      "NRNX1-seg-679 -> ['NDB_421', 'NDB_779', 'NDB_372']\n",
      "NRNX1-seg-681 -> ['NDB_472', 'NDB_386', 'NDB_513']\n",
      "NRNX1-seg-682 -> ['NDB_578', 'NDB_725', 'NDB_807']\n",
      "NRNX1-seg-683 -> ['NDB_815', 'NDB_431', 'NDB_768']\n",
      "NRNX1-seg-684 -> ['NDB_974', 'NDB_773', 'NDB_516']\n",
      "NRNX1-seg-685 -> ['NDB_443', 'NDB_737', 'NDB_657']\n",
      "NRNX1-seg-686 -> ['NDB_758', 'NDB_653', 'NDB_771']\n",
      "NRNX1-seg-687 -> ['NDB_866', 'NDB_347', 'NDB_615']\n",
      "NRNX1-seg-688 -> ['NDB_539', 'NDB_641', 'NDB_453']\n",
      "NRNX1-seg-689 -> ['NDB_881', 'NDB_734', 'NDB_828']\n",
      "NRNX1-seg-690 -> ['NDB_862', 'NDB_751', 'NDB_958']\n",
      "NRNX1-seg-691 -> ['NDB_418', 'NDB_517', 'NDB_712']\n",
      "NRNX1-seg-692 -> ['NDB_652', 'NDB_710', 'NDB_936']\n",
      "NRNX1-seg-693 -> ['NDB_844', 'NDB_554', 'NDB_804']\n",
      "NRNX1-seg-694 -> ['NDB_409', 'NDB_812', 'NDB_858']\n",
      "NRNX1-seg-695 -> ['NDB_754', 'NDB_698', 'NDB_438']\n",
      "NRNX1-seg-696 -> ['NDB_532', 'NDB_398', 'NDB_789']\n",
      "NRNX1-seg-697 -> ['NDB_730', 'NDB_752', 'NDB_1047']\n",
      "NRNX1-seg-698 -> ['NDB_853', 'NDB_899', 'NDB_618']\n",
      "NRNX1-seg-699 -> ['NDB_445', 'NDB_935', 'NDB_819']\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "check_readout_dict = pickle.load(open(os.path.join(library_folder, 'readout_usage.pkl'), 'rb'))\n",
    "\n",
    "reg_readout_dic, reg2readout_check = check._check_region_to_readouts(full_records, check_readout_dict, \n",
    "                                                                     total_readout_num=num_on_bits)\n",
    "print(reg2readout_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3e64afd0",
   "metadata": {},
   "source": [
    "### 4.1.4 check readout usage in the library"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "id": "1647e94f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- extracting probes from >region:NRNX1-seg-1, region:NRNX1-seg-2, region:NRNX1-seg-3, region:NRNX1-seg-4, region:NRNX1-seg-5, region:NRNX1-seg-6, region:NRNX1-seg-7, region:NRNX1-seg-9, region:NRNX1-seg-10, region:NRNX1-seg-13, region:NRNX1-seg-14, region:NRNX1-seg-15, region:NRNX1-seg-16, region:NRNX1-seg-17, region:NRNX1-seg-18, region:NRNX1-seg-19, region:NRNX1-seg-20, region:NRNX1-seg-21, region:NRNX1-seg-22, region:NRNX1-seg-23, region:NRNX1-seg-24, region:NRNX1-seg-25, region:NRNX1-seg-26, region:NRNX1-seg-27, region:NRNX1-seg-28, region:NRNX1-seg-30, region:NRNX1-seg-31, region:NRNX1-seg-32, region:NRNX1-seg-33, region:NRNX1-seg-34, region:NRNX1-seg-35, region:NRNX1-seg-36, region:NRNX1-seg-37, region:NRNX1-seg-38, region:NRNX1-seg-39, region:NRNX1-seg-40, region:NRNX1-seg-41, region:NRNX1-seg-42, region:NRNX1-seg-43, region:NRNX1-seg-44, region:NRNX1-seg-45, region:NRNX1-seg-46, region:NRNX1-seg-47, region:NRNX1-seg-48, region:NRNX1-seg-49, region:NRNX1-seg-50, region:NRNX1-seg-51, region:NRNX1-seg-52, region:NRNX1-seg-53, region:NRNX1-seg-54, region:NRNX1-seg-55, region:NRNX1-seg-56, region:NRNX1-seg-57, region:NRNX1-seg-58, region:NRNX1-seg-59, region:NRNX1-seg-60, region:NRNX1-seg-62, region:NRNX1-seg-63, region:NRNX1-seg-64, region:NRNX1-seg-65, region:NRNX1-seg-66, region:NRNX1-seg-67, region:NRNX1-seg-68, region:NRNX1-seg-69, region:NRNX1-seg-70, region:NRNX1-seg-71, region:NRNX1-seg-72, region:NRNX1-seg-73, region:NRNX1-seg-74, region:NRNX1-seg-75, region:NRNX1-seg-76, region:NRNX1-seg-77, region:NRNX1-seg-79, region:NRNX1-seg-80, region:NRNX1-seg-81, region:NRNX1-seg-82, region:NRNX1-seg-83, region:NRNX1-seg-84, region:NRNX1-seg-85, region:NRNX1-seg-86, region:NRNX1-seg-87, region:NRNX1-seg-88, region:NRNX1-seg-89, region:NRNX1-seg-90, region:NRNX1-seg-91, region:NRNX1-seg-92, region:NRNX1-seg-93, region:NRNX1-seg-94, region:NRNX1-seg-95, region:NRNX1-seg-96, region:NRNX1-seg-97, region:NRNX1-seg-98, region:NRNX1-seg-99, region:NRNX1-seg-100, region:NRNX1-seg-101, region:NRNX1-seg-102, region:NRNX1-seg-103, region:NRNX1-seg-104, region:NRNX1-seg-105, region:NRNX1-seg-106, region:NRNX1-seg-107, region:NRNX1-seg-108, region:NRNX1-seg-109, region:NRNX1-seg-110, region:NRNX1-seg-111, region:NRNX1-seg-112, region:NRNX1-seg-113, region:NRNX1-seg-114, region:NRNX1-seg-115, region:NRNX1-seg-117, region:NRNX1-seg-118, region:NRNX1-seg-119, region:NRNX1-seg-120, region:NRNX1-seg-121, region:NRNX1-seg-122, region:NRNX1-seg-123, region:NRNX1-seg-124, region:NRNX1-seg-125, region:NRNX1-seg-126, region:NRNX1-seg-127, region:NRNX1-seg-128, region:NRNX1-seg-129, region:NRNX1-seg-130, region:NRNX1-seg-131, region:NRNX1-seg-133, region:NRNX1-seg-134, region:NRNX1-seg-135, region:NRNX1-seg-136, region:NRNX1-seg-137, region:NRNX1-seg-138, region:NRNX1-seg-139, region:NRNX1-seg-140, region:NRNX1-seg-141, region:NRNX1-seg-142, region:NRNX1-seg-143, region:NRNX1-seg-144, region:NRNX1-seg-145, region:NRNX1-seg-146, region:NRNX1-seg-147, region:NRNX1-seg-148, region:NRNX1-seg-150, region:NRNX1-seg-151, region:NRNX1-seg-152, region:NRNX1-seg-154, region:NRNX1-seg-155, region:NRNX1-seg-156, region:NRNX1-seg-157, region:NRNX1-seg-158, region:NRNX1-seg-159, region:NRNX1-seg-160, region:NRNX1-seg-161, region:NRNX1-seg-162, region:NRNX1-seg-163, region:NRNX1-seg-164, region:NRNX1-seg-165, region:NRNX1-seg-166, region:NRNX1-seg-169, region:NRNX1-seg-170, region:NRNX1-seg-171, region:NRNX1-seg-172, region:NRNX1-seg-173, region:NRNX1-seg-174, region:NRNX1-seg-175, region:NRNX1-seg-176, region:NRNX1-seg-177, region:NRNX1-seg-178, region:NRNX1-seg-179, region:NRNX1-seg-180, region:NRNX1-seg-181, region:NRNX1-seg-182, region:NRNX1-seg-183, region:NRNX1-seg-184, region:NRNX1-seg-185, region:NRNX1-seg-186, region:NRNX1-seg-187, region:NRNX1-seg-188, region:NRNX1-seg-189, region:NRNX1-seg-190, region:NRNX1-seg-191, region:NRNX1-seg-192, region:NRNX1-seg-193, region:NRNX1-seg-194, region:NRNX1-seg-195, region:NRNX1-seg-196, region:NRNX1-seg-197, region:NRNX1-seg-198, region:NRNX1-seg-199, region:NRNX1-seg-200, region:NRNX1-seg-201, region:NRNX1-seg-202, region:NRNX1-seg-203, region:NRNX1-seg-204, region:NRNX1-seg-205, region:NRNX1-seg-206, region:NRNX1-seg-207, region:NRNX1-seg-208, region:NRNX1-seg-209, region:NRNX1-seg-210, region:NRNX1-seg-211, region:NRNX1-seg-212, region:NRNX1-seg-213, region:NRNX1-seg-214, region:NRNX1-seg-215, region:NRNX1-seg-216, region:NRNX1-seg-217, region:NRNX1-seg-218, region:NRNX1-seg-219, region:NRNX1-seg-220, region:NRNX1-seg-221, region:NRNX1-seg-222, region:NRNX1-seg-223, region:NRNX1-seg-224, region:NRNX1-seg-225, region:NRNX1-seg-226, region:NRNX1-seg-227, region:NRNX1-seg-228, region:NRNX1-seg-229, region:NRNX1-seg-230, region:NRNX1-seg-231, region:NRNX1-seg-232, region:NRNX1-seg-233, region:NRNX1-seg-234, region:NRNX1-seg-235, region:NRNX1-seg-236, region:NRNX1-seg-238, region:NRNX1-seg-239, region:NRNX1-seg-240, region:NRNX1-seg-241, region:NRNX1-seg-242, region:NRNX1-seg-244, region:NRNX1-seg-245, region:NRNX1-seg-246, region:NRNX1-seg-248, region:NRNX1-seg-249, region:NRNX1-seg-251, region:NRNX1-seg-252, region:NRNX1-seg-253, region:NRNX1-seg-254, region:NRNX1-seg-255, region:NRNX1-seg-256, region:NRNX1-seg-257, region:NRNX1-seg-258, region:NRNX1-seg-259, region:NRNX1-seg-260, region:NRNX1-seg-262, region:NRNX1-seg-263, region:NRNX1-seg-264, region:NRNX1-seg-265, region:NRNX1-seg-266, region:NRNX1-seg-267, region:NRNX1-seg-268, region:NRNX1-seg-269, region:NRNX1-seg-270, region:NRNX1-seg-271, region:NRNX1-seg-272, region:NRNX1-seg-273, region:NRNX1-seg-274, region:NRNX1-seg-275, region:NRNX1-seg-276, region:NRNX1-seg-277, region:NRNX1-seg-278, region:NRNX1-seg-279, region:NRNX1-seg-280, region:NRNX1-seg-281, region:NRNX1-seg-282, region:NRNX1-seg-283, region:NRNX1-seg-284, region:NRNX1-seg-285, region:NRNX1-seg-286, region:NRNX1-seg-287, region:NRNX1-seg-288, region:NRNX1-seg-289, region:NRNX1-seg-290, region:NRNX1-seg-291, region:NRNX1-seg-292, region:NRNX1-seg-293, region:NRNX1-seg-294, region:NRNX1-seg-295, region:NRNX1-seg-298, region:NRNX1-seg-299, region:NRNX1-seg-300, region:NRNX1-seg-301, region:NRNX1-seg-302, region:NRNX1-seg-303, region:NRNX1-seg-304, region:NRNX1-seg-305, region:NRNX1-seg-306, region:NRNX1-seg-307, region:NRNX1-seg-308, region:NRNX1-seg-311, region:NRNX1-seg-312, region:NRNX1-seg-313, region:NRNX1-seg-314, region:NRNX1-seg-315, region:NRNX1-seg-316, region:NRNX1-seg-317, region:NRNX1-seg-318, region:NRNX1-seg-319, region:NRNX1-seg-320, region:NRNX1-seg-321, region:NRNX1-seg-322, region:NRNX1-seg-323, region:NRNX1-seg-324, region:NRNX1-seg-325, region:NRNX1-seg-326, region:NRNX1-seg-327, region:NRNX1-seg-328, region:NRNX1-seg-329, region:NRNX1-seg-330, region:NRNX1-seg-331, region:NRNX1-seg-332, region:NRNX1-seg-333, region:NRNX1-seg-334, region:NRNX1-seg-335, region:NRNX1-seg-336, region:NRNX1-seg-337, region:NRNX1-seg-338, region:NRNX1-seg-339, region:NRNX1-seg-340, region:NRNX1-seg-341, region:NRNX1-seg-342, region:NRNX1-seg-343, region:NRNX1-seg-344, region:NRNX1-seg-345, region:NRNX1-seg-346, region:NRNX1-seg-347, region:NRNX1-seg-348, region:NRNX1-seg-349, region:NRNX1-seg-350, region:NRNX1-seg-351, region:NRNX1-seg-352, region:NRNX1-seg-353, region:NRNX1-seg-354, region:NRNX1-seg-355, region:NRNX1-seg-356, region:NRNX1-seg-357, region:NRNX1-seg-358, region:NRNX1-seg-359, region:NRNX1-seg-360, region:NRNX1-seg-361, region:NRNX1-seg-362, region:NRNX1-seg-363, region:NRNX1-seg-364, region:NRNX1-seg-365, region:NRNX1-seg-366, region:NRNX1-seg-367, region:NRNX1-seg-368, region:NRNX1-seg-369, region:NRNX1-seg-370, region:NRNX1-seg-371, region:NRNX1-seg-372, region:NRNX1-seg-373, region:NRNX1-seg-374, region:NRNX1-seg-375, region:NRNX1-seg-376, region:NRNX1-seg-379, region:NRNX1-seg-380, region:NRNX1-seg-381, region:NRNX1-seg-382, region:NRNX1-seg-383, region:NRNX1-seg-384, region:NRNX1-seg-385, region:NRNX1-seg-386, region:NRNX1-seg-387, region:NRNX1-seg-388, region:NRNX1-seg-389, region:NRNX1-seg-390, region:NRNX1-seg-391, region:NRNX1-seg-392, region:NRNX1-seg-393, region:NRNX1-seg-394, region:NRNX1-seg-395, region:NRNX1-seg-396, region:NRNX1-seg-397, region:NRNX1-seg-398, region:NRNX1-seg-399, region:NRNX1-seg-400, region:NRNX1-seg-401, region:NRNX1-seg-402, region:NRNX1-seg-403, region:NRNX1-seg-404, region:NRNX1-seg-405, region:NRNX1-seg-406, region:NRNX1-seg-407, region:NRNX1-seg-408, region:NRNX1-seg-409, region:NRNX1-seg-410, "
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "region:NRNX1-seg-411, region:NRNX1-seg-412, region:NRNX1-seg-413, region:NRNX1-seg-414, region:NRNX1-seg-415, region:NRNX1-seg-416, region:NRNX1-seg-417, region:NRNX1-seg-418, region:NRNX1-seg-419, region:NRNX1-seg-420, region:NRNX1-seg-421, region:NRNX1-seg-422, region:NRNX1-seg-423, region:NRNX1-seg-424, region:NRNX1-seg-425, region:NRNX1-seg-426, region:NRNX1-seg-427, region:NRNX1-seg-428, region:NRNX1-seg-429, region:NRNX1-seg-430, region:NRNX1-seg-431, region:NRNX1-seg-432, region:NRNX1-seg-433, region:NRNX1-seg-434, region:NRNX1-seg-435, region:NRNX1-seg-436, region:NRNX1-seg-437, region:NRNX1-seg-438, region:NRNX1-seg-439, region:NRNX1-seg-440, region:NRNX1-seg-441, region:NRNX1-seg-442, region:NRNX1-seg-443, region:NRNX1-seg-444, region:NRNX1-seg-445, region:NRNX1-seg-446, region:NRNX1-seg-447, region:NRNX1-seg-448, region:NRNX1-seg-449, region:NRNX1-seg-450, region:NRNX1-seg-452, region:NRNX1-seg-453, region:NRNX1-seg-454, region:NRNX1-seg-455, region:NRNX1-seg-456, region:NRNX1-seg-457, region:NRNX1-seg-458, region:NRNX1-seg-459, region:NRNX1-seg-460, region:NRNX1-seg-461, region:NRNX1-seg-462, region:NRNX1-seg-463, region:NRNX1-seg-464, region:NRNX1-seg-465, region:NRNX1-seg-466, region:NRNX1-seg-467, region:NRNX1-seg-468, region:NRNX1-seg-469, region:NRNX1-seg-470, region:NRNX1-seg-471, region:NRNX1-seg-472, region:NRNX1-seg-474, region:NRNX1-seg-475, region:NRNX1-seg-476, region:NRNX1-seg-477, region:NRNX1-seg-478, region:NRNX1-seg-479, region:NRNX1-seg-480, region:NRNX1-seg-481, region:NRNX1-seg-482, region:NRNX1-seg-483, region:NRNX1-seg-484, region:NRNX1-seg-485, region:NRNX1-seg-486, region:NRNX1-seg-487, region:NRNX1-seg-488, region:NRNX1-seg-489, region:NRNX1-seg-490, region:NRNX1-seg-491, region:NRNX1-seg-492, region:NRNX1-seg-493, region:NRNX1-seg-494, region:NRNX1-seg-495, region:NRNX1-seg-496, region:NRNX1-seg-497, region:NRNX1-seg-498, region:NRNX1-seg-499, region:NRNX1-seg-500, region:NRNX1-seg-501, region:NRNX1-seg-502, region:NRNX1-seg-503, region:NRNX1-seg-504, region:NRNX1-seg-505, region:NRNX1-seg-506, region:NRNX1-seg-507, region:NRNX1-seg-508, region:NRNX1-seg-509, region:NRNX1-seg-510, region:NRNX1-seg-511, region:NRNX1-seg-512, region:NRNX1-seg-513, region:NRNX1-seg-514, region:NRNX1-seg-515, region:NRNX1-seg-516, region:NRNX1-seg-517, region:NRNX1-seg-518, region:NRNX1-seg-519, region:NRNX1-seg-520, region:NRNX1-seg-521, region:NRNX1-seg-522, region:NRNX1-seg-523, region:NRNX1-seg-524, region:NRNX1-seg-525, region:NRNX1-seg-526, region:NRNX1-seg-527, region:NRNX1-seg-528, region:NRNX1-seg-529, region:NRNX1-seg-530, region:NRNX1-seg-531, region:NRNX1-seg-532, region:NRNX1-seg-533, region:NRNX1-seg-534, region:NRNX1-seg-535, region:NRNX1-seg-536, region:NRNX1-seg-537, region:NRNX1-seg-538, region:NRNX1-seg-539, region:NRNX1-seg-540, region:NRNX1-seg-541, region:NRNX1-seg-542, region:NRNX1-seg-543, region:NRNX1-seg-544, region:NRNX1-seg-545, region:NRNX1-seg-546, region:NRNX1-seg-547, region:NRNX1-seg-548, region:NRNX1-seg-549, region:NRNX1-seg-550, region:NRNX1-seg-551, region:NRNX1-seg-552, region:NRNX1-seg-553, region:NRNX1-seg-554, region:NRNX1-seg-555, region:NRNX1-seg-556, region:NRNX1-seg-557, region:NRNX1-seg-558, region:NRNX1-seg-559, region:NRNX1-seg-560, region:NRNX1-seg-561, region:NRNX1-seg-562, region:NRNX1-seg-563, region:NRNX1-seg-564, region:NRNX1-seg-565, region:NRNX1-seg-566, region:NRNX1-seg-567, region:NRNX1-seg-568, region:NRNX1-seg-569, region:NRNX1-seg-570, region:NRNX1-seg-571, region:NRNX1-seg-572, region:NRNX1-seg-573, region:NRNX1-seg-574, region:NRNX1-seg-575, region:NRNX1-seg-576, region:NRNX1-seg-577, region:NRNX1-seg-578, region:NRNX1-seg-579, region:NRNX1-seg-580, region:NRNX1-seg-581, region:NRNX1-seg-582, region:NRNX1-seg-583, region:NRNX1-seg-584, region:NRNX1-seg-585, region:NRNX1-seg-586, region:NRNX1-seg-587, region:NRNX1-seg-588, region:NRNX1-seg-589, region:NRNX1-seg-590, region:NRNX1-seg-591, region:NRNX1-seg-592, region:NRNX1-seg-593, region:NRNX1-seg-594, region:NRNX1-seg-595, region:NRNX1-seg-596, region:NRNX1-seg-597, region:NRNX1-seg-598, region:NRNX1-seg-599, region:NRNX1-seg-600, region:NRNX1-seg-601, region:NRNX1-seg-602, region:NRNX1-seg-603, region:NRNX1-seg-604, region:NRNX1-seg-605, region:NRNX1-seg-606, region:NRNX1-seg-607, region:NRNX1-seg-608, region:NRNX1-seg-609, region:NRNX1-seg-610, region:NRNX1-seg-611, region:NRNX1-seg-612, region:NRNX1-seg-613, region:NRNX1-seg-614, region:NRNX1-seg-615, region:NRNX1-seg-616, region:NRNX1-seg-617, region:NRNX1-seg-618, region:NRNX1-seg-619, region:NRNX1-seg-620, region:NRNX1-seg-621, region:NRNX1-seg-622, region:NRNX1-seg-623, region:NRNX1-seg-624, region:NRNX1-seg-625, region:NRNX1-seg-626, region:NRNX1-seg-627, region:NRNX1-seg-628, region:NRNX1-seg-629, region:NRNX1-seg-630, region:NRNX1-seg-631, region:NRNX1-seg-632, region:NRNX1-seg-633, region:NRNX1-seg-634, region:NRNX1-seg-635, region:NRNX1-seg-636, region:NRNX1-seg-637, region:NRNX1-seg-639, region:NRNX1-seg-640, region:NRNX1-seg-641, region:NRNX1-seg-642, region:NRNX1-seg-643, region:NRNX1-seg-644, region:NRNX1-seg-645, region:NRNX1-seg-646, region:NRNX1-seg-647, region:NRNX1-seg-648, region:NRNX1-seg-649, region:NRNX1-seg-650, region:NRNX1-seg-651, region:NRNX1-seg-652, region:NRNX1-seg-653, region:NRNX1-seg-654, region:NRNX1-seg-655, region:NRNX1-seg-656, region:NRNX1-seg-657, region:NRNX1-seg-658, region:NRNX1-seg-659, region:NRNX1-seg-660, region:NRNX1-seg-661, region:NRNX1-seg-662, region:NRNX1-seg-665, region:NRNX1-seg-666, region:NRNX1-seg-667, region:NRNX1-seg-668, region:NRNX1-seg-669, region:NRNX1-seg-670, region:NRNX1-seg-671, region:NRNX1-seg-672, region:NRNX1-seg-674, region:NRNX1-seg-675, region:NRNX1-seg-676, region:NRNX1-seg-678, region:NRNX1-seg-679, region:NRNX1-seg-681, region:NRNX1-seg-682, region:NRNX1-seg-683, region:NRNX1-seg-684, region:NRNX1-seg-685, region:NRNX1-seg-686, region:NRNX1-seg-687, region:NRNX1-seg-688, region:NRNX1-seg-689, region:NRNX1-seg-690, region:NRNX1-seg-691, region:NRNX1-seg-692, region:NRNX1-seg-693, region:NRNX1-seg-694, region:NRNX1-seg-695, region:NRNX1-seg-696, region:NRNX1-seg-697, region:NRNX1-seg-698, region:NRNX1-seg-699, {'NDB_376': ['NRNX1-seg-1', 'NRNX1-seg-1', 'NRNX1-seg-1', 'NRNX1-seg-1', 'NRNX1-seg-97', 'NRNX1-seg-97', 'NRNX1-seg-97', 'NRNX1-seg-97', 'NRNX1-seg-161', 'NRNX1-seg-161', 'NRNX1-seg-161', 'NRNX1-seg-161', 'NRNX1-seg-268', 'NRNX1-seg-268', 'NRNX1-seg-268', 'NRNX1-seg-268', 'NRNX1-seg-332', 'NRNX1-seg-332', 'NRNX1-seg-332', 'NRNX1-seg-332', 'NRNX1-seg-424', 'NRNX1-seg-424', 'NRNX1-seg-424', 'NRNX1-seg-424', 'NRNX1-seg-526', 'NRNX1-seg-526', 'NRNX1-seg-526', 'NRNX1-seg-526', 'NRNX1-seg-606', 'NRNX1-seg-606', 'NRNX1-seg-606', 'NRNX1-seg-606', 'NRNX1-seg-657', 'NRNX1-seg-657', 'NRNX1-seg-657', 'NRNX1-seg-657'], 'NDB_533': ['NRNX1-seg-1', 'NRNX1-seg-1', 'NRNX1-seg-1', 'NRNX1-seg-1', 'NRNX1-seg-55', 'NRNX1-seg-55', 'NRNX1-seg-55', 'NRNX1-seg-55', 'NRNX1-seg-173', 'NRNX1-seg-173', 'NRNX1-seg-173', 'NRNX1-seg-173', 'NRNX1-seg-233', 'NRNX1-seg-233', 'NRNX1-seg-233', 'NRNX1-seg-233', 'NRNX1-seg-278', 'NRNX1-seg-278', 'NRNX1-seg-278', 'NRNX1-seg-278', 'NRNX1-seg-394', 'NRNX1-seg-394', 'NRNX1-seg-394', 'NRNX1-seg-394', 'NRNX1-seg-455', 'NRNX1-seg-455', 'NRNX1-seg-455', 'NRNX1-seg-455', 'NRNX1-seg-536', 'NRNX1-seg-536', 'NRNX1-seg-536', 'NRNX1-seg-536', 'NRNX1-seg-636', 'NRNX1-seg-636', 'NRNX1-seg-636', 'NRNX1-seg-636'], 'NDB_936': ['NRNX1-seg-1', 'NRNX1-seg-1', 'NRNX1-seg-1', 'NRNX1-seg-1', 'NRNX1-seg-128', 'NRNX1-seg-128', 'NRNX1-seg-128', 'NRNX1-seg-128', 'NRNX1-seg-288', 'NRNX1-seg-288', 'NRNX1-seg-288', 'NRNX1-seg-288', 'NRNX1-seg-352', 'NRNX1-seg-352', 'NRNX1-seg-352', 'NRNX1-seg-352', 'NRNX1-seg-404', 'NRNX1-seg-404', 'NRNX1-seg-404', 'NRNX1-seg-404', 'NRNX1-seg-486', 'NRNX1-seg-486', 'NRNX1-seg-486', 'NRNX1-seg-486', 'NRNX1-seg-546', 'NRNX1-seg-546', 'NRNX1-seg-546', 'NRNX1-seg-546', 'NRNX1-seg-692', 'NRNX1-seg-692', 'NRNX1-seg-692', 'NRNX1-seg-692'], 'NDB_400': ['NRNX1-seg-2', 'NRNX1-seg-2', 'NRNX1-seg-2', 'NRNX1-seg-2', 'NRNX1-seg-98', 'NRNX1-seg-98', 'NRNX1-seg-98', 'NRNX1-seg-98', 'NRNX1-seg-162', 'NRNX1-seg-162', 'NRNX1-seg-162', 'NRNX1-seg-162', 'NRNX1-seg-269', 'NRNX1-seg-269', 'NRNX1-seg-269', 'NRNX1-seg-269', 'NRNX1-seg-333', 'NRNX1-seg-333', 'NRNX1-seg-333', 'NRNX1-seg-333', 'NRNX1-seg-425', 'NRNX1-seg-425', 'NRNX1-seg-425', 'NRNX1-seg-425', 'NRNX1-seg-527', 'NRNX1-seg-527', 'NRNX1-seg-527', 'NRNX1-seg-527', 'NRNX1-seg-607', 'NRNX1-seg-607', 'NRNX1-seg-607', 'NRNX1-seg-607', 'NRNX1-seg-658', 'NRNX1-seg-658', 'NRNX1-seg-658', 'NRNX1-seg-658'], 'NDB_464': ['NRNX1-seg-2', 'NRNX1-seg-2', 'NRNX1-seg-2', 'NRNX1-seg-2', 'NRNX1-seg-56', 'NRNX1-seg-56', 'NRNX1-seg-56', 'NRNX1-seg-56', 'NRNX1-seg-174', 'NRNX1-seg-174', 'NRNX1-seg-174', 'NRNX1-seg-174', 'NRNX1-seg-234', 'NRNX1-seg-234', 'NRNX1-seg-234', 'NRNX1-seg-234', 'NRNX1-seg-279', 'NRNX1-seg-279', 'NRNX1-seg-279', 'NRNX1-seg-279', 'NRNX1-seg-395', 'NRNX1-seg-395', 'NRNX1-seg-395', 'NRNX1-seg-395', 'NRNX1-seg-456', 'NRNX1-seg-456', 'NRNX1-seg-456', 'NRNX1-seg-456', 'NRNX1-seg-537', 'NRNX1-seg-537', 'NRNX1-seg-537', 'NRNX1-seg-537', 'NRNX1-seg-637', 'NRNX1-seg-637', 'NRNX1-seg-637', 'NRNX1-seg-637'], 'NDB_804': ['NRNX1-seg-2', 'NRNX1-seg-2', 'NRNX1-seg-2', 'NRNX1-seg-2', 'NRNX1-seg-129', 'NRNX1-seg-129', 'NRNX1-seg-129', 'NRNX1-seg-129', 'NRNX1-seg-289', 'NRNX1-seg-289', 'NRNX1-seg-289', 'NRNX1-seg-289', 'NRNX1-seg-353', 'NRNX1-seg-353', 'NRNX1-seg-353', 'NRNX1-seg-353', 'NRNX1-seg-405', 'NRNX1-seg-405', 'NRNX1-seg-405', 'NRNX1-seg-405', 'NRNX1-seg-487', 'NRNX1-seg-487', 'NRNX1-seg-487', 'NRNX1-seg-487', 'NRNX1-seg-547', 'NRNX1-seg-547', 'NRNX1-seg-547', 'NRNX1-seg-547', 'NRNX1-seg-693', 'NRNX1-seg-693', 'NRNX1-seg-693', 'NRNX1-seg-693'], 'NDB_511': ['NRNX1-seg-3', 'NRNX1-seg-3', 'NRNX1-seg-3', 'NRNX1-seg-3', 'NRNX1-seg-99', 'NRNX1-seg-99', 'NRNX1-seg-99', 'NRNX1-seg-99', 'NRNX1-seg-163', 'NRNX1-seg-163', 'NRNX1-seg-163', 'NRNX1-seg-163', 'NRNX1-seg-270', 'NRNX1-seg-270', 'NRNX1-seg-270', 'NRNX1-seg-270', 'NRNX1-seg-334', 'NRNX1-seg-334', 'NRNX1-seg-334', 'NRNX1-seg-334', 'NRNX1-seg-426', 'NRNX1-seg-426', 'NRNX1-seg-426', 'NRNX1-seg-426', 'NRNX1-seg-528', 'NRNX1-seg-528', 'NRNX1-seg-528', 'NRNX1-seg-528', 'NRNX1-seg-608', 'NRNX1-seg-608', 'NRNX1-seg-608', 'NRNX1-seg-608', 'NRNX1-seg-659', 'NRNX1-seg-659', 'NRNX1-seg-659', 'NRNX1-seg-659'], 'NDB_875': ['NRNX1-seg-3', 'NRNX1-seg-3', 'NRNX1-seg-3', 'NRNX1-seg-3', 'NRNX1-seg-57', 'NRNX1-seg-57', 'NRNX1-seg-57', 'NRNX1-seg-57', 'NRNX1-seg-175', 'NRNX1-seg-175', 'NRNX1-seg-175', 'NRNX1-seg-175', 'NRNX1-seg-235', 'NRNX1-seg-235', 'NRNX1-seg-235', 'NRNX1-seg-235', 'NRNX1-seg-280', 'NRNX1-seg-280', 'NRNX1-seg-280', 'NRNX1-seg-280', 'NRNX1-seg-396', 'NRNX1-seg-396', 'NRNX1-seg-396', 'NRNX1-seg-396', 'NRNX1-seg-457', 'NRNX1-seg-457', 'NRNX1-seg-457', 'NRNX1-seg-457', 'NRNX1-seg-538', 'NRNX1-seg-538', 'NRNX1-seg-538', 'NRNX1-seg-538', 'NRNX1-seg-639', 'NRNX1-seg-639', 'NRNX1-seg-639', 'NRNX1-seg-639'], 'NDB_858': ['NRNX1-seg-3', 'NRNX1-seg-3', 'NRNX1-seg-3', 'NRNX1-seg-3', 'NRNX1-seg-130', 'NRNX1-seg-130', 'NRNX1-seg-130', 'NRNX1-seg-130', 'NRNX1-seg-290', 'NRNX1-seg-290', 'NRNX1-seg-290', 'NRNX1-seg-290', 'NRNX1-seg-354', 'NRNX1-seg-354', 'NRNX1-seg-354', 'NRNX1-seg-354', 'NRNX1-seg-406', 'NRNX1-seg-406', 'NRNX1-seg-406', 'NRNX1-seg-406', 'NRNX1-seg-488', 'NRNX1-seg-488', 'NRNX1-seg-488', 'NRNX1-seg-488', 'NRNX1-seg-548', 'NRNX1-seg-548', 'NRNX1-seg-548', 'NRNX1-seg-548', 'NRNX1-seg-694', 'NRNX1-seg-694', 'NRNX1-seg-694', 'NRNX1-seg-694'], 'NDB_508': ['NRNX1-seg-4', 'NRNX1-seg-4', 'NRNX1-seg-4', 'NRNX1-seg-4', 'NRNX1-seg-100', 'NRNX1-seg-100', 'NRNX1-seg-100', 'NRNX1-seg-100', 'NRNX1-seg-164', 'NRNX1-seg-164', 'NRNX1-seg-164', 'NRNX1-seg-164', 'NRNX1-seg-271', 'NRNX1-seg-271', 'NRNX1-seg-271', 'NRNX1-seg-271', 'NRNX1-seg-335', 'NRNX1-seg-335', 'NRNX1-seg-335', 'NRNX1-seg-335', 'NRNX1-seg-427', 'NRNX1-seg-427', 'NRNX1-seg-427', 'NRNX1-seg-427', 'NRNX1-seg-529', 'NRNX1-seg-529', 'NRNX1-seg-529', 'NRNX1-seg-529', 'NRNX1-seg-609', 'NRNX1-seg-609', 'NRNX1-seg-609', 'NRNX1-seg-609', 'NRNX1-seg-660', 'NRNX1-seg-660', 'NRNX1-seg-660', 'NRNX1-seg-660'], 'NDB_470': ['NRNX1-seg-4', 'NRNX1-seg-4', 'NRNX1-seg-4', 'NRNX1-seg-4', 'NRNX1-seg-58', 'NRNX1-seg-58', 'NRNX1-seg-58', 'NRNX1-seg-58', 'NRNX1-seg-176', 'NRNX1-seg-176', 'NRNX1-seg-176', 'NRNX1-seg-176', 'NRNX1-seg-236', 'NRNX1-seg-236', 'NRNX1-seg-236', 'NRNX1-seg-236', 'NRNX1-seg-281', 'NRNX1-seg-281', 'NRNX1-seg-281', 'NRNX1-seg-281', 'NRNX1-seg-397', 'NRNX1-seg-397', 'NRNX1-seg-397', 'NRNX1-seg-397', 'NRNX1-seg-458', 'NRNX1-seg-458', 'NRNX1-seg-458', 'NRNX1-seg-458', 'NRNX1-seg-539', 'NRNX1-seg-539', 'NRNX1-seg-539', 'NRNX1-seg-539', 'NRNX1-seg-640', 'NRNX1-seg-640', 'NRNX1-seg-640', 'NRNX1-seg-640'], 'NDB_438': ['NRNX1-seg-4', 'NRNX1-seg-4', 'NRNX1-seg-4', 'NRNX1-seg-4', 'NRNX1-seg-131', 'NRNX1-seg-131', 'NRNX1-seg-131', 'NRNX1-seg-131', 'NRNX1-seg-291', 'NRNX1-seg-291', 'NRNX1-seg-291', 'NRNX1-seg-291', 'NRNX1-seg-355', 'NRNX1-seg-355', 'NRNX1-seg-355', 'NRNX1-seg-355', 'NRNX1-seg-407', 'NRNX1-seg-407', 'NRNX1-seg-407', 'NRNX1-seg-407', 'NRNX1-seg-489', 'NRNX1-seg-489', 'NRNX1-seg-489', 'NRNX1-seg-489', 'NRNX1-seg-549', 'NRNX1-seg-549', 'NRNX1-seg-549', 'NRNX1-seg-549', 'NRNX1-seg-695', 'NRNX1-seg-695', 'NRNX1-seg-695', 'NRNX1-seg-695'], 'NDB_883': ['NRNX1-seg-5', 'NRNX1-seg-5', 'NRNX1-seg-5', 'NRNX1-seg-5', 'NRNX1-seg-101', 'NRNX1-seg-101', 'NRNX1-seg-101', 'NRNX1-seg-101', 'NRNX1-seg-165', 'NRNX1-seg-165', 'NRNX1-seg-165', 'NRNX1-seg-165', 'NRNX1-seg-272', 'NRNX1-seg-272', 'NRNX1-seg-272', 'NRNX1-seg-272', 'NRNX1-seg-336', 'NRNX1-seg-336', 'NRNX1-seg-336', 'NRNX1-seg-336', 'NRNX1-seg-428', 'NRNX1-seg-428', 'NRNX1-seg-428', 'NRNX1-seg-428', 'NRNX1-seg-530', 'NRNX1-seg-530', 'NRNX1-seg-530', 'NRNX1-seg-530', 'NRNX1-seg-610', 'NRNX1-seg-610', 'NRNX1-seg-610', 'NRNX1-seg-610', 'NRNX1-seg-661', 'NRNX1-seg-661', 'NRNX1-seg-661', 'NRNX1-seg-661'], 'NDB_461': ['NRNX1-seg-5', 'NRNX1-seg-5', 'NRNX1-seg-5', 'NRNX1-seg-5', 'NRNX1-seg-59', 'NRNX1-seg-59', 'NRNX1-seg-59', 'NRNX1-seg-59', 'NRNX1-seg-177', 'NRNX1-seg-177', 'NRNX1-seg-177', 'NRNX1-seg-177', 'NRNX1-seg-238', 'NRNX1-seg-238', 'NRNX1-seg-238', 'NRNX1-seg-238', 'NRNX1-seg-282', 'NRNX1-seg-282', 'NRNX1-seg-282', 'NRNX1-seg-282', 'NRNX1-seg-398', 'NRNX1-seg-398', 'NRNX1-seg-398', 'NRNX1-seg-398', 'NRNX1-seg-459', 'NRNX1-seg-459', 'NRNX1-seg-459', 'NRNX1-seg-459', 'NRNX1-seg-540', 'NRNX1-seg-540', 'NRNX1-seg-540', 'NRNX1-seg-540', 'NRNX1-seg-641', 'NRNX1-seg-641', 'NRNX1-seg-641', 'NRNX1-seg-641'], 'NDB_789': ['NRNX1-seg-5', 'NRNX1-seg-5', 'NRNX1-seg-5', 'NRNX1-seg-5', 'NRNX1-seg-133', 'NRNX1-seg-133', 'NRNX1-seg-133', 'NRNX1-seg-133', 'NRNX1-seg-292', 'NRNX1-seg-292', 'NRNX1-seg-292', 'NRNX1-seg-292', 'NRNX1-seg-356', 'NRNX1-seg-356', 'NRNX1-seg-356', 'NRNX1-seg-356', 'NRNX1-seg-408', 'NRNX1-seg-408', 'NRNX1-seg-408', 'NRNX1-seg-408', 'NRNX1-seg-490', 'NRNX1-seg-490', 'NRNX1-seg-490', 'NRNX1-seg-490', 'NRNX1-seg-550', 'NRNX1-seg-550', 'NRNX1-seg-550', 'NRNX1-seg-550', 'NRNX1-seg-696', 'NRNX1-seg-696', 'NRNX1-seg-696', 'NRNX1-seg-696'], 'NDB_427': ['NRNX1-seg-6', 'NRNX1-seg-6', 'NRNX1-seg-6', 'NRNX1-seg-6', 'NRNX1-seg-102', 'NRNX1-seg-102', 'NRNX1-seg-102', 'NRNX1-seg-102', 'NRNX1-seg-166', 'NRNX1-seg-166', 'NRNX1-seg-166', 'NRNX1-seg-166', 'NRNX1-seg-273', 'NRNX1-seg-273', 'NRNX1-seg-273', 'NRNX1-seg-273', 'NRNX1-seg-337', 'NRNX1-seg-337', 'NRNX1-seg-337', 'NRNX1-seg-337', 'NRNX1-seg-429', 'NRNX1-seg-429', 'NRNX1-seg-429', 'NRNX1-seg-429', 'NRNX1-seg-531', 'NRNX1-seg-531', 'NRNX1-seg-531', 'NRNX1-seg-531', 'NRNX1-seg-611', 'NRNX1-seg-611', 'NRNX1-seg-611', 'NRNX1-seg-611', 'NRNX1-seg-662', 'NRNX1-seg-662', 'NRNX1-seg-662', 'NRNX1-seg-662'], 'NDB_506': ['NRNX1-seg-6', 'NRNX1-seg-6', 'NRNX1-seg-6', 'NRNX1-seg-6', 'NRNX1-seg-60', 'NRNX1-seg-60', 'NRNX1-seg-60', 'NRNX1-seg-60', 'NRNX1-seg-178', 'NRNX1-seg-178', 'NRNX1-seg-178', 'NRNX1-seg-178', 'NRNX1-seg-239', 'NRNX1-seg-239', 'NRNX1-seg-239', 'NRNX1-seg-239', 'NRNX1-seg-283', 'NRNX1-seg-283', 'NRNX1-seg-283', 'NRNX1-seg-283', 'NRNX1-seg-399', 'NRNX1-seg-399', 'NRNX1-seg-399', 'NRNX1-seg-399', 'NRNX1-seg-460', 'NRNX1-seg-460', 'NRNX1-seg-460', 'NRNX1-seg-460', 'NRNX1-seg-541', 'NRNX1-seg-541', 'NRNX1-seg-541', 'NRNX1-seg-541', 'NRNX1-seg-642', 'NRNX1-seg-642', 'NRNX1-seg-642', 'NRNX1-seg-642'], 'NDB_1047': ['NRNX1-seg-6', 'NRNX1-seg-6', 'NRNX1-seg-6', 'NRNX1-seg-6', 'NRNX1-seg-134', 'NRNX1-seg-134', 'NRNX1-seg-134', 'NRNX1-seg-134', 'NRNX1-seg-293', 'NRNX1-seg-293', 'NRNX1-seg-293', 'NRNX1-seg-293', 'NRNX1-seg-357', 'NRNX1-seg-357', 'NRNX1-seg-357', 'NRNX1-seg-357', 'NRNX1-seg-409', 'NRNX1-seg-409', 'NRNX1-seg-409', 'NRNX1-seg-409', 'NRNX1-seg-491', 'NRNX1-seg-491', 'NRNX1-seg-491', 'NRNX1-seg-491', 'NRNX1-seg-551', 'NRNX1-seg-551', 'NRNX1-seg-551', 'NRNX1-seg-551', 'NRNX1-seg-697', 'NRNX1-seg-697', 'NRNX1-seg-697', 'NRNX1-seg-697'], 'NDB_397': ['NRNX1-seg-7', 'NRNX1-seg-7', 'NRNX1-seg-7', 'NRNX1-seg-7', 'NRNX1-seg-103', 'NRNX1-seg-103', 'NRNX1-seg-103', 'NRNX1-seg-103', 'NRNX1-seg-169', 'NRNX1-seg-169', 'NRNX1-seg-169', 'NRNX1-seg-169', 'NRNX1-seg-274', 'NRNX1-seg-274', 'NRNX1-seg-274', 'NRNX1-seg-274', 'NRNX1-seg-338', 'NRNX1-seg-338', 'NRNX1-seg-338', 'NRNX1-seg-338', 'NRNX1-seg-430', 'NRNX1-seg-430', 'NRNX1-seg-430', 'NRNX1-seg-430', 'NRNX1-seg-532', 'NRNX1-seg-532', 'NRNX1-seg-532', 'NRNX1-seg-532', 'NRNX1-seg-612', 'NRNX1-seg-612', 'NRNX1-seg-612', 'NRNX1-seg-612', 'NRNX1-seg-665', 'NRNX1-seg-665', 'NRNX1-seg-665', 'NRNX1-seg-665'], 'NDB_989': ['NRNX1-seg-7', 'NRNX1-seg-7', 'NRNX1-seg-7', 'NRNX1-seg-7', 'NRNX1-seg-62', 'NRNX1-seg-62', 'NRNX1-seg-62', 'NRNX1-seg-62', 'NRNX1-seg-179', 'NRNX1-seg-179', 'NRNX1-seg-179', 'NRNX1-seg-179', 'NRNX1-seg-240', 'NRNX1-seg-240', 'NRNX1-seg-240', 'NRNX1-seg-240', 'NRNX1-seg-284', 'NRNX1-seg-284', 'NRNX1-seg-284', 'NRNX1-seg-284', 'NRNX1-seg-400', 'NRNX1-seg-400', 'NRNX1-seg-400', 'NRNX1-seg-400', 'NRNX1-seg-461', 'NRNX1-seg-461', 'NRNX1-seg-461', 'NRNX1-seg-461', 'NRNX1-seg-542', 'NRNX1-seg-542', 'NRNX1-seg-542', 'NRNX1-seg-542', 'NRNX1-seg-643', 'NRNX1-seg-643', 'NRNX1-seg-643', 'NRNX1-seg-643'], 'NDB_618': ['NRNX1-seg-7', 'NRNX1-seg-7', 'NRNX1-seg-7', 'NRNX1-seg-7', 'NRNX1-seg-135', 'NRNX1-seg-135', 'NRNX1-seg-135', 'NRNX1-seg-135', 'NRNX1-seg-294', 'NRNX1-seg-294', 'NRNX1-seg-294', 'NRNX1-seg-294', 'NRNX1-seg-358', 'NRNX1-seg-358', 'NRNX1-seg-358', 'NRNX1-seg-358', 'NRNX1-seg-410', 'NRNX1-seg-410', 'NRNX1-seg-410', 'NRNX1-seg-410', 'NRNX1-seg-492', 'NRNX1-seg-492', 'NRNX1-seg-492', 'NRNX1-seg-492', 'NRNX1-seg-552', 'NRNX1-seg-552', 'NRNX1-seg-552', 'NRNX1-seg-552', 'NRNX1-seg-698', 'NRNX1-seg-698', 'NRNX1-seg-698', 'NRNX1-seg-698'], 'NDB_514': ['NRNX1-seg-9', 'NRNX1-seg-9', 'NRNX1-seg-9', 'NRNX1-seg-9', 'NRNX1-seg-104', 'NRNX1-seg-104', 'NRNX1-seg-104', 'NRNX1-seg-104', 'NRNX1-seg-170', 'NRNX1-seg-170', 'NRNX1-seg-170', 'NRNX1-seg-170', 'NRNX1-seg-275', 'NRNX1-seg-275', 'NRNX1-seg-275', 'NRNX1-seg-275', 'NRNX1-seg-339', 'NRNX1-seg-339', 'NRNX1-seg-339', 'NRNX1-seg-339', 'NRNX1-seg-431', 'NRNX1-seg-431', 'NRNX1-seg-431', 'NRNX1-seg-431', 'NRNX1-seg-533', 'NRNX1-seg-533', 'NRNX1-seg-533', 'NRNX1-seg-533', 'NRNX1-seg-613', 'NRNX1-seg-613', 'NRNX1-seg-613', 'NRNX1-seg-613', 'NRNX1-seg-666', 'NRNX1-seg-666', 'NRNX1-seg-666', 'NRNX1-seg-666'], 'NDB_665': ['NRNX1-seg-9', 'NRNX1-seg-9', 'NRNX1-seg-9', 'NRNX1-seg-9', 'NRNX1-seg-63', 'NRNX1-seg-63', 'NRNX1-seg-63', 'NRNX1-seg-63', 'NRNX1-seg-180', 'NRNX1-seg-180', 'NRNX1-seg-180', 'NRNX1-seg-180', 'NRNX1-seg-241', 'NRNX1-seg-241', 'NRNX1-seg-241', 'NRNX1-seg-241', 'NRNX1-seg-285', 'NRNX1-seg-285', 'NRNX1-seg-285', 'NRNX1-seg-285', 'NRNX1-seg-401', 'NRNX1-seg-401', 'NRNX1-seg-401', 'NRNX1-seg-401', 'NRNX1-seg-462', 'NRNX1-seg-462', 'NRNX1-seg-462', 'NRNX1-seg-462', 'NRNX1-seg-543', 'NRNX1-seg-543', 'NRNX1-seg-543', 'NRNX1-seg-543', 'NRNX1-seg-644', 'NRNX1-seg-644', 'NRNX1-seg-644', 'NRNX1-seg-644'], 'NDB_819': ['NRNX1-seg-9', 'NRNX1-seg-9', 'NRNX1-seg-9', 'NRNX1-seg-9', 'NRNX1-seg-136', 'NRNX1-seg-136', 'NRNX1-seg-136', 'NRNX1-seg-136', 'NRNX1-seg-295', 'NRNX1-seg-295', 'NRNX1-seg-295', 'NRNX1-seg-295', 'NRNX1-seg-359', 'NRNX1-seg-359', 'NRNX1-seg-359', 'NRNX1-seg-359', 'NRNX1-seg-411', 'NRNX1-seg-411', 'NRNX1-seg-411', 'NRNX1-seg-411', 'NRNX1-seg-493', 'NRNX1-seg-493', 'NRNX1-seg-493', 'NRNX1-seg-493', 'NRNX1-seg-553', 'NRNX1-seg-553', 'NRNX1-seg-553', 'NRNX1-seg-553', 'NRNX1-seg-699', 'NRNX1-seg-699', 'NRNX1-seg-699', 'NRNX1-seg-699'], 'NDB_951': ['NRNX1-seg-10', 'NRNX1-seg-10', 'NRNX1-seg-10', 'NRNX1-seg-10', 'NRNX1-seg-126', 'NRNX1-seg-126', 'NRNX1-seg-126', 'NRNX1-seg-126', 'NRNX1-seg-211', 'NRNX1-seg-211', 'NRNX1-seg-211', 'NRNX1-seg-211', 'NRNX1-seg-320', 'NRNX1-seg-320', 'NRNX1-seg-320', 'NRNX1-seg-320', 'NRNX1-seg-360', 'NRNX1-seg-360', 'NRNX1-seg-360', 'NRNX1-seg-360', 'NRNX1-seg-463', 'NRNX1-seg-463', 'NRNX1-seg-463', 'NRNX1-seg-463', 'NRNX1-seg-514', 'NRNX1-seg-514', 'NRNX1-seg-514', 'NRNX1-seg-514', 'NRNX1-seg-645', 'NRNX1-seg-645', 'NRNX1-seg-645', 'NRNX1-seg-645'], 'NDB_401': ['NRNX1-seg-10', 'NRNX1-seg-10', 'NRNX1-seg-10', 'NRNX1-seg-10', 'NRNX1-seg-95', 'NRNX1-seg-95', 'NRNX1-seg-95', 'NRNX1-seg-95', 'NRNX1-seg-137', 'NRNX1-seg-137', 'NRNX1-seg-137', 'NRNX1-seg-137', 'NRNX1-seg-266', 'NRNX1-seg-266', 'NRNX1-seg-266', 'NRNX1-seg-266', 'NRNX1-seg-340', 'NRNX1-seg-340', 'NRNX1-seg-340', 'NRNX1-seg-340', 'NRNX1-seg-432', 'NRNX1-seg-432', 'NRNX1-seg-432', 'NRNX1-seg-432', 'NRNX1-seg-504', 'NRNX1-seg-504', 'NRNX1-seg-504', 'NRNX1-seg-504', 'NRNX1-seg-614', 'NRNX1-seg-614', 'NRNX1-seg-614', 'NRNX1-seg-614'], 'NDB_594': ['NRNX1-seg-10', 'NRNX1-seg-10', 'NRNX1-seg-10', 'NRNX1-seg-10', 'NRNX1-seg-53', 'NRNX1-seg-53', 'NRNX1-seg-53', 'NRNX1-seg-53', 'NRNX1-seg-147', 'NRNX1-seg-147', 'NRNX1-seg-147', 'NRNX1-seg-147', 'NRNX1-seg-242', 'NRNX1-seg-242', 'NRNX1-seg-242', 'NRNX1-seg-242', 'NRNX1-seg-392', 'NRNX1-seg-392', 'NRNX1-seg-392', 'NRNX1-seg-392', 'NRNX1-seg-442', 'NRNX1-seg-442', 'NRNX1-seg-442', 'NRNX1-seg-442', 'NRNX1-seg-484', 'NRNX1-seg-484', 'NRNX1-seg-484', 'NRNX1-seg-484', 'NRNX1-seg-534', 'NRNX1-seg-534', 'NRNX1-seg-534', 'NRNX1-seg-534', 'NRNX1-seg-584', 'NRNX1-seg-584', 'NRNX1-seg-584', 'NRNX1-seg-584'], 'NDB_735': ['NRNX1-seg-13', 'NRNX1-seg-13', 'NRNX1-seg-13', 'NRNX1-seg-13', 'NRNX1-seg-127', 'NRNX1-seg-127', 'NRNX1-seg-127', 'NRNX1-seg-127', 'NRNX1-seg-212', 'NRNX1-seg-212', 'NRNX1-seg-212', 'NRNX1-seg-212', 'NRNX1-seg-321', 'NRNX1-seg-321', 'NRNX1-seg-321', 'NRNX1-seg-321', 'NRNX1-seg-361', 'NRNX1-seg-361', 'NRNX1-seg-361', 'NRNX1-seg-361', 'NRNX1-seg-464', 'NRNX1-seg-464', 'NRNX1-seg-464', 'NRNX1-seg-464', 'NRNX1-seg-515', 'NRNX1-seg-515', 'NRNX1-seg-515', 'NRNX1-seg-515', 'NRNX1-seg-646', 'NRNX1-seg-646', 'NRNX1-seg-646', 'NRNX1-seg-646'], 'NDB_824': ['NRNX1-seg-13', 'NRNX1-seg-13', 'NRNX1-seg-13', 'NRNX1-seg-13', 'NRNX1-seg-96', 'NRNX1-seg-96', 'NRNX1-seg-96', 'NRNX1-seg-96', 'NRNX1-seg-138', 'NRNX1-seg-138', 'NRNX1-seg-138', 'NRNX1-seg-138', 'NRNX1-seg-267', 'NRNX1-seg-267', 'NRNX1-seg-267', 'NRNX1-seg-267', 'NRNX1-seg-341', 'NRNX1-seg-341', 'NRNX1-seg-341', 'NRNX1-seg-341', 'NRNX1-seg-433', 'NRNX1-seg-433', 'NRNX1-seg-433', 'NRNX1-seg-433', 'NRNX1-seg-505', 'NRNX1-seg-505', 'NRNX1-seg-505', 'NRNX1-seg-505', 'NRNX1-seg-615', 'NRNX1-seg-615', 'NRNX1-seg-615', 'NRNX1-seg-615'], 'NDB_903': ['NRNX1-seg-13', 'NRNX1-seg-13', 'NRNX1-seg-13', 'NRNX1-seg-13', 'NRNX1-seg-54', 'NRNX1-seg-54', 'NRNX1-seg-54', 'NRNX1-seg-54', 'NRNX1-seg-148', 'NRNX1-seg-148', 'NRNX1-seg-148', 'NRNX1-seg-148', 'NRNX1-seg-244', 'NRNX1-seg-244', 'NRNX1-seg-244', 'NRNX1-seg-244', 'NRNX1-seg-393', 'NRNX1-seg-393', 'NRNX1-seg-393', 'NRNX1-seg-393', 'NRNX1-seg-443', 'NRNX1-seg-443', 'NRNX1-seg-443', 'NRNX1-seg-443', 'NRNX1-seg-485', 'NRNX1-seg-485', 'NRNX1-seg-485', 'NRNX1-seg-485', 'NRNX1-seg-535', 'NRNX1-seg-535', 'NRNX1-seg-535', 'NRNX1-seg-535', 'NRNX1-seg-585', 'NRNX1-seg-585', 'NRNX1-seg-585', 'NRNX1-seg-585'], 'NDB_784': ['NRNX1-seg-14', 'NRNX1-seg-14', 'NRNX1-seg-14', 'NRNX1-seg-14', 'NRNX1-seg-107', 'NRNX1-seg-107', 'NRNX1-seg-107', 'NRNX1-seg-107', 'NRNX1-seg-257', 'NRNX1-seg-257', 'NRNX1-seg-257', 'NRNX1-seg-257', 'NRNX1-seg-362', 'NRNX1-seg-362', 'NRNX1-seg-362', 'NRNX1-seg-362', 'NRNX1-seg-424', 'NRNX1-seg-424', 'NRNX1-seg-424', 'NRNX1-seg-424', 'NRNX1-seg-496', 'NRNX1-seg-496', 'NRNX1-seg-496', 'NRNX1-seg-496', 'NRNX1-seg-546', 'NRNX1-seg-546', 'NRNX1-seg-546', 'NRNX1-seg-546', 'NRNX1-seg-586', 'NRNX1-seg-586', 'NRNX1-seg-586', 'NRNX1-seg-586', 'NRNX1-seg-626', 'NRNX1-seg-626', 'NRNX1-seg-626', 'NRNX1-seg-626'], 'NDB_801': ['NRNX1-seg-14', 'NRNX1-seg-14', 'NRNX1-seg-14', 'NRNX1-seg-14', 'NRNX1-seg-97', 'NRNX1-seg-97', 'NRNX1-seg-97', 'NRNX1-seg-97', 'NRNX1-seg-183', 'NRNX1-seg-183', 'NRNX1-seg-183', 'NRNX1-seg-183', 'NRNX1-seg-300', 'NRNX1-seg-300', 'NRNX1-seg-300', 'NRNX1-seg-300', 'NRNX1-seg-342', 'NRNX1-seg-342', 'NRNX1-seg-342', 'NRNX1-seg-342', 'NRNX1-seg-465', 'NRNX1-seg-465', 'NRNX1-seg-465', 'NRNX1-seg-465', 'NRNX1-seg-536', 'NRNX1-seg-536', 'NRNX1-seg-536', 'NRNX1-seg-536', 'NRNX1-seg-616', 'NRNX1-seg-616', 'NRNX1-seg-616', 'NRNX1-seg-616'], 'NDB_652': ['NRNX1-seg-14', 'NRNX1-seg-14', 'NRNX1-seg-14', 'NRNX1-seg-14', 'NRNX1-seg-87', 'NRNX1-seg-87', 'NRNX1-seg-87', 'NRNX1-seg-87', 'NRNX1-seg-139', 'NRNX1-seg-139', 'NRNX1-seg-139', 'NRNX1-seg-139', 'NRNX1-seg-213', 'NRNX1-seg-213', 'NRNX1-seg-213', 'NRNX1-seg-213', 'NRNX1-seg-278', 'NRNX1-seg-278', 'NRNX1-seg-278', 'NRNX1-seg-278', 'NRNX1-seg-372', 'NRNX1-seg-372', 'NRNX1-seg-372', 'NRNX1-seg-372', 'NRNX1-seg-434', 'NRNX1-seg-434', 'NRNX1-seg-434', 'NRNX1-seg-434', 'NRNX1-seg-526', 'NRNX1-seg-526', 'NRNX1-seg-526', 'NRNX1-seg-526', 'NRNX1-seg-692', 'NRNX1-seg-692', 'NRNX1-seg-692', 'NRNX1-seg-692'], 'NDB_643': ['NRNX1-seg-15', 'NRNX1-seg-15', 'NRNX1-seg-15', 'NRNX1-seg-15', 'NRNX1-seg-108', 'NRNX1-seg-108', 'NRNX1-seg-108', 'NRNX1-seg-108', 'NRNX1-seg-258', 'NRNX1-seg-258', 'NRNX1-seg-258', 'NRNX1-seg-258', 'NRNX1-seg-363', 'NRNX1-seg-363', 'NRNX1-seg-363', 'NRNX1-seg-363', 'NRNX1-seg-425', 'NRNX1-seg-425', 'NRNX1-seg-425', 'NRNX1-seg-425', 'NRNX1-seg-497', 'NRNX1-seg-497', 'NRNX1-seg-497', 'NRNX1-seg-497', 'NRNX1-seg-547', 'NRNX1-seg-547', 'NRNX1-seg-547', 'NRNX1-seg-547', 'NRNX1-seg-587', 'NRNX1-seg-587', 'NRNX1-seg-587', 'NRNX1-seg-587', 'NRNX1-seg-627', 'NRNX1-seg-627', 'NRNX1-seg-627', 'NRNX1-seg-627'], 'NDB_825': ['NRNX1-seg-15', 'NRNX1-seg-15', 'NRNX1-seg-15', 'NRNX1-seg-15', 'NRNX1-seg-98', 'NRNX1-seg-98', 'NRNX1-seg-98', 'NRNX1-seg-98', 'NRNX1-seg-184', 'NRNX1-seg-184', 'NRNX1-seg-184', 'NRNX1-seg-184', 'NRNX1-seg-301', 'NRNX1-seg-301', 'NRNX1-seg-301', 'NRNX1-seg-301', 'NRNX1-seg-343', 'NRNX1-seg-343', 'NRNX1-seg-343', 'NRNX1-seg-343', 'NRNX1-seg-466', 'NRNX1-seg-466', 'NRNX1-seg-466', 'NRNX1-seg-466', 'NRNX1-seg-537', 'NRNX1-seg-537', 'NRNX1-seg-537', 'NRNX1-seg-537', 'NRNX1-seg-617', 'NRNX1-seg-617', 'NRNX1-seg-617', 'NRNX1-seg-617'], 'NDB_844': ['NRNX1-seg-15', 'NRNX1-seg-15', 'NRNX1-seg-15', 'NRNX1-seg-15', 'NRNX1-seg-88', 'NRNX1-seg-88', 'NRNX1-seg-88', 'NRNX1-seg-88', 'NRNX1-seg-140', 'NRNX1-seg-140', 'NRNX1-seg-140', 'NRNX1-seg-140', 'NRNX1-seg-214', 'NRNX1-seg-214', 'NRNX1-seg-214', 'NRNX1-seg-214', 'NRNX1-seg-279', 'NRNX1-seg-279', 'NRNX1-seg-279', 'NRNX1-seg-279', 'NRNX1-seg-373', 'NRNX1-seg-373', 'NRNX1-seg-373', 'NRNX1-seg-373', 'NRNX1-seg-435', 'NRNX1-seg-435', 'NRNX1-seg-435', 'NRNX1-seg-435', 'NRNX1-seg-527', 'NRNX1-seg-527', 'NRNX1-seg-527', 'NRNX1-seg-527', 'NRNX1-seg-693', 'NRNX1-seg-693', 'NRNX1-seg-693', 'NRNX1-seg-693'], 'NDB_568': ['NRNX1-seg-16', 'NRNX1-seg-16', 'NRNX1-seg-16', 'NRNX1-seg-16', 'NRNX1-seg-109', 'NRNX1-seg-109', 'NRNX1-seg-109', 'NRNX1-seg-109', 'NRNX1-seg-259', 'NRNX1-seg-259', 'NRNX1-seg-259', 'NRNX1-seg-259', 'NRNX1-seg-364', 'NRNX1-seg-364', 'NRNX1-seg-364', 'NRNX1-seg-364', 'NRNX1-seg-426', 'NRNX1-seg-426', 'NRNX1-seg-426', 'NRNX1-seg-426', 'NRNX1-seg-498', 'NRNX1-seg-498', 'NRNX1-seg-498', 'NRNX1-seg-498', 'NRNX1-seg-548', 'NRNX1-seg-548', 'NRNX1-seg-548', 'NRNX1-seg-548', 'NRNX1-seg-588', 'NRNX1-seg-588', 'NRNX1-seg-588', 'NRNX1-seg-588', 'NRNX1-seg-628', 'NRNX1-seg-628', 'NRNX1-seg-628', 'NRNX1-seg-628'], 'NDB_375': ['NRNX1-seg-16', 'NRNX1-seg-16', 'NRNX1-seg-16', 'NRNX1-seg-16', 'NRNX1-seg-99', 'NRNX1-seg-99', 'NRNX1-seg-99', 'NRNX1-seg-99', 'NRNX1-seg-185', 'NRNX1-seg-185', 'NRNX1-seg-185', 'NRNX1-seg-185', 'NRNX1-seg-302', 'NRNX1-seg-302', 'NRNX1-seg-302', 'NRNX1-seg-302', 'NRNX1-seg-344', 'NRNX1-seg-344', 'NRNX1-seg-344', 'NRNX1-seg-344', 'NRNX1-seg-467', 'NRNX1-seg-467', 'NRNX1-seg-467', 'NRNX1-seg-467', 'NRNX1-seg-538', 'NRNX1-seg-538', 'NRNX1-seg-538', 'NRNX1-seg-538', 'NRNX1-seg-618', 'NRNX1-seg-618', 'NRNX1-seg-618', 'NRNX1-seg-618'], 'NDB_409': ['NRNX1-seg-16', 'NRNX1-seg-16', 'NRNX1-seg-16', 'NRNX1-seg-16', 'NRNX1-seg-89', 'NRNX1-seg-89', 'NRNX1-seg-89', 'NRNX1-seg-89', 'NRNX1-seg-141', 'NRNX1-seg-141', 'NRNX1-seg-141', 'NRNX1-seg-141', 'NRNX1-seg-215', 'NRNX1-seg-215', 'NRNX1-seg-215', 'NRNX1-seg-215', 'NRNX1-seg-280', 'NRNX1-seg-280', 'NRNX1-seg-280', 'NRNX1-seg-280', 'NRNX1-seg-374', 'NRNX1-seg-374', 'NRNX1-seg-374', 'NRNX1-seg-374', 'NRNX1-seg-436', 'NRNX1-seg-436', 'NRNX1-seg-436', 'NRNX1-seg-436', 'NRNX1-seg-528', 'NRNX1-seg-528', 'NRNX1-seg-528', 'NRNX1-seg-528', 'NRNX1-seg-694', 'NRNX1-seg-694', 'NRNX1-seg-694', 'NRNX1-seg-694'], 'NDB_868': ['NRNX1-seg-17', 'NRNX1-seg-17', 'NRNX1-seg-17', 'NRNX1-seg-17', 'NRNX1-seg-110', 'NRNX1-seg-110', 'NRNX1-seg-110', 'NRNX1-seg-110', 'NRNX1-seg-260', 'NRNX1-seg-260', 'NRNX1-seg-260', 'NRNX1-seg-260', 'NRNX1-seg-365', 'NRNX1-seg-365', 'NRNX1-seg-365', 'NRNX1-seg-365', 'NRNX1-seg-427', 'NRNX1-seg-427', 'NRNX1-seg-427', 'NRNX1-seg-427', 'NRNX1-seg-499', 'NRNX1-seg-499', 'NRNX1-seg-499', 'NRNX1-seg-499', 'NRNX1-seg-549', 'NRNX1-seg-549', 'NRNX1-seg-549', 'NRNX1-seg-549', 'NRNX1-seg-589', 'NRNX1-seg-589', 'NRNX1-seg-589', 'NRNX1-seg-589', 'NRNX1-seg-629', 'NRNX1-seg-629', 'NRNX1-seg-629', 'NRNX1-seg-629'], 'NDB_417': ['NRNX1-seg-17', 'NRNX1-seg-17', 'NRNX1-seg-17', 'NRNX1-seg-17', 'NRNX1-seg-100', 'NRNX1-seg-100', 'NRNX1-seg-100', 'NRNX1-seg-100', 'NRNX1-seg-186', 'NRNX1-seg-186', 'NRNX1-seg-186', 'NRNX1-seg-186', 'NRNX1-seg-303', 'NRNX1-seg-303', 'NRNX1-seg-303', 'NRNX1-seg-303', 'NRNX1-seg-345', 'NRNX1-seg-345', 'NRNX1-seg-345', 'NRNX1-seg-345', 'NRNX1-seg-468', 'NRNX1-seg-468', 'NRNX1-seg-468', 'NRNX1-seg-468', 'NRNX1-seg-539', 'NRNX1-seg-539', 'NRNX1-seg-539', 'NRNX1-seg-539', 'NRNX1-seg-619', 'NRNX1-seg-619', 'NRNX1-seg-619', 'NRNX1-seg-619'], 'NDB_754': ['NRNX1-seg-17', 'NRNX1-seg-17', 'NRNX1-seg-17', 'NRNX1-seg-17', 'NRNX1-seg-90', 'NRNX1-seg-90', 'NRNX1-seg-90', 'NRNX1-seg-90', 'NRNX1-seg-142', 'NRNX1-seg-142', 'NRNX1-seg-142', 'NRNX1-seg-142', 'NRNX1-seg-216', 'NRNX1-seg-216', 'NRNX1-seg-216', 'NRNX1-seg-216', 'NRNX1-seg-281', 'NRNX1-seg-281', 'NRNX1-seg-281', 'NRNX1-seg-281', 'NRNX1-seg-375', 'NRNX1-seg-375', 'NRNX1-seg-375', 'NRNX1-seg-375', 'NRNX1-seg-437', 'NRNX1-seg-437', 'NRNX1-seg-437', 'NRNX1-seg-437', 'NRNX1-seg-529', 'NRNX1-seg-529', 'NRNX1-seg-529', 'NRNX1-seg-529', 'NRNX1-seg-695', 'NRNX1-seg-695', 'NRNX1-seg-695', 'NRNX1-seg-695'], 'NDB_634': ['NRNX1-seg-18', 'NRNX1-seg-18', 'NRNX1-seg-18', 'NRNX1-seg-18', 'NRNX1-seg-111', 'NRNX1-seg-111', 'NRNX1-seg-111', 'NRNX1-seg-111', 'NRNX1-seg-262', 'NRNX1-seg-262', 'NRNX1-seg-262', 'NRNX1-seg-262', 'NRNX1-seg-366', 'NRNX1-seg-366', 'NRNX1-seg-366', 'NRNX1-seg-366', 'NRNX1-seg-428', 'NRNX1-seg-428', 'NRNX1-seg-428', 'NRNX1-seg-428', 'NRNX1-seg-500', 'NRNX1-seg-500', 'NRNX1-seg-500', 'NRNX1-seg-500', 'NRNX1-seg-550', 'NRNX1-seg-550', 'NRNX1-seg-550', 'NRNX1-seg-550', 'NRNX1-seg-590', 'NRNX1-seg-590', 'NRNX1-seg-590', 'NRNX1-seg-590', 'NRNX1-seg-630', 'NRNX1-seg-630', 'NRNX1-seg-630', 'NRNX1-seg-630'], 'NDB_954': ['NRNX1-seg-18', 'NRNX1-seg-18', 'NRNX1-seg-18', 'NRNX1-seg-18', 'NRNX1-seg-101', 'NRNX1-seg-101', 'NRNX1-seg-101', 'NRNX1-seg-101', 'NRNX1-seg-187', 'NRNX1-seg-187', 'NRNX1-seg-187', 'NRNX1-seg-187', 'NRNX1-seg-304', 'NRNX1-seg-304', 'NRNX1-seg-304', 'NRNX1-seg-304', 'NRNX1-seg-346', 'NRNX1-seg-346', 'NRNX1-seg-346', 'NRNX1-seg-346', 'NRNX1-seg-469', 'NRNX1-seg-469', 'NRNX1-seg-469', 'NRNX1-seg-469', 'NRNX1-seg-540', 'NRNX1-seg-540', 'NRNX1-seg-540', 'NRNX1-seg-540', 'NRNX1-seg-620', 'NRNX1-seg-620', 'NRNX1-seg-620', 'NRNX1-seg-620'], 'NDB_532': ['NRNX1-seg-18', 'NRNX1-seg-18', 'NRNX1-seg-18', 'NRNX1-seg-18', 'NRNX1-seg-91', 'NRNX1-seg-91', 'NRNX1-seg-91', 'NRNX1-seg-91', 'NRNX1-seg-143', 'NRNX1-seg-143', 'NRNX1-seg-143', 'NRNX1-seg-143', 'NRNX1-seg-217', 'NRNX1-seg-217', 'NRNX1-seg-217', 'NRNX1-seg-217', 'NRNX1-seg-282', 'NRNX1-seg-282', 'NRNX1-seg-282', 'NRNX1-seg-282', 'NRNX1-seg-376', 'NRNX1-seg-376', 'NRNX1-seg-376', 'NRNX1-seg-376', 'NRNX1-seg-438', 'NRNX1-seg-438', 'NRNX1-seg-438', 'NRNX1-seg-438', 'NRNX1-seg-530', 'NRNX1-seg-530', 'NRNX1-seg-530', 'NRNX1-seg-530', 'NRNX1-seg-696', 'NRNX1-seg-696', 'NRNX1-seg-696', 'NRNX1-seg-696'], 'NDB_388': ['NRNX1-seg-19', 'NRNX1-seg-19', 'NRNX1-seg-19', 'NRNX1-seg-19', 'NRNX1-seg-112', 'NRNX1-seg-112', 'NRNX1-seg-112', 'NRNX1-seg-112', 'NRNX1-seg-263', 'NRNX1-seg-263', 'NRNX1-seg-263', 'NRNX1-seg-263', 'NRNX1-seg-367', 'NRNX1-seg-367', 'NRNX1-seg-367', 'NRNX1-seg-367', 'NRNX1-seg-429', 'NRNX1-seg-429', 'NRNX1-seg-429', 'NRNX1-seg-429', 'NRNX1-seg-501', 'NRNX1-seg-501', 'NRNX1-seg-501', 'NRNX1-seg-501', 'NRNX1-seg-551', 'NRNX1-seg-551', 'NRNX1-seg-551', 'NRNX1-seg-551', 'NRNX1-seg-591', 'NRNX1-seg-591', 'NRNX1-seg-591', 'NRNX1-seg-591', 'NRNX1-seg-631', 'NRNX1-seg-631', 'NRNX1-seg-631', 'NRNX1-seg-631'], 'NDB_780': ['NRNX1-seg-19', 'NRNX1-seg-19', 'NRNX1-seg-19', 'NRNX1-seg-19', 'NRNX1-seg-102', 'NRNX1-seg-102', 'NRNX1-seg-102', 'NRNX1-seg-102', 'NRNX1-seg-188', 'NRNX1-seg-188', 'NRNX1-seg-188', 'NRNX1-seg-188', 'NRNX1-seg-305', 'NRNX1-seg-305', 'NRNX1-seg-305', 'NRNX1-seg-305', 'NRNX1-seg-347', 'NRNX1-seg-347', 'NRNX1-seg-347', 'NRNX1-seg-347', 'NRNX1-seg-470', 'NRNX1-seg-470', 'NRNX1-seg-470', 'NRNX1-seg-470', 'NRNX1-seg-541', 'NRNX1-seg-541', 'NRNX1-seg-541', 'NRNX1-seg-541', 'NRNX1-seg-621', 'NRNX1-seg-621', 'NRNX1-seg-621', 'NRNX1-seg-621'], 'NDB_730': ['NRNX1-seg-19', 'NRNX1-seg-19', 'NRNX1-seg-19', 'NRNX1-seg-19', 'NRNX1-seg-92', 'NRNX1-seg-92', 'NRNX1-seg-92', 'NRNX1-seg-92', 'NRNX1-seg-144', 'NRNX1-seg-144', 'NRNX1-seg-144', 'NRNX1-seg-144', 'NRNX1-seg-218', 'NRNX1-seg-218', 'NRNX1-seg-218', 'NRNX1-seg-218', 'NRNX1-seg-283', 'NRNX1-seg-283', 'NRNX1-seg-283', 'NRNX1-seg-283', 'NRNX1-seg-379', 'NRNX1-seg-379', 'NRNX1-seg-379', 'NRNX1-seg-379', 'NRNX1-seg-439', 'NRNX1-seg-439', 'NRNX1-seg-439', 'NRNX1-seg-439', 'NRNX1-seg-531', 'NRNX1-seg-531', 'NRNX1-seg-531', 'NRNX1-seg-531', 'NRNX1-seg-697', 'NRNX1-seg-697', 'NRNX1-seg-697', 'NRNX1-seg-697'], 'NDB_835': ['NRNX1-seg-20', 'NRNX1-seg-20', 'NRNX1-seg-20', 'NRNX1-seg-20', 'NRNX1-seg-113', 'NRNX1-seg-113', 'NRNX1-seg-113', 'NRNX1-seg-113', 'NRNX1-seg-264', 'NRNX1-seg-264', 'NRNX1-seg-264', 'NRNX1-seg-264', 'NRNX1-seg-368', 'NRNX1-seg-368', 'NRNX1-seg-368', 'NRNX1-seg-368', 'NRNX1-seg-430', 'NRNX1-seg-430', 'NRNX1-seg-430', 'NRNX1-seg-430', 'NRNX1-seg-502', 'NRNX1-seg-502', 'NRNX1-seg-502', 'NRNX1-seg-502', 'NRNX1-seg-552', 'NRNX1-seg-552', 'NRNX1-seg-552', 'NRNX1-seg-552', 'NRNX1-seg-592', 'NRNX1-seg-592', 'NRNX1-seg-592', 'NRNX1-seg-592', 'NRNX1-seg-632', 'NRNX1-seg-632', 'NRNX1-seg-632', 'NRNX1-seg-632'], 'NDB_630': ['NRNX1-seg-20', 'NRNX1-seg-20', 'NRNX1-seg-20', 'NRNX1-seg-20', 'NRNX1-seg-103', 'NRNX1-seg-103', 'NRNX1-seg-103', 'NRNX1-seg-103', 'NRNX1-seg-189', 'NRNX1-seg-189', 'NRNX1-seg-189', 'NRNX1-seg-189', 'NRNX1-seg-306', 'NRNX1-seg-306', 'NRNX1-seg-306', 'NRNX1-seg-306', 'NRNX1-seg-348', 'NRNX1-seg-348', 'NRNX1-seg-348', 'NRNX1-seg-348', 'NRNX1-seg-471', 'NRNX1-seg-471', 'NRNX1-seg-471', 'NRNX1-seg-471', 'NRNX1-seg-542', 'NRNX1-seg-542', 'NRNX1-seg-542', 'NRNX1-seg-542', 'NRNX1-seg-622', 'NRNX1-seg-622', 'NRNX1-seg-622', 'NRNX1-seg-622'], 'NDB_853': ['NRNX1-seg-20', 'NRNX1-seg-20', 'NRNX1-seg-20', 'NRNX1-seg-20', 'NRNX1-seg-93', 'NRNX1-seg-93', 'NRNX1-seg-93', 'NRNX1-seg-93', 'NRNX1-seg-145', 'NRNX1-seg-145', 'NRNX1-seg-145', 'NRNX1-seg-145', 'NRNX1-seg-219', 'NRNX1-seg-219', 'NRNX1-seg-219', 'NRNX1-seg-219', 'NRNX1-seg-284', 'NRNX1-seg-284', 'NRNX1-seg-284', 'NRNX1-seg-284', 'NRNX1-seg-380', 'NRNX1-seg-380', 'NRNX1-seg-380', 'NRNX1-seg-380', 'NRNX1-seg-440', 'NRNX1-seg-440', 'NRNX1-seg-440', 'NRNX1-seg-440', 'NRNX1-seg-532', 'NRNX1-seg-532', 'NRNX1-seg-532', 'NRNX1-seg-532', 'NRNX1-seg-698', 'NRNX1-seg-698', 'NRNX1-seg-698', 'NRNX1-seg-698'], 'NDB_838': ['NRNX1-seg-21', 'NRNX1-seg-21', 'NRNX1-seg-21', 'NRNX1-seg-21', 'NRNX1-seg-114', 'NRNX1-seg-114', 'NRNX1-seg-114', 'NRNX1-seg-114', 'NRNX1-seg-265', 'NRNX1-seg-265', 'NRNX1-seg-265', 'NRNX1-seg-265', 'NRNX1-seg-369', 'NRNX1-seg-369', 'NRNX1-seg-369', 'NRNX1-seg-369', 'NRNX1-seg-431', 'NRNX1-seg-431', 'NRNX1-seg-431', 'NRNX1-seg-431', 'NRNX1-seg-503', 'NRNX1-seg-503', 'NRNX1-seg-503', 'NRNX1-seg-503', 'NRNX1-seg-553', 'NRNX1-seg-553', 'NRNX1-seg-553', 'NRNX1-seg-553', 'NRNX1-seg-593', 'NRNX1-seg-593', 'NRNX1-seg-593', 'NRNX1-seg-593', 'NRNX1-seg-633', 'NRNX1-seg-633', 'NRNX1-seg-633', 'NRNX1-seg-633'], 'NDB_396': ['NRNX1-seg-21', 'NRNX1-seg-21', 'NRNX1-seg-21', 'NRNX1-seg-21', 'NRNX1-seg-104', 'NRNX1-seg-104', 'NRNX1-seg-104', 'NRNX1-seg-104', 'NRNX1-seg-190', 'NRNX1-seg-190', 'NRNX1-seg-190', 'NRNX1-seg-190', 'NRNX1-seg-307', 'NRNX1-seg-307', 'NRNX1-seg-307', 'NRNX1-seg-307', 'NRNX1-seg-349', 'NRNX1-seg-349', 'NRNX1-seg-349', 'NRNX1-seg-349', 'NRNX1-seg-472', 'NRNX1-seg-472', 'NRNX1-seg-472', 'NRNX1-seg-472', 'NRNX1-seg-543', 'NRNX1-seg-543', 'NRNX1-seg-543', 'NRNX1-seg-543', 'NRNX1-seg-623', 'NRNX1-seg-623', 'NRNX1-seg-623', 'NRNX1-seg-623'], 'NDB_445': ['NRNX1-seg-21', 'NRNX1-seg-21', 'NRNX1-seg-21', 'NRNX1-seg-21', 'NRNX1-seg-94', 'NRNX1-seg-94', 'NRNX1-seg-94', 'NRNX1-seg-94', 'NRNX1-seg-146', 'NRNX1-seg-146', 'NRNX1-seg-146', 'NRNX1-seg-146', 'NRNX1-seg-220', 'NRNX1-seg-220', 'NRNX1-seg-220', 'NRNX1-seg-220', 'NRNX1-seg-285', 'NRNX1-seg-285', 'NRNX1-seg-285', 'NRNX1-seg-285', 'NRNX1-seg-381', 'NRNX1-seg-381', 'NRNX1-seg-381', 'NRNX1-seg-381', 'NRNX1-seg-441', 'NRNX1-seg-441', 'NRNX1-seg-441', 'NRNX1-seg-441', 'NRNX1-seg-533', 'NRNX1-seg-533', 'NRNX1-seg-533', 'NRNX1-seg-533', 'NRNX1-seg-699', 'NRNX1-seg-699', 'NRNX1-seg-699', 'NRNX1-seg-699'], 'NDB_379': ['NRNX1-seg-22', 'NRNX1-seg-22', 'NRNX1-seg-22', 'NRNX1-seg-22', 'NRNX1-seg-159', 'NRNX1-seg-159', 'NRNX1-seg-159', 'NRNX1-seg-159', 'NRNX1-seg-255', 'NRNX1-seg-255', 'NRNX1-seg-255', 'NRNX1-seg-255', 'NRNX1-seg-308', 'NRNX1-seg-308', 'NRNX1-seg-308', 'NRNX1-seg-308', 'NRNX1-seg-402', 'NRNX1-seg-402', 'NRNX1-seg-402', 'NRNX1-seg-402', 'NRNX1-seg-453', 'NRNX1-seg-453', 'NRNX1-seg-453', 'NRNX1-seg-453', 'NRNX1-seg-524', 'NRNX1-seg-524', 'NRNX1-seg-524', 'NRNX1-seg-524', 'NRNX1-seg-645', 'NRNX1-seg-645', 'NRNX1-seg-645', 'NRNX1-seg-645'], 'NDB_977': ['NRNX1-seg-22', 'NRNX1-seg-22', 'NRNX1-seg-22', 'NRNX1-seg-22', 'NRNX1-seg-105', 'NRNX1-seg-105', 'NRNX1-seg-105', 'NRNX1-seg-105', 'NRNX1-seg-266', 'NRNX1-seg-266', 'NRNX1-seg-266', 'NRNX1-seg-266', 'NRNX1-seg-360', 'NRNX1-seg-360', 'NRNX1-seg-360', 'NRNX1-seg-360', 'NRNX1-seg-422', 'NRNX1-seg-422', 'NRNX1-seg-422', 'NRNX1-seg-422', 'NRNX1-seg-484', 'NRNX1-seg-484', 'NRNX1-seg-484', 'NRNX1-seg-484', 'NRNX1-seg-554', 'NRNX1-seg-554', 'NRNX1-seg-554', 'NRNX1-seg-554', 'NRNX1-seg-604', 'NRNX1-seg-604', 'NRNX1-seg-604', 'NRNX1-seg-604'], 'NDB_372': ['NRNX1-seg-22', 'NRNX1-seg-22', 'NRNX1-seg-22', 'NRNX1-seg-22', 'NRNX1-seg-85', 'NRNX1-seg-85', 'NRNX1-seg-85', 'NRNX1-seg-85', 'NRNX1-seg-126', 'NRNX1-seg-126', 'NRNX1-seg-126', 'NRNX1-seg-126', 'NRNX1-seg-191', 'NRNX1-seg-191', 'NRNX1-seg-191', 'NRNX1-seg-191', 'NRNX1-seg-286', 'NRNX1-seg-286', 'NRNX1-seg-286', 'NRNX1-seg-286', 'NRNX1-seg-350', 'NRNX1-seg-350', 'NRNX1-seg-350', 'NRNX1-seg-350', 'NRNX1-seg-442', 'NRNX1-seg-442', 'NRNX1-seg-442', 'NRNX1-seg-442', 'NRNX1-seg-624', 'NRNX1-seg-624', 'NRNX1-seg-624', 'NRNX1-seg-624', 'NRNX1-seg-679', 'NRNX1-seg-679', 'NRNX1-seg-679', 'NRNX1-seg-679'], 'NDB_544': ['NRNX1-seg-23', 'NRNX1-seg-23', 'NRNX1-seg-23', 'NRNX1-seg-23', 'NRNX1-seg-160', 'NRNX1-seg-160', 'NRNX1-seg-160', 'NRNX1-seg-160', 'NRNX1-seg-256', 'NRNX1-seg-256', 'NRNX1-seg-256', 'NRNX1-seg-256', 'NRNX1-seg-311', 'NRNX1-seg-311', 'NRNX1-seg-311', 'NRNX1-seg-311', 'NRNX1-seg-403', 'NRNX1-seg-403', 'NRNX1-seg-403', 'NRNX1-seg-403', 'NRNX1-seg-454', 'NRNX1-seg-454', 'NRNX1-seg-454', 'NRNX1-seg-454', 'NRNX1-seg-525', 'NRNX1-seg-525', 'NRNX1-seg-525', 'NRNX1-seg-525', 'NRNX1-seg-646', 'NRNX1-seg-646', 'NRNX1-seg-646', 'NRNX1-seg-646'], 'NDB_392': ['NRNX1-seg-23', 'NRNX1-seg-23', 'NRNX1-seg-23', 'NRNX1-seg-23', 'NRNX1-seg-106', 'NRNX1-seg-106', 'NRNX1-seg-106', 'NRNX1-seg-106', 'NRNX1-seg-267', 'NRNX1-seg-267', 'NRNX1-seg-267', 'NRNX1-seg-267', 'NRNX1-seg-361', 'NRNX1-seg-361', 'NRNX1-seg-361', 'NRNX1-seg-361', 'NRNX1-seg-423', 'NRNX1-seg-423', 'NRNX1-seg-423', 'NRNX1-seg-423', 'NRNX1-seg-485', 'NRNX1-seg-485', 'NRNX1-seg-485', 'NRNX1-seg-485', 'NRNX1-seg-555', 'NRNX1-seg-555', 'NRNX1-seg-555', 'NRNX1-seg-555', 'NRNX1-seg-605', 'NRNX1-seg-605', 'NRNX1-seg-605', 'NRNX1-seg-605'], 'NDB_513': ['NRNX1-seg-23', 'NRNX1-seg-23', 'NRNX1-seg-23', 'NRNX1-seg-23', 'NRNX1-seg-86', 'NRNX1-seg-86', 'NRNX1-seg-86', 'NRNX1-seg-86', 'NRNX1-seg-127', 'NRNX1-seg-127', 'NRNX1-seg-127', 'NRNX1-seg-127', 'NRNX1-seg-192', 'NRNX1-seg-192', 'NRNX1-seg-192', 'NRNX1-seg-192', 'NRNX1-seg-287', 'NRNX1-seg-287', 'NRNX1-seg-287', 'NRNX1-seg-287', 'NRNX1-seg-351', 'NRNX1-seg-351', 'NRNX1-seg-351', 'NRNX1-seg-351', 'NRNX1-seg-443', 'NRNX1-seg-443', 'NRNX1-seg-443', 'NRNX1-seg-443', 'NRNX1-seg-625', 'NRNX1-seg-625', 'NRNX1-seg-625', 'NRNX1-seg-625', 'NRNX1-seg-681', 'NRNX1-seg-681', 'NRNX1-seg-681', 'NRNX1-seg-681'], 'NDB_759': ['NRNX1-seg-24', 'NRNX1-seg-24', 'NRNX1-seg-24', 'NRNX1-seg-24', 'NRNX1-seg-76', 'NRNX1-seg-76', 'NRNX1-seg-76', 'NRNX1-seg-76', 'NRNX1-seg-183', 'NRNX1-seg-183', 'NRNX1-seg-183', 'NRNX1-seg-183', 'NRNX1-seg-245', 'NRNX1-seg-245', 'NRNX1-seg-245', 'NRNX1-seg-245', 'NRNX1-seg-312', 'NRNX1-seg-312', 'NRNX1-seg-312', 'NRNX1-seg-312', 'NRNX1-seg-394', 'NRNX1-seg-394', 'NRNX1-seg-394', 'NRNX1-seg-394', 'NRNX1-seg-546', 'NRNX1-seg-546', 'NRNX1-seg-546', 'NRNX1-seg-546', 'NRNX1-seg-606', 'NRNX1-seg-606', 'NRNX1-seg-606', 'NRNX1-seg-606', 'NRNX1-seg-669', 'NRNX1-seg-669', 'NRNX1-seg-669', 'NRNX1-seg-669'], 'NDB_713': ['NRNX1-seg-24', 'NRNX1-seg-24', 'NRNX1-seg-24', 'NRNX1-seg-24', 'NRNX1-seg-128', 'NRNX1-seg-128', 'NRNX1-seg-128', 'NRNX1-seg-128', 'NRNX1-seg-223', 'NRNX1-seg-223', 'NRNX1-seg-223', 'NRNX1-seg-223', 'NRNX1-seg-300', 'NRNX1-seg-300', 'NRNX1-seg-300', 'NRNX1-seg-300', 'NRNX1-seg-362', 'NRNX1-seg-362', 'NRNX1-seg-362', 'NRNX1-seg-362', 'NRNX1-seg-414', 'NRNX1-seg-414', 'NRNX1-seg-414', 'NRNX1-seg-414', 'NRNX1-seg-566', 'NRNX1-seg-566', 'NRNX1-seg-566', 'NRNX1-seg-566', 'NRNX1-seg-647', 'NRNX1-seg-647', 'NRNX1-seg-647', 'NRNX1-seg-647'], 'NDB_807': ['NRNX1-seg-24', 'NRNX1-seg-24', 'NRNX1-seg-24', 'NRNX1-seg-24', 'NRNX1-seg-107', 'NRNX1-seg-107', 'NRNX1-seg-107', 'NRNX1-seg-107', 'NRNX1-seg-150', 'NRNX1-seg-150', 'NRNX1-seg-150', 'NRNX1-seg-150', 'NRNX1-seg-278', 'NRNX1-seg-278', 'NRNX1-seg-278', 'NRNX1-seg-278', 'NRNX1-seg-332', 'NRNX1-seg-332', 'NRNX1-seg-332', 'NRNX1-seg-332', 'NRNX1-seg-444', 'NRNX1-seg-444', 'NRNX1-seg-444', 'NRNX1-seg-444', 'NRNX1-seg-506', 'NRNX1-seg-506', 'NRNX1-seg-506', 'NRNX1-seg-506', 'NRNX1-seg-616', 'NRNX1-seg-616', 'NRNX1-seg-616', 'NRNX1-seg-616', 'NRNX1-seg-682', 'NRNX1-seg-682', 'NRNX1-seg-682', 'NRNX1-seg-682'], 'NDB_456': ['NRNX1-seg-25', 'NRNX1-seg-25', 'NRNX1-seg-25', 'NRNX1-seg-25', 'NRNX1-seg-77', 'NRNX1-seg-77', 'NRNX1-seg-77', 'NRNX1-seg-77', 'NRNX1-seg-184', 'NRNX1-seg-184', 'NRNX1-seg-184', 'NRNX1-seg-184', 'NRNX1-seg-246', 'NRNX1-seg-246', 'NRNX1-seg-246', 'NRNX1-seg-246', 'NRNX1-seg-313', 'NRNX1-seg-313', 'NRNX1-seg-313', 'NRNX1-seg-313', 'NRNX1-seg-395', 'NRNX1-seg-395', 'NRNX1-seg-395', 'NRNX1-seg-395', 'NRNX1-seg-547', 'NRNX1-seg-547', 'NRNX1-seg-547', 'NRNX1-seg-547', 'NRNX1-seg-607', 'NRNX1-seg-607', 'NRNX1-seg-607', 'NRNX1-seg-607', 'NRNX1-seg-670', 'NRNX1-seg-670', 'NRNX1-seg-670', 'NRNX1-seg-670'], 'NDB_965': ['NRNX1-seg-25', 'NRNX1-seg-25', 'NRNX1-seg-25', 'NRNX1-seg-25', 'NRNX1-seg-129', 'NRNX1-seg-129', 'NRNX1-seg-129', 'NRNX1-seg-129', 'NRNX1-seg-224', 'NRNX1-seg-224', 'NRNX1-seg-224', 'NRNX1-seg-224', 'NRNX1-seg-301', 'NRNX1-seg-301', 'NRNX1-seg-301', 'NRNX1-seg-301', 'NRNX1-seg-363', 'NRNX1-seg-363', 'NRNX1-seg-363', 'NRNX1-seg-363', 'NRNX1-seg-415', 'NRNX1-seg-415', 'NRNX1-seg-415', 'NRNX1-seg-415', 'NRNX1-seg-567', 'NRNX1-seg-567', 'NRNX1-seg-567', 'NRNX1-seg-567', 'NRNX1-seg-648', 'NRNX1-seg-648', 'NRNX1-seg-648', 'NRNX1-seg-648'], 'NDB_768': ['NRNX1-seg-25', 'NRNX1-seg-25', 'NRNX1-seg-25', 'NRNX1-seg-25', 'NRNX1-seg-108', 'NRNX1-seg-108', 'NRNX1-seg-108', 'NRNX1-seg-108', 'NRNX1-seg-151', 'NRNX1-seg-151', 'NRNX1-seg-151', 'NRNX1-seg-151', 'NRNX1-seg-279', 'NRNX1-seg-279', 'NRNX1-seg-279', 'NRNX1-seg-279', 'NRNX1-seg-333', 'NRNX1-seg-333', 'NRNX1-seg-333', 'NRNX1-seg-333', 'NRNX1-seg-445', 'NRNX1-seg-445', 'NRNX1-seg-445', 'NRNX1-seg-445', 'NRNX1-seg-507', 'NRNX1-seg-507', 'NRNX1-seg-507', 'NRNX1-seg-507', 'NRNX1-seg-617', 'NRNX1-seg-617', 'NRNX1-seg-617', 'NRNX1-seg-617', 'NRNX1-seg-683', 'NRNX1-seg-683', 'NRNX1-seg-683', 'NRNX1-seg-683'], 'NDB_915': ['NRNX1-seg-26', 'NRNX1-seg-26', 'NRNX1-seg-26', 'NRNX1-seg-26', 'NRNX1-seg-79', 'NRNX1-seg-79', 'NRNX1-seg-79', 'NRNX1-seg-79', 'NRNX1-seg-185', 'NRNX1-seg-185', 'NRNX1-seg-185', 'NRNX1-seg-185', 'NRNX1-seg-248', 'NRNX1-seg-248', 'NRNX1-seg-248', 'NRNX1-seg-248', 'NRNX1-seg-314', 'NRNX1-seg-314', 'NRNX1-seg-314', 'NRNX1-seg-314', 'NRNX1-seg-396', 'NRNX1-seg-396', 'NRNX1-seg-396', 'NRNX1-seg-396', 'NRNX1-seg-548', 'NRNX1-seg-548', 'NRNX1-seg-548', 'NRNX1-seg-548', 'NRNX1-seg-608', 'NRNX1-seg-608', 'NRNX1-seg-608', 'NRNX1-seg-608', 'NRNX1-seg-671', 'NRNX1-seg-671', 'NRNX1-seg-671', 'NRNX1-seg-671'], 'NDB_746': ['NRNX1-seg-26', 'NRNX1-seg-26', 'NRNX1-seg-26', 'NRNX1-seg-26', 'NRNX1-seg-130', 'NRNX1-seg-130', 'NRNX1-seg-130', 'NRNX1-seg-130', 'NRNX1-seg-225', 'NRNX1-seg-225', 'NRNX1-seg-225', 'NRNX1-seg-225', 'NRNX1-seg-302', 'NRNX1-seg-302', 'NRNX1-seg-302', 'NRNX1-seg-302', 'NRNX1-seg-364', 'NRNX1-seg-364', 'NRNX1-seg-364', 'NRNX1-seg-364', 'NRNX1-seg-416', 'NRNX1-seg-416', 'NRNX1-seg-416', 'NRNX1-seg-416', 'NRNX1-seg-568', 'NRNX1-seg-568', 'NRNX1-seg-568', 'NRNX1-seg-568', 'NRNX1-seg-649', 'NRNX1-seg-649', 'NRNX1-seg-649', 'NRNX1-seg-649'], 'NDB_516': ['NRNX1-seg-26', 'NRNX1-seg-26', 'NRNX1-seg-26', 'NRNX1-seg-26', 'NRNX1-seg-109', 'NRNX1-seg-109', 'NRNX1-seg-109', 'NRNX1-seg-109', 'NRNX1-seg-152', 'NRNX1-seg-152', 'NRNX1-seg-152', 'NRNX1-seg-152', 'NRNX1-seg-280', 'NRNX1-seg-280', 'NRNX1-seg-280', 'NRNX1-seg-280', 'NRNX1-seg-334', 'NRNX1-seg-334', 'NRNX1-seg-334', 'NRNX1-seg-334', 'NRNX1-seg-446', 'NRNX1-seg-446', 'NRNX1-seg-446', 'NRNX1-seg-446', 'NRNX1-seg-508', 'NRNX1-seg-508', 'NRNX1-seg-508', 'NRNX1-seg-508', 'NRNX1-seg-618', 'NRNX1-seg-618', 'NRNX1-seg-618', 'NRNX1-seg-618', 'NRNX1-seg-684', 'NRNX1-seg-684', 'NRNX1-seg-684', 'NRNX1-seg-684'], 'NDB_957': ['NRNX1-seg-27', 'NRNX1-seg-27', 'NRNX1-seg-27', 'NRNX1-seg-27', 'NRNX1-seg-80', 'NRNX1-seg-80', 'NRNX1-seg-80', 'NRNX1-seg-80', 'NRNX1-seg-186', 'NRNX1-seg-186', 'NRNX1-seg-186', 'NRNX1-seg-186', 'NRNX1-seg-249', 'NRNX1-seg-249', 'NRNX1-seg-249', 'NRNX1-seg-249', 'NRNX1-seg-315', 'NRNX1-seg-315', 'NRNX1-seg-315', 'NRNX1-seg-315', 'NRNX1-seg-397', 'NRNX1-seg-397', 'NRNX1-seg-397', 'NRNX1-seg-397', 'NRNX1-seg-549', 'NRNX1-seg-549', 'NRNX1-seg-549', 'NRNX1-seg-549', 'NRNX1-seg-609', 'NRNX1-seg-609', 'NRNX1-seg-609', 'NRNX1-seg-609', 'NRNX1-seg-672', 'NRNX1-seg-672', 'NRNX1-seg-672', 'NRNX1-seg-672'], 'NDB_455': ['NRNX1-seg-27', 'NRNX1-seg-27', 'NRNX1-seg-27', 'NRNX1-seg-27', 'NRNX1-seg-131', 'NRNX1-seg-131', 'NRNX1-seg-131', 'NRNX1-seg-131', 'NRNX1-seg-226', 'NRNX1-seg-226', 'NRNX1-seg-226', 'NRNX1-seg-226', 'NRNX1-seg-303', 'NRNX1-seg-303', 'NRNX1-seg-303', 'NRNX1-seg-303', 'NRNX1-seg-365', 'NRNX1-seg-365', 'NRNX1-seg-365', 'NRNX1-seg-365', 'NRNX1-seg-417', 'NRNX1-seg-417', 'NRNX1-seg-417', 'NRNX1-seg-417', 'NRNX1-seg-569', 'NRNX1-seg-569', 'NRNX1-seg-569', 'NRNX1-seg-569', 'NRNX1-seg-650', 'NRNX1-seg-650', 'NRNX1-seg-650', 'NRNX1-seg-650'], 'NDB_657': ['NRNX1-seg-27', 'NRNX1-seg-27', 'NRNX1-seg-27', 'NRNX1-seg-27', 'NRNX1-seg-110', 'NRNX1-seg-110', 'NRNX1-seg-110', 'NRNX1-seg-110', 'NRNX1-seg-154', 'NRNX1-seg-154', 'NRNX1-seg-154', 'NRNX1-seg-154', 'NRNX1-seg-281', 'NRNX1-seg-281', 'NRNX1-seg-281', 'NRNX1-seg-281', 'NRNX1-seg-335', 'NRNX1-seg-335', 'NRNX1-seg-335', 'NRNX1-seg-335', 'NRNX1-seg-447', 'NRNX1-seg-447', 'NRNX1-seg-447', 'NRNX1-seg-447', 'NRNX1-seg-509', 'NRNX1-seg-509', 'NRNX1-seg-509', 'NRNX1-seg-509', 'NRNX1-seg-619', 'NRNX1-seg-619', 'NRNX1-seg-619', 'NRNX1-seg-619', 'NRNX1-seg-685', 'NRNX1-seg-685', 'NRNX1-seg-685', 'NRNX1-seg-685'], 'NDB_405': ['NRNX1-seg-28', 'NRNX1-seg-28', 'NRNX1-seg-28', 'NRNX1-seg-28', 'NRNX1-seg-81', 'NRNX1-seg-81', 'NRNX1-seg-81', 'NRNX1-seg-81', 'NRNX1-seg-187', 'NRNX1-seg-187', 'NRNX1-seg-187', 'NRNX1-seg-187', 'NRNX1-seg-251', 'NRNX1-seg-251', 'NRNX1-seg-251', 'NRNX1-seg-251', 'NRNX1-seg-316', 'NRNX1-seg-316', 'NRNX1-seg-316', 'NRNX1-seg-316', 'NRNX1-seg-398', 'NRNX1-seg-398', 'NRNX1-seg-398', 'NRNX1-seg-398', 'NRNX1-seg-550', 'NRNX1-seg-550', 'NRNX1-seg-550', 'NRNX1-seg-550', 'NRNX1-seg-610', 'NRNX1-seg-610', 'NRNX1-seg-610', 'NRNX1-seg-610', 'NRNX1-seg-674', 'NRNX1-seg-674', 'NRNX1-seg-674', 'NRNX1-seg-674'], 'NDB_536': ['NRNX1-seg-28', 'NRNX1-seg-28', 'NRNX1-seg-28', 'NRNX1-seg-28', 'NRNX1-seg-133', 'NRNX1-seg-133', 'NRNX1-seg-133', 'NRNX1-seg-133', 'NRNX1-seg-227', 'NRNX1-seg-227', 'NRNX1-seg-227', 'NRNX1-seg-227', 'NRNX1-seg-304', 'NRNX1-seg-304', 'NRNX1-seg-304', 'NRNX1-seg-304', 'NRNX1-seg-366', 'NRNX1-seg-366', 'NRNX1-seg-366', 'NRNX1-seg-366', 'NRNX1-seg-418', 'NRNX1-seg-418', 'NRNX1-seg-418', 'NRNX1-seg-418', 'NRNX1-seg-570', 'NRNX1-seg-570', 'NRNX1-seg-570', 'NRNX1-seg-570', 'NRNX1-seg-651', 'NRNX1-seg-651', 'NRNX1-seg-651', 'NRNX1-seg-651'], 'NDB_771': ['NRNX1-seg-28', 'NRNX1-seg-28', 'NRNX1-seg-28', 'NRNX1-seg-28', 'NRNX1-seg-111', 'NRNX1-seg-111', 'NRNX1-seg-111', 'NRNX1-seg-111', 'NRNX1-seg-155', 'NRNX1-seg-155', 'NRNX1-seg-155', 'NRNX1-seg-155', 'NRNX1-seg-282', 'NRNX1-seg-282', 'NRNX1-seg-282', 'NRNX1-seg-282', 'NRNX1-seg-336', 'NRNX1-seg-336', 'NRNX1-seg-336', 'NRNX1-seg-336', 'NRNX1-seg-448', 'NRNX1-seg-448', 'NRNX1-seg-448', 'NRNX1-seg-448', 'NRNX1-seg-510', 'NRNX1-seg-510', 'NRNX1-seg-510', 'NRNX1-seg-510', 'NRNX1-seg-620', 'NRNX1-seg-620', 'NRNX1-seg-620', 'NRNX1-seg-620', 'NRNX1-seg-686', 'NRNX1-seg-686', 'NRNX1-seg-686', 'NRNX1-seg-686'], 'NDB_390': ['NRNX1-seg-30', 'NRNX1-seg-30', 'NRNX1-seg-30', 'NRNX1-seg-30', 'NRNX1-seg-82', 'NRNX1-seg-82', 'NRNX1-seg-82', 'NRNX1-seg-82', 'NRNX1-seg-188', 'NRNX1-seg-188', 'NRNX1-seg-188', 'NRNX1-seg-188', 'NRNX1-seg-252', 'NRNX1-seg-252', 'NRNX1-seg-252', 'NRNX1-seg-252', 'NRNX1-seg-317', 'NRNX1-seg-317', 'NRNX1-seg-317', 'NRNX1-seg-317', 'NRNX1-seg-399', 'NRNX1-seg-399', 'NRNX1-seg-399', 'NRNX1-seg-399', 'NRNX1-seg-551', 'NRNX1-seg-551', 'NRNX1-seg-551', 'NRNX1-seg-551', 'NRNX1-seg-611', 'NRNX1-seg-611', 'NRNX1-seg-611', 'NRNX1-seg-611', 'NRNX1-seg-675', 'NRNX1-seg-675', 'NRNX1-seg-675', 'NRNX1-seg-675'], 'NDB_887': ['NRNX1-seg-30', 'NRNX1-seg-30', 'NRNX1-seg-30', 'NRNX1-seg-30', 'NRNX1-seg-134', 'NRNX1-seg-134', 'NRNX1-seg-134', 'NRNX1-seg-134', 'NRNX1-seg-228', 'NRNX1-seg-228', 'NRNX1-seg-228', 'NRNX1-seg-228', 'NRNX1-seg-305', 'NRNX1-seg-305', 'NRNX1-seg-305', 'NRNX1-seg-305', 'NRNX1-seg-367', 'NRNX1-seg-367', 'NRNX1-seg-367', 'NRNX1-seg-367', 'NRNX1-seg-419', 'NRNX1-seg-419', 'NRNX1-seg-419', 'NRNX1-seg-419', 'NRNX1-seg-571', 'NRNX1-seg-571', 'NRNX1-seg-571', 'NRNX1-seg-571', 'NRNX1-seg-652', 'NRNX1-seg-652', 'NRNX1-seg-652', 'NRNX1-seg-652'], 'NDB_615': ['NRNX1-seg-30', 'NRNX1-seg-30', 'NRNX1-seg-30', 'NRNX1-seg-30', 'NRNX1-seg-112', 'NRNX1-seg-112', 'NRNX1-seg-112', 'NRNX1-seg-112', 'NRNX1-seg-156', 'NRNX1-seg-156', 'NRNX1-seg-156', 'NRNX1-seg-156', 'NRNX1-seg-283', 'NRNX1-seg-283', 'NRNX1-seg-283', 'NRNX1-seg-283', 'NRNX1-seg-337', 'NRNX1-seg-337', 'NRNX1-seg-337', 'NRNX1-seg-337', 'NRNX1-seg-449', 'NRNX1-seg-449', 'NRNX1-seg-449', 'NRNX1-seg-449', 'NRNX1-seg-511', 'NRNX1-seg-511', 'NRNX1-seg-511', 'NRNX1-seg-511', 'NRNX1-seg-621', 'NRNX1-seg-621', 'NRNX1-seg-621', 'NRNX1-seg-621', 'NRNX1-seg-687', 'NRNX1-seg-687', 'NRNX1-seg-687', 'NRNX1-seg-687'], 'NDB_846': ['NRNX1-seg-31', 'NRNX1-seg-31', 'NRNX1-seg-31', 'NRNX1-seg-31', 'NRNX1-seg-83', 'NRNX1-seg-83', 'NRNX1-seg-83', 'NRNX1-seg-83', 'NRNX1-seg-189', 'NRNX1-seg-189', 'NRNX1-seg-189', 'NRNX1-seg-189', 'NRNX1-seg-253', 'NRNX1-seg-253', 'NRNX1-seg-253', 'NRNX1-seg-253', 'NRNX1-seg-318', 'NRNX1-seg-318', 'NRNX1-seg-318', 'NRNX1-seg-318', 'NRNX1-seg-400', 'NRNX1-seg-400', 'NRNX1-seg-400', 'NRNX1-seg-400', 'NRNX1-seg-552', 'NRNX1-seg-552', 'NRNX1-seg-552', 'NRNX1-seg-552', 'NRNX1-seg-612', 'NRNX1-seg-612', 'NRNX1-seg-612', 'NRNX1-seg-612', 'NRNX1-seg-676', 'NRNX1-seg-676', 'NRNX1-seg-676', 'NRNX1-seg-676'], 'NDB_896': ['NRNX1-seg-31', 'NRNX1-seg-31', 'NRNX1-seg-31', 'NRNX1-seg-31', 'NRNX1-seg-135', 'NRNX1-seg-135', 'NRNX1-seg-135', 'NRNX1-seg-135', 'NRNX1-seg-229', 'NRNX1-seg-229', 'NRNX1-seg-229', 'NRNX1-seg-229', 'NRNX1-seg-306', 'NRNX1-seg-306', 'NRNX1-seg-306', 'NRNX1-seg-306', 'NRNX1-seg-368', 'NRNX1-seg-368', 'NRNX1-seg-368', 'NRNX1-seg-368', 'NRNX1-seg-420', 'NRNX1-seg-420', 'NRNX1-seg-420', 'NRNX1-seg-420', 'NRNX1-seg-572', 'NRNX1-seg-572', 'NRNX1-seg-572', 'NRNX1-seg-572', 'NRNX1-seg-653', 'NRNX1-seg-653', 'NRNX1-seg-653', 'NRNX1-seg-653'], 'NDB_453': ['NRNX1-seg-31', 'NRNX1-seg-31', 'NRNX1-seg-31', 'NRNX1-seg-31', 'NRNX1-seg-113', 'NRNX1-seg-113', 'NRNX1-seg-113', 'NRNX1-seg-113', 'NRNX1-seg-157', 'NRNX1-seg-157', 'NRNX1-seg-157', 'NRNX1-seg-157', 'NRNX1-seg-284', 'NRNX1-seg-284', 'NRNX1-seg-284', 'NRNX1-seg-284', 'NRNX1-seg-338', 'NRNX1-seg-338', 'NRNX1-seg-338', 'NRNX1-seg-338', 'NRNX1-seg-450', 'NRNX1-seg-450', 'NRNX1-seg-450', 'NRNX1-seg-450', 'NRNX1-seg-512', 'NRNX1-seg-512', 'NRNX1-seg-512', 'NRNX1-seg-512', 'NRNX1-seg-622', 'NRNX1-seg-622', 'NRNX1-seg-622', 'NRNX1-seg-622', 'NRNX1-seg-688', 'NRNX1-seg-688', 'NRNX1-seg-688', 'NRNX1-seg-688'], 'NDB_687': ['NRNX1-seg-32', 'NRNX1-seg-32', 'NRNX1-seg-32', 'NRNX1-seg-32', 'NRNX1-seg-84', 'NRNX1-seg-84', 'NRNX1-seg-84', 'NRNX1-seg-84', 'NRNX1-seg-190', 'NRNX1-seg-190', 'NRNX1-seg-190', 'NRNX1-seg-190', 'NRNX1-seg-254', 'NRNX1-seg-254', 'NRNX1-seg-254', 'NRNX1-seg-254', 'NRNX1-seg-319', 'NRNX1-seg-319', 'NRNX1-seg-319', 'NRNX1-seg-319', 'NRNX1-seg-401', 'NRNX1-seg-401', 'NRNX1-seg-401', 'NRNX1-seg-401', 'NRNX1-seg-553', 'NRNX1-seg-553', 'NRNX1-seg-553', 'NRNX1-seg-553', 'NRNX1-seg-613', 'NRNX1-seg-613', 'NRNX1-seg-613', 'NRNX1-seg-613', 'NRNX1-seg-678', 'NRNX1-seg-678', 'NRNX1-seg-678', 'NRNX1-seg-678'], 'NDB_356': ['NRNX1-seg-32', 'NRNX1-seg-32', 'NRNX1-seg-32', 'NRNX1-seg-32', 'NRNX1-seg-136', 'NRNX1-seg-136', 'NRNX1-seg-136', 'NRNX1-seg-136', 'NRNX1-seg-230', 'NRNX1-seg-230', 'NRNX1-seg-230', 'NRNX1-seg-230', 'NRNX1-seg-307', 'NRNX1-seg-307', 'NRNX1-seg-307', 'NRNX1-seg-307', 'NRNX1-seg-369', 'NRNX1-seg-369', 'NRNX1-seg-369', 'NRNX1-seg-369', 'NRNX1-seg-421', 'NRNX1-seg-421', 'NRNX1-seg-421', 'NRNX1-seg-421', 'NRNX1-seg-573', 'NRNX1-seg-573', 'NRNX1-seg-573', 'NRNX1-seg-573', 'NRNX1-seg-654', 'NRNX1-seg-654', 'NRNX1-seg-654', 'NRNX1-seg-654'], 'NDB_828': ['NRNX1-seg-32', 'NRNX1-seg-32', 'NRNX1-seg-32', 'NRNX1-seg-32', 'NRNX1-seg-114', 'NRNX1-seg-114', 'NRNX1-seg-114', 'NRNX1-seg-114', 'NRNX1-seg-158', 'NRNX1-seg-158', 'NRNX1-seg-158', 'NRNX1-seg-158', 'NRNX1-seg-285', 'NRNX1-seg-285', 'NRNX1-seg-285', 'NRNX1-seg-285', 'NRNX1-seg-339', 'NRNX1-seg-339', 'NRNX1-seg-339', 'NRNX1-seg-339', 'NRNX1-seg-452', 'NRNX1-seg-452', 'NRNX1-seg-452', 'NRNX1-seg-452', 'NRNX1-seg-513', 'NRNX1-seg-513', 'NRNX1-seg-513', 'NRNX1-seg-513', 'NRNX1-seg-623', 'NRNX1-seg-623', 'NRNX1-seg-623', 'NRNX1-seg-623', 'NRNX1-seg-689', 'NRNX1-seg-689', 'NRNX1-seg-689', 'NRNX1-seg-689'], 'NDB_862': ['NRNX1-seg-33', 'NRNX1-seg-33', 'NRNX1-seg-33', 'NRNX1-seg-33', 'NRNX1-seg-137', 'NRNX1-seg-137', 'NRNX1-seg-137', 'NRNX1-seg-137', 'NRNX1-seg-255', 'NRNX1-seg-255', 'NRNX1-seg-255', 'NRNX1-seg-255', 'NRNX1-seg-330', 'NRNX1-seg-330', 'NRNX1-seg-330', 'NRNX1-seg-330', 'NRNX1-seg-422', 'NRNX1-seg-422', 'NRNX1-seg-422', 'NRNX1-seg-422', 'NRNX1-seg-534', 'NRNX1-seg-534', 'NRNX1-seg-534', 'NRNX1-seg-534', 'NRNX1-seg-594', 'NRNX1-seg-594', 'NRNX1-seg-594', 'NRNX1-seg-594', 'NRNX1-seg-690', 'NRNX1-seg-690', 'NRNX1-seg-690', 'NRNX1-seg-690'], 'NDB_491': ['NRNX1-seg-33', 'NRNX1-seg-33', 'NRNX1-seg-33', 'NRNX1-seg-33', 'NRNX1-seg-181', 'NRNX1-seg-181', 'NRNX1-seg-181', 'NRNX1-seg-181', 'NRNX1-seg-231', 'NRNX1-seg-231', 'NRNX1-seg-231', 'NRNX1-seg-231', 'NRNX1-seg-320', 'NRNX1-seg-320', 'NRNX1-seg-320', 'NRNX1-seg-320', 'NRNX1-seg-494', 'NRNX1-seg-494', 'NRNX1-seg-494', 'NRNX1-seg-494', 'NRNX1-seg-544', 'NRNX1-seg-544', 'NRNX1-seg-544', 'NRNX1-seg-544', 'NRNX1-seg-584', 'NRNX1-seg-584', 'NRNX1-seg-584', 'NRNX1-seg-584', 'NRNX1-seg-667', 'NRNX1-seg-667', 'NRNX1-seg-667', 'NRNX1-seg-667'], 'NDB_694': ['NRNX1-seg-33', 'NRNX1-seg-33', 'NRNX1-seg-33', 'NRNX1-seg-33', 'NRNX1-seg-85', 'NRNX1-seg-85', 'NRNX1-seg-85', 'NRNX1-seg-85', 'NRNX1-seg-171', 'NRNX1-seg-171', 'NRNX1-seg-171', 'NRNX1-seg-171', 'NRNX1-seg-211', 'NRNX1-seg-211', 'NRNX1-seg-211', 'NRNX1-seg-211', 'NRNX1-seg-276', 'NRNX1-seg-276', 'NRNX1-seg-276', 'NRNX1-seg-276', 'NRNX1-seg-392', 'NRNX1-seg-392', 'NRNX1-seg-392', 'NRNX1-seg-392', 'NRNX1-seg-453', 'NRNX1-seg-453', 'NRNX1-seg-453', 'NRNX1-seg-453', 'NRNX1-seg-614', 'NRNX1-seg-614', 'NRNX1-seg-614', 'NRNX1-seg-614', 'NRNX1-seg-655', 'NRNX1-seg-655', 'NRNX1-seg-655', 'NRNX1-seg-655'], 'NDB_418': ['NRNX1-seg-34', 'NRNX1-seg-34', 'NRNX1-seg-34', 'NRNX1-seg-34', 'NRNX1-seg-138', 'NRNX1-seg-138', 'NRNX1-seg-138', 'NRNX1-seg-138', 'NRNX1-seg-256', 'NRNX1-seg-256', 'NRNX1-seg-256', 'NRNX1-seg-256', 'NRNX1-seg-331', 'NRNX1-seg-331', 'NRNX1-seg-331', 'NRNX1-seg-331', 'NRNX1-seg-423', 'NRNX1-seg-423', 'NRNX1-seg-423', 'NRNX1-seg-423', 'NRNX1-seg-535', 'NRNX1-seg-535', 'NRNX1-seg-535', 'NRNX1-seg-535', 'NRNX1-seg-595', 'NRNX1-seg-595', 'NRNX1-seg-595', 'NRNX1-seg-595', 'NRNX1-seg-691', 'NRNX1-seg-691', 'NRNX1-seg-691', 'NRNX1-seg-691'], 'NDB_1028': ['NRNX1-seg-34', 'NRNX1-seg-34', 'NRNX1-seg-34', 'NRNX1-seg-34', 'NRNX1-seg-182', 'NRNX1-seg-182', 'NRNX1-seg-182', 'NRNX1-seg-182', 'NRNX1-seg-232', 'NRNX1-seg-232', 'NRNX1-seg-232', 'NRNX1-seg-232', 'NRNX1-seg-321', 'NRNX1-seg-321', 'NRNX1-seg-321', 'NRNX1-seg-321', 'NRNX1-seg-495', 'NRNX1-seg-495', 'NRNX1-seg-495', 'NRNX1-seg-495', 'NRNX1-seg-545', 'NRNX1-seg-545', 'NRNX1-seg-545', 'NRNX1-seg-545', 'NRNX1-seg-585', 'NRNX1-seg-585', 'NRNX1-seg-585', 'NRNX1-seg-585', 'NRNX1-seg-668', 'NRNX1-seg-668', 'NRNX1-seg-668', 'NRNX1-seg-668'], 'NDB_622': ['NRNX1-seg-34', 'NRNX1-seg-34', 'NRNX1-seg-34', 'NRNX1-seg-34', 'NRNX1-seg-86', 'NRNX1-seg-86', 'NRNX1-seg-86', 'NRNX1-seg-86', 'NRNX1-seg-172', 'NRNX1-seg-172', 'NRNX1-seg-172', 'NRNX1-seg-172', 'NRNX1-seg-212', 'NRNX1-seg-212', 'NRNX1-seg-212', 'NRNX1-seg-212', 'NRNX1-seg-277', 'NRNX1-seg-277', 'NRNX1-seg-277', 'NRNX1-seg-277', 'NRNX1-seg-393', 'NRNX1-seg-393', 'NRNX1-seg-393', 'NRNX1-seg-393', 'NRNX1-seg-454', 'NRNX1-seg-454', 'NRNX1-seg-454', 'NRNX1-seg-454', 'NRNX1-seg-615', 'NRNX1-seg-615', 'NRNX1-seg-615', 'NRNX1-seg-615', 'NRNX1-seg-656', 'NRNX1-seg-656', 'NRNX1-seg-656', 'NRNX1-seg-656'], 'NDB_826': ['NRNX1-seg-35', 'NRNX1-seg-35', 'NRNX1-seg-35', 'NRNX1-seg-35', 'NRNX1-seg-139', 'NRNX1-seg-139', 'NRNX1-seg-139', 'NRNX1-seg-139', 'NRNX1-seg-203', 'NRNX1-seg-203', 'NRNX1-seg-203', 'NRNX1-seg-203', 'NRNX1-seg-312', 'NRNX1-seg-312', 'NRNX1-seg-312', 'NRNX1-seg-312', 'NRNX1-seg-414', 'NRNX1-seg-414', 'NRNX1-seg-414', 'NRNX1-seg-414', 'NRNX1-seg-476', 'NRNX1-seg-476', 'NRNX1-seg-476', 'NRNX1-seg-476', 'NRNX1-seg-536', 'NRNX1-seg-536', 'NRNX1-seg-536', 'NRNX1-seg-536', 'NRNX1-seg-586', 'NRNX1-seg-586', 'NRNX1-seg-586', 'NRNX1-seg-586'], 'NDB_578': ['NRNX1-seg-35', 'NRNX1-seg-35', 'NRNX1-seg-35', 'NRNX1-seg-35', 'NRNX1-seg-245', 'NRNX1-seg-245', 'NRNX1-seg-245', 'NRNX1-seg-245', 'NRNX1-seg-322', 'NRNX1-seg-322', 'NRNX1-seg-322', 'NRNX1-seg-322', 'NRNX1-seg-372', 'NRNX1-seg-372', 'NRNX1-seg-372', 'NRNX1-seg-372', 'NRNX1-seg-424', 'NRNX1-seg-424', 'NRNX1-seg-424', 'NRNX1-seg-424', 'NRNX1-seg-516', 'NRNX1-seg-516', 'NRNX1-seg-516', 'NRNX1-seg-516', 'NRNX1-seg-596', 'NRNX1-seg-596', 'NRNX1-seg-596', 'NRNX1-seg-596', 'NRNX1-seg-682', 'NRNX1-seg-682', 'NRNX1-seg-682', 'NRNX1-seg-682'], 'NDB_710': ['NRNX1-seg-35', 'NRNX1-seg-35', 'NRNX1-seg-35', 'NRNX1-seg-35', 'NRNX1-seg-118', 'NRNX1-seg-118', 'NRNX1-seg-118', 'NRNX1-seg-118', 'NRNX1-seg-183', 'NRNX1-seg-183', 'NRNX1-seg-183', 'NRNX1-seg-183', 'NRNX1-seg-257', 'NRNX1-seg-257', 'NRNX1-seg-257', 'NRNX1-seg-257', 'NRNX1-seg-384', 'NRNX1-seg-384', 'NRNX1-seg-384', 'NRNX1-seg-384', 'NRNX1-seg-455', 'NRNX1-seg-455', 'NRNX1-seg-455', 'NRNX1-seg-455', 'NRNX1-seg-506', 'NRNX1-seg-506', 'NRNX1-seg-506', 'NRNX1-seg-506', 'NRNX1-seg-556', 'NRNX1-seg-556', 'NRNX1-seg-556', 'NRNX1-seg-556', 'NRNX1-seg-692', 'NRNX1-seg-692', 'NRNX1-seg-692', 'NRNX1-seg-692'], 'NDB_760': ['NRNX1-seg-36', 'NRNX1-seg-36', 'NRNX1-seg-36', 'NRNX1-seg-36', 'NRNX1-seg-140', 'NRNX1-seg-140', 'NRNX1-seg-140', 'NRNX1-seg-140', 'NRNX1-seg-204', 'NRNX1-seg-204', 'NRNX1-seg-204', 'NRNX1-seg-204', 'NRNX1-seg-313', 'NRNX1-seg-313', 'NRNX1-seg-313', 'NRNX1-seg-313', 'NRNX1-seg-415', 'NRNX1-seg-415', 'NRNX1-seg-415', 'NRNX1-seg-415', 'NRNX1-seg-477', 'NRNX1-seg-477', 'NRNX1-seg-477', 'NRNX1-seg-477', 'NRNX1-seg-537', 'NRNX1-seg-537', 'NRNX1-seg-537', 'NRNX1-seg-537', 'NRNX1-seg-587', 'NRNX1-seg-587', 'NRNX1-seg-587', 'NRNX1-seg-587'], 'NDB_815': ['NRNX1-seg-36', 'NRNX1-seg-36', 'NRNX1-seg-36', 'NRNX1-seg-36', 'NRNX1-seg-246', 'NRNX1-seg-246', 'NRNX1-seg-246', 'NRNX1-seg-246', 'NRNX1-seg-323', 'NRNX1-seg-323', 'NRNX1-seg-323', 'NRNX1-seg-323', 'NRNX1-seg-373', 'NRNX1-seg-373', 'NRNX1-seg-373', 'NRNX1-seg-373', 'NRNX1-seg-425', 'NRNX1-seg-425', 'NRNX1-seg-425', 'NRNX1-seg-425', 'NRNX1-seg-517', 'NRNX1-seg-517', 'NRNX1-seg-517', 'NRNX1-seg-517', 'NRNX1-seg-597', 'NRNX1-seg-597', 'NRNX1-seg-597', 'NRNX1-seg-597', 'NRNX1-seg-683', 'NRNX1-seg-683', 'NRNX1-seg-683', 'NRNX1-seg-683'], 'NDB_554': ['NRNX1-seg-36', 'NRNX1-seg-36', 'NRNX1-seg-36', 'NRNX1-seg-36', 'NRNX1-seg-119', 'NRNX1-seg-119', 'NRNX1-seg-119', 'NRNX1-seg-119', 'NRNX1-seg-184', 'NRNX1-seg-184', 'NRNX1-seg-184', 'NRNX1-seg-184', 'NRNX1-seg-258', 'NRNX1-seg-258', 'NRNX1-seg-258', 'NRNX1-seg-258', 'NRNX1-seg-385', 'NRNX1-seg-385', 'NRNX1-seg-385', 'NRNX1-seg-385', 'NRNX1-seg-456', 'NRNX1-seg-456', 'NRNX1-seg-456', 'NRNX1-seg-456', 'NRNX1-seg-507', 'NRNX1-seg-507', 'NRNX1-seg-507', 'NRNX1-seg-507', 'NRNX1-seg-557', 'NRNX1-seg-557', 'NRNX1-seg-557', 'NRNX1-seg-557', 'NRNX1-seg-693', 'NRNX1-seg-693', 'NRNX1-seg-693', 'NRNX1-seg-693'], 'NDB_466': ['NRNX1-seg-37', 'NRNX1-seg-37', 'NRNX1-seg-37', 'NRNX1-seg-37', 'NRNX1-seg-141', 'NRNX1-seg-141', 'NRNX1-seg-141', 'NRNX1-seg-141', 'NRNX1-seg-205', 'NRNX1-seg-205', 'NRNX1-seg-205', 'NRNX1-seg-205', 'NRNX1-seg-314', 'NRNX1-seg-314', 'NRNX1-seg-314', 'NRNX1-seg-314', 'NRNX1-seg-416', 'NRNX1-seg-416', 'NRNX1-seg-416', 'NRNX1-seg-416', 'NRNX1-seg-478', 'NRNX1-seg-478', 'NRNX1-seg-478', 'NRNX1-seg-478', 'NRNX1-seg-538', 'NRNX1-seg-538', 'NRNX1-seg-538', 'NRNX1-seg-538', 'NRNX1-seg-588', 'NRNX1-seg-588', 'NRNX1-seg-588', 'NRNX1-seg-588'], 'NDB_974': ['NRNX1-seg-37', 'NRNX1-seg-37', 'NRNX1-seg-37', 'NRNX1-seg-37', 'NRNX1-seg-248', 'NRNX1-seg-248', 'NRNX1-seg-248', 'NRNX1-seg-248', 'NRNX1-seg-324', 'NRNX1-seg-324', 'NRNX1-seg-324', 'NRNX1-seg-324', 'NRNX1-seg-374', 'NRNX1-seg-374', 'NRNX1-seg-374', 'NRNX1-seg-374', 'NRNX1-seg-426', 'NRNX1-seg-426', 'NRNX1-seg-426', 'NRNX1-seg-426', 'NRNX1-seg-518', 'NRNX1-seg-518', 'NRNX1-seg-518', 'NRNX1-seg-518', 'NRNX1-seg-598', 'NRNX1-seg-598', 'NRNX1-seg-598', 'NRNX1-seg-598', 'NRNX1-seg-684', 'NRNX1-seg-684', 'NRNX1-seg-684', 'NRNX1-seg-684'], 'NDB_812': ['NRNX1-seg-37', 'NRNX1-seg-37', 'NRNX1-seg-37', 'NRNX1-seg-37', 'NRNX1-seg-120', 'NRNX1-seg-120', 'NRNX1-seg-120', 'NRNX1-seg-120', 'NRNX1-seg-185', 'NRNX1-seg-185', 'NRNX1-seg-185', 'NRNX1-seg-185', 'NRNX1-seg-259', 'NRNX1-seg-259', 'NRNX1-seg-259', 'NRNX1-seg-259', 'NRNX1-seg-386', 'NRNX1-seg-386', 'NRNX1-seg-386', 'NRNX1-seg-386', 'NRNX1-seg-457', 'NRNX1-seg-457', 'NRNX1-seg-457', 'NRNX1-seg-457', 'NRNX1-seg-508', 'NRNX1-seg-508', 'NRNX1-seg-508', 'NRNX1-seg-508', 'NRNX1-seg-558', 'NRNX1-seg-558', 'NRNX1-seg-558', 'NRNX1-seg-558', 'NRNX1-seg-694', 'NRNX1-seg-694', 'NRNX1-seg-694', 'NRNX1-seg-694'], 'NDB_529': ['NRNX1-seg-38', 'NRNX1-seg-38', 'NRNX1-seg-38', 'NRNX1-seg-38', 'NRNX1-seg-142', 'NRNX1-seg-142', 'NRNX1-seg-142', 'NRNX1-seg-142', 'NRNX1-seg-206', 'NRNX1-seg-206', 'NRNX1-seg-206', 'NRNX1-seg-206', 'NRNX1-seg-315', 'NRNX1-seg-315', 'NRNX1-seg-315', 'NRNX1-seg-315', 'NRNX1-seg-417', 'NRNX1-seg-417', 'NRNX1-seg-417', 'NRNX1-seg-417', 'NRNX1-seg-479', 'NRNX1-seg-479', 'NRNX1-seg-479', 'NRNX1-seg-479', 'NRNX1-seg-539', 'NRNX1-seg-539', 'NRNX1-seg-539', 'NRNX1-seg-539', 'NRNX1-seg-589', 'NRNX1-seg-589', 'NRNX1-seg-589', 'NRNX1-seg-589'], 'NDB_443': ['NRNX1-seg-38', 'NRNX1-seg-38', 'NRNX1-seg-38', 'NRNX1-seg-38', 'NRNX1-seg-249', 'NRNX1-seg-249', 'NRNX1-seg-249', 'NRNX1-seg-249', 'NRNX1-seg-325', 'NRNX1-seg-325', 'NRNX1-seg-325', 'NRNX1-seg-325', 'NRNX1-seg-375', 'NRNX1-seg-375', 'NRNX1-seg-375', 'NRNX1-seg-375', 'NRNX1-seg-427', 'NRNX1-seg-427', 'NRNX1-seg-427', 'NRNX1-seg-427', 'NRNX1-seg-519', 'NRNX1-seg-519', 'NRNX1-seg-519', 'NRNX1-seg-519', 'NRNX1-seg-599', 'NRNX1-seg-599', 'NRNX1-seg-599', 'NRNX1-seg-599', 'NRNX1-seg-685', 'NRNX1-seg-685', 'NRNX1-seg-685', 'NRNX1-seg-685'], 'NDB_698': ['NRNX1-seg-38', 'NRNX1-seg-38', 'NRNX1-seg-38', 'NRNX1-seg-38', 'NRNX1-seg-121', 'NRNX1-seg-121', 'NRNX1-seg-121', 'NRNX1-seg-121', 'NRNX1-seg-186', 'NRNX1-seg-186', 'NRNX1-seg-186', 'NRNX1-seg-186', 'NRNX1-seg-260', 'NRNX1-seg-260', 'NRNX1-seg-260', 'NRNX1-seg-260', 'NRNX1-seg-387', 'NRNX1-seg-387', 'NRNX1-seg-387', 'NRNX1-seg-387', 'NRNX1-seg-458', 'NRNX1-seg-458', 'NRNX1-seg-458', 'NRNX1-seg-458', 'NRNX1-seg-509', 'NRNX1-seg-509', 'NRNX1-seg-509', 'NRNX1-seg-509', 'NRNX1-seg-559', 'NRNX1-seg-559', 'NRNX1-seg-559', 'NRNX1-seg-559', 'NRNX1-seg-695', 'NRNX1-seg-695', 'NRNX1-seg-695', 'NRNX1-seg-695'], 'NDB_715': ['NRNX1-seg-39', 'NRNX1-seg-39', 'NRNX1-seg-39', 'NRNX1-seg-39', 'NRNX1-seg-143', 'NRNX1-seg-143', 'NRNX1-seg-143', 'NRNX1-seg-143', 'NRNX1-seg-207', 'NRNX1-seg-207', 'NRNX1-seg-207', 'NRNX1-seg-207', 'NRNX1-seg-316', 'NRNX1-seg-316', 'NRNX1-seg-316', 'NRNX1-seg-316', 'NRNX1-seg-418', 'NRNX1-seg-418', 'NRNX1-seg-418', 'NRNX1-seg-418', 'NRNX1-seg-480', 'NRNX1-seg-480', 'NRNX1-seg-480', 'NRNX1-seg-480', 'NRNX1-seg-540', 'NRNX1-seg-540', 'NRNX1-seg-540', 'NRNX1-seg-540', 'NRNX1-seg-590', 'NRNX1-seg-590', 'NRNX1-seg-590', 'NRNX1-seg-590'], 'NDB_758': ['NRNX1-seg-39', 'NRNX1-seg-39', 'NRNX1-seg-39', 'NRNX1-seg-39', 'NRNX1-seg-251', 'NRNX1-seg-251', 'NRNX1-seg-251', 'NRNX1-seg-251', 'NRNX1-seg-326', 'NRNX1-seg-326', 'NRNX1-seg-326', 'NRNX1-seg-326', 'NRNX1-seg-376', 'NRNX1-seg-376', 'NRNX1-seg-376', 'NRNX1-seg-376', 'NRNX1-seg-428', 'NRNX1-seg-428', 'NRNX1-seg-428', 'NRNX1-seg-428', 'NRNX1-seg-520', 'NRNX1-seg-520', 'NRNX1-seg-520', 'NRNX1-seg-520', 'NRNX1-seg-600', 'NRNX1-seg-600', 'NRNX1-seg-600', 'NRNX1-seg-600', 'NRNX1-seg-686', 'NRNX1-seg-686', 'NRNX1-seg-686', 'NRNX1-seg-686'], 'NDB_398': ['NRNX1-seg-39', 'NRNX1-seg-39', 'NRNX1-seg-39', 'NRNX1-seg-39', 'NRNX1-seg-122', 'NRNX1-seg-122', 'NRNX1-seg-122', 'NRNX1-seg-122', 'NRNX1-seg-187', 'NRNX1-seg-187', 'NRNX1-seg-187', 'NRNX1-seg-187', 'NRNX1-seg-262', 'NRNX1-seg-262', 'NRNX1-seg-262', 'NRNX1-seg-262', 'NRNX1-seg-388', 'NRNX1-seg-388', 'NRNX1-seg-388', 'NRNX1-seg-388', 'NRNX1-seg-459', 'NRNX1-seg-459', 'NRNX1-seg-459', 'NRNX1-seg-459', 'NRNX1-seg-510', 'NRNX1-seg-510', 'NRNX1-seg-510', 'NRNX1-seg-510', 'NRNX1-seg-560', 'NRNX1-seg-560', 'NRNX1-seg-560', 'NRNX1-seg-560', 'NRNX1-seg-696', 'NRNX1-seg-696', 'NRNX1-seg-696', 'NRNX1-seg-696'], 'NDB_1060': ['NRNX1-seg-40', 'NRNX1-seg-40', 'NRNX1-seg-40', 'NRNX1-seg-40', 'NRNX1-seg-144', 'NRNX1-seg-144', 'NRNX1-seg-144', 'NRNX1-seg-144', 'NRNX1-seg-208', 'NRNX1-seg-208', 'NRNX1-seg-208', 'NRNX1-seg-208', 'NRNX1-seg-317', 'NRNX1-seg-317', 'NRNX1-seg-317', 'NRNX1-seg-317', 'NRNX1-seg-419', 'NRNX1-seg-419', 'NRNX1-seg-419', 'NRNX1-seg-419', 'NRNX1-seg-481', 'NRNX1-seg-481', 'NRNX1-seg-481', 'NRNX1-seg-481', 'NRNX1-seg-541', 'NRNX1-seg-541', 'NRNX1-seg-541', 'NRNX1-seg-541', 'NRNX1-seg-591', 'NRNX1-seg-591', 'NRNX1-seg-591', 'NRNX1-seg-591'], 'NDB_866': ['NRNX1-seg-40', 'NRNX1-seg-40', 'NRNX1-seg-40', 'NRNX1-seg-40', 'NRNX1-seg-252', 'NRNX1-seg-252', 'NRNX1-seg-252', 'NRNX1-seg-252', 'NRNX1-seg-327', 'NRNX1-seg-327', 'NRNX1-seg-327', 'NRNX1-seg-327', 'NRNX1-seg-379', 'NRNX1-seg-379', 'NRNX1-seg-379', 'NRNX1-seg-379', 'NRNX1-seg-429', 'NRNX1-seg-429', 'NRNX1-seg-429', 'NRNX1-seg-429', 'NRNX1-seg-521', 'NRNX1-seg-521', 'NRNX1-seg-521', 'NRNX1-seg-521', 'NRNX1-seg-601', 'NRNX1-seg-601', 'NRNX1-seg-601', 'NRNX1-seg-601', 'NRNX1-seg-687', 'NRNX1-seg-687', 'NRNX1-seg-687', 'NRNX1-seg-687'], 'NDB_752': ['NRNX1-seg-40', 'NRNX1-seg-40', 'NRNX1-seg-40', 'NRNX1-seg-40', 'NRNX1-seg-123', 'NRNX1-seg-123', 'NRNX1-seg-123', 'NRNX1-seg-123', 'NRNX1-seg-188', 'NRNX1-seg-188', 'NRNX1-seg-188', 'NRNX1-seg-188', 'NRNX1-seg-263', 'NRNX1-seg-263', 'NRNX1-seg-263', 'NRNX1-seg-263', 'NRNX1-seg-389', 'NRNX1-seg-389', 'NRNX1-seg-389', 'NRNX1-seg-389', 'NRNX1-seg-460', 'NRNX1-seg-460', 'NRNX1-seg-460', 'NRNX1-seg-460', 'NRNX1-seg-511', 'NRNX1-seg-511', 'NRNX1-seg-511', 'NRNX1-seg-511', 'NRNX1-seg-561', 'NRNX1-seg-561', 'NRNX1-seg-561', 'NRNX1-seg-561', 'NRNX1-seg-697', 'NRNX1-seg-697', 'NRNX1-seg-697', 'NRNX1-seg-697'], 'NDB_433': ['NRNX1-seg-41', 'NRNX1-seg-41', 'NRNX1-seg-41', 'NRNX1-seg-41', 'NRNX1-seg-145', 'NRNX1-seg-145', 'NRNX1-seg-145', 'NRNX1-seg-145', 'NRNX1-seg-209', 'NRNX1-seg-209', 'NRNX1-seg-209', 'NRNX1-seg-209', 'NRNX1-seg-318', 'NRNX1-seg-318', 'NRNX1-seg-318', 'NRNX1-seg-318', 'NRNX1-seg-420', 'NRNX1-seg-420', 'NRNX1-seg-420', 'NRNX1-seg-420', 'NRNX1-seg-482', 'NRNX1-seg-482', 'NRNX1-seg-482', 'NRNX1-seg-482', 'NRNX1-seg-542', 'NRNX1-seg-542', 'NRNX1-seg-542', 'NRNX1-seg-542', 'NRNX1-seg-592', 'NRNX1-seg-592', 'NRNX1-seg-592', 'NRNX1-seg-592'], 'NDB_539': ['NRNX1-seg-41', 'NRNX1-seg-41', 'NRNX1-seg-41', 'NRNX1-seg-41', 'NRNX1-seg-253', 'NRNX1-seg-253', 'NRNX1-seg-253', 'NRNX1-seg-253', 'NRNX1-seg-328', 'NRNX1-seg-328', 'NRNX1-seg-328', 'NRNX1-seg-328', 'NRNX1-seg-380', 'NRNX1-seg-380', 'NRNX1-seg-380', 'NRNX1-seg-380', 'NRNX1-seg-430', 'NRNX1-seg-430', 'NRNX1-seg-430', 'NRNX1-seg-430', 'NRNX1-seg-522', 'NRNX1-seg-522', 'NRNX1-seg-522', 'NRNX1-seg-522', 'NRNX1-seg-602', 'NRNX1-seg-602', 'NRNX1-seg-602', 'NRNX1-seg-602', 'NRNX1-seg-688', 'NRNX1-seg-688', 'NRNX1-seg-688', 'NRNX1-seg-688'], 'NDB_899': ['NRNX1-seg-41', 'NRNX1-seg-41', 'NRNX1-seg-41', 'NRNX1-seg-41', 'NRNX1-seg-124', 'NRNX1-seg-124', 'NRNX1-seg-124', 'NRNX1-seg-124', 'NRNX1-seg-189', 'NRNX1-seg-189', 'NRNX1-seg-189', 'NRNX1-seg-189', 'NRNX1-seg-264', 'NRNX1-seg-264', 'NRNX1-seg-264', 'NRNX1-seg-264', 'NRNX1-seg-390', 'NRNX1-seg-390', 'NRNX1-seg-390', 'NRNX1-seg-390', 'NRNX1-seg-461', 'NRNX1-seg-461', 'NRNX1-seg-461', 'NRNX1-seg-461', 'NRNX1-seg-512', 'NRNX1-seg-512', 'NRNX1-seg-512', 'NRNX1-seg-512', 'NRNX1-seg-562', 'NRNX1-seg-562', 'NRNX1-seg-562', 'NRNX1-seg-562', 'NRNX1-seg-698', 'NRNX1-seg-698', 'NRNX1-seg-698', 'NRNX1-seg-698'], 'NDB_994': ['NRNX1-seg-42', 'NRNX1-seg-42', 'NRNX1-seg-42', 'NRNX1-seg-42', 'NRNX1-seg-146', 'NRNX1-seg-146', 'NRNX1-seg-146', 'NRNX1-seg-146', 'NRNX1-seg-210', 'NRNX1-seg-210', 'NRNX1-seg-210', 'NRNX1-seg-210', 'NRNX1-seg-319', 'NRNX1-seg-319', 'NRNX1-seg-319', 'NRNX1-seg-319', 'NRNX1-seg-421', 'NRNX1-seg-421', 'NRNX1-seg-421', 'NRNX1-seg-421', 'NRNX1-seg-483', 'NRNX1-seg-483', 'NRNX1-seg-483', 'NRNX1-seg-483', 'NRNX1-seg-543', 'NRNX1-seg-543', 'NRNX1-seg-543', 'NRNX1-seg-543', 'NRNX1-seg-593', 'NRNX1-seg-593', 'NRNX1-seg-593', 'NRNX1-seg-593'], 'NDB_881': ['NRNX1-seg-42', 'NRNX1-seg-42', 'NRNX1-seg-42', 'NRNX1-seg-42', 'NRNX1-seg-254', 'NRNX1-seg-254', 'NRNX1-seg-254', 'NRNX1-seg-254', 'NRNX1-seg-329', 'NRNX1-seg-329', 'NRNX1-seg-329', 'NRNX1-seg-329', 'NRNX1-seg-381', 'NRNX1-seg-381', 'NRNX1-seg-381', 'NRNX1-seg-381', 'NRNX1-seg-431', 'NRNX1-seg-431', 'NRNX1-seg-431', 'NRNX1-seg-431', 'NRNX1-seg-523', 'NRNX1-seg-523', 'NRNX1-seg-523', 'NRNX1-seg-523', 'NRNX1-seg-603', 'NRNX1-seg-603', 'NRNX1-seg-603', 'NRNX1-seg-603', 'NRNX1-seg-689', 'NRNX1-seg-689', 'NRNX1-seg-689', 'NRNX1-seg-689'], 'NDB_935': ['NRNX1-seg-42', 'NRNX1-seg-42', 'NRNX1-seg-42', 'NRNX1-seg-42', 'NRNX1-seg-125', 'NRNX1-seg-125', 'NRNX1-seg-125', 'NRNX1-seg-125', 'NRNX1-seg-190', 'NRNX1-seg-190', 'NRNX1-seg-190', 'NRNX1-seg-190', 'NRNX1-seg-265', 'NRNX1-seg-265', 'NRNX1-seg-265', 'NRNX1-seg-265', 'NRNX1-seg-391', 'NRNX1-seg-391', 'NRNX1-seg-391', 'NRNX1-seg-391', 'NRNX1-seg-462', 'NRNX1-seg-462', 'NRNX1-seg-462', 'NRNX1-seg-462', 'NRNX1-seg-513', 'NRNX1-seg-513', 'NRNX1-seg-513', 'NRNX1-seg-513', 'NRNX1-seg-563', 'NRNX1-seg-563', 'NRNX1-seg-563', 'NRNX1-seg-563', 'NRNX1-seg-699', 'NRNX1-seg-699', 'NRNX1-seg-699', 'NRNX1-seg-699'], 'NDB_551': ['NRNX1-seg-43', 'NRNX1-seg-43', 'NRNX1-seg-43', 'NRNX1-seg-43', 'NRNX1-seg-115', 'NRNX1-seg-115', 'NRNX1-seg-115', 'NRNX1-seg-115', 'NRNX1-seg-181', 'NRNX1-seg-181', 'NRNX1-seg-181', 'NRNX1-seg-181', 'NRNX1-seg-308', 'NRNX1-seg-308', 'NRNX1-seg-308', 'NRNX1-seg-308', 'NRNX1-seg-382', 'NRNX1-seg-382', 'NRNX1-seg-382', 'NRNX1-seg-382', 'NRNX1-seg-442', 'NRNX1-seg-442', 'NRNX1-seg-442', 'NRNX1-seg-442', 'NRNX1-seg-504', 'NRNX1-seg-504', 'NRNX1-seg-504', 'NRNX1-seg-504', 'NRNX1-seg-594', 'NRNX1-seg-594', 'NRNX1-seg-594', 'NRNX1-seg-594'], 'NDB_705': ['NRNX1-seg-43', 'NRNX1-seg-43', 'NRNX1-seg-43', 'NRNX1-seg-43', 'NRNX1-seg-85', 'NRNX1-seg-85', 'NRNX1-seg-85', 'NRNX1-seg-85', 'NRNX1-seg-159', 'NRNX1-seg-159', 'NRNX1-seg-159', 'NRNX1-seg-159', 'NRNX1-seg-221', 'NRNX1-seg-221', 'NRNX1-seg-221', 'NRNX1-seg-221', 'NRNX1-seg-412', 'NRNX1-seg-412', 'NRNX1-seg-412', 'NRNX1-seg-412', 'NRNX1-seg-463', 'NRNX1-seg-463', 'NRNX1-seg-463', 'NRNX1-seg-463', 'NRNX1-seg-534', 'NRNX1-seg-534', 'NRNX1-seg-534', 'NRNX1-seg-534', 'NRNX1-seg-604', 'NRNX1-seg-604', 'NRNX1-seg-604', 'NRNX1-seg-604', 'NRNX1-seg-667', 'NRNX1-seg-667', 'NRNX1-seg-667', 'NRNX1-seg-667'], 'NDB_1053': ['NRNX1-seg-43', 'NRNX1-seg-43', 'NRNX1-seg-43', 'NRNX1-seg-43', 'NRNX1-seg-126', 'NRNX1-seg-126', 'NRNX1-seg-126', 'NRNX1-seg-126', 'NRNX1-seg-201', 'NRNX1-seg-201', 'NRNX1-seg-201', 'NRNX1-seg-201', 'NRNX1-seg-266', 'NRNX1-seg-266', 'NRNX1-seg-266', 'NRNX1-seg-266', 'NRNX1-seg-402', 'NRNX1-seg-402', 'NRNX1-seg-402', 'NRNX1-seg-402', 'NRNX1-seg-494', 'NRNX1-seg-494', 'NRNX1-seg-494', 'NRNX1-seg-494', 'NRNX1-seg-574', 'NRNX1-seg-574', 'NRNX1-seg-574', 'NRNX1-seg-574', 'NRNX1-seg-655', 'NRNX1-seg-655', 'NRNX1-seg-655', 'NRNX1-seg-655'], 'NDB_836': ['NRNX1-seg-44', 'NRNX1-seg-44', 'NRNX1-seg-44', 'NRNX1-seg-44', 'NRNX1-seg-117', 'NRNX1-seg-117', 'NRNX1-seg-117', 'NRNX1-seg-117', 'NRNX1-seg-182', 'NRNX1-seg-182', 'NRNX1-seg-182', 'NRNX1-seg-182', 'NRNX1-seg-311', 'NRNX1-seg-311', 'NRNX1-seg-311', 'NRNX1-seg-311', 'NRNX1-seg-383', 'NRNX1-seg-383', 'NRNX1-seg-383', 'NRNX1-seg-383', 'NRNX1-seg-443', 'NRNX1-seg-443', 'NRNX1-seg-443', 'NRNX1-seg-443', 'NRNX1-seg-505', 'NRNX1-seg-505', 'NRNX1-seg-505', 'NRNX1-seg-505', 'NRNX1-seg-595', 'NRNX1-seg-595', 'NRNX1-seg-595', 'NRNX1-seg-595'], 'NDB_519': ['NRNX1-seg-44', 'NRNX1-seg-44', 'NRNX1-seg-44', 'NRNX1-seg-44', 'NRNX1-seg-86', 'NRNX1-seg-86', 'NRNX1-seg-86', 'NRNX1-seg-86', 'NRNX1-seg-160', 'NRNX1-seg-160', 'NRNX1-seg-160', 'NRNX1-seg-160', 'NRNX1-seg-222', 'NRNX1-seg-222', 'NRNX1-seg-222', 'NRNX1-seg-222', 'NRNX1-seg-413', 'NRNX1-seg-413', 'NRNX1-seg-413', 'NRNX1-seg-413', 'NRNX1-seg-464', 'NRNX1-seg-464', 'NRNX1-seg-464', 'NRNX1-seg-464', 'NRNX1-seg-535', 'NRNX1-seg-535', 'NRNX1-seg-535', 'NRNX1-seg-535', 'NRNX1-seg-605', 'NRNX1-seg-605', 'NRNX1-seg-605', 'NRNX1-seg-605', 'NRNX1-seg-668', 'NRNX1-seg-668', 'NRNX1-seg-668', 'NRNX1-seg-668'], 'NDB_450': ['NRNX1-seg-44', 'NRNX1-seg-44', 'NRNX1-seg-44', 'NRNX1-seg-44', 'NRNX1-seg-127', 'NRNX1-seg-127', 'NRNX1-seg-127', 'NRNX1-seg-127', 'NRNX1-seg-202', 'NRNX1-seg-202', 'NRNX1-seg-202', 'NRNX1-seg-202', 'NRNX1-seg-267', 'NRNX1-seg-267', 'NRNX1-seg-267', 'NRNX1-seg-267', 'NRNX1-seg-403', 'NRNX1-seg-403', 'NRNX1-seg-403', 'NRNX1-seg-403', 'NRNX1-seg-495', 'NRNX1-seg-495', 'NRNX1-seg-495', 'NRNX1-seg-495', 'NRNX1-seg-575', 'NRNX1-seg-575', 'NRNX1-seg-575', 'NRNX1-seg-575', 'NRNX1-seg-656', 'NRNX1-seg-656', 'NRNX1-seg-656', 'NRNX1-seg-656'], 'NDB_755': ['NRNX1-seg-45', 'NRNX1-seg-45', 'NRNX1-seg-45', 'NRNX1-seg-45', 'NRNX1-seg-87', 'NRNX1-seg-87', 'NRNX1-seg-87', 'NRNX1-seg-87', 'NRNX1-seg-173', 'NRNX1-seg-173', 'NRNX1-seg-173', 'NRNX1-seg-173', 'NRNX1-seg-245', 'NRNX1-seg-245', 'NRNX1-seg-245', 'NRNX1-seg-245', 'NRNX1-seg-352', 'NRNX1-seg-352', 'NRNX1-seg-352', 'NRNX1-seg-352', 'NRNX1-seg-444', 'NRNX1-seg-444', 'NRNX1-seg-444', 'NRNX1-seg-444', 'NRNX1-seg-586', 'NRNX1-seg-586', 'NRNX1-seg-586', 'NRNX1-seg-586', 'NRNX1-seg-647', 'NRNX1-seg-647', 'NRNX1-seg-647', 'NRNX1-seg-647'], 'NDB_725': ['NRNX1-seg-45', 'NRNX1-seg-45', 'NRNX1-seg-45', 'NRNX1-seg-45', 'NRNX1-seg-128', 'NRNX1-seg-128', 'NRNX1-seg-128', 'NRNX1-seg-128', 'NRNX1-seg-193', 'NRNX1-seg-193', 'NRNX1-seg-193', 'NRNX1-seg-193', 'NRNX1-seg-233', 'NRNX1-seg-233', 'NRNX1-seg-233', 'NRNX1-seg-233', 'NRNX1-seg-342', 'NRNX1-seg-342', 'NRNX1-seg-342', 'NRNX1-seg-342', 'NRNX1-seg-434', 'NRNX1-seg-434', 'NRNX1-seg-434', 'NRNX1-seg-434', 'NRNX1-seg-556', 'NRNX1-seg-556', 'NRNX1-seg-556', 'NRNX1-seg-556', 'NRNX1-seg-606', 'NRNX1-seg-606', 'NRNX1-seg-606', 'NRNX1-seg-606', 'NRNX1-seg-682', 'NRNX1-seg-682', 'NRNX1-seg-682', 'NRNX1-seg-682'], 'NDB_743': ['NRNX1-seg-45', 'NRNX1-seg-45', 'NRNX1-seg-45', 'NRNX1-seg-45', 'NRNX1-seg-150', 'NRNX1-seg-150', 'NRNX1-seg-150', 'NRNX1-seg-150', 'NRNX1-seg-257', 'NRNX1-seg-257', 'NRNX1-seg-257', 'NRNX1-seg-257', 'NRNX1-seg-322', 'NRNX1-seg-322', 'NRNX1-seg-322', 'NRNX1-seg-322', 'NRNX1-seg-394', 'NRNX1-seg-394', 'NRNX1-seg-394', 'NRNX1-seg-394', 'NRNX1-seg-465', 'NRNX1-seg-465', 'NRNX1-seg-465', 'NRNX1-seg-465', 'NRNX1-seg-526', 'NRNX1-seg-526', 'NRNX1-seg-526', 'NRNX1-seg-526', 'NRNX1-seg-566', 'NRNX1-seg-566', 'NRNX1-seg-566', 'NRNX1-seg-566'], 'NDB_683': ['NRNX1-seg-46', 'NRNX1-seg-46', 'NRNX1-seg-46', 'NRNX1-seg-46', 'NRNX1-seg-88', 'NRNX1-seg-88', 'NRNX1-seg-88', 'NRNX1-seg-88', 'NRNX1-seg-174', 'NRNX1-seg-174', 'NRNX1-seg-174', 'NRNX1-seg-174', 'NRNX1-seg-246', 'NRNX1-seg-246', 'NRNX1-seg-246', 'NRNX1-seg-246', 'NRNX1-seg-353', 'NRNX1-seg-353', 'NRNX1-seg-353', 'NRNX1-seg-353', 'NRNX1-seg-445', 'NRNX1-seg-445', 'NRNX1-seg-445', 'NRNX1-seg-445', 'NRNX1-seg-587', 'NRNX1-seg-587', 'NRNX1-seg-587', 'NRNX1-seg-587', 'NRNX1-seg-648', 'NRNX1-seg-648', 'NRNX1-seg-648', 'NRNX1-seg-648'], 'NDB_431': ['NRNX1-seg-46', 'NRNX1-seg-46', 'NRNX1-seg-46', 'NRNX1-seg-46', 'NRNX1-seg-129', 'NRNX1-seg-129', 'NRNX1-seg-129', 'NRNX1-seg-129', 'NRNX1-seg-194', 'NRNX1-seg-194', 'NRNX1-seg-194', 'NRNX1-seg-194', 'NRNX1-seg-234', 'NRNX1-seg-234', 'NRNX1-seg-234', 'NRNX1-seg-234', 'NRNX1-seg-343', 'NRNX1-seg-343', 'NRNX1-seg-343', 'NRNX1-seg-343', 'NRNX1-seg-435', 'NRNX1-seg-435', 'NRNX1-seg-435', 'NRNX1-seg-435', 'NRNX1-seg-557', 'NRNX1-seg-557', 'NRNX1-seg-557', 'NRNX1-seg-557', 'NRNX1-seg-607', 'NRNX1-seg-607', 'NRNX1-seg-607', 'NRNX1-seg-607', 'NRNX1-seg-683', 'NRNX1-seg-683', 'NRNX1-seg-683', 'NRNX1-seg-683'], 'NDB_728': ['NRNX1-seg-46', 'NRNX1-seg-46', 'NRNX1-seg-46', 'NRNX1-seg-46', 'NRNX1-seg-151', 'NRNX1-seg-151', 'NRNX1-seg-151', 'NRNX1-seg-151', 'NRNX1-seg-258', 'NRNX1-seg-258', 'NRNX1-seg-258', 'NRNX1-seg-258', 'NRNX1-seg-323', 'NRNX1-seg-323', 'NRNX1-seg-323', 'NRNX1-seg-323', 'NRNX1-seg-395', 'NRNX1-seg-395', 'NRNX1-seg-395', 'NRNX1-seg-395', 'NRNX1-seg-466', 'NRNX1-seg-466', 'NRNX1-seg-466', 'NRNX1-seg-466', 'NRNX1-seg-527', 'NRNX1-seg-527', 'NRNX1-seg-527', 'NRNX1-seg-527', 'NRNX1-seg-567', 'NRNX1-seg-567', 'NRNX1-seg-567', 'NRNX1-seg-567'], 'NDB_458': ['NRNX1-seg-47', 'NRNX1-seg-47', 'NRNX1-seg-47', 'NRNX1-seg-47', 'NRNX1-seg-89', 'NRNX1-seg-89', 'NRNX1-seg-89', 'NRNX1-seg-89', 'NRNX1-seg-175', 'NRNX1-seg-175', 'NRNX1-seg-175', 'NRNX1-seg-175', 'NRNX1-seg-248', 'NRNX1-seg-248', 'NRNX1-seg-248', 'NRNX1-seg-248', 'NRNX1-seg-354', 'NRNX1-seg-354', 'NRNX1-seg-354', 'NRNX1-seg-354', 'NRNX1-seg-446', 'NRNX1-seg-446', 'NRNX1-seg-446', 'NRNX1-seg-446', 'NRNX1-seg-588', 'NRNX1-seg-588', 'NRNX1-seg-588', 'NRNX1-seg-588', 'NRNX1-seg-649', 'NRNX1-seg-649', 'NRNX1-seg-649', 'NRNX1-seg-649'], 'NDB_773': ['NRNX1-seg-47', 'NRNX1-seg-47', 'NRNX1-seg-47', 'NRNX1-seg-47', 'NRNX1-seg-130', 'NRNX1-seg-130', 'NRNX1-seg-130', 'NRNX1-seg-130', 'NRNX1-seg-195', 'NRNX1-seg-195', 'NRNX1-seg-195', 'NRNX1-seg-195', 'NRNX1-seg-235', 'NRNX1-seg-235', 'NRNX1-seg-235', 'NRNX1-seg-235', 'NRNX1-seg-344', 'NRNX1-seg-344', 'NRNX1-seg-344', 'NRNX1-seg-344', 'NRNX1-seg-436', 'NRNX1-seg-436', 'NRNX1-seg-436', 'NRNX1-seg-436', 'NRNX1-seg-558', 'NRNX1-seg-558', 'NRNX1-seg-558', 'NRNX1-seg-558', 'NRNX1-seg-608', 'NRNX1-seg-608', 'NRNX1-seg-608', 'NRNX1-seg-608', 'NRNX1-seg-684', 'NRNX1-seg-684', 'NRNX1-seg-684', 'NRNX1-seg-684'], 'NDB_860': ['NRNX1-seg-47', 'NRNX1-seg-47', 'NRNX1-seg-47', 'NRNX1-seg-47', 'NRNX1-seg-152', 'NRNX1-seg-152', 'NRNX1-seg-152', 'NRNX1-seg-152', 'NRNX1-seg-259', 'NRNX1-seg-259', 'NRNX1-seg-259', 'NRNX1-seg-259', 'NRNX1-seg-324', 'NRNX1-seg-324', 'NRNX1-seg-324', 'NRNX1-seg-324', 'NRNX1-seg-396', 'NRNX1-seg-396', 'NRNX1-seg-396', 'NRNX1-seg-396', 'NRNX1-seg-467', 'NRNX1-seg-467', 'NRNX1-seg-467', 'NRNX1-seg-467', 'NRNX1-seg-528', 'NRNX1-seg-528', 'NRNX1-seg-528', 'NRNX1-seg-528', 'NRNX1-seg-568', 'NRNX1-seg-568', 'NRNX1-seg-568', 'NRNX1-seg-568'], 'NDB_680': ['NRNX1-seg-48', 'NRNX1-seg-48', 'NRNX1-seg-48', 'NRNX1-seg-48', 'NRNX1-seg-90', 'NRNX1-seg-90', 'NRNX1-seg-90', 'NRNX1-seg-90', 'NRNX1-seg-176', 'NRNX1-seg-176', 'NRNX1-seg-176', 'NRNX1-seg-176', 'NRNX1-seg-249', 'NRNX1-seg-249', 'NRNX1-seg-249', 'NRNX1-seg-249', 'NRNX1-seg-355', 'NRNX1-seg-355', 'NRNX1-seg-355', 'NRNX1-seg-355', 'NRNX1-seg-447', 'NRNX1-seg-447', 'NRNX1-seg-447', 'NRNX1-seg-447', 'NRNX1-seg-589', 'NRNX1-seg-589', 'NRNX1-seg-589', 'NRNX1-seg-589', 'NRNX1-seg-650', 'NRNX1-seg-650', 'NRNX1-seg-650', 'NRNX1-seg-650'], 'NDB_737': ['NRNX1-seg-48', 'NRNX1-seg-48', 'NRNX1-seg-48', 'NRNX1-seg-48', 'NRNX1-seg-131', 'NRNX1-seg-131', 'NRNX1-seg-131', 'NRNX1-seg-131', 'NRNX1-seg-196', 'NRNX1-seg-196', 'NRNX1-seg-196', 'NRNX1-seg-196', 'NRNX1-seg-236', 'NRNX1-seg-236', 'NRNX1-seg-236', 'NRNX1-seg-236', 'NRNX1-seg-345', 'NRNX1-seg-345', 'NRNX1-seg-345', 'NRNX1-seg-345', 'NRNX1-seg-437', 'NRNX1-seg-437', 'NRNX1-seg-437', 'NRNX1-seg-437', 'NRNX1-seg-559', 'NRNX1-seg-559', 'NRNX1-seg-559', 'NRNX1-seg-559', 'NRNX1-seg-609', 'NRNX1-seg-609', 'NRNX1-seg-609', 'NRNX1-seg-609', 'NRNX1-seg-685', 'NRNX1-seg-685', 'NRNX1-seg-685', 'NRNX1-seg-685'], 'NDB_878': ['NRNX1-seg-48', 'NRNX1-seg-48', 'NRNX1-seg-48', 'NRNX1-seg-48', 'NRNX1-seg-154', 'NRNX1-seg-154', 'NRNX1-seg-154', 'NRNX1-seg-154', 'NRNX1-seg-260', 'NRNX1-seg-260', 'NRNX1-seg-260', 'NRNX1-seg-260', 'NRNX1-seg-325', 'NRNX1-seg-325', 'NRNX1-seg-325', 'NRNX1-seg-325', 'NRNX1-seg-397', 'NRNX1-seg-397', 'NRNX1-seg-397', 'NRNX1-seg-397', 'NRNX1-seg-468', 'NRNX1-seg-468', 'NRNX1-seg-468', 'NRNX1-seg-468', 'NRNX1-seg-529', 'NRNX1-seg-529', 'NRNX1-seg-529', 'NRNX1-seg-529', 'NRNX1-seg-569', 'NRNX1-seg-569', 'NRNX1-seg-569', 'NRNX1-seg-569'], 'NDB_332': ['NRNX1-seg-49', 'NRNX1-seg-49', 'NRNX1-seg-49', 'NRNX1-seg-49', 'NRNX1-seg-91', 'NRNX1-seg-91', 'NRNX1-seg-91', 'NRNX1-seg-91', 'NRNX1-seg-177', 'NRNX1-seg-177', 'NRNX1-seg-177', 'NRNX1-seg-177', 'NRNX1-seg-251', 'NRNX1-seg-251', 'NRNX1-seg-251', 'NRNX1-seg-251', 'NRNX1-seg-356', 'NRNX1-seg-356', 'NRNX1-seg-356', 'NRNX1-seg-356', 'NRNX1-seg-448', 'NRNX1-seg-448', 'NRNX1-seg-448', 'NRNX1-seg-448', 'NRNX1-seg-590', 'NRNX1-seg-590', 'NRNX1-seg-590', 'NRNX1-seg-590', 'NRNX1-seg-651', 'NRNX1-seg-651', 'NRNX1-seg-651', 'NRNX1-seg-651'], 'NDB_653': ['NRNX1-seg-49', 'NRNX1-seg-49', 'NRNX1-seg-49', 'NRNX1-seg-49', 'NRNX1-seg-133', 'NRNX1-seg-133', 'NRNX1-seg-133', 'NRNX1-seg-133', 'NRNX1-seg-197', 'NRNX1-seg-197', 'NRNX1-seg-197', 'NRNX1-seg-197', 'NRNX1-seg-238', 'NRNX1-seg-238', 'NRNX1-seg-238', 'NRNX1-seg-238', 'NRNX1-seg-346', 'NRNX1-seg-346', 'NRNX1-seg-346', 'NRNX1-seg-346', 'NRNX1-seg-438', 'NRNX1-seg-438', 'NRNX1-seg-438', 'NRNX1-seg-438', 'NRNX1-seg-560', 'NRNX1-seg-560', 'NRNX1-seg-560', 'NRNX1-seg-560', 'NRNX1-seg-610', 'NRNX1-seg-610', 'NRNX1-seg-610', 'NRNX1-seg-610', 'NRNX1-seg-686', 'NRNX1-seg-686', 'NRNX1-seg-686', 'NRNX1-seg-686'], 'NDB_467': ['NRNX1-seg-49', 'NRNX1-seg-49', 'NRNX1-seg-49', 'NRNX1-seg-49', 'NRNX1-seg-155', 'NRNX1-seg-155', 'NRNX1-seg-155', 'NRNX1-seg-155', 'NRNX1-seg-262', 'NRNX1-seg-262', 'NRNX1-seg-262', 'NRNX1-seg-262', 'NRNX1-seg-326', 'NRNX1-seg-326', 'NRNX1-seg-326', 'NRNX1-seg-326', 'NRNX1-seg-398', 'NRNX1-seg-398', 'NRNX1-seg-398', 'NRNX1-seg-398', 'NRNX1-seg-469', 'NRNX1-seg-469', 'NRNX1-seg-469', 'NRNX1-seg-469', 'NRNX1-seg-530', 'NRNX1-seg-530', 'NRNX1-seg-530', 'NRNX1-seg-530', 'NRNX1-seg-570', 'NRNX1-seg-570', 'NRNX1-seg-570', 'NRNX1-seg-570'], 'NDB_335': ['NRNX1-seg-50', 'NRNX1-seg-50', 'NRNX1-seg-50', 'NRNX1-seg-50', 'NRNX1-seg-92', 'NRNX1-seg-92', 'NRNX1-seg-92', 'NRNX1-seg-92', 'NRNX1-seg-178', 'NRNX1-seg-178', 'NRNX1-seg-178', 'NRNX1-seg-178', 'NRNX1-seg-252', 'NRNX1-seg-252', 'NRNX1-seg-252', 'NRNX1-seg-252', 'NRNX1-seg-357', 'NRNX1-seg-357', 'NRNX1-seg-357', 'NRNX1-seg-357', 'NRNX1-seg-449', 'NRNX1-seg-449', 'NRNX1-seg-449', 'NRNX1-seg-449', 'NRNX1-seg-591', 'NRNX1-seg-591', 'NRNX1-seg-591', 'NRNX1-seg-591', 'NRNX1-seg-652', 'NRNX1-seg-652', 'NRNX1-seg-652', 'NRNX1-seg-652'], 'NDB_347': ['NRNX1-seg-50', 'NRNX1-seg-50', 'NRNX1-seg-50', 'NRNX1-seg-50', 'NRNX1-seg-134', 'NRNX1-seg-134', 'NRNX1-seg-134', 'NRNX1-seg-134', 'NRNX1-seg-198', 'NRNX1-seg-198', 'NRNX1-seg-198', 'NRNX1-seg-198', 'NRNX1-seg-239', 'NRNX1-seg-239', 'NRNX1-seg-239', 'NRNX1-seg-239', 'NRNX1-seg-347', 'NRNX1-seg-347', 'NRNX1-seg-347', 'NRNX1-seg-347', 'NRNX1-seg-439', 'NRNX1-seg-439', 'NRNX1-seg-439', 'NRNX1-seg-439', 'NRNX1-seg-561', 'NRNX1-seg-561', 'NRNX1-seg-561', 'NRNX1-seg-561', 'NRNX1-seg-611', 'NRNX1-seg-611', 'NRNX1-seg-611', 'NRNX1-seg-611', 'NRNX1-seg-687', 'NRNX1-seg-687', 'NRNX1-seg-687', 'NRNX1-seg-687'], 'NDB_446': ['NRNX1-seg-50', 'NRNX1-seg-50', 'NRNX1-seg-50', 'NRNX1-seg-50', 'NRNX1-seg-156', 'NRNX1-seg-156', 'NRNX1-seg-156', 'NRNX1-seg-156', 'NRNX1-seg-263', 'NRNX1-seg-263', 'NRNX1-seg-263', 'NRNX1-seg-263', 'NRNX1-seg-327', 'NRNX1-seg-327', 'NRNX1-seg-327', 'NRNX1-seg-327', 'NRNX1-seg-399', 'NRNX1-seg-399', 'NRNX1-seg-399', 'NRNX1-seg-399', 'NRNX1-seg-470', 'NRNX1-seg-470', 'NRNX1-seg-470', 'NRNX1-seg-470', 'NRNX1-seg-531', 'NRNX1-seg-531', 'NRNX1-seg-531', 'NRNX1-seg-531', 'NRNX1-seg-571', 'NRNX1-seg-571', 'NRNX1-seg-571', 'NRNX1-seg-571'], 'NDB_473': ['NRNX1-seg-51', 'NRNX1-seg-51', 'NRNX1-seg-51', 'NRNX1-seg-51', 'NRNX1-seg-93', 'NRNX1-seg-93', 'NRNX1-seg-93', 'NRNX1-seg-93', 'NRNX1-seg-179', 'NRNX1-seg-179', 'NRNX1-seg-179', 'NRNX1-seg-179', 'NRNX1-seg-253', 'NRNX1-seg-253', 'NRNX1-seg-253', 'NRNX1-seg-253', 'NRNX1-seg-358', 'NRNX1-seg-358', 'NRNX1-seg-358', 'NRNX1-seg-358', 'NRNX1-seg-450', 'NRNX1-seg-450', 'NRNX1-seg-450', 'NRNX1-seg-450', 'NRNX1-seg-592', 'NRNX1-seg-592', 'NRNX1-seg-592', 'NRNX1-seg-592', 'NRNX1-seg-653', 'NRNX1-seg-653', 'NRNX1-seg-653', 'NRNX1-seg-653'], 'NDB_641': ['NRNX1-seg-51', 'NRNX1-seg-51', 'NRNX1-seg-51', 'NRNX1-seg-51', 'NRNX1-seg-135', 'NRNX1-seg-135', 'NRNX1-seg-135', 'NRNX1-seg-135', 'NRNX1-seg-199', 'NRNX1-seg-199', 'NRNX1-seg-199', 'NRNX1-seg-199', 'NRNX1-seg-240', 'NRNX1-seg-240', 'NRNX1-seg-240', 'NRNX1-seg-240', 'NRNX1-seg-348', 'NRNX1-seg-348', 'NRNX1-seg-348', 'NRNX1-seg-348', 'NRNX1-seg-440', 'NRNX1-seg-440', 'NRNX1-seg-440', 'NRNX1-seg-440', 'NRNX1-seg-562', 'NRNX1-seg-562', 'NRNX1-seg-562', 'NRNX1-seg-562', 'NRNX1-seg-612', 'NRNX1-seg-612', 'NRNX1-seg-612', 'NRNX1-seg-612', 'NRNX1-seg-688', 'NRNX1-seg-688', 'NRNX1-seg-688', 'NRNX1-seg-688'], 'NDB_452': ['NRNX1-seg-51', 'NRNX1-seg-51', 'NRNX1-seg-51', 'NRNX1-seg-51', 'NRNX1-seg-157', 'NRNX1-seg-157', 'NRNX1-seg-157', 'NRNX1-seg-157', 'NRNX1-seg-264', 'NRNX1-seg-264', 'NRNX1-seg-264', 'NRNX1-seg-264', 'NRNX1-seg-328', 'NRNX1-seg-328', 'NRNX1-seg-328', 'NRNX1-seg-328', 'NRNX1-seg-400', 'NRNX1-seg-400', 'NRNX1-seg-400', 'NRNX1-seg-400', 'NRNX1-seg-471', 'NRNX1-seg-471', 'NRNX1-seg-471', 'NRNX1-seg-471', 'NRNX1-seg-532', 'NRNX1-seg-532', 'NRNX1-seg-532', 'NRNX1-seg-532', 'NRNX1-seg-572', 'NRNX1-seg-572', 'NRNX1-seg-572', 'NRNX1-seg-572'], 'NDB_647': ['NRNX1-seg-52', 'NRNX1-seg-52', 'NRNX1-seg-52', 'NRNX1-seg-52', 'NRNX1-seg-94', 'NRNX1-seg-94', 'NRNX1-seg-94', 'NRNX1-seg-94', 'NRNX1-seg-180', 'NRNX1-seg-180', 'NRNX1-seg-180', 'NRNX1-seg-180', 'NRNX1-seg-254', 'NRNX1-seg-254', 'NRNX1-seg-254', 'NRNX1-seg-254', 'NRNX1-seg-359', 'NRNX1-seg-359', 'NRNX1-seg-359', 'NRNX1-seg-359', 'NRNX1-seg-452', 'NRNX1-seg-452', 'NRNX1-seg-452', 'NRNX1-seg-452', 'NRNX1-seg-593', 'NRNX1-seg-593', 'NRNX1-seg-593', 'NRNX1-seg-593', 'NRNX1-seg-654', 'NRNX1-seg-654', 'NRNX1-seg-654', 'NRNX1-seg-654'], 'NDB_734': ['NRNX1-seg-52', 'NRNX1-seg-52', 'NRNX1-seg-52', 'NRNX1-seg-52', 'NRNX1-seg-136', 'NRNX1-seg-136', 'NRNX1-seg-136', 'NRNX1-seg-136', 'NRNX1-seg-200', 'NRNX1-seg-200', 'NRNX1-seg-200', 'NRNX1-seg-200', 'NRNX1-seg-241', 'NRNX1-seg-241', 'NRNX1-seg-241', 'NRNX1-seg-241', 'NRNX1-seg-349', 'NRNX1-seg-349', 'NRNX1-seg-349', 'NRNX1-seg-349', 'NRNX1-seg-441', 'NRNX1-seg-441', 'NRNX1-seg-441', 'NRNX1-seg-441', 'NRNX1-seg-563', 'NRNX1-seg-563', 'NRNX1-seg-563', 'NRNX1-seg-563', 'NRNX1-seg-613', 'NRNX1-seg-613', 'NRNX1-seg-613', 'NRNX1-seg-613', 'NRNX1-seg-689', 'NRNX1-seg-689', 'NRNX1-seg-689', 'NRNX1-seg-689'], 'NDB_587': ['NRNX1-seg-52', 'NRNX1-seg-52', 'NRNX1-seg-52', 'NRNX1-seg-52', 'NRNX1-seg-158', 'NRNX1-seg-158', 'NRNX1-seg-158', 'NRNX1-seg-158', 'NRNX1-seg-265', 'NRNX1-seg-265', 'NRNX1-seg-265', 'NRNX1-seg-265', 'NRNX1-seg-329', 'NRNX1-seg-329', 'NRNX1-seg-329', 'NRNX1-seg-329', 'NRNX1-seg-401', 'NRNX1-seg-401', 'NRNX1-seg-401', 'NRNX1-seg-401', 'NRNX1-seg-472', 'NRNX1-seg-472', 'NRNX1-seg-472', 'NRNX1-seg-472', 'NRNX1-seg-533', 'NRNX1-seg-533', 'NRNX1-seg-533', 'NRNX1-seg-533', 'NRNX1-seg-573', 'NRNX1-seg-573', 'NRNX1-seg-573', 'NRNX1-seg-573'], 'NDB_628': ['NRNX1-seg-53', 'NRNX1-seg-53', 'NRNX1-seg-53', 'NRNX1-seg-53', 'NRNX1-seg-105', 'NRNX1-seg-105', 'NRNX1-seg-105', 'NRNX1-seg-105', 'NRNX1-seg-171', 'NRNX1-seg-171', 'NRNX1-seg-171', 'NRNX1-seg-171', 'NRNX1-seg-231', 'NRNX1-seg-231', 'NRNX1-seg-231', 'NRNX1-seg-231', 'NRNX1-seg-298', 'NRNX1-seg-298', 'NRNX1-seg-298', 'NRNX1-seg-298', 'NRNX1-seg-402', 'NRNX1-seg-402', 'NRNX1-seg-402', 'NRNX1-seg-402', 'NRNX1-seg-463', 'NRNX1-seg-463', 'NRNX1-seg-463', 'NRNX1-seg-463', 'NRNX1-seg-594', 'NRNX1-seg-594', 'NRNX1-seg-594', 'NRNX1-seg-594'], 'NDB_958': ['NRNX1-seg-53', 'NRNX1-seg-53', 'NRNX1-seg-53', 'NRNX1-seg-53', 'NRNX1-seg-115', 'NRNX1-seg-115', 'NRNX1-seg-115', 'NRNX1-seg-115', 'NRNX1-seg-201', 'NRNX1-seg-201', 'NRNX1-seg-201', 'NRNX1-seg-201', 'NRNX1-seg-276', 'NRNX1-seg-276', 'NRNX1-seg-276', 'NRNX1-seg-276', 'NRNX1-seg-360', 'NRNX1-seg-360', 'NRNX1-seg-360', 'NRNX1-seg-360', 'NRNX1-seg-412', 'NRNX1-seg-412', 'NRNX1-seg-412', 'NRNX1-seg-412', 'NRNX1-seg-544', 'NRNX1-seg-544', 'NRNX1-seg-544', 'NRNX1-seg-544', 'NRNX1-seg-624', 'NRNX1-seg-624', 'NRNX1-seg-624', 'NRNX1-seg-624', 'NRNX1-seg-690', 'NRNX1-seg-690', 'NRNX1-seg-690', 'NRNX1-seg-690'], 'NDB_526': ['NRNX1-seg-54', 'NRNX1-seg-54', 'NRNX1-seg-54', 'NRNX1-seg-54', 'NRNX1-seg-106', 'NRNX1-seg-106', 'NRNX1-seg-106', 'NRNX1-seg-106', 'NRNX1-seg-172', 'NRNX1-seg-172', 'NRNX1-seg-172', 'NRNX1-seg-172', 'NRNX1-seg-232', 'NRNX1-seg-232', 'NRNX1-seg-232', 'NRNX1-seg-232', 'NRNX1-seg-299', 'NRNX1-seg-299', 'NRNX1-seg-299', 'NRNX1-seg-299', 'NRNX1-seg-403', 'NRNX1-seg-403', 'NRNX1-seg-403', 'NRNX1-seg-403', 'NRNX1-seg-464', 'NRNX1-seg-464', 'NRNX1-seg-464', 'NRNX1-seg-464', 'NRNX1-seg-595', 'NRNX1-seg-595', 'NRNX1-seg-595', 'NRNX1-seg-595'], 'NDB_712': ['NRNX1-seg-54', 'NRNX1-seg-54', 'NRNX1-seg-54', 'NRNX1-seg-54', 'NRNX1-seg-117', 'NRNX1-seg-117', 'NRNX1-seg-117', 'NRNX1-seg-117', 'NRNX1-seg-202', 'NRNX1-seg-202', 'NRNX1-seg-202', 'NRNX1-seg-202', 'NRNX1-seg-277', 'NRNX1-seg-277', 'NRNX1-seg-277', 'NRNX1-seg-277', 'NRNX1-seg-361', 'NRNX1-seg-361', 'NRNX1-seg-361', 'NRNX1-seg-361', 'NRNX1-seg-413', 'NRNX1-seg-413', 'NRNX1-seg-413', 'NRNX1-seg-413', 'NRNX1-seg-545', 'NRNX1-seg-545', 'NRNX1-seg-545', 'NRNX1-seg-545', 'NRNX1-seg-625', 'NRNX1-seg-625', 'NRNX1-seg-625', 'NRNX1-seg-625', 'NRNX1-seg-691', 'NRNX1-seg-691', 'NRNX1-seg-691', 'NRNX1-seg-691'], 'NDB_817': ['NRNX1-seg-55', 'NRNX1-seg-55', 'NRNX1-seg-55', 'NRNX1-seg-55', 'NRNX1-seg-97', 'NRNX1-seg-97', 'NRNX1-seg-97', 'NRNX1-seg-97', 'NRNX1-seg-193', 'NRNX1-seg-193', 'NRNX1-seg-193', 'NRNX1-seg-193', 'NRNX1-seg-288', 'NRNX1-seg-288', 'NRNX1-seg-288', 'NRNX1-seg-288', 'NRNX1-seg-414', 'NRNX1-seg-414', 'NRNX1-seg-414', 'NRNX1-seg-414', 'NRNX1-seg-496', 'NRNX1-seg-496', 'NRNX1-seg-496', 'NRNX1-seg-496', 'NRNX1-seg-596', 'NRNX1-seg-596', 'NRNX1-seg-596', 'NRNX1-seg-596', 'NRNX1-seg-669', 'NRNX1-seg-669', 'NRNX1-seg-669', 'NRNX1-seg-669'], 'NDB_971': ['NRNX1-seg-55', 'NRNX1-seg-55', 'NRNX1-seg-55', 'NRNX1-seg-55', 'NRNX1-seg-118', 'NRNX1-seg-118', 'NRNX1-seg-118', 'NRNX1-seg-118', 'NRNX1-seg-161', 'NRNX1-seg-161', 'NRNX1-seg-161', 'NRNX1-seg-161', 'NRNX1-seg-213', 'NRNX1-seg-213', 'NRNX1-seg-213', 'NRNX1-seg-213', 'NRNX1-seg-312', 'NRNX1-seg-312', 'NRNX1-seg-312', 'NRNX1-seg-312', 'NRNX1-seg-404', 'NRNX1-seg-404', 'NRNX1-seg-404', 'NRNX1-seg-404', 'NRNX1-seg-516', 'NRNX1-seg-516', 'NRNX1-seg-516', 'NRNX1-seg-516', 'NRNX1-seg-616', 'NRNX1-seg-616', 'NRNX1-seg-616', 'NRNX1-seg-616'], 'NDB_778': ['NRNX1-seg-56', 'NRNX1-seg-56', 'NRNX1-seg-56', 'NRNX1-seg-56', 'NRNX1-seg-98', 'NRNX1-seg-98', 'NRNX1-seg-98', 'NRNX1-seg-98', 'NRNX1-seg-194', 'NRNX1-seg-194', 'NRNX1-seg-194', 'NRNX1-seg-194', 'NRNX1-seg-289', 'NRNX1-seg-289', 'NRNX1-seg-289', 'NRNX1-seg-289', 'NRNX1-seg-415', 'NRNX1-seg-415', 'NRNX1-seg-415', 'NRNX1-seg-415', 'NRNX1-seg-497', 'NRNX1-seg-497', 'NRNX1-seg-497', 'NRNX1-seg-497', 'NRNX1-seg-597', 'NRNX1-seg-597', 'NRNX1-seg-597', 'NRNX1-seg-597', 'NRNX1-seg-670', 'NRNX1-seg-670', 'NRNX1-seg-670', 'NRNX1-seg-670'], 'NDB_917': ['NRNX1-seg-56', 'NRNX1-seg-56', 'NRNX1-seg-56', 'NRNX1-seg-56', 'NRNX1-seg-119', 'NRNX1-seg-119', 'NRNX1-seg-119', 'NRNX1-seg-119', 'NRNX1-seg-162', 'NRNX1-seg-162', 'NRNX1-seg-162', 'NRNX1-seg-162', 'NRNX1-seg-214', 'NRNX1-seg-214', 'NRNX1-seg-214', 'NRNX1-seg-214', 'NRNX1-seg-313', 'NRNX1-seg-313', 'NRNX1-seg-313', 'NRNX1-seg-313', 'NRNX1-seg-405', 'NRNX1-seg-405', 'NRNX1-seg-405', 'NRNX1-seg-405', 'NRNX1-seg-517', 'NRNX1-seg-517', 'NRNX1-seg-517', 'NRNX1-seg-517', 'NRNX1-seg-617', 'NRNX1-seg-617', 'NRNX1-seg-617', 'NRNX1-seg-617'], 'NDB_487': ['NRNX1-seg-57', 'NRNX1-seg-57', 'NRNX1-seg-57', 'NRNX1-seg-57', 'NRNX1-seg-99', 'NRNX1-seg-99', 'NRNX1-seg-99', 'NRNX1-seg-99', 'NRNX1-seg-195', 'NRNX1-seg-195', 'NRNX1-seg-195', 'NRNX1-seg-195', 'NRNX1-seg-290', 'NRNX1-seg-290', 'NRNX1-seg-290', 'NRNX1-seg-290', 'NRNX1-seg-416', 'NRNX1-seg-416', 'NRNX1-seg-416', 'NRNX1-seg-416', 'NRNX1-seg-498', 'NRNX1-seg-498', 'NRNX1-seg-498', 'NRNX1-seg-498', 'NRNX1-seg-598', 'NRNX1-seg-598', 'NRNX1-seg-598', 'NRNX1-seg-598', 'NRNX1-seg-671', 'NRNX1-seg-671', 'NRNX1-seg-671', 'NRNX1-seg-671'], 'NDB_716': ['NRNX1-seg-57', 'NRNX1-seg-57', 'NRNX1-seg-57', 'NRNX1-seg-57', 'NRNX1-seg-120', 'NRNX1-seg-120', 'NRNX1-seg-120', 'NRNX1-seg-120', 'NRNX1-seg-163', 'NRNX1-seg-163', 'NRNX1-seg-163', 'NRNX1-seg-163', 'NRNX1-seg-215', 'NRNX1-seg-215', 'NRNX1-seg-215', 'NRNX1-seg-215', 'NRNX1-seg-314', 'NRNX1-seg-314', 'NRNX1-seg-314', 'NRNX1-seg-314', 'NRNX1-seg-406', 'NRNX1-seg-406', 'NRNX1-seg-406', 'NRNX1-seg-406', 'NRNX1-seg-518', 'NRNX1-seg-518', 'NRNX1-seg-518', 'NRNX1-seg-518', 'NRNX1-seg-618', 'NRNX1-seg-618', 'NRNX1-seg-618', 'NRNX1-seg-618'], 'NDB_1027': ['NRNX1-seg-58', 'NRNX1-seg-58', 'NRNX1-seg-58', 'NRNX1-seg-58', 'NRNX1-seg-100', 'NRNX1-seg-100', 'NRNX1-seg-100', 'NRNX1-seg-100', 'NRNX1-seg-196', 'NRNX1-seg-196', 'NRNX1-seg-196', 'NRNX1-seg-196', 'NRNX1-seg-291', 'NRNX1-seg-291', 'NRNX1-seg-291', 'NRNX1-seg-291', 'NRNX1-seg-417', 'NRNX1-seg-417', 'NRNX1-seg-417', 'NRNX1-seg-417', 'NRNX1-seg-499', 'NRNX1-seg-499', 'NRNX1-seg-499', 'NRNX1-seg-499', 'NRNX1-seg-599', 'NRNX1-seg-599', 'NRNX1-seg-599', 'NRNX1-seg-599', 'NRNX1-seg-672', 'NRNX1-seg-672', 'NRNX1-seg-672', 'NRNX1-seg-672'], 'NDB_863': ['NRNX1-seg-58', 'NRNX1-seg-58', 'NRNX1-seg-58', 'NRNX1-seg-58', 'NRNX1-seg-121', 'NRNX1-seg-121', 'NRNX1-seg-121', 'NRNX1-seg-121', 'NRNX1-seg-164', 'NRNX1-seg-164', 'NRNX1-seg-164', 'NRNX1-seg-164', 'NRNX1-seg-216', 'NRNX1-seg-216', 'NRNX1-seg-216', 'NRNX1-seg-216', 'NRNX1-seg-315', 'NRNX1-seg-315', 'NRNX1-seg-315', 'NRNX1-seg-315', 'NRNX1-seg-407', 'NRNX1-seg-407', 'NRNX1-seg-407', 'NRNX1-seg-407', 'NRNX1-seg-519', 'NRNX1-seg-519', 'NRNX1-seg-519', 'NRNX1-seg-519', 'NRNX1-seg-619', 'NRNX1-seg-619', 'NRNX1-seg-619', 'NRNX1-seg-619'], 'NDB_631': ['NRNX1-seg-59', 'NRNX1-seg-59', 'NRNX1-seg-59', 'NRNX1-seg-59', 'NRNX1-seg-101', 'NRNX1-seg-101', 'NRNX1-seg-101', 'NRNX1-seg-101', 'NRNX1-seg-197', 'NRNX1-seg-197', 'NRNX1-seg-197', 'NRNX1-seg-197', 'NRNX1-seg-292', 'NRNX1-seg-292', 'NRNX1-seg-292', 'NRNX1-seg-292', 'NRNX1-seg-418', 'NRNX1-seg-418', 'NRNX1-seg-418', 'NRNX1-seg-418', 'NRNX1-seg-500', 'NRNX1-seg-500', 'NRNX1-seg-500', 'NRNX1-seg-500', 'NRNX1-seg-600', 'NRNX1-seg-600', 'NRNX1-seg-600', 'NRNX1-seg-600', 'NRNX1-seg-674', 'NRNX1-seg-674', 'NRNX1-seg-674', 'NRNX1-seg-674'], 'NDB_905': ['NRNX1-seg-59', 'NRNX1-seg-59', 'NRNX1-seg-59', 'NRNX1-seg-59', 'NRNX1-seg-122', 'NRNX1-seg-122', 'NRNX1-seg-122', 'NRNX1-seg-122', 'NRNX1-seg-165', 'NRNX1-seg-165', 'NRNX1-seg-165', 'NRNX1-seg-165', 'NRNX1-seg-217', 'NRNX1-seg-217', 'NRNX1-seg-217', 'NRNX1-seg-217', 'NRNX1-seg-316', 'NRNX1-seg-316', 'NRNX1-seg-316', 'NRNX1-seg-316', 'NRNX1-seg-408', 'NRNX1-seg-408', 'NRNX1-seg-408', 'NRNX1-seg-408', 'NRNX1-seg-520', 'NRNX1-seg-520', 'NRNX1-seg-520', 'NRNX1-seg-520', 'NRNX1-seg-620', 'NRNX1-seg-620', 'NRNX1-seg-620', 'NRNX1-seg-620'], 'NDB_805': ['NRNX1-seg-60', 'NRNX1-seg-60', 'NRNX1-seg-60', 'NRNX1-seg-60', 'NRNX1-seg-102', 'NRNX1-seg-102', 'NRNX1-seg-102', 'NRNX1-seg-102', 'NRNX1-seg-198', 'NRNX1-seg-198', 'NRNX1-seg-198', 'NRNX1-seg-198', 'NRNX1-seg-293', 'NRNX1-seg-293', 'NRNX1-seg-293', 'NRNX1-seg-293', 'NRNX1-seg-419', 'NRNX1-seg-419', 'NRNX1-seg-419', 'NRNX1-seg-419', 'NRNX1-seg-501', 'NRNX1-seg-501', 'NRNX1-seg-501', 'NRNX1-seg-501', 'NRNX1-seg-601', 'NRNX1-seg-601', 'NRNX1-seg-601', 'NRNX1-seg-601', 'NRNX1-seg-675', 'NRNX1-seg-675', 'NRNX1-seg-675', 'NRNX1-seg-675'], 'NDB_482': ['NRNX1-seg-60', 'NRNX1-seg-60', 'NRNX1-seg-60', 'NRNX1-seg-60', 'NRNX1-seg-123', 'NRNX1-seg-123', 'NRNX1-seg-123', 'NRNX1-seg-123', 'NRNX1-seg-166', 'NRNX1-seg-166', 'NRNX1-seg-166', 'NRNX1-seg-166', 'NRNX1-seg-218', 'NRNX1-seg-218', 'NRNX1-seg-218', 'NRNX1-seg-218', 'NRNX1-seg-317', 'NRNX1-seg-317', 'NRNX1-seg-317', 'NRNX1-seg-317', 'NRNX1-seg-409', 'NRNX1-seg-409', 'NRNX1-seg-409', 'NRNX1-seg-409', 'NRNX1-seg-521', 'NRNX1-seg-521', 'NRNX1-seg-521', 'NRNX1-seg-521', 'NRNX1-seg-621', 'NRNX1-seg-621', 'NRNX1-seg-621', 'NRNX1-seg-621'], 'NDB_721': ['NRNX1-seg-62', 'NRNX1-seg-62', 'NRNX1-seg-62', 'NRNX1-seg-62', 'NRNX1-seg-103', 'NRNX1-seg-103', 'NRNX1-seg-103', 'NRNX1-seg-103', 'NRNX1-seg-199', 'NRNX1-seg-199', 'NRNX1-seg-199', 'NRNX1-seg-199', 'NRNX1-seg-294', 'NRNX1-seg-294', 'NRNX1-seg-294', 'NRNX1-seg-294', 'NRNX1-seg-420', 'NRNX1-seg-420', 'NRNX1-seg-420', 'NRNX1-seg-420', 'NRNX1-seg-502', 'NRNX1-seg-502', 'NRNX1-seg-502', 'NRNX1-seg-502', 'NRNX1-seg-602', 'NRNX1-seg-602', 'NRNX1-seg-602', 'NRNX1-seg-602', 'NRNX1-seg-676', 'NRNX1-seg-676', 'NRNX1-seg-676', 'NRNX1-seg-676'], 'NDB_941': ['NRNX1-seg-62', 'NRNX1-seg-62', 'NRNX1-seg-62', 'NRNX1-seg-62', 'NRNX1-seg-124', 'NRNX1-seg-124', 'NRNX1-seg-124', 'NRNX1-seg-124', 'NRNX1-seg-169', 'NRNX1-seg-169', 'NRNX1-seg-169', 'NRNX1-seg-169', 'NRNX1-seg-219', 'NRNX1-seg-219', 'NRNX1-seg-219', 'NRNX1-seg-219', 'NRNX1-seg-318', 'NRNX1-seg-318', 'NRNX1-seg-318', 'NRNX1-seg-318', 'NRNX1-seg-410', 'NRNX1-seg-410', 'NRNX1-seg-410', 'NRNX1-seg-410', 'NRNX1-seg-522', 'NRNX1-seg-522', 'NRNX1-seg-522', 'NRNX1-seg-522', 'NRNX1-seg-622', 'NRNX1-seg-622', 'NRNX1-seg-622', 'NRNX1-seg-622'], 'NDB_637': ['NRNX1-seg-63', 'NRNX1-seg-63', 'NRNX1-seg-63', 'NRNX1-seg-63', 'NRNX1-seg-104', 'NRNX1-seg-104', 'NRNX1-seg-104', 'NRNX1-seg-104', 'NRNX1-seg-200', 'NRNX1-seg-200', 'NRNX1-seg-200', 'NRNX1-seg-200', 'NRNX1-seg-295', 'NRNX1-seg-295', 'NRNX1-seg-295', 'NRNX1-seg-295', 'NRNX1-seg-421', 'NRNX1-seg-421', 'NRNX1-seg-421', 'NRNX1-seg-421', 'NRNX1-seg-503', 'NRNX1-seg-503', 'NRNX1-seg-503', 'NRNX1-seg-503', 'NRNX1-seg-603', 'NRNX1-seg-603', 'NRNX1-seg-603', 'NRNX1-seg-603', 'NRNX1-seg-678', 'NRNX1-seg-678', 'NRNX1-seg-678', 'NRNX1-seg-678'], 'NDB_788': ['NRNX1-seg-63', 'NRNX1-seg-63', 'NRNX1-seg-63', 'NRNX1-seg-63', 'NRNX1-seg-125', 'NRNX1-seg-125', 'NRNX1-seg-125', 'NRNX1-seg-125', 'NRNX1-seg-170', 'NRNX1-seg-170', 'NRNX1-seg-170', 'NRNX1-seg-170', 'NRNX1-seg-220', 'NRNX1-seg-220', 'NRNX1-seg-220', 'NRNX1-seg-220', 'NRNX1-seg-319', 'NRNX1-seg-319', 'NRNX1-seg-319', 'NRNX1-seg-319', 'NRNX1-seg-411', 'NRNX1-seg-411', 'NRNX1-seg-411', 'NRNX1-seg-411', 'NRNX1-seg-523', 'NRNX1-seg-523', 'NRNX1-seg-523', 'NRNX1-seg-523', 'NRNX1-seg-623', 'NRNX1-seg-623', 'NRNX1-seg-623', 'NRNX1-seg-623'], 'NDB_779': ['NRNX1-seg-64', 'NRNX1-seg-64', 'NRNX1-seg-64', 'NRNX1-seg-64', 'NRNX1-seg-159', 'NRNX1-seg-159', 'NRNX1-seg-159', 'NRNX1-seg-159', 'NRNX1-seg-276', 'NRNX1-seg-276', 'NRNX1-seg-276', 'NRNX1-seg-276', 'NRNX1-seg-340', 'NRNX1-seg-340', 'NRNX1-seg-340', 'NRNX1-seg-340', 'NRNX1-seg-484', 'NRNX1-seg-484', 'NRNX1-seg-484', 'NRNX1-seg-484', 'NRNX1-seg-564', 'NRNX1-seg-564', 'NRNX1-seg-564', 'NRNX1-seg-564', 'NRNX1-seg-634', 'NRNX1-seg-634', 'NRNX1-seg-634', 'NRNX1-seg-634', 'NRNX1-seg-679', 'NRNX1-seg-679', 'NRNX1-seg-679', 'NRNX1-seg-679'], 'NDB_751': ['NRNX1-seg-64', 'NRNX1-seg-64', 'NRNX1-seg-64', 'NRNX1-seg-64', 'NRNX1-seg-147', 'NRNX1-seg-147', 'NRNX1-seg-147', 'NRNX1-seg-147', 'NRNX1-seg-231', 'NRNX1-seg-231', 'NRNX1-seg-231', 'NRNX1-seg-231', 'NRNX1-seg-286', 'NRNX1-seg-286', 'NRNX1-seg-286', 'NRNX1-seg-286', 'NRNX1-seg-370', 'NRNX1-seg-370', 'NRNX1-seg-370', 'NRNX1-seg-370', 'NRNX1-seg-514', 'NRNX1-seg-514', 'NRNX1-seg-514', 'NRNX1-seg-514', 'NRNX1-seg-574', 'NRNX1-seg-574', 'NRNX1-seg-574', 'NRNX1-seg-574', 'NRNX1-seg-614', 'NRNX1-seg-614', 'NRNX1-seg-614', 'NRNX1-seg-614', 'NRNX1-seg-690', 'NRNX1-seg-690', 'NRNX1-seg-690', 'NRNX1-seg-690'], 'NDB_611': ['NRNX1-seg-64', 'NRNX1-seg-64', 'NRNX1-seg-64', 'NRNX1-seg-64', 'NRNX1-seg-137', 'NRNX1-seg-137', 'NRNX1-seg-137', 'NRNX1-seg-137', 'NRNX1-seg-242', 'NRNX1-seg-242', 'NRNX1-seg-242', 'NRNX1-seg-242', 'NRNX1-seg-298', 'NRNX1-seg-298', 'NRNX1-seg-298', 'NRNX1-seg-298', 'NRNX1-seg-350', 'NRNX1-seg-350', 'NRNX1-seg-350', 'NRNX1-seg-350', 'NRNX1-seg-474', 'NRNX1-seg-474', 'NRNX1-seg-474', 'NRNX1-seg-474', 'NRNX1-seg-544', 'NRNX1-seg-544', 'NRNX1-seg-544', 'NRNX1-seg-544', 'NRNX1-seg-645', 'NRNX1-seg-645', 'NRNX1-seg-645', 'NRNX1-seg-645'], 'NDB_386': ['NRNX1-seg-65', 'NRNX1-seg-65', 'NRNX1-seg-65', 'NRNX1-seg-65', 'NRNX1-seg-160', 'NRNX1-seg-160', 'NRNX1-seg-160', 'NRNX1-seg-160', 'NRNX1-seg-277', 'NRNX1-seg-277', 'NRNX1-seg-277', 'NRNX1-seg-277', 'NRNX1-seg-341', 'NRNX1-seg-341', 'NRNX1-seg-341', 'NRNX1-seg-341', 'NRNX1-seg-485', 'NRNX1-seg-485', 'NRNX1-seg-485', 'NRNX1-seg-485', 'NRNX1-seg-565', 'NRNX1-seg-565', 'NRNX1-seg-565', 'NRNX1-seg-565', 'NRNX1-seg-635', 'NRNX1-seg-635', 'NRNX1-seg-635', 'NRNX1-seg-635', 'NRNX1-seg-681', 'NRNX1-seg-681', 'NRNX1-seg-681', 'NRNX1-seg-681'], 'NDB_517': ['NRNX1-seg-65', 'NRNX1-seg-65', 'NRNX1-seg-65', 'NRNX1-seg-65', 'NRNX1-seg-148', 'NRNX1-seg-148', 'NRNX1-seg-148', 'NRNX1-seg-148', 'NRNX1-seg-232', 'NRNX1-seg-232', 'NRNX1-seg-232', 'NRNX1-seg-232', 'NRNX1-seg-287', 'NRNX1-seg-287', 'NRNX1-seg-287', 'NRNX1-seg-287', 'NRNX1-seg-371', 'NRNX1-seg-371', 'NRNX1-seg-371', 'NRNX1-seg-371', 'NRNX1-seg-515', 'NRNX1-seg-515', 'NRNX1-seg-515', 'NRNX1-seg-515', 'NRNX1-seg-575', 'NRNX1-seg-575', 'NRNX1-seg-575', 'NRNX1-seg-575', 'NRNX1-seg-615', 'NRNX1-seg-615', 'NRNX1-seg-615', 'NRNX1-seg-615', 'NRNX1-seg-691', 'NRNX1-seg-691', 'NRNX1-seg-691', 'NRNX1-seg-691'], 'NDB_614': ['NRNX1-seg-65', 'NRNX1-seg-65', 'NRNX1-seg-65', 'NRNX1-seg-65', 'NRNX1-seg-138', 'NRNX1-seg-138', 'NRNX1-seg-138', 'NRNX1-seg-138', 'NRNX1-seg-244', 'NRNX1-seg-244', 'NRNX1-seg-244', 'NRNX1-seg-244', 'NRNX1-seg-299', 'NRNX1-seg-299', 'NRNX1-seg-299', 'NRNX1-seg-299', 'NRNX1-seg-351', 'NRNX1-seg-351', 'NRNX1-seg-351', 'NRNX1-seg-351', 'NRNX1-seg-475', 'NRNX1-seg-475', 'NRNX1-seg-475', 'NRNX1-seg-475', 'NRNX1-seg-545', 'NRNX1-seg-545', 'NRNX1-seg-545', 'NRNX1-seg-545', 'NRNX1-seg-646', 'NRNX1-seg-646', 'NRNX1-seg-646', 'NRNX1-seg-646'], 'NDB_795': ['NRNX1-seg-66', 'NRNX1-seg-66', 'NRNX1-seg-66', 'NRNX1-seg-66', 'NRNX1-seg-161', 'NRNX1-seg-161', 'NRNX1-seg-161', 'NRNX1-seg-161', 'NRNX1-seg-233', 'NRNX1-seg-233', 'NRNX1-seg-233', 'NRNX1-seg-233', 'NRNX1-seg-288', 'NRNX1-seg-288', 'NRNX1-seg-288', 'NRNX1-seg-288', 'NRNX1-seg-372', 'NRNX1-seg-372', 'NRNX1-seg-372', 'NRNX1-seg-372', 'NRNX1-seg-465', 'NRNX1-seg-465', 'NRNX1-seg-465', 'NRNX1-seg-465', 'NRNX1-seg-506', 'NRNX1-seg-506', 'NRNX1-seg-506', 'NRNX1-seg-506', 'NRNX1-seg-576', 'NRNX1-seg-576', 'NRNX1-seg-576', 'NRNX1-seg-576', 'NRNX1-seg-647', 'NRNX1-seg-647', 'NRNX1-seg-647', 'NRNX1-seg-647'], 'NDB_865': ['NRNX1-seg-66', 'NRNX1-seg-66', 'NRNX1-seg-66', 'NRNX1-seg-66', 'NRNX1-seg-118', 'NRNX1-seg-118', 'NRNX1-seg-118', 'NRNX1-seg-118', 'NRNX1-seg-193', 'NRNX1-seg-193', 'NRNX1-seg-193', 'NRNX1-seg-193', 'NRNX1-seg-268', 'NRNX1-seg-268', 'NRNX1-seg-268', 'NRNX1-seg-268', 'NRNX1-seg-322', 'NRNX1-seg-322', 'NRNX1-seg-322', 'NRNX1-seg-322', 'NRNX1-seg-362', 'NRNX1-seg-362', 'NRNX1-seg-362', 'NRNX1-seg-362', 'NRNX1-seg-476', 'NRNX1-seg-476', 'NRNX1-seg-476', 'NRNX1-seg-476', 'NRNX1-seg-636', 'NRNX1-seg-636', 'NRNX1-seg-636', 'NRNX1-seg-636'], 'NDB_690': ['NRNX1-seg-66', 'NRNX1-seg-66', 'NRNX1-seg-66', 'NRNX1-seg-66', 'NRNX1-seg-107', 'NRNX1-seg-107', 'NRNX1-seg-107', 'NRNX1-seg-107', 'NRNX1-seg-203', 'NRNX1-seg-203', 'NRNX1-seg-203', 'NRNX1-seg-203', 'NRNX1-seg-300', 'NRNX1-seg-300', 'NRNX1-seg-300', 'NRNX1-seg-300', 'NRNX1-seg-352', 'NRNX1-seg-352', 'NRNX1-seg-352', 'NRNX1-seg-352', 'NRNX1-seg-455', 'NRNX1-seg-455', 'NRNX1-seg-455', 'NRNX1-seg-455', 'NRNX1-seg-516', 'NRNX1-seg-516', 'NRNX1-seg-516', 'NRNX1-seg-516', 'NRNX1-seg-657', 'NRNX1-seg-657', 'NRNX1-seg-657', 'NRNX1-seg-657'], 'NDB_738': ['NRNX1-seg-67', 'NRNX1-seg-67', 'NRNX1-seg-67', 'NRNX1-seg-67', 'NRNX1-seg-162', 'NRNX1-seg-162', 'NRNX1-seg-162', 'NRNX1-seg-162', 'NRNX1-seg-234', 'NRNX1-seg-234', 'NRNX1-seg-234', 'NRNX1-seg-234', 'NRNX1-seg-289', 'NRNX1-seg-289', 'NRNX1-seg-289', 'NRNX1-seg-289', 'NRNX1-seg-373', 'NRNX1-seg-373', 'NRNX1-seg-373', 'NRNX1-seg-373', 'NRNX1-seg-466', 'NRNX1-seg-466', 'NRNX1-seg-466', 'NRNX1-seg-466', 'NRNX1-seg-507', 'NRNX1-seg-507', 'NRNX1-seg-507', 'NRNX1-seg-507', 'NRNX1-seg-577', 'NRNX1-seg-577', 'NRNX1-seg-577', 'NRNX1-seg-577', 'NRNX1-seg-648', 'NRNX1-seg-648', 'NRNX1-seg-648', 'NRNX1-seg-648'], 'NDB_790': ['NRNX1-seg-67', 'NRNX1-seg-67', 'NRNX1-seg-67', 'NRNX1-seg-67', 'NRNX1-seg-119', 'NRNX1-seg-119', 'NRNX1-seg-119', 'NRNX1-seg-119', 'NRNX1-seg-194', 'NRNX1-seg-194', 'NRNX1-seg-194', 'NRNX1-seg-194', 'NRNX1-seg-269', 'NRNX1-seg-269', 'NRNX1-seg-269', 'NRNX1-seg-269', 'NRNX1-seg-323', 'NRNX1-seg-323', 'NRNX1-seg-323', 'NRNX1-seg-323', 'NRNX1-seg-363', 'NRNX1-seg-363', 'NRNX1-seg-363', 'NRNX1-seg-363', 'NRNX1-seg-477', 'NRNX1-seg-477', 'NRNX1-seg-477', 'NRNX1-seg-477', 'NRNX1-seg-637', 'NRNX1-seg-637', 'NRNX1-seg-637', 'NRNX1-seg-637'], 'NDB_852': ['NRNX1-seg-67', 'NRNX1-seg-67', 'NRNX1-seg-67', 'NRNX1-seg-67', 'NRNX1-seg-108', 'NRNX1-seg-108', 'NRNX1-seg-108', 'NRNX1-seg-108', 'NRNX1-seg-204', 'NRNX1-seg-204', 'NRNX1-seg-204', 'NRNX1-seg-204', 'NRNX1-seg-301', 'NRNX1-seg-301', 'NRNX1-seg-301', 'NRNX1-seg-301', 'NRNX1-seg-353', 'NRNX1-seg-353', 'NRNX1-seg-353', 'NRNX1-seg-353', 'NRNX1-seg-456', 'NRNX1-seg-456', 'NRNX1-seg-456', 'NRNX1-seg-456', 'NRNX1-seg-517', 'NRNX1-seg-517', 'NRNX1-seg-517', 'NRNX1-seg-517', 'NRNX1-seg-658', 'NRNX1-seg-658', 'NRNX1-seg-658', 'NRNX1-seg-658'], 'NDB_387': ['NRNX1-seg-68', 'NRNX1-seg-68', 'NRNX1-seg-68', 'NRNX1-seg-68', 'NRNX1-seg-163', 'NRNX1-seg-163', 'NRNX1-seg-163', 'NRNX1-seg-163', 'NRNX1-seg-235', 'NRNX1-seg-235', 'NRNX1-seg-235', 'NRNX1-seg-235', 'NRNX1-seg-290', 'NRNX1-seg-290', 'NRNX1-seg-290', 'NRNX1-seg-290', 'NRNX1-seg-374', 'NRNX1-seg-374', 'NRNX1-seg-374', 'NRNX1-seg-374', 'NRNX1-seg-467', 'NRNX1-seg-467', 'NRNX1-seg-467', 'NRNX1-seg-467', 'NRNX1-seg-508', 'NRNX1-seg-508', 'NRNX1-seg-508', 'NRNX1-seg-508', 'NRNX1-seg-578', 'NRNX1-seg-578', 'NRNX1-seg-578', 'NRNX1-seg-578', 'NRNX1-seg-649', 'NRNX1-seg-649', 'NRNX1-seg-649', 'NRNX1-seg-649'], 'NDB_901': ['NRNX1-seg-68', 'NRNX1-seg-68', 'NRNX1-seg-68', 'NRNX1-seg-68', 'NRNX1-seg-120', 'NRNX1-seg-120', 'NRNX1-seg-120', 'NRNX1-seg-120', 'NRNX1-seg-195', 'NRNX1-seg-195', 'NRNX1-seg-195', 'NRNX1-seg-195', 'NRNX1-seg-270', 'NRNX1-seg-270', 'NRNX1-seg-270', 'NRNX1-seg-270', 'NRNX1-seg-324', 'NRNX1-seg-324', 'NRNX1-seg-324', 'NRNX1-seg-324', 'NRNX1-seg-364', 'NRNX1-seg-364', 'NRNX1-seg-364', 'NRNX1-seg-364', 'NRNX1-seg-478', 'NRNX1-seg-478', 'NRNX1-seg-478', 'NRNX1-seg-478', 'NRNX1-seg-639', 'NRNX1-seg-639', 'NRNX1-seg-639', 'NRNX1-seg-639'], 'NDB_432': ['NRNX1-seg-68', 'NRNX1-seg-68', 'NRNX1-seg-68', 'NRNX1-seg-68', 'NRNX1-seg-109', 'NRNX1-seg-109', 'NRNX1-seg-109', 'NRNX1-seg-109', 'NRNX1-seg-205', 'NRNX1-seg-205', 'NRNX1-seg-205', 'NRNX1-seg-205', 'NRNX1-seg-302', 'NRNX1-seg-302', 'NRNX1-seg-302', 'NRNX1-seg-302', 'NRNX1-seg-354', 'NRNX1-seg-354', 'NRNX1-seg-354', 'NRNX1-seg-354', 'NRNX1-seg-457', 'NRNX1-seg-457', 'NRNX1-seg-457', 'NRNX1-seg-457', 'NRNX1-seg-518', 'NRNX1-seg-518', 'NRNX1-seg-518', 'NRNX1-seg-518', 'NRNX1-seg-659', 'NRNX1-seg-659', 'NRNX1-seg-659', 'NRNX1-seg-659'], 'NDB_465': ['NRNX1-seg-69', 'NRNX1-seg-69', 'NRNX1-seg-69', 'NRNX1-seg-69', 'NRNX1-seg-164', 'NRNX1-seg-164', 'NRNX1-seg-164', 'NRNX1-seg-164', 'NRNX1-seg-236', 'NRNX1-seg-236', 'NRNX1-seg-236', 'NRNX1-seg-236', 'NRNX1-seg-291', 'NRNX1-seg-291', 'NRNX1-seg-291', 'NRNX1-seg-291', 'NRNX1-seg-375', 'NRNX1-seg-375', 'NRNX1-seg-375', 'NRNX1-seg-375', 'NRNX1-seg-468', 'NRNX1-seg-468', 'NRNX1-seg-468', 'NRNX1-seg-468', 'NRNX1-seg-509', 'NRNX1-seg-509', 'NRNX1-seg-509', 'NRNX1-seg-509', 'NRNX1-seg-579', 'NRNX1-seg-579', 'NRNX1-seg-579', 'NRNX1-seg-579', 'NRNX1-seg-650', 'NRNX1-seg-650', 'NRNX1-seg-650', 'NRNX1-seg-650'], 'NDB_403': ['NRNX1-seg-69', 'NRNX1-seg-69', 'NRNX1-seg-69', 'NRNX1-seg-69', 'NRNX1-seg-121', 'NRNX1-seg-121', 'NRNX1-seg-121', 'NRNX1-seg-121', 'NRNX1-seg-196', 'NRNX1-seg-196', 'NRNX1-seg-196', 'NRNX1-seg-196', 'NRNX1-seg-271', 'NRNX1-seg-271', 'NRNX1-seg-271', 'NRNX1-seg-271', 'NRNX1-seg-325', 'NRNX1-seg-325', 'NRNX1-seg-325', 'NRNX1-seg-325', 'NRNX1-seg-365', 'NRNX1-seg-365', 'NRNX1-seg-365', 'NRNX1-seg-365', 'NRNX1-seg-479', 'NRNX1-seg-479', 'NRNX1-seg-479', 'NRNX1-seg-479', 'NRNX1-seg-640', 'NRNX1-seg-640', 'NRNX1-seg-640', 'NRNX1-seg-640'], 'NDB_1020': ['NRNX1-seg-69', 'NRNX1-seg-69', 'NRNX1-seg-69', 'NRNX1-seg-69', 'NRNX1-seg-110', 'NRNX1-seg-110', 'NRNX1-seg-110', 'NRNX1-seg-110', 'NRNX1-seg-206', 'NRNX1-seg-206', 'NRNX1-seg-206', 'NRNX1-seg-206', 'NRNX1-seg-303', 'NRNX1-seg-303', 'NRNX1-seg-303', 'NRNX1-seg-303', 'NRNX1-seg-355', 'NRNX1-seg-355', 'NRNX1-seg-355', 'NRNX1-seg-355', 'NRNX1-seg-458', 'NRNX1-seg-458', 'NRNX1-seg-458', 'NRNX1-seg-458', 'NRNX1-seg-519', 'NRNX1-seg-519', 'NRNX1-seg-519', 'NRNX1-seg-519', 'NRNX1-seg-660', 'NRNX1-seg-660', 'NRNX1-seg-660', 'NRNX1-seg-660'], 'NDB_1035': ['NRNX1-seg-70', 'NRNX1-seg-70', 'NRNX1-seg-70', 'NRNX1-seg-70', 'NRNX1-seg-165', 'NRNX1-seg-165', 'NRNX1-seg-165', 'NRNX1-seg-165', 'NRNX1-seg-238', 'NRNX1-seg-238', 'NRNX1-seg-238', 'NRNX1-seg-238', 'NRNX1-seg-292', 'NRNX1-seg-292', 'NRNX1-seg-292', 'NRNX1-seg-292', 'NRNX1-seg-376', 'NRNX1-seg-376', 'NRNX1-seg-376', 'NRNX1-seg-376', 'NRNX1-seg-469', 'NRNX1-seg-469', 'NRNX1-seg-469', 'NRNX1-seg-469', 'NRNX1-seg-510', 'NRNX1-seg-510', 'NRNX1-seg-510', 'NRNX1-seg-510', 'NRNX1-seg-580', 'NRNX1-seg-580', 'NRNX1-seg-580', 'NRNX1-seg-580', 'NRNX1-seg-651', 'NRNX1-seg-651', 'NRNX1-seg-651', 'NRNX1-seg-651'], 'NDB_1033': ['NRNX1-seg-70', 'NRNX1-seg-70', 'NRNX1-seg-70', 'NRNX1-seg-70', 'NRNX1-seg-122', 'NRNX1-seg-122', 'NRNX1-seg-122', 'NRNX1-seg-122', 'NRNX1-seg-197', 'NRNX1-seg-197', 'NRNX1-seg-197', 'NRNX1-seg-197', 'NRNX1-seg-272', 'NRNX1-seg-272', 'NRNX1-seg-272', 'NRNX1-seg-272', 'NRNX1-seg-326', 'NRNX1-seg-326', 'NRNX1-seg-326', 'NRNX1-seg-326', 'NRNX1-seg-366', 'NRNX1-seg-366', 'NRNX1-seg-366', 'NRNX1-seg-366', 'NRNX1-seg-480', 'NRNX1-seg-480', 'NRNX1-seg-480', 'NRNX1-seg-480', 'NRNX1-seg-641', 'NRNX1-seg-641', 'NRNX1-seg-641', 'NRNX1-seg-641'], 'NDB_741': ['NRNX1-seg-70', 'NRNX1-seg-70', 'NRNX1-seg-70', 'NRNX1-seg-70', 'NRNX1-seg-111', 'NRNX1-seg-111', 'NRNX1-seg-111', 'NRNX1-seg-111', 'NRNX1-seg-207', 'NRNX1-seg-207', 'NRNX1-seg-207', 'NRNX1-seg-207', 'NRNX1-seg-304', 'NRNX1-seg-304', 'NRNX1-seg-304', 'NRNX1-seg-304', 'NRNX1-seg-356', 'NRNX1-seg-356', 'NRNX1-seg-356', 'NRNX1-seg-356', 'NRNX1-seg-459', 'NRNX1-seg-459', 'NRNX1-seg-459', 'NRNX1-seg-459', 'NRNX1-seg-520', 'NRNX1-seg-520', 'NRNX1-seg-520', 'NRNX1-seg-520', 'NRNX1-seg-661', 'NRNX1-seg-661', 'NRNX1-seg-661', 'NRNX1-seg-661'], 'NDB_921': ['NRNX1-seg-71', 'NRNX1-seg-71', 'NRNX1-seg-71', 'NRNX1-seg-71', 'NRNX1-seg-166', 'NRNX1-seg-166', 'NRNX1-seg-166', 'NRNX1-seg-166', 'NRNX1-seg-239', 'NRNX1-seg-239', 'NRNX1-seg-239', 'NRNX1-seg-239', 'NRNX1-seg-293', 'NRNX1-seg-293', 'NRNX1-seg-293', 'NRNX1-seg-293', 'NRNX1-seg-379', 'NRNX1-seg-379', 'NRNX1-seg-379', 'NRNX1-seg-379', 'NRNX1-seg-470', 'NRNX1-seg-470', 'NRNX1-seg-470', 'NRNX1-seg-470', 'NRNX1-seg-511', 'NRNX1-seg-511', 'NRNX1-seg-511', 'NRNX1-seg-511', 'NRNX1-seg-581', 'NRNX1-seg-581', 'NRNX1-seg-581', 'NRNX1-seg-581', 'NRNX1-seg-652', 'NRNX1-seg-652', 'NRNX1-seg-652', 'NRNX1-seg-652'], 'NDB_619': ['NRNX1-seg-71', 'NRNX1-seg-71', 'NRNX1-seg-71', 'NRNX1-seg-71', 'NRNX1-seg-123', 'NRNX1-seg-123', 'NRNX1-seg-123', 'NRNX1-seg-123', 'NRNX1-seg-198', 'NRNX1-seg-198', 'NRNX1-seg-198', 'NRNX1-seg-198', 'NRNX1-seg-273', 'NRNX1-seg-273', 'NRNX1-seg-273', 'NRNX1-seg-273', 'NRNX1-seg-327', 'NRNX1-seg-327', 'NRNX1-seg-327', 'NRNX1-seg-327', 'NRNX1-seg-367', 'NRNX1-seg-367', 'NRNX1-seg-367', 'NRNX1-seg-367', 'NRNX1-seg-481', 'NRNX1-seg-481', 'NRNX1-seg-481', 'NRNX1-seg-481', 'NRNX1-seg-642', 'NRNX1-seg-642', 'NRNX1-seg-642', 'NRNX1-seg-642'], 'NDB_507': ['NRNX1-seg-71', 'NRNX1-seg-71', 'NRNX1-seg-71', 'NRNX1-seg-71', 'NRNX1-seg-112', 'NRNX1-seg-112', 'NRNX1-seg-112', 'NRNX1-seg-112', 'NRNX1-seg-208', 'NRNX1-seg-208', 'NRNX1-seg-208', 'NRNX1-seg-208', 'NRNX1-seg-305', 'NRNX1-seg-305', 'NRNX1-seg-305', 'NRNX1-seg-305', 'NRNX1-seg-357', 'NRNX1-seg-357', 'NRNX1-seg-357', 'NRNX1-seg-357', 'NRNX1-seg-460', 'NRNX1-seg-460', 'NRNX1-seg-460', 'NRNX1-seg-460', 'NRNX1-seg-521', 'NRNX1-seg-521', 'NRNX1-seg-521', 'NRNX1-seg-521', 'NRNX1-seg-662', 'NRNX1-seg-662', 'NRNX1-seg-662', 'NRNX1-seg-662'], 'NDB_864': ['NRNX1-seg-72', 'NRNX1-seg-72', 'NRNX1-seg-72', 'NRNX1-seg-72', 'NRNX1-seg-169', 'NRNX1-seg-169', 'NRNX1-seg-169', 'NRNX1-seg-169', 'NRNX1-seg-240', 'NRNX1-seg-240', 'NRNX1-seg-240', 'NRNX1-seg-240', 'NRNX1-seg-294', 'NRNX1-seg-294', 'NRNX1-seg-294', 'NRNX1-seg-294', 'NRNX1-seg-380', 'NRNX1-seg-380', 'NRNX1-seg-380', 'NRNX1-seg-380', 'NRNX1-seg-471', 'NRNX1-seg-471', 'NRNX1-seg-471', 'NRNX1-seg-471', 'NRNX1-seg-512', 'NRNX1-seg-512', 'NRNX1-seg-512', 'NRNX1-seg-512', 'NRNX1-seg-582', 'NRNX1-seg-582', 'NRNX1-seg-582', 'NRNX1-seg-582', 'NRNX1-seg-653', 'NRNX1-seg-653', 'NRNX1-seg-653', 'NRNX1-seg-653'], 'NDB_946': ['NRNX1-seg-72', 'NRNX1-seg-72', 'NRNX1-seg-72', 'NRNX1-seg-72', 'NRNX1-seg-124', 'NRNX1-seg-124', 'NRNX1-seg-124', 'NRNX1-seg-124', 'NRNX1-seg-199', 'NRNX1-seg-199', 'NRNX1-seg-199', 'NRNX1-seg-199', 'NRNX1-seg-274', 'NRNX1-seg-274', 'NRNX1-seg-274', 'NRNX1-seg-274', 'NRNX1-seg-328', 'NRNX1-seg-328', 'NRNX1-seg-328', 'NRNX1-seg-328', 'NRNX1-seg-368', 'NRNX1-seg-368', 'NRNX1-seg-368', 'NRNX1-seg-368', 'NRNX1-seg-482', 'NRNX1-seg-482', 'NRNX1-seg-482', 'NRNX1-seg-482', 'NRNX1-seg-643', 'NRNX1-seg-643', 'NRNX1-seg-643', 'NRNX1-seg-643'], 'NDB_816': ['NRNX1-seg-72', 'NRNX1-seg-72', 'NRNX1-seg-72', 'NRNX1-seg-72', 'NRNX1-seg-113', 'NRNX1-seg-113', 'NRNX1-seg-113', 'NRNX1-seg-113', 'NRNX1-seg-209', 'NRNX1-seg-209', 'NRNX1-seg-209', 'NRNX1-seg-209', 'NRNX1-seg-306', 'NRNX1-seg-306', 'NRNX1-seg-306', 'NRNX1-seg-306', 'NRNX1-seg-358', 'NRNX1-seg-358', 'NRNX1-seg-358', 'NRNX1-seg-358', 'NRNX1-seg-461', 'NRNX1-seg-461', 'NRNX1-seg-461', 'NRNX1-seg-461', 'NRNX1-seg-522', 'NRNX1-seg-522', 'NRNX1-seg-522', 'NRNX1-seg-522', 'NRNX1-seg-665', 'NRNX1-seg-665', 'NRNX1-seg-665', 'NRNX1-seg-665'], 'NDB_498': ['NRNX1-seg-73', 'NRNX1-seg-73', 'NRNX1-seg-73', 'NRNX1-seg-73', 'NRNX1-seg-170', 'NRNX1-seg-170', 'NRNX1-seg-170', 'NRNX1-seg-170', 'NRNX1-seg-241', 'NRNX1-seg-241', 'NRNX1-seg-241', 'NRNX1-seg-241', 'NRNX1-seg-295', 'NRNX1-seg-295', 'NRNX1-seg-295', 'NRNX1-seg-295', 'NRNX1-seg-381', 'NRNX1-seg-381', 'NRNX1-seg-381', 'NRNX1-seg-381', 'NRNX1-seg-472', 'NRNX1-seg-472', 'NRNX1-seg-472', 'NRNX1-seg-472', 'NRNX1-seg-513', 'NRNX1-seg-513', 'NRNX1-seg-513', 'NRNX1-seg-513', 'NRNX1-seg-583', 'NRNX1-seg-583', 'NRNX1-seg-583', 'NRNX1-seg-583', 'NRNX1-seg-654', 'NRNX1-seg-654', 'NRNX1-seg-654', 'NRNX1-seg-654'], 'NDB_1066': ['NRNX1-seg-73', 'NRNX1-seg-73', 'NRNX1-seg-73', 'NRNX1-seg-73', 'NRNX1-seg-125', 'NRNX1-seg-125', 'NRNX1-seg-125', 'NRNX1-seg-125', 'NRNX1-seg-200', 'NRNX1-seg-200', 'NRNX1-seg-200', 'NRNX1-seg-200', 'NRNX1-seg-275', 'NRNX1-seg-275', 'NRNX1-seg-275', 'NRNX1-seg-275', 'NRNX1-seg-329', 'NRNX1-seg-329', 'NRNX1-seg-329', 'NRNX1-seg-329', 'NRNX1-seg-369', 'NRNX1-seg-369', 'NRNX1-seg-369', 'NRNX1-seg-369', 'NRNX1-seg-483', 'NRNX1-seg-483', 'NRNX1-seg-483', 'NRNX1-seg-483', 'NRNX1-seg-644', 'NRNX1-seg-644', 'NRNX1-seg-644', 'NRNX1-seg-644'], 'NDB_660': ['NRNX1-seg-73', 'NRNX1-seg-73', 'NRNX1-seg-73', 'NRNX1-seg-73', 'NRNX1-seg-114', 'NRNX1-seg-114', 'NRNX1-seg-114', 'NRNX1-seg-114', 'NRNX1-seg-210', 'NRNX1-seg-210', 'NRNX1-seg-210', 'NRNX1-seg-210', 'NRNX1-seg-307', 'NRNX1-seg-307', 'NRNX1-seg-307', 'NRNX1-seg-307', 'NRNX1-seg-359', 'NRNX1-seg-359', 'NRNX1-seg-359', 'NRNX1-seg-359', 'NRNX1-seg-462', 'NRNX1-seg-462', 'NRNX1-seg-462', 'NRNX1-seg-462', 'NRNX1-seg-523', 'NRNX1-seg-523', 'NRNX1-seg-523', 'NRNX1-seg-523', 'NRNX1-seg-666', 'NRNX1-seg-666', 'NRNX1-seg-666', 'NRNX1-seg-666'], 'NDB_459': ['NRNX1-seg-74', 'NRNX1-seg-74', 'NRNX1-seg-74', 'NRNX1-seg-74', 'NRNX1-seg-181', 'NRNX1-seg-181', 'NRNX1-seg-181', 'NRNX1-seg-181', 'NRNX1-seg-286', 'NRNX1-seg-286', 'NRNX1-seg-286', 'NRNX1-seg-286', 'NRNX1-seg-340', 'NRNX1-seg-340', 'NRNX1-seg-340', 'NRNX1-seg-340', 'NRNX1-seg-392', 'NRNX1-seg-392', 'NRNX1-seg-392', 'NRNX1-seg-392', 'NRNX1-seg-474', 'NRNX1-seg-474', 'NRNX1-seg-474', 'NRNX1-seg-474', 'NRNX1-seg-524', 'NRNX1-seg-524', 'NRNX1-seg-524', 'NRNX1-seg-524', 'NRNX1-seg-604', 'NRNX1-seg-604', 'NRNX1-seg-604', 'NRNX1-seg-604'], 'NDB_879': ['NRNX1-seg-74', 'NRNX1-seg-74', 'NRNX1-seg-74', 'NRNX1-seg-74', 'NRNX1-seg-115', 'NRNX1-seg-115', 'NRNX1-seg-115', 'NRNX1-seg-115', 'NRNX1-seg-191', 'NRNX1-seg-191', 'NRNX1-seg-191', 'NRNX1-seg-191', 'NRNX1-seg-255', 'NRNX1-seg-255', 'NRNX1-seg-255', 'NRNX1-seg-255', 'NRNX1-seg-370', 'NRNX1-seg-370', 'NRNX1-seg-370', 'NRNX1-seg-370', 'NRNX1-seg-432', 'NRNX1-seg-432', 'NRNX1-seg-432', 'NRNX1-seg-432', 'NRNX1-seg-584', 'NRNX1-seg-584', 'NRNX1-seg-584', 'NRNX1-seg-584', 'NRNX1-seg-655', 'NRNX1-seg-655', 'NRNX1-seg-655', 'NRNX1-seg-655'], 'NDB_341': ['NRNX1-seg-74', 'NRNX1-seg-74', 'NRNX1-seg-74', 'NRNX1-seg-74', 'NRNX1-seg-147', 'NRNX1-seg-147', 'NRNX1-seg-147', 'NRNX1-seg-147', 'NRNX1-seg-221', 'NRNX1-seg-221', 'NRNX1-seg-221', 'NRNX1-seg-221', 'NRNX1-seg-320', 'NRNX1-seg-320', 'NRNX1-seg-320', 'NRNX1-seg-320', 'NRNX1-seg-422', 'NRNX1-seg-422', 'NRNX1-seg-422', 'NRNX1-seg-422', 'NRNX1-seg-504', 'NRNX1-seg-504', 'NRNX1-seg-504', 'NRNX1-seg-504', 'NRNX1-seg-564', 'NRNX1-seg-564', 'NRNX1-seg-564', 'NRNX1-seg-564', 'NRNX1-seg-624', 'NRNX1-seg-624', 'NRNX1-seg-624', 'NRNX1-seg-624'], 'NDB_933': ['NRNX1-seg-75', 'NRNX1-seg-75', 'NRNX1-seg-75', 'NRNX1-seg-75', 'NRNX1-seg-182', 'NRNX1-seg-182', 'NRNX1-seg-182', 'NRNX1-seg-182', 'NRNX1-seg-287', 'NRNX1-seg-287', 'NRNX1-seg-287', 'NRNX1-seg-287', 'NRNX1-seg-341', 'NRNX1-seg-341', 'NRNX1-seg-341', 'NRNX1-seg-341', 'NRNX1-seg-393', 'NRNX1-seg-393', 'NRNX1-seg-393', 'NRNX1-seg-393', 'NRNX1-seg-475', 'NRNX1-seg-475', 'NRNX1-seg-475', 'NRNX1-seg-475', 'NRNX1-seg-525', 'NRNX1-seg-525', 'NRNX1-seg-525', 'NRNX1-seg-525', 'NRNX1-seg-605', 'NRNX1-seg-605', 'NRNX1-seg-605', 'NRNX1-seg-605'], 'NDB_840': ['NRNX1-seg-75', 'NRNX1-seg-75', 'NRNX1-seg-75', 'NRNX1-seg-75', 'NRNX1-seg-117', 'NRNX1-seg-117', 'NRNX1-seg-117', 'NRNX1-seg-117', 'NRNX1-seg-192', 'NRNX1-seg-192', 'NRNX1-seg-192', 'NRNX1-seg-192', 'NRNX1-seg-256', 'NRNX1-seg-256', 'NRNX1-seg-256', 'NRNX1-seg-256', 'NRNX1-seg-371', 'NRNX1-seg-371', 'NRNX1-seg-371', 'NRNX1-seg-371', 'NRNX1-seg-433', 'NRNX1-seg-433', 'NRNX1-seg-433', 'NRNX1-seg-433', 'NRNX1-seg-585', 'NRNX1-seg-585', 'NRNX1-seg-585', 'NRNX1-seg-585', 'NRNX1-seg-656', 'NRNX1-seg-656', 'NRNX1-seg-656', 'NRNX1-seg-656'], 'NDB_395': ['NRNX1-seg-75', 'NRNX1-seg-75', 'NRNX1-seg-75', 'NRNX1-seg-75', 'NRNX1-seg-148', 'NRNX1-seg-148', 'NRNX1-seg-148', 'NRNX1-seg-148', 'NRNX1-seg-222', 'NRNX1-seg-222', 'NRNX1-seg-222', 'NRNX1-seg-222', 'NRNX1-seg-321', 'NRNX1-seg-321', 'NRNX1-seg-321', 'NRNX1-seg-321', 'NRNX1-seg-423', 'NRNX1-seg-423', 'NRNX1-seg-423', 'NRNX1-seg-423', 'NRNX1-seg-505', 'NRNX1-seg-505', 'NRNX1-seg-505', 'NRNX1-seg-505', 'NRNX1-seg-565', 'NRNX1-seg-565', 'NRNX1-seg-565', 'NRNX1-seg-565', 'NRNX1-seg-625', 'NRNX1-seg-625', 'NRNX1-seg-625', 'NRNX1-seg-625'], 'NDB_699': ['NRNX1-seg-76', 'NRNX1-seg-76', 'NRNX1-seg-76', 'NRNX1-seg-76', 'NRNX1-seg-139', 'NRNX1-seg-139', 'NRNX1-seg-139', 'NRNX1-seg-139', 'NRNX1-seg-223', 'NRNX1-seg-223', 'NRNX1-seg-223', 'NRNX1-seg-223', 'NRNX1-seg-444', 'NRNX1-seg-444', 'NRNX1-seg-444', 'NRNX1-seg-444', 'NRNX1-seg-486', 'NRNX1-seg-486', 'NRNX1-seg-486', 'NRNX1-seg-486', 'NRNX1-seg-556', 'NRNX1-seg-556', 'NRNX1-seg-556', 'NRNX1-seg-556', 'NRNX1-seg-596', 'NRNX1-seg-596', 'NRNX1-seg-596', 'NRNX1-seg-596', 'NRNX1-seg-657', 'NRNX1-seg-657', 'NRNX1-seg-657', 'NRNX1-seg-657'], 'NDB_847': ['NRNX1-seg-76', 'NRNX1-seg-76', 'NRNX1-seg-76', 'NRNX1-seg-76', 'NRNX1-seg-203', 'NRNX1-seg-203', 'NRNX1-seg-203', 'NRNX1-seg-203', 'NRNX1-seg-332', 'NRNX1-seg-332', 'NRNX1-seg-332', 'NRNX1-seg-332', 'NRNX1-seg-384', 'NRNX1-seg-384', 'NRNX1-seg-384', 'NRNX1-seg-384', 'NRNX1-seg-434', 'NRNX1-seg-434', 'NRNX1-seg-434', 'NRNX1-seg-434', 'NRNX1-seg-496', 'NRNX1-seg-496', 'NRNX1-seg-496', 'NRNX1-seg-496', 'NRNX1-seg-576', 'NRNX1-seg-576', 'NRNX1-seg-576', 'NRNX1-seg-576', 'NRNX1-seg-636', 'NRNX1-seg-636', 'NRNX1-seg-636', 'NRNX1-seg-636'], 'NDB_762': ['NRNX1-seg-77', 'NRNX1-seg-77', 'NRNX1-seg-77', 'NRNX1-seg-77', 'NRNX1-seg-140', 'NRNX1-seg-140', 'NRNX1-seg-140', 'NRNX1-seg-140', 'NRNX1-seg-224', 'NRNX1-seg-224', 'NRNX1-seg-224', 'NRNX1-seg-224', 'NRNX1-seg-445', 'NRNX1-seg-445', 'NRNX1-seg-445', 'NRNX1-seg-445', 'NRNX1-seg-487', 'NRNX1-seg-487', 'NRNX1-seg-487', 'NRNX1-seg-487', 'NRNX1-seg-557', 'NRNX1-seg-557', 'NRNX1-seg-557', 'NRNX1-seg-557', 'NRNX1-seg-597', 'NRNX1-seg-597', 'NRNX1-seg-597', 'NRNX1-seg-597', 'NRNX1-seg-658', 'NRNX1-seg-658', 'NRNX1-seg-658', 'NRNX1-seg-658'], 'NDB_661': ['NRNX1-seg-77', 'NRNX1-seg-77', 'NRNX1-seg-77', 'NRNX1-seg-77', 'NRNX1-seg-204', 'NRNX1-seg-204', 'NRNX1-seg-204', 'NRNX1-seg-204', 'NRNX1-seg-333', 'NRNX1-seg-333', 'NRNX1-seg-333', 'NRNX1-seg-333', 'NRNX1-seg-385', 'NRNX1-seg-385', 'NRNX1-seg-385', 'NRNX1-seg-385', 'NRNX1-seg-435', 'NRNX1-seg-435', 'NRNX1-seg-435', 'NRNX1-seg-435', 'NRNX1-seg-497', 'NRNX1-seg-497', 'NRNX1-seg-497', 'NRNX1-seg-497', 'NRNX1-seg-577', 'NRNX1-seg-577', 'NRNX1-seg-577', 'NRNX1-seg-577', 'NRNX1-seg-637', 'NRNX1-seg-637', 'NRNX1-seg-637', 'NRNX1-seg-637'], 'NDB_786': ['NRNX1-seg-79', 'NRNX1-seg-79', 'NRNX1-seg-79', 'NRNX1-seg-79', 'NRNX1-seg-141', 'NRNX1-seg-141', 'NRNX1-seg-141', 'NRNX1-seg-141', 'NRNX1-seg-225', 'NRNX1-seg-225', 'NRNX1-seg-225', 'NRNX1-seg-225', 'NRNX1-seg-446', 'NRNX1-seg-446', 'NRNX1-seg-446', 'NRNX1-seg-446', 'NRNX1-seg-488', 'NRNX1-seg-488', 'NRNX1-seg-488', 'NRNX1-seg-488', 'NRNX1-seg-558', 'NRNX1-seg-558', 'NRNX1-seg-558', 'NRNX1-seg-558', 'NRNX1-seg-598', 'NRNX1-seg-598', 'NRNX1-seg-598', 'NRNX1-seg-598', 'NRNX1-seg-659', 'NRNX1-seg-659', 'NRNX1-seg-659', 'NRNX1-seg-659'], 'NDB_448': ['NRNX1-seg-79', 'NRNX1-seg-79', 'NRNX1-seg-79', 'NRNX1-seg-79', 'NRNX1-seg-205', 'NRNX1-seg-205', 'NRNX1-seg-205', 'NRNX1-seg-205', 'NRNX1-seg-334', 'NRNX1-seg-334', 'NRNX1-seg-334', 'NRNX1-seg-334', 'NRNX1-seg-386', 'NRNX1-seg-386', 'NRNX1-seg-386', 'NRNX1-seg-386', 'NRNX1-seg-436', 'NRNX1-seg-436', 'NRNX1-seg-436', 'NRNX1-seg-436', 'NRNX1-seg-498', 'NRNX1-seg-498', 'NRNX1-seg-498', 'NRNX1-seg-498', 'NRNX1-seg-578', 'NRNX1-seg-578', 'NRNX1-seg-578', 'NRNX1-seg-578', 'NRNX1-seg-639', 'NRNX1-seg-639', 'NRNX1-seg-639', 'NRNX1-seg-639'], 'NDB_681': ['NRNX1-seg-80', 'NRNX1-seg-80', 'NRNX1-seg-80', 'NRNX1-seg-80', 'NRNX1-seg-142', 'NRNX1-seg-142', 'NRNX1-seg-142', 'NRNX1-seg-142', 'NRNX1-seg-226', 'NRNX1-seg-226', 'NRNX1-seg-226', 'NRNX1-seg-226', 'NRNX1-seg-447', 'NRNX1-seg-447', 'NRNX1-seg-447', 'NRNX1-seg-447', 'NRNX1-seg-489', 'NRNX1-seg-489', 'NRNX1-seg-489', 'NRNX1-seg-489', 'NRNX1-seg-559', 'NRNX1-seg-559', 'NRNX1-seg-559', 'NRNX1-seg-559', 'NRNX1-seg-599', 'NRNX1-seg-599', 'NRNX1-seg-599', 'NRNX1-seg-599', 'NRNX1-seg-660', 'NRNX1-seg-660', 'NRNX1-seg-660', 'NRNX1-seg-660'], 'NDB_391': ['NRNX1-seg-80', 'NRNX1-seg-80', 'NRNX1-seg-80', 'NRNX1-seg-80', 'NRNX1-seg-206', 'NRNX1-seg-206', 'NRNX1-seg-206', 'NRNX1-seg-206', 'NRNX1-seg-335', 'NRNX1-seg-335', 'NRNX1-seg-335', 'NRNX1-seg-335', 'NRNX1-seg-387', 'NRNX1-seg-387', 'NRNX1-seg-387', 'NRNX1-seg-387', 'NRNX1-seg-437', 'NRNX1-seg-437', 'NRNX1-seg-437', 'NRNX1-seg-437', 'NRNX1-seg-499', 'NRNX1-seg-499', 'NRNX1-seg-499', 'NRNX1-seg-499', 'NRNX1-seg-579', 'NRNX1-seg-579', 'NRNX1-seg-579', 'NRNX1-seg-579', 'NRNX1-seg-640', 'NRNX1-seg-640', 'NRNX1-seg-640', 'NRNX1-seg-640'], 'NDB_444': ['NRNX1-seg-81', 'NRNX1-seg-81', 'NRNX1-seg-81', 'NRNX1-seg-81', 'NRNX1-seg-143', 'NRNX1-seg-143', 'NRNX1-seg-143', 'NRNX1-seg-143', 'NRNX1-seg-227', 'NRNX1-seg-227', 'NRNX1-seg-227', 'NRNX1-seg-227', 'NRNX1-seg-448', 'NRNX1-seg-448', 'NRNX1-seg-448', 'NRNX1-seg-448', 'NRNX1-seg-490', 'NRNX1-seg-490', 'NRNX1-seg-490', 'NRNX1-seg-490', 'NRNX1-seg-560', 'NRNX1-seg-560', 'NRNX1-seg-560', 'NRNX1-seg-560', 'NRNX1-seg-600', 'NRNX1-seg-600', 'NRNX1-seg-600', 'NRNX1-seg-600', 'NRNX1-seg-661', 'NRNX1-seg-661', 'NRNX1-seg-661', 'NRNX1-seg-661'], 'NDB_1075': ['NRNX1-seg-81', 'NRNX1-seg-81', 'NRNX1-seg-81', 'NRNX1-seg-81', 'NRNX1-seg-207', 'NRNX1-seg-207', 'NRNX1-seg-207', 'NRNX1-seg-207', 'NRNX1-seg-336', 'NRNX1-seg-336', 'NRNX1-seg-336', 'NRNX1-seg-336', 'NRNX1-seg-388', 'NRNX1-seg-388', 'NRNX1-seg-388', 'NRNX1-seg-388', 'NRNX1-seg-438', 'NRNX1-seg-438', 'NRNX1-seg-438', 'NRNX1-seg-438', 'NRNX1-seg-500', 'NRNX1-seg-500', 'NRNX1-seg-500', 'NRNX1-seg-500', 'NRNX1-seg-580', 'NRNX1-seg-580', 'NRNX1-seg-580', 'NRNX1-seg-580', 'NRNX1-seg-641', 'NRNX1-seg-641', 'NRNX1-seg-641', 'NRNX1-seg-641'], 'NDB_747': ['NRNX1-seg-82', 'NRNX1-seg-82', 'NRNX1-seg-82', 'NRNX1-seg-82', 'NRNX1-seg-144', 'NRNX1-seg-144', 'NRNX1-seg-144', 'NRNX1-seg-144', 'NRNX1-seg-228', 'NRNX1-seg-228', 'NRNX1-seg-228', 'NRNX1-seg-228', 'NRNX1-seg-449', 'NRNX1-seg-449', 'NRNX1-seg-449', 'NRNX1-seg-449', 'NRNX1-seg-491', 'NRNX1-seg-491', 'NRNX1-seg-491', 'NRNX1-seg-491', 'NRNX1-seg-561', 'NRNX1-seg-561', 'NRNX1-seg-561', 'NRNX1-seg-561', 'NRNX1-seg-601', 'NRNX1-seg-601', 'NRNX1-seg-601', 'NRNX1-seg-601', 'NRNX1-seg-662', 'NRNX1-seg-662', 'NRNX1-seg-662', 'NRNX1-seg-662'], 'NDB_832': ['NRNX1-seg-82', 'NRNX1-seg-82', 'NRNX1-seg-82', 'NRNX1-seg-82', 'NRNX1-seg-208', 'NRNX1-seg-208', 'NRNX1-seg-208', 'NRNX1-seg-208', 'NRNX1-seg-337', 'NRNX1-seg-337', 'NRNX1-seg-337', 'NRNX1-seg-337', 'NRNX1-seg-389', 'NRNX1-seg-389', 'NRNX1-seg-389', 'NRNX1-seg-389', 'NRNX1-seg-439', 'NRNX1-seg-439', 'NRNX1-seg-439', 'NRNX1-seg-439', 'NRNX1-seg-501', 'NRNX1-seg-501', 'NRNX1-seg-501', 'NRNX1-seg-501', 'NRNX1-seg-581', 'NRNX1-seg-581', 'NRNX1-seg-581', 'NRNX1-seg-581', 'NRNX1-seg-642', 'NRNX1-seg-642', 'NRNX1-seg-642', 'NRNX1-seg-642'], 'NDB_1044': ['NRNX1-seg-83', 'NRNX1-seg-83', 'NRNX1-seg-83', 'NRNX1-seg-83', 'NRNX1-seg-145', 'NRNX1-seg-145', 'NRNX1-seg-145', 'NRNX1-seg-145', 'NRNX1-seg-229', 'NRNX1-seg-229', 'NRNX1-seg-229', 'NRNX1-seg-229', 'NRNX1-seg-450', 'NRNX1-seg-450', 'NRNX1-seg-450', 'NRNX1-seg-450', 'NRNX1-seg-492', 'NRNX1-seg-492', 'NRNX1-seg-492', 'NRNX1-seg-492', 'NRNX1-seg-562', 'NRNX1-seg-562', 'NRNX1-seg-562', 'NRNX1-seg-562', 'NRNX1-seg-602', 'NRNX1-seg-602', 'NRNX1-seg-602', 'NRNX1-seg-602', 'NRNX1-seg-665', 'NRNX1-seg-665', 'NRNX1-seg-665', 'NRNX1-seg-665'], 'NDB_385': ['NRNX1-seg-83', 'NRNX1-seg-83', 'NRNX1-seg-83', 'NRNX1-seg-83', 'NRNX1-seg-209', 'NRNX1-seg-209', 'NRNX1-seg-209', 'NRNX1-seg-209', 'NRNX1-seg-338', 'NRNX1-seg-338', 'NRNX1-seg-338', 'NRNX1-seg-338', 'NRNX1-seg-390', 'NRNX1-seg-390', 'NRNX1-seg-390', 'NRNX1-seg-390', 'NRNX1-seg-440', 'NRNX1-seg-440', 'NRNX1-seg-440', 'NRNX1-seg-440', 'NRNX1-seg-502', 'NRNX1-seg-502', 'NRNX1-seg-502', 'NRNX1-seg-502', 'NRNX1-seg-582', 'NRNX1-seg-582', 'NRNX1-seg-582', 'NRNX1-seg-582', 'NRNX1-seg-643', 'NRNX1-seg-643', 'NRNX1-seg-643', 'NRNX1-seg-643'], 'NDB_711': ['NRNX1-seg-84', 'NRNX1-seg-84', 'NRNX1-seg-84', 'NRNX1-seg-84', 'NRNX1-seg-146', 'NRNX1-seg-146', 'NRNX1-seg-146', 'NRNX1-seg-146', 'NRNX1-seg-230', 'NRNX1-seg-230', 'NRNX1-seg-230', 'NRNX1-seg-230', 'NRNX1-seg-452', 'NRNX1-seg-452', 'NRNX1-seg-452', 'NRNX1-seg-452', 'NRNX1-seg-493', 'NRNX1-seg-493', 'NRNX1-seg-493', 'NRNX1-seg-493', 'NRNX1-seg-563', 'NRNX1-seg-563', 'NRNX1-seg-563', 'NRNX1-seg-563', 'NRNX1-seg-603', 'NRNX1-seg-603', 'NRNX1-seg-603', 'NRNX1-seg-603', 'NRNX1-seg-666', 'NRNX1-seg-666', 'NRNX1-seg-666', 'NRNX1-seg-666'], 'NDB_889': ['NRNX1-seg-84', 'NRNX1-seg-84', 'NRNX1-seg-84', 'NRNX1-seg-84', 'NRNX1-seg-210', 'NRNX1-seg-210', 'NRNX1-seg-210', 'NRNX1-seg-210', 'NRNX1-seg-339', 'NRNX1-seg-339', 'NRNX1-seg-339', 'NRNX1-seg-339', 'NRNX1-seg-391', 'NRNX1-seg-391', 'NRNX1-seg-391', 'NRNX1-seg-391', 'NRNX1-seg-441', 'NRNX1-seg-441', 'NRNX1-seg-441', 'NRNX1-seg-441', 'NRNX1-seg-503', 'NRNX1-seg-503', 'NRNX1-seg-503', 'NRNX1-seg-503', 'NRNX1-seg-583', 'NRNX1-seg-583', 'NRNX1-seg-583', 'NRNX1-seg-583', 'NRNX1-seg-644', 'NRNX1-seg-644', 'NRNX1-seg-644', 'NRNX1-seg-644'], 'NDB_718': ['NRNX1-seg-87', 'NRNX1-seg-87', 'NRNX1-seg-87', 'NRNX1-seg-87', 'NRNX1-seg-150', 'NRNX1-seg-150', 'NRNX1-seg-150', 'NRNX1-seg-150', 'NRNX1-seg-223', 'NRNX1-seg-223', 'NRNX1-seg-223', 'NRNX1-seg-223', 'NRNX1-seg-342', 'NRNX1-seg-342', 'NRNX1-seg-342', 'NRNX1-seg-342', 'NRNX1-seg-404', 'NRNX1-seg-404', 'NRNX1-seg-404', 'NRNX1-seg-404', 'NRNX1-seg-476', 'NRNX1-seg-476', 'NRNX1-seg-476', 'NRNX1-seg-476', 'NRNX1-seg-576', 'NRNX1-seg-576', 'NRNX1-seg-576', 'NRNX1-seg-576', 'NRNX1-seg-626', 'NRNX1-seg-626', 'NRNX1-seg-626', 'NRNX1-seg-626'], 'NDB_961': ['NRNX1-seg-88', 'NRNX1-seg-88', 'NRNX1-seg-88', 'NRNX1-seg-88', 'NRNX1-seg-151', 'NRNX1-seg-151', 'NRNX1-seg-151', 'NRNX1-seg-151', 'NRNX1-seg-224', 'NRNX1-seg-224', 'NRNX1-seg-224', 'NRNX1-seg-224', 'NRNX1-seg-343', 'NRNX1-seg-343', 'NRNX1-seg-343', 'NRNX1-seg-343', 'NRNX1-seg-405', 'NRNX1-seg-405', 'NRNX1-seg-405', 'NRNX1-seg-405', 'NRNX1-seg-477', 'NRNX1-seg-477', 'NRNX1-seg-477', 'NRNX1-seg-477', 'NRNX1-seg-577', 'NRNX1-seg-577', 'NRNX1-seg-577', 'NRNX1-seg-577', 'NRNX1-seg-627', 'NRNX1-seg-627', 'NRNX1-seg-627', 'NRNX1-seg-627'], 'NDB_571': ['NRNX1-seg-89', 'NRNX1-seg-89', 'NRNX1-seg-89', 'NRNX1-seg-89', 'NRNX1-seg-152', 'NRNX1-seg-152', 'NRNX1-seg-152', 'NRNX1-seg-152', 'NRNX1-seg-225', 'NRNX1-seg-225', 'NRNX1-seg-225', 'NRNX1-seg-225', 'NRNX1-seg-344', 'NRNX1-seg-344', 'NRNX1-seg-344', 'NRNX1-seg-344', 'NRNX1-seg-406', 'NRNX1-seg-406', 'NRNX1-seg-406', 'NRNX1-seg-406', 'NRNX1-seg-478', 'NRNX1-seg-478', 'NRNX1-seg-478', 'NRNX1-seg-478', 'NRNX1-seg-578', 'NRNX1-seg-578', 'NRNX1-seg-578', 'NRNX1-seg-578', 'NRNX1-seg-628', 'NRNX1-seg-628', 'NRNX1-seg-628', 'NRNX1-seg-628'], 'NDB_856': ['NRNX1-seg-90', 'NRNX1-seg-90', 'NRNX1-seg-90', 'NRNX1-seg-90', 'NRNX1-seg-154', 'NRNX1-seg-154', 'NRNX1-seg-154', 'NRNX1-seg-154', 'NRNX1-seg-226', 'NRNX1-seg-226', 'NRNX1-seg-226', 'NRNX1-seg-226', 'NRNX1-seg-345', 'NRNX1-seg-345', 'NRNX1-seg-345', 'NRNX1-seg-345', 'NRNX1-seg-407', 'NRNX1-seg-407', 'NRNX1-seg-407', 'NRNX1-seg-407', 'NRNX1-seg-479', 'NRNX1-seg-479', 'NRNX1-seg-479', 'NRNX1-seg-479', 'NRNX1-seg-579', 'NRNX1-seg-579', 'NRNX1-seg-579', 'NRNX1-seg-579', 'NRNX1-seg-629', 'NRNX1-seg-629', 'NRNX1-seg-629', 'NRNX1-seg-629'], 'NDB_616': ['NRNX1-seg-91', 'NRNX1-seg-91', 'NRNX1-seg-91', 'NRNX1-seg-91', 'NRNX1-seg-155', 'NRNX1-seg-155', 'NRNX1-seg-155', 'NRNX1-seg-155', 'NRNX1-seg-227', 'NRNX1-seg-227', 'NRNX1-seg-227', 'NRNX1-seg-227', 'NRNX1-seg-346', 'NRNX1-seg-346', 'NRNX1-seg-346', 'NRNX1-seg-346', 'NRNX1-seg-408', 'NRNX1-seg-408', 'NRNX1-seg-408', 'NRNX1-seg-408', 'NRNX1-seg-480', 'NRNX1-seg-480', 'NRNX1-seg-480', 'NRNX1-seg-480', 'NRNX1-seg-580', 'NRNX1-seg-580', 'NRNX1-seg-580', 'NRNX1-seg-580', 'NRNX1-seg-630', 'NRNX1-seg-630', 'NRNX1-seg-630', 'NRNX1-seg-630'], 'NDB_880': ['NRNX1-seg-92', 'NRNX1-seg-92', 'NRNX1-seg-92', 'NRNX1-seg-92', 'NRNX1-seg-156', 'NRNX1-seg-156', 'NRNX1-seg-156', 'NRNX1-seg-156', 'NRNX1-seg-228', 'NRNX1-seg-228', 'NRNX1-seg-228', 'NRNX1-seg-228', 'NRNX1-seg-347', 'NRNX1-seg-347', 'NRNX1-seg-347', 'NRNX1-seg-347', 'NRNX1-seg-409', 'NRNX1-seg-409', 'NRNX1-seg-409', 'NRNX1-seg-409', 'NRNX1-seg-481', 'NRNX1-seg-481', 'NRNX1-seg-481', 'NRNX1-seg-481', 'NRNX1-seg-581', 'NRNX1-seg-581', 'NRNX1-seg-581', 'NRNX1-seg-581', 'NRNX1-seg-631', 'NRNX1-seg-631', 'NRNX1-seg-631', 'NRNX1-seg-631'], 'NDB_505': ['NRNX1-seg-93', 'NRNX1-seg-93', 'NRNX1-seg-93', 'NRNX1-seg-93', 'NRNX1-seg-157', 'NRNX1-seg-157', 'NRNX1-seg-157', 'NRNX1-seg-157', 'NRNX1-seg-229', 'NRNX1-seg-229', 'NRNX1-seg-229', 'NRNX1-seg-229', 'NRNX1-seg-348', 'NRNX1-seg-348', 'NRNX1-seg-348', 'NRNX1-seg-348', 'NRNX1-seg-410', 'NRNX1-seg-410', 'NRNX1-seg-410', 'NRNX1-seg-410', 'NRNX1-seg-482', 'NRNX1-seg-482', 'NRNX1-seg-482', 'NRNX1-seg-482', 'NRNX1-seg-582', 'NRNX1-seg-582', 'NRNX1-seg-582', 'NRNX1-seg-582', 'NRNX1-seg-632', 'NRNX1-seg-632', 'NRNX1-seg-632', 'NRNX1-seg-632'], 'NDB_706': ['NRNX1-seg-94', 'NRNX1-seg-94', 'NRNX1-seg-94', 'NRNX1-seg-94', 'NRNX1-seg-158', 'NRNX1-seg-158', 'NRNX1-seg-158', 'NRNX1-seg-158', 'NRNX1-seg-230', 'NRNX1-seg-230', 'NRNX1-seg-230', 'NRNX1-seg-230', 'NRNX1-seg-349', 'NRNX1-seg-349', 'NRNX1-seg-349', 'NRNX1-seg-349', 'NRNX1-seg-411', 'NRNX1-seg-411', 'NRNX1-seg-411', 'NRNX1-seg-411', 'NRNX1-seg-483', 'NRNX1-seg-483', 'NRNX1-seg-483', 'NRNX1-seg-483', 'NRNX1-seg-583', 'NRNX1-seg-583', 'NRNX1-seg-583', 'NRNX1-seg-583', 'NRNX1-seg-633', 'NRNX1-seg-633', 'NRNX1-seg-633', 'NRNX1-seg-633'], 'NDB_1014': ['NRNX1-seg-95', 'NRNX1-seg-95', 'NRNX1-seg-95', 'NRNX1-seg-95', 'NRNX1-seg-201', 'NRNX1-seg-201', 'NRNX1-seg-201', 'NRNX1-seg-201', 'NRNX1-seg-298', 'NRNX1-seg-298', 'NRNX1-seg-298', 'NRNX1-seg-298', 'NRNX1-seg-382', 'NRNX1-seg-382', 'NRNX1-seg-382', 'NRNX1-seg-382', 'NRNX1-seg-453', 'NRNX1-seg-453', 'NRNX1-seg-453', 'NRNX1-seg-453', 'NRNX1-seg-514', 'NRNX1-seg-514', 'NRNX1-seg-514', 'NRNX1-seg-514', 'NRNX1-seg-564', 'NRNX1-seg-564', 'NRNX1-seg-564', 'NRNX1-seg-564', 'NRNX1-seg-667', 'NRNX1-seg-667', 'NRNX1-seg-667', 'NRNX1-seg-667'], 'NDB_421': ['NRNX1-seg-95', 'NRNX1-seg-95', 'NRNX1-seg-95', 'NRNX1-seg-95', 'NRNX1-seg-211', 'NRNX1-seg-211', 'NRNX1-seg-211', 'NRNX1-seg-211', 'NRNX1-seg-330', 'NRNX1-seg-330', 'NRNX1-seg-330', 'NRNX1-seg-330', 'NRNX1-seg-370', 'NRNX1-seg-370', 'NRNX1-seg-370', 'NRNX1-seg-370', 'NRNX1-seg-412', 'NRNX1-seg-412', 'NRNX1-seg-412', 'NRNX1-seg-412', 'NRNX1-seg-474', 'NRNX1-seg-474', 'NRNX1-seg-474', 'NRNX1-seg-474', 'NRNX1-seg-554', 'NRNX1-seg-554', 'NRNX1-seg-554', 'NRNX1-seg-554', 'NRNX1-seg-679', 'NRNX1-seg-679', 'NRNX1-seg-679', 'NRNX1-seg-679'], 'NDB_627': ['NRNX1-seg-96', 'NRNX1-seg-96', 'NRNX1-seg-96', 'NRNX1-seg-96', 'NRNX1-seg-202', 'NRNX1-seg-202', 'NRNX1-seg-202', 'NRNX1-seg-202', 'NRNX1-seg-299', 'NRNX1-seg-299', 'NRNX1-seg-299', 'NRNX1-seg-299', 'NRNX1-seg-383', 'NRNX1-seg-383', 'NRNX1-seg-383', 'NRNX1-seg-383', 'NRNX1-seg-454', 'NRNX1-seg-454', 'NRNX1-seg-454', 'NRNX1-seg-454', 'NRNX1-seg-515', 'NRNX1-seg-515', 'NRNX1-seg-515', 'NRNX1-seg-515', 'NRNX1-seg-565', 'NRNX1-seg-565', 'NRNX1-seg-565', 'NRNX1-seg-565', 'NRNX1-seg-668', 'NRNX1-seg-668', 'NRNX1-seg-668', 'NRNX1-seg-668'], 'NDB_472': ['NRNX1-seg-96', 'NRNX1-seg-96', 'NRNX1-seg-96', 'NRNX1-seg-96', 'NRNX1-seg-212', 'NRNX1-seg-212', 'NRNX1-seg-212', 'NRNX1-seg-212', 'NRNX1-seg-331', 'NRNX1-seg-331', 'NRNX1-seg-331', 'NRNX1-seg-331', 'NRNX1-seg-371', 'NRNX1-seg-371', 'NRNX1-seg-371', 'NRNX1-seg-371', 'NRNX1-seg-413', 'NRNX1-seg-413', 'NRNX1-seg-413', 'NRNX1-seg-413', 'NRNX1-seg-475', 'NRNX1-seg-475', 'NRNX1-seg-475', 'NRNX1-seg-475', 'NRNX1-seg-555', 'NRNX1-seg-555', 'NRNX1-seg-555', 'NRNX1-seg-555', 'NRNX1-seg-681', 'NRNX1-seg-681', 'NRNX1-seg-681', 'NRNX1-seg-681'], 'NDB_557': ['NRNX1-seg-105', 'NRNX1-seg-105', 'NRNX1-seg-105', 'NRNX1-seg-105', 'NRNX1-seg-191', 'NRNX1-seg-191', 'NRNX1-seg-191', 'NRNX1-seg-191', 'NRNX1-seg-242', 'NRNX1-seg-242', 'NRNX1-seg-242', 'NRNX1-seg-242', 'NRNX1-seg-330', 'NRNX1-seg-330', 'NRNX1-seg-330', 'NRNX1-seg-330', 'NRNX1-seg-382', 'NRNX1-seg-382', 'NRNX1-seg-382', 'NRNX1-seg-382', 'NRNX1-seg-524', 'NRNX1-seg-524', 'NRNX1-seg-524', 'NRNX1-seg-524', 'NRNX1-seg-574', 'NRNX1-seg-574', 'NRNX1-seg-574', 'NRNX1-seg-574', 'NRNX1-seg-634', 'NRNX1-seg-634', 'NRNX1-seg-634', 'NRNX1-seg-634'], 'NDB_929': ['NRNX1-seg-106', 'NRNX1-seg-106', 'NRNX1-seg-106', 'NRNX1-seg-106', 'NRNX1-seg-192', 'NRNX1-seg-192', 'NRNX1-seg-192', 'NRNX1-seg-192', 'NRNX1-seg-244', 'NRNX1-seg-244', 'NRNX1-seg-244', 'NRNX1-seg-244', 'NRNX1-seg-331', 'NRNX1-seg-331', 'NRNX1-seg-331', 'NRNX1-seg-331', 'NRNX1-seg-383', 'NRNX1-seg-383', 'NRNX1-seg-383', 'NRNX1-seg-383', 'NRNX1-seg-525', 'NRNX1-seg-525', 'NRNX1-seg-525', 'NRNX1-seg-525', 'NRNX1-seg-575', 'NRNX1-seg-575', 'NRNX1-seg-575', 'NRNX1-seg-575', 'NRNX1-seg-635', 'NRNX1-seg-635', 'NRNX1-seg-635', 'NRNX1-seg-635'], 'NDB_742': ['NRNX1-seg-171', 'NRNX1-seg-171', 'NRNX1-seg-171', 'NRNX1-seg-171', 'NRNX1-seg-221', 'NRNX1-seg-221', 'NRNX1-seg-221', 'NRNX1-seg-221', 'NRNX1-seg-308', 'NRNX1-seg-308', 'NRNX1-seg-308', 'NRNX1-seg-308', 'NRNX1-seg-350', 'NRNX1-seg-350', 'NRNX1-seg-350', 'NRNX1-seg-350', 'NRNX1-seg-432', 'NRNX1-seg-432', 'NRNX1-seg-432', 'NRNX1-seg-432', 'NRNX1-seg-494', 'NRNX1-seg-494', 'NRNX1-seg-494', 'NRNX1-seg-494', 'NRNX1-seg-554', 'NRNX1-seg-554', 'NRNX1-seg-554', 'NRNX1-seg-554', 'NRNX1-seg-634', 'NRNX1-seg-634', 'NRNX1-seg-634', 'NRNX1-seg-634'], 'NDB_925': ['NRNX1-seg-172', 'NRNX1-seg-172', 'NRNX1-seg-172', 'NRNX1-seg-172', 'NRNX1-seg-222', 'NRNX1-seg-222', 'NRNX1-seg-222', 'NRNX1-seg-222', 'NRNX1-seg-311', 'NRNX1-seg-311', 'NRNX1-seg-311', 'NRNX1-seg-311', 'NRNX1-seg-351', 'NRNX1-seg-351', 'NRNX1-seg-351', 'NRNX1-seg-351', 'NRNX1-seg-433', 'NRNX1-seg-433', 'NRNX1-seg-433', 'NRNX1-seg-433', 'NRNX1-seg-495', 'NRNX1-seg-495', 'NRNX1-seg-495', 'NRNX1-seg-495', 'NRNX1-seg-555', 'NRNX1-seg-555', 'NRNX1-seg-555', 'NRNX1-seg-555', 'NRNX1-seg-635', 'NRNX1-seg-635', 'NRNX1-seg-635', 'NRNX1-seg-635'], 'NDB_777': ['NRNX1-seg-173', 'NRNX1-seg-173', 'NRNX1-seg-173', 'NRNX1-seg-173', 'NRNX1-seg-213', 'NRNX1-seg-213', 'NRNX1-seg-213', 'NRNX1-seg-213', 'NRNX1-seg-268', 'NRNX1-seg-268', 'NRNX1-seg-268', 'NRNX1-seg-268', 'NRNX1-seg-384', 'NRNX1-seg-384', 'NRNX1-seg-384', 'NRNX1-seg-384', 'NRNX1-seg-486', 'NRNX1-seg-486', 'NRNX1-seg-486', 'NRNX1-seg-486', 'NRNX1-seg-566', 'NRNX1-seg-566', 'NRNX1-seg-566', 'NRNX1-seg-566', 'NRNX1-seg-626', 'NRNX1-seg-626', 'NRNX1-seg-626', 'NRNX1-seg-626', 'NRNX1-seg-669', 'NRNX1-seg-669', 'NRNX1-seg-669', 'NRNX1-seg-669'], 'NDB_750': ['NRNX1-seg-174', 'NRNX1-seg-174', 'NRNX1-seg-174', 'NRNX1-seg-174', 'NRNX1-seg-214', 'NRNX1-seg-214', 'NRNX1-seg-214', 'NRNX1-seg-214', 'NRNX1-seg-269', 'NRNX1-seg-269', 'NRNX1-seg-269', 'NRNX1-seg-269', 'NRNX1-seg-385', 'NRNX1-seg-385', 'NRNX1-seg-385', 'NRNX1-seg-385', 'NRNX1-seg-487', 'NRNX1-seg-487', 'NRNX1-seg-487', 'NRNX1-seg-487', 'NRNX1-seg-567', 'NRNX1-seg-567', 'NRNX1-seg-567', 'NRNX1-seg-567', 'NRNX1-seg-627', 'NRNX1-seg-627', 'NRNX1-seg-627', 'NRNX1-seg-627', 'NRNX1-seg-670', 'NRNX1-seg-670', 'NRNX1-seg-670', 'NRNX1-seg-670'], 'NDB_621': ['NRNX1-seg-175', 'NRNX1-seg-175', 'NRNX1-seg-175', 'NRNX1-seg-175', 'NRNX1-seg-215', 'NRNX1-seg-215', 'NRNX1-seg-215', 'NRNX1-seg-215', 'NRNX1-seg-270', 'NRNX1-seg-270', 'NRNX1-seg-270', 'NRNX1-seg-270', 'NRNX1-seg-386', 'NRNX1-seg-386', 'NRNX1-seg-386', 'NRNX1-seg-386', 'NRNX1-seg-488', 'NRNX1-seg-488', 'NRNX1-seg-488', 'NRNX1-seg-488', 'NRNX1-seg-568', 'NRNX1-seg-568', 'NRNX1-seg-568', 'NRNX1-seg-568', 'NRNX1-seg-628', 'NRNX1-seg-628', 'NRNX1-seg-628', 'NRNX1-seg-628', 'NRNX1-seg-671', 'NRNX1-seg-671', 'NRNX1-seg-671', 'NRNX1-seg-671'], 'NDB_489': ['NRNX1-seg-176', 'NRNX1-seg-176', 'NRNX1-seg-176', 'NRNX1-seg-176', 'NRNX1-seg-216', 'NRNX1-seg-216', 'NRNX1-seg-216', 'NRNX1-seg-216', 'NRNX1-seg-271', 'NRNX1-seg-271', 'NRNX1-seg-271', 'NRNX1-seg-271', 'NRNX1-seg-387', 'NRNX1-seg-387', 'NRNX1-seg-387', 'NRNX1-seg-387', 'NRNX1-seg-489', 'NRNX1-seg-489', 'NRNX1-seg-489', 'NRNX1-seg-489', 'NRNX1-seg-569', 'NRNX1-seg-569', 'NRNX1-seg-569', 'NRNX1-seg-569', 'NRNX1-seg-629', 'NRNX1-seg-629', 'NRNX1-seg-629', 'NRNX1-seg-629', 'NRNX1-seg-672', 'NRNX1-seg-672', 'NRNX1-seg-672', 'NRNX1-seg-672'], 'NDB_663': ['NRNX1-seg-177', 'NRNX1-seg-177', 'NRNX1-seg-177', 'NRNX1-seg-177', 'NRNX1-seg-217', 'NRNX1-seg-217', 'NRNX1-seg-217', 'NRNX1-seg-217', 'NRNX1-seg-272', 'NRNX1-seg-272', 'NRNX1-seg-272', 'NRNX1-seg-272', 'NRNX1-seg-388', 'NRNX1-seg-388', 'NRNX1-seg-388', 'NRNX1-seg-388', 'NRNX1-seg-490', 'NRNX1-seg-490', 'NRNX1-seg-490', 'NRNX1-seg-490', 'NRNX1-seg-570', 'NRNX1-seg-570', 'NRNX1-seg-570', 'NRNX1-seg-570', 'NRNX1-seg-630', 'NRNX1-seg-630', 'NRNX1-seg-630', 'NRNX1-seg-630', 'NRNX1-seg-674', 'NRNX1-seg-674', 'NRNX1-seg-674', 'NRNX1-seg-674'], 'NDB_501': ['NRNX1-seg-178', 'NRNX1-seg-178', 'NRNX1-seg-178', 'NRNX1-seg-178', 'NRNX1-seg-218', 'NRNX1-seg-218', 'NRNX1-seg-218', 'NRNX1-seg-218', 'NRNX1-seg-273', 'NRNX1-seg-273', 'NRNX1-seg-273', 'NRNX1-seg-273', 'NRNX1-seg-389', 'NRNX1-seg-389', 'NRNX1-seg-389', 'NRNX1-seg-389', 'NRNX1-seg-491', 'NRNX1-seg-491', 'NRNX1-seg-491', 'NRNX1-seg-491', 'NRNX1-seg-571', 'NRNX1-seg-571', 'NRNX1-seg-571', 'NRNX1-seg-571', 'NRNX1-seg-631', 'NRNX1-seg-631', 'NRNX1-seg-631', 'NRNX1-seg-631', 'NRNX1-seg-675', 'NRNX1-seg-675', 'NRNX1-seg-675', 'NRNX1-seg-675'], 'NDB_813': ['NRNX1-seg-179', 'NRNX1-seg-179', 'NRNX1-seg-179', 'NRNX1-seg-179', 'NRNX1-seg-219', 'NRNX1-seg-219', 'NRNX1-seg-219', 'NRNX1-seg-219', 'NRNX1-seg-274', 'NRNX1-seg-274', 'NRNX1-seg-274', 'NRNX1-seg-274', 'NRNX1-seg-390', 'NRNX1-seg-390', 'NRNX1-seg-390', 'NRNX1-seg-390', 'NRNX1-seg-492', 'NRNX1-seg-492', 'NRNX1-seg-492', 'NRNX1-seg-492', 'NRNX1-seg-572', 'NRNX1-seg-572', 'NRNX1-seg-572', 'NRNX1-seg-572', 'NRNX1-seg-632', 'NRNX1-seg-632', 'NRNX1-seg-632', 'NRNX1-seg-632', 'NRNX1-seg-676', 'NRNX1-seg-676', 'NRNX1-seg-676', 'NRNX1-seg-676'], 'NDB_1023': ['NRNX1-seg-180', 'NRNX1-seg-180', 'NRNX1-seg-180', 'NRNX1-seg-180', 'NRNX1-seg-220', 'NRNX1-seg-220', 'NRNX1-seg-220', 'NRNX1-seg-220', 'NRNX1-seg-275', 'NRNX1-seg-275', 'NRNX1-seg-275', 'NRNX1-seg-275', 'NRNX1-seg-391', 'NRNX1-seg-391', 'NRNX1-seg-391', 'NRNX1-seg-391', 'NRNX1-seg-493', 'NRNX1-seg-493', 'NRNX1-seg-493', 'NRNX1-seg-493', 'NRNX1-seg-573', 'NRNX1-seg-573', 'NRNX1-seg-573', 'NRNX1-seg-573', 'NRNX1-seg-633', 'NRNX1-seg-633', 'NRNX1-seg-633', 'NRNX1-seg-633', 'NRNX1-seg-678', 'NRNX1-seg-678', 'NRNX1-seg-678', 'NRNX1-seg-678']}\n",
      " > Done.\n",
      "-- inverting region_to_readout dict\n",
      "NDB_1014 -> ['NRNX1-seg-201', 'NRNX1-seg-298', 'NRNX1-seg-382', 'NRNX1-seg-453', 'NRNX1-seg-514', 'NRNX1-seg-564', 'NRNX1-seg-667', 'NRNX1-seg-95']\n",
      "NDB_1020 -> ['NRNX1-seg-110', 'NRNX1-seg-206', 'NRNX1-seg-303', 'NRNX1-seg-355', 'NRNX1-seg-458', 'NRNX1-seg-519', 'NRNX1-seg-660', 'NRNX1-seg-69']\n",
      "NDB_1023 -> ['NRNX1-seg-180', 'NRNX1-seg-220', 'NRNX1-seg-275', 'NRNX1-seg-391', 'NRNX1-seg-493', 'NRNX1-seg-573', 'NRNX1-seg-633', 'NRNX1-seg-678']\n",
      "NDB_1027 -> ['NRNX1-seg-100', 'NRNX1-seg-196', 'NRNX1-seg-291', 'NRNX1-seg-417', 'NRNX1-seg-499', 'NRNX1-seg-58', 'NRNX1-seg-599', 'NRNX1-seg-672']\n",
      "NDB_1028 -> ['NRNX1-seg-182', 'NRNX1-seg-232', 'NRNX1-seg-321', 'NRNX1-seg-34', 'NRNX1-seg-495', 'NRNX1-seg-545', 'NRNX1-seg-585', 'NRNX1-seg-668']\n",
      "NDB_1033 -> ['NRNX1-seg-122', 'NRNX1-seg-197', 'NRNX1-seg-272', 'NRNX1-seg-326', 'NRNX1-seg-366', 'NRNX1-seg-480', 'NRNX1-seg-641', 'NRNX1-seg-70']\n",
      "NDB_1035 -> ['NRNX1-seg-165', 'NRNX1-seg-238', 'NRNX1-seg-292', 'NRNX1-seg-376', 'NRNX1-seg-469', 'NRNX1-seg-510', 'NRNX1-seg-580', 'NRNX1-seg-651', 'NRNX1-seg-70']\n",
      "NDB_1044 -> ['NRNX1-seg-145', 'NRNX1-seg-229', 'NRNX1-seg-450', 'NRNX1-seg-492', 'NRNX1-seg-562', 'NRNX1-seg-602', 'NRNX1-seg-665', 'NRNX1-seg-83']\n",
      "NDB_1047 -> ['NRNX1-seg-134', 'NRNX1-seg-293', 'NRNX1-seg-357', 'NRNX1-seg-409', 'NRNX1-seg-491', 'NRNX1-seg-551', 'NRNX1-seg-6', 'NRNX1-seg-697']\n",
      "NDB_1053 -> ['NRNX1-seg-126', 'NRNX1-seg-201', 'NRNX1-seg-266', 'NRNX1-seg-402', 'NRNX1-seg-43', 'NRNX1-seg-494', 'NRNX1-seg-574', 'NRNX1-seg-655']\n",
      "NDB_1060 -> ['NRNX1-seg-144', 'NRNX1-seg-208', 'NRNX1-seg-317', 'NRNX1-seg-40', 'NRNX1-seg-419', 'NRNX1-seg-481', 'NRNX1-seg-541', 'NRNX1-seg-591']\n",
      "NDB_1066 -> ['NRNX1-seg-125', 'NRNX1-seg-200', 'NRNX1-seg-275', 'NRNX1-seg-329', 'NRNX1-seg-369', 'NRNX1-seg-483', 'NRNX1-seg-644', 'NRNX1-seg-73']\n",
      "NDB_1075 -> ['NRNX1-seg-207', 'NRNX1-seg-336', 'NRNX1-seg-388', 'NRNX1-seg-438', 'NRNX1-seg-500', 'NRNX1-seg-580', 'NRNX1-seg-641', 'NRNX1-seg-81']\n",
      "NDB_332 -> ['NRNX1-seg-177', 'NRNX1-seg-251', 'NRNX1-seg-356', 'NRNX1-seg-448', 'NRNX1-seg-49', 'NRNX1-seg-590', 'NRNX1-seg-651', 'NRNX1-seg-91']\n",
      "NDB_335 -> ['NRNX1-seg-178', 'NRNX1-seg-252', 'NRNX1-seg-357', 'NRNX1-seg-449', 'NRNX1-seg-50', 'NRNX1-seg-591', 'NRNX1-seg-652', 'NRNX1-seg-92']\n",
      "NDB_341 -> ['NRNX1-seg-147', 'NRNX1-seg-221', 'NRNX1-seg-320', 'NRNX1-seg-422', 'NRNX1-seg-504', 'NRNX1-seg-564', 'NRNX1-seg-624', 'NRNX1-seg-74']\n",
      "NDB_347 -> ['NRNX1-seg-134', 'NRNX1-seg-198', 'NRNX1-seg-239', 'NRNX1-seg-347', 'NRNX1-seg-439', 'NRNX1-seg-50', 'NRNX1-seg-561', 'NRNX1-seg-611', 'NRNX1-seg-687']\n",
      "NDB_356 -> ['NRNX1-seg-136', 'NRNX1-seg-230', 'NRNX1-seg-307', 'NRNX1-seg-32', 'NRNX1-seg-369', 'NRNX1-seg-421', 'NRNX1-seg-573', 'NRNX1-seg-654']\n",
      "NDB_372 -> ['NRNX1-seg-126', 'NRNX1-seg-191', 'NRNX1-seg-22', 'NRNX1-seg-286', 'NRNX1-seg-350', 'NRNX1-seg-442', 'NRNX1-seg-624', 'NRNX1-seg-679', 'NRNX1-seg-85']\n",
      "NDB_375 -> ['NRNX1-seg-16', 'NRNX1-seg-185', 'NRNX1-seg-302', 'NRNX1-seg-344', 'NRNX1-seg-467', 'NRNX1-seg-538', 'NRNX1-seg-618', 'NRNX1-seg-99']\n",
      "NDB_376 -> ['NRNX1-seg-1', 'NRNX1-seg-161', 'NRNX1-seg-268', 'NRNX1-seg-332', 'NRNX1-seg-424', 'NRNX1-seg-526', 'NRNX1-seg-606', 'NRNX1-seg-657', 'NRNX1-seg-97']\n",
      "NDB_379 -> ['NRNX1-seg-159', 'NRNX1-seg-22', 'NRNX1-seg-255', 'NRNX1-seg-308', 'NRNX1-seg-402', 'NRNX1-seg-453', 'NRNX1-seg-524', 'NRNX1-seg-645']\n",
      "NDB_385 -> ['NRNX1-seg-209', 'NRNX1-seg-338', 'NRNX1-seg-390', 'NRNX1-seg-440', 'NRNX1-seg-502', 'NRNX1-seg-582', 'NRNX1-seg-643', 'NRNX1-seg-83']\n",
      "NDB_386 -> ['NRNX1-seg-160', 'NRNX1-seg-277', 'NRNX1-seg-341', 'NRNX1-seg-485', 'NRNX1-seg-565', 'NRNX1-seg-635', 'NRNX1-seg-65', 'NRNX1-seg-681']\n",
      "NDB_387 -> ['NRNX1-seg-163', 'NRNX1-seg-235', 'NRNX1-seg-290', 'NRNX1-seg-374', 'NRNX1-seg-467', 'NRNX1-seg-508', 'NRNX1-seg-578', 'NRNX1-seg-649', 'NRNX1-seg-68']\n",
      "NDB_388 -> ['NRNX1-seg-112', 'NRNX1-seg-19', 'NRNX1-seg-263', 'NRNX1-seg-367', 'NRNX1-seg-429', 'NRNX1-seg-501', 'NRNX1-seg-551', 'NRNX1-seg-591', 'NRNX1-seg-631']\n",
      "NDB_390 -> ['NRNX1-seg-188', 'NRNX1-seg-252', 'NRNX1-seg-30', 'NRNX1-seg-317', 'NRNX1-seg-399', 'NRNX1-seg-551', 'NRNX1-seg-611', 'NRNX1-seg-675', 'NRNX1-seg-82']\n",
      "NDB_391 -> ['NRNX1-seg-206', 'NRNX1-seg-335', 'NRNX1-seg-387', 'NRNX1-seg-437', 'NRNX1-seg-499', 'NRNX1-seg-579', 'NRNX1-seg-640', 'NRNX1-seg-80']\n",
      "NDB_392 -> ['NRNX1-seg-106', 'NRNX1-seg-23', 'NRNX1-seg-267', 'NRNX1-seg-361', 'NRNX1-seg-423', 'NRNX1-seg-485', 'NRNX1-seg-555', 'NRNX1-seg-605']\n",
      "NDB_395 -> ['NRNX1-seg-148', 'NRNX1-seg-222', 'NRNX1-seg-321', 'NRNX1-seg-423', 'NRNX1-seg-505', 'NRNX1-seg-565', 'NRNX1-seg-625', 'NRNX1-seg-75']\n",
      "NDB_396 -> ['NRNX1-seg-104', 'NRNX1-seg-190', 'NRNX1-seg-21', 'NRNX1-seg-307', 'NRNX1-seg-349', 'NRNX1-seg-472', 'NRNX1-seg-543', 'NRNX1-seg-623']\n",
      "NDB_397 -> ['NRNX1-seg-103', 'NRNX1-seg-169', 'NRNX1-seg-274', 'NRNX1-seg-338', 'NRNX1-seg-430', 'NRNX1-seg-532', 'NRNX1-seg-612', 'NRNX1-seg-665', 'NRNX1-seg-7']\n",
      "NDB_398 -> ['NRNX1-seg-122', 'NRNX1-seg-187', 'NRNX1-seg-262', 'NRNX1-seg-388', 'NRNX1-seg-39', 'NRNX1-seg-459', 'NRNX1-seg-510', 'NRNX1-seg-560', 'NRNX1-seg-696']\n",
      "NDB_400 -> ['NRNX1-seg-162', 'NRNX1-seg-2', 'NRNX1-seg-269', 'NRNX1-seg-333', 'NRNX1-seg-425', 'NRNX1-seg-527', 'NRNX1-seg-607', 'NRNX1-seg-658', 'NRNX1-seg-98']\n",
      "NDB_401 -> ['NRNX1-seg-10', 'NRNX1-seg-137', 'NRNX1-seg-266', 'NRNX1-seg-340', 'NRNX1-seg-432', 'NRNX1-seg-504', 'NRNX1-seg-614', 'NRNX1-seg-95']\n",
      "NDB_403 -> ['NRNX1-seg-121', 'NRNX1-seg-196', 'NRNX1-seg-271', 'NRNX1-seg-325', 'NRNX1-seg-365', 'NRNX1-seg-479', 'NRNX1-seg-640', 'NRNX1-seg-69']\n",
      "NDB_405 -> ['NRNX1-seg-187', 'NRNX1-seg-251', 'NRNX1-seg-28', 'NRNX1-seg-316', 'NRNX1-seg-398', 'NRNX1-seg-550', 'NRNX1-seg-610', 'NRNX1-seg-674', 'NRNX1-seg-81']\n",
      "NDB_409 -> ['NRNX1-seg-141', 'NRNX1-seg-16', 'NRNX1-seg-215', 'NRNX1-seg-280', 'NRNX1-seg-374', 'NRNX1-seg-436', 'NRNX1-seg-528', 'NRNX1-seg-694', 'NRNX1-seg-89']\n",
      "NDB_417 -> ['NRNX1-seg-100', 'NRNX1-seg-17', 'NRNX1-seg-186', 'NRNX1-seg-303', 'NRNX1-seg-345', 'NRNX1-seg-468', 'NRNX1-seg-539', 'NRNX1-seg-619']\n",
      "NDB_418 -> ['NRNX1-seg-138', 'NRNX1-seg-256', 'NRNX1-seg-331', 'NRNX1-seg-34', 'NRNX1-seg-423', 'NRNX1-seg-535', 'NRNX1-seg-595', 'NRNX1-seg-691']\n",
      "NDB_421 -> ['NRNX1-seg-211', 'NRNX1-seg-330', 'NRNX1-seg-370', 'NRNX1-seg-412', 'NRNX1-seg-474', 'NRNX1-seg-554', 'NRNX1-seg-679', 'NRNX1-seg-95']\n",
      "NDB_427 -> ['NRNX1-seg-102', 'NRNX1-seg-166', 'NRNX1-seg-273', 'NRNX1-seg-337', 'NRNX1-seg-429', 'NRNX1-seg-531', 'NRNX1-seg-6', 'NRNX1-seg-611', 'NRNX1-seg-662']\n",
      "NDB_431 -> ['NRNX1-seg-129', 'NRNX1-seg-194', 'NRNX1-seg-234', 'NRNX1-seg-343', 'NRNX1-seg-435', 'NRNX1-seg-46', 'NRNX1-seg-557', 'NRNX1-seg-607', 'NRNX1-seg-683']\n",
      "NDB_432 -> ['NRNX1-seg-109', 'NRNX1-seg-205', 'NRNX1-seg-302', 'NRNX1-seg-354', 'NRNX1-seg-457', 'NRNX1-seg-518', 'NRNX1-seg-659', 'NRNX1-seg-68']\n",
      "NDB_433 -> ['NRNX1-seg-145', 'NRNX1-seg-209', 'NRNX1-seg-318', 'NRNX1-seg-41', 'NRNX1-seg-420', 'NRNX1-seg-482', 'NRNX1-seg-542', 'NRNX1-seg-592']\n",
      "NDB_438 -> ['NRNX1-seg-131', 'NRNX1-seg-291', 'NRNX1-seg-355', 'NRNX1-seg-4', 'NRNX1-seg-407', 'NRNX1-seg-489', 'NRNX1-seg-549', 'NRNX1-seg-695']\n",
      "NDB_443 -> ['NRNX1-seg-249', 'NRNX1-seg-325', 'NRNX1-seg-375', 'NRNX1-seg-38', 'NRNX1-seg-427', 'NRNX1-seg-519', 'NRNX1-seg-599', 'NRNX1-seg-685']\n",
      "NDB_444 -> ['NRNX1-seg-143', 'NRNX1-seg-227', 'NRNX1-seg-448', 'NRNX1-seg-490', 'NRNX1-seg-560', 'NRNX1-seg-600', 'NRNX1-seg-661', 'NRNX1-seg-81']\n",
      "NDB_445 -> ['NRNX1-seg-146', 'NRNX1-seg-21', 'NRNX1-seg-220', 'NRNX1-seg-285', 'NRNX1-seg-381', 'NRNX1-seg-441', 'NRNX1-seg-533', 'NRNX1-seg-699', 'NRNX1-seg-94']\n",
      "NDB_446 -> ['NRNX1-seg-156', 'NRNX1-seg-263', 'NRNX1-seg-327', 'NRNX1-seg-399', 'NRNX1-seg-470', 'NRNX1-seg-50', 'NRNX1-seg-531', 'NRNX1-seg-571']\n",
      "NDB_448 -> ['NRNX1-seg-205', 'NRNX1-seg-334', 'NRNX1-seg-386', 'NRNX1-seg-436', 'NRNX1-seg-498', 'NRNX1-seg-578', 'NRNX1-seg-639', 'NRNX1-seg-79']\n",
      "NDB_450 -> ['NRNX1-seg-127', 'NRNX1-seg-202', 'NRNX1-seg-267', 'NRNX1-seg-403', 'NRNX1-seg-44', 'NRNX1-seg-495', 'NRNX1-seg-575', 'NRNX1-seg-656']\n",
      "NDB_452 -> ['NRNX1-seg-157', 'NRNX1-seg-264', 'NRNX1-seg-328', 'NRNX1-seg-400', 'NRNX1-seg-471', 'NRNX1-seg-51', 'NRNX1-seg-532', 'NRNX1-seg-572']\n",
      "NDB_453 -> ['NRNX1-seg-113', 'NRNX1-seg-157', 'NRNX1-seg-284', 'NRNX1-seg-31', 'NRNX1-seg-338', 'NRNX1-seg-450', 'NRNX1-seg-512', 'NRNX1-seg-622', 'NRNX1-seg-688']\n",
      "NDB_455 -> ['NRNX1-seg-131', 'NRNX1-seg-226', 'NRNX1-seg-27', 'NRNX1-seg-303', 'NRNX1-seg-365', 'NRNX1-seg-417', 'NRNX1-seg-569', 'NRNX1-seg-650']\n",
      "NDB_456 -> ['NRNX1-seg-184', 'NRNX1-seg-246', 'NRNX1-seg-25', 'NRNX1-seg-313', 'NRNX1-seg-395', 'NRNX1-seg-547', 'NRNX1-seg-607', 'NRNX1-seg-670', 'NRNX1-seg-77']\n",
      "NDB_458 -> ['NRNX1-seg-175', 'NRNX1-seg-248', 'NRNX1-seg-354', 'NRNX1-seg-446', 'NRNX1-seg-47', 'NRNX1-seg-588', 'NRNX1-seg-649', 'NRNX1-seg-89']\n",
      "NDB_459 -> ['NRNX1-seg-181', 'NRNX1-seg-286', 'NRNX1-seg-340', 'NRNX1-seg-392', 'NRNX1-seg-474', 'NRNX1-seg-524', 'NRNX1-seg-604', 'NRNX1-seg-74']\n",
      "NDB_461 -> ['NRNX1-seg-177', 'NRNX1-seg-238', 'NRNX1-seg-282', 'NRNX1-seg-398', 'NRNX1-seg-459', 'NRNX1-seg-5', 'NRNX1-seg-540', 'NRNX1-seg-59', 'NRNX1-seg-641']\n",
      "NDB_464 -> ['NRNX1-seg-174', 'NRNX1-seg-2', 'NRNX1-seg-234', 'NRNX1-seg-279', 'NRNX1-seg-395', 'NRNX1-seg-456', 'NRNX1-seg-537', 'NRNX1-seg-56', 'NRNX1-seg-637']\n",
      "NDB_465 -> ['NRNX1-seg-164', 'NRNX1-seg-236', 'NRNX1-seg-291', 'NRNX1-seg-375', 'NRNX1-seg-468', 'NRNX1-seg-509', 'NRNX1-seg-579', 'NRNX1-seg-650', 'NRNX1-seg-69']\n",
      "NDB_466 -> ['NRNX1-seg-141', 'NRNX1-seg-205', 'NRNX1-seg-314', 'NRNX1-seg-37', 'NRNX1-seg-416', 'NRNX1-seg-478', 'NRNX1-seg-538', 'NRNX1-seg-588']\n",
      "NDB_467 -> ['NRNX1-seg-155', 'NRNX1-seg-262', 'NRNX1-seg-326', 'NRNX1-seg-398', 'NRNX1-seg-469', 'NRNX1-seg-49', 'NRNX1-seg-530', 'NRNX1-seg-570']\n",
      "NDB_470 -> ['NRNX1-seg-176', 'NRNX1-seg-236', 'NRNX1-seg-281', 'NRNX1-seg-397', 'NRNX1-seg-4', 'NRNX1-seg-458', 'NRNX1-seg-539', 'NRNX1-seg-58', 'NRNX1-seg-640']\n",
      "NDB_472 -> ['NRNX1-seg-212', 'NRNX1-seg-331', 'NRNX1-seg-371', 'NRNX1-seg-413', 'NRNX1-seg-475', 'NRNX1-seg-555', 'NRNX1-seg-681', 'NRNX1-seg-96']\n",
      "NDB_473 -> ['NRNX1-seg-179', 'NRNX1-seg-253', 'NRNX1-seg-358', 'NRNX1-seg-450', 'NRNX1-seg-51', 'NRNX1-seg-592', 'NRNX1-seg-653', 'NRNX1-seg-93']\n",
      "NDB_482 -> ['NRNX1-seg-123', 'NRNX1-seg-166', 'NRNX1-seg-218', 'NRNX1-seg-317', 'NRNX1-seg-409', 'NRNX1-seg-521', 'NRNX1-seg-60', 'NRNX1-seg-621']\n",
      "NDB_487 -> ['NRNX1-seg-195', 'NRNX1-seg-290', 'NRNX1-seg-416', 'NRNX1-seg-498', 'NRNX1-seg-57', 'NRNX1-seg-598', 'NRNX1-seg-671', 'NRNX1-seg-99']\n",
      "NDB_489 -> ['NRNX1-seg-176', 'NRNX1-seg-216', 'NRNX1-seg-271', 'NRNX1-seg-387', 'NRNX1-seg-489', 'NRNX1-seg-569', 'NRNX1-seg-629', 'NRNX1-seg-672']\n",
      "NDB_491 -> ['NRNX1-seg-181', 'NRNX1-seg-231', 'NRNX1-seg-320', 'NRNX1-seg-33', 'NRNX1-seg-494', 'NRNX1-seg-544', 'NRNX1-seg-584', 'NRNX1-seg-667']\n",
      "NDB_498 -> ['NRNX1-seg-170', 'NRNX1-seg-241', 'NRNX1-seg-295', 'NRNX1-seg-381', 'NRNX1-seg-472', 'NRNX1-seg-513', 'NRNX1-seg-583', 'NRNX1-seg-654', 'NRNX1-seg-73']\n",
      "NDB_501 -> ['NRNX1-seg-178', 'NRNX1-seg-218', 'NRNX1-seg-273', 'NRNX1-seg-389', 'NRNX1-seg-491', 'NRNX1-seg-571', 'NRNX1-seg-631', 'NRNX1-seg-675']\n",
      "NDB_505 -> ['NRNX1-seg-157', 'NRNX1-seg-229', 'NRNX1-seg-348', 'NRNX1-seg-410', 'NRNX1-seg-482', 'NRNX1-seg-582', 'NRNX1-seg-632', 'NRNX1-seg-93']\n",
      "NDB_506 -> ['NRNX1-seg-178', 'NRNX1-seg-239', 'NRNX1-seg-283', 'NRNX1-seg-399', 'NRNX1-seg-460', 'NRNX1-seg-541', 'NRNX1-seg-6', 'NRNX1-seg-60', 'NRNX1-seg-642']\n",
      "NDB_507 -> ['NRNX1-seg-112', 'NRNX1-seg-208', 'NRNX1-seg-305', 'NRNX1-seg-357', 'NRNX1-seg-460', 'NRNX1-seg-521', 'NRNX1-seg-662', 'NRNX1-seg-71']\n",
      "NDB_508 -> ['NRNX1-seg-100', 'NRNX1-seg-164', 'NRNX1-seg-271', 'NRNX1-seg-335', 'NRNX1-seg-4', 'NRNX1-seg-427', 'NRNX1-seg-529', 'NRNX1-seg-609', 'NRNX1-seg-660']\n",
      "NDB_511 -> ['NRNX1-seg-163', 'NRNX1-seg-270', 'NRNX1-seg-3', 'NRNX1-seg-334', 'NRNX1-seg-426', 'NRNX1-seg-528', 'NRNX1-seg-608', 'NRNX1-seg-659', 'NRNX1-seg-99']\n",
      "NDB_513 -> ['NRNX1-seg-127', 'NRNX1-seg-192', 'NRNX1-seg-23', 'NRNX1-seg-287', 'NRNX1-seg-351', 'NRNX1-seg-443', 'NRNX1-seg-625', 'NRNX1-seg-681', 'NRNX1-seg-86']\n",
      "NDB_514 -> ['NRNX1-seg-104', 'NRNX1-seg-170', 'NRNX1-seg-275', 'NRNX1-seg-339', 'NRNX1-seg-431', 'NRNX1-seg-533', 'NRNX1-seg-613', 'NRNX1-seg-666', 'NRNX1-seg-9']\n",
      "NDB_516 -> ['NRNX1-seg-109', 'NRNX1-seg-152', 'NRNX1-seg-26', 'NRNX1-seg-280', 'NRNX1-seg-334', 'NRNX1-seg-446', 'NRNX1-seg-508', 'NRNX1-seg-618', 'NRNX1-seg-684']\n",
      "NDB_517 -> ['NRNX1-seg-148', 'NRNX1-seg-232', 'NRNX1-seg-287', 'NRNX1-seg-371', 'NRNX1-seg-515', 'NRNX1-seg-575', 'NRNX1-seg-615', 'NRNX1-seg-65', 'NRNX1-seg-691']\n",
      "NDB_519 -> ['NRNX1-seg-160', 'NRNX1-seg-222', 'NRNX1-seg-413', 'NRNX1-seg-44', 'NRNX1-seg-464', 'NRNX1-seg-535', 'NRNX1-seg-605', 'NRNX1-seg-668', 'NRNX1-seg-86']\n",
      "NDB_526 -> ['NRNX1-seg-106', 'NRNX1-seg-172', 'NRNX1-seg-232', 'NRNX1-seg-299', 'NRNX1-seg-403', 'NRNX1-seg-464', 'NRNX1-seg-54', 'NRNX1-seg-595']\n",
      "NDB_529 -> ['NRNX1-seg-142', 'NRNX1-seg-206', 'NRNX1-seg-315', 'NRNX1-seg-38', 'NRNX1-seg-417', 'NRNX1-seg-479', 'NRNX1-seg-539', 'NRNX1-seg-589']\n",
      "NDB_532 -> ['NRNX1-seg-143', 'NRNX1-seg-18', 'NRNX1-seg-217', 'NRNX1-seg-282', 'NRNX1-seg-376', 'NRNX1-seg-438', 'NRNX1-seg-530', 'NRNX1-seg-696', 'NRNX1-seg-91']\n",
      "NDB_533 -> ['NRNX1-seg-1', 'NRNX1-seg-173', 'NRNX1-seg-233', 'NRNX1-seg-278', 'NRNX1-seg-394', 'NRNX1-seg-455', 'NRNX1-seg-536', 'NRNX1-seg-55', 'NRNX1-seg-636']\n",
      "NDB_536 -> ['NRNX1-seg-133', 'NRNX1-seg-227', 'NRNX1-seg-28', 'NRNX1-seg-304', 'NRNX1-seg-366', 'NRNX1-seg-418', 'NRNX1-seg-570', 'NRNX1-seg-651']\n",
      "NDB_539 -> ['NRNX1-seg-253', 'NRNX1-seg-328', 'NRNX1-seg-380', 'NRNX1-seg-41', 'NRNX1-seg-430', 'NRNX1-seg-522', 'NRNX1-seg-602', 'NRNX1-seg-688']\n",
      "NDB_544 -> ['NRNX1-seg-160', 'NRNX1-seg-23', 'NRNX1-seg-256', 'NRNX1-seg-311', 'NRNX1-seg-403', 'NRNX1-seg-454', 'NRNX1-seg-525', 'NRNX1-seg-646']\n",
      "NDB_551 -> ['NRNX1-seg-115', 'NRNX1-seg-181', 'NRNX1-seg-308', 'NRNX1-seg-382', 'NRNX1-seg-43', 'NRNX1-seg-442', 'NRNX1-seg-504', 'NRNX1-seg-594']\n",
      "NDB_554 -> ['NRNX1-seg-119', 'NRNX1-seg-184', 'NRNX1-seg-258', 'NRNX1-seg-36', 'NRNX1-seg-385', 'NRNX1-seg-456', 'NRNX1-seg-507', 'NRNX1-seg-557', 'NRNX1-seg-693']\n",
      "NDB_557 -> ['NRNX1-seg-105', 'NRNX1-seg-191', 'NRNX1-seg-242', 'NRNX1-seg-330', 'NRNX1-seg-382', 'NRNX1-seg-524', 'NRNX1-seg-574', 'NRNX1-seg-634']\n",
      "NDB_568 -> ['NRNX1-seg-109', 'NRNX1-seg-16', 'NRNX1-seg-259', 'NRNX1-seg-364', 'NRNX1-seg-426', 'NRNX1-seg-498', 'NRNX1-seg-548', 'NRNX1-seg-588', 'NRNX1-seg-628']\n",
      "NDB_571 -> ['NRNX1-seg-152', 'NRNX1-seg-225', 'NRNX1-seg-344', 'NRNX1-seg-406', 'NRNX1-seg-478', 'NRNX1-seg-578', 'NRNX1-seg-628', 'NRNX1-seg-89']\n",
      "NDB_578 -> ['NRNX1-seg-245', 'NRNX1-seg-322', 'NRNX1-seg-35', 'NRNX1-seg-372', 'NRNX1-seg-424', 'NRNX1-seg-516', 'NRNX1-seg-596', 'NRNX1-seg-682']\n",
      "NDB_587 -> ['NRNX1-seg-158', 'NRNX1-seg-265', 'NRNX1-seg-329', 'NRNX1-seg-401', 'NRNX1-seg-472', 'NRNX1-seg-52', 'NRNX1-seg-533', 'NRNX1-seg-573']\n",
      "NDB_594 -> ['NRNX1-seg-10', 'NRNX1-seg-147', 'NRNX1-seg-242', 'NRNX1-seg-392', 'NRNX1-seg-442', 'NRNX1-seg-484', 'NRNX1-seg-53', 'NRNX1-seg-534', 'NRNX1-seg-584']\n",
      "NDB_611 -> ['NRNX1-seg-137', 'NRNX1-seg-242', 'NRNX1-seg-298', 'NRNX1-seg-350', 'NRNX1-seg-474', 'NRNX1-seg-544', 'NRNX1-seg-64', 'NRNX1-seg-645']\n",
      "NDB_614 -> ['NRNX1-seg-138', 'NRNX1-seg-244', 'NRNX1-seg-299', 'NRNX1-seg-351', 'NRNX1-seg-475', 'NRNX1-seg-545', 'NRNX1-seg-646', 'NRNX1-seg-65']\n",
      "NDB_615 -> ['NRNX1-seg-112', 'NRNX1-seg-156', 'NRNX1-seg-283', 'NRNX1-seg-30', 'NRNX1-seg-337', 'NRNX1-seg-449', 'NRNX1-seg-511', 'NRNX1-seg-621', 'NRNX1-seg-687']\n",
      "NDB_616 -> ['NRNX1-seg-155', 'NRNX1-seg-227', 'NRNX1-seg-346', 'NRNX1-seg-408', 'NRNX1-seg-480', 'NRNX1-seg-580', 'NRNX1-seg-630', 'NRNX1-seg-91']\n",
      "NDB_618 -> ['NRNX1-seg-135', 'NRNX1-seg-294', 'NRNX1-seg-358', 'NRNX1-seg-410', 'NRNX1-seg-492', 'NRNX1-seg-552', 'NRNX1-seg-698', 'NRNX1-seg-7']\n",
      "NDB_619 -> ['NRNX1-seg-123', 'NRNX1-seg-198', 'NRNX1-seg-273', 'NRNX1-seg-327', 'NRNX1-seg-367', 'NRNX1-seg-481', 'NRNX1-seg-642', 'NRNX1-seg-71']\n",
      "NDB_621 -> ['NRNX1-seg-175', 'NRNX1-seg-215', 'NRNX1-seg-270', 'NRNX1-seg-386', 'NRNX1-seg-488', 'NRNX1-seg-568', 'NRNX1-seg-628', 'NRNX1-seg-671']\n",
      "NDB_622 -> ['NRNX1-seg-172', 'NRNX1-seg-212', 'NRNX1-seg-277', 'NRNX1-seg-34', 'NRNX1-seg-393', 'NRNX1-seg-454', 'NRNX1-seg-615', 'NRNX1-seg-656', 'NRNX1-seg-86']\n",
      "NDB_627 -> ['NRNX1-seg-202', 'NRNX1-seg-299', 'NRNX1-seg-383', 'NRNX1-seg-454', 'NRNX1-seg-515', 'NRNX1-seg-565', 'NRNX1-seg-668', 'NRNX1-seg-96']\n",
      "NDB_628 -> ['NRNX1-seg-105', 'NRNX1-seg-171', 'NRNX1-seg-231', 'NRNX1-seg-298', 'NRNX1-seg-402', 'NRNX1-seg-463', 'NRNX1-seg-53', 'NRNX1-seg-594']\n",
      "NDB_630 -> ['NRNX1-seg-103', 'NRNX1-seg-189', 'NRNX1-seg-20', 'NRNX1-seg-306', 'NRNX1-seg-348', 'NRNX1-seg-471', 'NRNX1-seg-542', 'NRNX1-seg-622']\n",
      "NDB_631 -> ['NRNX1-seg-101', 'NRNX1-seg-197', 'NRNX1-seg-292', 'NRNX1-seg-418', 'NRNX1-seg-500', 'NRNX1-seg-59', 'NRNX1-seg-600', 'NRNX1-seg-674']\n",
      "NDB_634 -> ['NRNX1-seg-111', 'NRNX1-seg-18', 'NRNX1-seg-262', 'NRNX1-seg-366', 'NRNX1-seg-428', 'NRNX1-seg-500', 'NRNX1-seg-550', 'NRNX1-seg-590', 'NRNX1-seg-630']\n",
      "NDB_637 -> ['NRNX1-seg-104', 'NRNX1-seg-200', 'NRNX1-seg-295', 'NRNX1-seg-421', 'NRNX1-seg-503', 'NRNX1-seg-603', 'NRNX1-seg-63', 'NRNX1-seg-678']\n",
      "NDB_641 -> ['NRNX1-seg-135', 'NRNX1-seg-199', 'NRNX1-seg-240', 'NRNX1-seg-348', 'NRNX1-seg-440', 'NRNX1-seg-51', 'NRNX1-seg-562', 'NRNX1-seg-612', 'NRNX1-seg-688']\n",
      "NDB_643 -> ['NRNX1-seg-108', 'NRNX1-seg-15', 'NRNX1-seg-258', 'NRNX1-seg-363', 'NRNX1-seg-425', 'NRNX1-seg-497', 'NRNX1-seg-547', 'NRNX1-seg-587', 'NRNX1-seg-627']\n",
      "NDB_647 -> ['NRNX1-seg-180', 'NRNX1-seg-254', 'NRNX1-seg-359', 'NRNX1-seg-452', 'NRNX1-seg-52', 'NRNX1-seg-593', 'NRNX1-seg-654', 'NRNX1-seg-94']\n",
      "NDB_652 -> ['NRNX1-seg-139', 'NRNX1-seg-14', 'NRNX1-seg-213', 'NRNX1-seg-278', 'NRNX1-seg-372', 'NRNX1-seg-434', 'NRNX1-seg-526', 'NRNX1-seg-692', 'NRNX1-seg-87']\n",
      "NDB_653 -> ['NRNX1-seg-133', 'NRNX1-seg-197', 'NRNX1-seg-238', 'NRNX1-seg-346', 'NRNX1-seg-438', 'NRNX1-seg-49', 'NRNX1-seg-560', 'NRNX1-seg-610', 'NRNX1-seg-686']\n",
      "NDB_657 -> ['NRNX1-seg-110', 'NRNX1-seg-154', 'NRNX1-seg-27', 'NRNX1-seg-281', 'NRNX1-seg-335', 'NRNX1-seg-447', 'NRNX1-seg-509', 'NRNX1-seg-619', 'NRNX1-seg-685']\n",
      "NDB_660 -> ['NRNX1-seg-114', 'NRNX1-seg-210', 'NRNX1-seg-307', 'NRNX1-seg-359', 'NRNX1-seg-462', 'NRNX1-seg-523', 'NRNX1-seg-666', 'NRNX1-seg-73']\n",
      "NDB_661 -> ['NRNX1-seg-204', 'NRNX1-seg-333', 'NRNX1-seg-385', 'NRNX1-seg-435', 'NRNX1-seg-497', 'NRNX1-seg-577', 'NRNX1-seg-637', 'NRNX1-seg-77']\n",
      "NDB_663 -> ['NRNX1-seg-177', 'NRNX1-seg-217', 'NRNX1-seg-272', 'NRNX1-seg-388', 'NRNX1-seg-490', 'NRNX1-seg-570', 'NRNX1-seg-630', 'NRNX1-seg-674']\n",
      "NDB_665 -> ['NRNX1-seg-180', 'NRNX1-seg-241', 'NRNX1-seg-285', 'NRNX1-seg-401', 'NRNX1-seg-462', 'NRNX1-seg-543', 'NRNX1-seg-63', 'NRNX1-seg-644', 'NRNX1-seg-9']\n",
      "NDB_680 -> ['NRNX1-seg-176', 'NRNX1-seg-249', 'NRNX1-seg-355', 'NRNX1-seg-447', 'NRNX1-seg-48', 'NRNX1-seg-589', 'NRNX1-seg-650', 'NRNX1-seg-90']\n",
      "NDB_681 -> ['NRNX1-seg-142', 'NRNX1-seg-226', 'NRNX1-seg-447', 'NRNX1-seg-489', 'NRNX1-seg-559', 'NRNX1-seg-599', 'NRNX1-seg-660', 'NRNX1-seg-80']\n",
      "NDB_683 -> ['NRNX1-seg-174', 'NRNX1-seg-246', 'NRNX1-seg-353', 'NRNX1-seg-445', 'NRNX1-seg-46', 'NRNX1-seg-587', 'NRNX1-seg-648', 'NRNX1-seg-88']\n",
      "NDB_687 -> ['NRNX1-seg-190', 'NRNX1-seg-254', 'NRNX1-seg-319', 'NRNX1-seg-32', 'NRNX1-seg-401', 'NRNX1-seg-553', 'NRNX1-seg-613', 'NRNX1-seg-678', 'NRNX1-seg-84']\n",
      "NDB_690 -> ['NRNX1-seg-107', 'NRNX1-seg-203', 'NRNX1-seg-300', 'NRNX1-seg-352', 'NRNX1-seg-455', 'NRNX1-seg-516', 'NRNX1-seg-657', 'NRNX1-seg-66']\n",
      "NDB_694 -> ['NRNX1-seg-171', 'NRNX1-seg-211', 'NRNX1-seg-276', 'NRNX1-seg-33', 'NRNX1-seg-392', 'NRNX1-seg-453', 'NRNX1-seg-614', 'NRNX1-seg-655', 'NRNX1-seg-85']\n",
      "NDB_698 -> ['NRNX1-seg-121', 'NRNX1-seg-186', 'NRNX1-seg-260', 'NRNX1-seg-38', 'NRNX1-seg-387', 'NRNX1-seg-458', 'NRNX1-seg-509', 'NRNX1-seg-559', 'NRNX1-seg-695']\n",
      "NDB_699 -> ['NRNX1-seg-139', 'NRNX1-seg-223', 'NRNX1-seg-444', 'NRNX1-seg-486', 'NRNX1-seg-556', 'NRNX1-seg-596', 'NRNX1-seg-657', 'NRNX1-seg-76']\n",
      "NDB_705 -> ['NRNX1-seg-159', 'NRNX1-seg-221', 'NRNX1-seg-412', 'NRNX1-seg-43', 'NRNX1-seg-463', 'NRNX1-seg-534', 'NRNX1-seg-604', 'NRNX1-seg-667', 'NRNX1-seg-85']\n",
      "NDB_706 -> ['NRNX1-seg-158', 'NRNX1-seg-230', 'NRNX1-seg-349', 'NRNX1-seg-411', 'NRNX1-seg-483', 'NRNX1-seg-583', 'NRNX1-seg-633', 'NRNX1-seg-94']\n",
      "NDB_710 -> ['NRNX1-seg-118', 'NRNX1-seg-183', 'NRNX1-seg-257', 'NRNX1-seg-35', 'NRNX1-seg-384', 'NRNX1-seg-455', 'NRNX1-seg-506', 'NRNX1-seg-556', 'NRNX1-seg-692']\n",
      "NDB_711 -> ['NRNX1-seg-146', 'NRNX1-seg-230', 'NRNX1-seg-452', 'NRNX1-seg-493', 'NRNX1-seg-563', 'NRNX1-seg-603', 'NRNX1-seg-666', 'NRNX1-seg-84']\n",
      "NDB_712 -> ['NRNX1-seg-117', 'NRNX1-seg-202', 'NRNX1-seg-277', 'NRNX1-seg-361', 'NRNX1-seg-413', 'NRNX1-seg-54', 'NRNX1-seg-545', 'NRNX1-seg-625', 'NRNX1-seg-691']\n",
      "NDB_713 -> ['NRNX1-seg-128', 'NRNX1-seg-223', 'NRNX1-seg-24', 'NRNX1-seg-300', 'NRNX1-seg-362', 'NRNX1-seg-414', 'NRNX1-seg-566', 'NRNX1-seg-647']\n",
      "NDB_715 -> ['NRNX1-seg-143', 'NRNX1-seg-207', 'NRNX1-seg-316', 'NRNX1-seg-39', 'NRNX1-seg-418', 'NRNX1-seg-480', 'NRNX1-seg-540', 'NRNX1-seg-590']\n",
      "NDB_716 -> ['NRNX1-seg-120', 'NRNX1-seg-163', 'NRNX1-seg-215', 'NRNX1-seg-314', 'NRNX1-seg-406', 'NRNX1-seg-518', 'NRNX1-seg-57', 'NRNX1-seg-618']\n",
      "NDB_718 -> ['NRNX1-seg-150', 'NRNX1-seg-223', 'NRNX1-seg-342', 'NRNX1-seg-404', 'NRNX1-seg-476', 'NRNX1-seg-576', 'NRNX1-seg-626', 'NRNX1-seg-87']\n",
      "NDB_721 -> ['NRNX1-seg-103', 'NRNX1-seg-199', 'NRNX1-seg-294', 'NRNX1-seg-420', 'NRNX1-seg-502', 'NRNX1-seg-602', 'NRNX1-seg-62', 'NRNX1-seg-676']\n",
      "NDB_725 -> ['NRNX1-seg-128', 'NRNX1-seg-193', 'NRNX1-seg-233', 'NRNX1-seg-342', 'NRNX1-seg-434', 'NRNX1-seg-45', 'NRNX1-seg-556', 'NRNX1-seg-606', 'NRNX1-seg-682']\n",
      "NDB_728 -> ['NRNX1-seg-151', 'NRNX1-seg-258', 'NRNX1-seg-323', 'NRNX1-seg-395', 'NRNX1-seg-46', 'NRNX1-seg-466', 'NRNX1-seg-527', 'NRNX1-seg-567']\n",
      "NDB_730 -> ['NRNX1-seg-144', 'NRNX1-seg-19', 'NRNX1-seg-218', 'NRNX1-seg-283', 'NRNX1-seg-379', 'NRNX1-seg-439', 'NRNX1-seg-531', 'NRNX1-seg-697', 'NRNX1-seg-92']\n",
      "NDB_734 -> ['NRNX1-seg-136', 'NRNX1-seg-200', 'NRNX1-seg-241', 'NRNX1-seg-349', 'NRNX1-seg-441', 'NRNX1-seg-52', 'NRNX1-seg-563', 'NRNX1-seg-613', 'NRNX1-seg-689']\n",
      "NDB_735 -> ['NRNX1-seg-127', 'NRNX1-seg-13', 'NRNX1-seg-212', 'NRNX1-seg-321', 'NRNX1-seg-361', 'NRNX1-seg-464', 'NRNX1-seg-515', 'NRNX1-seg-646']\n",
      "NDB_737 -> ['NRNX1-seg-131', 'NRNX1-seg-196', 'NRNX1-seg-236', 'NRNX1-seg-345', 'NRNX1-seg-437', 'NRNX1-seg-48', 'NRNX1-seg-559', 'NRNX1-seg-609', 'NRNX1-seg-685']\n",
      "NDB_738 -> ['NRNX1-seg-162', 'NRNX1-seg-234', 'NRNX1-seg-289', 'NRNX1-seg-373', 'NRNX1-seg-466', 'NRNX1-seg-507', 'NRNX1-seg-577', 'NRNX1-seg-648', 'NRNX1-seg-67']\n",
      "NDB_741 -> ['NRNX1-seg-111', 'NRNX1-seg-207', 'NRNX1-seg-304', 'NRNX1-seg-356', 'NRNX1-seg-459', 'NRNX1-seg-520', 'NRNX1-seg-661', 'NRNX1-seg-70']\n",
      "NDB_742 -> ['NRNX1-seg-171', 'NRNX1-seg-221', 'NRNX1-seg-308', 'NRNX1-seg-350', 'NRNX1-seg-432', 'NRNX1-seg-494', 'NRNX1-seg-554', 'NRNX1-seg-634']\n",
      "NDB_743 -> ['NRNX1-seg-150', 'NRNX1-seg-257', 'NRNX1-seg-322', 'NRNX1-seg-394', 'NRNX1-seg-45', 'NRNX1-seg-465', 'NRNX1-seg-526', 'NRNX1-seg-566']\n",
      "NDB_746 -> ['NRNX1-seg-130', 'NRNX1-seg-225', 'NRNX1-seg-26', 'NRNX1-seg-302', 'NRNX1-seg-364', 'NRNX1-seg-416', 'NRNX1-seg-568', 'NRNX1-seg-649']\n",
      "NDB_747 -> ['NRNX1-seg-144', 'NRNX1-seg-228', 'NRNX1-seg-449', 'NRNX1-seg-491', 'NRNX1-seg-561', 'NRNX1-seg-601', 'NRNX1-seg-662', 'NRNX1-seg-82']\n",
      "NDB_750 -> ['NRNX1-seg-174', 'NRNX1-seg-214', 'NRNX1-seg-269', 'NRNX1-seg-385', 'NRNX1-seg-487', 'NRNX1-seg-567', 'NRNX1-seg-627', 'NRNX1-seg-670']\n",
      "NDB_751 -> ['NRNX1-seg-147', 'NRNX1-seg-231', 'NRNX1-seg-286', 'NRNX1-seg-370', 'NRNX1-seg-514', 'NRNX1-seg-574', 'NRNX1-seg-614', 'NRNX1-seg-64', 'NRNX1-seg-690']\n",
      "NDB_752 -> ['NRNX1-seg-123', 'NRNX1-seg-188', 'NRNX1-seg-263', 'NRNX1-seg-389', 'NRNX1-seg-40', 'NRNX1-seg-460', 'NRNX1-seg-511', 'NRNX1-seg-561', 'NRNX1-seg-697']\n",
      "NDB_754 -> ['NRNX1-seg-142', 'NRNX1-seg-17', 'NRNX1-seg-216', 'NRNX1-seg-281', 'NRNX1-seg-375', 'NRNX1-seg-437', 'NRNX1-seg-529', 'NRNX1-seg-695', 'NRNX1-seg-90']\n",
      "NDB_755 -> ['NRNX1-seg-173', 'NRNX1-seg-245', 'NRNX1-seg-352', 'NRNX1-seg-444', 'NRNX1-seg-45', 'NRNX1-seg-586', 'NRNX1-seg-647', 'NRNX1-seg-87']\n",
      "NDB_758 -> ['NRNX1-seg-251', 'NRNX1-seg-326', 'NRNX1-seg-376', 'NRNX1-seg-39', 'NRNX1-seg-428', 'NRNX1-seg-520', 'NRNX1-seg-600', 'NRNX1-seg-686']\n",
      "NDB_759 -> ['NRNX1-seg-183', 'NRNX1-seg-24', 'NRNX1-seg-245', 'NRNX1-seg-312', 'NRNX1-seg-394', 'NRNX1-seg-546', 'NRNX1-seg-606', 'NRNX1-seg-669', 'NRNX1-seg-76']\n",
      "NDB_760 -> ['NRNX1-seg-140', 'NRNX1-seg-204', 'NRNX1-seg-313', 'NRNX1-seg-36', 'NRNX1-seg-415', 'NRNX1-seg-477', 'NRNX1-seg-537', 'NRNX1-seg-587']\n",
      "NDB_762 -> ['NRNX1-seg-140', 'NRNX1-seg-224', 'NRNX1-seg-445', 'NRNX1-seg-487', 'NRNX1-seg-557', 'NRNX1-seg-597', 'NRNX1-seg-658', 'NRNX1-seg-77']\n",
      "NDB_768 -> ['NRNX1-seg-108', 'NRNX1-seg-151', 'NRNX1-seg-25', 'NRNX1-seg-279', 'NRNX1-seg-333', 'NRNX1-seg-445', 'NRNX1-seg-507', 'NRNX1-seg-617', 'NRNX1-seg-683']\n",
      "NDB_771 -> ['NRNX1-seg-111', 'NRNX1-seg-155', 'NRNX1-seg-28', 'NRNX1-seg-282', 'NRNX1-seg-336', 'NRNX1-seg-448', 'NRNX1-seg-510', 'NRNX1-seg-620', 'NRNX1-seg-686']\n",
      "NDB_773 -> ['NRNX1-seg-130', 'NRNX1-seg-195', 'NRNX1-seg-235', 'NRNX1-seg-344', 'NRNX1-seg-436', 'NRNX1-seg-47', 'NRNX1-seg-558', 'NRNX1-seg-608', 'NRNX1-seg-684']\n",
      "NDB_777 -> ['NRNX1-seg-173', 'NRNX1-seg-213', 'NRNX1-seg-268', 'NRNX1-seg-384', 'NRNX1-seg-486', 'NRNX1-seg-566', 'NRNX1-seg-626', 'NRNX1-seg-669']\n",
      "NDB_778 -> ['NRNX1-seg-194', 'NRNX1-seg-289', 'NRNX1-seg-415', 'NRNX1-seg-497', 'NRNX1-seg-56', 'NRNX1-seg-597', 'NRNX1-seg-670', 'NRNX1-seg-98']\n",
      "NDB_779 -> ['NRNX1-seg-159', 'NRNX1-seg-276', 'NRNX1-seg-340', 'NRNX1-seg-484', 'NRNX1-seg-564', 'NRNX1-seg-634', 'NRNX1-seg-64', 'NRNX1-seg-679']\n",
      "NDB_780 -> ['NRNX1-seg-102', 'NRNX1-seg-188', 'NRNX1-seg-19', 'NRNX1-seg-305', 'NRNX1-seg-347', 'NRNX1-seg-470', 'NRNX1-seg-541', 'NRNX1-seg-621']\n",
      "NDB_784 -> ['NRNX1-seg-107', 'NRNX1-seg-14', 'NRNX1-seg-257', 'NRNX1-seg-362', 'NRNX1-seg-424', 'NRNX1-seg-496', 'NRNX1-seg-546', 'NRNX1-seg-586', 'NRNX1-seg-626']\n",
      "NDB_786 -> ['NRNX1-seg-141', 'NRNX1-seg-225', 'NRNX1-seg-446', 'NRNX1-seg-488', 'NRNX1-seg-558', 'NRNX1-seg-598', 'NRNX1-seg-659', 'NRNX1-seg-79']\n",
      "NDB_788 -> ['NRNX1-seg-125', 'NRNX1-seg-170', 'NRNX1-seg-220', 'NRNX1-seg-319', 'NRNX1-seg-411', 'NRNX1-seg-523', 'NRNX1-seg-623', 'NRNX1-seg-63']\n",
      "NDB_789 -> ['NRNX1-seg-133', 'NRNX1-seg-292', 'NRNX1-seg-356', 'NRNX1-seg-408', 'NRNX1-seg-490', 'NRNX1-seg-5', 'NRNX1-seg-550', 'NRNX1-seg-696']\n",
      "NDB_790 -> ['NRNX1-seg-119', 'NRNX1-seg-194', 'NRNX1-seg-269', 'NRNX1-seg-323', 'NRNX1-seg-363', 'NRNX1-seg-477', 'NRNX1-seg-637', 'NRNX1-seg-67']\n",
      "NDB_795 -> ['NRNX1-seg-161', 'NRNX1-seg-233', 'NRNX1-seg-288', 'NRNX1-seg-372', 'NRNX1-seg-465', 'NRNX1-seg-506', 'NRNX1-seg-576', 'NRNX1-seg-647', 'NRNX1-seg-66']\n",
      "NDB_801 -> ['NRNX1-seg-14', 'NRNX1-seg-183', 'NRNX1-seg-300', 'NRNX1-seg-342', 'NRNX1-seg-465', 'NRNX1-seg-536', 'NRNX1-seg-616', 'NRNX1-seg-97']\n",
      "NDB_804 -> ['NRNX1-seg-129', 'NRNX1-seg-2', 'NRNX1-seg-289', 'NRNX1-seg-353', 'NRNX1-seg-405', 'NRNX1-seg-487', 'NRNX1-seg-547', 'NRNX1-seg-693']\n",
      "NDB_805 -> ['NRNX1-seg-102', 'NRNX1-seg-198', 'NRNX1-seg-293', 'NRNX1-seg-419', 'NRNX1-seg-501', 'NRNX1-seg-60', 'NRNX1-seg-601', 'NRNX1-seg-675']\n",
      "NDB_807 -> ['NRNX1-seg-107', 'NRNX1-seg-150', 'NRNX1-seg-24', 'NRNX1-seg-278', 'NRNX1-seg-332', 'NRNX1-seg-444', 'NRNX1-seg-506', 'NRNX1-seg-616', 'NRNX1-seg-682']\n",
      "NDB_812 -> ['NRNX1-seg-120', 'NRNX1-seg-185', 'NRNX1-seg-259', 'NRNX1-seg-37', 'NRNX1-seg-386', 'NRNX1-seg-457', 'NRNX1-seg-508', 'NRNX1-seg-558', 'NRNX1-seg-694']\n",
      "NDB_813 -> ['NRNX1-seg-179', 'NRNX1-seg-219', 'NRNX1-seg-274', 'NRNX1-seg-390', 'NRNX1-seg-492', 'NRNX1-seg-572', 'NRNX1-seg-632', 'NRNX1-seg-676']\n",
      "NDB_815 -> ['NRNX1-seg-246', 'NRNX1-seg-323', 'NRNX1-seg-36', 'NRNX1-seg-373', 'NRNX1-seg-425', 'NRNX1-seg-517', 'NRNX1-seg-597', 'NRNX1-seg-683']\n",
      "NDB_816 -> ['NRNX1-seg-113', 'NRNX1-seg-209', 'NRNX1-seg-306', 'NRNX1-seg-358', 'NRNX1-seg-461', 'NRNX1-seg-522', 'NRNX1-seg-665', 'NRNX1-seg-72']\n",
      "NDB_817 -> ['NRNX1-seg-193', 'NRNX1-seg-288', 'NRNX1-seg-414', 'NRNX1-seg-496', 'NRNX1-seg-55', 'NRNX1-seg-596', 'NRNX1-seg-669', 'NRNX1-seg-97']\n",
      "NDB_819 -> ['NRNX1-seg-136', 'NRNX1-seg-295', 'NRNX1-seg-359', 'NRNX1-seg-411', 'NRNX1-seg-493', 'NRNX1-seg-553', 'NRNX1-seg-699', 'NRNX1-seg-9']\n",
      "NDB_824 -> ['NRNX1-seg-13', 'NRNX1-seg-138', 'NRNX1-seg-267', 'NRNX1-seg-341', 'NRNX1-seg-433', 'NRNX1-seg-505', 'NRNX1-seg-615', 'NRNX1-seg-96']\n",
      "NDB_825 -> ['NRNX1-seg-15', 'NRNX1-seg-184', 'NRNX1-seg-301', 'NRNX1-seg-343', 'NRNX1-seg-466', 'NRNX1-seg-537', 'NRNX1-seg-617', 'NRNX1-seg-98']\n",
      "NDB_826 -> ['NRNX1-seg-139', 'NRNX1-seg-203', 'NRNX1-seg-312', 'NRNX1-seg-35', 'NRNX1-seg-414', 'NRNX1-seg-476', 'NRNX1-seg-536', 'NRNX1-seg-586']\n",
      "NDB_828 -> ['NRNX1-seg-114', 'NRNX1-seg-158', 'NRNX1-seg-285', 'NRNX1-seg-32', 'NRNX1-seg-339', 'NRNX1-seg-452', 'NRNX1-seg-513', 'NRNX1-seg-623', 'NRNX1-seg-689']\n",
      "NDB_832 -> ['NRNX1-seg-208', 'NRNX1-seg-337', 'NRNX1-seg-389', 'NRNX1-seg-439', 'NRNX1-seg-501', 'NRNX1-seg-581', 'NRNX1-seg-642', 'NRNX1-seg-82']\n",
      "NDB_835 -> ['NRNX1-seg-113', 'NRNX1-seg-20', 'NRNX1-seg-264', 'NRNX1-seg-368', 'NRNX1-seg-430', 'NRNX1-seg-502', 'NRNX1-seg-552', 'NRNX1-seg-592', 'NRNX1-seg-632']\n",
      "NDB_836 -> ['NRNX1-seg-117', 'NRNX1-seg-182', 'NRNX1-seg-311', 'NRNX1-seg-383', 'NRNX1-seg-44', 'NRNX1-seg-443', 'NRNX1-seg-505', 'NRNX1-seg-595']\n",
      "NDB_838 -> ['NRNX1-seg-114', 'NRNX1-seg-21', 'NRNX1-seg-265', 'NRNX1-seg-369', 'NRNX1-seg-431', 'NRNX1-seg-503', 'NRNX1-seg-553', 'NRNX1-seg-593', 'NRNX1-seg-633']\n",
      "NDB_840 -> ['NRNX1-seg-117', 'NRNX1-seg-192', 'NRNX1-seg-256', 'NRNX1-seg-371', 'NRNX1-seg-433', 'NRNX1-seg-585', 'NRNX1-seg-656', 'NRNX1-seg-75']\n",
      "NDB_844 -> ['NRNX1-seg-140', 'NRNX1-seg-15', 'NRNX1-seg-214', 'NRNX1-seg-279', 'NRNX1-seg-373', 'NRNX1-seg-435', 'NRNX1-seg-527', 'NRNX1-seg-693', 'NRNX1-seg-88']\n",
      "NDB_846 -> ['NRNX1-seg-189', 'NRNX1-seg-253', 'NRNX1-seg-31', 'NRNX1-seg-318', 'NRNX1-seg-400', 'NRNX1-seg-552', 'NRNX1-seg-612', 'NRNX1-seg-676', 'NRNX1-seg-83']\n",
      "NDB_847 -> ['NRNX1-seg-203', 'NRNX1-seg-332', 'NRNX1-seg-384', 'NRNX1-seg-434', 'NRNX1-seg-496', 'NRNX1-seg-576', 'NRNX1-seg-636', 'NRNX1-seg-76']\n",
      "NDB_852 -> ['NRNX1-seg-108', 'NRNX1-seg-204', 'NRNX1-seg-301', 'NRNX1-seg-353', 'NRNX1-seg-456', 'NRNX1-seg-517', 'NRNX1-seg-658', 'NRNX1-seg-67']\n",
      "NDB_853 -> ['NRNX1-seg-145', 'NRNX1-seg-20', 'NRNX1-seg-219', 'NRNX1-seg-284', 'NRNX1-seg-380', 'NRNX1-seg-440', 'NRNX1-seg-532', 'NRNX1-seg-698', 'NRNX1-seg-93']\n",
      "NDB_856 -> ['NRNX1-seg-154', 'NRNX1-seg-226', 'NRNX1-seg-345', 'NRNX1-seg-407', 'NRNX1-seg-479', 'NRNX1-seg-579', 'NRNX1-seg-629', 'NRNX1-seg-90']\n",
      "NDB_858 -> ['NRNX1-seg-130', 'NRNX1-seg-290', 'NRNX1-seg-3', 'NRNX1-seg-354', 'NRNX1-seg-406', 'NRNX1-seg-488', 'NRNX1-seg-548', 'NRNX1-seg-694']\n",
      "NDB_860 -> ['NRNX1-seg-152', 'NRNX1-seg-259', 'NRNX1-seg-324', 'NRNX1-seg-396', 'NRNX1-seg-467', 'NRNX1-seg-47', 'NRNX1-seg-528', 'NRNX1-seg-568']\n",
      "NDB_862 -> ['NRNX1-seg-137', 'NRNX1-seg-255', 'NRNX1-seg-33', 'NRNX1-seg-330', 'NRNX1-seg-422', 'NRNX1-seg-534', 'NRNX1-seg-594', 'NRNX1-seg-690']\n",
      "NDB_863 -> ['NRNX1-seg-121', 'NRNX1-seg-164', 'NRNX1-seg-216', 'NRNX1-seg-315', 'NRNX1-seg-407', 'NRNX1-seg-519', 'NRNX1-seg-58', 'NRNX1-seg-619']\n",
      "NDB_864 -> ['NRNX1-seg-169', 'NRNX1-seg-240', 'NRNX1-seg-294', 'NRNX1-seg-380', 'NRNX1-seg-471', 'NRNX1-seg-512', 'NRNX1-seg-582', 'NRNX1-seg-653', 'NRNX1-seg-72']\n",
      "NDB_865 -> ['NRNX1-seg-118', 'NRNX1-seg-193', 'NRNX1-seg-268', 'NRNX1-seg-322', 'NRNX1-seg-362', 'NRNX1-seg-476', 'NRNX1-seg-636', 'NRNX1-seg-66']\n",
      "NDB_866 -> ['NRNX1-seg-252', 'NRNX1-seg-327', 'NRNX1-seg-379', 'NRNX1-seg-40', 'NRNX1-seg-429', 'NRNX1-seg-521', 'NRNX1-seg-601', 'NRNX1-seg-687']\n",
      "NDB_868 -> ['NRNX1-seg-110', 'NRNX1-seg-17', 'NRNX1-seg-260', 'NRNX1-seg-365', 'NRNX1-seg-427', 'NRNX1-seg-499', 'NRNX1-seg-549', 'NRNX1-seg-589', 'NRNX1-seg-629']\n",
      "NDB_875 -> ['NRNX1-seg-175', 'NRNX1-seg-235', 'NRNX1-seg-280', 'NRNX1-seg-3', 'NRNX1-seg-396', 'NRNX1-seg-457', 'NRNX1-seg-538', 'NRNX1-seg-57', 'NRNX1-seg-639']\n",
      "NDB_878 -> ['NRNX1-seg-154', 'NRNX1-seg-260', 'NRNX1-seg-325', 'NRNX1-seg-397', 'NRNX1-seg-468', 'NRNX1-seg-48', 'NRNX1-seg-529', 'NRNX1-seg-569']\n",
      "NDB_879 -> ['NRNX1-seg-115', 'NRNX1-seg-191', 'NRNX1-seg-255', 'NRNX1-seg-370', 'NRNX1-seg-432', 'NRNX1-seg-584', 'NRNX1-seg-655', 'NRNX1-seg-74']\n",
      "NDB_880 -> ['NRNX1-seg-156', 'NRNX1-seg-228', 'NRNX1-seg-347', 'NRNX1-seg-409', 'NRNX1-seg-481', 'NRNX1-seg-581', 'NRNX1-seg-631', 'NRNX1-seg-92']\n",
      "NDB_881 -> ['NRNX1-seg-254', 'NRNX1-seg-329', 'NRNX1-seg-381', 'NRNX1-seg-42', 'NRNX1-seg-431', 'NRNX1-seg-523', 'NRNX1-seg-603', 'NRNX1-seg-689']\n",
      "NDB_883 -> ['NRNX1-seg-101', 'NRNX1-seg-165', 'NRNX1-seg-272', 'NRNX1-seg-336', 'NRNX1-seg-428', 'NRNX1-seg-5', 'NRNX1-seg-530', 'NRNX1-seg-610', 'NRNX1-seg-661']\n",
      "NDB_887 -> ['NRNX1-seg-134', 'NRNX1-seg-228', 'NRNX1-seg-30', 'NRNX1-seg-305', 'NRNX1-seg-367', 'NRNX1-seg-419', 'NRNX1-seg-571', 'NRNX1-seg-652']\n",
      "NDB_889 -> ['NRNX1-seg-210', 'NRNX1-seg-339', 'NRNX1-seg-391', 'NRNX1-seg-441', 'NRNX1-seg-503', 'NRNX1-seg-583', 'NRNX1-seg-644', 'NRNX1-seg-84']\n",
      "NDB_896 -> ['NRNX1-seg-135', 'NRNX1-seg-229', 'NRNX1-seg-306', 'NRNX1-seg-31', 'NRNX1-seg-368', 'NRNX1-seg-420', 'NRNX1-seg-572', 'NRNX1-seg-653']\n",
      "NDB_899 -> ['NRNX1-seg-124', 'NRNX1-seg-189', 'NRNX1-seg-264', 'NRNX1-seg-390', 'NRNX1-seg-41', 'NRNX1-seg-461', 'NRNX1-seg-512', 'NRNX1-seg-562', 'NRNX1-seg-698']\n",
      "NDB_901 -> ['NRNX1-seg-120', 'NRNX1-seg-195', 'NRNX1-seg-270', 'NRNX1-seg-324', 'NRNX1-seg-364', 'NRNX1-seg-478', 'NRNX1-seg-639', 'NRNX1-seg-68']\n",
      "NDB_903 -> ['NRNX1-seg-13', 'NRNX1-seg-148', 'NRNX1-seg-244', 'NRNX1-seg-393', 'NRNX1-seg-443', 'NRNX1-seg-485', 'NRNX1-seg-535', 'NRNX1-seg-54', 'NRNX1-seg-585']\n",
      "NDB_905 -> ['NRNX1-seg-122', 'NRNX1-seg-165', 'NRNX1-seg-217', 'NRNX1-seg-316', 'NRNX1-seg-408', 'NRNX1-seg-520', 'NRNX1-seg-59', 'NRNX1-seg-620']\n",
      "NDB_915 -> ['NRNX1-seg-185', 'NRNX1-seg-248', 'NRNX1-seg-26', 'NRNX1-seg-314', 'NRNX1-seg-396', 'NRNX1-seg-548', 'NRNX1-seg-608', 'NRNX1-seg-671', 'NRNX1-seg-79']\n",
      "NDB_917 -> ['NRNX1-seg-119', 'NRNX1-seg-162', 'NRNX1-seg-214', 'NRNX1-seg-313', 'NRNX1-seg-405', 'NRNX1-seg-517', 'NRNX1-seg-56', 'NRNX1-seg-617']\n",
      "NDB_921 -> ['NRNX1-seg-166', 'NRNX1-seg-239', 'NRNX1-seg-293', 'NRNX1-seg-379', 'NRNX1-seg-470', 'NRNX1-seg-511', 'NRNX1-seg-581', 'NRNX1-seg-652', 'NRNX1-seg-71']\n",
      "NDB_925 -> ['NRNX1-seg-172', 'NRNX1-seg-222', 'NRNX1-seg-311', 'NRNX1-seg-351', 'NRNX1-seg-433', 'NRNX1-seg-495', 'NRNX1-seg-555', 'NRNX1-seg-635']\n",
      "NDB_929 -> ['NRNX1-seg-106', 'NRNX1-seg-192', 'NRNX1-seg-244', 'NRNX1-seg-331', 'NRNX1-seg-383', 'NRNX1-seg-525', 'NRNX1-seg-575', 'NRNX1-seg-635']\n",
      "NDB_933 -> ['NRNX1-seg-182', 'NRNX1-seg-287', 'NRNX1-seg-341', 'NRNX1-seg-393', 'NRNX1-seg-475', 'NRNX1-seg-525', 'NRNX1-seg-605', 'NRNX1-seg-75']\n",
      "NDB_935 -> ['NRNX1-seg-125', 'NRNX1-seg-190', 'NRNX1-seg-265', 'NRNX1-seg-391', 'NRNX1-seg-42', 'NRNX1-seg-462', 'NRNX1-seg-513', 'NRNX1-seg-563', 'NRNX1-seg-699']\n",
      "NDB_936 -> ['NRNX1-seg-1', 'NRNX1-seg-128', 'NRNX1-seg-288', 'NRNX1-seg-352', 'NRNX1-seg-404', 'NRNX1-seg-486', 'NRNX1-seg-546', 'NRNX1-seg-692']\n",
      "NDB_941 -> ['NRNX1-seg-124', 'NRNX1-seg-169', 'NRNX1-seg-219', 'NRNX1-seg-318', 'NRNX1-seg-410', 'NRNX1-seg-522', 'NRNX1-seg-62', 'NRNX1-seg-622']\n",
      "NDB_946 -> ['NRNX1-seg-124', 'NRNX1-seg-199', 'NRNX1-seg-274', 'NRNX1-seg-328', 'NRNX1-seg-368', 'NRNX1-seg-482', 'NRNX1-seg-643', 'NRNX1-seg-72']\n",
      "NDB_951 -> ['NRNX1-seg-10', 'NRNX1-seg-126', 'NRNX1-seg-211', 'NRNX1-seg-320', 'NRNX1-seg-360', 'NRNX1-seg-463', 'NRNX1-seg-514', 'NRNX1-seg-645']\n",
      "NDB_954 -> ['NRNX1-seg-101', 'NRNX1-seg-18', 'NRNX1-seg-187', 'NRNX1-seg-304', 'NRNX1-seg-346', 'NRNX1-seg-469', 'NRNX1-seg-540', 'NRNX1-seg-620']\n",
      "NDB_957 -> ['NRNX1-seg-186', 'NRNX1-seg-249', 'NRNX1-seg-27', 'NRNX1-seg-315', 'NRNX1-seg-397', 'NRNX1-seg-549', 'NRNX1-seg-609', 'NRNX1-seg-672', 'NRNX1-seg-80']\n",
      "NDB_958 -> ['NRNX1-seg-115', 'NRNX1-seg-201', 'NRNX1-seg-276', 'NRNX1-seg-360', 'NRNX1-seg-412', 'NRNX1-seg-53', 'NRNX1-seg-544', 'NRNX1-seg-624', 'NRNX1-seg-690']\n",
      "NDB_961 -> ['NRNX1-seg-151', 'NRNX1-seg-224', 'NRNX1-seg-343', 'NRNX1-seg-405', 'NRNX1-seg-477', 'NRNX1-seg-577', 'NRNX1-seg-627', 'NRNX1-seg-88']\n",
      "NDB_965 -> ['NRNX1-seg-129', 'NRNX1-seg-224', 'NRNX1-seg-25', 'NRNX1-seg-301', 'NRNX1-seg-363', 'NRNX1-seg-415', 'NRNX1-seg-567', 'NRNX1-seg-648']\n",
      "NDB_971 -> ['NRNX1-seg-118', 'NRNX1-seg-161', 'NRNX1-seg-213', 'NRNX1-seg-312', 'NRNX1-seg-404', 'NRNX1-seg-516', 'NRNX1-seg-55', 'NRNX1-seg-616']\n",
      "NDB_974 -> ['NRNX1-seg-248', 'NRNX1-seg-324', 'NRNX1-seg-37', 'NRNX1-seg-374', 'NRNX1-seg-426', 'NRNX1-seg-518', 'NRNX1-seg-598', 'NRNX1-seg-684']\n",
      "NDB_977 -> ['NRNX1-seg-105', 'NRNX1-seg-22', 'NRNX1-seg-266', 'NRNX1-seg-360', 'NRNX1-seg-422', 'NRNX1-seg-484', 'NRNX1-seg-554', 'NRNX1-seg-604']\n",
      "NDB_989 -> ['NRNX1-seg-179', 'NRNX1-seg-240', 'NRNX1-seg-284', 'NRNX1-seg-400', 'NRNX1-seg-461', 'NRNX1-seg-542', 'NRNX1-seg-62', 'NRNX1-seg-643', 'NRNX1-seg-7']\n",
      "NDB_994 -> ['NRNX1-seg-146', 'NRNX1-seg-210', 'NRNX1-seg-319', 'NRNX1-seg-42', 'NRNX1-seg-421', 'NRNX1-seg-483', 'NRNX1-seg-543', 'NRNX1-seg-593']\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "readout_reg_dic, readout2reg_check = check._check_readout_to_region(reg_readout_dic, full_records, \n",
    "                                                                    check_readout_dict, primer_len=primer_len, \n",
    "                                                                    target_len=target_len,\n",
    "                                                                    probe_readout_num=num_readout_per_probe)\n",
    "print(readout2reg_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7445b10e",
   "metadata": {},
   "source": [
    "### 4.1.5 check bewteen probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "id": "3a6a50cc",
   "metadata": {},
   "outputs": [],
   "source": [
    "# construct internal map\n",
    "reload(check)\n",
    "int_map = check._construct_internal_map(full_records, library_folder, save=False) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "id": "11132a68",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "True\n"
     ]
    }
   ],
   "source": [
    "# internal check probes\n",
    "readout_count_dic, readout_count_check = check._check_readout_in_probes(readout_reg_dic, reg_size_dic, int_map, readout_dict)\n",
    "print(readout_count_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "id": "056864be",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_2:50850000-50855000_gene_NRNX1-seg-571_pb_51_pos_4749_strand_+_readouts_[NDB_887_c,NDB_887_c,NDB_887_c,NDB_887_c]_primers_[W1A01_primer_0,W1A02_primer_1] got hits:103, dumped\n",
      "-- total probes removed by internal screening: 1\n"
     ]
    }
   ],
   "source": [
    "# check between probes\n",
    "kept_records, removed_count = check._check_between_probes(full_records, int_map, \n",
    "                                                          primer_len=primer_len, \n",
    "                                                          target_len=target_len)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 148,
   "id": "495f8a7d",
   "metadata": {},
   "outputs": [],
   "source": [
    "# clear memory\n",
    "del(int_map)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ef0c0c1c",
   "metadata": {},
   "source": [
    "### 4.1.6 finish checking, save"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "id": "0b11845a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "34067 34068\n"
     ]
    }
   ],
   "source": [
    "print(len(kept_records), len(cand_probes))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "id": "56768f43",
   "metadata": {},
   "outputs": [],
   "source": [
    "# save kept records\n",
    "with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'w') as output_handle:\n",
    "    SeqIO.write(kept_records, output_handle, \"fasta\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4c40e93f",
   "metadata": {},
   "source": [
    "## 4.2 BLAST screening of probes\n",
    "\n",
    "create the local blast database by:\n",
    "```bash\n",
    "makeblastdb -in C:\\Users\\puzheng\\Documents\\Tools\\lowercase_masked\\Mus_musculus.GRCm38.dna_sm.merged.fa -dbtype nucl -parse_seqids -out GRCm38_ensembl -title \"Mouse chromosomes soft masked\" -blastdb_version 4![image.png](attachment:image.png)\n",
    "```\n",
    "\n",
    "The outputs are:\n",
    "```bash\n",
    "Building a new DB, current time: 03/30/2021 21:49:18\n",
    "New DB name:   C:\\Users\\puzheng\\Documents\\Tools\\GRCm38_ensembl\n",
    "New DB title:  Mouse chromosomes soft masked\n",
    "Sequence type: Nucleotide\n",
    "Keep MBits: T\n",
    "Maximum file size: 1000000000B\n",
    "Adding sequences from FASTA; added 66 sequences in 42.2668 seconds.\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "id": "5045ff8f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<module 'ImageAnalysis3.library_tools.quality_check' from 'E:\\\\Users\\\\puzheng_new\\\\Documents\\\\Chromatin_Analysis_Scripts\\\\Library_design\\\\CTP-14-human_brain_genome\\\\..\\\\..\\\\..\\\\..\\\\Documents\\\\ImageAnalysis3\\\\library_tools\\\\quality_check.py'>"
      ]
     },
     "execution_count": 68,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "id": "bef107d6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "34067\n"
     ]
    }
   ],
   "source": [
    "if 'kept_records' not in locals() or len(kept_records) == 0:\n",
    "    kept_records = []\n",
    "    with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'r') as handle:\n",
    "        for record in SeqIO.parse(handle, \"fasta\"):\n",
    "            kept_records.append(record)\n",
    "print(len(kept_records))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "id": "75e15498",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Blast probe against \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\blast_database\\hg38\n",
      "-- number of probes loaded: 34067\n",
      "-- processing region: NRNX1-seg-1 with 36 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-1.fasta\n",
      "--- *blasting region: NRNX1-seg-1\n",
      "--- total time for blast NRNX1-seg-1: 4.198s.\n",
      "-- processing region: NRNX1-seg-10 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-10.fasta\n",
      "--- *blasting region: NRNX1-seg-10\n",
      "--- total time for blast NRNX1-seg-10: 5.124s.\n",
      "-- processing region: NRNX1-seg-100 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-100.fasta\n",
      "--- *blasting region: NRNX1-seg-100\n",
      "--- total time for blast NRNX1-seg-100: 4.354s.\n",
      "-- processing region: NRNX1-seg-101 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-101.fasta\n",
      "--- *blasting region: NRNX1-seg-101\n",
      "--- total time for blast NRNX1-seg-101: 3.562s.\n",
      "-- processing region: NRNX1-seg-102 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-102.fasta\n",
      "--- *blasting region: NRNX1-seg-102\n",
      "--- total time for blast NRNX1-seg-102: 3.459s.\n",
      "-- processing region: NRNX1-seg-103 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-103.fasta\n",
      "--- *blasting region: NRNX1-seg-103\n",
      "--- total time for blast NRNX1-seg-103: 3.499s.\n",
      "-- processing region: NRNX1-seg-104 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-104.fasta\n",
      "--- *blasting region: NRNX1-seg-104\n",
      "--- total time for blast NRNX1-seg-104: 4.255s.\n",
      "-- processing region: NRNX1-seg-105 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-105.fasta\n",
      "--- *blasting region: NRNX1-seg-105\n",
      "--- total time for blast NRNX1-seg-105: 2.657s.\n",
      "-- processing region: NRNX1-seg-106 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-106.fasta\n",
      "--- *blasting region: NRNX1-seg-106\n",
      "--- total time for blast NRNX1-seg-106: 4.820s.\n",
      "-- processing region: NRNX1-seg-107 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-107.fasta\n",
      "--- *blasting region: NRNX1-seg-107\n",
      "--- total time for blast NRNX1-seg-107: 3.346s.\n",
      "-- processing region: NRNX1-seg-108 with 32 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-108.fasta\n",
      "--- *blasting region: NRNX1-seg-108\n",
      "--- total time for blast NRNX1-seg-108: 2.415s.\n",
      "-- processing region: NRNX1-seg-109 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-109.fasta\n",
      "--- *blasting region: NRNX1-seg-109\n",
      "--- total time for blast NRNX1-seg-109: 3.388s.\n",
      "-- processing region: NRNX1-seg-110 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-110.fasta\n",
      "--- *blasting region: NRNX1-seg-110\n",
      "--- total time for blast NRNX1-seg-110: 4.306s.\n",
      "-- processing region: NRNX1-seg-111 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-111.fasta\n",
      "--- *blasting region: NRNX1-seg-111\n",
      "--- total time for blast NRNX1-seg-111: 3.319s.\n",
      "-- processing region: NRNX1-seg-112 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-112.fasta\n",
      "--- *blasting region: NRNX1-seg-112\n",
      "--- total time for blast NRNX1-seg-112: 2.285s.\n",
      "-- processing region: NRNX1-seg-113 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-113.fasta\n",
      "--- *blasting region: NRNX1-seg-113\n",
      "--- total time for blast NRNX1-seg-113: 3.997s.\n",
      "-- processing region: NRNX1-seg-114 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-114.fasta\n",
      "--- *blasting region: NRNX1-seg-114\n",
      "--- total time for blast NRNX1-seg-114: 4.468s.\n",
      "-- processing region: NRNX1-seg-115 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-115.fasta\n",
      "--- *blasting region: NRNX1-seg-115\n",
      "--- total time for blast NRNX1-seg-115: 3.668s.\n",
      "-- processing region: NRNX1-seg-117 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-117.fasta\n",
      "--- *blasting region: NRNX1-seg-117\n",
      "--- total time for blast NRNX1-seg-117: 4.576s.\n",
      "-- processing region: NRNX1-seg-118 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-118.fasta\n",
      "--- *blasting region: NRNX1-seg-118\n",
      "--- total time for blast NRNX1-seg-118: 3.653s.\n",
      "-- processing region: NRNX1-seg-119 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-119.fasta\n",
      "--- *blasting region: NRNX1-seg-119\n",
      "--- total time for blast NRNX1-seg-119: 13.723s.\n",
      "-- processing region: NRNX1-seg-120 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-120.fasta\n",
      "--- *blasting region: NRNX1-seg-120\n",
      "--- total time for blast NRNX1-seg-120: 3.624s.\n",
      "-- processing region: NRNX1-seg-121 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-121.fasta\n",
      "--- *blasting region: NRNX1-seg-121\n",
      "--- total time for blast NRNX1-seg-121: 5.506s.\n",
      "-- processing region: NRNX1-seg-122 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-122.fasta\n",
      "--- *blasting region: NRNX1-seg-122\n",
      "--- total time for blast NRNX1-seg-122: 4.297s.\n",
      "-- processing region: NRNX1-seg-123 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-123.fasta\n",
      "--- *blasting region: NRNX1-seg-123\n",
      "--- total time for blast NRNX1-seg-123: 4.138s.\n",
      "-- processing region: NRNX1-seg-124 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-124.fasta\n",
      "--- *blasting region: NRNX1-seg-124\n",
      "--- total time for blast NRNX1-seg-124: 3.945s.\n",
      "-- processing region: NRNX1-seg-125 with 36 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-125.fasta\n",
      "--- *blasting region: NRNX1-seg-125\n",
      "--- total time for blast NRNX1-seg-125: 2.940s.\n",
      "-- processing region: NRNX1-seg-126 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-126.fasta\n",
      "--- *blasting region: NRNX1-seg-126\n",
      "--- total time for blast NRNX1-seg-126: 7.602s.\n",
      "-- processing region: NRNX1-seg-127 with 38 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-127.fasta\n",
      "--- *blasting region: NRNX1-seg-127\n",
      "--- total time for blast NRNX1-seg-127: 10.098s.\n",
      "-- processing region: NRNX1-seg-128 with 36 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-128.fasta\n",
      "--- *blasting region: NRNX1-seg-128\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-128: 4.503s.\n",
      "-- processing region: NRNX1-seg-129 with 30 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-129.fasta\n",
      "--- *blasting region: NRNX1-seg-129\n",
      "--- total time for blast NRNX1-seg-129: 2.776s.\n",
      "-- processing region: NRNX1-seg-13 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-13.fasta\n",
      "--- *blasting region: NRNX1-seg-13\n",
      "--- total time for blast NRNX1-seg-13: 4.400s.\n",
      "-- processing region: NRNX1-seg-130 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-130.fasta\n",
      "--- *blasting region: NRNX1-seg-130\n",
      "--- total time for blast NRNX1-seg-130: 3.228s.\n",
      "-- processing region: NRNX1-seg-131 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-131.fasta\n",
      "--- *blasting region: NRNX1-seg-131\n",
      "--- total time for blast NRNX1-seg-131: 2.878s.\n",
      "-- processing region: NRNX1-seg-133 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-133.fasta\n",
      "--- *blasting region: NRNX1-seg-133\n",
      "--- total time for blast NRNX1-seg-133: 2.573s.\n",
      "-- processing region: NRNX1-seg-134 with 28 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-134.fasta\n",
      "--- *blasting region: NRNX1-seg-134\n",
      "--- total time for blast NRNX1-seg-134: 2.239s.\n",
      "-- processing region: NRNX1-seg-135 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-135.fasta\n",
      "--- *blasting region: NRNX1-seg-135\n",
      "--- total time for blast NRNX1-seg-135: 3.511s.\n",
      "-- processing region: NRNX1-seg-136 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-136.fasta\n",
      "--- *blasting region: NRNX1-seg-136\n",
      "--- total time for blast NRNX1-seg-136: 3.585s.\n",
      "-- processing region: NRNX1-seg-137 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-137.fasta\n",
      "--- *blasting region: NRNX1-seg-137\n",
      "--- total time for blast NRNX1-seg-137: 3.562s.\n",
      "-- processing region: NRNX1-seg-138 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-138.fasta\n",
      "--- *blasting region: NRNX1-seg-138\n",
      "--- total time for blast NRNX1-seg-138: 4.056s.\n",
      "-- processing region: NRNX1-seg-139 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-139.fasta\n",
      "--- *blasting region: NRNX1-seg-139\n",
      "--- total time for blast NRNX1-seg-139: 4.216s.\n",
      "-- processing region: NRNX1-seg-14 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-14.fasta\n",
      "--- *blasting region: NRNX1-seg-14\n",
      "--- total time for blast NRNX1-seg-14: 4.607s.\n",
      "-- processing region: NRNX1-seg-140 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-140.fasta\n",
      "--- *blasting region: NRNX1-seg-140\n",
      "--- total time for blast NRNX1-seg-140: 10.591s.\n",
      "-- processing region: NRNX1-seg-141 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-141.fasta\n",
      "--- *blasting region: NRNX1-seg-141\n",
      "--- total time for blast NRNX1-seg-141: 5.891s.\n",
      "-- processing region: NRNX1-seg-142 with 31 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-142.fasta\n",
      "--- *blasting region: NRNX1-seg-142\n",
      "--- total time for blast NRNX1-seg-142: 3.945s.\n",
      "-- processing region: NRNX1-seg-143 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-143.fasta\n",
      "--- *blasting region: NRNX1-seg-143\n",
      "--- total time for blast NRNX1-seg-143: 5.026s.\n",
      "-- processing region: NRNX1-seg-144 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-144.fasta\n",
      "--- *blasting region: NRNX1-seg-144\n",
      "--- total time for blast NRNX1-seg-144: 4.247s.\n",
      "-- processing region: NRNX1-seg-145 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-145.fasta\n",
      "--- *blasting region: NRNX1-seg-145\n",
      "--- total time for blast NRNX1-seg-145: 4.447s.\n",
      "-- processing region: NRNX1-seg-146 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-146.fasta\n",
      "--- *blasting region: NRNX1-seg-146\n",
      "--- total time for blast NRNX1-seg-146: 4.044s.\n",
      "-- processing region: NRNX1-seg-147 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-147.fasta\n",
      "--- *blasting region: NRNX1-seg-147\n",
      "--- total time for blast NRNX1-seg-147: 4.781s.\n",
      "-- processing region: NRNX1-seg-148 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-148.fasta\n",
      "--- *blasting region: NRNX1-seg-148\n",
      "--- total time for blast NRNX1-seg-148: 3.639s.\n",
      "-- processing region: NRNX1-seg-15 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-15.fasta\n",
      "--- *blasting region: NRNX1-seg-15\n",
      "--- total time for blast NRNX1-seg-15: 4.290s.\n",
      "-- processing region: NRNX1-seg-150 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-150.fasta\n",
      "--- *blasting region: NRNX1-seg-150\n",
      "--- total time for blast NRNX1-seg-150: 3.639s.\n",
      "-- processing region: NRNX1-seg-151 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-151.fasta\n",
      "--- *blasting region: NRNX1-seg-151\n",
      "--- total time for blast NRNX1-seg-151: 4.262s.\n",
      "-- processing region: NRNX1-seg-152 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-152.fasta\n",
      "--- *blasting region: NRNX1-seg-152\n",
      "--- total time for blast NRNX1-seg-152: 3.990s.\n",
      "-- processing region: NRNX1-seg-154 with 35 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-154.fasta\n",
      "--- *blasting region: NRNX1-seg-154\n",
      "--- total time for blast NRNX1-seg-154: 4.087s.\n",
      "-- processing region: NRNX1-seg-155 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-155.fasta\n",
      "--- *blasting region: NRNX1-seg-155\n",
      "--- total time for blast NRNX1-seg-155: 3.919s.\n",
      "-- processing region: NRNX1-seg-156 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-156.fasta\n",
      "--- *blasting region: NRNX1-seg-156\n",
      "--- total time for blast NRNX1-seg-156: 4.278s.\n",
      "-- processing region: NRNX1-seg-157 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-157.fasta\n",
      "--- *blasting region: NRNX1-seg-157\n",
      "--- total time for blast NRNX1-seg-157: 5.368s.\n",
      "-- processing region: NRNX1-seg-158 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-158.fasta\n",
      "--- *blasting region: NRNX1-seg-158\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-158: 4.853s.\n",
      "-- processing region: NRNX1-seg-159 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-159.fasta\n",
      "--- *blasting region: NRNX1-seg-159\n",
      "--- total time for blast NRNX1-seg-159: 4.523s.\n",
      "-- processing region: NRNX1-seg-16 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-16.fasta\n",
      "--- *blasting region: NRNX1-seg-16\n",
      "--- total time for blast NRNX1-seg-16: 4.236s.\n",
      "-- processing region: NRNX1-seg-160 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-160.fasta\n",
      "--- *blasting region: NRNX1-seg-160\n",
      "--- total time for blast NRNX1-seg-160: 5.216s.\n",
      "-- processing region: NRNX1-seg-161 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-161.fasta\n",
      "--- *blasting region: NRNX1-seg-161\n",
      "--- total time for blast NRNX1-seg-161: 4.871s.\n",
      "-- processing region: NRNX1-seg-162 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-162.fasta\n",
      "--- *blasting region: NRNX1-seg-162\n",
      "--- total time for blast NRNX1-seg-162: 4.608s.\n",
      "-- processing region: NRNX1-seg-163 with 34 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-163.fasta\n",
      "--- *blasting region: NRNX1-seg-163\n",
      "--- total time for blast NRNX1-seg-163: 4.469s.\n",
      "-- processing region: NRNX1-seg-164 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-164.fasta\n",
      "--- *blasting region: NRNX1-seg-164\n",
      "--- total time for blast NRNX1-seg-164: 4.505s.\n",
      "-- processing region: NRNX1-seg-165 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-165.fasta\n",
      "--- *blasting region: NRNX1-seg-165\n",
      "--- total time for blast NRNX1-seg-165: 5.205s.\n",
      "-- processing region: NRNX1-seg-166 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-166.fasta\n",
      "--- *blasting region: NRNX1-seg-166\n",
      "--- total time for blast NRNX1-seg-166: 5.262s.\n",
      "-- processing region: NRNX1-seg-169 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-169.fasta\n",
      "--- *blasting region: NRNX1-seg-169\n",
      "--- total time for blast NRNX1-seg-169: 4.976s.\n",
      "-- processing region: NRNX1-seg-17 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-17.fasta\n",
      "--- *blasting region: NRNX1-seg-17\n",
      "--- total time for blast NRNX1-seg-17: 4.112s.\n",
      "-- processing region: NRNX1-seg-170 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-170.fasta\n",
      "--- *blasting region: NRNX1-seg-170\n",
      "--- total time for blast NRNX1-seg-170: 4.400s.\n",
      "-- processing region: NRNX1-seg-171 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-171.fasta\n",
      "--- *blasting region: NRNX1-seg-171\n",
      "--- total time for blast NRNX1-seg-171: 4.677s.\n",
      "-- processing region: NRNX1-seg-172 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-172.fasta\n",
      "--- *blasting region: NRNX1-seg-172\n",
      "--- total time for blast NRNX1-seg-172: 5.141s.\n",
      "-- processing region: NRNX1-seg-173 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-173.fasta\n",
      "--- *blasting region: NRNX1-seg-173\n",
      "--- total time for blast NRNX1-seg-173: 4.402s.\n",
      "-- processing region: NRNX1-seg-174 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-174.fasta\n",
      "--- *blasting region: NRNX1-seg-174\n",
      "--- total time for blast NRNX1-seg-174: 5.779s.\n",
      "-- processing region: NRNX1-seg-175 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-175.fasta\n",
      "--- *blasting region: NRNX1-seg-175\n",
      "--- total time for blast NRNX1-seg-175: 4.696s.\n",
      "-- processing region: NRNX1-seg-176 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-176.fasta\n",
      "--- *blasting region: NRNX1-seg-176\n",
      "--- total time for blast NRNX1-seg-176: 4.442s.\n",
      "-- processing region: NRNX1-seg-177 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-177.fasta\n",
      "--- *blasting region: NRNX1-seg-177\n",
      "--- total time for blast NRNX1-seg-177: 4.023s.\n",
      "-- processing region: NRNX1-seg-178 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-178.fasta\n",
      "--- *blasting region: NRNX1-seg-178\n",
      "--- total time for blast NRNX1-seg-178: 4.802s.\n",
      "-- processing region: NRNX1-seg-179 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-179.fasta\n",
      "--- *blasting region: NRNX1-seg-179\n",
      "--- total time for blast NRNX1-seg-179: 4.564s.\n",
      "-- processing region: NRNX1-seg-18 with 36 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-18.fasta\n",
      "--- *blasting region: NRNX1-seg-18\n",
      "--- total time for blast NRNX1-seg-18: 4.984s.\n",
      "-- processing region: NRNX1-seg-180 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-180.fasta\n",
      "--- *blasting region: NRNX1-seg-180\n",
      "--- total time for blast NRNX1-seg-180: 4.431s.\n",
      "-- processing region: NRNX1-seg-181 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-181.fasta\n",
      "--- *blasting region: NRNX1-seg-181\n",
      "--- total time for blast NRNX1-seg-181: 4.754s.\n",
      "-- processing region: NRNX1-seg-182 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-182.fasta\n",
      "--- *blasting region: NRNX1-seg-182\n",
      "--- total time for blast NRNX1-seg-182: 5.130s.\n",
      "-- processing region: NRNX1-seg-183 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-183.fasta\n",
      "--- *blasting region: NRNX1-seg-183\n",
      "--- total time for blast NRNX1-seg-183: 3.750s.\n",
      "-- processing region: NRNX1-seg-184 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-184.fasta\n",
      "--- *blasting region: NRNX1-seg-184\n",
      "--- total time for blast NRNX1-seg-184: 3.266s.\n",
      "-- processing region: NRNX1-seg-185 with 43 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-185.fasta\n",
      "--- *blasting region: NRNX1-seg-185\n",
      "--- total time for blast NRNX1-seg-185: 3.999s.\n",
      "-- processing region: NRNX1-seg-186 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-186.fasta\n",
      "--- *blasting region: NRNX1-seg-186\n",
      "--- total time for blast NRNX1-seg-186: 3.580s.\n",
      "-- processing region: NRNX1-seg-187 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-187.fasta\n",
      "--- *blasting region: NRNX1-seg-187\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-187: 4.743s.\n",
      "-- processing region: NRNX1-seg-188 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-188.fasta\n",
      "--- *blasting region: NRNX1-seg-188\n",
      "--- total time for blast NRNX1-seg-188: 3.971s.\n",
      "-- processing region: NRNX1-seg-189 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-189.fasta\n",
      "--- *blasting region: NRNX1-seg-189\n",
      "--- total time for blast NRNX1-seg-189: 3.611s.\n",
      "-- processing region: NRNX1-seg-19 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-19.fasta\n",
      "--- *blasting region: NRNX1-seg-19\n",
      "--- total time for blast NRNX1-seg-19: 4.646s.\n",
      "-- processing region: NRNX1-seg-190 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-190.fasta\n",
      "--- *blasting region: NRNX1-seg-190\n",
      "--- total time for blast NRNX1-seg-190: 4.553s.\n",
      "-- processing region: NRNX1-seg-191 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-191.fasta\n",
      "--- *blasting region: NRNX1-seg-191\n",
      "--- total time for blast NRNX1-seg-191: 4.300s.\n",
      "-- processing region: NRNX1-seg-192 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-192.fasta\n",
      "--- *blasting region: NRNX1-seg-192\n",
      "--- total time for blast NRNX1-seg-192: 4.942s.\n",
      "-- processing region: NRNX1-seg-193 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-193.fasta\n",
      "--- *blasting region: NRNX1-seg-193\n",
      "--- total time for blast NRNX1-seg-193: 4.279s.\n",
      "-- processing region: NRNX1-seg-194 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-194.fasta\n",
      "--- *blasting region: NRNX1-seg-194\n",
      "--- total time for blast NRNX1-seg-194: 5.221s.\n",
      "-- processing region: NRNX1-seg-195 with 72 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-195.fasta\n",
      "--- *blasting region: NRNX1-seg-195\n",
      "--- total time for blast NRNX1-seg-195: 4.640s.\n",
      "-- processing region: NRNX1-seg-196 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-196.fasta\n",
      "--- *blasting region: NRNX1-seg-196\n",
      "--- total time for blast NRNX1-seg-196: 5.052s.\n",
      "-- processing region: NRNX1-seg-197 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-197.fasta\n",
      "--- *blasting region: NRNX1-seg-197\n",
      "--- total time for blast NRNX1-seg-197: 4.786s.\n",
      "-- processing region: NRNX1-seg-198 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-198.fasta\n",
      "--- *blasting region: NRNX1-seg-198\n",
      "--- total time for blast NRNX1-seg-198: 3.594s.\n",
      "-- processing region: NRNX1-seg-199 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-199.fasta\n",
      "--- *blasting region: NRNX1-seg-199\n",
      "--- total time for blast NRNX1-seg-199: 4.455s.\n",
      "-- processing region: NRNX1-seg-2 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-2.fasta\n",
      "--- *blasting region: NRNX1-seg-2\n",
      "--- total time for blast NRNX1-seg-2: 3.529s.\n",
      "-- processing region: NRNX1-seg-20 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-20.fasta\n",
      "--- *blasting region: NRNX1-seg-20\n",
      "--- total time for blast NRNX1-seg-20: 3.308s.\n",
      "-- processing region: NRNX1-seg-200 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-200.fasta\n",
      "--- *blasting region: NRNX1-seg-200\n",
      "--- total time for blast NRNX1-seg-200: 4.728s.\n",
      "-- processing region: NRNX1-seg-201 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-201.fasta\n",
      "--- *blasting region: NRNX1-seg-201\n",
      "--- total time for blast NRNX1-seg-201: 7.159s.\n",
      "-- processing region: NRNX1-seg-202 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-202.fasta\n",
      "--- *blasting region: NRNX1-seg-202\n",
      "--- total time for blast NRNX1-seg-202: 3.126s.\n",
      "-- processing region: NRNX1-seg-203 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-203.fasta\n",
      "--- *blasting region: NRNX1-seg-203\n",
      "--- total time for blast NRNX1-seg-203: 3.663s.\n",
      "-- processing region: NRNX1-seg-204 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-204.fasta\n",
      "--- *blasting region: NRNX1-seg-204\n",
      "--- total time for blast NRNX1-seg-204: 5.250s.\n",
      "-- processing region: NRNX1-seg-205 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-205.fasta\n",
      "--- *blasting region: NRNX1-seg-205\n",
      "--- total time for blast NRNX1-seg-205: 3.854s.\n",
      "-- processing region: NRNX1-seg-206 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-206.fasta\n",
      "--- *blasting region: NRNX1-seg-206\n",
      "--- total time for blast NRNX1-seg-206: 4.460s.\n",
      "-- processing region: NRNX1-seg-207 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-207.fasta\n",
      "--- *blasting region: NRNX1-seg-207\n",
      "--- total time for blast NRNX1-seg-207: 4.110s.\n",
      "-- processing region: NRNX1-seg-208 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-208.fasta\n",
      "--- *blasting region: NRNX1-seg-208\n",
      "--- total time for blast NRNX1-seg-208: 3.777s.\n",
      "-- processing region: NRNX1-seg-209 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-209.fasta\n",
      "--- *blasting region: NRNX1-seg-209\n",
      "--- total time for blast NRNX1-seg-209: 5.169s.\n",
      "-- processing region: NRNX1-seg-21 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-21.fasta\n",
      "--- *blasting region: NRNX1-seg-21\n",
      "--- total time for blast NRNX1-seg-21: 4.433s.\n",
      "-- processing region: NRNX1-seg-210 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-210.fasta\n",
      "--- *blasting region: NRNX1-seg-210\n",
      "--- total time for blast NRNX1-seg-210: 4.860s.\n",
      "-- processing region: NRNX1-seg-211 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-211.fasta\n",
      "--- *blasting region: NRNX1-seg-211\n",
      "--- total time for blast NRNX1-seg-211: 4.559s.\n",
      "-- processing region: NRNX1-seg-212 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-212.fasta\n",
      "--- *blasting region: NRNX1-seg-212\n",
      "--- total time for blast NRNX1-seg-212: 6.393s.\n",
      "-- processing region: NRNX1-seg-213 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-213.fasta\n",
      "--- *blasting region: NRNX1-seg-213\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-213: 10.822s.\n",
      "-- processing region: NRNX1-seg-214 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-214.fasta\n",
      "--- *blasting region: NRNX1-seg-214\n",
      "--- total time for blast NRNX1-seg-214: 5.849s.\n",
      "-- processing region: NRNX1-seg-215 with 30 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-215.fasta\n",
      "--- *blasting region: NRNX1-seg-215\n",
      "--- total time for blast NRNX1-seg-215: 4.832s.\n",
      "-- processing region: NRNX1-seg-216 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-216.fasta\n",
      "--- *blasting region: NRNX1-seg-216\n",
      "--- total time for blast NRNX1-seg-216: 5.235s.\n",
      "-- processing region: NRNX1-seg-217 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-217.fasta\n",
      "--- *blasting region: NRNX1-seg-217\n",
      "--- total time for blast NRNX1-seg-217: 2.284s.\n",
      "-- processing region: NRNX1-seg-218 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-218.fasta\n",
      "--- *blasting region: NRNX1-seg-218\n",
      "--- total time for blast NRNX1-seg-218: 5.914s.\n",
      "-- processing region: NRNX1-seg-219 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-219.fasta\n",
      "--- *blasting region: NRNX1-seg-219\n",
      "--- total time for blast NRNX1-seg-219: 3.190s.\n",
      "-- processing region: NRNX1-seg-22 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-22.fasta\n",
      "--- *blasting region: NRNX1-seg-22\n",
      "--- total time for blast NRNX1-seg-22: 5.012s.\n",
      "-- processing region: NRNX1-seg-220 with 35 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-220.fasta\n",
      "--- *blasting region: NRNX1-seg-220\n",
      "--- total time for blast NRNX1-seg-220: 4.476s.\n",
      "-- processing region: NRNX1-seg-221 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-221.fasta\n",
      "--- *blasting region: NRNX1-seg-221\n",
      "--- total time for blast NRNX1-seg-221: 4.538s.\n",
      "-- processing region: NRNX1-seg-222 with 71 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-222.fasta\n",
      "--- *blasting region: NRNX1-seg-222\n",
      "--- total time for blast NRNX1-seg-222: 4.123s.\n",
      "-- processing region: NRNX1-seg-223 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-223.fasta\n",
      "--- *blasting region: NRNX1-seg-223\n",
      "--- total time for blast NRNX1-seg-223: 4.744s.\n",
      "-- processing region: NRNX1-seg-224 with 74 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-224.fasta\n",
      "--- *blasting region: NRNX1-seg-224\n",
      "--- total time for blast NRNX1-seg-224: 4.508s.\n",
      "-- processing region: NRNX1-seg-225 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-225.fasta\n",
      "--- *blasting region: NRNX1-seg-225\n",
      "--- total time for blast NRNX1-seg-225: 4.109s.\n",
      "-- processing region: NRNX1-seg-226 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-226.fasta\n",
      "--- *blasting region: NRNX1-seg-226\n",
      "--- total time for blast NRNX1-seg-226: 4.127s.\n",
      "-- processing region: NRNX1-seg-227 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-227.fasta\n",
      "--- *blasting region: NRNX1-seg-227\n",
      "--- total time for blast NRNX1-seg-227: 3.080s.\n",
      "-- processing region: NRNX1-seg-228 with 29 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-228.fasta\n",
      "--- *blasting region: NRNX1-seg-228\n",
      "--- total time for blast NRNX1-seg-228: 2.914s.\n",
      "-- processing region: NRNX1-seg-229 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-229.fasta\n",
      "--- *blasting region: NRNX1-seg-229\n",
      "--- total time for blast NRNX1-seg-229: 4.453s.\n",
      "-- processing region: NRNX1-seg-23 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-23.fasta\n",
      "--- *blasting region: NRNX1-seg-23\n",
      "--- total time for blast NRNX1-seg-23: 4.170s.\n",
      "-- processing region: NRNX1-seg-230 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-230.fasta\n",
      "--- *blasting region: NRNX1-seg-230\n",
      "--- total time for blast NRNX1-seg-230: 4.009s.\n",
      "-- processing region: NRNX1-seg-231 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-231.fasta\n",
      "--- *blasting region: NRNX1-seg-231\n",
      "--- total time for blast NRNX1-seg-231: 4.706s.\n",
      "-- processing region: NRNX1-seg-232 with 29 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-232.fasta\n",
      "--- *blasting region: NRNX1-seg-232\n",
      "--- total time for blast NRNX1-seg-232: 2.405s.\n",
      "-- processing region: NRNX1-seg-233 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-233.fasta\n",
      "--- *blasting region: NRNX1-seg-233\n",
      "--- total time for blast NRNX1-seg-233: 3.807s.\n",
      "-- processing region: NRNX1-seg-234 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-234.fasta\n",
      "--- *blasting region: NRNX1-seg-234\n",
      "--- total time for blast NRNX1-seg-234: 4.513s.\n",
      "-- processing region: NRNX1-seg-235 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-235.fasta\n",
      "--- *blasting region: NRNX1-seg-235\n",
      "--- total time for blast NRNX1-seg-235: 5.176s.\n",
      "-- processing region: NRNX1-seg-236 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-236.fasta\n",
      "--- *blasting region: NRNX1-seg-236\n",
      "--- total time for blast NRNX1-seg-236: 3.262s.\n",
      "-- processing region: NRNX1-seg-238 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-238.fasta\n",
      "--- *blasting region: NRNX1-seg-238\n",
      "--- total time for blast NRNX1-seg-238: 5.805s.\n",
      "-- processing region: NRNX1-seg-239 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-239.fasta\n",
      "--- *blasting region: NRNX1-seg-239\n",
      "--- total time for blast NRNX1-seg-239: 4.609s.\n",
      "-- processing region: NRNX1-seg-24 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-24.fasta\n",
      "--- *blasting region: NRNX1-seg-24\n",
      "--- total time for blast NRNX1-seg-24: 4.656s.\n",
      "-- processing region: NRNX1-seg-240 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-240.fasta\n",
      "--- *blasting region: NRNX1-seg-240\n",
      "--- total time for blast NRNX1-seg-240: 4.424s.\n",
      "-- processing region: NRNX1-seg-241 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-241.fasta\n",
      "--- *blasting region: NRNX1-seg-241\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-241: 3.398s.\n",
      "-- processing region: NRNX1-seg-242 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-242.fasta\n",
      "--- *blasting region: NRNX1-seg-242\n",
      "--- total time for blast NRNX1-seg-242: 4.661s.\n",
      "-- processing region: NRNX1-seg-244 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-244.fasta\n",
      "--- *blasting region: NRNX1-seg-244\n",
      "--- total time for blast NRNX1-seg-244: 4.257s.\n",
      "-- processing region: NRNX1-seg-245 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-245.fasta\n",
      "--- *blasting region: NRNX1-seg-245\n",
      "--- total time for blast NRNX1-seg-245: 3.814s.\n",
      "-- processing region: NRNX1-seg-246 with 38 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-246.fasta\n",
      "--- *blasting region: NRNX1-seg-246\n",
      "--- total time for blast NRNX1-seg-246: 4.142s.\n",
      "-- processing region: NRNX1-seg-248 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-248.fasta\n",
      "--- *blasting region: NRNX1-seg-248\n",
      "--- total time for blast NRNX1-seg-248: 4.746s.\n",
      "-- processing region: NRNX1-seg-249 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-249.fasta\n",
      "--- *blasting region: NRNX1-seg-249\n",
      "--- total time for blast NRNX1-seg-249: 7.338s.\n",
      "-- processing region: NRNX1-seg-25 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-25.fasta\n",
      "--- *blasting region: NRNX1-seg-25\n",
      "--- total time for blast NRNX1-seg-25: 4.210s.\n",
      "-- processing region: NRNX1-seg-251 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-251.fasta\n",
      "--- *blasting region: NRNX1-seg-251\n",
      "--- total time for blast NRNX1-seg-251: 4.358s.\n",
      "-- processing region: NRNX1-seg-252 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-252.fasta\n",
      "--- *blasting region: NRNX1-seg-252\n",
      "--- total time for blast NRNX1-seg-252: 3.912s.\n",
      "-- processing region: NRNX1-seg-253 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-253.fasta\n",
      "--- *blasting region: NRNX1-seg-253\n",
      "--- total time for blast NRNX1-seg-253: 4.190s.\n",
      "-- processing region: NRNX1-seg-254 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-254.fasta\n",
      "--- *blasting region: NRNX1-seg-254\n",
      "--- total time for blast NRNX1-seg-254: 3.537s.\n",
      "-- processing region: NRNX1-seg-255 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-255.fasta\n",
      "--- *blasting region: NRNX1-seg-255\n",
      "--- total time for blast NRNX1-seg-255: 4.661s.\n",
      "-- processing region: NRNX1-seg-256 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-256.fasta\n",
      "--- *blasting region: NRNX1-seg-256\n",
      "--- total time for blast NRNX1-seg-256: 4.123s.\n",
      "-- processing region: NRNX1-seg-257 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-257.fasta\n",
      "--- *blasting region: NRNX1-seg-257\n",
      "--- total time for blast NRNX1-seg-257: 4.222s.\n",
      "-- processing region: NRNX1-seg-258 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-258.fasta\n",
      "--- *blasting region: NRNX1-seg-258\n",
      "--- total time for blast NRNX1-seg-258: 3.792s.\n",
      "-- processing region: NRNX1-seg-259 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-259.fasta\n",
      "--- *blasting region: NRNX1-seg-259\n",
      "--- total time for blast NRNX1-seg-259: 4.259s.\n",
      "-- processing region: NRNX1-seg-26 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-26.fasta\n",
      "--- *blasting region: NRNX1-seg-26\n",
      "--- total time for blast NRNX1-seg-26: 3.663s.\n",
      "-- processing region: NRNX1-seg-260 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-260.fasta\n",
      "--- *blasting region: NRNX1-seg-260\n",
      "--- total time for blast NRNX1-seg-260: 5.755s.\n",
      "-- processing region: NRNX1-seg-262 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-262.fasta\n",
      "--- *blasting region: NRNX1-seg-262\n",
      "--- total time for blast NRNX1-seg-262: 3.181s.\n",
      "-- processing region: NRNX1-seg-263 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-263.fasta\n",
      "--- *blasting region: NRNX1-seg-263\n",
      "--- total time for blast NRNX1-seg-263: 2.679s.\n",
      "-- processing region: NRNX1-seg-264 with 35 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-264.fasta\n",
      "--- *blasting region: NRNX1-seg-264\n",
      "--- total time for blast NRNX1-seg-264: 2.561s.\n",
      "-- processing region: NRNX1-seg-265 with 32 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-265.fasta\n",
      "--- *blasting region: NRNX1-seg-265\n",
      "--- total time for blast NRNX1-seg-265: 5.292s.\n",
      "-- processing region: NRNX1-seg-266 with 33 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-266.fasta\n",
      "--- *blasting region: NRNX1-seg-266\n",
      "--- total time for blast NRNX1-seg-266: 3.702s.\n",
      "-- processing region: NRNX1-seg-267 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-267.fasta\n",
      "--- *blasting region: NRNX1-seg-267\n",
      "--- total time for blast NRNX1-seg-267: 4.240s.\n",
      "-- processing region: NRNX1-seg-268 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-268.fasta\n",
      "--- *blasting region: NRNX1-seg-268\n",
      "--- total time for blast NRNX1-seg-268: 4.568s.\n",
      "-- processing region: NRNX1-seg-269 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-269.fasta\n",
      "--- *blasting region: NRNX1-seg-269\n",
      "--- total time for blast NRNX1-seg-269: 4.861s.\n",
      "-- processing region: NRNX1-seg-27 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-27.fasta\n",
      "--- *blasting region: NRNX1-seg-27\n",
      "--- total time for blast NRNX1-seg-27: 3.681s.\n",
      "-- processing region: NRNX1-seg-270 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-270.fasta\n",
      "--- *blasting region: NRNX1-seg-270\n",
      "--- total time for blast NRNX1-seg-270: 3.941s.\n",
      "-- processing region: NRNX1-seg-271 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-271.fasta\n",
      "--- *blasting region: NRNX1-seg-271\n",
      "--- total time for blast NRNX1-seg-271: 4.635s.\n",
      "-- processing region: NRNX1-seg-272 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-272.fasta\n",
      "--- *blasting region: NRNX1-seg-272\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-272: 4.316s.\n",
      "-- processing region: NRNX1-seg-273 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-273.fasta\n",
      "--- *blasting region: NRNX1-seg-273\n",
      "--- total time for blast NRNX1-seg-273: 5.532s.\n",
      "-- processing region: NRNX1-seg-274 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-274.fasta\n",
      "--- *blasting region: NRNX1-seg-274\n",
      "--- total time for blast NRNX1-seg-274: 3.964s.\n",
      "-- processing region: NRNX1-seg-275 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-275.fasta\n",
      "--- *blasting region: NRNX1-seg-275\n",
      "--- total time for blast NRNX1-seg-275: 3.813s.\n",
      "-- processing region: NRNX1-seg-276 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-276.fasta\n",
      "--- *blasting region: NRNX1-seg-276\n",
      "--- total time for blast NRNX1-seg-276: 4.106s.\n",
      "-- processing region: NRNX1-seg-277 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-277.fasta\n",
      "--- *blasting region: NRNX1-seg-277\n",
      "--- total time for blast NRNX1-seg-277: 3.534s.\n",
      "-- processing region: NRNX1-seg-278 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-278.fasta\n",
      "--- *blasting region: NRNX1-seg-278\n",
      "--- total time for blast NRNX1-seg-278: 7.645s.\n",
      "-- processing region: NRNX1-seg-279 with 28 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-279.fasta\n",
      "--- *blasting region: NRNX1-seg-279\n",
      "--- total time for blast NRNX1-seg-279: 16.551s.\n",
      "-- processing region: NRNX1-seg-28 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-28.fasta\n",
      "--- *blasting region: NRNX1-seg-28\n",
      "--- total time for blast NRNX1-seg-28: 4.379s.\n",
      "-- processing region: NRNX1-seg-280 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-280.fasta\n",
      "--- *blasting region: NRNX1-seg-280\n",
      "--- total time for blast NRNX1-seg-280: 4.715s.\n",
      "-- processing region: NRNX1-seg-281 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-281.fasta\n",
      "--- *blasting region: NRNX1-seg-281\n",
      "--- total time for blast NRNX1-seg-281: 4.450s.\n",
      "-- processing region: NRNX1-seg-282 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-282.fasta\n",
      "--- *blasting region: NRNX1-seg-282\n",
      "--- total time for blast NRNX1-seg-282: 6.247s.\n",
      "-- processing region: NRNX1-seg-283 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-283.fasta\n",
      "--- *blasting region: NRNX1-seg-283\n",
      "--- total time for blast NRNX1-seg-283: 4.169s.\n",
      "-- processing region: NRNX1-seg-284 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-284.fasta\n",
      "--- *blasting region: NRNX1-seg-284\n",
      "--- total time for blast NRNX1-seg-284: 4.659s.\n",
      "-- processing region: NRNX1-seg-285 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-285.fasta\n",
      "--- *blasting region: NRNX1-seg-285\n",
      "--- total time for blast NRNX1-seg-285: 3.982s.\n",
      "-- processing region: NRNX1-seg-286 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-286.fasta\n",
      "--- *blasting region: NRNX1-seg-286\n",
      "--- total time for blast NRNX1-seg-286: 3.574s.\n",
      "-- processing region: NRNX1-seg-287 with 26 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-287.fasta\n",
      "--- *blasting region: NRNX1-seg-287\n",
      "--- total time for blast NRNX1-seg-287: 4.525s.\n",
      "-- processing region: NRNX1-seg-288 with 29 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-288.fasta\n",
      "--- *blasting region: NRNX1-seg-288\n",
      "--- total time for blast NRNX1-seg-288: 2.214s.\n",
      "-- processing region: NRNX1-seg-289 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-289.fasta\n",
      "--- *blasting region: NRNX1-seg-289\n",
      "--- total time for blast NRNX1-seg-289: 4.893s.\n",
      "-- processing region: NRNX1-seg-290 with 36 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-290.fasta\n",
      "--- *blasting region: NRNX1-seg-290\n",
      "--- total time for blast NRNX1-seg-290: 2.998s.\n",
      "-- processing region: NRNX1-seg-291 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-291.fasta\n",
      "--- *blasting region: NRNX1-seg-291\n",
      "--- total time for blast NRNX1-seg-291: 3.014s.\n",
      "-- processing region: NRNX1-seg-292 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-292.fasta\n",
      "--- *blasting region: NRNX1-seg-292\n",
      "--- total time for blast NRNX1-seg-292: 4.085s.\n",
      "-- processing region: NRNX1-seg-293 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-293.fasta\n",
      "--- *blasting region: NRNX1-seg-293\n",
      "--- total time for blast NRNX1-seg-293: 3.302s.\n",
      "-- processing region: NRNX1-seg-294 with 31 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-294.fasta\n",
      "--- *blasting region: NRNX1-seg-294\n",
      "--- total time for blast NRNX1-seg-294: 3.662s.\n",
      "-- processing region: NRNX1-seg-295 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-295.fasta\n",
      "--- *blasting region: NRNX1-seg-295\n",
      "--- total time for blast NRNX1-seg-295: 3.940s.\n",
      "-- processing region: NRNX1-seg-298 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-298.fasta\n",
      "--- *blasting region: NRNX1-seg-298\n",
      "--- total time for blast NRNX1-seg-298: 3.868s.\n",
      "-- processing region: NRNX1-seg-299 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-299.fasta\n",
      "--- *blasting region: NRNX1-seg-299\n",
      "--- total time for blast NRNX1-seg-299: 4.262s.\n",
      "-- processing region: NRNX1-seg-3 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-3.fasta\n",
      "--- *blasting region: NRNX1-seg-3\n",
      "--- total time for blast NRNX1-seg-3: 3.865s.\n",
      "-- processing region: NRNX1-seg-30 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-30.fasta\n",
      "--- *blasting region: NRNX1-seg-30\n",
      "--- total time for blast NRNX1-seg-30: 3.332s.\n",
      "-- processing region: NRNX1-seg-300 with 36 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-300.fasta\n",
      "--- *blasting region: NRNX1-seg-300\n",
      "--- total time for blast NRNX1-seg-300: 6.138s.\n",
      "-- processing region: NRNX1-seg-301 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-301.fasta\n",
      "--- *blasting region: NRNX1-seg-301\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-301: 3.728s.\n",
      "-- processing region: NRNX1-seg-302 with 31 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-302.fasta\n",
      "--- *blasting region: NRNX1-seg-302\n",
      "--- total time for blast NRNX1-seg-302: 2.410s.\n",
      "-- processing region: NRNX1-seg-303 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-303.fasta\n",
      "--- *blasting region: NRNX1-seg-303\n",
      "--- total time for blast NRNX1-seg-303: 3.574s.\n",
      "-- processing region: NRNX1-seg-304 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-304.fasta\n",
      "--- *blasting region: NRNX1-seg-304\n",
      "--- total time for blast NRNX1-seg-304: 4.112s.\n",
      "-- processing region: NRNX1-seg-305 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-305.fasta\n",
      "--- *blasting region: NRNX1-seg-305\n",
      "--- total time for blast NRNX1-seg-305: 2.782s.\n",
      "-- processing region: NRNX1-seg-306 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-306.fasta\n",
      "--- *blasting region: NRNX1-seg-306\n",
      "--- total time for blast NRNX1-seg-306: 6.918s.\n",
      "-- processing region: NRNX1-seg-307 with 22 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-307.fasta\n",
      "--- *blasting region: NRNX1-seg-307\n",
      "--- total time for blast NRNX1-seg-307: 4.527s.\n",
      "-- processing region: NRNX1-seg-308 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-308.fasta\n",
      "--- *blasting region: NRNX1-seg-308\n",
      "--- total time for blast NRNX1-seg-308: 4.196s.\n",
      "-- processing region: NRNX1-seg-31 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-31.fasta\n",
      "--- *blasting region: NRNX1-seg-31\n",
      "--- total time for blast NRNX1-seg-31: 3.287s.\n",
      "-- processing region: NRNX1-seg-311 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-311.fasta\n",
      "--- *blasting region: NRNX1-seg-311\n",
      "--- total time for blast NRNX1-seg-311: 6.263s.\n",
      "-- processing region: NRNX1-seg-312 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-312.fasta\n",
      "--- *blasting region: NRNX1-seg-312\n",
      "--- total time for blast NRNX1-seg-312: 3.956s.\n",
      "-- processing region: NRNX1-seg-313 with 76 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-313.fasta\n",
      "--- *blasting region: NRNX1-seg-313\n",
      "--- total time for blast NRNX1-seg-313: 7.757s.\n",
      "-- processing region: NRNX1-seg-314 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-314.fasta\n",
      "--- *blasting region: NRNX1-seg-314\n",
      "--- total time for blast NRNX1-seg-314: 3.630s.\n",
      "-- processing region: NRNX1-seg-315 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-315.fasta\n",
      "--- *blasting region: NRNX1-seg-315\n",
      "--- total time for blast NRNX1-seg-315: 4.145s.\n",
      "-- processing region: NRNX1-seg-316 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-316.fasta\n",
      "--- *blasting region: NRNX1-seg-316\n",
      "--- total time for blast NRNX1-seg-316: 3.367s.\n",
      "-- processing region: NRNX1-seg-317 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-317.fasta\n",
      "--- *blasting region: NRNX1-seg-317\n",
      "--- total time for blast NRNX1-seg-317: 4.257s.\n",
      "-- processing region: NRNX1-seg-318 with 34 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-318.fasta\n",
      "--- *blasting region: NRNX1-seg-318\n",
      "--- total time for blast NRNX1-seg-318: 3.260s.\n",
      "-- processing region: NRNX1-seg-319 with 35 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-319.fasta\n",
      "--- *blasting region: NRNX1-seg-319\n",
      "--- total time for blast NRNX1-seg-319: 5.723s.\n",
      "-- processing region: NRNX1-seg-32 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-32.fasta\n",
      "--- *blasting region: NRNX1-seg-32\n",
      "--- total time for blast NRNX1-seg-32: 5.920s.\n",
      "-- processing region: NRNX1-seg-320 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-320.fasta\n",
      "--- *blasting region: NRNX1-seg-320\n",
      "--- total time for blast NRNX1-seg-320: 3.609s.\n",
      "-- processing region: NRNX1-seg-321 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-321.fasta\n",
      "--- *blasting region: NRNX1-seg-321\n",
      "--- total time for blast NRNX1-seg-321: 3.755s.\n",
      "-- processing region: NRNX1-seg-322 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-322.fasta\n",
      "--- *blasting region: NRNX1-seg-322\n",
      "--- total time for blast NRNX1-seg-322: 3.494s.\n",
      "-- processing region: NRNX1-seg-323 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-323.fasta\n",
      "--- *blasting region: NRNX1-seg-323\n",
      "--- total time for blast NRNX1-seg-323: 3.339s.\n",
      "-- processing region: NRNX1-seg-324 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-324.fasta\n",
      "--- *blasting region: NRNX1-seg-324\n",
      "--- total time for blast NRNX1-seg-324: 3.796s.\n",
      "-- processing region: NRNX1-seg-325 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-325.fasta\n",
      "--- *blasting region: NRNX1-seg-325\n",
      "--- total time for blast NRNX1-seg-325: 4.182s.\n",
      "-- processing region: NRNX1-seg-326 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-326.fasta\n",
      "--- *blasting region: NRNX1-seg-326\n",
      "--- total time for blast NRNX1-seg-326: 5.038s.\n",
      "-- processing region: NRNX1-seg-327 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-327.fasta\n",
      "--- *blasting region: NRNX1-seg-327\n",
      "--- total time for blast NRNX1-seg-327: 5.170s.\n",
      "-- processing region: NRNX1-seg-328 with 34 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-328.fasta\n",
      "--- *blasting region: NRNX1-seg-328\n",
      "--- total time for blast NRNX1-seg-328: 3.039s.\n",
      "-- processing region: NRNX1-seg-329 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-329.fasta\n",
      "--- *blasting region: NRNX1-seg-329\n",
      "--- total time for blast NRNX1-seg-329: 4.743s.\n",
      "-- processing region: NRNX1-seg-33 with 34 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-33.fasta\n",
      "--- *blasting region: NRNX1-seg-33\n",
      "--- total time for blast NRNX1-seg-33: 4.957s.\n",
      "-- processing region: NRNX1-seg-330 with 31 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-330.fasta\n",
      "--- *blasting region: NRNX1-seg-330\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-330: 3.532s.\n",
      "-- processing region: NRNX1-seg-331 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-331.fasta\n",
      "--- *blasting region: NRNX1-seg-331\n",
      "--- total time for blast NRNX1-seg-331: 3.927s.\n",
      "-- processing region: NRNX1-seg-332 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-332.fasta\n",
      "--- *blasting region: NRNX1-seg-332\n",
      "--- total time for blast NRNX1-seg-332: 5.004s.\n",
      "-- processing region: NRNX1-seg-333 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-333.fasta\n",
      "--- *blasting region: NRNX1-seg-333\n",
      "--- total time for blast NRNX1-seg-333: 4.341s.\n",
      "-- processing region: NRNX1-seg-334 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-334.fasta\n",
      "--- *blasting region: NRNX1-seg-334\n",
      "--- total time for blast NRNX1-seg-334: 6.627s.\n",
      "-- processing region: NRNX1-seg-335 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-335.fasta\n",
      "--- *blasting region: NRNX1-seg-335\n",
      "--- total time for blast NRNX1-seg-335: 3.800s.\n",
      "-- processing region: NRNX1-seg-336 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-336.fasta\n",
      "--- *blasting region: NRNX1-seg-336\n",
      "--- total time for blast NRNX1-seg-336: 4.026s.\n",
      "-- processing region: NRNX1-seg-337 with 35 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-337.fasta\n",
      "--- *blasting region: NRNX1-seg-337\n",
      "--- total time for blast NRNX1-seg-337: 3.886s.\n",
      "-- processing region: NRNX1-seg-338 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-338.fasta\n",
      "--- *blasting region: NRNX1-seg-338\n",
      "--- total time for blast NRNX1-seg-338: 5.425s.\n",
      "-- processing region: NRNX1-seg-339 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-339.fasta\n",
      "--- *blasting region: NRNX1-seg-339\n",
      "--- total time for blast NRNX1-seg-339: 5.123s.\n",
      "-- processing region: NRNX1-seg-34 with 43 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-34.fasta\n",
      "--- *blasting region: NRNX1-seg-34\n",
      "--- total time for blast NRNX1-seg-34: 3.434s.\n",
      "-- processing region: NRNX1-seg-340 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-340.fasta\n",
      "--- *blasting region: NRNX1-seg-340\n",
      "--- total time for blast NRNX1-seg-340: 3.654s.\n",
      "-- processing region: NRNX1-seg-341 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-341.fasta\n",
      "--- *blasting region: NRNX1-seg-341\n",
      "--- total time for blast NRNX1-seg-341: 3.597s.\n",
      "-- processing region: NRNX1-seg-342 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-342.fasta\n",
      "--- *blasting region: NRNX1-seg-342\n",
      "--- total time for blast NRNX1-seg-342: 4.419s.\n",
      "-- processing region: NRNX1-seg-343 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-343.fasta\n",
      "--- *blasting region: NRNX1-seg-343\n",
      "--- total time for blast NRNX1-seg-343: 3.961s.\n",
      "-- processing region: NRNX1-seg-344 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-344.fasta\n",
      "--- *blasting region: NRNX1-seg-344\n",
      "--- total time for blast NRNX1-seg-344: 3.753s.\n",
      "-- processing region: NRNX1-seg-345 with 38 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-345.fasta\n",
      "--- *blasting region: NRNX1-seg-345\n",
      "--- total time for blast NRNX1-seg-345: 3.268s.\n",
      "-- processing region: NRNX1-seg-346 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-346.fasta\n",
      "--- *blasting region: NRNX1-seg-346\n",
      "--- total time for blast NRNX1-seg-346: 6.234s.\n",
      "-- processing region: NRNX1-seg-347 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-347.fasta\n",
      "--- *blasting region: NRNX1-seg-347\n",
      "--- total time for blast NRNX1-seg-347: 6.033s.\n",
      "-- processing region: NRNX1-seg-348 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-348.fasta\n",
      "--- *blasting region: NRNX1-seg-348\n",
      "--- total time for blast NRNX1-seg-348: 4.976s.\n",
      "-- processing region: NRNX1-seg-349 with 35 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-349.fasta\n",
      "--- *blasting region: NRNX1-seg-349\n",
      "--- total time for blast NRNX1-seg-349: 3.031s.\n",
      "-- processing region: NRNX1-seg-35 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-35.fasta\n",
      "--- *blasting region: NRNX1-seg-35\n",
      "--- total time for blast NRNX1-seg-35: 3.570s.\n",
      "-- processing region: NRNX1-seg-350 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-350.fasta\n",
      "--- *blasting region: NRNX1-seg-350\n",
      "--- total time for blast NRNX1-seg-350: 4.719s.\n",
      "-- processing region: NRNX1-seg-351 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-351.fasta\n",
      "--- *blasting region: NRNX1-seg-351\n",
      "--- total time for blast NRNX1-seg-351: 3.776s.\n",
      "-- processing region: NRNX1-seg-352 with 36 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-352.fasta\n",
      "--- *blasting region: NRNX1-seg-352\n",
      "--- total time for blast NRNX1-seg-352: 4.578s.\n",
      "-- processing region: NRNX1-seg-353 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-353.fasta\n",
      "--- *blasting region: NRNX1-seg-353\n",
      "--- total time for blast NRNX1-seg-353: 5.252s.\n",
      "-- processing region: NRNX1-seg-354 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-354.fasta\n",
      "--- *blasting region: NRNX1-seg-354\n",
      "--- total time for blast NRNX1-seg-354: 3.651s.\n",
      "-- processing region: NRNX1-seg-355 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-355.fasta\n",
      "--- *blasting region: NRNX1-seg-355\n",
      "--- total time for blast NRNX1-seg-355: 4.518s.\n",
      "-- processing region: NRNX1-seg-356 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-356.fasta\n",
      "--- *blasting region: NRNX1-seg-356\n",
      "--- total time for blast NRNX1-seg-356: 3.551s.\n",
      "-- processing region: NRNX1-seg-357 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-357.fasta\n",
      "--- *blasting region: NRNX1-seg-357\n",
      "--- total time for blast NRNX1-seg-357: 4.136s.\n",
      "-- processing region: NRNX1-seg-358 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-358.fasta\n",
      "--- *blasting region: NRNX1-seg-358\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-358: 3.639s.\n",
      "-- processing region: NRNX1-seg-359 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-359.fasta\n",
      "--- *blasting region: NRNX1-seg-359\n",
      "--- total time for blast NRNX1-seg-359: 4.241s.\n",
      "-- processing region: NRNX1-seg-36 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-36.fasta\n",
      "--- *blasting region: NRNX1-seg-36\n",
      "--- total time for blast NRNX1-seg-36: 5.209s.\n",
      "-- processing region: NRNX1-seg-360 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-360.fasta\n",
      "--- *blasting region: NRNX1-seg-360\n",
      "--- total time for blast NRNX1-seg-360: 3.963s.\n",
      "-- processing region: NRNX1-seg-361 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-361.fasta\n",
      "--- *blasting region: NRNX1-seg-361\n",
      "--- total time for blast NRNX1-seg-361: 4.487s.\n",
      "-- processing region: NRNX1-seg-362 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-362.fasta\n",
      "--- *blasting region: NRNX1-seg-362\n",
      "--- total time for blast NRNX1-seg-362: 4.747s.\n",
      "-- processing region: NRNX1-seg-363 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-363.fasta\n",
      "--- *blasting region: NRNX1-seg-363\n",
      "--- total time for blast NRNX1-seg-363: 4.361s.\n",
      "-- processing region: NRNX1-seg-364 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-364.fasta\n",
      "--- *blasting region: NRNX1-seg-364\n",
      "--- total time for blast NRNX1-seg-364: 4.532s.\n",
      "-- processing region: NRNX1-seg-365 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-365.fasta\n",
      "--- *blasting region: NRNX1-seg-365\n",
      "--- total time for blast NRNX1-seg-365: 3.171s.\n",
      "-- processing region: NRNX1-seg-366 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-366.fasta\n",
      "--- *blasting region: NRNX1-seg-366\n",
      "--- total time for blast NRNX1-seg-366: 7.966s.\n",
      "-- processing region: NRNX1-seg-367 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-367.fasta\n",
      "--- *blasting region: NRNX1-seg-367\n",
      "--- total time for blast NRNX1-seg-367: 4.610s.\n",
      "-- processing region: NRNX1-seg-368 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-368.fasta\n",
      "--- *blasting region: NRNX1-seg-368\n",
      "--- total time for blast NRNX1-seg-368: 3.286s.\n",
      "-- processing region: NRNX1-seg-369 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-369.fasta\n",
      "--- *blasting region: NRNX1-seg-369\n",
      "--- total time for blast NRNX1-seg-369: 3.959s.\n",
      "-- processing region: NRNX1-seg-37 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-37.fasta\n",
      "--- *blasting region: NRNX1-seg-37\n",
      "--- total time for blast NRNX1-seg-37: 6.762s.\n",
      "-- processing region: NRNX1-seg-370 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-370.fasta\n",
      "--- *blasting region: NRNX1-seg-370\n",
      "--- total time for blast NRNX1-seg-370: 6.608s.\n",
      "-- processing region: NRNX1-seg-371 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-371.fasta\n",
      "--- *blasting region: NRNX1-seg-371\n",
      "--- total time for blast NRNX1-seg-371: 7.031s.\n",
      "-- processing region: NRNX1-seg-372 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-372.fasta\n",
      "--- *blasting region: NRNX1-seg-372\n",
      "--- total time for blast NRNX1-seg-372: 4.309s.\n",
      "-- processing region: NRNX1-seg-373 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-373.fasta\n",
      "--- *blasting region: NRNX1-seg-373\n",
      "--- total time for blast NRNX1-seg-373: 3.725s.\n",
      "-- processing region: NRNX1-seg-374 with 31 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-374.fasta\n",
      "--- *blasting region: NRNX1-seg-374\n",
      "--- total time for blast NRNX1-seg-374: 3.867s.\n",
      "-- processing region: NRNX1-seg-375 with 21 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-375.fasta\n",
      "--- *blasting region: NRNX1-seg-375\n",
      "--- total time for blast NRNX1-seg-375: 11.264s.\n",
      "-- processing region: NRNX1-seg-376 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-376.fasta\n",
      "--- *blasting region: NRNX1-seg-376\n",
      "--- total time for blast NRNX1-seg-376: 3.389s.\n",
      "-- processing region: NRNX1-seg-379 with 30 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-379.fasta\n",
      "--- *blasting region: NRNX1-seg-379\n",
      "--- total time for blast NRNX1-seg-379: 3.295s.\n",
      "-- processing region: NRNX1-seg-38 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-38.fasta\n",
      "--- *blasting region: NRNX1-seg-38\n",
      "--- total time for blast NRNX1-seg-38: 3.927s.\n",
      "-- processing region: NRNX1-seg-380 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-380.fasta\n",
      "--- *blasting region: NRNX1-seg-380\n",
      "--- total time for blast NRNX1-seg-380: 4.865s.\n",
      "-- processing region: NRNX1-seg-381 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-381.fasta\n",
      "--- *blasting region: NRNX1-seg-381\n",
      "--- total time for blast NRNX1-seg-381: 4.263s.\n",
      "-- processing region: NRNX1-seg-382 with 43 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-382.fasta\n",
      "--- *blasting region: NRNX1-seg-382\n",
      "--- total time for blast NRNX1-seg-382: 4.144s.\n",
      "-- processing region: NRNX1-seg-383 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-383.fasta\n",
      "--- *blasting region: NRNX1-seg-383\n",
      "--- total time for blast NRNX1-seg-383: 4.331s.\n",
      "-- processing region: NRNX1-seg-384 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-384.fasta\n",
      "--- *blasting region: NRNX1-seg-384\n",
      "--- total time for blast NRNX1-seg-384: 3.220s.\n",
      "-- processing region: NRNX1-seg-385 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-385.fasta\n",
      "--- *blasting region: NRNX1-seg-385\n",
      "--- total time for blast NRNX1-seg-385: 4.425s.\n",
      "-- processing region: NRNX1-seg-386 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-386.fasta\n",
      "--- *blasting region: NRNX1-seg-386\n",
      "--- total time for blast NRNX1-seg-386: 3.453s.\n",
      "-- processing region: NRNX1-seg-387 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-387.fasta\n",
      "--- *blasting region: NRNX1-seg-387\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-387: 5.168s.\n",
      "-- processing region: NRNX1-seg-388 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-388.fasta\n",
      "--- *blasting region: NRNX1-seg-388\n",
      "--- total time for blast NRNX1-seg-388: 4.158s.\n",
      "-- processing region: NRNX1-seg-389 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-389.fasta\n",
      "--- *blasting region: NRNX1-seg-389\n",
      "--- total time for blast NRNX1-seg-389: 4.423s.\n",
      "-- processing region: NRNX1-seg-39 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-39.fasta\n",
      "--- *blasting region: NRNX1-seg-39\n",
      "--- total time for blast NRNX1-seg-39: 3.623s.\n",
      "-- processing region: NRNX1-seg-390 with 32 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-390.fasta\n",
      "--- *blasting region: NRNX1-seg-390\n",
      "--- total time for blast NRNX1-seg-390: 2.112s.\n",
      "-- processing region: NRNX1-seg-391 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-391.fasta\n",
      "--- *blasting region: NRNX1-seg-391\n",
      "--- total time for blast NRNX1-seg-391: 3.666s.\n",
      "-- processing region: NRNX1-seg-392 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-392.fasta\n",
      "--- *blasting region: NRNX1-seg-392\n",
      "--- total time for blast NRNX1-seg-392: 4.839s.\n",
      "-- processing region: NRNX1-seg-393 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-393.fasta\n",
      "--- *blasting region: NRNX1-seg-393\n",
      "--- total time for blast NRNX1-seg-393: 3.429s.\n",
      "-- processing region: NRNX1-seg-394 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-394.fasta\n",
      "--- *blasting region: NRNX1-seg-394\n",
      "--- total time for blast NRNX1-seg-394: 3.387s.\n",
      "-- processing region: NRNX1-seg-395 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-395.fasta\n",
      "--- *blasting region: NRNX1-seg-395\n",
      "--- total time for blast NRNX1-seg-395: 4.304s.\n",
      "-- processing region: NRNX1-seg-396 with 43 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-396.fasta\n",
      "--- *blasting region: NRNX1-seg-396\n",
      "--- total time for blast NRNX1-seg-396: 3.006s.\n",
      "-- processing region: NRNX1-seg-397 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-397.fasta\n",
      "--- *blasting region: NRNX1-seg-397\n",
      "--- total time for blast NRNX1-seg-397: 3.328s.\n",
      "-- processing region: NRNX1-seg-398 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-398.fasta\n",
      "--- *blasting region: NRNX1-seg-398\n",
      "--- total time for blast NRNX1-seg-398: 3.392s.\n",
      "-- processing region: NRNX1-seg-399 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-399.fasta\n",
      "--- *blasting region: NRNX1-seg-399\n",
      "--- total time for blast NRNX1-seg-399: 4.235s.\n",
      "-- processing region: NRNX1-seg-4 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-4.fasta\n",
      "--- *blasting region: NRNX1-seg-4\n",
      "--- total time for blast NRNX1-seg-4: 4.812s.\n",
      "-- processing region: NRNX1-seg-40 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-40.fasta\n",
      "--- *blasting region: NRNX1-seg-40\n",
      "--- total time for blast NRNX1-seg-40: 4.705s.\n",
      "-- processing region: NRNX1-seg-400 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-400.fasta\n",
      "--- *blasting region: NRNX1-seg-400\n",
      "--- total time for blast NRNX1-seg-400: 4.269s.\n",
      "-- processing region: NRNX1-seg-401 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-401.fasta\n",
      "--- *blasting region: NRNX1-seg-401\n",
      "--- total time for blast NRNX1-seg-401: 4.298s.\n",
      "-- processing region: NRNX1-seg-402 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-402.fasta\n",
      "--- *blasting region: NRNX1-seg-402\n",
      "--- total time for blast NRNX1-seg-402: 4.871s.\n",
      "-- processing region: NRNX1-seg-403 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-403.fasta\n",
      "--- *blasting region: NRNX1-seg-403\n",
      "--- total time for blast NRNX1-seg-403: 4.506s.\n",
      "-- processing region: NRNX1-seg-404 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-404.fasta\n",
      "--- *blasting region: NRNX1-seg-404\n",
      "--- total time for blast NRNX1-seg-404: 4.570s.\n",
      "-- processing region: NRNX1-seg-405 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-405.fasta\n",
      "--- *blasting region: NRNX1-seg-405\n",
      "--- total time for blast NRNX1-seg-405: 3.144s.\n",
      "-- processing region: NRNX1-seg-406 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-406.fasta\n",
      "--- *blasting region: NRNX1-seg-406\n",
      "--- total time for blast NRNX1-seg-406: 5.968s.\n",
      "-- processing region: NRNX1-seg-407 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-407.fasta\n",
      "--- *blasting region: NRNX1-seg-407\n",
      "--- total time for blast NRNX1-seg-407: 4.465s.\n",
      "-- processing region: NRNX1-seg-408 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-408.fasta\n",
      "--- *blasting region: NRNX1-seg-408\n",
      "--- total time for blast NRNX1-seg-408: 3.140s.\n",
      "-- processing region: NRNX1-seg-409 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-409.fasta\n",
      "--- *blasting region: NRNX1-seg-409\n",
      "--- total time for blast NRNX1-seg-409: 3.286s.\n",
      "-- processing region: NRNX1-seg-41 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-41.fasta\n",
      "--- *blasting region: NRNX1-seg-41\n",
      "--- total time for blast NRNX1-seg-41: 3.587s.\n",
      "-- processing region: NRNX1-seg-410 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-410.fasta\n",
      "--- *blasting region: NRNX1-seg-410\n",
      "--- total time for blast NRNX1-seg-410: 3.529s.\n",
      "-- processing region: NRNX1-seg-411 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-411.fasta\n",
      "--- *blasting region: NRNX1-seg-411\n",
      "--- total time for blast NRNX1-seg-411: 3.206s.\n",
      "-- processing region: NRNX1-seg-412 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-412.fasta\n",
      "--- *blasting region: NRNX1-seg-412\n",
      "--- total time for blast NRNX1-seg-412: 3.442s.\n",
      "-- processing region: NRNX1-seg-413 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-413.fasta\n",
      "--- *blasting region: NRNX1-seg-413\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-413: 4.485s.\n",
      "-- processing region: NRNX1-seg-414 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-414.fasta\n",
      "--- *blasting region: NRNX1-seg-414\n",
      "--- total time for blast NRNX1-seg-414: 4.104s.\n",
      "-- processing region: NRNX1-seg-415 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-415.fasta\n",
      "--- *blasting region: NRNX1-seg-415\n",
      "--- total time for blast NRNX1-seg-415: 4.232s.\n",
      "-- processing region: NRNX1-seg-416 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-416.fasta\n",
      "--- *blasting region: NRNX1-seg-416\n",
      "--- total time for blast NRNX1-seg-416: 3.602s.\n",
      "-- processing region: NRNX1-seg-417 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-417.fasta\n",
      "--- *blasting region: NRNX1-seg-417\n",
      "--- total time for blast NRNX1-seg-417: 4.330s.\n",
      "-- processing region: NRNX1-seg-418 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-418.fasta\n",
      "--- *blasting region: NRNX1-seg-418\n",
      "--- total time for blast NRNX1-seg-418: 2.805s.\n",
      "-- processing region: NRNX1-seg-419 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-419.fasta\n",
      "--- *blasting region: NRNX1-seg-419\n",
      "--- total time for blast NRNX1-seg-419: 3.379s.\n",
      "-- processing region: NRNX1-seg-42 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-42.fasta\n",
      "--- *blasting region: NRNX1-seg-42\n",
      "--- total time for blast NRNX1-seg-42: 4.176s.\n",
      "-- processing region: NRNX1-seg-420 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-420.fasta\n",
      "--- *blasting region: NRNX1-seg-420\n",
      "--- total time for blast NRNX1-seg-420: 3.408s.\n",
      "-- processing region: NRNX1-seg-421 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-421.fasta\n",
      "--- *blasting region: NRNX1-seg-421\n",
      "--- total time for blast NRNX1-seg-421: 4.277s.\n",
      "-- processing region: NRNX1-seg-422 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-422.fasta\n",
      "--- *blasting region: NRNX1-seg-422\n",
      "--- total time for blast NRNX1-seg-422: 3.625s.\n",
      "-- processing region: NRNX1-seg-423 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-423.fasta\n",
      "--- *blasting region: NRNX1-seg-423\n",
      "--- total time for blast NRNX1-seg-423: 4.474s.\n",
      "-- processing region: NRNX1-seg-424 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-424.fasta\n",
      "--- *blasting region: NRNX1-seg-424\n",
      "--- total time for blast NRNX1-seg-424: 4.584s.\n",
      "-- processing region: NRNX1-seg-425 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-425.fasta\n",
      "--- *blasting region: NRNX1-seg-425\n",
      "--- total time for blast NRNX1-seg-425: 3.388s.\n",
      "-- processing region: NRNX1-seg-426 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-426.fasta\n",
      "--- *blasting region: NRNX1-seg-426\n",
      "--- total time for blast NRNX1-seg-426: 3.693s.\n",
      "-- processing region: NRNX1-seg-427 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-427.fasta\n",
      "--- *blasting region: NRNX1-seg-427\n",
      "--- total time for blast NRNX1-seg-427: 4.503s.\n",
      "-- processing region: NRNX1-seg-428 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-428.fasta\n",
      "--- *blasting region: NRNX1-seg-428\n",
      "--- total time for blast NRNX1-seg-428: 3.362s.\n",
      "-- processing region: NRNX1-seg-429 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-429.fasta\n",
      "--- *blasting region: NRNX1-seg-429\n",
      "--- total time for blast NRNX1-seg-429: 3.675s.\n",
      "-- processing region: NRNX1-seg-43 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-43.fasta\n",
      "--- *blasting region: NRNX1-seg-43\n",
      "--- total time for blast NRNX1-seg-43: 4.543s.\n",
      "-- processing region: NRNX1-seg-430 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-430.fasta\n",
      "--- *blasting region: NRNX1-seg-430\n",
      "--- total time for blast NRNX1-seg-430: 4.078s.\n",
      "-- processing region: NRNX1-seg-431 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-431.fasta\n",
      "--- *blasting region: NRNX1-seg-431\n",
      "--- total time for blast NRNX1-seg-431: 3.545s.\n",
      "-- processing region: NRNX1-seg-432 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-432.fasta\n",
      "--- *blasting region: NRNX1-seg-432\n",
      "--- total time for blast NRNX1-seg-432: 3.399s.\n",
      "-- processing region: NRNX1-seg-433 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-433.fasta\n",
      "--- *blasting region: NRNX1-seg-433\n",
      "--- total time for blast NRNX1-seg-433: 4.826s.\n",
      "-- processing region: NRNX1-seg-434 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-434.fasta\n",
      "--- *blasting region: NRNX1-seg-434\n",
      "--- total time for blast NRNX1-seg-434: 3.726s.\n",
      "-- processing region: NRNX1-seg-435 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-435.fasta\n",
      "--- *blasting region: NRNX1-seg-435\n",
      "--- total time for blast NRNX1-seg-435: 3.965s.\n",
      "-- processing region: NRNX1-seg-436 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-436.fasta\n",
      "--- *blasting region: NRNX1-seg-436\n",
      "--- total time for blast NRNX1-seg-436: 3.760s.\n",
      "-- processing region: NRNX1-seg-437 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-437.fasta\n",
      "--- *blasting region: NRNX1-seg-437\n",
      "--- total time for blast NRNX1-seg-437: 3.355s.\n",
      "-- processing region: NRNX1-seg-438 with 34 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-438.fasta\n",
      "--- *blasting region: NRNX1-seg-438\n",
      "--- total time for blast NRNX1-seg-438: 9.511s.\n",
      "-- processing region: NRNX1-seg-439 with 38 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-439.fasta\n",
      "--- *blasting region: NRNX1-seg-439\n",
      "--- total time for blast NRNX1-seg-439: 3.505s.\n",
      "-- processing region: NRNX1-seg-44 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-44.fasta\n",
      "--- *blasting region: NRNX1-seg-44\n",
      "--- total time for blast NRNX1-seg-44: 4.079s.\n",
      "-- processing region: NRNX1-seg-440 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-440.fasta\n",
      "--- *blasting region: NRNX1-seg-440\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-440: 11.107s.\n",
      "-- processing region: NRNX1-seg-441 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-441.fasta\n",
      "--- *blasting region: NRNX1-seg-441\n",
      "--- total time for blast NRNX1-seg-441: 3.865s.\n",
      "-- processing region: NRNX1-seg-442 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-442.fasta\n",
      "--- *blasting region: NRNX1-seg-442\n",
      "--- total time for blast NRNX1-seg-442: 4.617s.\n",
      "-- processing region: NRNX1-seg-443 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-443.fasta\n",
      "--- *blasting region: NRNX1-seg-443\n",
      "--- total time for blast NRNX1-seg-443: 3.028s.\n",
      "-- processing region: NRNX1-seg-444 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-444.fasta\n",
      "--- *blasting region: NRNX1-seg-444\n",
      "--- total time for blast NRNX1-seg-444: 3.042s.\n",
      "-- processing region: NRNX1-seg-445 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-445.fasta\n",
      "--- *blasting region: NRNX1-seg-445\n",
      "--- total time for blast NRNX1-seg-445: 3.819s.\n",
      "-- processing region: NRNX1-seg-446 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-446.fasta\n",
      "--- *blasting region: NRNX1-seg-446\n",
      "--- total time for blast NRNX1-seg-446: 4.439s.\n",
      "-- processing region: NRNX1-seg-447 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-447.fasta\n",
      "--- *blasting region: NRNX1-seg-447\n",
      "--- total time for blast NRNX1-seg-447: 3.628s.\n",
      "-- processing region: NRNX1-seg-448 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-448.fasta\n",
      "--- *blasting region: NRNX1-seg-448\n",
      "--- total time for blast NRNX1-seg-448: 3.467s.\n",
      "-- processing region: NRNX1-seg-449 with 43 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-449.fasta\n",
      "--- *blasting region: NRNX1-seg-449\n",
      "--- total time for blast NRNX1-seg-449: 2.927s.\n",
      "-- processing region: NRNX1-seg-45 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-45.fasta\n",
      "--- *blasting region: NRNX1-seg-45\n",
      "--- total time for blast NRNX1-seg-45: 8.932s.\n",
      "-- processing region: NRNX1-seg-450 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-450.fasta\n",
      "--- *blasting region: NRNX1-seg-450\n",
      "--- total time for blast NRNX1-seg-450: 6.640s.\n",
      "-- processing region: NRNX1-seg-452 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-452.fasta\n",
      "--- *blasting region: NRNX1-seg-452\n",
      "--- total time for blast NRNX1-seg-452: 4.448s.\n",
      "-- processing region: NRNX1-seg-453 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-453.fasta\n",
      "--- *blasting region: NRNX1-seg-453\n",
      "--- total time for blast NRNX1-seg-453: 4.386s.\n",
      "-- processing region: NRNX1-seg-454 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-454.fasta\n",
      "--- *blasting region: NRNX1-seg-454\n",
      "--- total time for blast NRNX1-seg-454: 3.078s.\n",
      "-- processing region: NRNX1-seg-455 with 43 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-455.fasta\n",
      "--- *blasting region: NRNX1-seg-455\n",
      "--- total time for blast NRNX1-seg-455: 3.114s.\n",
      "-- processing region: NRNX1-seg-456 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-456.fasta\n",
      "--- *blasting region: NRNX1-seg-456\n",
      "--- total time for blast NRNX1-seg-456: 3.997s.\n",
      "-- processing region: NRNX1-seg-457 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-457.fasta\n",
      "--- *blasting region: NRNX1-seg-457\n",
      "--- total time for blast NRNX1-seg-457: 2.984s.\n",
      "-- processing region: NRNX1-seg-458 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-458.fasta\n",
      "--- *blasting region: NRNX1-seg-458\n",
      "--- total time for blast NRNX1-seg-458: 4.058s.\n",
      "-- processing region: NRNX1-seg-459 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-459.fasta\n",
      "--- *blasting region: NRNX1-seg-459\n",
      "--- total time for blast NRNX1-seg-459: 4.330s.\n",
      "-- processing region: NRNX1-seg-46 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-46.fasta\n",
      "--- *blasting region: NRNX1-seg-46\n",
      "--- total time for blast NRNX1-seg-46: 3.295s.\n",
      "-- processing region: NRNX1-seg-460 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-460.fasta\n",
      "--- *blasting region: NRNX1-seg-460\n",
      "--- total time for blast NRNX1-seg-460: 4.666s.\n",
      "-- processing region: NRNX1-seg-461 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-461.fasta\n",
      "--- *blasting region: NRNX1-seg-461\n",
      "--- total time for blast NRNX1-seg-461: 3.936s.\n",
      "-- processing region: NRNX1-seg-462 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-462.fasta\n",
      "--- *blasting region: NRNX1-seg-462\n",
      "--- total time for blast NRNX1-seg-462: 4.218s.\n",
      "-- processing region: NRNX1-seg-463 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-463.fasta\n",
      "--- *blasting region: NRNX1-seg-463\n",
      "--- total time for blast NRNX1-seg-463: 4.216s.\n",
      "-- processing region: NRNX1-seg-464 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-464.fasta\n",
      "--- *blasting region: NRNX1-seg-464\n",
      "--- total time for blast NRNX1-seg-464: 3.551s.\n",
      "-- processing region: NRNX1-seg-465 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-465.fasta\n",
      "--- *blasting region: NRNX1-seg-465\n",
      "--- total time for blast NRNX1-seg-465: 3.344s.\n",
      "-- processing region: NRNX1-seg-466 with 34 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-466.fasta\n",
      "--- *blasting region: NRNX1-seg-466\n",
      "--- total time for blast NRNX1-seg-466: 2.381s.\n",
      "-- processing region: NRNX1-seg-467 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-467.fasta\n",
      "--- *blasting region: NRNX1-seg-467\n",
      "--- total time for blast NRNX1-seg-467: 3.515s.\n",
      "-- processing region: NRNX1-seg-468 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-468.fasta\n",
      "--- *blasting region: NRNX1-seg-468\n",
      "--- total time for blast NRNX1-seg-468: 3.975s.\n",
      "-- processing region: NRNX1-seg-469 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-469.fasta\n",
      "--- *blasting region: NRNX1-seg-469\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-469: 5.280s.\n",
      "-- processing region: NRNX1-seg-47 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-47.fasta\n",
      "--- *blasting region: NRNX1-seg-47\n",
      "--- total time for blast NRNX1-seg-47: 3.260s.\n",
      "-- processing region: NRNX1-seg-470 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-470.fasta\n",
      "--- *blasting region: NRNX1-seg-470\n",
      "--- total time for blast NRNX1-seg-470: 4.095s.\n",
      "-- processing region: NRNX1-seg-471 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-471.fasta\n",
      "--- *blasting region: NRNX1-seg-471\n",
      "--- total time for blast NRNX1-seg-471: 3.534s.\n",
      "-- processing region: NRNX1-seg-472 with 26 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-472.fasta\n",
      "--- *blasting region: NRNX1-seg-472\n",
      "--- total time for blast NRNX1-seg-472: 3.859s.\n",
      "-- processing region: NRNX1-seg-474 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-474.fasta\n",
      "--- *blasting region: NRNX1-seg-474\n",
      "--- total time for blast NRNX1-seg-474: 4.153s.\n",
      "-- processing region: NRNX1-seg-475 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-475.fasta\n",
      "--- *blasting region: NRNX1-seg-475\n",
      "--- total time for blast NRNX1-seg-475: 4.911s.\n",
      "-- processing region: NRNX1-seg-476 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-476.fasta\n",
      "--- *blasting region: NRNX1-seg-476\n",
      "--- total time for blast NRNX1-seg-476: 5.303s.\n",
      "-- processing region: NRNX1-seg-477 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-477.fasta\n",
      "--- *blasting region: NRNX1-seg-477\n",
      "--- total time for blast NRNX1-seg-477: 4.201s.\n",
      "-- processing region: NRNX1-seg-478 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-478.fasta\n",
      "--- *blasting region: NRNX1-seg-478\n",
      "--- total time for blast NRNX1-seg-478: 2.867s.\n",
      "-- processing region: NRNX1-seg-479 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-479.fasta\n",
      "--- *blasting region: NRNX1-seg-479\n",
      "--- total time for blast NRNX1-seg-479: 6.422s.\n",
      "-- processing region: NRNX1-seg-48 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-48.fasta\n",
      "--- *blasting region: NRNX1-seg-48\n",
      "--- total time for blast NRNX1-seg-48: 3.411s.\n",
      "-- processing region: NRNX1-seg-480 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-480.fasta\n",
      "--- *blasting region: NRNX1-seg-480\n",
      "--- total time for blast NRNX1-seg-480: 3.841s.\n",
      "-- processing region: NRNX1-seg-481 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-481.fasta\n",
      "--- *blasting region: NRNX1-seg-481\n",
      "--- total time for blast NRNX1-seg-481: 4.225s.\n",
      "-- processing region: NRNX1-seg-482 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-482.fasta\n",
      "--- *blasting region: NRNX1-seg-482\n",
      "--- total time for blast NRNX1-seg-482: 4.345s.\n",
      "-- processing region: NRNX1-seg-483 with 25 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-483.fasta\n",
      "--- *blasting region: NRNX1-seg-483\n",
      "--- total time for blast NRNX1-seg-483: 3.719s.\n",
      "-- processing region: NRNX1-seg-484 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-484.fasta\n",
      "--- *blasting region: NRNX1-seg-484\n",
      "--- total time for blast NRNX1-seg-484: 4.645s.\n",
      "-- processing region: NRNX1-seg-485 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-485.fasta\n",
      "--- *blasting region: NRNX1-seg-485\n",
      "--- total time for blast NRNX1-seg-485: 4.662s.\n",
      "-- processing region: NRNX1-seg-486 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-486.fasta\n",
      "--- *blasting region: NRNX1-seg-486\n",
      "--- total time for blast NRNX1-seg-486: 3.840s.\n",
      "-- processing region: NRNX1-seg-487 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-487.fasta\n",
      "--- *blasting region: NRNX1-seg-487\n",
      "--- total time for blast NRNX1-seg-487: 4.693s.\n",
      "-- processing region: NRNX1-seg-488 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-488.fasta\n",
      "--- *blasting region: NRNX1-seg-488\n",
      "--- total time for blast NRNX1-seg-488: 10.272s.\n",
      "-- processing region: NRNX1-seg-489 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-489.fasta\n",
      "--- *blasting region: NRNX1-seg-489\n",
      "--- total time for blast NRNX1-seg-489: 3.631s.\n",
      "-- processing region: NRNX1-seg-49 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-49.fasta\n",
      "--- *blasting region: NRNX1-seg-49\n",
      "--- total time for blast NRNX1-seg-49: 3.811s.\n",
      "-- processing region: NRNX1-seg-490 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-490.fasta\n",
      "--- *blasting region: NRNX1-seg-490\n",
      "--- total time for blast NRNX1-seg-490: 3.821s.\n",
      "-- processing region: NRNX1-seg-491 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-491.fasta\n",
      "--- *blasting region: NRNX1-seg-491\n",
      "--- total time for blast NRNX1-seg-491: 3.579s.\n",
      "-- processing region: NRNX1-seg-492 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-492.fasta\n",
      "--- *blasting region: NRNX1-seg-492\n",
      "--- total time for blast NRNX1-seg-492: 3.494s.\n",
      "-- processing region: NRNX1-seg-493 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-493.fasta\n",
      "--- *blasting region: NRNX1-seg-493\n",
      "--- total time for blast NRNX1-seg-493: 3.268s.\n",
      "-- processing region: NRNX1-seg-494 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-494.fasta\n",
      "--- *blasting region: NRNX1-seg-494\n",
      "--- total time for blast NRNX1-seg-494: 3.495s.\n",
      "-- processing region: NRNX1-seg-495 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-495.fasta\n",
      "--- *blasting region: NRNX1-seg-495\n",
      "--- total time for blast NRNX1-seg-495: 4.112s.\n",
      "-- processing region: NRNX1-seg-496 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-496.fasta\n",
      "--- *blasting region: NRNX1-seg-496\n",
      "--- total time for blast NRNX1-seg-496: 4.591s.\n",
      "-- processing region: NRNX1-seg-497 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-497.fasta\n",
      "--- *blasting region: NRNX1-seg-497\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-497: 4.477s.\n",
      "-- processing region: NRNX1-seg-498 with 80 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-498.fasta\n",
      "--- *blasting region: NRNX1-seg-498\n",
      "--- total time for blast NRNX1-seg-498: 4.499s.\n",
      "-- processing region: NRNX1-seg-499 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-499.fasta\n",
      "--- *blasting region: NRNX1-seg-499\n",
      "--- total time for blast NRNX1-seg-499: 4.181s.\n",
      "-- processing region: NRNX1-seg-5 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-5.fasta\n",
      "--- *blasting region: NRNX1-seg-5\n",
      "--- total time for blast NRNX1-seg-5: 4.293s.\n",
      "-- processing region: NRNX1-seg-50 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-50.fasta\n",
      "--- *blasting region: NRNX1-seg-50\n",
      "--- total time for blast NRNX1-seg-50: 5.747s.\n",
      "-- processing region: NRNX1-seg-500 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-500.fasta\n",
      "--- *blasting region: NRNX1-seg-500\n",
      "--- total time for blast NRNX1-seg-500: 4.461s.\n",
      "-- processing region: NRNX1-seg-501 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-501.fasta\n",
      "--- *blasting region: NRNX1-seg-501\n",
      "--- total time for blast NRNX1-seg-501: 3.373s.\n",
      "-- processing region: NRNX1-seg-502 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-502.fasta\n",
      "--- *blasting region: NRNX1-seg-502\n",
      "--- total time for blast NRNX1-seg-502: 3.817s.\n",
      "-- processing region: NRNX1-seg-503 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-503.fasta\n",
      "--- *blasting region: NRNX1-seg-503\n",
      "--- total time for blast NRNX1-seg-503: 3.662s.\n",
      "-- processing region: NRNX1-seg-504 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-504.fasta\n",
      "--- *blasting region: NRNX1-seg-504\n",
      "--- total time for blast NRNX1-seg-504: 4.088s.\n",
      "-- processing region: NRNX1-seg-505 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-505.fasta\n",
      "--- *blasting region: NRNX1-seg-505\n",
      "--- total time for blast NRNX1-seg-505: 2.990s.\n",
      "-- processing region: NRNX1-seg-506 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-506.fasta\n",
      "--- *blasting region: NRNX1-seg-506\n",
      "--- total time for blast NRNX1-seg-506: 3.620s.\n",
      "-- processing region: NRNX1-seg-507 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-507.fasta\n",
      "--- *blasting region: NRNX1-seg-507\n",
      "--- total time for blast NRNX1-seg-507: 3.710s.\n",
      "-- processing region: NRNX1-seg-508 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-508.fasta\n",
      "--- *blasting region: NRNX1-seg-508\n",
      "--- total time for blast NRNX1-seg-508: 3.837s.\n",
      "-- processing region: NRNX1-seg-509 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-509.fasta\n",
      "--- *blasting region: NRNX1-seg-509\n",
      "--- total time for blast NRNX1-seg-509: 4.520s.\n",
      "-- processing region: NRNX1-seg-51 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-51.fasta\n",
      "--- *blasting region: NRNX1-seg-51\n",
      "--- total time for blast NRNX1-seg-51: 4.568s.\n",
      "-- processing region: NRNX1-seg-510 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-510.fasta\n",
      "--- *blasting region: NRNX1-seg-510\n",
      "--- total time for blast NRNX1-seg-510: 3.632s.\n",
      "-- processing region: NRNX1-seg-511 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-511.fasta\n",
      "--- *blasting region: NRNX1-seg-511\n",
      "--- total time for blast NRNX1-seg-511: 3.571s.\n",
      "-- processing region: NRNX1-seg-512 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-512.fasta\n",
      "--- *blasting region: NRNX1-seg-512\n",
      "--- total time for blast NRNX1-seg-512: 3.486s.\n",
      "-- processing region: NRNX1-seg-513 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-513.fasta\n",
      "--- *blasting region: NRNX1-seg-513\n",
      "--- total time for blast NRNX1-seg-513: 3.300s.\n",
      "-- processing region: NRNX1-seg-514 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-514.fasta\n",
      "--- *blasting region: NRNX1-seg-514\n",
      "--- total time for blast NRNX1-seg-514: 5.547s.\n",
      "-- processing region: NRNX1-seg-515 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-515.fasta\n",
      "--- *blasting region: NRNX1-seg-515\n",
      "--- total time for blast NRNX1-seg-515: 3.940s.\n",
      "-- processing region: NRNX1-seg-516 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-516.fasta\n",
      "--- *blasting region: NRNX1-seg-516\n",
      "--- total time for blast NRNX1-seg-516: 7.654s.\n",
      "-- processing region: NRNX1-seg-517 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-517.fasta\n",
      "--- *blasting region: NRNX1-seg-517\n",
      "--- total time for blast NRNX1-seg-517: 4.031s.\n",
      "-- processing region: NRNX1-seg-518 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-518.fasta\n",
      "--- *blasting region: NRNX1-seg-518\n",
      "--- total time for blast NRNX1-seg-518: 3.685s.\n",
      "-- processing region: NRNX1-seg-519 with 62 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-519.fasta\n",
      "--- *blasting region: NRNX1-seg-519\n",
      "--- total time for blast NRNX1-seg-519: 4.517s.\n",
      "-- processing region: NRNX1-seg-52 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-52.fasta\n",
      "--- *blasting region: NRNX1-seg-52\n",
      "--- total time for blast NRNX1-seg-52: 4.648s.\n",
      "-- processing region: NRNX1-seg-520 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-520.fasta\n",
      "--- *blasting region: NRNX1-seg-520\n",
      "--- total time for blast NRNX1-seg-520: 4.124s.\n",
      "-- processing region: NRNX1-seg-521 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-521.fasta\n",
      "--- *blasting region: NRNX1-seg-521\n",
      "--- total time for blast NRNX1-seg-521: 4.047s.\n",
      "-- processing region: NRNX1-seg-522 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-522.fasta\n",
      "--- *blasting region: NRNX1-seg-522\n",
      "--- total time for blast NRNX1-seg-522: 4.315s.\n",
      "-- processing region: NRNX1-seg-523 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-523.fasta\n",
      "--- *blasting region: NRNX1-seg-523\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-523: 4.938s.\n",
      "-- processing region: NRNX1-seg-524 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-524.fasta\n",
      "--- *blasting region: NRNX1-seg-524\n",
      "--- total time for blast NRNX1-seg-524: 4.648s.\n",
      "-- processing region: NRNX1-seg-525 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-525.fasta\n",
      "--- *blasting region: NRNX1-seg-525\n",
      "--- total time for blast NRNX1-seg-525: 4.124s.\n",
      "-- processing region: NRNX1-seg-526 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-526.fasta\n",
      "--- *blasting region: NRNX1-seg-526\n",
      "--- total time for blast NRNX1-seg-526: 3.142s.\n",
      "-- processing region: NRNX1-seg-527 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-527.fasta\n",
      "--- *blasting region: NRNX1-seg-527\n",
      "--- total time for blast NRNX1-seg-527: 4.440s.\n",
      "-- processing region: NRNX1-seg-528 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-528.fasta\n",
      "--- *blasting region: NRNX1-seg-528\n",
      "--- total time for blast NRNX1-seg-528: 3.162s.\n",
      "-- processing region: NRNX1-seg-529 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-529.fasta\n",
      "--- *blasting region: NRNX1-seg-529\n",
      "--- total time for blast NRNX1-seg-529: 4.815s.\n",
      "-- processing region: NRNX1-seg-53 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-53.fasta\n",
      "--- *blasting region: NRNX1-seg-53\n",
      "--- total time for blast NRNX1-seg-53: 4.318s.\n",
      "-- processing region: NRNX1-seg-530 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-530.fasta\n",
      "--- *blasting region: NRNX1-seg-530\n",
      "--- total time for blast NRNX1-seg-530: 4.531s.\n",
      "-- processing region: NRNX1-seg-531 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-531.fasta\n",
      "--- *blasting region: NRNX1-seg-531\n",
      "--- total time for blast NRNX1-seg-531: 3.972s.\n",
      "-- processing region: NRNX1-seg-532 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-532.fasta\n",
      "--- *blasting region: NRNX1-seg-532\n",
      "--- total time for blast NRNX1-seg-532: 4.068s.\n",
      "-- processing region: NRNX1-seg-533 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-533.fasta\n",
      "--- *blasting region: NRNX1-seg-533\n",
      "--- total time for blast NRNX1-seg-533: 4.792s.\n",
      "-- processing region: NRNX1-seg-534 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-534.fasta\n",
      "--- *blasting region: NRNX1-seg-534\n",
      "--- total time for blast NRNX1-seg-534: 4.081s.\n",
      "-- processing region: NRNX1-seg-535 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-535.fasta\n",
      "--- *blasting region: NRNX1-seg-535\n",
      "--- total time for blast NRNX1-seg-535: 3.440s.\n",
      "-- processing region: NRNX1-seg-536 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-536.fasta\n",
      "--- *blasting region: NRNX1-seg-536\n",
      "--- total time for blast NRNX1-seg-536: 4.759s.\n",
      "-- processing region: NRNX1-seg-537 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-537.fasta\n",
      "--- *blasting region: NRNX1-seg-537\n",
      "--- total time for blast NRNX1-seg-537: 3.274s.\n",
      "-- processing region: NRNX1-seg-538 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-538.fasta\n",
      "--- *blasting region: NRNX1-seg-538\n",
      "--- total time for blast NRNX1-seg-538: 3.676s.\n",
      "-- processing region: NRNX1-seg-539 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-539.fasta\n",
      "--- *blasting region: NRNX1-seg-539\n",
      "--- total time for blast NRNX1-seg-539: 3.901s.\n",
      "-- processing region: NRNX1-seg-54 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-54.fasta\n",
      "--- *blasting region: NRNX1-seg-54\n",
      "--- total time for blast NRNX1-seg-54: 5.330s.\n",
      "-- processing region: NRNX1-seg-540 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-540.fasta\n",
      "--- *blasting region: NRNX1-seg-540\n",
      "--- total time for blast NRNX1-seg-540: 3.249s.\n",
      "-- processing region: NRNX1-seg-541 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-541.fasta\n",
      "--- *blasting region: NRNX1-seg-541\n",
      "--- total time for blast NRNX1-seg-541: 3.187s.\n",
      "-- processing region: NRNX1-seg-542 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-542.fasta\n",
      "--- *blasting region: NRNX1-seg-542\n",
      "--- total time for blast NRNX1-seg-542: 4.156s.\n",
      "-- processing region: NRNX1-seg-543 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-543.fasta\n",
      "--- *blasting region: NRNX1-seg-543\n",
      "--- total time for blast NRNX1-seg-543: 4.856s.\n",
      "-- processing region: NRNX1-seg-544 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-544.fasta\n",
      "--- *blasting region: NRNX1-seg-544\n",
      "--- total time for blast NRNX1-seg-544: 3.300s.\n",
      "-- processing region: NRNX1-seg-545 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-545.fasta\n",
      "--- *blasting region: NRNX1-seg-545\n",
      "--- total time for blast NRNX1-seg-545: 4.130s.\n",
      "-- processing region: NRNX1-seg-546 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-546.fasta\n",
      "--- *blasting region: NRNX1-seg-546\n",
      "--- total time for blast NRNX1-seg-546: 3.825s.\n",
      "-- processing region: NRNX1-seg-547 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-547.fasta\n",
      "--- *blasting region: NRNX1-seg-547\n",
      "--- total time for blast NRNX1-seg-547: 3.400s.\n",
      "-- processing region: NRNX1-seg-548 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-548.fasta\n",
      "--- *blasting region: NRNX1-seg-548\n",
      "--- total time for blast NRNX1-seg-548: 3.263s.\n",
      "-- processing region: NRNX1-seg-549 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-549.fasta\n",
      "--- *blasting region: NRNX1-seg-549\n",
      "--- total time for blast NRNX1-seg-549: 3.184s.\n",
      "-- processing region: NRNX1-seg-55 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-55.fasta\n",
      "--- *blasting region: NRNX1-seg-55\n",
      "--- total time for blast NRNX1-seg-55: 4.003s.\n",
      "-- processing region: NRNX1-seg-550 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-550.fasta\n",
      "--- *blasting region: NRNX1-seg-550\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-550: 3.650s.\n",
      "-- processing region: NRNX1-seg-551 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-551.fasta\n",
      "--- *blasting region: NRNX1-seg-551\n",
      "--- total time for blast NRNX1-seg-551: 2.906s.\n",
      "-- processing region: NRNX1-seg-552 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-552.fasta\n",
      "--- *blasting region: NRNX1-seg-552\n",
      "--- total time for blast NRNX1-seg-552: 4.188s.\n",
      "-- processing region: NRNX1-seg-553 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-553.fasta\n",
      "--- *blasting region: NRNX1-seg-553\n",
      "--- total time for blast NRNX1-seg-553: 4.062s.\n",
      "-- processing region: NRNX1-seg-554 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-554.fasta\n",
      "--- *blasting region: NRNX1-seg-554\n",
      "--- total time for blast NRNX1-seg-554: 4.299s.\n",
      "-- processing region: NRNX1-seg-555 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-555.fasta\n",
      "--- *blasting region: NRNX1-seg-555\n",
      "--- total time for blast NRNX1-seg-555: 4.255s.\n",
      "-- processing region: NRNX1-seg-556 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-556.fasta\n",
      "--- *blasting region: NRNX1-seg-556\n",
      "--- total time for blast NRNX1-seg-556: 4.824s.\n",
      "-- processing region: NRNX1-seg-557 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-557.fasta\n",
      "--- *blasting region: NRNX1-seg-557\n",
      "--- total time for blast NRNX1-seg-557: 5.379s.\n",
      "-- processing region: NRNX1-seg-558 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-558.fasta\n",
      "--- *blasting region: NRNX1-seg-558\n",
      "--- total time for blast NRNX1-seg-558: 4.187s.\n",
      "-- processing region: NRNX1-seg-559 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-559.fasta\n",
      "--- *blasting region: NRNX1-seg-559\n",
      "--- total time for blast NRNX1-seg-559: 7.451s.\n",
      "-- processing region: NRNX1-seg-56 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-56.fasta\n",
      "--- *blasting region: NRNX1-seg-56\n",
      "--- total time for blast NRNX1-seg-56: 3.785s.\n",
      "-- processing region: NRNX1-seg-560 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-560.fasta\n",
      "--- *blasting region: NRNX1-seg-560\n",
      "--- total time for blast NRNX1-seg-560: 4.765s.\n",
      "-- processing region: NRNX1-seg-561 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-561.fasta\n",
      "--- *blasting region: NRNX1-seg-561\n",
      "--- total time for blast NRNX1-seg-561: 3.229s.\n",
      "-- processing region: NRNX1-seg-562 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-562.fasta\n",
      "--- *blasting region: NRNX1-seg-562\n",
      "--- total time for blast NRNX1-seg-562: 3.618s.\n",
      "-- processing region: NRNX1-seg-563 with 43 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-563.fasta\n",
      "--- *blasting region: NRNX1-seg-563\n",
      "--- total time for blast NRNX1-seg-563: 3.416s.\n",
      "-- processing region: NRNX1-seg-564 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-564.fasta\n",
      "--- *blasting region: NRNX1-seg-564\n",
      "--- total time for blast NRNX1-seg-564: 6.539s.\n",
      "-- processing region: NRNX1-seg-565 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-565.fasta\n",
      "--- *blasting region: NRNX1-seg-565\n",
      "--- total time for blast NRNX1-seg-565: 3.313s.\n",
      "-- processing region: NRNX1-seg-566 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-566.fasta\n",
      "--- *blasting region: NRNX1-seg-566\n",
      "--- total time for blast NRNX1-seg-566: 3.246s.\n",
      "-- processing region: NRNX1-seg-567 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-567.fasta\n",
      "--- *blasting region: NRNX1-seg-567\n",
      "--- total time for blast NRNX1-seg-567: 4.211s.\n",
      "-- processing region: NRNX1-seg-568 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-568.fasta\n",
      "--- *blasting region: NRNX1-seg-568\n",
      "--- total time for blast NRNX1-seg-568: 3.747s.\n",
      "-- processing region: NRNX1-seg-569 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-569.fasta\n",
      "--- *blasting region: NRNX1-seg-569\n",
      "--- total time for blast NRNX1-seg-569: 3.925s.\n",
      "-- processing region: NRNX1-seg-57 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-57.fasta\n",
      "--- *blasting region: NRNX1-seg-57\n",
      "--- total time for blast NRNX1-seg-57: 3.941s.\n",
      "-- processing region: NRNX1-seg-570 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-570.fasta\n",
      "--- *blasting region: NRNX1-seg-570\n",
      "--- total time for blast NRNX1-seg-570: 4.276s.\n",
      "-- processing region: NRNX1-seg-571 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-571.fasta\n",
      "--- *blasting region: NRNX1-seg-571\n",
      "--- total time for blast NRNX1-seg-571: 3.481s.\n",
      "-- processing region: NRNX1-seg-572 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-572.fasta\n",
      "--- *blasting region: NRNX1-seg-572\n",
      "--- total time for blast NRNX1-seg-572: 3.525s.\n",
      "-- processing region: NRNX1-seg-573 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-573.fasta\n",
      "--- *blasting region: NRNX1-seg-573\n",
      "--- total time for blast NRNX1-seg-573: 4.301s.\n",
      "-- processing region: NRNX1-seg-574 with 43 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-574.fasta\n",
      "--- *blasting region: NRNX1-seg-574\n",
      "--- total time for blast NRNX1-seg-574: 4.160s.\n",
      "-- processing region: NRNX1-seg-575 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-575.fasta\n",
      "--- *blasting region: NRNX1-seg-575\n",
      "--- total time for blast NRNX1-seg-575: 3.236s.\n",
      "-- processing region: NRNX1-seg-576 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-576.fasta\n",
      "--- *blasting region: NRNX1-seg-576\n",
      "--- total time for blast NRNX1-seg-576: 4.149s.\n",
      "-- processing region: NRNX1-seg-577 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-577.fasta\n",
      "--- *blasting region: NRNX1-seg-577\n",
      "--- total time for blast NRNX1-seg-577: 3.597s.\n",
      "-- processing region: NRNX1-seg-578 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-578.fasta\n",
      "--- *blasting region: NRNX1-seg-578\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-578: 3.155s.\n",
      "-- processing region: NRNX1-seg-579 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-579.fasta\n",
      "--- *blasting region: NRNX1-seg-579\n",
      "--- total time for blast NRNX1-seg-579: 3.011s.\n",
      "-- processing region: NRNX1-seg-58 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-58.fasta\n",
      "--- *blasting region: NRNX1-seg-58\n",
      "--- total time for blast NRNX1-seg-58: 4.958s.\n",
      "-- processing region: NRNX1-seg-580 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-580.fasta\n",
      "--- *blasting region: NRNX1-seg-580\n",
      "--- total time for blast NRNX1-seg-580: 3.531s.\n",
      "-- processing region: NRNX1-seg-581 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-581.fasta\n",
      "--- *blasting region: NRNX1-seg-581\n",
      "--- total time for blast NRNX1-seg-581: 3.749s.\n",
      "-- processing region: NRNX1-seg-582 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-582.fasta\n",
      "--- *blasting region: NRNX1-seg-582\n",
      "--- total time for blast NRNX1-seg-582: 4.196s.\n",
      "-- processing region: NRNX1-seg-583 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-583.fasta\n",
      "--- *blasting region: NRNX1-seg-583\n",
      "--- total time for blast NRNX1-seg-583: 3.617s.\n",
      "-- processing region: NRNX1-seg-584 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-584.fasta\n",
      "--- *blasting region: NRNX1-seg-584\n",
      "--- total time for blast NRNX1-seg-584: 2.639s.\n",
      "-- processing region: NRNX1-seg-585 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-585.fasta\n",
      "--- *blasting region: NRNX1-seg-585\n",
      "--- total time for blast NRNX1-seg-585: 4.120s.\n",
      "-- processing region: NRNX1-seg-586 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-586.fasta\n",
      "--- *blasting region: NRNX1-seg-586\n",
      "--- total time for blast NRNX1-seg-586: 4.287s.\n",
      "-- processing region: NRNX1-seg-587 with 35 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-587.fasta\n",
      "--- *blasting region: NRNX1-seg-587\n",
      "--- total time for blast NRNX1-seg-587: 3.238s.\n",
      "-- processing region: NRNX1-seg-588 with 38 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-588.fasta\n",
      "--- *blasting region: NRNX1-seg-588\n",
      "--- total time for blast NRNX1-seg-588: 3.571s.\n",
      "-- processing region: NRNX1-seg-589 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-589.fasta\n",
      "--- *blasting region: NRNX1-seg-589\n",
      "--- total time for blast NRNX1-seg-589: 5.708s.\n",
      "-- processing region: NRNX1-seg-59 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-59.fasta\n",
      "--- *blasting region: NRNX1-seg-59\n",
      "--- total time for blast NRNX1-seg-59: 3.249s.\n",
      "-- processing region: NRNX1-seg-590 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-590.fasta\n",
      "--- *blasting region: NRNX1-seg-590\n",
      "--- total time for blast NRNX1-seg-590: 4.092s.\n",
      "-- processing region: NRNX1-seg-591 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-591.fasta\n",
      "--- *blasting region: NRNX1-seg-591\n",
      "--- total time for blast NRNX1-seg-591: 11.368s.\n",
      "-- processing region: NRNX1-seg-592 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-592.fasta\n",
      "--- *blasting region: NRNX1-seg-592\n",
      "--- total time for blast NRNX1-seg-592: 4.048s.\n",
      "-- processing region: NRNX1-seg-593 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-593.fasta\n",
      "--- *blasting region: NRNX1-seg-593\n",
      "--- total time for blast NRNX1-seg-593: 4.222s.\n",
      "-- processing region: NRNX1-seg-594 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-594.fasta\n",
      "--- *blasting region: NRNX1-seg-594\n",
      "--- total time for blast NRNX1-seg-594: 3.678s.\n",
      "-- processing region: NRNX1-seg-595 with 43 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-595.fasta\n",
      "--- *blasting region: NRNX1-seg-595\n",
      "--- total time for blast NRNX1-seg-595: 3.554s.\n",
      "-- processing region: NRNX1-seg-596 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-596.fasta\n",
      "--- *blasting region: NRNX1-seg-596\n",
      "--- total time for blast NRNX1-seg-596: 3.299s.\n",
      "-- processing region: NRNX1-seg-597 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-597.fasta\n",
      "--- *blasting region: NRNX1-seg-597\n",
      "--- total time for blast NRNX1-seg-597: 3.122s.\n",
      "-- processing region: NRNX1-seg-598 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-598.fasta\n",
      "--- *blasting region: NRNX1-seg-598\n",
      "--- total time for blast NRNX1-seg-598: 3.503s.\n",
      "-- processing region: NRNX1-seg-599 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-599.fasta\n",
      "--- *blasting region: NRNX1-seg-599\n",
      "--- total time for blast NRNX1-seg-599: 4.193s.\n",
      "-- processing region: NRNX1-seg-6 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-6.fasta\n",
      "--- *blasting region: NRNX1-seg-6\n",
      "--- total time for blast NRNX1-seg-6: 3.883s.\n",
      "-- processing region: NRNX1-seg-60 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-60.fasta\n",
      "--- *blasting region: NRNX1-seg-60\n",
      "--- total time for blast NRNX1-seg-60: 2.080s.\n",
      "-- processing region: NRNX1-seg-600 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-600.fasta\n",
      "--- *blasting region: NRNX1-seg-600\n",
      "--- total time for blast NRNX1-seg-600: 4.000s.\n",
      "-- processing region: NRNX1-seg-601 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-601.fasta\n",
      "--- *blasting region: NRNX1-seg-601\n",
      "--- total time for blast NRNX1-seg-601: 5.540s.\n",
      "-- processing region: NRNX1-seg-602 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-602.fasta\n",
      "--- *blasting region: NRNX1-seg-602\n",
      "--- total time for blast NRNX1-seg-602: 3.076s.\n",
      "-- processing region: NRNX1-seg-603 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-603.fasta\n",
      "--- *blasting region: NRNX1-seg-603\n",
      "--- total time for blast NRNX1-seg-603: 7.425s.\n",
      "-- processing region: NRNX1-seg-604 with 38 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-604.fasta\n",
      "--- *blasting region: NRNX1-seg-604\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-604: 6.430s.\n",
      "-- processing region: NRNX1-seg-605 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-605.fasta\n",
      "--- *blasting region: NRNX1-seg-605\n",
      "--- total time for blast NRNX1-seg-605: 4.218s.\n",
      "-- processing region: NRNX1-seg-606 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-606.fasta\n",
      "--- *blasting region: NRNX1-seg-606\n",
      "--- total time for blast NRNX1-seg-606: 3.543s.\n",
      "-- processing region: NRNX1-seg-607 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-607.fasta\n",
      "--- *blasting region: NRNX1-seg-607\n",
      "--- total time for blast NRNX1-seg-607: 4.442s.\n",
      "-- processing region: NRNX1-seg-608 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-608.fasta\n",
      "--- *blasting region: NRNX1-seg-608\n",
      "--- total time for blast NRNX1-seg-608: 3.811s.\n",
      "-- processing region: NRNX1-seg-609 with 32 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-609.fasta\n",
      "--- *blasting region: NRNX1-seg-609\n",
      "--- total time for blast NRNX1-seg-609: 2.703s.\n",
      "-- processing region: NRNX1-seg-610 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-610.fasta\n",
      "--- *blasting region: NRNX1-seg-610\n",
      "--- total time for blast NRNX1-seg-610: 3.761s.\n",
      "-- processing region: NRNX1-seg-611 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-611.fasta\n",
      "--- *blasting region: NRNX1-seg-611\n",
      "--- total time for blast NRNX1-seg-611: 4.008s.\n",
      "-- processing region: NRNX1-seg-612 with 30 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-612.fasta\n",
      "--- *blasting region: NRNX1-seg-612\n",
      "--- total time for blast NRNX1-seg-612: 2.503s.\n",
      "-- processing region: NRNX1-seg-613 with 28 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-613.fasta\n",
      "--- *blasting region: NRNX1-seg-613\n",
      "--- total time for blast NRNX1-seg-613: 3.473s.\n",
      "-- processing region: NRNX1-seg-614 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-614.fasta\n",
      "--- *blasting region: NRNX1-seg-614\n",
      "--- total time for blast NRNX1-seg-614: 3.671s.\n",
      "-- processing region: NRNX1-seg-615 with 33 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-615.fasta\n",
      "--- *blasting region: NRNX1-seg-615\n",
      "--- total time for blast NRNX1-seg-615: 2.230s.\n",
      "-- processing region: NRNX1-seg-616 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-616.fasta\n",
      "--- *blasting region: NRNX1-seg-616\n",
      "--- total time for blast NRNX1-seg-616: 4.933s.\n",
      "-- processing region: NRNX1-seg-617 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-617.fasta\n",
      "--- *blasting region: NRNX1-seg-617\n",
      "--- total time for blast NRNX1-seg-617: 3.398s.\n",
      "-- processing region: NRNX1-seg-618 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-618.fasta\n",
      "--- *blasting region: NRNX1-seg-618\n",
      "--- total time for blast NRNX1-seg-618: 4.421s.\n",
      "-- processing region: NRNX1-seg-619 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-619.fasta\n",
      "--- *blasting region: NRNX1-seg-619\n",
      "--- total time for blast NRNX1-seg-619: 4.927s.\n",
      "-- processing region: NRNX1-seg-62 with 43 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-62.fasta\n",
      "--- *blasting region: NRNX1-seg-62\n",
      "--- total time for blast NRNX1-seg-62: 3.383s.\n",
      "-- processing region: NRNX1-seg-620 with 75 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-620.fasta\n",
      "--- *blasting region: NRNX1-seg-620\n",
      "--- total time for blast NRNX1-seg-620: 3.992s.\n",
      "-- processing region: NRNX1-seg-621 with 56 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-621.fasta\n",
      "--- *blasting region: NRNX1-seg-621\n",
      "--- total time for blast NRNX1-seg-621: 3.499s.\n",
      "-- processing region: NRNX1-seg-622 with 65 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-622.fasta\n",
      "--- *blasting region: NRNX1-seg-622\n",
      "--- total time for blast NRNX1-seg-622: 4.342s.\n",
      "-- processing region: NRNX1-seg-623 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-623.fasta\n",
      "--- *blasting region: NRNX1-seg-623\n",
      "--- total time for blast NRNX1-seg-623: 3.465s.\n",
      "-- processing region: NRNX1-seg-624 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-624.fasta\n",
      "--- *blasting region: NRNX1-seg-624\n",
      "--- total time for blast NRNX1-seg-624: 5.216s.\n",
      "-- processing region: NRNX1-seg-625 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-625.fasta\n",
      "--- *blasting region: NRNX1-seg-625\n",
      "--- total time for blast NRNX1-seg-625: 4.597s.\n",
      "-- processing region: NRNX1-seg-626 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-626.fasta\n",
      "--- *blasting region: NRNX1-seg-626\n",
      "--- total time for blast NRNX1-seg-626: 3.626s.\n",
      "-- processing region: NRNX1-seg-627 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-627.fasta\n",
      "--- *blasting region: NRNX1-seg-627\n",
      "--- total time for blast NRNX1-seg-627: 5.013s.\n",
      "-- processing region: NRNX1-seg-628 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-628.fasta\n",
      "--- *blasting region: NRNX1-seg-628\n",
      "--- total time for blast NRNX1-seg-628: 3.668s.\n",
      "-- processing region: NRNX1-seg-629 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-629.fasta\n",
      "--- *blasting region: NRNX1-seg-629\n",
      "--- total time for blast NRNX1-seg-629: 3.449s.\n",
      "-- processing region: NRNX1-seg-63 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-63.fasta\n",
      "--- *blasting region: NRNX1-seg-63\n",
      "--- total time for blast NRNX1-seg-63: 4.174s.\n",
      "-- processing region: NRNX1-seg-630 with 34 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-630.fasta\n",
      "--- *blasting region: NRNX1-seg-630\n",
      "--- total time for blast NRNX1-seg-630: 3.054s.\n",
      "-- processing region: NRNX1-seg-631 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-631.fasta\n",
      "--- *blasting region: NRNX1-seg-631\n",
      "--- total time for blast NRNX1-seg-631: 6.069s.\n",
      "-- processing region: NRNX1-seg-632 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-632.fasta\n",
      "--- *blasting region: NRNX1-seg-632\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-632: 3.630s.\n",
      "-- processing region: NRNX1-seg-633 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-633.fasta\n",
      "--- *blasting region: NRNX1-seg-633\n",
      "--- total time for blast NRNX1-seg-633: 3.294s.\n",
      "-- processing region: NRNX1-seg-634 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-634.fasta\n",
      "--- *blasting region: NRNX1-seg-634\n",
      "--- total time for blast NRNX1-seg-634: 5.327s.\n",
      "-- processing region: NRNX1-seg-635 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-635.fasta\n",
      "--- *blasting region: NRNX1-seg-635\n",
      "--- total time for blast NRNX1-seg-635: 3.681s.\n",
      "-- processing region: NRNX1-seg-636 with 32 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-636.fasta\n",
      "--- *blasting region: NRNX1-seg-636\n",
      "--- total time for blast NRNX1-seg-636: 3.505s.\n",
      "-- processing region: NRNX1-seg-637 with 22 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-637.fasta\n",
      "--- *blasting region: NRNX1-seg-637\n",
      "--- total time for blast NRNX1-seg-637: 7.287s.\n",
      "-- processing region: NRNX1-seg-639 with 32 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-639.fasta\n",
      "--- *blasting region: NRNX1-seg-639\n",
      "--- total time for blast NRNX1-seg-639: 9.119s.\n",
      "-- processing region: NRNX1-seg-64 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-64.fasta\n",
      "--- *blasting region: NRNX1-seg-64\n",
      "--- total time for blast NRNX1-seg-64: 3.009s.\n",
      "-- processing region: NRNX1-seg-640 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-640.fasta\n",
      "--- *blasting region: NRNX1-seg-640\n",
      "--- total time for blast NRNX1-seg-640: 3.801s.\n",
      "-- processing region: NRNX1-seg-641 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-641.fasta\n",
      "--- *blasting region: NRNX1-seg-641\n",
      "--- total time for blast NRNX1-seg-641: 4.315s.\n",
      "-- processing region: NRNX1-seg-642 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-642.fasta\n",
      "--- *blasting region: NRNX1-seg-642\n",
      "--- total time for blast NRNX1-seg-642: 3.475s.\n",
      "-- processing region: NRNX1-seg-643 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-643.fasta\n",
      "--- *blasting region: NRNX1-seg-643\n",
      "--- total time for blast NRNX1-seg-643: 3.943s.\n",
      "-- processing region: NRNX1-seg-644 with 58 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-644.fasta\n",
      "--- *blasting region: NRNX1-seg-644\n",
      "--- total time for blast NRNX1-seg-644: 3.978s.\n",
      "-- processing region: NRNX1-seg-645 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-645.fasta\n",
      "--- *blasting region: NRNX1-seg-645\n",
      "--- total time for blast NRNX1-seg-645: 5.727s.\n",
      "-- processing region: NRNX1-seg-646 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-646.fasta\n",
      "--- *blasting region: NRNX1-seg-646\n",
      "--- total time for blast NRNX1-seg-646: 3.936s.\n",
      "-- processing region: NRNX1-seg-647 with 33 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-647.fasta\n",
      "--- *blasting region: NRNX1-seg-647\n",
      "--- total time for blast NRNX1-seg-647: 9.174s.\n",
      "-- processing region: NRNX1-seg-648 with 41 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-648.fasta\n",
      "--- *blasting region: NRNX1-seg-648\n",
      "--- total time for blast NRNX1-seg-648: 3.704s.\n",
      "-- processing region: NRNX1-seg-649 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-649.fasta\n",
      "--- *blasting region: NRNX1-seg-649\n",
      "--- total time for blast NRNX1-seg-649: 3.506s.\n",
      "-- processing region: NRNX1-seg-65 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-65.fasta\n",
      "--- *blasting region: NRNX1-seg-65\n",
      "--- total time for blast NRNX1-seg-65: 2.427s.\n",
      "-- processing region: NRNX1-seg-650 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-650.fasta\n",
      "--- *blasting region: NRNX1-seg-650\n",
      "--- total time for blast NRNX1-seg-650: 2.896s.\n",
      "-- processing region: NRNX1-seg-651 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-651.fasta\n",
      "--- *blasting region: NRNX1-seg-651\n",
      "--- total time for blast NRNX1-seg-651: 3.511s.\n",
      "-- processing region: NRNX1-seg-652 with 35 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-652.fasta\n",
      "--- *blasting region: NRNX1-seg-652\n",
      "--- total time for blast NRNX1-seg-652: 3.946s.\n",
      "-- processing region: NRNX1-seg-653 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-653.fasta\n",
      "--- *blasting region: NRNX1-seg-653\n",
      "--- total time for blast NRNX1-seg-653: 3.969s.\n",
      "-- processing region: NRNX1-seg-654 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-654.fasta\n",
      "--- *blasting region: NRNX1-seg-654\n",
      "--- total time for blast NRNX1-seg-654: 4.379s.\n",
      "-- processing region: NRNX1-seg-655 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-655.fasta\n",
      "--- *blasting region: NRNX1-seg-655\n",
      "--- total time for blast NRNX1-seg-655: 3.836s.\n",
      "-- processing region: NRNX1-seg-656 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-656.fasta\n",
      "--- *blasting region: NRNX1-seg-656\n",
      "--- total time for blast NRNX1-seg-656: 17.038s.\n",
      "-- processing region: NRNX1-seg-657 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-657.fasta\n",
      "--- *blasting region: NRNX1-seg-657\n",
      "--- total time for blast NRNX1-seg-657: 4.337s.\n",
      "-- processing region: NRNX1-seg-658 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-658.fasta\n",
      "--- *blasting region: NRNX1-seg-658\n",
      "--- total time for blast NRNX1-seg-658: 4.491s.\n",
      "-- processing region: NRNX1-seg-659 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-659.fasta\n",
      "--- *blasting region: NRNX1-seg-659\n",
      "--- total time for blast NRNX1-seg-659: 4.251s.\n",
      "-- processing region: NRNX1-seg-66 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-66.fasta\n",
      "--- *blasting region: NRNX1-seg-66\n",
      "--- total time for blast NRNX1-seg-66: 3.805s.\n",
      "-- processing region: NRNX1-seg-660 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-660.fasta\n",
      "--- *blasting region: NRNX1-seg-660\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-660: 3.856s.\n",
      "-- processing region: NRNX1-seg-661 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-661.fasta\n",
      "--- *blasting region: NRNX1-seg-661\n",
      "--- total time for blast NRNX1-seg-661: 3.951s.\n",
      "-- processing region: NRNX1-seg-662 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-662.fasta\n",
      "--- *blasting region: NRNX1-seg-662\n",
      "--- total time for blast NRNX1-seg-662: 3.830s.\n",
      "-- processing region: NRNX1-seg-665 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-665.fasta\n",
      "--- *blasting region: NRNX1-seg-665\n",
      "--- total time for blast NRNX1-seg-665: 4.835s.\n",
      "-- processing region: NRNX1-seg-666 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-666.fasta\n",
      "--- *blasting region: NRNX1-seg-666\n",
      "--- total time for blast NRNX1-seg-666: 4.788s.\n",
      "-- processing region: NRNX1-seg-667 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-667.fasta\n",
      "--- *blasting region: NRNX1-seg-667\n",
      "--- total time for blast NRNX1-seg-667: 3.030s.\n",
      "-- processing region: NRNX1-seg-668 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-668.fasta\n",
      "--- *blasting region: NRNX1-seg-668\n",
      "--- total time for blast NRNX1-seg-668: 3.687s.\n",
      "-- processing region: NRNX1-seg-669 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-669.fasta\n",
      "--- *blasting region: NRNX1-seg-669\n",
      "--- total time for blast NRNX1-seg-669: 3.159s.\n",
      "-- processing region: NRNX1-seg-67 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-67.fasta\n",
      "--- *blasting region: NRNX1-seg-67\n",
      "--- total time for blast NRNX1-seg-67: 12.931s.\n",
      "-- processing region: NRNX1-seg-670 with 33 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-670.fasta\n",
      "--- *blasting region: NRNX1-seg-670\n",
      "--- total time for blast NRNX1-seg-670: 4.179s.\n",
      "-- processing region: NRNX1-seg-671 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-671.fasta\n",
      "--- *blasting region: NRNX1-seg-671\n",
      "--- total time for blast NRNX1-seg-671: 4.163s.\n",
      "-- processing region: NRNX1-seg-672 with 38 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-672.fasta\n",
      "--- *blasting region: NRNX1-seg-672\n",
      "--- total time for blast NRNX1-seg-672: 6.849s.\n",
      "-- processing region: NRNX1-seg-674 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-674.fasta\n",
      "--- *blasting region: NRNX1-seg-674\n",
      "--- total time for blast NRNX1-seg-674: 3.652s.\n",
      "-- processing region: NRNX1-seg-675 with 43 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-675.fasta\n",
      "--- *blasting region: NRNX1-seg-675\n",
      "--- total time for blast NRNX1-seg-675: 3.844s.\n",
      "-- processing region: NRNX1-seg-676 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-676.fasta\n",
      "--- *blasting region: NRNX1-seg-676\n",
      "--- total time for blast NRNX1-seg-676: 6.332s.\n",
      "-- processing region: NRNX1-seg-678 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-678.fasta\n",
      "--- *blasting region: NRNX1-seg-678\n",
      "--- total time for blast NRNX1-seg-678: 5.130s.\n",
      "-- processing region: NRNX1-seg-679 with 22 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-679.fasta\n",
      "--- *blasting region: NRNX1-seg-679\n",
      "--- total time for blast NRNX1-seg-679: 7.023s.\n",
      "-- processing region: NRNX1-seg-68 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-68.fasta\n",
      "--- *blasting region: NRNX1-seg-68\n",
      "--- total time for blast NRNX1-seg-68: 2.947s.\n",
      "-- processing region: NRNX1-seg-681 with 31 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-681.fasta\n",
      "--- *blasting region: NRNX1-seg-681\n",
      "--- total time for blast NRNX1-seg-681: 5.305s.\n",
      "-- processing region: NRNX1-seg-682 with 39 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-682.fasta\n",
      "--- *blasting region: NRNX1-seg-682\n",
      "--- total time for blast NRNX1-seg-682: 3.445s.\n",
      "-- processing region: NRNX1-seg-683 with 53 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-683.fasta\n",
      "--- *blasting region: NRNX1-seg-683\n",
      "--- total time for blast NRNX1-seg-683: 3.672s.\n",
      "-- processing region: NRNX1-seg-684 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-684.fasta\n",
      "--- *blasting region: NRNX1-seg-684\n",
      "--- total time for blast NRNX1-seg-684: 2.553s.\n",
      "-- processing region: NRNX1-seg-685 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-685.fasta\n",
      "--- *blasting region: NRNX1-seg-685\n",
      "--- total time for blast NRNX1-seg-685: 3.623s.\n",
      "-- processing region: NRNX1-seg-686 with 28 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-686.fasta\n",
      "--- *blasting region: NRNX1-seg-686\n",
      "--- total time for blast NRNX1-seg-686: 10.265s.\n",
      "-- processing region: NRNX1-seg-687 with 27 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-687.fasta\n",
      "--- *blasting region: NRNX1-seg-687\n",
      "--- total time for blast NRNX1-seg-687: 4.112s.\n",
      "-- processing region: NRNX1-seg-688 with 28 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-688.fasta\n",
      "--- *blasting region: NRNX1-seg-688\n",
      "--- total time for blast NRNX1-seg-688: 3.862s.\n",
      "-- processing region: NRNX1-seg-689 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-689.fasta\n",
      "--- *blasting region: NRNX1-seg-689\n",
      "--- total time for blast NRNX1-seg-689: 7.453s.\n",
      "-- processing region: NRNX1-seg-69 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-69.fasta\n",
      "--- *blasting region: NRNX1-seg-69\n",
      "--- total time for blast NRNX1-seg-69: 3.222s.\n",
      "-- processing region: NRNX1-seg-690 with 43 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-690.fasta\n",
      "--- *blasting region: NRNX1-seg-690\n",
      "--- total time for blast NRNX1-seg-690: 3.321s.\n",
      "-- processing region: NRNX1-seg-691 with 36 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-691.fasta\n",
      "--- *blasting region: NRNX1-seg-691\n",
      "--- total time for blast NRNX1-seg-691: 5.131s.\n",
      "-- processing region: NRNX1-seg-692 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-692.fasta\n",
      "--- *blasting region: NRNX1-seg-692\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-692: 3.738s.\n",
      "-- processing region: NRNX1-seg-693 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-693.fasta\n",
      "--- *blasting region: NRNX1-seg-693\n",
      "--- total time for blast NRNX1-seg-693: 3.407s.\n",
      "-- processing region: NRNX1-seg-694 with 49 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-694.fasta\n",
      "--- *blasting region: NRNX1-seg-694\n",
      "--- total time for blast NRNX1-seg-694: 5.311s.\n",
      "-- processing region: NRNX1-seg-695 with 57 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-695.fasta\n",
      "--- *blasting region: NRNX1-seg-695\n",
      "--- total time for blast NRNX1-seg-695: 4.652s.\n",
      "-- processing region: NRNX1-seg-696 with 40 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-696.fasta\n",
      "--- *blasting region: NRNX1-seg-696\n",
      "--- total time for blast NRNX1-seg-696: 4.317s.\n",
      "-- processing region: NRNX1-seg-697 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-697.fasta\n",
      "--- *blasting region: NRNX1-seg-697\n",
      "--- total time for blast NRNX1-seg-697: 3.358s.\n",
      "-- processing region: NRNX1-seg-698 with 61 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-698.fasta\n",
      "--- *blasting region: NRNX1-seg-698\n",
      "--- total time for blast NRNX1-seg-698: 6.916s.\n",
      "-- processing region: NRNX1-seg-699 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-699.fasta\n",
      "--- *blasting region: NRNX1-seg-699\n",
      "--- total time for blast NRNX1-seg-699: 2.784s.\n",
      "-- processing region: NRNX1-seg-7 with 34 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-7.fasta\n",
      "--- *blasting region: NRNX1-seg-7\n",
      "--- total time for blast NRNX1-seg-7: 2.671s.\n",
      "-- processing region: NRNX1-seg-70 with 42 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-70.fasta\n",
      "--- *blasting region: NRNX1-seg-70\n",
      "--- total time for blast NRNX1-seg-70: 4.099s.\n",
      "-- processing region: NRNX1-seg-71 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-71.fasta\n",
      "--- *blasting region: NRNX1-seg-71\n",
      "--- total time for blast NRNX1-seg-71: 3.905s.\n",
      "-- processing region: NRNX1-seg-72 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-72.fasta\n",
      "--- *blasting region: NRNX1-seg-72\n",
      "--- total time for blast NRNX1-seg-72: 3.355s.\n",
      "-- processing region: NRNX1-seg-73 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-73.fasta\n",
      "--- *blasting region: NRNX1-seg-73\n",
      "--- total time for blast NRNX1-seg-73: 3.313s.\n",
      "-- processing region: NRNX1-seg-74 with 36 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-74.fasta\n",
      "--- *blasting region: NRNX1-seg-74\n",
      "--- total time for blast NRNX1-seg-74: 3.289s.\n",
      "-- processing region: NRNX1-seg-75 with 28 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-75.fasta\n",
      "--- *blasting region: NRNX1-seg-75\n",
      "--- total time for blast NRNX1-seg-75: 6.095s.\n",
      "-- processing region: NRNX1-seg-76 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-76.fasta\n",
      "--- *blasting region: NRNX1-seg-76\n",
      "--- total time for blast NRNX1-seg-76: 3.196s.\n",
      "-- processing region: NRNX1-seg-77 with 34 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-77.fasta\n",
      "--- *blasting region: NRNX1-seg-77\n",
      "--- total time for blast NRNX1-seg-77: 5.057s.\n",
      "-- processing region: NRNX1-seg-79 with 32 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-79.fasta\n",
      "--- *blasting region: NRNX1-seg-79\n",
      "--- total time for blast NRNX1-seg-79: 3.583s.\n",
      "-- processing region: NRNX1-seg-80 with 51 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-80.fasta\n",
      "--- *blasting region: NRNX1-seg-80\n",
      "--- total time for blast NRNX1-seg-80: 3.505s.\n",
      "-- processing region: NRNX1-seg-81 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-81.fasta\n",
      "--- *blasting region: NRNX1-seg-81\n",
      "--- total time for blast NRNX1-seg-81: 3.353s.\n",
      "-- processing region: NRNX1-seg-82 with 48 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-82.fasta\n",
      "--- *blasting region: NRNX1-seg-82\n",
      "--- total time for blast NRNX1-seg-82: 9.671s.\n",
      "-- processing region: NRNX1-seg-83 with 63 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-83.fasta\n",
      "--- *blasting region: NRNX1-seg-83\n",
      "--- total time for blast NRNX1-seg-83: 4.333s.\n",
      "-- processing region: NRNX1-seg-84 with 69 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-84.fasta\n",
      "--- *blasting region: NRNX1-seg-84\n",
      "--- total time for blast NRNX1-seg-84: 4.042s.\n",
      "-- processing region: NRNX1-seg-85 with 55 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-85.fasta\n",
      "--- *blasting region: NRNX1-seg-85\n",
      "--- total time for blast NRNX1-seg-85: 3.841s.\n",
      "-- processing region: NRNX1-seg-86 with 46 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-86.fasta\n",
      "--- *blasting region: NRNX1-seg-86\n",
      "--- total time for blast NRNX1-seg-86: 3.491s.\n",
      "-- processing region: NRNX1-seg-87 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-87.fasta\n",
      "--- *blasting region: NRNX1-seg-87\n",
      "--- total time for blast NRNX1-seg-87: 4.313s.\n",
      "-- processing region: NRNX1-seg-88 with 47 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-88.fasta\n",
      "--- *blasting region: NRNX1-seg-88\n",
      "--- total time for blast NRNX1-seg-88: 3.774s.\n",
      "-- processing region: NRNX1-seg-89 with 77 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-89.fasta\n",
      "--- *blasting region: NRNX1-seg-89\n",
      "--- total time for blast NRNX1-seg-89: 7.532s.\n",
      "-- processing region: NRNX1-seg-9 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-9.fasta\n",
      "--- *blasting region: NRNX1-seg-9\n",
      "--- total time for blast NRNX1-seg-9: 6.968s.\n",
      "-- processing region: NRNX1-seg-90 with 52 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-90.fasta\n",
      "--- *blasting region: NRNX1-seg-90\n",
      "--- total time for blast NRNX1-seg-90: 3.815s.\n",
      "-- processing region: NRNX1-seg-91 with 50 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-91.fasta\n",
      "--- *blasting region: NRNX1-seg-91\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-91: 3.394s.\n",
      "-- processing region: NRNX1-seg-92 with 38 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-92.fasta\n",
      "--- *blasting region: NRNX1-seg-92\n",
      "--- total time for blast NRNX1-seg-92: 3.221s.\n",
      "-- processing region: NRNX1-seg-93 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-93.fasta\n",
      "--- *blasting region: NRNX1-seg-93\n",
      "--- total time for blast NRNX1-seg-93: 4.483s.\n",
      "-- processing region: NRNX1-seg-94 with 45 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-94.fasta\n",
      "--- *blasting region: NRNX1-seg-94\n",
      "--- total time for blast NRNX1-seg-94: 3.529s.\n",
      "-- processing region: NRNX1-seg-95 with 29 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-95.fasta\n",
      "--- *blasting region: NRNX1-seg-95\n",
      "--- total time for blast NRNX1-seg-95: 2.358s.\n",
      "-- processing region: NRNX1-seg-96 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-96.fasta\n",
      "--- *blasting region: NRNX1-seg-96\n",
      "--- total time for blast NRNX1-seg-96: 3.960s.\n",
      "-- processing region: NRNX1-seg-97 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-97.fasta\n",
      "--- *blasting region: NRNX1-seg-97\n",
      "--- total time for blast NRNX1-seg-97: 3.866s.\n",
      "-- processing region: NRNX1-seg-98 with 37 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-98.fasta\n",
      "--- *blasting region: NRNX1-seg-98\n",
      "--- total time for blast NRNX1-seg-98: 3.530s.\n",
      "-- processing region: NRNX1-seg-99 with 32 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\blast\\probe_gene_NRNX1-seg-99.fasta\n",
      "--- *blasting region: NRNX1-seg-99\n",
      "--- total time for blast NRNX1-seg-99: 6.595s.\n"
     ]
    }
   ],
   "source": [
    "blast_database = os.path.join(reference_folder, 'blast_database', 'hg38')\n",
    "\n",
    "check.Blast_probes(kept_records, library_folder, num_threads=44, \n",
    "                   blast_db=blast_database, blast_evalue=100, overwrite=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7c6a9506",
   "metadata": {},
   "source": [
    "### 4.2.1 screen probe by blast results\n",
    "\n",
    "This step has 2 different cutoffs:\n",
    "\n",
    "hard_cutoff: whether a probe has any ~40mer hit somewhere else, if so just dump the probe\n",
    "\n",
    "soft_cutoff: whether a probe has a lot of ~20mer off-target hits, if exceed certain amount then dump the probe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 77,
   "id": "6f6749b3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Number of probes loaded: 34067\n",
      "- Number of regions in this library: 668\n",
      "-- checking probes in region: NRNX1-seg-1\n",
      "--- gene=NRNX1-seg-1, id=6 removed by soft count = 240\n",
      "--- gene=NRNX1-seg-1, id=8 removed by soft count = 135\n",
      "--- number of probes: 36 , kept by blast: 34 , if remove dups: 34\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: NRNX1-seg-2\n",
      "--- gene=NRNX1-seg-2, id=13 removed by soft count = 52\n",
      "--- gene=NRNX1-seg-2, id=18 removed by soft count = 87\n",
      "--- number of probes: 40 , kept by blast: 38 , if remove dups: 38\n",
      "-- number of probes kept for this region: 38\n",
      "-- checking probes in region: NRNX1-seg-3\n",
      "--- gene=NRNX1-seg-3, id=0 removed by soft count = 68\n",
      "--- gene=NRNX1-seg-3, id=1 removed by soft count = 102\n",
      "--- gene=NRNX1-seg-3, id=22 removed by soft count = 215\n",
      "--- gene=NRNX1-seg-3, id=27 removed by soft count = 349\n",
      "--- number of probes: 40 , kept by blast: 36 , if remove dups: 36\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: NRNX1-seg-4\n",
      "--- gene=NRNX1-seg-4, id=40 removed by soft count = 211\n",
      "--- gene=NRNX1-seg-4, id=41 removed by soft count = 234\n",
      "--- gene=NRNX1-seg-4, id=42 removed by soft count = 334\n",
      "--- number of probes: 53 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-5\n",
      "--- number of probes: 64 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-6\n",
      "--- gene=NRNX1-seg-6, id=36 removed by soft count = 56\n",
      "--- gene=NRNX1-seg-6, id=50 removed by soft count = 90\n",
      "--- number of probes: 53 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: NRNX1-seg-7\n",
      "--- gene=NRNX1-seg-7, id=22 removed by soft count = 120\n",
      "--- number of probes: 34 , kept by blast: 33 , if remove dups: 33\n",
      "-- number of probes kept for this region: 33\n",
      "-- checking probes in region: NRNX1-seg-9\n",
      "--- gene=NRNX1-seg-9, id=18 removed by soft count = 1420\n",
      "--- gene=NRNX1-seg-9, id=20 removed by soft count = 2692\n",
      "--- gene=NRNX1-seg-9, id=45 removed by soft count = 75\n",
      "--- number of probes: 54 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: NRNX1-seg-10\n",
      "--- number of probes: 66 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-13\n",
      "--- gene=NRNX1-seg-13, id=0 removed by soft count = 60\n",
      "--- gene=NRNX1-seg-13, id=1 removed by soft count = 60\n",
      "--- gene=NRNX1-seg-13, id=2 removed by soft count = 100\n",
      "--- gene=NRNX1-seg-13, id=5 removed by soft count = 83\n",
      "--- gene=NRNX1-seg-13, id=7 removed by soft count = 77\n",
      "--- gene=NRNX1-seg-13, id=8 removed by soft count = 131\n",
      "--- gene=NRNX1-seg-13, id=10 removed by soft count = 60\n",
      "--- gene=NRNX1-seg-13, id=11 removed by soft count = 160\n",
      "--- gene=NRNX1-seg-13, id=12 removed by soft count = 272\n",
      "--- gene=NRNX1-seg-13, id=13 removed by soft count = 1367\n",
      "--- number of probes: 55 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-14\n",
      "--- gene=NRNX1-seg-14, id=27 removed by soft count = 62\n",
      "--- gene=NRNX1-seg-14, id=36 removed by soft count = 59\n",
      "--- number of probes: 67 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-15\n",
      "--- number of probes: 78 , kept by blast: 78 , if remove dups: 78\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-16\n",
      "--- number of probes: 70 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-17\n",
      "--- gene=NRNX1-seg-17, id=5 removed by soft count = 125\n",
      "--- gene=NRNX1-seg-17, id=9 removed by soft count = 166\n",
      "--- number of probes: 54 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-18\n",
      "--- gene=NRNX1-seg-18, id=7 removed by soft count = 1175\n",
      "--- gene=NRNX1-seg-18, id=9 removed by soft count = 248\n",
      "--- number of probes: 36 , kept by blast: 34 , if remove dups: 34\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: NRNX1-seg-19\n",
      "--- number of probes: 72 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-20\n",
      "--- number of probes: 45 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-21\n",
      "--- gene=NRNX1-seg-21, id=2 removed by soft count = 312\n",
      "--- gene=NRNX1-seg-21, id=14 removed by soft count = 83\n",
      "--- gene=NRNX1-seg-21, id=19 removed by soft count = 158\n",
      "--- gene=NRNX1-seg-21, id=23 removed by soft count = 85\n",
      "--- number of probes: 42 , kept by blast: 38 , if remove dups: 38\n",
      "-- number of probes kept for this region: 38\n",
      "-- checking probes in region: NRNX1-seg-22\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-23\n",
      "--- gene=NRNX1-seg-23, id=37 removed by soft count = 87\n",
      "--- number of probes: 70 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-24\n",
      "--- gene=NRNX1-seg-24, id=43 removed by soft count = 625\n",
      "--- gene=NRNX1-seg-24, id=52 removed by soft count = 233\n",
      "--- number of probes: 57 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-25\n",
      "--- gene=NRNX1-seg-25, id=27 removed by soft count = 176\n",
      "--- gene=NRNX1-seg-25, id=32 removed by soft count = 805\n",
      "--- number of probes: 45 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-26\n",
      "--- number of probes: 55 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-27\n",
      "--- number of probes: 50 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-28\n",
      "--- gene=NRNX1-seg-28, id=32 removed by soft count = 163\n",
      "--- gene=NRNX1-seg-28, id=34 removed by soft count = 151\n",
      "--- gene=NRNX1-seg-28, id=35 removed by soft count = 117\n",
      "--- number of probes: 50 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-30\n",
      "--- gene=NRNX1-seg-30, id=14 removed by soft count = 155\n",
      "--- gene=NRNX1-seg-30, id=30 removed by soft count = 71\n",
      "--- gene=NRNX1-seg-30, id=36 removed by soft count = 52\n",
      "--- number of probes: 47 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-31\n",
      "--- gene=NRNX1-seg-31, id=18 removed by soft count = 65\n",
      "--- gene=NRNX1-seg-31, id=19 removed by soft count = 104\n",
      "--- gene=NRNX1-seg-31, id=20 removed by soft count = 138\n",
      "--- number of probes: 41 , kept by blast: 38 , if remove dups: 38\n",
      "-- number of probes kept for this region: 38\n",
      "-- checking probes in region: NRNX1-seg-32\n",
      "--- gene=NRNX1-seg-32, id=37 removed by soft count = 103\n",
      "--- gene=NRNX1-seg-32, id=38 removed by soft count = 585\n",
      "--- gene=NRNX1-seg-32, id=41 removed by soft count = 276\n",
      "--- gene=NRNX1-seg-32, id=42 removed by soft count = 746\n",
      "--- gene=NRNX1-seg-32, id=43 removed by soft count = 364\n",
      "--- gene=NRNX1-seg-32, id=45 removed by soft count = 1250\n",
      "--- gene=NRNX1-seg-32, id=46 removed by soft count = 232\n",
      "--- gene=NRNX1-seg-32, id=47 removed by soft count = 253\n",
      "--- number of probes: 49 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: NRNX1-seg-33\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=NRNX1-seg-33, id=0 removed by soft count = 467\n",
      "--- gene=NRNX1-seg-33, id=1 removed by soft count = 455\n",
      "--- gene=NRNX1-seg-33, id=2 removed by soft count = 297\n",
      "--- gene=NRNX1-seg-33, id=3 removed by soft count = 216\n",
      "--- gene=NRNX1-seg-33, id=4 removed by soft count = 1573\n",
      "--- number of probes: 34 , kept by blast: 29 , if remove dups: 29\n",
      "-- number of probes kept for this region: 29\n",
      "-- checking probes in region: NRNX1-seg-34\n",
      "--- gene=NRNX1-seg-34, id=22 removed by soft count = 127\n",
      "--- gene=NRNX1-seg-34, id=25 removed by soft count = 112\n",
      "--- gene=NRNX1-seg-34, id=33 removed by soft count = 57\n",
      "--- gene=NRNX1-seg-34, id=35 removed by soft count = 279\n",
      "--- number of probes: 43 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-35\n",
      "--- gene=NRNX1-seg-35, id=28 removed by soft count = 178\n",
      "--- number of probes: 42 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: NRNX1-seg-36\n",
      "--- gene=NRNX1-seg-36, id=8 removed by soft count = 269\n",
      "--- gene=NRNX1-seg-36, id=10 removed by soft count = 210\n",
      "--- gene=NRNX1-seg-36, id=11 removed by soft count = 262\n",
      "--- gene=NRNX1-seg-36, id=18 removed by soft count = 303\n",
      "--- gene=NRNX1-seg-36, id=32 removed by soft count = 67\n",
      "--- gene=NRNX1-seg-36, id=33 removed by soft count = 65\n",
      "--- gene=NRNX1-seg-36, id=35 removed by soft count = 104\n",
      "--- gene=NRNX1-seg-36, id=37 removed by soft count = 75\n",
      "--- gene=NRNX1-seg-36, id=38 removed by soft count = 685\n",
      "--- number of probes: 53 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-37\n",
      "--- gene=NRNX1-seg-37, id=4 removed by soft count = 67\n",
      "--- gene=NRNX1-seg-37, id=5 removed by soft count = 105\n",
      "--- gene=NRNX1-seg-37, id=40 removed by soft count = 3418\n",
      "--- number of probes: 44 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: NRNX1-seg-38\n",
      "--- gene=NRNX1-seg-38, id=17 removed by soft count = 158\n",
      "--- gene=NRNX1-seg-38, id=21 removed by soft count = 156\n",
      "--- gene=NRNX1-seg-38, id=25 removed by soft count = 811\n",
      "--- gene=NRNX1-seg-38, id=43 removed by soft count = 166\n",
      "--- gene=NRNX1-seg-38, id=45 removed by soft count = 103\n",
      "--- gene=NRNX1-seg-38, id=46 removed by soft count = 349\n",
      "--- gene=NRNX1-seg-38, id=47 removed by soft count = 161\n",
      "--- number of probes: 48 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: NRNX1-seg-39\n",
      "--- gene=NRNX1-seg-39, id=0 removed by soft count = 115\n",
      "--- gene=NRNX1-seg-39, id=1 removed by soft count = 89\n",
      "--- number of probes: 47 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-40\n",
      "--- gene=NRNX1-seg-40, id=41 removed by soft count = 55\n",
      "--- gene=NRNX1-seg-40, id=42 removed by soft count = 62\n",
      "--- number of probes: 70 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-41\n",
      "--- gene=NRNX1-seg-41, id=30 removed by soft count = 70\n",
      "--- number of probes: 55 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-42\n",
      "--- number of probes: 61 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-43\n",
      "--- gene=NRNX1-seg-43, id=28 removed by soft count = 78\n",
      "--- number of probes: 69 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-44\n",
      "--- gene=NRNX1-seg-44, id=5 removed by soft count = 59\n",
      "--- gene=NRNX1-seg-44, id=9 removed by soft count = 154\n",
      "--- gene=NRNX1-seg-44, id=25 removed by soft count = 189\n",
      "--- gene=NRNX1-seg-44, id=27 removed by soft count = 97\n",
      "--- number of probes: 54 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-45\n",
      "--- gene=NRNX1-seg-45, id=24 removed by soft count = 1306\n",
      "--- number of probes: 50 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-46\n",
      "--- number of probes: 42 , kept by blast: 42 , if remove dups: 42\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: NRNX1-seg-47\n",
      "--- number of probes: 41 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: NRNX1-seg-48\n",
      "--- gene=NRNX1-seg-48, id=27 removed by soft count = 327\n",
      "--- gene=NRNX1-seg-48, id=28 removed by soft count = 133\n",
      "--- gene=NRNX1-seg-48, id=29 removed by soft count = 86\n",
      "--- gene=NRNX1-seg-48, id=30 removed by soft count = 54\n",
      "--- gene=NRNX1-seg-48, id=31 removed by soft count = 156\n",
      "--- number of probes: 40 , kept by blast: 35 , if remove dups: 35\n",
      "-- number of probes kept for this region: 35\n",
      "-- checking probes in region: NRNX1-seg-49\n",
      "--- gene=NRNX1-seg-49, id=10 removed by soft count = 485\n",
      "--- number of probes: 58 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-50\n",
      "--- gene=NRNX1-seg-50, id=6 removed by soft count = 273\n",
      "--- gene=NRNX1-seg-50, id=8 removed by soft count = 73\n",
      "--- gene=NRNX1-seg-50, id=24 removed by soft count = 4130\n",
      "--- gene=NRNX1-seg-50, id=25 removed by soft count = 61\n",
      "--- number of probes: 50 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-51\n",
      "--- gene=NRNX1-seg-51, id=7 removed by soft count = 375\n",
      "--- gene=NRNX1-seg-51, id=8 removed by soft count = 67\n",
      "--- gene=NRNX1-seg-51, id=9 removed by soft count = 100\n",
      "--- gene=NRNX1-seg-51, id=50 removed by soft count = 71\n",
      "--- gene=NRNX1-seg-51, id=51 removed by soft count = 127\n",
      "--- number of probes: 52 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-52\n",
      "--- gene=NRNX1-seg-52, id=58 removed by soft count = 179\n",
      "--- number of probes: 59 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-53\n",
      "--- gene=NRNX1-seg-53, id=0 removed by soft count = 130\n",
      "--- gene=NRNX1-seg-53, id=19 removed by soft count = 285\n",
      "--- gene=NRNX1-seg-53, id=21 removed by soft count = 439\n",
      "--- number of probes: 53 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-54\n",
      "--- gene=NRNX1-seg-54, id=21 removed by soft count = 250\n",
      "--- gene=NRNX1-seg-54, id=23 removed by soft count = 118\n",
      "--- gene=NRNX1-seg-54, id=24 removed by soft count = 804\n",
      "--- gene=NRNX1-seg-54, id=25 removed by soft count = 58\n",
      "--- number of probes: 66 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-55\n",
      "--- number of probes: 57 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-56\n",
      "--- gene=NRNX1-seg-56, id=3 removed by soft count = 204\n",
      "--- gene=NRNX1-seg-56, id=25 removed by soft count = 134\n",
      "--- number of probes: 44 , kept by blast: 42 , if remove dups: 42\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: NRNX1-seg-57\n",
      "--- number of probes: 54 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-58\n",
      "--- gene=NRNX1-seg-58, id=0 removed by soft count = 449\n",
      "--- gene=NRNX1-seg-58, id=18 removed by soft count = 72\n",
      "--- gene=NRNX1-seg-58, id=20 removed by soft count = 254\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=NRNX1-seg-58, id=31 removed by soft count = 724\n",
      "--- gene=NRNX1-seg-58, id=35 removed by soft count = 90\n",
      "--- gene=NRNX1-seg-58, id=39 removed by soft count = 109\n",
      "--- number of probes: 48 , kept by blast: 42 , if remove dups: 42\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: NRNX1-seg-59\n",
      "--- gene=NRNX1-seg-59, id=27 removed by soft count = 52\n",
      "--- number of probes: 48 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-60\n",
      "--- number of probes: 23 , kept by blast: 23 , if remove dups: 23\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: NRNX1-seg-62\n",
      "--- number of probes: 43 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-63\n",
      "--- gene=NRNX1-seg-63, id=0 removed by soft count = 136\n",
      "--- number of probes: 57 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-64\n",
      "--- gene=NRNX1-seg-64, id=32 removed by soft count = 139\n",
      "--- number of probes: 44 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-65\n",
      "--- gene=NRNX1-seg-65, id=17 removed by soft count = 183\n",
      "--- number of probes: 24 , kept by blast: 23 , if remove dups: 23\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: NRNX1-seg-66\n",
      "--- gene=NRNX1-seg-66, id=15 removed by soft count = 239\n",
      "--- gene=NRNX1-seg-66, id=16 removed by soft count = 282\n",
      "--- gene=NRNX1-seg-66, id=17 removed by soft count = 83\n",
      "--- gene=NRNX1-seg-66, id=28 removed by soft count = 137\n",
      "--- gene=NRNX1-seg-66, id=29 removed by soft count = 92\n",
      "--- gene=NRNX1-seg-66, id=31 removed by soft count = 208\n",
      "--- number of probes: 45 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-67\n",
      "--- gene=NRNX1-seg-67, id=15 removed by soft count = 6253\n",
      "--- number of probes: 51 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-68\n",
      "--- gene=NRNX1-seg-68, id=16 removed by soft count = 166\n",
      "--- gene=NRNX1-seg-68, id=17 removed by soft count = 84\n",
      "--- gene=NRNX1-seg-68, id=19 removed by soft count = 476\n",
      "--- number of probes: 24 , kept by blast: 21 , if remove dups: 21\n",
      "-- number of probes kept for this region: 21\n",
      "-- checking probes in region: NRNX1-seg-69\n",
      "--- number of probes: 45 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-70\n",
      "--- gene=NRNX1-seg-70, id=3 removed by soft count = 208\n",
      "--- number of probes: 42 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: NRNX1-seg-71\n",
      "--- gene=NRNX1-seg-71, id=20 removed by soft count = 74\n",
      "--- number of probes: 54 , kept by blast: 53 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-72\n",
      "--- number of probes: 44 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-73\n",
      "--- gene=NRNX1-seg-73, id=13 removed by soft count = 344\n",
      "--- gene=NRNX1-seg-73, id=14 removed by soft count = 390\n",
      "--- gene=NRNX1-seg-73, id=15 removed by soft count = 772\n",
      "--- gene=NRNX1-seg-73, id=17 removed by soft count = 131\n",
      "--- gene=NRNX1-seg-73, id=18 removed by soft count = 343\n",
      "--- number of probes: 24 , kept by blast: 19 , if remove dups: 19\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: NRNX1-seg-74\n",
      "--- gene=NRNX1-seg-74, id=35 removed by soft count = 255\n",
      "--- number of probes: 36 , kept by blast: 35 , if remove dups: 35\n",
      "-- number of probes kept for this region: 35\n",
      "-- checking probes in region: NRNX1-seg-75\n",
      "--- gene=NRNX1-seg-75, id=9 removed by soft count = 1526\n",
      "--- gene=NRNX1-seg-75, id=18 removed by soft count = 452\n",
      "--- number of probes: 28 , kept by blast: 26 , if remove dups: 26\n",
      "-- number of probes kept for this region: 26\n",
      "-- checking probes in region: NRNX1-seg-76\n",
      "--- gene=NRNX1-seg-76, id=45 removed by soft count = 209\n",
      "--- number of probes: 46 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-77\n",
      "--- gene=NRNX1-seg-77, id=0 removed by soft count = 480\n",
      "--- gene=NRNX1-seg-77, id=4 removed by soft count = 91\n",
      "--- gene=NRNX1-seg-77, id=5 removed by soft count = 195\n",
      "--- gene=NRNX1-seg-77, id=6 removed by soft count = 564\n",
      "--- gene=NRNX1-seg-77, id=11 removed by soft count = 471\n",
      "--- gene=NRNX1-seg-77, id=12 removed by soft count = 146\n",
      "--- gene=NRNX1-seg-77, id=25 removed by soft count = 51\n",
      "--- gene=NRNX1-seg-77, id=27 removed by soft count = 66\n",
      "--- gene=NRNX1-seg-77, id=33 removed by soft count = 131\n",
      "--- number of probes: 34 , kept by blast: 25 , if remove dups: 25\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: NRNX1-seg-79\n",
      "--- gene=NRNX1-seg-79, id=0 removed by soft count = 135\n",
      "--- gene=NRNX1-seg-79, id=1 removed by soft count = 727\n",
      "--- gene=NRNX1-seg-79, id=5 removed by soft count = 53\n",
      "--- gene=NRNX1-seg-79, id=8 removed by soft count = 147\n",
      "--- gene=NRNX1-seg-79, id=14 removed by soft count = 107\n",
      "--- number of probes: 32 , kept by blast: 27 , if remove dups: 27\n",
      "-- number of probes kept for this region: 27\n",
      "-- checking probes in region: NRNX1-seg-80\n",
      "--- gene=NRNX1-seg-80, id=48 removed by soft count = 86\n",
      "--- number of probes: 51 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-81\n",
      "--- gene=NRNX1-seg-81, id=29 removed by soft count = 165\n",
      "--- number of probes: 37 , kept by blast: 36 , if remove dups: 36\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: NRNX1-seg-82\n",
      "--- gene=NRNX1-seg-82, id=7 removed by soft count = 5711\n",
      "--- number of probes: 48 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-83\n",
      "--- number of probes: 63 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-84\n",
      "--- gene=NRNX1-seg-84, id=38 removed by soft count = 51\n",
      "--- number of probes: 69 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-85\n",
      "--- gene=NRNX1-seg-85, id=25 removed by soft count = 91\n",
      "--- gene=NRNX1-seg-85, id=51 removed by soft count = 90\n",
      "--- number of probes: 55 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-86\n",
      "--- gene=NRNX1-seg-86, id=7 removed by soft count = 142\n",
      "--- number of probes: 46 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-87\n",
      "--- gene=NRNX1-seg-87, id=1 removed by soft count = 135\n",
      "--- gene=NRNX1-seg-87, id=6 removed by soft count = 82\n",
      "--- gene=NRNX1-seg-87, id=8 removed by soft count = 157\n",
      "--- gene=NRNX1-seg-87, id=22 removed by soft count = 117\n",
      "--- gene=NRNX1-seg-87, id=46 removed by soft count = 57\n",
      "--- gene=NRNX1-seg-87, id=48 removed by soft count = 147\n",
      "--- number of probes: 50 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-88\n",
      "--- gene=NRNX1-seg-88, id=2 removed by soft count = 61\n",
      "--- number of probes: 47 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-89\n",
      "--- number of probes: 77 , kept by blast: 77 , if remove dups: 77\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-90\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 52 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-91\n",
      "--- number of probes: 50 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-92\n",
      "--- gene=NRNX1-seg-92, id=0 removed by soft count = 109\n",
      "--- number of probes: 38 , kept by blast: 37 , if remove dups: 37\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: NRNX1-seg-93\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-94\n",
      "--- gene=NRNX1-seg-94, id=24 removed by soft count = 61\n",
      "--- gene=NRNX1-seg-94, id=36 removed by soft count = 94\n",
      "--- number of probes: 45 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-95\n",
      "--- number of probes: 29 , kept by blast: 29 , if remove dups: 29\n",
      "-- number of probes kept for this region: 29\n",
      "-- checking probes in region: NRNX1-seg-96\n",
      "--- gene=NRNX1-seg-96, id=6 removed by soft count = 61\n",
      "--- gene=NRNX1-seg-96, id=9 removed by soft count = 186\n",
      "--- gene=NRNX1-seg-96, id=12 removed by soft count = 303\n",
      "--- gene=NRNX1-seg-96, id=16 removed by soft count = 72\n",
      "--- number of probes: 44 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: NRNX1-seg-97\n",
      "--- gene=NRNX1-seg-97, id=23 removed by soft count = 58\n",
      "--- gene=NRNX1-seg-97, id=24 removed by soft count = 257\n",
      "--- gene=NRNX1-seg-97, id=25 removed by soft count = 732\n",
      "--- gene=NRNX1-seg-97, id=39 removed by soft count = 59\n",
      "--- number of probes: 44 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: NRNX1-seg-98\n",
      "--- gene=NRNX1-seg-98, id=2 removed by soft count = 393\n",
      "--- number of probes: 37 , kept by blast: 36 , if remove dups: 35\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: NRNX1-seg-99\n",
      "--- gene=NRNX1-seg-99, id=30 removed by soft count = 804\n",
      "--- gene=NRNX1-seg-99, id=31 removed by soft count = 899\n",
      "--- number of probes: 32 , kept by blast: 30 , if remove dups: 30\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: NRNX1-seg-100\n",
      "--- gene=NRNX1-seg-100, id=13 removed by soft count = 131\n",
      "--- gene=NRNX1-seg-100, id=14 removed by soft count = 63\n",
      "--- gene=NRNX1-seg-100, id=17 removed by soft count = 160\n",
      "--- gene=NRNX1-seg-100, id=20 removed by soft count = 57\n",
      "--- gene=NRNX1-seg-100, id=33 removed by soft count = 661\n",
      "--- number of probes: 44 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-101\n",
      "--- gene=NRNX1-seg-101, id=8 removed by soft count = 204\n",
      "--- gene=NRNX1-seg-101, id=22 removed by soft count = 101\n",
      "--- gene=NRNX1-seg-101, id=26 removed by soft count = 95\n",
      "--- number of probes: 42 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-102\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-103\n",
      "--- gene=NRNX1-seg-103, id=12 removed by soft count = 141\n",
      "--- number of probes: 52 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: NRNX1-seg-104\n",
      "--- number of probes: 58 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-105\n",
      "--- number of probes: 23 , kept by blast: 23 , if remove dups: 23\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: NRNX1-seg-106\n",
      "--- number of probes: 69 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-107\n",
      "--- number of probes: 46 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-108\n",
      "--- number of probes: 32 , kept by blast: 32 , if remove dups: 32\n",
      "-- number of probes kept for this region: 32\n",
      "-- checking probes in region: NRNX1-seg-109\n",
      "--- gene=NRNX1-seg-109, id=4 removed by soft count = 85\n",
      "--- gene=NRNX1-seg-109, id=8 removed by soft count = 256\n",
      "--- gene=NRNX1-seg-109, id=21 removed by soft count = 58\n",
      "--- number of probes: 42 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-110\n",
      "--- number of probes: 66 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-111\n",
      "--- number of probes: 47 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-112\n",
      "--- gene=NRNX1-seg-112, id=15 removed by soft count = 99\n",
      "--- gene=NRNX1-seg-112, id=16 removed by soft count = 189\n",
      "--- gene=NRNX1-seg-112, id=17 removed by soft count = 72\n",
      "--- number of probes: 25 , kept by blast: 22 , if remove dups: 22\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: NRNX1-seg-113\n",
      "--- gene=NRNX1-seg-113, id=59 removed by soft count = 221\n",
      "--- number of probes: 62 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-114\n",
      "--- gene=NRNX1-seg-114, id=1 removed by soft count = 103\n",
      "--- gene=NRNX1-seg-114, id=10 removed by soft count = 51\n",
      "--- gene=NRNX1-seg-114, id=12 removed by soft count = 79\n",
      "--- gene=NRNX1-seg-114, id=55 removed by soft count = 285\n",
      "--- gene=NRNX1-seg-114, id=56 removed by soft count = 599\n",
      "--- number of probes: 57 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-115\n",
      "--- gene=NRNX1-seg-115, id=44 removed by soft count = 53\n",
      "--- gene=NRNX1-seg-115, id=45 removed by soft count = 307\n",
      "--- number of probes: 50 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: NRNX1-seg-117\n",
      "--- gene=NRNX1-seg-117, id=0 removed by soft count = 58\n",
      "--- number of probes: 57 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-118\n",
      "--- number of probes: 58 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-119\n",
      "--- gene=NRNX1-seg-119, id=28 removed by soft count = 15091\n",
      "--- number of probes: 53 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-120\n",
      "--- number of probes: 56 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-121\n",
      "--- gene=NRNX1-seg-121, id=28 removed by soft count = 126\n",
      "--- gene=NRNX1-seg-121, id=29 removed by soft count = 249\n",
      "--- gene=NRNX1-seg-121, id=52 removed by soft count = 76\n",
      "--- gene=NRNX1-seg-121, id=54 removed by soft count = 251\n",
      "--- gene=NRNX1-seg-121, id=55 removed by soft count = 336\n",
      "--- number of probes: 65 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-122\n",
      "--- gene=NRNX1-seg-122, id=9 removed by soft count = 133\n",
      "--- gene=NRNX1-seg-122, id=10 removed by soft count = 461\n",
      "--- number of probes: 49 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-123\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=NRNX1-seg-123, id=8 removed by soft count = 90\n",
      "--- gene=NRNX1-seg-123, id=36 removed by soft count = 3187\n",
      "--- number of probes: 37 , kept by blast: 35 , if remove dups: 35\n",
      "-- number of probes kept for this region: 35\n",
      "-- checking probes in region: NRNX1-seg-124\n",
      "--- gene=NRNX1-seg-124, id=18 removed by soft count = 213\n",
      "--- number of probes: 59 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-125\n",
      "--- number of probes: 36 , kept by blast: 36 , if remove dups: 36\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: NRNX1-seg-126\n",
      "--- gene=NRNX1-seg-126, id=1 removed by soft count = 74\n",
      "--- gene=NRNX1-seg-126, id=2 removed by soft count = 671\n",
      "--- gene=NRNX1-seg-126, id=3 removed by soft count = 160\n",
      "--- gene=NRNX1-seg-126, id=7 removed by soft count = 119\n",
      "--- gene=NRNX1-seg-126, id=14 removed by soft count = 566\n",
      "--- gene=NRNX1-seg-126, id=15 removed by soft count = 805\n",
      "--- gene=NRNX1-seg-126, id=16 removed by soft count = 386\n",
      "--- gene=NRNX1-seg-126, id=24 removed by soft count = 126\n",
      "--- number of probes: 27 , kept by blast: 19 , if remove dups: 19\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: NRNX1-seg-127\n",
      "--- gene=NRNX1-seg-127, id=0 removed by soft count = 230\n",
      "--- gene=NRNX1-seg-127, id=1 removed by soft count = 5818\n",
      "--- gene=NRNX1-seg-127, id=2 removed by soft count = 470\n",
      "--- gene=NRNX1-seg-127, id=3 removed by soft count = 1787\n",
      "--- gene=NRNX1-seg-127, id=25 removed by soft count = 237\n",
      "--- gene=NRNX1-seg-127, id=28 removed by soft count = 66\n",
      "--- gene=NRNX1-seg-127, id=29 removed by soft count = 278\n",
      "--- gene=NRNX1-seg-127, id=30 removed by soft count = 3571\n",
      "--- gene=NRNX1-seg-127, id=32 removed by soft count = 57\n",
      "--- gene=NRNX1-seg-127, id=33 removed by soft count = 89\n",
      "--- gene=NRNX1-seg-127, id=34 removed by soft count = 1360\n",
      "--- gene=NRNX1-seg-127, id=37 removed by soft count = 58\n",
      "--- number of probes: 38 , kept by blast: 26 , if remove dups: 26\n",
      "-- number of probes kept for this region: 26\n",
      "-- checking probes in region: NRNX1-seg-128\n",
      "--- gene=NRNX1-seg-128, id=0 removed by soft count = 67\n",
      "--- gene=NRNX1-seg-128, id=1 removed by soft count = 170\n",
      "--- gene=NRNX1-seg-128, id=2 removed by soft count = 62\n",
      "--- gene=NRNX1-seg-128, id=5 removed by soft count = 110\n",
      "--- gene=NRNX1-seg-128, id=12 removed by soft count = 83\n",
      "--- gene=NRNX1-seg-128, id=28 removed by soft count = 172\n",
      "--- gene=NRNX1-seg-128, id=30 removed by soft count = 197\n",
      "--- gene=NRNX1-seg-128, id=31 removed by soft count = 129\n",
      "--- gene=NRNX1-seg-128, id=32 removed by soft count = 425\n",
      "--- gene=NRNX1-seg-128, id=34 removed by soft count = 654\n",
      "--- gene=NRNX1-seg-128, id=35 removed by soft count = 86\n",
      "--- number of probes: 36 , kept by blast: 25 , if remove dups: 25\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: NRNX1-seg-129\n",
      "--- number of probes: 30 , kept by blast: 30 , if remove dups: 30\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: NRNX1-seg-130\n",
      "--- number of probes: 46 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-131\n",
      "--- gene=NRNX1-seg-131, id=36 removed by soft count = 77\n",
      "--- number of probes: 40 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-133\n",
      "--- gene=NRNX1-seg-133, id=17 removed by soft count = 512\n",
      "--- number of probes: 24 , kept by blast: 23 , if remove dups: 23\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: NRNX1-seg-134\n",
      "--- gene=NRNX1-seg-134, id=0 removed by soft count = 67\n",
      "--- gene=NRNX1-seg-134, id=21 removed by soft count = 73\n",
      "--- number of probes: 28 , kept by blast: 26 , if remove dups: 26\n",
      "-- number of probes kept for this region: 26\n",
      "-- checking probes in region: NRNX1-seg-135\n",
      "--- gene=NRNX1-seg-135, id=30 removed by soft count = 185\n",
      "--- gene=NRNX1-seg-135, id=32 removed by soft count = 147\n",
      "--- number of probes: 42 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: NRNX1-seg-136\n",
      "--- number of probes: 51 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: NRNX1-seg-137\n",
      "--- number of probes: 50 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-138\n",
      "--- gene=NRNX1-seg-138, id=17 removed by soft count = 51\n",
      "--- gene=NRNX1-seg-138, id=18 removed by soft count = 413\n",
      "--- number of probes: 51 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-139\n",
      "--- number of probes: 67 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-140\n",
      "--- gene=NRNX1-seg-140, id=0 removed by soft count = 22435\n",
      "--- gene=NRNX1-seg-140, id=1 removed by soft count = 215\n",
      "--- gene=NRNX1-seg-140, id=3 removed by soft count = 385\n",
      "--- number of probes: 61 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-141\n",
      "--- gene=NRNX1-seg-141, id=37 removed by soft count = 5154\n",
      "--- gene=NRNX1-seg-141, id=38 removed by soft count = 3129\n",
      "--- number of probes: 39 , kept by blast: 37 , if remove dups: 37\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: NRNX1-seg-142\n",
      "--- gene=NRNX1-seg-142, id=0 removed by soft count = 2565\n",
      "--- gene=NRNX1-seg-142, id=1 removed by soft count = 1976\n",
      "--- gene=NRNX1-seg-142, id=2 removed by soft count = 93\n",
      "--- gene=NRNX1-seg-142, id=11 removed by soft count = 172\n",
      "--- number of probes: 31 , kept by blast: 27 , if remove dups: 27\n",
      "-- number of probes kept for this region: 27\n",
      "-- checking probes in region: NRNX1-seg-143\n",
      "--- gene=NRNX1-seg-143, id=41 removed by soft count = 96\n",
      "--- number of probes: 71 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-144\n",
      "--- gene=NRNX1-seg-144, id=36 removed by soft count = 122\n",
      "--- number of probes: 69 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-145\n",
      "--- number of probes: 69 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-146\n",
      "--- number of probes: 69 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-147\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-148\n",
      "--- number of probes: 57 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-150\n",
      "--- number of probes: 59 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: NRNX1-seg-151\n",
      "--- number of probes: 69 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-152\n",
      "--- number of probes: 66 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-154\n",
      "--- gene=NRNX1-seg-154, id=0 removed by soft count = 1050\n",
      "--- gene=NRNX1-seg-154, id=1 removed by soft count = 863\n",
      "--- gene=NRNX1-seg-154, id=3 removed by soft count = 390\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 35 , kept by blast: 32 , if remove dups: 32\n",
      "-- number of probes kept for this region: 32\n",
      "-- checking probes in region: NRNX1-seg-155\n",
      "--- gene=NRNX1-seg-155, id=57 removed by soft count = 84\n",
      "--- number of probes: 59 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-156\n",
      "--- number of probes: 66 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-157\n",
      "--- gene=NRNX1-seg-157, id=2 removed by soft count = 174\n",
      "--- gene=NRNX1-seg-157, id=35 removed by soft count = 58\n",
      "--- gene=NRNX1-seg-157, id=36 removed by soft count = 404\n",
      "--- gene=NRNX1-seg-157, id=38 removed by soft count = 341\n",
      "--- gene=NRNX1-seg-157, id=39 removed by soft count = 55\n",
      "--- gene=NRNX1-seg-157, id=40 removed by soft count = 57\n",
      "--- gene=NRNX1-seg-157, id=41 removed by soft count = 81\n",
      "--- number of probes: 49 , kept by blast: 42 , if remove dups: 42\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: NRNX1-seg-158\n",
      "--- gene=NRNX1-seg-158, id=28 removed by soft count = 97\n",
      "--- number of probes: 67 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-159\n",
      "--- gene=NRNX1-seg-159, id=53 removed by soft count = 189\n",
      "--- number of probes: 58 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-160\n",
      "--- gene=NRNX1-seg-160, id=45 removed by soft count = 264\n",
      "--- number of probes: 64 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-161\n",
      "--- number of probes: 75 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-162\n",
      "--- number of probes: 66 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-163\n",
      "--- gene=NRNX1-seg-163, id=12 removed by soft count = 869\n",
      "--- gene=NRNX1-seg-163, id=13 removed by soft count = 1454\n",
      "--- gene=NRNX1-seg-163, id=14 removed by soft count = 179\n",
      "--- gene=NRNX1-seg-163, id=15 removed by soft count = 51\n",
      "--- gene=NRNX1-seg-163, id=32 removed by soft count = 71\n",
      "--- number of probes: 34 , kept by blast: 29 , if remove dups: 29\n",
      "-- number of probes kept for this region: 29\n",
      "-- checking probes in region: NRNX1-seg-164\n",
      "--- number of probes: 56 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-165\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-166\n",
      "--- gene=NRNX1-seg-166, id=13 removed by soft count = 228\n",
      "--- gene=NRNX1-seg-166, id=14 removed by soft count = 52\n",
      "--- gene=NRNX1-seg-166, id=18 removed by soft count = 104\n",
      "--- gene=NRNX1-seg-166, id=21 removed by soft count = 52\n",
      "--- gene=NRNX1-seg-166, id=22 removed by soft count = 60\n",
      "--- gene=NRNX1-seg-166, id=24 removed by soft count = 55\n",
      "--- gene=NRNX1-seg-166, id=26 removed by soft count = 67\n",
      "--- gene=NRNX1-seg-166, id=28 removed by soft count = 85\n",
      "--- gene=NRNX1-seg-166, id=30 removed by soft count = 205\n",
      "--- gene=NRNX1-seg-166, id=32 removed by soft count = 117\n",
      "--- gene=NRNX1-seg-166, id=33 removed by soft count = 52\n",
      "--- gene=NRNX1-seg-166, id=35 removed by soft count = 59\n",
      "--- gene=NRNX1-seg-166, id=36 removed by soft count = 138\n",
      "--- gene=NRNX1-seg-166, id=37 removed by soft count = 124\n",
      "--- gene=NRNX1-seg-166, id=39 removed by soft count = 59\n",
      "--- gene=NRNX1-seg-166, id=40 removed by soft count = 72\n",
      "--- gene=NRNX1-seg-166, id=41 removed by soft count = 54\n",
      "--- gene=NRNX1-seg-166, id=42 removed by soft count = 126\n",
      "--- gene=NRNX1-seg-166, id=43 removed by soft count = 118\n",
      "--- gene=NRNX1-seg-166, id=44 removed by soft count = 182\n",
      "--- gene=NRNX1-seg-166, id=45 removed by soft count = 215\n",
      "--- gene=NRNX1-seg-166, id=46 removed by soft count = 344\n",
      "--- gene=NRNX1-seg-166, id=47 removed by soft count = 658\n",
      "--- gene=NRNX1-seg-166, id=49 removed by soft count = 220\n",
      "--- number of probes: 51 , kept by blast: 27 , if remove dups: 27\n",
      "-- number of probes kept for this region: 27\n",
      "-- checking probes in region: NRNX1-seg-169\n",
      "--- gene=NRNX1-seg-169, id=3 removed by soft count = 102\n",
      "--- gene=NRNX1-seg-169, id=5 removed by soft count = 57\n",
      "--- gene=NRNX1-seg-169, id=6 removed by soft count = 82\n",
      "--- gene=NRNX1-seg-169, id=7 removed by soft count = 64\n",
      "--- gene=NRNX1-seg-169, id=8 removed by soft count = 68\n",
      "--- gene=NRNX1-seg-169, id=10 removed by soft count = 723\n",
      "--- gene=NRNX1-seg-169, id=15 removed by soft count = 139\n",
      "--- gene=NRNX1-seg-169, id=16 removed by soft count = 58\n",
      "--- gene=NRNX1-seg-169, id=17 removed by soft count = 112\n",
      "--- gene=NRNX1-seg-169, id=18 removed by soft count = 79\n",
      "--- gene=NRNX1-seg-169, id=19 removed by soft count = 115\n",
      "--- gene=NRNX1-seg-169, id=21 removed by soft count = 163\n",
      "--- number of probes: 62 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-170\n",
      "--- gene=NRNX1-seg-170, id=61 removed by soft count = 108\n",
      "--- number of probes: 71 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-171\n",
      "--- gene=NRNX1-seg-171, id=14 removed by soft count = 143\n",
      "--- gene=NRNX1-seg-171, id=17 removed by soft count = 65\n",
      "--- number of probes: 63 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-172\n",
      "--- gene=NRNX1-seg-172, id=25 removed by soft count = 77\n",
      "--- gene=NRNX1-seg-172, id=74 removed by soft count = 575\n",
      "--- number of probes: 76 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-173\n",
      "--- gene=NRNX1-seg-173, id=52 removed by soft count = 306\n",
      "--- gene=NRNX1-seg-173, id=53 removed by soft count = 173\n",
      "--- gene=NRNX1-seg-173, id=54 removed by soft count = 77\n",
      "--- number of probes: 68 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-174\n",
      "--- gene=NRNX1-seg-174, id=37 removed by soft count = 1421\n",
      "--- gene=NRNX1-seg-174, id=38 removed by soft count = 904\n",
      "--- number of probes: 58 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-175\n",
      "--- number of probes: 71 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-176\n",
      "--- number of probes: 70 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-177\n",
      "--- gene=NRNX1-seg-177, id=43 removed by soft count = 120\n",
      "--- number of probes: 64 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-178\n",
      "--- gene=NRNX1-seg-178, id=16 removed by soft count = 545\n",
      "--- number of probes: 66 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-179\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=NRNX1-seg-179, id=61 removed by soft count = 207\n",
      "--- number of probes: 71 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-180\n",
      "--- gene=NRNX1-seg-180, id=51 removed by soft count = 60\n",
      "--- number of probes: 69 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-181\n",
      "--- number of probes: 71 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-182\n",
      "--- number of probes: 71 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-183\n",
      "--- gene=NRNX1-seg-183, id=19 removed by soft count = 60\n",
      "--- gene=NRNX1-seg-183, id=20 removed by soft count = 108\n",
      "--- number of probes: 56 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-184\n",
      "--- number of probes: 48 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: NRNX1-seg-185\n",
      "--- gene=NRNX1-seg-185, id=7 removed by soft count = 97\n",
      "--- gene=NRNX1-seg-185, id=8 removed by soft count = 124\n",
      "--- gene=NRNX1-seg-185, id=12 removed by soft count = 192\n",
      "--- gene=NRNX1-seg-185, id=14 removed by soft count = 106\n",
      "--- gene=NRNX1-seg-185, id=22 removed by soft count = 104\n",
      "--- gene=NRNX1-seg-185, id=31 removed by soft count = 79\n",
      "--- gene=NRNX1-seg-185, id=39 removed by soft count = 81\n",
      "--- number of probes: 43 , kept by blast: 36 , if remove dups: 36\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: NRNX1-seg-186\n",
      "--- number of probes: 54 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-187\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-188\n",
      "--- gene=NRNX1-seg-188, id=54 removed by soft count = 62\n",
      "--- number of probes: 59 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-189\n",
      "--- number of probes: 63 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-190\n",
      "--- number of probes: 64 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-191\n",
      "--- gene=NRNX1-seg-191, id=42 removed by soft count = 245\n",
      "--- gene=NRNX1-seg-191, id=43 removed by soft count = 117\n",
      "--- gene=NRNX1-seg-191, id=46 removed by soft count = 99\n",
      "--- number of probes: 52 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-192\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-193\n",
      "--- number of probes: 71 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-194\n",
      "--- number of probes: 67 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-195\n",
      "--- number of probes: 72 , kept by blast: 72 , if remove dups: 72\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-196\n",
      "--- gene=NRNX1-seg-196, id=1 removed by soft count = 100\n",
      "--- gene=NRNX1-seg-196, id=4 removed by soft count = 87\n",
      "--- gene=NRNX1-seg-196, id=8 removed by soft count = 202\n",
      "--- gene=NRNX1-seg-196, id=38 removed by soft count = 1872\n",
      "--- number of probes: 49 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-197\n",
      "--- number of probes: 74 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-198\n",
      "--- gene=NRNX1-seg-198, id=42 removed by soft count = 94\n",
      "--- number of probes: 48 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-199\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-200\n",
      "--- number of probes: 70 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-201\n",
      "--- gene=NRNX1-seg-201, id=37 removed by soft count = 3255\n",
      "--- number of probes: 77 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-202\n",
      "--- gene=NRNX1-seg-202, id=14 removed by soft count = 395\n",
      "--- number of probes: 41 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: NRNX1-seg-203\n",
      "--- gene=NRNX1-seg-203, id=26 removed by soft count = 273\n",
      "--- gene=NRNX1-seg-203, id=29 removed by soft count = 62\n",
      "--- number of probes: 45 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-204\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-205\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-206\n",
      "--- number of probes: 71 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-207\n",
      "--- number of probes: 70 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-208\n",
      "--- number of probes: 51 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: NRNX1-seg-209\n",
      "--- number of probes: 75 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-210\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-211\n",
      "--- gene=NRNX1-seg-211, id=61 removed by soft count = 84\n",
      "--- number of probes: 62 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-212\n",
      "--- gene=NRNX1-seg-212, id=4 removed by soft count = 79\n",
      "--- gene=NRNX1-seg-212, id=7 removed by soft count = 700\n",
      "--- gene=NRNX1-seg-212, id=9 removed by soft count = 82\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=NRNX1-seg-212, id=10 removed by soft count = 525\n",
      "--- gene=NRNX1-seg-212, id=13 removed by soft count = 64\n",
      "--- gene=NRNX1-seg-212, id=14 removed by soft count = 1151\n",
      "--- gene=NRNX1-seg-212, id=16 removed by soft count = 97\n",
      "--- gene=NRNX1-seg-212, id=17 removed by soft count = 111\n",
      "--- gene=NRNX1-seg-212, id=18 removed by soft count = 104\n",
      "--- gene=NRNX1-seg-212, id=21 removed by soft count = 154\n",
      "--- gene=NRNX1-seg-212, id=22 removed by soft count = 137\n",
      "--- number of probes: 23 , kept by blast: 12 , if remove dups: 12\n",
      "-- number of probes kept for this region: 12\n",
      "-- checking probes in region: NRNX1-seg-213\n",
      "--- gene=NRNX1-seg-213, id=1 removed by soft count = 80\n",
      "--- gene=NRNX1-seg-213, id=21 removed by soft count = 510\n",
      "--- gene=NRNX1-seg-213, id=22 removed by soft count = 310\n",
      "--- gene=NRNX1-seg-213, id=25 removed by soft count = 1394\n",
      "--- gene=NRNX1-seg-213, id=26 removed by soft count = 112\n",
      "--- gene=NRNX1-seg-213, id=27 removed by soft count = 2533\n",
      "--- number of probes: 49 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-214\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-215\n",
      "--- gene=NRNX1-seg-215, id=15 removed by soft count = 346\n",
      "--- gene=NRNX1-seg-215, id=25 removed by soft count = 741\n",
      "--- gene=NRNX1-seg-215, id=26 removed by soft count = 415\n",
      "--- gene=NRNX1-seg-215, id=27 removed by soft count = 60\n",
      "--- gene=NRNX1-seg-215, id=29 removed by soft count = 840\n",
      "--- number of probes: 30 , kept by blast: 25 , if remove dups: 25\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: NRNX1-seg-216\n",
      "--- gene=NRNX1-seg-216, id=0 removed by soft count = 126\n",
      "--- gene=NRNX1-seg-216, id=1 removed by soft count = 129\n",
      "--- gene=NRNX1-seg-216, id=4 removed by soft count = 720\n",
      "--- gene=NRNX1-seg-216, id=8 removed by soft count = 968\n",
      "--- gene=NRNX1-seg-216, id=11 removed by soft count = 955\n",
      "--- gene=NRNX1-seg-216, id=13 removed by soft count = 172\n",
      "--- gene=NRNX1-seg-216, id=14 removed by soft count = 901\n",
      "--- number of probes: 27 , kept by blast: 20 , if remove dups: 20\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: NRNX1-seg-217\n",
      "--- gene=NRNX1-seg-217, id=5 removed by soft count = 102\n",
      "--- number of probes: 24 , kept by blast: 23 , if remove dups: 23\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: NRNX1-seg-218\n",
      "--- gene=NRNX1-seg-218, id=8 removed by soft count = 1209\n",
      "--- number of probes: 44 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-219\n",
      "--- gene=NRNX1-seg-219, id=3 removed by soft count = 132\n",
      "--- gene=NRNX1-seg-219, id=10 removed by soft count = 109\n",
      "--- gene=NRNX1-seg-219, id=11 removed by soft count = 54\n",
      "--- number of probes: 23 , kept by blast: 20 , if remove dups: 20\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: NRNX1-seg-220\n",
      "--- gene=NRNX1-seg-220, id=2 removed by soft count = 136\n",
      "--- gene=NRNX1-seg-220, id=8 removed by soft count = 319\n",
      "--- gene=NRNX1-seg-220, id=9 removed by soft count = 64\n",
      "--- gene=NRNX1-seg-220, id=10 removed by soft count = 175\n",
      "--- gene=NRNX1-seg-220, id=31 removed by soft count = 435\n",
      "--- number of probes: 35 , kept by blast: 30 , if remove dups: 30\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: NRNX1-seg-221\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-222\n",
      "--- gene=NRNX1-seg-222, id=44 removed by soft count = 57\n",
      "--- number of probes: 71 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-223\n",
      "--- number of probes: 75 , kept by blast: 75 , if remove dups: 75\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-224\n",
      "--- gene=NRNX1-seg-224, id=27 removed by soft count = 147\n",
      "--- gene=NRNX1-seg-224, id=56 removed by soft count = 68\n",
      "--- gene=NRNX1-seg-224, id=59 removed by soft count = 97\n",
      "--- number of probes: 74 , kept by blast: 71 , if remove dups: 71\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-225\n",
      "--- gene=NRNX1-seg-225, id=44 removed by soft count = 172\n",
      "--- number of probes: 60 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: NRNX1-seg-226\n",
      "--- gene=NRNX1-seg-226, id=4 removed by soft count = 51\n",
      "--- number of probes: 54 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-227\n",
      "--- number of probes: 41 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: NRNX1-seg-228\n",
      "--- gene=NRNX1-seg-228, id=4 removed by soft count = 180\n",
      "--- gene=NRNX1-seg-228, id=5 removed by soft count = 67\n",
      "--- gene=NRNX1-seg-228, id=9 removed by soft count = 150\n",
      "--- number of probes: 29 , kept by blast: 26 , if remove dups: 26\n",
      "-- number of probes kept for this region: 26\n",
      "-- checking probes in region: NRNX1-seg-229\n",
      "--- gene=NRNX1-seg-229, id=2 removed by soft count = 96\n",
      "--- gene=NRNX1-seg-229, id=3 removed by soft count = 189\n",
      "--- number of probes: 62 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-230\n",
      "--- number of probes: 70 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-231\n",
      "--- number of probes: 70 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-232\n",
      "--- number of probes: 29 , kept by blast: 29 , if remove dups: 29\n",
      "-- number of probes kept for this region: 29\n",
      "-- checking probes in region: NRNX1-seg-233\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-234\n",
      "--- gene=NRNX1-seg-234, id=8 removed by soft count = 99\n",
      "--- gene=NRNX1-seg-234, id=34 removed by soft count = 67\n",
      "--- number of probes: 54 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-235\n",
      "--- gene=NRNX1-seg-235, id=0 removed by soft count = 58\n",
      "--- gene=NRNX1-seg-235, id=1 removed by soft count = 231\n",
      "--- gene=NRNX1-seg-235, id=2 removed by soft count = 591\n",
      "--- gene=NRNX1-seg-235, id=4 removed by soft count = 1170\n",
      "--- gene=NRNX1-seg-235, id=5 removed by soft count = 90\n",
      "--- number of probes: 64 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: NRNX1-seg-236\n",
      "--- gene=NRNX1-seg-236, id=43 removed by soft count = 71\n",
      "--- number of probes: 54 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-238\n",
      "--- gene=NRNX1-seg-238, id=0 removed by soft count = 3214\n",
      "--- gene=NRNX1-seg-238, id=1 removed by soft count = 266\n",
      "--- gene=NRNX1-seg-238, id=2 removed by soft count = 164\n",
      "--- gene=NRNX1-seg-238, id=3 removed by soft count = 1336\n",
      "--- gene=NRNX1-seg-238, id=4 removed by soft count = 92\n",
      "--- gene=NRNX1-seg-238, id=21 removed by soft count = 51\n",
      "--- number of probes: 53 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-239\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-240\n",
      "--- number of probes: 66 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-241\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-242\n",
      "--- gene=NRNX1-seg-242, id=15 removed by soft count = 96\n",
      "--- gene=NRNX1-seg-242, id=16 removed by soft count = 201\n",
      "--- gene=NRNX1-seg-242, id=19 removed by soft count = 158\n",
      "--- gene=NRNX1-seg-242, id=20 removed by soft count = 171\n",
      "--- gene=NRNX1-seg-242, id=27 removed by soft count = 77\n",
      "--- gene=NRNX1-seg-242, id=28 removed by soft count = 53\n",
      "--- gene=NRNX1-seg-242, id=34 removed by soft count = 61\n",
      "--- gene=NRNX1-seg-242, id=36 removed by soft count = 128\n",
      "--- gene=NRNX1-seg-242, id=38 removed by soft count = 162\n",
      "--- gene=NRNX1-seg-242, id=39 removed by soft count = 203\n",
      "--- number of probes: 53 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-244\n",
      "--- gene=NRNX1-seg-244, id=1 removed by soft count = 59\n",
      "--- gene=NRNX1-seg-244, id=4 removed by soft count = 62\n",
      "--- number of probes: 54 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-245\n",
      "--- gene=NRNX1-seg-245, id=8 removed by soft count = 228\n",
      "--- number of probes: 64 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-246\n",
      "--- gene=NRNX1-seg-246, id=34 removed by soft count = 1600\n",
      "--- number of probes: 38 , kept by blast: 37 , if remove dups: 37\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: NRNX1-seg-248\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-249\n",
      "--- gene=NRNX1-seg-249, id=29 removed by soft count = 77\n",
      "--- gene=NRNX1-seg-249, id=30 removed by soft count = 317\n",
      "--- gene=NRNX1-seg-249, id=39 removed by soft count = 7599\n",
      "--- gene=NRNX1-seg-249, id=40 removed by soft count = 8511\n",
      "--- number of probes: 41 , kept by blast: 37 , if remove dups: 37\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: NRNX1-seg-251\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-252\n",
      "--- number of probes: 53 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-253\n",
      "--- gene=NRNX1-seg-253, id=21 removed by soft count = 88\n",
      "--- gene=NRNX1-seg-253, id=47 removed by soft count = 54\n",
      "--- number of probes: 68 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-254\n",
      "--- number of probes: 50 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-255\n",
      "--- gene=NRNX1-seg-255, id=57 removed by soft count = 341\n",
      "--- gene=NRNX1-seg-255, id=60 removed by soft count = 157\n",
      "--- gene=NRNX1-seg-255, id=61 removed by soft count = 379\n",
      "--- number of probes: 64 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-256\n",
      "--- gene=NRNX1-seg-256, id=29 removed by soft count = 119\n",
      "--- gene=NRNX1-seg-256, id=30 removed by soft count = 131\n",
      "--- gene=NRNX1-seg-256, id=32 removed by soft count = 64\n",
      "--- gene=NRNX1-seg-256, id=60 removed by soft count = 180\n",
      "--- number of probes: 63 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: NRNX1-seg-257\n",
      "--- gene=NRNX1-seg-257, id=0 removed by soft count = 74\n",
      "--- gene=NRNX1-seg-257, id=1 removed by soft count = 389\n",
      "--- number of probes: 52 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-258\n",
      "--- number of probes: 63 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-259\n",
      "--- gene=NRNX1-seg-259, id=6 removed by soft count = 143\n",
      "--- gene=NRNX1-seg-259, id=7 removed by soft count = 63\n",
      "--- gene=NRNX1-seg-259, id=9 removed by soft count = 269\n",
      "--- gene=NRNX1-seg-259, id=10 removed by soft count = 112\n",
      "--- gene=NRNX1-seg-259, id=33 removed by soft count = 199\n",
      "--- number of probes: 54 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-260\n",
      "--- gene=NRNX1-seg-260, id=12 removed by soft count = 191\n",
      "--- gene=NRNX1-seg-260, id=35 removed by soft count = 166\n",
      "--- gene=NRNX1-seg-260, id=47 removed by soft count = 370\n",
      "--- gene=NRNX1-seg-260, id=48 removed by soft count = 1038\n",
      "--- gene=NRNX1-seg-260, id=49 removed by soft count = 4064\n",
      "--- number of probes: 50 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-262\n",
      "--- gene=NRNX1-seg-262, id=25 removed by soft count = 54\n",
      "--- number of probes: 42 , kept by blast: 41 , if remove dups: 40\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: NRNX1-seg-263\n",
      "--- gene=NRNX1-seg-263, id=10 removed by soft count = 61\n",
      "--- gene=NRNX1-seg-263, id=11 removed by soft count = 149\n",
      "--- gene=NRNX1-seg-263, id=12 removed by soft count = 154\n",
      "--- number of probes: 25 , kept by blast: 22 , if remove dups: 22\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: NRNX1-seg-264\n",
      "--- number of probes: 35 , kept by blast: 35 , if remove dups: 35\n",
      "-- number of probes kept for this region: 35\n",
      "-- checking probes in region: NRNX1-seg-265\n",
      "--- gene=NRNX1-seg-265, id=20 removed by soft count = 293\n",
      "--- gene=NRNX1-seg-265, id=21 removed by soft count = 153\n",
      "--- gene=NRNX1-seg-265, id=22 removed by soft count = 2146\n",
      "--- gene=NRNX1-seg-265, id=24 removed by soft count = 269\n",
      "--- gene=NRNX1-seg-265, id=25 removed by soft count = 109\n",
      "--- gene=NRNX1-seg-265, id=27 removed by soft count = 253\n",
      "--- gene=NRNX1-seg-265, id=28 removed by soft count = 385\n",
      "--- gene=NRNX1-seg-265, id=29 removed by soft count = 107\n",
      "--- number of probes: 32 , kept by blast: 24 , if remove dups: 24\n",
      "-- number of probes kept for this region: 24\n",
      "-- checking probes in region: NRNX1-seg-266\n",
      "--- gene=NRNX1-seg-266, id=4 removed by soft count = 64\n",
      "--- gene=NRNX1-seg-266, id=14 removed by soft count = 102\n",
      "--- gene=NRNX1-seg-266, id=18 removed by soft count = 113\n",
      "--- gene=NRNX1-seg-266, id=19 removed by soft count = 117\n",
      "--- number of probes: 33 , kept by blast: 29 , if remove dups: 29\n",
      "-- number of probes kept for this region: 29\n",
      "-- checking probes in region: NRNX1-seg-267\n",
      "--- gene=NRNX1-seg-267, id=8 removed by soft count = 386\n",
      "--- number of probes: 55 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-268\n",
      "--- number of probes: 70 , kept by blast: 70 , if remove dups: 70\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-269\n",
      "--- number of probes: 63 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-270\n",
      "--- gene=NRNX1-seg-270, id=3 removed by soft count = 74\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=NRNX1-seg-270, id=26 removed by soft count = 477\n",
      "--- gene=NRNX1-seg-270, id=27 removed by soft count = 328\n",
      "--- gene=NRNX1-seg-270, id=45 removed by soft count = 81\n",
      "--- gene=NRNX1-seg-270, id=47 removed by soft count = 59\n",
      "--- number of probes: 50 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-271\n",
      "--- number of probes: 59 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: NRNX1-seg-272\n",
      "--- number of probes: 64 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-273\n",
      "--- gene=NRNX1-seg-273, id=26 removed by soft count = 1676\n",
      "--- gene=NRNX1-seg-273, id=27 removed by soft count = 266\n",
      "--- gene=NRNX1-seg-273, id=28 removed by soft count = 356\n",
      "--- gene=NRNX1-seg-273, id=29 removed by soft count = 2386\n",
      "--- gene=NRNX1-seg-273, id=30 removed by soft count = 1681\n",
      "--- number of probes: 37 , kept by blast: 32 , if remove dups: 32\n",
      "-- number of probes kept for this region: 32\n",
      "-- checking probes in region: NRNX1-seg-274\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-275\n",
      "--- gene=NRNX1-seg-275, id=31 removed by soft count = 214\n",
      "--- number of probes: 56 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-276\n",
      "--- gene=NRNX1-seg-276, id=48 removed by soft count = 439\n",
      "--- number of probes: 57 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-277\n",
      "--- number of probes: 46 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-278\n",
      "--- gene=NRNX1-seg-278, id=13 removed by soft count = 1106\n",
      "--- gene=NRNX1-seg-278, id=14 removed by soft count = 379\n",
      "--- gene=NRNX1-seg-278, id=15 removed by soft count = 4137\n",
      "--- gene=NRNX1-seg-278, id=16 removed by soft count = 216\n",
      "--- gene=NRNX1-seg-278, id=17 removed by soft count = 69\n",
      "--- gene=NRNX1-seg-278, id=18 removed by soft count = 260\n",
      "--- gene=NRNX1-seg-278, id=19 removed by soft count = 270\n",
      "--- gene=NRNX1-seg-278, id=20 removed by soft count = 100\n",
      "--- gene=NRNX1-seg-278, id=22 removed by soft count = 1674\n",
      "--- gene=NRNX1-seg-278, id=23 removed by soft count = 257\n",
      "--- number of probes: 24 , kept by blast: 14 , if remove dups: 14\n",
      "-- number of probes kept for this region: 14\n",
      "-- checking probes in region: NRNX1-seg-279\n",
      "--- gene=NRNX1-seg-279, id=0 removed by soft count = 129\n",
      "--- gene=NRNX1-seg-279, id=1 removed by soft count = 148\n",
      "--- gene=NRNX1-seg-279, id=3 removed by soft count = 35095\n",
      "--- gene=NRNX1-seg-279, id=4 removed by soft count = 1411\n",
      "--- gene=NRNX1-seg-279, id=5 removed by soft count = 198\n",
      "--- gene=NRNX1-seg-279, id=7 removed by soft count = 5909\n",
      "--- gene=NRNX1-seg-279, id=8 removed by soft count = 4352\n",
      "--- gene=NRNX1-seg-279, id=9 removed by soft count = 60\n",
      "--- gene=NRNX1-seg-279, id=11 removed by soft count = 318\n",
      "--- number of probes: 28 , kept by blast: 19 , if remove dups: 19\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: NRNX1-seg-280\n",
      "--- gene=NRNX1-seg-280, id=16 removed by soft count = 1058\n",
      "--- gene=NRNX1-seg-280, id=18 removed by soft count = 137\n",
      "--- number of probes: 50 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: NRNX1-seg-281\n",
      "--- number of probes: 63 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-282\n",
      "--- gene=NRNX1-seg-282, id=8 removed by soft count = 314\n",
      "--- gene=NRNX1-seg-282, id=10 removed by soft count = 107\n",
      "--- gene=NRNX1-seg-282, id=12 removed by soft count = 387\n",
      "--- gene=NRNX1-seg-282, id=13 removed by soft count = 200\n",
      "--- gene=NRNX1-seg-282, id=15 removed by soft count = 2078\n",
      "--- gene=NRNX1-seg-282, id=40 removed by soft count = 102\n",
      "--- gene=NRNX1-seg-282, id=41 removed by soft count = 157\n",
      "--- gene=NRNX1-seg-282, id=42 removed by soft count = 77\n",
      "--- gene=NRNX1-seg-282, id=44 removed by soft count = 167\n",
      "--- gene=NRNX1-seg-282, id=45 removed by soft count = 86\n",
      "--- number of probes: 46 , kept by blast: 36 , if remove dups: 36\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: NRNX1-seg-283\n",
      "--- gene=NRNX1-seg-283, id=1 removed by soft count = 140\n",
      "--- gene=NRNX1-seg-283, id=20 removed by soft count = 80\n",
      "--- number of probes: 57 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-284\n",
      "--- gene=NRNX1-seg-284, id=13 removed by soft count = 54\n",
      "--- number of probes: 61 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-285\n",
      "--- gene=NRNX1-seg-285, id=19 removed by soft count = 93\n",
      "--- gene=NRNX1-seg-285, id=20 removed by soft count = 82\n",
      "--- gene=NRNX1-seg-285, id=21 removed by soft count = 123\n",
      "--- gene=NRNX1-seg-285, id=22 removed by soft count = 68\n",
      "--- gene=NRNX1-seg-285, id=37 removed by soft count = 74\n",
      "--- gene=NRNX1-seg-285, id=38 removed by soft count = 129\n",
      "--- gene=NRNX1-seg-285, id=46 removed by soft count = 162\n",
      "--- number of probes: 51 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-286\n",
      "--- gene=NRNX1-seg-286, id=0 removed by soft count = 65\n",
      "--- gene=NRNX1-seg-286, id=2 removed by soft count = 74\n",
      "--- gene=NRNX1-seg-286, id=4 removed by soft count = 91\n",
      "--- number of probes: 39 , kept by blast: 36 , if remove dups: 36\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: NRNX1-seg-287\n",
      "--- gene=NRNX1-seg-287, id=0 removed by soft count = 6279\n",
      "--- number of probes: 26 , kept by blast: 25 , if remove dups: 25\n",
      "-- number of probes kept for this region: 25\n",
      "-- checking probes in region: NRNX1-seg-288\n",
      "--- number of probes: 29 , kept by blast: 29 , if remove dups: 29\n",
      "-- number of probes kept for this region: 29\n",
      "-- checking probes in region: NRNX1-seg-289\n",
      "--- gene=NRNX1-seg-289, id=4 removed by soft count = 174\n",
      "--- gene=NRNX1-seg-289, id=5 removed by soft count = 580\n",
      "--- gene=NRNX1-seg-289, id=30 removed by soft count = 78\n",
      "--- gene=NRNX1-seg-289, id=34 removed by soft count = 172\n",
      "--- number of probes: 45 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: NRNX1-seg-290\n",
      "--- gene=NRNX1-seg-290, id=13 removed by soft count = 221\n",
      "--- number of probes: 36 , kept by blast: 35 , if remove dups: 35\n",
      "-- number of probes kept for this region: 35\n",
      "-- checking probes in region: NRNX1-seg-291\n",
      "--- number of probes: 39 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-292\n",
      "--- number of probes: 57 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-293\n",
      "--- gene=NRNX1-seg-293, id=12 removed by soft count = 107\n",
      "--- gene=NRNX1-seg-293, id=13 removed by soft count = 92\n",
      "--- gene=NRNX1-seg-293, id=14 removed by soft count = 161\n",
      "--- gene=NRNX1-seg-293, id=16 removed by soft count = 59\n",
      "--- gene=NRNX1-seg-293, id=18 removed by soft count = 111\n",
      "--- gene=NRNX1-seg-293, id=19 removed by soft count = 84\n",
      "--- gene=NRNX1-seg-293, id=23 removed by soft count = 82\n",
      "--- gene=NRNX1-seg-293, id=24 removed by soft count = 69\n",
      "--- gene=NRNX1-seg-293, id=25 removed by soft count = 88\n",
      "--- gene=NRNX1-seg-293, id=26 removed by soft count = 315\n",
      "--- number of probes: 27 , kept by blast: 17 , if remove dups: 17\n",
      "-- number of probes kept for this region: 17\n",
      "-- checking probes in region: NRNX1-seg-294\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=NRNX1-seg-294, id=0 removed by soft count = 299\n",
      "--- gene=NRNX1-seg-294, id=1 removed by soft count = 155\n",
      "--- gene=NRNX1-seg-294, id=2 removed by soft count = 422\n",
      "--- gene=NRNX1-seg-294, id=3 removed by soft count = 707\n",
      "--- gene=NRNX1-seg-294, id=6 removed by soft count = 69\n",
      "--- gene=NRNX1-seg-294, id=7 removed by soft count = 194\n",
      "--- gene=NRNX1-seg-294, id=8 removed by soft count = 860\n",
      "--- gene=NRNX1-seg-294, id=10 removed by soft count = 113\n",
      "--- gene=NRNX1-seg-294, id=11 removed by soft count = 449\n",
      "--- gene=NRNX1-seg-294, id=15 removed by soft count = 334\n",
      "--- gene=NRNX1-seg-294, id=30 removed by soft count = 71\n",
      "--- number of probes: 31 , kept by blast: 20 , if remove dups: 20\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: NRNX1-seg-295\n",
      "--- gene=NRNX1-seg-295, id=1 removed by soft count = 139\n",
      "--- gene=NRNX1-seg-295, id=8 removed by soft count = 82\n",
      "--- gene=NRNX1-seg-295, id=21 removed by soft count = 63\n",
      "--- gene=NRNX1-seg-295, id=42 removed by soft count = 227\n",
      "--- number of probes: 44 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: NRNX1-seg-298\n",
      "--- gene=NRNX1-seg-298, id=33 removed by soft count = 131\n",
      "--- gene=NRNX1-seg-298, id=37 removed by soft count = 153\n",
      "--- number of probes: 42 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: NRNX1-seg-299\n",
      "--- gene=NRNX1-seg-299, id=31 removed by soft count = 131\n",
      "--- gene=NRNX1-seg-299, id=32 removed by soft count = 73\n",
      "--- gene=NRNX1-seg-299, id=34 removed by soft count = 115\n",
      "--- gene=NRNX1-seg-299, id=35 removed by soft count = 170\n",
      "--- gene=NRNX1-seg-299, id=38 removed by soft count = 277\n",
      "--- number of probes: 39 , kept by blast: 34 , if remove dups: 34\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: NRNX1-seg-300\n",
      "--- gene=NRNX1-seg-300, id=0 removed by soft count = 162\n",
      "--- gene=NRNX1-seg-300, id=1 removed by soft count = 5361\n",
      "--- gene=NRNX1-seg-300, id=4 removed by soft count = 201\n",
      "--- number of probes: 36 , kept by blast: 33 , if remove dups: 33\n",
      "-- number of probes kept for this region: 33\n",
      "-- checking probes in region: NRNX1-seg-301\n",
      "--- gene=NRNX1-seg-301, id=39 removed by soft count = 52\n",
      "--- number of probes: 54 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-302\n",
      "--- gene=NRNX1-seg-302, id=5 removed by soft count = 54\n",
      "--- gene=NRNX1-seg-302, id=7 removed by soft count = 51\n",
      "--- gene=NRNX1-seg-302, id=8 removed by soft count = 128\n",
      "--- number of probes: 31 , kept by blast: 28 , if remove dups: 28\n",
      "-- number of probes kept for this region: 28\n",
      "-- checking probes in region: NRNX1-seg-303\n",
      "--- number of probes: 50 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-304\n",
      "--- gene=NRNX1-seg-304, id=45 removed by soft count = 75\n",
      "--- number of probes: 58 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-305\n",
      "--- number of probes: 46 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-306\n",
      "--- gene=NRNX1-seg-306, id=42 removed by soft count = 3209\n",
      "--- gene=NRNX1-seg-306, id=43 removed by soft count = 419\n",
      "--- gene=NRNX1-seg-306, id=44 removed by soft count = 80\n",
      "--- gene=NRNX1-seg-306, id=45 removed by soft count = 1828\n",
      "--- gene=NRNX1-seg-306, id=46 removed by soft count = 1039\n",
      "--- gene=NRNX1-seg-306, id=47 removed by soft count = 172\n",
      "--- number of probes: 49 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-307\n",
      "--- gene=NRNX1-seg-307, id=0 removed by soft count = 700\n",
      "--- gene=NRNX1-seg-307, id=1 removed by soft count = 289\n",
      "--- gene=NRNX1-seg-307, id=2 removed by soft count = 290\n",
      "--- gene=NRNX1-seg-307, id=3 removed by soft count = 330\n",
      "--- gene=NRNX1-seg-307, id=9 removed by soft count = 137\n",
      "--- gene=NRNX1-seg-307, id=17 removed by soft count = 52\n",
      "--- gene=NRNX1-seg-307, id=18 removed by soft count = 210\n",
      "--- gene=NRNX1-seg-307, id=19 removed by soft count = 101\n",
      "--- gene=NRNX1-seg-307, id=20 removed by soft count = 95\n",
      "--- number of probes: 22 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: NRNX1-seg-308\n",
      "--- gene=NRNX1-seg-308, id=2 removed by soft count = 224\n",
      "--- gene=NRNX1-seg-308, id=4 removed by soft count = 612\n",
      "--- gene=NRNX1-seg-308, id=6 removed by soft count = 124\n",
      "--- gene=NRNX1-seg-308, id=7 removed by soft count = 881\n",
      "--- gene=NRNX1-seg-308, id=9 removed by soft count = 495\n",
      "--- gene=NRNX1-seg-308, id=10 removed by soft count = 239\n",
      "--- gene=NRNX1-seg-308, id=11 removed by soft count = 198\n",
      "--- gene=NRNX1-seg-308, id=12 removed by soft count = 166\n",
      "--- gene=NRNX1-seg-308, id=13 removed by soft count = 419\n",
      "--- gene=NRNX1-seg-308, id=14 removed by soft count = 312\n",
      "--- gene=NRNX1-seg-308, id=21 removed by soft count = 772\n",
      "--- gene=NRNX1-seg-308, id=22 removed by soft count = 352\n",
      "--- number of probes: 24 , kept by blast: 12 , if remove dups: 12\n",
      "-- number of probes kept for this region: 12\n",
      "-- checking probes in region: NRNX1-seg-311\n",
      "--- gene=NRNX1-seg-311, id=38 removed by soft count = 4523\n",
      "--- number of probes: 58 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-312\n",
      "--- number of probes: 57 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-313\n",
      "--- number of probes: 76 , kept by blast: 76 , if remove dups: 76\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-314\n",
      "--- number of probes: 54 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-315\n",
      "--- gene=NRNX1-seg-315, id=15 removed by soft count = 598\n",
      "--- gene=NRNX1-seg-315, id=17 removed by soft count = 289\n",
      "--- gene=NRNX1-seg-315, id=18 removed by soft count = 139\n",
      "--- gene=NRNX1-seg-315, id=19 removed by soft count = 67\n",
      "--- number of probes: 42 , kept by blast: 38 , if remove dups: 38\n",
      "-- number of probes kept for this region: 38\n",
      "-- checking probes in region: NRNX1-seg-316\n",
      "--- gene=NRNX1-seg-316, id=36 removed by soft count = 287\n",
      "--- gene=NRNX1-seg-316, id=37 removed by soft count = 95\n",
      "--- gene=NRNX1-seg-316, id=38 removed by soft count = 60\n",
      "--- number of probes: 48 , kept by blast: 45 , if remove dups: 44\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-317\n",
      "--- gene=NRNX1-seg-317, id=2 removed by soft count = 71\n",
      "--- gene=NRNX1-seg-317, id=18 removed by soft count = 1363\n",
      "--- number of probes: 44 , kept by blast: 42 , if remove dups: 42\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: NRNX1-seg-318\n",
      "--- gene=NRNX1-seg-318, id=3 removed by soft count = 467\n",
      "--- gene=NRNX1-seg-318, id=32 removed by soft count = 76\n",
      "--- number of probes: 34 , kept by blast: 32 , if remove dups: 32\n",
      "-- number of probes kept for this region: 32\n",
      "-- checking probes in region: NRNX1-seg-319\n",
      "--- gene=NRNX1-seg-319, id=19 removed by soft count = 64\n",
      "--- gene=NRNX1-seg-319, id=21 removed by soft count = 129\n",
      "--- gene=NRNX1-seg-319, id=23 removed by soft count = 319\n",
      "--- gene=NRNX1-seg-319, id=24 removed by soft count = 368\n",
      "--- gene=NRNX1-seg-319, id=26 removed by soft count = 68\n",
      "--- gene=NRNX1-seg-319, id=27 removed by soft count = 2315\n",
      "--- gene=NRNX1-seg-319, id=28 removed by soft count = 441\n",
      "--- gene=NRNX1-seg-319, id=29 removed by soft count = 524\n",
      "--- gene=NRNX1-seg-319, id=30 removed by soft count = 86\n",
      "--- gene=NRNX1-seg-319, id=31 removed by soft count = 405\n",
      "--- gene=NRNX1-seg-319, id=32 removed by soft count = 461\n",
      "--- gene=NRNX1-seg-319, id=33 removed by soft count = 1000\n",
      "--- gene=NRNX1-seg-319, id=34 removed by soft count = 58\n",
      "--- number of probes: 35 , kept by blast: 22 , if remove dups: 22\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: NRNX1-seg-320\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=NRNX1-seg-320, id=19 removed by soft count = 84\n",
      "--- gene=NRNX1-seg-320, id=20 removed by soft count = 120\n",
      "--- number of probes: 50 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: NRNX1-seg-321\n",
      "--- number of probes: 69 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-322\n",
      "--- number of probes: 55 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-323\n",
      "--- gene=NRNX1-seg-323, id=8 removed by soft count = 505\n",
      "--- number of probes: 41 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: NRNX1-seg-324\n",
      "--- number of probes: 57 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-325\n",
      "--- gene=NRNX1-seg-325, id=30 removed by soft count = 172\n",
      "--- gene=NRNX1-seg-325, id=32 removed by soft count = 60\n",
      "--- number of probes: 59 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-326\n",
      "--- number of probes: 64 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-327\n",
      "--- gene=NRNX1-seg-327, id=39 removed by soft count = 53\n",
      "--- gene=NRNX1-seg-327, id=62 removed by soft count = 271\n",
      "--- gene=NRNX1-seg-327, id=63 removed by soft count = 222\n",
      "--- number of probes: 64 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-328\n",
      "--- gene=NRNX1-seg-328, id=0 removed by soft count = 57\n",
      "--- gene=NRNX1-seg-328, id=24 removed by soft count = 196\n",
      "--- gene=NRNX1-seg-328, id=25 removed by soft count = 477\n",
      "--- gene=NRNX1-seg-328, id=26 removed by soft count = 211\n",
      "--- gene=NRNX1-seg-328, id=27 removed by soft count = 250\n",
      "--- gene=NRNX1-seg-328, id=29 removed by soft count = 100\n",
      "--- gene=NRNX1-seg-328, id=30 removed by soft count = 275\n",
      "--- number of probes: 34 , kept by blast: 27 , if remove dups: 27\n",
      "-- number of probes kept for this region: 27\n",
      "-- checking probes in region: NRNX1-seg-329\n",
      "--- gene=NRNX1-seg-329, id=3 removed by soft count = 103\n",
      "--- number of probes: 61 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-330\n",
      "--- gene=NRNX1-seg-330, id=6 removed by soft count = 116\n",
      "--- gene=NRNX1-seg-330, id=9 removed by soft count = 292\n",
      "--- gene=NRNX1-seg-330, id=10 removed by soft count = 70\n",
      "--- gene=NRNX1-seg-330, id=11 removed by soft count = 127\n",
      "--- gene=NRNX1-seg-330, id=12 removed by soft count = 416\n",
      "--- number of probes: 31 , kept by blast: 26 , if remove dups: 26\n",
      "-- number of probes kept for this region: 26\n",
      "-- checking probes in region: NRNX1-seg-331\n",
      "--- number of probes: 53 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-332\n",
      "--- gene=NRNX1-seg-332, id=52 removed by soft count = 1170\n",
      "--- gene=NRNX1-seg-332, id=53 removed by soft count = 528\n",
      "--- number of probes: 60 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-333\n",
      "--- gene=NRNX1-seg-333, id=3 removed by soft count = 86\n",
      "--- gene=NRNX1-seg-333, id=41 removed by soft count = 630\n",
      "--- gene=NRNX1-seg-333, id=42 removed by soft count = 196\n",
      "--- gene=NRNX1-seg-333, id=48 removed by soft count = 87\n",
      "--- gene=NRNX1-seg-333, id=51 removed by soft count = 129\n",
      "--- number of probes: 52 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-334\n",
      "--- gene=NRNX1-seg-334, id=1 removed by soft count = 106\n",
      "--- gene=NRNX1-seg-334, id=2 removed by soft count = 187\n",
      "--- gene=NRNX1-seg-334, id=6 removed by soft count = 66\n",
      "--- gene=NRNX1-seg-334, id=21 removed by soft count = 91\n",
      "--- gene=NRNX1-seg-334, id=22 removed by soft count = 425\n",
      "--- gene=NRNX1-seg-334, id=27 removed by soft count = 130\n",
      "--- gene=NRNX1-seg-334, id=29 removed by soft count = 60\n",
      "--- gene=NRNX1-seg-334, id=30 removed by soft count = 132\n",
      "--- gene=NRNX1-seg-334, id=43 removed by soft count = 150\n",
      "--- gene=NRNX1-seg-334, id=45 removed by soft count = 69\n",
      "--- gene=NRNX1-seg-334, id=46 removed by soft count = 77\n",
      "--- gene=NRNX1-seg-334, id=48 removed by soft count = 715\n",
      "--- gene=NRNX1-seg-334, id=52 removed by soft count = 61\n",
      "--- gene=NRNX1-seg-334, id=53 removed by soft count = 258\n",
      "--- gene=NRNX1-seg-334, id=56 removed by soft count = 95\n",
      "--- gene=NRNX1-seg-334, id=57 removed by soft count = 126\n",
      "--- gene=NRNX1-seg-334, id=58 removed by soft count = 3085\n",
      "--- number of probes: 65 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: NRNX1-seg-335\n",
      "--- number of probes: 52 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-336\n",
      "--- number of probes: 61 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-337\n",
      "--- gene=NRNX1-seg-337, id=1 removed by soft count = 96\n",
      "--- gene=NRNX1-seg-337, id=7 removed by soft count = 72\n",
      "--- gene=NRNX1-seg-337, id=8 removed by soft count = 143\n",
      "--- gene=NRNX1-seg-337, id=31 removed by soft count = 165\n",
      "--- number of probes: 35 , kept by blast: 31 , if remove dups: 31\n",
      "-- number of probes kept for this region: 31\n",
      "-- checking probes in region: NRNX1-seg-338\n",
      "--- gene=NRNX1-seg-338, id=0 removed by soft count = 571\n",
      "--- gene=NRNX1-seg-338, id=2 removed by soft count = 402\n",
      "--- gene=NRNX1-seg-338, id=3 removed by soft count = 106\n",
      "--- gene=NRNX1-seg-338, id=6 removed by soft count = 64\n",
      "--- gene=NRNX1-seg-338, id=39 removed by soft count = 145\n",
      "--- gene=NRNX1-seg-338, id=42 removed by soft count = 78\n",
      "--- gene=NRNX1-seg-338, id=48 removed by soft count = 226\n",
      "--- number of probes: 49 , kept by blast: 42 , if remove dups: 42\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: NRNX1-seg-339\n",
      "--- gene=NRNX1-seg-339, id=0 removed by soft count = 1584\n",
      "--- gene=NRNX1-seg-339, id=19 removed by soft count = 287\n",
      "--- gene=NRNX1-seg-339, id=20 removed by soft count = 800\n",
      "--- gene=NRNX1-seg-339, id=21 removed by soft count = 317\n",
      "--- number of probes: 50 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-340\n",
      "--- number of probes: 46 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-341\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-342\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-343\n",
      "--- gene=NRNX1-seg-343, id=35 removed by soft count = 72\n",
      "--- gene=NRNX1-seg-343, id=41 removed by soft count = 65\n",
      "--- gene=NRNX1-seg-343, id=42 removed by soft count = 52\n",
      "--- gene=NRNX1-seg-343, id=45 removed by soft count = 64\n",
      "--- gene=NRNX1-seg-343, id=46 removed by soft count = 109\n",
      "--- gene=NRNX1-seg-343, id=47 removed by soft count = 128\n",
      "--- gene=NRNX1-seg-343, id=49 removed by soft count = 67\n",
      "--- number of probes: 50 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-344\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 56 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-345\n",
      "--- gene=NRNX1-seg-345, id=26 removed by soft count = 60\n",
      "--- number of probes: 38 , kept by blast: 37 , if remove dups: 37\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: NRNX1-seg-346\n",
      "--- gene=NRNX1-seg-346, id=14 removed by soft count = 874\n",
      "--- gene=NRNX1-seg-346, id=34 removed by soft count = 2375\n",
      "--- number of probes: 42 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: NRNX1-seg-347\n",
      "--- gene=NRNX1-seg-347, id=4 removed by soft count = 468\n",
      "--- gene=NRNX1-seg-347, id=6 removed by soft count = 56\n",
      "--- gene=NRNX1-seg-347, id=7 removed by soft count = 247\n",
      "--- gene=NRNX1-seg-347, id=12 removed by soft count = 876\n",
      "--- gene=NRNX1-seg-347, id=13 removed by soft count = 2414\n",
      "--- gene=NRNX1-seg-347, id=14 removed by soft count = 504\n",
      "--- gene=NRNX1-seg-347, id=15 removed by soft count = 2529\n",
      "--- gene=NRNX1-seg-347, id=16 removed by soft count = 1294\n",
      "--- number of probes: 25 , kept by blast: 17 , if remove dups: 17\n",
      "-- number of probes kept for this region: 17\n",
      "-- checking probes in region: NRNX1-seg-348\n",
      "--- gene=NRNX1-seg-348, id=15 removed by soft count = 160\n",
      "--- gene=NRNX1-seg-348, id=26 removed by soft count = 175\n",
      "--- gene=NRNX1-seg-348, id=37 removed by soft count = 58\n",
      "--- gene=NRNX1-seg-348, id=39 removed by soft count = 113\n",
      "--- gene=NRNX1-seg-348, id=40 removed by soft count = 2623\n",
      "--- number of probes: 44 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-349\n",
      "--- number of probes: 35 , kept by blast: 35 , if remove dups: 35\n",
      "-- number of probes kept for this region: 35\n",
      "-- checking probes in region: NRNX1-seg-350\n",
      "--- gene=NRNX1-seg-350, id=5 removed by soft count = 344\n",
      "--- gene=NRNX1-seg-350, id=10 removed by soft count = 204\n",
      "--- number of probes: 37 , kept by blast: 35 , if remove dups: 35\n",
      "-- number of probes kept for this region: 35\n",
      "-- checking probes in region: NRNX1-seg-351\n",
      "--- number of probes: 53 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-352\n",
      "--- gene=NRNX1-seg-352, id=21 removed by soft count = 3022\n",
      "--- number of probes: 36 , kept by blast: 35 , if remove dups: 35\n",
      "-- number of probes kept for this region: 35\n",
      "-- checking probes in region: NRNX1-seg-353\n",
      "--- gene=NRNX1-seg-353, id=43 removed by soft count = 276\n",
      "--- number of probes: 59 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-354\n",
      "--- number of probes: 50 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-355\n",
      "--- gene=NRNX1-seg-355, id=13 removed by soft count = 625\n",
      "--- gene=NRNX1-seg-355, id=18 removed by soft count = 54\n",
      "--- gene=NRNX1-seg-355, id=21 removed by soft count = 62\n",
      "--- number of probes: 60 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-356\n",
      "--- gene=NRNX1-seg-356, id=19 removed by soft count = 262\n",
      "--- gene=NRNX1-seg-356, id=23 removed by soft count = 96\n",
      "--- number of probes: 40 , kept by blast: 38 , if remove dups: 38\n",
      "-- number of probes kept for this region: 38\n",
      "-- checking probes in region: NRNX1-seg-357\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-358\n",
      "--- gene=NRNX1-seg-358, id=0 removed by soft count = 55\n",
      "--- gene=NRNX1-seg-358, id=37 removed by soft count = 150\n",
      "--- gene=NRNX1-seg-358, id=38 removed by soft count = 483\n",
      "--- gene=NRNX1-seg-358, id=40 removed by soft count = 60\n",
      "--- number of probes: 47 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-359\n",
      "--- gene=NRNX1-seg-359, id=20 removed by soft count = 66\n",
      "--- gene=NRNX1-seg-359, id=28 removed by soft count = 77\n",
      "--- gene=NRNX1-seg-359, id=29 removed by soft count = 149\n",
      "--- gene=NRNX1-seg-359, id=32 removed by soft count = 72\n",
      "--- gene=NRNX1-seg-359, id=37 removed by soft count = 352\n",
      "--- gene=NRNX1-seg-359, id=38 removed by soft count = 72\n",
      "--- number of probes: 52 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-360\n",
      "--- number of probes: 56 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-361\n",
      "--- gene=NRNX1-seg-361, id=31 removed by soft count = 597\n",
      "--- gene=NRNX1-seg-361, id=32 removed by soft count = 946\n",
      "--- number of probes: 61 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: NRNX1-seg-362\n",
      "--- gene=NRNX1-seg-362, id=8 removed by soft count = 1660\n",
      "--- gene=NRNX1-seg-362, id=9 removed by soft count = 299\n",
      "--- gene=NRNX1-seg-362, id=10 removed by soft count = 111\n",
      "--- number of probes: 40 , kept by blast: 37 , if remove dups: 37\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: NRNX1-seg-363\n",
      "--- number of probes: 55 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-364\n",
      "--- gene=NRNX1-seg-364, id=15 removed by soft count = 881\n",
      "--- gene=NRNX1-seg-364, id=16 removed by soft count = 384\n",
      "--- gene=NRNX1-seg-364, id=25 removed by soft count = 257\n",
      "--- number of probes: 57 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-365\n",
      "--- gene=NRNX1-seg-365, id=36 removed by soft count = 227\n",
      "--- number of probes: 37 , kept by blast: 36 , if remove dups: 36\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: NRNX1-seg-366\n",
      "--- gene=NRNX1-seg-366, id=0 removed by soft count = 9640\n",
      "--- gene=NRNX1-seg-366, id=1 removed by soft count = 3991\n",
      "--- gene=NRNX1-seg-366, id=2 removed by soft count = 793\n",
      "--- gene=NRNX1-seg-366, id=4 removed by soft count = 62\n",
      "--- gene=NRNX1-seg-366, id=5 removed by soft count = 124\n",
      "--- gene=NRNX1-seg-366, id=6 removed by soft count = 57\n",
      "--- number of probes: 42 , kept by blast: 36 , if remove dups: 36\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: NRNX1-seg-367\n",
      "--- gene=NRNX1-seg-367, id=33 removed by soft count = 1222\n",
      "--- number of probes: 64 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-368\n",
      "--- number of probes: 47 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-369\n",
      "--- gene=NRNX1-seg-369, id=4 removed by soft count = 53\n",
      "--- gene=NRNX1-seg-369, id=43 removed by soft count = 71\n",
      "--- gene=NRNX1-seg-369, id=50 removed by soft count = 55\n",
      "--- number of probes: 56 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-370\n",
      "--- gene=NRNX1-seg-370, id=39 removed by soft count = 3311\n",
      "--- gene=NRNX1-seg-370, id=42 removed by soft count = 911\n",
      "--- number of probes: 51 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-371\n",
      "--- gene=NRNX1-seg-371, id=43 removed by soft count = 2738\n",
      "--- number of probes: 47 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-372\n",
      "--- gene=NRNX1-seg-372, id=32 removed by soft count = 235\n",
      "--- number of probes: 65 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-373\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=NRNX1-seg-373, id=43 removed by soft count = 193\n",
      "--- number of probes: 55 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-374\n",
      "--- gene=NRNX1-seg-374, id=13 removed by soft count = 709\n",
      "--- gene=NRNX1-seg-374, id=15 removed by soft count = 470\n",
      "--- gene=NRNX1-seg-374, id=16 removed by soft count = 62\n",
      "--- gene=NRNX1-seg-374, id=17 removed by soft count = 87\n",
      "--- gene=NRNX1-seg-374, id=18 removed by soft count = 122\n",
      "--- gene=NRNX1-seg-374, id=20 removed by soft count = 797\n",
      "--- gene=NRNX1-seg-374, id=21 removed by soft count = 309\n",
      "--- gene=NRNX1-seg-374, id=22 removed by soft count = 92\n",
      "--- gene=NRNX1-seg-374, id=23 removed by soft count = 376\n",
      "--- gene=NRNX1-seg-374, id=24 removed by soft count = 195\n",
      "--- number of probes: 31 , kept by blast: 21 , if remove dups: 20\n",
      "-- number of probes kept for this region: 21\n",
      "-- checking probes in region: NRNX1-seg-375\n",
      "--- gene=NRNX1-seg-375, id=7 removed by soft count = 18080\n",
      "--- gene=NRNX1-seg-375, id=20 removed by soft count = 150\n",
      "--- number of probes: 21 , kept by blast: 19 , if remove dups: 19\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: NRNX1-seg-376\n",
      "--- gene=NRNX1-seg-376, id=2 removed by soft count = 402\n",
      "--- gene=NRNX1-seg-376, id=7 removed by soft count = 73\n",
      "--- gene=NRNX1-seg-376, id=9 removed by soft count = 120\n",
      "--- gene=NRNX1-seg-376, id=10 removed by soft count = 367\n",
      "--- gene=NRNX1-seg-376, id=11 removed by soft count = 197\n",
      "--- gene=NRNX1-seg-376, id=13 removed by soft count = 54\n",
      "--- gene=NRNX1-seg-376, id=15 removed by soft count = 53\n",
      "--- gene=NRNX1-seg-376, id=16 removed by soft count = 51\n",
      "--- gene=NRNX1-seg-376, id=18 removed by soft count = 53\n",
      "--- gene=NRNX1-seg-376, id=19 removed by soft count = 57\n",
      "--- gene=NRNX1-seg-376, id=22 removed by soft count = 162\n",
      "--- gene=NRNX1-seg-376, id=23 removed by soft count = 174\n",
      "--- gene=NRNX1-seg-376, id=24 removed by soft count = 540\n",
      "--- number of probes: 25 , kept by blast: 12 , if remove dups: 12\n",
      "-- number of probes kept for this region: 12\n",
      "-- checking probes in region: NRNX1-seg-379\n",
      "--- gene=NRNX1-seg-379, id=3 removed by soft count = 53\n",
      "--- gene=NRNX1-seg-379, id=8 removed by soft count = 319\n",
      "--- gene=NRNX1-seg-379, id=9 removed by soft count = 150\n",
      "--- gene=NRNX1-seg-379, id=12 removed by soft count = 61\n",
      "--- number of probes: 30 , kept by blast: 26 , if remove dups: 26\n",
      "-- number of probes kept for this region: 26\n",
      "-- checking probes in region: NRNX1-seg-380\n",
      "--- gene=NRNX1-seg-380, id=26 removed by soft count = 214\n",
      "--- gene=NRNX1-seg-380, id=28 removed by soft count = 444\n",
      "--- gene=NRNX1-seg-380, id=35 removed by soft count = 1569\n",
      "--- number of probes: 49 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-381\n",
      "--- gene=NRNX1-seg-381, id=35 removed by soft count = 101\n",
      "--- number of probes: 60 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: NRNX1-seg-382\n",
      "--- gene=NRNX1-seg-382, id=4 removed by soft count = 258\n",
      "--- number of probes: 43 , kept by blast: 42 , if remove dups: 42\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: NRNX1-seg-383\n",
      "--- gene=NRNX1-seg-383, id=1 removed by soft count = 762\n",
      "--- number of probes: 52 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: NRNX1-seg-384\n",
      "--- gene=NRNX1-seg-384, id=31 removed by soft count = 72\n",
      "--- number of probes: 41 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: NRNX1-seg-385\n",
      "--- number of probes: 55 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-386\n",
      "--- number of probes: 58 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-387\n",
      "--- gene=NRNX1-seg-387, id=19 removed by soft count = 2244\n",
      "--- number of probes: 45 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-388\n",
      "--- number of probes: 62 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-389\n",
      "--- number of probes: 60 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-390\n",
      "--- number of probes: 32 , kept by blast: 32 , if remove dups: 32\n",
      "-- number of probes kept for this region: 32\n",
      "-- checking probes in region: NRNX1-seg-391\n",
      "--- number of probes: 61 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-392\n",
      "--- number of probes: 67 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-393\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-394\n",
      "--- gene=NRNX1-seg-394, id=48 removed by soft count = 145\n",
      "--- number of probes: 53 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-395\n",
      "--- number of probes: 56 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-396\n",
      "--- gene=NRNX1-seg-396, id=20 removed by soft count = 68\n",
      "--- number of probes: 43 , kept by blast: 42 , if remove dups: 42\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: NRNX1-seg-397\n",
      "--- number of probes: 52 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-398\n",
      "--- number of probes: 47 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-399\n",
      "--- gene=NRNX1-seg-399, id=16 removed by soft count = 96\n",
      "--- gene=NRNX1-seg-399, id=17 removed by soft count = 153\n",
      "--- gene=NRNX1-seg-399, id=18 removed by soft count = 53\n",
      "--- gene=NRNX1-seg-399, id=19 removed by soft count = 242\n",
      "--- gene=NRNX1-seg-399, id=20 removed by soft count = 170\n",
      "--- gene=NRNX1-seg-399, id=21 removed by soft count = 84\n",
      "--- gene=NRNX1-seg-399, id=22 removed by soft count = 155\n",
      "--- gene=NRNX1-seg-399, id=26 removed by soft count = 265\n",
      "--- number of probes: 48 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: NRNX1-seg-400\n",
      "--- gene=NRNX1-seg-400, id=14 removed by soft count = 455\n",
      "--- number of probes: 55 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-401\n",
      "--- number of probes: 61 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-402\n",
      "--- gene=NRNX1-seg-402, id=9 removed by soft count = 55\n",
      "--- gene=NRNX1-seg-402, id=28 removed by soft count = 255\n",
      "--- number of probes: 55 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-403\n",
      "--- gene=NRNX1-seg-403, id=58 removed by soft count = 54\n",
      "--- number of probes: 64 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-404\n",
      "--- gene=NRNX1-seg-404, id=20 removed by soft count = 108\n",
      "--- gene=NRNX1-seg-404, id=21 removed by soft count = 245\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 58 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-405\n",
      "--- number of probes: 47 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-406\n",
      "--- gene=NRNX1-seg-406, id=26 removed by soft count = 2166\n",
      "--- gene=NRNX1-seg-406, id=45 removed by soft count = 66\n",
      "--- gene=NRNX1-seg-406, id=48 removed by soft count = 70\n",
      "--- number of probes: 54 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: NRNX1-seg-407\n",
      "--- gene=NRNX1-seg-407, id=2 removed by soft count = 163\n",
      "--- gene=NRNX1-seg-407, id=31 removed by soft count = 96\n",
      "--- gene=NRNX1-seg-407, id=32 removed by soft count = 899\n",
      "--- gene=NRNX1-seg-407, id=34 removed by soft count = 113\n",
      "--- number of probes: 39 , kept by blast: 35 , if remove dups: 35\n",
      "-- number of probes kept for this region: 35\n",
      "-- checking probes in region: NRNX1-seg-408\n",
      "--- gene=NRNX1-seg-408, id=33 removed by soft count = 59\n",
      "--- number of probes: 45 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-409\n",
      "--- number of probes: 44 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-410\n",
      "--- number of probes: 51 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: NRNX1-seg-411\n",
      "--- number of probes: 46 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-412\n",
      "--- gene=NRNX1-seg-412, id=34 removed by soft count = 72\n",
      "--- gene=NRNX1-seg-412, id=35 removed by soft count = 178\n",
      "--- gene=NRNX1-seg-412, id=36 removed by soft count = 81\n",
      "--- number of probes: 37 , kept by blast: 34 , if remove dups: 34\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: NRNX1-seg-413\n",
      "--- gene=NRNX1-seg-413, id=0 removed by soft count = 1054\n",
      "--- gene=NRNX1-seg-413, id=1 removed by soft count = 899\n",
      "--- number of probes: 41 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-414\n",
      "--- number of probes: 58 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-415\n",
      "--- gene=NRNX1-seg-415, id=12 removed by soft count = 127\n",
      "--- gene=NRNX1-seg-415, id=14 removed by soft count = 683\n",
      "--- number of probes: 49 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-416\n",
      "--- number of probes: 52 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-417\n",
      "--- gene=NRNX1-seg-417, id=0 removed by soft count = 196\n",
      "--- gene=NRNX1-seg-417, id=3 removed by soft count = 141\n",
      "--- gene=NRNX1-seg-417, id=36 removed by soft count = 1060\n",
      "--- gene=NRNX1-seg-417, id=39 removed by soft count = 595\n",
      "--- gene=NRNX1-seg-417, id=40 removed by soft count = 373\n",
      "--- gene=NRNX1-seg-417, id=41 removed by soft count = 1121\n",
      "--- number of probes: 45 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-418\n",
      "--- number of probes: 41 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: NRNX1-seg-419\n",
      "--- number of probes: 54 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-420\n",
      "--- number of probes: 55 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-421\n",
      "--- number of probes: 62 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-422\n",
      "--- number of probes: 48 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: NRNX1-seg-423\n",
      "--- gene=NRNX1-seg-423, id=48 removed by soft count = 110\n",
      "--- gene=NRNX1-seg-423, id=49 removed by soft count = 90\n",
      "--- gene=NRNX1-seg-423, id=51 removed by soft count = 422\n",
      "--- gene=NRNX1-seg-423, id=53 removed by soft count = 481\n",
      "--- number of probes: 54 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-424\n",
      "--- number of probes: 59 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: NRNX1-seg-425\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-426\n",
      "--- number of probes: 53 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-427\n",
      "--- number of probes: 67 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-428\n",
      "--- gene=NRNX1-seg-428, id=21 removed by soft count = 159\n",
      "--- number of probes: 46 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-429\n",
      "--- gene=NRNX1-seg-429, id=15 removed by soft count = 260\n",
      "--- gene=NRNX1-seg-429, id=19 removed by soft count = 162\n",
      "--- number of probes: 46 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-430\n",
      "--- number of probes: 58 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-431\n",
      "--- gene=NRNX1-seg-431, id=20 removed by soft count = 72\n",
      "--- gene=NRNX1-seg-431, id=28 removed by soft count = 57\n",
      "--- number of probes: 41 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-432\n",
      "--- number of probes: 55 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-433\n",
      "--- gene=NRNX1-seg-433, id=4 removed by soft count = 859\n",
      "--- number of probes: 54 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-434\n",
      "--- gene=NRNX1-seg-434, id=10 removed by soft count = 290\n",
      "--- gene=NRNX1-seg-434, id=11 removed by soft count = 84\n",
      "--- number of probes: 52 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-435\n",
      "--- gene=NRNX1-seg-435, id=0 removed by soft count = 231\n",
      "--- gene=NRNX1-seg-435, id=5 removed by soft count = 129\n",
      "--- gene=NRNX1-seg-435, id=6 removed by soft count = 142\n",
      "--- number of probes: 47 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-436\n",
      "--- number of probes: 52 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-437\n",
      "--- gene=NRNX1-seg-437, id=2 removed by soft count = 363\n",
      "--- number of probes: 41 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: NRNX1-seg-438\n",
      "--- gene=NRNX1-seg-438, id=29 removed by soft count = 19107\n",
      "--- number of probes: 34 , kept by blast: 33 , if remove dups: 33\n",
      "-- number of probes kept for this region: 33\n",
      "-- checking probes in region: NRNX1-seg-439\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 38 , kept by blast: 38 , if remove dups: 38\n",
      "-- number of probes kept for this region: 38\n",
      "-- checking probes in region: NRNX1-seg-440\n",
      "--- gene=NRNX1-seg-440, id=11 removed by soft count = 3361\n",
      "--- gene=NRNX1-seg-440, id=13 removed by soft count = 7160\n",
      "--- gene=NRNX1-seg-440, id=20 removed by soft count = 222\n",
      "--- gene=NRNX1-seg-440, id=21 removed by soft count = 612\n",
      "--- number of probes: 59 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-441\n",
      "--- number of probes: 57 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-442\n",
      "--- number of probes: 50 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-443\n",
      "--- number of probes: 39 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-444\n",
      "--- number of probes: 46 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-445\n",
      "--- number of probes: 62 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-446\n",
      "--- gene=NRNX1-seg-446, id=37 removed by soft count = 79\n",
      "--- number of probes: 61 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-447\n",
      "--- number of probes: 41 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: NRNX1-seg-448\n",
      "--- gene=NRNX1-seg-448, id=34 removed by soft count = 96\n",
      "--- number of probes: 53 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-449\n",
      "--- number of probes: 43 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-450\n",
      "--- gene=NRNX1-seg-450, id=31 removed by soft count = 71\n",
      "--- gene=NRNX1-seg-450, id=37 removed by soft count = 554\n",
      "--- gene=NRNX1-seg-450, id=38 removed by soft count = 1445\n",
      "--- gene=NRNX1-seg-450, id=50 removed by soft count = 169\n",
      "--- number of probes: 51 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-452\n",
      "--- gene=NRNX1-seg-452, id=32 removed by soft count = 239\n",
      "--- gene=NRNX1-seg-452, id=34 removed by soft count = 514\n",
      "--- gene=NRNX1-seg-452, id=35 removed by soft count = 267\n",
      "--- gene=NRNX1-seg-452, id=38 removed by soft count = 532\n",
      "--- gene=NRNX1-seg-452, id=40 removed by soft count = 162\n",
      "--- number of probes: 50 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-453\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-454\n",
      "--- number of probes: 48 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: NRNX1-seg-455\n",
      "--- number of probes: 43 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-456\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-457\n",
      "--- number of probes: 47 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-458\n",
      "--- gene=NRNX1-seg-458, id=0 removed by soft count = 65\n",
      "--- gene=NRNX1-seg-458, id=11 removed by soft count = 496\n",
      "--- gene=NRNX1-seg-458, id=27 removed by soft count = 538\n",
      "--- number of probes: 44 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: NRNX1-seg-459\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-460\n",
      "--- gene=NRNX1-seg-460, id=13 removed by soft count = 84\n",
      "--- gene=NRNX1-seg-460, id=15 removed by soft count = 164\n",
      "--- gene=NRNX1-seg-460, id=16 removed by soft count = 174\n",
      "--- number of probes: 49 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-461\n",
      "--- number of probes: 52 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-462\n",
      "--- gene=NRNX1-seg-462, id=30 removed by soft count = 277\n",
      "--- number of probes: 49 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: NRNX1-seg-463\n",
      "--- number of probes: 59 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: NRNX1-seg-464\n",
      "--- number of probes: 53 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-465\n",
      "--- number of probes: 60 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-466\n",
      "--- number of probes: 34 , kept by blast: 34 , if remove dups: 34\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: NRNX1-seg-467\n",
      "--- number of probes: 61 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-468\n",
      "--- number of probes: 51 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: NRNX1-seg-469\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-470\n",
      "--- number of probes: 79 , kept by blast: 79 , if remove dups: 79\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-471\n",
      "--- gene=NRNX1-seg-471, id=1 removed by soft count = 216\n",
      "--- gene=NRNX1-seg-471, id=2 removed by soft count = 89\n",
      "--- number of probes: 62 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-472\n",
      "--- gene=NRNX1-seg-472, id=20 removed by soft count = 183\n",
      "--- gene=NRNX1-seg-472, id=21 removed by soft count = 911\n",
      "--- gene=NRNX1-seg-472, id=22 removed by soft count = 1174\n",
      "--- gene=NRNX1-seg-472, id=23 removed by soft count = 810\n",
      "--- gene=NRNX1-seg-472, id=24 removed by soft count = 65\n",
      "--- gene=NRNX1-seg-472, id=25 removed by soft count = 623\n",
      "--- number of probes: 26 , kept by blast: 20 , if remove dups: 20\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: NRNX1-seg-474\n",
      "--- gene=NRNX1-seg-474, id=33 removed by soft count = 64\n",
      "--- number of probes: 61 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-475\n",
      "--- number of probes: 60 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-476\n",
      "--- gene=NRNX1-seg-476, id=20 removed by soft count = 1504\n",
      "--- gene=NRNX1-seg-476, id=30 removed by soft count = 346\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 51 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-477\n",
      "--- number of probes: 58 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-478\n",
      "--- number of probes: 52 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-479\n",
      "--- gene=NRNX1-seg-479, id=5 removed by soft count = 96\n",
      "--- gene=NRNX1-seg-479, id=6 removed by soft count = 3877\n",
      "--- number of probes: 58 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-480\n",
      "--- number of probes: 58 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-481\n",
      "--- number of probes: 66 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-482\n",
      "--- gene=NRNX1-seg-482, id=28 removed by soft count = 118\n",
      "--- gene=NRNX1-seg-482, id=30 removed by soft count = 163\n",
      "--- gene=NRNX1-seg-482, id=31 removed by soft count = 74\n",
      "--- number of probes: 60 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-483\n",
      "--- gene=NRNX1-seg-483, id=16 removed by soft count = 1622\n",
      "--- gene=NRNX1-seg-483, id=18 removed by soft count = 239\n",
      "--- gene=NRNX1-seg-483, id=20 removed by soft count = 170\n",
      "--- number of probes: 25 , kept by blast: 22 , if remove dups: 22\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: NRNX1-seg-484\n",
      "--- gene=NRNX1-seg-484, id=18 removed by soft count = 101\n",
      "--- gene=NRNX1-seg-484, id=19 removed by soft count = 279\n",
      "--- number of probes: 59 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-485\n",
      "--- number of probes: 63 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-486\n",
      "--- number of probes: 50 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-487\n",
      "--- number of probes: 64 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-488\n",
      "--- gene=NRNX1-seg-488, id=4 removed by soft count = 6620\n",
      "--- gene=NRNX1-seg-488, id=6 removed by soft count = 2635\n",
      "--- number of probes: 49 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-489\n",
      "--- number of probes: 57 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-490\n",
      "--- number of probes: 58 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-491\n",
      "--- number of probes: 50 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-492\n",
      "--- gene=NRNX1-seg-492, id=40 removed by soft count = 126\n",
      "--- number of probes: 47 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-493\n",
      "--- number of probes: 54 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-494\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-495\n",
      "--- gene=NRNX1-seg-495, id=15 removed by soft count = 149\n",
      "--- number of probes: 59 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-496\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-497\n",
      "--- gene=NRNX1-seg-497, id=31 removed by soft count = 82\n",
      "--- gene=NRNX1-seg-497, id=33 removed by soft count = 85\n",
      "--- gene=NRNX1-seg-497, id=34 removed by soft count = 521\n",
      "--- number of probes: 52 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-498\n",
      "--- number of probes: 80 , kept by blast: 80 , if remove dups: 80\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-499\n",
      "--- number of probes: 66 , kept by blast: 66 , if remove dups: 66\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-500\n",
      "--- number of probes: 67 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-501\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-502\n",
      "--- gene=NRNX1-seg-502, id=63 removed by soft count = 107\n",
      "--- number of probes: 65 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-503\n",
      "--- gene=NRNX1-seg-503, id=0 removed by soft count = 85\n",
      "--- number of probes: 49 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: NRNX1-seg-504\n",
      "--- gene=NRNX1-seg-504, id=0 removed by soft count = 252\n",
      "--- gene=NRNX1-seg-504, id=1 removed by soft count = 200\n",
      "--- gene=NRNX1-seg-504, id=2 removed by soft count = 63\n",
      "--- number of probes: 47 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-505\n",
      "--- number of probes: 44 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-506\n",
      "--- number of probes: 60 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-507\n",
      "--- number of probes: 58 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-508\n",
      "--- number of probes: 62 , kept by blast: 62 , if remove dups: 62\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-509\n",
      "--- gene=NRNX1-seg-509, id=9 removed by soft count = 1586\n",
      "--- number of probes: 45 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-510\n",
      "--- gene=NRNX1-seg-510, id=7 removed by soft count = 66\n",
      "--- number of probes: 55 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-511\n",
      "--- number of probes: 52 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-512\n",
      "--- number of probes: 56 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-513\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 52 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-514\n",
      "--- gene=NRNX1-seg-514, id=42 removed by soft count = 2718\n",
      "--- gene=NRNX1-seg-514, id=43 removed by soft count = 516\n",
      "--- gene=NRNX1-seg-514, id=44 removed by soft count = 106\n",
      "--- number of probes: 51 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: NRNX1-seg-515\n",
      "--- number of probes: 63 , kept by blast: 63 , if remove dups: 63\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-516\n",
      "--- gene=NRNX1-seg-516, id=46 removed by soft count = 3013\n",
      "--- number of probes: 52 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: NRNX1-seg-517\n",
      "--- number of probes: 59 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: NRNX1-seg-518\n",
      "--- number of probes: 55 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-519\n",
      "--- gene=NRNX1-seg-519, id=49 removed by soft count = 115\n",
      "--- number of probes: 62 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-520\n",
      "--- number of probes: 51 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: NRNX1-seg-521\n",
      "--- number of probes: 61 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-522\n",
      "--- gene=NRNX1-seg-522, id=6 removed by soft count = 202\n",
      "--- gene=NRNX1-seg-522, id=7 removed by soft count = 83\n",
      "--- gene=NRNX1-seg-522, id=8 removed by soft count = 138\n",
      "--- gene=NRNX1-seg-522, id=33 removed by soft count = 129\n",
      "--- number of probes: 50 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-523\n",
      "--- number of probes: 59 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: NRNX1-seg-524\n",
      "--- number of probes: 55 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-525\n",
      "--- number of probes: 58 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-526\n",
      "--- number of probes: 44 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-527\n",
      "--- number of probes: 57 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-528\n",
      "--- number of probes: 47 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-529\n",
      "--- number of probes: 50 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-530\n",
      "--- number of probes: 61 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-531\n",
      "--- gene=NRNX1-seg-531, id=30 removed by soft count = 164\n",
      "--- gene=NRNX1-seg-531, id=32 removed by soft count = 379\n",
      "--- gene=NRNX1-seg-531, id=39 removed by soft count = 117\n",
      "--- number of probes: 48 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-532\n",
      "--- number of probes: 61 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-533\n",
      "--- number of probes: 67 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-534\n",
      "--- number of probes: 52 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-535\n",
      "--- gene=NRNX1-seg-535, id=38 removed by soft count = 641\n",
      "--- number of probes: 39 , kept by blast: 38 , if remove dups: 38\n",
      "-- number of probes kept for this region: 38\n",
      "-- checking probes in region: NRNX1-seg-536\n",
      "--- gene=NRNX1-seg-536, id=1 removed by soft count = 309\n",
      "--- gene=NRNX1-seg-536, id=12 removed by soft count = 173\n",
      "--- gene=NRNX1-seg-536, id=14 removed by soft count = 487\n",
      "--- number of probes: 59 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-537\n",
      "--- number of probes: 51 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: NRNX1-seg-538\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-539\n",
      "--- gene=NRNX1-seg-539, id=44 removed by soft count = 373\n",
      "--- number of probes: 55 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-540\n",
      "--- number of probes: 54 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-541\n",
      "--- number of probes: 45 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-542\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-543\n",
      "--- gene=NRNX1-seg-543, id=15 removed by soft count = 387\n",
      "--- gene=NRNX1-seg-543, id=16 removed by soft count = 602\n",
      "--- gene=NRNX1-seg-543, id=19 removed by soft count = 214\n",
      "--- number of probes: 52 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-544\n",
      "--- gene=NRNX1-seg-544, id=28 removed by soft count = 141\n",
      "--- number of probes: 47 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-545\n",
      "--- number of probes: 60 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-546\n",
      "--- number of probes: 52 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-547\n",
      "--- gene=NRNX1-seg-547, id=34 removed by soft count = 104\n",
      "--- number of probes: 46 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-548\n",
      "--- gene=NRNX1-seg-548, id=17 removed by soft count = 375\n",
      "--- number of probes: 47 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-549\n",
      "--- number of probes: 46 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-550\n",
      "--- gene=NRNX1-seg-550, id=7 removed by soft count = 112\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 56 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-551\n",
      "--- number of probes: 48 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: NRNX1-seg-552\n",
      "--- gene=NRNX1-seg-552, id=38 removed by soft count = 188\n",
      "--- number of probes: 58 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-553\n",
      "--- gene=NRNX1-seg-553, id=19 removed by soft count = 88\n",
      "--- gene=NRNX1-seg-553, id=21 removed by soft count = 531\n",
      "--- gene=NRNX1-seg-553, id=22 removed by soft count = 63\n",
      "--- number of probes: 52 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-554\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-555\n",
      "--- number of probes: 59 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: NRNX1-seg-556\n",
      "--- gene=NRNX1-seg-556, id=51 removed by soft count = 649\n",
      "--- gene=NRNX1-seg-556, id=52 removed by soft count = 1280\n",
      "--- number of probes: 53 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: NRNX1-seg-557\n",
      "--- gene=NRNX1-seg-557, id=0 removed by soft count = 4519\n",
      "--- number of probes: 57 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-558\n",
      "--- gene=NRNX1-seg-558, id=3 removed by soft count = 107\n",
      "--- number of probes: 59 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-559\n",
      "--- gene=NRNX1-seg-559, id=44 removed by soft count = 239\n",
      "--- number of probes: 52 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: NRNX1-seg-560\n",
      "--- gene=NRNX1-seg-560, id=7 removed by soft count = 62\n",
      "--- gene=NRNX1-seg-560, id=24 removed by soft count = 117\n",
      "--- gene=NRNX1-seg-560, id=26 removed by soft count = 61\n",
      "--- gene=NRNX1-seg-560, id=58 removed by soft count = 67\n",
      "--- number of probes: 61 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-561\n",
      "--- number of probes: 40 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: NRNX1-seg-562\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-563\n",
      "--- number of probes: 43 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-564\n",
      "--- gene=NRNX1-seg-564, id=27 removed by soft count = 63\n",
      "--- gene=NRNX1-seg-564, id=30 removed by soft count = 1135\n",
      "--- gene=NRNX1-seg-564, id=32 removed by soft count = 119\n",
      "--- gene=NRNX1-seg-564, id=34 removed by soft count = 99\n",
      "--- gene=NRNX1-seg-564, id=37 removed by soft count = 1081\n",
      "--- gene=NRNX1-seg-564, id=39 removed by soft count = 352\n",
      "--- number of probes: 45 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-565\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-566\n",
      "--- gene=NRNX1-seg-566, id=3 removed by soft count = 57\n",
      "--- gene=NRNX1-seg-566, id=4 removed by soft count = 63\n",
      "--- gene=NRNX1-seg-566, id=6 removed by soft count = 139\n",
      "--- gene=NRNX1-seg-566, id=7 removed by soft count = 218\n",
      "--- number of probes: 41 , kept by blast: 37 , if remove dups: 37\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: NRNX1-seg-567\n",
      "--- gene=NRNX1-seg-567, id=34 removed by soft count = 444\n",
      "--- number of probes: 50 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-568\n",
      "--- gene=NRNX1-seg-568, id=33 removed by soft count = 342\n",
      "--- number of probes: 46 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-569\n",
      "--- gene=NRNX1-seg-569, id=54 removed by soft count = 148\n",
      "--- gene=NRNX1-seg-569, id=55 removed by soft count = 58\n",
      "--- gene=NRNX1-seg-569, id=56 removed by soft count = 108\n",
      "--- number of probes: 64 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-570\n",
      "--- gene=NRNX1-seg-570, id=23 removed by soft count = 91\n",
      "--- gene=NRNX1-seg-570, id=35 removed by soft count = 681\n",
      "--- gene=NRNX1-seg-570, id=53 removed by soft count = 51\n",
      "--- number of probes: 61 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-571\n",
      "--- number of probes: 54 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-572\n",
      "--- number of probes: 58 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-573\n",
      "--- number of probes: 61 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-574\n",
      "--- gene=NRNX1-seg-574, id=36 removed by soft count = 710\n",
      "--- number of probes: 43 , kept by blast: 42 , if remove dups: 42\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: NRNX1-seg-575\n",
      "--- number of probes: 47 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-576\n",
      "--- number of probes: 61 , kept by blast: 61 , if remove dups: 61\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-577\n",
      "--- number of probes: 45 , kept by blast: 45 , if remove dups: 44\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-578\n",
      "--- number of probes: 39 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-579\n",
      "--- number of probes: 41 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: NRNX1-seg-580\n",
      "--- number of probes: 52 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-581\n",
      "--- number of probes: 52 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-582\n",
      "--- gene=NRNX1-seg-582, id=23 removed by soft count = 51\n",
      "--- gene=NRNX1-seg-582, id=26 removed by soft count = 68\n",
      "--- gene=NRNX1-seg-582, id=27 removed by soft count = 90\n",
      "--- gene=NRNX1-seg-582, id=32 removed by soft count = 58\n",
      "--- number of probes: 59 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-583\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-584\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 39 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-585\n",
      "--- gene=NRNX1-seg-585, id=26 removed by soft count = 74\n",
      "--- number of probes: 51 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-586\n",
      "--- number of probes: 54 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-587\n",
      "--- gene=NRNX1-seg-587, id=17 removed by soft count = 423\n",
      "--- gene=NRNX1-seg-587, id=18 removed by soft count = 106\n",
      "--- gene=NRNX1-seg-587, id=19 removed by soft count = 125\n",
      "--- number of probes: 35 , kept by blast: 32 , if remove dups: 32\n",
      "-- number of probes kept for this region: 32\n",
      "-- checking probes in region: NRNX1-seg-588\n",
      "--- gene=NRNX1-seg-588, id=3 removed by soft count = 57\n",
      "--- gene=NRNX1-seg-588, id=26 removed by soft count = 69\n",
      "--- gene=NRNX1-seg-588, id=27 removed by soft count = 108\n",
      "--- gene=NRNX1-seg-588, id=30 removed by soft count = 120\n",
      "--- number of probes: 38 , kept by blast: 34 , if remove dups: 34\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: NRNX1-seg-589\n",
      "--- gene=NRNX1-seg-589, id=5 removed by soft count = 211\n",
      "--- gene=NRNX1-seg-589, id=6 removed by soft count = 230\n",
      "--- gene=NRNX1-seg-589, id=8 removed by soft count = 660\n",
      "--- gene=NRNX1-seg-589, id=10 removed by soft count = 86\n",
      "--- gene=NRNX1-seg-589, id=11 removed by soft count = 650\n",
      "--- gene=NRNX1-seg-589, id=12 removed by soft count = 149\n",
      "--- gene=NRNX1-seg-589, id=14 removed by soft count = 395\n",
      "--- gene=NRNX1-seg-589, id=15 removed by soft count = 1917\n",
      "--- gene=NRNX1-seg-589, id=17 removed by soft count = 185\n",
      "--- gene=NRNX1-seg-589, id=18 removed by soft count = 95\n",
      "--- gene=NRNX1-seg-589, id=19 removed by soft count = 74\n",
      "--- number of probes: 41 , kept by blast: 30 , if remove dups: 30\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: NRNX1-seg-590\n",
      "--- gene=NRNX1-seg-590, id=2 removed by soft count = 935\n",
      "--- gene=NRNX1-seg-590, id=3 removed by soft count = 78\n",
      "--- number of probes: 52 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-591\n",
      "--- gene=NRNX1-seg-591, id=35 removed by soft count = 16742\n",
      "--- number of probes: 49 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: NRNX1-seg-592\n",
      "--- gene=NRNX1-seg-592, id=45 removed by soft count = 173\n",
      "--- number of probes: 50 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-593\n",
      "--- number of probes: 55 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-594\n",
      "--- gene=NRNX1-seg-594, id=44 removed by soft count = 124\n",
      "--- number of probes: 45 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-595\n",
      "--- gene=NRNX1-seg-595, id=0 removed by soft count = 62\n",
      "--- gene=NRNX1-seg-595, id=1 removed by soft count = 126\n",
      "--- gene=NRNX1-seg-595, id=18 removed by soft count = 121\n",
      "--- number of probes: 43 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: NRNX1-seg-596\n",
      "--- number of probes: 45 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-597\n",
      "--- number of probes: 39 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-598\n",
      "--- number of probes: 46 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-599\n",
      "--- gene=NRNX1-seg-599, id=0 removed by soft count = 253\n",
      "--- number of probes: 59 , kept by blast: 58 , if remove dups: 58\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 58\n",
      "-- checking probes in region: NRNX1-seg-600\n",
      "--- gene=NRNX1-seg-600, id=27 removed by soft count = 52\n",
      "--- number of probes: 54 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-601\n",
      "--- gene=NRNX1-seg-601, id=20 removed by soft count = 64\n",
      "--- gene=NRNX1-seg-601, id=48 removed by soft count = 1380\n",
      "--- number of probes: 53 , kept by blast: 51 , if remove dups: 51\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 51\n",
      "-- checking probes in region: NRNX1-seg-602\n",
      "--- gene=NRNX1-seg-602, id=4 removed by soft count = 138\n",
      "--- number of probes: 42 , kept by blast: 41 , if remove dups: 41\n",
      "-- number of probes kept for this region: 41\n",
      "-- checking probes in region: NRNX1-seg-603\n",
      "--- gene=NRNX1-seg-603, id=6 removed by soft count = 1038\n",
      "--- number of probes: 60 , kept by blast: 59 , if remove dups: 59\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: NRNX1-seg-604\n",
      "--- gene=NRNX1-seg-604, id=6 removed by soft count = 1280\n",
      "--- gene=NRNX1-seg-604, id=8 removed by soft count = 2408\n",
      "--- gene=NRNX1-seg-604, id=9 removed by soft count = 59\n",
      "--- number of probes: 38 , kept by blast: 35 , if remove dups: 35\n",
      "-- number of probes kept for this region: 35\n",
      "-- checking probes in region: NRNX1-seg-605\n",
      "--- number of probes: 55 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-606\n",
      "--- number of probes: 67 , kept by blast: 67 , if remove dups: 67\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-607\n",
      "--- number of probes: 68 , kept by blast: 68 , if remove dups: 68\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-608\n",
      "--- number of probes: 56 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-609\n",
      "--- number of probes: 32 , kept by blast: 32 , if remove dups: 32\n",
      "-- number of probes kept for this region: 32\n",
      "-- checking probes in region: NRNX1-seg-610\n",
      "--- gene=NRNX1-seg-610, id=19 removed by soft count = 52\n",
      "--- gene=NRNX1-seg-610, id=20 removed by soft count = 247\n",
      "--- gene=NRNX1-seg-610, id=22 removed by soft count = 1242\n",
      "--- gene=NRNX1-seg-610, id=25 removed by soft count = 64\n",
      "--- gene=NRNX1-seg-610, id=26 removed by soft count = 168\n",
      "--- number of probes: 27 , kept by blast: 22 , if remove dups: 22\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: NRNX1-seg-611\n",
      "--- gene=NRNX1-seg-611, id=0 removed by soft count = 215\n",
      "--- gene=NRNX1-seg-611, id=12 removed by soft count = 141\n",
      "--- number of probes: 47 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-612\n",
      "--- gene=NRNX1-seg-612, id=24 removed by soft count = 67\n",
      "--- gene=NRNX1-seg-612, id=25 removed by soft count = 70\n",
      "--- number of probes: 30 , kept by blast: 28 , if remove dups: 28\n",
      "-- number of probes kept for this region: 28\n",
      "-- checking probes in region: NRNX1-seg-613\n",
      "--- gene=NRNX1-seg-613, id=0 removed by soft count = 62\n",
      "--- gene=NRNX1-seg-613, id=1 removed by soft count = 59\n",
      "--- gene=NRNX1-seg-613, id=4 removed by soft count = 118\n",
      "--- gene=NRNX1-seg-613, id=7 removed by soft count = 192\n",
      "--- gene=NRNX1-seg-613, id=9 removed by soft count = 59\n",
      "--- gene=NRNX1-seg-613, id=13 removed by soft count = 200\n",
      "--- number of probes: 28 , kept by blast: 22 , if remove dups: 22\n",
      "-- number of probes kept for this region: 22\n",
      "-- checking probes in region: NRNX1-seg-614\n",
      "--- gene=NRNX1-seg-614, id=47 removed by soft count = 77\n",
      "--- number of probes: 49 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: NRNX1-seg-615\n",
      "--- gene=NRNX1-seg-615, id=3 removed by soft count = 53\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 33 , kept by blast: 32 , if remove dups: 32\n",
      "-- number of probes kept for this region: 32\n",
      "-- checking probes in region: NRNX1-seg-616\n",
      "--- gene=NRNX1-seg-616, id=7 removed by soft count = 194\n",
      "--- gene=NRNX1-seg-616, id=8 removed by soft count = 1053\n",
      "--- gene=NRNX1-seg-616, id=10 removed by soft count = 90\n",
      "--- gene=NRNX1-seg-616, id=11 removed by soft count = 131\n",
      "--- gene=NRNX1-seg-616, id=12 removed by soft count = 95\n",
      "--- gene=NRNX1-seg-616, id=13 removed by soft count = 230\n",
      "--- gene=NRNX1-seg-616, id=14 removed by soft count = 81\n",
      "--- gene=NRNX1-seg-616, id=31 removed by soft count = 1573\n",
      "--- gene=NRNX1-seg-616, id=32 removed by soft count = 289\n",
      "--- number of probes: 42 , kept by blast: 33 , if remove dups: 33\n",
      "-- number of probes kept for this region: 33\n",
      "-- checking probes in region: NRNX1-seg-617\n",
      "--- number of probes: 48 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: NRNX1-seg-618\n",
      "--- gene=NRNX1-seg-618, id=17 removed by soft count = 130\n",
      "--- gene=NRNX1-seg-618, id=18 removed by soft count = 538\n",
      "--- gene=NRNX1-seg-618, id=19 removed by soft count = 382\n",
      "--- number of probes: 60 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-619\n",
      "--- number of probes: 73 , kept by blast: 73 , if remove dups: 73\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-620\n",
      "--- gene=NRNX1-seg-620, id=6 removed by soft count = 74\n",
      "--- number of probes: 75 , kept by blast: 74 , if remove dups: 74\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-621\n",
      "--- number of probes: 56 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-622\n",
      "--- number of probes: 65 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-623\n",
      "--- number of probes: 50 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-624\n",
      "--- gene=NRNX1-seg-624, id=23 removed by soft count = 150\n",
      "--- gene=NRNX1-seg-624, id=24 removed by soft count = 91\n",
      "--- gene=NRNX1-seg-624, id=25 removed by soft count = 338\n",
      "--- number of probes: 58 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-625\n",
      "--- gene=NRNX1-seg-625, id=0 removed by soft count = 64\n",
      "--- gene=NRNX1-seg-625, id=3 removed by soft count = 279\n",
      "--- gene=NRNX1-seg-625, id=4 removed by soft count = 182\n",
      "--- gene=NRNX1-seg-625, id=5 removed by soft count = 467\n",
      "--- gene=NRNX1-seg-625, id=7 removed by soft count = 248\n",
      "--- gene=NRNX1-seg-625, id=8 removed by soft count = 517\n",
      "--- gene=NRNX1-seg-625, id=9 removed by soft count = 310\n",
      "--- gene=NRNX1-seg-625, id=10 removed by soft count = 338\n",
      "--- gene=NRNX1-seg-625, id=12 removed by soft count = 89\n",
      "--- gene=NRNX1-seg-625, id=13 removed by soft count = 138\n",
      "--- gene=NRNX1-seg-625, id=15 removed by soft count = 316\n",
      "--- gene=NRNX1-seg-625, id=16 removed by soft count = 462\n",
      "--- gene=NRNX1-seg-625, id=17 removed by soft count = 508\n",
      "--- gene=NRNX1-seg-625, id=20 removed by soft count = 160\n",
      "--- gene=NRNX1-seg-625, id=22 removed by soft count = 166\n",
      "--- gene=NRNX1-seg-625, id=23 removed by soft count = 72\n",
      "--- gene=NRNX1-seg-625, id=24 removed by soft count = 431\n",
      "--- gene=NRNX1-seg-625, id=25 removed by soft count = 135\n",
      "--- gene=NRNX1-seg-625, id=26 removed by soft count = 386\n",
      "--- gene=NRNX1-seg-625, id=29 removed by soft count = 121\n",
      "--- number of probes: 39 , kept by blast: 19 , if remove dups: 18\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: NRNX1-seg-626\n",
      "--- number of probes: 48 , kept by blast: 48 , if remove dups: 48\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: NRNX1-seg-627\n",
      "--- gene=NRNX1-seg-627, id=37 removed by soft count = 106\n",
      "--- gene=NRNX1-seg-627, id=38 removed by soft count = 94\n",
      "--- gene=NRNX1-seg-627, id=40 removed by soft count = 701\n",
      "--- gene=NRNX1-seg-627, id=42 removed by soft count = 1315\n",
      "--- gene=NRNX1-seg-627, id=43 removed by soft count = 939\n",
      "--- number of probes: 44 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-628\n",
      "--- gene=NRNX1-seg-628, id=0 removed by soft count = 223\n",
      "--- gene=NRNX1-seg-628, id=1 removed by soft count = 536\n",
      "--- gene=NRNX1-seg-628, id=34 removed by soft count = 81\n",
      "--- number of probes: 37 , kept by blast: 34 , if remove dups: 34\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: NRNX1-seg-629\n",
      "--- number of probes: 40 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: NRNX1-seg-630\n",
      "--- gene=NRNX1-seg-630, id=6 removed by soft count = 448\n",
      "--- number of probes: 34 , kept by blast: 33 , if remove dups: 33\n",
      "-- number of probes kept for this region: 33\n",
      "-- checking probes in region: NRNX1-seg-631\n",
      "--- gene=NRNX1-seg-631, id=7 removed by soft count = 2443\n",
      "--- gene=NRNX1-seg-631, id=9 removed by soft count = 92\n",
      "--- gene=NRNX1-seg-631, id=10 removed by soft count = 1394\n",
      "--- gene=NRNX1-seg-631, id=11 removed by soft count = 155\n",
      "--- gene=NRNX1-seg-631, id=33 removed by soft count = 95\n",
      "--- number of probes: 55 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-632\n",
      "--- number of probes: 57 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-633\n",
      "--- gene=NRNX1-seg-633, id=44 removed by soft count = 78\n",
      "--- gene=NRNX1-seg-633, id=45 removed by soft count = 102\n",
      "--- number of probes: 51 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-634\n",
      "--- gene=NRNX1-seg-634, id=0 removed by soft count = 103\n",
      "--- gene=NRNX1-seg-634, id=1 removed by soft count = 353\n",
      "--- gene=NRNX1-seg-634, id=2 removed by soft count = 740\n",
      "--- gene=NRNX1-seg-634, id=3 removed by soft count = 337\n",
      "--- gene=NRNX1-seg-634, id=4 removed by soft count = 91\n",
      "--- gene=NRNX1-seg-634, id=7 removed by soft count = 70\n",
      "--- gene=NRNX1-seg-634, id=8 removed by soft count = 283\n",
      "--- gene=NRNX1-seg-634, id=10 removed by soft count = 138\n",
      "--- gene=NRNX1-seg-634, id=11 removed by soft count = 189\n",
      "--- gene=NRNX1-seg-634, id=16 removed by soft count = 67\n",
      "--- gene=NRNX1-seg-634, id=17 removed by soft count = 101\n",
      "--- gene=NRNX1-seg-634, id=18 removed by soft count = 136\n",
      "--- number of probes: 64 , kept by blast: 52 , if remove dups: 52\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: NRNX1-seg-635\n",
      "--- gene=NRNX1-seg-635, id=44 removed by soft count = 213\n",
      "--- number of probes: 45 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-636\n",
      "--- gene=NRNX1-seg-636, id=11 removed by soft count = 71\n",
      "--- gene=NRNX1-seg-636, id=12 removed by soft count = 203\n",
      "--- gene=NRNX1-seg-636, id=14 removed by soft count = 162\n",
      "--- gene=NRNX1-seg-636, id=17 removed by soft count = 304\n",
      "--- gene=NRNX1-seg-636, id=18 removed by soft count = 184\n",
      "--- gene=NRNX1-seg-636, id=19 removed by soft count = 201\n",
      "--- gene=NRNX1-seg-636, id=20 removed by soft count = 303\n",
      "--- gene=NRNX1-seg-636, id=21 removed by soft count = 197\n",
      "--- gene=NRNX1-seg-636, id=25 removed by soft count = 229\n",
      "--- gene=NRNX1-seg-636, id=26 removed by soft count = 83\n",
      "--- gene=NRNX1-seg-636, id=27 removed by soft count = 124\n",
      "--- gene=NRNX1-seg-636, id=28 removed by soft count = 127\n",
      "--- gene=NRNX1-seg-636, id=29 removed by soft count = 906\n",
      "--- gene=NRNX1-seg-636, id=30 removed by soft count = 691\n",
      "--- gene=NRNX1-seg-636, id=31 removed by soft count = 172\n",
      "--- number of probes: 32 , kept by blast: 17 , if remove dups: 17\n",
      "-- number of probes kept for this region: 17\n",
      "-- checking probes in region: NRNX1-seg-637\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=NRNX1-seg-637, id=0 removed by soft count = 89\n",
      "--- gene=NRNX1-seg-637, id=3 removed by soft count = 402\n",
      "--- gene=NRNX1-seg-637, id=4 removed by soft count = 52\n",
      "--- gene=NRNX1-seg-637, id=5 removed by soft count = 1215\n",
      "--- gene=NRNX1-seg-637, id=6 removed by soft count = 1773\n",
      "--- gene=NRNX1-seg-637, id=7 removed by soft count = 109\n",
      "--- gene=NRNX1-seg-637, id=8 removed by soft count = 364\n",
      "--- gene=NRNX1-seg-637, id=9 removed by soft count = 1673\n",
      "--- gene=NRNX1-seg-637, id=10 removed by soft count = 261\n",
      "--- gene=NRNX1-seg-637, id=11 removed by soft count = 1587\n",
      "--- gene=NRNX1-seg-637, id=12 removed by soft count = 167\n",
      "--- number of probes: 22 , kept by blast: 11 , if remove dups: 11\n",
      "-- number of probes kept for this region: 11\n",
      "-- checking probes in region: NRNX1-seg-639\n",
      "--- gene=NRNX1-seg-639, id=1 removed by soft count = 11025\n",
      "--- gene=NRNX1-seg-639, id=2 removed by soft count = 285\n",
      "--- gene=NRNX1-seg-639, id=4 removed by soft count = 117\n",
      "--- gene=NRNX1-seg-639, id=6 removed by soft count = 354\n",
      "--- gene=NRNX1-seg-639, id=7 removed by soft count = 69\n",
      "--- gene=NRNX1-seg-639, id=8 removed by soft count = 10202\n",
      "--- gene=NRNX1-seg-639, id=10 removed by soft count = 84\n",
      "--- gene=NRNX1-seg-639, id=11 removed by soft count = 55\n",
      "--- gene=NRNX1-seg-639, id=19 removed by soft count = 55\n",
      "--- gene=NRNX1-seg-639, id=21 removed by soft count = 108\n",
      "--- gene=NRNX1-seg-639, id=24 removed by soft count = 286\n",
      "--- gene=NRNX1-seg-639, id=27 removed by soft count = 178\n",
      "--- number of probes: 32 , kept by blast: 20 , if remove dups: 20\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: NRNX1-seg-640\n",
      "--- gene=NRNX1-seg-640, id=22 removed by soft count = 246\n",
      "--- number of probes: 54 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-641\n",
      "--- gene=NRNX1-seg-641, id=35 removed by soft count = 412\n",
      "--- gene=NRNX1-seg-641, id=36 removed by soft count = 628\n",
      "--- number of probes: 46 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-642\n",
      "--- number of probes: 45 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-643\n",
      "--- gene=NRNX1-seg-643, id=7 removed by soft count = 67\n",
      "--- number of probes: 57 , kept by blast: 56 , if remove dups: 56\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 56\n",
      "-- checking probes in region: NRNX1-seg-644\n",
      "--- gene=NRNX1-seg-644, id=10 removed by soft count = 53\n",
      "--- gene=NRNX1-seg-644, id=11 removed by soft count = 116\n",
      "--- gene=NRNX1-seg-644, id=12 removed by soft count = 89\n",
      "--- gene=NRNX1-seg-644, id=14 removed by soft count = 201\n",
      "--- number of probes: 58 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-645\n",
      "--- gene=NRNX1-seg-645, id=43 removed by soft count = 149\n",
      "--- gene=NRNX1-seg-645, id=44 removed by soft count = 145\n",
      "--- gene=NRNX1-seg-645, id=45 removed by soft count = 559\n",
      "--- number of probes: 49 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-646\n",
      "--- number of probes: 69 , kept by blast: 69 , if remove dups: 69\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-647\n",
      "--- gene=NRNX1-seg-647, id=0 removed by soft count = 11368\n",
      "--- gene=NRNX1-seg-647, id=27 removed by soft count = 566\n",
      "--- gene=NRNX1-seg-647, id=28 removed by soft count = 2036\n",
      "--- gene=NRNX1-seg-647, id=29 removed by soft count = 180\n",
      "--- gene=NRNX1-seg-647, id=30 removed by soft count = 648\n",
      "--- gene=NRNX1-seg-647, id=31 removed by soft count = 130\n",
      "--- number of probes: 33 , kept by blast: 27 , if remove dups: 27\n",
      "-- number of probes kept for this region: 27\n",
      "-- checking probes in region: NRNX1-seg-648\n",
      "--- gene=NRNX1-seg-648, id=0 removed by soft count = 110\n",
      "--- gene=NRNX1-seg-648, id=1 removed by soft count = 201\n",
      "--- number of probes: 41 , kept by blast: 39 , if remove dups: 39\n",
      "-- number of probes kept for this region: 39\n",
      "-- checking probes in region: NRNX1-seg-649\n",
      "--- number of probes: 45 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-650\n",
      "--- gene=NRNX1-seg-650, id=19 removed by soft count = 91\n",
      "--- number of probes: 44 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-651\n",
      "--- gene=NRNX1-seg-651, id=11 removed by soft count = 115\n",
      "--- gene=NRNX1-seg-651, id=29 removed by soft count = 61\n",
      "--- gene=NRNX1-seg-651, id=34 removed by soft count = 185\n",
      "--- gene=NRNX1-seg-651, id=41 removed by soft count = 85\n",
      "--- gene=NRNX1-seg-651, id=44 removed by soft count = 86\n",
      "--- number of probes: 47 , kept by blast: 42 , if remove dups: 42\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: NRNX1-seg-652\n",
      "--- gene=NRNX1-seg-652, id=2 removed by soft count = 470\n",
      "--- gene=NRNX1-seg-652, id=3 removed by soft count = 951\n",
      "--- gene=NRNX1-seg-652, id=6 removed by soft count = 73\n",
      "--- gene=NRNX1-seg-652, id=8 removed by soft count = 54\n",
      "--- number of probes: 35 , kept by blast: 31 , if remove dups: 31\n",
      "-- number of probes kept for this region: 31\n",
      "-- checking probes in region: NRNX1-seg-653\n",
      "--- gene=NRNX1-seg-653, id=22 removed by soft count = 94\n",
      "--- gene=NRNX1-seg-653, id=26 removed by soft count = 734\n",
      "--- number of probes: 49 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-654\n",
      "--- gene=NRNX1-seg-654, id=5 removed by soft count = 101\n",
      "--- gene=NRNX1-seg-654, id=9 removed by soft count = 226\n",
      "--- gene=NRNX1-seg-654, id=10 removed by soft count = 131\n",
      "--- gene=NRNX1-seg-654, id=13 removed by soft count = 129\n",
      "--- number of probes: 57 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-655\n",
      "--- number of probes: 55 , kept by blast: 55 , if remove dups: 55\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: NRNX1-seg-656\n",
      "--- gene=NRNX1-seg-656, id=0 removed by soft count = 137\n",
      "--- gene=NRNX1-seg-656, id=40 removed by soft count = 5331\n",
      "--- gene=NRNX1-seg-656, id=44 removed by soft count = 3235\n",
      "--- gene=NRNX1-seg-656, id=45 removed by soft count = 11773\n",
      "--- number of probes: 50 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: NRNX1-seg-657\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-658\n",
      "--- number of probes: 60 , kept by blast: 60 , if remove dups: 60\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-659\n",
      "--- gene=NRNX1-seg-659, id=1 removed by soft count = 1157\n",
      "--- number of probes: 50 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-660\n",
      "--- number of probes: 50 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-661\n",
      "--- gene=NRNX1-seg-661, id=26 removed by soft count = 93\n",
      "--- number of probes: 51 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-662\n",
      "--- gene=NRNX1-seg-662, id=34 removed by soft count = 276\n",
      "--- number of probes: 46 , kept by blast: 45 , if remove dups: 44\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-665\n",
      "--- gene=NRNX1-seg-665, id=21 removed by soft count = 77\n",
      "--- gene=NRNX1-seg-665, id=31 removed by soft count = 1128\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=NRNX1-seg-665, id=52 removed by soft count = 186\n",
      "--- number of probes: 53 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-666\n",
      "--- gene=NRNX1-seg-666, id=6 removed by soft count = 99\n",
      "--- gene=NRNX1-seg-666, id=7 removed by soft count = 72\n",
      "--- gene=NRNX1-seg-666, id=27 removed by soft count = 57\n",
      "--- number of probes: 68 , kept by blast: 65 , if remove dups: 65\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the front\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: NRNX1-seg-667\n",
      "--- number of probes: 50 , kept by blast: 50 , if remove dups: 50\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 50\n",
      "-- checking probes in region: NRNX1-seg-668\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-669\n",
      "--- number of probes: 45 , kept by blast: 45 , if remove dups: 45\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: NRNX1-seg-670\n",
      "--- gene=NRNX1-seg-670, id=23 removed by soft count = 276\n",
      "--- number of probes: 33 , kept by blast: 32 , if remove dups: 32\n",
      "-- number of probes kept for this region: 32\n",
      "-- checking probes in region: NRNX1-seg-671\n",
      "--- gene=NRNX1-seg-671, id=50 removed by soft count = 252\n",
      "--- number of probes: 54 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-672\n",
      "--- gene=NRNX1-seg-672, id=9 removed by soft count = 8430\n",
      "--- number of probes: 38 , kept by blast: 37 , if remove dups: 37\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: NRNX1-seg-674\n",
      "--- number of probes: 47 , kept by blast: 47 , if remove dups: 47\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: NRNX1-seg-675\n",
      "--- gene=NRNX1-seg-675, id=12 removed by soft count = 108\n",
      "--- number of probes: 43 , kept by blast: 42 , if remove dups: 42\n",
      "-- number of probes kept for this region: 42\n",
      "-- checking probes in region: NRNX1-seg-676\n",
      "--- gene=NRNX1-seg-676, id=7 removed by soft count = 5283\n",
      "--- gene=NRNX1-seg-676, id=9 removed by soft count = 61\n",
      "--- gene=NRNX1-seg-676, id=10 removed by soft count = 1624\n",
      "--- gene=NRNX1-seg-676, id=13 removed by soft count = 737\n",
      "--- gene=NRNX1-seg-676, id=14 removed by soft count = 1019\n",
      "--- gene=NRNX1-seg-676, id=16 removed by soft count = 979\n",
      "--- gene=NRNX1-seg-676, id=17 removed by soft count = 1055\n",
      "--- number of probes: 40 , kept by blast: 33 , if remove dups: 33\n",
      "-- number of probes kept for this region: 33\n",
      "-- checking probes in region: NRNX1-seg-678\n",
      "--- gene=NRNX1-seg-678, id=16 removed by soft count = 176\n",
      "--- gene=NRNX1-seg-678, id=17 removed by soft count = 344\n",
      "--- gene=NRNX1-seg-678, id=21 removed by soft count = 86\n",
      "--- gene=NRNX1-seg-678, id=25 removed by soft count = 145\n",
      "--- gene=NRNX1-seg-678, id=26 removed by soft count = 200\n",
      "--- gene=NRNX1-seg-678, id=28 removed by soft count = 252\n",
      "--- gene=NRNX1-seg-678, id=35 removed by soft count = 362\n",
      "--- gene=NRNX1-seg-678, id=37 removed by soft count = 1351\n",
      "--- gene=NRNX1-seg-678, id=38 removed by soft count = 503\n",
      "--- gene=NRNX1-seg-678, id=41 removed by soft count = 222\n",
      "--- gene=NRNX1-seg-678, id=43 removed by soft count = 93\n",
      "--- number of probes: 44 , kept by blast: 33 , if remove dups: 33\n",
      "-- number of probes kept for this region: 33\n",
      "-- checking probes in region: NRNX1-seg-679\n",
      "--- gene=NRNX1-seg-679, id=1 removed by soft count = 53\n",
      "--- gene=NRNX1-seg-679, id=3 removed by soft count = 85\n",
      "--- gene=NRNX1-seg-679, id=5 removed by soft count = 66\n",
      "--- gene=NRNX1-seg-679, id=6 removed by soft count = 108\n",
      "--- gene=NRNX1-seg-679, id=8 removed by soft count = 146\n",
      "--- gene=NRNX1-seg-679, id=9 removed by soft count = 64\n",
      "--- gene=NRNX1-seg-679, id=13 removed by soft count = 147\n",
      "--- gene=NRNX1-seg-679, id=14 removed by soft count = 73\n",
      "--- gene=NRNX1-seg-679, id=16 removed by soft count = 624\n",
      "--- gene=NRNX1-seg-679, id=17 removed by soft count = 499\n",
      "--- gene=NRNX1-seg-679, id=18 removed by soft count = 86\n",
      "--- gene=NRNX1-seg-679, id=19 removed by soft count = 2362\n",
      "--- gene=NRNX1-seg-679, id=20 removed by soft count = 1616\n",
      "--- gene=NRNX1-seg-679, id=21 removed by soft count = 9532\n",
      "--- number of probes: 22 , kept by blast: 8 , if remove dups: 8\n",
      "-- number of probes kept for this region: 8\n",
      "-- checking probes in region: NRNX1-seg-681\n",
      "--- gene=NRNX1-seg-681, id=21 removed by soft count = 781\n",
      "--- gene=NRNX1-seg-681, id=22 removed by soft count = 2216\n",
      "--- gene=NRNX1-seg-681, id=23 removed by soft count = 3345\n",
      "--- gene=NRNX1-seg-681, id=24 removed by soft count = 68\n",
      "--- gene=NRNX1-seg-681, id=26 removed by soft count = 936\n",
      "--- gene=NRNX1-seg-681, id=27 removed by soft count = 64\n",
      "--- gene=NRNX1-seg-681, id=28 removed by soft count = 79\n",
      "--- gene=NRNX1-seg-681, id=30 removed by soft count = 111\n",
      "--- number of probes: 31 , kept by blast: 23 , if remove dups: 23\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: NRNX1-seg-682\n",
      "--- gene=NRNX1-seg-682, id=2 removed by soft count = 87\n",
      "--- gene=NRNX1-seg-682, id=9 removed by soft count = 74\n",
      "--- gene=NRNX1-seg-682, id=15 removed by soft count = 103\n",
      "--- gene=NRNX1-seg-682, id=19 removed by soft count = 187\n",
      "--- number of probes: 39 , kept by blast: 35 , if remove dups: 35\n",
      "-- number of probes kept for this region: 35\n",
      "-- checking probes in region: NRNX1-seg-683\n",
      "--- number of probes: 53 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-684\n",
      "--- gene=NRNX1-seg-684, id=14 removed by soft count = 228\n",
      "--- gene=NRNX1-seg-684, id=16 removed by soft count = 166\n",
      "--- gene=NRNX1-seg-684, id=19 removed by soft count = 117\n",
      "--- gene=NRNX1-seg-684, id=20 removed by soft count = 178\n",
      "--- number of probes: 23 , kept by blast: 19 , if remove dups: 19\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: NRNX1-seg-685\n",
      "--- gene=NRNX1-seg-685, id=8 removed by soft count = 79\n",
      "--- gene=NRNX1-seg-685, id=27 removed by soft count = 182\n",
      "--- gene=NRNX1-seg-685, id=28 removed by soft count = 818\n",
      "--- number of probes: 37 , kept by blast: 34 , if remove dups: 34\n",
      "-- number of probes kept for this region: 34\n",
      "-- checking probes in region: NRNX1-seg-686\n",
      "--- gene=NRNX1-seg-686, id=0 removed by soft count = 53\n",
      "--- gene=NRNX1-seg-686, id=2 removed by soft count = 84\n",
      "--- gene=NRNX1-seg-686, id=3 removed by soft count = 2113\n",
      "--- gene=NRNX1-seg-686, id=4 removed by soft count = 490\n",
      "--- gene=NRNX1-seg-686, id=6 removed by soft count = 3584\n",
      "--- gene=NRNX1-seg-686, id=8 removed by soft count = 1484\n",
      "--- gene=NRNX1-seg-686, id=13 removed by soft count = 110\n",
      "--- gene=NRNX1-seg-686, id=14 removed by soft count = 58\n",
      "--- gene=NRNX1-seg-686, id=16 removed by soft count = 526\n",
      "--- number of probes: 28 , kept by blast: 19 , if remove dups: 19\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: NRNX1-seg-687\n",
      "--- gene=NRNX1-seg-687, id=5 removed by soft count = 55\n",
      "--- gene=NRNX1-seg-687, id=12 removed by soft count = 2484\n",
      "--- gene=NRNX1-seg-687, id=13 removed by soft count = 125\n",
      "--- gene=NRNX1-seg-687, id=15 removed by soft count = 119\n",
      "--- gene=NRNX1-seg-687, id=22 removed by soft count = 51\n",
      "--- gene=NRNX1-seg-687, id=23 removed by soft count = 69\n",
      "--- number of probes: 27 , kept by blast: 21 , if remove dups: 21\n",
      "-- number of probes kept for this region: 21\n",
      "-- checking probes in region: NRNX1-seg-688\n",
      "--- gene=NRNX1-seg-688, id=4 removed by soft count = 185\n",
      "--- gene=NRNX1-seg-688, id=5 removed by soft count = 66\n",
      "--- gene=NRNX1-seg-688, id=6 removed by soft count = 577\n",
      "--- gene=NRNX1-seg-688, id=7 removed by soft count = 716\n",
      "--- gene=NRNX1-seg-688, id=8 removed by soft count = 530\n",
      "--- gene=NRNX1-seg-688, id=9 removed by soft count = 1878\n",
      "--- gene=NRNX1-seg-688, id=16 removed by soft count = 86\n",
      "--- number of probes: 28 , kept by blast: 21 , if remove dups: 21\n",
      "-- number of probes kept for this region: 21\n",
      "-- checking probes in region: NRNX1-seg-689\n",
      "--- gene=NRNX1-seg-689, id=16 removed by soft count = 5273\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=NRNX1-seg-689, id=17 removed by soft count = 930\n",
      "--- number of probes: 45 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-690\n",
      "--- number of probes: 43 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: NRNX1-seg-691\n",
      "--- gene=NRNX1-seg-691, id=0 removed by soft count = 368\n",
      "--- gene=NRNX1-seg-691, id=1 removed by soft count = 1447\n",
      "--- gene=NRNX1-seg-691, id=2 removed by soft count = 295\n",
      "--- gene=NRNX1-seg-691, id=3 removed by soft count = 151\n",
      "--- gene=NRNX1-seg-691, id=4 removed by soft count = 1071\n",
      "--- gene=NRNX1-seg-691, id=9 removed by soft count = 448\n",
      "--- number of probes: 36 , kept by blast: 30 , if remove dups: 30\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: NRNX1-seg-692\n",
      "--- gene=NRNX1-seg-692, id=0 removed by soft count = 271\n",
      "--- gene=NRNX1-seg-692, id=1 removed by soft count = 138\n",
      "--- gene=NRNX1-seg-692, id=39 removed by soft count = 72\n",
      "--- number of probes: 47 , kept by blast: 44 , if remove dups: 44\n",
      "-- number of probes kept for this region: 44\n",
      "-- checking probes in region: NRNX1-seg-693\n",
      "--- number of probes: 54 , kept by blast: 54 , if remove dups: 54\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 54\n",
      "-- checking probes in region: NRNX1-seg-694\n",
      "--- number of probes: 49 , kept by blast: 49 , if remove dups: 49\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: NRNX1-seg-695\n",
      "--- gene=NRNX1-seg-695, id=45 removed by soft count = 108\n",
      "--- gene=NRNX1-seg-695, id=48 removed by soft count = 147\n",
      "--- gene=NRNX1-seg-695, id=51 removed by soft count = 1042\n",
      "--- gene=NRNX1-seg-695, id=52 removed by soft count = 744\n",
      "--- number of probes: 57 , kept by blast: 53 , if remove dups: 53\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 53\n",
      "-- checking probes in region: NRNX1-seg-696\n",
      "--- gene=NRNX1-seg-696, id=14 removed by soft count = 61\n",
      "--- gene=NRNX1-seg-696, id=16 removed by soft count = 168\n",
      "--- gene=NRNX1-seg-696, id=23 removed by soft count = 128\n",
      "--- number of probes: 40 , kept by blast: 37 , if remove dups: 37\n",
      "-- number of probes kept for this region: 37\n",
      "-- checking probes in region: NRNX1-seg-697\n",
      "--- gene=NRNX1-seg-697, id=7 removed by soft count = 121\n",
      "--- gene=NRNX1-seg-697, id=10 removed by soft count = 178\n",
      "--- number of probes: 42 , kept by blast: 40 , if remove dups: 40\n",
      "-- number of probes kept for this region: 40\n",
      "-- checking probes in region: NRNX1-seg-698\n",
      "--- gene=NRNX1-seg-698, id=3 removed by soft count = 109\n",
      "--- gene=NRNX1-seg-698, id=4 removed by soft count = 213\n",
      "--- gene=NRNX1-seg-698, id=8 removed by soft count = 55\n",
      "--- gene=NRNX1-seg-698, id=9 removed by soft count = 4088\n",
      "--- number of probes: 61 , kept by blast: 57 , if remove dups: 57\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 57\n",
      "-- checking probes in region: NRNX1-seg-699\n",
      "--- number of probes: 46 , kept by blast: 46 , if remove dups: 46\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 46\n",
      "- summarize\n",
      "-- region: NRNX1-seg-1 , number of probes: 34\n",
      "-- region: NRNX1-seg-2 , number of probes: 38\n",
      "-- region: NRNX1-seg-3 , number of probes: 36\n",
      "-- region: NRNX1-seg-4 , number of probes: 50\n",
      "-- region: NRNX1-seg-5 , number of probes: 60\n",
      "-- region: NRNX1-seg-6 , number of probes: 51\n",
      "-- region: NRNX1-seg-7 , number of probes: 33\n",
      "-- region: NRNX1-seg-9 , number of probes: 51\n",
      "-- region: NRNX1-seg-10 , number of probes: 60\n",
      "-- region: NRNX1-seg-13 , number of probes: 45\n",
      "-- region: NRNX1-seg-14 , number of probes: 60\n",
      "-- region: NRNX1-seg-15 , number of probes: 60\n",
      "-- region: NRNX1-seg-16 , number of probes: 60\n",
      "-- region: NRNX1-seg-17 , number of probes: 52\n",
      "-- region: NRNX1-seg-18 , number of probes: 34\n",
      "-- region: NRNX1-seg-19 , number of probes: 60\n",
      "-- region: NRNX1-seg-20 , number of probes: 45\n",
      "-- region: NRNX1-seg-21 , number of probes: 38\n",
      "-- region: NRNX1-seg-22 , number of probes: 60\n",
      "-- region: NRNX1-seg-23 , number of probes: 60\n",
      "-- region: NRNX1-seg-24 , number of probes: 55\n",
      "-- region: NRNX1-seg-25 , number of probes: 43\n",
      "-- region: NRNX1-seg-26 , number of probes: 55\n",
      "-- region: NRNX1-seg-27 , number of probes: 50\n",
      "-- region: NRNX1-seg-28 , number of probes: 47\n",
      "-- region: NRNX1-seg-30 , number of probes: 44\n",
      "-- region: NRNX1-seg-31 , number of probes: 38\n",
      "-- region: NRNX1-seg-32 , number of probes: 41\n",
      "-- region: NRNX1-seg-33 , number of probes: 29\n",
      "-- region: NRNX1-seg-34 , number of probes: 39\n",
      "-- region: NRNX1-seg-35 , number of probes: 41\n",
      "-- region: NRNX1-seg-36 , number of probes: 44\n",
      "-- region: NRNX1-seg-37 , number of probes: 41\n",
      "-- region: NRNX1-seg-38 , number of probes: 41\n",
      "-- region: NRNX1-seg-39 , number of probes: 45\n",
      "-- region: NRNX1-seg-40 , number of probes: 60\n",
      "-- region: NRNX1-seg-41 , number of probes: 54\n",
      "-- region: NRNX1-seg-42 , number of probes: 60\n",
      "-- region: NRNX1-seg-43 , number of probes: 60\n",
      "-- region: NRNX1-seg-44 , number of probes: 50\n",
      "-- region: NRNX1-seg-45 , number of probes: 49\n",
      "-- region: NRNX1-seg-46 , number of probes: 42\n",
      "-- region: NRNX1-seg-47 , number of probes: 41\n",
      "-- region: NRNX1-seg-48 , number of probes: 35\n",
      "-- region: NRNX1-seg-49 , number of probes: 57\n",
      "-- region: NRNX1-seg-50 , number of probes: 46\n",
      "-- region: NRNX1-seg-51 , number of probes: 47\n",
      "-- region: NRNX1-seg-52 , number of probes: 58\n",
      "-- region: NRNX1-seg-53 , number of probes: 50\n",
      "-- region: NRNX1-seg-54 , number of probes: 60\n",
      "-- region: NRNX1-seg-55 , number of probes: 57\n",
      "-- region: NRNX1-seg-56 , number of probes: 42\n",
      "-- region: NRNX1-seg-57 , number of probes: 54\n",
      "-- region: NRNX1-seg-58 , number of probes: 42\n",
      "-- region: NRNX1-seg-59 , number of probes: 47\n",
      "-- region: NRNX1-seg-60 , number of probes: 23\n",
      "-- region: NRNX1-seg-62 , number of probes: 43\n",
      "-- region: NRNX1-seg-63 , number of probes: 56\n",
      "-- region: NRNX1-seg-64 , number of probes: 43\n",
      "-- region: NRNX1-seg-65 , number of probes: 23\n",
      "-- region: NRNX1-seg-66 , number of probes: 39\n",
      "-- region: NRNX1-seg-67 , number of probes: 50\n",
      "-- region: NRNX1-seg-68 , number of probes: 21\n",
      "-- region: NRNX1-seg-69 , number of probes: 45\n",
      "-- region: NRNX1-seg-70 , number of probes: 41\n",
      "-- region: NRNX1-seg-71 , number of probes: 52\n",
      "-- region: NRNX1-seg-72 , number of probes: 44\n",
      "-- region: NRNX1-seg-73 , number of probes: 19\n",
      "-- region: NRNX1-seg-74 , number of probes: 35\n",
      "-- region: NRNX1-seg-75 , number of probes: 26\n",
      "-- region: NRNX1-seg-76 , number of probes: 45\n",
      "-- region: NRNX1-seg-77 , number of probes: 25\n",
      "-- region: NRNX1-seg-79 , number of probes: 27\n",
      "-- region: NRNX1-seg-80 , number of probes: 50\n",
      "-- region: NRNX1-seg-81 , number of probes: 36\n",
      "-- region: NRNX1-seg-82 , number of probes: 47\n",
      "-- region: NRNX1-seg-83 , number of probes: 60\n",
      "-- region: NRNX1-seg-84 , number of probes: 60\n",
      "-- region: NRNX1-seg-85 , number of probes: 53\n",
      "-- region: NRNX1-seg-86 , number of probes: 45\n",
      "-- region: NRNX1-seg-87 , number of probes: 44\n",
      "-- region: NRNX1-seg-88 , number of probes: 46\n",
      "-- region: NRNX1-seg-89 , number of probes: 60\n",
      "-- region: NRNX1-seg-90 , number of probes: 52\n",
      "-- region: NRNX1-seg-91 , number of probes: 50\n",
      "-- region: NRNX1-seg-92 , number of probes: 37\n",
      "-- region: NRNX1-seg-93 , number of probes: 60\n",
      "-- region: NRNX1-seg-94 , number of probes: 43\n",
      "-- region: NRNX1-seg-95 , number of probes: 29\n",
      "-- region: NRNX1-seg-96 , number of probes: 40\n",
      "-- region: NRNX1-seg-97 , number of probes: 40\n",
      "-- region: NRNX1-seg-98 , number of probes: 36\n",
      "-- region: NRNX1-seg-99 , number of probes: 30\n",
      "-- region: NRNX1-seg-100 , number of probes: 39\n",
      "-- region: NRNX1-seg-101 , number of probes: 39\n",
      "-- region: NRNX1-seg-102 , number of probes: 49\n",
      "-- region: NRNX1-seg-103 , number of probes: 51\n",
      "-- region: NRNX1-seg-104 , number of probes: 58\n",
      "-- region: NRNX1-seg-105 , number of probes: 23\n",
      "-- region: NRNX1-seg-106 , number of probes: 60\n",
      "-- region: NRNX1-seg-107 , number of probes: 46\n",
      "-- region: NRNX1-seg-108 , number of probes: 32\n",
      "-- region: NRNX1-seg-109 , number of probes: 39\n",
      "-- region: NRNX1-seg-110 , number of probes: 60\n",
      "-- region: NRNX1-seg-111 , number of probes: 47\n",
      "-- region: NRNX1-seg-112 , number of probes: 22\n",
      "-- region: NRNX1-seg-113 , number of probes: 60\n",
      "-- region: NRNX1-seg-114 , number of probes: 52\n",
      "-- region: NRNX1-seg-115 , number of probes: 48\n",
      "-- region: NRNX1-seg-117 , number of probes: 56\n",
      "-- region: NRNX1-seg-118 , number of probes: 58\n",
      "-- region: NRNX1-seg-119 , number of probes: 52\n",
      "-- region: NRNX1-seg-120 , number of probes: 56\n",
      "-- region: NRNX1-seg-121 , number of probes: 60\n",
      "-- region: NRNX1-seg-122 , number of probes: 47\n",
      "-- region: NRNX1-seg-123 , number of probes: 35\n",
      "-- region: NRNX1-seg-124 , number of probes: 58\n",
      "-- region: NRNX1-seg-125 , number of probes: 36\n",
      "-- region: NRNX1-seg-126 , number of probes: 19\n",
      "-- region: NRNX1-seg-127 , number of probes: 26\n",
      "-- region: NRNX1-seg-128 , number of probes: 25\n",
      "-- region: NRNX1-seg-129 , number of probes: 30\n",
      "-- region: NRNX1-seg-130 , number of probes: 46\n",
      "-- region: NRNX1-seg-131 , number of probes: 39\n",
      "-- region: NRNX1-seg-133 , number of probes: 23\n",
      "-- region: NRNX1-seg-134 , number of probes: 26\n",
      "-- region: NRNX1-seg-135 , number of probes: 40\n",
      "-- region: NRNX1-seg-136 , number of probes: 51\n",
      "-- region: NRNX1-seg-137 , number of probes: 50\n",
      "-- region: NRNX1-seg-138 , number of probes: 49\n",
      "-- region: NRNX1-seg-139 , number of probes: 60\n",
      "-- region: NRNX1-seg-140 , number of probes: 58\n",
      "-- region: NRNX1-seg-141 , number of probes: 37\n",
      "-- region: NRNX1-seg-142 , number of probes: 27\n",
      "-- region: NRNX1-seg-143 , number of probes: 60\n",
      "-- region: NRNX1-seg-144 , number of probes: 60\n",
      "-- region: NRNX1-seg-145 , number of probes: 60\n",
      "-- region: NRNX1-seg-146 , number of probes: 60\n",
      "-- region: NRNX1-seg-147 , number of probes: 60\n",
      "-- region: NRNX1-seg-148 , number of probes: 57\n",
      "-- region: NRNX1-seg-150 , number of probes: 59\n",
      "-- region: NRNX1-seg-151 , number of probes: 60\n",
      "-- region: NRNX1-seg-152 , number of probes: 60\n",
      "-- region: NRNX1-seg-154 , number of probes: 32\n",
      "-- region: NRNX1-seg-155 , number of probes: 58\n",
      "-- region: NRNX1-seg-156 , number of probes: 60\n",
      "-- region: NRNX1-seg-157 , number of probes: 42\n",
      "-- region: NRNX1-seg-158 , number of probes: 60\n",
      "-- region: NRNX1-seg-159 , number of probes: 57\n",
      "-- region: NRNX1-seg-160 , number of probes: 60\n",
      "-- region: NRNX1-seg-161 , number of probes: 60\n",
      "-- region: NRNX1-seg-162 , number of probes: 60\n",
      "-- region: NRNX1-seg-163 , number of probes: 29\n",
      "-- region: NRNX1-seg-164 , number of probes: 56\n",
      "-- region: NRNX1-seg-165 , number of probes: 60\n",
      "-- region: NRNX1-seg-166 , number of probes: 27\n",
      "-- region: NRNX1-seg-169 , number of probes: 50\n",
      "-- region: NRNX1-seg-170 , number of probes: 60\n",
      "-- region: NRNX1-seg-171 , number of probes: 60\n",
      "-- region: NRNX1-seg-172 , number of probes: 60\n",
      "-- region: NRNX1-seg-173 , number of probes: 60\n",
      "-- region: NRNX1-seg-174 , number of probes: 56\n",
      "-- region: NRNX1-seg-175 , number of probes: 60\n",
      "-- region: NRNX1-seg-176 , number of probes: 60\n",
      "-- region: NRNX1-seg-177 , number of probes: 60\n",
      "-- region: NRNX1-seg-178 , number of probes: 60\n",
      "-- region: NRNX1-seg-179 , number of probes: 60\n",
      "-- region: NRNX1-seg-180 , number of probes: 60\n",
      "-- region: NRNX1-seg-181 , number of probes: 60\n",
      "-- region: NRNX1-seg-182 , number of probes: 60\n",
      "-- region: NRNX1-seg-183 , number of probes: 54\n",
      "-- region: NRNX1-seg-184 , number of probes: 48\n",
      "-- region: NRNX1-seg-185 , number of probes: 36\n",
      "-- region: NRNX1-seg-186 , number of probes: 54\n",
      "-- region: NRNX1-seg-187 , number of probes: 60\n",
      "-- region: NRNX1-seg-188 , number of probes: 58\n",
      "-- region: NRNX1-seg-189 , number of probes: 60\n",
      "-- region: NRNX1-seg-190 , number of probes: 60\n",
      "-- region: NRNX1-seg-191 , number of probes: 49\n",
      "-- region: NRNX1-seg-192 , number of probes: 60\n",
      "-- region: NRNX1-seg-193 , number of probes: 60\n",
      "-- region: NRNX1-seg-194 , number of probes: 60\n",
      "-- region: NRNX1-seg-195 , number of probes: 60\n",
      "-- region: NRNX1-seg-196 , number of probes: 45\n",
      "-- region: NRNX1-seg-197 , number of probes: 60\n",
      "-- region: NRNX1-seg-198 , number of probes: 47\n",
      "-- region: NRNX1-seg-199 , number of probes: 60\n",
      "-- region: NRNX1-seg-200 , number of probes: 60\n",
      "-- region: NRNX1-seg-201 , number of probes: 60\n",
      "-- region: NRNX1-seg-202 , number of probes: 40\n",
      "-- region: NRNX1-seg-203 , number of probes: 43\n",
      "-- region: NRNX1-seg-204 , number of probes: 60\n",
      "-- region: NRNX1-seg-205 , number of probes: 60\n",
      "-- region: NRNX1-seg-206 , number of probes: 60\n",
      "-- region: NRNX1-seg-207 , number of probes: 60\n",
      "-- region: NRNX1-seg-208 , number of probes: 51\n",
      "-- region: NRNX1-seg-209 , number of probes: 60\n",
      "-- region: NRNX1-seg-210 , number of probes: 60\n",
      "-- region: NRNX1-seg-211 , number of probes: 60\n",
      "-- region: NRNX1-seg-212 , number of probes: 12\n",
      "-- region: NRNX1-seg-213 , number of probes: 43\n",
      "-- region: NRNX1-seg-214 , number of probes: 60\n",
      "-- region: NRNX1-seg-215 , number of probes: 25\n",
      "-- region: NRNX1-seg-216 , number of probes: 20\n",
      "-- region: NRNX1-seg-217 , number of probes: 23\n",
      "-- region: NRNX1-seg-218 , number of probes: 43\n",
      "-- region: NRNX1-seg-219 , number of probes: 20\n",
      "-- region: NRNX1-seg-220 , number of probes: 30\n",
      "-- region: NRNX1-seg-221 , number of probes: 60\n",
      "-- region: NRNX1-seg-222 , number of probes: 60\n",
      "-- region: NRNX1-seg-223 , number of probes: 60\n",
      "-- region: NRNX1-seg-224 , number of probes: 60\n",
      "-- region: NRNX1-seg-225 , number of probes: 59\n",
      "-- region: NRNX1-seg-226 , number of probes: 53\n",
      "-- region: NRNX1-seg-227 , number of probes: 41\n",
      "-- region: NRNX1-seg-228 , number of probes: 26\n",
      "-- region: NRNX1-seg-229 , number of probes: 60\n",
      "-- region: NRNX1-seg-230 , number of probes: 60\n",
      "-- region: NRNX1-seg-231 , number of probes: 60\n",
      "-- region: NRNX1-seg-232 , number of probes: 29\n",
      "-- region: NRNX1-seg-233 , number of probes: 60\n",
      "-- region: NRNX1-seg-234 , number of probes: 52\n",
      "-- region: NRNX1-seg-235 , number of probes: 59\n",
      "-- region: NRNX1-seg-236 , number of probes: 53\n",
      "-- region: NRNX1-seg-238 , number of probes: 47\n",
      "-- region: NRNX1-seg-239 , number of probes: 60\n",
      "-- region: NRNX1-seg-240 , number of probes: 60\n",
      "-- region: NRNX1-seg-241 , number of probes: 49\n",
      "-- region: NRNX1-seg-242 , number of probes: 43\n",
      "-- region: NRNX1-seg-244 , number of probes: 52\n",
      "-- region: NRNX1-seg-245 , number of probes: 60\n",
      "-- region: NRNX1-seg-246 , number of probes: 37\n",
      "-- region: NRNX1-seg-248 , number of probes: 60\n",
      "-- region: NRNX1-seg-249 , number of probes: 37\n",
      "-- region: NRNX1-seg-251 , number of probes: 60\n",
      "-- region: NRNX1-seg-252 , number of probes: 53\n",
      "-- region: NRNX1-seg-253 , number of probes: 60\n",
      "-- region: NRNX1-seg-254 , number of probes: 50\n",
      "-- region: NRNX1-seg-255 , number of probes: 60\n",
      "-- region: NRNX1-seg-256 , number of probes: 59\n",
      "-- region: NRNX1-seg-257 , number of probes: 50\n",
      "-- region: NRNX1-seg-258 , number of probes: 60\n",
      "-- region: NRNX1-seg-259 , number of probes: 49\n",
      "-- region: NRNX1-seg-260 , number of probes: 45\n",
      "-- region: NRNX1-seg-262 , number of probes: 41\n",
      "-- region: NRNX1-seg-263 , number of probes: 22\n",
      "-- region: NRNX1-seg-264 , number of probes: 35\n",
      "-- region: NRNX1-seg-265 , number of probes: 24\n",
      "-- region: NRNX1-seg-266 , number of probes: 29\n",
      "-- region: NRNX1-seg-267 , number of probes: 54\n",
      "-- region: NRNX1-seg-268 , number of probes: 60\n",
      "-- region: NRNX1-seg-269 , number of probes: 60\n",
      "-- region: NRNX1-seg-270 , number of probes: 45\n",
      "-- region: NRNX1-seg-271 , number of probes: 59\n",
      "-- region: NRNX1-seg-272 , number of probes: 60\n",
      "-- region: NRNX1-seg-273 , number of probes: 32\n",
      "-- region: NRNX1-seg-274 , number of probes: 60\n",
      "-- region: NRNX1-seg-275 , number of probes: 55\n",
      "-- region: NRNX1-seg-276 , number of probes: 56\n",
      "-- region: NRNX1-seg-277 , number of probes: 46\n",
      "-- region: NRNX1-seg-278 , number of probes: 14\n",
      "-- region: NRNX1-seg-279 , number of probes: 19\n",
      "-- region: NRNX1-seg-280 , number of probes: 48\n",
      "-- region: NRNX1-seg-281 , number of probes: 60\n",
      "-- region: NRNX1-seg-282 , number of probes: 36\n",
      "-- region: NRNX1-seg-283 , number of probes: 55\n",
      "-- region: NRNX1-seg-284 , number of probes: 60\n",
      "-- region: NRNX1-seg-285 , number of probes: 44\n",
      "-- region: NRNX1-seg-286 , number of probes: 36\n",
      "-- region: NRNX1-seg-287 , number of probes: 25\n",
      "-- region: NRNX1-seg-288 , number of probes: 29\n",
      "-- region: NRNX1-seg-289 , number of probes: 41\n",
      "-- region: NRNX1-seg-290 , number of probes: 35\n",
      "-- region: NRNX1-seg-291 , number of probes: 39\n",
      "-- region: NRNX1-seg-292 , number of probes: 57\n",
      "-- region: NRNX1-seg-293 , number of probes: 17\n",
      "-- region: NRNX1-seg-294 , number of probes: 20\n",
      "-- region: NRNX1-seg-295 , number of probes: 40\n",
      "-- region: NRNX1-seg-298 , number of probes: 40\n",
      "-- region: NRNX1-seg-299 , number of probes: 34\n",
      "-- region: NRNX1-seg-300 , number of probes: 33\n",
      "-- region: NRNX1-seg-301 , number of probes: 53\n",
      "-- region: NRNX1-seg-302 , number of probes: 28\n",
      "-- region: NRNX1-seg-303 , number of probes: 50\n",
      "-- region: NRNX1-seg-304 , number of probes: 57\n",
      "-- region: NRNX1-seg-305 , number of probes: 46\n",
      "-- region: NRNX1-seg-306 , number of probes: 43\n",
      "-- region: NRNX1-seg-307 , number of probes: 13\n",
      "-- region: NRNX1-seg-308 , number of probes: 12\n",
      "-- region: NRNX1-seg-311 , number of probes: 57\n",
      "-- region: NRNX1-seg-312 , number of probes: 57\n",
      "-- region: NRNX1-seg-313 , number of probes: 60\n",
      "-- region: NRNX1-seg-314 , number of probes: 54\n",
      "-- region: NRNX1-seg-315 , number of probes: 38\n",
      "-- region: NRNX1-seg-316 , number of probes: 44\n",
      "-- region: NRNX1-seg-317 , number of probes: 42\n",
      "-- region: NRNX1-seg-318 , number of probes: 32\n",
      "-- region: NRNX1-seg-319 , number of probes: 22\n",
      "-- region: NRNX1-seg-320 , number of probes: 48\n",
      "-- region: NRNX1-seg-321 , number of probes: 60\n",
      "-- region: NRNX1-seg-322 , number of probes: 55\n",
      "-- region: NRNX1-seg-323 , number of probes: 40\n",
      "-- region: NRNX1-seg-324 , number of probes: 57\n",
      "-- region: NRNX1-seg-325 , number of probes: 57\n",
      "-- region: NRNX1-seg-326 , number of probes: 60\n",
      "-- region: NRNX1-seg-327 , number of probes: 60\n",
      "-- region: NRNX1-seg-328 , number of probes: 27\n",
      "-- region: NRNX1-seg-329 , number of probes: 60\n",
      "-- region: NRNX1-seg-330 , number of probes: 26\n",
      "-- region: NRNX1-seg-331 , number of probes: 53\n",
      "-- region: NRNX1-seg-332 , number of probes: 58\n",
      "-- region: NRNX1-seg-333 , number of probes: 47\n",
      "-- region: NRNX1-seg-334 , number of probes: 48\n",
      "-- region: NRNX1-seg-335 , number of probes: 52\n",
      "-- region: NRNX1-seg-336 , number of probes: 60\n",
      "-- region: NRNX1-seg-337 , number of probes: 31\n",
      "-- region: NRNX1-seg-338 , number of probes: 42\n",
      "-- region: NRNX1-seg-339 , number of probes: 46\n",
      "-- region: NRNX1-seg-340 , number of probes: 46\n",
      "-- region: NRNX1-seg-341 , number of probes: 49\n",
      "-- region: NRNX1-seg-342 , number of probes: 60\n",
      "-- region: NRNX1-seg-343 , number of probes: 43\n",
      "-- region: NRNX1-seg-344 , number of probes: 56\n",
      "-- region: NRNX1-seg-345 , number of probes: 37\n",
      "-- region: NRNX1-seg-346 , number of probes: 40\n",
      "-- region: NRNX1-seg-347 , number of probes: 17\n",
      "-- region: NRNX1-seg-348 , number of probes: 39\n",
      "-- region: NRNX1-seg-349 , number of probes: 35\n",
      "-- region: NRNX1-seg-350 , number of probes: 35\n",
      "-- region: NRNX1-seg-351 , number of probes: 53\n",
      "-- region: NRNX1-seg-352 , number of probes: 35\n",
      "-- region: NRNX1-seg-353 , number of probes: 58\n",
      "-- region: NRNX1-seg-354 , number of probes: 50\n",
      "-- region: NRNX1-seg-355 , number of probes: 57\n",
      "-- region: NRNX1-seg-356 , number of probes: 38\n",
      "-- region: NRNX1-seg-357 , number of probes: 60\n",
      "-- region: NRNX1-seg-358 , number of probes: 43\n",
      "-- region: NRNX1-seg-359 , number of probes: 46\n",
      "-- region: NRNX1-seg-360 , number of probes: 56\n",
      "-- region: NRNX1-seg-361 , number of probes: 59\n",
      "-- region: NRNX1-seg-362 , number of probes: 37\n",
      "-- region: NRNX1-seg-363 , number of probes: 55\n",
      "-- region: NRNX1-seg-364 , number of probes: 54\n",
      "-- region: NRNX1-seg-365 , number of probes: 36\n",
      "-- region: NRNX1-seg-366 , number of probes: 36\n",
      "-- region: NRNX1-seg-367 , number of probes: 60\n",
      "-- region: NRNX1-seg-368 , number of probes: 47\n",
      "-- region: NRNX1-seg-369 , number of probes: 53\n",
      "-- region: NRNX1-seg-370 , number of probes: 49\n",
      "-- region: NRNX1-seg-371 , number of probes: 46\n",
      "-- region: NRNX1-seg-372 , number of probes: 60\n",
      "-- region: NRNX1-seg-373 , number of probes: 54\n",
      "-- region: NRNX1-seg-374 , number of probes: 21\n",
      "-- region: NRNX1-seg-375 , number of probes: 19\n",
      "-- region: NRNX1-seg-376 , number of probes: 12\n",
      "-- region: NRNX1-seg-379 , number of probes: 26\n",
      "-- region: NRNX1-seg-380 , number of probes: 46\n",
      "-- region: NRNX1-seg-381 , number of probes: 59\n",
      "-- region: NRNX1-seg-382 , number of probes: 42\n",
      "-- region: NRNX1-seg-383 , number of probes: 51\n",
      "-- region: NRNX1-seg-384 , number of probes: 40\n",
      "-- region: NRNX1-seg-385 , number of probes: 55\n",
      "-- region: NRNX1-seg-386 , number of probes: 58\n",
      "-- region: NRNX1-seg-387 , number of probes: 44\n",
      "-- region: NRNX1-seg-388 , number of probes: 60\n",
      "-- region: NRNX1-seg-389 , number of probes: 60\n",
      "-- region: NRNX1-seg-390 , number of probes: 32\n",
      "-- region: NRNX1-seg-391 , number of probes: 60\n",
      "-- region: NRNX1-seg-392 , number of probes: 60\n",
      "-- region: NRNX1-seg-393 , number of probes: 49\n",
      "-- region: NRNX1-seg-394 , number of probes: 52\n",
      "-- region: NRNX1-seg-395 , number of probes: 56\n",
      "-- region: NRNX1-seg-396 , number of probes: 42\n",
      "-- region: NRNX1-seg-397 , number of probes: 52\n",
      "-- region: NRNX1-seg-398 , number of probes: 47\n",
      "-- region: NRNX1-seg-399 , number of probes: 40\n",
      "-- region: NRNX1-seg-400 , number of probes: 54\n",
      "-- region: NRNX1-seg-401 , number of probes: 60\n",
      "-- region: NRNX1-seg-402 , number of probes: 53\n",
      "-- region: NRNX1-seg-403 , number of probes: 60\n",
      "-- region: NRNX1-seg-404 , number of probes: 56\n",
      "-- region: NRNX1-seg-405 , number of probes: 47\n",
      "-- region: NRNX1-seg-406 , number of probes: 51\n",
      "-- region: NRNX1-seg-407 , number of probes: 35\n",
      "-- region: NRNX1-seg-408 , number of probes: 44\n",
      "-- region: NRNX1-seg-409 , number of probes: 44\n",
      "-- region: NRNX1-seg-410 , number of probes: 51\n",
      "-- region: NRNX1-seg-411 , number of probes: 46\n",
      "-- region: NRNX1-seg-412 , number of probes: 34\n",
      "-- region: NRNX1-seg-413 , number of probes: 39\n",
      "-- region: NRNX1-seg-414 , number of probes: 58\n",
      "-- region: NRNX1-seg-415 , number of probes: 47\n",
      "-- region: NRNX1-seg-416 , number of probes: 52\n",
      "-- region: NRNX1-seg-417 , number of probes: 39\n",
      "-- region: NRNX1-seg-418 , number of probes: 41\n",
      "-- region: NRNX1-seg-419 , number of probes: 54\n",
      "-- region: NRNX1-seg-420 , number of probes: 55\n",
      "-- region: NRNX1-seg-421 , number of probes: 60\n",
      "-- region: NRNX1-seg-422 , number of probes: 48\n",
      "-- region: NRNX1-seg-423 , number of probes: 50\n",
      "-- region: NRNX1-seg-424 , number of probes: 59\n",
      "-- region: NRNX1-seg-425 , number of probes: 49\n",
      "-- region: NRNX1-seg-426 , number of probes: 53\n",
      "-- region: NRNX1-seg-427 , number of probes: 60\n",
      "-- region: NRNX1-seg-428 , number of probes: 45\n",
      "-- region: NRNX1-seg-429 , number of probes: 44\n",
      "-- region: NRNX1-seg-430 , number of probes: 58\n",
      "-- region: NRNX1-seg-431 , number of probes: 39\n",
      "-- region: NRNX1-seg-432 , number of probes: 55\n",
      "-- region: NRNX1-seg-433 , number of probes: 53\n",
      "-- region: NRNX1-seg-434 , number of probes: 50\n",
      "-- region: NRNX1-seg-435 , number of probes: 44\n",
      "-- region: NRNX1-seg-436 , number of probes: 52\n",
      "-- region: NRNX1-seg-437 , number of probes: 40\n",
      "-- region: NRNX1-seg-438 , number of probes: 33\n",
      "-- region: NRNX1-seg-439 , number of probes: 38\n",
      "-- region: NRNX1-seg-440 , number of probes: 55\n",
      "-- region: NRNX1-seg-441 , number of probes: 57\n",
      "-- region: NRNX1-seg-442 , number of probes: 50\n",
      "-- region: NRNX1-seg-443 , number of probes: 39\n",
      "-- region: NRNX1-seg-444 , number of probes: 46\n",
      "-- region: NRNX1-seg-445 , number of probes: 60\n",
      "-- region: NRNX1-seg-446 , number of probes: 60\n",
      "-- region: NRNX1-seg-447 , number of probes: 41\n",
      "-- region: NRNX1-seg-448 , number of probes: 52\n",
      "-- region: NRNX1-seg-449 , number of probes: 43\n",
      "-- region: NRNX1-seg-450 , number of probes: 47\n",
      "-- region: NRNX1-seg-452 , number of probes: 45\n",
      "-- region: NRNX1-seg-453 , number of probes: 60\n",
      "-- region: NRNX1-seg-454 , number of probes: 48\n",
      "-- region: NRNX1-seg-455 , number of probes: 43\n",
      "-- region: NRNX1-seg-456 , number of probes: 49\n",
      "-- region: NRNX1-seg-457 , number of probes: 47\n",
      "-- region: NRNX1-seg-458 , number of probes: 41\n",
      "-- region: NRNX1-seg-459 , number of probes: 60\n",
      "-- region: NRNX1-seg-460 , number of probes: 46\n",
      "-- region: NRNX1-seg-461 , number of probes: 52\n",
      "-- region: NRNX1-seg-462 , number of probes: 48\n",
      "-- region: NRNX1-seg-463 , number of probes: 59\n",
      "-- region: NRNX1-seg-464 , number of probes: 53\n",
      "-- region: NRNX1-seg-465 , number of probes: 60\n",
      "-- region: NRNX1-seg-466 , number of probes: 34\n",
      "-- region: NRNX1-seg-467 , number of probes: 60\n",
      "-- region: NRNX1-seg-468 , number of probes: 51\n",
      "-- region: NRNX1-seg-469 , number of probes: 60\n",
      "-- region: NRNX1-seg-470 , number of probes: 60\n",
      "-- region: NRNX1-seg-471 , number of probes: 60\n",
      "-- region: NRNX1-seg-472 , number of probes: 20\n",
      "-- region: NRNX1-seg-474 , number of probes: 60\n",
      "-- region: NRNX1-seg-475 , number of probes: 60\n",
      "-- region: NRNX1-seg-476 , number of probes: 49\n",
      "-- region: NRNX1-seg-477 , number of probes: 58\n",
      "-- region: NRNX1-seg-478 , number of probes: 52\n",
      "-- region: NRNX1-seg-479 , number of probes: 56\n",
      "-- region: NRNX1-seg-480 , number of probes: 58\n",
      "-- region: NRNX1-seg-481 , number of probes: 60\n",
      "-- region: NRNX1-seg-482 , number of probes: 57\n",
      "-- region: NRNX1-seg-483 , number of probes: 22\n",
      "-- region: NRNX1-seg-484 , number of probes: 57\n",
      "-- region: NRNX1-seg-485 , number of probes: 60\n",
      "-- region: NRNX1-seg-486 , number of probes: 50\n",
      "-- region: NRNX1-seg-487 , number of probes: 60\n",
      "-- region: NRNX1-seg-488 , number of probes: 47\n",
      "-- region: NRNX1-seg-489 , number of probes: 57\n",
      "-- region: NRNX1-seg-490 , number of probes: 58\n",
      "-- region: NRNX1-seg-491 , number of probes: 50\n",
      "-- region: NRNX1-seg-492 , number of probes: 46\n",
      "-- region: NRNX1-seg-493 , number of probes: 54\n",
      "-- region: NRNX1-seg-494 , number of probes: 49\n",
      "-- region: NRNX1-seg-495 , number of probes: 58\n",
      "-- region: NRNX1-seg-496 , number of probes: 60\n",
      "-- region: NRNX1-seg-497 , number of probes: 49\n",
      "-- region: NRNX1-seg-498 , number of probes: 60\n",
      "-- region: NRNX1-seg-499 , number of probes: 60\n",
      "-- region: NRNX1-seg-500 , number of probes: 60\n",
      "-- region: NRNX1-seg-501 , number of probes: 49\n",
      "-- region: NRNX1-seg-502 , number of probes: 60\n",
      "-- region: NRNX1-seg-503 , number of probes: 48\n",
      "-- region: NRNX1-seg-504 , number of probes: 44\n",
      "-- region: NRNX1-seg-505 , number of probes: 44\n",
      "-- region: NRNX1-seg-506 , number of probes: 60\n",
      "-- region: NRNX1-seg-507 , number of probes: 58\n",
      "-- region: NRNX1-seg-508 , number of probes: 60\n",
      "-- region: NRNX1-seg-509 , number of probes: 44\n",
      "-- region: NRNX1-seg-510 , number of probes: 54\n",
      "-- region: NRNX1-seg-511 , number of probes: 52\n",
      "-- region: NRNX1-seg-512 , number of probes: 56\n",
      "-- region: NRNX1-seg-513 , number of probes: 52\n",
      "-- region: NRNX1-seg-514 , number of probes: 48\n",
      "-- region: NRNX1-seg-515 , number of probes: 60\n",
      "-- region: NRNX1-seg-516 , number of probes: 51\n",
      "-- region: NRNX1-seg-517 , number of probes: 59\n",
      "-- region: NRNX1-seg-518 , number of probes: 55\n",
      "-- region: NRNX1-seg-519 , number of probes: 60\n",
      "-- region: NRNX1-seg-520 , number of probes: 51\n",
      "-- region: NRNX1-seg-521 , number of probes: 60\n",
      "-- region: NRNX1-seg-522 , number of probes: 46\n",
      "-- region: NRNX1-seg-523 , number of probes: 59\n",
      "-- region: NRNX1-seg-524 , number of probes: 55\n",
      "-- region: NRNX1-seg-525 , number of probes: 58\n",
      "-- region: NRNX1-seg-526 , number of probes: 44\n",
      "-- region: NRNX1-seg-527 , number of probes: 57\n",
      "-- region: NRNX1-seg-528 , number of probes: 47\n",
      "-- region: NRNX1-seg-529 , number of probes: 50\n",
      "-- region: NRNX1-seg-530 , number of probes: 60\n",
      "-- region: NRNX1-seg-531 , number of probes: 45\n",
      "-- region: NRNX1-seg-532 , number of probes: 60\n",
      "-- region: NRNX1-seg-533 , number of probes: 60\n",
      "-- region: NRNX1-seg-534 , number of probes: 52\n",
      "-- region: NRNX1-seg-535 , number of probes: 38\n",
      "-- region: NRNX1-seg-536 , number of probes: 56\n",
      "-- region: NRNX1-seg-537 , number of probes: 51\n",
      "-- region: NRNX1-seg-538 , number of probes: 60\n",
      "-- region: NRNX1-seg-539 , number of probes: 54\n",
      "-- region: NRNX1-seg-540 , number of probes: 54\n",
      "-- region: NRNX1-seg-541 , number of probes: 45\n",
      "-- region: NRNX1-seg-542 , number of probes: 60\n",
      "-- region: NRNX1-seg-543 , number of probes: 49\n",
      "-- region: NRNX1-seg-544 , number of probes: 46\n",
      "-- region: NRNX1-seg-545 , number of probes: 60\n",
      "-- region: NRNX1-seg-546 , number of probes: 52\n",
      "-- region: NRNX1-seg-547 , number of probes: 45\n",
      "-- region: NRNX1-seg-548 , number of probes: 46\n",
      "-- region: NRNX1-seg-549 , number of probes: 46\n",
      "-- region: NRNX1-seg-550 , number of probes: 55\n",
      "-- region: NRNX1-seg-551 , number of probes: 48\n",
      "-- region: NRNX1-seg-552 , number of probes: 57\n",
      "-- region: NRNX1-seg-553 , number of probes: 49\n",
      "-- region: NRNX1-seg-554 , number of probes: 60\n",
      "-- region: NRNX1-seg-555 , number of probes: 59\n",
      "-- region: NRNX1-seg-556 , number of probes: 51\n",
      "-- region: NRNX1-seg-557 , number of probes: 56\n",
      "-- region: NRNX1-seg-558 , number of probes: 58\n",
      "-- region: NRNX1-seg-559 , number of probes: 51\n",
      "-- region: NRNX1-seg-560 , number of probes: 57\n",
      "-- region: NRNX1-seg-561 , number of probes: 40\n",
      "-- region: NRNX1-seg-562 , number of probes: 49\n",
      "-- region: NRNX1-seg-563 , number of probes: 43\n",
      "-- region: NRNX1-seg-564 , number of probes: 39\n",
      "-- region: NRNX1-seg-565 , number of probes: 49\n",
      "-- region: NRNX1-seg-566 , number of probes: 37\n",
      "-- region: NRNX1-seg-567 , number of probes: 49\n",
      "-- region: NRNX1-seg-568 , number of probes: 45\n",
      "-- region: NRNX1-seg-569 , number of probes: 60\n",
      "-- region: NRNX1-seg-570 , number of probes: 58\n",
      "-- region: NRNX1-seg-571 , number of probes: 54\n",
      "-- region: NRNX1-seg-572 , number of probes: 58\n",
      "-- region: NRNX1-seg-573 , number of probes: 60\n",
      "-- region: NRNX1-seg-574 , number of probes: 42\n",
      "-- region: NRNX1-seg-575 , number of probes: 47\n",
      "-- region: NRNX1-seg-576 , number of probes: 60\n",
      "-- region: NRNX1-seg-577 , number of probes: 44\n",
      "-- region: NRNX1-seg-578 , number of probes: 39\n",
      "-- region: NRNX1-seg-579 , number of probes: 41\n",
      "-- region: NRNX1-seg-580 , number of probes: 52\n",
      "-- region: NRNX1-seg-581 , number of probes: 52\n",
      "-- region: NRNX1-seg-582 , number of probes: 55\n",
      "-- region: NRNX1-seg-583 , number of probes: 60\n",
      "-- region: NRNX1-seg-584 , number of probes: 39\n",
      "-- region: NRNX1-seg-585 , number of probes: 50\n",
      "-- region: NRNX1-seg-586 , number of probes: 54\n",
      "-- region: NRNX1-seg-587 , number of probes: 32\n",
      "-- region: NRNX1-seg-588 , number of probes: 34\n",
      "-- region: NRNX1-seg-589 , number of probes: 30\n",
      "-- region: NRNX1-seg-590 , number of probes: 50\n",
      "-- region: NRNX1-seg-591 , number of probes: 48\n",
      "-- region: NRNX1-seg-592 , number of probes: 49\n",
      "-- region: NRNX1-seg-593 , number of probes: 55\n",
      "-- region: NRNX1-seg-594 , number of probes: 44\n",
      "-- region: NRNX1-seg-595 , number of probes: 40\n",
      "-- region: NRNX1-seg-596 , number of probes: 45\n",
      "-- region: NRNX1-seg-597 , number of probes: 39\n",
      "-- region: NRNX1-seg-598 , number of probes: 46\n",
      "-- region: NRNX1-seg-599 , number of probes: 58\n",
      "-- region: NRNX1-seg-600 , number of probes: 53\n",
      "-- region: NRNX1-seg-601 , number of probes: 51\n",
      "-- region: NRNX1-seg-602 , number of probes: 41\n",
      "-- region: NRNX1-seg-603 , number of probes: 59\n",
      "-- region: NRNX1-seg-604 , number of probes: 35\n",
      "-- region: NRNX1-seg-605 , number of probes: 55\n",
      "-- region: NRNX1-seg-606 , number of probes: 60\n",
      "-- region: NRNX1-seg-607 , number of probes: 60\n",
      "-- region: NRNX1-seg-608 , number of probes: 56\n",
      "-- region: NRNX1-seg-609 , number of probes: 32\n",
      "-- region: NRNX1-seg-610 , number of probes: 22\n",
      "-- region: NRNX1-seg-611 , number of probes: 45\n",
      "-- region: NRNX1-seg-612 , number of probes: 28\n",
      "-- region: NRNX1-seg-613 , number of probes: 22\n",
      "-- region: NRNX1-seg-614 , number of probes: 48\n",
      "-- region: NRNX1-seg-615 , number of probes: 32\n",
      "-- region: NRNX1-seg-616 , number of probes: 33\n",
      "-- region: NRNX1-seg-617 , number of probes: 48\n",
      "-- region: NRNX1-seg-618 , number of probes: 57\n",
      "-- region: NRNX1-seg-619 , number of probes: 60\n",
      "-- region: NRNX1-seg-620 , number of probes: 60\n",
      "-- region: NRNX1-seg-621 , number of probes: 56\n",
      "-- region: NRNX1-seg-622 , number of probes: 60\n",
      "-- region: NRNX1-seg-623 , number of probes: 50\n",
      "-- region: NRNX1-seg-624 , number of probes: 55\n",
      "-- region: NRNX1-seg-625 , number of probes: 19\n",
      "-- region: NRNX1-seg-626 , number of probes: 48\n",
      "-- region: NRNX1-seg-627 , number of probes: 39\n",
      "-- region: NRNX1-seg-628 , number of probes: 34\n",
      "-- region: NRNX1-seg-629 , number of probes: 40\n",
      "-- region: NRNX1-seg-630 , number of probes: 33\n",
      "-- region: NRNX1-seg-631 , number of probes: 50\n",
      "-- region: NRNX1-seg-632 , number of probes: 57\n",
      "-- region: NRNX1-seg-633 , number of probes: 49\n",
      "-- region: NRNX1-seg-634 , number of probes: 52\n",
      "-- region: NRNX1-seg-635 , number of probes: 44\n",
      "-- region: NRNX1-seg-636 , number of probes: 17\n",
      "-- region: NRNX1-seg-637 , number of probes: 11\n",
      "-- region: NRNX1-seg-639 , number of probes: 20\n",
      "-- region: NRNX1-seg-640 , number of probes: 53\n",
      "-- region: NRNX1-seg-641 , number of probes: 44\n",
      "-- region: NRNX1-seg-642 , number of probes: 45\n",
      "-- region: NRNX1-seg-643 , number of probes: 56\n",
      "-- region: NRNX1-seg-644 , number of probes: 54\n",
      "-- region: NRNX1-seg-645 , number of probes: 46\n",
      "-- region: NRNX1-seg-646 , number of probes: 60\n",
      "-- region: NRNX1-seg-647 , number of probes: 27\n",
      "-- region: NRNX1-seg-648 , number of probes: 39\n",
      "-- region: NRNX1-seg-649 , number of probes: 45\n",
      "-- region: NRNX1-seg-650 , number of probes: 43\n",
      "-- region: NRNX1-seg-651 , number of probes: 42\n",
      "-- region: NRNX1-seg-652 , number of probes: 31\n",
      "-- region: NRNX1-seg-653 , number of probes: 47\n",
      "-- region: NRNX1-seg-654 , number of probes: 53\n",
      "-- region: NRNX1-seg-655 , number of probes: 55\n",
      "-- region: NRNX1-seg-656 , number of probes: 46\n",
      "-- region: NRNX1-seg-657 , number of probes: 49\n",
      "-- region: NRNX1-seg-658 , number of probes: 60\n",
      "-- region: NRNX1-seg-659 , number of probes: 49\n",
      "-- region: NRNX1-seg-660 , number of probes: 50\n",
      "-- region: NRNX1-seg-661 , number of probes: 50\n",
      "-- region: NRNX1-seg-662 , number of probes: 44\n",
      "-- region: NRNX1-seg-665 , number of probes: 50\n",
      "-- region: NRNX1-seg-666 , number of probes: 60\n",
      "-- region: NRNX1-seg-667 , number of probes: 50\n",
      "-- region: NRNX1-seg-668 , number of probes: 49\n",
      "-- region: NRNX1-seg-669 , number of probes: 45\n",
      "-- region: NRNX1-seg-670 , number of probes: 32\n",
      "-- region: NRNX1-seg-671 , number of probes: 53\n",
      "-- region: NRNX1-seg-672 , number of probes: 37\n",
      "-- region: NRNX1-seg-674 , number of probes: 47\n",
      "-- region: NRNX1-seg-675 , number of probes: 42\n",
      "-- region: NRNX1-seg-676 , number of probes: 33\n",
      "-- region: NRNX1-seg-678 , number of probes: 33\n",
      "-- region: NRNX1-seg-679 , number of probes: 8\n",
      "-- region: NRNX1-seg-681 , number of probes: 23\n",
      "-- region: NRNX1-seg-682 , number of probes: 35\n",
      "-- region: NRNX1-seg-683 , number of probes: 53\n",
      "-- region: NRNX1-seg-684 , number of probes: 19\n",
      "-- region: NRNX1-seg-685 , number of probes: 34\n",
      "-- region: NRNX1-seg-686 , number of probes: 19\n",
      "-- region: NRNX1-seg-687 , number of probes: 21\n",
      "-- region: NRNX1-seg-688 , number of probes: 21\n",
      "-- region: NRNX1-seg-689 , number of probes: 43\n",
      "-- region: NRNX1-seg-690 , number of probes: 43\n",
      "-- region: NRNX1-seg-691 , number of probes: 30\n",
      "-- region: NRNX1-seg-692 , number of probes: 44\n",
      "-- region: NRNX1-seg-693 , number of probes: 54\n",
      "-- region: NRNX1-seg-694 , number of probes: 49\n",
      "-- region: NRNX1-seg-695 , number of probes: 53\n",
      "-- region: NRNX1-seg-696 , number of probes: 37\n",
      "-- region: NRNX1-seg-697 , number of probes: 40\n",
      "-- region: NRNX1-seg-698 , number of probes: 57\n",
      "-- region: NRNX1-seg-699 , number of probes: 46\n",
      "- Number of probes kept: 31808\n",
      "- Saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\.\\blast_full_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "# screen\n",
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "num_probes_per_region = 60\n",
    "kept_pbs, blast_keep_dic, hard_count_list, soft_count_list = check.Screening_Probes_by_Blast(library_folder, \n",
    "                                                                                             num_probes_per_region,\n",
    "                                                                                             keep_mode='front',\n",
    "                                                                                             hard_thres=target_len,\n",
    "                                                                                             soft_count_th=50,\n",
    "                                                                                             )"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d06655b7",
   "metadata": {},
   "source": [
    "# 5. Summary Table and plots"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "id": "b792aa76",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "31808\n"
     ]
    }
   ],
   "source": [
    "blasted_records = []\n",
    "with open(os.path.join(library_folder, 'blast_full_probes.fasta'), 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        blasted_records.append(record)\n",
    "print(len(blasted_records))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "id": "27cafd06",
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "\n",
    "region_info_dict = {\n",
    "    'gene_name':[],\n",
    "    'locus_info':[],\n",
    "    'chr':[],\n",
    "    #'chr_order':[],\n",
    "    'num_probes':[],\n",
    "    'readouts': [],\n",
    "    #'type':[],\n",
    "}\n",
    "\n",
    "probe_name_re_string = r'loc_(?P<loc>.+)_gene_(?P<gene>.+)_pb_(?P<pb>[0-9]+)_pos_(?P<pos>[0-9]+)(.+)_readouts_\\[(?P<readouts>.+?)\\]_primers_\\[(?P<fwd_primers>.+?),(?P<rev_primers>.+?)\\]'\n",
    "\n",
    "for _pb in blasted_records:\n",
    "    \n",
    "    _match = re.match(probe_name_re_string, _pb.id, )\n",
    "    # append all if not exist\n",
    "    if _match.groupdict()['gene'] not in region_info_dict['gene_name']:\n",
    "        # append new\n",
    "        region_info_dict['gene_name'].append(_match.groupdict()['gene'])\n",
    "        region_info_dict['locus_info'].append(_match.groupdict()['loc'])\n",
    "        region_info_dict['chr'].append(_match.groupdict()['loc'].split(':')[0])\n",
    "        region_info_dict['num_probes'].append(1)\n",
    "        region_info_dict['readouts'].append(list(np.unique(_match.groupdict()['readouts'].split(','))))\n",
    "    else:\n",
    "        _ind = region_info_dict['gene_name'].index(_match.groupdict()['gene'])\n",
    "        region_info_dict['num_probes'][_ind] += 1\n",
    "        for _rd in np.unique(_match.groupdict()['readouts'].split(',')):\n",
    "            if _rd not in region_info_dict['readouts'][_ind]:\n",
    "                region_info_dict['readouts'][_ind].append(_rd)\n",
    "        \n",
    "    #break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "id": "2979cac5",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>gene_name</th>\n",
       "      <th>locus_info</th>\n",
       "      <th>chr</th>\n",
       "      <th>num_probes</th>\n",
       "      <th>readouts</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NRNX1-seg-1</td>\n",
       "      <td>2:48000000-48005000</td>\n",
       "      <td>2</td>\n",
       "      <td>34</td>\n",
       "      <td>[NDB_376_c, NDB_533_c, NDB_936_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NRNX1-seg-2</td>\n",
       "      <td>2:48005000-48010000</td>\n",
       "      <td>2</td>\n",
       "      <td>38</td>\n",
       "      <td>[NDB_400_c, NDB_464_c, NDB_804_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NRNX1-seg-3</td>\n",
       "      <td>2:48010000-48015000</td>\n",
       "      <td>2</td>\n",
       "      <td>36</td>\n",
       "      <td>[NDB_858_c, NDB_511_c, NDB_875_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NRNX1-seg-4</td>\n",
       "      <td>2:48015000-48020000</td>\n",
       "      <td>2</td>\n",
       "      <td>50</td>\n",
       "      <td>[NDB_508_c, NDB_470_c, NDB_438_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>NRNX1-seg-5</td>\n",
       "      <td>2:48020000-48025000</td>\n",
       "      <td>2</td>\n",
       "      <td>60</td>\n",
       "      <td>[NDB_883_c, NDB_461_c, NDB_789_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>663</th>\n",
       "      <td>NRNX1-seg-695</td>\n",
       "      <td>2:51470000-51475000</td>\n",
       "      <td>2</td>\n",
       "      <td>53</td>\n",
       "      <td>[NDB_754_c, NDB_698_c, NDB_438_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>664</th>\n",
       "      <td>NRNX1-seg-696</td>\n",
       "      <td>2:51475000-51480000</td>\n",
       "      <td>2</td>\n",
       "      <td>37</td>\n",
       "      <td>[NDB_532_c, NDB_398_c, NDB_789_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>665</th>\n",
       "      <td>NRNX1-seg-697</td>\n",
       "      <td>2:51480000-51485000</td>\n",
       "      <td>2</td>\n",
       "      <td>40</td>\n",
       "      <td>[NDB_730_c, NDB_752_c, NDB_1047_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>666</th>\n",
       "      <td>NRNX1-seg-698</td>\n",
       "      <td>2:51485000-51490000</td>\n",
       "      <td>2</td>\n",
       "      <td>57</td>\n",
       "      <td>[NDB_853_c, NDB_899_c, NDB_618_c]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>667</th>\n",
       "      <td>NRNX1-seg-699</td>\n",
       "      <td>2:51490000-51495000</td>\n",
       "      <td>2</td>\n",
       "      <td>46</td>\n",
       "      <td>[NDB_445_c, NDB_935_c, NDB_819_c]</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>668 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "         gene_name           locus_info chr  num_probes  \\\n",
       "0      NRNX1-seg-1  2:48000000-48005000   2          34   \n",
       "1      NRNX1-seg-2  2:48005000-48010000   2          38   \n",
       "2      NRNX1-seg-3  2:48010000-48015000   2          36   \n",
       "3      NRNX1-seg-4  2:48015000-48020000   2          50   \n",
       "4      NRNX1-seg-5  2:48020000-48025000   2          60   \n",
       "..             ...                  ...  ..         ...   \n",
       "663  NRNX1-seg-695  2:51470000-51475000   2          53   \n",
       "664  NRNX1-seg-696  2:51475000-51480000   2          37   \n",
       "665  NRNX1-seg-697  2:51480000-51485000   2          40   \n",
       "666  NRNX1-seg-698  2:51485000-51490000   2          57   \n",
       "667  NRNX1-seg-699  2:51490000-51495000   2          46   \n",
       "\n",
       "                               readouts  \n",
       "0     [NDB_376_c, NDB_533_c, NDB_936_c]  \n",
       "1     [NDB_400_c, NDB_464_c, NDB_804_c]  \n",
       "2     [NDB_858_c, NDB_511_c, NDB_875_c]  \n",
       "3     [NDB_508_c, NDB_470_c, NDB_438_c]  \n",
       "4     [NDB_883_c, NDB_461_c, NDB_789_c]  \n",
       "..                                  ...  \n",
       "663   [NDB_754_c, NDB_698_c, NDB_438_c]  \n",
       "664   [NDB_532_c, NDB_398_c, NDB_789_c]  \n",
       "665  [NDB_730_c, NDB_752_c, NDB_1047_c]  \n",
       "666   [NDB_853_c, NDB_899_c, NDB_618_c]  \n",
       "667   [NDB_445_c, NDB_935_c, NDB_819_c]  \n",
       "\n",
       "[668 rows x 5 columns]"
      ]
     },
     "execution_count": 85,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "region_info_df = pd.DataFrame(region_info_dict)\n",
    "region_info_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "id": "f822e25a",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Add additional information\n",
    "_types = []\n",
    "for _g in region_info_df['gene_name']:\n",
    "    if 'hSE-' in _g:\n",
    "        _types.append('SE')\n",
    "    elif 'enhancer-' in _g:\n",
    "        _types.append('enhancer')\n",
    "    else:\n",
    "        _types.append('genome')\n",
    "# add to this df\n",
    "region_info_df['type'] = _types\n",
    "\n",
    "region_info_df['chr_order'] = -1\n",
    "for _chr in np.unique(region_info_df['chr']):\n",
    "    _reg_starts = [int(_info.split(':')[1].split('-')[0])\n",
    "                   for _info in region_info_df.loc[region_info_df['chr']==_chr,\n",
    "                                                   'locus_info']\n",
    "                  ]\n",
    "    _reg_orders = np.array(_reg_starts).argsort().argsort().astype(np.int32)\n",
    "    region_info_df.loc[region_info_df['chr']==_chr,'chr_order'] = _reg_orders"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "id": "d319419e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>gene_name</th>\n",
       "      <th>locus_info</th>\n",
       "      <th>chr</th>\n",
       "      <th>num_probes</th>\n",
       "      <th>readouts</th>\n",
       "      <th>type</th>\n",
       "      <th>chr_order</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NRNX1-seg-1</td>\n",
       "      <td>2:48000000-48005000</td>\n",
       "      <td>2</td>\n",
       "      <td>34</td>\n",
       "      <td>[NDB_376_c, NDB_533_c, NDB_936_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NRNX1-seg-2</td>\n",
       "      <td>2:48005000-48010000</td>\n",
       "      <td>2</td>\n",
       "      <td>38</td>\n",
       "      <td>[NDB_400_c, NDB_464_c, NDB_804_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NRNX1-seg-3</td>\n",
       "      <td>2:48010000-48015000</td>\n",
       "      <td>2</td>\n",
       "      <td>36</td>\n",
       "      <td>[NDB_858_c, NDB_511_c, NDB_875_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NRNX1-seg-4</td>\n",
       "      <td>2:48015000-48020000</td>\n",
       "      <td>2</td>\n",
       "      <td>50</td>\n",
       "      <td>[NDB_508_c, NDB_470_c, NDB_438_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>NRNX1-seg-5</td>\n",
       "      <td>2:48020000-48025000</td>\n",
       "      <td>2</td>\n",
       "      <td>60</td>\n",
       "      <td>[NDB_883_c, NDB_461_c, NDB_789_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>663</th>\n",
       "      <td>NRNX1-seg-695</td>\n",
       "      <td>2:51470000-51475000</td>\n",
       "      <td>2</td>\n",
       "      <td>53</td>\n",
       "      <td>[NDB_754_c, NDB_698_c, NDB_438_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>663</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>664</th>\n",
       "      <td>NRNX1-seg-696</td>\n",
       "      <td>2:51475000-51480000</td>\n",
       "      <td>2</td>\n",
       "      <td>37</td>\n",
       "      <td>[NDB_532_c, NDB_398_c, NDB_789_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>664</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>665</th>\n",
       "      <td>NRNX1-seg-697</td>\n",
       "      <td>2:51480000-51485000</td>\n",
       "      <td>2</td>\n",
       "      <td>40</td>\n",
       "      <td>[NDB_730_c, NDB_752_c, NDB_1047_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>665</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>666</th>\n",
       "      <td>NRNX1-seg-698</td>\n",
       "      <td>2:51485000-51490000</td>\n",
       "      <td>2</td>\n",
       "      <td>57</td>\n",
       "      <td>[NDB_853_c, NDB_899_c, NDB_618_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>666</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>667</th>\n",
       "      <td>NRNX1-seg-699</td>\n",
       "      <td>2:51490000-51495000</td>\n",
       "      <td>2</td>\n",
       "      <td>46</td>\n",
       "      <td>[NDB_445_c, NDB_935_c, NDB_819_c]</td>\n",
       "      <td>genome</td>\n",
       "      <td>667</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>668 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "         gene_name           locus_info chr  num_probes  \\\n",
       "0      NRNX1-seg-1  2:48000000-48005000   2          34   \n",
       "1      NRNX1-seg-2  2:48005000-48010000   2          38   \n",
       "2      NRNX1-seg-3  2:48010000-48015000   2          36   \n",
       "3      NRNX1-seg-4  2:48015000-48020000   2          50   \n",
       "4      NRNX1-seg-5  2:48020000-48025000   2          60   \n",
       "..             ...                  ...  ..         ...   \n",
       "663  NRNX1-seg-695  2:51470000-51475000   2          53   \n",
       "664  NRNX1-seg-696  2:51475000-51480000   2          37   \n",
       "665  NRNX1-seg-697  2:51480000-51485000   2          40   \n",
       "666  NRNX1-seg-698  2:51485000-51490000   2          57   \n",
       "667  NRNX1-seg-699  2:51490000-51495000   2          46   \n",
       "\n",
       "                               readouts    type  chr_order  \n",
       "0     [NDB_376_c, NDB_533_c, NDB_936_c]  genome          0  \n",
       "1     [NDB_400_c, NDB_464_c, NDB_804_c]  genome          1  \n",
       "2     [NDB_858_c, NDB_511_c, NDB_875_c]  genome          2  \n",
       "3     [NDB_508_c, NDB_470_c, NDB_438_c]  genome          3  \n",
       "4     [NDB_883_c, NDB_461_c, NDB_789_c]  genome          4  \n",
       "..                                  ...     ...        ...  \n",
       "663   [NDB_754_c, NDB_698_c, NDB_438_c]  genome        663  \n",
       "664   [NDB_532_c, NDB_398_c, NDB_789_c]  genome        664  \n",
       "665  [NDB_730_c, NDB_752_c, NDB_1047_c]  genome        665  \n",
       "666   [NDB_853_c, NDB_899_c, NDB_618_c]  genome        666  \n",
       "667   [NDB_445_c, NDB_935_c, NDB_819_c]  genome        667  \n",
       "\n",
       "[668 rows x 7 columns]"
      ]
     },
     "execution_count": 87,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "region_info_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "id": "807aa9f2",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "saving region info to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\summary_region_info.csv\n"
     ]
    }
   ],
   "source": [
    "overwrite_region_info = True\n",
    "region_info_filename = os.path.join(library_folder, r'summary_region_info.csv')\n",
    "\n",
    "if not os.path.exists(region_info_filename) or overwrite_region_info:\n",
    "    print(f\"saving region info to file: {region_info_filename}\")\n",
    "    region_info_df.to_csv(region_info_filename, index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0d1a3125",
   "metadata": {},
   "source": [
    "## Generate summary plots"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1bbf42ac",
   "metadata": {},
   "outputs": [],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "# get region size dict\n",
    "min_num_probes = 50\n",
    "reg_size_dic, len_check = check._check_region_size(blasted_records, \n",
    "                                                   min_size=min_num_probes)\n",
    "print(len_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a330f8a4",
   "metadata": {},
   "outputs": [],
   "source": [
    "plt.style.use('default')\n",
    "#plt.style.use('dark_background')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e4fb5b25",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Plot number of probes per region\n",
    "plt.figure(figsize=(6,3), dpi=100)\n",
    "plt.plot(list(reg_size_dic.values()))\n",
    "#plt.plot(np.ones(len(pb_nums))*20)\n",
    "plt.ylabel(f\"Num of probes\", labelpad=1)\n",
    "plt.xlabel(f\"Region id\", labelpad=1)\n",
    "plt.ylim([0, np.max(list(reg_size_dic.values()))*1.1])\n",
    "plt.savefig(os.path.join(report_folder, 'afterblast_nProbes_per_region.png'),\n",
    "            transparent=True, dpi=300)\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5d0d61c0",
   "metadata": {},
   "source": [
    "## number and location on each chr"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6e65ec21",
   "metadata": {},
   "outputs": [],
   "source": [
    "chrs = []\n",
    "chr_2_centers = {}\n",
    "chr_2_types = {}\n",
    "for _loc_name, _type in zip(region_info_df['locus_info'], region_info_df['type']):\n",
    "    _chr = _loc_name.split(':')[0]\n",
    "    _start, _end = _loc_name.split(':')[1].split('-')\n",
    "    _center = (int(_start)+int(_end))/2\n",
    "    if _chr not in chrs:\n",
    "        chrs.append(_chr)\n",
    "    # create\n",
    "    if _chr not in chr_2_centers:\n",
    "        chr_2_centers[_chr] = []\n",
    "        chr_2_types[_chr] = []\n",
    "    # append\n",
    "    if _center not in chr_2_centers[_chr]:\n",
    "        chr_2_centers[_chr].append(_center)\n",
    "        chr_2_types[_chr].append(_type)\n",
    "# region number\n",
    "chr_2_region_num = {_chr:len(cts) for _chr, cts in chr_2_centers.items()}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "9f5911e4",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Load chromosome sizes\n",
    "import pandas as pd\n",
    "from ImageAnalysis3.structure_tools.distance import sort_chr\n",
    "\n",
    "sorted_chrs = sorted(chrs, key=lambda _c: sort_chr(_c))\n",
    "\n",
    "chrom_size_df = pd.read_csv(r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\hg38.chrom.sizes',\n",
    "            delimiter='\\t', header=None)\n",
    "\n",
    "chr_2_len = {}\n",
    "for _chr in chrs:\n",
    "    chr_2_len[_chr] = chrom_size_df.loc[chrom_size_df[0]==f'chr{_chr}', 1].values[0]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "932982a9",
   "metadata": {},
   "outputs": [],
   "source": [
    "%matplotlib inline\n",
    "color_map = {\n",
    "    'genome':'blue',\n",
    "}\n",
    "#plt.style.use('dark_background')\n",
    "from matplotlib.lines import Line2D\n",
    "\n",
    "fig, ax = plt.subplots(figsize=(6,4),dpi=150)\n",
    "for _i, _chr in enumerate(sorted_chrs):\n",
    "    ax.hlines(_i, 0, chr_2_len[_chr], color='black', linewidth=1)\n",
    "    ax.scatter(chr_2_centers[_chr], \n",
    "               np.ones(len(chr_2_centers[_chr]))* _i, \n",
    "               marker='+', linewidth=1,\n",
    "               s=15, c=[color_map[_t] for _t in chr_2_types[_chr]],\n",
    "               )\n",
    "# create legend patch\n",
    "legend_element = [Line2D([0],[0], color=[0,0,0,0], \n",
    "                         label=_type, marker='+', markersize=8,\n",
    "                         markeredgecolor=_c, markerfacecolor=_c,)\n",
    "                  for _type,_c in color_map.items()]\n",
    "ax.legend(handles=legend_element)\n",
    "ax.set_yticks(np.arange(len(sorted_chrs)))\n",
    "ax.set_yticklabels(sorted_chrs)\n",
    "#ax.set_xlim([0, chr_2_len['1']])\n",
    "ax.set_xlabel(\"Genomic Location\")\n",
    "ax.set_ylabel(\"Chromosome\")\n",
    "#ax.legend([_sc])\n",
    "plt.savefig(os.path.join(report_folder, 'target_location_summary.png'), \n",
    "            dpi=300, transparent=True)\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7b68187e",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "fig, ax1 = plt.subplots(figsize=(6,4),dpi=150)\n",
    "ax2 = ax1.twinx()\n",
    "_bar_p = ax1.bar(sorted_chrs, \n",
    "                 [chr_2_region_num[_c] for _c in sorted_chrs], alpha=0.5)\n",
    "ax1.set_ylabel(\"Num. of regions\")\n",
    "ax2.plot(sorted_chrs, [chr_2_len[_c] for _c in sorted_chrs], color='r')\n",
    "ax2.set_ylim([0, np.max(list(chr_2_len.values()))*1.1])\n",
    "ax2.set_ylabel(\"Chr size (bp)\")\n",
    "plt.savefig(os.path.join(report_folder, 'target_chr_stat_summary.png'), \n",
    "            dpi=300, transparent=True)\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a9429fce",
   "metadata": {},
   "source": [
    "# Create count table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "id": "3cbb34c9",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n",
      "- Start multi-processing comsume 31808 sequences 44 threads, finish in 16.891s\n",
      "- Total sequences loaded: 4643968\n",
      "Time to compute unique and clip: 1.302971601486206\n",
      "Time to update matrix: 2.5318892002105713\n",
      "- start saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\ctp14-NRNX1-tracing_17w.npy\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "from tqdm import tqdm\n",
    "overwrite_table = False\n",
    "# library\n",
    "library_table_file = os.path.join(reference_folder, 'ctp14-NRNX1-tracing_17w.npy')\n",
    "\n",
    "if not os.path.exists(library_table_file) or overwrite_table:\n",
    "    # library\n",
    "    _library_filenames = [os.path.join(library_folder, 'blast_full_probes.fasta')]\n",
    "    print(len(_library_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=library_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_library_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=44) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {library_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "532acdcb",
   "metadata": {},
   "source": [
    "# Design corresponding sequential as validation\n",
    "\n",
    "start from here"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "id": "153261f4",
   "metadata": {},
   "outputs": [],
   "source": [
    "## load DNA-MERFISH count-table to select probes\n",
    "library_table_file = os.path.join(reference_folder, 'ctp14-NRNX1-tracing_17w.npy')\n",
    "if 'lib_table' not in locals():\n",
    "    lib_table = library_tools.design.countTable(save_file=library_table_file, word=17)\n",
    "    lib_table.load()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "00df1bc3",
   "metadata": {},
   "source": [
    "## select based on groups:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "id": "e3601b3a",
   "metadata": {},
   "outputs": [],
   "source": [
    "region_2_group_id = pickle.load(open(os.path.join(library_folder, 'region_2_group_id.pkl'), 'rb'))\n",
    "group_2_regions = {}\n",
    "for _r, _g in region_2_group_id.items():\n",
    "    if _g not in group_2_regions:\n",
    "        group_2_regions[_g] = [_r]\n",
    "    else:\n",
    "        group_2_regions[_g].append(_r)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "id": "61b7e60d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0 3 67\n",
      "1 4 67\n",
      "2 3 67\n",
      "3 1 67\n",
      "4 4 67\n",
      "5 4 67\n",
      "6 2 67\n",
      "7 1 67\n",
      "8 3 66\n",
      "9 3 66\n",
      "28\n"
     ]
    }
   ],
   "source": [
    "# split blasted probes into genes\n",
    "gene_2_probes = check.split_probe_by_gene(blasted_records,)\n",
    "\n",
    "sel_sequential_pb_dict = {}\n",
    "kept_group_th = 0.01 # percentage of regions that could be sequentially labeled\n",
    "_library_hit_th = 14\n",
    "_sequential_probe_num_th = 12\n",
    "\n",
    "# for each group, check pb_dict\n",
    "for _g, _region_names in group_2_regions.items():\n",
    "    _sel_regions = []\n",
    "    for _rname in _region_names:\n",
    "        if _rname not in pb_dict:\n",
    "            print(f\"{_rname} doesn't exist.\")\n",
    "            continue\n",
    "        # final probe num\n",
    "        _final_probe_num = len(gene_2_probes[_rname])\n",
    "        _design_probe_num = len(pb_dict[_rname].kept_probes)\n",
    "        if _design_probe_num - _final_probe_num > 12:\n",
    "            #print(_rname, _design_probe_num, _final_probe_num)\n",
    "            _sel_regions.append(_rname)\n",
    "    # check if all regions in this group have extra probes:\n",
    "    print(_g, len(_sel_regions), len(_region_names))\n",
    "    if len(_sel_regions) >= len(_region_names) * kept_group_th:\n",
    "        #print(_g, len(_sel_regions), len(_region_names))\n",
    "        # for probes in these regions, add to sel_sequential_pb_dict\n",
    "        \n",
    "        for _rname in _sel_regions:\n",
    "            # collect used\n",
    "            _used_pb_pos = [int(_r.id.split('pos_')[1].split('_')[0]) for _r in gene_2_probes[_rname]]\n",
    "            # re-assemble kept_pbs for each region\n",
    "            _kept_pbs = {}\n",
    "            for _seq, _info in pb_dict[_rname].kept_probes.items():\n",
    "                # map with multiplexed library\n",
    "                _library_hits = lib_table.get(_seq)\n",
    "                if _library_hits > _library_hit_th:\n",
    "                    #print(_library_hits, '*')\n",
    "                    continue\n",
    "                if _info['pb_index'] not in _used_pb_pos:\n",
    "                    _new_info = {_k:_v for _k,_v in _info.items()}\n",
    "                    # add sequential tag\n",
    "                    _new_info['reg_name'] = _new_info['reg_name'].replace('Chr7-6Mb-seg', 'Chr7-6Mb-sequential-seg')\n",
    "                    # store group info_\n",
    "                    _new_info['reg_name'] += f'_group_{_g}'\n",
    "                    _new_info['group_id'] = _g\n",
    "                    # map library\n",
    "                    _new_info['map_library'] = _library_hits\n",
    "                    # append\n",
    "                    _kept_pbs[_seq] = _new_info\n",
    "            if len(_kept_pbs) < _sequential_probe_num_th:\n",
    "                continue\n",
    "            #print(len(_kept_pbs))\n",
    "            _splitted_pbde = library_tools.design.pb_reports_class()\n",
    "            _splitted_pbde.kept_probes = _kept_pbs\n",
    "            \n",
    "            sel_sequential_pb_dict[_rname] = _splitted_pbde\n",
    "            \n",
    "print(len(sel_sequential_pb_dict))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "79167b52",
   "metadata": {},
   "source": [
    "## add sequential tag"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "id": "0d0fe667",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "4"
      ]
     },
     "execution_count": 98,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_readout_per_probe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 99,
   "id": "2a43dec8",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'NRNX1-seg-161': ['u0', 'u0', 'u0', 'u0'], 'NRNX1-seg-223': ['u1', 'u1', 'u1', 'u1'], 'NRNX1-seg-636': ['u2', 'u2', 'u2', 'u2'], 'NRNX1-seg-15': ['u3', 'u3', 'u3', 'u3'], 'NRNX1-seg-214': ['u4', 'u4', 'u4', 'u4'], 'NRNX1-seg-224': ['u5', 'u5', 'u5', 'u5'], 'NRNX1-seg-313': ['u6', 'u6', 'u6', 'u6'], 'NRNX1-seg-89': ['u7', 'u7', 'u7', 'u7'], 'NRNX1-seg-334': ['u8', 'u8', 'u8', 'u8'], 'NRNX1-seg-498': ['u9', 'u9', 'u9', 'u9'], 'NRNX1-seg-619': ['u10', 'u10', 'u10', 'u10'], 'NRNX1-seg-165': ['u11', 'u11', 'u11', 'u11'], 'NRNX1-seg-197': ['u12', 'u12', 'u12', 'u12'], 'NRNX1-seg-376': ['u13', 'u13', 'u13', 'u13'], 'NRNX1-seg-620': ['u14', 'u14', 'u14', 'u14'], 'NRNX1-seg-166': ['u15', 'u15', 'u15', 'u15'], 'NRNX1-seg-239': ['u16', 'u16', 'u16', 'u16'], 'NRNX1-seg-357': ['u17', 'u17', 'u17', 'u17'], 'NRNX1-seg-470': ['u18', 'u18', 'u18', 'u18'], 'NRNX1-seg-93': ['u19', 'u19', 'u19', 'u19'], 'NRNX1-seg-209': ['u20', 'u20', 'u20', 'u20'], 'NRNX1-seg-319': ['u21', 'u21', 'u21', 'u21'], 'NRNX1-seg-201': ['u22', 'u22', 'u22', 'u22'], 'NRNX1-seg-221': ['u23', 'u23', 'u23', 'u23'], 'NRNX1-seg-679': ['u24', 'u24', 'u24', 'u24'], 'NRNX1-seg-172': ['u25', 'u25', 'u25', 'u25'], 'NRNX1-seg-192': ['u26', 'u26', 'u26', 'u26'], 'NRNX1-seg-625': ['u27', 'u27', 'u27', 'u27']}\n"
     ]
    }
   ],
   "source": [
    "sequential_region_2_readout_dict = {}\n",
    "for _i, _reg in enumerate(sel_sequential_pb_dict):\n",
    "    sequential_region_2_readout_dict[_reg] = [f\"u{_i}\"] * num_readout_per_probe\n",
    "print(sequential_region_2_readout_dict)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "43a60d6b",
   "metadata": {},
   "source": [
    "## loading components"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1ebaa560",
   "metadata": {},
   "source": [
    "### primers doesn't change"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "59d9f6e0",
   "metadata": {},
   "source": [
    "### readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 100,
   "id": "03ce4e17",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'u': 28}"
      ]
     },
     "execution_count": 100,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# collect types from region_2_readout_dict\n",
    "sequential_readout_usage_by_type = {}\n",
    "for _region, _infos in sequential_region_2_readout_dict.items():\n",
    "    for _info in _infos:\n",
    "        if _info[0] not in sequential_readout_usage_by_type:\n",
    "            sequential_readout_usage_by_type[_info[0]] = []\n",
    "        if int(_info[1:]) not in sequential_readout_usage_by_type[_info[0]]:\n",
    "            sequential_readout_usage_by_type[_info[0]].append(int(_info[1:]))\n",
    "\n",
    "readout_usage_dict = {_type:len(_v) for _type,_v in sequential_readout_usage_by_type.items()}\n",
    "\n",
    "# collect types from region_2_readout_dict\n",
    "sequential_readout_usage_dict = {}\n",
    "for _region, _infos in sequential_region_2_readout_dict.items():\n",
    "    for _info in _infos:\n",
    "        if _info[0] not in sequential_readout_usage_by_type:\n",
    "            sequential_readout_usage_by_type[_info[0]] = []\n",
    "        if int(_info[1:]) not in sequential_readout_usage_by_type[_info[0]]:\n",
    "            sequential_readout_usage_by_type[_info[0]].append(int(_info[1:]))\n",
    "\n",
    "sequential_readout_usage_dict = {_type:len(_v) for _type,_v in sequential_readout_usage_by_type.items()}\n",
    "sequential_readout_usage_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 108,
   "id": "5fb824f4",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "81\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\selected_NDB_sets\\highest_snr_w_stv_750.fasta 175 81 10\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\selected_NDB_sets\\highest_snr_w_stv_647.fasta 174 81 10\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\selected_NDB_sets\\highest_snr_w_stv_561.fasta 180 81 10\n",
      "[10, 10, 10]\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3.classes import _allowed_kwds\n",
    "reload(ia.library_tools.assemble)\n",
    "sequential_readout_dict ={}\n",
    "overwrite_readout_usage = True\n",
    "readout_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\selected_NDB_sets'\n",
    "\n",
    "# combo\n",
    "readout_type = 'unique'\n",
    "# \n",
    "_type_char = _allowed_kwds[readout_type]\n",
    "_readout_start = 81\n",
    "print(_readout_start)\n",
    "_num_color = 3\n",
    "\n",
    "sequential_readout_dict[_type_char] = library_tools.assemble.load_readouts(\n",
    "    sequential_readout_usage_dict[_type_char], \n",
    "    _type='highest_snr_w_stv', \n",
    "    _readout_folder=readout_folder, \n",
    "    _num_colors=_num_color, # 750,647,561\n",
    "    _start_id=_readout_start,\n",
    "    )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 109,
   "id": "386bd3b4",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'u': [SeqRecord(seq=Seq('GACCTATCATTCGGGCTAGCAACTGCGATG'), id='NDB_940', name='NDB_940', description='NDB_940', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCGAGTAACGTGTGATGACGGTGTACCCG'), id='NDB_581', name='NDB_581', description='NDB_581', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCGTCGGTCAGATTTACGCTATTCAGGCC'), id='NDB_441', name='NDB_441', description='NDB_441', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCAATTATCGAAGTCGGTCGTTCTGACATA'), id='NDB_577', name='NDB_577', description='NDB_577', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCGTTGGACTTAAGTATTACGCCTGACGAC'), id='NDB_350', name='NDB_350', description='NDB_350', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTCAAGTCGAGTCCTATGTAACCGTGCAG'), id='NDB_402', name='NDB_402', description='NDB_402', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAACACGAAGATCCATAGGCGACCTTTACT'), id='NDB_556', name='NDB_556', description='NDB_556', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAACTGAGCTTATCGCTTCGCACGAAATGG'), id='NDB_371', name='NDB_371', description='NDB_371', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCATGCCGGGTCGTGATCAACGTATTAGG'), id='NDB_939', name='NDB_939', description='NDB_939', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCGTGAGTATTCGCGCGCATTTGGGAAGT'), id='NDB_934', name='NDB_934', description='NDB_934', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTACAGGATACGAGCCGCCATGCGAATAGT'), id='NDB_476', name='NDB_476', description='NDB_476', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTCAAGGGCATTACGGCCTTAACGGAACG'), id='NDB_798', name='NDB_798', description='NDB_798', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGGCTACGTTAGGTGCCTCGCTGCGTATAT'), id='NDB_481', name='NDB_481', description='NDB_481', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCACATAAGGTGCGTACTAGCTGACTATCT'), id='NDB_428', name='NDB_428', description='NDB_428', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACGCAGTACGATGACGTCCCATATTTGTAA'), id='NDB_585', name='NDB_585', description='NDB_585', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATCTGTTTAGCGCTCCGGAGTTCACCAGT'), id='NDB_910', name='NDB_910', description='NDB_910', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGGTTACTGGGTCGTCTATACCAACCTTA'), id='NDB_818', name='NDB_818', description='NDB_818', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACTTGGGATACGTTGACCATGCCGCCTTA'), id='NDB_606', name='NDB_606', description='NDB_606', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACGCATGTAGTTGCACGCGGGTCGAGTAAC'), id='NDB_775', name='NDB_775', description='NDB_775', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCGTCATTGAACAACCGGTCTTGCCAGAG'), id='NDB_365', name='NDB_365', description='NDB_365', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCGTGTTTCGCAATAGGGCGGCTACTTTG'), id='NDB_702', name='NDB_702', description='NDB_702', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCCAATAGCTCGGTGAGGATCTCTACGAGA'), id='NDB_439', name='NDB_439', description='NDB_439', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACTTGTGTCCAATCCGTACGTTTGTGACT'), id='NDB_494', name='NDB_494', description='NDB_494', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTTTAAACTCCGAATAAGCGCTTCTTGCC'), id='NDB_591', name='NDB_591', description='NDB_591', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCGAAGATCGTAGTTAAGTACCGCCATTT'), id='NDB_493', name='NDB_493', description='NDB_493', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGGGTTCAATTACGAATGGAGCTTCCTATC'), id='NDB_479', name='NDB_479', description='NDB_479', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATCAAGTTGCGCTAGGTATCTTGTGAACC'), id='NDB_483', name='NDB_483', description='NDB_483', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGTTCCAATGTACTGCCGGCTACATGCTG'), id='NDB_547', name='NDB_547', description='NDB_547', dbxrefs=[])]}"
      ]
     },
     "execution_count": 109,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sequential_readout_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 106,
   "id": "bc5e5a67",
   "metadata": {},
   "outputs": [],
   "source": [
    "reload(library_tools.assemble)\n",
    "from ImageAnalysis3.library_tools.assemble import Assemble_probes\n",
    "\n",
    "# params\n",
    "assemble_params = {\n",
    "    'primer_len':20,\n",
    "    'num_readout_per_probe':4, \n",
    "}\n",
    "sequential_assemble_folder = os.path.join(library_folder, 'Sequential')\n",
    "if not os.path.exists(sequential_assemble_folder):\n",
    "    os.makedirs(sequential_assemble_folder)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 107,
   "id": "fc400c48",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "save sequential readout usage dict\n"
     ]
    }
   ],
   "source": [
    "# save\n",
    "sequential_readout_usage_file = os.path.join(sequential_assemble_folder, 'sequential_readout_usage.pkl')\n",
    "if not os.path.isfile(sequential_readout_usage_file) or overwrite_readout_usage:\n",
    "    print('save sequential readout usage dict')\n",
    "    pickle.dump(readout_dict, open(sequential_readout_usage_file, 'wb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 110,
   "id": "448a361d",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Assemble probes by given target sequences, readouts and primers.\n",
      "-- included readout types: ['u']\n",
      "--- assemblying 15 probes in region: NRNX1-seg-161\n",
      "--- assemblying 15 probes in region: NRNX1-seg-223\n",
      "--- assemblying 15 probes in region: NRNX1-seg-636\n",
      "--- assemblying 18 probes in region: NRNX1-seg-15\n",
      "--- assemblying 13 probes in region: NRNX1-seg-214\n",
      "--- assemblying 14 probes in region: NRNX1-seg-224\n",
      "--- assemblying 16 probes in region: NRNX1-seg-313\n",
      "--- assemblying 17 probes in region: NRNX1-seg-89\n",
      "--- assemblying 17 probes in region: NRNX1-seg-334\n",
      "--- assemblying 21 probes in region: NRNX1-seg-498\n",
      "--- assemblying 13 probes in region: NRNX1-seg-619\n",
      "--- assemblying 23 probes in region: NRNX1-seg-165\n",
      "--- assemblying 14 probes in region: NRNX1-seg-197\n",
      "--- assemblying 13 probes in region: NRNX1-seg-376\n",
      "--- assemblying 15 probes in region: NRNX1-seg-620\n",
      "--- assemblying 24 probes in region: NRNX1-seg-166\n",
      "--- assemblying 13 probes in region: NRNX1-seg-239\n",
      "--- assemblying 13 probes in region: NRNX1-seg-357\n",
      "--- assemblying 19 probes in region: NRNX1-seg-470\n",
      "--- assemblying 13 probes in region: NRNX1-seg-93\n",
      "--- assemblying 15 probes in region: NRNX1-seg-209\n",
      "--- assemblying 13 probes in region: NRNX1-seg-319\n",
      "--- assemblying 17 probes in region: NRNX1-seg-201\n",
      "--- assemblying 13 probes in region: NRNX1-seg-221\n",
      "--- assemblying 14 probes in region: NRNX1-seg-679\n",
      "--- assemblying 16 probes in region: NRNX1-seg-172\n",
      "--- assemblying 20 probes in region: NRNX1-seg-192\n",
      "--- assemblying 20 probes in region: NRNX1-seg-625\n",
      "-- 449 probes assembled in total.\n",
      "-- saving 449 probes into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\candidate_probes.fasta\n",
      "-- saving readout_summary into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\readout_summary.pkl\n"
     ]
    }
   ],
   "source": [
    "# run assembly\n",
    "\n",
    "cand_sequential_probes, readout_summary = Assemble_probes(\n",
    "    library_folder, sel_sequential_pb_dict, \n",
    "    sequential_region_2_readout_dict, sequential_readout_dict, \n",
    "    primers, \n",
    "    primer_len=assemble_params['primer_len'], \n",
    "    num_readout_per_probe=assemble_params['num_readout_per_probe'], \n",
    "    unique_readout_per_probe=True,\n",
    "    save_folder=sequential_assemble_folder,\n",
    "    rc_targets=False, overwrite=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 111,
   "id": "ec06201c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7t3QuUXWV5N/BnkEIiaFAgWUACpKAfWlDwk1RREsEFpaj1UqFqvUTMwhYqSEQsN5ukASksIIpcbClR6yJ80ioSxCKxjURugaKVhVhECcQQJQ1g5BKBON/aBycmk5mzb++encn5nbWyajnv++x9fu+zz/znnTN7+vr7+/vDgwABAgQIECBAgECPCPQJwD2y0l4mAQIECBAgQIBAR0AA1ggECBAgQIAAAQI9JSAA99Rye7EECBAgQIAAAQICsB4gQIAAAQIECBDoKQEBuKeW24slQIAAAQIECBAQgPUAAQIECBAgQIBATwkIwD213F4sAQIECBAgQICAAKwHCBAgQIAAAQIEekpAAO6p5fZiCRAgQIAAAQIEBGA9QIAAAQIECBAg0FMCAnBPLbcXS4AAAQIECBAgIADrAQIECBAgQIAAgZ4SEIB7arm9WAIECBAgQIAAAQFYDxAgQIAAAQIECPSUgADcU8vtxRIgQIAAAQIECAjAeoAAAQIECBAgQKCnBATgnlpuL5YAAQIECBAgQEAA1gMECBAgQIAAAQI9JSAA99Rye7EECBAgQIAAAQICsB4gQIAAAQIECBDoKQEBuKeW24slQIAAAQIECBAQgPUAAQIECBAgQIBATwkIwD213F4sAQIECBAgQICAAKwHCBAgQIAAAQIEekpAAO6p5fZiCRAgQIAAAQIEBGA9QIAAAQIECBAg0FMCAnBPLbcXS4AAAQIECBAgIADrAQIECBAgQIAAgZ4SEIB7arm9WAIECBAgQIAAAQFYDxAgQIAAAQIECPSUgADcU8vtxRIgQIAAAQIECAjAeoAAAQIECBAgQKCnBATgnlpuL5YAAQIECBAgQEAA1gMECBAgQIAAAQI9JSAA99Rye7EECBAgQIAAAQICsB4gQIAAAQIECBDoKQEBuKeW24slQIAAAQIECBAQgPUAAQIECBAgQIBATwkIwD213F4sAQIECBAgQICAAKwHCBAgQIAAAQIEekpAAO6p5fZiCRAgQIAAAQIEBGA9QIAAAQIECBAg0FMCAnBPLbcXS4AAAQIECBAgIADrAQIECBAgQIAAgZ4SEIB7arm9WAIECBAgQIAAAQFYDxAgQIAAAQIECPSUgADcU8vtxRIgQIAAAQIECAjAeoAAAQIECBAgQKCnBATgnlpuL5YAAQIECBAgQEAA1gMECBAgQIAAAQI9JSAA99Rye7EECBAgQIAAAQICsB4gQIAAAQIECBDoKQEBuKeW24slQIAAAQIECBAQgPUAAQIECBAgQIBATwkIwD213F4sAQIECBAgQICAAKwHCBAgQIAAAQIEekpAAO6p5fZiCRAgQIAAAQIEBGA9QIAAAQIECBAg0FMCAnBPLbcXS4AAAQIECBAgIADrAQIECBAgQIAAgZ4SEIB7arm9WAIECBAgQIAAAQG4wR5Yu3Zt3H333bHzzjvH1ltv3eCRlCZAgAABAgQIbCzw3HPPxapVq2K//faLMWPG4NlAQABusB3uuOOOmDJlSoNHUJoAAQIECBAg0F1g6dKlceCBB2ISgEemB5YtWxaTJ0+OrPF22WWXkTmooxAgQIAAAQIEImLlypWdjbgHHngg9txzTyYC8Mj0wM9//vOYNGlSLF++PCZOnDgyB3UUAgQIECBAgEBEyCHDt4GPQDR4iWi8BnGVJkCAAAECBLoKyCECcCuXiMZrhd1BCRAgQIAAATvAXXvADnCDl4gA3CCu0gQIECBAgIAd4Io9IABXhCsyTQAuomQMAQIECBAg0ISAHDK8qgDcRMf9rqbGaxBXaQIECBAgQMAOcMUeEIArwhWZJgAXUTKGAAECBAgQaEJADrED3ERf5dbUeLlEBhAgQIAAAQINCcghAnBDrdW9rMZrhd1BCRAgQIAAAXeB6NoDPgLR4CUiADeIqzQBAgQIECDQVUAOsQPcyiWi8Vphd1ACBAgQIEDADrAd4EcffTROO+20uO6662LNmjXxqle9Ks4555x44xvfOCTO6tWr44QTTojrr7++8/xRRx0VF154YWy33XalLigBuBSXwQQIECBAgEBCATmkx3eADz/88HjkkUfi85//fOy8885x8cUXx+WXXx533XVX7LPPPpvoHHLIIfH000/HJZdcEo8//ngcc8wxMW3atPjSl75Uqi01XikugwkQINB7As88GfHDrz7/78lVEWNfGvGSPSIefzDiqUcjtts54lVHP/9vm+0iyozPNDesndV65dsj+voi7rnm+eMN1N/nrRE/vu735zH4uMOtzODzyTvf4Y5X9nUXOc5wrzWzrGtTtna3NSxqXeHqkEN6OADff//98bKXvSxuvvnmOOiggzoS/f398fKXvzze+973xpw5czbSufXWWzvj7r333vXh+Nvf/nYcccQRkTXSrrvuWrgFNV5hKgMJECDQewL/+5OIL789Ys2KiL6tIvp/u6nBwH9/8W4Rb/tsxMITi43fbvzzQfeJX25Quy/7Cvi7Y/zufw/UH+7/Zsf94DcidnrZpuc21Pnnne9wxxlcvWidYV26vNa6NlGydpnXMpx1xatDDunhAJzt4N5yyy3x5je/Obbddtv1EnvvvXcn1Ga7whs+zj333Jg3b148/PDD6//zM888E2PHjo0FCxbE0Uf/7jvHAs2o8QogGUKAAIFeFMh2Tj9/YMSvVw4dfDcx6Xs+0HZ2cYYIyk0ZZuHtRbtE/M0dz+9ADzxyzz/V+ebVyXu+KZgqdXPOdTjrKof63Rw5pIcD8FAv/eqrr+4E2WuvvTbe9ra3bTQk++zv7bff3vm34WP8+PFxyimnxMknn1y4FTVeYSoDCRAg0FsCd86PuO7jo+c1Z7vP/3f67893tJ3/6JF+fqd/Q+sa5y6HCMDrBbKPQmQ7v9mO8DXXXLOJzIwZM+K+++6Lm266aaPndt999zj22GPjjDPOGFYz+wW77N/AY+XKlTFlypRYvnx5TJw4sUYLm0qAAAECW5TAFX8asfy2kd3NrQqY7Uzu/vqIDz//i+Gdx2g6/6qvu415Q1nXOA8BWADuCHzjG9+I973vffG6172uc0eI7GMNgx8f+9jHYunSpUPuAJ966qlx0kknDas5a9asmD179ibPC8A1rl5TCRAgsCUKXPTaiNU/GT2vbMeXRXzszt+f72g7/9EjHTHYusa5C8ACcOezvieeeGK8613viq985SsbfR54Q57sM8AXXXRRZ9d24DHwGeCrrrqqc0u04R52gGtcpaYSIECglwRG0w6qHeCR60w7wCNm3RN/Ce7SSy+N4447rnNv3+x+vltttdWwwLfddlu8/vWvj5/85CeR/aJc9rjhhhviyCOPjIceeih22223wovjO6/CVAYSIECgtwRG22dofQZ45PrTZ4BHxHqLD8DZ53n33XffeOtb39q5r++Gj+wjENtvv32sWrUqxo0b1/lIRHaLtIMPPjjWrl0bWXB+4oknOvcBftOb3hTz588vtSgCcCkugwkQINA7Arl3URhM0dLdDtwFImFPugtEQszapbb4AHz22WfH6aefPiTUhz70ocg+tzt58uROuJ0+/fnfcM3+aMbxxx8f3/rWtzqhOPvYwwUXXBBjxowpBS4Al+IymAABAr0l4D7A1e9/nHdv3bL36u02fvC9i8vWzjvXDZ93H+ARew/Y4gPwiEkOcSABuE19xyZAgMAoEMh2gu+++vm/wPbEIxEv3PH5vwT3WPaX4FZHbD/++b8Ct99Rv/9LcEXHZy9/w7FZrewvmGUB7kfXPH+8gfr/5y0R//PN35/H4OMORzn4/PPOd7jjlX3dRY4z3GvNLOvalK3dbQ2LWldoZzlkeDQBuEJDFZ2i8YpKGUeAAAECBAikFpBDBODUPVWonsYrxGQQAQIECBAg0ICAHCIAN9BW+SU1Xr6REQQIECBAgEAzAnKIANxMZ+VU1XitsDsoAQIECBAgEBFyiADcyoWg8Vphd1ACBAgQIEBAAO7aA34JrsFLRABuEFdpAgQIECBAoKuAHGIHuJVLROO1wu6gBAgQIECAgB1gO8BtXQUCcFvyjkuAAAECBAjIIXaAW7kKNF4r7A5KgAABAgQI2AG2A9zWVSAAtyXvuAQIECBAgIAcYge4latA47XC7qAECBAgQICAHWA7wG1dBQJwW/KOS4AAAQIECMghdoBbuQo0XivsDkqAAAECBAjYAbYD3NZVIAC3Je+4BAgQIECAgBxiB7iVq0DjtcLuoAQIECBAgIAdYDvAbV0FAnBb8o5LgAABAgQIyCF2gFu5CjReK+wOSoAAAQIECNgBtgPc1lUgALcl77gECBAgQICAHGIHuJWrQOO1wu6gBAgQIECAgB1gO8BtXQUCcFvyjkuAAAECBAjIIXaAW7kKNF4r7A5KgAABAgQI2AG2A9zWVSAAtyXvuAQIECBAgIAcYge4latA47XC7qAECBAgQICAHWA7wG1dBQJwW/KOS4AAAQIECMghdoBbuQo0XivsDkqAAAECBAjYAbYD3NZVIAC3Je+4BAgQIECAgBxiB7iVq0DjtcLuoAQIECBAgIAdYDvAbV0FAnBb8o5LgAABAgQIyCF2gFu5CjReK+wOSoAAAQIECNgBtgPc1lUgALcl77gECBAgQICAHGIHuJWrQOO1wu6gBAgQIECAgB1gO8BtXQUCcFvyjkuAAAECBAjIIXaAW7kKNF4r7A5KgAABAgQI2AG2A9zWVSAAtyXvuAQIECBAgIAcYge4latA47XC7qAECBAgQICAHWA7wG1dBQJwW/KOS4AAAQIECMghdoBbuQo0XivsDkqAAAECBAjYAbYD3NZVIAC3Je+4BAgQIECAgBxiB7iVq0DjtcLuoAQIECBAgIAdYDvAbV0FAnBb8o5LgAABAgQIyCF2gFu5CjReK+wOSoAAAQIECNgBtgPc1lUgALcl77gECBAgQICAHGIHuJWrQOO1wu6gBAgQIECAgB1gO8BtXQUCcFvyjkuAAAECBAjIIXaAW7kKNF4r7A5KgAABAgQI2AG2A9zWVSAAtyXvuAQIECBAgIAcYge4latA47XC7qAECBAgQICAHWA7wG1dBQJwW/KOS4AAAQIECMghdoBbuQo0XivsDkqAAAECBAjYAbYD3NZVIAC3Je+4BAgQIECAgBxiB7iVq0DjtcLuoAQIECBAgIAdYDvAbV0FAnBb8o5LgAABAgQIyCF2gFu5CjReK+wOSoAAAQIECNgBtgPc1lUgALcl77gECBAgQICAHGIHuJWrQOO1wu6gBAgQIECAgB1gO8BtXQUCcFvyjkuAAAECBAjIIXaAW7kKNF4r7A5KgAABAgQI2AG2A9zWVSAAtyXvuAQIECBAgIAcYge4latA47XC7qAECBAgQICAHWA7wG1dBQJwW/KOS4AAAQIECMghdoBbuQo0XivsDkqAAAECBAjYAbYD3NZVIAC3Je+4BAgQIECAgBxiB3gjgblz58aiRYti8eLFw8r84he/iJNOOiluvPHGzphDDz00Lrjggpg4cWLhK0rjFaYykAABAgQIEEgsIIcIwOsF5s2bFzNnzoypU6d2DcDTpk2LdevWxUUXXdSZe9xxx8Wzzz4bd955Z+H21HiFqQwkQIAAAQIEEgvIIQJwrFixImbMmBFLliyJSZMmxYQJE4YNwI8//ni85CUviWuvvTbe9ra3dfSy//32t789/vd//zd23HHHQi2q8QoxGUSAAAECBAg0ICCHCMCxcOHCWLBgQWQff5gzZ04sW7Zs2AC8du3a2GmnneKII46IK664Ivr6+uKYY46JH/7wh3HvvffGVlttVahNNV4hJoMIECBAgACBBgTkEAF4I4Hp06d3DcDZ4K9+9atx/PHHx+rVqzsBeJdddonvfve7sddeexVuUY1XmMpAAgQIECBAILGAHCIAlwrA/f39ceaZZ8bdd98dp5xySjz33HNx+umnx5o1a+Lmm2+OF73oRUOKZs9n/wYeK1eujClTpsTy5ctL/fJc4v5XjgABAgQIEOhBAQFYAC4VgLOPSmS/9PbQQw+tD7uPPfZY7LHHHvH3f//3ceKJJw4pOmvWrJg9e/YmzwnAPfiu4yUTIECAAIGWBQRgAbhUAM4++vBf//Vfcdttt20078ADD+zs6F588cV2gFu+qB2eAAECBAgQ6C4gAAvApQLwOeecE+edd17nzhFjxozpzH3qqac6d4/IPhrx8Y9/vNA1p/EKMRlEgAABAgQINCAghwjAXQNwdr/fVatWxbhx42Ls2LGRfXZ3v/32ize84Q2djzwMfCY42xW+5557YocddijUphqvEJNBBAgQIECAQAMCcogA3DUAZ7dEmzx5csyfPz+yO0Rkj+x2Z5/61Kfilltu6dz27OCDD47zzz8/9txzz8ItqvEKUxlIgAABAgQIJBaQQwTgxC1VrJzGK+ZkFAECBAgQIJBeQA4RgNN3VYGKGq8AkiEECBAgQIBAIwJyiADcSGPlFdV4eUKeJ0CAAAECBJoSkEME4KZ6q2tdjdcKu4MSIECAAAECESGHCMCtXAgarxV2ByVAgAABAgQE4K490Nef3ePLoxEBAbgRVkUJECBAgACBAgJyiB3gAm2SfojGS2+qIgECBAgQIFBMQA4RgIt1SuJRGi8xqHIECBAgQIBAYQE5RAAu3CwpB2q8lJpqESBAgAABAmUE5BABuEy/JBur8ZJRKkSAAAECBAiUFJBDBOCSLZNmuMZL46gKAQIECBAgUF5ADhGAy3dNghkaLwGiEgQIECBAgEAlATlEAK7UOHUnaby6guYTIECAAAECVQXkEAG4au/UmqfxavGZTIAAAQIECNQQkEME4BrtU32qxqtuZyYBAgQIECBQT0AOEYDrdVDF2RqvIpxpBAgQIECAQG0BOUQArt1EVQpovCpq5hAgQIAAAQIpBOQQAThFH5WuofFKk5lAgAABAgQIJBKQQwTgRK1UrozGK+dlNAECBAgQIJBOQA4RgNN1U4lKGq8ElqEECBAgQIBAUgE5RABO2lBFi2m8olLGESBAgAABAqkF5BABOHVPFaqn8QoxGUSAAAECBAg0ICCHCMANtFV+SY2Xb2QEAQIECBAg0IyAHCIAN9NZOVU1XivsDkqAAAECBAhEhBwiALdyIWi8VtgdlAABAgQIEBCAu/ZAX39/f78uaUZAAG7GVVUCBAgQIEAgX0AOsQOc3yUNjNB4DaAqSYAAAQIECBQSkEME4EKNknqQxkstqh4BAgQIECBQVEAOEYCL9krScRovKadiBAgQIECAQAkBOUQALtEu6YZqvHSWKhEgQIAAAQLlBOQQAbhcxyQarfESQSpDgAABAgQIlBaQQwTg0k2TYoLGS6GoBgECBAgQIFBFQA4RgKv0Te05Gq82oQIECBAgQIBARQE5RACu2Dr1pmm8en5mEyBAgAABAtUF5BABuHr31Jip8WrgmUqAAAECBAjUEpBDBOBaDVR1ssarKmceAQIECBAgUFdADhGA6/ZQpfkarxKbSQQIECBAgEACATlEAE7QRuVLaLzyZmYQIECAAAECaQTkEAE4TSeVrKLxSoIZToAAAQIECCQTkEME4GTNVKaQxiujZSwBAgQIECCQUkAOEYBT9lPhWhqvMJWBBAgQIECAQGIBOUQATtxSxcppvGJORhEgQIAAAQLpBeQQATh9VxWoqPEKIBlCgAABAgQINCIghwjAjTRWXlGNlyfkeQIECBAgQKApATlEAG6qt7rW1XitsDsoAQIECBAgEBFyiADcyoWg8Vphd1ACBAgQIEBAAO7aA339/f39uqQZAQG4GVdVCRAgQIAAgXwBOcQOcH6XNDBC4zWAqiQBAgQIECBQSEAOEYALNUrqQRovtah6BAgQIECAQFEBOUQALtorScdpvKScihEgQIAAAQIlBOQQAbhEu6QbqvHSWapEgAABAgQIlBOQQwTgch2TaLTGSwSpDAECBAgQIFBaQA4RgEs3TYoJGi+FohoECBAgQIBAFQE5RACu0je152i82oQKECBAgAABAhUF5BABuGLr1Jum8er5mU2AAAECBAhUF5BDBODq3VNjpsargWcqAQIECBAgUEtADhGAazVQ1ckar6qceQQIECBAgEBdATlEAK7bQ5Xma7xKbCYRIECAAAECCQTkEAE4QRuVL6HxypuZQYAAAQIECKQRkEME4DSdVLKKxisJZjgBAgQIECCQTEAOEYCTNVOZQhqvjJaxBAgQIECAQEoBOUQATtlPhWtpvMJUBhIgQIAAAQKJBeQQAThxSxUrp/GKORlFgAABAgQIpBeQQwTgjQTmzp0bixYtisWLFw8r8+yzz8anP/3p+PKXvxyPP/54vPa1r43Pfvazsf/++xfuUI1XmMpAAgQIECBAILGAHCIArxeYN29ezJw5M6ZOndo1AM+YMSOuvfbamD9/fuy1115x2mmnxa233ho//vGPY9y4cYVaVOMVYjKIAAECBAgQaEBADhGAY8WKFZGF2iVLlsSkSZNiwoQJwwbgBx54IP7wD/8wrrvuunjLW97S0ct2gQ844ID453/+5zj00EMLtanGK8RkEAECBAgQINCAgBwiAMfChQtjwYIFkX38Yc6cObFs2bJhA/Bll10Wp556aqxevTq22mqryi2p8SrTmUiAAAECBAjUFJBDBOCNBKZPn941AGcfkfjud7/bCcGf+cxnOrvHr3nNa+L888+PV7ziFYXbUeMVpjKQAAECBAgQSCwghwjApQJw9lGJf/3Xf4099tgjzjvvvNhhhx3irLPOiptvvjl+9KMfxfjx44cUXbNmTWT/Bh4rV66MKVOmxPLly2PixImJ21o5AgQIECBAgMDwAgKwAFwqAB933HFx6aWXdsLuwI7v008/3fns8Kc+9an45Cc/OaTorFmzYvbs2Zs8JwB7eyJAgAABAgRGWkAAFoBLBeCzzz47/u7v/i6yW6Ft+Mh2c7PboV1yySV2gEf6KnY8AgQIECBAoJSAACwAlwrA3/ve9+Lggw+OO+64oxN4s0e2A5x9jOH000/v3EatyEPjFVEyhgABAgQIEGhCQA4RgLsG4HXr1sWqVas69/cdO3ZsZ+xhhx3W+eW3L3zhC7Hjjjt2doRvuummuOeee2KnnXYq1KcarxCTQQQIECBAgEADAnKIANw1AGe3RJs8eXLnj15kd4jIHr/+9a87n/e9+uqr46mnnoo3vOENkf0RjVe+8pWFW1TjFaYykAABAgQIEEgsIIcIwIlbqlg5jVfMySgCBAgQIEAgvYAcIgCn76oCFTVeASRDCBAgQIAAgUYE5BABuJHGyiuq8fKEPE+AAAECBAg0JSCHCMBN9VbXuhqvFXYHJUCAAAECBCJCDhGAW7kQNF4r7A5KgAABAgQICMBde6Cvv7+/X5c0IyAAN+OqKgECBAgQIJAvIIfYAc7vkgZGaLwGUJUkQIAAAQIECgnIIQJwoUZJPUjjpRZVjwABAgQIECgqIIcIwEV7Jek4jZeUUzECBAgQIECghIAcIgCXaJd0QzVeOkuVCBAgQIAAgXICcogAXK5jEo3WeIkglSFAgAABAgRKC8ghAnDppkkxQeOlUFSDAAECBAgQqCIghwjAVfqm9hyNV5tQAQIECBAgQKCigBwiAFdsnXrTNF49P7MJECBAgACB6gJyiABcvXtqzNR4NfBMJUCAAAECBGoJyCECcK0GqjpZ41WVM48AAQIECBCoKyCHCMB1e6jSfI1Xic0kAgQIECBAIIGAHCIAJ2ij8iU0XnkzMwgQIECAAIE0AnKIAJymk0pW0XglwQwnQIAAAQIEkgnIIQJwsmYqU0jjldEylgABAgQIEEgpIIcIwCn7qXAtjVeYykACBAgQIEAgsYAcIgAnbqli5TReMSejCBAgQIAAgfQCcogAnL6rClTUeAWQDCFAgAABAgQaEZBDBOBGGiuvqMbLE/I8AQIECBAg0JSAHCIAN9VbXetqvFbYHZQAAQIECBCICDlEAG7lQtB4rbA7KAECBAgQICAAd+2Bvv7+/n5d0oyAANyMq6oECBAgQIBAvoAcYgc4v0saGKHxGkBVkgABAgQIECgkIIcIwIUaJfUgjZdaVD0CBAgQIECgqIAcIgAX7ZWk4zReUk7FCBAgQIAAgRICcogAXKJd0g3VeOksVSJAgAABAgTKCcghAnC5jkk0WuMlglSGAAECBAgQKC0ghwjApZsmxQSNl0JRDQIECBAgQKCKgBwiAFfpm9pzNF5tQgUIECBAgACBigJyiABcsXXqTdN49fzMJkCAAAECBKoLyCECcPXuqTFT49XAM5UAAQIECBCoJSCHCMC1GqjqZI1XVc48AgQIECBAoK6AHCIA1+2hSvM1XiU2kwgQIECAAIEEAnKIAJygjcqX0HjlzcwgQIAAAQIE0gjIIQJwmk4qWUXjlQQznAABAgQIEEgmIIcIwMmaqUwhjVdGy1gCBAgQIEAgpYAcIgCn7KfCtTReYSoDCRAgQIAAgcQCcogAnLilipXTeMWcjCJAgAABAgTSC8ghAnD6ripQUeMVQDKEAAECBAgQaERADhGAG2msvKIaL0/I8wQIECBAgEBTAnKIANxUb3Wtq/FaYXdQAgQIECBAICLkEAG4lQtB47XC7qAECBAgQICAANy1B/r6+/v7dUkzAgJwM66qEiBAgAABAvkCcogd4PwuaWCExmsAVUkCBAgQIECgkIAcIgAXapTUgzRealH1CBAgQIAAgaICcogAXLRXko7TeEk5FSNAgAABAgRKCMghAnCJdkk3VOOls1SJAAECBAgQKCcghwjA5Tom0WiNlwhSGQIECBAgQKC0gBwiAJdumhQTNF4KRTUIECBAgACBKgJyiABcpW9qz9F4tQkVIECAAAECBCoKyCECcMXWqTdN49XzM5sAAQIECBCoLiCHCMDVu6fGTI1XA89UAgQIECBAoJaAHCIA12qgqpM1XlU58wgQIECAAIGLD9fZAAAgAElEQVS6AnKIAFy3hyrN13iV2EwiQIAAAQIEEgjIIQJwgjYqX0LjlTczgwABAgQIEEgjIIcIwGk6qWQVjVcSzHACBAgQIEAgmYAcIgAna6YyhTReGS1jCRAgQIAAgZQCcogAnLKfCtfSeIWpDCRAgAABAgQSC8ghAvBGAnPnzo1FixbF4sWLC7XalVdeGX/5l38ZDzzwQOy5556F5mSDNF5hKgMJECBAgACBxAJyiAC8XmDevHkxc+bMmDp1aqEA/OCDD8arX/3q+NWvfiUAJ74wlSNAgAABAgSaExCABeBYsWJFzJgxI5YsWRKTJk2KCRMm5Abg3/72tzFt2rTYZptt4j/+4z8E4OauUZUJECBAgACBxAICsAAcCxcujAULFkT28Yc5c+bEsmXLcgNwNjabd9ZZZ8Vhhx0mACe+MJUjQIAAAQIEmhMQgAXgjQSmT5+eG4CXLl0ahx9+eNxxxx2d3eNDDjlEAG7uGlWZAAECBAgQSCwgAAvApQLwk08+Gfvvv3+cfPLJ8dGPfrSzU1wkAK9ZsyayfwOPlStXxpQpU2L58uUxceLExG2tHAECBAgQIEBgeAEBWAAuFYA/8pGPxC9+8Yv45je/2ZlXNADPmjUrZs+evYm2AOztiQABAgQIEBhpAQFYAC4VgPv6+mLbbbeNrbfeujNv3bp1sXbt2njhC18YH/jAB+Kyyy4bUtQO8Ehf2o5HgAABAgQIDCcgAAvApQLw/fffv9H422+/Pd7//vd3doJf8YpXxPjx4wtdbRqvEJNBBAgQIECAQAMCcogA3DUAZzu8q1atinHjxsXYsWM30Sr6EYjBEzVeA1ezkgQIECBAgEAhATlEAO4agLNbok2ePDnmz58f2R0iBj8E4ELXmUEECBAgQIDAZiQgAAvArbSjxmuF3UEJECBAgACBiJBDBOBWLgSN1wq7gxIgQIAAAQICcNce6Ovv7+/XJc0ICMDNuKpKgAABAgQI5AvIIXaA87ukgREarwFUJQkQIECAAIFCAnKIAFyoUVIP0nipRdUjQIAAAQIEigrIIQJw0V5JOk7jJeVUjAABAgQIECghIIcIwCXaJd1QjZfOUiUCBAgQIECgnIAcIgCX65hEozVeIkhlCBAgQIAAgdICcogAXLppUkzQeCkU1SBAgAABAgSqCMghAnCVvqk9R+PVJlSAAAECBAgQqCgghwjAFVun3jSNV8/PbAIECBAgQKC6gBwiAFfvnhozNV4NPFMJECBAgACBWgJyiABcq4GqTtZ4VeXMI0CAAAECBOoKyCECcN0eqjRf41ViM4kAAQIECBBIICCHCMAJ2qh8CY1X3swMAgQIECBAII2AHCIAp+mkklU0XkkwwwkQIECAAIFkAnKIAJysmcoU0nhltIwlQIAAAQIEUgrIIQJwyn4qXEvjFaYykAABAgQIEEgsIIcIwIlbqlg5jVfMySgCBAgQIEAgvYAcIgCn76oCFTVeASRDCBAgQIAAgUYE5BABuJHGyiuq8fKEPE+AAAECBAg0JSCHCMBN9VbXuhqvFXYHJUCAAAECBCJCDhGAW7kQNF4r7A5KgAABAgQICMBde6Cvv7+/X5c0IyAAN+OqKgECBAgQIJAvIIfYAc7vkgZGaLwGUJUkQIAAAQIECgnIIQJwoUZJPUjjpRZVjwABAgQIECgqIIcIwEV7Jek4jZeUUzECBAgQIECghIAcIgCXaJd0QzVeOkuVCBAgQIAAgXICcogAXK5jEo3WeIkglSFAgAABAgRKC8ghAnDppkkxQeOlUFSDAAECBAgQqCIghwjAVfqm9hyNV5tQAQIECBAgQKCigBwiAFdsnXrTNF49P7MJECBAgACB6gJyiABcvXtqzNR4NfBMJUCAAAECBGoJyCECcK0GqjpZ41WVM48AAQIECBCoKyCHCMB1e6jSfI1Xic0kAgQIECBAIIGAHCIAJ2ij8iU0XnkzMwgQIECAAIE0AnKIAJymk0pW0XglwQwnQIAAAQIEkgnIIQJwsmYqU0jjldEylgABAgQIEEgpIIcIwCn7qXAtjVeYykACBAgQIEAgsYAcIgAnbqli5TReMSejCBAgQIAAgfQCcogAnL6rClTUeAWQDCFAgAABAgQaEZBDBOBGGiuvqMbLE/I8AQIECBAg0JSAHCIAN9VbXetqvFbYHZQAAQIECBCICDlEAG7lQtB4rbA7KAECBAgQICAAd+2Bvv7+/n5d0oyAANyMq6oECBAgQIBAvoAcYgc4v0saGKHxGkBVkgABAgQIECgkIIcIwIUaJfUgjZdaVD0CBAgQIECgqIAcIgAX7ZWk4zReUk7FCBAgQIAAgRICcogAXKJd0g3VeOksVSJAgAABAgTKCcghAnC5jkk0WuMlglSGAAECBAgQKC0ghwjApZsmxQSNl0JRDQIECBAgQKCKgBwiAFfpm9pzNF5tQgUIECBAgACBigJyiABcsXXqTdN49fzMJkCAAAECBKoLyCECcPXuqTFT49XAM5UAAQIECBCoJSCHCMC1GqjqZI1XVc48AgQIECBAoK6AHCIA1+2hSvM1XiU2kwgQIECAAIEEAnKIAJygjcqX0HjlzcwgQIAAAQIE0gjIIQJwmk4qWUXjlQQznAABAgQIEEgmIIcIwMmaqUwhjVdGy1gCBAgQIEAgpYAcIgCn7KfCtTReYSoDCRAgQIAAgcQCcogAnLilipXTeMWcjCJAgAABAgTSC8ghAnD6ripQUeMVQDKEAAECBAgQaERADhGAG2msvKIaL0/I8wQIECBAgEBTAnKIALyRwNy5c2PRokWxePHiYWXuueeeOOWUU+K2226LF7zgBTFt2rQ4//zzY/fddy/cpxqvMJWBBAgQIECAQGIBOUQAXi8wb968mDlzZkydOnXYALx69erYd999O6H3zDPPjLVr18bJJ58cv/zlL+Ouu+6KMWPGFGpRjVeIySACBAgQIECgAQE5RACOFStWxIwZM2LJkiUxadKkmDBhwrAB+PLLL49PfOITncA7EHYHmug73/lOHHrooYXaVOMVYjKIAAECBAgQaEBADhGAY+HChbFgwYLIPv4wZ86cWLZs2bABOHvugQceiEMOOWS93EATXX311fHud7+7UJtqvEJMBhEgQIAAAQINCMghAvBGAtOnT+8agIfiOv744+OKK66IBx98MMaPH1+oTTVeISaDCBAgQIAAgQYE5BABuFYA/tznPhcnnnhiXHjhhfHxj398WM01a9ZE9m/gsXLlypgyZUosX748Jk6c2EBrK0mAAAECBAgQGFpAABaAKwXg/v7++PSnP9352MSpp54aZ599dtdrbNasWTF79uxNxgjA3poIECBAgACBkRYQgAXg0gH42WefjQ9/+MNx5ZVXxrnnntu5C0Teww5wnpDnCRAgQIAAgZESEIAF4NIB+D3veU987Wtfiy9/+cuR/e8qD41XRc0cAgQIECBAIIWAHCIAdw3A69ati1WrVsW4ceNi7Nix8cUvfrGz+3veeefF+9///o3mDowp0pgar4iSMQQIECBAgEATAnKIANw1AGe3PZs8eXLMnz8/sjtEHH744XHjjTcOqTYwpkijarwiSsYQIECAAAECTQjIIQJwE32VW1Pj5RIZQIAAAQIECDQkIIcIwA21VveyGq8VdgclQIAAAQIEIkIOEYBbuRA0XivsDkqAAAECBAgIwF17oK8/u9mtRyMCAnAjrIoSIECAAAECBQTkEDvABdok/RCNl95URQIECBAgQKCYgBwiABfrlMSjNF5iUOUIECBAgACBwgJyiABcuFlSDtR4KTXVIkCAAAECBMoIyCECcJl+STZW4yWjVIgAAQIECBAoKSCHCMAlWybNcI2XxlEVAgQIECBAoLyAHCIAl++aBDM0XgJEJQgQIECAAIFKAnKIAFypcepO0nh1Bc0nQIAAAQIEqgrIIQJw1d6pNU/j1eIzmQABAgQIEKghIIcIwDXap/pUjVfdzkwCBAgQIECgnoAcIgDX66CKszVeRTjTCBAgQIAAgdoCcogAXLuJqhTQeFXUzCFAgAABAgRSCMghAnCKPipdQ+OVJjOBAAECBAgQSCQghwjAiVqpXBmNV87LaAIECBAgQCCdgBwiAKfrphKVNF4JLEMJECBAgACBpAJyiACctKGKFtN4RaWMI0CAAAECBFILyCECcOqeKlRP4xViMogAAQIECBBoQEAOEYAbaKv8khov38gIAgQIECBAoBkBOUQAbqazcqpqvFbYHZQAAQIECBCICDlEAG7lQtB4rbA7KAECBAgQICAAd+2Bvv7+/n5d0oyAANyMq6oECBAgQIBAvoAcYgc4v0saGKHxGkBVkgABAgQIECgkIIcIwIUaJfUgjZdaVD0CBAgQIECgqIAcIgAX7ZWk4zReUk7FCBAgQIAAgRICcogAXKJd0g3VeOksVSJAgAABAgTKCcghAnC5jkk0WuMlglSGAAECBAgQKC0ghwjApZsmxQSNl0JRDQIECBAgQKCKgBwiAFfpm9pzNF5tQgUIECBAgACBigJyiABcsXXqTdN49fzMJkCAAAECBKoLyCECcPXuqTFT49XAM5UAAQIECBCoJSCHCMC1GqjqZI1XVc48AgQIECBAoK6AHCIA1+2hSvM1XiU2kwgQIECAAIEEAnKIAJygjcqX0HjlzcwgQIAAAQIE0gjIIQJwmk4qWUXjlQQznAABAgQIEEgmIIcIwMmaqUwhjVdGy1gCBAgQIEAgpYAcIgCn7KfCtTReYSoDCRAgQIAAgcQCcogAnLilipXTeMWcjCJAgAABAgTSC8ghAnD6ripQUeMVQDKEAAECBAgQaERADhGAG2msvKIaL0/I8wQIECBAgEBTAnKIANxUb3Wtq/FaYXdQAgQIECBAICLkEAG4lQtB47XC7qAECBAgQICAANy1B/r6+/v7dUkzAgJwM66qEiBAgAABAvkCcogd4PwuaWCExmsAVUkCBAgQIECgkIAcIgAXapTUgzRealH1CBAgQIAAgaICcogAXLRXko7TeEk5FSNAgAABAgRKCMghAnCJdkk3VOOls1SJAAECBAgQKCcghwjA5Tom0WiNlwhSGQIECBAgQKC0gBwiAJdumhQTNF4KRTUIECBAgACBKgJyiABcpW9qz9F4tQkVIECAAAECBCoKyCECcMXWqTdN49XzM5sAAQIECBCoLiCHCMDVu6fGTI1XA89UAgQIECBAoJaAHCIA12qgqpM1XlU58wgQIECAAIG6AnKIAFy3hyrN13iV2EwiQIAAAQIEEgjIIQJwgjYqX0LjlTczgwABAgQIEEgjIIcIwGk6qWQVjVcSzHACBAgQIEAgmYAcIgAna6YyhTReGS1jCRAgQIAAgZQCcogAnLKfCtfSeIWpDCRAgAABAgQSC8ghAnDilipWTuMVczKKAAECBAgQSC8ghwjA6buqQEWNVwDJEAIECBAgQKARATlEAG6ksfKKarw8Ic8TIECAAAECTQnIIQLwRgJz586NRYsWxeLFi4eVWb16dZxwwglx/fXXd8YcddRRceGFF8Z2221XuE+bbLynnnkurvn+w/H17/88Vj/5TOy43Tbxlv12jejrj2/+cOX6//bOAybGOw7YNV64zdYx1JwNn89e2OAxO4z9g9j9pdvF8seejMeeerZznIE52fjB55A9d/gfjY9v3/NI13MrW7fbaxvqPKpaDD73oc5zqNqD/1u311dkLfLWqsj6D3cOQ61PkT4pYpPXG1kvDtc3Rfq07Fp3O1YTz9VZ27x1GXjjyeuNoa7jbu8Pecct8p5Q5vos25dF+i7veh/Orsy55B0jz6noscq+d6R6L83rg+G+8OX143DPF1nXIu9L3d43yvRl2WukbO0yr6VwyCgwsMkcUuDwm/WQvv7+/v7N+gwTn9y8efNi5syZMXXq1K4B+JBDDomnn346Lrnkknj88cfjmGOOiWnTpsWXvvSlwmfUVOP9dNUT8f7Lb4+Vv1obfX0Rg1ewLyKyRR14bpdxY+Iz79ovTv3a3RvN2fD5r8z4487r6lY3e35gzs7bb9P5f1b9+jfr/9vAc1v1Rfy2//djNwQbOLfBiN3qDjW/6vg8i27nvsk5/865zOvLO37V58ucw+DXWLRP8myK9Ea3vsl77UPNLdsbdc+xzvnnvb5u65Jdn3vtvH0Mde2XvY4Hvz+U7Ydub4CDa+et2YbvKcOdR17fdeuBzLyb3XDvQ0Xew/Kum7wvFEXeLzf0Ga5/Ur2XFum/wcfK68duX3fy1jXPN+/5su8N3caX6Ye67xED/ZrXP0WfbyqHFD3+5jyuZwLwihUrYsaMGbFkyZKYNGlSTJgwYdgAfOutt8ZBBx0U9957b+yzzz6d9fv2t78dRxxxRGTNtOuuz+9i5T2aaLzsu+k3n//d+OWatZ2QWeSRfVHK3iyyx1Bzsot75xdtG9mQR379m8J1ixx7cxuTZ9H0+eYdv+7zdc4/79h1aheZ2/bxi5xjtzF555/3/HC1s+tzwovHxMKPvSHedtHNw177qa/jqudb1zHl/KJ2dY45Uk4jdZzBFgOG3/nEtM5PEgceeV+LUp1vXp285+usbeq5eec6nHWd82gih9Q5n81pbs8E4IULF8aCBQsi+/jDnDlzYtmyZcMG4HPPPTeyneKHH354/Vo988wzMXbs2E6No48+utAaNtF4V97+UJz29bsLHd8gAgS2HIF3HrBbfP37K7acFzSCr4RdfexsN/e9U3ZfX8jXovqmw1UYbF3nSE3kkDrnsznN7ZkAvCH69OnTuwbg7LO/t99+e+ffho/x48fHKaecEieffPKQa7hmzZrI/g08Vq5cGVOmTInly5fHxIkTk6z7UZfdEnc++NgmH3tIUlwRAgQ2S4HsJzjbb7N1PPHMc679kivEriTYEMMzwyl7vjT+30dfv/5ZX4vquw5VYSjrOkcSgIfXE4CHsMk+KnHffffFTTfdtNGzu+++exx77LFxxhlnDCk6a9asmD179ibPpQzAh56/OH626sk614O5BAiMQoE/eEFfPLuu4OeeRuHra/KU2dXX/cOdt4v/+MSb1hfytai+6XAVBlvXOZIALABvJJC3A/yxj30sli5dOuQO8KmnnhonnXTSkKJ2gOtcpuYSIDCcgF3M6r3BrrrdwEw7wPUNi1awA1xUqv44O8BDGGafAb7ooos6H10YeAx8Bviqq67q3BKtyKOJ77x87qqIvDEEtjwBn2OtvqbsqtsNzPQZ4PqGRSv4DHBRqXrjBOAh/G677bZ4/etfHz/5yU9i77337oy44YYb4sgjj4yHHnoodtttt0LqTQTgvN+8HerEivzmqbtAFFrS2oPy1qLu83VOMO/YdWoXmdv28YucY7cxeeef9/xwtYveycBdIDYVLGpXZ+2rrmvZY47UcQafl7tAlF2p4cfnraG7QKSzLlJJAI6IdevWxapVq2LcuHGdOz1kt0Y++OCDY+3atXHppZfGE0880bkP8Jve9KaYP39+EdfOmCYCcFY39z7Av7s3cN59EsvePzQ7dpH7qHa9Z+IQ9y3Oq7sh+MDxC98HuKRF3r0phzqXvP+24evLuxds1efLnEPZ+76WvV9pt/tg1rlHZt49ZYv0RpH+ber889a2yH1Y8+67WuR+3oOdyvZD128CBl1veWu24bWR5D7AQ1zvye8DXPA9Je8LRdnrKvl9gAe9jiL9N/g15fXjZnMf4JJ9mXeNlP2aVPZrcXbP71SPpnJIqvNrs44AHNG5I8TkyZM74Tb7fHD2eOSRR+L444+Pb33rW51QnH3s4YILLogxY8YUXq8mGy/bCf7GDx6Oa76/IlY98ZvYeftt48j9dun8aYbr7/7F+v/2jgN2i7fv//u/BDd4zobPZy9scN2XvnCbmPTSF8byR5+KR596pnOcgTnZ+KHqHfbK8XHjjx7pem5l63Z7bUOdR1WLwec+1HkOVXvwf+v2+gb+Wli3tRhqffPWsug5DLU+ebWz54vY5PVG1ovD9U2RPi271t2O1cRzddY2b10G3njyemOo67jb+0PecYu8J5S5Poe7NoY7jyJ9l3e9D2dX5lzyjpHnVPRYZd87Ur2X5vXBcF/48vpxuOeLrGuR96Vu7xtl+rLsNVK2dpnXUjhkFBjYZA4pcPjNekhPBuCRWhGNN1LSjkOAAAECBAgMFpBDhu8JAbjB60XjNYirNAECBAgQINBVQA4RgFu5RDReK+wOSoAAAQIECDT4u0hbAq4d4AZXUQBuEFdpAgQIECBAwA5wxR4QgCvCFZkmABdRMoYAAQIECBBoQkAOGV5VAG6i435XU+M1iKs0AQIECBAgYAe4Yg8IwBXhikwTgIsoGUOAAAECBAg0ISCH2AFuoq9ya2q8XCIDCBAgQIAAgYYE5BABuKHW6l5W47XC7qAECBAgQICAu0B07QEfgWjwEhGAG8RVmgABAgQIEOgqIIfYAW7lEtF4rbA7KAECBAgQIGAH2A5wW1eBANyWvOMSIECAAAECcogd4FaugmXLlsXkyZNj6dKlscsuu7RyDg5KgAABAgQI9KbAypUrY8qUKfHAAw/Ennvu2ZsIw7xqnwFusB3uuOOOTuN5ECBAgAABAgTaEsg24g488MC2Dr9ZHlcAbnBZ1q5dG3fffXfsvPPOsfXWWyc90sB3dXaXk7LWKmZNavEln2w9kpPWLmhNahMmLWA9knImKZZ6TZ577rlYtWpV7LfffjFmzJgk57ilFBGAR+lK+lzP5rdw1mTzWhPrsXmtR3Y21mTzWhPrsXmth2tkZNdDAB5Z72RH88aVjDJZIWuSjDJJIeuRhDFpEWuSlLN2MetRmzB5AWuSnHTYggLwyFknPZKLJClnkmLWJAljsiLWIxllskLWJBllkkLWIwlj0iLWJCln12IC8MhZJz3SmjVr4oILLoiZM2fGi1/84qS1FasmYE2quTU1y3o0JVu9rjWpbtfETOvRhGq9mtaknl+Z2QJwGS1jCRAgQIAAAQIERr2AADzql9ALIECAAAECBAgQKCMgAJfRMpYAAQIECBAgQGDUCwjAo34JvQACBAgQIECAAIEyAgJwGS1jCRAgQIAAAQIERr2AADwKl/C3v/1tzJ49Oy6//PJ47LHH4o1vfGNccsklsffee4/CV7N5n/LcuXNj0aJFsXjx4vUn+oMf/CBOPPHEuPPOO2PHHXeME044IU4++eT1zxdZn7wam7fKyJ7do48+Gqeddlpcd911kf2G9Kte9ao455xzOn2fPfIsrUfa9XrkkUfiE5/4RPz7v/97PP300zFt2rQ477zz4pWvfKX1SEtdqdp9990Xr3nNa+Lzn/98TJ8+3ZpUUqw/6cEHH4w999xzk0L/9E//FDNmzPC+VZ+4dgUBuDbhyBfIwm8WeOfPnx+77bZbnHLKKfGzn/0s7rnnnthmm21G/oS20CPOmzevc5u5qVOnrg/Aq1evjn322Sfe8Y53dJ677bbb4rjjjuusx4c//OGORN76FKmxhZJWelmHH354ZKEr+4Ke/Vnxiy++uPPN31133dX5/61HJdbKk173utfFVlttFZ/73Odiu+22izPPPDNuueWWuP/++zuB2HpUpq098dlnn42DDjqo88159vUhC8BF3m+8Z9Wm36TAwoUL4+ijj+58be7r61v//Lhx4+Kpp55ynaQnL11RAC5N1u6EZ555Jnbaaac499xz46/+6q86J/P444/HrrvuGldccUW85z3vafcEt4Cjr1ixovMd+pIlS2LSpEkxYcKE9QH4M5/5TCeALVu2LLbeeuvOq812J7/2ta/Fj3/84yiyPnk1tgDCZC8hC1Uve9nL4uabb+58Yc8e/f398fKXvzze+973xtixY61HMu38QlmY+pu/+Zs444wz4o/+6I86E/77v/879t9//1i6dGnnpyWuj3zHpkZk70XZN+X/+Z//uT4A573feM9qZjXOPvvs+OpXv9rZ6R38sCbNmJetKgCXFWt5fPZF5o//+I/jf/7nfzohYOCR/Tg4+9FwthPpUU8g+859wYIFkX38Yc6cOZ2wO/ARiCOPPDJ22GGHuPLKK9cfJPuif9hhh8Uvf/nLzti89cmrMX78+HovYAuanX1zl+0uvvnNb45tt912/SvLPu5zxBFHdHZXrEd7C571fPbxn+z6yL4BPOqoo6xHS8tx0003xTvf+c5O4Np9993XB+C89xvvWc0sWLYZlf2kZMOvFQNHsibNmJetKgCXFWt5fLbT+Od//uedH6Fku18Dj+xHLdl/yz4n6ZFOIPsR4oYBOPsm40//9E/jH/7hH9Yf5Ec/+lFnN+yOO+6Ihx56KHd98mq89rWvTfcCtsBKV199dedHi9dee22cfvrp1qOlNT722GMj+zxj9o1JthbZR1Xyetv10cxiZd8ovvrVr45sZ/F973tf50fuAx+BsCbNmOdVzb4m7LLLLvGb3/wmss9lZz/Jyj4u9Cd/8ieukzy8EXpeAB4h6FSH+cpXvhIf+MAHYt26dZ3vLgceH/zgB+Phhx/u/AjSI53A4ACc7TxmX2CyneGBR7YLuddee3U+MpGF5bz1yasx8Mtd6V7FllMp+yhEtvOb7Qhfc801nV/8tB7trG/2jV/2md9LL720s8v1ve99r/ONifUY+fXIPg6UfTToqquu6hx8wwDsGhn59cg+VpJ9Pv6AAw7o/ILo9ttvH9nX7s9+9rNx4403xkc/+lHXycgvyyZHFIA3g0Uocwr/9m//Fu9+97uH3AHOvtP8xje+UaacsTkCgwPwfvvtF9mPr4baAc5+KSsLw3nrk1cje9P02FQg6+0sXGW/hJX9pCP7CUiepfVovpOyu2xk65B99Cf7KYjro3nzDY/wL//yL3HqqafG3XffHS95yUs2CcCukZFdj4GjPfnkk53fE9nwo1vZN+/ZNyc///nPXSftLMtGRxWAN4NFKHMKA58Bzn45KNt1HHhku4bZj4N6X9wAAAUHSURBVMCyX0DxSCcwOABnX9yzW59lX3QGHkN9Brjb+uTV8BngTdcvuwNEduu5d73rXZ2dlIEvKnmWA59vtB5pronsbhzf+c53Oju9L3jBC9YXzT77m92iLvtvro801kWrHHLIIZ3d9w2DVha+sv8/+xqxxx57WJOimA2P++QnP9m5fWD2y9Wuk4axC5QXgAsgbU5Dsl3eLCBdcMEF8ZGPfKRzau4C0dwKDQ7A2f1nL7vssvjpT3+6PgBkuy9f//rXO78EVGR98mo092pGZ+XsR+zZreay+y1feOGFG330J8/SeqRd8+wXrLKfUGQh+NBDD+0Uz269ld367M/+7M86d0xxfaQ1z6uW3bUm+yjKho/s86bZT6n+4i/+ovMLvdYkTzHt8z/84Q87d63Jwu6GH2nLvlnJrpHsrinWJK15lWoCcBW1ludkv/jzhS98oXPbs+xG29l3ldlOV/YjMPcBTrs4gwNwtgM28MU+u/9ytiP/13/91503sw996EOdg+etT5EaaV/F6K2W/fLIvvvuG29961s3ucNJ9hGILOBaj5Fb3+xzptkvgWbvN9kvwGU/cj/rrLPihhtu6Nx9YMyYMdZj5JZj2CNt+BngIu833rPSLlr2saAsAGc78dk38NmtS//xH/+x8x6WfUwoC8Het9KaV6kmAFdRa3lO9gtw2f0es9/yzb7zz/5QQ/bRh6H+6kzLpzrqDz84AGcvKHsDy3Yjv//973d+yzf7q1jZvVEHHkXWJ6/GqIdL9AKye2lmX5yHemTfcHzxi1+0Homsi5b51a9+1fnMafZLiNlPnw4++ODOT6QG7guc19uuj6LS1cdtGIC9Z1V3rDNz1apV8bd/+7dx/fXXd66T7K/zZT+xyq4Xa1JHNt1cATidpUoECBAgQIAAAQKjQEAAHgWL5BQJECBAgAABAgTSCQjA6SxVIkCAAAECBAgQGAUCAvAoWCSnSIAAAQIECBAgkE5AAE5nqRIBAgQIECBAgMAoEBCAR8EiOUUCBAgQIECAAIF0AgJwOkuVCBAgQIAAAQIERoGAADwKFskpEiBAgAABAgQIpBMQgNNZqkSAAAECBAgQIDAKBATgUbBITpEAAQIECBAgQCCdgACczlIlAgQIECBAgACBUSAgAI+CRXKKBAgQIECAAAEC6QQE4HSWKhEgQIAAAQIECIwCAQF4FCySUyRAgAABAgQIEEgnIACns1SJAAECBAgQIEBgFAgIwKNgkZwiAQIECBAgQIBAOgEBOJ2lSgQIECBAgAABAqNAQAAeBYvkFAkQIECAAAECBNIJCMDpLFUiQIAAAQIECBAYBQIC8ChYJKdIgAABAgQIECCQTkAATmepEgECBAgQIECAwCgQEIBHwSI5RQIECBAgQIAAgXQCAnA6S5UIECBAgAABAgRGgYAAPAoWySkSIECAAAECBAikExCA01mqRIAAAQIECBAgMAoEBOBRsEhOkQABAgQIECBAIJ2AAJzOUiUCBAgQIECAAIFRICAAj4JFcooECBAgQIAAAQLpBATgdJYqESBAgAABAgQIjAIBAXgULJJTJECAAAECBAgQSCcgAKezVIkAAQIECBAgQGAUCAjAo2CRnCIBAgQIECBAgEA6AQE4naVKBAgQIECAAAECo0BAAB4Fi+QUCRAgQIAAAQIE0gkIwOksVSJAgAABAgQIEBgFAgLwKFgkp0iAAAECBAgQIJBO4P8D414v+8QAtZ4AAAAASUVORK5CYII=\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<matplotlib.collections.PathCollection at 0x214082ed4f0>"
      ]
     },
     "execution_count": 111,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sel_region = 'NRNX1-seg-192'\n",
    "plt.figure()\n",
    "plt.scatter([int(_info['pb_index']) \n",
    "             for _info in pb_dict[sel_region].kept_probes.values()],\n",
    "             np.ones(len(pb_dict[sel_region].kept_probes)))\n",
    "plt.scatter([int(_info['pb_index']) \n",
    "             for _info in sel_sequential_pb_dict[sel_region].kept_probes.values()],\n",
    "             2*np.ones(len(sel_sequential_pb_dict[sel_region].kept_probes)))\n",
    "\n",
    "#plt.scatter([int(_p) for _p in genome_name_2_pb_pos['5']], \n",
    "#            2*np.ones(len(genome_name_2_pb_pos['5'])) )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "id": "1bf27455",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: NRNX1-seg-161 -> 15\n",
      "gene: NRNX1-seg-223 -> 15\n",
      "gene: NRNX1-seg-636 -> 15\n",
      "gene: NRNX1-seg-15 -> 18\n",
      "gene: NRNX1-seg-214 -> 13\n",
      "gene: NRNX1-seg-224 -> 14\n",
      "gene: NRNX1-seg-313 -> 16\n",
      "gene: NRNX1-seg-89 -> 17\n",
      "gene: NRNX1-seg-334 -> 17\n",
      "gene: NRNX1-seg-498 -> 21\n",
      "gene: NRNX1-seg-619 -> 13\n",
      "gene: NRNX1-seg-165 -> 23\n",
      "gene: NRNX1-seg-197 -> 14\n",
      "gene: NRNX1-seg-376 -> 13\n",
      "gene: NRNX1-seg-620 -> 15\n",
      "gene: NRNX1-seg-166 -> 24\n",
      "gene: NRNX1-seg-239 -> 13\n",
      "gene: NRNX1-seg-357 -> 13\n",
      "gene: NRNX1-seg-470 -> 19\n",
      "gene: NRNX1-seg-93 -> 13\n",
      "gene: NRNX1-seg-209 -> 15\n",
      "gene: NRNX1-seg-319 -> 13\n",
      "gene: NRNX1-seg-201 -> 17\n",
      "gene: NRNX1-seg-221 -> 13\n",
      "gene: NRNX1-seg-679 -> 14\n",
      "gene: NRNX1-seg-172 -> 16\n",
      "gene: NRNX1-seg-192 -> 20\n",
      "gene: NRNX1-seg-625 -> 20\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "reg_size_dic, len_check = check._check_region_size(cand_sequential_probes, \n",
    "                                                   species_marker = 'gene_',\n",
    "                                                   min_size=12)\n",
    "print(len_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "id": "cd424910",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- total probes removed by internal screening: 0\n"
     ]
    }
   ],
   "source": [
    "# check these probes with int_map\n",
    "\n",
    "kept_probes = library_tools.quality_check._check_between_probes(cand_sequential_probes, lib_table,\n",
    "                                                  primer_len=primer_len, \n",
    "                                                  target_len=target_len)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a9836188",
   "metadata": {},
   "source": [
    "## Blast cand_sequential_probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 117,
   "id": "e430025b",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Blast probe against \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\blast_database\\hg38\n",
      "-- number of probes loaded: 449\n",
      "-- processing region: NRNX1-seg-15 with 18 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-15.fasta\n",
      "--- *blasting region: NRNX1-seg-15\n",
      "--- total time for blast NRNX1-seg-15: 2.314s.\n",
      "-- processing region: NRNX1-seg-161 with 15 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-161.fasta\n",
      "--- *blasting region: NRNX1-seg-161\n",
      "--- total time for blast NRNX1-seg-161: 1.421s.\n",
      "-- processing region: NRNX1-seg-165 with 23 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-165.fasta\n",
      "--- *blasting region: NRNX1-seg-165\n",
      "--- total time for blast NRNX1-seg-165: 1.659s.\n",
      "-- processing region: NRNX1-seg-166 with 24 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-166.fasta\n",
      "--- *blasting region: NRNX1-seg-166\n",
      "--- total time for blast NRNX1-seg-166: 3.584s.\n",
      "-- processing region: NRNX1-seg-172 with 16 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-172.fasta\n",
      "--- *blasting region: NRNX1-seg-172\n",
      "--- total time for blast NRNX1-seg-172: 2.687s.\n",
      "-- processing region: NRNX1-seg-192 with 20 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-192.fasta\n",
      "--- *blasting region: NRNX1-seg-192\n",
      "--- total time for blast NRNX1-seg-192: 1.873s.\n",
      "-- processing region: NRNX1-seg-197 with 14 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-197.fasta\n",
      "--- *blasting region: NRNX1-seg-197\n",
      "--- total time for blast NRNX1-seg-197: 1.511s.\n",
      "-- processing region: NRNX1-seg-201 with 17 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-201.fasta\n",
      "--- *blasting region: NRNX1-seg-201\n",
      "--- total time for blast NRNX1-seg-201: 3.787s.\n",
      "-- processing region: NRNX1-seg-209 with 15 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-209.fasta\n",
      "--- *blasting region: NRNX1-seg-209\n",
      "--- total time for blast NRNX1-seg-209: 1.610s.\n",
      "-- processing region: NRNX1-seg-214 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-214.fasta\n",
      "--- *blasting region: NRNX1-seg-214\n",
      "--- total time for blast NRNX1-seg-214: 1.473s.\n",
      "-- processing region: NRNX1-seg-221 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-221.fasta\n",
      "--- *blasting region: NRNX1-seg-221\n",
      "--- total time for blast NRNX1-seg-221: 1.318s.\n",
      "-- processing region: NRNX1-seg-223 with 15 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-223.fasta\n",
      "--- *blasting region: NRNX1-seg-223\n",
      "--- total time for blast NRNX1-seg-223: 1.535s.\n",
      "-- processing region: NRNX1-seg-224 with 14 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-224.fasta\n",
      "--- *blasting region: NRNX1-seg-224\n",
      "--- total time for blast NRNX1-seg-224: 1.705s.\n",
      "-- processing region: NRNX1-seg-239 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-239.fasta\n",
      "--- *blasting region: NRNX1-seg-239\n",
      "--- total time for blast NRNX1-seg-239: 1.432s.\n",
      "-- processing region: NRNX1-seg-313 with 16 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-313.fasta\n",
      "--- *blasting region: NRNX1-seg-313\n",
      "--- total time for blast NRNX1-seg-313: 1.566s.\n",
      "-- processing region: NRNX1-seg-319 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-319.fasta\n",
      "--- *blasting region: NRNX1-seg-319\n",
      "--- total time for blast NRNX1-seg-319: 4.364s.\n",
      "-- processing region: NRNX1-seg-334 with 17 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-334.fasta\n",
      "--- *blasting region: NRNX1-seg-334\n",
      "--- total time for blast NRNX1-seg-334: 4.125s.\n",
      "-- processing region: NRNX1-seg-357 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-357.fasta\n",
      "--- *blasting region: NRNX1-seg-357\n",
      "--- total time for blast NRNX1-seg-357: 1.281s.\n",
      "-- processing region: NRNX1-seg-376 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-376.fasta\n",
      "--- *blasting region: NRNX1-seg-376\n",
      "--- total time for blast NRNX1-seg-376: 2.460s.\n",
      "-- processing region: NRNX1-seg-470 with 19 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-470.fasta\n",
      "--- *blasting region: NRNX1-seg-470\n",
      "--- total time for blast NRNX1-seg-470: 1.535s.\n",
      "-- processing region: NRNX1-seg-498 with 21 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-498.fasta\n",
      "--- *blasting region: NRNX1-seg-498\n",
      "--- total time for blast NRNX1-seg-498: 1.999s.\n",
      "-- processing region: NRNX1-seg-619 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-619.fasta\n",
      "--- *blasting region: NRNX1-seg-619\n",
      "--- total time for blast NRNX1-seg-619: 1.191s.\n",
      "-- processing region: NRNX1-seg-620 with 15 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-620.fasta\n",
      "--- *blasting region: NRNX1-seg-620\n",
      "--- total time for blast NRNX1-seg-620: 2.239s.\n",
      "-- processing region: NRNX1-seg-625 with 20 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-625.fasta\n",
      "--- *blasting region: NRNX1-seg-625\n",
      "--- total time for blast NRNX1-seg-625: 3.549s.\n",
      "-- processing region: NRNX1-seg-636 with 15 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-636.fasta\n",
      "--- *blasting region: NRNX1-seg-636\n",
      "--- total time for blast NRNX1-seg-636: 3.104s.\n",
      "-- processing region: NRNX1-seg-679 with 14 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-679.fasta\n",
      "--- *blasting region: NRNX1-seg-679\n",
      "--- total time for blast NRNX1-seg-679: 6.215s.\n",
      "-- processing region: NRNX1-seg-89 with 17 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-89.fasta\n",
      "--- *blasting region: NRNX1-seg-89\n",
      "--- total time for blast NRNX1-seg-89: 1.793s.\n",
      "-- processing region: NRNX1-seg-93 with 13 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\blast_sequential\\probe_gene_NRNX1-seg-93.fasta\n",
      "--- *blasting region: NRNX1-seg-93\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast NRNX1-seg-93: 1.522s.\n"
     ]
    }
   ],
   "source": [
    "blast_database = os.path.join(reference_folder, 'blast_database', 'hg38')\n",
    "\n",
    "check.Blast_probes(cand_sequential_probes, sequential_assemble_folder, \n",
    "                   blast_subfolder='blast_sequential',\n",
    "                   num_threads=44, \n",
    "                   blast_db=blast_database, blast_evalue=100, overwrite=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 118,
   "id": "eb86ab06",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Number of probes loaded: 449\n",
      "- Number of regions in this library: 28\n",
      "-- checking probes in region: NRNX1-seg-161\n",
      "--- number of probes: 15 , kept by blast: 15 , if remove dups: 15\n",
      "-- number of probes kept for this region: 15\n",
      "-- checking probes in region: NRNX1-seg-223\n",
      "--- number of probes: 15 , kept by blast: 15 , if remove dups: 15\n",
      "-- number of probes kept for this region: 15\n",
      "-- checking probes in region: NRNX1-seg-636\n",
      "--- gene=NRNX1-seg-636, id=0 removed by soft count = 60\n",
      "--- gene=NRNX1-seg-636, id=1 removed by soft count = 203\n",
      "--- gene=NRNX1-seg-636, id=2 removed by soft count = 205\n",
      "--- gene=NRNX1-seg-636, id=3 removed by soft count = 371\n",
      "--- gene=NRNX1-seg-636, id=4 removed by soft count = 273\n",
      "--- gene=NRNX1-seg-636, id=5 removed by soft count = 208\n",
      "--- gene=NRNX1-seg-636, id=6 removed by soft count = 200\n",
      "--- gene=NRNX1-seg-636, id=7 removed by soft count = 234\n",
      "--- gene=NRNX1-seg-636, id=8 removed by soft count = 224\n",
      "--- gene=NRNX1-seg-636, id=9 removed by soft count = 83\n",
      "--- gene=NRNX1-seg-636, id=10 removed by soft count = 104\n",
      "--- gene=NRNX1-seg-636, id=11 removed by soft count = 133\n",
      "--- gene=NRNX1-seg-636, id=12 removed by soft count = 884\n",
      "--- gene=NRNX1-seg-636, id=13 removed by soft count = 651\n",
      "--- gene=NRNX1-seg-636, id=14 removed by soft count = 166\n",
      "--- number of probes: 15 , kept by blast: 0 , if remove dups: 0\n",
      "-- number of probes kept for this region: 0\n",
      "-- checking probes in region: NRNX1-seg-15\n",
      "--- number of probes: 18 , kept by blast: 18 , if remove dups: 18\n",
      "-- number of probes kept for this region: 18\n",
      "-- checking probes in region: NRNX1-seg-214\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "E:\\Users\\puzheng_new\\Documents\\Chromatin_Analysis_Scripts\\Library_design\\CTP-14-human_brain_genome\\..\\..\\..\\..\\Documents\\ImageAnalysis3\\library_tools\\quality_check.py:578: RuntimeWarning: invalid value encountered in true_divide\n",
      "  if sum(_keep_dic[_reg] * (_hard_cts == 1)) / float(sum(_keep_dic[_reg])) >= smallest_region_ratio and sum(_keep_dic[_reg]) >= smallest_region_ratio*probe_per_region:\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 13 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: NRNX1-seg-224\n",
      "--- gene=NRNX1-seg-224, id=0 removed by soft count = 91\n",
      "--- gene=NRNX1-seg-224, id=1 removed by soft count = 64\n",
      "--- gene=NRNX1-seg-224, id=2 removed by soft count = 92\n",
      "--- number of probes: 14 , kept by blast: 11 , if remove dups: 11\n",
      "-- number of probes kept for this region: 11\n",
      "-- checking probes in region: NRNX1-seg-313\n",
      "--- number of probes: 16 , kept by blast: 16 , if remove dups: 16\n",
      "-- number of probes kept for this region: 16\n",
      "-- checking probes in region: NRNX1-seg-89\n",
      "--- number of probes: 17 , kept by blast: 17 , if remove dups: 17\n",
      "-- number of probes kept for this region: 17\n",
      "-- checking probes in region: NRNX1-seg-334\n",
      "--- gene=NRNX1-seg-334, id=0 removed by soft count = 141\n",
      "--- gene=NRNX1-seg-334, id=1 removed by soft count = 188\n",
      "--- gene=NRNX1-seg-334, id=3 removed by soft count = 90\n",
      "--- gene=NRNX1-seg-334, id=4 removed by soft count = 433\n",
      "--- gene=NRNX1-seg-334, id=5 removed by soft count = 122\n",
      "--- gene=NRNX1-seg-334, id=7 removed by soft count = 175\n",
      "--- gene=NRNX1-seg-334, id=8 removed by soft count = 160\n",
      "--- gene=NRNX1-seg-334, id=9 removed by soft count = 65\n",
      "--- gene=NRNX1-seg-334, id=10 removed by soft count = 71\n",
      "--- gene=NRNX1-seg-334, id=11 removed by soft count = 817\n",
      "--- gene=NRNX1-seg-334, id=12 removed by soft count = 60\n",
      "--- gene=NRNX1-seg-334, id=13 removed by soft count = 608\n",
      "--- gene=NRNX1-seg-334, id=14 removed by soft count = 90\n",
      "--- gene=NRNX1-seg-334, id=15 removed by soft count = 121\n",
      "--- gene=NRNX1-seg-334, id=16 removed by soft count = 1699\n",
      "--- number of probes: 17 , kept by blast: 2 , if remove dups: 2\n",
      "-- number of probes kept for this region: 2\n",
      "-- checking probes in region: NRNX1-seg-498\n",
      "--- number of probes: 21 , kept by blast: 21 , if remove dups: 21\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 21\n",
      "-- checking probes in region: NRNX1-seg-619\n",
      "--- number of probes: 13 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: NRNX1-seg-165\n",
      "--- number of probes: 23 , kept by blast: 23 , if remove dups: 23\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 23\n",
      "-- checking probes in region: NRNX1-seg-197\n",
      "--- number of probes: 14 , kept by blast: 14 , if remove dups: 14\n",
      "-- number of probes kept for this region: 14\n",
      "-- checking probes in region: NRNX1-seg-376\n",
      "--- gene=NRNX1-seg-376, id=0 removed by soft count = 498\n",
      "--- gene=NRNX1-seg-376, id=1 removed by soft count = 71\n",
      "--- gene=NRNX1-seg-376, id=2 removed by soft count = 104\n",
      "--- gene=NRNX1-seg-376, id=3 removed by soft count = 339\n",
      "--- gene=NRNX1-seg-376, id=4 removed by soft count = 193\n",
      "--- gene=NRNX1-seg-376, id=5 removed by soft count = 52\n",
      "--- gene=NRNX1-seg-376, id=6 removed by soft count = 53\n",
      "--- gene=NRNX1-seg-376, id=7 removed by soft count = 53\n",
      "--- gene=NRNX1-seg-376, id=8 removed by soft count = 54\n",
      "--- gene=NRNX1-seg-376, id=9 removed by soft count = 52\n",
      "--- gene=NRNX1-seg-376, id=10 removed by soft count = 174\n",
      "--- gene=NRNX1-seg-376, id=11 removed by soft count = 248\n",
      "--- gene=NRNX1-seg-376, id=12 removed by soft count = 405\n",
      "--- number of probes: 13 , kept by blast: 0 , if remove dups: 0\n",
      "-- number of probes kept for this region: 0\n",
      "-- checking probes in region: NRNX1-seg-620\n",
      "--- gene=NRNX1-seg-620, id=0 removed by soft count = 90\n",
      "--- number of probes: 15 , kept by blast: 14 , if remove dups: 14\n",
      "-- number of probes kept for this region: 14\n",
      "-- checking probes in region: NRNX1-seg-166\n",
      "--- gene=NRNX1-seg-166, id=0 removed by soft count = 174\n",
      "--- gene=NRNX1-seg-166, id=2 removed by soft count = 99\n",
      "--- gene=NRNX1-seg-166, id=4 removed by soft count = 66\n",
      "--- gene=NRNX1-seg-166, id=5 removed by soft count = 56\n",
      "--- gene=NRNX1-seg-166, id=6 removed by soft count = 67\n",
      "--- gene=NRNX1-seg-166, id=8 removed by soft count = 203\n",
      "--- gene=NRNX1-seg-166, id=9 removed by soft count = 492\n",
      "--- gene=NRNX1-seg-166, id=10 removed by soft count = 55\n",
      "--- gene=NRNX1-seg-166, id=11 removed by soft count = 61\n",
      "--- gene=NRNX1-seg-166, id=12 removed by soft count = 142\n",
      "--- gene=NRNX1-seg-166, id=13 removed by soft count = 77\n",
      "--- gene=NRNX1-seg-166, id=15 removed by soft count = 91\n",
      "--- gene=NRNX1-seg-166, id=17 removed by soft count = 127\n",
      "--- gene=NRNX1-seg-166, id=18 removed by soft count = 90\n",
      "--- gene=NRNX1-seg-166, id=19 removed by soft count = 170\n",
      "--- gene=NRNX1-seg-166, id=20 removed by soft count = 209\n",
      "--- gene=NRNX1-seg-166, id=21 removed by soft count = 518\n",
      "--- gene=NRNX1-seg-166, id=22 removed by soft count = 654\n",
      "--- gene=NRNX1-seg-166, id=23 removed by soft count = 225\n",
      "--- number of probes: 24 , kept by blast: 5 , if remove dups: 5\n",
      "-- number of probes kept for this region: 5\n",
      "-- checking probes in region: NRNX1-seg-239\n",
      "--- number of probes: 13 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: NRNX1-seg-357\n",
      "--- number of probes: 13 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: NRNX1-seg-470\n",
      "--- number of probes: 19 , kept by blast: 19 , if remove dups: 19\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 19\n",
      "-- checking probes in region: NRNX1-seg-93\n",
      "--- number of probes: 13 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: NRNX1-seg-209\n",
      "--- number of probes: 15 , kept by blast: 15 , if remove dups: 15\n",
      "-- number of probes kept for this region: 15\n",
      "-- checking probes in region: NRNX1-seg-319\n",
      "--- gene=NRNX1-seg-319, id=0 removed by soft count = 72\n",
      "--- gene=NRNX1-seg-319, id=1 removed by soft count = 160\n",
      "--- gene=NRNX1-seg-319, id=2 removed by soft count = 418\n",
      "--- gene=NRNX1-seg-319, id=3 removed by soft count = 345\n",
      "--- gene=NRNX1-seg-319, id=4 removed by soft count = 61\n",
      "--- gene=NRNX1-seg-319, id=5 removed by soft count = 1795\n",
      "--- gene=NRNX1-seg-319, id=6 removed by soft count = 395\n",
      "--- gene=NRNX1-seg-319, id=7 removed by soft count = 525\n",
      "--- gene=NRNX1-seg-319, id=8 removed by soft count = 98\n",
      "--- gene=NRNX1-seg-319, id=9 removed by soft count = 402\n",
      "--- gene=NRNX1-seg-319, id=10 removed by soft count = 525\n",
      "--- gene=NRNX1-seg-319, id=11 removed by soft count = 857\n",
      "--- gene=NRNX1-seg-319, id=12 removed by soft count = 62\n",
      "--- number of probes: 13 , kept by blast: 0 , if remove dups: 0\n",
      "-- number of probes kept for this region: 0\n",
      "-- checking probes in region: NRNX1-seg-201\n",
      "--- gene=NRNX1-seg-201, id=0 removed by soft count = 3056\n",
      "--- number of probes: 17 , kept by blast: 16 , if remove dups: 16\n",
      "-- number of probes kept for this region: 16\n",
      "-- checking probes in region: NRNX1-seg-221\n",
      "--- number of probes: 13 , kept by blast: 13 , if remove dups: 13\n",
      "-- number of probes kept for this region: 13\n",
      "-- checking probes in region: NRNX1-seg-679\n",
      "--- gene=NRNX1-seg-679, id=0 removed by soft count = 51\n",
      "--- gene=NRNX1-seg-679, id=1 removed by soft count = 84\n",
      "--- gene=NRNX1-seg-679, id=2 removed by soft count = 58\n",
      "--- gene=NRNX1-seg-679, id=3 removed by soft count = 112\n",
      "--- gene=NRNX1-seg-679, id=4 removed by soft count = 137\n",
      "--- gene=NRNX1-seg-679, id=5 removed by soft count = 63\n",
      "--- gene=NRNX1-seg-679, id=6 removed by soft count = 125\n",
      "--- gene=NRNX1-seg-679, id=7 removed by soft count = 62\n",
      "--- gene=NRNX1-seg-679, id=8 removed by soft count = 601\n",
      "--- gene=NRNX1-seg-679, id=9 removed by soft count = 486\n",
      "--- gene=NRNX1-seg-679, id=10 removed by soft count = 81\n",
      "--- gene=NRNX1-seg-679, id=11 removed by soft count = 2423\n",
      "--- gene=NRNX1-seg-679, id=12 removed by soft count = 1644\n",
      "--- gene=NRNX1-seg-679, id=13 removed by soft count = 8692\n",
      "--- number of probes: 14 , kept by blast: 0 , if remove dups: 0\n",
      "-- number of probes kept for this region: 0\n",
      "-- checking probes in region: NRNX1-seg-172\n",
      "--- gene=NRNX1-seg-172, id=0 removed by soft count = 90\n",
      "--- gene=NRNX1-seg-172, id=14 removed by soft count = 877\n",
      "--- number of probes: 16 , kept by blast: 14 , if remove dups: 14\n",
      "-- number of probes kept for this region: 14\n",
      "-- checking probes in region: NRNX1-seg-192\n",
      "--- number of probes: 20 , kept by blast: 20 , if remove dups: 20\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 20\n",
      "-- checking probes in region: NRNX1-seg-625\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=NRNX1-seg-625, id=1 removed by soft count = 375\n",
      "--- gene=NRNX1-seg-625, id=3 removed by soft count = 443\n",
      "--- gene=NRNX1-seg-625, id=4 removed by soft count = 249\n",
      "--- gene=NRNX1-seg-625, id=5 removed by soft count = 412\n",
      "--- gene=NRNX1-seg-625, id=6 removed by soft count = 304\n",
      "--- gene=NRNX1-seg-625, id=7 removed by soft count = 335\n",
      "--- gene=NRNX1-seg-625, id=8 removed by soft count = 99\n",
      "--- gene=NRNX1-seg-625, id=9 removed by soft count = 140\n",
      "--- gene=NRNX1-seg-625, id=10 removed by soft count = 320\n",
      "--- gene=NRNX1-seg-625, id=11 removed by soft count = 463\n",
      "--- gene=NRNX1-seg-625, id=12 removed by soft count = 515\n",
      "--- gene=NRNX1-seg-625, id=13 removed by soft count = 161\n",
      "--- gene=NRNX1-seg-625, id=14 removed by soft count = 209\n",
      "--- gene=NRNX1-seg-625, id=15 removed by soft count = 68\n",
      "--- gene=NRNX1-seg-625, id=16 removed by soft count = 460\n",
      "--- gene=NRNX1-seg-625, id=17 removed by soft count = 134\n",
      "--- gene=NRNX1-seg-625, id=18 removed by soft count = 390\n",
      "--- gene=NRNX1-seg-625, id=19 removed by soft count = 111\n",
      "--- number of probes: 20 , kept by blast: 2 , if remove dups: 2\n",
      "-- number of probes kept for this region: 2\n",
      "- summarize\n",
      "-- region: NRNX1-seg-161 , number of probes: 15\n",
      "-- region: NRNX1-seg-223 , number of probes: 15\n",
      "-- region: NRNX1-seg-636 , number of probes: 0\n",
      "-- region: NRNX1-seg-15 , number of probes: 18\n",
      "-- region: NRNX1-seg-214 , number of probes: 13\n",
      "-- region: NRNX1-seg-224 , number of probes: 11\n",
      "-- region: NRNX1-seg-313 , number of probes: 16\n",
      "-- region: NRNX1-seg-89 , number of probes: 17\n",
      "-- region: NRNX1-seg-334 , number of probes: 2\n",
      "-- region: NRNX1-seg-498 , number of probes: 21\n",
      "-- region: NRNX1-seg-619 , number of probes: 13\n",
      "-- region: NRNX1-seg-165 , number of probes: 23\n",
      "-- region: NRNX1-seg-197 , number of probes: 14\n",
      "-- region: NRNX1-seg-376 , number of probes: 0\n",
      "-- region: NRNX1-seg-620 , number of probes: 14\n",
      "-- region: NRNX1-seg-166 , number of probes: 5\n",
      "-- region: NRNX1-seg-239 , number of probes: 13\n",
      "-- region: NRNX1-seg-357 , number of probes: 13\n",
      "-- region: NRNX1-seg-470 , number of probes: 19\n",
      "-- region: NRNX1-seg-93 , number of probes: 13\n",
      "-- region: NRNX1-seg-209 , number of probes: 15\n",
      "-- region: NRNX1-seg-319 , number of probes: 0\n",
      "-- region: NRNX1-seg-201 , number of probes: 16\n",
      "-- region: NRNX1-seg-221 , number of probes: 13\n",
      "-- region: NRNX1-seg-679 , number of probes: 0\n",
      "-- region: NRNX1-seg-172 , number of probes: 14\n",
      "-- region: NRNX1-seg-192 , number of probes: 20\n",
      "-- region: NRNX1-seg-625 , number of probes: 2\n",
      "- Number of probes kept: 335\n",
      "- Saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_NRNX1_5kb\\Sequential\\.\\blast_sequential_full_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "# screen\n",
    "import ImageAnalysis3.library_tools.quality_check as qc\n",
    "reload(check)\n",
    "num_probes_per_region = 25\n",
    "kept_pbs, blast_keep_dic, hard_count_list, soft_count_list = qc.Screening_Probes_by_Blast(\n",
    "    sequential_assemble_folder, \n",
    "    num_probes_per_region,\n",
    "    probe_filename='candidate_probes.fasta',\n",
    "    blast_subfolder='blast_sequential',\n",
    "    keep_mode='end',\n",
    "    hard_thres=target_len,\n",
    "    soft_count_th=50,\n",
    "    save_filename='blast_sequential_full_probes.fasta',\n",
    "    )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "id": "f7c9188d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7snQd4FFUXhj8SepfeOyG00AOh1xAQEEGqBVBEpEqRX3roCAqKVEVFaUpvQmihBEKAEHoIvQQILUBCC/1/7qwTN5BkZ2ZndzbZ7z7P//yYvfWdM7vf3Dn3nBSvX79+DRYSIAESIAESIAESIAEScBICKSiAneRKc5kkQAIkQAIkQAIkQAISAQpgGgIJkAAJkAAJkAAJkIBTEaAAdqrLzcWSAAmQAAmQAAmQAAlQANMGSIAESIAESIAESIAEnIoABbBTXW4ulgRIgARIgARIgARIgAKYNkACJEACJEACJEACJOBUBCiAnepyc7EkQAIkQAIkQAIkQAIUwLQBEiABEiABEiABEiABpyJAAexUl5uLJQESIAESIAESIAESoACmDZAACZAACZAACZAACTgVAQpgp7rcXCwJkAAJkAAJkAAJkAAFMG2ABEiABEiABEiABEjAqQhQADvV5eZiSYAESIAESIAESIAEKIBpAyRAAiRAAiRAAiRAAk5FgALYqS43F0sCJEACJEACJEACJEABTBsgARIgARIgARIgARJwKgIUwE51ublYEiABEiABEiABEiABCmDaAAmQAAmQAAmQAAmQgFMRoAB2qsvNxZIACZAACZAACZAACVAA0wZIgARIgARIgARIgAScigAFsFNdbi6WBEiABEiABEiABEiAApg2QAIkQAIkQAIkQAIk4FQEKICd6nJzsSRAAiRAAiRAAiRAAhTAtAESIAESIAESIAESIAGnIkAB7FSXm4slARIgARIgARIgARKgAKYNkAAJkAAJkAAJkAAJOBUBCmCnutxcLAmQAAmQAAmQAAmQAAUwbYAESIAESIAESIAESMCpCFAAO9Xl5mJJgARIgARIgARIgAQogGkDJEACJEACJEACJEACTkWAAtipLjcXSwIkQAIkQAIkQAIkQAFMGyABEiABEiABEiABEnAqAhTATnW5uVgSIAESIAESIAESIAEKYNoACZAACZAACZAACZCAUxGgAHaqy83FkgAJkAAJkAAJkAAJUADTBkiABEiABEiABEiABJyKAAWwU11uLpYESIAESIAESIAESIACmDZAAiRAAiRAAiRAAiTgVAQogJ3qcnOxJEACJEACJEACJEACFMC0ARIgARIgARIgARIgAaciQAHsVJebiyUBEiABEiABEiABEqAApg2QAAmQAAmQAAmQAAk4FQEKYKe63FwsCZAACZAACZAACZAABTBtgARIgARIgARIgARIwKkIUAA71eXmYkmABEiABEiABEiABCiAaQMkQAIkQAIkQAIkQAJORYAC2KkuNxdLAiRAAiRAAiRAAiRAAUwbIAESIAESIAESIAEScCoCFMBOdbm5WBIgARIgARIgARIgAQpg2gAJkAAJkAAJkAAJkIBTEaAAdqrLzcWSAAmQAAmQAAmQAAlQANMGSIAESIAESIAESIAEnIoABbBTXW4ulgRIgARIgARIgARIgAKYNkACJEACJEACJEACJOBUBCiAnepyc7EkQAIkQAIkQAIkQAIUwLQBEiABEiABEiABEiABpyJAAexUl5uLJQESIAESIAESIAEScFoBfOvWLQwaNAh+fn548uQJ6tWrh6lTp6JMmTKSVRw5cgT9+/dHcHAwsmfPjn79+mHw4MG0GBIgARIgARIgARIggSROwGkFcI0aNeDi4oIZM2YgQ4YMGDlyJAIDA3Hu3DlJELu7u6N169YYOHAggoKC0KtXL8yePRvdunVL4pec0ycBEiABEiABEiAB5ybglAI4MjISffr0wYgRI1C2bFnJAo4ePYqKFSviwIED2LZtG2bNmoVLly4hZcqU0ufDhg3DqlWrEBYW5twWw9WTAAmQAAmQAAmQQBIn4JQC+M1rdvPmTcm9YefOnZLAbdeuHbJmzYolS5bEVhWiuEmTJhB1c+XKlcQvO6dPAiRAAiRAAiRAAs5LwOkFcI8ePfDLL78gTZo0WLduHby9veHh4YFmzZrh22+/jbWM0NBQabf44MGDqFq1arwWEx0dDfE/ucTExCA8PBxFixaN3Ul2XlPjykmABEiABEiABOxJ4MWLF7h9+zbKly+PtGnT2nNohx/L6QWwELbC53fOnDnSju+ePXvQvn17dO7cGWPHjo29gBcuXEDx4sUREBCA2rVrx3thfX19MWbMGIe/6JwgCZAACZAACZCA8xAQ7p3VqlVzngUrWKnTC2CZ0atXr6QnpOrVq0u7vM2bN493BzgkJASVKlVStAMsdn9r1qwp+RXnzZtXweVgFRIgARIgARIgARLQh0BERAQ8PT1x8eJFFClSRJ9Ok0kvTimARQi07du3Szu9rq6usZdS+P4KFwbxNxH6bOHChbGfafEBvnr1KgoWLCi5QRQoUCCZmAyXQQIkQAIkQAIkkBQIUIckfJWcUgCLGL9iF1eI4IYNG0p0nj9/LoU+a9WqFXLnzo25c+fi/PnzsQJ56NChWL16taooEDS8pPD1wDmSAAmQAAmQQPIkQB1CARyHwOvXr6VDbiLMmTgA984772DChAnYvHmzlABDOIrLYnjIkCGSC8OXX34pieIuXboovktoeIpRsSIJkAAJkAAJkIDOBKhDKIDfIhAVFQWxq7tmzRrcv38fderUwbRp02LjAgs/YJH97fDhw5L/rsgaJ2IHqyk0PDW0WJcESIAESIAESEBPAtQhFMB62pPivmh4ilGxIgmQAAmQAAmQgM4EqEMogHU2KWXd0fCUcWItEiABEiABEiAB/QlQh1AA629VCnqk4SmAxCokQAIkQAIkQAI2IUAdQgFsE8Oy1CkNzxIhfk4CJEACJEACJGArAtQhFMC2sq1E+6XhGYKdg5IACZAACZAACQCgDqEANuRGoOEZgp2DkgAJkAAJkAAJUAAnagNOmQjDXncFBbC9SHMcEiABEiABEiCBNwlQh3AH2JC7goZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH2JC7gIZnCHYOSgIkQAIkQAIkwB1g7gAbdRdQABtFnuOSAAmQAAmQAAlQh3AH+C0Cd+/exbBhw7BhwwZER0fDw8MDkydPRu3ataW63bp1w4IFC+K0y58/P4QxKS00PKWkWI8ESIAESIAESEBvAtQhFMBvEfD29satW7cwc+ZM5MyZE7NmzcL8+fMREhICd3d3VKlSBT4+Pujbt29sW1dXV6mu0kLDU0qK9UiABEiABEiABPQmQB1CARyHwLlz51CyZEns3bsXNWvWlD57/fo13Nzc0KlTJ4wePRoZM2bE0qVL0bp1a832SMPTjI4NSYAESIAESIAErCRAHUIBHIfA/fv3ERgYiEaNGiFNmjSxn5UoUULa9e3Tpw9Kly6N06dPS6JYa6HhaSXHdiRAAiRAAiRAAtYSoA6hALZoQ8uXL0f79u2xbt06xMTEoEOHDujZsyc2bdoEFxcXNG/eHOPHj0eWLFkS7Ev4Eov/ySUiIgKenp4IDw9HgQIFLM6BFUiABEiABEiABEhALwIUwBTAidqScIUQO79iR3jNmjWSC8SECRMwceJESfiePXsWgwcPRsGCBeHv7y8J4viKr68vxowZ89ZHFMB63crshwRIgARIgARIQCkBCmAK4AQJrF27Fp07d0aNGjWkiBDp0qWT/IGjoqKQNWvW2HZBQUHw8vKC+P/q1avH2x93gJXekqxHAiRAAiRAAiRgawIUwBTA8RIQESD69++PNm3aYNGiRXH8gd9s8OjRI+lg3IoVK9C2bVtFNkvDU4SJlUiABEiABEiABGxAgDqEAvgtAnPmzEGvXr3Qr18/TJ8+PY5bg9gRjoyMxObNm2Pb7dy5Ew0aNEBoaKh0QE5JoeEpocQ6JEACJEACJEACtiBAHUIBHIfAmTNnUK5cObRo0QKzZ8+O85lwgQgICECrVq0wbtw4dOzYEaJ+7969JReIxYsXK7ZRGp5iVKxIAiRAAiRAAiSgMwHqEArgOATE4bbhw4fHS6VLly5SBriVK1dKB+HCwsIkX2CxKyyiQKRNm1axedLwFKNiRRIgARIgARIgAZ0JUIdQAOtsUsq6o+Ep48RaJEACJEACJEAC+hOgDqEA1t+qFPRIw1MAiVVIgARIgARIgARsQoA6hALYJoZlqVManiVC/JwESIAESIAESMBWBKhDKIBtZVuJ9kvDMwQ7ByUBEiABEiABEgBAHUIBbMiNQMMzBDsHJQESIAESIAESoABO1AZSvBZpz1hsQoAC2CZY2SkJkAAJkAAJkIACAtQh3AFWYCb6V6Hh6c+UPZIACZAACZAACSgjQB1CAazMUnSuRcPTGSi7IwESIAESIAESUEyAOoQCWLGx6FmRhqcnTfZFAiRAAiRAAiSghgB1CAWwGnvRrS4NTzeU7IgESIAESIAESEAlAeoQCmCVJqNPdRqePhzZCwmQAAmQAAmQgHoC1CEUwOqtRocWNDwdILILEiABEiABEiABTQSoQyiANRmOtY1oeNYSZHsSIAESIAESIAGtBKhDKIC12o5V7Wh4VuFjYxIgARIgARIgASsIUIdQAFthPtqb0vC0s2NLEiABEiABEiAB6whQh1AAW2dBGlvT8DSCYzMSIAESIAESIAGrCVCHUABbbURaOqDhaaHGNiRAAiRAAiRAAnoQoA6hANbDjlT3QcNTjYwNSIAESIAESIAEdCJAHUIBrJMpqeuGhqeOF2uTAAmQAAmQAAnoR4A6hAJYP2tS0RMNTwUsViUBEiABEiABEtCVAHUIBbCuBqW0MxqeUlKsRwK2I/Dw6Qt0+e0A3PNkwoT3y9tuIPZMAiRAAg5GgDqEAtgQk6ThGYKdg5JAHAKbjkfgy8Uh0t8Oj2yCdzKkJiESIAEScAoC1CEUwIYYOg3PEOwclATiEJi36zwmbQqT/rbySy9UKZyNhEiABEjAKQhQh1AAG2LoNDxDsHNQEohDYNjq41iy/4r0t6kfeKBd1YIkRAIkQAJOQYA6hALYEEOn4RmCnYOSQBwCH83fjz3n7kh/+7J+cfzPx52ESIAESMApCFCHUAAbYug0PEOwc1ASiEOgzhR/hN99Iv3Np2wezP24CgmRAAmQgFMQoA6hADbE0Gl4hmDnoCQQS+D5y1dwH+mHl69eS39zy50RWwbUIyESIAEScAoC1CEUwIYYOg3PEOwclARiCVyOfIR6U3fG/nfqlC44NdYHri4pSIkESIAEkj0B6hAKYEOMnIZnCHYOSgKxBHafuY1PfjsQh0jAkAYomC09KZEACZBAsidAHUIBbIiR0/AMwc5BSSCWwKKgyxix5gQypU2JBzEvpL//8akn6rnlJCUSIAESSPYEqEMogA0xchqeIdg5KAnEEpi48RR+3n0BlQtlxfX7MbgRHYPRLcugW62ipEQCJEACyZ4AdQgFsCFGTsMzBDsHJYFYAl8sDMbmkzfxfqX8uBkdg8Dzkfi4RmGMa12OlEiABEgg2ROgDqEANsTIaXiGYOegJBBLoNmPATgVEY3+jUrizsOnWLz/CmqXyIFF3auTEgmQAAkkewLUIRTAhhg5Dc8Q7ByUBCQCr1+/RrnRm/Ho2Ut8364C7j95jnEbQpEvS1oEDm1ESiRAAiSQ7AlQh1AAG2LkNDxDsHNQEpAIiB3fquO3Sf9e0dNLOgTXbcFB6b9FKLR0qV1JigRIgASSNQHqEApgQwychmcIdg5KAhKBQ5fvoe2cQOnfB4Y3wpNnL2NjAm/sVwdl8mUmKRIgARJI1gSoQyiADTFwGp4h2DkoCUgE1hy+hq/+PoJ0qVwROraplA2u9Cg/PH/5GrM6V8a7HnlJigRIgASSNQHqEApgQwychmcIdg5KAhKBH7edxfRtZ1AqdyZsHlBX+lvjabtw7tZDDGrihr6NSpIUCZAACSRrAtQhFMCGGDgNzxDsHJQEJAIDlx3BqpBraFImN375pKr0t8//DMbW0JtoUyk/pnWoSFIkQAIkkKwJUIdQABti4DQ8Q7BzUBKQCLSbG4iDl+6he+2iGNGijPS3SZtOYd6uC6hQMCvW9q5FUiRAAiSQrAlQh1AAG2LgNDxDsHNQEpAIeE7YhlsPnmLse2XxiVcR6W/LDoZjyMpjUmrkY6O9kSJFCtIiARIggWRLgDqEAtgQ46bhGYKdg5KAFPFBHHgTZUG3aqhfKpf074OX7qLd3H3Sv4NHNEaOjGlIiwRIgASSLQHqEApgQ4ybhmcIdg5KAjhz8wG8p++WSOwYXB9Fc2SQ/h358Cmq/BsbeNkXXvAsmo20SIAESCDZEqAOoQA2xLhpeIZg56AkIB10EwfeXFIAYeOaIXVKF4mKyA5XYcwWRMe8wLdty6NDtUKkRQIkQALJlgB1CAWwIcZNwzMEOwclAcwPuIDx/5xC/qzpsPebhnGItJ61F0fC7+OLusUwtHlp0iIBEiCBZEuAOoQC2BDjpuEZgp2DkgBGrT2BP/ddRs3i2bHk8xpxiAz8+whWHY4bHo3ISIAESCA5EqAOoQA2xK5peIZg56AkgK6/H8DO07fRybMgJrXxiENkpv9ZfLflDIrnzIDtg+qTFgmQAAkkWwLUIRTAhhg3Dc8Q7BzUYAI/bT+LsBsP8H37CkibytWQ2TT8bicu3HmEIT6l0Kt+iThz+OdYBHovCUEq1xQ4NdYHKV1N/sEsJEACJJDcCFCHUAAbYtM0PEOwc1ADCTx98RKlR/rh1Wtg7keV4VMur91n8/LVa7iP3ITnL19jVufKeNcj7hxORUSj2Y8B0rx2Dq6PIv9GiLD7RDkgCZAACdiYAHUIBbCNTSz+7ml4hmDnoAYSuBkdg+oTt0szGNDYDf0bl7T7bK7df4Jak/2lcTf0rY1y+bPEmUPM85dwH2mKEfxb16po6J7b7nPkgCRAAiRgDwLUIRTA9rCzt8ag4RmCnYMaSOD0jQdo+oMp/q7YeRU7sPYugefvoPMv+6Vhj472RpZ0qd6aghDIQiiPeLc0utcpZu8pcjwSIAESsAsB6hAKYLsY2puD0PAMwc5BDSSw/0IkOvwcJM2gZK6M2Dqwnt1n89eBK/hm1XFkTZ8KR0Z5xzv+x7/uR8DZO+hcvRAmvl/e7nPkgCRAAiRgDwLUIRTA9rCzt8ag4RmCnYMaSGDzyRv4YuEhaQYpXVIgdKxPbBIKe01ril8YZu88jwoFsmBtn9rxDiuHSfMqlh1Le8QNk2aveXIcEiABErA1AeoQCmBb21i8/dPwDMHOQQ0k8PfBK/jfyuOxM9gyoC7ccmey64xEhAcR6aGFR17MTMAFY8Hei/BdH4rcmdNg/7DGdp0fByMBEiABexGgDqEAtpetxRmHhmcIdg5qIIF5u85j0qaw2Bn81KkSWlbIZ9cZtfxpD45fi0LvBsXxdVP3eMfedeY2uvx2QPrs5JimyJAmpV3nyMFIgARIwB4EqEMogO1hZ2+NQcMzBDsHNZDAt35hmLPzfOwM+jUsgYHepew6Iw/fzYiOeYEpbT3QvlrBeMcOv/sYdabskD6LL1KEXSfMwUiABEjARgSoQyiA3yJw9+5dDBs2DBs2bEB0dDQ8PDwwefJk1K5t8hk8cuQI+vfvj+DgYGTPnh39+vXD4MGDVZkoDU8VLlZOBgSGrjqOpQeuxK6kadncmPdxVbutLOrxc1QYu0Ua768eNVCjWPZ4xxaxgkuP8sOzF6/wY8eKeK9ifrvNkQORAAmQgL0IUIdQAL9FwNvbG7du3cLMmTORM2dOzJo1C/Pnz0dISIj03+7u7mjdujUGDhyIoKAg9OrVC7Nnz0a3bt0U2y0NTzEqVkwmBL5cdAibTtyIXU2xHBngP9h+6YaPXb2PVjP3SuMHftMQ+bKmS5Bs0+m7cfrmA3zVuCS+auyWTK4Al0ECJEAC/xGgDqEAjkPg3LlzKFmyJPbu3YuaNWtKn71+/Rpubm7o1KkT0qVLJwniS5cuIWVKk2+g2C1etWoVwsL+82+0dJPR8CwR4ufJjUCnn4Ow70IkCmVLjyt3H8MlBaRIEPZKibz+6HX0XXpYijwRNtYHLmICCRRZrLeqkA8zOlVKbpeC6yEBEiABUIdQAMchcP/+fQQGBqJRo0ZIkyZN7GclSpSAj48PLly4gKxZs2LJkiWxn23btg1NmjTBzZs3kStXLkW3FQ1PESZWSkYERIphkWq4beUCWBlyVVrZP/1qo2y+uNnYbLXkWTvOYerm0yieMwO2D0p851kOl1Y+fxas7xt/uDRbzZP9kgAJkIA9CFCHUABbtLPly5ejffv2WLduHYYPH45mzZrh22+/jW0XGhqKsmXL4uDBg6haVZlPIw3PInZWSGYEak7ajutRMRjTqiwmbTqFmOev8EOHimhdyT4+tv9bcQx/B4ejQamc+L2bZ6J0Vxy6isHLjyJDalecGNMUKVIkvFuczC4Tl0MCJKCRgDg3cPHOI7jlzpgkvjOoQyiAEzV14Qohdn7FjvCaNWsgdoI7d+6MsWPHxrYTu8LFixdHQEBA7EG5NzsVh+nE/+QSEREBT09PhIeHo0CBAhpvNzYjgaRDoPRIPzx5/hIi/Nm83edx4lo0vqxfHP/ziT8cmd4r6/jzPgRduIuuNYvAt1XZRLs/dPke2s4JlOocGNYIuTKn1Xs67I8ESCCZEZCT6Pi2LIOutYo6/OoogCmAEySwdu1aSezWqFFDiggh/H/Lly+P5s2bx7sDLA7JVaoUv7+gr68vxowZ89ZYFMAO/x3BCepAIOb5S7iP9JN6WviZJ1YfvoZVIdfQuHQuzO9STYcRLHdRa7I/rt1/glEtyuDT2on/ON1//AwVx26VOl36eQ14FY8/YoTlUVmDBEjAWQh4TdqOiKgYfFClAL5rV8Hhl00BTAEcLwERAUKEOmvTpg0WLVoU6w8sxK8IfbZw4cLYdkp8gLkD7PDfBZygDQncio6B58Tt0gjr+9TG3vN3MHlTmHQgbveQBjYc2dT10xcmAf76NTD/k6poXCa3xTErjd2Ce4+fY+L75dG5eiGL9VmBBEjAeQlcv/8ENSf7SwAauefCr13t82BvDXEKYArgtwjMmTNHCm0m4vtOnz4dLi4usXVEPOC5c+fi/PnzcHV1lf4+dOhQrF69mlEgrLkT2TZZEzh94wGa/rBbWmPAkAY4e+sBPl0QLP136NimSJ/attnWzt9+iEbf75LG2zqgLkoqSMEsXCCEK0T32kUxokWZZH19uDgSIAHrCMhRZkQvlQtlxapetazr0A6tKYApgOMQOHPmDMqVK4cWLVpIsX3Ni3CBePr0qRQHuFWrVhgyZAgOHDiAL7/8UhLFXbp0UWyyNDzFqFgxGRDYfyESHX4OklZy3NcbUU+eo/a3pmxr6/rUgkeBrDZd5Y6wW+i24KA0Rtg4ZaHXxCE4cRiuoXsu/JYEdnNsCpCdkwAJJEpgzPqT+H3vJamOvWOca7001CEUwHEITJw4UYr0EF8RAnfBggVStAexO3z48GHkzZsXgwYNQp8+fVTZIA1PFS5WTuIENp+8gS8WHoKrSwqcm9BMWk250Zvx6NlLyVdO+MzZsvwReAmj151E7sxpsH9YY0VDzd55DlP8TqPCvZU/AAAgAElEQVRojgzYYceEHYomx0okQAIOReC9mXtw9GqUNKes6VPhyChvh5pffJOhDqEANsRIaXiGYOegBhFYdjAcQ1YeQ/YMqXFoZBNpFq1n7cWR8PvoUbcYhjUvbdOZjV0fit/2XoRnkWxY1tNL0Vh+J26g5yKTaD811kdKoMFCAiRAAm8SePLsJcr7bsaLV6+lj0TUxHMTmkvfHY5cqEMogA2xTxqeIdg5qEEE5u06j0mbwlAsZwb4/5uEYsiKo1gWfBX1S+XEAgtxea2ddvc/DmLbqVuqTmefufkA3tNNfsvbB9VD8ZwZrZ0G25MACSRDAuYuXvLyDo9sgncypHbo1VKHUAAbYqA0PEOwc1CDCHzrF4Y5O8+jSuF3sPJLU4rx+QEXMP6fU8iXJS0Chzay6cy8p+/CmZsPMaiJG/o2KqloLPPIEb98UhVNFESOUNQxK5EACSQrArK7VCrXFHj+0rQL7D+oHoo5+EMzdQgFsCE3Ig3PEOwc1CACQ1cdx9IDV+KEB9p95jY++e2ANCNxMC5T2lQ2md3r169RepSflHnux44V8V5F5Znn6kzxR/jdJxjW3B096ha3yfzYKQmQQNImIL9hqlMyBwLO3pEWs/JLL1QpnM2hF0YdQgFsiIHS8AzBzkENItBr8SFsPH4DbSrnx7T2FaVZ3IiKQY1JptjAq3rVROVC79hkdjejY1D93xjEq3vVRCUV43T57QB2nbmNjtUKYnJbD5vMj52SAAkkXQLiAbvK+G24++gZRrYogwn/hEK4AiuNN27kyqlDKIANsT8aniHYOahBBDr9HIR9FyLxWe2i0o+EKOKHw2PMFjyIeYHJbcqjo6dtkk0cvHQX7ebuk8YMGdkE2VT45fmuO4kFgZfgWTQbln2h7PCcQYg5LAmQgAEELt55hAbf7ZRGFiEdxUOzSKAz9QMPtKta0IAZKR+SOoQCWLm16FiThqcjTHbl8ASa/RiAUxHRb/ngfjAnEMGX7+HTWkUxqqVtkk2IWL4ipm/GNCklV4sU4oi2wrJw3yWMXHsSOTKmQfAIZeHTFHbNaiRAAsmAgPz9kjaVC477NkXT6btx4c6jJOE2RR1CAWzILUjDMwQ7BzWIQM1J23E9Kgbj3iuLj72KxM5C9g2uXSIHFnWvbpPZTdtyGjP8z6FM3szY2L+OqjH2nL2Dj37dL7U55uuNzDbyU1Y1KVbWlcDxq1H4PfAietUvjhK5MunaNztL/gTk7zD5LZGcQfLL+sXxPx93hwZAHUIBbIiB0vAMwc5BDSJQZpQfHj97iRmdKqFVhXyxs1iw9yJ814ciV6Y0ODDcNjus/f86jLVHrqNZuTyY81EVVQSu33+CmpP9pTZre9dChYK2zVinanKsrAuBj3/dLx1calkhH37qVEmXPtmJ8xAQO76nbz6ALHjlA3GdPAtiUhvHPjdAHUIBbMidSsMzBDsHNYCACCdWaoSfNPLCzzxRp2TO2FkEnruDzvNNO6xHRjVB1vT6x818f/ZeHL5yH1/ULYahKhNuvHr1GmVHb8aT5y8xvUMFvF/JthnrDLg8Tj3ky1evUWHMFjx8+sIu4ficGnYyXLxI6V5x7Ba8Njv0JqdQ9ymbB3M/VvfAbW9E1CEUwPa2OWk8Gp4h2DmoAQRuRcfA898oDOv71Eb5AlliZ3H7wVNUm7BN+m9xyEy8RtS7VBm3FZGPnmHC++XwYfXCqruX/Zf7NSyBgd6lVLdnA8clIPzSxfWVS+A3DZEvazrHnTBn5lAERIQYcehNFPmA7cSNp/Dz7gtJ4uAsdQgFsCE3FA3PEOwc1AAC5hnVAoY0QMFs6WNnYR5CaHzrcviohnqBmtiSxM5eudGbpSqLPquO2iVzqCbQe0kI/jkWgXc98mJW58qq27OB4xJYFHQZI9aciJ3gzM6V0MLjPxcdx505Z+YIBKZvPYMft59FsRwZ4D+4vjQlOSmGW+6M2DKgniNMM8E5UIdQABtioDQ8Q7BzUAMImKcJje8gWYd5+7D/4l108SqMMe+V03WGodej0XyGaYfvTfGtdKDvt5zGTxoP0Skdg/WMITBw2RGsCrkWO7gto5EYs0KOaksCsv9428oF8H37CtJQIuGPOBiXM1MaHLTRuQa91kQdQgGsly2p6oeGpwoXKydhAptP3sAXCw/B1SUFzk1o9lYYspFrTmBh0GXUKJYNf/XQN9au34kI9FwUgpQuKRA2zgcpXV1Uk1x9+CoG/H0U6VK54uSYpnBxUR5GTfVgbGBXAvWn7sClyMdI7eqCZy9fSYccxWFHFhKwRMDcf3zi++XRubopjrn5d87ZeL7vLPVrz8+pQyiA7WlvsWPR8AzBzkENILDsYDiGrDyG7BlS49DIJm/NQIhfIYIT+tyaKc/bdR6TNoWhcPb02PV1A01dHQm/j9az9kpt9w1tiLxZ6COqCaSDNbrz8Cmqjjf5n4tMf38dDJcelE6MaYq0qVwdbLacjqMRMPcf3/xVXZTKYwqhZ/7Gy5Yp3vXgQR1CAayHHanug4anGhkbJFECP+8+j4kbw1AsZwb4DzL5yZkX8x8MkWxCJJ3QqwxffRyL919BnZI5sPAzbXGGxUlvESlAlMXdq6NWCfV+xHqth/3oR2DLyRvosfAQRF4UIWC8p++WOrfVYUz9Zs6eHIGA7D+eKU1KHB3tHftmKLEzD44wb/M5UIdQABtikzQ8Q7BzUAMIfOsXhjk7z6NyoaxY1evt18v3Hj1DpXFbpZkt+bw6ahbXT2DKPnofVi+ECe+X17z6quO34s7DZ28l8tDcIRsaTmDSplOYt+sC3PNkwqb+deA1yR83omOk5AUipisLCSRGYODfR7Dq8DXUdcuJPz/1jK1660EMPCdsl/5bpEb2KOC4scOpQyiADbnLaXiGYOegBhCQMyU1cs+FX7tWi3cGIhSaCIk2plVZdKn5X6Y4a6dbd8oOXLn72Oq0pO3n7sOBS3fRrVYRjG5Z1tppsb0DEGg3NxAHL92D/HDUe3EI/jkegcalc2N+l6oOMENOwZEJyP7jAxq7oX/jkrFTffbiFdxGbJL+e0G3aqhfKpfDLoM6hALYEOOk4RmCnYMaQKDX4kPYePwG2lTOj2ntK8Y7gw/nB2HvuUjpIIk4UKJHef7yFdxH+kEcVpn7URX4lMujudtvVh6TfETrueXEH2a7PZo7ZENDCQiRUt53M56+eIVp7SugTeUC+HXPRYzbECr5ogtXnBTCN4KFBOIhYO4/Hl94xfKjN+PB0xf4oUNFtK6U32EZUodQABtinDQ8Q7BzUAMIdP4lCIHnI5FYiKkx60/i972XUK3IO1jes6Yus7wS+Rh1p+6Q+trYrw7K5MusuV/5MF2hbOmxe4i2w3SaB2dD3QmYH2zc9XV9FM6eAeZ/2zm4PorkyKD7uOwweRAw9x8/NtobmdKmirOwOlP8EX73CUa3LINutYo67KKpQyiADTFOGp512B/EPEealK5InVJ9WCvrRmZrtQTkTGqDmrihb6P/XhWa9yPHzsyc1nSgRI/dt4Czt/Hxr6YsTeJkf8Y0KdVOPbb+1tCb+PzPYIgIaKfG+Ui2x5J0Cci7vTkyppZitQp7M98V/r5dBbStwrTXSfcK23bm5v7jfl/VfWuwVjP34NjVKPRrVBIDm7jZdjJW9E4dQgFshflob0rD085OvH5qMHUncmZOI53eTqUhtqv20dlSLYGak7bjelRMogfIDl2+h7ZzAqWu9w9rhNyZ06od5q368iltIXKCR7wdfk3NAOduPUTjabukJlsH1EXJ3KaQRyxJk4Ds79ukTG788sl//r6yX7CerjhJkxBnnRiBN/3H36wr0iOLNMkf1yiMca31Te6j55WhDqEA1tOeFPdFw1OM6q2K5vnXl/f0QrUi2bR3xpY2J1BmlB8eP3uJGZ0qoVWF+NPMRsc8h4evKdTYws88UadkTqvnNWnjKczbfSHB6BNqBhC7g6VH6eNPrGZc1tWfgEi/XWPSdtyMfopvmrmjZ73/Ij5Y2tnTfzbsMakREN8F5Xw3S28MZP/xN9fw1V+HsebIdbTwyIuZDpw+nTqEAtiQ+4+Gpx37msPX8NXfR6QO+jUsgYHepbR3xpY2JfD0xUuUGuEnjSFCBYmQQQkVr0nbEREVgxHvlkb3OsWsnlfPhYfgd/IGWlfMhx86VrK6P/nUN8NkWY3S0A6u3X+CWpP9pTm8+QBt7tspXHEyv+HbaejEObhDEIjPf/zNifmuO4kFgZdQu0QOLOquLf64PRZLHUIBbA87e2sMGp527L/vvYgx60OlDioVyorV8cSW1d47W+pJ4FZ0DDwnmmJiru9TG+ULZEmwe/m1YYeqBfHtBx5WT0P2PdbLD+/TBQfhH3YL7aoUwNR2FayeHzswhsD6o9fRd+lhpHJNgeO+cbO+mZ/u1+tNhDGr5Ki2IhCf//ibY/247SymbzuDMnkzY2P/OraaitX9UodQAFttRFo6oOFpoWZqM23rGczYflb6tziUdHikN7Kkj3sKV3vvbKknATVZkSb8E4pfAi7q8lAjXnOX992Ch09fQK8DTSJElvjxq1L4Haz8Up9IFXqyZl/KCMi7cxULZsWa3m8nZpF3+r9qXBJfNXbcA0zKVstaehNIyH/cfJyF+y5h5NqTyJclLQKHNtJ7Crr1Rx1CAaybManpiIanhlbcuiPXnMDCoMuxf5z7UWX4lMurvUO2tBmBAxfvov28fVL/x3wTf6W8LDgcQ1Yck6I1HPe1LhJE5MOnqDJ+mzSuXn7ii/dfxvDVJ5AtQ2qEjLTuUJ3NgLNjiwTkE/qf1S6KkS3KvFV/4LIjWBVyzar02RYnwQpJkkBi/uPmC5LfMqRL5SpFjXHUQh1CAWyIbdLwtGPvsyQEG45FxHZgbZpb7TNhS0sEZJ9KV5cUODehWaLhzY6G38d7s/ZKXe79piHyZ01nqfsEPw+5cg9tZpuiShwY1gi5dIgqse98JDr9EiT1eWRUE2RNn1rz/NjQGAKPn72Q3gyI5CizOlfGux5vPzjLDzqZ0qTEkdHeELbLQgKCQGL+4+aE9py9g49+3S/9KWycD9KmcsywidQhFMCG3Nk0PO3YP5q/H3vO3ZHcH169BpicQDtLW7dcdjAcQ1Yek7JrHbKwa/ro6QuUHb1ZmtLvXauhgbv2FKLyQcm0qVxwaqyPLnGFb0bHoPq//syretVE5ULv2Bof+9eZQNCFSHT82fQQs29oQ+TN8vZD1qmIaAj/cVH8vqoD9zzaE6joPH12ZzCBdUevo18C/uPmUztxLQotftqTqJ0ZvBRpeOoQCmBD7JCGpx37uzMCcPJ6NGqVyC6lzxVFzuakvVe2tAWBn3efx8SNYSiWMwP8B9W3OIScQWloM3d8YRaeymLDNyoIH3HhK+6WOyO2DKintnm89cXrz3KjN+PRs5f4rl0FfMBECbpwtWcns3acw9TNp6W3C+ItQ3xF7A5XGGPyHxdpuUVMYBYSEAQs+Y/LlK7ff4Ka/0YasTYLpS3JU4dQANvSvhLsm4anHbucWEH4732/5bQUY3Z863L4qEZh7Z2ypU0ITPELw+yd5xXH4v1swUFsD7uFtpUL4Pv22iMtDFp2FCtDrqJx6dyY3+W/RAfWLrLFTwE4cS0avRsUx9dN3a3tju3tTKD7Hwex7dQttKyQDz91Sjg03se/7kfA2TtW26Gdl8fhbEzAkv+4PPyTZy+luOGiLOleHTVL5LDxzLR1Tx1CAazNcqxsRcPTDrD0SD88ef5S+gETr7qFYPIpmwdzP66ivVO2tAmBoauOQ6Q5buieC791rWZxjMmbwjB313mUz58F6/vWtlg/oQpypqaEDjpp7Vi8/hSvQZuVy4M5H9HetHI0op3Ywa88bivuPX4O35Zl0LVW0QSnMX3rGfy4/SyK5siAHYMtv7kwYj0c074EzP3HZ39YGc3LJ37wutSITXj64hVmdq6EFh7xJwCy7wreHo06hALYEBuk4WnDHvP8JdxHmp6sF31WHWdvPZBiAmdKmxKHRzZBSqZF1gbWRq16LT6EjcdvoE3l/JjWvqLFUVYfvooBfx+F8N0NHeMDF40HkKpP3CZl+hr7Xll84lXE4rhKK8jCqFTuTNg8oK7SZqznAAQu3H6Iht+b0lmv61MLHgWyJjir3Wdu45PfDkifHxrRGNkzpnGAFXAKRhIw9x8PGtoIebIknq5dTuwjUiGLlMiOWKhDKIANsUsanjbsEVFP4DXJlMVpQ9/a0unaxtNMP2o8mKSNqS1bdf4lCIHnI/FpraIY1fLtkFNvjm1+eESrX7f5Q9Lv3aqhQSnth+nenN/aI9fQ/68jSJ3SBWFjtQt0WzJn3/ETWB4cjq9XHJMerkQCjFSJPCyL1NzCD/j1a+CXT6qiSZncxOrkBJT4j5sjkhPxDGrihr6NSjokPeoQCmBDDJOGpw176PVoNJ9hOqEtDrGIQOPisIFIoTuwiRtE1i8WE4FDl+9iR9htdK1VBDkM2sFq/mMAQiOiFV8bIV7LjPKTontoFR7myTf8B9VDsZwZdTOJ41ej0HKm6XR3wJAGKJgtvW59syPbEhi66hiWHghH9aLZ8PcXXhYHazp9N07ffICe9Yrjm2b097YILJlXkM8nWPIflzGoffg3Ah91CAWwEXbH8CMaqQeeu4PO803xFUV4q3SpXfH18qNYfugqqhV5B8t7MkOXOMUuoiD85H9WEpL9GpbAQO9SGolb10w+sKjGFaHBdztx8c4jfN20FHo3KKF6AttCb6L7n8FIkQI4Pa6ZtFurVxGRAUQkCFH+/NQTdd1y6tU1+7ExAe/pu3Dm5kP0ql8cQ3wsC1rZf92zaDYsUyCYbTx9dm8gATX+4/I05Yxx71fKj+kdLLt/GbE8CmAKYCPsjgJYI/V/jkWg95IQpEnpgtPjm0m9yLEZU7qkwOFRTZAprfOmRb4RFYP+fx3G/ot3Ywm3qZQf0wz6Aha7uSJKx4xOldCqgrKDID3+DMaW0Jt4r2I+/Ngx4ZP6CZmQSFcs0hYnFupKo/lJzWT/YksHqawZg231JRD15DkqjjW5NPzapSoalbbs0rDi0FUMXn5U+q45MSZxlwl9Z8veHI2Auf/4+j61Ub5AFotTHLHmOBYFXUH9UjmxoJunxfpGVKAApgA2wu4ogDVSFymQRSrkPJnTImiYKce6edpbra/NNU7HoZrtCLuFQcuP4u6jZ9K83kmfSjrxXqdkDiz8rLrd5/r0xUuUGmE6sKhmt1SEtvvJ/xxK582MTf3rqJ736LUn8Me+y/Aqlh1Le9RQ3d5Sg44/70PQhbvo4lUYY94rZ6k6P3cAArvO3EaXfw+1iTTWIp21pSLeQoi3EaKs7V0LFQomfGjOUl/8PGkTkP3HRWpjkdI9Mf9xeaXy95iwG2E/jlgogCmADbFLGp427D9tP4vvt555SxzJ8VmdUZQ8e/EKUzeH4ZeAixLUDKldMeH98rgU+Qg/bDurWUhqu0L/tbr1IAaeE7ZLf7B06t58LHlHX7guhI5pqjqyR9ffD2Dn6dvoWK0gJrf1sHYZb7Uftvo4luy/YtiDhe4LcoIORVIU4RZULEcG+CsMayZee1cZv016oBzVogw+rZ1w2DQnQOjUS1TrPy5gyW+iHDlTKXUIBbAhNzYNTxv2setD8dvei6hZPDuWfP7f7p4cP1ZpxjFtozteqyuRj9F3aQiOXo2SJlc2X2bM7FxZil+6KOgyRqw5gZyZ0uDg8MZ2n7z5YTQ1B8bCbkTD5wfTQcftg+qhuMpDbA2/34kLt7X7EFsCNT/gAsb/c8pmLhaWxufn6gnI6dNF9j6RxU9p6f5HMLaduol3PfJiVufKSpuxXjIjoNZ/XCxfDukoQnSKqCOOWKhDKIANsUsanjbsA/8+glWHr+Hd8nkx68P/fpD2nruDD/89HCeiQwj/z+ReNhy7jqErj+PB0xfSUrvWLIKhzd2RJqWr9N+bT97AFwsPwdUlBc6Ob6Y5pq5Wjgcu3kX7efuk5uK1YWaFvtliR1v4Dr949RpzP6oMn3KJB5w3n584ACgSpTx7absA9P5hN/HpAtMhO3EQU4TiY3FcAsImPHxNKawntSmPTp7KUxvP3nkOU/xOI2+WtNg31ORyxeJcBIT/uAiJJ4pS/3FRd8fpW+j2+0Gp3bkJzVS/ybIHZeoQCmB72NlbY9DwtGGXX29/WL2Q9JpfLiJ8ljjkEvP8Fb5tWx4dqin/kdM2E+NaibWK5B8iw5ooWdKlwtQPPOBdNk+cSR26fA9t5wRKfzMimP+WkzfQ418BLn4AUgjFqLCI2M7nbj3EgMZu6N9YeWi7a/efoNZkU5xopYdVFE4ptpq5b6jwURa+yiyOS8A8dOKWAXXhljuT4snuvxCJDj8HSfUDRdhFJ3iwVgzHSSruPH0LXf8Vskr9xwWaI+H30XrWXolS8IjGhoWiTOwyUYdQABtyG9PwtGF/b9ZeHA2/j74NS2DQG6G9ROYmkcEpOb+uPHvzAfosOSzFJxWlauF38GOnSvHueAv3iLpTd0j1Nn9VF6XyKP/h13Z14rZaFhyOISuOSQeOxA+HmiKHEFJ7Lfedj0SnX0yC5ehob+nhQO/y4uUrKRuh2KFWkhJV7/HZnzoC8sFZ8Sr66ChvVW9Cnjx7ifK+m6Vr7cgpbdURYW01BGL9x3NmgP8g5WmxL0c+Qr2ppkOUWwfURUkVD15q5mdNXeoQCmBr7EdzWxqeNnT1pu7A5cjHGNmiDD5741CK7Jspoh8cGtFE1Q+dttnYr5U4kLM8+CpGrTsh7XKLzVQRz1TskCaU/lnkri8zyhSzVqSNrl0yh/0mDODn3ecxcWMYtPhl/7DtjHSAzy13RmwZUE/xvP8+eAX/W3lcEr5CANuqyH7Gg73d0Keh8h1qW82H/SZMQHabqueWE398qj4c1Xsz90g+9t1qFcHolmWJ2skIyP7j7aoUwFQV/uMim6CHr8l1QsSRFvGkHa1Qh1AAG2KTNDxt2MVuzIOYF5jWvgLaVC4QpxPzw1O2ev2tbdbWtXoQ81w6zLb2yHWpI5HV7YcOFRUJWjkOr6jfulJ+6yaisvUUvzDM3nkelQtlxape6sIAbTwegV6LQyBiO4eO9VGczEIe06NAFqzrU1vljJVXlw9HtamcH9PaO2aQe+WrSd415Ydmte40MpUx60/i972XUKFAFqy1oU0l76uQNFdn7j8+uU15dFThPy42LUoO3yS9PZj3cRU0fcNFzRGIUIdQABtihzQ89djFq+cSwzdJDX/vWg0N3HPF6UR84XhO3I7bD55qziKmfla2bSFS74ooD5ciH0sDiZi+QnCJyA5KSt0pO3Dl7mOMeLc0utcppqSJbnXkcGEN3XPht67VVPUr/H+FH7Aoavw2+ywJwYZjEWjhkVeKhmGrMnHjKfy8+wIqFsyKNQ4a49NWa09K/YrvgmoTtklT1voWRBw2FW5H4mFMnOYX2SdZnIOANf7jglDV8Vtx5+Ezhz2XQh1CAWzInUzDU4/9zsOnqDre9GMmRIcQH28W+XWnrZIgqJ+1thZCzItdp0mbTuH5y9dSJIdB3m7oWbe4KteONrP3IuTKfXxRrxiGNiutbTIaW/VafAgbj9+Alkx04mFHuG+IaA4/daqElgqzyLWauQfHrkYpTnercWn468AVfLPqODILv9LR3qoO+Gkdk+3UE5AjobikMPmEa8kSGRH1BF6TTAcr/+5RA9WLZVc/EbZIkgSs8R8XC5YP8/7Pxx1f1i/ucAyoQyiADTFKGp567OduPUDjabulhru+ro/C2TO81cmqkKsYuOwoUrmmkH7w0qdOqX4gg1vce/QMX684JsUfFUWEdJvRqSKqFFbvQ/bFwmBsPnkTbSsXwPftlcc/1QNB51+CEHg+Ep/WKopRLcuo7tLnh90Iu/EA/RqWwMA3Djwm1JkIVyTCFtk6Eoh5iDcjImyohumkDSZtPIV5uy9YnQzGa9J2RETFwFGFjL0ur3gwVxPNxV7zstU4A/4+gtWHr0Gr/3j7uftw4NJdfFG3GIY2t+8GhBIm1CEUwErsRPc6NDz1SM1FR0In/G9Fx0huEKL83q0aGpSK6yahflT7thAhzt6dEYDztx9JA3uXyY0pH3gga3rLqVvjm+nw1cexeP8VzV/g1qy++Y8BCI2IxsAmbujXSP1BsX5LD0NkhfMpmwdzP65icSpRj5+jwljToZOln9eAV3Hb7dSZv1pf0dMLVYuofzixuCBWsJrAB3MCEXz5Hj6qUQjjW/8XNlFtx72XhOCfYxFoXDo35nepqrZ5kq8vvpfem7kXGdK4SgmInCX2tew/rvU7rMefwdgSehPtqxbAlA/suwGhxOioQyiAldiJ7nVoeOqRmid2SCyurLxzKKJEiGgRSansOnMbXX47IE15TKuy+MSrsFU7LtO3nsGP289KGeL+6VfHrihEPF4Rl3fse2IdRVSPPdP/LL7bckZx+lrhL91y5h5pHFsnQxE7YR5jtkgHMqe09UD7agVVr48NbEtAJFQp57sZ4v+nd6iA9yvFPTSrZvTf9lzE2A2hUkg/sePvTLuggpN5PGS1h8HUcHakuuYPuYu7V0etEuqj6PxvxTH8HRzusA9O1CEUwIbcczQ89djlEFc5MqZG8IiE48pO+CcUvwRcVB1CS/2M9G8xfkMo5u+5iFK5M2HzgLpWDyD7sOXOnAb7h9k3HbIcgeLHjhXxXkX1ESjkRBrCf1NEgrC06yQfVkrt6oJT43wkv2lbFjk8lhH+1bZcV3Lp+/CVe3h/tikRTEIuU0rXKmKPixjkouwYXF9KNe5MZdPxCHy5OERacolcGbHlq7qqziIkRVbm/uPHfJsiYxr17nSTN4Vh7q7zUrz2FV/WdDgM1CEUwIYYJQ1PPfY5O8/jW78w6Qt428CEY8Oa76LuH9YIuTOnVT+YQS303r32OxGBnotC7J4O+VVB1QsAACAASURBVOmLlyg1wk+i+OennqjrllM10Ut3HqH+d6ZA8hv71UGZfIlnXJu14xymbj6tKe6w6skBkP0DhZvKz58432txLczs2UaOCy4emA8Ot27X9vnLV1JCDBGD+7t2FfBBFe27yfZkoNdY8oO03N9vXauioXtuvbp3yH708B+ft+s8Jm0KQ/GcGbBdRRINewGhDqEAtpetxRmHhqceu/yFVK3IO1jeM+GnaZG9SRyGEhEEktKPlS38l4Mv3cUHc/dJsNWk8VR/deK2uPUgBp4TTL7Y6/rUgkeBtyN2WBpDxOAUu8hPX7yS4h5bimP8zcpj+OtgOOqXyokF3dQnPLA0nzc/n7H9LESWKEsPZGr7ZX19CMhRSPR6QGk/bx/EOYROnoUwqY12f2J9VmffXuTENPKoNYplw189vOw7CTuPpof/+LKD4Riy8hiyC9cZldkw7bFc6hAKYHvY2Vtj0PDUYx+y4iiWBV+VDoZZ2nH7cH4Q9p6LROuK+fBDx0rqBzOghRzBQrzCPzK6iS4RLMx3UdXE07V2+SJlc5Pppogdu79ugELZ02vqssVPAThxLVpRWLNOPwdh34VIdPEqjDHvldM0nppGssuFiDgSNq6ZzV0u1MzN2esKH+0ak7bjZvRTDG3mji/qWR+CSn6d7Z4nE/y+st49KSldo1FrT+DPfZeRPrUrHj97KU09OSUbevNa6OU/bu7GdW5Cc4dzG6EOoQA25HuIhqce++d/BmNr6E10qFoQ337gkWgHsruEyJp2YFgjh/viiW/ytohh/PDpC5QbbUqHvKR7ddTUcJBD/ZWCtFMmdsxESShih5J+ZSaNS+fC/C6JJ9OQD93FlyZbyVhq65y8HoV3Z5gO3VnrY6p2bNZPnMDVe49R+9sdUiW9onSI7x7xHSTSkAubzpw2ldNcht6LQ/DP8Qi0qpAPR6/el9LRi3/P6JQ0NhfUXihz/3FrHuDN38AdHeWNLOkdy2aoQyiA1d4butSn4anHKL+SUnLo6MS1KLT4ySROlPiPqp+Nvi3Ms9gN8SmFXvVL6DKA6FcklHjy/CW0HkbTMhF550McREssYoelvuUHmULZ0mP3kAYJVhc7NqVGbsLr18D8T6qicRnb+yc+fvZCYitKUgy5Z4l9Uv5chM8TYfTE7rzI3mbpAKWStUY+fIoq/ybi0erXrmQcR6zTYd4+7L94FyKyTpHs6TFy7UnpjYd48Cvwjra3O464TnlO//mPp8HB4Y00R/0wz2i5c3B9FHGww5PUIRTAhtyHNDz12Bt9v1OKj6vklearV6+lFKiRj55hWHN39Khr/StQ9TNW3iLsRjR8fgiQGuj9arHOFH+E330ihYQTP2D2KMuCwzFkxTEpbJTwPdZa/MNu4tMFpl23k2OaJugWcuH2QzT8Xn3qZK3zktvVnLQd16Ni7MrW2jk7Q3vfdSexIPCS7qmqG3y3ExfvPEL/RiUxoImbM6CU1thk2i6cvfUQ4uG8W82iqDl5O+49fi59nyS1UJNKLprsP960bG7M+1j7AVfzh6ZVvWqicqF3lAxvtzrUIRTAiRrb+PHjsW3bNuzcaTqNLkq3bt2wYMGCOO3y588PYUxKCw1PKan/6lUZt1UStErjrvZdehjrj15HnZI5sPCz6uoHtGMLecfhnfSpcGhEE11dNt6fvReHr9yXUnGKTFb2KL/svoAJG08pjuGb0JzC7z5GnSmmV9mJHabbcfoWuv1+UKoXNs5yyDS9GMi+5tYmWtBrPuzHRKDlT3tw/FqU7gJt0LKjWBlyNUl8p+hpC5XHbcXdR89iMyx+v+U0fvI/hwypXRE4tBGypHOsV/vWrN3cf9zazROR0r3E8E3SdBwxcgZ1CAVwggR++OEHDBw4EHXr1o0jgKtUqQIfHx/07ds3tq2rqyty5lQe6omGp+4rSuzolhi+Ea9eAz9/XAXeZfNY7EDehUyT0kXy2dPjNajFQTVW+OS3A9h95jZaeOTFzM6VNfYSfzPZd1qEbhJRMexRpviFYfbO86hcKCtW9aqleUhx3UX4qUfPXiYa0eOPwEsYve4k7B3veOSaExAhomoWzy5lyGIxnoBwTSnvuwUiisjsDyujefm8uk1qyf4rGLb6ODKlSYkjo72d4uCjEHElR8R1LxJJIoTPvYi0o+SNnG4XwA4d6e0/Lqdn/75dBbR1sPB51CEUwG8RuHbtGrp3746AgAAULFgQuXPnjhXAL1++RMaMGbF06VK0bt1a8+1Iw1OHzjzN7fKeXqimIPVsRNQTeE3ylwZa9Fl11C6pPpOPullqqy3SjFYcu0WKMfpt2/LoUK2Qto4SaDV01XEsPXDFbuHBxDSESBBioaF7LvzWNfHDa5YWKxIQiEQEPeoWw7DmpeOtPm5DKH7dcxGWQuRZGkvt53KGsDyZ0yJoWCO1zVnfBgT2nY9Ep1+CpJ71jgNu7qrk91UduOdJPDa1DZZn9y7NM6Kt7lUTlf59jS+HHRS2L/zzU6d0sfvcbDHg2iPX0P+vI7r5j9efugOXIh9jxLul0b1OMVtMWXOf1CEUwG8RWL9+vSRwhfvD2LFjcenSpVgBHBYWhtKlS+P06dNwc9PuA0bDU3fPmofzEkkwROxVJaXxtF0QBxG+qFsMQxMQT0r6sWWdvefu4MP5+6UhbJHCd9qW05jhfw7l8mfGhr72SYcsnxpvUyk/pnWoaBW+r5cfxfJDVxMV8N3/CMa2UzfRtnIBfN/ePrvcYlE7T99C139dL4SPcgYN2aKsgsPGbxGQE6Lkz5pOup/0LGJXuaJIgf30BSa8Xw4fVi+sZ/cO2dfpGw/Q9AdTSMOAIQ1QMJvp0Nu5Ww/QeJrp79ammnakhcv+45UKZcVqK95eyWtqPWsvjoTfR+8GxfF1U/u4oCnlSR1CAZyorXTt2jWOAF6+fDk6dOiAnj17YtOmTXBxcUHz5s0lsZwlSxaldif5C4vd5fDwcBQooH9WIbFTsfH4DcXzsVRRnMJvWzm/5tOwlvq39Ll5WJpDIxoje8Y0lppIn8tfZmXyZsbG/vYRf4omZlZJji9aLGcG+NsgW9Cf+y5h1NqTsOcuZedfghB4PhLdahXB6JZl1SKJU1/2j05M0HhP34UzNx9iYBM39GtU0qrx1DS+EvkYdaeafJQ39K2NcvmVfweoGYd1lRP4bMFBbA+7ZbMwXR//uh8BZ++gTeX8mNbeuoe7hFZ1KiIaodejpTFSiBOgBpbAc3fQ+d8H9Dcf8j5dcBD+YbdQWny/9qtt+Fz1wCT7j3evXRQjWpSxukuZUefqhTDxfcdKoGJrHWI1PAM7SPFaeIM7eXlTAI8ePRoTJkzAxIkTJeF79uxZDB48WBKz/v7+kiCOr0RHR0P8Ty4RERHw9PS0mQCWX+PoefmWfl4DXsWz69ml4r7kaACigQirldJV2eu27adu4rM/gqVxRDrUnJmUCWfFE9Oh4rszAnDyerTNEjhsOh6BLxeHSK/0zoxvZpcfqeY/BiA0IloXQWqe2vq4rzcyvRF/VXxNlR7lJ7mQKMkYp8Mli+1C7AiWHukn+UL+1KkSWlbIp2f37EslAWELlcZtxf3Hz+Hbsgy61tI/6omcFU2EA9v5dcKh+VROPbb65chHaPZjgJRwYkG3aqhfKpfWrnRpJ4eUS5vKBafG+sT5/jB3N3FkNzOlIMz9x+d8WBnNdPAfH7jsCFaFXEPz8nkw+8MqSqdil3oUwAljpgAG8KYAFl+wUVFRyJr1v9SuQUFB8PLygvj/6tXjjzbg6+uLMWPGvEXbVjvA4kDV91vP6HITnboeLf3Ai6dX8RRrRFl56CoGLT+KzGlT4phvU8VTePT0heRf+/zla7vGwVU6QfMwObaKX3vw0l20+zcd8pFRTZA1fWql09NcT05KMfa9svjEq4jmfkRDc1/u+EIJmaeQNiLUkBwiakBjN/RvbL/dZ6ugJtPG528/RKN/w+HpHU5QRhZw9jY+/vWA9J/BIxpDJNvRq4hDnx1/DsKBS3elLke1KINP7RS6MKE1LNh7Eb7rQxHfGxjxe9hq5l4p4kY9t5z441PbpyDXi3V8/djCf1w+n+CI6aMpgCmAE72f3hTA8VV+9OiRdDBuxYoVaNu2bbz92XsHWM8vCbEbIV7J2fv1svka5NfgWnZdREYykZnMnlEQlPKXd1dSuqSQTpVntIEPqYhbKuKXirJtYF2UyJVJ6fQ01ys7yk+K3KBH8g3xI+sh/C5jXsR7SNBc4Ktxj9G8uDcafrEwGJtP3kxSabf1Wruj9SNHfkmXyhXHfL2RSuGbIjXreBDzXLJH8X70l0+qoomOSVfk7zl5Pr3qF8cQO4UuTIiBHPLMo0AWrOtT+61q5m8bN39VF6Xy2P77Rc31UlPXFv7jM/3P4rstZ+CIKbQpgCmAVQngzp07IzIyEps3mzJAiSJiBDdo0AChoaHSATklJSkZnuzz9nGNwhjXupyS5ele57vNpzFzxzlNge3lLyARIitoqPasProvCoB8wMuzSDYs6+lliyEgfrBFWChR7OHGIrKyuY0wxb4UO0JiZ8ja0nZOIA5dvodPaxXFqJZx/fLktwPi4UG4SNjbZ/JbvzCIjHUJCQRr1872ygnIkQlsvdvm88NuhN14gJ71iuObZvocbBKHdZvPCIC4f+TSrkoBTLVT6MKEKMtRZBqUyonfu729w/v85SvUn7oT1+4/cchNBuXWA8j+unqmeV68/zKGrz5h9xCNStadlHSIkvXoWYcuEPG4QGzYsAGtWrXCuHHj0LFjR5w5cwa9e/eWXCAWL16smH9SMryBfx/BqsPX0KxcHsz5yBgfJjmsVkJfwomBFydwxUlcUbYOqIuSuR1jh0LsbNac7I+IqBib7q6LcdxH+uHpi1eY0amSdDjIluXWgxh4TtguDZFY8go1c5B/hONLajJt6xnM2H5WOoizyYCDjvKuo1ECXA3H5F5Xdkex9Yl7+ftIrwdXEWu37dx9Urg/cU5BhPMTh5i1fN/pfY3lNxyJvUGTd67FOYO9/2uIXJnT6j0Nm/dn7j8+plVZdKlpneuWPOGNxyPQa3GIFCbu9Li4PtQ2X5SFAZKSDrE3KwrgeASwuAgrV66UDsKJkGjCF1jsCosoEGnTKr/pk5LhTdx4Cj/vvmD3GKvmBi+nptRy8locVBKZjKKePHeolLXmYYRs7bsq++Taw6fw7M0HaDLdFB5p99cNUCi7KWySNeX3vRcxZn0ocmVKgwPDG8fp6qu/DmPNkevwKZsHcz+2/wPaoct30XbOPmlOB4Y3Qq5Myr8HrGHCtnEJiPtbJB0Q5dcuVdGodG6bIZLfOogkO8d9m1odA1d+9S4mLM4CHLt6XwpdWD5/Fqzv+7bbgc0WFk/HH8wJRPDle4mGkhRvmWpO8pfCwzmC24YWPub+43pGdAk8fwedfzGFuXS0UIlJSYdouabWtKEAtoZeMnry+nn3eUzcGIaiOTJgx+D6NqSScNedfg7CvguRmlObygLaEXZU5FXKok4c7AsZ2URxZAstF0BOJmGPHydzn1yRgU+PNKnmsZKPjvJGlvT/pV6VUz0bFetZpIgVD1ii/NWjBmoUMyZSiha7SE5tzGMyi/spWwbbHfY0j0u+pnctyTVLaxHnK1rN3CMd1JV3WeXQhXmzpMW+ocYmWBHnB8Q5AktpgSdtPIV5uy9IB5XFnJNaTGxz/3HhSqU00pCl626ePGXP/xqgwDvWbwhYGlPp5xTACZOiAFZqRRrqJSXDWxVyFQOXHZXSfx4fozwCgwYsCTaRfe4Ge7uhT0P1J+3lFKbicMyR0U2QJqWrntPT1Jccr9QeO5dyooj2VQtgyge2TRSx5eQN9Fh4CC4pRMi65nAR/7CymGejejMTYNXxW3Hn4TOMb10OH9UwJjGBiDQiQm9NalMenTyNiZRiJeIk31xO+GKreNrmgMTr8qrjtyHy0TOr3ioJf1/xcCpEcL4saeE3oC4yp02F2Nfmri44Pd7Y1+YevpsRHfMCllL5imgtdb7dgRevXtssBJ0tjVT2H/cqlh1Le+iX1vxmdAyqTzS5hOm5s6wHi6SkQ/RYr5o+KIDV0FJZNykZngip9slvprA/YeN8kDaV/cVj9YnbcDP6qWaRE373MepMMSUssMdBMEvmIH74hGgSsT7tkVFq6KpjWHogXJfUxJbWJu+kiB04sROnRxGCQ+yy3nv8PI4NPHz6AuVGmw6kLvzME3VKWn/gTst85V3oz+sUxfB3rQ+er2UOzt7mw/lB2HsuEvY6OPb5n8HYGnoT73rkxazOlTXhl0X7m/Yb5y3KG288NA2ksZH5gdbfu1VDAwsxiQf8fQSrD19DwWzpsGNQfd12UTVOX1Uz2X+8T4MSGNy0lKq2iVV++uIlSo3wM/w7Kr45JiUdotsFUdgRBbBCUFqqJSXDE7sTIhSaKEa8whHip5RINvDiFWZ/WBnNNQYnl3Oy2/qAjBJ72H8hEh1+DpKq6uUnm9i4lkIZKZmz0jq/7L6ACRtPoViODPDX0WWmw7x92H/xbpyEISJbljg5by+OCTEYtOwoVoZcRePSuTC/SzWlqFhPJwLiEJnw/xWh9ya3KY+OdtiFn7vrPEQWR61uCuLAW5s5gRBnFD6qUQjjW/+XJSxu6ELlqd91whnbzY2oGNSYZNq9VBJX+eT1KLw7Y49UXzwUiIeDpFCiHj9HhbEm//HfulZFQ3d9/cfLjPKTNjv0CAupJ8+kpEP0XLeSviiAlVDSWCcpGZ7562dr/d204BLZecqMMu3yLfm8OmoWz6GlG4xccwILgy47RLgqOaybSDG9e4j+2aTeBPRH4CWMXndSes0aaGOfwqmbwzBrx3lUKpQVq3vV0nSt4mskXz/zEFd+J26g56JDcHVJIb2dsEXcVyULkA8x6S36lYzNOoC58LJXpBcRW1zEGBcl8JuGyJc1neJLEfP8JVr8tAci9Jn4DhDRS8x9Zs1DFxrpV37iWpQ0TzVr/Gj+fuw5dwcVCmbFml417R6WUPFFMKu44/QtdPv9oPSXwyOb4B2d/cflQ8h6RpfQss432yQlHaLHetX0QQGshpbKuknJ8MQORcnhG/HKBoHflWC7eu8xan9rcl8QPxQi3JWWsvnkDXyx8BBSpABCRuj/JadmTvKhtA+rF8IEO+SH/+dYBHovCUFqO/gUyiGiGrrnwm9d9dsNXbjvEkauPYnsGVLj0L+uFfIBTXs9SCR0jeV00yKhySkDhbgaG0xOdcWDrXhAEgewjozy1sXv3BIfIWKF+43weVWbBluOrCO+i/76vAaqv3Fw0jx0odq+Lc1bzeda3N/MDyO+6a+vZmx71pVdUYrnzIDtg/Q/6N3ipwCcuBaNrxqXxFeN3ey5tETHSko6xN7QKIBtSDypGZ580MiIQz7muxD7hzVCbo0xJqNjnqPS2K3SK8eZnSuhhYdt4+EmZD73H5uiBogHirkfVYZPOdu/JjR3uXgzioLeZt57cQj+OR6BNpXyY1qHirp1b74GOQXt8NXHsXj/FcQXH1i3gRV0dPrGAzT9wRT6zX9QPRTLmVFBK1bRi4Dse2rvdLzyg2zXmkXg26qsouUEi9Tk8/ZJmeQ+q11UOkQXX5F3DUe3LINutYoq6lvvSqsPX8WAv49KGSpPKDwALcS7zw8BOC3CIZbJLWXLc/Qi+4/b6pCwnEyqi1dhjHnPmGRS8V2DpKZD7GlHFMA2pJ3UDE+OwjCoiRv6NlIfhcEalAFnb+PjX02H8MSJaGsiOMgZxTpWK4jJbT2smZbmtvIJbxEc4bA44JLuv5Bemju10NA8xuW2gbb1KZR/TLrVKoLRLZWJAiXrvvfoGSr9G25MdoWRf1jstZOe0DzFbmDpUX6SqBFxXBvrmB5XCZv46gi/2C6/H5DcQn7tUk1yE0mupe6UHbhy97FNE8rEx27s+lD8tveiYrcq4c4lzlNcjnwMEa1iY786CR4qlsW1kWcW5AQXhbOnx66vlbtqLQ8Ox9crjklv27YPdOwHQrEhIiJdCP/xb9uWR4dq+kdx6bv0MNYfvS4lIRLJiBylJDUdYk9uFMA2pJ3UDE/26zLiCVbONZ8+tStCx/pYdVWmbz2DH7efRf6s6aQDffZOmysmL2c109tHNjEw5kkCbO1T2PzHAIRGRGNAYzf0b6zvw5IIPXXn4VPIvnT1pu6QxMTQZu74ol5xq2zD2sY1J23H9agYhwkBdfxqFFrONPlvOlr4JWtZm7c3zzy4uHt11Cqh7YyAljnJrkXC9UUkxEiXOvEIOaPXnsAf+y5LIQJXflkTlQq9k+Cw3f84iG2nbqFD1YL49gNjHtblNN+VC2XFKhX+/CLygXBbE+dHjH44tXRdzSME/dOvNsrmy2KpierP5etu9JuqNyee1HSIavBWNKAAtgKepaZJzfDkbFvvls+LWR9qC/ljiUlCn8sHuIRo3ftNQ63dSO3Ms3YZ8apavB4U4diu3nuCfg1LYKC3fuF2EgNjHknD1u4f8qvbse+VxSde+qQTldcm7y6LH1UhgkWKZ+GDaS9XksQY2zPZiJKbYPupm/jsj2CpqpF+pErmak0d+SCkEJXHfJtKr+vtVUTsW69J/tJwf/d425fXfB7myVyU7OrKcWkbuefCrzr60qthM2TFUSwLFtFNcmN+F3WuDPLBUJEtTxwSzJ4xjZqh7Vb30OV7EG8Gpd+HEY1tMk9548URMvuZg01qOsRuRgGAAtiGtJOa4Y3fEIr5ey7Cs2g2LPvCy4Zk3u76h21n8MO2syiXPzM29K1j1djitbDwAxYpO20h0CxNzjyDlL0PiMjC1LdlGXS1oU9h2VF+0utEW4T88V13EgsCL0lpub9vVxF1p5oOR4pXyWXyaTscaemaKf1cjgsrZ/NS2s5W9ZYeuCK9bRBlYBM39LOz65Kt1vVmv/KBsjJ5M2Njf+u+H7TMWd75H+JTCr3ql4i3CxHVQfjFXrv/BO55MmFtn1oWXbnk0IUVCmTB2j7GpEOWk/VocRkTZx1qTvaXwn/Z4m2QlmsVXxu/ExHouSgEYhf/zPhmNjlAuWDvRfiuD5XePFq7iaPXukU/SU2H6Ll2S31RAFsiZMXnSc3w5u06j0mbwiS/NX8bnJJNDKUsevR6fSQLFS27GlZccqmpHMlA7FIdHtXErmG73pu5B0evRkHvQO/mTMwD5//xqSfEoSQ9iyzqhN+02MmWfcPFAR177vzFtyb5QJ69D2IlxPfHbWcxfdsZ6ePWFfPhh46O43uop018MCcQwZfvvRVLV88xEuurz5IQbDgWkWgMaHk3V4gsIX6VvGbX882XVhatZ+3FkfD7ULJjHd8Y8ne3iNwihJ8RSZQsrd0eaadlN74MqV1x0ko3PkvrUfN5UtMhatZmbV0KYGsJJtI+qRneykNXMWj5UWRKm1LydbNn6bf0MNYdvY6WFfJJr3KtLUaK0B5/BmNL6E1NrxStXbc1uzlKxzb3x1zbu5YUC1TPYu7C0r9RScmf2zwsmp5jqe1LflMhwvSJcH1GlxFrjmNR0BVpGh4FsmCdQbuItuQgfE3L+26RkuRM71AB71cqYMvh4u37970XMWZ9KN5Jn0rKfPjmuYIdYbfQbYEpxqyaQ8T2DF2YELQ6U/wRfveJ5nTPVyIfo/53O6SINxPfL4/O1fU/YGbtBbdHkiDzcHLWHuS2dr3m7ZOaDtFz7Zb6ogC2RMiKz5Oa4e06cxtdDEqHLJ/0/8SrMMbqEELGPMvSip5eqFokmxVXUnlTc/cLIwKi/2/FMfwdHA5b+hSeFaGPppvCgdkiw535Yb6y+TLj5PVo3RNuKL+icWsu2X8FIgZyjoypETxCnxTQWuci2skPW+LfmdKkxDFfb0MOfVqzBkttQ67cQ5vZJv9NW9ibpfHF58eu3kermXulqm+eKxBuAN7Td+PWg6fSQ8iqL2sqTg8cJ3ThaPtEi3lzvXpkMJPDIoq3h9sG1LOJi4GS65RQHfl70ZZZHM0PpFoTytOadcbXNqnpEL3Xn1h/FMA2pJ3UDM8805J4lSV8mexVWv60B8evRUHs+A1oYn0Q8TgH0RqVlPwj7VGMPoAnZ2izpU/hQRHjdK4pO9ZRG/1oe03ajoiomNhL9l7FfPjRAV7vbwu9ie5/Bkuhn86Ob6ZY6NjK9uTX13L/B4Y3Qq5MaW01nCH9ymG6cmRMg4PDGxki8J+/fIXyvpsR8/wVvmtXAcIHXC7y4eHUKV3wT9/aKJk7k2JO5qELtw+qh+J2ji395JkptJ8oiz6rjtoltUXXOHzlHt7/9yHFUUIEml+Ebr8fwI7Tt9HJsxBEnHtbFPNIE35f1YF7HmPPK8hrTGo6xBbXJqE+KYBtSDupGd6t6Bh4TjTlhLfFq+3EUNvi8NbQVcew9EA41Ib3scYk5FfkRoVgk1/V2vIgxtbQmxA+1uJE/rkJzW2y2/PJbwcgXinKxZ7RNBK7/uY7gUFDGyFPFmPFpnzfyHO2FKXAGts2qu2Xiw5h04kbaFo2N+Z9rC5KgZ5z7jBvH/ZfvBtHRMmHq8Q4w5q7o0dddWH6zN92GHHtzDNwWnvItN3cQBy8dM+QQ9SWrvO7MwKkN0l6bbDEN96jpy9QdvRm6aOln9eAV/HslqZll8+Tmg6xC5R/B6EAtiHtpGZ44vV9yRGbpED/v3apikalc9uQTtyubRFVQPavs2cyCjkJh1FxPTccu44+Sw5D7EadHudjk92yZcHhGLLimOQPKZJ82KLIEUnkvt/cdbPFmEr6vBEVgxqTTA+J6/vURvkC+scTVTIPUUcKezfCD89evoptYkQWR6Xz1VJPrLH6xO2Se4EWgallzITaTPELw+yd51EqdyZsHlAXkQ+fSq4PkY+eoWrhd/D3F16qE5GYX8NZnSvj3f+zdx1QVR1dd8eu2Bv2jgUEBbEjooKisccWe4saY6zRLyr23jXWGBO7sSQao1FACAnccAAAIABJREFUROwNUEQRBRtiwd4Ltm/NfY65Eh7vlpn7CnfW+tefT2bOPbNn3nvnzpyztwt/xUjx/CKuPwKh9iPtxKgGyK9QgZOMDzx3G33Whgm2tD5AMbXOVacECXzFk1tWROcaxU11V/R38Vou7eSGxs7arqUxp60tDlEEvsJBegCsEDgpw6xx4xH53gfPE7mp5SSHm5hVYE3PavBkxCpAcvOIqhgJ6Jd1rgLfigWkLJviPpYgw3zs8n10WH5MmAPJB82eib0C3S8HLmPKrvMoldcOwT94KcYrpYE0yKZ9tKaTM+YbuQova6aXxKQ+kf1deeIe4Z8zpE0jBMJ9PEthVJMKXNbEHEbFJ5Ra5vInN1dx+svpsQ1B8kr9z91G5vRphYLIEnntFEFET/HNUTMgLt6LmdJYFWPN+/cf0GDufpD6i6YuBbGoo7Zc8sbAJypwDqN3CUV6y7tUQUMnfr8D1aYECS9rU1pVRKfqfAJtuZvMGuMQuXNU2l8PgJUiJ2GcNW68RvMOCPruwxuVw3f1kue7lDB1WV3EqResT9UoLVjnGsUwuSWf3C86WXoCQvJDw/18kMsugywcWHSOvfMM3nP3C6Z4iYDQPGOeKneElonkt9JmSUUlVSbtEU79zH3aejHhqXACSRrhTCbXzzyLfFjsT7k2KLVU+rQGFTZzUmyRgwFyQEDaV25F8Gd4vPDfarnG6XfU9/XLYJhGojmfXiw/yhkTykGSz6+2rTt2DX5/nRXSo4isctHcWdSaVD1ezFrz13e1UZkxa43YQXP8fpoCyBrjEFNzYvV3PQBmhWQydqxx41EVru61SmB8cyeO6PxrOvr2E4FAnrSDI9h+ac4OuIBF+2IhV+deycTH/HUWa49dMysd1eMXb1BpYqDgPhEzIaImrBvlwq1XLh9W9qjG2rxgT5xPlyl9GpyfyCedQ4nzvvMPIPr2U7MT/x+KuYfOvx4XpkB4n8k+NweHtxIMpY6h8rI8X7ak+kL61Z8dgsv3nn8aUqt0HqF4LA2J+BQ2Sl34dbWimNZaWznkZfsvYTpD7ndSVFd7RrBwi9ijdgmMa6bNb0hK0J+98RhNFxrkwolaXSGOxd0dlh/FscsP0NujJPyaOircEWyHWWMcwhYB49b0AJgj0ta48QZtPIXtp28KuWgkJ02LdvTSfXz9i+HanrXYgTglgDeFUr3ZIcL1X3+v0hjhW14L6P7zDC1yCinlUSvXwpjXvjK3eXrMCBbkpMvaZ0XgkLrcniPXMC3Q0+JWISXftobHY+jmCEEcZHZbl09KV+cn+aq6ypaLR9L+5Co86HwC7j57rdYUSLrN1fsvLCag+GFLBP4IM5z8EtxJtX+RXOpOObWg6DK2EFP+icIvB6+gWonc2NyPjfrn3D0X8dPeGGTJkBZHf2yAHFnYp2HJ2VjiNA+iAkfqI3g1WrBJbgjmtKvE6zGy7FpjHCJrgio66wGwCvBMDbXGjTdpZxR+PXQF1UvmFoo6tGi7I2/h2/XhQh4jIRBPSjKvxgeSX1x5YqAg1clTHU1MgbPhm+qoVVoZnZCaudKxlEKMV04hvSXgfcJDT8Ys7VqfBkENHe2xvKv5WAk+KTfmtcPSzlXQaL4hHYJX6ovUvUkLMaX2l9JvSSc3NLGAoiKx9PSMr5zRvqp60YdP1IVFcwrFY1q2oZtPY2v4Dfg6FcCyLlWYPPres9eCPDL57uVZdCbV2c0nr2PEn3yLdqkvRJac7JH65fPjt+5VpbrItZ81xiFcAREZ1wNgjkhb48ZbGnIJM/yjUTqfHfZqJIdMxQXyZ8uIE6O9ma/I2O1nseboNUEHflv/2lwq9+kPIymIOT3OBxnTpWU+D6kGKacyr5xCSik0xLssBnk7SHVLdj9yxU/2IslHZ1UYKduJZAYQn8jnhOQSkpxCczX6skrSXEjxKOFzNQeDS9L5U2lcctKWO4v6PHgH+6xY3sUdmTOY7zNF50gKXckpXzn77BjTtAKTl3UtqAuN7VEifEQEkDpVL4YprdjVSLRbdhQnrj7AN3VKYvSX5k0FWBQcg9mBFz+xd/D8vGpRHyHXf2uMQ+TOUWl/PQBWipyEcda48bYwLoqQABMW74vFrIAL3L6gnr1+C5K3Sa/Td3zvwTxA7b8+DLsib8OrXD6s4pQXKwVL0udf0nc+OYXmrFqXigHPfuYMWMTz+v73U9gRcfNTxT1NGRndpAK+8SzFE4IUbdMbAtZBldkmxPnBZA3JWmZMlwbRnKgLjU2h6cKDOHvjCQYyFguie5OVtL2aJaAHIHUc8mJtr+pqTJkcS0VbSua1wz5ODDkmnUjSwRrjELlzVNpfD4CVIidhnDVuvH0X7qDHSoOmPe98KQoh5XytUSo3Nvbhk3YhzgXuV7c0fmzMLkeX0OyQ6nBCaj+mqSN6eZSUsDv4dRnxRwQ2h8bDu4I9VnRjf0XPg7OZHxrsLVN+aR4pO3K8pcIMPWuXxNhmjqC5yTzVrqT45z45COQanFcKjhQfrKmPuAYicnxDZONAXWgMj1rT9uLm41eqmSyS2ueRW6x0TfutDRPo6lq7FcbcdvxqFoh/JD+cpEixYtVQOmfxOGuMQ1jMW4oNPQCWgpLCPta48cQVs0dH1kfBHPzlkIdtjhAohRpXLCDkMvJqE3acw8rDVwWKni39aqFK8VxMHiUmkw8c4omyMqRQmTiQxAgl7OdxRS/mbF7dsxrqMuJs5oEDL5tiKejTY32Qk8E1vxJfKSMBeZkjL3U09UDL/P2kfoupwsydC68EU3OMEVMXklNDcnqoRRMKZsf4C7m6izq6oqlLIWaPpSehWrDvmHK69ZLDCI97JHxGWB58JPfcvecT0Gu1QSqdqGSmVcEOYmpeUv9ujXGI1Lmp7acHwGoRTGG8NW68hCevBOUl0lhz8hqDqueqkwiOvsNVp508m1D0kPxVQmNEfmSI9CeLvEKaY2afPSOIPC7LIj4l2/O3Q1cwcWcUiuTKjEP/q6/EhNExRE2JqCqRZmlqT0wnmoKxa/efo+6sEKGHOV94nMcF4Onrt5jbrhJauxXBmqNXMXb7OfDKpZeCr/imJczPG3myZpQyLFX3EVMXain48vTVGziPN1AmspbuNWdaR9LNVGdmMK4/eKnJ7Vx43EO0XnJEcCF8jA9ym4ELPun8rTEO0eoLQQ+AOSJtjRuPKF05jN4toPJbd3fUL89fDrnVksM4FfdIE/ow8gXVZukRQRWIFdcxvYq2FOqbvyNuYuDvp8CDPzcm4Sl8Poov7B/uheJ5tDmt4vgxlW36ReJbOI4NEMYRDlgPB+0ZP8jLHCl6I21tr2qo45APYl5gra/SKYhrj17FmO3nkMcuA8LG+MjGNjUOEFMXasl2IX6R2zPEEw4Mb64s5ZaEYFt+jD9ev32PhV+7guQk82yEBpPQYZK2d1hdlM6XlefjJNm2xjhE0sQYdNIDYAYgGjNhrRvPdWIgHr54g5lfuaBd1aIcETKYpvy5fl9WQO86/It3aBU/ebbaa1oi2EBo1t68+4D57SujpWth7niZesCRS/fQ8ReDQALrQEj8w0aUo0iuW2psFccFgBRX0tNXrTEQBy8Bgz1RrkA23Hj0EqRAkTStbm+SzpuKwdQslQe/96mhNSxW+zxKXahWVU4OAGHXHuKrpYbTStan9XH3X8Bz1j7BNt2fcnxj1ZfUZVSaYDjl3tSnBqqXysPKdLJ2xPLk5pbupg5aaxzCdaE+GtcDYI4oW+vG85m7HzF3nmkmh0wCyEcv3mB220poU6UIxxUxmH799h2aLzwsSD6TNAH/wZ4Cqb2SFhydgJ6rQoWhJ0d7I18281/5ik9pWecU7olKwDdrQoU8apLjpkYBSwneljKG5t+ObFwefeuW1twt8YvIqTEG2W0iQOE4zh+v3rzHgg6V0aKy9i9j7X4+ihNXHqBbzeKY0KKi5rhY6wMpdeHA+mUwVCM5ZCrdTj7LMYzzVV+9eSecvJJmzlqB2DtP4T3XwI/N+rswub1GPoNlRu8Sbhh/6eoOH0f+N6im9ry1xiGm5sXi73oAzAJFIzasdeN9vfwYjl6+r4mUJWFQIF8YWvOXkmK/losP4+37D6pyj2lhXYWC2bF7UB2Ou0m66YfPE+E6aY8wgHVOIaXJy5UlPU6NbSjdKRvrSdNeCOMHYf7QulEmivRpvxDYWmjeOZVpZk1rJWV+5LqZsKGQ2yNLEECQ4rOl9PmXurAYprVmx8eb0vw2nojDj1sjkTdrBoT6sU9X0fomMbm5im/DWKuMGsOWfAZIMejMNi5o587/BtXUHrbWOMTUvFj8XQ+AWaBoYwGwlhyO4qrxP79lx8wgZVnnB13E/KAYoeuqHlXhVS6/lGGf9fGeux+kiruPZymMalJB9ngeA8gpRLkxu4W0jKWd3NCYoYIWkaadsuu8UERoKTyXPDA0ZXPAhnDsPHNLyCkkuYVat1WHr2D8jigUypEJR0Y2+PR4KlNtDr/EBZKsX7y0xlfr5w3fEoEtYfHCiSE5OdSiUf51XlLj9GVsmE9ZfN+An2BOSlhtP30DgzaeFmSZoyb6agEr6s8JweW7z2Gu26Gkk9QDYOPLrgfAHD8S1rrx6KmmFnl8l+4+Q4M5+4VV0OKKSrzcpOCPFOARIvgC2TMJuWpydOtvPX6JmtMMOZe0EInjdpJlusbUvbj95BUmtXBCl5olZI1NqbMlKh0xm5wMQxN3ROG3w9pKhovdo1R3lZLI584OuIBF+2LhVCg7/hmo7Y3E4dh76LTCkHseMbahrM+SDOhtsitdT9diOQW1Si0a7+95XipzcrChL+wl8mRByPB6coYq7kvyqkl+tRa0a1KctNY4RMrc1PbRA2C1CKYw3lo33pKQWMz0vwCH/FmxZ2hdjggBoVcfoM2yo8IzzMGpejHhKZr+dAiJ796jtWthzG0vnSh9c+h1jPjjDIjk65lxDZEpvfmlWuli8VJ4Gr0tEuuPx6FeuXxYaWbFO64b04TxZfsvYfruaJTKa4dgMyg+EbJ9Qrqf9MRwa3g8hm6OEE68zk1opCklH1XIMycNmzn3hJpnU+rCorkz4+AIttSFxvwiTDGEMaapS0Es6uimxv1kx/7vjzPYFHod3hXyY0W3qsztSzFoDkGO3qtDEXQ+AR2qFsX0r1ykuMm1j7XGIVxB+WhcD4A5omytG48GdlrkeVpCURUNZshW+LlLFTRyKiBpV9AfEC0kNiU5JOrUfeUJhFy4i47Vi2FqK3Y5hd9tCAfJP23lWhjzZLwsyPXf0vvTQDNbxnSInNBIc3ep6ltSueHT1x8Jue2kEU7qAjkyaebbyK1n8PuJ67DEz4NmICh8EKUuzJyeXNVr8+LSecVxHIq9x61gce6ei/hpbwwqFs6Ond9rextBl2HQxlPYfvomvnQpiMUcgvzklpumszRyssfPXbRJZ0lp21lrHKLwoyRrmB4Ay4JLXmdr3Xj7ou+gxyqDHHLMlMZInzaNvInL6E2DbUIYTojDzdFIIV7bZUcEtSBSEEJSIUwR+JM8WyIIcf95osXkeomxM3ZCqBbfTiuO4XDsfWYcymr9Mdd4Mefu+Ym+TARV5MyF5lcO8S6LQd7/5leKaZ/UUvzJ8Yf0pVe/VJpZ7vjU3N8cxVp0Dw31KQtSNMm6rT9+DaO3nRWYcQhDjjlax1+O4cglbb+vpu46j+UHLqNaidzY3K+mOab92TOtNQ7RAjg9AOaIsrVuvMj4x2i26JCADO9TpOUHLmHqrmiUymeH4GFeHFcjZdOEwLzxggMChVQT5wLCaUFKim5iyeh/BnrAqVAOs/me3IPJ9Tw52WadU0iU9M7dfIKkgZdFTV4DZy7cfopG8w30SuYQBHGfvAf3niUKp/vklF/c6N+0ZGIgDBAuEwLx9NVbzPjKGe2rfu6TBkti1Y8QUxeG/OCFEhrIIZMXeFK4OKVVRXSqXpw5fmJZYMJUwvMgxZjztEh5hG859Pcqw3yOyRnUMoVQyoSsNQ6RMje1ffQAWC2CKYy31o0nLu7a+b0HKhbmF9zRQK1K8VwgLBDmbLSynvjw09euaJ6CahBNmyAnxidGeVscH+6Kg5cx+Z/zYJ1TSIQWiODChOZO6FaLXXGdOdddybN5Us2Z8uctUWv02y1QB67o6g7vJFyj5Dbj5NWH0JKiTfydsbV/LbgVy2VqGvrfRQiI95MWAgrkBqus326BBnJZZzf4VizIfD3EhwRHfqyPQjkzM3+GKYMu4wPw5NVbzTjmiT//0stlRKifeU6+xbhYaxxiam1Z/F0PgFmgaMSGtW68xLfvhS9H0lb2qIp6CujBpML6459nsPGkeQslqK/kR4FUsRMOZKJwRuRB82dPPoeSpgK0qFwICzpoT4NlCl9K/0NyCs9PYkf/QxXQzCW0YGreWv1dLF9LbgtIjqFW7fbjV6gxba/wuL8H1IZLkZyfPZoWH2lZqLj/4l2Qqn/SWKsPaoWrOZ+jRUAqnp9YsYwXZZ2YFm9b/1pw1filSCzGsaZnNXiWzafJEvufvY1+68KQLs0XQgphSjeJWjhkrXGIFtjoATBHlK154xH5SJJPOKuNC9pyJPPuuzYUAecS0LZKEcxqW4njakgzff3BC5DcuOeJ79CgPKledv/PF9jLxHeoNDEQ5EVBK/U6ad7/2+tI7D10/EhJRdgA7BQq3YmfK34xUsqbLHceltyfnoaPb+aI7rVLaubqmfhHaL7IeKEbvZ0onicL9mtE/URvHArnzIzDP2rDYqAZ4Bo9qPrUICQ8eY1JLSuiSw32KQniaRDucpIeQFrwsLoolS8r81mK+ch5nTKn5DT5Lq8z0yDH7D+4DsoXyM58jskZPH75PtovPyb86cz4hsieybxy8dYch/BeMD0A5oiwNW+8BnNCcOnuc/zPtzy+9eIn9UqlUy1JSOL3E3EYuTVS2BnJqfmIT7t450gr3Z6E3q3hPLY5quITne3f1QbhoE3NjbAtENaF/l6lMcK3vGZQmMqtFEvcRk9qLND08W608t2rXD6sSsX0eGpwptSFgxo4YIhPWTWmTI4lctXku5e0iHENhRsvHs1cL4lkLoSLlxRmCv/t522ysJnV/MX53AeG10OxPFlYmVZkx5rjEEUTljFID4BlgCW3qzVvvA7Lj+LY5Qfc8wh95u5HzJ1n0LJIwdQ6kuvt7itPggS6hObKf4gnyMkWbZRbUgueZFO+Gvv75wp7NVGleG6lpj6Ni73zFN5z2QbVqp0yo4Fv1oSC0Pi1qVJEuAnQqtEXtDx2GRCWDHOKeJ2ChnqiTP5s3F1rsfgwIq4/Ql/PUhhpIYqI3CfN+AGUujAptR3jxwjmdkfewrfrw5FUSpv1s4jQ0Km4R8IhCjlM0bL5n72FfuvChVQEUoSXJs0Xmjze0g4KrDkO4b1gegDMEWFr3nhU6pUUgpGCMF7NfXIQ7j17jemtndGhmuVUjpM8y4bz9gsFFB5l8gpKbzSXi9IHWTLdE7l+JIVShOJtWecq8K0ojds4pXU+efUB2n4ULdGVvgAqCkJyC0mOoVZtQVAM5gVdRPkC2eA/2PM/jyWpKhXG+gtrv7xLFTSUyGut1H+y1yqOD8CLxHcWmxKkdG5ajqPUhQ0d7bGcsxzy2mPXMOavs7DPnhHHR/Er1Pp2XRh2n70tW2SIBe5rjl7F2O3nUDBHJhwVyYWzsJ2SDaIw6jBamxoaKXOx5jhEyvzU9NEDYDXomRhrzRtv/N/nsOrIVdQqnQcbvqnBBSVy0kq+KAyVyGyCNJaObjsVjyGbIgSTNC/vztNXqDbFUIC0sntV1Cufn+UjmdqqNiUId56yyym0BNESpgCpNEYD0QoFs2P3IO2I/qUE3l6z9uHq/Rf4sXF5QZKVZxPnWiZXlMfz2bZkmzLiuBXLia2c5ZDp3nUsmB27OO5dLX5HjO0BKgteqUgObB/goelWcR4XgKev32Je+0po5VpE02cnfZg1xyG8gdMDYI4IW/PGW7wvFrMCLqCsfVYEDuEjh/zk1Ru4jA8UVmBTnxqoXioPx9WQb5oE6H3XhiEwKgGETYEUUoTHPRSCYnJ1SHLnsmRIJ9+wRiOaLDiIqFtPwCqncEvodQz/4wy0UAjUCCJVj6GpCIQKL9RPOxEXKakXPVaewL4Ld9HevShmtOErxxocnYCeq0LxxRcQ5Jct+TOhasE5D6aFhMVyZ8GBEfW4Pm3s9rNYc/Qad9W+T5LhZuB5N6cUs+fMfYh78AJjmzqip4d2BbLJbRprjkO4fggA6AEwR4SteeNtOhmH//0ZCZ4KbXH3X8BzlqFKN3CIJ8ra889VlLvcJJ+LCB6QnFqi7FMoZyb8dfomapTKjY19zK/yk9J8jMnlysWA9qc/0CXz2mHfD+YTLVHqP+tx4mK0mMmNkY6jYqLYdynFd5N2RuHXQ1dQtUQubOnHl197acglzPCPhhaBG+s1tCR7lLowSwYih8yOujC5OX63Phz/RN5Cy8qFMJ8jjSO9RcuaMR3OaiwZTl8Cv65WDNNas5ODl7JnWiw6hIj4xxhYvwyGNiwnZQi3PtYch3AD5aNhPQDmiLA1bzwtftxJ0QwpniGNSGUSyUxLbLsib6H/+nDBNXLKRQQIhjcqh+/qaaMspBSTYZsj8Gd4PFhp0s8KiMbifZdQuWhO/PVdbaVu2cw4U3RkvCZKK+vHNXNEDyP0a+uOXYPfX2dhrFCOpW9DN53G1lM34F2B0AZWZWk6Vdk6HHtP4CEnLWoi35N0rYqczSHxTDcNVa1kdQMmZzMSTmxSRN25RjFMbqlt8J3UT2uOQ+RgrqSvHgArQU3iGGveeOLg9MSoBkYFISRCkWy3kAt3BLYF0ghhuDmkMqX6P/D3U/g74uan7taQ6zht93n8vP8yWKns0dxTLQUWpK6POfqZEqTg4ZNUAQ5x4MG7YJHSd2lNB8cDX3PaFFMX8qbPouw7vF/kL999hvpzDHzDQUProkx+9nzDxtaMt9RzSntl8MZTwk0hEcghQjnmbNYch/DGTQ+AOSJszRvv5qOXqDU9WEDnn4EecCrEXg6ZXo8RqrFIja/H5C47UU7ymXcAJCUiZ5b0CPPzQVqNaHXk+kr705QFVoII320Ixz9nbqGVa2HMa19ZqVs2M04sSfxrN3c0qGDPfW5SJZjFwTlPFS7CNOE41h+v377H/PaV0dK1MHcMbPUBn1MX1hJeXHk1t0l7hLSuGV85o31Vfuw7z1+/hdO4AGEaG3pXR60yeXlN6TO7ZF86jN6F9x+AX7q6wyeJXDhvJybsOIeVh6+idpk8WN+bTxG51DlYcxwidY5K++kBsFLkJIyz5o33+u07lPPzF2a5umc11OUgI/nboSuYuDMKRXNnxsERlq8eRa4oCeF/l5oluIqDSNhakrr8deoGBm86DbsMaXGOQU5h5xXHcSj2HrrXKoHxzZ0k+WDrndwn78G9Z4lCjiHJNeTdLtx+KuSkk7Z/uBeK57FL9pHkpJgEHoSabG67SmjtxqcS/eq95/CaHSL4sGtgHTgW0kZtizfO5rAvpi78uUsVNOJEX0eCwzKjdwmpXCu6usObc3BIGRF47sOk6yVm6yHpWiRtS8v2094YzN1zEVozxCQ3R2uOQ3ivmR4Ac0TY2jeey/gAgQeXl9zv3MAL+Ck4FuagqeG47BZj+lDMPXT+lV1OIc2pG+ztgMHefJWqLAZEE45QTugh3mUxyNuBu9ty1pSu14B6ZfBDIz6FOAHnbgtMKeQyhBRuZUqfljsGtvwASl04uWVFdOYkh0x41wn/Omlb+9eCWzF+J83kGURymUgvayl2dPbGYzRdeEiY45Ef66OQSMhIi/2z9uhVjDEDB7EeAMtbXT0AloeXrN7WHgDXnxOCy3efc+MS9fsrEuuOxQmny+SUWW9sEYi+/QS+8w8KRlnkFNLiqwnNndCtVgm2zlqpNdZMG6Zg2Boej6GbIyClqv77309hR8RNfOlcEIs78clDXBQcg9mBF1Eqrx2CdWYQU8tn8u+UupDnS6b4FoHF94KpSXVacQyHY+9renO0L/oOeqwy1JcQFTgt5MDFOOw8cxMDNpxCpvRpQOTIzdmsPQ7hiZ0eAHNE19o3HtGKJ5rxvT1Kwq+pI3OkaE4pbyoe5o5biUHxSc+f36rPKaw4LgDPXr/Fgg6V0aKynutJtgFV7yI5hiTXkHf7xKsqIeCct+ciFuyNMaoYx8JXWhzq61QAy7pUYWEyVdugL1Q82QOOxN5Dx49sE4S32S4jXy7zoZtPY2v4DWi5R7Sg8Uxpo4oZPc5P9EXmDOa7GbH2OITnF4IeAHNE19o3Hm+uSHoy0KN2CYxrpueUst6KJNev7Ec5ZLU5hWJ5z1U9qsKrnOUq4LHGMSV7M/2jsSREO2q4iTui8NvhK6heMjc29U2Zh5ryypJTqKgJvkjDoWiTpoBYAt+pluvO61laBIvkVoDcDmRMR04nfT9JvPOak9afETKPhXtjMGePcblwXnOlds/dfIwvfzJfCoZ4ftYeh/BcKz0A5oiutW+8cdvPYvXRa/AokxfreldnjhS97hvqUxYDG/DPn2Q+ASswyIoKiLBfEFukmaOoxFKhXnX4CsbviELhnJlx+Ef+hZwDNoRj55lbaFapEBZ+7ZoiLJHxj9FskeFHmPhGfGTZyEuR09gAJL57j0UdXdHUpRBL86nS1rRd5/HzgctwL54Lf3zLR8BE6z27+shVjPv7HArmyISjIxtosq5aKd0Zm4wWLEpSgbT2OETqPJX00wNgJahJHGPtG4/3W3TNaXtx6/ErTGrhJDAr6I09Ao0XHMT5W0+gtkgr9s5TeM81zT7AfgaWbZHQwpFUngxp0+DCZP6nae1/PorjVx6gZ+2SGNss5bSkp6/ewPmj1PjaXtVQxyEfUzDFe8JSlRyZTlgDY5S6sESeLAgZzkcOeU7gBSwMjoVLkRz4e4AH91n5n72NfuuAwHZbAAAgAElEQVTCBNpIopjI4yYi6ST6rQ2D/7nbaO1WGHPbaU/Z+DLxHSqMNbAore9dHbU1on9LbjGtPQ7huUH1AJgjuta+8TaeiMOPWyORN2sGhPr5MEeq/JjdePXmvXCSRU609MYegS6/HsfBmHvoUqM4JrWsqPgBoVcfoM2yo8J43sIKip00w8CTVx+g7UdcTo3xQS67DFy9qD87BJfvSS9MpawCE1s4oSvjl0yqkJguzRcCA4TWhUZcgTaTcUpdKKXIUamLo7ZFYsPxOGglaHP6+iMQ+W7SToxugPzZMil1XfK41ksOIzzuEfrVLS0UcZujWcrvm7XHITzXTg+AAUyePBlBQUEICTHwWZJ2+vRpDBo0CKGhociTJw8GDhyIH374QdZaWPvGC4pKQO81oYL8L3lzT5c2jaz5p9TZkt6QmU3KAg1RmdrGFQtgaWflRUp7ohLwzZpQge4qdkoTTU5xLBDO/7h07f5z1J1l+N7Q4hSUFiJK5VSlJ8Y8uJvnB13E/KAYlLXPisAhda1huSzeRzHNHa/iqb5rQxFwLgFfuRXBnHaVuGMiFmXZMcADzkXYiyolnYTHjGDEP3yJsU0d0dOjJPc5JvcAS7nhtPY4hOfipfoAeP78+Rg6dCg8PT0/BcD3799H+fLl0bJlS+Fvx44dQ//+/bFkyRL06NFD8npY+8bj+eZuSTlSkhfUCjtO3XUeyxnkFG4JvY7hf5xBrizpcWpsQytEgo/L4he5db2qw8OBn9LVi8S3cBxrUNWS+qyRWyPx+4k4eJbNhzWMqQb7rw/DrsjbFiH3ymd1tbcqpi48OKIeiubOwtyJNkuPIPTaQ/T1LIWRTSowt5/UIFFMJMW4RJVNC+ENIgJTfoxBndCct4uWUuNi7XEIzw2aagPgGzduoHfv3jh48CCKFi0Ke3v7TwHwtGnTsHjxYly9ehXp0hkoYkaNGoWtW7ciOjpa8npY+8aLf/gCHjP2CfNlrfJkSVWykhfUCjsuP3AJU3dFQ21OIc1NLJnXDvt0vtfPdoJWSldKTpt/OXAZU3adR5FcmXHof2yL9KjAgdr8civ8WHFzWQuRCppGM6pJefTxLM1tLmLDWgh80Oc9fvkGlSYECv9zU58aqF4qjyZzTPoQS2E5svY4hOfipdoAeMeOHfj999+F9IeJEycKwS5NgWjSpAly5syJDRs2fMKepEj4+PggISEB+fNLo4Cy9o336s074U2aNHJ6RE6RWDVL4klkNSdLtLPtVDyGbJImnJCS/7MDLmDRvlhBUpSwQOjtXwTk5uUqxU5JvvHe8wnotdqQxkSu1FkptRGpdMIA8fb9Byzr7AbfigWVTksfJ0KAUBc6jN4lnJYu71IFDTnIIfNW+ExuQZsvOoQz8Y/xff0yGNaQjyohfa64OJO8rJOXdnM0S+G5t/Y4hOfapdoAWAxq9+7dPwuAXVxc0LhxY8yYMeNTt6ioKDg5OeHkyZNwd0+e8P7Jkycg/0fbrVu3UK1aNVy/fh1FihThuY7cbPM63bIkpRxu4FmA4YMxd9Hl1xOCJ4TzU2kANHpbJNYfj4NXuXxY1UNX7RMvrRxmBjVbQgnjxJV7z1FvtiFHOWCwJ8oVyKbGhU9jxVf1e4fVRel8WZnY1Y1AkCkmJ8FTWzmjY/ViTCFJfGtIRyBtZY+qqKcRn3fv1aEIOp+Adu5FMLMN37xjrYU+jC2QpSid6gGw8Y+QHgADSBoAlylTBh07dhROhmm7fPkySpcuLaRMeHgkTx0zfvx4TJgw4T9oW3MATH48yY8o6+syS9JKZ/oLY2HGCAUaoUIjTU1OIT3NaOVaGPPaa08rZGGwfuYOlRyWws2rZh4rD1/BBJmcw4Srt8IYf+YntX9H3ARRgSPMD1ETGjEtkFWDkS2MpdSFPPjRE568QvWpewWYtCpII8+iwSCPXPSka06ZNOwypMW5ib5m2xJzAy/gp+BYVCqSA9s1oJszNlE9ANYD4BQ/BEkDYGdnZ5A0iOROgMPDw+HqmjwBvS2eALdbdhQnrj5AH89SGMWwYOKnvTGYu+ciKhTMjt2D6pjtS8rWHywWsNjWvxZci+VSNOXOK47jUOw98GATUOSQBQ2atDMKvx6Sps6mxu0Z/tFYqkB1rv6cEFy++xzDG5XDd/XKqHHh01iaEqN/fpnA+ZkRSl3YtWZxTGyhnLowOc/EtRc8xFGMobEoOAazAy+inH02BAzxZA+ayCLNe1db96DWyd8OXcHEnVEomjszDo5gm38vxzc9ANYDYFkBMAl+CfXZ2rVrP41LjTnAZPK00pv1yd+EHeew8vBV1C6TB+t715Dzedb7ykBAnFP4S1d3+Djayxj9b9emCw/i7I0nGOztgMHeZRXZsNVBP++/hGm7o1Eqrx2CORYI/rAlAn+ExaOhoz2Wd00+DSs5jHuvPomg83eY0l71WROKwKgEtKhcCAs6pKxIZ6vrzmtelLqwiXMBLOmknLowOf8OXLyLrr+pT4mSO/fNodcx4o8zyJE5PSLG8WWRmfJPFH45eAXVSuTG5n4py4XLnYec/rT+IlvGdIic0EjOUKZ99QBYD4BlBcDTp0/HsmXLcOnSJaRNm1YYO3LkSGzbti1VsUCQefOSlByy6TS2nbqhUygx/apL3pj75D249yxRVU5h7enBuPHoJcY3c0T32ubh1dQAKkWP2Boej6Gb1Rcamno4CVxIANOpejFMaeVsqvunv1MqPLdiObG1P5sCRpoaxfJUWfKEbLwjXa+qJXJhSz+2csisimLlLoGWgfegjaew/fRNs/+27LtwBz1WnhSgipnSGOkZ8ujLwV8PgPUAWFYAfOfOHYEHuHnz5hgxYgROnDiBb7/9VgiKu3XrJnnv2cLGo6kK5Qtkg/9gdldX3X47gf0X76JzjWKY3FL6j7lk8PWOnxDwnX8A0befQk1OIRVgWNChMlpULqyjK0JALF4QNbERsmQwUCeybnQd5dKOER5gwgecM0t6nGbA4UzYYYjM64cPgJpbBdb42Io9Sl3Ig3KQ0hkWy50FB0bwkVpObh0uJjxFw3naSKl/vfwYjl6+jx61S2BcMyezbYuI64/Q4qMC3snR3siXLaNZfLGFOIQXcHoRXDJFcARswvZA1N9OnTqFggULYtiwYRgwYICsdbCFjUckM4l0Zt6sGRHq5y1r/il1brHoECI0osVh5rSVGqL5u0pzCkkhlcNoQ+X4qh5V4aVR5bi1wK3Vj3uVSXtw/3kiprV2xtfVpLMDHL98H+2XHxPgZCHXfPbGYzRdeEiwt3+4F4rnMQ/NlLXsD7l+0hsFHlfnNI/ctVhObGN0GyBlflpy81J+6v/5lse3XtrwHCeHQdz9F/CcZeDR10Il0tg62EIcImWPKemjB8BKUJM4xhY2XuC52+izNkyQwI2Z0gRpyX8waJ4z9yHuwQuzSlUymIZVmKDpJkpzCsXk/IQDmHAB6+1fBB69SETliXuEf9jSryaqlsjNHB7xS8iv3dzRoIL0XO47T1+h2hRD5f+f39ZEleLq/KPX6JnSEwYIX10Wm/Fq80wXGPFHBDaHxsO7gj1WdJOeR652ikSdjdwavHrzHj997YrmlQqpNWl0vDl4jpNz5smrN3AZb35BDluIQ3htFj0A5oUsAFvYeOFxD9F6yREBJXICTE6CWTTKLzyvfSW0crVOjmQWOGhhQ21RiJhYPuQHL5QwE7G8FlgpeQb5cS/n54/Ed++xuKObkHvIut1+/Ao1pimjryL+kR/ip6/fYlYbF7R1L6rKvem7o7Fs/yU4F86BHd8nTwmp6gGpfHDUzSdo8pOBuvDQ/+qhSC52csi0ILK9e1HMaOOiKdJes/bh6v0XGN2kAr7xLMXl2TzFm+Q6TD535ObMIBZTBb4VC8g1waS/LcQhTIBIxogeAPNC1kYC4OsPXqDOTMM1jv/gOihfILtqxMSnWVqSsat23EoNqGUpCL36AG2WHRVmHzG2IXJkSW+lSPBzmxYJjmvmiB4cigTPxD9C80WHhQkcG9kABXJkkjUZqsRFroTJ1bCa1mvVSeyNZssqocYfWxsrpi5kfePScvFhnL7+CP29SmOEyn0gF3cqGNPLoyTGNHWUO1xSfx6/V5IebKQTFTWZ3toZHWSkLal5ZtKxegBsHE09AGa505LYsoWNJ36jXterOjwc8qpGTPwFv/272qikX6mrxjQlA3+GxWPYlghky5QOkePl0/EERSWg9xqDnO6lKU30K+9kwOYdWIjXIGZyY9nCE4M3nsJfp2/C16kAlnVRR63lMSMY8Q9fYmTj8uhb13w5llw/NGY0LqYuXNHVHd4KqQuTm0KdmcG4/uClEICSQFTLRoRTiIBKU5eCWNTRjcujw649wFdLDS/r4WN8kNsuA5fnSDXqM3c/Yu48wwjfcujvxYaDW+qzaT9biEPkzllqfz0AloqUgn62svEoAwCrdIWYhKfw+VgRfGB4PRTLw+6KT8Ey2fwQwrZBWDdIUyKHvCX0Oob/cYYZi4AtAk55cdtUKYLZbdlLvf5bjJoBoX4+siGkbC5l7bMicEhd2ePpgOev38JpXIDwP1d2r4p65fMrtqUPNI4ApS6UW/BoClOnsf54nvgO5mBz4UnvRuftf/YW+q0LR7o0X+Di5MZmf1nnJSRlap3Ff7eVOETOnKX21QNgqUgp6GcrG4917pa4Kv3M+IbInkm/UlewvSQPEas/KckppNRJ5lZWkjxhM3TkLfW6ICgG84KUKyfuiLgJItlMpIvPT/RVXMwqpnbSUknMDEtq1kdSyrthPmXxfQMHJr68TDTQ15G2tlc11HHIx8SuVCNELZGoJvJURltz9CrGbj+Hgjky4ejIBlJd49av79pQBJxLQNsqRTCLw4uxFMdtJQ6RMle5ffQAWC5iMvrbysZrs/QIQq89RF/PUhjJQA7Z/+xt9FsXJrylE4LwL8jdut64IXDnyStUm2oooFKSU0hlbwn7Axmvt/8iQANU1nzZ9Emjt0Vi/fE4eJbNhzU9q8leAvFL0MER9VA0t7JbF6rolZWoW41vqH92Za+EtAGUurBbzeKYwEgOOf7hC3jMMNRz7BpYB46F1NdzSJuNodfOMzcxYIPhJezCJF8ue4d+V1UqkgPbB5i/QPPHP89g48nr8K6QHyu6VZUDF7O+thKHMANEZEgPgHmg+tGmrWy8fmvD4H/uNlq7FcbcdpVVI7bxRBx+3MqeW1i1YzZq4C3h8fXbLQgXKMkppKebXuXyYVUP+cGXjcL62bSo2EQeuwwIGyM/RcEURt+sCcWeqAQoTbF4kfgWjmMNqQtquJwpo4jWPLKm8LG1v9Oc7S+dC2JxJzb5suJCyhOjGiB/dnmFlGoxFhfTsuCjTs4fc9G8GcOGMqZUKZ4Lf37LVtVP6nrYShwidb5y+ukBsBy0ZPa1lY3H+np3SUgsZvpfgEP+rNgzVHk+oszlSNXdqYiCkmrk7zaE458zt9CyciHM7+CaqnE0Nvm95xPQa7WhUJDkHrKWPSWKUiT9QE31fs1pe3Hr8SuoYaqgcszmoNFKTRtv8s4orDh0BdVK5sbmvjWZTH1f9B30WGU+aV4xQ8PuQXVQoSD7E+juK08g5MJddKxeTJB+N3f7xMCTzw7Bw7zM4o6txCE8wNMDYB6o2tgJ8Pygi5gfFAPHgtmxa1Ad1YjRYohqJXJjcz82X+6qnbJxA43mHcCFhKf4oWFZDKgvL6eQXsd2r1UC45ubT1rUkpcoMv4xmi0yqKMpoSkzNbda0/bi5uNXGN/MEd0V0qx1/OUYjly6jy41imNSy4qmHpns32kQ7fdlBfSuw4fLVZFjNjaIR+BEi1lzZE6PiHENNUfs9dt3Al82abzoL7/86SDO3XyCQQ0cMMSnrOZzTPpAmjJE2CgIK4U5mh4AG0ddD4A57khb2Xjrj1/D6G1nkT9bRpwYrV4OefiWCGwJi0dDR3ss76qdGhHHpbZ4051WHMPh2PtQEsQ2XXgQZ288wWBvBwz2Nv+PiiWCnfDkFap/zLP+e0BtuBRhp5ZHCPXL+u3Gm3cfsKSTG5o4KxPaoDc5HmXyYl3v6rJhFCtbmaOISrbDVjyAUhdmz5QOZxRQFyY3dSJeQq7kS5nxNNJt0h48eJ6IGV85o31V6XLeUpey6pQgEJrNKa0qolP14lKHcetH0pZI+hIRUI01E4WkrcQhPBZJD4B5oPrRpq1svIBzt9F3bZhQOU44SNOolEPuvToUQecT0KFqUUz/Sls1Io7LbdGmB208he2nbwoqZUStTE6jvK9qTh/lPM8a+6rNs05pzg+fJ8J1knqpZVqFXyhHJhxRUCEv5lg9PqoB7DXOIbXGfaHUZzF14YXJvsiYLq1SU5/GaUFDZsrJxgsO4vytJxjiXRaDvOXdRJmyLeZP/qWrO3wY8ieberaxv4vznk+P9UHOLNrzEttKHKJ0DVIapwfAPFC1sQA47NpDfLXUIIcc5ueNPCrlkIktYrNf3dL4sbE6VSqOy2dTpgn9EAmAlOQUUh7o+e0ro6VrYZvCheVkeHG3Xrj9FI3mHxBc3T/cC8Xz2Clye9+FO+ix0pADGjWxEbJkSCfLDi30I6eS5ApdZ2+RBZ+szmLWDlZ0c0M3n8bW8BtMxFBkTUbUucfKE9h34S6+rlYMhOOYZROz3ViKwNKlu8/QYM5+YZr7fvBCSTPIyOsBsPFdpgfALD+BSWzZysaLu/8CnrMM9DkBgz1RrkA2VajVnxOCy3ef60pSqlCUN3hpyCXM8Jd//SmWrVbDHiDPW+vszet062DMXXT51SBkQjh8M2dQdhoo/hwrocGasOMcVh6+iqolcmFLP/NUtFvnzpDvNY9gjojhkJPlTtWLYYqZCsQoLViD8vnxa3e2tGBnbzxG04WGPPyjI+ujYI7M8oFnPIKke5C0D9IICwRhg9C62UocwgM3PQDmgepHm7ay8cQUSut7V0ftMurkkGke2Mw2LmjnXpTjCuimKQK0AEZuTuG9Z69B9OxJU8IhnJpWgFeA8UnKmnDvTpAvZU3XgFwRVxjjj8R377GooyuauhSStTw0j9xSKuxlOW9lncUpNb92c0eDCvaqZ9Bs4SFE3niMgfXLYGjDcqrtKTEwb89FLNgbA6dC2fHPQPUF1WIfxCwXhImF8A2bu5HPXJnRuwQKSlbrKHdOthKHyJ23lP56ACwFJYV9bGnjOY71xwsGEprvP34hvP8AWEqelsLltaphIRfuoPvH6285OYWxd57Be67hCi/kBy+UMMMVnrUATYs7Se4h2dusGsviJZ+5+xFz5xmUKIzRAqMJzZ3QrVYJVtPT7RhBgHXBGGUSMef60TSavFmVSXqntFk2nYzD//6MhDkZF5Lzr9KEQDx++UaQSCc83lo3W4pDWGOnB8CsERXZs6WN5zlzH+IevIBa+qNHLxJReaLhSuiPfjXhXiI3xxXQTVMExNeDR36sj0I5pV0PWkIRh7Ws4kz/aCwJuYRKRXOC5CCyahN3ROG3w1dQvWRubFLJCdtnTSgCoxLQyrUw5rWXLmojLsTb8E111Cqt7haIFTa2bKfhvP24mPAMwxuVw3f1yqiaKmESKUdO/98qO/1X9XDR4ODoBPRcFSr8C+tT2oV7YzBnz0XwUmNUikG92SG4cu+56t9Opc+3pThEKQbGxukBMGtEbTQAZlW4Rr4IyBcCaUFD66JM/qwcV0A3TRFQStMVFJWA3msMAg+XzETjYy2ruOrwFYzfEYXCOTODFC6xagM2hGPnmVtoVqkQFn6tToiEKlPJDdKPX76P9suPCVMK9fNGXpWFsKywsWU7aqgLk+Ly7PVbkGJW0sz5AsOjuI/OdcxfZ7H22DXUcciLtb3k0/zx2kutlhzGqTh1IjZqfNMDYOPo6QGwmp1lYqwtbby+a0MRcE65FCuFKjzuIVovMTBKEGJwcl2lN/4IiIvZfuvujvrlpeUU/hEWjx+2RCBnlvQ4PVZ78nz+yLB7wq7IW+i/PhwZ0qYBSTNhxZLQ7uejOHHlAXp5lMSYpo6qHN588jpG/HkG2Qi/rAwmBxJYkACDl9SzqknZ6GBKXdjUpSAWyaQuTArJtfvPUXeW4eAhcIgnytqrK2RWCvn9Z69R5WNNAeuisH5rw+B/7jZauxXG3HbSbzeUzkXquJ6rTiI4+g4X5gspPthSHCJlvnL66AGwHLRk9rWljTd6WyTWH49D3bL5sLpnNZlI/NtdLBlLiMEJt7DetEHAdWIgHr54g5lfuaBdVWnFhysOXsbkf86jRJ4sCBleTxtHrfQp4nSRU2N8kIvRyx29Qh3ZuDz61i2tCh2xj3JOcunpWo1SubGxj67eqGoRJA6m1IUsUl9YU1lKnMJ/upEakHJj1Iu6JPf81ksOIzzukcXRaw7bHIE/w+PRuGIBLO1cRSl0isfZUhyiGAQjA/UAmDWiInu2tPFYVe/SE0VzyXFyXG6LN00LoOTkFM4OuIBF+2KZ57VaPFgKHGRNF0hdoDzM89pXQitXdUU04hO4zX1rCrzQUlr7n4/i+JUH6FazOCa0UCajLOU5ep9/EaDUhaXz2WHvMC9V0IgVyWLMfPBQe3owbjx6iXHNHNFDoax3cmBQwZ6xTR3R06OkKrxYDp68MworDrHJ4Vfily3FIUrmn9IYPQBmjaiNBsD0CtQ+e0YcH6VcDpmeKBJCcEIMrjftEPh6+TEcvSxPDpnK53qVy4dVPZSf/Gs3S/M96WXiO1QY6y84sK5XdXg4qC8UE1MQsrBJiqFIESqpSp/e2hkdqpmWoyVjCCMBuT2Y3LIiOtcwv8Ss+VZZuydT6kIWhwUbT8Thx62RFpHCQutJ+tYthZGNKzAB1FKK/JKbzOJ9sZgVcAHl7LMhYIgnk/nKMaIHwMbR0gNgOTtJZl9b2nj+Z2+h37pwpEvzhVC9q1QOeVZANBbvuwTXYjmxrT+7SnmZS5Mquw/8/RT+jrgJOTmFtACrZeVCmN9BXQFWagDdeVwAnr5+i7ntKqG1m7rTWoLX1XvP4fWxaJRV7iYtyunrWQojm5gOQO4+fQ1CgUaanFPj1LDePOcoVu5Ty5hAg7Cy9lkROKQuT7dN2u6/Pgy7Im+D5XfK4xdvUGlioEXu0fXHr2H0trPIny0jToxWfnhkElgjHWwpDlGKgbFxegDMGlGRPVvaeKzyG0dujQThgqxfPj9+Y6wExHEpbcI0pdOSk8fZecVxHIq9h+61SmB8cyebwIHnJKjKIZH4JlLfahspfiNFcKSdHuuDnFnUF41SSVzvCvZY0c00X/GR2HvouOI4Ux/U4pIaxrNUNqOf/Zql8uD3PjXMCh9VFGTpS+ydp/Cea5ALN5fksDFQd0fewrccimOlLqItxSFS5yy1nx4AS0VKQT9b2njik6g9QzzhoLCK+Nt1Ydh91vIqdRUsr9UNWRISi5n+FwTqOUJBJ6U1XXgQZ288waAGDhjiU1bKkFTdp8Pyozh2+QF61i6Jsc3UMTYQIP85cwvfbQgXVK0uTGLDLEFPA6Xmlq48fAUTdkSZ7QQrtW4oMXXhjgEecC6SQzEUlFHiS5eCWKySUUKxEx8H/rz/EqbtjkapvHYIZpQGJ35JOzehEewyplPrJrPxRy/dx9e/GCgEz05ohKwa+2ZLcQizRfloSA+AWSMqsmdLG+/567dwYsAjSQOE3h4l4aeS0onj0tmk6c2h1zHijzOyKM1oYcn4Zo7ozrBgxSYBBvD976ewQ2aaSUpY0OCTJbcwpWtLn/YLnJ/oi3RpU5aMpbc2HmXyYl1vy+FXtdU9ROclpi5c2b0q6pXPr3jK9CbHEooYt5++gUEbTyNLhrQgwSoLusC/Tt3A4E2nYUdsTvRVjBOPgdG3n8B3/kHB9MER9VA0dxYejzFq05biENbA6QEwa0RtNAAm06owxh8v37zDgg6V0aJyYUXINZp3ABcSnjJRN1LkQCoetC/6DnqsOikgIDWnkOa0zm9fGS1dla15aoKcJXUVwW2GfzQIG0DlojnxFyN1OfEPshR56zZLjyD02kNmp9qpaT+onesn6sI2LmjnLo26MLln+s4/gOjbTzHUpywGNnBQ65aq8eIT0TPjGyJ7pvSq7JHByw9cwtRd0bDE4mqWJ/lKgNIDYOOo6QGwkh0lcYytbbw6M4Nx/cFLgYyfkPIradWmBOHO09eY0qoiOlXXq8mVYKh0TGT8YzRbdEgYfnRkfRTMkbIc8mcnUD2qol455SdQSn22tnGsr3cph2hDR3ss72o6X1cKXq/eGNgqPnwATImikOp6lwmBePrqrWTWCCk+6H2kIaCEujA5y/R71xJYPD5XA/VEmfzqRTmm/BOFXw5eEWj9SKGmJbXXb9+hnJ+BHWZNz2rwLJtPU/dsLQ5hCZ4eALNEM4ktW9t4tHr8W6/S+J9vednICVQ1fv5IfPceSzq5oYlzQdk29AHKEbj1+CVqTgsWDEjJKbz37DXcP6o2kdNHcgqpt5QR2HYqHkM2RQh5fiTfT23r8utxHIy5h841imFyS2e15j6Np1ysfl9WQO86pYzavf34FWpM2yv8fWv/WnArlouZD7oh0whQ6sIetUtgXDNlRahEfKKs3268ff8Byzq7wbeieb93WVP7ERQtKcc5uVV1GuuP54nqbk9N75bke9haHKIUh+TG6QEwSzRtPAD+Zk0oCKF62ypFMKttJdnIifOIf/+mBmqWziPbhj5AOQKJb98LP4SkSckpjL3zDN5z9wv9pVyVK/fMdkYejr2HTh8ZE6ImNkKWDOqKcXhdXdPAumP1Ypjaynhgvf/iXXT77YSwQJHjGyIbg+tq21lt/jOhOeXNKhXCwq+V0RA+epEocD+TtqVfTVQtIU38hOfsnMcHCLcKs9tWQpsq6ukCWbwo8JwvfeE0Ry2FHgAbX1k9AOa4621t49FimHrl8mGlAlGE6w9eoM7MfQLi/oProHyB7BzR100nh13cDhcAACAASURBVEDliYF4ROSQJeQUhl17gK+WsqXgsvVVuZjwFA3nGeiYWLw0VJm0B/efJ2Jaa2d8LUG0Qiq+47afxeqj12CKEo8K1xTKkQlHRjaQal7vxwgBFpRhl+4+Q4M5hhfZvcPqonS+rIy8U26GVWoH9aDBnBBcuvtcuJkkN5SW1potPITIG4/NwqZja3EIy7XVA2CWaCaxZWsbb27gBfwUHAvnwjmw43sP2ciJc1CPj2oA++yZZNvQB6hDgJzokpPdEb7l0N+rTIrGgqIS0HtNKL74Aog1s3yqullrN5rlaZs4B/vXbu5oUMGe2URWH7mKcX+fgyllxxF/RGBzaDx0JUBm0MsyRCnrHPJnxR6J1IVJHyDmko4Y2xA5sqgvOpM1iWQ60xuIrjWLYyIDaW16ojynbSV8xeBEWe38ko5nPV85/tlaHCJn7qb66gGwKYRU/N3WNt7ao1cxZvs5FMyRCUcVnAYduHgXXT9ep16Y7IuM6dKqQFcfqgQBOTy1f4TF44ctEbJo05T4ZEtjxJKshG+V8K4qbXJztuU8R/xZTImbtMXiw4i4/gh9PEthlATVODk+6H1NI0CpC3NlSY9TYxuaHpBMDyrEQFQ8Y6Y0ZkI7psgR0SCWxZ2kqLP8GPMVmUnBgqpwqkllkfKc5PrYWhyiFIfkxukBMEs0k9iytY1Hv0gJfyih0ZLL30j5Hy2Rq5HjNrAo01TauHmlQvjJRE4hvf4ukScLQobXs6h5WLIzNN9vXDNH9FDBnUwCTxKAksb6xiT+4Qt4zDCkI+383gMVC/9XZIEE8xXHBQjFO6xyNS153SzRNzF1IQle05vgbE5uDuuOXYPfX+aT4k3Op1kB0Vi87xIqFc2J7Srp/awhtY6mHNVxyIu1vbTl0ra1OITl51QPgFmiaeMB8MmrD9B2mfKc0FWHr2D8jiiwJPXnuHw2aXr83+ew6shV1CqdBxu+SVkSdU7gBSwMjmXyI2WTYBqZlFq2FGpWnIISM7mxScEKORgTZgBChfb67XujvN7iIPnvAbXhUkRnAZGDMYu+4rSxYyMboEAO+WljC4JiMC/oIioUzI7dg+qwcEu1DXqbWCB7JhwbpS63XFyrED7GB7nt1MuFq55gEgPzgy5iflAMKhbOjp3fa7sGegBsfDX1AJj1ThfZs7WNp5a/cd6ei1iw1zxfAhyX2apMy8kp9PsrEuuOxaFu2XxY3bOaVc3TnM72WROKwKgEobqdnJwqbRuOx2HUtkjkzZoBoX4+Ss0YHUcZJgZ7O2Cw939lroOjE9BzVagwngWjBfMJpAKD4jQYYyf1pmAYu/0s1hy9BnOcPhrzLfDcbfRZG4a0aQy3ieT/K21qbyaVPlfOOJpzb47DH1uLQ+TgbqqvHgCbQkjF321t4z199QbO4wMFRJTQmJnzGkjFMtrU0E0n4/C/PyMhJaeQpku0rFwI8zsoo2CyKfAkToa+OBDCe0J8r7TRUyNeJ3ffrgvD7rO3YSwdZtn+S5i+OxrFcmfBgRF6CozSdVQz7jPqQoViNN9tCMc/Z27Bkj7H4vSeE6MaIL+Kgug1R69i7PZzsGSmErH8c5TGUs22Foeo+TwlHasHwCzRTGLL1jYeyQkkxQbk2pRwUpKEfjmNclpKyT+VY1fvKx0B8ameqZxCWrncvVYJjG+ujIRfume20/OnvTGYu+ciyhfIBv/BnoonRk5/ySkwrxN4modpjNVl6KbT2HrqBrwr5MeKblUVz0MfqA6BShMC8fjlG8xq44K2CuSQ5RS+qvNU+mixPLDa9JrZARewaF8sKhXJge0D5LMTSfdaeU9x0Wn0JF9kSq9dAbitxSHKV+G/I/UAmCWaNh4Ak+mpKfChAVW3msUxgQH1Dcels1nTZ+IfofkiaYVVTRcexNkbT8zCXWnNC7DxRBx+3BqJPHYZEDZGeeqCWuEZUxhSlg9SlEqYIJIWtdL1V6r8aOr5+t+lIaCW45Y15640r1Pu9e79BziM3oX3H4DlXaqgoVMBxWYpVZ93BXus6MZGLlyxM0YGnr3xGE0XGmToleZyK/VJD4CNI6cHwEp3lYRxtrjxKC3Sd/VKY3gjeXLI9AfVWM6hBEj1LioRuPnoJWpNN8ghm8op9JgRjPiHL6GWzUCly1Y3XM4pe0qTU/NZkwJaeNxDtF5yROialGWCBChO4/zx6s17zG9fGS1dC0sxqffhgAA9we3lURJjmjrKfgIVU5nxlTPaVy0mezyvAdWnBiHhyWtMauGELjVLKH5M95UnEHLhLkypGip+AIOB4oJSUohI0pq0arYYh7DCTg+AWSGZjB1b3Hi9V4ci6HwC2rsXxYw2LrLQo6fHE5o7oVst5V94sh6qd/4Mgddv36Gcn4Ezc1WPqvAql98oQs7jAvD09Vs9AJK5h1hU7pNH1pq2FzcfvwIv+VSxaEfSnP6r957Da3aIMPN/BnrAqdB/adJkwqJ3V4gAzcVvUbkQFsjMxScvMmVG78KHD8AvXd3h48hOTEXhdD4Na7HoECLiH2NAvTL4oVE5xeaaLDiIqFtPYMkHK89fv4XTuABhjhu+qY5apfMqnq/cgbYYh8jFwFh/PQBmhWQqCYBHbj2D309cR/3y+fFbd3l5gY5j/fEi8Z1R2iWOS6GbFiHgMj4AT169TZHbVaxCtlJh8U1qBZ1FfiPJty/rtxtv3n3Akk5uaOKsXFAjpXVwm7QHD54nYkqriuhUvfinrrRKnxTnk6IdLXMWU+u+MTZvSl1Yu0werO+dMnVhUhv3nr2G++Qg4Z+39q8Ft2K5LAZeVmwpVacE4e7T15jaylk4BbbEJhbI4fl5Tm7uegBsfEfoATDHT4stbjzKDetSJAf+llFwID55JJXxpEJeb+ZBoP6cEFy++xz/8y0Pkt+ZXBP/cG7rXwuuFvTDaR7UpD/17bv3cPDbLZy6rejqDm8Fp24kKCXBKWl/9KsJ9xK5pTsgo2ebpUcQeu0henuUhJ/oep3S5ZXKa4fgH7xkWNS7skaArkVZ+6wIHFJXlvkLt5+i0fwDwpgDw+uhWJ4sssbz7Dzmr7NYe0wdPZs4l9jSTriTYkdTPia3rIjONf592eSJMbFti3EIK8z0AJgVksnYscWNR/kM5VLOiE/FTOWeclwS3TSAdj8fxYkrD5BSTmHsnWfwnrtfwCvkBy+UyGunYycDAXLqRl4ilJ5KaRW4DN8SgS1h8f+50aHSrY2c7PFzF8ssLJKxHFbdlVIXEoEHIvQgpx25dA8dfzkuDElJ8lqOTVZ91QT21Ic7T16h2tS9wv8kinJEWc5SG+Xd/qFhWQyo76CZm7YYh7ACTw+AWSGZSgJgwidJeCUzpE2DC5N9Jcshn7/1BI0XHBRQOvS/eiiSy3JOIjhuAYs0TXlBU8opFKsrnR7rg5xZLE9dySLB/eiU2rzEgzF30eXXE4K18xN9kTkDH9qkpSGXMMM/GiXz2mGf6KSX/lgPrF8GQxsqz8+05DWyFt/2nk9Ar9Wh+OILQK4i4I6ImyD0kxnTpQGh35IrX88TI8pCkj1TOpwZ30jRo8TsCkdH1kfBHJkV2dFikNpiRqU+6gGwceT0AFjprpIwzhY33vHL99F++TFh9hHjGiJH5vQSkADEJxHnJjSCXcZ0ksbpndgjICWnUPyjGzuliSqlJvYzsHyL3X47gf0X76JT9WKY0spZtsN/hsVj2JYIZMuYDpETlAUHUh7qf/Y2+q0zKHKRQDtDujQgKRyOYwOQ+E4Z37eU5+p9pCOgRjTCkuXnWbzksWJckb4aynv2Xx+GXZG30dqtMOa2q6zckMyRthiHyITAaHc9AGaFZDJ2bHHjXbr7DA3mGK7G9w6ri9L5skpCcFfkLfRfL//kWJJxvZMsBBYFx2B24EWUs8+GgCHJCzXQ0xnygkNedPQmDwGaWkCq7kluotxGT2ZL5bND8DB+ObgxCU/hM8+QI0o/z+L0l4DBnihXIJtc9/X+DBG48eilwL9OmlxGjrmBF/BTcCyMiZ0wdFO2KfHeU5pmpSY9RLbDKgdQYZt65fJhZQ/lCpFy3bDFOEQuBsb66wEwKyRTSQD85NUbuHyUQ97Upwaql8ojCcH1x69h9LazsM+eEcdHeUsao3fig4AUoYYVBy9j8j/nUSJPFoQM12Vw5a4EVVkjOYkkN1Fum7DjHFYevooapXJjY5+acodL7k+KUyuM8RcECWgR0e7IW/h2fTjSpflCYIAgp8J6Mx8C4gLi1T2rCcqAUhsNurzK5cMqDYMuKf6Jf0s29qmBGhJ/S8S2F+6NwRwGqotS/FXbh34nVC6aE38p+E5Q+nw9ADaOnB4AK91VEsbZ4sYT07ks7uiGL12k0TPRU0e18rASYNe7mEAgKCoBvdeknFNI2T6UBnCpfRGUFotS3Cj3qxay4Z4z9yHuwQuMbFwefeuWxvygi5gfFAOH/FmxZ6g81oHUvu685k+pC+e0rYSvqhSR/Ji+a0MRcC4BX7kVwZx2lSSP06Ij+S0h3LhqqDFZMEloMVfyDHMdKthiHMJqzfQAmBWSydix1Y1HBS3kEPRP2hmFXw9d4X6ixXE5bcb06euP0HKxQQ75xOgGyJ8t03/mRn9YyGkTOXXSmzwEaMpP+rRf4OLkxrKLj6QwdcjzyHhvqqTVoWpRTP/KBd+tD8c/kbeEl1vykqs38yNAqQt/bFwe/eomT12YnJdtlx3ByasP0cezFEY1qWD+iSTxoN7sEFy59/zTy5dcBy05wE86F5rXr6boTy4+pL+txiFKsEg6Rg+AWaBoxIatbjwlCj5DN5/G1vAbaOJcAEs6VeGIum7aFAJiWU5jOYVq1KdMPT81/D306gO0WXZUmOqpMT7IZSePRUNtYCAHY5puUa1EbmzuVxM+c/cj5s4zDPEui0He2tE1yfE5tfWlL0RJ+ZpN4VB/dgguqwgwTdlX+3fKjNCzdkmMbSZf5rn1ksMIj3sk8JkTXnNLbuKCvdgpjZEurTapRbYah7BYaz0AZoFiKguAe606ib3Rd0BPjKRA2GPlCeyzcL12KfOwhT6v3rxD+TEGOWRjOYVdfj2OgzH30L1WCYxv7mQL09Z0DnH3X8Bz1j7hmUoKyZzG+uN54jvMa18JrVylX3krmSQRIyAn/nmzZsSRH+uDKDa+ff8BSzu5oTEnBTolfqbmMfRUvmXlQpgvQw5ZiuqjOXEdvPEU/jp9E186F8TiTvJvGzxmBCP+4UuMbeqInh4lzTkVk88Oj3uI1kuOCP3C/LyRJ2tGk2NYdNADYOMo6gEwix2WygLg//1xBptCr8O7Qn6s6CZNDplcuZOr9+/qlcbwRpb9ps5xS1iMaefxAXj66i2M5RQ2W3gIkTceY1ADBwzxKWsxfluLI+KXjLW9qqGOg/TCpeev3wq5kaSt710dtcvk5Trtw7H30GmFQSyBqM7Rk2s5LC9cHdSNY9z2s1h99Bo8yuTFut7VJSGS+Pa9IKdNmqXKmU/bdR4/H7iMKsVz4c9va0maF+0krkdZ1NEVTV0KyRqvdeer957Da3aI8NigoXVRJr80BiW1fuoBsB4Aq91Disbb6sZTUuHuNWsfrt5/Ab8vK6B3nVKK8NQHsUPA1NUoPVkZ18wRPWpb9skKO1TYWjL1kmHsaeIfyj1DPOFgz5eG7Nbjl6g5zUCzRa7YVxy6IgjdRE1spNk1LVvkbc+aErYDsfrm3wNqw6WI5amk/XboCibujEKRXJlx6H/1ZS3c4xdvUGlioDBmc9+aqFaSj1y4LKdS6Cz2d0u/mqjKSd48qQu2GoewWBf9BJgFikZs2OrGW3n4CibsiELhnJlx+EdpX1qVJgTi8cs3Rk8cOS6DbjoZBNotO4oTVx/gmzolMfrL/+beOY8LwNPXbzW5grfVBVJauERkqknOJ2laqPC9f2+oxn/55h2K5s6M6w9eokLB7Ng9qI6tLo3VzYtSF+bNmgGhftLkkM/dfIwvfzokzJV8T5Pva0travjhWfAIa4kH+ZyVGb1LoBxc3qUKGjoV0OTxthqHsABPD4BZoJjKAuCdZ25iwAbp8ppEWarMaMNV3G/d3VG/vD1H1HXTUhCgqkStXAtjXvvPVYnevHsPh4/rZalXp1LmaO4+Sgt86OeL8O9e0Ei+lko3U8xSksk2N66p8fmUujANkUOWqMx44OJddP3NIKdNZJAzpecjp61mPcSS63LzYo/E3kPHj6k71qIu6jZpDx48T8TMr1zQrmpRNdBJHqsHwMah0gNgydtIfkdb3XjHLt9Hh49yyGfGN0T2TCnLId9/9hpVJgcJAG7tXwtuxXLJB1MfwRSBsdvPYs3Ra6jjkBdre32eUyher239a8FVXy9F2A/8/RT+jriJpi4FsUgGnRi9FpZzw6LIQdGg7zaE458ztz79y/BG5fBdvTJqzerjGSEgpi48Odob+bKZLqD669QNDN50GnYZ0uLcRF9GnrA1I2ak2TWwDhwLZZf8AGuYX9LJNJgTgkt3n0MunZ1kUJLpaKtxiBpM6Fg9ADaC4rVr11CiRIn//PWXX35B7969JWFvqxtPLJUaPKwuSpmQQxb3Vyp5KQlwvZNkBH7aG4O5RhSUxOu17wcvlMxrJ9mu3vFfBCbvjBLyaUluIslRlNpm+EeDSCG7FsuJbf3lq8hJfY64H5XMpf9GVeGU2NLHsEdASaBIhReK5c6CAyMsU83xs0K97lVRr3x+yeAtP3AJU3dFC99P5HvKGlqbpUcQeu0h+tYthZGNteFlttU4hMV66wGwERR37NiBdu3a4fLly5+R2OfIkQOZM0vLpbLVjSe3+ODk1Qdo+5ETNWJsQ+TIkvKJMYuNrdtIGYENx+NAZFKTyykUX0sq4bDVsTcgQH+gS+W1Q7CMH+hhmyPwZ3g8GjnZ4+cu7prAue1UPIZsivj0rP3DvVA8j/7iown4Eh4iZhVZ07MaPCXIIZvjRUrCVP7TpcqkPbj/PBHTWjvj62rFJJtQ+oIp+QEcOvZeHYqg8wlo714UM9q4cHjCf03aahzCAjw9ADaC4tSpU7F582acPn1aMc62uvEE+hk/fyS+e48lndzQxARXaOC52+izNgwkfy12ShOkIf+hN7MisCcqAd+sCRXWJGlO4d7zCei12iCVTNYrrb5eitaKBpVZM6bD2QmNJNugHMydaxTD5JbOksep6Rhx/RFafFQHzJQ+DaIm+OqfUzWAchhLC1PntquE1m6muaGV0FVycNukSZp/LpdyUWmKkUmHOHYY8UcENofGo6GjPZZ31ebl1lbjEBbLpAfARlDs0KED0qRJgw0bNijG2ZY3Xs1pe3Hr8StMbOGErjX/myoiBm3zyesY8ecZ5LbLgPAx0iqYFYOuD5SEwKm4h2j1kZQ9aU4hlezMkTk9IsY1lGRP7/RfBMT8uoRSLEuGdJJg8p1/ANG3n2KoT1kMbKCNEtuTV2/gMt5AKeVcOAd2fO8hyVe9k3YIUHXAUU3Ko4+naTnk3qtPIuj8HU1PG5Wg0XPVSQRH38HX1YpiWmvpp6JfLz+Go5fvo0ftEhjXzDrEeijvcdUSubClnzzeYyXYkjG2HIcoxYSO0wNgIwg6OTmhYMGCeP36NS5evAgHBweMGTMGjRoZP8l58uQJyP/RduvWLVSrVg3Xr19HkSKm39jVLqaW46lQwsD6ZTC0YbkUH/3z/kuYtjsapfPZYe8w68jV0hJLczzr+oMXqDPToFRG6K4I7RVtNHeweJ4s2D/cMnMHzYGZ3GcqpWmileLTWzujg4wrYbn+Je3vPjkI9569Rmu3wpjb7nNmELW29fHqEWi77AhOXn2IPp6lMKqJ6fxRKj7U36s0RliwTPDIrZH4/UQc6pXLh5U9qkkGihaUEQlkIoVsDY3k9pPUFCKCQcQwtGh6AGwcZT0ATgabxMRE2NnZwdXVFbNmzULWrFmxbt06LFiwAHv27EGDBg2SRXT8+PGYMGHCf/5miwHwv2/txYTcrZTatN3n8fP+y3Avngt/yFT70eILIjU+IyWlsjmBF7AwOBaViubE9u+0KcKyxTWQmytPMBBT0GlNGUjldme1cUFbd20ommxx3XnNiVIXtnYtjLlJqAuTe6bnzH2Ie2D54kPzgy5iflCMbO5ppUIzvNZHil0lfM5S7KbURw+A9QBY9h56/vw50qVLh4wZ/6Wb8fX1FQridu82cNombanpBJjmMvk42oNUjKfU/s1Fs8eKbtrkPcle8FQ4oOK4ADxLRuxizF9nsfbYNdQtmw+re0o/kUmFEKY4ZbFU6+KObvjSpaBJiMSqbDu/90DFwjlMjmHV4WXiO1xMeCqkQOh5+qxQZWcnJerC5J7iNNYfzxPfYX77ymjpWpidI4wt0aAwj10GhElMkVMjNc7YfVnmAs7dRt+1YUJdReyUxp8V2MsyJKOzHgDrAbCM7WK86/Dhw+Hv74/IyEhJ9mx54830j8aSkEuoXDQn/jJxSthnTSgCoxLQtkoRzGpbSRJ2eif+CFB56tFNKuAbz3/lqb///RR2RNyELoagfg1qTw/GjUcvIVVSWlyMdnxUA9hnz6TeCd2CTSCQEnVh0glaU4C478Id9Fh5UpjChcm+yJjOtGCHOIUrYLAnyhXgKxfOagOJVR5JfQWps+DdbDkOUYudngKRDIJnzpxBrVq1hGDXw+PfYpB69erB3t4eGzdulIS7LW88ORruVHa3r2cpjJSQuyYJXL2TagQoJ2XSnELKQtCtZnFMaFFR9XNSs4FWSw7jVNwjIUeR5CqaapSdgzBwxExujHRp05gaov89lSDwL3VhRoT6eac4a/LSRV6+SPtnoAecCml3kyB3OaJuPkGTnw4Kww6OqIeiubOYNCGmaiSF1aTA2hqauC5AK6pBW45D1K65HgAng+D79++FAJikQSxduhR58+bF8uXLsWTJEpw8eRLOztKoiWx54xGFK0JDQyiTzk80pIYYa95z94OIK1hTsYLaD5Y1jP92XRh2n72NpDmFtMBRLi2RNcxZax/7rg1FwLkEfOVWBHPamb79WH/8GkZvO4u8WU0HOVrPRX+eeREQ00makkM+E/8IzRcdFhy29JsEIg1MCj9J+/PbmqhSPLdJoHdH3sK368ORPu0XuDhZm1QCk05J6HD36WtUnWJQRSU3p+QGlXez5ThELXZ6AGwEwbt37+LHH3/Erl278OjRI7i5uWH69OmoU6eOZMxteeMduXQPHX85LmBBOE4J16mx5j55D+49S4TWVe2SFyqVdqS5vknlkOvMDMb1B9Kv7VMpfJKmbQxjY4NpQZBjwezYNUj6d40kZ/ROVo1AeNxDtP5IXUhOgMlLkrG2L/oOeqwypBWQADFDOsu9SRDzykvNlV995CrG/X0OhXJkwpGRyRelW+Jii4tcV8pUvlM6H1uOQ5RiQsfpAbBaBFMYb8sbTyrFE/lyKzN6N969/4Cfu1RBI6cCHBHXTctBYEFQDOYFXfxP9TUl3J/XvhJaudoWfZ8cfFj0lZO3SZ5H1PnIVbdegMgCfduyIc579R9cB+UL/EtdmHSmf4TF44ctEUKOqTVweXvMCEb8w5cY09QRvTxKmly4WQHRWLzvklUy1VD2CqmCJibBMNHBluMQtdjoAbBaBFNpAPzweSJcP15b/dGvJtxLJH9t9fjlG1SaYCDY39y3JqqVNH29xXFJdNMiBOh1e75sGUHEMEgzxwmFLS+K3Ap3KpWqF4za8q5QNjfC0lFhrL8weF2v6vBwyGvUEOVelyvDrcwz9aOM1SMYsyyHhUi9d2wtUHo6qcG+2qfrAbBxBPUAWO3uSqUB8Pv3H1BuzG68efcByzq7wbdi8hRP1+4/R91ZIQJKe4Z4wsHeOqp1OW4LizEtpuQhBVeE+ur+s9eoMtmQo7atfy24FstlMf5aoyPB0QnouSpUcD1mSmOkN1HU1mLRIUTEP8Z39UpjeCPTRXPWiInus3IEKHWhKWqzqbvOY/mBy9BScUz5rIDvNoTjnzO3JDPPdF95AiEX7qJj9WKY2kpaTY4a/1iOJZLjhO3l+/plMMyEiBSL5+oBsB4As9hHsm3Y+sarMXUvbj95hUktnNDFiBzy6euPQBSJSEsquSsbUH0AUwTCrj3EV0uPCDbD/LyRJ2tGXLr7DA3m7Bf+bd8PXiiZ147pM1ObsbM3HqPpwkPCtI+OrI+COTKnCEGtaXtx8/ErTGjuhG61UpYYT21Y6vMFKHWh35cV0LvOv9SFSbEZuvk0tobfgK9TASzrUsXioZu0Mwq/HrqC6iVzY1Pfmib9bbLgIKJuPcFgbwcM9i5rsr8ldaDBe6fqxTBFg+Dd1uMQNWurnwCrQc/EWFvfeE0XHsTZG08wsIEDhvok/yUk5niUcgLGcTl000kQiLv/Ap6zDHLINKdQTC90aowPclkJvZClLu6dJ69Qbepewb2/B9SGSxHjVd8kX76sn+FWZWknNzR2Ni2cYanz1v3igwBNFTBFKWltJ6TLD1zC1F3RKJEnC0IkyK9T2W5y+ktOga2pDdl0GttO3cCXzgWxuJMbd9dtPQ5RA6AeAKtBL5UHwFK+ZLeGx2Po5ghky5QOkeMbcURbNy0XgReJb+E4NkAYRnMK955PQK/VoSCsdrFTmgiKRXpTjsDbd+/h4LcbHz4AK7q6w9vR3qgxMR1USnn1yr3RR1o7Av3WhsH/3G20diuMue0qG50OpTIcWL8Mhmpwza4W1+2nb2DQxtPInD4toiY2SpFWkxRUO4zehfcSPlNq/eIxfsKOc1h5+Cpqlc6DDd/U4PGIz2zqAbBxiPUAmOP2s/WNR6qMSbVxQ0d7LDcih0yutcj1VrHcWXBgRD2OaOumlSCQVC71z7B4DLOi6nElc9Z6jNTTqujbT+A73yAIcGB4PRTLY1oQQOu56M8zLwJ+f0Vi3bE4eJbNhzUpyJRbWyrN8cv30X75MQFcUwpp4luV7d/VFpggrKkt3BuDOXsuSFZqaQAAIABJREFUonyBbPAf7MnddVuPQ9QAqAfAatAzMdbWN9703dFYtv8S3IrlxNb+tZNFY07gBSwMjrVKuhqOW8NiTNedtQ/X7r8AzSmkLyzF82TBfglXkRYzEQt2RGq+4oGLd9H1txPCTKIn+SJTetOSsBY8bd01DghI4YkmqTTlx/jj9dv3WPi1K5pVKsTBE7Ymr957Dq/ZhmLpwCGeKJtCsbTcvHq2nqq3tvbYNRB+8ALZM+HYKP4cxrYeh6hZET0AVoNeKg+AVxy8jMn/nE/xdHf0tkisPx4Hr3L5sKrH/9s7D/icrjeO/xB7hCJGzJLYiR2xiRGqpajVWqUtNepvtWZtRa3WprSlqFWrxJ4hgpghIbFH7VWb9P85N+7r9eZNcu+b932Te+/vfD79fJTnnnOe73ly89xznvM8FR1Im13bQkBcghOX4b6q8T4GNCgG0wdLHles6f62DLgtffOZaAJKQoWEnLz7znAhWk5sBBYFXcTg1SfhljE1gt+kLrSU/ff5K4hsEaIt/sIHlQvFni4tqZB+9vK15LSLtrBTRVTzyB7r1OTMKiJMSxT5iC+zSlLRUZ7H+uPX0H3xEaR2SY7wUQ0cPj06wLEjpgPsQPPTu+HJcVvpUom4LX+rJLv9EYK/T1zHx2XcMbll7DFrDlwGdh0HActSvXLlsviOWAlVOQGlOUtn7ozEuIAwFMqeHtv61FQ+ACUNQ8Ba6kJL5c1TT8a3m5qUwIl88SJv/ITmXvikfN5Yp6Y2t3ZS0lHMZV/EbbSZF11F9fQIf6RN5diTHr37IQlZXzrACaEXz7N6NzzzH+TQ4fWR3ko55DZzg7Av8g46VimA7z8s4UDa7NoWAvIOvVx5rMeSI1h37JrifJy2jGm0Z5RWrZIvx1R6/z0s/TL+VFBG40h9IZ3WyKkLQ4bUxXtWsrSoKZmclJjWm7wLZ278i771PNG9tkesU3N2DK29GZ269hANf4qO9Q/8rjbcM8edGjGh4+vdD0kIHzrACaFncAf4zI1HqDd5t0RhV7+ayJ81Zs7YBlP34PT1h+hT1xM9/GJ/qTlwGdh1HAQmbzmDqdvOokTuTPi7ZzW0/eUA9py9jfa++TG8cUmyswOB3/ZdwPdrQ5HbNQ32DYg95k8uBvCRd2781LqMHUZmF3ojYJ66MLbd3S2nbuCL37WXyUV+93xWKR9GNYm9uIV8SlXNIxsWdvLR3BJff/AUvmO3S/Ne36MqSrq7OlQHOsCx46UD7EDT07vhmadtWtnVF+XyxyxzbCqW0aQk2lbK70Da7NoWAvKFDDmm0JQ+KY7czraMY+RnNp64jq5/hCBlimRSzGIyEbxopbWYtR/BF+6ic9WCGNyouJGRUfdYCJinLvyjsw+qFI4Z36vVEIF+y49h+eErqFs8B+bGklVIYLEM29KasZjHO8dX0toeuundD0kIIzrACaEXz7N6NzxRDlnkOBV5GWd9Vg7+JXPGIFJk8EbpNvK0NmXQyCvp30Z2oDkkya4DTv6DLosOS/l+RTnkGj/uwOW7T/H9h8XRsUrBJDlnrU3KvLhIbMfWQqdaP+7E+duPMbBhUXxZvZDW1OR8nUSg+NAAPHnxGlNblUbj0u4xRp2+IwITNoXDwy0DtvSu4aRZJXyYHzeFY9qOCHjlccXaOC7gfjwjEEcu3UfXmoXwrb82y4UXGxKApy9fSyc94sTHkU3vfkhC2NEBTgg9gzvAQv2Ko7fi5qPnGNWkJD6z2OF9+uI1ig2Nvtkb226FA/GzawUELJ0zkRbt0bNXmNzSGx+XyaOgB4rER+Dy3SeoNj664t6mXtVRJGdGq49Y5mSOr1/+uzEJVB+/A5fuvk1daElhxLpTmB94HlqLJbc8jYptdav8sB1X72v7I13O0zyicQm083VsyXM6wLG/J+gAO/AdagTDiyvHqXhJiZeVaBt6VkPx3JkcSJtd20LA/Ma4WCP5csaCDhVQq6ibLV3yGQsCSlI8PX7+CiXepK7ixyJNKC4CTWcEIuTSfXSpUQjfNYi5A/rN0iNYc/QaPvDKheltHF9q116rJccui+KTIlTIJUXyGF2LHMdFhgTgxasoSTehoxab/Hvzf3U88U0dx96NMYIfYqsN0AG2lZyC54xgeCJxv0jgb+3igtYTlitYYs2LmDteImm+yAIh2qqvK6Nsviya1y+pKFBq2CZpZ33iJ95oVi7mzroIfRAhEKJt+V91eMRRCCCp6MR5JA4BOQa2ebk8+PET7xiT+GzeAeyN0N5F1hNXHuDDaXslfYIG+CGna5oYuj148hLeIzZLf7/sK19ULBjz3knirIq6UT+dF4TAiDvoULkAhn3k2OxIRvBD1NF/K00H2FZyCp4zguH1WXYMK0OuwL9ETsxqW+4dKnvP3sZnv0TnO2RlKwUGk0gicjzaN34eUkYI0Xb0rYmC2WJm9UikKWp+WL+JOxF567EUsyhiFy3bO6Vgh9aDa7qUmteZCjiGQHzFheTMO87YXbSnhuYljld3q4LSVkocn73xCHXfZB7a2bcmCmj0HdV9cQjWH7/ulHSTRvBDbLVDOsC2klPwnBEMb+zG05i96xzK5c+ClV0rv0NF5JMVO4ppU6bA6ZHWC2UowEgRBxOoNn67dPFNHCf+ffy6NNqRIXWRxUqOUQdPRbfdt54ThP3n7uDzKgUx9MOYGR7k6lCpRHWokf6xZorQLSAqppiAnLqwpHsmrO9RLcZzcd3LUDxIIgiKy9Seby5Vz25bDvVLxLxUHRhxG5++KSJxakR9pEvlkggzTfiQziw4ZAQ/xNYVoQNsKzkFzxnB8ORyyPmzpsOufrXeofL7/gsYuib+/KcKUFLEgQTkW9VFcmRE+I1H0kiRYxpKmSHY7ENAjsts5JUL06zEZc7fex4j1p9Cnixpsffb2vYZlL3okoB8WSxHptQ4MLDOOzqKGFmPQRvxSsrMUxb+JbUVI+s7dhuuP3iG2C6HrT5yFb3+PIoMqV1wcnh9za7vpM3h+Gl7/Bkv7KGgEfwQWznRAbaVnILnjGB48gspfaoUCLUohzx161lM3noGxXNlwoZvYu5UKEBIEScQ+PL3Q9h86gZSpUiOF6+jkCmNC44P0+4vFycgUz3EqPWnMG/veSlmUcQuWrYfNoZh1q5IlMmXGX99XUV1/3zAOAQCTl5Hl0UhcEkenVc6udmHqtZjZBtPD8Sxy/fxdc1C6G8lxdmc3ZEYsyFMCs8SYVpabQsCz2P4Oud88BrBD7HVDugA20pOwXNGMDzzOF/LuubD1obi130XUKVwVvzRuZICYhRJDAID/zqBxQcumYa2tpufGPPS05jx/eLuvewoVoVcRf0SOTC7bXk9qU5d7Ezg0IW7aD5rv9SrZahS5K1/4Tdxl/Rv2/rUQKHsGew8umO7ky/4NS3rjkktSscYLL4PScfOzn69O3Mn2wh+iK0rQwfYVnIKnjOC4YX98xD+U6Lrmu/pXwt530tnItNr6RGsPnoNsR37KkBIEScQmLTlDH56c/lNDOedxxVr4khE74Qp6W6IuE5KhLJyGVhRLXFkE5ag1p0B2FGhC7cfo2YsGUOCz99Fi9nRzvExDV6m/H7NSfy2/yKqFs6GRZ1jljnuueQI1h7T/u+UneE30WHBQWmdxC6+iP13VDOCH2IrOzrAtpJT8JwRDO/2v89RftRWiYZl6iw5RRp/qSswlkQUWbj/AoasCTXNoLpndvz+ecVEnJH+ht4XcRtt3lzeCR1eH+lTv3t5x3/KboT98wh96nqih59j84Lqj66xNPr3+SuUfJMzevEXPqhc6G05ZPPwiLOjYy+7nVSJyVXsCrtlwFYrVexazdmPoHN3Y71MmlT1spyXCPMQ4R6iBQ/yg1vGmCnf7KWLEfwQW1nRAbaVnILnjGB45jd357Qth3pmN3c/mrYXx688QM/ahdG7XhEFxCiSGAQ2nriOrn+EmIYWpTlFiU42+xGIL31T2ZFbcPfxC/zQtBRaVcxnv4HZk+4IiItuxYduslpKd1HQRQxefRJuGVMjeNC7F+S0AGLl4Svos/wYMqZ2wQkrl9zkdIKiAIgoBKLVprQ6pD30M4IfYisnOsC2klPwnFEMr8Lorbj16DlGf1wSn/rkN5GR02t9/2FxdKxSUAExiiQGgYMX7uKTNzGFYvz2vvkxvDGP4e25FnFdThJVrUT6J9HmdyiP2kVz2HNo9qVDAvK7dWij4vi86tt3q3zxuFiuTNiowYvH5mnOrJ2UxFdQRitL/ejZS5QaFl3QY+mXlVDp/awOm7pR/BBbANIBtoWawmeMYnixJV4Xx3TiuG5Ky9JoUsZdITWKOZuAeRUyMXZPPw/0ruvp7GnoejzzEq7T2pRBI6/cJn2v3X+Kym9Khq/vURUl3V11zYLKJZyAnLrQMltCfDG0CR/ZsT1E3HyEOpN2S4Ns71MD75td4lNSUtyxs7Nf785MV2cUP8SW1aEDbAs1hc8YxfCsXeAx39X6tWMF1CzippAaxZxNwHw3Qoxtuavk7Pnodbyq47bjyr2nsDwReScecKAf3DI5Lh5Qr2yNptcXvx/CllM38Em5PJhgVg652+IQqZhN49K5MbWV9sKYzN9FlvHNl+48QfUJO6Sl3tSrOorkzKjpZRd3Z8QdmrFNS6G1A8OejOKH2GIMdIBtoabwGaMYXu8/j2LVkatoUDInZn4WXQ755qNnqDh6m/TnNd2qwNtKWUuFGCnmYAJiN6LY0AA8exkljTS5pTc+LpPHwaMar/umMwIRcum+VApZlESWm3BkhEMj0rmeHc0CJMazDPUaD1h1AkuCL6FWkexY0PHthVU9XBIrMTQAj1+8jnFyGFf6N/UEE/+JepN34cyNf9GvfhF0q1XYYRMyih9iC0A6wLZQU/iMUQxvzIbTmLP7HCoUyILlXaLLIZ+58Qj13tRst0yPphAfxZxIQN6dFEMu6FABtYpyx97e+OUcp83K5sHEFt6m7v84cBGD/jqJbBlS49Bg7V1csjcn9hc/AbmSWCl3V6zrUdX0gLOcqvhnaLtE7Yk7ce7WY1hedJMv66ZMEV0AJFkybVeqFOnqRNq6L6oVxKAPYpZHt53gu08axQ+xhRcdYFuoKXzGKIZnLcl/0Lk7aDUnSCJ1Ylg9ZEyTUiE1iiUGgSbTA3H08n1paMt0dokxHz2OOWT1SYgyttU8smFhp7c5TidvOYOp286yYqIeF91BOsll5nO5psH+AX6mUcqN3II7Gs8m0npOEPafu4MOlQtg2EclTLr9tu8Cvl8bityuabDPTGcHIXZ4t/IHcfNyefCjWRiLvQc2ih9iCzc6wLZQU/iMUQxvVcgV9F72buoareejVLjEuhHr/NshbD19Q9LH8vKJbpRMZEV+3nYWE7ecQdGcGRHQq7ppNvJxds0i2fGr2XF2Ik+XwydhAvJuqChfHj7KX9oNFSkpCw/agP/+A+a2K4+6xbWZTeR/fx7FXxYhdWIpJmwKw/QdkVI4nQir03obsOo4lgRfhl9RN/zSoYLD1DGKH2ILQDrAtlBT+IxRDG/P2Vto+0uwRCVspD/SpEwhxaeJX+w81lVoLIksJr+MxTQsy6sm8tR0M/yfBy/h25UnkDV9KhweUtekl/zx0aJ8Hoxv/jY0QjeKUxG7EzBPXShXfIurKJHdJ+DADn/YGIZZuyJRNl9mrPr6raPbb/kxLD98RXLshYOv9TYuIAwzd8bU0956GcUPsYUbHWBbqCl8xiiGd/r6Q4hUaKLJ8b5yRR/PHBmw+X81FBKjWGIRmLg5HD9vj5CGjxzDi1iOWIcdYTfR8dfo8qeiSlfKFNHlTxtP24tjVx6ge63C6FufBWMcwV5vfZqnLhQV00TlNPN7F7v61UT+rOk1qfaCwPMYvu4U3DOnReB3tU06dFgQjJ3ht/CpTz6M/riUJnUzn7QcOvh+tvTY3remw/Qxih9iC0A6wLZQU/iMUQxPFMEQxTBE++vryiiTLwtG/30Kc/ecR8WC72HZV74KiVEssQjIMYWZ0rjg+LD6iTUNXY978uoDNPp5r6Tj/gG1kcs1rfRn37HbcP3BMwz/qATaVy6gawZUzj4EzNOFLfmiEnwLZcW+yNtoM/eANMDJ4fWRwaLctn1GdnwvcniHS/Loy27JRXoUAA2n7sGp6w/Rq44HetXRfp7yZYcuo/+K48iSLiWODK3nMLBG8UNsAUgH2BZqCp8xiuGJ2DOPQRsQZRZ71nf5Maw4fAX1S+TA7LbaP65SuOSaFbt6/ynazw+W1qtf/bcpujSrUBKc+M2Hz1BxzLupAaOi/pOqwL2K+g8zPy2LBqVyJcGZc0pJjYBIXVh0SACev4rCz63L4EPv3Fh37Bp6LDmC1C7JpVA0rWZJCLl0D01n7JOQi6woIoxONDlv7piPS6GNj/bLhW89dQNfLToshUQFDfAzOfr2tjWj+CG2cKMDbAs1hc8YyfDKj9qC2/++MCX17vzbQWw9fROtK+bF2KZeColRjAT0S8Dah+Ldxy9QduQWSemVXX1RLv97+gVAzexKoMoP2yE+XOXCKnrJkiB0ErqJJldGfPU6Ch6DN0oX/Oa1K486Gr3gZ24A4n0gNrcd/aFiJD9E7Q8YHWC1xFTIG8nw/KfsRtg/j9Cnrid6+HkgtqT/KvBRlAR0R0CEComQIXkXK+yfh/Cf8m78vO6UpkIOIdB4eiBEFcFutQpJpzax5QZ2yOAO7PTl6yjpVEQ4u7+0Lw+/Yjlw4+Ez+Lw5PVnbvQq88mR24Az01bWR/BC1K0cHWC0xFfJGMrzP5h3A3ojbaO+bH8Mbl0TtH3fi3O3HGNiwKL6sXkgFNYqSgH4JfPDTHoReexvHuPvMLbSb/24GFf1qT83sSUA+ZWtZPi/GNffCwL9OYPGBS9BDOj3LcAfz+HkRLpDTleXCldqSkfwQpUxkOTrAaompkDeS4cm5Gz8olQvTPy2LMiM2496Tlxjf3AstyudVQY2iJKBfAvJNdhHDKHaBRZy8iJfPmMYFJ3j5UL8L7wDNLPPIdll4GAGh/6BpWXdMalHaASM6r8tGP+/ByasP0dPPA73remJ72A18/ushiOJv4mKcnEHFeTPS7khG8kPUrhIdYLXEVMgbyfBMWR8KvIclX1YyJWTXS7yWimWnKAnESqD/imNYdugK6hTLgXnty2PGzgiMDwhHoezpsa2P41IhcUn0R0BOXeidxxVrulfFJ7P24eCFe/iy+vsY2LCYphXu9OtBbAu7CXl3e2nwJXy3KmYObU0r6aTJG8kPUYuUDrBaYirkjWR4s3dFYuzGMIichiu7VkYZXuxRYSkUNQqBHzeFY9qOCFM1q+HrQrEg8AJ8388qfTiykYBSAvKlNzlfrhx2NqBBUXxVQ9thZ3I4Rw3P7Pjt84r4adtZTLJSRVEpKyPLGckPUbvOdIDVElMhbyTDW3n4Cvq8OcoVZSprT9wlkdrWpwYKZc+gghpFSUC/BOR8y7ld02DfAD90WxyCv49fR+PSuTG1VRn9Kk7N7E5A2I2wH7kccukRW/Dg6UtMaO6FTzQedmbp8A5ZfRILgy6iumd2/P55Rbuz1HOHRvJD1K4jHWC1xFTIG8nwdp25JeWRFW3xFz6mhOwsq6vCYCiqewJykv+UKaKT/LecHYTgC3fRuWpBDG5UXPf6U0H7EThw7g5azgmSOjw8uA7KjYouRrSgQwXUKupmv4ESoSe5bLhcJOKrhYewKfQGmpfLgx8/YblwNUtiJD9EDRchSwdYLTEV8kYyvNBrD/DBT9FVrkY2LoEha0KlCwsRo1lWV4XJUFTnBA5fvItmM/dLWoYMqSulC7xw5wmzpeh83R2hXuStf+H35qRNVINrPTfaGdZDmrCd4TfRYUF02XBR1EPoduTSfXStWQjf+rNQjxp7MpIfooYLHWC1tFTKG8nwzKtctfPNj9/3X0TmdClx1IElHlUuB8VJINEJXL77BNXG75DmEdCrmlTx6smL15jSsjSalHFP9PlxAtohIMIdvIdvliY87MPiGLbulPTnvd/WQp4s6bSjiJWZWubHbjUn6J2iH5pWzsmTN5IfohYtd4DVElMhbyTDM6/UIy707D93BwWzpceOvrzZrsJkKKpzAs9evpZK2Io267Ny6LLosPTnxZ19ULlwNp1rT/XsSUCUQy4yJAAvXkVJFTeXBF+Wuj89wh9pU6Ww51BO7+ve4xemi9TLvvKFyDP/4nUUprcpiw+8WC5czYIYyQ9Rw0XI0gFWS0yFvNEMr9zILbjz+AWyZUgllUUumy8zVn1dRQUxipKA/gl4DduEh89eoWftwvhpe4Sk8Nbe1VHYLaP+laeGdiUgl0Muky+zFCKQPlUKhI7wt+sYidGZuXM/qklJDF59UprG8i6+qFCA5cLVrInR/BA1bOgAq6GlUtZohld/8m6E33hkouRX1A2/dKigkhrFSUDfBPwm7kTkrceoXdQN28NuSsoeG1oPrulS6ltxamd3Ao2n7cWxKw8kx/fxi9fI91467O5fy+7jJEaH1cfvwKW7T9CqQl4sPRi9u72zb00UyJY+Maaj2TGN5oeoWSg6wGpoqZQ1muF9Oi8IgRF3TJSalc2DiS14Y1el2VBc5wRazwmSQoTcMqbGzUfPkcolOcJH+iOZuDXKRgIqCMgFI+RHSufNjNXd9HHqJhf2KJE7k1Q+XLRTI+ojXSoXFYQoajQ/RM2K0wFWQ0ulrNEM75ulR7Dm6DUTJaZ2UmkwFDcEAcufkzxZ0mLvt7UNoTuVtC+Bb1ccx5+HondHRatTzA3z2uvj1K374hCsP34dLsmT4VXUf8iQ2gUnh9e3L0AD9GY0P0TNktIBVkNLpazRDG/k+lP4Ze95E6V+9YugW63CKqlRnAT0TUAuGy5ryVh5fa+3I7WbsCkM03dEmoaQSwc7ckxn9T1q/SnMM/t9IqqMbuelatX4jeaHqAFEB1gNLZWyRjO8mTsjMS4gzERpzMel0MYnn0pqFCcBfROYu/scRm84bVLSv0ROzGpbTt9KUzuHEFgQeB7D36Q/EwPoKU/uvD3nMOrvtz8nPgXfw59f+TqEo547NZofomYt6QCroaVS1miGt/zQZfRbcdxEaeanZdGgFFPWqDQbiuucwOojV9Hrz6MmLdtWyo+RTUrqXGuq5wgC645dQ48lR0xdD/6gGDpXe98RQzm9z7XHrqGnmW6NvHJhWpuyTp+H1gc0mh+iZr3oAKuhpVLWaIZnXr1HoFr6ZSVUej+rSmoUJwF9E9gXcRtt5h0wKdmnrid6+HnoW2lq5xACQefuQBSJkJueCqoEn7+LFrOjqyaK9nmVghj6IcuFqzUko/khavjQAVZDS6Ws0Qzv5NUHaPRzdDlk0Tb1qo4iOZnbVKXZUFznBCJuPkKdSbtNWo5rVgotKzBUSOfL7hD1Im7+izqTdpn6/v3ziqjumd0hYzm700t3nqD6hOiqiaJ916AoutQo5OxpaH48o/khahaMDnAstKKiojB8+HDMmzcP9+7dQ9WqVTFjxgwULqz8UpfRDO/Gw2fwGbPNRDR4oB/cMqVRY4+UJQHdEzAvYSuUXdChAmoVddO93lTQ/gQePHkJ7xHR5ZBF+7tnVZTI7Wr/gRKhR/OqiWL4SS280bRsnkSYibaHNJofoma16ADHQks4v8LhXbBgAdzd3dG/f3+cO3cOoaGhSJUqlSLGRjO8l6+j4DFoo4nNmVENpBynbCRAAm8JiCpXohzy81dR0l+u71EVJd314bRwnZ1LQKqYNjhAKhMs2oGBfsiho02H0iM24/6Tl5Juizr5oKoHy4WrtTCj+SFq+NABtkLrxYsXyJYtG8aPH48uXbpIEvfv30fu3Lkxf/58tGrVShFjIxpemRGbce/JS+ZsVGQhFDIqgarjtuPKvaeS+jwpMaoV2Edv37HbcP3BM6kzvW06+E/ZjbB/oquLMqTONnsxoh+ilBQdYCukgoOD4ePjg/DwcHh6epokRBiEl5eXtDOspBnR8OpO2oWzN/8Fk/srsRDKGJVA0xmBCLl0H8mTAWdHN0QK8Qc2ErCBwIc/78WJqw+QKY0Ljg/TV6GIdvODsfvMLYnKkSF1kSW9stNXGzDq9hEj+iFKF5MOsBVSq1atQrNmzfDkyROkTZvWJNGiRQvp79avX6+IrxENTy7zWsrdFet6VFXEiUIkYDQCXRYeRkDoP8ieMTUODqpjNPWprx0JdFwQjB3ht6DHQhH9VxzDskNXkCpFcoSPYrlwW8zGiH6IUk50gK2QWrRoEdq2bYvXr18jefK3Mazt2rXDtWvXsHXrVqt8Hz58CPGf3C5fvozKlStD7CjnymWMfLhDV5/E5lM3ULHge/ipdRmldkg5EjAUgR83hWHF4avwzJkBv3/uYyjdqax9CYjKguuOXYdXHlfMaVfevp0ncm9zdkVifuAF5HRNg9XdqiTybLQ5/PXr11GxYkWcP38eBQoU0KYSDpo1HWArYFeuXInmzZtb3QF+/vw51qxZY3U5hg0bJmWOYCMBEiABEiABEiCBpEJAbMRVqFAhqUwnScyDDrCVZZBjgCMiIlCo0Nu8gyIG2NvbG9OnT1e0A/zs2TOIXeCCBQvCxcXFIQsuf90ZaZfZISAVdkreCkHZSYy87QRSQTdkrQCSHUXI244wFXRlVN6vXr3CrVu3UKpUKaRJw7Sk5qZCB9jKD47Y5XVzc8OkSZPQqVMnScKWLBAKfiYTLML4ngQjVNUBeavClWBh8k4wQsUdkLViVHYRJG+7YFTcCXkrRmUYQTrAsSz1oEGDMHv2bCntmYib6devHy5cuIATJ04ozgPsDCviD7UzKL8dg7zJ27kEnDcabdt5rMVI5E3eziXA0SwJ0AGOxSbEBbiBAwdKhTCePn2K6tWrS6EPSS2InC9R5/5Qkzd5O5eA80ajbTuPNR1g57Imb+fz1sKIdIC1sEpxzFFknRChGr1790amTJn8cNk/AAAJtUlEQVQ0rk3Snz55O3eNyNt5vMnaeazFSORN3s4lwNG4A0wbIAESIAESIAESIAESMDQB7gAbevmpPAmQAAmQAAmQAAkYjwAdYOOtOTUmARIgARIgARIgAUMToANs6OWn8iRAAiRAAiRAAiRgPAJ0gI235tSYBEiABEiABEiABAxNgA6wRpc/KipKKrs8b9483Lt3D6JK3YwZM1C4cGGNapS0p33x4kWrKfDmzp2Lzp07J+3Ja2x2o0aNwtatW7Fz507TzI8ePYpvvvkGhw4dQtasWdGzZ0/07dtXY5olzela492xY0f8+uuv70zY3d1dyl3Lpo7A3bt3pZSa69evlzI/eHl54YcffpDe2aLRttXxjE86Pt607fgIGuff6QBrdK2F8yscXpGnWPxi6t+/P86dO4fQ0NAkVahDo3hjTHvdunVo0aKFxDhZsmSmf3d1dUXatGn1omai6zFlyhQppZ/Iuy07wHfu3EHRokXRpEkT6d+CgoLw9ddfS/Yvfpmx2U7AGm/RW7ly5eDv748ePXqYOk+RIgWyZ89u+2AGfbJevXq4efMmpk2bJvET+eTFxkVISIj0/7Rt+xpGXLwFa9q2fXlruTc6wBpcvRcvXiBbtmwYP348unTpImmQVEs1axCv1SmPGTMGy5Ytk3Zr2OxP4OrVq9JO+p49e5A3b17kyJHD5ACPHTtWchpEJUYXFxdpcLGjtmrVKoSFhdl/MgboMS7eoghQhgwZsGTJEumjg812AhEREfDw8EBgYCAqV64sdfTff//B09MTrVu3lj6eadu287V8Mj7e33//PW3bfrg13xMdYA0uYXBwMHx8fBAeHi69SOUmjtTE8ZrYGWOzL4FWrVohefLkWLx4sX07Zm8SAbHDLhwucRw/YsQIydmVd4AbNmyIzJkzv8NehEjUrVsXN27cgJubGymqJBAXb/FRUaxYsRjvF5VDUPzNxsS+ffvg5+eH1KlTm5iIUDWxwy5OlGjb9jMVsREUF+/u3bvTtu2HW/M90QHW4BKKna9mzZrhyZMn7xy/iyN68Xci1ozNvgRKlCiBXLly4fnz5zhz5oy0qzNkyBDUr1/fvgOxN3To0OEdB1h81DVo0ADjxo0z0Tl16hTEmhw8eBDly5cntQQQsOS9fPlytGzZUjpd2rhxo/ThJz5CxMeJCPlhSxgBwVe8q9euXYtBgwbRthOGM96nzXk/e/aMth0vMeMI0AHW4FovWrQIbdu2hTiqFL+c5NauXTtcu3ZNukDEZj8CIuQkffr0KFOmDCZMmCAdoYk1mDp1KrZs2SLt7rDZj4ClQyZ2y9q0aSPtDMtN7JwVKlRICpmQLxPZbwbG6smStzgmHj16NETYj3B8z549K104FKEp27dvf+edYyxSCddWhEKInV/xzli9erV0aZm2nXCusfVgyZu27TjWWuyZDrAGV23lypVo3ry51R1gsUO5Zs0aDWqVtKf8+PFjKf7U/BhT/CITF+LELhmb/QhYOmSlSpWSHDFrO8DiIpH4MGGznYAlbxGj+uDBA+loXm7i4qGvr690AVGEX7GpJyDey8LZrVSpknRKJ+J/advqOSp9whpv2rZSesaQowOswXWWY4BFwL/YBZOb2Anz9vaWLlWwOZ5Av379EBAQgBMnTjh+MAONYOmQCedXpD5buHChiQJjgO1nEJa8rfUsPgDFyceKFSuk8Cs2dQREBgiRxq9p06bS6ZH8IU3bVsdRqXRsvGnbSgkaQ44OsAbXWezyios/kyZNQqdOnSQNmAXCcQt5/Phx6Qa3cHbNj9tr1aolZStYunSp4wY3YM+WDpnImTpr1ixERkZCpOISbcCAAfjrr7+YBcIO9mHJW+xSitRzmzZtMvUuLiQKexex1+KCHJtyAjNnzpTS9onc1ZMnT34nhIS2rZyjUsm4eNO2lVI0hhwdYI2us7g8MXv2bMyfP18q0CB2I8XNebEbmSpVKo1qlTSnLYqOCAdY7IKJl6tIQTdnzhwp24a4hCWOMdnsR8DSIRM5VEX+zo8++kjKdy1OQLp27So5xe3bt7ffwAbtyZK3OJ4XrEeOHAmR/URc+uzWrZsUAvHHH38YlJJtagt2JUuWRKNGjWJk5xEhEGIzg7ZtG1trT8XHW9wZoG3bj7fWe6IDrNEVFBfgRC5UUQjj6dOnUuEAEfognGE2+xO4desWvvvuO2zYsEHabS9btqxUzalatWr2H8zgPVo7khcfGmIH7ciRI1I2jj59+kCkNGJLOAFrvMU9A3ERTqREE7HAYudMZIFIkyZNwgc0UA/iIqHYrLDWxMebqLZH27afQSjhTdu2H2+t90QHWOsryPmTAAmQAAmQAAmQAAmoIkAHWBUuCpMACZAACZAACZAACWidAB1gra8g508CJEACJEACJEACJKCKAB1gVbgoTAIkQAIkQAIkQAIkoHUCdIC1voKcPwmQAAmQAAmQAAmQgCoCdIBV4aIwCZAACZAACZAACZCA1gnQAdb6CnL+JEACJEACJEACJEACqgjQAVaFi8IkQAIkQAIkQAIkQAJaJ0AHWOsryPmTAAmQAAmQAAmQAAmoIkAHWBUuCpMACZAACZAACZAACWidAB1gra8g508CJEACJEACJEACJKCKAB1gVbgoTAIkQAIkQAIkQAIkoHUCdIC1voKcPwmQAAmQAAmQAAmQgCoCdIBV4aIwCZAACZAACZAACZCA1gnQAdb6CnL+JEACJEACJEACJEACqgjQAVaFi8IkQAIkQAIkQAIkQAJaJ0AHWOsryPmTAAmQAAmQAAmQAAmoIkAHWBUuCpMACZAACZAACZAACWidAB1gra8g508CJEACJEACJEACJKCKAB1gVbgoTAIkQAIkQAIkQAIkoHUCdIC1voKcPwmQAAmQAAmQAAmQgCoCdIBV4aIwCZAACZAACZAACZCA1gnQAdb6CnL+JEACJEACJEACJEACqgjQAVaFi8IkQAIkQAIkQAIkQAJaJ0AHWOsryPmTAAmQAAmQAAmQAAmoIkAHWBUuCpMACZAACZAACZAACWidAB1gra8g508CJEACJEACJEACJKCKAB1gVbgoTAIkQAIkQAIkQAIkoHUCdIC1voKcPwmQAAmQAAmQAAmQgCoCdIBV4aIwCZAACZAACZAACZCA1gnQAdb6CnL+JEACJEACJEACJEACqgjQAVaFi8IkQAIkQAIkQAIkQAJaJ0AHWOsryPmTAAmQAAmQAAmQAAmoIkAHWBUuCpMACZAACZAACZAACWidAB1gra8g508CJEACJEACJEACJKCKAB1gVbgoTAIkQAIkQAIkQAIkoHUC/wfE6g6dqGKrywAAAABJRU5ErkJggg==\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "fig, ax = plt.subplots()\n",
    "ax.plot([np.sum(_flg) for _flg in blast_keep_dic.values()])\n",
    "ax.set_ylim([0,30])\n",
    "fig.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "cfadcb1f",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "postanalysis",
   "language": "python",
   "name": "postanalysis"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
