{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "9153e2bb",
   "metadata": {},
   "source": [
    "# Design CTP-14, mouse, next to CTP11,12,13 Chr7 regions\n",
    "\n",
    "by Pu Zheng\n",
    "\n",
    "2022.12.12\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "088da425",
   "metadata": {},
   "source": [
    "# Table of contents\n",
    "\n",
    "\n",
    "> 0. [Minimum required packages and settings](#0)\n",
    ">>\n",
    ">> 0.1: [import required packages](#0.1)\n",
    ">\n",
    "> 1. [Extract region sequences](#1)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3b459614",
   "metadata": {},
   "source": [
    "<a id='0'></a>\n",
    "# 0 Minimum required packages and settings"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4846ea43",
   "metadata": {},
   "source": [
    "<a id='0.1'></a>\n",
    "## 0.1 load required packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "2971864f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "23100\n"
     ]
    }
   ],
   "source": [
    "%run \"..\\..\\Startup_py3.py\"\n",
    "sys.path.append(r\"..\\..\\..\\..\\Documents\")\n",
    "\n",
    "import ImageAnalysis3 as ia\n",
    "%matplotlib notebook\n",
    "\n",
    "from ImageAnalysis3 import *\n",
    "from ImageAnalysis3 import library_tools\n",
    "print(os.getpid())"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bd4cf4d8",
   "metadata": {},
   "source": [
    "<a id='1'></a>\n",
    "# 1 Extract region sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "02b04f27",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Use pool_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\n"
     ]
    }
   ],
   "source": [
    "## Some folders\n",
    "# human genome\n",
    "reference_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl'\n",
    "genome_folder = os.path.join(reference_folder, 'Genome')\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain'\n",
    "if not os.path.exists(pool_folder):\n",
    "    print(f\"Create pool_folder: {pool_folder}\")\n",
    "    os.makedirs(pool_folder)\n",
    "else:\n",
    "    print(f\"Use pool_folder: {pool_folder}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "997a4dad",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "create library folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\n",
      "create sequence folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\n",
      "create report folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\reports\n",
      "-- library_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\n",
      "-- sequence_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\n",
      "-- report_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\reports\n"
     ]
    }
   ],
   "source": [
    "# folder for sub-pool\n",
    "library_folder = os.path.join(pool_folder, f'mouse_brain_relabel_chr7')\n",
    "if not os.path.exists(library_folder):\n",
    "    print(f\"create library folder: {library_folder}\")\n",
    "    os.makedirs(library_folder)\n",
    "# folder for fasta sequences\n",
    "sequence_folder = os.path.join(library_folder, 'sequences')\n",
    "if not os.path.exists(sequence_folder):\n",
    "    print(f\"create sequence folder: {sequence_folder}\")\n",
    "    os.makedirs(sequence_folder)\n",
    "# folder to save result probes\n",
    "report_folder = os.path.join(library_folder, 'reports')\n",
    "if not os.path.exists(report_folder):\n",
    "    print(f\"create report folder: {report_folder}\")\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "print(f\"-- library_folder: {library_folder}\")\n",
    "print(f\"-- sequence_folder: {sequence_folder}\")\n",
    "print(f\"-- report_folder: {report_folder}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "e988dce2",
   "metadata": {},
   "outputs": [],
   "source": [
    "sel_chr = '7'"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6f849117",
   "metadata": {},
   "source": [
    "<a id='1.1'></a>\n",
    "## 1.1 select regions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "cd85eb01",
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm.notebook import tqdm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "4e404415",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- load sequence: 1, size=195471971\n",
      "-- load sequence: 2, size=182113224\n",
      "-- load sequence: 3, size=160039680\n",
      "-- load sequence: 4, size=156508116\n",
      "-- load sequence: 5, size=151834684\n",
      "-- load sequence: 6, size=149736546\n",
      "-- load sequence: 7, size=145441459\n",
      "-- load sequence: 8, size=129401213\n",
      "-- load sequence: 9, size=124595110\n",
      "-- load sequence: 10, size=130694993\n",
      "-- load sequence: 11, size=122082543\n",
      "-- load sequence: 12, size=120129022\n",
      "-- load sequence: 13, size=120421639\n",
      "-- load sequence: 14, size=124902244\n",
      "-- load sequence: 15, size=104043685\n",
      "-- load sequence: 16, size=98207768\n",
      "-- load sequence: 17, size=94987271\n",
      "-- load sequence: 18, size=90702639\n",
      "-- load sequence: 19, size=61431566\n",
      "-- load sequence: MT, size=16299\n",
      "-- load sequence: X, size=171031299\n",
      "-- load sequence: Y, size=91744698\n",
      "-- load sequence: JH584299.1, size=953012\n",
      "-- load sequence: GL456233.1, size=336933\n",
      "-- load sequence: JH584301.1, size=259875\n",
      "-- load sequence: GL456211.1, size=241735\n",
      "-- load sequence: GL456350.1, size=227966\n",
      "-- load sequence: JH584293.1, size=207968\n",
      "-- load sequence: GL456221.1, size=206961\n",
      "-- load sequence: JH584297.1, size=205776\n",
      "-- load sequence: JH584296.1, size=199368\n",
      "-- load sequence: GL456354.1, size=195993\n",
      "-- load sequence: JH584294.1, size=191905\n",
      "-- load sequence: JH584298.1, size=184189\n",
      "-- load sequence: JH584300.1, size=182347\n",
      "-- load sequence: GL456219.1, size=175968\n",
      "-- load sequence: GL456210.1, size=169725\n",
      "-- load sequence: JH584303.1, size=158099\n",
      "-- load sequence: JH584302.1, size=155838\n",
      "-- load sequence: GL456212.1, size=153618\n",
      "-- load sequence: JH584304.1, size=114452\n",
      "-- load sequence: GL456379.1, size=72385\n",
      "-- load sequence: GL456216.1, size=66673\n",
      "-- load sequence: GL456393.1, size=55711\n",
      "-- load sequence: GL456366.1, size=47073\n",
      "-- load sequence: GL456367.1, size=42057\n",
      "-- load sequence: GL456239.1, size=40056\n",
      "-- load sequence: GL456213.1, size=39340\n",
      "-- load sequence: GL456383.1, size=38659\n",
      "-- load sequence: GL456385.1, size=35240\n",
      "-- load sequence: GL456360.1, size=31704\n",
      "-- load sequence: GL456378.1, size=31602\n",
      "-- load sequence: GL456389.1, size=28772\n",
      "-- load sequence: GL456372.1, size=28664\n",
      "-- load sequence: GL456370.1, size=26764\n",
      "-- load sequence: GL456381.1, size=25871\n",
      "-- load sequence: GL456387.1, size=24685\n",
      "-- load sequence: GL456390.1, size=24668\n",
      "-- load sequence: GL456394.1, size=24323\n",
      "-- load sequence: GL456392.1, size=23629\n",
      "-- load sequence: GL456382.1, size=23158\n",
      "-- load sequence: GL456359.1, size=22974\n",
      "-- load sequence: GL456396.1, size=21240\n",
      "-- load sequence: GL456368.1, size=20208\n",
      "-- load sequence: JH584292.1, size=14945\n",
      "-- load sequence: JH584295.1, size=1976\n",
      "- 66 sequences loaded.\n"
     ]
    }
   ],
   "source": [
    "# prepare sequence reader\n",
    "reload(ia.library_tools.sequences)\n",
    "resolution = 0\n",
    "flanking = 0\n",
    "\n",
    "seq_rd = ia.library_tools.sequences.sequence_reader(genome_folder, resolution=resolution, flanking=flanking)\n",
    "seq_rd.load_ref_sequences() # load genomic sequences"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "cf743401",
   "metadata": {},
   "source": [
    "<a id='1.2'></a>\n",
    "## 1.2 input region information from previous codebook"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "c5e958d6",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>name</th>\n",
       "      <th>id</th>\n",
       "      <th>NDB_784</th>\n",
       "      <th>NDB_755</th>\n",
       "      <th>NDB_826</th>\n",
       "      <th>NDB_713</th>\n",
       "      <th>NDB_865</th>\n",
       "      <th>NDB_725</th>\n",
       "      <th>NDB_817</th>\n",
       "      <th>NDB_710</th>\n",
       "      <th>...</th>\n",
       "      <th>NDB_479</th>\n",
       "      <th>NDB_562</th>\n",
       "      <th>NDB_608</th>\n",
       "      <th>NDB_460</th>\n",
       "      <th>NDB_563</th>\n",
       "      <th>NDB_592</th>\n",
       "      <th>NDB_368</th>\n",
       "      <th>NDB_436</th>\n",
       "      <th>NDB_629</th>\n",
       "      <th>NDB_604</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1:3742742-3759944</td>\n",
       "      <td>1</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1:6245958-6258969</td>\n",
       "      <td>2</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1:8740008-8759916</td>\n",
       "      <td>3</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1:11247744-11257616</td>\n",
       "      <td>4</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1:13741888-13757922</td>\n",
       "      <td>5</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1977</th>\n",
       "      <td>X:101961397-101974103</td>\n",
       "      <td>987</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1978</th>\n",
       "      <td>X:136808741-136818926</td>\n",
       "      <td>988</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1979</th>\n",
       "      <td>X:140517829-140527051</td>\n",
       "      <td>989</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1980</th>\n",
       "      <td>X:167157164-167167452</td>\n",
       "      <td>990</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1981</th>\n",
       "      <td>X:169963295-170005197</td>\n",
       "      <td>991</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1982 rows × 228 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                             name   id  NDB_784  NDB_755  NDB_826  NDB_713  \\\n",
       "Unnamed: 0                                                                   \n",
       "0               1:3742742-3759944    1      0.0      0.0      0.0      0.0   \n",
       "1               1:6245958-6258969    2      0.0      0.0      0.0      0.0   \n",
       "2               1:8740008-8759916    3      0.0      0.0      0.0      0.0   \n",
       "3             1:11247744-11257616    4      0.0      0.0      0.0      0.0   \n",
       "4             1:13741888-13757922    5      0.0      0.0      0.0      0.0   \n",
       "...                           ...  ...      ...      ...      ...      ...   \n",
       "1977        X:101961397-101974103  987      0.0      0.0      0.0      0.0   \n",
       "1978        X:136808741-136818926  988      0.0      0.0      0.0      0.0   \n",
       "1979        X:140517829-140527051  989      0.0      0.0      0.0      0.0   \n",
       "1980        X:167157164-167167452  990      0.0      0.0      0.0      0.0   \n",
       "1981        X:169963295-170005197  991      0.0      0.0      0.0      0.0   \n",
       "\n",
       "            NDB_865  NDB_725  NDB_817  NDB_710  ...  NDB_479  NDB_562  \\\n",
       "Unnamed: 0                                      ...                     \n",
       "0               0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "1               0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "2               0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "3               0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "4               0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "...             ...      ...      ...      ...  ...      ...      ...   \n",
       "1977            0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "1978            0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "1979            0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "1980            0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "1981            0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "\n",
       "            NDB_608  NDB_460  NDB_563  NDB_592  NDB_368  NDB_436  NDB_629  \\\n",
       "Unnamed: 0                                                                  \n",
       "0               0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "1               0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "2               0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "3               0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "4               0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "...             ...      ...      ...      ...      ...      ...      ...   \n",
       "1977            0.0      1.0      0.0      0.0      0.0      0.0      0.0   \n",
       "1978            0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "1979            0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "1980            0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "1981            0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "\n",
       "            NDB_604  \n",
       "Unnamed: 0           \n",
       "0               0.0  \n",
       "1               0.0  \n",
       "2               0.0  \n",
       "3               0.0  \n",
       "4               0.0  \n",
       "...             ...  \n",
       "1977            0.0  \n",
       "1978            0.0  \n",
       "1979            0.0  \n",
       "1980            0.0  \n",
       "1981            0.0  \n",
       "\n",
       "[1982 rows x 228 columns]"
      ]
     },
     "execution_count": 45,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "prev_codebook_filename = os.path.join(\n",
    "    r'\\\\10.245.74.158\\Chromatin_NAS_8\\Exported_data\\0814-Sample_Result_Blair',\n",
    "    'merged_codebook.csv')\n",
    "import pandas as pd\n",
    "prev_codebook = pd.read_csv(prev_codebook_filename).set_index('Unnamed: 0')\n",
    "prev_codebook"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "id": "9a927d1b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>name</th>\n",
       "      <th>id</th>\n",
       "      <th>NDB_784</th>\n",
       "      <th>NDB_755</th>\n",
       "      <th>NDB_826</th>\n",
       "      <th>NDB_713</th>\n",
       "      <th>NDB_865</th>\n",
       "      <th>NDB_725</th>\n",
       "      <th>NDB_817</th>\n",
       "      <th>NDB_710</th>\n",
       "      <th>...</th>\n",
       "      <th>NDB_479</th>\n",
       "      <th>NDB_562</th>\n",
       "      <th>NDB_608</th>\n",
       "      <th>NDB_460</th>\n",
       "      <th>NDB_563</th>\n",
       "      <th>NDB_592</th>\n",
       "      <th>NDB_368</th>\n",
       "      <th>NDB_436</th>\n",
       "      <th>NDB_629</th>\n",
       "      <th>NDB_604</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>383</th>\n",
       "      <td>7:6245207-6259885</td>\n",
       "      <td>403</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>384</th>\n",
       "      <td>7:16248012-16259975</td>\n",
       "      <td>407</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>385</th>\n",
       "      <td>7:18740071-18759137</td>\n",
       "      <td>408</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>386</th>\n",
       "      <td>7:26244483-26259941</td>\n",
       "      <td>411</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>387</th>\n",
       "      <td>7:28750208-28760000</td>\n",
       "      <td>412</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1437</th>\n",
       "      <td>7:132256661-132264746</td>\n",
       "      <td>432</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1438</th>\n",
       "      <td>7:134668240-134677180</td>\n",
       "      <td>433</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1439</th>\n",
       "      <td>7:136070464-136079456</td>\n",
       "      <td>434</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1440</th>\n",
       "      <td>7:138213625-138222514</td>\n",
       "      <td>435</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1441</th>\n",
       "      <td>7:142382518-142390093</td>\n",
       "      <td>436</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>96 rows × 228 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                             name   id  NDB_784  NDB_755  NDB_826  NDB_713  \\\n",
       "Unnamed: 0                                                                   \n",
       "383             7:6245207-6259885  403      0.0      0.0      0.0      0.0   \n",
       "384           7:16248012-16259975  407      0.0      0.0      0.0      0.0   \n",
       "385           7:18740071-18759137  408      0.0      0.0      0.0      0.0   \n",
       "386           7:26244483-26259941  411      0.0      0.0      0.0      0.0   \n",
       "387           7:28750208-28760000  412      0.0      1.0      0.0      0.0   \n",
       "...                           ...  ...      ...      ...      ...      ...   \n",
       "1437        7:132256661-132264746  432      0.0      0.0      0.0      0.0   \n",
       "1438        7:134668240-134677180  433      0.0      0.0      0.0      0.0   \n",
       "1439        7:136070464-136079456  434      0.0      0.0      0.0      0.0   \n",
       "1440        7:138213625-138222514  435      0.0      0.0      0.0      0.0   \n",
       "1441        7:142382518-142390093  436      0.0      0.0      0.0      0.0   \n",
       "\n",
       "            NDB_865  NDB_725  NDB_817  NDB_710  ...  NDB_479  NDB_562  \\\n",
       "Unnamed: 0                                      ...                     \n",
       "383             0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "384             0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "385             0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "386             0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "387             0.0      1.0      0.0      1.0  ...      0.0      0.0   \n",
       "...             ...      ...      ...      ...  ...      ...      ...   \n",
       "1437            0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "1438            0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "1439            0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "1440            0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "1441            0.0      0.0      0.0      0.0  ...      0.0      0.0   \n",
       "\n",
       "            NDB_608  NDB_460  NDB_563  NDB_592  NDB_368  NDB_436  NDB_629  \\\n",
       "Unnamed: 0                                                                  \n",
       "383             0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "384             0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "385             0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "386             0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "387             0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "...             ...      ...      ...      ...      ...      ...      ...   \n",
       "1437            0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "1438            0.0      0.0      0.0      1.0      0.0      0.0      0.0   \n",
       "1439            0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "1440            0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "1441            0.0      0.0      0.0      0.0      0.0      0.0      0.0   \n",
       "\n",
       "            NDB_604  \n",
       "Unnamed: 0           \n",
       "383             0.0  \n",
       "384             0.0  \n",
       "385             0.0  \n",
       "386             0.0  \n",
       "387             0.0  \n",
       "...             ...  \n",
       "1437            0.0  \n",
       "1438            0.0  \n",
       "1439            0.0  \n",
       "1440            0.0  \n",
       "1441            0.0  \n",
       "\n",
       "[96 rows x 228 columns]"
      ]
     },
     "execution_count": 66,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "prev_codebook.loc[prev_codebook['chr'] == sel_chr]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "id": "be888d9d",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "# loop through regions in selected chr\n",
    "length = 20000\n",
    "region_dicts = []\n",
    "for _i, (_reg_name, _rid) in prev_codebook.loc[prev_codebook['chr'] == sel_chr, ['name','id']].iterrows():\n",
    "    _start,_end = _reg_name.split(f\"{sel_chr}:\")[1].split('-')\n",
    "    #print(int(_end)-int(_start))\n",
    "    region_dicts.append(\n",
    "        {\n",
    "            'Chr': sel_chr,\n",
    "            'Start': f\"{int(_start) - length}\",\n",
    "            'End': f\"{_start}\",\n",
    "            'Region':f\"{sel_chr}:{int(_start) - length}-{_start}\",\n",
    "            'Strand':'+',\n",
    "            'Gene': _i,\n",
    "        }\n",
    "    )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "id": "61de86f6",
   "metadata": {},
   "outputs": [],
   "source": [
    "seq_rd.seq_dict = {}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "id": "54b411f9",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n"
     ]
    }
   ],
   "source": [
    "seqs = seq_rd.find_sequence_for_region(region_dicts)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "id": "1ea77da4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "96"
      ]
     },
     "execution_count": 85,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(seq_rd.seq_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "id": "42aacac9",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- saving sequences into folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\383-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\384-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\385-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\386-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\387-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\388-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\389-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\390-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\391-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\392-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\393-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\394-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\395-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\396-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\397-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\398-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\399-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\400-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\401-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\402-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\403-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\404-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\405-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\406-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\407-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\408-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\409-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\410-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\411-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\412-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\413-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\414-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\415-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\416-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\417-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\418-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\419-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\420-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\421-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\422-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\423-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\424-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\425-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\426-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\427-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\428-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\429-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\430-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\431-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\989-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1010-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1397-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1398-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1399-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1400-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1401-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1402-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1403-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1404-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1405-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1406-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1407-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1408-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1409-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1410-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1411-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1412-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1413-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1414-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1415-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1416-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1417-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1418-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1419-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1420-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1421-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1422-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1423-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1424-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1425-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1426-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1427-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1428-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1429-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1430-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1431-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1432-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1433-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1434-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1435-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1436-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1437-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1438-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1439-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1440-seg-0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1441-seg-0.fasta\n"
     ]
    }
   ],
   "source": [
    "seq_rd.save_sequences(save_folder=sequence_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9252f4d3",
   "metadata": {},
   "source": [
    "<a id='2'></a>\n",
    "# 2. Design probe targeting sequences by probe_designer\n",
    "\n",
    "run probe_designer, remeber to clear the memory usage because each table should take ~32GB"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8e85674f",
   "metadata": {},
   "source": [
    "<a id='2.1'></a>\n",
    "## 2.1 Construct count table with all the 17-mers in the genome\n",
    "\n",
    "Only do this if you don't have pre-built 17-mer\n",
    "\n",
    "However you can do almost the same thing for your own library during quality check\n",
    "\n",
    "This library requires mm10 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "id": "74ff1314",
   "metadata": {},
   "outputs": [],
   "source": [
    "overwrite_table = False\n",
    "from ImageAnalysis3 import library_tools"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2e5c8ab3",
   "metadata": {},
   "source": [
    "### construct map for whole genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "80be081a",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\GRCm38_genome_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.design)\n",
    " \n",
    "genome_table_file = os.path.join(reference_folder, 'GRCm38_genome_17w.npy')\n",
    "\n",
    "if not os.path.exists(genome_table_file) or overwrite_table:\n",
    "    # genome\n",
    "    _genome_filenames = [os.path.join(genome_folder, _fl) \n",
    "         for _fl in os.listdir(genome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_genome_filenames))\n",
    "\n",
    "    ct = library_tools.design.countTable(word=17,save_file=genome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_genome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "\n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {genome_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1acc14b4",
   "metadata": {},
   "source": [
    "### construct map for transcriptome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "id": "1e244aad",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\GRCm38_transcriptome_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "from tqdm import tqdm\n",
    "# transcriptome\n",
    "transcriptome_folder = os.path.join(reference_folder, 'Transcriptome')\n",
    "\n",
    "transcriptome_table_file = os.path.join(reference_folder, 'GRCm38_transcriptome_17w.npy')\n",
    "\n",
    "if not os.path.exists(transcriptome_table_file) or overwrite_table:\n",
    "    # transcriptome\n",
    "    _transcriptome_filenames = [os.path.join(transcriptome_folder, _fl) \n",
    "         for _fl in os.listdir(transcriptome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_transcriptome_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=transcriptome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_transcriptome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {transcriptome_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0b477fab",
   "metadata": {},
   "source": [
    "### construct map for repeats from RepBase"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "id": "556095da",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2\n",
      "- Start multi-processing comsume 1810 sequences 24 threads, finish in 6.695s\n",
      "- Total sequences loaded: 2602955\n",
      "Time to compute unique and clip: 0.8485946655273438\n",
      "Time to update matrix: 2.1205813884735107\n",
      "- start saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Repbase_v2603_repeat_17w.npy\n",
      "Wall time: 54.9 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "# repeat\n",
    "repeat_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\Repbase'\n",
    "\n",
    "repeat_table_file = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "\n",
    "if not os.path.exists(repeat_table_file) or overwrite_table:\n",
    "    # repeat\n",
    "    _repeat_filenames = [os.path.join(repeat_folder, _fl) \n",
    "         for _fl in os.listdir(repeat_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_repeat_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=repeat_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_repeat_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {repeat_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ed546b8b",
   "metadata": {},
   "source": [
    "### construct map for CTP11-13"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "1daaff8b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mouse brain library table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\mouse_brain_library_ctp11-12-13_17w.npy\n",
      "3 input files.\n",
      "- Start multi-processing comsume 233855 sequences 44 threads, finish in 70.172s\n",
      "- Total sequences loaded: 29391810\n",
      "Time to compute unique and clip: 7.444552183151245\n",
      "Time to update matrix: 10.16057801246643\n",
      "- start saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\mouse_brain_library_ctp11-12-13_17w.npy\n",
      "Wall time: 2min 16s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "# final ctp11,12,13\n",
    "mouse_brain_library_file = os.path.join(reference_folder, 'mouse_brain_library_ctp11-12-13_17w.npy')\n",
    "print(f\"mouse brain library table: {mouse_brain_library_file}\")\n",
    "if not os.path.exists(mouse_brain_library_file) or overwrite_table:\n",
    "    # directly input library files\n",
    "    _library_filenames = [\n",
    "        r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-11_brain\\mouse_genome_1000\\blast_full_probes.fasta',\n",
    "        r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-12_mop_markers\\marker_gene_TSS\\blast_full_probes.fasta',\n",
    "        r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\blast_full_probes.fasta',\n",
    "    ]\n",
    "    print(f\"{len(_library_filenames)} input files.\")\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=mouse_brain_library_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_library_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=44) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {mouse_brain_library_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d38ed029",
   "metadata": {},
   "source": [
    "<a id='2.2'></a>\n",
    "## 2.2 Design probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "id": "9958b6d9",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\reports\n"
     ]
    }
   ],
   "source": [
    "print(report_folder)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "id": "8e476dee",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "96 regions loaded to design probes.\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\reports\\mouse_brain_chr7_relabel.pbr\n"
     ]
    }
   ],
   "source": [
    "# requires pre_defined genome_folder and library_folder\n",
    "# Indices\n",
    "genome_index = os.path.join(reference_folder, 'GRCm38_genome_17w.npy')\n",
    "#transcriptome_index = os.path.join(reference_folder, 'GRCm38_transcriptome_17w.npy') \n",
    "repeat_index = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "ref_library_index = os.path.join(reference_folder, 'mouse_brain_library_ctp11-12-13_17w.npy') # dna-merfish\n",
    "# get input files \n",
    "input_files = [os.path.join(sequence_folder, _fl) for _fl in os.listdir(sequence_folder) if '.fasta' in _fl]\n",
    "\n",
    "print(f\"{len(input_files)} regions loaded to design probes.\")\n",
    "\n",
    "if not os.path.exists(report_folder):\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "# filename to save probe reports\n",
    "probe_savefile = os.path.join(report_folder, f'mouse_brain_chr7_relabel.pbr')\n",
    "print(probe_savefile)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "722e6b4b",
   "metadata": {},
   "source": [
    "### create pb_designer class"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "id": "02cf472d",
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Probe designer derived from Bogdan Bintu:\n",
      "https://github.com/BogdanBintu/ChromatinImaging/blob/master/LibraryDesign/LibraryDesigner.py\n",
      "by Pu Zheng, 2020.11\n",
      "\n",
      "Major changes:\n",
      "    1. allow design of two strands\n",
      "    2. separate reverse_complement (rev_com) and from two strands (two_stranded) as \n",
      "    two different inputs for map_dic and sequence_dic\n",
      "    3. replace 'local_genome' with 'self_sequences' to be more explicit, and only \n",
      "    exclude the counts for the corresponding self_sequence within each input. \n",
      "\n",
      "Key information:\n",
      "    - number of input_sequence(s): 96\n",
      "    - save_file location: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\reports\\mouse_brain_chr7_relabel.pbr\n",
      "\n",
      "- Fail to load from savefile: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\reports\\mouse_brain_chr7_relabel.pbr, file doesn't exist.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "False"
      ]
     },
     "execution_count": 88,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "pb_designer = library_tools.design.pb_reports_class(\n",
    "    sequence_dic={'file':input_files,\n",
    "                  'rev_com':True, # design two strands\n",
    "                  'two_stranded':True},\n",
    "    map_dic={'genome':{'file':genome_index,'rev_com':False,'two_stranded':True},\n",
    "             #'transcriptome':{'file':transcriptome_index,'rev_com':True,'two_stranded':False},\n",
    "             'rep_genome':{'file':repeat_index,'rev_com':False,'two_stranded':True},\n",
    "             'self_sequences':{'file':input_files,'force_list':True,'rev_com':False,'two_stranded':True},\n",
    "             'ref_library':{'file':ref_library_index,'rev_com':False,'two_stranded':True},\n",
    "             },\n",
    "        save_file=probe_savefile,\n",
    "    params_dic={'word_size':17,'pb_len':42,'buffer_len':0,'max_count':2**16-1,\n",
    "                'check_on_go': False, # whether automatically check probes\n",
    "                'auto': False, # whether automatically convert reference maps\n",
    "               },\n",
    "    check_dic={('genome','self_sequences'): 50, # 25\n",
    "               'rep_genome': 0,\n",
    "               #'transcriptome': 25, # 14\n",
    "               'ref_library': 14,\n",
    "               'gc':[0.25,0.85],'tm': 37+0.62*50+5,\n",
    "               }\n",
    "    )\n",
    "print(pb_designer)\n",
    "pb_designer.load_from_file(load_probes_only=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b84dae63",
   "metadata": {},
   "source": [
    "### calculate probe reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "id": "35dcbca3",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- setting attribute: map_genome\n",
      "--- finish map_genome in 69.628s.\n",
      "-- setting attribute: map_rep_genome\n",
      "--- finish map_rep_genome in 64.836s.\n",
      "-- setting attribute: map_ref_library\n",
      "--- finish map_ref_library in 64.071s.\n",
      "Time(s): 198.53534173965454\n",
      "- Designing targeting sequence for 96 regions\n",
      "-- designing region: 7:6225207-6245207_strand_+_gene_383 -- region: 0, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\383-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.220s.\n",
      "- Designed 39920 candidate probes in 10.702s.\n",
      "-- designing region: 7:16228012-16248012_strand_+_gene_384 -- region: 1, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\384-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.604s.\n",
      "- Designed 39920 candidate probes in 15.012s.\n",
      "-- designing region: 7:18720071-18740071_strand_+_gene_385 -- region: 2, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\385-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.602s.\n",
      "- Designed 39920 candidate probes in 15.120s.\n",
      "-- designing region: 7:26224483-26244483_strand_+_gene_386 -- region: 3, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\386-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.560s.\n",
      "- Designed 39920 candidate probes in 23.626s.\n",
      "-- designing region: 7:28730208-28750208_strand_+_gene_387 -- region: 4, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\387-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.680s.\n",
      "- Designed 39920 candidate probes in 15.275s.\n",
      "-- designing region: 7:31221413-31241413_strand_+_gene_388 -- region: 5, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\388-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.886s.\n",
      "- Designed 39920 candidate probes in 16.508s.\n",
      "-- designing region: 7:33720106-33740106_strand_+_gene_389 -- region: 6, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\389-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.642s.\n",
      "- Designed 39920 candidate probes in 11.153s.\n",
      "-- designing region: 7:36230665-36250665_strand_+_gene_390 -- region: 7, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\390-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.641s.\n",
      "- Designed 39920 candidate probes in 11.547s.\n",
      "-- designing region: 7:43720140-43740140_strand_+_gene_391 -- region: 8, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\391-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.497s.\n",
      "- Designed 39920 candidate probes in 11.045s.\n",
      "-- designing region: 7:46231598-46251598_strand_+_gene_392 -- region: 9, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\392-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.479s.\n",
      "- Designed 39920 candidate probes in 11.005s.\n",
      "-- designing region: 7:48731492-48751492_strand_+_gene_393 -- region: 10, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\393-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.479s.\n",
      "- Designed 39920 candidate probes in 10.979s.\n",
      "-- designing region: 7:51231184-51251184_strand_+_gene_394 -- region: 11, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\394-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.471s.\n",
      "- Designed 39920 candidate probes in 10.980s.\n",
      "-- designing region: 7:53729537-53749537_strand_+_gene_395 -- region: 12, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\395-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.490s.\n",
      "- Designed 39920 candidate probes in 10.985s.\n",
      "-- designing region: 7:56227769-56247769_strand_+_gene_396 -- region: 13, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\396-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.491s.\n",
      "- Designed 39920 candidate probes in 11.543s.\n",
      "-- designing region: 7:58732210-58752210_strand_+_gene_397 -- region: 14, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\397-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.646s.\n",
      "- Designed 39920 candidate probes in 11.166s.\n",
      "-- designing region: 7:61221628-61241628_strand_+_gene_398 -- region: 15, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\398-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.617s.\n",
      "- Designed 39920 candidate probes in 11.117s.\n",
      "-- designing region: 7:63733717-63753717_strand_+_gene_399 -- region: 16, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\399-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.565s.\n",
      "- Designed 39920 candidate probes in 11.071s.\n",
      "-- designing region: 7:66232902-66252902_strand_+_gene_400 -- region: 17, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\400-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.551s.\n",
      "- Designed 39920 candidate probes in 11.124s.\n",
      "-- designing region: 7:68728543-68748543_strand_+_gene_401 -- region: 18, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\401-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.492s.\n",
      "- Designed 39920 candidate probes in 10.998s.\n",
      "-- designing region: 7:71229261-71249261_strand_+_gene_402 -- region: 19, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\402-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.480s.\n",
      "- Designed 39920 candidate probes in 11.654s.\n",
      "-- designing region: 7:73731066-73751066_strand_+_gene_403 -- region: 20, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\403-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.633s.\n",
      "- Designed 39920 candidate probes in 11.127s.\n",
      "-- designing region: 7:76225000-76245000_strand_+_gene_404 -- region: 21, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\404-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.572s.\n",
      "- Designed 39920 candidate probes in 11.065s.\n",
      "-- designing region: 7:78732414-78752414_strand_+_gene_405 -- region: 22, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\405-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.557s.\n",
      "- Designed 39920 candidate probes in 11.067s.\n",
      "-- designing region: 7:81227806-81247806_strand_+_gene_406 -- region: 23, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\406-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.535s.\n",
      "- Designed 39920 candidate probes in 11.033s.\n",
      "-- designing region: 7:83728496-83748496_strand_+_gene_407 -- region: 24, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\407-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.506s.\n",
      "- Designed 39920 candidate probes in 11.028s.\n",
      "-- designing region: 7:86220611-86240611_strand_+_gene_408 -- region: 25, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\408-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.481s.\n",
      "- Designed 39920 candidate probes in 10.969s.\n",
      "-- designing region: 7:88725370-88745370_strand_+_gene_409 -- region: 26, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\409-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.513s.\n",
      "- Designed 39920 candidate probes in 11.028s.\n",
      "-- designing region: 7:91227463-91247463_strand_+_gene_410 -- region: 27, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\410-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.498s.\n",
      "- Designed 39920 candidate probes in 11.859s.\n",
      "-- designing region: 7:93720090-93740090_strand_+_gene_411 -- region: 28, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\411-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.557s.\n",
      "- Designed 39920 candidate probes in 11.040s.\n",
      "-- designing region: 7:96232288-96252288_strand_+_gene_412 -- region: 29, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\412-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.481s.\n",
      "- Designed 39920 candidate probes in 10.986s.\n",
      "-- designing region: 7:98732625-98752625_strand_+_gene_413 -- region: 30, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\413-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.548s.\n",
      "- Designed 39920 candidate probes in 11.089s.\n",
      "-- designing region: 7:101226642-101246642_strand_+_gene_414 -- region: 31, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\414-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.507s.\n",
      "- Designed 39920 candidate probes in 11.003s.\n",
      "-- designing region: 7:103724092-103744092_strand_+_gene_415 -- region: 32, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\415-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.514s.\n",
      "- Designed 39920 candidate probes in 10.993s.\n",
      "-- designing region: 7:106220051-106240051_strand_+_gene_416 -- region: 33, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\416-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.492s.\n",
      "- Designed 39920 candidate probes in 11.002s.\n",
      "-- designing region: 7:108729723-108749723_strand_+_gene_417 -- region: 34, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\417-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.516s.\n",
      "- Designed 39920 candidate probes in 11.051s.\n",
      "-- designing region: 7:111229436-111249436_strand_+_gene_418 -- region: 35, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\418-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.499s.\n",
      "- Designed 39920 candidate probes in 11.120s.\n",
      "-- designing region: 7:113731352-113751352_strand_+_gene_419 -- region: 36, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\419-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.628s.\n",
      "- Designed 39920 candidate probes in 11.155s.\n",
      "-- designing region: 7:116229796-116249796_strand_+_gene_420 -- region: 37, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\420-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.642s.\n",
      "- Designed 39920 candidate probes in 12.241s.\n",
      "-- designing region: 7:118731335-118751335_strand_+_gene_421 -- region: 38, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\421-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.495s.\n",
      "- Designed 39920 candidate probes in 10.984s.\n",
      "-- designing region: 7:121232567-121252567_strand_+_gene_422 -- region: 39, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\422-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.641s.\n",
      "- Designed 39920 candidate probes in 11.149s.\n",
      "-- designing region: 7:123731541-123751541_strand_+_gene_423 -- region: 40, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\423-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.643s.\n",
      "- Designed 39920 candidate probes in 11.097s.\n",
      "-- designing region: 7:126228882-126248882_strand_+_gene_424 -- region: 41, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\424-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.596s.\n",
      "- Designed 39920 candidate probes in 11.083s.\n",
      "-- designing region: 7:128728355-128748355_strand_+_gene_425 -- region: 42, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\425-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.559s.\n",
      "- Designed 39920 candidate probes in 16.292s.\n",
      "-- designing region: 7:131229978-131249978_strand_+_gene_426 -- region: 43, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\426-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 31.206s.\n",
      "- Designed 39920 candidate probes in 44.501s.\n",
      "-- designing region: 7:133728640-133748640_strand_+_gene_427 -- region: 44, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\427-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.642s.\n",
      "- Designed 39920 candidate probes in 16.764s.\n",
      "-- designing region: 7:136229793-136249793_strand_+_gene_428 -- region: 45, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\428-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 1.862s.\n",
      "- Designed 39920 candidate probes in 18.602s.\n",
      "-- designing region: 7:138731656-138751656_strand_+_gene_429 -- region: 46, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\429-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.813s.\n",
      "- Designed 39920 candidate probes in 13.158s.\n",
      "-- designing region: 7:141230556-141250556_strand_+_gene_430 -- region: 47, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\430-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.824s.\n",
      "- Designed 39920 candidate probes in 15.561s.\n",
      "-- designing region: 7:143732889-143752889_strand_+_gene_431 -- region: 48, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\431-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.621s.\n",
      "- Designed 39920 candidate probes in 11.244s.\n",
      "-- designing region: 7:45133949-45153949_strand_+_gene_989 -- region: 49, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\989-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.524s.\n",
      "- Designed 39920 candidate probes in 11.163s.\n",
      "-- designing region: 7:142636816-142656816_strand_+_gene_1010 -- region: 50, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1010-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.643s.\n",
      "- Designed 39920 candidate probes in 12.636s.\n",
      "-- designing region: 7:4474500-4494500_strand_+_gene_1397 -- region: 51, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1397-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.644s.\n",
      "- Designed 39920 candidate probes in 11.249s.\n",
      "-- designing region: 7:25617152-25637152_strand_+_gene_1398 -- region: 52, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1398-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.640s.\n",
      "- Designed 39920 candidate probes in 11.232s.\n",
      "-- designing region: 7:28170401-28190401_strand_+_gene_1399 -- region: 53, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1399-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.642s.\n",
      "- Designed 39920 candidate probes in 11.236s.\n",
      "-- designing region: 7:28560710-28580710_strand_+_gene_1400 -- region: 54, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1400-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.642s.\n",
      "- Designed 39920 candidate probes in 11.245s.\n",
      "-- designing region: 7:30258706-30278706_strand_+_gene_1401 -- region: 55, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1401-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.639s.\n",
      "- Designed 39920 candidate probes in 11.219s.\n",
      "-- designing region: 7:30887838-30907838_strand_+_gene_1402 -- region: 56, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1402-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.641s.\n",
      "- Designed 39920 candidate probes in 11.204s.\n",
      "-- designing region: 7:35710849-35730849_strand_+_gene_1403 -- region: 57, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1403-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.637s.\n",
      "- Designed 39920 candidate probes in 11.212s.\n",
      "-- designing region: 7:36045257-36065257_strand_+_gene_1404 -- region: 58, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1404-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.640s.\n",
      "- Designed 39920 candidate probes in 11.239s.\n",
      "-- designing region: 7:36913930-36933930_strand_+_gene_1405 -- region: 59, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1405-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.640s.\n",
      "- Designed 39920 candidate probes in 11.214s.\n",
      "-- designing region: 7:43538219-43558219_strand_+_gene_1406 -- region: 60, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1406-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.637s.\n",
      "- Designed 39920 candidate probes in 11.212s.\n",
      "-- designing region: 7:47100967-47120967_strand_+_gene_1407 -- region: 61, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1407-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.640s.\n",
      "- Designed 39920 candidate probes in 11.231s.\n",
      "-- designing region: 7:47062742-47082742_strand_+_gene_1408 -- region: 62, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1408-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.641s.\n",
      "- Designed 39920 candidate probes in 11.231s.\n",
      "-- designing region: 7:50530714-50550714_strand_+_gene_1409 -- region: 63, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1409-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.640s.\n",
      "- Designed 39920 candidate probes in 11.230s.\n",
      "-- designing region: 7:64309437-64329437_strand_+_gene_1410 -- region: 64, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1410-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.640s.\n",
      "- Designed 39920 candidate probes in 11.224s.\n",
      "-- designing region: 7:65882429-65902429_strand_+_gene_1411 -- region: 65, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1411-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.638s.\n",
      "- Designed 39920 candidate probes in 11.230s.\n",
      "-- designing region: 7:67833347-67853347_strand_+_gene_1412 -- region: 66, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1412-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.576s.\n",
      "- Designed 39920 candidate probes in 12.882s.\n",
      "-- designing region: 7:70321561-70341561_strand_+_gene_1413 -- region: 67, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1413-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.638s.\n",
      "- Designed 39920 candidate probes in 11.236s.\n",
      "-- designing region: 7:81275338-81295338_strand_+_gene_1414 -- region: 68, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1414-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.641s.\n",
      "- Designed 39920 candidate probes in 11.248s.\n",
      "-- designing region: 7:82809265-82829265_strand_+_gene_1415 -- region: 69, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1415-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.638s.\n",
      "- Designed 39920 candidate probes in 11.238s.\n",
      "-- designing region: 7:83053025-83073025_strand_+_gene_1416 -- region: 70, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1416-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.637s.\n",
      "- Designed 39920 candidate probes in 11.419s.\n",
      "-- designing region: 7:83636608-83656608_strand_+_gene_1417 -- region: 71, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1417-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.639s.\n",
      "- Designed 39920 candidate probes in 11.219s.\n",
      "-- designing region: 7:83798755-83818755_strand_+_gene_1418 -- region: 72, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1418-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.646s.\n",
      "- Designed 39920 candidate probes in 11.234s.\n",
      "-- designing region: 7:84252672-84272672_strand_+_gene_1419 -- region: 73, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1419-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.642s.\n",
      "- Designed 39920 candidate probes in 11.244s.\n",
      "-- designing region: 7:89260544-89280544_strand_+_gene_1420 -- region: 74, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1420-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.644s.\n",
      "- Designed 39920 candidate probes in 11.210s.\n",
      "-- designing region: 7:97751442-97771442_strand_+_gene_1421 -- region: 75, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1421-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.652s.\n",
      "- Designed 39920 candidate probes in 11.253s.\n",
      "-- designing region: 7:98133101-98153101_strand_+_gene_1422 -- region: 76, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1422-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.677s.\n",
      "- Designed 39920 candidate probes in 11.265s.\n",
      "-- designing region: 7:101883725-101903725_strand_+_gene_1423 -- region: 77, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1423-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.679s.\n",
      "- Designed 39920 candidate probes in 11.245s.\n",
      "-- designing region: 7:107664577-107684577_strand_+_gene_1424 -- region: 78, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1424-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.672s.\n",
      "- Designed 39920 candidate probes in 11.271s.\n",
      "-- designing region: 7:109914012-109934012_strand_+_gene_1425 -- region: 79, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1425-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.678s.\n",
      "- Designed 39920 candidate probes in 11.256s.\n",
      "-- designing region: 7:112802310-112822310_strand_+_gene_1426 -- region: 80, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1426-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.676s.\n",
      "- Designed 39920 candidate probes in 11.258s.\n",
      "-- designing region: 7:114289196-114309196_strand_+_gene_1427 -- region: 81, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1427-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.682s.\n",
      "- Designed 39920 candidate probes in 11.258s.\n",
      "-- designing region: 7:115493233-115513233_strand_+_gene_1428 -- region: 82, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1428-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.678s.\n",
      "- Designed 39920 candidate probes in 11.245s.\n",
      "-- designing region: 7:115653962-115673962_strand_+_gene_1429 -- region: 83, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1429-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.672s.\n",
      "- Designed 39920 candidate probes in 11.236s.\n",
      "-- designing region: 7:118121089-118141089_strand_+_gene_1430 -- region: 84, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1430-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.676s.\n",
      "- Designed 39920 candidate probes in 11.252s.\n",
      "-- designing region: 7:119706311-119726311_strand_+_gene_1431 -- region: 85, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1431-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.681s.\n",
      "- Designed 39920 candidate probes in 11.267s.\n",
      "-- designing region: 7:121240027-121260027_strand_+_gene_1432 -- region: 86, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1432-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.679s.\n",
      "- Designed 39920 candidate probes in 13.509s.\n",
      "-- designing region: 7:126247521-126267521_strand_+_gene_1433 -- region: 87, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1433-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.499s.\n",
      "- Designed 39920 candidate probes in 11.091s.\n",
      "-- designing region: 7:127086699-127106699_strand_+_gene_1434 -- region: 88, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1434-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.676s.\n",
      "- Designed 39920 candidate probes in 11.250s.\n",
      "-- designing region: 7:127652637-127672637_strand_+_gene_1435 -- region: 89, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1435-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.676s.\n",
      "- Designed 39920 candidate probes in 11.236s.\n",
      "-- designing region: 7:130218885-130238885_strand_+_gene_1436 -- region: 90, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1436-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.679s.\n",
      "- Designed 39920 candidate probes in 11.251s.\n",
      "-- designing region: 7:132236661-132256661_strand_+_gene_1437 -- region: 91, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1437-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.680s.\n",
      "- Designed 39920 candidate probes in 11.251s.\n",
      "-- designing region: 7:134648240-134668240_strand_+_gene_1438 -- region: 92, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1438-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.682s.\n",
      "- Designed 39920 candidate probes in 11.245s.\n",
      "-- designing region: 7:136050464-136070464_strand_+_gene_1439 -- region: 93, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1439-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.685s.\n",
      "- Designed 39920 candidate probes in 11.281s.\n",
      "-- designing region: 7:138193625-138213625_strand_+_gene_1440 -- region: 94, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1440-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.684s.\n",
      "- Designed 39920 candidate probes in 11.264s.\n",
      "-- designing region: 7:142362518-142382518_strand_+_gene_1441 -- region: 95, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\sequences\\1441-seg-0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.682s.\n",
      "- Designed 39920 candidate probes in 11.267s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\reports\\mouse_brain_chr7_relabel.pbr\n",
      "Wall time: 23min 50s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "pb_designer.computeOTmaps() # load the tables \n",
    "pb_designer.compute_pb_report() # design candidate probes"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e23bacce",
   "metadata": {},
   "source": [
    "### check probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 90,
   "id": "a1b4e1e9",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:0 7:6225207-6245207_strand_+_gene_383, 39192 candidate probes\n",
      "--- 12326 probes passed check_dic selection.\n",
      "finish in 1.549s, 195 probes kept.\n",
      "-- check region:1 7:16228012-16248012_strand_+_gene_384, 39456 candidate probes\n",
      "--- 17790 probes passed check_dic selection.\n",
      "finish in 1.285s, 246 probes kept.\n",
      "-- check region:2 7:18720071-18740071_strand_+_gene_385, 39832 candidate probes\n",
      "--- 14724 probes passed check_dic selection.\n",
      "finish in 1.174s, 247 probes kept.\n",
      "-- check region:3 7:26224483-26244483_strand_+_gene_386, 38860 candidate probes\n",
      "--- 6548 probes passed check_dic selection.\n",
      "finish in 0.849s, 100 probes kept.\n",
      "-- check region:4 7:28730208-28750208_strand_+_gene_387, 39138 candidate probes\n",
      "--- 23402 probes passed check_dic selection.\n",
      "finish in 1.485s, 271 probes kept.\n",
      "-- check region:5 7:31221413-31241413_strand_+_gene_388, 36662 candidate probes\n",
      "--- 2942 probes passed check_dic selection.\n",
      "finish in 0.686s, 70 probes kept.\n",
      "-- check region:6 7:33720106-33740106_strand_+_gene_389, 37966 candidate probes\n",
      "--- 10722 probes passed check_dic selection.\n",
      "finish in 0.992s, 157 probes kept.\n",
      "-- check region:7 7:36230665-36250665_strand_+_gene_390, 39724 candidate probes\n",
      "--- 19804 probes passed check_dic selection.\n",
      "finish in 1.333s, 259 probes kept.\n",
      "-- check region:8 7:43720140-43740140_strand_+_gene_391, 39708 candidate probes\n",
      "--- 17064 probes passed check_dic selection.\n",
      "finish in 1.271s, 254 probes kept.\n",
      "-- check region:9 7:46231598-46251598_strand_+_gene_392, 39784 candidate probes\n",
      "--- 27738 probes passed check_dic selection.\n",
      "finish in 1.650s, 310 probes kept.\n",
      "-- check region:10 7:48731492-48751492_strand_+_gene_393, 36478 candidate probes\n",
      "--- 4464 probes passed check_dic selection.\n",
      "finish in 0.743s, 79 probes kept.\n",
      "-- check region:11 7:51231184-51251184_strand_+_gene_394, 39154 candidate probes\n",
      "--- 10558 probes passed check_dic selection.\n",
      "finish in 0.987s, 172 probes kept.\n",
      "-- check region:12 7:53729537-53749537_strand_+_gene_395, 39464 candidate probes\n",
      "--- 12456 probes passed check_dic selection.\n",
      "finish in 1.054s, 189 probes kept.\n",
      "-- check region:13 7:56227769-56247769_strand_+_gene_396, 39802 candidate probes\n",
      "--- 25712 probes passed check_dic selection.\n",
      "finish in 1.591s, 289 probes kept.\n",
      "-- check region:14 7:58732210-58752210_strand_+_gene_397, 39590 candidate probes\n",
      "--- 23178 probes passed check_dic selection.\n",
      "finish in 1.494s, 283 probes kept.\n",
      "-- check region:15 7:61221628-61241628_strand_+_gene_398, 39052 candidate probes\n",
      "--- 5482 probes passed check_dic selection.\n",
      "finish in 0.807s, 117 probes kept.\n",
      "-- check region:16 7:63733717-63753717_strand_+_gene_399, 39042 candidate probes\n",
      "--- 26164 probes passed check_dic selection.\n",
      "finish in 1.575s, 295 probes kept.\n",
      "-- check region:17 7:66232902-66252902_strand_+_gene_400, 39326 candidate probes\n",
      "--- 17198 probes passed check_dic selection.\n",
      "finish in 1.237s, 215 probes kept.\n",
      "-- check region:18 7:68728543-68748543_strand_+_gene_401, 39644 candidate probes\n",
      "--- 25424 probes passed check_dic selection.\n",
      "finish in 1.575s, 303 probes kept.\n",
      "-- check region:19 7:71229261-71249261_strand_+_gene_402, 39874 candidate probes\n",
      "--- 22058 probes passed check_dic selection.\n",
      "finish in 1.452s, 290 probes kept.\n",
      "-- check region:20 7:73731066-73751066_strand_+_gene_403, 39796 candidate probes\n",
      "--- 23812 probes passed check_dic selection.\n",
      "finish in 1.523s, 290 probes kept.\n",
      "-- check region:21 7:76225000-76245000_strand_+_gene_404, 38272 candidate probes\n",
      "--- 15882 probes passed check_dic selection.\n",
      "finish in 1.202s, 198 probes kept.\n",
      "-- check region:22 7:78732414-78752414_strand_+_gene_405, 39002 candidate probes\n",
      "--- 22082 probes passed check_dic selection.\n",
      "finish in 1.429s, 271 probes kept.\n",
      "-- check region:23 7:81227806-81247806_strand_+_gene_406, 39244 candidate probes\n",
      "--- 20162 probes passed check_dic selection.\n",
      "finish in 1.374s, 268 probes kept.\n",
      "-- check region:24 7:83728496-83748496_strand_+_gene_407, 39850 candidate probes\n",
      "--- 26224 probes passed check_dic selection.\n",
      "finish in 1.617s, 293 probes kept.\n",
      "-- check region:25 7:86220611-86240611_strand_+_gene_408, 39446 candidate probes\n",
      "--- 6314 probes passed check_dic selection.\n",
      "finish in 0.843s, 120 probes kept.\n",
      "-- check region:26 7:88725370-88745370_strand_+_gene_409, 39840 candidate probes\n",
      "--- 10912 probes passed check_dic selection.\n",
      "finish in 1.015s, 188 probes kept.\n",
      "-- check region:27 7:91227463-91247463_strand_+_gene_410, 39714 candidate probes\n",
      "--- 15804 probes passed check_dic selection.\n",
      "finish in 1.187s, 219 probes kept.\n",
      "-- check region:28 7:93720090-93740090_strand_+_gene_411, 39858 candidate probes\n",
      "--- 12090 probes passed check_dic selection.\n",
      "finish in 1.048s, 205 probes kept.\n",
      "-- check region:29 7:96232288-96252288_strand_+_gene_412, 39744 candidate probes\n",
      "--- 23520 probes passed check_dic selection.\n",
      "finish in 1.503s, 286 probes kept.\n",
      "-- check region:30 7:98732625-98752625_strand_+_gene_413, 39722 candidate probes\n",
      "--- 28994 probes passed check_dic selection.\n",
      "finish in 1.727s, 310 probes kept.\n",
      "-- check region:31 7:101226642-101246642_strand_+_gene_414, 39590 candidate probes\n",
      "--- 11518 probes passed check_dic selection.\n",
      "finish in 1.052s, 184 probes kept.\n",
      "-- check region:32 7:103724092-103744092_strand_+_gene_415, 39792 candidate probes\n",
      "--- 10348 probes passed check_dic selection.\n",
      "finish in 0.993s, 178 probes kept.\n",
      "-- check region:33 7:106220051-106240051_strand_+_gene_416, 38444 candidate probes\n",
      "--- 4752 probes passed check_dic selection.\n",
      "finish in 0.776s, 95 probes kept.\n",
      "-- check region:34 7:108729723-108749723_strand_+_gene_417, 39656 candidate probes\n",
      "--- 5530 probes passed check_dic selection.\n",
      "finish in 0.828s, 140 probes kept.\n",
      "-- check region:35 7:111229436-111249436_strand_+_gene_418, 38974 candidate probes\n",
      "--- 17994 probes passed check_dic selection.\n",
      "finish in 1.295s, 252 probes kept.\n",
      "-- check region:36 7:113731352-113751352_strand_+_gene_419, 39870 candidate probes\n",
      "--- 26862 probes passed check_dic selection.\n",
      "finish in 1.605s, 315 probes kept.\n",
      "-- check region:37 7:116229796-116249796_strand_+_gene_420, 39654 candidate probes\n",
      "--- 26818 probes passed check_dic selection.\n",
      "finish in 1.643s, 301 probes kept.\n",
      "-- check region:38 7:118731335-118751335_strand_+_gene_421, 39528 candidate probes\n",
      "--- 23826 probes passed check_dic selection.\n",
      "finish in 1.524s, 300 probes kept.\n",
      "-- check region:39 7:121232567-121252567_strand_+_gene_422, 14700 candidate probes\n",
      "--- 7816 probes passed check_dic selection.\n",
      "finish in 0.517s, 103 probes kept.\n",
      "-- check region:40 7:123731541-123751541_strand_+_gene_423, 38962 candidate probes\n",
      "--- 16728 probes passed check_dic selection.\n",
      "finish in 1.211s, 225 probes kept.\n",
      "-- check region:41 7:126228882-126248882_strand_+_gene_424, 36060 candidate probes\n",
      "--- 18136 probes passed check_dic selection.\n",
      "finish in 1.245s, 239 probes kept.\n",
      "-- check region:42 7:128728355-128748355_strand_+_gene_425, 39496 candidate probes\n",
      "--- 18802 probes passed check_dic selection.\n",
      "finish in 1.314s, 254 probes kept.\n",
      "-- check region:43 7:131229978-131249978_strand_+_gene_426, 38754 candidate probes\n",
      "--- 18238 probes passed check_dic selection.\n",
      "finish in 1.269s, 225 probes kept.\n",
      "-- check region:44 7:133728640-133748640_strand_+_gene_427, 39760 candidate probes\n",
      "--- 22752 probes passed check_dic selection.\n",
      "finish in 1.449s, 271 probes kept.\n",
      "-- check region:45 7:136229793-136249793_strand_+_gene_428, 39278 candidate probes\n",
      "--- 19132 probes passed check_dic selection.\n",
      "finish in 1.320s, 221 probes kept.\n",
      "-- check region:46 7:138731656-138751656_strand_+_gene_429, 39520 candidate probes\n",
      "--- 26776 probes passed check_dic selection.\n",
      "finish in 1.604s, 301 probes kept.\n",
      "-- check region:47 7:141230556-141250556_strand_+_gene_430, 39346 candidate probes\n",
      "--- 19912 probes passed check_dic selection.\n",
      "finish in 1.351s, 257 probes kept.\n",
      "-- check region:48 7:143732889-143752889_strand_+_gene_431, 39830 candidate probes\n",
      "--- 26552 probes passed check_dic selection.\n",
      "finish in 1.615s, 291 probes kept.\n",
      "-- check region:49 7:45133949-45153949_strand_+_gene_989, 39470 candidate probes\n",
      "--- 26186 probes passed check_dic selection.\n",
      "finish in 1.620s, 290 probes kept.\n",
      "-- check region:50 7:142636816-142656816_strand_+_gene_1010, 39834 candidate probes\n",
      "--- 28820 probes passed check_dic selection.\n",
      "finish in 1.724s, 320 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:51 7:4474500-4494500_strand_+_gene_1397, 39882 candidate probes\n",
      "--- 29306 probes passed check_dic selection.\n",
      "finish in 1.746s, 325 probes kept.\n",
      "-- check region:52 7:25617152-25637152_strand_+_gene_1398, 39908 candidate probes\n",
      "--- 27436 probes passed check_dic selection.\n",
      "finish in 1.672s, 310 probes kept.\n",
      "-- check region:53 7:28170401-28190401_strand_+_gene_1399, 39706 candidate probes\n",
      "--- 29420 probes passed check_dic selection.\n",
      "finish in 1.748s, 319 probes kept.\n",
      "-- check region:54 7:28560710-28580710_strand_+_gene_1400, 39918 candidate probes\n",
      "--- 32078 probes passed check_dic selection.\n",
      "finish in 1.795s, 329 probes kept.\n",
      "-- check region:55 7:30258706-30278706_strand_+_gene_1401, 39520 candidate probes\n",
      "--- 23338 probes passed check_dic selection.\n",
      "finish in 1.503s, 280 probes kept.\n",
      "-- check region:56 7:30887838-30907838_strand_+_gene_1402, 38856 candidate probes\n",
      "--- 22048 probes passed check_dic selection.\n",
      "finish in 1.436s, 258 probes kept.\n",
      "-- check region:57 7:35710849-35730849_strand_+_gene_1403, 39528 candidate probes\n",
      "--- 18734 probes passed check_dic selection.\n",
      "finish in 1.316s, 262 probes kept.\n",
      "-- check region:58 7:36045257-36065257_strand_+_gene_1404, 39920 candidate probes\n",
      "--- 26604 probes passed check_dic selection.\n",
      "finish in 1.634s, 309 probes kept.\n",
      "-- check region:59 7:36913930-36933930_strand_+_gene_1405, 39686 candidate probes\n",
      "--- 26478 probes passed check_dic selection.\n",
      "finish in 1.624s, 305 probes kept.\n",
      "-- check region:60 7:43538219-43558219_strand_+_gene_1406, 39408 candidate probes\n",
      "--- 20908 probes passed check_dic selection.\n",
      "finish in 1.399s, 281 probes kept.\n",
      "-- check region:61 7:47100967-47120967_strand_+_gene_1407, 39852 candidate probes\n",
      "--- 28978 probes passed check_dic selection.\n",
      "finish in 1.722s, 321 probes kept.\n",
      "-- check region:62 7:47062742-47082742_strand_+_gene_1408, 39592 candidate probes\n",
      "--- 29422 probes passed check_dic selection.\n",
      "finish in 1.746s, 323 probes kept.\n",
      "-- check region:63 7:50530714-50550714_strand_+_gene_1409, 39246 candidate probes\n",
      "--- 20308 probes passed check_dic selection.\n",
      "finish in 1.361s, 254 probes kept.\n",
      "-- check region:64 7:64309437-64329437_strand_+_gene_1410, 39684 candidate probes\n",
      "--- 20598 probes passed check_dic selection.\n",
      "finish in 1.384s, 264 probes kept.\n",
      "-- check region:65 7:65882429-65902429_strand_+_gene_1411, 39428 candidate probes\n",
      "--- 25666 probes passed check_dic selection.\n",
      "finish in 1.583s, 304 probes kept.\n",
      "-- check region:66 7:67833347-67853347_strand_+_gene_1412, 39920 candidate probes\n",
      "--- 19874 probes passed check_dic selection.\n",
      "finish in 1.404s, 245 probes kept.\n",
      "-- check region:67 7:70321561-70341561_strand_+_gene_1413, 39534 candidate probes\n",
      "--- 27816 probes passed check_dic selection.\n",
      "finish in 1.674s, 312 probes kept.\n",
      "-- check region:68 7:81275338-81295338_strand_+_gene_1414, 39914 candidate probes\n",
      "--- 20972 probes passed check_dic selection.\n",
      "finish in 1.408s, 270 probes kept.\n",
      "-- check region:69 7:82809265-82829265_strand_+_gene_1415, 39648 candidate probes\n",
      "--- 18580 probes passed check_dic selection.\n",
      "finish in 1.325s, 230 probes kept.\n",
      "-- check region:70 7:83053025-83073025_strand_+_gene_1416, 39404 candidate probes\n",
      "--- 17768 probes passed check_dic selection.\n",
      "finish in 1.283s, 247 probes kept.\n",
      "-- check region:71 7:83636608-83656608_strand_+_gene_1417, 39904 candidate probes\n",
      "--- 30834 probes passed check_dic selection.\n",
      "finish in 1.805s, 327 probes kept.\n",
      "-- check region:72 7:83798755-83818755_strand_+_gene_1418, 39632 candidate probes\n",
      "--- 25364 probes passed check_dic selection.\n",
      "finish in 1.581s, 299 probes kept.\n",
      "-- check region:73 7:84252672-84272672_strand_+_gene_1419, 39812 candidate probes\n",
      "--- 28108 probes passed check_dic selection.\n",
      "finish in 1.690s, 314 probes kept.\n",
      "-- check region:74 7:89260544-89280544_strand_+_gene_1420, 38672 candidate probes\n",
      "--- 10166 probes passed check_dic selection.\n",
      "finish in 0.984s, 161 probes kept.\n",
      "-- check region:75 7:97751442-97771442_strand_+_gene_1421, 39556 candidate probes\n",
      "--- 20410 probes passed check_dic selection.\n",
      "finish in 1.368s, 266 probes kept.\n",
      "-- check region:76 7:98133101-98153101_strand_+_gene_1422, 39618 candidate probes\n",
      "--- 28480 probes passed check_dic selection.\n",
      "finish in 1.705s, 305 probes kept.\n",
      "-- check region:77 7:101883725-101903725_strand_+_gene_1423, 39722 candidate probes\n",
      "--- 20682 probes passed check_dic selection.\n",
      "finish in 1.377s, 262 probes kept.\n",
      "-- check region:78 7:107664577-107684577_strand_+_gene_1424, 39744 candidate probes\n",
      "--- 26576 probes passed check_dic selection.\n",
      "finish in 1.630s, 314 probes kept.\n",
      "-- check region:79 7:109914012-109934012_strand_+_gene_1425, 38846 candidate probes\n",
      "--- 20094 probes passed check_dic selection.\n",
      "finish in 1.353s, 251 probes kept.\n",
      "-- check region:80 7:112802310-112822310_strand_+_gene_1426, 39682 candidate probes\n",
      "--- 24122 probes passed check_dic selection.\n",
      "finish in 1.535s, 291 probes kept.\n",
      "-- check region:81 7:114289196-114309196_strand_+_gene_1427, 39802 candidate probes\n",
      "--- 18130 probes passed check_dic selection.\n",
      "finish in 1.304s, 251 probes kept.\n",
      "-- check region:82 7:115493233-115513233_strand_+_gene_1428, 39394 candidate probes\n",
      "--- 21892 probes passed check_dic selection.\n",
      "finish in 1.407s, 266 probes kept.\n",
      "-- check region:83 7:115653962-115673962_strand_+_gene_1429, 39248 candidate probes\n",
      "--- 20258 probes passed check_dic selection.\n",
      "finish in 1.366s, 262 probes kept.\n",
      "-- check region:84 7:118121089-118141089_strand_+_gene_1430, 39598 candidate probes\n",
      "--- 19916 probes passed check_dic selection.\n",
      "finish in 1.369s, 278 probes kept.\n",
      "-- check region:85 7:119706311-119726311_strand_+_gene_1431, 39868 candidate probes\n",
      "--- 22638 probes passed check_dic selection.\n",
      "finish in 1.476s, 286 probes kept.\n",
      "-- check region:86 7:121240027-121260027_strand_+_gene_1432, 39856 candidate probes\n",
      "--- 17330 probes passed check_dic selection.\n",
      "finish in 1.302s, 261 probes kept.\n",
      "-- check region:87 7:126247521-126267521_strand_+_gene_1433, 39700 candidate probes\n",
      "--- 17842 probes passed check_dic selection.\n",
      "finish in 1.299s, 252 probes kept.\n",
      "-- check region:88 7:127086699-127106699_strand_+_gene_1434, 39900 candidate probes\n",
      "--- 21458 probes passed check_dic selection.\n",
      "finish in 1.430s, 261 probes kept.\n",
      "-- check region:89 7:127652637-127672637_strand_+_gene_1435, 39742 candidate probes\n",
      "--- 20382 probes passed check_dic selection.\n",
      "finish in 1.392s, 258 probes kept.\n",
      "-- check region:90 7:130218885-130238885_strand_+_gene_1436, 39786 candidate probes\n",
      "--- 26314 probes passed check_dic selection.\n",
      "finish in 1.610s, 301 probes kept.\n",
      "-- check region:91 7:132236661-132256661_strand_+_gene_1437, 39794 candidate probes\n",
      "--- 29762 probes passed check_dic selection.\n",
      "finish in 1.760s, 330 probes kept.\n",
      "-- check region:92 7:134648240-134668240_strand_+_gene_1438, 39840 candidate probes\n",
      "--- 21380 probes passed check_dic selection.\n",
      "finish in 1.405s, 278 probes kept.\n",
      "-- check region:93 7:136050464-136070464_strand_+_gene_1439, 39918 candidate probes\n",
      "--- 26364 probes passed check_dic selection.\n",
      "finish in 1.619s, 305 probes kept.\n",
      "-- check region:94 7:138193625-138213625_strand_+_gene_1440, 39862 candidate probes\n",
      "--- 29704 probes passed check_dic selection.\n",
      "finish in 1.746s, 325 probes kept.\n",
      "-- check region:95 7:142362518-142382518_strand_+_gene_1441, 39908 candidate probes\n",
      "--- 29130 probes passed check_dic selection.\n",
      "finish in 1.742s, 315 probes kept.\n",
      "Wall time: 3min 38s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "pbs, pb_scores = pb_designer.check_probes()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ae777bea",
   "metadata": {},
   "source": [
    "### save probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "id": "91a45df7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\reports\\mouse_brain_chr7_relabel.pbr\n",
      "-- number of probes kept: 24518\n"
     ]
    }
   ],
   "source": [
    "overwrite_savefile = True \n",
    "if not os.path.exists(pb_designer.save_file) or overwrite_savefile:\n",
    "    #pb_designer.plots()\n",
    "    #pb_designer.save_csv()\n",
    "    pb_designer.save_to_file()\n",
    "print(f\"-- number of probes kept: {len(pb_designer.kept_probes)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e57f03ab",
   "metadata": {},
   "outputs": [],
   "source": [
    "del(pb_designer)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e728e379",
   "metadata": {},
   "source": [
    "## Reload from here:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "id": "9e89bb39",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "pb_designer already exists in RAM, skip.\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "if 'probe_savefile' not in locals():\n",
    "    probe_savefile = os.path.join(report_folder, f'mouse_brain_chr7_relabel.pbr')\n",
    "    print(probe_savefile)\n",
    "\n",
    "# Parse segemented list\n",
    "if 'pb_designer' not in locals():\n",
    "    pb_designer = library_tools.design.pb_reports_class()\n",
    "    pb_designer.load_from_file(probe_savefile)\n",
    "else:\n",
    "    print(\"pb_designer already exists in RAM, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2abb0fb7",
   "metadata": {},
   "source": [
    "## assemble pb_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "id": "5537dab3",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Sort genes\n",
      "- 96 genes\n",
      "-- appending region:383 with 195 probes\n",
      "-- appending region:384 with 246 probes\n",
      "-- appending region:385 with 247 probes\n",
      "-- appending region:386 with 100 probes\n",
      "-- appending region:387 with 271 probes\n",
      "-- appending region:388 with 70 probes\n",
      "-- appending region:389 with 157 probes\n",
      "-- appending region:390 with 259 probes\n",
      "-- appending region:391 with 254 probes\n",
      "-- appending region:392 with 310 probes\n",
      "-- appending region:393 with 79 probes\n",
      "-- appending region:394 with 172 probes\n",
      "-- appending region:395 with 189 probes\n",
      "-- appending region:396 with 289 probes\n",
      "-- appending region:397 with 283 probes\n",
      "-- appending region:398 with 117 probes\n",
      "-- appending region:399 with 295 probes\n",
      "-- appending region:400 with 215 probes\n",
      "-- appending region:401 with 303 probes\n",
      "-- appending region:402 with 290 probes\n",
      "-- appending region:403 with 290 probes\n",
      "-- appending region:404 with 198 probes\n",
      "-- appending region:405 with 271 probes\n",
      "-- appending region:406 with 268 probes\n",
      "-- appending region:407 with 293 probes\n",
      "-- appending region:408 with 120 probes\n",
      "-- appending region:409 with 188 probes\n",
      "-- appending region:410 with 219 probes\n",
      "-- appending region:411 with 205 probes\n",
      "-- appending region:412 with 286 probes\n",
      "-- appending region:413 with 310 probes\n",
      "-- appending region:414 with 184 probes\n",
      "-- appending region:415 with 178 probes\n",
      "-- appending region:416 with 95 probes\n",
      "-- appending region:417 with 140 probes\n",
      "-- appending region:418 with 252 probes\n",
      "-- appending region:419 with 315 probes\n",
      "-- appending region:420 with 301 probes\n",
      "-- appending region:421 with 300 probes\n",
      "-- appending region:422 with 103 probes\n",
      "-- appending region:423 with 225 probes\n",
      "-- appending region:424 with 239 probes\n",
      "-- appending region:425 with 254 probes\n",
      "-- appending region:426 with 225 probes\n",
      "-- appending region:427 with 271 probes\n",
      "-- appending region:428 with 221 probes\n",
      "-- appending region:429 with 301 probes\n",
      "-- appending region:430 with 257 probes\n",
      "-- appending region:431 with 291 probes\n",
      "-- appending region:989 with 290 probes\n",
      "-- appending region:1010 with 320 probes\n",
      "-- appending region:1397 with 325 probes\n",
      "-- appending region:1398 with 310 probes\n",
      "-- appending region:1399 with 319 probes\n",
      "-- appending region:1400 with 329 probes\n",
      "-- appending region:1401 with 280 probes\n",
      "-- appending region:1402 with 258 probes\n",
      "-- appending region:1403 with 262 probes\n",
      "-- appending region:1404 with 309 probes\n",
      "-- appending region:1405 with 305 probes\n",
      "-- appending region:1406 with 281 probes\n",
      "-- appending region:1407 with 321 probes\n",
      "-- appending region:1408 with 323 probes\n",
      "-- appending region:1409 with 254 probes\n",
      "-- appending region:1410 with 264 probes\n",
      "-- appending region:1411 with 304 probes\n",
      "-- appending region:1412 with 245 probes\n",
      "-- appending region:1413 with 312 probes\n",
      "-- appending region:1414 with 270 probes\n",
      "-- appending region:1415 with 230 probes\n",
      "-- appending region:1416 with 247 probes\n",
      "-- appending region:1417 with 327 probes\n",
      "-- appending region:1418 with 299 probes\n",
      "-- appending region:1419 with 314 probes\n",
      "-- appending region:1420 with 161 probes\n",
      "-- appending region:1421 with 266 probes\n",
      "-- appending region:1422 with 305 probes\n",
      "-- appending region:1423 with 262 probes\n",
      "-- appending region:1424 with 314 probes\n",
      "-- appending region:1425 with 251 probes\n",
      "-- appending region:1426 with 291 probes\n",
      "-- appending region:1427 with 251 probes\n",
      "-- appending region:1428 with 266 probes\n",
      "-- appending region:1429 with 262 probes\n",
      "-- appending region:1430 with 278 probes\n",
      "-- appending region:1431 with 286 probes\n",
      "-- appending region:1432 with 261 probes\n",
      "-- appending region:1433 with 252 probes\n",
      "-- appending region:1434 with 261 probes\n",
      "-- appending region:1435 with 258 probes\n",
      "-- appending region:1436 with 301 probes\n",
      "-- appending region:1437 with 330 probes\n",
      "-- appending region:1438 with 278 probes\n",
      "-- appending region:1439 with 305 probes\n",
      "-- appending region:1440 with 325 probes\n",
      "-- appending region:1441 with 315 probes\n",
      "Saving pb_dict into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\reports\\pb_dict.pkl\n"
     ]
    }
   ],
   "source": [
    "overwrite_pb_dict = False\n",
    "verbose = True\n",
    "pb_dict_filename = os.path.join(report_folder, 'pb_dict.pkl')\n",
    "\n",
    "cand_prob_nums = []\n",
    "pb_dict = {}\n",
    "\n",
    "if not os.path.exists(pb_dict_filename) or overwrite_pb_dict:\n",
    "    # get genes\n",
    "    print(\"Sort genes\")\n",
    "    gene_names = np.unique([_v['reg_name'].split('gene_')[1] for _s, _v in pb_designer.kept_probes.items()])\n",
    "    #region_inds = np.unique()\n",
    "    print(f\"- {len(gene_names)} genes\")\n",
    "    for _gene_name in sorted(gene_names, key=lambda v:int(v)):\n",
    "        _sel_pbs = {_s:_v for _s, _v in pb_designer.kept_probes.items() if _v['reg_name'].split('gene_')[1]==_gene_name}\n",
    "\n",
    "        if len(_sel_pbs) > 0:\n",
    "            cand_prob_nums.append(len(_sel_pbs))\n",
    "            if verbose:\n",
    "                print(f\"-- appending region:{_gene_name} with {len(_sel_pbs)} probes\")\n",
    "            _splitted_pbde = library_tools.design.pb_reports_class()\n",
    "            _splitted_pbde.kept_probes = _sel_pbs\n",
    "            pb_dict[_gene_name] = _splitted_pbde\n",
    "    # sazve\n",
    "    print(f\"Saving pb_dict into file: {pb_dict_filename}\")\n",
    "    pickle.dump(pb_dict, open(pb_dict_filename, 'wb'))\n",
    "else:\n",
    "    print(f\"Loading pb_dict from file: {pb_dict_filename}\")\n",
    "    pb_dict = pickle.load(open(pb_dict_filename, 'rb'))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "id": "0e6acea2",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7snQvcz+X9/9/OpKKS5lA5TwlFoySdC9v8LZJZyLRCSnNqKFFqaOmwnJaoKdqKIlu1KE1rSCiHUU5RjBayX7co7v/jfW3fe/fh+73vz+f7OV7f7/N6PHps7vv6XIfn9fp879f3/Xl/rqtUbm5urlAgAAEIQAACEIAABCCQJQRKYYCzZKWZJgQgAAEIQAACEICAIYABRggQgAAEIAABCEAAAllFAAOcVcvNZCEAAQhAAAIQgAAEMMBoAAIQgAAEIAABCEAgqwhggLNquZksBCAAAQhAAAIQgAAGGA1AAAIQgAAEIAABCGQVAQxwVi03k4UABCAAAQhAAAIQwACjAQhAAAIQgAAEIACBrCKAAc6q5WayEIAABCAAAQhAAAIYYDQAAQhAAAIQgAAEIJBVBDDAWbXcTBYCEIAABCAAAQhAAAOMBiAAAQhAAAIQgAAEsooABjirlpvJQgACEIAABCAAAQhggNEABCAAAQhAAAIQgEBWEcAAZ9VyM1kIQAACEIAABCAAAQwwGoAABCAAAQhAAAIQyCoCGOCsWm4mCwEIQAACEIAABCCAAUYDEIAABCAAAQhAAAJZRQADnFXLzWQhAAEIQAACEIAABDDAaAACEIAABCAAAQhAIKsIYICzarmZLAQgAAEIQAACEIAABhgNQAACEIAABCAAAQhkFQEMcFYtN5OFAAQgAAEIQAACEMAAowEIQAACEIAABCAAgawigAHOquVmshCAAAQgAAEIQAACGGA0AAEIQAACEIAABCCQVQQwwFm13EwWAhCAAAQgAAEIQAADjAYgAAEIQAACEIAABLKKAAY4q5abyUIAAhCAAAQgAAEIYIDRAAQgAAEIQAACEIBAVhHAAGfVcjNZCEAAAhCAAAQgAAEMMBqAAAQgAAEIQAACEMgqAhjgrFpuJgsBCEAAAhCAAAQggAFGAxCAAAQgAAEIQAACWUUAA5xVy81kIQABCEAAAhCAAAQwwGgAAhCAAAQgAAEIQCCrCGCAs2q5mSwEIAABCEAAAhCAAAYYDUAAAhCAAAQgAAEIZBUBDHBWLTeThQAEIAABCEAAAhDAAKMBCGQIgZtvvlmeffbZArMpVaqUnHDCCdKwYUPp1auXDBo0SEqXLu3LjHfs2CF169aVu+++W8aPH+9LmzY0cvnll8umTZvkn//8ZyTD1TW98cYb5YUXXvCt/yeffNKs4b/+9S/p3Lmzr22nM8jc3Fy59957Zfr06fJ///d/cscdd8i+ffuMvg8fPiwVK1aUhN4T/9Z+jh8/Ltu3b5f69eun023Sa+rUqSPf+973ZPny5b61SUMQgED0BDDA0a8BI4CALwQShuDRRx+VatWqmTbVSKiBePnll+XNN980RuKJJ57wpT8McGYY4PXr10vTpk3lggsukP79+xvzeOWVV/qikXQbWbRokfz4xz+Wq666Srp37y7NmzeX7777TrZu3So9evQwX+IKG+BDhw7J1Vdfbcbu5xeyV155RSpVqiTXXXddutPhOghAIIYEMMAxXBSGBIF0CCQMgUbANGqVvxw7dkzatGkjq1atks8++0xq1KiRThcFrsEAZ4YBnjNnjvzsZz+TP/zhD9KtWzfPuvCjgYceekhGjRolK1askFatWiVtsrABzlY9+sGbNiCQjQQwwNm46sw5IwkUZ4B1wpMmTZIhQ4bI/Pnz5Sc/+YlnBtlqODItBeKZZ56RPn36yGuvvSbt27f3rAs/GhgzZoyMHTtW/vGPf0jjxo0xwH5ApQ0IQKAAAQwwgoBAhhAoyQA/9thj8stf/jIv0qf13333XfnVr35l8nj1EfOUKVNMNPDgwYMyevRokzqxd+9eqV27tsk71bxMzSnWkjDAaqrLlSsnM2fONOkWl156qXkEff755xcgqwbrwQcflDVr1kiZMmWkbdu2Mm7cOGnRokWxK5AY54svvihDhw41uZiaA/r//t//k9/85jdy6qmnmutTmaZp06aZR/tvv/22qHlNjFvHq4/U9X91vq1btzY5p+XLl5c777xT3nrrLalatarJnX7ggQfMmLUkDLDmow4bNkw2b94s9erVM+klAwYMKDCXzz//XO655x7585//bPrQXGwdy+23355XL2FA582bZ9ZBI/Q///nPZfLkyUm5JHKAf/CDH4iuqebtKkMdY+HUBWWlXN577z3RpwAXXnihWVdNLUjM5Z133inQT+IJgo5D1zsxdp2jGmVd7wSLpUuXyhVXXCGzZs0yqTUbN26Ua665Rl599VXT5uzZs80Y9eeqG+3317/+dbE5uvr04tNPPy0wJk3lKRzxzf9vnaeOI39JzOPIkSNGdxrp3rVrl5xxxhlyww03GC4nnXRSAS3rl0Qd+9/+9jc599xz5YMPPjBrmz8HWNe/bNmyMmLECBOl/vDDD41OfvrTn4pGrlWbifLxxx+bNVXGmp+s+dXXX3+90W5Cjxny8cM0IGAdAQywdUvGgCGQnEBJBviHP/yhMTOffPKJNGjQwBiKP/7xj1K5cmVjgtWg3XLLLVKlShW55JJLjLH7xS9+Ic2aNTOm8/e//735uRpDNYkJI6nX6zVqANVEq+FR07Fy5Uo555xzzGDV5KmpU9PbtWtXY5RnzJghe/bskSVLlpj0jFRFx7lgwQJR46cG4qKLLjLGXc2VGhmdQzoGWE39KaecIrfddpsxRmqmv//978vXX39tzNTFF18szz//vPz1r3+V3/3ud4ZFwjT+/e9/N+Pp3bu3yZ/VMSxbtsyYUDW8Wnbv3i1qUr/99ltjetV4vfHGG7Jw4UJjsB9//PE8Nmos1YzpWGrVqmXG0aFDh5QGWJmrCdOXGvU6NctqWP/yl7/kGUHN+f7Rj35kIqhq4hOGdN26deZLkK6D1nnppZfM/NTM61rr04Evv/zSfCHQvFo19WpKta6uQ37mCQOs49FcXTXiJ598stx0000mgqsmU3N5NbKsL7HpFyw14praoBpMVjTnVrWmX74mTpxo0nW0veIM8FdffSVz5841X/B0zvplTeehZlT7VgOq2tb56fx1vprzrGtWoUKFAlpWA6/X6LoNHDjQzL2wAVZj++9//9to+rzzzhMds95bylDHrGXnzp3SsmVLc08k1umpp56SL774Qvbv348B5oMcAhETwABHvAB0DwG/CCQMwurVq+XMM880zarZUHOnUc6pU6dKly5djOHRkqivv1MDligjR440UbrnnnvORIMT5ZFHHjERWDW4+gc9vwHWR9WJPtVgaPRXDYj2pSZKf6eRMzVQiaKmRc3j6aefbiJtqUpinBpV1mhaoqhJVSOs7Wh00W0EuHr16rJly5a8KKAaOx1v/hcFtW2tp6ZKI7RadB5qqJTH4MGDzc/U5LRr106UvUZ9TzvtNMNXTZxGCPPnZCs7jZbqz9WQJSLAalIL7+KRjIkab/1Pv2BoRFeLRuk1utyoUSOT563RRv23Gnw16xqh16JfTDRCr5rQKKt+kUmWAqEvmqmhVIOoX1oSRdnojhFq+DSKmTDAOvf8keRt27aZ/tVAJoy+tqGmUA2jmswEz2RzTLaWxRlgNbrJUnIScyuc9qPropFYXQedU+Ja3dVENZF/p5RkBljnqrtwqNHWkuD9zTffmPXXouZY1/P999/Pe8qhetL565cVIsB+ffLRDgTSI4ABTo8bV0EgdgSSbYOWGKRGC9XUqHlJPPZN1Fezon/4E0Uf/eofck0PUKOVKEePHjVmVSNnanwSpuHWW281qQP5y7XXXmseu+sffDVLGm1UM9CxY8cC9fRxvBpzNQQa+UxWUkW21XzqjhdqqtRguzXAGlXUKHKiaORWH5UXNibatv6n89GiBliNrqYeqIFMlMTLZGoc9WUyTc3QKHL+PrSufllQw6h96ZeNhEkr/EUklcB0TZSvRpPzF10HjTBqVF1NmJpjjUj369evQD2NvOvje33Mr5H3wgZYvzTpI329XlnkL2q0NRqqkW+9LmGAdR014psoiS9L+rRAv+TkL/plS6P++sVIdZms+GWANfqsZlU1XrjolwWdo0bNE1pO9iUkmQFWLeiTgsQXC21bDbVG9/XLkKZs6E4sGhHXyHn+MmHCBPPEBQOcSuH8HALhEMAAh8OZXiAQOIGEUdTIrT5u16KRLE1P0Mfg+pg6f0nU13SE/L/TLZ/U5GnObuGi5vfAgQPGMCRMg+ZN6qPn/EUfm6uxVTOmxlf/4BdX1FCoWUxWEuNUw5HIP9Z62qaaiUSup1sDrLmsmvaQKInrNV81kbqhv0tmgJSBRnDzF43IatqARs/V5KlRLK4kvjgkDKhu/aVpKiUVNcD5UygS9fXRu0bINeKr0V1NSSiuqGHXvNXCBjhhctU46xoWLhpV1m3J1PwmDLB+scqf16wpH5p7XVzRFJFUu5H4ZYB1HXXP5lRFTbCm+iS0rE84Hn744QLVk62/akRTOvIX5a2pJWp+NYVEDbDeB4VzufUpiKbyYIBLUjq/h0CwBDDAwfKldQiERqCkHODCA0l2kIDW0cfJml6QzACr8dGDBzQHMmEaEikR+dtX86RRYc131KikRjrVCKjhSFY0V1KNVbKSapxODbCaM33MXfgluMIHeKQy0MkMkEa29WW+/EXzpNXEq4HS1JGaNWtKp06dTN/Jiv5eo+1ud2FQA3zXXXeZ6Hf+oikiGtnVR+6a560Rf32JTaPNyUqTJk2MAS3cvx7woT9PZYD1C5UafY2cJgywGuX8kWbNZdY8W00p0frJiqZW5H9hLH8dvwyw5lJr2odGvZMV/bKnee3F7WiSbP2THYSS3wAnGOoXQ/2CmL9o5F5zjDHAoX000hEEkhLAACMMCGQIAb8MsEbNNAVCHxsXToHQqJa+2KWPsBOmQXNa1QTnL2qg165da1720VzJRE5p4aikmkZ9FK4mLZUZcmqA9XG/PorXftWoJ4qaQN1twk8DrHnOGgFM7IagfWmkW8eq+ab66F2Nn+bbvv766wXY6JcCfbFOH7+fffbZaRlgzb/V1JL8RaPOOgZNzdCopqY3aDS6cPRdx65pI2pAdYypUiD0S4ka3PxFI/pq3DW/9emnn05pgLVf/dKj0Wh9aTF/Ue2orlQj+fUVhAHWwyv06YJG7AunW+iLi5qnrE81/DbAmhOsaSSqa43s5y+Fv5BlyMcP04CAdQQwwNYtGQOGQHICfhngRGS18Etwmi6gb7lr5FEjkAnToI/6Na9V/+Br0QikRgg1n1LNlT4O1hxajXZq3qm+da9FzbG+BKaPjPWRfap8UKcGWCPNmlagW3LpNVo0b1nzMDds2OCrAda8Us31TRh67UeNnr5ApfnMuhOC5gHri15q8vVLQ6Ik0kP+9Kc/mZzodCLA+mVBH8MncrfV0Gqaix4aoaZVdzDQyKWmwOhJb4korEZDdZwawdRx6st6yfpPRDMLvwSnL7VpJF9fItPH+KkiwMpbX/bSCLjWTbxUpukq+nONPmvKSKqSTgRYX+w766yzCuzEoLtOaGqGaldTXhJFza++wJbYtcFvA6z9aJ60pkRoqoxGorXoF0u9Nz766CMiwHyQQyBiAhjgiBeA7iHgFwG/DLBuh6ZGSiPAaijzb4OmUcvCW0epidKcY8371OijRoPV5OrODrrVmBZ9214jxdpWz549jdnVHFFNpVAjmXibPhkLpwY4YbS1bzU2avrUDGuOs0ZE/YwAq6nRl53UEOrLe/plQbf20rQPZaZF+anZ0ZQRNb267Ze+FKamSHN99YUpNYbpGGBlro/v9YuIjkP5alqG7oqR2H9ZjafubKFfPnQLN+Wh24up8dTc6eHDh5txJutfDaFqQPOudexqtDXlQfNXNfqsbWv0NpUB1nbVcOrjf42C60uQOTk55iVM1Yi+GKY/99MAK2d9wVPNpqYe6I4nmtt+2WWXmS8h+tKjRr01PURNvH5x0zXTXT6CMMD6pUS/fOkXPM3ZTkTbVfM6VmWnY6NAAALREMAAR8OdXiHgOwG/DLAOTE3KfffdZx6z6//XyJq+MKU5pmq8tCRMg0bXNIKrJlAjj7rFlf5MDxDIX9Q06c81RUF3T9DdAbS9VPvdJq51aoC1vqYWaARbd2nQnGKNQuuOCVdffbWvBlhNtR6CoOPXqKaaLt1FovBRwvo7TctQ86gGVVMeND9YDXqCYzoGWHfAUEOlEU5NIdF0B8091sf5+YtGqjX9Q42eGjGNEusXETWDiZKqf11TnZOmcOiet2rgNfVBry98EEbhHOBE2xqV19/pE4ITTzzRpH2orgqnRRS+GdKJAGsbykBzodVg6r68+jKnmnhloFFfjXrrlzXVqPaR2LovCAOs49Eovb5Yp18alZl+EVCt6JePZOkhvn8o0CAEIJCSAAYYcUAAAhCAAAR8JqA54rptYOE858Q2aLrNYOEviT4PgeYgAIFiCGCAkQcEIAABCEDAZwIafda8ZE15SETMdY9l3SlEI866U0T+Azd87p7mIACBEghggJEIBCAAAQhAwGcCerBJ3759zW4XmvqgO0PotnCaluL00BOfh0RzEIBAPgIYYOQAAQhAAAIQCICAvvCou6borhtqgHV7Ps3/1t0xKBCAQLQEMt4A6+MnfRNX3z5ObI2kL+HoixyrVq0y2wDpG7r6okKi6AeVHuupm6fr/pH65rC+bKIvgVAgAAEIQAACEIAABOwmkNEGWN9I17ej1egm9gbVrZL0TWjdw1LfpNbtcXSbHzW4upG8FjW/+m+9Rrc40jd2dUsj3dtS316nQAACEIAABCAAAQjYSyCjDbCeRKQGV/f/TBhgPaFI94DUlxASG+9rPT29SR9T6Yb2etrVxIkT84721H1R9fQjzdsqfJKVvUvPyCEAAQhAAAIQgEB2EshYA6z7gf7kJz8xe47qHqYJA6wnL+mJVXPmzMlb8cWLF5t9Iffu3WuMsW5erxvnN2rUKK+OpkHoJv4aGaZAAAIQgAAEIAABCNhLICMNsEZs9WUDjfb26NHD7MOYMMBqYnXjfd2LMVF0s3I9mlOPcNXTe/QEIT21KLFRvdbTDe71Z4XPdc+/9Lohvf6XKHrspW6Do6copTrm1V7pMHIIQAACEIAABOJMQE+K/OKLL8zBQ3qEOuV/BDLSAOuJVXrq0QsvvGBmmt8A64tsaorvv//+PAqa31u/fn1zWo9GgPWoVt2vMf8ejXqi1O7du0WjxalK4vQiBAYBCEAAAhCAAATiQkCPQP/BD34Ql+HEYhwZZ4Bnz55tjiddt26dOQq1sAHWb0GaBpEsAqzHp6oZTpxbXzgCfOTIEVmwYEHKhSscAdbor76Ep8KrUaNGLBacQUAAAhCAAAQgkB0E9uzZI61atTJHttepUyc7Ju1wlhlngHXT8XfffVcqVKiQh0DPgtd/a5T37LPPNlufqVFOlGQ5wFu2bDH1E0VzgDWtQl+gc1r03Hk9a16NcO3atZ1eRj0IQAACEIAABCDgmQA+JDXCjDPAn3/+uRw+fLjAjBs2bGgivjfeeKPMnTtXpk2bJnoOe+J4So0Yv/zyy2YXCI3yVq9eXSZNmmRO8dGS7i4QCM/zvUsDEIAABCAAAQikSQAfkkUGONlU8+cA79u3z+wDrCfx6P6+mp7Qv39/Y4p79+5tLh81apRMnz7dbHumjwz05B7NDda0Cjf7ACO8NO9YLoMABCAAAQhAwDMBfAgGOG8XCEWhuz3o6W9r1qwxublDhgyRgQMH5lHSF+B0b2DdOUKjye3atTOpD27zZxCe53uXBiAAAQhAAAIQSJMAPiTLDXCauvF8GcLzjJAGIAABCEAAAhBIkwA+BAOcpnS8XYbwvPHjaghAAAIQgAAE0ieAD8EAp68eD1ciPA/wuBQCEIAABCAAAU8E8CEYYE8CSvdihJcuOa6DAAQgAAEIQMArAXwIBtirhtK6HuGlhY2LIAABCEAAAhDwgQA+BAPsg4zcN4Hw3DPjCghAAAIQgAAE/CGAD8EA+6Mkl60gPJfAqA4BCEAAAhCAgG8E8CEYYN/E5KYhhOeGFnUhAAEIQAACEPCTAD4EA+ynnhy3hfAco6IiBCAAAQhAAAI+E8CHYIB9lpSz5hCeM07UggAEIAABCEDAfwL4EAyw/6py0CLCcwCJKhCAAAQgAAEIBEIAH4IBDkRYJTWK8EoixO8hAAEIQAACEAiKAD4EAxyUtoptF+FFgp1OIQABCEAAAhAQEXwIBjiSGwHhRYKdTiEAAQhAAAIQwAAXq4FSubm5uagkGAIY4GC40ioEIAABCEAAAiUTwIcQAS5ZJQHUQHgBQKVJCEAAAhCAAAQcEcCHYIAdCcXvSgjPb6K0BwEIQAACEICAUwL4EAywU634Wg/h+YqTxiAAAQhAAAIQcEEAH4IBdiEX/6oiPP9Y0hIEIAABCEAAAu4I4EMwwO4U41NthOcTSJqBAAQgAAEIQMA1AXwIBti1aPy4AOH5QZE2IAABCEAAAhBIhwA+BAOcjm48X4PwPCOkAQhAAAIQgAAE0iSAD8EApykdb5chPG/8uBoCEIAABCAAgfQJ4EMwwOmrx8OVCM8DPC6FAAQgAAEIQMATAXwIBtiTgNK9GOGlS47rIAABCEAAAhDwSgAfggH2qqG0rkd4aWHjIghAAAIQgAAEfCCAD8EA+yAj900gPPfMuAICEIAABCAAAX8I4EMwwP4oyWUrCM8lMKpDAAIQgAAEIOAbAXwIBtg3MblpCOG5oUVdCEAAAhCAAAT8JIAPwQD7qSfHbSE8x6ioCAEIQAACEICAzwTwIRhgnyXlrDmE54wTtSAAAQhAAAIQ8J8APgQD7L+qHLSI8BxAogoEIAABCEAAAoEQwIdggAMRVkmNIrySCPF7CEAAAhCAAASCIoAPwQAHpa1i20V4kWCnUwhAAAIQgAAERAQfggGO5EZAeJFgp1MIQAACEIAABDDAxWqgVG5ubi4qCYYABjgYrrQKAQhAAAIQgEDJBPAhRIBLVkkANRBeAFBpEgIQgAAEIAABRwTwIRhgR0LxuxLC85so7UEAAhCAAAQg4JQAPgQD7FQrvtZDeL7ipDEIQAACEIAABFwQwIdggF3Ixb+qCM8/lrQEAQhAAAIQgIA7AviQLDPA+/btkyFDhsjrr78uhw8flssuu0wefvhhOffccw2JPn36yDPPPFOASq1atcx2IVqOHz8uY8eOlRkzZsiBAwekbdu2MmXKFGnQoIEr5SE8V7ioDAEIQAACEICAjwTwIVlmgC+66CIpXbq0PPHEE1K5cmW599575b333pMtW7bICSecIC1btpT27dvLHXfckUemTJkycvrpp5t/q/lVwztr1ixRYzx8+HDZtm2bbNiwQcqXL+9YmgjPMSoqQgACEIAABCDgMwF8SBYZ4C+//FIGDhwo99xzjzRp0sTM/MMPP5Tzzz9fVq5cKS1atJATTzxR5s6dK507dy5C5ujRo1KtWjWZOHGi9OvXz/z+4MGDUrNmTZk5c6Z0797dsTwRnmNUVIQABCAAAQhAwGcC+JAsMsCFp7p3714ZOnSoLF26VDZt2iS7du2Sc845RzZv3iyNGjUqQkZNcuvWrYv8XtMgmjVrZiLDTgvCc0qKehCAAAQgAAEI+E0AH5KlBvjWW2+Vp556SipUqCALFy6Ua6+9Vl588UW58cYbTXT3tddeM6kSHTt2lHHjxkmVKlVk/vz50qVLF8nJyZFKlSrlkevWrZv52aJFi1LSPHTokOh/ibJnzx5p1aqVMd21a9f2W9e0BwEIQAACEIAABFISwABnqQHeuHGjeQlu6tSpMmfOHHn33XdlwYIF8uCDD8pDDz1kjO8nn3xiIsRnnnmmvPXWW6Zez5495dixY8YcJ0qvXr1k9+7dsnjx4pQ0x4wZY/KHCxcMMJ9OEIAABCAAAQiETQADnKUGODFt3dWhadOmJrXh6aeflq+++kqqVq2aR2X58uVy8cUXi/6viqVr165JI8BHjhwxBjpVIQIc9q1NfxCAAAQgAAEIpCKAAc4iA6xboC1ZskQ0ZUF3dkiUG264waQnvPHGG0VofP311+bFuJdeeslEgtUo644R9evXz6urOcDNmzeXyZMnO77TEJ5jVFSEAAQgAAEIQMBnAviQLDLAa9eulQsuuMCY4CuvvNLM/Ntvv5XGjRtLp06dRF+K050i8hthfUHuiiuuEE2ZqFevnlSvXl0mTZokffv2NdezC4TPdyTNQQACEIAABCAQOAEMcBYZ4NzcXOnQoYPs2LHDvAB3yimnmJxfNbxqjj/66CNjhB944AGzpdnHH38st99+u0mBeP755w2pUaNGyfTp0822Z3Xq1JFhw4aZ9tatW8c+wIHfrnQAAQhAAAIQgIAfBDDAWWSAdaqa4ztixAh55ZVXTPT20ksvNRHdxL7A8+bNM6ZYt0XTXOAePXqYXSAqVqxoSOkLcCNHjjQHYehLdO3atTOpD2qG3RSE54YWdSEAAQhAAAIQ8JMAPiTLDLCf4vHSFsLzQo9rIQABCEAAAhDwQgAfggH2op+0r0V4aaPjQghAAAIQgAAEPBLAh2CAPUoovcsRXnrcuAoCEIAABCAAAe8E8CEYYO8qSqMFhJcGNC6BAAQgAAEIQMAXAvgQDLAvQnLbCMJzS4z6EIAABCAAAQj4RQAfggH2S0uu2kF4rnBRGQIQgAAEIAABHwngQzDAPsrJeVMIzzkrakIAAhCAAAQg4C8BfAgG2F9FOWwN4TkERTUIQAACEIAABHwngA/BAPsuKicNIjwnlKgDAQhAAAIQgEAQBPAhGOAgdFVimwivRERUgAAEIAABCEAgIAL4EAxwQNIqvlmEFwl2OoUABCAAAQhAQETwIRjgSG4EhBcJdjqFAAQgAAEIQAADXKwGSuXm5uaikmAIYICD4UqrEIAABCAAgbAIHP3uuMxb/Zns3J8jZ596gnRpWVvKlSkdVvee+sGHEAH2JKB0L0ZCifEmAAAgAElEQVR46ZLjOghAIC4EbP7jHxeGjMNeAqr/Xk+vkOXb9+dN4qJ6p8nsvq2sMMH4EAxwJHcfwosEO51CAAI+EbD9j79PGGgmiwnMXblTRsxfV4TA+OubSvdWZ8WeDD4EAxyJSBFeJNjpFAIQ8ImA7X/8fcIQaDNE2APF67nxCa9vkqlLtxZpp//l9eXu9o09tx90A/gQDHDQGkvaPsKLBDudQgACPhGw/Y+/TxgCa4YIe2BofWvY9i+B+BAMsG83g5uGEJ4bWtSFAASCIOAlwmj7H/8gePrZJnz9pBlMW358SfFyD3qdFT4EA+xVQ2ldj/DSwsZFEICATwS8/vH2er1P08jYZoiw27G03x47LvM++Ew+TWMXiKjvIXwIBjiSuwzhRYKdTiEAgf8S8CPC6OWPf5gLEWWULd15+rE+6fbNdeEQiHqN8SEY4HCUXqgXhBcJdjqFAAT+SyBbIoxRR9nSFZyt4053vtl4XdT3ID4EAxzJfYfwIsFOpxCAgI8R4LjBTBbpfemDz6zdqsqWCHvcdGDLeIgAx3elOAkuwLXBAAcIl6YhAIEiBAqbwx83ryl9n3nf2k38C08wVcS0ee0qMv2v24rwsGWrKqScuQSijvLjQ4gAR3J3IbxIsNMpBLKSQKo/tDNvvlAWrt3t6AWeuOfRpoqmXX9BLZm/5vMi627LYQVZKdgsmnSUUX58CAY4klsN4UWCnU4hkJUEvD5qjTpS5WTRUuVT3tqunny062DGRLqdsKAOBJwQwIdggJ3oxPc6CM93pDQIAQikIOD1ZRuvBjqMhSlujF1a1k57q6owxk4fEIiCAD4EAxyF7gThRYKdTiGQlQS8GlivBjoM6DZEqcPgQB8QcEoAH4IBdqoVX+shPF9x0liWE4h7fmrUy+PVHHo10GHNP8p8yrDmSD+ZTyCszzN8CAY4krsJ4UWCnU4zkIBXc5eBSJJOyYs5hHH8VRKWaQqrn/gTD2aEYd5r+BAMcDAqLqFVhBcJdjrNQAK2RCdtR+/FQNs+97iPPyzTFFY/cecd5PjC/DzDh2CAg9RyyrYRXiTY6TQDCdiQn5qB2JlSEgJRRUfDMk1h9ZPN4grz8wwfggGO5F5DeJFgp9MMJMAf5QxcVAunFGV0NCzTFFY/Fi6/b0MO8/MMH4IB9k24bhpCeKlpRRVFcbN+1C1IIMo1i9J4ZKIOolxLm3mGaVwKcwqr77D6sVkHXsce5ucZPgQD7FWvaV2P8JJjC/PmT2vhuKgIgTisGfmp/ggzDmtpqwGPMjoa1rqF1Y8/ara3lbA+z/AhGOBI7hKElxw7EYZI5OipU9bME75YXRz1WtpssKJmF5ZpCqufWN0YGToYfAgGOBJpI7zk2KOMokQihAzolDXLgEX87xSiXsuoTaSXlbTZvHuZN9faSwAfggGORL0IjwhwJMILoFObTUsAOKxuMuq1jNqAe108oqNeCXJ9mATwIRjgMPWW1xfCS46dKEokcvTUKWvmCV+sLo56LVMZ8HGdm0iZ0qVl5/4cOfvUE6RLy9pSrkzpWLGzfTC25l7bwj2OfPEhGOBI7h+Elxo7UZRIJOmpU9bME75YXRzlWiYz4K3qnmr4rNy+P4/TRfVOk9l9W4ViguNoXPwWTNRffPyeT9zaiytffEiWGeB9+/bJkCFD5PXXX5fDhw/LZZddJg8//LCce+65hsTatWtl0KBBsmrVKjnttNPkzjvvlKFDh+ZROn78uIwdO1ZmzJghBw4ckLZt28qUKVOkQYMGru45hOcKlzWVs+GPpTWLwUCtJFDYgH93PFfueWV9kbmMv76pdG91VqBzjKtx8XvSUae++D2fuLUXV774kCwzwBdddJGULl1annjiCalcubLce++98t5778mWLVuMIW7cuLF07txZBg8eLMuXL5cBAwYYg9unTx9DSs2v/nvWrFlSq1YtGT58uGzbtk02bNgg5cuXd3zfITzHqKypmC1/LK1ZEAaaEQSizAuOq3Hxe2GjZOz3XOLYXlz54kOyyAB/+eWXMnDgQLnnnnukSZMmZuYffvihnH/++bJy5UpZvHixTJ48WXbs2CFly5Y1vx85cqTMnz9fNm3aJEePHpVq1arJxIkTpV+/fub3Bw8elJo1a8rMmTOle/fuju89hOcYlTUVs+WPpS0LYnM03uax+62PKO+ruBqXTGLs91yCbC/d+zJKDRfHAx+SRQa48FT37t1r0huWLl1qDO4NN9wgVatWlTlz5uRVVVN8zTXXiNZVY9y6dWvZvHmzNGrUKK+OpkE0a9bMRIadFoTnlJQ99bLlj6UNK5IqGv907wtl4Ye7Y/0yFU8SCiosSh5xNS5+34NRMvZ7LkG154WRl2uDmo+2iw/JUgN86623ylNPPSUVKlSQhQsXyrXXXmtMbIcOHWTChAl5VDZu3Giixe+//77s3LlTunTpIjk5OVKpUqW8Ot26dTM/W7RokWOtIjzHqKypmC1/LG1YkFRrUbdaZdn+r6/zphDmy1ROuaGjoqSiejEvrsYl3UhkcRqMirHT+yLqel7vyzjyxYdkqQFWY6s5v1OnTjUR33fffVfUyPbo0UPuv//+PCqa31u/fn1ZtmyZiQD37NlTjh07ZvKIE6VXr16ye/duk0KRqhw6dEj0v0TZs2ePtGrVSnbt2iW1a9eO+t6mfx8IxPWPpQ9Ts66JVNH4ZBMJ42UqNwB5kuCGVvB142Zc+JwJfs2T9ZCJ9yUGOEsNcGLauqtD06ZNTWqDRnk7duyYNAK8evVq87Jb165dk0aAjxw5IgsWLEhJc8yYMeYFusIFAxzNh1lQvcbtj2VQ84x7u6miNcnG3f/y+nJ3+8axmZLXSFNsJsJAAiGAPgLBWmKjmcgdA5xFBli3QFuyZImJ9JYpUyZv5pr7q9FZ/ZlufTZ79uy83yXLAdYdIzQqnCiaA9y8eXPzAl2qQgS4xM8XKkDANwLJomSF0x8SncUtAkyEzzcZZGRDmRiJtGGhMvG+xABnkQHWPX4vuOACY4KvvPJKM/Nvv/3WbH3WqVMnOeOMM2TatGmydevWPIM8YsQIefnll81LchrlrV69ukyaNEn69u1rrmcXCBs+uryNMYh8O28j4monBApH43/cvKb0feZ9WR7RgQpOxpyow5MEN7Syq24mRiJtWcFMuy8xwFlkgHNzc81LbprLqy/AnXLKKfLggw/KG2+8YQ7AqFixYp4Z1v19dWu0/v37G1Pcu3dvQ2rUqFEyffp0s+1ZnTp1ZNiwYaa9devWsQ+wLZ9iLsaZid/6XUw/46pm2h+wjFsgJlQiAT6TSkREBYcEMMBZZIB1ql999ZVoVPeVV14x0dtLL73URHQT+wJrHrCe/rZmzRqpUaOGOTVO9w5OFH0BTvcG1oMw9CW6du3amdQHNcNuCsJzQyu6ujZFW4hUR6cTeoZAmAT4Ihcm7cztCx+SZQY4LlJGeHFZieLHYUu+HVEhO/TEKCEAAQjEhQA+BAMciRYRXiTYXXdqSwTYlnG6XgAuKJYAUX8EElcCaDOuK/O/ceFDMMCRqBThRYLddad+RFaT/SHIzRWZt/oz304ksyVS7XoBuCAlgeK06be+WAYIuCHgx+emm/6omx4BfAgGOD3leLwK4XkE6PJyL9EIL/l2yf4QtKp7qhn9Sh93IyAC7FIQGVA91ZqP69xEFn24x4rdLjJgGZhCEgJ8HtkhC3wIBjgSpSK88LBHGY1wcyCDl/1oo5xjeCtJT/kJpIr6X9bodHnn4y+KwPKiL8hDwA0Bnki5oRVdXXwIBjgS9SG88LBHGY1wcySv1xPJvESqw1sNevKLQCpdX97odFmaxAB71Zdf46adzCcQ5Wdu5tP1b4b4EAywf2py0RLCcwHLY9UooxFhRYA9InJ1uZd0ElcdUblYAqmi/j9qVkPueWU9EWD0ExmBsJ9I8ZmU3lLjQzDA6SnH41UIzyNAF5dHGY0IKwfYBQ5PVcP+w+ZpsFlwcbKov74A1+vpFeQAZ8H6x3mKYT2R4jMpfRXgQzDA6avHw5UIzwM8l5dG/QGZ7A+BTmHeB5/Jp/tz5OxTT5AuLWtLuTKlXc4s/OpRfpkIf7b29hiW+bCXECPPFAJ8JqW/kvgQDHD66vFwJcLzAC+NS1NFyvzciiyNYVl3SZTpJNbBYsAQgEDgBPhMSh8xPgQDnL56PFyJ8DzA8+HSqKPCPkwhkiaItkSCnU4hAIEUBPhMSl8a+BAMcPrq8XAlwvMAz4dL+dBMDyJfHNLjxlX+E+DFJ/+Z2tgin0nprxo+BAOcvno8XInwPMDz4VIem6UPkfzS9Nllw5VhGFNMTzYoyfkc+Uxyzip/TXwIBjg95Xi8CuF5BOjxciLAHgFyOQSSEAjLmHL/Ij8IeCeAD8EAe1dRGi0gvDSg+XhJWH+ofRwyTUEg9gTCMqY8wYm9FAIZYBhPF/wYuC3jxIdggP3Qu+s2EJ5rZL5fwGMz35HSYJYTCMuYhmW0s3w5YzV9W4IWtoxTFxcfggGO5CZHeJFgp1MIQCBAAmEZU5tMRoC4s6rpsLTlFaot48QAF7/SpXJz9UwhShAEMMBBUKVNCEAgSgJhGlOe4ES50uH3HdbTBa8zs2WcGGAMsFetp309BjhtdFwIAQjEmADGNMaLY/HQbIms2jJODDAGOLKPAwxwZOjpGAIQgAAELCMQ5tMFL2hsGScG2BIDfPz4cZk7d668++67cvToUSmcmTFz5kwveo3kWgxwJNjpFAIQgAAELCVgy9MFW8aJD0l9I8QmB3jw4MHy+OOPS/PmzaVKlSpFRvz2229bdzvbKDxbtnaxTgwlDBjumbOirGXmrCUzgYDtBGz0IWExj40BPv3002Xs2LEyYMCAsOYeeD+2Cc+mxzqBL16IHcA9RNgBd8VaBgw4ps3zpSemC8Ow2AatGA3ExgCfdNJJsnbtWqlfv37GSNY2A2xTYn/GiERE4J45q8laZs5aOp0JX3qckqJeFARs8yFhMoqNAe7atatcccUVcvvtt4c5/0D7sk14Nm3tEujChdw43EMGHmB3rGWAcGPaNF96YrowDMsQsM2HhLlssTHAEydOlDFjxkj79u3lnHPOkQoVKhTgMHr06DC5+NKXbcLjg9yXZXfdSCru4zo3kTKlS8vO/Tly9qknSJeWtaVcmdKu2+eC8AhwD4XHOi498aUnLivBOJIRsM2HhLmKsTHAdevWTTnvUqVKybZt28Lk4ktftgmPR3m+LLvrRpJxb1X3VNPOyu3789q7qN5pMrtvK0ywa8LhXcA9FB7ruPTEl564rATjwAC700BsDLC7YdtR2zYDrFRt2drFDgU4H2Vh7t8dz5V7XllfpIHx1zeV7q3Oct4wNUMnwD0UOvJIO4zjl544vpQXxzFFKpyQOrfRh4SERmJngN944w356KOPpFy5ctKkSRO58sorpUyZMmHx8LUfhOcrzqxqjMeqWbXcTNZyAnH60hNXQ97r6RWynCdaoSsdH5IaeWwM8MGDB+Xaa6+VVatWSdWqVUUPxjh06JC0bNlS3nzzTfMz2wrCs23F4jNeHqvGZy0YSWYRyPRIZBw/O+I4psxSderZ4EMsMMC33HKLrFixwpwGd95555kRf/jhh3LTTTdJ27ZtZerUqdbpFeFZt2SxGXAcozixgcNAIJAmgWy4r+L49CiOY0pTQtZdhg+xwADrQRjz5s2Tdu3aFRjtO++8IzfeeKP885//RHjWEWDAXgjE6bGql3lwbfYSiFu0NRsikW7nGMYasdNNdJ8BGGALDLCmOCxfvlwaN25cYLSbNm2SFi1aSE5OTnQKSrNnhJcmOC6DAASsJxDHaGs2RCLdcHdT14sgg9rpxql5d1rPyxzjei0+xAIDfNVVV5nUh8cff7zAaO+44w754IMP5L333ourvlKOC+FZt2QMGAIQ8ImA20ikT90W20wcxxTEvJ0+PQqTh9873Tg1707rBbEOcWgTH2KBAf773/9uToJr3ry5XHLJJaJ7/y5btswcj/zaa6+JGmTbSrYJL5u/ZdumTcYLgaAJxDHamu1mqPCaR7lGXvt2at6d1gv6foiq/WzzIW44x2YXCB30ypUr5ZFHHpH169dLbm6uNG3aVH75y1/KRRdd5GZOsambTcLjD0tsZMdAIBAJgcJfgOO6l7XT6GgkEEPuNEpz6LVvpwY6Vb1b29WTutUqZ/xJm9nkQ9zePrEywG4HH/f62SQ8rx9mcV9LxgeBMAjY+hQlqBzPMJhncx9RBi689u30b06qemp+t//r67zlz9STNrPJh7i9lyM1wD//+c9Nzu9JJ50k+v+LKzNnznQ7t8jrZ5PwnH4bj3xRGAAEYkrAqyGIclqpTMaDnc+TMqVLyaf7c+TsU0+QLi1rc5R3lAuVpO8oI+Je+nZ6vySrV9j8JrBk4kmb2eRD3N5akRpgzfl9+eWXzSEX+v+LK2+//bbbuUVeP5uE5/TbeOSLwgAgEFMCNt9DfAGOqagyfFhODXThehr5nf7XbUXo9L+8vtzdvuBOVLYjzCYf4natIjXAbgdrW/1sEl5x38Zzc0Xmrf4s43OtbNMn43VOIIzUBJtNZFzNexjr5lxF1IwLgbjqNQg+2eRD3PKLjQH+61//mnTsuhtE+fLlpXbt2lKrVi1H89u/f7+MHDlSFi1aZI5TbtasmYwfP96cKKelT58+8swzzxRoS9tWoWjRY5jHjh0rM2bMkAMHDpjrpkyZIg0aNHDUf6JStgkv2bdxNb+cAe9KNlSOGQGnj1q9DtvmP8phMXLDOI5jcjN+6gZHIJu0kW0+xI1qYmOAy5UrZ4ynFt0BQoua3/zl8ssvN6fFacpEceXaa6+Vffv2yZNPPil6wtzkyZONmV29erU5aKNly5bSvn170T2GE6VMmTKmrhY1v2p4Z82aZUz38OHDZdu2bbJhwwZjxp0WhCdi8x91p+tMPf8JxClyF5aGbf+j7PRxtP9qSd5iWOsW1nzoxxkBp58dcdOrs9m5r4UPSc0sNgZ4zpw5JmqrpvXSSy81I9aT4QYOHCj9+/c3W6ENHjxYzj//fJk2bVrKGW3ZskUaNmwof/vb36RNmzZ5hrpRo0by05/+VO677z458cQTZe7cudK5c+ci7Rw9elSqVasmEydOlH79+pnfHzx4UGrWrCn6Il737t0dKxDhidj8WNfxQlPRVwJxM4Jhajhb/ij7KpgUjYW5bmHMJ+o+nBpLHaebun7OK26fHX7OLd228CEWGGBNL9BI7XXXXVdgtEuWLDFG9JNPPhE9LOP666+XPXv2pJyRmlU9NU4PzqhQoUJePW1fo75qqM855xzZvHmzqCkuXHQv4tatWxf5vaZBaCqFRoadFoRHBNipVqj3PwJxi9zFbTxoxRkB1s0ZJye13BhLN3Wd9O2mDmtelBY+xAIDXLlyZXPksaYo5C+bNm2SCy64QA4fPiw7d+40v8/JyXFzT8iLL74o3bp1k4ULF8o333wjN954ozHVesJc6dKlpWPHjjJu3DipUqWKzJ8/X7p06WL6qFSpUl4/er3+TPOKnRaE959IADnAThVDPSUQt8gdGrZTl6ybf+vmxli6qevfCP/TUtw+O/yeXzrt4UMsMMCXXXaZ1KtXz+Tqaj6ulmPHjsktt9wi//jHP0w6xOzZs83LbJqL67RoKoRGfjUi/Morr5gUiAcffFAeeughY3w1sjx06FA588wz5a233hJNxejZs6fpW81xovTq1Ut2794tixcvTtm1vnCn/yWKRqpbtWolu3btMi/xZWvhsW62rnx6847yD2iqEaPh9NYy6qtYN39WwI2xdFPXn9H9r5U4fnb4PUe37WGALTDAGv1Vk3rKKaeYl9T0hTj92VdffSWvv/66+be+BPfb3/5WbrvtNkcaWLBggfTo0cPkD2vkViO6+oKdtpn/RTo11xdffLEx2SqWrl27Jo0AHzlyRLTNVGXMmDHmBbrCJWoDHFU+lqNFohIEChEgcockMpWArZ/Fboylm7p+rzOfHUWJYoAtMMA6RI2Y6gtua9askbJly0rz5s1lwIABZncGjQJ//vnncvXVVzu6Z/RlukGDBpmc4eeee65APnDhBr7++mvzYtxLL71kIsGaA6wv09WvXz+vquYA63g0TzlViUMEuPAH7I+b15S+z7wvy7fvzxt2ph756EgYVLKCAJE7K5aJQbogYLM5czN2N3Vd4HNclc+OgqgwwJYY4MQwNdKq240V3gbN6R0wdepUY5zvvPNOefTRRwukMmhE+Msvv5Q33ngjr7mlS5eak+g2btxo0jCqV68ukyZNkr59+5o6tuwCke1HPjrVB/UgAAEIlETA72htlJHRkubq5PdujKWbuk76pk76BDDAlhhgjf7q9mP6stvHH38sDz/8sNSoUUNGjx7tePX1uvPOO09+9KMfFdmxQVMgli1bJp06dZIHHnjAbGmm9W+//XaTAvH888+bfkaNGiXTp083257VqVNHhg0bJjt27JB169bFeh/gVB+wyeBl4pGPjkVCRQhAAALFEAgiihllbiyLnb0EMMAWGGB9+UyjtnfddZcxwevXrzd5u3fffbfJq9XDKJwUfblNDWyy0rt3b3MCnB6moS/C6Q4TmgusUWHdBaJixYrmMn0BTvck1oMwdPeJdu3amdQHNcNuStjCS/UBm2zM469vKt1bneVmOtSFAAQgkBUEgojWBtFmViwGk/REIGwf4mmwIV8cm4MwWrRoYXJ21aSedNJJ8uGHH5p0hGeffdaYU92twbYStvBSfcDWrVZZtv/r6zx85ADbpiTGCwEIhEkgiGhtEFHlMJnQl50EwvYhNlGKjQHWfYA16lu3bt0CBnj79u1y7rnnmkisbSVs4aX6gJ1584WycO1u+XR/jpx96gnSpWVtKVfmf1u82caV8UIAAhAIkkBQ0VpyY4NcNdpORiBsH2LTKsTGAOuOC7pzQ4cOHQoYYN3BQffu3bp1q01czVijEB4fsNbJhAFDAAIxI0C0NmYLwnDSJhCFD0l7sCFfGBsDrHm/unvDI488InrohJ7epi+e3XPPPTJ48OCUeb0h83LVHcJzhYvKEIgFAb/f/o/FpBiEawIEE1wj44IYEsCHpF6U2BhgHaK+ePbYY4+Z44q16F7AemSx/iz/qWwx1FjSISE8W1aKcULgPwSI/KEECEAgkwjgQywwwO+88460adNGvv32W7Mfr5781rhxYzn55JOt1SLCs3bpGHiWEggq9zNLcTJtCEAgYgL4EAsMsJ72podT6G4QmVIQXqasJPPIFgJBvP2fLeyYJwQgED8C+BALDHCTJk3MS3B6IlumFISXKSvJPLKFABHgbFlp5gmB7CCAD7HAAOuLbnoSXMeOHaVhw4aip7blL25Og4uLrBFeXFaCcUDAGQFygJ1xohYEIGAHAXyIBQZY9/9NVUqVKiXbtm2zQ235RonwrFsyBgwB4e1/RAABCGQKAXyIBQY4U8SWfx4ILxNXlTlBAAIQgAAE7CCAD8EAR6JUhBcJdjqFAAQgAAEIQCCiA7lsAR+rfYBtgeZ0nBhgp6SoBwEIQCCzCHCgSmatp62zwYcQAY5EuwgvEux0CgEIQCBSArxMGSl+Os9HAB9igQHOzc0VfdktkwrCy6TVZC4QgAAEnBFgOz1nnKgVPAF8iAUG+MILL5SZM2dKs2bNgldESD0gvJBA0w0EIACBGBHgQJUYLUaWDwUfYoEBrlatmqxYsULq16+fMXJFeBmzlEwEAhCAgGMCRIAdo6JiwATwIRYY4Icfflj+9Kc/ybBhw6RBgwZFDsI466yzApaJ/80jPP+Z0iIE/CTg5UUlL9f6OQfaih8BcoDjtybZOiJ8iAUGuHTp0nmjzJ8LnMgNPnbsmHX6RXjWLRkDziICXkyKl2uzCHFWT5UDVbJ6+WMzeXyIBQb4nXfeKVYwl112WWwE5XQgCM8pKepBIHwCXh5Te7k2/JnSIwQgkK0E8CEWGOD8Qzxy5IhUqFDBer0iPOuXkAlkMAEvLyp5uTaDkTI1CEAgZgTwIZYY4GnTpsmECRNk165d8vHHH4vmBdeoUUNGjx4dM0k5Gw7Cc8aJWhCIgoCXKK7ba8kXjmKF6RMCEMCHWGCA58yZIwMGDJC77rpLJk6cKOvXr5dFixbJ3XffLWPHjpXhw4dbp2SEZ92SMeAsIuAlj9fNtW7qZhF+pgoBCIRAAB9igQFu0aKFDBo0SHr37i0nnXSSfPjhh1KvXj159tlnZdy4cfLJJ5+EIBV/u0B4/vKkNe8EiEQWZOjlRSWn17qNFntfZVqAAAQg8B8C+BALDHDlypVN1Ldu3boFDPD27dvl3HPPlcOHD1unZ4Rn3ZJl9ICJREazvOQLR8OdXiEAAQxwcRoolav7jMWg6AEYTz75pHTo0KGAAX7uuefkvvvuk61bt8ZglO6GgAF2x4vawRIgEhks31Stwz0a7vQKAQhggK0wwJr3O3XqVHnkkUekV69e8uKLL8qOHTvknnvukcGDB8uoUaOs0zIG2Loly+gBE4mMZnmJvEfDnV4hAAEMsBUGWAc5cuRIeeyxx+Sbb74xYy5btqz069fP/Cz/QRm2iBoDbMtKpR5nJuXMEomMTo9O84WjGyE9QwACmUgAH5J6VWOTApEYYk5OjmzcuFGOHz8ujRs3lpNPPtlaTSI8a5fODDzTIneZNh+71cXoIQABCARPAB9iiQH+97//LS+88IKsW7dOypQpIy1btpSuXbtKxYoVg1dJAD0gvACghthkJkZMiUSGKCC6ggAEIBAxAXyIBQZ406ZNcsUVV4ia4EaNGpkIsG59VqtWLXnrrbekdu3aEcvIffcIzz2zOF1BzmycVoOxQAACEICAWwL4EAsM8DXXXCPlypWT559/Xk455RQz4n/961/So0cPOfHEE2X+/Plu1z3y+ggv8iXwNIBMjAB7AsLFEIAABCBgFQF8iAUGWN6fRJcAACAASURBVE3uihUrpEmTJgVGqwdiXHrppXLo0CGrRKeDRXjWLVmBAZMza/f6MXoIQAAC2U4AH2KBAda0h9/+9rdy3XXXFRjtsmXL5Gc/+5ns3LnTOh0jPOuWrMiAyZm1fw2ZAQQgAIFsJYAPscAAv/rqq3LXXXfJpEmT5LLLLjPpEO+//74MGDBAbrnlFvMyXKKcddZZVmgZ4VmxTAwSAhCAAAQgkJEE8CEWGOD8+/yWKlUqb8SJg+r0Z/r/9X+PHTtmhVARnhXLxCAhAAEIQAACGUkAH2KBAX7nnXcci08jxDYUhGfDKjFGCEAAAhCAQGYSwIdYYIAzUXoILxNXlTlBAAIQgAAE7CCAD8EAR6JUhBcJdjp1SSCTjnt2OXWqQwACEMhoAvgQDHAkAkd4kWCnUxcE2OrNBSyqQgACELCMAD4kywzw/v37ZeTIkbJo0SKzf3CzZs1k/Pjx0rZtW0Ni7dq1MmjQIFm1apWcdtppcuedd8rQoUPzKOkpdGPHjpUZM2bIgQMHzHVTpkyRBg0auJI+wnOFi8oREOCwjwig0yUEIACBkAjgQ7LMAF977bWyb98+efLJJ+X000+XyZMnGzO7evVq8+/GjRtL586dZfDgwbJ8+XKz1Zoa3D59+hhSan7137NmzTJHMQ8fPly2bdsmGzZskPLlyzuWLcJzjIqKERHguOeIwNMtBCAAgRAI4EMsMcBz5syRdu3aSe3atWXcuHHywgsvyCWXXCKPP/64VKxY0ZFUtmzZIg0bNpS//e1v0qZNG3ONbp+mB2389Kc/lUqVKhlDvGPHDilbtqz5vUaL9ajlTZs2ydGjR6VatWoyceJE6devn/n9wYMHpWbNmjJz5kzp3r27o3FoJYTnGBUVIyJABDgi8HQLAQhAIAQC+BALDLAa3gcffFCWLFli9vpV46sHYCxdulQ6duwojz32mCOpqFl977335KqrrpIKFSrkXaPpC+3btzeR3KpVq4qa7URZvHixXHPNNbJ3715jjFu3bi2bN282pjlRNA1CUyk0Muy0IDynpKgXFQFygKMiT78QgAAEgieAD7HAANerV08mTJggN9xwgwwbNsxEcNXI6lHIGnX9/PPP01bKiy++KN26dZOFCxfKqFGjpEOHDqavRNm4caM0adLEnDynRy536dJFcnJyTLQ4UfR6/ZnmFTstCM8pKepFSYDjnqOkT98QgAAEgiOAD7HAAGuKg6YvaPrD+eefLz/5yU/kvvvuk08//VTOOeccYz7TKWqkNfKrEeFXXnnFvMjWo0cPuf/++/Oa06hw/fr1jdnWCHDPnj3NaXP5T6fr1auX7N69WzRanKroC3f6X6Ls2bNHWrVqJbt27TLzokCALcfQAAQgAAEIhEUAA2yBAVZj+rvf/U7OPvvsvBzeiy++WJ577jl54IEHTEqC27JgwQJjdi+66CITudWIbtOmTU1KRbIIsL4kp2a4a9euSSPAR44cEW0zVRkzZox5ga5wwQC7XbnMrE+6QWauK7OCAAQgEFcCGGALDPBvfvMbeeihh4xJ1Z0adKsyzbfVdAg1lfm3KXMiNN0BQrc6u/76642JTuQDq/nVrc9mz56d10yyHGCNRmtUOFE0B7h58+bmBbpUhQiwk5XJ3jq8cJa9a8/MIQABCERBAANsgQHWIb766qsmAnvTTTcZk/r888/L4cOHzctwbsrUqVPN1ma6v++jjz5aIJVB9wOeNm2abN26VcqUKWOaHTFihLz88stmFwiN8lavXl0mTZokffv2Nb+P6y4QPE53o4ro67LlWPRrwAggAAEIZBMBDLAlBjgxTDWh+XdwcCPWjz/+WM477zz50Y9+VGTHBo0ua9u6D3CnTp3M/r4rV66U/v37G1Pcu3dv05W+KDd9+nSz7VmdOnVMFFpzg9etWxebfYB5nO5GFfGoSwQ4HuvAKCAAAQhkCwEMsCUGWE2o5uZqzqwa2Ycfflhq1Kgho0ePdqxVTaNQA5usqMF95plnzG4PGh1es2aNaX/IkCEycODAvEv0BTjdG1gPwtAItO5NrKkPaobdlCCFh5lysxLxqMuXlnisA6OAAAQgkC0EgvQhtjMslaunRMSg6L68mrZw1113mUMo1q9fb15cu/vuu00OsEZrbStBCo/H6bap4T/jZcsxO9eNUUMAAhCwkUCQPsRGHvnHHBsD3KJFC/PSmkZpTzrpJPnwww9F9wZ+9tlnzalwn3zyiXWsgxQeEWDr5MCAIQABCEAAAqESCNKHhDqRADqLjQGuXLmyifrWrVu3gAHevn27nHvuuSYVwbYSpPB4nG6bGhgvBCAAAQhAIFwCQfqQcGfif2+xMcC65ZhuXaantOWPAOsWZnoghu7aYFsJWnjZ+jid3S9suxMYLwQgAAEIREEgaB8SxZz86jM2BljzfnX7skceeUT01DU9vlh3Xrjnnntk8ODBKV9s8wtEEO0gPP+pEvl2zpQvCs5ZURMCEIBAJhLAh6Re1dgYYB2i7rzw2GOPyTfffGNGXLZsWenXr5/5Wf5jiW0RKcLzf6XIfXbGlC8KzjhRCwIQgEAmE8CHWGKAdZg5OTmyceNGOX78uNmv9+STT7ZWmwjP/6Vj9wtnTPmi4IwTtSAAAQhkMgF8iCUGWCO/uh2avgynB2E0bdpUunXrZiLBNhaE5/+qYeycMeWLgjNO1IIABCCQyQTwIRYYYN3toW3btnLo0CFp1KiR6GEUehjG2WefLa+99prrQyjiIGiE5/8q8GjfGdMwvyiQa+xsTagFAQhAIGwC+BALDPC1114rFStWlNmzZ0uVKlXMiL/44gvp3r27nHDCCfLqq6+GrRvP/SE8zwiTNpCtu1+4oRnWF4Ww+nEzd+pCAAIQgMB/COBDLDDAanJXrVpl9vzNX9auXSuXXHKJfP3119bpGeFZt2QZNeAwviiEGWnOqMVhMhCAAARCIIAPscAAa9rDlClT5Oqrry4w2rfffltuvvlm+fTTT0OQir9dIDx/edJa/AiQaxy/NWFEEIAABBIE8CExNcA7d+7MG9nChQvlN7/5jTkMQ3OBy5QpIx988IHceuutcu+990rPnj2tUzTCs27JGLBLAkSAXQKjOgQgAIEQCeBDYmqAdW/fUqVK5Y0uNzfX/P/CP9N/60txthWEZ9uKMV63BMgBdkuM+hCAAATCI4APiakBfueddxyr4LLLLnNcNy4VEV5cVoJxBEkgWa6xfpedt/oz2bk/R84+9QTp0rK2lCtTOshh0DYEIAABCBQigA+JqQFONSzd/aFcuXJStWpVq8WM8KxePgafJgGiwmmC4zIIQAACPhPAh1higCdPniwPPvig7N2714z4e9/7ngwbNkzuuusunyURTnMILxzO9BIvAuQFx2s9GA0EIJC9BPAhFhjgGTNmyMCBA81/7dq1Mzm/f/3rX2XatGmixvjnP/+5dQpGeNYtGQP2gQA7Q/gAkSYgAAEI+EAAH2KBAW7cuLEMGjRI+vfvX2C0an7VBK9bt84HKYTbBMILlze9xYMAEeB4rAOjgAAEIIAPscAAV6pUSdavXy/169cvMNqtW7fKeeedJ4cPH7ZOyQjPuiVjwD4QIAfYB4g0AQEIQMAHAvgQCwywRoDHjh0rN954Y4HRzp07V0aMGCE7duzwQQrhNoHwvPNWM8VuAt45ht1CGKfQhT0n+oMABCBgGwF8iAUG+NFHHzUvwD3wwAPm6GPd+3fZsmUyevRoueOOO+S+++6zTXecwe1xxYgkegTI5RCAAAQgkNUEMMAWGODjx4/L4MGDZerUqfLdd9+ZEetpcHoS3BNPPCF6aIZtBeF5WzFySb3x42oIQAACEMhuAvgQCwywHorRpk0b+eabb2TTpk2ip8JpWsTJJ59srXoRnrelYzcBb/y4GgIQgAAEspsAPsQCA3z66afLG2+8IS1atMgYtSI8b0tJBNgbP66GAAQgAIHsJoAPscAAN2nSRJ588km54oorMkatCM/bUpID7I0fV0MAAhCAQHYTwIdYYIA1/1f3++3YsaM0bNhQdFu0/EVfhrOtIDzvK8ZuAt4Z0gIEIAABCGQnAXyIBQa4bt26KUepO0Js27bNOvUiPOuWjAFDAAIQgAAEMoYAPsQCA5wxass3EYSXiavKnCAAAQhAAAJ2EMCHWGKAdSu0JUuWmGOPdQu0li1bStu2be1QWZJRIjxrl46BQwACEIAABKwngA+xwADv2bNHrrvuOnMcctWqVc02aF999ZU5FGPRokVSpUoV64SI8KxbMgYMAQhAAAIQyBgC+BALDPD1118vu3btkueee06+//3vmxFv2LBBevXqJc2aNZNZs2ZZJ0iEZ92SMWAIQAACVhHguHirliv0weJDLDDAeuDF22+/bdIe8pcVK1ZIhw4dZP/+/aELx2uHCM8rQa6HAAQgAIFUBNgqEm2URAAfYoEBPuuss+QPf/iDXHzxxQVG+8EHH5it0fbu3VvSOsfu9wgvdkvCgCAAAQhkDAEOC8qYpQxsIvgQCwzwM888I4888ohJgWjevLkZ8fbt26Vnz57So0cPGTBgQGACCaphhBcUWdqFAAQgAAGOi0cDJRHAh1hggM8880zZt2+ffPfdd3LSSSdJuXLlTNqDvgyn+wDnL8eOHStpzWPx+0wWHnlnsZAYg4AABLKYABHgLF58h1PPZB/iEEHKaqVy1WHGoDz77LOOR9G7d2/HdaOsmKnCI+8sSlXRNwQgAIH/EOCzGCWURCBTfUhJ83by+9gYYCeDta1OpgqPqINtSmS8EIBAphLguPhMXVl/5pWpPsQPOhhgPyimaCNThUfeWYCioWkIQAACEICATwQy1Yf4gQcD7AfFLDPARIADFA1NQwACEIAABHwigAFODTLjDfC4ceNk8eLFsnTp0jwKffr0Ed11In+pVauWqFC06JHMY8eOlRkzZsiBAwfMccxTpkyRBg0auJJkpgqPvDNXMqAyBCAAAQhAIBICmepD/ICZ0Qb4sccek8GDB0u7du0KGGA9bKN9+/Zyxx135DEsU6aMnH766ebfan7V8Orpc2qMhw8fLtu2bTMn05UvX94x90wWHnlnjmVARQhAAAIQgEAkBDLZh3gFGisDvHHjRlm/fr188803RealRyI7LZ9//rnccsstsmzZMtHt1c4444w8A6xbqJ144okyd+5c6dy5c5Emjx49KtWqVZOJEydKv379zO8PHjwoNWvWlJkzZ0r37t2dDsNElLV/PeK5du3ajq+jIgQgAAEIQAACEPBKAB+SmmBsDPD48eNl5MiRSUeq+wC72fv31VdfNQZX0x/uv/9+2bFjR54B3rRpk5xzzjmyefNmadSoUZH+Vq5cKa1bty7ye02DaNasmYkMOy0Izykp6kEAAhCAAAQg4DcBfIgFBvh73/ueibiOGDFCKlSo4JsGbr755gIG+MUXX5Qbb7zR9PXaa69J6dKlzVHLaparVKki8+fPly5dukhOTo5UqlQpbxzdunUzP1u0aFHKsR06dEj0v0TZs2ePtGrVigiwb6tJQxCAAAQgAAEIOCWAAbbAAJ966qmi0Ve3L5qVJILCBvi+++6TBx98UB566CFjfD/55BMZOnSoSVV46623ZM6cOeb4ZY04qzlOFE3B2L17t3mhLlUZM2aMyR8uXEiBKGmV+D0EIAABCEAAAn4TwABbYIDvuusuE/mdMGGCr+tf2ADrwXdfffWVVK1aNa+f5cuXy8UXXyz6vyqWrl27Jo0AHzlyRBYsWEAE2NcVojEIQAACEIAABIIggAG2wABrdFVzbCtXrix169YtEH3V4Wt0Np1S2AAna+Prr782L8a99NJLJhKsOcBbtmyR+vXr51XXHODmzZvL5MmTHQ8D4TlGRUUIQAACEIAABHwmgA+xwABrOsKKFSvk6quvlhNOOKHIiHVLsnRKYQPco0cP+fLLL+WNN97Ia073CL7iiitEd6GoV6+eVK9eXSZNmiR9+/Y1ddgFIh3yXAMBCEAAAhCAQJQEMMAWGGA1vUuWLDGpCH6WwgZYX2Lr1KmTPPDAA2ZLs48//lhuv/120+/zzz9vuh41apRMnz7dbHtWp04dGTZsmHmRbt26dewD7Ofi0BYEIAABCEAAAoERwABbYIA18rpw4UI577zzfBVCshSIefPmmRfhdEs0zQXWqLDuAlGxYkXTt74Ap1uyadT58OHD5iANTX1QM+ymIDw3tKgLAQhAAAIQgICfBPAhFhjgP/7xjzJt2jRjNHV/Xj2ZzfaC8GxfQcYPAQhAAAIQsJcAPsQCA9ywYUPZuXOnfPfdd0lH6+YgjLhIFeHFZSUYBwQgAAEIQCD7COBDLDDAzz77bLHK7N27t3XKRXjWLRkDDpnA0e+Oy7zVn8nO/Tly9qknSJeWtaVcmf/tvx3ycOgOAhCAQEYRwIdYYIAzSnH/nQzCy8RVZU5+EVDz2+vpFbJ8+/68Ji+qd5rM7tsKE+wXZNqBAASymgA+xAIDfP/99xcr0tGjR1snYoRn3ZIx4BAJzF25U0bMX1ekx/HXN5Xurc4KcSR0BQEIQCAzCeBDLDDAevhF/vLtt9/Kvn37zLZjbdq0kb/85S/WqRPhWbdkDDhEAhNe3yRTl24t0mP/y+vL3e0bhzgSuoIABCCQmQTwIRYY4GRDPHTokGjurx5Sceedd1qnToRn3ZIx4BAJEAEOETZdQQACWUkAH2KpAdZhr1+/Xn74wx/Kp59+ap14EZ51S8aAQyRADnCIsOkKAhDISgL4EIsN8Jo1a+SSSy6RnJwc68SL8KxbMgYcMoFvjx2XeR98Jp+yC0TI5OkOAhDIBgL4EAsM8O9///sCo8zNzZWvvvrKHElco0YNWbx4sXVaRXjWLRkDhgAEIAABCGQMAXyIBQa4dOmie3+WK1fORH/1hDg9Hc62gvBsWzHGCwEIQAACEMgcAvgQCwxw5sjtfzNBeJm4qswJAhCAAAQgYAcBfAgGOBKlIrxIsNMpBCAAAQhAAAIigg+JqQG+8sorHQm0VKlSsmTJEkd141QJ4cVpNRgLBCAAAQhAILsI4ENiaoD79OlTrBKXLVsm27Ztk5NPPlkOHjxonWoRnnVLxoAhAAEIQAACGUMAHxJTA5xqWP/+979l8ODB8vTTT8vVV18tM2bMkLPOsu9oVISXMZ8hTAQCEIAABCBgHQF8iEUG+M0335Rf/OIXJuI7ceJEufXWW60TXGLACM/apWPgEIAABCAAAesJ4EMsMMD/93//Z6K+Gu296qqrTPTXxqhvftQIz/rPDiYAAQhAAAIQsJYAPiTmBjgR9T1w4ICJ+t52223Wig0DnBFLxyQgAAEIQAAC1hPAAMfUAGvUd8iQIQWivmeeeab1giMFImOWkIlAAAIQgAAErCWAAY6pAa5Tp47s2rVL6tWrJzfddJPodmepyujRo60TIMKzbskYMAQgAAEIQCBjCOBDYmyAizO9iWFrHd0OzbaC8GxbMcYLAQhAAAIQyBwC+JCYGuDMkVjymSC8TF/heMzv6HfHZd7qz2Tn/hw5+9QTpEvL2lKuTOl4DI5RQAACEIBAZATwIRjgSMSH8CLBnlWdqvnt9fQKWb59f968L6p3mszu2woTnFVKYLIQgAAEihLAh2CAI7kvEF4k2LOq07krd8qI+euKzHn89U2leyv7Do/JqsVjshCAAAQCJoAPwQAHLLHkzSO8SLCX2GkmpQxMeH2TTF26tcic+19eX+5u37hEFlSAAAQgAIHMJYAPwQBHom6EFwn2YjvNtJQBIsDx0xgjggAEIBAXAvgQDHAkWkR4kWAvttNMM4yZZujjpxhGBAEIQMBeAvgQDHAk6kV4kWAvttNMTBn49thxmffBZ/Ipu0DET3CMCAIQgECEBPAhGOBI5IfwIsGeVRHg+BFmRBCAAAQgEBcC+BAMcCRaRHiRYC+2U1IG4rcmjAgCEIAABIIhgA/BAAejrBJaRXiRYC+xU1IGSkREBQhAAAIQyAAC+BAMcCQyRniRYKdTCEAAAhCAAAREBB+CAY7kRkB4kWCnUwhAAAIQgAAEMMDFaqBUbm5uLioJhgAGOBiutAoBCEAAAhCAQMkE8CFEgEtWSQA1EF4AUGkSAhCAAAQgAAFHBPAhGGBHQvG7EsLzmyjtQQACEIAABCDglAA+BAPsVCu+1kN4vuKkMQhAAAIQgAAEXBDAh2CAXcjFv6oIzz+WtAQBCEAAAhCAgDsC+BAMsDvF+FQb4fkEkmYg4JGAHoAyb/VnspPjoj2S5HIIQMAmAviQLDbA48aNk8WLF8vSpUvzKKxdu1YGDRokq1atktNOO03uvPNOGTp0aN7vjx8/LmPHjpUZM2bIgQMHpG3btjJlyhRp0KCBK90jPFe4qAyBQAhw+l8gWGkUAhCwgAA+JEsN8GOPPSaDBw+Wdu3a5RngL7/8Uho3biydO3c2v1u+fLkMGDDAGNw+ffoYUmp+9d+zZs2SWrVqyfDhw2Xbtm2yYcMGKV++vGPJIzzHqKgIgcAIzF25U0bMX1ek/fHXN5Xurc4KrF8ahgAEIBA1AXxIlhngzz//XG655RZZtmyZnHnmmXLGGWfkGeBf//rXMnnyZNmxY4eULVvWkBk5cqTMnz9fNm3aJEePHpVq1arJxIkTpV+/fub3Bw8elJo1a8rMmTOle/fujvUcZ+HxSNjxMlLRcgITXt8kU5duLTKL/pfXl7vbN7Z8dgwfAhCAQGoCcfYhUa9bRh6E8eqrr8rcuXNF0x/uv/9+Y3YTKRAdO3aUqlWrypw5c/LYa4rENddcI3v37jV1W7duLZs3b5ZGjRrl1dE0iGbNmpnIsNMSV+HxSNjpClIvEwgQAc6EVWQOEIBAOgTi6kPSmYvf12SkAc4P6eabby5ggNXEdujQQSZMmJBXbePGjdKkSRN5//33ZefOndKlSxfJycmRSpUq5dXp1q2b+dmiRYtSrsGhQ4dE/0uUPXv2SKtWrWTXrl1Su3Ztv9cu7fYwBGmj40ILCfCFz8JFY8gQgIAvBDDAqTFmnQHWF9l69OhhIsOJovm99evXNykTGgHu2bOnHDt2TEqXLp1Xp1evXrJ7927zQl2qMmbMGJM/XLjEzQDzSNiXzxUasYjAt8eOy7wPPpNP2QXColVjqBCAgFcCGGAMcF4KRNOmTUXTIJJFgFevXm1eduvatWvSCPCRI0dkwYIFRIC93pFcDwEIQAACEIBA4AQwwBjgAjnAuvXZ7Nmz86gkywHesmWLiQoniuYAN2/e3LxA57TEVXg8Ena6gtTzkwAvXvpJk7YgAAEIlEwgrj6k5JEHXyPrUiDGjx8v06ZNk61bt0qZMmUM4REjRsjLL79sdoHQKG/16tVl0qRJ0rdvX/P7TNwFgkfCwd9c9PA/AnzpQg0QgAAEwieAASYCnBcB3rdvn9kHuFOnTmZ/35UrV0r//v2NKe7du7chNWrUKJk+fbrZ9qxOnToybNgwkxu8bt069gEO//6lxwwgwIuXGbCITAECELCOAAYYA1zgJDjd7UFPf1uzZo3UqFFDhgwZIgMHDsyjpC/A6d7AehDG4cOHzUEamvqgZthNQXhuaFE3kwnw4mUmry5zgwAE4koAH5LFBjhKUSK8KOnTd5wIEAGO02owFghAIFsI4EMwwJFoHeFFgp1OY0iAHOAYLgpDggAEMp4APgQDHInIEV4k2Ok0pgR48TKmC8OwIACBjCWAD8EARyJuhBcJdjqFAAQgAAEIQEBE8CEY4EhuBIQXCXY6hQAEIAABCEAAA1ysBjJ+H+Ao7wAMcJT06RsCEIAABCCQ3QTwIUSAI7kDEF4k2OkUAhCAAAQgAAEiwESAo7oLMMBRkadfCEAAAhCAAATwIUSAI7kLEF4k2OkUAhCAAAQgAAEiwESAo7oLMMBRkadfCEAAAhCAAATwIUSAI7kLEF4k2OkUAhCAAAQgAAEiwESAo7oLMMBRkadfCEAAAhCAAATwIUSAI7kLEF4k2OkUAhCAAAQgAAEiwESAo7oLMMDuyB/97rjMW/2Z7NyfI2efeoJ0aVlbypUp7a4RakMAAhCAAAQgYAjgQ4gAR3IrIDzn2NX89np6hSzfvj/voovqnSaz+7bCBDvHSE0IQAACEIBAHgF8CAY4ktsB4TnHPnflThkxf12RC8Zf31S6tzrLeUPUhAAEIAABCECACHAJGuAo5ABvEgywc7gTXt8kU5duLXJB/8vry93tGztviJoQgAAEIAABCGCAMcDR3QUYYOfsiQA7Z0VNCEAAAhCAgBMC+JDUlIgAO1FQmnUQnnNw5AA7Z0VNCEAAAhCAgBMC+BAMsBOd+F4H4blD+u2x4zLvg8/kU3aBcAeO2hCAAAQgAIEkBPAhGOBIbgyEFwl2OoUABCAAAQhAgG3QitUAKRAB3iIY4ADh0jQEIAABCEAAAsUSwIcQAY7kFskU4XFARSTyoVMIQAACEICAJwKZ4kM8QUhxMRHgIKj+t81MEB4vpwUoEJqGAAQgAAEIBEggE3xIUHgwwEGRzZDcG7YnC1AgNA0BCEAAAhAIkAAGmBSIAOWVuulMEB4HVEQiHTqFAAQgAAEIeCaQCT7EM4QUDRABDoosEeAAydI0BCAAAQhAAAIlEcAAEwEuSSOB/D4ThEcOcCDSoFEIQAACEIBA4AQywYcEBYkIcFBkMyQCrHg4oCJAkdA0BCAAAQhAICACGGAiwAFJq/hmEV4k2OkUAhCAAAQgAIEMCsQFsZhEgIOg+t82McABwqVpCEAAAhCAAASKJYAPIQIcyS2C8CLBTqcQgAAEIAABCBABLlYDRIADvEUwwAHCpWkIQAACEIAABIgAp6kBDHCa4JxchgF2Qok6EIAABCAAAQgEQQAfkpoqBjgIxf23TYQXIFyahgAEIAABCECACHCaGsAApwnOyWUYYCeUqAMBCEAAAhCAQBAE8CFEgIPQVYltIrwSEVEBAhCAAAQgAIGACOBDyqGIIAAAGNlJREFUMMABSav4ZhFeJNjpFAIQgAAEIAABdoEoVgOkQAR4i2CAA4RL0xCAAAQgAAEIFEsAH0IEOJJbBOFFgp1OIQABCEAAAhAgAkwEOBmBTz/9VOrUqVPkV0899ZTccsstsnbtWhk0aJCsWrVKTjvtNLnzzjtl6NChrm4oDLArXFSGAAQgAAEIQMBHAvgQIsBFCLz66qvSrVs32bZtm5QqVSrv91WqVJGcnBxp3LixdO7cWQYPHizLly+XAQMGyJQpU6RPnz6OpYnwHKOiIgQgAAEIQAACPhPAh2CAixB46KGH5I9//KOJ9BYuv/71r2Xy5MmyY8cOKVu2rPn1yJEjZf78+bJp0ybH8kR4jlFREQIQgAAEIAABnwngQzDARQh0795dSpcuLXPmzCnyu44dO0rVqlUL/G7x4sVyzTXXyN69e6V69eqOJIrwHGGiEgQgAAEIQAACARDAh2CAixBo0qSJ1KhRQ44cOSIff/yxNGzYUO6991657rrrpFmzZtKhQweZMGFC3nUbN24Uveb999+XCy+8MCnRQ4cOif6XKHv27JFWrVrJrl27pHbt2gFImyYhAAEIQAACEIBAcgIYYAxwAQJHjx6VypUrywUXXCAPP/ywnHjiifLcc8/J448/Lm+++abcdttt0qNHD7n//vvzrtNc4fr168uyZcukbdu2SYmOGTNGxo4dW+R3GGA+miAAAQhAAAIQCJsABhgDXITA119/bfJ7K1SokPe79u3bmxfiVDCaBpEsArx69WpjnJMVIsBh39r0BwEIQAACEIBAKgIYYAywo7tj2LBh8vrrr8uZZ55ptj6bPXt23nXkADtCSCUIQAACEIAABGJCAAOMAS5A4KOPPpI2bdoYs5s/neGKK66QM844Q84//3yZNm2abN26VcqUKWOuHTFihLz88svsAhGTm5phQAACEIAABCBQPAEMMAa4AIHjx48bA6xpEFOnTpVq1arJ7373O7PPr77kpiZY9wHu1KmTDB8+XFauXCn9+/c3prh3796O7zeE5xgVFSEAAQhAAAIQ8JkAPgQDXITAF198Ib/61a/kz3/+sxw8eFBatGgh48ePl0svvdTUVSOsp7+tWbPG7BYxZMgQGThwoCtpIjxXuKgMAQhAAAIQgICPBPAhGGAf5eS8KYTnnBU1IQABCEAAAhDwlwA+BAPsr6IctobwHIKiGgQgAAEIQAACvhPAh2CAfReVkwYRnhNK1IEABCAAAQhAIAgC+BAMcBC6KrFNhFciIipAAAIQgAAEIBAQAXwIBjggaRXfLMKLBDudQgACEIAABCAgYg720rMNOJG2qBxK5ebm5qKSYAggvGC40ioEIAABCEAAAiUTwIcQAS5ZJQHUQHgBQKVJCEAAAhCAAAQcEcCHYIAdCcXvSgjPb6K0BwEIQAACEICAUwL4EAywU634Wg/h+YqTxiAAAQhAAAIQcEEAH4IBdiEX/6oiPP9Y0hIEIAABCEAAAu4I4EMwwO4U41NthOcTSJqBAAQgAAEIQMA1AXwIBti1aPy4AOH5QZE2IAABCEAAAhBIhwA+BAOcjm48X4PwPCOkAQhAAAIQgAAE0iSAD8EApykdb5chPG/8uBoCEIAABCAAgfQJ4EMwwOmrx8OVCM8DPC6FAAQgAAEIQMATAXwIBtiTgNK9GOGlS47rIAABCEAAAhDwSgAfggH2qqG0rkd4aWHjIghAAAIQgAAEfCCAD8EA+yAj900gPPfMuAICEIAABCAAAX8I4EMwwP4oyWUrCM8lMKpDAAIQgAAEIOAbAXwIBtg3MblpCOG5oUVdCEAAAhCAAAT8JIAPwQD7qSfHbSE8x6ioCAEIQAACEICAzwTwIRhgnyXlrDmE54wTtSAAAQhAAAIQ8J8APgQD7L+qHLSI8BxAogoEIAABCEAAAoEQwIdggAMRVkmNIrySCPF7CEAAAhCAAASCIoAPwQAHpa1i20V4kWCnUwhAAAIQgAAERAQfggGO5EZAeJFgp1MIQAACEIAABDDAxWqgVG5ubi4qCYYABjgYrrQKAQhAAAIQgEDJBPAhRIBLVkkANRBeAFBpEgIQgAAEIAABRwTwIRhgR0LxuxLC85so7UEAAhCAAAQg4JQAPgQD7FQrvtZDeL7ipDEIQAACEIAABFwQwIdggF3Ixb+qCM8/lrQEAQhAAAIQgIA7AvgQDLA7xfhUG+H5BJJmIAABCEAAAhBwTQAfggF2LRo/LkB4flCkDQhAAAIQgAAE0iGAD8EAp6Mbz9cgPM8IaQACEIAABCAAgTQJ4EMwwGlKx9tlCM8bP66GAAQgAAEIQCB9AvgQDHD66vFwJcLzAI9LIQABCEAAAhDwRAAfggH2JKB0L0Z46ZLjOghAAAIQgAAEvBLAh2CAvWooresRXlrYuAgCEIAABCAAAR8I4EMwwD7IyH0TCM89M66AAAQgAAEIQMAfAvgQDLBrJR0/flzGjh0rM2bMkAMHDkjbtm1lypQp0qBBA8dtITzHqKgIAQhAAAIQgIDPBPAhGGDXklLzq4Z31qxZUqtWLRk+fLhs27ZNNmzYIOXLl3fUHsJzhIlKEIAABCAAAQgEQAAfggF2JaujR49KtWrVZOLEidKvXz9z7cGDB6VmzZoyc+ZM6d69u6P2EJ4jTFSCAAQgAAEIQCAAAvgQDLArWa1cuVJat24tmzdvlkaNGuVdq2kQzZo1M5FhJwXhOaFEHQhAAAIQgAAEgiCAD8EAu9LV/PnzpUuXLpKTkyOVKlXKu7Zbt27mZ4sWLXLUHsJzhIlKEIAABCAAAQgEQAAfggF2JavnnntOevbsKceOHZPSpUvnXdurVy/ZvXu3LF68OGl7hw4dEv0vUXbt2iVt2rQRjSjXqFHD1RioDAEIQAACEIAABLwQ2LNnj7Rq1Uq2b98uderU8dJUxl1bKjc3NzfjZuVxQvPmzZOuXbsmjQAfOXJEFixYkLSHMWPGmJ0jKBCAAAQgAAEIQCAuBDQQ94Mf/CAuw4nFODDASZYhkQO8ZcsWqV+/fl4NzQFu3ry5TJ482VEE+JtvvhGNAtetW1fKli0byIInvt0RZQ4Er+dGWR/PCANvgDUKHLGnDlgfT/gCv5j1CRyxpw6+++47+eKLL6Rp06ZSsWJFT21l2sUY4CQrqlHe6tWry6RJk6Rv376mRjq7QIQhFvJ7wqCcfh+sT/rswrqSNQqLdHr9sD7pcQvrKtYnLNL04zcBDHAKoqNGjZLp06ebbc80b2bYsGGyY8cOWbduneN9gP1erGTt8eETBuX0+2B90mcX1pWsUVik0+uH9UmPW1hXsT5hkaYfvwlggFMQ1RfgRo4caQ7COHz4sLRr186kPsQtiZwPH79vCX/bY3385RlEa6xREFT9a5P18Y9lEC2xPkFQpc0wCGCAw6AcYB+664SmagwePFhOPvnkAHui6XQIsD7pUAv3GtYoXN5ue2N93BILtz7rEy5vevOPAAbYP5a0BAEIQAACEIAABCBgAQEMsAWLxBAhAAEIQAACEIAABPwjgAH2jyUtQQACEIAABCAAAQhYQAADbMEiMUQIQAACEIAABCAAAf8IYID9Y0lLEIAABCAAAQhAAAIWEMAAW7BIyYZ4/Phxc+zyjBkz5MCBA6Kn1E2ZMkUaNGhg6YzsHvb+/fvNtnmLFi0SfSu6WbNmMn78eLMuWtb+//bOPPim8o/jn1C0oYRpCjG0ITSWZGwjskSiiJlCmizZyjJpk7UwyNhpEca+ljKWRglZSmZCthZmmEFqmilCpXl/5nfvXFx+9zbfe8+53/t6/uHL+Z7neV7ve+55n8/5PJ9n507r3bu3ffXVV1akSBHr1auX9evXL7MnnaGj379/v91///02ceJE69ixI/qESMdZs2b5dfPDDz/4LpzaXv6JJ55Ao5BodO7cOddk9uzZft+pXLmyjRw50h588EE0ColGDCNxAhjgxFmF6kiZXxle1Sm+7bbbbMCAAX7T2L17d6g26ggVtBQOplGjRnb8+HE3VUWLFvWa0Xo42bFjh/989913W8uWLb1c3ZYtW6x79+6uX6dOnVI4Kk59MQHdwHWz1oOIrh0Z4JMnT6JPCD4qc+bMcT3GjBljzZo1s3nz5rnZ2rhxo915551oFAKNBg0aZNOnT7cPPvjAypQp4+Z34cKF9t1331n+/PnRKAQaMYTECWCAE2cVmiPPnj1rt9xyi40aNcq6du3q4wrrVs2hgZbCgRw8eNDKlStnmzZtikZCzp8/7zftdu3a2bXXXuuGWDsJ5suXz0eiaPHSpUtt7969KRwZp76YgLjrAWT9+vVRA/zmm2+iT8AfFV0vMlSPP/64jR49Ojqahx9+2OrVq+c/cw0FLJKZR3wbNGjgDylqettVqFAhW7x4senNChoFrxEjSJwABjhxVqE5ctu2bVajRg3bt2+fm6xI0+t2vXpXZJGWPgJ6+Ni8ebPfGBQFiTSlozRu3Ngj84ULF7a5c+dG/2/dunXWsGFDO3bsmBUrVix9g83injZs2GCPPfaYp6OULFkyaoCbNm2KPgF/LvQgeM899/gbkypVqlwyGjQKWKD/da/vOO38tnr1aitRooSNGzfOXn31VX/z2LNnT66jcMjEKBIkgAFOEFSYDlPksHXr1nbq1CmPLkZamzZt/N+Uh0oLlsCiRYtMenz44Yf2yiuvWJMmTfx1YaTt2bPHypcvb9u3b7eqVasGO9gs6F0PKZUqVTJFe9u3b29XXXVV1ADroRF9gv0Q6Dp59NFH7YsvvrChQ4faN998Y6VLl3Zz1bx5c3+wR6NgNVLvu3btsrZt25q+v/LmzWt58uTx6G+LFi3QKHh5GEGSBDDASQILw+HKlXvqqafs77//9i+gSHv66aft6NGjpugiLTgCSoVQ5FfRkuXLl/vCRJmuIUOGRAcVWeSjG35koVxwI879PSsVRa/Z58+f75ONNcDoE7z+ke80mV7lmephZcmSJTZ8+HBbu3atdenShWsoeJlMD/bjx4+3/v37+9qTGTNm2IIFC0xvV1q1aoVGIdCIISROAAOcOKvQHKkbg3Ll4kWAz5w5YytWrAjNWLNtIGIvs/vAAw94JF4R+ooVK5pe4caLAF/ulW+2cUvlfLVifeDAgfbtt9/aTTfddIkBRp9U0k/s3FpIpcii0re6desW/SUthlM7fPgw11BiKFN21KFDh3ytw6effmq1a9eO9qO/a02K1kLwPZcy/Jw4BQQwwCmAmupTRnKA9YWjUkGRpkiiIidaiEBLPwFVgFCpM0VCFNGK5APrpqDSZzJikUYOcPr0qV+/vlcSiM3P/uOPP/xnXT+lSpVCn/TJEbcn5dDXqlXLUyBi34iouo0eJO+44w40ClijSFrX6dOnrUCBAtHRKBr8ySefcB0FrA/dJ08AA5w8s8B/Q1FeLZwaO3asde7c2cdDFYhgZZkyZYqXNlN9Xy0MiU1NUV3TqVOn2vfff+95c2qKSC5btowqEGmQ7ciRI6abdmxTJEsReUUdVW4LfdIgxBW6kD7Fixe3119//YL62Mr7vfrqq726ChoFq5Gqp9SsWdO2bt1q1atXjw5G6V4yxHrrhUbBakTvyRHAACfHKzRHa2HVtGnT7L333vPoiJ7CVWZLr3mvueaa0IwzGwai8j8VKlSwRx555JIKHEqB0AOL6gBroYgiWorg6zWvbhYdOnTIBkShm2NsDrDqN6NP8BINGzbMSzvqe00GS/naMsR65X7vvfeiUcASafMllaQ7ceKEf8/dfvvtpo1LtLBUb1hUxo7rKGCR6D4pAhjgpHCF52AtgFNNUxXzV/SkTp06nvogM0xLL4ERI0Z4pYd4TQZ35syZXu1B0WGtbr/11lutb9++1qNHj/QOlN6iBGINsP4RfcLx4dBbLaUSKWqvsmja8EfVIdAoHPpo9zdV5lBaiv6u/Hl9/9WtWxeNwiERo0iCAAY4CVgcCgEIQAACEIAABCCQ+QQwwJmvITOAAAQgAAEIQAACEEiCAAY4CVgcCgEIQAACEIAABCCQ+QQwwJmvITOAAAQgAAEIQAACEEiCAAY4CVgcCgEIQAACEIAABCCQ+QQwwJmvITOAAAQgAAEIQAACEEiCAAY4CVgcCgEIQAACEIAABCCQ+QQwwJmvITOAAAQgAAEIQAACEEiCAAY4CVgcCgEIQAACEIAABCCQ+QQwwJmvITOAAARSREBbv37++ecXnF1bjWs3v5YtW/o2sNruOqeadnLs2LGjvfHGGzl1ykvOc3Ef2hUvtuXPn9+3uW3btq3vcHjdddelbCycGAIQgEBQBDDAQZGnXwhAIPQEZICLFy9u48ePj471999/tzVr1lifPn3sueee8617c6qdOHHCDfUNN9yQU6dMyAC//fbbbnjVNL9t27ZZv379fDvi1atXW758+VI2Hk4MAQhAIAgCGOAgqNMnBCCQEQRkgBUxnTlz5iXjfeaZZ2zFihV28uTJjJhLZJDxIsDvv/++R55j29dff23VqlWzd955xzRXGgQgAIHcRAADnJvUZC4QgECOEriSAe7evbstXrzYjh8/7n2ePXvWXnvtNZszZ4799ttvVqFCBRsyZIg1atQoOiZFjgcOHGh79uyxMmXKWN++fa1z5872448/utG+2Jx+/PHHNnToUNu1a5cVLFjQ2rVrZ8OHD7cCBQr4OZW+MH36dFu4cKFt3LjRbr75ZuvRo4f3cbmWqAHW79epU8eU8rFu3boc5crJIAABCARNAAMctAL0DwEIhJZAPAP8119/eVrAk08+6SkQY8aM8fG3b9/edu/ebUonUA7tRx99ZC+99JItW7bMmjVrZjt37vSI6gsvvOCmVz9369bNfv3117gGePny5da6dWvPB27Tpo0dOHDAj69evbotWbIkaoALFSrkaRj699mzZ9uwYcNsw4YNVrt27bhckzHAzz//vC1YsMB+/vnn0GrEwCAAAQj8FwIY4P9Cjd+BAASygoAM8KZNm0wLwyLt9OnTVqpUKU8ZePnllz0/9uDBg1auXDnbvn27Va1aNXpshw4d7NChQ/bZZ5+Z/r5//3778ssvo/8/YcIE69WrV1wDXKNGDStRooRHmSNNprpFixYeQVZ+riLAvXv3dtMdaYoC9+/f/7JR4GQMsBbBjRo1ys6dO5cVejNJCEAgewhggLNHa2YKAQgkSUAGuEiRIjZy5Ej7559/bOvWrR7BbdWqlU2ePDm6OGzRokUepb3++usv6EHGsXDhwnbs2DGrWLGiNWzY0MaOHRs9RlHgKlWqxDXAqr4wYsQIX2wXaUqt0Pnmz5/vi9ZkgKdOnWpdunSJHiOD26lTJxs0aFDc2SZjgJVOob6IACf5weFwCEAg9AQwwKGXiAFCAAJBEYiXArF+/Xp76KGH3HTKBKspTUApETt27LAbb7zxguHmzZvXSpcu7Ua3bt26F0Rrr2SAVQ3irbfe8ghvpP3yyy9uyJUCIRMuA3zxArb/V0otGQNcs2ZNn49yl2kQgAAEchMBDHBuUpO5QAACOUrgcovgBgwYYKNHj7ZVq1ZZ48aNPfdXi95Wrlzp+b6RphSCPHny+EI2pUzs3bvXtmzZEv3/adOmWdeuXeNGgJXTW7JkyQtSIFR1QvWHY1MgUmWAVQVC6RyqgKH0DRoEIACB3EQAA5yb1GQuEIBAjhK4nAH+888/7b777rMzZ864+VXd3ubNm/vCtkmTJrkZXrp0qckov/vuu56SoEoOlStX9vq6KismE6tFdKr9+9NPP3lecWx0VmkVSnMYPHiw/6n8YS2C0zmUC6yWUxHgeHWAtYCvbNmytnbtWlMUmwYBCEAgNxHAAOcmNZkLBCCQowSuVAZNO8TVr1/fevbs6RtlnDp1yndOUzqEUhVU5uzFF1+0Z599NjomGVeVKFNFh7vuuita5eHo0aO+u9zF6Qnz5s3z3eb27dtnRYsW9UoTMsSR3edyygDHQlO5NaVsqC8t0IuUXMtRsJwMAhCAQMAEMMABC0D3EIBAdhBQhQhVjFAucKTNnTvXo8HafY3d1rLjc8AsIQCBcBDAAIdDB0YBAQjkcgIzZszw8mSzZs3yNAaVTtNCulq1asXdaS6X42B6EIAABAIlgAEOFD+dQwAC2ULg/PnzvhhOi8qOHDlixYoV853dYlMasoUF84QABCAQNAEMcNAK0D8EIAABCEAAAhCAQFoJYIDTipvOIAABCEAAAhCAAASCJoABDloB+ocABCAAAQhAAAIQSCsBDHBacdMZBCAAAQhAAAIQgEDQBDDAQStA/xCAAAQgAAEIQAACaSWAAU4rbjqDAAQgAAEIQAACEAiaAAY4aAXoHwIQgAAEIAABCEAgrQQwwGnFTWcQgAAEIAABCEAAAkETwAAHrQD9QwACEIAABCAAAQiklQAGOK246QwCEIAABCAAAQhAIGgCGOCgFaB/CEAAAhCAAAQgAIG0EsAApxU3nUEAAhCAAAQgAAEIBE0AAxy0AvQPAQhAAAIQgAAEIJBWAv8C/c4+crbONBMAAAAASUVORK5CYII=\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# number of probes histogram\n",
    "fig, ax = plt.subplots()\n",
    "ax.plot(cand_prob_nums, '.')\n",
    "ax.set_ylabel(\"Num probes per region\")\n",
    "ax.set_xlabel(\"Region ID\")\n",
    "ax.set_title(\"Probe number before filtering\")\n",
    "ax.set_ylim([0,400])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "id": "1e0f008c",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Screen probes by genome counts.\n",
      "-- filtering region:383, screen genome to keep 120 probes\n",
      "-- filtering region:384, screen genome to keep 120 probes\n",
      "-- filtering region:385, screen genome to keep 120 probes\n",
      "-- filtering region:386, directly append 100 probes\n",
      "-- filtering region:387, screen genome to keep 120 probes\n",
      "-- filtering region:388, directly append 70 probes\n",
      "-- filtering region:389, screen genome to keep 120 probes\n",
      "-- filtering region:390, screen genome to keep 120 probes\n",
      "-- filtering region:391, screen genome to keep 120 probes\n",
      "-- filtering region:392, screen genome to keep 120 probes\n",
      "-- filtering region:393, directly append 79 probes\n",
      "-- filtering region:394, screen genome to keep 120 probes\n",
      "-- filtering region:395, screen genome to keep 120 probes\n",
      "-- filtering region:396, screen genome to keep 120 probes\n",
      "-- filtering region:397, screen genome to keep 120 probes\n",
      "-- filtering region:398, directly append 117 probes\n",
      "-- filtering region:399, screen genome to keep 120 probes\n",
      "-- filtering region:400, screen genome to keep 120 probes\n",
      "-- filtering region:401, screen genome to keep 120 probes\n",
      "-- filtering region:402, screen genome to keep 120 probes\n",
      "-- filtering region:403, screen genome to keep 120 probes\n",
      "-- filtering region:404, screen genome to keep 120 probes\n",
      "-- filtering region:405, screen genome to keep 120 probes\n",
      "-- filtering region:406, screen genome to keep 120 probes\n",
      "-- filtering region:407, screen genome to keep 120 probes\n",
      "-- filtering region:408, directly append 120 probes\n",
      "-- filtering region:409, screen genome to keep 120 probes\n",
      "-- filtering region:410, screen genome to keep 120 probes\n",
      "-- filtering region:411, screen genome to keep 120 probes\n",
      "-- filtering region:412, screen genome to keep 120 probes\n",
      "-- filtering region:413, screen genome to keep 120 probes\n",
      "-- filtering region:414, screen genome to keep 120 probes\n",
      "-- filtering region:415, screen genome to keep 120 probes\n",
      "-- filtering region:416, directly append 95 probes\n",
      "-- filtering region:417, screen genome to keep 120 probes\n",
      "-- filtering region:418, screen genome to keep 120 probes\n",
      "-- filtering region:419, screen genome to keep 120 probes\n",
      "-- filtering region:420, screen genome to keep 120 probes\n",
      "-- filtering region:421, screen genome to keep 120 probes\n",
      "-- filtering region:422, directly append 103 probes\n",
      "-- filtering region:423, screen genome to keep 120 probes\n",
      "-- filtering region:424, screen genome to keep 120 probes\n",
      "-- filtering region:425, screen genome to keep 120 probes\n",
      "-- filtering region:426, screen genome to keep 120 probes\n",
      "-- filtering region:427, screen genome to keep 120 probes\n",
      "-- filtering region:428, screen genome to keep 120 probes\n",
      "-- filtering region:429, screen genome to keep 120 probes\n",
      "-- filtering region:430, screen genome to keep 120 probes\n",
      "-- filtering region:431, screen genome to keep 120 probes\n",
      "-- filtering region:989, screen genome to keep 120 probes\n",
      "-- filtering region:1010, screen genome to keep 120 probes\n",
      "-- filtering region:1397, screen genome to keep 120 probes\n",
      "-- filtering region:1398, screen genome to keep 120 probes\n",
      "-- filtering region:1399, screen genome to keep 120 probes\n",
      "-- filtering region:1400, screen genome to keep 120 probes\n",
      "-- filtering region:1401, screen genome to keep 120 probes\n",
      "-- filtering region:1402, screen genome to keep 120 probes\n",
      "-- filtering region:1403, screen genome to keep 120 probes\n",
      "-- filtering region:1404, screen genome to keep 120 probes\n",
      "-- filtering region:1405, screen genome to keep 120 probes\n",
      "-- filtering region:1406, screen genome to keep 120 probes\n",
      "-- filtering region:1407, screen genome to keep 120 probes\n",
      "-- filtering region:1408, screen genome to keep 120 probes\n",
      "-- filtering region:1409, screen genome to keep 120 probes\n",
      "-- filtering region:1410, screen genome to keep 120 probes\n",
      "-- filtering region:1411, screen genome to keep 120 probes\n",
      "-- filtering region:1412, screen genome to keep 120 probes\n",
      "-- filtering region:1413, screen genome to keep 120 probes\n",
      "-- filtering region:1414, screen genome to keep 120 probes\n",
      "-- filtering region:1415, screen genome to keep 120 probes\n",
      "-- filtering region:1416, screen genome to keep 120 probes\n",
      "-- filtering region:1417, screen genome to keep 120 probes\n",
      "-- filtering region:1418, screen genome to keep 120 probes\n",
      "-- filtering region:1419, screen genome to keep 120 probes\n",
      "-- filtering region:1420, screen genome to keep 120 probes\n",
      "-- filtering region:1421, screen genome to keep 120 probes\n",
      "-- filtering region:1422, screen genome to keep 120 probes\n",
      "-- filtering region:1423, screen genome to keep 120 probes\n",
      "-- filtering region:1424, screen genome to keep 120 probes\n",
      "-- filtering region:1425, screen genome to keep 120 probes\n",
      "-- filtering region:1426, screen genome to keep 120 probes\n",
      "-- filtering region:1427, screen genome to keep 120 probes\n",
      "-- filtering region:1428, screen genome to keep 120 probes\n",
      "-- filtering region:1429, screen genome to keep 120 probes\n",
      "-- filtering region:1430, screen genome to keep 120 probes\n",
      "-- filtering region:1431, screen genome to keep 120 probes\n",
      "-- filtering region:1432, screen genome to keep 120 probes\n",
      "-- filtering region:1433, screen genome to keep 120 probes\n",
      "-- filtering region:1434, screen genome to keep 120 probes\n",
      "-- filtering region:1435, screen genome to keep 120 probes\n",
      "-- filtering region:1436, screen genome to keep 120 probes\n",
      "-- filtering region:1437, screen genome to keep 120 probes\n",
      "-- filtering region:1438, screen genome to keep 120 probes\n",
      "-- filtering region:1439, screen genome to keep 120 probes\n",
      "-- filtering region:1440, screen genome to keep 120 probes\n",
      "-- filtering region:1441, screen genome to keep 120 probes\n"
     ]
    }
   ],
   "source": [
    "# filter by genome counts\n",
    "reload(library_tools.quality_check)\n",
    "\n",
    "num_probes_kept = 120 # max num of probes kept for each region\n",
    "filtered_pb_dict = library_tools.quality_check.Screen_probe_by_hit(library_folder, pb_dict, num_probes_kept,\n",
    "                                                                  hit_type='genome')\n",
    "\n",
    "## filter by minimum number of probes in each region\n",
    "min_num_probes = 21\n",
    "removed_regions = []\n",
    "for _rid, _pb_class in filtered_pb_dict.items():\n",
    "    if len(_pb_class.kept_probes) < min_num_probes:\n",
    "        print(f\"region: {_rid} removed with {len(_pb_class.kept_probes)} probes\")\n",
    "        removed_regions.append(_rid)\n",
    "for _rid in removed_regions:\n",
    "    del(filtered_pb_dict[_rid])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 99,
   "id": "7e4c6885",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(96, 96)"
      ]
     },
     "execution_count": 99,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(pb_dict), len(filtered_pb_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "id": "9a4e2d8b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7s3Qm8jnX+//GPLWStLCNL9iFbUUeEMipLjTEImWwxhWzZCmUnyxAz9mQZQoTEJKFINUgoy9jXymCskzWc/+Pz/f3ve85yn3Ou676v65z7uu/X9Xj0mMm5ru/1vZ/fz9H7fM/3+l7pYmNjY4UDAQQQQAABBBBAAIEoEUhHAI6SkeZjIoAAAggggAACCBgBAjCFgAACCCCAAAIIIBBVAgTgqBpuPiwCCCCAAAIIIIAAAZgaQAABBBBAAAEEEIgqAQJwVA03HxYBBBBAAAEEEECAAEwNIIAAAggggAACCESVAAE4qoabD4sAAggggAACCCBAAKYGEEAAAQQQQAABBKJKgAAcVcPNh0UAAQQQQAABBBAgAFMDCCCAAAIIIIAAAlElQACOquHmwyKAAAIIIIAAAggQgKkBBBBAAAEEEEAAgagSIABH1XDzYRFAAAEEEEAAAQQIwNQAAggggAACCCCAQFQJEICjarj5sAgggAACCCCAAAIEYGoAAQQQQAABBBBAIKoECMBRNdx8WAQQQAABBBBAAAECMDWAAAIIIIAAAgggEFUCBOCoGm4+LAIIIIAAAggggAABmBpAAAEEEEAAAQQQiCoBAnBUDTcfFgEEEEAAAQQQQIAATA0ggAACCCCAAAIIRJUAATiqhpsPiwACCCCAAAIIIEAApgYQQAABBBBAAAEEokqAABxVw82HRQABBBBAAAEEECAAUwMIIIAAAggggAACUSVAAI6q4ebDIoAAAggggAACCBCAqQEEEEAAAQQQQACBqBIgAEfVcPNhEUAAAQQQQAABBAjA1AACCCCAAAIIIIBAVAkQgKNquPmwCCCAAAIIIIAAAgRgagABBBBAAAEEEEAgqgQIwFE13HxYBBBAAAEEEEAAAQIwNYAAAggggAACCCAQVQIE4Kgabj4sAggggAACCCCAAAGYGkAAAQQQQAABBBCIKgECcFQNNx8WAQQQQAABBBBAgABMDSCAAAIIIIAAAghElQABOKqGmw+LAAIIIIAAAgggQACmBhCIAoG2bdvK3Llz433SdOnSyd133y2lSpWS1q1bS/fu3SV9+vSOaBw7dkyKFSsmr7/+uowaNcqRNr3QyJNPPin79u2Tf//732Hd3Zs3b0rXrl3lgw8+kFu3bsnbb79t/v3MmTOSJUsWyZkzp6v9//HHH0Vr8ptvvjH3W7t2rTzyyCPSvHlzWbRokbm31mfcf9c/c7p/GzZskNq1a8vUqVOlY8eOrn5mGkcAgfASIACH13jQGwRcEfAF4HfeeUfy5Mlj7hEbGyu//PKLLF++3AQQDUB//etfHbk/ATi8A/CkSZPMeDdr1kyeeeYZefzxx+Xo0aPSsmVL+ec//yllypRxpA6SaqRp06aydOlSGTBggBQpUsTc96OPPpKiRYtKjRo1Agbg1atXO96/06dPm9qvWrWq+UGQAwEEokeAABw9Y80njWIBXwDWkKMhI+5x+/ZtqV69umzbtk10Zq5AgQIhSxGAwzsAv/zyy/Luu++KBsB8+fKZ8R48eLAMGTJE/vWvf7kegEuXLi0ZM2aUvXv3JllrCWeAU7N/IX8D0AACCIS9AAE47IeIDiIQukByAVhbHz9+vPTq1UuWLVsmf/zjH0O+IQE4vAOwrx6uXbtmliCkdgDWH8J+85vfyObNmwnAIX+30QACCAQjQAAORo1rEPCYQEoBeMKECfLaa6+ZNaH6a3E9/6uvvpI33njDrOPVdaJTpkyRP/3pT3Lx4kUZOHCgWTqhM4iFChUyazXfeusts6ZYD18A1lCdKVMmmTVrllluUbNmTbMm+KGHHoonqL/eHjFihOzYsUMyZMhgfg0+fPhwqVy5crLSvn4uWbJEevfubQKVBro//OEP8pe//EXuvffeZMPdtGnTpFOnTvLFF1+Irt/19Vv7e/jwYdNv/bz6K/Lp06fLXXfdJd26dZPPP/9ccufObdZODxs2zPRZD98aYF1v3adPH9m/f78UL17cLDfo3LlzvM/y008/yZtvvimffPKJuYf+Cl778uqrr/rPmzNnjrRr184sF9Bx0Bn6l156SSZPnhzQRY1Hjx5tfpA5cuSIWdOts626vlut9NCZ1bjHAw88YPodd434E088Ibo+Vg9d06z91M+sgblcuXLSt29fUye+Q2dndbx0GcMrr7xiPo/Wg9ZP3MO35jbun7Vp00b0cyac8Y377wnXsDvVv8ceeyzeGmDf+M+ePVuOHz9uxl9r/Le//a0MGjRIGjduHO/zLFiwQMaMGWPGuWDBgqYG9Tcp69atM7XEgQAC4StAAA7fsaFnCDgmkFIAfvbZZ00QO3jwoJQsWdKEpcWLF0u2bNlMiNFA06FDB8mVK5dZL6r/wf/zn/8sFStWNKHz73//u/lzDUkaEn1BQq/XazQAaojWoH3jxg3ZunWrlC1b1nw+DT8a6jT06tpQDXEzZ86UU6dOyfr1683yjKQO7eeKFStMeGrUqJFooNHgPm/ePHn++efNZ9AjqV+fJxWANdTfc889JsydPHnShGkNQVeuXDGBqVq1avL+++/Ll19+KTNmzDAWvgCsa2i1PxrsKlSoYPqwadMmE5Q1SOrx888/y6OPPiq//vqrCb358+eXNWvWyMcff2wC9sSJE/02GoBz5Mhh+qIhS/tRv379gCTaty1btpgQret49WE87d+JEydMoNXgOH/+fPNn2icNePo59f5jx441P9RooNMfUJ5++mnZtWuXGRf9QUKXTeh4ahjXz62/NdAfmny+Q4cONW3pn+kPBE899ZT5jHEP35pbPUfrQselRIkSxjO5AKymbvTPN56+h+B8das/FOgSDX0wTj+Lflb9geWHH36Q8uXLm4+k1+gPNVpzuoZZr9W11dmzZzfjRQB27K8vGkLAFQECsCusNIpAeAn4AvD27dulcOHCpnO69lfDnYYg/Y95kyZN5MMPPzRf852vX9MA5jv69+9vdgzQEKWzwb5j3LhxZvZLA67ONsYNwLqm1HdPDVQarnSZhd7r8uXL5ms6A6lB1ndcunTJhMe8efPKd999lySmr586q6wzpL5Dg6AGYW1HZ6XtBmBdF3vo0CETZPTQMK39jfugoLat5z333HMmFOqhn2Pjxo2iHj179jR/psG/Vq1aovYaou677z7jq2Hz+++/j7cmW+30QUT9c/3hwjcDrDPNCXfxSIjy7bffSkxMjOiDjj169PB/WdtS87h9t7oEQj+PziRr8NMZbz3u3LljauXTTz81wVrHyOervxnQdcQpHYGWQCQXgLW9QGMYav8S7gLhq1v9gUB/yNOQroeOqd5L619/U6E/pPl+GPn666/Nbzn00Jl3tdEATQBOqQr4OgJpK0AATlt/7o5AqggE2gbNd2Od6dIZLJ298gU+3/kafnQ7M9/x4IMPyvXr183ygLi/StdttTQIPfzww2am0RckdNZQlw7EPXTXAd3+SgOk/spcZ3013DVo0CDeeRqmNJjrr/01bAQ6kprZ1vCpQVADmgZsuwH4xRdfNLPIvkNnbjX4+JZK+P5c29Z/9PPooSFJg+5//vMfMxPuO/RX5foDw8KFC83SAZ1R1VnPuPfQc/WHBQ3Lei8NW74AnPAHkaSKRk11CUjmzJnNKbrTh/4AoTOx+oOMtqOHlQB87tw5s2OIjqH2J+6hs9VqpLPgWjs+X/0NgP7wkdLhRAB2on9JBWDfsgzf59Af1DQM628q3nvvPfPDmv7GIeEPgnq+/mZDl4oQgFOqAr6OQNoKEIDT1p+7I5AqAr7Ao//B1tktPXR9qP5HXX9Vrr/ajnv4zteZrrhfy5o1qwl5umY34aHh98KFC+Y//L4AHPfX5L7z9dfGGmx1iYMG34TrRBO2q+FSw2Kgw9dP/VW2b/2xnqdt6lpY364XdgOwrl3WZQ++w3e97lrgW7qhX0sY5NRGDXTWNe6hSz50HbHOnmsQ1QfAkjt8Pzj4AvCqVatEl6mkdFy9elV0/aoGO13OorPYaqNH3FBnJQDrUgr99X5yx8iRI6Vfv37+ALx7926zRjilw4kA7ET/kgrAun5bl4L4Dv2hT2vfZ+j7jYeOa8JlHvrbDV3LTgBOqQr4OgJpK0AATlt/7o5AqgiktAY4YScCBSQ9R2cXdYYvUACuVKmSmfk6cOCAPwD7lkTEbV/XVeqs8NmzZ81WXDrTqQ916cNagY4qVaqYtaWBjqT6aTUA+/bDTfgQXMIXeCQVoAMFYJ2F1QAU99B10hridR2rzgTff//90rBhQ7MsIdChX9fZdl8AVu969eolWyu6TlvX62rwrVOnjuh46NIHDWi6ztZuANZ1t7r+WsO4LgEJdOgDfvqP3S3KnAjATvQvqQCccPwTBmBdcqPB37dUJa7NCy+8YPZSJgCnyl9t3ASBoAUIwEHTcSEC3hFwKgDr7KeGAV0akXAJhP66XMOWPrjmmwHWNa0aguMeGqB37twp58+fN2/90l+h69KAFi1aJAqN+qtnXRLg26rLalBPGID1ATRdUqH31WDoO3SnAt29wMkArOuc9Y1lvp0h9F46061joGtEf//735uZd90RQ9fRxj30hwJ9wEzfiqbrSO0EYH0ITXcq+Mc//hFvOYmOVTABWGfoNYjrw3f6sGDcQ2fWdV2wzhDrbxTSIgA70b9gA7CuB9cfCnT3EV3CE/fQsdMlMARg7/z9SE+jU4AAHJ3jzqeOMgGnArAvWCZc+6jLBfTXxr4HsHwBWH/Vr+tafQ9Q6YNauhRAH+rScKfrOHUNrc526sNEvrWrGo71ITBdw6rbUek65UCH1RlgnWnWmUxdHuDbDkzXLes2a3v27HE0AOsDU3EDvd5Hg6IuR9D1zPqaYV0HrA/O6cxw3F+h+5aH+EKsnQDse4Au4ctOdJZZZ7p15lnHTY9Abr4fEtRDx0MPfahOl33on2kg10PHRB/8011DfEseUiMAu9G/YAOw/mCm69J1XPWHGN8PO1rDOgvPQ3BR9hcsH9eTAgRgTw4bnUbAnoBTAVh/za6hSGcVNVDG3QZNZ750ay0Nsb4ArDse6AyhbvWls2I6G6xf1wezdKsxPXTXAw1v2larVq1M2NUZR11KoUFS9xhO6rAagH1BW++tQV1nYDUM6xpnfdrfyRlg/bW47vzQpUsXE5I0dOp6VV32oWZ6qJ/+IKBLRjT06tZz+gCZ7sOsa311OzRdo20nAH/22WdSt25dM8Pt25ZNZ5w15GlA063TfDttBHLTrdF0trd9+/ZmeYb+o/3WGXtdB6791LcE6u4Vei8dU90bWo/UCMBu9C/YAKyfWZft6Bjr9n9ao7q1nday7pKhP/jpDyIcCCAQvgIE4PAdG3qGgGMCTgVg7ZAGWf1Vu+7goP+/SJEiousedU2kPiikhy8A68ywzuBqCNQ9b3VvWf0zXTca99BQpX+uSxR09wTdAk3bS2q/W9+1VgOwnq9LC3QGW3dp0DXFOgutO1LofrVOBmAN1QMGDDD91xCk+/bqLhJxXxyh/dGv6bIMDZO6blhnDXWWVgO6z9FOANY29XxdZ6wBWz+j7lmr99Y/09lJ3RdYfQO5aR+0jzqDrTt/6My9HhroNeDqn+vyF11OoQFb9xr2zXymRgB2o3+hBGC10V019IE4nd3X7wNdz65BXR+E1BeIcCCAQPgKEIDDd2zoGQIIIIBAGAroy1x0x41AD2fqOnndElB/4OJAAIHwFSAAh+/Y0DMEEEAAgTAU8P2GI+GLP3S2XLcD1H2o426jF4YfgS4hEPUCBOCoLwEAEEAAAQTsCujuJLpGWtdC65IdfcBR97fWQ5fypLTXs937cT4CCDgrQAB21pPWEEAAAQSiQEAfCNUXgeiDhhp+9e1+uqZcd6vwvfo7Chj4iAh4ViAiA7BuoaQPI+jbk3S7Gn26XDcu1+1pfD+d61Pn27ZtE31KvVu3btK7d2//IOpTvPo++5kzZ5qHGfQ6fdpZn9TmQAABBBBAAAEEEPC2QEQGYP0pXDei170v9WEE3a5Gw6w+/a3/rq9+1fe46zot3YdTt/fRgKuvKNVDw6/+u26TpNsY9e3b1zxVrXth6hPUHAgggAACCCCAAALeFYi4AKzb0ZQqVcps+aOv8dRDN27X16zqVk26vZAGYn2Iwbe5vs4W66+xdNsa3bRe32il74HXV7bqob/q0jci6ZY3Cd9W5d2hp+cIIIAAAggggEB0CkRcANaw+s0330idOnX8b5XSodXlC/Xq1TMzufpWqgULFvhHfN26dWZ/0tOnT5tgrBvU6+b4Gpp9hy6D0KUUvo3fo7Nc+NQIIIAAAggggID3BSIuAAcaEn1fu27wrm9X0g3qdXP90aNH+0/VV32WK1dO9DWtJ06ckCZNmpg9Hn2b0euJer3+ma4rTurQ9cb6j+/QTeNPnjxpNpVP6lWu3i8hPgECCCCAAAIIhKOAvpXy7NmzZqeSLFmyhGMX06xPER+AdSmEzvzqjLC+uUpnglu2bClDhw71o+ussL7dSF/jqjPA+jrW27dvm1eR+g59a5S+6lJni5M6fG9DSrPR5MYIIIAAAggggEACga1bt8qjjz6KSxyBiA7A+t57DbuPPfaYmbnVGV39KahBgwYBZ4D1ITkNw02bNg04A6xv/9E2rc4A6+yvrkPWwitQoACFhwACCCCAAAIIpJrAqVOnJCYmxrx6vWjRoql2Xy/cKGIDsO4AoVudNW7cWObPn+9fD6zhV7c+mzdvnn98Aq0B1ofpdFbYd+ga4EqVKpkH6Kweujek7gepQbhQoUJWL+M8BBBAAAEEEEAgZAFySNKEERmA9W08urWZ7u/7zjvvxFvKoPsBT5s2TQ4fPiwZMmQwMv369ZPly5ebXSB0ljdfvnwyfvx4ad++vfl6sLtAUHghf+/SAAIIIIAAAggEKUAOiaIAfODAASlfvrw899xziXZs0CUQGnB1H+CGDRua/X11eYK+ylJDcZs2bYyUPig3ffp0s+2Z/sqgT58+Zm3wrl27bO0DTOEF+R3LZQgggAACCCAQsgA5JIoCsL6aUgNsoEMD7pw5c8xuDzo7vGPHDrM2t1evXtKlSxf/JfoAnO4NrC/CuHbtmug733Xpg931MxReyN+7NIAAAggggAACQQqQQ6IoAAdZI65cRuG5wkqjCCCAAAIIIGBBgBxCALZQJs6fQuE5b0qLCCCAAAIIIGBNgBxCALZWKQ6fReE5DEpzCCCAAAIIIGBZgBxCALZcLE6eSOE5qUlbCCCAAAIIIGBHgBxCALZTL46dS+E5RklDCCCAAAIIIGBTgBxCALZZMs6cTuE540grCCCAAAIIIGBfgBxCALZfNQ5cQeE5gEgTCCCAAAIIIBCUADmEABxU4YR6EYUXqiDXI4AAAggggECwAuQQAnCwtRPSdRReSHxcjAACCCCAAAIhCJBDCMAhlE/wl1J4wdtxJQIIIIAAAgiEJkAOIQCHVkFBXk3hBQnHZQgggAACCCAQsgA5hAAcchEF0wCFF4wa1yCAAAIIIICAEwLkEAKwE3Vkuw0KzzYZFyCAAAIIIICAQwLkEAKwQ6VkrxkKz54XZyOAAAIIIICAcwLkEAKwc9VkoyUKzwYWpyKAAAIIIICAowLkEAKwowVltTEKz6oU5yGAAAIIIICA0wLkEAKw0zVlqT0KzxITJyGAAAIIIICACwLkEAKwC2WVcpMUXspGnIEAAggggAAC7giQQwjA7lRWCq1SeGnCzk0RQAABBBBAQETIIQTgNPlGoPDShJ2bIoAAAggggAABONkaSBcbGxtLlbgjQAB2x5VWEUAAAQQQQCBlAXIIM8ApV4kLZ1B4LqDSJAIIIIAAAghYEiCHEIAtFYrTJ1F4TovSHgIIIIAAAghYFSCHEICt1oqj51F4jnLSGAIIIIAAAgjYECCHEIBtlItzp1J4zlnSEgIIIIAAAgjYEyCHEIDtVYxDZ1N4DkHSDAIIIIAAAgjYFiCHEIBtF40TF1B4TijSBgIIIIAAAggEI0AOIQAHUzchX0PhhUxIAwgggAACCCAQpAA5hAAcZOmEdhmFF5ofVyOAAAIIIIBA8ALkEAJw8NUTwpUUXgh4XIoAAggggAACIQmQQwjAIRVQsBdTeMHKcR0CCCCAAAIIhCpADiEAh1pDQV1P4QXFxkUIIIAAAggg4IAAOYQA7EAZ2W+CwrNvxhUIIIAAAggg4IwAOYQA7Ewl2WyFwrMJxukIIIAAAggg4JgAOYQA7Fgx2WmIwrOjxbkIIIAAAggg4KQAOYQA7GQ9WW6LwrNMxYkIIIAAAggg4LAAOYQA7HBJWWuOwrPmxFkIIIAAAggg4LwAOYQA7HxVWWiRwrOAxCkIIIAAAggg4IoAOYQA7EphpdQohZeSEF9HAAEEEEAAAbcEyCEEYLdqK9l2Kbw0YeemCCCAAAIIICAi5JAoDsDDhw+XdevWyYYNG4zCk08+KRs3bgwoMnfuXGndurUcP35cihYtmuicd999Vzp06GD5m4rCs0zFiQgggAACCCDgsAA5JEoD8IQJE6Rnz55Sq1YtfwA+f/683Lx5M56IhtrDhw/L1q1bJUeOHLJy5Upp1qyZHDlyRNKlS+c/N1euXJI1a1bL5UnhWabiRAQQQAABBBBwWIAcEmUB+KeffjIztZs2bZLChQtL/vz5/QE4IcXChQvNrO/27dulQoUK5ssjR46UxYsXy86dO0MqRQovJD4uRgABBBBAAIEQBMghURaAdQZXg60ufxg6dKgcO3YsYAC+cuWK/Pa3v5WWLVvKmDFj/EotWrSQ9OnTy4IFC0IoO9behITHxQgggAACCCAQkgABOMoCcNyP27Zt2yQD8Lhx40xAPnHihOjyBt9Rrlw5KVCggNy4cUMOHDggpUqVkrfeekvq1q2bbCFevnxZ9B/fcerUKYmJiZGTJ09KoUKFQipiLkYAAQQQQAABBOwIEIAJwIlmgG/fvi3Fixc3s79vv/22X0jXB2fLlk0efvhhGTt2rGTPnl3mz58vEydOlLVr10qdOnWS1Bw8eLAMGTIk0dcJwHa+XTkXAQQQQAABBJwQIAATgBMF4PXr18tTTz1lHn7TIBz30KURGTNmlMyZM/v/uF69euaBuNWrVyepyQywE9+utIEAAggggAACTggQgAnAiQJw165dza4PW7ZssVRjffr0kU8//VR27dpl6Xw9icKzTMWJCCCAAAIIIOCwADmEAJwoAOsSB53Vjbv8QZl++OEHqV69ugm7NWrU8MvVrl3b7CaxaNEiy+VJ4Vmm4kQEEEAAAQQQcFiAHEIAjheAdf2v7uc7Z84cswY47nHnzh0TgHUZxNSpUyVPnjwyY8YMmTJlinz77bf+rdKs1CiFZ0WJcxBAAAEEEEDADQFyCAE4XgA+c+aMmc3VWd5AOzucPXtW3njjDfnkk0/k4sWLUrlyZRk1apTUrFnTVn1SeLa4OBkBBBBAAAEEHBQgh0RxAHawjmw3ReHZJuMCBBBAAAEEEHBIgBxCAHaolOw1Q+HZ8+JsBBBAAAEEEHBOgBxCAHaummy0ROHZwOJUBBBAAAEEEHBUgBxCAHa0oKw2RuFZleI8BBBAAAEEEHBagBxCAHa6piy1R+FZYuIkBBBAAAEEEHBBgBxCAHahrFJuksJL2YgzEEAAAQQQQMAdAXIIAdidykqhVQovTdi5KQIIIIAAAgjwRtpkayBdbGxsLFXijgAB2B1XWkUAAQQQQACBlAXIIcwAp1wlLpxB4bmASpMIIIAAAgggYEmAHEIAtlQoTp9E4TktSnsIIIAAAgggYFWAHEIAtlorjp5H4TnKSWMIIIAAAgggYEOAHEIAtlEuzp1K4TlnSUsIIIAAAgggYE+AHEIAtlcxDp1N4TkESTMIIIAAAgggYFuAHEIAtl00TlxA4TmhSBsIIIAAAgggEIwAOYQAHEzdhHwNhRcyIQ0ggAACCCCAQJAC5BACcJClE9plFF5oflyNAAIIIIAAAsELkEMIwMFXTwhXUngh4HEpAggggAACCIQkQA4hAIdUQMFeTOEFK8d1CCCAAAIIIBCqADmEABxqDQV1PYUXFBsXIYAAAggggIADAuQQArADZWS/CQrPvhlXIIAAAggggIAzAuQQArAzlWSzFQrPJhinI4AAAggggIBjAuQQArBjxWSnIQrPjhbnIoAAAggggICTAuQQArCT9WS5LQrPMhUnIoAAAggggIDDAuQQArDDJWWtOQrPmhNnIYAAAggggIDzAuQQArDzVWWhRQrPAhKnIIAAAggggIArAuQQArArhZVSoxReSkJ8HQEEEEAAAQTcEiCHEIDdqq1k26Xw0oSdmyKAAAIIIICAiJBDCMBp8o1A4aUJOzdFAAEEEEAAAQJwsjWQLjY2NpYqcUeAAOyOK60igAACCCCAQMoC5BBmgFOuEhfOoPBcQKVJBBBAAAEEELAkQA4hAFsqFKdPovCcFqU9BBBAAAEEELAqQA4hAFutFUfPo/Ac5aQxBBBAAAEEELAhQA4hANsoF+dOpfCcs6QlBBBAAAEEELAnQA4hANurGIfOpvAcgqQZBBBAAAEEELAtQA4hANsuGicuoPCcUKQNBBBAAAEEEAhGgBxCAA6mbkK+hsILmZAGEEAAAQQQQCBIAXIIATjI0gntMgovND+uRgABBBBAAIHgBcghBODgqyeEKym8EPC4FAEEEEAAAQRCEiCHEIBDKqBgL6bwgpXjOgQQQAABBBAIVYAcEsUBePjw4bJu3TrZsGGDX6Fdu3YyZ86ceCoFCxYULRQ97ty5I0OGDJGZM2fKhQsXpEaNGjJlyhQpWbKkrVqk8GxxcTICCCCAAAIIOChADonSADxhwgTp2bOn1KpVK14ArlKlitSrV0+6du3ql8mQIYPkzZvX/LuGXw28s2fPFg3Gffv2lSNHjsiePXvkrrvuslyaFJ5lKk5EAAEEEEAAAYcFyCFRFoB/+ul7/oV6AAAgAElEQVQn6dChg2zatEkKFy4s+fPn9wfg27dvS/bs2WXhwoXSqFGjRDI3b96UPHnyyJgxY6Rjx47m6xcvXpT7779fZs2aJS1atLBcnhSeZSpORAABBBBAAAGHBcghURaAV65caQKuLn8YOnSoHDt2zB+A9+3bJ2XLlpX9+/dL6dKlE8ls3bpVqlatmujrugyiYsWKZmbY6kHhWZXiPAQQQAABBBBwWoAcEmUBOO7Hbdu2bbwAvGTJEmnevLmZ3V29erWkT59eGjRoYMJyrly5ZNmyZdKkSRO5evWqZM2a1d9Us2bNzJ+tWrXKcn1SeJapOBEBBBBAAAEEHBYghxCA/TPAgwYNkhEjRsjIkSNN8D148KD07t3bLJX4/PPPZcGCBdKqVSvRpRIajn1H69at5eeffzYP1CV1XL58WfQf33Hq1CmJiYmRkydPSqFChRwua5pDAAEEEEAAAQSSFiAAE4D9ATg2NlYuXbokuXPn9qts3rxZqlWrJvq/WixNmzYNOAN848YNWbFiRZKagwcPNg/QJTwIwPz1hAACCCCAAAKpLUAAJgDH2wUiIceVK1fMg3EffvihmQnWNcCHDh2SEiVK+E/VNcCVKlWSyZMnMwOc2t/B3A8BBBBAAAEEbAsQgAnA/gDcsmVLOXfunKxZs8avonsE165dW/bu3SvFixeXfPnyyfjx46V9+/bmHHaBsP09xwUIIIAAAgggkMYCBGACsD8A60NsDRs2lGHDhpktzQ4cOCCvvvqqWQLx/vvvG6kBAwbI9OnTzbZnRYsWlT59+pgH6Xbt2sU+wGn8zcztEUAAAQQQQMCaAAGYABxvCcTSpUvNg3C6JZquBdZZYd0FIkuWLEZKH4Dr37+/eRHGtWvXzIs0dOmDhmE7B4VnR4tzEUAAAQQQQMBJAXJIFAdgJwvJblsUnl0xzkcAAQQQQAABpwTIIQRgp2rJVjsUni0uTkYAAQQQQAABBwXIIQRgB8vJelMUnnUrzkQAAQQQQAABZwXIIQRgZyvKYmsUnkUoTkMAAQQQQAABxwXIIQRgx4vKSoMUnhUlzkEAAQQQQAABNwTIIQRgN+oqxTYpvBSJOAEBBBBAAAEEXBIghxCAXSqt5Jul8NKEnZsigAACCCCAgIiQQwjAafKNQOGlCTs3RQABBBBAAAECcLI1kC42NjaWKnFHgADsjiutIoAAAggggEDKAuQQZoBTrhIXzqDwXEClSQQQQAABBBCwJEAOIQBbKhSnT6LwnBalPQQQQAABBBCwKkAOIQBbrRVHz6PwHOWkMQQQQAABBBCwIUAOIQDbKBfnTqXwnLOkJQQQQAABBBCwJ0AOIQDbqxiHzqbwHIKkGQQQQAABBBCwLUAOIQDbLhonLqDwnFCkDQQQQAABBBAIRoAcQgAOpm5CvobCC5mQBhBAAAEEEEAgSAFyCAE4yNIJ7TIKLzQ/rkYAAQQQQACB4AXIIQTg4KsnhCspvBDwuBQBBBBAAAEEQhIghxCAQyqgYC+m8IKV4zoEEEAAAQQQCFWAHEIADrWGgrqewguKjYsQQAABBBBAwAEBcogHAvCdO3dk4cKF8tVXX8nNmzclNjY2Xq9nzZrlQCmkbhMUXup6czcEEEAAAQQQ+J8AOcQDAbhnz54yceJEqVSpkuTKlStRj7/44gvP1TSF57kho8MIIIAAAghEjAA5xAMBOG/evDJkyBDp3LkzhRcxAnwQBBBAAAEEEEgrAQKwBwJwjhw5ZOfOnVKiRIm0qhPH70vhOU5KgwgggAACCCBgUYAc4oEA3LRpU6ldu7a8+uqrFoc1/E+j8MJ/jOghAggggAACkSpADvFAAB4zZowMHjxY6tWrJ2XLlpXMmTPH6/XAgQM9V58UnueGjA4jgAACCCAQMQLkEA8E4GLFiiXZy3Tp0smRI0c8V5AUnueGjA4jgAACCCAQMQLkEA8E4IiptjgfhMKLxFHlMyGAAAIIIOANAXKIhwLwmjVr5IcffpBMmTJJuXLl5He/+51kyJDBG5WWoJcUnieHjU4jgAACCCAQEQLkEA8E4IsXL8ozzzwj27Ztk9y5c4u+GOPy5ctSpUoVWbt2rfkzrx0UntdGjP4igAACCCAQOQLkEA8E4A4dOsiWLVvM2+DKly9vevz999/Liy++KDVq1JCpU6d6riIpPM8NGR1GAAEEEEAgYgTIIR4IwPoijKVLl0qtWrXi9Xbjxo3SvHlz+fe//+25gqTwPDdkdBgBBBBAAIGIESCHeCAA6xKHzZs3S5kyZeL1dt++fVK5cmW5evWq5wqSwvPckNFhBBBAAAEEIkaAHOKBAFynTh2z9GHixInxetu1a1f57rvv5JtvvvFcQVJ4nhsyOowAAggggEDECJBDPBCA//nPf5o3wVWqVEkef/xx0b1/N23aZF6PvHr1atGA7LWDwvPaiNFfBBBAAAEEIkeAHOKBAKxd3Lp1q4wbN052794tsbGxUqFCBXnttdfkscce82Q1UnieHDY6jQACCCCAQEQIkEM8EoAjotrifAgKL9JGlM+DAAIIIICAdwTIIWEagF966SWz5jdHjhyi/z+5Y9asWd6puP/fUwrPc0NGhxFAAAEEEIgYAXJImAZgXfO7fPly85IL/f/JHV988YXnCpLC89yQ0WEEEEAAAQQiRoAcEqYBOGIqLIkPQuFF+gjz+RBAAAEEEAhfAXKIBwLwl19+GbCXuhvEXXfdJYUKFZKCBQvarrLhw4fLunXrZMOGDf5rV65cKUOHDpV//etfkidPHnn++efNv2fNmtWcc/z4cSlatGiie7377ruib6yzelB4VqU4DwEEEEAAAQScFiCHeCAAZ8qUSe7cuWN6qjtA6KHhN+7x5JNPmrfF6ZIJK8eECROkZ8+e5u1yvgCsW6vpcgsNxk2bNpVDhw7Jyy+/bLZZmz17tmlWA3KzZs3kyJEj8fqQK1cuf0i2cn8Kz4oS5yCAAAIIIICAGwLkEA8E4AULFkj//v1l0qRJUrNmTdNjfTNcly5dpFOnTmYrNA2zDz30kEybNi3ZOvnpp5/MTK2G3cKFC0v+/Pn9AfjFF1+UM2fOyGeffeZvY/78+dKuXTv55ZdfJHPmzDJy5EhZvHix2YM4lIPCC0WPaxFAAAEEEEAgFAFyiAcCcMmSJWXy5MlSt27deL1dv369dOzYUQ4ePCj6sozGjRvLqVOnkq0HncFduHChmeXVpQ3Hjh3zB+AdO3ZI+vTpzQs3fIcG4FatWsnZs2fNkogWLVqYczSUh3JQeKHocS0CCCCAAAIIhCJADvFAAM6WLZt55XGZMmXi9Xbfvn3y8MMPy7Vr1+TEiRPm61evXrVcD23bto0XgBNeePPmTalataoJvHp/PcqVKycFChSQGzduyIEDB6RUqVLy1ltvJQrnKXWCwktJiK8jgAACCCCAgFsC5BAPBOAnnnhCihcvLjNnzpQMGTKYHt++fdssZdCH1XQ5xLx582TUqFGyZ88ey7WSXAC+deuW/OlPf5KPPvpI9CE8DcIaiDWMa+geO3asZM+eXXSGWPcrXrt2bbKvZL58+bLoP75DZ6pjYmLk5MmT5iE+DgQQQAABBBBAILUECMAeCMA6+6oPot1zzz1SpUoV80Cc/tmlS5fk008/Nf+uD8H97W9/k1deecVy7SQVgP/73/+aB910f+ElS5bI73//e3+bV65ckYwZM5r1wL6jXr165oG41atXJ3nvwYMHy5AhQxJ9nQBsebg4EQEEEEAAAQQcEiAAeyAAaxd1xlQfcNN1uhpAdZ1u586dJW/evGYWWB9ue+qpp2yVRaAArPepX7++HD161Mz+pvQSDr1hnz59TBDftWtXkvdnBtjW0HAyAggggAACCLgoQAD2SAD2dVPX3urevwm3QQumRhIG4AsXLkj16tXNUgUNtBUqVIjX7A8//GC+rl+rUaOG/2saknU3iUWLFlnuBoVnmYoTEUAAAQQQQMBhAXKIRwKwzv6OGTPGPOymD5/pGlx9GG3gwIFBl0TCAKz/rjtEaMAtW7ZsvHZ1pllDtwZgXQYxdepUsyvEjBkzZMqUKfLtt98mCszJdYzCC3rYuBABBBBAAAEEQhQgh3ggAOuWY7rcoUePHiYE7969W1atWiWvv/66WVfbt2/foMogbgDWdcT6gNv169cDtqVLIvQNcLod2htvvCGffPKJXLx4USpXrmwevvPtT2y1IxSeVSnOQwABBBBAAAGnBcghHgjAGjK7d+8ubdq0kRw5csj3339vdoWYO3eu2c9X9wH22kHheW3E6C8CCCCAAAKRI0AO8UAA1plZnfUtVqxYvACss7IPPvig2QfYaweF57URo78IIIAAAghEjgA5xAMBuESJEuY1yLo7Q9wZYN2Dd9CgQXL48GHPVaTbhXfz1h1Zuv1HOXH+qjxw793SpEohiY0VS3+WKUP6gJ6B2gx0rtV767V2zk3YKTvX2jnXyn3s9N2qe6Dz3LiPG22G2nerdeRG3+3Ue7iNZajuVj+PG+70Pfi/n63+fcb4BmccjrWZ1N9ToYQft3NIKH1L62vTxcZqGaT9oet+9aGzcePGSevWrc3evPoK4zfffFN69uwpAwYMSPtO2uyBm4Wnfzm2fm+LbD563t+rmGL3mv+/NYU/e6z4fTKvfYwk/GYL1Gagc63eW699r80j0n7Otyn2M9zuY6fvVt0DnefGfdxoM9S+Wx1fN/pup97DbSxDdbf6edxwp+8p/12MUcpG0VKbSf09ZTN2JDrdzRwSat/S+vqwCcAK0b9/f5kwYYL/ITXdC7hjx47mz/RVxV473Cy8hVtPSL9lSe9JnJLVqMYVpEVMkXinJdVmwnPt3LvxwwVl2Y6fUuqO+Xq43cdO3y19wCROcuM+brQZqPt27mN1fO20adXdTr1bbTNUj3C7jxvuqWXk5b5jlPJ3gpfH107fA/09lbJO8me4mUNC7VtaXx82AXjjxo1m+7Fff/1V9u7da978VqZMGcmZM2daGwV9fzcLb/Sn+2TqhuCXhXR6soS8Xq9MvM+WVJsJz7Vz74eL5JYdJy5aMgy3+9jpu6UPmMRJbtzHjTYDdd/OfayOr502rbrbqXerbYbqEW73ccM9tYy83HeMUv5O8PL42ul7oL+nUtYhAAdrFDYBWPfgXbNmjdlyLFIONwOwnVnYQJ52ZsSsztyFOpMRbvex85N7KDXrxn3caDO1xteNvtup93Aby1DdrX4eN9zpu1X9+Oe5MRZutMn4pjy+dtyZAU7Z08kzwiYAlytXzjwEZ+W1xE4CuNmWmwHY6jrcpNaYsQb4f0tqklr7bHX9stV1ltG83o81wCmvdbRaR26siXSjzWiud6tjiVHK3xfRUpusAXYzjQVuO2wCsD7opm+Ca9CggZQqVUqyZs0ar8ehvA0u9Vn/745uBmBt/9fbd2Tpdz/K8Ti7QOifW/mzpJ42DdRmoHOt3luvtXNuwrGyc63VcwM9/RvIzU7frbqn1n3Cse9W68iNvtup93Aby2iuGatj4UbNpJZ7at3Hy0bR0nd2gUjdtBY2AVj3/03q0NcTHzlyJHVlHLib2wHYgS5GXRNWd7qIOhg+MAIIIIBAxAmQQ5Ie0rAJwBFXdakwAxyJZm5/Jqs7XbjdD9pHAAEEEEDAbQECMAHY7RoL2D6Flybsyd7U6k4X4ddzeoQAAggggIA9AXIIAdhexTh0NoXnEKSDzTAD7CAmTSGAAAIIhLUAOYQAnCYFSuGlCXuyN2UNcPiNCT1CAAEEEHBHgBzigQCsb2TWh90i6aDwwnM0re50EZ69p1cIIIAAAghYEyCHeCAAP/LIIzJr1iypWLGitVH1wFkUngcGiS4igAACCCAQoQLkEA8E4Dx58siWLVukRIkSEVOGFF7EDCUfBAEEEEAAAc8JkEM8EIDHjh0r//jHP6RPnz5SsmTJRC/CKFKkCIXnOQE6jAACCCCAAAJpJUAA9kAATp/+f6+mjbsW2Lc2+Pbt22lVP0Hfl8ILmo4LEUAAAQQQQCBEAXKIBwLwxo0bkx3mJ554IsQySP3LKbzUN+eOCCCAAAIIIPB/AuQQDwTguF28ceOGZM6c2fP1S+F5fgj5AAgggAACCHhWgBzikQA8bdo0GT16tJw8eVIOHDggui64QIECMnDgQE8WH4XnyWGj0wgggAACCESEADnEAwF4wYIF0rlzZ+nRo4eMGTNGdu/eLatWrZLXX39dhgwZIn379vVcMVJ4nhsyOowAAggggEDECJBDPBCAK1euLN27d5c2bdpIjhw55Pvvv5fixYvL3LlzZfjw4XLw4EHPFSSF57kho8MIIIAAAghEjAA5xAMBOFu2bGbWt1ixYvEC8NGjR+XBBx+Ua9euea4gKTzPDRkdRgABBBBAIGIEyCEeCMD6AoxJkyZJ/fr14wXg+fPny6BBg+Tw4cOeK0gKz3NDRocRQAABBBCIGAFyiAcCsK77nTp1qowbN05at24tS5YskWPHjsmbb74pPXv2lAEDBniuICk8zw0ZHUYAAQQQQCBiBMghHgjA2sX+/fvLhAkT5Pr166bHGTNmlI4dO5o/i/uiDK9UJoXnlZGinwgggAACCESeADnEIwFYu3n16lXZu3ev3LlzR8qUKSM5c+b0bEVSeJ4dOjqOAAIIIICA5wXIIR4JwP/9739l0aJFsmvXLsmQIYNUqVJFmjZtKlmyZPFkEVJ4nhw2Oo0AAggggEBECJBDPBCA9+3bJ7Vr1xYNwaVLlzYzwLr1WcGCBeXzzz+XQoUKea4YKTzPDRkdRgABBBBAIGIEyCEeCMBPP/20ZMqUSd5//3255557TI//85//SMuWLSV79uyybNkyzxUkhee5IaPDCCCAAAIIRIwAOcQDAVhD7pYtW6RcuXLxeqsvxKhZs6ZcvnzZcwVJ4XluyOgwAggggAACESNADvFAANZlD3/729+kbt268Xq7adMm+dOf/iQnTpzwXEFSeJ4bMjqMAAIIIIBAxAiQQzwQgFeuXCk9evSQ8ePHyxNPPGGWQ3z77bfSuXNn6dChg3kYzncUKVLEE8VJ4XlimOgkAggggAACESlADvFAAI67z2+6dOn8PY6NjTX/X/9M/7/+7+3btz1RqBSeJ4aJTiKAAAIIIBCRAuQQDwTgjRs3Wi4+nSH2wkHheWGU6CMCCCCAAAKRKUAO8UAAjsTSo/AicVT5TF4UuHnrjizd/qOcOH9VHrj3bmlSpZBkypDeix+FPiOAAAKWBcghBGDLxeLkiRSek5q0hUBwAhp+W7+3RTYfPe9v4LHi98m89jGE4OBIuQoBBDwiQA4hAKdJqVJ4acLOTRGIJ7Bw6wnpt2xXIpVRjStIixhvPFDLkCKAAALBCJBDojgADx8+XNatWycbNmzwK+zcuVO6d+8u27Ztk/vuu0+6desmvXv39n9d30I3ZMgQmTlzply4cEFq1KghU6ZMkZIlS9qqPwrPFhcnI+CKwOhP98nUDYcTtd3pyRLyer0yrtyTRhFAAIFwECCHRGkAnjBhgvTs2VNq1arlD8Dnzp2TMmXKSKNGjczXNm/ebLZa04Dbrl07I6XhV/999uzZ5lXMffv2lSNHjsiePXvkrrvuslzTFJ5lKk5EwDUBZoBdo6VhBBAIcwFyiEcC8IIFC0xYLVSokOjM7aJFi+Txxx+XiRMnSpYsWSyX2U8//WT2DtaXaBQuXFjy58/vD8Bvv/22TJ48WY4dOyYZM2Y0bfbv39+8annfvn1y8+ZNyZMnj4wZM0Y6duxovn7x4kW5//77ZdasWdKiRQvL/aDwLFNxIgKuCbAG2DVaGkYAgTAXIId4IABr4B0xYoSsX7/e7PWrwVdDrC5daNCggehsrtVDX6qxcOFCE6KHDh1qwq5vCYS2lTt3btGw7Tt0icTTTz8tp0+fNudWrVpV9u/fL/p2Ot+hyyAqVqxoZoatHuFceDwVb3UUOS8SBH69fUeWfvejHGcXiEgYTj4DAghYFAjnHGLxI7h2WrpY35smXLuFtYaLFy8uo0ePlueff1769OkjX3/9tXzzzTdmFldnXXVWN5ijbdu28QKwhtj69eube/mOvXv3Srly5cyb5/SVy02aNJGrV69K1qxZ/ec0a9bM/NmqVassdyNcC48ZMctDyIkIIIAAAgh4ViBcc0g4gIZNANYlDocOHTLLHx566CH54x//KIMGDZLjx49L2bJlTfgM5kgYgPVBtpYtW5qZYd+h63tLlChhwrbOALdq1cq8bS7u2+lat24tP//8s3mgLqnj8uXLov/4jlOnTklMTIycPHnSfK5wOVgTGS4jQT8QQAABBBBwT4AAnLRt2ARgDaYzZsyQBx54QEqVKmVmgKtVqybz58+XYcOGmSUJwRwJA3CFChXMkopAM8Dbt283D7s1bdo04AzwjRs3ZMWKFUl2Y/DgweYBuoRHuAVgnooPppK4BgEEEEAAAW8JEIA9EID/8pe/yMiRI82yg7x584puVabrbXU5hIbKuNuU2Sm/hAFYw69ufTZv3jx/M4HWAOtstM4K+w5dA1ypUiXzAF1SBzPAdkaGcxFAAAEEEEDATQECsAcCsHZRH17TGdgXX3zRhNT3339frl27Zh6GC/ZIGIBHjRol06ZNk8OHD0uGDBlMs/369ZPly5ebXSB0ljdfvnwyfvx4ad++vfl6pO0CwRrgYKuJ6xBAAAEEEPCOAAHYIwHY100NoZkzZ3akwhIG4DNnzph9gBs2bGj29926dat06tTJhOI2bdqYew4YMECmT59utj0rWrSomYXWtcG7du2KmH2AeSrekfKiEQQQQAABBMJWgADskQCsIVTX5uqa2QMHDsjYsWOlQIECMnDgwKCLK2EA1oZ0twd9+9uOHTtM+7169ZIuXbr476EPwOnewPoiDJ2B1r2JdemDhmE7B4VnR4tzEUAAAQQQQMBJAXKIBwKw7surb2Tr0aOHeQnF7t27zZZjr7/+ulkDrLO1XjsoPK+NGP1FAAEEEEAgcgTIIR4IwJUrV5bu3bubZQg5cuSQ77//XnRv4Llz55oXWhw8eNBzFUnheW7I6DACCCCAAAIRI0AO8UAAzpYtm5n1LVasWLwAfPToUXnwwQfNUgSvHRSe10aM/ropwNsH3dSlbQQQQCCxADnEAwFYtxybNGmSeUtb3Blg3QdYX4ihuzZ47aDwvDZi9NctAXYecUuWdhFAAIGkBcghHgjAuu536tSpMm7cONG3ri1ZssTsvPDmm29Kz549zc4MXjsoPK+NGP11S4C3D7olS7sIIIAAATiYGgibN8Fp53XnhQkTJsj169fNZ8mYMaN07NjR/Fnc1xIH80HT4hoCcFqoc89wFODtg+E4KvQJAQQiXYAc4oEZYF8Xr169Knv37pU7d+6Y/Xpz5szp2fqk8Dw7dHTcYQFmgB0GpTkEEEDAggA5xCMBWGd+dTs0fRhOX4RRoUIFadasmZkJ9uJB4Xlx1OizGwKsAXZDlTYRQACB5AXIIR4IwLrbQ40aNeTy5ctSunRp0ZdR6MswHnjgAVm9erXtl1CEwzcFhRcOo0AfwkWAtw+Gy0jQDwQQiBYBcogHAvAzzzwjWbJkkXnz5kmuXLlMj8+ePSstWrSQu+++W1auXOm5eqXwPDdkdBgBBBBAAIGIESCHeCAAa8jdtm2b2fM37rFz5055/PHH5cqVK54rSArPc0NGhxFAAAEEEIgYAXKIBwKwLnuYMmWKPPXUU/F6+8UXX0jbtm3l+PHjnitICs9zQ0aHEUAAAQQQiBgBckiYBuATJ074e/bxxx/LX/7yF/MyDF0LnCFDBvnuu+/k5ZdflrfeektatWrluYKk8Dw3ZHQYAQQQQACBiBEgh4RpANa9fdOlS+fvXWxsrPn/Cf9M/10fivPaQeF5bcToLwIIIIAAApEjQA4J0wC8ceNGy1X2xBNPWD43XE6k8MJlJOgHAggggAAC0SdADgnTAJxUt3T3h0yZMknu3Lk9Xa0UnqeHj84jgAACCCDgaQFyiEcC8OTJk2XEiBFy+vRp0+Pf/OY30qdPH+nRo4cnC5DC8+Sw0WkEEEAAAQQiQoAc4oEAPHPmTOnSpYv5p1atWmbN75dffinTpk0TDcYvvfSS54qRwvPckNFhBBBAAAEEIkaAHOKBAFymTBnp3r27dOrUKV5vNfxqCN61a5fnCpLC89yQ0WEEEEAAAQQiRoAc4oEAnDVrVtm9e7eUKFEiXm8PHz4s5cuXl2vXrnmuICk8zw0ZHUYAAQQQQCBiBMghHgjAOgM8ZMgQad68ebzeLly4UPr16yfHjh3zXEFSeJ4bMjqMAAIIIIBAxAiQQzwQgN955x3zANywYcPMq491799NmzbJwIEDpWvXrjJo0CDPFSSF57kho8MIIIAAAghEjAA5xAMB+M6dO9KzZ0+ZOnWq3Lp1y/RY3wanb4L761//KvrSDK8dFJ7XRoz+IoAAAgggEDkC5BAPBGB9KUb16tXl+vXrsm/fPtG3wumyiJw5c3q2Eik8zw4dHUcAAQQQQMDzAuQQDwTgvHnzypo1a6Ry5cqeLzjfB6DwImYo+SAIIIAAAgh4ToAc4oEAXK5cOZk0aZLUrl3bcwWWVIcpvIgZSj4IAggggAACnhMgh3ggAOv6X93vt0GDBlKqVCnRbdHiHvownNcOCs9rI0Z/EUAAAQQQiBwBcogHAnCxYsWS7KXuCHHkyBHPVSSF57kho8MIIIAAAghEjAA5xAMBOGKqLc4HofAicVT5TAgggAACCHhDgBzikQCsW6GtX7/evPZYt0CrUqWK1KhRwxtVFqCXFJ5nh46OI4AAAggg4HkBcogHAvCpU6ekbt265nXIuXPnNtugXbp0ybwUY9WqVZIrVy7PFSKF57kho8MIIIAAAghEjAA5xAMBuHHjxnLy5EmZP3++/Pa3vzU93rNnj7Ru3VoqVqwos2fP9lxBUnieGzI6jAACCCCAQMQIkA4/umoAACAASURBVEM8EID1hRdffPGFWfYQ99iyZYvUr19fzp8/77mCpPA8N2R0GAEEEEAAgYgRIId4IAAXKVJEPvjgA6lWrVq83n733Xdma7TTp097riApPM8NGR1GAAEEEEAgYgTIIR4IwHPmzJFx48aZJRCVKlUyPT569Ki0atVKWrZsKZ07d/ZcQVJ49obs5q07snT7j3Li/FV54N67pUmVQpIpQ3p7jXA2AggggAACCBgBcogHAnDhwoXlzJkzcuvWLcmRI4dkypTJLHvQh+F0H+C4x+3btz1R2hSe9WHS8Nv6vS2y+ej/lro8Vvw+mdc+hhBsnZEzEUAAAQQQ8AuQQzwQgOfOnWu5ZNu0aWP53LQ8kcKzrr9w6wnpt2xXogtGNa4gLWKKWG+IMxFAAAEEEECAGeAUaiBdrE6xcrgiQAC2zjr6030ydcPhRBd0erKEvF6vjPWGOBMBBBBAAAEECMAE4LT7LiAAW7dnBti6FWcigAACCCBgRYAckrQSM8BWKijIcyg863CsAbZuxZkIIIAAAghYESCHEIDjCWzYsEFq164dUKVYsWJy5MgRadeunejOFHGPggULmicqrR4UnlWp/zvv19t3ZOl3P8pxdoGwB8fZCCCAAAIIBBAghxCA4wncvHkz0Ys1du3aZfYbnjp1qnTo0MG8kKNevXrStWtX/7UZMmSQvHnzWv4mo/AsU3EiAggggAACCDgsQA7xSADeu3ev7N69W65fv56ox/pKZLeOX3/9VR5++GEpX768LFq0SHSbtezZs8vChQulUaNGQd+WwguajgsRQAABBBBAIEQBcogHAvCoUaOkf//+AXuq+wC7uffvhAkTZNCgQbJv3z4pUKCA+d+yZcvK/v37pXTp0kGXH4UXNB0XIoAAAggggECIAuQQDwTg3/zmN9KxY0fp16+fZM6cOcQht365zjbrSzj0TXNDhgwxFy5ZskSaN29u+rN69WpJnz69WR4xfPhwyZUrV5KNX758WfQf33Hq1CmJiYmRkydPSqFChax3ijMRQAABBBBAAIEQBQjAHgjA9957r2zdulVKliwZ4nDbu/zdd9+V1157TY4fPy733XefuVhng0eMGCEjR440wffgwYPSu3dvE5Q///xzE4gDHYMHD/aH6LhfJwDbGxPORgABBBBAAIHQBQjAHgjAPXr0MDO/o0ePDn3EbbRQs2ZNKV68uMR9E52+G+TSpUuSO3duf0ubN2+WatWqif5v1apVA96BGWAb8JyKAAIIIIAAAq4KEIA9EIB//vlnqVixomTLlk10K7KEs6w68+r0cfbsWdGlF6tWrZL69esn2/yVK1fMg3EffvihNGnSxFJXKDxLTJyEAAIIIIAAAi4IkEM8EIB1qcGWLVvkqaeekrvvvjtRj2fPnu14aaxYscKEWZ25jXvPli1byrlz52TNmjX+e/r2DtadKvQBOSsHhWdFiXMQQAABBBBAwA0BcogHArAG0PXr15tlBql1DB06VObPny8HDhyId0udEW7YsKEMGzZMWrRoYb7+6quvmr69//77lrtH4Vmm4kQEEEAAAQQQcFiAHOKBAKzrcD/++GOzF29qHbrzw44dO+Sf//xnolsuXbrUPAinW6LpWmCdFdZdILJkyWK5exSeZSpORAABBBBAAAGHBcghHgjAixcvlmnTpsnkyZPN3rv61jWvHxSe10eQ/iOAAAIIIOBdAXKIBwJwqVKl5MSJE3Lr1q2AvXXzRRhulTaF55Ys7SKAAAIIIIBASgLkEA8E4LjbkAXqbps2bVIa57D7eqQU3s1bd2Tp9h/lxPmr8sC9d0uTKoUkU4bAeyGH3SDQIQQQQAABBKJUIFJyiBvDly5WN73lcEUgEgpPw2/r97bI5qPn/UaPFb9P5rWPIQS7UjU0igACCCCAgDMCkZBDnJFI3ErYBGDdkSG5Y+DAgW4ZuNZuJBTewq0npN+yXYmMRjWuIC1iirhmR8MIIIAAAgggEJpAJOSQ0ASSvjpsArC+/CLu8euvv8qZM2fkrrvukurVq8tnn33mloFr7UZC4Y3+dJ9M3XA4kVGnJ0vI6/XKuGZHwwgggAACCCAQmkAk5JDQBDwQgAN1UV9QoWt/a9euLd26dXPLwLV2I6HwmAF2rTxoGAEEEEAAAVcFIiGHuAUUNjPASX3A3bt3y7PPPivHjx93y8C1diOh8FgD7Fp50DACCCCAAAKuCkRCDnELKOwDsL6o4vHHH5erV6+6ZeBau5FSeL/eviNLv/tRjrMLhGu1QsMIIIAAAgg4LRApOcRpF20vbALw3//+93ifTzenuHTpkkyfPl0KFCgg69atc+Pzu9omhecqL40jgAACCCCAQDIC5JCkccImAKdPn3hf2UyZMpnZX31DnL4dzmsHhee1EaO/CCCAAAIIRI4AOcQDAThyyu1/n4TCi8RR5TMhgAACCCDgDQFyCAE4TSqVwksTdm6KAAIIIIAAAiJCDgnTAPy73/3OUoGmS5dO1q9fb+nccDqJwgun0aAvCCCAAAIIRJcAOSRMA3C7du2SrcRNmzbJkSNHJGfOnHLx4kXPVS2F57kho8MIIIAAAghEjAA5JEwDcFLd+u9//ys9e/aU9957T5566imZOXOmFCnivdfuUngR83cIHwQBBBBAAAHPCZBDPBSA165dK3/+85/NjO+YMWPk5Zdf9lzB+TpM4Xl26Og4AggggAACnhcgh3ggAP/yyy9m1ldne+vUqWNmf7046xuXmsLz/N8dfAAEEEAAAQQ8K0AOCfMA7Jv1vXDhgpn1feWVVzxbbATgiBg6PgQCCCCAAAKeFyAAh2kA1lnfXr16xZv1LVy4sOcLjiUQETOEfBAEEEAAAQQ8K0AADtMAXLRoUTl58qQUL15cXnzxRdHtzpI6Bg4c6LkCpPA8N2R0GAEEEEAAgYgRIIeEcQBOLvT6uq3n6HZoXjsoPK+NGP1FAAEEEEAgcgTIIWEagCOnxAJ/Egov0keYz4cAAggggED4CpBDCMBpUp0UXpqwc1MEEEAAAQQQ4FXIydZAutjY2FiqxB0BArA7rrSKAAIIIIAAAikLkEOYAU65Slw4g8JzAZUmEUAAAQQQQMCSADmEAGypUJw+icJzWpT2EEAAAQQQQMCqADmEAGy1Vhw9j8JzlJPGEEAAAQQQQMCGADmEAGyjXJw7lcJzzpKWEEAAAQQQQMCeADmEAGyvYhw6m8JzCJJmEEAAAQQQQMC2ADmEAGy7aJy4gMJzQpE2EEAAAQQQQCAYAXIIATiYugn5GgovZEIaQAABBBBAAIEgBcghBOAgSye0yyi80Py4GgEEEEAAAQSCFyCHEICDr54QrqTwQsDjUgQQQAABBBAISYAcQgAOqYCCvZjCC1aO6xBAAAEEEEAgVAFyCAE41BoK6noKLyg2LkIAAQQQQAABBwTIIQRgB8rIfhMUnn0zrkAAAQQQQAABZwTIIQRgZyrJZisUnk0wTkcAAQQQQAABxwTIIQRgx4rJTkMUnh0tzkUAAQQQQAABJwXIIQRgJ+vJclsUnmUqTkQAAQQQQAABhwXIIQTgRALHjx+XokWLJvrzd999Vzp06CA7d+6U7t27y7Zt2+S+++6Tbt26Se/evW2VJoVni4uTEUAAAQQQQMBBAXIIATiRwMqVK6VZs2Zy5MgRSZcunf/ruXLlkqtXr0qZMmWkUaNG0rNnT9m8ebN07txZpkyZIu3atbNcmhSeZSpORAABBBBAAAGHBcghBOBEAiNHjpTFixebmd6Ex9tvvy2TJ0+WY8eOScaMGc2X+/fvL8uWLZN9+/ZZLk8KzzIVJyKAAAIIIICAwwLkEAJwIoEWLVpI+vTpZcGCBYm+1qBBA8mdO3e8r61bt06efvppOX36tOTLl89SiVJ4lpg4CQEEEEAAAQRcECCHEIATCZQrV04KFCggN27ckAMHDkipUqXkrbfekrp160rFihWlfv36Mnr0aP91e/fuFb3m22+/lUceeSSg6OXLl0X/8R2nTp2SmJgYOXnypBQqVMiF0qZJBBBAAAEEEEAgsAABmAAcT+DmzZuSLVs2efjhh2Xs2LGSPXt2mT9/vkycOFHWrl0rr7zyirRs2VKGDh3qv07XCpcoUUI2bdokNWrUCCg6ePBgGTJkSKKvEYD5qwkBBBBAAAEEUluAAEwATiRw5coVs743c+bM/q/Vq1fPPBCnBaPLIALNAG/fvt0E50AHM8Cp/a3N/RBAAAEEEEAgKQECMAHY0ndHnz595NNPP5XChQubrc/mzZvnv441wJYIOQkBBBBAAAEEwkSAAEwAjifwww8/SPXq1U3YjbucoXbt2pI/f3556KGHZNq0aXL48GHJkCGDubZfv36yfPlydoEIk29quoEAAggggAACyQsQgAnA8QTu3LljArAug5g6darkyZNHZsyYYfb51YfcNATrPsANGzaUvn37ytatW6VTp04mFLdp08by9xuFZ5mKExFAAAEEEEDAYQFyCAE4kcDZs2fljTfekE8++UQuXrwolStXllGjRknNmjXNuRqE9e1vO3bsMLtF9OrVS7p06WKrNCk8W1ycjAACCCCAAAIOCpBDCMAOlpP1pig861aciQACCCCAAALOCpBDCMDOVpTF1ig8i1CchgACCCCAAAKOC5BDCMCOF5WVBik8K0qcgwACCCCAAAJuCJBDCMBu1FWKbVJ4KRJxAgIIIIAAAgi4JEAOIQC7VFrJN0vhpQk7N0UAAQQQQAABEfNiL323AW+kTVwO6WJjY2OpEncEKDx3XGkVAQQQQAABBFIWIIcwA5xylbhwBoXnAipNIoAAAggggIAlAXIIAdhSoTh9EoXntCjtIYAAAggggIBVAXIIAdhqrTh6HoXnKCeNIYAAAggggIANAXIIAdhGuTh3KoXnnCUtIYAAAggggIA9AXIIAdhexTh0NoXnECTNIIAAAggggIBtAXIIAdh20ThxAYXnhCJtIIAAAggggEAwAuQQAnAwdRPyNRReyIQ0gAACCCCAAAJBCpBDCMBBlk5ol1F4oflxNQIIIIAAAggEL0AOIQAHXz0hXEnhhYDHpQgggAACCCAQkgA5hAAcUgEFezGFF6wc1yGAAAIIIIBAqALkEAJwqDUU1PUUXlBsXIQAAggggAACDgiQQwjADpSR/SYoPPtmXIEAAggggAACzgiQQwjAzlSSzVYoPJtgnI4AAggggAACjgmQQwjAjhWTnYYoPDtanIsAAggggAACTgqQQwjATtaT5bYoPMtUnIgAAggggAACDguQQwjADpeUteYoPGtOnIUAAggggAACzguQQwjAzleVhRYpPAtInIIAAggggAACrgiQQwjArhRWSo1SeCkJ8XUEEEAAAQQQcEuAHEIAdqu2km2XwksTdm6KAAIIIIAAAiJCDiEAp8k3AoWXJuzcFAEEEEAAAQQIwMnWQLrY2NhYqsQdAQKwO660igACCCCAAAIpC5BDmAFOuUpcOIPCcwGVJhFAAAEEEEDAkgA5hABsqVCcPonCc1qU9hBAAAEEEEDAqgA5hABstVYcPY/Cc5STxhBAAAEEEEDAhgA5hABso1ycO5XCc86SlhBAAAEEEEDAngA5hABsr2IcOpvCcwiSZhBAAAEEEEDAtgA5hABsu2icuIDCc0KRNhBAAAEEEEAgGAFyCAE4mLoJ+RoKL2RCGkAAAQQQQACBIAXIIQTgIEsntMsovND8uBoBBBBAAAEEghcghxCAg6+eEK6k8ELA41IEEEAAAQQQCEmAHEIADqmAgr2YwgtWjusQQAABBBBAIFQBcggBONQaCup6Ci8oNi5CAAEEEEAAAQcEyCEEYAfKyH4TFJ59M65AAAEEEEAAAWcEyCEEYGcqyWYrFJ5NME5HAAEEEEAAAccEyCEE4EQC58+fl/79+8uqVavk8uXLUrFiRRk1apTUqFHDnNuuXTuZM2dOvOsKFiwoWkxWDwrPqhTnIYAAAggggIDTAuQQAnAigWeeeUbOnDkjkyZNkrx588rkyZNl5syZsn37dilTpoxUqVJF6tWrJ127dvVfmyFDBnOu1YPCsyrFeQgggAACCCDgtAA5hAAcT+DQoUNSqlQp+frrr6V69erma7GxsVK6dGl54YUXZNCgQZI9e3ZZuHChNGrUKOh6pPCCpuNCBBBAAAEEEAhRgBxCAI4ncPHiRfnmm2+kTp06kjlzZv/XSpYsaWZ9u3TpImXLlpX9+/ebUBzsQeEFK8d1CCCAAAIIIBCqADmEAJxiDS1ZskSaNWsmH3/8sVy/fl2aN28uHTt2lNWrV0v69OmlQYMGMnz4cMmVK1eSbelaYv3Hd5w6dUpiYmLk5MmTUqhQoRT7wAkIIIAAAggggIBTAgRgAnCytaRLIXTmV2eEP/roI7MEYsSIETJy5EgTfA8ePCi9e/eWwoULy+eff24CcaBj8ODBMmTIkERfIgA79a1MOwgggAACCCBgVYAATABOUmDFihXSsmVLeeyxx8yOEFmzZjXrgS9duiS5c+f2X7d582apVq2a6P9WrVo1YHvMAFv9luQ8BBBAAAEEEHBbgABMAA4ooDtAdO/eXRo3bizz58+Ptx444QVXrlwxD8Z9+OGH0qRJE0s1S+FZYuIkBBBAAAEEEHBBgBxCAE4kMHXqVOncubN069ZN3nnnnXjLGnRG+Ny5c7JmzRr/dRs2bJDatWvL3r17zQNyVg4Kz4oS5yCAAAIIIICAGwLkEAJwPIEDBw5I+fLl5bnnnpMpU6bE+5ougdi0aZM0bNhQhg0bJi1atBA9/9VXXzVLIN5//33LNUrhWabiRAQQQAABBBBwWIAcQgCOJ6APtw0YMCCgSps2bcwb4JYuXWoehNu3b59ZC6yzwroLRJYsWSyXJ4VnmYoTEUAAAQQQQMBhAXIIAdjhkrLWHIVnzYmzEEAAAQQQQMB5AXIIAdj5qrLQIoVnAYlTEEAAAQQQQMAVAXIIAdiVwkqpUQovJSG+jgACCCCAAAJuCZBDCMBu1Vay7VJ4acLOTRFAAAEEEEBARMghBOA0+Uag8NKEnZsigAACCCCAAAE42RpIF6uvPeNwRYAA7AorjSKAAAIIIICABQFyCDPAFsrE+VMoPOdNaREBBBBAAAEErAmQQwjA1irF4bMoPIdBaQ4BBBBAAAEELAuQQwjAlovFyRMpPCc1aQsBBBBAAAEE7AiQQwjAdurFsXMpPMcoaQgBBBBAAAEEbAqQQwjANkvGmdMpPGccaQUBBBBAAAEE7AuQQwjA9qvGgSsoPAcQaQIBBBBAAAEEghIghxCAgyqcUC+i8EIV5HoEEEAAAQQQCFaAHEIADrZ2QrqOwguJj4sRQAABBBBAIAQBcggBOITyCf5SCi94O65EAAEEEEAAgdAEyCEE4NAqKMirKbwg4bgMAQQQQAABBEIWIIcQgEMuomAaoPCCUeMaBBBAAAEEEHBCgBxCAHaijmy3QeHZJuMCBBBAAAEEEHBIgBxCAHaolOw1Q+HZ8+JsBBBAAAEEEHBOgBxCAHaummy0ROHZwOJUBBBAAAEEEHBUgBxCAHa0oKw2RuFZleI8BBBAAAEEEHBagBxCAHa6piy1R+FZYuIkBBBAAAEEEHBBgBxCAHahrFJuksJL2YgzEEAAAQQQQMAdAXIIAdidykqhVQovTdi5KQIIIIAAAgiICDmEAJwm3wgUXpqwc1MEEEAAAQQQIAAnWwPpYmNjY6kSdwQIwO640ioCCCCAAAIIpCxADmEGOOUqceEMCs8FVJpEAAEEEEAAAUsC5BACsKVCcfokCs9pUdpDAAEEEEAAAasC5BACsNVacfQ8Cs9RThpDAAEEEEAAARsC5BACsI1yce5UCs85S1pCAAEEEEAAAXsC5BACsL2KcehsCs8hSJpBAAEEEEAAAdsC5BACsO2iceICCs8JRdpAAAEEEEAAgWAEyCEE4GDqJuRrKLyQCWkAAQQQQAABBIIUIIcQgIMsndAuo/BC8+NqBBBAAAEEEAhegBxCAA6+ekK4ksILAY9LEUAAAQQQQCAkAXIIATikAgr2YgovWDmuQwABBBBAAIFQBcghBOBQayio6ym8oNi4CAEEEEAAAQQcECCHEIAdKCP7TVB49s24AgEEEEAAAQScESCHEICdqSSbrVB4NsE4HQEEEEAAAQQcEyCHEIBtF9OdO3dkyJAhMnPmTLlw4YLUqFFDpkyZIiVLlrTcFoVnmYoTEUAAAQQQQMBhAXIIAdh2SWn41cA7e/ZsKViwoPTt21eOHDkie/bskbvuustSexSeJSZOQgABBBBAAAEXBMghBGBbZXXz5k3JkyePjBkzRjp27GiuvXjxotx///0ya9YsadGihaX2KDxLTJyEAAIIIIAAAi4IkEMIwLbKauvWrVK1alXZv3+/lC5d2n+tLoOoWLGimRm2clB4VpQ4BwEEEEAAAQTcECCHEIBt1dWyZcukSZMmcvXqVcmaNav/2mbNmpk/W7VqlaX2KDxLTJyEAAIIIIAAAi4IkEMIwLbKav78+dKqVSu5ffu2pE+f3n9t69at5eeff5Z169YFbO/y5cui//iOkydPSvXq1UVnlAsUKGCrD5yMAAIIIIAAAgiEInDq1CmJiYmRo0ePStGiRUNpKuKuTRcbGxsbcZ8qxA+0dOlSadq0acAZ4Bs3bsiKFSsC3mHw4MFm5wgOBBBAAAEEEEAgXAR0Iu7RRx8Nl+6ERT8IwAGGwbcG+NChQ1KiRAn/GboGuFKlSjJ58mRLM8DXr18XnQUuVqyYZMyY0ZUB9/10xyyzK7whN8r4hEzoegOMkevEId2A8QmJz/WLGR/XiUO6wa1bt+Ts2bNSoUIFyZIlS0htRdrFBOAAI6qzvPny5ZPx48dL+/btzRnB7AKRGsXC+p7UUA7+HoxP8HapdSVjlFrSwd2H8QnOLbWuYnxSS5r7OC1AAE5CdMCAATJ9+nSz7Zmum+nTp48cO3ZMdu3aZXkfYKcHK1B7/OWTGsrB34PxCd4uta5kjFJLOrj7MD7BuaXWVYxPaklzH6cFCMBJiOoDcP379zcvwrh27ZrUqlXLLH0It0Xk/OXj9LeEs+0xPs56utEaY+SGqnNtMj7OWbrREuPjhiptpoYAATg1lF28h+46oUs1evbsKTlz5nTxTjQdjADjE4xa6l7DGKWut927MT52xVL3fMYndb25m3MCBGDnLGkJAQQQQAABBBBAwAMCBGAPDBJdRAABBBBAAAEEEHBOgADsnCUtIYAAAggggAACCHhAgADsgUGiiwgggAACCCCAAALOCRCAnbOkJQQQQAABBBBAAAEPCBCAPTBIgbp4584d89rlmTNnyoULF0TfUjdlyhQpWbKkRz+Rt7t9/vx5s23eqlWrRJ+KrlixoowaNcqMix47d+6U7t27y7Zt2+S+++6Tbt26Se/evb39oT3a+wMHDkjlypVl0qRJ0rZtW8YnjMbx73//u/m+OXLkiHkLp75e/vnnn2eMwmSMfv31VzMm8+bNM//deeihh2T06NFSvXp1xihMxohuWBcgAFu3CqszNfxq4NV9igsWLCh9+/Y1/9HYs2dPWL2oI6zQXOzMM888I2fOnDGhKm/evGbPaP3hZPv27ebfy5QpI40aNTLb1W3evFk6d+5sxq9du3Yu9oqmEwrof8D1P9b6g4h+72gAPnfuHOMTBqUyf/58Mx7jxo2TZ599VhYuXGjC1ldffSWlS5dmjMJgjAYNGiQzZsyQuXPnSvHixU34Xbx4sfzrX/+SzJkzM0ZhMEZ0wboAAdi6VdicefPmTcmTJ4+MGTNGOnbsaPoVrq9qDhs0Fzty6NAhKVWqlHz99df+mZDY2FjzH+0XXnhBsmbNagKxvkkwY8aMpic6W7xs2TLZt2+fiz2j6YQC6q4/gHzxxRf+APz2228zPmlcKvr9ooGqadOmMnbsWH9v6tatK08++aT5d76H0niQRMyMb506dcwPKXrob7ty5colH374oehvVhijtB8jemBdgABs3Spszty6datUrVpV9u/fb0KW79Bft+uv3nVmkSP1BPSHj2+++cb8h0FnQXyHLkepV6+emZnPnTu3LFiwwP+1devWydNPPy2nT5+WfPnypV5no/hOX375pfzxj380y1GKFCniD8ANGjRgfNK4LvQHwbJly5rfmDz88MOJesMYpfEA/f/b699x+ua3NWvWSOHCheWdd96RN9980/zmsWvXrnwfhccw0QuLAgRgi1DhdJrOHDZp0kSuXr1qZhd9R7Nmzcyf6TpUjrQVWLJkieh4fPzxxzJgwACpX7+++XWh79i7d6+UK1dOvv32W3nkkUfStrNRcHf9IaVSpUqis70tW7aUdOnS+QOw/tDI+KRtEej3yR/+8AfZtGmTDBs2THbs2CHFihUz4er3v/+9+cGeMUrbMdK77969W5o3by7691eGDBkkffr0Zva3YcOGjFHaDw89sClAALYJFg6n61q5Vq1aye3bt81fQL6jdevW8vPPP4vOLnKknYAuhdCZX50t+eijj8yDiRq6hg4d6u+U7yEf/Q++70G5tOtx5N9Zl6Lor9kXLVpkPmzcAMz4pP34+/5O09Cr60z1h5WlS5fKiBEjZO3atfLKK6/wPZT2wyT6g/3EiROlT58+5tmTd999Vz744APR3640btyYMQqDMaIL1gUIwNatwuZM/Q+DrpULNAN848YNWbFiRdj0Ndo6ovYadh977DEzE68z9BUqVBD9FW6gGeCkfuUbbW5ufl59Yr1fv36ya9cuueeeexIFYMbHTX1rbeuDVDqzqMu3OnXq5L9IH4bT48SJE3wPWaN07azjx4+bZx3Wr18vNWvW9N9H/78+k6LPQvD3nGv8NOyCAAHYBVS3m/StAda/cHSrIN+hM4k6c6IPInCkvoDuAKFbnelMiM5o+dYD638UdOszDWK+gzXAqTc+tWvXNjsJxF2ffeXKFfPv+v3zwAMPSDmFuwAACXlJREFUMD6pNxwB76Rr6B9//HGzBCLub0R0dxv9QbJo0aKMURqPkW9Z17Vr1yRLliz+3uhs8CeffML3URqPD7e3L0AAtm+W5lfoLK8+ODV+/Hhp37696Q+7QKTtsEydOtVsbab7++qDIXGXpui+ptOmTZPDhw+bdXN66Izk8uXL2QUiFYbtp59+Ev2PdtxDZ7J0Rl5nHXW7LcYnFQYimVvo+OTPn18GDhwYb39sXfebKVMms7sKY5S2Y6S7p1SrVk22bNkiMTEx/s7oci8NxPpbL8YobceIu9sTIADb8wqbs/XBqunTp8usWbPM7Ij+FK7bbOmvee+6666w6Wc0dES3/ylfvrw899xziXbg0CUQ+gOL7gOsD4rojJbO4OuvefU/Fm3atIkGorD7jHHXAOv+zYxP2g/R8OHDzdaO+veaBixdr62BWH/l/uCDDzJGaTxE+vIl3ZLu7Nmz5u+5QoUKib64RB8s1d+w6DZ2fB+l8SBxe1sCBGBbXOFzsj4Ap3ua6mb+OntSq1Yts/RBwzBH6gqMHDnS7PQQ6NCAO2fOHLPbg84O69PtBQoUkF69ekmXLl1St6PczS8QNwDrHzI+4VEc+lstXUqks/a6LZq+8Ed3h2CMwmN89O1vujOHLkvR/6/r5/XvvyeeeIIxCo8hohc2BAjANrA4FQEEEEAAAQQQQMD7AgRg748hnwABBBBAAAEEEEDAhgAB2AYWpyKAAAIIIIAAAgh4X4AA7P0x5BMggAACCCCAAAII2BAgANvA4lQEEEAAAQQQQAAB7wsQgL0/hnwCBBBAAAEEEEAAARsCBGAbWJyKAAIIIIAAAggg4H0BArD3x5BPgAACCCCAAAIIIGBDgABsA4tTEUAAAQQQQAABBLwvQAD2/hjyCRBAwCUBffXrxo0b47WurxrXt/k1atTIvAZWX3ft1KFvcmzbtq0MHjzYqSYTtZPwHvpWvLhH5syZzWtumzdvbt5wePfdd7vWFxpGAAEE0kqAAJxW8twXAQTCXkADcP78+WXixIn+vv7yyy/y2WefSY8ePeTll182r+516jh79qwJ1NmzZ3eqSUsBeMKECSbw6qGfb+vWrdK7d2/zOuI1a9ZIxowZXesPDSOAAAJpIUAATgt17okAAp4Q0ACsM6Zz5sxJ1N+XXnpJVqxYIefOnfPEZ/F1MtAM8OzZs83Mc9zju+++k0cffVRmzpwp+lk5EEAAgUgSIABH0mjyWRBAwFGB5AJw586d5cMPP5QzZ86Ye968eVPeeustmT9/vly6dEnKly8vQ4cOlWeeecbfJ5057tevn+zdu1eKFy8uvXr1kvbt28vRo0dN0E4YTv/xj3/IsGHDZPfu3ZIzZ0554YUXZMSIEZIlSxbTpi5fmDFjhixevFi++uoruffee6VLly7mHkkdVgOwXl+rVi3RJR/r1q1z1JXGEEAAgbQWIACn9QhwfwQQCFuBQAH41q1bZllAixYtzBKIcePGmf63bNlS9uzZI7qcQNfQrly5Ut544w1Zvny5PPvss7Jz504zo/raa6+Z0Kv/3qlTJ7lw4ULAAPzRRx9JkyZNzHrgZs2aycGDB835MTExsnTpUn8AzpUrl1mGoX8+b948GT58uHz55ZdSs2bNgK52AvCrr74qH3zwgfznP/8J2zGiYwgggEAwAgTg/9fe/bvSH8VxHH/HKKXEhhSZiFKSQYqyMPgHpBQLSjFYZJEfZRGRLAaUMimDgUVImZGBgbIY/Ujx7fWuzw1dvt1v9/s5uvd5Fupe99zzOIZXp/d5f/5Fjb9BAIGsEFAAPjw8NF0Mi8bT05OVlZV5ycDY2JjXx15dXVllZaWdnp5afX194r3d3d12c3NjBwcHpt8vLy/t6Ogo8fr8/LwNDg4mDcANDQ1WUlLip8zRUKju7Oz0E2TV5+oEeGhoyEN3NHQKPDIy8u0pcCoBWJfgZmZm7PX1NSv2m0UigED2CBCAs2evWSkCCKQooABcWFho09PT9vb2ZicnJ36C29XVZYuLi4nLYVtbW35Km5eX92kGBceCggK7v7+36upqa2trs7m5ucR7dApcV1eXNACr+8Lk5KRftouGSiv0eZubm35pTQF4aWnJ+vr6Eu9RwO3p6bHx8fGkq00lAKucQnNxApziPw5vRwCBXy9AAP71W8QXRACBUALJSiD29/ettbXVQ6dCsIbKBFQScXZ2Zvn5+Z++bm5urpWXl3vQbW5u/nRa+1MAVjeIqakpP+GNxsPDgwdylUAohCsAf73A9rdWaqkE4MbGRl+PapcZCCCAQCYJEIAzaTdZCwIIpFXgu0two6OjNjs7a7u7u9be3u61v7r0trOz4/W+0VAJQU5Ojl9kU8nE+fm5HR8fJ15fXl62/v7+pCfAquktLS39VAKhrhPqP/yxBOJ/BWB1gVA5hzpgqHyDgQACCGSSAAE4k3aTtSCAQFoFvgvAz8/PVlNTYy8vLx5+1be3o6PDL7YtLCx4GN7e3jYF5dXVVS9JUCeH2tpa76+rtmIKsbpEp96/19fXXlf88XRWZRUqc5iYmPCfqh/WJTh9hmqBNdJ1ApysD7Au8FVUVNje3p7pFJuBAAIIZJIAATiTdpO1IIBAWgV+aoOmJ8S1tLTYwMCAPyjj8fHRn5ymcgiVKqjN2fDwsPX29ia+k4KrWpSpo0NVVVWiy8Pd3Z0/Xe5recLGxoY/be7i4sKKioq804QCcfT0uXQF4I9oaremkg3NpQt6Ucu1tMLyYQgggEBgAQJw4A1gegQQyA4BdYhQxwjVAkdjfX3dT4P19DWetpYd/wesEgEEfocAAfh37APfAgEEMlxgZWXF25Otra15GYNap+kiXVNTU9InzWU4B8tDAAEEggoQgIPyMzkCCGSLwPv7u1+G06Wy29tbKy4u9ie7fSxpyBYL1okAAgiEFiAAh94B5kcAAQQQQAABBBCIVYAAHCs3kyGAAAIIIIAAAgiEFiAAh94B5kcAAQQQQAABBBCIVYAAHCs3kyGAAAIIIIAAAgiEFiAAh94B5kcAAQQQQAABBBCIVYAAHCs3kyGAAAIIIIAAAgiEFiAAh94B5kcAAQQQQAABBBCIVYAAHCs3kyGAAAIIIIAAAgiEFiAAh94B5kcAAQQQQAABBBCIVYAAHCs3kyGAAAIIIIAAAgiEFiAAh94B5kcAAQQQQAABBBCIVYAAHCs3kyGAAAIIIIAAAgiEFiAAh94B5kcAAQQQQAABBBCIVeAP9KLGvhKePYAAAAAASUVORK5CYII=\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "pb_num_dict = {_k:len(_v.kept_probes) for _k,_v in filtered_pb_dict.items()}\n",
    "\n",
    "pb_nums = np.array(list(pb_num_dict.values()))\n",
    "# number of probes histogram\n",
    "fig, ax = plt.subplots()\n",
    "ax.plot(pb_nums, '.')\n",
    "ax.set_ylabel(\"Num probes per region\")\n",
    "ax.set_xlabel(\"Region ID\")\n",
    "ax.set_title(\"Probe number after filtering\")\n",
    "ax.set_ylim([0,200])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 102,
   "id": "f2eb1b00",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\chr_2_final_region_num.pkl\n"
     ]
    }
   ],
   "source": [
    "chr_2_region_savefile = os.path.join(library_folder, 'chr_2_final_region_num.pkl')\n",
    "overwrite = True\n",
    "\n",
    "filtered_reg_names = {}\n",
    "for _reg, _pbde in filtered_pb_dict.items():\n",
    "    if len(_pbde.kept_probes) > 0:\n",
    "        filtered_reg_names[_reg] = list(_pbde.kept_probes.values())[0]['reg_name']\n",
    "\n",
    "if not os.path.exists(chr_2_region_savefile) or overwrite:\n",
    "    chr_2_region_num = {}\n",
    "    for _reg, _info in filtered_reg_names.items():\n",
    "        _chr = _info.split('_')[0].split(':')[0]\n",
    "        if _chr not in chr_2_region_num:\n",
    "            chr_2_region_num[_chr] = 1\n",
    "        else:\n",
    "            chr_2_region_num[_chr] += 1\n",
    "    print(f\"Saving into file: {chr_2_region_savefile}\")\n",
    "\n",
    "    pickle.dump(chr_2_region_num, open(chr_2_region_savefile, 'wb'))\n",
    "else:\n",
    "    print(f\"loading from file: {chr_2_region_savefile}\")\n",
    "    chr_2_region_num = pickle.load(open(chr_2_region_savefile, 'rb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "id": "ad002f0a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'7': 96}"
      ]
     },
     "execution_count": 103,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "chr_2_region_num"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "id": "8e416446",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving region names into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\chr_2_region_names.pkl\n"
     ]
    }
   ],
   "source": [
    "#\n",
    "reg_2_start = {}\n",
    "\n",
    "for _name, _info in filtered_reg_names.items():\n",
    "    _locus = _info.split('_')[0]\n",
    "    _start,_end = _locus.split(':')[1].split('-')\n",
    "    reg_2_start[_name] = int(_start)\n",
    "#    \n",
    "chr_2_region_names = {}\n",
    "\n",
    "for _reg, _info in filtered_reg_names.items():\n",
    "    _chr = _info.split('_')[0].split(':')[0]\n",
    "    if _chr not in chr_2_region_names:\n",
    "        chr_2_region_names[_chr] = [_reg]\n",
    "    else:\n",
    "        chr_2_region_names[_chr].append(_reg)\n",
    "# sort\n",
    "for _chr, _regs in chr_2_region_names.items():\n",
    "    _sorted_regs = sorted(_regs, key=lambda _r: reg_2_start[_r])\n",
    "    chr_2_region_names[_chr] = _sorted_regs\n",
    "    \n",
    "overwrite_region_names = True\n",
    "chr_2_region_name_savefile = os.path.join(library_folder, 'chr_2_region_names.pkl')\n",
    "\n",
    "if not os.path.exists(chr_2_region_name_savefile) or overwrite_region_names:\n",
    "    print(f\"Saving region names into file: {chr_2_region_name_savefile}\")\n",
    "    pickle.dump(chr_2_region_names, open(chr_2_region_name_savefile, 'wb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2c3d9549",
   "metadata": {},
   "source": [
    "<a id='3'></a>\n",
    "# 3. Load and asssemble probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 106,
   "id": "136b756f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'383': ['u0', 'u96'], '384': ['u1', 'u97'], '385': ['u2', 'u98'], '386': ['u3', 'u99'], '387': ['u4', 'u100'], '388': ['u5', 'u101'], '389': ['u6', 'u102'], '390': ['u7', 'u103'], '391': ['u8', 'u104'], '392': ['u9', 'u105'], '393': ['u10', 'u106'], '394': ['u11', 'u107'], '395': ['u12', 'u108'], '396': ['u13', 'u109'], '397': ['u14', 'u110'], '398': ['u15', 'u111'], '399': ['u16', 'u112'], '400': ['u17', 'u113'], '401': ['u18', 'u114'], '402': ['u19', 'u115'], '403': ['u20', 'u116'], '404': ['u21', 'u117'], '405': ['u22', 'u118'], '406': ['u23', 'u119'], '407': ['u24', 'u120'], '408': ['u25', 'u121'], '409': ['u26', 'u122'], '410': ['u27', 'u123'], '411': ['u28', 'u124'], '412': ['u29', 'u125'], '413': ['u30', 'u126'], '414': ['u31', 'u127'], '415': ['u32', 'u128'], '416': ['u33', 'u129'], '417': ['u34', 'u130'], '418': ['u35', 'u131'], '419': ['u36', 'u132'], '420': ['u37', 'u133'], '421': ['u38', 'u134'], '422': ['u39', 'u135'], '423': ['u40', 'u136'], '424': ['u41', 'u137'], '425': ['u42', 'u138'], '426': ['u43', 'u139'], '427': ['u44', 'u140'], '428': ['u45', 'u141'], '429': ['u46', 'u142'], '430': ['u47', 'u143'], '431': ['u48', 'u144'], '989': ['u49', 'u145'], '1010': ['u50', 'u146'], '1397': ['u51', 'u147'], '1398': ['u52', 'u148'], '1399': ['u53', 'u149'], '1400': ['u54', 'u150'], '1401': ['u55', 'u151'], '1402': ['u56', 'u152'], '1403': ['u57', 'u153'], '1404': ['u58', 'u154'], '1405': ['u59', 'u155'], '1406': ['u60', 'u156'], '1407': ['u61', 'u157'], '1408': ['u62', 'u158'], '1409': ['u63', 'u159'], '1410': ['u64', 'u160'], '1411': ['u65', 'u161'], '1412': ['u66', 'u162'], '1413': ['u67', 'u163'], '1414': ['u68', 'u164'], '1415': ['u69', 'u165'], '1416': ['u70', 'u166'], '1417': ['u71', 'u167'], '1418': ['u72', 'u168'], '1419': ['u73', 'u169'], '1420': ['u74', 'u170'], '1421': ['u75', 'u171'], '1422': ['u76', 'u172'], '1423': ['u77', 'u173'], '1424': ['u78', 'u174'], '1425': ['u79', 'u175'], '1426': ['u80', 'u176'], '1427': ['u81', 'u177'], '1428': ['u82', 'u178'], '1429': ['u83', 'u179'], '1430': ['u84', 'u180'], '1431': ['u85', 'u181'], '1432': ['u86', 'u182'], '1433': ['u87', 'u183'], '1434': ['u88', 'u184'], '1435': ['u89', 'u185'], '1436': ['u90', 'u186'], '1437': ['u91', 'u187'], '1438': ['u92', 'u188'], '1439': ['u93', 'u189'], '1440': ['u94', 'u190'], '1441': ['u95', 'u191']}\n"
     ]
    }
   ],
   "source": [
    "region_2_readout_dict = {}\n",
    "for _i, _reg in enumerate(pb_dict):\n",
    "    region_2_readout_dict[_reg] = [f\"u{_i}\", f\"u{_i+len(pb_dict)}\"]\n",
    "print(region_2_readout_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 107,
   "id": "5846cb9e",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'383': ['u0', 'u96'],\n",
       " '384': ['u1', 'u97'],\n",
       " '385': ['u2', 'u98'],\n",
       " '386': ['u3', 'u99'],\n",
       " '387': ['u4', 'u100'],\n",
       " '388': ['u5', 'u101'],\n",
       " '389': ['u6', 'u102'],\n",
       " '390': ['u7', 'u103'],\n",
       " '391': ['u8', 'u104'],\n",
       " '392': ['u9', 'u105'],\n",
       " '393': ['u10', 'u106'],\n",
       " '394': ['u11', 'u107'],\n",
       " '395': ['u12', 'u108'],\n",
       " '396': ['u13', 'u109'],\n",
       " '397': ['u14', 'u110'],\n",
       " '398': ['u15', 'u111'],\n",
       " '399': ['u16', 'u112'],\n",
       " '400': ['u17', 'u113'],\n",
       " '401': ['u18', 'u114'],\n",
       " '402': ['u19', 'u115'],\n",
       " '403': ['u20', 'u116'],\n",
       " '404': ['u21', 'u117'],\n",
       " '405': ['u22', 'u118'],\n",
       " '406': ['u23', 'u119'],\n",
       " '407': ['u24', 'u120'],\n",
       " '408': ['u25', 'u121'],\n",
       " '409': ['u26', 'u122'],\n",
       " '410': ['u27', 'u123'],\n",
       " '411': ['u28', 'u124'],\n",
       " '412': ['u29', 'u125'],\n",
       " '413': ['u30', 'u126'],\n",
       " '414': ['u31', 'u127'],\n",
       " '415': ['u32', 'u128'],\n",
       " '416': ['u33', 'u129'],\n",
       " '417': ['u34', 'u130'],\n",
       " '418': ['u35', 'u131'],\n",
       " '419': ['u36', 'u132'],\n",
       " '420': ['u37', 'u133'],\n",
       " '421': ['u38', 'u134'],\n",
       " '422': ['u39', 'u135'],\n",
       " '423': ['u40', 'u136'],\n",
       " '424': ['u41', 'u137'],\n",
       " '425': ['u42', 'u138'],\n",
       " '426': ['u43', 'u139'],\n",
       " '427': ['u44', 'u140'],\n",
       " '428': ['u45', 'u141'],\n",
       " '429': ['u46', 'u142'],\n",
       " '430': ['u47', 'u143'],\n",
       " '431': ['u48', 'u144'],\n",
       " '989': ['u49', 'u145'],\n",
       " '1010': ['u50', 'u146'],\n",
       " '1397': ['u51', 'u147'],\n",
       " '1398': ['u52', 'u148'],\n",
       " '1399': ['u53', 'u149'],\n",
       " '1400': ['u54', 'u150'],\n",
       " '1401': ['u55', 'u151'],\n",
       " '1402': ['u56', 'u152'],\n",
       " '1403': ['u57', 'u153'],\n",
       " '1404': ['u58', 'u154'],\n",
       " '1405': ['u59', 'u155'],\n",
       " '1406': ['u60', 'u156'],\n",
       " '1407': ['u61', 'u157'],\n",
       " '1408': ['u62', 'u158'],\n",
       " '1409': ['u63', 'u159'],\n",
       " '1410': ['u64', 'u160'],\n",
       " '1411': ['u65', 'u161'],\n",
       " '1412': ['u66', 'u162'],\n",
       " '1413': ['u67', 'u163'],\n",
       " '1414': ['u68', 'u164'],\n",
       " '1415': ['u69', 'u165'],\n",
       " '1416': ['u70', 'u166'],\n",
       " '1417': ['u71', 'u167'],\n",
       " '1418': ['u72', 'u168'],\n",
       " '1419': ['u73', 'u169'],\n",
       " '1420': ['u74', 'u170'],\n",
       " '1421': ['u75', 'u171'],\n",
       " '1422': ['u76', 'u172'],\n",
       " '1423': ['u77', 'u173'],\n",
       " '1424': ['u78', 'u174'],\n",
       " '1425': ['u79', 'u175'],\n",
       " '1426': ['u80', 'u176'],\n",
       " '1427': ['u81', 'u177'],\n",
       " '1428': ['u82', 'u178'],\n",
       " '1429': ['u83', 'u179'],\n",
       " '1430': ['u84', 'u180'],\n",
       " '1431': ['u85', 'u181'],\n",
       " '1432': ['u86', 'u182'],\n",
       " '1433': ['u87', 'u183'],\n",
       " '1434': ['u88', 'u184'],\n",
       " '1435': ['u89', 'u185'],\n",
       " '1436': ['u90', 'u186'],\n",
       " '1437': ['u91', 'u187'],\n",
       " '1438': ['u92', 'u188'],\n",
       " '1439': ['u93', 'u189'],\n",
       " '1440': ['u94', 'u190'],\n",
       " '1441': ['u95', 'u191']}"
      ]
     },
     "execution_count": 107,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "region_2_readout_dict"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a4dd2783",
   "metadata": {},
   "source": [
    "<a id='3.2'></a>\n",
    "## 3.2 loading components"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8e899029",
   "metadata": {},
   "source": [
    "### primers\n",
    "pick primer set 20,21"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 109,
   "id": "fd766707",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Picked primer: ID: W1A05_primer_4\n",
      "Name: W1A05_primer_4\n",
      "Description: W1A05_primer_4\n",
      "Number of features: 0\n",
      "Seq('CATTCAGCATTGCGCAACGG')\n",
      "- Picked primer: ID: W1A04_primer_3\n",
      "Name: W1A04_primer_3\n",
      "Description: W1A04_primer_3\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGCGTTGTATGCCCTCCACGC')\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "primer_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Primers'\n",
    "primers = library_tools.assemble.load_primers([4,3], _primer_folder=primer_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3dfa7962",
   "metadata": {},
   "source": [
    "### readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 110,
   "id": "e778a2e8",
   "metadata": {},
   "outputs": [],
   "source": [
    "# collect types from region_2_readout_dict\n",
    "readout_usage_by_type = {}\n",
    "for _region, _infos in region_2_readout_dict.items():\n",
    "    for _info in _infos:\n",
    "        if _info[0] not in readout_usage_by_type:\n",
    "            readout_usage_by_type[_info[0]] = []\n",
    "        if int(_info[1:]) not in readout_usage_by_type[_info[0]]:\n",
    "            readout_usage_by_type[_info[0]].append(int(_info[1:]))\n",
    "\n",
    "readout_usage_dict = {_type:len(_v) for _type,_v in readout_usage_by_type.items()}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "id": "e92509a7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\designed_readouts_750.fasta 453 0 64\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\designed_readouts_647.fasta 453 0 64\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts\\designed_readouts_561.fasta 452 0 64\n",
      "[64, 64, 64]\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3.classes import _allowed_kwds\n",
    "\n",
    "readout_dict ={}\n",
    "overwrite_readout_usage = True\n",
    "readout_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts'\n",
    "\n",
    "# unique\n",
    "readout_type = 'unique'\n",
    "# \n",
    "_type_char = _allowed_kwds[readout_type]\n",
    "_readout_start = 0\n",
    "_num_color = 3\n",
    "\n",
    "readout_dict[_type_char] = library_tools.assemble.load_readouts(readout_usage_dict[_type_char], \n",
    "                                                       _type='designed_readouts', \n",
    "                                                       _readout_folder=readout_folder, \n",
    "                                                       _num_colors=_num_color, # 750,647,561\n",
    "                                                       _start_id=_readout_start,\n",
    "                                                       )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "id": "ee209a25",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'u': [SeqRecord(seq=Seq('ACAAATCCGACCAGATCGGACGATCATGGG'), id='Stv_3', name='Stv_3', description='Stv_3 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGACGACGCACGTTCGTACCGCGTACTTCG'), id='Stv_32', name='Stv_32', description='Stv_32 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCCCGCCAATGACGTCGGTTTGGACGAGA'), id='Stv_91', name='Stv_91', description='Stv_91 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAAGTATGCAGCGCGATTGACCGTCTCGTT'), id='Stv_4', name='Stv_4', description='Stv_4 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTGCTCGCAAGTGCGCACGAGTTGAACTG'), id='Stv_33', name='Stv_33', description='Stv_33 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCGTTGATGTCCCTTGTGAGCGCCCGACAT'), id='Stv_92', name='Stv_92', description='Stv_92 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGTCGTACGCCGATGCGCAGCAATTCACT'), id='Stv_5', name='Stv_5', description='Stv_5 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTTTCCATACGCGGCGTGAATATCGCGGC'), id='Stv_35', name='Stv_35', description='Stv_35 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGCTTATCGATGTCAGGTCCGCATGGGTCG'), id='Stv_94', name='Stv_94', description='Stv_94 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGAAACATCGGCCACGGTCCCGTTGAACTT'), id='Stv_6', name='Stv_6', description='Stv_6 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTACAGCGTTACGGGCCCAGACGTTTCAT'), id='Stv_36', name='Stv_36', description='Stv_36 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACGTCATCGGTGGATCCCAGAGTGCCAAA'), id='Stv_95', name='Stv_95', description='Stv_95 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACGAATCCACCGTCCAGCGCGTCAAACAGA'), id='Stv_7', name='Stv_7', description='Stv_7 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTTATCGTGCGTGTCCGCCAGTACCGTGAG'), id='Stv_37', name='Stv_37', description='Stv_37 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTCCGATGAAACGTCCCGTGTGCTGTCGCG'), id='Stv_99', name='Stv_99', description='Stv_99 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGCGAAATCCCCGTAACGAGCGTCCCTTGC'), id='Stv_8', name='Stv_8', description='Stv_8 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGTGCGGGAACCGTATCCCTGGCGCGGACT'), id='Stv_39', name='Stv_39', description='Stv_39 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACACGACGTCCAAAGTTGGTCTCGCGCAA'), id='Stv_100', name='Stv_100', description='Stv_100 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCATGAGTTGCCTGGCGTTGCGACGACTAA'), id='Stv_9', name='Stv_9', description='Stv_9 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AATCCGGCGTACGTAAGGCAGCTTGCGTTA'), id='Stv_40', name='Stv_40', description='Stv_40 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGGCCACCCGACTGCGACGAACTCGATGC'), id='Stv_101', name='Stv_101', description='Stv_101 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCGTCGTCTCCGGTCCACCGTTGCGCTTAC'), id='Stv_10', name='Stv_10', description='Stv_10 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCGGGATTCGTACGAGCCCTCTTGGACGGG'), id='Stv_42', name='Stv_42', description='Stv_42 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGGCGTTGCGCGATTGCCGCTTCAATCAA'), id='Stv_104', name='Stv_104', description='Stv_104 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCCAATGGCCCAGGTCCGTCACGCAATTT'), id='Stv_11', name='Stv_11', description='Stv_11 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTTTGTTCGGCTCGGAGCGTTACTGCGGGC'), id='Stv_44', name='Stv_44', description='Stv_44 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCACGCTTCAACAAACGGCACACTCGGAT'), id='Stv_105', name='Stv_105', description='Stv_105 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGATCGAATCGGAGCGTAGCGGAATCTGC'), id='Stv_12', name='Stv_12', description='Stv_12 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAATTCCGGTATCGGGCCCTTTTGTCTGAC'), id='Stv_45', name='Stv_45', description='Stv_45 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGACTCCCGACCTGCGCACTTGCCCGTGTT'), id='Stv_106', name='Stv_106', description='Stv_106 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGCGCGGATCCGCTTGTCGGGAACGGATAC'), id='Stv_13', name='Stv_13', description='Stv_13 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGTGAGAATGCGTCGCGATACTGGTGTAAG'), id='Stv_46', name='Stv_46', description='Stv_46 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCCCACACGCCAAGCGCAGACGAACGGCA'), id='Stv_107', name='Stv_107', description='Stv_107 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCCTCGATTACGACGGATGTAATTCGGCCG'), id='Stv_14', name='Stv_14', description='Stv_14 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTCATGGTCGCCGGACCGTGATGCCGTGT'), id='Stv_48', name='Stv_48', description='Stv_48 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCATCGCCCGAAATTCCCTGGTGCGGTAAC'), id='Stv_109', name='Stv_109', description='Stv_109 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCGGGAAGCACGTGGATTAGGGCATCGACC'), id='Stv_16', name='Stv_16', description='Stv_16 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCCATGGTACGACGGCAAGAGAGCGTGCGT'), id='Stv_50', name='Stv_50', description='Stv_50 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCGACCCTCGCCTGGATTTGCGCGAGGTG'), id='Stv_118', name='Stv_118', description='Stv_118 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGTGTATCCCGCCGGATCTTTGAATCAAC'), id='Stv_19', name='Stv_19', description='Stv_19 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACACCCGATACGTCGTGGGACGTGGACCGT'), id='Stv_53', name='Stv_53', description='Stv_53 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAAACGCACCTCACGTTCCTGCGCGGACTT'), id='Stv_119', name='Stv_119', description='Stv_119 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGCTAGTTCGATCGCGCAATTGCGTAGTTC'), id='Stv_20', name='Stv_20', description='Stv_20 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGACGATGATCGCAATCGGAGAACTACACC'), id='Stv_54', name='Stv_54', description='Stv_54 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTCGCGAATCGGATAGCGAAATTGGATGA'), id='Stv_120', name='Stv_120', description='Stv_120 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGACACGGGATCGACGGTAGTGGATCGATT'), id='Stv_21', name='Stv_21', description='Stv_21 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGCGACGGGACTCTATCGGCATGGAGGTA'), id='Stv_59', name='Stv_59', description='Stv_59 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCTTTACCGTCAAATCGTGTGAATCGTGC'), id='Stv_121', name='Stv_121', description='Stv_121 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGACCTTAGGTTCCGCGTCATGTACCGGTT'), id='Stv_22', name='Stv_22', description='Stv_22 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTCTGGGTCGTCCGTAGGCAAAGGGTCCG'), id='Stv_60', name='Stv_60', description='Stv_60 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGTGCAACCGGTTCAAGTCGGTCGCGCATT'), id='Stv_125', name='Stv_125', description='Stv_125 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTTTACGACGTGGGCTGCTGCGATTTCAC'), id='Stv_23', name='Stv_23', description='Stv_23 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCCATGCGTCGACAATGGTGCGAAAGACCC'), id='Stv_61', name='Stv_61', description='Stv_61 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCGTTCGAGATCCAAGTTGGCTGCCGTTCG'), id='Stv_127', name='Stv_127', description='Stv_127 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GATGGTCGACTGGCGGTCTTAATATGCCCA'), id='Stv_25', name='Stv_25', description='Stv_25 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGAGCGTTGCGCGAGGAATTGGCGCATCG'), id='Stv_63', name='Stv_63', description='Stv_63 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGTCGATACATCCAACTCCAAACCGTCGGC'), id='Stv_129', name='Stv_129', description='Stv_129 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCCATATGACCGGCGGCCTTTTCTCGGAC'), id='Stv_26', name='Stv_26', description='Stv_26 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCGCGGTCCCAAGATGCCGTGAGCCTTTCA'), id='Stv_64', name='Stv_64', description='Stv_64 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTGGCGACTGTGATCCGCAACGTGGTGACA'), id='Stv_130', name='Stv_130', description='Stv_130 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGATCCACGACCGAGCAGGTTAGTTGACG'), id='Stv_27', name='Stv_27', description='Stv_27 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCGAATCGGCTTCCAATCCCTAAGGCCAC'), id='Stv_65', name='Stv_65', description='Stv_65 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGCACGCAACCAGCGCGAACCCTGAGCAAA'), id='Stv_131', name='Stv_131', description='Stv_131 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAGACGCACCACGCTCGACAAGAACAGGAC'), id='Stv_28', name='Stv_28', description='Stv_28 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGGTGCCAAGCTTCACCGCGTTTGTTCAAC'), id='Stv_86', name='Stv_86', description='Stv_86 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGCGAGGGCACTCACGCTGGAAGTGATACG'), id='Stv_133', name='Stv_133', description='Stv_133 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTCGTTTAACGGCCGGTTGGTCTACGGAT'), id='Stv_29', name='Stv_29', description='Stv_29 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGCCGATTGCCCAGAGTCGGACATCTCCCA'), id='Stv_87', name='Stv_87', description='Stv_87 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGGTCACGCAACCGAACCGACGTCGCAAAT'), id='Stv_136', name='Stv_136', description='Stv_136 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGTACTCGCGTCTGGAACGACGGATTCAG'), id='Stv_30', name='Stv_30', description='Stv_30 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGGCCGACGCGACACGTCCCTCAATCGAAC'), id='Stv_88', name='Stv_88', description='Stv_88 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAATCCGGGCAAGTCGGTTGGGAGCGTCCC'), id='Stv_145', name='Stv_145', description='Stv_145 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AACATCGGATCGGTGCGGTGGGATGGATAA'), id='Stv_31', name='Stv_31', description='Stv_31 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGGCGGGAGTTGGACTTCCGGTCGATATCC'), id='Stv_90', name='Stv_90', description='Stv_90 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCGGTTCTGCATTTGGGTCATTCCCACCGA'), id='Stv_182', name='Stv_182', description='Stv_182 old_barcode', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTAACCGATGTTGACGGACTAATCCTGAC'), id='NDB_1', name='NDB_1', description='NDB_1', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCATATAGTAGTTCAGACGCCGTTAAGCGC'), id='NDB_2', name='NDB_2', description='NDB_2', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAACACTGACGTGTGAGGCGCTAGAGCATA'), id='NDB_3', name='NDB_3', description='NDB_3', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCTTGGTATGGCACGCCTAATCTGGACAC'), id='NDB_4', name='NDB_4', description='NDB_4', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGCTTGCTATTACGAGCGCTTGGATCCCGT'), id='NDB_5', name='NDB_5', description='NDB_5', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCTGATTAACCGAACTGACGGCCATCAAGG'), id='NDB_6', name='NDB_6', description='NDB_6', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTGTTCGGTTTCAAGGATGATCCGCGCTT'), id='NDB_7', name='NDB_7', description='NDB_7', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TATATAGAGACTGCCCGACACATCTTAGTG'), id='NDB_8', name='NDB_8', description='NDB_8', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTAAACGTCGTATTATACCCGCCATGGAA'), id='NDB_9', name='NDB_9', description='NDB_9', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAAGATGCACCGTTCATATGTTATCGGGAC'), id='NDB_10', name='NDB_10', description='NDB_10', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TAGCTAGAGACTTACACCCGTAGACGTCGG'), id='NDB_11', name='NDB_11', description='NDB_11', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGTTATGAGCTGTCTCGAACCGAAGGCAC'), id='NDB_12', name='NDB_12', description='NDB_12', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACACTTCGGGCGGTTCAACTTACTGGTAGA'), id='NDB_13', name='NDB_13', description='NDB_13', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTGTCTAAGTGGCGCCGATTGTTTGTCCA'), id='NDB_14', name='NDB_14', description='NDB_14', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AATTAGTGATTGTCGCCTTGACGAGCCTAC'), id='NDB_15', name='NDB_15', description='NDB_15', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAGATTTAGCTCTATTAGCCGCGATGCGC'), id='NDB_16', name='NDB_16', description='NDB_16', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCCCTGTTGCGATGTTCTACCGGATACAAG'), id='NDB_17', name='NDB_17', description='NDB_17', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTAATTGGTATCGATTTGTCGAACCGAGCC'), id='NDB_18', name='NDB_18', description='NDB_18', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTGTGGATCCAAATCGCGGTTCCTTATCA'), id='NDB_19', name='NDB_19', description='NDB_19', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGATACGTTCGCACCGTGTCCATCGTATTT'), id='NDB_20', name='NDB_20', description='NDB_20', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCAAACCGTATGGGCGTCATAAAGGGAGC'), id='NDB_21', name='NDB_21', description='NDB_21', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCCCTTGGGATGGCGTCGTCTTAGCGTTAT'), id='NDB_22', name='NDB_22', description='NDB_22', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTAAAGCTAACTGGAACGTCCCGGAATTGC'), id='NDB_23', name='NDB_23', description='NDB_23', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCTAAACGGGAGTCCGAGGTCGTACCTAG'), id='NDB_24', name='NDB_24', description='NDB_24', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGGATCTCACGTTATATTCGTCTCAGTAC'), id='NDB_25', name='NDB_25', description='NDB_25', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCGAAGTACGAGCACTAGAGGAGTTCGCG'), id='NDB_26', name='NDB_26', description='NDB_26', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCGAAAGTAAGCTCGCGATGTAGACGTTT'), id='NDB_27', name='NDB_27', description='NDB_27', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACGTTGAGCGACAATCGACTTTCGTGGATC'), id='NDB_28', name='NDB_28', description='NDB_28', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAGGTGATTCGTGCGCGTCCATGCCTAAGT'), id='NDB_29', name='NDB_29', description='NDB_29', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TACCAAGTGTTTCGAGTCGTTTCCACTACT'), id='NDB_30', name='NDB_30', description='NDB_30', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCTGTTTGCGTCGGGACAACTATCTACAAT'), id='NDB_31', name='NDB_31', description='NDB_31', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCTATAGATCATACGATCCGGACCTGGAG'), id='NDB_32', name='NDB_32', description='NDB_32', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCGTGTGTTTACGTGCTTCGTTTGGATCC'), id='NDB_33', name='NDB_33', description='NDB_33', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TACTAATCGTTCAGGAATGCCGTTCTCGAC'), id='NDB_34', name='NDB_34', description='NDB_34', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGATATCATACGTAATGGAGGCTCGCCTTG'), id='NDB_35', name='NDB_35', description='NDB_35', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCTTACGTCTAGTGTGGTACGGTATCCAC'), id='NDB_36', name='NDB_36', description='NDB_36', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GACGATAGCCGTTATCGATTAATCGTCCTG'), id='NDB_37', name='NDB_37', description='NDB_37', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TATCACGGGTATGCTTGCCTACTAACTGTT'), id='NDB_38', name='NDB_38', description='NDB_38', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTATGTCTGTATGGCCGACTTTACTCGTC'), id='NDB_39', name='NDB_39', description='NDB_39', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCTTGACGATGACAACGCGCAGTTCTTAC'), id='NDB_40', name='NDB_40', description='NDB_40', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCATGTCTAAAGTATGCGTCGCGGCATGA'), id='NDB_41', name='NDB_41', description='NDB_41', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCACTGGTTATCGCATGTCGACAAGGACGT'), id='NDB_42', name='NDB_42', description='NDB_42', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCATCGAATTGATCCGAAGAGCGTGCACA'), id='NDB_43', name='NDB_43', description='NDB_43', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCCATGGATAAACCGGCTAGGTCGCAGAG'), id='NDB_44', name='NDB_44', description='NDB_44', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGGTATCGGTGACGGCGATTCTCACATTT'), id='NDB_45', name='NDB_45', description='NDB_45', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATTGTGGGATCGTTTCACCCTTTCGAGTCG'), id='NDB_46', name='NDB_46', description='NDB_46', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGATCTGATACTCGACTCGCCATACTTGG'), id='NDB_47', name='NDB_47', description='NDB_47', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGAGTGACACGCACGTGACTCTAATGGACG'), id='NDB_48', name='NDB_48', description='NDB_48', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCGAATACGGCAGTTGTGGCTCCCGTTTA'), id='NDB_49', name='NDB_49', description='NDB_49', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGTCAAGACGTACAGTAATATTCGCTTCGT'), id='NDB_50', name='NDB_50', description='NDB_50', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCTAGAGCGAGTTTCATTCGATACCTCGA'), id='NDB_51', name='NDB_51', description='NDB_51', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCACAGTTAGCTCGTGCGTTATCGGTTCTT'), id='NDB_52', name='NDB_52', description='NDB_52', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACACTGACATGATTGCCGTTCCGCAAGTTT'), id='NDB_53', name='NDB_53', description='NDB_53', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGTTAGTGTCTTATCGGCAAGCGCTACCCA'), id='NDB_54', name='NDB_54', description='NDB_54', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCCTTCTCTAGTCGGCGTGATACATAGAG'), id='NDB_55', name='NDB_55', description='NDB_55', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACCAGTTAAGGATCGCCGAGCTAGTACAG'), id='NDB_56', name='NDB_56', description='NDB_56', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTCGTTTCGGGAACTCTTTACGCGACCAAG'), id='NDB_57', name='NDB_57', description='NDB_57', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACCGTAGTGTAAAGGCACGCTCCGATGTGA'), id='NDB_58', name='NDB_58', description='NDB_58', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAACTATCTTGTTAGCCGTTGTAGCGCCTT'), id='NDB_59', name='NDB_59', description='NDB_59', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCGCTTGTTTATTCGGTGGGCCAGGTCTAG'), id='NDB_60', name='NDB_60', description='NDB_60', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCAAAGAGAGGGTTTACCGGCCTACTCCGA'), id='NDB_61', name='NDB_61', description='NDB_61', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACATTCAAAGTCGGATGCGCCGTCTTAGTG'), id='NDB_62', name='NDB_62', description='NDB_62', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTCATCACAAGGCGTATGATACTAGTAGC'), id='NDB_63', name='NDB_63', description='NDB_63', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGTTCCTTAATCTGACCGCACGGAATATG'), id='NDB_64', name='NDB_64', description='NDB_64', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCCAGGTATGTCGCGTCCGTTACAGTAAG'), id='NDB_65', name='NDB_65', description='NDB_65', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGAAACATAGTTCAACGCCTATTCGGGCG'), id='NDB_66', name='NDB_66', description='NDB_66', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCGAACGCGGGTTAGACCACCGTCTTAAT'), id='NDB_67', name='NDB_67', description='NDB_67', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACGTATCATCGGTAGACAGCTGCGAATGTC'), id='NDB_68', name='NDB_68', description='NDB_68', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCACAATAGAACTGGTACGGAGGCTCGCGT'), id='NDB_69', name='NDB_69', description='NDB_69', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCATATTAGTAAGCTACTTGCGGACTCGCA'), id='NDB_70', name='NDB_70', description='NDB_70', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGTATGGGACTATCGCTCACCGGTGCTAAT'), id='NDB_71', name='NDB_71', description='NDB_71', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTGTTTACGTTCGATGTATGCCGCCTTGG'), id='NDB_72', name='NDB_72', description='NDB_72', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAACTACGTGCCGGCTCTCGCTTATTTGAA'), id='NDB_73', name='NDB_73', description='NDB_73', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATCTATTGCGCTGGTCATCGTCCATACTAA'), id='NDB_74', name='NDB_74', description='NDB_74', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CAGTTGAATGGTCCGCTTCTATTGCGCAGG'), id='NDB_75', name='NDB_75', description='NDB_75', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ATGGTAATTTCCCTACGAACCTTGCGTTAC'), id='NDB_76', name='NDB_76', description='NDB_76', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGGTTGTATCGGCGGCCCGTCATTGCTATT'), id='NDB_77', name='NDB_77', description='NDB_77', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGGCTATCGTGCGGACGAGACAGCATAGAT'), id='NDB_78', name='NDB_78', description='NDB_78', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCTAGAGTCGAGCTAGCATAGGAGTTCAC'), id='NDB_79', name='NDB_79', description='NDB_79', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTCTGATTTCGCTACACGAGCAGGTCTAC'), id='NDB_80', name='NDB_80', description='NDB_80', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGATTAGTACCCGCGCCTTTGAAGCTTATG'), id='NDB_81', name='NDB_81', description='NDB_81', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCGTTATTGCGCTTTCGCAGTATTGTTTC'), id='NDB_82', name='NDB_82', description='NDB_82', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCCTATGCGCGAGTTATTTACGCTCGTAT'), id='NDB_83', name='NDB_83', description='NDB_83', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTGAACTCTGCTGCGCCGTTAACTAGTCTT'), id='NDB_84', name='NDB_84', description='NDB_84', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCCATTGCATTTACGGACGGCTACAGTCT'), id='NDB_85', name='NDB_85', description='NDB_85', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CATTAGGAGCGATGACCGTAACCTATCTGA'), id='NDB_86', name='NDB_86', description='NDB_86', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGCTAACGTTGCTAGCGCTCCGAGGTTTAG'), id='NDB_87', name='NDB_87', description='NDB_87', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACGCAGGTCGGTTCATAACGTCCTTAGCTA'), id='NDB_88', name='NDB_88', description='NDB_88', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTCTGTATATGTTCGCGCACGGTGTAGAC'), id='NDB_89', name='NDB_89', description='NDB_89', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGAGAAATTTCGCTGTCTACGTACTGCTAC'), id='NDB_90', name='NDB_90', description='NDB_90', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGTAATAGCGTACACTTTGCGATCGGCCG'), id='NDB_91', name='NDB_91', description='NDB_91', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTGTAAACAAGTACACGCCGTCATTCTCAT'), id='NDB_92', name='NDB_92', description='NDB_92', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGTGTAGTTCGGTACTATCGCCAGCATACA'), id='NDB_93', name='NDB_93', description='NDB_93', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GAGTAGTACCACATTCGGCATTGCCTGTTA'), id='NDB_94', name='NDB_94', description='NDB_94', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTATATGAGGGTGCTCTACCGGGTTCACA'), id='NDB_95', name='NDB_95', description='NDB_95', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CTCGTACGGAGAGTTGACTACAACCCAGTT'), id='NDB_96', name='NDB_96', description='NDB_96', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AAAGTTTGTTCGTTCCCGTCGCTGAACTTC'), id='NDB_97', name='NDB_97', description='NDB_97', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GCTCAAGCTAAGTGCGCCGTACGGATAAGT'), id='NDB_98', name='NDB_98', description='NDB_98', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCGAATTTATGCGTGCGCTGTGCCACTAGA'), id='NDB_99', name='NDB_99', description='NDB_99', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCTAGGCGTATATTGAGTTAGACACGTCC'), id='NDB_100', name='NDB_100', description='NDB_100', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGCCTAATAGGTCGGTCCGCGGGTCATTAT'), id='NDB_101', name='NDB_101', description='NDB_101', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCGATGCGCAATAGTGCCCGTGTTTAATAA'), id='NDB_102', name='NDB_102', description='NDB_102', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGTCAAAGATGTTCGACCTACGCTCGTAGA'), id='NDB_103', name='NDB_103', description='NDB_103', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CACGATACGGGTATAGTCGCGTCGAGTATC'), id='NDB_104', name='NDB_104', description='NDB_104', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTTTACGGACTTCCCACCGATACATAAGAT'), id='NDB_105', name='NDB_105', description='NDB_105', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('ACTCAAGGATCGGGCCGTATGTCTAGTTGC'), id='NDB_106', name='NDB_106', description='NDB_106', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CCCATGGACAACTGTCGGGTAGGTATCCTT'), id='NDB_107', name='NDB_107', description='NDB_107', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCCGTTTGAAGGATAGGTCGGACCAGGCCA'), id='NDB_108', name='NDB_108', description='NDB_108', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('AGGCTCTATGACCGTTTACGTTCTTATCGA'), id='NDB_109', name='NDB_109', description='NDB_109', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('CGTGAATATATCGTCGGGCTCGTCCATCTT'), id='NDB_110', name='NDB_110', description='NDB_110', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TGCCATTTGTCGACCGATTCATGCGAATAT'), id='NDB_111', name='NDB_111', description='NDB_111', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GGTTAGCGGAATCCTCCACCGAGATCGATG'), id='NDB_112', name='NDB_112', description='NDB_112', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCACTAGGTCGAAGTGTCGCGTAAACTGGC'), id='NDB_113', name='NDB_113', description='NDB_113', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TCGAAAGTTAAGCACTCTCGCGATCGATGA'), id='NDB_114', name='NDB_114', description='NDB_114', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TATGACCGACTATCGTAATGCCTTACGCGG'), id='NDB_115', name='NDB_115', description='NDB_115', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('GTTCTCATCGTAGGTGCATATTCGGTGCCG'), id='NDB_116', name='NDB_116', description='NDB_116', dbxrefs=[]),\n",
       "  SeqRecord(seq=Seq('TTCCAAAGTGTCGATTTACGGGTAGCCGCA'), id='NDB_117', name='NDB_117', description='NDB_117', dbxrefs=[])]}"
      ]
     },
     "execution_count": 114,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "readout_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "id": "86e337a4",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "save readout usage dict\n"
     ]
    }
   ],
   "source": [
    "# save\n",
    "readout_usage_file = os.path.join(library_folder, 'readout_usage.pkl')\n",
    "if not os.path.isfile(readout_usage_file) or overwrite_readout_usage:\n",
    "    print('save readout usage dict')\n",
    "    pickle.dump(readout_dict, open(readout_usage_file, 'wb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3d1eeb3b",
   "metadata": {},
   "source": [
    "<a id='3.3'></a>\n",
    "## 3.3 assemble probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 115,
   "id": "043d7089",
   "metadata": {},
   "outputs": [],
   "source": [
    "reload(library_tools.assemble)\n",
    "from ImageAnalysis3.library_tools.assemble import Assemble_probes\n",
    "\n",
    "# params\n",
    "assemble_params = {\n",
    "    'primer_len':20,\n",
    "    'num_readout_per_probe':4, \n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 116,
   "id": "d7c74256",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Assemble probes by given target sequences, readouts and primers.\n",
      "-- included readout types: ['u']\n",
      "--- assemblying 120 probes in region: 383\n",
      "--- assemblying 120 probes in region: 384\n",
      "--- assemblying 120 probes in region: 385\n",
      "--- assemblying 100 probes in region: 386\n",
      "--- assemblying 120 probes in region: 387\n",
      "--- assemblying 70 probes in region: 388\n",
      "--- assemblying 120 probes in region: 389\n",
      "--- assemblying 120 probes in region: 390\n",
      "--- assemblying 120 probes in region: 391\n",
      "--- assemblying 120 probes in region: 392\n",
      "--- assemblying 79 probes in region: 393\n",
      "--- assemblying 120 probes in region: 394\n",
      "--- assemblying 120 probes in region: 395\n",
      "--- assemblying 120 probes in region: 396\n",
      "--- assemblying 120 probes in region: 397\n",
      "--- assemblying 117 probes in region: 398\n",
      "--- assemblying 120 probes in region: 399\n",
      "--- assemblying 120 probes in region: 400\n",
      "--- assemblying 120 probes in region: 401\n",
      "--- assemblying 120 probes in region: 402\n",
      "--- assemblying 120 probes in region: 403\n",
      "--- assemblying 120 probes in region: 404\n",
      "--- assemblying 120 probes in region: 405\n",
      "--- assemblying 120 probes in region: 406\n",
      "--- assemblying 120 probes in region: 407\n",
      "--- assemblying 120 probes in region: 408\n",
      "--- assemblying 120 probes in region: 409\n",
      "--- assemblying 120 probes in region: 410\n",
      "--- assemblying 120 probes in region: 411\n",
      "--- assemblying 120 probes in region: 412\n",
      "--- assemblying 120 probes in region: 413\n",
      "--- assemblying 120 probes in region: 414\n",
      "--- assemblying 120 probes in region: 415\n",
      "--- assemblying 95 probes in region: 416\n",
      "--- assemblying 120 probes in region: 417\n",
      "--- assemblying 120 probes in region: 418\n",
      "--- assemblying 120 probes in region: 419\n",
      "--- assemblying 120 probes in region: 420\n",
      "--- assemblying 120 probes in region: 421\n",
      "--- assemblying 103 probes in region: 422\n",
      "--- assemblying 120 probes in region: 423\n",
      "--- assemblying 120 probes in region: 424\n",
      "--- assemblying 120 probes in region: 425\n",
      "--- assemblying 120 probes in region: 426\n",
      "--- assemblying 120 probes in region: 427\n",
      "--- assemblying 120 probes in region: 428\n",
      "--- assemblying 120 probes in region: 429\n",
      "--- assemblying 120 probes in region: 430\n",
      "--- assemblying 120 probes in region: 431\n",
      "--- assemblying 120 probes in region: 989\n",
      "--- assemblying 120 probes in region: 1010\n",
      "--- assemblying 120 probes in region: 1397\n",
      "--- assemblying 120 probes in region: 1398\n",
      "--- assemblying 120 probes in region: 1399\n",
      "--- assemblying 120 probes in region: 1400\n",
      "--- assemblying 120 probes in region: 1401\n",
      "--- assemblying 120 probes in region: 1402\n",
      "--- assemblying 120 probes in region: 1403\n",
      "--- assemblying 120 probes in region: 1404\n",
      "--- assemblying 120 probes in region: 1405\n",
      "--- assemblying 120 probes in region: 1406\n",
      "--- assemblying 120 probes in region: 1407\n",
      "--- assemblying 120 probes in region: 1408\n",
      "--- assemblying 120 probes in region: 1409\n",
      "--- assemblying 120 probes in region: 1410\n",
      "--- assemblying 120 probes in region: 1411\n",
      "--- assemblying 120 probes in region: 1412\n",
      "--- assemblying 120 probes in region: 1413\n",
      "--- assemblying 120 probes in region: 1414\n",
      "--- assemblying 120 probes in region: 1415\n",
      "--- assemblying 120 probes in region: 1416\n",
      "--- assemblying 120 probes in region: 1417\n",
      "--- assemblying 120 probes in region: 1418\n",
      "--- assemblying 120 probes in region: 1419\n",
      "--- assemblying 120 probes in region: 1420\n",
      "--- assemblying 120 probes in region: 1421\n",
      "--- assemblying 120 probes in region: 1422\n",
      "--- assemblying 120 probes in region: 1423\n",
      "--- assemblying 120 probes in region: 1424\n",
      "--- assemblying 120 probes in region: 1425\n",
      "--- assemblying 120 probes in region: 1426\n",
      "--- assemblying 120 probes in region: 1427\n",
      "--- assemblying 120 probes in region: 1428\n",
      "--- assemblying 120 probes in region: 1429\n",
      "--- assemblying 120 probes in region: 1430\n",
      "--- assemblying 120 probes in region: 1431\n",
      "--- assemblying 120 probes in region: 1432\n",
      "--- assemblying 120 probes in region: 1433\n",
      "--- assemblying 120 probes in region: 1434\n",
      "--- assemblying 120 probes in region: 1435\n",
      "--- assemblying 120 probes in region: 1436\n",
      "--- assemblying 120 probes in region: 1437\n",
      "--- assemblying 120 probes in region: 1438\n",
      "--- assemblying 120 probes in region: 1439\n",
      "--- assemblying 120 probes in region: 1440\n",
      "--- assemblying 120 probes in region: 1441\n",
      "-- 11364 probes assembled in total.\n",
      "-- saving 11364 probes into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\candidate_probes.fasta\n",
      "-- saving readout_summary into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\readout_summary.pkl\n"
     ]
    }
   ],
   "source": [
    "# run assembly\n",
    "cand_probes, readout_summary = Assemble_probes(library_folder, filtered_pb_dict, \n",
    "                                               region_2_readout_dict, readout_dict, primers, \n",
    "                                               primer_len=assemble_params['primer_len'], \n",
    "                                               num_readout_per_probe=assemble_params['num_readout_per_probe'], \n",
    "                                               unique_readout_per_probe=True,\n",
    "                                               rc_targets=False, overwrite=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9265b92d",
   "metadata": {},
   "source": [
    "# 4.Check quality"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8dca4823",
   "metadata": {},
   "source": [
    "\n",
    "## 4.1 Load probes\n",
    "\n",
    "Checking procedures could restart from here"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 117,
   "id": "a651e7eb",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total probe loaded: 11364\n"
     ]
    }
   ],
   "source": [
    "# folder for this library\n",
    "# biopython imports\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "\n",
    "# candidate full-length probe filename\n",
    "candidate_basename = 'candidate_probes.fasta' # old version\n",
    "# load full probes\n",
    "full_records = []\n",
    "with open(os.path.join(library_folder, candidate_basename), 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        full_records.append(record)\n",
    "\n",
    "print(f\"Total probe loaded: {len(full_records)}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fe532250",
   "metadata": {},
   "source": [
    "### Manually input the following info to double check!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "id": "cf1d4bf8",
   "metadata": {},
   "outputs": [],
   "source": [
    "num_readout_per_probe = 4\n",
    "primer_len=20\n",
    "target_len=42\n",
    "min_num_probes = 21\n",
    "# merfish related\n",
    "num_on_bits = 2"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2da410b1",
   "metadata": {},
   "source": [
    "### 4.1.1 check primer usage"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "id": "3cba3f0b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Checking primer usage, total probes: 11364\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "primer_check = check._check_primer_usage(full_records, primers[0], primers[1], primer_len=primer_len)\n",
    "print(primer_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "30ce1025",
   "metadata": {},
   "source": [
    "### 4.1.2 check region probe number"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 121,
   "id": "761720bd",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: 383 -> 120\n",
      "gene: 384 -> 120\n",
      "gene: 385 -> 120\n",
      "gene: 386 -> 100\n",
      "gene: 387 -> 120\n",
      "gene: 388 -> 70\n",
      "gene: 389 -> 120\n",
      "gene: 390 -> 120\n",
      "gene: 391 -> 120\n",
      "gene: 392 -> 120\n",
      "gene: 393 -> 79\n",
      "gene: 394 -> 120\n",
      "gene: 395 -> 120\n",
      "gene: 396 -> 120\n",
      "gene: 397 -> 120\n",
      "gene: 398 -> 117\n",
      "gene: 399 -> 120\n",
      "gene: 400 -> 120\n",
      "gene: 401 -> 120\n",
      "gene: 402 -> 120\n",
      "gene: 403 -> 120\n",
      "gene: 404 -> 120\n",
      "gene: 405 -> 120\n",
      "gene: 406 -> 120\n",
      "gene: 407 -> 120\n",
      "gene: 408 -> 120\n",
      "gene: 409 -> 120\n",
      "gene: 410 -> 120\n",
      "gene: 411 -> 120\n",
      "gene: 412 -> 120\n",
      "gene: 413 -> 120\n",
      "gene: 414 -> 120\n",
      "gene: 415 -> 120\n",
      "gene: 416 -> 95\n",
      "gene: 417 -> 120\n",
      "gene: 418 -> 120\n",
      "gene: 419 -> 120\n",
      "gene: 420 -> 120\n",
      "gene: 421 -> 120\n",
      "gene: 422 -> 103\n",
      "gene: 423 -> 120\n",
      "gene: 424 -> 120\n",
      "gene: 425 -> 120\n",
      "gene: 426 -> 120\n",
      "gene: 427 -> 120\n",
      "gene: 428 -> 120\n",
      "gene: 429 -> 120\n",
      "gene: 430 -> 120\n",
      "gene: 431 -> 120\n",
      "gene: 989 -> 120\n",
      "gene: 1010 -> 120\n",
      "gene: 1397 -> 120\n",
      "gene: 1398 -> 120\n",
      "gene: 1399 -> 120\n",
      "gene: 1400 -> 120\n",
      "gene: 1401 -> 120\n",
      "gene: 1402 -> 120\n",
      "gene: 1403 -> 120\n",
      "gene: 1404 -> 120\n",
      "gene: 1405 -> 120\n",
      "gene: 1406 -> 120\n",
      "gene: 1407 -> 120\n",
      "gene: 1408 -> 120\n",
      "gene: 1409 -> 120\n",
      "gene: 1410 -> 120\n",
      "gene: 1411 -> 120\n",
      "gene: 1412 -> 120\n",
      "gene: 1413 -> 120\n",
      "gene: 1414 -> 120\n",
      "gene: 1415 -> 120\n",
      "gene: 1416 -> 120\n",
      "gene: 1417 -> 120\n",
      "gene: 1418 -> 120\n",
      "gene: 1419 -> 120\n",
      "gene: 1420 -> 120\n",
      "gene: 1421 -> 120\n",
      "gene: 1422 -> 120\n",
      "gene: 1423 -> 120\n",
      "gene: 1424 -> 120\n",
      "gene: 1425 -> 120\n",
      "gene: 1426 -> 120\n",
      "gene: 1427 -> 120\n",
      "gene: 1428 -> 120\n",
      "gene: 1429 -> 120\n",
      "gene: 1430 -> 120\n",
      "gene: 1431 -> 120\n",
      "gene: 1432 -> 120\n",
      "gene: 1433 -> 120\n",
      "gene: 1434 -> 120\n",
      "gene: 1435 -> 120\n",
      "gene: 1436 -> 120\n",
      "gene: 1437 -> 120\n",
      "gene: 1438 -> 120\n",
      "gene: 1439 -> 120\n",
      "gene: 1440 -> 120\n",
      "gene: 1441 -> 120\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "reg_size_dic, len_check = check._check_region_size(full_records, \n",
    "                                                   species_marker = 'gene_',\n",
    "                                                   min_size=min_num_probes)\n",
    "print(len_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "id": "95f60f31",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7t3QuwVtV99/F/wBoYi8hwMXIrKGGAKrRawCsEtFwMqTYiNaIEBSkNFQwIvAjKRVQuIxeVi5aLKCO0FAgBsSIEBxsEYsEMBaoioFRQEUFEQa7v/Neb53nPgXNYD0uXm//e3z3DJHLWetbZn9868GOf/ezzg5MnT54UDgQQQAABBBBAAAEEMiLwAwpwRpLmNBFAAAEEEEAAAQScAAWYjYAAAggggAACCCCQKQEKcKbi5mQRQAABBBBAAAEEKMDsAQQQQAABBBBAAIFMCVCAMxU3J4sAAggggAACCCBAAWYPIIAAAggggAACCGRKgAKcqbg5WQQQQAABBBBAAAEKMHsAAQQQQAABBBBAIFMCFOBMxc3JIoAAAggggAACCFCA2QMIIIAAAggggAACmRKgAGcqbk4WAQQQQAABBBBAgALMHkAAAQQQQAABBBDIlAAFOFNxc7IIIIAAAggggAACFGD2AAIIIIAAAggggECmBCjAmYqbk0UAAQQQQAABBBCgALMHEEAAAQQQQAABBDIlQAHOVNycLAIIIIAAAggggAAFmD2AAAIIIIAAAgggkCkBCnCm4uZkEUAAAQQQQAABBCjA7AEEEEAAAQQQQACBTAlQgDMVNyeLAAIIIIAAAgggQAFmDyCAAAIIIIAAAghkSoACnKm4OVkEEEAAAQQQQAABCjB7AAEEEEAAAQQQQCBTAhTgTMXNySKAAAIIIIAAAghQgNkDCCCAAAIIIIAAApkSoABnKm5OFgEEEEAAAQQQQIACzB5AAAEEEEAAAQQQyJQABThTcXOyCCCAAAIIIIAAAhRg9gACCCCAAAIIIIBApgQowJmKm5NFAAEEEEAAAQQQoACzBxBAAAEEEEAAAQQyJUABzlTcnCwCCCCAAAIIIIAABZg9gAACCCCAAAIIIJApAQpwpuLmZBFAAAEEEEAAAQQowOwBBBBAAAEEEEAAgUwJUIAzFTcniwACCCCAAAIIIEABZg8ggAACCCCAAAIIZEqAApypuDlZBBBAAAEEEEAAAQowewABBBBAAAEEEEAgUwIU4EzFzckigAACCCCAAAIIUIDZAwgggAACCCCAAAKZEqAAZypuThYBBBBAAAEEEECAAsweQAABBBBAAAEEEMiUAAU4U3FzsggggAACCCCAAAIUYPYAAggggAACCCCAQKYEKMCZipuTRQABBBBAAAEEEKAAswcQQAABBBBAAAEEMiVAAc5U3JwsAggggAACCCCAAAWYPYAAAggggAACCCCQKQEKcKbi5mQRQAABBBBAAAEEKMDsAQQQQAABBBBAAIFMCVCAMxU3J4sAAggggAACCCBAAWYPIIAAAggggAACCGRKgAKcqbg5WQQQQAABBBBAAAEKMHsAAQQQQAABBBBAIFMCFOBMxc3JIoAAAggggAACCFCA2QMIIIAAAggggAACmRKgAGcqbk4WAQQQQAABBBBAgALMHkAAAQQQQAABBBDIlAAFOFNxc7IIIIAAAggggAACFOCIe+Dw4cOyceNGqVq1qpx33nkRV+KlEUAAAQQQQACB4gLHjh2TPXv2yBVXXCHlypWDp4gABTjidvjDH/4gzZo1i7gCL40AAggggAACCJxZYN26ddK0aVOYKMDfzx7YsWOH1K1bV3TjXXLJJd/PoqyCAAIIIIAAAgiIyO7du92FuO3bt0udOnUwoQB/P3vgf//3f6VWrVqyc+dOqVmz5vezKKsggAACCCCAAAIiQg8pfRtwC0TELxE2XkRcXhoBBBBAAAEEzihAD6EAJ/IlwsZLhJ1FEUAAAQQQQIArwGfcA1wBjvglQgGOiMtLI4AAAggggABXgAP3AAU4EK6QaRTgQpQYgwACCCCAAAIxBOghpatSgGPsuD+9JhsvIi4vjQACCCCAAAJcAQ7cAxTgQLhCplGAC1FiDAIIIIAAAgjEEKCHcAU4xr7yviYbz0vEAAQQQAABBBCIJEAPoQBH2lpnflk2XiLsLIoAAggggAACPAXijHuAWyAifolQgCPi8tIIIIAAAgggcEYBeghXgBP5EmHjJcLOoggggAACCCDAFWCuACf1VUABTkqedRFAAAEEEECAHpLyK8Cff/65PPTQQ7JkyRI5cOCANG7cWEaNGiXXX399iWe+d+9e6d27tyxdutR9/Pbbb5fx48fLBRdckB8/b948GTp0qGzbtk3q168vY8aMkXbt2p3VVxMb76y4GIwAAggggAAC36EAPSTlBbhNmzby6aefyjPPPCNVq1aVSZMmybRp02T9+vXSoEGD086+VatWcujQIZk8ebLs379f7r33XmnZsqXMmjXLjV25cqW0bdtWxo0bJzfddJNMnz5dnn76admwYYM0bNiw4K3JxiuYioEIIIAAAggg8B0L0ENSXIC3bt0qP/7xj+X3v/+9XHvtte5MT5486a7a/uIXv5ARI0YUO/s333zTjduyZUu+HC9btsxd3dWNUr16dVd+K1WqJHPnzs3P1Tl6ZXnq1KkFb082XsFUDEQAAQQQQACB71iAHpLiAqxXcFevXi033nij/PCHP8yfab169Vyp1avCRQ+9lWHChAmya9eu/G8fOXJEypcvL3PmzJGOHTvKhRde6K7+9ujRIz9myJAhsmDBAtm8eXPB25ONVzAVAxFAAAEEEEDgOxagh6S4AJd0anr/bqdOneS3v/2t/OxnPys2RO/9Xbt2rftV9KhWrZoMGDDA3Q5RuXJld39w+/bt80P0dgn9+MGDBwvenmy8gqkYiAACCCCAAALfsQA9JEMFWG+F0Cu/ekX4N7/5zWln3r17d3n33Xdl1apVxT5Wu3Ztd8W3a9euUqtWLVmxYoW0bt06P2bGjBnu48eOHStVU9+Ap79yx+7du6VZs2ayc+dOqVmz5ne8rXk5BBBAAAEEEECgdAEKcEYK8KJFi+TOO++Uq6++2j0RQm9rOPW4//77Zd26dSVeAR40aJB06dJFqlSpUuIVYP34F198UarmsGHDZPjw4ad9nALMH08IIIAAAggUF6jzf16OQrJj1E+jvK7FF6UAZ6AA672+ffr0kZ///Ocye/bsYvcDFz19vQdYn+igpTR35O4B1je96T3AFSpUkIkTJ0q3bt3yY/Qe4IULF8qmTZtK1eQKsMU/HvicEUAAAQSSEKAAx1enAKe8AE+ZMkV+9atfuWf76vN8y5QpU+oZr1mzRq655hp57733RN8op8err74qN998s3z44YdSo0YNdwuFPk7txRdfzL8OT4GI/4XKCggggAAC2RGgAMfPmgKc4gKs9/Nefvnl0qFDB/dc36KH3gLx53/+57Jnzx6pWLGiuyVCH5F2ww03yOHDh0WLs76pTd/49pOf/ERmzpzpputj0bQQ69Vi/V+9/1evMJf2XOHSeNl48b+4WQEBBBBAwKYABTh+bvSQFBfgxx9/XAYPHlziGf7yl78UvS+3bt26rtzqG9z00B+a0atXL3nllVdcKdafBKePPStXrlz+dfTqrz5DWDdPo0aNZOzYscXeFFfItmXjFaLEGAQQQACBLApQgOOnTg9JcQGOv33CV2DjhdsxEwEEEEAg3QIU4Pj50kMowPF3WQkrsPESYWdRBBBAAAEDAhTg+CHRQyjA8XcZBTgRYxZFAAEEELApQAGOnxsFmAIcf5dRgBMxZlEEEEAAAZsCFOD4uVGAKcDxdxkFOBFjFkUAAQQQsClAAY6fGwWYAhx/l1GAEzFmUQQQQAABmwIU4Pi5UYApwPF3GQU4EWMWRQABBBCwKUABjp8bBZgCHH+XUYATMWZRBBBAAAGbAhTg+LlRgCnA8XcZBTgRYxZFAAEEELApQAGOnxsFmAIcf5dRgBMxZlEEEEAAAZsCFOD4uVGAKcDxdxkFOBFjFkUAAQQQsClAAY6fGwWYAhx/l1GAEzFmUQQQQAABmwIU4Pi5UYApwPF3GQU4EWMWRQABBBCwKUABjp8bBZgCHH+XUYATMWZRBBBAAAGbAhTg+LlRgCnA8XcZBTgRYxZFAAEEELApQAGOnxsFmAIcf5dRgBMxZlEEEEAAAZsCFOD4uVGAKcDxdxkFOBFjFkUAAQQQsClAAY6fGwWYAhx/l1GAEzFmUQQQQAABmwIU4Pi5UYApwPF3GQU4EWMWRQABBBCwKUABjp8bBZgCHH+XUYATMWZRBBBAAAGbAhTg+LlRgCnA8XcZBTgRYxZFAAEEELApQAGOnxsFmAIcf5dRgBMxZlEEEEAAAZsCFOD4uVGAKcDxdxkFOBFjFkUAAQQQsClAAY6fGwWYAhx/l1GAEzFmUQQQQAABmwIU4Pi5UYApwPF3GQU4EWMWRQABBBCwKUABjp8bBZgCHH+XUYATMWZRBBBAAAGbAhTg+LlRgCnA8XcZBTgRYxZFAAEEELApQAGOnxsFmAIcf5dRgBMxZlEEEEAAAZsCFOD4uVGAKcDxdxkFOBFjFkUAAQQQsClAAY6fGwWYAhx/l1GAEzFmUQQQQAABmwIU4Pi5UYApwPF3GQU4EWMWRQABBBCwKUABjp8bBZgCHH+XUYATMWZRBBBAAAGbAhTg+LlRgCnA8XcZBTgRYxZFAAEEELApQAGOnxsFmAIcf5dRgBMxZlEEEEAAAZsCFOD4uVGAKcDxdxkFOBFjFkUAAQQQsClAAY6fGwWYAhx/l1GAEzFmUQQQQAABmwIU4Pi5UYApwPF3GQU4EWMWRQABBBCwKUABjp8bBZgCHH+XUYATMWZRBBBAAAGbAhTg+LlRgDNUgEeOHCnLly+X119/vcSzHjZsmAwfPrzEj91zzz0yY8YM97G6devKjh07io3r3LmzzJ49u+Ady8YrmIqBCCCAAAIZE6AAxw+cHpKRAjxhwgTp27evtGjRotQCfPDgQdFfRY/p06fLY489Jm+++aY0adJEvvzyS6lYsaIsXrxYrrrqqvzQ8uXLu98v9GDjFSrFOAQQQACBrAlQgOMnTg9JeQH+6KOPpHv37vLGG29IrVq15OKLLy61AJ9K8f7770vjxo3lySeflJ49e7oPr169Wq677jrZt2+fXHTRRcE7lI0XTMdEBBBAAIGUC1CA4wdMD0l5AdYrtXPmzBG9/WHEiBHu1oXSboE4leLWW2+V3bt3u6u/ZcqUcR+eOnWqe51du3Z9q93JxvtWfExGAAEEEEixAAU4frj0kJQX4KKn17Vr14IL8FtvvSVNmzaV3/3ud9KqVav8y/Tq1UuWLVvmrgzr1eBq1aqJ3h/cu3fvfEkuifTAgQOiv3KHFutmzZrJzp07pWbNmvF3OisggAACCCBgRIACHD8oCjAFuEQBfVOb3gKxZs2aYh9v2bKlbNmyRSZNmiQNGjSQVatWycCBA939xXpluLSjtDfYUYDjf5GzAgIIIICALQEKcPy8KMAU4NMEvvrqK6lSpYpMnjzZXd0tehw9elQOHz4sFSpUyP/2qFGj3C0WeoU3d6vEqS/KFeD4X8ysgAACCCCQDgEKcPwcKcAU4NME5s+fL3feead88sknBb3R7eWXX5YOHTrIZ599JpUrVy5o17LxCmJiEAIIIIBABgUowPFDp4dQgE8T+PWvf+1ufdA3vxU9Tpw4IZdeeqncd999Mnjw4PyH9PYGfXPcxx9/XPCOZeMVTMVABBBAAIGMCVCA4wdOD8lwAT5+/Ljs2bPHPb9Xn+ObO1q3bi316tWT55577jSd/v37u9+fNm2aXHnllbJixQp54IEHZOLEia4YF3qw8QqVYhwCCCCAQNYEKMDxE6eHZLgA6yPR9Ke6zZw5U/QJEbmjUaNGcsstt8gTTzxxms6xY8dk9OjR7qfC6ebR+f369Tur8qsvysaL/8XNCggggAACNgUowPFzo4dkqADH306Fr8DGK9yKkQgggAAC2RKgAMfPmx5CAY6/y0pYgY2XCDuLIoAAAggYEKAAxw+JHkIBjr/LKMCJGLMoAggggIBNAQpw/NwowBTg+LuMApyIMYsigAACCNgUoADHz40CTAGOv8sowIkYsygCCCCAgE0BCnD83CjAFOD4u4wCnIgxiyKAAAII2BSgAMfPjQJMAY6/yyjAiRizKAIIIICATQEKcPzcKMAU4Pi7jAKciDGLIoAAAgjYFKAAx8+NAkwBjr/LKMCJGLMoAggggIBNAQpw/NwowBTg+LuMApyIMYsigAACCNgUoADHz40CTAGOv8sowIkYsygCCCCAgE0BCnD83CjAFOD4u4wCnIgxiyKAAAII2BSgAMfPjQJMAY6/yyjAiRizKAIIIICATQEKcPzcKMAU4Pi7jAKciDGLIoAAAgjYFKAAx8+NAkwBjr/LKMCJGLMoAggggIBNAQpw/NwowBTg+LuMApyIMYsigAACCNgUoADHz40CTAGOv8sowIkYsygCCCCAgE0BCnD83CjAFOD4u4wCnIgxiyKAAAII2BSgAMfPjQJMAY6/yyjAiRizKAIIIICATQEKcPzcKMAU4Pi7jAKciDGLIoAAAgjYFKAAx8+NAkwBjr/LKMCJGLMoAggggIBNAQpw/NwowBTg+LuMApyIMYsigAACCNgUoADHz40CTAGOv8sowIkYsygCCCCAgE0BCnD83CjAFOD4u4wCnIgxiyKAAAII2BSgAMfPjQJMAY6/yyjAiRizKAIIIICATQEKcPzcKMAU4Pi7jAKciDGLIoAAAgjYFKAAx8+NAkwBjr/LKMCJGLMoAggggIBNAQpw/NwowBTg+LuMApyIMYsigAACCNgUoADHz40CTAGOv8sowIkYsygCCCCAgE0BCnD83CjAFOD4u4wCnIgxiyKAAAII2BSgAMfPjQJMAY6/yyjAiRizKAIIIICATQEKcPzcKMAU4Pi7jAKciDGLIoAAAgjYFKAAx8+NAkwBjr/LKMCJGLMoAggggIBNAQpw/NwowBTg+LuMApyIMYsigAACCNgUoADHz40CTAGOv8sowIkYsygCCCCAgE0BCnD83CjAFOD4u4wCnIgxiyKAAAII2BSgAMfPjQJMAY6/yyjAiRizKAIIIICATQEKcPzcKMAZKsAjR46U5cuXy+uvv17qWc+aNUu6du162sffe+89qVevnvv9efPmydChQ2Xbtm1Sv359GTNmjLRr1+6sdisb76y4GIwAAgggkCEBCnD8sOkhGSnAEyZMkL59+0qLFi3OWID79esn69evlzlz5hSTqVq1qpQtW1ZWrlwpbdu2lXHjxslNN90k06dPl6efflo2bNggDRs2LHjHsvEKpmIgAggggEDGBCjA8QOnh6S8AH/00UfSvXt3eeONN6RWrVpy8cUXn7EAt2nTRho1aiRamEs6tPxWqlRJ5s6dm//wtddeK40bN5apU6cWvGPZeAVTMRABBBBAIGMCFOD4gdNDUl6AFy9e7K7m6u0PI0aMkB07dpyxAP/oRz9y43r06HGazIkTJ+TCCy90V3+LfnzIkCGyYMEC2bx5c8E7lo1XMBUDEUAAAQQyJkABjh84PSTlBbjo6em9vWcqwHv27JFq1apJ586d5e2335bPP/9cmjdvLqNHj3b3+up/V65cWZYuXSrt27fPv/TkyZNlwIABcvDgwYJ3LBuvYCoGIoAAAghkTIACHD9weggFOC+g9/e2bt1aunTpIn369HGF9tFHH5U//vGPsnHjRjl69Ki7jWLFihVuXO6YMWOGuyJ87NixUjUPHDgg+it37N69W5o1ayY7d+6UmjVrxt/prIAAAggggIARAQpw/KAowBTgYgL79u1z9/jmjq+++kpq167trvDqvcRVqlQp8QrwoEGD5IsvvihVc9iwYTJ8+PDTPk4Bjv9FzgoIIIAAArYEKMDx86IAU4C9u6xp06Zy9dVXy1NPPSUVKlSQiRMnSrdu3fLz9B7ghQsXyqZNm7gC7NVkAAIIIIAAAmcWoADH3yEUYApwXmDKlCny8MMPu9sSypcv735fb1vQ2x70PuCePXu65/3qI9FefPHF/DyeAhH/C5UVEEAAAQSyI0ABjp81BTjDBfj48eOib3yrWLGiK7xafJs0aSI33nij6C0Lhw4dEr21Yfv27e4eYB2zbNkyufnmm90Pv9D/1ft/n3nmGffs4AYNGhS8Y9l4BVMxEAEEEEAgYwIU4PiB00MyXID1iRB169aVmTNn5n/6mz79YeDAgbJ27Vo5efKk6HOB9bFnehU4d+jVX31Umm4efWbw2LFji70prpBty8YrRIkxCCCAAAJZFKAAx0+dHpKhAhx/OxW+AhuvcCtGIoAAAghkS4ACHD9veggFOP4uK2EFNl4i7CyKAAIIIGBAgAIcPyR6CAU4/i6jACdizKIIIIAAAjYFKMDxc6MAU4Dj7zIKcCLGLIoAAgggYFOAAhw/NwowBTj+LqMAJ2LMoggggAACNgUowPFzowBTgOPvMgpwIsYsigACCCBgU4ACHD83CjAFOP4uowAnYsyiCCCAAAI2BSjA8XOjAFOA4+8yCnAixiyKAAIIIGBTgAIcPzcKMAU4/i6jACdizKIIIIAAAjYFKMDxc6MAU4Dj7zIKcCLGLIoAAgggYFOAAhw/NwowBTj+LqMAJ2LMoggggAACNgUowPFzowBTgOPvMgpwIsYsigACCCBgU4ACHD83CjAFOP4uowAnYsyiCCCAAAI2BSjA8XOjAFOA4+8yCnAixiyKAAIIIGBTgAIcPzcKMAU4/i6jACdizKIIIIAAAjYFKMDxc6MAU4Dj7zIKcCLGLIoAAgggYFOAAhw/NwowBTj+LqMAJ2LMoggggAACNgUowPFzowBTgOPvMgpwIsYsigACCCBgU4ACHD83CjAFOP4uowAnYsyiCCCAAAI2BSjA8XOjAFOA4+8yCnAixiyKAAIIIGBTgAIcPzcKMAU4/i6jACdizKIIIIAAAjYFKMDxc6MAU4Dj7zIKcCLGLIoAAgggYFOAAhw/NwowBTj+LqMAJ2LMoggggAACNgUowPFzowBTgOPvMgpwIsYsigACCCBgU4ACHD83CjAFOP4uowAnYsyiCCCAAAI2BSjA8XOjAFOA4+8yCnAixiyKAAIIIGBTgAIcPzcKMAU4/i6jACdizKIIIIAAAjYFKMDxc6MAU4Dj7zIKcCLGLIoAAgggYFOAAhw/NwowBTj+LqMAJ2LMoggggAACNgUowPFzowBTgOPvMgpwIsYsigACCCBgU4ACHD83CjAFOP4uowAnYsyiCCCAAAI2BSjA8XOjAFOA4+8yCnAixiyKAAIIIGBTgAIcPzcKMAU4/i6jACdizKIIIIAAAjYFKMDxc6MAU4Dj7zIKcCLGLIoAAgggYFOAAhw/NwowBTj+LqMAJ2LMoggggAACNgUowPFzowBTgOPvMgpwIsYsigACCCBgU4ACHD83CjAFOP4uowAnYsyiCCCAAAI2BSjA8XOjAGeoAI8cOVKWL18ur7/+eqlnvWnTJhkwYICsWbNGypYtKy1btpQnn3xSateunZ9Tt25d2bFjR7HX6Ny5s8yePbvgHcvGK5iKgQgggAACGROgAMcPnB6SkQI8YcIE6du3r7Ro0aLUArx37165/PLLXel9+OGH5fDhw/Lggw/KJ598IuvXr5dy5crJl19+KRUrVpTFixfLVVddldcrX768+/1CDzZeoVKMQwABBBDImgAFOH7i9JCUF+CPPvpIunfvLm+88YbUqlVLLr744lIL8LRp06Rfv36u8GrZ1SO3QVasWCGtW7eW1atXy3XXXSf79u2Tiy66KHiHsvGC6ZiIAAIIIJByAQpw/IDpISkvwHqlds6cOaK3P4wYMcLdulDaLRD6se3bt0urVq3yKrkNMm/ePOnYsaNMnTrVvc6uXbu+1e5k430rPiYjgAACCKRYgAIcP1x6SMoLcNHT69q16xkLcEkUvXr1khkzZsgHH3wg1apVE/3vZcuWSePGjd3VYP29e+65R3r37i1lypQpeMey8QqmYiACCCCAQMYEKMDxA6eHUIBLFXjqqaekT58+Mn78eHnggQfcOL0/eMuWLTJp0iRp0KCBrFq1SgYOHOjuL9Yrw6UdBw4cEP2VO3bv3i3NmjWTnTt3Ss2aNePvdFZAAAEEEEDAiAAFOH5QFGAK8GkCJ0+elEceecTdNjFo0CB5/PHH82OOHj3q3hxXoUKF/O+NGjXKjdWCW9pV4GHDhsnw4cNPW4sCHP+LnBUQQAABBGwJUIDj50UBpgAXE9CCq7c0vPTSSzJmzBj3FAjf8fLLL0uHDh3ks88+k8qVK5c4nCvAPkU+jgACCCCAwP8ToADH3wkUYApwMYE77rhDFixYIC+88ILo/y96nDhxQi699FK57777ZPDgwfkP6dVdfXPcxx9/XPCOZeMVTMVABBBAAIGMCVCA4wdOD8lwAT5+/Ljs2bPHPb9Xn+P7/PPPu6u/Y8eOlbvuuquYTG5M//795bnnnhN9ZNqVV14p+ng0vT944sSJrhgXerDxCpViHAIIIIBA1gQowPETp4dkuADrY8/0p7rNnDlT9AkRbdq0kddee61EkdyYY8eOyejRo92TIXTz6Hx9dvDZlF9dgI0X/4ubFRBAAAEEbApQgOPnRg/JUAGOv50KX4GNV7gVIxFAAAEEsiVAAY6fNz2EAhx/l5WwAhsvEXYWRQABBBAwIEABjh8SPYQCHH+XUYATMWZRBBBAAAGbAhTg+LlRgCnA8XcZBTgRYxZFAAEEELApQAGOnxsFmAIcf5dRgBMxZlEEEEAAAZsCFOD4uVGAKcDxdxkFOBFjFkUAAQQQsClAAY6fGwWYAhx/l1GAEzFmUQQQQAABmwIU4Pi5UYApwPF3GQU4EWMWRQABBBCwKUABjp8bBZgCHH+XUYATMWZRBBBAAAGbAhTg+LlRgCnA8XcZBTgRYxZFAAEEELApQAGOnxsFmAIcf5dRgBMxZlEEEEAAAZsCFOD4uVGAKcDxdxkFOBFjFkUAAQQQsClAAY6fGwWYAhx/l1GAEzFmUQQQQAChNzWAAAAgAElEQVQBmwIU4Pi5UYApwPF3GQU4EWMWRQABBBCwKUABjp8bBZgCHH+XUYATMWZRBBBAAAGbAhTg+LlRgCnA8XcZBTgRYxZFAAEEELApQAGOnxsFmAIcf5dRgBMxZlEEEEAAAZsCFOD4uVGAKcDxdxkFOBFjFkUAAQQQsClAAY6fGwWYAhx/l1GAEzFmUQQQQAABmwIU4Pi5UYApwPF3GQU4EWMWRQABBBCwKUABjp8bBZgCHH+XUYATMWZRBBBAAAGbAhTg+LlRgCnA8XcZBTgRYxZFAAEEELApQAGOnxsFmAIcf5dRgBMxZlEEEEAAAZsCFOD4uVGAKcDxdxkFOBFjFkUAAQQQsClAAY6fGwWYAhx/l1GAEzFmUQQQQAABmwIU4Pi5UYApwPF3GQU4EWMWRQABBBCwKUABjp8bBZgCHH+XUYATMWZRBBBAAAGbAhTg+LlRgCnA8XcZBTgRYxZFAAEEELApQAGOnxsFmAIcf5dRgBMxZlEEEEAAAZsCFOD4uVGAKcDxdxkFOBFjFkUAAQQQsClAAY6fGwWYAhx/l1GAEzFmUQQQQAABmwIU4Pi5UYApwPF3GQU4EWMWRQABBBCwKUABjp8bBZgCHH+XUYATMWZRBBBAAAGbAhTg+LlRgCnA8XcZBTgRYxZFAAEEELApQAGOnxsFmAIcf5dRgBMxZlEEEEAAAZsCFOD4uVGAKcDxdxkFOBFjFkUAAQQQsClAAY6fGwWYAhx/l1GAEzFmUQQQQAABmwIU4Pi5UYApwPF3GQU4EWMWRQABBBCwKUABjp8bBThDBXjkyJGyfPlyef3110s9671790rv3r1l6dKlbsztt98u48ePlwsuuCA/Z968eTJ06FDZtm2b1K9fX8aMGSPt2rU7q93KxjsrLgYjgAACCGRIgAIcP2x6SEYK8IQJE6Rv377SokWLMxbgVq1ayaFDh2Ty5Mmyf/9+uffee6Vly5Yya9YsJ7Vy5Upp27atjBs3Tm666SaZPn26PP3007JhwwZp2LBhwTuWjVcwFQMRQAABBDImQAGOHzg9JOUF+KOPPpLu3bvLG2+8IbVq1ZKLL7641AL85ptvyrXXXitbtmyRBg0aOJlly5a5q7u6UapXr+7Kb6VKlWTu3Ll5OZ3TuHFjmTp1asE7lo1XMBUDEUAAAQQyJkABjh84PSTlBXjx4sUyZ84c0dsfRowYITt27Ci1AOutDHqleNeuXXmVI0eOSPny5d1rdOzYUS688EJ39bdHjx75MUOGDJEFCxbI5s2bC96xbLyCqRiIAAIIIJAxAQpw/MDpISkvwEVPr2vXrmcswHrv79q1a92voke1atVkwIAB7naIypUru/uD27dvnx+it0voxw8ePFiq5oEDB0R/5Y7du3dLs2bNZOfOnVKzZs34O50VEEAAAQQQMCJAAY4fFAWYApwX0Fsl3n33XVm1alUxldq1a7srvlqg9TaKFStWSOvWrfNjZsyY4T5+7NixUjWHDRsmw4cPP+3jFOD4X+SsgAACtgRilZ8YCjtG/TTGy5p6TfIyFVf+k6UAU4DzAvfff7+sW7euxCvAgwYNki5dukiVKlVKvAKsH//iiy+4AmzzzwE+awQQOIcEKFTnUBgFfCrkVQDSOTiEAkwBzgvoPcD6RAe9Kps7cvcA65ve9B7gChUqyMSJE6Vbt275MXoP8MKFC2XTpk0Fb3E2XsFUDEQAgYwJUKhsBU5etvLKfbb0EApwXmDNmjVyzTXXyHvvvSf16tVzv//qq6/KzTffLB9++KHUqFHDPRGiatWq8uKLL+bn8RQIm1/8fNYIIHBuClCozs1cSvusyMtWXhRgf14/OHny5En/MDsjTn0T3PHjx2XPnj1SsWJF96QHPd0bbrhBDh8+LFOmTHFvatM3vv3kJz+RmTNnuhPVx6JpIdarxfq/ev/vM888I+vXr88/Oq0QEf7lVYgSYxBAIIsCFCpbqZOXrbwowP68Ul+A9ZFodevWdeVWy7Een376qfTq1UteeeUVV4r1J8HpY8/KlSuXF9Orv/pINS2xjRo1krFjxxZ7U5yfVtxcfUMdb4IrRIsxCCCQJQEKla20yctWXhRgf16pK8D+U/7+RlCAvz9rVkIAAVsCFCryiiXAUzv+vyw9pPRdRgGO9RUoXAGOSMtLI4CAcQEKsK0AyctWXlwB9udFAfYbBY/gX17BdExEAIGUC1CobAVMXrbyogD786IA+42CR1CAg+mYiAACKRegUNkKmLxs5UUB9udFAfYbBY+gAAfTMREBBFIuQKGyFTB52cqLAuzPiwLsNwoeQQEOpmMiAgikXIBCZStg8rKVFwXYnxcF2G8UPIICHEzHRAQQSLkAhcpWwORlKy8KsD8vCrDfKHgEBTiYjokIIJByAQqVrYDJy1ZeFGB/XhRgv1HwCApwMB0TEUAg5QIUKlsBk5etvCjA/rwowH6j4BEU4GA6JiKAQMoFKFS2AiYvW3lRgP15UYD9RsEjKMDBdExEAIGUC1CobAVMXrbyogD786IA+42CR1CAg+mYiAACKRegUNkKmLxs5UUB9udFAfYbBY+gAAfTMREBBFIuQKGyFTB52cqLAuzPiwLsNwoeQQEOpmMiAgikXIBCZStg8rKVFwXYnxcF2G8UPIICHEzHRAQQSLkAhcpWwORlKy8KsD8vCrDfKHgEBTiYjokIIJByAQqVrYDJy1ZeFGB/XhRgv1HwCApwMB0TEUAg5QIUKlsBk5etvCjA/rwowH6j4BEU4GA6JiKAQMoFKFS2AiYvW3lRgP15UYD9RsEjKMDBdExEAIGUC1CobAVMXrbyogD786IA+42CR1CAg+mYiAACKRegUNkKmLxs5UUB9udFAfYbBY+gAAfTMREBBFIuQKGyFTB52cqLAuzPiwLsNwoeQQEOpmMiAgikXIBCZStg8rKVFwXYnxcF2G8UPIICHEzHRAQQSLkAhcpWwORlKy8KsD8vCrDfKHgEBTiYjokIIJByAQqVrYDJy1ZeFGB/XhRgv1HwCApwMB0TEUAg5QIUKlsBk5etvCjA/rwowH6j4BEU4GA6JiKAQMoFKFS2AiYvW3lRgP15UYD9RsEjKMDBdExEAIGUC1CobAVMXrbyogD786IA+42CR1CAg+mYiAACKRegUNkKmLxs5UUB9udFAfYbBY+gAAfTMREBBFIuQKGyFTB52cqLAuzPiwLsNwoeQQEOpmMiAgikXIBCZStg8rKVFwXYnxcF2G8UPIICHEzHRAQQSLkAhcpWwORlKy8KsD8vCrDfKHgEBTiYjokIIJByAQqVrYDJy1ZeFGB/XhRgv1HwCApwMB0TEUAg5QIUKlsBk5etvCjA/rwowH6j4BEU4GA6JiKAQMoFKFS2AiYvW3lRgP15UYD9RsEjKMDBdExEAIGUC1CobAVMXrbyogD786IA+42CR1CAg+mYiAACKRegUNkKmLxs5UUB9udFAfYbBY+gAAfTMREBBFIuQKGyFTB52cqLAuzPKxUF+MSJEzJ8+HCZNm2a7Nu3T66//nqZPHmy1KtX7zSBYcOGubElHffcc4/MmDHDfahu3bqyY8eOYsM6d+4ss2fP9qv+aQQFuGAqBiKAQMYEKFS2AicvW3lRgP15paIAa6HVwjtz5kypUaOGDBgwQLZt2yabNm2S888/v5jCwYMHRX8VPaZPny6PPfaYvPnmm9KkSRP58ssvpWLFirJ48WK56qqr8kPLly/vfr/QgwJcqBTjEEAgawIUKluJk5etvCjA/rzMF+AjR45IlSpVZMyYMdKzZ093xvv375fq1au7q7l33HHHGRXef/99ady4sTz55JP5+atXr5brrrvOXU2+6KKL/IqljKAAB9MxEQEEUi5AobIVMHnZyosC7M/LfAFet26dNG/eXN555x2pX79+/oz1Nggttnpl+EzHrbfeKrt373ZXf8uUKeOGTp06VUaMGCG7du3yC55hBAX4W/ExGQEEUixAobIVLnnZyosC7M/LfAFesGCB3HbbbfL111+L3qKQOzp16uR+b8mSJaUqvPXWW9K0aVP53e9+J61atcqP69WrlyxbtswVaL0aXK1aNdH7g3v37p0vyX5aEQpwIUqMQQCBLApQqGylTl628qIA+/MyX4D1TWl33323HD9+vFg57dKli7uCu3z58lIV9E1tegvEmjVrio1p2bKlbNmyRSZNmiQNGjSQVatWycCBA6Vv377uynBpx4EDB0R/5Q69stysWTPZuXOn1KxZ058GIxBAAIGMCFCobAVNXrbyogD78zJfgOfPny8dO3Ys8QrwN998I4sWLSpR4auvvnL3DustEnp1t+hx9OhROXz4sFSoUCH/26NGjZKRI0e6gpu7VeLUFy7tCRMUYP9GZAQCCGRLgEJlK2/yspUXBdifl/kCnLsHeOvWrXLZZZflz1jvAdYnOuhV3JIOLc533nmnfPLJJwW90e3ll1+WDh06yGeffSaVK1cu8TW5AuzfcIxAAAEEVIBCZWsfkJetvCjA/rzMF2C9yqv36I4bN066devmzriQp0D8+te/drc+6Jvfih76TOFLL71U7rvvPhk8eHD+Q3p1V98c9/HHH/tV/zSCe4ALpmIgAghkTIBCZStw8rKVFwXYn5f5AqynqEX12WefdY89q1OnjvTv39/9EIuNGzdK2bJlZc+ePe75vUXfJNe6dWv3gzKee+6505R0vv6+/mCNK6+8UlasWCEPPPCATJw40RXjQg8KcKFSjEMAgawJUKhsJU5etvKiAPvzSkUB1jfAPfTQQ+4HYRw6dEhatGjhbn3QMqxFWH+qm36sa9eueZFGjRrJLbfcIk888cRpSseOHZPRo0e7Qq0lVuf369fvrMqvvigF2L8BGYEAAtkUoFDZyp28bOVFAfbnlYoC7D/NZEZQgJNxZ1UEEDj3BShU535GRT9D8rKVFwXYnxcF2G8UPIICHEzHRAQQSLkAhcpWwORlKy8KsD8vCrDfKHgEBTiYjokIIJByAQqVrYDJy1ZeFGB/XhRgv1HwCApwMB0TEUAg5QIUKlsBk5etvCjA/rwowH6j4BEU4GA6JiKAQMoFKFS2AiYvW3lRgP15UYD9RsEjKMDBdExEAIGUC1CobAVMXrbyogD786IA+42CR1CAg+mYiAACKRegUNkKmLxs5UUB9udFAfYbBY+gAAfTMREBBFIuQKGyFTB52cqLAuzPiwLsNwoeQQEOpmMiAgikXIBCZStg8rKVFwXYnxcF2G8UPIICHEzHRAQQSLkAhcpWwORlKy8KsD8vCrDfKHgEBTiYjokIIJByAQqVrYDJy1ZeFGB/XhRgv1HwCApwMB0TEUAg5QIUKlsBk5etvCjA/rwowH6j4BEU4GA6JiKAQMoFKFS2AiYvW3lRgP15UYD9RsEjKMDBdExEAIGUC1CobAVMXrbyogD786IA+42CR1CAg+mYiAACKRegUNkKmLxs5UUB9udFAfYbBY+gAAfTMREBBFIuQKGyFTB52cqLAuzPiwLsNwoeQQEOpmMiAgikXIBCZStg8rKVFwXYnxcF2G8UPIICHEzHRAQQSLkAhcpWwORlKy8KsD8vCrDfKHgEBTiYjokIIJByAQqVrYDJy1ZeFGB/XhRgv1HwCApwMB0TEUAg5QIUKlsBk5etvCjA/rwowH6j4BEU4GA6JiKAQMoFKFS2AiYvW3lRgP15UYD9RsEjKMDBdExEAIGUC1CobAVMXrbyogD786IA+42CR1CAg+mYiAACKRegUNkKmLxs5UUB9udFAfYbBY+gAAfTMREBBFIuQKGyFTB52cqLAuzPiwLsNwoeQQEOpmMiAgikXIBCZStg8rKVFwXYnxcF2G8UPIICHEzHRAQQSLkAhcpWwORlKy8KsD8vCrDfKHgEBTiYjokIIJByAQqVrYDJy1ZeFGB/XhRgv1HwCApwMB0TEUAg5QIUKlsBk5etvCjA/rwowH6j4BEU4GA6JiKAQMoFKFS2AiYvW3lRgP15UYD9RsEjKMDBdExEAIGUC1CobAVMXrbyogD786IA+42CR1CAg+mYiAACKRegUNkKmLxs5UUB9udFAfYbBY+gAAfTMREBBFIuQKGyFTB52cqLAuzPiwLsNwoeQQEOpmMiAgikXIBCZStg8rKVFwXYnxcF2G8UPIICHEzHRAQQSLkAhcpWwORlKy8KsD8vCrDfKHgEBTiYjokIIJByAQqVrYDJy1ZeFGB/XhRgv1HwCApwMB0TEUAg5QIUKlsBk5etvCjA/rwowH6j4BEU4GA6JiKAQMoFKFS2AiYvW3lRgP15UYD9RsEjKMDBdExEAIGUC1CobAVMXrbyogD780pFAT5x4oQMHz5cpk2bJvv27ZPrr79eJk+eLPXq1StRYNasWdK1a9fTPvbee+/l58ybN0+GDh0q27Ztk/r168uYMWOkXbt2ftEiIyjAZ8XFYAQQyJAAhcpW2ORlKy8KsD+vVBRgLb9aeGfOnCk1atSQAQMGuOK6adMmOf/8809T6Nevn6xfv17mzJlT7GNVq1aVsmXLysqVK6Vt27Yybtw4uemmm2T69Ony9NNPy4YNG6Rhw4Z+1T+NoAAXTMVABBDImACFylbg5GUrLwqwPy/zBfjIkSNSpUoVd4W2Z8+e7oz3798v1atXlxkzZsgdd9xxmkKbNm2kUaNGMmHChBKFtPxWqlRJ5s6dm//4tddeK40bN5apU6f6VSnABRsxEAEEsilAobKVO3nZyosC7M/LfAFet26dNG/eXN555x13q0Lu0NsgtLDqleFTjx/96EcyYsQI6dGjx2kf09spLrzwQnf1t+jHhwwZIgsWLJDNmzf7VSnABRsxEAEEsilAobKVO3nZyosC7M/LfAHWUnrbbbfJ119/LeXLl8+fcadOndzvLVmypJjCnj17pFq1atK5c2d5++235fPPP3cFevTo0a5A639XrlxZli5dKu3bt8/P1SKtt1YcPHiwVNUDBw6I/sodu3fvlmbNmsnOnTulZs2a/jQYgQACCGREgEJlK2jyspUXBdifl/kCPHv2bLn77rvl+PHjUqZMmfwZd+nSRXbt2iXLly8vpqD397Zu3Vr043369HGF9tFHH5U//vGPsnHjRjl69KjUqlVLVqxY4cblDr2dQq8IHzt2rFTVYcOGuTfjnXpQgP0bkREIIJAtAQqVrbzJy1ZeFGB/XuYL8Pz586Vjx44lXgH+5ptvZNGiRacp6JMi9B7f3PHVV19J7dq13RXe7t27u3uKS7oCPGjQIPniiy+4AuzfV4xAAAEEzihAobK1QcjLVl4UYH9e5gtw7h7grVu3ymWXXZY/Y70HuEmTJjJp0iS/gog0bdpUrr76annqqaekQoUKMnHiROnWrVt+rt4DvHDhQvdkiUIPngJRqBTjEEAgawIUKluJk5etvCjA/rzMF2C9yqv39Oqb1nKF9UxPgZgyZYo8/PDD7r7c3D3Det+u3vag9wHrkyT0eb/6SLQXX3wxL8hTIPybiREIIIBAoQIUqkKlzo1x5HVu5HC2nwUX4koXM1+A9dQGDx4szz77rHvsWZ06daR///6yY8cOd0+vPtdX3/hWsWJFV3i1+OqV4RtvvFH0nt1Dhw6J3tqwfft2N17HLFu2TG6++Wb3aDX9X33dZ555xj07uEGDBgXvPzZewVQMRACBjAlQqGwFTl628uIKsD+vVBRgfQPcQw895H4QhhbaFi1auFsftAxrEa5bt677WO6nv+nTHwYOHChr166VkydPij4XWK8g61Xg3KFXf/VRaVpi9ZnBY8eOLfamOD+tuLn6mrwJrhAtxiCAQJYEKFS20iYvW3lRgP15paIA+08zmREU4GTcWRUBBM59AQrVuZ9R0c+QvGzlRQH250UB9hsFj6AAB9MxEQEEUi5AobIVMHnZyosC7M+LAuw3Ch5BAQ6mYyICCKRcgEJlK2DyspUXBdifFwXYbxQ8ggIcTMdEBBBIuQCFylbA5GUrLwqwPy8KsN8oeAQFOJiOiQggkHIBCpWtgMnLVl4UYH9eFGC/UfAICnAwHRMRQCDlAhQqWwGTl628KMD+vCjAfqPgERTgYDomIoBAygUoVLYCJi9beVGA/XlRgP1GwSMowMF0TEQAgZQLUKhsBUxetvKiAPvzogD7jYJHUICD6ZiIAAIpF6BQ2QqYvGzlRQH250UB9hsFj6AAB9MxEQEEUi5AobIVMHnZyosC7M+LAuw3Ch5BAQ6mYyICCKRcgEJlK2DyspUXBdifFwXYbxQ8ggIcTMdEBBBIuQCFylbA5GUrLwqwPy8KsN8oeAQFOJiOiQggkHIBCpWtgMnLVl4UYH9eFGC/UfAICnAwHRMRQCDlAhQqWwGTl628KMD+vCjAfqPgERTgYDomIoBAygUoVLYCJi9beVGA/XlRgP1GwSMowMF0TEQAgZQLUKhsBUxetvKiAPvzogD7jYJHUICD6ZiIAAIpF6BQ2QqYvGzlRQH250UB9hsFj6AAB9MxEQEEUi5AobIVMHnZyosC7M+LAuw3Ch5BAQ6mYyICCKRcgEJlK2DyspUXBdifFwXYbxQ8ggIcTMdEBBBIuQCFylbA5GUrLwqwPy8KsN8oeAQFOJiOiQggkHIBCpWtgMnLVl4UYH9eFGC/UfAICnAwHRMRQCDlAhQqWwGTl628KMD+vCjAfqPgERTgYDomIoBAygUoVLYCJi9beVGA/XlRgP1GwSMowMF0TEQAgZQLUKhsBUxetvKiAPvzogD7jYJHUICD6ZiIAAIpF6BQ2QqYvGzlRQH250UB9hsFj6AAB9MxEQEEUi5AobIVMHnZyosC7M+LAuw3Ch5BAQ6mYyICCKRcgEJlK2DyspUXBdifFwXYbxQ8ggIcTMdEBBBIuQCFylbA5GUrLwqwPy8KsN8oeAQFOJiOiQggkHIBCpWtgMnLVl4UYH9eFGC/UfAICnAwHRMRQCDlAhQqWwGTl628KMD+vCjAfqPgERTgYDomIoBAygUoVLYCJi9beVGA/XlRgP1GwSMowMF0TEQAgZQLUKhsBUxetvKiAPvzogD7jYJHUICD6ZiIAAIpF6BQ2QqYvGzlRQH250UB9hsFj6AAB9MxEQEEUi5AobIVMHnZyosC7M+LAuw3Ch5BAQ6mYyICCKRcgEJlK2DyspUXBdifFwXYbxQ8ggIcTMdEBBBIuQCFylbA5GUrLwqwPy8KsN8oeAQFOJiOiQggkHIBCpWtgMnLVl4UYH9eqSjAJ06ckOHDh8u0adNk3759cv3118vkyZOlXr16JQps2rRJBgwYIGvWrJGyZctKy5Yt5cknn5TatWvnx9etW1d27NhRbH7nzp1l9uzZftU/jaAAF0zFQAQQyJgAhcpW4ORlKy8KsD+vVBRgLb9aeGfOnCk1atRw5Xbbtm2iRff8888vprB37165/PLLXel9+OGH5fDhw/Lggw/KJ598IuvXr5dy5crJl19+KRUrVpTFixfLVVddlZ9fvnx59/uFHhTgQqUYhwACWROgUNlKnLxs5UUB9udlvgAfOXJEqlSpImPGjJGePXu6M96/f79Ur15dZsyYIXfccUcxBb1K3K9fP1d4tezqkSuqK1askNatW8vq1avluuuuc1eTL7roIr9iKSMowMF0TEQAgZQLUKhsBUxetvKiAPvzMl+A161bJ82bN5d33nlH6tevnz9jvQ2icePG7spw0UNva9i+fbu0atUq/9u5ojpv3jzp2LGjTJ06VUaMGCG7du3yC55hBAX4W/ExGQEEUixAobIVLnnZyosC7M/LfAFesGCB3HbbbfL111+L3qKQOzp16uR+b8mSJV6FXr16uavFH3zwgVSrVk30v5ctW+YKtF4N1t+75557pHfv3lKmTBnv67HxCiZiIAIIZFSAQmUrePKylRc9xJ+X+QKsb0q7++675fjx48XKaZcuXdwV3OXLl59R4amnnpI+ffrI+PHj5YEHHnBj9f7gLVu2yKRJk6RBgwayatUqGThwoPTt29ddGS7tOHDggOiv3LF7925p1qyZ7Ny5U2rWrOlPgxEIIIBARgQoVLaCJi9beVGA/XmZL8Dz5893ty2UdAX4m2++kUWLFpWocPLkSXnkkUdk5MiRMmjQIHn88cfz444ePereHFehQoX8740aNcqN1YJb2lXgYcOGuadRnHpQgP0bkREIIJAtAQqVrbzJy1ZeFGB/XuYLcO4e4K1bt8pll12WP2O9B7hJkybuKu6phxZcvaXhpZdecm+e06dA+I6XX35ZOnToIJ999plUrly5xOFcAfYp8nEEEEDg/wlQqGztBPKylRcF2J+X+QKsV3n1Ht1x48ZJt27d3Bmf6SkQ+nF9MoTeO/zCCy+c9pQIfabwpZdeKvfdd58MHjw4L6hXd/XNcR9//LFf9U8jeBNcwVQMRACBjAlQqGwFTl628qIA+/MyX4D1FLWoPvvss+6NbHXq1JH+/fu7H2KxceNG94Mu9uzZ457fq2+Se/75593V37Fjx8pdd91VTCg3Ruc/99xz7gdrXHnllaKPR9P7gydOnOiKcaEHBbhQKcYhgEDWBChUthInL1t5UYD9eaWiAOsb4B566CH3gzAOHTokLVq0cLc+aBnWIqw/1U0/1rVrV2nTpo289tprJcrkxhw7dkxGjx7tCrWWWJ2vzw4+m/KrC1CA/RuQEQggkE0BCpWt3MnLVl4UYH9eqSjA/tNMZgQFOBl3VkUAgXNfgEJ17mdU9DMkL1t5UYD9eVGA/UbBIyjAwXRMRACBlAtQqGwFTF628qIA+/OiAPuNgkdQgIPpmIgAAikXoFDZCpi8bOVFAfbnRQH2GwWPoAAH0zERAQRSLkChshUwednKiwLsz4sC7DcKHkEBDqZjIgIIpFyAQmUrYPKylRcF2J8XBdhvFDyCAhxMx0QEEEi5AIXKVsDkZSsvCrA/Lwqw3yh4BAU4mI6JCCCQcgEKla2AyctWXvDLv0IAABMvSURBVBRgf14UYL9R8AgKcDAdExFAIOUCFCpbAZOXrbwowP68KMB+o+ARFOBgOiYigEDKBShUtgImL1t5UYD9eVGA/UbBIyjAwXRMRACBlAtQqGwFTF628qIA+/OiAPuNgkdQgIPpmIgAAikXoFDZCpi8bOVFAfbnRQH2GwWPoAAH0zERAQRSLkChshUwednKiwLsz4sC7DcKHkEBDqZjIgIIpFyAQmUrYPKylRcF2J8XBdhvFDyCAhxMx0QEEEi5AIXKVsDkZSsvCrA/Lwqw3yh4BAU4mI6JCCCQcgEKla2AyctWXhRgf14UYL9R8AgKcDAdExFAIOUCFCpbAZOXrbwowP68KMB+o+ARFOBgOiYigEDKBShUtgImL1t5UYD9eVGA/UbBIyjAwXRMRACBlAtQqGwFTF628qIA+/OiAPuNgkdQgIPpmIgAAikXoFDZCpi8bOVFAfbnRQH2GwWPoAAH0zERAQRSLkChshUwednKiwLsz4sC7DcKHkEBDqZjIgIIpFyAQmUrYPKylRcF2J8XBdhvFDyCAhxMx0QEEEi5AIXKVsDkZSsvCrA/Lwqw3yh4BAU4mI6JCCCQcgEKla2AyctWXhRgf14UYL9R8AgKcDAdExFAIOUCFCpbAZOXrbwowP68KMB+o+ARFOBgOiYigEDKBShUtgImL1t5UYD9eVGA/UbBIyjAwXRMRACBlAtQqGwFTF628qIA+/OiAPuNgkdQgIPpmIgAAikXoFDZCpi8bOVFAfbnRQH2GwWPoAAH0zERAQRSLkChshUwednKiwLsz4sC7DcKHkEBDqZjIgIIpFyAQmUrYPKylRcF2J8XBdhvFDyCAhxMx0QEEEi5AIXKVsDkZSsvCrA/Lwqw3yh4BAU4mI6JCCCQcgEKla2AyctWXhRgf14UYL9R8AgKcDAdExFAIOUCFCpbAZOXrbwowP68KMB+o+ARFOBgOiYigEDKBShUtgImL1t5UYD9eVGA/UbBIyjAwXRMRACBlAtQqGwFTF628qIA+/OiAPuNgkdQgIPpmIgAAikXoFDZCpi8bOVFAfbnRQH2GwWPoAAH0zERAQRSLkChshUwednKiwLsz4sC7DcKHkEBDqZjIgIIpFyAQmUrYPKylRcF2J9XKgrwiRMnZPjw4TJt2jTZt2+fXH/99TJ58mSpV69eiQJ79+6V3r17y9KlS93Hb7/9dhk/frxccMEF+fHz5s2ToUOHyrZt26R+/foyZswYadeunV+0yAgK8FlxMRgBBDIkQKGyFTZ52cqLAuzPKxUFWMuvFt6ZM2dKjRo1ZMCAAa64btq0Sc4///zTFFq1aiWHDh1yc/bv3y/33nuvtGzZUmbNmuXGrly5Utq2bSvjxo2Tm266SaZPny5PP/20bNiwQRo2bOhX/dMICnDBVAxEAIGMCVCobAVOXrbyogD78zJfgI8cOSJVqlRxV2h79uzpzlhLbfXq1WXGjBlyxx13FFN488035dprr5UtW7ZIgwYN3MeWLVvmru5qYdV5Wn4rVaokc+fOzc/VOY0bN5apU6f6VSnABRsxEAEEsilAobKVO3nZyosC7M/LfAFet26dNG/eXN555x13q0Lu0NsgtLDqVd6ihxblCRMmyK5du/K/rSW6fPnyMmfOHOnYsaNceOGF7upvjx498mOGDBkiCxYskM2bN/tVKcAFGzEQAQSyKUChspU7ednKiwLsz8t8AdZSetttt8nXX3/tSmzu6NSpk/u9JUuWFFPQe3/Xrl3rfhU9qlWr5m6d0NshKleu7O4Pbt++fX6IFmn9+MGDB/2qFOCCjRiIAALZFKBQ2cqdvGzlRQH252W+AM+ePVvuvvtuOX78uJQpUyZ/xl26dHFXeZcvX15MoXv37vLuu+/KqlWriv1+7dq13RXfrl27Sq1atWTFihXSunXr/Bi9nUI/fuzYsVJVDxw4IPord+zcudPdbqFXqS+55BJ/GoxAAAEEMiJw9eMrzJzpmoduNPO5xvpEySuWbNzX3b17tzRr1ky2b98uderUibuYsVc3X4Dnz5/vblso6QrwN998I4sWLSoWyf333+8KaUlXgAcNGiRanPWe4pKuAOvHv/jii1IjHjZsmHsaBQcCCCCAAAIIIHCuCGjvadq06bny6ZwTn4f5Apy7B3jr1q1y2WWX5VH1HuAmTZrIpEmTikHrPcD6RAe9Ops7cvcA65vetExXqFBBJk6cKN26dcuP0XuAFy5c6J4sUdpx6hXgw4cPu3Xq1q0r5513XpTAc/+64yrzd8uL63frmXs1XOO46qtiG8cWV1zjCMR51VP3q37Xes+ePXLFFVdIuXLl4ixq9FXNF2C9yqv37+qb1nKF9UxPgVizZo1cc8018t577+WfE/zqq6/KzTffLB9++KF7jJo+EaJq1ary4osv5mMNeQrE97EneNRaHGVccY0jEO9V2bNxbHHFNY5AnFdlvxbuar4A66kOHjxYnn32WffYM73HpX///rJjxw7ZuHGjlC1b1v3rp2LFiu5NcidPnpQbbrhB9OrslClT3Jva9I1vP/nJT9xzhPXQx6JpIdarxfq/+rrPPPOMrF+/Pv/otMKJ445ks8fxxRXXOALxXpU9G8cWV1zjCMR5VfZr4a6pKMD6BriHHnrIFVj9ARctWrRwtz5oGdYirLcg6Mf0DW56fPrpp9KrVy955ZVXXCnWnwSnV5CLfntAr/6OGDHCPRu4UaNGMnbs2GJviiucOO5INnscX1xxjSMQ71XZs3FsccU1jkCcV2W/Fu6aigJc+Ommb6Ted6zlvW/fvu75xRzfjQCu343jqa+CaxxXfVVs49jiimscgTivyn4t3JUCXLgVIxFAAAEEEEAAAQRSIEABTkGInAICCCCAAAIIIIBA4QIU4MKtGIkAAggggAACCCCQAgEKcApC5BQQQAABBBBAAAEEChegABduxUgEEEAAAQQQQACBFAhQgI2GeOLECfdjl6dNmyb79u0T/cl3kydPzv9wD6On9b1/2p9//rl7hN6SJUvcu+gbN24so0aNcp56vP3229KnTx956623pHLlytK7d2958MEHv/fP0/KC7777rlx55ZXuWdq5RxHi+u0SfeGFF9w+3bZtm/sJmPpj2PVxjuzZcNejR486R30Epv6Z+ld/9VcyevRo0R+ChGuY68iRI2X58uXy+uuv51/A97XP321+65JcFy9e7B7dumXLFqlSpYr780D/Wx/1qgeup7tSgP177ZwcoeVXC68+31h/et2AAQPcX4b6o5rPP//8c/JzPhc/qTZt2rjnQms505/+p8+P1n9U6A890f9u0KCB3Hrrre4xc/pTBH/1q18593vuuedcPJ1z7nPSUqEFQv8BkXsW9969e3H9FknNnj3b/UPiySeflJ/+9KcyZ84cV9z+8z//U+rXr49toO3QoUPlueeek1mzZsmll17qyu+//du/uULxwx/+ENezdJ0wYYL7c1Ofy58rwIV87fN325mhS3J94403pFWrVqLFuGPHjrJ161bp0aOH3Hjjjfkf8IUrBfgsv4TPzeFHjhxx/8LTn1TXs2dP90me6cc/n5tnkfxnpX9I/PjHP5bf//73+as8+pMCtUT84he/cP9y1kKsP0zlvPPOc5+wXi1esGCB/M///E/yJ2DgM1Av/YfDypUr8wX4iSeewDUwO92fWs70Lzn94Ty5o23btu6nWerBng3D1Su+Whj0HxZ66HeE9CeI/vu//7vodzFwLcz1o48+ku7du4uWslq1asnFF1+cL8C+r33+bivd+Eyud911l7uQoz/FNnfoP5T1Qo3+tNsf/OAHdIYSaLkCXNjX9Dk1at26ddK8eXN55513XFnLHfpte/0Wvl6h5PAL6D8aVq9e7f7S0ys8uaNevXrSrl07d0X9oosukpdeein/Mf123t/+7d/KJ598ItWqVfMvkuERq1atkr//+793t5HUrl07X4D1x4vjGrYx9B9eDRs2dN+h+Ou//uvTXgTbMFedpX8O6E/RevXVV11xGz9+vAwZMsR9V+3+++9nzxZIq9+K1+9K6NVI/Ra8XkDIXQH27U8dy99tJUOfyXXDhg1SpkwZadKkSbECfPfdd8uePXvc32W4nu5KAS7wi/pcGqZXIG+77Tb5+uuv8/f36OfXqVMn93t6PytHmMC8efOc429/+1sZPHiwtG/f3n0rNHds3rxZ/vIv/1L+8Ic/yN/8zd+ELZKBWfqPC/3DWK/43Hnnne4KRO4WCP1HGq5hm0D35S233OKurj366KOif/Hpj3rXovazn/3M/QMY2zDb//7v/5Z/+Id/EP0aL1u2rCsUevX37/7u73ANI3W36hQtwL79+eGHH/J3WwHWp7qeOkWvpGvh1T38X//1X+67lnQGCnABW+vcH6Lf2tB/2R0/ftxt8NzRpUsX2bVrl3vTAcfZC+itEHrlV68E/eY3v3FvKNTyplcxckfuTUdaQHJvlDv7ldI/Q28h0W/Xz507151s0QKMa3j+ua99Lb16z6r+I2P+/Pny2GOPyWuvvSb/+I//yJ4N5NV//E6cOFH69+/v3lfxL//yL/Kv//qvot/J+PnPf45rgOupRc33ta9lmb/b/NBnKsDHjh2Tzp07u7/DdO9qEaYzlGzKFWD/XjvnRuhfeHoPYElXgL/55htZtGjROfc5n+ufkJpp2b366qvdFXS9//eKK64Q/ZZdSVeAS/sW9Ll+nt/H56fvoh80aJBs3LhRKlWqdFoBxjU8BX1Tll6l1Nuc/umf/in/QvpmOD30Chp79ux9P/jgA/d+gBUrVsgNN9yQfwH9//p+C32/AK5n73pqUfN97esFBv5u8zuXVoC//PJL9x1Mfc+F/oNOvyukB52BAuzfVUZG5O4B1j+U9RFIuUOvSOoVIX2zBkfhAvoECH3UmV7l0X8p5+4H1r/w9NFnWuhyB/cA+1313cj6RIKi91V/9dVX7r91v/7FX/wFrn7GEkfoPevXXXeduwWi6Hcg9Ckw+g+3OnXqYBtgm7v16dChQ1KuXLn8K+jV4KVLl7JnA0x1yqlFzfdnau4eYP5uOzN4SQV49+7d7van7du3u6u/+udw7qAzUIADv4TPvWl6lVffgDVu3Djp1q2b+wR5CkRYTlOmTHGPNtPn++qbXoreUqLPWZ06daq8//777p5APfTK5sKFC3kKxBm49d3KWiSKHnp1Ta+k69VLfYMMrmH7VV31XfWPPPJIsedR6198f/Znf+aeZoLt2dvqk0quueYaWbt2rTRr1iz/AnpLlBZi/c4QrmfvempR8/2Zyt9thRmf6qrPrdavfX1yyX/8x3+4714WPXClABe2s4yM0jdoPfvsszJjxgx31UevVOi/nvXbzjwHuLAQ9dFGl19+uXTo0OG0J2foLRD6h4Y+B1jfBKNX2PRf0fptZ/2L8Je//GVhizDKCRS9B1gf14Nr+MbQd9frIxD161/Lmt5nrYVYv33fqFEjbANo9YcE6GPk9B3zentJzZo1RX/YiL6JU7+boY+eY8+ePeypRa2Qr33+bvM7n+qq/60XFrT86lNiih76PHu9gIPr6a7cA+zfa+fkCH0DnD5jVd9Zr1eF9GHjeuuDlmGOwgQef/xx94dCSYcW3Oeff9497UGvDuu77S+55BLp16+f/PM//3NhCzAqL1C0AOtv4vrtNod+90dv3dGr7foXnj7kXp8OgW24q15F06dp6K0k+v/1Kpr+GdGyZUtcA1lL+la972ufv9v82EVd9R9vF1xwgRw+fLjEiXpLhPYCXCnA/p3FCAQQQAABBBBAAIFUC3AFONXxcnIIIIAAAggggAACpwpQgNkTCCCAAAIIIIAAApkSoABnKm5OFgEEEEAAAQQQQIACzB5AAAEEEEAAAQQQyJQABThTcXOyCCCAAAIIIIAAAhRg9gACCCCAAAIIIIBApgQowJmKm5NFAAEEEEAAAQQQoACzBxBAAAEEEEAAAQQyJUABzlTcnCwCCCCAAAIIIIAABZg9gAACCCCAAAIIIJApAQpwpuLmZBFAAAEEEEAAAQQowOwBBBBAAAEEEEAAgUwJUIAzFTcniwACCCCAAAIIIEABZg8ggAACCCCAAAIIZEqAApypuDlZBBBAAAEEEEAAAQowewABBBBAAAEEEEAgUwIU4EzFzckigAACCCCAAAIIUIDZAwgggAACCCCAAAKZEqAAZypuThYBBBBAAAEEEECAAsweQAABBBBAAAEEEMiUAAU4U3FzsggggAACCCCAAAIUYPYAAggggAACCCCAQKYEKMCZipuTRQABBBBAAAEEEKAAswcQQAABBBBAAAEEMiVAAc5U3JwsAggggAACCCCAAAWYPYAAAggggAACCCCQKQEKcKbi5mQRQAABBBBAAAEEKMDsAQQQQAABBBBAAIFMCVCAMxU3J4sAAggggAACCCBAAWYPIIAAAggggAACCGRKgAKcqbg5WQQQQAABBBBAAAEKMHsAAQQQQAABBBBAIFMCFOBMxc3JIoAAAggggAACCFCA2QMIIIAAAggggAACmRKgAGcqbk4WAQQQQAABBBBA4P8ChFxP3Rcpvx8AAAAASUVORK5CYII=\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "plt.figure()\n",
    "plt.hist(list(reg_size_dic.values()), bins=np.arange(0,120,5))\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2db97b4d",
   "metadata": {},
   "source": [
    "### 4.1.3 check readout usage for each gene"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "id": "8d0a6c3d",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "383 -> ['Stv_3', 'NDB_22']\n",
      "384 -> ['Stv_32', 'NDB_23']\n",
      "385 -> ['Stv_91', 'NDB_24']\n",
      "386 -> ['Stv_4', 'NDB_25']\n",
      "387 -> ['Stv_33', 'NDB_26']\n",
      "388 -> ['Stv_92', 'NDB_27']\n",
      "389 -> ['Stv_5', 'NDB_28']\n",
      "390 -> ['Stv_35', 'NDB_29']\n",
      "391 -> ['Stv_94', 'NDB_30']\n",
      "392 -> ['Stv_6', 'NDB_31']\n",
      "393 -> ['Stv_36', 'NDB_32']\n",
      "394 -> ['Stv_95', 'NDB_33']\n",
      "395 -> ['Stv_7', 'NDB_34']\n",
      "396 -> ['Stv_37', 'NDB_35']\n",
      "397 -> ['Stv_99', 'NDB_36']\n",
      "398 -> ['Stv_8', 'NDB_37']\n",
      "399 -> ['Stv_39', 'NDB_38']\n",
      "400 -> ['Stv_100', 'NDB_39']\n",
      "401 -> ['Stv_9', 'NDB_40']\n",
      "402 -> ['Stv_40', 'NDB_41']\n",
      "403 -> ['Stv_101', 'NDB_42']\n",
      "404 -> ['Stv_10', 'NDB_43']\n",
      "405 -> ['Stv_42', 'NDB_44']\n",
      "406 -> ['Stv_104', 'NDB_45']\n",
      "407 -> ['Stv_11', 'NDB_46']\n",
      "408 -> ['Stv_44', 'NDB_47']\n",
      "409 -> ['Stv_105', 'NDB_48']\n",
      "410 -> ['Stv_12', 'NDB_49']\n",
      "411 -> ['Stv_45', 'NDB_50']\n",
      "412 -> ['Stv_106', 'NDB_51']\n",
      "413 -> ['Stv_13', 'NDB_52']\n",
      "414 -> ['Stv_46', 'NDB_53']\n",
      "415 -> ['Stv_107', 'NDB_54']\n",
      "416 -> ['Stv_14', 'NDB_55']\n",
      "417 -> ['Stv_48', 'NDB_56']\n",
      "418 -> ['Stv_109', 'NDB_57']\n",
      "419 -> ['Stv_16', 'NDB_58']\n",
      "420 -> ['Stv_50', 'NDB_59']\n",
      "421 -> ['Stv_118', 'NDB_60']\n",
      "422 -> ['Stv_19', 'NDB_61']\n",
      "423 -> ['Stv_53', 'NDB_62']\n",
      "424 -> ['Stv_119', 'NDB_63']\n",
      "425 -> ['Stv_20', 'NDB_64']\n",
      "426 -> ['Stv_54', 'NDB_65']\n",
      "427 -> ['Stv_120', 'NDB_66']\n",
      "428 -> ['Stv_21', 'NDB_67']\n",
      "429 -> ['Stv_59', 'NDB_68']\n",
      "430 -> ['Stv_121', 'NDB_69']\n",
      "431 -> ['Stv_22', 'NDB_70']\n",
      "989 -> ['Stv_60', 'NDB_71']\n",
      "1010 -> ['Stv_125', 'NDB_72']\n",
      "1397 -> ['Stv_23', 'NDB_73']\n",
      "1398 -> ['Stv_61', 'NDB_74']\n",
      "1399 -> ['Stv_127', 'NDB_75']\n",
      "1400 -> ['Stv_25', 'NDB_76']\n",
      "1401 -> ['Stv_63', 'NDB_77']\n",
      "1402 -> ['Stv_129', 'NDB_78']\n",
      "1403 -> ['Stv_26', 'NDB_79']\n",
      "1404 -> ['Stv_64', 'NDB_80']\n",
      "1405 -> ['Stv_130', 'NDB_81']\n",
      "1406 -> ['Stv_27', 'NDB_82']\n",
      "1407 -> ['Stv_65', 'NDB_83']\n",
      "1408 -> ['Stv_131', 'NDB_84']\n",
      "1409 -> ['Stv_28', 'NDB_85']\n",
      "1410 -> ['Stv_86', 'NDB_86']\n",
      "1411 -> ['Stv_133', 'NDB_87']\n",
      "1412 -> ['Stv_29', 'NDB_88']\n",
      "1413 -> ['Stv_87', 'NDB_89']\n",
      "1414 -> ['Stv_136', 'NDB_90']\n",
      "1415 -> ['Stv_30', 'NDB_91']\n",
      "1416 -> ['Stv_88', 'NDB_92']\n",
      "1417 -> ['Stv_145', 'NDB_93']\n",
      "1418 -> ['Stv_31', 'NDB_94']\n",
      "1419 -> ['Stv_90', 'NDB_95']\n",
      "1420 -> ['Stv_182', 'NDB_96']\n",
      "1421 -> ['NDB_1', 'NDB_97']\n",
      "1422 -> ['NDB_2', 'NDB_98']\n",
      "1423 -> ['NDB_3', 'NDB_99']\n",
      "1424 -> ['NDB_4', 'NDB_100']\n",
      "1425 -> ['NDB_5', 'NDB_101']\n",
      "1426 -> ['NDB_6', 'NDB_102']\n",
      "1427 -> ['NDB_7', 'NDB_103']\n",
      "1428 -> ['NDB_8', 'NDB_104']\n",
      "1429 -> ['NDB_9', 'NDB_105']\n",
      "1430 -> ['NDB_10', 'NDB_106']\n",
      "1431 -> ['NDB_11', 'NDB_107']\n",
      "1432 -> ['NDB_12', 'NDB_108']\n",
      "1433 -> ['NDB_13', 'NDB_109']\n",
      "1434 -> ['NDB_14', 'NDB_110']\n",
      "1435 -> ['NDB_15', 'NDB_111']\n",
      "1436 -> ['NDB_16', 'NDB_112']\n",
      "1437 -> ['NDB_17', 'NDB_113']\n",
      "1438 -> ['NDB_18', 'NDB_114']\n",
      "1439 -> ['NDB_19', 'NDB_115']\n",
      "1440 -> ['NDB_20', 'NDB_116']\n",
      "1441 -> ['NDB_21', 'NDB_117']\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "check_readout_dict = pickle.load(open(os.path.join(library_folder, 'readout_usage.pkl'), 'rb'))\n",
    "\n",
    "reg_readout_dic, reg2readout_check = check._check_region_to_readouts(full_records, check_readout_dict, \n",
    "                                                                     total_readout_num=num_on_bits)\n",
    "print(reg2readout_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3e64afd0",
   "metadata": {},
   "source": [
    "### 4.1.4 check readout usage in the library"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "id": "1647e94f",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- extracting probes from >region:383, region:384, region:385, region:386, region:387, region:388, region:389, region:390, region:391, region:392, region:393, region:394, region:395, region:396, region:397, region:398, region:399, region:400, region:401, region:402, region:403, region:404, region:405, region:406, region:407, region:408, region:409, region:410, region:411, region:412, region:413, region:414, region:415, region:416, region:417, region:418, region:419, region:420, region:421, region:422, region:423, region:424, region:425, region:426, region:427, region:428, region:429, region:430, region:431, region:989, region:1010, region:1397, region:1398, region:1399, region:1400, region:1401, region:1402, region:1403, region:1404, region:1405, region:1406, region:1407, region:1408, region:1409, region:1410, region:1411, region:1412, region:1413, region:1414, region:1415, region:1416, region:1417, region:1418, region:1419, region:1420, region:1421, region:1422, region:1423, region:1424, region:1425, region:1426, region:1427, region:1428, region:1429, region:1430, region:1431, region:1432, region:1433, region:1434, region:1435, region:1436, region:1437, region:1438, region:1439, region:1440, region:1441, {'Stv_3': ['383', '383', '383', '383'], 'NDB_22': ['383', '383', '383', '383'], 'Stv_32': ['384', '384', '384', '384'], 'NDB_23': ['384', '384', '384', '384'], 'Stv_91': ['385', '385', '385', '385'], 'NDB_24': ['385', '385', '385', '385'], 'Stv_4': ['386', '386', '386', '386'], 'NDB_25': ['386', '386', '386', '386'], 'Stv_33': ['387', '387', '387', '387'], 'NDB_26': ['387', '387', '387', '387'], 'Stv_92': ['388', '388', '388', '388'], 'NDB_27': ['388', '388', '388', '388'], 'Stv_5': ['389', '389', '389', '389'], 'NDB_28': ['389', '389', '389', '389'], 'Stv_35': ['390', '390', '390', '390'], 'NDB_29': ['390', '390', '390', '390'], 'Stv_94': ['391', '391', '391', '391'], 'NDB_30': ['391', '391', '391', '391'], 'Stv_6': ['392', '392', '392', '392'], 'NDB_31': ['392', '392', '392', '392'], 'Stv_36': ['393', '393', '393', '393'], 'NDB_32': ['393', '393', '393', '393'], 'Stv_95': ['394', '394', '394', '394'], 'NDB_33': ['394', '394', '394', '394'], 'Stv_7': ['395', '395', '395', '395'], 'NDB_34': ['395', '395', '395', '395'], 'Stv_37': ['396', '396', '396', '396'], 'NDB_35': ['396', '396', '396', '396'], 'Stv_99': ['397', '397', '397', '397'], 'NDB_36': ['397', '397', '397', '397'], 'Stv_8': ['398', '398', '398', '398'], 'NDB_37': ['398', '398', '398', '398'], 'Stv_39': ['399', '399', '399', '399'], 'NDB_38': ['399', '399', '399', '399'], 'Stv_100': ['400', '400', '400', '400'], 'NDB_39': ['400', '400', '400', '400'], 'Stv_9': ['401', '401', '401', '401'], 'NDB_40': ['401', '401', '401', '401'], 'Stv_40': ['402', '402', '402', '402'], 'NDB_41': ['402', '402', '402', '402'], 'Stv_101': ['403', '403', '403', '403'], 'NDB_42': ['403', '403', '403', '403'], 'Stv_10': ['404', '404', '404', '404'], 'NDB_43': ['404', '404', '404', '404'], 'Stv_42': ['405', '405', '405', '405'], 'NDB_44': ['405', '405', '405', '405'], 'Stv_104': ['406', '406', '406', '406'], 'NDB_45': ['406', '406', '406', '406'], 'Stv_11': ['407', '407', '407', '407'], 'NDB_46': ['407', '407', '407', '407'], 'Stv_44': ['408', '408', '408', '408'], 'NDB_47': ['408', '408', '408', '408'], 'Stv_105': ['409', '409', '409', '409'], 'NDB_48': ['409', '409', '409', '409'], 'Stv_12': ['410', '410', '410', '410'], 'NDB_49': ['410', '410', '410', '410'], 'Stv_45': ['411', '411', '411', '411'], 'NDB_50': ['411', '411', '411', '411'], 'Stv_106': ['412', '412', '412', '412'], 'NDB_51': ['412', '412', '412', '412'], 'Stv_13': ['413', '413', '413', '413'], 'NDB_52': ['413', '413', '413', '413'], 'Stv_46': ['414', '414', '414', '414'], 'NDB_53': ['414', '414', '414', '414'], 'Stv_107': ['415', '415', '415', '415'], 'NDB_54': ['415', '415', '415', '415'], 'Stv_14': ['416', '416', '416', '416'], 'NDB_55': ['416', '416', '416', '416'], 'Stv_48': ['417', '417', '417', '417'], 'NDB_56': ['417', '417', '417', '417'], 'Stv_109': ['418', '418', '418', '418'], 'NDB_57': ['418', '418', '418', '418'], 'Stv_16': ['419', '419', '419', '419'], 'NDB_58': ['419', '419', '419', '419'], 'Stv_50': ['420', '420', '420', '420'], 'NDB_59': ['420', '420', '420', '420'], 'Stv_118': ['421', '421', '421', '421'], 'NDB_60': ['421', '421', '421', '421'], 'Stv_19': ['422', '422', '422', '422'], 'NDB_61': ['422', '422', '422', '422'], 'Stv_53': ['423', '423', '423', '423'], 'NDB_62': ['423', '423', '423', '423'], 'Stv_119': ['424', '424', '424', '424'], 'NDB_63': ['424', '424', '424', '424'], 'Stv_20': ['425', '425', '425', '425'], 'NDB_64': ['425', '425', '425', '425'], 'Stv_54': ['426', '426', '426', '426'], 'NDB_65': ['426', '426', '426', '426'], 'Stv_120': ['427', '427', '427', '427'], 'NDB_66': ['427', '427', '427', '427'], 'Stv_21': ['428', '428', '428', '428'], 'NDB_67': ['428', '428', '428', '428'], 'Stv_59': ['429', '429', '429', '429'], 'NDB_68': ['429', '429', '429', '429'], 'Stv_121': ['430', '430', '430', '430'], 'NDB_69': ['430', '430', '430', '430'], 'Stv_22': ['431', '431', '431', '431'], 'NDB_70': ['431', '431', '431', '431'], 'Stv_60': ['989', '989', '989', '989'], 'NDB_71': ['989', '989', '989', '989'], 'Stv_125': ['1010', '1010', '1010', '1010'], 'NDB_72': ['1010', '1010', '1010', '1010'], 'Stv_23': ['1397', '1397', '1397', '1397'], 'NDB_73': ['1397', '1397', '1397', '1397'], 'Stv_61': ['1398', '1398', '1398', '1398'], 'NDB_74': ['1398', '1398', '1398', '1398'], 'Stv_127': ['1399', '1399', '1399', '1399'], 'NDB_75': ['1399', '1399', '1399', '1399'], 'Stv_25': ['1400', '1400', '1400', '1400'], 'NDB_76': ['1400', '1400', '1400', '1400'], 'Stv_63': ['1401', '1401', '1401', '1401'], 'NDB_77': ['1401', '1401', '1401', '1401'], 'Stv_129': ['1402', '1402', '1402', '1402'], 'NDB_78': ['1402', '1402', '1402', '1402'], 'Stv_26': ['1403', '1403', '1403', '1403'], 'NDB_79': ['1403', '1403', '1403', '1403'], 'Stv_64': ['1404', '1404', '1404', '1404'], 'NDB_80': ['1404', '1404', '1404', '1404'], 'Stv_130': ['1405', '1405', '1405', '1405'], 'NDB_81': ['1405', '1405', '1405', '1405'], 'Stv_27': ['1406', '1406', '1406', '1406'], 'NDB_82': ['1406', '1406', '1406', '1406'], 'Stv_65': ['1407', '1407', '1407', '1407'], 'NDB_83': ['1407', '1407', '1407', '1407'], 'Stv_131': ['1408', '1408', '1408', '1408'], 'NDB_84': ['1408', '1408', '1408', '1408'], 'Stv_28': ['1409', '1409', '1409', '1409'], 'NDB_85': ['1409', '1409', '1409', '1409'], 'Stv_86': ['1410', '1410', '1410', '1410'], 'NDB_86': ['1410', '1410', '1410', '1410'], 'Stv_133': ['1411', '1411', '1411', '1411'], 'NDB_87': ['1411', '1411', '1411', '1411'], 'Stv_29': ['1412', '1412', '1412', '1412'], 'NDB_88': ['1412', '1412', '1412', '1412'], 'Stv_87': ['1413', '1413', '1413', '1413'], 'NDB_89': ['1413', '1413', '1413', '1413'], 'Stv_136': ['1414', '1414', '1414', '1414'], 'NDB_90': ['1414', '1414', '1414', '1414'], 'Stv_30': ['1415', '1415', '1415', '1415'], 'NDB_91': ['1415', '1415', '1415', '1415'], 'Stv_88': ['1416', '1416', '1416', '1416'], 'NDB_92': ['1416', '1416', '1416', '1416'], 'Stv_145': ['1417', '1417', '1417', '1417'], 'NDB_93': ['1417', '1417', '1417', '1417'], 'Stv_31': ['1418', '1418', '1418', '1418'], 'NDB_94': ['1418', '1418', '1418', '1418'], 'Stv_90': ['1419', '1419', '1419', '1419'], 'NDB_95': ['1419', '1419', '1419', '1419'], 'Stv_182': ['1420', '1420', '1420', '1420'], 'NDB_96': ['1420', '1420', '1420', '1420'], 'NDB_1': ['1421', '1421', '1421', '1421'], 'NDB_97': ['1421', '1421', '1421', '1421'], 'NDB_2': ['1422', '1422', '1422', '1422'], 'NDB_98': ['1422', '1422', '1422', '1422'], 'NDB_3': ['1423', '1423', '1423', '1423'], 'NDB_99': ['1423', '1423', '1423', '1423'], 'NDB_4': ['1424', '1424', '1424', '1424'], 'NDB_100': ['1424', '1424', '1424', '1424'], 'NDB_5': ['1425', '1425', '1425', '1425'], 'NDB_101': ['1425', '1425', '1425', '1425'], 'NDB_6': ['1426', '1426', '1426', '1426'], 'NDB_102': ['1426', '1426', '1426', '1426'], 'NDB_7': ['1427', '1427', '1427', '1427'], 'NDB_103': ['1427', '1427', '1427', '1427'], 'NDB_8': ['1428', '1428', '1428', '1428'], 'NDB_104': ['1428', '1428', '1428', '1428'], 'NDB_9': ['1429', '1429', '1429', '1429'], 'NDB_105': ['1429', '1429', '1429', '1429'], 'NDB_10': ['1430', '1430', '1430', '1430'], 'NDB_106': ['1430', '1430', '1430', '1430'], 'NDB_11': ['1431', '1431', '1431', '1431'], 'NDB_107': ['1431', '1431', '1431', '1431'], 'NDB_12': ['1432', '1432', '1432', '1432'], 'NDB_108': ['1432', '1432', '1432', '1432'], 'NDB_13': ['1433', '1433', '1433', '1433'], 'NDB_109': ['1433', '1433', '1433', '1433'], 'NDB_14': ['1434', '1434', '1434', '1434'], 'NDB_110': ['1434', '1434', '1434', '1434'], 'NDB_15': ['1435', '1435', '1435', '1435'], 'NDB_111': ['1435', '1435', '1435', '1435'], 'NDB_16': ['1436', '1436', '1436', '1436'], 'NDB_112': ['1436', '1436', '1436', '1436'], 'NDB_17': ['1437', '1437', '1437', '1437'], 'NDB_113': ['1437', '1437', '1437', '1437'], 'NDB_18': ['1438', '1438', '1438', '1438'], 'NDB_114': ['1438', '1438', '1438', '1438'], 'NDB_19': ['1439', '1439', '1439', '1439'], 'NDB_115': ['1439', '1439', '1439', '1439'], 'NDB_20': ['1440', '1440', '1440', '1440'], 'NDB_116': ['1440', '1440', '1440', '1440'], 'NDB_21': ['1441', '1441', '1441', '1441'], 'NDB_117': ['1441', '1441', '1441', '1441']}\n",
      " > Done.\n",
      "-- inverting region_to_readout dict\n",
      "NDB_1 -> ['1421']\n",
      "NDB_10 -> ['1430']\n",
      "NDB_100 -> ['1424']\n",
      "NDB_101 -> ['1425']\n",
      "NDB_102 -> ['1426']\n",
      "NDB_103 -> ['1427']\n",
      "NDB_104 -> ['1428']\n",
      "NDB_105 -> ['1429']\n",
      "NDB_106 -> ['1430']\n",
      "NDB_107 -> ['1431']\n",
      "NDB_108 -> ['1432']\n",
      "NDB_109 -> ['1433']\n",
      "NDB_11 -> ['1431']\n",
      "NDB_110 -> ['1434']\n",
      "NDB_111 -> ['1435']\n",
      "NDB_112 -> ['1436']\n",
      "NDB_113 -> ['1437']\n",
      "NDB_114 -> ['1438']\n",
      "NDB_115 -> ['1439']\n",
      "NDB_116 -> ['1440']\n",
      "NDB_117 -> ['1441']\n",
      "NDB_12 -> ['1432']\n",
      "NDB_13 -> ['1433']\n",
      "NDB_14 -> ['1434']\n",
      "NDB_15 -> ['1435']\n",
      "NDB_16 -> ['1436']\n",
      "NDB_17 -> ['1437']\n",
      "NDB_18 -> ['1438']\n",
      "NDB_19 -> ['1439']\n",
      "NDB_2 -> ['1422']\n",
      "NDB_20 -> ['1440']\n",
      "NDB_21 -> ['1441']\n",
      "NDB_22 -> ['383']\n",
      "NDB_23 -> ['384']\n",
      "NDB_24 -> ['385']\n",
      "NDB_25 -> ['386']\n",
      "NDB_26 -> ['387']\n",
      "NDB_27 -> ['388']\n",
      "NDB_28 -> ['389']\n",
      "NDB_29 -> ['390']\n",
      "NDB_3 -> ['1423']\n",
      "NDB_30 -> ['391']\n",
      "NDB_31 -> ['392']\n",
      "NDB_32 -> ['393']\n",
      "NDB_33 -> ['394']\n",
      "NDB_34 -> ['395']\n",
      "NDB_35 -> ['396']\n",
      "NDB_36 -> ['397']\n",
      "NDB_37 -> ['398']\n",
      "NDB_38 -> ['399']\n",
      "NDB_39 -> ['400']\n",
      "NDB_4 -> ['1424']\n",
      "NDB_40 -> ['401']\n",
      "NDB_41 -> ['402']\n",
      "NDB_42 -> ['403']\n",
      "NDB_43 -> ['404']\n",
      "NDB_44 -> ['405']\n",
      "NDB_45 -> ['406']\n",
      "NDB_46 -> ['407']\n",
      "NDB_47 -> ['408']\n",
      "NDB_48 -> ['409']\n",
      "NDB_49 -> ['410']\n",
      "NDB_5 -> ['1425']\n",
      "NDB_50 -> ['411']\n",
      "NDB_51 -> ['412']\n",
      "NDB_52 -> ['413']\n",
      "NDB_53 -> ['414']\n",
      "NDB_54 -> ['415']\n",
      "NDB_55 -> ['416']\n",
      "NDB_56 -> ['417']\n",
      "NDB_57 -> ['418']\n",
      "NDB_58 -> ['419']\n",
      "NDB_59 -> ['420']\n",
      "NDB_6 -> ['1426']\n",
      "NDB_60 -> ['421']\n",
      "NDB_61 -> ['422']\n",
      "NDB_62 -> ['423']\n",
      "NDB_63 -> ['424']\n",
      "NDB_64 -> ['425']\n",
      "NDB_65 -> ['426']\n",
      "NDB_66 -> ['427']\n",
      "NDB_67 -> ['428']\n",
      "NDB_68 -> ['429']\n",
      "NDB_69 -> ['430']\n",
      "NDB_7 -> ['1427']\n",
      "NDB_70 -> ['431']\n",
      "NDB_71 -> ['989']\n",
      "NDB_72 -> ['1010']\n",
      "NDB_73 -> ['1397']\n",
      "NDB_74 -> ['1398']\n",
      "NDB_75 -> ['1399']\n",
      "NDB_76 -> ['1400']\n",
      "NDB_77 -> ['1401']\n",
      "NDB_78 -> ['1402']\n",
      "NDB_79 -> ['1403']\n",
      "NDB_8 -> ['1428']\n",
      "NDB_80 -> ['1404']\n",
      "NDB_81 -> ['1405']\n",
      "NDB_82 -> ['1406']\n",
      "NDB_83 -> ['1407']\n",
      "NDB_84 -> ['1408']\n",
      "NDB_85 -> ['1409']\n",
      "NDB_86 -> ['1410']\n",
      "NDB_87 -> ['1411']\n",
      "NDB_88 -> ['1412']\n",
      "NDB_89 -> ['1413']\n",
      "NDB_9 -> ['1429']\n",
      "NDB_90 -> ['1414']\n",
      "NDB_91 -> ['1415']\n",
      "NDB_92 -> ['1416']\n",
      "NDB_93 -> ['1417']\n",
      "NDB_94 -> ['1418']\n",
      "NDB_95 -> ['1419']\n",
      "NDB_96 -> ['1420']\n",
      "NDB_97 -> ['1421']\n",
      "NDB_98 -> ['1422']\n",
      "NDB_99 -> ['1423']\n",
      "Stv_10 -> ['404']\n",
      "Stv_100 -> ['400']\n",
      "Stv_101 -> ['403']\n",
      "Stv_104 -> ['406']\n",
      "Stv_105 -> ['409']\n",
      "Stv_106 -> ['412']\n",
      "Stv_107 -> ['415']\n",
      "Stv_109 -> ['418']\n",
      "Stv_11 -> ['407']\n",
      "Stv_118 -> ['421']\n",
      "Stv_119 -> ['424']\n",
      "Stv_12 -> ['410']\n",
      "Stv_120 -> ['427']\n",
      "Stv_121 -> ['430']\n",
      "Stv_125 -> ['1010']\n",
      "Stv_127 -> ['1399']\n",
      "Stv_129 -> ['1402']\n",
      "Stv_13 -> ['413']\n",
      "Stv_130 -> ['1405']\n",
      "Stv_131 -> ['1408']\n",
      "Stv_133 -> ['1411']\n",
      "Stv_136 -> ['1414']\n",
      "Stv_14 -> ['416']\n",
      "Stv_145 -> ['1417']\n",
      "Stv_16 -> ['419']\n",
      "Stv_182 -> ['1420']\n",
      "Stv_19 -> ['422']\n",
      "Stv_20 -> ['425']\n",
      "Stv_21 -> ['428']\n",
      "Stv_22 -> ['431']\n",
      "Stv_23 -> ['1397']\n",
      "Stv_25 -> ['1400']\n",
      "Stv_26 -> ['1403']\n",
      "Stv_27 -> ['1406']\n",
      "Stv_28 -> ['1409']\n",
      "Stv_29 -> ['1412']\n",
      "Stv_3 -> ['383']\n",
      "Stv_30 -> ['1415']\n",
      "Stv_31 -> ['1418']\n",
      "Stv_32 -> ['384']\n",
      "Stv_33 -> ['387']\n",
      "Stv_35 -> ['390']\n",
      "Stv_36 -> ['393']\n",
      "Stv_37 -> ['396']\n",
      "Stv_39 -> ['399']\n",
      "Stv_4 -> ['386']\n",
      "Stv_40 -> ['402']\n",
      "Stv_42 -> ['405']\n",
      "Stv_44 -> ['408']\n",
      "Stv_45 -> ['411']\n",
      "Stv_46 -> ['414']\n",
      "Stv_48 -> ['417']\n",
      "Stv_5 -> ['389']\n",
      "Stv_50 -> ['420']\n",
      "Stv_53 -> ['423']\n",
      "Stv_54 -> ['426']\n",
      "Stv_59 -> ['429']\n",
      "Stv_6 -> ['392']\n",
      "Stv_60 -> ['989']\n",
      "Stv_61 -> ['1398']\n",
      "Stv_63 -> ['1401']\n",
      "Stv_64 -> ['1404']\n",
      "Stv_65 -> ['1407']\n",
      "Stv_7 -> ['395']\n",
      "Stv_8 -> ['398']\n",
      "Stv_86 -> ['1410']\n",
      "Stv_87 -> ['1413']\n",
      "Stv_88 -> ['1416']\n",
      "Stv_9 -> ['401']\n",
      "Stv_90 -> ['1419']\n",
      "Stv_91 -> ['385']\n",
      "Stv_92 -> ['388']\n",
      "Stv_94 -> ['391']\n",
      "Stv_95 -> ['394']\n",
      "Stv_99 -> ['397']\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "readout_reg_dic, readout2reg_check = check._check_readout_to_region(\n",
    "    reg_readout_dic, full_records, \n",
    "    check_readout_dict, primer_len=primer_len, \n",
    "    target_len=target_len,\n",
    "    probe_readout_num=num_readout_per_probe,\n",
    ")\n",
    "print(readout2reg_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7445b10e",
   "metadata": {},
   "source": [
    "### 4.1.5 check bewteen probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "id": "3a6a50cc",
   "metadata": {},
   "outputs": [],
   "source": [
    "# construct internal map\n",
    "reload(check)\n",
    "int_map = check._construct_internal_map(full_records, library_folder, save=False) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "id": "11132a68",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "True\n"
     ]
    }
   ],
   "source": [
    "# internal check probes\n",
    "readout_count_dic, readout_count_check = check._check_readout_in_probes(readout_reg_dic, reg_size_dic, int_map, readout_dict)\n",
    "print(readout_count_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 128,
   "id": "056864be",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_7:35710849-35730849_gene_1403_pb_67_pos_15264_strand_+_readouts_[NDB_79_u,NDB_79_u,NDB_79_u,NDB_79_u]_primers_[W1A05_primer_4,W1A04_primer_3] got hits:80, dumped\n",
      "--- Sequence:loc_7:35710849-35730849_gene_1403_pb_68_pos_15345_strand_+_readouts_[Stv_26_u,Stv_26_u,Stv_26_u,Stv_26_u]_primers_[W1A05_primer_4,W1A04_primer_3] got hits:79, dumped\n",
      "--- Sequence:loc_7:35710849-35730849_gene_1403_pb_69_pos_15393_strand_+_readouts_[NDB_79_u,NDB_79_u,NDB_79_u,NDB_79_u]_primers_[W1A05_primer_4,W1A04_primer_3] got hits:52, dumped\n",
      "--- Sequence:loc_7:35710849-35730849_gene_1403_pb_70_pos_15454_strand_+_readouts_[Stv_26_u,Stv_26_u,Stv_26_u,Stv_26_u]_primers_[W1A05_primer_4,W1A04_primer_3] got hits:72, dumped\n",
      "-- total probes removed by internal screening: 4\n"
     ]
    }
   ],
   "source": [
    "# check between probes\n",
    "kept_records, removed_count = check._check_between_probes(full_records, int_map, \n",
    "                                                          primer_len=primer_len, \n",
    "                                                          target_len=target_len)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 129,
   "id": "495f8a7d",
   "metadata": {},
   "outputs": [],
   "source": [
    "# clear memory\n",
    "del(int_map)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ef0c0c1c",
   "metadata": {},
   "source": [
    "### 4.1.6 finish checking, save"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 130,
   "id": "0b11845a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11360 11364\n"
     ]
    }
   ],
   "source": [
    "print(len(kept_records), len(cand_probes))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 131,
   "id": "56768f43",
   "metadata": {},
   "outputs": [],
   "source": [
    "# save kept records\n",
    "with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'w') as output_handle:\n",
    "    SeqIO.write(kept_records, output_handle, \"fasta\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4c40e93f",
   "metadata": {},
   "source": [
    "## 4.2 BLAST screening of probes\n",
    "\n",
    "create the local blast database by:\n",
    "```bash\n",
    "makeblastdb -in C:\\Users\\puzheng\\Documents\\Tools\\lowercase_masked\\Mus_musculus.GRCm38.dna_sm.merged.fa -dbtype nucl -parse_seqids -out GRCm38_ensembl -title \"Mouse chromosomes soft masked\" -blastdb_version 4![image.png](attachment:image.png)\n",
    "```\n",
    "\n",
    "The outputs are:\n",
    "```bash\n",
    "Building a new DB, current time: 03/30/2021 21:49:18\n",
    "New DB name:   C:\\Users\\puzheng\\Documents\\Tools\\GRCm38_ensembl\n",
    "New DB title:  Mouse chromosomes soft masked\n",
    "Sequence type: Nucleotide\n",
    "Keep MBits: T\n",
    "Maximum file size: 1000000000B\n",
    "Adding sequences from FASTA; added 66 sequences in 42.2668 seconds.\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 132,
   "id": "5045ff8f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<module 'ImageAnalysis3.library_tools.quality_check' from 'E:\\\\Users\\\\puzheng_new\\\\Documents\\\\Chromatin_Analysis_Scripts\\\\Library_design\\\\CTP-14-human_brain_genome\\\\..\\\\..\\\\..\\\\..\\\\Documents\\\\ImageAnalysis3\\\\library_tools\\\\quality_check.py'>"
      ]
     },
     "execution_count": 132,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 133,
   "id": "bef107d6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "11360\n"
     ]
    }
   ],
   "source": [
    "if 'kept_records' not in locals() or len(kept_records) == 0:\n",
    "    kept_records = []\n",
    "    with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'r') as handle:\n",
    "        for record in SeqIO.parse(handle, \"fasta\"):\n",
    "            kept_records.append(record)\n",
    "print(len(kept_records))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 134,
   "id": "75e15498",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Blast probe against \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\blast_database\\GRCm38_ensembl\n",
      "-- number of probes loaded: 11360\n",
      "-- processing region: 1010 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1010.fasta\n",
      "--- *blasting region: 1010\n",
      "--- total time for blast 1010: 6.221s.\n",
      "-- processing region: 1397 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1397.fasta\n",
      "--- *blasting region: 1397\n",
      "--- total time for blast 1397: 7.999s.\n",
      "-- processing region: 1398 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1398.fasta\n",
      "--- *blasting region: 1398\n",
      "--- total time for blast 1398: 6.052s.\n",
      "-- processing region: 1399 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1399.fasta\n",
      "--- *blasting region: 1399\n",
      "--- total time for blast 1399: 4.805s.\n",
      "-- processing region: 1400 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1400.fasta\n",
      "--- *blasting region: 1400\n",
      "--- total time for blast 1400: 5.276s.\n",
      "-- processing region: 1401 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1401.fasta\n",
      "--- *blasting region: 1401\n",
      "--- total time for blast 1401: 4.981s.\n",
      "-- processing region: 1402 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1402.fasta\n",
      "--- *blasting region: 1402\n",
      "--- total time for blast 1402: 6.821s.\n",
      "-- processing region: 1403 with 116 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1403.fasta\n",
      "--- *blasting region: 1403\n",
      "--- total time for blast 1403: 6.157s.\n",
      "-- processing region: 1404 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1404.fasta\n",
      "--- *blasting region: 1404\n",
      "--- total time for blast 1404: 6.346s.\n",
      "-- processing region: 1405 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1405.fasta\n",
      "--- *blasting region: 1405\n",
      "--- total time for blast 1405: 6.245s.\n",
      "-- processing region: 1406 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1406.fasta\n",
      "--- *blasting region: 1406\n",
      "--- total time for blast 1406: 8.953s.\n",
      "-- processing region: 1407 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1407.fasta\n",
      "--- *blasting region: 1407\n",
      "--- total time for blast 1407: 6.973s.\n",
      "-- processing region: 1408 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1408.fasta\n",
      "--- *blasting region: 1408\n",
      "--- total time for blast 1408: 5.832s.\n",
      "-- processing region: 1409 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1409.fasta\n",
      "--- *blasting region: 1409\n",
      "--- total time for blast 1409: 5.915s.\n",
      "-- processing region: 1410 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1410.fasta\n",
      "--- *blasting region: 1410\n",
      "--- total time for blast 1410: 5.883s.\n",
      "-- processing region: 1411 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1411.fasta\n",
      "--- *blasting region: 1411\n",
      "--- total time for blast 1411: 4.882s.\n",
      "-- processing region: 1412 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1412.fasta\n",
      "--- *blasting region: 1412\n",
      "--- total time for blast 1412: 5.969s.\n",
      "-- processing region: 1413 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1413.fasta\n",
      "--- *blasting region: 1413\n",
      "--- total time for blast 1413: 6.030s.\n",
      "-- processing region: 1414 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1414.fasta\n",
      "--- *blasting region: 1414\n",
      "--- total time for blast 1414: 5.167s.\n",
      "-- processing region: 1415 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1415.fasta\n",
      "--- *blasting region: 1415\n",
      "--- total time for blast 1415: 6.428s.\n",
      "-- processing region: 1416 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1416.fasta\n",
      "--- *blasting region: 1416\n",
      "--- total time for blast 1416: 9.258s.\n",
      "-- processing region: 1417 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1417.fasta\n",
      "--- *blasting region: 1417\n",
      "--- total time for blast 1417: 6.188s.\n",
      "-- processing region: 1418 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1418.fasta\n",
      "--- *blasting region: 1418\n",
      "--- total time for blast 1418: 6.217s.\n",
      "-- processing region: 1419 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1419.fasta\n",
      "--- *blasting region: 1419\n",
      "--- total time for blast 1419: 6.207s.\n",
      "-- processing region: 1420 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1420.fasta\n",
      "--- *blasting region: 1420\n",
      "--- total time for blast 1420: 11.509s.\n",
      "-- processing region: 1421 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1421.fasta\n",
      "--- *blasting region: 1421\n",
      "--- total time for blast 1421: 7.621s.\n",
      "-- processing region: 1422 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1422.fasta\n",
      "--- *blasting region: 1422\n",
      "--- total time for blast 1422: 4.324s.\n",
      "-- processing region: 1423 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1423.fasta\n",
      "--- *blasting region: 1423\n",
      "--- total time for blast 1423: 8.275s.\n",
      "-- processing region: 1424 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1424.fasta\n",
      "--- *blasting region: 1424\n",
      "--- total time for blast 1424: 5.741s.\n",
      "-- processing region: 1425 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1425.fasta\n",
      "--- *blasting region: 1425\n",
      "--- total time for blast 1425: 5.523s.\n",
      "-- processing region: 1426 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1426.fasta\n",
      "--- *blasting region: 1426\n",
      "--- total time for blast 1426: 6.211s.\n",
      "-- processing region: 1427 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1427.fasta\n",
      "--- *blasting region: 1427\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 1427: 7.159s.\n",
      "-- processing region: 1428 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1428.fasta\n",
      "--- *blasting region: 1428\n",
      "--- total time for blast 1428: 4.990s.\n",
      "-- processing region: 1429 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1429.fasta\n",
      "--- *blasting region: 1429\n",
      "--- total time for blast 1429: 7.178s.\n",
      "-- processing region: 1430 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1430.fasta\n",
      "--- *blasting region: 1430\n",
      "--- total time for blast 1430: 4.690s.\n",
      "-- processing region: 1431 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1431.fasta\n",
      "--- *blasting region: 1431\n",
      "--- total time for blast 1431: 6.149s.\n",
      "-- processing region: 1432 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1432.fasta\n",
      "--- *blasting region: 1432\n",
      "--- total time for blast 1432: 7.381s.\n",
      "-- processing region: 1433 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1433.fasta\n",
      "--- *blasting region: 1433\n",
      "--- total time for blast 1433: 5.076s.\n",
      "-- processing region: 1434 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1434.fasta\n",
      "--- *blasting region: 1434\n",
      "--- total time for blast 1434: 6.795s.\n",
      "-- processing region: 1435 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1435.fasta\n",
      "--- *blasting region: 1435\n",
      "--- total time for blast 1435: 5.070s.\n",
      "-- processing region: 1436 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1436.fasta\n",
      "--- *blasting region: 1436\n",
      "--- total time for blast 1436: 5.089s.\n",
      "-- processing region: 1437 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1437.fasta\n",
      "--- *blasting region: 1437\n",
      "--- total time for blast 1437: 5.279s.\n",
      "-- processing region: 1438 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1438.fasta\n",
      "--- *blasting region: 1438\n",
      "--- total time for blast 1438: 5.462s.\n",
      "-- processing region: 1439 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1439.fasta\n",
      "--- *blasting region: 1439\n",
      "--- total time for blast 1439: 4.922s.\n",
      "-- processing region: 1440 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1440.fasta\n",
      "--- *blasting region: 1440\n",
      "--- total time for blast 1440: 5.998s.\n",
      "-- processing region: 1441 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_1441.fasta\n",
      "--- *blasting region: 1441\n",
      "--- total time for blast 1441: 4.594s.\n",
      "-- processing region: 383 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_383.fasta\n",
      "--- *blasting region: 383\n",
      "--- total time for blast 383: 13.267s.\n",
      "-- processing region: 384 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_384.fasta\n",
      "--- *blasting region: 384\n",
      "--- total time for blast 384: 5.870s.\n",
      "-- processing region: 385 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_385.fasta\n",
      "--- *blasting region: 385\n",
      "--- total time for blast 385: 5.879s.\n",
      "-- processing region: 386 with 100 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_386.fasta\n",
      "--- *blasting region: 386\n",
      "--- total time for blast 386: 30.157s.\n",
      "-- processing region: 387 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_387.fasta\n",
      "--- *blasting region: 387\n",
      "--- total time for blast 387: 6.321s.\n",
      "-- processing region: 388 with 70 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_388.fasta\n",
      "--- *blasting region: 388\n",
      "--- total time for blast 388: 8.219s.\n",
      "-- processing region: 389 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_389.fasta\n",
      "--- *blasting region: 389\n",
      "--- total time for blast 389: 13.827s.\n",
      "-- processing region: 390 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_390.fasta\n",
      "--- *blasting region: 390\n",
      "--- total time for blast 390: 6.906s.\n",
      "-- processing region: 391 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_391.fasta\n",
      "--- *blasting region: 391\n",
      "--- total time for blast 391: 5.714s.\n",
      "-- processing region: 392 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_392.fasta\n",
      "--- *blasting region: 392\n",
      "--- total time for blast 392: 5.152s.\n",
      "-- processing region: 393 with 79 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_393.fasta\n",
      "--- *blasting region: 393\n",
      "--- total time for blast 393: 12.192s.\n",
      "-- processing region: 394 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_394.fasta\n",
      "--- *blasting region: 394\n",
      "--- total time for blast 394: 8.919s.\n",
      "-- processing region: 395 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_395.fasta\n",
      "--- *blasting region: 395\n",
      "--- total time for blast 395: 5.433s.\n",
      "-- processing region: 396 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_396.fasta\n",
      "--- *blasting region: 396\n",
      "--- total time for blast 396: 5.429s.\n",
      "-- processing region: 397 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_397.fasta\n",
      "--- *blasting region: 397\n",
      "--- total time for blast 397: 5.678s.\n",
      "-- processing region: 398 with 117 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_398.fasta\n",
      "--- *blasting region: 398\n",
      "--- total time for blast 398: 16.799s.\n",
      "-- processing region: 399 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_399.fasta\n",
      "--- *blasting region: 399\n",
      "--- total time for blast 399: 8.139s.\n",
      "-- processing region: 400 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_400.fasta\n",
      "--- *blasting region: 400\n",
      "--- total time for blast 400: 11.505s.\n",
      "-- processing region: 401 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_401.fasta\n",
      "--- *blasting region: 401\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 401: 5.571s.\n",
      "-- processing region: 402 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_402.fasta\n",
      "--- *blasting region: 402\n",
      "--- total time for blast 402: 6.408s.\n",
      "-- processing region: 403 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_403.fasta\n",
      "--- *blasting region: 403\n",
      "--- total time for blast 403: 4.411s.\n",
      "-- processing region: 404 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_404.fasta\n",
      "--- *blasting region: 404\n",
      "--- total time for blast 404: 6.273s.\n",
      "-- processing region: 405 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_405.fasta\n",
      "--- *blasting region: 405\n",
      "--- total time for blast 405: 7.605s.\n",
      "-- processing region: 406 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_406.fasta\n",
      "--- *blasting region: 406\n",
      "--- total time for blast 406: 7.235s.\n",
      "-- processing region: 407 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_407.fasta\n",
      "--- *blasting region: 407\n",
      "--- total time for blast 407: 6.041s.\n",
      "-- processing region: 408 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_408.fasta\n",
      "--- *blasting region: 408\n",
      "--- total time for blast 408: 17.250s.\n",
      "-- processing region: 409 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_409.fasta\n",
      "--- *blasting region: 409\n",
      "--- total time for blast 409: 8.002s.\n",
      "-- processing region: 410 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_410.fasta\n",
      "--- *blasting region: 410\n",
      "--- total time for blast 410: 5.417s.\n",
      "-- processing region: 411 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_411.fasta\n",
      "--- *blasting region: 411\n",
      "--- total time for blast 411: 7.796s.\n",
      "-- processing region: 412 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_412.fasta\n",
      "--- *blasting region: 412\n",
      "--- total time for blast 412: 6.831s.\n",
      "-- processing region: 413 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_413.fasta\n",
      "--- *blasting region: 413\n",
      "--- total time for blast 413: 4.306s.\n",
      "-- processing region: 414 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_414.fasta\n",
      "--- *blasting region: 414\n",
      "--- total time for blast 414: 5.817s.\n",
      "-- processing region: 415 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_415.fasta\n",
      "--- *blasting region: 415\n",
      "--- total time for blast 415: 9.919s.\n",
      "-- processing region: 416 with 95 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_416.fasta\n",
      "--- *blasting region: 416\n",
      "--- total time for blast 416: 5.507s.\n",
      "-- processing region: 417 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_417.fasta\n",
      "--- *blasting region: 417\n",
      "--- total time for blast 417: 14.944s.\n",
      "-- processing region: 418 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_418.fasta\n",
      "--- *blasting region: 418\n",
      "--- total time for blast 418: 9.868s.\n",
      "-- processing region: 419 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_419.fasta\n",
      "--- *blasting region: 419\n",
      "--- total time for blast 419: 8.425s.\n",
      "-- processing region: 420 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_420.fasta\n",
      "--- *blasting region: 420\n",
      "--- total time for blast 420: 4.927s.\n",
      "-- processing region: 421 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_421.fasta\n",
      "--- *blasting region: 421\n",
      "--- total time for blast 421: 6.414s.\n",
      "-- processing region: 422 with 103 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_422.fasta\n",
      "--- *blasting region: 422\n",
      "--- total time for blast 422: 4.572s.\n",
      "-- processing region: 423 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_423.fasta\n",
      "--- *blasting region: 423\n",
      "--- total time for blast 423: 5.118s.\n",
      "-- processing region: 424 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_424.fasta\n",
      "--- *blasting region: 424\n",
      "--- total time for blast 424: 5.456s.\n",
      "-- processing region: 425 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_425.fasta\n",
      "--- *blasting region: 425\n",
      "--- total time for blast 425: 4.361s.\n",
      "-- processing region: 426 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_426.fasta\n",
      "--- *blasting region: 426\n",
      "--- total time for blast 426: 7.690s.\n",
      "-- processing region: 427 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_427.fasta\n",
      "--- *blasting region: 427\n",
      "--- total time for blast 427: 4.152s.\n",
      "-- processing region: 428 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_428.fasta\n",
      "--- *blasting region: 428\n",
      "--- total time for blast 428: 5.581s.\n",
      "-- processing region: 429 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_429.fasta\n",
      "--- *blasting region: 429\n",
      "--- total time for blast 429: 6.621s.\n",
      "-- processing region: 430 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_430.fasta\n",
      "--- *blasting region: 430\n",
      "--- total time for blast 430: 5.766s.\n",
      "-- processing region: 431 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_431.fasta\n",
      "--- *blasting region: 431\n",
      "--- total time for blast 431: 5.008s.\n",
      "-- processing region: 989 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\blast\\probe_gene_989.fasta\n",
      "--- *blasting region: 989\n",
      "--- total time for blast 989: 4.771s.\n"
     ]
    }
   ],
   "source": [
    "blast_database = os.path.join(reference_folder, 'blast_database', 'GRCm38_ensembl')\n",
    "\n",
    "check.Blast_probes(kept_records, library_folder, num_threads=44, \n",
    "                   blast_db=blast_database, blast_evalue=100, overwrite=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7c6a9506",
   "metadata": {},
   "source": [
    "### 4.2.1 screen probe by blast results\n",
    "\n",
    "This step has 2 different cutoffs:\n",
    "\n",
    "hard_cutoff: whether a probe has any ~40mer hit somewhere else, if so just dump the probe\n",
    "\n",
    "soft_cutoff: whether a probe has a lot of ~20mer off-target hits, if exceed certain amount then dump the probe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 135,
   "id": "6f6749b3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Number of probes loaded: 11360\n",
      "- Number of regions in this library: 96\n",
      "-- checking probes in region: 383\n",
      "--- gene=383, id=16 removed by soft count = 600\n",
      "--- gene=383, id=23 removed by soft count = 476\n",
      "--- gene=383, id=24 removed by soft count = 562\n",
      "--- gene=383, id=28 removed by soft count = 183\n",
      "--- gene=383, id=29 removed by soft count = 294\n",
      "--- gene=383, id=31 removed by soft count = 8137\n",
      "--- gene=383, id=35 removed by soft count = 1828\n",
      "--- gene=383, id=75 removed by soft count = 60\n",
      "--- gene=383, id=87 removed by soft count = 182\n",
      "--- gene=383, id=88 removed by soft count = 874\n",
      "--- gene=383, id=91 removed by soft count = 799\n",
      "--- number of probes: 120 , kept by blast: 109 , if remove dups: 109\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 384\n",
      "--- gene=384, id=67 removed by soft count = 236\n",
      "--- gene=384, id=79 removed by soft count = 84\n",
      "--- gene=384, id=94 removed by soft count = 72\n",
      "--- number of probes: 120 , kept by blast: 117 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 385\n",
      "--- gene=385, id=8 removed by soft count = 554\n",
      "--- gene=385, id=10 removed by soft count = 84\n",
      "--- gene=385, id=61 removed by soft count = 171\n",
      "--- gene=385, id=62 removed by soft count = 54\n",
      "--- gene=385, id=63 removed by soft count = 252\n",
      "--- gene=385, id=103 removed by soft count = 53\n",
      "--- number of probes: 120 , kept by blast: 114 , if remove dups: 108\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 386\n",
      "--- gene=386, id=0 removed by soft count = 216\n",
      "--- gene=386, id=1 removed by soft count = 1189\n",
      "--- gene=386, id=2 removed by soft count = 1245\n",
      "--- gene=386, id=4 removed by soft count = 59\n",
      "--- gene=386, id=8 removed by soft count = 2124\n",
      "--- gene=386, id=83 removed by soft count = 2876\n",
      "--- gene=386, id=84 removed by soft count = 705\n",
      "--- gene=386, id=85 removed by soft count = 7579\n",
      "--- gene=386, id=86 removed by soft count = 2782\n",
      "--- gene=386, id=87 removed by soft count = 2573\n",
      "--- gene=386, id=89 removed by soft count = 267\n",
      "--- gene=386, id=90 removed by soft count = 6056\n",
      "--- gene=386, id=91 removed by soft count = 818\n",
      "--- gene=386, id=92 removed by soft count = 28441\n",
      "--- gene=386, id=93 removed by soft count = 31763\n",
      "--- gene=386, id=94 removed by soft count = 2035\n",
      "--- gene=386, id=95 removed by soft count = 260\n",
      "--- number of probes: 100 , kept by blast: 83 , if remove dups: 83\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 387\n",
      "--- gene=387, id=46 removed by soft count = 422\n",
      "--- gene=387, id=99 removed by soft count = 63\n",
      "--- gene=387, id=100 removed by soft count = 127\n",
      "--- number of probes: 120 , kept by blast: 117 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 388\n",
      "--- gene=388, id=0 removed by soft count = 936\n",
      "--- gene=388, id=1 removed by soft count = 3362\n",
      "--- gene=388, id=2 removed by soft count = 557\n",
      "--- gene=388, id=3 removed by soft count = 95\n",
      "--- gene=388, id=4 removed by soft count = 173\n",
      "--- gene=388, id=5 removed by soft count = 78\n",
      "--- gene=388, id=6 removed by soft count = 60\n",
      "--- gene=388, id=7 removed by soft count = 58\n",
      "--- gene=388, id=12 removed by soft count = 81\n",
      "--- gene=388, id=16 removed by soft count = 66\n",
      "--- gene=388, id=19 removed by soft count = 92\n",
      "--- gene=388, id=20 removed by soft count = 134\n",
      "--- gene=388, id=21 removed by soft count = 501\n",
      "--- gene=388, id=24 removed by soft count = 84\n",
      "--- gene=388, id=26 removed by soft count = 67\n",
      "--- gene=388, id=28 removed by soft count = 592\n",
      "--- gene=388, id=29 removed by soft count = 536\n",
      "--- gene=388, id=30 removed by soft count = 181\n",
      "--- gene=388, id=31 removed by soft count = 420\n",
      "--- gene=388, id=35 removed by soft count = 255\n",
      "--- gene=388, id=64 removed by soft count = 208\n",
      "--- gene=388, id=65 removed by soft count = 222\n",
      "--- gene=388, id=66 removed by soft count = 199\n",
      "--- gene=388, id=67 removed by soft count = 479\n",
      "--- gene=388, id=68 removed by soft count = 239\n",
      "--- number of probes: 70 , kept by blast: 45 , if remove dups: 45\n",
      "-- number of probes kept for this region: 45\n",
      "-- checking probes in region: 389\n",
      "--- gene=389, id=1 removed by soft count = 24894\n",
      "--- gene=389, id=9 removed by soft count = 68\n",
      "--- gene=389, id=12 removed by soft count = 84\n",
      "--- gene=389, id=19 removed by soft count = 69\n",
      "--- gene=389, id=20 removed by soft count = 87\n",
      "--- gene=389, id=43 removed by soft count = 230\n",
      "--- gene=389, id=61 removed by soft count = 61\n",
      "--- gene=389, id=63 removed by soft count = 1129\n",
      "--- number of probes: 120 , kept by blast: 112 , if remove dups: 112\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 390\n",
      "--- gene=390, id=4 removed by soft count = 342\n",
      "--- gene=390, id=73 removed by soft count = 95\n",
      "--- gene=390, id=74 removed by soft count = 1138\n",
      "--- number of probes: 120 , kept by blast: 117 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 391\n",
      "--- gene=391, id=3 removed by soft count = 83\n",
      "--- number of probes: 120 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 392\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 393\n",
      "--- gene=393, id=7 removed by soft count = 524\n",
      "--- gene=393, id=9 removed by soft count = 704\n",
      "--- gene=393, id=10 removed by soft count = 429\n",
      "--- gene=393, id=11 removed by soft count = 3203\n",
      "--- gene=393, id=12 removed by soft count = 714\n",
      "--- gene=393, id=14 removed by soft count = 234\n",
      "--- gene=393, id=18 removed by soft count = 520\n",
      "--- gene=393, id=19 removed by soft count = 427\n",
      "--- gene=393, id=20 removed by soft count = 104\n",
      "--- gene=393, id=21 removed by soft count = 133\n",
      "--- gene=393, id=22 removed by soft count = 1474\n",
      "--- gene=393, id=24 removed by soft count = 75\n",
      "--- gene=393, id=26 removed by soft count = 139\n",
      "--- gene=393, id=27 removed by soft count = 9967\n",
      "--- gene=393, id=29 removed by soft count = 626\n",
      "--- gene=393, id=30 removed by soft count = 92\n",
      "--- gene=393, id=31 removed by soft count = 228\n",
      "--- gene=393, id=33 removed by soft count = 718\n",
      "--- gene=393, id=38 removed by soft count = 444\n",
      "--- gene=393, id=39 removed by soft count = 272\n",
      "--- number of probes: 79 , kept by blast: 59 , if remove dups: 58\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: 394\n",
      "--- gene=394, id=1 removed by soft count = 116\n",
      "--- gene=394, id=5 removed by soft count = 107\n",
      "--- gene=394, id=8 removed by soft count = 237\n",
      "--- gene=394, id=9 removed by soft count = 268\n",
      "--- gene=394, id=12 removed by soft count = 100\n",
      "--- gene=394, id=15 removed by soft count = 60\n",
      "--- gene=394, id=113 removed by soft count = 110\n",
      "--- number of probes: 120 , kept by blast: 113 , if remove dups: 113\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 395\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 396\n",
      "--- gene=396, id=11 removed by soft count = 56\n",
      "--- gene=396, id=20 removed by soft count = 55\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 397\n",
      "--- gene=397, id=85 removed by soft count = 474\n",
      "--- number of probes: 120 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 398\n",
      "--- gene=398, id=25 removed by soft count = 106\n",
      "--- gene=398, id=43 removed by soft count = 84\n",
      "--- gene=398, id=44 removed by soft count = 120\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=398, id=45 removed by soft count = 218\n",
      "--- gene=398, id=46 removed by soft count = 1585\n",
      "--- gene=398, id=47 removed by soft count = 22682\n",
      "--- gene=398, id=48 removed by soft count = 516\n",
      "--- gene=398, id=63 removed by soft count = 509\n",
      "--- gene=398, id=91 removed by soft count = 245\n",
      "--- gene=398, id=92 removed by soft count = 277\n",
      "--- gene=398, id=115 removed by soft count = 210\n",
      "--- gene=398, id=116 removed by soft count = 2802\n",
      "--- number of probes: 117 , kept by blast: 105 , if remove dups: 105\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 399\n",
      "--- gene=399, id=12 removed by soft count = 272\n",
      "--- gene=399, id=20 removed by soft count = 2862\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 400\n",
      "--- gene=400, id=3 removed by soft count = 618\n",
      "--- gene=400, id=6 removed by soft count = 93\n",
      "--- gene=400, id=7 removed by soft count = 977\n",
      "--- gene=400, id=8 removed by soft count = 134\n",
      "--- gene=400, id=9 removed by soft count = 1729\n",
      "--- gene=400, id=10 removed by soft count = 265\n",
      "--- gene=400, id=111 removed by soft count = 71\n",
      "--- number of probes: 120 , kept by blast: 113 , if remove dups: 113\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 401\n",
      "--- gene=401, id=100 removed by soft count = 284\n",
      "--- number of probes: 120 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 402\n",
      "--- gene=402, id=61 removed by soft count = 60\n",
      "--- gene=402, id=62 removed by soft count = 160\n",
      "--- gene=402, id=64 removed by soft count = 842\n",
      "--- gene=402, id=78 removed by soft count = 70\n",
      "--- number of probes: 120 , kept by blast: 116 , if remove dups: 116\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 403\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 404\n",
      "--- gene=404, id=31 removed by soft count = 424\n",
      "--- gene=404, id=109 removed by soft count = 110\n",
      "--- gene=404, id=114 removed by soft count = 103\n",
      "--- gene=404, id=116 removed by soft count = 398\n",
      "--- gene=404, id=117 removed by soft count = 426\n",
      "--- gene=404, id=118 removed by hard count = 3\n",
      "--- number of probes: 120 , kept by blast: 114 , if remove dups: 114\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 405\n",
      "--- gene=405, id=52 removed by soft count = 1027\n",
      "--- gene=405, id=84 removed by soft count = 54\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 406\n",
      "--- gene=406, id=92 removed by soft count = 148\n",
      "--- gene=406, id=95 removed by soft count = 80\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 407\n",
      "--- gene=407, id=36 removed by soft count = 109\n",
      "--- gene=407, id=49 removed by soft count = 246\n",
      "--- gene=407, id=96 removed by soft count = 455\n",
      "--- number of probes: 120 , kept by blast: 117 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 408\n",
      "--- gene=408, id=0 removed by soft count = 59\n",
      "--- gene=408, id=1 removed by soft count = 342\n",
      "--- gene=408, id=2 removed by soft count = 250\n",
      "--- gene=408, id=3 removed by soft count = 314\n",
      "--- gene=408, id=4 removed by soft count = 7205\n",
      "--- gene=408, id=5 removed by soft count = 1110\n",
      "--- gene=408, id=7 removed by soft count = 220\n",
      "--- gene=408, id=10 removed by soft count = 126\n",
      "--- gene=408, id=11 removed by soft count = 253\n",
      "--- gene=408, id=94 removed by soft count = 4520\n",
      "--- gene=408, id=95 removed by soft count = 666\n",
      "--- number of probes: 120 , kept by blast: 109 , if remove dups: 109\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 409\n",
      "--- gene=409, id=0 removed by soft count = 527\n",
      "--- gene=409, id=15 removed by soft count = 61\n",
      "--- gene=409, id=24 removed by soft count = 127\n",
      "--- gene=409, id=78 removed by soft count = 123\n",
      "--- number of probes: 120 , kept by blast: 116 , if remove dups: 116\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 410\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 411\n",
      "--- gene=411, id=1 removed by soft count = 152\n",
      "--- gene=411, id=2 removed by soft count = 433\n",
      "--- gene=411, id=3 removed by soft count = 94\n",
      "--- gene=411, id=4 removed by soft count = 157\n",
      "--- gene=411, id=5 removed by soft count = 172\n",
      "--- gene=411, id=8 removed by soft count = 77\n",
      "--- gene=411, id=57 removed by soft count = 83\n",
      "--- gene=411, id=59 removed by soft count = 75\n",
      "--- gene=411, id=61 removed by soft count = 699\n",
      "--- gene=411, id=70 removed by soft count = 126\n",
      "--- gene=411, id=73 removed by soft count = 278\n",
      "--- number of probes: 120 , kept by blast: 109 , if remove dups: 109\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 412\n",
      "--- gene=412, id=9 removed by soft count = 64\n",
      "--- gene=412, id=22 removed by soft count = 285\n",
      "--- gene=412, id=42 removed by soft count = 72\n",
      "--- gene=412, id=70 removed by soft count = 347\n",
      "--- number of probes: 120 , kept by blast: 116 , if remove dups: 116\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 413\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 414\n",
      "--- gene=414, id=20 removed by soft count = 54\n",
      "--- gene=414, id=33 removed by soft count = 75\n",
      "--- gene=414, id=58 removed by soft count = 87\n",
      "--- number of probes: 120 , kept by blast: 117 , if remove dups: 115\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 415\n",
      "--- gene=415, id=1 removed by soft count = 137\n",
      "--- gene=415, id=3 removed by soft count = 86\n",
      "--- gene=415, id=4 removed by soft count = 87\n",
      "--- gene=415, id=9 removed by soft count = 60\n",
      "--- gene=415, id=53 removed by soft count = 62\n",
      "--- gene=415, id=58 removed by soft count = 533\n",
      "--- gene=415, id=65 removed by soft count = 71\n",
      "--- gene=415, id=66 removed by soft count = 12429\n",
      "--- gene=415, id=85 removed by soft count = 58\n",
      "--- number of probes: 120 , kept by blast: 111 , if remove dups: 111\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 416\n",
      "--- gene=416, id=0 removed by soft count = 158\n",
      "--- gene=416, id=1 removed by hard count = 3\n",
      "--- gene=416, id=21 removed by soft count = 414\n",
      "--- gene=416, id=24 removed by soft count = 68\n",
      "--- gene=416, id=74 removed by soft count = 179\n",
      "--- gene=416, id=76 removed by soft count = 77\n",
      "--- gene=416, id=80 removed by soft count = 148\n",
      "--- gene=416, id=81 removed by soft count = 510\n",
      "--- number of probes: 95 , kept by blast: 87 , if remove dups: 63\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 417\n",
      "--- gene=417, id=1 removed by soft count = 126\n",
      "--- gene=417, id=3 removed by soft count = 55\n",
      "--- gene=417, id=4 removed by soft count = 188\n",
      "--- gene=417, id=5 removed by soft count = 734\n",
      "--- gene=417, id=6 removed by soft count = 186\n",
      "--- gene=417, id=7 removed by soft count = 323\n",
      "--- gene=417, id=8 removed by soft count = 199\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=417, id=18 removed by soft count = 86\n",
      "--- gene=417, id=33 removed by soft count = 54\n",
      "--- gene=417, id=44 removed by soft count = 86\n",
      "--- gene=417, id=46 removed by soft count = 88\n",
      "--- gene=417, id=47 removed by soft count = 183\n",
      "--- gene=417, id=48 removed by soft count = 301\n",
      "--- gene=417, id=50 removed by soft count = 126\n",
      "--- gene=417, id=51 removed by soft count = 213\n",
      "--- gene=417, id=52 removed by soft count = 374\n",
      "--- gene=417, id=54 removed by soft count = 3385\n",
      "--- gene=417, id=55 removed by soft count = 237\n",
      "--- gene=417, id=57 removed by soft count = 215\n",
      "--- gene=417, id=58 removed by soft count = 1477\n",
      "--- gene=417, id=59 removed by soft count = 75\n",
      "--- gene=417, id=60 removed by soft count = 294\n",
      "--- gene=417, id=61 removed by soft count = 246\n",
      "--- gene=417, id=62 removed by soft count = 383\n",
      "--- gene=417, id=64 removed by soft count = 60\n",
      "--- gene=417, id=65 removed by soft count = 2562\n",
      "--- gene=417, id=70 removed by soft count = 225\n",
      "--- gene=417, id=75 removed by soft count = 128\n",
      "--- gene=417, id=77 removed by soft count = 101\n",
      "--- gene=417, id=78 removed by soft count = 108\n",
      "--- gene=417, id=79 removed by soft count = 110\n",
      "--- gene=417, id=82 removed by soft count = 167\n",
      "--- gene=417, id=83 removed by soft count = 155\n",
      "--- gene=417, id=84 removed by soft count = 231\n",
      "--- gene=417, id=85 removed by soft count = 819\n",
      "--- gene=417, id=86 removed by soft count = 172\n",
      "--- gene=417, id=87 removed by soft count = 125\n",
      "--- gene=417, id=90 removed by soft count = 1215\n",
      "--- gene=417, id=92 removed by soft count = 100\n",
      "--- gene=417, id=93 removed by soft count = 353\n",
      "--- gene=417, id=96 removed by soft count = 246\n",
      "--- gene=417, id=98 removed by soft count = 64\n",
      "--- gene=417, id=99 removed by soft count = 79\n",
      "--- gene=417, id=101 removed by soft count = 182\n",
      "--- gene=417, id=103 removed by soft count = 110\n",
      "--- gene=417, id=104 removed by soft count = 123\n",
      "--- gene=417, id=106 removed by soft count = 70\n",
      "--- gene=417, id=107 removed by soft count = 70\n",
      "--- gene=417, id=108 removed by soft count = 601\n",
      "--- gene=417, id=109 removed by soft count = 176\n",
      "--- gene=417, id=110 removed by soft count = 59\n",
      "--- gene=417, id=112 removed by soft count = 183\n",
      "--- gene=417, id=113 removed by soft count = 159\n",
      "--- gene=417, id=114 removed by soft count = 747\n",
      "--- gene=417, id=116 removed by soft count = 100\n",
      "--- gene=417, id=119 removed by soft count = 471\n",
      "--- number of probes: 120 , kept by blast: 64 , if remove dups: 64\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 64\n",
      "-- checking probes in region: 418\n",
      "--- gene=418, id=1 removed by soft count = 161\n",
      "--- gene=418, id=17 removed by soft count = 63\n",
      "--- gene=418, id=22 removed by soft count = 7530\n",
      "--- gene=418, id=52 removed by soft count = 127\n",
      "--- gene=418, id=53 removed by soft count = 71\n",
      "--- gene=418, id=54 removed by soft count = 94\n",
      "--- gene=418, id=55 removed by soft count = 107\n",
      "--- gene=418, id=63 removed by soft count = 57\n",
      "--- number of probes: 120 , kept by blast: 112 , if remove dups: 112\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 419\n",
      "--- gene=419, id=0 removed by soft count = 9262\n",
      "--- gene=419, id=84 removed by soft count = 77\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 420\n",
      "--- gene=420, id=45 removed by soft count = 90\n",
      "--- gene=420, id=117 removed by soft count = 385\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 421\n",
      "--- gene=421, id=33 removed by soft count = 95\n",
      "--- gene=421, id=62 removed by soft count = 152\n",
      "--- gene=421, id=63 removed by soft count = 54\n",
      "--- gene=421, id=64 removed by soft count = 82\n",
      "--- gene=421, id=78 removed by soft count = 259\n",
      "--- gene=421, id=115 removed by soft count = 85\n",
      "--- gene=421, id=118 removed by soft count = 65\n",
      "--- number of probes: 120 , kept by blast: 113 , if remove dups: 113\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 422\n",
      "--- number of probes: 103 , kept by blast: 103 , if remove dups: 103\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 423\n",
      "--- gene=423, id=48 removed by soft count = 460\n",
      "--- gene=423, id=53 removed by soft count = 719\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 424\n",
      "--- gene=424, id=48 removed by soft count = 486\n",
      "--- gene=424, id=73 removed by soft count = 63\n",
      "--- gene=424, id=77 removed by soft count = 158\n",
      "--- number of probes: 120 , kept by blast: 117 , if remove dups: 115\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 425\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 426\n",
      "--- gene=426, id=29 removed by soft count = 352\n",
      "--- gene=426, id=31 removed by soft count = 178\n",
      "--- gene=426, id=64 removed by soft count = 202\n",
      "--- gene=426, id=65 removed by soft count = 470\n",
      "--- gene=426, id=79 removed by soft count = 69\n",
      "--- gene=426, id=104 removed by soft count = 65\n",
      "--- number of probes: 120 , kept by blast: 114 , if remove dups: 114\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 427\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 428\n",
      "--- gene=428, id=57 removed by soft count = 287\n",
      "--- gene=428, id=70 removed by soft count = 106\n",
      "--- gene=428, id=73 removed by soft count = 53\n",
      "--- gene=428, id=74 removed by soft count = 330\n",
      "--- gene=428, id=75 removed by soft count = 72\n",
      "--- gene=428, id=82 removed by soft count = 187\n",
      "--- number of probes: 120 , kept by blast: 114 , if remove dups: 114\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 429\n",
      "--- gene=429, id=57 removed by soft count = 175\n",
      "--- number of probes: 120 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 430\n",
      "--- gene=430, id=38 removed by soft count = 352\n",
      "--- gene=430, id=70 removed by soft count = 132\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 431\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 989\n",
      "--- gene=989, id=77 removed by soft count = 94\n",
      "--- gene=989, id=108 removed by soft count = 67\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1010\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1397\n",
      "--- gene=1397, id=3 removed by soft count = 147\n",
      "--- number of probes: 120 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1398\n",
      "--- gene=1398, id=94 removed by soft count = 273\n",
      "--- gene=1398, id=95 removed by soft count = 483\n",
      "--- gene=1398, id=96 removed by soft count = 624\n",
      "--- gene=1398, id=97 removed by soft count = 124\n",
      "--- gene=1398, id=98 removed by soft count = 196\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 120 , kept by blast: 115 , if remove dups: 115\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1399\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1400\n",
      "--- gene=1400, id=90 removed by soft count = 68\n",
      "--- number of probes: 120 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1401\n",
      "--- gene=1401, id=0 removed by soft count = 122\n",
      "--- gene=1401, id=91 removed by soft count = 310\n",
      "--- gene=1401, id=93 removed by soft count = 186\n",
      "--- number of probes: 120 , kept by blast: 117 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1402\n",
      "--- gene=1402, id=89 removed by soft count = 824\n",
      "--- gene=1402, id=90 removed by soft count = 213\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1403\n",
      "--- gene=1403, id=6 removed by soft count = 426\n",
      "--- gene=1403, id=61 removed by soft count = 108\n",
      "--- number of probes: 116 , kept by blast: 114 , if remove dups: 114\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1404\n",
      "--- gene=1404, id=32 removed by soft count = 56\n",
      "--- number of probes: 120 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1405\n",
      "--- gene=1405, id=18 removed by soft count = 64\n",
      "--- gene=1405, id=62 removed by soft count = 365\n",
      "--- gene=1405, id=107 removed by soft count = 78\n",
      "--- number of probes: 120 , kept by blast: 117 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1406\n",
      "--- gene=1406, id=9 removed by soft count = 57\n",
      "--- gene=1406, id=10 removed by soft count = 133\n",
      "--- gene=1406, id=11 removed by soft count = 51\n",
      "--- gene=1406, id=12 removed by soft count = 773\n",
      "--- gene=1406, id=16 removed by soft count = 711\n",
      "--- gene=1406, id=17 removed by soft count = 791\n",
      "--- gene=1406, id=21 removed by soft count = 471\n",
      "--- gene=1406, id=23 removed by soft count = 810\n",
      "--- gene=1406, id=24 removed by soft count = 813\n",
      "--- gene=1406, id=26 removed by soft count = 815\n",
      "--- gene=1406, id=27 removed by soft count = 87\n",
      "--- gene=1406, id=28 removed by soft count = 850\n",
      "--- gene=1406, id=32 removed by soft count = 113\n",
      "--- gene=1406, id=33 removed by soft count = 764\n",
      "--- gene=1406, id=38 removed by soft count = 51\n",
      "--- gene=1406, id=43 removed by soft count = 180\n",
      "--- gene=1406, id=44 removed by soft count = 69\n",
      "--- gene=1406, id=47 removed by soft count = 705\n",
      "--- gene=1406, id=50 removed by soft count = 435\n",
      "--- gene=1406, id=52 removed by soft count = 59\n",
      "--- gene=1406, id=59 removed by soft count = 80\n",
      "--- gene=1406, id=61 removed by soft count = 696\n",
      "--- gene=1406, id=115 removed by soft count = 593\n",
      "--- number of probes: 120 , kept by blast: 97 , if remove dups: 97\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1407\n",
      "--- gene=1407, id=100 removed by soft count = 2036\n",
      "--- number of probes: 120 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1408\n",
      "--- gene=1408, id=5 removed by soft count = 155\n",
      "--- number of probes: 120 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1409\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1410\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1411\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1412\n",
      "--- gene=1412, id=103 removed by soft count = 76\n",
      "--- gene=1412, id=110 removed by soft count = 194\n",
      "--- gene=1412, id=112 removed by soft count = 81\n",
      "--- number of probes: 120 , kept by blast: 117 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1413\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1414\n",
      "--- gene=1414, id=59 removed by soft count = 57\n",
      "--- gene=1414, id=106 removed by soft count = 63\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1415\n",
      "--- gene=1415, id=0 removed by soft count = 391\n",
      "--- gene=1415, id=103 removed by soft count = 86\n",
      "--- gene=1415, id=104 removed by soft count = 217\n",
      "--- gene=1415, id=105 removed by soft count = 53\n",
      "--- number of probes: 120 , kept by blast: 116 , if remove dups: 116\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1416\n",
      "--- gene=1416, id=1 removed by soft count = 59\n",
      "--- gene=1416, id=80 removed by soft count = 199\n",
      "--- gene=1416, id=91 removed by soft count = 2148\n",
      "--- gene=1416, id=104 removed by soft count = 187\n",
      "--- gene=1416, id=108 removed by soft count = 3459\n",
      "--- gene=1416, id=117 removed by soft count = 59\n",
      "--- number of probes: 120 , kept by blast: 114 , if remove dups: 114\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1417\n",
      "--- gene=1417, id=118 removed by soft count = 1906\n",
      "--- number of probes: 120 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1418\n",
      "--- gene=1418, id=96 removed by soft count = 89\n",
      "--- gene=1418, id=99 removed by soft count = 443\n",
      "--- gene=1418, id=106 removed by soft count = 76\n",
      "--- number of probes: 120 , kept by blast: 117 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1419\n",
      "--- gene=1419, id=22 removed by soft count = 117\n",
      "--- number of probes: 120 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1420\n",
      "--- gene=1420, id=87 removed by soft count = 2190\n",
      "--- gene=1420, id=94 removed by soft count = 307\n",
      "--- gene=1420, id=96 removed by soft count = 567\n",
      "--- gene=1420, id=97 removed by soft count = 1065\n",
      "--- gene=1420, id=98 removed by soft count = 151\n",
      "--- gene=1420, id=99 removed by soft count = 51\n",
      "--- gene=1420, id=102 removed by soft count = 75\n",
      "--- gene=1420, id=103 removed by soft count = 66\n",
      "--- gene=1420, id=106 removed by soft count = 55\n",
      "--- number of probes: 120 , kept by blast: 111 , if remove dups: 110\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1421\n",
      "--- gene=1421, id=10 removed by soft count = 2934\n",
      "--- gene=1421, id=11 removed by soft count = 82\n",
      "--- gene=1421, id=12 removed by soft count = 93\n",
      "--- gene=1421, id=26 removed by soft count = 68\n",
      "--- gene=1421, id=33 removed by soft count = 58\n",
      "--- gene=1421, id=35 removed by hard count = 0\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1421, id=69 removed by soft count = 65\n",
      "--- gene=1421, id=84 removed by soft count = 68\n",
      "--- gene=1421, id=97 removed by soft count = 332\n",
      "--- number of probes: 120 , kept by blast: 111 , if remove dups: 111\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1422\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1423\n",
      "--- gene=1423, id=11 removed by soft count = 908\n",
      "--- gene=1423, id=30 removed by soft count = 1225\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 116\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1424\n",
      "--- gene=1424, id=93 removed by soft count = 97\n",
      "--- number of probes: 120 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1425\n",
      "--- gene=1425, id=6 removed by soft count = 53\n",
      "--- gene=1425, id=8 removed by soft count = 250\n",
      "--- gene=1425, id=93 removed by soft count = 96\n",
      "--- number of probes: 120 , kept by blast: 117 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1426\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1427\n",
      "--- gene=1427, id=6 removed by soft count = 252\n",
      "--- gene=1427, id=7 removed by soft count = 277\n",
      "--- gene=1427, id=72 removed by soft count = 185\n",
      "--- gene=1427, id=90 removed by soft count = 95\n",
      "--- number of probes: 120 , kept by blast: 116 , if remove dups: 116\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1428\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1429\n",
      "--- gene=1429, id=25 removed by soft count = 147\n",
      "--- number of probes: 120 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1430\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 103\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1431\n",
      "--- gene=1431, id=0 removed by soft count = 1092\n",
      "--- gene=1431, id=112 removed by soft count = 80\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1432\n",
      "--- gene=1432, id=57 removed by soft count = 146\n",
      "--- gene=1432, id=94 removed by soft count = 135\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1433\n",
      "--- gene=1433, id=43 removed by soft count = 185\n",
      "--- gene=1433, id=115 removed by soft count = 68\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1434\n",
      "--- gene=1434, id=2 removed by soft count = 58\n",
      "--- gene=1434, id=23 removed by soft count = 88\n",
      "--- gene=1434, id=25 removed by soft count = 246\n",
      "--- gene=1434, id=65 removed by soft count = 75\n",
      "--- gene=1434, id=70 removed by soft count = 195\n",
      "--- gene=1434, id=73 removed by soft count = 385\n",
      "--- number of probes: 120 , kept by blast: 114 , if remove dups: 114\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1435\n",
      "--- gene=1435, id=61 removed by soft count = 78\n",
      "--- number of probes: 120 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1436\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1437\n",
      "--- gene=1437, id=41 removed by soft count = 102\n",
      "--- number of probes: 120 , kept by blast: 119 , if remove dups: 119\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1438\n",
      "--- gene=1438, id=79 removed by soft count = 209\n",
      "--- number of probes: 120 , kept by blast: 119 , if remove dups: 115\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1439\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1440\n",
      "--- gene=1440, id=95 removed by soft count = 81\n",
      "--- gene=1440, id=110 removed by soft count = 113\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 1441\n",
      "--- number of probes: 120 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the end\n",
      "-- number of probes kept for this region: 80\n",
      "- summarize\n",
      "-- region: 383 , number of probes: 80\n",
      "-- region: 384 , number of probes: 80\n",
      "-- region: 385 , number of probes: 80\n",
      "-- region: 386 , number of probes: 80\n",
      "-- region: 387 , number of probes: 80\n",
      "-- region: 388 , number of probes: 45\n",
      "-- region: 389 , number of probes: 80\n",
      "-- region: 390 , number of probes: 80\n",
      "-- region: 391 , number of probes: 80\n",
      "-- region: 392 , number of probes: 80\n",
      "-- region: 393 , number of probes: 59\n",
      "-- region: 394 , number of probes: 80\n",
      "-- region: 395 , number of probes: 80\n",
      "-- region: 396 , number of probes: 80\n",
      "-- region: 397 , number of probes: 80\n",
      "-- region: 398 , number of probes: 80\n",
      "-- region: 399 , number of probes: 80\n",
      "-- region: 400 , number of probes: 80\n",
      "-- region: 401 , number of probes: 80\n",
      "-- region: 402 , number of probes: 80\n",
      "-- region: 403 , number of probes: 80\n",
      "-- region: 404 , number of probes: 80\n",
      "-- region: 405 , number of probes: 80\n",
      "-- region: 406 , number of probes: 80\n",
      "-- region: 407 , number of probes: 80\n",
      "-- region: 408 , number of probes: 80\n",
      "-- region: 409 , number of probes: 80\n",
      "-- region: 410 , number of probes: 80\n",
      "-- region: 411 , number of probes: 80\n",
      "-- region: 412 , number of probes: 80\n",
      "-- region: 413 , number of probes: 80\n",
      "-- region: 414 , number of probes: 80\n",
      "-- region: 415 , number of probes: 80\n",
      "-- region: 416 , number of probes: 80\n",
      "-- region: 417 , number of probes: 64\n",
      "-- region: 418 , number of probes: 80\n",
      "-- region: 419 , number of probes: 80\n",
      "-- region: 420 , number of probes: 80\n",
      "-- region: 421 , number of probes: 80\n",
      "-- region: 422 , number of probes: 80\n",
      "-- region: 423 , number of probes: 80\n",
      "-- region: 424 , number of probes: 80\n",
      "-- region: 425 , number of probes: 80\n",
      "-- region: 426 , number of probes: 80\n",
      "-- region: 427 , number of probes: 80\n",
      "-- region: 428 , number of probes: 80\n",
      "-- region: 429 , number of probes: 80\n",
      "-- region: 430 , number of probes: 80\n",
      "-- region: 431 , number of probes: 80\n",
      "-- region: 989 , number of probes: 80\n",
      "-- region: 1010 , number of probes: 80\n",
      "-- region: 1397 , number of probes: 80\n",
      "-- region: 1398 , number of probes: 80\n",
      "-- region: 1399 , number of probes: 80\n",
      "-- region: 1400 , number of probes: 80\n",
      "-- region: 1401 , number of probes: 80\n",
      "-- region: 1402 , number of probes: 80\n",
      "-- region: 1403 , number of probes: 80\n",
      "-- region: 1404 , number of probes: 80\n",
      "-- region: 1405 , number of probes: 80\n",
      "-- region: 1406 , number of probes: 80\n",
      "-- region: 1407 , number of probes: 80\n",
      "-- region: 1408 , number of probes: 80\n",
      "-- region: 1409 , number of probes: 80\n",
      "-- region: 1410 , number of probes: 80\n",
      "-- region: 1411 , number of probes: 80\n",
      "-- region: 1412 , number of probes: 80\n",
      "-- region: 1413 , number of probes: 80\n",
      "-- region: 1414 , number of probes: 80\n",
      "-- region: 1415 , number of probes: 80\n",
      "-- region: 1416 , number of probes: 80\n",
      "-- region: 1417 , number of probes: 80\n",
      "-- region: 1418 , number of probes: 80\n",
      "-- region: 1419 , number of probes: 80\n",
      "-- region: 1420 , number of probes: 80\n",
      "-- region: 1421 , number of probes: 80\n",
      "-- region: 1422 , number of probes: 80\n",
      "-- region: 1423 , number of probes: 80\n",
      "-- region: 1424 , number of probes: 80\n",
      "-- region: 1425 , number of probes: 80\n",
      "-- region: 1426 , number of probes: 80\n",
      "-- region: 1427 , number of probes: 80\n",
      "-- region: 1428 , number of probes: 80\n",
      "-- region: 1429 , number of probes: 80\n",
      "-- region: 1430 , number of probes: 80\n",
      "-- region: 1431 , number of probes: 80\n",
      "-- region: 1432 , number of probes: 80\n",
      "-- region: 1433 , number of probes: 80\n",
      "-- region: 1434 , number of probes: 80\n",
      "-- region: 1435 , number of probes: 80\n",
      "-- region: 1436 , number of probes: 80\n",
      "-- region: 1437 , number of probes: 80\n",
      "-- region: 1438 , number of probes: 80\n",
      "-- region: 1439 , number of probes: 80\n",
      "-- region: 1440 , number of probes: 80\n",
      "-- region: 1441 , number of probes: 80\n",
      "- Number of probes kept: 7608\n",
      "- Saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\.\\blast_full_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "# screen\n",
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "num_probes_per_region = 80\n",
    "kept_pbs, blast_keep_dic, hard_count_list, soft_count_list = check.Screening_Probes_by_Blast(\n",
    "    library_folder, \n",
    "    num_probes_per_region,\n",
    "    keep_mode='end',\n",
    "    hard_thres=target_len,\n",
    "    soft_count_th=50,\n",
    ")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d06655b7",
   "metadata": {},
   "source": [
    "# 5. Summary Table and plots"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 137,
   "id": "0bd8cae6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "7608\n"
     ]
    }
   ],
   "source": [
    "blasted_records = []\n",
    "with open(os.path.join(library_folder, 'blast_full_probes.fasta'), 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        blasted_records.append(record)\n",
    "print(len(blasted_records))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 138,
   "id": "35c03434",
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "\n",
    "region_info_dict = {\n",
    "    'gene_name':[],\n",
    "    'locus_info':[],\n",
    "    'chr':[],\n",
    "    #'chr_order':[],\n",
    "    'num_probes':[],\n",
    "    'readouts': [],\n",
    "    #'type':[],\n",
    "}\n",
    "\n",
    "probe_name_re_string = r'loc_(?P<loc>.+)_gene_(?P<gene>.+)_pb_(?P<pb>[0-9]+)_pos_(?P<pos>[0-9]+)(.+)_readouts_\\[(?P<readouts>.+?)\\]_primers_\\[(?P<fwd_primers>.+?),(?P<rev_primers>.+?)\\]'\n",
    "\n",
    "for _pb in blasted_records:\n",
    "    \n",
    "    _match = re.match(probe_name_re_string, _pb.id, )\n",
    "    # append all if not exist\n",
    "    if _match.groupdict()['gene'] not in region_info_dict['gene_name']:\n",
    "        # append new\n",
    "        region_info_dict['gene_name'].append(_match.groupdict()['gene'])\n",
    "        region_info_dict['locus_info'].append(_match.groupdict()['loc'])\n",
    "        region_info_dict['chr'].append(_match.groupdict()['loc'].split(':')[0])\n",
    "        region_info_dict['num_probes'].append(1)\n",
    "        region_info_dict['readouts'].append(list(np.unique(_match.groupdict()['readouts'].split(','))))\n",
    "    else:\n",
    "        _ind = region_info_dict['gene_name'].index(_match.groupdict()['gene'])\n",
    "        region_info_dict['num_probes'][_ind] += 1\n",
    "        for _rd in np.unique(_match.groupdict()['readouts'].split(',')):\n",
    "            if _rd not in region_info_dict['readouts'][_ind]:\n",
    "                region_info_dict['readouts'][_ind].append(_rd)\n",
    "        \n",
    "    #break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 139,
   "id": "187a2a99",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>gene_name</th>\n",
       "      <th>locus_info</th>\n",
       "      <th>chr</th>\n",
       "      <th>num_probes</th>\n",
       "      <th>readouts</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>383</td>\n",
       "      <td>7:6225207-6245207</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[Stv_3_u, NDB_22_u]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>384</td>\n",
       "      <td>7:16228012-16248012</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[NDB_23_u, Stv_32_u]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>385</td>\n",
       "      <td>7:18720071-18740071</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[Stv_91_u, NDB_24_u]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>386</td>\n",
       "      <td>7:26224483-26244483</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[NDB_25_u, Stv_4_u]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>387</td>\n",
       "      <td>7:28730208-28750208</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[NDB_26_u, Stv_33_u]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>1437</td>\n",
       "      <td>7:132236661-132256661</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[NDB_113_u, NDB_17_u]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>1438</td>\n",
       "      <td>7:134648240-134668240</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[NDB_114_u, NDB_18_u]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>1439</td>\n",
       "      <td>7:136050464-136070464</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[NDB_19_u, NDB_115_u]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>1440</td>\n",
       "      <td>7:138193625-138213625</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[NDB_20_u, NDB_116_u]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>1441</td>\n",
       "      <td>7:142362518-142382518</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[NDB_21_u, NDB_117_u]</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>96 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   gene_name             locus_info chr  num_probes               readouts\n",
       "0        383      7:6225207-6245207   7          80    [Stv_3_u, NDB_22_u]\n",
       "1        384    7:16228012-16248012   7          80   [NDB_23_u, Stv_32_u]\n",
       "2        385    7:18720071-18740071   7          80   [Stv_91_u, NDB_24_u]\n",
       "3        386    7:26224483-26244483   7          80    [NDB_25_u, Stv_4_u]\n",
       "4        387    7:28730208-28750208   7          80   [NDB_26_u, Stv_33_u]\n",
       "..       ...                    ...  ..         ...                    ...\n",
       "91      1437  7:132236661-132256661   7          80  [NDB_113_u, NDB_17_u]\n",
       "92      1438  7:134648240-134668240   7          80  [NDB_114_u, NDB_18_u]\n",
       "93      1439  7:136050464-136070464   7          80  [NDB_19_u, NDB_115_u]\n",
       "94      1440  7:138193625-138213625   7          80  [NDB_20_u, NDB_116_u]\n",
       "95      1441  7:142362518-142382518   7          80  [NDB_21_u, NDB_117_u]\n",
       "\n",
       "[96 rows x 5 columns]"
      ]
     },
     "execution_count": 139,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "region_info_df = pd.DataFrame(region_info_dict)\n",
    "region_info_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 140,
   "id": "9dd327f0",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Add additional information\n",
    "_types = []\n",
    "for _g in region_info_df['gene_name']:\n",
    "    if 'hSE-' in _g:\n",
    "        _types.append('SE')\n",
    "    elif 'enhancer-' in _g:\n",
    "        _types.append('enhancer')\n",
    "    else:\n",
    "        _types.append('genome')\n",
    "# add to this df\n",
    "region_info_df['type'] = _types\n",
    "\n",
    "region_info_df['chr_order'] = -1\n",
    "for _chr in np.unique(region_info_df['chr']):\n",
    "    _reg_starts = [int(_info.split(':')[1].split('-')[0])\n",
    "                   for _info in region_info_df.loc[region_info_df['chr']==_chr,\n",
    "                                                   'locus_info']\n",
    "                  ]\n",
    "    _reg_orders = np.array(_reg_starts).argsort().argsort().astype(np.int32)\n",
    "    region_info_df.loc[region_info_df['chr']==_chr,'chr_order'] = _reg_orders"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 141,
   "id": "c9d8517b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>gene_name</th>\n",
       "      <th>locus_info</th>\n",
       "      <th>chr</th>\n",
       "      <th>num_probes</th>\n",
       "      <th>readouts</th>\n",
       "      <th>type</th>\n",
       "      <th>chr_order</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>383</td>\n",
       "      <td>7:6225207-6245207</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[Stv_3_u, NDB_22_u]</td>\n",
       "      <td>genome</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>384</td>\n",
       "      <td>7:16228012-16248012</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[NDB_23_u, Stv_32_u]</td>\n",
       "      <td>genome</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>385</td>\n",
       "      <td>7:18720071-18740071</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[Stv_91_u, NDB_24_u]</td>\n",
       "      <td>genome</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>386</td>\n",
       "      <td>7:26224483-26244483</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[NDB_25_u, Stv_4_u]</td>\n",
       "      <td>genome</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>387</td>\n",
       "      <td>7:28730208-28750208</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[NDB_26_u, Stv_33_u]</td>\n",
       "      <td>genome</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>1437</td>\n",
       "      <td>7:132236661-132256661</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[NDB_113_u, NDB_17_u]</td>\n",
       "      <td>genome</td>\n",
       "      <td>85</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>1438</td>\n",
       "      <td>7:134648240-134668240</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[NDB_114_u, NDB_18_u]</td>\n",
       "      <td>genome</td>\n",
       "      <td>87</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>1439</td>\n",
       "      <td>7:136050464-136070464</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[NDB_19_u, NDB_115_u]</td>\n",
       "      <td>genome</td>\n",
       "      <td>88</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>1440</td>\n",
       "      <td>7:138193625-138213625</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[NDB_20_u, NDB_116_u]</td>\n",
       "      <td>genome</td>\n",
       "      <td>90</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>1441</td>\n",
       "      <td>7:142362518-142382518</td>\n",
       "      <td>7</td>\n",
       "      <td>80</td>\n",
       "      <td>[NDB_21_u, NDB_117_u]</td>\n",
       "      <td>genome</td>\n",
       "      <td>93</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>96 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   gene_name             locus_info chr  num_probes               readouts  \\\n",
       "0        383      7:6225207-6245207   7          80    [Stv_3_u, NDB_22_u]   \n",
       "1        384    7:16228012-16248012   7          80   [NDB_23_u, Stv_32_u]   \n",
       "2        385    7:18720071-18740071   7          80   [Stv_91_u, NDB_24_u]   \n",
       "3        386    7:26224483-26244483   7          80    [NDB_25_u, Stv_4_u]   \n",
       "4        387    7:28730208-28750208   7          80   [NDB_26_u, Stv_33_u]   \n",
       "..       ...                    ...  ..         ...                    ...   \n",
       "91      1437  7:132236661-132256661   7          80  [NDB_113_u, NDB_17_u]   \n",
       "92      1438  7:134648240-134668240   7          80  [NDB_114_u, NDB_18_u]   \n",
       "93      1439  7:136050464-136070464   7          80  [NDB_19_u, NDB_115_u]   \n",
       "94      1440  7:138193625-138213625   7          80  [NDB_20_u, NDB_116_u]   \n",
       "95      1441  7:142362518-142382518   7          80  [NDB_21_u, NDB_117_u]   \n",
       "\n",
       "      type  chr_order  \n",
       "0   genome          1  \n",
       "1   genome          2  \n",
       "2   genome          3  \n",
       "3   genome          5  \n",
       "4   genome          8  \n",
       "..     ...        ...  \n",
       "91  genome         85  \n",
       "92  genome         87  \n",
       "93  genome         88  \n",
       "94  genome         90  \n",
       "95  genome         93  \n",
       "\n",
       "[96 rows x 7 columns]"
      ]
     },
     "execution_count": 141,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "region_info_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 142,
   "id": "e012503a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "saving region info to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\mouse_brain_relabel_chr7\\summary_region_info.csv\n"
     ]
    }
   ],
   "source": [
    "overwrite_region_info = True\n",
    "region_info_filename = os.path.join(library_folder, r'summary_region_info.csv')\n",
    "\n",
    "if not os.path.exists(region_info_filename) or overwrite_region_info:\n",
    "    print(f\"saving region info to file: {region_info_filename}\")\n",
    "    region_info_df.to_csv(region_info_filename, index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0d1a3125",
   "metadata": {},
   "source": [
    "## Generate summary plots"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 143,
   "id": "ddf27206",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: 383 -> 80\n",
      "gene: 384 -> 80\n",
      "gene: 385 -> 80\n",
      "gene: 386 -> 80\n",
      "gene: 387 -> 80\n",
      "gene: 388 -> 45\n",
      "gene: 389 -> 80\n",
      "gene: 390 -> 80\n",
      "gene: 391 -> 80\n",
      "gene: 392 -> 80\n",
      "gene: 393 -> 59\n",
      "gene: 394 -> 80\n",
      "gene: 395 -> 80\n",
      "gene: 396 -> 80\n",
      "gene: 397 -> 80\n",
      "gene: 398 -> 80\n",
      "gene: 399 -> 80\n",
      "gene: 400 -> 80\n",
      "gene: 401 -> 80\n",
      "gene: 402 -> 80\n",
      "gene: 403 -> 80\n",
      "gene: 404 -> 80\n",
      "gene: 405 -> 80\n",
      "gene: 406 -> 80\n",
      "gene: 407 -> 80\n",
      "gene: 408 -> 80\n",
      "gene: 409 -> 80\n",
      "gene: 410 -> 80\n",
      "gene: 411 -> 80\n",
      "gene: 412 -> 80\n",
      "gene: 413 -> 80\n",
      "gene: 414 -> 80\n",
      "gene: 415 -> 80\n",
      "gene: 416 -> 80\n",
      "gene: 417 -> 64\n",
      "gene: 418 -> 80\n",
      "gene: 419 -> 80\n",
      "gene: 420 -> 80\n",
      "gene: 421 -> 80\n",
      "gene: 422 -> 80\n",
      "gene: 423 -> 80\n",
      "gene: 424 -> 80\n",
      "gene: 425 -> 80\n",
      "gene: 426 -> 80\n",
      "gene: 427 -> 80\n",
      "gene: 428 -> 80\n",
      "gene: 429 -> 80\n",
      "gene: 430 -> 80\n",
      "gene: 431 -> 80\n",
      "gene: 989 -> 80\n",
      "gene: 1010 -> 80\n",
      "gene: 1397 -> 80\n",
      "gene: 1398 -> 80\n",
      "gene: 1399 -> 80\n",
      "gene: 1400 -> 80\n",
      "gene: 1401 -> 80\n",
      "gene: 1402 -> 80\n",
      "gene: 1403 -> 80\n",
      "gene: 1404 -> 80\n",
      "gene: 1405 -> 80\n",
      "gene: 1406 -> 80\n",
      "gene: 1407 -> 80\n",
      "gene: 1408 -> 80\n",
      "gene: 1409 -> 80\n",
      "gene: 1410 -> 80\n",
      "gene: 1411 -> 80\n",
      "gene: 1412 -> 80\n",
      "gene: 1413 -> 80\n",
      "gene: 1414 -> 80\n",
      "gene: 1415 -> 80\n",
      "gene: 1416 -> 80\n",
      "gene: 1417 -> 80\n",
      "gene: 1418 -> 80\n",
      "gene: 1419 -> 80\n",
      "gene: 1420 -> 80\n",
      "gene: 1421 -> 80\n",
      "gene: 1422 -> 80\n",
      "gene: 1423 -> 80\n",
      "gene: 1424 -> 80\n",
      "gene: 1425 -> 80\n",
      "gene: 1426 -> 80\n",
      "gene: 1427 -> 80\n",
      "gene: 1428 -> 80\n",
      "gene: 1429 -> 80\n",
      "gene: 1430 -> 80\n",
      "gene: 1431 -> 80\n",
      "gene: 1432 -> 80\n",
      "gene: 1433 -> 80\n",
      "gene: 1434 -> 80\n",
      "gene: 1435 -> 80\n",
      "gene: 1436 -> 80\n",
      "gene: 1437 -> 80\n",
      "gene: 1438 -> 80\n",
      "gene: 1439 -> 80\n",
      "gene: 1440 -> 80\n",
      "gene: 1441 -> 80\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "# get region size dict\n",
    "min_num_probes = 21\n",
    "reg_size_dic, len_check = check._check_region_size(blasted_records, \n",
    "                                                   min_size=min_num_probes)\n",
    "print(len_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 144,
   "id": "8ee3ec1c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox ≥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAApQAAAFKCAYAAACjE8/+AAAAAXNSR0IArs4c6QAAIABJREFUeF7t3Qd0FUX7x/EfHUFA6b1IlS69C9KCogh2QbAgUoS/YEM6CIKoKCgqiq8FOwKCSJHee+9VinREmgoS4H9m8EYCCeTm7ia7yXfP4byvubuzu595cufJ7M5MkosXL14UGwIIIIAAAggggAACsRRIQkIZSzkOQwABBBBAAAEEELACJJQEAgIIIIAAAggggEBIAiSUIfFxMAIIIIAAAggggAAJJTGAAAIIIIAAAgggEJIACWVIfByMAAIIIIAAAgggQEJJDCCAAAIIIIAAAgiEJEBCGRIfByOAAAIIIIAAAgiQUBIDCCCAAAIIIIAAAiEJkFCGxMfBCCCAAAIIIIAAAiSUxAACCCCAAAIIIIBASAIklCHxcTACCCCAAAIIIIAACSUxgAACCCCAAAIIIBCSAAllSHwcjAACCCCAAAIIIEBCSQwggAACCCCAAAIIhCRAQhkSHwcjgAACCCCAAAIIkFASAwgggAACCCCAAAIhCZBQhsTHwQgggAACCCCAAAIklMQAAggggAACCCCAQEgCJJQh8XEwAggggAACCCCAAAklMYAAAggggAACCCAQkgAJZUh8HIwAAggggAACCCBAQkkMIIAAAggggAACCIQkQEIZEh8HI4AAAggggAACCJBQEgMIIIAAAggggAACIQmQUIbEx8EIIIAAAggggAACJJTEAAIIIIAAAggggEBIAiSUIfFxMAIIIIAAAggggAAJJTGAAAIIIIAAAgggEJIACWVIfByMAAIIIIAAAgggQEJJDCCAAAIIIIAAAgiEJEBCGRIfByOAAAIIIIAAAgiQUBIDCCCAAAIIIIAAAiEJkFCGxMfBCCCAAAIIIIAAAiSUxAACCCCAAAIIIIBASAIklCHxcTACCCCAAAIIIIAACSUxgAACCCCAAAIIIBCSAAllSHwcjAACCCCAAAIIIEBCSQwggAACCCCAAAIIhCRAQhkS338HnzlzRuvWrVOWLFmUPHlyh0qlGAQQQAABBBDwqkB4eLiOHDmiUqVKKXXq1F69zDi5LhJKh5iXLVumSpUqOVQaxSCAAAIIIICAXwSWLl2qihUr+uVyXblOEkqHWHft2qUCBQrIBFWOHDkcKpViEEAAAQQQQMCrAgcOHLCdSb/++qvy58/v1cuMk+sioXSI+bffflOePHm0d+9e5c6d26FSKQYBBBBAAAEEvCpA2/9fzZBQOhSlBJVDkBSDAAIIIICATwRo+0koHQ9VgspxUgpEAAEEEEDA0wK0/SSUjgcoQeU4KQUigAACCCDgaQHafhJKxwOUoHKclAIRQAABBBDwtABtPwml4wFKUDlOSoEIIIAAAgh4WoC2n4TS8QAlqBwnpUAEEEAAAQQ8LUDbT0LpeIASVI6TUiACCCCAAAKeFqDtJ6F0PEAJKsdJKRABBBBAAAFPC9D2k1A6HqAEleOkFIgAAggggICnBWj7SSgdD1CCynFSCkQAAQQQQMDTArT9JJSOByhB5TgpBSKAAAIIIOBpAdp+EkrHA5SgcpyUAhFAAAEEEPC0AG0/CaXjAUpQOU5KgQgggAACCHhagLafhNLxACWoHCelQAQQQAABBDwtQNufCBLKH3/8UQMHDtSmTZt0ww03qHr16va/ixYtGik4J02apO7du9v9cufOrS5duqh9+/ZBBzBBFTQZByCAAAIIIOBrAdr+BJ5QTp8+XQ0aNFCLFi302GOP6fjx4+rTp49OnjypDRs2KH369FZg0aJFqlWrllq2bGn3XbBggXr37q0RI0aodevWQQU5QRUUFzsjgAACCCDgewHa/gSeUJpkcMaMGdq5c6eSJEli73bp0qWqXLmyTI9ko0aN7M/M/x47dkxLliyJEGnTpo0mTpwoEyRJkyaNcbATVDGmYkcEEEAAAQQShABtfwJPKFu1aqVVq1Zp7dq1EXe6detW+7j7559/1p133qmzZ8/anspBgwapc+fOEfvNmTNHtWvX1vLly1W+fPkYBzxBFWMqdkQAAQQQQCBBCND2J/CEcu7cuapbt66GDBkS8ci7U6dO2r59u000U6VKpY0bN6pEiRKaPHmywsLCIkSOHDmirFmzatSoUfYxeEw3t4Jq88GTmr/taEwvI0b7Zboxpe4qlVMpk0ffA7tx/0kt3BH789YumlWFst4Y7fX8/c95/bR2v07+fS5G18xO8S+QJV0q3VUqh5Iniz5u1uw9rmW7jsX/xXIFCCCAwGUCaVIm16OV8zpu4lbb7/iFxkGBSS5evHgxDs4T56cwj60fffRRnTp1yp67ePHimjp1qh14YzbzvmSNGjXse5RVqlSJuL7w8HClSJFCQ4cOlUlCo9vM+5jmX2A7cOCAKlWqpL1790acw4mb/mbpHr0ydp0TRUUqY/B9pfVgxTxRlht+/oIqDJiu43/FPtnLmzGN5r5UJ9rrfn/2dg2essXx+6JAdwXefeQ23V0mZ5Qn+fNsuCoOmK6//jnv7kVQOgIIIBCkQLb0qbSkW70gj7r+7iSU/xklyIRy4cKF9rH2448/rnvuuUcnTpzQa6+9pjNnzthE0jzqDiSUixcvtu9WBrZAQjls2DB17Ngx2mgyg3z69u171edOJ5QT1+7XkF+2Xj+qY7jHvuN/62z4BXW8o5CebxB5xHugiON//aOy/abZ/8yZIbVSp0gWw9Klv8+d14ETZ+z+2wY0UopoerOe/36Nxqz8TTemSq6s6VLFuHx2jB+BmMSN6U0Pe2eevUDzB0XypJfeX2ZDAAEE4lvAPJkb3baa45dBQpnAE8oKFSoob968Gjt2bMSdmkfZpnfSTB1kpgYK9ZF3XPVQOh39zUcu1oLtv+uJ6vnV++4SURb/2x9/qcbrs+xnk/+vpm7NcWlUfEy2tb8d1z3vLbC7Lu1WV1nTp47ysCc+XapZW47YRxCvNS0Vk6LZJx4Fnv5iuaZtPKQHyufWGw+UifJKZm85rMc/XWY/2/xqWFB/iMTjrXFqBBBAINYCJJQJPKFMkyaNunbtql69ekUKkgIFCtiey+HDhyfaQTltvliuXzYe0kMV8uj1+0tH+Uu09dApNXh7rv1s3kt1lCdjmhj/su099pdqDr5+Mtrkvfla89sJPVunkF5oGHVPaYxPyo6uC/T8cb1GLd6tmoUza9RT//XoX37i75bt0ctj1unmNCm0qlcD16+JEyCAAALxLUBCmcATyltvvVVFihTR+PHjI+704MGDttdywIABevHFF+3PzbRBZo5K8x5lYGvbtq0mTJiQYKcN6vzdao1btU+NS+fQe4+Wi/J3cdWeP9T0/YX2sxU96inTjTF/JP3XP+Eq3muqPfbr1pVVrVDmKM9Rc/BM7T32t3o1Lq4naxSI7+8Ezn8dgeGztuuNqVtUJNuN+qXz7VHuPXT6Nr09fauKZU+nKc/VwhQBBBBI8AIklAk8oXzvvffs+48dOnRQkyZNbNJo3qHcvXu3ndg8R44cViAwsbl517J58+b2vUrTq5mQJzbv8eM6fbl4j+oUzaJPn6gU5S+7GVXe4pNLc3PG5tFlsZ6TdebcBV1rAEeJXlP05z/nNfThsmpSNleC/9Lx+w3+sOI3vTB6jdKlTq51fRpGeTtm8JgZRFa7aBZ9Fk1s+d2B60cAAQQuFyChTOAJpRm4/vHHH+v999+3UwXdeOONdgS26Z0sVSry+3pmovNu3bpFWnrRJKLBbn4JqoGTNmnE3J2qVCCjvn+mapS3OXXDQT0zaoWSJU2i7QMaRUwOH1OTagNnaP+JM+p7Twm1qpb/qsPOnDuvYj2n2J9/8WQl1SqSJaZFs188CVz+R8bGfg1lpuC4cnvys2WaufmwHq6YR4Pui/p1ini6fE6LAAIIuCLgl7bflZu/otAEOco7LuCuPIdfgirwWLJEzvT6uVPNKKnGrvxNXb6/dm/UtYwbvztP6/ed1P/VLazO9YtcteuBE3+r6sCZ9ucTO9ZQyVwZ4qPKOGcQAtsPn1K9IZfeq535/O26JcvVc4w2GjpPmw5EX+9BnI5dEUAAAV8I+KXtjwtMEkqHlP0SVCPn7VT/nzepQOa0mvVC7Sjv3gy+MIMwcmRIrUWv1A1aqOX/lmru1iN6rEo+vXpvyauOX7/vhBq/O9/+fNErdyhHhhuCPgcHxK3AqTPnVKrPL/akXz9dWdUKXv1ubLlXp+nYn/9oULNSeriS8xMIx+0dczYEEEDg+gJ+afuvfyeh70FCGbqhLcEvQRWYKN3M/bi0e9STvI6Ys0MDJ29WwSxpNeP5qJPOa7E99+0q/bh6v11VZXjzqwf+zNt2RI99stQWEZt3NB2qMooJUqBk76k6fTZcbz9URk1vu7RAQGA7G35eRXtceo3h0ycqqk7RrEGWzu4IIICA/wT80vbHhSwJpUPKfgmqCWv2q9M3q5Q2ZTJt6PffkpOXMwz5ZYuGzdyuMrkzaPyzNYIW6vfTRv1vwa+qeksmfdPmv1WIAgWNX71P//ft6mteQ9An5QDXBeq+NVs7jvypl8OKqV3tgpHOF9Ppoly/SE6AAAIIxKGAX9r+uCAhoXRI2S9BNWPTIT31+XIlSSLtGHCnkkaxmsmrEzfqk/nRJ4TXIwtMMVM0WzpN7Xz19DH/m/+r+k3cqDwZb9C8l+64XnF87hGBwKT4j1fLrz73RJ4U36zf/cCHl6bfWtWzvm5Om9IjV81lIIAAAu4J+KXtd0/gv5JJKB1S9ktQLdrxux75eLG96w19GyptqqtH63Yds1bfLturerdm1chWFYMW+nrJHnUbt06Zb0yl5T2ufqz+1i9b9G4IPaBBXxAHOCLQ5fvVGrtyn8JKZNeHj5WPVOZPa/ar4zerlDJ5Um15NSzomQEcuUAKQQABBOJYwC9tf1ywkFA6pOyXoFr32wnd/d6lATFLu9dV1nRXL41oEgOTIDQpm1NDH74taKEp6w+q7ZfRTztkkk2TdF5rLsygT8oBrgsMnrJZ78/eoTJ5btL4DtUjnS8w2Mus4T33pTquXwsnQAABBLwg4Je2Py6sSCgdUvZLUO04clp135pj73r2C7WVP3PaqwSe+myZZmw+rEcq5dXAZsGvs3354881vRooQ5oUkc7RdtQKTdlwUPeVy623Hox6XWiHqoViHBT4YtEu9Rq/QdnTp9bibpFH//efuFEj5/+qSvkz6vu2Uc9v6uClUBQCCCDgCQG/tP1xgUVC6ZCyX4Lq0MkzqvzaDHvX0c0B+dCIRVry6zE9XbOAut9VPGih7YdPq96QS0lrVHMWPjhikZaGUH7QF8QBjghcPuH91v6N7MT3ga3D1yv189oDuqdMTg17JPhebUcukEIQQACBOBbwS9sfFywklA4p+yWoLp9P0KyUY1bMuXK7+935WrfvRLQTk1+P7I8//9Ftr06zu41pV1Xl80U+h0k2TdIZ1Wjh65XN5/EnsGbvcTUZvsBewJJudZUt/X+vS9z/wUIt3/2H2tS6Rd3uvDX+LpIzI4AAAnEo4Je2Py5ISCgdUvZLUJ2/cFEFu02yd/3p4xVVp9jV8wXe8eZs7Tz6p7rdWUxtakWeHiYmXBcuXFSh7pN04aL00WPl1aBE9kiHBSbAHnxfaT1YMU9MimQfDwgcPnlGlf7t3TbvUJp3KQNbjddn6rc//lbPxsX1VI0CHrhaLgEBBBBwX8Avbb/7EhIJpUPKfgqqW3tO0d/nzuu9R29T49I5rxKo/Np0HTp5VgOallTzyvliJVT+1Wn6PYpVU0xCW/jfZPPjlhVUv3i2WJXPQXEvYOquSI/JMv874rHyavjvHwoXL160k5r/c/6Chj9aTneVzhH3F8cZEUAAgXgQ8FPb7zYPCaVDwn4Kqgr9p+no6X8UXQ9hqd5TdepsuN55qKzuvS1XrITqD5mjbYdP66Wwompfu1BEGWZpPtNDabYx7aqpfL6bY1U+B8WPQNWBM3TgxBn1a1JCLavmtxfx++mzKt9/+r91evUrDvFzpZwVAQQQcF/AT22/2xoklA4J+ymoag2epT3H/lKvxsX15BWPJ01vk3kkbh5Xh9KDGBjY07pGAfVo/N/Anu2HT6nekLlWPbpR5g5VCcW4IHDv8AVavfe4XSnHvANrtg37T+iuYZemopr/ch3lvjmNC2emSAQQQMB7An5q+93WI6F0SNhPQdVo6DxtOnBSLzQoomfvKBxJ4O9/zuvWXpfWZP66dWVVK5Q5VkLtvlyhyesPqtltuTTkobIRZZjR3WaUt9nW9G6gDDdEnlIoVifjoDgTCEz5dHm9ztx8SE9+ttxegxn9bSY3Z0MAAQQSg4Cf2n6364OE0iFhPwVVYERu29sLqmujS71Mge3IqbOqOODS48sfO1RX2csGXgRD1X3cOn21ZI9qF82iz56oFHHolPUH1PbLlUqRLIlNPpKYNSDZfCPQZ8IGfbZwl6oVzKSvn760Tvv1Vkbyzc1xoQgggECQAn5q+4O8taB3J6EMmizqA/wUVK3+t1Rzth5Ry6r51K9JyUg3tPv3P3X7G7Ptz6Z3qaVCWdPFSmjIL1s0bOZ2lc6dQROerRFRxldLdqv7uPXKmi6Vlna/elnGWJ2Mg+JM4IPZO/T6lM26JUtazXy+tj3vkGlbNWzGNpXMlV4TO9aMs2vhRAgggEB8C/ip7XfbioTSIWE/BVX7r1Zo0rqoV6rZuP+k7hw2z6os7HqHct50Q6yEPl3wq/r+tFG5brpBC7reEVHGezO36c1ftqpY9nSa8lytWJXNQfEnMG7Vb+r83RqlTZlMG/qF2Qt56Yc1+n75b7Fe+z3+7oYzI4AAAqEJ+KntD+1Or380CeX1jWK0h5+C6oXRa/TDit8UViK7PnysfKT7i7RsYgjvOI5fvU//9+1qpUmZTBv/TTzMifr+tEGfLoj8yDRGwOzkCYGFO47q0Y+X2GtZ16eB0qVOoZb/W6q5W4+oeeW8GtA0+KU6PXFjXAQCCCAQCwE/tf2xuL2gDiGhDIor+p39FFSB9+BqFs6sUU9VjnRTs7cc1uOfLrM/2zagkVIki90Ai/nbjqrFJ5cSj82vhil1imT2/z/37Sr9uHq/GpfOofceLeeQPsXElcCvR/9UnTcjvxLR8O252nLolJ6vX0Qd60Ye5BVX18V5EEAAgfgQ8FPb77YPCaVDwn4KqsFTNuv92TtULu9NGtu+eiQBsx6zWZfZjNQ1g2Ziu10+lYx55G0efZvtsU+WaN62o1G+vxnbc3Fc3AlcPgvAqKcqqWbhLCrT9xed+PucBt9fWg9WYOWjuKsNzoQAAvEt4Ke2320rEkqHhP0UVMNnbdcbU7dE+R7j98v36qUf1ipj2pRa2bN+rHUOnjijKgNn2OMndqyhkrky2P9/17B52rD/pJ6rV1jP1SsS6/I5MP4ESveZqpNnwvXG/aV1d5mcKtbz0jRTXzxZSbWKZIm/C+PMCCCAQBwL+Kntd5uGhNIhYT8F1WcLflWfnzYqT8YbNO+l/wbMGIrAZ7lvvkHzX478WTBUZ8PP2+X4zPb5k5V0+7+JRrWBM7T/ipVWgimXfeNfoMHbc7T10Gk7j6lZurP2v4/Ap3WupcLZYjcrQPzfFVeAAAIIBC/gp7Y/+LsL7ggSyuC8ot3bT0F1rV7Ia/VeBktVsvdUnT4brrcfKqOmt+WWWYXH9GadDb8Q7TriwZ6D/eNeIPDaQosqeW1C+fBHi+1FrO3TQOlTM1F93NcIZ0QAgfgS8FPb77YRCaVDwn4KqknrDqj9VyuVKnlSbbniPclrvV8ZLFVgiceejYvrqRoF9OfZcJXoPdUW8/XTlVWtYOxW4Qn2OtjfWYEXR6/R6BVmmqBsurtMjojR/Bv6NmSiemepKQ0BBDwu4Ke2321KEkqHhP0UVJeP5N4+oJGSXzaSu/f49fp80W5FNQI8WKomwxdozd7j6lCnoF5sWEx7j/2lmoNn2WKmPFdTxbKnD7ZI9veAwFu/bNG7M7erVK4MdrT+wMmbdUvmtJr5wqWJztkQQACBxCLgp7bf7TohoXRI2E9BtXzXMd3/YdTraV9rjspgqZ78bJlmbj6sRyrl0cBmpW1yaZJMsy3tXldZ06UOtkj294DAl4t3q8eP65UlXSrdVSqHXYqx6i2Z9E2bS0sxsiGAAAKJRcBPbb/bdUJC6ZCwn4Lq8tVwFr1yh3Jk+G81nMAqOs3K5dKQB8uGpBNIThuWyKYRj1XQrM2H9cRnoc9xGdJFcXDIAjM2HdJTny+XWYa9brFsmr7pkJrelktvPxRavIR8YRSAAAIIxLGAn9p+t2lIKB0S9lNQXWu97sCqJ1Gt8x0s1WuTNumjuTtVMf/NGt22msas+E3Pj16j9KmTa22fhsEWx/4eEVi/74QavzvfXk329Kl18OQZtb29oLo2KuaRK+QyEEAAgbgR8FPb77YICaVDwn4KqqOnz6pC/+n2zsd3qK4yeW6KULj/g4VavvsPRxKED+fs0CDzfl2WtJr5fG19PHenBkzapAKZ02oW79s5FHlxX8zl8RM4e997SqhVtfxxfzGcEQEEEIhHAT+1/W4zkVA6JOynoLp8tZOvW1dWtUL/jbYOe2euNh88ZecYfPaO0JbRC0xPdHOaFFrVq4FNLk2SWT7fzRrTrppD8hQT1wIXLlxU0Z6Tde78xYhTf9iivMJKZo/rS+F8CCCAQLwK+KntdxuKhNIhYT8FlZkPsmC3SbpwUfq4ZQXVL54tQiEw1U+vxsX1ZI0CIelM33hIrb+49K7d9gF3qtvYdfpu+V473czIVhVCKpuD41eg+qCZ2nf874iL+LFDdZW9rKc7fq+OsyOAAAJxI+Cntt9tERJKh4T9FlSlek/VqbPhGvpwWTUpmytCoUL/aTp6+h+9fl8pPVQxb0g6K/f8oWbvL7RlLO9RT13HrLMDOB6qkEev3186pLI5OH4F7vtgoVbs/iPiIha/UlfZMzBqP35rhbMjgEBcC/it7XfTh4TSIV2/BVXl16br0MmzGtC0pJpXzhehUKznZJ0558xKNpcP/jHL8r08Zq1W7jmudrUL6uUwBnA4FHrxUkyHr1bq53UH7LmTJpG29o88n2m8XBQnRQABBOJYwG9tv5s8JJQO6fotqO54a7Z2HvlT3e+8VU/XusUqnL9w6VG42T59vKLqFMsaks7JM+dUus8vtoxv21TRK2PX6dejkc8Z0gk4ON4E+v20Uf9b8Ks9f7b0qbSkW714uxZOjAACCMSXgN/afjedSCgd0vVbUN397nyt23dC/1e3sDrXL2IVLk8Av3+mqioVyBiSjnlXs0iPS4M33m9eTl3HrNXJM+F664Eyuq987pDK5uD4Ffho7g69NmmzvYgyuTNo/LM14veCODsCCCAQDwJ+a/vdJCKhdEjXb0H18EeLtHjnMT1ds4C631XcKhw8cUZVBs6w/39ixxoqmStDyDqBR+t97i6uPj9ttOV9+kRF1SkaWu9nyBdGASEJTFizX52+WWXLCExcH1KBHIwAAgj4UMBvbb+bxCSUDun6Laie+myZZmw+rEcr59VrTUtZhR1HTqvuW3Ps/zfzRJr5IkPdAtMQPVYln0Yt3m2Lu3Luy1DPwfFxL7D012N6cMSl5TtbVc2nvk1Kxv1FcEYEEEAgngX81va7yUVC6ZCu34Kq4zer9NOa/WpSNqeGPnybVVj723Hd856za203H7lYC7b/btd6XrTzd3ueeS/VUZ6MaRySp5j4ENjz+1+q9cYse+qXwoqqfe1C8XEZnBMBBBCIVwG/tf1uYpFQOqTrt6B6ZexafbM08pyQi3b8rkc+XmxFNvRtqLSpkoesE0hcs6RLpSOnztryNvZrqDQpQy875IujgFgLnDl3XsV6TrHHD3mwjJqV453YWGNyIAII+FbAb22/m9AklA7p+i2oXp24UZ/M/9X2HH7TpopVuHwi8h0D7lRSMx9MiFvv8ev1+aJLj7rNlip5Um1+NUxJzGznbL4W6PD1SplH3+Z922zpmYPS15XJxSOAQKwE/Nb2x+omY3gQCWUMoa63m9+Casi0rRo2Y1ukEbrjV+/T/327WmlTJtOGfmHXu+UYfT50+ja9PX1rxL45M6TWwlfqxuhYdvK+gFmG0Yk/PLx/p1whAgggcLWA39p+N+vQdwnlP//8o5QpU7ppEquy/RZUI+bs0MDJm1Uo642a3uV2e8/fLN1j54o0j6eXdXdmXkEzEKfnj+sjTEvmSq+JHWvGypiDEEAAAQQQ8JKA39p+N+08m1COGjVKx48fV8eOHe39r1+/Xk2bNtWvv/6qGjVq6Pvvv1fWrN6ZesZvQRVI9HJkSK1F//YYjpy3U/1/3mRHd5tR3k5sP689IPNoNLDVLJxZo56q7ETRlIEAAggggEC8Cvit7XcTy7MJZenSpfXMM8+oQ4cO9v7r1Kmjw4cPq127dnr33XdVs2ZNjRw50k2boMr2W1CNW/WbOn+3RulTJ9faPg3tvQYeT5fImV4/d3KmF/HygT7mHPeWzal3/h1VHhQwOyOAAAIIIOAxAb+1/W7yeTahTJ8+vcaPH28TyaNHjyp79uyaOHGiwsLC9N133+mFF17Q3r17r2nzySefaNiwYdqyZYtMeVWqVNGECRMijpk0aZK6d++uTZs2KXfu3OrSpYvat28fK2+/BdXUDQf1zKgVSpY0ibYPaGQHybw2aZM+mrvTrpBjVspxYtt66JQavD03oqgnqxdQr7svTaTOhgACCCCAgJ8F/Nb2u2nt2YTypptusoljw4YNNXr0aLVs2dI+Ak+VKpXmzp1rf/73339Ha9OnTx+9/fbbNmGsXLmyjh07pilTpmjEiBH2mEWLFqlWrVq23BYtWmjBggXq3bu3/bx169ZBm/stqOZvO6oWnyyx92lGXadOkUzdx63TV0v2qE7RLPr0iUpBG0R1gJkqqOKA6REfvdCgiJ69o7AjZVMIAggggAAC8Sngt7bfTSvPJpS33367TFL52muvqW3btraH8eeff7bVIKjTAAAgAElEQVQWX375pXr06KFdu3ZFaWN6HEuVKiXTA9mgQYMo92nUqJFNMpcsuZRUma1Nmza2F9QESNKkSYNy91tQrdrzh5q+v9De48qe9ZUxbUp1/m61xq3ap7tK59DwR8sFdf/R7Rx+/oIKdZ8c8bFZlceszsOGAAIIIICA3wX81va76e3ZhNL0GDZu3FgnT55UunTpNH36dFWoUMFa3HfffUqWLJkdmBPV9vLLL2vcuHHauvW/6Wou3+/s2bM2QR00aJA6d+4c8dGcOXNUu3ZtLV++XOXLlw/K3W9Bdfmj6MDKNU9/sVzTNh7SQxXy6PX7Swd1/9fauWy/X3T8r3N2lw9blFdYyeyOlU1BCCCAAAIIxJeA39p+N508m1Camz516pRNCgsWLGh7KwOb6XksVKiQihQpEqWN6d3MnDmzypYtawfwmEflVatW1dChQ+3PNm7cqBIlSmjy5Mn2nczAduTIETty3IwwN4/Bg9n8FlT7jv+t6oNm2luc8lxNFcueXoFlEp+onl+97y4RzO1fc9873pytnUf/tPuYdzPNO5psCCCAAAII+F3Ab22/m96eTihje+NFixbV/v37lStXLvvI3Mxb2bdvX/uIfNu2bdqwYYOdesi8R2kG6gS28PBwpUiRwiaenTp1uubpTc+p+RfYDhw4oEqVKtmBQmaAj9e343/9o7L9ptnLHNOuqsrny6gmwxdozd7j6nhHIT3foKhjt3D/Bwu1fPcftjwz56WZ+5INAQQQQAABvwuQUP5Xg55OKE3i9+qrr2rZsmX2vUaTAJYrV84OtDEJoXkPMqqtcOHC2r59u5270vREms0kfAUKFFC/fv1UvXp1e/zixYvtgJ0rE0ozMjww/2V0wW4G/Zgk9crNLwnlP+EXVKTHpXcbv3iykmoVyaL6Q+Zo2+HTejmsmNrVLujY73mbL5brl42HbHmretbXzWm9NzG9YzdLQQgggAACiUaAhNIHCeW0adN011132QSybt26GjhwoH230fy3SeZWrFihn376KcqgNUni7t27dfDgwUifm8fdZcqUkXnHMtRH3n7voTQwRbpP1j/nL+jDFuUUVjKHfQRuHoX3a1JCLavmd+wLoeuYtfp22V6ZpcG3O7RGuGMXR0EIIIAAAgjEUoCE0gcJpRmAU7x4cX3xxRcyj6LNY+tAQmnmpzQTnpuKjGp7/PHH7RRBVyaUJpk0CemHH36Y6AflGLfAYJk3Hyij+8vnVpm+v+jE3+f01gNldF955x7bD56yWe/P3qFMaVNqRc/6sfy15TAEEEAAAQS8JUBC6YOEMnXq1LYHsn79+jp//rx9tzGQUJrR2GYeyjNnzkQZWT/88IMeeOABrVu3TiVLlrT77Nu3T7fccotef/11Pffcc/ZxuRmsYx6jBzYzPZGZ+DwxTBtk7vnKHsnC3Sfp3PmLET2WTv3ajl6+Vy/+sFbl892sMe2qOVUs5SCAAAIIIBCvAiSUPkgoc+bMaR9zt2rV6qqE8uOPP7aDbcy63lFtJgE1A2TMKPH+/fvb3k3z7qRZutGsmpM2bdqIic1Nb2bz5s3txOa9evVKNBObG7cGb8/R1kOn9VJYUT1Vo4CK9phiOUc9VUk1C2dx7JfUvK85Yc1+Vcx/s/JlSutYuRSEAAIIIIBAfAqQUPogoTRLIJrpgaZOnWqnCDI9lOa9yRw5cshMC3T33XfrzTffjDaOTPJo5pg0k6GfO3fOHmNWzjEjwAObKb9bt26Rll4MrB0ebID6MajuHb5Aq/ce17N1CtmE8rZXA6O+q9neRDYEEEAAAQQQiF7Aj22/W/Xp2VHeJ06cUL169bR27Vq76s3KlSvtgJqdO3fapHDmzJm68UbvTD/jx6BqMXKJ5m8/KjPvpEkoa7w+y8ZZYF5Kt4KOchFAAAEEEEgIAn5s+91y92xCaW7Y9CyaZRbNiO+jR48qY8aMNsk062+bx9he2vwYVIHpfB6skFtP1bhFDd+Za0kDK+d4yZdrQQABBBBAwGsCfmz73TL0dELp1k27Ua4fg6rLd6s19t+1u00PZbMr1vZ2w4kyEUAAAQQQSCgCfmz73bL3fEJpll5cunSpnZjcDNQx0wld/h6kWzDBluvHoOrx4zp9uXiPahfNYh95P/bJUnvbW/qHKVXyZMESsD8CCCCAAAKJSsCPbb9bFeTZhPL06dNq06aNvv/+e124cMEOyjGPwJMmTWqnBDIjvXmHMrSwGDhpk0bM3alK+TPqyRoF1PbLFUqeNIm2DWikJEmShFY4RyOAAAIIIJDABUgo/6tgzyaUTzzxhMaMGaN33nlHzZo100033WTnjTQ/M6O377vvPn366aeeCVU/BtWwGds0ZNpWlciZ3vZQdvl+jdKnTq61fRp6xpULQQABBBBAwKsCfmz73bL0bEKZPn16Owl5u3btrrr3999/X6+88orMSHCvbH4MqpHzdqr/z5uUP1Mam1D2HL9BOTOk1sJX6nqFletAAAEEEEDAswJ+bPvdwvRsQpk1a1a77GJYWNhV926WVWzRooUd+e2VzY9B9e3SPeo6dp2ypEtlE8pBkzerUNYbNb3L7V5h5ToQQAABBBDwrIAf2363MD2bUL788svavHmzfvzxx0jv8128eFFNmjSxA3PeeOMNt1yCLtePQWVWr+n0zSqlSZnMJpTvztyuMnlu0vgO1YO+fw5AAAEEEEAgsQn4se13q448lVAOGTIk4j7Dw8M1bNgwmTW9zao4psfSrH5j1vc+e/asOnXqpBdffNEtl6DL9WNQzdh0SE99vtze6+PV8uuzhbtUrWAmff10laDvnwMQQAABBBBIbAJ+bPvdqiNPJZRmBHdMNzMK2azZ7ZXNj0G1eOfvevijxZbw7jI59dOa/ap3azaNbFXBK6xcBwIIIIAAAp4V8GPb7xampxJKt24yLsr1Y1Ct++2E7n5vvuWpXCCjlvx6TPeWzal3Hr4tLsg4BwIIIIAAAr4W8GPb7xY4CaVDsn4Mqp1HTuuOt+ZYgVsyp9XOo3/q0cp59VrTUg6pUAwCCCCAAAIJV8CPbb9bteHphPL333/X8OHDNW/ePB07dsyu5V2rVi21b99emTJlcsskVuX6MagOnTyjyq/NsPd7Y6rkOn02XG1q3aJud94aKwMOQgABBBBAIDEJ+LHtd6t+PJtQ7tixwyaPZmqgatWqKXv27Dp48KAWLlyozJkza+7cuSpYsKBbLkGX68egOnXmnEr1+SXSvT5Xr7Ceq1ck6PvnAAQQQAABBBKbgB/bfrfqyLMJ5b333iuzjreZczJv3rwR97937147N2WRIkU0btw4t1yCLtePQXX+wkUV7DYp0r12v/NWPV3rlqDvnwMQQAABBBBIbAJ+bPvdqiPPJpQZMmTQJ598ovvvv/+qex89erRat27NSjkORMWtPafo73P/jZY370+a9yjZEEAAAQQQQODaAiSU//l4NqFMly6dRo0aJdNTeeVmeiZbtmypU6dOeSbW/RpUFfpP19HTZyMchz5cVk3K5vKMKxeCAAIIIICAVwX82va74enZhNI81j5y5IimTZtmB+MEtj/++EP169dXlixZNHnyZDdMYlWmX4Pq9jdmaffvf0Xc88iWFVSveLZYGXAQAggggAACiUnAr22/G3Xk2YRy/fr1qlmzpsyKOXfccYcdlHPo0CHNmDFDKVOmtINySpQo4YZJrMr0a1A1GjpPmw6cjLjnr5+urGoFM8fKgIMQQAABBBBITAJ+bfvdqCPPJpTmZk1FmeUY58+fL9MzaXoqa9Sooc6dOyt37txueMS6TL8G1QMfLtSyXX9E3PeEZ6urdO6bYu3AgQgggAACCCQWAb+2/W7UjycTyjNnzth1uh977DFVqlTJjft2vEy/BlWr/y3VnK1HIjymd7ldhbLe6LgPBSKAAAIIIJDQBPza9rtRD55MKM2N3njjjfr55591++23u3Hfjpfp16Bq/9UKTVp3MMJj0St3KEeGGxz3oUAEEEAAAQQSmoBf23436sGzCWWDBg1Ur149vfTSS27ct+Nl+jWoXhy9RqNX/BbhsbZPA6VPncJxHwpEAAEEEEAgoQn4te13ox48m1AuXrxYLVq0sMssNmrUSFmzZlWSJEkiGVw++tsNnGDK9GtQ9ZmwQZ8t3BVxq9sHNFLyZEmDuXX2RQABBBBAIFEK+LXtd6OyPJtQJk36X1JzZSIZgDh//r8Jud3ACaZMvwbVG1M3a/isHfZWU6dIqs2vNgrmttkXAQQQQACBRCvg17bfjQrzbEL52WefXdUjeSVAq1at3DCJVZl+Darhs7brjalb7D1nSptSK3rWj9X9cxACCCCAAAKJTcCvbb8b9eTZhNKNm3WzTL8G1WcLflWfnzZamjwZb9C8l+5wk4myEUAAAQQQSDACfm373agAzyeUJ06c0Lp163TgwAHlyJFDpUqVklnn22ubX4Nq9PK9evGHtZazWPZ0mvJcLa/Rcj0IIIAAAgh4UsCvbb8bmJ5NKC9cuKAePXro3Xff1Z9//hlx72nTptWzzz6r/v37K1myZG6YxKpMvwbVpHUH1P6rlfaey+e7WWPaVYvV/XMQAggggAACiU3Ar22/G/Xk2YTy+eeft8lk165d1axZMzvK+/DhwxozZowGDx5sk8q33nrLDZNYlenXoDKTmpvJzc1Wq0gWffGkPyaSj1UlcRACCCCAAAIOCvi17XeQIKIozyaUmTJlsnNQvvzyy1fd98CBA/Xmm2/q999/d8MkVmX6NaiW7zqm+z9cZO+5Ucns+qBF+VjdPwchgAACCCCQ2AT82va7UU+eTShvuukmjR49WvXrXz3q+JdfftGDDz6o48ePu2ESqzL9GlQb95/UncPm2Xu+r1xuvfVgmVjdPwchgAACCCCQ2AT82va7UU+eTShbt26tixcv6pNPPrnqvp988kn7s//9739umMSqTL8G1Z7f/1KtN2bZe25VNZ/6NikZq/vnIAQQQAABBBKbgF/bfjfqybMJ5RdffKHu3burUKFCuvfeeyPeoRw3bpx27NihAQMG2PW+A5t5zzI+N78G1dHTZ1Wh/3RL1652Qb0cViw+GTk3AggggAACvhHwa9vvBrBnE8rLV8q53o2blXTie9UcvwbVmXPnVaznFEv8YsOi6lCn0PW4+RwBBBBAAAEEJPm17Xej8jybUO7evTuo+82XL19Q+zu9s1+DyrxWULDbJF24KPW+u7ieqF7AaRrKQwABBBBAIEEK+LXtd6MyPJtQunGzbpbp56Aq3WeqTp4J1+D7S+vBCnncZKJsBBBAAAEEEoyAn9t+pyuBhNIhUT8H1bsztmnG5sP6uGUFZUmXyiERikEAAQQQQCBhC/i57Xe6ZkgoHRIlqByCpBgEEEAAAQR8IkDb/19FkVA6FLQElUOQFIMAAggggIBPBGj7SSgdD1WCynFSCkQAAQQQQMDTArT9Hk0oM2bMqOnTp6tcuXIyk5f37NlTBQr4Y9QxQeXp33kuDgEEEEAAAccFaPs9mlCmTp1aM2fOVLVq1WTmoVy8eLEqVarkeAC4USBB5YYqZSKAAAIIIOBdAdp+jyaUZcqUUd68eXXffffZHspevXrplltuiTaSWrZs6ZkoI6g8UxVcCAIIIIAAAnEiQNvv0YRy2rRpatWqlQ4ePCiz+o2ZdDu6zQur41x+bQRVnPzuchIEEEAAAQQ8I0Db79GEMnBZx48fl3mfcurUqfZ9yui2TJkyxSioTp8+rWLFimnfvn1atmyZKlSoEHHcpEmT7JrhmzZtUu7cudWlSxe1b98+RuWSUAbNxAEIIIAAAggkGAESSo8nlObyPv/8czVu3FgxTRqvFZ0vv/yyLe/QoUOREspFixapVq1aMo/OW7RooQULFqh3794aMWKEWrduHVTAE1RBcbEzAggggAACvheg7fdBQhm4xF27dtlE79ixY7bXskaNGgpm3e7NmzfbHsm33npLbdu2jZRQNmrUyJa7ZMmSCJE2bdpo4sSJdsF3MzAophtBFVMp9kMAAQQQQCBhCND2+yChPH/+vE0AP/30U124cCHiik2SZwbsfPjhhzFK+Bo0aKDSpUvb3s46depEJJRnz55V+vTpNWjQIHXu3Dmi/Dlz5qh27dpavny5ypcvH+OIJ6hiTMWOCCCAAAIIJAgB2n4fJJRmDsrBgwerb9++euSRR5Q9e3Y7WOfbb7+1j6XNY2zz2bW2H374QR06dNC2bdu0cuXKSAnlxo0bVaJECU2ePFlhYWERxRw5ckRZs2bVqFGj7GPwmG4EVUyl2A8BBBBAAIGEIUDb74OE0jzWbteunbp27XpV1JlexQ8++EC7d++ONiL/+usvOxCnT58+tkdz9uzZkRJK8xjdPD4371FWqVIlopzw8HClSJFCQ4cOVadOnaIt/+TJkzL/AtuBAwfsnJl79+61g3vYEEAAAQQQQCBhC5BQ+iChNJOc//TTT6pfv/5V0WimF7r77rt15syZaCO1W7duMvuZ9yPNY/LoEkozeXrlypWvSiiHDRumjh07Rlu+SVSj6iEloUzYXx7cHQIIIIAAAgEBEkofJJTFixdXvXr1ZBK7KzfTc2iWaDSPraPaTM9lkSJFNG7cOLvqjtnmz59vk9BZs2bZQTp79uwJ6ZE3PZR8oSCAAAIIIJC4BUgofZBQfvTRR3ZQzkMPPaQHH3zQvkNppv35/vvv9d1339lBOU8//XSUkRzojYwuzE2PpBl8w6CcxP1FwN0jgAACCCAQigAJpQ8SSnOJ7777rl599VUdPXo0YuWcLFmy2CUZzWCb6DYzMfrq1asjfWz+24zmNoloxYoV7YTpZtogs695jzKwmSR2woQJTBsUym8YxyKAAAIIIJAIBEgofZJQmss0UwaZuST/+OMPOw9l0aJFYzRd0JVxfOU7lObzwMTmjz/+uJo3b27nuzTJKhObJ4JvAW4RAQQQQACBEAVIKH2UUIZY1xGHR5VQmg/N0otmAM/lSy9eq/czuushqJyqKcpBAAEEEEDAHwK0/YkwoXQ7NAkqt4UpHwEEEEAAAW8J0PaTUDoekQSV46QUiAACCCCAgKcFaPtJKB0PUILKcVIKRAABBBBAwNMCtP0klI4HKEHlOCkFIoAAAggg4GkB2n4SSscDlKBynJQCEUAAAQQQ8LQAbb9PEkqzLOLo0aPt+thXLrOYJEkSjR8/3jOBRlB5piq4EAQQQAABBOJEgLbfBwnl0KFD7UTkWbNmVcGCBZUyZcqrgsMso+iVjaDySk1wHQgggAACCMSNAG2/DxLKPHnyqGnTpnrnnXdiNZF53ITSf2chqOJanPMhgAACCCAQvwK0/T5IKDNlymTX7a5bt278RksMz05QxRCK3RBAAAEEEEggArT9Pkgo27Vrp7Rp0+rNN9/0RdgRVL6oJi4SAQQQQAABxwRo+32QUJ49e1ZPPfWUXcu7Xr16uummm64KgGbNmjkWFKEWRFCFKsjxCCCAAAII+EuAtt8HCeWqVavsO5R79uyJMrrMKO/z5897JvIIKs9UBReCAAIIIIBAnAjQ9vsgoSxXrpzCw8M1cOBAFSlSJMpR3vny5YuTgInJSQiqmCixDwIIIIAAAglHgLbfBwllmjRpNHbsWIWFhfki8ggqX1QTF4kAAggggIBjArT9Pkgoq1Spovbt26tly5aOVbybBRFUbupSNgIIIIAAAt4ToO33QUK5cuVKOyhn8ODBqlOnjpInT+69SLrsiggqT1cPF4cAAggggIDjArT9Pkgo06VLp3Pnztl/SZMm1Q033BApEMygnBMnTjgeHLEtkKCKrRzHIYAAAggg4E8B2n4fJJR9+vSRSRqvtfXu3dszEUhQeaYquBAEEEAAAQTiRIC23wcJZZxEgoMnIagcxKQoBBBAAAEEfCBA209C6XiYElSOk1IgAggggAACnhag7fdBQnnHHXdcN4hmzpx53X3iageCKq6kOQ8CCCCAAALeEKDt90FC2bhx46veoTx27JjWrl1rl2G87bbbNGHCBG9ElCSCyjNVwYUggAACCCAQJwK0/T5IKKOLhMOHD+uee+7R888/rwceeCBOAiYmJyGoYqLEPggggAACCCQcAdp+HyeU5tLHjRunrl27asuWLZ6JSoLKM1XBhSCAAAIIIBAnArT9Pk8ox4wZoyeeeEInT56Mk4CJyUkIqpgosQ8CCCCAAAIJR4C23wcJpVkp58rtn3/+0aZNm9S3b18VKlRI06dP90xUElSeqQouBAEEEEAAgTgRoO33QUJpVse5cmLzixcv2iuvXLmyvv76axUoUCBOAiYmJyGoYqLEPggggAACCCQcAdp+HySUc+bMuSriUqdOrdy5cytXrlyei0aCynNVwgUhgAACCCDgqgBtvw8SSlcjwIXCCSoXUCkSAQQQQAABDwvQ9pNQOh6eBJXjpBSIAAIIIICApwVo+z2aUKZLl+6q9yajiyTzfuWJEyc8E2gElWeqggtBAAEEEEAgTgRo+z2aUPbp0+e6CeX8+fM1Y8YMu9/58+fjJGBichKCKiZK7IMAAggggEDCEaDt92hCea0QW7BggUzCaZLJ0qVLq1evXmrWrJlnopKg8kxVcCEIIIAAAgjEiQBtv48Synnz5tl5J2fNmqUyZcrYRPLee++Nk0AJ5iQEVTBa7IsAAggggID/BWj7fZBQzp071/ZIzp49W+XKlbOJpFnD26sbQeXVmuG6EEAAAQQQcEeAtt/DCaVJIE2PpJmHsmLFijaRvOuuu9yJBAdLJagcxKQoBBBAAAEEfCBA2+/RhLJ27doyj7jNSjgmkQwLC/NBOF26RILKN1XFhSKAAAIIIOCIAG2/RxNKs9yi2dKmTXvd0d5MG+TI7wKFIIAAAggggEAsBUgoPZpQmkfdwWy9e/cOZndX9yWoXOWlcAQQQAABBDwnQNvv0YTSc5ESxAURVEFgsSsCCCCAAAIJQIC2n4TS8TAmqBwnpUAEEEAAAQQ8LUDbT0LpeIASVI6TUiACCCCAAAKeFqDtJ6F0PEAJKsdJKRABBBBAAAFPC9D2k1A6HqAEleOkFIgAAggggICnBWj7SSgdD1CCynFSCkQAAQQQQMDTArT9JJSOByhB5TgpBSKAAAIIIOBpAdr+BJ5Qjh49Wl999ZVWrFihY8eOqWDBgmrXrp2eeeYZBSZPNwSTJk1S9+7dtWnTJuXOnVtdunRR+/btYxW8BFWs2DgIAQQQQAAB3wrQ9ifwhLJKlSrKly+fmjZtqmzZsmnWrFkaOHCgnnvuOb3xxhv27hctWqRatWqpZcuWatGihRYsWCAzUfqIESPUunXroIOboAqajAMQQAABBBDwtQBtfwJPKI8cOaIsWbJEClLT+/jBBx/o+PHjSpUqlRo1amR7L5csWRKxX5s2bTRx4kS7LvflPZkxiXaCKiZK7IMAAggggEDCEaDtT+AJZVShOmrUKNsbuX//fmXMmFHp06fXoEGD1Llz54jd58yZo9q1a2v58uUqX758UBFPUAXFxc4IIIAAAgj4XoC2PxEmlKb3ccyYMTp8+LC2bNmiEiVKaPLkyQoLC4vQMD2bWbNmlUk+zWPwYDaCKhgt9kUAAQQQQMD/ArT9iSyhND2OVatWte9I9ujRw74vWaNGDfsepXnfMrCFh4crRYoUGjp0qDp16nTNSD958qTMv8B24MABVapUSXv37rUDfNgQQAABBBBAIGELkFAmooTy4MGDqly5sk3yZs+ebRPGQEK5ePFi+9mVCeWwYcPUsWPHa/4W9OnTR3379r1qHxLKhP3lwd0hgAACCCAQECChTCQJ5YkTJ+w7kWfOnNH8+fOVKVMme+cbN24M+ZE3PZR8oSCAAAIIIJC4BUgoE0FCaZLIhg0batu2bfbRtplGKLCdPXuWQTmJ+zuAu0cAAQQQQCBkARLKBJ5QmnchmzVrprlz59p/pUuXvipozLRBZgohk2wGtrZt22rChAlMGxTyrxgFIIAAAgggkPAFSCgTeEJpVsT56KOPNHjwYNWsWTNSRBcvXtz2TgYmNn/88cfVvHlz+15lr169mNg84f/+c4cIIIAAAgg4IkBCmcATyvz582v37t1RBotZNce8V2k2s/Rit27dIi292KFDh1gFGUEVKzYOQgABBBBAwLcCtP0JPKGMj8gkqOJDnXMigAACCCAQfwK0/SSUjkcfQeU4KQUigAACCCDgaQHafhJKxwOUoHKclAIRQAABBBDwtABtPwml4wFKUDlOSoEIIIAAAgh4WoC2n4TS8QAlqBwnpUAEEEAAAQQ8LUDbT0LpeIASVI6TUiACCCCAAAKeFqDtJ6F0PEAJKsdJKRABBBBAAAFPC9D2k1A6HqAEleOkFIgAAggggICnBWj7SSgdD1CCynFSCkQAAQQQQMDTArT9JJSOByhB5TgpBSKAAAIIIOBpAdp+EkrHA5SgcpyUAhFAAAEEEPC0AG0/CaXjAUpQOU5KgQgggAACCHhagLafhNLxACWoHCelQAQQQAABBDwtQNtPQul4gBJUjpNSIAIIIIAAAp4WoO0noXQ8QAkqx0kpEAEEEEAAAU8L0PaTUDoeoASV46QUiAACCCCAgKcFaPtJKB0PUILKcVIKRAABBBBAwNMCtP0klI4HKEHlOCkFIoAAAggg4GkB2n4SSscDlKBynJQCEUAAAQQQ8LQAbT8JpeMBSlA5TkqBCCCAAAIIeFqAtp+E0vEAJagcJ6VABBBAAAEEPC1A209C6XiAElSOk1IgAggggAACnhag7SehdDxACSrHSSkQAQQQQAABTwvQ9pNQOh6gBJXjpBSIAAIIIICApwVo+0koHQ9QgspxUgpEAAEEEEDA0wK0/SSUjgcoQeU4KQUigAACCCDgaQHafhJKxwOUoHKclAIRQAABBBDwtABtPwml4wFKUDlOSoEIIIAAAgh4WoC2n4TS8QAlqBwnpUAEEEAAAQQ8LUDbT7RS1UUAABAkSURBVELpeIASVI6TUiACCCCAAAKeFqDtJ6F0PEAJKsdJKRABBBBAAAFPC9D2k1A6HqAEleOkFIgAAggggICnBWj7SSgdD1CCynFSCkQAAQQQQMDTArT9JJSOByhB5TgpBSKAAAIIIOBpAdp+EkrHA5SgcpyUAhFAAAEEEPC0AG0/CaXjAUpQOU5KgQgggAACCHhagLafhNLxACWoHCelQAQQQAABBDwtQNtPQul4gBJUjpNSIAIIIIAAAp4WoO0noXQ8QAkqx0kpEAEEEEAAAU8L0PaTUDoeoASV46QUiAACCCCAgKcFaPtJKB0PUILKcVIKRAABBBBAwNMCtP0klI4HKEHlOCkFIoAAAggg4GkB2n4SSscDlKBynJQCEUAAAQQQ8LQAbT8JpeMBSlA5TkqBCCCAAAIIeFqAtp+E0vEAJagcJ6VABBBAAAEEPC1A209C6XiAElSOk1IgAggggAACnhag7SehjBDYunWrOnXqpHnz5ilt2rR65JFHNGjQIN1www1BBTFBFRQXOyOAAAIIIOB7Adp+EkorcPz4cZUsWVL58uVTz549dfjwYXXp0kVhYWH68ssvgwp0giooLnZGAAEEEEDA9wK0/SSUVuD1119Xv379tHv3bmXOnNn+7Ouvv1bz5s21ceNG3XrrrTEOdoIqxlTsiAACCCCAQIIQoO0nobQCt99+u2666SaNHz8+QuTs2bPKkCGDBgwYoOeffz7GAU9QxZiKHRFAAAEEEEgQArT9JJRWIGvWrHryySftO5OXbyVKlFDVqlU1cuTIGAc8QRVjKnZEAAEEEEAgQQjQ9pNQWoEUKVLo1VdfVdeuXSMFdo0aNWyyOXbs2GgD/uTJkzL/AtvevXtVrVo1LV26VDly5EgQvyjcBAIIIIAAAghEL3DgwAFVqlRJv/76q/Lnz5+oqZJcvHjxYmIVMAll//799fLLL0ciqF69urJnz64xY8ZES9OnTx/17ds3sdJx3wgggAACCCDwr4DpTKpYsWKi9kjUCWUoj7yv7KE8c+aMTC9lgQIFlDx5ckeDKvAXEL2fjrIGXRj1EDSZKwdQD66wBlUodRAUl2s7Uw+u0ca44PDwcB05ckSlSpVS6tSpY3xcQtwxUSeUTg7KcTM4eEfDTd2Yl009xNzKzT2pBzd1Y1Y2dRAzJ7f3oh7cFqb8YAQSdUJppg0y71CaaYMyZcpk3b799ls7uXmw0wYFgx7svnxpBCvmzv7UgzuuwZZKPQQr5vz+1IHzprEpkXqIjRrHuCWQqBPKwMTm5kXayyc2b9iwYdATm7tVQaZcvjTc1I152dRDzK3c3JN6cFM3ZmVTBzFzcnsv6sFtYcoPRiBRJ5QGyiy92LFjR82fP19p0qSxvZOm5zLYpReDQQ92X/O+5pAhQ+wqPunTpw/2cPZ3SIB6cAgyxGKohxABHTicOnAA0YEiqAcHECnCMYFEn1A6JklBCCCAAAIIIIBAIhUgoUykFc9tI4AAAggggAACTgmQUDolSTkIIIAAAggggEAiFSChTKQVz20jgAACCCCAAAJOCZBQOiXpQjlmwFCnTp00b948pU2b1g4YMuuOe2nAkAu3HW9Fjh49Wl999ZVWrFihY8eOqWDBgmrXrp2eeeYZJU2aNOK6Jk2apO7du2vTpk3KnTu3HSzVvn37eLvuhH7i06dPq1ixYtq3b5+WLVumChUqUBdxWOmffPKJhg0bpi1btthBgVWqVNGECROogzisgx9//FEDBw603znm+9+s5mb+u2jRopGugu+mOKwUTnWVAAmlR4MiMKVRvnz5Ik1pFBYW5qkpjTzKF6vLMg2l8W7atKmyZcumWbNm2S/t5557Tm+88YYtc9GiRapVq5ZatmypFi1aaMGCBerdu7dGjBih1q1bx+q8HHRtAbM06ueff65Dhw5FSiipC/cjxywx+/bbb9s/oCpXrmz/0JoyZYqNd34f3Pc3Z5g+fboaNGhgv28ee+wxmbbB1IsZ4b1hw4aImT/4fYib+uAs0QuQUHo0OszURf369bOTrmfOnNle5ddff63mzZt7atJ1j/LF6rLM8llZsmSJdKzpffzggw/sl3iqVKnUqFEj26guWbIkYr82bdpo4sSJdr7Qy3syY3URHBRJYPPmzbZH8q233lLbtm0jJZTUhbvBYnrDzHJyptfLJDRRbdSBu3VgSjd/qM6YMUM7d+5UkiRJ7AnNMrwmwTd1Y+rAbNSF+3XBGa4tQELp0Qjxy7KQHuVz7LJGjRpleyP379+vjBkz2t4A89pB586dI84xZ84c1a5dW8uXL1f58uUdOzcFySYypUuXVuPGjVWnTp2IhPLs2bPUhcsBYnqGx40bZ+fqjWqjDlyugH+Lb9WqlVatWqW1a9dGnNDUiXnc/fPPP+vOO+8UdRE3dcFZSCh9GQNZs2bVk08+aZOXy7cSJUqoatWqGjlypC/vy28XbXofx4wZo8OHD9t3yIz/5MmTZV49CGymZ9PUl0k+zWMpNmcEfvjhB3Xo0EHbtm3TypUrIyWUZmlU6sIZ5+hKMX/UmqcjZcuW1bvvvmt76c13z9ChQ+3PqAN3/QOlz507V3Xr1rWLWwQeeZt367dv324TTfPkhLqIm7rgLCSUvoyBFClS2HXGu3btGun6a9SoYZOXsWPH+vK+/HTRpsfRNKDmHckePXrY9yWNv3lXybxvGdjCw8Nl6ss0tOaLni10gb/++ssOxDHvipk/rGbPnh0poaQuQje+XgmmB8z0zOfKlUuvvfaaUqZMqb59+2rXrl02yTfv7/H7cD1FZz43r9Q8+uijOnXqlC2wePHimjp1qh0UaDZ+H5xxppTQBHjkHZqfa0ebBKV///4yj50u38zovuzZs9teMzb3BA4ePGjfUTJf2CaZMfUR+NJevHix/ezKhNKMhDXLeLKFLtCtWzdNmzbNvqtq3kuNLqGkLkK3jq6EwoUL216w9evX295gsx04cEAFChSw73eb7yKTUFIH7tWBKXnhwoX2sfbjjz+ue+65RydOnLAJ/pkzZ+x3knkNh+8md+uA0mMmQEIZM6c434tH3nFOHnFC84Vt3ok0X9hmjfdMmTLZz3isFDd1YgaiFSlSxL6/V61aNXtSUw933323HXlvBuns2bOHR94uV4f5o8nUhfnj6vLNPO4uU6aM/WOX1w5crgTJxnvevHkjPZUyr9mYP3bNLBRm4CDfTe7XA2e4vgAJ5fWN4mUPBuXEC7tNIhs2bGgf6ZlH22YaocDGi+9xUyeB3sjozmYSHTMQigFS7taH6REzUwRdmVCaZLJcuXL68MMPqQN3q8CWniZNGvvqU69evSKdzfQUm57L4cOHMygnDuqBU1xfgITy+kbxsoeZNsi8Q2l6CAI9ZN9++62d3Nz8NXrrrbfGy3Ul5JOadyGbNWsm8xK8+WdGF1+5mak5zOAEk2wGNjOdjZnomWmDnIkO47t69epIhZn/NiPrTRJTsWJFm9BQF854R1eKGRT1wAMPaN26dSpZsqTdzUwuf8stt8h8P5n5WakDd+vAlG6+602P/fjx4yNOZpJ802s5YMAAvfjii/bn1IX7dcEZri1AQunRCAlMbJ4/f/5IE5ub3rMvv/zSo1ft78syK+J89NFHGjx4sGrWrBnpZsxL8KZHLDB5sOm9MXOCmneXTM8BE5u7W/dXvkNpzkZduGt+/vx5VapUyQ4EMe9zm0E55t3JwIwHZvUu6sDdOjClv/fee/bdbDPjQZMmTewftOYdStPZYAZG5ciRw14EdeF+XXAGEkrfxoCZa8x8kZj3x8xjD9M7aXoGWHrRnSo1ybv5ko5qM+/umfcqzWYmEzaDRi5fetF82bO5JxBVQklduOcdKNkkj6Zn2Mx3eO7cOZlXcczKOZcv+cfvg7v1cPHiRX388cd6//337SCpG2+80Sb6pnfSTDx/+UZduFsXlE5CSQwggAACCCCAAAIIuCjAI28XcSkaAQQQQAABBBBIDAIklImhlrlHBBBAAAEEEEDARQESShdxKRoBBBBAAAEEEEgMAiSUiaGWuUcEEEAAAQQQQMBFARJKF3EpGgEE3BMw63ybtaUDm5mv1az/bUbgmwmf3diiG23uxrkCZSZJkkRvvPGGXnjhhWhPY9adN/NzXj4bgZvXRNkIIIDAlQIklMQEAgj4UsAklGbO0JkzZ9rr379/vwYNGqQVK1Zo3rx5Ecs2OnlzJ0+etAsLmOlazDyMcbGZtbLNik2B+QajOicJZVzUBOdAAIFrCZBQEh8IIOBLAZNQvvnmmzp9+nTE9R84cEC5cuXS008/bSebTywbCWViqWnuEwHvCpBQerduuDIEELiGQFQJpdk9W7ZsdmnGyZMnRxxtVhHp3r27lixZouTJk+uuu+7SO++8o6xZs0bsY5bONMtozpgxQxkzZrRLC5qJ7idOnKhdu3bZ/aJ65G3Wfzdlm6VRjx49apfJe+WVV/Too49GlG1WVjJJn1n1xEwUbhYtKFGihD744AOVL1/+mvUc1SNvs3KNKcsk0w0aNFDr1q3tPfHIm18ZBBCILwESyviS57wIIBCSQFQJpUmwbr75ZpllNE3CZTaTTJpVjsx7lU899ZT+/PNP9ejRQ+adS/M42WxmNZIKFSro0KFDGjhwoDJkyGAfp5uEMlmyZNdMKO+77z67epJJ8sya1yax/Oyzz/TFF1/oscces+WbhNIkpjlz5lTXrl3tMp7mf80j9B07dihFihTRWlyZUAaW4jPvVNarV0+//PKLvvvuO7vONgllSCHFwQggEIIACWUIeByKAALxJxBIKM3axmYzj7tNkjZlyhSbKBYuXNj+3CwXGB4ebpcwNcmZ2cwayOY9SJPkmUTTJISmh2/u3LkR67ibZC937ty2tzK6Hsq1a9eqTJkyGj58uNq3bx+B0bBhQ23ZsiXiOJNQmgRz3bp1tmfSbNOnT1f9+vXt+541atSIUUJp1tfOmzev6tata8sLbKY39JtvviGhjL9w5MwIJHoBEspEHwIAIOBPgStHeZu7ML2JEyZMiBjl/ddff9neQPOu5bPPPhvpRs1AlzZt2qh37952tLh5BP7HH39E2qdJkyZas2ZNtAmlSSRNueZRt+nxDGymh/KJJ57Qnj17lCdPHttDaRJI81g9sJlBROZ9T5MIPvzwwzFKKE2PqVlzfuzYsWratGnEMea/TU8pPZT+jGWuGoGEIEBCmRBqkXtAIBEKBEZ5m17FCxcuaNu2bbaH0iSR69evt6OizWNg08sY3WYegY8cOdK+O2lGi5t3Gy/fzOCeadOmRZtQmsfcJhk9d+5cpONML2mjRo20evVq24MZeIfSXFdgM0lolixZ9Omnn9rPo9suf+Rtel6rVq1qe1urV68ecciCBQtsLycJZSL8ReCWEfCIAAmlRyqCy0AAgeAEonqH0kwZVKlSJdvzaAa8mPcl06VLZ+emvPfee686QebMmW2PX2x7KN9//3116NBBv//+u300Htii6qE0g3JCTSjpoQwuRtgbAQTiTuD/Af8vrHtqG2laAAAAAElFTkSuQmCC\" width=\"599.9999869953505\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "plt.style.use('default')\n",
    "#plt.style.use('dark_background')\n",
    "\n",
    "# Plot number of probes per region\n",
    "plt.figure(figsize=(6,3), dpi=100)\n",
    "plt.plot(list(reg_size_dic.values()))\n",
    "#plt.plot(np.ones(len(pb_nums))*20)\n",
    "plt.ylabel(f\"Num of probes\", labelpad=1)\n",
    "plt.xlabel(f\"Region id\", labelpad=1)\n",
    "plt.ylim([0, np.max(list(reg_size_dic.values()))*1.1])\n",
    "plt.savefig(os.path.join(report_folder, 'afterblast_nProbes_per_region.png'),\n",
    "            transparent=True, dpi=300)\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a9429fce",
   "metadata": {},
   "source": [
    "# Create count table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3512c771",
   "metadata": {},
   "outputs": [],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "from tqdm import tqdm\n",
    "overwrite_table = False\n",
    "# library\n",
    "library_table_file = os.path.join(reference_folder, 'ctp11-12-13-mouse-chr7_17w.npy')\n",
    "\n",
    "if not os.path.exists(library_table_file) or overwrite_table:\n",
    "    # library\n",
    "    _library_filenames = [os.path.join(library_folder, 'blast_full_probes.fasta')]\n",
    "    print(len(_library_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=library_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_library_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=44) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {library_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a4d77713",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "postanalysis",
   "language": "python",
   "name": "postanalysis"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
