{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "deadly-absence",
   "metadata": {},
   "source": [
    "# Design CTP-10 Aire DNA-MERFISH library for all super enhancers\n",
    "\n",
    "by Pu Zheng\n",
    "\n",
    "2021.4.5\n",
    "\n",
    "Super-enhancers are called by ..."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "delayed-belgium",
   "metadata": {},
   "source": [
    "# Table of contents\n",
    "\n",
    "\n",
    "> 0. [Minimum required packages and settings](#0)\n",
    ">>\n",
    ">> 0.1: [import required packages](#0.1)\n",
    ">\n",
    "> 1. [Extract region sequences](#1)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fuzzy-reasoning",
   "metadata": {},
   "source": [
    "<a id='0.1'></a>\n",
    "## 0.1 load required packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "packed-worse",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "15908\n"
     ]
    }
   ],
   "source": [
    "%run \"..\\..\\Startup_py3.py\"\n",
    "sys.path.append(r\"..\\..\\..\\..\\Documents\")\n",
    "\n",
    "import ImageAnalysis3 as ia\n",
    "%matplotlib notebook\n",
    "\n",
    "from ImageAnalysis3 import *\n",
    "print(os.getpid())\n",
    "\n",
    "# library design specific tools\n",
    "from ImageAnalysis3.library_tools import LibraryDesigner as ld\n",
    "from ImageAnalysis3.library_tools import LibraryTools as lt\n",
    "# biopython imports\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "from Bio.Blast.Applications import NcbiblastnCommandline\n",
    "from Bio.Blast import NCBIXML"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "severe-obligation",
   "metadata": {},
   "source": [
    "<a id='1'></a>\n",
    "# 1 Extract region sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "single-palmer",
   "metadata": {},
   "outputs": [],
   "source": [
    "## Some folders\n",
    "# human genome\n",
    "reference_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl'\n",
    "genome_folder = os.path.join(reference_folder, 'Genome')\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "sunset-queen",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- library_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\n",
      "-- sequence_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\n",
      "-- report_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\reports\n"
     ]
    }
   ],
   "source": [
    "resolution = 0\n",
    "flanking = 10000\n",
    "# folder for sub-pool\n",
    "library_folder = os.path.join(pool_folder, f'Genes_TSS_DNA')\n",
    "if not os.path.exists(library_folder):\n",
    "    print(f\"create library folder: {library_folder}\")\n",
    "    os.makedirs(library_folder)\n",
    "# folder for fasta sequences\n",
    "sequence_folder = os.path.join(library_folder, 'sequences')\n",
    "if not os.path.exists(sequence_folder):\n",
    "    print(f\"create sequence folder: {sequence_folder}\")\n",
    "    os.makedirs(sequence_folder)\n",
    "# folder to save result probes\n",
    "report_folder = os.path.join(library_folder, 'reports')\n",
    "if not os.path.exists(report_folder):\n",
    "    print(f\"create report folder: {report_folder}\")\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "print(f\"-- library_folder: {library_folder}\")\n",
    "print(f\"-- sequence_folder: {sequence_folder}\")\n",
    "print(f\"-- report_folder: {report_folder}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "improved-cabinet",
   "metadata": {},
   "source": [
    "## 1.1 load gene list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "willing-award",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Cluster</th>\n",
       "      <th>Gene</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Immature MEC</td>\n",
       "      <td>Ccl21a</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Immature MEC</td>\n",
       "      <td>Krt14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Immature MEC</td>\n",
       "      <td>Krt5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Immature MEC</td>\n",
       "      <td>Col6a1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Immature MEC</td>\n",
       "      <td>Lifr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>204</th>\n",
       "      <td>Aire-stage</td>\n",
       "      <td>Ltf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>205</th>\n",
       "      <td>Aire-stage</td>\n",
       "      <td>Clps</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>206</th>\n",
       "      <td>Aire-stage</td>\n",
       "      <td>Col1a1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>207</th>\n",
       "      <td>Aire-stage</td>\n",
       "      <td>Gpx6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>208</th>\n",
       "      <td>Aire-stage</td>\n",
       "      <td>Plvap</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>209 rows Ã— 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "          Cluster    Gene\n",
       "0    Immature MEC  Ccl21a\n",
       "1    Immature MEC   Krt14\n",
       "2    Immature MEC    Krt5\n",
       "3    Immature MEC  Col6a1\n",
       "4    Immature MEC    Lifr\n",
       "..            ...     ...\n",
       "204    Aire-stage     Ltf\n",
       "205    Aire-stage    Clps\n",
       "206    Aire-stage  Col1a1\n",
       "207    Aire-stage    Gpx6\n",
       "208    Aire-stage   Plvap\n",
       "\n",
       "[209 rows x 2 columns]"
      ]
     },
     "execution_count": 49,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gene_list_folder = os.path.join(pool_folder, 'Gene_list')\n",
    "gene_list_filename = os.path.join(gene_list_folder, 'uniqued_clustered_genes_for_yuan_2021-04-22.txt')\n",
    "import pandas as pd\n",
    "gene_df = pd.read_csv(gene_list_filename, delimiter = \"\\t\", header=None)\n",
    "gene_df.columns = ['Cluster', 'Gene']\n",
    "gene_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "careful-stroke",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['Aire-stage', 'Ciliated', 'Gut/Liver', 'Immature MEC', 'Lung',\n",
       "       'Mcell', 'Muscle', 'Neuroendocrine', 'Skin', 'Tuft1', 'Tuft2'],\n",
       "      dtype='<U14')"
      ]
     },
     "execution_count": 53,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "np.unique(gene_df['Cluster'].tolist())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "registered-involvement",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- loading from save_file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Transcriptome\\Mus_musculus.GRCm38.102.chr.pkl\n",
      "opening ref_file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Transcriptome\\Mus_musculus.GRCm38.102.chr.gff3\n"
     ]
    }
   ],
   "source": [
    "# load gene reference\n",
    "reload(library_tools.references)\n",
    "reload(library_tools.sequences)\n",
    "ref_filename = os.path.join(reference_folder, 'Transcriptome', 'Mus_musculus.GRCm38.102.chr.gff3')\n",
    "with library_tools.references.gff3_reader(ref_filename, auto_read=False, load_savefile=True) as ref_rd:\n",
    "    ## example commands\n",
    "    #infos = _handle.load_all()\n",
    "    #ref_rd._load_headers()\n",
    "    #gene_infos = ref_rd.load_gene_by_id('ENSMUSG00000064842')\n",
    "    #gene_dict = ref_rd.parse_gene_info(gene_infos)\n",
    "    #gene_info_dict = ref_rd._batch_parse_gene_info()\n",
    "    ref_save_dict = ref_rd._save_to_file(overwrite=False)\n",
    "# searching example\n",
    "matched_gene = ref_rd._search_gene_by_id('gene:ENSMUSG00000089613')\n",
    "print(matched_gene[0]['infos']['Name'])\n",
    "matched_gene = ref_rd._search_gene_by_name('Ccl21a')\n",
    "print(matched_gene[0]['infos']['ID'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "id": "functional-holiday",
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm.notebook import tqdm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "id": "humanitarian-carbon",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "20f3fc377e5a454196dc64b6ada6312d",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "  0%|          | 0/209 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "gene_dicts = []\n",
    "for _gene in tqdm(gene_df['Gene']):\n",
    "    if _gene == 'Lect1': # there is one gene that actually has different name\n",
    "        _gene = \"Cnmd\" \n",
    "    _gds = ref_rd._search_gene_by_name(_gene)\n",
    "    if len(_gds) == 1:\n",
    "        gene_dicts.append(_gds[0])\n",
    "    else:\n",
    "        print(_gene)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "id": "patient-homework",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Cluster</th>\n",
       "      <th>Gene</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Immature MEC</td>\n",
       "      <td>Ccl21a</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Immature MEC</td>\n",
       "      <td>Krt14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Immature MEC</td>\n",
       "      <td>Krt5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Immature MEC</td>\n",
       "      <td>Col6a1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Immature MEC</td>\n",
       "      <td>Lifr</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>204</th>\n",
       "      <td>Aire-stage</td>\n",
       "      <td>Ltf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>205</th>\n",
       "      <td>Aire-stage</td>\n",
       "      <td>Clps</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>206</th>\n",
       "      <td>Aire-stage</td>\n",
       "      <td>Col1a1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>207</th>\n",
       "      <td>Aire-stage</td>\n",
       "      <td>Gpx6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>208</th>\n",
       "      <td>Aire-stage</td>\n",
       "      <td>Plvap</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>209 rows Ã— 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "          Cluster    Gene\n",
       "0    Immature MEC  Ccl21a\n",
       "1    Immature MEC   Krt14\n",
       "2    Immature MEC    Krt5\n",
       "3    Immature MEC  Col6a1\n",
       "4    Immature MEC    Lifr\n",
       "..            ...     ...\n",
       "204    Aire-stage     Ltf\n",
       "205    Aire-stage    Clps\n",
       "206    Aire-stage  Col1a1\n",
       "207    Aire-stage    Gpx6\n",
       "208    Aire-stage   Plvap\n",
       "\n",
       "[209 rows x 2 columns]"
      ]
     },
     "execution_count": 94,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gene_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 111,
   "id": "valued-completion",
   "metadata": {},
   "outputs": [],
   "source": [
    "# convert reg_dict into reg_start_dict\n",
    "reg_dicts = [library_tools.sequences.gene_dict_2_reg_dict(_gd) for _gd in gene_dicts]\n",
    "tss_dicts = []\n",
    "reg_size = 10000\n",
    "for _rd in reg_dicts:\n",
    "    _tss_d = {_k:_v for _k,_v in _rd.items()}\n",
    "    if _tss_d['Strand'] == '+':\n",
    "        _tss_d[\"Start\"] = int(_rd['Start']) - int(reg_size/2)\n",
    "        _tss_d[\"End\"] = int(_rd['Start']) + int(reg_size/2)\n",
    "        _tss_d['Region'] = f\"{_rd['Chr']}:{_tss_d['Start']}-{_tss_d['End']}\"\n",
    "    else:\n",
    "        _tss_d[\"Start\"] = int(_rd['End']) - int(reg_size/2)\n",
    "        _tss_d[\"End\"] = int(_rd['End']) + int(reg_size/2)\n",
    "        _tss_d['Region'] = f\"{_rd['Chr']}:{_tss_d['Start']}-{_tss_d['End']}\"\n",
    "    # append\n",
    "    tss_dicts.append(_tss_d)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "id": "environmental-street",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "({'Chr': '4',\n",
       "  'Start': '42772860',\n",
       "  'End': '42773993',\n",
       "  'Name': 'gene:ENSMUSG00000094686-Ccl21a',\n",
       "  'Gene': 'Ccl21a',\n",
       "  'Region': '4:42772860-42773993',\n",
       "  'Strand': '-'},\n",
       " {'Chr': '4',\n",
       "  'Start': 42768993,\n",
       "  'End': 42778993,\n",
       "  'Name': 'gene:ENSMUSG00000094686-Ccl21a',\n",
       "  'Gene': 'Ccl21a',\n",
       "  'Region': '4:42768993-42778993',\n",
       "  'Strand': '-'})"
      ]
     },
     "execution_count": 112,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "reg_dicts[0], tss_dicts[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "id": "assured-technology",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- load sequence: 1, size=195471971\n",
      "-- load sequence: 10, size=130694993\n",
      "-- load sequence: 11, size=122082543\n",
      "-- load sequence: 12, size=120129022\n",
      "-- load sequence: 13, size=120421639\n",
      "-- load sequence: 14, size=124902244\n",
      "-- load sequence: 15, size=104043685\n",
      "-- load sequence: 16, size=98207768\n",
      "-- load sequence: 17, size=94987271\n",
      "-- load sequence: 18, size=90702639\n",
      "-- load sequence: 19, size=61431566\n",
      "-- load sequence: 2, size=182113224\n",
      "-- load sequence: 3, size=160039680\n",
      "-- load sequence: 4, size=156508116\n",
      "-- load sequence: 5, size=151834684\n",
      "-- load sequence: 6, size=149736546\n",
      "-- load sequence: 7, size=145441459\n",
      "-- load sequence: 8, size=129401213\n",
      "-- load sequence: 9, size=124595110\n",
      "-- load sequence: MT, size=16299\n",
      "-- load sequence: X, size=171031299\n",
      "-- load sequence: Y, size=91744698\n",
      "-- load sequence: JH584299.1, size=953012\n",
      "-- load sequence: GL456233.1, size=336933\n",
      "-- load sequence: JH584301.1, size=259875\n",
      "-- load sequence: GL456211.1, size=241735\n",
      "-- load sequence: GL456350.1, size=227966\n",
      "-- load sequence: JH584293.1, size=207968\n",
      "-- load sequence: GL456221.1, size=206961\n",
      "-- load sequence: JH584297.1, size=205776\n",
      "-- load sequence: JH584296.1, size=199368\n",
      "-- load sequence: GL456354.1, size=195993\n",
      "-- load sequence: JH584294.1, size=191905\n",
      "-- load sequence: JH584298.1, size=184189\n",
      "-- load sequence: JH584300.1, size=182347\n",
      "-- load sequence: GL456219.1, size=175968\n",
      "-- load sequence: GL456210.1, size=169725\n",
      "-- load sequence: JH584303.1, size=158099\n",
      "-- load sequence: JH584302.1, size=155838\n",
      "-- load sequence: GL456212.1, size=153618\n",
      "-- load sequence: JH584304.1, size=114452\n",
      "-- load sequence: GL456379.1, size=72385\n",
      "-- load sequence: GL456216.1, size=66673\n",
      "-- load sequence: GL456393.1, size=55711\n",
      "-- load sequence: GL456366.1, size=47073\n",
      "-- load sequence: GL456367.1, size=42057\n",
      "-- load sequence: GL456239.1, size=40056\n",
      "-- load sequence: GL456213.1, size=39340\n",
      "-- load sequence: GL456383.1, size=38659\n",
      "-- load sequence: GL456385.1, size=35240\n",
      "-- load sequence: GL456360.1, size=31704\n",
      "-- load sequence: GL456378.1, size=31602\n",
      "-- load sequence: GL456389.1, size=28772\n",
      "-- load sequence: GL456372.1, size=28664\n",
      "-- load sequence: GL456370.1, size=26764\n",
      "-- load sequence: GL456381.1, size=25871\n",
      "-- load sequence: GL456387.1, size=24685\n",
      "-- load sequence: GL456390.1, size=24668\n",
      "-- load sequence: GL456394.1, size=24323\n",
      "-- load sequence: GL456392.1, size=23629\n",
      "-- load sequence: GL456382.1, size=23158\n",
      "-- load sequence: GL456359.1, size=22974\n",
      "-- load sequence: GL456396.1, size=21240\n",
      "-- load sequence: GL456368.1, size=20208\n",
      "-- load sequence: JH584292.1, size=14945\n",
      "-- load sequence: JH584295.1, size=1976\n",
      "- 66 sequences loaded.\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.references)\n",
    "reload(library_tools.sequences)\n",
    "seq_rd = library_tools.sequences.RNA_sequence_reader(genome_folder, flanking=flanking)\n",
    "seq_rd.load_sequences()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "id": "fluid-ukraine",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 10.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 8.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 10.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 16.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 10.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 5.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 13.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 8.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 5.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 12.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 16.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 10.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 12.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 16.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 18.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 19.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 18.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 12.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 10.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 16.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 19.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 19.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 12.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 12.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 13.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 19.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 5.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 8.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 12.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 16.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 10.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 16.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 5.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 10.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 16.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 8.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 16.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 12.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 5.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 8.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 5.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 12.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 13.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: X.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 8.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 1.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 4.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 8.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 7.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 2.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 3.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 6.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 14.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 5.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 15.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 9.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 17.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 11.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 13.\n",
      "-- searching among 1 references\n",
      "-- a match found in record: 8.\n",
      "209\n"
     ]
    }
   ],
   "source": [
    "for _tss_d in tss_dicts:\n",
    "    seq_rd.find_sequence_for_region(_tss_d)\n",
    "print(len(seq_rd.seq_dict))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 116,
   "id": "revolutionary-niagara",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- saving sequences into folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl21a_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt14_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt5_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Col6a1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lifr_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Itga6_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Col4a1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ascl1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cald1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt17_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\C3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Trp63_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Dclk1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Trpm5_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ly6g6f_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Avil_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Alox5ap_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hck_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Plk2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pik3r5_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Agt_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Nrgn_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Plac8_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fxyd6_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pou2f3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl20_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl9_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl6_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rnase1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gp2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fabp1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sox8_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Spib_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fxyd3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tnfrsf11b_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ifi27l2b_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Guca2a_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pigr_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Muc13_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Saa1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Aldob_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lgals2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gsta2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lcn2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Apoa1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hnf4a_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hnf4g_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Igf1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Grhl1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lypd3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krtdap_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tacstd2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt16_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pdzk1ip1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tmem45a_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt10_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gm94_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Acpp_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cnfn_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fam25c_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Elovl4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Efna3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cebpb_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ly6d_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gsta4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Serpinb11_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gabrp_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cyp2f2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gsto1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Anxa8_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mlph_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cd14_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Foxa1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ttc36_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Foxj1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fam183b_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccdc30_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Meig1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Spag8_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Vpreb3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Riiad1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rsph1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Odf3b_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Car8_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rnase4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tspan1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Stxbp5l_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Igfbp5_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cpn1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rap1gap2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Celf3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Syt7_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gpx3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Foxa3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Foxa2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ckb_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Meg3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Myog_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ckm_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Jpt1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mymx_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Dbn1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\S100a9_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pyy_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fabp6_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Slpi_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gal_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Crabp1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Spp1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mt4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pomc_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tff3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sst_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ptgds_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cck_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Psca_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tac2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gstm3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pcp4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pla2g1b_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Nts_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Apod_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ctrb1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lypd2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pgc_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rarres1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Retnla_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ins2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cga_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Iapp_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Nppa_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Camp_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cyp1a2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Dlk1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gm13539_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt13_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cilp_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Igfbp6_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Apoc2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sparc_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tac1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Npy_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Saa2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mpo_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gatm_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Xcl1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\H19_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ly6c2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cxcl13_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Penk_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ecm1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sprr1b_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ly6c1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hbb-y_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Klk7_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Dppa3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hba-a2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mt3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt71_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cryaa_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tff2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Timp2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Klk14_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fgg_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Oas1d_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rsad2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Apoa2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Nefh_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Idi2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Olfm1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Bgn_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ngp_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Car4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lgals3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hp_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\C1qb_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mal_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Alpl_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt77_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cnmd_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ang4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt19_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mgp_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Igf2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Alpi_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ghrl_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hamp2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gm2a_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt9_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Reg4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Isg15_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Calb2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Zg16_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Aldh3a1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sftpd_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lcn3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sprr2i_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Reg3b_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sftpa1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Oas1h_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt76_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gsdma_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ltf_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Clps_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Col1a1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gpx6_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Plvap_reg_1.fasta\n"
     ]
    }
   ],
   "source": [
    "seq_rd.save_sequences(sequence_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "infinite-musical",
   "metadata": {},
   "source": [
    "<a id='2'></a>\n",
    "# 2. Design probe targeting sequences by probe_designer\n",
    "\n",
    "run probe_designer, remeber to clear the memory usage because each table should take ~32GB"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "acting-religion",
   "metadata": {},
   "source": [
    "<a id='2.1'></a>\n",
    "## 2.1 Construct count table with all the 17-mers in the genome\n",
    "\n",
    "Only do this if you don't have pre-built 17-mer\n",
    "\n",
    "However you can do almost the same thing for your own library during quality check\n",
    "\n",
    "This library requires mm10 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "greatest-architecture",
   "metadata": {},
   "outputs": [],
   "source": [
    "overwrite_table = False"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "hollow-turner",
   "metadata": {},
   "source": [
    "### construct map for whole genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "close-mount",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "reload(library_tools.design)\n",
    " \n",
    "genome_table_file = os.path.join(reference_folder, 'GRCm38_genome_17w.npy')\n",
    "\n",
    "if not os.path.exists(genome_table_file) or overwrite_table:\n",
    "    # genome\n",
    "    _genome_filenames = [os.path.join(genome_folder, _fl) \n",
    "         for _fl in os.listdir(genome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_genome_filenames))\n",
    "\n",
    "    ct = library_tools.design.countTable(word=17,save_file=genome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_genome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "\n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "immune-strike",
   "metadata": {},
   "source": [
    "### construct map for transcriptome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "opponent-explorer",
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm import tqdm\n",
    "# transcriptome\n",
    "transcriptome_folder = os.path.join(reference_folder, 'Transcriptome')\n",
    "\n",
    "transcriptome_table_file = os.path.join(reference_folder, 'GRCm38_transcriptome_17w.npy')\n",
    "\n",
    "if not os.path.exists(transcriptome_table_file) or overwrite_table:\n",
    "    # transcriptome\n",
    "    _transcriptome_filenames = [os.path.join(transcriptome_folder, _fl) \n",
    "         for _fl in os.listdir(transcriptome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_transcriptome_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=transcriptome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_transcriptome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "psychological-router",
   "metadata": {},
   "source": [
    "### construct map for repeats from RepBase"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "statistical-killer",
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm import tqdm\n",
    "# repeat\n",
    "repeat_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\Repbase'\n",
    "\n",
    "repeat_table_file = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "\n",
    "if not os.path.exists(repeat_table_file) or overwrite_table:\n",
    "    # repeat\n",
    "    _repeat_filenames = [os.path.join(repeat_folder, _fl) \n",
    "         for _fl in os.listdir(repeat_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_repeat_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=repeat_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_repeat_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "accomplished-transparency",
   "metadata": {},
   "source": [
    "<a id='2.2'></a>\n",
    "## 2.2 Design probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "binary-berlin",
   "metadata": {},
   "outputs": [],
   "source": [
    "# required parameters\n",
    "resolution = 0\n",
    "\n",
    "## required folders\n",
    "## Some folders\n",
    "# human genome\n",
    "reference_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl'\n",
    "genome_folder = os.path.join(reference_folder, 'Genome')\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire'\n",
    "\n",
    "library_folder = os.path.join(pool_folder, f'Genes_TSS_DNA')\n",
    "sequence_folder = os.path.join(library_folder, 'sequences')\n",
    "report_folder = os.path.join(library_folder, 'reports')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "beautiful-bookmark",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "209 regions loaded to design probes.\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\reports\\merged_probes.pbr\n"
     ]
    }
   ],
   "source": [
    "# requires pre_defined genome_folder and library_folder\n",
    "# Indices\n",
    "genome_index = os.path.join(reference_folder, 'GRCm38_genome_17w.npy')\n",
    "transcriptome_index = os.path.join(reference_folder, 'GRCm38_transcriptome_17w.npy') \n",
    "repeat_index = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "#ref_merfish_index = os.path.join(reference_folder, 'M1_meng_MERFISH_17w.npy') # merfish designed by Meng\n",
    "# get input files \n",
    "input_files = glob.glob(os.path.join(sequence_folder, '*.fasta'))\n",
    "\n",
    "print(f\"{len(input_files)} regions loaded to design probes.\")\n",
    "\n",
    "if not os.path.exists(report_folder):\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "# filename to save probe reports\n",
    "save_file = os.path.join(report_folder, f'merged_probes.pbr')\n",
    "print(save_file)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "forbidden-provider",
   "metadata": {},
   "source": [
    "### create pb_designer class"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "id": "automated-stomach",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "73.0"
      ]
     },
     "execution_count": 93,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "massive-yield",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Probe designer derived from Bogdan Bintu:\n",
      "https://github.com/BogdanBintu/ChromatinImaging/blob/master/LibraryDesign/LibraryDesigner.py\n",
      "by Pu Zheng, 2020.11\n",
      "\n",
      "Major changes:\n",
      "    1. allow design of two strands\n",
      "    2. separate reverse_complement (rev_com) and from two strands (two_stranded) as \n",
      "    two different inputs for map_dic and sequence_dic\n",
      "    3. replace 'local_genome' with 'self_sequences' to be more explicit, and only \n",
      "    exclude the counts for the corresponding self_sequence within each input. \n",
      "\n",
      "Key information:\n",
      "    - number of input_sequence(s): 209\n",
      "    - save_file location: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\reports\\merged_probes.pbr\n",
      "\n",
      "- Loading from savefile: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\reports\\merged_probes.pbr.\n",
      "0\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "pb_designer = library_tools.design.pb_reports_class(\n",
    "    sequence_dic={'file':input_files,\n",
    "                  'rev_com':True, # design two strands\n",
    "                  'two_stranded':True},\n",
    "    map_dic={'genome':{'file':genome_index,'rev_com':False,'two_stranded':True},\n",
    "             'transcriptome':{'file':transcriptome_index,'rev_com':True,'two_stranded':False},\n",
    "             'rep_genome':{'file':repeat_index,'rev_com':False,'two_stranded':True},\n",
    "             'self_sequences':{'file':input_files,'force_list':True,'rev_com':False,'two_stranded':True},\n",
    "             #'ref_merfish':{'file':ref_merfish_index,'rev_com':False,'two_stranded':True},\n",
    "             },\n",
    "        save_file=save_file,\n",
    "    params_dic={'word_size':17,'pb_len':40,'buffer_len':1,'max_count':2**16-1,\n",
    "                'check_on_go': False, # whether automatically check probes\n",
    "                'auto': False, # whether automatically convert reference maps\n",
    "               },\n",
    "    check_dic={('genome','self_sequences'): 25,\n",
    "               'rep_genome': 0,\n",
    "               'transcriptome': 14,\n",
    "               #'ref_merfish': 14,\n",
    "               'gc':[0.25,0.85],'tm': 37+0.62*50+5,\n",
    "               }\n",
    "    )\n",
    "print(pb_designer)\n",
    "pb_designer.load_from_file(load_probes_only=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "mediterranean-adjustment",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(pb_designer.kept_probes)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "incoming-texture",
   "metadata": {},
   "source": [
    "### calculate probe reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "weird-starter",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- setting attribute: map_genome\n",
      "--- finish map_genome in 96.933s.\n",
      "-- setting attribute: map_transcriptome\n",
      "--- finish map_transcriptome in 96.102s.\n",
      "-- setting attribute: map_rep_genome\n",
      "--- finish map_rep_genome in 88.888s.\n",
      "Time(s): 281.9225420951843\n",
      "- Designing targeting sequence for 209 regions\n",
      "-- designing region: 9:104322748-104352748_reg_Acpp:- 0, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Acpp_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.202s.\n",
      "in 15.760s.\n",
      "-- designing region: 8:124554706-124584706_reg_Agt:- 1, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Agt_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.172s.\n",
      "in 16.250s.\n",
      "-- designing region: 11:61192537-61222537_reg_Aldh3a1:+ 2, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Aldh3a1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.159s.\n",
      "in 16.314s.\n",
      "-- designing region: 4:49534546-49564546_reg_Aldob:- 3, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Aldob_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.165s.\n",
      "in 16.251s.\n",
      "-- designing region: 5:149249767-149279767_reg_Alox5ap:+ 4, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Alox5ap_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.144s.\n",
      "in 16.220s.\n",
      "-- designing region: 1:87086606-87116606_reg_Alpi:- 5, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Alpi_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.181s.\n",
      "in 16.278s.\n",
      "-- designing region: 4:137781384-137811384_reg_Alpl:- 6, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Alpl_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.263s.\n",
      "-- designing region: 14:51758590-51788590_reg_Ang4:- 7, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ang4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.144s.\n",
      "in 16.194s.\n",
      "-- designing region: 14:34070981-34100981_reg_Anxa8:+ 8, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Anxa8_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.225s.\n",
      "-- designing region: 9:46213580-46243580_reg_Apoa1:+ 9, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Apoa1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.254s.\n",
      "-- designing region: 1:171210054-171240054_reg_Apoa2:+ 10, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Apoa2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.223s.\n",
      "-- designing region: 7:19662941-19692941_reg_Apoc2:- 11, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Apoc2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.269s.\n",
      "-- designing region: 16:31299808-31329808_reg_Apod:- 12, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Apod_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.203s.\n",
      "-- designing region: 10:87478660-87508660_reg_Ascl1:- 13, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ascl1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.200s.\n",
      "-- designing region: 10:126985709-127015709_reg_Avil:+ 14, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Avil_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 23.334s.\n",
      "-- designing region: X:73468602-73498602_reg_Bgn:+ 15, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Bgn_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.241s.\n",
      "-- designing region: 4:136871187-136901187_reg_C1qb:- 16, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\C1qb_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.324s.\n",
      "-- designing region: 17:57213136-57243136_reg_C3:- 17, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\C3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "in 16.320s.\n",
      "-- designing region: 8:110153210-110183210_reg_Calb2:- 18, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Calb2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.255s.\n",
      "-- designing region: 6:34583500-34613500_reg_Cald1:+ 19, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cald1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.126s.\n",
      "in 16.202s.\n",
      "-- designing region: 9:109834617-109864617_reg_Camp:- 20, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Camp_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.183s.\n",
      "in 16.272s.\n",
      "-- designing region: 11:84942786-84972786_reg_Car4:+ 21, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Car4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "in 16.246s.\n",
      "-- designing region: 4:8224041-8254041_reg_Car8:- 22, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Car8_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.233s.\n",
      "-- designing region: 4:119400521-119430521_reg_Ccdc30:- 23, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccdc30_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.289s.\n",
      "-- designing region: 9:121480694-121510694_reg_Cck:- 24, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cck_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.288s.\n",
      "-- designing region: 11:82290690-82320690_reg_Ccl1:- 25, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "in 16.225s.\n",
      "-- designing region: 1:83101766-83131766_reg_Ccl20:+ 26, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl20_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.187s.\n",
      "-- designing region: 4:42758993-42788993_reg_Ccl21a:- 27, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl21a_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 16.273s.\n",
      "-- designing region: 11:82020571-82050571_reg_Ccl2:+ 28, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "in 16.447s.\n",
      "-- designing region: 11:83578087-83608087_reg_Ccl6:- 29, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl6_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.135s.\n",
      "in 16.438s.\n",
      "-- designing region: 11:83563636-83593636_reg_Ccl9:- 30, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ccl9_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.135s.\n",
      "in 16.642s.\n",
      "-- designing region: 18:36711798-36741798_reg_Cd14:- 31, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cd14_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.135s.\n",
      "in 16.304s.\n",
      "-- designing region: 2:167673915-167703915_reg_Cebpb:+ 32, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cebpb_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.212s.\n",
      "-- designing region: 3:94463295-94493295_reg_Celf3:+ 33, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Celf3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.226s.\n",
      "-- designing region: 4:34878779-34908779_reg_Cga:+ 34, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cga_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "in 16.236s.\n",
      "-- designing region: 9:65250180-65280180_reg_Cilp:+ 35, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cilp_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.234s.\n",
      "-- designing region: 12:111657338-111687338_reg_Ckb:- 36, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ckb_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "in 16.263s.\n",
      "-- designing region: 7:19389776-19419776_reg_Ckm:+ 37, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ckm_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "in 16.565s.\n",
      "-- designing region: 17:28545766-28575766_reg_Clps:- 38, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Clps_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "in 17.489s.\n",
      "-- designing region: 7:25354724-25384724_reg_Cnfn:- 39, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cnfn_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.142s.\n",
      "in 16.608s.\n",
      "-- designing region: 14:79647170-79677170_reg_Cnmd:- 40, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cnmd_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.347s.\n",
      "-- designing region: 11:94921224-94951224_reg_Col1a1:+ 41, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Col1a1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "in 16.345s.\n",
      "-- designing region: 8:11297826-11327826_reg_Col4a1:- 42, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Col4a1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "in 16.290s.\n",
      "-- designing region: 10:76711168-76741168_reg_Col6a1:- 43, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Col6a1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.375s.\n",
      "-- designing region: 19:43971556-44001556_reg_Cpn1:- 44, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cpn1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.120s.\n",
      "in 16.284s.\n",
      "-- designing region: 9:54749748-54779748_reg_Crabp1:+ 45, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Crabp1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.332s.\n",
      "-- designing region: 17:31662807-31692807_reg_Cryaa:+ 46, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cryaa_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.416s.\n",
      "-- designing region: 8:111676010-111706010_reg_Ctrb1:- 47, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ctrb1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.515s.\n",
      "-- designing region: 5:95941951-95971951_reg_Cxcl13:+ 48, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cxcl13_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.281s.\n",
      "-- designing region: 9:57668703-57698703_reg_Cyp1a2:- 49, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cyp1a2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.157s.\n",
      "in 16.389s.\n",
      "-- designing region: 7:27104909-27134909_reg_Cyp2f2:+ 50, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Cyp2f2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.139s.\n",
      "in 16.241s.\n",
      "-- designing region: 13:55473111-55503111_reg_Dbn1:- 51, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Dbn1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.312s.\n",
      "-- designing region: 3:55227364-55257364_reg_Dclk1:+ 52, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Dclk1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.291s.\n",
      "-- designing region: 12:109437823-109467823_reg_Dlk1:+ 53, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Dlk1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.333s.\n",
      "-- designing region: 6:122611410-122641410_reg_Dppa3:+ 54, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Dppa3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.259s.\n",
      "-- designing region: 3:95724569-95754569_reg_Ecm1:- 55, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ecm1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.310s.\n",
      "-- designing region: 3:89307965-89337965_reg_Efna3:- 56, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Efna3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 16.278s.\n",
      "-- designing region: 9:83791277-83821277_reg_Elovl4:- 57, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Elovl4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.156s.\n",
      "in 16.333s.\n",
      "-- designing region: 6:71184827-71214827_reg_Fabp1:+ 58, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fabp1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.196s.\n",
      "in 16.337s.\n",
      "-- designing region: 11:43586540-43616540_reg_Fabp6:- 59, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fabp6_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.283s.\n",
      "-- designing region: 11:58786960-58816960_reg_Fam183b:- 60, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fam183b_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.143s.\n",
      "in 16.295s.\n",
      "-- designing region: 14:34340433-34370433_reg_Fam25c:- 61, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fam25c_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.369s.\n",
      "-- designing region: 3:82992724-83022724_reg_Fgg:+ 62, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fgg_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.144s.\n",
      "in 16.327s.\n",
      "-- designing region: 12:57531916-57561916_reg_Foxa1:- 63, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Foxa1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "in 16.308s.\n",
      "-- designing region: 2:148031969-148061969_reg_Foxa2:- 64, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Foxa2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "in 16.458s.\n",
      "-- designing region: 7:19008538-19038538_reg_Foxa3:- 65, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Foxa3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.158s.\n",
      "in 16.436s.\n",
      "-- designing region: 11:116320399-116350399_reg_Foxj1:- 66, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Foxj1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.120s.\n",
      "in 16.341s.\n",
      "-- designing region: 7:31061704-31091704_reg_Fxyd3:- 67, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fxyd3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.364s.\n",
      "-- designing region: 9:45355185-45385185_reg_Fxyd6:+ 68, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Fxyd6_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.288s.\n",
      "-- designing region: 11:33563959-33593959_reg_Gabrp:- 69, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gabrp_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.153s.\n",
      "in 25.337s.\n",
      "-- designing region: 19:3399544-3429544_reg_Gal:- 70, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gal_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.334s.\n",
      "-- designing region: 2:122596303-122626303_reg_Gatm:- 71, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gatm_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.335s.\n",
      "-- designing region: 6:113704880-113734880_reg_Ghrl:- 72, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ghrl_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.316s.\n",
      "-- designing region: 2:25767257-25797257_reg_Gm13539:+ 73, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gm13539_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.377s.\n",
      "-- designing region: 11:55083115-55113115_reg_Gm2a:+ 74, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gm2a_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.346s.\n",
      "-- designing region: 18:43777878-43807878_reg_Gm94:- 75, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gm94_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.554s.\n",
      "-- designing region: 7:119444285-119474285_reg_Gp2:- 76, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gp2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "in 16.533s.\n",
      "-- designing region: 11:54887453-54917453_reg_Gpx3:+ 77, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gpx3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "in 16.357s.\n",
      "-- designing region: 13:21297125-21327125_reg_Gpx6:+ 78, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gpx6_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.126s.\n",
      "in 16.403s.\n",
      "-- designing region: 12:24557283-24587283_reg_Grhl1:+ 79, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Grhl1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.141s.\n",
      "in 16.335s.\n",
      "-- designing region: 11:98649351-98679351_reg_Gsdma:+ 80, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gsdma_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.309s.\n",
      "-- designing region: 9:78340788-78370788_reg_Gsta2:- 81, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gsta2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.358s.\n",
      "-- designing region: 9:78167774-78197774_reg_Gsta4:+ 82, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gsta4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.334s.\n",
      "-- designing region: 3:107954283-107984283_reg_Gstm3:- 83, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gstm3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.120s.\n",
      "in 16.334s.\n",
      "-- designing region: 19:47839970-47869970_reg_Gsto1:+ 84, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Gsto1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.292s.\n",
      "-- designing region: 4:119622704-119652704_reg_Guca2a:+ 85, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Guca2a_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 16.392s.\n",
      "-- designing region: 7:142563143-142593143_reg_H19:- 86, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\H19_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.350s.\n",
      "-- designing region: 7:30909681-30939681_reg_Hamp2:- 87, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hamp2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "in 16.345s.\n",
      "-- designing region: 11:32281489-32311489_reg_Hba-a2:+ 88, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hba-a2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.276s.\n",
      "-- designing region: 7:103838216-103868216_reg_Hbb-y:- 89, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hbb-y_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.384s.\n",
      "-- designing region: 2:153093468-153123468_reg_Hck:+ 90, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hck_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.317s.\n",
      "-- designing region: 2:163491808-163521808_reg_Hnf4a:+ 91, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hnf4a_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.125s.\n",
      "in 16.328s.\n",
      "-- designing region: 3:3493030-3523030_reg_Hnf4g:+ 92, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hnf4g_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.300s.\n",
      "-- designing region: 8:109564172-109594172_reg_Hp:- 93, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Hp_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.333s.\n",
      "-- designing region: 6:142283423-142313423_reg_Iapp:+ 94, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Iapp_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.298s.\n",
      "-- designing region: 13:8937863-8967863_reg_Idi2:+ 95, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Idi2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "in 17.434s.\n",
      "-- designing region: 12:103442223-103472223_reg_Ifi27l2b:- 96, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ifi27l2b_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.364s.\n",
      "-- designing region: 10:87843265-87873265_reg_Igf1:+ 97, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Igf1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.566s.\n",
      "-- designing region: 7:142651816-142681816_reg_Igf2:- 98, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Igf2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.153s.\n",
      "in 16.600s.\n",
      "-- designing region: 1:72859884-72889884_reg_Igfbp5:- 99, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Igfbp5_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.575s.\n",
      "-- designing region: 15:102129362-102159362_reg_Igfbp6:+ 100, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Igfbp6_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.165s.\n",
      "in 16.554s.\n",
      "-- designing region: 7:142728381-142758381_reg_Ins2:- 101, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ins2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.135s.\n",
      "in 16.387s.\n",
      "-- designing region: 4:156185796-156215796_reg_Isg15:- 102, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Isg15_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.149s.\n",
      "in 16.358s.\n",
      "-- designing region: 2:71730616-71760616_reg_Itga6:+ 103, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Itga6_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.374s.\n",
      "-- designing region: 11:115499387-115529387_reg_Jpt1:- 104, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Jpt1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.126s.\n",
      "in 16.539s.\n",
      "-- designing region: 7:43675418-43705418_reg_Klk14:+ 105, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Klk14_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.575s.\n",
      "-- designing region: 7:43796294-43826294_reg_Klk7:+ 106, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Klk7_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "in 16.548s.\n",
      "-- designing region: 11:99374364-99404364_reg_Krt10:- 107, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt10_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.139s.\n",
      "in 16.556s.\n",
      "-- designing region: 11:100106566-100136566_reg_Krt13:- 108, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt13_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.337s.\n",
      "-- designing region: 11:100192548-100222548_reg_Krt14:- 109, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt14_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.156s.\n",
      "in 16.334s.\n",
      "-- designing region: 11:100233902-100263902_reg_Krt16:- 110, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt16_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.308s.\n",
      "-- designing region: 11:100246029-100276029_reg_Krt17:- 111, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt17_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.389s.\n",
      "-- designing region: 11:100133665-100163665_reg_Krt19:- 112, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt19_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.360s.\n",
      "-- designing region: 15:101697898-101727898_reg_Krt5:- 113, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt5_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.351s.\n",
      "-- designing region: 15:101728109-101758109_reg_Krt71:- 114, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt71_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 16.282s.\n",
      "-- designing region: 15:101877920-101907920_reg_Krt76:- 115, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt76_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "in 16.375s.\n",
      "-- designing region: 15:101854705-101884705_reg_Krt77:- 116, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt77_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.384s.\n",
      "-- designing region: 11:100178246-100208246_reg_Krt9:- 117, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krt9_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.445s.\n",
      "-- designing region: 7:30772896-30802896_reg_Krtdap:+ 118, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Krtdap_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.138s.\n",
      "in 16.317s.\n",
      "-- designing region: 2:32373252-32403252_reg_Lcn2:- 119, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lcn2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.146s.\n",
      "in 16.393s.\n",
      "-- designing region: 2:25750569-25780569_reg_Lcn3:+ 120, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lcn3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.149s.\n",
      "in 16.372s.\n",
      "-- designing region: 15:78840529-78870529_reg_Lgals2:- 121, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lgals2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.373s.\n",
      "-- designing region: 14:47352751-47382751_reg_Lgals3:+ 122, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lgals3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "in 16.321s.\n",
      "-- designing region: 15:7075614-7105614_reg_Lifr:+ 123, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lifr_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.123s.\n",
      "in 16.333s.\n",
      "-- designing region: 9:111004271-111034271_reg_Ltf:+ 124, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ltf_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.307s.\n",
      "-- designing region: 15:75033830-75063830_reg_Ly6c1:- 125, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ly6c1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.341s.\n",
      "-- designing region: 15:75096970-75126970_reg_Ly6c2:- 126, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ly6c2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.130s.\n",
      "in 16.484s.\n",
      "-- designing region: 15:74748620-74778620_reg_Ly6d:- 127, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ly6d_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.323s.\n",
      "-- designing region: 17:35070610-35100610_reg_Ly6g6f:- 128, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ly6g6f_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.329s.\n",
      "-- designing region: 15:74719329-74749329_reg_Lypd2:- 129, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lypd2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.414s.\n",
      "-- designing region: 7:24621550-24651550_reg_Lypd3:+ 130, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Lypd3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.153s.\n",
      "in 16.547s.\n",
      "-- designing region: 2:127641695-127671695_reg_Mal:- 131, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mal_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "in 16.517s.\n",
      "-- designing region: 12:109526001-109556001_reg_Meg3:+ 132, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Meg3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.341s.\n",
      "-- designing region: 2:3407648-3437648_reg_Meig1:- 133, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Meig1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.128s.\n",
      "in 16.300s.\n",
      "-- designing region: 6:136860823-136890823_reg_Mgp:- 134, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mgp_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.310s.\n",
      "-- designing region: 1:90900085-90930085_reg_Mlph:+ 135, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mlph_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.317s.\n",
      "-- designing region: 11:87778581-87808581_reg_Mpo:+ 136, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mpo_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.139s.\n",
      "in 16.308s.\n",
      "-- designing region: 8:94137607-94167607_reg_Mt3:+ 137, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mt3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.143s.\n",
      "in 16.558s.\n",
      "-- designing region: 8:94122197-94152197_reg_Mt4:+ 138, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mt4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.123s.\n",
      "in 27.472s.\n",
      "-- designing region: 16:33779037-33809037_reg_Muc13:+ 139, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Muc13_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.369s.\n",
      "-- designing region: 17:45587102-45617102_reg_Mymx:- 140, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Mymx_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.310s.\n",
      "-- designing region: 1:134274989-134304989_reg_Myog:+ 141, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Myog_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.138s.\n",
      "in 16.375s.\n",
      "-- designing region: 11:4933064-4963064_reg_Nefh:- 142, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Nefh_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.338s.\n",
      "-- designing region: 9:110404747-110434747_reg_Ngp:+ 143, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ngp_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 16.352s.\n",
      "-- designing region: 4:147985722-148015722_reg_Nppa:+ 144, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Nppa_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.290s.\n",
      "-- designing region: 6:49807710-49837710_reg_Npy:+ 145, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Npy_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.298s.\n",
      "-- designing region: 9:37537904-37567904_reg_Nrgn:- 146, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Nrgn_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "in 16.333s.\n",
      "-- designing region: 10:102475486-102505486_reg_Nts:- 147, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Nts_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.139s.\n",
      "in 16.455s.\n",
      "-- designing region: 5:120899536-120929536_reg_Oas1d:+ 148, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Oas1d_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "in 16.524s.\n",
      "-- designing region: 5:120846421-120876421_reg_Oas1h:+ 149, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Oas1h_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.127s.\n",
      "in 16.559s.\n",
      "-- designing region: 15:89364587-89394587_reg_Odf3b:- 150, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Odf3b_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "in 16.539s.\n",
      "-- designing region: 2:28177992-28207992_reg_Olfm1:+ 151, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Olfm1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.138s.\n",
      "in 17.169s.\n",
      "-- designing region: 16:96452606-96482606_reg_Pcp4:+ 152, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pcp4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "in 16.592s.\n",
      "-- designing region: 4:115073708-115103708_reg_Pdzk1ip1:+ 153, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pdzk1ip1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "in 16.600s.\n",
      "-- designing region: 4:4123819-4153819_reg_Penk:- 154, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Penk_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.138s.\n",
      "in 16.583s.\n",
      "-- designing region: 17:47711842-47741842_reg_Pgc:+ 155, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pgc_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.135s.\n",
      "in 16.477s.\n",
      "-- designing region: 1:130811684-130841684_reg_Pigr:+ 156, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pigr_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.341s.\n",
      "-- designing region: 11:68417121-68447121_reg_Pik3r5:+ 157, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pik3r5_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "in 16.354s.\n",
      "-- designing region: 5:115451262-115481262_reg_Pla2g1b:+ 158, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pla2g1b_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.121s.\n",
      "in 16.526s.\n",
      "-- designing region: 5:100557245-100587245_reg_Plac8:- 159, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Plac8_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "in 16.597s.\n",
      "-- designing region: 13:110380046-110410046_reg_Plk2:+ 160, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Plk2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "in 16.607s.\n",
      "-- designing region: 8:71496752-71526752_reg_Plvap:- 161, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Plvap_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.149s.\n",
      "in 16.598s.\n",
      "-- designing region: 12:3939951-3969951_reg_Pomc:+ 162, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pomc_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.137s.\n",
      "in 16.554s.\n",
      "-- designing region: 9:43195369-43225369_reg_Pou2f3:- 163, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pou2f3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "in 16.584s.\n",
      "-- designing region: 15:74699839-74729839_reg_Psca:+ 164, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Psca_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.139s.\n",
      "in 16.638s.\n",
      "-- designing region: 2:25455046-25485046_reg_Ptgds:- 165, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ptgds_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.130s.\n",
      "in 16.621s.\n",
      "-- designing region: 11:102092832-102122832_reg_Pyy:- 166, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Pyy_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.146s.\n",
      "in 16.556s.\n",
      "-- designing region: 11:74595915-74625915_reg_Rap1gap2:- 167, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rap1gap2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.626s.\n",
      "-- designing region: 3:67500523-67530523_reg_Rarres1:- 168, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rarres1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.130s.\n",
      "in 16.587s.\n",
      "-- designing region: 6:78355657-78385657_reg_Reg3b:+ 169, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Reg3b_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.587s.\n",
      "-- designing region: 3:98207156-98237156_reg_Reg4:+ 170, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Reg4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "in 16.594s.\n",
      "-- designing region: 16:48827551-48857551_reg_Retnla:+ 171, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Retnla_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.135s.\n",
      "in 16.542s.\n",
      "-- designing region: 3:94465341-94495341_reg_Riiad1:- 172, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Riiad1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.128s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 16.646s.\n",
      "-- designing region: 14:51131785-51161785_reg_Rnase1:- 173, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rnase1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.545s.\n",
      "-- designing region: 14:51076077-51106077_reg_Rnase4:+ 174, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rnase4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.130s.\n",
      "in 16.344s.\n",
      "-- designing region: 12:26441452-26471452_reg_Rsad2:- 175, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rsad2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.138s.\n",
      "in 16.336s.\n",
      "-- designing region: 17:31262383-31292383_reg_Rsph1:- 176, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Rsph1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "in 16.389s.\n",
      "-- designing region: 3:90680721-90710721_reg_S100a9:- 177, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\S100a9_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.152s.\n",
      "in 16.351s.\n",
      "-- designing region: 7:46727980-46757980_reg_Saa1:- 178, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Saa1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.140s.\n",
      "in 16.358s.\n",
      "-- designing region: 7:46736790-46766790_reg_Saa2:+ 179, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Saa2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.524s.\n",
      "-- designing region: 1:107346198-107376198_reg_Serpinb11:+ 180, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Serpinb11_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.150s.\n",
      "in 16.373s.\n",
      "-- designing region: 14:41116782-41146782_reg_Sftpa1:+ 181, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sftpa1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.129s.\n",
      "in 16.312s.\n",
      "-- designing region: 14:41170149-41200149_reg_Sftpd:- 182, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sftpd_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.350s.\n",
      "-- designing region: 2:164374095-164404095_reg_Slpi:- 183, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Slpi_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.396s.\n",
      "-- designing region: 17:25555686-25585686_reg_Sox8:- 184, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sox8_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.130s.\n",
      "in 16.440s.\n",
      "-- designing region: 4:43638594-43668594_reg_Spag8:- 185, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Spag8_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.120s.\n",
      "in 16.337s.\n",
      "-- designing region: 11:55408183-55438183_reg_Sparc:- 186, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sparc_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.133s.\n",
      "in 16.312s.\n",
      "-- designing region: 7:44517071-44547071_reg_Spib:- 187, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Spib_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.522s.\n",
      "-- designing region: 5:104420118-104450118_reg_Spp1:+ 188, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Spp1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.135s.\n",
      "in 16.589s.\n",
      "-- designing region: 3:92423789-92453789_reg_Sprr1b:- 189, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sprr1b_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.551s.\n",
      "-- designing region: 3:92392991-92422991_reg_Sprr2i:+ 190, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sprr2i_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.138s.\n",
      "in 16.572s.\n",
      "-- designing region: 16:23875958-23905958_reg_Sst:- 191, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Sst_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.138s.\n",
      "in 16.553s.\n",
      "-- designing region: 16:37369962-37399962_reg_Stxbp5l:- 192, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Stxbp5l_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.314s.\n",
      "-- designing region: 19:10374090-10404090_reg_Syt7:+ 193, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Syt7_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.338s.\n",
      "-- designing region: 6:7539879-7569879_reg_Tac1:+ 194, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tac1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.141s.\n",
      "in 16.429s.\n",
      "-- designing region: 10:127709478-127739478_reg_Tac2:+ 195, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tac2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.130s.\n",
      "in 23.259s.\n",
      "-- designing region: 6:67520796-67550796_reg_Tacstd2:- 196, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tacstd2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.536s.\n",
      "-- designing region: 17:31129282-31159282_reg_Tff2:- 197, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tff2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.728s.\n",
      "-- designing region: 17:31114646-31144646_reg_Tff3:- 198, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tff3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.131s.\n",
      "in 16.829s.\n",
      "-- designing region: 11:118340740-118370740_reg_Timp2:- 199, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Timp2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.605s.\n",
      "-- designing region: 16:56871166-56901166_reg_Tmem45a:- 200, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tmem45a_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.133s.\n",
      "in 16.555s.\n",
      "-- designing region: 15:54263484-54293484_reg_Tnfrsf11b:- 201, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tnfrsf11b_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 16.502s.\n",
      "-- designing region: 16:25668763-25698763_reg_Trp63:+ 202, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Trp63_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.149s.\n",
      "in 16.543s.\n",
      "-- designing region: 7:143079642-143109642_reg_Trpm5:- 203, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Trpm5_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.145s.\n",
      "in 16.385s.\n",
      "-- designing region: 4:116152601-116182601_reg_Tspan1:- 204, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Tspan1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.372s.\n",
      "-- designing region: 9:44788072-44818072_reg_Ttc36:- 205, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Ttc36_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.118s.\n",
      "in 16.506s.\n",
      "-- designing region: 10:75928057-75958057_reg_Vpreb3:+ 206, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Vpreb3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.136s.\n",
      "in 16.299s.\n",
      "-- designing region: 1:164920527-164950527_reg_Xcl1:- 207, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Xcl1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.132s.\n",
      "in 17.001s.\n",
      "-- designing region: 7:127072328-127102328_reg_Zg16:- 208, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\sequences\\Zg16_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.134s.\n",
      "in 16.400s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\reports\\merged_probes.pbr\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "pb_designer.computeOTmaps() # load the tables \n",
    "pb_designer.compute_pb_report() # design candidate probes"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "rapid-intermediate",
   "metadata": {},
   "source": [
    "### check probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "restricted-scientist",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:0 9:104322748-104352748_reg_Acpp:-, 58448 candidate probes\n",
      "--- 30602 probes passed check_dic selection.\n",
      "finish in 2.378s, 425 probes kept.\n",
      "-- check region:1 8:124554706-124584706_reg_Agt:-, 53018 candidate probes\n",
      "--- 25277 probes passed check_dic selection.\n",
      "finish in 2.133s, 352 probes kept.\n",
      "-- check region:2 11:61192537-61222537_reg_Aldh3a1:+, 59118 candidate probes\n",
      "--- 31463 probes passed check_dic selection.\n",
      "finish in 2.457s, 433 probes kept.\n",
      "-- check region:3 4:49534546-49564546_reg_Aldob:-, 57512 candidate probes\n",
      "--- 29093 probes passed check_dic selection.\n",
      "finish in 2.298s, 414 probes kept.\n",
      "-- check region:4 5:149249767-149279767_reg_Alox5ap:+, 59050 candidate probes\n",
      "--- 28144 probes passed check_dic selection.\n",
      "finish in 2.329s, 398 probes kept.\n",
      "-- check region:5 1:87086606-87116606_reg_Alpi:-, 59514 candidate probes\n",
      "--- 25109 probes passed check_dic selection.\n",
      "finish in 2.213s, 399 probes kept.\n",
      "-- check region:6 4:137781384-137811384_reg_Alpl:-, 58836 candidate probes\n",
      "--- 27924 probes passed check_dic selection.\n",
      "finish in 2.297s, 395 probes kept.\n",
      "-- check region:7 14:51758590-51788590_reg_Ang4:-, 51186 candidate probes\n",
      "--- 7687 probes passed check_dic selection.\n",
      "finish in 1.556s, 175 probes kept.\n",
      "-- check region:8 14:34070981-34100981_reg_Anxa8:+, 59610 candidate probes\n",
      "--- 36881 probes passed check_dic selection.\n",
      "finish in 2.705s, 497 probes kept.\n",
      "-- check region:9 9:46213580-46243580_reg_Apoa1:+, 59446 candidate probes\n",
      "--- 31090 probes passed check_dic selection.\n",
      "finish in 2.413s, 454 probes kept.\n",
      "-- check region:10 1:171210054-171240054_reg_Apoa2:+, 58882 candidate probes\n",
      "--- 22933 probes passed check_dic selection.\n",
      "finish in 2.245s, 373 probes kept.\n",
      "-- check region:11 7:19662941-19692941_reg_Apoc2:-, 56036 candidate probes\n",
      "--- 19622 probes passed check_dic selection.\n",
      "finish in 1.961s, 320 probes kept.\n",
      "-- check region:12 16:31299808-31329808_reg_Apod:-, 59030 candidate probes\n",
      "--- 27391 probes passed check_dic selection.\n",
      "finish in 2.331s, 414 probes kept.\n",
      "-- check region:13 10:87478660-87508660_reg_Ascl1:-, 57926 candidate probes\n",
      "--- 28404 probes passed check_dic selection.\n",
      "finish in 2.361s, 415 probes kept.\n",
      "-- check region:14 10:126985709-127015709_reg_Avil:+, 58794 candidate probes\n",
      "--- 29126 probes passed check_dic selection.\n",
      "finish in 2.357s, 434 probes kept.\n",
      "-- check region:15 X:73468602-73498602_reg_Bgn:+, 59380 candidate probes\n",
      "--- 31594 probes passed check_dic selection.\n",
      "finish in 2.541s, 475 probes kept.\n",
      "-- check region:16 4:136871187-136901187_reg_C1qb:-, 58528 candidate probes\n",
      "--- 25895 probes passed check_dic selection.\n",
      "finish in 2.235s, 409 probes kept.\n",
      "-- check region:17 17:57213136-57243136_reg_C3:-, 59432 candidate probes\n",
      "--- 25685 probes passed check_dic selection.\n",
      "finish in 2.314s, 404 probes kept.\n",
      "-- check region:18 8:110153210-110183210_reg_Calb2:-, 59190 candidate probes\n",
      "--- 35610 probes passed check_dic selection.\n",
      "finish in 2.686s, 456 probes kept.\n",
      "-- check region:19 6:34583500-34613500_reg_Cald1:+, 57594 candidate probes\n",
      "--- 30046 probes passed check_dic selection.\n",
      "finish in 2.467s, 402 probes kept.\n",
      "-- check region:20 9:109834617-109864617_reg_Camp:-, 56924 candidate probes\n",
      "--- 22637 probes passed check_dic selection.\n",
      "finish in 2.031s, 329 probes kept.\n",
      "-- check region:21 11:84942786-84972786_reg_Car4:+, 59260 candidate probes\n",
      "--- 30770 probes passed check_dic selection.\n",
      "finish in 2.389s, 429 probes kept.\n",
      "-- check region:22 4:8224041-8254041_reg_Car8:-, 57068 candidate probes\n",
      "--- 24797 probes passed check_dic selection.\n",
      "finish in 2.102s, 374 probes kept.\n",
      "-- check region:23 4:119400521-119430521_reg_Ccdc30:-, 59082 candidate probes\n",
      "--- 24423 probes passed check_dic selection.\n",
      "finish in 2.140s, 389 probes kept.\n",
      "-- check region:24 9:121480694-121510694_reg_Cck:-, 58886 candidate probes\n",
      "--- 27891 probes passed check_dic selection.\n",
      "finish in 2.298s, 404 probes kept.\n",
      "-- check region:25 11:82290690-82320690_reg_Ccl1:-, 59606 candidate probes\n",
      "--- 31558 probes passed check_dic selection.\n",
      "finish in 2.348s, 443 probes kept.\n",
      "-- check region:26 1:83101766-83131766_reg_Ccl20:+, 58752 candidate probes\n",
      "--- 29857 probes passed check_dic selection.\n",
      "finish in 2.330s, 425 probes kept.\n",
      "-- check region:27 4:42758993-42788993_reg_Ccl21a:-, 58856 candidate probes\n",
      "--- 27 probes passed check_dic selection.\n",
      "finish in 1.151s, 5 probes kept.\n",
      "-- check region:28 11:82020571-82050571_reg_Ccl2:+, 59788 candidate probes\n",
      "--- 32708 probes passed check_dic selection.\n",
      "finish in 2.526s, 472 probes kept.\n",
      "-- check region:29 11:83578087-83608087_reg_Ccl6:-, 28754 candidate probes\n",
      "--- 13948 probes passed check_dic selection.\n",
      "finish in 1.149s, 200 probes kept.\n",
      "-- check region:30 11:83563636-83593636_reg_Ccl9:-, 58104 candidate probes\n",
      "--- 24699 probes passed check_dic selection.\n",
      "finish in 2.221s, 394 probes kept.\n",
      "-- check region:31 18:36711798-36741798_reg_Cd14:-, 57850 candidate probes\n",
      "--- 20919 probes passed check_dic selection.\n",
      "finish in 2.160s, 353 probes kept.\n",
      "-- check region:32 2:167673915-167703915_reg_Cebpb:+, 58686 candidate probes\n",
      "--- 26061 probes passed check_dic selection.\n",
      "finish in 2.174s, 411 probes kept.\n",
      "-- check region:33 3:94463295-94493295_reg_Celf3:+, 3988 candidate probes\n",
      "--- 1627 probes passed check_dic selection.\n",
      "finish in 0.179s, 26 probes kept.\n",
      "-- check region:34 4:34878779-34908779_reg_Cga:+, 57418 candidate probes\n",
      "--- 21767 probes passed check_dic selection.\n",
      "finish in 2.003s, 346 probes kept.\n",
      "-- check region:35 9:65250180-65280180_reg_Cilp:+, 59032 candidate probes\n",
      "--- 24645 probes passed check_dic selection.\n",
      "finish in 2.140s, 396 probes kept.\n",
      "-- check region:36 12:111657338-111687338_reg_Ckb:-, 59058 candidate probes\n",
      "--- 29468 probes passed check_dic selection.\n",
      "finish in 2.348s, 429 probes kept.\n",
      "-- check region:37 7:19389776-19419776_reg_Ckm:+, 58948 candidate probes\n",
      "--- 22799 probes passed check_dic selection.\n",
      "finish in 2.098s, 392 probes kept.\n",
      "-- check region:38 17:28545766-28575766_reg_Clps:-, 57758 candidate probes\n",
      "--- 21936 probes passed check_dic selection.\n",
      "finish in 2.068s, 329 probes kept.\n",
      "-- check region:39 7:25354724-25384724_reg_Cnfn:-, 59336 candidate probes\n",
      "--- 29590 probes passed check_dic selection.\n",
      "finish in 2.423s, 453 probes kept.\n",
      "-- check region:40 14:79647170-79677170_reg_Cnmd:-, 57040 candidate probes\n",
      "--- 32165 probes passed check_dic selection.\n",
      "finish in 2.412s, 445 probes kept.\n",
      "-- check region:41 11:94921224-94951224_reg_Col1a1:+, 59068 candidate probes\n",
      "--- 33072 probes passed check_dic selection.\n",
      "finish in 2.512s, 471 probes kept.\n",
      "-- check region:42 8:11297826-11327826_reg_Col4a1:-, 59510 candidate probes\n",
      "--- 36211 probes passed check_dic selection.\n",
      "finish in 2.585s, 468 probes kept.\n",
      "-- check region:43 10:76711168-76741168_reg_Col6a1:-, 59678 candidate probes\n",
      "--- 34278 probes passed check_dic selection.\n",
      "finish in 2.517s, 464 probes kept.\n",
      "-- check region:44 19:43971556-44001556_reg_Cpn1:-, 59248 candidate probes\n",
      "--- 28397 probes passed check_dic selection.\n",
      "finish in 2.265s, 395 probes kept.\n",
      "-- check region:45 9:54749748-54779748_reg_Crabp1:+, 59532 candidate probes\n",
      "--- 34458 probes passed check_dic selection.\n",
      "finish in 2.482s, 465 probes kept.\n",
      "-- check region:46 17:31662807-31692807_reg_Cryaa:+, 58760 candidate probes\n",
      "--- 25827 probes passed check_dic selection.\n",
      "finish in 2.146s, 389 probes kept.\n",
      "-- check region:47 8:111676010-111706010_reg_Ctrb1:-, 57908 candidate probes\n",
      "--- 24482 probes passed check_dic selection.\n",
      "finish in 2.125s, 362 probes kept.\n",
      "-- check region:48 5:95941951-95971951_reg_Cxcl13:+, 58768 candidate probes\n",
      "--- 28956 probes passed check_dic selection.\n",
      "finish in 2.275s, 393 probes kept.\n",
      "-- check region:49 9:57668703-57698703_reg_Cyp1a2:-, 59106 candidate probes\n",
      "--- 25524 probes passed check_dic selection.\n",
      "finish in 2.400s, 402 probes kept.\n",
      "-- check region:50 7:27104909-27134909_reg_Cyp2f2:+, 58014 candidate probes\n",
      "--- 21372 probes passed check_dic selection.\n",
      "finish in 2.135s, 350 probes kept.\n",
      "-- check region:51 13:55473111-55503111_reg_Dbn1:-, 59344 candidate probes\n",
      "--- 29857 probes passed check_dic selection.\n",
      "finish in 2.419s, 433 probes kept.\n",
      "-- check region:52 3:55227364-55257364_reg_Dclk1:+, 59692 candidate probes\n",
      "--- 29723 probes passed check_dic selection.\n",
      "finish in 2.493s, 444 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:53 12:109437823-109467823_reg_Dlk1:+, 59602 candidate probes\n",
      "--- 32652 probes passed check_dic selection.\n",
      "finish in 2.559s, 466 probes kept.\n",
      "-- check region:54 6:122611410-122641410_reg_Dppa3:+, 56332 candidate probes\n",
      "--- 17421 probes passed check_dic selection.\n",
      "finish in 1.977s, 279 probes kept.\n",
      "-- check region:55 3:95724569-95754569_reg_Ecm1:-, 59310 candidate probes\n",
      "--- 21974 probes passed check_dic selection.\n",
      "finish in 2.098s, 402 probes kept.\n",
      "-- check region:56 3:89307965-89337965_reg_Efna3:-, 59680 candidate probes\n",
      "--- 29802 probes passed check_dic selection.\n",
      "finish in 2.424s, 437 probes kept.\n",
      "-- check region:57 9:83791277-83821277_reg_Elovl4:-, 59490 candidate probes\n",
      "--- 28628 probes passed check_dic selection.\n",
      "finish in 2.344s, 418 probes kept.\n",
      "-- check region:58 6:71184827-71214827_reg_Fabp1:+, 59242 candidate probes\n",
      "--- 25601 probes passed check_dic selection.\n",
      "finish in 2.233s, 378 probes kept.\n",
      "-- check region:59 11:43586540-43616540_reg_Fabp6:-, 58550 candidate probes\n",
      "--- 25589 probes passed check_dic selection.\n",
      "finish in 2.144s, 390 probes kept.\n",
      "-- check region:60 11:58786960-58816960_reg_Fam183b:-, 59244 candidate probes\n",
      "--- 23847 probes passed check_dic selection.\n",
      "finish in 2.081s, 370 probes kept.\n",
      "-- check region:61 14:34340433-34370433_reg_Fam25c:-, 59158 candidate probes\n",
      "--- 28687 probes passed check_dic selection.\n",
      "finish in 2.254s, 415 probes kept.\n",
      "-- check region:62 3:82992724-83022724_reg_Fgg:+, 59390 candidate probes\n",
      "--- 30570 probes passed check_dic selection.\n",
      "finish in 2.402s, 443 probes kept.\n",
      "-- check region:63 12:57531916-57561916_reg_Foxa1:-, 59446 candidate probes\n",
      "--- 32143 probes passed check_dic selection.\n",
      "finish in 2.428s, 448 probes kept.\n",
      "-- check region:64 2:148031969-148061969_reg_Foxa2:-, 59804 candidate probes\n",
      "--- 35315 probes passed check_dic selection.\n",
      "finish in 2.626s, 464 probes kept.\n",
      "-- check region:65 7:19008538-19038538_reg_Foxa3:-, 58090 candidate probes\n",
      "--- 24443 probes passed check_dic selection.\n",
      "finish in 2.197s, 403 probes kept.\n",
      "-- check region:66 11:116320399-116350399_reg_Foxj1:-, 59384 candidate probes\n",
      "--- 32377 probes passed check_dic selection.\n",
      "finish in 2.464s, 468 probes kept.\n",
      "-- check region:67 7:31061704-31091704_reg_Fxyd3:-, 59668 candidate probes\n",
      "--- 25530 probes passed check_dic selection.\n",
      "finish in 2.214s, 402 probes kept.\n",
      "-- check region:68 9:45355185-45385185_reg_Fxyd6:+, 59098 candidate probes\n",
      "--- 33688 probes passed check_dic selection.\n",
      "finish in 2.478s, 462 probes kept.\n",
      "-- check region:69 11:33563959-33593959_reg_Gabrp:-, 58802 candidate probes\n",
      "--- 29388 probes passed check_dic selection.\n",
      "finish in 2.451s, 406 probes kept.\n",
      "-- check region:70 19:3399544-3429544_reg_Gal:-, 59350 candidate probes\n",
      "--- 30657 probes passed check_dic selection.\n",
      "finish in 2.486s, 454 probes kept.\n",
      "-- check region:71 2:122596303-122626303_reg_Gatm:-, 58774 candidate probes\n",
      "--- 29457 probes passed check_dic selection.\n",
      "finish in 2.401s, 420 probes kept.\n",
      "-- check region:72 6:113704880-113734880_reg_Ghrl:-, 59344 candidate probes\n",
      "--- 28065 probes passed check_dic selection.\n",
      "finish in 2.540s, 439 probes kept.\n",
      "-- check region:73 2:25767257-25797257_reg_Gm13539:+, 33266 candidate probes\n",
      "--- 14842 probes passed check_dic selection.\n",
      "finish in 1.309s, 228 probes kept.\n",
      "-- check region:74 11:55083115-55113115_reg_Gm2a:+, 58900 candidate probes\n",
      "--- 28013 probes passed check_dic selection.\n",
      "finish in 2.451s, 416 probes kept.\n",
      "-- check region:75 18:43777878-43807878_reg_Gm94:-, 58494 candidate probes\n",
      "--- 26544 probes passed check_dic selection.\n",
      "finish in 2.295s, 396 probes kept.\n",
      "-- check region:76 7:119444285-119474285_reg_Gp2:-, 58928 candidate probes\n",
      "--- 30376 probes passed check_dic selection.\n",
      "finish in 2.376s, 422 probes kept.\n",
      "-- check region:77 11:54887453-54917453_reg_Gpx3:+, 58856 candidate probes\n",
      "--- 26876 probes passed check_dic selection.\n",
      "finish in 2.297s, 393 probes kept.\n",
      "-- check region:78 13:21297125-21327125_reg_Gpx6:+, 58982 candidate probes\n",
      "--- 31125 probes passed check_dic selection.\n",
      "finish in 2.434s, 437 probes kept.\n",
      "-- check region:79 12:24557283-24587283_reg_Grhl1:+, 59038 candidate probes\n",
      "--- 32178 probes passed check_dic selection.\n",
      "finish in 2.587s, 455 probes kept.\n",
      "-- check region:80 11:98649351-98679351_reg_Gsdma:+, 58042 candidate probes\n",
      "--- 17038 probes passed check_dic selection.\n",
      "finish in 1.897s, 319 probes kept.\n",
      "-- check region:81 9:78340788-78370788_reg_Gsta2:-, 58128 candidate probes\n",
      "--- 14828 probes passed check_dic selection.\n",
      "finish in 1.949s, 274 probes kept.\n",
      "-- check region:82 9:78167774-78197774_reg_Gsta4:+, 58390 candidate probes\n",
      "--- 20063 probes passed check_dic selection.\n",
      "finish in 2.080s, 338 probes kept.\n",
      "-- check region:83 3:107954283-107984283_reg_Gstm3:-, 59490 candidate probes\n",
      "--- 25142 probes passed check_dic selection.\n",
      "finish in 2.198s, 396 probes kept.\n",
      "-- check region:84 19:47839970-47869970_reg_Gsto1:+, 58954 candidate probes\n",
      "--- 23994 probes passed check_dic selection.\n",
      "finish in 2.178s, 338 probes kept.\n",
      "-- check region:85 4:119622704-119652704_reg_Guca2a:+, 52096 candidate probes\n",
      "--- 21081 probes passed check_dic selection.\n",
      "finish in 1.887s, 315 probes kept.\n",
      "-- check region:86 7:142563143-142593143_reg_H19:-, 59866 candidate probes\n",
      "--- 40724 probes passed check_dic selection.\n",
      "finish in 2.798s, 501 probes kept.\n",
      "-- check region:87 7:30909681-30939681_reg_Hamp2:-, 57016 candidate probes\n",
      "--- 14628 probes passed check_dic selection.\n",
      "finish in 1.799s, 238 probes kept.\n",
      "-- check region:88 11:32281489-32311489_reg_Hba-a2:+, 54118 candidate probes\n",
      "--- 24903 probes passed check_dic selection.\n",
      "finish in 2.064s, 380 probes kept.\n",
      "-- check region:89 7:103838216-103868216_reg_Hbb-y:-, 58158 candidate probes\n",
      "--- 24895 probes passed check_dic selection.\n",
      "finish in 2.143s, 403 probes kept.\n",
      "-- check region:90 2:153093468-153123468_reg_Hck:+, 48344 candidate probes\n",
      "--- 25408 probes passed check_dic selection.\n",
      "finish in 2.016s, 356 probes kept.\n",
      "-- check region:91 2:163491808-163521808_reg_Hnf4a:+, 59306 candidate probes\n",
      "--- 25237 probes passed check_dic selection.\n",
      "finish in 2.163s, 378 probes kept.\n",
      "-- check region:92 3:3493030-3523030_reg_Hnf4g:+, 58354 candidate probes\n",
      "--- 24405 probes passed check_dic selection.\n",
      "finish in 2.126s, 373 probes kept.\n",
      "-- check region:93 8:109564172-109594172_reg_Hp:-, 59218 candidate probes\n",
      "--- 27800 probes passed check_dic selection.\n",
      "finish in 2.380s, 444 probes kept.\n",
      "-- check region:94 6:142283423-142313423_reg_Iapp:+, 56302 candidate probes\n",
      "--- 18866 probes passed check_dic selection.\n",
      "finish in 1.868s, 306 probes kept.\n",
      "-- check region:95 13:8937863-8967863_reg_Idi2:+, 41940 candidate probes\n",
      "--- 13213 probes passed check_dic selection.\n",
      "finish in 1.374s, 245 probes kept.\n",
      "-- check region:96 12:103442223-103472223_reg_Ifi27l2b:-, 48848 candidate probes\n",
      "--- 12049 probes passed check_dic selection.\n",
      "finish in 1.666s, 193 probes kept.\n",
      "-- check region:97 10:87843265-87873265_reg_Igf1:+, 59642 candidate probes\n",
      "--- 33585 probes passed check_dic selection.\n",
      "finish in 2.558s, 473 probes kept.\n",
      "-- check region:98 7:142651816-142681816_reg_Igf2:-, 59670 candidate probes\n",
      "--- 30499 probes passed check_dic selection.\n",
      "finish in 2.437s, 476 probes kept.\n",
      "-- check region:99 1:72859884-72889884_reg_Igfbp5:-, 59572 candidate probes\n",
      "--- 31950 probes passed check_dic selection.\n",
      "finish in 2.427s, 455 probes kept.\n",
      "-- check region:100 15:102129362-102159362_reg_Igfbp6:+, 58934 candidate probes\n",
      "--- 25826 probes passed check_dic selection.\n",
      "finish in 2.314s, 387 probes kept.\n",
      "-- check region:101 7:142728381-142758381_reg_Ins2:-, 56480 candidate probes\n",
      "--- 22538 probes passed check_dic selection.\n",
      "finish in 1.980s, 332 probes kept.\n",
      "-- check region:102 4:156185796-156215796_reg_Isg15:-, 58976 candidate probes\n",
      "--- 28774 probes passed check_dic selection.\n",
      "finish in 2.382s, 421 probes kept.\n",
      "-- check region:103 2:71730616-71760616_reg_Itga6:+, 59210 candidate probes\n",
      "--- 32407 probes passed check_dic selection.\n",
      "finish in 2.522s, 459 probes kept.\n",
      "-- check region:104 11:115499387-115529387_reg_Jpt1:-, 58398 candidate probes\n",
      "--- 21543 probes passed check_dic selection.\n",
      "finish in 2.130s, 348 probes kept.\n",
      "-- check region:105 7:43675418-43705418_reg_Klk14:+, 58680 candidate probes\n",
      "--- 22669 probes passed check_dic selection.\n",
      "finish in 2.180s, 378 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:106 7:43796294-43826294_reg_Klk7:+, 59190 candidate probes\n",
      "--- 25128 probes passed check_dic selection.\n",
      "finish in 2.340s, 390 probes kept.\n",
      "-- check region:107 11:99374364-99404364_reg_Krt10:-, 59524 candidate probes\n",
      "--- 31335 probes passed check_dic selection.\n",
      "finish in 2.464s, 445 probes kept.\n",
      "-- check region:108 11:100106566-100136566_reg_Krt13:-, 54122 candidate probes\n",
      "--- 30844 probes passed check_dic selection.\n",
      "finish in 2.358s, 440 probes kept.\n",
      "-- check region:109 11:100192548-100222548_reg_Krt14:-, 28148 candidate probes\n",
      "--- 13284 probes passed check_dic selection.\n",
      "finish in 1.264s, 195 probes kept.\n",
      "-- check region:110 11:100233902-100263902_reg_Krt16:-, 23626 candidate probes\n",
      "--- 4982 probes passed check_dic selection.\n",
      "finish in 0.823s, 109 probes kept.\n",
      "-- check region:111 11:100246029-100276029_reg_Krt17:-, 58810 candidate probes\n",
      "--- 31040 probes passed check_dic selection.\n",
      "finish in 2.446s, 439 probes kept.\n",
      "-- check region:112 11:100133665-100163665_reg_Krt19:-, 58888 candidate probes\n",
      "--- 29924 probes passed check_dic selection.\n",
      "finish in 2.437s, 428 probes kept.\n",
      "-- check region:113 15:101697898-101727898_reg_Krt5:-, 59662 candidate probes\n",
      "--- 32064 probes passed check_dic selection.\n",
      "finish in 2.504s, 429 probes kept.\n",
      "-- check region:114 15:101728109-101758109_reg_Krt71:-, 58580 candidate probes\n",
      "--- 30667 probes passed check_dic selection.\n",
      "finish in 2.396s, 429 probes kept.\n",
      "-- check region:115 15:101877920-101907920_reg_Krt76:-, 46270 candidate probes\n",
      "--- 23908 probes passed check_dic selection.\n",
      "finish in 1.916s, 335 probes kept.\n",
      "-- check region:116 15:101854705-101884705_reg_Krt77:-, 59860 candidate probes\n",
      "--- 34080 probes passed check_dic selection.\n",
      "finish in 2.606s, 478 probes kept.\n",
      "-- check region:117 11:100178246-100208246_reg_Krt9:-, 59090 candidate probes\n",
      "--- 29171 probes passed check_dic selection.\n",
      "finish in 2.300s, 425 probes kept.\n",
      "-- check region:118 7:30772896-30802896_reg_Krtdap:+, 58290 candidate probes\n",
      "--- 27445 probes passed check_dic selection.\n",
      "finish in 2.272s, 405 probes kept.\n",
      "-- check region:119 2:32373252-32403252_reg_Lcn2:-, 59100 candidate probes\n",
      "--- 26311 probes passed check_dic selection.\n",
      "finish in 2.261s, 445 probes kept.\n",
      "-- check region:120 2:25750569-25780569_reg_Lcn3:+, 59752 candidate probes\n",
      "--- 30168 probes passed check_dic selection.\n",
      "finish in 2.403s, 424 probes kept.\n",
      "-- check region:121 15:78840529-78870529_reg_Lgals2:-, 58844 candidate probes\n",
      "--- 27730 probes passed check_dic selection.\n",
      "finish in 2.299s, 417 probes kept.\n",
      "-- check region:122 14:47352751-47382751_reg_Lgals3:+, 59298 candidate probes\n",
      "--- 29244 probes passed check_dic selection.\n",
      "finish in 2.303s, 415 probes kept.\n",
      "-- check region:123 15:7075614-7105614_reg_Lifr:+, 59624 candidate probes\n",
      "--- 32513 probes passed check_dic selection.\n",
      "finish in 2.506s, 453 probes kept.\n",
      "-- check region:124 9:111004271-111034271_reg_Ltf:+, 58258 candidate probes\n",
      "--- 27334 probes passed check_dic selection.\n",
      "finish in 2.290s, 416 probes kept.\n",
      "-- check region:125 15:75033830-75063830_reg_Ly6c1:-, 47374 candidate probes\n",
      "--- 3834 probes passed check_dic selection.\n",
      "finish in 1.182s, 126 probes kept.\n",
      "-- check region:126 15:75096970-75126970_reg_Ly6c2:-, 58210 candidate probes\n",
      "--- 4235 probes passed check_dic selection.\n",
      "finish in 1.344s, 147 probes kept.\n",
      "-- check region:127 15:74748620-74778620_reg_Ly6d:-, 56848 candidate probes\n",
      "--- 28531 probes passed check_dic selection.\n",
      "finish in 2.292s, 409 probes kept.\n",
      "-- check region:128 17:35070610-35100610_reg_Ly6g6f:-, 58932 candidate probes\n",
      "--- 22950 probes passed check_dic selection.\n",
      "finish in 2.166s, 391 probes kept.\n",
      "-- check region:129 15:74719329-74749329_reg_Lypd2:-, 38682 candidate probes\n",
      "--- 23532 probes passed check_dic selection.\n",
      "finish in 1.789s, 318 probes kept.\n",
      "-- check region:130 7:24621550-24651550_reg_Lypd3:+, 59096 candidate probes\n",
      "--- 23726 probes passed check_dic selection.\n",
      "finish in 2.100s, 385 probes kept.\n",
      "-- check region:131 2:127641695-127671695_reg_Mal:-, 59118 candidate probes\n",
      "--- 31111 probes passed check_dic selection.\n",
      "finish in 2.412s, 427 probes kept.\n",
      "-- check region:132 12:109526001-109556001_reg_Meg3:+, 59806 candidate probes\n",
      "--- 35676 probes passed check_dic selection.\n",
      "finish in 2.571s, 487 probes kept.\n",
      "-- check region:133 2:3407648-3437648_reg_Meig1:-, 58532 candidate probes\n",
      "--- 28679 probes passed check_dic selection.\n",
      "finish in 2.319s, 427 probes kept.\n",
      "-- check region:134 6:136860823-136890823_reg_Mgp:-, 58416 candidate probes\n",
      "--- 28732 probes passed check_dic selection.\n",
      "finish in 2.287s, 416 probes kept.\n",
      "-- check region:135 1:90900085-90930085_reg_Mlph:+, 59078 candidate probes\n",
      "--- 31266 probes passed check_dic selection.\n",
      "finish in 2.425s, 432 probes kept.\n",
      "-- check region:136 11:87778581-87808581_reg_Mpo:+, 59520 candidate probes\n",
      "--- 27194 probes passed check_dic selection.\n",
      "finish in 2.362s, 436 probes kept.\n",
      "-- check region:137 8:94137607-94167607_reg_Mt3:+, 30150 candidate probes\n",
      "--- 11481 probes passed check_dic selection.\n",
      "finish in 1.050s, 187 probes kept.\n",
      "-- check region:138 8:94122197-94152197_reg_Mt4:+, 59242 candidate probes\n",
      "--- 28981 probes passed check_dic selection.\n",
      "finish in 2.403s, 413 probes kept.\n",
      "-- check region:139 16:33779037-33809037_reg_Muc13:+, 58042 candidate probes\n",
      "--- 23753 probes passed check_dic selection.\n",
      "finish in 2.200s, 370 probes kept.\n",
      "-- check region:140 17:45587102-45617102_reg_Mymx:-, 58956 candidate probes\n",
      "--- 25497 probes passed check_dic selection.\n",
      "finish in 2.244s, 391 probes kept.\n",
      "-- check region:141 1:134274989-134304989_reg_Myog:+, 59366 candidate probes\n",
      "--- 30720 probes passed check_dic selection.\n",
      "finish in 2.398s, 454 probes kept.\n",
      "-- check region:142 11:4933064-4963064_reg_Nefh:-, 58996 candidate probes\n",
      "--- 27354 probes passed check_dic selection.\n",
      "finish in 2.317s, 420 probes kept.\n",
      "-- check region:143 9:110404747-110434747_reg_Ngp:+, 58770 candidate probes\n",
      "--- 24592 probes passed check_dic selection.\n",
      "finish in 2.153s, 405 probes kept.\n",
      "-- check region:144 4:147985722-148015722_reg_Nppa:+, 59394 candidate probes\n",
      "--- 27719 probes passed check_dic selection.\n",
      "finish in 2.308s, 419 probes kept.\n",
      "-- check region:145 6:49807710-49837710_reg_Npy:+, 59690 candidate probes\n",
      "--- 32576 probes passed check_dic selection.\n",
      "finish in 2.628s, 448 probes kept.\n",
      "-- check region:146 9:37537904-37567904_reg_Nrgn:-, 59582 candidate probes\n",
      "--- 31271 probes passed check_dic selection.\n",
      "finish in 2.598s, 453 probes kept.\n",
      "-- check region:147 10:102475486-102505486_reg_Nts:-, 59361 candidate probes\n",
      "--- 26439 probes passed check_dic selection.\n",
      "finish in 2.282s, 413 probes kept.\n",
      "-- check region:148 5:120899536-120929536_reg_Oas1d:+, 58068 candidate probes\n",
      "--- 13112 probes passed check_dic selection.\n",
      "finish in 1.708s, 261 probes kept.\n",
      "-- check region:149 5:120846421-120876421_reg_Oas1h:+, 57490 candidate probes\n",
      "--- 17466 probes passed check_dic selection.\n",
      "finish in 1.964s, 316 probes kept.\n",
      "-- check region:150 15:89364587-89394587_reg_Odf3b:-, 59686 candidate probes\n",
      "--- 27711 probes passed check_dic selection.\n",
      "finish in 2.480s, 451 probes kept.\n",
      "-- check region:151 2:28177992-28207992_reg_Olfm1:+, 59524 candidate probes\n",
      "--- 32910 probes passed check_dic selection.\n",
      "finish in 2.512s, 452 probes kept.\n",
      "-- check region:152 16:96452606-96482606_reg_Pcp4:+, 59466 candidate probes\n",
      "--- 31521 probes passed check_dic selection.\n",
      "finish in 2.495s, 454 probes kept.\n",
      "-- check region:153 4:115073708-115103708_reg_Pdzk1ip1:+, 59480 candidate probes\n",
      "--- 32513 probes passed check_dic selection.\n",
      "finish in 2.495s, 452 probes kept.\n",
      "-- check region:154 4:4123819-4153819_reg_Penk:-, 58636 candidate probes\n",
      "--- 27054 probes passed check_dic selection.\n",
      "finish in 2.208s, 391 probes kept.\n",
      "-- check region:155 17:47711842-47741842_reg_Pgc:+, 59038 candidate probes\n",
      "--- 28760 probes passed check_dic selection.\n",
      "finish in 2.369s, 424 probes kept.\n",
      "-- check region:156 1:130811684-130841684_reg_Pigr:+, 59814 candidate probes\n",
      "--- 36942 probes passed check_dic selection.\n",
      "finish in 2.652s, 493 probes kept.\n",
      "-- check region:157 11:68417121-68447121_reg_Pik3r5:+, 59132 candidate probes\n",
      "--- 28860 probes passed check_dic selection.\n",
      "finish in 2.375s, 421 probes kept.\n",
      "-- check region:158 5:115451262-115481262_reg_Pla2g1b:+, 58576 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 20073 probes passed check_dic selection.\n",
      "finish in 1.982s, 333 probes kept.\n",
      "-- check region:159 5:100557245-100587245_reg_Plac8:-, 58798 candidate probes\n",
      "--- 25295 probes passed check_dic selection.\n",
      "finish in 2.149s, 376 probes kept.\n",
      "-- check region:160 13:110380046-110410046_reg_Plk2:+, 59592 candidate probes\n",
      "--- 23915 probes passed check_dic selection.\n",
      "finish in 2.144s, 395 probes kept.\n",
      "-- check region:161 8:71496752-71526752_reg_Plvap:-, 58214 candidate probes\n",
      "--- 21266 probes passed check_dic selection.\n",
      "finish in 2.028s, 379 probes kept.\n",
      "-- check region:162 12:3939951-3969951_reg_Pomc:+, 58864 candidate probes\n",
      "--- 25526 probes passed check_dic selection.\n",
      "finish in 2.151s, 411 probes kept.\n",
      "-- check region:163 9:43195369-43225369_reg_Pou2f3:-, 59132 candidate probes\n",
      "--- 26730 probes passed check_dic selection.\n",
      "finish in 2.212s, 392 probes kept.\n",
      "-- check region:164 15:74699839-74729839_reg_Psca:+, 59642 candidate probes\n",
      "--- 31807 probes passed check_dic selection.\n",
      "finish in 2.484s, 501 probes kept.\n",
      "-- check region:165 2:25455046-25485046_reg_Ptgds:-, 59554 candidate probes\n",
      "--- 35150 probes passed check_dic selection.\n",
      "finish in 2.582s, 479 probes kept.\n",
      "-- check region:166 11:102092832-102122832_reg_Pyy:-, 59360 candidate probes\n",
      "--- 28190 probes passed check_dic selection.\n",
      "finish in 2.273s, 404 probes kept.\n",
      "-- check region:167 11:74595915-74625915_reg_Rap1gap2:-, 58804 candidate probes\n",
      "--- 23803 probes passed check_dic selection.\n",
      "finish in 2.171s, 359 probes kept.\n",
      "-- check region:168 3:67500523-67530523_reg_Rarres1:-, 58966 candidate probes\n",
      "--- 32474 probes passed check_dic selection.\n",
      "finish in 2.458s, 433 probes kept.\n",
      "-- check region:169 6:78355657-78385657_reg_Reg3b:+, 57644 candidate probes\n",
      "--- 21934 probes passed check_dic selection.\n",
      "finish in 2.030s, 351 probes kept.\n",
      "-- check region:170 3:98207156-98237156_reg_Reg4:+, 59424 candidate probes\n",
      "--- 27729 probes passed check_dic selection.\n",
      "finish in 2.366s, 399 probes kept.\n",
      "-- check region:171 16:48827551-48857551_reg_Retnla:+, 58654 candidate probes\n",
      "--- 16102 probes passed check_dic selection.\n",
      "finish in 1.826s, 246 probes kept.\n",
      "-- check region:172 3:94465341-94495341_reg_Riiad1:-, 59500 candidate probes\n",
      "--- 28985 probes passed check_dic selection.\n",
      "finish in 2.364s, 439 probes kept.\n",
      "-- check region:173 14:51131785-51161785_reg_Rnase1:-, 56892 candidate probes\n",
      "--- 11659 probes passed check_dic selection.\n",
      "finish in 1.565s, 228 probes kept.\n",
      "-- check region:174 14:51076077-51106077_reg_Rnase4:+, 59468 candidate probes\n",
      "--- 20982 probes passed check_dic selection.\n",
      "finish in 2.031s, 349 probes kept.\n",
      "-- check region:175 12:26441452-26471452_reg_Rsad2:-, 59268 candidate probes\n",
      "--- 28287 probes passed check_dic selection.\n",
      "finish in 2.322s, 437 probes kept.\n",
      "-- check region:176 17:31262383-31292383_reg_Rsph1:-, 56674 candidate probes\n",
      "--- 27101 probes passed check_dic selection.\n",
      "finish in 2.226s, 373 probes kept.\n",
      "-- check region:177 3:90680721-90710721_reg_S100a9:-, 59278 candidate probes\n",
      "--- 26105 probes passed check_dic selection.\n",
      "finish in 2.144s, 376 probes kept.\n",
      "-- check region:178 7:46727980-46757980_reg_Saa1:-, 17450 candidate probes\n",
      "--- 8353 probes passed check_dic selection.\n",
      "finish in 0.750s, 135 probes kept.\n",
      "-- check region:179 7:46736790-46766790_reg_Saa2:+, 56998 candidate probes\n",
      "--- 27168 probes passed check_dic selection.\n",
      "finish in 2.232s, 447 probes kept.\n",
      "-- check region:180 1:107346198-107376198_reg_Serpinb11:+, 59258 candidate probes\n",
      "--- 26876 probes passed check_dic selection.\n",
      "finish in 2.448s, 415 probes kept.\n",
      "-- check region:181 14:41116782-41146782_reg_Sftpa1:+, 59110 candidate probes\n",
      "--- 27424 probes passed check_dic selection.\n",
      "finish in 2.233s, 414 probes kept.\n",
      "-- check region:182 14:41170149-41200149_reg_Sftpd:-, 59674 candidate probes\n",
      "--- 27142 probes passed check_dic selection.\n",
      "finish in 2.358s, 406 probes kept.\n",
      "-- check region:183 2:164374095-164404095_reg_Slpi:-, 59258 candidate probes\n",
      "--- 28890 probes passed check_dic selection.\n",
      "finish in 2.393s, 424 probes kept.\n",
      "-- check region:184 17:25555686-25585686_reg_Sox8:-, 59338 candidate probes\n",
      "--- 33515 probes passed check_dic selection.\n",
      "finish in 2.772s, 464 probes kept.\n",
      "-- check region:185 4:43638594-43668594_reg_Spag8:-, 59834 candidate probes\n",
      "--- 29661 probes passed check_dic selection.\n",
      "finish in 2.572s, 466 probes kept.\n",
      "-- check region:186 11:55408183-55438183_reg_Sparc:-, 58954 candidate probes\n",
      "--- 26474 probes passed check_dic selection.\n",
      "finish in 2.398s, 392 probes kept.\n",
      "-- check region:187 7:44517071-44547071_reg_Spib:-, 59566 candidate probes\n",
      "--- 29552 probes passed check_dic selection.\n",
      "finish in 2.466s, 457 probes kept.\n",
      "-- check region:188 5:104420118-104450118_reg_Spp1:+, 59054 candidate probes\n",
      "--- 25092 probes passed check_dic selection.\n",
      "finish in 2.217s, 379 probes kept.\n",
      "-- check region:189 3:92423789-92453789_reg_Sprr1b:-, 59420 candidate probes\n",
      "--- 20117 probes passed check_dic selection.\n",
      "finish in 2.022s, 338 probes kept.\n",
      "-- check region:190 3:92392991-92422991_reg_Sprr2i:+, 59356 candidate probes\n",
      "--- 22887 probes passed check_dic selection.\n",
      "finish in 2.109s, 377 probes kept.\n",
      "-- check region:191 16:23875958-23905958_reg_Sst:-, 59762 candidate probes\n",
      "--- 30124 probes passed check_dic selection.\n",
      "finish in 2.408s, 434 probes kept.\n",
      "-- check region:192 16:37369962-37399962_reg_Stxbp5l:-, 59606 candidate probes\n",
      "--- 23200 probes passed check_dic selection.\n",
      "finish in 2.232s, 396 probes kept.\n",
      "-- check region:193 19:10374090-10404090_reg_Syt7:+, 59614 candidate probes\n",
      "--- 35167 probes passed check_dic selection.\n",
      "finish in 2.836s, 466 probes kept.\n",
      "-- check region:194 6:7539879-7569879_reg_Tac1:+, 58070 candidate probes\n",
      "--- 23404 probes passed check_dic selection.\n",
      "finish in 2.062s, 376 probes kept.\n",
      "-- check region:195 10:127709478-127739478_reg_Tac2:+, 59792 candidate probes\n",
      "--- 32224 probes passed check_dic selection.\n",
      "finish in 2.574s, 473 probes kept.\n",
      "-- check region:196 6:67520796-67550796_reg_Tacstd2:-, 58880 candidate probes\n",
      "--- 26264 probes passed check_dic selection.\n",
      "finish in 2.413s, 382 probes kept.\n",
      "-- check region:197 17:31129282-31159282_reg_Tff2:-, 28996 candidate probes\n",
      "--- 14277 probes passed check_dic selection.\n",
      "finish in 1.188s, 196 probes kept.\n",
      "-- check region:198 17:31114646-31144646_reg_Tff3:-, 59642 candidate probes\n",
      "--- 28757 probes passed check_dic selection.\n",
      "finish in 2.458s, 432 probes kept.\n",
      "-- check region:199 11:118340740-118370740_reg_Timp2:-, 59702 candidate probes\n",
      "--- 30746 probes passed check_dic selection.\n",
      "finish in 2.508s, 433 probes kept.\n",
      "-- check region:200 16:56871166-56901166_reg_Tmem45a:-, 59382 candidate probes\n",
      "--- 21392 probes passed check_dic selection.\n",
      "finish in 2.150s, 329 probes kept.\n",
      "-- check region:201 15:54263484-54293484_reg_Tnfrsf11b:-, 59462 candidate probes\n",
      "--- 26345 probes passed check_dic selection.\n",
      "finish in 2.248s, 386 probes kept.\n",
      "-- check region:202 16:25668763-25698763_reg_Trp63:+, 59652 candidate probes\n",
      "--- 23614 probes passed check_dic selection.\n",
      "finish in 2.189s, 382 probes kept.\n",
      "-- check region:203 7:143079642-143109642_reg_Trpm5:-, 59890 candidate probes\n",
      "--- 33397 probes passed check_dic selection.\n",
      "finish in 2.748s, 478 probes kept.\n",
      "-- check region:204 4:116152601-116182601_reg_Tspan1:-, 59714 candidate probes\n",
      "--- 26362 probes passed check_dic selection.\n",
      "finish in 2.407s, 415 probes kept.\n",
      "-- check region:205 9:44788072-44818072_reg_Ttc36:-, 59852 candidate probes\n",
      "--- 24442 probes passed check_dic selection.\n",
      "finish in 2.247s, 445 probes kept.\n",
      "-- check region:206 10:75928057-75958057_reg_Vpreb3:+, 59480 candidate probes\n",
      "--- 26158 probes passed check_dic selection.\n",
      "finish in 2.373s, 391 probes kept.\n",
      "-- check region:207 1:164920527-164950527_reg_Xcl1:-, 59846 candidate probes\n",
      "--- 25900 probes passed check_dic selection.\n",
      "finish in 2.488s, 409 probes kept.\n",
      "-- check region:208 7:127072328-127102328_reg_Zg16:-, 59334 candidate probes\n",
      "--- 21752 probes passed check_dic selection.\n",
      "finish in 2.063s, 347 probes kept.\n"
     ]
    }
   ],
   "source": [
    "pbs, pb_scores = pb_designer.check_probes()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "occasional-charter",
   "metadata": {},
   "source": [
    "### save probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "helpful-manual",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\reports\\merged_probes.pbr\n",
      "-- number of probes kept: 81511\n"
     ]
    }
   ],
   "source": [
    "overwrite_savefile = True \n",
    "if not os.path.exists(pb_designer.save_file) or overwrite_savefile:\n",
    "    #pb_designer.plots()\n",
    "    #pb_designer.save_csv()\n",
    "    pb_designer.save_to_file()\n",
    "print(f\"-- number of probes kept: {len(pb_designer.kept_probes)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "completed-winter",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "# after finish designing, remove all loaded OT maps to freeup memory usage\n",
    "pb_designer.release_OTmaps()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "further-scottish",
   "metadata": {},
   "source": [
    "## 2.3 summarize into a pb_dict\n",
    "\n",
    "start over from here is possible"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "valid-porter",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\reports\\merged_probes.pbr\n"
     ]
    }
   ],
   "source": [
    "# filename to save probe reports\n",
    "save_file = os.path.join(report_folder, f'merged_probes.pbr')\n",
    "print(save_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "ahead-owner",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Loading from savefile: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\reports\\merged_probes.pbr.\n",
      "81511\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "verbose=True\n",
    "cand_prob_nums = []\n",
    "pb_dict = {}\n",
    "# Parse segemented list\n",
    "_pbde = library_tools.design.pb_reports_class()\n",
    "_pbde.load_from_file(save_file)\n",
    "\n",
    "gene_names = np.sort(np.unique([_v['reg_name'].split('reg_')[1].split('_')[0] for _s, _v in _pbde.kept_probes.items()]))\n",
    "for _gene_name in gene_names:\n",
    "    _sel_pbs = {_s:_v for _s, _v in _pbde.kept_probes.items() if _v['reg_name'].split('_')[1]==_gene_name}\n",
    "\n",
    "    if len(_sel_pbs) > 0:\n",
    "        _region_gid = list(_sel_pbs.values())[0]['reg_name'].split('_')[-1]\n",
    "        cand_prob_nums.append(len(_sel_pbs))\n",
    "        if verbose:\n",
    "            print(f\"-- appending gene:{_gene_name} with {len(_sel_pbs)} probes\")\n",
    "        _splitted_pbde = ld.pb_reports_class()\n",
    "        _splitted_pbde.kept_probes = _sel_pbs\n",
    "        pb_dict[_gene_name] = _splitted_pbde"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "consolidated-north",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending gene:Acpp with 425 probes\n",
      "-- appending gene:Agt with 352 probes\n",
      "-- appending gene:Aldh3a1 with 433 probes\n",
      "-- appending gene:Aldob with 414 probes\n",
      "-- appending gene:Alox5ap with 398 probes\n",
      "-- appending gene:Alpi with 399 probes\n",
      "-- appending gene:Alpl with 395 probes\n",
      "-- appending gene:Ang4 with 175 probes\n",
      "-- appending gene:Anxa8 with 497 probes\n",
      "-- appending gene:Apoa1 with 454 probes\n",
      "-- appending gene:Apoa2 with 373 probes\n",
      "-- appending gene:Apoc2 with 320 probes\n",
      "-- appending gene:Apod with 414 probes\n",
      "-- appending gene:Ascl1 with 415 probes\n",
      "-- appending gene:Avil with 434 probes\n",
      "-- appending gene:Bgn with 475 probes\n",
      "-- appending gene:C1qb with 409 probes\n",
      "-- appending gene:C3 with 404 probes\n",
      "-- appending gene:Calb2 with 456 probes\n",
      "-- appending gene:Cald1 with 402 probes\n",
      "-- appending gene:Camp with 329 probes\n",
      "-- appending gene:Car4 with 429 probes\n",
      "-- appending gene:Car8 with 374 probes\n",
      "-- appending gene:Ccdc30 with 389 probes\n",
      "-- appending gene:Cck with 404 probes\n",
      "-- appending gene:Ccl1 with 443 probes\n",
      "-- appending gene:Ccl2 with 472 probes\n",
      "-- appending gene:Ccl20 with 425 probes\n",
      "-- appending gene:Ccl21a with 5 probes\n",
      "-- appending gene:Ccl6 with 200 probes\n",
      "-- appending gene:Ccl9 with 394 probes\n",
      "-- appending gene:Cd14 with 353 probes\n",
      "-- appending gene:Cebpb with 411 probes\n",
      "-- appending gene:Celf3 with 26 probes\n",
      "-- appending gene:Cga with 346 probes\n",
      "-- appending gene:Cilp with 396 probes\n",
      "-- appending gene:Ckb with 429 probes\n",
      "-- appending gene:Ckm with 392 probes\n",
      "-- appending gene:Clps with 329 probes\n",
      "-- appending gene:Cnfn with 453 probes\n",
      "-- appending gene:Cnmd with 445 probes\n",
      "-- appending gene:Col1a1 with 471 probes\n",
      "-- appending gene:Col4a1 with 468 probes\n",
      "-- appending gene:Col6a1 with 464 probes\n",
      "-- appending gene:Cpn1 with 395 probes\n",
      "-- appending gene:Crabp1 with 465 probes\n",
      "-- appending gene:Cryaa with 389 probes\n",
      "-- appending gene:Ctrb1 with 362 probes\n",
      "-- appending gene:Cxcl13 with 393 probes\n",
      "-- appending gene:Cyp1a2 with 402 probes\n",
      "-- appending gene:Cyp2f2 with 350 probes\n",
      "-- appending gene:Dbn1 with 433 probes\n",
      "-- appending gene:Dclk1 with 444 probes\n",
      "-- appending gene:Dlk1 with 466 probes\n",
      "-- appending gene:Dppa3 with 279 probes\n",
      "-- appending gene:Ecm1 with 402 probes\n",
      "-- appending gene:Efna3 with 437 probes\n",
      "-- appending gene:Elovl4 with 418 probes\n",
      "-- appending gene:Fabp1 with 378 probes\n",
      "-- appending gene:Fabp6 with 390 probes\n",
      "-- appending gene:Fam183b with 370 probes\n",
      "-- appending gene:Fam25c with 415 probes\n",
      "-- appending gene:Fgg with 443 probes\n",
      "-- appending gene:Foxa1 with 448 probes\n",
      "-- appending gene:Foxa2 with 464 probes\n",
      "-- appending gene:Foxa3 with 403 probes\n",
      "-- appending gene:Foxj1 with 468 probes\n",
      "-- appending gene:Fxyd3 with 402 probes\n",
      "-- appending gene:Fxyd6 with 462 probes\n",
      "-- appending gene:Gabrp with 406 probes\n",
      "-- appending gene:Gal with 454 probes\n",
      "-- appending gene:Gatm with 420 probes\n",
      "-- appending gene:Ghrl with 439 probes\n",
      "-- appending gene:Gm13539 with 228 probes\n",
      "-- appending gene:Gm2a with 416 probes\n",
      "-- appending gene:Gm94 with 396 probes\n",
      "-- appending gene:Gp2 with 422 probes\n",
      "-- appending gene:Gpx3 with 393 probes\n",
      "-- appending gene:Gpx6 with 437 probes\n",
      "-- appending gene:Grhl1 with 455 probes\n",
      "-- appending gene:Gsdma with 319 probes\n",
      "-- appending gene:Gsta2 with 274 probes\n",
      "-- appending gene:Gsta4 with 338 probes\n",
      "-- appending gene:Gstm3 with 396 probes\n",
      "-- appending gene:Gsto1 with 338 probes\n",
      "-- appending gene:Guca2a with 315 probes\n",
      "-- appending gene:H19 with 501 probes\n",
      "-- appending gene:Hamp2 with 238 probes\n",
      "-- appending gene:Hba-a2 with 380 probes\n",
      "-- appending gene:Hbb-y with 403 probes\n",
      "-- appending gene:Hck with 356 probes\n",
      "-- appending gene:Hnf4a with 378 probes\n",
      "-- appending gene:Hnf4g with 373 probes\n",
      "-- appending gene:Hp with 444 probes\n",
      "-- appending gene:Iapp with 306 probes\n",
      "-- appending gene:Idi2 with 245 probes\n",
      "-- appending gene:Ifi27l2b with 193 probes\n",
      "-- appending gene:Igf1 with 473 probes\n",
      "-- appending gene:Igf2 with 476 probes\n",
      "-- appending gene:Igfbp5 with 455 probes\n",
      "-- appending gene:Igfbp6 with 387 probes\n",
      "-- appending gene:Ins2 with 332 probes\n",
      "-- appending gene:Isg15 with 421 probes\n",
      "-- appending gene:Itga6 with 459 probes\n",
      "-- appending gene:Jpt1 with 348 probes\n",
      "-- appending gene:Klk14 with 378 probes\n",
      "-- appending gene:Klk7 with 390 probes\n",
      "-- appending gene:Krt10 with 445 probes\n",
      "-- appending gene:Krt13 with 440 probes\n",
      "-- appending gene:Krt14 with 195 probes\n",
      "-- appending gene:Krt16 with 109 probes\n",
      "-- appending gene:Krt17 with 439 probes\n",
      "-- appending gene:Krt19 with 428 probes\n",
      "-- appending gene:Krt5 with 429 probes\n",
      "-- appending gene:Krt71 with 429 probes\n",
      "-- appending gene:Krt76 with 335 probes\n",
      "-- appending gene:Krt77 with 478 probes\n",
      "-- appending gene:Krt9 with 425 probes\n",
      "-- appending gene:Krtdap with 405 probes\n",
      "-- appending gene:Lcn2 with 445 probes\n",
      "-- appending gene:Lcn3 with 424 probes\n",
      "-- appending gene:Lgals2 with 417 probes\n",
      "-- appending gene:Lgals3 with 415 probes\n",
      "-- appending gene:Lifr with 453 probes\n",
      "-- appending gene:Ltf with 416 probes\n",
      "-- appending gene:Ly6c1 with 126 probes\n",
      "-- appending gene:Ly6c2 with 147 probes\n",
      "-- appending gene:Ly6d with 409 probes\n",
      "-- appending gene:Ly6g6f with 391 probes\n",
      "-- appending gene:Lypd2 with 318 probes\n",
      "-- appending gene:Lypd3 with 385 probes\n",
      "-- appending gene:Mal with 427 probes\n",
      "-- appending gene:Meg3 with 487 probes\n",
      "-- appending gene:Meig1 with 427 probes\n",
      "-- appending gene:Mgp with 416 probes\n",
      "-- appending gene:Mlph with 432 probes\n",
      "-- appending gene:Mpo with 436 probes\n",
      "-- appending gene:Mt3 with 187 probes\n",
      "-- appending gene:Mt4 with 413 probes\n",
      "-- appending gene:Muc13 with 370 probes\n",
      "-- appending gene:Mymx with 391 probes\n",
      "-- appending gene:Myog with 454 probes\n",
      "-- appending gene:Nefh with 420 probes\n",
      "-- appending gene:Ngp with 405 probes\n",
      "-- appending gene:Nppa with 419 probes\n",
      "-- appending gene:Npy with 448 probes\n",
      "-- appending gene:Nrgn with 453 probes\n",
      "-- appending gene:Nts with 413 probes\n",
      "-- appending gene:Oas1d with 261 probes\n",
      "-- appending gene:Oas1h with 316 probes\n",
      "-- appending gene:Odf3b with 451 probes\n",
      "-- appending gene:Olfm1 with 452 probes\n",
      "-- appending gene:Pcp4 with 454 probes\n",
      "-- appending gene:Pdzk1ip1 with 452 probes\n",
      "-- appending gene:Penk with 391 probes\n",
      "-- appending gene:Pgc with 424 probes\n",
      "-- appending gene:Pigr with 493 probes\n",
      "-- appending gene:Pik3r5 with 421 probes\n",
      "-- appending gene:Pla2g1b with 333 probes\n",
      "-- appending gene:Plac8 with 376 probes\n",
      "-- appending gene:Plk2 with 395 probes\n",
      "-- appending gene:Plvap with 379 probes\n",
      "-- appending gene:Pomc with 411 probes\n",
      "-- appending gene:Pou2f3 with 392 probes\n",
      "-- appending gene:Psca with 501 probes\n",
      "-- appending gene:Ptgds with 479 probes\n",
      "-- appending gene:Pyy with 404 probes\n",
      "-- appending gene:Rap1gap2 with 359 probes\n",
      "-- appending gene:Rarres1 with 433 probes\n",
      "-- appending gene:Reg3b with 351 probes\n",
      "-- appending gene:Reg4 with 399 probes\n",
      "-- appending gene:Retnla with 246 probes\n",
      "-- appending gene:Riiad1 with 439 probes\n",
      "-- appending gene:Rnase1 with 228 probes\n",
      "-- appending gene:Rnase4 with 349 probes\n",
      "-- appending gene:Rsad2 with 437 probes\n",
      "-- appending gene:Rsph1 with 373 probes\n",
      "-- appending gene:S100a9 with 376 probes\n",
      "-- appending gene:Saa1 with 135 probes\n",
      "-- appending gene:Saa2 with 447 probes\n",
      "-- appending gene:Serpinb11 with 415 probes\n",
      "-- appending gene:Sftpa1 with 414 probes\n",
      "-- appending gene:Sftpd with 406 probes\n",
      "-- appending gene:Slpi with 424 probes\n",
      "-- appending gene:Sox8 with 464 probes\n",
      "-- appending gene:Spag8 with 466 probes\n",
      "-- appending gene:Sparc with 392 probes\n",
      "-- appending gene:Spib with 457 probes\n",
      "-- appending gene:Spp1 with 379 probes\n",
      "-- appending gene:Sprr1b with 338 probes\n",
      "-- appending gene:Sprr2i with 377 probes\n",
      "-- appending gene:Sst with 434 probes\n",
      "-- appending gene:Stxbp5l with 396 probes\n",
      "-- appending gene:Syt7 with 466 probes\n",
      "-- appending gene:Tac1 with 376 probes\n",
      "-- appending gene:Tac2 with 473 probes\n",
      "-- appending gene:Tacstd2 with 382 probes\n",
      "-- appending gene:Tff2 with 196 probes\n",
      "-- appending gene:Tff3 with 432 probes\n",
      "-- appending gene:Timp2 with 433 probes\n",
      "-- appending gene:Tmem45a with 329 probes\n",
      "-- appending gene:Tnfrsf11b with 386 probes\n",
      "-- appending gene:Trp63 with 382 probes\n",
      "-- appending gene:Trpm5 with 478 probes\n",
      "-- appending gene:Tspan1 with 415 probes\n",
      "-- appending gene:Ttc36 with 445 probes\n",
      "-- appending gene:Vpreb3 with 391 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending gene:Xcl1 with 409 probes\n",
      "-- appending gene:Zg16 with 347 probes\n"
     ]
    }
   ],
   "source": [
    "gene_names = np.sort(np.unique([_v['reg_name'].split('reg_')[1].split('_')[0].split(':')[0] for _s, _v in _pbde.kept_probes.items()]))\n",
    "for _gene_name in gene_names:\n",
    "    _sel_pbs = {_s:_v for _s, _v in _pbde.kept_probes.items() \n",
    "                if _v['reg_name'].split('reg_')[1].split('_')[0].split(':')[0]==_gene_name}\n",
    "\n",
    "    if len(_sel_pbs) > 0:\n",
    "        _region_gid = list(_sel_pbs.values())[0]['reg_name'].split('_')[-1]\n",
    "        cand_prob_nums.append(len(_sel_pbs))\n",
    "        if verbose:\n",
    "            print(f\"-- appending gene:{_gene_name} with {len(_sel_pbs)} probes\")\n",
    "        _splitted_pbde = ld.pb_reports_class()\n",
    "        _splitted_pbde.kept_probes = _sel_pbs\n",
    "        pb_dict[_gene_name] = _splitted_pbde"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "considerable-works",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Screen probes by genome counts.\n",
      "-- filtering region:Acpp, screen genome to keep 300 probes\n",
      "-- filtering region:Agt, screen genome to keep 300 probes\n",
      "-- filtering region:Aldh3a1, screen genome to keep 300 probes\n",
      "-- filtering region:Aldob, screen genome to keep 300 probes\n",
      "-- filtering region:Alox5ap, screen genome to keep 300 probes\n",
      "-- filtering region:Alpi, screen genome to keep 300 probes\n",
      "-- filtering region:Alpl, screen genome to keep 300 probes\n",
      "-- filtering region:Ang4, directly append 175 probes\n",
      "-- filtering region:Anxa8, screen genome to keep 300 probes\n",
      "-- filtering region:Apoa1, screen genome to keep 300 probes\n",
      "-- filtering region:Apoa2, screen genome to keep 300 probes\n",
      "-- filtering region:Apoc2, screen genome to keep 300 probes\n",
      "-- filtering region:Apod, screen genome to keep 300 probes\n",
      "-- filtering region:Ascl1, screen genome to keep 300 probes\n",
      "-- filtering region:Avil, screen genome to keep 300 probes\n",
      "-- filtering region:Bgn, screen genome to keep 300 probes\n",
      "-- filtering region:C1qb, screen genome to keep 300 probes\n",
      "-- filtering region:C3, screen genome to keep 300 probes\n",
      "-- filtering region:Calb2, screen genome to keep 300 probes\n",
      "-- filtering region:Cald1, screen genome to keep 300 probes\n",
      "-- filtering region:Camp, screen genome to keep 300 probes\n",
      "-- filtering region:Car4, screen genome to keep 300 probes\n",
      "-- filtering region:Car8, screen genome to keep 300 probes\n",
      "-- filtering region:Ccdc30, screen genome to keep 300 probes\n",
      "-- filtering region:Cck, screen genome to keep 300 probes\n",
      "-- filtering region:Ccl1, screen genome to keep 300 probes\n",
      "-- filtering region:Ccl2, screen genome to keep 300 probes\n",
      "-- filtering region:Ccl20, screen genome to keep 300 probes\n",
      "-- filtering region:Ccl21a, directly append 5 probes\n",
      "-- filtering region:Ccl6, directly append 200 probes\n",
      "-- filtering region:Ccl9, screen genome to keep 300 probes\n",
      "-- filtering region:Cd14, screen genome to keep 300 probes\n",
      "-- filtering region:Cebpb, screen genome to keep 300 probes\n",
      "-- filtering region:Celf3, directly append 26 probes\n",
      "-- filtering region:Cga, screen genome to keep 300 probes\n",
      "-- filtering region:Cilp, screen genome to keep 300 probes\n",
      "-- filtering region:Ckb, screen genome to keep 300 probes\n",
      "-- filtering region:Ckm, screen genome to keep 300 probes\n",
      "-- filtering region:Clps, screen genome to keep 300 probes\n",
      "-- filtering region:Cnfn, screen genome to keep 300 probes\n",
      "-- filtering region:Cnmd, screen genome to keep 300 probes\n",
      "-- filtering region:Col1a1, screen genome to keep 300 probes\n",
      "-- filtering region:Col4a1, screen genome to keep 300 probes\n",
      "-- filtering region:Col6a1, screen genome to keep 300 probes\n",
      "-- filtering region:Cpn1, screen genome to keep 300 probes\n",
      "-- filtering region:Crabp1, screen genome to keep 300 probes\n",
      "-- filtering region:Cryaa, screen genome to keep 300 probes\n",
      "-- filtering region:Ctrb1, screen genome to keep 300 probes\n",
      "-- filtering region:Cxcl13, screen genome to keep 300 probes\n",
      "-- filtering region:Cyp1a2, screen genome to keep 300 probes\n",
      "-- filtering region:Cyp2f2, screen genome to keep 300 probes\n",
      "-- filtering region:Dbn1, screen genome to keep 300 probes\n",
      "-- filtering region:Dclk1, screen genome to keep 300 probes\n",
      "-- filtering region:Dlk1, screen genome to keep 300 probes\n",
      "-- filtering region:Dppa3, directly append 279 probes\n",
      "-- filtering region:Ecm1, screen genome to keep 300 probes\n",
      "-- filtering region:Efna3, screen genome to keep 300 probes\n",
      "-- filtering region:Elovl4, screen genome to keep 300 probes\n",
      "-- filtering region:Fabp1, screen genome to keep 300 probes\n",
      "-- filtering region:Fabp6, screen genome to keep 300 probes\n",
      "-- filtering region:Fam183b, screen genome to keep 300 probes\n",
      "-- filtering region:Fam25c, screen genome to keep 300 probes\n",
      "-- filtering region:Fgg, screen genome to keep 300 probes\n",
      "-- filtering region:Foxa1, screen genome to keep 300 probes\n",
      "-- filtering region:Foxa2, screen genome to keep 300 probes\n",
      "-- filtering region:Foxa3, screen genome to keep 300 probes\n",
      "-- filtering region:Foxj1, screen genome to keep 300 probes\n",
      "-- filtering region:Fxyd3, screen genome to keep 300 probes\n",
      "-- filtering region:Fxyd6, screen genome to keep 300 probes\n",
      "-- filtering region:Gabrp, screen genome to keep 300 probes\n",
      "-- filtering region:Gal, screen genome to keep 300 probes\n",
      "-- filtering region:Gatm, screen genome to keep 300 probes\n",
      "-- filtering region:Ghrl, screen genome to keep 300 probes\n",
      "-- filtering region:Gm13539, directly append 228 probes\n",
      "-- filtering region:Gm2a, screen genome to keep 300 probes\n",
      "-- filtering region:Gm94, screen genome to keep 300 probes\n",
      "-- filtering region:Gp2, screen genome to keep 300 probes\n",
      "-- filtering region:Gpx3, screen genome to keep 300 probes\n",
      "-- filtering region:Gpx6, screen genome to keep 300 probes\n",
      "-- filtering region:Grhl1, screen genome to keep 300 probes\n",
      "-- filtering region:Gsdma, screen genome to keep 300 probes\n",
      "-- filtering region:Gsta2, directly append 274 probes\n",
      "-- filtering region:Gsta4, screen genome to keep 300 probes\n",
      "-- filtering region:Gstm3, screen genome to keep 300 probes\n",
      "-- filtering region:Gsto1, screen genome to keep 300 probes\n",
      "-- filtering region:Guca2a, screen genome to keep 300 probes\n",
      "-- filtering region:H19, screen genome to keep 300 probes\n",
      "-- filtering region:Hamp2, directly append 238 probes\n",
      "-- filtering region:Hba-a2, screen genome to keep 300 probes\n",
      "-- filtering region:Hbb-y, screen genome to keep 300 probes\n",
      "-- filtering region:Hck, screen genome to keep 300 probes\n",
      "-- filtering region:Hnf4a, screen genome to keep 300 probes\n",
      "-- filtering region:Hnf4g, screen genome to keep 300 probes\n",
      "-- filtering region:Hp, screen genome to keep 300 probes\n",
      "-- filtering region:Iapp, screen genome to keep 300 probes\n",
      "-- filtering region:Idi2, directly append 245 probes\n",
      "-- filtering region:Ifi27l2b, directly append 193 probes\n",
      "-- filtering region:Igf1, screen genome to keep 300 probes\n",
      "-- filtering region:Igf2, screen genome to keep 300 probes\n",
      "-- filtering region:Igfbp5, screen genome to keep 300 probes\n",
      "-- filtering region:Igfbp6, screen genome to keep 300 probes\n",
      "-- filtering region:Ins2, screen genome to keep 300 probes\n",
      "-- filtering region:Isg15, screen genome to keep 300 probes\n",
      "-- filtering region:Itga6, screen genome to keep 300 probes\n",
      "-- filtering region:Jpt1, screen genome to keep 300 probes\n",
      "-- filtering region:Klk14, screen genome to keep 300 probes\n",
      "-- filtering region:Klk7, screen genome to keep 300 probes\n",
      "-- filtering region:Krt10, screen genome to keep 300 probes\n",
      "-- filtering region:Krt13, screen genome to keep 300 probes\n",
      "-- filtering region:Krt14, directly append 195 probes\n",
      "-- filtering region:Krt16, directly append 109 probes\n",
      "-- filtering region:Krt17, screen genome to keep 300 probes\n",
      "-- filtering region:Krt19, screen genome to keep 300 probes\n",
      "-- filtering region:Krt5, screen genome to keep 300 probes\n",
      "-- filtering region:Krt71, screen genome to keep 300 probes\n",
      "-- filtering region:Krt76, screen genome to keep 300 probes\n",
      "-- filtering region:Krt77, screen genome to keep 300 probes\n",
      "-- filtering region:Krt9, screen genome to keep 300 probes\n",
      "-- filtering region:Krtdap, screen genome to keep 300 probes\n",
      "-- filtering region:Lcn2, screen genome to keep 300 probes\n",
      "-- filtering region:Lcn3, screen genome to keep 300 probes\n",
      "-- filtering region:Lgals2, screen genome to keep 300 probes\n",
      "-- filtering region:Lgals3, screen genome to keep 300 probes\n",
      "-- filtering region:Lifr, screen genome to keep 300 probes\n",
      "-- filtering region:Ltf, screen genome to keep 300 probes\n",
      "-- filtering region:Ly6c1, directly append 126 probes\n",
      "-- filtering region:Ly6c2, directly append 147 probes\n",
      "-- filtering region:Ly6d, screen genome to keep 300 probes\n",
      "-- filtering region:Ly6g6f, screen genome to keep 300 probes\n",
      "-- filtering region:Lypd2, screen genome to keep 300 probes\n",
      "-- filtering region:Lypd3, screen genome to keep 300 probes\n",
      "-- filtering region:Mal, screen genome to keep 300 probes\n",
      "-- filtering region:Meg3, screen genome to keep 300 probes\n",
      "-- filtering region:Meig1, screen genome to keep 300 probes\n",
      "-- filtering region:Mgp, screen genome to keep 300 probes\n",
      "-- filtering region:Mlph, screen genome to keep 300 probes\n",
      "-- filtering region:Mpo, screen genome to keep 300 probes\n",
      "-- filtering region:Mt3, directly append 187 probes\n",
      "-- filtering region:Mt4, screen genome to keep 300 probes\n",
      "-- filtering region:Muc13, screen genome to keep 300 probes\n",
      "-- filtering region:Mymx, screen genome to keep 300 probes\n",
      "-- filtering region:Myog, screen genome to keep 300 probes\n",
      "-- filtering region:Nefh, screen genome to keep 300 probes\n",
      "-- filtering region:Ngp, screen genome to keep 300 probes\n",
      "-- filtering region:Nppa, screen genome to keep 300 probes\n",
      "-- filtering region:Npy, screen genome to keep 300 probes\n",
      "-- filtering region:Nrgn, screen genome to keep 300 probes\n",
      "-- filtering region:Nts, screen genome to keep 300 probes\n",
      "-- filtering region:Oas1d, directly append 261 probes\n",
      "-- filtering region:Oas1h, screen genome to keep 300 probes\n",
      "-- filtering region:Odf3b, screen genome to keep 300 probes\n",
      "-- filtering region:Olfm1, screen genome to keep 300 probes\n",
      "-- filtering region:Pcp4, screen genome to keep 300 probes\n",
      "-- filtering region:Pdzk1ip1, screen genome to keep 300 probes\n",
      "-- filtering region:Penk, screen genome to keep 300 probes\n",
      "-- filtering region:Pgc, screen genome to keep 300 probes\n",
      "-- filtering region:Pigr, screen genome to keep 300 probes\n",
      "-- filtering region:Pik3r5, screen genome to keep 300 probes\n",
      "-- filtering region:Pla2g1b, screen genome to keep 300 probes\n",
      "-- filtering region:Plac8, screen genome to keep 300 probes\n",
      "-- filtering region:Plk2, screen genome to keep 300 probes\n",
      "-- filtering region:Plvap, screen genome to keep 300 probes\n",
      "-- filtering region:Pomc, screen genome to keep 300 probes\n",
      "-- filtering region:Pou2f3, screen genome to keep 300 probes\n",
      "-- filtering region:Psca, screen genome to keep 300 probes\n",
      "-- filtering region:Ptgds, screen genome to keep 300 probes\n",
      "-- filtering region:Pyy, screen genome to keep 300 probes\n",
      "-- filtering region:Rap1gap2, screen genome to keep 300 probes\n",
      "-- filtering region:Rarres1, screen genome to keep 300 probes\n",
      "-- filtering region:Reg3b, screen genome to keep 300 probes\n",
      "-- filtering region:Reg4, screen genome to keep 300 probes\n",
      "-- filtering region:Retnla, directly append 246 probes\n",
      "-- filtering region:Riiad1, screen genome to keep 300 probes\n",
      "-- filtering region:Rnase1, directly append 228 probes\n",
      "-- filtering region:Rnase4, screen genome to keep 300 probes\n",
      "-- filtering region:Rsad2, screen genome to keep 300 probes\n",
      "-- filtering region:Rsph1, screen genome to keep 300 probes\n",
      "-- filtering region:S100a9, screen genome to keep 300 probes\n",
      "-- filtering region:Saa1, directly append 135 probes\n",
      "-- filtering region:Saa2, screen genome to keep 300 probes\n",
      "-- filtering region:Serpinb11, screen genome to keep 300 probes\n",
      "-- filtering region:Sftpa1, screen genome to keep 300 probes\n",
      "-- filtering region:Sftpd, screen genome to keep 300 probes\n",
      "-- filtering region:Slpi, screen genome to keep 300 probes\n",
      "-- filtering region:Sox8, screen genome to keep 300 probes\n",
      "-- filtering region:Spag8, screen genome to keep 300 probes\n",
      "-- filtering region:Sparc, screen genome to keep 300 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:Spib, screen genome to keep 300 probes\n",
      "-- filtering region:Spp1, screen genome to keep 300 probes\n",
      "-- filtering region:Sprr1b, screen genome to keep 300 probes\n",
      "-- filtering region:Sprr2i, screen genome to keep 300 probes\n",
      "-- filtering region:Sst, screen genome to keep 300 probes\n",
      "-- filtering region:Stxbp5l, screen genome to keep 300 probes\n",
      "-- filtering region:Syt7, screen genome to keep 300 probes\n",
      "-- filtering region:Tac1, screen genome to keep 300 probes\n",
      "-- filtering region:Tac2, screen genome to keep 300 probes\n",
      "-- filtering region:Tacstd2, screen genome to keep 300 probes\n",
      "-- filtering region:Tff2, directly append 196 probes\n",
      "-- filtering region:Tff3, screen genome to keep 300 probes\n",
      "-- filtering region:Timp2, screen genome to keep 300 probes\n",
      "-- filtering region:Tmem45a, screen genome to keep 300 probes\n",
      "-- filtering region:Tnfrsf11b, screen genome to keep 300 probes\n",
      "-- filtering region:Trp63, screen genome to keep 300 probes\n",
      "-- filtering region:Trpm5, screen genome to keep 300 probes\n",
      "-- filtering region:Tspan1, screen genome to keep 300 probes\n",
      "-- filtering region:Ttc36, screen genome to keep 300 probes\n",
      "-- filtering region:Vpreb3, screen genome to keep 300 probes\n",
      "-- filtering region:Xcl1, screen genome to keep 300 probes\n",
      "-- filtering region:Zg16, screen genome to keep 300 probes\n",
      "region: Ccl21a removed with 5 probes\n",
      "region: Celf3 removed with 26 probes\n"
     ]
    }
   ],
   "source": [
    "# filter by genome counts\n",
    "reload(library_tools.quality_check)\n",
    "\n",
    "num_probes_kept = 300 # max num of probes kept for each region\n",
    "filtered_pb_dict = library_tools.quality_check.Screen_probe_by_hit(library_folder, pb_dict, num_probes_kept,\n",
    "                                                                  hit_type='genome')\n",
    "\n",
    "## filter by minimum number of probes in each region\n",
    "min_num_probes = 100\n",
    "removed_regions = []\n",
    "for _rid, _pb_class in filtered_pb_dict.items():\n",
    "    if len(_pb_class.kept_probes) < min_num_probes:\n",
    "        print(f\"region: {_rid} removed with {len(_pb_class.kept_probes)} probes\")\n",
    "        removed_regions.append(_rid)\n",
    "for _rid in removed_regions:\n",
    "    del(filtered_pb_dict[_rid])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "black-fitting",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox â‰¥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_dpi_ratio', { dpi_ratio: fig.ratio });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    var cursor = msg['cursor'];\n",
       "    switch (cursor) {\n",
       "        case 0:\n",
       "            cursor = 'pointer';\n",
       "            break;\n",
       "        case 1:\n",
       "            cursor = 'default';\n",
       "            break;\n",
       "        case 2:\n",
       "            cursor = 'crosshair';\n",
       "            break;\n",
       "        case 3:\n",
       "            cursor = 'move';\n",
       "            break;\n",
       "    }\n",
       "    fig.rubberband_canvas.style.cursor = cursor;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            /* FIXME: We get \"Resource interpreted as Image but\n",
       "             * transferred with MIME type text/plain:\" errors on\n",
       "             * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "             * to be part of the websocket stream */\n",
       "            evt.data.type = 'image/png';\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                evt.data\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from http://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * http://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.which === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.which;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.which !== 17) {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    if (event.altKey && event.which !== 18) {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    if (event.shiftKey && event.which !== 16) {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k';\n",
       "    value += event.which.toString();\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(msg['content']['data']);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    var manager = IPython.notebook.keyboard_manager;\n",
       "    if (!manager) {\n",
       "        manager = IPython.keyboard_manager;\n",
       "    }\n",
       "\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAApQAAAFKCAYAAACjE8/+AAAgAElEQVR4XuxdBXiURxN+cXd3CCEESXAIAYIEhxaKO6W0FCiFFmtxhwIt0BYtFGixFocCwV1D0ECIIEGDBnf7n9ljL1+OhJx8d9njn3me//lLbnd29/32buebnXknwdu3b9+ChRFgBBgBRoARYAQYAUaAEbASgQRsUFqJHHdjBBgBRoARYAQYAUaAERAIsEHJG4ERYAQYAUaAEWAEGAFGwCYE2KC0CT7uzAgwAowAI8AIMAKMACPABiXvAUaAEWAEGAFGgBFgBBgBmxBgg9Im+LgzI8AIMAKMACPACDACjAAblLwHGAFGgBFgBBgBRoARYARsQoANSpvg486MACPACDACjAAjwAgwAmxQ8h5gBBgBRoARYAQYAUaAEbAJATYobYKPOzMCjAAjwAgwAowAI8AIsEHJe4ARYAQYAUaAEWAEGAFGwCYE2KC0CT7uzAgwAowAI8AIMAKMACPABiXvAUaAEWAEGAFGgBFgBBgBmxBgg9Im+LgzI8AIMAKMACPACDACjAAblLwHGAFGgBFgBBgBRoARYARsQoANSpvg486MACPACDACjAAjwAgwAmxQ8h5gBBgBRoARYAQYAUaAEbAJATYobYKPOzMCjAAjwAgwAowAI8AIsEHJe4ARYAQYAUaAEWAEGAFGwCYE2KC0CT7uzAgwAowAI8AIMAKMACPABiXvAUaAEWAEGAFGgBFgBBgBmxBgg9Im+LgzI8AIMAKMACPACDACjAAblLwHGAFGgBFgBBgBRoARYARsQoANSpvg486MACPACDACjAAjwAgwAmxQ8h5gBBgBRoARYAQYAUaAEbAJATYobYKPOzMCjAAjwAgwAowAI8AIsEHJe4ARYAQYAUaAEWAEGAFGwCYE2KC0CT7uzAgwAowAI8AIMAKMACPABiXvAUaAEWAEGAFGgBFgBBgBmxBgg9Im+LgzI8AIMAKMACPACDACjAAblLwHGAFGgBFgBBgBRoARYARsQoANSpvg486MACPACDACjAAjwAgwAmxQ8h5gBBgBRoARYAQYAUaAEbAJATYobYKPOzMCjAAjwAgwAowAI8AIsEHJe4ARYAQYAUaAEWAEGAFGwCYE2KC0CT7uzAgwAowAI8AIMAKMACPABiXvAUaAEWAEGAFGgBFgBBgBmxBgg9Im+LgzI8AIMAKMACPACDACjAAblLwHGAFGgBFgBBgBRoARYARsQoANSpvgi+r87NkzBAYGIkuWLEicOLFOWlkNI8AIMAKMACPACKiKwKtXr3Dr1i14eHggefLkqk7TIfNig1InmA8fPozy5cvrpI3VMAKMACPACDACjICzIODv749y5co5y3TtMk82KHWCNTw8HAUKFABtqhw5cuikldUwAowAI8AIMAKMgKoIRERECGfShQsXkD9/flWn6ZB5sUGpE8xXrlxBnjx5cPnyZeTOnVsnrayGEWAEGAFGgBFgBFRFgM/+qCfDBqVOu5Q3lU5AshpGgBFgBBgBRsBJEOCznw1K3bcqbyrdIWWFjAAjwAgwAoyA0gjw2c8Gpe4blDeV7pCyQkaAEWAEGAFGQGkE+Oxng1L3DcqbSndIWSEjwAgwAowAI6A0Anz2s0Gp+wblTaU7pKyQEWAEGAFGgBFQGgE++9mg1H2D8qbSHVJWyAgwAowAI8AIKI0An/1sUOq+QXlT6Q4pK2QEGAFGgBFgBJRGgM9+Nih136C8qXSHlBUyAowAI8AIMAJKI8BnPxuUum9Q3lS6Q8oKGQFGgBFgBBgBpRHgs58NSt03KG8q3SFlhYwAI8AIMAKMgNII8NnPBqXuG5Q3le6QskJGgBFgBBgBRkBpBPjsZ4NS9w3Km0p3SFkhI8AIMAKMACOgNAJ89rNBqfsG5U2lO6SskBFgBBgBRoARUBoBPvvZoNR9g/Km0h1SVsgIMAKMACPACCiNAJ/9H5FBefPmTfTp0wcbN27E06dPUbVqVUycOBFFixYVqzx+/Dh69eqFgIAAZMqUCT179kTfvn2NCLx58wYjRozAnDlzcPfuXVSuXBnTp0+Hq6urRZuYN5VFcHFjRoARYAQYAUbA6RHgs/8jMii9vLyQMGFC/Pbbb0iVKhWGDBmC/fv34+zZs8LAdHd3R+PGjdG7d28cPHgQ3bt3FwZjp06dBApkTNK/582bh1y5cqF///44f/48Tp8+jaRJk5q92XlTmQ0VN2QEGAFGgBFgBD4KBPjs/0gMyjt37qBHjx4YPHgwihUrJlZ14sQJlCxZEv7+/ti6dSumTZuG8PBwJE6cWHw+cOBArFy5EsHBwXjx4gUyZ86MCRMmoGvXruLze/fuIWfOnJg7dy5atWpl9obnTWU2VNyQEWAEGAFGgBH4KBDgs/8jMShNd+ONGzfEdfbOnTuFwdi8eXOkT58eixcvNjYlI7NWrVqgtmRoVqhQASEhIXBzczO2oWtvT09P4bk0V3hTmYsUt2MEGAFGgBFgBD4OBPjs/wgNyi5dumD27NlIliwZ1q5di9q1awujsF69ehg/frxxxUFBQcKbefjwYVy6dAlNmzbFkydPkCJFCmObFi1aiL+tW7fO7B1vr00VfP0B9obdNnse2oZZ0iRDfY8cSJIoId6+fYtNp2/gyt0nVulydKdECROgdrHsyJXe8FzO33qEHSG3xDqcXbwLZkbRnGnFMm4+fAa/wOt4+fqNcstKkCABahbJinyZUom5XY58gi1BN/DGimdQ1S0LCmVLI/RE3H+KTaeu49WbDz/LDCmTooFnDiRPkkj02xF8E+duPbIYp7QpkqChZw6kTGq4pZBCukinvcUjVzpUcMkUbZg3b95ic9B1XLn71N7Dv6ffJUsq1HDP9t7f94TdQsj1h7rOJ2f6FKhbLDsSJkyA12/ewu9UBK7ff6brGEVypEUl18xC55MXr7DuRAQePHup6xiOVFYqbwaUyZdBDHn/6UvsDLmJ6u5ZkTZ5EvG3gPBIHL98L8Yp0XfW1z0r8mc2fGfp956+s4S9apI4YQLULZ4D2dMlF1M7e/Mhdobcsts06fvfpkJe3fXb6+zXfaIOUJjg7cdwQgMgQ5FiJmfMmCE8knv37gUZhm3atMHIkSONUFJ8ZMGCBbFnzx7hoWzfvj1ev34t4jCldOjQAdeuXRNX5rHJgwcPQP+TEhERgfLly+Py5cvInTu3bo9uif8lDFgZaLW+31uXwiclcuLQ+Tto+cdBq/XER0cvl4z4p0tFMbTvLztx7tbj+JiG7mOmT5kEhwfVFIb+V38HiB98VYUOa79eVcT0Gk/bF+tBFtf8c6ZLjn0/1gAdeG3nHMS+s3fi6iI+H9moGDpUzI8zEQ9Q79c9ZvWJqVG/OoXxTfXoiXZVJ+7AxTv2f8Gil6ODA3xBL3hS9p+9jTZzDlm9Hls7ru9ZGcVypjOqCb/9GNV+3mmr2hj7//1Fefi4ZcGm09fx9YIjuo+RIAGwu1915MmYEr9tC8OkLaG6j+FIhUkTJxS/D+lSJMHwtacxf384uvi4YGD9Irj7+AXKj92Kl69jNxALZ0uDTd/7iCk3n7kfh8PvOnL6Fo1VrXAWzO9UXvSp9NN2XL1nvxesbGmT4dDAmhbNz5zGbFBGofTRGJRySZS17eHhIa6yyQtZv379GD2UR48eFck3zZo1i9FD+fz5c6xZsybW/TR8+HCR0GMqehuU605ew6TNlv9AXr77RPzoDKzvji4+BbH62FV89+9x0I9vgXceJ3O+LPHRht7K7zx+gTwZU2BP/xpiCu5D/PDs5RtkTZMMqZNF9zTFxxytGfPlmze4HGn4wdzZt5rwIlQctw0R958hc+pkSJtcnXU9fP4Ktx4+R5pkiRE4oo6Yc9nRW3D70QuL5vr81RvjIXFiWG1xSJYZtUU8XzKwSH9Mcu3+U/G8v6pSAIMaFI1mjLi8876Y8wxuPHiGxy9eo1W5PPipqWe0Lq4DNwgvKR00qUy8l+bojqsN6b4UaTBY131bGcVzRRlwK45cQZ9lJ5AwAZDfgd/H87cNL2V/diwL3yJRXsr9526jzWyDgZs/U0okpB8KG+Vi5BPhGfupiQdalc+LefsuYMR/QSDPVN6MKW3UDpBJdeHdehZ2roDKhTLju3+OYfXxa+I3gn4rnEm0vw9bvvcRHv1O8/zFzUx9j+yY3rZMtBcrwpCwlCK/s/Q7cnK44TsrjbRMqZKK754qcu/pS0Q+foFCWVNjS++q4uapwIANYnr2+j5mSp0Uy7p66w4BG5RRkDq1QUmUQdu2bROeyESJDNdiJBQ7Sd5D+htRBS1YsMD4WUwxlJQRTl5LKRRDWaJECZHQE5s4ykNp7e4nbw55daRnZmnAZfRffhKZUydFwOBa1qp1SL9Fhy5i0KpT4oeF3ijpx8Zl4AbQTevsDmVRq+j713UOmZiNg9B1XNGhm4SW+Z3KwcslE9yHbBT/nvd5OXGtpYpsPHUdXRceEQfW2bH1xbSKDd0ojLNfW5VEo5K5zJrqxTuPUXWiwfO18bsqwniSayYMqhWOec1fzD+M7cE3xRXV2M88sOrYFXz/7wmkSZ4Yge8OS3Mm8P2/x7Hq2FVxdT6tTWljF7pypj0lsO9UDtVjmYc5Y8TWhg7M0qO2iI9XdvdG6byGa0ySpYcvo/8Kx38fpRE9q30Z1CmW3Tif3aG30GGuv/h36Oh6IC+ZrSKNGellnrXrHMb5BcM1a2ps7V3VVvWiv3zRlDcxHef6Y1foLbT3yodRjYvrMoajlGj3y6ru3qCr7xYzD8A/PBLVC2fBvE7lcfTSXTSZvl9M6diQWsiQKoqJZO2Ja+i55BhSJk2EoJF1RRuvsdtw/cEzjPmsONpWyOeopcQ5zp97L2DUuiDky5QSu/pVx4tXb+A22E/0kx7tOJUo0oANyqgH4dQGJXFMlipVShiVNWoYPFkvX74UVEGffvopsmXLhpkzZ+LcuXNGg3PAgAFYtWqVSNohL2TWrFkxadIkdO7cWfT/WLK8G03dixNX7qOnbyH0ruWGhQcvYvDqU8iRLjkODPBV5KsY8zSWH7mCvstOgK6Gjw+tLeILCw1yzh8b0xWWH7MVNx8+F1e5FV0yodbk3aLJtj5VUTBLamWei9bACBtTTxiWBQduAIViWWLUP3/1GoUHvzOaO5UT3nF5tbr5ex+4vYurNF14j8VHse5kBBqXzIkprUoZ92/2tMlxcKD5+3fI6lNYcPAiKIbzry8MV2skz16+Nhq2i7+sAO93MXh6PoBHz1+h+DDDC8Q/XbzEC4SUxYcuYeCqQONLk57jfkhX4cF+IK8xGddkZEvZHnwDX8wPEP+8MK6+CE2wVar/vFN4EAc3KIIvq7jg921h+GVLKLRhFLaOIT38oxoVQ/uK+fHp1L04Sb97NVzRu3ZhW9U7tL/2u7Kgc3lUKZRFhHmQY0CG/2hDJYJH1TXGF9NE/QIj0G3RUSRNlBChY+qJuctbhQlNPdGiXB6HrudDg9F3kr6b8jz60HdFmUnHMhE2KKOAcWqDkjxXlHRDsZCUkJMhQwaMGTMGmzZtEoTmyZMnNxqXxC9JVELdunUTRmbHjh0FCoMGDcKsWbMETVD+/PnRr18/oS8wMNCpeSibzdiPgIt30a1aQfxQ19143UTXJLv7V1f6O2r6pu3MPzamQEuPwxeVCqBiwUwihpJurc6MqotkiaO87PH9gA6HR6L5zANiGnRVnSxxQqMBJq8XzZ2jPNTI05g/c0rj1Wrg8NpI8y7RwFTXjytO4p/Dl4U3mgzYP3afw9gNwSiYJRW29alm7tAYvzEYM3aeQ9l8GbC8W9R1F4VVlBixWehZ3rUiyubPaLZOcxt+yOuy4EA4hqw5LZLOKLbUUUIGLn2fTL3MMr6R4j3PvfNI2zqnOpN3I+TGQ/SvWxjdq7li0uYQ/Lb9LErkSY8131SyVb3oLw2u72u6oVfNQqgyYbsIKxnasCi+qFxAlzEcqcRtkB9evH6DGW1Lo55HDuN6SuROhzU9KmNr0A18+XeACF06Pza64b/59HV0WXBE/J6cH9dATLvkyM249+QlfmleAk3L6Bfbbysm0kNPV/FHhtTCnUfPUWa0IWfB1Jtv61j27s8G5UdiUNIy7t+/D/I6rl69WngXq1SpIjyOkpeS4iipOs6xY8eQI0cOUVWHuCulUEIOcVMSsTkl9fj4+IirbjIuLRHVNlXrPw7iwPk7+LJyAQxuWNTqA9kSDPRqa3q4xXQVpNdYjtbTb9kJLDtyRWRikkE5ev0Z5M6QAnt/cJxRYc6aT129j4a/7xVNDw30FQZlyZGG69sV3SqiTD7zDbBPft+LwKv38W0NV3HlTbGDFON26l1sZkzzGflfEObuu4BKrpmw6EsvTN4Sil+3hcEzdzqs7VHZnCWINtN2nMXETSFwz54GG78zJCqQUHxouTGGA2xtj0rwzJ3ebJ3mNtTGhc3pUBY1NaEa8/ddwPD/gkQsoSNf8MiIJmN6UosSaFI6ysBYfzIC3yw+Kp5zyGiDd8tWafj7Hpy6+gDf1SyE72q6YZzfGczadR7l8mfQLZatzeyD2H/uDj73zo/hnxaDx7BNoFjCyS1L4LNS6hhQ5mJJIRL0ezehmSdalM2DUiM34+6Tl3DLlhqbv6+K/05cw7cm19pSNzEWdJp/WPyTjE3KrI/tBcLc+dirnYzpl78DxPxQcdx2MZxpvLG95qCXXtXOfr3WZY0ep/ZQWrNge/VRbVNRPBRdW3asmA8jGhWP9WC1Fx626N0RchOd5hl+GM+OqYdbj54bf2xMs1NtGSc++k7dHoafN4eKODK68qarn8qumbHwywrxMZ1YxyRKHd9fdonPKYGIYuq8fzL84G/oWcVIe2TOpGUme9PSuUGUNWTgyWD82Pr/sjkEv28/i5J50mP1N5UwZn0QZu+5YLz6M2dcavPX/nAMW/u+J5CySSnGj4RiO92zG2ic9BaKCyNPpekV85w958XLBCUYbe9rvsfV1vnJhCjTK9A1x6+i1z/H4zT0LRm/yfR9OHrpHrpXK4j+dd1h+pJgia7Y2pIRTMYwhUZMbF7CGBrzofhcPca1lw6fCTtEIhd5WDtVyi/iCim5Ur50fij2NqY4WBljarr/7DV/c/WaXs9rY6239vaBa1YDxZgziGpnf3xixgalTuirtqm+/Oswtp65idbl82JcEw+rPTw6wWORGm3GadDIOsKbJBM7KJifjDFnFelhoDincgUyCPqcthXyYsxnHkot6dq9p9EMyGRJEhoNzB19q6GABZnWw9acwl8HLsK7YCbRb9GhS4JGhoLvYxO6pqbraumZIeosotAiz+6fn5czGyvTeFzZkWL7KMaPZHufqnCxU/xqbB4zmaASl2Ft9kLNbFhh7FbcePBcJDppOflk1rmMWzZT3QebtfrjAA6ejzRm6g9eHYiFBy8ZE0z0GEPqpBjZic09UX7MNqGWrtTpat3ZpP6vexAU8UDEvRNVkExgk8mU8gUpplsN099N4l2MLQkrvnHZduYGOv8VFbN79uYjYzw5UUDlzWQ7C4Cj1qja2e+odcc0DhuUOqGv2qbqtvAI/E5dR7MyufFz8xKYsDEY03eeE4S5KzSxZDotX1c1Ry5GoukMQ/ze8aG1RBJL7XfJK3v6G/jmnFUCr9zHJ1MNV8kpkiTC05evjdROKq3p3pMXmitub3EVqr0Cz5bWQEZsjszcdQ4/+QULbxxRJVH2dkw0Plpdfx8Ix1BNjGGvf45hzfFrglOVMnrNFW22OiUXyWQTIvCuM8WQELX3h+rIncE+e0rGj45v6oGW5aJIlWO7ijd3Xda2k5nXMolF6vn38CX8sCJQUEIFDNaHq6/9n4ewJ+y28Zak//ITWBpwBbWLZsMfHcpau4Ro/aQnm2IMyUMpfyeczSiRi2ox6wD8L0QKY5L+V/ZdXKHM3DZ90dKCQf2oP8lJik9OlthIxWNKE6UL+DYoIRL99n8aWAVCRtdF2I1Hxt8X4myVZOc2DOGwrqqd/Q5beAwDsUGpE/qqbSqij6DklkYlc+LXVqWsvjLUCR6L1JjG75GHUhoz/gN9kdUCY8aigR3QWJsMIoczpXBxwDTiHEKbBU1JOOSh1CbpWMJpJ69TkydJKGIog68/hEyiiG0i0rOYIWUSHBtaG1/+FYCtZ27EaYia6tt39jbaviMQ12bFag17/0G+yJrGfAM5TvA0DbzHbcO1+89gasBJAu7iudJi3bcG4nhHiPZKVZu0Yg8WiKhbkjwY18QTksKJqhZN1VA42bLuuXsvYOS6IMFZ+3OzEsbiDWRQycoytuh3dN/O8w9jW7DhZulrHxcjI4JMwpm8NUyQt8skHe38tJRCR4fUEry2ru/YMYjhgLy4qoi20AYl54XeeISmMwx0SDT3jBo6JFXmHNs8VDv74xMvNih1Ql+1TdVn6QmsOHrFSIgrKy5UKZQZCzqrFa9n+ghCbzyM5pGk8oRaj2X6lFHcazo9PoeqkYH2ctBN3/mgcHa1YoZMuT/JQyl5CsnTR1V+zBVtxjhd9VMWq0w6iE2HMcYqcULBiyiTLyg7fugnRc0dWlT2oQo/JOR5Iw8cidYLfmJobaRLaR/S52oTdyD8zhMjdY6cOFVzic0wMHtxVjSs8ctOnL/1GIPqF8FXPi5GDTJJSFtMwAr10bp0X3QEGwKvg2Jnf2lRAt8sOor1gRFoUjoXJrUoaat60d/IT5ossbjy7rrwqKC40nqjdRnIQUq0nngyKOWLNA1PL0S0b/7YfR4VCmTEv18bqohJMX1JIoNaXpnbixrLWlhMv5fkoWw921DJ7fSIOkjlRMUrVDv7rX0mevRjg1IPFKlm6pUryJMnj+6lF62d3oCVJ7HE/zJqFsmGOR3LCs474r6zNAbN2vFt6WcaoE1X3rKKx5mRdZEiqTr0Otas07SEoaprkkTmv7UuJbjtTInOzV071ROuPH5HtOaSZy82HUROTSTVJGQcNJt5ACcu3xOZ4n0s4Bek2Kyak6KSi2SNY228mT3xrz15l/C+SOocud6Jm4Ixbcc5lM6bHiu760OhY87zkPMhKjGiFJMik4QoxpViZPUQ0zAFa73MH5qLNoGPiMyJ21DPa3s9cLBEx6BVgSLGmIjMqcKZNLJIBxGZ/7IlJNY41KBrD1D/N0N50v0/1gDVsJc8qP928Xqvnrwl89K7relcw24+ivZ9t+SFVe+5WapPtbPf0vnr2Z4NSp3QVG1TDV1zCn8fiCJ0lnQ1dYtlx8z2ZXRatX3UaCkkKKubDEqZ9U0cecSV58wiy8PRGuxVX1YPfLTxf8SRSaU7taUYzR2DiOkpW5UqHUmJK7kqIDxSGJEk5EFsNnM/6NAxNYTimgOVXqww1pCooaUj2RlyE5+/YxKw556S1Dm9fAvh+1puxulKCp3y+TNiadfonqa41mTL55K3sU8tN3zrW8ioSsbm6ZkkZPqbI6vYdKiYDyMb6VPFRuvpIu81UU3puQZbsLamr3ZfUAwlcU5KISPxl82h0W6etGOE3XhoTGyhWHOqKhVF9eUt4udVEdMXPfpuExtETPyaqsw5tnmodvbHJ15sUOqEvmqbispaUXkryqxd/JWXscatpUkNOsFjkRrTknUUQ/n1guhlAC1SqFhjed1J0ypfICOWmlxdqTJdSRI9/JOiSJo4kfByU31k/0GWJ23I7GK5NsrepyzU2MTUg0Hxm0T1YxqLGBdWsZHiSxJoPYm8Y5qLpM6RBQZkm9HrgjBn7wVBHbWki1dcy9Dtc1lJxtTAtUcVG+ltq+GeFXM/LwdTblw9FnXpzhP4TDR4v6sVzoKdIbfgaCNdj3VIHdrnQFfe9BInhV7CJm0JiRZGoB37/K1HqPGO6ouYC8hDKZN6VMt6vxz5BFUmGJ4bVcyiMKcei4/pyoOq53P5kC7Vzn5HrTumcdig1Al91TYVZdVSdq0kEbZH/JJO0L2nxtQIIA+laY1ae43tCL0rj15B76UnxFAtyubGhGYlHDGsxWNor2vJQ6mtvWupMu01PyX0UPWdD4lp2AN5K62p+EE1uwsOMtSB15KLrzt5TRxglGlPVYrsJdKI6ly5AIY0jIr9jK+Y5s+m78OxS/fQo7or+taJKk0oX3IsJY7/EG5yjZJn1bR6lx6YP3j2Ep7DDRWPqC70xTtPdM0i12OOluiYt+8CRvxnSDKiK2+6wpdCHnbKat8RcgvtvPJidOPoVGOmRhrFUHqNM3jnVePvNb05IIOSfhPJqxo4vI4lkMV7W9XO/vgEhA1KndBXbVOZljmT5NJx0bXoBIdNarS1uyk7kTyUVNtbZvzapFyBzkcu3jVmNParUxjfVHdVYFbvT8FYD76GqyA2J0J204oz5k5cJmhQe3N03H70PJp3hTyUlMwzs11p1C0eVYPanPElF6S23KA06s0xbs0ZI7Y28pq3vVc+UIyfFFljnLxq8zvFzsdpy9gx9W0+cz8Oh9/F11VdMKBeEWMTSSumZ0znuA1nMGv3eaMXPjbvqC1rpOSxQoP88OrNW1FykGrNO8NvXGxrXhZwGf2WnxS/dWRQEherFLrJIIPy0IUobk+tHlPu2LQpEhtjl1VL/ItOS1ZR0Ab9uFJf2ipb9pUlfVU7+y2Zu95t2aDUCVHVNpW8OimaIy029KqCz+f5i+sg04NNp+XrqsY0w5iyvAetOqV0vKElAGjr1qpWwUK7Di0xNRmUlERSKm96rLIiiURWSSH98gr0Q5g9ffEaRYZuFE3opUIm6BAZOpGiWyJeY7fh+oNnGN24ONp55RNd//G/JA6wLGmS4bAVV/jmji9f5FqWzYPxzTyN3SRRe80iWTGno/lE7eaOG1u72K6dx244I7KH9QzB+HlTCKbuiKp2VHfKbkEZpfdLlIz1lWvuWrUgfqznbitU8dJ/46kIkameJFECfFXFRXAHS6HvATkKTly5j56+hQT5uVbod1ISu1M5UfJQVntH3r+tT1UUtBN5vzVAPewe4X8AACAASURBVHnxCkWHbhJdF39VARRTqeWdtUZnfPVR7eyPLxxoXDYodUJftU0lyaRlgHrbOQdFVRZLaVd0gsdiNbJk2NQ2pYSHkq6BHF332OJJm9mBDGbKmKbDlWKbVKVB6jTPX1yvUSUfuvLW1tY2c6nGZjKLmP5gTmUg7UsFEfOTh5rE0jri1IeyvOnAIiODjA0SU+J0S9djbntZGvCzUrkwuWUUVY4k+a5TLBtmtdeH5NucOUmycVn7WvYZ8d9pzNsXbqydbo6uuNpIrk35UhsbZVFceuL6vNakXSJhS8qAeu74+t1zjquvap/vDbuNdn8eEtMiT+s/hy8bpzizXRkRQ0msATElp5nGnpNBKRkOdvWrhnyZUimz3Fev3xg5MqlMJnkox2w4Iypp6cUy4KjFqnb2O2rdMY3DBqVO6Ku2qSghh2Le8mdKiZ39qqPFzAPwD49876pLp+XrrsZz+CY8ePYKk1qUEFneFBNKJRcpMP1jETKaZOUWFdekjbslHkoDDZV1HjWqt0zGFUnf2m7oUSMqwzi2tRPlCcXT0uEpr/78elVBkRyW1d2W8ZtayiFH1dLu/e9xrDx2FQ08cmBa29LGpfZeehwrj77/d3vvg9huKkxZIfSYh+lLbeXx23Hl7lOM+LQYOnrn12MIoUP+tkmFpnXKdRvIAYq0WetERE70WVImtywhsrxjw1AbT0rX43TlXXeKgUZo3481kCt9CgeswPwhXAasFyEKf7QvI14IJm4KQeFsabDpex/zlSjQUrWzPz4hYYNSJ/RV21QLDoRjiKZ0XUyHqk5Lt4uacmO2Cs8k1SGn/6ekgWI502J9T8dVFbHLwpxIqSTHr1c8u8i+XH38GqytcqKt4vFL8xJoWiZ3nEiUH7NVvExIOhjqYE1JvXZzDmHv2dvQeuUcVfrwxxUnhZfJ1BCXHI2flsgJ4vl0lJhWr5Hj2uMKXr7UUrLMrn7VIZ+naR1xW9fe5e8AbA66YVQzu0NZ1CqazVa18dJfS6dDxlXIjYfGeRBu5KG8/egFYjKatWEidI1MHkqVK4wVGbJRlJ6lWyjyupJHW8+kMEc9QNXOfketO6Zx2KDUCX3VNpVpjFiD3/bg9LUHIu6G4m9UF1lzmLwZFBtkS/ye6mtVdX7axBEiNqdD29qsdG1W55KvvFCxYKY4l1395524cPuxqN/934lror222k2cCt416LrgCDaejqprT3+evCUUvzrgAJOeP9MKVT0WH8W6kxFoUioXJmmuws1dk7XtJBbNy+QWta+l2IOndsHBiyJLOUe65DgwwBeyQhSFMDQz44XC3DX+sPwk/g2Iuhpe3rUiyubPaG53pdppvydUOpFuaaQMblBEvFg/efFa1LOn74VWXrwy8L2SUKwx0QbJKlFHBtdEpndVolRZcIkRm0GlaOkFkzyU5NEumy8DlnfzVmWKZs1DtbPfrEnbqREblDoBq9qmWnHkCvosO4H0KZPg+NDaiK1Chk7L112NjLcaWN9deChn77kAL5eM+KeL40igdV+UkymUiRpU5o2ScvaE3UbHivkwwkpSaqKRoUxUuvo1pxKGfAmi537wfKRAz5qqNlpP64x2BlJ/ukInMm97H2Bj1geJvWtaKq/bwiPwO3UdpoadvbeIjOk0NWTtUWd76eHL6L/iJDKnToqAwbWgrbxEnlm9RFKkSX1xkebrNa499Dx+/grFhhmSVUyFyOgnbQ19jwJLtiOKLJeBG8Q/531eTlDwGIsDDKsNYjRQSbS3UBRDSTHakjdZpXnGNRfVzv645mvPz9mg1Ald1TbV2hPXBHdjqqSJcHpkXUhvD73lflklqoavTsvXXY3MCKV4O7r2pKo/lN1Lb94sjkFAy01IHsqAi3dFUoujMmhlbFzBLKlw7tZjQQtDVW0sjTuNifPRlPjfXojKEoum2fEy+7t1+TwY1yQq+9te85B6Tcshyr9Lj2njkjkxpZU+V/DGOtvvuAXdBvlZTf30IVxm7TqHcX5R9DoqeuPMfa7aZDTTPsRlSmEEJIu+rIBKrpnfU6uNSyQPZas/1K2Prb2FIh5KKjnpaBotc5/Lh9qpdvbrsSZrdbBBaS1yJv1U21SSfoIMgdAx9SC/vCMbFUOHivoFxOsE33tqGk3bZ6zdTB5KikOjuCiKj2JxDALapAryUFLIxPc13dCrpmNCJmQCibz6s6bsIyFlSl9Df3MUD+SvW8Mweev78b9fzD+M7cE3zcp41/NpS29tfY/smN42qgRrTGEBto4rE7Eo/jZ4VF0UGGDwns39vCxquOsX4yg9oXK+Z8fUQ+JECW2dfrz1l7ypphMgpoBVx66KP6/s7o3Sed8vpUhX3nT1PaNtaaRJnsSYMR4yuq5galBJavy8E+dvP8ag+kVEpZxlR644JSm9amd/fD5jNih1Ql+1TbXtzA10/stQB/bCuPqinjF5+ijJpXX5vDqt2n5qjFnpPi5i3vRD2sAzB4i3kcUxCPy1PxzD1p4W2aHJkiTE+VuGH/+vfBzj4ZZZ5nK11tY9j6lOtYy7q100G/6w40tKbDWyO8z1x+7QWzaFEFizC2Jbd2zJOtaMIftsCboh6jOTZ5mqERUebOAVXdi5AioXet+7Zu1Ysowm9bc3Ub21c7SkX8Vx2xBx/9l7XbRZ37GxHRQdulHEWFKiF72IOaJevSVr07bV8pKSQbnm+DURF0rxoc4kqp398YkdG5Q6oa/apqLDig4tkrAx9USG5d0nL6F3QLxO8L2nRsuXRx7K9YERaFI6Fya1iOLys9fYrNeAwNKAy+i//CQypUoqYijpkKNqL0SO7wiRiSJyLJfMqbC9bzWLhzZNDiEF9ogZjGlippnOsk188cJSPfbFhy69Ry4fW0Ufi8HWdNgZctNo0JwYWhslRhpKJP7bxQsVXOJOyjJ37IDwSGOsoKRJM7eviu1MeTXJM//w+St45EqHwKv3xZRj45WUdGtEMZQmWRJ8+XcAEiQgp0ID5ZaqrcRFSTkUU9y0dG780kLNUrSxAaja2R+fD5oNSp3QV21THTh3B61nG+JngkbWERUUiNNPW35Op6XbRU3n+YexLfim8KbeevgMW8/Qfzs23swuC3MipZRZ/e2SY0iZNJEwKK2ppW3LcmXso9RRPFdarPvWctqo1ceu4rt/j0N7Ze6o2vbSmM2eNjkODvQ1wtFy1gFRQq+LjwsG1o8qgWgLXub0HbbmFP6KIR5ZGridKuXHsE+KmaMqzjb7z91Gm9kGkm7iQaSwG5JV3b1RKobr2jgVxtJAS7VTMk96rP6mkrWqlOgn663LydCLFF0N003B1XtPxZ/9B/oia9rk78239KgtIILzCc08BW0QFVCgqjthY+orsTbtJLRlQCkph0JA6PeebtGcSVQ7++MTOzYodUJftU115GIkms44IFZH3oFyY7caY2vqeVhWC1kniCxSI2s/k1eSPJSUYWxa3cMihdzYYgS2Bt0wejgoK5tis6a3LY36Dto/MqFFTtzasoDy6pU8NeffJfV8+VcAtp65YfeXFBnflzFVUhwdUsv4DJrN2C+SnLpVKyiI2x0lsgRmJddMWPSll3HYFrMOwF9nA1frOaRa0nWm7Bbjrfu2MornSqfbkrWlTKsXzoJ5DqyNrtsiNIrk7Yz8EzEE0MtH8iQJ8ezlG/Hnk8NrC4PRVLRcn0Rs3mPxMaRIkkiEHKgm2pcYMihNuWJVm29s81Ht7I9P3Nig1Al91TbVySv38OnUfWJ1/oN8RQzl27fAnA5lUdMJSH+/++eYkUibYij1Pux0euwftZr9Z2+jzRyDh0kK0ZFUd8/qkHVL8nE5mLXGgtZbf3pEHaRKlliEgzgihnHN8avo9c9xpE6WGKdG1DHiFl+FBmKr2d1k+j4cvXQP3asVRH+dDFztb9CKbt5oOmO/WP+W731QKFsa3fYQlfErNNhP/L59DGEx8mVaAkTFBIizVCsUxhQT9ZZMvhzVqJhIyhGe+XdZ9roBrpMibWlXMiipkpujPfZ6LEW1s1+PNVmrgw1Ka5Ez6afapjoT8QD1fjWU3drTvzqqTNgh/vuvL8qDgrtVF5k8QJnd5KGkkmTa0nmqz/9jmJ+2uo1czz9dvOClY/zbh3Cav+8Chv8XZGxibVLWqav3jRVDDg30Rba0ydHqjwOC2/KrKgUwqEFRuz0uv8AIdFt0FJJtQQ70ye97RTzcdzUL4buabnYb31TxhI3BmL7zHMrkywAy8qQY49l8C4niB3pI8PUHxtJ/RPclY7rtUVdakmQTtc6QhvZ7nnrgEpcOWeddttNWiqK/fegK22fCDlyKfIKhDYsKYvO+y04gQ8okODa0dlzDOvxzLcl+6M1HRlaPPrULO3wutgyo2tlvy1ps7csGpa0Ivuuv2qbSxhVt/K6K8YedSnJ5F9Qvw1In+N5TI2ldyPglDyUZyETs+60TVPmxFyaO1qs1COTYa76phBJ50jtkKjIpSA7WsmwejG9mOWcjVdshHlaSbX2qomCW1JAeuW+qF0S/Ova7cjZlW5AcmvSyR3va3LrmegE+aXMIftt+VjxDepZS6v+6B0E6f8fO3XoE3192iSGIxoYMa5L9P9ZATp3rSksKmn51CuOb6q56wRUvemRYghxcW8ue/kbZ2yeHR3m7tZOUBSGIjYE8kz+uDESWNMlweFDNeFnLhwYlnmTiSyaSe0rKcdbfeNXO/vh80GxQ6oS+apvq0p0n8Jlo8Epqr5ucpSzZ6HVBmLPXUB2HPJREbE1Vc7r4FNTpibGauBC4eOcxqk40GGJS9L6u/NAcJI+hbGNtwgjtH6rKQSIN4oa/78Gpq/bn1dwTdgvt/zSwLWi5AOOrcpXkxTRNcLLHfC5HPjHejExs5ol+y08KHKwpnxnXXv338CUsC7giMoTzZUoVV3OlP5cFBWiSFDc5uEFRDF59yjjnD9FnyefYv25hEWNJ/WTpS9UWTd7T5UeuoG6x7Ai9+VDQkg2o546vqzrXb7xqZ398Pmc2KHVCX7VNFXH/KSqOM2RVLuhc3nioOdLDZAu08mqudN70wkN55e5TDP+kKD6vVMAWtdzXAgSohjqxA2iFwifyZExpgRbrm+4IuYlO8w4bFfSo7oq+dSy/Dnv28jXchxg4EGWFEXsYUDGt9ND5O2j5rlqJNpFC60lyFK8nzU/GpbpnT4ON3/kYpyw9fHpW0rr54BnKjzXsn2GfFMWId+ELsSWUWL9TPq6es3efx5gNZ8SiMqdOJowsKqMrpUDmVNgRC32W1tNMHkoKGcmTMQX29K+hHEhaCivioaTfeNonnZzsN161sz8+HzQblDqhr9qmuv3oOcqONnhlZrYrI+gjSDb0rIKiOdPqtGr7qdFWGCEPkzORstsPFcdqfvjsJTyGG7gDpTiyrB0lYlH2sRTyunSvZvl1JpWzcx3kh9dv3mJW+zKoUyw7qk3cgfA7T0S8HcXd2UuOXbqLz6YbklG0nrmqE3fg4h1DrNsXdhzfdF2yTKFr1tSgmtdSqkzYjsuR+h7odx+/QKlRW8QQdLX/8+ZQ8d9UNSd5ErWqttjr+Vujd4n/JQxYGSi6kvHYv05hY7gA/a1IjrQgYvOY5NOpe3Hyyn309C0krsZHrz8jdMRmgFozP736jPjvNObtC0dl18yiUg79xo/9zANtKqhfeEOLgWpnv17Pxxo9Tm9QRkZGYuDAgVi3bh0ePHgAT09P/PTTT6hcubLAo1OnTpg/f340bHLlygXaBCRv3rzBiBEjMGfOHNy9e1f0mz59OlxdLTu4VNtU95++BAWqk0xqUQK9lxrecLf29oFrVv0yLK3ZdOb00Zb9ox8aWg+to0np3OZ05zY6IEDZs2SIaYU4TVMmTayD9rhVaJNpqPWIT4uho7d1ZUNl0sYvzUugaZnc8B63DdccQNR++tp9NPhtr1isNnaw8vjtwiPj6FKoc/acj9HIkNVZ9CSuf/z8FYoN2yTWTjXg6TtNQtRNCal8DkuMCFBcIcUXklBoAiWpaD31dGuzsnvMXJva2GDK8v7JLxiFsqbGFs3Lgyqwj/M7g1m7zqN8/oziypt4bp2l8IYWQ9XO/vh8vk5vUNauXRs3b97E1KlTkSVLFkybNk0Yh0ePHoW7uzvKlCmDunXr4ttvvzXinChRItFWHFIjRggDct68eSBDs3///jh//jxOnz6NpEmTmv1sVNtUT1+8RpGhhms+OiQoyYXEHhmWZoNkQcO5ey9g5Log5M2YUsRQPn35GlPblEJDz5wWaOGmtiLgNsgPL14buO8cbQxok2lobIrDa142j1VLMq1lX3b0Ftx+9AITmnqiRTnrdJozEW1y3M6+1ZA/syG+TxpwYz4rjrYVHFN5iMaVmfO5M6TA3h+irkHpNoNuNX5q4oFWOpVmfUl0Pu9eSNp55cXCg5eQOGECnB2rHsm2Oc/SUW12BN9Ep/mGUA+KIScWgFbvwibob+TRW/hlhRinoy1ZS1neEzeFwDS8wVHriGscY4JY7nQIvfFI/MZTyUhK0nEmUe3sj0/snNqgPHv2LAoVKoR9+/bB29tAgUHXW25ubmjdujWGDRuG1KlTY8mSJWjcuPF7OL948QKZM2fGhAkT0LVrV/H5vXv3kDNnTsydOxetWrUy+9motqm03iWKwRnnFyzWYo8MS7NBsqDhokMXMWjVKVCFEYrle/MWmN2hLIhGiMVxCMhSbjQiJQgEj6rnsMG1MXg0qC2k6nUm70bIjYeQWcAewzfh4bNXmNKyJBqXymW3NWkTUzZ/7wO3d/yLlCREL0rjm3qgZTnHXfEtPHgxxkSNkiM36+4hot/iAgM2CGw/K5ULq45dVZZk224bwArFh8Mj0XymIdSjZpFs6OnrauQUln+b07FsjJrbzD6I/efuiDAOiqGcsjVMlGz871vDjZ1KMnV7mAiDIIOXsrwpJIXCs+oWz67SNOOci2pnf5wTtmMDpzYoyfjbv38/fH19kSxZMiNMdF1NXskePXqgSJEiCAkJEUamqfj7+6NChQrvfU7X3nR1Tp5Lc0W1TUU/5i4DNwiyX4qn+W1bmFgK0UcQjYTqsizgssgKpR9FOvhJnIVDU3VsLZmf19htuP7gmejiaD477ZUpjT+/UzlUK2wdqTqRah/RVKYpPNgPzx1Q+efGg2eiqACJtkKMLJFni9fVkuco2/7jfylGKpniwzbZpTSr22A/UWGJXgSpYlG6FElwYph6nIjWYGmvPloOYTLEiWy+1mRDlSES8uCRJy8mkVV2OlbMJ3gof99+FqqWo/xj9zmM3RAMqr9O8cwkjiycoNfzU+3s12td1uhxaoMypgUvW7YMLVq0wNq1a/Hs2TO0bNlSeB/9/PyQMGFC1K9fH6NHj0a6dOmwcuVKNG3aFE+ePEGKFCmM6qg//Y3iMmMTitek/0mJiIhA+fLlcfnyZeTOrUacnzw0v6xcQFDwkNCPOf2oqy4yjojK5ZFRTOJIUm3V8XHU/Ii/ka6eSaiWMNVkdpS8eWN4KZKyrGtFlMuf0arhZWWcDhXziVhM+bJl78pR2sQUou8iQnES6fmd3LIEPivluN8L+aJm+nIgfyumtSkNIpDXS6Sh6l0wk/CcUdYyJSexxI6A1qvd3isfvq7qgsrjDRRwJB/iY/1i/mFRE7tthbyiUg7FrZbLnwHLukaR2KuC/V/7wzFs7WmxJyjcgmTxlxXg7ao+T7IWQzYoo9D4qAxKuvomzyR5LFevXi2uvMeMGYOxY8cKQzIsLAx9+/ZFnjx5sH37dixevBjt27fH69evhbEppUOHDrh27Rq2bjVkScckw4cPF/GXpqKSQekxbBMePn+F1uXzgjIHSc6MrIsUSdXPsNx46roxM11ivLK7N0rnNRzILI5BQBJw02gFs6TCtj7VHDPwu1GKDt2IJy9ei3/ZwlDwzaKjWB8YgSalcglydBnbRxVcfOxYOUrrZdUWFSg2dCMev3B8zNjqY1djLMdXcOCGaFnwej1k6YktkTsdTly57/CXEr3W4Ug99568QMmRhux4qvVODoEy7xg76G+fe+fH8E+LxTilLn8HYHPQDWF00u0OORIqumTCki5RddsduZYPjSWz2ZMlTihuC0ichSdZuy42KKPQ+GgMyjVr1qBNmzbw8vISnkXyONK17/3795E+fVRlj4MHD6JixYqg/6eN0KxZsxg9lM+fPwfpjE2cwUMpf8wblcyJNceviaWcHVMPiRNFGc+q/LiYzsOUg5A+X9+zMorlTKfqlD/KecmrYlpcfMRiyWQRGt+WhDJtKU+Km5TZx/b2emsTU7RX9o66cjfdlP+duIZvlxxDyqSJEDSyrvhYG+s49/OyqOGuX5yyDJlwyZJKEFerSmGj0pdXu2co5pdKL8oES2lkUvWcmETWAaea5kRsPn9/OKoUyowFnWNO4onPda84ciUavybNZW2PSvDM7ZhKXHqtnQ3Kj8ygpAzvXr16oUmTJli4cGG0eErTTfP48WORqLN8+XLhqaQYSkruKVgwip2fYihLlCghMsbNFRU3VYWxW3HjwXMR2L31zA3Q9TFRdsjyb+auLT7a7T97G23mHIo2tLNQHsUHXvYas92cQ9h79rZQT/QeS7tWtNdQMeqVfI30oS3xv7KcHV290rWu5Ee0t9ebjDXy/lFS2R/ty6B2MUPCgevADXil4cV0FKgx1RbXGjB6e2wlvyVVd6HfIrdsqbH5+yj+S0et29nGcR/ih2cv3whaqXYV8kUL/aBa6xQXH5PIcobkRCAPJWXWVy+cBfM6lVcOAvlyo50YlQl2z64+T7J2ziqe/fH1sJ3eQzljxgx0794dPXv2xOTJk6NdXZPH8s6dO9i0ycCFRrJz505Ur14dQUFBcHFxQdasWTFp0iR07txZfP6xZHnTWiTXnYxfSpo4IUJHOy5L15ZNfeRiJJrOiCK1Jl2OrNJiy9w/pr5f/R0gkilI6GqYDA5HivbK3RYOTFnOjryslCEbU6KMvdYljQNJe6X1CP7ZsSx8i+jnEYxrDfQs6ZkSDeT5cQ1Ecy3FmPZaPi5d5nzu+8tOUTY1dbLEIunHtOSjOTr+H9vUnbIbwdcfGhPR5B4iLD5Uzaj3v8ex8thVNPDIgbQpEmOJ/2XhUIgtKzw+sd18+jq6LDAU3JBCBOzkxXYmYYMy6mk5tUEZGhqK4sWLo2HDhu9lZNOV9549e/Dpp59i1KhRggKI2n/zzTfiynvRokUChUGDBmHWrFmCJih//vzo168fwsPDERgY6NQ8lLQ2WU6tRJ70OHH5nvhRPzWijlN8VwOv3McnUw2E0FIODfRFtrTJnWL+H8skpceD1kM1d2e2L+PQpTWbsR8BF+8K7/q5MdYTYstydi6ZUwm2gCoTDEkOjqhNLhNwJDE/0aOQ15LElsx1ax6EluNQEow/ePYSnu8qItmS+BTTfLQvBPT5h0i5rVnPx9qHEnOoekz1wlkFCbwMX6L1jm5cHO28YuYu7b/8BJYGXEGdYtlEUg7Vyq5XPDtmtHPs99ac57Iz5CY+15RWpT6U9EfJf84kbFB+JAYlJduQQRiTdOzYUVTIWbFihUjMCQ4OFrGU5LWkLO/kyQ2GCSXkUKUdIjZ/+vQpfHx8xFU3GZeWiIqbSnLvUaUE4vlyNO2LJfiZtqUf09oaqgz6/PjQWkif0nyyeVvG574GBH5ccRL/HL4s/psoTCa3LOlQaGR2dqqkiXD6XcyfNROQCQCUUUpxkzUn7RJqbInLNHcepqThz1+9RuHBhqIDCztXQOVCjstq3R16C4QpSdiYekiSKCEiH78QBguJ3iEAjabuFck4UioUyIh/v3Zs2IS5z0nldpKYn+b4oYphVLKR9rqve1ZBG0Tcnw09c2Bqm9LKLW//udtoMzt6WJMtYS3xtUAVz/74wsKpPZTxBVpM46q4qRr+vgenrj4Qb3xX7z1F1jTJ4D/IOSg7wm8/RrWfd0aD2pYrT5X2ijPNZfja0yKwn4Rq7FKtXUdKt4VH4HfquuBOpcPGWll38hp6LD4myNlXdquE+r/tEaoODKiBHOns6xExrdLz5MUrFB1qCMNZ8pUXKhbMZO2yLO6nPcQl44OWQP6/HpXhkVu/xLfmM/fjcPhd4zxVTRCxGEgHd6AXIKq6RDKjbWnU84iZ2mnomlP4+8BFVHXLImIo152MiJcXQXPgiSmsyVlo7bTrU/HsNwd/e7Rhg1InVFXcVJ9N34djl+4hfcokogqGo3kEbYE24v5TVBy3PZoKZ8lQt2XdqvUdvzEYM3YaajBT9Y0hDYs6dIp9lp7AiqNXBPnxzn7VrR5be71G1CTN3lUiOTK4JjKlti/Rv+TyHFS/CL7ycYE9r5jjAsj/QiRazDLEJgcOry2uRellk4xeEr9eVVAkh35JEW3nHMS+s3eM06pZJCvmdCwX1zT5cxMEPvl9LwKvGjy98zqVE1fhMcmI/05j3r5wVHLNJLK86WWsWZncoka2ahJTWFPwqLpInkR9Wjs2KGPeTWxQ6vQtU9GgpIODDpCkiRKKeszORNlx59HzaNxriRImwDmuAazTbjVfze/bwvDLllDR4dsaruhTu7D5nXVoKT0uRXOkxYZeVazWSFVyiAKJZFb7Mvj6XTKANKqsVmxGR9Oyj1qyc72vmOOajhYHGUJy8c5jVJ1ouA3Qm0nh83n+2BlyyzgtVeP54sItvj+Xv+U0j3+7eKGCS8xe7bEbzuCP3edBoQX0skDsHq3L58G4Jp7xvYT3xo8prEnG9So32Q9MSMWzP77wY4NSJ+RV3FRayhdapjNRdoiM0GFR2fla3jydHhmrMQOBOXvOY/T6M6KlrINtRjfdmkzbcRYTN4WgWuEsmG8D9Yn28BrXxAMUa0YSMroukiW2r0fk06l7cfLKfUH1QpQvVMObanmTOJp37+SVe8a60DJeja5SZUzpzr7VkF/HLFtJtC03BNHZ/Noq5rKBum2aL9r1GQAAIABJREFUj1BRx7n+2BVqMMw/tGcmbAzG9J3nREUmuvImY56q7YxqXFw5VEzDmpIkSoCwMfWVm2dcE1Lx7I9rzvb6nA1KnZBVcVN1muePHRrvQLGcabG+p/VeHp2gMksN1f+lOsBS6Nr++FCuAWwWeDo2WnToIgatOiU0DvukKDpVKqCj9rhVUdWQhQcvom7x7HDNmibuDrG0uHbvKbzfXeuSYUxGKsmFcfbnZZWZ6l2rFsSP9dyhre/taLL+09fuo8FvBvaEgwN8kT1dcgRff4C6UwwxpXt/qI7cGVJajbNpx28WH8X6kxHGP6t6/arbgu2kqOuCI9h4+rrQ/iEv8qTNIfht+1kQs0fa5ImxJ+z2Byvr2Gm6ZqnVfiepg62Jd2YNaodGKp79dlimWSrZoDQLprgbqbipTL0DJfOkx+pvKsW9GAVaEFefrLdM0yFi5EMDrU/KUGBJTjmFlUevoPfSE2LuPzXxQKvyeZ1yHQ+fvYTHO2qcTpXyizgzCgUJHWN/XtY2sw+KOtZU8WToJ0WjxSw6msg55PpD1JmyO5rxeOrqfTT83WBk6k3NJXkR5aaJj8Qup9ywJpPW4vgho/+3bWEgzlVyHpCH8uD5SHxVpQAGNXBs7LM5mFP9bmJAkOJMLCTa9al49puDvz3asEGpE6oqbipT70B8VDqxBV5Zno505MmYAnv617BFHfe1AgFZWYW6/tqqJBqVzGWFlvjv8oa4HwdtwNu3wKclcmLtiWsO42WVcYTtvPJidGMPXLrzBD4TDTyYescsxoV0TNfbxy/fQ+Np+0RXvZOUtLRTpP9Ddajjmvv/8+cDVwVi8aFLcT4jGSLinj2NMCgpw156xlXDT/uSR3PLnjY5Dg70VW2acc5HxbM/zknbqQEblDoBq+Km+v7f44KHTApl/i360kunFdtfjSSEppFcs6bG1t5css3+qEcfQZsdrS0d6Oh56DGex7BNePj8lciApczjjKmS4uiQWnqo/qAOeVPQomxuTGhWAudvPUKNXww8mI6uDBI9Aaeq+F4FhEcas971pm0ZsvoUFhy8aMSni48LBtYvYnfMP7YBRq8Lwpy9F8SyJN1TTGv8Y/c5jN0QjIJZUomkHHpZiI9kOnPw1/KxUvu8GVNid3/rmRzMGdMebVQ8++2xTnN0skFpDkpmtFFxU8mqCXL6tiY2mAGDrk0kITQptTXLV9eJ/R8pO3T+Dlr+cVCseEHn8qhSKIvTrr7iuG2IuP8M5L2hsnaO8oj0WHxU8AHKhJSwGw9R6x1pv6PLiV65+wSVxxu8o5u+80Hh7Glw4NwdtJ5teMZ6c72OWheEP98ZQqS/R3VX9K3jWKYAp92wmon/sjkEv28/K/7yoUxowpowz5cppaANIqqh72oWwnc13ZSDQVuClCbnrE4DFc/++HrYbFDqhLyKm2rQqkAsendNQsusVTQbZncoq9OK7a9GWx2iVN70WNXdOeI/7Y+M40bQcsWt6FYRZfJldNzgOo9Ue/IuhN54JEjSKdPaUR6R3kuPY+XRq8YSeGciHoBKEpLs/7EGcjqw1Nz1+8/gNW6bGHvdt5VRPFc67Am7hfZ/GqrnhI6uh6SJE+qGvJbHlJRSljtlu7NYhsD0nWcxYWMIUiRJhDOj6sba+e8D4Ri65rTgHKYrb3pxig92BnNX5zbIT1DakTir00DFs99c/PVuxwalToiquKm0VU5omQ08cmBaW/VKcMX2CGQtcvqcS7bptFEtVHP25kPUnGRI4nB0RrKFU42zeZPp+3D00j0kTpgAr968dZhHRFsO78/Py0GbBOM/0BdZHVifXpsIseabSiIbOKb63nGCaWYDShChRBEpP9R1R7dqBc3szc0kAvP2XcCI/4LiDNOQrAyUxEgeSiq5S8wCFEepohA1HFHEkThT0qgWSxXP/vh61mxQ6oS8iptKktzKJTYumRNTnIgDru6U3eINm8THLQv+/qK8Tk+L1ZiLwOPnr4RH6/WbtzgwwBfpUiQxt6ty7bRcfjQ5R3lE5IudLDtozySYuEAnGqaSIw11u1d08xZ8hZtPX0eXBUeQMAFwflyDuFRY9LlMEpGdqNISVVxisQwBGZZQ0SUTlnSJPQ5+6eHL6L/iJDKnTipiKC/cfozBDYrgyyoulg3ooNZlRm3BnccvxGjO6jRQ8ex30ON7bxhlDMqnT58iUaJESJo0Kc6cOYP169fD29tb/M8ZRMVNNXFTMKbtMJTNI2leJjcmKliCK7bn22jqXpy4Yig3VrNINszp6DzX9c6wZ82dI9V6fvMWgrPQmcWU9cBRHhH5Yle+QEYs/boitDWMTwytjXQpHWekazNrZcWVDYER6L7oqLjqpitvPWX27vMYs8FAjE9CBNtEtM1iOQJ0W0DhESmTJo6184ojV9Bn2Qnx4kdX3lfuPsXwT4ricwfzx5q7OhnXTO2dtc67ime/ufjr3U4Jg3L37t1o3Lgxli9fjqJFi8LNzQ0JEybEo0ePsGTJEjRv3lzvdeuuT8VNNWVrKKZsjbpucjYOuBYzD8A/PFI8qwaeOTCtjfNc1+u+wVihzQiYUthIA89mxXEo+HlTCKbuOGu80tMmOp0aUUfQFzlKnr18DfchG8Vwi7+sAG/XzFhz/Cp6/XPcLsTS8/ddwPD/gozLG9/UAy3LOSeXqaOekS3jyGdJe4oMSkpCG924ONopasRXnbgDF+88EUt21jrvKp79tuwhW/oqYVBWrlxZGJG//vor5s6di4kTJyIsLEz895w5c3Ds2DFb1uiQvipuKtPrJmfjgNOWjmxSKhcmtSzpkGfJg3ycCIxZH4TZewzUKySO8ohIsukiOdLCr1cV7D97G23mHBJzCB5VF8mT2Lf0o/Zpvnz9BoUGGSpQUQgJhZJovVpEG6SnEHcicShKmdyyBD4rlVvPIViXBgHpbU6WOKG48qaYWZWN+FqTdok4T5L6HtkxvW0Zp3ueKp798QWiEgZlypQpcerUKbi4uKB+/frIkycPZs2ahYsXL8Ld3R10Ha66qLipTK+bVK2YENuz7Tz/MLYF3xQfty6fB+OaeKq+DXh+CiPw69YwTN4aapyhr3tWUJKMvWXmrnP4yc/ADbitTzXsDr2FDnMNWdVhY+ohSSL9sqrjWouWqmXe5+VQ3T0r/j18CT+sCBRxdwGD9eXlXBZwGf2WnzROa2qbUmjomTOuafLnViKw6fR1fL3giEg8Iw/l3Scv8XPzEqCSlypKg9/24PS1B2JqzhbjL/FU8eyPr2ethEGZJUsW7NmzBwUKFECmTJkwb948cc196NAhNGrUCNevG2qYqiwqbirT66bu1Qqif113lWGMNrduC4/A75Th2XesmA8jGhV3mrnzRNVDYO7eCxi5Lur6tV7x7JjRzv4eETmurPZkz6xqc1AvOHCDSLKSRPVUK33w6lN24eWUV7ByXs5Ojm8OvvHZZnvwDXwxP0BMIU2yxILIX+UKV5J5gebbsmwejG/mfE4DFc/++NqDShiULVq0EF7IjBkzYsWKFYiIiMC5c+fw9ddfC68lxVGqLipuKkkhIbHr5VsI39dSj+A2tmf73T/HsPr4NfExV9hQ/Rug/vyWBlxGf423TBKN23vmWioXqke/JegGvvo7wC5Z1easxW2wH168eoMZbUujnkcO/LU/HMPWnkbuDCmw9wd9y5tuPBWBrguPGqc1v1M5VCuc1ZxpchsrENgVegvEZkBCteqJ41Flr3CrPw6IeuMklKxFSVvOJiqe/fGFoRIG5a1bt9C1a1ecP38ew4YNEwk6ffr0ER5KStTJnj17fOFj9rgqbirTA1RlgtuYgNZW+uEKG2ZvRW4YCwKmxo2jWA/ktW+GlElwbGhtyHnQgR86Rt+sanMeftGhG/HkxWv83roUPimRE3P2nMfo9WdQIHMqUQpST9l25gY6/2XwmJHIRCA9x2BdUQho43PlX2e2K4O6xdU8Qyn0g0JASIhOimilnE1UPPvjC0MlDMqYFv/ixQtBIeQsouKmWn3sKr7797gRwoH13dHFR02C25ies7YOcJ9abviWK2w4y9dByXnuDbuNdn8akmFI2lbIizGfedh9rqZZ1OtPRoAojJInSYjgUY43KD2Hb8KDZ68gE2RkjGehrKmxpXdVXfHQVuEhxcu7VkTZ/M5bbUlXcOygTMsgINXP6VAWNYtms8Notqv88q8AbD1zQygiwnsivnc2UfHsjy8MlTEo6Zp79uzZCA4OxpQpU7Br1y54eHiIpBxnEBU3lTy4JH7DPimKTorykcX0jLV1gAfUc8fXilZ7cIb9yXMEtITihEenSvkx7JNidodGeiSTJEqAsDH1jTQ9RO1CtEGOltKjtiDy8QtMbOaJ5mXzYOr2MPy8ORQyC13P+ZgaOGt7VIJn7vR6DsG6NAgcuXgXTWfsj4aJymEGWm5YVWuOx7XBVDz745qzvT5XwqA8e/YsKlSogHTp0oEeDhmVP/zwA/z8/LB582anIDdXcVPJChhy86jMRxbTBp+wMRjTdxqI2VUm57XXl5P16ovAuVuP4PvLLqPSr6u6YEC9IvoOEoM2baLE+bH1sfr4VfReaiCf1pumx5zFlBuzVdQy/6mJB1qVz4vJW0Lx67YweOZOh7U9Kpujwuw2Ry/dRZPpUQYO0SaR4cpiHwROXL6HRtP2RVO+sHMFVC6U2T4D2qi197/HsfLYVaGlf93C6F7N1UaNju+u4tnveBQMIyphUFImN2V6k4cybdq0OHHiBPLly4fPP/8cly5dEt5K1UXFTbUj5CY6zTtshG5CU0+0KJdHdSiN89MSs4/9zANEzM7CCFiLAFX8KT92m7F7zxqu6F27sLXqzO6nvWon3sm1J66J5KBMqZLiyBB9aXrMmZT3uG24dv+ZsWqNrKhVKm96rOpeyRwVZrfR1i2nTtv6VEXBLKnN7s8NLUPg9LX7aPDb3mid/uniBS+XTJYpclBrbbEBlUtEfggOFc9+Bz2+94ZRwqAkY5KMRqqSkyZNGmFQUnY3eSrJc3n/vqH8nsqi4qbad/Y22r4jUCbsnI1UeMbOcxi/MVg89l+al0BTRbnUVN6XPLcoBJ6+eI0iQw1VYkj61nZDjxqF7A7R4fBINJ95QIxDHkkinx6wMhBZ0ySD/6Cadh/fdIAqE7bjcmRUSb5xG85g1u7zKJ8/I5Z2rajrfMJuPEStybuNOvf0r448GVPqOgYri0Ig9MZD1NbgTZ+s6FYRZfKpGbc6dM0p/H3goljAqEbF0L5ifqd7nCqe/fEFohIGZYYMGQQPZfHixaMZlJTlXa9ePURGGmgFVBYVN5X/hUi0mGU4yEhUpo+I6dn+ufcCKI6SRGakqrwHeG5qI0Ck3q6D/AQHI4mjktS015CHB9XExtPXQQlnOdMlx/4Bvg4HrfrPO3Hh9mNIj9DI/4Iwd98FeBfMhMVfeek6n/Dbj1Ht551GnYcG+iJbWueuCa8rQDorMw3rIPWrv6kkyn6qKKPXBWHOXkP1KpUr+nwIOxXP/vh61koYlK1atRLrX7RoEdKnT4+TJ0+Kq+8mTZoga9asWLZsWXzhY/a4Km6qY5fu4jNN/NKs9mVQp5ia9BExAS0Jl+kzJkQ2eytyww8gUHLkZtx78lK0cFRc7pmIB6j36x4x5r4fa2DL6euivrUkOnf0A6s5aRfO3nyEH+u5o2vVgpBeIirDSOUY9ZRr957C+6ftRpXHhtRChlTOw96hJxaO0HXpzhP4TNwRbah131ZG8VzpHDG8xWPIcAvqOKVlSTQulctiHfHdQcWzP74wUcKgvHbtGqpVq4Y7d+6I623K7Kayi0R0vnv3bhFPqbqouKlM45dkqTXVsZTz0/Joqpyp6Cx48jyByuO348pdQynXcU080Lq8/eNytV4j4nkkbkZ78T6a84zrTtmN4OsPIXlp6fp9if8l2KMUJSX/UBKQFMpqp+x2FvsgcPXeU1TSGPA0ysbvqsA9u5qJUNpyqNPblkZ9jxz2AcaOWlU8++243A+qVsKgpBk+efJEVMQ5duwY3rx5I66/27VrJzyVHxK6Dh84cCDWrVuHBw8ewNPTEz/99BMqVzZkKx4/fhy9evVCQECAKOvYs2dP9O3b16iSxhoxYgTmzJmDu3fvin7Tp0+Hq6tl2WYqbirTeBqVs/1iesbasm1LvvJCxYJqBpbH15eXx7UcAWlMUU9HxeVejnyCKhMMXqNN3/lgZ8hNjPMLhmvW1NiqM++jOYg0/H0PTl19gO9ruqFXzULot+wElh25gjrFsmFW+7LmqDC7zf2nL1FixGZj+9DR9ZA0seNql5s90Y+koWniGS2L9hjtNRVFGyevMl/mh7BT8eyPr2etjEFJADx79gyhoaFIliyZSMpJkiRJnLjUrl0bN2/exNSpU0Wm+LRp04RxePToUfFv8nZS5Z3evXvj4MGD6N69uzAYO3XqJHSTMUn/pvrhuXLlQv/+/UXFntOnT1tErK7ipqI4KYqXkvJvFy9UUDTbL6YHra1ssqKbN8rkyxDnfuAGjMCHEGgx8wD8ww0x2Y6Ky9Ue8sTDuCfsNiZuCoF79jTY+J2Pwx8Y0cpQXKfMcpfULQ08c2Bam9K6zufZy9dwHxKVCHVhXH0kSJBA1zFYWRQCdx49R5nRUR5h+mRn32rInzmVkjBp4+Qp3ILCLpxNVDz74wtDJQzK169fC6/hzJkzQRVySFKkSCG4KAcPHhzrDxDxVxYqVAj79u0zclVS4L2bmxtat24tdJCBGR4ejsSJDdcs5M1cuXKlyCCnsTJnzowJEyaI0o8k9+7dQ86cOTF37lzI2E5zHo6Km+rK3SeoPD4qnmZld2+Uzus8RtmO4JvoNN9Ae6RyHJA5+4PbqIHAF/MPY3vwTTEZR8UU33vyAiVHbhFjUqWY/efuYNKWUBTLmRbre1ZxODDNZuxHwMW7xsok3y45hv9OXEPjkjkxpVUpXedDCVAFB24QOuOr1KSuC1JcmalHmKa794fqyJ1Bzcx6bZy8szk85FZQ8eyPr22qhEE5aNAgYcCNGjVK0ASRgUlZ38OHD0ePHj2EFzEmIeNv//798PX1FV5NKXRdXbduXeFppCSfxYsXGz/bunUratWqhRs3bghDk8YLCQkRRqgUuvamq3PyXJorKm4q0+sPZzPKtLRHW773QaFsacx9HNyOEYgRgV7/HMOa49fEZ/M6lUP1wlntjpSWrohqWR+8EInftoWhRO50WKMzkbg5i2k56wAOXYhEFx8XDKxfBN0WHoHfqetoViY3fm5ewhwVFrUhg5IMy/iqDGTRZJ288ePnr1Bs2KZoqzg4wBfZ06mZWa+Nk1/V3RulnMjhwQbl+18WJQzKbNmyiSvn+vXrR5vhihUrhEFJZRnNFcoIb9GiBdauXQsyVIl2aPz48cbuQUFBKFasGA4fPixI05s2bSriN8mbKYX6098oLtNcUdGgvPv4BUqNMnhGSCh+q3B25zHKAsIj0ewdf9/uftWRN5Oab9nm7hFuF/8IDFoViEWHLomJkHHn7Wr/CiJaLx0ZsbSvp+04h9J502OlzkTi5iDcds5B7Dt7B19UKoChnxSFrKfcunwejGviaY4Ki9oUGbIRT1++RsZUSXE0HojcLZqskzd+/uo1Cg+OCjGg5QQMronMqaMcLiotURsnv75nZRTLqWY2+ocwU/Hsj69nrIRBSV7EAwcOoEiR6GXQyPgjD+LDhw/NwoeuvskzSR7L1atXi8SaNm3aYOTIkcb+5LUsWLCg8ICSh7J9+/bCI5owYVSgeIcOHUCZ5+TNjE0oAYj+J4WM3vLly+Py5cvInTu3WfO1d6NHz1+huOZtdXufqnBxoioV2qoPxN+XJY2aP4r2fo6sXz8EfvILxsxdhnKedP1cNr9jCJ+ll46u2akc4axd51EufwYs6+qt3+LM1NRxrj92hd5Ch4r5MLJRcXw+zx87Q26hvVc+UT1Hb/EcvgkPnr1CtrTJcGig44nc9V6Pyvq0Ly9ynseH1kL6lGpSNfkFRqDboqNiqlt7+8A1q/M4PCS+bFBGfSOUMCgpC5sSa/76669oiTBffPGFiH38448/4vwOr1mzRhiPXl5ewrNIHkcPDw/h9YzJQ0lJO2RcNmvWLEYP5fPnz0E6YxO6jo/pKl4lg9L0bdXZqlS8efMWvZceFz+Gwz8tFuce4AaMQFwITNtxViTEkFCCjGduxxA+Sy8dJQKdvHIPs/dcgJdLRvzTRd/KNHGtnz7vPP8wtgXfFKVMqaRpuzmHsPfsbXzund8u37Oyo7fi9qPn8ca7aQ4mH1ObAgPW462Bu1+IylRNRKHV+a8AMU9nvYVig1IBg7JGjRrGWbx69Qp79+5Fjhw5UK5cOSRKlEjQB5FxRnW+ly9f/sHvO2V4k1FKROgLFy40xlOSMUlUQQsWLDD2jymGkpJ7yGsphWIoS5QoIRJ6YhNn8FCSQebyLiCe1sFVKj6mY4PXYg0Cfx8Ix9A1p0VXR/LzEXUOJUxQjCJ53uftC0dl18xY+GUFa5ZhU58ufwdgc9ANtCybB+ObeULGVH5VpQAGNShqk+6YOhMvIvEjFsySCtv6VNNdPyuMjoDbID+8eP3G+EeqH588SSIlYdLWuXfW84kNSgUMSknbY84up/jK2GTGjBmCCoj4JSdPnhzt6pr4KClz/Ny5c8JIJRkwYABWrVolsrzJC0mVeCZNmoTOnTuLzz+mLG9aj+vADXj1rtQcxS9RHBMLI/D/isDKo1fQe+kJsXxHhoAQuTeRfJNHMPj6A1G/uKpbFvylc2Uac57rN4uOYn1gBJqUzoVJLUqi6Yz9OKLJ+jZHhyVtqk3cgfA7T+KNJsmSuX4MbaU3XK7l7Jh6SJxITe5PbZy8ylfzH9oXbFAqYFDq8cUlzkoiQG/YsOF7Gdl05U0GI/FQfvrpp4Jf0t/fH926dRNGZseOHcUUKHFn1qxZIss8f/786Nevn4itDAwMdHoeSlqf9sclcHhtpEkeN7enHs+GdTACKiKw+fR1dFlwREzNkXQqskIPlXsMvfkIiw/ZpzKNOZj3XHIMa09cQ6OSOfFrq1JoNHUvTly5b+SlNEeHJW1qT96F0BuP4i2r3ZK5fgxtPYZvwsNnr4xLUZn788WrN6CY3twZUmCiHRgGHPE82aBU0KCkh0JXzFTHmwjNKRP766+/Rt68sZdGGzt2rDAIYxIyGOfPny+yucl7SVfodKXep08fkTkuhRJyiJuSvKBPnz6Fj4+PmAcZl5aIqptKBsTTWlS++rAEa27LCFiLwIFzd9B69kHR3X+QL7KmcQydiu8vO3Hu1mMMqOeO87ce49+Ay6hdNBv+6KBvZRpzcDElMq//6x4ERTxA71pu6OlbyBwVFrVp8NsenL72AGXzZcDybo5PQrJosh9B41IjN+Puu3r1iRMmwNmx0dlTPoIlKrUEVc/++ABJiaScU6dOCUMuZcqUIlOajDwqlfj48WNBWk7Gpeqi6qYqO3oLbj8ykMWfG1sfiRJylQrV9xLPz34IUCWRSuO3I0PKpNjdvzqSOOgqUBpVVO7wUuQTrDh6BfU9smN62zL2W2wsmvsvP4GlAVGlFqUHsX/dwuhezbKSs+ZM/rPp+3Ds0j14F8yExV95mdOF29iAgAyvIBXJkyRE8Kh6NmjjrnEhoOrZH9e87fG5EgYlcUWmSpUKixYtMibUUBlGquVN/28JH6Q9QDJHp6qbquK4bYi4/0wYkmRQsjAC/+8IUMZxssQJHRr+oY1TjLj3FKuPX8MnJXKK8o+OlgErA7HE/xJqFsmKOR3LocYvO4XXdFD9IvjKx0X36cikn2qFs2B+p/K662eF0RGQv/n0VyaTt//uUPXst//K3x9BCYMyderUos42xUNqha6/yXNJiTKqi6qbymfCDuER4TdV1XcQz+9jRqDN7IOi5GKnSvlFcs66kxH4rFQuTG5Z0uHLHrL6FBYcvAhp4MnfiGGfFEWnSgV0n0/7Pw+J+uW1imbD7Hi44td9QYorrDJhOy5HPhWzTJ8yCY4Pra34jJ17eqqe/fGBqhIGJcU2kheyTJno1z8U/0hlEtmgtH5ryNitNMkTI3B4HesVcU9GgBGwGgFZQ5y4H6mClT1LHcY1yeFrT2P+/nBUKZQZCzpXgPe4bbh2/5kgNSdyc71F8l428MyBaW1K662e9ZkgUP3nnbhw+7H4a+bUSREwuBZjZEcE2KCMAlcJg5KutqkyDfFNZsxoqFxx+/ZtUUKRquisXLnSjttBH9Wqbqq6U3Yj+PpDZEqVFEe47Jk+D5u1MAIWItB90RFsCLwuqHooA3dL0A20KpcHPzXVv9RhXFMbsz4oGrG6jLn7qYkHWpWPPQkyLr2xff7N4qNYfzLCbrXCrZ3Xx9qv1qRdCLv5SCwve9rkODjQ92NdqhLrUvXsjw9wlDAo6YF4e3sjMjISbm5uSJAgAUJCQpAhQwbs2rULLi76x/XoDbaqm+rTqXtx8sp9/mHR+4GzPkbAAgS+//c4Vh27igYeOURd6+3BN9G2Ql6M+czDAi36NJXlJ2XpR5kVTKTrzcroXzZ2/7nbmLI1DD/ULYwy+RxT6lIfpJxTS71f9+BMhKEsMNHx7P0hqoiIc65I7VmrevbHB2pKGJRv374VGd1U0YYyvunfVDaRSimmS+ccxeJV3VQyGSBPxhTY059/WOLjS8ZjMgIDVp7EEv/LIhHmxeu32B16Cx0r5sOIRvrXzo4L7Z83hWDqjrMolTc9VnWvBI9hm/Dw+StMaVkSjUvliqs7f644Ap/8vheBV++LWebPlBI7+1VXfMbOPT1Vz/74QFUJg7Js2bKCWNzT0/HXP3qBruqmavXHARw8HwmXLKmwncue6fW4WQ8jYBECMm6Ryi2+xVvsO3sHX1QqgKGf6F/qMK6JTdkaKjyGHrnS4b9vK8N9iB+evXyDqW1KoaFnzri68+eKIyBpmmiarln/uI/jAAAgAElEQVRTY2vvqorP2Lmnp+rZHx+oKmFQZs6cGYcOHYpWTzs+wLBlTFU3lcywdM+eBhu/87FlidyXEWAErERAXjMTuTdReB26EAl71c6Oa4pTt4fh582hKJIjLfx6VTGWZ53ZrgzqFs8eV3f+XHEEms/cj8Phd8Us+Xff/g9L1bPf/it/fwQlDMqJEydi/fr1ouyhq6srqGyiVj5ULSc+QItpTFU3lcywlN4IVfDieTAC/08ITN4Sil+3GbyCxIEZcPEuulYtiB/ruTschhk7z2H8xmC4ZUuNTd/5oMCADWIOf3YsC98i2Rw+Hx5QXwRa/3EQB87fEUqL50qLdd9W0XcA1hYNAVXP/vh4TEoYlAkTRhWup4QcKRRLSf+myjmqi6qbquuCI9h4+jpK502Pld0rqQ4jz48R+CgRkEZcoaypkTp5YlE5pkd1V/StU9jh652z5zxGrz8Dl8ypsOl7HxQa5Cfm8NcX5VHVLYvD58MD6ouAvJUirSXypMeab/h3X1+Eo2tT9ey355pj062EQUmZ3B+SqlXVjwFRdVN9u+QY/jtxDRUKZMS/X1eMjz3GYzIC//cIzNt3ASP+CwIlx1HZR2Je6OVbCN/XcnM4NnIueTOmFB7KIkM3ijks/rICvF0zO3w+PKC+CHSa548dIbeEUq6fri+2MWlT9ez/X3tnAl7jtb3xV2Ke1VRDSmoeIsYYilbN6qqihF6z1lClF1FDFTHzr+HWWIp7Kb3GqrlSRagUjTEaU0xFQ429Yo7/s7Z7jiSCk9rnO/vkvPt5+rRNvqy99m+t7+z37NH5LX+yBiMEZVy35Oggb29vt9ndbfPd1KTqs2QfVoSfsx9i7IokY50k4OkE5KpDufIwZ6Y0yJkxDQ5fuIG+dYvio9pFLEezYOcpDFkVgXxZ02HDxzXgN+x75cOSrlUR4MtjfSwPiOYKu/xrD0J+jVZWOZCgGW4i5kzt+53fcoMFpayjnDJlCi5cuKC89PX1xSeffIL333/fFVySXKepSTVg+QF8s/ssahXLiXm8RzfJceUfkIAOAivCf0OfJfuROW1K5MmSDkei/0T/BsXQ443COswnyYZN3ObKlEaNUJYbsUn9/Yoe1VD+lWxJssWHzSPQfeEv6iYmKXKqwMIulc1zMhl5ZGrf7wrERoxQjhs3DsHBwejVqxeqVq2K2NhY7NixA9OnT8ekSZPwwQcfuIJNkuo0NanWHDiPPv/Zj0GNiqODE+7pTRIkPkwCHkpg3cEL6PF1OFKn9IJPtnQ4cemmeic/qFnIciJL9pxF/2UH1O1Z6z+ugYBRPygfVvesDr/87nHur+XQ3KjCnovC1V3xUmRNrKyNZXEeAVP7fue1+OmWjRCUPj4+GD16NNq2bRvPUzmbcsyYMTh27Jgr2CSpTpOT6s79B0iT0jtJ7eHDJEAC+ghsjoxGp/l7lEFZu3jmSgyGNC6JztV99VXioCXbaGmWdKnUsUHVxm5Wfyn/LUcJsbg3AdutTNIKOUh/TvtK7t0gw703ue+3Gp0RgjJjxowIDw9X1y7GLSIk/f39ERMTYzWXJNfHpEoyMv4BCXgMgR3H/8B7c35W7c2RMQ3++O8dDG9SCu2rFbScwXf7z6PX4r3ImCYl1vWqgZoTflQ+hPSpicK5MlnuDyvUSyBo6X4s/eU3ZbRBqZcxs20FvRXQWjwC7Psf4zBCULZv3x6ZMmXC1KlT4wVq+PDh6irGpUuXGp/CTCrjQ0QHScBlBH45fQXNZ+xU9adL5a3u8x7ZtDT+XqWA5T7Zpt/lPMx1vWug9uePTtn4sd8b8M2RwXJ/WKFeArZrPsXqW2XyYFqb8noroDUKyqfkgBGCsk+fPmq9ZIkSJSBHBKVKlQq7d+9GaGgo3n77bWTNmtXuvkyDm1goKE2MCn0iATMIHDp3HY2/2B7PmbHN/BAY8IrlDm6M+B1dF/yClF4psLZXDdSfvE35ENq/FnxeSm+5P6xQL4Eh3x7CgrDTymjTsnkxObCc3gpojYLSZEFZq5bjl9f/+OOj6RnTCgWlaRGhPyRgDoHjF/+LOhPjn7c7oUUZvFvRx3Inf/g1Gp3/9Wg955qPqtuFbtjA2ng5S1rL/WGFegnY7o0Xq83L58fnLf31VkBrFJQmC8rkkJ8UlMkhimwDCTiHwNkrMagxPv6X4Umt/PFOufzOqfAZVrcevYT2c3epJ5Z3r4bmM35S/73n0zpqfSeLexMYtfYwZoeeVI0IrOSDsc3LuHeDDPeeff/jABkx5W14vjjkHpPKIUx8iAQ8ksClP++g0qiQeG3/Z+tyaOKf13IecTcIfd2lsn2z0P7P6iFL+lSW+8MK9RIYuz4SM7eeUEbfq/wKRr3jp7cCWotHgH0/BaX2V4JJpR0pDZJAsiHw5+179htpbI2SzRKyacLqEhZ1GYFfhqlqv2pf0T79HTG8PjKkSWm1O6xPM4HPvz+CLzYfV1Y7VCuIYU1Kaa6B5uISYN9PQan9jWBSaUdKgySQbAjcexCLIoPXx2vPzL9XQIPSL1vexrg7zkXUfrgoXPlwZGQDnldreTT0Vzg55Cgmhzw6u1nOOZXzTlmcR4B9vwGC8rvvvkO9evWQNm3yWATOpHLeC0vLJODuBB4+fIhCg9Yh9uHjlsxpVxF1Sua2vGn7zl5D02k7VL2fv+uPvkv3q/+OGt0IXl4pLPeHFeolMO3H45iw8Ygy2vX1VzGwYQm9FdBaPALs+w0QlHKYeWRkJPLnz49XX31VHROUPXt2t01VJpXbho6Ok4AlBEp+tgExdx/Y65rXsRJqFctlSd1xK4l7hNGod0pj8MpDEB0ZNeYty31hhfoJzNp6AmPWRyrDPWsVRr/6xfRXQot2Auz7DRCUuXLlQteuXVGnTh3IsUErV65EtmzZEk3TmjVrGp++TCrjQ0QHScClBMqP2IQrN+/afVjQOQA1iuS03KfI32+gweRQVa9Mh45Yc1jdMX50ZEPLfWGF+gnMCY3CyLW/KsO9axfBP+rGv4FOf42ebZF9vwGCcty4cRg0aBBkKihFihTq34kV+d2DB4+/1ZuaukwqUyNDv0jADALVxvyA89dv251Z9H5lVCuUw3Lnjl/8E3UmPjrMPKh+MTU9miG1NyKCG1juCyvUT+BfP53C0O8ilOF+9Yqi55tF9FdCi3YC7PsNEJTiwvXr13Ht2jX4+vpi165dyJkz8W/rBQo4dj3ZyJEjERISgi1btthb2LFjR8yfPz9e+ufLlw+SBFJiY2MhVzzOmTMHV69eRfXq1dWtPYULF07SK8OkShIuPkwCHkeg1v9twck/btrbvaRrVQT4vmQ5B/FBfJHy0ZuF1Y7gLOlSYf/Qepb7wgr1E1gYdhqffntIGf6kQXF0f6OQ/kpokYIykRww4hzKrVu34rXXXkPKlH/9yIrJkydDrnCU6fG4grJChQpo0KABPvroI3vzvb297eJVxKQIyHnz5kGEZv/+/REVFYWIiAikTp3a4deGgtJhVHyQBDySQIPJ2xD5+5/2tsuh4hUKJL7Mx5mA4h6y/n4NX3UIdvYMqfHLkLrOrJa2LSLwn91n8Mnyg6q2T98qgS41XrWoZs+shn3/47gbISjFnTVr1mDUqFE4cOCAusu7ZMmSCAoKwjvvvPPMLD137hy6dOmi7v328fFB7ty57YJSpspl88/ixYvRtGnTJ+zcvXsXOXLkwPjx49GtWzf1exkxzZs3L+TO8MDAQIffECaVw6j4IAl4JAHZWS07rG1l1Yevwd8nq+UsLly/hapjNqt6/17lFSwMO4PcmdPg50F1LPeFFeonsOyX39Dvfzv3h/6tJDq+5qu/Elq0E2Dfb5igXLVqFZo1a6bEo4wwihDctm0bVq9ejRUrVqBJkyZPTV95RgSjTHcHBwfj1KlTdkEpu8hLlCiBI0eOoGjRJxcmyzR75cqVn/i9THuXKVNGjVw6WphUjpLicyTgmQRazdqJn09esTde7tEunS+L5TAu/nkbAaN+UPW2qJAfIkDyZU2HHQPetNwXVqifwKp959D7m33K8IimpdG2imNLxvR74hkW2fcbJijLly+vxOSQIUPiZaAIRBm5FOHnSOnQoUM8Qbl06VK0atVKjT6uX78eXl5eaNSokRKfWbJkUWK1efPmiImJQbp06exVtGzZUv1M6n5auXHjBuQfW7lw4QICAgJw9uxZdRQSCwmQAAnEJSD3Z8s92ray4eMaKP5yZsshXb15F+VGbFL1Ni6TB2sOXEDB7OmxJaiW5b6wQv0E1hw4j56L9irDY5r5oXXAK/oroUU7AQpKwwSliLmDBw8+sRHm2LFj8Pf3V+LOkZJQUA4dOlRNo48ePVoJSbHXr18/NTW+efNmLFq0CG3btlUjoiI2baVdu3Y4f/682uDztDJs2DC1mSdhoaB0JFJ8hgQ8j0DXBXuwMSLa3vCQPjVROFcmy0HcuH0PZYZ9r+qtUyI3Qn6NRuFcGRHS53XLfWGF+glsOPQ7ui38RRme0KIM3q3oo78SWqSgTCQHjFhDWahQIXzxxRdK9MUta9euxfvvv6/EnSMloaCUo4hkJ3nWrI/XKYWFhaFq1aqQf8s3ixYtWiQ6Qnnnzh3IVPzTCkcoHYkInyEBErAR6P3NXqza9/iz7Md+b8A3RwbLAcXcvY+Sn21U9dYokgOhx/5A8ZczYcPH5p/3azksN6ww5HA0uvx7j/J8Uit/vFOOM2bODCNHKB/TNUJQylT3ggULMGPGDLXbW8r27dvRo0cPvP3225gyZYpD+ZBQUCb2Rzdv3lQbdZYtW6ZGKmUN5fHjxyGi1lZkDaWMjE6bNs2heuUhJpXDqPggCXgkgf7L9mPJnkfHlUnZFlQLr2RPbzmLO/cfoNinG1S9ZX2yqo1C/vmzYFXP6pb7wgr1E9hy5CI6zNutDH/Ruhz+5p9XfyW0aCfAvt8wQXn79m21o1ru95aDzKXI6KKMWC5ZsgTp0zv2oZtQULZp0waXL1/Gxo2Pvo1LkSOF5Gaew4cPqysf5caeiRMnonPnzur33OXNTwoSIAFnEPhs1SH8e+dpu2nZBCObYawusbEP8eqgdapaOX/y+q17aFj6Zcz4ewWrXWF9TiCw4/gfeG/Oz8ryzL+XR4PSeZxQC03aCFBQGiYobe7IrmxZSyli0s/PT+3QTkpJKChlU43sEB8xYoQSrEePHsWHH36opry//vprZXrw4MGYNWuWOiaoYMGC6qgi2SkufvAcyqTQ57MkQALPIjB63a/4cluU/ZGfB9VG7sxpXQLNd+BaxL2crNvrhTCgYXGX+MJK9RIIi7qMwC/DlNHZ7SqibsnceiugtXgEKCgNFZQvmqeJTXkvX75cbcwRsSprKWXUUnZ5p0376INcNuTIFZBysPmtW7fUsUUy1S3iMimFSZUUWnyWBDyPwMTvj+Cfm4/bG77n0zrIkTGNS0AUGbwO9x48vu52bDM/BHI3sEtiobvSX05fQfMZO5XZeR0roVaxXLqroL04BNj3J1NB6cosZ1K5kj7rJgHzCUz78bi6N9tW9n1WF1nTO34bl84WlhiyAbfuPbCbdNW94jrbRFuPCMiaWDlEX8qCzgGoUSTxK43JSw8B9v0UlHoyid9StHOkQRJIrgTmhEZh5Npf7c07OKweMqVN5ZLm+g3diD/v3LfX/dOAN5HXBes5XdL4ZF7poXPX0fiL7aqVi9+vgqqFsifzFru2eRSUFJTaM5BJpR0pDZJAsiKwMOw0Pv32kL1NvwY3QLrU3i5pY9ng73Et5p6qO3VKL0QGN4CX16MNkSzuTSDy9xtoMDlUNWJpt6qoVPAl926Q4d6z76eg1J6iTCrtSGmQBJIVgaV7ziJo2QF7m46ObKjEnCtKxZEh+OO/d1TVRXJlxCYeau6KMDilzuMX/4s6E7cq2yt7VEO5V7I5pR4afUSAfb9hgvLkyZP45JNPcOjQIcgRQglLVNTjnZGmJjGTytTI0C8SMIPA6v3n8dHiR1fiSYka3chlo4JVRv+A3288+qytUyIX5rSvZAYkevHCBE79cRNv/N8WZWd1z+rwy2/9ffEv3Ag3MsC+3zBBKQeJy13Ycoe2bfd13HySKxRNL0wq0yNE/0jAtQQ2HY7G+/+7wUSO2z055i2XOfTa2M04d+2Wqr9zdV8MaVzSZb6wYr0ELly/hapjNiujGz+uiWIvW3+9p94WmW2Nfb9hgjJDhgwIDQ1F+fLlzc6cZ3jHpHLb0NFxErCEQOixS2j71S5VVyrvFDg2Kv5Vs5Y48b9K3pjwI05djlH/N+LtUmhbNWnHpFnpK+tKGgE5x7nv0v24eec+ZrxXwWWj4Enz2n2fZt9vmKAUITl58mR1BqS7FiaVu0aOfpOANQR2nbyClrMenQ+YNpUXIkc0tKbiRGqp/fkWnLh0U/3m350CULMoj5ZxWTBYsVsTYN9vmKAMCwtD9+7d0atXL/j6+sLLK/5CdXcQmkwqt/5MoPMk4HQCB367hiZTH50PmDFNShwaXt/pdT6tgvqTtuFI9J/q11uD3kCB7Blc5gsrJgF3JsC+3zBB+dVXXylBef/+43PRbC7K3d5ym43phUlleoToHwm4lsDR6D9Rb9I25YTcob1/aD2XOfTWP0MRcf4GUnqlQOSIBkjp7Zrd5i4DwIpJQBMB9v2GCUofHx80btxYjVCmT5/+iTAXKFBAU+idZ4ZJ5Ty2tEwCyYHAmcsxqDnhR9WUlzKkRviQui5r1ttTt2P/b9dRMHt6bAmq5TI/WDEJuDsB9v2GCUrZlBMREZHk+7NNSkQmlUnRoC8kYB6B6Bu3UXn0D8qxnJnSYPfgOi5zstn0HQg/c02tnZQ1lCwkQAJ/jQD7fsMEZYsWLdCoUSN06tTpr0XUgL9iUhkQBLpAAgYTuB5zD/7B3ysP82RJi50Da7vM2zazw/DTictoX7UAhr9d2mV+sGIScHcC7PsNE5RTp07FwIED0bBhQxQrVgypUsW/3/azzz4zPueYVMaHiA6SgEsJ3L73AMWHbFA+5M+WDts/edNl/vzwazTm/3QKnzUuiSK5eU6hywLBit2eAPt+wwSl7Ox+WpFNObwpx+3fOTaABDyegJwP6DtwneLAtYsenw4EkEwIUFAaJiiTQ14xqZJDFNkGEnAugWKfrsed+7EolDMDfuj7hnMro3USIAGnE2DfT0GpPcmYVNqR0iAJJDsC/sO/x/Vb91AsdyZs/If7XuSQ7ALDBpHAXyTAvt8wQSkHmcvU9tMKz6H8i5nOPyMBEjCKQOXRIYi+cQcl82TGut41jPKNzpAACSSdAAWlYYJy/vz58QTlvXv3cOzYMcjPJ06ciPfeey/pUbb4L5hUFgNndSTghgRqjv8RZ67EoEz+LPiuZ3U3bAFdJgESiEuAfb9hgvJp6blo0SIsWLAA69evNz6DmVTGh4gOkoDLCdSbtBVHo/+Lcq9kxcoer7ncHzpAAiTwYgTY97uJoDx9+jRKliyJmzdvvljELfhrJpUFkFkFCbg5gSZTt+PAb9dRqWA2LO1Wzc1bQ/dJgATY97uJoBw7diymTZuGs2fPGp+1TCrjQ0QHScDlBFrO3Ildp66gyqsv4ZsPqrrcHzpAAiTwYgTY9xsmKOUcyribcuS8ths3buDatWsYNWoUBgwY8GIRt+CvmVQWQGYVJODmBNp+9TNCj/2B6oVzYGGXym7eGrpPAiTAvt8wQTls2LAndnmnTp0ar732GmrWdI+jNZhU/GAhARJ4HoEu/9qDkF+jeYf280Dx9yTgJgTY9xsmKN0kb57pJpMqOUSRbSAB5xLouSgcaw5cwJvFc2Fuh0rOrYzWSYAEnE6Afb+BgnLHjh3Yvn077t69C5nyjlt4l7fT3wlWQAIkYAGBoKX7sfSX31C/VG7MalvRghpZBQmQgDMJUFAaJihl882gQYOQLVs2ZM6cOV7seZe3M18F2iYBErCSwM4TlxG85jD61SuK2iVyW1k16yIBEnACAQpKwwRl/vz50alTJwQHB79QuEeOHImQkBBs2bLFbmffvn3o3bs39uzZg+zZs6NXr17o16+f/fexsbEYPnw45syZg6tXr6J69eqYPn06ChcunCRfmFRJwsWHSYAESIAESMDtCbDvN0xQpk+fHgcOHEiyiIubiZMnT0afPn3UJh6boLx8+TKKFy+Opk2bqt+FhYWhR48eSjB27NhR/bmISfn/efPmIV++fOjfvz+ioqIQEREB2RjkaGFSOUqKz5EACZAACZBA8iDAvt8wQVmvXj0l8Fq3bp3kDDt37hy6dOmC0NBQ+Pj4IHfu3HZBOWbMGHWO5alTp5AyZUplW6bWV6xYgcjISLVeM0eOHBg/fjy6deumfi9HFeXNmxdz585FYGCgw/4wqRxGxQdJgARIgARIIFkQYN9vmKCU6WYZGRRRKSOKadKkiZdo7dq1e2rirV69GosXL4ZMd8uUuYhH2whlo0aNkDVrVsgVjrYiU+J169ZFdHS0erZy5co4cuQIihYtan9Gpr3LlCmjRi4dLUwqR0nxORIgARIgARJIHgTY9xsmKL28vJ6aWbIp58GDBw5lXocOHeIJShGFDRs2xLhx4+x/f/jwYZQqVQq7d+/GmTNn0Lx5c8TExCBdunT2Z1q2bKl+tmbNGofqlYeYVA6j4oMkQAIkQAIkkCwIsO83TFDqyqqEglI21rRp0ybeZh9ZH1moUCE1RS4jlG3btlWCNa6olRHR8+fPqw0+Tytyk4/8YysXLlxAQECAuiZSNhmxkAAJkAAJkAAJJG8CFJQeIij9/Pwg096JjVCGh4erzTctWrRIdITyzp07WLVq1VPfBLndRzb0JCwUlMn7w4OtIwESIAESIAEbAQpKDxGUIiblqKAFCxbYW5zYGsrjx4+rUUtbkTWU/v7+akMPRyj5wUECJEACJEACJJAYAQpKDxGUcmD6zJkzceLECXh7e6tWDxw4ECtXrlS7vGUUMleuXJg4cSI6d+6sfs9d3vzQIAESIAESIAEScIQABaWHCMqLFy+qXeNNmjRRu8h37dqF7t27K5HZvn17RWHw4MGYNWuWOiaoYMGCCAoKUmsrDx48yHMoHXmb+AwJkAAJkAAJeCgBCkoPEZTSTNnNLbfj7N27F3ny5EHfvn3Rs2dPOwHZkCNnU8rB5rdu3VIHo8tUt4jLpBQmVVJo8VkSIAESIAEScH8C7PuTqaB0ZWoyqVxJn3WTAAmQAAmQgPUE2PdTUGrPOiaVdqQ0SAIkQAIkQAJGE2DfT0GpPUGZVNqR0iAJkAAJkAAJGE2AfT8FpfYEZVJpR0qDJEACJEACJGA0Afb9FJTaE5RJpR0pDZIACZAACZCA0QTY91NQak9QJpV2pDRIAiRAAiRAAkYTYN9PQak9QZlU2pHSIAmQAAmQAAkYTYB9PwWl9gRlUmlHSoMkQAIkQAIkYDQB9v0UlNoTlEmlHSkNkgAJkAAJkIDRBNj3U1BqT1AmlXakNEgCJEACJEACRhNg309BqT1BmVTakdIgCZAACZAACRhNgH0/BaX2BGVSaUdKgyRAAiRAAiRgNAH2/RSU2hOUSaUdKQ2SAAmQAAmQgNEE2PdTUGpPUCaVdqQ0SAIkQAIkQAJGE2DfT0GpPUGZVNqR0iAJkAAJkAAJGE2AfT8FpfYEZVJpR0qDJEACJEACJGA0Afb9FJTaE5RJpR0pDZIACZAACZCA0QTY91NQak9QJpV2pDRIAiRAAiRAAkYTYN9PQak9QZlU2pHSIAmQAAmQAAkYTYB9PwWl9gRlUmlHSoMkQAIkQAIkYDQB9v0UlNoTlEmlHSkNkgAJkAAJkIDRBNj3U1BqT1AmlXakNEgCJEACJEACRhNg309BqT1BmVTakdIgCZAACZAACRhNgH0/BaX2BGVSaUdKgyRAAiRAAiRgNAH2/RSU2hOUSaUdKQ2SAAmQAAmQgNEE2PdTUGpPUCaVdqQ0SAIkQAIkQAJGE2DfT0GpPUGZVNqR0iAJkAAJkAAJGE2AfT8FpfYEZVJpR0qDJEACJEACJGA0Afb9HiQoT58+jYIFCz6RkLNnz0aXLl2wb98+9O7dG3v27EH27NnRq1cv9OvXL8kJzKRKMjL+AQmQAAmQAAm4NQH2/R4kKFevXo2WLVsiKioKKVKksLc8S5YsiImJQfHixdG0aVP06dMHYWFh6NGjB6ZPn46OHTsmKcmZVEnCxYdJgARIgARIwO0JsO/3IEE5evRoLFmyRI1EJixjxozBtGnTcOrUKaRMmVL9etCgQVixYgUiIyOTlOhMqiTh4sMkQAIkQAIk4PYE2Pd7kKAMDAyEl5cXFi1a9ETiNmrUCFmzZo33u5CQENStWxfR0dHIlSuXw8nOpHIYFR8kARIgARIggWRBgH2/BwnKUqVKIU+ePLhz5w6OHj2KIkWKYMiQIahfvz7KlCmDhg0bYty4cXYihw8fhvzN7t27UbFixacm/I0bNyD/2MqFCxcQEBCAs2fPIn/+/MniRWEjSIAESIAESIAEnk6AgtJDBOXdu3eRIUMGlCtXDhMmTEDGjBmxcOFCTJkyBZs2bULXrl3Rpk0bBAcH24nIWstChQohNDQU1atXf2oWDRs2DMOHD3/i9xSU/OghARIgARIgAc8gQEHpIYJSmnnz5k21PjJNmjT2Vjdo0EBt0JFEkGnvxEYow8PDlRB9WuEIpWd8WLCVJEACJEACJPA0AhSUHiQoE0uCoKAgbNiwAT4+PuqooAULFtgf4xpKfnCQAAmQAAmQAAk4QoCC0kME5YEDB1CtWjUlHuNOX9eqVQu5c+dG2bJlMXPmTJw4cQLe3t6KysCBA7Fy5Uru8nbkTeIzJEACJEACJODBBCgoPURQxsbGKkEp094zZsxAjhw58OWXX6pzJmXTjYhKOYeySZMm6N+/P3bt2oXu3bsrkdm+ffskvSJMqiTh4sMkQAIkQAIk4PYE2Pd7iKCUZl66dCc2U7cAABC5SURBVAkDBgzAunXrcO3aNZQvXx5jx45FjRo1FAURlnI7zt69e9Vu8L59+6Jnz55JTnImVZKR8Q9IgARIgARIwK0JsO/3IEFpVaYyqawizXpIgARIgARIwAwC7PspKLVnIpNKO1IaJAESIAESIAGjCbDvp6DUnqBMKu1IaZAESIAESIAEjCbAvp+CUnuCMqm0I6VBEiABEiABEjCaAPt+CkrtCcqk0o6UBkmABEiABEjAaALs+ykotScok0o7UhokARIgARIgAaMJsO+noNSeoEwq7UhpkARIgARIgASMJsC+n4JSe4IyqbQjpUESIAESIAESMJoA+34KSu0JyqTSjpQGSYAESIAESMBoAuz7KSi1JyiTSjtSGiQBEiABEiABowmw76eg1J6gTCrtSGmQBEiABEiABIwmwL6fglJ7gjKptCOlQRIgARIgARIwmgD7fgpK7QnKpNKOlAZJgARIgARIwGgC7PspKLUnKJNKO1IaJAESIAESIAGjCbDvp6DUnqBMKu1IaZAESIAESIAEjCbAvp+CUnuCMqm0I6VBEiABEiABEjCaAPt+CkrtCcqk0o6UBkmABEiABEjAaALs+ykotScok0o7UhokARIgARIgAaMJsO+noNSeoEwq7UhpkARIgARIgASMJsC+n4JSe4IyqbQjpUESIAESIAESMJoA+34KSu0JyqTSjpQGSYAESIAESMBoAuz7KSi1JyiTSjtSGiQBEiABEiABowmw76eg1J6gTCrtSGmQBEiABEiABIwmwL6fglJ7gjKptCOlQRIgARIgARIwmgD7fgpK7QnKpNKOlAZJgARIgARIwGgC7PspKLUnKJNKO1IaJAESIAESIAGjCbDvp6DUnqBMKu1IaZAESIAESIAEjCbAvp+CMl6CxsbGYvjw4ZgzZw6uXr2K6tWrY/r06ShcuLDDicykchgVHyQBEiABEiCBZEGAfT8FZbxEFjEpAnLevHnIly8f+vfvj6ioKERERCB16tQOJT2TyiFMfIgESIAESIAEkg0B9v0UlHYCd+/eRY4cOTB+/Hh069ZN/fzatWvImzcv5s6di8DAQIcSn0nlECY+RAIkQAIkQALJhgD7fgpKO4Fdu3ahcuXKOHLkCIoWLWr/uUx7lylTRo1cOlKYVI5Q4jMkQAIkQAIkkHwIsO+noLQTWLFiBZo3b46YmBikS5fO/vOWLVuqn61Zs8ahzGdSOYSJD5EACZAACZBAsiHAvp+C0k5g4cKFaNu2LR48eAAvLy/7z9u1a4fz588jJCQk0cS/ceMG5B9bOXv2LKpVqwYZ8cyTJ0+yeVnYEBIgARIgARIggcQJXLhwAQEBATh58iQKFizo0ZhSPHz48KEnE1i+fDlatGiR6AjlnTt3sGrVqkTxDBs2TO0MZyEBEiABEiABEvBsAjKYVKlSJY+G4PGC0raG8vjx4yhUqJA9GWQNpb+/P6ZNm+bQCOXt27cho5S+vr5ImTKl1qSyfQPi6KdWrJYaYwwtxa29MsZPO1LLDTKGliPXXqGJMbx//z4uXboEPz8/pE2bVnub3cmgxwtKGYXMlSsXJk6ciM6dO6vY/ZVd3s4MOtdoOJOuNbYZQ2s4O6sWxs9ZZK2zyxhax9pZNTGGziKrx67HC0rBOHjwYMyaNUsdEyRrIIKCgnDq1CkcPHjQ4XMo9YQjcSt8iZxJ1xrbjKE1nJ1VC+PnLLLW2WUMrWPtrJoYQ2eR1WOXghJQG3IGDRqkDja/desWatasqaa6TVlgy5dIT7K70gpj6Er6L1434/fiDF1tgTF0dQRevH7G8MUZOtMCBaUz6WqyLbvJZUq+T58+yJw5syarNGMlAcbQStr662L89DO12iJjaDVx/fUxhvqZ6rRIQamTJm2RAAmQAAmQAAmQgAcSoKD0wKCzySRAAiRAAiRAAiSgkwAFpU6atEUCJEACJEACJEACHkiAgtIDg84mkwAJkAAJkAAJkIBOAhSUOmlqthUbG6tu45kzZw6uXr0KOWx9+vTpKFy4sOaaaE4XgdOnTyd6OsDs2bPRpUsX7Nu3D71798aePXuQPXt29OrVC/369dNVPe28AIGRI0eqq1a3bNlit/K8ePEdfQHgTvjTxGLYsWNHzJ8/P15t+fLlg+wYlsIYOiEQSTR55coVddLKmjVr1JXGZcqUwdixY1WfJ4XvYRKBuuhxCkoXgXekWhGTIiDlOCP5AOzfvz+ioqIQERFhxPmYjrTB055ZvXo1WrZsqeKUIkUKe/OzZMmirvcsXrw4mjZtqnbsh4WFoUePHirG0umxuI7A5MmTVUzkyDCboLx8+fJz48V31HUxS1hzYjGUZypUqIAGDRrgo48+sv+Jt7c3cubMqf6fMXR9DOvVq4eLFy9i6tSpKi5ybJ8MpISHh6v/f97nJmPo+hiKBxSUZsThCS/u3r2LHDlyYPz48ejWrZv6vWk3+BiKzqVujR49GkuWLFHfqBOWMWPGqA9KOTTfdj2nfCtfsWIFIiMjXeq3p1Z+7tw5NXIcGhoKHx8f5M6d2y4onxcvvqNmZM2zYihnDGfMmBGLFy9WX+QSFsbQ9TGUa4+LFCmCHTt2oFq1asqhhw8fomjRomjdujXSpUv3zM9NxtD1MbR5QEFpTizieWK7Y/zIkSPqxbIVmQKQ6QAZ1WIxj0BgYCC8vLywaNGiJ5xr1KgRsmbNGu93MsVat25dREdHqytAWawlICPKIjZkqjQ4OFiJfdsI5fPiJc9WrlwZfEetjVnC2p4VQ/miVqJEiSdiZLPBz1nXxs42UPLTTz+hdu3aSJMmjd0hWdolI8sy2/Osz02+h66PIQWlOTFI1BMZtWrevLmaJpVvaLYi06nyM1lrwmIegVKlSiFPnjyQO+KPHj2qvnkPGTIE9evXV18EGjZsiHHjxtkdP3z4MORvdu/ejYoVK5rXIA/yqEOHDvEE5fPidebMGb6jhuVHwhguXboUrVq1UrM869evV1/25IuCfIGQZSj8nDUsgP9zR+Imfd13332nrkZ+1ucm30NzYsgRSnNiEc+ThQsXom3btupaSPkQtJV27drh/PnzavMAi1kEZOolQ4YMKFeuHCZMmKCm2iSOU6ZMwaZNm9C1a1e0adNGjYTZinz7LlSokJpytS1AN6tVnuNNQjEiIyTPipeMjPAdNSs/EsZw6NChGDVqFGQpigjJY8eOqU1wsrxh8+bNaraAMTQrhjL1LSOTMmL57bffqk2ofA/NitHTvKGgNDROy5cvR4sWLRIdoZTRr1WrVhnquWe7dfPmTbU+Mu7UjXw4ygYd2VUqnVpiI5Sy+FyEKIvrCCQUI35+fs+Ml3wZ4DvqunglVnPCGMpavOvXr6spU1uRzXBVq1ZVm+LknWQMzYmh9GsiHqtUqaJm4WR2ju+hOfF5nicUlM8j5KLf29b2yIJlGcGyFRnF8vf3V4uUWdyDQFBQEDZs2KBGReSooAULFtgd5xpKc2KYUIyI+H9WvGxrt/iOmhvDxDyTL30ye7Bs2TL1Tso6WMbQ9TGUHd5ypFqzZs3UzI7tSznfQ9fHxlEPKCgdJWXxczIKKZs0Jk6ciM6dO6vaucvb4iAksboDBw6oXYoiHuNOX9eqVUvtHi5btixmzpyJEydOQI4tkTJw4ECsXLmSu7yTyNoZjycUlHIO3rPixXfUGVF4MZsJYyijXXL808aNG+2GZdOVvJOyfvnVV1/l5+yLIdfy1zNmzFBHqMm5vJMmTYq3zIvvoRbElhihoLQE81+rRBYjz5o1C3PnzlWHZctIl4yKHDx4kOdQ/jWkTv0rOSBZBKWMgMgHpBz79OWXX6od+bLpRkSlnKfWpEkTdaaojEJ3795diZb27ds71Tcafz6BhGJEzsV7Xrz4jj6fq5VPJIyhTJvK+zZixAjICQyyUe7DDz9UU95ff/21co0xtDJCT9YlMSldujQaN278xOklMuUtX9z4Hro2Ro7WTkHpKCkXPCcbcuScQjnY/NatW+rQZZnqFnHJYiaBS5cuYcCAAVi3bp0aUS5fvry68aFGjRrKYRGW8i187969ajd437590bNnTzMb42FeJRQjjsSL76hZSZJYDGU9umzMkSOEZC2ljFrKLu+0adMq5xlD18ZQNkyJqE+syBdtueXoeZ+bjKFrY2irnYLSjDjQCxIgARIgARIgARJwWwIUlG4bOjpOAiRAAiRAAiRAAmYQoKA0Iw70ggRIgARIgARIgATclgAFpduGjo6TAAmQAAmQAAmQgBkEKCjNiAO9IAESIAESIAESIAG3JUBB6baho+MkQAJWE5ATFmQn8bBhw5xSte2MxJMnTz71NAdn++CUhtEoCZBAsidAQZnsQ8wGkgAJ6CIgx0LJ2Xhy04ozitwHf+XKFeTMmdN++H3CeigonUGeNkmABF6UAAXlixLk35MACZCAhQQoKC2EzapIgAQcJkBB6TAqPkgCJOAuBFKkSKEOS5Z70+Wmja1bt8LX1xdDhgxR9wRfv35d3c4RHByMevXq2Zv1/fffq+swbdfyycHzcvWpbQo6oZhbu3atuoXl0KFDyJw5M1q3bq0O0bYdmi1+yG1JS5Yswfbt2/HSSy+pg+yljsRKwilv8VMOwl+1apW6HUsuOpg8ebJTp93dJcb0kwRIwCwCFJRmxYPekAAJaCAgQk6uvly/fj3u37+PKlWqqBtSIiIilCDLnz8/Vq9erW41krvU33rrLezbtw+VKlXCP/7xDyUi5f/lasyrV68mKii//fZbNG/eXK2nbNmyJY4dO6aeDwgIgNzOIkX8yJIlC6ZOnap+LgJXbmnZtm2b/fakuM1NKCjr16+PM2fOKFGaKVMmdbPS5s2bMXToUKet49SAnyZIgAQ8kAAFpQcGnU0mgeROQIScCMOJEyeqph4/fhxFihRRV7hVrFjR3ny52u306dMQISf/LfcK79y50/77L774Qo0QJjZCWblyZfj4+GDZsmX250Wkyt3RMsJZokQJJSh79+6tRKytyChlUFBQoqOUcQWl7Q7jkJAQ1K5dW/15dHQ0ChQooISwszYGJffcYPtIgAScQ4CC0jlcaZUESMCFBETITZ8+XY0YSlm6dKkaRcyQIUM8r+7du6fudxah5ufnh7p169pFqDwoo5TlypVLVFCmT58ecg/xxx9/bLcpU9Ri75tvvkGrVq2UoJw5cya6du1qf0amzTt27KhGGROWuIJyz549ePfdd9X0vEyn20rJkiVVWygoXZhgrJoESOAJAhSUTAoSIIFkR0CE3Lx589RaQyn/+c9/EBgYiPDwcDV1HLd4e3ur9ZUiHF9//fV4o4nPEpSy23vs2LFqBNJWZId29uzZ1ZR3s2bNlKCM64c896xNNYkJSplyF5FqK/7+/njnnXcoKJNd1rJBJODeBCgo3Tt+9J4ESCARAgmFnKydlE04a9asUeslbUU27nh5eamNNSI+IyMjERYWZv/9rFmz0K1bt0RHKGVN5CuvvBJvyls2zzRt2jTelPdfFZQPHjxA4cKF4/l87do15MuXT02Zc4SSqU8CJGASAQpKk6JBX0iABLQQSGxk8G9/+5uawp42bZoSlytWrED//v3x1VdfqSlo2aldtmxZ9OvXD506dVKi8IMPPoCcPXnq1Cm1djHu6KJMo8u09vDhw9W/Zf2lTLGLDVlLKeVFRiilLpnalnWfs2fPxssvv6zWXcpGo08//ZSCUkum0AgJkIAuAv8PmylJaj6HK1MAAAAASUVORK5CYII=\" width=\"599.9999869953505\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "pb_num_dict = {_k:len(_v.kept_probes) for _k,_v in filtered_pb_dict.items()}\n",
    "\n",
    "pb_nums = np.array(list(pb_num_dict.values()))\n",
    "plt.figure(figsize=(6,3), dpi=100)\n",
    "plt.plot(pb_nums)\n",
    "#plt.plot(np.ones(len(pb_nums))*20)\n",
    "plt.ylabel(f\"num of probes\", labelpad=1)\n",
    "plt.xlabel(f\"region id\", labelpad=1)\n",
    "plt.ylim([0, num_probes_kept*1.1])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "familiar-cement",
   "metadata": {},
   "source": [
    "# 3. Load and asssemble probes"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "parliamentary-flavor",
   "metadata": {},
   "source": [
    "## 3.1 design readout mapping dict (region_to_readout)\n",
    "\n",
    "load directly from http://localhost:8889/notebooks/Chromatin_Analysis_Scripts/Library_design/CTP-10_Aire/CTP-10_Aire_introns_DNA-MERFISH.ipynb#3.3-assemble-probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "correct-tooth",
   "metadata": {},
   "outputs": [],
   "source": [
    "# summarize total readout usage\n",
    "encoding_folder = os.path.join(pool_folder, f'Genes_intronic_RNA')\n",
    "gene_2_readout_dict = pickle.load(open(os.path.join(encoding_folder, 'gene_2_readout.pkl'), 'rb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ahead-priority",
   "metadata": {},
   "source": [
    "## 3.2 loading components"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "substantial-engineering",
   "metadata": {},
   "source": [
    "### primers\n",
    "pick primer set 6,11"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "dressed-chamber",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Picked primer: ID: W1A07_primer_6\n",
      "Name: W1A07_primer_6\n",
      "Description: W1A07_primer_6\n",
      "Number of features: 0\n",
      "Seq('CGCAAACTGGTGCGGAAGGC')\n",
      "- Picked primer: ID: W1A12_primer_11\n",
      "Name: W1A12_primer_11\n",
      "Description: W1A12_primer_11\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGCCATTGCCCGCGAGGTCGAG')\n"
     ]
    }
   ],
   "source": [
    "primer_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Primers'\n",
    "primers = library_tools.assemble.load_primers([6,11], _primer_folder=primer_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "designed-filing",
   "metadata": {},
   "source": [
    "### readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "weekly-blues",
   "metadata": {},
   "outputs": [],
   "source": [
    "# summarize total readout usage\n",
    "encoding_folder = os.path.join(pool_folder, f'Genes_intronic_RNA')\n",
    "gene_2_readout_dict = pickle.load(open(os.path.join(encoding_folder, 'gene_2_readout.pkl'), 'rb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "formed-roulette",
   "metadata": {},
   "outputs": [],
   "source": [
    "# collect types\n",
    "readout_usage_by_type = {}\n",
    "for _gene, _infos in gene_2_readout_dict.items():\n",
    "    for _info in _infos:\n",
    "        if _info[0] not in readout_usage_by_type:\n",
    "            readout_usage_by_type[_info[0]] = []\n",
    "        if int(_info[1:]) not in readout_usage_by_type[_info[0]]:\n",
    "            readout_usage_by_type[_info[0]].append(int(_info[1:]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "initial-passion",
   "metadata": {},
   "outputs": [],
   "source": [
    "readout_usage_dict = {_type:len(_v) for _type,_v in readout_usage_by_type.items()}\n",
    "readout_dict ={}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "polar-sleep",
   "metadata": {},
   "outputs": [],
   "source": [
    "reload(library_tools.assemble)\n",
    "overwrite_readout_usage = True\n",
    "readout_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Readouts'\n",
    "readout_start = 0\n",
    "num_color = 2\n",
    "comb_readouts = library_tools.assemble.load_readouts(readout_usage_dict['c'], \n",
    "                                                       _type='snr_after_630', \n",
    "                                                       _readout_folder=readout_folder, \n",
    "                                                       _num_colors=2, # 750, 647\n",
    "                                                       _start_id=readout_start,\n",
    "                                                       )\n",
    "readout_dict['c'] = comb_readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "electoral-insured",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "save\n"
     ]
    }
   ],
   "source": [
    "unique_readouts = library_tools.assemble.load_readouts(readout_usage_dict['u'], \n",
    "                                                       _type='designed_readouts', \n",
    "                                                       _readout_folder=readout_folder, \n",
    "                                                       _num_colors=2, # 750, 647\n",
    "                                                       _start_id=int(sum(list(readout_usage_dict.values()))/2)+1,\n",
    "                                                       )\n",
    "\n",
    "readout_dict['u'] = unique_readouts\n",
    "\n",
    "# save\n",
    "readout_usage_file = os.path.join(library_folder, 'readout_usage.pkl')\n",
    "if not os.path.isfile(readout_usage_file) or overwrite_readout_usage:\n",
    "    print('save')\n",
    "    pickle.dump(readout_dict, open(readout_usage_file, 'wb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "chubby-savannah",
   "metadata": {},
   "source": [
    "## 3.3 assemble probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "interstate-intensity",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Assemble probes by given target sequences, readouts and primers.\n",
      "-- included readout types: ['u', 'c']\n",
      "--- assemblying 300 probes in region: Acpp\n",
      "--- assemblying 300 probes in region: Agt\n",
      "--- assemblying 300 probes in region: Aldh3a1\n",
      "--- assemblying 300 probes in region: Aldob\n",
      "--- assemblying 300 probes in region: Alox5ap\n",
      "--- assemblying 300 probes in region: Alpi\n",
      "--- assemblying 300 probes in region: Alpl\n",
      "--- assemblying 175 probes in region: Ang4\n",
      "--- assemblying 300 probes in region: Anxa8\n",
      "--- assemblying 300 probes in region: Apoa1\n",
      "--- assemblying 300 probes in region: Apoa2\n",
      "--- assemblying 300 probes in region: Apoc2\n",
      "--- assemblying 300 probes in region: Apod\n",
      "--- assemblying 300 probes in region: Ascl1\n",
      "--- assemblying 300 probes in region: Avil\n",
      "--- assemblying 300 probes in region: Bgn\n",
      "--- assemblying 300 probes in region: C1qb\n",
      "--- assemblying 300 probes in region: C3\n",
      "--- assemblying 300 probes in region: Calb2\n",
      "--- assemblying 300 probes in region: Cald1\n",
      "--- assemblying 300 probes in region: Camp\n",
      "--- assemblying 300 probes in region: Car4\n",
      "--- assemblying 300 probes in region: Car8\n",
      "--- assemblying 300 probes in region: Ccdc30\n",
      "--- assemblying 300 probes in region: Cck\n",
      "--- assemblying 300 probes in region: Ccl1\n",
      "--- assemblying 300 probes in region: Ccl2\n",
      "--- assemblying 300 probes in region: Ccl20\n",
      "--- assemblying 200 probes in region: Ccl6\n",
      "--- assemblying 300 probes in region: Ccl9\n",
      "--- assemblying 300 probes in region: Cd14\n",
      "--- assemblying 300 probes in region: Cebpb\n",
      "--- assemblying 300 probes in region: Cga\n",
      "--- assemblying 300 probes in region: Cilp\n",
      "--- assemblying 300 probes in region: Ckb\n",
      "--- assemblying 300 probes in region: Ckm\n",
      "--- assemblying 300 probes in region: Clps\n",
      "--- assemblying 300 probes in region: Cnfn\n",
      "--- assemblying 300 probes in region: Cnmd\n",
      "--- assemblying 300 probes in region: Col1a1\n",
      "--- assemblying 300 probes in region: Col4a1\n",
      "--- assemblying 300 probes in region: Col6a1\n",
      "--- assemblying 300 probes in region: Cpn1\n",
      "--- assemblying 300 probes in region: Crabp1\n",
      "--- assemblying 300 probes in region: Cryaa\n",
      "--- assemblying 300 probes in region: Ctrb1\n",
      "--- assemblying 300 probes in region: Cxcl13\n",
      "--- assemblying 300 probes in region: Cyp1a2\n",
      "--- assemblying 300 probes in region: Cyp2f2\n",
      "--- assemblying 300 probes in region: Dbn1\n",
      "--- assemblying 300 probes in region: Dclk1\n",
      "--- assemblying 300 probes in region: Dlk1\n",
      "--- assemblying 279 probes in region: Dppa3\n",
      "--- assemblying 300 probes in region: Ecm1\n",
      "--- assemblying 300 probes in region: Efna3\n",
      "--- assemblying 300 probes in region: Elovl4\n",
      "--- assemblying 300 probes in region: Fabp1\n",
      "--- assemblying 300 probes in region: Fabp6\n",
      "--- assemblying 300 probes in region: Fam183b\n",
      "--- assemblying 300 probes in region: Fam25c\n",
      "--- assemblying 300 probes in region: Fgg\n",
      "--- assemblying 300 probes in region: Foxa1\n",
      "--- assemblying 300 probes in region: Foxa2\n",
      "--- assemblying 300 probes in region: Foxa3\n",
      "--- assemblying 300 probes in region: Foxj1\n",
      "--- assemblying 300 probes in region: Fxyd3\n",
      "--- assemblying 300 probes in region: Fxyd6\n",
      "--- assemblying 300 probes in region: Gabrp\n",
      "--- assemblying 300 probes in region: Gal\n",
      "--- assemblying 300 probes in region: Gatm\n",
      "--- assemblying 300 probes in region: Ghrl\n",
      "--- assemblying 228 probes in region: Gm13539\n",
      "--- assemblying 300 probes in region: Gm2a\n",
      "--- assemblying 300 probes in region: Gm94\n",
      "--- assemblying 300 probes in region: Gp2\n",
      "--- assemblying 300 probes in region: Gpx3\n",
      "--- assemblying 300 probes in region: Gpx6\n",
      "--- assemblying 300 probes in region: Grhl1\n",
      "--- assemblying 300 probes in region: Gsdma\n",
      "--- assemblying 274 probes in region: Gsta2\n",
      "--- assemblying 300 probes in region: Gsta4\n",
      "--- assemblying 300 probes in region: Gstm3\n",
      "--- assemblying 300 probes in region: Gsto1\n",
      "--- assemblying 300 probes in region: Guca2a\n",
      "--- assemblying 300 probes in region: H19\n",
      "--- assemblying 238 probes in region: Hamp2\n",
      "--- assemblying 300 probes in region: Hba-a2\n",
      "--- assemblying 300 probes in region: Hbb-y\n",
      "--- assemblying 300 probes in region: Hck\n",
      "--- assemblying 300 probes in region: Hnf4a\n",
      "--- assemblying 300 probes in region: Hnf4g\n",
      "--- assemblying 300 probes in region: Hp\n",
      "--- assemblying 300 probes in region: Iapp\n",
      "--- assemblying 245 probes in region: Idi2\n",
      "--- assemblying 193 probes in region: Ifi27l2b\n",
      "--- assemblying 300 probes in region: Igf1\n",
      "--- assemblying 300 probes in region: Igf2\n",
      "--- assemblying 300 probes in region: Igfbp5\n",
      "--- assemblying 300 probes in region: Igfbp6\n",
      "--- assemblying 300 probes in region: Ins2\n",
      "--- assemblying 300 probes in region: Isg15\n",
      "--- assemblying 300 probes in region: Itga6\n",
      "--- assemblying 300 probes in region: Jpt1\n",
      "--- assemblying 300 probes in region: Klk14\n",
      "--- assemblying 300 probes in region: Klk7\n",
      "--- assemblying 300 probes in region: Krt10\n",
      "--- assemblying 300 probes in region: Krt13\n",
      "--- assemblying 195 probes in region: Krt14\n",
      "--- assemblying 109 probes in region: Krt16\n",
      "--- assemblying 300 probes in region: Krt17\n",
      "--- assemblying 300 probes in region: Krt19\n",
      "--- assemblying 300 probes in region: Krt5\n",
      "--- assemblying 300 probes in region: Krt71\n",
      "--- assemblying 300 probes in region: Krt76\n",
      "--- assemblying 300 probes in region: Krt77\n",
      "--- assemblying 300 probes in region: Krt9\n",
      "--- assemblying 300 probes in region: Krtdap\n",
      "--- assemblying 300 probes in region: Lcn2\n",
      "--- assemblying 300 probes in region: Lcn3\n",
      "--- assemblying 300 probes in region: Lgals2\n",
      "--- assemblying 300 probes in region: Lgals3\n",
      "--- assemblying 300 probes in region: Lifr\n",
      "--- assemblying 300 probes in region: Ltf\n",
      "--- assemblying 126 probes in region: Ly6c1\n",
      "--- assemblying 147 probes in region: Ly6c2\n",
      "--- assemblying 300 probes in region: Ly6d\n",
      "--- assemblying 300 probes in region: Ly6g6f\n",
      "--- assemblying 300 probes in region: Lypd2\n",
      "--- assemblying 300 probes in region: Lypd3\n",
      "--- assemblying 300 probes in region: Mal\n",
      "--- assemblying 300 probes in region: Meg3\n",
      "--- assemblying 300 probes in region: Meig1\n",
      "--- assemblying 300 probes in region: Mgp\n",
      "--- assemblying 300 probes in region: Mlph\n",
      "--- assemblying 300 probes in region: Mpo\n",
      "--- assemblying 187 probes in region: Mt3\n",
      "--- assemblying 300 probes in region: Mt4\n",
      "--- assemblying 300 probes in region: Muc13\n",
      "--- assemblying 300 probes in region: Mymx\n",
      "--- assemblying 300 probes in region: Myog\n",
      "--- assemblying 300 probes in region: Nefh\n",
      "--- assemblying 300 probes in region: Ngp\n",
      "--- assemblying 300 probes in region: Nppa\n",
      "--- assemblying 300 probes in region: Npy\n",
      "--- assemblying 300 probes in region: Nrgn\n",
      "--- assemblying 300 probes in region: Nts\n",
      "--- assemblying 261 probes in region: Oas1d\n",
      "--- assemblying 300 probes in region: Oas1h\n",
      "--- assemblying 300 probes in region: Odf3b\n",
      "--- assemblying 300 probes in region: Olfm1\n",
      "--- assemblying 300 probes in region: Pcp4\n",
      "--- assemblying 300 probes in region: Pdzk1ip1\n",
      "--- assemblying 300 probes in region: Penk\n",
      "--- assemblying 300 probes in region: Pgc\n",
      "--- assemblying 300 probes in region: Pigr\n",
      "--- assemblying 300 probes in region: Pik3r5\n",
      "--- assemblying 300 probes in region: Pla2g1b\n",
      "--- assemblying 300 probes in region: Plac8\n",
      "--- assemblying 300 probes in region: Plk2\n",
      "--- assemblying 300 probes in region: Plvap\n",
      "--- assemblying 300 probes in region: Pomc\n",
      "--- assemblying 300 probes in region: Pou2f3\n",
      "--- assemblying 300 probes in region: Psca\n",
      "--- assemblying 300 probes in region: Ptgds\n",
      "--- assemblying 300 probes in region: Pyy\n",
      "--- assemblying 300 probes in region: Rap1gap2\n",
      "--- assemblying 300 probes in region: Rarres1\n",
      "--- assemblying 300 probes in region: Reg3b\n",
      "--- assemblying 300 probes in region: Reg4\n",
      "--- assemblying 246 probes in region: Retnla\n",
      "--- assemblying 300 probes in region: Riiad1\n",
      "--- assemblying 228 probes in region: Rnase1\n",
      "--- assemblying 300 probes in region: Rnase4\n",
      "--- assemblying 300 probes in region: Rsad2\n",
      "--- assemblying 300 probes in region: Rsph1\n",
      "--- assemblying 300 probes in region: S100a9\n",
      "--- assemblying 135 probes in region: Saa1\n",
      "--- assemblying 300 probes in region: Saa2\n",
      "--- assemblying 300 probes in region: Serpinb11\n",
      "--- assemblying 300 probes in region: Sftpa1\n",
      "--- assemblying 300 probes in region: Sftpd\n",
      "--- assemblying 300 probes in region: Slpi\n",
      "--- assemblying 300 probes in region: Sox8\n",
      "--- assemblying 300 probes in region: Spag8\n",
      "--- assemblying 300 probes in region: Sparc\n",
      "--- assemblying 300 probes in region: Spib\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 300 probes in region: Spp1\n",
      "--- assemblying 300 probes in region: Sprr1b\n",
      "--- assemblying 300 probes in region: Sprr2i\n",
      "--- assemblying 300 probes in region: Sst\n",
      "--- assemblying 300 probes in region: Stxbp5l\n",
      "--- assemblying 300 probes in region: Syt7\n",
      "--- assemblying 300 probes in region: Tac1\n",
      "--- assemblying 300 probes in region: Tac2\n",
      "--- assemblying 300 probes in region: Tacstd2\n",
      "--- assemblying 196 probes in region: Tff2\n",
      "--- assemblying 300 probes in region: Tff3\n",
      "--- assemblying 300 probes in region: Timp2\n",
      "--- assemblying 300 probes in region: Tmem45a\n",
      "--- assemblying 300 probes in region: Tnfrsf11b\n",
      "--- assemblying 300 probes in region: Trp63\n",
      "--- assemblying 300 probes in region: Trpm5\n",
      "--- assemblying 300 probes in region: Tspan1\n",
      "--- assemblying 300 probes in region: Ttc36\n",
      "--- assemblying 300 probes in region: Vpreb3\n",
      "--- assemblying 300 probes in region: Xcl1\n",
      "--- assemblying 300 probes in region: Zg16\n",
      "-- 60362 probes assembled in total.\n",
      "-- saving 60362 probes into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\candidate_probes.fasta\n",
      "-- saving readout_summary into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\readout_summary.pkl\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.assemble)\n",
    "from ImageAnalysis3.library_tools.assemble import Assemble_probes\n",
    "\n",
    "cand_probes, readout_summary = Assemble_probes(library_folder, filtered_pb_dict, \n",
    "                                               gene_2_readout_dict, readout_dict, primers, \n",
    "                                               primer_len=20, \n",
    "                                               num_readout_per_probe=3, \n",
    "                                               unique_readout_per_probe=True,\n",
    "                                               rc_targets=False, overwrite=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "behavioral-metro",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'loc_9:104322748-104352748_gene_Acpp:-_pb_0_pos_15_strand_+_readouts_[NDB_401_u,NDB_401_u,NDB_401_u]'"
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cand_probes[0].id"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "national-subscriber",
   "metadata": {},
   "source": [
    "# 4.Check quality"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "welsh-financing",
   "metadata": {},
   "source": [
    "## 4.1 Load probes\n",
    "\n",
    "Checking procedures could restart from here"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "alone-husband",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total probe loaded: 60362\n"
     ]
    }
   ],
   "source": [
    "# folder for this library\n",
    "# candidate full-length probe filename\n",
    "candidate_basename = 'candidate_probes.fasta' # old version\n",
    "# load full probes\n",
    "full_records = []\n",
    "with open(os.path.join(library_folder, candidate_basename), 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        full_records.append(record)\n",
    "\n",
    "print(f\"Total probe loaded: {len(full_records)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "magnetic-niagara",
   "metadata": {},
   "outputs": [],
   "source": [
    "num_readout_per_probe = 3\n",
    "primer_len=20\n",
    "target_len=40\n",
    "min_num_probes = 100"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "noticed-raise",
   "metadata": {},
   "source": [
    "### 4.1.1 check primer usage"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "equipped-testament",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Checking primer usage, total probes: 60362\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "primer_check = check._check_primer_usage(full_records, primers[0], primers[1], primer_len=primer_len)\n",
    "print(primer_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "small-packaging",
   "metadata": {},
   "source": [
    "### 4.1.2 check region probe number"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "practical-mechanism",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: Acpp:- -> 300\n",
      "gene: Agt:- -> 300\n",
      "gene: Aldh3a1:+ -> 300\n",
      "gene: Aldob:- -> 300\n",
      "gene: Alox5ap:+ -> 300\n",
      "gene: Alpi:- -> 300\n",
      "gene: Alpl:- -> 300\n",
      "gene: Ang4:- -> 175\n",
      "gene: Anxa8:+ -> 300\n",
      "gene: Apoa1:+ -> 300\n",
      "gene: Apoa2:+ -> 300\n",
      "gene: Apoc2:- -> 300\n",
      "gene: Apod:- -> 300\n",
      "gene: Ascl1:- -> 300\n",
      "gene: Avil:+ -> 300\n",
      "gene: Bgn:+ -> 300\n",
      "gene: C1qb:- -> 300\n",
      "gene: C3:- -> 300\n",
      "gene: Calb2:- -> 300\n",
      "gene: Cald1:+ -> 300\n",
      "gene: Camp:- -> 300\n",
      "gene: Car4:+ -> 300\n",
      "gene: Car8:- -> 300\n",
      "gene: Ccdc30:- -> 300\n",
      "gene: Cck:- -> 300\n",
      "gene: Ccl1:- -> 300\n",
      "gene: Ccl2:+ -> 300\n",
      "gene: Ccl20:+ -> 300\n",
      "gene: Ccl6:- -> 200\n",
      "gene: Ccl9:- -> 300\n",
      "gene: Cd14:- -> 300\n",
      "gene: Cebpb:+ -> 300\n",
      "gene: Cga:+ -> 300\n",
      "gene: Cilp:+ -> 300\n",
      "gene: Ckb:- -> 300\n",
      "gene: Ckm:+ -> 300\n",
      "gene: Clps:- -> 300\n",
      "gene: Cnfn:- -> 300\n",
      "gene: Cnmd:- -> 300\n",
      "gene: Col1a1:+ -> 300\n",
      "gene: Col4a1:- -> 300\n",
      "gene: Col6a1:- -> 300\n",
      "gene: Cpn1:- -> 300\n",
      "gene: Crabp1:+ -> 300\n",
      "gene: Cryaa:+ -> 300\n",
      "gene: Ctrb1:- -> 300\n",
      "gene: Cxcl13:+ -> 300\n",
      "gene: Cyp1a2:- -> 300\n",
      "gene: Cyp2f2:+ -> 300\n",
      "gene: Dbn1:- -> 300\n",
      "gene: Dclk1:+ -> 300\n",
      "gene: Dlk1:+ -> 300\n",
      "gene: Dppa3:+ -> 279\n",
      "gene: Ecm1:- -> 300\n",
      "gene: Efna3:- -> 300\n",
      "gene: Elovl4:- -> 300\n",
      "gene: Fabp1:+ -> 300\n",
      "gene: Fabp6:- -> 300\n",
      "gene: Fam183b:- -> 300\n",
      "gene: Fam25c:- -> 300\n",
      "gene: Fgg:+ -> 300\n",
      "gene: Foxa1:- -> 300\n",
      "gene: Foxa2:- -> 300\n",
      "gene: Foxa3:- -> 300\n",
      "gene: Foxj1:- -> 300\n",
      "gene: Fxyd3:- -> 300\n",
      "gene: Fxyd6:+ -> 300\n",
      "gene: Gabrp:- -> 300\n",
      "gene: Gal:- -> 300\n",
      "gene: Gatm:- -> 300\n",
      "gene: Ghrl:- -> 300\n",
      "gene: Gm13539:+ -> 228\n",
      "gene: Gm2a:+ -> 300\n",
      "gene: Gm94:- -> 300\n",
      "gene: Gp2:- -> 300\n",
      "gene: Gpx3:+ -> 300\n",
      "gene: Gpx6:+ -> 300\n",
      "gene: Grhl1:+ -> 300\n",
      "gene: Gsdma:+ -> 300\n",
      "gene: Gsta2:- -> 274\n",
      "gene: Gsta4:+ -> 300\n",
      "gene: Gstm3:- -> 300\n",
      "gene: Gsto1:+ -> 300\n",
      "gene: Guca2a:+ -> 300\n",
      "gene: H19:- -> 300\n",
      "gene: Hamp2:- -> 238\n",
      "gene: Hba-a2:+ -> 300\n",
      "gene: Hbb-y:- -> 300\n",
      "gene: Hck:+ -> 300\n",
      "gene: Hnf4a:+ -> 300\n",
      "gene: Hnf4g:+ -> 300\n",
      "gene: Hp:- -> 300\n",
      "gene: Iapp:+ -> 300\n",
      "gene: Idi2:+ -> 245\n",
      "gene: Ifi27l2b:- -> 193\n",
      "gene: Igf1:+ -> 300\n",
      "gene: Igf2:- -> 300\n",
      "gene: Igfbp5:- -> 300\n",
      "gene: Igfbp6:+ -> 300\n",
      "gene: Ins2:- -> 300\n",
      "gene: Isg15:- -> 300\n",
      "gene: Itga6:+ -> 300\n",
      "gene: Jpt1:- -> 300\n",
      "gene: Klk14:+ -> 300\n",
      "gene: Klk7:+ -> 300\n",
      "gene: Krt10:- -> 300\n",
      "gene: Krt13:- -> 300\n",
      "gene: Krt14:- -> 195\n",
      "gene: Krt16:- -> 109\n",
      "gene: Krt17:- -> 300\n",
      "gene: Krt19:- -> 300\n",
      "gene: Krt5:- -> 300\n",
      "gene: Krt71:- -> 300\n",
      "gene: Krt76:- -> 300\n",
      "gene: Krt77:- -> 300\n",
      "gene: Krt9:- -> 300\n",
      "gene: Krtdap:+ -> 300\n",
      "gene: Lcn2:- -> 300\n",
      "gene: Lcn3:+ -> 300\n",
      "gene: Lgals2:- -> 300\n",
      "gene: Lgals3:+ -> 300\n",
      "gene: Lifr:+ -> 300\n",
      "gene: Ltf:+ -> 300\n",
      "gene: Ly6c1:- -> 126\n",
      "gene: Ly6c2:- -> 147\n",
      "gene: Ly6d:- -> 300\n",
      "gene: Ly6g6f:- -> 300\n",
      "gene: Lypd2:- -> 300\n",
      "gene: Lypd3:+ -> 300\n",
      "gene: Mal:- -> 300\n",
      "gene: Meg3:+ -> 300\n",
      "gene: Meig1:- -> 300\n",
      "gene: Mgp:- -> 300\n",
      "gene: Mlph:+ -> 300\n",
      "gene: Mpo:+ -> 300\n",
      "gene: Mt3:+ -> 187\n",
      "gene: Mt4:+ -> 300\n",
      "gene: Muc13:+ -> 300\n",
      "gene: Mymx:- -> 300\n",
      "gene: Myog:+ -> 300\n",
      "gene: Nefh:- -> 300\n",
      "gene: Ngp:+ -> 300\n",
      "gene: Nppa:+ -> 300\n",
      "gene: Npy:+ -> 300\n",
      "gene: Nrgn:- -> 300\n",
      "gene: Nts:- -> 300\n",
      "gene: Oas1d:+ -> 261\n",
      "gene: Oas1h:+ -> 300\n",
      "gene: Odf3b:- -> 300\n",
      "gene: Olfm1:+ -> 300\n",
      "gene: Pcp4:+ -> 300\n",
      "gene: Pdzk1ip1:+ -> 300\n",
      "gene: Penk:- -> 300\n",
      "gene: Pgc:+ -> 300\n",
      "gene: Pigr:+ -> 300\n",
      "gene: Pik3r5:+ -> 300\n",
      "gene: Pla2g1b:+ -> 300\n",
      "gene: Plac8:- -> 300\n",
      "gene: Plk2:+ -> 300\n",
      "gene: Plvap:- -> 300\n",
      "gene: Pomc:+ -> 300\n",
      "gene: Pou2f3:- -> 300\n",
      "gene: Psca:+ -> 300\n",
      "gene: Ptgds:- -> 300\n",
      "gene: Pyy:- -> 300\n",
      "gene: Rap1gap2:- -> 300\n",
      "gene: Rarres1:- -> 300\n",
      "gene: Reg3b:+ -> 300\n",
      "gene: Reg4:+ -> 300\n",
      "gene: Retnla:+ -> 246\n",
      "gene: Riiad1:- -> 300\n",
      "gene: Rnase1:- -> 228\n",
      "gene: Rnase4:+ -> 300\n",
      "gene: Rsad2:- -> 300\n",
      "gene: Rsph1:- -> 300\n",
      "gene: S100a9:- -> 300\n",
      "gene: Saa1:- -> 135\n",
      "gene: Saa2:+ -> 300\n",
      "gene: Serpinb11:+ -> 300\n",
      "gene: Sftpa1:+ -> 300\n",
      "gene: Sftpd:- -> 300\n",
      "gene: Slpi:- -> 300\n",
      "gene: Sox8:- -> 300\n",
      "gene: Spag8:- -> 300\n",
      "gene: Sparc:- -> 300\n",
      "gene: Spib:- -> 300\n",
      "gene: Spp1:+ -> 300\n",
      "gene: Sprr1b:- -> 300\n",
      "gene: Sprr2i:+ -> 300\n",
      "gene: Sst:- -> 300\n",
      "gene: Stxbp5l:- -> 300\n",
      "gene: Syt7:+ -> 300\n",
      "gene: Tac1:+ -> 300\n",
      "gene: Tac2:+ -> 300\n",
      "gene: Tacstd2:- -> 300\n",
      "gene: Tff2:- -> 196\n",
      "gene: Tff3:- -> 300\n",
      "gene: Timp2:- -> 300\n",
      "gene: Tmem45a:- -> 300\n",
      "gene: Tnfrsf11b:- -> 300\n",
      "gene: Trp63:+ -> 300\n",
      "gene: Trpm5:- -> 300\n",
      "gene: Tspan1:- -> 300\n",
      "gene: Ttc36:- -> 300\n",
      "gene: Vpreb3:+ -> 300\n",
      "gene: Xcl1:- -> 300\n",
      "gene: Zg16:- -> 300\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "reg_size_dic, len_check = check._check_region_size(full_records, \n",
    "                                                   min_size=min_num_probes)\n",
    "print(len_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "statistical-evening",
   "metadata": {},
   "source": [
    "### 4.1.3 check readout usage for each gene"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "id": "amber-thong",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "3"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "num_readout_per_probe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "considerable-episode",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Acpp:- -> ['NDB_401', 'NDB_812', 'NDB_860', 'NDB_878']\n",
      "Agt:- -> ['NDB_346', 'NDB_754', 'NDB_773', 'NDB_817']\n",
      "Aldh3a1:+ -> ['NDB_608', 'NDB_860', 'NDB_961', 'NDB_974']\n",
      "Aldob:- -> ['NDB_376', 'NDB_743', 'NDB_844', 'NDB_863']\n",
      "Alox5ap:+ -> ['NDB_340', 'NDB_634', 'NDB_971', 'NDB_1033']\n",
      "Alpi:- -> ['NDB_595', 'NDB_844', 'NDB_860', 'NDB_865']\n",
      "Alpl:- -> ['NDB_584', 'NDB_713', 'NDB_856', 'NDB_971']\n",
      "Ang4:- -> ['NDB_589', 'NDB_715', 'NDB_754', 'NDB_860']\n",
      "Anxa8:+ -> ['NDB_419', 'NDB_680', 'NDB_728', 'NDB_901']\n",
      "Apoa1:+ -> ['NDB_382', 'NDB_773', 'NDB_917', 'NDB_971']\n",
      "Apoa2:+ -> ['NDB_566', 'NDB_631', 'NDB_643', 'NDB_773']\n",
      "Apoc2:- -> ['NDB_523', 'NDB_683', 'NDB_790', 'NDB_865']\n",
      "Apod:- -> ['NDB_497', 'NDB_737', 'NDB_773', 'NDB_844']\n",
      "Ascl1:- -> ['NDB_326', 'NDB_661', 'NDB_698', 'NDB_773']\n",
      "Avil:+ -> ['NDB_338', 'NDB_653', 'NDB_743', 'NDB_815']\n",
      "Bgn:+ -> ['NDB_572', 'NDB_680', 'NDB_784', 'NDB_971']\n",
      "C1qb:- -> ['NDB_581', 'NDB_698', 'NDB_826', 'NDB_875']\n",
      "C3:- -> ['NDB_331', 'NDB_778', 'NDB_875', 'NDB_971']\n",
      "Calb2:- -> ['NDB_605', 'NDB_652', 'NDB_746', 'NDB_760']\n",
      "Cald1:+ -> ['NDB_328', 'NDB_718', 'NDB_737', 'NDB_974']\n",
      "Camp:- -> ['NDB_512', 'NDB_634', 'NDB_653', 'NDB_680']\n",
      "Car4:+ -> ['NDB_575', 'NDB_661', 'NDB_815', 'NDB_868']\n",
      "Car8:- -> ['NDB_440', 'NDB_631', 'NDB_634', 'NDB_847']\n",
      "Ccdc30:- -> ['NDB_430', 'NDB_643', 'NDB_653', 'NDB_817']\n",
      "Cck:- -> ['NDB_487', 'NDB_716', 'NDB_875', 'NDB_883']\n",
      "Ccl1:- -> ['NDB_547', 'NDB_746', 'NDB_812', 'NDB_1033']\n",
      "Ccl2:+ -> ['NDB_578', 'NDB_725', 'NDB_784', 'NDB_817']\n",
      "Ccl20:+ -> ['NDB_353', 'NDB_683', 'NDB_713', 'NDB_716']\n",
      "Ccl6:- -> ['NDB_356', 'NDB_710', 'NDB_863', 'NDB_901']\n",
      "Ccl9:- -> ['NDB_355', 'NDB_643', 'NDB_680', 'NDB_1075']\n",
      "Cd14:- -> ['NDB_422', 'NDB_728', 'NDB_847', 'NDB_860']\n",
      "Cebpb:+ -> ['NDB_409', 'NDB_718', 'NDB_743', 'NDB_878']\n",
      "Cga:+ -> ['NDB_508', 'NDB_725', 'NDB_754', 'NDB_868']\n",
      "Cilp:+ -> ['NDB_520', 'NDB_790', 'NDB_1027', 'NDB_1075']\n",
      "Ckb:- -> ['NDB_458', 'NDB_856', 'NDB_865', 'NDB_1033']\n",
      "Ckm:+ -> ['NDB_463', 'NDB_713', 'NDB_815', 'NDB_901']\n",
      "Clps:- -> ['NDB_623', 'NDB_758', 'NDB_817', 'NDB_856']\n",
      "Cnfn:- -> ['NDB_403', 'NDB_718', 'NDB_844', 'NDB_965']\n",
      "Cnmd:- -> ['NDB_587', 'NDB_760', 'NDB_815', 'NDB_847']\n",
      "Col1a1:+ -> ['NDB_625', 'NDB_728', 'NDB_844', 'NDB_878']\n",
      "Col4a1:- -> ['NDB_325', 'NDB_698', 'NDB_718', 'NDB_971']\n",
      "Col6a1:- -> ['NDB_320', 'NDB_826', 'NDB_863', 'NDB_974']\n",
      "Cpn1:- -> ['NDB_448', 'NDB_778', 'NDB_817', 'NDB_965']\n",
      "Crabp1:+ -> ['NDB_476', 'NDB_710', 'NDB_755', 'NDB_865']\n",
      "Cryaa:+ -> ['NDB_556', 'NDB_790', 'NDB_961', 'NDB_1033']\n",
      "Ctrb1:- -> ['NDB_499', 'NDB_634', 'NDB_784', 'NDB_875']\n",
      "Cxcl13:+ -> ['NDB_538', 'NDB_680', 'NDB_826', 'NDB_1027']\n",
      "Cyp1a2:- -> ['NDB_514', 'NDB_698', 'NDB_974', 'NDB_1033']\n",
      "Cyp2f2:+ -> ['NDB_416', 'NDB_652', 'NDB_710', 'NDB_784']\n",
      "Dbn1:- -> ['NDB_467', 'NDB_652', 'NDB_718', 'NDB_758']\n",
      "Dclk1:+ -> ['NDB_334', 'NDB_683', 'NDB_755', 'NDB_817']\n",
      "Dlk1:+ -> ['NDB_515', 'NDB_661', 'NDB_863', 'NDB_875']\n",
      "Dppa3:+ -> ['NDB_550', 'NDB_754', 'NDB_760', 'NDB_961']\n",
      "Ecm1:- -> ['NDB_541', 'NDB_718', 'NDB_826', 'NDB_961']\n",
      "Efna3:- -> ['NDB_407', 'NDB_643', 'NDB_790', 'NDB_860']\n",
      "Elovl4:- -> ['NDB_406', 'NDB_661', 'NDB_713', 'NDB_965']\n",
      "Fabp1:+ -> ['NDB_361', 'NDB_743', 'NDB_758', 'NDB_1033']\n",
      "Fabp6:- -> ['NDB_472', 'NDB_743', 'NDB_784', 'NDB_847']\n",
      "Fam183b:- -> ['NDB_428', 'NDB_713', 'NDB_755', 'NDB_778']\n",
      "Fam25c:- -> ['NDB_404', 'NDB_755', 'NDB_868', 'NDB_971']\n",
      "Fgg:+ -> ['NDB_562', 'NDB_778', 'NDB_784', 'NDB_856']\n",
      "Foxa1:- -> ['NDB_424', 'NDB_725', 'NDB_743', 'NDB_755']\n",
      "Foxa2:- -> ['NDB_457', 'NDB_847', 'NDB_974', 'NDB_1027']\n",
      "Foxa3:- -> ['NDB_455', 'NDB_661', 'NDB_878', 'NDB_1033']\n",
      "Foxj1:- -> ['NDB_427', 'NDB_815', 'NDB_917', 'NDB_1033']\n",
      "Fxyd3:- -> ['NDB_365', 'NDB_661', 'NDB_817', 'NDB_961']\n",
      "Fxyd6:+ -> ['NDB_350', 'NDB_631', 'NDB_743', 'NDB_754']\n",
      "Gabrp:- -> ['NDB_415', 'NDB_715', 'NDB_844', 'NDB_971']\n",
      "Gal:- -> ['NDB_475', 'NDB_634', 'NDB_643', 'NDB_754']\n",
      "Gatm:- -> ['NDB_532', 'NDB_713', 'NDB_725', 'NDB_737']\n",
      "Ghrl:- -> ['NDB_596', 'NDB_634', 'NDB_878', 'NDB_965']\n",
      "Gm13539:+ -> ['NDB_517', 'NDB_680', 'NDB_790', 'NDB_856']\n",
      "Gm2a:+ -> ['NDB_599', 'NDB_653', 'NDB_683', 'NDB_1027']\n",
      "Gm94:- -> ['NDB_400', 'NDB_725', 'NDB_856', 'NDB_1027']\n",
      "Gp2:- -> ['NDB_359', 'NDB_743', 'NDB_760', 'NDB_875']\n",
      "Gpx3:+ -> ['NDB_454', 'NDB_718', 'NDB_754', 'NDB_883']\n",
      "Gpx6:+ -> ['NDB_626', 'NDB_643', 'NDB_865', 'NDB_883']\n",
      "Grhl1:+ -> ['NDB_388', 'NDB_713', 'NDB_728', 'NDB_758']\n",
      "Gsdma:+ -> ['NDB_620', 'NDB_634', 'NDB_652', 'NDB_790']\n",
      "Gsta2:- -> ['NDB_379', 'NDB_847', 'NDB_863', 'NDB_961']\n",
      "Gsta4:+ -> ['NDB_412', 'NDB_760', 'NDB_778', 'NDB_868']\n",
      "Gstm3:- -> ['NDB_491', 'NDB_754', 'NDB_812', 'NDB_971']\n",
      "Gsto1:+ -> ['NDB_418', 'NDB_715', 'NDB_773', 'NDB_863']\n",
      "Guca2a:+ -> ['NDB_370', 'NDB_728', 'NDB_760', 'NDB_917']\n",
      "H19:- -> ['NDB_535', 'NDB_737', 'NDB_778', 'NDB_961']\n",
      "Hamp2:- -> ['NDB_598', 'NDB_715', 'NDB_746', 'NDB_847']\n",
      "Hba-a2:+ -> ['NDB_551', 'NDB_643', 'NDB_652', 'NDB_965']\n",
      "Hbb-y:- -> ['NDB_545', 'NDB_634', 'NDB_725', 'NDB_773']\n",
      "Hck:+ -> ['NDB_341', 'NDB_784', 'NDB_901', 'NDB_965']\n",
      "Hnf4a:+ -> ['NDB_383', 'NDB_631', 'NDB_860', 'NDB_1033']\n",
      "Hnf4g:+ -> ['NDB_385', 'NDB_631', 'NDB_710', 'NDB_716']\n",
      "Hp:- -> ['NDB_580', 'NDB_643', 'NDB_743', 'NDB_961']\n",
      "Iapp:+ -> ['NDB_509', 'NDB_847', 'NDB_883', 'NDB_917']\n",
      "Idi2:+ -> ['NDB_569', 'NDB_653', 'NDB_737', 'NDB_917']\n",
      "Ifi27l2b:- -> ['NDB_368', 'NDB_653', 'NDB_860', 'NDB_901']\n",
      "Igf1:+ -> ['NDB_386', 'NDB_683', 'NDB_875', 'NDB_878']\n",
      "Igf2:- -> ['NDB_593', 'NDB_631', 'NDB_790', 'NDB_971']\n",
      "Igfbp5:- -> ['NDB_446', 'NDB_661', 'NDB_758', 'NDB_971']\n",
      "Igfbp6:+ -> ['NDB_521', 'NDB_961', 'NDB_971', 'NDB_1027']\n",
      "Ins2:- -> ['NDB_506', 'NDB_812', 'NDB_844', 'NDB_974']\n",
      "Isg15:- -> ['NDB_604', 'NDB_652', 'NDB_737', 'NDB_901']\n",
      "Itga6:+ -> ['NDB_323', 'NDB_653', 'NDB_760', 'NDB_844']\n",
      "Jpt1:- -> ['NDB_464', 'NDB_716', 'NDB_743', 'NDB_971']\n",
      "Klk14:+ -> ['NDB_560', 'NDB_653', 'NDB_755', 'NDB_883']\n",
      "Klk7:+ -> ['NDB_548', 'NDB_680', 'NDB_812', 'NDB_815']\n",
      "Krt10:- -> ['NDB_398', 'NDB_713', 'NDB_817', 'NDB_826']\n",
      "Krt13:- -> ['NDB_518', 'NDB_725', 'NDB_760', 'NDB_860']\n",
      "Krt14:- -> ['NDB_317', 'NDB_754', 'NDB_758', 'NDB_974']\n",
      "Krt16:- -> ['NDB_394', 'NDB_683', 'NDB_698', 'NDB_847']\n",
      "Krt17:- -> ['NDB_329', 'NDB_653', 'NDB_710', 'NDB_1075']\n",
      "Krt19:- -> ['NDB_590', 'NDB_773', 'NDB_856', 'NDB_883']\n",
      "Krt5:- -> ['NDB_319', 'NDB_683', 'NDB_754', 'NDB_965']\n",
      "Krt71:- -> ['NDB_554', 'NDB_710', 'NDB_715', 'NDB_817']\n",
      "Krt76:- -> ['NDB_619', 'NDB_725', 'NDB_878', 'NDB_901']\n",
      "Krt77:- -> ['NDB_586', 'NDB_643', 'NDB_737', 'NDB_812']\n",
      "Krt9:- -> ['NDB_601', 'NDB_875', 'NDB_901', 'NDB_1027']\n",
      "Krtdap:+ -> ['NDB_391', 'NDB_856', 'NDB_863', 'NDB_917']\n",
      "Lcn2:- -> ['NDB_380', 'NDB_698', 'NDB_817', 'NDB_917']\n",
      "Lcn3:+ -> ['NDB_611', 'NDB_716', 'NDB_755', 'NDB_815']\n",
      "Lgals2:- -> ['NDB_377', 'NDB_746', 'NDB_844', 'NDB_917']\n",
      "Lgals3:+ -> ['NDB_577', 'NDB_856', 'NDB_878', 'NDB_1075']\n",
      "Lifr:+ -> ['NDB_322', 'NDB_634', 'NDB_661', 'NDB_826']\n",
      "Ltf:+ -> ['NDB_622', 'NDB_728', 'NDB_737', 'NDB_784']\n",
      "Ly6c1:- -> ['NDB_544', 'NDB_631', 'NDB_815', 'NDB_856']\n",
      "Ly6c2:- -> ['NDB_536', 'NDB_728', 'NDB_790', 'NDB_863']\n",
      "Ly6d:- -> ['NDB_410', 'NDB_755', 'NDB_760', 'NDB_1075']\n",
      "Ly6g6f:- -> ['NDB_337', 'NDB_713', 'NDB_812', 'NDB_847']\n",
      "Lypd2:- -> ['NDB_500', 'NDB_865', 'NDB_868', 'NDB_875']\n",
      "Lypd3:+ -> ['NDB_389', 'NDB_698', 'NDB_778', 'NDB_860']\n",
      "Mal:- -> ['NDB_583', 'NDB_683', 'NDB_773', 'NDB_961']\n",
      "Meg3:+ -> ['NDB_460', 'NDB_652', 'NDB_778', 'NDB_826']\n",
      "Meig1:- -> ['NDB_431', 'NDB_643', 'NDB_778', 'NDB_863']\n",
      "Mgp:- -> ['NDB_592', 'NDB_860', 'NDB_863', 'NDB_1075']\n",
      "Mlph:+ -> ['NDB_421', 'NDB_710', 'NDB_725', 'NDB_965']\n",
      "Mpo:+ -> ['NDB_530', 'NDB_698', 'NDB_737', 'NDB_965']\n",
      "Mt3:+ -> ['NDB_553', 'NDB_815', 'NDB_965', 'NDB_974']\n",
      "Mt4:+ -> ['NDB_479', 'NDB_680', 'NDB_737', 'NDB_883']\n",
      "Muc13:+ -> ['NDB_373', 'NDB_631', 'NDB_746', 'NDB_817']\n",
      "Mymx:- -> ['NDB_466', 'NDB_653', 'NDB_784', 'NDB_868']\n",
      "Myog:+ -> ['NDB_461', 'NDB_737', 'NDB_875', 'NDB_1075']\n",
      "Nefh:- -> ['NDB_568', 'NDB_746', 'NDB_856', 'NDB_868']\n",
      "Ngp:+ -> ['NDB_574', 'NDB_683', 'NDB_826', 'NDB_856']\n",
      "Nppa:+ -> ['NDB_511', 'NDB_683', 'NDB_778', 'NDB_974']\n",
      "Npy:+ -> ['NDB_527', 'NDB_715', 'NDB_728', 'NDB_865']\n",
      "Nrgn:- -> ['NDB_347', 'NDB_725', 'NDB_746', 'NDB_883']\n",
      "Nts:- -> ['NDB_496', 'NDB_631', 'NDB_901', 'NDB_961']\n",
      "Oas1d:+ -> ['NDB_563', 'NDB_817', 'NDB_860', 'NDB_868']\n",
      "Oas1h:+ -> ['NDB_617', 'NDB_746', 'NDB_754', 'NDB_755']\n",
      "Odf3b:- -> ['NDB_439', 'NDB_698', 'NDB_758', 'NDB_883']\n",
      "Olfm1:+ -> ['NDB_571', 'NDB_634', 'NDB_868', 'NDB_883']\n",
      "Pcp4:+ -> ['NDB_493', 'NDB_778', 'NDB_878', 'NDB_917']\n",
      "Pdzk1ip1:+ -> ['NDB_395', 'NDB_680', 'NDB_965', 'NDB_1033']\n",
      "Penk:- -> ['NDB_539', 'NDB_718', 'NDB_784', 'NDB_1075']\n",
      "Pgc:+ -> ['NDB_502', 'NDB_652', 'NDB_844', 'NDB_1027']\n",
      "Pigr:+ -> ['NDB_371', 'NDB_743', 'NDB_826', 'NDB_901']\n",
      "Pik3r5:+ -> ['NDB_344', 'NDB_631', 'NDB_758', 'NDB_875']\n",
      "Pla2g1b:+ -> ['NDB_494', 'NDB_812', 'NDB_917', 'NDB_1075']\n",
      "Plac8:- -> ['NDB_349', 'NDB_710', 'NDB_778', 'NDB_847']\n",
      "Plk2:+ -> ['NDB_343', 'NDB_683', 'NDB_725', 'NDB_844']\n",
      "Plvap:- -> ['NDB_628', 'NDB_716', 'NDB_812', 'NDB_863']\n",
      "Pomc:+ -> ['NDB_481', 'NDB_710', 'NDB_718', 'NDB_868']\n",
      "Pou2f3:- -> ['NDB_352', 'NDB_715', 'NDB_716', 'NDB_758']\n",
      "Psca:+ -> ['NDB_488', 'NDB_653', 'NDB_713', 'NDB_974']\n",
      "Ptgds:- -> ['NDB_485', 'NDB_710', 'NDB_728', 'NDB_875']\n",
      "Pyy:- -> ['NDB_470', 'NDB_634', 'NDB_715', 'NDB_755']\n",
      "Rap1gap2:- -> ['NDB_449', 'NDB_718', 'NDB_728', 'NDB_773']\n",
      "Rarres1:- -> ['NDB_503', 'NDB_680', 'NDB_698', 'NDB_725']\n",
      "Reg3b:+ -> ['NDB_614', 'NDB_652', 'NDB_653', 'NDB_661']\n",
      "Reg4:+ -> ['NDB_602', 'NDB_737', 'NDB_746', 'NDB_1027']\n",
      "Retnla:+ -> ['NDB_505', 'NDB_683', 'NDB_812', 'NDB_883']\n",
      "Riiad1:- -> ['NDB_436', 'NDB_661', 'NDB_715', 'NDB_743']\n",
      "Rnase1:- -> ['NDB_358', 'NDB_631', 'NDB_653', 'NDB_718']\n",
      "Rnase4:+ -> ['NDB_442', 'NDB_710', 'NDB_1027', 'NDB_1033']\n",
      "Rsad2:- -> ['NDB_565', 'NDB_715', 'NDB_784', 'NDB_790']\n",
      "Rsph1:- -> ['NDB_437', 'NDB_680', 'NDB_746', 'NDB_878']\n",
      "S100a9:- -> ['NDB_469', 'NDB_713', 'NDB_1033', 'NDB_1075']\n",
      "Saa1:- -> ['NDB_374', 'NDB_754', 'NDB_826', 'NDB_1075']\n",
      "Saa2:+ -> ['NDB_529', 'NDB_643', 'NDB_758', 'NDB_1027']\n",
      "Serpinb11:+ -> ['NDB_413', 'NDB_784', 'NDB_863', 'NDB_883']\n",
      "Sftpa1:+ -> ['NDB_616', 'NDB_725', 'NDB_790', 'NDB_917']\n",
      "Sftpd:- -> ['NDB_610', 'NDB_716', 'NDB_826', 'NDB_844']\n",
      "Slpi:- -> ['NDB_473', 'NDB_652', 'NDB_715', 'NDB_812']\n",
      "Sox8:- -> ['NDB_362', 'NDB_634', 'NDB_865', 'NDB_974']\n",
      "Spag8:- -> ['NDB_433', 'NDB_755', 'NDB_773', 'NDB_790']\n",
      "Sparc:- -> ['NDB_524', 'NDB_716', 'NDB_865', 'NDB_917']\n",
      "Spib:- -> ['NDB_364', 'NDB_631', 'NDB_652', 'NDB_878']\n",
      "Spp1:+ -> ['NDB_478', 'NDB_713', 'NDB_784', 'NDB_865']\n",
      "Sprr1b:- -> ['NDB_542', 'NDB_652', 'NDB_773', 'NDB_868']\n",
      "Sprr2i:+ -> ['NDB_613', 'NDB_815', 'NDB_865', 'NDB_878']\n",
      "Sst:- -> ['NDB_484', 'NDB_634', 'NDB_961', 'NDB_1075']\n",
      "Stxbp5l:- -> ['NDB_445', 'NDB_643', 'NDB_760', 'NDB_974']\n",
      "Syt7:+ -> ['NDB_452', 'NDB_737', 'NDB_758', 'NDB_878']\n",
      "Tac1:+ -> ['NDB_526', 'NDB_698', 'NDB_812', 'NDB_901']\n",
      "Tac2:+ -> ['NDB_490', 'NDB_716', 'NDB_790', 'NDB_847']\n",
      "Tacstd2:- -> ['NDB_392', 'NDB_746', 'NDB_773', 'NDB_826']\n",
      "Tff2:- -> ['NDB_557', 'NDB_661', 'NDB_755', 'NDB_917']\n",
      "Tff3:- -> ['NDB_482', 'NDB_683', 'NDB_715', 'NDB_718']\n",
      "Timp2:- -> ['NDB_559', 'NDB_760', 'NDB_826', 'NDB_865']\n",
      "Tmem45a:- -> ['NDB_397', 'NDB_698', 'NDB_728', 'NDB_754']\n",
      "Tnfrsf11b:- -> ['NDB_367', 'NDB_680', 'NDB_716', 'NDB_778']\n",
      "Trp63:+ -> ['NDB_332', 'NDB_715', 'NDB_965', 'NDB_1027']\n",
      "Trpm5:- -> ['NDB_335', 'NDB_680', 'NDB_817', 'NDB_863']\n",
      "Tspan1:- -> ['NDB_443', 'NDB_746', 'NDB_758', 'NDB_865']\n",
      "Ttc36:- -> ['NDB_425', 'NDB_643', 'NDB_718', 'NDB_815']\n",
      "Vpreb3:+ -> ['NDB_434', 'NDB_760', 'NDB_856', 'NDB_965']\n",
      "Xcl1:- -> ['NDB_533', 'NDB_728', 'NDB_815', 'NDB_1027']\n",
      "Zg16:- -> ['NDB_607', 'NDB_716', 'NDB_728', 'NDB_868']\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "check_readout_dict = pickle.load(open(os.path.join(library_folder, 'readout_usage.pkl'), 'rb'))\n",
    "\n",
    "reg_readout_dic, reg2readout_check = check._check_region_to_readouts(full_records, check_readout_dict, \n",
    "                                                                     total_readout_num=num_readout_per_probe+1)\n",
    "print(reg2readout_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "regular-samba",
   "metadata": {},
   "source": [
    "### 4.1.4 check readout usage in the library"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "id": "funny-vanilla",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- extracting probes from >region:Acpp:-, region:Agt:-, region:Aldh3a1:+, region:Aldob:-, region:Alox5ap:+, region:Alpi:-, region:Alpl:-, region:Ang4:-, region:Anxa8:+, region:Apoa1:+, region:Apoa2:+, region:Apoc2:-, region:Apod:-, region:Ascl1:-, region:Avil:+, region:Bgn:+, region:C1qb:-, region:C3:-, region:Calb2:-, region:Cald1:+, region:Camp:-, region:Car4:+, region:Car8:-, region:Ccdc30:-, region:Cck:-, region:Ccl1:-, region:Ccl2:+, region:Ccl20:+, region:Ccl6:-, region:Ccl9:-, region:Cd14:-, region:Cebpb:+, region:Cga:+, region:Cilp:+, region:Ckb:-, region:Ckm:+, region:Clps:-, region:Cnfn:-, region:Cnmd:-, region:Col1a1:+, region:Col4a1:-, region:Col6a1:-, region:Cpn1:-, region:Crabp1:+, region:Cryaa:+, region:Ctrb1:-, region:Cxcl13:+, region:Cyp1a2:-, region:Cyp2f2:+, region:Dbn1:-, region:Dclk1:+, region:Dlk1:+, region:Dppa3:+, region:Ecm1:-, region:Efna3:-, region:Elovl4:-, region:Fabp1:+, region:Fabp6:-, region:Fam183b:-, region:Fam25c:-, region:Fgg:+, region:Foxa1:-, region:Foxa2:-, region:Foxa3:-, region:Foxj1:-, region:Fxyd3:-, region:Fxyd6:+, region:Gabrp:-, region:Gal:-, region:Gatm:-, region:Ghrl:-, region:Gm13539:+, region:Gm2a:+, region:Gm94:-, region:Gp2:-, region:Gpx3:+, region:Gpx6:+, region:Grhl1:+, region:Gsdma:+, region:Gsta2:-, region:Gsta4:+, region:Gstm3:-, region:Gsto1:+, region:Guca2a:+, region:H19:-, region:Hamp2:-, region:Hba-a2:+, region:Hbb-y:-, region:Hck:+, region:Hnf4a:+, region:Hnf4g:+, region:Hp:-, region:Iapp:+, region:Idi2:+, region:Ifi27l2b:-, region:Igf1:+, region:Igf2:-, region:Igfbp5:-, region:Igfbp6:+, region:Ins2:-, region:Isg15:-, region:Itga6:+, region:Jpt1:-, region:Klk14:+, region:Klk7:+, region:Krt10:-, region:Krt13:-, region:Krt14:-, region:Krt16:-, region:Krt17:-, region:Krt19:-, region:Krt5:-, region:Krt71:-, region:Krt76:-, region:Krt77:-, region:Krt9:-, region:Krtdap:+, region:Lcn2:-, region:Lcn3:+, region:Lgals2:-, region:Lgals3:+, region:Lifr:+, region:Ltf:+, region:Ly6c1:-, region:Ly6c2:-, region:Ly6d:-, region:Ly6g6f:-, region:Lypd2:-, region:Lypd3:+, region:Mal:-, region:Meg3:+, region:Meig1:-, region:Mgp:-, region:Mlph:+, region:Mpo:+, region:Mt3:+, region:Mt4:+, region:Muc13:+, region:Mymx:-, region:Myog:+, region:Nefh:-, region:Ngp:+, region:Nppa:+, region:Npy:+, region:Nrgn:-, region:Nts:-, region:Oas1d:+, region:Oas1h:+, region:Odf3b:-, region:Olfm1:+, region:Pcp4:+, region:Pdzk1ip1:+, region:Penk:-, region:Pgc:+, region:Pigr:+, region:Pik3r5:+, region:Pla2g1b:+, region:Plac8:-, region:Plk2:+, region:Plvap:-, region:Pomc:+, region:Pou2f3:-, region:Psca:+, region:Ptgds:-, region:Pyy:-, region:Rap1gap2:-, region:Rarres1:-, region:Reg3b:+, region:Reg4:+, region:Retnla:+, region:Riiad1:-, region:Rnase1:-, region:Rnase4:+, region:Rsad2:-, region:Rsph1:-, region:S100a9:-, region:Saa1:-, region:Saa2:+, region:Serpinb11:+, region:Sftpa1:+, region:Sftpd:-, region:Slpi:-, region:Sox8:-, region:Spag8:-, region:Sparc:-, region:Spib:-, region:Spp1:+, region:Sprr1b:-, region:Sprr2i:+, region:Sst:-, region:Stxbp5l:-, region:Syt7:+, region:Tac1:+, region:Tac2:+, region:Tacstd2:-, region:Tff2:-, region:Tff3:-, region:Timp2:-, region:Tmem45a:-, region:Tnfrsf11b:-, region:Trp63:+, region:Trpm5:-, region:Tspan1:-, region:Ttc36:-, region:Vpreb3:+, region:Xcl1:-, region:Zg16:-,  > Done.\n",
      "-- inverting region_to_readout dict\n",
      "NDB_1027 -> ['Cilp:+', 'Cxcl13:+', 'Foxa2:-', 'Gm2a:+', 'Gm94:-', 'Igfbp6:+', 'Krt9:-', 'Pgc:+', 'Reg4:+', 'Rnase4:+', 'Saa2:+', 'Trp63:+', 'Xcl1:-']\n",
      "NDB_1033 -> ['Alox5ap:+', 'Ccl1:-', 'Ckb:-', 'Cryaa:+', 'Cyp1a2:-', 'Fabp1:+', 'Foxa3:-', 'Foxj1:-', 'Hnf4a:+', 'Pdzk1ip1:+', 'Rnase4:+', 'S100a9:-']\n",
      "NDB_1075 -> ['Ccl9:-', 'Cilp:+', 'Krt17:-', 'Lgals3:+', 'Ly6d:-', 'Mgp:-', 'Myog:+', 'Penk:-', 'Pla2g1b:+', 'S100a9:-', 'Saa1:-', 'Sst:-']\n",
      "NDB_317 -> ['Krt14:-']\n",
      "NDB_319 -> ['Krt5:-']\n",
      "NDB_320 -> ['Col6a1:-']\n",
      "NDB_322 -> ['Lifr:+']\n",
      "NDB_323 -> ['Itga6:+']\n",
      "NDB_325 -> ['Col4a1:-']\n",
      "NDB_326 -> ['Ascl1:-']\n",
      "NDB_328 -> ['Cald1:+']\n",
      "NDB_329 -> ['Krt17:-']\n",
      "NDB_331 -> ['C3:-']\n",
      "NDB_332 -> ['Trp63:+']\n",
      "NDB_334 -> ['Dclk1:+']\n",
      "NDB_335 -> ['Trpm5:-']\n",
      "NDB_337 -> ['Ly6g6f:-']\n",
      "NDB_338 -> ['Avil:+']\n",
      "NDB_340 -> ['Alox5ap:+']\n",
      "NDB_341 -> ['Hck:+']\n",
      "NDB_343 -> ['Plk2:+']\n",
      "NDB_344 -> ['Pik3r5:+']\n",
      "NDB_346 -> ['Agt:-']\n",
      "NDB_347 -> ['Nrgn:-']\n",
      "NDB_349 -> ['Plac8:-']\n",
      "NDB_350 -> ['Fxyd6:+']\n",
      "NDB_352 -> ['Pou2f3:-']\n",
      "NDB_353 -> ['Ccl20:+']\n",
      "NDB_355 -> ['Ccl9:-']\n",
      "NDB_356 -> ['Ccl6:-']\n",
      "NDB_358 -> ['Rnase1:-']\n",
      "NDB_359 -> ['Gp2:-']\n",
      "NDB_361 -> ['Fabp1:+']\n",
      "NDB_362 -> ['Sox8:-']\n",
      "NDB_364 -> ['Spib:-']\n",
      "NDB_365 -> ['Fxyd3:-']\n",
      "NDB_367 -> ['Tnfrsf11b:-']\n",
      "NDB_368 -> ['Ifi27l2b:-']\n",
      "NDB_370 -> ['Guca2a:+']\n",
      "NDB_371 -> ['Pigr:+']\n",
      "NDB_373 -> ['Muc13:+']\n",
      "NDB_374 -> ['Saa1:-']\n",
      "NDB_376 -> ['Aldob:-']\n",
      "NDB_377 -> ['Lgals2:-']\n",
      "NDB_379 -> ['Gsta2:-']\n",
      "NDB_380 -> ['Lcn2:-']\n",
      "NDB_382 -> ['Apoa1:+']\n",
      "NDB_383 -> ['Hnf4a:+']\n",
      "NDB_385 -> ['Hnf4g:+']\n",
      "NDB_386 -> ['Igf1:+']\n",
      "NDB_388 -> ['Grhl1:+']\n",
      "NDB_389 -> ['Lypd3:+']\n",
      "NDB_391 -> ['Krtdap:+']\n",
      "NDB_392 -> ['Tacstd2:-']\n",
      "NDB_394 -> ['Krt16:-']\n",
      "NDB_395 -> ['Pdzk1ip1:+']\n",
      "NDB_397 -> ['Tmem45a:-']\n",
      "NDB_398 -> ['Krt10:-']\n",
      "NDB_400 -> ['Gm94:-']\n",
      "NDB_401 -> ['Acpp:-']\n",
      "NDB_403 -> ['Cnfn:-']\n",
      "NDB_404 -> ['Fam25c:-']\n",
      "NDB_406 -> ['Elovl4:-']\n",
      "NDB_407 -> ['Efna3:-']\n",
      "NDB_409 -> ['Cebpb:+']\n",
      "NDB_410 -> ['Ly6d:-']\n",
      "NDB_412 -> ['Gsta4:+']\n",
      "NDB_413 -> ['Serpinb11:+']\n",
      "NDB_415 -> ['Gabrp:-']\n",
      "NDB_416 -> ['Cyp2f2:+']\n",
      "NDB_418 -> ['Gsto1:+']\n",
      "NDB_419 -> ['Anxa8:+']\n",
      "NDB_421 -> ['Mlph:+']\n",
      "NDB_422 -> ['Cd14:-']\n",
      "NDB_424 -> ['Foxa1:-']\n",
      "NDB_425 -> ['Ttc36:-']\n",
      "NDB_427 -> ['Foxj1:-']\n",
      "NDB_428 -> ['Fam183b:-']\n",
      "NDB_430 -> ['Ccdc30:-']\n",
      "NDB_431 -> ['Meig1:-']\n",
      "NDB_433 -> ['Spag8:-']\n",
      "NDB_434 -> ['Vpreb3:+']\n",
      "NDB_436 -> ['Riiad1:-']\n",
      "NDB_437 -> ['Rsph1:-']\n",
      "NDB_439 -> ['Odf3b:-']\n",
      "NDB_440 -> ['Car8:-']\n",
      "NDB_442 -> ['Rnase4:+']\n",
      "NDB_443 -> ['Tspan1:-']\n",
      "NDB_445 -> ['Stxbp5l:-']\n",
      "NDB_446 -> ['Igfbp5:-']\n",
      "NDB_448 -> ['Cpn1:-']\n",
      "NDB_449 -> ['Rap1gap2:-']\n",
      "NDB_452 -> ['Syt7:+']\n",
      "NDB_454 -> ['Gpx3:+']\n",
      "NDB_455 -> ['Foxa3:-']\n",
      "NDB_457 -> ['Foxa2:-']\n",
      "NDB_458 -> ['Ckb:-']\n",
      "NDB_460 -> ['Meg3:+']\n",
      "NDB_461 -> ['Myog:+']\n",
      "NDB_463 -> ['Ckm:+']\n",
      "NDB_464 -> ['Jpt1:-']\n",
      "NDB_466 -> ['Mymx:-']\n",
      "NDB_467 -> ['Dbn1:-']\n",
      "NDB_469 -> ['S100a9:-']\n",
      "NDB_470 -> ['Pyy:-']\n",
      "NDB_472 -> ['Fabp6:-']\n",
      "NDB_473 -> ['Slpi:-']\n",
      "NDB_475 -> ['Gal:-']\n",
      "NDB_476 -> ['Crabp1:+']\n",
      "NDB_478 -> ['Spp1:+']\n",
      "NDB_479 -> ['Mt4:+']\n",
      "NDB_481 -> ['Pomc:+']\n",
      "NDB_482 -> ['Tff3:-']\n",
      "NDB_484 -> ['Sst:-']\n",
      "NDB_485 -> ['Ptgds:-']\n",
      "NDB_487 -> ['Cck:-']\n",
      "NDB_488 -> ['Psca:+']\n",
      "NDB_490 -> ['Tac2:+']\n",
      "NDB_491 -> ['Gstm3:-']\n",
      "NDB_493 -> ['Pcp4:+']\n",
      "NDB_494 -> ['Pla2g1b:+']\n",
      "NDB_496 -> ['Nts:-']\n",
      "NDB_497 -> ['Apod:-']\n",
      "NDB_499 -> ['Ctrb1:-']\n",
      "NDB_500 -> ['Lypd2:-']\n",
      "NDB_502 -> ['Pgc:+']\n",
      "NDB_503 -> ['Rarres1:-']\n",
      "NDB_505 -> ['Retnla:+']\n",
      "NDB_506 -> ['Ins2:-']\n",
      "NDB_508 -> ['Cga:+']\n",
      "NDB_509 -> ['Iapp:+']\n",
      "NDB_511 -> ['Nppa:+']\n",
      "NDB_512 -> ['Camp:-']\n",
      "NDB_514 -> ['Cyp1a2:-']\n",
      "NDB_515 -> ['Dlk1:+']\n",
      "NDB_517 -> ['Gm13539:+']\n",
      "NDB_518 -> ['Krt13:-']\n",
      "NDB_520 -> ['Cilp:+']\n",
      "NDB_521 -> ['Igfbp6:+']\n",
      "NDB_523 -> ['Apoc2:-']\n",
      "NDB_524 -> ['Sparc:-']\n",
      "NDB_526 -> ['Tac1:+']\n",
      "NDB_527 -> ['Npy:+']\n",
      "NDB_529 -> ['Saa2:+']\n",
      "NDB_530 -> ['Mpo:+']\n",
      "NDB_532 -> ['Gatm:-']\n",
      "NDB_533 -> ['Xcl1:-']\n",
      "NDB_535 -> ['H19:-']\n",
      "NDB_536 -> ['Ly6c2:-']\n",
      "NDB_538 -> ['Cxcl13:+']\n",
      "NDB_539 -> ['Penk:-']\n",
      "NDB_541 -> ['Ecm1:-']\n",
      "NDB_542 -> ['Sprr1b:-']\n",
      "NDB_544 -> ['Ly6c1:-']\n",
      "NDB_545 -> ['Hbb-y:-']\n",
      "NDB_547 -> ['Ccl1:-']\n",
      "NDB_548 -> ['Klk7:+']\n",
      "NDB_550 -> ['Dppa3:+']\n",
      "NDB_551 -> ['Hba-a2:+']\n",
      "NDB_553 -> ['Mt3:+']\n",
      "NDB_554 -> ['Krt71:-']\n",
      "NDB_556 -> ['Cryaa:+']\n",
      "NDB_557 -> ['Tff2:-']\n",
      "NDB_559 -> ['Timp2:-']\n",
      "NDB_560 -> ['Klk14:+']\n",
      "NDB_562 -> ['Fgg:+']\n",
      "NDB_563 -> ['Oas1d:+']\n",
      "NDB_565 -> ['Rsad2:-']\n",
      "NDB_566 -> ['Apoa2:+']\n",
      "NDB_568 -> ['Nefh:-']\n",
      "NDB_569 -> ['Idi2:+']\n",
      "NDB_571 -> ['Olfm1:+']\n",
      "NDB_572 -> ['Bgn:+']\n",
      "NDB_574 -> ['Ngp:+']\n",
      "NDB_575 -> ['Car4:+']\n",
      "NDB_577 -> ['Lgals3:+']\n",
      "NDB_578 -> ['Ccl2:+']\n",
      "NDB_580 -> ['Hp:-']\n",
      "NDB_581 -> ['C1qb:-']\n",
      "NDB_583 -> ['Mal:-']\n",
      "NDB_584 -> ['Alpl:-']\n",
      "NDB_586 -> ['Krt77:-']\n",
      "NDB_587 -> ['Cnmd:-']\n",
      "NDB_589 -> ['Ang4:-']\n",
      "NDB_590 -> ['Krt19:-']\n",
      "NDB_592 -> ['Mgp:-']\n",
      "NDB_593 -> ['Igf2:-']\n",
      "NDB_595 -> ['Alpi:-']\n",
      "NDB_596 -> ['Ghrl:-']\n",
      "NDB_598 -> ['Hamp2:-']\n",
      "NDB_599 -> ['Gm2a:+']\n",
      "NDB_601 -> ['Krt9:-']\n",
      "NDB_602 -> ['Reg4:+']\n",
      "NDB_604 -> ['Isg15:-']\n",
      "NDB_605 -> ['Calb2:-']\n",
      "NDB_607 -> ['Zg16:-']\n",
      "NDB_608 -> ['Aldh3a1:+']\n",
      "NDB_610 -> ['Sftpd:-']\n",
      "NDB_611 -> ['Lcn3:+']\n",
      "NDB_613 -> ['Sprr2i:+']\n",
      "NDB_614 -> ['Reg3b:+']\n",
      "NDB_616 -> ['Sftpa1:+']\n",
      "NDB_617 -> ['Oas1h:+']\n",
      "NDB_619 -> ['Krt76:-']\n",
      "NDB_620 -> ['Gsdma:+']\n",
      "NDB_622 -> ['Ltf:+']\n",
      "NDB_623 -> ['Clps:-']\n",
      "NDB_625 -> ['Col1a1:+']\n",
      "NDB_626 -> ['Gpx6:+']\n",
      "NDB_628 -> ['Plvap:-']\n",
      "NDB_631 -> ['Apoa2:+', 'Car8:-', 'Fxyd6:+', 'Hnf4a:+', 'Hnf4g:+', 'Igf2:-', 'Ly6c1:-', 'Muc13:+', 'Nts:-', 'Pik3r5:+', 'Rnase1:-', 'Spib:-']\n",
      "NDB_634 -> ['Alox5ap:+', 'Camp:-', 'Car8:-', 'Ctrb1:-', 'Gal:-', 'Ghrl:-', 'Gsdma:+', 'Hbb-y:-', 'Lifr:+', 'Olfm1:+', 'Pyy:-', 'Sox8:-', 'Sst:-']\n",
      "NDB_643 -> ['Apoa2:+', 'Ccdc30:-', 'Ccl9:-', 'Efna3:-', 'Gal:-', 'Gpx6:+', 'Hba-a2:+', 'Hp:-', 'Krt77:-', 'Meig1:-', 'Saa2:+', 'Stxbp5l:-', 'Ttc36:-']\n",
      "NDB_652 -> ['Calb2:-', 'Cyp2f2:+', 'Dbn1:-', 'Gsdma:+', 'Hba-a2:+', 'Isg15:-', 'Meg3:+', 'Pgc:+', 'Reg3b:+', 'Slpi:-', 'Spib:-', 'Sprr1b:-']\n",
      "NDB_653 -> ['Avil:+', 'Camp:-', 'Ccdc30:-', 'Gm2a:+', 'Idi2:+', 'Ifi27l2b:-', 'Itga6:+', 'Klk14:+', 'Krt17:-', 'Mymx:-', 'Psca:+', 'Reg3b:+', 'Rnase1:-']\n",
      "NDB_661 -> ['Ascl1:-', 'Car4:+', 'Dlk1:+', 'Elovl4:-', 'Foxa3:-', 'Fxyd3:-', 'Igfbp5:-', 'Lifr:+', 'Reg3b:+', 'Riiad1:-', 'Tff2:-']\n",
      "NDB_680 -> ['Anxa8:+', 'Bgn:+', 'Camp:-', 'Ccl9:-', 'Cxcl13:+', 'Gm13539:+', 'Klk7:+', 'Mt4:+', 'Pdzk1ip1:+', 'Rarres1:-', 'Rsph1:-', 'Tnfrsf11b:-', 'Trpm5:-']\n",
      "NDB_683 -> ['Apoc2:-', 'Ccl20:+', 'Dclk1:+', 'Gm2a:+', 'Igf1:+', 'Krt16:-', 'Krt5:-', 'Mal:-', 'Ngp:+', 'Nppa:+', 'Plk2:+', 'Retnla:+', 'Tff3:-']\n",
      "NDB_698 -> ['Ascl1:-', 'C1qb:-', 'Col4a1:-', 'Cyp1a2:-', 'Krt16:-', 'Lcn2:-', 'Lypd3:+', 'Mpo:+', 'Odf3b:-', 'Rarres1:-', 'Tac1:+', 'Tmem45a:-']\n",
      "NDB_710 -> ['Ccl6:-', 'Crabp1:+', 'Cyp2f2:+', 'Hnf4g:+', 'Krt17:-', 'Krt71:-', 'Mlph:+', 'Plac8:-', 'Pomc:+', 'Ptgds:-', 'Rnase4:+']\n",
      "NDB_713 -> ['Alpl:-', 'Ccl20:+', 'Ckm:+', 'Elovl4:-', 'Fam183b:-', 'Gatm:-', 'Grhl1:+', 'Krt10:-', 'Ly6g6f:-', 'Psca:+', 'S100a9:-', 'Spp1:+']\n",
      "NDB_715 -> ['Ang4:-', 'Gabrp:-', 'Gsto1:+', 'Hamp2:-', 'Krt71:-', 'Npy:+', 'Pou2f3:-', 'Pyy:-', 'Riiad1:-', 'Rsad2:-', 'Slpi:-', 'Tff3:-', 'Trp63:+']\n",
      "NDB_716 -> ['Cck:-', 'Ccl20:+', 'Hnf4g:+', 'Jpt1:-', 'Lcn3:+', 'Plvap:-', 'Pou2f3:-', 'Sftpd:-', 'Sparc:-', 'Tac2:+', 'Tnfrsf11b:-', 'Zg16:-']\n",
      "NDB_718 -> ['Cald1:+', 'Cebpb:+', 'Cnfn:-', 'Col4a1:-', 'Dbn1:-', 'Ecm1:-', 'Gpx3:+', 'Penk:-', 'Pomc:+', 'Rap1gap2:-', 'Rnase1:-', 'Tff3:-', 'Ttc36:-']\n",
      "NDB_725 -> ['Ccl2:+', 'Cga:+', 'Foxa1:-', 'Gatm:-', 'Gm94:-', 'Hbb-y:-', 'Krt13:-', 'Krt76:-', 'Mlph:+', 'Nrgn:-', 'Plk2:+', 'Rarres1:-', 'Sftpa1:+']\n",
      "NDB_728 -> ['Anxa8:+', 'Cd14:-', 'Col1a1:+', 'Grhl1:+', 'Guca2a:+', 'Ltf:+', 'Ly6c2:-', 'Npy:+', 'Ptgds:-', 'Rap1gap2:-', 'Tmem45a:-', 'Xcl1:-', 'Zg16:-']\n",
      "NDB_737 -> ['Apod:-', 'Cald1:+', 'Gatm:-', 'H19:-', 'Idi2:+', 'Isg15:-', 'Krt77:-', 'Ltf:+', 'Mpo:+', 'Mt4:+', 'Myog:+', 'Reg4:+', 'Syt7:+']\n",
      "NDB_743 -> ['Aldob:-', 'Avil:+', 'Cebpb:+', 'Fabp1:+', 'Fabp6:-', 'Foxa1:-', 'Fxyd6:+', 'Gp2:-', 'Hp:-', 'Jpt1:-', 'Pigr:+', 'Riiad1:-']\n",
      "NDB_746 -> ['Calb2:-', 'Ccl1:-', 'Hamp2:-', 'Lgals2:-', 'Muc13:+', 'Nefh:-', 'Nrgn:-', 'Oas1h:+', 'Reg4:+', 'Rsph1:-', 'Tacstd2:-', 'Tspan1:-']\n",
      "NDB_754 -> ['Agt:-', 'Ang4:-', 'Cga:+', 'Dppa3:+', 'Fxyd6:+', 'Gal:-', 'Gpx3:+', 'Gstm3:-', 'Krt14:-', 'Krt5:-', 'Oas1h:+', 'Saa1:-', 'Tmem45a:-']\n",
      "NDB_755 -> ['Crabp1:+', 'Dclk1:+', 'Fam183b:-', 'Fam25c:-', 'Foxa1:-', 'Klk14:+', 'Lcn3:+', 'Ly6d:-', 'Oas1h:+', 'Pyy:-', 'Spag8:-', 'Tff2:-']\n",
      "NDB_758 -> ['Clps:-', 'Dbn1:-', 'Fabp1:+', 'Grhl1:+', 'Igfbp5:-', 'Krt14:-', 'Odf3b:-', 'Pik3r5:+', 'Pou2f3:-', 'Saa2:+', 'Syt7:+', 'Tspan1:-']\n",
      "NDB_760 -> ['Calb2:-', 'Cnmd:-', 'Dppa3:+', 'Gp2:-', 'Gsta4:+', 'Guca2a:+', 'Itga6:+', 'Krt13:-', 'Ly6d:-', 'Stxbp5l:-', 'Timp2:-', 'Vpreb3:+']\n",
      "NDB_773 -> ['Agt:-', 'Apoa1:+', 'Apoa2:+', 'Apod:-', 'Ascl1:-', 'Gsto1:+', 'Hbb-y:-', 'Krt19:-', 'Mal:-', 'Rap1gap2:-', 'Spag8:-', 'Sprr1b:-', 'Tacstd2:-']\n",
      "NDB_778 -> ['C3:-', 'Cpn1:-', 'Fam183b:-', 'Fgg:+', 'Gsta4:+', 'H19:-', 'Lypd3:+', 'Meg3:+', 'Meig1:-', 'Nppa:+', 'Pcp4:+', 'Plac8:-', 'Tnfrsf11b:-']\n",
      "NDB_784 -> ['Bgn:+', 'Ccl2:+', 'Ctrb1:-', 'Cyp2f2:+', 'Fabp6:-', 'Fgg:+', 'Hck:+', 'Ltf:+', 'Mymx:-', 'Penk:-', 'Rsad2:-', 'Serpinb11:+', 'Spp1:+']\n",
      "NDB_790 -> ['Apoc2:-', 'Cilp:+', 'Cryaa:+', 'Efna3:-', 'Gm13539:+', 'Gsdma:+', 'Igf2:-', 'Ly6c2:-', 'Rsad2:-', 'Sftpa1:+', 'Spag8:-', 'Tac2:+']\n",
      "NDB_812 -> ['Acpp:-', 'Ccl1:-', 'Gstm3:-', 'Ins2:-', 'Klk7:+', 'Krt77:-', 'Ly6g6f:-', 'Pla2g1b:+', 'Plvap:-', 'Retnla:+', 'Slpi:-', 'Tac1:+']\n",
      "NDB_815 -> ['Avil:+', 'Car4:+', 'Ckm:+', 'Cnmd:-', 'Foxj1:-', 'Klk7:+', 'Lcn3:+', 'Ly6c1:-', 'Mt3:+', 'Sprr2i:+', 'Ttc36:-', 'Xcl1:-']\n",
      "NDB_817 -> ['Agt:-', 'Ccdc30:-', 'Ccl2:+', 'Clps:-', 'Cpn1:-', 'Dclk1:+', 'Fxyd3:-', 'Krt10:-', 'Krt71:-', 'Lcn2:-', 'Muc13:+', 'Oas1d:+', 'Trpm5:-']\n",
      "NDB_826 -> ['C1qb:-', 'Col6a1:-', 'Cxcl13:+', 'Ecm1:-', 'Krt10:-', 'Lifr:+', 'Meg3:+', 'Ngp:+', 'Pigr:+', 'Saa1:-', 'Sftpd:-', 'Tacstd2:-', 'Timp2:-']\n",
      "NDB_844 -> ['Aldob:-', 'Alpi:-', 'Apod:-', 'Cnfn:-', 'Col1a1:+', 'Gabrp:-', 'Ins2:-', 'Itga6:+', 'Lgals2:-', 'Pgc:+', 'Plk2:+', 'Sftpd:-']\n",
      "NDB_847 -> ['Car8:-', 'Cd14:-', 'Cnmd:-', 'Fabp6:-', 'Foxa2:-', 'Gsta2:-', 'Hamp2:-', 'Iapp:+', 'Krt16:-', 'Ly6g6f:-', 'Plac8:-', 'Tac2:+']\n",
      "NDB_856 -> ['Alpl:-', 'Ckb:-', 'Clps:-', 'Fgg:+', 'Gm13539:+', 'Gm94:-', 'Krt19:-', 'Krtdap:+', 'Lgals3:+', 'Ly6c1:-', 'Nefh:-', 'Ngp:+', 'Vpreb3:+']\n",
      "NDB_860 -> ['Acpp:-', 'Aldh3a1:+', 'Alpi:-', 'Ang4:-', 'Cd14:-', 'Efna3:-', 'Hnf4a:+', 'Ifi27l2b:-', 'Krt13:-', 'Lypd3:+', 'Mgp:-', 'Oas1d:+']\n",
      "NDB_863 -> ['Aldob:-', 'Ccl6:-', 'Col6a1:-', 'Dlk1:+', 'Gsta2:-', 'Gsto1:+', 'Krtdap:+', 'Ly6c2:-', 'Meig1:-', 'Mgp:-', 'Plvap:-', 'Serpinb11:+', 'Trpm5:-']\n",
      "NDB_865 -> ['Alpi:-', 'Apoc2:-', 'Ckb:-', 'Crabp1:+', 'Gpx6:+', 'Lypd2:-', 'Npy:+', 'Sox8:-', 'Sparc:-', 'Spp1:+', 'Sprr2i:+', 'Timp2:-', 'Tspan1:-']\n",
      "NDB_868 -> ['Car4:+', 'Cga:+', 'Fam25c:-', 'Gsta4:+', 'Lypd2:-', 'Mymx:-', 'Nefh:-', 'Oas1d:+', 'Olfm1:+', 'Pomc:+', 'Sprr1b:-', 'Zg16:-']\n",
      "NDB_875 -> ['C1qb:-', 'C3:-', 'Cck:-', 'Ctrb1:-', 'Dlk1:+', 'Gp2:-', 'Igf1:+', 'Krt9:-', 'Lypd2:-', 'Myog:+', 'Pik3r5:+', 'Ptgds:-']\n",
      "NDB_878 -> ['Acpp:-', 'Cebpb:+', 'Col1a1:+', 'Foxa3:-', 'Ghrl:-', 'Igf1:+', 'Krt76:-', 'Lgals3:+', 'Pcp4:+', 'Rsph1:-', 'Spib:-', 'Sprr2i:+', 'Syt7:+']\n",
      "NDB_883 -> ['Cck:-', 'Gpx3:+', 'Gpx6:+', 'Iapp:+', 'Klk14:+', 'Krt19:-', 'Mt4:+', 'Nrgn:-', 'Odf3b:-', 'Olfm1:+', 'Retnla:+', 'Serpinb11:+']\n",
      "NDB_901 -> ['Anxa8:+', 'Ccl6:-', 'Ckm:+', 'Hck:+', 'Ifi27l2b:-', 'Isg15:-', 'Krt76:-', 'Krt9:-', 'Nts:-', 'Pigr:+', 'Tac1:+']\n",
      "NDB_917 -> ['Apoa1:+', 'Foxj1:-', 'Guca2a:+', 'Iapp:+', 'Idi2:+', 'Krtdap:+', 'Lcn2:-', 'Lgals2:-', 'Pcp4:+', 'Pla2g1b:+', 'Sftpa1:+', 'Sparc:-', 'Tff2:-']\n",
      "NDB_961 -> ['Aldh3a1:+', 'Cryaa:+', 'Dppa3:+', 'Ecm1:-', 'Fxyd3:-', 'Gsta2:-', 'H19:-', 'Hp:-', 'Igfbp6:+', 'Mal:-', 'Nts:-', 'Sst:-']\n",
      "NDB_965 -> ['Cnfn:-', 'Cpn1:-', 'Elovl4:-', 'Ghrl:-', 'Hba-a2:+', 'Hck:+', 'Krt5:-', 'Mlph:+', 'Mpo:+', 'Mt3:+', 'Pdzk1ip1:+', 'Trp63:+', 'Vpreb3:+']\n",
      "NDB_971 -> ['Alox5ap:+', 'Alpl:-', 'Apoa1:+', 'Bgn:+', 'C3:-', 'Col4a1:-', 'Fam25c:-', 'Gabrp:-', 'Gstm3:-', 'Igf2:-', 'Igfbp5:-', 'Igfbp6:+', 'Jpt1:-']\n",
      "NDB_974 -> ['Aldh3a1:+', 'Cald1:+', 'Col6a1:-', 'Cyp1a2:-', 'Foxa2:-', 'Ins2:-', 'Krt14:-', 'Mt3:+', 'Nppa:+', 'Psca:+', 'Sox8:-', 'Stxbp5l:-']\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "readout_reg_dic, readout2reg_check = check._check_readout_to_region(reg_readout_dic, full_records, \n",
    "                                                                    check_readout_dict, primer_len=primer_len, \n",
    "                                                                    target_len=target_len,\n",
    "                                                                    probe_readout_num=num_readout_per_probe)\n",
    "\n",
    "print(readout2reg_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bronze-cambodia",
   "metadata": {},
   "source": [
    "### 4.1.5 check bewteen probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "qualified-vacation",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- saving internal 17-mer map to file:\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\probe_table_17.npz\n"
     ]
    }
   ],
   "source": [
    "# construct internal map\n",
    "reload(check)\n",
    "int_map = check._construct_internal_map(full_records, library_folder) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "crazy-threat",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "True\n"
     ]
    }
   ],
   "source": [
    "# internal check probes\n",
    "readout_count_dic, readout_count_check = check._check_readout_in_probes(readout_reg_dic, reg_size_dic, int_map, readout_dict)\n",
    "print(readout_count_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "honest-tracker",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_4:8224041-8254041_gene_Car8:-_pb_243_pos_23900_strand_-_readouts_[NDB_631_c,NDB_631_c,NDB_631_c] got hits:52, dumped\n",
      "--- Sequence:loc_4:8224041-8254041_gene_Car8:-_pb_269_pos_26158_strand_-_readouts_[NDB_847_c,NDB_847_c,NDB_847_c] got hits:51, dumped\n",
      "--- Sequence:loc_9:121480694-121510694_gene_Cck:-_pb_46_pos_4608_strand_+_readouts_[NDB_716_c,NDB_716_c,NDB_716_c] got hits:89, dumped\n",
      "--- Sequence:loc_9:121480694-121510694_gene_Cck:-_pb_47_pos_4657_strand_+_readouts_[NDB_883_c,NDB_883_c,NDB_883_c] got hits:60, dumped\n",
      "--- Sequence:loc_9:121480694-121510694_gene_Cck:-_pb_48_pos_4717_strand_+_readouts_[NDB_487_u,NDB_487_u,NDB_487_u] got hits:89, dumped\n",
      "--- Sequence:loc_9:121480694-121510694_gene_Cck:-_pb_49_pos_4791_strand_-_readouts_[NDB_875_c,NDB_875_c,NDB_875_c] got hits:60, dumped\n",
      "--- Sequence:loc_9:121480694-121510694_gene_Cck:-_pb_51_pos_4903_strand_+_readouts_[NDB_883_c,NDB_883_c,NDB_883_c] got hits:71, dumped\n",
      "--- Sequence:loc_17:31662807-31692807_gene_Cryaa:+_pb_31_pos_3445_strand_+_readouts_[NDB_1033_c,NDB_1033_c,NDB_1033_c] got hits:55, dumped\n",
      "--- Sequence:loc_17:31662807-31692807_gene_Cryaa:+_pb_32_pos_3502_strand_+_readouts_[NDB_556_u,NDB_556_u,NDB_556_u] got hits:55, dumped\n",
      "--- Sequence:loc_19:47839970-47869970_gene_Gsto1:+_pb_167_pos_21256_strand_+_readouts_[NDB_863_c,NDB_863_c,NDB_863_c] got hits:70, dumped\n",
      "--- Sequence:loc_19:47839970-47869970_gene_Gsto1:+_pb_168_pos_21358_strand_+_readouts_[NDB_418_u,NDB_418_u,NDB_418_u] got hits:60, dumped\n",
      "--- Sequence:loc_19:47839970-47869970_gene_Gsto1:+_pb_169_pos_21453_strand_+_readouts_[NDB_773_c,NDB_773_c,NDB_773_c] got hits:72, dumped\n",
      "--- Sequence:loc_19:47839970-47869970_gene_Gsto1:+_pb_170_pos_21495_strand_+_readouts_[NDB_715_c,NDB_715_c,NDB_715_c] got hits:75, dumped\n",
      "--- Sequence:loc_19:47839970-47869970_gene_Gsto1:+_pb_171_pos_21542_strand_+_readouts_[NDB_863_c,NDB_863_c,NDB_863_c] got hits:53, dumped\n",
      "--- Sequence:loc_19:47839970-47869970_gene_Gsto1:+_pb_172_pos_21594_strand_+_readouts_[NDB_418_u,NDB_418_u,NDB_418_u] got hits:63, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_2_pos_456_strand_+_readouts_[NDB_965_c,NDB_965_c,NDB_965_c] got hits:66, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_3_pos_577_strand_+_readouts_[NDB_901_c,NDB_901_c,NDB_901_c] got hits:157, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_4_pos_1617_strand_-_readouts_[NDB_341_u,NDB_341_u,NDB_341_u] got hits:263, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_5_pos_1863_strand_+_readouts_[NDB_784_c,NDB_784_c,NDB_784_c] got hits:118, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_6_pos_2859_strand_-_readouts_[NDB_965_c,NDB_965_c,NDB_965_c] got hits:227, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_7_pos_3473_strand_+_readouts_[NDB_901_c,NDB_901_c,NDB_901_c] got hits:95, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_8_pos_3516_strand_+_readouts_[NDB_341_u,NDB_341_u,NDB_341_u] got hits:156, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_9_pos_3989_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c] got hits:112, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_10_pos_4247_strand_-_readouts_[NDB_965_c,NDB_965_c,NDB_965_c] got hits:99, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_11_pos_4342_strand_-_readouts_[NDB_901_c,NDB_901_c,NDB_901_c] got hits:162, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_12_pos_4413_strand_-_readouts_[NDB_341_u,NDB_341_u,NDB_341_u] got hits:343, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_13_pos_4470_strand_+_readouts_[NDB_784_c,NDB_784_c,NDB_784_c] got hits:338, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_14_pos_4644_strand_+_readouts_[NDB_965_c,NDB_965_c,NDB_965_c] got hits:149, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_15_pos_4706_strand_+_readouts_[NDB_901_c,NDB_901_c,NDB_901_c] got hits:218, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_16_pos_4750_strand_-_readouts_[NDB_341_u,NDB_341_u,NDB_341_u] got hits:259, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_17_pos_4860_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c] got hits:156, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_18_pos_4938_strand_+_readouts_[NDB_965_c,NDB_965_c,NDB_965_c] got hits:250, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_19_pos_5024_strand_-_readouts_[NDB_901_c,NDB_901_c,NDB_901_c] got hits:360, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_20_pos_5092_strand_+_readouts_[NDB_341_u,NDB_341_u,NDB_341_u] got hits:231, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_21_pos_5134_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c] got hits:247, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_22_pos_5211_strand_-_readouts_[NDB_965_c,NDB_965_c,NDB_965_c] got hits:304, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_23_pos_5255_strand_-_readouts_[NDB_901_c,NDB_901_c,NDB_901_c] got hits:331, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_24_pos_5305_strand_-_readouts_[NDB_341_u,NDB_341_u,NDB_341_u] got hits:306, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_25_pos_5346_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c] got hits:228, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_26_pos_6355_strand_+_readouts_[NDB_965_c,NDB_965_c,NDB_965_c] got hits:141, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_27_pos_6427_strand_+_readouts_[NDB_901_c,NDB_901_c,NDB_901_c] got hits:319, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_28_pos_6587_strand_+_readouts_[NDB_341_u,NDB_341_u,NDB_341_u] got hits:355, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_29_pos_6663_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c] got hits:341, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_30_pos_6723_strand_-_readouts_[NDB_965_c,NDB_965_c,NDB_965_c] got hits:120, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_31_pos_6856_strand_-_readouts_[NDB_901_c,NDB_901_c,NDB_901_c] got hits:251, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_32_pos_6903_strand_-_readouts_[NDB_341_u,NDB_341_u,NDB_341_u] got hits:358, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_33_pos_6987_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c] got hits:285, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_34_pos_7029_strand_-_readouts_[NDB_965_c,NDB_965_c,NDB_965_c] got hits:192, dumped\n",
      "--- Sequence:loc_2:153093468-153123468_gene_Hck:+_pb_35_pos_7088_strand_+_readouts_[NDB_901_c,NDB_901_c,NDB_901_c] got hits:226, dumped\n",
      "--- Sequence:loc_13:8937863-8967863_gene_Idi2:+_pb_49_pos_7418_strand_+_readouts_[NDB_917_c,NDB_917_c,NDB_917_c] got hits:59, dumped\n",
      "--- Sequence:loc_13:8937863-8967863_gene_Idi2:+_pb_58_pos_11919_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c] got hits:59, dumped\n",
      "--- Sequence:loc_2:71730616-71760616_gene_Itga6:+_pb_64_pos_6569_strand_+_readouts_[NDB_323_u,NDB_323_u,NDB_323_u] got hits:67, dumped\n",
      "--- Sequence:loc_2:71730616-71760616_gene_Itga6:+_pb_65_pos_6804_strand_+_readouts_[NDB_760_c,NDB_760_c,NDB_760_c] got hits:90, dumped\n",
      "--- Sequence:loc_2:71730616-71760616_gene_Itga6:+_pb_66_pos_6853_strand_+_readouts_[NDB_844_c,NDB_844_c,NDB_844_c] got hits:82, dumped\n",
      "--- Sequence:loc_2:71730616-71760616_gene_Itga6:+_pb_67_pos_6911_strand_-_readouts_[NDB_653_c,NDB_653_c,NDB_653_c] got hits:66, dumped\n",
      "--- Sequence:loc_2:71730616-71760616_gene_Itga6:+_pb_68_pos_6952_strand_+_readouts_[NDB_323_u,NDB_323_u,NDB_323_u] got hits:89, dumped\n",
      "--- Sequence:loc_11:100133665-100163665_gene_Krt19:-_pb_3_pos_188_strand_+_readouts_[NDB_883_c,NDB_883_c,NDB_883_c] got hits:69, dumped\n",
      "--- Sequence:loc_11:100133665-100163665_gene_Krt19:-_pb_4_pos_241_strand_+_readouts_[NDB_590_u,NDB_590_u,NDB_590_u] got hits:60, dumped\n",
      "--- Sequence:loc_11:100133665-100163665_gene_Krt19:-_pb_5_pos_315_strand_+_readouts_[NDB_773_c,NDB_773_c,NDB_773_c] got hits:60, dumped\n",
      "--- Sequence:loc_11:100133665-100163665_gene_Krt19:-_pb_6_pos_385_strand_+_readouts_[NDB_856_c,NDB_856_c,NDB_856_c] got hits:83, dumped\n",
      "--- Sequence:loc_11:100133665-100163665_gene_Krt19:-_pb_7_pos_426_strand_+_readouts_[NDB_883_c,NDB_883_c,NDB_883_c] got hits:81, dumped\n",
      "--- Sequence:loc_11:100133665-100163665_gene_Krt19:-_pb_8_pos_500_strand_+_readouts_[NDB_590_u,NDB_590_u,NDB_590_u] got hits:55, dumped\n",
      "--- Sequence:loc_11:100133665-100163665_gene_Krt19:-_pb_9_pos_541_strand_+_readouts_[NDB_773_c,NDB_773_c,NDB_773_c] got hits:62, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_2:3407648-3437648_gene_Meig1:-_pb_27_pos_3473_strand_-_readouts_[NDB_863_c,NDB_863_c,NDB_863_c] got hits:68, dumped\n",
      "--- Sequence:loc_2:3407648-3437648_gene_Meig1:-_pb_28_pos_3518_strand_-_readouts_[NDB_431_u,NDB_431_u,NDB_431_u] got hits:68, dumped\n",
      "--- Sequence:loc_2:3407648-3437648_gene_Meig1:-_pb_30_pos_3619_strand_-_readouts_[NDB_778_c,NDB_778_c,NDB_778_c] got hits:73, dumped\n",
      "--- Sequence:loc_2:3407648-3437648_gene_Meig1:-_pb_35_pos_4017_strand_+_readouts_[NDB_863_c,NDB_863_c,NDB_863_c] got hits:114, dumped\n",
      "--- Sequence:loc_2:3407648-3437648_gene_Meig1:-_pb_36_pos_4078_strand_-_readouts_[NDB_431_u,NDB_431_u,NDB_431_u] got hits:162, dumped\n",
      "--- Sequence:loc_2:3407648-3437648_gene_Meig1:-_pb_37_pos_4133_strand_-_readouts_[NDB_643_c,NDB_643_c,NDB_643_c] got hits:132, dumped\n",
      "--- Sequence:loc_2:3407648-3437648_gene_Meig1:-_pb_38_pos_4185_strand_+_readouts_[NDB_778_c,NDB_778_c,NDB_778_c] got hits:149, dumped\n",
      "--- Sequence:loc_2:3407648-3437648_gene_Meig1:-_pb_39_pos_4226_strand_+_readouts_[NDB_863_c,NDB_863_c,NDB_863_c] got hits:87, dumped\n",
      "--- Sequence:loc_2:3407648-3437648_gene_Meig1:-_pb_40_pos_4268_strand_-_readouts_[NDB_431_u,NDB_431_u,NDB_431_u] got hits:147, dumped\n",
      "--- Sequence:loc_2:3407648-3437648_gene_Meig1:-_pb_41_pos_4312_strand_+_readouts_[NDB_643_c,NDB_643_c,NDB_643_c] got hits:55, dumped\n",
      "--- Sequence:loc_2:3407648-3437648_gene_Meig1:-_pb_42_pos_4493_strand_+_readouts_[NDB_778_c,NDB_778_c,NDB_778_c] got hits:187, dumped\n",
      "--- Sequence:loc_2:3407648-3437648_gene_Meig1:-_pb_43_pos_4590_strand_+_readouts_[NDB_863_c,NDB_863_c,NDB_863_c] got hits:152, dumped\n",
      "--- Sequence:loc_2:3407648-3437648_gene_Meig1:-_pb_44_pos_4658_strand_+_readouts_[NDB_431_u,NDB_431_u,NDB_431_u] got hits:142, dumped\n",
      "--- Sequence:loc_2:3407648-3437648_gene_Meig1:-_pb_45_pos_4735_strand_+_readouts_[NDB_643_c,NDB_643_c,NDB_643_c] got hits:169, dumped\n",
      "--- Sequence:loc_2:3407648-3437648_gene_Meig1:-_pb_46_pos_4776_strand_+_readouts_[NDB_778_c,NDB_778_c,NDB_778_c] got hits:106, dumped\n",
      "--- Sequence:loc_2:3407648-3437648_gene_Meig1:-_pb_47_pos_4818_strand_+_readouts_[NDB_863_c,NDB_863_c,NDB_863_c] got hits:143, dumped\n",
      "--- Sequence:loc_11:4933064-4963064_gene_Nefh:-_pb_203_pos_22279_strand_+_readouts_[NDB_856_c,NDB_856_c,NDB_856_c] got hits:90, dumped\n",
      "--- Sequence:loc_11:4933064-4963064_gene_Nefh:-_pb_206_pos_22423_strand_+_readouts_[NDB_868_c,NDB_868_c,NDB_868_c] got hits:91, dumped\n",
      "--- Sequence:loc_11:4933064-4963064_gene_Nefh:-_pb_211_pos_22711_strand_+_readouts_[NDB_856_c,NDB_856_c,NDB_856_c] got hits:53, dumped\n",
      "--- Sequence:loc_8:71496752-71526752_gene_Plvap:-_pb_219_pos_22612_strand_+_readouts_[NDB_863_c,NDB_863_c,NDB_863_c] got hits:102, dumped\n",
      "--- Sequence:loc_8:71496752-71526752_gene_Plvap:-_pb_221_pos_22735_strand_+_readouts_[NDB_812_c,NDB_812_c,NDB_812_c] got hits:96, dumped\n",
      "--- Sequence:loc_8:71496752-71526752_gene_Plvap:-_pb_222_pos_22932_strand_+_readouts_[NDB_716_c,NDB_716_c,NDB_716_c] got hits:91, dumped\n",
      "--- Sequence:loc_8:71496752-71526752_gene_Plvap:-_pb_223_pos_22974_strand_+_readouts_[NDB_863_c,NDB_863_c,NDB_863_c] got hits:100, dumped\n",
      "--- Sequence:loc_8:71496752-71526752_gene_Plvap:-_pb_224_pos_23017_strand_+_readouts_[NDB_628_u,NDB_628_u,NDB_628_u] got hits:102, dumped\n",
      "--- Sequence:loc_8:71496752-71526752_gene_Plvap:-_pb_225_pos_23058_strand_+_readouts_[NDB_812_c,NDB_812_c,NDB_812_c] got hits:102, dumped\n",
      "--- Sequence:loc_8:71496752-71526752_gene_Plvap:-_pb_228_pos_23314_strand_-_readouts_[NDB_628_u,NDB_628_u,NDB_628_u] got hits:76, dumped\n",
      "--- Sequence:loc_2:164374095-164404095_gene_Slpi:-_pb_74_pos_5971_strand_+_readouts_[NDB_812_c,NDB_812_c,NDB_812_c] got hits:64, dumped\n",
      "--- Sequence:loc_2:164374095-164404095_gene_Slpi:-_pb_76_pos_6079_strand_+_readouts_[NDB_473_u,NDB_473_u,NDB_473_u] got hits:70, dumped\n",
      "--- Sequence:loc_2:164374095-164404095_gene_Slpi:-_pb_77_pos_6171_strand_-_readouts_[NDB_652_c,NDB_652_c,NDB_652_c] got hits:106, dumped\n",
      "--- Sequence:loc_2:164374095-164404095_gene_Slpi:-_pb_78_pos_6225_strand_+_readouts_[NDB_812_c,NDB_812_c,NDB_812_c] got hits:80, dumped\n",
      "--- Sequence:loc_2:164374095-164404095_gene_Slpi:-_pb_79_pos_6278_strand_+_readouts_[NDB_715_c,NDB_715_c,NDB_715_c] got hits:85, dumped\n",
      "--- Sequence:loc_2:164374095-164404095_gene_Slpi:-_pb_81_pos_6377_strand_+_readouts_[NDB_652_c,NDB_652_c,NDB_652_c] got hits:74, dumped\n",
      "--- Sequence:loc_2:164374095-164404095_gene_Slpi:-_pb_82_pos_6449_strand_-_readouts_[NDB_812_c,NDB_812_c,NDB_812_c] got hits:104, dumped\n",
      "--- Sequence:loc_2:164374095-164404095_gene_Slpi:-_pb_84_pos_6563_strand_+_readouts_[NDB_473_u,NDB_473_u,NDB_473_u] got hits:94, dumped\n",
      "--- Sequence:loc_2:164374095-164404095_gene_Slpi:-_pb_85_pos_6633_strand_-_readouts_[NDB_652_c,NDB_652_c,NDB_652_c] got hits:66, dumped\n",
      "--- Sequence:loc_2:164374095-164404095_gene_Slpi:-_pb_86_pos_6709_strand_-_readouts_[NDB_812_c,NDB_812_c,NDB_812_c] got hits:58, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_274_pos_26586_strand_-_readouts_[NDB_812_c,NDB_812_c,NDB_812_c] got hits:139, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_275_pos_26628_strand_+_readouts_[NDB_698_c,NDB_698_c,NDB_698_c] got hits:73, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_276_pos_26702_strand_-_readouts_[NDB_526_u,NDB_526_u,NDB_526_u] got hits:82, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_277_pos_26921_strand_+_readouts_[NDB_901_c,NDB_901_c,NDB_901_c] got hits:123, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_278_pos_27038_strand_-_readouts_[NDB_812_c,NDB_812_c,NDB_812_c] got hits:52, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_279_pos_27090_strand_-_readouts_[NDB_698_c,NDB_698_c,NDB_698_c] got hits:97, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_280_pos_27228_strand_+_readouts_[NDB_526_u,NDB_526_u,NDB_526_u] got hits:114, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_281_pos_27326_strand_-_readouts_[NDB_901_c,NDB_901_c,NDB_901_c] got hits:102, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_282_pos_27413_strand_+_readouts_[NDB_812_c,NDB_812_c,NDB_812_c] got hits:129, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_283_pos_27513_strand_-_readouts_[NDB_698_c,NDB_698_c,NDB_698_c] got hits:113, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_284_pos_27554_strand_+_readouts_[NDB_526_u,NDB_526_u,NDB_526_u] got hits:96, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_285_pos_27611_strand_+_readouts_[NDB_901_c,NDB_901_c,NDB_901_c] got hits:89, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_286_pos_27655_strand_+_readouts_[NDB_812_c,NDB_812_c,NDB_812_c] got hits:97, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_287_pos_27732_strand_+_readouts_[NDB_698_c,NDB_698_c,NDB_698_c] got hits:83, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_288_pos_27779_strand_-_readouts_[NDB_526_u,NDB_526_u,NDB_526_u] got hits:138, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_289_pos_27884_strand_-_readouts_[NDB_901_c,NDB_901_c,NDB_901_c] got hits:90, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_290_pos_27941_strand_+_readouts_[NDB_812_c,NDB_812_c,NDB_812_c] got hits:58, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_291_pos_27997_strand_+_readouts_[NDB_698_c,NDB_698_c,NDB_698_c] got hits:81, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_293_pos_28080_strand_+_readouts_[NDB_901_c,NDB_901_c,NDB_901_c] got hits:143, dumped\n",
      "--- Sequence:loc_6:7539879-7569879_gene_Tac1:+_pb_294_pos_28152_strand_-_readouts_[NDB_812_c,NDB_812_c,NDB_812_c] got hits:124, dumped\n",
      "-- total probes removed by internal screening: 119\n"
     ]
    }
   ],
   "source": [
    "# check between probes\n",
    "kept_records, removed_count = check._check_between_probes(full_records, int_map, \n",
    "                                                          primer_len=primer_len, target_len=target_len)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "drawn-maria",
   "metadata": {},
   "outputs": [],
   "source": [
    "del(int_map)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "thousand-sister",
   "metadata": {},
   "source": [
    "### 4.1.6 finish checking, save"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "clear-cosmetic",
   "metadata": {},
   "outputs": [],
   "source": [
    "# save kept records\n",
    "with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'w') as output_handle:\n",
    "    SeqIO.write(kept_records, output_handle, \"fasta\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "mineral-status",
   "metadata": {},
   "source": [
    "## 4.2 BLAST screening of probes\n",
    "\n",
    "create the local blast database by:\n",
    "```bash\n",
    "makeblastdb -in C:\\Users\\puzheng\\Documents\\Tools\\lowercase_masked\\Mus_musculus.GRCm38.dna_sm.merged.fa -dbtype nucl -parse_seqids -out GRCm38_ensembl -title \"Mouse chromosomes soft masked\" -blastdb_version 4![image.png](attachment:image.png)\n",
    "```\n",
    "\n",
    "The outputs are:\n",
    "```bash\n",
    "Building a new DB, current time: 03/30/2021 21:49:18\n",
    "New DB name:   C:\\Users\\puzheng\\Documents\\Tools\\GRCm38_ensembl\n",
    "New DB title:  Mouse chromosomes soft masked\n",
    "Sequence type: Nucleotide\n",
    "Keep MBits: T\n",
    "Maximum file size: 1000000000B\n",
    "Adding sequences from FASTA; added 66 sequences in 42.2668 seconds.\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "elder-mexico",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Blast probe against \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\GRCm38_ensembl\\blast_database\\GRCm38_ensembl\n",
      "-- number of probes loaded: 60243\n",
      "-- processing region: Acpp:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Acpp:-.fasta\n",
      "--- *blasting region: Acpp:-\n",
      "--- total time for blast Acpp:-: 27.854001760482788\n",
      "-- processing region: Agt:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Agt:-.fasta\n",
      "--- *blasting region: Agt:-\n",
      "--- total time for blast Agt:-: 51.85732460021973\n",
      "-- processing region: Aldh3a1:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Aldh3a1:+.fasta\n",
      "--- *blasting region: Aldh3a1:+\n",
      "--- total time for blast Aldh3a1:+: 29.487189531326294\n",
      "-- processing region: Aldob:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Aldob:-.fasta\n",
      "--- *blasting region: Aldob:-\n",
      "--- total time for blast Aldob:-: 32.75022292137146\n",
      "-- processing region: Alox5ap:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Alox5ap:+.fasta\n",
      "--- *blasting region: Alox5ap:+\n",
      "--- total time for blast Alox5ap:+: 33.147716760635376\n",
      "-- processing region: Alpi:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Alpi:-.fasta\n",
      "--- *blasting region: Alpi:-\n",
      "--- total time for blast Alpi:-: 25.099304676055908\n",
      "-- processing region: Alpl:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Alpl:-.fasta\n",
      "--- *blasting region: Alpl:-\n",
      "--- total time for blast Alpl:-: 35.19901776313782\n",
      "-- processing region: Ang4:- with 175 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ang4:-.fasta\n",
      "--- *blasting region: Ang4:-\n",
      "--- total time for blast Ang4:-: 83.18445062637329\n",
      "-- processing region: Anxa8:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Anxa8:+.fasta\n",
      "--- *blasting region: Anxa8:+\n",
      "--- total time for blast Anxa8:+: 20.839163541793823\n",
      "-- processing region: Apoa1:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Apoa1:+.fasta\n",
      "--- *blasting region: Apoa1:+\n",
      "--- total time for blast Apoa1:+: 22.7909996509552\n",
      "-- processing region: Apoa2:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Apoa2:+.fasta\n",
      "--- *blasting region: Apoa2:+\n",
      "--- total time for blast Apoa2:+: 32.540807485580444\n",
      "-- processing region: Apoc2:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Apoc2:-.fasta\n",
      "--- *blasting region: Apoc2:-\n",
      "--- total time for blast Apoc2:-: 36.02543878555298\n",
      "-- processing region: Apod:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Apod:-.fasta\n",
      "--- *blasting region: Apod:-\n",
      "--- total time for blast Apod:-: 26.284001350402832\n",
      "-- processing region: Ascl1:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ascl1:-.fasta\n",
      "--- *blasting region: Ascl1:-\n",
      "--- total time for blast Ascl1:-: 27.851239442825317\n",
      "-- processing region: Avil:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Avil:+.fasta\n",
      "--- *blasting region: Avil:+\n",
      "--- total time for blast Avil:+: 15.353884935379028\n",
      "-- processing region: Bgn:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Bgn:+.fasta\n",
      "--- *blasting region: Bgn:+\n",
      "--- total time for blast Bgn:+: 26.16600012779236\n",
      "-- processing region: C1qb:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_C1qb:-.fasta\n",
      "--- *blasting region: C1qb:-\n",
      "--- total time for blast C1qb:-: 28.94277811050415\n",
      "-- processing region: C3:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_C3:-.fasta\n",
      "--- *blasting region: C3:-\n",
      "--- total time for blast C3:-: 36.90557909011841\n",
      "-- processing region: Calb2:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Calb2:-.fasta\n",
      "--- *blasting region: Calb2:-\n",
      "--- total time for blast Calb2:-: 16.18400001525879\n",
      "-- processing region: Cald1:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Cald1:+.fasta\n",
      "--- *blasting region: Cald1:+\n",
      "--- total time for blast Cald1:+: 43.70377540588379\n",
      "-- processing region: Camp:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Camp:-.fasta\n",
      "--- *blasting region: Camp:-\n",
      "--- total time for blast Camp:-: 25.20500135421753\n",
      "-- processing region: Car4:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Car4:+.fasta\n",
      "--- *blasting region: Car4:+\n",
      "--- total time for blast Car4:+: 33.36202263832092\n",
      "-- processing region: Car8:- with 298 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Car8:-.fasta\n",
      "--- *blasting region: Car8:-\n",
      "--- total time for blast Car8:-: 31.80901527404785\n",
      "-- processing region: Ccdc30:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ccdc30:-.fasta\n",
      "--- *blasting region: Ccdc30:-\n",
      "--- total time for blast Ccdc30:-: 49.21833920478821\n",
      "-- processing region: Cck:- with 295 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Cck:-.fasta\n",
      "--- *blasting region: Cck:-\n",
      "--- total time for blast Cck:-: 27.061463356018066\n",
      "-- processing region: Ccl1:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ccl1:-.fasta\n",
      "--- *blasting region: Ccl1:-\n",
      "--- total time for blast Ccl1:-: 59.66955924034119\n",
      "-- processing region: Ccl20:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ccl20:+.fasta\n",
      "--- *blasting region: Ccl20:+\n",
      "--- total time for blast Ccl20:+: 28.097837924957275\n",
      "-- processing region: Ccl2:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ccl2:+.fasta\n",
      "--- *blasting region: Ccl2:+\n",
      "--- total time for blast Ccl2:+: 27.923444032669067\n",
      "-- processing region: Ccl6:- with 200 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ccl6:-.fasta\n",
      "--- *blasting region: Ccl6:-\n",
      "--- total time for blast Ccl6:-: 19.089410066604614\n",
      "-- processing region: Ccl9:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ccl9:-.fasta\n",
      "--- *blasting region: Ccl9:-\n",
      "--- total time for blast Ccl9:-: 25.907999515533447\n",
      "-- processing region: Cd14:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Cd14:-.fasta\n",
      "--- *blasting region: Cd14:-\n",
      "--- total time for blast Cd14:-: 45.96295523643494\n",
      "-- processing region: Cebpb:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Cebpb:+.fasta\n",
      "--- *blasting region: Cebpb:+\n",
      "--- total time for blast Cebpb:+: 29.3145649433136\n",
      "-- processing region: Cga:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Cga:+.fasta\n",
      "--- *blasting region: Cga:+\n",
      "--- total time for blast Cga:+: 44.98504400253296\n",
      "-- processing region: Cilp:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Cilp:+.fasta\n",
      "--- *blasting region: Cilp:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Cilp:+: 35.61765432357788\n",
      "-- processing region: Ckb:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ckb:-.fasta\n",
      "--- *blasting region: Ckb:-\n",
      "--- total time for blast Ckb:-: 26.817999601364136\n",
      "-- processing region: Ckm:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ckm:+.fasta\n",
      "--- *blasting region: Ckm:+\n",
      "--- total time for blast Ckm:+: 26.764604806900024\n",
      "-- processing region: Clps:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Clps:-.fasta\n",
      "--- *blasting region: Clps:-\n",
      "--- total time for blast Clps:-: 37.27298021316528\n",
      "-- processing region: Cnfn:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Cnfn:-.fasta\n",
      "--- *blasting region: Cnfn:-\n",
      "--- total time for blast Cnfn:-: 31.26802921295166\n",
      "-- processing region: Cnmd:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Cnmd:-.fasta\n",
      "--- *blasting region: Cnmd:-\n",
      "--- total time for blast Cnmd:-: 25.95000171661377\n",
      "-- processing region: Col1a1:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Col1a1:+.fasta\n",
      "--- *blasting region: Col1a1:+\n",
      "--- total time for blast Col1a1:+: 17.637000560760498\n",
      "-- processing region: Col4a1:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Col4a1:-.fasta\n",
      "--- *blasting region: Col4a1:-\n",
      "--- total time for blast Col4a1:-: 29.133001804351807\n",
      "-- processing region: Col6a1:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Col6a1:-.fasta\n",
      "--- *blasting region: Col6a1:-\n",
      "--- total time for blast Col6a1:-: 79.17779326438904\n",
      "-- processing region: Cpn1:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Cpn1:-.fasta\n",
      "--- *blasting region: Cpn1:-\n",
      "--- total time for blast Cpn1:-: 54.647106885910034\n",
      "-- processing region: Crabp1:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Crabp1:+.fasta\n",
      "--- *blasting region: Crabp1:+\n",
      "--- total time for blast Crabp1:+: 24.40615677833557\n",
      "-- processing region: Cryaa:+ with 298 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Cryaa:+.fasta\n",
      "--- *blasting region: Cryaa:+\n",
      "--- total time for blast Cryaa:+: 43.643637895584106\n",
      "-- processing region: Ctrb1:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ctrb1:-.fasta\n",
      "--- *blasting region: Ctrb1:-\n",
      "--- total time for blast Ctrb1:-: 24.679771900177002\n",
      "-- processing region: Cxcl13:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Cxcl13:+.fasta\n",
      "--- *blasting region: Cxcl13:+\n",
      "--- total time for blast Cxcl13:+: 26.599658727645874\n",
      "-- processing region: Cyp1a2:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Cyp1a2:-.fasta\n",
      "--- *blasting region: Cyp1a2:-\n",
      "--- total time for blast Cyp1a2:-: 54.238237142562866\n",
      "-- processing region: Cyp2f2:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Cyp2f2:+.fasta\n",
      "--- *blasting region: Cyp2f2:+\n",
      "--- total time for blast Cyp2f2:+: 31.39371681213379\n",
      "-- processing region: Dbn1:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Dbn1:-.fasta\n",
      "--- *blasting region: Dbn1:-\n",
      "--- total time for blast Dbn1:-: 40.31660866737366\n",
      "-- processing region: Dclk1:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Dclk1:+.fasta\n",
      "--- *blasting region: Dclk1:+\n",
      "--- total time for blast Dclk1:+: 31.332133293151855\n",
      "-- processing region: Dlk1:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Dlk1:+.fasta\n",
      "--- *blasting region: Dlk1:+\n",
      "--- total time for blast Dlk1:+: 21.41300058364868\n",
      "-- processing region: Dppa3:+ with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Dppa3:+.fasta\n",
      "--- *blasting region: Dppa3:+\n",
      "--- total time for blast Dppa3:+: 176.61443710327148\n",
      "-- processing region: Ecm1:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ecm1:-.fasta\n",
      "--- *blasting region: Ecm1:-\n",
      "--- total time for blast Ecm1:-: 55.60395646095276\n",
      "-- processing region: Efna3:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Efna3:-.fasta\n",
      "--- *blasting region: Efna3:-\n",
      "--- total time for blast Efna3:-: 22.79399800300598\n",
      "-- processing region: Elovl4:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Elovl4:-.fasta\n",
      "--- *blasting region: Elovl4:-\n",
      "--- total time for blast Elovl4:-: 24.63499927520752\n",
      "-- processing region: Fabp1:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Fabp1:+.fasta\n",
      "--- *blasting region: Fabp1:+\n",
      "--- total time for blast Fabp1:+: 27.93300199508667\n",
      "-- processing region: Fabp6:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Fabp6:-.fasta\n",
      "--- *blasting region: Fabp6:-\n",
      "--- total time for blast Fabp6:-: 51.48018789291382\n",
      "-- processing region: Fam183b:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Fam183b:-.fasta\n",
      "--- *blasting region: Fam183b:-\n",
      "--- total time for blast Fam183b:-: 44.813772201538086\n",
      "-- processing region: Fam25c:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Fam25c:-.fasta\n",
      "--- *blasting region: Fam25c:-\n",
      "--- total time for blast Fam25c:-: 50.72065091133118\n",
      "-- processing region: Fgg:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Fgg:+.fasta\n",
      "--- *blasting region: Fgg:+\n",
      "--- total time for blast Fgg:+: 37.68916368484497\n",
      "-- processing region: Foxa1:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Foxa1:-.fasta\n",
      "--- *blasting region: Foxa1:-\n",
      "--- total time for blast Foxa1:-: 30.476847648620605\n",
      "-- processing region: Foxa2:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Foxa2:-.fasta\n",
      "--- *blasting region: Foxa2:-\n",
      "--- total time for blast Foxa2:-: 26.03247308731079\n",
      "-- processing region: Foxa3:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Foxa3:-.fasta\n",
      "--- *blasting region: Foxa3:-\n",
      "--- total time for blast Foxa3:-: 26.976001024246216\n",
      "-- processing region: Foxj1:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Foxj1:-.fasta\n",
      "--- *blasting region: Foxj1:-\n",
      "--- total time for blast Foxj1:-: 29.4875271320343\n",
      "-- processing region: Fxyd3:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Fxyd3:-.fasta\n",
      "--- *blasting region: Fxyd3:-\n",
      "--- total time for blast Fxyd3:-: 42.44445872306824\n",
      "-- processing region: Fxyd6:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Fxyd6:+.fasta\n",
      "--- *blasting region: Fxyd6:+\n",
      "--- total time for blast Fxyd6:+: 18.430001735687256\n",
      "-- processing region: Gabrp:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Gabrp:-.fasta\n",
      "--- *blasting region: Gabrp:-\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Gabrp:-: 33.45302176475525\n",
      "-- processing region: Gal:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Gal:-.fasta\n",
      "--- *blasting region: Gal:-\n",
      "--- total time for blast Gal:-: 20.840999841690063\n",
      "-- processing region: Gatm:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Gatm:-.fasta\n",
      "--- *blasting region: Gatm:-\n",
      "--- total time for blast Gatm:-: 30.25599956512451\n",
      "-- processing region: Ghrl:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ghrl:-.fasta\n",
      "--- *blasting region: Ghrl:-\n",
      "--- total time for blast Ghrl:-: 25.295257091522217\n",
      "-- processing region: Gm13539:+ with 228 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Gm13539:+.fasta\n",
      "--- *blasting region: Gm13539:+\n",
      "--- total time for blast Gm13539:+: 29.12114119529724\n",
      "-- processing region: Gm2a:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Gm2a:+.fasta\n",
      "--- *blasting region: Gm2a:+\n",
      "--- total time for blast Gm2a:+: 20.286999702453613\n",
      "-- processing region: Gm94:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Gm94:-.fasta\n",
      "--- *blasting region: Gm94:-\n",
      "--- total time for blast Gm94:-: 34.9642379283905\n",
      "-- processing region: Gp2:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Gp2:-.fasta\n",
      "--- *blasting region: Gp2:-\n",
      "--- total time for blast Gp2:-: 21.273000717163086\n",
      "-- processing region: Gpx3:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Gpx3:+.fasta\n",
      "--- *blasting region: Gpx3:+\n",
      "--- total time for blast Gpx3:+: 44.24056673049927\n",
      "-- processing region: Gpx6:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Gpx6:+.fasta\n",
      "--- *blasting region: Gpx6:+\n",
      "--- total time for blast Gpx6:+: 21.687001705169678\n",
      "-- processing region: Grhl1:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Grhl1:+.fasta\n",
      "--- *blasting region: Grhl1:+\n",
      "--- total time for blast Grhl1:+: 26.64399790763855\n",
      "-- processing region: Gsdma:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Gsdma:+.fasta\n",
      "--- *blasting region: Gsdma:+\n",
      "--- total time for blast Gsdma:+: 55.53229784965515\n",
      "-- processing region: Gsta2:- with 274 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Gsta2:-.fasta\n",
      "--- *blasting region: Gsta2:-\n",
      "--- total time for blast Gsta2:-: 175.08213448524475\n",
      "-- processing region: Gsta4:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Gsta4:+.fasta\n",
      "--- *blasting region: Gsta4:+\n",
      "--- total time for blast Gsta4:+: 39.09202241897583\n",
      "-- processing region: Gstm3:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Gstm3:-.fasta\n",
      "--- *blasting region: Gstm3:-\n",
      "--- total time for blast Gstm3:-: 21.320998907089233\n",
      "-- processing region: Gsto1:+ with 294 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Gsto1:+.fasta\n",
      "--- *blasting region: Gsto1:+\n",
      "--- total time for blast Gsto1:+: 31.45580792427063\n",
      "-- processing region: Guca2a:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Guca2a:+.fasta\n",
      "--- *blasting region: Guca2a:+\n",
      "--- total time for blast Guca2a:+: 25.47800064086914\n",
      "-- processing region: H19:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_H19:-.fasta\n",
      "--- *blasting region: H19:-\n",
      "--- total time for blast H19:-: 22.29099988937378\n",
      "-- processing region: Hamp2:- with 238 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Hamp2:-.fasta\n",
      "--- *blasting region: Hamp2:-\n",
      "--- total time for blast Hamp2:-: 20.202886819839478\n",
      "-- processing region: Hba-a2:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Hba-a2:+.fasta\n",
      "--- *blasting region: Hba-a2:+\n",
      "--- total time for blast Hba-a2:+: 37.443501234054565\n",
      "-- processing region: Hbb-y:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Hbb-y:-.fasta\n",
      "--- *blasting region: Hbb-y:-\n",
      "--- total time for blast Hbb-y:-: 45.31233859062195\n",
      "-- processing region: Hck:+ with 266 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Hck:+.fasta\n",
      "--- *blasting region: Hck:+\n",
      "--- total time for blast Hck:+: 20.34900188446045\n",
      "-- processing region: Hnf4a:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Hnf4a:+.fasta\n",
      "--- *blasting region: Hnf4a:+\n",
      "--- total time for blast Hnf4a:+: 41.422048568725586\n",
      "-- processing region: Hnf4g:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Hnf4g:+.fasta\n",
      "--- *blasting region: Hnf4g:+\n",
      "--- total time for blast Hnf4g:+: 40.948397397994995\n",
      "-- processing region: Hp:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Hp:-.fasta\n",
      "--- *blasting region: Hp:-\n",
      "--- total time for blast Hp:-: 17.23599934577942\n",
      "-- processing region: Iapp:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Iapp:+.fasta\n",
      "--- *blasting region: Iapp:+\n",
      "--- total time for blast Iapp:+: 42.96697807312012\n",
      "-- processing region: Idi2:+ with 243 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Idi2:+.fasta\n",
      "--- *blasting region: Idi2:+\n",
      "--- total time for blast Idi2:+: 19.292999029159546\n",
      "-- processing region: Ifi27l2b:- with 193 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ifi27l2b:-.fasta\n",
      "--- *blasting region: Ifi27l2b:-\n",
      "--- total time for blast Ifi27l2b:-: 61.73206543922424\n",
      "-- processing region: Igf1:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Igf1:+.fasta\n",
      "--- *blasting region: Igf1:+\n",
      "--- total time for blast Igf1:+: 22.695002794265747\n",
      "-- processing region: Igf2:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Igf2:-.fasta\n",
      "--- *blasting region: Igf2:-\n",
      "--- total time for blast Igf2:-: 20.183997631072998\n",
      "-- processing region: Igfbp5:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Igfbp5:-.fasta\n",
      "--- *blasting region: Igfbp5:-\n",
      "--- total time for blast Igfbp5:-: 40.43601131439209\n",
      "-- processing region: Igfbp6:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Igfbp6:+.fasta\n",
      "--- *blasting region: Igfbp6:+\n",
      "--- total time for blast Igfbp6:+: 30.715787649154663\n",
      "-- processing region: Ins2:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ins2:-.fasta\n",
      "--- *blasting region: Ins2:-\n",
      "--- total time for blast Ins2:-: 83.39453601837158\n",
      "-- processing region: Isg15:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Isg15:-.fasta\n",
      "--- *blasting region: Isg15:-\n",
      "--- total time for blast Isg15:-: 26.09200119972229\n",
      "-- processing region: Itga6:+ with 295 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Itga6:+.fasta\n",
      "--- *blasting region: Itga6:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Itga6:+: 19.453999757766724\n",
      "-- processing region: Jpt1:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Jpt1:-.fasta\n",
      "--- *blasting region: Jpt1:-\n",
      "--- total time for blast Jpt1:-: 45.60643124580383\n",
      "-- processing region: Klk14:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Klk14:+.fasta\n",
      "--- *blasting region: Klk14:+\n",
      "--- total time for blast Klk14:+: 26.691261768341064\n",
      "-- processing region: Klk7:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Klk7:+.fasta\n",
      "--- *blasting region: Klk7:+\n",
      "--- total time for blast Klk7:+: 56.79631781578064\n",
      "-- processing region: Krt10:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Krt10:-.fasta\n",
      "--- *blasting region: Krt10:-\n",
      "--- total time for blast Krt10:-: 32.67199945449829\n",
      "-- processing region: Krt13:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Krt13:-.fasta\n",
      "--- *blasting region: Krt13:-\n",
      "--- total time for blast Krt13:-: 19.368000745773315\n",
      "-- processing region: Krt14:- with 195 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Krt14:-.fasta\n",
      "--- *blasting region: Krt14:-\n",
      "--- total time for blast Krt14:-: 26.98841381072998\n",
      "-- processing region: Krt16:- with 109 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Krt16:-.fasta\n",
      "--- *blasting region: Krt16:-\n",
      "--- total time for blast Krt16:-: 14.240000486373901\n",
      "-- processing region: Krt17:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Krt17:-.fasta\n",
      "--- *blasting region: Krt17:-\n",
      "--- total time for blast Krt17:-: 18.92799973487854\n",
      "-- processing region: Krt19:- with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Krt19:-.fasta\n",
      "--- *blasting region: Krt19:-\n",
      "--- total time for blast Krt19:-: 23.770395755767822\n",
      "-- processing region: Krt5:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Krt5:-.fasta\n",
      "--- *blasting region: Krt5:-\n",
      "--- total time for blast Krt5:-: 18.835999965667725\n",
      "-- processing region: Krt71:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Krt71:-.fasta\n",
      "--- *blasting region: Krt71:-\n",
      "--- total time for blast Krt71:-: 24.702192544937134\n",
      "-- processing region: Krt76:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Krt76:-.fasta\n",
      "--- *blasting region: Krt76:-\n",
      "--- total time for blast Krt76:-: 38.6921648979187\n",
      "-- processing region: Krt77:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Krt77:-.fasta\n",
      "--- *blasting region: Krt77:-\n",
      "--- total time for blast Krt77:-: 19.822000741958618\n",
      "-- processing region: Krt9:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Krt9:-.fasta\n",
      "--- *blasting region: Krt9:-\n",
      "--- total time for blast Krt9:-: 28.228177309036255\n",
      "-- processing region: Krtdap:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Krtdap:+.fasta\n",
      "--- *blasting region: Krtdap:+\n",
      "--- total time for blast Krtdap:+: 36.22653913497925\n",
      "-- processing region: Lcn2:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Lcn2:-.fasta\n",
      "--- *blasting region: Lcn2:-\n",
      "--- total time for blast Lcn2:-: 21.79699683189392\n",
      "-- processing region: Lcn3:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Lcn3:+.fasta\n",
      "--- *blasting region: Lcn3:+\n",
      "--- total time for blast Lcn3:+: 77.94216084480286\n",
      "-- processing region: Lgals2:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Lgals2:-.fasta\n",
      "--- *blasting region: Lgals2:-\n",
      "--- total time for blast Lgals2:-: 26.83177137374878\n",
      "-- processing region: Lgals3:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Lgals3:+.fasta\n",
      "--- *blasting region: Lgals3:+\n",
      "--- total time for blast Lgals3:+: 29.79700255393982\n",
      "-- processing region: Lifr:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Lifr:+.fasta\n",
      "--- *blasting region: Lifr:+\n",
      "--- total time for blast Lifr:+: 25.896000146865845\n",
      "-- processing region: Ltf:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ltf:+.fasta\n",
      "--- *blasting region: Ltf:+\n",
      "--- total time for blast Ltf:+: 48.10684657096863\n",
      "-- processing region: Ly6c1:- with 126 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ly6c1:-.fasta\n",
      "--- *blasting region: Ly6c1:-\n",
      "--- total time for blast Ly6c1:-: 24.28769326210022\n",
      "-- processing region: Ly6c2:- with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ly6c2:-.fasta\n",
      "--- *blasting region: Ly6c2:-\n",
      "--- total time for blast Ly6c2:-: 76.00926232337952\n",
      "-- processing region: Ly6d:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ly6d:-.fasta\n",
      "--- *blasting region: Ly6d:-\n",
      "--- total time for blast Ly6d:-: 33.53857660293579\n",
      "-- processing region: Ly6g6f:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ly6g6f:-.fasta\n",
      "--- *blasting region: Ly6g6f:-\n",
      "--- total time for blast Ly6g6f:-: 26.018903493881226\n",
      "-- processing region: Lypd2:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Lypd2:-.fasta\n",
      "--- *blasting region: Lypd2:-\n",
      "--- total time for blast Lypd2:-: 19.854512214660645\n",
      "-- processing region: Lypd3:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Lypd3:+.fasta\n",
      "--- *blasting region: Lypd3:+\n",
      "--- total time for blast Lypd3:+: 28.50300097465515\n",
      "-- processing region: Mal:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Mal:-.fasta\n",
      "--- *blasting region: Mal:-\n",
      "--- total time for blast Mal:-: 26.760387182235718\n",
      "-- processing region: Meg3:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Meg3:+.fasta\n",
      "--- *blasting region: Meg3:+\n",
      "--- total time for blast Meg3:+: 24.552000522613525\n",
      "-- processing region: Meig1:- with 284 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Meig1:-.fasta\n",
      "--- *blasting region: Meig1:-\n",
      "--- total time for blast Meig1:-: 34.67103171348572\n",
      "-- processing region: Mgp:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Mgp:-.fasta\n",
      "--- *blasting region: Mgp:-\n",
      "--- total time for blast Mgp:-: 28.688628911972046\n",
      "-- processing region: Mlph:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Mlph:+.fasta\n",
      "--- *blasting region: Mlph:+\n",
      "--- total time for blast Mlph:+: 24.153629302978516\n",
      "-- processing region: Mpo:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Mpo:+.fasta\n",
      "--- *blasting region: Mpo:+\n",
      "--- total time for blast Mpo:+: 22.338000297546387\n",
      "-- processing region: Mt3:+ with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Mt3:+.fasta\n",
      "--- *blasting region: Mt3:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Mt3:+: 17.86700177192688\n",
      "-- processing region: Mt4:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Mt4:+.fasta\n",
      "--- *blasting region: Mt4:+\n",
      "--- total time for blast Mt4:+: 24.226999044418335\n",
      "-- processing region: Muc13:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Muc13:+.fasta\n",
      "--- *blasting region: Muc13:+\n",
      "--- total time for blast Muc13:+: 38.94497990608215\n",
      "-- processing region: Mymx:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Mymx:-.fasta\n",
      "--- *blasting region: Mymx:-\n",
      "--- total time for blast Mymx:-: 21.89799952507019\n",
      "-- processing region: Myog:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Myog:+.fasta\n",
      "--- *blasting region: Myog:+\n",
      "--- total time for blast Myog:+: 36.670111894607544\n",
      "-- processing region: Nefh:- with 297 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Nefh:-.fasta\n",
      "--- *blasting region: Nefh:-\n",
      "--- total time for blast Nefh:-: 33.52408528327942\n",
      "-- processing region: Ngp:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ngp:+.fasta\n",
      "--- *blasting region: Ngp:+\n",
      "--- total time for blast Ngp:+: 29.099337577819824\n",
      "-- processing region: Nppa:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Nppa:+.fasta\n",
      "--- *blasting region: Nppa:+\n",
      "--- total time for blast Nppa:+: 43.59628129005432\n",
      "-- processing region: Npy:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Npy:+.fasta\n",
      "--- *blasting region: Npy:+\n",
      "--- total time for blast Npy:+: 25.15137553215027\n",
      "-- processing region: Nrgn:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Nrgn:-.fasta\n",
      "--- *blasting region: Nrgn:-\n",
      "--- total time for blast Nrgn:-: 21.243996143341064\n",
      "-- processing region: Nts:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Nts:-.fasta\n",
      "--- *blasting region: Nts:-\n",
      "--- total time for blast Nts:-: 28.690001964569092\n",
      "-- processing region: Oas1d:+ with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Oas1d:+.fasta\n",
      "--- *blasting region: Oas1d:+\n",
      "--- total time for blast Oas1d:+: 46.92449450492859\n",
      "-- processing region: Oas1h:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Oas1h:+.fasta\n",
      "--- *blasting region: Oas1h:+\n",
      "--- total time for blast Oas1h:+: 45.69177722930908\n",
      "-- processing region: Odf3b:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Odf3b:-.fasta\n",
      "--- *blasting region: Odf3b:-\n",
      "--- total time for blast Odf3b:-: 18.083000421524048\n",
      "-- processing region: Olfm1:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Olfm1:+.fasta\n",
      "--- *blasting region: Olfm1:+\n",
      "--- total time for blast Olfm1:+: 27.588000535964966\n",
      "-- processing region: Pcp4:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Pcp4:+.fasta\n",
      "--- *blasting region: Pcp4:+\n",
      "--- total time for blast Pcp4:+: 35.76228904724121\n",
      "-- processing region: Pdzk1ip1:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Pdzk1ip1:+.fasta\n",
      "--- *blasting region: Pdzk1ip1:+\n",
      "--- total time for blast Pdzk1ip1:+: 22.114999055862427\n",
      "-- processing region: Penk:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Penk:-.fasta\n",
      "--- *blasting region: Penk:-\n",
      "--- total time for blast Penk:-: 38.26625204086304\n",
      "-- processing region: Pgc:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Pgc:+.fasta\n",
      "--- *blasting region: Pgc:+\n",
      "--- total time for blast Pgc:+: 23.85766911506653\n",
      "-- processing region: Pigr:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Pigr:+.fasta\n",
      "--- *blasting region: Pigr:+\n",
      "--- total time for blast Pigr:+: 19.453001976013184\n",
      "-- processing region: Pik3r5:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Pik3r5:+.fasta\n",
      "--- *blasting region: Pik3r5:+\n",
      "--- total time for blast Pik3r5:+: 31.426660299301147\n",
      "-- processing region: Pla2g1b:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Pla2g1b:+.fasta\n",
      "--- *blasting region: Pla2g1b:+\n",
      "--- total time for blast Pla2g1b:+: 33.78257656097412\n",
      "-- processing region: Plac8:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Plac8:-.fasta\n",
      "--- *blasting region: Plac8:-\n",
      "--- total time for blast Plac8:-: 29.69112753868103\n",
      "-- processing region: Plk2:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Plk2:+.fasta\n",
      "--- *blasting region: Plk2:+\n",
      "--- total time for blast Plk2:+: 59.9879105091095\n",
      "-- processing region: Plvap:- with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Plvap:-.fasta\n",
      "--- *blasting region: Plvap:-\n",
      "--- total time for blast Plvap:-: 31.15019178390503\n",
      "-- processing region: Pomc:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Pomc:+.fasta\n",
      "--- *blasting region: Pomc:+\n",
      "--- total time for blast Pomc:+: 33.663307905197144\n",
      "-- processing region: Pou2f3:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Pou2f3:-.fasta\n",
      "--- *blasting region: Pou2f3:-\n",
      "--- total time for blast Pou2f3:-: 24.77238154411316\n",
      "-- processing region: Psca:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Psca:+.fasta\n",
      "--- *blasting region: Psca:+\n",
      "--- total time for blast Psca:+: 25.895999670028687\n",
      "-- processing region: Ptgds:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ptgds:-.fasta\n",
      "--- *blasting region: Ptgds:-\n",
      "--- total time for blast Ptgds:-: 17.2509982585907\n",
      "-- processing region: Pyy:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Pyy:-.fasta\n",
      "--- *blasting region: Pyy:-\n",
      "--- total time for blast Pyy:-: 27.00753426551819\n",
      "-- processing region: Rap1gap2:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Rap1gap2:-.fasta\n",
      "--- *blasting region: Rap1gap2:-\n",
      "--- total time for blast Rap1gap2:-: 55.17457103729248\n",
      "-- processing region: Rarres1:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Rarres1:-.fasta\n",
      "--- *blasting region: Rarres1:-\n",
      "--- total time for blast Rarres1:-: 21.31900119781494\n",
      "-- processing region: Reg3b:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Reg3b:+.fasta\n",
      "--- *blasting region: Reg3b:+\n",
      "--- total time for blast Reg3b:+: 29.79343891143799\n",
      "-- processing region: Reg4:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Reg4:+.fasta\n",
      "--- *blasting region: Reg4:+\n",
      "--- total time for blast Reg4:+: 20.27000093460083\n",
      "-- processing region: Retnla:+ with 246 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Retnla:+.fasta\n",
      "--- *blasting region: Retnla:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Retnla:+: 90.42463755607605\n",
      "-- processing region: Riiad1:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Riiad1:-.fasta\n",
      "--- *blasting region: Riiad1:-\n",
      "--- total time for blast Riiad1:-: 21.012001276016235\n",
      "-- processing region: Rnase1:- with 228 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Rnase1:-.fasta\n",
      "--- *blasting region: Rnase1:-\n",
      "--- total time for blast Rnase1:-: 26.64335322380066\n",
      "-- processing region: Rnase4:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Rnase4:+.fasta\n",
      "--- *blasting region: Rnase4:+\n",
      "--- total time for blast Rnase4:+: 48.45356822013855\n",
      "-- processing region: Rsad2:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Rsad2:-.fasta\n",
      "--- *blasting region: Rsad2:-\n",
      "--- total time for blast Rsad2:-: 27.324000597000122\n",
      "-- processing region: Rsph1:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Rsph1:-.fasta\n",
      "--- *blasting region: Rsph1:-\n",
      "--- total time for blast Rsph1:-: 22.526000499725342\n",
      "-- processing region: S100a9:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_S100a9:-.fasta\n",
      "--- *blasting region: S100a9:-\n",
      "--- total time for blast S100a9:-: 35.36658549308777\n",
      "-- processing region: Saa1:- with 135 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Saa1:-.fasta\n",
      "--- *blasting region: Saa1:-\n",
      "--- total time for blast Saa1:-: 14.000043869018555\n",
      "-- processing region: Saa2:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Saa2:+.fasta\n",
      "--- *blasting region: Saa2:+\n",
      "--- total time for blast Saa2:+: 408.51230001449585\n",
      "-- processing region: Serpinb11:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Serpinb11:+.fasta\n",
      "--- *blasting region: Serpinb11:+\n",
      "--- total time for blast Serpinb11:+: 43.301687240600586\n",
      "-- processing region: Sftpa1:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Sftpa1:+.fasta\n",
      "--- *blasting region: Sftpa1:+\n",
      "--- total time for blast Sftpa1:+: 33.58064889907837\n",
      "-- processing region: Sftpd:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Sftpd:-.fasta\n",
      "--- *blasting region: Sftpd:-\n",
      "--- total time for blast Sftpd:-: 32.67578864097595\n",
      "-- processing region: Slpi:- with 290 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Slpi:-.fasta\n",
      "--- *blasting region: Slpi:-\n",
      "--- total time for blast Slpi:-: 16.46541714668274\n",
      "-- processing region: Sox8:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Sox8:-.fasta\n",
      "--- *blasting region: Sox8:-\n",
      "--- total time for blast Sox8:-: 26.11400055885315\n",
      "-- processing region: Spag8:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Spag8:-.fasta\n",
      "--- *blasting region: Spag8:-\n",
      "--- total time for blast Spag8:-: 30.537042140960693\n",
      "-- processing region: Sparc:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Sparc:-.fasta\n",
      "--- *blasting region: Sparc:-\n",
      "--- total time for blast Sparc:-: 30.794347524642944\n",
      "-- processing region: Spib:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Spib:-.fasta\n",
      "--- *blasting region: Spib:-\n",
      "--- total time for blast Spib:-: 61.904258251190186\n",
      "-- processing region: Spp1:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Spp1:+.fasta\n",
      "--- *blasting region: Spp1:+\n",
      "--- total time for blast Spp1:+: 42.19728469848633\n",
      "-- processing region: Sprr1b:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Sprr1b:-.fasta\n",
      "--- *blasting region: Sprr1b:-\n",
      "--- total time for blast Sprr1b:-: 54.55359435081482\n",
      "-- processing region: Sprr2i:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Sprr2i:+.fasta\n",
      "--- *blasting region: Sprr2i:+\n",
      "--- total time for blast Sprr2i:+: 49.91100001335144\n",
      "-- processing region: Sst:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Sst:-.fasta\n",
      "--- *blasting region: Sst:-\n",
      "--- total time for blast Sst:-: 27.91900134086609\n",
      "-- processing region: Stxbp5l:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Stxbp5l:-.fasta\n",
      "--- *blasting region: Stxbp5l:-\n",
      "--- total time for blast Stxbp5l:-: 32.464998722076416\n",
      "-- processing region: Syt7:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Syt7:+.fasta\n",
      "--- *blasting region: Syt7:+\n",
      "--- total time for blast Syt7:+: 21.515999794006348\n",
      "-- processing region: Tac1:+ with 280 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Tac1:+.fasta\n",
      "--- *blasting region: Tac1:+\n",
      "--- total time for blast Tac1:+: 24.661001920700073\n",
      "-- processing region: Tac2:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Tac2:+.fasta\n",
      "--- *blasting region: Tac2:+\n",
      "--- total time for blast Tac2:+: 30.416581630706787\n",
      "-- processing region: Tacstd2:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Tacstd2:-.fasta\n",
      "--- *blasting region: Tacstd2:-\n",
      "--- total time for blast Tacstd2:-: 34.228795528411865\n",
      "-- processing region: Tff2:- with 196 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Tff2:-.fasta\n",
      "--- *blasting region: Tff2:-\n",
      "--- total time for blast Tff2:-: 15.256006479263306\n",
      "-- processing region: Tff3:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Tff3:-.fasta\n",
      "--- *blasting region: Tff3:-\n",
      "--- total time for blast Tff3:-: 56.41916346549988\n",
      "-- processing region: Timp2:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Timp2:-.fasta\n",
      "--- *blasting region: Timp2:-\n",
      "--- total time for blast Timp2:-: 23.80388617515564\n",
      "-- processing region: Tmem45a:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Tmem45a:-.fasta\n",
      "--- *blasting region: Tmem45a:-\n",
      "--- total time for blast Tmem45a:-: 46.704998254776\n",
      "-- processing region: Tnfrsf11b:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Tnfrsf11b:-.fasta\n",
      "--- *blasting region: Tnfrsf11b:-\n",
      "--- total time for blast Tnfrsf11b:-: 34.95264649391174\n",
      "-- processing region: Trp63:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Trp63:+.fasta\n",
      "--- *blasting region: Trp63:+\n",
      "--- total time for blast Trp63:+: 235.22224283218384\n",
      "-- processing region: Trpm5:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Trpm5:-.fasta\n",
      "--- *blasting region: Trpm5:-\n",
      "--- total time for blast Trpm5:-: 27.091126918792725\n",
      "-- processing region: Tspan1:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Tspan1:-.fasta\n",
      "--- *blasting region: Tspan1:-\n",
      "--- total time for blast Tspan1:-: 21.798001050949097\n",
      "-- processing region: Ttc36:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Ttc36:-.fasta\n",
      "--- *blasting region: Ttc36:-\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast Ttc36:-: 30.132728338241577\n",
      "-- processing region: Vpreb3:+ with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Vpreb3:+.fasta\n",
      "--- *blasting region: Vpreb3:+\n",
      "--- total time for blast Vpreb3:+: 33.7911422252655\n",
      "-- processing region: Xcl1:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Xcl1:-.fasta\n",
      "--- *blasting region: Xcl1:-\n",
      "--- total time for blast Xcl1:-: 22.802998542785645\n",
      "-- processing region: Zg16:- with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\blast\\probe_gene_Zg16:-.fasta\n",
      "--- *blasting region: Zg16:-\n",
      "--- total time for blast Zg16:-: 37.00647568702698\n"
     ]
    }
   ],
   "source": [
    "blast_database = os.path.join(reference_folder, 'blast_database', 'GRCm38_ensembl')\n",
    "\n",
    "check.Blast_probes(kept_records, library_folder, num_threads=32, \n",
    "                   blast_db=blast_database, blast_evalue=100, overwrite=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "rubber-hayes",
   "metadata": {},
   "source": [
    "### 4.2.1 screen probe by blast results\n",
    "\n",
    "This step has 2 different cutoffs:\n",
    "\n",
    "hard_cutoff: whether a probe has any ~40mer hit somewhere else, if so just dump the probe\n",
    "\n",
    "soft_cutoff: whether a probe has a lot of ~20mer off-target hits, if exceed certain amount then dump the probe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "entertaining-example",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Number of probes loaded: 60243\n",
      "- Number of regions in this library: 207\n",
      "-- checking probes in region: Acpp:-\n",
      "--- gene=Acpp:-, id=19 removed by soft count = 57\n",
      "--- gene=Acpp:-, id=259 removed by soft count = 176\n",
      "--- gene=Acpp:-, id=261 removed by soft count = 219\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Agt:-\n",
      "--- gene=Agt:-, id=23 removed by soft count = 64\n",
      "--- gene=Agt:-, id=240 removed by soft count = 5686\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Aldh3a1:+\n",
      "--- gene=Aldh3a1:+, id=5 removed by soft count = 90\n",
      "--- gene=Aldh3a1:+, id=52 removed by soft count = 57\n",
      "--- gene=Aldh3a1:+, id=60 removed by soft count = 186\n",
      "--- gene=Aldh3a1:+, id=62 removed by soft count = 73\n",
      "--- gene=Aldh3a1:+, id=205 removed by soft count = 116\n",
      "--- gene=Aldh3a1:+, id=280 removed by soft count = 215\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Aldob:-\n",
      "--- gene=Aldob:-, id=122 removed by soft count = 111\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Alox5ap:+\n",
      "--- gene=Alox5ap:+, id=102 removed by soft count = 335\n",
      "--- gene=Alox5ap:+, id=103 removed by soft count = 98\n",
      "--- gene=Alox5ap:+, id=145 removed by soft count = 1551\n",
      "--- gene=Alox5ap:+, id=146 removed by soft count = 61\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Alpi:-\n",
      "--- gene=Alpi:-, id=5 removed by soft count = 190\n",
      "--- gene=Alpi:-, id=43 removed by soft count = 56\n",
      "--- gene=Alpi:-, id=44 removed by hard count = 0\n",
      "--- gene=Alpi:-, id=63 removed by soft count = 51\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Alpl:-\n",
      "--- gene=Alpl:-, id=27 removed by soft count = 211\n",
      "--- gene=Alpl:-, id=78 removed by soft count = 205\n",
      "--- gene=Alpl:-, id=79 removed by soft count = 159\n",
      "--- gene=Alpl:-, id=81 removed by soft count = 528\n",
      "--- gene=Alpl:-, id=146 removed by soft count = 280\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ang4:-\n",
      "--- gene=Ang4:-, id=40 removed by soft count = 5348\n",
      "--- gene=Ang4:-, id=41 removed by soft count = 2506\n",
      "--- gene=Ang4:-, id=42 removed by soft count = 79\n",
      "--- gene=Ang4:-, id=43 removed by soft count = 207\n",
      "--- gene=Ang4:-, id=85 removed by soft count = 495\n",
      "--- gene=Ang4:-, id=86 removed by soft count = 267\n",
      "--- gene=Ang4:-, id=88 removed by soft count = 133\n",
      "--- gene=Ang4:-, id=138 removed by soft count = 251\n",
      "--- number of probes: 175 , kept by blast: 167 , if remove dups: 167\n",
      "-- number of probes kept for this region: 167\n",
      "-- checking probes in region: Anxa8:+\n",
      "--- gene=Anxa8:+, id=242 removed by soft count = 51\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Apoa1:+\n",
      "--- gene=Apoa1:+, id=248 removed by soft count = 183\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Apoa2:+\n",
      "--- gene=Apoa2:+, id=13 removed by soft count = 442\n",
      "--- gene=Apoa2:+, id=17 removed by soft count = 307\n",
      "--- gene=Apoa2:+, id=18 removed by soft count = 324\n",
      "--- gene=Apoa2:+, id=131 removed by soft count = 249\n",
      "--- gene=Apoa2:+, id=202 removed by soft count = 63\n",
      "--- gene=Apoa2:+, id=226 removed by soft count = 200\n",
      "--- gene=Apoa2:+, id=238 removed by soft count = 131\n",
      "--- number of probes: 300 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Apoc2:-\n",
      "--- gene=Apoc2:-, id=63 removed by soft count = 622\n",
      "--- gene=Apoc2:-, id=65 removed by soft count = 56\n",
      "--- gene=Apoc2:-, id=174 removed by soft count = 63\n",
      "--- gene=Apoc2:-, id=214 removed by soft count = 52\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Apod:-\n",
      "--- gene=Apod:-, id=15 removed by soft count = 63\n",
      "--- gene=Apod:-, id=208 removed by soft count = 252\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ascl1:-\n",
      "--- gene=Ascl1:-, id=51 removed by soft count = 62\n",
      "--- gene=Ascl1:-, id=73 removed by soft count = 262\n",
      "--- gene=Ascl1:-, id=282 removed by soft count = 107\n",
      "--- gene=Ascl1:-, id=283 removed by soft count = 92\n",
      "--- gene=Ascl1:-, id=284 removed by soft count = 851\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Avil:+\n",
      "--- gene=Avil:+, id=119 removed by soft count = 82\n",
      "--- gene=Avil:+, id=270 removed by soft count = 347\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Bgn:+\n",
      "--- gene=Bgn:+, id=58 removed by soft count = 77\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: C1qb:-\n",
      "--- gene=C1qb:-, id=7 removed by soft count = 485\n",
      "--- gene=C1qb:-, id=159 removed by soft count = 116\n",
      "--- gene=C1qb:-, id=234 removed by soft count = 391\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: C3:-\n",
      "--- gene=C3:-, id=15 removed by soft count = 58\n",
      "--- gene=C3:-, id=21 removed by soft count = 319\n",
      "--- gene=C3:-, id=22 removed by soft count = 813\n",
      "--- gene=C3:-, id=23 removed by soft count = 1334\n",
      "--- gene=C3:-, id=24 removed by soft count = 524\n",
      "--- gene=C3:-, id=235 removed by soft count = 78\n",
      "--- gene=C3:-, id=248 removed by soft count = 52\n",
      "--- number of probes: 300 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Calb2:-\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Cald1:+\n",
      "--- gene=Cald1:+, id=265 removed by soft count = 118\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Camp:-\n",
      "--- gene=Camp:-, id=54 removed by soft count = 71\n",
      "--- gene=Camp:-, id=68 removed by soft count = 146\n",
      "--- gene=Camp:-, id=269 removed by soft count = 871\n",
      "--- gene=Camp:-, id=280 removed by soft count = 52\n",
      "--- gene=Camp:-, id=286 removed by soft count = 269\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Car4:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Car4:+, id=34 removed by soft count = 60\n",
      "--- gene=Car4:+, id=281 removed by soft count = 52\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Car8:-\n",
      "--- gene=Car8:-, id=36 removed by soft count = 65\n",
      "--- gene=Car8:-, id=38 removed by soft count = 378\n",
      "--- gene=Car8:-, id=39 removed by soft count = 171\n",
      "--- gene=Car8:-, id=40 removed by soft count = 78\n",
      "--- gene=Car8:-, id=42 removed by soft count = 114\n",
      "--- gene=Car8:-, id=53 removed by soft count = 62\n",
      "--- gene=Car8:-, id=55 removed by soft count = 78\n",
      "--- gene=Car8:-, id=56 removed by soft count = 77\n",
      "--- gene=Car8:-, id=232 removed by soft count = 131\n",
      "--- gene=Car8:-, id=268 removed by hard count = 8\n",
      "--- gene=Car8:-, id=296 removed by soft count = 545\n",
      "--- gene=Car8:-, id=297 removed by soft count = 52\n",
      "--- number of probes: 298 , kept by blast: 286 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ccdc30:-\n",
      "--- gene=Ccdc30:-, id=28 removed by soft count = 376\n",
      "--- gene=Ccdc30:-, id=29 removed by soft count = 75\n",
      "--- gene=Ccdc30:-, id=220 removed by soft count = 53\n",
      "--- gene=Ccdc30:-, id=225 removed by soft count = 1719\n",
      "--- gene=Ccdc30:-, id=248 removed by soft count = 51\n",
      "--- gene=Ccdc30:-, id=265 removed by soft count = 202\n",
      "--- gene=Ccdc30:-, id=279 removed by soft count = 64\n",
      "--- number of probes: 300 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Cck:-\n",
      "--- gene=Cck:-, id=0 removed by soft count = 859\n",
      "--- gene=Cck:-, id=1 removed by soft count = 85\n",
      "--- gene=Cck:-, id=2 removed by soft count = 55\n",
      "--- gene=Cck:-, id=233 removed by soft count = 72\n",
      "--- number of probes: 295 , kept by blast: 291 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ccl1:-\n",
      "--- gene=Ccl1:-, id=29 removed by soft count = 251\n",
      "--- gene=Ccl1:-, id=31 removed by soft count = 148\n",
      "--- gene=Ccl1:-, id=145 removed by soft count = 82\n",
      "--- gene=Ccl1:-, id=146 removed by soft count = 58\n",
      "--- gene=Ccl1:-, id=152 removed by soft count = 277\n",
      "--- gene=Ccl1:-, id=153 removed by soft count = 89\n",
      "--- gene=Ccl1:-, id=159 removed by soft count = 88\n",
      "--- gene=Ccl1:-, id=162 removed by soft count = 169\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ccl2:+\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ccl20:+\n",
      "--- gene=Ccl20:+, id=79 removed by soft count = 215\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ccl6:-\n",
      "--- gene=Ccl6:-, id=29 removed by soft count = 616\n",
      "--- gene=Ccl6:-, id=111 removed by soft count = 76\n",
      "--- gene=Ccl6:-, id=125 removed by soft count = 52\n",
      "--- gene=Ccl6:-, id=126 removed by soft count = 58\n",
      "--- number of probes: 200 , kept by blast: 196 , if remove dups: 195\n",
      "-- number of probes kept for this region: 196\n",
      "-- checking probes in region: Ccl9:-\n",
      "--- gene=Ccl9:-, id=186 removed by soft count = 204\n",
      "--- gene=Ccl9:-, id=203 removed by soft count = 945\n",
      "--- gene=Ccl9:-, id=206 removed by soft count = 134\n",
      "--- gene=Ccl9:-, id=261 removed by soft count = 66\n",
      "--- gene=Ccl9:-, id=262 removed by soft count = 144\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Cd14:-\n",
      "--- gene=Cd14:-, id=101 removed by soft count = 235\n",
      "--- gene=Cd14:-, id=182 removed by soft count = 97\n",
      "--- gene=Cd14:-, id=183 removed by soft count = 66\n",
      "--- gene=Cd14:-, id=191 removed by soft count = 113\n",
      "--- gene=Cd14:-, id=192 removed by soft count = 151\n",
      "--- gene=Cd14:-, id=206 removed by soft count = 235\n",
      "--- gene=Cd14:-, id=237 removed by soft count = 121\n",
      "--- gene=Cd14:-, id=244 removed by soft count = 68\n",
      "--- gene=Cd14:-, id=251 removed by soft count = 844\n",
      "--- gene=Cd14:-, id=272 removed by soft count = 59\n",
      "--- gene=Cd14:-, id=299 removed by soft count = 78\n",
      "--- number of probes: 300 , kept by blast: 289 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Cebpb:+\n",
      "--- gene=Cebpb:+, id=52 removed by soft count = 109\n",
      "--- gene=Cebpb:+, id=64 removed by soft count = 53\n",
      "--- gene=Cebpb:+, id=87 removed by soft count = 55\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Cga:+\n",
      "--- gene=Cga:+, id=1 removed by soft count = 1261\n",
      "--- gene=Cga:+, id=2 removed by soft count = 332\n",
      "--- gene=Cga:+, id=3 removed by soft count = 1126\n",
      "--- gene=Cga:+, id=88 removed by soft count = 62\n",
      "--- gene=Cga:+, id=196 removed by soft count = 65\n",
      "--- gene=Cga:+, id=206 removed by soft count = 95\n",
      "--- gene=Cga:+, id=220 removed by soft count = 66\n",
      "--- gene=Cga:+, id=237 removed by soft count = 174\n",
      "--- gene=Cga:+, id=240 removed by soft count = 75\n",
      "--- gene=Cga:+, id=282 removed by soft count = 63\n",
      "--- number of probes: 300 , kept by blast: 290 , if remove dups: 281\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Cilp:+\n",
      "--- gene=Cilp:+, id=42 removed by soft count = 51\n",
      "--- gene=Cilp:+, id=53 removed by soft count = 297\n",
      "--- gene=Cilp:+, id=59 removed by soft count = 57\n",
      "--- gene=Cilp:+, id=71 removed by soft count = 78\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ckb:-\n",
      "--- gene=Ckb:-, id=243 removed by soft count = 150\n",
      "--- gene=Ckb:-, id=252 removed by soft count = 178\n",
      "--- gene=Ckb:-, id=253 removed by soft count = 170\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ckm:+\n",
      "--- gene=Ckm:+, id=132 removed by soft count = 142\n",
      "--- gene=Ckm:+, id=164 removed by soft count = 197\n",
      "--- gene=Ckm:+, id=165 removed by soft count = 109\n",
      "--- gene=Ckm:+, id=167 removed by soft count = 180\n",
      "--- gene=Ckm:+, id=236 removed by soft count = 686\n",
      "--- gene=Ckm:+, id=275 removed by soft count = 76\n",
      "--- gene=Ckm:+, id=280 removed by soft count = 117\n",
      "--- gene=Ckm:+, id=281 removed by soft count = 450\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Clps:-\n",
      "--- gene=Clps:-, id=22 removed by soft count = 57\n",
      "--- gene=Clps:-, id=59 removed by soft count = 101\n",
      "--- gene=Clps:-, id=65 removed by soft count = 80\n",
      "--- gene=Clps:-, id=75 removed by soft count = 236\n",
      "--- gene=Clps:-, id=77 removed by soft count = 64\n",
      "--- gene=Clps:-, id=78 removed by soft count = 303\n",
      "--- gene=Clps:-, id=79 removed by soft count = 54\n",
      "--- gene=Clps:-, id=186 removed by soft count = 83\n",
      "--- gene=Clps:-, id=187 removed by soft count = 52\n",
      "--- gene=Clps:-, id=277 removed by soft count = 697\n",
      "--- number of probes: 300 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Cnfn:-\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Cnmd:-\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Cnmd:-, id=60 removed by soft count = 96\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Col1a1:+\n",
      "--- gene=Col1a1:+, id=85 removed by hard count = 0\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Col4a1:-\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Col6a1:-\n",
      "--- gene=Col6a1:-, id=9 removed by soft count = 1411\n",
      "--- gene=Col6a1:-, id=30 removed by soft count = 86\n",
      "--- gene=Col6a1:-, id=32 removed by soft count = 84\n",
      "--- gene=Col6a1:-, id=144 removed by soft count = 299\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Cpn1:-\n",
      "--- gene=Cpn1:-, id=62 removed by soft count = 5354\n",
      "--- gene=Cpn1:-, id=145 removed by soft count = 87\n",
      "--- gene=Cpn1:-, id=214 removed by soft count = 99\n",
      "--- gene=Cpn1:-, id=215 removed by soft count = 214\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Crabp1:+\n",
      "--- gene=Crabp1:+, id=104 removed by soft count = 58\n",
      "--- gene=Crabp1:+, id=241 removed by soft count = 115\n",
      "--- gene=Crabp1:+, id=246 removed by soft count = 160\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Cryaa:+\n",
      "--- gene=Cryaa:+, id=33 removed by soft count = 110\n",
      "--- gene=Cryaa:+, id=38 removed by soft count = 1319\n",
      "--- gene=Cryaa:+, id=69 removed by soft count = 79\n",
      "--- gene=Cryaa:+, id=196 removed by soft count = 129\n",
      "--- gene=Cryaa:+, id=218 removed by soft count = 279\n",
      "--- gene=Cryaa:+, id=254 removed by soft count = 103\n",
      "--- number of probes: 298 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ctrb1:-\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Cxcl13:+\n",
      "--- gene=Cxcl13:+, id=3 removed by soft count = 190\n",
      "--- gene=Cxcl13:+, id=79 removed by soft count = 89\n",
      "--- gene=Cxcl13:+, id=98 removed by soft count = 51\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Cyp1a2:-\n",
      "--- gene=Cyp1a2:-, id=213 removed by soft count = 93\n",
      "--- gene=Cyp1a2:-, id=224 removed by soft count = 114\n",
      "--- gene=Cyp1a2:-, id=279 removed by soft count = 149\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Cyp2f2:+\n",
      "--- gene=Cyp2f2:+, id=19 removed by soft count = 784\n",
      "--- gene=Cyp2f2:+, id=87 removed by soft count = 61\n",
      "--- gene=Cyp2f2:+, id=142 removed by soft count = 65\n",
      "--- gene=Cyp2f2:+, id=151 removed by soft count = 56\n",
      "--- gene=Cyp2f2:+, id=254 removed by soft count = 56\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Dbn1:-\n",
      "--- gene=Dbn1:-, id=78 removed by soft count = 2268\n",
      "--- gene=Dbn1:-, id=186 removed by soft count = 83\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Dclk1:+\n",
      "--- gene=Dclk1:+, id=181 removed by soft count = 146\n",
      "--- gene=Dclk1:+, id=246 removed by soft count = 55\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Dlk1:+\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Dppa3:+\n",
      "--- gene=Dppa3:+, id=26 removed by soft count = 5409\n",
      "--- gene=Dppa3:+, id=76 removed by soft count = 1110\n",
      "--- gene=Dppa3:+, id=79 removed by soft count = 674\n",
      "--- gene=Dppa3:+, id=82 removed by soft count = 55\n",
      "--- gene=Dppa3:+, id=121 removed by soft count = 91\n",
      "--- gene=Dppa3:+, id=210 removed by soft count = 106\n",
      "--- gene=Dppa3:+, id=257 removed by soft count = 101\n",
      "--- number of probes: 279 , kept by blast: 272 , if remove dups: 270\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ecm1:-\n",
      "--- gene=Ecm1:-, id=171 removed by soft count = 66\n",
      "--- gene=Ecm1:-, id=232 removed by soft count = 96\n",
      "--- gene=Ecm1:-, id=290 removed by soft count = 228\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Efna3:-\n",
      "--- gene=Efna3:-, id=252 removed by soft count = 74\n",
      "--- gene=Efna3:-, id=253 removed by soft count = 72\n",
      "--- gene=Efna3:-, id=265 removed by soft count = 112\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Elovl4:-\n",
      "--- gene=Elovl4:-, id=186 removed by soft count = 53\n",
      "--- gene=Elovl4:-, id=224 removed by soft count = 131\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Fabp1:+\n",
      "--- gene=Fabp1:+, id=78 removed by soft count = 74\n",
      "--- gene=Fabp1:+, id=79 removed by soft count = 58\n",
      "--- gene=Fabp1:+, id=82 removed by soft count = 188\n",
      "--- gene=Fabp1:+, id=83 removed by soft count = 111\n",
      "--- gene=Fabp1:+, id=221 removed by soft count = 154\n",
      "--- gene=Fabp1:+, id=222 removed by soft count = 102\n",
      "--- gene=Fabp1:+, id=268 removed by soft count = 90\n",
      "--- gene=Fabp1:+, id=280 removed by soft count = 156\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Fabp6:-\n",
      "--- gene=Fabp6:-, id=86 removed by soft count = 320\n",
      "--- gene=Fabp6:-, id=130 removed by soft count = 54\n",
      "--- gene=Fabp6:-, id=194 removed by soft count = 254\n",
      "--- gene=Fabp6:-, id=212 removed by soft count = 119\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Fam183b:-\n",
      "--- gene=Fam183b:-, id=64 removed by soft count = 420\n",
      "--- gene=Fam183b:-, id=65 removed by soft count = 75\n",
      "--- gene=Fam183b:-, id=66 removed by soft count = 187\n",
      "--- gene=Fam183b:-, id=81 removed by soft count = 208\n",
      "--- gene=Fam183b:-, id=150 removed by soft count = 359\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Fam25c:-\n",
      "--- gene=Fam25c:-, id=12 removed by soft count = 75\n",
      "--- gene=Fam25c:-, id=33 removed by soft count = 79\n",
      "--- gene=Fam25c:-, id=277 removed by soft count = 303\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Fgg:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Fgg:+, id=54 removed by hard count = 4\n",
      "--- gene=Fgg:+, id=221 removed by soft count = 290\n",
      "--- gene=Fgg:+, id=222 removed by soft count = 210\n",
      "--- gene=Fgg:+, id=227 removed by soft count = 97\n",
      "--- gene=Fgg:+, id=229 removed by soft count = 89\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Foxa1:-\n",
      "--- gene=Foxa1:-, id=14 removed by soft count = 1233\n",
      "--- gene=Foxa1:-, id=15 removed by soft count = 1268\n",
      "--- gene=Foxa1:-, id=16 removed by soft count = 63\n",
      "--- gene=Foxa1:-, id=18 removed by soft count = 68\n",
      "--- gene=Foxa1:-, id=19 removed by soft count = 120\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Foxa2:-\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Foxa3:-\n",
      "--- gene=Foxa3:-, id=148 removed by soft count = 111\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Foxj1:-\n",
      "--- gene=Foxj1:-, id=63 removed by soft count = 260\n",
      "--- gene=Foxj1:-, id=127 removed by soft count = 216\n",
      "--- gene=Foxj1:-, id=239 removed by soft count = 577\n",
      "--- gene=Foxj1:-, id=265 removed by soft count = 165\n",
      "--- gene=Foxj1:-, id=296 removed by soft count = 187\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Fxyd3:-\n",
      "--- gene=Fxyd3:-, id=10 removed by soft count = 124\n",
      "--- gene=Fxyd3:-, id=21 removed by soft count = 55\n",
      "--- gene=Fxyd3:-, id=240 removed by soft count = 2159\n",
      "--- gene=Fxyd3:-, id=295 removed by soft count = 2265\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Fxyd6:+\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Gabrp:-\n",
      "--- gene=Gabrp:-, id=63 removed by soft count = 82\n",
      "--- gene=Gabrp:-, id=65 removed by soft count = 248\n",
      "--- gene=Gabrp:-, id=69 removed by soft count = 168\n",
      "--- gene=Gabrp:-, id=123 removed by soft count = 697\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Gal:-\n",
      "--- gene=Gal:-, id=62 removed by soft count = 77\n",
      "--- gene=Gal:-, id=218 removed by soft count = 158\n",
      "--- gene=Gal:-, id=220 removed by soft count = 88\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Gatm:-\n",
      "--- gene=Gatm:-, id=33 removed by soft count = 89\n",
      "--- gene=Gatm:-, id=34 removed by soft count = 135\n",
      "--- gene=Gatm:-, id=37 removed by soft count = 58\n",
      "--- gene=Gatm:-, id=73 removed by soft count = 56\n",
      "--- gene=Gatm:-, id=195 removed by soft count = 69\n",
      "--- gene=Gatm:-, id=207 removed by soft count = 143\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ghrl:-\n",
      "--- gene=Ghrl:-, id=193 removed by soft count = 83\n",
      "--- gene=Ghrl:-, id=277 removed by soft count = 115\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Gm13539:+\n",
      "--- gene=Gm13539:+, id=68 removed by soft count = 117\n",
      "--- gene=Gm13539:+, id=95 removed by soft count = 1139\n",
      "--- gene=Gm13539:+, id=108 removed by soft count = 84\n",
      "--- gene=Gm13539:+, id=114 removed by soft count = 178\n",
      "--- gene=Gm13539:+, id=120 removed by soft count = 58\n",
      "--- gene=Gm13539:+, id=122 removed by soft count = 167\n",
      "--- gene=Gm13539:+, id=125 removed by soft count = 488\n",
      "--- gene=Gm13539:+, id=162 removed by soft count = 57\n",
      "--- number of probes: 228 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 220\n",
      "-- checking probes in region: Gm2a:+\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Gm94:-\n",
      "--- gene=Gm94:-, id=23 removed by soft count = 134\n",
      "--- gene=Gm94:-, id=37 removed by soft count = 230\n",
      "--- gene=Gm94:-, id=42 removed by soft count = 80\n",
      "--- gene=Gm94:-, id=147 removed by soft count = 94\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Gp2:-\n",
      "--- gene=Gp2:-, id=0 removed by soft count = 51\n",
      "--- gene=Gp2:-, id=65 removed by soft count = 59\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Gpx3:+\n",
      "--- gene=Gpx3:+, id=37 removed by soft count = 61\n",
      "--- gene=Gpx3:+, id=63 removed by soft count = 120\n",
      "--- gene=Gpx3:+, id=89 removed by soft count = 104\n",
      "--- gene=Gpx3:+, id=278 removed by soft count = 72\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Gpx6:+\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Grhl1:+\n",
      "--- gene=Grhl1:+, id=158 removed by soft count = 71\n",
      "--- gene=Grhl1:+, id=198 removed by soft count = 227\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Gsdma:+\n",
      "--- gene=Gsdma:+, id=7 removed by soft count = 358\n",
      "--- gene=Gsdma:+, id=9 removed by soft count = 83\n",
      "--- gene=Gsdma:+, id=18 removed by soft count = 61\n",
      "--- gene=Gsdma:+, id=74 removed by soft count = 372\n",
      "--- gene=Gsdma:+, id=100 removed by soft count = 841\n",
      "--- gene=Gsdma:+, id=259 removed by soft count = 103\n",
      "--- gene=Gsdma:+, id=287 removed by soft count = 481\n",
      "--- gene=Gsdma:+, id=298 removed by soft count = 110\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Gsta2:-\n",
      "--- gene=Gsta2:-, id=7 removed by soft count = 1159\n",
      "--- gene=Gsta2:-, id=72 removed by soft count = 111\n",
      "--- gene=Gsta2:-, id=73 removed by soft count = 167\n",
      "--- gene=Gsta2:-, id=74 removed by soft count = 71\n",
      "--- gene=Gsta2:-, id=95 removed by soft count = 66\n",
      "--- gene=Gsta2:-, id=109 removed by soft count = 52\n",
      "--- gene=Gsta2:-, id=145 removed by soft count = 1824\n",
      "--- gene=Gsta2:-, id=147 removed by soft count = 253\n",
      "--- gene=Gsta2:-, id=173 removed by soft count = 341\n",
      "--- gene=Gsta2:-, id=223 removed by hard count = 3\n",
      "--- gene=Gsta2:-, id=234 removed by hard count = 3\n",
      "--- gene=Gsta2:-, id=270 removed by soft count = 81\n",
      "--- number of probes: 274 , kept by blast: 262 , if remove dups: 259\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 259\n",
      "-- checking probes in region: Gsta4:+\n",
      "--- gene=Gsta4:+, id=6 removed by soft count = 170\n",
      "--- gene=Gsta4:+, id=98 removed by soft count = 106\n",
      "--- gene=Gsta4:+, id=140 removed by soft count = 1607\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Gsta4:+, id=143 removed by soft count = 133\n",
      "--- gene=Gsta4:+, id=169 removed by soft count = 142\n",
      "--- gene=Gsta4:+, id=214 removed by soft count = 318\n",
      "--- gene=Gsta4:+, id=224 removed by soft count = 90\n",
      "--- gene=Gsta4:+, id=225 removed by soft count = 55\n",
      "--- gene=Gsta4:+, id=290 removed by soft count = 62\n",
      "--- number of probes: 300 , kept by blast: 291 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Gstm3:-\n",
      "--- gene=Gstm3:-, id=255 removed by soft count = 57\n",
      "--- gene=Gstm3:-, id=289 removed by soft count = 123\n",
      "--- gene=Gstm3:-, id=290 removed by soft count = 98\n",
      "--- gene=Gstm3:-, id=293 removed by soft count = 247\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Gsto1:+\n",
      "--- gene=Gsto1:+, id=19 removed by soft count = 87\n",
      "--- gene=Gsto1:+, id=20 removed by soft count = 101\n",
      "--- gene=Gsto1:+, id=21 removed by soft count = 124\n",
      "--- gene=Gsto1:+, id=22 removed by soft count = 227\n",
      "--- gene=Gsto1:+, id=23 removed by soft count = 232\n",
      "--- gene=Gsto1:+, id=24 removed by soft count = 100\n",
      "--- gene=Gsto1:+, id=25 removed by soft count = 594\n",
      "--- gene=Gsto1:+, id=71 removed by soft count = 61\n",
      "--- gene=Gsto1:+, id=189 removed by soft count = 199\n",
      "--- number of probes: 294 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Guca2a:+\n",
      "--- gene=Guca2a:+, id=26 removed by soft count = 166\n",
      "--- gene=Guca2a:+, id=295 removed by soft count = 73\n",
      "--- gene=Guca2a:+, id=297 removed by soft count = 104\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: H19:-\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Hamp2:-\n",
      "--- gene=Hamp2:-, id=27 removed by soft count = 266\n",
      "--- gene=Hamp2:-, id=88 removed by soft count = 223\n",
      "--- gene=Hamp2:-, id=138 removed by soft count = 64\n",
      "--- gene=Hamp2:-, id=233 removed by soft count = 96\n",
      "--- gene=Hamp2:-, id=234 removed by soft count = 63\n",
      "--- gene=Hamp2:-, id=237 removed by soft count = 59\n",
      "--- number of probes: 238 , kept by blast: 232 , if remove dups: 215\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 215\n",
      "-- checking probes in region: Hba-a2:+\n",
      "--- gene=Hba-a2:+, id=41 removed by soft count = 741\n",
      "--- gene=Hba-a2:+, id=42 removed by soft count = 118\n",
      "--- gene=Hba-a2:+, id=119 removed by soft count = 67\n",
      "--- gene=Hba-a2:+, id=144 removed by soft count = 128\n",
      "--- gene=Hba-a2:+, id=212 removed by soft count = 62\n",
      "--- gene=Hba-a2:+, id=213 removed by soft count = 79\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 277\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Hbb-y:-\n",
      "--- gene=Hbb-y:-, id=12 removed by hard count = 0\n",
      "--- gene=Hbb-y:-, id=45 removed by soft count = 55\n",
      "--- gene=Hbb-y:-, id=127 removed by soft count = 56\n",
      "--- gene=Hbb-y:-, id=129 removed by soft count = 355\n",
      "--- gene=Hbb-y:-, id=158 removed by soft count = 1589\n",
      "--- gene=Hbb-y:-, id=163 removed by soft count = 68\n",
      "--- gene=Hbb-y:-, id=212 removed by soft count = 830\n",
      "--- gene=Hbb-y:-, id=214 removed by soft count = 116\n",
      "--- gene=Hbb-y:-, id=264 removed by soft count = 109\n",
      "--- number of probes: 300 , kept by blast: 291 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Hck:+\n",
      "--- gene=Hck:+, id=148 removed by soft count = 144\n",
      "--- number of probes: 266 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 265\n",
      "-- checking probes in region: Hnf4a:+\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Hnf4g:+\n",
      "--- gene=Hnf4g:+, id=3 removed by soft count = 61\n",
      "--- gene=Hnf4g:+, id=9 removed by soft count = 394\n",
      "--- gene=Hnf4g:+, id=155 removed by soft count = 429\n",
      "--- gene=Hnf4g:+, id=157 removed by soft count = 55\n",
      "--- gene=Hnf4g:+, id=158 removed by soft count = 132\n",
      "--- gene=Hnf4g:+, id=252 removed by soft count = 71\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Hp:-\n",
      "--- gene=Hp:-, id=13 removed by soft count = 76\n",
      "--- gene=Hp:-, id=16 removed by soft count = 55\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Iapp:+\n",
      "--- gene=Iapp:+, id=292 removed by soft count = 2932\n",
      "--- gene=Iapp:+, id=293 removed by soft count = 655\n",
      "--- gene=Iapp:+, id=296 removed by soft count = 174\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Idi2:+\n",
      "--- gene=Idi2:+, id=48 removed by soft count = 183\n",
      "--- gene=Idi2:+, id=49 removed by soft count = 113\n",
      "--- gene=Idi2:+, id=59 removed by soft count = 78\n",
      "--- gene=Idi2:+, id=63 removed by soft count = 59\n",
      "--- gene=Idi2:+, id=103 removed by soft count = 76\n",
      "--- gene=Idi2:+, id=116 removed by hard count = 3\n",
      "--- gene=Idi2:+, id=117 removed by hard count = 3\n",
      "--- gene=Idi2:+, id=120 removed by hard count = 3\n",
      "--- gene=Idi2:+, id=121 removed by hard count = 3\n",
      "--- gene=Idi2:+, id=122 removed by hard count = 3\n",
      "--- gene=Idi2:+, id=123 removed by hard count = 3\n",
      "--- gene=Idi2:+, id=214 removed by soft count = 327\n",
      "--- gene=Idi2:+, id=215 removed by soft count = 242\n",
      "--- gene=Idi2:+, id=216 removed by soft count = 136\n",
      "--- gene=Idi2:+, id=218 removed by soft count = 127\n",
      "--- gene=Idi2:+, id=219 removed by soft count = 228\n",
      "--- gene=Idi2:+, id=220 removed by soft count = 197\n",
      "--- gene=Idi2:+, id=242 removed by soft count = 130\n",
      "--- number of probes: 243 , kept by blast: 225 , if remove dups: 124\n",
      "-- number of probes kept for this region: 225\n",
      "-- checking probes in region: Ifi27l2b:-\n",
      "--- gene=Ifi27l2b:-, id=7 removed by soft count = 106\n",
      "--- gene=Ifi27l2b:-, id=8 removed by soft count = 705\n",
      "--- gene=Ifi27l2b:-, id=9 removed by soft count = 13460\n",
      "--- gene=Ifi27l2b:-, id=10 removed by soft count = 427\n",
      "--- gene=Ifi27l2b:-, id=46 removed by soft count = 72\n",
      "--- gene=Ifi27l2b:-, id=52 removed by soft count = 215\n",
      "--- gene=Ifi27l2b:-, id=120 removed by soft count = 53\n",
      "--- gene=Ifi27l2b:-, id=128 removed by soft count = 85\n",
      "--- gene=Ifi27l2b:-, id=161 removed by soft count = 736\n",
      "--- gene=Ifi27l2b:-, id=162 removed by soft count = 560\n",
      "--- gene=Ifi27l2b:-, id=163 removed by soft count = 648\n",
      "--- gene=Ifi27l2b:-, id=182 removed by hard count = 3\n",
      "--- number of probes: 193 , kept by blast: 181 , if remove dups: 164\n",
      "-- number of probes kept for this region: 181\n",
      "-- checking probes in region: Igf1:+\n",
      "--- gene=Igf1:+, id=0 removed by soft count = 204\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Igf2:-\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Igfbp5:-\n",
      "--- gene=Igfbp5:-, id=47 removed by soft count = 73\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Igfbp6:+\n",
      "--- gene=Igfbp6:+, id=25 removed by soft count = 76\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Igfbp6:+, id=65 removed by soft count = 240\n",
      "--- gene=Igfbp6:+, id=104 removed by soft count = 123\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ins2:-\n",
      "--- gene=Ins2:-, id=63 removed by soft count = 103\n",
      "--- gene=Ins2:-, id=64 removed by soft count = 84\n",
      "--- gene=Ins2:-, id=65 removed by soft count = 138\n",
      "--- gene=Ins2:-, id=146 removed by soft count = 12030\n",
      "--- gene=Ins2:-, id=170 removed by soft count = 198\n",
      "--- gene=Ins2:-, id=172 removed by soft count = 104\n",
      "--- gene=Ins2:-, id=275 removed by soft count = 62\n",
      "--- gene=Ins2:-, id=276 removed by soft count = 73\n",
      "--- gene=Ins2:-, id=278 removed by soft count = 79\n",
      "--- gene=Ins2:-, id=279 removed by soft count = 113\n",
      "--- gene=Ins2:-, id=281 removed by soft count = 175\n",
      "--- gene=Ins2:-, id=284 removed by soft count = 92\n",
      "--- gene=Ins2:-, id=285 removed by soft count = 60\n",
      "--- gene=Ins2:-, id=291 removed by soft count = 73\n",
      "--- number of probes: 300 , kept by blast: 286 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Isg15:-\n",
      "--- gene=Isg15:-, id=9 removed by soft count = 1479\n",
      "--- gene=Isg15:-, id=47 removed by soft count = 313\n",
      "--- gene=Isg15:-, id=58 removed by soft count = 133\n",
      "--- gene=Isg15:-, id=59 removed by soft count = 92\n",
      "--- gene=Isg15:-, id=79 removed by soft count = 145\n",
      "--- gene=Isg15:-, id=81 removed by soft count = 154\n",
      "--- gene=Isg15:-, id=257 removed by soft count = 69\n",
      "--- gene=Isg15:-, id=258 removed by soft count = 139\n",
      "--- gene=Isg15:-, id=260 removed by soft count = 81\n",
      "--- number of probes: 300 , kept by blast: 291 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Itga6:+\n",
      "--- gene=Itga6:+, id=87 removed by soft count = 97\n",
      "--- gene=Itga6:+, id=113 removed by soft count = 70\n",
      "--- number of probes: 295 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Jpt1:-\n",
      "--- gene=Jpt1:-, id=55 removed by soft count = 95\n",
      "--- gene=Jpt1:-, id=90 removed by soft count = 345\n",
      "--- gene=Jpt1:-, id=106 removed by soft count = 1789\n",
      "--- gene=Jpt1:-, id=241 removed by soft count = 76\n",
      "--- gene=Jpt1:-, id=288 removed by soft count = 317\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Klk14:+\n",
      "--- gene=Klk14:+, id=13 removed by soft count = 65\n",
      "--- gene=Klk14:+, id=109 removed by soft count = 59\n",
      "--- gene=Klk14:+, id=225 removed by soft count = 176\n",
      "--- gene=Klk14:+, id=253 removed by soft count = 100\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Klk7:+\n",
      "--- gene=Klk7:+, id=59 removed by soft count = 62\n",
      "--- gene=Klk7:+, id=60 removed by soft count = 3788\n",
      "--- gene=Klk7:+, id=251 removed by soft count = 309\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Krt10:-\n",
      "--- gene=Krt10:-, id=137 removed by hard count = 0\n",
      "--- gene=Krt10:-, id=177 removed by soft count = 51\n",
      "--- gene=Krt10:-, id=201 removed by soft count = 97\n",
      "--- gene=Krt10:-, id=204 removed by soft count = 91\n",
      "--- gene=Krt10:-, id=222 removed by soft count = 190\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Krt13:-\n",
      "--- gene=Krt13:-, id=247 removed by soft count = 116\n",
      "--- gene=Krt13:-, id=248 removed by soft count = 257\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Krt14:-\n",
      "--- gene=Krt14:-, id=68 removed by soft count = 147\n",
      "--- gene=Krt14:-, id=105 removed by soft count = 191\n",
      "--- gene=Krt14:-, id=119 removed by soft count = 447\n",
      "--- gene=Krt14:-, id=151 removed by soft count = 1497\n",
      "--- number of probes: 195 , kept by blast: 191 , if remove dups: 191\n",
      "-- number of probes kept for this region: 191\n",
      "-- checking probes in region: Krt16:-\n",
      "--- gene=Krt16:-, id=34 removed by soft count = 56\n",
      "--- gene=Krt16:-, id=36 removed by soft count = 90\n",
      "--- gene=Krt16:-, id=40 removed by soft count = 180\n",
      "--- gene=Krt16:-, id=41 removed by soft count = 107\n",
      "--- gene=Krt16:-, id=43 removed by soft count = 128\n",
      "--- gene=Krt16:-, id=49 removed by soft count = 82\n",
      "--- gene=Krt16:-, id=56 removed by soft count = 79\n",
      "--- gene=Krt16:-, id=58 removed by soft count = 158\n",
      "--- gene=Krt16:-, id=59 removed by soft count = 264\n",
      "--- gene=Krt16:-, id=60 removed by soft count = 230\n",
      "--- gene=Krt16:-, id=61 removed by soft count = 177\n",
      "--- gene=Krt16:-, id=65 removed by soft count = 120\n",
      "--- gene=Krt16:-, id=66 removed by soft count = 195\n",
      "--- gene=Krt16:-, id=68 removed by soft count = 104\n",
      "--- gene=Krt16:-, id=70 removed by soft count = 297\n",
      "--- gene=Krt16:-, id=76 removed by soft count = 339\n",
      "--- gene=Krt16:-, id=83 removed by soft count = 53\n",
      "--- gene=Krt16:-, id=84 removed by soft count = 112\n",
      "--- gene=Krt16:-, id=108 removed by soft count = 52\n",
      "--- number of probes: 109 , kept by blast: 90 , if remove dups: 90\n",
      "-- number of probes kept for this region: 90\n",
      "-- checking probes in region: Krt17:-\n",
      "--- gene=Krt17:-, id=7 removed by soft count = 225\n",
      "--- gene=Krt17:-, id=235 removed by soft count = 320\n",
      "--- gene=Krt17:-, id=237 removed by soft count = 65\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Krt19:-\n",
      "--- gene=Krt19:-, id=22 removed by soft count = 67\n",
      "--- number of probes: 293 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Krt5:-\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Krt71:-\n",
      "--- gene=Krt71:-, id=9 removed by soft count = 55\n",
      "--- gene=Krt71:-, id=122 removed by soft count = 421\n",
      "--- gene=Krt71:-, id=239 removed by soft count = 216\n",
      "--- gene=Krt71:-, id=241 removed by soft count = 63\n",
      "--- gene=Krt71:-, id=292 removed by soft count = 59\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Krt76:-\n",
      "--- gene=Krt76:-, id=16 removed by soft count = 85\n",
      "--- gene=Krt76:-, id=17 removed by soft count = 56\n",
      "--- gene=Krt76:-, id=20 removed by soft count = 357\n",
      "--- gene=Krt76:-, id=110 removed by soft count = 308\n",
      "--- gene=Krt76:-, id=116 removed by soft count = 293\n",
      "--- gene=Krt76:-, id=160 removed by soft count = 130\n",
      "--- gene=Krt76:-, id=163 removed by soft count = 91\n",
      "--- gene=Krt76:-, id=164 removed by soft count = 294\n",
      "--- gene=Krt76:-, id=205 removed by soft count = 3929\n",
      "--- number of probes: 300 , kept by blast: 291 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Krt77:-\n",
      "--- gene=Krt77:-, id=38 removed by soft count = 87\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Krt9:-\n",
      "--- gene=Krt9:-, id=74 removed by soft count = 684\n",
      "--- gene=Krt9:-, id=75 removed by soft count = 66\n",
      "--- gene=Krt9:-, id=167 removed by hard count = 3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Krt9:-, id=221 removed by soft count = 204\n",
      "--- gene=Krt9:-, id=257 removed by soft count = 55\n",
      "--- gene=Krt9:-, id=274 removed by soft count = 108\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Krtdap:+\n",
      "--- gene=Krtdap:+, id=198 removed by soft count = 64\n",
      "--- gene=Krtdap:+, id=222 removed by soft count = 63\n",
      "--- gene=Krtdap:+, id=260 removed by soft count = 65\n",
      "--- gene=Krtdap:+, id=261 removed by soft count = 288\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Lcn2:-\n",
      "--- gene=Lcn2:-, id=20 removed by soft count = 132\n",
      "--- gene=Lcn2:-, id=37 removed by soft count = 108\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Lcn3:+\n",
      "--- gene=Lcn3:+, id=36 removed by soft count = 144\n",
      "--- gene=Lcn3:+, id=37 removed by soft count = 661\n",
      "--- gene=Lcn3:+, id=38 removed by soft count = 1185\n",
      "--- gene=Lcn3:+, id=40 removed by soft count = 175\n",
      "--- gene=Lcn3:+, id=41 removed by soft count = 124\n",
      "--- gene=Lcn3:+, id=42 removed by soft count = 14114\n",
      "--- gene=Lcn3:+, id=141 removed by soft count = 61\n",
      "--- gene=Lcn3:+, id=144 removed by soft count = 620\n",
      "--- gene=Lcn3:+, id=145 removed by soft count = 124\n",
      "--- number of probes: 300 , kept by blast: 291 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Lgals2:-\n",
      "--- gene=Lgals2:-, id=84 removed by soft count = 759\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Lgals3:+\n",
      "--- gene=Lgals3:+, id=230 removed by soft count = 327\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Lifr:+\n",
      "--- gene=Lifr:+, id=208 removed by soft count = 204\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ltf:+\n",
      "--- gene=Ltf:+, id=289 removed by soft count = 390\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ly6c1:-\n",
      "--- gene=Ly6c1:-, id=0 removed by soft count = 70\n",
      "--- gene=Ly6c1:-, id=21 removed by soft count = 232\n",
      "--- gene=Ly6c1:-, id=23 removed by soft count = 908\n",
      "--- gene=Ly6c1:-, id=31 removed by soft count = 898\n",
      "--- gene=Ly6c1:-, id=32 removed by soft count = 851\n",
      "--- gene=Ly6c1:-, id=34 removed by soft count = 784\n",
      "--- gene=Ly6c1:-, id=35 removed by soft count = 827\n",
      "--- gene=Ly6c1:-, id=36 removed by soft count = 830\n",
      "--- gene=Ly6c1:-, id=38 removed by soft count = 816\n",
      "--- gene=Ly6c1:-, id=40 removed by soft count = 776\n",
      "--- gene=Ly6c1:-, id=41 removed by soft count = 706\n",
      "--- gene=Ly6c1:-, id=49 removed by soft count = 51\n",
      "--- gene=Ly6c1:-, id=52 removed by soft count = 128\n",
      "--- gene=Ly6c1:-, id=54 removed by soft count = 86\n",
      "--- gene=Ly6c1:-, id=55 removed by soft count = 51\n",
      "--- gene=Ly6c1:-, id=58 removed by soft count = 71\n",
      "--- gene=Ly6c1:-, id=59 removed by soft count = 58\n",
      "--- gene=Ly6c1:-, id=120 removed by soft count = 263\n",
      "--- gene=Ly6c1:-, id=123 removed by soft count = 191\n",
      "--- gene=Ly6c1:-, id=125 removed by soft count = 176\n",
      "--- number of probes: 126 , kept by blast: 106 , if remove dups: 104\n",
      "-- number of probes kept for this region: 106\n",
      "-- checking probes in region: Ly6c2:-\n",
      "--- gene=Ly6c2:-, id=0 removed by soft count = 5083\n",
      "--- gene=Ly6c2:-, id=2 removed by soft count = 690\n",
      "--- gene=Ly6c2:-, id=3 removed by soft count = 820\n",
      "--- gene=Ly6c2:-, id=6 removed by soft count = 696\n",
      "--- gene=Ly6c2:-, id=7 removed by soft count = 757\n",
      "--- gene=Ly6c2:-, id=8 removed by soft count = 759\n",
      "--- gene=Ly6c2:-, id=9 removed by soft count = 51\n",
      "--- gene=Ly6c2:-, id=12 removed by soft count = 764\n",
      "--- gene=Ly6c2:-, id=15 removed by soft count = 795\n",
      "--- gene=Ly6c2:-, id=16 removed by soft count = 892\n",
      "--- gene=Ly6c2:-, id=18 removed by soft count = 794\n",
      "--- gene=Ly6c2:-, id=20 removed by soft count = 684\n",
      "--- gene=Ly6c2:-, id=22 removed by soft count = 80\n",
      "--- gene=Ly6c2:-, id=23 removed by soft count = 156\n",
      "--- gene=Ly6c2:-, id=26 removed by soft count = 451\n",
      "--- gene=Ly6c2:-, id=27 removed by soft count = 914\n",
      "--- gene=Ly6c2:-, id=28 removed by soft count = 53\n",
      "--- gene=Ly6c2:-, id=30 removed by soft count = 595\n",
      "--- gene=Ly6c2:-, id=48 removed by soft count = 63\n",
      "--- gene=Ly6c2:-, id=62 removed by soft count = 260\n",
      "--- gene=Ly6c2:-, id=76 removed by soft count = 51\n",
      "--- gene=Ly6c2:-, id=141 removed by soft count = 123\n",
      "--- gene=Ly6c2:-, id=142 removed by soft count = 107\n",
      "--- gene=Ly6c2:-, id=143 removed by soft count = 140\n",
      "--- gene=Ly6c2:-, id=144 removed by soft count = 1105\n",
      "--- gene=Ly6c2:-, id=146 removed by soft count = 143\n",
      "--- number of probes: 147 , kept by blast: 121 , if remove dups: 118\n",
      "-- number of probes kept for this region: 121\n",
      "-- checking probes in region: Ly6d:-\n",
      "--- gene=Ly6d:-, id=219 removed by soft count = 2450\n",
      "--- gene=Ly6d:-, id=220 removed by soft count = 537\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ly6g6f:-\n",
      "--- gene=Ly6g6f:-, id=219 removed by hard count = 0\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Lypd2:-\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Lypd3:+\n",
      "--- gene=Lypd3:+, id=80 removed by soft count = 147\n",
      "--- gene=Lypd3:+, id=85 removed by hard count = 0\n",
      "--- gene=Lypd3:+, id=191 removed by soft count = 191\n",
      "--- gene=Lypd3:+, id=210 removed by soft count = 59\n",
      "--- gene=Lypd3:+, id=211 removed by soft count = 153\n",
      "--- gene=Lypd3:+, id=229 removed by hard count = 0\n",
      "--- gene=Lypd3:+, id=238 removed by soft count = 138\n",
      "--- gene=Lypd3:+, id=264 removed by soft count = 68\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Mal:-\n",
      "--- gene=Mal:-, id=73 removed by soft count = 79\n",
      "--- gene=Mal:-, id=214 removed by soft count = 82\n",
      "--- gene=Mal:-, id=216 removed by soft count = 133\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Meg3:+\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Meig1:-\n",
      "--- gene=Meig1:-, id=87 removed by soft count = 76\n",
      "--- gene=Meig1:-, id=172 removed by soft count = 73\n",
      "--- gene=Meig1:-, id=254 removed by soft count = 93\n",
      "--- number of probes: 284 , kept by blast: 281 , if remove dups: 281\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Mgp:-\n",
      "--- gene=Mgp:-, id=41 removed by soft count = 93\n",
      "--- gene=Mgp:-, id=69 removed by soft count = 257\n",
      "--- gene=Mgp:-, id=72 removed by soft count = 65\n",
      "--- gene=Mgp:-, id=75 removed by soft count = 75\n",
      "--- gene=Mgp:-, id=149 removed by soft count = 88\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Mlph:+\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Mlph:+, id=148 removed by soft count = 56\n",
      "--- gene=Mlph:+, id=256 removed by soft count = 136\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Mpo:+\n",
      "--- gene=Mpo:+, id=291 removed by soft count = 402\n",
      "--- gene=Mpo:+, id=292 removed by soft count = 236\n",
      "--- gene=Mpo:+, id=293 removed by soft count = 467\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Mt3:+\n",
      "--- gene=Mt3:+, id=81 removed by hard count = 4\n",
      "--- gene=Mt3:+, id=102 removed by soft count = 323\n",
      "--- gene=Mt3:+, id=152 removed by soft count = 63\n",
      "--- gene=Mt3:+, id=169 removed by soft count = 765\n",
      "--- number of probes: 187 , kept by blast: 183 , if remove dups: 183\n",
      "-- number of probes kept for this region: 183\n",
      "-- checking probes in region: Mt4:+\n",
      "--- gene=Mt4:+, id=163 removed by soft count = 57\n",
      "--- gene=Mt4:+, id=165 removed by soft count = 96\n",
      "--- gene=Mt4:+, id=208 removed by hard count = 3\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Muc13:+\n",
      "--- gene=Muc13:+, id=4 removed by soft count = 119\n",
      "--- gene=Muc13:+, id=8 removed by soft count = 54\n",
      "--- gene=Muc13:+, id=39 removed by soft count = 179\n",
      "--- gene=Muc13:+, id=40 removed by soft count = 163\n",
      "--- gene=Muc13:+, id=178 removed by hard count = 6\n",
      "--- gene=Muc13:+, id=193 removed by soft count = 62\n",
      "--- gene=Muc13:+, id=220 removed by soft count = 96\n",
      "--- gene=Muc13:+, id=229 removed by soft count = 426\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Mymx:-\n",
      "--- gene=Mymx:-, id=37 removed by soft count = 107\n",
      "--- gene=Mymx:-, id=71 removed by soft count = 71\n",
      "--- gene=Mymx:-, id=72 removed by soft count = 75\n",
      "--- gene=Mymx:-, id=103 removed by soft count = 69\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Myog:+\n",
      "--- gene=Myog:+, id=159 removed by soft count = 1771\n",
      "--- gene=Myog:+, id=161 removed by soft count = 71\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Nefh:-\n",
      "--- gene=Nefh:-, id=30 removed by hard count = 4\n",
      "--- gene=Nefh:-, id=79 removed by hard count = 0\n",
      "--- gene=Nefh:-, id=234 removed by soft count = 988\n",
      "--- gene=Nefh:-, id=290 removed by soft count = 102\n",
      "--- number of probes: 297 , kept by blast: 293 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ngp:+\n",
      "--- gene=Ngp:+, id=137 removed by soft count = 99\n",
      "--- gene=Ngp:+, id=139 removed by soft count = 151\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Nppa:+\n",
      "--- gene=Nppa:+, id=40 removed by soft count = 313\n",
      "--- gene=Nppa:+, id=41 removed by soft count = 51\n",
      "--- gene=Nppa:+, id=43 removed by soft count = 99\n",
      "--- gene=Nppa:+, id=44 removed by soft count = 67\n",
      "--- gene=Nppa:+, id=54 removed by soft count = 118\n",
      "--- gene=Nppa:+, id=68 removed by soft count = 147\n",
      "--- gene=Nppa:+, id=165 removed by soft count = 161\n",
      "--- gene=Nppa:+, id=166 removed by soft count = 95\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Npy:+\n",
      "--- gene=Npy:+, id=273 removed by soft count = 51\n",
      "--- gene=Npy:+, id=286 removed by soft count = 2739\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Nrgn:-\n",
      "--- gene=Nrgn:-, id=61 removed by soft count = 59\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Nts:-\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Oas1d:+\n",
      "--- gene=Oas1d:+, id=41 removed by soft count = 495\n",
      "--- gene=Oas1d:+, id=191 removed by soft count = 234\n",
      "--- gene=Oas1d:+, id=198 removed by soft count = 4176\n",
      "--- gene=Oas1d:+, id=212 removed by soft count = 654\n",
      "--- number of probes: 261 , kept by blast: 257 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 252\n",
      "-- checking probes in region: Oas1h:+\n",
      "--- gene=Oas1h:+, id=71 removed by soft count = 76\n",
      "--- gene=Oas1h:+, id=218 removed by soft count = 78\n",
      "--- gene=Oas1h:+, id=221 removed by soft count = 86\n",
      "--- gene=Oas1h:+, id=235 removed by soft count = 71\n",
      "--- gene=Oas1h:+, id=236 removed by soft count = 170\n",
      "--- gene=Oas1h:+, id=238 removed by soft count = 276\n",
      "--- gene=Oas1h:+, id=239 removed by soft count = 94\n",
      "--- gene=Oas1h:+, id=240 removed by soft count = 310\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Odf3b:-\n",
      "--- gene=Odf3b:-, id=4 removed by soft count = 51\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Olfm1:+\n",
      "--- gene=Olfm1:+, id=55 removed by soft count = 108\n",
      "--- gene=Olfm1:+, id=56 removed by soft count = 128\n",
      "--- gene=Olfm1:+, id=57 removed by soft count = 53\n",
      "--- gene=Olfm1:+, id=59 removed by soft count = 1876\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Pcp4:+\n",
      "--- gene=Pcp4:+, id=14 removed by soft count = 307\n",
      "--- gene=Pcp4:+, id=56 removed by soft count = 134\n",
      "--- gene=Pcp4:+, id=256 removed by soft count = 73\n",
      "--- gene=Pcp4:+, id=259 removed by soft count = 91\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Pdzk1ip1:+\n",
      "--- gene=Pdzk1ip1:+, id=25 removed by soft count = 175\n",
      "--- gene=Pdzk1ip1:+, id=26 removed by soft count = 52\n",
      "--- gene=Pdzk1ip1:+, id=251 removed by soft count = 51\n",
      "--- gene=Pdzk1ip1:+, id=253 removed by soft count = 82\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Penk:-\n",
      "--- gene=Penk:-, id=44 removed by soft count = 252\n",
      "--- gene=Penk:-, id=46 removed by soft count = 106\n",
      "--- gene=Penk:-, id=266 removed by soft count = 95\n",
      "--- gene=Penk:-, id=268 removed by soft count = 319\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Pgc:+\n",
      "--- gene=Pgc:+, id=21 removed by soft count = 51\n",
      "--- gene=Pgc:+, id=26 removed by soft count = 123\n",
      "--- gene=Pgc:+, id=61 removed by soft count = 91\n",
      "--- gene=Pgc:+, id=221 removed by soft count = 481\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Pigr:+\n",
      "--- gene=Pigr:+, id=12 removed by soft count = 118\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Pigr:+, id=159 removed by soft count = 57\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Pik3r5:+\n",
      "--- gene=Pik3r5:+, id=20 removed by soft count = 534\n",
      "--- gene=Pik3r5:+, id=47 removed by soft count = 512\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Pla2g1b:+\n",
      "--- gene=Pla2g1b:+, id=88 removed by soft count = 99\n",
      "--- gene=Pla2g1b:+, id=105 removed by soft count = 379\n",
      "--- gene=Pla2g1b:+, id=124 removed by soft count = 51\n",
      "--- gene=Pla2g1b:+, id=130 removed by soft count = 139\n",
      "--- gene=Pla2g1b:+, id=132 removed by soft count = 556\n",
      "--- gene=Pla2g1b:+, id=201 removed by soft count = 131\n",
      "--- gene=Pla2g1b:+, id=219 removed by soft count = 106\n",
      "--- gene=Pla2g1b:+, id=276 removed by soft count = 164\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Plac8:-\n",
      "--- gene=Plac8:-, id=240 removed by soft count = 138\n",
      "--- gene=Plac8:-, id=241 removed by soft count = 184\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Plk2:+\n",
      "--- gene=Plk2:+, id=7 removed by soft count = 124\n",
      "--- gene=Plk2:+, id=61 removed by soft count = 72\n",
      "--- gene=Plk2:+, id=70 removed by soft count = 191\n",
      "--- gene=Plk2:+, id=71 removed by soft count = 84\n",
      "--- gene=Plk2:+, id=72 removed by soft count = 71\n",
      "--- gene=Plk2:+, id=73 removed by soft count = 162\n",
      "--- gene=Plk2:+, id=74 removed by soft count = 104\n",
      "--- gene=Plk2:+, id=75 removed by soft count = 330\n",
      "--- gene=Plk2:+, id=78 removed by soft count = 693\n",
      "--- gene=Plk2:+, id=85 removed by soft count = 682\n",
      "--- gene=Plk2:+, id=88 removed by soft count = 51\n",
      "--- gene=Plk2:+, id=96 removed by soft count = 68\n",
      "--- gene=Plk2:+, id=97 removed by soft count = 72\n",
      "--- gene=Plk2:+, id=228 removed by soft count = 59\n",
      "--- number of probes: 300 , kept by blast: 286 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Plvap:-\n",
      "--- gene=Plvap:-, id=44 removed by soft count = 102\n",
      "--- gene=Plvap:-, id=201 removed by soft count = 58\n",
      "--- gene=Plvap:-, id=230 removed by soft count = 147\n",
      "--- gene=Plvap:-, id=282 removed by soft count = 70\n",
      "--- gene=Plvap:-, id=285 removed by soft count = 207\n",
      "--- number of probes: 293 , kept by blast: 288 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Pomc:+\n",
      "--- gene=Pomc:+, id=63 removed by soft count = 129\n",
      "--- gene=Pomc:+, id=265 removed by hard count = 3\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Pou2f3:-\n",
      "--- gene=Pou2f3:-, id=31 removed by soft count = 1453\n",
      "--- gene=Pou2f3:-, id=62 removed by soft count = 100\n",
      "--- gene=Pou2f3:-, id=95 removed by soft count = 175\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Psca:+\n",
      "--- gene=Psca:+, id=98 removed by soft count = 85\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ptgds:-\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Pyy:-\n",
      "--- gene=Pyy:-, id=43 removed by soft count = 81\n",
      "--- gene=Pyy:-, id=71 removed by soft count = 71\n",
      "--- gene=Pyy:-, id=76 removed by soft count = 100\n",
      "--- gene=Pyy:-, id=95 removed by soft count = 62\n",
      "--- gene=Pyy:-, id=126 removed by soft count = 325\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Rap1gap2:-\n",
      "--- gene=Rap1gap2:-, id=62 removed by soft count = 285\n",
      "--- gene=Rap1gap2:-, id=165 removed by soft count = 83\n",
      "--- gene=Rap1gap2:-, id=178 removed by soft count = 69\n",
      "--- gene=Rap1gap2:-, id=219 removed by soft count = 80\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Rarres1:-\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Reg3b:+\n",
      "--- gene=Reg3b:+, id=61 removed by soft count = 51\n",
      "--- gene=Reg3b:+, id=62 removed by soft count = 88\n",
      "--- gene=Reg3b:+, id=63 removed by soft count = 62\n",
      "--- gene=Reg3b:+, id=237 removed by soft count = 77\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Reg4:+\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Retnla:+\n",
      "--- gene=Retnla:+, id=1 removed by soft count = 71\n",
      "--- gene=Retnla:+, id=87 removed by soft count = 3034\n",
      "--- gene=Retnla:+, id=169 removed by soft count = 67\n",
      "--- gene=Retnla:+, id=171 removed by soft count = 653\n",
      "--- gene=Retnla:+, id=172 removed by soft count = 354\n",
      "--- gene=Retnla:+, id=173 removed by soft count = 321\n",
      "--- gene=Retnla:+, id=174 removed by soft count = 70\n",
      "--- gene=Retnla:+, id=175 removed by soft count = 145\n",
      "--- gene=Retnla:+, id=176 removed by soft count = 673\n",
      "--- gene=Retnla:+, id=177 removed by soft count = 748\n",
      "--- gene=Retnla:+, id=218 removed by soft count = 137\n",
      "--- number of probes: 246 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 235\n",
      "-- checking probes in region: Riiad1:-\n",
      "--- gene=Riiad1:-, id=176 removed by soft count = 55\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Rnase1:-\n",
      "--- gene=Rnase1:-, id=30 removed by soft count = 121\n",
      "--- gene=Rnase1:-, id=47 removed by soft count = 93\n",
      "--- gene=Rnase1:-, id=48 removed by soft count = 55\n",
      "--- gene=Rnase1:-, id=105 removed by soft count = 629\n",
      "--- gene=Rnase1:-, id=107 removed by soft count = 176\n",
      "--- gene=Rnase1:-, id=178 removed by soft count = 71\n",
      "--- gene=Rnase1:-, id=216 removed by soft count = 196\n",
      "--- gene=Rnase1:-, id=220 removed by soft count = 94\n",
      "--- number of probes: 228 , kept by blast: 220 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 219\n",
      "-- checking probes in region: Rnase4:+\n",
      "--- gene=Rnase4:+, id=70 removed by soft count = 102\n",
      "--- gene=Rnase4:+, id=193 removed by soft count = 575\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Rsad2:-\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Rsph1:-\n",
      "--- gene=Rsph1:-, id=137 removed by soft count = 67\n",
      "--- gene=Rsph1:-, id=185 removed by soft count = 86\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: S100a9:-\n",
      "--- gene=S100a9:-, id=3 removed by soft count = 829\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=S100a9:-, id=15 removed by soft count = 69\n",
      "--- gene=S100a9:-, id=18 removed by soft count = 66\n",
      "--- gene=S100a9:-, id=24 removed by soft count = 122\n",
      "--- gene=S100a9:-, id=53 removed by soft count = 65\n",
      "--- gene=S100a9:-, id=58 removed by soft count = 53\n",
      "--- gene=S100a9:-, id=59 removed by soft count = 66\n",
      "--- gene=S100a9:-, id=60 removed by soft count = 69\n",
      "--- gene=S100a9:-, id=62 removed by soft count = 92\n",
      "--- gene=S100a9:-, id=68 removed by soft count = 57\n",
      "--- number of probes: 300 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Saa1:-\n",
      "--- gene=Saa1:-, id=74 removed by soft count = 698\n",
      "--- number of probes: 135 , kept by blast: 134 , if remove dups: 134\n",
      "-- number of probes kept for this region: 134\n",
      "-- checking probes in region: Saa2:+\n",
      "--- gene=Saa2:+, id=70 removed by soft count = 797\n",
      "--- gene=Saa2:+, id=72 removed by soft count = 49017\n",
      "--- gene=Saa2:+, id=118 removed by soft count = 52\n",
      "--- gene=Saa2:+, id=178 removed by soft count = 76\n",
      "--- gene=Saa2:+, id=279 removed by soft count = 61\n",
      "--- gene=Saa2:+, id=287 removed by soft count = 157\n",
      "--- gene=Saa2:+, id=290 removed by soft count = 211\n",
      "--- number of probes: 300 , kept by blast: 293 , if remove dups: 275\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Serpinb11:+\n",
      "--- gene=Serpinb11:+, id=17 removed by soft count = 62\n",
      "--- gene=Serpinb11:+, id=18 removed by soft count = 67\n",
      "--- gene=Serpinb11:+, id=32 removed by soft count = 117\n",
      "--- gene=Serpinb11:+, id=109 removed by soft count = 83\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Sftpa1:+\n",
      "--- gene=Sftpa1:+, id=79 removed by soft count = 138\n",
      "--- gene=Sftpa1:+, id=99 removed by soft count = 510\n",
      "--- gene=Sftpa1:+, id=120 removed by soft count = 173\n",
      "--- gene=Sftpa1:+, id=244 removed by soft count = 66\n",
      "--- gene=Sftpa1:+, id=251 removed by soft count = 101\n",
      "--- gene=Sftpa1:+, id=277 removed by soft count = 331\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Sftpd:-\n",
      "--- gene=Sftpd:-, id=49 removed by soft count = 51\n",
      "--- gene=Sftpd:-, id=57 removed by soft count = 55\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Slpi:-\n",
      "--- gene=Slpi:-, id=25 removed by soft count = 56\n",
      "--- gene=Slpi:-, id=202 removed by soft count = 68\n",
      "--- gene=Slpi:-, id=261 removed by soft count = 92\n",
      "--- gene=Slpi:-, id=278 removed by soft count = 118\n",
      "--- number of probes: 290 , kept by blast: 286 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Sox8:-\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Spag8:-\n",
      "--- gene=Spag8:-, id=11 removed by soft count = 223\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Sparc:-\n",
      "--- gene=Sparc:-, id=70 removed by soft count = 203\n",
      "--- gene=Sparc:-, id=79 removed by soft count = 400\n",
      "--- gene=Sparc:-, id=81 removed by soft count = 65\n",
      "--- gene=Sparc:-, id=224 removed by soft count = 111\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Spib:-\n",
      "--- gene=Spib:-, id=19 removed by soft count = 62\n",
      "--- gene=Spib:-, id=59 removed by soft count = 311\n",
      "--- gene=Spib:-, id=128 removed by soft count = 194\n",
      "--- gene=Spib:-, id=267 removed by soft count = 58\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Spp1:+\n",
      "--- gene=Spp1:+, id=230 removed by soft count = 381\n",
      "--- gene=Spp1:+, id=232 removed by soft count = 270\n",
      "--- gene=Spp1:+, id=240 removed by soft count = 398\n",
      "--- gene=Spp1:+, id=244 removed by soft count = 122\n",
      "--- gene=Spp1:+, id=268 removed by soft count = 317\n",
      "--- gene=Spp1:+, id=299 removed by soft count = 93\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Sprr1b:-\n",
      "--- gene=Sprr1b:-, id=44 removed by soft count = 432\n",
      "--- gene=Sprr1b:-, id=291 removed by soft count = 121\n",
      "--- gene=Sprr1b:-, id=296 removed by soft count = 189\n",
      "--- gene=Sprr1b:-, id=297 removed by soft count = 442\n",
      "--- gene=Sprr1b:-, id=298 removed by soft count = 147\n",
      "--- gene=Sprr1b:-, id=299 removed by soft count = 499\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 288\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Sprr2i:+\n",
      "--- gene=Sprr2i:+, id=37 removed by soft count = 363\n",
      "--- gene=Sprr2i:+, id=38 removed by soft count = 61\n",
      "--- gene=Sprr2i:+, id=124 removed by soft count = 83\n",
      "--- gene=Sprr2i:+, id=146 removed by soft count = 182\n",
      "--- gene=Sprr2i:+, id=148 removed by soft count = 1041\n",
      "--- gene=Sprr2i:+, id=197 removed by soft count = 119\n",
      "--- gene=Sprr2i:+, id=199 removed by soft count = 65\n",
      "--- gene=Sprr2i:+, id=200 removed by soft count = 205\n",
      "--- gene=Sprr2i:+, id=201 removed by soft count = 639\n",
      "--- gene=Sprr2i:+, id=202 removed by soft count = 195\n",
      "--- number of probes: 300 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Sst:-\n",
      "--- gene=Sst:-, id=5 removed by soft count = 117\n",
      "--- gene=Sst:-, id=8 removed by soft count = 55\n",
      "--- gene=Sst:-, id=29 removed by soft count = 290\n",
      "--- gene=Sst:-, id=30 removed by soft count = 137\n",
      "--- gene=Sst:-, id=210 removed by soft count = 713\n",
      "--- gene=Sst:-, id=211 removed by soft count = 75\n",
      "--- gene=Sst:-, id=264 removed by soft count = 326\n",
      "--- number of probes: 300 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Stxbp5l:-\n",
      "--- gene=Stxbp5l:-, id=73 removed by soft count = 74\n",
      "--- gene=Stxbp5l:-, id=186 removed by soft count = 81\n",
      "--- gene=Stxbp5l:-, id=275 removed by soft count = 56\n",
      "--- gene=Stxbp5l:-, id=285 removed by soft count = 105\n",
      "--- gene=Stxbp5l:-, id=287 removed by soft count = 144\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Syt7:+\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Tac1:+\n",
      "--- gene=Tac1:+, id=39 removed by soft count = 351\n",
      "--- gene=Tac1:+, id=40 removed by soft count = 58\n",
      "--- gene=Tac1:+, id=41 removed by soft count = 128\n",
      "--- gene=Tac1:+, id=72 removed by soft count = 103\n",
      "--- number of probes: 280 , kept by blast: 276 , if remove dups: 276\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Tac2:+\n",
      "--- gene=Tac2:+, id=218 removed by soft count = 742\n",
      "--- gene=Tac2:+, id=244 removed by soft count = 146\n",
      "--- gene=Tac2:+, id=248 removed by soft count = 167\n",
      "--- gene=Tac2:+, id=259 removed by soft count = 75\n",
      "--- gene=Tac2:+, id=284 removed by soft count = 352\n",
      "--- gene=Tac2:+, id=292 removed by soft count = 77\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Tacstd2:-\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=Tacstd2:-, id=260 removed by soft count = 79\n",
      "--- gene=Tacstd2:-, id=264 removed by soft count = 136\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Tff2:-\n",
      "--- number of probes: 196 , kept by blast: 196 , if remove dups: 196\n",
      "-- number of probes kept for this region: 196\n",
      "-- checking probes in region: Tff3:-\n",
      "--- gene=Tff3:-, id=49 removed by soft count = 576\n",
      "--- gene=Tff3:-, id=82 removed by soft count = 127\n",
      "--- gene=Tff3:-, id=93 removed by soft count = 517\n",
      "--- gene=Tff3:-, id=117 removed by soft count = 58\n",
      "--- gene=Tff3:-, id=230 removed by soft count = 2014\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Timp2:-\n",
      "--- gene=Timp2:-, id=22 removed by soft count = 57\n",
      "--- gene=Timp2:-, id=73 removed by hard count = 0\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Tmem45a:-\n",
      "--- gene=Tmem45a:-, id=20 removed by soft count = 163\n",
      "--- gene=Tmem45a:-, id=43 removed by soft count = 125\n",
      "--- gene=Tmem45a:-, id=44 removed by soft count = 84\n",
      "--- gene=Tmem45a:-, id=45 removed by soft count = 255\n",
      "--- gene=Tmem45a:-, id=46 removed by soft count = 174\n",
      "--- gene=Tmem45a:-, id=47 removed by soft count = 123\n",
      "--- gene=Tmem45a:-, id=48 removed by soft count = 271\n",
      "--- gene=Tmem45a:-, id=56 removed by soft count = 141\n",
      "--- gene=Tmem45a:-, id=62 removed by soft count = 244\n",
      "--- gene=Tmem45a:-, id=64 removed by soft count = 103\n",
      "--- gene=Tmem45a:-, id=65 removed by soft count = 165\n",
      "--- gene=Tmem45a:-, id=103 removed by soft count = 110\n",
      "--- gene=Tmem45a:-, id=104 removed by soft count = 1282\n",
      "--- gene=Tmem45a:-, id=249 removed by soft count = 725\n",
      "--- gene=Tmem45a:-, id=253 removed by soft count = 161\n",
      "--- number of probes: 300 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Tnfrsf11b:-\n",
      "--- gene=Tnfrsf11b:-, id=0 removed by soft count = 63\n",
      "--- gene=Tnfrsf11b:-, id=21 removed by soft count = 52\n",
      "--- gene=Tnfrsf11b:-, id=72 removed by soft count = 496\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Trp63:+\n",
      "--- gene=Trp63:+, id=258 removed by soft count = 71\n",
      "--- gene=Trp63:+, id=259 removed by soft count = 283\n",
      "--- gene=Trp63:+, id=260 removed by soft count = 314\n",
      "--- gene=Trp63:+, id=261 removed by soft count = 63\n",
      "--- gene=Trp63:+, id=263 removed by soft count = 3113\n",
      "--- gene=Trp63:+, id=264 removed by soft count = 28348\n",
      "--- gene=Trp63:+, id=265 removed by soft count = 1536\n",
      "--- gene=Trp63:+, id=266 removed by soft count = 631\n",
      "--- gene=Trp63:+, id=267 removed by soft count = 944\n",
      "--- gene=Trp63:+, id=268 removed by soft count = 1435\n",
      "--- gene=Trp63:+, id=269 removed by soft count = 332\n",
      "--- gene=Trp63:+, id=287 removed by soft count = 302\n",
      "--- gene=Trp63:+, id=288 removed by soft count = 741\n",
      "--- number of probes: 300 , kept by blast: 287 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Trpm5:-\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Tspan1:-\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Ttc36:-\n",
      "--- gene=Ttc36:-, id=81 removed by soft count = 149\n",
      "--- gene=Ttc36:-, id=286 removed by soft count = 72\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Vpreb3:+\n",
      "--- gene=Vpreb3:+, id=170 removed by soft count = 531\n",
      "--- gene=Vpreb3:+, id=171 removed by soft count = 812\n",
      "--- gene=Vpreb3:+, id=240 removed by soft count = 70\n",
      "--- gene=Vpreb3:+, id=251 removed by soft count = 195\n",
      "--- gene=Vpreb3:+, id=288 removed by soft count = 69\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Xcl1:-\n",
      "--- gene=Xcl1:-, id=145 removed by soft count = 151\n",
      "--- gene=Xcl1:-, id=232 removed by soft count = 68\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "-- checking probes in region: Zg16:-\n",
      "--- gene=Zg16:-, id=37 removed by soft count = 167\n",
      "--- gene=Zg16:-, id=39 removed by soft count = 104\n",
      "--- gene=Zg16:-, id=81 removed by soft count = 312\n",
      "--- gene=Zg16:-, id=126 removed by soft count = 67\n",
      "--- gene=Zg16:-, id=127 removed by soft count = 74\n",
      "--- gene=Zg16:-, id=134 removed by soft count = 83\n",
      "--- gene=Zg16:-, id=135 removed by soft count = 192\n",
      "--- gene=Zg16:-, id=139 removed by soft count = 182\n",
      "--- gene=Zg16:-, id=140 removed by soft count = 200\n",
      "--- gene=Zg16:-, id=141 removed by soft count = 169\n",
      "--- gene=Zg16:-, id=168 removed by soft count = 110\n",
      "--- gene=Zg16:-, id=170 removed by soft count = 75\n",
      "--- gene=Zg16:-, id=192 removed by soft count = 122\n",
      "--- gene=Zg16:-, id=223 removed by soft count = 948\n",
      "--- gene=Zg16:-, id=248 removed by soft count = 105\n",
      "--- number of probes: 300 , kept by blast: 285 , if remove dups: 282\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 270\n",
      "- summarize\n",
      "-- region: Acpp:- , number of probes: 270\n",
      "-- region: Agt:- , number of probes: 270\n",
      "-- region: Aldh3a1:+ , number of probes: 270\n",
      "-- region: Aldob:- , number of probes: 270\n",
      "-- region: Alox5ap:+ , number of probes: 270\n",
      "-- region: Alpi:- , number of probes: 270\n",
      "-- region: Alpl:- , number of probes: 270\n",
      "-- region: Ang4:- , number of probes: 167\n",
      "-- region: Anxa8:+ , number of probes: 270\n",
      "-- region: Apoa1:+ , number of probes: 270\n",
      "-- region: Apoa2:+ , number of probes: 270\n",
      "-- region: Apoc2:- , number of probes: 270\n",
      "-- region: Apod:- , number of probes: 270\n",
      "-- region: Ascl1:- , number of probes: 270\n",
      "-- region: Avil:+ , number of probes: 270\n",
      "-- region: Bgn:+ , number of probes: 270\n",
      "-- region: C1qb:- , number of probes: 270\n",
      "-- region: C3:- , number of probes: 270\n",
      "-- region: Calb2:- , number of probes: 270\n",
      "-- region: Cald1:+ , number of probes: 270\n",
      "-- region: Camp:- , number of probes: 270\n",
      "-- region: Car4:+ , number of probes: 270\n",
      "-- region: Car8:- , number of probes: 270\n",
      "-- region: Ccdc30:- , number of probes: 270\n",
      "-- region: Cck:- , number of probes: 270\n",
      "-- region: Ccl1:- , number of probes: 270\n",
      "-- region: Ccl2:+ , number of probes: 270\n",
      "-- region: Ccl20:+ , number of probes: 270\n",
      "-- region: Ccl6:- , number of probes: 196\n",
      "-- region: Ccl9:- , number of probes: 270\n",
      "-- region: Cd14:- , number of probes: 270\n",
      "-- region: Cebpb:+ , number of probes: 270\n",
      "-- region: Cga:+ , number of probes: 270\n",
      "-- region: Cilp:+ , number of probes: 270\n",
      "-- region: Ckb:- , number of probes: 270\n",
      "-- region: Ckm:+ , number of probes: 270\n",
      "-- region: Clps:- , number of probes: 270\n",
      "-- region: Cnfn:- , number of probes: 270\n",
      "-- region: Cnmd:- , number of probes: 270\n",
      "-- region: Col1a1:+ , number of probes: 270\n",
      "-- region: Col4a1:- , number of probes: 270\n",
      "-- region: Col6a1:- , number of probes: 270\n",
      "-- region: Cpn1:- , number of probes: 270\n",
      "-- region: Crabp1:+ , number of probes: 270\n",
      "-- region: Cryaa:+ , number of probes: 270\n",
      "-- region: Ctrb1:- , number of probes: 270\n",
      "-- region: Cxcl13:+ , number of probes: 270\n",
      "-- region: Cyp1a2:- , number of probes: 270\n",
      "-- region: Cyp2f2:+ , number of probes: 270\n",
      "-- region: Dbn1:- , number of probes: 270\n",
      "-- region: Dclk1:+ , number of probes: 270\n",
      "-- region: Dlk1:+ , number of probes: 270\n",
      "-- region: Dppa3:+ , number of probes: 270\n",
      "-- region: Ecm1:- , number of probes: 270\n",
      "-- region: Efna3:- , number of probes: 270\n",
      "-- region: Elovl4:- , number of probes: 270\n",
      "-- region: Fabp1:+ , number of probes: 270\n",
      "-- region: Fabp6:- , number of probes: 270\n",
      "-- region: Fam183b:- , number of probes: 270\n",
      "-- region: Fam25c:- , number of probes: 270\n",
      "-- region: Fgg:+ , number of probes: 270\n",
      "-- region: Foxa1:- , number of probes: 270\n",
      "-- region: Foxa2:- , number of probes: 270\n",
      "-- region: Foxa3:- , number of probes: 270\n",
      "-- region: Foxj1:- , number of probes: 270\n",
      "-- region: Fxyd3:- , number of probes: 270\n",
      "-- region: Fxyd6:+ , number of probes: 270\n",
      "-- region: Gabrp:- , number of probes: 270\n",
      "-- region: Gal:- , number of probes: 270\n",
      "-- region: Gatm:- , number of probes: 270\n",
      "-- region: Ghrl:- , number of probes: 270\n",
      "-- region: Gm13539:+ , number of probes: 220\n",
      "-- region: Gm2a:+ , number of probes: 270\n",
      "-- region: Gm94:- , number of probes: 270\n",
      "-- region: Gp2:- , number of probes: 270\n",
      "-- region: Gpx3:+ , number of probes: 270\n",
      "-- region: Gpx6:+ , number of probes: 270\n",
      "-- region: Grhl1:+ , number of probes: 270\n",
      "-- region: Gsdma:+ , number of probes: 270\n",
      "-- region: Gsta2:- , number of probes: 259\n",
      "-- region: Gsta4:+ , number of probes: 270\n",
      "-- region: Gstm3:- , number of probes: 270\n",
      "-- region: Gsto1:+ , number of probes: 270\n",
      "-- region: Guca2a:+ , number of probes: 270\n",
      "-- region: H19:- , number of probes: 270\n",
      "-- region: Hamp2:- , number of probes: 215\n",
      "-- region: Hba-a2:+ , number of probes: 270\n",
      "-- region: Hbb-y:- , number of probes: 270\n",
      "-- region: Hck:+ , number of probes: 265\n",
      "-- region: Hnf4a:+ , number of probes: 270\n",
      "-- region: Hnf4g:+ , number of probes: 270\n",
      "-- region: Hp:- , number of probes: 270\n",
      "-- region: Iapp:+ , number of probes: 270\n",
      "-- region: Idi2:+ , number of probes: 225\n",
      "-- region: Ifi27l2b:- , number of probes: 181\n",
      "-- region: Igf1:+ , number of probes: 270\n",
      "-- region: Igf2:- , number of probes: 270\n",
      "-- region: Igfbp5:- , number of probes: 270\n",
      "-- region: Igfbp6:+ , number of probes: 270\n",
      "-- region: Ins2:- , number of probes: 270\n",
      "-- region: Isg15:- , number of probes: 270\n",
      "-- region: Itga6:+ , number of probes: 270\n",
      "-- region: Jpt1:- , number of probes: 270\n",
      "-- region: Klk14:+ , number of probes: 270\n",
      "-- region: Klk7:+ , number of probes: 270\n",
      "-- region: Krt10:- , number of probes: 270\n",
      "-- region: Krt13:- , number of probes: 270\n",
      "-- region: Krt14:- , number of probes: 191\n",
      "-- region: Krt16:- , number of probes: 90\n",
      "-- region: Krt17:- , number of probes: 270\n",
      "-- region: Krt19:- , number of probes: 270\n",
      "-- region: Krt5:- , number of probes: 270\n",
      "-- region: Krt71:- , number of probes: 270\n",
      "-- region: Krt76:- , number of probes: 270\n",
      "-- region: Krt77:- , number of probes: 270\n",
      "-- region: Krt9:- , number of probes: 270\n",
      "-- region: Krtdap:+ , number of probes: 270\n",
      "-- region: Lcn2:- , number of probes: 270\n",
      "-- region: Lcn3:+ , number of probes: 270\n",
      "-- region: Lgals2:- , number of probes: 270\n",
      "-- region: Lgals3:+ , number of probes: 270\n",
      "-- region: Lifr:+ , number of probes: 270\n",
      "-- region: Ltf:+ , number of probes: 270\n",
      "-- region: Ly6c1:- , number of probes: 106\n",
      "-- region: Ly6c2:- , number of probes: 121\n",
      "-- region: Ly6d:- , number of probes: 270\n",
      "-- region: Ly6g6f:- , number of probes: 270\n",
      "-- region: Lypd2:- , number of probes: 270\n",
      "-- region: Lypd3:+ , number of probes: 270\n",
      "-- region: Mal:- , number of probes: 270\n",
      "-- region: Meg3:+ , number of probes: 270\n",
      "-- region: Meig1:- , number of probes: 270\n",
      "-- region: Mgp:- , number of probes: 270\n",
      "-- region: Mlph:+ , number of probes: 270\n",
      "-- region: Mpo:+ , number of probes: 270\n",
      "-- region: Mt3:+ , number of probes: 183\n",
      "-- region: Mt4:+ , number of probes: 270\n",
      "-- region: Muc13:+ , number of probes: 270\n",
      "-- region: Mymx:- , number of probes: 270\n",
      "-- region: Myog:+ , number of probes: 270\n",
      "-- region: Nefh:- , number of probes: 270\n",
      "-- region: Ngp:+ , number of probes: 270\n",
      "-- region: Nppa:+ , number of probes: 270\n",
      "-- region: Npy:+ , number of probes: 270\n",
      "-- region: Nrgn:- , number of probes: 270\n",
      "-- region: Nts:- , number of probes: 270\n",
      "-- region: Oas1d:+ , number of probes: 252\n",
      "-- region: Oas1h:+ , number of probes: 270\n",
      "-- region: Odf3b:- , number of probes: 270\n",
      "-- region: Olfm1:+ , number of probes: 270\n",
      "-- region: Pcp4:+ , number of probes: 270\n",
      "-- region: Pdzk1ip1:+ , number of probes: 270\n",
      "-- region: Penk:- , number of probes: 270\n",
      "-- region: Pgc:+ , number of probes: 270\n",
      "-- region: Pigr:+ , number of probes: 270\n",
      "-- region: Pik3r5:+ , number of probes: 270\n",
      "-- region: Pla2g1b:+ , number of probes: 270\n",
      "-- region: Plac8:- , number of probes: 270\n",
      "-- region: Plk2:+ , number of probes: 270\n",
      "-- region: Plvap:- , number of probes: 270\n",
      "-- region: Pomc:+ , number of probes: 270\n",
      "-- region: Pou2f3:- , number of probes: 270\n",
      "-- region: Psca:+ , number of probes: 270\n",
      "-- region: Ptgds:- , number of probes: 270\n",
      "-- region: Pyy:- , number of probes: 270\n",
      "-- region: Rap1gap2:- , number of probes: 270\n",
      "-- region: Rarres1:- , number of probes: 270\n",
      "-- region: Reg3b:+ , number of probes: 270\n",
      "-- region: Reg4:+ , number of probes: 270\n",
      "-- region: Retnla:+ , number of probes: 235\n",
      "-- region: Riiad1:- , number of probes: 270\n",
      "-- region: Rnase1:- , number of probes: 219\n",
      "-- region: Rnase4:+ , number of probes: 270\n",
      "-- region: Rsad2:- , number of probes: 270\n",
      "-- region: Rsph1:- , number of probes: 270\n",
      "-- region: S100a9:- , number of probes: 270\n",
      "-- region: Saa1:- , number of probes: 134\n",
      "-- region: Saa2:+ , number of probes: 270\n",
      "-- region: Serpinb11:+ , number of probes: 270\n",
      "-- region: Sftpa1:+ , number of probes: 270\n",
      "-- region: Sftpd:- , number of probes: 270\n",
      "-- region: Slpi:- , number of probes: 270\n",
      "-- region: Sox8:- , number of probes: 270\n",
      "-- region: Spag8:- , number of probes: 270\n",
      "-- region: Sparc:- , number of probes: 270\n",
      "-- region: Spib:- , number of probes: 270\n",
      "-- region: Spp1:+ , number of probes: 270\n",
      "-- region: Sprr1b:- , number of probes: 270\n",
      "-- region: Sprr2i:+ , number of probes: 270\n",
      "-- region: Sst:- , number of probes: 270\n",
      "-- region: Stxbp5l:- , number of probes: 270\n",
      "-- region: Syt7:+ , number of probes: 270\n",
      "-- region: Tac1:+ , number of probes: 270\n",
      "-- region: Tac2:+ , number of probes: 270\n",
      "-- region: Tacstd2:- , number of probes: 270\n",
      "-- region: Tff2:- , number of probes: 196\n",
      "-- region: Tff3:- , number of probes: 270\n",
      "-- region: Timp2:- , number of probes: 270\n",
      "-- region: Tmem45a:- , number of probes: 270\n",
      "-- region: Tnfrsf11b:- , number of probes: 270\n",
      "-- region: Trp63:+ , number of probes: 270\n",
      "-- region: Trpm5:- , number of probes: 270\n",
      "-- region: Tspan1:- , number of probes: 270\n",
      "-- region: Ttc36:- , number of probes: 270\n",
      "-- region: Vpreb3:+ , number of probes: 270\n",
      "-- region: Xcl1:- , number of probes: 270\n",
      "-- region: Zg16:- , number of probes: 270\n",
      "- Number of probes kept: 54485\n",
      "- Saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-10_Aire\\Genes_TSS_DNA\\.\\blast_full_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "# screen\n",
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "num_probes_per_region = 270\n",
    "kept_pbs, blast_keep_dic, hard_count_list, soft_count_list = check.Screening_Probes_by_Blast(library_folder, \n",
    "                                                                                             num_probes_per_region,\n",
    "                                                                                             hard_thres=target_len,\n",
    "                                                                                             soft_count_th=50,\n",
    "                                                                                             )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "waiting-convention",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox â‰¥ 6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_dpi_ratio', { dpi_ratio: fig.ratio });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    var cursor = msg['cursor'];\n",
       "    switch (cursor) {\n",
       "        case 0:\n",
       "            cursor = 'pointer';\n",
       "            break;\n",
       "        case 1:\n",
       "            cursor = 'default';\n",
       "            break;\n",
       "        case 2:\n",
       "            cursor = 'crosshair';\n",
       "            break;\n",
       "        case 3:\n",
       "            cursor = 'move';\n",
       "            break;\n",
       "    }\n",
       "    fig.rubberband_canvas.style.cursor = cursor;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            /* FIXME: We get \"Resource interpreted as Image but\n",
       "             * transferred with MIME type text/plain:\" errors on\n",
       "             * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "             * to be part of the websocket stream */\n",
       "            evt.data.type = 'image/png';\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                evt.data\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from http://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * http://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.which === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.which;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.which !== 17) {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    if (event.altKey && event.which !== 18) {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    if (event.shiftKey && event.which !== 16) {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k';\n",
       "    value += event.which.toString();\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(msg['content']['data']);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    var manager = IPython.notebook.keyboard_manager;\n",
       "    if (!manager) {\n",
       "        manager = IPython.keyboard_manager;\n",
       "    }\n",
       "\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAApQAAAFKCAYAAACjE8/+AAAgAElEQVR4XuxdB3gU1RY+9N577yGU0Am9d1Ckd1QEERGRrvQOCgoWpImCIiiKFKUTeu8ldELvvYcO7/vv5mxuJrPZ2d3ZzYR3z/e970l25pZ/7sw995T/xHr9+vVrUqIQUAgoBBQCCgGFgEJAIaAQcBOBWEqhdBM5dZtCQCGgEFAIKAQUAgoBhYBAQCmUaiEoBBQCCgGFgEJAIaAQUAh4hIBSKD2CT92sEFAIKAQUAgoBhYBCQCGgFEq1BhQCCgGFgEJAIaAQUAgoBDxCQCmUHsGnblYIKAQUAgoBhYBCQCGgEFAKpVoDCgGFgEJAIaAQUAgoBBQCHiGgFEqP4FM3KwQUAgoBhYBCQCGgEFAIKIVSrQGFgEJAIaAQUAgoBBQCCgGPEFAKpUfwqZsVAgoBhYBCQCGgEFAIKASUQqnWgEJAIaAQUAgoBBQCCgGFgEcIKIXSI/jUzQoBhYBCQCGgEFAIKAQUAkqhVGtAIaAQUAgoBBQCCgGFgELAIwSUQukRfOpmhYBCQCGgEFAIKAQUAgoBpVCqNaAQUAgoBBQCCgGFgEJAIeARAkqh9Ag+dbNCQCGgEFAIKAQUAgoBhYBSKNUaUAgoBBQCCgGFgEJAIaAQ8AgBpVB6BJ+6WSGgEFAIKAQUAgoBhYBCQCmUag0oBBQCCgGFgEJAIaAQUAh4hIBSKD2CT92sEFAIKAQUAgoBhYBCQCGgFEq1BhQCCgGFgEJAIaAQUAgoBDxCQCmUHsGnblYIKAQUAgoBhYBCQCGgEFAKpVoDCgGFgEJAIaAQUAgoBBQCHiGgFEqP4FM3KwQUAgoBhYBCQCGgEFAIKIVSrQGFgEJAIaAQUAgoBBQCCgGPEFAKpUfwqZsVAgoBhYBCQCGgEFAIKASUQqnWgEJAIaAQUAgoBBQCCgGFgEcIKIXSI/jUzQoBhYBCQCGgEFAIKAQUAkqhVGtAIaAQUAgoBBQCCgGFgELAIwSUQukRfOpmhYBCQCGgEFAIKAQUAgoBpVCqNaAQUAgoBBQCCgGFgEJAIeARAkqh9Ag+dbNCQCGgEFAIKAQUAgoBhYBSKNUaUAgoBBQCCgGFgEJAIaAQ8AgBpVB6BJ+6WSGgEFAIKAQUAgoBhYBCQCmUag0oBBQCCgGFgEJAIaAQUAh4hIBSKD2CT92sEFAIKAQUAgoBhYBCQCGgFEq1BhQCCgGFgEJAIaAQUAgoBDxCQCmUHsGnblYIKAQUAgoBhYBCQCGgEFAKpVoDCgGFgEJAIaAQUAgoBBQCHiGgFEqP4Au/+cmTJxQcHEzp0qWjuHHjmtSqakYhoBBQCCgEFAIKAasi8OLFC7px4wYFBARQwoQJrTpMn4xLKZQmwbxr1y4KDAw0qTXVjEJAIaAQUAgoBBQCMQWBnTt3UunSpWPKcL0yTqVQmgTr2bNnKVeuXIRFlSlTJpNaVc0oBBQCCgGFgEJAIWBVBK5cuSKMSWfOnKGcOXNadZg+GZdSKE2C+eLFi5QtWza6cOECZc2a1aRWVTMKAYWAQkAhoBBQCFgVAbX3hz8ZpVCatErVojIJSNWMQkAhoBBQCCgEYggCau9XCqXpS1UtKtMhVQ0qBBQCCgGFgELA0giovV8plKYvULWoTIdUNagQUAgoBBQCCgFLI6D2/jdIobx+/Tr17t2bVqxYQY8fP6YqVarQ+PHjqWDBgmKW+/fvp88++4x2795NadKkoe7du1OfPn3sCLx69YqGDx9OM2bMoDt37lDFihVp8uTJlDdvXpcWsVpULsGlLlYIKAQUAgoBhUCMR0Dt/W+QQlm2bFmKHTs2ff/995QkSRIaPHgwbd26lUJCQoSC6e/vT40aNaJevXrR9u3bqWvXrkJh7NChg0AByiT+PXPmTMqSJQv169ePTp8+TYcPH6b48eMbXuxqURmGSl2oEFAIKAQUAgqBNwIBtfe/IQrlrVu3qFu3bjRo0CAqVKiQmNWBAweoWLFigr4nKCiIfvzxRwKlD5ONDxgwgBYsWEDHjh2jZ8+eUdq0aWncuHHUpUsXcf/du3cpc+bM9Msvv1CrVq0ML3i1qAxDpS5UCCgEFAIKAYXAG4GA2vvfEIVSuxqvXbsm3Nnr168XCmPz5s0pZcqUNHfuXPulUDJr1apFuBaKZpkyZej48ePk5+dnvwZu7yJFigjLpVFRi8ooUuo6hYBCQCGgEFAIvBkIqL3/DVQoO3fuTD/99BMlSJCA/v33X6pdu7ZQCuvVq0dfffWVfcZHjhwR1kxUtjl//jw1bdqUQkNDKVGiRPZrWrRoIf62ZMkShyv+/v37hP+xMLnp/yMP5a9bz9LUDaeoRoH01KVKHsqaKrFbX4p/9lykr1cdp0r50lLXqnkpZ9okbrUT1U2jlhyhhfsuUfNS2ahTpVyUNmkCev7ylfjbL5vPUMJ4cejXDwIpRaJ4UfY9ae1JmrvjPH3TohiVy5PG0DgPX75H3f/YR4WzpKDvWhU3dI+rFx26dI86/rqL3iqSmQa/ZYsj1sqRy/fpw992U+1CGWjo2zbLvhH5LugkLdp/iSa3LUEFMiXXvWXs8qO0eN9lmtSmOJXKmdpIs7rX/LDmJP2x8zxNbV+SimRNGWU7e8/foS6z99B75XPSJ9Vci30GFpPWnaRNJ25S3cIZqWu1vJTLC+vOCBDfrzlJf++5QFPblaRCmVMYueX/9hq8s4v2XaIZm85Q7NixaGHX8uLdtZJcv/+Epm88TUsOXqGWpbNRz1rhRguM88+d52n8yuM0unFhqlvYtWIYr1+/psGLD9G6Yzfot46BlCddUitNXYzl1sOn1GHWLjG2iS2LmTo+7BUDFwXTk+evKE7sWNSkeBYa3TiA4seNbWo/RhpTCmU4Sm8MDyUURcRMTpkyRVgkN2/eTFAM27RpQyNGjLDPGPGRefLkoU2bNgkLZfv27enly5ciDpPl3XffpcuXLwuXuSMZNmyYiL/Uii8Vyk0nb9DopUepR818Tj9Id0OfUcdfd9P526FiyP4Zk9H09qUoUXxjH+FdZ2/ToIWHxKbdpkx2+7RfvXpNpUcH0a1Hz8Tf4saORUMbFqL2ZXMYeRcjXFPjm/V06sYj8bfYsYg+r+tPH1XJ43I7jm548fIVFR62UnyEIPHjxKYUiePRk2cv6cHTF/bbulfPS71q53fY78tXr6nYiFX04MkLypA8Aa3qUUW0E5UcuHCX2v+8g+4/sfVzeHgdSpLA/JrvUBRXH7km+oBSB8VSK62nb6dtp2+JP//brYJThY3vr/DlWrp09zGVy52G/uhcNlK7R6/cp3rfbRJ/z5IyEa3oUYmSJYwaFz3MNpy4Qe/9slP81K1aXupTx/GzwDXjVhyjyetPUdIEcWn3oJqGFYsR/x2hX7aciTAErLuOFXNR/3oFhKLiK4GCVHT4Kgp99pJqFshAM94rZe8aysOgRYfo2v2n9F2rYk7XzV+7LtC3QSfo+avXusPPnTYJzexQmhLHj7z+9py7Q1PWn6IDF++Ke5MliEtfNStCpT04HJiJ4dMXL2n+notijBfvPLY3DYWyePZUZnblUVs4nH654hg9e2H71uC7uLV/dUqfzFbr+c6jZ1R53Drx3cE3ZEPfaoTvSq+/9ot3ZnyzIhQrVizaeOIGjVt5jAbUK0Dl86a1jwkY9Pn7gPg31qujw6Mrk8Cz7zv/gPiuQWoVzECjGxUW43BHsAa/DTopbt0/pBalTGw8J8FZfy2mbaOdZ25HuKy6f3px2PX1wUIplOGP4Y1RKHlKyNpGkXa4smGFrF+/vq6Fcu/evSL5plmzZroWyqdPn9LixYsdruvotlBik6nz7UY6ce0hJY4fh5Z/VolypHFs0Vt68Ap9MndvhPlMaFGUmpRwXtUHyij6woaGD+OiTyoIKxtk3/k71HjyVvvmgw9kgrixaVO/apQuWQL6cV0Inbz+kMY1K0IJ4jpWXnGaDxyzRrSD+WBjzZ0uCa3tXdXZt8Xw77LCg5Msf+y5gWypE9GF248pSfw4tOnz6pQ6if4H8ODFu9Rw0hZ7v+8Uy+zQ4ogNcMHeSzRm6dEISuvWL6pT5pThVnE0hme67dQtmrPzPOXPkIy618hneG648PLdx1Txq7XEekTKxPFoVY/KlD65bRODbA25SW1m7LD/u4pfOmGR1QqsC8P+O0JV/dJR05K2NVJq1Gq6+dB2cJjbqQxlS52Yxiw7KtZC16p56KPZe2hVmDKLa5qXzErjmxd1aQ73Qp9T7W83iLUG+ahybupfv0CUbQxedIhmbz8nrpnaroTTwxU3VnDICrHO8NzrFspIi/ZfphsPbP22Kp1NWDxg/XBXcIAZs+wYvabXNOStglFuzNjMm06xvUfoEusPSjnk+NUH4v2DDHu7IL1fIZd9SHhvhv13mAKypKSPq9oOX1gDsrKlN348czx7lifPX1LXOXtp7bHrkS7XKrhR4YH1PnTxYXr64hV9UCEXBWQ1Zmldf/w6/bLlLH1aPa9QXvEuQCF7+vyVUJbwHGDh7zhrN129/yTSELAeZYXr0dMXQuHKlyEZ9aiRz/DhAJav33ecE9+GuHFi08dVctvX008bT9OJaw8iWcJwGEBfcWPHps6Vc9Oqw1fpm9UnxBihkD9/9UocYvvU9qNu1W3v9FcrjgmlmGVQgwLiMA2rPGTLF7bnj282vt2wwE0Is/JdvBNK9b7dZP+ewKK+ro9r30ngO27lcXrw5DmNaFhY4IM5QFGVZWyTAGodGG5AMPouABOsQ36P1/SuYtiKigPlrC1n6It6BSh/xmS6XZYcuVoYMd4vbyt1OGvrWfH/WNO/vF9arBd4LiatDaGeNf0Mr0Oj85OvUwplOBoxWqEEZdCaNWuEJTJOnHBlBbGTUPjwN1AFzZ492z5jvRhKZITDasmCGMqiRYuKhB6j4q1FhQ8tvywYS8FMyalP7fy0+9wdwimNpVSOVDTvo3ION8CF+y5Sz3kHKGG82EJZOXDxHlXLn45mdghXJoIv3qM5O84J6yUskXwyhZv23wOX7X3lS5+U/vu0ojgJTlh1nL5fG0KZUySkJd0rUZWwU/d75XJQiRyp6LM/94v7cHKsH+DYrbN4/yVxbbw4sahXrfzig4uNflO/6rqP4N7j50JJq1kwgzhJs2AT/+/gZdoacktYN/OmD3cFzdt1nj7/J1hYsjb0rUobT94QGxYO4LBuQIGs9NU6evz8ZQRFZv+FuzR/zwWx0b1TLItw73+5/FiEcZXPk0a4W2r4p6f25WwfuXXHrtOAhcF05Z5tA4Si+ujZS/HfOADAbQzLLzaqZy9fCWXm8OXwMIpVPSuTXwb9D6oeKN+sOk4/rA2hZAnjEizH6Asf2J/fKyU2R2wiUFr2nr8rNjq2ys58vzQdvXpfKKRYW7Ak9Jt/gP7afZEyJk9I2wfUEN0VGbbSbmHF2O+FPqPLYXOrWSA9BR21KSMlc6QiKEiQn94tFeH5OHufevy5Tyh2LB0q5HTqlpfvaRCQiX5sW0Lczso8MP6sRr5IB658A5fR85evaVr7klSnUEaCUtV/QbAIf4C0LJVNWOeMCNYdXJt4X98tn5NKZE9FCIv4epVNsVjbuwrljsItKV+L62XL7MwtZ2j4f0dEO1jPq3tWFu8mnlebn7bT2Vs2r8OeQTXpxavXVCbsYPZJtTyUTRN+8sWCYHEt1kSNAuHvzZ5zt6npFNv3BH1AiQi5/oD+2HmBEsWLQ/uG1LJbfnDA/G3bObr+4AkNqF8ggqUTIQ/TNpy2Q4Z1AUUoecJ4hMMClMQqfmkjKP3Lg6/Qp3/sE2MPzJWa/vqonFAGmoQdVLE+q+G9+nkHbTp5U7yv+JZ8UjUvNfhhE71+TTTj3VLiW8DC7zr+3aREFhrfrKjTw8GMTadp1NKjER53phQJacvn1YVlvtK4deK3KW1LUD3pW4Zn/v7MXZGWCb4JU9qVpPErj9Hv28+LbyQOCrfDrJP4zvB7KH8b0FBQryriOXSctYvWHLtOvK7xDredsYO2nrolvpVYv0bWl3ZwKw9fFQdAyLzOZalM7jTi24D3tmr+dCIMaEvILXG4X/FZZcqexrUwphWHrlCX38MNGHimeLYs1+4/IYR4wIMihwxduB1Kdb/dKL5dfCDFnBEKlTJRfPqwcm5h3S0+crVo6o8Py1LZ3KlpYtBJ0R4EVnysj+rfrBcGggZFMtGPbWzfBG+It/Z+b4zV223GaIUSHJPFixcXSmX16jbF4/nz54IqqGHDhpQhQwaaOnUqnTp1yq5w9u/fnxYuXCiSdmCFTJ8+PU2YMIE6duwo7rdaljdOrNjkZMFp9uDFe0LJQ6wflCvIR1VyU69afrqWwL93X6C+8w8Kpal/PX/x37A27hpYk568eEkDFgTTuuM37N10gtuvfgGave2ssFRBYIlDn/iAf1gpFw1sUJAafL9JKEHtymanUY0CCHF2E4NOCHdygnix7e4T+XSut6g/n3+Q5u2+QIE5U1PDYpmFi09WZrT3wPKJ+COc4nGah0CBg7XmXNgGCyV7/sfl7bdCuUPcIz5Af3Yup/tusfsUivfbRTILSw+7h4EX+gJ2cEW9VSQTXX/wNJLrBdafXGmSUL3vbB9GbID1CmcU8X0Nvt9s/xDiQ9pq+jbafjqi64YHxq73kOsP6Z+9F4ViBkUFiicUfyjGiM2CiwwWlfJfrqWbD58Ky5BfhqTEygPGiRgmxJxh7BAoetjoYOGWpU6hDCLUoNbEjcIFB+U0eFgdcYn/4OX2cAFHH6acaRLTyp6VqdX07bTv/F2qkDcNzekU2T2O+6Fg/Lz5NDUslkVsnlBUio2wbRS8WbYtk11YhKIS3nRxDZ7b7kG1hJUI64OVeViOoPywYJPK1X+Z+KesXGHOeBewFiGwtMMS60iAO3CdvD7ErthhDH3r+NPYZUeFkgRZ8mlFu1Vfry0ohlASYFnBGBDbCys2DilyGAPuxSaaNVUiav3T9giWSGyaWGuwNKKd4GG1I7m1c/dfKizY09uXpNqFMtqHAss42oNwOAYUVqwpyKwOpcXhBAeWaRtO2Q9G2LxxyIJAccchF98HWUHia/7afYH6zT8olJQdA2qIdbvk4GVxkMScIcKyNKgW/bTpNE1aFyL+1rh4FhrYoAAFjg4SY4e3o0WpbOI3tjJ/37o4NSwaHt7R9+8D9LdkbYMykyOK53j/yXNaedgWKlIsW0oqkys1TdtoU4yhDO0+d5vGrTgu/t27lh99KnkP8Oz5NwYUWOGggkP3sav3qe63tlAQhKGsP35DWALx/s79sIxQnHmd8P28XtrO2C4UO7YSQ+FixRY4jFxyRHxjsSd0qpTb4Tq9cu8xzd52TjwrGAQQmnL82gNxPWI425bJQSVGrhbKLv6Nb02diRvpTuhzcT0wSZ00vnCvs9veYWdExOPma2SDAp51y2nbhFGkRPaUtKBrBXEZDsFYgzvCXNnFs6ekhV0rRPCC4X2EMtpsqu3wg/0LnjAIwmRg3YTF9oMKOWnw4sPi7wFZUggDiLdEKZThyMZohRKbApJuEAuJhJxUqVLR6NGjaeXKlYLQPGHChHblEvySoBL6+OOPhZL53nvvCRQGDhxI06ZNEzRBOXPmpL59+4r2goODLcFDufvsbVocZrGBywfWJWwymDtOpyPeKUT7z9+lBWFWFZyo4SZsFZg9QiwJAsChYODlw+m39KggYRUb2agwwc0DKxxE3gjSJo1vd3Hig4yN7MvlR+mnTWfExvVty2J2C+Qv75ei6v4ZhAsFsUH4EMkCK8GEFo4DsyuNWytOk7AkZU6ZUFgS0T+UAwhiEKG4whUJ9wx/sDCOYyPrCiUaGw4UPFnYEoi/vfXDJjp06b5wS8nKhXw9lBpYKeWYSvl3WI5+3nxGWDG/ahog5gzLBlynW07dpNM3HlH6ZAkoS6pEQqFKlTieUF7husEzyzdwudg8kHiBJJDqX6+n0zcfEZRfuI6xEcHi8eu2c+LDCGtU7YkbxTWQotlS0rEr94VLEZI8YVyx4WJT4TUA9xLi5IYsPmx3BcvPEh/qBR+XJ1gp2IoANyu7yqEUstUL4QvHR9UTfbEywuECUAygmMKix64yVh5+335OHArgdt83uJauu7fnvP3iXiRhze5YhnacvkUtp9uUGuAhrPClstK4ZlG7zZtP3Uq7ztosohC0B0uWLLC4wX3Hgk0tzwCbQql1/z5+9pIKDFkhfmPrWITGwsIT/t59kb5bc1JYr1g4XEN7PcJEsCnrCSyjRYavEocCrC9WpH5oXVxYWjhel++FsgPFAtZhHNxwSIQbGMo33gPEhTraRNkqqw0N2BJyU1i+IHifOA4N1qJjVx/Qu+VyCAtTt7n7IkwBB1iEZsDFDCUFMdp50iWhxd0qiu8AFJQvmwSI75FsacV3C9YpXPPw6QtxD9YcngvmDXfwkSs2az2+ST1q+tHoZUfFgWHPoFr2OFIOw8C72LJ0uGuW3yscSvVc5LoPgkjEByN+Fc+x5oQNwg2NeHCsRYTMQLQhLt3m7hXWaVhj8bxwqIOHRw7x0a5RtAPcgN/AhcE0Z8d5+2ECv/3zcXlh6W82Zavom0NTZOV08+fVhKcEfUd1cEN7w/49LDxdsIi2LpNdJAuxwAvQo4YfFR2xSvwJSm75PGkJluOP50QMk8K3Ye6HZSOF68h4nrrxkGp8syECxCPfKWT33MgeHiirq3tVEddqLcQ4zB4cWltYyWEMgEAhf/jkhdjL8O07MLS2/dsihyLJ1lt8g/YPqe3okXv8d6VQhkMYoxVKTOPevXsEq+OiRYuEdbFSpUrC4si8lIijRHWcffv2UaZMmURVHXBXsiAhB9yUIDZHUk/lypWFqxvKpSvii0UFZQfKBStNfNLHOAcsPCRO+7AOQGDhgGL5QcVc4kPFGzwUzm39a9itHvhAc5IKPvI4+ff+6wAtDb5inz426K+bF6UMyRMKt+DbP2wWcZFsTYHSgReWE3xgwRgb5hKGYgNlSD6JanGVT91wv8Aq2PvvA8L6ig8GhBXOb5oXFS4MJDCwUgV3IhTlgGG2D+LElkVp/IrjYsNlCxfGXXjoSqHMOUpW4XGtO36d5u++KDa3eHFji9M6lB189LGBQxGHaK1XiK9664fNEWIzte4x7QYIZeFu6PMIVhfZ/YjQASiXWsEmiwMFj4V/lzcWKLAjlxyNkHhSNGsKYa2E+xW/QzlGolCbwOwiIF+riEHRPD22gcCCFbChbxcUsWStSmcXCi4sC7AoQSGCFRYKv+yy1IsXRYxmubFrxfh5Y8AahVUByi9ccIgdg7LsLEOUlR4tRlBKX71+LQ5hTUtkpW9ahCumWA/+g21K45xOZaiClPCAv3EC0sD6BYSbTRaOP5Nj4GDZ7VYtHyVPFJfa/LQjgpKJe+d3KRch6x1tYJ3lTpuULt+D69qmzAErWPE2h9ykIllT0Mh3CtM7P9ridREvJoe/4L2b/m4p2nnmFv247pQ4RGATRjgLrh3WMHIGf/5By8V7A2sm3iMWWNzfDUuEOjm6HsWLY0tShBUbbcMTgG8FFKzSOVNR7FixhCWJ3ZL8fcE3AQkyyMwvN3aNsBCPalSY2pXNEUFhgCKBJApYAeFSX9+3qnB7I9ECSh17BXh8rKjLIQ3ydwFrskNYbCmUWFjbIEg+ung7VFj5nAms5Fi//B2buPqEODBoDwmFMienpd0r2ZuDaxUHSVau9frhkB78hkNwo2JZaEzjANEXPEzoCwcFVuBYqcO3NvjSPXsinKw07RxYgzafvEm9/kL8ZiwRluAoCe6DWbt042MFhvnSUu/a+alR2Drb3r8GZUxhi7tGLCM8KHiPEF+L7yes4zAu6Fnu8V63/3mnOAzgm5w6cXxhCYWhAFnuUMrfmbTF/t2S4z85LhLeFfac7BxQQyTc8bqHMQDvDowaentK5992R4jj5mcBa707CYLO1gx+98Xeb2QcVrgmxiuUVgDRl4tKjtdBtjU+Siwnrz0QLx8+Xmxt4sQbUPsM/few+Bhs/ry6cF0jNpJFtgQhHgxB5VfuPrbHg8k4g5oGHx9202BjQCA0CzZrxHzBOoCPCk6XsNTtG1JbKB97zt8RcZr8giMrtd8/B8WGBQVyxaGrwvIJpenwiLqi2QKDVwirIFMKsXsOv8HClCZJfKHMQaDoAQPEr6ENxABiI+QPpjM3pt6aggIFZZ6FcdReK5+y9ZQh3nwQdgAXVd6ByyLFgAEjuLVkyxesH0iggiWwaNaUQmEIff5C0KbAeotzBDY+ZCcXzBxO6cNK4+6zd0R2PvBzlLUJtxjcXFAwsenx4SRkdD2hvLLVTpsAoYdX6LMXVGjoSt34NlwPZQxxsiywtsBygVgzxJ5h7cDiaiT+iZU/bPQcg4o2YGlCuAgs/HD7T5LiqGBRw/ggHEMmz4PdZ9o4Sq2SDgV+yFuFIiQPIGliwuoTVDxbSrvbDRuwHCvGLmYoU6VzpRYhFLy5ysodrIJQsnBAhNJVZnSQcDfjPrjqkYgiWxfZ0gwr39uSC5jnxu+R9ncotx3C4gBPj6lvT2KRDzfcBhRGYIpNnsNHhiw+JOIqZQYAPgRyIpH2mfOBFMlECLOQD6LoS4735r61llWsVygsfevkt1NGgeUAYQJYwzjoOqMAc7SHIIa05oTwd56vA/YIC8DBydk653vwTYU1EeFFUHz1aH7w3ucOs5pDEa6WPz3VnrhBKFewVsJqCY8Vu3uROY3vfMlRq8V7pj28yvNi74z8N4QIYB/AYaFPHT8RY4+5HRlRR/cbEXTkmginwCGQvQpye4ipbTdjhxgve7CQkAg3NIdEfTR7tz20APfK39Fc/ZeKecB4wRnsUKx/WBNiP2BA6cZ3DuFZekl/sODCUo52ED6186wtnGhZ90oRvouOnrk7f1cKZThqSqF0ZwXp3OPLRYWNF/GCsNxoM4UxtLM3H1HzadtErB1bWGCJQrwNXBbr+z5SoMIAACAASURBVFYTH8KSI4OEkuYOxQt4AjmTEW5zRzRB20/fEvF0ELg+EWCOjwUSbuZ2sp1yOamCP1LLgq+IDxdcqyfC3K3sqsMmhLhBxEKywAICtx/ugavj2Mh6dOvRU2FlgiIEdwsULriAodjudeCCdbYUEPfD8T2OXLHYFAYtPkTX7j0RyQjazazJ5C3CYoasaCiUbEn55+NyVDJHeNA64u84hgvjkl33zsbpye84sCDAvUKeNOJwAsEGAxxhFYZorW2O+nNkuYG1s+rX60SIAwti+qAcYwOAsgy3M+IYaxfMIKxwUUnAsJXC5Y/YL/Dy4RljTcJty8lF2nYQv8kuPi326AvvCt4ZbRwuH35wDaySP7Qu4ZD7DklB+QfZrKCzOwZSpXzhWdUL9l4UliVZ+IAIpbXltO32zRDXYPNHnCBCVxAbiAMJ83wKl/mwVRGs1dv6V6dMKSKyCKAdWOnhYpZjH/H3NUevCVox8f34soF9WHhWsKpzCAsONjPeKy2wAUYcw8wJM3LMK7udOb5P/mZwB3DBbvq8mkgEw4EYsbssSHBDggjHwunRQuGQiHAdOYmJE4OQfIh4Xk8Eygm7uhGywKFBOACBb9eZJd7Vvv0GLhfPkRVnhAQgjADW6n+7VYzA0oD3EtRP/E2BWxyxrnoHRg4HQgwkuFdhZS6eI5X9EIGY+Rmbz4hEQXxrHAmvfyiMO/rXsDNIQGFuOnWbONziUANvQOPiWQUNEpRKxJAjQYmp4bBucGDmOHnZAwJlGjGwYJTAtxvUQ0xLBwMB1gruxTugRyuHmFYcsr5qWoSqjF8vvCuceOfq8zByvS/3fiPjic5rlEJpEvpWW1R8csfJHxaA6RtPCQoTxCqtCaPigcUAyTqgdsEJ2BXBB+SzefuFe/q3DoEOeRgRQM1Zpwu6lhdur2XBV0VXyHpEYsDcneeFu7Rf3fyC0JwtDOxulRMocJ8c74d/IxkJGYCweMkuFI5tQj950icV7lxHNDlG5v7fgcvCLQdB/Gij4rZkBFekw8yd4nQN5QEfd4410mYBwwrMFletdc2V/ty9VmsJgUIJvlGIUe5Kxh+UPCAoZ8FhCITHELhooQzCJQaLF9x/iHVEvDCslVAoZklMBLgHGwSUNWymUODz6Fh5ua9Bi4J124FrruQo23z04htxYIFlXY4BRZ/Vxq8XoRSVw7Ln2TWsh3MEi1NYpjJfJ/MI8t/kUAw5nhS/cxyio+cpJ3fJiWra6zlTX0sZhiSmzrP3iDCWU2PqR7iNY13xx6XdKwrSdb4e7yJibKt9vV58C+TkkFoTNojQmC/q+YuCB+xClhsHhy7iIyF4z2GZZ8ojHBQRZ8zJODJ1DrfByUxIRBvyto3In+MVtR4cd94FOeEGyhpnc+O/q+ZPbw8liipW2JV+A4auFPHbrPCXGRMkqHfg0VnRo7KIr+YxwHMA9gZOdkI/csIS94v3BQdyWDNl1gU51AjfzTM3H9mzyR2NWTZEyBRWyLCGVR4iH1ZAK4Z4TVgL531UlgoOWSmMGLDsIwyB4+S1IShoD4d3fPsQI6on2sx+vWuYQstZ0pIrz0h7rdX2fk/m4um9SqH0FMGw+622qOp/t0nEsXB2NX8YzTi1uwIZNgm4FpG0gvhHKH3axBm0BwsjMvGwGWrdby/Dklm0/XICEeKqELuG4G1ZYQQNUpMpW+zUGrjfGWl5VHOD0otkIFhr4H5yx5XGmzNcuR3K54zgvpKJf4Eb3HZw4cJd6o2qQVHNNUKs1oAagiQbFl+IUWspr7nsqRPTxn7V7N1xnBMsPjnSJBbuU8R2IvYPAmwRDzxzy9lIyQbABVYphDAguSxJgjj22Fk91zWTl8MFjmQCFvmgo5eBDTcz03KBLB0uZ8STgfEA1hnwe4Lj0Jk4yqpmSw/cd8AHcY1Q1OU4L7b6oQ/QXEXFMytv6HB1w6WtJ8VHrBLrF27FZmH8oriOaV4QI3xitC0JiwVcs4iLQ3w1K20yp6tIHPt2o3AzyhnzbN3jbxDHY+IbxFzZf3UpJyiFWIYuPmSPGebYWySUIIQFiVvMf8vXd/p1l6Crah2YjcY2KSIOGoilxrtqlGc3qmcIN27zqdvEM/q1QyBV+GqtiAsFNyYOhMwcoV1fztaFo985xnpc0yLUonQ2kZCFGGvm5JUVecQ2Q/BOQMmEaxlWXLyfcnyjzPG7+JMKIu4ZggNPwaErIjA3gGoKDAVRCR8U2Q2P72zjybYQKCRNftk0nGqLwxgwfiQCMosDnhe+14ifPjisjkjm5Bh4vMeLD1wWXig5flWOX8f41vep6vS7yEUcHMUUu/uc5PustvebMSd321AKpbvIae6z2qJqOGmzoBZiCwC7m8BjuSwKl4ZJcERohqmFEFPIHH+IeZoa5lJ9v0JOQXWTKoxIHIHm7X62JSkcH1WXXr0ie/ye3DDi2+AWhTsoSfy4Is4GSSzD3ylsvwzKKfjWmMTcVV5Es/HgbMuKedOKzFlYhWDlCRkdHrdmdp/utKfNJn3xEm7q9aIpoyTFsjXl4LDadsUB1ixYQ2C5Am04J3DxOBFAj0xn8BnCsgGlg0VWBBEzhs2RqW304qSYQJo3P24HLjNWkFHRxz9jxFKScmIHNjgkmcB6hgxeI4lC3I/foOVi7Wl5WJkODDHGcB1CwdK6KmGlRrgIYkP/7Fw2SmJ02aI8vGEhkWWsJ5z4wAoLX8OFDxC3iJARPYHiwmOE2xzucwgsVUwtJlva+RvECRkcxqF9pnJfsEyDRgcWYLyrzgQeA3gOOPNaJoh3poQ7a1vvd1bymTWAXe5wGQ9yUOrUlX44HpizojnmlWMNcdBCpr3MvoD28V4gxhsWfoQK/CzFtMveDm0oBBseeIzag4be2GUOS1j3e83bLxIvMUZYUaHUsrAlHodvHIzrf2+jT4LnDO8mFMYjI+pG4JbEgRKu8xFLbHR1EFQTQogB89tCuYTLHxbaqIRDXrSYuPJMnF1rtb3f2Xi9+btSKE1C12qLCidG0NZwbBG7mzgWx6RpG2qGT7ScUY7YSCgNUFLgYtOWypLjLg8NryMyDBEjJgsyARHcjqQLxMzB/QklgS0H8rVQUGHtixsnlkjYMbMEmCEApIv4OYDWBXGnSEaCdRZxnVaS0zceUvUw6g8oCbA+cFKS0aQmWHcCR9uqH8nExsx3B6UmU8qEwvrFwu5aJmmHFRObFou8NhAzWSpHansVGb1xyXjLXHTnbj0S8VUQJpHW4s/WPPSDJB6EjGC9rulVxallhNsqNGSFSKLRxiyCR3Tgwqi5VtEGXIygn3FWsQdE1KKa1b0nYlN3xJ2JsAXEVmsroHCSnpwI52w9Mj5IykMGMMYIyiEOA9B+gzgu1Zk1D+5PZC07UxYwPuavBRMDFFAUCQC9ECzKuwZCUXe/0pHe/PlAyDRqhcIsfGZYQ9Efxx0j9r1TpVwiSQeWX1CR7RxYkzj2VuaH5XFyQqDMjoHf1h67Rh/MsiUpISZdDtPQFq1guqKonj2eD2jn4JpnqyEOxVAYwc4gi+xtQlwoaMowju9bFRfhQ2wRl78V/3WrSLdDn9nLr6I9xNcjCx9eC4hRTxsbUrBfrOppoycyW6y295s9P1faUwqlK2hFca3VFhXHESF2CZagr1ceF1YfJos1adqGmmHlgC/WJjpoG5EzS5HJiBggjneDIgYaDVg9ULmGLZmcleworgYxc5A0SW0kuNElzMWHpCSQCSPz0+wSk2bMTY6vggUPyj/HdIKUGhRSRqTUqCBh1eN4q4hcnCVEYgmuYeHqTezC1VK0yET/cKWWzZ3GHjZwABm9mprqTICv3YBkrjxHrjN+h2D1hlsVBxZXy0kikQlWI4R7cAlLzBUFA5Bsgvjerf1tlYg8FfQDxRLKlCMpO2aN4GVkKh++DuTsPeahjnQ4kb2z8bAFEpnBiIvjhD++r8XUbSKxCDHOYB9gd7ZehrCzvhz9Llv8f+9URhwwsUY8iZWOaiwo8YlSn2CVQGUycFVC9Kzc7sxJDhPoXDkPwcINYZYMrgAkc/RGeoYJ4lLwcFsxAghzEGPMezQHVy5EwdcicZI9RVGNH9RyKLYAwUECMeV6rAJwh789yca+wRyrsDaObhRAncIy8cEqgLhkOaQmeaJ49n/jXliAEe6ANQrR0kc5Giuv66iy1915TvI9Vtv7PZ2PJ/crhdIT9KR7rbaoOHaEK9pAcUF2OLL7/u4SXj3GpOlH2Yw2o9VZfWa4O5h7D5UQYKHkxB5QlsBaBkoWbPBsZeIBIJ7LSGybL+at14dsYUDSANy6zhTs6BirHHeF2D5gzqXwjG46GDe4DUGDw1nxMl0Pc+1x4gGuZ+WDKWZk4mP8zkH++G+ESUA54QQfJJNoLXlstdEqO3J9bM7Y1eLcf8FBEecll6k0Gj/KbTki3tbSePnqGWtdqtwvihuA+9UVEuhP5uyNwFerLeXK3yDEGsJzwATeWpoxT+bOla2Yk1DrAvekbb175YpCqM8OtyysbIdH1Ilg+XO3X3aho6Y5OBc5rpAtx1EdRNjKzG5kHgMfzjixRx4bhzrgb1rLZlRz2Bpyk9rM2CFYNcB0gOQpPZGrLUHJR5wnvA4IxZKTi+RveVCvyoKfFbH3OKhAkGyE7yR7TYzGwsvGCY6FdvfZOLrPanu/2fNzpT2lULqCVhTXWm1RcawP10IevfSIIIONquygSVBEagZB/Y3D6vLiR23ZN+0NiKPiEoWI+YGFsuJXtjq6K3tUtnP+wRoDombm3MTvcpUPb83Hk3bZ/YQ2QLaNUz6XVfOkXbPvlcsgglYHWd5M/4QwBDlOKqq++SBTOEtyWvJpJQLXJQjNIZwMw5nv+Bu7Dh0pgp1+3U1BR20l8hoVyyxqPGs5S+XxONqA5TUmEznL92ord7ij+DuyCP6y+YxQRrQJS2Y/R217nPUqZ+jiGs4U1rNiORqTtm63nGmNe/gbxAkR7J42QgVlFAeugc7KEq8lIyU7jfYhX4dwAWY74GosURVtcLUPztaHMon/sfWeXcOO3gv0w8qhNr6SmQ4Ql/rbB4ERhiQfrLThJc7GDnc2aH9ANeRI5OxtrC3Q/9QPyCi8M3JlJmT2s7WXY19l7kwkE8E7FThmjfB4GKUBkg/GMEYUz+4am4kzDPC71fZ+I2P21jVKoTQJWastqvdn7hQ1Y5H4MeKdwjT8v8Mi/gTJIHAN+VJk5QT9OjspyiTiiIuDdQyJHBBtQghbXPAbVwHy5dxc7Us+McNajJKBRsoLutqPp9cjdg8UHxBYEqHUc6wj4rAQB2tE2OWETQ7KPkioua4xxzxy9i/aYyWTFUEtBQ42HdQ2h2CDhHISVd13Ry5C2QruaD3K8V/oTxsHaWT+rMDJlVxwHysGMs2VkfY8vabK+HWi1r021lguzQqvgBHh6jh8rZaPlquzsHLHnIRQKCa3DaeRMtKXo2uYDxNsARv6VrOXKmRLtydt692LkA2EMYD8H4IQgd87lrFnTnvaHxPqi4S9yrntB2m0e2ZsfZqy4ZSoG6613ON3FITo8vseEX8aIlE/MasCsvqRdCMLsuKR+INDuR4tk6fzwf1MhcRtwX2NmFcus4p4eiiUcPdDtnxRXbB9MD8x/sacm+ABRZIRkqKcxRXjPjwvVMQCi4K23rsZc0MbVtv7zZqXO+0ohdId1HTusdqiYjoN5mLjShbeii1yBiMH8BvZQLUJIVBmmPBYm3ghk42jigLimqwsUIb4JI5YN5y2vbX5eYIDeEbzDrTFb4FzD8+Aia+xsRlNdmDXGG8KqKXOdDyoigQ3G9cMxkYI6yeStPQUHIzBf/ByOw0ULBaonYxsUb0NFn3aYwM1cWVyNjDidPUSteQ4UlhXtvavHqE+sxF8OaNdW8KReWF9HT/LZOPa8RhNEpLnDPclFCAWKFYV86W1/xuJcOCUZSqZz/7cJyiimKTdCH7OrpGz5aEIcxlOuXKOszZc/Z1jaxEegDlrqYxcbU++npU/MFh0rpI7Qk1sMF4gFARE39rYYrQhVwg6E0YphL8jfAgHKEeUQJwI5C3Mqo5fJ+q0s+Awg1h+OYQGCiXHWqLcYvrkCYnjn7XhKq7iy2Tq3pqf1fZ+V/Ex83qlUJqEptUWFZe44tJxzJfmTfqEqKBsOmWroHzQOyVr75M38lU9K4uEEKab0NJeoOoOqCkgVrT0aecGBVJOQsHvTPxs0lI0pRmc7DnDFCEKiGNFhqYeT2FUHcp8hYhVPHrlgb0s3qkwqiTQ6oxaekRski1LZxfN6cX0nb8VSpXH20IfILBigC4GFX0cuR31qi7hXpk43JELHzx9HMfF5QFdBZdL5zFpP9/vKEbU1fZdvZ43V22VEUcW4ajalw9+uI4tS3xP1zl7RBEDrqPOMZdNSmShCS2KuTp03eu5RjZCMPAc2WMRFXWSpx3Dy/DnzguixrufAS5SV/rjGFCEcyAph797aAPzm7I+RNRW10uulAsGyOUzy49dI5JeHGGCg8HSg5epX13/KBO6XJmHfC1/++1rv20JQQFkVyAH1hAWSm2MNhIpcVisF5BJlKF0V9hbx1yl7rbj6D6r7f1mz8+V9pRC6QpaUVxrtUWl/Xh7I37JFejgPkTc2KAGBSPUPdZr4+q9J1R2rI1uBpU5oFDKSTrg7mORsxS9dQJ1ZZ7OrkXcZ74wyx9fq+UEdNaGr35HfCpcRT+2KUEvXr0SsYq8cRsdg5ZAHDFbSP5gQmNH7eglGGgtYsjcbFoyi6iE48jyjvrDyCaFyJZV2XIaVdwtLGAo44asaHfoppjnr1ctP+peI599umx90UuUMIqtO9c5UnCZtB3sA5v6VTfUtBwfh5CGoyPq2muAowGmpGGOSLa+mXnwk4m+kZQF4my9rHpDE7LARSg5iNKaKFUIl7ccew5qMSiUiIXX88bI78fJ0TZ6IBwMkSmOGOioan17c+r83LkPxEMiZEZ2cV+8HRrBBS4T/Hs6NmTlIzvfW+FeVtv7PcXLk/uVQukJetK9VltU7F7ijznTPJgZv2QSdJGaka14+PhAmWk6ZZu4TksNI2eQy6XrvDU2M9plbkJuy1mSkhl9utMG18ie2LKoUOr7zj9opy8x2p5czxoJAXD5IxnFmeLClVuQ+HBytK0UICs9ct+op73y8DVRom1SmxKRhrXp5A177CdchuB0hMibL5ewMzonV657Z9JmUQFIm5XK/HjOaie70peRa9klzNVr+B6ORXTVvcgZ+nq8gCC8XrDvEqEqFA4lHWftojXHrouyo2MaBxgZrtNr5CIIOBjAouzt2s1OB+XBBZxAg4x5WChb/7Td3hrouiavCxGVhPSol7RYIHREJujXq1nvwVAN38pUTnzDzoE16P7j51Rzgq1uO5JwYKGUk3S03MSGO9O5kMNLOM7Wk7b07rXa3m/2/FxpTymUrqAVxbVWW1QcAM8fcw5wjqosm0lQeNzMvdDnVHSEjch8fpdytpJe020f1sPD61ASqRLDrrO3RWk0CAhxA7Km8Lh/bzfALijuB3MEH6PVRKa8efmKRJk5cMjtGGAsaYPnI5N7o0KOoxgwef5yNjxbFpnHELFrKEcHgYXv2NUHIkgfZN1akV3bCP5ny4de+97An919Wpc5W9Y5+90bfeu1yVWretb0o89qhltMmRDc1ZjOZlO20u5zd0hbrx19c5USKP3T2peyU0hpq1l5Mnc5FhYhFcwGwZRUnrQdHffK5O+wUDK1DsaC+HGUMwWVlV7okkxpxN9JueKV0YIEZs9b5iHGAfH4yHp04U5ohMICF++E2ueqR//lyZg4RhsZ6QiZMhr/bbRPq+39RsftjeuUQmkSqlZbVPwx5w89V6txpWycSdC43IzMVYjqC7BQcoaxbGVCw4+fvRQVQvChQklJtkC53KkPb2DyYu7SaClDHw5RdFVu7BpRtxhl4GD1QXk9LgHnylg4rg1UNeduhwq2AWfVUmTLImeVM6clV2bBGLjWryNuU5myas+gmnZiey4fp82IdWVeRq7lpDHmg+V7Jqw+QeAH9HXlqrd/2GwvDNCzlp99CuCoBcWTo+QmR3NFtjruQ/Zwo+JZIlzGlilWftrO2E5bQm4J/lCuCW4Ew6iukWN0F3Qtb4/DiymHS+3cmFcTpUKhUKJsrPydmLzulKAag0t8SruImfJy/XkudQoOWLw3kOiiVJM9C0yTpS19euH2Y3tstZxQ5On6wP0IzcD3SzZEmNEut2G1vd/MubnallIoXUXMwfVWW1T8MWeOw49/30PLD101lBRjEiRuNyO7SWd3DBTuViavloPNuQNkJOPUaYRGwu1BmXgjc81xk4iNQvlFqwlTzAxqUECUf0NJO1ctWJiTrMTAtYUNUc+iJc9fLrHI1pZK49YSNh7Eyo5feTwCXFoXLv945PJ9e2LD1i+qU+aUicRPnKyj5ewz+xm0m7GDNofcJOZi5PYdlZY0u39te5zxC+Ls3rXz23/2JKYT76veQU7LLIHsfig9zgobuIKBXEJzaruSgjYHsq5PVQKjREwTJiGH5Roub8ShsoBUHwlocu1yeX56zAVcS9tZzLI3ccJ4kWwE4dhPmR8SVGFIxPx4jutJf94ct9G2rbb3Gx23N65TCqVJqFptUXEgMlev0FJ4mDRtrzSD7FpkGENmvl9alJPrPHuPUBjhDonp0mX2Hlpx+KqYBkpGhoyOXOHFCnOsNWEDnbz+kD6v6y+yvKHEuZNEwgTX4Na7fPeJICZn9gFH85Q3R1TmSZwgjuCTg2KLms095+2nh09tXICQEe8UonfL5YzUnEzTJJdYdKd2tTvPhDNM25XNTqMahbvkZUsU6if7SppM3kJ7z9+lrlXziKxeFo7pLJgpubD0myEj/jtCv2w5QxXypqE5ncqSo7496UuuAY01MGTxYdGcM65bT/r05r3TNpyiscuPibrVsFD2m3/Q3h3iyeHyRsywHlsGOBpRaUeeP64HbyXaQ8366JCtp25Sm592iK6Z61KO7YRlGQdNKM+u1JKPjrno9Wm1vT86cVEKpUnoW21RcY1bDt7WkgybNG2vNZO7/1JBtouEFWQofjJ3L3nbmuS1yWga/uKfg/Tnrgvir66UuvPV+LgfOd4OCuV3brpoOUEM8bvI+oaVSusC1s5NrgEMXjpk7jIXKTbGDrN2CmslCxKHGhfPGgkiLQUV07xwMpcr5ebcwZ8r+zAXI7fBVWZ8XQqVORS7VMkj6KpYvg06IWJbwe3536cV3ZlqpHu4TGZgrtT010flyJ6gVCMfIevdDMGhovBQGwE/14rGf0eXe9fTOcnZ9rBQwjDAglhrWJLXHb+hm9ikfWfA5cj7ACv1no7PnfvlQhXMhXn/yXMqMswWJ4+1gfcU7A/efh/dGb+ze6y29zsbrzd/VwqlSehabVGNWnKEZmw+Q+Vyp6E/Opf1SkC8SdDpNsOUNZPblhAWSlDWoKZy8PA63uzWJ22PXXaUpm08LfrKnTYJre1T1Sf9utpJ48lbaN95GyEylHtwJ7pTfpA3NdB2IIMfSTSOXNQ8Rm2t7VM3HtlJtKEsIDYR2dMsM94tRTULZog0Rdm1JsfVuVNq0FX8cD2HmjAXI7fBypavyfjZ7Qzr14D6BexTmrDqOH2/NkRUfIElzAzRWmGZQql3LT/6VKJQ8qQvxMblCfNmgI7or90XRTw14m7NTr7wZJxG72Wi9vTJEggL5ailR+23zu1URri89UIocJFeeAdzgUZn7Dz4JEuOChLzAP1Wu7I5RFwjPA4QzAtJOp//E0xpk8an3YNqGYXLEtdZbe+PTlCUQmkS+lZbVFxDOTBnavqrSznyRkC8SdDpNgOrA6wPKHcHCyUIzFMljkf7htT2Zrc+aZvdUOjMzDrAZg+elQ9YE6FQglqGDyiu9MUZzSAuv/PomSBZduSi5nZl0mzEw8HKgQQFDnuQ63/jnnmdy1KZ3GkiDUtbk7xkDls2/dwd593OWndl7kxUzfRdfC9n87qDpyv9a69tPX07bTt9izpWzCXKL7Jw+Usz1yMnHhXNmoIWd6tIdSZuFKU3tSTvnswH9/Lhs2r+dKLcbEz+Tizcd5F6zrNZ6qBQyrHCqFiFpJydZ/XjUOVDGGd0cxa+mXGrrj4vKP3Fhq+iB09fiDKu5fOkJbkS168fBAoLJUqoxoTyudr5W23vd/X5mHm9UihNQtNqi+rrlcdp0roQu8LC2aZay4RJ0ze9mWIjVglqmG+aF6VnL18RkoxAaG60zrDpAzKxQS5zhyZrFkhPM94rbWLr5jXFsY9IKIHL+7dt5xwSiEfVK1dhyZwioXBdP3r20mldbK2rGtm8sFJzjBXTYnG/y7pXooKZk0cahrYmOTYziDuVYdxB1s7FGJCJfmwbzpM5/L/DItvdW2TLjsbqKEmID6BmuuA5wYTjMrnEn7bsozu4yvdwWVdwemKdcCaxp+1Gx/1yshiUQFiNWRD+8+P6U6KMojapCteEXH8QgdsxR5ok5Kj0p6/nBo7MUzceEuKoYTkG4Xqu/rY4eXgXYKEc/p9zflpfj9tIf1bb+42M2VvXKIXSJGSttqgmrj4hYt7YOsAnVXdLyJkEk+FmUJ4Q7tEvmwQIl/fgxYdFqT2Ud4vpsvTgFRETCmleMiuNb17UklNiImpwPGIDQNwnswa4MmDO8kR1DJRZhCDZqpq/43Jq2go7R67cFwkKyIZHVjyHdPA4HHHsyZaQmR1K20u4oWoTCNa9RXbM42L6rtoFM9D0d0vZYWNOTT2CalewdfVapl7SckF6wwXPlngmPa88bh2dvx1KQ98uSB0q5HJ16A6vZ1oquEtvPnxGZiYWmTZIgw2tOXqNOv5qq+yEw//0sNAY/BuFG1B2EUqztvISfgfHKxRICFORMTbOPAIGh2fqZfkGLhPep6ntSoh4aMEiYeEQIEeTt9reb+pDcrExpVC6CFhMWVSc1CdPygAAIABJREFUtQk349LulUT2H7IA9U62JkFgajN2DsRGhen5i1c+2fxNnUAUjaGUH1eFsLLFWI7/e02vacHeS9RAY2kzgpk8X74emc3g2nMkcI0XH7la/Iws0MOX7olDBaycW/vXEMkJsjtQW0FJbhcxdtrqKe4SeRuZr3wNyODhXgd35i/vh1uiuSIK3LSzOgS62qzb1zvKOpcJted+WNbt9uUbuUIJU02xcgNe0/Y6GfnudlpzwgZRgQmMCWAB4CQgd9uLzvvkajc4yCGmkgWJZ5PWhhDiiZFQhcQqWeRa96t7VqZ8GZLZuWRHNy5MbcvkiM6pReq74JAVFPrsJf3QuriwUCIb3S9DUlrVM3qy0d0FRymU4cgphdLdVaS5z2qLSmsdYC7AHjXzUY+a5mRYmgSdbjPMOQhrBqxaoNLIky4JreltzQQWV7A4dOkevfXDZnELKHlgNbaicC1mZGfDQrnk4BVyJ7j/8OV71OB723xZgnpVprzpkzmctpy9i/jI4Ev3RIICuAURU/nnzvP0xYJg+/1RVdcoMHgFPX5u27gwF4j2/fAW/o4skaxo6lU88dZY0K5sdZYrC2lZIcwYg9YKHDg6iK4/eCoqGkFZMksaTtpMB6UELV9jatY80M7us7epWVjlL1Q5gzeD5aumAcJCCSvvkLcK0gcVI1p5tWTh/hmTU+nRQXTjgc3T08pEzM2Yc5FhK+n+kxcERfn8rcc0MegEsQHEjPZ91YbV9n5fzVuvH6VQmoS+1RYVWwdYCePqLM6ya02Cw+Nmqn+9nk7ffESIt0IMpbsciB4PxAsNyPGBVvzQ85SRCAViZJCQI4Zy1ZFrhEzacc1cc9FfufeYyo1dGwFJ1PNNnyyhQ3RlcvvfO5ah/Rfu0NerTth5MLnSDRpwxl3H8bio5gL+Pog3eBf1JuMo+Yapo9wJIfBkSTIfrZYHVEtC7kkffK82TrXEyNWitvS4ZkWoRalsZnQh2uD4cG6wUbHM9G2r4qa178uGZOofhENsOnnT3v3IRoXpx7UhdPX+E3u2tDy2q/eeUNmxa8SfQBZeOEsKYszHNytCzU3E3AxMSo5cTbfC1gO+iT+YzDJgxhiNtGG1vd/ImL11jVIoTULWaouKrQM50ySm9X2rUe2JG+jENRtJtVUtYvKj4PEiIxQWSrM58kx67G41I3OwTWtfkuoUyuhWO96+SXbXwkIJ/jstQbeRMcgUIXy9M55ALbn93vN3Imw4cg131OjdPqCGw6GwZWxM4wDB3wfRZiAbmYc71zjim9TWuXanbXfuYVJ9bezuwIXBNEfHNe9OH3wPZ9KDAmfnwJokW6T0OEPd7Uub8e/OGnW3b7PvkzkbUZZTtrwiKx+hHkIpb1qEWpSOqJTLJO//dqtARbKm9BrmZsy7zJggunb/KcEdjxhKlP90h5bMjLF40obV9n5P5uLpvUqh9BTBsPuttqh+23ZWVI3g2sveyrA0Cb5IzTCpNoLPYa2Cq6d49pS0sKs5HHneGreRdqGcNZ68lU5dfyiC50FAbEWR+SMRQ4k6zB0q5KShbxdyebjsdsaNRgnq8w5YRi9eIWi/JO05d5t+2nTGXroNGaM1vtkgxuGs/nTFr9aKShyoJf5+WDKIN2hy9EBhtoVi2VLSIonfsfdfBxzWZHYZXBduYF7CJiWy0IQWxex39l9wkP7Y6V7SlaPutVyfcswchx64MHSHl34yZy8tDQ53DWtJ283ow1dtyKUkYQw4eyvU3jWMAVAoEQ7ybctikWqna6vPlMieSpC+4/rvWxenhmHhHr6ai7N+OKZ2eMNCwo3vLi2Zs368/bvV9n5vzzeq9pVCaRL6VltUTE3DvF5cl1kv9sYkCExthmsOd6+el56+eCWIwGNysL0WHCSJQFFOHD+uqbiZ2Zi90klOG3ej4L+rkpv61wsnxDbaX/mxawT/JMQo/ZOsgMAiCdqiyn7p6LcPAgWfJSftOONOrPHNepHI0L+eP30UlsjA5PLeXlNcgQa1mZd8Gl7S0E4nVCQT/dgmnE7IKJ7uXtdt7l4RC6t1C3vDYsqcilxH2m/QcuFtmNK2BNULyOTuFCLdx6EZ/ANqvX9SLa9p7fuyIZndAHyad0Kf27vH4RpUcMAQBR/qazC8F/qcio6wVZ9BVZ1SOVOT/+Dl9OT5K7HGEJNpJWFKo0ENCggeyl+3nSNfsx6YgYfV9n4z5uRuG0qhdBc5zX1WW1Tzdp0XlQd487YyfYTeI5Bpjp4+fxWhJrBJj0w14wQBtq6hegoE/HdQ8HvVzu8ydmxxxo1G6woXHb5K8FZOaFGUtp++JaqgMP0OXOJ5By4ThOtV/NIRyJEdCccPyxVavJHVrNc/Z6MzdQ5fI5ejRLKQr8RRv2wxrR+QkSa3LWnKcLheeuL4cejIiLrE5VRRi72WTlUjdzvl+E++34oUOUbnJhPxx45FYn2zyKUl9SpDaRPZQPQfTs1TkuoWtlZoTa0JG+jk9YciYx0WSj02BKO4Red1Vtv7oxMLpVCahL7VFhWSKXByZ96+smPWiGBuK9JH6D2CVtO30fbTtprPOGHP3u4eqbZJj/f/shmZmBr7Gvjv3C2bxyTpANKZRZHBZi5SZLduPXWLFu+/LLK0WQHjoP63imSiSVFY+djaLVNmcfY1Wzy99YCnbTglGAqYOseuHDiwFHprHNxuz3n7aeG+S8JaJVtG+e/OsHRlfMuDr9DHc/YS+EePQaEMK5Eo84G60p6jazlOlX/HAaRJich13c3oy9ttPH72kgoMsZUk1ArCTUCGD4GVHmtXFvlelDMsnzetXYl3VJrU2/OJqn05UfTcrVD6e89FqlMoA01rH87XGp3jM9q31fZ+o+P2xnVKoTQJVastqkX7LlGPefspWcK4FDysDsmbc8vS5lF2mARfpGbkKi1wDZsd3+Wtcb9J7SJIHhVUYFGEgOtPdhu7MlcuQYh7quVPRzMNcC+ymxzZrZtP3qCVh69FIIJn/kFQ0MgUONpxcQlJufycN5JQ9PDg5Dht9RaO+9PGMrqCqTvXOordZIoobYlId/rge1YdvkqdZ+8hWNqOjaxHcHlDZncMpEr5IipDnvTDpT25DVSUqW3RRDdn85ST0SKt47Ba5fj7Hx+WpXJ5IpYalZkRgHGFPGm9psQ7m4eR35nuqXuNfMLlrXfQMdJOdF9jtb0/OvFQCqVJ6FttUS05eJm6zd1H7G7i8mQydYpJU/dKM5y52bZMdhFDCYtrvcIZaUo7c9xxXhn0G9aorAxhao7474xMmy2CuNao0sJxv8hu3XTyhqjTLGfwctUXZ9yq8uFkWENbQtHn8w/SvN0XhOsVLlhviVx2EoTsLJxtDRojvJO+EkeVe1C5CZyHTYpnoQktw5N1PBnXumPXqcOsXaKJQ8PriAQRR8qQJ/0wST23oadsedK+r+/1G7hcUKVpBeEIy4Kvij/rFQZAXDZI/CGo+10hb1rKN9CmxOtZNH09L21/TadspT3n7lDXqnnEt0Uvtje6x2ikf6vt/UbG7K1rlEJpErJWW1QrDl2hLr/b3E0nRtWjgGEr6cET/exAkyAwtRmZL+/Ji5eR3J2mdqYa00UAYQaDFx0i0PKgCsmVe/r8d0bg41KguBb1fEe8U9jpbXKM1frj10UIRKeKuWjQWwXFvcevPiAcnFBrPE3SBA7b0yPz9kbMoN4AUOlErw595992C15PLR+kU1A8vMBRNjdXRTJTwcUhoP3PO8WIdw+qKbwkkL+7lKPSYYleHk5H3P779nM0aNEhe1PMwWhG29HRRsDQlfTg6YtIXcPFvfHEDfF3vTnK9bF/fq+UUCj9B9vc5+wCj475OOqT+UNRLQzZ7VoPhJXGGtVYrLb3RyduSqE0CX2rLarVR64RlLI4sWMRqoh4i7LDJPgiNSPTmyApB7QgvnYPemtuMaXdv3ZdoH7/HKQ0SeKLITMJsTuk1L9uPUtD/z0s2kFyQZ86zhN76n+3iVDDG3Gba49fp33n79In1fJQ3zr+LkHIa6lpiaz0TQubNdBXSTF/775AfecfJGTs7htS2z7uTr/uoqCj16l1YDYa26SIS/Px5GJHpSC9oeBuPXWT2vy0Qwx38+fVqOJX68R/L+xanopnd1x209X5Ldh7kXr9dcB+2/o+VSln2iSuNmOZ60uNWi1qkmsFpUph0YOs6lmZ/DJErjTFZUbh9odCWSjMKoxqU0jSsZK0m7GDNofcFFRkKBu55hjeh6jDV6w0fh6L1fb+6MRIKZQmoW+1RSW7m86MrU/5B6/wCmWHSfBFaoY3fHCngRg7Oqw53ppbTGnXHoebwEZtBKuJHv+dkfks3n+JPvtzv7gU1Y8+rJzb6W0yddTqo9fdTgrq8ec+WrT/MskJJ95w8epNSMYweHgd+yUfzNpFa49dJ4R0jG4c4BQLsy6AxVkvwS1cwTVvQ5fJ54N6VSHEvELMtiCyN4Yx2jOoZpQWa7Ow9FY7zMjB7SeKF0eUDgVTwPFrD8SfUX4UZUi1wu5yUDMhKQdMCRCmEfLWmN1pV64rf/72Y2F9Neq9cKc/b91jtb3fW/M00q5SKI2gZOAaqy0q8XL+YnM3hYyuJxRKxNjElIB12SWJ7EVUafH15mvgsb/RlywLvkJdw7J0YxGJWFY9/jsjIMjuT2RtG0kMaz51K+06e4dAVL3qyFU6feMRDajvT50ru1b7XC9u8KPZu33iYuNY5oTxYovEFJb3ftlJG07coPZlcxCSjnwljmp2c8yymVVm9p2/Iwj8WYnk+vXLP6tEBTIlN23KwBF4shwfVZcSxI1jWvu+boiLUHC/qDSEGuhZUiYi1OuGbPmiuvi3Vph3clKb4lQ+T1pRehFitlXYDEw6/bqbgo5eo1alsxGyvLedvkUdK+YixEzHJLHa3h+d2CmF0iT0rbaotobcpDYzbO4mlLnjWJpf3i9F1f0zmDRr7zXDtY6RNAELJWraIlaOkyq817NqmRHgsAn8GzGUr1+TSGBxh0Pw0KV7xArF1HYlqG5h5yTLbX7aLuiCPqiQi1C7G5upOxyDbJWrmj8dzQrLLveGRU5v5aw4dJW6/L6H4saORSFj6tsv0UsU8sXKG/HfEV1OV05weq9cDhpuIL7VyFjlutT/fFyOmk7ZJm4L6lWZ8qaP7K410qbeNbIlNH6c2HRidLji7m6b0Xkf0+nwGEA5hcNUysTx6G4Y0fmugTUFx7BWuDLOd62KiSzwwNG22t7/datIAVlTROe0IvUth6Kcv/1IHB7dLZwQnROz2t4fnVgohdIk9K22qHacvkUtp28XszswpLa9ggIIoEEEbXUZtCiYft9+XlDMwN3DnJQDG8Ss06vVcY5qfFrLD651d/1AGYQrD2I0QYBdYrDiLT90RcSVGbVuyvNiEvNyudPQH53Lip98ZSFcc/Qadfx1t+gToSexoJkTUdsZ20UpSyjLQ9723ZoeteQIzdh8hsrmTk1/di5nh0mOZ3OntKbeOgJvKZQjyO8dy1C7n20HXLNjHOXDCvPuxuT3rtGPW2j/hbv2KaCwAIoK4FCCUqTimz60NqVIFC/SNLleOrg4YaEsO9amUC7rXokKZjbPKmwGvjJVFSyUmLPR+Goz+jerDavt/WbNy512lELpDmo691htUSF4G7QMkJ0DalDgGNuHZU6nMiJY2+oi15GGQon5fFw1D6GerRLfILDt1C1q/ZPtUMLiLiXLi5evqNbEjXTjwVPa2K+aINx3JpzpD5cYKEVQCQSWl3eKZXF2a4Tfv1pxjKasP0VIagDdCsQbCpTeoGSl/OToehQvTmxxGRP3y1nrLk3KzYvtJSdzpqa/uoQrlK2nbxcuRzPHc/LaA/HMISDW7vSbTbHe1K8aZUud2M0ZRL7t9I2HVD2srnuONIlpQ99qprUdHQ1x9jP3jXKE8NDIcnREXUoUP7Jbn+nhxjcrIiyUnAi1skdlyp/RPKuwGbj0+ms/Ldh7iRoEZKKztx7R4cv3qWdNP/qsZj4zmvdZG1bb+302cZ2OlEJpEvpWW1Q40SKpgT/glcbZMiz/7FyWylos20/vEYxeeoR+2nSGyuRKLVzeBy7eIxDgop6tEt8gIB9KuEe4LkvmsNX2dlXwHMGvlzxhZMuKXlsy+fd/By7T85evyai7XG6P62kHZElB/31aUfwkV2LyptVbDj2RlQA9snVX8XTnej3lGu14Yzxnbj4i1GuGIKYPvLiQ7f1rUMYUCd0Zvu49V+89sVviCmVOTku7h9dMN60THzbE4Qfcpcw/yX8DcwcYPLQiF7DAd77KeBv+SIriAgU+nEqUXclcsMjyRsJRTKzDbrW9Pzqfr1IoTULfaotKdgMhZqnmBJulwIrZfnqPQN74kJTD9DGf1ohZp1eTlle0NCPHwPEAfBmLxdnZsGCANgriTtm+yetDaNyK4+SXISmt6llFtCPXivem1XvnmdtCWYMcHFbbrkxz/0g4Qi1jXwnXZy+ePSUt7FrB3q038ED1Ez7IwmIG+iSIo/g/dzFAvXfOZsYBdN5H4ZZXd9uMzvuYwonHAK5SkPCzMBWc3hjLjAmia/efihK7UChrhFluzQ4zMAMfOazp3O1Qt5PuzBiLJ21Ybe/3ZC6e3qsUSk8RDLvfaovq2NX7VPdbW/wSaDo4IcKK2X56j2DCquP0/doQKpo1BYU+e0knrz8U7m64vZX4BgEQh9f51nYQYVnRoxL5Z/RNLBZnZ8suP3dc7jM2naZRS49SzjSJaX2YO5Tj1OT63t5Ade/5O9QkLNN57+Badld/k8lbaK+bvJqejFN+rxZ3s1lrIY0nbxE8n2bGsF2595jKjbXFzSKTHclRkP1DalHKxM5DHozO8/nLV/aKMDULZKAZ73mv8pHRMXlynVymFO0g8/nnzWfsTYJG6OjIurpdMOXQyHcKCYWSQw7MDjPwZH58r8w4AJf3hduPaejbBalDhVxmNO+zNqy29/ts4jodxXiF8vbt2zRgwABasmQJ3b9/n4oUKUJffvklVaxo+1h26NCBZs2aFWHqWbJkISwCyKtXr2j48OE0Y8YMunPnjrhv8uTJlDdvXpeei9UWVcj1B3arpJxh6UsLk0sAai7+Yc1J+mb1CSqYKblIyoH7bFCDAtSpknP+Qk/6VfeGIyC7LPmva3tXodzpbLW9vS1cb7tYtpT2JIUFXctTCRdJsbniT+YUCYnLH771wyY6dMn7MVuylRexzOmT21y9zLHpbYVW+4zY/V84S3Ja8mm4a/idSZtNDytBvGzp0bbqOHh3odRDUIYxaRi3qVlrKN/AZSIkonHxLDTRpNKRZo3N1Xb6/H1AlJplQWnRb4NO2v+NZBwk5ehJpXFr7YoZFEpOitr6RXXKrEMz5OrYzLyew5qQIIakHFTiwsEDSXgxSay290cndjFeoaxduzZdv36dJk2aROnSpaMff/xRKId79+4lf39/KlmyJNWtW5c+/fRTO85x4sQR10KgTEKBnDlzJkHR7NevH50+fZoOHz5M8eMbP0VbbVHJygCyaplCyIrZfnovAJIo4PaGmxIWyot3HtPwhoXovfI5o/N9+b/qW87M5omj4knWVOYlVEQFKFsw8qRLQqduPBKXurN+5Yo/ewbXEu3U/XYjHbvq/ZgtOdNZ5g58+4fNFHzpHn1WIx/19GFcMB/UwAMJPkgWbyjYd0OfUbERNh7EPrX96OtVJ8R/g8YsYTxzeSI5u9nXvJ7e+CDwQYrbBi8jmApY0iZNIEpZ6knV8evo7K1QocBDoWTPlHyY8caY3WlTDmtC6UWwOHzZJIBaBWZ3p7lou8dqe3+0AQF6udcoABpDJSQkhPLly0dbtmyh8uVt2ZuYjp+fH7Vu3ZqGDh1KSZMmpT/++IMaNWoUaZbPnj2jtGnT0rhx46hLly7i97t371LmzJnpl19+oVatWhlGxmqLSo5fmvl+aeowa5eYi6OSXYYn6qML2U2ZO20SevTshT0uqG2ZmHV69RFcXulGtjBxB77cmMYsO0rTN56mDMkTiOcPWdO7CuVx0UKqV60GVVtCrj+k/vX86aMq3gujkD0FG/pWpRxpbNVNGny/SWS1IskMyWa+kh/XhdD4lcfJP2MyWtGjsr1b5j6E4tetujnjefDkOQUMs1VqgSt90roQ8d8otBA3LNvdrHmXHbOGrt5/8kYwQTDNFWMzrlkR6hcWf4q/gdAchxM9qfHNenH4QgGAwFxpCKEdECtWD5LDL6AEIxb2m+ZFqWnJrGYtC5+0Y7W93yeTdtBJjFYoofxt3bqVatSoQQkShJO8wl0Nq2S3bt2oQIECdPz4caFkamXnzp1UpkyZSL/D7Q3XOSyXRsVqi0qOX0JmbJff97q9IRvFwMzrZm05Q8P+O0LZUiei0KcvPaojbea4/p/auhf63M5fyvM2O/4tKjzHrzxGP647RYnjxxFWaoijCiFRtbM8+Ap9HFbx58QoG+k1so99EUZx9uYjqhqW6Sxn2rKF1EwFzsjaZMt/vvRJaXUvW4ISpM7EjSLLtl/d/NS1qmvhPo76RVY/F1SQ4wBlPk4jYzZyTa0JG0ScNRKckOgUk2XcimM0ef0pMQVwT8KFj7hKFpRcROlFPak9cQOduPZQPMcyudLYqePARZwisTF2BV9hJ4c1wUL56NlLt2jBfDVeR/1Ybe+PTjxitEKpB9zff/9NLVq0oH///ZeePHlCLVu2FNbH5cuXU+zYsal+/fo0atQoSpEiBS1YsICaNm1KoaGhlChReBkr3I+/IS7TqFhtUV1/8MReJQHcfVxHWbaSGJ1bdFz3+/ZzNGjRIcqUIiE9evqC7j95QRNbFqXGxWPW6TU6sDOrT2TXFxiyIkJzh4fXoSQmx785Gu93QSdpYpDNTcoCVx9cfq6IHrl4xa/WijCKYW8XpPe9mARw8U6onQtQTmjyhgJnBJPpG0/RmGXHCNVX1vYOV0rYYmumQiYny6Bs6pwd5yNVDDIyZiPXLNh7UXAaftk0wGchGUbG5c418rpPEj8OfdOimKi2xIKa3it7hluX5T5kSzMslMwwEDysNiUzSNflzpjduUcOazp7M1RQirlb2tWd/s26x2p7v1nzcqedN0qhhOsblklYLBctWiRc3qNHj6YxY8YIRfLkyZPUp08fypYtG61du5bmzp1L7du3p5cvXwplk+Xdd9+ly5cvU1CQLaBcT5AAhP+xXLlyhQIDA+nChQuUNWv0Kz23Hz2z13Ed17QI9fvHRtnhyxg4dxYk3zNv13n6/J9gSps0Pj16+lIk5vzQuji9XTSzJ82qe11AALXf8wxYFuEOmZzbhabcupTpfuSb3UnokOuIc53ncmPXiCSAUY0KUzsvJgFcu/+EyoQVFQDbQuEstvJ33lDgjIDMoSRaK1f1r9fT6ZuPaGD9AvRhZXMS3xB+lKu/bf00LZGV/tl7kbQ1zY2M+f/tmqkbTtGXy4+JaaMAANzAHLKEv2kTqmR8OBYWiTyBuVJTm59s1YkcEaFHJ7ZyWNOZW488Ku0anfNQCmU4+m+MQrl48WJq06YNlS1bVlgWYXHEB+3evXuUMmVK+4y3b99O5cqVI/w/FkKzZs10LZRPnz4ltOlIhg0bJhJ6tGIVhVLmZgOFxODFh8VQzSYV9taL/M+ei9T77wOivFjosxdhpNYlqW7hjN7qUrWrgwAUSiiWLN5wVzoCnjcc+Xd3FFqZC5ItNUwA7e0kgFsPn1LJUbaDqZyhXv2b9YJ3z0wFzsgC/mXzGRqx5AhlT51YVCxiqTJ+nci0RQII3NNmSd4By0S5wAZFMtHSg1dEdjcOBUocI8DhPrgCzATjmxeltjNsiiGkRPaUtEDiEJVbsmfrV89LpXOlpvY/7xQ/I9Qjftxwo4kV8Od5Yo6X7z0RQ3KHZza656IUyvAn8EYolMjw/uyzz6hJkyb0+++/R4in1C62R48eiUSd+fPnC0slYiiR3JMnT3jcDWIoixYtKjLGHYnVLZRwExcaulIMH5vW6GU2yg6zSYW99TIv3n9JuOnh8kFsDeTn90pRjQIZvNWlalcHAf/By+nJ81fiF2xIHIPoC7B+3XqWhv5rOwhBoiJ0jmo8qBHMyQnsMucSdV83L0rNvJgEcP/JcyoSlpjy10flhNUIImfj+pIKizHVJnZ4KwQg/6Dl9PTFK6pZID0FHb1OKRPHo/1D9ClvfLGmYkIff+w8T/0XBIuhwpL8VdMidtc1/qatwy7PSeY3LZ0zNb0/05aMeXpMfYqtU1knOvGYs+McDVx4iFIljkd3Qp+LoaDme8V81i8NLOOmFMpwNGK8QjllyhTq2rUrde/enSZOnBjBdQ2L5a1bt2jlSptiBVm/fj1Vq1aNjhw5Qrlz56b06dPThAkTqGPHjuL3NyXLWw6Ilyk79g2uRakM1FGOzg8N+uZECigRbCH79YNAquJno3tS4hsEmI4FvSVLEJeCfWhdmrvjPA1YaNtYIUjOOTJCn9A5KjSOXL5P9b+3kfwzH1/A0JX04OkL+rZlMWpU3LXa4K4gL8ehgr6rfF7bZinzBfqSyJk5ORGbvK1/DftUyo9dI6xE8Ga0L2ceNVfhoStFDfYKedPQlpBbIv7VEeWNK7i+ydciHrTXXwfEFJGN/2XTIvYDEf5W2S8d/fZBoC4EzadupV1n79BHVXJTYM7U1PFXW/30s182sBxkTOeFMAg+tM7rXJbKxIDSwDKYSqEMRyNGK5QnTpygwoUL01tvvRUpIxsu702bNlHDhg1p5MiRggII13/yySfC5T1nzhyBwsCBA2natGmCJihnzpzUt29fOnv2LAUHB8doHko5/g3kyT+stVF2yOXfLPeFkQa0+sg1+vA328eQZe6HZah8nph1erUyxkbGxq5hXIt4LlR78ZWA3Bkkzyzu9n/qxsNIJegKDF4h4nJRY/qtIt6Ly5UTU+QDEVc0GfFOIXrXRAXO2bNhJR1UTDtUXayoAAAgAElEQVQGhHMZyiX7zKTmKjZiFd0NfU4lc6Qi1IbPmDwhbR8Qrsg6G+//4+8IDfhkro2VA5XCxjQJoAbfb7ZDEVU1oJbTttGOM7fpw0q5qFTO1PTR7D1eS4Ty9NnIijO39c/H5cVaiUmiFMo3RKFEsg0UQj157733RIWcf/75RyTmHDt2TMRSwmqJLO+ECW0VK5CQg0o7IDZ//PgxVa5cWbi6oVy6IlZbVHJAfOfKuQWfH+TIiDqUOH5cV6YWLdeuO36dOoS5a3gAf3cpR3DjKPEdAqz4oEdfKwMc9sCz1VrVjKIgc7Ku7FGZ8mdMRlxZBZRadQtnMtqUy9fJ76EcssFJQb6uDPLnzvP0xQIku0W0FHorprTUqNWCsLpQ5uSCdzNrqkS0+XN9DkWXwX1Db5BZCRAiMaZxYXvVM0y5fkBGmty2pO7s287YLizBH1SAQpmKus7ZSwnixqbjYXRZVoLs3wOXqbtEh4Sx/dutAhXJGp7zYKXxOhqL1fb+6MQsRlsooxM4bd9WXFQcEP9euRz067ZzYsic5Wol7PTGsvnkTWr3c3ggOq6JKXXIrY6tK+NjvkbcA07QTf18pwxw2AOPNyr+vajmdP3+EwoMy7Tm0qO5+i8VWaUz3i1FNQt6Ny6X38Op7cKTytgiOKZxALUp47vKIH/tviBIsrXW3hIjVxOYIUCi3aJUNleWSJTXMuE4aIqQhOTuMzRtQDGgIfnbhzr2oxsFUOXx6+wjb1QsM33bqrjuTNr/vIM2nbxJ+ObDQgn+SsShH3YjVMTbUGnfb/SH6k2o4hSTxIp7f3ThpxRKk5C34qLigPhWpbPRn7suiJmeGlNfJDdYXbafvkWtpm+PMEyZdsXq439TxsdEyZgPSiCukbgLvT3HoCPXqJMU9qCt7mK0f5mgHXXti2VLZadD8kVWKSc2ye51b1kEnWEisyfI9aCLDl/llUolbOFGEhBKeeZNn5RA8K7EMQK7zt6m5lO3iQvg3h7ZqBCVG7vWfkOLUllpXLOiug10mLmT1h2/Qe3KZqdSOVJTj3n7KVnCuBQ8zHqZ9dr3GxMK6lWZ8qZPFqOWhxX3/ugCUCmUJiFvxUVVaMgKkSGNE+2i/ZfFTH1J++IJtHvO3aamU2wfVRZ2V3rSrrrXNQSY1w53aes/u9aS61dvPHGD3v3FRnsCKZotJS3+pILLDWkTY0rmTEX5B9kI232RVaqXAOQti6AzcOxlKDVKhreSlDibPU2S+KLalbuHAmfzepN+P3jxLjWcZCuZCLqlEQ0L2amn8Dcoi6MaBehOudOvu0Q2fevAbFQyR2oRg4ws6n0WzKxff/y6PQudJ7O+T1XKmdZWnjSmiBX3/ujCTimUJiFvxUXFGbr1Cmek5YeuWjY4W+8RHLhwl94Jq0PLv6/tXYVyu1jH2aTH+3/bDNOQCIUuawpa3K2iz7DYduoWtf4p3EqNeDJQ77gqcoIaLJLIfmVKrT8+LEvl8qRxtUmXrmeKovHNilDzMHcyJ6v4unaxTMclu0ELDlkhylt+37o4NTSxeAATuDP9V1Sk3C6B+gZffOLaA6o9caOYIQjhhzYsaKeewt86VMhJQ98upItA599206oj1whWTCS32IpDWDOzfkvIzQj8mpiQO6VVo3spWHHvjy5MlEJpEvJWXFQlR64WVoHq/ulp7bHrlg3O1nsEhy/fi5DZiGs29atG2VInNumJqWaMINBq+jbafvq2uLR0zlT0d5fyRm4z5RqtlToquhRnHXIc47T2Jals7jQEFy9kfpdyItbMm1J6dBDdePCUxjYJoNaBtnjJgGEr6UE0lBNdcvAydZu7L1LFGg6P+bFNCWEVM0u4ZjmibMCPXyxbSlrkhpXZrPHEhHZQ17rK+PViqIivHfJWQXtNdPwNlED96xXQnUrXOXtoWfBVoYhCoQTtlq+T6YxivOP0LWqpCWvaOaAGpU9uS5iNKWLFvT+6sFMKpUnIW3FRBY4OousPnto54KwanK33CORTOv++rX91ypQivOa6SY9ONRMFAhzkj0vK50lDcz8s6zO8ZNcfOq1VMAP99G4pt/pnCxzKd2IeXL3GF4leehyPzM/4Xati9E4x7/FgasHiRAgtSb3fwOWilrLZWe9yyATG4utDiVuLJZpvunrvCZUdu0aMAtbIwQ0KUm6pBGr36nmpV+38uqPsNncvLTl4RYQ5QaFEhTQtiX00T8/e/d7zd6jJ5K0RhgNaMiSMxSSx4t4fXfgphdIk5K24qHgjK5UjFe0+d4eSJ4xLBy0YnK33CM7cfETIMJaFq5yY9MhUMwYQ4JgsXFo1fzqa1UGfUNlAUy5fcvTKfar3nY2QHII67lAI3RF2MaMyTuV8ae1Z375I9Ko8bh2dvx0qLE0fhJU1ZB5MX9enX3n4qi43IZfYnN6+JNUuZF55U1QoQqUilqiqvLjzXN/Ee+48ekbFR64WU+tSJQ99Uc+f2MKOv6FQRbfq+XSn3uPPfSJeHu8KSjQO/y9ymU2rYBZ88R69PSmcXxPj4tKoVhmjkXFYce83Mm5vXKMUSpNQteKi4mocAVlSUPClez4npvYEWpk7kNs5MKQ2pUgcz5Nm1b0uIsAuNNxWu2AGmu6mhdDFbsXlMiE5/o0SiVAI3RGZpqdK/nSE7GOIL2hKuG73gPr+1LmyrcQru5gnty1B9QPMczE7w4YLBsSKhQS98OopOb9YKm41u7xpsylbxWGWBTQ4szuWcTbM/+vf5SSyHjXzUY+afsQWdgDTv54/fVQlvFSwDFavv/bTgr2XqEFAJiqePSWNWnqUcqdNQmv7VLUcptoDIwZ4bGRdShgvjuXGGtWArLj3RxeAllEoQSoeJ04cUZ3m6NGjtHTpUipfvrz4X0wQKy4q5hDMlz4pnbz+kNIlSyBqeccEkd0+PN7Dw+tQkgTWJ2WPCfgaHSNbPHA9YusQY+cr0R4qospudTYmPlwNe7sgVffPYOf18wVNSZ2JG+n4tQfUt05++qRaXjFUXxGra3FZe+wafTDLVoGKGR9evXptd6maTaMkx+Ciz2r509FMH1q5na0LK/4uJ5F9XtefPq6ah9jCjvEOfbsgOSrX2ffvA/T3notUp1AGKpE9FY1dfozw/V9tQaqmkOsPIhC2Y24ho+tR3DixrfhYHI7Jint/dAFoCYVy48aN1KhRI5o/fz4VLFiQ/Pz8RE3uhw8f0h9//EHNmzePLnwM92vFRcUZltlTJxYuN3crjRgGwcQLbz58SuDqk+XEqHqE2C8lvkOANyj02KR4FprQspjPOtceKjpWzEWD3yroVv81vllPp248EtYdEJnX+GaDaGddn6qCbNub0uD7TaJKTM+afvRZTZurMnf/pSJJxWwXs7N5yFQtzEkrl4dEjWgkP5kl7WbsoM0hN+3NRVU20Kw+34R2+MDBYRKc2IW5jW5cmByVx+y/4CD9sfOC4K+EhXL8yuOWpWo6e/MRVZXCmmA1Pz2mPsXCf8QgseLeH13wWUKhrFixolAiv/vuO1FTe/z48XTy5Enx3zNmzKB9+/ZFFz6G+7XiouIMS9TtvXb/aYwqeyaTUfNDiCkcmoYXTQy4cODCYJqz47wYactS2eirZkV8NupbD59G4N/7pFoe6lvH363+63+3iY5cuU+9a/lRncIZ7bQsvmAOAP0VaLA+rZ6XeoclU3jLxewMHJnbkw9oT56/tGcRz+lUhirkTeusGcO/vz9zJ60/fsN+PSjMprTTLxtouNH/gws5aYuVR7kEqkw/pYWC31dYgotnT0UTVp8QZS+Xdq9kOdRAdM+hJxhc/Dix6cToepYbp7MBWXHvdzZmb/1uCYUyceLEdOjQIcqdOzfVr1+fsmXLRtOmTaNz586Rv7+/qLFtdbHiouIMy5SJ49Hd0OeUM01iWt+3mtWhFON79PSFnSsQ/0Z1H1hUlPgWgeH/HaaZW/7H3nVARXV04U9Q7GLFikoRsWAXrNgbGjVqbIldY9dYE+y95rfEnhg1sSV2Y1esWLGgolhA7L1jxI7/uYNvXQnKrr59b3a5c05O1Dd7595v7u58b2buvZfEoC1K5QLVntaqRT5/9UH+vd7VPNCjStzBCPHppASHEKmrVTAr/H6JCfbRInOAco9QCbD44Ii5dUlU8nSKT33Vnhvn/lPuq0W9fI38Q7aIMdTOy9n+jyMIOHPboP+XBFapBoIVCFKuSShXEIxLoH4qM8CQtafw54HLYpeZUjT9sj1M8/yxpsJrXBKVPpPCwR6hEpaIjM8eGdf++HS21HMpCGWmTJkQGBgIFxcXZMiQAfPnzxfH3IcOHUK9evVw69YtS9mvmlwZnUrZGUmexB7PXr3RvHTel4D74vUbQzUTkpMsiR3OjrS+t9cvwUCGz47ddAZzdkcIVb7kyPlzbDHeOaPPGwe1mCuv8ZwDCLr4AN/7uqJOoayGSiRBA6vAKbVl894p9wjbl3PBoDr58fpNNNwHbhIm/NHWGxVUPGKODxfjZPHKneQnz1/Ba1hMXk5KHE8J5NVqnRYexebT73+/vy6aHZM1vDahlh1ay6EsF2dvRqJGgSyws0sEhWCSHrO+LYZaHwnkUl4Ay7lnFIRy+s5wEe29qov5FaYsbTPVjqeKUUpzTJ4ExuVALT2+WvJlXPvVss1cOVIQysaNG4tdyPTp02PlypW4efMmLly4gI4dO4pdS7pHKXuT0akaztqPo5cfgq6kvH0L5M2cGlt6+coOpdDPeBeH/p46aWKEDJevHq1VgPkFSk7aeg6/7AgXEpQdti8QZ9ZHjYMT6IPD6xZAqzK5zZKhdFbyabYukxt1i2Qz5L8LHlwN6Syc98547GF1C8D4ZWlhO2+Uz6PencX4wCFSTeSampKiJXatcyrZp1ZT8iIq8r4pngMTPzNSXy2drFHOV9P2ikwd1D4ViT9qfSjm7r0ISs9ENetn774gbe7P2CcQGVM54MigalY3PTKu/XqBKAWhvHv3Ljp16oSIiAgMHTpUBOj06dNH7FBSoE6WLOrlRbMU0DI6lbIro9icP2sabOwp312aj82JkhuPnstaj9ZS/iSL3Gnbw/C/beeFOnTcTMfOWjZjHxjf0AtNSsZUmjG3va9xnBO0S6aQqpPDqiNNMsumomozPwg7z93Ftz45MfprLxjvvC5p74MyKt5ZjA+XI5ceoNHsGEKppOEy3ilSO9F7r7+PY3XwdYNaVGN6bAPt7uHGh4e1PDcugfqplxDlRIHKixZ2dsRvgTHk8q/vzS9ZamlsYp9AyFrRJz4cZFz749PZUs+lIJRxGffy5UuRQshamoxO1fy3g9h/4b4BwkI5HPGPhrWYv3TulFx9JMcpdVIEWUnKoy+1W6bPz9l9QaQeofaphMqW0tlz8CY8fxUtxE9pUgT1i35eVRkln2aDYtlFPsvmvx0SMs+MqInkDpbNe2dcX3lCo8If3A9W+85ifPNgXJ1EqUpCZSEpipjaP93KolCOtPGJMfl53+UnsOLoNUN/re/hmqyo5B2N0y/9/X0p+LjGXX9+wuazmLnrgqiSQ7/3dP+Zjr8XtZcv96fx1Q+C3zl9cgT2ryz5TPxXPRnXfr1AlIZQ0jH3b7/9hrNnz2LKlCnYvXs3vLy8RFCONTQZncq4bB5hSGkkVkt4l+Zj86tEOtJzWcuHWYNvfomO8/ddFNU2qH3JHcbP1UGpeU2f/5KygMpOGd2fbFzCGS3nBQmVtEhF1XXxMWwIuWlIu2R81Kf2ncX4cKZoc7pbTY1y0lJu2tuRz+EzJqbUn9qVg5Q0NopeVEpw6FcF4lOTn8dCwPi3fFWXMiLHZFztf1vPYdqOcBR2TosiORzxx4HL4o4u3dWVrb19+xYu/hsNasmagD0+3GRc++PT2VLPpSCU4eHh8PHxgaOjI2hyiFT++OOP2LRpE7Zu3WoVyc1ldKq2Cw5jx9k7Bt+xtjq6xsl8c2VIgd1WEqFuqS+rHnIXH7qMgatPiaEpKXjrsi6aqlFi1Dbc+/elGHP+F0RE/7jiJP4+clVU+2nmnRNtFhwWMinvHQU9WLL1/CsYa9+Vw6NSi5a8sxifHaeuP0adaTHl7oIGVIFTmmS48egZylioctCgNSFYdDAm7RS1DuVdMLD25+USjc82W35uXAL1U6R/SsB5TAkIA1VHox1KSvlV2dMJ81qXlBIeJd8mKeeRORW29qogpZ6fUkrGtV8vEKUglBTJTZHetEOZJk0anDhxArly5ULr1q1x5coVsVspe5PRqTr8eQRUak1pst6l+djcUmJzSnBOzd0pFQIkrPYgu19+qX7LjlxF/xUnhZhPJVT+0nE+9vnSY7fj5uPn4vGSDj4o4/Z5ORIHrzmFhQdjdmvoLuP3C4+CeGSEUflBS9nQZ9kJrDx2DUoORuM7i5/abbKEPqdvPEbtX2IIpZIyybgi0dZevvDInFq1oYf9cxoL9seknaJGVV+o+gs38xAwLoG6rZcv8nxkjpQ7z/myphHpgv46fBXV8mfGbxqWTDXHMuOSkrLmy4zPHhnX/vh0ttRzKQglkUkijVQlJ3Xq1IJQUnQ37VTSzuXjxzHRbTI3GZ2q86Kj2HTqfcoOWe/SmEImPLOkxuYfrCNCXWY/NVe3tcevo+dfx8XHPpVQ2Vy5pvb3nbBTVHmitrJzGXE37HOaEv1a2jUDWpbOhc6Lj4mqS3Tkben208qTYmFXqsQYV4Fa27WsOJ7Uqp29FYmaU2JycO79sRJypEuBy/efosLEXeLf6KWNXt7UaqM3hIrAEKX1qOyO3u+Su6s1RkKQo+xyk627+1VErgxxV3eauSscEzafExk9vHI4ivurMieTLzRsCyKfvxZTSN8D+j5YW5Nx7dcLQykIZbp06UQeyoIFC35AKCnKu1atWnjw4IFe+Jg8roxOFTtlh6x3aT4GslJ/mZ7TEc667uVMng/uqA4Cm0JuCvJF7VMJldUZ7b9SlJKJ9GRDj3IokM3xs4YyDlag1EM9lgZrlkhZOfatmDcTFrTxhnFC53XdyomFX6sWdvsJqk3eI4ZTqgRF3P0XlS1UinLcprMidY3SjMtPamWzLYxjXAL1U8n4lSA6eikolN0Rq4Kvi7yr05sXkxKG4iO34f7TmCstJXKlw4rOZaTU81NKybj26wWiFISyadOmwv7Fixcjbdq0OHnypDj6btCgAZycnLB8+XK98DF5XBmd6oe/grHm+A2DDVU8nfC7pHdp4gK68s+7EHHvqXgka3Jekx3ESjtuP3Mb7f44IrT/VEJlS5lXa2ogztyMFOK/ZPdMuVtWMHsakaC9198nkCZZYpwcZvncpsqxr3JCYFyj/EtI8udgfuHuv4Y65rv6VkTujCkRfucJqk6KIZl7+lVCzgwpPkd0nJ9RgkSUh/1q5EXXSu6qyU8ogoxLoB4ZVBUZUyWN0/S5gREYteGMqE9PL+H/nLiBekWyYWrTolJC5TMmQJQFpkanB0u/LyWlnkwoTZsWKQjljRs3ULFiRdy/f18cb1NkN5VdpETne/bsEfcpZW8yEkrl7paCHQUk/CrpXZq45te4OgRV76CIWG7aImBc+/lTCZUtpVW96Xtx4lrMlRfliPZzxqJdMtoto50bqpZD90LTp3QApc6xdFOOfX1c0uPvjqVhXMN4U8/yoPtuWjWqwEJl/Kht71MBbplS4dytJ6gxJYZQfgnGcdkwNSAMkwNi8phS86/liY4V3LQy12bGMb6L+qncqUpWhpzpUwhCKbILFMuOSY2LSImFcY3y8nkyYmE7+dIbxQecjGt/fDpb6rkUhJKMi4qKEhVxgoODER0dLY6/v/vuO7FTaQ1NRqdS7m4p+Pl5ZcHMb4tbA5xCx9q/BOL0jZjdqbLuGbC4vfW9vVoN2B9R9GDEfTT99aB4+mdbb1EjWMum1MGmMZU0N58z/sIDlzB47WmRfqpbZXf4rwrRLLfp+M1nMetdbkC6B2rJIJj4sLlyPwq+E3eKbkpwR+iNSIvVNp+xMxwTt5wzqDW4Tn6xQ8zNPATGbjyDOXtiSqAqNdjjkvDngUsY8s7PiVBS2cvGJXKA8p/K2IxrlMscjf4p7GRc+/Waa2kIJQHw/PlznD9/HkmTJhVBOUmSWLaChZqgy+hUxsckZOtXhbOB0pZYS1NqkZO+yv0za9HdVvQ0ToT91/elUOojCZUtZW+zXw/iQERMcn6lVODnjLXy6DX0WX4CaVMkQZ9qHoJcZnNMhv3+VT5HnFmfUcpXUtTt2m7lLBoEE59i1x5Godz4GEK55Qdf5M2SGnGlEopPjqnPjRPj02dG1CuAlqU/r3ymqWPaYr+ft5wTdbmpfSrV1aKDlzFozSlQ1ZmC2R0RcOa2SJM1toGXlLBUm7QbYXf+FbrVKJAZc1qUkFJPJpSmTYsUhPLNmzfo27cvZs+eDaqQQy158uQiF+WgQYOQiIpRS95kJJRD154SiW2VRiXnJjeR8+gjruk13p1SImQldwObU8+YbHxJlPXnAkMJyOnYndqXJCHffOomOi06Bgd7O/j7eYpk7VpV5vhlexgmbTsPpfSpcRDMjj4V4JpJvajq+HC++fgZSo/dIbpt7FEe+bOlgXGy80/dz4tPdlzPlTt9yjM9Uk99jt6yfUbxocR2iRA+xu+j6v0VdAU/rQoRCesLZksjSn5+VyonRtWXk1Aa35GuXSgrZkgaPMSE0rRvhBSEcuDAgZg3bx5Gjhwp0gQRwaSo72HDhqFbt24YPny4adbo2EtGQjliXSjm7XufsoNKzv38jZxHH3FNnXG5MZlTX+jodhYf2jgqWO0qKqYo3/6PI2KXhXJGXhjj99kvl8Z3QfvXzCtSq2hVmUNJ5aIkbg6/8y+qTorJrfupFDCm4GNun7iq4hjvQgcProZ0KdUrefvH/ksY+s9pg5oTGhZC45LO5qqd4Psrd4BTONgjdETNj+Kh5I2l+8GU1zEw7B5al8mNYXXlrE5Ud/penHx3R7p+kWyYImnwEBNK076CUhDKzJkzY/78+fDz+/DNa+XKlYJQUllG2ZuMhHLMxjP49d29G8KvaUlnjGtYSHYoDfoZlxurWzgbfrGi43qrATkeRY3v+wX09oW7k3pJr03BSEnonDyJPc6M/PhCGp+so5cfoOGsA6JbpwpuIpVNHqdU2KZBsvzf9kRg9MYzBgJ7/vYTVI+Vuic+/dV6bly3W8mBeeTSAzSaHYPNiaHV4ZhcvatGxpWWSP6kxoXRoFgOtcxJMHJ+33sRI9eHiisbx4dU/6jdq45dQ+9lJ8QcUkaDfeH3xZ1VursqY2s4az+OXn4oVPumeA5MtKINDwVPGdd+veZaCkJJqYIOHDiAfPnyfYBDaGio2LF88uSJXviYPK6MTqUEAyhGUIWQ0V/LefQRF9Bt5geJIxtqMkcqmuwkVtiR6u32XX4SL99E45emRT57h/BzTVcSOqdLkQTBn1hI45NPqYfoeI0a3SlbGnRFRFdTlLWlmxJ5qxyxG+uy76fKIlBIq2ZcpWd1lzIomjMdDkXcR5N3gVenhtdAqqSJVVPn78NX8OPKEIM8PXKZqmaMjoKUu5FOqZMiaGDVj2qiFCJInTSxuM5w6OIDdPR1hb/fh2urjqZ8MHSTOQeEjsr3Uta7np/CS8a1X6/5lYJQ9uzZE3fu3MEff/wBB4f3xy1t27ZF4sSJ8euvv+qFj8njyuhUSjCAYkSr0rkwvF5Bk23Su6Nx6cgmJZwxvpH17K7qjZ2tjN93+QlR7YOCDA4O+PwAGuPoZrqrteHkTc2S5VPJRyr9mNUxGQ74V0Fc5Q+1mq9HUS9RZMQ2MdzKzqVRPFd67L9wD81/OyT+7cyImkjuYK+aOkowlCJw5rfF4OeVVTX5CUUQnRS0mhckkpR/qtLQuhM30H1pMGhHn468j1x+KHW5y+/mHsLe8HtiGqmC1QgrWp8U35Nx7dfre6EboaxcubLB5tevX2Pv3r3ImjUrSpYsCXt7e5E+6OrVq6A63ytWrNALH5PHldGpYueAa1vWBUO+kvPoIy6guy4+JvKoUZP5YrnJTsIdzUaA0vvQbmLuDCmwq18lsz+vfMC43CHlu6O7ZUVzpsXqLpYv9aYESlAyagp6Cbn2GF9Nj6mnHTSgCpzSJPtsu8z9YOTzVyg0bKv4GOV1pfyue8Pu4bvfYwjluVE1kTSxeoTSuHQnyZ/TojhqFMhirtrc30QElMpWVFaUgsCOX32E7pXd0UfScpet5wdh17tTKJmP5j8Fv4xrv4nuono33QhlmzZtTDaG7lfK3mR0qtg54Cih8wBJjz7iml/j+rUyXyyX3TetWT8loTPVJt7S6/NruT97+Qb5hmwWUFD6HkqWXjJ3OizvZPlSb7TDSjutyv03S0ZVxzfXT1+8RoGhW0S3pR1KobRbBuw+f1fsflELH10Lie3t4hNj8nPaCe66JKZ0J7V5rUugsmdmkz/PHc1DYMvpW+i48Cjs7RKJHUoKePmhah78UNXDPEEa9VaC7mi4jhVc4V9LzqN5JpSmOYRuhNI09aynl4yEUokMVFDsXNENP9b0tBpQjSv9dCjvgoG1rWd31WpAllxRpcqMksPxc9Wlu6CuAzbi7VuI3c5L96M0K/Wm7NLR3US6o2gcVU2VeigiV6v2/NUbeA6OIdZL2vugjHtG7Dx7B20WHBb/dnHs50fSx2WDQnCUZ3okx9cKWxnGMS6VSjuUoTcj0be6B7pVziODev/RQQm6owfdKrmjb428UurJhNK0aZGGUBIhmzFjhqjjTQnNCxQogI4dOyJnzpymWaJzLxkJZewccDIffcQ1fcaVfrpUdEN/KyLDOrujzQyvRAmrkVKk4NAt+PfFa0HgKDhFq1Jvyi5d0sR2ODeqFowjzk8MqQ7HFOpFVcc38S9ev0HeQTGEcmE7b5TPkwkBobfR/s8joHS/F8fWjk+EWc+NCY4xiTVLCHc2GYGd5+6gzfyYlwPKYkBJwylNVmUyHwcAACAASURBVJeKctZP77E0WNQbp9arqgd6VpWT+DKhNM0FpSCUp06dgq+vL1KkSAFvb2+Rh/LIkSN4+vQp9u3bJ8il7E1GQqlElyrYyXz0Edf8DloTgkUHr4hHParkQe9qch7byO6b1qzfm+i3OHzpgaj68aXRx96jA3DnyQtQcujX0W9RKW8mzG/jbXF4tp6+he/fHUNSLs2giw/QeE5Mmp5P1WW2hGKv30TDfeAmIXpBm5KomNcJyi5ifEmzP0cf4+N0+rxyb/NzZPFn4kfAON9qrgwpcPl+FAb4eeJ7Xznrp/dedhyrjl0XhvWrkRddK8lJfJlQxu971EMKQlmrVi2kTJkSixcvFmUXqVEZRqrlTf9fv369adbo2EtGQqnUL1ZgkfnoI66pU+7P0TMql9e9ivW9verokjx0LASM6wbTI62qLxkfKVPZPEqT0uy3mProp4fXQEoV0/TEN+l09O/iv1F0m9+6JCp5OsE4kIOqEanZ9offQ/O5MQE/1FZ1KYNiOdOpOQTLMkLAGG/KKnDz8XORg1LW+uk/rjiJv49cFRbITHyZUJr2NZOCUKZKlQoHDx5EwYIfprSh42/auXz06JFp1ujYS0ZCSdGxFCWrNLo/SfcoraUp9+dI359qeYqE1NwYgc9FoPYvgTh9I9Lwca2qLwWG3UWL32OCXiiK+uilhwaSpXaaHlOwcfHfIO6Szm1ZAlXzZ8b6kzfQbUlMqpkvSR4f19jGOS7p+T/dyqJQjrSmqMl9PgOBgxH30fRdTtEMKR1w/+lLDPsqP1qXdfkMaZb/iPEp1NCv8qONpHoyoTTNF6QglJQuiHYhixcv/oHWhw8fRrVq1ZhQmjaX/+mllOFSHgz0y4cOvq6fKU37jxknZh9UOx/al7ce3bVHi0eMD4HGsw8g6FJMEmVqlNNvuga1gw9cuP/BjiQF5RgTTDXT9MSHAT13G7ARdJVASeETO2jIFBmm9qEqKFQNRWlK/XBTP8/9zEOArod8867qEV0RoTvDI+sXRItSucwTpFFv41MomfVkQmmaQ0hBKOlo+8aNGyLfZPr06YXm9+7dQ+PGjUFVdFatWmWaNTr2knGHUinDpcAypE5+tC0n55tqXFM3adt5/LI9TDwaXrcAWpXJreMM89DWjoBxzjuy5eui2TG5SRGLm/VBacMh1XH82iOLpekxxRiPgZtE5aNZ3xZDLa+sWB18Db3+PoE0yRLj5LAapogwuc/Ja49Qd/o+Q/+tvXzhkVnb8p0mK2sDHY0zCDjY24l5puozVB1KxmZ8CjWugReaSqonE0rTvEcKQklkrEyZMnjw4AE8PDxEebdz584hXbp02L17N1xd5d+ZkpFQxk4qPKJeAbQsbT2kbNr2MPxv23nhyWO+9kJzHzl/FE37qnEvvREwTpRPujQqngM/a1A7mJJL158RQ6qUxOZKmh66U2lnl0hTaDwHb8LzV9GY0bwYqGqQkifzS8tbxmWEcVUger6jTwW4Zkqlqb0JaTDjHKeK3RMaFULjEs5SwmB8CvW/bwqjYXHrq/Mu49qv12RLQSjpojhFdC9cuBAU8U1/9/LyQvPmzeHo6KgXNmaNK6NTxU4qPPrrgvjWR86jj7jAnrXrAugHh5rMP4pmOQp31g2BfstPYPnRa4bxm5Z0xriGli/neer6Y9SZFlMZ5+C70ovt/jgi/n5pnLppekwBt8CQzXj68g1+aVYUdQtnw7LDV9F/5UlkTOWAI4OqmSLC5D7nbz9B9cl7DP0D+1eCc/oUJn+eO5qHgLGvKZ+UmagZlwe21jrvMq795nmNer2lIJQlSpTAvHnzUKiQ5X/c1YPuQ0kyOtXmU7fQadFRg6LjG3qhSUnr2eUzzqM5uUlhfF3U+t5eLeVvLNd8BIzva9GntSrnee7WE9SYEkOqiFCduRkp0gjRxmSEynkfTUHFa+gWPHnxGlOaFEH9otlFaUsK3nNKnRRBA6uaIsLkPhfu/osq/9tt6L//p8rIlja5yZ/njuYhQL5Va2rgBx+SmagZn0JZa513Gdd+87xGvd5SEMqMGTPi0KFDcHOz3iheGZ0qdlJhOt6jYz5raQv2XcSwdaFC3enNi6JOoWzWojrrKSECEzafxcxdFwyaaVXO05hU7exbEeduRaLTomNIYp8IYaP9NEeq8PCtePzsFZSdq0UHL2PQmlOgNDMH/Kuoqs+V+1HwnbjTIDNoYBU4pdaudrmqxliBsNg7wrL/dhqfQv3WsgSq5be+spwyrv16uaoUhHLixInYsGED+vXrB3d3dyRP/uEbrDVUy5HRqXadu4PW76omkIMpOxJ6OZu54yoLHX1u9nfFUbNgFnNFcH9GwIBA7Nr27cu5YFAdy5fzNCZV23r5iuolXRYfg1I5R+spKjZym6gUNLFRIXxTwhl/HriEIWtPI3va5Nj3U2VV1bnx6BnKjNthkBk8uBrSaVhqUlVjrEBY7B3hmN/OYqhZMKuU2hufQs1vUxKV8jpJqeenlJJx7dcLRCkIpZ2dncF+CshRGt2lpL9T5ZyPNQrkGTBggEg7FBkZKY7Nx40bh3LlyomPHD9+HD179hSVdzJkyIAePXqgb9++BnHR0dEYPnw45s6di4cPH4rPzZw5UxBbc5qMTrU37B6++/19UuFpzYriq8LWs8v39+Er+HFlTB7N31uVQJV81vf2ao4PcV/LIhC7clTHCq7wr5XPsoMCMCZVlDaHFv3uSy2T99EUY0qMCsC9f19AuQKj4JIzfQrs6V/JFBEm97kT+RzeY7Yb+mtdGchkRW2k46V7T1Hx510fWPNri+KoXkDOl3HjU6hF7XxQLk9Gq5sJGdd+vUCUglBSJPenWoUKFT76uHr16rhz5w6mT5+OTJkyiXrgRA6PHTsm/u7p6Yn69eujd+/eInl6ly5dBGFs06aNkElkkv4+f/58ZM+eHf3790dERAROnz4NBwcHk+dFRqcyzn9HhihpQkw2SueOK49eQ5/lJ4QWf7b1hq9HJp014uGtGQEl+ESxoVsld/StkdfiJt158hzeo2NIFSX2vnjvKXr+dVyUkjw1XN00PaYYo5SgVDInKLtELhlTgo7k1Wy0E0o7okrTI5G7mvbILuvqgyiUn/D+igHpO691CVT2lPNlfPGhyxi4+pSA9e/vS8HHNYPsEP9HPxnXfr1AlIJQGhtPO4729vYmRXeHh4cjT548ot43pR2iRrualHqoWbNm4uicCOalS5eQOHFi8Zx2Mymv5dmzZ/Hy5UvQ/c0JEyagU6dO4jlV5cmWLZsIEmratKnJ8yKjUxknuSVDZH5TjQto47RHSzr4oIyb9b29muxA3NHiCCgVYZSBelbJg14a1Id/+PQlir4jVSs7l8GVB09F3sfUyRIjROW8j6aAWHrsdlGST0kk/eueCxiz8SzcMqXE9j7qEsrI569QaNhWg1pU2tEh8fsTKVP05T6mIxD7igF98o+23qgg6cu48UsefTeK57K+spwyrv2me4y6PaUhlHSPcurUqbh586aw0MXFBT/++CM6dOjwUYuJ/O3fvx9VqlQx1ACnznRcXbNmTbHTSInRlyxZYpAREBAgqu/cvn1bEE0fHx+R85JIqNLo2JuOzmnn0tQmo1MZJ7klO2R+U40LZ6XGMD1b3qk0SuaOSXrPjRH4HASMa2rT57Wqbf/k+St4vSNVtAtz7eEzsfOeNkUSHB9S/XNM+aLPlB23A9cfPYOSl1YJjPDInApbe338NOhzBo16+Rr5h2wxfPTiWD9xjYmbZRC4HfkcPkZXDGiUxe19UNZdzpdx4+Ib1lqWU8a13zLeFb9UKQjl+PHjMWLECHG/sXTp0qB7jbTrSIRu8uTJ+P777+O35F2P5cuXiwo7//zzDwYOHIhatWqB5CstNDQUBQoUAJV1vHLlCho2bIioqKgPAoHo8/RvdC/T1CajU8WuUiHzm2pcOG8LvY0Of8bk61vTtSyKOHMNYFP9kfv9F4HYdaW1qm3//NUbeA7eLBSixZ3IXP8VJ5E+pQOODVY376Mp815+wg5cffAMSu1kJVjJM0tqbP7B1xQRJvd5+ToaHoM2if72dolwYYz2Ue0mK2sDHeluLN2RNW5LO5RCaTc5j5L/OXEDPZYGC3U39SyPfFnTWN0syLj26wWiFITS2dkZY8aMQYsWLT7AgY6dx44di7CwmPJ78TUiobQzSTuWa9asETuVlBydyKrSaNeS0hMFBgaKHUoak4J+jAODWrZsKUpB0m7mxxoFANF/SqOdVW9vb1y9ehU5csiRmif0RiT8fnmfk0zmN9W4cN557g7avItSX9+9HApmt44k9/H5KT/XB4HYSZ+1qg//+k003AfGkKoFbUri1uPn+GlVCDKmSioq52jdKk7ciUv3o6DYT+VNqcxpgWxpsKFHeVXViY5+C9cBG4VMOuqmI29ulkPA+HqFMorMpzvGp1ABvX3h7mR9ZTmZUL73ZykIZapUqUQQjfGxM6lIRLJw4cJitzC+tnbtWkEeS5UqJXYW6f4kVdvx8/OLc4eSxiNy2ahRozh3KF+8eAGS+bE2bNgwEdATu8lEKGPnJPvr+1IoZUWXno2j1Lf84Iu8WazvxyY+v+Xn2iEQcfdfVDZKsq3s0FlaA7rX7eIfQ6rmtiyBO09eYMDqEGROkxSHBmhPKCv/bxci7j7FQL986ODrisnbzmPq9jAUyuGIf7rFZMdQs7n6b0D0WyClgz1Oj6ippmiWFQsByi9KeUaN26ouZVAsp5x3EwNCb6P9u1OoXX0rInfGlFY3p0woJSOUrVq1QurUqUWktnEjwkalGOkY+1ONPkepgRo0aIBFixYZ7lMSmaRUQVTSUWlx3aGk4B7jpOp0h5KILAX0fKxZww5l7JxkKzqVRgkruod4MOI+mv56UEwB1wC2ut9Z6RSOfb9MCUrRQtE8Azfi1Zu3Iifg3X9fYrCFEombYku1SbtFLsyfanmiUwU3/G/rOUzbES6ulNDVErUbHXnT0bdj8iQ4MVT7O6Nq2yOzvKcvXqPA0Pd3VklXme8mGudKphyolAvV2hoTSskIJaX0ofuS+fLlA6UISpIkibjjSMfS9erVE4E1SqNjcOM2a9YskQqI7l/SfUvjo2vKRzl79mxcuHBBRI5T8/f3x+rVq0WUN+1COjk5YdKkSWjXrp14bktR3pfvP0WFie9zkq3uUgZFJX1TjetH5OjlB2g464B4xDWAre1nVj59jYNjSLuxDbzQzFubUqT5Bm/Gs1dvQLlgH0a9tFgicVNQrzF5D87dfoJ+NfKiayV3KBWEKMKWIm3VbvmHbEbUyzfIkNIBR3W4M6q2PTLLM76vq+gp83WhfeH38O3cmFzJQQOqwCmN9VVRYkIpGaGsVMn0ZLo7d77PsXX+/HkULFgQderU+U9ENh15E2GkPJR169YV+SWDgoLQuXNnQTJpV5QaBe7MmTNHpAnKnTu3qNZDdytDQkKsPg/ltYdRKDf+PV7rupWDVw7ruYd44uoj1JuxT8zTAf/KyOpofW+vMi8+CU23N9Fv4fbuPh/ZrlSK0QKHQsO2IPL5a1BN+kdRrzB8XSic0ydHYH91K9OYYgvVeqaaz32qeaB7lTwYu+kM5uyOQMnc6bC8k/qEUrFdryN+UzCxlT6v3kQjz7v7uopNm38oD88scga7GAfKUYAaBapZW2NCKRmh/FwHokAeIoRxNSKMCxYsEDudtHsZHByMrFmzok+fPujWrZvhIxSQQ7kpKbH5s2fP4OvrK466iVya02R0Krr8X2rs+yoVVKUjfzY5f1jiwvrfF69BAQSpkyUBlaxLbM/568zxSe77XwQ8B2/C81fR4gGRu6+LahNAV3zkNtx/+hITGhXCk+evMXJ9KHJnSIFd/Ux/mVZrPutMC8Sp65HoVdUDPavmwegNofgt8CJ8XNLj746l1RrGIEex3RKlHVVX1soFGgdBKabIHOzyKOqlOEWjl43NPX1hZ2d9KaVkXPv1cmMpgnL0Ml7NcWV0qtgpJLb28oVHZusKbHn28o1IN8LJkNX01oQrSyE3hMAvzYqirkalSH3GBOB25AuM/rogol68weiNZ+CaMSV2qFyZxpSZrTd9L05ce4weVfKgdzUPjFgXinn7LqKMWwYs6VDKFBFm9VEq8+TKkAK7dSDQZilrA51z/7ThAyuo+hFVQZK1Ua7SJPZ24j9rbDKu/XrhyIRSJeRldCp6+ysy4n3Zs+19KsAtUyqVLGYxjID1IaDkYCTNtSxFqiQTH163gLhLOW7TWbg7pUJAb3UTiZsyI/Vn7MPxq4+glJ4c9s9pLNh/CeXzZMTCdj6miDCrj2K7JSrxmKVIAunsPmAjXlNY/bu2p18l5MyQIoFYr72ZMq792qMQMyITSpWQl9GpYgch7O5XEbkyyPumqtJUsBhG4KMI1JyyB2dvPRHPtSxFapz78eWbaEzYfA55M6fGll7qJhI3ZeobztqPo5cfonNFN1Byd4o4X3jwsijPR8UP1G6+E3biyoMo3exV2x7Z5SlR9Yqe1ho9LTvOin4yrv16YacboaRKNtWrV0eyZNYX1RXXZMnoVHRcnG9ITIUOant/rIQc6fhNVa8vG4+rPwINZu7DsSuPhCLzW5dEJU8nTZSqOmk3wt+l6qFE5z9vPQ9LVKYxxZjGsw8g6NIDdKzgCv9a+UROzCWHrqCypxPmtS5pigiz+ih5Ly2RON0sRRJIZyWqXjH30IAqyGyF0dPWMl0yrv16YacboaRk5pS6h6rKuLq6iuAZyhlprU1GpzIue0a4HvSvgiyOtkHgrdVPWG99EWjx+yEEht0TSvzZ1hu+Hpk0UUjZGaX64XQaaanKNKYY0/TXAzgY8QAdyrtgYO38+GnlSfx1+Cqq5suMua1KmCLCrD5KmqLCzmmx1gJ5Ls1SJgF09hq6BU9evDZYStWYqCoTN8sgIOPabxlL45eqG6Gk/I8dO3ZE1apVQWmDKDdkunRxZ/OnyGvZm4xOFTvi7/DAqsiUmn9YZPcl1s9yCHRceARbTt8WAyzp4IMybhktN5iR5K+m7UXI9cfoWSWP+FeqTOOV3RHruqtfmSY+g5r/dhD7L9xH27IuGPJVfvRbfgLLj15DjQKZMaeF+oTSb2ogQm9GwlJ5LuOzN6E9p0o5VDFHacGDqyGdFabjsZZ5k3Ht1ws73Qjl+PHjRboeKkuWKFEi8f+4Gj2j1D6yN1mdSil7RvjxD4vsXsT6WRqB3n8fx6rg62KYZR1Lw9slvaWHFPK/nrkPwVceoWslN9glSiQq0+i1Y6fs0rYukxvD6hZA72XHserYdfh5ZcHMb4urjocSVW6ptESqK2zlAo0zGZApJ4dVR5pkSazcKnnVl3Xt1wMx3QglGfv48WNRmcbFxUUkHc+UKe7jp1y5cumBjVljyupUHgM3gYIA+IfFrOnkzjaKwKA1IVh08IqwTssax4Z7i76uSGyfCDN2XkDRnGmxuov6pQ7jm7pW84Kw+/xdtCydCyPqFcQPfwVjzfEbqFMoK6Y3Lxbfx81+rgQBlXPPiEXt1Y8iN1shG/9AydEBuPvkhcHK0BE1kMIhsY1brZ95sq79eiCiK6FUDN69ezfKli2LxImt1+lldSql5BthzT8senzFeEyZEBi78Qzm7IkQKmlZ49j4mJlyqs7efQElcqXDCguUOowP7zbzg7Dz3F1865MTo7/2QvelwVh34gbqFcmGqU2Lxvdxs583nnMAQRcfWCyK3GyFbPwDpcZsx63I5wYrz42qiaSJY0oPc1MfAVnXfvUtjV+iFISS1Fy/fj1Gjx6NkydPilre+fPnF2UQv/766/itkKCHrE5lfEGbf1gkcBRWQVcEpgaEYXLAeaGDlpWjWs4Lwp53u4LJk9gLUkvH7XTsrnVr/8dhBJy5g2bezhjboBC6Lj6GDSE30aBodkxqUkR1db6dexD7wu+jaj4nzG2lfhS56gpbuUAl76diRvjoWlxlzIJzKuvab0GTPypaCkK5du1aNGjQQJBHCsChO5N79uzBunXrsGrVKlGLW/Ymq1MVGbFV1A6mdmGMn6g6w40RSKgIzA2MwKgNZ4T5WlaOarfgMLafJRKXEykd7DF370WUck2Pv77XnlB+/+cRbA29jSYlnDG+USF0WngUm0/fQqPiOfDzN4VVdw2FTFsq6Ed1ha1coJL3UzHj4lg/EafAzTIIyLr2W8baT0uVglAWK1ZMkMnBgwd/oO2IESPEziXdr5S9yepUJUZtw71/Xwr4+IdFdi9i/SyNAOVbpLyL1Hb0qQBXjSpHKdHl3xTPIWrTU6nDsu4ZsLi9+qUO48Ow86Kj2HTqPYHs8OcRbDMimPF93tznCpmuXSgrZljgjqa5+th6/8o/70LEvafCTNpAoI0EbpZDQNa133IWf1yyFIQyefLkCAkJgbu7+weahoWFoXDhwoiKitIDG7PGlNWplBrCie0SIZx/WMyaU+5sewisPX4dPf86LgzTsiRd1yXHsOHkTXxdNDsckyexaKnD+GYt9hH3+yPwnBjbwCu+j5v9XCHT9YtkwxQL3NE0WyEb/4CSRJ/MpPu650fVsnGL9TVP1rVfD1SkIJRubm6YNm0a/Pw+fJPasGEDOnTogBs3buiBjVljyupUyn2apIntcI5/WMyaU+5sewjQThztyFHb/1NlZEubXBMjjSOp06VwsGipw/gMih2EowTpfFcqJ0bVV59QjlofKo74u1d2R5/qeeNTj59/IQJKInkSQ/d1z4ys+YUS+eOfQkDWtV+PWZOCUNJR98KFCzFr1iwR7U1t79696NKlC+rVq4epU6fqgY1ZY8rqVBUm7sTl+1Hi3tbpEfzDYtakcmebQ2B/+D00n3tI2BU0oAqcNCpJ13f5Caw4eg01C2RBhlQOWHzoCirlzYT5bdSvnR3fpCnk9qvC2TCtWVEodxxblc6F4fUKxvdxs59HvXyNQxcfoLRrBiRLwtHGZgNo5geURPL0sdRJEyNkeA0zJXB3cxCQde03xwa1+kpBKJ8/f46mTZuC6nsrl4cp0TntWC5btgwpUshff1pWp1Lq6KZJlhgnh/EPi1pfHJZjnQiE33mCqpP2wMHeDsFDqiFlUm1SlfmvOomlQVTe0ElUq1L+rEfUs5LIvLZXVsz4thi+m3sIe8PvoU3Z3Bj6VQHrnFjW2oCAUpWJ/iFtiiQ4PqQ6o2NBBGRd+y1o8kdFS0EoFe2otjfdpSQy6eXlhXz58umByWeNKatTVZ+8G+dv/4v0KR1wbHC1z7KNP8QI2BICyw5fFQtt9QJZNDNr8JpThmPuLGmS4e8jV1E9f2b82lL9UofxGWW8Wzq7RXE0+/UgDkTcR/tyLhhUJ398H+fnkiNQf8Y+HL/6SGiZMZUDjgzi331LTpmsa78lbf6YbKkIpR4AqDWmrE5Va2ogztyMFLsiVMubGyPACGiPwIh1oYbI7qyOyQ3H30TotG4/rjgpCG21/JnxW8sSUBKPd/R1hb+f9bzEa42btYynVCYifZ1SJ0UQ/+5bdOpkXfstavRHhDOhVAl1WZ1KOf7I6pgMB/yrqGQti2EEGAFzEFAq9FAy8xzpkova2cqRszly1OjrvyoES4OuGBKNN5q1H0cuP0Tnim74saanGkOwDB0RUMp8kgrZHJNhP//uW3Q2ZF37LWo0E0rLwiurU309cx+CrzwSi9jeHytbFgSWzggwAnEiMHHLWVG/u1jOtMiZPoVFa2fHNwVKPXMlKEj5jehWyR19a3AUdnz4yf5cucJAejqnT47A/vy7b8k5k3Xtt6TNH5PNO5QqoS6rU30zez8OX3qI3BlSYFe/SipZy2IYAUbAHAQmbzuPqdvD4JXdEbkzprRo7ez49Bqy9hT+PHAZvh6Z8Gdbb9Sbvhcnrj1Gjyp50LuaR3wf5+eSI6AEWZGaLhlTYmffipJrbN3qybr264EqE0qVUJfVqZr+egAHIx7ALVNKbO/DPywqTTeLYQTMQmD6jjD8vPU8PLOkhptTKkOS88kWqJ0dn2LD/jktEquXc8+IRe19UGdaIE5dj0Svqh7oWTVPfB/n55Ij0GpeEHafvyu05N99y0+WrGu/5S3/7whSEMqLFy/ixx9/xKlTp0AphGK3iIgIPbAxa0xZnUp5W82bOTW29PI1yybuzAgwAuogMHv3BYzbdBbuTqmQxymVKH3YsFgO/K+x+rWz49N45PpQ/L73Isq4ZcCSDqWgBO71re6BbpWZUMaHn+zP2y44jB1n7wg1+Xff8rMl69pvecslJZTlypXDzZs30bhxYyRLluw/Wg4dOlQPbMwaU1anUt5W82dNg409y5tlE3dmBBgBdRCYGxiBURvOiKsnebOkxpbTt9G4RA5MaKQ9oRyz8Qx+3RMBH5f0+LtjaSiVVfrXzIsuFT8sf6uO9SxFSwTa/3EEAWduiyH5d9/yyMu69lveckkJZcqUKREYGIhixYrpgYEqY8rqVO0WHMb2s3dQKIcj/ulWThVbWQgjwAiYh8Af+y9h6D+nkT1tcuTLmkYs+E1LOmNcw0LmCVKh99hNZzBndwRK5k6H5Z3KQKn97F/LEx0ruKkwAovQE4FOC49i8+lbQgX+3bf8TMi69lveckkJJRHJKVOmwNfXeo9kZXWqjguPiN2QojnTYnWXmLKW3BgBRkBbBBYfuoyBq08hc5qkKJjNUbzkNffJiTFfq187Oz7LJmw+i5m7LqB4rnRY2bkMKv+8CxH3nmJQ7XxoX941vo/zc8kR6Lr4GDaE3BRaFnFOizVd+XffklMm69pvSZs/JluKO5QHDx5E586d0aNHD7i4uMDOzu4Dfa2BaMrqVMqPi7IboYeT8ZiMQEJHgKrz9F95EhlSOsArhyN2nbuL70rlxKj62hPK/209h2k7wg1ko8LEnbh8PwpD6uRH23IuCX2qrN7+7kuDRRYBaiVypcOKzmWs3iaZDZB17dcDMykI5e+//y4I+ChV4wAAIABJREFU5evXr/+DAdX2fvPmjR7YmDWmrE7VY2kw/jlxA6Vc0+Ov70ubZRN3ZgQYAXUQWHXsGnovO4E0yRKjsHNaBIbdQ6vSuTC8XkF1BjBDyqRt5/HL9jDDcWi58Ttw7eEzDK9bAK3K5DZDEneVEYFefx/H6uDrQjXlnqyMetqKTrKu/XrgKwWhdHZ2Rp06dcQOZYoUKf6DQ65cufTAxqwxZXWq3n8fx6rg64YUIWYZxZ0ZAUZAFQTopY5e7lI42KNYznTYG34PrcvkxrC6BVSRb46QqQFhmBxwHgWypcGGHuVRZux23Hj8HCPrF0SLUvL/1ppja0Ls22fZCaw8dk2YXtY9Axa3L5UQYdDMZlnXfs0AMBpICkJJQTmnT59G7tzW+3Ysq1P1W34Cy49eQwWPTPijrbcePsZjMgIJHoFNITfRefExONjbibuLByLuo105Fwyuk19zbJScmBQctKlnefiMCcDtyBfiPifd6+Rm3QgotdrJivJ5MmJhOx/rNkhy7WVd+/WATQpC2ahRI/j5+aFt27Z6YKDKmLI61cDVIVh8iOr2ZsbcViVUsZWFMAKMgHkIbAu9jQ5/HkGiRIB37vQ4dPEBvvd1xQC/fOYJUqH3zF3hmLD5nCFHYYlRAbj37wuMb+iFJiWZUKoAsa4ilFrtpIRSXlNXhWx8cFnXfj1gl4JQTp8+Hf7+/qhVqxby5s2LJEmSfIDFkCFD9MDGrDFldaojlx5g2LrT+KGKB6rmz2yWTdyZEWAE1EFg57k7aDP/sBBGO5RHLz9Exwqu8K+lPaE0TrIe0LsCio3chgdPX2Jio0L4poSzOgazFN0QUGq1kwK8kWD5aZB17be85f8dQQpCSZHdH2sUlMOVcvRwDR6TEWAE1EJgb9g9fPf7ISGuYPY0otRhl4pu6F/TU60hTJbz254IjN54Bq4ZU2JH34ooPHwrHj97hUmNC6NBsRwmy+GOciKglNYk7WoWyILZLYrLqaiNaMWE8v1ESkEobcGv2KlsYRbZBkbAMggcjLiPpr8eFMKp9GLYnX/RrZI7+tbIa5kBPyGVyi5S+UWq2rOrXyV4Dd2CJy9eY2rTIqhXJLvm+vCA6iIwYl0o5u27KITW9sqKGd9ab8EQdZGxjDRe+5lQqu5Z7FSqQ8oCGQGbQeDo5QdoOOuAsCdXhhQi72OPKnnQu5qH5jYu2HcRw9aFwjl9cgT2r4z8QzYj6uUbTGtWFF8Vzqa5PjyguggopTVJat3C2fBLs6LqDsDSPkCA137JCCUlMqej7Y81zkPJ32BGgBGwZgROXH2EejP2CROypEmGW5HP0auqB3pWzaO5WX8euIQha2PKQO77qTLyDtqEF6+jMfPbYvDzyqq5PjygugiM23QWdE+WWoOi2TGpSRF1B2BpTCg/4gNSHHkvWLDgA0L56tUrhIWFgf590qRJ+Pbbb6V3YX5LkX6KWEFGQDcEQm9Ewu+XQDF++pQOIgimTzUPdK+iPaFcdPAyBq05JYjtwQFVkGfgRrx68xazvyuOmgWz6IYRD6wOAhO3nMWMnTGEslHxHPj5m8LqCGYpcSLAa/97WKQglB/z0yVLlmDhwoXYtGmT9K7MTiX9FLGCjIBuCITdfoJqk/eI8VM62OPpyzfoVyMvulZy11ynpUFXQKllnFInRdDAqnD134Dot8BvLUugGmeC0Hw+1B5QqYREcpuWdMa4hoXUHoLlGSHAa7+VEMrLly8jf/78ePr0qfQOzE4l/RSxgoyAbghcvPcUlX7eJcZPYp9I7Aj+WNMTnSu6aa6TUlc8YyoHHB5YFS7+G4UO81qXQGVPTi2m+YSoPKBSCYnEfuuTE6O/1r5evMomSS2O134rIZTjxo3DjBkzcPXqVakdipRjp5J+ilhBRkA3BK4+iEL5CTs/GH+Anye+99WeUC4/chX9VpxEuhRJcGRQNbgNiCGUC9qURMW8TrphxAOrg4BSCYmk6VUvXh1LrEMKr/2SEUrKQ2kclPP27VtERkbi0aNHGD16NH766SfpPYudSvopYgUZAd0QuPX4OUqN3f7B+INq50P78q6a67Tq2DX0XnYCjsmTiB1Kj0ExV4oWtvNG+TyZNNeHB1QXgVm7LmD85rNCaJuyuTH0K+3rxatrkdzSeO2XjFAOGzbsP1HeDg4OKFu2LHx9feX2pnfasVNZxTSxkoyALghQaUMqcWjchtTJj7blPl7UwVKKrj1+HT3/Oo7USRPj8KCq8By8WQy1pL0PyrhntNSwLFcjBH7dcwFjNsYQyg7lXTCwtvb14jUyVYpheO2XjFBK4RVfqAQ71RcCyB9nBGwYgcdRr1B4xNYPLBxetwBalcmtudXrTtxA96XBSOFgL3YoCwzdInT46/tSKOWaQXN9eEB1EVAS15PUThXc8FMt7asxqWuR3NJ47ZeQUO7btw979+7Fy5cvQUfeH7zJcy1vub9RrB0jwAh8EoGnL14biJvScWS9AmhRWntCueHkTXRdcgxJE9uJHcpCw2KI7vJOpVEyd3qeSStHQElcT2boVY3JyiE0S30mlJIRSgq+GTBgANKlS4c0adJ8MJnm1PIeNWoUAgICsGtXTDQltTZt2oh8lsYte/bsIoiGWnR0NIYPH465c+fi4cOHKFeuHGbOnAl3d/PSebBTmfUd5M6MQIJC4MXrN8g7KOZoWWmjvy6Ib31yaY7D5lM30WnRMTjY2yFoYBUUGbFN6LCycxkUz5VOc314QHURWHjgEgavPS2E6lWNSV2L5JbGa79khDJHjhxo27YtRowY8dmeM2XKFPTu3VvcuTQmlMWLF0fNmjXRvXt3g2x7e3tkyhRz+ZzIJBHI+fPng4hm//79ERERgdOnT4PucZra2KlMRYr7MQIJD4Ho6LdwfRdNrVg/toEXmnnn1ByMradv4fuFR2FvlwhBA6qg+Lu7nWu6lkUR57Sa68MDqovAkkNXMGB1iBBKpT2JVHKzHAK89ktGKFOkSIGTJ0+avStIZly/fh3t27dHYGAgnJ2dkTlzZgOhpJKNqVKlwtKlS1G/fv3/eBQdr2fMmBETJkxAp06dxHOKLM+WLRvmzZuHpk2bmuyF7FQmQ8UdGYEEiYCSQFwxfkLDQmhc0llzLAJCb6P9n0fEuLRD6T06Jvp8Xbdy8MrhqLk+PKC6CPx9+Ap+XBlDKPVKnq+uRXJL47VfMkJZvXp1cTTdrFkzsz1n3bp1gjDScTftcF66dMlAKM+ePYt8+fLh3Llz8PDw+I/soKAg+Pj4/Oc5HXsXKlRI7Fya2tipTEWK+zECCRMBSs/z8nW0wfiJjQrhmxLaE8qdZ++gzYLDQo8D/pVReuwO8ecNPcqhQDYmlNbunSuOXkPf5SeEGf61PNGxgva5Tq0dQ3P057VfMkJJ9xfpqJlIpaenJ5ImTfrBfLZs2dKk+W3duvUHhHL58uVo0qSJ2H2k8o12dnbw8/MT5NPR0RGrVq1Cw4YNERUVheTJkxvGaNy4sfi39evXf3RcypNJ/ynt5s2b8Pb2FknY6QifGyPACDACxggUGLJZlFxU2qTGhdGgmPa/FbvP30WreUFCjT39KsF3YkzC9c0/lIdnlg/vsPMMWh8Cq4OvodffMYRSr1yn1ofa52vMhFIyQklE72ONgnLo6NqUFptQDh06VCRGHzNmjCCSYWFh6Nu3rzga37FjB6hWeIsWLYR8Yx2IwN64cUME+HysUe5Mun8ZuzGhNGWmuA8jkPAQKDx8Kx4/e2UwfGrTIqhXJLvmQOwNu4fvfj8kxt3epwKq/G+3+PO2Xr7Ikzm15vrwgOoioOQZJalDv8qPNmW1z3WqrkVyS2NCKRmhVMtdYhNKSj/0+PFjpE37/qL5wYMHUbp0adD/yREaNWoU5w7lixcvsHbt2o+qxjuUas0ay2EEEgYCJUZtw71/XxqM/aVZUdQtnE1z4/eH30PzuTGEknYla04JNJBLt0ypNNeHB1QXASUtFEnVKzWVuhbJLY0JZQIhlHG54dOnT0WgzooVK8ROJd2hDA8Ph5vb+3smdIeycOHCoo64qY2dylSkuB8jkDARKDVmO25FPjcYP6N5MdQulFVzMA5G3EfTXw+KcSkQ56vpe8Wfd/WtiNwZU2quDw+oLgJKWiiSOuZrLzT30T6TgLoWyS2N1/4EQiibN2+O+/fvY8uWmEoQ4kdz1y5UqlQJoaGhcHV1hZOTEyZNmoR27dqJ5xzlLfeXl7VjBKwVgXLjd+Daw2cG9Wd9Wwy1vLQnlIcvPcA3sw8IPSj3ZMNZ+8WfA/tXgnP6FNYKL+v9DgElLRT9dXxDLzQpyYTSks7BhDKBEEoKqqlbty5GjhwpUgCdP38eXbt2FUfeixcvFigMHDgQc+bMEWmCcufOjX79+onAnpCQEM5DaclvIctmBBIYApV+3oWL954arJ7TojhqFMiiOQpHLz80kMi/vy+FJu92K/f9VBnZ074PTtRcMR5QFQR2nL2Ntgti0kL9/E1hNCqufeCXKoZYiRAmlAmEUIo38JUrRWAOpRCiu5S0a0lR3smSJRMoUEAOVemhxObPnj0TidHpqJvIpTmNncoctLgvI5DwEKg2aTfC7vxrMPy3liVQLX9mzYEIvvIQX8+M2ZVc1M7HEKBz0L8KsjjG/C5ys14Edp27g9bzY9JCTWlSBPWLah/4Zb3oma85r/02SijNdwX1PsFOpR6WLIkRsEUEak0NxJmb71ONzWtdApU9tSeUJ689Qt3p+wTE81uXNOSkpCTnTqmZUFq77xlH8U9rVhRf6RD4Ze0YmqM/r/1MKM3xF5P6slOZBBN3YgQSLAL1pu/FiWuPDfYvaFMSFfM6aY7HqeuPUWdaTCDOry2KizKM1I4OqooMqT7MAay5cjzgFyOw/8I9NP8tJop/5rfF4KfDPd0vNsKKBPDaz4RSdXdlp1IdUhbICNgUAhT8QvcXlfZnW2/4emTS3EbaJaXdUmoUad51yTHx5+NDqiFtCgfN9eEB1UXgUMR9w71Yve7pqmuR3NJ47WdCqbqHslOpDikLZARsCoEmcw7g0MUHBpsWt/dBWfeMmtt47tYT1JiyR4xLd+x++Pu4+POJodXhmDyJ5vrwgOoicPTyAzScFRPF/3urEqiST/trFepaJLc0XvuZUKruoexUqkPKAhkBm0Lgu7mHsDf8nsGmJR18UMZNe0IZfucJqk6KIZQTGhVC/xUnxZ9PDa+BVEkT2xTmCdEY46Arva5VJCTcee1nQqm6v7NTqQ4pC2QEbAqB1vODsOvcXYNNlLLHxzWD5jZG3P0Xld+VW6TE1wNWhwgdzoyoieQO9prrwwOqi0DItceGZPUL23mjfB7tr1Woa5Hc0njtZ0KpuoeyU6kOKQtkBGwKgfZ/HEHAmdsGm5Z3Ko2SudNrbuOle09R8eddYtzhdQtg6D+nxZ/PjaqJpImZUGo+ISoPePrGY9T+JSboSq9dcJVNklocr/1MKFV3UHYq1SFlgYyATSHQedFRbDp1y2ATVakpniud5jZefRCF8hN2inEH1c6HURvOiD+Hj66FxPZ2muvDA6qLwNlbkYb67Ms6loa3i/YvLepaJLc0XvuZUKruoexUqkPKAhkBm0Kg+9JgrDtxw2DTmq5lUcQ5reY2Xn/0DGXH7RDj9q+ZFxM2nxN/vjjWD4kSJdJcHx5QXQSM78jq9dKirkVyS+O1nwml6h7KTqU6pCyQEbApBHr/fRyrgq8bbPqnW1kUyqE9obz1+DlKjd0u9OhdzQOTtp0H8ciLY2vbFN4J1RjjO7J6vbQkJOx57WdCqbq/s1OpDikLZARsCoH+K05g2ZFrBpvWdy+HgtkdNbfxTuRzeI+JIZTdK7tj2o5wJLZLhPAxfprrwgOqj8Dl+09RYWLMHVm9fEx9q+SVyGs/E0rVvZOdSnVIWSAjYFMIUDT1kkNXDDZt7FEe+bOl0dzGe/++QIlRAWLcjhVcMWd3BBwS2+H8qFqa68IDqo/AtYdRKDc+5o7spp7lkS+r9j6mvlXySuS1nwml6t7JTqU6pCyQEbApBIauPYU/Dlw22LTlB1/kzZJacxsfPn2JoiO3iXHblnXBvH0XkTyJPc6MrKm5Ljyg+ghER79F4zkH8PTlG6ztWla8LHCzHAK89jOhVN272KlUh5QFMgI2hcDI9aH4fe9Fg03bevkiT2btCeXjZ69QePhWoUfL0rnw54HLIqE5JTbnZhsIvH37VhjCQVaWn09e+5lQqu5l7FSqQ8oCGQGbQmDspjPieFlp2/tUgFumVJrb+OT5K3gNiyGUzbydsTToKtIkS4yTw5hQaj4ZPKDVI8BrPxNK1Z2YnUp1SFkgI2BTCPy85Rym7ww32LSrb0XkzphScxujXr5G/iFbxLgNi+XAymPXkC5FEgQPqa65LjwgI2DtCPDaz4RSdR9mp1IdUhbICNgUAlMCzmNKQJjBpj39KiFnhhSa2/j81Rt4Dt4sxq1XJBvWHr+BjKkccGRQNc114QEZAWtHgNd+JpSq+zA7leqQskBGwKYQmLEzHBO3xCQRp7b3x0rIkU57QvnqTTTyDNwkdPDzyoKNIbfglDopggZWtSm82RhGQAsEeO1nQqm6n7FTqQ4pC2QEbAqBObsvYOymswab9v9UGdnSJtfcxjfRb+E2YKMYt2q+zKK+eFbHZDjgX0VzXXhARsDaEeC1nwml6j7MTqU6pCyQEbApBCjCmyK9lXZoQBVkTpNMcxspAtjFP4ZQVvDIhN3n7yJ72uTY91NlzXXhARkBa0eA134mlKr7MDuV6pCyQEbAphBYeOASBq89bbDp8MCqyJQ6qS42uvpvQPRboKx7BuwLv4+c6VNgT/9KuujCgzIC1owAr/1MKFX3X3Yq1SFlgYyATSGwNOgK/FeFGGw6OqgqMqTSh1DmGbgRr968hbdLegRdfACXjCmxs29Fm8KbjWEEtECA134mlKr7GTuV6pCyQEbAphBYfuQq+q04abDp+JBqSJvCQRcbPQZtwsvX0SiaMy2CrzyCW6aU2N6HCaUuk8GDWjUCvPYzoVTdgdmpVIeUBTICNoXAmuDr+OHv4wabTgytDsfkSXSxMd/gzXj26g0KZk+DU9cj4ZE5Fbb2qqCLLjwoI2DNCPDaz4RSdf9lp1IdUhbICNgUAutP3kC3JcEGm6jUIZU81KMVHLoF/754Dc8sqXH21hPx/80/+OqhCo/JCFg1Arz2M6FU3YHZqVSHlAUyAjaFwOZTt9Bp0VGDTaEjaiCFgz6E0mvYFjx5/hqumVIi4u5TFMiWBht6lLcpvNkYRkALBHjtZ0Kpup+xU6kOKQtkBGwKge1nbqPdH0cMNp0dWRPJktjrYmOREVvxKOqVOHJ//OyVCM5Z1rG0LrrwoIyANSPAaz8TStX9l51KdUhZICNgUwhQvsdW84IMNoWNroUk9na62Fh85Dbcf/rSMHaj4jnw8zeFddGFB2UErBkBXvuZUKruv+xUqkPKAhkBm0Jgf/g9NJ97yGDThTF+sLdLpIuNJUcH4O6TF4ax+1TzQPcqeXTRhQdlBKwZAV77mVCq7r/sVKpDygIZAZtCgPI9Np5zwGDTxbF+SJRIH0LpMyYAtyPfE8pfmhVF3cLZbApvNoYR0AIBXvuZUKruZ+xUqkPKAhkBm0Lg2JWHaDBzv7CJeOTFsbV1s6/M2O248fi5Yfy1XcuisHNa3fThgRkBa0WA134mlKr7LjuV6pCyQEbAphAIufYYX03fK2xKbJcI4WP8dLOv3PgduPbwmWF8PZOs6wYCD8wIqIAAr/1MKFVwow9FsFOpDikLZARsCoEzNyNRa2qgsMnB3g7nR9fSzb4KE3fi8v0oMX6aZIlxclgN3XThgRkBa0aA134mlKr7LzuV6pCyQEbAphAIv/MEVSftETYlS2KHsyP1I5SVft6Fi/eeCl28sjtiXfdyNoU1G8MIaIUAr/1MKFX3NXYq1SFlgYyATSFw6d5TVPx5l7AppYM9To+oqZt9Vf63CxfuxhDKOoWyYnrzYrrpwgMzAtaMAK/9TChV9192KtUhZYGMgE0hcO1hFMqN3ylsSp00MUKG63fMXH3ybpy//a/QpWslN/Sr4WlTWLMxjIBWCPDaz4RSdV9jp1IdUhbICNgUArcjn8NnzHZhE1WoOTG0um721ZyyR9TwpjahYSE0Lumsmy48MCNgzQjw2s+EUnX/ZadSHVIWyAjYFAL3/32B4qMChE3pUiRB8BD9CKXf1ECE3owUuvz1fSmUcs1gU1izMYyAVgjw2s+EUnVfY6dSHVIWyAjYFAJUM7vw8K3CpoypHHBkUDXd7Ptq2l6EXH8sxj/gXxlZHZPrpgsPzAhYMwK89jOhVN1/2alUh5QFMgI2hcCzl2+Qb8hmYVOm1ElxeGBV3eyrN2MfTlx9hKSJ7XBmRE3Y6VQCUjcAeGBGQCUEeO1nQqmSK70Xw06lOqQskBGwKQRevYlGnoGbhE1Z0iTDwQFVdLOvwcx9OHblEfI4pcK23hV004MHZgSsHQFe+5lQqu7D7FSqQ8oCGQGbQuDt27dw8d8obMqeNjn2/VRZN/sazdqPI5cfomo+J8xtVVI3PXhgRsDaEeC1nwml6j7MTqU6pCyQEbA5BNwHbMTr6LfIkS459v6oH6FsNS8Iu8/fRUdfV/j75bM5nNkgRkArBHjtZ0Kpuq+xU6kOKQtkBGwOAc/Bm/D8VTRyZUiB3f0q6Wbf8auPsOLoVXSrlAdZHJPppgcPzAhYOwK89tsooRw1ahQCAgKwa1dMNQpqx48fR8+ePXHkyBFkyJABPXr0QN++fQ3Po6OjMXz4cMydOxcPHz5EuXLlMHPmTLi7u5vl5+xUZsHFnRmBBImA19AtePLiNVwypsTOvhUTJAZsNCNgSwjw2m+DhHLKlCno3bs3fH19DYTy/v378PT0RP369cWzgwcPokuXLoIwtmnTRqBAZJL+Pn/+fGTPnh39+/dHREQETp8+DQcHB5P9np3KZKi4IyOQYBEoOmIrHka9glumlNjehwllgnUENtxmEOC134YI5fXr19G+fXsEBgbC2dkZmTNnNhDKsWPHYsaMGbh06RISJ04srB4wYABWrVqFs2fP4uXLl8iYMSMmTJiATp06ieePHj1CtmzZMG/ePDRt2tRkp2enMhkq7sgIJFgESo4OwN0nLzi6OsF6ABtuawjw2m9DhHLdunVYunQp6Lh7xIgRgjwqR95+fn5ImzYtlixZYrCYjsSrVauG27dvi74+Pj44d+4cPDw8DH3o2LtQoUJi59LUxk5lKlLcjxFIuAiUGbsdNx4/h2eW1Nj8g2/CBYItZwRsBAFe+22IUBr7ZOvWrT8glEQKa9WqhfHjxxu6hYaGokCBAjh8+DCuXLmChg0bIioqCsmTv68U0bhxY/Fv69ev/6jLR0ZGgv5T2s2bN+Ht7Y2rV68iR44cNvJVYTMYAUZATQR8J+zElQdRyJ81DTb2LK+maJbFCDACOiDAhDKBEEoKrGnevLnYuVQa3Y90c3MTR+S0Q9miRQu8efMGdnZ2hj4tW7bEjRs3RIDPx9qwYcPE/cvYjQmlDt9oHpIRsBIEKv9vFyLuPkXB7GmwvjsTSiuZNlaTEfgoAkwoEwih9PLyAh17x7VDeezYMRF806hRozh3KF+8eIG1a9fyDiX/kDACjIBqCNSYvAfnbj9B4RyOWNutnGpyWRAjwAjogwATygRCKIlMUqqghQsXGiyO6w5leHi42LVUGt2hLFy4sAjoMbWxU5mKFPdjBBIuArV/CcTpG5Eo4pwWa7qWTbhAsOWMgI0gwGt/AiGU48aNw+zZs3HhwgXY29sLq/39/bF69WoR5U27kE5OTpg0aRLatWsnnnOUt418y9kMRkBCBOrN2IcTVx+heK50WNm5jIQaskqMACNgDgJMKBMIobxz547IQ1m3bl2RXzIoKAidO3cWJLNVq1YChYEDB2LOnDkiTVDu3LnRr18/cbcyJCSE81Ca863ivowAIxAvAkoN7ZK502F5JyaU8QLGHRgByRFgQplACCWZSdHcVB0nODgYWbNmRZ8+fdCtWzcDAhSQQ7kpKbH5s2fPRGJ0OuomcmlOY6cyBy3uywgkTASa/noAByMewMclPf7uWDphgsBWMwI2hACv/TZKKPX0UXYqPdHnsRkB60Cgxe+HEBh2D2XcMmBJh1LWoTRryQgwAh9FgNd+JpSqfz3YqVSHlAUyAjaHQJv5Qdh57i7KuWfEovY+NmcfG8QIJDQEeO1nQqm6z7NTqQ4pC2QEbA6BDn8ewbbQ2/D1yIQ/23rbnH1sECOQ0BDgtZ8Jpeo+z06lOqQskBGwOQS6Lj6GDSE3UTFvJixow4TS5iaYDUpwCPDaz4RSdadnp1IdUhbICNgcAj3/Csba4zdQxdMJv7cuaXP2sUGMQEJDgNd+JpSq+zw7leqQskBGwOYQ6LPsBFYeu4Zq+TPjt5YlbM4+NogRSGgI8NrPhFJ1n2enUh1SFsgI2BwCs3dfwLhNZ9Gxgiv8a+WzOfvYIEYgoSHAaz8TStV9np1KdUhZICNgcwhER79F6M1IeGZJjcT2djZnHxvECCQ0BHjtZ0Kpus+zU6kOKQtkBBgBRoARYASkRoDXfiaUqjsoO5XqkLJARoARYAQYAUZAagR47WdCqbqDslOpDikLZAQYAUaAEWAEpEaA134mlKo7KDuV6pCyQEaAEWAEGAFGQGoEeO1nQqm6g7JTqQ4pC2QEGAFGgBFgBKRGgNd+JpSqOyg7leqQskBGgBFgBBgBRkBqBHjtZ0KpuoOyU6kOKQtkBBgBRoARYASkRoDXfiaUqjsoO5XqkLJARoARYAQYAUZAagR47WdCqbqDslOpDikLZAQYAUaAEWAEpEaA134mlKo76KVLl+Di4oKgoCBkzZpVdfkskBFgBBgBRoARYATkQuDmzZvw9vbGxYsXkTt3brmU01ibRG/fvn2r8Ziq/TNvAAAMfUlEQVQ2Odzhw4eFU3FjBBgBRoARYAQYgYSFAG0mlSxZMmEZHctaJpQqTf/z588REhKCTJkyIXHixCpJjRGjvAHx7qeqsGoqjOdQU7hVH4znT3VINRfIc6g55KoPKOMcvn79Gnfv3oWXlxeSJUumus3WJJAJpRXMFt/RsIJJikdFnkPrnkOeP+ueP9Ke55Dn0PoRkNsCJpRyz4/Qjn8IrWCSmFBa/yR9wgL+Dlr/9PIc8hxaPwJyW8CEUu75YUJpBfNjioq8mJmCkrx9eP7knRtTNeM5NBUpefvxHMo7N6QZE0q550doFxkZiUmTJqF3795IkyaNFWjMKsZGgOfQun2C58+6549/R61//ngO5Z9DJpTyzxFryAgwAowAI8AIMAKMgNQIMKGUenpYOUaAEWAEGAFGgBFgBORHgAml/HPEGjICjAAjwAgwAowAIyA1AkwopZ4eVo4RYAQYAUaAEWAEGAH5EWBCKfEcRUdHY/jw4Zg7dy4ePnyIcuXKYebMmXB3d5dY64St2uXLl+Msv/Xbb7+hffv2OH78OHr27IkjR44gQ4YM6NGjB/r27ZuwQZPE+lGjRiEgIAC7du0yaBTffPF3VJLJe6dGXHPYpk0bLFiw4ANFs2fPLtKxUeM51H8OHzx4gAEDBmD9+vUiCLVQoUIYN26cWPOo8fdQ/zkyRQMmlKagpFMfIpNEIOfPnw/6Aezfvz8iIiJw+vRpODg46KQVD/spBNatW4fGjRuLeUqUKJGhq6OjI6KiouDp6Yn69euLiP2DBw+iS5cuYo5p0eOmHwJTpkwRc+Lr62sglPfv3493vvg7qt+cxR45rjmkPsWLF0fNmjXRvXt3w0fs7e1FVTNqPIf6z2H16tVx584dTJ8+XczLjBkzxEbKsWPHxN/j+93kOdR/DkkDJpRyzMN/tHj58iUyZsyICRMmoFOnTuL5o0ePkC1bNsybNw9NmzaVVPOErdaYMWOwbNky8UYdu40dO1b8UF66dMlQnpPeyletWoWzZ88mbOB0sv769eti5zgwMBDOzs7InDmzgVDGN1/8HdVp0mIN+6k5fPPmDVKlSoWlS5eKF7nYjedQ/zkMDw9Hnjx5sG/fPpQpU0Yo9PbtW3h4eKBZs2ZInjz5J383eQ71n0NFAyaU8szFB5pQ3W4fHx+cO3dOfLGURkcAdBxAu1rc5EOAiL6dnR2WLFnyH+X8/PyQNm3aD57REWu1atVw+/ZtODk5yWeQjWtEO8pENuiodMSIEYLsK0fe8c0X9eXvqP4O8qk5pBe1fPny/ed3VNGaf2f1nz/aKNm/fz+qVKmCpEmTGhSiq120s0ynPZ/63eTvof5zyIRSnjmIUxPatWrYsKE4JqU3NKXRcSr9G9014SYfAgUKFEDWrFnx4sULnD9/Xrx5Dx48GDVq1BAvArVq1cL48eMNioeGhoI+c/jwYZQoUUI+gxKQRq1bt/6AUMY3X1euXOHvqGT+EXsOly9fjiZNmohTnk2bNomXPXpRoBcIuobCv7OSTeA7dWjeaK37559/MHDgwE/+bvL3UJ455B1KeebiA00WLVqEFi1agI5s6EdQaS1btsSNGzdE8AA3uRCgo5eUKVOiaNGimDhxojhqo3mcOnUqtm3bho4dO6J58+ZiJ0xp9Pbt5uYmjlyVC+hyWZVwtIlNRmiH5FPzRTsj/B2Vyz9iz+HQoUMxevRo0FUUIpJhYWEiCI6uN+zYsUOcFvAcyjWHdPRNO5O0Y7lmzRoRhMrfQ7nm6GPaMKGUdJ5WrlyJRo0axblDSbtfa9eulVTzhK3W06dPxf1I46Mb+nGkAB2KKqVFLa4dSrp8TkSUm34IxCYjXl5en5wvehng76h+8xXXyLHnkO7iPX78WByZKo2C4UqXLi2C4ug7yXMozxzSukbksVSpUuIUjk7n+Hsoz/zEpwkTyvgQ0um5creHLizTDpbSaBercOHC4pIyN+tAoF+/fti8ebPYFaFUQQsXLjQoznco5ZnD2GSEyP+n5ku5u8XfUXnnMC7N6KWPTg9WrFghvpN0D5bnUP85pAhvSqnWoEEDcbKjvJTz91D/uTFVAyaUpiKlcT/ahaQgjUmTJqFdu3ZidI7y1ngSzBzu5MmTIkqRyKPx8XWlSpVE9HCRIkUwe/ZsXLhwAZS2hJq/vz9Wr17NUd5mYm2J7rEJJeXB+9R88XfUErPwZTJjzyHtdlH6py1bthgEU9AVfSfp/rKrqyv/zn4Z5Kp8etasWSKFGuXlnTx58gfXvPh7qArEmghhQqkJzJ83CF1GnjNnjkgTlDt3btBOF+2KhISEcB7Kz4PUop+iBMlEKGkHhH4gKe3Tr7/+KiLyKeiGSCXlU6tbt67IKUq70J07dxakpVWrVhbVjYXHj0BsMkJ58eKbL/6Oxo+rlj1izyEdm9L3beTIkSLVGgXKde3aVRx5L168WKjGc6jlDP13LJqTggULok6dOv/JXkJH3vTixt9DfefI1NGZUJqKlA79KCCH8hRSYvNnz56JpMt01E3kkpucCNy9exc//fQTNm7cKHaUixUrJio+lC9fXihMxJLewoODg0U0eJ8+fdCtWzc5jUlgWsUmI6bMF39H5XKSuOaQ7qNTYA6lEKK7lLRrSVHeyZIlE8rzHOo7hxQwRaQ+rkYv2lTlKL7fTZ5DfedQGZ0JpRzzwFowAowAI8AIMAKMACNgtQgwobTaqWPFGQFGgBFgBBgBRoARkAMBJpRyzANrwQgwAowAI8AIMAKMgNUiwITSaqeOFWcEGAFGgBFgBBgBRkAOBJhQyjEPrAUjwAgwAowAI8AIMAJWiwATSqudOlacEWAEtEaAMixQJPGwYcMsMrSSI/HixYsfzeZgaR0sYhgLZQQYAZtHgAmlzU8xG8gIMAJqIUBpoSg3HlVasUSjevAPHjxApkyZDMnvY4/DhNISyLNMRoAR+FIEmFB+KYL8eUaAEWAENESACaWGYPNQjAAjYDICTChNhoo7MgKMgLUgkChRIpEsmeqmU6WN3bt3w8XFBYMHDxZ1gh8/fiyqc4wYMQLVq1c3mLV161ZRDlMpy0eJ56n0qXIEHZvMbdiwQVRhOXXqFNKkSYNmzZqJJNpK0mzSg6olLVu2DHv37kX69OlFInsaI64W+8ib9KRE+GvXrhXVsajQwZQpUyx67G4tc8x6MgKMgFwIMKGUaz5YG0aAEVABASJyVPpy06ZNeP36NUqVKiUqpJw+fVoQshw5cmDdunWiqhHVUq9duzaOHz+OkiVLolevXoJE0t+pNObDhw/jJJRr1qxBw4YNxX3Kxo0bIywsTPT39vYGVWehRno4Ojpi+vTp4t+J4FKVlj179hiqJxmbG5tQ1qhRA1euXBGkNHXq1KKy0o4dOzB06FCL3eNUAX4WwQgwAgkQASaUCXDS2WRGwNYRICJHxHDSpEnC1PDwcOTJk0eUcCtRooTBfCrtdvnyZRCRoz9TXeEDBw4Ynk+bNk3sEMa1Q+nj4wNnZ2esWLHC0J9IKtWOph3OfPnyCULZs2dPQWKVRruU/fr1i3OX0phQKjWMAwICUKVKFfHx27dvI1euXIIIWyowyNZ9g+1jBBgByyDw//bOXkVhKIjCg3Za2giBVFaCqE9glVJQIWirhVpaiJWNXV5AUURslYCVD2Eh1pY+gr0IyxlISDRY7Lr4w5nGwsvk5uQWH3NmEgLl/+jKrFSACrxQAYDcZDLRiiHCdV2tIiaTydCuLpeLft8ZoJbL5cSyLB9CsRBVymKxGAmUiURC8B3iXq/n54RFjXyr1Urq9boC5XQ6lU6n46+Bbd5sNrXKeBtBoNzv92LbttrzsNO9yGazei8EyhceMF6aClCBOwUIlDwUVIAKfJ0CALnlcqm9hoj1ei2NRkMOh4Nax8GIx+PaXwlwLJVKoWriI6DEtLfjOFqB9AIT2qlUSi3vWq2mQBncB9Y9GqqJAkpY7oBUL/L5vFSrVQLl151a3hAV+GwFCJSf/fy4eypABSIUuAU59E5iCGe73Wq/pBcY3InFYjpYA/g8Ho+y2+38/2ezmXS73cgKJXoiTdMMWd4YnqlUKiHL+7dAeb1eJZPJhPZ8Pp/FMAy1zFmh5NGnAlTgnRQgUL7T0+BeqAAVeIoCUZXBcrmsFvZ4PFa43Gw2MhgMZLFYqAWNSe1CoSD9fl9arZZCYbvdFrx78nQ6ae9isLoIGx229mg00l/0X8JiRw70UiL+UqHEtWBto+9zPp9LOp3WvksMGg2HQwLlU04Kk1ABKvAsBX4Ail2paLFJU7gAAAAASUVORK5CYII=\" width=\"599.9999869953505\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "ename": "NameError",
     "evalue": "name 'num_probes_kept' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-9-147c62ea5723>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m      7\u001b[0m \u001b[0mplt\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mylabel\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34mf\"num of probes\"\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mlabelpad\u001b[0m\u001b[1;33m=\u001b[0m\u001b[1;36m1\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      8\u001b[0m \u001b[0mplt\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mxlabel\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34mf\"region id\"\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mlabelpad\u001b[0m\u001b[1;33m=\u001b[0m\u001b[1;36m1\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m----> 9\u001b[1;33m \u001b[0mplt\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mylim\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;36m0\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mnum_probes_kept\u001b[0m\u001b[1;33m*\u001b[0m\u001b[1;36m1.1\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     10\u001b[0m \u001b[0mplt\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mshow\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mNameError\u001b[0m: name 'num_probes_kept' is not defined"
     ]
    }
   ],
   "source": [
    "pb_num_dict = {_k:np.sum(_v) for _k,_v in blast_keep_dic.items()}\n",
    "\n",
    "pb_nums = np.array(list(pb_num_dict.values()))\n",
    "plt.figure(figsize=(6,3), dpi=100)\n",
    "plt.plot(pb_nums)\n",
    "#plt.plot(np.ones(len(pb_nums))*20)\n",
    "plt.ylabel(f\"num of probes\", labelpad=1)\n",
    "plt.xlabel(f\"region id\", labelpad=1)\n",
    "plt.ylim([0, num_probes_kept*1.1])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "artistic-packet",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "160"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(kept_pbs[0].seq)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "pacific-bonus",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
