{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "0691c309",
   "metadata": {},
   "source": [
    "# Design CTP-14 human genome DNA-MERFISH library \n",
    "\n",
    "by Pu Zheng\n",
    "\n",
    "2022.10.14\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "137597b3",
   "metadata": {},
   "source": [
    "# Table of contents\n",
    "\n",
    "\n",
    "> 0. [Minimum required packages and settings](#0)\n",
    ">>\n",
    ">> 0.1: [import required packages](#0.1)\n",
    ">\n",
    "> 1. [Extract region sequences](#1)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3b459614",
   "metadata": {},
   "source": [
    "<a id='0'></a>\n",
    "# 0 Minimum required packages and settings"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4846ea43",
   "metadata": {},
   "source": [
    "<a id='0.1'></a>\n",
    "## 0.1 load required packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "99a843b8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "44820\n"
     ]
    }
   ],
   "source": [
    "%run \"..\\..\\Startup_py3.py\"\n",
    "sys.path.append(r\"..\\..\\..\\..\\Documents\")\n",
    "\n",
    "import ImageAnalysis3 as ia\n",
    "%matplotlib notebook\n",
    "\n",
    "from ImageAnalysis3 import *\n",
    "print(os.getpid())\n",
    "\n",
    "# library design specific tools\n",
    "from ImageAnalysis3.library_tools import LibraryDesigner as ld\n",
    "from ImageAnalysis3.library_tools import LibraryTools as lt\n",
    "# biopython imports\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "from Bio.Blast.Applications import NcbiblastnCommandline\n",
    "from Bio.Blast import NCBIXML"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bd4cf4d8",
   "metadata": {},
   "source": [
    "<a id='1'></a>\n",
    "# 1 Extract region sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "d1426dad",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Use pool_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\n"
     ]
    }
   ],
   "source": [
    "## Some folders\n",
    "# human genome\n",
    "reference_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38'\n",
    "genome_folder = os.path.join(reference_folder, 'Genome')\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain'\n",
    "if not os.path.exists(pool_folder):\n",
    "    print(f\"Create pool_folder: {pool_folder}\")\n",
    "    os.makedirs(pool_folder)\n",
    "else:\n",
    "    print(f\"Use pool_folder: {pool_folder}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "714f9640",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- library_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\n",
      "-- sequence_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\n",
      "-- report_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\reports\n"
     ]
    }
   ],
   "source": [
    "resolution = 0\n",
    "flanking = 10000\n",
    "# folder for sub-pool\n",
    "library_folder = os.path.join(pool_folder, f'human_genome_library')\n",
    "if not os.path.exists(library_folder):\n",
    "    print(f\"create library folder: {library_folder}\")\n",
    "    os.makedirs(library_folder)\n",
    "# folder for fasta sequences\n",
    "sequence_folder = os.path.join(library_folder, 'sequences')\n",
    "if not os.path.exists(sequence_folder):\n",
    "    print(f\"create sequence folder: {sequence_folder}\")\n",
    "    os.makedirs(sequence_folder)\n",
    "# folder to save result probes\n",
    "report_folder = os.path.join(library_folder, 'reports')\n",
    "if not os.path.exists(report_folder):\n",
    "    print(f\"create report folder: {report_folder}\")\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "print(f\"-- library_folder: {library_folder}\")\n",
    "print(f\"-- sequence_folder: {sequence_folder}\")\n",
    "print(f\"-- report_folder: {report_folder}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "217df8b4",
   "metadata": {},
   "source": [
    "<a id='1.1'></a>\n",
    "## 1.1 select regions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "980fa9c6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- load sequence: 1, size=248956422\n",
      "-- load sequence: 10, size=133797422\n",
      "-- load sequence: 11, size=135086622\n",
      "-- load sequence: 12, size=133275309\n",
      "-- load sequence: 13, size=114364328\n",
      "-- load sequence: 14, size=107043718\n",
      "-- load sequence: 15, size=101991189\n",
      "-- load sequence: 16, size=90338345\n",
      "-- load sequence: 17, size=83257441\n",
      "-- load sequence: 18, size=80373285\n",
      "-- load sequence: 19, size=58617616\n",
      "-- load sequence: 2, size=242193529\n",
      "-- load sequence: 20, size=64444167\n",
      "-- load sequence: 21, size=46709983\n",
      "-- load sequence: 22, size=50818468\n",
      "-- load sequence: 3, size=198295559\n",
      "-- load sequence: 4, size=190214555\n",
      "-- load sequence: 5, size=181538259\n",
      "-- load sequence: 6, size=170805979\n",
      "-- load sequence: 7, size=159345973\n",
      "-- load sequence: 8, size=145138636\n",
      "-- load sequence: 9, size=138394717\n",
      "-- load sequence: MT, size=16569\n",
      "-- load sequence: X, size=156040895\n",
      "-- load sequence: Y, size=57227415\n",
      "-- load sequence: KI270728.1, size=1872759\n",
      "-- load sequence: KI270727.1, size=448248\n",
      "-- load sequence: KI270442.1, size=392061\n",
      "-- load sequence: KI270729.1, size=280839\n",
      "-- load sequence: GL000225.1, size=211173\n",
      "-- load sequence: KI270743.1, size=210658\n",
      "-- load sequence: GL000008.2, size=209709\n",
      "-- load sequence: GL000009.2, size=201709\n",
      "-- load sequence: KI270747.1, size=198735\n",
      "-- load sequence: KI270722.1, size=194050\n",
      "-- load sequence: GL000194.1, size=191469\n",
      "-- load sequence: KI270742.1, size=186739\n",
      "-- load sequence: GL000205.2, size=185591\n",
      "-- load sequence: GL000195.1, size=182896\n",
      "-- load sequence: KI270736.1, size=181920\n",
      "-- load sequence: KI270733.1, size=179772\n",
      "-- load sequence: GL000224.1, size=179693\n",
      "-- load sequence: GL000219.1, size=179198\n",
      "-- load sequence: KI270719.1, size=176845\n",
      "-- load sequence: GL000216.2, size=176608\n",
      "-- load sequence: KI270712.1, size=176043\n",
      "-- load sequence: KI270706.1, size=175055\n",
      "-- load sequence: KI270725.1, size=172810\n",
      "-- load sequence: KI270744.1, size=168472\n",
      "-- load sequence: KI270734.1, size=165050\n",
      "-- load sequence: GL000213.1, size=164239\n",
      "-- load sequence: GL000220.1, size=161802\n",
      "-- load sequence: KI270715.1, size=161471\n",
      "-- load sequence: GL000218.1, size=161147\n",
      "-- load sequence: KI270749.1, size=158759\n",
      "-- load sequence: KI270741.1, size=157432\n",
      "-- load sequence: GL000221.1, size=155397\n",
      "-- load sequence: KI270716.1, size=153799\n",
      "-- load sequence: KI270731.1, size=150754\n",
      "-- load sequence: KI270751.1, size=150742\n",
      "-- load sequence: KI270750.1, size=148850\n",
      "-- load sequence: KI270519.1, size=138126\n",
      "-- load sequence: GL000214.1, size=137718\n",
      "-- load sequence: KI270708.1, size=127682\n",
      "-- load sequence: KI270730.1, size=112551\n",
      "-- load sequence: KI270438.1, size=112505\n",
      "-- load sequence: KI270737.1, size=103838\n",
      "-- load sequence: KI270721.1, size=100316\n",
      "-- load sequence: KI270738.1, size=99375\n",
      "-- load sequence: KI270748.1, size=93321\n",
      "-- load sequence: KI270435.1, size=92983\n",
      "-- load sequence: GL000208.1, size=92689\n",
      "-- load sequence: KI270538.1, size=91309\n",
      "-- load sequence: KI270756.1, size=79590\n",
      "-- load sequence: KI270739.1, size=73985\n",
      "-- load sequence: KI270757.1, size=71251\n",
      "-- load sequence: KI270709.1, size=66860\n",
      "-- load sequence: KI270746.1, size=66486\n",
      "-- load sequence: KI270753.1, size=62944\n",
      "-- load sequence: KI270589.1, size=44474\n",
      "-- load sequence: KI270726.1, size=43739\n",
      "-- load sequence: KI270735.1, size=42811\n",
      "-- load sequence: KI270711.1, size=42210\n",
      "-- load sequence: KI270745.1, size=41891\n",
      "-- load sequence: KI270714.1, size=41717\n",
      "-- load sequence: KI270732.1, size=41543\n",
      "-- load sequence: KI270713.1, size=40745\n",
      "-- load sequence: KI270754.1, size=40191\n",
      "-- load sequence: KI270710.1, size=40176\n",
      "-- load sequence: KI270717.1, size=40062\n",
      "-- load sequence: KI270724.1, size=39555\n",
      "-- load sequence: KI270720.1, size=39050\n",
      "-- load sequence: KI270723.1, size=38115\n",
      "-- load sequence: KI270718.1, size=38054\n",
      "-- load sequence: KI270317.1, size=37690\n",
      "-- load sequence: KI270740.1, size=37240\n",
      "-- load sequence: KI270755.1, size=36723\n",
      "-- load sequence: KI270707.1, size=32032\n",
      "-- load sequence: KI270579.1, size=31033\n",
      "-- load sequence: KI270752.1, size=27745\n",
      "-- load sequence: KI270512.1, size=22689\n",
      "-- load sequence: KI270322.1, size=21476\n",
      "-- load sequence: GL000226.1, size=15008\n",
      "-- load sequence: KI270311.1, size=12399\n",
      "-- load sequence: KI270366.1, size=8320\n",
      "-- load sequence: KI270511.1, size=8127\n",
      "-- load sequence: KI270448.1, size=7992\n",
      "-- load sequence: KI270521.1, size=7642\n",
      "-- load sequence: KI270581.1, size=7046\n",
      "-- load sequence: KI270582.1, size=6504\n",
      "-- load sequence: KI270515.1, size=6361\n",
      "-- load sequence: KI270588.1, size=6158\n",
      "-- load sequence: KI270591.1, size=5796\n",
      "-- load sequence: KI270522.1, size=5674\n",
      "-- load sequence: KI270507.1, size=5353\n",
      "-- load sequence: KI270590.1, size=4685\n",
      "-- load sequence: KI270584.1, size=4513\n",
      "-- load sequence: KI270320.1, size=4416\n",
      "-- load sequence: KI270382.1, size=4215\n",
      "-- load sequence: KI270468.1, size=4055\n",
      "-- load sequence: KI270467.1, size=3920\n",
      "-- load sequence: KI270362.1, size=3530\n",
      "-- load sequence: KI270517.1, size=3253\n",
      "-- load sequence: KI270593.1, size=3041\n",
      "-- load sequence: KI270528.1, size=2983\n",
      "-- load sequence: KI270587.1, size=2969\n",
      "-- load sequence: KI270364.1, size=2855\n",
      "-- load sequence: KI270371.1, size=2805\n",
      "-- load sequence: KI270333.1, size=2699\n",
      "-- load sequence: KI270374.1, size=2656\n",
      "-- load sequence: KI270411.1, size=2646\n",
      "-- load sequence: KI270414.1, size=2489\n",
      "-- load sequence: KI270510.1, size=2415\n",
      "-- load sequence: KI270390.1, size=2387\n",
      "-- load sequence: KI270375.1, size=2378\n",
      "-- load sequence: KI270420.1, size=2321\n",
      "-- load sequence: KI270509.1, size=2318\n",
      "-- load sequence: KI270315.1, size=2276\n",
      "-- load sequence: KI270302.1, size=2274\n",
      "-- load sequence: KI270518.1, size=2186\n",
      "-- load sequence: KI270530.1, size=2168\n",
      "-- load sequence: KI270304.1, size=2165\n",
      "-- load sequence: KI270418.1, size=2145\n",
      "-- load sequence: KI270424.1, size=2140\n",
      "-- load sequence: KI270417.1, size=2043\n",
      "-- load sequence: KI270508.1, size=1951\n",
      "-- load sequence: KI270303.1, size=1942\n",
      "-- load sequence: KI270381.1, size=1930\n",
      "-- load sequence: KI270529.1, size=1899\n",
      "-- load sequence: KI270425.1, size=1884\n",
      "-- load sequence: KI270396.1, size=1880\n",
      "-- load sequence: KI270363.1, size=1803\n",
      "-- load sequence: KI270386.1, size=1788\n",
      "-- load sequence: KI270465.1, size=1774\n",
      "-- load sequence: KI270383.1, size=1750\n",
      "-- load sequence: KI270384.1, size=1658\n",
      "-- load sequence: KI270330.1, size=1652\n",
      "-- load sequence: KI270372.1, size=1650\n",
      "-- load sequence: KI270548.1, size=1599\n",
      "-- load sequence: KI270580.1, size=1553\n",
      "-- load sequence: KI270387.1, size=1537\n",
      "-- load sequence: KI270391.1, size=1484\n",
      "-- load sequence: KI270305.1, size=1472\n",
      "-- load sequence: KI270373.1, size=1451\n",
      "-- load sequence: KI270422.1, size=1445\n",
      "-- load sequence: KI270316.1, size=1444\n",
      "-- load sequence: KI270340.1, size=1428\n",
      "-- load sequence: KI270338.1, size=1428\n",
      "-- load sequence: KI270583.1, size=1400\n",
      "-- load sequence: KI270334.1, size=1368\n",
      "-- load sequence: KI270429.1, size=1361\n",
      "-- load sequence: KI270393.1, size=1308\n",
      "-- load sequence: KI270516.1, size=1300\n",
      "-- load sequence: KI270389.1, size=1298\n",
      "-- load sequence: KI270466.1, size=1233\n",
      "-- load sequence: KI270388.1, size=1216\n",
      "-- load sequence: KI270544.1, size=1202\n",
      "-- load sequence: KI270310.1, size=1201\n",
      "-- load sequence: KI270412.1, size=1179\n",
      "-- load sequence: KI270395.1, size=1143\n",
      "-- load sequence: KI270376.1, size=1136\n",
      "-- load sequence: KI270337.1, size=1121\n",
      "-- load sequence: KI270335.1, size=1048\n",
      "-- load sequence: KI270378.1, size=1048\n",
      "-- load sequence: KI270379.1, size=1045\n",
      "-- load sequence: KI270329.1, size=1040\n",
      "-- load sequence: KI270419.1, size=1029\n",
      "-- load sequence: KI270336.1, size=1026\n",
      "-- load sequence: KI270312.1, size=998\n",
      "-- load sequence: KI270539.1, size=993\n",
      "-- load sequence: KI270385.1, size=990\n",
      "-- load sequence: KI270423.1, size=981\n",
      "-- load sequence: KI270392.1, size=971\n",
      "-- load sequence: KI270394.1, size=970\n",
      "- 194 sequences loaded.\n"
     ]
    }
   ],
   "source": [
    "# prepare sequence reader\n",
    "reload(ia.library_tools.sequences)\n",
    "seq_rd = ia.library_tools.sequences.sequence_reader(genome_folder)\n",
    "seq_rd.load_ref_sequences() # load genomic sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "id": "5a5d3385",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3088269832\n"
     ]
    }
   ],
   "source": [
    "# calculate total length\n",
    "min_chr_len = 2000000 # ONLY taking care of chromosomes longer than 1Mb\n",
    "chr_len_dict = {_name:len(_record) for _name, _record in seq_rd.ref_seq_dict.items() if len(_record) > min_chr_len}\n",
    "total_len = sum(list(chr_len_dict.values()))\n",
    "print(total_len)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1904a504",
   "metadata": {},
   "source": [
    "## select resolution for each chromosome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "5859125b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Loading from file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\chr_2_design_region_num.pkl\n",
      "Loading from file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\chr_2_design_resolution.pkl\n"
     ]
    }
   ],
   "source": [
    "overwrite = False\n",
    "region_num_filename = os.path.join(library_folder, 'chr_2_design_region_num.pkl')\n",
    "resolution_filename = os.path.join(library_folder, 'chr_2_design_resolution.pkl')\n",
    "\n",
    "# if both files exists, directly load\n",
    "if os.path.exists(region_num_filename) and os.path.exists(resolution_filename) and not overwrite:\n",
    "    print(f\"Loading from file: {region_num_filename}\")\n",
    "    chr_2_region_num = pickle.load(open(region_num_filename, 'rb'))\n",
    "    print(f\"Loading from file: {resolution_filename}\")\n",
    "    chr_2_resolution = pickle.load(open(resolution_filename, 'rb'))\n",
    "else:\n",
    "    # Selecte regions in an adaptive manner to gaurentee \n",
    "    allowed_resolutions = (np.arange(25,0,-5) * 1e5).astype(np.int32)\n",
    "    min_chr_region_num = 40\n",
    "    chr_2_region_num = {}\n",
    "    chr_2_resolution = {}\n",
    "    for _name, _len in sorted(chr_len_dict.items(), key=lambda v:v[1]): \n",
    "        # iterative\n",
    "        for _resolution in allowed_resolutions:\n",
    "            # select the shortest chromosome first\n",
    "            _curr_num = np.floor(_len/_resolution).astype(np.int32)\n",
    "            if _curr_num >= min_chr_region_num:\n",
    "                chr_2_region_num[_name] = _curr_num\n",
    "                chr_2_resolution[_name] = _resolution\n",
    "                print(_name, _resolution, _curr_num)\n",
    "                min_chr_region_num = max(_curr_num * 0.8, min_chr_region_num)\n",
    "                break    \n",
    "    print(np.sum(list(chr_2_region_num.values())))\n",
    "    # save\n",
    "    print(f\"Saving to file: {region_num_filename}\")\n",
    "    pickle.dump(chr_2_region_num, open(region_num_filename, 'wb'))\n",
    "    print(f\"Saving to file: {resolution_filename}\")\n",
    "    pickle.dump(chr_2_resolution, open(resolution_filename, 'wb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b8d42f5b",
   "metadata": {},
   "source": [
    "## For each chromosome, calculate region center:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "16b3b917",
   "metadata": {},
   "outputs": [],
   "source": [
    "from ImageAnalysis3.structure_tools.distance import sort_chr\n",
    "region_size = 30000 # extract 30kb around the targets"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "id": "ea7a5258",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Loading from file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\region_dicts.pkl\n"
     ]
    }
   ],
   "source": [
    "region_dict_savefile = os.path.join(sequence_folder, 'region_dicts.pkl')\n",
    "if os.path.exists(region_dict_savefile) and not overwrite:\n",
    "    print(f\"Loading from file: {region_dict_savefile}\")\n",
    "    region_dicts = pickle.load(open(region_dict_savefile, 'rb'))\n",
    "else:\n",
    "    region_seqs = []\n",
    "    region_dicts = []\n",
    "    region_id = 0\n",
    "    for _chr in sorted(chr_2_region_num.keys(), key=lambda _v:sort_chr(_v)):\n",
    "        print(_chr)\n",
    "        _region_num = chr_2_region_num[_chr]\n",
    "        _resolution = chr_2_resolution[_chr]\n",
    "        # get region center\n",
    "        _reg_centers = (np.arange(0.5,_region_num+0.5, 1) * _resolution ).astype(np.int32)\n",
    "        _reg_starts = _reg_centers - int(region_size/2) + 1\n",
    "        _reg_ends = _reg_centers + int(region_size/2)\n",
    "        _seqs = [seq_rd.ref_seq_dict[_chr][_s:_e] for _s,_e in zip(_reg_starts, _reg_ends)]\n",
    "        region_seqs.extend(_seqs)\n",
    "\n",
    "        for _s,_e in zip(_reg_starts, _reg_ends):\n",
    "            _reg_dict = {\n",
    "                'Chr':_chr,\n",
    "                'Start':_s,\n",
    "                'End':_e,\n",
    "                'Region':f\"{_chr}:{_s}-{_e}\",\n",
    "                'Strand':'+',\n",
    "                'Gene': region_id,\n",
    "            }\n",
    "            region_dicts.append(_reg_dict)\n",
    "            region_id += 1\n",
    "    # save\n",
    "    print(f\"Saving to file: {region_dict_savefile}\")\n",
    "    pickle.dump(region_dicts, open(region_dict_savefile, 'wb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "0ab776c8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- searching among 24 references\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 1.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 2.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 3.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 4.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 5.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 6.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 7.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 8.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 9.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 10.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 11.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 12.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 13.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 14.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 15.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 16.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 17.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 18.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 19.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 20.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 21.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: 22.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: X.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n",
      "-- a match found in record: Y.\n"
     ]
    }
   ],
   "source": [
    "seqs = seq_rd.find_sequence_for_region(region_dicts)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "2970dec8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- saving sequences into folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\0_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\2_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\3_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\4_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\5_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\6_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\7_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\8_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\9_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\10_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\11_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\12_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\13_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\14_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\15_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\16_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\17_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\18_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\19_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\20_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\21_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\22_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\23_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\24_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\25_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\26_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\27_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\28_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\29_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\30_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\31_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\32_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\33_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\34_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\35_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\36_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\37_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\38_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\39_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\40_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\41_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\42_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\43_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\44_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\45_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\46_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\47_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\48_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\49_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\50_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\51_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\52_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\53_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\54_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\55_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\56_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\57_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\58_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\59_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\60_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\61_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\62_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\63_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\64_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\65_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\66_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\67_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\68_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\69_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\70_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\71_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\72_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\73_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\74_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\75_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\76_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\77_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\78_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\79_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\80_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\81_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\82_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\83_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\84_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\85_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\86_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\87_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\88_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\89_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\90_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\91_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\92_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\93_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\94_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\95_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\96_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\97_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\98_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\99_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\100_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\101_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\102_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\103_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\104_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\105_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\106_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\107_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\108_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\109_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\110_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\111_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\112_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\113_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\114_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\115_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\116_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\117_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\118_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\119_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\120_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\121_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\122_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\123_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\124_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\125_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\126_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\127_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\128_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\129_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\130_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\131_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\132_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\133_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\134_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\135_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\136_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\137_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\138_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\139_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\140_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\141_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\142_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\143_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\144_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\145_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\146_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\147_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\148_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\149_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\150_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\151_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\152_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\153_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\154_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\155_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\156_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\157_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\158_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\159_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\160_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\161_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\162_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\163_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\164_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\165_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\166_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\167_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\168_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\169_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\170_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\171_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\172_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\173_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\174_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\175_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\176_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\177_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\178_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\179_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\180_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\181_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\182_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\183_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\184_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\185_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\186_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\187_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\188_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\189_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\190_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\191_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\192_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\193_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\194_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\195_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\196_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\197_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\198_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\199_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\200_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\201_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\202_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\203_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\204_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\205_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\206_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\207_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\208_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\209_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\210_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\211_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\212_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\213_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\214_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\215_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\216_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\217_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\218_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\219_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\220_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\221_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\222_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\223_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\224_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\225_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\226_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\227_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\228_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\229_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\230_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\231_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\232_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\233_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\234_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\235_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\236_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\237_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\238_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\239_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\240_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\241_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\242_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\243_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\244_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\245_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\246_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\247_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\248_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\249_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\250_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\251_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\252_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\253_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\254_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\255_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\256_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\257_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\258_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\259_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\260_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\261_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\262_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\263_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\264_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\265_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\266_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\267_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\268_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\269_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\270_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\271_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\272_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\273_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\274_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\275_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\276_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\277_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\278_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\279_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\280_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\281_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\282_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\283_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\284_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\285_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\286_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\287_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\288_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\289_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\290_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\291_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\292_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\293_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\294_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\295_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\296_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\297_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\298_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\299_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\300_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\301_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\302_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\303_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\304_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\305_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\306_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\307_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\308_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\309_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\310_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\311_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\312_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\313_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\314_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\315_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\316_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\317_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\318_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\319_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\320_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\321_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\322_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\323_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\324_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\325_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\326_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\327_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\328_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\329_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\330_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\331_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\332_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\333_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\334_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\335_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\336_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\337_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\338_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\339_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\340_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\341_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\342_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\343_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\344_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\345_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\346_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\347_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\348_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\349_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\350_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\351_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\352_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\353_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\354_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\355_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\356_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\357_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\358_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\359_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\360_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\361_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\362_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\363_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\364_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\365_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\366_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\367_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\368_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\369_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\370_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\371_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\372_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\373_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\374_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\375_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\376_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\377_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\378_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\379_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\380_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\381_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\382_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\383_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\384_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\385_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\386_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\387_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\388_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\389_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\390_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\391_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\392_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\393_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\394_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\395_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\396_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\397_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\398_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\399_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\400_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\401_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\402_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\403_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\404_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\405_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\406_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\407_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\408_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\409_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\410_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\411_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\412_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\413_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\414_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\415_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\416_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\417_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\418_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\419_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\420_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\421_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\422_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\423_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\424_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\425_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\426_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\427_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\428_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\429_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\430_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\431_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\432_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\433_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\434_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\435_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\436_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\437_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\438_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\439_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\440_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\441_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\442_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\443_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\444_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\445_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\446_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\447_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\448_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\449_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\450_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\451_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\452_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\453_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\454_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\455_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\456_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\457_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\458_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\459_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\460_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\461_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\462_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\463_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\464_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\465_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\466_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\467_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\468_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\469_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\470_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\471_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\472_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\473_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\474_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\475_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\476_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\477_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\478_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\479_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\480_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\481_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\482_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\483_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\484_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\485_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\486_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\487_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\488_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\489_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\490_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\491_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\492_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\493_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\494_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\495_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\496_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\497_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\498_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\499_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\500_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\501_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\502_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\503_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\504_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\505_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\506_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\507_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\508_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\509_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\510_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\511_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\512_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\513_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\514_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\515_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\516_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\517_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\518_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\519_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\520_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\521_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\522_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\523_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\524_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\525_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\526_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\527_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\528_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\529_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\530_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\531_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\532_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\533_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\534_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\535_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\536_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\537_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\538_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\539_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\540_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\541_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\542_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\543_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\544_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\545_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\546_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\547_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\548_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\549_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\550_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\551_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\552_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\553_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\554_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\555_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\556_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\557_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\558_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\559_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\560_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\561_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\562_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\563_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\564_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\565_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\566_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\567_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\568_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\569_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\570_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\571_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\572_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\573_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\574_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\575_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\576_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\577_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\578_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\579_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\580_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\581_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\582_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\583_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\584_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\585_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\586_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\587_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\588_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\589_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\590_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\591_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\592_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\593_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\594_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\595_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\596_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\597_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\598_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\599_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\600_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\601_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\602_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\603_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\604_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\605_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\606_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\607_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\608_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\609_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\610_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\611_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\612_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\613_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\614_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\615_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\616_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\617_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\618_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\619_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\620_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\621_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\622_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\623_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\624_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\625_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\626_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\627_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\628_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\629_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\630_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\631_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\632_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\633_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\634_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\635_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\636_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\637_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\638_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\639_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\640_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\641_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\642_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\643_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\644_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\645_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\646_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\647_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\648_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\649_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\650_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\651_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\652_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\653_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\654_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\655_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\656_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\657_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\658_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\659_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\660_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\661_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\662_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\663_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\664_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\665_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\666_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\667_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\668_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\669_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\670_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\671_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\672_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\673_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\674_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\675_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\676_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\677_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\678_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\679_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\680_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\681_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\682_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\683_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\684_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\685_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\686_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\687_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\688_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\689_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\690_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\691_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\692_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\693_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\694_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\695_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\696_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\697_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\698_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\699_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\700_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\701_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\702_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\703_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\704_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\705_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\706_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\707_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\708_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\709_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\710_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\711_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\712_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\713_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\714_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\715_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\716_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\717_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\718_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\719_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\720_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\721_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\722_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\723_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\724_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\725_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\726_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\727_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\728_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\729_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\730_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\731_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\732_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\733_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\734_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\735_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\736_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\737_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\738_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\739_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\740_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\741_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\742_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\743_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\744_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\745_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\746_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\747_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\748_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\749_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\750_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\751_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\752_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\753_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\754_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\755_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\756_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\757_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\758_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\759_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\760_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\761_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\762_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\763_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\764_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\765_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\766_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\767_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\768_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\769_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\770_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\771_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\772_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\773_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\774_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\775_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\776_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\777_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\778_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\779_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\780_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\781_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\782_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\783_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\784_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\785_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\786_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\787_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\788_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\789_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\790_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\791_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\792_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\793_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\794_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\795_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\796_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\797_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\798_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\799_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\800_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\801_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\802_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\803_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\804_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\805_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\806_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\807_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\808_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\809_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\810_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\811_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\812_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\813_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\814_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\815_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\816_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\817_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\818_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\819_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\820_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\821_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\822_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\823_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\824_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\825_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\826_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\827_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\828_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\829_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\830_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\831_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\832_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\833_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\834_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\835_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\836_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\837_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\838_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\839_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\840_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\841_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\842_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\843_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\844_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\845_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\846_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\847_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\848_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\849_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\850_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\851_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\852_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\853_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\854_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\855_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\856_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\857_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\858_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\859_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\860_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\861_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\862_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\863_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\864_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\865_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\866_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\867_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\868_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\869_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\870_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\871_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\872_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\873_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\874_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\875_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\876_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\877_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\878_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\879_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\880_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\881_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\882_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\883_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\884_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\885_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\886_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\887_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\888_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\889_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\890_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\891_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\892_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\893_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\894_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\895_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\896_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\897_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\898_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\899_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\900_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\901_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\902_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\903_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\904_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\905_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\906_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\907_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\908_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\909_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\910_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\911_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\912_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\913_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\914_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\915_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\916_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\917_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\918_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\919_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\920_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\921_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\922_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\923_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\924_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\925_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\926_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\927_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\928_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\929_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\930_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\931_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\932_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\933_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\934_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\935_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\936_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\937_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\938_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\939_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\940_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\941_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\942_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\943_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\944_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\945_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\946_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\947_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\948_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\949_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\950_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\951_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\952_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\953_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\954_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\955_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\956_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\957_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\958_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\959_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\960_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\961_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\962_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\963_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\964_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\965_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\966_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\967_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\968_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\969_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\970_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\971_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\972_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\973_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\974_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\975_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\976_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\977_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\978_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\979_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\980_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\981_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\982_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\983_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\984_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\985_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\986_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\987_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\988_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\989_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\990_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\991_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\992_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\993_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\994_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\995_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\996_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\997_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\998_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\999_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1000_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1001_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1002_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1003_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1004_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1005_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1006_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1007_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1008_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1009_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1010_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1011_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1012_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1013_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1014_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1015_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1016_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1017_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1018_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1019_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1020_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1021_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1022_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1023_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1024_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1025_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1026_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1027_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1028_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1029_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1030_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1031_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1032_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1033_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1034_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1035_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1036_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1037_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1038_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1039_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1040_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1041_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1042_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1043_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1044_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1045_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1046_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1047_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1048_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1049_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1050_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1051_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1052_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1053_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1054_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1055_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1056_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1057_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1058_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1059_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1060_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1061_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1062_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1063_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1064_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1065_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1066_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1067_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1068_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1069_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1070_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1071_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1072_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1073_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1074_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1075_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1076_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1077_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1078_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1079_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1080_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1081_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1082_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1083_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1084_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1085_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1086_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1087_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1088_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1089_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1090_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1091_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1092_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1093_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1094_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1095_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1096_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1097_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1098_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1099_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1100_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1101_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1102_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1103_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1104_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1105_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1106_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1107_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1108_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1109_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1110_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1111_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1112_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1113_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1114_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1115_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1116_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1117_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1118_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1119_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1120_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1121_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1122_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1123_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1124_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1125_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1126_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1127_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1128_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1129_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1130_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1131_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1132_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1133_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1134_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1135_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1136_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1137_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1138_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1139_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1140_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1141_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1142_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1143_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1144_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1145_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1146_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1147_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1148_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1149_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1150_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1151_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1152_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1153_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1154_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1155_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1156_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1157_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1158_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1159_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1160_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1161_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1162_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1163_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1164_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1165_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1166_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1167_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1168_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1169_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1170_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1171_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1172_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1173_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1174_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1175_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1176_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1177_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1178_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1179_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1180_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1181_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1182_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1183_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1184_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1185_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1186_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1187_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1188_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1189_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1190_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1191_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1192_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1193_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1194_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1195_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1196_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1197_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1198_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1199_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1200_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1201_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1202_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1203_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1204_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1205_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1206_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1207_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1208_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1209_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1210_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1211_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1212_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1213_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1214_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1215_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1216_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1217_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1218_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1219_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1220_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1221_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1222_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1223_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1224_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1225_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1226_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1227_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1228_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1229_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1230_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1231_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1232_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1233_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1234_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1235_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1236_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1237_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1238_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1239_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1240_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1241_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1242_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1243_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1244_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1245_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1246_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1247_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1248_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1249_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1250_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1251_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1252_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1253_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1254_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1255_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1256_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1257_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1258_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1259_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1260_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1261_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1262_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1263_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1264_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1265_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1266_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1267_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1268_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1269_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1270_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1271_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1272_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1273_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1274_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1275_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1276_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1277_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1278_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1279_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1280_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1281_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1282_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1283_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1284_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1285_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1286_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1287_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1288_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1289_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1290_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1291_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1292_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1293_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1294_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1295_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1296_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1297_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1298_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1299_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1300_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1301_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1302_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1303_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1304_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1305_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1306_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1307_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1308_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1309_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1310_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1311_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1312_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1313_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1314_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1315_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1316_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1317_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1318_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1319_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1320_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1321_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1322_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1323_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1324_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1325_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1326_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1327_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1328_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1329_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1330_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1331_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1332_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1333_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1334_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1335_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1336_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1337_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1338_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1339_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1340_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1341_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1342_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1343_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1344_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1345_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1346_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1347_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1348_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1349_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1350_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1351_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1352_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1353_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1354_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1355_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1356_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1357_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1358_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1359_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1360_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1361_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1362_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1363_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1364_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1365_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1366_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1367_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1368_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1369_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1370_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1371_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1372_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1373_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1374_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1375_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1376_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1377_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1378_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1379_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1380_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1381_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1382_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1383_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1384_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1385_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1386_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1387_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1388_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1389_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1390_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1391_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1392_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1393_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1394_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1395_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1396_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1397_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1398_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1399_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1400_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1401_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1402_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1403_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1404_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1405_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1406_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1407_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1408_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1409_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1410_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1411_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1412_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1413_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1414_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1415_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1416_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1417_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1418_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1419_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1420_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1421_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1422_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1423_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1424_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1425_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1426_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1427_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1428_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1429_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1430_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1431_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1432_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1433_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1434_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1435_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1436_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1437_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1438_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1439_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1440_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1441_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1442_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1443_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1444_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1445_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1446_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1447_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1448_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1449_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1450_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1451_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1452_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1453_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1454_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1455_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1456_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1457_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1458_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1459_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1460_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1461_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1462_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1463_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1464_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1465_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1466_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1467_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1468_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1469_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1470_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1471_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1472_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1473_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1474_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1475_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1476_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1477_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1478_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1479_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1480_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1481_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1482_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1483_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1484_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1485_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1486_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1487_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1488_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1489_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1490_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1491_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1492_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1493_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1494_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1495_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1496_reg_0.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1497_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1498_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1499_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1500_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1501_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1502_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1503_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1504_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1505_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1506_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1507_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1508_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1509_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1510_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1511_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1512_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1513_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1514_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1515_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1516_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1517_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1518_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1519_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1520_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1521_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1522_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1523_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1524_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1525_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1526_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1527_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1528_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1529_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1530_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1531_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1532_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1533_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1534_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1535_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1536_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1537_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1538_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1539_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1540_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1541_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1542_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1543_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1544_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1545_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1546_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1547_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1548_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1549_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1550_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1551_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1552_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1553_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1554_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1555_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1556_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1557_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1558_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1559_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1560_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1561_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1562_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1563_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1564_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1565_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1566_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1567_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1568_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1569_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1570_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1571_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1572_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1573_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1574_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1575_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1576_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1577_reg_0.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1578_reg_0.fasta\n"
     ]
    }
   ],
   "source": [
    "seq_rd.save_sequences(save_folder=sequence_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fb106a4a",
   "metadata": {},
   "source": [
    "<a id='2'></a>\n",
    "# 2. Design probe targeting sequences by probe_designer\n",
    "\n",
    "run probe_designer, remeber to clear the memory usage because each table should take ~32GB"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "cfad443e",
   "metadata": {},
   "source": [
    "<a id='2.1'></a>\n",
    "## 2.1 Construct count table with all the 17-mers in the genome\n",
    "\n",
    "Only do this if you don't have pre-built 17-mer\n",
    "\n",
    "However you can do almost the same thing for your own library during quality check\n",
    "\n",
    "This library requires hg38 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "id": "414635fc",
   "metadata": {},
   "outputs": [],
   "source": [
    "overwrite_table = False"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "85dda5fb",
   "metadata": {},
   "source": [
    "### construct map for whole genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "35c7e4f5",
   "metadata": {},
   "outputs": [],
   "source": [
    "from ImageAnalysis3 import library_tools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "id": "dc089b78",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\hg38_genome_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.design)\n",
    " \n",
    "genome_table_file = os.path.join(reference_folder, 'hg38_genome_17w.npy')\n",
    "\n",
    "if not os.path.exists(genome_table_file) or overwrite_table:\n",
    "    # genome\n",
    "    _genome_filenames = [os.path.join(genome_folder, _fl) \n",
    "         for _fl in os.listdir(genome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_genome_filenames))\n",
    "\n",
    "    ct = library_tools.design.countTable(word=17,save_file=genome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_genome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "\n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {genome_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "29731c46",
   "metadata": {},
   "source": [
    "### construct map for transcriptome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "88940c0a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\hg38_transcriptome_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "from tqdm import tqdm\n",
    "# transcriptome\n",
    "transcriptome_folder = os.path.join(reference_folder, 'Transcriptome')\n",
    "\n",
    "transcriptome_table_file = os.path.join(reference_folder, 'hg38_transcriptome_17w.npy')\n",
    "\n",
    "if not os.path.exists(transcriptome_table_file) or overwrite_table:\n",
    "    # transcriptome\n",
    "    _transcriptome_filenames = [os.path.join(transcriptome_folder, _fl) \n",
    "         for _fl in os.listdir(transcriptome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_transcriptome_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=transcriptome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_transcriptome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {transcriptome_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0b477fab",
   "metadata": {},
   "source": [
    "### construct map for repeats from RepBase"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "dac1a780",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\Repbase_v2603_repeat_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "from tqdm import tqdm\n",
    "# repeat\n",
    "repeat_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\Repbase'\n",
    "\n",
    "repeat_table_file = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "\n",
    "if not os.path.exists(repeat_table_file) or overwrite_table:\n",
    "    # repeat\n",
    "    _repeat_filenames = [os.path.join(repeat_folder, _fl) \n",
    "         for _fl in os.listdir(repeat_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_repeat_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=repeat_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_repeat_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {repeat_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "688a066f",
   "metadata": {},
   "source": [
    "<a id='2.2'></a>\n",
    "## 2.2 Design probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "id": "e7c4541c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1579 regions loaded to design probes.\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\reports\\merged_probes.pbr\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "# requires pre_defined genome_folder and library_folder\n",
    "# Indices\n",
    "genome_index = os.path.join(reference_folder, 'hg38_genome_17w.npy')\n",
    "transcriptome_index = os.path.join(reference_folder, 'hg38_transcriptome_17w.npy') \n",
    "repeat_index = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "#ref_merfish_index = os.path.join(reference_folder, 'M1_meng_MERFISH_17w.npy') # merfish designed by Meng\n",
    "# get input files \n",
    "input_files = glob.glob(os.path.join(sequence_folder, '*.fasta'))\n",
    "\n",
    "print(f\"{len(input_files)} regions loaded to design probes.\")\n",
    "\n",
    "if not os.path.exists(report_folder):\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "# filename to save probe reports\n",
    "probe_savefile = os.path.join(report_folder, f'merged_probes.pbr')\n",
    "print(probe_savefile)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "722e6b4b",
   "metadata": {},
   "source": [
    "### create pb_designer class"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "id": "f3c242de",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Probe designer derived from Bogdan Bintu:\n",
      "https://github.com/BogdanBintu/ChromatinImaging/blob/master/LibraryDesign/LibraryDesigner.py\n",
      "by Pu Zheng, 2020.11\n",
      "\n",
      "Major changes:\n",
      "    1. allow design of two strands\n",
      "    2. separate reverse_complement (rev_com) and from two strands (two_stranded) as \n",
      "    two different inputs for map_dic and sequence_dic\n",
      "    3. replace 'local_genome' with 'self_sequences' to be more explicit, and only \n",
      "    exclude the counts for the corresponding self_sequence within each input. \n",
      "\n",
      "Key information:\n",
      "    - number of input_sequence(s): 1579\n",
      "    - save_file location: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\reports\\merged_probes.pbr\n",
      "\n",
      "- Fail to load from savefile: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\reports\\merged_probes.pbr, file doesn't exist.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "False"
      ]
     },
     "execution_count": 57,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "pb_designer = library_tools.design.pb_reports_class(\n",
    "    sequence_dic={'file':input_files,\n",
    "                  'rev_com':True, # design two strands\n",
    "                  'two_stranded':True},\n",
    "    map_dic={'genome':{'file':genome_index,'rev_com':False,'two_stranded':True},\n",
    "             'transcriptome':{'file':transcriptome_index,'rev_com':True,'two_stranded':False},\n",
    "             'rep_genome':{'file':repeat_index,'rev_com':False,'two_stranded':True},\n",
    "             'self_sequences':{'file':input_files,'force_list':True,'rev_com':False,'two_stranded':True},\n",
    "             #'ref_merfish':{'file':ref_merfish_index,'rev_com':False,'two_stranded':True},\n",
    "             },\n",
    "        save_file=probe_savefile,\n",
    "    params_dic={'word_size':17,'pb_len':42,'buffer_len':2,'max_count':2**16-1,\n",
    "                'check_on_go': False, # whether automatically check probes\n",
    "                'auto': False, # whether automatically convert reference maps\n",
    "               },\n",
    "    check_dic={('genome','self_sequences'): 25,\n",
    "               'rep_genome': 0,\n",
    "               'transcriptome': 25, # 14\n",
    "               #'ref_merfish': 14,\n",
    "               'gc':[0.25,0.75],'tm': 37+0.62*50+5,\n",
    "               }\n",
    "    )\n",
    "print(pb_designer)\n",
    "pb_designer.load_from_file(load_probes_only=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b84dae63",
   "metadata": {},
   "source": [
    "### calculate probe reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "id": "686689a5",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- setting attribute: map_genome\n",
      "--- finish map_genome in 117.838s.\n",
      "-- setting attribute: map_transcriptome\n",
      "--- finish map_transcriptome in 121.144s.\n",
      "-- setting attribute: map_rep_genome\n",
      "--- finish map_rep_genome in 111.222s.\n",
      "Time(s): 350.2070834636688\n",
      "- Designing targeting sequence for 1579 regions\n",
      "-- designing region: 1:1235001-1265000_strand_+_gene_0 -- region: 0, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\0_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.308s.\n",
      "- Designed 59918 candidate probes in 16.210s.\n",
      "-- designing region: 1:3735001-3765000_strand_+_gene_1 -- region: 1, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.343s.\n",
      "- Designed 59918 candidate probes in 23.067s.\n",
      "-- designing region: 1:6235001-6265000_strand_+_gene_2 -- region: 2, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\2_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.335s.\n",
      "- Designed 59918 candidate probes in 19.631s.\n",
      "-- designing region: 1:8735001-8765000_strand_+_gene_3 -- region: 3, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\3_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.425s.\n",
      "-- designing region: 1:11235001-11265000_strand_+_gene_4 -- region: 4, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\4_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.494s.\n",
      "-- designing region: 1:13735001-13765000_strand_+_gene_5 -- region: 5, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\5_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.456s.\n",
      "-- designing region: 1:16235001-16265000_strand_+_gene_6 -- region: 6, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\6_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.448s.\n",
      "-- designing region: 1:18735001-18765000_strand_+_gene_7 -- region: 7, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\7_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 19.049s.\n",
      "-- designing region: 1:21235001-21265000_strand_+_gene_8 -- region: 8, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\8_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.419s.\n",
      "-- designing region: 1:23735001-23765000_strand_+_gene_9 -- region: 9, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\9_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.416s.\n",
      "-- designing region: 1:26235001-26265000_strand_+_gene_10 -- region: 10, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\10_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.424s.\n",
      "-- designing region: 1:28735001-28765000_strand_+_gene_11 -- region: 11, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\11_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.458s.\n",
      "-- designing region: 1:31235001-31265000_strand_+_gene_12 -- region: 12, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\12_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.379s.\n",
      "-- designing region: 1:33735001-33765000_strand_+_gene_13 -- region: 13, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\13_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.405s.\n",
      "-- designing region: 1:36235001-36265000_strand_+_gene_14 -- region: 14, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\14_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.388s.\n",
      "-- designing region: 1:38735001-38765000_strand_+_gene_15 -- region: 15, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\15_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.395s.\n",
      "-- designing region: 1:41235001-41265000_strand_+_gene_16 -- region: 16, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\16_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.388s.\n",
      "-- designing region: 1:43735001-43765000_strand_+_gene_17 -- region: 17, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\17_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.381s.\n",
      "-- designing region: 1:46235001-46265000_strand_+_gene_18 -- region: 18, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\18_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.379s.\n",
      "-- designing region: 1:48735001-48765000_strand_+_gene_19 -- region: 19, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\19_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.443s.\n",
      "-- designing region: 1:51235001-51265000_strand_+_gene_20 -- region: 20, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\20_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.409s.\n",
      "-- designing region: 1:53735001-53765000_strand_+_gene_21 -- region: 21, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\21_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.431s.\n",
      "-- designing region: 1:56235001-56265000_strand_+_gene_22 -- region: 22, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\22_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.444s.\n",
      "-- designing region: 1:58735001-58765000_strand_+_gene_23 -- region: 23, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\23_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 1:61235001-61265000_strand_+_gene_24 -- region: 24, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\24_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.401s.\n",
      "-- designing region: 1:63735001-63765000_strand_+_gene_25 -- region: 25, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\25_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.409s.\n",
      "-- designing region: 1:66235001-66265000_strand_+_gene_26 -- region: 26, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\26_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: 1:68735001-68765000_strand_+_gene_27 -- region: 27, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\27_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.440s.\n",
      "-- designing region: 1:71235001-71265000_strand_+_gene_28 -- region: 28, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\28_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 16.000s.\n",
      "-- designing region: 1:73735001-73765000_strand_+_gene_29 -- region: 29, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\29_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 19.696s.\n",
      "-- designing region: 1:76235001-76265000_strand_+_gene_30 -- region: 30, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\30_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.345s.\n",
      "- Designed 59918 candidate probes in 29.179s.\n",
      "-- designing region: 1:78735001-78765000_strand_+_gene_31 -- region: 31, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\31_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.258s.\n",
      "- Designed 59918 candidate probes in 18.508s.\n",
      "-- designing region: 1:81235001-81265000_strand_+_gene_32 -- region: 32, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\32_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 20.955s.\n",
      "-- designing region: 1:83735001-83765000_strand_+_gene_33 -- region: 33, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\33_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.654s.\n",
      "- Designed 59918 candidate probes in 16.666s.\n",
      "-- designing region: 1:86235001-86265000_strand_+_gene_34 -- region: 34, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\34_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.516s.\n",
      "-- designing region: 1:88735001-88765000_strand_+_gene_35 -- region: 35, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\35_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 1:91235001-91265000_strand_+_gene_36 -- region: 36, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\36_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 1:93735001-93765000_strand_+_gene_37 -- region: 37, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\37_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.733s.\n",
      "-- designing region: 1:96235001-96265000_strand_+_gene_38 -- region: 38, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\38_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.311s.\n",
      "- Designed 59918 candidate probes in 15.626s.\n",
      "-- designing region: 1:98735001-98765000_strand_+_gene_39 -- region: 39, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\39_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.625s.\n",
      "-- designing region: 1:101235001-101265000_strand_+_gene_40 -- region: 40, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\40_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.305s.\n",
      "- Designed 59918 candidate probes in 15.668s.\n",
      "-- designing region: 1:103735001-103765000_strand_+_gene_41 -- region: 41, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\41_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.734s.\n",
      "-- designing region: 1:106235001-106265000_strand_+_gene_42 -- region: 42, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\42_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.644s.\n",
      "-- designing region: 1:108735001-108765000_strand_+_gene_43 -- region: 43, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\43_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.644s.\n",
      "-- designing region: 1:111235001-111265000_strand_+_gene_44 -- region: 44, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\44_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.674s.\n",
      "-- designing region: 1:113735001-113765000_strand_+_gene_45 -- region: 45, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\45_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Designed 59918 candidate probes in 15.656s.\n",
      "-- designing region: 1:116235001-116265000_strand_+_gene_46 -- region: 46, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\46_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.691s.\n",
      "-- designing region: 1:118735001-118765000_strand_+_gene_47 -- region: 47, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\47_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.892s.\n",
      "-- designing region: 1:121235001-121265000_strand_+_gene_48 -- region: 48, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\48_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.594s.\n",
      "-- designing region: 1:123735001-123765000_strand_+_gene_49 -- region: 49, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\49_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.222s.\n",
      "- Designed 59918 candidate probes in 15.390s.\n",
      "-- designing region: 1:126235001-126265000_strand_+_gene_50 -- region: 50, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\50_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.219s.\n",
      "- Designed 0 candidate probes in 0.513s.\n",
      "-- designing region: 1:128735001-128765000_strand_+_gene_51 -- region: 51, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\51_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.584s.\n",
      "- Designed 0 candidate probes in 0.879s.\n",
      "-- designing region: 1:131235001-131265000_strand_+_gene_52 -- region: 52, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\52_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.586s.\n",
      "- Designed 0 candidate probes in 0.880s.\n",
      "-- designing region: 1:133735001-133765000_strand_+_gene_53 -- region: 53, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\53_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.580s.\n",
      "- Designed 0 candidate probes in 0.877s.\n",
      "-- designing region: 1:136235001-136265000_strand_+_gene_54 -- region: 54, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\54_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.592s.\n",
      "- Designed 0 candidate probes in 0.885s.\n",
      "-- designing region: 1:138735001-138765000_strand_+_gene_55 -- region: 55, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\55_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.575s.\n",
      "- Designed 0 candidate probes in 0.870s.\n",
      "-- designing region: 1:141235001-141265000_strand_+_gene_56 -- region: 56, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\56_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.590s.\n",
      "- Designed 0 candidate probes in 0.886s.\n",
      "-- designing region: 1:143735001-143765000_strand_+_gene_57 -- region: 57, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\57_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.653s.\n",
      "- Designed 59918 candidate probes in 15.938s.\n",
      "-- designing region: 1:146235001-146265000_strand_+_gene_58 -- region: 58, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\58_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 1:148735001-148765000_strand_+_gene_59 -- region: 59, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\59_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.584s.\n",
      "-- designing region: 1:151235001-151265000_strand_+_gene_60 -- region: 60, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\60_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 1:153735001-153765000_strand_+_gene_61 -- region: 61, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\61_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.471s.\n",
      "-- designing region: 1:156235001-156265000_strand_+_gene_62 -- region: 62, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\62_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.659s.\n",
      "-- designing region: 1:158735001-158765000_strand_+_gene_63 -- region: 63, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\63_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 21.260s.\n",
      "-- designing region: 1:161235001-161265000_strand_+_gene_64 -- region: 64, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\64_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 1:163735001-163765000_strand_+_gene_65 -- region: 65, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\65_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: 1:166235001-166265000_strand_+_gene_66 -- region: 66, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\66_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.620s.\n",
      "-- designing region: 1:168735001-168765000_strand_+_gene_67 -- region: 67, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\67_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.553s.\n",
      "-- designing region: 1:171235001-171265000_strand_+_gene_68 -- region: 68, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\68_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 1:173735001-173765000_strand_+_gene_69 -- region: 69, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\69_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.318s.\n",
      "-- designing region: 1:176235001-176265000_strand_+_gene_70 -- region: 70, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\70_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.563s.\n",
      "-- designing region: 1:178735001-178765000_strand_+_gene_71 -- region: 71, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\71_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.554s.\n",
      "-- designing region: 1:181235001-181265000_strand_+_gene_72 -- region: 72, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\72_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.658s.\n",
      "-- designing region: 1:183735001-183765000_strand_+_gene_73 -- region: 73, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\73_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.559s.\n",
      "-- designing region: 1:186235001-186265000_strand_+_gene_74 -- region: 74, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\74_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.587s.\n",
      "-- designing region: 1:188735001-188765000_strand_+_gene_75 -- region: 75, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\75_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.608s.\n",
      "-- designing region: 1:191235001-191265000_strand_+_gene_76 -- region: 76, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\76_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.548s.\n",
      "-- designing region: 1:193735001-193765000_strand_+_gene_77 -- region: 77, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\77_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.473s.\n",
      "-- designing region: 1:196235001-196265000_strand_+_gene_78 -- region: 78, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\78_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.556s.\n",
      "-- designing region: 1:198735001-198765000_strand_+_gene_79 -- region: 79, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\79_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.556s.\n",
      "-- designing region: 1:201235001-201265000_strand_+_gene_80 -- region: 80, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\80_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 1:203735001-203765000_strand_+_gene_81 -- region: 81, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\81_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 1:206235001-206265000_strand_+_gene_82 -- region: 82, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\82_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 1:208735001-208765000_strand_+_gene_83 -- region: 83, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\83_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 1:211235001-211265000_strand_+_gene_84 -- region: 84, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\84_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.564s.\n",
      "-- designing region: 1:213735001-213765000_strand_+_gene_85 -- region: 85, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\85_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 1:216235001-216265000_strand_+_gene_86 -- region: 86, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\86_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 1:218735001-218765000_strand_+_gene_87 -- region: 87, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\87_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.602s.\n",
      "-- designing region: 1:221235001-221265000_strand_+_gene_88 -- region: 88, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\88_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.550s.\n",
      "-- designing region: 1:223735001-223765000_strand_+_gene_89 -- region: 89, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\89_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 1:226235001-226265000_strand_+_gene_90 -- region: 90, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\90_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.535s.\n",
      "-- designing region: 1:228735001-228765000_strand_+_gene_91 -- region: 91, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\91_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.635s.\n",
      "-- designing region: 1:231235001-231265000_strand_+_gene_92 -- region: 92, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\92_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 1:233735001-233765000_strand_+_gene_93 -- region: 93, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\93_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.612s.\n",
      "-- designing region: 1:236235001-236265000_strand_+_gene_94 -- region: 94, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\94_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.563s.\n",
      "-- designing region: 1:238735001-238765000_strand_+_gene_95 -- region: 95, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\95_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 1:241235001-241265000_strand_+_gene_96 -- region: 96, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\96_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 23.354s.\n",
      "-- designing region: 1:243735001-243765000_strand_+_gene_97 -- region: 97, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\97_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 18.353s.\n",
      "-- designing region: 1:246235001-246265000_strand_+_gene_98 -- region: 98, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\98_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.497s.\n",
      "-- designing region: 2:1235001-1265000_strand_+_gene_99 -- region: 99, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\99_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.439s.\n",
      "-- designing region: 2:3735001-3765000_strand_+_gene_100 -- region: 100, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\100_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.522s.\n",
      "-- designing region: 2:6235001-6265000_strand_+_gene_101 -- region: 101, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\101_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.490s.\n",
      "-- designing region: 2:8735001-8765000_strand_+_gene_102 -- region: 102, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\102_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 16.071s.\n",
      "-- designing region: 2:11235001-11265000_strand_+_gene_103 -- region: 103, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\103_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.448s.\n",
      "-- designing region: 2:13735001-13765000_strand_+_gene_104 -- region: 104, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\104_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.465s.\n",
      "-- designing region: 2:16235001-16265000_strand_+_gene_105 -- region: 105, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\105_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.482s.\n",
      "-- designing region: 2:18735001-18765000_strand_+_gene_106 -- region: 106, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\106_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.445s.\n",
      "-- designing region: 2:21235001-21265000_strand_+_gene_107 -- region: 107, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\107_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.474s.\n",
      "-- designing region: 2:23735001-23765000_strand_+_gene_108 -- region: 108, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\108_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.472s.\n",
      "-- designing region: 2:26235001-26265000_strand_+_gene_109 -- region: 109, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\109_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.439s.\n",
      "-- designing region: 2:28735001-28765000_strand_+_gene_110 -- region: 110, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\110_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.448s.\n",
      "-- designing region: 2:31235001-31265000_strand_+_gene_111 -- region: 111, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\111_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.491s.\n",
      "-- designing region: 2:33735001-33765000_strand_+_gene_112 -- region: 112, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\112_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.486s.\n",
      "-- designing region: 2:36235001-36265000_strand_+_gene_113 -- region: 113, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\113_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.456s.\n",
      "-- designing region: 2:38735001-38765000_strand_+_gene_114 -- region: 114, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\114_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.466s.\n",
      "-- designing region: 2:41235001-41265000_strand_+_gene_115 -- region: 115, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\115_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.440s.\n",
      "-- designing region: 2:43735001-43765000_strand_+_gene_116 -- region: 116, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\116_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.455s.\n",
      "-- designing region: 2:46235001-46265000_strand_+_gene_117 -- region: 117, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\117_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.493s.\n",
      "-- designing region: 2:48735001-48765000_strand_+_gene_118 -- region: 118, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\118_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.488s.\n",
      "-- designing region: 2:51235001-51265000_strand_+_gene_119 -- region: 119, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\119_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.446s.\n",
      "-- designing region: 2:53735001-53765000_strand_+_gene_120 -- region: 120, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\120_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.483s.\n",
      "-- designing region: 2:56235001-56265000_strand_+_gene_121 -- region: 121, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\121_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.497s.\n",
      "-- designing region: 2:58735001-58765000_strand_+_gene_122 -- region: 122, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\122_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.527s.\n",
      "-- designing region: 2:61235001-61265000_strand_+_gene_123 -- region: 123, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\123_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.498s.\n",
      "-- designing region: 2:63735001-63765000_strand_+_gene_124 -- region: 124, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\124_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.561s.\n",
      "-- designing region: 2:66235001-66265000_strand_+_gene_125 -- region: 125, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\125_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.532s.\n",
      "-- designing region: 2:68735001-68765000_strand_+_gene_126 -- region: 126, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\126_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.657s.\n",
      "-- designing region: 2:71235001-71265000_strand_+_gene_127 -- region: 127, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\127_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.541s.\n",
      "-- designing region: 2:73735001-73765000_strand_+_gene_128 -- region: 128, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\128_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.672s.\n",
      "-- designing region: 2:76235001-76265000_strand_+_gene_129 -- region: 129, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\129_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 2:78735001-78765000_strand_+_gene_130 -- region: 130, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\130_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.526s.\n",
      "-- designing region: 2:81235001-81265000_strand_+_gene_131 -- region: 131, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\131_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: 2:83735001-83765000_strand_+_gene_132 -- region: 132, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\132_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.609s.\n",
      "-- designing region: 2:86235001-86265000_strand_+_gene_133 -- region: 133, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\133_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 2:88735001-88765000_strand_+_gene_134 -- region: 134, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\134_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.570s.\n",
      "-- designing region: 2:91235001-91265000_strand_+_gene_135 -- region: 135, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\135_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.211s.\n",
      "- Designed 0 candidate probes in 0.516s.\n",
      "-- designing region: 2:93735001-93765000_strand_+_gene_136 -- region: 136, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\136_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.597s.\n",
      "- Designed 59918 candidate probes in 15.678s.\n",
      "-- designing region: 2:96235001-96265000_strand_+_gene_137 -- region: 137, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\137_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.570s.\n",
      "-- designing region: 2:98735001-98765000_strand_+_gene_138 -- region: 138, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\138_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.558s.\n",
      "-- designing region: 2:101235001-101265000_strand_+_gene_139 -- region: 139, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\139_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 25.058s.\n",
      "-- designing region: 2:103735001-103765000_strand_+_gene_140 -- region: 140, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\140_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.551s.\n",
      "-- designing region: 2:106235001-106265000_strand_+_gene_141 -- region: 141, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\141_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.516s.\n",
      "-- designing region: 2:108735001-108765000_strand_+_gene_142 -- region: 142, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\142_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.333s.\n",
      "-- designing region: 2:111235001-111265000_strand_+_gene_143 -- region: 143, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\143_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.625s.\n",
      "-- designing region: 2:113735001-113765000_strand_+_gene_144 -- region: 144, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\144_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.543s.\n",
      "-- designing region: 2:116235001-116265000_strand_+_gene_145 -- region: 145, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\145_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.561s.\n",
      "-- designing region: 2:118735001-118765000_strand_+_gene_146 -- region: 146, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\146_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 2:121235001-121265000_strand_+_gene_147 -- region: 147, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\147_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.632s.\n",
      "-- designing region: 2:123735001-123765000_strand_+_gene_148 -- region: 148, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\148_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.570s.\n",
      "-- designing region: 2:126235001-126265000_strand_+_gene_149 -- region: 149, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\149_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.569s.\n",
      "-- designing region: 2:128735001-128765000_strand_+_gene_150 -- region: 150, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\150_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.603s.\n",
      "-- designing region: 2:131235001-131265000_strand_+_gene_151 -- region: 151, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\151_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.590s.\n",
      "-- designing region: 2:133735001-133765000_strand_+_gene_152 -- region: 152, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\152_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 2:136235001-136265000_strand_+_gene_153 -- region: 153, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\153_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 2:138735001-138765000_strand_+_gene_154 -- region: 154, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\154_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.302s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 2:141235001-141265000_strand_+_gene_155 -- region: 155, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\155_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.620s.\n",
      "-- designing region: 2:143735001-143765000_strand_+_gene_156 -- region: 156, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\156_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 2:146235001-146265000_strand_+_gene_157 -- region: 157, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\157_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.930s.\n",
      "-- designing region: 2:148735001-148765000_strand_+_gene_158 -- region: 158, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\158_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 17.022s.\n",
      "-- designing region: 2:151235001-151265000_strand_+_gene_159 -- region: 159, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\159_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 2.802s.\n",
      "- Designed 59918 candidate probes in 18.827s.\n",
      "-- designing region: 2:153735001-153765000_strand_+_gene_160 -- region: 160, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\160_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.515s.\n",
      "-- designing region: 2:156235001-156265000_strand_+_gene_161 -- region: 161, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\161_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.531s.\n",
      "-- designing region: 2:158735001-158765000_strand_+_gene_162 -- region: 162, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\162_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.563s.\n",
      "-- designing region: 2:161235001-161265000_strand_+_gene_163 -- region: 163, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\163_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.503s.\n",
      "-- designing region: 2:163735001-163765000_strand_+_gene_164 -- region: 164, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\164_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.510s.\n",
      "-- designing region: 2:166235001-166265000_strand_+_gene_165 -- region: 165, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\165_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.466s.\n",
      "-- designing region: 2:168735001-168765000_strand_+_gene_166 -- region: 166, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\166_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.506s.\n",
      "-- designing region: 2:171235001-171265000_strand_+_gene_167 -- region: 167, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\167_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.491s.\n",
      "-- designing region: 2:173735001-173765000_strand_+_gene_168 -- region: 168, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\168_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.354s.\n",
      "-- designing region: 2:176235001-176265000_strand_+_gene_169 -- region: 169, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\169_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.498s.\n",
      "-- designing region: 2:178735001-178765000_strand_+_gene_170 -- region: 170, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\170_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.564s.\n",
      "-- designing region: 2:181235001-181265000_strand_+_gene_171 -- region: 171, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\171_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.513s.\n",
      "-- designing region: 2:183735001-183765000_strand_+_gene_172 -- region: 172, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\172_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.479s.\n",
      "-- designing region: 2:186235001-186265000_strand_+_gene_173 -- region: 173, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\173_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.453s.\n",
      "-- designing region: 2:188735001-188765000_strand_+_gene_174 -- region: 174, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\174_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.442s.\n",
      "-- designing region: 2:191235001-191265000_strand_+_gene_175 -- region: 175, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\175_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.473s.\n",
      "-- designing region: 2:193735001-193765000_strand_+_gene_176 -- region: 176, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\176_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.473s.\n",
      "-- designing region: 2:196235001-196265000_strand_+_gene_177 -- region: 177, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\177_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.470s.\n",
      "-- designing region: 2:198735001-198765000_strand_+_gene_178 -- region: 178, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\178_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: 2:201235001-201265000_strand_+_gene_179 -- region: 179, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\179_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.453s.\n",
      "-- designing region: 2:203735001-203765000_strand_+_gene_180 -- region: 180, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\180_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: 2:206235001-206265000_strand_+_gene_181 -- region: 181, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\181_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.747s.\n",
      "-- designing region: 2:208735001-208765000_strand_+_gene_182 -- region: 182, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\182_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.494s.\n",
      "-- designing region: 2:211235001-211265000_strand_+_gene_183 -- region: 183, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\183_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.496s.\n",
      "-- designing region: 2:213735001-213765000_strand_+_gene_184 -- region: 184, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\184_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.301s.\n",
      "- Designed 59918 candidate probes in 15.456s.\n",
      "-- designing region: 2:216235001-216265000_strand_+_gene_185 -- region: 185, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\185_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.542s.\n",
      "-- designing region: 2:218735001-218765000_strand_+_gene_186 -- region: 186, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\186_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.507s.\n",
      "-- designing region: 2:221235001-221265000_strand_+_gene_187 -- region: 187, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\187_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.488s.\n",
      "-- designing region: 2:223735001-223765000_strand_+_gene_188 -- region: 188, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\188_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.502s.\n",
      "-- designing region: 2:226235001-226265000_strand_+_gene_189 -- region: 189, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\189_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.500s.\n",
      "-- designing region: 2:228735001-228765000_strand_+_gene_190 -- region: 190, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\190_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 27.791s.\n",
      "-- designing region: 2:231235001-231265000_strand_+_gene_191 -- region: 191, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\191_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.565s.\n",
      "-- designing region: 2:233735001-233765000_strand_+_gene_192 -- region: 192, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\192_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.583s.\n",
      "-- designing region: 2:236235001-236265000_strand_+_gene_193 -- region: 193, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\193_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.655s.\n",
      "-- designing region: 2:238735001-238765000_strand_+_gene_194 -- region: 194, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\194_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.624s.\n",
      "-- designing region: 3:1235001-1265000_strand_+_gene_195 -- region: 195, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\195_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.589s.\n",
      "-- designing region: 3:3735001-3765000_strand_+_gene_196 -- region: 196, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\196_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.606s.\n",
      "-- designing region: 3:6235001-6265000_strand_+_gene_197 -- region: 197, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\197_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.584s.\n",
      "-- designing region: 3:8735001-8765000_strand_+_gene_198 -- region: 198, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\198_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.308s.\n",
      "- Designed 59918 candidate probes in 15.653s.\n",
      "-- designing region: 3:11235001-11265000_strand_+_gene_199 -- region: 199, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\199_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 16.750s.\n",
      "-- designing region: 3:13735001-13765000_strand_+_gene_200 -- region: 200, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\200_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 3:16235001-16265000_strand_+_gene_201 -- region: 201, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\201_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.473s.\n",
      "-- designing region: 3:18735001-18765000_strand_+_gene_202 -- region: 202, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\202_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.302s.\n",
      "- Designed 59918 candidate probes in 15.523s.\n",
      "-- designing region: 3:21235001-21265000_strand_+_gene_203 -- region: 203, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\203_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.560s.\n",
      "-- designing region: 3:23735001-23765000_strand_+_gene_204 -- region: 204, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\204_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 3:26235001-26265000_strand_+_gene_205 -- region: 205, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\205_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 3:28735001-28765000_strand_+_gene_206 -- region: 206, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\206_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.304s.\n",
      "- Designed 59918 candidate probes in 15.547s.\n",
      "-- designing region: 3:31235001-31265000_strand_+_gene_207 -- region: 207, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\207_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.554s.\n",
      "-- designing region: 3:33735001-33765000_strand_+_gene_208 -- region: 208, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\208_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.509s.\n",
      "-- designing region: 3:36235001-36265000_strand_+_gene_209 -- region: 209, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\209_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.559s.\n",
      "-- designing region: 3:38735001-38765000_strand_+_gene_210 -- region: 210, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\210_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.546s.\n",
      "-- designing region: 3:41235001-41265000_strand_+_gene_211 -- region: 211, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\211_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.584s.\n",
      "-- designing region: 3:43735001-43765000_strand_+_gene_212 -- region: 212, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\212_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.549s.\n",
      "-- designing region: 3:46235001-46265000_strand_+_gene_213 -- region: 213, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\213_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.558s.\n",
      "-- designing region: 3:48735001-48765000_strand_+_gene_214 -- region: 214, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\214_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.537s.\n",
      "-- designing region: 3:51235001-51265000_strand_+_gene_215 -- region: 215, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\215_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.572s.\n",
      "-- designing region: 3:53735001-53765000_strand_+_gene_216 -- region: 216, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\216_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.673s.\n",
      "-- designing region: 3:56235001-56265000_strand_+_gene_217 -- region: 217, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\217_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.611s.\n",
      "-- designing region: 3:58735001-58765000_strand_+_gene_218 -- region: 218, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\218_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.859s.\n",
      "-- designing region: 3:61235001-61265000_strand_+_gene_219 -- region: 219, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\219_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 18.882s.\n",
      "-- designing region: 3:63735001-63765000_strand_+_gene_220 -- region: 220, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\220_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 25.131s.\n",
      "-- designing region: 3:66235001-66265000_strand_+_gene_221 -- region: 221, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\221_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.338s.\n",
      "-- designing region: 3:68735001-68765000_strand_+_gene_222 -- region: 222, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\222_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 3:71235001-71265000_strand_+_gene_223 -- region: 223, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\223_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.642s.\n",
      "-- designing region: 3:73735001-73765000_strand_+_gene_224 -- region: 224, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\224_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.303s.\n",
      "- Designed 59918 candidate probes in 15.684s.\n",
      "-- designing region: 3:76235001-76265000_strand_+_gene_225 -- region: 225, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\225_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.580s.\n",
      "-- designing region: 3:78735001-78765000_strand_+_gene_226 -- region: 226, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\226_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.312s.\n",
      "- Designed 59918 candidate probes in 15.609s.\n",
      "-- designing region: 3:81235001-81265000_strand_+_gene_227 -- region: 227, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\227_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: 3:83735001-83765000_strand_+_gene_228 -- region: 228, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\228_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.312s.\n",
      "- Designed 59918 candidate probes in 15.601s.\n",
      "-- designing region: 3:86235001-86265000_strand_+_gene_229 -- region: 229, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\229_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.567s.\n",
      "-- designing region: 3:88735001-88765000_strand_+_gene_230 -- region: 230, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\230_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 3:91235001-91265000_strand_+_gene_231 -- region: 231, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\231_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.236s.\n",
      "- Designed 41860 candidate probes in 11.001s.\n",
      "-- designing region: 3:93735001-93765000_strand_+_gene_232 -- region: 232, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\232_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.481s.\n",
      "-- designing region: 3:96235001-96265000_strand_+_gene_233 -- region: 233, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\233_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.475s.\n",
      "-- designing region: 3:98735001-98765000_strand_+_gene_234 -- region: 234, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\234_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.551s.\n",
      "-- designing region: 3:101235001-101265000_strand_+_gene_235 -- region: 235, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\235_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 3:103735001-103765000_strand_+_gene_236 -- region: 236, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\236_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.546s.\n",
      "-- designing region: 3:106235001-106265000_strand_+_gene_237 -- region: 237, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\237_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 3:108735001-108765000_strand_+_gene_238 -- region: 238, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\238_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 3:111235001-111265000_strand_+_gene_239 -- region: 239, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\239_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.614s.\n",
      "-- designing region: 3:113735001-113765000_strand_+_gene_240 -- region: 240, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\240_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.585s.\n",
      "-- designing region: 3:116235001-116265000_strand_+_gene_241 -- region: 241, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\241_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.604s.\n",
      "-- designing region: 3:118735001-118765000_strand_+_gene_242 -- region: 242, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\242_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.680s.\n",
      "-- designing region: 3:121235001-121265000_strand_+_gene_243 -- region: 243, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\243_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 3:123735001-123765000_strand_+_gene_244 -- region: 244, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\244_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.607s.\n",
      "-- designing region: 3:126235001-126265000_strand_+_gene_245 -- region: 245, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\245_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 3:128735001-128765000_strand_+_gene_246 -- region: 246, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\246_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.647s.\n",
      "-- designing region: 3:131235001-131265000_strand_+_gene_247 -- region: 247, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\247_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.638s.\n",
      "-- designing region: 3:133735001-133765000_strand_+_gene_248 -- region: 248, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\248_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 3:136235001-136265000_strand_+_gene_249 -- region: 249, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\249_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.602s.\n",
      "-- designing region: 3:138735001-138765000_strand_+_gene_250 -- region: 250, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\250_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.626s.\n",
      "-- designing region: 3:141235001-141265000_strand_+_gene_251 -- region: 251, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\251_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.624s.\n",
      "-- designing region: 3:143735001-143765000_strand_+_gene_252 -- region: 252, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\252_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.676s.\n",
      "-- designing region: 3:146235001-146265000_strand_+_gene_253 -- region: 253, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\253_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 3:148735001-148765000_strand_+_gene_254 -- region: 254, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\254_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.559s.\n",
      "-- designing region: 3:151235001-151265000_strand_+_gene_255 -- region: 255, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\255_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 31.099s.\n",
      "-- designing region: 3:153735001-153765000_strand_+_gene_256 -- region: 256, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\256_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.519s.\n",
      "-- designing region: 3:156235001-156265000_strand_+_gene_257 -- region: 257, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\257_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.616s.\n",
      "-- designing region: 3:158735001-158765000_strand_+_gene_258 -- region: 258, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\258_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.604s.\n",
      "-- designing region: 3:161235001-161265000_strand_+_gene_259 -- region: 259, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\259_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.528s.\n",
      "-- designing region: 3:163735001-163765000_strand_+_gene_260 -- region: 260, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\260_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 3:166235001-166265000_strand_+_gene_261 -- region: 261, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\261_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.530s.\n",
      "-- designing region: 3:168735001-168765000_strand_+_gene_262 -- region: 262, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\262_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.460s.\n",
      "-- designing region: 3:171235001-171265000_strand_+_gene_263 -- region: 263, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\263_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.563s.\n",
      "-- designing region: 3:173735001-173765000_strand_+_gene_264 -- region: 264, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\264_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.523s.\n",
      "-- designing region: 3:176235001-176265000_strand_+_gene_265 -- region: 265, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\265_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.533s.\n",
      "-- designing region: 3:178735001-178765000_strand_+_gene_266 -- region: 266, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\266_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.485s.\n",
      "-- designing region: 3:181235001-181265000_strand_+_gene_267 -- region: 267, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\267_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.545s.\n",
      "-- designing region: 3:183735001-183765000_strand_+_gene_268 -- region: 268, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\268_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.550s.\n",
      "-- designing region: 3:186235001-186265000_strand_+_gene_269 -- region: 269, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\269_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.568s.\n",
      "-- designing region: 3:188735001-188765000_strand_+_gene_270 -- region: 270, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\270_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 3:191235001-191265000_strand_+_gene_271 -- region: 271, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\271_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.537s.\n",
      "-- designing region: 3:193735001-193765000_strand_+_gene_272 -- region: 272, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\272_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 3:196235001-196265000_strand_+_gene_273 -- region: 273, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\273_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.540s.\n",
      "-- designing region: 4:1235001-1265000_strand_+_gene_274 -- region: 274, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\274_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.615s.\n",
      "-- designing region: 4:3735001-3765000_strand_+_gene_275 -- region: 275, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\275_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.673s.\n",
      "-- designing region: 4:6235001-6265000_strand_+_gene_276 -- region: 276, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\276_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.686s.\n",
      "-- designing region: 4:8735001-8765000_strand_+_gene_277 -- region: 277, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\277_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.580s.\n",
      "-- designing region: 4:11235001-11265000_strand_+_gene_278 -- region: 278, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\278_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.828s.\n",
      "-- designing region: 4:13735001-13765000_strand_+_gene_279 -- region: 279, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\279_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 18.755s.\n",
      "-- designing region: 4:16235001-16265000_strand_+_gene_280 -- region: 280, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\280_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 23.827s.\n",
      "-- designing region: 4:18735001-18765000_strand_+_gene_281 -- region: 281, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\281_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.507s.\n",
      "-- designing region: 4:21235001-21265000_strand_+_gene_282 -- region: 282, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\282_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 4:23735001-23765000_strand_+_gene_283 -- region: 283, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\283_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.637s.\n",
      "-- designing region: 4:26235001-26265000_strand_+_gene_284 -- region: 284, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\284_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.556s.\n",
      "-- designing region: 4:28735001-28765000_strand_+_gene_285 -- region: 285, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\285_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.562s.\n",
      "-- designing region: 4:31235001-31265000_strand_+_gene_286 -- region: 286, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\286_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.729s.\n",
      "-- designing region: 4:33735001-33765000_strand_+_gene_287 -- region: 287, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\287_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 4:36235001-36265000_strand_+_gene_288 -- region: 288, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\288_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.585s.\n",
      "-- designing region: 4:38735001-38765000_strand_+_gene_289 -- region: 289, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\289_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.621s.\n",
      "-- designing region: 4:41235001-41265000_strand_+_gene_290 -- region: 290, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\290_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.300s.\n",
      "- Designed 59918 candidate probes in 15.670s.\n",
      "-- designing region: 4:43735001-43765000_strand_+_gene_291 -- region: 291, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\291_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 4:46235001-46265000_strand_+_gene_292 -- region: 292, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\292_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: 4:48735001-48765000_strand_+_gene_293 -- region: 293, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\293_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.562s.\n",
      "-- designing region: 4:51235001-51265000_strand_+_gene_294 -- region: 294, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\294_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.237s.\n",
      "- Designed 59918 candidate probes in 15.361s.\n",
      "-- designing region: 4:53735001-53765000_strand_+_gene_295 -- region: 295, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\295_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.595s.\n",
      "-- designing region: 4:56235001-56265000_strand_+_gene_296 -- region: 296, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\296_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 4:58735001-58765000_strand_+_gene_297 -- region: 297, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\297_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: 4:61235001-61265000_strand_+_gene_298 -- region: 298, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\298_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.602s.\n",
      "-- designing region: 4:63735001-63765000_strand_+_gene_299 -- region: 299, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\299_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.300s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 4:66235001-66265000_strand_+_gene_300 -- region: 300, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\300_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.594s.\n",
      "-- designing region: 4:68735001-68765000_strand_+_gene_301 -- region: 301, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\301_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 4:71235001-71265000_strand_+_gene_302 -- region: 302, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\302_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.633s.\n",
      "-- designing region: 4:73735001-73765000_strand_+_gene_303 -- region: 303, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\303_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.641s.\n",
      "-- designing region: 4:76235001-76265000_strand_+_gene_304 -- region: 304, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\304_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.637s.\n",
      "-- designing region: 4:78735001-78765000_strand_+_gene_305 -- region: 305, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\305_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: 4:81235001-81265000_strand_+_gene_306 -- region: 306, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\306_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 4:83735001-83765000_strand_+_gene_307 -- region: 307, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\307_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.701s.\n",
      "-- designing region: 4:86235001-86265000_strand_+_gene_308 -- region: 308, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\308_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 4:88735001-88765000_strand_+_gene_309 -- region: 309, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\309_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 4:91235001-91265000_strand_+_gene_310 -- region: 310, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\310_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 4:93735001-93765000_strand_+_gene_311 -- region: 311, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\311_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 4:96235001-96265000_strand_+_gene_312 -- region: 312, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\312_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 4:98735001-98765000_strand_+_gene_313 -- region: 313, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\313_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.614s.\n",
      "-- designing region: 4:101235001-101265000_strand_+_gene_314 -- region: 314, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\314_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.644s.\n",
      "-- designing region: 4:103735001-103765000_strand_+_gene_315 -- region: 315, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\315_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.571s.\n",
      "-- designing region: 4:106235001-106265000_strand_+_gene_316 -- region: 316, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\316_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.503s.\n",
      "-- designing region: 4:108735001-108765000_strand_+_gene_317 -- region: 317, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\317_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.626s.\n",
      "-- designing region: 4:111235001-111265000_strand_+_gene_318 -- region: 318, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\318_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.555s.\n",
      "-- designing region: 4:113735001-113765000_strand_+_gene_319 -- region: 319, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\319_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 4:116235001-116265000_strand_+_gene_320 -- region: 320, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\320_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.550s.\n",
      "-- designing region: 4:118735001-118765000_strand_+_gene_321 -- region: 321, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\321_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 4:121235001-121265000_strand_+_gene_322 -- region: 322, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\322_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 4:123735001-123765000_strand_+_gene_323 -- region: 323, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\323_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: 4:126235001-126265000_strand_+_gene_324 -- region: 324, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\324_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.529s.\n",
      "-- designing region: 4:128735001-128765000_strand_+_gene_325 -- region: 325, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\325_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.547s.\n",
      "-- designing region: 4:131235001-131265000_strand_+_gene_326 -- region: 326, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\326_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.568s.\n",
      "-- designing region: 4:133735001-133765000_strand_+_gene_327 -- region: 327, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\327_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.535s.\n",
      "-- designing region: 4:136235001-136265000_strand_+_gene_328 -- region: 328, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\328_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.592s.\n",
      "-- designing region: 4:138735001-138765000_strand_+_gene_329 -- region: 329, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\329_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.521s.\n",
      "-- designing region: 4:141235001-141265000_strand_+_gene_330 -- region: 330, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\330_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.530s.\n",
      "-- designing region: 4:143735001-143765000_strand_+_gene_331 -- region: 331, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\331_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.496s.\n",
      "-- designing region: 4:146235001-146265000_strand_+_gene_332 -- region: 332, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\332_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.665s.\n",
      "-- designing region: 4:148735001-148765000_strand_+_gene_333 -- region: 333, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\333_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.525s.\n",
      "-- designing region: 4:151235001-151265000_strand_+_gene_334 -- region: 334, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\334_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.551s.\n",
      "-- designing region: 4:153735001-153765000_strand_+_gene_335 -- region: 335, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\335_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 35.304s.\n",
      "-- designing region: 4:156235001-156265000_strand_+_gene_336 -- region: 336, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\336_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.563s.\n",
      "-- designing region: 4:158735001-158765000_strand_+_gene_337 -- region: 337, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\337_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.716s.\n",
      "-- designing region: 4:161235001-161265000_strand_+_gene_338 -- region: 338, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\338_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.556s.\n",
      "-- designing region: 4:163735001-163765000_strand_+_gene_339 -- region: 339, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\339_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 25.287s.\n",
      "-- designing region: 4:166235001-166265000_strand_+_gene_340 -- region: 340, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\340_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.700s.\n",
      "- Designed 59918 candidate probes in 18.302s.\n",
      "-- designing region: 4:168735001-168765000_strand_+_gene_341 -- region: 341, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\341_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.527s.\n",
      "-- designing region: 4:171235001-171265000_strand_+_gene_342 -- region: 342, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\342_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 4:173735001-173765000_strand_+_gene_343 -- region: 343, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\343_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.653s.\n",
      "-- designing region: 4:176235001-176265000_strand_+_gene_344 -- region: 344, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\344_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.622s.\n",
      "-- designing region: 4:178735001-178765000_strand_+_gene_345 -- region: 345, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\345_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.595s.\n",
      "-- designing region: 4:181235001-181265000_strand_+_gene_346 -- region: 346, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\346_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 4:183735001-183765000_strand_+_gene_347 -- region: 347, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\347_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.671s.\n",
      "-- designing region: 4:186235001-186265000_strand_+_gene_348 -- region: 348, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\348_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.595s.\n",
      "-- designing region: 4:188735001-188765000_strand_+_gene_349 -- region: 349, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\349_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 5:1235001-1265000_strand_+_gene_350 -- region: 350, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\350_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.677s.\n",
      "-- designing region: 5:3735001-3765000_strand_+_gene_351 -- region: 351, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\351_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 5:6235001-6265000_strand_+_gene_352 -- region: 352, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\352_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 5:8735001-8765000_strand_+_gene_353 -- region: 353, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\353_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 5:11235001-11265000_strand_+_gene_354 -- region: 354, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\354_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.635s.\n",
      "-- designing region: 5:13735001-13765000_strand_+_gene_355 -- region: 355, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\355_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.611s.\n",
      "-- designing region: 5:16235001-16265000_strand_+_gene_356 -- region: 356, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\356_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: 5:18735001-18765000_strand_+_gene_357 -- region: 357, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\357_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.572s.\n",
      "-- designing region: 5:21235001-21265000_strand_+_gene_358 -- region: 358, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\358_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.568s.\n",
      "-- designing region: 5:23735001-23765000_strand_+_gene_359 -- region: 359, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\359_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.534s.\n",
      "-- designing region: 5:26235001-26265000_strand_+_gene_360 -- region: 360, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\360_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.533s.\n",
      "-- designing region: 5:28735001-28765000_strand_+_gene_361 -- region: 361, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\361_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.612s.\n",
      "-- designing region: 5:31235001-31265000_strand_+_gene_362 -- region: 362, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\362_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.614s.\n",
      "-- designing region: 5:33735001-33765000_strand_+_gene_363 -- region: 363, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\363_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 5:36235001-36265000_strand_+_gene_364 -- region: 364, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\364_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 5:38735001-38765000_strand_+_gene_365 -- region: 365, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\365_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.654s.\n",
      "-- designing region: 5:41235001-41265000_strand_+_gene_366 -- region: 366, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\366_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.609s.\n",
      "-- designing region: 5:43735001-43765000_strand_+_gene_367 -- region: 367, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\367_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 5:46235001-46265000_strand_+_gene_368 -- region: 368, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\368_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.259s.\n",
      "- Designed 59918 candidate probes in 15.511s.\n",
      "-- designing region: 5:48735001-48765000_strand_+_gene_369 -- region: 369, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\369_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.233s.\n",
      "- Designed 59918 candidate probes in 15.364s.\n",
      "-- designing region: 5:51235001-51265000_strand_+_gene_370 -- region: 370, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\370_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.486s.\n",
      "-- designing region: 5:53735001-53765000_strand_+_gene_371 -- region: 371, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\371_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.533s.\n",
      "-- designing region: 5:56235001-56265000_strand_+_gene_372 -- region: 372, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\372_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 5:58735001-58765000_strand_+_gene_373 -- region: 373, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\373_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.615s.\n",
      "-- designing region: 5:61235001-61265000_strand_+_gene_374 -- region: 374, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\374_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 5:63735001-63765000_strand_+_gene_375 -- region: 375, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\375_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.528s.\n",
      "-- designing region: 5:66235001-66265000_strand_+_gene_376 -- region: 376, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\376_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.587s.\n",
      "-- designing region: 5:68735001-68765000_strand_+_gene_377 -- region: 377, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\377_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 5:71235001-71265000_strand_+_gene_378 -- region: 378, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\378_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.525s.\n",
      "-- designing region: 5:73735001-73765000_strand_+_gene_379 -- region: 379, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\379_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.539s.\n",
      "-- designing region: 5:76235001-76265000_strand_+_gene_380 -- region: 380, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\380_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.548s.\n",
      "-- designing region: 5:78735001-78765000_strand_+_gene_381 -- region: 381, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\381_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: 5:81235001-81265000_strand_+_gene_382 -- region: 382, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\382_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.588s.\n",
      "-- designing region: 5:83735001-83765000_strand_+_gene_383 -- region: 383, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\383_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.503s.\n",
      "-- designing region: 5:86235001-86265000_strand_+_gene_384 -- region: 384, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\384_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.535s.\n",
      "-- designing region: 5:88735001-88765000_strand_+_gene_385 -- region: 385, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\385_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.519s.\n",
      "-- designing region: 5:91235001-91265000_strand_+_gene_386 -- region: 386, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\386_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.529s.\n",
      "-- designing region: 5:93735001-93765000_strand_+_gene_387 -- region: 387, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\387_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.508s.\n",
      "-- designing region: 5:96235001-96265000_strand_+_gene_388 -- region: 388, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\388_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.548s.\n",
      "-- designing region: 5:98735001-98765000_strand_+_gene_389 -- region: 389, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\389_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.522s.\n",
      "-- designing region: 5:101235001-101265000_strand_+_gene_390 -- region: 390, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\390_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.502s.\n",
      "-- designing region: 5:103735001-103765000_strand_+_gene_391 -- region: 391, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\391_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.510s.\n",
      "-- designing region: 5:106235001-106265000_strand_+_gene_392 -- region: 392, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\392_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.510s.\n",
      "-- designing region: 5:108735001-108765000_strand_+_gene_393 -- region: 393, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\393_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 18.066s.\n",
      "-- designing region: 5:111235001-111265000_strand_+_gene_394 -- region: 394, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\394_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.469s.\n",
      "-- designing region: 5:113735001-113765000_strand_+_gene_395 -- region: 395, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\395_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.446s.\n",
      "-- designing region: 5:116235001-116265000_strand_+_gene_396 -- region: 396, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\396_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.544s.\n",
      "-- designing region: 5:118735001-118765000_strand_+_gene_397 -- region: 397, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\397_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.553s.\n",
      "-- designing region: 5:121235001-121265000_strand_+_gene_398 -- region: 398, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\398_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.724s.\n",
      "-- designing region: 5:123735001-123765000_strand_+_gene_399 -- region: 399, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\399_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 16.902s.\n",
      "-- designing region: 5:126235001-126265000_strand_+_gene_400 -- region: 400, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\400_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 2.634s.\n",
      "- Designed 59918 candidate probes in 26.355s.\n",
      "-- designing region: 5:128735001-128765000_strand_+_gene_401 -- region: 401, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\401_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.500s.\n",
      "-- designing region: 5:131235001-131265000_strand_+_gene_402 -- region: 402, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\402_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.571s.\n",
      "-- designing region: 5:133735001-133765000_strand_+_gene_403 -- region: 403, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\403_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.633s.\n",
      "-- designing region: 5:136235001-136265000_strand_+_gene_404 -- region: 404, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\404_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 5:138735001-138765000_strand_+_gene_405 -- region: 405, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\405_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.675s.\n",
      "-- designing region: 5:141235001-141265000_strand_+_gene_406 -- region: 406, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\406_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.611s.\n",
      "-- designing region: 5:143735001-143765000_strand_+_gene_407 -- region: 407, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\407_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.587s.\n",
      "-- designing region: 5:146235001-146265000_strand_+_gene_408 -- region: 408, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\408_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.566s.\n",
      "-- designing region: 5:148735001-148765000_strand_+_gene_409 -- region: 409, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\409_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.302s.\n",
      "- Designed 59918 candidate probes in 15.609s.\n",
      "-- designing region: 5:151235001-151265000_strand_+_gene_410 -- region: 410, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\410_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.657s.\n",
      "-- designing region: 5:153735001-153765000_strand_+_gene_411 -- region: 411, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\411_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.645s.\n",
      "-- designing region: 5:156235001-156265000_strand_+_gene_412 -- region: 412, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\412_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.627s.\n",
      "-- designing region: 5:158735001-158765000_strand_+_gene_413 -- region: 413, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\413_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.585s.\n",
      "-- designing region: 5:161235001-161265000_strand_+_gene_414 -- region: 414, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\414_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 5:163735001-163765000_strand_+_gene_415 -- region: 415, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\415_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 5:166235001-166265000_strand_+_gene_416 -- region: 416, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\416_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 5:168735001-168765000_strand_+_gene_417 -- region: 417, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\417_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.615s.\n",
      "-- designing region: 5:171235001-171265000_strand_+_gene_418 -- region: 418, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\418_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.577s.\n",
      "-- designing region: 5:173735001-173765000_strand_+_gene_419 -- region: 419, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\419_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 5:176235001-176265000_strand_+_gene_420 -- region: 420, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\420_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 5:178735001-178765000_strand_+_gene_421 -- region: 421, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\421_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 6:1235001-1265000_strand_+_gene_422 -- region: 422, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\422_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.604s.\n",
      "-- designing region: 6:3735001-3765000_strand_+_gene_423 -- region: 423, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\423_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 6:6235001-6265000_strand_+_gene_424 -- region: 424, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\424_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 6:8735001-8765000_strand_+_gene_425 -- region: 425, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\425_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.543s.\n",
      "-- designing region: 6:11235001-11265000_strand_+_gene_426 -- region: 426, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\426_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.658s.\n",
      "-- designing region: 6:13735001-13765000_strand_+_gene_427 -- region: 427, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\427_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.611s.\n",
      "-- designing region: 6:16235001-16265000_strand_+_gene_428 -- region: 428, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\428_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.583s.\n",
      "-- designing region: 6:18735001-18765000_strand_+_gene_429 -- region: 429, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\429_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.607s.\n",
      "-- designing region: 6:21235001-21265000_strand_+_gene_430 -- region: 430, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\430_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 6:23735001-23765000_strand_+_gene_431 -- region: 431, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\431_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.617s.\n",
      "-- designing region: 6:26235001-26265000_strand_+_gene_432 -- region: 432, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\432_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.715s.\n",
      "-- designing region: 6:28735001-28765000_strand_+_gene_433 -- region: 433, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\433_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.572s.\n",
      "-- designing region: 6:31235001-31265000_strand_+_gene_434 -- region: 434, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\434_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.576s.\n",
      "-- designing region: 6:33735001-33765000_strand_+_gene_435 -- region: 435, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\435_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 40.413s.\n",
      "-- designing region: 6:36235001-36265000_strand_+_gene_436 -- region: 436, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\436_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 6:38735001-38765000_strand_+_gene_437 -- region: 437, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\437_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: 6:41235001-41265000_strand_+_gene_438 -- region: 438, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\438_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.562s.\n",
      "-- designing region: 6:43735001-43765000_strand_+_gene_439 -- region: 439, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\439_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 6:46235001-46265000_strand_+_gene_440 -- region: 440, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\440_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.544s.\n",
      "-- designing region: 6:48735001-48765000_strand_+_gene_441 -- region: 441, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\441_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.576s.\n",
      "-- designing region: 6:51235001-51265000_strand_+_gene_442 -- region: 442, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\442_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 6:53735001-53765000_strand_+_gene_443 -- region: 443, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\443_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.558s.\n",
      "-- designing region: 6:56235001-56265000_strand_+_gene_444 -- region: 444, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\444_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.544s.\n",
      "-- designing region: 6:58735001-58765000_strand_+_gene_445 -- region: 445, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\445_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.226s.\n",
      "- Designed 59918 candidate probes in 15.332s.\n",
      "-- designing region: 6:61235001-61265000_strand_+_gene_446 -- region: 446, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\446_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.526s.\n",
      "-- designing region: 6:63735001-63765000_strand_+_gene_447 -- region: 447, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\447_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.558s.\n",
      "-- designing region: 6:66235001-66265000_strand_+_gene_448 -- region: 448, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\448_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.532s.\n",
      "-- designing region: 6:68735001-68765000_strand_+_gene_449 -- region: 449, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\449_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.534s.\n",
      "-- designing region: 6:71235001-71265000_strand_+_gene_450 -- region: 450, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\450_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.543s.\n",
      "-- designing region: 6:73735001-73765000_strand_+_gene_451 -- region: 451, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\451_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.572s.\n",
      "-- designing region: 6:76235001-76265000_strand_+_gene_452 -- region: 452, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\452_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.554s.\n",
      "-- designing region: 6:78735001-78765000_strand_+_gene_453 -- region: 453, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\453_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.540s.\n",
      "-- designing region: 6:81235001-81265000_strand_+_gene_454 -- region: 454, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\454_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.520s.\n",
      "-- designing region: 6:83735001-83765000_strand_+_gene_455 -- region: 455, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\455_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.567s.\n",
      "-- designing region: 6:86235001-86265000_strand_+_gene_456 -- region: 456, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\456_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.572s.\n",
      "-- designing region: 6:88735001-88765000_strand_+_gene_457 -- region: 457, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\457_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.565s.\n",
      "-- designing region: 6:91235001-91265000_strand_+_gene_458 -- region: 458, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\458_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.767s.\n",
      "-- designing region: 6:93735001-93765000_strand_+_gene_459 -- region: 459, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\459_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 22.793s.\n",
      "-- designing region: 6:96235001-96265000_strand_+_gene_460 -- region: 460, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\460_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.649s.\n",
      "- Designed 59918 candidate probes in 19.907s.\n",
      "-- designing region: 6:98735001-98765000_strand_+_gene_461 -- region: 461, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\461_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.553s.\n",
      "-- designing region: 6:101235001-101265000_strand_+_gene_462 -- region: 462, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\462_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 6:103735001-103765000_strand_+_gene_463 -- region: 463, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\463_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.661s.\n",
      "-- designing region: 6:106235001-106265000_strand_+_gene_464 -- region: 464, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\464_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.621s.\n",
      "-- designing region: 6:108735001-108765000_strand_+_gene_465 -- region: 465, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\465_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.667s.\n",
      "-- designing region: 6:111235001-111265000_strand_+_gene_466 -- region: 466, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\466_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.695s.\n",
      "-- designing region: 6:113735001-113765000_strand_+_gene_467 -- region: 467, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\467_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.648s.\n",
      "-- designing region: 6:116235001-116265000_strand_+_gene_468 -- region: 468, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\468_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.662s.\n",
      "-- designing region: 6:118735001-118765000_strand_+_gene_469 -- region: 469, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\469_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: 6:121235001-121265000_strand_+_gene_470 -- region: 470, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\470_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.670s.\n",
      "-- designing region: 6:123735001-123765000_strand_+_gene_471 -- region: 471, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\471_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.613s.\n",
      "-- designing region: 6:126235001-126265000_strand_+_gene_472 -- region: 472, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\472_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.625s.\n",
      "-- designing region: 6:128735001-128765000_strand_+_gene_473 -- region: 473, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\473_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.636s.\n",
      "-- designing region: 6:131235001-131265000_strand_+_gene_474 -- region: 474, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\474_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.638s.\n",
      "-- designing region: 6:133735001-133765000_strand_+_gene_475 -- region: 475, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\475_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 6:136235001-136265000_strand_+_gene_476 -- region: 476, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\476_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: 6:138735001-138765000_strand_+_gene_477 -- region: 477, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\477_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.608s.\n",
      "-- designing region: 6:141235001-141265000_strand_+_gene_478 -- region: 478, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\478_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 6:143735001-143765000_strand_+_gene_479 -- region: 479, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\479_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.669s.\n",
      "-- designing region: 6:146235001-146265000_strand_+_gene_480 -- region: 480, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\480_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.611s.\n",
      "-- designing region: 6:148735001-148765000_strand_+_gene_481 -- region: 481, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\481_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.602s.\n",
      "-- designing region: 6:151235001-151265000_strand_+_gene_482 -- region: 482, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\482_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.607s.\n",
      "-- designing region: 6:153735001-153765000_strand_+_gene_483 -- region: 483, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\483_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.689s.\n",
      "-- designing region: 6:156235001-156265000_strand_+_gene_484 -- region: 484, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\484_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.621s.\n",
      "-- designing region: 6:158735001-158765000_strand_+_gene_485 -- region: 485, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\485_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.632s.\n",
      "-- designing region: 6:161235001-161265000_strand_+_gene_486 -- region: 486, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\486_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 6:163735001-163765000_strand_+_gene_487 -- region: 487, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\487_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.672s.\n",
      "-- designing region: 6:166235001-166265000_strand_+_gene_488 -- region: 488, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\488_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.633s.\n",
      "-- designing region: 6:168735001-168765000_strand_+_gene_489 -- region: 489, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\489_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.627s.\n",
      "-- designing region: 7:1235001-1265000_strand_+_gene_490 -- region: 490, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\490_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.666s.\n",
      "-- designing region: 7:3735001-3765000_strand_+_gene_491 -- region: 491, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\491_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 7:6235001-6265000_strand_+_gene_492 -- region: 492, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\492_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 7:8735001-8765000_strand_+_gene_493 -- region: 493, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\493_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.569s.\n",
      "-- designing region: 7:11235001-11265000_strand_+_gene_494 -- region: 494, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\494_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.587s.\n",
      "-- designing region: 7:13735001-13765000_strand_+_gene_495 -- region: 495, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\495_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.563s.\n",
      "-- designing region: 7:16235001-16265000_strand_+_gene_496 -- region: 496, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\496_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.303s.\n",
      "- Designed 59918 candidate probes in 15.570s.\n",
      "-- designing region: 7:18735001-18765000_strand_+_gene_497 -- region: 497, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\497_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.543s.\n",
      "-- designing region: 7:21235001-21265000_strand_+_gene_498 -- region: 498, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\498_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.565s.\n",
      "-- designing region: 7:23735001-23765000_strand_+_gene_499 -- region: 499, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\499_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.547s.\n",
      "-- designing region: 7:26235001-26265000_strand_+_gene_500 -- region: 500, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\500_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.599s.\n",
      "-- designing region: 7:28735001-28765000_strand_+_gene_501 -- region: 501, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\501_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.603s.\n",
      "-- designing region: 7:31235001-31265000_strand_+_gene_502 -- region: 502, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\502_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.592s.\n",
      "-- designing region: 7:33735001-33765000_strand_+_gene_503 -- region: 503, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\503_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 7:36235001-36265000_strand_+_gene_504 -- region: 504, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\504_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 7:38735001-38765000_strand_+_gene_505 -- region: 505, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\505_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.569s.\n",
      "-- designing region: 7:41235001-41265000_strand_+_gene_506 -- region: 506, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\506_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.304s.\n",
      "- Designed 59918 candidate probes in 15.604s.\n",
      "-- designing region: 7:43735001-43765000_strand_+_gene_507 -- region: 507, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\507_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.493s.\n",
      "-- designing region: 7:46235001-46265000_strand_+_gene_508 -- region: 508, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\508_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.489s.\n",
      "-- designing region: 7:48735001-48765000_strand_+_gene_509 -- region: 509, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\509_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.538s.\n",
      "-- designing region: 7:51235001-51265000_strand_+_gene_510 -- region: 510, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\510_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.529s.\n",
      "-- designing region: 7:53735001-53765000_strand_+_gene_511 -- region: 511, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\511_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.536s.\n",
      "-- designing region: 7:56235001-56265000_strand_+_gene_512 -- region: 512, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\512_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.545s.\n",
      "-- designing region: 7:58735001-58765000_strand_+_gene_513 -- region: 513, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\513_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.228s.\n",
      "- Designed 59636 candidate probes in 15.237s.\n",
      "-- designing region: 7:61235001-61265000_strand_+_gene_514 -- region: 514, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\514_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.229s.\n",
      "- Designed 59918 candidate probes in 15.422s.\n",
      "-- designing region: 7:63735001-63765000_strand_+_gene_515 -- region: 515, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\515_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.253s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: 7:66235001-66265000_strand_+_gene_516 -- region: 516, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\516_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.560s.\n",
      "-- designing region: 7:68735001-68765000_strand_+_gene_517 -- region: 517, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\517_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.665s.\n",
      "-- designing region: 7:71235001-71265000_strand_+_gene_518 -- region: 518, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\518_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.648s.\n",
      "-- designing region: 7:73735001-73765000_strand_+_gene_519 -- region: 519, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\519_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.267s.\n",
      "- Designed 59918 candidate probes in 15.720s.\n",
      "-- designing region: 7:76235001-76265000_strand_+_gene_520 -- region: 520, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\520_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 21.128s.\n",
      "-- designing region: 7:78735001-78765000_strand_+_gene_521 -- region: 521, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\521_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.693s.\n",
      "- Designed 59918 candidate probes in 18.523s.\n",
      "-- designing region: 7:81235001-81265000_strand_+_gene_522 -- region: 522, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\522_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.527s.\n",
      "-- designing region: 7:83735001-83765000_strand_+_gene_523 -- region: 523, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\523_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.594s.\n",
      "-- designing region: 7:86235001-86265000_strand_+_gene_524 -- region: 524, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\524_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.657s.\n",
      "-- designing region: 7:88735001-88765000_strand_+_gene_525 -- region: 525, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\525_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.614s.\n",
      "-- designing region: 7:91235001-91265000_strand_+_gene_526 -- region: 526, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\526_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.616s.\n",
      "-- designing region: 7:93735001-93765000_strand_+_gene_527 -- region: 527, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\527_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.690s.\n",
      "-- designing region: 7:96235001-96265000_strand_+_gene_528 -- region: 528, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\528_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.637s.\n",
      "-- designing region: 7:98735001-98765000_strand_+_gene_529 -- region: 529, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\529_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.651s.\n",
      "-- designing region: 7:101235001-101265000_strand_+_gene_530 -- region: 530, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\530_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 7:103735001-103765000_strand_+_gene_531 -- region: 531, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\531_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.653s.\n",
      "-- designing region: 7:106235001-106265000_strand_+_gene_532 -- region: 532, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\532_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.632s.\n",
      "-- designing region: 7:108735001-108765000_strand_+_gene_533 -- region: 533, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\533_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 7:111235001-111265000_strand_+_gene_534 -- region: 534, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\534_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.633s.\n",
      "-- designing region: 7:113735001-113765000_strand_+_gene_535 -- region: 535, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\535_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.560s.\n",
      "-- designing region: 7:116235001-116265000_strand_+_gene_536 -- region: 536, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\536_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 7:118735001-118765000_strand_+_gene_537 -- region: 537, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\537_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 7:121235001-121265000_strand_+_gene_538 -- region: 538, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\538_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 7:123735001-123765000_strand_+_gene_539 -- region: 539, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\539_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.606s.\n",
      "-- designing region: 7:126235001-126265000_strand_+_gene_540 -- region: 540, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\540_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 7:128735001-128765000_strand_+_gene_541 -- region: 541, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\541_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.647s.\n",
      "-- designing region: 7:131235001-131265000_strand_+_gene_542 -- region: 542, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\542_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.662s.\n",
      "-- designing region: 7:133735001-133765000_strand_+_gene_543 -- region: 543, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\543_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.606s.\n",
      "-- designing region: 7:136235001-136265000_strand_+_gene_544 -- region: 544, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\544_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 7:138735001-138765000_strand_+_gene_545 -- region: 545, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\545_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.661s.\n",
      "-- designing region: 7:141235001-141265000_strand_+_gene_546 -- region: 546, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\546_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.671s.\n",
      "-- designing region: 7:143735001-143765000_strand_+_gene_547 -- region: 547, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\547_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: 7:146235001-146265000_strand_+_gene_548 -- region: 548, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\548_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: 7:148735001-148765000_strand_+_gene_549 -- region: 549, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\549_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 7:151235001-151265000_strand_+_gene_550 -- region: 550, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\550_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.742s.\n",
      "-- designing region: 7:153735001-153765000_strand_+_gene_551 -- region: 551, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\551_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 7:156235001-156265000_strand_+_gene_552 -- region: 552, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\552_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.675s.\n",
      "-- designing region: 8:1235001-1265000_strand_+_gene_553 -- region: 553, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\553_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.251s.\n",
      "- Designed 59918 candidate probes in 15.478s.\n",
      "-- designing region: 8:3735001-3765000_strand_+_gene_554 -- region: 554, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\554_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.537s.\n",
      "-- designing region: 8:6235001-6265000_strand_+_gene_555 -- region: 555, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\555_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.554s.\n",
      "-- designing region: 8:8735001-8765000_strand_+_gene_556 -- region: 556, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\556_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.657s.\n",
      "-- designing region: 8:11235001-11265000_strand_+_gene_557 -- region: 557, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\557_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.574s.\n",
      "-- designing region: 8:13735001-13765000_strand_+_gene_558 -- region: 558, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\558_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.525s.\n",
      "-- designing region: 8:16235001-16265000_strand_+_gene_559 -- region: 559, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\559_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.522s.\n",
      "-- designing region: 8:18735001-18765000_strand_+_gene_560 -- region: 560, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\560_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 46.748s.\n",
      "-- designing region: 8:21235001-21265000_strand_+_gene_561 -- region: 561, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\561_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 8:23735001-23765000_strand_+_gene_562 -- region: 562, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\562_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.559s.\n",
      "-- designing region: 8:26235001-26265000_strand_+_gene_563 -- region: 563, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\563_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.538s.\n",
      "-- designing region: 8:28735001-28765000_strand_+_gene_564 -- region: 564, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\564_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.638s.\n",
      "-- designing region: 8:31235001-31265000_strand_+_gene_565 -- region: 565, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\565_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 8:33735001-33765000_strand_+_gene_566 -- region: 566, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\566_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.509s.\n",
      "-- designing region: 8:36235001-36265000_strand_+_gene_567 -- region: 567, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\567_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.540s.\n",
      "-- designing region: 8:38735001-38765000_strand_+_gene_568 -- region: 568, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\568_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.554s.\n",
      "-- designing region: 8:41235001-41265000_strand_+_gene_569 -- region: 569, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\569_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.604s.\n",
      "-- designing region: 8:43735001-43765000_strand_+_gene_570 -- region: 570, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\570_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.253s.\n",
      "- Designed 59918 candidate probes in 15.495s.\n",
      "-- designing region: 8:46235001-46265000_strand_+_gene_571 -- region: 571, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\571_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 8:48735001-48765000_strand_+_gene_572 -- region: 572, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\572_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.550s.\n",
      "-- designing region: 8:51235001-51265000_strand_+_gene_573 -- region: 573, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\573_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.507s.\n",
      "-- designing region: 8:53735001-53765000_strand_+_gene_574 -- region: 574, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\574_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.571s.\n",
      "-- designing region: 8:56235001-56265000_strand_+_gene_575 -- region: 575, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\575_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 8:58735001-58765000_strand_+_gene_576 -- region: 576, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\576_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.637s.\n",
      "-- designing region: 8:61235001-61265000_strand_+_gene_577 -- region: 577, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\577_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.574s.\n",
      "-- designing region: 8:63735001-63765000_strand_+_gene_578 -- region: 578, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\578_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.710s.\n",
      "-- designing region: 8:66235001-66265000_strand_+_gene_579 -- region: 579, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\579_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 18.588s.\n",
      "-- designing region: 8:68735001-68765000_strand_+_gene_580 -- region: 580, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\580_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.527s.\n",
      "- Designed 59918 candidate probes in 22.086s.\n",
      "-- designing region: 8:71235001-71265000_strand_+_gene_581 -- region: 581, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\581_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.521s.\n",
      "-- designing region: 8:73735001-73765000_strand_+_gene_582 -- region: 582, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\582_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 8:76235001-76265000_strand_+_gene_583 -- region: 583, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\583_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.306s.\n",
      "- Designed 59918 candidate probes in 15.603s.\n",
      "-- designing region: 8:78735001-78765000_strand_+_gene_584 -- region: 584, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\584_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 8:81235001-81265000_strand_+_gene_585 -- region: 585, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\585_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.302s.\n",
      "- Designed 59918 candidate probes in 15.696s.\n",
      "-- designing region: 8:83735001-83765000_strand_+_gene_586 -- region: 586, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\586_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.653s.\n",
      "-- designing region: 8:86235001-86265000_strand_+_gene_587 -- region: 587, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\587_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.621s.\n",
      "-- designing region: 8:88735001-88765000_strand_+_gene_588 -- region: 588, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\588_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.704s.\n",
      "-- designing region: 8:91235001-91265000_strand_+_gene_589 -- region: 589, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\589_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 8:93735001-93765000_strand_+_gene_590 -- region: 590, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\590_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.701s.\n",
      "-- designing region: 8:96235001-96265000_strand_+_gene_591 -- region: 591, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\591_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.662s.\n",
      "-- designing region: 8:98735001-98765000_strand_+_gene_592 -- region: 592, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\592_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.653s.\n",
      "-- designing region: 8:101235001-101265000_strand_+_gene_593 -- region: 593, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\593_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.602s.\n",
      "-- designing region: 8:103735001-103765000_strand_+_gene_594 -- region: 594, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\594_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.640s.\n",
      "-- designing region: 8:106235001-106265000_strand_+_gene_595 -- region: 595, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\595_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.551s.\n",
      "-- designing region: 8:108735001-108765000_strand_+_gene_596 -- region: 596, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\596_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 8:111235001-111265000_strand_+_gene_597 -- region: 597, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\597_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.300s.\n",
      "- Designed 59918 candidate probes in 15.647s.\n",
      "-- designing region: 8:113735001-113765000_strand_+_gene_598 -- region: 598, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\598_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.585s.\n",
      "-- designing region: 8:116235001-116265000_strand_+_gene_599 -- region: 599, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\599_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.688s.\n",
      "-- designing region: 8:118735001-118765000_strand_+_gene_600 -- region: 600, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\600_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 8:121235001-121265000_strand_+_gene_601 -- region: 601, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\601_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 8:123735001-123765000_strand_+_gene_602 -- region: 602, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\602_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.589s.\n",
      "-- designing region: 8:126235001-126265000_strand_+_gene_603 -- region: 603, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\603_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 8:128735001-128765000_strand_+_gene_604 -- region: 604, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\604_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.608s.\n",
      "-- designing region: 8:131235001-131265000_strand_+_gene_605 -- region: 605, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\605_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.617s.\n",
      "-- designing region: 8:133735001-133765000_strand_+_gene_606 -- region: 606, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\606_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.680s.\n",
      "-- designing region: 8:136235001-136265000_strand_+_gene_607 -- region: 607, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\607_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 8:138735001-138765000_strand_+_gene_608 -- region: 608, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\608_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.640s.\n",
      "-- designing region: 8:141235001-141265000_strand_+_gene_609 -- region: 609, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\609_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.633s.\n",
      "-- designing region: 8:143735001-143765000_strand_+_gene_610 -- region: 610, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\610_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.654s.\n",
      "-- designing region: 9:985001-1015000_strand_+_gene_611 -- region: 611, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\611_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.615s.\n",
      "-- designing region: 9:2985001-3015000_strand_+_gene_612 -- region: 612, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\612_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 9:4985001-5015000_strand_+_gene_613 -- region: 613, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\613_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 9:6985001-7015000_strand_+_gene_614 -- region: 614, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\614_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.564s.\n",
      "-- designing region: 9:8985001-9015000_strand_+_gene_615 -- region: 615, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\615_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.583s.\n",
      "-- designing region: 9:10985001-11015000_strand_+_gene_616 -- region: 616, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\616_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.500s.\n",
      "-- designing region: 9:12985001-13015000_strand_+_gene_617 -- region: 617, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\617_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.555s.\n",
      "-- designing region: 9:14985001-15015000_strand_+_gene_618 -- region: 618, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\618_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.536s.\n",
      "-- designing region: 9:16985001-17015000_strand_+_gene_619 -- region: 619, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\619_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.513s.\n",
      "-- designing region: 9:18985001-19015000_strand_+_gene_620 -- region: 620, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\620_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.556s.\n",
      "-- designing region: 9:20985001-21015000_strand_+_gene_621 -- region: 621, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\621_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.531s.\n",
      "-- designing region: 9:22985001-23015000_strand_+_gene_622 -- region: 622, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\622_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.302s.\n",
      "- Designed 59918 candidate probes in 15.580s.\n",
      "-- designing region: 9:24985001-25015000_strand_+_gene_623 -- region: 623, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\623_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.266s.\n",
      "- Designed 59918 candidate probes in 15.510s.\n",
      "-- designing region: 9:26985001-27015000_strand_+_gene_624 -- region: 624, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\624_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.517s.\n",
      "-- designing region: 9:28985001-29015000_strand_+_gene_625 -- region: 625, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\625_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.553s.\n",
      "-- designing region: 9:30985001-31015000_strand_+_gene_626 -- region: 626, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\626_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.543s.\n",
      "-- designing region: 9:32985001-33015000_strand_+_gene_627 -- region: 627, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\627_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.592s.\n",
      "-- designing region: 9:34985001-35015000_strand_+_gene_628 -- region: 628, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\628_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 9:36985001-37015000_strand_+_gene_629 -- region: 629, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\629_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 9:38985001-39015000_strand_+_gene_630 -- region: 630, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\630_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.576s.\n",
      "-- designing region: 9:40985001-41015000_strand_+_gene_631 -- region: 631, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\631_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.621s.\n",
      "-- designing region: 9:42985001-43015000_strand_+_gene_632 -- region: 632, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\632_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.541s.\n",
      "-- designing region: 9:44985001-45015000_strand_+_gene_633 -- region: 633, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\633_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.225s.\n",
      "- Designed 59918 candidate probes in 15.290s.\n",
      "-- designing region: 9:46985001-47015000_strand_+_gene_634 -- region: 634, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\634_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.213s.\n",
      "- Designed 0 candidate probes in 0.507s.\n",
      "-- designing region: 9:48985001-49015000_strand_+_gene_635 -- region: 635, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\635_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.558s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: 9:50985001-51015000_strand_+_gene_636 -- region: 636, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\636_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.585s.\n",
      "- Designed 0 candidate probes in 0.877s.\n",
      "-- designing region: 9:52985001-53015000_strand_+_gene_637 -- region: 637, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\637_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.579s.\n",
      "- Designed 0 candidate probes in 0.872s.\n",
      "-- designing region: 9:54985001-55015000_strand_+_gene_638 -- region: 638, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\638_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.574s.\n",
      "- Designed 0 candidate probes in 0.868s.\n",
      "-- designing region: 9:56985001-57015000_strand_+_gene_639 -- region: 639, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\639_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.558s.\n",
      "- Designed 0 candidate probes in 0.852s.\n",
      "-- designing region: 9:58985001-59015000_strand_+_gene_640 -- region: 640, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\640_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.581s.\n",
      "- Designed 0 candidate probes in 0.875s.\n",
      "-- designing region: 9:60985001-61015000_strand_+_gene_641 -- region: 641, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\641_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.604s.\n",
      "- Designed 37692 candidate probes in 10.403s.\n",
      "-- designing region: 9:62985001-63015000_strand_+_gene_642 -- region: 642, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\642_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.233s.\n",
      "- Designed 13176 candidate probes in 4.141s.\n",
      "-- designing region: 9:64985001-65015000_strand_+_gene_643 -- region: 643, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\643_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.223s.\n",
      "- Designed 26166 candidate probes in 7.211s.\n",
      "-- designing region: 9:66985001-67015000_strand_+_gene_644 -- region: 644, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\644_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.545s.\n",
      "-- designing region: 9:68985001-69015000_strand_+_gene_645 -- region: 645, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\645_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.264s.\n",
      "- Designed 59918 candidate probes in 15.567s.\n",
      "-- designing region: 9:70985001-71015000_strand_+_gene_646 -- region: 646, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\646_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.547s.\n",
      "-- designing region: 9:72985001-73015000_strand_+_gene_647 -- region: 647, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\647_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 9:74985001-75015000_strand_+_gene_648 -- region: 648, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\648_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 9:76985001-77015000_strand_+_gene_649 -- region: 649, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\649_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 20.911s.\n",
      "-- designing region: 9:78985001-79015000_strand_+_gene_650 -- region: 650, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\650_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 18.399s.\n",
      "-- designing region: 9:80985001-81015000_strand_+_gene_651 -- region: 651, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\651_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.541s.\n",
      "-- designing region: 9:82985001-83015000_strand_+_gene_652 -- region: 652, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\652_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.690s.\n",
      "-- designing region: 9:84985001-85015000_strand_+_gene_653 -- region: 653, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\653_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.644s.\n",
      "-- designing region: 9:86985001-87015000_strand_+_gene_654 -- region: 654, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\654_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.669s.\n",
      "-- designing region: 9:88985001-89015000_strand_+_gene_655 -- region: 655, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\655_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.695s.\n",
      "-- designing region: 9:90985001-91015000_strand_+_gene_656 -- region: 656, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\656_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.738s.\n",
      "-- designing region: 9:92985001-93015000_strand_+_gene_657 -- region: 657, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\657_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.709s.\n",
      "-- designing region: 9:94985001-95015000_strand_+_gene_658 -- region: 658, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\658_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.673s.\n",
      "-- designing region: 9:96985001-97015000_strand_+_gene_659 -- region: 659, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\659_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 9:98985001-99015000_strand_+_gene_660 -- region: 660, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\660_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.747s.\n",
      "-- designing region: 9:100985001-101015000_strand_+_gene_661 -- region: 661, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\661_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.690s.\n",
      "-- designing region: 9:102985001-103015000_strand_+_gene_662 -- region: 662, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\662_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.538s.\n",
      "-- designing region: 9:104985001-105015000_strand_+_gene_663 -- region: 663, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\663_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 9:106985001-107015000_strand_+_gene_664 -- region: 664, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\664_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.654s.\n",
      "-- designing region: 9:108985001-109015000_strand_+_gene_665 -- region: 665, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\665_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 9:110985001-111015000_strand_+_gene_666 -- region: 666, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\666_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.594s.\n",
      "-- designing region: 9:112985001-113015000_strand_+_gene_667 -- region: 667, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\667_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 9:114985001-115015000_strand_+_gene_668 -- region: 668, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\668_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.570s.\n",
      "-- designing region: 9:116985001-117015000_strand_+_gene_669 -- region: 669, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\669_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.643s.\n",
      "-- designing region: 9:118985001-119015000_strand_+_gene_670 -- region: 670, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\670_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.601s.\n",
      "-- designing region: 9:120985001-121015000_strand_+_gene_671 -- region: 671, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\671_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 9:122985001-123015000_strand_+_gene_672 -- region: 672, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\672_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 9:124985001-125015000_strand_+_gene_673 -- region: 673, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\673_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.577s.\n",
      "-- designing region: 9:126985001-127015000_strand_+_gene_674 -- region: 674, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\674_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.738s.\n",
      "-- designing region: 9:128985001-129015000_strand_+_gene_675 -- region: 675, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\675_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.684s.\n",
      "-- designing region: 9:130985001-131015000_strand_+_gene_676 -- region: 676, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\676_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.696s.\n",
      "-- designing region: 9:132985001-133015000_strand_+_gene_677 -- region: 677, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\677_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 9:134985001-135015000_strand_+_gene_678 -- region: 678, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\678_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.698s.\n",
      "-- designing region: 9:136985001-137015000_strand_+_gene_679 -- region: 679, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\679_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.665s.\n",
      "-- designing region: 10:985001-1015000_strand_+_gene_680 -- region: 680, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\680_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.638s.\n",
      "-- designing region: 10:2985001-3015000_strand_+_gene_681 -- region: 681, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\681_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.690s.\n",
      "-- designing region: 10:4985001-5015000_strand_+_gene_682 -- region: 682, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\682_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 10:6985001-7015000_strand_+_gene_683 -- region: 683, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\683_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.624s.\n",
      "-- designing region: 10:8985001-9015000_strand_+_gene_684 -- region: 684, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\684_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 10:10985001-11015000_strand_+_gene_685 -- region: 685, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\685_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 10:12985001-13015000_strand_+_gene_686 -- region: 686, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\686_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 10:14985001-15015000_strand_+_gene_687 -- region: 687, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\687_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 10:16985001-17015000_strand_+_gene_688 -- region: 688, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\688_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.636s.\n",
      "-- designing region: 10:18985001-19015000_strand_+_gene_689 -- region: 689, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\689_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.539s.\n",
      "-- designing region: 10:20985001-21015000_strand_+_gene_690 -- region: 690, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\690_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.621s.\n",
      "-- designing region: 10:22985001-23015000_strand_+_gene_691 -- region: 691, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\691_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.567s.\n",
      "-- designing region: 10:24985001-25015000_strand_+_gene_692 -- region: 692, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\692_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.588s.\n",
      "-- designing region: 10:26985001-27015000_strand_+_gene_693 -- region: 693, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\693_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 16.019s.\n",
      "-- designing region: 10:28985001-29015000_strand_+_gene_694 -- region: 694, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\694_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.265s.\n",
      "- Designed 59918 candidate probes in 15.585s.\n",
      "-- designing region: 10:30985001-31015000_strand_+_gene_695 -- region: 695, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\695_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 10:32985001-33015000_strand_+_gene_696 -- region: 696, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\696_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.583s.\n",
      "-- designing region: 10:34985001-35015000_strand_+_gene_697 -- region: 697, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\697_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: 10:36985001-37015000_strand_+_gene_698 -- region: 698, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\698_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.544s.\n",
      "-- designing region: 10:38985001-39015000_strand_+_gene_699 -- region: 699, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\699_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 10:40985001-41015000_strand_+_gene_700 -- region: 700, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\700_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.227s.\n",
      "- Designed 59918 candidate probes in 15.326s.\n",
      "-- designing region: 10:42985001-43015000_strand_+_gene_701 -- region: 701, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\701_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.602s.\n",
      "-- designing region: 10:44985001-45015000_strand_+_gene_702 -- region: 702, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\702_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 10:46985001-47015000_strand_+_gene_703 -- region: 703, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\703_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 10:48985001-49015000_strand_+_gene_704 -- region: 704, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\704_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.633s.\n",
      "-- designing region: 10:50985001-51015000_strand_+_gene_705 -- region: 705, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\705_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.541s.\n",
      "-- designing region: 10:52985001-53015000_strand_+_gene_706 -- region: 706, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\706_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.580s.\n",
      "-- designing region: 10:54985001-55015000_strand_+_gene_707 -- region: 707, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\707_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 10:56985001-57015000_strand_+_gene_708 -- region: 708, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\708_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.779s.\n",
      "-- designing region: 10:58985001-59015000_strand_+_gene_709 -- region: 709, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\709_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.594s.\n",
      "-- designing region: 10:60985001-61015000_strand_+_gene_710 -- region: 710, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\710_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 22.786s.\n",
      "-- designing region: 10:62985001-63015000_strand_+_gene_711 -- region: 711, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\711_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.632s.\n",
      "- Designed 59918 candidate probes in 17.746s.\n",
      "-- designing region: 10:64985001-65015000_strand_+_gene_712 -- region: 712, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\712_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.572s.\n",
      "-- designing region: 10:66985001-67015000_strand_+_gene_713 -- region: 713, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\713_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.644s.\n",
      "-- designing region: 10:68985001-69015000_strand_+_gene_714 -- region: 714, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\714_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.643s.\n",
      "-- designing region: 10:70985001-71015000_strand_+_gene_715 -- region: 715, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\715_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.727s.\n",
      "-- designing region: 10:72985001-73015000_strand_+_gene_716 -- region: 716, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\716_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.669s.\n",
      "-- designing region: 10:74985001-75015000_strand_+_gene_717 -- region: 717, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\717_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.673s.\n",
      "-- designing region: 10:76985001-77015000_strand_+_gene_718 -- region: 718, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\718_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.730s.\n",
      "-- designing region: 10:78985001-79015000_strand_+_gene_719 -- region: 719, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\719_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.689s.\n",
      "-- designing region: 10:80985001-81015000_strand_+_gene_720 -- region: 720, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\720_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 10:82985001-83015000_strand_+_gene_721 -- region: 721, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\721_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.642s.\n",
      "-- designing region: 10:84985001-85015000_strand_+_gene_722 -- region: 722, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\722_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.665s.\n",
      "-- designing region: 10:86985001-87015000_strand_+_gene_723 -- region: 723, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\723_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 54.988s.\n",
      "-- designing region: 10:88985001-89015000_strand_+_gene_724 -- region: 724, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\724_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.673s.\n",
      "-- designing region: 10:90985001-91015000_strand_+_gene_725 -- region: 725, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\725_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.614s.\n",
      "-- designing region: 10:92985001-93015000_strand_+_gene_726 -- region: 726, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\726_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 10:94985001-95015000_strand_+_gene_727 -- region: 727, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\727_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.301s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 10:96985001-97015000_strand_+_gene_728 -- region: 728, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\728_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.656s.\n",
      "-- designing region: 10:98985001-99015000_strand_+_gene_729 -- region: 729, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\729_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 10:100985001-101015000_strand_+_gene_730 -- region: 730, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\730_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.681s.\n",
      "-- designing region: 10:102985001-103015000_strand_+_gene_731 -- region: 731, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\731_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.613s.\n",
      "-- designing region: 10:104985001-105015000_strand_+_gene_732 -- region: 732, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\732_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.302s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 10:106985001-107015000_strand_+_gene_733 -- region: 733, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\733_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.651s.\n",
      "-- designing region: 10:108985001-109015000_strand_+_gene_734 -- region: 734, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\734_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.599s.\n",
      "-- designing region: 10:110985001-111015000_strand_+_gene_735 -- region: 735, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\735_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 10:112985001-113015000_strand_+_gene_736 -- region: 736, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\736_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.651s.\n",
      "-- designing region: 10:114985001-115015000_strand_+_gene_737 -- region: 737, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\737_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.595s.\n",
      "-- designing region: 10:116985001-117015000_strand_+_gene_738 -- region: 738, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\738_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 10:118985001-119015000_strand_+_gene_739 -- region: 739, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\739_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.610s.\n",
      "-- designing region: 10:120985001-121015000_strand_+_gene_740 -- region: 740, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\740_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.632s.\n",
      "-- designing region: 10:122985001-123015000_strand_+_gene_741 -- region: 741, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\741_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 10:124985001-125015000_strand_+_gene_742 -- region: 742, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\742_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.640s.\n",
      "-- designing region: 10:126985001-127015000_strand_+_gene_743 -- region: 743, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\743_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.603s.\n",
      "-- designing region: 10:128985001-129015000_strand_+_gene_744 -- region: 744, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\744_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 10:130985001-131015000_strand_+_gene_745 -- region: 745, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\745_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.640s.\n",
      "-- designing region: 11:985001-1015000_strand_+_gene_746 -- region: 746, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\746_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 11:2985001-3015000_strand_+_gene_747 -- region: 747, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\747_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.622s.\n",
      "-- designing region: 11:4985001-5015000_strand_+_gene_748 -- region: 748, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\748_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.556s.\n",
      "-- designing region: 11:6985001-7015000_strand_+_gene_749 -- region: 749, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\749_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.583s.\n",
      "-- designing region: 11:8985001-9015000_strand_+_gene_750 -- region: 750, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\750_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.594s.\n",
      "-- designing region: 11:10985001-11015000_strand_+_gene_751 -- region: 751, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\751_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 11:12985001-13015000_strand_+_gene_752 -- region: 752, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\752_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 11:14985001-15015000_strand_+_gene_753 -- region: 753, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\753_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.599s.\n",
      "-- designing region: 11:16985001-17015000_strand_+_gene_754 -- region: 754, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\754_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: 11:18985001-19015000_strand_+_gene_755 -- region: 755, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\755_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.562s.\n",
      "-- designing region: 11:20985001-21015000_strand_+_gene_756 -- region: 756, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\756_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 11:22985001-23015000_strand_+_gene_757 -- region: 757, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\757_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.586s.\n",
      "-- designing region: 11:24985001-25015000_strand_+_gene_758 -- region: 758, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\758_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.627s.\n",
      "-- designing region: 11:26985001-27015000_strand_+_gene_759 -- region: 759, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\759_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.533s.\n",
      "-- designing region: 11:28985001-29015000_strand_+_gene_760 -- region: 760, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\760_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.518s.\n",
      "-- designing region: 11:30985001-31015000_strand_+_gene_761 -- region: 761, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\761_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.505s.\n",
      "-- designing region: 11:32985001-33015000_strand_+_gene_762 -- region: 762, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\762_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.507s.\n",
      "-- designing region: 11:34985001-35015000_strand_+_gene_763 -- region: 763, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\763_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 11:36985001-37015000_strand_+_gene_764 -- region: 764, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\764_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: 11:38985001-39015000_strand_+_gene_765 -- region: 765, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\765_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.565s.\n",
      "-- designing region: 11:40985001-41015000_strand_+_gene_766 -- region: 766, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\766_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.517s.\n",
      "-- designing region: 11:42985001-43015000_strand_+_gene_767 -- region: 767, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\767_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.761s.\n",
      "-- designing region: 11:44985001-45015000_strand_+_gene_768 -- region: 768, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\768_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.520s.\n",
      "-- designing region: 11:46985001-47015000_strand_+_gene_769 -- region: 769, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\769_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 22.997s.\n",
      "-- designing region: 11:48985001-49015000_strand_+_gene_770 -- region: 770, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\770_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.306s.\n",
      "- Designed 59918 candidate probes in 15.603s.\n",
      "-- designing region: 11:50985001-51015000_strand_+_gene_771 -- region: 771, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\771_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.210s.\n",
      "- Designed 0 candidate probes in 0.509s.\n",
      "-- designing region: 11:52985001-53015000_strand_+_gene_772 -- region: 772, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\772_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.595s.\n",
      "- Designed 59918 candidate probes in 15.892s.\n",
      "-- designing region: 11:54985001-55015000_strand_+_gene_773 -- region: 773, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\773_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.261s.\n",
      "- Designed 59918 candidate probes in 15.514s.\n",
      "-- designing region: 11:56985001-57015000_strand_+_gene_774 -- region: 774, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\774_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 11:58985001-59015000_strand_+_gene_775 -- region: 775, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\775_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.606s.\n",
      "-- designing region: 11:60985001-61015000_strand_+_gene_776 -- region: 776, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\776_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.682s.\n",
      "-- designing region: 11:62985001-63015000_strand_+_gene_777 -- region: 777, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\777_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.751s.\n",
      "-- designing region: 11:64985001-65015000_strand_+_gene_778 -- region: 778, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\778_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.655s.\n",
      "-- designing region: 11:66985001-67015000_strand_+_gene_779 -- region: 779, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\779_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.711s.\n",
      "-- designing region: 11:68985001-69015000_strand_+_gene_780 -- region: 780, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\780_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.707s.\n",
      "-- designing region: 11:70985001-71015000_strand_+_gene_781 -- region: 781, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\781_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.204s.\n",
      "- Designed 0 candidate probes in 0.511s.\n",
      "-- designing region: 11:72985001-73015000_strand_+_gene_782 -- region: 782, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\782_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.640s.\n",
      "- Designed 59918 candidate probes in 16.006s.\n",
      "-- designing region: 11:74985001-75015000_strand_+_gene_783 -- region: 783, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\783_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.635s.\n",
      "-- designing region: 11:76985001-77015000_strand_+_gene_784 -- region: 784, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\784_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.678s.\n",
      "-- designing region: 11:78985001-79015000_strand_+_gene_785 -- region: 785, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\785_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.650s.\n",
      "-- designing region: 11:80985001-81015000_strand_+_gene_786 -- region: 786, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\786_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.636s.\n",
      "-- designing region: 11:82985001-83015000_strand_+_gene_787 -- region: 787, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\787_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: 11:84985001-85015000_strand_+_gene_788 -- region: 788, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\788_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 11:86985001-87015000_strand_+_gene_789 -- region: 789, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\789_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.589s.\n",
      "-- designing region: 11:88985001-89015000_strand_+_gene_790 -- region: 790, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\790_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.634s.\n",
      "-- designing region: 11:90985001-91015000_strand_+_gene_791 -- region: 791, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\791_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.596s.\n",
      "-- designing region: 11:92985001-93015000_strand_+_gene_792 -- region: 792, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\792_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.668s.\n",
      "-- designing region: 11:94985001-95015000_strand_+_gene_793 -- region: 793, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\793_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 11:96985001-97015000_strand_+_gene_794 -- region: 794, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\794_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 20.724s.\n",
      "-- designing region: 11:98985001-99015000_strand_+_gene_795 -- region: 795, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\795_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.471s.\n",
      "-- designing region: 11:100985001-101015000_strand_+_gene_796 -- region: 796, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\796_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.453s.\n",
      "-- designing region: 11:102985001-103015000_strand_+_gene_797 -- region: 797, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\797_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.410s.\n",
      "-- designing region: 11:104985001-105015000_strand_+_gene_798 -- region: 798, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\798_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.447s.\n",
      "-- designing region: 11:106985001-107015000_strand_+_gene_799 -- region: 799, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\799_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.527s.\n",
      "-- designing region: 11:108985001-109015000_strand_+_gene_800 -- region: 800, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\800_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.588s.\n",
      "-- designing region: 11:110985001-111015000_strand_+_gene_801 -- region: 801, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\801_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.555s.\n",
      "-- designing region: 11:112985001-113015000_strand_+_gene_802 -- region: 802, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\802_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.541s.\n",
      "-- designing region: 11:114985001-115015000_strand_+_gene_803 -- region: 803, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\803_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.536s.\n",
      "-- designing region: 11:116985001-117015000_strand_+_gene_804 -- region: 804, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\804_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.547s.\n",
      "-- designing region: 11:118985001-119015000_strand_+_gene_805 -- region: 805, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\805_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 11:120985001-121015000_strand_+_gene_806 -- region: 806, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\806_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.566s.\n",
      "-- designing region: 11:122985001-123015000_strand_+_gene_807 -- region: 807, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\807_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.518s.\n",
      "-- designing region: 11:124985001-125015000_strand_+_gene_808 -- region: 808, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\808_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.540s.\n",
      "-- designing region: 11:126985001-127015000_strand_+_gene_809 -- region: 809, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\809_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: 11:128985001-129015000_strand_+_gene_810 -- region: 810, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\810_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.514s.\n",
      "-- designing region: 11:130985001-131015000_strand_+_gene_811 -- region: 811, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\811_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.545s.\n",
      "-- designing region: 11:132985001-133015000_strand_+_gene_812 -- region: 812, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\812_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.569s.\n",
      "-- designing region: 12:985001-1015000_strand_+_gene_813 -- region: 813, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\813_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.530s.\n",
      "-- designing region: 12:2985001-3015000_strand_+_gene_814 -- region: 814, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\814_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 12:4985001-5015000_strand_+_gene_815 -- region: 815, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\815_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.610s.\n",
      "-- designing region: 12:6985001-7015000_strand_+_gene_816 -- region: 816, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\816_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.533s.\n",
      "-- designing region: 12:8985001-9015000_strand_+_gene_817 -- region: 817, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\817_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.539s.\n",
      "-- designing region: 12:10985001-11015000_strand_+_gene_818 -- region: 818, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\818_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.484s.\n",
      "-- designing region: 12:12985001-13015000_strand_+_gene_819 -- region: 819, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\819_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.570s.\n",
      "-- designing region: 12:14985001-15015000_strand_+_gene_820 -- region: 820, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\820_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.560s.\n",
      "-- designing region: 12:16985001-17015000_strand_+_gene_821 -- region: 821, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\821_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.524s.\n",
      "-- designing region: 12:18985001-19015000_strand_+_gene_822 -- region: 822, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\822_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.460s.\n",
      "-- designing region: 12:20985001-21015000_strand_+_gene_823 -- region: 823, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\823_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.479s.\n",
      "-- designing region: 12:22985001-23015000_strand_+_gene_824 -- region: 824, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\824_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.585s.\n",
      "-- designing region: 12:24985001-25015000_strand_+_gene_825 -- region: 825, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\825_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.483s.\n",
      "-- designing region: 12:26985001-27015000_strand_+_gene_826 -- region: 826, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\826_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.547s.\n",
      "-- designing region: 12:28985001-29015000_strand_+_gene_827 -- region: 827, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\827_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.578s.\n",
      "-- designing region: 12:30985001-31015000_strand_+_gene_828 -- region: 828, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\828_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.531s.\n",
      "-- designing region: 12:32985001-33015000_strand_+_gene_829 -- region: 829, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\829_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.526s.\n",
      "-- designing region: 12:34985001-35015000_strand_+_gene_830 -- region: 830, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\830_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.217s.\n",
      "- Designed 59918 candidate probes in 15.718s.\n",
      "-- designing region: 12:36985001-37015000_strand_+_gene_831 -- region: 831, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\831_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.228s.\n",
      "- Designed 59918 candidate probes in 15.279s.\n",
      "-- designing region: 12:38985001-39015000_strand_+_gene_832 -- region: 832, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\832_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 20.195s.\n",
      "-- designing region: 12:40985001-41015000_strand_+_gene_833 -- region: 833, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\833_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 18.488s.\n",
      "-- designing region: 12:42985001-43015000_strand_+_gene_834 -- region: 834, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\834_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 12:44985001-45015000_strand_+_gene_835 -- region: 835, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\835_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.567s.\n",
      "-- designing region: 12:46985001-47015000_strand_+_gene_836 -- region: 836, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\836_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.633s.\n",
      "-- designing region: 12:48985001-49015000_strand_+_gene_837 -- region: 837, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\837_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.647s.\n",
      "-- designing region: 12:50985001-51015000_strand_+_gene_838 -- region: 838, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\838_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.625s.\n",
      "-- designing region: 12:52985001-53015000_strand_+_gene_839 -- region: 839, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\839_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.231s.\n",
      "- Designed 59918 candidate probes in 15.544s.\n",
      "-- designing region: 12:54985001-55015000_strand_+_gene_840 -- region: 840, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\840_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.641s.\n",
      "-- designing region: 12:56985001-57015000_strand_+_gene_841 -- region: 841, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\841_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.651s.\n",
      "-- designing region: 12:58985001-59015000_strand_+_gene_842 -- region: 842, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\842_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.675s.\n",
      "-- designing region: 12:60985001-61015000_strand_+_gene_843 -- region: 843, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\843_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.553s.\n",
      "-- designing region: 12:62985001-63015000_strand_+_gene_844 -- region: 844, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\844_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.605s.\n",
      "-- designing region: 12:64985001-65015000_strand_+_gene_845 -- region: 845, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\845_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 12:66985001-67015000_strand_+_gene_846 -- region: 846, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\846_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 12:68985001-69015000_strand_+_gene_847 -- region: 847, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\847_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.637s.\n",
      "-- designing region: 12:70985001-71015000_strand_+_gene_848 -- region: 848, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\848_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: 12:72985001-73015000_strand_+_gene_849 -- region: 849, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\849_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.636s.\n",
      "-- designing region: 12:74985001-75015000_strand_+_gene_850 -- region: 850, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\850_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.577s.\n",
      "-- designing region: 12:76985001-77015000_strand_+_gene_851 -- region: 851, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\851_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.604s.\n",
      "-- designing region: 12:78985001-79015000_strand_+_gene_852 -- region: 852, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\852_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 12:80985001-81015000_strand_+_gene_853 -- region: 853, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\853_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.586s.\n",
      "-- designing region: 12:82985001-83015000_strand_+_gene_854 -- region: 854, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\854_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 12:84985001-85015000_strand_+_gene_855 -- region: 855, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\855_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.535s.\n",
      "-- designing region: 12:86985001-87015000_strand_+_gene_856 -- region: 856, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\856_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.533s.\n",
      "-- designing region: 12:88985001-89015000_strand_+_gene_857 -- region: 857, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\857_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.608s.\n",
      "-- designing region: 12:90985001-91015000_strand_+_gene_858 -- region: 858, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\858_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: 12:92985001-93015000_strand_+_gene_859 -- region: 859, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\859_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 12:94985001-95015000_strand_+_gene_860 -- region: 860, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\860_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.556s.\n",
      "-- designing region: 12:96985001-97015000_strand_+_gene_861 -- region: 861, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\861_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.586s.\n",
      "-- designing region: 12:98985001-99015000_strand_+_gene_862 -- region: 862, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\862_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.563s.\n",
      "-- designing region: 12:100985001-101015000_strand_+_gene_863 -- region: 863, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\863_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 12:102985001-103015000_strand_+_gene_864 -- region: 864, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\864_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.592s.\n",
      "-- designing region: 12:104985001-105015000_strand_+_gene_865 -- region: 865, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\865_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.577s.\n",
      "-- designing region: 12:106985001-107015000_strand_+_gene_866 -- region: 866, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\866_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.541s.\n",
      "-- designing region: 12:108985001-109015000_strand_+_gene_867 -- region: 867, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\867_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.599s.\n",
      "-- designing region: 12:110985001-111015000_strand_+_gene_868 -- region: 868, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\868_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.560s.\n",
      "-- designing region: 12:112985001-113015000_strand_+_gene_869 -- region: 869, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\869_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.266s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 12:114985001-115015000_strand_+_gene_870 -- region: 870, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\870_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.583s.\n",
      "-- designing region: 12:116985001-117015000_strand_+_gene_871 -- region: 871, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\871_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.516s.\n",
      "-- designing region: 12:118985001-119015000_strand_+_gene_872 -- region: 872, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\872_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.311s.\n",
      "- Designed 59918 candidate probes in 15.635s.\n",
      "-- designing region: 12:120985001-121015000_strand_+_gene_873 -- region: 873, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\873_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.537s.\n",
      "-- designing region: 12:122985001-123015000_strand_+_gene_874 -- region: 874, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\874_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.601s.\n",
      "-- designing region: 12:124985001-125015000_strand_+_gene_875 -- region: 875, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\875_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.531s.\n",
      "-- designing region: 12:126985001-127015000_strand_+_gene_876 -- region: 876, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\876_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.571s.\n",
      "-- designing region: 12:128985001-129015000_strand_+_gene_877 -- region: 877, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\877_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.574s.\n",
      "-- designing region: 12:130985001-131015000_strand_+_gene_878 -- region: 878, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\878_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.489s.\n",
      "-- designing region: 13:985001-1015000_strand_+_gene_879 -- region: 879, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\879_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.209s.\n",
      "- Designed 0 candidate probes in 0.500s.\n",
      "-- designing region: 13:2985001-3015000_strand_+_gene_880 -- region: 880, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\880_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.565s.\n",
      "- Designed 0 candidate probes in 0.859s.\n",
      "-- designing region: 13:4985001-5015000_strand_+_gene_881 -- region: 881, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\881_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.844s.\n",
      "-- designing region: 13:6985001-7015000_strand_+_gene_882 -- region: 882, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\882_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.555s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: 13:8985001-9015000_strand_+_gene_883 -- region: 883, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\883_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.846s.\n",
      "-- designing region: 13:10985001-11015000_strand_+_gene_884 -- region: 884, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\884_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.559s.\n",
      "- Designed 0 candidate probes in 0.850s.\n",
      "-- designing region: 13:12985001-13015000_strand_+_gene_885 -- region: 885, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\885_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.555s.\n",
      "- Designed 0 candidate probes in 0.846s.\n",
      "-- designing region: 13:14985001-15015000_strand_+_gene_886 -- region: 886, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\886_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.561s.\n",
      "- Designed 0 candidate probes in 0.853s.\n",
      "-- designing region: 13:16985001-17015000_strand_+_gene_887 -- region: 887, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\887_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.570s.\n",
      "- Designed 59918 candidate probes in 15.641s.\n",
      "-- designing region: 13:18985001-19015000_strand_+_gene_888 -- region: 888, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\888_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.453s.\n",
      "-- designing region: 13:20985001-21015000_strand_+_gene_889 -- region: 889, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\889_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: 13:22985001-23015000_strand_+_gene_890 -- region: 890, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\890_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.530s.\n",
      "-- designing region: 13:24985001-25015000_strand_+_gene_891 -- region: 891, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\891_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.457s.\n",
      "-- designing region: 13:26985001-27015000_strand_+_gene_892 -- region: 892, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\892_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.496s.\n",
      "-- designing region: 13:28985001-29015000_strand_+_gene_893 -- region: 893, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\893_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.476s.\n",
      "-- designing region: 13:30985001-31015000_strand_+_gene_894 -- region: 894, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\894_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.460s.\n",
      "-- designing region: 13:32985001-33015000_strand_+_gene_895 -- region: 895, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\895_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.476s.\n",
      "-- designing region: 13:34985001-35015000_strand_+_gene_896 -- region: 896, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\896_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.459s.\n",
      "-- designing region: 13:36985001-37015000_strand_+_gene_897 -- region: 897, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\897_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.440s.\n",
      "-- designing region: 13:38985001-39015000_strand_+_gene_898 -- region: 898, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\898_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.462s.\n",
      "-- designing region: 13:40985001-41015000_strand_+_gene_899 -- region: 899, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\899_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.437s.\n",
      "-- designing region: 13:42985001-43015000_strand_+_gene_900 -- region: 900, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\900_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.588s.\n",
      "-- designing region: 13:44985001-45015000_strand_+_gene_901 -- region: 901, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\901_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 17.715s.\n",
      "-- designing region: 13:46985001-47015000_strand_+_gene_902 -- region: 902, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\902_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 2.364s.\n",
      "- Designed 59918 candidate probes in 22.833s.\n",
      "-- designing region: 13:48985001-49015000_strand_+_gene_903 -- region: 903, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\903_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.496s.\n",
      "- Designed 59918 candidate probes in 15.739s.\n",
      "-- designing region: 13:50985001-51015000_strand_+_gene_904 -- region: 904, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\904_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 13:52985001-53015000_strand_+_gene_905 -- region: 905, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\905_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.663s.\n",
      "-- designing region: 13:54985001-55015000_strand_+_gene_906 -- region: 906, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\906_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.659s.\n",
      "-- designing region: 13:56985001-57015000_strand_+_gene_907 -- region: 907, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\907_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.616s.\n",
      "-- designing region: 13:58985001-59015000_strand_+_gene_908 -- region: 908, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\908_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 13:60985001-61015000_strand_+_gene_909 -- region: 909, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\909_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.644s.\n",
      "-- designing region: 13:62985001-63015000_strand_+_gene_910 -- region: 910, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\910_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.616s.\n",
      "-- designing region: 13:64985001-65015000_strand_+_gene_911 -- region: 911, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\911_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.570s.\n",
      "-- designing region: 13:66985001-67015000_strand_+_gene_912 -- region: 912, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\912_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.648s.\n",
      "-- designing region: 13:68985001-69015000_strand_+_gene_913 -- region: 913, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\913_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.584s.\n",
      "-- designing region: 13:70985001-71015000_strand_+_gene_914 -- region: 914, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\914_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.571s.\n",
      "-- designing region: 13:72985001-73015000_strand_+_gene_915 -- region: 915, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\915_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 13:74985001-75015000_strand_+_gene_916 -- region: 916, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\916_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.210s.\n",
      "- Designed 59918 candidate probes in 15.522s.\n",
      "-- designing region: 13:76985001-77015000_strand_+_gene_917 -- region: 917, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\917_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 13:78985001-79015000_strand_+_gene_918 -- region: 918, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\918_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.589s.\n",
      "-- designing region: 13:80985001-81015000_strand_+_gene_919 -- region: 919, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\919_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.669s.\n",
      "-- designing region: 13:82985001-83015000_strand_+_gene_920 -- region: 920, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\920_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.545s.\n",
      "-- designing region: 13:84985001-85015000_strand_+_gene_921 -- region: 921, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\921_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.518s.\n",
      "-- designing region: 13:86985001-87015000_strand_+_gene_922 -- region: 922, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\922_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.643s.\n",
      "-- designing region: 13:88985001-89015000_strand_+_gene_923 -- region: 923, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\923_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.487s.\n",
      "-- designing region: 13:90985001-91015000_strand_+_gene_924 -- region: 924, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\924_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.567s.\n",
      "-- designing region: 13:92985001-93015000_strand_+_gene_925 -- region: 925, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\925_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 64.615s.\n",
      "-- designing region: 13:94985001-95015000_strand_+_gene_926 -- region: 926, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\926_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.587s.\n",
      "-- designing region: 13:96985001-97015000_strand_+_gene_927 -- region: 927, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\927_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.615s.\n",
      "-- designing region: 13:98985001-99015000_strand_+_gene_928 -- region: 928, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\928_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 13:100985001-101015000_strand_+_gene_929 -- region: 929, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\929_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.610s.\n",
      "-- designing region: 13:102985001-103015000_strand_+_gene_930 -- region: 930, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\930_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 13:104985001-105015000_strand_+_gene_931 -- region: 931, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\931_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.565s.\n",
      "-- designing region: 13:106985001-107015000_strand_+_gene_932 -- region: 932, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\932_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.592s.\n",
      "-- designing region: 13:108985001-109015000_strand_+_gene_933 -- region: 933, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\933_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.563s.\n",
      "-- designing region: 13:110985001-111015000_strand_+_gene_934 -- region: 934, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\934_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.583s.\n",
      "-- designing region: 13:112985001-113015000_strand_+_gene_935 -- region: 935, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\935_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 14:735001-765000_strand_+_gene_936 -- region: 936, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\936_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.210s.\n",
      "- Designed 0 candidate probes in 0.502s.\n",
      "-- designing region: 14:2235001-2265000_strand_+_gene_937 -- region: 937, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\937_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.550s.\n",
      "- Designed 0 candidate probes in 0.844s.\n",
      "-- designing region: 14:3735001-3765000_strand_+_gene_938 -- region: 938, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\938_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: 14:5235001-5265000_strand_+_gene_939 -- region: 939, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\939_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.555s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: 14:6735001-6765000_strand_+_gene_940 -- region: 940, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\940_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.555s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: 14:8235001-8265000_strand_+_gene_941 -- region: 941, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\941_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.552s.\n",
      "- Designed 0 candidate probes in 0.845s.\n",
      "-- designing region: 14:9735001-9765000_strand_+_gene_942 -- region: 942, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\942_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.564s.\n",
      "- Designed 0 candidate probes in 0.858s.\n",
      "-- designing region: 14:11235001-11265000_strand_+_gene_943 -- region: 943, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\943_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.549s.\n",
      "- Designed 0 candidate probes in 0.842s.\n",
      "-- designing region: 14:12735001-12765000_strand_+_gene_944 -- region: 944, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\944_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.574s.\n",
      "- Designed 0 candidate probes in 0.868s.\n",
      "-- designing region: 14:14235001-14265000_strand_+_gene_945 -- region: 945, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\945_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.557s.\n",
      "- Designed 0 candidate probes in 0.850s.\n",
      "-- designing region: 14:15735001-15765000_strand_+_gene_946 -- region: 946, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\946_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.557s.\n",
      "- Designed 0 candidate probes in 0.852s.\n",
      "-- designing region: 14:17235001-17265000_strand_+_gene_947 -- region: 947, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\947_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.574s.\n",
      "- Designed 59918 candidate probes in 15.722s.\n",
      "-- designing region: 14:18735001-18765000_strand_+_gene_948 -- region: 948, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\948_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.211s.\n",
      "- Designed 0 candidate probes in 0.505s.\n",
      "-- designing region: 14:20235001-20265000_strand_+_gene_949 -- region: 949, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\949_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.626s.\n",
      "- Designed 59918 candidate probes in 15.687s.\n",
      "-- designing region: 14:21735001-21765000_strand_+_gene_950 -- region: 950, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\950_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.540s.\n",
      "-- designing region: 14:23235001-23265000_strand_+_gene_951 -- region: 951, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\951_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.267s.\n",
      "- Designed 59918 candidate probes in 15.521s.\n",
      "-- designing region: 14:24735001-24765000_strand_+_gene_952 -- region: 952, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\952_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.529s.\n",
      "-- designing region: 14:26235001-26265000_strand_+_gene_953 -- region: 953, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\953_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.447s.\n",
      "-- designing region: 14:27735001-27765000_strand_+_gene_954 -- region: 954, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\954_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: 14:29235001-29265000_strand_+_gene_955 -- region: 955, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\955_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.266s.\n",
      "- Designed 59918 candidate probes in 15.406s.\n",
      "-- designing region: 14:30735001-30765000_strand_+_gene_956 -- region: 956, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\956_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.441s.\n",
      "-- designing region: 14:32235001-32265000_strand_+_gene_957 -- region: 957, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\957_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.529s.\n",
      "-- designing region: 14:33735001-33765000_strand_+_gene_958 -- region: 958, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\958_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.477s.\n",
      "-- designing region: 14:35235001-35265000_strand_+_gene_959 -- region: 959, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\959_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.266s.\n",
      "- Designed 59918 candidate probes in 15.423s.\n",
      "-- designing region: 14:36735001-36765000_strand_+_gene_960 -- region: 960, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\960_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.445s.\n",
      "-- designing region: 14:38235001-38265000_strand_+_gene_961 -- region: 961, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\961_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.461s.\n",
      "-- designing region: 14:39735001-39765000_strand_+_gene_962 -- region: 962, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\962_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.434s.\n",
      "-- designing region: 14:41235001-41265000_strand_+_gene_963 -- region: 963, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\963_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.421s.\n",
      "-- designing region: 14:42735001-42765000_strand_+_gene_964 -- region: 964, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\964_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.387s.\n",
      "-- designing region: 14:44235001-44265000_strand_+_gene_965 -- region: 965, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\965_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.413s.\n",
      "-- designing region: 14:45735001-45765000_strand_+_gene_966 -- region: 966, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\966_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.417s.\n",
      "-- designing region: 14:47235001-47265000_strand_+_gene_967 -- region: 967, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\967_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.412s.\n",
      "-- designing region: 14:48735001-48765000_strand_+_gene_968 -- region: 968, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\968_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.461s.\n",
      "-- designing region: 14:50235001-50265000_strand_+_gene_969 -- region: 969, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\969_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 14:51735001-51765000_strand_+_gene_970 -- region: 970, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\970_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.430s.\n",
      "-- designing region: 14:53235001-53265000_strand_+_gene_971 -- region: 971, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\971_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 19.509s.\n",
      "-- designing region: 14:54735001-54765000_strand_+_gene_972 -- region: 972, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\972_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.614s.\n",
      "- Designed 59918 candidate probes in 19.927s.\n",
      "-- designing region: 14:56235001-56265000_strand_+_gene_973 -- region: 973, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\973_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 14:57735001-57765000_strand_+_gene_974 -- region: 974, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\974_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.657s.\n",
      "-- designing region: 14:59235001-59265000_strand_+_gene_975 -- region: 975, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\975_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.666s.\n",
      "-- designing region: 14:60735001-60765000_strand_+_gene_976 -- region: 976, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\976_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.661s.\n",
      "-- designing region: 14:62235001-62265000_strand_+_gene_977 -- region: 977, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\977_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.670s.\n",
      "-- designing region: 14:63735001-63765000_strand_+_gene_978 -- region: 978, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\978_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.625s.\n",
      "-- designing region: 14:65235001-65265000_strand_+_gene_979 -- region: 979, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\979_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.689s.\n",
      "-- designing region: 14:66735001-66765000_strand_+_gene_980 -- region: 980, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\980_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.613s.\n",
      "-- designing region: 14:68235001-68265000_strand_+_gene_981 -- region: 981, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\981_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.634s.\n",
      "-- designing region: 14:69735001-69765000_strand_+_gene_982 -- region: 982, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\982_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 14:71235001-71265000_strand_+_gene_983 -- region: 983, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\983_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.657s.\n",
      "-- designing region: 14:72735001-72765000_strand_+_gene_984 -- region: 984, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\984_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.674s.\n",
      "-- designing region: 14:74235001-74265000_strand_+_gene_985 -- region: 985, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\985_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.671s.\n",
      "-- designing region: 14:75735001-75765000_strand_+_gene_986 -- region: 986, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\986_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.642s.\n",
      "-- designing region: 14:77235001-77265000_strand_+_gene_987 -- region: 987, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\987_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.718s.\n",
      "-- designing region: 14:78735001-78765000_strand_+_gene_988 -- region: 988, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\988_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.634s.\n",
      "-- designing region: 14:80235001-80265000_strand_+_gene_989 -- region: 989, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\989_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.594s.\n",
      "-- designing region: 14:81735001-81765000_strand_+_gene_990 -- region: 990, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\990_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.603s.\n",
      "-- designing region: 14:83235001-83265000_strand_+_gene_991 -- region: 991, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\991_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.559s.\n",
      "-- designing region: 14:84735001-84765000_strand_+_gene_992 -- region: 992, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\992_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 14:86235001-86265000_strand_+_gene_993 -- region: 993, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\993_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.584s.\n",
      "-- designing region: 14:87735001-87765000_strand_+_gene_994 -- region: 994, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\994_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.612s.\n",
      "-- designing region: 14:89235001-89265000_strand_+_gene_995 -- region: 995, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\995_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.642s.\n",
      "-- designing region: 14:90735001-90765000_strand_+_gene_996 -- region: 996, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\996_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 14:92235001-92265000_strand_+_gene_997 -- region: 997, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\997_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.632s.\n",
      "-- designing region: 14:93735001-93765000_strand_+_gene_998 -- region: 998, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\998_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.681s.\n",
      "-- designing region: 14:95235001-95265000_strand_+_gene_999 -- region: 999, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\999_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: 14:96735001-96765000_strand_+_gene_1000 -- region: 1000, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1000_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.687s.\n",
      "-- designing region: 14:98235001-98265000_strand_+_gene_1001 -- region: 1001, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1001_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.569s.\n",
      "-- designing region: 14:99735001-99765000_strand_+_gene_1002 -- region: 1002, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1002_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 14:101235001-101265000_strand_+_gene_1003 -- region: 1003, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1003_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 14:102735001-102765000_strand_+_gene_1004 -- region: 1004, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1004_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.504s.\n",
      "-- designing region: 14:104235001-104265000_strand_+_gene_1005 -- region: 1005, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1005_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.488s.\n",
      "-- designing region: 14:105735001-105765000_strand_+_gene_1006 -- region: 1006, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1006_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.477s.\n",
      "-- designing region: 15:735001-765000_strand_+_gene_1007 -- region: 1007, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1007_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.200s.\n",
      "- Designed 0 candidate probes in 0.496s.\n",
      "-- designing region: 15:2235001-2265000_strand_+_gene_1008 -- region: 1008, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1008_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.562s.\n",
      "- Designed 0 candidate probes in 0.854s.\n",
      "-- designing region: 15:3735001-3765000_strand_+_gene_1009 -- region: 1009, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1009_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.558s.\n",
      "- Designed 0 candidate probes in 0.851s.\n",
      "-- designing region: 15:5235001-5265000_strand_+_gene_1010 -- region: 1010, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1010_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.557s.\n",
      "- Designed 0 candidate probes in 0.851s.\n",
      "-- designing region: 15:6735001-6765000_strand_+_gene_1011 -- region: 1011, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1011_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: 15:8235001-8265000_strand_+_gene_1012 -- region: 1012, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1012_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.560s.\n",
      "- Designed 0 candidate probes in 0.854s.\n",
      "-- designing region: 15:9735001-9765000_strand_+_gene_1013 -- region: 1013, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1013_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.846s.\n",
      "-- designing region: 15:11235001-11265000_strand_+_gene_1014 -- region: 1014, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1014_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.559s.\n",
      "- Designed 0 candidate probes in 0.853s.\n",
      "-- designing region: 15:12735001-12765000_strand_+_gene_1015 -- region: 1015, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1015_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.513s.\n",
      "- Designed 0 candidate probes in 0.808s.\n",
      "-- designing region: 15:14235001-14265000_strand_+_gene_1016 -- region: 1016, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1016_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.557s.\n",
      "- Designed 0 candidate probes in 0.851s.\n",
      "-- designing region: 15:15735001-15765000_strand_+_gene_1017 -- region: 1017, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1017_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.556s.\n",
      "- Designed 0 candidate probes in 0.850s.\n",
      "-- designing region: 15:17235001-17265000_strand_+_gene_1018 -- region: 1018, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1018_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.593s.\n",
      "- Designed 59918 candidate probes in 15.650s.\n",
      "-- designing region: 15:18735001-18765000_strand_+_gene_1019 -- region: 1019, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1019_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.221s.\n",
      "- Designed 59918 candidate probes in 15.232s.\n",
      "-- designing region: 15:20235001-20265000_strand_+_gene_1020 -- region: 1020, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1020_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.265s.\n",
      "- Designed 59918 candidate probes in 15.316s.\n",
      "-- designing region: 15:21735001-21765000_strand_+_gene_1021 -- region: 1021, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1021_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.453s.\n",
      "-- designing region: 15:23235001-23265000_strand_+_gene_1022 -- region: 1022, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1022_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.202s.\n",
      "- Designed 0 candidate probes in 0.495s.\n",
      "-- designing region: 15:24735001-24765000_strand_+_gene_1023 -- region: 1023, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1023_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.627s.\n",
      "- Designed 59918 candidate probes in 15.783s.\n",
      "-- designing region: 15:26235001-26265000_strand_+_gene_1024 -- region: 1024, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1024_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.442s.\n",
      "-- designing region: 15:27735001-27765000_strand_+_gene_1025 -- region: 1025, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1025_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.476s.\n",
      "-- designing region: 15:29235001-29265000_strand_+_gene_1026 -- region: 1026, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1026_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.424s.\n",
      "-- designing region: 15:30735001-30765000_strand_+_gene_1027 -- region: 1027, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1027_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.442s.\n",
      "-- designing region: 15:32235001-32265000_strand_+_gene_1028 -- region: 1028, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1028_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.454s.\n",
      "-- designing region: 15:33735001-33765000_strand_+_gene_1029 -- region: 1029, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1029_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "- Designed 59918 candidate probes in 15.437s.\n",
      "-- designing region: 15:35235001-35265000_strand_+_gene_1030 -- region: 1030, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1030_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.446s.\n",
      "-- designing region: 15:36735001-36765000_strand_+_gene_1031 -- region: 1031, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1031_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.414s.\n",
      "-- designing region: 15:38235001-38265000_strand_+_gene_1032 -- region: 1032, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1032_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.449s.\n",
      "-- designing region: 15:39735001-39765000_strand_+_gene_1033 -- region: 1033, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1033_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.415s.\n",
      "-- designing region: 15:41235001-41265000_strand_+_gene_1034 -- region: 1034, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1034_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.424s.\n",
      "-- designing region: 15:42735001-42765000_strand_+_gene_1035 -- region: 1035, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1035_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.428s.\n",
      "-- designing region: 15:44235001-44265000_strand_+_gene_1036 -- region: 1036, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1036_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.444s.\n",
      "-- designing region: 15:45735001-45765000_strand_+_gene_1037 -- region: 1037, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1037_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.448s.\n",
      "-- designing region: 15:47235001-47265000_strand_+_gene_1038 -- region: 1038, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1038_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.414s.\n",
      "-- designing region: 15:48735001-48765000_strand_+_gene_1039 -- region: 1039, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1039_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.430s.\n",
      "-- designing region: 15:50235001-50265000_strand_+_gene_1040 -- region: 1040, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1040_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.458s.\n",
      "-- designing region: 15:51735001-51765000_strand_+_gene_1041 -- region: 1041, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1041_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.478s.\n",
      "-- designing region: 15:53235001-53265000_strand_+_gene_1042 -- region: 1042, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1042_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: 15:54735001-54765000_strand_+_gene_1043 -- region: 1043, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1043_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.502s.\n",
      "-- designing region: 15:56235001-56265000_strand_+_gene_1044 -- region: 1044, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1044_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 19.604s.\n",
      "-- designing region: 15:57735001-57765000_strand_+_gene_1045 -- region: 1045, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1045_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.598s.\n",
      "- Designed 59918 candidate probes in 21.340s.\n",
      "-- designing region: 15:59235001-59265000_strand_+_gene_1046 -- region: 1046, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1046_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.532s.\n",
      "-- designing region: 15:60735001-60765000_strand_+_gene_1047 -- region: 1047, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1047_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.309s.\n",
      "- Designed 59918 candidate probes in 15.689s.\n",
      "-- designing region: 15:62235001-62265000_strand_+_gene_1048 -- region: 1048, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1048_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.267s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 15:63735001-63765000_strand_+_gene_1049 -- region: 1049, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1049_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.624s.\n",
      "-- designing region: 15:65235001-65265000_strand_+_gene_1050 -- region: 1050, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1050_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.636s.\n",
      "-- designing region: 15:66735001-66765000_strand_+_gene_1051 -- region: 1051, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1051_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.672s.\n",
      "-- designing region: 15:68235001-68265000_strand_+_gene_1052 -- region: 1052, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1052_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.577s.\n",
      "-- designing region: 15:69735001-69765000_strand_+_gene_1053 -- region: 1053, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1053_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.693s.\n",
      "-- designing region: 15:71235001-71265000_strand_+_gene_1054 -- region: 1054, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1054_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 15:72735001-72765000_strand_+_gene_1055 -- region: 1055, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1055_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.613s.\n",
      "-- designing region: 15:74235001-74265000_strand_+_gene_1056 -- region: 1056, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1056_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.668s.\n",
      "-- designing region: 15:75735001-75765000_strand_+_gene_1057 -- region: 1057, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1057_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.688s.\n",
      "-- designing region: 15:77235001-77265000_strand_+_gene_1058 -- region: 1058, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1058_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 15:78735001-78765000_strand_+_gene_1059 -- region: 1059, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1059_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 15:80235001-80265000_strand_+_gene_1060 -- region: 1060, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1060_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.620s.\n",
      "-- designing region: 15:81735001-81765000_strand_+_gene_1061 -- region: 1061, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1061_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.676s.\n",
      "-- designing region: 15:83235001-83265000_strand_+_gene_1062 -- region: 1062, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1062_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.637s.\n",
      "-- designing region: 15:84735001-84765000_strand_+_gene_1063 -- region: 1063, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1063_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.606s.\n",
      "-- designing region: 15:86235001-86265000_strand_+_gene_1064 -- region: 1064, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1064_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.636s.\n",
      "-- designing region: 15:87735001-87765000_strand_+_gene_1065 -- region: 1065, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1065_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 15:89235001-89265000_strand_+_gene_1066 -- region: 1066, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1066_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.301s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 15:90735001-90765000_strand_+_gene_1067 -- region: 1067, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1067_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.496s.\n",
      "-- designing region: 15:92235001-92265000_strand_+_gene_1068 -- region: 1068, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1068_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.561s.\n",
      "-- designing region: 15:93735001-93765000_strand_+_gene_1069 -- region: 1069, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1069_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.565s.\n",
      "-- designing region: 15:95235001-95265000_strand_+_gene_1070 -- region: 1070, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1070_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.585s.\n",
      "-- designing region: 15:96735001-96765000_strand_+_gene_1071 -- region: 1071, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1071_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.555s.\n",
      "-- designing region: 15:98235001-98265000_strand_+_gene_1072 -- region: 1072, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1072_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.574s.\n",
      "-- designing region: 15:99735001-99765000_strand_+_gene_1073 -- region: 1073, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1073_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 16:735001-765000_strand_+_gene_1074 -- region: 1074, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1074_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 16:2235001-2265000_strand_+_gene_1075 -- region: 1075, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1075_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 16:3735001-3765000_strand_+_gene_1076 -- region: 1076, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1076_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.536s.\n",
      "-- designing region: 16:5235001-5265000_strand_+_gene_1077 -- region: 1077, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1077_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.543s.\n",
      "-- designing region: 16:6735001-6765000_strand_+_gene_1078 -- region: 1078, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1078_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.564s.\n",
      "-- designing region: 16:8235001-8265000_strand_+_gene_1079 -- region: 1079, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1079_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.405s.\n",
      "-- designing region: 16:9735001-9765000_strand_+_gene_1080 -- region: 1080, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1080_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.526s.\n",
      "-- designing region: 16:11235001-11265000_strand_+_gene_1081 -- region: 1081, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1081_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.346s.\n",
      "-- designing region: 16:12735001-12765000_strand_+_gene_1082 -- region: 1082, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1082_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.394s.\n",
      "-- designing region: 16:14235001-14265000_strand_+_gene_1083 -- region: 1083, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1083_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.407s.\n",
      "-- designing region: 16:15735001-15765000_strand_+_gene_1084 -- region: 1084, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1084_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.433s.\n",
      "-- designing region: 16:17235001-17265000_strand_+_gene_1085 -- region: 1085, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1085_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.405s.\n",
      "-- designing region: 16:18735001-18765000_strand_+_gene_1086 -- region: 1086, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1086_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.418s.\n",
      "-- designing region: 16:20235001-20265000_strand_+_gene_1087 -- region: 1087, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1087_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.408s.\n",
      "-- designing region: 16:21735001-21765000_strand_+_gene_1088 -- region: 1088, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1088_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.423s.\n",
      "-- designing region: 16:23235001-23265000_strand_+_gene_1089 -- region: 1089, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1089_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.370s.\n",
      "-- designing region: 16:24735001-24765000_strand_+_gene_1090 -- region: 1090, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1090_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.429s.\n",
      "-- designing region: 16:26235001-26265000_strand_+_gene_1091 -- region: 1091, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1091_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.424s.\n",
      "-- designing region: 16:27735001-27765000_strand_+_gene_1092 -- region: 1092, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1092_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.462s.\n",
      "-- designing region: 16:29235001-29265000_strand_+_gene_1093 -- region: 1093, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1093_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: 16:30735001-30765000_strand_+_gene_1094 -- region: 1094, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1094_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.470s.\n",
      "-- designing region: 16:32235001-32265000_strand_+_gene_1095 -- region: 1095, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1095_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.455s.\n",
      "-- designing region: 16:33735001-33765000_strand_+_gene_1096 -- region: 1096, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1096_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.374s.\n",
      "-- designing region: 16:35235001-35265000_strand_+_gene_1097 -- region: 1097, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1097_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.419s.\n",
      "-- designing region: 16:36735001-36765000_strand_+_gene_1098 -- region: 1098, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1098_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.218s.\n",
      "- Designed 59918 candidate probes in 15.283s.\n",
      "-- designing region: 16:38235001-38265000_strand_+_gene_1099 -- region: 1099, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1099_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.212s.\n",
      "- Designed 59918 candidate probes in 15.224s.\n",
      "-- designing region: 16:39735001-39765000_strand_+_gene_1100 -- region: 1100, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1100_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.202s.\n",
      "- Designed 0 candidate probes in 0.496s.\n",
      "-- designing region: 16:41235001-41265000_strand_+_gene_1101 -- region: 1101, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1101_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.556s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: 16:42735001-42765000_strand_+_gene_1102 -- region: 1102, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1102_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.844s.\n",
      "-- designing region: 16:44235001-44265000_strand_+_gene_1103 -- region: 1103, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1103_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.844s.\n",
      "-- designing region: 16:45735001-45765000_strand_+_gene_1104 -- region: 1104, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1104_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.564s.\n",
      "- Designed 0 candidate probes in 0.859s.\n",
      "-- designing region: 16:47235001-47265000_strand_+_gene_1105 -- region: 1105, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1105_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.620s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 16:48735001-48765000_strand_+_gene_1106 -- region: 1106, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1106_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.455s.\n",
      "-- designing region: 16:50235001-50265000_strand_+_gene_1107 -- region: 1107, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1107_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.477s.\n",
      "-- designing region: 16:51735001-51765000_strand_+_gene_1108 -- region: 1108, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1108_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.465s.\n",
      "-- designing region: 16:53235001-53265000_strand_+_gene_1109 -- region: 1109, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1109_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.539s.\n",
      "-- designing region: 16:54735001-54765000_strand_+_gene_1110 -- region: 1110, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1110_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.664s.\n",
      "-- designing region: 16:56235001-56265000_strand_+_gene_1111 -- region: 1111, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1111_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 2.378s.\n",
      "- Designed 59918 candidate probes in 28.245s.\n",
      "-- designing region: 16:57735001-57765000_strand_+_gene_1112 -- region: 1112, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1112_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.575s.\n",
      "-- designing region: 16:59235001-59265000_strand_+_gene_1113 -- region: 1113, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1113_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.654s.\n",
      "-- designing region: 16:60735001-60765000_strand_+_gene_1114 -- region: 1114, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1114_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.628s.\n",
      "-- designing region: 16:62235001-62265000_strand_+_gene_1115 -- region: 1115, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1115_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 16:63735001-63765000_strand_+_gene_1116 -- region: 1116, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1116_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.607s.\n",
      "-- designing region: 16:65235001-65265000_strand_+_gene_1117 -- region: 1117, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1117_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 16:66735001-66765000_strand_+_gene_1118 -- region: 1118, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1118_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.614s.\n",
      "-- designing region: 16:68235001-68265000_strand_+_gene_1119 -- region: 1119, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1119_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.659s.\n",
      "-- designing region: 16:69735001-69765000_strand_+_gene_1120 -- region: 1120, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1120_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 16:71235001-71265000_strand_+_gene_1121 -- region: 1121, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1121_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.611s.\n",
      "-- designing region: 16:72735001-72765000_strand_+_gene_1122 -- region: 1122, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1122_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 16:74235001-74265000_strand_+_gene_1123 -- region: 1123, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1123_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 16:75735001-75765000_strand_+_gene_1124 -- region: 1124, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1124_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.718s.\n",
      "-- designing region: 16:77235001-77265000_strand_+_gene_1125 -- region: 1125, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1125_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.595s.\n",
      "-- designing region: 16:78735001-78765000_strand_+_gene_1126 -- region: 1126, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1126_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 16:80235001-80265000_strand_+_gene_1127 -- region: 1127, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1127_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.587s.\n",
      "-- designing region: 16:81735001-81765000_strand_+_gene_1128 -- region: 1128, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1128_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 16:83235001-83265000_strand_+_gene_1129 -- region: 1129, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1129_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.567s.\n",
      "-- designing region: 16:84735001-84765000_strand_+_gene_1130 -- region: 1130, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1130_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.624s.\n",
      "-- designing region: 16:86235001-86265000_strand_+_gene_1131 -- region: 1131, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1131_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.620s.\n",
      "-- designing region: 16:87735001-87765000_strand_+_gene_1132 -- region: 1132, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1132_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 16:89235001-89265000_strand_+_gene_1133 -- region: 1133, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1133_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.602s.\n",
      "-- designing region: 17:735001-765000_strand_+_gene_1134 -- region: 1134, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1134_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 17:2235001-2265000_strand_+_gene_1135 -- region: 1135, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1135_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.588s.\n",
      "-- designing region: 17:3735001-3765000_strand_+_gene_1136 -- region: 1136, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1136_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: 17:5235001-5265000_strand_+_gene_1137 -- region: 1137, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1137_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.564s.\n",
      "-- designing region: 17:6735001-6765000_strand_+_gene_1138 -- region: 1138, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1138_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 17:8235001-8265000_strand_+_gene_1139 -- region: 1139, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1139_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 17:9735001-9765000_strand_+_gene_1140 -- region: 1140, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1140_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.587s.\n",
      "-- designing region: 17:11235001-11265000_strand_+_gene_1141 -- region: 1141, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1141_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.608s.\n",
      "-- designing region: 17:12735001-12765000_strand_+_gene_1142 -- region: 1142, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1142_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 17:14235001-14265000_strand_+_gene_1143 -- region: 1143, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1143_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.555s.\n",
      "-- designing region: 17:15735001-15765000_strand_+_gene_1144 -- region: 1144, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1144_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.564s.\n",
      "-- designing region: 17:17235001-17265000_strand_+_gene_1145 -- region: 1145, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1145_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.554s.\n",
      "-- designing region: 17:18735001-18765000_strand_+_gene_1146 -- region: 1146, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1146_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.401s.\n",
      "-- designing region: 17:20235001-20265000_strand_+_gene_1147 -- region: 1147, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1147_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.489s.\n",
      "-- designing region: 17:21735001-21765000_strand_+_gene_1148 -- region: 1148, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1148_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.510s.\n",
      "-- designing region: 17:23235001-23265000_strand_+_gene_1149 -- region: 1149, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1149_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.215s.\n",
      "- Designed 59918 candidate probes in 15.257s.\n",
      "-- designing region: 17:24735001-24765000_strand_+_gene_1150 -- region: 1150, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1150_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.220s.\n",
      "- Designed 59918 candidate probes in 15.215s.\n",
      "-- designing region: 17:26235001-26265000_strand_+_gene_1151 -- region: 1151, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1151_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.223s.\n",
      "- Designed 59918 candidate probes in 15.257s.\n",
      "-- designing region: 17:27735001-27765000_strand_+_gene_1152 -- region: 1152, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1152_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.364s.\n",
      "-- designing region: 17:29235001-29265000_strand_+_gene_1153 -- region: 1153, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1153_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.458s.\n",
      "-- designing region: 17:30735001-30765000_strand_+_gene_1154 -- region: 1154, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1154_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.391s.\n",
      "-- designing region: 17:32235001-32265000_strand_+_gene_1155 -- region: 1155, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1155_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.265s.\n",
      "- Designed 59918 candidate probes in 15.392s.\n",
      "-- designing region: 17:33735001-33765000_strand_+_gene_1156 -- region: 1156, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1156_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.433s.\n",
      "-- designing region: 17:35235001-35265000_strand_+_gene_1157 -- region: 1157, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1157_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.267s.\n",
      "- Designed 59918 candidate probes in 15.418s.\n",
      "-- designing region: 17:36735001-36765000_strand_+_gene_1158 -- region: 1158, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1158_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.480s.\n",
      "-- designing region: 17:38235001-38265000_strand_+_gene_1159 -- region: 1159, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1159_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.260s.\n",
      "- Designed 59918 candidate probes in 15.354s.\n",
      "-- designing region: 17:39735001-39765000_strand_+_gene_1160 -- region: 1160, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1160_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.455s.\n",
      "-- designing region: 17:41235001-41265000_strand_+_gene_1161 -- region: 1161, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1161_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "- Designed 59918 candidate probes in 15.425s.\n",
      "-- designing region: 17:42735001-42765000_strand_+_gene_1162 -- region: 1162, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1162_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.470s.\n",
      "-- designing region: 17:44235001-44265000_strand_+_gene_1163 -- region: 1163, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1163_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.262s.\n",
      "- Designed 59918 candidate probes in 15.439s.\n",
      "-- designing region: 17:45735001-45765000_strand_+_gene_1164 -- region: 1164, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1164_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.475s.\n",
      "-- designing region: 17:47235001-47265000_strand_+_gene_1165 -- region: 1165, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1165_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.471s.\n",
      "-- designing region: 17:48735001-48765000_strand_+_gene_1166 -- region: 1166, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1166_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.483s.\n",
      "-- designing region: 17:50235001-50265000_strand_+_gene_1167 -- region: 1167, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1167_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.457s.\n",
      "-- designing region: 17:51735001-51765000_strand_+_gene_1168 -- region: 1168, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1168_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.470s.\n",
      "-- designing region: 17:53235001-53265000_strand_+_gene_1169 -- region: 1169, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1169_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.474s.\n",
      "-- designing region: 17:54735001-54765000_strand_+_gene_1170 -- region: 1170, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1170_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.568s.\n",
      "-- designing region: 17:56235001-56265000_strand_+_gene_1171 -- region: 1171, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1171_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.467s.\n",
      "-- designing region: 17:57735001-57765000_strand_+_gene_1172 -- region: 1172, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1172_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 19.991s.\n",
      "-- designing region: 17:59235001-59265000_strand_+_gene_1173 -- region: 1173, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1173_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.583s.\n",
      "- Designed 59918 candidate probes in 21.880s.\n",
      "-- designing region: 17:60735001-60765000_strand_+_gene_1174 -- region: 1174, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1174_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.522s.\n",
      "-- designing region: 17:62235001-62265000_strand_+_gene_1175 -- region: 1175, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1175_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.643s.\n",
      "-- designing region: 17:63735001-63765000_strand_+_gene_1176 -- region: 1176, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1176_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.576s.\n",
      "-- designing region: 17:65235001-65265000_strand_+_gene_1177 -- region: 1177, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1177_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.650s.\n",
      "-- designing region: 17:66735001-66765000_strand_+_gene_1178 -- region: 1178, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1178_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.697s.\n",
      "-- designing region: 17:68235001-68265000_strand_+_gene_1179 -- region: 1179, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1179_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.715s.\n",
      "-- designing region: 17:69735001-69765000_strand_+_gene_1180 -- region: 1180, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1180_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.671s.\n",
      "-- designing region: 17:71235001-71265000_strand_+_gene_1181 -- region: 1181, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1181_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.621s.\n",
      "-- designing region: 17:72735001-72765000_strand_+_gene_1182 -- region: 1182, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1182_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.702s.\n",
      "-- designing region: 17:74235001-74265000_strand_+_gene_1183 -- region: 1183, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1183_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 17:75735001-75765000_strand_+_gene_1184 -- region: 1184, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1184_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.676s.\n",
      "-- designing region: 17:77235001-77265000_strand_+_gene_1185 -- region: 1185, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1185_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.634s.\n",
      "-- designing region: 17:78735001-78765000_strand_+_gene_1186 -- region: 1186, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1186_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.641s.\n",
      "-- designing region: 17:80235001-80265000_strand_+_gene_1187 -- region: 1187, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1187_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.650s.\n",
      "-- designing region: 17:81735001-81765000_strand_+_gene_1188 -- region: 1188, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1188_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.224s.\n",
      "- Designed 15002 candidate probes in 4.639s.\n",
      "-- designing region: 18:735001-765000_strand_+_gene_1189 -- region: 1189, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1189_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.568s.\n",
      "-- designing region: 18:2235001-2265000_strand_+_gene_1190 -- region: 1190, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1190_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.632s.\n",
      "-- designing region: 18:3735001-3765000_strand_+_gene_1191 -- region: 1191, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1191_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.606s.\n",
      "-- designing region: 18:5235001-5265000_strand_+_gene_1192 -- region: 1192, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1192_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.595s.\n",
      "-- designing region: 18:6735001-6765000_strand_+_gene_1193 -- region: 1193, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1193_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.571s.\n",
      "-- designing region: 18:8235001-8265000_strand_+_gene_1194 -- region: 1194, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1194_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 18:9735001-9765000_strand_+_gene_1195 -- region: 1195, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1195_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 77.578s.\n",
      "-- designing region: 18:11235001-11265000_strand_+_gene_1196 -- region: 1196, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1196_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.727s.\n",
      "-- designing region: 18:12735001-12765000_strand_+_gene_1197 -- region: 1197, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1197_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 18:14235001-14265000_strand_+_gene_1198 -- region: 1198, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1198_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 18:15735001-15765000_strand_+_gene_1199 -- region: 1199, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1199_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.221s.\n",
      "- Designed 59918 candidate probes in 15.394s.\n",
      "-- designing region: 18:17235001-17265000_strand_+_gene_1200 -- region: 1200, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1200_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.229s.\n",
      "- Designed 59918 candidate probes in 15.532s.\n",
      "-- designing region: 18:18735001-18765000_strand_+_gene_1201 -- region: 1201, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1201_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.224s.\n",
      "- Designed 59918 candidate probes in 15.335s.\n",
      "-- designing region: 18:20235001-20265000_strand_+_gene_1202 -- region: 1202, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1202_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.220s.\n",
      "- Designed 59918 candidate probes in 15.303s.\n",
      "-- designing region: 18:21735001-21765000_strand_+_gene_1203 -- region: 1203, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1203_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 18:23235001-23265000_strand_+_gene_1204 -- region: 1204, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1204_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.664s.\n",
      "-- designing region: 18:24735001-24765000_strand_+_gene_1205 -- region: 1205, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1205_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.636s.\n",
      "-- designing region: 18:26235001-26265000_strand_+_gene_1206 -- region: 1206, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1206_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.660s.\n",
      "-- designing region: 18:27735001-27765000_strand_+_gene_1207 -- region: 1207, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1207_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 18:29235001-29265000_strand_+_gene_1208 -- region: 1208, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1208_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.592s.\n",
      "-- designing region: 18:30735001-30765000_strand_+_gene_1209 -- region: 1209, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1209_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.558s.\n",
      "-- designing region: 18:32235001-32265000_strand_+_gene_1210 -- region: 1210, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1210_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.573s.\n",
      "-- designing region: 18:33735001-33765000_strand_+_gene_1211 -- region: 1211, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1211_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.458s.\n",
      "-- designing region: 18:35235001-35265000_strand_+_gene_1212 -- region: 1212, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1212_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.482s.\n",
      "-- designing region: 18:36735001-36765000_strand_+_gene_1213 -- region: 1213, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1213_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.490s.\n",
      "-- designing region: 18:38235001-38265000_strand_+_gene_1214 -- region: 1214, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1214_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.478s.\n",
      "-- designing region: 18:39735001-39765000_strand_+_gene_1215 -- region: 1215, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1215_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.317s.\n",
      "- Designed 59918 candidate probes in 15.507s.\n",
      "-- designing region: 18:41235001-41265000_strand_+_gene_1216 -- region: 1216, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1216_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.402s.\n",
      "-- designing region: 18:42735001-42765000_strand_+_gene_1217 -- region: 1217, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1217_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.437s.\n",
      "-- designing region: 18:44235001-44265000_strand_+_gene_1218 -- region: 1218, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1218_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.405s.\n",
      "-- designing region: 18:45735001-45765000_strand_+_gene_1219 -- region: 1219, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1219_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.459s.\n",
      "-- designing region: 18:47235001-47265000_strand_+_gene_1220 -- region: 1220, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1220_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.451s.\n",
      "-- designing region: 18:48735001-48765000_strand_+_gene_1221 -- region: 1221, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1221_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.489s.\n",
      "-- designing region: 18:50235001-50265000_strand_+_gene_1222 -- region: 1222, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1222_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.408s.\n",
      "-- designing region: 18:51735001-51765000_strand_+_gene_1223 -- region: 1223, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1223_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.466s.\n",
      "-- designing region: 18:53235001-53265000_strand_+_gene_1224 -- region: 1224, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1224_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.429s.\n",
      "-- designing region: 18:54735001-54765000_strand_+_gene_1225 -- region: 1225, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1225_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.400s.\n",
      "-- designing region: 18:56235001-56265000_strand_+_gene_1226 -- region: 1226, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1226_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.454s.\n",
      "-- designing region: 18:57735001-57765000_strand_+_gene_1227 -- region: 1227, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1227_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.431s.\n",
      "-- designing region: 18:59235001-59265000_strand_+_gene_1228 -- region: 1228, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1228_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.487s.\n",
      "-- designing region: 18:60735001-60765000_strand_+_gene_1229 -- region: 1229, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1229_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: 18:62235001-62265000_strand_+_gene_1230 -- region: 1230, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1230_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.304s.\n",
      "- Designed 59918 candidate probes in 15.430s.\n",
      "-- designing region: 18:63735001-63765000_strand_+_gene_1231 -- region: 1231, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1231_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 2.262s.\n",
      "- Designed 59918 candidate probes in 28.285s.\n",
      "-- designing region: 18:65235001-65265000_strand_+_gene_1232 -- region: 1232, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1232_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.535s.\n",
      "-- designing region: 18:66735001-66765000_strand_+_gene_1233 -- region: 1233, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1233_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.622s.\n",
      "-- designing region: 18:68235001-68265000_strand_+_gene_1234 -- region: 1234, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1234_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.608s.\n",
      "-- designing region: 18:69735001-69765000_strand_+_gene_1235 -- region: 1235, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1235_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.594s.\n",
      "-- designing region: 18:71235001-71265000_strand_+_gene_1236 -- region: 1236, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1236_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.591s.\n",
      "-- designing region: 18:72735001-72765000_strand_+_gene_1237 -- region: 1237, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1237_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.588s.\n",
      "-- designing region: 18:74235001-74265000_strand_+_gene_1238 -- region: 1238, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1238_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.697s.\n",
      "-- designing region: 18:75735001-75765000_strand_+_gene_1239 -- region: 1239, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1239_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.603s.\n",
      "-- designing region: 18:77235001-77265000_strand_+_gene_1240 -- region: 1240, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1240_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.652s.\n",
      "-- designing region: 18:78735001-78765000_strand_+_gene_1241 -- region: 1241, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1241_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.306s.\n",
      "- Designed 59918 candidate probes in 15.667s.\n",
      "-- designing region: 19:485001-515000_strand_+_gene_1242 -- region: 1242, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1242_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.639s.\n",
      "-- designing region: 19:1485001-1515000_strand_+_gene_1243 -- region: 1243, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1243_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.613s.\n",
      "-- designing region: 19:2485001-2515000_strand_+_gene_1244 -- region: 1244, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1244_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 19:3485001-3515000_strand_+_gene_1245 -- region: 1245, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1245_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.668s.\n",
      "-- designing region: 19:4485001-4515000_strand_+_gene_1246 -- region: 1246, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1246_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.609s.\n",
      "-- designing region: 19:5485001-5515000_strand_+_gene_1247 -- region: 1247, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1247_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.572s.\n",
      "-- designing region: 19:6485001-6515000_strand_+_gene_1248 -- region: 1248, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1248_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.571s.\n",
      "-- designing region: 19:7485001-7515000_strand_+_gene_1249 -- region: 1249, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1249_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.587s.\n",
      "-- designing region: 19:8485001-8515000_strand_+_gene_1250 -- region: 1250, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1250_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.640s.\n",
      "-- designing region: 19:9485001-9515000_strand_+_gene_1251 -- region: 1251, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1251_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.593s.\n",
      "-- designing region: 19:10485001-10515000_strand_+_gene_1252 -- region: 1252, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1252_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 19:11485001-11515000_strand_+_gene_1253 -- region: 1253, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1253_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 19:12485001-12515000_strand_+_gene_1254 -- region: 1254, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1254_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.611s.\n",
      "-- designing region: 19:13485001-13515000_strand_+_gene_1255 -- region: 1255, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1255_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 19:14485001-14515000_strand_+_gene_1256 -- region: 1256, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1256_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.533s.\n",
      "-- designing region: 19:15485001-15515000_strand_+_gene_1257 -- region: 1257, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1257_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.560s.\n",
      "-- designing region: 19:16485001-16515000_strand_+_gene_1258 -- region: 1258, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1258_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.553s.\n",
      "-- designing region: 19:17485001-17515000_strand_+_gene_1259 -- region: 1259, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1259_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.609s.\n",
      "-- designing region: 19:18485001-18515000_strand_+_gene_1260 -- region: 1260, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1260_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.656s.\n",
      "-- designing region: 19:19485001-19515000_strand_+_gene_1261 -- region: 1261, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1261_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.444s.\n",
      "-- designing region: 19:20485001-20515000_strand_+_gene_1262 -- region: 1262, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1262_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.531s.\n",
      "-- designing region: 19:21485001-21515000_strand_+_gene_1263 -- region: 1263, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1263_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.561s.\n",
      "-- designing region: 19:22485001-22515000_strand_+_gene_1264 -- region: 1264, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1264_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.527s.\n",
      "-- designing region: 19:23485001-23515000_strand_+_gene_1265 -- region: 1265, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1265_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.504s.\n",
      "-- designing region: 19:24485001-24515000_strand_+_gene_1266 -- region: 1266, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1266_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.224s.\n",
      "- Designed 31958 candidate probes in 8.605s.\n",
      "-- designing region: 19:25485001-25515000_strand_+_gene_1267 -- region: 1267, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1267_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.221s.\n",
      "- Designed 59918 candidate probes in 15.242s.\n",
      "-- designing region: 19:26485001-26515000_strand_+_gene_1268 -- region: 1268, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1268_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.217s.\n",
      "- Designed 59918 candidate probes in 15.274s.\n",
      "-- designing region: 19:27485001-27515000_strand_+_gene_1269 -- region: 1269, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1269_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.260s.\n",
      "- Designed 59918 candidate probes in 15.277s.\n",
      "-- designing region: 19:28485001-28515000_strand_+_gene_1270 -- region: 1270, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1270_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.473s.\n",
      "-- designing region: 19:29485001-29515000_strand_+_gene_1271 -- region: 1271, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1271_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.470s.\n",
      "-- designing region: 19:30485001-30515000_strand_+_gene_1272 -- region: 1272, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1272_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.429s.\n",
      "-- designing region: 19:31485001-31515000_strand_+_gene_1273 -- region: 1273, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1273_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "- Designed 59918 candidate probes in 15.440s.\n",
      "-- designing region: 19:32485001-32515000_strand_+_gene_1274 -- region: 1274, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1274_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.403s.\n",
      "-- designing region: 19:33485001-33515000_strand_+_gene_1275 -- region: 1275, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1275_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.474s.\n",
      "-- designing region: 19:34485001-34515000_strand_+_gene_1276 -- region: 1276, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1276_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.441s.\n",
      "-- designing region: 19:35485001-35515000_strand_+_gene_1277 -- region: 1277, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1277_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.466s.\n",
      "-- designing region: 19:36485001-36515000_strand_+_gene_1278 -- region: 1278, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1278_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.267s.\n",
      "- Designed 59918 candidate probes in 15.412s.\n",
      "-- designing region: 19:37485001-37515000_strand_+_gene_1279 -- region: 1279, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1279_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.414s.\n",
      "-- designing region: 19:38485001-38515000_strand_+_gene_1280 -- region: 1280, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1280_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.456s.\n",
      "-- designing region: 19:39485001-39515000_strand_+_gene_1281 -- region: 1281, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1281_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.431s.\n",
      "-- designing region: 19:40485001-40515000_strand_+_gene_1282 -- region: 1282, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1282_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.433s.\n",
      "-- designing region: 19:41485001-41515000_strand_+_gene_1283 -- region: 1283, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1283_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.449s.\n",
      "-- designing region: 19:42485001-42515000_strand_+_gene_1284 -- region: 1284, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1284_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.422s.\n",
      "-- designing region: 19:43485001-43515000_strand_+_gene_1285 -- region: 1285, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1285_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.414s.\n",
      "-- designing region: 19:44485001-44515000_strand_+_gene_1286 -- region: 1286, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1286_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.413s.\n",
      "-- designing region: 19:45485001-45515000_strand_+_gene_1287 -- region: 1287, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1287_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.464s.\n",
      "-- designing region: 19:46485001-46515000_strand_+_gene_1288 -- region: 1288, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1288_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.450s.\n",
      "-- designing region: 19:47485001-47515000_strand_+_gene_1289 -- region: 1289, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1289_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.539s.\n",
      "-- designing region: 19:48485001-48515000_strand_+_gene_1290 -- region: 1290, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1290_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.534s.\n",
      "-- designing region: 19:49485001-49515000_strand_+_gene_1291 -- region: 1291, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1291_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.579s.\n",
      "-- designing region: 19:50485001-50515000_strand_+_gene_1292 -- region: 1292, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1292_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.465s.\n",
      "-- designing region: 19:51485001-51515000_strand_+_gene_1293 -- region: 1293, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1293_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 2.511s.\n",
      "- Designed 59918 candidate probes in 26.966s.\n",
      "-- designing region: 19:52485001-52515000_strand_+_gene_1294 -- region: 1294, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1294_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.663s.\n",
      "- Designed 59918 candidate probes in 17.707s.\n",
      "-- designing region: 19:53485001-53515000_strand_+_gene_1295 -- region: 1295, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1295_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.559s.\n",
      "-- designing region: 19:54485001-54515000_strand_+_gene_1296 -- region: 1296, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1296_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.648s.\n",
      "-- designing region: 19:55485001-55515000_strand_+_gene_1297 -- region: 1297, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1297_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.646s.\n",
      "-- designing region: 19:56485001-56515000_strand_+_gene_1298 -- region: 1298, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1298_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.658s.\n",
      "-- designing region: 19:57485001-57515000_strand_+_gene_1299 -- region: 1299, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1299_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 20:485001-515000_strand_+_gene_1300 -- region: 1300, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1300_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.510s.\n",
      "-- designing region: 20:1485001-1515000_strand_+_gene_1301 -- region: 1301, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1301_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "- Designed 59918 candidate probes in 15.631s.\n",
      "-- designing region: 20:2485001-2515000_strand_+_gene_1302 -- region: 1302, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1302_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.690s.\n",
      "-- designing region: 20:3485001-3515000_strand_+_gene_1303 -- region: 1303, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1303_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.660s.\n",
      "-- designing region: 20:4485001-4515000_strand_+_gene_1304 -- region: 1304, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1304_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.665s.\n",
      "-- designing region: 20:5485001-5515000_strand_+_gene_1305 -- region: 1305, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1305_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.704s.\n",
      "-- designing region: 20:6485001-6515000_strand_+_gene_1306 -- region: 1306, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1306_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.658s.\n",
      "-- designing region: 20:7485001-7515000_strand_+_gene_1307 -- region: 1307, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1307_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.549s.\n",
      "-- designing region: 20:8485001-8515000_strand_+_gene_1308 -- region: 1308, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1308_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.616s.\n",
      "-- designing region: 20:9485001-9515000_strand_+_gene_1309 -- region: 1309, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1309_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.676s.\n",
      "-- designing region: 20:10485001-10515000_strand_+_gene_1310 -- region: 1310, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1310_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.687s.\n",
      "-- designing region: 20:11485001-11515000_strand_+_gene_1311 -- region: 1311, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1311_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.600s.\n",
      "-- designing region: 20:12485001-12515000_strand_+_gene_1312 -- region: 1312, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1312_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 20:13485001-13515000_strand_+_gene_1313 -- region: 1313, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1313_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.672s.\n",
      "-- designing region: 20:14485001-14515000_strand_+_gene_1314 -- region: 1314, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1314_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.618s.\n",
      "-- designing region: 20:15485001-15515000_strand_+_gene_1315 -- region: 1315, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1315_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.659s.\n",
      "-- designing region: 20:16485001-16515000_strand_+_gene_1316 -- region: 1316, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1316_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: 20:17485001-17515000_strand_+_gene_1317 -- region: 1317, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1317_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.754s.\n",
      "-- designing region: 20:18485001-18515000_strand_+_gene_1318 -- region: 1318, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1318_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 20:19485001-19515000_strand_+_gene_1319 -- region: 1319, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1319_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.645s.\n",
      "-- designing region: 20:20485001-20515000_strand_+_gene_1320 -- region: 1320, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1320_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.697s.\n",
      "-- designing region: 20:21485001-21515000_strand_+_gene_1321 -- region: 1321, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1321_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.675s.\n",
      "-- designing region: 20:22485001-22515000_strand_+_gene_1322 -- region: 1322, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1322_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.664s.\n",
      "-- designing region: 20:23485001-23515000_strand_+_gene_1323 -- region: 1323, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1323_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.468s.\n",
      "-- designing region: 20:24485001-24515000_strand_+_gene_1324 -- region: 1324, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1324_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.627s.\n",
      "-- designing region: 20:25485001-25515000_strand_+_gene_1325 -- region: 1325, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1325_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.627s.\n",
      "-- designing region: 20:26485001-26515000_strand_+_gene_1326 -- region: 1326, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1326_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.218s.\n",
      "- Designed 59918 candidate probes in 15.355s.\n",
      "-- designing region: 20:27485001-27515000_strand_+_gene_1327 -- region: 1327, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1327_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.215s.\n",
      "- Designed 59636 candidate probes in 15.368s.\n",
      "-- designing region: 20:28485001-28515000_strand_+_gene_1328 -- region: 1328, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1328_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.237s.\n",
      "- Designed 48460 candidate probes in 12.533s.\n",
      "-- designing region: 20:29485001-29515000_strand_+_gene_1329 -- region: 1329, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1329_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.475s.\n",
      "-- designing region: 20:30485001-30515000_strand_+_gene_1330 -- region: 1330, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1330_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.509s.\n",
      "-- designing region: 20:31485001-31515000_strand_+_gene_1331 -- region: 1331, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1331_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.261s.\n",
      "- Designed 59918 candidate probes in 15.451s.\n",
      "-- designing region: 20:32485001-32515000_strand_+_gene_1332 -- region: 1332, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1332_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.500s.\n",
      "-- designing region: 20:33485001-33515000_strand_+_gene_1333 -- region: 1333, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1333_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.464s.\n",
      "-- designing region: 20:34485001-34515000_strand_+_gene_1334 -- region: 1334, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1334_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.429s.\n",
      "-- designing region: 20:35485001-35515000_strand_+_gene_1335 -- region: 1335, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1335_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.265s.\n",
      "- Designed 59918 candidate probes in 15.481s.\n",
      "-- designing region: 20:36485001-36515000_strand_+_gene_1336 -- region: 1336, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1336_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.500s.\n",
      "-- designing region: 20:37485001-37515000_strand_+_gene_1337 -- region: 1337, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1337_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.502s.\n",
      "-- designing region: 20:38485001-38515000_strand_+_gene_1338 -- region: 1338, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1338_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.471s.\n",
      "-- designing region: 20:39485001-39515000_strand_+_gene_1339 -- region: 1339, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1339_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.492s.\n",
      "-- designing region: 20:40485001-40515000_strand_+_gene_1340 -- region: 1340, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1340_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.500s.\n",
      "-- designing region: 20:41485001-41515000_strand_+_gene_1341 -- region: 1341, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1341_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.504s.\n",
      "-- designing region: 20:42485001-42515000_strand_+_gene_1342 -- region: 1342, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1342_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.453s.\n",
      "-- designing region: 20:43485001-43515000_strand_+_gene_1343 -- region: 1343, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1343_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.448s.\n",
      "-- designing region: 20:44485001-44515000_strand_+_gene_1344 -- region: 1344, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1344_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.471s.\n",
      "-- designing region: 20:45485001-45515000_strand_+_gene_1345 -- region: 1345, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1345_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.489s.\n",
      "-- designing region: 20:46485001-46515000_strand_+_gene_1346 -- region: 1346, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1346_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.454s.\n",
      "-- designing region: 20:47485001-47515000_strand_+_gene_1347 -- region: 1347, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1347_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.267s.\n",
      "- Designed 59918 candidate probes in 15.436s.\n",
      "-- designing region: 20:48485001-48515000_strand_+_gene_1348 -- region: 1348, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1348_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.253s.\n",
      "- Designed 59918 candidate probes in 15.430s.\n",
      "-- designing region: 20:49485001-49515000_strand_+_gene_1349 -- region: 1349, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1349_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.476s.\n",
      "-- designing region: 20:50485001-50515000_strand_+_gene_1350 -- region: 1350, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1350_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.495s.\n",
      "-- designing region: 20:51485001-51515000_strand_+_gene_1351 -- region: 1351, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1351_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.551s.\n",
      "-- designing region: 20:52485001-52515000_strand_+_gene_1352 -- region: 1352, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1352_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.568s.\n",
      "-- designing region: 20:53485001-53515000_strand_+_gene_1353 -- region: 1353, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1353_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.521s.\n",
      "-- designing region: 20:54485001-54515000_strand_+_gene_1354 -- region: 1354, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1354_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 18.235s.\n",
      "-- designing region: 20:55485001-55515000_strand_+_gene_1355 -- region: 1355, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1355_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.422s.\n",
      "- Designed 59918 candidate probes in 22.270s.\n",
      "-- designing region: 20:56485001-56515000_strand_+_gene_1356 -- region: 1356, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1356_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.598s.\n",
      "-- designing region: 20:57485001-57515000_strand_+_gene_1357 -- region: 1357, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1357_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.721s.\n",
      "-- designing region: 20:58485001-58515000_strand_+_gene_1358 -- region: 1358, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1358_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.737s.\n",
      "-- designing region: 20:59485001-59515000_strand_+_gene_1359 -- region: 1359, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1359_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.686s.\n",
      "-- designing region: 20:60485001-60515000_strand_+_gene_1360 -- region: 1360, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1360_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.673s.\n",
      "-- designing region: 20:61485001-61515000_strand_+_gene_1361 -- region: 1361, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1361_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.727s.\n",
      "-- designing region: 20:62485001-62515000_strand_+_gene_1362 -- region: 1362, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1362_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.725s.\n",
      "-- designing region: 20:63485001-63515000_strand_+_gene_1363 -- region: 1363, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1363_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.717s.\n",
      "-- designing region: 21:485001-515000_strand_+_gene_1364 -- region: 1364, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1364_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.211s.\n",
      "- Designed 0 candidate probes in 0.503s.\n",
      "-- designing region: 21:1485001-1515000_strand_+_gene_1365 -- region: 1365, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1365_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.576s.\n",
      "- Designed 0 candidate probes in 0.872s.\n",
      "-- designing region: 21:2485001-2515000_strand_+_gene_1366 -- region: 1366, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1366_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.588s.\n",
      "- Designed 0 candidate probes in 0.883s.\n",
      "-- designing region: 21:3485001-3515000_strand_+_gene_1367 -- region: 1367, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1367_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.581s.\n",
      "- Designed 0 candidate probes in 0.876s.\n",
      "-- designing region: 21:4485001-4515000_strand_+_gene_1368 -- region: 1368, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1368_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.577s.\n",
      "- Designed 0 candidate probes in 0.877s.\n",
      "-- designing region: 21:5485001-5515000_strand_+_gene_1369 -- region: 1369, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1369_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.617s.\n",
      "- Designed 31894 candidate probes in 9.145s.\n",
      "-- designing region: 21:6485001-6515000_strand_+_gene_1370 -- region: 1370, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1370_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.679s.\n",
      "-- designing region: 21:7485001-7515000_strand_+_gene_1371 -- region: 1371, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1371_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.249s.\n",
      "- Designed 31698 candidate probes in 8.763s.\n",
      "-- designing region: 21:8485001-8515000_strand_+_gene_1372 -- region: 1372, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1372_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.213s.\n",
      "- Designed 0 candidate probes in 0.509s.\n",
      "-- designing region: 21:9485001-9515000_strand_+_gene_1373 -- region: 1373, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1373_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.578s.\n",
      "- Designed 0 candidate probes in 0.876s.\n",
      "-- designing region: 21:10485001-10515000_strand_+_gene_1374 -- region: 1374, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1374_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.659s.\n",
      "- Designed 59918 candidate probes in 16.001s.\n",
      "-- designing region: 21:11485001-11515000_strand_+_gene_1375 -- region: 1375, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1375_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.225s.\n",
      "- Designed 59918 candidate probes in 15.499s.\n",
      "-- designing region: 21:12485001-12515000_strand_+_gene_1376 -- region: 1376, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1376_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.218s.\n",
      "- Designed 59918 candidate probes in 15.473s.\n",
      "-- designing region: 21:13485001-13515000_strand_+_gene_1377 -- region: 1377, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1377_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.513s.\n",
      "-- designing region: 21:14485001-14515000_strand_+_gene_1378 -- region: 1378, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1378_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.635s.\n",
      "-- designing region: 21:15485001-15515000_strand_+_gene_1379 -- region: 1379, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1379_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.675s.\n",
      "-- designing region: 21:16485001-16515000_strand_+_gene_1380 -- region: 1380, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1380_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.620s.\n",
      "-- designing region: 21:17485001-17515000_strand_+_gene_1381 -- region: 1381, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1381_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.679s.\n",
      "-- designing region: 21:18485001-18515000_strand_+_gene_1382 -- region: 1382, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1382_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.654s.\n",
      "-- designing region: 21:19485001-19515000_strand_+_gene_1383 -- region: 1383, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1383_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.692s.\n",
      "-- designing region: 21:20485001-20515000_strand_+_gene_1384 -- region: 1384, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1384_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.649s.\n",
      "-- designing region: 21:21485001-21515000_strand_+_gene_1385 -- region: 1385, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1385_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.635s.\n",
      "-- designing region: 21:22485001-22515000_strand_+_gene_1386 -- region: 1386, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1386_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.607s.\n",
      "-- designing region: 21:23485001-23515000_strand_+_gene_1387 -- region: 1387, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1387_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: 21:24485001-24515000_strand_+_gene_1388 -- region: 1388, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1388_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.499s.\n",
      "-- designing region: 21:25485001-25515000_strand_+_gene_1389 -- region: 1389, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1389_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.658s.\n",
      "-- designing region: 21:26485001-26515000_strand_+_gene_1390 -- region: 1390, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1390_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.557s.\n",
      "-- designing region: 21:27485001-27515000_strand_+_gene_1391 -- region: 1391, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1391_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.595s.\n",
      "-- designing region: 21:28485001-28515000_strand_+_gene_1392 -- region: 1392, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1392_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.610s.\n",
      "-- designing region: 21:29485001-29515000_strand_+_gene_1393 -- region: 1393, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1393_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 21:30485001-30515000_strand_+_gene_1394 -- region: 1394, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1394_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.581s.\n",
      "-- designing region: 21:31485001-31515000_strand_+_gene_1395 -- region: 1395, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1395_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.629s.\n",
      "-- designing region: 21:32485001-32515000_strand_+_gene_1396 -- region: 1396, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1396_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.608s.\n",
      "-- designing region: 21:33485001-33515000_strand_+_gene_1397 -- region: 1397, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1397_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.612s.\n",
      "-- designing region: 21:34485001-34515000_strand_+_gene_1398 -- region: 1398, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1398_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.509s.\n",
      "-- designing region: 21:35485001-35515000_strand_+_gene_1399 -- region: 1399, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1399_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.531s.\n",
      "-- designing region: 21:36485001-36515000_strand_+_gene_1400 -- region: 1400, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1400_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.527s.\n",
      "-- designing region: 21:37485001-37515000_strand_+_gene_1401 -- region: 1401, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1401_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.466s.\n",
      "-- designing region: 21:38485001-38515000_strand_+_gene_1402 -- region: 1402, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1402_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.496s.\n",
      "-- designing region: 21:39485001-39515000_strand_+_gene_1403 -- region: 1403, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1403_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.506s.\n",
      "-- designing region: 21:40485001-40515000_strand_+_gene_1404 -- region: 1404, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1404_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.479s.\n",
      "-- designing region: 21:41485001-41515000_strand_+_gene_1405 -- region: 1405, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1405_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "- Designed 59918 candidate probes in 15.511s.\n",
      "-- designing region: 21:42485001-42515000_strand_+_gene_1406 -- region: 1406, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1406_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.506s.\n",
      "-- designing region: 21:43485001-43515000_strand_+_gene_1407 -- region: 1407, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1407_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.528s.\n",
      "-- designing region: 21:44485001-44515000_strand_+_gene_1408 -- region: 1408, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1408_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.521s.\n",
      "-- designing region: 21:45485001-45515000_strand_+_gene_1409 -- region: 1409, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1409_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.501s.\n",
      "-- designing region: 22:485001-515000_strand_+_gene_1410 -- region: 1410, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1410_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.209s.\n",
      "- Designed 0 candidate probes in 0.504s.\n",
      "-- designing region: 22:1485001-1515000_strand_+_gene_1411 -- region: 1411, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1411_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: 22:2485001-2515000_strand_+_gene_1412 -- region: 1412, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1412_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.560s.\n",
      "- Designed 0 candidate probes in 0.854s.\n",
      "-- designing region: 22:3485001-3515000_strand_+_gene_1413 -- region: 1413, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1413_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.550s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: 22:4485001-4515000_strand_+_gene_1414 -- region: 1414, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1414_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: 22:5485001-5515000_strand_+_gene_1415 -- region: 1415, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1415_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.551s.\n",
      "- Designed 0 candidate probes in 0.844s.\n",
      "-- designing region: 22:6485001-6515000_strand_+_gene_1416 -- region: 1416, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1416_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.556s.\n",
      "- Designed 0 candidate probes in 0.852s.\n",
      "-- designing region: 22:7485001-7515000_strand_+_gene_1417 -- region: 1417, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1417_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.848s.\n",
      "-- designing region: 22:8485001-8515000_strand_+_gene_1418 -- region: 1418, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1418_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: 22:9485001-9515000_strand_+_gene_1419 -- region: 1419, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1419_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: 22:10485001-10515000_strand_+_gene_1420 -- region: 1420, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1420_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.565s.\n",
      "- Designed 9918 candidate probes in 3.683s.\n",
      "-- designing region: 22:11485001-11515000_strand_+_gene_1421 -- region: 1421, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1421_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.230s.\n",
      "- Designed 24592 candidate probes in 6.814s.\n",
      "-- designing region: 22:12485001-12515000_strand_+_gene_1422 -- region: 1422, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1422_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.237s.\n",
      "- Designed 52538 candidate probes in 13.543s.\n",
      "-- designing region: 22:13485001-13515000_strand_+_gene_1423 -- region: 1423, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1423_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.228s.\n",
      "- Designed 59918 candidate probes in 15.335s.\n",
      "-- designing region: 22:14485001-14515000_strand_+_gene_1424 -- region: 1424, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1424_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.220s.\n",
      "- Designed 59918 candidate probes in 15.280s.\n",
      "-- designing region: 22:15485001-15515000_strand_+_gene_1425 -- region: 1425, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1425_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.397s.\n",
      "-- designing region: 22:16485001-16515000_strand_+_gene_1426 -- region: 1426, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1426_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.476s.\n",
      "-- designing region: 22:17485001-17515000_strand_+_gene_1427 -- region: 1427, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1427_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.481s.\n",
      "-- designing region: 22:18485001-18515000_strand_+_gene_1428 -- region: 1428, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1428_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.483s.\n",
      "-- designing region: 22:19485001-19515000_strand_+_gene_1429 -- region: 1429, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1429_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.478s.\n",
      "-- designing region: 22:20485001-20515000_strand_+_gene_1430 -- region: 1430, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1430_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "- Designed 59918 candidate probes in 15.464s.\n",
      "-- designing region: 22:21485001-21515000_strand_+_gene_1431 -- region: 1431, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1431_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.533s.\n",
      "-- designing region: 22:22485001-22515000_strand_+_gene_1432 -- region: 1432, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1432_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.548s.\n",
      "-- designing region: 22:23485001-23515000_strand_+_gene_1433 -- region: 1433, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1433_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.363s.\n",
      "-- designing region: 22:24485001-24515000_strand_+_gene_1434 -- region: 1434, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1434_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 17.608s.\n",
      "-- designing region: 22:25485001-25515000_strand_+_gene_1435 -- region: 1435, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1435_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.418s.\n",
      "- Designed 59918 candidate probes in 23.789s.\n",
      "-- designing region: 22:26485001-26515000_strand_+_gene_1436 -- region: 1436, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1436_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.571s.\n",
      "-- designing region: 22:27485001-27515000_strand_+_gene_1437 -- region: 1437, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1437_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.716s.\n",
      "-- designing region: 22:28485001-28515000_strand_+_gene_1438 -- region: 1438, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1438_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.630s.\n",
      "-- designing region: 22:29485001-29515000_strand_+_gene_1439 -- region: 1439, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1439_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.668s.\n",
      "-- designing region: 22:30485001-30515000_strand_+_gene_1440 -- region: 1440, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1440_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.694s.\n",
      "-- designing region: 22:31485001-31515000_strand_+_gene_1441 -- region: 1441, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1441_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.615s.\n",
      "-- designing region: 22:32485001-32515000_strand_+_gene_1442 -- region: 1442, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1442_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.695s.\n",
      "-- designing region: 22:33485001-33515000_strand_+_gene_1443 -- region: 1443, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1443_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.679s.\n",
      "-- designing region: 22:34485001-34515000_strand_+_gene_1444 -- region: 1444, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1444_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.700s.\n",
      "-- designing region: 22:35485001-35515000_strand_+_gene_1445 -- region: 1445, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1445_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "- Designed 59918 candidate probes in 15.673s.\n",
      "-- designing region: 22:36485001-36515000_strand_+_gene_1446 -- region: 1446, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1446_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.634s.\n",
      "-- designing region: 22:37485001-37515000_strand_+_gene_1447 -- region: 1447, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1447_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.694s.\n",
      "-- designing region: 22:38485001-38515000_strand_+_gene_1448 -- region: 1448, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1448_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.701s.\n",
      "-- designing region: 22:39485001-39515000_strand_+_gene_1449 -- region: 1449, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1449_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.674s.\n",
      "-- designing region: 22:40485001-40515000_strand_+_gene_1450 -- region: 1450, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1450_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 16.330s.\n",
      "-- designing region: 22:41485001-41515000_strand_+_gene_1451 -- region: 1451, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1451_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.632s.\n",
      "-- designing region: 22:42485001-42515000_strand_+_gene_1452 -- region: 1452, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1452_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.602s.\n",
      "-- designing region: 22:43485001-43515000_strand_+_gene_1453 -- region: 1453, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1453_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "- Designed 59918 candidate probes in 15.625s.\n",
      "-- designing region: 22:44485001-44515000_strand_+_gene_1454 -- region: 1454, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1454_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.672s.\n",
      "-- designing region: 22:45485001-45515000_strand_+_gene_1455 -- region: 1455, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1455_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.728s.\n",
      "-- designing region: 22:46485001-46515000_strand_+_gene_1456 -- region: 1456, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1456_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.656s.\n",
      "-- designing region: 22:47485001-47515000_strand_+_gene_1457 -- region: 1457, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1457_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.675s.\n",
      "-- designing region: 22:48485001-48515000_strand_+_gene_1458 -- region: 1458, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1458_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.699s.\n",
      "-- designing region: 22:49485001-49515000_strand_+_gene_1459 -- region: 1459, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1459_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.730s.\n",
      "-- designing region: X:1235001-1265000_strand_+_gene_1460 -- region: 1460, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1460_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "- Designed 59918 candidate probes in 15.552s.\n",
      "-- designing region: X:3735001-3765000_strand_+_gene_1461 -- region: 1461, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1461_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.599s.\n",
      "-- designing region: X:6235001-6265000_strand_+_gene_1462 -- region: 1462, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1462_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.698s.\n",
      "-- designing region: X:8735001-8765000_strand_+_gene_1463 -- region: 1463, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1463_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.612s.\n",
      "-- designing region: X:11235001-11265000_strand_+_gene_1464 -- region: 1464, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1464_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.647s.\n",
      "-- designing region: X:13735001-13765000_strand_+_gene_1465 -- region: 1465, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1465_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.664s.\n",
      "-- designing region: X:16235001-16265000_strand_+_gene_1466 -- region: 1466, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1466_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.656s.\n",
      "-- designing region: X:18735001-18765000_strand_+_gene_1467 -- region: 1467, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1467_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.608s.\n",
      "-- designing region: X:21235001-21265000_strand_+_gene_1468 -- region: 1468, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1468_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.582s.\n",
      "-- designing region: X:23735001-23765000_strand_+_gene_1469 -- region: 1469, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1469_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.566s.\n",
      "-- designing region: X:26235001-26265000_strand_+_gene_1470 -- region: 1470, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1470_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.471s.\n",
      "-- designing region: X:28735001-28765000_strand_+_gene_1471 -- region: 1471, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1471_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.524s.\n",
      "-- designing region: X:31235001-31265000_strand_+_gene_1472 -- region: 1472, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1472_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.289s.\n",
      "- Designed 59918 candidate probes in 15.497s.\n",
      "-- designing region: X:33735001-33765000_strand_+_gene_1473 -- region: 1473, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1473_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.499s.\n",
      "-- designing region: X:36235001-36265000_strand_+_gene_1474 -- region: 1474, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1474_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.509s.\n",
      "-- designing region: X:38735001-38765000_strand_+_gene_1475 -- region: 1475, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1475_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.489s.\n",
      "-- designing region: X:41235001-41265000_strand_+_gene_1476 -- region: 1476, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1476_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.482s.\n",
      "-- designing region: X:43735001-43765000_strand_+_gene_1477 -- region: 1477, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1477_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.503s.\n",
      "-- designing region: X:46235001-46265000_strand_+_gene_1478 -- region: 1478, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1478_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 15.488s.\n",
      "-- designing region: X:48735001-48765000_strand_+_gene_1479 -- region: 1479, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1479_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.259s.\n",
      "- Designed 59918 candidate probes in 15.463s.\n",
      "-- designing region: X:51235001-51265000_strand_+_gene_1480 -- region: 1480, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1480_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.482s.\n",
      "-- designing region: X:53735001-53765000_strand_+_gene_1481 -- region: 1481, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1481_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.450s.\n",
      "-- designing region: X:56235001-56265000_strand_+_gene_1482 -- region: 1482, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1482_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.472s.\n",
      "-- designing region: X:58735001-58765000_strand_+_gene_1483 -- region: 1483, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1483_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.234s.\n",
      "- Designed 59636 candidate probes in 15.285s.\n",
      "-- designing region: X:61235001-61265000_strand_+_gene_1484 -- region: 1484, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1484_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.225s.\n",
      "- Designed 59918 candidate probes in 15.341s.\n",
      "-- designing region: X:63735001-63765000_strand_+_gene_1485 -- region: 1485, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1485_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.436s.\n",
      "-- designing region: X:66235001-66265000_strand_+_gene_1486 -- region: 1486, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1486_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.525s.\n",
      "-- designing region: X:68735001-68765000_strand_+_gene_1487 -- region: 1487, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1487_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.501s.\n",
      "-- designing region: X:71235001-71265000_strand_+_gene_1488 -- region: 1488, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1488_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "- Designed 59918 candidate probes in 15.506s.\n",
      "-- designing region: X:73735001-73765000_strand_+_gene_1489 -- region: 1489, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1489_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.462s.\n",
      "-- designing region: X:76235001-76265000_strand_+_gene_1490 -- region: 1490, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1490_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.457s.\n",
      "-- designing region: X:78735001-78765000_strand_+_gene_1491 -- region: 1491, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1491_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "- Designed 59918 candidate probes in 15.518s.\n",
      "-- designing region: X:81235001-81265000_strand_+_gene_1492 -- region: 1492, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1492_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "- Designed 59918 candidate probes in 15.490s.\n",
      "-- designing region: X:83735001-83765000_strand_+_gene_1493 -- region: 1493, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1493_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.502s.\n",
      "-- designing region: X:86235001-86265000_strand_+_gene_1494 -- region: 1494, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1494_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "- Designed 59918 candidate probes in 15.532s.\n",
      "-- designing region: X:88735001-88765000_strand_+_gene_1495 -- region: 1495, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1495_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.488s.\n",
      "-- designing region: X:91235001-91265000_strand_+_gene_1496 -- region: 1496, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1496_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 20.020s.\n",
      "-- designing region: X:93735001-93765000_strand_+_gene_1497 -- region: 1497, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1497_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.622s.\n",
      "- Designed 59918 candidate probes in 22.269s.\n",
      "-- designing region: X:96235001-96265000_strand_+_gene_1498 -- region: 1498, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1498_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "- Designed 59918 candidate probes in 15.564s.\n",
      "-- designing region: X:98735001-98765000_strand_+_gene_1499 -- region: 1499, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1499_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "- Designed 59918 candidate probes in 15.633s.\n",
      "-- designing region: X:101235001-101265000_strand_+_gene_1500 -- region: 1500, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1500_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.597s.\n",
      "-- designing region: X:103735001-103765000_strand_+_gene_1501 -- region: 1501, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1501_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "- Designed 59918 candidate probes in 15.687s.\n",
      "-- designing region: X:106235001-106265000_strand_+_gene_1502 -- region: 1502, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1502_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.670s.\n",
      "-- designing region: X:108735001-108765000_strand_+_gene_1503 -- region: 1503, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1503_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.653s.\n",
      "-- designing region: X:111235001-111265000_strand_+_gene_1504 -- region: 1504, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1504_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.678s.\n",
      "-- designing region: X:113735001-113765000_strand_+_gene_1505 -- region: 1505, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1505_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.668s.\n",
      "-- designing region: X:116235001-116265000_strand_+_gene_1506 -- region: 1506, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1506_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.695s.\n",
      "-- designing region: X:118735001-118765000_strand_+_gene_1507 -- region: 1507, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1507_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.723s.\n",
      "-- designing region: X:121235001-121265000_strand_+_gene_1508 -- region: 1508, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1508_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "- Designed 59918 candidate probes in 15.671s.\n",
      "-- designing region: X:123735001-123765000_strand_+_gene_1509 -- region: 1509, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1509_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "- Designed 59918 candidate probes in 15.969s.\n",
      "-- designing region: X:126235001-126265000_strand_+_gene_1510 -- region: 1510, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1510_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "- Designed 59918 candidate probes in 15.657s.\n",
      "-- designing region: X:128735001-128765000_strand_+_gene_1511 -- region: 1511, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1511_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.619s.\n",
      "-- designing region: X:131235001-131265000_strand_+_gene_1512 -- region: 1512, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1512_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "- Designed 59918 candidate probes in 15.678s.\n",
      "-- designing region: X:133735001-133765000_strand_+_gene_1513 -- region: 1513, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1513_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.636s.\n",
      "-- designing region: X:136235001-136265000_strand_+_gene_1514 -- region: 1514, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1514_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.669s.\n",
      "-- designing region: X:138735001-138765000_strand_+_gene_1515 -- region: 1515, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1515_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "- Designed 59918 candidate probes in 92.578s.\n",
      "-- designing region: X:141235001-141265000_strand_+_gene_1516 -- region: 1516, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1516_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "- Designed 59918 candidate probes in 15.938s.\n",
      "-- designing region: X:143735001-143765000_strand_+_gene_1517 -- region: 1517, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1517_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.217s.\n",
      "- Designed 59918 candidate probes in 15.807s.\n",
      "-- designing region: X:146235001-146265000_strand_+_gene_1518 -- region: 1518, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1518_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.862s.\n",
      "-- designing region: X:148735001-148765000_strand_+_gene_1519 -- region: 1519, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1519_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.243s.\n",
      "- Designed 59918 candidate probes in 16.043s.\n",
      "-- designing region: X:151235001-151265000_strand_+_gene_1520 -- region: 1520, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1520_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "- Designed 59918 candidate probes in 15.940s.\n",
      "-- designing region: X:153735001-153765000_strand_+_gene_1521 -- region: 1521, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1521_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.891s.\n",
      "-- designing region: Y:485001-515000_strand_+_gene_1522 -- region: 1522, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1522_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.145s.\n",
      "- Designed 0 candidate probes in 0.440s.\n",
      "-- designing region: Y:1485001-1515000_strand_+_gene_1523 -- region: 1523, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1523_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.459s.\n",
      "- Designed 0 candidate probes in 0.762s.\n",
      "-- designing region: Y:2485001-2515000_strand_+_gene_1524 -- region: 1524, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1524_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.450s.\n",
      "- Designed 0 candidate probes in 0.746s.\n",
      "-- designing region: Y:3485001-3515000_strand_+_gene_1525 -- region: 1525, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1525_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.581s.\n",
      "- Designed 59918 candidate probes in 16.273s.\n",
      "-- designing region: Y:4485001-4515000_strand_+_gene_1526 -- region: 1526, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1526_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "- Designed 59918 candidate probes in 15.882s.\n",
      "-- designing region: Y:5485001-5515000_strand_+_gene_1527 -- region: 1527, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1527_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "- Designed 59918 candidate probes in 15.870s.\n",
      "-- designing region: Y:6485001-6515000_strand_+_gene_1528 -- region: 1528, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1528_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.919s.\n",
      "-- designing region: Y:7485001-7515000_strand_+_gene_1529 -- region: 1529, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1529_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.213s.\n",
      "- Designed 59918 candidate probes in 15.770s.\n",
      "-- designing region: Y:8485001-8515000_strand_+_gene_1530 -- region: 1530, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1530_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.798s.\n",
      "-- designing region: Y:9485001-9515000_strand_+_gene_1531 -- region: 1531, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1531_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.256s.\n",
      "- Designed 59918 candidate probes in 15.623s.\n",
      "-- designing region: Y:10485001-10515000_strand_+_gene_1532 -- region: 1532, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1532_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.229s.\n",
      "- Designed 59918 candidate probes in 15.355s.\n",
      "-- designing region: Y:11485001-11515000_strand_+_gene_1533 -- region: 1533, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1533_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.239s.\n",
      "- Designed 59918 candidate probes in 15.424s.\n",
      "-- designing region: Y:12485001-12515000_strand_+_gene_1534 -- region: 1534, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1534_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.354s.\n",
      "- Designed 59918 candidate probes in 15.528s.\n",
      "-- designing region: Y:13485001-13515000_strand_+_gene_1535 -- region: 1535, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1535_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.547s.\n",
      "-- designing region: Y:14485001-14515000_strand_+_gene_1536 -- region: 1536, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1536_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.537s.\n",
      "-- designing region: Y:15485001-15515000_strand_+_gene_1537 -- region: 1537, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1537_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "- Designed 59918 candidate probes in 15.534s.\n",
      "-- designing region: Y:16485001-16515000_strand_+_gene_1538 -- region: 1538, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1538_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "- Designed 59918 candidate probes in 15.484s.\n",
      "-- designing region: Y:17485001-17515000_strand_+_gene_1539 -- region: 1539, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1539_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.304s.\n",
      "- Designed 59918 candidate probes in 15.531s.\n",
      "-- designing region: Y:18485001-18515000_strand_+_gene_1540 -- region: 1540, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1540_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "- Designed 59918 candidate probes in 15.457s.\n",
      "-- designing region: Y:19485001-19515000_strand_+_gene_1541 -- region: 1541, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1541_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.283s.\n",
      "- Designed 59918 candidate probes in 15.506s.\n",
      "-- designing region: Y:20485001-20515000_strand_+_gene_1542 -- region: 1542, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1542_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "- Designed 59918 candidate probes in 15.500s.\n",
      "-- designing region: Y:21485001-21515000_strand_+_gene_1543 -- region: 1543, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1543_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "- Designed 59918 candidate probes in 15.535s.\n",
      "-- designing region: Y:22485001-22515000_strand_+_gene_1544 -- region: 1544, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1544_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "- Designed 59918 candidate probes in 15.527s.\n",
      "-- designing region: Y:23485001-23515000_strand_+_gene_1545 -- region: 1545, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1545_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.491s.\n",
      "-- designing region: Y:24485001-24515000_strand_+_gene_1546 -- region: 1546, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1546_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "- Designed 59918 candidate probes in 15.514s.\n",
      "-- designing region: Y:25485001-25515000_strand_+_gene_1547 -- region: 1547, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1547_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.521s.\n",
      "-- designing region: Y:26485001-26515000_strand_+_gene_1548 -- region: 1548, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1548_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "- Designed 59918 candidate probes in 15.495s.\n",
      "-- designing region: Y:27485001-27515000_strand_+_gene_1549 -- region: 1549, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1549_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.200s.\n",
      "- Designed 0 candidate probes in 0.495s.\n",
      "-- designing region: Y:28485001-28515000_strand_+_gene_1550 -- region: 1550, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1550_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.563s.\n",
      "- Designed 0 candidate probes in 0.857s.\n",
      "-- designing region: Y:29485001-29515000_strand_+_gene_1551 -- region: 1551, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1551_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: Y:30485001-30515000_strand_+_gene_1552 -- region: 1552, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1552_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.561s.\n",
      "- Designed 0 candidate probes in 0.856s.\n",
      "-- designing region: Y:31485001-31515000_strand_+_gene_1553 -- region: 1553, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1553_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.550s.\n",
      "- Designed 0 candidate probes in 0.841s.\n",
      "-- designing region: Y:32485001-32515000_strand_+_gene_1554 -- region: 1554, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1554_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.560s.\n",
      "- Designed 0 candidate probes in 0.854s.\n",
      "-- designing region: Y:33485001-33515000_strand_+_gene_1555 -- region: 1555, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1555_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.555s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: Y:34485001-34515000_strand_+_gene_1556 -- region: 1556, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1556_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.851s.\n",
      "-- designing region: Y:35485001-35515000_strand_+_gene_1557 -- region: 1557, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1557_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.846s.\n",
      "-- designing region: Y:36485001-36515000_strand_+_gene_1558 -- region: 1558, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1558_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.552s.\n",
      "- Designed 0 candidate probes in 0.842s.\n",
      "-- designing region: Y:37485001-37515000_strand_+_gene_1559 -- region: 1559, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1559_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.555s.\n",
      "- Designed 0 candidate probes in 0.850s.\n",
      "-- designing region: Y:38485001-38515000_strand_+_gene_1560 -- region: 1560, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1560_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.551s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: Y:39485001-39515000_strand_+_gene_1561 -- region: 1561, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1561_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: Y:40485001-40515000_strand_+_gene_1562 -- region: 1562, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1562_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.561s.\n",
      "- Designed 0 candidate probes in 0.855s.\n",
      "-- designing region: Y:41485001-41515000_strand_+_gene_1563 -- region: 1563, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1563_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: Y:42485001-42515000_strand_+_gene_1564 -- region: 1564, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1564_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.559s.\n",
      "- Designed 0 candidate probes in 0.854s.\n",
      "-- designing region: Y:43485001-43515000_strand_+_gene_1565 -- region: 1565, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1565_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.551s.\n",
      "- Designed 0 candidate probes in 0.846s.\n",
      "-- designing region: Y:44485001-44515000_strand_+_gene_1566 -- region: 1566, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1566_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.558s.\n",
      "- Designed 0 candidate probes in 0.851s.\n",
      "-- designing region: Y:45485001-45515000_strand_+_gene_1567 -- region: 1567, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1567_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.846s.\n",
      "-- designing region: Y:46485001-46515000_strand_+_gene_1568 -- region: 1568, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1568_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.551s.\n",
      "- Designed 0 candidate probes in 0.847s.\n",
      "-- designing region: Y:47485001-47515000_strand_+_gene_1569 -- region: 1569, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1569_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: Y:48485001-48515000_strand_+_gene_1570 -- region: 1570, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1570_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: Y:49485001-49515000_strand_+_gene_1571 -- region: 1571, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1571_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.552s.\n",
      "- Designed 0 candidate probes in 0.848s.\n",
      "-- designing region: Y:50485001-50515000_strand_+_gene_1572 -- region: 1572, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1572_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.848s.\n",
      "-- designing region: Y:51485001-51515000_strand_+_gene_1573 -- region: 1573, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1573_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.850s.\n",
      "-- designing region: Y:52485001-52515000_strand_+_gene_1574 -- region: 1574, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1574_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.564s.\n",
      "- Designed 0 candidate probes in 0.860s.\n",
      "-- designing region: Y:53485001-53515000_strand_+_gene_1575 -- region: 1575, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1575_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: Y:54485001-54515000_strand_+_gene_1576 -- region: 1576, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1576_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.561s.\n",
      "- Designed 0 candidate probes in 0.855s.\n",
      "-- designing region: Y:55485001-55515000_strand_+_gene_1577 -- region: 1577, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1577_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "- Designed 0 candidate probes in 0.849s.\n",
      "-- designing region: Y:56485001-56515000_strand_+_gene_1578 -- region: 1578, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\sequences\\1578_reg_0.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "- Designed 0 candidate probes in 0.848s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\reports\\merged_probes.pbr\n",
      "CPU times: total: 7h 7min 3s\n",
      "Wall time: 7h 14min 10s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "pb_designer.computeOTmaps() # load the tables \n",
    "pb_designer.compute_pb_report() # design candidate probes"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e23bacce",
   "metadata": {},
   "source": [
    "### check probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "8541e32f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:0 1:1235001-1265000_strand_+_gene_0, 57068 candidate probes\n",
      "--- 19930 probes passed check_dic selection.\n",
      "finish in 1.719s, 344 probes kept.\n",
      "-- check region:1 1:3735001-3765000_strand_+_gene_1, 57548 candidate probes\n",
      "--- 17833 probes passed check_dic selection.\n",
      "finish in 1.605s, 319 probes kept.\n",
      "-- check region:2 1:6235001-6265000_strand_+_gene_2, 58584 candidate probes\n",
      "--- 24326 probes passed check_dic selection.\n",
      "finish in 1.832s, 377 probes kept.\n",
      "-- check region:3 1:8735001-8765000_strand_+_gene_3, 58690 candidate probes\n",
      "--- 12490 probes passed check_dic selection.\n",
      "finish in 1.391s, 242 probes kept.\n",
      "-- check region:4 1:11235001-11265000_strand_+_gene_4, 55772 candidate probes\n",
      "--- 15903 probes passed check_dic selection.\n",
      "finish in 1.482s, 256 probes kept.\n",
      "-- check region:5 1:13735001-13765000_strand_+_gene_5, 58958 candidate probes\n",
      "--- 16480 probes passed check_dic selection.\n",
      "finish in 1.511s, 303 probes kept.\n",
      "-- check region:6 1:16235001-16265000_strand_+_gene_6, 57896 candidate probes\n",
      "--- 16713 probes passed check_dic selection.\n",
      "finish in 1.550s, 300 probes kept.\n",
      "-- check region:7 1:18735001-18765000_strand_+_gene_7, 58950 candidate probes\n",
      "--- 19815 probes passed check_dic selection.\n",
      "finish in 1.670s, 341 probes kept.\n",
      "-- check region:8 1:21235001-21265000_strand_+_gene_8, 58040 candidate probes\n",
      "--- 20685 probes passed check_dic selection.\n",
      "finish in 1.693s, 323 probes kept.\n",
      "-- check region:9 1:23735001-23765000_strand_+_gene_9, 57028 candidate probes\n",
      "--- 14310 probes passed check_dic selection.\n",
      "finish in 1.445s, 264 probes kept.\n",
      "-- check region:10 1:26235001-26265000_strand_+_gene_10, 56510 candidate probes\n",
      "--- 11115 probes passed check_dic selection.\n",
      "finish in 1.315s, 216 probes kept.\n",
      "-- check region:11 1:28735001-28765000_strand_+_gene_11, 57208 candidate probes\n",
      "--- 10131 probes passed check_dic selection.\n",
      "finish in 1.286s, 212 probes kept.\n",
      "-- check region:12 1:31235001-31265000_strand_+_gene_12, 57094 candidate probes\n",
      "--- 13737 probes passed check_dic selection.\n",
      "finish in 1.412s, 240 probes kept.\n",
      "-- check region:13 1:33735001-33765000_strand_+_gene_13, 58082 candidate probes\n",
      "--- 21173 probes passed check_dic selection.\n",
      "finish in 1.715s, 341 probes kept.\n",
      "-- check region:14 1:36235001-36265000_strand_+_gene_14, 54616 candidate probes\n",
      "--- 8851 probes passed check_dic selection.\n",
      "finish in 1.180s, 176 probes kept.\n",
      "-- check region:15 1:38735001-38765000_strand_+_gene_15, 58356 candidate probes\n",
      "--- 23078 probes passed check_dic selection.\n",
      "finish in 1.780s, 343 probes kept.\n",
      "-- check region:16 1:41235001-41265000_strand_+_gene_16, 58510 candidate probes\n",
      "--- 12680 probes passed check_dic selection.\n",
      "finish in 1.388s, 245 probes kept.\n",
      "-- check region:17 1:43735001-43765000_strand_+_gene_17, 57914 candidate probes\n",
      "--- 10224 probes passed check_dic selection.\n",
      "finish in 1.285s, 203 probes kept.\n",
      "-- check region:18 1:46235001-46265000_strand_+_gene_18, 53492 candidate probes\n",
      "--- 11883 probes passed check_dic selection.\n",
      "finish in 1.282s, 204 probes kept.\n",
      "-- check region:19 1:48735001-48765000_strand_+_gene_19, 59554 candidate probes\n",
      "--- 24548 probes passed check_dic selection.\n",
      "finish in 1.879s, 391 probes kept.\n",
      "-- check region:20 1:51235001-51265000_strand_+_gene_20, 58438 candidate probes\n",
      "--- 9236 probes passed check_dic selection.\n",
      "finish in 1.275s, 206 probes kept.\n",
      "-- check region:21 1:53735001-53765000_strand_+_gene_21, 58628 candidate probes\n",
      "--- 14481 probes passed check_dic selection.\n",
      "finish in 1.458s, 255 probes kept.\n",
      "-- check region:22 1:56235001-56265000_strand_+_gene_22, 56386 candidate probes\n",
      "--- 17122 probes passed check_dic selection.\n",
      "finish in 1.519s, 303 probes kept.\n",
      "-- check region:23 1:58735001-58765000_strand_+_gene_23, 58276 candidate probes\n",
      "--- 19752 probes passed check_dic selection.\n",
      "finish in 1.649s, 333 probes kept.\n",
      "-- check region:24 1:61235001-61265000_strand_+_gene_24, 58850 candidate probes\n",
      "--- 14774 probes passed check_dic selection.\n",
      "finish in 1.454s, 279 probes kept.\n",
      "-- check region:25 1:63735001-63765000_strand_+_gene_25, 58436 candidate probes\n",
      "--- 13647 probes passed check_dic selection.\n",
      "finish in 1.424s, 238 probes kept.\n",
      "-- check region:26 1:66235001-66265000_strand_+_gene_26, 59758 candidate probes\n",
      "--- 17731 probes passed check_dic selection.\n",
      "finish in 1.589s, 328 probes kept.\n",
      "-- check region:27 1:68735001-68765000_strand_+_gene_27, 50588 candidate probes\n",
      "--- 6119 probes passed check_dic selection.\n",
      "finish in 1.018s, 149 probes kept.\n",
      "-- check region:28 1:71235001-71265000_strand_+_gene_28, 59792 candidate probes\n",
      "--- 12680 probes passed check_dic selection.\n",
      "finish in 1.387s, 262 probes kept.\n",
      "-- check region:29 1:73735001-73765000_strand_+_gene_29, 59876 candidate probes\n",
      "--- 9387 probes passed check_dic selection.\n",
      "finish in 1.246s, 199 probes kept.\n",
      "-- check region:30 1:76235001-76265000_strand_+_gene_30, 58912 candidate probes\n",
      "--- 14145 probes passed check_dic selection.\n",
      "finish in 1.438s, 263 probes kept.\n",
      "-- check region:31 1:78735001-78765000_strand_+_gene_31, 59318 candidate probes\n",
      "--- 9584 probes passed check_dic selection.\n",
      "finish in 1.263s, 215 probes kept.\n",
      "-- check region:32 1:81235001-81265000_strand_+_gene_32, 56240 candidate probes\n",
      "--- 9269 probes passed check_dic selection.\n",
      "finish in 1.211s, 190 probes kept.\n",
      "-- check region:33 1:83735001-83765000_strand_+_gene_33, 59506 candidate probes\n",
      "--- 19081 probes passed check_dic selection.\n",
      "finish in 1.633s, 338 probes kept.\n",
      "-- check region:34 1:86235001-86265000_strand_+_gene_34, 54984 candidate probes\n",
      "--- 12953 probes passed check_dic selection.\n",
      "finish in 1.324s, 230 probes kept.\n",
      "-- check region:35 1:88735001-88765000_strand_+_gene_35, 59162 candidate probes\n",
      "--- 9335 probes passed check_dic selection.\n",
      "finish in 1.247s, 203 probes kept.\n",
      "-- check region:36 1:91235001-91265000_strand_+_gene_36, 53656 candidate probes\n",
      "--- 6720 probes passed check_dic selection.\n",
      "finish in 1.088s, 154 probes kept.\n",
      "-- check region:37 1:93735001-93765000_strand_+_gene_37, 59440 candidate probes\n",
      "--- 23915 probes passed check_dic selection.\n",
      "finish in 1.839s, 358 probes kept.\n",
      "-- check region:38 1:96235001-96265000_strand_+_gene_38, 51066 candidate probes\n",
      "--- 10362 probes passed check_dic selection.\n",
      "finish in 1.177s, 211 probes kept.\n",
      "-- check region:39 1:98735001-98765000_strand_+_gene_39, 59716 candidate probes\n",
      "--- 11984 probes passed check_dic selection.\n",
      "finish in 1.354s, 237 probes kept.\n",
      "-- check region:40 1:101235001-101265000_strand_+_gene_40, 58420 candidate probes\n",
      "--- 15801 probes passed check_dic selection.\n",
      "finish in 1.513s, 293 probes kept.\n",
      "-- check region:41 1:103735001-103765000_strand_+_gene_41, 59728 candidate probes\n",
      "--- 676 probes passed check_dic selection.\n",
      "finish in 0.963s, 22 probes kept.\n",
      "-- check region:42 1:106235001-106265000_strand_+_gene_42, 55970 candidate probes\n",
      "--- 9129 probes passed check_dic selection.\n",
      "finish in 1.193s, 201 probes kept.\n",
      "-- check region:43 1:108735001-108765000_strand_+_gene_43, 59108 candidate probes\n",
      "--- 12044 probes passed check_dic selection.\n",
      "finish in 1.361s, 235 probes kept.\n",
      "-- check region:44 1:111235001-111265000_strand_+_gene_44, 59668 candidate probes\n",
      "--- 19819 probes passed check_dic selection.\n",
      "finish in 1.701s, 317 probes kept.\n",
      "-- check region:45 1:113735001-113765000_strand_+_gene_45, 58218 candidate probes\n",
      "--- 7562 probes passed check_dic selection.\n",
      "finish in 1.180s, 187 probes kept.\n",
      "-- check region:46 1:116235001-116265000_strand_+_gene_46, 59238 candidate probes\n",
      "--- 18078 probes passed check_dic selection.\n",
      "finish in 1.601s, 309 probes kept.\n",
      "-- check region:47 1:118735001-118765000_strand_+_gene_47, 57846 candidate probes\n",
      "--- 10969 probes passed check_dic selection.\n",
      "finish in 1.301s, 213 probes kept.\n",
      "-- check region:48 1:121235001-121265000_strand_+_gene_48, 41546 candidate probes\n",
      "--- 82 probes passed check_dic selection.\n",
      "finish in 0.670s, 3 probes kept.\n",
      "-- check region:49 1:123735001-123765000_strand_+_gene_49, 20180 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.324s, 0 probes kept.\n",
      "-- check region:50 1:126235001-126265000_strand_+_gene_50, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:51 1:128735001-128765000_strand_+_gene_51, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:52 1:131235001-131265000_strand_+_gene_52, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:53 1:133735001-133765000_strand_+_gene_53, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:54 1:136235001-136265000_strand_+_gene_54, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:55 1:138735001-138765000_strand_+_gene_55, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:56 1:141235001-141265000_strand_+_gene_56, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:57 1:143735001-143765000_strand_+_gene_57, 57134 candidate probes\n",
      "--- 1466 probes passed check_dic selection.\n",
      "finish in 0.973s, 60 probes kept.\n",
      "-- check region:58 1:146235001-146265000_strand_+_gene_58, 57912 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.925s, 0 probes kept.\n",
      "-- check region:59 1:148735001-148765000_strand_+_gene_59, 58022 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.911s, 0 probes kept.\n",
      "-- check region:60 1:151235001-151265000_strand_+_gene_60, 56816 candidate probes\n",
      "--- 8909 probes passed check_dic selection.\n",
      "finish in 1.247s, 187 probes kept.\n",
      "-- check region:61 1:153735001-153765000_strand_+_gene_61, 57570 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.949s, 0 probes kept.\n",
      "-- check region:62 1:156235001-156265000_strand_+_gene_62, 57474 candidate probes\n",
      "--- 18009 probes passed check_dic selection.\n",
      "finish in 1.608s, 330 probes kept.\n",
      "-- check region:63 1:158735001-158765000_strand_+_gene_63, 55158 candidate probes\n",
      "--- 11401 probes passed check_dic selection.\n",
      "finish in 1.281s, 241 probes kept.\n",
      "-- check region:64 1:161235001-161265000_strand_+_gene_64, 55970 candidate probes\n",
      "--- 11378 probes passed check_dic selection.\n",
      "finish in 1.320s, 199 probes kept.\n",
      "-- check region:65 1:163735001-163765000_strand_+_gene_65, 51598 candidate probes\n",
      "--- 5205 probes passed check_dic selection.\n",
      "finish in 0.996s, 114 probes kept.\n",
      "-- check region:66 1:166235001-166265000_strand_+_gene_66, 43668 candidate probes\n",
      "--- 6591 probes passed check_dic selection.\n",
      "finish in 0.932s, 127 probes kept.\n",
      "-- check region:67 1:168735001-168765000_strand_+_gene_67, 55396 candidate probes\n",
      "--- 16568 probes passed check_dic selection.\n",
      "finish in 1.483s, 271 probes kept.\n",
      "-- check region:68 1:171235001-171265000_strand_+_gene_68, 58478 candidate probes\n",
      "--- 13146 probes passed check_dic selection.\n",
      "finish in 1.408s, 226 probes kept.\n",
      "-- check region:69 1:173735001-173765000_strand_+_gene_69, 57446 candidate probes\n",
      "--- 5044 probes passed check_dic selection.\n",
      "finish in 1.082s, 128 probes kept.\n",
      "-- check region:70 1:176235001-176265000_strand_+_gene_70, 48472 candidate probes\n",
      "--- 5579 probes passed check_dic selection.\n",
      "finish in 0.957s, 120 probes kept.\n",
      "-- check region:71 1:178735001-178765000_strand_+_gene_71, 57946 candidate probes\n",
      "--- 5882 probes passed check_dic selection.\n",
      "finish in 1.124s, 137 probes kept.\n",
      "-- check region:72 1:181235001-181265000_strand_+_gene_72, 58242 candidate probes\n",
      "--- 16338 probes passed check_dic selection.\n",
      "finish in 1.533s, 267 probes kept.\n",
      "-- check region:73 1:183735001-183765000_strand_+_gene_73, 57770 candidate probes\n",
      "--- 9739 probes passed check_dic selection.\n",
      "finish in 1.311s, 198 probes kept.\n",
      "-- check region:74 1:186235001-186265000_strand_+_gene_74, 53270 candidate probes\n",
      "--- 9123 probes passed check_dic selection.\n",
      "finish in 1.177s, 173 probes kept.\n",
      "-- check region:75 1:188735001-188765000_strand_+_gene_75, 51052 candidate probes\n",
      "--- 6244 probes passed check_dic selection.\n",
      "finish in 1.018s, 149 probes kept.\n",
      "-- check region:76 1:191235001-191265000_strand_+_gene_76, 58588 candidate probes\n",
      "--- 3622 probes passed check_dic selection.\n",
      "finish in 1.039s, 121 probes kept.\n",
      "-- check region:77 1:193735001-193765000_strand_+_gene_77, 59194 candidate probes\n",
      "--- 10009 probes passed check_dic selection.\n",
      "finish in 1.270s, 218 probes kept.\n",
      "-- check region:78 1:196235001-196265000_strand_+_gene_78, 59454 candidate probes\n",
      "--- 4442 probes passed check_dic selection.\n",
      "finish in 1.067s, 142 probes kept.\n",
      "-- check region:79 1:198735001-198765000_strand_+_gene_79, 59752 candidate probes\n",
      "--- 8681 probes passed check_dic selection.\n",
      "finish in 1.236s, 198 probes kept.\n",
      "-- check region:80 1:201235001-201265000_strand_+_gene_80, 51892 candidate probes\n",
      "--- 14788 probes passed check_dic selection.\n",
      "finish in 1.357s, 247 probes kept.\n",
      "-- check region:81 1:203735001-203765000_strand_+_gene_81, 56642 candidate probes\n",
      "--- 11938 probes passed check_dic selection.\n",
      "finish in 1.363s, 223 probes kept.\n",
      "-- check region:82 1:206235001-206265000_strand_+_gene_82, 59142 candidate probes\n",
      "--- 124 probes passed check_dic selection.\n",
      "finish in 0.958s, 8 probes kept.\n",
      "-- check region:83 1:208735001-208765000_strand_+_gene_83, 57958 candidate probes\n",
      "--- 14684 probes passed check_dic selection.\n",
      "finish in 1.442s, 259 probes kept.\n",
      "-- check region:84 1:211235001-211265000_strand_+_gene_84, 55774 candidate probes\n",
      "--- 9538 probes passed check_dic selection.\n",
      "finish in 1.217s, 191 probes kept.\n",
      "-- check region:85 1:213735001-213765000_strand_+_gene_85, 59034 candidate probes\n",
      "--- 19858 probes passed check_dic selection.\n",
      "finish in 1.657s, 313 probes kept.\n",
      "-- check region:86 1:216235001-216265000_strand_+_gene_86, 59444 candidate probes\n",
      "--- 10988 probes passed check_dic selection.\n",
      "finish in 1.316s, 237 probes kept.\n",
      "-- check region:87 1:218735001-218765000_strand_+_gene_87, 59202 candidate probes\n",
      "--- 11848 probes passed check_dic selection.\n",
      "finish in 1.368s, 236 probes kept.\n",
      "-- check region:88 1:221235001-221265000_strand_+_gene_88, 55744 candidate probes\n",
      "--- 9701 probes passed check_dic selection.\n",
      "finish in 1.227s, 204 probes kept.\n",
      "-- check region:89 1:223735001-223765000_strand_+_gene_89, 58376 candidate probes\n",
      "--- 21410 probes passed check_dic selection.\n",
      "finish in 1.720s, 335 probes kept.\n",
      "-- check region:90 1:226235001-226265000_strand_+_gene_90, 55972 candidate probes\n",
      "--- 4457 probes passed check_dic selection.\n",
      "finish in 1.037s, 118 probes kept.\n",
      "-- check region:91 1:228735001-228765000_strand_+_gene_91, 59636 candidate probes\n",
      "--- 22761 probes passed check_dic selection.\n",
      "finish in 1.803s, 363 probes kept.\n",
      "-- check region:92 1:231235001-231265000_strand_+_gene_92, 58162 candidate probes\n",
      "--- 15167 probes passed check_dic selection.\n",
      "finish in 1.481s, 271 probes kept.\n",
      "-- check region:93 1:233735001-233765000_strand_+_gene_93, 58296 candidate probes\n",
      "--- 11461 probes passed check_dic selection.\n",
      "finish in 1.318s, 212 probes kept.\n",
      "-- check region:94 1:236235001-236265000_strand_+_gene_94, 57698 candidate probes\n",
      "--- 8821 probes passed check_dic selection.\n",
      "finish in 1.217s, 178 probes kept.\n",
      "-- check region:95 1:238735001-238765000_strand_+_gene_95, 58698 candidate probes\n",
      "--- 9685 probes passed check_dic selection.\n",
      "finish in 1.260s, 214 probes kept.\n",
      "-- check region:96 1:241235001-241265000_strand_+_gene_96, 57548 candidate probes\n",
      "--- 14944 probes passed check_dic selection.\n",
      "finish in 1.448s, 258 probes kept.\n",
      "-- check region:97 1:243735001-243765000_strand_+_gene_97, 59166 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.920s, 0 probes kept.\n",
      "-- check region:98 1:246235001-246265000_strand_+_gene_98, 54048 candidate probes\n",
      "--- 7653 probes passed check_dic selection.\n",
      "finish in 1.120s, 168 probes kept.\n",
      "-- check region:99 2:1235001-1265000_strand_+_gene_99, 58542 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.926s, 0 probes kept.\n",
      "-- check region:100 2:3735001-3765000_strand_+_gene_100, 59230 candidate probes\n",
      "--- 22508 probes passed check_dic selection.\n",
      "finish in 1.763s, 344 probes kept.\n",
      "-- check region:101 2:6235001-6265000_strand_+_gene_101, 58343 candidate probes\n",
      "--- 15962 probes passed check_dic selection.\n",
      "finish in 1.491s, 286 probes kept.\n",
      "-- check region:102 2:8735001-8765000_strand_+_gene_102, 59048 candidate probes\n",
      "--- 11359 probes passed check_dic selection.\n",
      "finish in 1.335s, 240 probes kept.\n",
      "-- check region:103 2:11235001-11265000_strand_+_gene_103, 57902 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 8477 probes passed check_dic selection.\n",
      "finish in 1.215s, 205 probes kept.\n",
      "-- check region:104 2:13735001-13765000_strand_+_gene_104, 52880 candidate probes\n",
      "--- 3896 probes passed check_dic selection.\n",
      "finish in 0.969s, 119 probes kept.\n",
      "-- check region:105 2:16235001-16265000_strand_+_gene_105, 59376 candidate probes\n",
      "--- 23556 probes passed check_dic selection.\n",
      "finish in 1.818s, 352 probes kept.\n",
      "-- check region:106 2:18735001-18765000_strand_+_gene_106, 57758 candidate probes\n",
      "--- 13112 probes passed check_dic selection.\n",
      "finish in 1.377s, 244 probes kept.\n",
      "-- check region:107 2:21235001-21265000_strand_+_gene_107, 58946 candidate probes\n",
      "--- 19363 probes passed check_dic selection.\n",
      "finish in 1.624s, 305 probes kept.\n",
      "-- check region:108 2:23735001-23765000_strand_+_gene_108, 53196 candidate probes\n",
      "--- 9170 probes passed check_dic selection.\n",
      "finish in 1.173s, 213 probes kept.\n",
      "-- check region:109 2:26235001-26265000_strand_+_gene_109, 57444 candidate probes\n",
      "--- 10774 probes passed check_dic selection.\n",
      "finish in 1.303s, 225 probes kept.\n",
      "-- check region:110 2:28735001-28765000_strand_+_gene_110, 58032 candidate probes\n",
      "--- 12274 probes passed check_dic selection.\n",
      "finish in 1.354s, 230 probes kept.\n",
      "-- check region:111 2:31235001-31265000_strand_+_gene_111, 58774 candidate probes\n",
      "--- 25061 probes passed check_dic selection.\n",
      "finish in 1.865s, 358 probes kept.\n",
      "-- check region:112 2:33735001-33765000_strand_+_gene_112, 56124 candidate probes\n",
      "--- 7636 probes passed check_dic selection.\n",
      "finish in 1.158s, 167 probes kept.\n",
      "-- check region:113 2:36235001-36265000_strand_+_gene_113, 59548 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.937s, 0 probes kept.\n",
      "-- check region:114 2:38735001-38765000_strand_+_gene_114, 55546 candidate probes\n",
      "--- 11253 probes passed check_dic selection.\n",
      "finish in 1.294s, 215 probes kept.\n",
      "-- check region:115 2:41235001-41265000_strand_+_gene_115, 59450 candidate probes\n",
      "--- 11107 probes passed check_dic selection.\n",
      "finish in 1.319s, 224 probes kept.\n",
      "-- check region:116 2:43735001-43765000_strand_+_gene_116, 58272 candidate probes\n",
      "--- 11170 probes passed check_dic selection.\n",
      "finish in 1.319s, 228 probes kept.\n",
      "-- check region:117 2:46235001-46265000_strand_+_gene_117, 58614 candidate probes\n",
      "--- 16184 probes passed check_dic selection.\n",
      "finish in 1.524s, 276 probes kept.\n",
      "-- check region:118 2:48735001-48765000_strand_+_gene_118, 49048 candidate probes\n",
      "--- 12350 probes passed check_dic selection.\n",
      "finish in 1.227s, 216 probes kept.\n",
      "-- check region:119 2:51235001-51265000_strand_+_gene_119, 59480 candidate probes\n",
      "--- 10455 probes passed check_dic selection.\n",
      "finish in 1.291s, 221 probes kept.\n",
      "-- check region:120 2:53735001-53765000_strand_+_gene_120, 59252 candidate probes\n",
      "--- 8651 probes passed check_dic selection.\n",
      "finish in 1.245s, 200 probes kept.\n",
      "-- check region:121 2:56235001-56265000_strand_+_gene_121, 59064 candidate probes\n",
      "--- 16258 probes passed check_dic selection.\n",
      "finish in 1.524s, 295 probes kept.\n",
      "-- check region:122 2:58735001-58765000_strand_+_gene_122, 59346 candidate probes\n",
      "--- 15901 probes passed check_dic selection.\n",
      "finish in 1.515s, 270 probes kept.\n",
      "-- check region:123 2:61235001-61265000_strand_+_gene_123, 58406 candidate probes\n",
      "--- 7260 probes passed check_dic selection.\n",
      "finish in 1.178s, 182 probes kept.\n",
      "-- check region:124 2:63735001-63765000_strand_+_gene_124, 59256 candidate probes\n",
      "--- 13150 probes passed check_dic selection.\n",
      "finish in 1.411s, 235 probes kept.\n",
      "-- check region:125 2:66235001-66265000_strand_+_gene_125, 55576 candidate probes\n",
      "--- 14592 probes passed check_dic selection.\n",
      "finish in 1.411s, 251 probes kept.\n",
      "-- check region:126 2:68735001-68765000_strand_+_gene_126, 59500 candidate probes\n",
      "--- 16537 probes passed check_dic selection.\n",
      "finish in 1.556s, 291 probes kept.\n",
      "-- check region:127 2:71235001-71265000_strand_+_gene_127, 55764 candidate probes\n",
      "--- 12597 probes passed check_dic selection.\n",
      "finish in 1.355s, 222 probes kept.\n",
      "-- check region:128 2:73735001-73765000_strand_+_gene_128, 57684 candidate probes\n",
      "--- 18209 probes passed check_dic selection.\n",
      "finish in 1.582s, 275 probes kept.\n",
      "-- check region:129 2:76235001-76265000_strand_+_gene_129, 59578 candidate probes\n",
      "--- 10650 probes passed check_dic selection.\n",
      "finish in 1.310s, 216 probes kept.\n",
      "-- check region:130 2:78735001-78765000_strand_+_gene_130, 57898 candidate probes\n",
      "--- 6208 probes passed check_dic selection.\n",
      "finish in 1.130s, 162 probes kept.\n",
      "-- check region:131 2:81235001-81265000_strand_+_gene_131, 59160 candidate probes\n",
      "--- 8527 probes passed check_dic selection.\n",
      "finish in 1.214s, 190 probes kept.\n",
      "-- check region:132 2:83735001-83765000_strand_+_gene_132, 59078 candidate probes\n",
      "--- 9011 probes passed check_dic selection.\n",
      "finish in 1.243s, 192 probes kept.\n",
      "-- check region:133 2:86235001-86265000_strand_+_gene_133, 57158 candidate probes\n",
      "--- 19043 probes passed check_dic selection.\n",
      "finish in 1.612s, 312 probes kept.\n",
      "-- check region:134 2:88735001-88765000_strand_+_gene_134, 58082 candidate probes\n",
      "--- 13769 probes passed check_dic selection.\n",
      "finish in 1.431s, 237 probes kept.\n",
      "-- check region:135 2:91235001-91265000_strand_+_gene_135, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:136 2:93735001-93765000_strand_+_gene_136, 19752 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.319s, 0 probes kept.\n",
      "-- check region:137 2:96235001-96265000_strand_+_gene_137, 57980 candidate probes\n",
      "--- 19278 probes passed check_dic selection.\n",
      "finish in 1.651s, 315 probes kept.\n",
      "-- check region:138 2:98735001-98765000_strand_+_gene_138, 58622 candidate probes\n",
      "--- 16089 probes passed check_dic selection.\n",
      "finish in 1.525s, 285 probes kept.\n",
      "-- check region:139 2:101235001-101265000_strand_+_gene_139, 56614 candidate probes\n",
      "--- 8754 probes passed check_dic selection.\n",
      "finish in 1.217s, 174 probes kept.\n",
      "-- check region:140 2:103735001-103765000_strand_+_gene_140, 59398 candidate probes\n",
      "--- 14226 probes passed check_dic selection.\n",
      "finish in 1.443s, 261 probes kept.\n",
      "-- check region:141 2:106235001-106265000_strand_+_gene_141, 57460 candidate probes\n",
      "--- 3231 probes passed check_dic selection.\n",
      "finish in 1.043s, 74 probes kept.\n",
      "-- check region:142 2:108735001-108765000_strand_+_gene_142, 57586 candidate probes\n",
      "--- 1138 probes passed check_dic selection.\n",
      "finish in 0.942s, 39 probes kept.\n",
      "-- check region:143 2:111235001-111265000_strand_+_gene_143, 58894 candidate probes\n",
      "--- 10479 probes passed check_dic selection.\n",
      "finish in 1.317s, 186 probes kept.\n",
      "-- check region:144 2:113735001-113765000_strand_+_gene_144, 57568 candidate probes\n",
      "--- 7818 probes passed check_dic selection.\n",
      "finish in 1.182s, 187 probes kept.\n",
      "-- check region:145 2:116235001-116265000_strand_+_gene_145, 58094 candidate probes\n",
      "--- 7994 probes passed check_dic selection.\n",
      "finish in 1.196s, 178 probes kept.\n",
      "-- check region:146 2:118735001-118765000_strand_+_gene_146, 59292 candidate probes\n",
      "--- 18064 probes passed check_dic selection.\n",
      "finish in 1.597s, 294 probes kept.\n",
      "-- check region:147 2:121235001-121265000_strand_+_gene_147, 58632 candidate probes\n",
      "--- 24527 probes passed check_dic selection.\n",
      "finish in 1.859s, 355 probes kept.\n",
      "-- check region:148 2:123735001-123765000_strand_+_gene_148, 57596 candidate probes\n",
      "--- 8024 probes passed check_dic selection.\n",
      "finish in 1.168s, 193 probes kept.\n",
      "-- check region:149 2:126235001-126265000_strand_+_gene_149, 59054 candidate probes\n",
      "--- 18165 probes passed check_dic selection.\n",
      "finish in 1.598s, 309 probes kept.\n",
      "-- check region:150 2:128735001-128765000_strand_+_gene_150, 58502 candidate probes\n",
      "--- 15266 probes passed check_dic selection.\n",
      "finish in 1.489s, 240 probes kept.\n",
      "-- check region:151 2:131235001-131265000_strand_+_gene_151, 58856 candidate probes\n",
      "--- 34 probes passed check_dic selection.\n",
      "finish in 0.925s, 3 probes kept.\n",
      "-- check region:152 2:133735001-133765000_strand_+_gene_152, 58456 candidate probes\n",
      "--- 14344 probes passed check_dic selection.\n",
      "finish in 1.443s, 248 probes kept.\n",
      "-- check region:153 2:136235001-136265000_strand_+_gene_153, 57306 candidate probes\n",
      "--- 19389 probes passed check_dic selection.\n",
      "finish in 1.639s, 307 probes kept.\n",
      "-- check region:154 2:138735001-138765000_strand_+_gene_154, 59390 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 16035 probes passed check_dic selection.\n",
      "finish in 1.518s, 283 probes kept.\n",
      "-- check region:155 2:141235001-141265000_strand_+_gene_155, 59734 candidate probes\n",
      "--- 11963 probes passed check_dic selection.\n",
      "finish in 1.351s, 252 probes kept.\n",
      "-- check region:156 2:143735001-143765000_strand_+_gene_156, 59532 candidate probes\n",
      "--- 17151 probes passed check_dic selection.\n",
      "finish in 1.552s, 292 probes kept.\n",
      "-- check region:157 2:146235001-146265000_strand_+_gene_157, 59104 candidate probes\n",
      "--- 11906 probes passed check_dic selection.\n",
      "finish in 1.345s, 255 probes kept.\n",
      "-- check region:158 2:148735001-148765000_strand_+_gene_158, 59332 candidate probes\n",
      "--- 10875 probes passed check_dic selection.\n",
      "finish in 1.304s, 227 probes kept.\n",
      "-- check region:159 2:151235001-151265000_strand_+_gene_159, 57610 candidate probes\n",
      "--- 12441 probes passed check_dic selection.\n",
      "finish in 1.362s, 231 probes kept.\n",
      "-- check region:160 2:153735001-153765000_strand_+_gene_160, 59614 candidate probes\n",
      "--- 5641 probes passed check_dic selection.\n",
      "finish in 1.124s, 136 probes kept.\n",
      "-- check region:161 2:156235001-156265000_strand_+_gene_161, 50744 candidate probes\n",
      "--- 5130 probes passed check_dic selection.\n",
      "finish in 0.973s, 134 probes kept.\n",
      "-- check region:162 2:158735001-158765000_strand_+_gene_162, 59256 candidate probes\n",
      "--- 22055 probes passed check_dic selection.\n",
      "finish in 1.742s, 338 probes kept.\n",
      "-- check region:163 2:161235001-161265000_strand_+_gene_163, 59112 candidate probes\n",
      "--- 11385 probes passed check_dic selection.\n",
      "finish in 1.331s, 234 probes kept.\n",
      "-- check region:164 2:163735001-163765000_strand_+_gene_164, 59834 candidate probes\n",
      "--- 15486 probes passed check_dic selection.\n",
      "finish in 1.487s, 303 probes kept.\n",
      "-- check region:165 2:166235001-166265000_strand_+_gene_165, 59524 candidate probes\n",
      "--- 10991 probes passed check_dic selection.\n",
      "finish in 1.314s, 244 probes kept.\n",
      "-- check region:166 2:168735001-168765000_strand_+_gene_166, 58398 candidate probes\n",
      "--- 17450 probes passed check_dic selection.\n",
      "finish in 1.552s, 304 probes kept.\n",
      "-- check region:167 2:171235001-171265000_strand_+_gene_167, 57682 candidate probes\n",
      "--- 11079 probes passed check_dic selection.\n",
      "finish in 1.318s, 217 probes kept.\n",
      "-- check region:168 2:173735001-173765000_strand_+_gene_168, 58986 candidate probes\n",
      "--- 18991 probes passed check_dic selection.\n",
      "finish in 1.624s, 301 probes kept.\n",
      "-- check region:169 2:176235001-176265000_strand_+_gene_169, 58332 candidate probes\n",
      "--- 12767 probes passed check_dic selection.\n",
      "finish in 1.375s, 238 probes kept.\n",
      "-- check region:170 2:178735001-178765000_strand_+_gene_170, 59638 candidate probes\n",
      "--- 11977 probes passed check_dic selection.\n",
      "finish in 1.383s, 303 probes kept.\n",
      "-- check region:171 2:181235001-181265000_strand_+_gene_171, 59462 candidate probes\n",
      "--- 14568 probes passed check_dic selection.\n",
      "finish in 1.445s, 272 probes kept.\n",
      "-- check region:172 2:183735001-183765000_strand_+_gene_172, 56990 candidate probes\n",
      "--- 10101 probes passed check_dic selection.\n",
      "finish in 1.240s, 200 probes kept.\n",
      "-- check region:173 2:186235001-186265000_strand_+_gene_173, 55870 candidate probes\n",
      "--- 7584 probes passed check_dic selection.\n",
      "finish in 1.140s, 173 probes kept.\n",
      "-- check region:174 2:188735001-188765000_strand_+_gene_174, 59046 candidate probes\n",
      "--- 8218 probes passed check_dic selection.\n",
      "finish in 1.203s, 198 probes kept.\n",
      "-- check region:175 2:191235001-191265000_strand_+_gene_175, 59416 candidate probes\n",
      "--- 18199 probes passed check_dic selection.\n",
      "finish in 1.595s, 317 probes kept.\n",
      "-- check region:176 2:193735001-193765000_strand_+_gene_176, 59382 candidate probes\n",
      "--- 5916 probes passed check_dic selection.\n",
      "finish in 1.124s, 152 probes kept.\n",
      "-- check region:177 2:196235001-196265000_strand_+_gene_177, 58500 candidate probes\n",
      "--- 14625 probes passed check_dic selection.\n",
      "finish in 1.453s, 279 probes kept.\n",
      "-- check region:178 2:198735001-198765000_strand_+_gene_178, 59618 candidate probes\n",
      "--- 15309 probes passed check_dic selection.\n",
      "finish in 1.479s, 280 probes kept.\n",
      "-- check region:179 2:201235001-201265000_strand_+_gene_179, 56916 candidate probes\n",
      "--- 12584 probes passed check_dic selection.\n",
      "finish in 1.350s, 230 probes kept.\n",
      "-- check region:180 2:203735001-203765000_strand_+_gene_180, 57976 candidate probes\n",
      "--- 11272 probes passed check_dic selection.\n",
      "finish in 1.335s, 226 probes kept.\n",
      "-- check region:181 2:206235001-206265000_strand_+_gene_181, 56772 candidate probes\n",
      "--- 14550 probes passed check_dic selection.\n",
      "finish in 1.427s, 248 probes kept.\n",
      "-- check region:182 2:208735001-208765000_strand_+_gene_182, 47466 candidate probes\n",
      "--- 4136 probes passed check_dic selection.\n",
      "finish in 0.889s, 93 probes kept.\n",
      "-- check region:183 2:211235001-211265000_strand_+_gene_183, 55966 candidate probes\n",
      "--- 10636 probes passed check_dic selection.\n",
      "finish in 1.262s, 204 probes kept.\n",
      "-- check region:184 2:213735001-213765000_strand_+_gene_184, 57980 candidate probes\n",
      "--- 8299 probes passed check_dic selection.\n",
      "finish in 1.191s, 173 probes kept.\n",
      "-- check region:185 2:216235001-216265000_strand_+_gene_185, 58664 candidate probes\n",
      "--- 19004 probes passed check_dic selection.\n",
      "finish in 1.634s, 313 probes kept.\n",
      "-- check region:186 2:218735001-218765000_strand_+_gene_186, 58428 candidate probes\n",
      "--- 16735 probes passed check_dic selection.\n",
      "finish in 1.558s, 301 probes kept.\n",
      "-- check region:187 2:221235001-221265000_strand_+_gene_187, 58854 candidate probes\n",
      "--- 12326 probes passed check_dic selection.\n",
      "finish in 1.361s, 239 probes kept.\n",
      "-- check region:188 2:223735001-223765000_strand_+_gene_188, 57874 candidate probes\n",
      "--- 13246 probes passed check_dic selection.\n",
      "finish in 1.400s, 233 probes kept.\n",
      "-- check region:189 2:226235001-226265000_strand_+_gene_189, 57792 candidate probes\n",
      "--- 8688 probes passed check_dic selection.\n",
      "finish in 1.204s, 191 probes kept.\n",
      "-- check region:190 2:228735001-228765000_strand_+_gene_190, 47512 candidate probes\n",
      "--- 7527 probes passed check_dic selection.\n",
      "finish in 1.018s, 150 probes kept.\n",
      "-- check region:191 2:231235001-231265000_strand_+_gene_191, 56606 candidate probes\n",
      "--- 17496 probes passed check_dic selection.\n",
      "finish in 1.536s, 274 probes kept.\n",
      "-- check region:192 2:233735001-233765000_strand_+_gene_192, 56054 candidate probes\n",
      "--- 14438 probes passed check_dic selection.\n",
      "finish in 1.428s, 270 probes kept.\n",
      "-- check region:193 2:236235001-236265000_strand_+_gene_193, 59022 candidate probes\n",
      "--- 22615 probes passed check_dic selection.\n",
      "finish in 1.775s, 357 probes kept.\n",
      "-- check region:194 2:238735001-238765000_strand_+_gene_194, 58294 candidate probes\n",
      "--- 18206 probes passed check_dic selection.\n",
      "finish in 1.592s, 297 probes kept.\n",
      "-- check region:195 3:1235001-1265000_strand_+_gene_195, 57398 candidate probes\n",
      "--- 10304 probes passed check_dic selection.\n",
      "finish in 1.268s, 199 probes kept.\n",
      "-- check region:196 3:3735001-3765000_strand_+_gene_196, 59590 candidate probes\n",
      "--- 18284 probes passed check_dic selection.\n",
      "finish in 1.616s, 317 probes kept.\n",
      "-- check region:197 3:6235001-6265000_strand_+_gene_197, 59196 candidate probes\n",
      "--- 12831 probes passed check_dic selection.\n",
      "finish in 1.394s, 251 probes kept.\n",
      "-- check region:198 3:8735001-8765000_strand_+_gene_198, 59624 candidate probes\n",
      "--- 24162 probes passed check_dic selection.\n",
      "finish in 1.846s, 379 probes kept.\n",
      "-- check region:199 3:11235001-11265000_strand_+_gene_199, 58214 candidate probes\n",
      "--- 12341 probes passed check_dic selection.\n",
      "finish in 1.388s, 275 probes kept.\n",
      "-- check region:200 3:13735001-13765000_strand_+_gene_200, 58840 candidate probes\n",
      "--- 14436 probes passed check_dic selection.\n",
      "finish in 1.458s, 229 probes kept.\n",
      "-- check region:201 3:16235001-16265000_strand_+_gene_201, 59164 candidate probes\n",
      "--- 22549 probes passed check_dic selection.\n",
      "finish in 1.782s, 361 probes kept.\n",
      "-- check region:202 3:18735001-18765000_strand_+_gene_202, 59750 candidate probes\n",
      "--- 16468 probes passed check_dic selection.\n",
      "finish in 1.543s, 331 probes kept.\n",
      "-- check region:203 3:21235001-21265000_strand_+_gene_203, 50848 candidate probes\n",
      "--- 9906 probes passed check_dic selection.\n",
      "finish in 1.145s, 190 probes kept.\n",
      "-- check region:204 3:23735001-23765000_strand_+_gene_204, 58004 candidate probes\n",
      "--- 17337 probes passed check_dic selection.\n",
      "finish in 1.553s, 265 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:205 3:26235001-26265000_strand_+_gene_205, 53482 candidate probes\n",
      "--- 12872 probes passed check_dic selection.\n",
      "finish in 1.297s, 241 probes kept.\n",
      "-- check region:206 3:28735001-28765000_strand_+_gene_206, 59384 candidate probes\n",
      "--- 14205 probes passed check_dic selection.\n",
      "finish in 1.451s, 262 probes kept.\n",
      "-- check region:207 3:31235001-31265000_strand_+_gene_207, 59732 candidate probes\n",
      "--- 18090 probes passed check_dic selection.\n",
      "finish in 1.596s, 324 probes kept.\n",
      "-- check region:208 3:33735001-33765000_strand_+_gene_208, 56604 candidate probes\n",
      "--- 8194 probes passed check_dic selection.\n",
      "finish in 1.188s, 174 probes kept.\n",
      "-- check region:209 3:36235001-36265000_strand_+_gene_209, 55888 candidate probes\n",
      "--- 8315 probes passed check_dic selection.\n",
      "finish in 1.174s, 180 probes kept.\n",
      "-- check region:210 3:38735001-38765000_strand_+_gene_210, 59248 candidate probes\n",
      "--- 23150 probes passed check_dic selection.\n",
      "finish in 1.814s, 365 probes kept.\n",
      "-- check region:211 3:41235001-41265000_strand_+_gene_211, 59596 candidate probes\n",
      "--- 21119 probes passed check_dic selection.\n",
      "finish in 1.730s, 349 probes kept.\n",
      "-- check region:212 3:43735001-43765000_strand_+_gene_212, 58990 candidate probes\n",
      "--- 21172 probes passed check_dic selection.\n",
      "finish in 1.707s, 330 probes kept.\n",
      "-- check region:213 3:46235001-46265000_strand_+_gene_213, 56570 candidate probes\n",
      "--- 10030 probes passed check_dic selection.\n",
      "finish in 1.265s, 200 probes kept.\n",
      "-- check region:214 3:48735001-48765000_strand_+_gene_214, 56314 candidate probes\n",
      "--- 11716 probes passed check_dic selection.\n",
      "finish in 1.336s, 219 probes kept.\n",
      "-- check region:215 3:51235001-51265000_strand_+_gene_215, 52006 candidate probes\n",
      "--- 7476 probes passed check_dic selection.\n",
      "finish in 1.088s, 142 probes kept.\n",
      "-- check region:216 3:53735001-53765000_strand_+_gene_216, 59534 candidate probes\n",
      "--- 29186 probes passed check_dic selection.\n",
      "finish in 2.036s, 408 probes kept.\n",
      "-- check region:217 3:56235001-56265000_strand_+_gene_217, 58812 candidate probes\n",
      "--- 15237 probes passed check_dic selection.\n",
      "finish in 1.480s, 278 probes kept.\n",
      "-- check region:218 3:58735001-58765000_strand_+_gene_218, 53942 candidate probes\n",
      "--- 14116 probes passed check_dic selection.\n",
      "finish in 1.369s, 252 probes kept.\n",
      "-- check region:219 3:61235001-61265000_strand_+_gene_219, 57882 candidate probes\n",
      "--- 14729 probes passed check_dic selection.\n",
      "finish in 1.445s, 258 probes kept.\n",
      "-- check region:220 3:63735001-63765000_strand_+_gene_220, 52330 candidate probes\n",
      "--- 9463 probes passed check_dic selection.\n",
      "finish in 1.160s, 214 probes kept.\n",
      "-- check region:221 3:66235001-66265000_strand_+_gene_221, 58814 candidate probes\n",
      "--- 14483 probes passed check_dic selection.\n",
      "finish in 1.443s, 256 probes kept.\n",
      "-- check region:222 3:68735001-68765000_strand_+_gene_222, 59486 candidate probes\n",
      "--- 16427 probes passed check_dic selection.\n",
      "finish in 1.535s, 292 probes kept.\n",
      "-- check region:223 3:71235001-71265000_strand_+_gene_223, 58364 candidate probes\n",
      "--- 20247 probes passed check_dic selection.\n",
      "finish in 1.671s, 318 probes kept.\n",
      "-- check region:224 3:73735001-73765000_strand_+_gene_224, 59628 candidate probes\n",
      "--- 20134 probes passed check_dic selection.\n",
      "finish in 1.681s, 322 probes kept.\n",
      "-- check region:225 3:76235001-76265000_strand_+_gene_225, 57928 candidate probes\n",
      "--- 11379 probes passed check_dic selection.\n",
      "finish in 1.304s, 234 probes kept.\n",
      "-- check region:226 3:78735001-78765000_strand_+_gene_226, 59290 candidate probes\n",
      "--- 10209 probes passed check_dic selection.\n",
      "finish in 1.274s, 220 probes kept.\n",
      "-- check region:227 3:81235001-81265000_strand_+_gene_227, 59290 candidate probes\n",
      "--- 12039 probes passed check_dic selection.\n",
      "finish in 1.348s, 242 probes kept.\n",
      "-- check region:228 3:83735001-83765000_strand_+_gene_228, 57548 candidate probes\n",
      "--- 6074 probes passed check_dic selection.\n",
      "finish in 1.104s, 148 probes kept.\n",
      "-- check region:229 3:86235001-86265000_strand_+_gene_229, 58872 candidate probes\n",
      "--- 8771 probes passed check_dic selection.\n",
      "finish in 1.214s, 197 probes kept.\n",
      "-- check region:230 3:88735001-88765000_strand_+_gene_230, 56192 candidate probes\n",
      "--- 8239 probes passed check_dic selection.\n",
      "finish in 1.183s, 166 probes kept.\n",
      "-- check region:231 3:91235001-91265000_strand_+_gene_231, 18710 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.296s, 0 probes kept.\n",
      "-- check region:232 3:93735001-93765000_strand_+_gene_232, 53752 candidate probes\n",
      "--- 294 probes passed check_dic selection.\n",
      "finish in 0.856s, 23 probes kept.\n",
      "-- check region:233 3:96235001-96265000_strand_+_gene_233, 59240 candidate probes\n",
      "--- 9866 probes passed check_dic selection.\n",
      "finish in 1.300s, 196 probes kept.\n",
      "-- check region:234 3:98735001-98765000_strand_+_gene_234, 58976 candidate probes\n",
      "--- 12682 probes passed check_dic selection.\n",
      "finish in 1.389s, 255 probes kept.\n",
      "-- check region:235 3:101235001-101265000_strand_+_gene_235, 58610 candidate probes\n",
      "--- 11582 probes passed check_dic selection.\n",
      "finish in 1.348s, 233 probes kept.\n",
      "-- check region:236 3:103735001-103765000_strand_+_gene_236, 57596 candidate probes\n",
      "--- 6626 probes passed check_dic selection.\n",
      "finish in 1.130s, 182 probes kept.\n",
      "-- check region:237 3:106235001-106265000_strand_+_gene_237, 58878 candidate probes\n",
      "--- 7412 probes passed check_dic selection.\n",
      "finish in 1.183s, 179 probes kept.\n",
      "-- check region:238 3:108735001-108765000_strand_+_gene_238, 48358 candidate probes\n",
      "--- 15053 probes passed check_dic selection.\n",
      "finish in 1.328s, 251 probes kept.\n",
      "-- check region:239 3:111235001-111265000_strand_+_gene_239, 57338 candidate probes\n",
      "--- 11428 probes passed check_dic selection.\n",
      "finish in 1.311s, 231 probes kept.\n",
      "-- check region:240 3:113735001-113765000_strand_+_gene_240, 58244 candidate probes\n",
      "--- 12043 probes passed check_dic selection.\n",
      "finish in 1.338s, 234 probes kept.\n",
      "-- check region:241 3:116235001-116265000_strand_+_gene_241, 59358 candidate probes\n",
      "--- 14479 probes passed check_dic selection.\n",
      "finish in 1.443s, 272 probes kept.\n",
      "-- check region:242 3:118735001-118765000_strand_+_gene_242, 58754 candidate probes\n",
      "--- 16899 probes passed check_dic selection.\n",
      "finish in 1.535s, 288 probes kept.\n",
      "-- check region:243 3:121235001-121265000_strand_+_gene_243, 58780 candidate probes\n",
      "--- 6093 probes passed check_dic selection.\n",
      "finish in 1.132s, 165 probes kept.\n",
      "-- check region:244 3:123735001-123765000_strand_+_gene_244, 59342 candidate probes\n",
      "--- 19997 probes passed check_dic selection.\n",
      "finish in 1.677s, 333 probes kept.\n",
      "-- check region:245 3:126235001-126265000_strand_+_gene_245, 59608 candidate probes\n",
      "--- 13194 probes passed check_dic selection.\n",
      "finish in 1.429s, 238 probes kept.\n",
      "-- check region:246 3:128735001-128765000_strand_+_gene_246, 56974 candidate probes\n",
      "--- 10794 probes passed check_dic selection.\n",
      "finish in 1.304s, 205 probes kept.\n",
      "-- check region:247 3:131235001-131265000_strand_+_gene_247, 57664 candidate probes\n",
      "--- 13218 probes passed check_dic selection.\n",
      "finish in 1.385s, 234 probes kept.\n",
      "-- check region:248 3:133735001-133765000_strand_+_gene_248, 56368 candidate probes\n",
      "--- 16119 probes passed check_dic selection.\n",
      "finish in 1.502s, 277 probes kept.\n",
      "-- check region:249 3:136235001-136265000_strand_+_gene_249, 57468 candidate probes\n",
      "--- 13977 probes passed check_dic selection.\n",
      "finish in 1.426s, 242 probes kept.\n",
      "-- check region:250 3:138735001-138765000_strand_+_gene_250, 57170 candidate probes\n",
      "--- 6813 probes passed check_dic selection.\n",
      "finish in 1.146s, 172 probes kept.\n",
      "-- check region:251 3:141235001-141265000_strand_+_gene_251, 59574 candidate probes\n",
      "--- 24158 probes passed check_dic selection.\n",
      "finish in 1.838s, 357 probes kept.\n",
      "-- check region:252 3:143735001-143765000_strand_+_gene_252, 57258 candidate probes\n",
      "--- 16249 probes passed check_dic selection.\n",
      "finish in 1.492s, 287 probes kept.\n",
      "-- check region:253 3:146235001-146265000_strand_+_gene_253, 59396 candidate probes\n",
      "--- 7039 probes passed check_dic selection.\n",
      "finish in 1.174s, 188 probes kept.\n",
      "-- check region:254 3:148735001-148765000_strand_+_gene_254, 58592 candidate probes\n",
      "--- 15835 probes passed check_dic selection.\n",
      "finish in 1.503s, 282 probes kept.\n",
      "-- check region:255 3:151235001-151265000_strand_+_gene_255, 51454 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 12065 probes passed check_dic selection.\n",
      "finish in 1.245s, 225 probes kept.\n",
      "-- check region:256 3:153735001-153765000_strand_+_gene_256, 58366 candidate probes\n",
      "--- 11123 probes passed check_dic selection.\n",
      "finish in 1.316s, 218 probes kept.\n",
      "-- check region:257 3:156235001-156265000_strand_+_gene_257, 59370 candidate probes\n",
      "--- 17780 probes passed check_dic selection.\n",
      "finish in 1.580s, 288 probes kept.\n",
      "-- check region:258 3:158735001-158765000_strand_+_gene_258, 57960 candidate probes\n",
      "--- 17467 probes passed check_dic selection.\n",
      "finish in 1.574s, 314 probes kept.\n",
      "-- check region:259 3:161235001-161265000_strand_+_gene_259, 59342 candidate probes\n",
      "--- 6225 probes passed check_dic selection.\n",
      "finish in 1.142s, 160 probes kept.\n",
      "-- check region:260 3:163735001-163765000_strand_+_gene_260, 56448 candidate probes\n",
      "--- 7752 probes passed check_dic selection.\n",
      "finish in 1.147s, 162 probes kept.\n",
      "-- check region:261 3:166235001-166265000_strand_+_gene_261, 59188 candidate probes\n",
      "--- 6472 probes passed check_dic selection.\n",
      "finish in 1.140s, 159 probes kept.\n",
      "-- check region:262 3:168735001-168765000_strand_+_gene_262, 55842 candidate probes\n",
      "--- 8709 probes passed check_dic selection.\n",
      "finish in 1.179s, 179 probes kept.\n",
      "-- check region:263 3:171235001-171265000_strand_+_gene_263, 58874 candidate probes\n",
      "--- 18708 probes passed check_dic selection.\n",
      "finish in 1.604s, 324 probes kept.\n",
      "-- check region:264 3:173735001-173765000_strand_+_gene_264, 59812 candidate probes\n",
      "--- 12823 probes passed check_dic selection.\n",
      "finish in 1.391s, 259 probes kept.\n",
      "-- check region:265 3:176235001-176265000_strand_+_gene_265, 59200 candidate probes\n",
      "--- 9727 probes passed check_dic selection.\n",
      "finish in 1.268s, 220 probes kept.\n",
      "-- check region:266 3:178735001-178765000_strand_+_gene_266, 57116 candidate probes\n",
      "--- 13646 probes passed check_dic selection.\n",
      "finish in 1.392s, 251 probes kept.\n",
      "-- check region:267 3:181235001-181265000_strand_+_gene_267, 58802 candidate probes\n",
      "--- 13988 probes passed check_dic selection.\n",
      "finish in 1.424s, 263 probes kept.\n",
      "-- check region:268 3:183735001-183765000_strand_+_gene_268, 54334 candidate probes\n",
      "--- 9242 probes passed check_dic selection.\n",
      "finish in 1.193s, 175 probes kept.\n",
      "-- check region:269 3:186235001-186265000_strand_+_gene_269, 58764 candidate probes\n",
      "--- 24235 probes passed check_dic selection.\n",
      "finish in 1.828s, 351 probes kept.\n",
      "-- check region:270 3:188735001-188765000_strand_+_gene_270, 59176 candidate probes\n",
      "--- 17886 probes passed check_dic selection.\n",
      "finish in 1.572s, 313 probes kept.\n",
      "-- check region:271 3:191235001-191265000_strand_+_gene_271, 59214 candidate probes\n",
      "--- 9298 probes passed check_dic selection.\n",
      "finish in 1.256s, 220 probes kept.\n",
      "-- check region:272 3:193735001-193765000_strand_+_gene_272, 58370 candidate probes\n",
      "--- 12730 probes passed check_dic selection.\n",
      "finish in 1.382s, 232 probes kept.\n",
      "-- check region:273 3:196235001-196265000_strand_+_gene_273, 55152 candidate probes\n",
      "--- 11823 probes passed check_dic selection.\n",
      "finish in 1.327s, 231 probes kept.\n",
      "-- check region:274 4:1235001-1265000_strand_+_gene_274, 58124 candidate probes\n",
      "--- 20310 probes passed check_dic selection.\n",
      "finish in 1.694s, 330 probes kept.\n",
      "-- check region:275 4:3735001-3765000_strand_+_gene_275, 59168 candidate probes\n",
      "--- 25816 probes passed check_dic selection.\n",
      "finish in 1.918s, 390 probes kept.\n",
      "-- check region:276 4:6235001-6265000_strand_+_gene_276, 58618 candidate probes\n",
      "--- 21027 probes passed check_dic selection.\n",
      "finish in 1.721s, 325 probes kept.\n",
      "-- check region:277 4:8735001-8765000_strand_+_gene_277, 59046 candidate probes\n",
      "--- 20298 probes passed check_dic selection.\n",
      "finish in 1.696s, 313 probes kept.\n",
      "-- check region:278 4:11235001-11265000_strand_+_gene_278, 51360 candidate probes\n",
      "--- 11205 probes passed check_dic selection.\n",
      "finish in 1.215s, 224 probes kept.\n",
      "-- check region:279 4:13735001-13765000_strand_+_gene_279, 57268 candidate probes\n",
      "--- 11460 probes passed check_dic selection.\n",
      "finish in 1.314s, 210 probes kept.\n",
      "-- check region:280 4:16235001-16265000_strand_+_gene_280, 59292 candidate probes\n",
      "--- 16471 probes passed check_dic selection.\n",
      "finish in 1.539s, 309 probes kept.\n",
      "-- check region:281 4:18735001-18765000_strand_+_gene_281, 59206 candidate probes\n",
      "--- 12124 probes passed check_dic selection.\n",
      "finish in 1.359s, 239 probes kept.\n",
      "-- check region:282 4:21235001-21265000_strand_+_gene_282, 55818 candidate probes\n",
      "--- 14508 probes passed check_dic selection.\n",
      "finish in 1.400s, 262 probes kept.\n",
      "-- check region:283 4:23735001-23765000_strand_+_gene_283, 59586 candidate probes\n",
      "--- 16401 probes passed check_dic selection.\n",
      "finish in 1.537s, 293 probes kept.\n",
      "-- check region:284 4:26235001-26265000_strand_+_gene_284, 57248 candidate probes\n",
      "--- 10912 probes passed check_dic selection.\n",
      "finish in 1.300s, 215 probes kept.\n",
      "-- check region:285 4:28735001-28765000_strand_+_gene_285, 58894 candidate probes\n",
      "--- 10900 probes passed check_dic selection.\n",
      "finish in 1.297s, 234 probes kept.\n",
      "-- check region:286 4:31235001-31265000_strand_+_gene_286, 59308 candidate probes\n",
      "--- 8805 probes passed check_dic selection.\n",
      "finish in 1.247s, 187 probes kept.\n",
      "-- check region:287 4:33735001-33765000_strand_+_gene_287, 58596 candidate probes\n",
      "--- 8397 probes passed check_dic selection.\n",
      "finish in 1.212s, 194 probes kept.\n",
      "-- check region:288 4:36235001-36265000_strand_+_gene_288, 59066 candidate probes\n",
      "--- 12385 probes passed check_dic selection.\n",
      "finish in 1.365s, 249 probes kept.\n",
      "-- check region:289 4:38735001-38765000_strand_+_gene_289, 58170 candidate probes\n",
      "--- 15084 probes passed check_dic selection.\n",
      "finish in 1.529s, 250 probes kept.\n",
      "-- check region:290 4:41235001-41265000_strand_+_gene_290, 57280 candidate probes\n",
      "--- 12929 probes passed check_dic selection.\n",
      "finish in 1.370s, 253 probes kept.\n",
      "-- check region:291 4:43735001-43765000_strand_+_gene_291, 56116 candidate probes\n",
      "--- 8611 probes passed check_dic selection.\n",
      "finish in 1.212s, 195 probes kept.\n",
      "-- check region:292 4:46235001-46265000_strand_+_gene_292, 58820 candidate probes\n",
      "--- 8920 probes passed check_dic selection.\n",
      "finish in 1.226s, 226 probes kept.\n",
      "-- check region:293 4:48735001-48765000_strand_+_gene_293, 54528 candidate probes\n",
      "--- 7654 probes passed check_dic selection.\n",
      "finish in 1.136s, 157 probes kept.\n",
      "-- check region:294 4:51235001-51265000_strand_+_gene_294, 19210 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.304s, 0 probes kept.\n",
      "-- check region:295 4:53735001-53765000_strand_+_gene_295, 55208 candidate probes\n",
      "--- 11082 probes passed check_dic selection.\n",
      "finish in 1.276s, 209 probes kept.\n",
      "-- check region:296 4:56235001-56265000_strand_+_gene_296, 57570 candidate probes\n",
      "--- 16329 probes passed check_dic selection.\n",
      "finish in 1.511s, 274 probes kept.\n",
      "-- check region:297 4:58735001-58765000_strand_+_gene_297, 58324 candidate probes\n",
      "--- 8317 probes passed check_dic selection.\n",
      "finish in 1.196s, 194 probes kept.\n",
      "-- check region:298 4:61235001-61265000_strand_+_gene_298, 59286 candidate probes\n",
      "--- 13121 probes passed check_dic selection.\n",
      "finish in 1.388s, 253 probes kept.\n",
      "-- check region:299 4:63735001-63765000_strand_+_gene_299, 57076 candidate probes\n",
      "--- 4696 probes passed check_dic selection.\n",
      "finish in 1.038s, 141 probes kept.\n",
      "-- check region:300 4:66235001-66265000_strand_+_gene_300, 52848 candidate probes\n",
      "--- 6661 probes passed check_dic selection.\n",
      "finish in 1.069s, 151 probes kept.\n",
      "-- check region:301 4:68735001-68765000_strand_+_gene_301, 59446 candidate probes\n",
      "--- 550 probes passed check_dic selection.\n",
      "finish in 0.961s, 25 probes kept.\n",
      "-- check region:302 4:71235001-71265000_strand_+_gene_302, 59776 candidate probes\n",
      "--- 17706 probes passed check_dic selection.\n",
      "finish in 1.575s, 328 probes kept.\n",
      "-- check region:303 4:73735001-73765000_strand_+_gene_303, 57074 candidate probes\n",
      "--- 10599 probes passed check_dic selection.\n",
      "finish in 1.267s, 213 probes kept.\n",
      "-- check region:304 4:76235001-76265000_strand_+_gene_304, 58618 candidate probes\n",
      "--- 12953 probes passed check_dic selection.\n",
      "finish in 1.394s, 232 probes kept.\n",
      "-- check region:305 4:78735001-78765000_strand_+_gene_305, 57434 candidate probes\n",
      "--- 11524 probes passed check_dic selection.\n",
      "finish in 1.310s, 222 probes kept.\n",
      "-- check region:306 4:81235001-81265000_strand_+_gene_306, 58416 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 12242 probes passed check_dic selection.\n",
      "finish in 1.355s, 242 probes kept.\n",
      "-- check region:307 4:83735001-83765000_strand_+_gene_307, 59474 candidate probes\n",
      "--- 11535 probes passed check_dic selection.\n",
      "finish in 1.342s, 231 probes kept.\n",
      "-- check region:308 4:86235001-86265000_strand_+_gene_308, 52754 candidate probes\n",
      "--- 10618 probes passed check_dic selection.\n",
      "finish in 1.207s, 203 probes kept.\n",
      "-- check region:309 4:88735001-88765000_strand_+_gene_309, 58880 candidate probes\n",
      "--- 18640 probes passed check_dic selection.\n",
      "finish in 1.613s, 343 probes kept.\n",
      "-- check region:310 4:91235001-91265000_strand_+_gene_310, 55942 candidate probes\n",
      "--- 6910 probes passed check_dic selection.\n",
      "finish in 1.112s, 166 probes kept.\n",
      "-- check region:311 4:93735001-93765000_strand_+_gene_311, 59373 candidate probes\n",
      "--- 13572 probes passed check_dic selection.\n",
      "finish in 1.411s, 267 probes kept.\n",
      "-- check region:312 4:96235001-96265000_strand_+_gene_312, 59822 candidate probes\n",
      "--- 11560 probes passed check_dic selection.\n",
      "finish in 1.344s, 241 probes kept.\n",
      "-- check region:313 4:98735001-98765000_strand_+_gene_313, 54682 candidate probes\n",
      "--- 9726 probes passed check_dic selection.\n",
      "finish in 1.212s, 182 probes kept.\n",
      "-- check region:314 4:101235001-101265000_strand_+_gene_314, 59568 candidate probes\n",
      "--- 12113 probes passed check_dic selection.\n",
      "finish in 1.351s, 253 probes kept.\n",
      "-- check region:315 4:103735001-103765000_strand_+_gene_315, 59362 candidate probes\n",
      "--- 9957 probes passed check_dic selection.\n",
      "finish in 1.278s, 228 probes kept.\n",
      "-- check region:316 4:106235001-106265000_strand_+_gene_316, 59386 candidate probes\n",
      "--- 7965 probes passed check_dic selection.\n",
      "finish in 1.200s, 183 probes kept.\n",
      "-- check region:317 4:108735001-108765000_strand_+_gene_317, 58312 candidate probes\n",
      "--- 13293 probes passed check_dic selection.\n",
      "finish in 1.406s, 251 probes kept.\n",
      "-- check region:318 4:111235001-111265000_strand_+_gene_318, 54424 candidate probes\n",
      "--- 8236 probes passed check_dic selection.\n",
      "finish in 1.141s, 178 probes kept.\n",
      "-- check region:319 4:113735001-113765000_strand_+_gene_319, 59692 candidate probes\n",
      "--- 11581 probes passed check_dic selection.\n",
      "finish in 1.335s, 249 probes kept.\n",
      "-- check region:320 4:116235001-116265000_strand_+_gene_320, 55300 candidate probes\n",
      "--- 6259 probes passed check_dic selection.\n",
      "finish in 1.099s, 155 probes kept.\n",
      "-- check region:321 4:118735001-118765000_strand_+_gene_321, 59562 candidate probes\n",
      "--- 13916 probes passed check_dic selection.\n",
      "finish in 1.436s, 279 probes kept.\n",
      "-- check region:322 4:121235001-121265000_strand_+_gene_322, 57988 candidate probes\n",
      "--- 14672 probes passed check_dic selection.\n",
      "finish in 1.442s, 258 probes kept.\n",
      "-- check region:323 4:123735001-123765000_strand_+_gene_323, 59171 candidate probes\n",
      "--- 13605 probes passed check_dic selection.\n",
      "finish in 1.416s, 248 probes kept.\n",
      "-- check region:324 4:126235001-126265000_strand_+_gene_324, 59134 candidate probes\n",
      "--- 7381 probes passed check_dic selection.\n",
      "finish in 1.174s, 171 probes kept.\n",
      "-- check region:325 4:128735001-128765000_strand_+_gene_325, 57776 candidate probes\n",
      "--- 8482 probes passed check_dic selection.\n",
      "finish in 1.218s, 176 probes kept.\n",
      "-- check region:326 4:131235001-131265000_strand_+_gene_326, 59414 candidate probes\n",
      "--- 8441 probes passed check_dic selection.\n",
      "finish in 1.210s, 191 probes kept.\n",
      "-- check region:327 4:133735001-133765000_strand_+_gene_327, 53616 candidate probes\n",
      "--- 4203 probes passed check_dic selection.\n",
      "finish in 0.991s, 106 probes kept.\n",
      "-- check region:328 4:136235001-136265000_strand_+_gene_328, 58890 candidate probes\n",
      "--- 6531 probes passed check_dic selection.\n",
      "finish in 1.153s, 170 probes kept.\n",
      "-- check region:329 4:138735001-138765000_strand_+_gene_329, 58016 candidate probes\n",
      "--- 17038 probes passed check_dic selection.\n",
      "finish in 1.538s, 295 probes kept.\n",
      "-- check region:330 4:141235001-141265000_strand_+_gene_330, 59480 candidate probes\n",
      "--- 10940 probes passed check_dic selection.\n",
      "finish in 1.332s, 218 probes kept.\n",
      "-- check region:331 4:143735001-143765000_strand_+_gene_331, 58548 candidate probes\n",
      "--- 5834 probes passed check_dic selection.\n",
      "finish in 1.119s, 154 probes kept.\n",
      "-- check region:332 4:146235001-146265000_strand_+_gene_332, 59396 candidate probes\n",
      "--- 22539 probes passed check_dic selection.\n",
      "finish in 1.831s, 377 probes kept.\n",
      "-- check region:333 4:148735001-148765000_strand_+_gene_333, 57418 candidate probes\n",
      "--- 11058 probes passed check_dic selection.\n",
      "finish in 1.299s, 214 probes kept.\n",
      "-- check region:334 4:151235001-151265000_strand_+_gene_334, 57116 candidate probes\n",
      "--- 11642 probes passed check_dic selection.\n",
      "finish in 1.314s, 234 probes kept.\n",
      "-- check region:335 4:153735001-153765000_strand_+_gene_335, 58926 candidate probes\n",
      "--- 15490 probes passed check_dic selection.\n",
      "finish in 1.493s, 271 probes kept.\n",
      "-- check region:336 4:156235001-156265000_strand_+_gene_336, 59654 candidate probes\n",
      "--- 11363 probes passed check_dic selection.\n",
      "finish in 1.336s, 244 probes kept.\n",
      "-- check region:337 4:158735001-158765000_strand_+_gene_337, 56642 candidate probes\n",
      "--- 8361 probes passed check_dic selection.\n",
      "finish in 1.203s, 181 probes kept.\n",
      "-- check region:338 4:161235001-161265000_strand_+_gene_338, 59884 candidate probes\n",
      "--- 7976 probes passed check_dic selection.\n",
      "finish in 1.217s, 222 probes kept.\n",
      "-- check region:339 4:163735001-163765000_strand_+_gene_339, 57884 candidate probes\n",
      "--- 12569 probes passed check_dic selection.\n",
      "finish in 1.352s, 257 probes kept.\n",
      "-- check region:340 4:166235001-166265000_strand_+_gene_340, 55378 candidate probes\n",
      "--- 10964 probes passed check_dic selection.\n",
      "finish in 1.245s, 201 probes kept.\n",
      "-- check region:341 4:168735001-168765000_strand_+_gene_341, 58206 candidate probes\n",
      "--- 15769 probes passed check_dic selection.\n",
      "finish in 1.483s, 284 probes kept.\n",
      "-- check region:342 4:171235001-171265000_strand_+_gene_342, 58712 candidate probes\n",
      "--- 5718 probes passed check_dic selection.\n",
      "finish in 1.112s, 157 probes kept.\n",
      "-- check region:343 4:173735001-173765000_strand_+_gene_343, 57288 candidate probes\n",
      "--- 14086 probes passed check_dic selection.\n",
      "finish in 1.392s, 260 probes kept.\n",
      "-- check region:344 4:176235001-176265000_strand_+_gene_344, 58226 candidate probes\n",
      "--- 11960 probes passed check_dic selection.\n",
      "finish in 1.338s, 234 probes kept.\n",
      "-- check region:345 4:178735001-178765000_strand_+_gene_345, 59584 candidate probes\n",
      "--- 10358 probes passed check_dic selection.\n",
      "finish in 1.292s, 225 probes kept.\n",
      "-- check region:346 4:181235001-181265000_strand_+_gene_346, 54286 candidate probes\n",
      "--- 16090 probes passed check_dic selection.\n",
      "finish in 1.439s, 279 probes kept.\n",
      "-- check region:347 4:183735001-183765000_strand_+_gene_347, 56876 candidate probes\n",
      "--- 11453 probes passed check_dic selection.\n",
      "finish in 1.300s, 202 probes kept.\n",
      "-- check region:348 4:186235001-186265000_strand_+_gene_348, 56416 candidate probes\n",
      "--- 11895 probes passed check_dic selection.\n",
      "finish in 1.334s, 223 probes kept.\n",
      "-- check region:349 4:188735001-188765000_strand_+_gene_349, 58986 candidate probes\n",
      "--- 7998 probes passed check_dic selection.\n",
      "finish in 1.208s, 161 probes kept.\n",
      "-- check region:350 5:1235001-1265000_strand_+_gene_350, 58844 candidate probes\n",
      "--- 28944 probes passed check_dic selection.\n",
      "finish in 2.026s, 415 probes kept.\n",
      "-- check region:351 5:3735001-3765000_strand_+_gene_351, 58624 candidate probes\n",
      "--- 21753 probes passed check_dic selection.\n",
      "finish in 1.730s, 319 probes kept.\n",
      "-- check region:352 5:6235001-6265000_strand_+_gene_352, 59232 candidate probes\n",
      "--- 19360 probes passed check_dic selection.\n",
      "finish in 1.632s, 325 probes kept.\n",
      "-- check region:353 5:8735001-8765000_strand_+_gene_353, 54702 candidate probes\n",
      "--- 7310 probes passed check_dic selection.\n",
      "finish in 1.129s, 158 probes kept.\n",
      "-- check region:354 5:11235001-11265000_strand_+_gene_354, 58940 candidate probes\n",
      "--- 17777 probes passed check_dic selection.\n",
      "finish in 1.575s, 293 probes kept.\n",
      "-- check region:355 5:13735001-13765000_strand_+_gene_355, 56876 candidate probes\n",
      "--- 12344 probes passed check_dic selection.\n",
      "finish in 1.340s, 234 probes kept.\n",
      "-- check region:356 5:16235001-16265000_strand_+_gene_356, 59652 candidate probes\n",
      "--- 13984 probes passed check_dic selection.\n",
      "finish in 1.437s, 257 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:357 5:18735001-18765000_strand_+_gene_357, 59204 candidate probes\n",
      "--- 7013 probes passed check_dic selection.\n",
      "finish in 1.180s, 170 probes kept.\n",
      "-- check region:358 5:21235001-21265000_strand_+_gene_358, 58456 candidate probes\n",
      "--- 9329 probes passed check_dic selection.\n",
      "finish in 1.248s, 216 probes kept.\n",
      "-- check region:359 5:23735001-23765000_strand_+_gene_359, 58770 candidate probes\n",
      "--- 8580 probes passed check_dic selection.\n",
      "finish in 1.214s, 179 probes kept.\n",
      "-- check region:360 5:26235001-26265000_strand_+_gene_360, 59062 candidate probes\n",
      "--- 8788 probes passed check_dic selection.\n",
      "finish in 1.227s, 190 probes kept.\n",
      "-- check region:361 5:28735001-28765000_strand_+_gene_361, 53544 candidate probes\n",
      "--- 8271 probes passed check_dic selection.\n",
      "finish in 1.122s, 175 probes kept.\n",
      "-- check region:362 5:31235001-31265000_strand_+_gene_362, 59162 candidate probes\n",
      "--- 17331 probes passed check_dic selection.\n",
      "finish in 1.577s, 315 probes kept.\n",
      "-- check region:363 5:33735001-33765000_strand_+_gene_363, 59068 candidate probes\n",
      "--- 21668 probes passed check_dic selection.\n",
      "finish in 1.746s, 350 probes kept.\n",
      "-- check region:364 5:36235001-36265000_strand_+_gene_364, 59388 candidate probes\n",
      "--- 13534 probes passed check_dic selection.\n",
      "finish in 1.414s, 253 probes kept.\n",
      "-- check region:365 5:38735001-38765000_strand_+_gene_365, 59046 candidate probes\n",
      "--- 16349 probes passed check_dic selection.\n",
      "finish in 1.529s, 284 probes kept.\n",
      "-- check region:366 5:41235001-41265000_strand_+_gene_366, 53282 candidate probes\n",
      "--- 11233 probes passed check_dic selection.\n",
      "finish in 1.238s, 214 probes kept.\n",
      "-- check region:367 5:43735001-43765000_strand_+_gene_367, 54478 candidate probes\n",
      "--- 4208 probes passed check_dic selection.\n",
      "finish in 1.015s, 102 probes kept.\n",
      "-- check region:368 5:46235001-46265000_strand_+_gene_368, 59530 candidate probes\n",
      "--- 1032 probes passed check_dic selection.\n",
      "finish in 0.966s, 64 probes kept.\n",
      "-- check region:369 5:48735001-48765000_strand_+_gene_369, 24304 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.391s, 0 probes kept.\n",
      "-- check region:370 5:51235001-51265000_strand_+_gene_370, 50960 candidate probes\n",
      "--- 6825 probes passed check_dic selection.\n",
      "finish in 1.035s, 158 probes kept.\n",
      "-- check region:371 5:53735001-53765000_strand_+_gene_371, 54598 candidate probes\n",
      "--- 3175 probes passed check_dic selection.\n",
      "finish in 0.980s, 84 probes kept.\n",
      "-- check region:372 5:56235001-56265000_strand_+_gene_372, 58392 candidate probes\n",
      "--- 13554 probes passed check_dic selection.\n",
      "finish in 1.409s, 236 probes kept.\n",
      "-- check region:373 5:58735001-58765000_strand_+_gene_373, 58844 candidate probes\n",
      "--- 14895 probes passed check_dic selection.\n",
      "finish in 1.461s, 270 probes kept.\n",
      "-- check region:374 5:61235001-61265000_strand_+_gene_374, 52802 candidate probes\n",
      "--- 13482 probes passed check_dic selection.\n",
      "finish in 1.320s, 224 probes kept.\n",
      "-- check region:375 5:63735001-63765000_strand_+_gene_375, 58246 candidate probes\n",
      "--- 11151 probes passed check_dic selection.\n",
      "finish in 1.314s, 226 probes kept.\n",
      "-- check region:376 5:66235001-66265000_strand_+_gene_376, 52526 candidate probes\n",
      "--- 13116 probes passed check_dic selection.\n",
      "finish in 1.316s, 222 probes kept.\n",
      "-- check region:377 5:68735001-68765000_strand_+_gene_377, 59138 candidate probes\n",
      "--- 21259 probes passed check_dic selection.\n",
      "finish in 1.718s, 340 probes kept.\n",
      "-- check region:378 5:71235001-71265000_strand_+_gene_378, 55834 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.868s, 0 probes kept.\n",
      "-- check region:379 5:73735001-73765000_strand_+_gene_379, 58704 candidate probes\n",
      "--- 16691 probes passed check_dic selection.\n",
      "finish in 1.519s, 287 probes kept.\n",
      "-- check region:380 5:76235001-76265000_strand_+_gene_380, 57776 candidate probes\n",
      "--- 12588 probes passed check_dic selection.\n",
      "finish in 1.376s, 224 probes kept.\n",
      "-- check region:381 5:78735001-78765000_strand_+_gene_381, 58622 candidate probes\n",
      "--- 17248 probes passed check_dic selection.\n",
      "finish in 1.561s, 278 probes kept.\n",
      "-- check region:382 5:81235001-81265000_strand_+_gene_382, 57584 candidate probes\n",
      "--- 18961 probes passed check_dic selection.\n",
      "finish in 1.625s, 318 probes kept.\n",
      "-- check region:383 5:83735001-83765000_strand_+_gene_383, 57522 candidate probes\n",
      "--- 7500 probes passed check_dic selection.\n",
      "finish in 1.176s, 176 probes kept.\n",
      "-- check region:384 5:86235001-86265000_strand_+_gene_384, 58670 candidate probes\n",
      "--- 7344 probes passed check_dic selection.\n",
      "finish in 1.164s, 166 probes kept.\n",
      "-- check region:385 5:88735001-88765000_strand_+_gene_385, 58144 candidate probes\n",
      "--- 13547 probes passed check_dic selection.\n",
      "finish in 1.398s, 272 probes kept.\n",
      "-- check region:386 5:91235001-91265000_strand_+_gene_386, 58938 candidate probes\n",
      "--- 10074 probes passed check_dic selection.\n",
      "finish in 1.282s, 215 probes kept.\n",
      "-- check region:387 5:93735001-93765000_strand_+_gene_387, 56008 candidate probes\n",
      "--- 8698 probes passed check_dic selection.\n",
      "finish in 1.204s, 192 probes kept.\n",
      "-- check region:388 5:96235001-96265000_strand_+_gene_388, 57526 candidate probes\n",
      "--- 15719 probes passed check_dic selection.\n",
      "finish in 1.461s, 277 probes kept.\n",
      "-- check region:389 5:98735001-98765000_strand_+_gene_389, 57504 candidate probes\n",
      "--- 7234 probes passed check_dic selection.\n",
      "finish in 1.157s, 170 probes kept.\n",
      "-- check region:390 5:101235001-101265000_strand_+_gene_390, 59332 candidate probes\n",
      "--- 6865 probes passed check_dic selection.\n",
      "finish in 1.173s, 164 probes kept.\n",
      "-- check region:391 5:103735001-103765000_strand_+_gene_391, 58602 candidate probes\n",
      "--- 12027 probes passed check_dic selection.\n",
      "finish in 1.351s, 257 probes kept.\n",
      "-- check region:392 5:106235001-106265000_strand_+_gene_392, 58906 candidate probes\n",
      "--- 8898 probes passed check_dic selection.\n",
      "finish in 1.259s, 202 probes kept.\n",
      "-- check region:393 5:108735001-108765000_strand_+_gene_393, 58444 candidate probes\n",
      "--- 11195 probes passed check_dic selection.\n",
      "finish in 1.310s, 229 probes kept.\n",
      "-- check region:394 5:111235001-111265000_strand_+_gene_394, 59592 candidate probes\n",
      "--- 18167 probes passed check_dic selection.\n",
      "finish in 1.596s, 309 probes kept.\n",
      "-- check region:395 5:113735001-113765000_strand_+_gene_395, 58296 candidate probes\n",
      "--- 13054 probes passed check_dic selection.\n",
      "finish in 1.385s, 256 probes kept.\n",
      "-- check region:396 5:116235001-116265000_strand_+_gene_396, 53912 candidate probes\n",
      "--- 7453 probes passed check_dic selection.\n",
      "finish in 1.107s, 182 probes kept.\n",
      "-- check region:397 5:118735001-118765000_strand_+_gene_397, 58838 candidate probes\n",
      "--- 14321 probes passed check_dic selection.\n",
      "finish in 1.442s, 245 probes kept.\n",
      "-- check region:398 5:121235001-121265000_strand_+_gene_398, 56712 candidate probes\n",
      "--- 6667 probes passed check_dic selection.\n",
      "finish in 1.116s, 162 probes kept.\n",
      "-- check region:399 5:123735001-123765000_strand_+_gene_399, 59612 candidate probes\n",
      "--- 11138 probes passed check_dic selection.\n",
      "finish in 1.316s, 196 probes kept.\n",
      "-- check region:400 5:126235001-126265000_strand_+_gene_400, 58076 candidate probes\n",
      "--- 12039 probes passed check_dic selection.\n",
      "finish in 1.342s, 222 probes kept.\n",
      "-- check region:401 5:128735001-128765000_strand_+_gene_401, 58658 candidate probes\n",
      "--- 12589 probes passed check_dic selection.\n",
      "finish in 1.368s, 237 probes kept.\n",
      "-- check region:402 5:131235001-131265000_strand_+_gene_402, 57384 candidate probes\n",
      "--- 10030 probes passed check_dic selection.\n",
      "finish in 1.253s, 191 probes kept.\n",
      "-- check region:403 5:133735001-133765000_strand_+_gene_403, 59326 candidate probes\n",
      "--- 26130 probes passed check_dic selection.\n",
      "finish in 1.907s, 377 probes kept.\n",
      "-- check region:404 5:136235001-136265000_strand_+_gene_404, 59770 candidate probes\n",
      "--- 20940 probes passed check_dic selection.\n",
      "finish in 1.713s, 331 probes kept.\n",
      "-- check region:405 5:138735001-138765000_strand_+_gene_405, 56450 candidate probes\n",
      "--- 10844 probes passed check_dic selection.\n",
      "finish in 1.289s, 205 probes kept.\n",
      "-- check region:406 5:141235001-141265000_strand_+_gene_406, 58602 candidate probes\n",
      "--- 10206 probes passed check_dic selection.\n",
      "finish in 1.301s, 261 probes kept.\n",
      "-- check region:407 5:143735001-143765000_strand_+_gene_407, 59248 candidate probes\n",
      "--- 17059 probes passed check_dic selection.\n",
      "finish in 1.556s, 306 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:408 5:146235001-146265000_strand_+_gene_408, 56880 candidate probes\n",
      "--- 6809 probes passed check_dic selection.\n",
      "finish in 1.143s, 163 probes kept.\n",
      "-- check region:409 5:148735001-148765000_strand_+_gene_409, 58354 candidate probes\n",
      "--- 12846 probes passed check_dic selection.\n",
      "finish in 1.386s, 221 probes kept.\n",
      "-- check region:410 5:151235001-151265000_strand_+_gene_410, 58452 candidate probes\n",
      "--- 16114 probes passed check_dic selection.\n",
      "finish in 1.519s, 271 probes kept.\n",
      "-- check region:411 5:153735001-153765000_strand_+_gene_411, 59156 candidate probes\n",
      "--- 16064 probes passed check_dic selection.\n",
      "finish in 1.512s, 282 probes kept.\n",
      "-- check region:412 5:156235001-156265000_strand_+_gene_412, 58598 candidate probes\n",
      "--- 10484 probes passed check_dic selection.\n",
      "finish in 1.281s, 212 probes kept.\n",
      "-- check region:413 5:158735001-158765000_strand_+_gene_413, 59412 candidate probes\n",
      "--- 18989 probes passed check_dic selection.\n",
      "finish in 1.622s, 339 probes kept.\n",
      "-- check region:414 5:161235001-161265000_strand_+_gene_414, 55674 candidate probes\n",
      "--- 9346 probes passed check_dic selection.\n",
      "finish in 1.224s, 188 probes kept.\n",
      "-- check region:415 5:163735001-163765000_strand_+_gene_415, 59108 candidate probes\n",
      "--- 13183 probes passed check_dic selection.\n",
      "finish in 1.403s, 252 probes kept.\n",
      "-- check region:416 5:166235001-166265000_strand_+_gene_416, 59350 candidate probes\n",
      "--- 13622 probes passed check_dic selection.\n",
      "finish in 1.407s, 252 probes kept.\n",
      "-- check region:417 5:168735001-168765000_strand_+_gene_417, 58344 candidate probes\n",
      "--- 20345 probes passed check_dic selection.\n",
      "finish in 1.686s, 315 probes kept.\n",
      "-- check region:418 5:171235001-171265000_strand_+_gene_418, 58362 candidate probes\n",
      "--- 9558 probes passed check_dic selection.\n",
      "finish in 1.287s, 200 probes kept.\n",
      "-- check region:419 5:173735001-173765000_strand_+_gene_419, 58678 candidate probes\n",
      "--- 23552 probes passed check_dic selection.\n",
      "finish in 1.809s, 355 probes kept.\n",
      "-- check region:420 5:176235001-176265000_strand_+_gene_420, 54728 candidate probes\n",
      "--- 1000 probes passed check_dic selection.\n",
      "finish in 0.900s, 37 probes kept.\n",
      "-- check region:421 5:178735001-178765000_strand_+_gene_421, 57852 candidate probes\n",
      "--- 10228 probes passed check_dic selection.\n",
      "finish in 1.286s, 212 probes kept.\n",
      "-- check region:422 6:1235001-1265000_strand_+_gene_422, 58392 candidate probes\n",
      "--- 19603 probes passed check_dic selection.\n",
      "finish in 1.647s, 298 probes kept.\n",
      "-- check region:423 6:3735001-3765000_strand_+_gene_423, 59398 candidate probes\n",
      "--- 22974 probes passed check_dic selection.\n",
      "finish in 1.787s, 345 probes kept.\n",
      "-- check region:424 6:6235001-6265000_strand_+_gene_424, 58358 candidate probes\n",
      "--- 18492 probes passed check_dic selection.\n",
      "finish in 1.593s, 323 probes kept.\n",
      "-- check region:425 6:8735001-8765000_strand_+_gene_425, 59282 candidate probes\n",
      "--- 11841 probes passed check_dic selection.\n",
      "finish in 1.354s, 221 probes kept.\n",
      "-- check region:426 6:11235001-11265000_strand_+_gene_426, 59502 candidate probes\n",
      "--- 22652 probes passed check_dic selection.\n",
      "finish in 1.773s, 350 probes kept.\n",
      "-- check region:427 6:13735001-13765000_strand_+_gene_427, 56734 candidate probes\n",
      "--- 12281 probes passed check_dic selection.\n",
      "finish in 1.342s, 217 probes kept.\n",
      "-- check region:428 6:16235001-16265000_strand_+_gene_428, 55176 candidate probes\n",
      "--- 15424 probes passed check_dic selection.\n",
      "finish in 1.442s, 243 probes kept.\n",
      "-- check region:429 6:18735001-18765000_strand_+_gene_429, 55708 candidate probes\n",
      "--- 9154 probes passed check_dic selection.\n",
      "finish in 1.204s, 193 probes kept.\n",
      "-- check region:430 6:21235001-21265000_strand_+_gene_430, 58278 candidate probes\n",
      "--- 13443 probes passed check_dic selection.\n",
      "finish in 1.400s, 236 probes kept.\n",
      "-- check region:431 6:23735001-23765000_strand_+_gene_431, 58448 candidate probes\n",
      "--- 11542 probes passed check_dic selection.\n",
      "finish in 1.330s, 237 probes kept.\n",
      "-- check region:432 6:26235001-26265000_strand_+_gene_432, 58348 candidate probes\n",
      "--- 14722 probes passed check_dic selection.\n",
      "finish in 1.456s, 270 probes kept.\n",
      "-- check region:433 6:28735001-28765000_strand_+_gene_433, 57786 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.921s, 0 probes kept.\n",
      "-- check region:434 6:31235001-31265000_strand_+_gene_434, 53592 candidate probes\n",
      "--- 108 probes passed check_dic selection.\n",
      "finish in 0.856s, 5 probes kept.\n",
      "-- check region:435 6:33735001-33765000_strand_+_gene_435, 58186 candidate probes\n",
      "--- 22674 probes passed check_dic selection.\n",
      "finish in 1.770s, 357 probes kept.\n",
      "-- check region:436 6:36235001-36265000_strand_+_gene_436, 58796 candidate probes\n",
      "--- 18289 probes passed check_dic selection.\n",
      "finish in 1.612s, 294 probes kept.\n",
      "-- check region:437 6:38735001-38765000_strand_+_gene_437, 55554 candidate probes\n",
      "--- 7860 probes passed check_dic selection.\n",
      "finish in 1.156s, 188 probes kept.\n",
      "-- check region:438 6:41235001-41265000_strand_+_gene_438, 56446 candidate probes\n",
      "--- 10300 probes passed check_dic selection.\n",
      "finish in 1.273s, 206 probes kept.\n",
      "-- check region:439 6:43735001-43765000_strand_+_gene_439, 58982 candidate probes\n",
      "--- 19427 probes passed check_dic selection.\n",
      "finish in 1.664s, 310 probes kept.\n",
      "-- check region:440 6:46235001-46265000_strand_+_gene_440, 59438 candidate probes\n",
      "--- 17864 probes passed check_dic selection.\n",
      "finish in 1.585s, 321 probes kept.\n",
      "-- check region:441 6:48735001-48765000_strand_+_gene_441, 51024 candidate probes\n",
      "--- 7988 probes passed check_dic selection.\n",
      "finish in 1.076s, 183 probes kept.\n",
      "-- check region:442 6:51235001-51265000_strand_+_gene_442, 59470 candidate probes\n",
      "--- 14199 probes passed check_dic selection.\n",
      "finish in 1.437s, 276 probes kept.\n",
      "-- check region:443 6:53735001-53765000_strand_+_gene_443, 58526 candidate probes\n",
      "--- 12697 probes passed check_dic selection.\n",
      "finish in 1.380s, 241 probes kept.\n",
      "-- check region:444 6:56235001-56265000_strand_+_gene_444, 59790 candidate probes\n",
      "--- 16212 probes passed check_dic selection.\n",
      "finish in 1.511s, 307 probes kept.\n",
      "-- check region:445 6:58735001-58765000_strand_+_gene_445, 14622 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.238s, 0 probes kept.\n",
      "-- check region:446 6:61235001-61265000_strand_+_gene_446, 55292 candidate probes\n",
      "--- 4728 probes passed check_dic selection.\n",
      "finish in 1.032s, 108 probes kept.\n",
      "-- check region:447 6:63735001-63765000_strand_+_gene_447, 54288 candidate probes\n",
      "--- 8054 probes passed check_dic selection.\n",
      "finish in 1.145s, 181 probes kept.\n",
      "-- check region:448 6:66235001-66265000_strand_+_gene_448, 58510 candidate probes\n",
      "--- 4580 probes passed check_dic selection.\n",
      "finish in 1.079s, 127 probes kept.\n",
      "-- check region:449 6:68735001-68765000_strand_+_gene_449, 59456 candidate probes\n",
      "--- 11111 probes passed check_dic selection.\n",
      "finish in 1.317s, 228 probes kept.\n",
      "-- check region:450 6:71235001-71265000_strand_+_gene_450, 59491 candidate probes\n",
      "--- 13566 probes passed check_dic selection.\n",
      "finish in 1.417s, 268 probes kept.\n",
      "-- check region:451 6:73735001-73765000_strand_+_gene_451, 58302 candidate probes\n",
      "--- 13529 probes passed check_dic selection.\n",
      "finish in 1.407s, 258 probes kept.\n",
      "-- check region:452 6:76235001-76265000_strand_+_gene_452, 59190 candidate probes\n",
      "--- 8912 probes passed check_dic selection.\n",
      "finish in 1.244s, 197 probes kept.\n",
      "-- check region:453 6:78735001-78765000_strand_+_gene_453, 52894 candidate probes\n",
      "--- 2144 probes passed check_dic selection.\n",
      "finish in 0.910s, 67 probes kept.\n",
      "-- check region:454 6:81235001-81265000_strand_+_gene_454, 57654 candidate probes\n",
      "--- 6470 probes passed check_dic selection.\n",
      "finish in 1.121s, 145 probes kept.\n",
      "-- check region:455 6:83735001-83765000_strand_+_gene_455, 58740 candidate probes\n",
      "--- 5414 probes passed check_dic selection.\n",
      "finish in 1.105s, 142 probes kept.\n",
      "-- check region:456 6:86235001-86265000_strand_+_gene_456, 57828 candidate probes\n",
      "--- 11139 probes passed check_dic selection.\n",
      "finish in 1.313s, 217 probes kept.\n",
      "-- check region:457 6:88735001-88765000_strand_+_gene_457, 59404 candidate probes\n",
      "--- 9824 probes passed check_dic selection.\n",
      "finish in 1.277s, 207 probes kept.\n",
      "-- check region:458 6:91235001-91265000_strand_+_gene_458, 59664 candidate probes\n",
      "--- 12553 probes passed check_dic selection.\n",
      "finish in 1.385s, 236 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:459 6:93735001-93765000_strand_+_gene_459, 59612 candidate probes\n",
      "--- 8077 probes passed check_dic selection.\n",
      "finish in 1.198s, 180 probes kept.\n",
      "-- check region:460 6:96235001-96265000_strand_+_gene_460, 58726 candidate probes\n",
      "--- 8874 probes passed check_dic selection.\n",
      "finish in 1.232s, 202 probes kept.\n",
      "-- check region:461 6:98735001-98765000_strand_+_gene_461, 59402 candidate probes\n",
      "--- 10817 probes passed check_dic selection.\n",
      "finish in 1.317s, 209 probes kept.\n",
      "-- check region:462 6:101235001-101265000_strand_+_gene_462, 56344 candidate probes\n",
      "--- 7306 probes passed check_dic selection.\n",
      "finish in 1.138s, 170 probes kept.\n",
      "-- check region:463 6:103735001-103765000_strand_+_gene_463, 57636 candidate probes\n",
      "--- 6226 probes passed check_dic selection.\n",
      "finish in 1.119s, 143 probes kept.\n",
      "-- check region:464 6:106235001-106265000_strand_+_gene_464, 59076 candidate probes\n",
      "--- 9690 probes passed check_dic selection.\n",
      "finish in 1.264s, 211 probes kept.\n",
      "-- check region:465 6:108735001-108765000_strand_+_gene_465, 58654 candidate probes\n",
      "--- 21972 probes passed check_dic selection.\n",
      "finish in 1.739s, 352 probes kept.\n",
      "-- check region:466 6:111235001-111265000_strand_+_gene_466, 57222 candidate probes\n",
      "--- 10460 probes passed check_dic selection.\n",
      "finish in 1.277s, 189 probes kept.\n",
      "-- check region:467 6:113735001-113765000_strand_+_gene_467, 57176 candidate probes\n",
      "--- 10480 probes passed check_dic selection.\n",
      "finish in 1.274s, 200 probes kept.\n",
      "-- check region:468 6:116235001-116265000_strand_+_gene_468, 59232 candidate probes\n",
      "--- 7985 probes passed check_dic selection.\n",
      "finish in 1.235s, 222 probes kept.\n",
      "-- check region:469 6:118735001-118765000_strand_+_gene_469, 58286 candidate probes\n",
      "--- 15356 probes passed check_dic selection.\n",
      "finish in 1.487s, 260 probes kept.\n",
      "-- check region:470 6:121235001-121265000_strand_+_gene_470, 58672 candidate probes\n",
      "--- 5042 probes passed check_dic selection.\n",
      "finish in 1.095s, 131 probes kept.\n",
      "-- check region:471 6:123735001-123765000_strand_+_gene_471, 59022 candidate probes\n",
      "--- 8233 probes passed check_dic selection.\n",
      "finish in 1.207s, 199 probes kept.\n",
      "-- check region:472 6:126235001-126265000_strand_+_gene_472, 54670 candidate probes\n",
      "--- 10094 probes passed check_dic selection.\n",
      "finish in 1.216s, 203 probes kept.\n",
      "-- check region:473 6:128735001-128765000_strand_+_gene_473, 59134 candidate probes\n",
      "--- 10493 probes passed check_dic selection.\n",
      "finish in 1.301s, 203 probes kept.\n",
      "-- check region:474 6:131235001-131265000_strand_+_gene_474, 53924 candidate probes\n",
      "--- 12981 probes passed check_dic selection.\n",
      "finish in 1.319s, 235 probes kept.\n",
      "-- check region:475 6:133735001-133765000_strand_+_gene_475, 59592 candidate probes\n",
      "--- 13540 probes passed check_dic selection.\n",
      "finish in 1.414s, 263 probes kept.\n",
      "-- check region:476 6:136235001-136265000_strand_+_gene_476, 58416 candidate probes\n",
      "--- 11193 probes passed check_dic selection.\n",
      "finish in 1.321s, 236 probes kept.\n",
      "-- check region:477 6:138735001-138765000_strand_+_gene_477, 57216 candidate probes\n",
      "--- 11848 probes passed check_dic selection.\n",
      "finish in 1.334s, 215 probes kept.\n",
      "-- check region:478 6:141235001-141265000_strand_+_gene_478, 57824 candidate probes\n",
      "--- 7652 probes passed check_dic selection.\n",
      "finish in 1.157s, 189 probes kept.\n",
      "-- check region:479 6:143735001-143765000_strand_+_gene_479, 58634 candidate probes\n",
      "--- 17328 probes passed check_dic selection.\n",
      "finish in 1.560s, 287 probes kept.\n",
      "-- check region:480 6:146235001-146265000_strand_+_gene_480, 58952 candidate probes\n",
      "--- 14006 probes passed check_dic selection.\n",
      "finish in 1.429s, 257 probes kept.\n",
      "-- check region:481 6:148735001-148765000_strand_+_gene_481, 57956 candidate probes\n",
      "--- 16188 probes passed check_dic selection.\n",
      "finish in 1.491s, 269 probes kept.\n",
      "-- check region:482 6:151235001-151265000_strand_+_gene_482, 57186 candidate probes\n",
      "--- 13863 probes passed check_dic selection.\n",
      "finish in 1.416s, 230 probes kept.\n",
      "-- check region:483 6:153735001-153765000_strand_+_gene_483, 59198 candidate probes\n",
      "--- 10037 probes passed check_dic selection.\n",
      "finish in 1.293s, 220 probes kept.\n",
      "-- check region:484 6:156235001-156265000_strand_+_gene_484, 58910 candidate probes\n",
      "--- 14100 probes passed check_dic selection.\n",
      "finish in 1.446s, 265 probes kept.\n",
      "-- check region:485 6:158735001-158765000_strand_+_gene_485, 56788 candidate probes\n",
      "--- 15744 probes passed check_dic selection.\n",
      "finish in 1.475s, 255 probes kept.\n",
      "-- check region:486 6:161235001-161265000_strand_+_gene_486, 58420 candidate probes\n",
      "--- 20027 probes passed check_dic selection.\n",
      "finish in 1.658s, 294 probes kept.\n",
      "-- check region:487 6:163735001-163765000_strand_+_gene_487, 59034 candidate probes\n",
      "--- 21320 probes passed check_dic selection.\n",
      "finish in 1.740s, 342 probes kept.\n",
      "-- check region:488 6:166235001-166265000_strand_+_gene_488, 58184 candidate probes\n",
      "--- 52 probes passed check_dic selection.\n",
      "finish in 0.935s, 3 probes kept.\n",
      "-- check region:489 6:168735001-168765000_strand_+_gene_489, 58548 candidate probes\n",
      "--- 11867 probes passed check_dic selection.\n",
      "finish in 1.353s, 216 probes kept.\n",
      "-- check region:490 7:1235001-1265000_strand_+_gene_490, 59058 candidate probes\n",
      "--- 25941 probes passed check_dic selection.\n",
      "finish in 1.874s, 384 probes kept.\n",
      "-- check region:491 7:3735001-3765000_strand_+_gene_491, 58846 candidate probes\n",
      "--- 15596 probes passed check_dic selection.\n",
      "finish in 1.495s, 276 probes kept.\n",
      "-- check region:492 7:6235001-6265000_strand_+_gene_492, 57965 candidate probes\n",
      "--- 13241 probes passed check_dic selection.\n",
      "finish in 1.400s, 229 probes kept.\n",
      "-- check region:493 7:8735001-8765000_strand_+_gene_493, 58774 candidate probes\n",
      "--- 12135 probes passed check_dic selection.\n",
      "finish in 1.354s, 239 probes kept.\n",
      "-- check region:494 7:11235001-11265000_strand_+_gene_494, 59314 candidate probes\n",
      "--- 14673 probes passed check_dic selection.\n",
      "finish in 1.466s, 293 probes kept.\n",
      "-- check region:495 7:13735001-13765000_strand_+_gene_495, 59226 candidate probes\n",
      "--- 13267 probes passed check_dic selection.\n",
      "finish in 1.389s, 259 probes kept.\n",
      "-- check region:496 7:16235001-16265000_strand_+_gene_496, 56808 candidate probes\n",
      "--- 6990 probes passed check_dic selection.\n",
      "finish in 1.137s, 167 probes kept.\n",
      "-- check region:497 7:18735001-18765000_strand_+_gene_497, 57740 candidate probes\n",
      "--- 10322 probes passed check_dic selection.\n",
      "finish in 1.265s, 228 probes kept.\n",
      "-- check region:498 7:21235001-21265000_strand_+_gene_498, 58528 candidate probes\n",
      "--- 13638 probes passed check_dic selection.\n",
      "finish in 1.401s, 262 probes kept.\n",
      "-- check region:499 7:23735001-23765000_strand_+_gene_499, 57398 candidate probes\n",
      "--- 7698 probes passed check_dic selection.\n",
      "finish in 1.179s, 162 probes kept.\n",
      "-- check region:500 7:26235001-26265000_strand_+_gene_500, 55900 candidate probes\n",
      "--- 5262 probes passed check_dic selection.\n",
      "finish in 1.084s, 142 probes kept.\n",
      "-- check region:501 7:28735001-28765000_strand_+_gene_501, 57528 candidate probes\n",
      "--- 18329 probes passed check_dic selection.\n",
      "finish in 1.564s, 304 probes kept.\n",
      "-- check region:502 7:31235001-31265000_strand_+_gene_502, 59096 candidate probes\n",
      "--- 16295 probes passed check_dic selection.\n",
      "finish in 1.520s, 269 probes kept.\n",
      "-- check region:503 7:33735001-33765000_strand_+_gene_503, 59038 candidate probes\n",
      "--- 19694 probes passed check_dic selection.\n",
      "finish in 1.663s, 318 probes kept.\n",
      "-- check region:504 7:36235001-36265000_strand_+_gene_504, 56492 candidate probes\n",
      "--- 12168 probes passed check_dic selection.\n",
      "finish in 1.340s, 220 probes kept.\n",
      "-- check region:505 7:38735001-38765000_strand_+_gene_505, 59808 candidate probes\n",
      "--- 17556 probes passed check_dic selection.\n",
      "finish in 1.572s, 308 probes kept.\n",
      "-- check region:506 7:41235001-41265000_strand_+_gene_506, 59594 candidate probes\n",
      "--- 17450 probes passed check_dic selection.\n",
      "finish in 1.569s, 306 probes kept.\n",
      "-- check region:507 7:43735001-43765000_strand_+_gene_507, 57900 candidate probes\n",
      "--- 10945 probes passed check_dic selection.\n",
      "finish in 1.307s, 202 probes kept.\n",
      "-- check region:508 7:46235001-46265000_strand_+_gene_508, 59208 candidate probes\n",
      "--- 10168 probes passed check_dic selection.\n",
      "finish in 1.284s, 221 probes kept.\n",
      "-- check region:509 7:48735001-48765000_strand_+_gene_509, 59608 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 15186 probes passed check_dic selection.\n",
      "finish in 1.479s, 272 probes kept.\n",
      "-- check region:510 7:51235001-51265000_strand_+_gene_510, 59338 candidate probes\n",
      "--- 21452 probes passed check_dic selection.\n",
      "finish in 1.729s, 336 probes kept.\n",
      "-- check region:511 7:53735001-53765000_strand_+_gene_511, 59534 candidate probes\n",
      "--- 8992 probes passed check_dic selection.\n",
      "finish in 1.248s, 184 probes kept.\n",
      "-- check region:512 7:56235001-56265000_strand_+_gene_512, 56492 candidate probes\n",
      "--- 9995 probes passed check_dic selection.\n",
      "finish in 1.270s, 197 probes kept.\n",
      "-- check region:513 7:58735001-58765000_strand_+_gene_513, 11780 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.191s, 0 probes kept.\n",
      "-- check region:514 7:61235001-61265000_strand_+_gene_514, 22414 candidate probes\n",
      "--- 18 probes passed check_dic selection.\n",
      "finish in 0.359s, 2 probes kept.\n",
      "-- check region:515 7:63735001-63765000_strand_+_gene_515, 33010 candidate probes\n",
      "--- 322 probes passed check_dic selection.\n",
      "finish in 0.547s, 20 probes kept.\n",
      "-- check region:516 7:66235001-66265000_strand_+_gene_516, 57870 candidate probes\n",
      "--- 10584 probes passed check_dic selection.\n",
      "finish in 1.297s, 227 probes kept.\n",
      "-- check region:517 7:68735001-68765000_strand_+_gene_517, 57906 candidate probes\n",
      "--- 14651 probes passed check_dic selection.\n",
      "finish in 1.459s, 262 probes kept.\n",
      "-- check region:518 7:71235001-71265000_strand_+_gene_518, 58108 candidate probes\n",
      "--- 15277 probes passed check_dic selection.\n",
      "finish in 1.485s, 264 probes kept.\n",
      "-- check region:519 7:73735001-73765000_strand_+_gene_519, 55418 candidate probes\n",
      "--- 10501 probes passed check_dic selection.\n",
      "finish in 1.282s, 188 probes kept.\n",
      "-- check region:520 7:76235001-76265000_strand_+_gene_520, 55754 candidate probes\n",
      "--- 13254 probes passed check_dic selection.\n",
      "finish in 1.380s, 212 probes kept.\n",
      "-- check region:521 7:78735001-78765000_strand_+_gene_521, 58934 candidate probes\n",
      "--- 12509 probes passed check_dic selection.\n",
      "finish in 1.367s, 265 probes kept.\n",
      "-- check region:522 7:81235001-81265000_strand_+_gene_522, 59296 candidate probes\n",
      "--- 7588 probes passed check_dic selection.\n",
      "finish in 1.196s, 187 probes kept.\n",
      "-- check region:523 7:83735001-83765000_strand_+_gene_523, 52294 candidate probes\n",
      "--- 7339 probes passed check_dic selection.\n",
      "finish in 1.080s, 152 probes kept.\n",
      "-- check region:524 7:86235001-86265000_strand_+_gene_524, 56958 candidate probes\n",
      "--- 8650 probes passed check_dic selection.\n",
      "finish in 1.188s, 196 probes kept.\n",
      "-- check region:525 7:88735001-88765000_strand_+_gene_525, 59626 candidate probes\n",
      "--- 10108 probes passed check_dic selection.\n",
      "finish in 1.278s, 216 probes kept.\n",
      "-- check region:526 7:91235001-91265000_strand_+_gene_526, 58114 candidate probes\n",
      "--- 12610 probes passed check_dic selection.\n",
      "finish in 1.366s, 223 probes kept.\n",
      "-- check region:527 7:93735001-93765000_strand_+_gene_527, 58410 candidate probes\n",
      "--- 12178 probes passed check_dic selection.\n",
      "finish in 1.343s, 255 probes kept.\n",
      "-- check region:528 7:96235001-96265000_strand_+_gene_528, 58342 candidate probes\n",
      "--- 14230 probes passed check_dic selection.\n",
      "finish in 1.434s, 249 probes kept.\n",
      "-- check region:529 7:98735001-98765000_strand_+_gene_529, 56898 candidate probes\n",
      "--- 17912 probes passed check_dic selection.\n",
      "finish in 1.562s, 269 probes kept.\n",
      "-- check region:530 7:101235001-101265000_strand_+_gene_530, 56806 candidate probes\n",
      "--- 12896 probes passed check_dic selection.\n",
      "finish in 1.408s, 223 probes kept.\n",
      "-- check region:531 7:103735001-103765000_strand_+_gene_531, 54226 candidate probes\n",
      "--- 10681 probes passed check_dic selection.\n",
      "finish in 1.236s, 201 probes kept.\n",
      "-- check region:532 7:106235001-106265000_strand_+_gene_532, 55740 candidate probes\n",
      "--- 12049 probes passed check_dic selection.\n",
      "finish in 1.312s, 232 probes kept.\n",
      "-- check region:533 7:108735001-108765000_strand_+_gene_533, 58946 candidate probes\n",
      "--- 10228 probes passed check_dic selection.\n",
      "finish in 1.292s, 201 probes kept.\n",
      "-- check region:534 7:111235001-111265000_strand_+_gene_534, 48940 candidate probes\n",
      "--- 8227 probes passed check_dic selection.\n",
      "finish in 1.053s, 181 probes kept.\n",
      "-- check region:535 7:113735001-113765000_strand_+_gene_535, 45718 candidate probes\n",
      "--- 5035 probes passed check_dic selection.\n",
      "finish in 0.894s, 116 probes kept.\n",
      "-- check region:536 7:116235001-116265000_strand_+_gene_536, 58996 candidate probes\n",
      "--- 12561 probes passed check_dic selection.\n",
      "finish in 1.383s, 259 probes kept.\n",
      "-- check region:537 7:118735001-118765000_strand_+_gene_537, 59498 candidate probes\n",
      "--- 11385 probes passed check_dic selection.\n",
      "finish in 1.329s, 231 probes kept.\n",
      "-- check region:538 7:121235001-121265000_strand_+_gene_538, 55112 candidate probes\n",
      "--- 11315 probes passed check_dic selection.\n",
      "finish in 1.273s, 214 probes kept.\n",
      "-- check region:539 7:123735001-123765000_strand_+_gene_539, 59296 candidate probes\n",
      "--- 12614 probes passed check_dic selection.\n",
      "finish in 1.366s, 242 probes kept.\n",
      "-- check region:540 7:126235001-126265000_strand_+_gene_540, 58816 candidate probes\n",
      "--- 8154 probes passed check_dic selection.\n",
      "finish in 1.195s, 179 probes kept.\n",
      "-- check region:541 7:128735001-128765000_strand_+_gene_541, 58330 candidate probes\n",
      "--- 13976 probes passed check_dic selection.\n",
      "finish in 1.431s, 248 probes kept.\n",
      "-- check region:542 7:131235001-131265000_strand_+_gene_542, 57724 candidate probes\n",
      "--- 13859 probes passed check_dic selection.\n",
      "finish in 1.413s, 227 probes kept.\n",
      "-- check region:543 7:133735001-133765000_strand_+_gene_543, 59574 candidate probes\n",
      "--- 16651 probes passed check_dic selection.\n",
      "finish in 1.542s, 302 probes kept.\n",
      "-- check region:544 7:136235001-136265000_strand_+_gene_544, 59234 candidate probes\n",
      "--- 14305 probes passed check_dic selection.\n",
      "finish in 1.454s, 276 probes kept.\n",
      "-- check region:545 7:138735001-138765000_strand_+_gene_545, 57266 candidate probes\n",
      "--- 16229 probes passed check_dic selection.\n",
      "finish in 1.510s, 267 probes kept.\n",
      "-- check region:546 7:141235001-141265000_strand_+_gene_546, 59082 candidate probes\n",
      "--- 19321 probes passed check_dic selection.\n",
      "finish in 1.630s, 305 probes kept.\n",
      "-- check region:547 7:143735001-143765000_strand_+_gene_547, 55556 candidate probes\n",
      "--- 1807 probes passed check_dic selection.\n",
      "finish in 0.947s, 44 probes kept.\n",
      "-- check region:548 7:146235001-146265000_strand_+_gene_548, 57240 candidate probes\n",
      "--- 9504 probes passed check_dic selection.\n",
      "finish in 1.237s, 191 probes kept.\n",
      "-- check region:549 7:148735001-148765000_strand_+_gene_549, 58218 candidate probes\n",
      "--- 10746 probes passed check_dic selection.\n",
      "finish in 1.299s, 206 probes kept.\n",
      "-- check region:550 7:151235001-151265000_strand_+_gene_550, 58426 candidate probes\n",
      "--- 23572 probes passed check_dic selection.\n",
      "finish in 1.831s, 378 probes kept.\n",
      "-- check region:551 7:153735001-153765000_strand_+_gene_551, 59062 candidate probes\n",
      "--- 3248 probes passed check_dic selection.\n",
      "finish in 1.039s, 90 probes kept.\n",
      "-- check region:552 7:156235001-156265000_strand_+_gene_552, 59514 candidate probes\n",
      "--- 17668 probes passed check_dic selection.\n",
      "finish in 1.580s, 264 probes kept.\n",
      "-- check region:553 8:1235001-1265000_strand_+_gene_553, 53214 candidate probes\n",
      "--- 1480 probes passed check_dic selection.\n",
      "finish in 0.920s, 49 probes kept.\n",
      "-- check region:554 8:3735001-3765000_strand_+_gene_554, 59280 candidate probes\n",
      "--- 18404 probes passed check_dic selection.\n",
      "finish in 1.594s, 306 probes kept.\n",
      "-- check region:555 8:6235001-6265000_strand_+_gene_555, 57736 candidate probes\n",
      "--- 13714 probes passed check_dic selection.\n",
      "finish in 1.405s, 241 probes kept.\n",
      "-- check region:556 8:8735001-8765000_strand_+_gene_556, 58102 candidate probes\n",
      "--- 13515 probes passed check_dic selection.\n",
      "finish in 1.402s, 234 probes kept.\n",
      "-- check region:557 8:11235001-11265000_strand_+_gene_557, 59186 candidate probes\n",
      "--- 15948 probes passed check_dic selection.\n",
      "finish in 1.514s, 273 probes kept.\n",
      "-- check region:558 8:13735001-13765000_strand_+_gene_558, 59458 candidate probes\n",
      "--- 12585 probes passed check_dic selection.\n",
      "finish in 1.387s, 251 probes kept.\n",
      "-- check region:559 8:16235001-16265000_strand_+_gene_559, 59112 candidate probes\n",
      "--- 11281 probes passed check_dic selection.\n",
      "finish in 1.326s, 221 probes kept.\n",
      "-- check region:560 8:18735001-18765000_strand_+_gene_560, 58018 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 13812 probes passed check_dic selection.\n",
      "finish in 1.410s, 261 probes kept.\n",
      "-- check region:561 8:21235001-21265000_strand_+_gene_561, 58978 candidate probes\n",
      "--- 13366 probes passed check_dic selection.\n",
      "finish in 1.396s, 240 probes kept.\n",
      "-- check region:562 8:23735001-23765000_strand_+_gene_562, 58728 candidate probes\n",
      "--- 15209 probes passed check_dic selection.\n",
      "finish in 1.483s, 249 probes kept.\n",
      "-- check region:563 8:26235001-26265000_strand_+_gene_563, 57378 candidate probes\n",
      "--- 10265 probes passed check_dic selection.\n",
      "finish in 1.287s, 193 probes kept.\n",
      "-- check region:564 8:28735001-28765000_strand_+_gene_564, 56384 candidate probes\n",
      "--- 14050 probes passed check_dic selection.\n",
      "finish in 1.415s, 270 probes kept.\n",
      "-- check region:565 8:31235001-31265000_strand_+_gene_565, 57680 candidate probes\n",
      "--- 11456 probes passed check_dic selection.\n",
      "finish in 1.328s, 218 probes kept.\n",
      "-- check region:566 8:33735001-33765000_strand_+_gene_566, 58946 candidate probes\n",
      "--- 8690 probes passed check_dic selection.\n",
      "finish in 1.251s, 169 probes kept.\n",
      "-- check region:567 8:36235001-36265000_strand_+_gene_567, 58530 candidate probes\n",
      "--- 13148 probes passed check_dic selection.\n",
      "finish in 1.403s, 229 probes kept.\n",
      "-- check region:568 8:38735001-38765000_strand_+_gene_568, 58086 candidate probes\n",
      "--- 12693 probes passed check_dic selection.\n",
      "finish in 1.376s, 222 probes kept.\n",
      "-- check region:569 8:41235001-41265000_strand_+_gene_569, 59066 candidate probes\n",
      "--- 17171 probes passed check_dic selection.\n",
      "finish in 1.566s, 292 probes kept.\n",
      "-- check region:570 8:43735001-43765000_strand_+_gene_570, 58986 candidate probes\n",
      "--- 2317 probes passed check_dic selection.\n",
      "finish in 1.017s, 104 probes kept.\n",
      "-- check region:571 8:46235001-46265000_strand_+_gene_571, 59692 candidate probes\n",
      "--- 18551 probes passed check_dic selection.\n",
      "finish in 1.656s, 356 probes kept.\n",
      "-- check region:572 8:48735001-48765000_strand_+_gene_572, 58682 candidate probes\n",
      "--- 17341 probes passed check_dic selection.\n",
      "finish in 1.548s, 280 probes kept.\n",
      "-- check region:573 8:51235001-51265000_strand_+_gene_573, 59564 candidate probes\n",
      "--- 9165 probes passed check_dic selection.\n",
      "finish in 1.260s, 206 probes kept.\n",
      "-- check region:574 8:53735001-53765000_strand_+_gene_574, 58094 candidate probes\n",
      "--- 13846 probes passed check_dic selection.\n",
      "finish in 1.411s, 258 probes kept.\n",
      "-- check region:575 8:56235001-56265000_strand_+_gene_575, 55372 candidate probes\n",
      "--- 10284 probes passed check_dic selection.\n",
      "finish in 1.235s, 204 probes kept.\n",
      "-- check region:576 8:58735001-58765000_strand_+_gene_576, 59668 candidate probes\n",
      "--- 17149 probes passed check_dic selection.\n",
      "finish in 1.552s, 291 probes kept.\n",
      "-- check region:577 8:61235001-61265000_strand_+_gene_577, 56988 candidate probes\n",
      "--- 10344 probes passed check_dic selection.\n",
      "finish in 1.276s, 207 probes kept.\n",
      "-- check region:578 8:63735001-63765000_strand_+_gene_578, 59666 candidate probes\n",
      "--- 12430 probes passed check_dic selection.\n",
      "finish in 1.381s, 246 probes kept.\n",
      "-- check region:579 8:66235001-66265000_strand_+_gene_579, 58238 candidate probes\n",
      "--- 6592 probes passed check_dic selection.\n",
      "finish in 1.149s, 151 probes kept.\n",
      "-- check region:580 8:68735001-68765000_strand_+_gene_580, 52198 candidate probes\n",
      "--- 10313 probes passed check_dic selection.\n",
      "finish in 1.185s, 202 probes kept.\n",
      "-- check region:581 8:71235001-71265000_strand_+_gene_581, 59522 candidate probes\n",
      "--- 13847 probes passed check_dic selection.\n",
      "finish in 1.421s, 250 probes kept.\n",
      "-- check region:582 8:73735001-73765000_strand_+_gene_582, 57024 candidate probes\n",
      "--- 10981 probes passed check_dic selection.\n",
      "finish in 1.289s, 199 probes kept.\n",
      "-- check region:583 8:76235001-76265000_strand_+_gene_583, 59650 candidate probes\n",
      "--- 7832 probes passed check_dic selection.\n",
      "finish in 1.200s, 181 probes kept.\n",
      "-- check region:584 8:78735001-78765000_strand_+_gene_584, 58456 candidate probes\n",
      "--- 5299 probes passed check_dic selection.\n",
      "finish in 1.092s, 139 probes kept.\n",
      "-- check region:585 8:81235001-81265000_strand_+_gene_585, 59522 candidate probes\n",
      "--- 12095 probes passed check_dic selection.\n",
      "finish in 1.371s, 241 probes kept.\n",
      "-- check region:586 8:83735001-83765000_strand_+_gene_586, 59586 candidate probes\n",
      "--- 9264 probes passed check_dic selection.\n",
      "finish in 1.248s, 218 probes kept.\n",
      "-- check region:587 8:86235001-86265000_strand_+_gene_587, 59368 candidate probes\n",
      "--- 5999 probes passed check_dic selection.\n",
      "finish in 1.140s, 156 probes kept.\n",
      "-- check region:588 8:88735001-88765000_strand_+_gene_588, 57440 candidate probes\n",
      "--- 7672 probes passed check_dic selection.\n",
      "finish in 1.168s, 177 probes kept.\n",
      "-- check region:589 8:91235001-91265000_strand_+_gene_589, 59736 candidate probes\n",
      "--- 13972 probes passed check_dic selection.\n",
      "finish in 1.426s, 275 probes kept.\n",
      "-- check region:590 8:93735001-93765000_strand_+_gene_590, 58602 candidate probes\n",
      "--- 11231 probes passed check_dic selection.\n",
      "finish in 1.342s, 225 probes kept.\n",
      "-- check region:591 8:96235001-96265000_strand_+_gene_591, 59180 candidate probes\n",
      "--- 16137 probes passed check_dic selection.\n",
      "finish in 1.513s, 283 probes kept.\n",
      "-- check region:592 8:98735001-98765000_strand_+_gene_592, 52732 candidate probes\n",
      "--- 6568 probes passed check_dic selection.\n",
      "finish in 1.062s, 144 probes kept.\n",
      "-- check region:593 8:101235001-101265000_strand_+_gene_593, 59166 candidate probes\n",
      "--- 17489 probes passed check_dic selection.\n",
      "finish in 1.574s, 306 probes kept.\n",
      "-- check region:594 8:103735001-103765000_strand_+_gene_594, 54296 candidate probes\n",
      "--- 7094 probes passed check_dic selection.\n",
      "finish in 1.100s, 155 probes kept.\n",
      "-- check region:595 8:106235001-106265000_strand_+_gene_595, 59424 candidate probes\n",
      "--- 7965 probes passed check_dic selection.\n",
      "finish in 1.209s, 180 probes kept.\n",
      "-- check region:596 8:108735001-108765000_strand_+_gene_596, 59488 candidate probes\n",
      "--- 12145 probes passed check_dic selection.\n",
      "finish in 1.363s, 254 probes kept.\n",
      "-- check region:597 8:111235001-111265000_strand_+_gene_597, 59800 candidate probes\n",
      "--- 14840 probes passed check_dic selection.\n",
      "finish in 1.468s, 277 probes kept.\n",
      "-- check region:598 8:113735001-113765000_strand_+_gene_598, 59640 candidate probes\n",
      "--- 11169 probes passed check_dic selection.\n",
      "finish in 1.318s, 226 probes kept.\n",
      "-- check region:599 8:116235001-116265000_strand_+_gene_599, 59356 candidate probes\n",
      "--- 17227 probes passed check_dic selection.\n",
      "finish in 1.558s, 304 probes kept.\n",
      "-- check region:600 8:118735001-118765000_strand_+_gene_600, 59846 candidate probes\n",
      "--- 16373 probes passed check_dic selection.\n",
      "finish in 1.546s, 294 probes kept.\n",
      "-- check region:601 8:121235001-121265000_strand_+_gene_601, 59528 candidate probes\n",
      "--- 16894 probes passed check_dic selection.\n",
      "finish in 1.541s, 299 probes kept.\n",
      "-- check region:602 8:123735001-123765000_strand_+_gene_602, 58336 candidate probes\n",
      "--- 14617 probes passed check_dic selection.\n",
      "finish in 1.456s, 261 probes kept.\n",
      "-- check region:603 8:126235001-126265000_strand_+_gene_603, 58372 candidate probes\n",
      "--- 12673 probes passed check_dic selection.\n",
      "finish in 1.370s, 251 probes kept.\n",
      "-- check region:604 8:128735001-128765000_strand_+_gene_604, 48616 candidate probes\n",
      "--- 102 probes passed check_dic selection.\n",
      "finish in 0.766s, 6 probes kept.\n",
      "-- check region:605 8:131235001-131265000_strand_+_gene_605, 51326 candidate probes\n",
      "--- 11282 probes passed check_dic selection.\n",
      "finish in 1.222s, 187 probes kept.\n",
      "-- check region:606 8:133735001-133765000_strand_+_gene_606, 56408 candidate probes\n",
      "--- 18163 probes passed check_dic selection.\n",
      "finish in 1.550s, 293 probes kept.\n",
      "-- check region:607 8:136235001-136265000_strand_+_gene_607, 59599 candidate probes\n",
      "--- 16356 probes passed check_dic selection.\n",
      "finish in 1.534s, 281 probes kept.\n",
      "-- check region:608 8:138735001-138765000_strand_+_gene_608, 58988 candidate probes\n",
      "--- 23497 probes passed check_dic selection.\n",
      "finish in 1.801s, 355 probes kept.\n",
      "-- check region:609 8:141235001-141265000_strand_+_gene_609, 55248 candidate probes\n",
      "--- 23012 probes passed check_dic selection.\n",
      "finish in 1.744s, 323 probes kept.\n",
      "-- check region:610 8:143735001-143765000_strand_+_gene_610, 57796 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.923s, 0 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:611 9:985001-1015000_strand_+_gene_611, 58302 candidate probes\n",
      "--- 17781 probes passed check_dic selection.\n",
      "finish in 1.571s, 302 probes kept.\n",
      "-- check region:612 9:2985001-3015000_strand_+_gene_612, 59036 candidate probes\n",
      "--- 19552 probes passed check_dic selection.\n",
      "finish in 1.650s, 311 probes kept.\n",
      "-- check region:613 9:4985001-5015000_strand_+_gene_613, 58348 candidate probes\n",
      "--- 8937 probes passed check_dic selection.\n",
      "finish in 1.233s, 200 probes kept.\n",
      "-- check region:614 9:6985001-7015000_strand_+_gene_614, 58838 candidate probes\n",
      "--- 15060 probes passed check_dic selection.\n",
      "finish in 1.467s, 277 probes kept.\n",
      "-- check region:615 9:8985001-9015000_strand_+_gene_615, 59752 candidate probes\n",
      "--- 13016 probes passed check_dic selection.\n",
      "finish in 1.392s, 258 probes kept.\n",
      "-- check region:616 9:10985001-11015000_strand_+_gene_616, 58558 candidate probes\n",
      "--- 7764 probes passed check_dic selection.\n",
      "finish in 1.193s, 173 probes kept.\n",
      "-- check region:617 9:12985001-13015000_strand_+_gene_617, 59440 candidate probes\n",
      "--- 12918 probes passed check_dic selection.\n",
      "finish in 1.387s, 254 probes kept.\n",
      "-- check region:618 9:14985001-15015000_strand_+_gene_618, 58940 candidate probes\n",
      "--- 9758 probes passed check_dic selection.\n",
      "finish in 1.284s, 192 probes kept.\n",
      "-- check region:619 9:16985001-17015000_strand_+_gene_619, 59236 candidate probes\n",
      "--- 15747 probes passed check_dic selection.\n",
      "finish in 1.498s, 282 probes kept.\n",
      "-- check region:620 9:18985001-19015000_strand_+_gene_620, 56436 candidate probes\n",
      "--- 10313 probes passed check_dic selection.\n",
      "finish in 1.267s, 196 probes kept.\n",
      "-- check region:621 9:20985001-21015000_strand_+_gene_621, 59524 candidate probes\n",
      "--- 11207 probes passed check_dic selection.\n",
      "finish in 1.337s, 248 probes kept.\n",
      "-- check region:622 9:22985001-23015000_strand_+_gene_622, 58370 candidate probes\n",
      "--- 11766 probes passed check_dic selection.\n",
      "finish in 1.328s, 247 probes kept.\n",
      "-- check region:623 9:24985001-25015000_strand_+_gene_623, 59076 candidate probes\n",
      "--- 8287 probes passed check_dic selection.\n",
      "finish in 1.202s, 185 probes kept.\n",
      "-- check region:624 9:26985001-27015000_strand_+_gene_624, 57982 candidate probes\n",
      "--- 4332 probes passed check_dic selection.\n",
      "finish in 1.054s, 132 probes kept.\n",
      "-- check region:625 9:28985001-29015000_strand_+_gene_625, 52910 candidate probes\n",
      "--- 7284 probes passed check_dic selection.\n",
      "finish in 1.088s, 168 probes kept.\n",
      "-- check region:626 9:30985001-31015000_strand_+_gene_626, 56480 candidate probes\n",
      "--- 4242 probes passed check_dic selection.\n",
      "finish in 1.040s, 118 probes kept.\n",
      "-- check region:627 9:32985001-33015000_strand_+_gene_627, 56252 candidate probes\n",
      "--- 14177 probes passed check_dic selection.\n",
      "finish in 1.411s, 253 probes kept.\n",
      "-- check region:628 9:34985001-35015000_strand_+_gene_628, 57820 candidate probes\n",
      "--- 15020 probes passed check_dic selection.\n",
      "finish in 1.512s, 273 probes kept.\n",
      "-- check region:629 9:36985001-37015000_strand_+_gene_629, 59692 candidate probes\n",
      "--- 23904 probes passed check_dic selection.\n",
      "finish in 1.830s, 360 probes kept.\n",
      "-- check region:630 9:38985001-39015000_strand_+_gene_630, 58402 candidate probes\n",
      "--- 778 probes passed check_dic selection.\n",
      "finish in 0.955s, 37 probes kept.\n",
      "-- check region:631 9:40985001-41015000_strand_+_gene_631, 48250 candidate probes\n",
      "--- 314 probes passed check_dic selection.\n",
      "finish in 0.763s, 21 probes kept.\n",
      "-- check region:632 9:42985001-43015000_strand_+_gene_632, 51826 candidate probes\n",
      "--- 95 probes passed check_dic selection.\n",
      "finish in 0.807s, 9 probes kept.\n",
      "-- check region:633 9:44985001-45015000_strand_+_gene_633, 14458 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.232s, 0 probes kept.\n",
      "-- check region:634 9:46985001-47015000_strand_+_gene_634, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:635 9:48985001-49015000_strand_+_gene_635, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:636 9:50985001-51015000_strand_+_gene_636, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:637 9:52985001-53015000_strand_+_gene_637, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:638 9:54985001-55015000_strand_+_gene_638, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:639 9:56985001-57015000_strand_+_gene_639, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:640 9:58985001-59015000_strand_+_gene_640, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:641 9:60985001-61015000_strand_+_gene_641, 33884 candidate probes\n",
      "--- 156 probes passed check_dic selection.\n",
      "finish in 0.543s, 5 probes kept.\n",
      "-- check region:642 9:62985001-63015000_strand_+_gene_642, 13162 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.203s, 0 probes kept.\n",
      "-- check region:643 9:64985001-65015000_strand_+_gene_643, 25074 candidate probes\n",
      "--- 8 probes passed check_dic selection.\n",
      "finish in 0.393s, 1 probes kept.\n",
      "-- check region:644 9:66985001-67015000_strand_+_gene_644, 58532 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.934s, 0 probes kept.\n",
      "-- check region:645 9:68985001-69015000_strand_+_gene_645, 57344 candidate probes\n",
      "--- 15338 probes passed check_dic selection.\n",
      "finish in 1.477s, 257 probes kept.\n",
      "-- check region:646 9:70985001-71015000_strand_+_gene_646, 59258 candidate probes\n",
      "--- 12671 probes passed check_dic selection.\n",
      "finish in 1.389s, 241 probes kept.\n",
      "-- check region:647 9:72985001-73015000_strand_+_gene_647, 59194 candidate probes\n",
      "--- 10255 probes passed check_dic selection.\n",
      "finish in 1.279s, 219 probes kept.\n",
      "-- check region:648 9:74985001-75015000_strand_+_gene_648, 58784 candidate probes\n",
      "--- 10421 probes passed check_dic selection.\n",
      "finish in 1.295s, 215 probes kept.\n",
      "-- check region:649 9:76985001-77015000_strand_+_gene_649, 58082 candidate probes\n",
      "--- 15798 probes passed check_dic selection.\n",
      "finish in 1.489s, 262 probes kept.\n",
      "-- check region:650 9:78985001-79015000_strand_+_gene_650, 58878 candidate probes\n",
      "--- 11398 probes passed check_dic selection.\n",
      "finish in 1.320s, 241 probes kept.\n",
      "-- check region:651 9:80985001-81015000_strand_+_gene_651, 57854 candidate probes\n",
      "--- 6058 probes passed check_dic selection.\n",
      "finish in 1.164s, 146 probes kept.\n",
      "-- check region:652 9:82985001-83015000_strand_+_gene_652, 59538 candidate probes\n",
      "--- 17144 probes passed check_dic selection.\n",
      "finish in 1.561s, 306 probes kept.\n",
      "-- check region:653 9:84985001-85015000_strand_+_gene_653, 59666 candidate probes\n",
      "--- 24656 probes passed check_dic selection.\n",
      "finish in 1.864s, 373 probes kept.\n",
      "-- check region:654 9:86985001-87015000_strand_+_gene_654, 59664 candidate probes\n",
      "--- 13367 probes passed check_dic selection.\n",
      "finish in 1.423s, 271 probes kept.\n",
      "-- check region:655 9:88985001-89015000_strand_+_gene_655, 59446 candidate probes\n",
      "--- 18315 probes passed check_dic selection.\n",
      "finish in 1.664s, 401 probes kept.\n",
      "-- check region:656 9:90985001-91015000_strand_+_gene_656, 58472 candidate probes\n",
      "--- 17236 probes passed check_dic selection.\n",
      "finish in 1.544s, 292 probes kept.\n",
      "-- check region:657 9:92985001-93015000_strand_+_gene_657, 54310 candidate probes\n",
      "--- 10507 probes passed check_dic selection.\n",
      "finish in 1.258s, 186 probes kept.\n",
      "-- check region:658 9:94985001-95015000_strand_+_gene_658, 58864 candidate probes\n",
      "--- 18646 probes passed check_dic selection.\n",
      "finish in 1.597s, 311 probes kept.\n",
      "-- check region:659 9:96985001-97015000_strand_+_gene_659, 57732 candidate probes\n",
      "--- 1321 probes passed check_dic selection.\n",
      "finish in 0.962s, 54 probes kept.\n",
      "-- check region:660 9:98985001-99015000_strand_+_gene_660, 59066 candidate probes\n",
      "--- 17208 probes passed check_dic selection.\n",
      "finish in 1.577s, 278 probes kept.\n",
      "-- check region:661 9:100985001-101015000_strand_+_gene_661, 56326 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.878s, 0 probes kept.\n",
      "-- check region:662 9:102985001-103015000_strand_+_gene_662, 58904 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 6849 probes passed check_dic selection.\n",
      "finish in 1.153s, 184 probes kept.\n",
      "-- check region:663 9:104985001-105015000_strand_+_gene_663, 57592 candidate probes\n",
      "--- 11358 probes passed check_dic selection.\n",
      "finish in 1.331s, 205 probes kept.\n",
      "-- check region:664 9:106985001-107015000_strand_+_gene_664, 58432 candidate probes\n",
      "--- 13332 probes passed check_dic selection.\n",
      "finish in 1.401s, 255 probes kept.\n",
      "-- check region:665 9:108985001-109015000_strand_+_gene_665, 55922 candidate probes\n",
      "--- 14603 probes passed check_dic selection.\n",
      "finish in 1.408s, 266 probes kept.\n",
      "-- check region:666 9:110985001-111015000_strand_+_gene_666, 59410 candidate probes\n",
      "--- 13341 probes passed check_dic selection.\n",
      "finish in 1.408s, 273 probes kept.\n",
      "-- check region:667 9:112985001-113015000_strand_+_gene_667, 58482 candidate probes\n",
      "--- 6954 probes passed check_dic selection.\n",
      "finish in 1.171s, 172 probes kept.\n",
      "-- check region:668 9:114985001-115015000_strand_+_gene_668, 59146 candidate probes\n",
      "--- 20907 probes passed check_dic selection.\n",
      "finish in 1.717s, 328 probes kept.\n",
      "-- check region:669 9:116985001-117015000_strand_+_gene_669, 59744 candidate probes\n",
      "--- 17506 probes passed check_dic selection.\n",
      "finish in 1.586s, 312 probes kept.\n",
      "-- check region:670 9:118985001-119015000_strand_+_gene_670, 59286 candidate probes\n",
      "--- 17504 probes passed check_dic selection.\n",
      "finish in 1.574s, 311 probes kept.\n",
      "-- check region:671 9:120985001-121015000_strand_+_gene_671, 58408 candidate probes\n",
      "--- 9760 probes passed check_dic selection.\n",
      "finish in 1.264s, 199 probes kept.\n",
      "-- check region:672 9:122985001-123015000_strand_+_gene_672, 58572 candidate probes\n",
      "--- 6881 probes passed check_dic selection.\n",
      "finish in 1.164s, 171 probes kept.\n",
      "-- check region:673 9:124985001-125015000_strand_+_gene_673, 51506 candidate probes\n",
      "--- 5614 probes passed check_dic selection.\n",
      "finish in 1.028s, 120 probes kept.\n",
      "-- check region:674 9:126985001-127015000_strand_+_gene_674, 52094 candidate probes\n",
      "--- 16587 probes passed check_dic selection.\n",
      "finish in 1.436s, 258 probes kept.\n",
      "-- check region:675 9:128985001-129015000_strand_+_gene_675, 57862 candidate probes\n",
      "--- 17152 probes passed check_dic selection.\n",
      "finish in 1.577s, 327 probes kept.\n",
      "-- check region:676 9:130985001-131015000_strand_+_gene_676, 58506 candidate probes\n",
      "--- 22148 probes passed check_dic selection.\n",
      "finish in 1.755s, 330 probes kept.\n",
      "-- check region:677 9:132985001-133015000_strand_+_gene_677, 58040 candidate probes\n",
      "--- 19348 probes passed check_dic selection.\n",
      "finish in 1.663s, 308 probes kept.\n",
      "-- check region:678 9:134985001-135015000_strand_+_gene_678, 58590 candidate probes\n",
      "--- 24332 probes passed check_dic selection.\n",
      "finish in 1.833s, 347 probes kept.\n",
      "-- check region:679 9:136985001-137015000_strand_+_gene_679, 57460 candidate probes\n",
      "--- 16796 probes passed check_dic selection.\n",
      "finish in 1.532s, 320 probes kept.\n",
      "-- check region:680 10:985001-1015000_strand_+_gene_680, 57276 candidate probes\n",
      "--- 16217 probes passed check_dic selection.\n",
      "finish in 1.517s, 294 probes kept.\n",
      "-- check region:681 10:2985001-3015000_strand_+_gene_681, 58898 candidate probes\n",
      "--- 19585 probes passed check_dic selection.\n",
      "finish in 1.656s, 308 probes kept.\n",
      "-- check region:682 10:4985001-5015000_strand_+_gene_682, 59714 candidate probes\n",
      "--- 5166 probes passed check_dic selection.\n",
      "finish in 1.133s, 147 probes kept.\n",
      "-- check region:683 10:6985001-7015000_strand_+_gene_683, 58992 candidate probes\n",
      "--- 17639 probes passed check_dic selection.\n",
      "finish in 1.570s, 285 probes kept.\n",
      "-- check region:684 10:8985001-9015000_strand_+_gene_684, 58622 candidate probes\n",
      "--- 13481 probes passed check_dic selection.\n",
      "finish in 1.393s, 253 probes kept.\n",
      "-- check region:685 10:10985001-11015000_strand_+_gene_685, 59622 candidate probes\n",
      "--- 19918 probes passed check_dic selection.\n",
      "finish in 1.659s, 328 probes kept.\n",
      "-- check region:686 10:12985001-13015000_strand_+_gene_686, 58560 candidate probes\n",
      "--- 15810 probes passed check_dic selection.\n",
      "finish in 1.500s, 266 probes kept.\n",
      "-- check region:687 10:14985001-15015000_strand_+_gene_687, 56296 candidate probes\n",
      "--- 1778 probes passed check_dic selection.\n",
      "finish in 0.976s, 78 probes kept.\n",
      "-- check region:688 10:16985001-17015000_strand_+_gene_688, 58772 candidate probes\n",
      "--- 16248 probes passed check_dic selection.\n",
      "finish in 1.535s, 278 probes kept.\n",
      "-- check region:689 10:18985001-19015000_strand_+_gene_689, 52136 candidate probes\n",
      "--- 7588 probes passed check_dic selection.\n",
      "finish in 1.102s, 160 probes kept.\n",
      "-- check region:690 10:20985001-21015000_strand_+_gene_690, 59368 candidate probes\n",
      "--- 16381 probes passed check_dic selection.\n",
      "finish in 1.523s, 289 probes kept.\n",
      "-- check region:691 10:22985001-23015000_strand_+_gene_691, 59236 candidate probes\n",
      "--- 15827 probes passed check_dic selection.\n",
      "finish in 1.511s, 270 probes kept.\n",
      "-- check region:692 10:24985001-25015000_strand_+_gene_692, 56912 candidate probes\n",
      "--- 8981 probes passed check_dic selection.\n",
      "finish in 1.218s, 189 probes kept.\n",
      "-- check region:693 10:26985001-27015000_strand_+_gene_693, 57314 candidate probes\n",
      "--- 12806 probes passed check_dic selection.\n",
      "finish in 1.366s, 246 probes kept.\n",
      "-- check region:694 10:28985001-29015000_strand_+_gene_694, 57602 candidate probes\n",
      "--- 14624 probes passed check_dic selection.\n",
      "finish in 1.444s, 236 probes kept.\n",
      "-- check region:695 10:30985001-31015000_strand_+_gene_695, 58752 candidate probes\n",
      "--- 12652 probes passed check_dic selection.\n",
      "finish in 1.381s, 246 probes kept.\n",
      "-- check region:696 10:32985001-33015000_strand_+_gene_696, 57946 candidate probes\n",
      "--- 9196 probes passed check_dic selection.\n",
      "finish in 1.240s, 170 probes kept.\n",
      "-- check region:697 10:34985001-35015000_strand_+_gene_697, 57302 candidate probes\n",
      "--- 12472 probes passed check_dic selection.\n",
      "finish in 1.361s, 220 probes kept.\n",
      "-- check region:698 10:36985001-37015000_strand_+_gene_698, 58858 candidate probes\n",
      "--- 9130 probes passed check_dic selection.\n",
      "finish in 1.259s, 227 probes kept.\n",
      "-- check region:699 10:38985001-39015000_strand_+_gene_699, 54966 candidate probes\n",
      "--- 1946 probes passed check_dic selection.\n",
      "finish in 0.942s, 66 probes kept.\n",
      "-- check region:700 10:40985001-41015000_strand_+_gene_700, 17262 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.277s, 0 probes kept.\n",
      "-- check region:701 10:42985001-43015000_strand_+_gene_701, 59490 candidate probes\n",
      "--- 17204 probes passed check_dic selection.\n",
      "finish in 1.585s, 289 probes kept.\n",
      "-- check region:702 10:44985001-45015000_strand_+_gene_702, 52884 candidate probes\n",
      "--- 16162 probes passed check_dic selection.\n",
      "finish in 1.464s, 291 probes kept.\n",
      "-- check region:703 10:46985001-47015000_strand_+_gene_703, 56504 candidate probes\n",
      "--- 9847 probes passed check_dic selection.\n",
      "finish in 1.232s, 181 probes kept.\n",
      "-- check region:704 10:48985001-49015000_strand_+_gene_704, 57466 candidate probes\n",
      "--- 16334 probes passed check_dic selection.\n",
      "finish in 1.520s, 284 probes kept.\n",
      "-- check region:705 10:50985001-51015000_strand_+_gene_705, 59426 candidate probes\n",
      "--- 16551 probes passed check_dic selection.\n",
      "finish in 1.526s, 292 probes kept.\n",
      "-- check region:706 10:52985001-53015000_strand_+_gene_706, 59542 candidate probes\n",
      "--- 10349 probes passed check_dic selection.\n",
      "finish in 1.291s, 246 probes kept.\n",
      "-- check region:707 10:54985001-55015000_strand_+_gene_707, 58968 candidate probes\n",
      "--- 7118 probes passed check_dic selection.\n",
      "finish in 1.164s, 182 probes kept.\n",
      "-- check region:708 10:56985001-57015000_strand_+_gene_708, 57900 candidate probes\n",
      "--- 11011 probes passed check_dic selection.\n",
      "finish in 1.295s, 213 probes kept.\n",
      "-- check region:709 10:58985001-59015000_strand_+_gene_709, 59226 candidate probes\n",
      "--- 16056 probes passed check_dic selection.\n",
      "finish in 1.513s, 289 probes kept.\n",
      "-- check region:710 10:60985001-61015000_strand_+_gene_710, 57470 candidate probes\n",
      "--- 12368 probes passed check_dic selection.\n",
      "finish in 1.345s, 234 probes kept.\n",
      "-- check region:711 10:62985001-63015000_strand_+_gene_711, 59222 candidate probes\n",
      "--- 15436 probes passed check_dic selection.\n",
      "finish in 1.522s, 290 probes kept.\n",
      "-- check region:712 10:64985001-65015000_strand_+_gene_712, 59138 candidate probes\n",
      "--- 9931 probes passed check_dic selection.\n",
      "finish in 1.260s, 201 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:713 10:66985001-67015000_strand_+_gene_713, 59136 candidate probes\n",
      "--- 11383 probes passed check_dic selection.\n",
      "finish in 1.322s, 236 probes kept.\n",
      "-- check region:714 10:68985001-69015000_strand_+_gene_714, 58172 candidate probes\n",
      "--- 9358 probes passed check_dic selection.\n",
      "finish in 1.261s, 196 probes kept.\n",
      "-- check region:715 10:70985001-71015000_strand_+_gene_715, 59488 candidate probes\n",
      "--- 24131 probes passed check_dic selection.\n",
      "finish in 1.849s, 353 probes kept.\n",
      "-- check region:716 10:72985001-73015000_strand_+_gene_716, 55342 candidate probes\n",
      "--- 5170 probes passed check_dic selection.\n",
      "finish in 1.056s, 136 probes kept.\n",
      "-- check region:717 10:74985001-75015000_strand_+_gene_717, 58428 candidate probes\n",
      "--- 14826 probes passed check_dic selection.\n",
      "finish in 1.460s, 282 probes kept.\n",
      "-- check region:718 10:76985001-77015000_strand_+_gene_718, 59802 candidate probes\n",
      "--- 24600 probes passed check_dic selection.\n",
      "finish in 1.872s, 391 probes kept.\n",
      "-- check region:719 10:78985001-79015000_strand_+_gene_719, 59394 candidate probes\n",
      "--- 24352 probes passed check_dic selection.\n",
      "finish in 1.856s, 376 probes kept.\n",
      "-- check region:720 10:80985001-81015000_strand_+_gene_720, 58174 candidate probes\n",
      "--- 11806 probes passed check_dic selection.\n",
      "finish in 1.340s, 218 probes kept.\n",
      "-- check region:721 10:82985001-83015000_strand_+_gene_721, 59694 candidate probes\n",
      "--- 13924 probes passed check_dic selection.\n",
      "finish in 1.431s, 270 probes kept.\n",
      "-- check region:722 10:84985001-85015000_strand_+_gene_722, 59558 candidate probes\n",
      "--- 18092 probes passed check_dic selection.\n",
      "finish in 1.602s, 309 probes kept.\n",
      "-- check region:723 10:86985001-87015000_strand_+_gene_723, 57284 candidate probes\n",
      "--- 1851 probes passed check_dic selection.\n",
      "finish in 0.986s, 73 probes kept.\n",
      "-- check region:724 10:88985001-89015000_strand_+_gene_724, 59616 candidate probes\n",
      "--- 14502 probes passed check_dic selection.\n",
      "finish in 1.465s, 288 probes kept.\n",
      "-- check region:725 10:90985001-91015000_strand_+_gene_725, 59298 candidate probes\n",
      "--- 15469 probes passed check_dic selection.\n",
      "finish in 1.496s, 271 probes kept.\n",
      "-- check region:726 10:92985001-93015000_strand_+_gene_726, 59100 candidate probes\n",
      "--- 9080 probes passed check_dic selection.\n",
      "finish in 1.238s, 209 probes kept.\n",
      "-- check region:727 10:94985001-95015000_strand_+_gene_727, 58790 candidate probes\n",
      "--- 4922 probes passed check_dic selection.\n",
      "finish in 1.109s, 138 probes kept.\n",
      "-- check region:728 10:96985001-97015000_strand_+_gene_728, 59228 candidate probes\n",
      "--- 16797 probes passed check_dic selection.\n",
      "finish in 1.601s, 365 probes kept.\n",
      "-- check region:729 10:98985001-99015000_strand_+_gene_729, 53948 candidate probes\n",
      "--- 10135 probes passed check_dic selection.\n",
      "finish in 1.211s, 200 probes kept.\n",
      "-- check region:730 10:100985001-101015000_strand_+_gene_730, 58554 candidate probes\n",
      "--- 20086 probes passed check_dic selection.\n",
      "finish in 1.698s, 379 probes kept.\n",
      "-- check region:731 10:102985001-103015000_strand_+_gene_731, 57494 candidate probes\n",
      "--- 9173 probes passed check_dic selection.\n",
      "finish in 1.238s, 174 probes kept.\n",
      "-- check region:732 10:104985001-105015000_strand_+_gene_732, 59442 candidate probes\n",
      "--- 15676 probes passed check_dic selection.\n",
      "finish in 1.504s, 296 probes kept.\n",
      "-- check region:733 10:106985001-107015000_strand_+_gene_733, 57964 candidate probes\n",
      "--- 15393 probes passed check_dic selection.\n",
      "finish in 1.466s, 274 probes kept.\n",
      "-- check region:734 10:108985001-109015000_strand_+_gene_734, 55870 candidate probes\n",
      "--- 11633 probes passed check_dic selection.\n",
      "finish in 1.297s, 225 probes kept.\n",
      "-- check region:735 10:110985001-111015000_strand_+_gene_735, 58520 candidate probes\n",
      "--- 8803 probes passed check_dic selection.\n",
      "finish in 1.224s, 206 probes kept.\n",
      "-- check region:736 10:112985001-113015000_strand_+_gene_736, 58910 candidate probes\n",
      "--- 24659 probes passed check_dic selection.\n",
      "finish in 1.847s, 362 probes kept.\n",
      "-- check region:737 10:114985001-115015000_strand_+_gene_737, 58240 candidate probes\n",
      "--- 14856 probes passed check_dic selection.\n",
      "finish in 1.456s, 252 probes kept.\n",
      "-- check region:738 10:116985001-117015000_strand_+_gene_738, 57842 candidate probes\n",
      "--- 11235 probes passed check_dic selection.\n",
      "finish in 1.308s, 222 probes kept.\n",
      "-- check region:739 10:118985001-119015000_strand_+_gene_739, 58221 candidate probes\n",
      "--- 13436 probes passed check_dic selection.\n",
      "finish in 1.413s, 229 probes kept.\n",
      "-- check region:740 10:120985001-121015000_strand_+_gene_740, 59016 candidate probes\n",
      "--- 13317 probes passed check_dic selection.\n",
      "finish in 1.421s, 249 probes kept.\n",
      "-- check region:741 10:122985001-123015000_strand_+_gene_741, 59000 candidate probes\n",
      "--- 13316 probes passed check_dic selection.\n",
      "finish in 1.415s, 266 probes kept.\n",
      "-- check region:742 10:124985001-125015000_strand_+_gene_742, 59540 candidate probes\n",
      "--- 23122 probes passed check_dic selection.\n",
      "finish in 1.844s, 353 probes kept.\n",
      "-- check region:743 10:126985001-127015000_strand_+_gene_743, 58574 candidate probes\n",
      "--- 19516 probes passed check_dic selection.\n",
      "finish in 1.644s, 309 probes kept.\n",
      "-- check region:744 10:128985001-129015000_strand_+_gene_744, 57656 candidate probes\n",
      "--- 20176 probes passed check_dic selection.\n",
      "finish in 1.648s, 308 probes kept.\n",
      "-- check region:745 10:130985001-131015000_strand_+_gene_745, 59644 candidate probes\n",
      "--- 25898 probes passed check_dic selection.\n",
      "finish in 1.897s, 367 probes kept.\n",
      "-- check region:746 11:985001-1015000_strand_+_gene_746, 58442 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.944s, 0 probes kept.\n",
      "-- check region:747 11:2985001-3015000_strand_+_gene_747, 58292 candidate probes\n",
      "--- 655 probes passed check_dic selection.\n",
      "finish in 0.961s, 23 probes kept.\n",
      "-- check region:748 11:4985001-5015000_strand_+_gene_748, 59264 candidate probes\n",
      "--- 13069 probes passed check_dic selection.\n",
      "finish in 1.406s, 254 probes kept.\n",
      "-- check region:749 11:6985001-7015000_strand_+_gene_749, 59584 candidate probes\n",
      "--- 8405 probes passed check_dic selection.\n",
      "finish in 1.235s, 199 probes kept.\n",
      "-- check region:750 11:8985001-9015000_strand_+_gene_750, 57640 candidate probes\n",
      "--- 17277 probes passed check_dic selection.\n",
      "finish in 1.534s, 287 probes kept.\n",
      "-- check region:751 11:10985001-11015000_strand_+_gene_751, 59254 candidate probes\n",
      "--- 17511 probes passed check_dic selection.\n",
      "finish in 1.575s, 293 probes kept.\n",
      "-- check region:752 11:12985001-13015000_strand_+_gene_752, 59354 candidate probes\n",
      "--- 17723 probes passed check_dic selection.\n",
      "finish in 1.586s, 307 probes kept.\n",
      "-- check region:753 11:14985001-15015000_strand_+_gene_753, 58204 candidate probes\n",
      "--- 5896 probes passed check_dic selection.\n",
      "finish in 1.120s, 145 probes kept.\n",
      "-- check region:754 11:16985001-17015000_strand_+_gene_754, 58510 candidate probes\n",
      "--- 21750 probes passed check_dic selection.\n",
      "finish in 1.727s, 344 probes kept.\n",
      "-- check region:755 11:18985001-19015000_strand_+_gene_755, 56760 candidate probes\n",
      "--- 1584 probes passed check_dic selection.\n",
      "finish in 0.959s, 59 probes kept.\n",
      "-- check region:756 11:20985001-21015000_strand_+_gene_756, 59242 candidate probes\n",
      "--- 19074 probes passed check_dic selection.\n",
      "finish in 1.616s, 319 probes kept.\n",
      "-- check region:757 11:22985001-23015000_strand_+_gene_757, 55970 candidate probes\n",
      "--- 7886 probes passed check_dic selection.\n",
      "finish in 1.170s, 192 probes kept.\n",
      "-- check region:758 11:24985001-25015000_strand_+_gene_758, 55938 candidate probes\n",
      "--- 6178 probes passed check_dic selection.\n",
      "finish in 1.099s, 141 probes kept.\n",
      "-- check region:759 11:26985001-27015000_strand_+_gene_759, 59694 candidate probes\n",
      "--- 16300 probes passed check_dic selection.\n",
      "finish in 1.531s, 306 probes kept.\n",
      "-- check region:760 11:28985001-29015000_strand_+_gene_760, 59466 candidate probes\n",
      "--- 9687 probes passed check_dic selection.\n",
      "finish in 1.262s, 220 probes kept.\n",
      "-- check region:761 11:30985001-31015000_strand_+_gene_761, 59162 candidate probes\n",
      "--- 11361 probes passed check_dic selection.\n",
      "finish in 1.330s, 222 probes kept.\n",
      "-- check region:762 11:32985001-33015000_strand_+_gene_762, 57980 candidate probes\n",
      "--- 5871 probes passed check_dic selection.\n",
      "finish in 1.139s, 141 probes kept.\n",
      "-- check region:763 11:34985001-35015000_strand_+_gene_763, 59402 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 14029 probes passed check_dic selection.\n",
      "finish in 1.431s, 270 probes kept.\n",
      "-- check region:764 11:36985001-37015000_strand_+_gene_764, 59224 candidate probes\n",
      "--- 14174 probes passed check_dic selection.\n",
      "finish in 1.430s, 261 probes kept.\n",
      "-- check region:765 11:38985001-39015000_strand_+_gene_765, 56146 candidate probes\n",
      "--- 8819 probes passed check_dic selection.\n",
      "finish in 1.192s, 191 probes kept.\n",
      "-- check region:766 11:40985001-41015000_strand_+_gene_766, 59026 candidate probes\n",
      "--- 12936 probes passed check_dic selection.\n",
      "finish in 1.385s, 254 probes kept.\n",
      "-- check region:767 11:42985001-43015000_strand_+_gene_767, 52574 candidate probes\n",
      "--- 9651 probes passed check_dic selection.\n",
      "finish in 1.182s, 199 probes kept.\n",
      "-- check region:768 11:44985001-45015000_strand_+_gene_768, 55880 candidate probes\n",
      "--- 18220 probes passed check_dic selection.\n",
      "finish in 1.570s, 278 probes kept.\n",
      "-- check region:769 11:46985001-47015000_strand_+_gene_769, 54456 candidate probes\n",
      "--- 7673 probes passed check_dic selection.\n",
      "finish in 1.136s, 160 probes kept.\n",
      "-- check region:770 11:48985001-49015000_strand_+_gene_770, 58544 candidate probes\n",
      "--- 2150 probes passed check_dic selection.\n",
      "finish in 0.988s, 81 probes kept.\n",
      "-- check region:771 11:50985001-51015000_strand_+_gene_771, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:772 11:52985001-53015000_strand_+_gene_772, 17150 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.279s, 0 probes kept.\n",
      "-- check region:773 11:54985001-55015000_strand_+_gene_773, 58334 candidate probes\n",
      "--- 796 probes passed check_dic selection.\n",
      "finish in 0.953s, 46 probes kept.\n",
      "-- check region:774 11:56985001-57015000_strand_+_gene_774, 58692 candidate probes\n",
      "--- 7743 probes passed check_dic selection.\n",
      "finish in 1.207s, 202 probes kept.\n",
      "-- check region:775 11:58985001-59015000_strand_+_gene_775, 59228 candidate probes\n",
      "--- 7492 probes passed check_dic selection.\n",
      "finish in 1.212s, 202 probes kept.\n",
      "-- check region:776 11:60985001-61015000_strand_+_gene_776, 58014 candidate probes\n",
      "--- 20452 probes passed check_dic selection.\n",
      "finish in 1.665s, 317 probes kept.\n",
      "-- check region:777 11:62985001-63015000_strand_+_gene_777, 59040 candidate probes\n",
      "--- 20294 probes passed check_dic selection.\n",
      "finish in 1.711s, 373 probes kept.\n",
      "-- check region:778 11:64985001-65015000_strand_+_gene_778, 56550 candidate probes\n",
      "--- 15225 probes passed check_dic selection.\n",
      "finish in 1.466s, 252 probes kept.\n",
      "-- check region:779 11:66985001-67015000_strand_+_gene_779, 56500 candidate probes\n",
      "--- 11773 probes passed check_dic selection.\n",
      "finish in 1.336s, 208 probes kept.\n",
      "-- check region:780 11:68985001-69015000_strand_+_gene_780, 58790 candidate probes\n",
      "--- 21252 probes passed check_dic selection.\n",
      "finish in 1.742s, 349 probes kept.\n",
      "-- check region:781 11:70985001-71015000_strand_+_gene_781, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:782 11:72985001-73015000_strand_+_gene_782, 55200 candidate probes\n",
      "--- 10501 probes passed check_dic selection.\n",
      "finish in 1.241s, 206 probes kept.\n",
      "-- check region:783 11:74985001-75015000_strand_+_gene_783, 58832 candidate probes\n",
      "--- 15098 probes passed check_dic selection.\n",
      "finish in 1.489s, 286 probes kept.\n",
      "-- check region:784 11:76985001-77015000_strand_+_gene_784, 58832 candidate probes\n",
      "--- 8764 probes passed check_dic selection.\n",
      "finish in 1.237s, 196 probes kept.\n",
      "-- check region:785 11:78985001-79015000_strand_+_gene_785, 59296 candidate probes\n",
      "--- 17557 probes passed check_dic selection.\n",
      "finish in 1.573s, 295 probes kept.\n",
      "-- check region:786 11:80985001-81015000_strand_+_gene_786, 59696 candidate probes\n",
      "--- 11432 probes passed check_dic selection.\n",
      "finish in 1.335s, 234 probes kept.\n",
      "-- check region:787 11:82985001-83015000_strand_+_gene_787, 59090 candidate probes\n",
      "--- 16890 probes passed check_dic selection.\n",
      "finish in 1.546s, 293 probes kept.\n",
      "-- check region:788 11:84985001-85015000_strand_+_gene_788, 58364 candidate probes\n",
      "--- 8371 probes passed check_dic selection.\n",
      "finish in 1.204s, 182 probes kept.\n",
      "-- check region:789 11:86985001-87015000_strand_+_gene_789, 57600 candidate probes\n",
      "--- 10658 probes passed check_dic selection.\n",
      "finish in 1.302s, 209 probes kept.\n",
      "-- check region:790 11:88985001-89015000_strand_+_gene_790, 50500 candidate probes\n",
      "--- 2720 probes passed check_dic selection.\n",
      "finish in 0.884s, 91 probes kept.\n",
      "-- check region:791 11:90985001-91015000_strand_+_gene_791, 58256 candidate probes\n",
      "--- 6472 probes passed check_dic selection.\n",
      "finish in 1.125s, 164 probes kept.\n",
      "-- check region:792 11:92985001-93015000_strand_+_gene_792, 56370 candidate probes\n",
      "--- 8040 probes passed check_dic selection.\n",
      "finish in 1.175s, 168 probes kept.\n",
      "-- check region:793 11:94985001-95015000_strand_+_gene_793, 59377 candidate probes\n",
      "--- 11026 probes passed check_dic selection.\n",
      "finish in 1.337s, 231 probes kept.\n",
      "-- check region:794 11:96985001-97015000_strand_+_gene_794, 59492 candidate probes\n",
      "--- 15335 probes passed check_dic selection.\n",
      "finish in 1.488s, 298 probes kept.\n",
      "-- check region:795 11:98985001-99015000_strand_+_gene_795, 55680 candidate probes\n",
      "--- 5489 probes passed check_dic selection.\n",
      "finish in 1.060s, 126 probes kept.\n",
      "-- check region:796 11:100985001-101015000_strand_+_gene_796, 59050 candidate probes\n",
      "--- 15093 probes passed check_dic selection.\n",
      "finish in 1.477s, 289 probes kept.\n",
      "-- check region:797 11:102985001-103015000_strand_+_gene_797, 59512 candidate probes\n",
      "--- 12901 probes passed check_dic selection.\n",
      "finish in 1.409s, 249 probes kept.\n",
      "-- check region:798 11:104985001-105015000_strand_+_gene_798, 59742 candidate probes\n",
      "--- 10260 probes passed check_dic selection.\n",
      "finish in 1.297s, 244 probes kept.\n",
      "-- check region:799 11:106985001-107015000_strand_+_gene_799, 59180 candidate probes\n",
      "--- 16220 probes passed check_dic selection.\n",
      "finish in 1.506s, 289 probes kept.\n",
      "-- check region:800 11:108985001-109015000_strand_+_gene_800, 58707 candidate probes\n",
      "--- 10232 probes passed check_dic selection.\n",
      "finish in 1.300s, 196 probes kept.\n",
      "-- check region:801 11:110985001-111015000_strand_+_gene_801, 59620 candidate probes\n",
      "--- 17803 probes passed check_dic selection.\n",
      "finish in 1.589s, 320 probes kept.\n",
      "-- check region:802 11:112985001-113015000_strand_+_gene_802, 58526 candidate probes\n",
      "--- 13569 probes passed check_dic selection.\n",
      "finish in 1.399s, 257 probes kept.\n",
      "-- check region:803 11:114985001-115015000_strand_+_gene_803, 59880 candidate probes\n",
      "--- 17121 probes passed check_dic selection.\n",
      "finish in 1.576s, 289 probes kept.\n",
      "-- check region:804 11:116985001-117015000_strand_+_gene_804, 58776 candidate probes\n",
      "--- 11908 probes passed check_dic selection.\n",
      "finish in 1.359s, 220 probes kept.\n",
      "-- check region:805 11:118985001-119015000_strand_+_gene_805, 55404 candidate probes\n",
      "--- 7656 probes passed check_dic selection.\n",
      "finish in 1.183s, 168 probes kept.\n",
      "-- check region:806 11:120985001-121015000_strand_+_gene_806, 58954 candidate probes\n",
      "--- 15925 probes passed check_dic selection.\n",
      "finish in 1.513s, 291 probes kept.\n",
      "-- check region:807 11:122985001-123015000_strand_+_gene_807, 57380 candidate probes\n",
      "--- 12288 probes passed check_dic selection.\n",
      "finish in 1.348s, 233 probes kept.\n",
      "-- check region:808 11:124985001-125015000_strand_+_gene_808, 59206 candidate probes\n",
      "--- 8711 probes passed check_dic selection.\n",
      "finish in 1.240s, 197 probes kept.\n",
      "-- check region:809 11:126985001-127015000_strand_+_gene_809, 59906 candidate probes\n",
      "--- 25728 probes passed check_dic selection.\n",
      "finish in 1.927s, 405 probes kept.\n",
      "-- check region:810 11:128985001-129015000_strand_+_gene_810, 58772 candidate probes\n",
      "--- 6511 probes passed check_dic selection.\n",
      "finish in 1.145s, 160 probes kept.\n",
      "-- check region:811 11:130985001-131015000_strand_+_gene_811, 53680 candidate probes\n",
      "--- 13312 probes passed check_dic selection.\n",
      "finish in 1.327s, 234 probes kept.\n",
      "-- check region:812 11:132985001-133015000_strand_+_gene_812, 59558 candidate probes\n",
      "--- 23353 probes passed check_dic selection.\n",
      "finish in 1.813s, 358 probes kept.\n",
      "-- check region:813 12:985001-1015000_strand_+_gene_813, 57412 candidate probes\n",
      "--- 9561 probes passed check_dic selection.\n",
      "finish in 1.252s, 187 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:814 12:2985001-3015000_strand_+_gene_814, 57616 candidate probes\n",
      "--- 18014 probes passed check_dic selection.\n",
      "finish in 1.584s, 275 probes kept.\n",
      "-- check region:815 12:4985001-5015000_strand_+_gene_815, 58606 candidate probes\n",
      "--- 21198 probes passed check_dic selection.\n",
      "finish in 1.711s, 332 probes kept.\n",
      "-- check region:816 12:6985001-7015000_strand_+_gene_816, 55888 candidate probes\n",
      "--- 11805 probes passed check_dic selection.\n",
      "finish in 1.324s, 209 probes kept.\n",
      "-- check region:817 12:8985001-9015000_strand_+_gene_817, 58298 candidate probes\n",
      "--- 9355 probes passed check_dic selection.\n",
      "finish in 1.258s, 204 probes kept.\n",
      "-- check region:818 12:10985001-11015000_strand_+_gene_818, 59662 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.925s, 0 probes kept.\n",
      "-- check region:819 12:12985001-13015000_strand_+_gene_819, 59104 candidate probes\n",
      "--- 14508 probes passed check_dic selection.\n",
      "finish in 1.461s, 268 probes kept.\n",
      "-- check region:820 12:14985001-15015000_strand_+_gene_820, 59774 candidate probes\n",
      "--- 13800 probes passed check_dic selection.\n",
      "finish in 1.434s, 255 probes kept.\n",
      "-- check region:821 12:16985001-17015000_strand_+_gene_821, 53870 candidate probes\n",
      "--- 5701 probes passed check_dic selection.\n",
      "finish in 1.059s, 142 probes kept.\n",
      "-- check region:822 12:18985001-19015000_strand_+_gene_822, 58762 candidate probes\n",
      "--- 14919 probes passed check_dic selection.\n",
      "finish in 1.460s, 257 probes kept.\n",
      "-- check region:823 12:20985001-21015000_strand_+_gene_823, 58088 candidate probes\n",
      "--- 5722 probes passed check_dic selection.\n",
      "finish in 1.110s, 129 probes kept.\n",
      "-- check region:824 12:22985001-23015000_strand_+_gene_824, 59410 candidate probes\n",
      "--- 15742 probes passed check_dic selection.\n",
      "finish in 1.514s, 288 probes kept.\n",
      "-- check region:825 12:24985001-25015000_strand_+_gene_825, 58612 candidate probes\n",
      "--- 9910 probes passed check_dic selection.\n",
      "finish in 1.285s, 225 probes kept.\n",
      "-- check region:826 12:26985001-27015000_strand_+_gene_826, 57806 candidate probes\n",
      "--- 13023 probes passed check_dic selection.\n",
      "finish in 1.383s, 263 probes kept.\n",
      "-- check region:827 12:28985001-29015000_strand_+_gene_827, 59638 candidate probes\n",
      "--- 15388 probes passed check_dic selection.\n",
      "finish in 1.502s, 283 probes kept.\n",
      "-- check region:828 12:30985001-31015000_strand_+_gene_828, 59740 candidate probes\n",
      "--- 22934 probes passed check_dic selection.\n",
      "finish in 1.829s, 377 probes kept.\n",
      "-- check region:829 12:32985001-33015000_strand_+_gene_829, 57316 candidate probes\n",
      "--- 12425 probes passed check_dic selection.\n",
      "finish in 1.361s, 237 probes kept.\n",
      "-- check region:830 12:34985001-35015000_strand_+_gene_830, 10682 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.172s, 0 probes kept.\n",
      "-- check region:831 12:36985001-37015000_strand_+_gene_831, 19554 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.314s, 0 probes kept.\n",
      "-- check region:832 12:38985001-39015000_strand_+_gene_832, 54924 candidate probes\n",
      "--- 7497 probes passed check_dic selection.\n",
      "finish in 1.122s, 168 probes kept.\n",
      "-- check region:833 12:40985001-41015000_strand_+_gene_833, 58984 candidate probes\n",
      "--- 9233 probes passed check_dic selection.\n",
      "finish in 1.254s, 204 probes kept.\n",
      "-- check region:834 12:42985001-43015000_strand_+_gene_834, 58858 candidate probes\n",
      "--- 17164 probes passed check_dic selection.\n",
      "finish in 1.549s, 310 probes kept.\n",
      "-- check region:835 12:44985001-45015000_strand_+_gene_835, 58648 candidate probes\n",
      "--- 11650 probes passed check_dic selection.\n",
      "finish in 1.335s, 236 probes kept.\n",
      "-- check region:836 12:46985001-47015000_strand_+_gene_836, 52818 candidate probes\n",
      "--- 7349 probes passed check_dic selection.\n",
      "finish in 1.087s, 164 probes kept.\n",
      "-- check region:837 12:48985001-49015000_strand_+_gene_837, 58254 candidate probes\n",
      "--- 17667 probes passed check_dic selection.\n",
      "finish in 1.594s, 302 probes kept.\n",
      "-- check region:838 12:50985001-51015000_strand_+_gene_838, 57504 candidate probes\n",
      "--- 11374 probes passed check_dic selection.\n",
      "finish in 1.342s, 226 probes kept.\n",
      "-- check region:839 12:52985001-53015000_strand_+_gene_839, 55362 candidate probes\n",
      "--- 10306 probes passed check_dic selection.\n",
      "finish in 1.263s, 194 probes kept.\n",
      "-- check region:840 12:54985001-55015000_strand_+_gene_840, 59194 candidate probes\n",
      "--- 14043 probes passed check_dic selection.\n",
      "finish in 1.449s, 264 probes kept.\n",
      "-- check region:841 12:56985001-57015000_strand_+_gene_841, 59618 candidate probes\n",
      "--- 20008 probes passed check_dic selection.\n",
      "finish in 1.706s, 373 probes kept.\n",
      "-- check region:842 12:58985001-59015000_strand_+_gene_842, 56162 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.872s, 0 probes kept.\n",
      "-- check region:843 12:60985001-61015000_strand_+_gene_843, 53988 candidate probes\n",
      "--- 6448 probes passed check_dic selection.\n",
      "finish in 1.064s, 148 probes kept.\n",
      "-- check region:844 12:62985001-63015000_strand_+_gene_844, 59420 candidate probes\n",
      "--- 12253 probes passed check_dic selection.\n",
      "finish in 1.368s, 256 probes kept.\n",
      "-- check region:845 12:64985001-65015000_strand_+_gene_845, 59110 candidate probes\n",
      "--- 16355 probes passed check_dic selection.\n",
      "finish in 1.524s, 300 probes kept.\n",
      "-- check region:846 12:66985001-67015000_strand_+_gene_846, 58866 candidate probes\n",
      "--- 13819 probes passed check_dic selection.\n",
      "finish in 1.417s, 261 probes kept.\n",
      "-- check region:847 12:68985001-69015000_strand_+_gene_847, 52944 candidate probes\n",
      "--- 8607 probes passed check_dic selection.\n",
      "finish in 1.157s, 174 probes kept.\n",
      "-- check region:848 12:70985001-71015000_strand_+_gene_848, 58702 candidate probes\n",
      "--- 16329 probes passed check_dic selection.\n",
      "finish in 1.502s, 294 probes kept.\n",
      "-- check region:849 12:72985001-73015000_strand_+_gene_849, 58162 candidate probes\n",
      "--- 5079 probes passed check_dic selection.\n",
      "finish in 1.096s, 123 probes kept.\n",
      "-- check region:850 12:74985001-75015000_strand_+_gene_850, 55610 candidate probes\n",
      "--- 7859 probes passed check_dic selection.\n",
      "finish in 1.147s, 181 probes kept.\n",
      "-- check region:851 12:76985001-77015000_strand_+_gene_851, 59040 candidate probes\n",
      "--- 12968 probes passed check_dic selection.\n",
      "finish in 1.392s, 241 probes kept.\n",
      "-- check region:852 12:78985001-79015000_strand_+_gene_852, 58806 candidate probes\n",
      "--- 14154 probes passed check_dic selection.\n",
      "finish in 1.421s, 279 probes kept.\n",
      "-- check region:853 12:80985001-81015000_strand_+_gene_853, 56986 candidate probes\n",
      "--- 9669 probes passed check_dic selection.\n",
      "finish in 1.232s, 194 probes kept.\n",
      "-- check region:854 12:82985001-83015000_strand_+_gene_854, 59092 candidate probes\n",
      "--- 9228 probes passed check_dic selection.\n",
      "finish in 1.258s, 215 probes kept.\n",
      "-- check region:855 12:84985001-85015000_strand_+_gene_855, 59566 candidate probes\n",
      "--- 8968 probes passed check_dic selection.\n",
      "finish in 1.245s, 199 probes kept.\n",
      "-- check region:856 12:86985001-87015000_strand_+_gene_856, 59418 candidate probes\n",
      "--- 6328 probes passed check_dic selection.\n",
      "finish in 1.142s, 163 probes kept.\n",
      "-- check region:857 12:88985001-89015000_strand_+_gene_857, 59386 candidate probes\n",
      "--- 15567 probes passed check_dic selection.\n",
      "finish in 1.494s, 300 probes kept.\n",
      "-- check region:858 12:90985001-91015000_strand_+_gene_858, 59488 candidate probes\n",
      "--- 11415 probes passed check_dic selection.\n",
      "finish in 1.332s, 252 probes kept.\n",
      "-- check region:859 12:92985001-93015000_strand_+_gene_859, 58844 candidate probes\n",
      "--- 12656 probes passed check_dic selection.\n",
      "finish in 1.392s, 240 probes kept.\n",
      "-- check region:860 12:94985001-95015000_strand_+_gene_860, 58144 candidate probes\n",
      "--- 6965 probes passed check_dic selection.\n",
      "finish in 1.173s, 144 probes kept.\n",
      "-- check region:861 12:96985001-97015000_strand_+_gene_861, 59646 candidate probes\n",
      "--- 19339 probes passed check_dic selection.\n",
      "finish in 1.637s, 333 probes kept.\n",
      "-- check region:862 12:98985001-99015000_strand_+_gene_862, 59154 candidate probes\n",
      "--- 12494 probes passed check_dic selection.\n",
      "finish in 1.376s, 233 probes kept.\n",
      "-- check region:863 12:100985001-101015000_strand_+_gene_863, 59114 candidate probes\n",
      "--- 16091 probes passed check_dic selection.\n",
      "finish in 1.520s, 282 probes kept.\n",
      "-- check region:864 12:102985001-103015000_strand_+_gene_864, 59516 candidate probes\n",
      "--- 13166 probes passed check_dic selection.\n",
      "finish in 1.410s, 250 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:865 12:104985001-105015000_strand_+_gene_865, 58218 candidate probes\n",
      "--- 8145 probes passed check_dic selection.\n",
      "finish in 1.231s, 223 probes kept.\n",
      "-- check region:866 12:106985001-107015000_strand_+_gene_866, 58960 candidate probes\n",
      "--- 9151 probes passed check_dic selection.\n",
      "finish in 1.271s, 216 probes kept.\n",
      "-- check region:867 12:108985001-109015000_strand_+_gene_867, 49866 candidate probes\n",
      "--- 9620 probes passed check_dic selection.\n",
      "finish in 1.151s, 164 probes kept.\n",
      "-- check region:868 12:110985001-111015000_strand_+_gene_868, 58634 candidate probes\n",
      "--- 17830 probes passed check_dic selection.\n",
      "finish in 1.591s, 289 probes kept.\n",
      "-- check region:869 12:112985001-113015000_strand_+_gene_869, 59554 candidate probes\n",
      "--- 15892 probes passed check_dic selection.\n",
      "finish in 1.538s, 301 probes kept.\n",
      "-- check region:870 12:114985001-115015000_strand_+_gene_870, 59480 candidate probes\n",
      "--- 20547 probes passed check_dic selection.\n",
      "finish in 1.683s, 324 probes kept.\n",
      "-- check region:871 12:116985001-117015000_strand_+_gene_871, 59214 candidate probes\n",
      "--- 21254 probes passed check_dic selection.\n",
      "finish in 1.710s, 329 probes kept.\n",
      "-- check region:872 12:118985001-119015000_strand_+_gene_872, 59702 candidate probes\n",
      "--- 21898 probes passed check_dic selection.\n",
      "finish in 1.766s, 360 probes kept.\n",
      "-- check region:873 12:120985001-121015000_strand_+_gene_873, 57502 candidate probes\n",
      "--- 13716 probes passed check_dic selection.\n",
      "finish in 1.449s, 265 probes kept.\n",
      "-- check region:874 12:122985001-123015000_strand_+_gene_874, 59552 candidate probes\n",
      "--- 23039 probes passed check_dic selection.\n",
      "finish in 1.828s, 370 probes kept.\n",
      "-- check region:875 12:124985001-125015000_strand_+_gene_875, 57088 candidate probes\n",
      "--- 13122 probes passed check_dic selection.\n",
      "finish in 1.377s, 212 probes kept.\n",
      "-- check region:876 12:126985001-127015000_strand_+_gene_876, 58578 candidate probes\n",
      "--- 14792 probes passed check_dic selection.\n",
      "finish in 1.456s, 262 probes kept.\n",
      "-- check region:877 12:128985001-129015000_strand_+_gene_877, 57628 candidate probes\n",
      "--- 16059 probes passed check_dic selection.\n",
      "finish in 1.501s, 268 probes kept.\n",
      "-- check region:878 12:130985001-131015000_strand_+_gene_878, 57340 candidate probes\n",
      "--- 24179 probes passed check_dic selection.\n",
      "finish in 1.797s, 347 probes kept.\n",
      "-- check region:879 13:985001-1015000_strand_+_gene_879, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.002s, 0 probes kept.\n",
      "-- check region:880 13:2985001-3015000_strand_+_gene_880, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:881 13:4985001-5015000_strand_+_gene_881, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:882 13:6985001-7015000_strand_+_gene_882, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:883 13:8985001-9015000_strand_+_gene_883, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:884 13:10985001-11015000_strand_+_gene_884, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:885 13:12985001-13015000_strand_+_gene_885, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:886 13:14985001-15015000_strand_+_gene_886, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:887 13:16985001-17015000_strand_+_gene_887, 18844 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.303s, 0 probes kept.\n",
      "-- check region:888 13:18985001-19015000_strand_+_gene_888, 57388 candidate probes\n",
      "--- 10784 probes passed check_dic selection.\n",
      "finish in 1.315s, 215 probes kept.\n",
      "-- check region:889 13:20985001-21015000_strand_+_gene_889, 59070 candidate probes\n",
      "--- 17426 probes passed check_dic selection.\n",
      "finish in 1.570s, 279 probes kept.\n",
      "-- check region:890 13:22985001-23015000_strand_+_gene_890, 57672 candidate probes\n",
      "--- 10204 probes passed check_dic selection.\n",
      "finish in 1.276s, 205 probes kept.\n",
      "-- check region:891 13:24985001-25015000_strand_+_gene_891, 56834 candidate probes\n",
      "--- 7417 probes passed check_dic selection.\n",
      "finish in 1.180s, 169 probes kept.\n",
      "-- check region:892 13:26985001-27015000_strand_+_gene_892, 59168 candidate probes\n",
      "--- 20167 probes passed check_dic selection.\n",
      "finish in 1.680s, 314 probes kept.\n",
      "-- check region:893 13:28985001-29015000_strand_+_gene_893, 55352 candidate probes\n",
      "--- 15305 probes passed check_dic selection.\n",
      "finish in 1.438s, 252 probes kept.\n",
      "-- check region:894 13:30985001-31015000_strand_+_gene_894, 58262 candidate probes\n",
      "--- 19787 probes passed check_dic selection.\n",
      "finish in 1.630s, 308 probes kept.\n",
      "-- check region:895 13:32985001-33015000_strand_+_gene_895, 59404 candidate probes\n",
      "--- 16427 probes passed check_dic selection.\n",
      "finish in 1.533s, 294 probes kept.\n",
      "-- check region:896 13:34985001-35015000_strand_+_gene_896, 58196 candidate probes\n",
      "--- 7400 probes passed check_dic selection.\n",
      "finish in 1.184s, 158 probes kept.\n",
      "-- check region:897 13:36985001-37015000_strand_+_gene_897, 59138 candidate probes\n",
      "--- 14687 probes passed check_dic selection.\n",
      "finish in 1.464s, 286 probes kept.\n",
      "-- check region:898 13:38985001-39015000_strand_+_gene_898, 49564 candidate probes\n",
      "--- 7616 probes passed check_dic selection.\n",
      "finish in 1.057s, 182 probes kept.\n",
      "-- check region:899 13:40985001-41015000_strand_+_gene_899, 58998 candidate probes\n",
      "--- 9082 probes passed check_dic selection.\n",
      "finish in 1.243s, 211 probes kept.\n",
      "-- check region:900 13:42985001-43015000_strand_+_gene_900, 58270 candidate probes\n",
      "--- 13735 probes passed check_dic selection.\n",
      "finish in 1.423s, 251 probes kept.\n",
      "-- check region:901 13:44985001-45015000_strand_+_gene_901, 58374 candidate probes\n",
      "--- 11458 probes passed check_dic selection.\n",
      "finish in 1.338s, 230 probes kept.\n",
      "-- check region:902 13:46985001-47015000_strand_+_gene_902, 59684 candidate probes\n",
      "--- 13844 probes passed check_dic selection.\n",
      "finish in 1.439s, 271 probes kept.\n",
      "-- check region:903 13:48985001-49015000_strand_+_gene_903, 56732 candidate probes\n",
      "--- 5572 probes passed check_dic selection.\n",
      "finish in 1.073s, 144 probes kept.\n",
      "-- check region:904 13:50985001-51015000_strand_+_gene_904, 55228 candidate probes\n",
      "--- 21081 probes passed check_dic selection.\n",
      "finish in 1.645s, 320 probes kept.\n",
      "-- check region:905 13:52985001-53015000_strand_+_gene_905, 59084 candidate probes\n",
      "--- 18729 probes passed check_dic selection.\n",
      "finish in 1.621s, 311 probes kept.\n",
      "-- check region:906 13:54985001-55015000_strand_+_gene_906, 59224 candidate probes\n",
      "--- 7730 probes passed check_dic selection.\n",
      "finish in 1.198s, 194 probes kept.\n",
      "-- check region:907 13:56985001-57015000_strand_+_gene_907, 55990 candidate probes\n",
      "--- 5812 probes passed check_dic selection.\n",
      "finish in 1.061s, 143 probes kept.\n",
      "-- check region:908 13:58985001-59015000_strand_+_gene_908, 58968 candidate probes\n",
      "--- 11075 probes passed check_dic selection.\n",
      "finish in 1.317s, 216 probes kept.\n",
      "-- check region:909 13:60985001-61015000_strand_+_gene_909, 59142 candidate probes\n",
      "--- 8744 probes passed check_dic selection.\n",
      "finish in 1.243s, 181 probes kept.\n",
      "-- check region:910 13:62985001-63015000_strand_+_gene_910, 59572 candidate probes\n",
      "--- 6135 probes passed check_dic selection.\n",
      "finish in 1.148s, 155 probes kept.\n",
      "-- check region:911 13:64985001-65015000_strand_+_gene_911, 59538 candidate probes\n",
      "--- 8840 probes passed check_dic selection.\n",
      "finish in 1.225s, 187 probes kept.\n",
      "-- check region:912 13:66985001-67015000_strand_+_gene_912, 58460 candidate probes\n",
      "--- 11459 probes passed check_dic selection.\n",
      "finish in 1.312s, 230 probes kept.\n",
      "-- check region:913 13:68985001-69015000_strand_+_gene_913, 59516 candidate probes\n",
      "--- 6367 probes passed check_dic selection.\n",
      "finish in 1.144s, 169 probes kept.\n",
      "-- check region:914 13:70985001-71015000_strand_+_gene_914, 59698 candidate probes\n",
      "--- 8288 probes passed check_dic selection.\n",
      "finish in 1.214s, 176 probes kept.\n",
      "-- check region:915 13:72985001-73015000_strand_+_gene_915, 56930 candidate probes\n",
      "--- 7842 probes passed check_dic selection.\n",
      "finish in 1.177s, 166 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:916 13:74985001-75015000_strand_+_gene_916, 58368 candidate probes\n",
      "--- 12510 probes passed check_dic selection.\n",
      "finish in 1.363s, 250 probes kept.\n",
      "-- check region:917 13:76985001-77015000_strand_+_gene_917, 58920 candidate probes\n",
      "--- 8109 probes passed check_dic selection.\n",
      "finish in 1.261s, 250 probes kept.\n",
      "-- check region:918 13:78985001-79015000_strand_+_gene_918, 59502 candidate probes\n",
      "--- 13807 probes passed check_dic selection.\n",
      "finish in 1.423s, 259 probes kept.\n",
      "-- check region:919 13:80985001-81015000_strand_+_gene_919, 58502 candidate probes\n",
      "--- 5788 probes passed check_dic selection.\n",
      "finish in 1.127s, 157 probes kept.\n",
      "-- check region:920 13:82985001-83015000_strand_+_gene_920, 58926 candidate probes\n",
      "--- 8356 probes passed check_dic selection.\n",
      "finish in 1.198s, 187 probes kept.\n",
      "-- check region:921 13:84985001-85015000_strand_+_gene_921, 59250 candidate probes\n",
      "--- 7460 probes passed check_dic selection.\n",
      "finish in 1.172s, 173 probes kept.\n",
      "-- check region:922 13:86985001-87015000_strand_+_gene_922, 59534 candidate probes\n",
      "--- 7916 probes passed check_dic selection.\n",
      "finish in 1.199s, 174 probes kept.\n",
      "-- check region:923 13:88985001-89015000_strand_+_gene_923, 59682 candidate probes\n",
      "--- 5132 probes passed check_dic selection.\n",
      "finish in 1.090s, 141 probes kept.\n",
      "-- check region:924 13:90985001-91015000_strand_+_gene_924, 59060 candidate probes\n",
      "--- 11831 probes passed check_dic selection.\n",
      "finish in 1.342s, 236 probes kept.\n",
      "-- check region:925 13:92985001-93015000_strand_+_gene_925, 59556 candidate probes\n",
      "--- 10512 probes passed check_dic selection.\n",
      "finish in 1.302s, 214 probes kept.\n",
      "-- check region:926 13:94985001-95015000_strand_+_gene_926, 59078 candidate probes\n",
      "--- 16807 probes passed check_dic selection.\n",
      "finish in 1.537s, 293 probes kept.\n",
      "-- check region:927 13:96985001-97015000_strand_+_gene_927, 59754 candidate probes\n",
      "--- 17233 probes passed check_dic selection.\n",
      "finish in 1.569s, 314 probes kept.\n",
      "-- check region:928 13:98985001-99015000_strand_+_gene_928, 59354 candidate probes\n",
      "--- 20029 probes passed check_dic selection.\n",
      "finish in 1.667s, 334 probes kept.\n",
      "-- check region:929 13:100985001-101015000_strand_+_gene_929, 58822 candidate probes\n",
      "--- 15102 probes passed check_dic selection.\n",
      "finish in 1.468s, 261 probes kept.\n",
      "-- check region:930 13:102985001-103015000_strand_+_gene_930, 59748 candidate probes\n",
      "--- 14654 probes passed check_dic selection.\n",
      "finish in 1.469s, 266 probes kept.\n",
      "-- check region:931 13:104985001-105015000_strand_+_gene_931, 59566 candidate probes\n",
      "--- 12651 probes passed check_dic selection.\n",
      "finish in 1.373s, 249 probes kept.\n",
      "-- check region:932 13:106985001-107015000_strand_+_gene_932, 59598 candidate probes\n",
      "--- 17996 probes passed check_dic selection.\n",
      "finish in 1.583s, 297 probes kept.\n",
      "-- check region:933 13:108985001-109015000_strand_+_gene_933, 56530 candidate probes\n",
      "--- 12977 probes passed check_dic selection.\n",
      "finish in 1.350s, 224 probes kept.\n",
      "-- check region:934 13:110985001-111015000_strand_+_gene_934, 59304 candidate probes\n",
      "--- 11902 probes passed check_dic selection.\n",
      "finish in 1.363s, 209 probes kept.\n",
      "-- check region:935 13:112985001-113015000_strand_+_gene_935, 57404 candidate probes\n",
      "--- 32292 probes passed check_dic selection.\n",
      "finish in 2.148s, 426 probes kept.\n",
      "-- check region:936 14:735001-765000_strand_+_gene_936, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.002s, 0 probes kept.\n",
      "-- check region:937 14:2235001-2265000_strand_+_gene_937, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:938 14:3735001-3765000_strand_+_gene_938, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:939 14:5235001-5265000_strand_+_gene_939, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:940 14:6735001-6765000_strand_+_gene_940, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:941 14:8235001-8265000_strand_+_gene_941, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:942 14:9735001-9765000_strand_+_gene_942, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:943 14:11235001-11265000_strand_+_gene_943, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:944 14:12735001-12765000_strand_+_gene_944, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:945 14:14235001-14265000_strand_+_gene_945, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:946 14:15735001-15765000_strand_+_gene_946, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:947 14:17235001-17265000_strand_+_gene_947, 18830 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.300s, 0 probes kept.\n",
      "-- check region:948 14:18735001-18765000_strand_+_gene_948, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:949 14:20235001-20265000_strand_+_gene_949, 55628 candidate probes\n",
      "--- 6152 probes passed check_dic selection.\n",
      "finish in 1.102s, 160 probes kept.\n",
      "-- check region:950 14:21735001-21765000_strand_+_gene_950, 54778 candidate probes\n",
      "--- 9054 probes passed check_dic selection.\n",
      "finish in 1.193s, 178 probes kept.\n",
      "-- check region:951 14:23235001-23265000_strand_+_gene_951, 58102 candidate probes\n",
      "--- 8736 probes passed check_dic selection.\n",
      "finish in 1.235s, 188 probes kept.\n",
      "-- check region:952 14:24735001-24765000_strand_+_gene_952, 54004 candidate probes\n",
      "--- 10023 probes passed check_dic selection.\n",
      "finish in 1.218s, 190 probes kept.\n",
      "-- check region:953 14:26235001-26265000_strand_+_gene_953, 59552 candidate probes\n",
      "--- 12971 probes passed check_dic selection.\n",
      "finish in 1.395s, 245 probes kept.\n",
      "-- check region:954 14:27735001-27765000_strand_+_gene_954, 59542 candidate probes\n",
      "--- 2627 probes passed check_dic selection.\n",
      "finish in 1.019s, 90 probes kept.\n",
      "-- check region:955 14:29235001-29265000_strand_+_gene_955, 56294 candidate probes\n",
      "--- 6829 probes passed check_dic selection.\n",
      "finish in 1.117s, 174 probes kept.\n",
      "-- check region:956 14:30735001-30765000_strand_+_gene_956, 58516 candidate probes\n",
      "--- 16637 probes passed check_dic selection.\n",
      "finish in 1.518s, 265 probes kept.\n",
      "-- check region:957 14:32235001-32265000_strand_+_gene_957, 54616 candidate probes\n",
      "--- 13878 probes passed check_dic selection.\n",
      "finish in 1.389s, 258 probes kept.\n",
      "-- check region:958 14:33735001-33765000_strand_+_gene_958, 58938 candidate probes\n",
      "--- 19137 probes passed check_dic selection.\n",
      "finish in 1.620s, 328 probes kept.\n",
      "-- check region:959 14:35235001-35265000_strand_+_gene_959, 58264 candidate probes\n",
      "--- 10905 probes passed check_dic selection.\n",
      "finish in 1.314s, 213 probes kept.\n",
      "-- check region:960 14:36735001-36765000_strand_+_gene_960, 58292 candidate probes\n",
      "--- 17287 probes passed check_dic selection.\n",
      "finish in 1.546s, 279 probes kept.\n",
      "-- check region:961 14:38235001-38265000_strand_+_gene_961, 59362 candidate probes\n",
      "--- 13119 probes passed check_dic selection.\n",
      "finish in 1.411s, 237 probes kept.\n",
      "-- check region:962 14:39735001-39765000_strand_+_gene_962, 59388 candidate probes\n",
      "--- 10430 probes passed check_dic selection.\n",
      "finish in 1.297s, 226 probes kept.\n",
      "-- check region:963 14:41235001-41265000_strand_+_gene_963, 59582 candidate probes\n",
      "--- 9588 probes passed check_dic selection.\n",
      "finish in 1.275s, 195 probes kept.\n",
      "-- check region:964 14:42735001-42765000_strand_+_gene_964, 59552 candidate probes\n",
      "--- 5992 probes passed check_dic selection.\n",
      "finish in 1.130s, 149 probes kept.\n",
      "-- check region:965 14:44235001-44265000_strand_+_gene_965, 59194 candidate probes\n",
      "--- 8340 probes passed check_dic selection.\n",
      "finish in 1.210s, 191 probes kept.\n",
      "-- check region:966 14:45735001-45765000_strand_+_gene_966, 58030 candidate probes\n",
      "--- 6482 probes passed check_dic selection.\n",
      "finish in 1.140s, 161 probes kept.\n",
      "-- check region:967 14:47235001-47265000_strand_+_gene_967, 59204 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 10562 probes passed check_dic selection.\n",
      "finish in 1.316s, 229 probes kept.\n",
      "-- check region:968 14:48735001-48765000_strand_+_gene_968, 59742 candidate probes\n",
      "--- 11408 probes passed check_dic selection.\n",
      "finish in 1.343s, 228 probes kept.\n",
      "-- check region:969 14:50235001-50265000_strand_+_gene_969, 56888 candidate probes\n",
      "--- 6524 probes passed check_dic selection.\n",
      "finish in 1.136s, 156 probes kept.\n",
      "-- check region:970 14:51735001-51765000_strand_+_gene_970, 59320 candidate probes\n",
      "--- 3847 probes passed check_dic selection.\n",
      "finish in 1.096s, 110 probes kept.\n",
      "-- check region:971 14:53235001-53265000_strand_+_gene_971, 59384 candidate probes\n",
      "--- 9337 probes passed check_dic selection.\n",
      "finish in 1.271s, 173 probes kept.\n",
      "-- check region:972 14:54735001-54765000_strand_+_gene_972, 59426 candidate probes\n",
      "--- 26923 probes passed check_dic selection.\n",
      "finish in 1.954s, 392 probes kept.\n",
      "-- check region:973 14:56235001-56265000_strand_+_gene_973, 59256 candidate probes\n",
      "--- 15202 probes passed check_dic selection.\n",
      "finish in 1.473s, 270 probes kept.\n",
      "-- check region:974 14:57735001-57765000_strand_+_gene_974, 56600 candidate probes\n",
      "--- 11856 probes passed check_dic selection.\n",
      "finish in 1.318s, 209 probes kept.\n",
      "-- check region:975 14:59235001-59265000_strand_+_gene_975, 58950 candidate probes\n",
      "--- 17523 probes passed check_dic selection.\n",
      "finish in 1.563s, 315 probes kept.\n",
      "-- check region:976 14:60735001-60765000_strand_+_gene_976, 58238 candidate probes\n",
      "--- 8622 probes passed check_dic selection.\n",
      "finish in 1.215s, 194 probes kept.\n",
      "-- check region:977 14:62235001-62265000_strand_+_gene_977, 59486 candidate probes\n",
      "--- 16162 probes passed check_dic selection.\n",
      "finish in 1.525s, 297 probes kept.\n",
      "-- check region:978 14:63735001-63765000_strand_+_gene_978, 55934 candidate probes\n",
      "--- 5197 probes passed check_dic selection.\n",
      "finish in 1.072s, 130 probes kept.\n",
      "-- check region:979 14:65235001-65265000_strand_+_gene_979, 57636 candidate probes\n",
      "--- 14862 probes passed check_dic selection.\n",
      "finish in 1.463s, 255 probes kept.\n",
      "-- check region:980 14:66735001-66765000_strand_+_gene_980, 58796 candidate probes\n",
      "--- 6549 probes passed check_dic selection.\n",
      "finish in 1.144s, 173 probes kept.\n",
      "-- check region:981 14:68235001-68265000_strand_+_gene_981, 59020 candidate probes\n",
      "--- 17812 probes passed check_dic selection.\n",
      "finish in 1.576s, 302 probes kept.\n",
      "-- check region:982 14:69735001-69765000_strand_+_gene_982, 57262 candidate probes\n",
      "--- 9839 probes passed check_dic selection.\n",
      "finish in 1.265s, 171 probes kept.\n",
      "-- check region:983 14:71235001-71265000_strand_+_gene_983, 57784 candidate probes\n",
      "--- 14720 probes passed check_dic selection.\n",
      "finish in 1.454s, 236 probes kept.\n",
      "-- check region:984 14:72735001-72765000_strand_+_gene_984, 59518 candidate probes\n",
      "--- 17910 probes passed check_dic selection.\n",
      "finish in 1.606s, 269 probes kept.\n",
      "-- check region:985 14:74235001-74265000_strand_+_gene_985, 58210 candidate probes\n",
      "--- 23978 probes passed check_dic selection.\n",
      "finish in 1.821s, 342 probes kept.\n",
      "-- check region:986 14:75735001-75765000_strand_+_gene_986, 59016 candidate probes\n",
      "--- 11477 probes passed check_dic selection.\n",
      "finish in 1.337s, 233 probes kept.\n",
      "-- check region:987 14:77235001-77265000_strand_+_gene_987, 59074 candidate probes\n",
      "--- 24179 probes passed check_dic selection.\n",
      "finish in 1.855s, 374 probes kept.\n",
      "-- check region:988 14:78735001-78765000_strand_+_gene_988, 59596 candidate probes\n",
      "--- 20464 probes passed check_dic selection.\n",
      "finish in 1.683s, 348 probes kept.\n",
      "-- check region:989 14:80235001-80265000_strand_+_gene_989, 55414 candidate probes\n",
      "--- 7318 probes passed check_dic selection.\n",
      "finish in 1.131s, 154 probes kept.\n",
      "-- check region:990 14:81735001-81765000_strand_+_gene_990, 59612 candidate probes\n",
      "--- 13638 probes passed check_dic selection.\n",
      "finish in 1.420s, 281 probes kept.\n",
      "-- check region:991 14:83235001-83265000_strand_+_gene_991, 59444 candidate probes\n",
      "--- 11090 probes passed check_dic selection.\n",
      "finish in 1.320s, 247 probes kept.\n",
      "-- check region:992 14:84735001-84765000_strand_+_gene_992, 59438 candidate probes\n",
      "--- 11365 probes passed check_dic selection.\n",
      "finish in 1.333s, 237 probes kept.\n",
      "-- check region:993 14:86235001-86265000_strand_+_gene_993, 56026 candidate probes\n",
      "--- 7062 probes passed check_dic selection.\n",
      "finish in 1.112s, 153 probes kept.\n",
      "-- check region:994 14:87735001-87765000_strand_+_gene_994, 59232 candidate probes\n",
      "--- 15416 probes passed check_dic selection.\n",
      "finish in 1.511s, 286 probes kept.\n",
      "-- check region:995 14:89235001-89265000_strand_+_gene_995, 58874 candidate probes\n",
      "--- 22293 probes passed check_dic selection.\n",
      "finish in 1.769s, 337 probes kept.\n",
      "-- check region:996 14:90735001-90765000_strand_+_gene_996, 58914 candidate probes\n",
      "--- 15697 probes passed check_dic selection.\n",
      "finish in 1.496s, 255 probes kept.\n",
      "-- check region:997 14:92235001-92265000_strand_+_gene_997, 59290 candidate probes\n",
      "--- 19992 probes passed check_dic selection.\n",
      "finish in 1.677s, 304 probes kept.\n",
      "-- check region:998 14:93735001-93765000_strand_+_gene_998, 59220 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.973s, 0 probes kept.\n",
      "-- check region:999 14:95235001-95265000_strand_+_gene_999, 58842 candidate probes\n",
      "--- 22857 probes passed check_dic selection.\n",
      "finish in 1.783s, 341 probes kept.\n",
      "-- check region:1000 14:96735001-96765000_strand_+_gene_1000, 59126 candidate probes\n",
      "--- 21089 probes passed check_dic selection.\n",
      "finish in 1.710s, 326 probes kept.\n",
      "-- check region:1001 14:98235001-98265000_strand_+_gene_1001, 58978 candidate probes\n",
      "--- 21920 probes passed check_dic selection.\n",
      "finish in 1.745s, 341 probes kept.\n",
      "-- check region:1002 14:99735001-99765000_strand_+_gene_1002, 59442 candidate probes\n",
      "--- 26380 probes passed check_dic selection.\n",
      "finish in 1.939s, 381 probes kept.\n",
      "-- check region:1003 14:101235001-101265000_strand_+_gene_1003, 58358 candidate probes\n",
      "--- 24596 probes passed check_dic selection.\n",
      "finish in 1.834s, 356 probes kept.\n",
      "-- check region:1004 14:102735001-102765000_strand_+_gene_1004, 56204 candidate probes\n",
      "--- 20535 probes passed check_dic selection.\n",
      "finish in 1.650s, 283 probes kept.\n",
      "-- check region:1005 14:104235001-104265000_strand_+_gene_1005, 57338 candidate probes\n",
      "--- 19117 probes passed check_dic selection.\n",
      "finish in 1.613s, 321 probes kept.\n",
      "-- check region:1006 14:105735001-105765000_strand_+_gene_1006, 56982 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.930s, 0 probes kept.\n",
      "-- check region:1007 15:735001-765000_strand_+_gene_1007, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1008 15:2235001-2265000_strand_+_gene_1008, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1009 15:3735001-3765000_strand_+_gene_1009, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1010 15:5235001-5265000_strand_+_gene_1010, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1011 15:6735001-6765000_strand_+_gene_1011, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1012 15:8235001-8265000_strand_+_gene_1012, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1013 15:9735001-9765000_strand_+_gene_1013, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1014 15:11235001-11265000_strand_+_gene_1014, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1015 15:12735001-12765000_strand_+_gene_1015, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1016 15:14235001-14265000_strand_+_gene_1016, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1017 15:15735001-15765000_strand_+_gene_1017, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1018 15:17235001-17265000_strand_+_gene_1018, 20232 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.319s, 0 probes kept.\n",
      "-- check region:1019 15:18735001-18765000_strand_+_gene_1019, 9492 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.153s, 0 probes kept.\n",
      "-- check region:1020 15:20235001-20265000_strand_+_gene_1020, 57112 candidate probes\n",
      "--- 1290 probes passed check_dic selection.\n",
      "finish in 0.985s, 52 probes kept.\n",
      "-- check region:1021 15:21735001-21765000_strand_+_gene_1021, 58998 candidate probes\n",
      "--- 176 probes passed check_dic selection.\n",
      "finish in 0.948s, 11 probes kept.\n",
      "-- check region:1022 15:23235001-23265000_strand_+_gene_1022, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1023 15:24735001-24765000_strand_+_gene_1023, 59194 candidate probes\n",
      "--- 10244 probes passed check_dic selection.\n",
      "finish in 1.302s, 217 probes kept.\n",
      "-- check region:1024 15:26235001-26265000_strand_+_gene_1024, 58096 candidate probes\n",
      "--- 14110 probes passed check_dic selection.\n",
      "finish in 1.429s, 242 probes kept.\n",
      "-- check region:1025 15:27735001-27765000_strand_+_gene_1025, 59374 candidate probes\n",
      "--- 18759 probes passed check_dic selection.\n",
      "finish in 1.630s, 306 probes kept.\n",
      "-- check region:1026 15:29235001-29265000_strand_+_gene_1026, 58654 candidate probes\n",
      "--- 354 probes passed check_dic selection.\n",
      "finish in 0.940s, 17 probes kept.\n",
      "-- check region:1027 15:30735001-30765000_strand_+_gene_1027, 59116 candidate probes\n",
      "--- 64 probes passed check_dic selection.\n",
      "finish in 0.945s, 3 probes kept.\n",
      "-- check region:1028 15:32235001-32265000_strand_+_gene_1028, 58334 candidate probes\n",
      "--- 20 probes passed check_dic selection.\n",
      "finish in 0.915s, 1 probes kept.\n",
      "-- check region:1029 15:33735001-33765000_strand_+_gene_1029, 57146 candidate probes\n",
      "--- 16079 probes passed check_dic selection.\n",
      "finish in 1.518s, 235 probes kept.\n",
      "-- check region:1030 15:35235001-35265000_strand_+_gene_1030, 56208 candidate probes\n",
      "--- 10526 probes passed check_dic selection.\n",
      "finish in 1.262s, 209 probes kept.\n",
      "-- check region:1031 15:36735001-36765000_strand_+_gene_1031, 58514 candidate probes\n",
      "--- 16434 probes passed check_dic selection.\n",
      "finish in 1.508s, 290 probes kept.\n",
      "-- check region:1032 15:38235001-38265000_strand_+_gene_1032, 59318 candidate probes\n",
      "--- 13933 probes passed check_dic selection.\n",
      "finish in 1.415s, 275 probes kept.\n",
      "-- check region:1033 15:39735001-39765000_strand_+_gene_1033, 59910 candidate probes\n",
      "--- 10895 probes passed check_dic selection.\n",
      "finish in 1.324s, 234 probes kept.\n",
      "-- check region:1034 15:41235001-41265000_strand_+_gene_1034, 56752 candidate probes\n",
      "--- 10290 probes passed check_dic selection.\n",
      "finish in 1.276s, 183 probes kept.\n",
      "-- check region:1035 15:42735001-42765000_strand_+_gene_1035, 58436 candidate probes\n",
      "--- 9185 probes passed check_dic selection.\n",
      "finish in 1.270s, 201 probes kept.\n",
      "-- check region:1036 15:44235001-44265000_strand_+_gene_1036, 48536 candidate probes\n",
      "--- 6845 probes passed check_dic selection.\n",
      "finish in 1.018s, 130 probes kept.\n",
      "-- check region:1037 15:45735001-45765000_strand_+_gene_1037, 56914 candidate probes\n",
      "--- 12719 probes passed check_dic selection.\n",
      "finish in 1.363s, 237 probes kept.\n",
      "-- check region:1038 15:47235001-47265000_strand_+_gene_1038, 59030 candidate probes\n",
      "--- 11325 probes passed check_dic selection.\n",
      "finish in 1.327s, 227 probes kept.\n",
      "-- check region:1039 15:48735001-48765000_strand_+_gene_1039, 59338 candidate probes\n",
      "--- 11135 probes passed check_dic selection.\n",
      "finish in 1.329s, 246 probes kept.\n",
      "-- check region:1040 15:50235001-50265000_strand_+_gene_1040, 59072 candidate probes\n",
      "--- 20458 probes passed check_dic selection.\n",
      "finish in 1.690s, 350 probes kept.\n",
      "-- check region:1041 15:51735001-51765000_strand_+_gene_1041, 59178 candidate probes\n",
      "--- 13355 probes passed check_dic selection.\n",
      "finish in 1.395s, 254 probes kept.\n",
      "-- check region:1042 15:53235001-53265000_strand_+_gene_1042, 59444 candidate probes\n",
      "--- 19562 probes passed check_dic selection.\n",
      "finish in 1.644s, 325 probes kept.\n",
      "-- check region:1043 15:54735001-54765000_strand_+_gene_1043, 56686 candidate probes\n",
      "--- 5321 probes passed check_dic selection.\n",
      "finish in 1.081s, 122 probes kept.\n",
      "-- check region:1044 15:56235001-56265000_strand_+_gene_1044, 59420 candidate probes\n",
      "--- 13851 probes passed check_dic selection.\n",
      "finish in 1.413s, 253 probes kept.\n",
      "-- check region:1045 15:57735001-57765000_strand_+_gene_1045, 59656 candidate probes\n",
      "--- 22466 probes passed check_dic selection.\n",
      "finish in 1.776s, 347 probes kept.\n",
      "-- check region:1046 15:59235001-59265000_strand_+_gene_1046, 58744 candidate probes\n",
      "--- 14904 probes passed check_dic selection.\n",
      "finish in 1.467s, 263 probes kept.\n",
      "-- check region:1047 15:60735001-60765000_strand_+_gene_1047, 59146 candidate probes\n",
      "--- 22824 probes passed check_dic selection.\n",
      "finish in 1.774s, 350 probes kept.\n",
      "-- check region:1048 15:62235001-62265000_strand_+_gene_1048, 57960 candidate probes\n",
      "--- 12330 probes passed check_dic selection.\n",
      "finish in 1.391s, 296 probes kept.\n",
      "-- check region:1049 15:63735001-63765000_strand_+_gene_1049, 58796 candidate probes\n",
      "--- 8083 probes passed check_dic selection.\n",
      "finish in 1.207s, 176 probes kept.\n",
      "-- check region:1050 15:65235001-65265000_strand_+_gene_1050, 56824 candidate probes\n",
      "--- 17031 probes passed check_dic selection.\n",
      "finish in 1.526s, 267 probes kept.\n",
      "-- check region:1051 15:66735001-66765000_strand_+_gene_1051, 59416 candidate probes\n",
      "--- 26041 probes passed check_dic selection.\n",
      "finish in 1.912s, 384 probes kept.\n",
      "-- check region:1052 15:68235001-68265000_strand_+_gene_1052, 57294 candidate probes\n",
      "--- 5566 probes passed check_dic selection.\n",
      "finish in 1.092s, 135 probes kept.\n",
      "-- check region:1053 15:69735001-69765000_strand_+_gene_1053, 59552 candidate probes\n",
      "--- 23263 probes passed check_dic selection.\n",
      "finish in 1.812s, 351 probes kept.\n",
      "-- check region:1054 15:71235001-71265000_strand_+_gene_1054, 58938 candidate probes\n",
      "--- 20713 probes passed check_dic selection.\n",
      "finish in 1.696s, 320 probes kept.\n",
      "-- check region:1055 15:72735001-72765000_strand_+_gene_1055, 58468 candidate probes\n",
      "--- 12035 probes passed check_dic selection.\n",
      "finish in 1.381s, 268 probes kept.\n",
      "-- check region:1056 15:74235001-74265000_strand_+_gene_1056, 59028 candidate probes\n",
      "--- 22794 probes passed check_dic selection.\n",
      "finish in 1.788s, 361 probes kept.\n",
      "-- check region:1057 15:75735001-75765000_strand_+_gene_1057, 57102 candidate probes\n",
      "--- 5366 probes passed check_dic selection.\n",
      "finish in 1.116s, 155 probes kept.\n",
      "-- check region:1058 15:77235001-77265000_strand_+_gene_1058, 53196 candidate probes\n",
      "--- 5149 probes passed check_dic selection.\n",
      "finish in 1.014s, 131 probes kept.\n",
      "-- check region:1059 15:78735001-78765000_strand_+_gene_1059, 59200 candidate probes\n",
      "--- 5507 probes passed check_dic selection.\n",
      "finish in 1.169s, 164 probes kept.\n",
      "-- check region:1060 15:80235001-80265000_strand_+_gene_1060, 59040 candidate probes\n",
      "--- 20713 probes passed check_dic selection.\n",
      "finish in 1.698s, 315 probes kept.\n",
      "-- check region:1061 15:81735001-81765000_strand_+_gene_1061, 56942 candidate probes\n",
      "--- 20454 probes passed check_dic selection.\n",
      "finish in 1.638s, 309 probes kept.\n",
      "-- check region:1062 15:83235001-83265000_strand_+_gene_1062, 58602 candidate probes\n",
      "--- 12134 probes passed check_dic selection.\n",
      "finish in 1.376s, 240 probes kept.\n",
      "-- check region:1063 15:84735001-84765000_strand_+_gene_1063, 58352 candidate probes\n",
      "--- 15700 probes passed check_dic selection.\n",
      "finish in 1.481s, 266 probes kept.\n",
      "-- check region:1064 15:86235001-86265000_strand_+_gene_1064, 59552 candidate probes\n",
      "--- 18547 probes passed check_dic selection.\n",
      "finish in 1.620s, 327 probes kept.\n",
      "-- check region:1065 15:87735001-87765000_strand_+_gene_1065, 59450 candidate probes\n",
      "--- 19490 probes passed check_dic selection.\n",
      "finish in 1.644s, 318 probes kept.\n",
      "-- check region:1066 15:89235001-89265000_strand_+_gene_1066, 58344 candidate probes\n",
      "--- 10108 probes passed check_dic selection.\n",
      "finish in 1.287s, 207 probes kept.\n",
      "-- check region:1067 15:90735001-90765000_strand_+_gene_1067, 58724 candidate probes\n",
      "--- 9631 probes passed check_dic selection.\n",
      "finish in 1.270s, 204 probes kept.\n",
      "-- check region:1068 15:92235001-92265000_strand_+_gene_1068, 55828 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 6624 probes passed check_dic selection.\n",
      "finish in 1.112s, 147 probes kept.\n",
      "-- check region:1069 15:93735001-93765000_strand_+_gene_1069, 57714 candidate probes\n",
      "--- 7185 probes passed check_dic selection.\n",
      "finish in 1.170s, 160 probes kept.\n",
      "-- check region:1070 15:95235001-95265000_strand_+_gene_1070, 59286 candidate probes\n",
      "--- 12443 probes passed check_dic selection.\n",
      "finish in 1.378s, 250 probes kept.\n",
      "-- check region:1071 15:96735001-96765000_strand_+_gene_1071, 58072 candidate probes\n",
      "--- 16991 probes passed check_dic selection.\n",
      "finish in 1.534s, 291 probes kept.\n",
      "-- check region:1072 15:98235001-98265000_strand_+_gene_1072, 59102 candidate probes\n",
      "--- 13463 probes passed check_dic selection.\n",
      "finish in 1.412s, 259 probes kept.\n",
      "-- check region:1073 15:99735001-99765000_strand_+_gene_1073, 58492 candidate probes\n",
      "--- 16026 probes passed check_dic selection.\n",
      "finish in 1.503s, 263 probes kept.\n",
      "-- check region:1074 16:735001-765000_strand_+_gene_1074, 55932 candidate probes\n",
      "--- 15767 probes passed check_dic selection.\n",
      "finish in 1.497s, 273 probes kept.\n",
      "-- check region:1075 16:2235001-2265000_strand_+_gene_1075, 57606 candidate probes\n",
      "--- 13675 probes passed check_dic selection.\n",
      "finish in 1.428s, 256 probes kept.\n",
      "-- check region:1076 16:3735001-3765000_strand_+_gene_1076, 57836 candidate probes\n",
      "--- 17336 probes passed check_dic selection.\n",
      "finish in 1.569s, 293 probes kept.\n",
      "-- check region:1077 16:5235001-5265000_strand_+_gene_1077, 59308 candidate probes\n",
      "--- 3292 probes passed check_dic selection.\n",
      "finish in 1.075s, 88 probes kept.\n",
      "-- check region:1078 16:6735001-6765000_strand_+_gene_1078, 58632 candidate probes\n",
      "--- 13331 probes passed check_dic selection.\n",
      "finish in 1.395s, 241 probes kept.\n",
      "-- check region:1079 16:8235001-8265000_strand_+_gene_1079, 59702 candidate probes\n",
      "--- 17077 probes passed check_dic selection.\n",
      "finish in 1.558s, 302 probes kept.\n",
      "-- check region:1080 16:9735001-9765000_strand_+_gene_1080, 59256 candidate probes\n",
      "--- 17743 probes passed check_dic selection.\n",
      "finish in 1.621s, 362 probes kept.\n",
      "-- check region:1081 16:11235001-11265000_strand_+_gene_1081, 58996 candidate probes\n",
      "--- 18797 probes passed check_dic selection.\n",
      "finish in 1.617s, 298 probes kept.\n",
      "-- check region:1082 16:12735001-12765000_strand_+_gene_1082, 58850 candidate probes\n",
      "--- 13954 probes passed check_dic selection.\n",
      "finish in 1.448s, 263 probes kept.\n",
      "-- check region:1083 16:14235001-14265000_strand_+_gene_1083, 59282 candidate probes\n",
      "--- 17516 probes passed check_dic selection.\n",
      "finish in 1.590s, 325 probes kept.\n",
      "-- check region:1084 16:15735001-15765000_strand_+_gene_1084, 58414 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.959s, 0 probes kept.\n",
      "-- check region:1085 16:17235001-17265000_strand_+_gene_1085, 59198 candidate probes\n",
      "--- 23024 probes passed check_dic selection.\n",
      "finish in 1.785s, 341 probes kept.\n",
      "-- check region:1086 16:18735001-18765000_strand_+_gene_1086, 57596 candidate probes\n",
      "--- 668 probes passed check_dic selection.\n",
      "finish in 0.965s, 28 probes kept.\n",
      "-- check region:1087 16:20235001-20265000_strand_+_gene_1087, 58992 candidate probes\n",
      "--- 20531 probes passed check_dic selection.\n",
      "finish in 1.701s, 336 probes kept.\n",
      "-- check region:1088 16:21735001-21765000_strand_+_gene_1088, 58806 candidate probes\n",
      "--- 688 probes passed check_dic selection.\n",
      "finish in 0.978s, 25 probes kept.\n",
      "-- check region:1089 16:23235001-23265000_strand_+_gene_1089, 58860 candidate probes\n",
      "--- 12590 probes passed check_dic selection.\n",
      "finish in 1.393s, 207 probes kept.\n",
      "-- check region:1090 16:24735001-24765000_strand_+_gene_1090, 59486 candidate probes\n",
      "--- 14238 probes passed check_dic selection.\n",
      "finish in 1.450s, 261 probes kept.\n",
      "-- check region:1091 16:26235001-26265000_strand_+_gene_1091, 59670 candidate probes\n",
      "--- 11069 probes passed check_dic selection.\n",
      "finish in 1.339s, 226 probes kept.\n",
      "-- check region:1092 16:27735001-27765000_strand_+_gene_1092, 59368 candidate probes\n",
      "--- 22538 probes passed check_dic selection.\n",
      "finish in 1.821s, 355 probes kept.\n",
      "-- check region:1093 16:29235001-29265000_strand_+_gene_1093, 58836 candidate probes\n",
      "--- 8098 probes passed check_dic selection.\n",
      "finish in 1.244s, 203 probes kept.\n",
      "-- check region:1094 16:30735001-30765000_strand_+_gene_1094, 57978 candidate probes\n",
      "--- 15674 probes passed check_dic selection.\n",
      "finish in 1.534s, 317 probes kept.\n",
      "-- check region:1095 16:32235001-32265000_strand_+_gene_1095, 59838 candidate probes\n",
      "--- 14 probes passed check_dic selection.\n",
      "finish in 0.955s, 2 probes kept.\n",
      "-- check region:1096 16:33735001-33765000_strand_+_gene_1096, 57806 candidate probes\n",
      "--- 248 probes passed check_dic selection.\n",
      "finish in 0.911s, 14 probes kept.\n",
      "-- check region:1097 16:35235001-35265000_strand_+_gene_1097, 59852 candidate probes\n",
      "--- 9304 probes passed check_dic selection.\n",
      "finish in 1.286s, 230 probes kept.\n",
      "-- check region:1098 16:36735001-36765000_strand_+_gene_1098, 8216 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.133s, 0 probes kept.\n",
      "-- check region:1099 16:38235001-38265000_strand_+_gene_1099, 9408 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.153s, 0 probes kept.\n",
      "-- check region:1100 16:39735001-39765000_strand_+_gene_1100, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1101 16:41235001-41265000_strand_+_gene_1101, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1102 16:42735001-42765000_strand_+_gene_1102, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1103 16:44235001-44265000_strand_+_gene_1103, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1104 16:45735001-45765000_strand_+_gene_1104, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1105 16:47235001-47265000_strand_+_gene_1105, 58574 candidate probes\n",
      "--- 11166 probes passed check_dic selection.\n",
      "finish in 1.319s, 223 probes kept.\n",
      "-- check region:1106 16:48735001-48765000_strand_+_gene_1106, 57616 candidate probes\n",
      "--- 10715 probes passed check_dic selection.\n",
      "finish in 1.305s, 189 probes kept.\n",
      "-- check region:1107 16:50235001-50265000_strand_+_gene_1107, 58822 candidate probes\n",
      "--- 17599 probes passed check_dic selection.\n",
      "finish in 1.579s, 267 probes kept.\n",
      "-- check region:1108 16:51735001-51765000_strand_+_gene_1108, 59908 candidate probes\n",
      "--- 17933 probes passed check_dic selection.\n",
      "finish in 1.581s, 295 probes kept.\n",
      "-- check region:1109 16:53235001-53265000_strand_+_gene_1109, 59266 candidate probes\n",
      "--- 7204 probes passed check_dic selection.\n",
      "finish in 1.183s, 184 probes kept.\n",
      "-- check region:1110 16:54735001-54765000_strand_+_gene_1110, 59154 candidate probes\n",
      "--- 20774 probes passed check_dic selection.\n",
      "finish in 1.691s, 336 probes kept.\n",
      "-- check region:1111 16:56235001-56265000_strand_+_gene_1111, 59834 candidate probes\n",
      "--- 26866 probes passed check_dic selection.\n",
      "finish in 1.934s, 381 probes kept.\n",
      "-- check region:1112 16:57735001-57765000_strand_+_gene_1112, 58976 candidate probes\n",
      "--- 24770 probes passed check_dic selection.\n",
      "finish in 1.876s, 398 probes kept.\n",
      "-- check region:1113 16:59235001-59265000_strand_+_gene_1113, 59218 candidate probes\n",
      "--- 13972 probes passed check_dic selection.\n",
      "finish in 1.430s, 266 probes kept.\n",
      "-- check region:1114 16:60735001-60765000_strand_+_gene_1114, 59284 candidate probes\n",
      "--- 10830 probes passed check_dic selection.\n",
      "finish in 1.314s, 217 probes kept.\n",
      "-- check region:1115 16:62235001-62265000_strand_+_gene_1115, 59560 candidate probes\n",
      "--- 12400 probes passed check_dic selection.\n",
      "finish in 1.376s, 253 probes kept.\n",
      "-- check region:1116 16:63735001-63765000_strand_+_gene_1116, 59230 candidate probes\n",
      "--- 7748 probes passed check_dic selection.\n",
      "finish in 1.201s, 184 probes kept.\n",
      "-- check region:1117 16:65235001-65265000_strand_+_gene_1117, 55438 candidate probes\n",
      "--- 16117 probes passed check_dic selection.\n",
      "finish in 1.454s, 288 probes kept.\n",
      "-- check region:1118 16:66735001-66765000_strand_+_gene_1118, 57134 candidate probes\n",
      "--- 11571 probes passed check_dic selection.\n",
      "finish in 1.329s, 231 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:1119 16:68235001-68265000_strand_+_gene_1119, 58084 candidate probes\n",
      "--- 21599 probes passed check_dic selection.\n",
      "finish in 1.730s, 328 probes kept.\n",
      "-- check region:1120 16:69735001-69765000_strand_+_gene_1120, 57914 candidate probes\n",
      "--- 12618 probes passed check_dic selection.\n",
      "finish in 1.393s, 228 probes kept.\n",
      "-- check region:1121 16:71235001-71265000_strand_+_gene_1121, 59502 candidate probes\n",
      "--- 17116 probes passed check_dic selection.\n",
      "finish in 1.564s, 298 probes kept.\n",
      "-- check region:1122 16:72735001-72765000_strand_+_gene_1122, 59080 candidate probes\n",
      "--- 11320 probes passed check_dic selection.\n",
      "finish in 1.336s, 227 probes kept.\n",
      "-- check region:1123 16:74235001-74265000_strand_+_gene_1123, 58092 candidate probes\n",
      "--- 15410 probes passed check_dic selection.\n",
      "finish in 1.479s, 267 probes kept.\n",
      "-- check region:1124 16:75735001-75765000_strand_+_gene_1124, 55488 candidate probes\n",
      "--- 10625 probes passed check_dic selection.\n",
      "finish in 1.267s, 201 probes kept.\n",
      "-- check region:1125 16:77235001-77265000_strand_+_gene_1125, 57718 candidate probes\n",
      "--- 11950 probes passed check_dic selection.\n",
      "finish in 1.345s, 208 probes kept.\n",
      "-- check region:1126 16:78735001-78765000_strand_+_gene_1126, 59246 candidate probes\n",
      "--- 21069 probes passed check_dic selection.\n",
      "finish in 1.713s, 320 probes kept.\n",
      "-- check region:1127 16:80235001-80265000_strand_+_gene_1127, 57418 candidate probes\n",
      "--- 14243 probes passed check_dic selection.\n",
      "finish in 1.406s, 241 probes kept.\n",
      "-- check region:1128 16:81735001-81765000_strand_+_gene_1128, 58144 candidate probes\n",
      "--- 13753 probes passed check_dic selection.\n",
      "finish in 1.436s, 235 probes kept.\n",
      "-- check region:1129 16:83235001-83265000_strand_+_gene_1129, 59594 candidate probes\n",
      "--- 20187 probes passed check_dic selection.\n",
      "finish in 1.670s, 335 probes kept.\n",
      "-- check region:1130 16:84735001-84765000_strand_+_gene_1130, 58968 candidate probes\n",
      "--- 19618 probes passed check_dic selection.\n",
      "finish in 1.640s, 313 probes kept.\n",
      "-- check region:1131 16:86235001-86265000_strand_+_gene_1131, 49850 candidate probes\n",
      "--- 18095 probes passed check_dic selection.\n",
      "finish in 1.457s, 272 probes kept.\n",
      "-- check region:1132 16:87735001-87765000_strand_+_gene_1132, 56974 candidate probes\n",
      "--- 19986 probes passed check_dic selection.\n",
      "finish in 1.645s, 290 probes kept.\n",
      "-- check region:1133 16:89235001-89265000_strand_+_gene_1133, 55660 candidate probes\n",
      "--- 11862 probes passed check_dic selection.\n",
      "finish in 1.313s, 205 probes kept.\n",
      "-- check region:1134 17:735001-765000_strand_+_gene_1134, 56602 candidate probes\n",
      "--- 17190 probes passed check_dic selection.\n",
      "finish in 1.574s, 331 probes kept.\n",
      "-- check region:1135 17:2235001-2265000_strand_+_gene_1135, 57046 candidate probes\n",
      "--- 17374 probes passed check_dic selection.\n",
      "finish in 1.534s, 268 probes kept.\n",
      "-- check region:1136 17:3735001-3765000_strand_+_gene_1136, 56836 candidate probes\n",
      "--- 20544 probes passed check_dic selection.\n",
      "finish in 1.668s, 294 probes kept.\n",
      "-- check region:1137 17:5235001-5265000_strand_+_gene_1137, 54046 candidate probes\n",
      "--- 10212 probes passed check_dic selection.\n",
      "finish in 1.246s, 196 probes kept.\n",
      "-- check region:1138 17:6735001-6765000_strand_+_gene_1138, 58168 candidate probes\n",
      "--- 12176 probes passed check_dic selection.\n",
      "finish in 1.374s, 222 probes kept.\n",
      "-- check region:1139 17:8235001-8265000_strand_+_gene_1139, 57506 candidate probes\n",
      "--- 14953 probes passed check_dic selection.\n",
      "finish in 1.480s, 256 probes kept.\n",
      "-- check region:1140 17:9735001-9765000_strand_+_gene_1140, 57798 candidate probes\n",
      "--- 14699 probes passed check_dic selection.\n",
      "finish in 1.440s, 242 probes kept.\n",
      "-- check region:1141 17:11235001-11265000_strand_+_gene_1141, 59408 candidate probes\n",
      "--- 25557 probes passed check_dic selection.\n",
      "finish in 1.887s, 375 probes kept.\n",
      "-- check region:1142 17:12735001-12765000_strand_+_gene_1142, 58088 candidate probes\n",
      "--- 14630 probes passed check_dic selection.\n",
      "finish in 1.475s, 263 probes kept.\n",
      "-- check region:1143 17:14235001-14265000_strand_+_gene_1143, 58124 candidate probes\n",
      "--- 10590 probes passed check_dic selection.\n",
      "finish in 1.302s, 206 probes kept.\n",
      "-- check region:1144 17:15735001-15765000_strand_+_gene_1144, 59342 candidate probes\n",
      "--- 4051 probes passed check_dic selection.\n",
      "finish in 1.104s, 122 probes kept.\n",
      "-- check region:1145 17:17235001-17265000_strand_+_gene_1145, 57004 candidate probes\n",
      "--- 12463 probes passed check_dic selection.\n",
      "finish in 1.369s, 215 probes kept.\n",
      "-- check region:1146 17:18735001-18765000_strand_+_gene_1146, 56140 candidate probes\n",
      "--- 2810 probes passed check_dic selection.\n",
      "finish in 1.008s, 94 probes kept.\n",
      "-- check region:1147 17:20235001-20265000_strand_+_gene_1147, 58944 candidate probes\n",
      "--- 11917 probes passed check_dic selection.\n",
      "finish in 1.367s, 257 probes kept.\n",
      "-- check region:1148 17:21735001-21765000_strand_+_gene_1148, 58912 candidate probes\n",
      "--- 3517 probes passed check_dic selection.\n",
      "finish in 1.082s, 101 probes kept.\n",
      "-- check region:1149 17:23235001-23265000_strand_+_gene_1149, 7348 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.120s, 0 probes kept.\n",
      "-- check region:1150 17:24735001-24765000_strand_+_gene_1150, 7626 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.124s, 0 probes kept.\n",
      "-- check region:1151 17:26235001-26265000_strand_+_gene_1151, 19024 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.313s, 0 probes kept.\n",
      "-- check region:1152 17:27735001-27765000_strand_+_gene_1152, 59528 candidate probes\n",
      "--- 8242 probes passed check_dic selection.\n",
      "finish in 1.271s, 194 probes kept.\n",
      "-- check region:1153 17:29235001-29265000_strand_+_gene_1153, 57844 candidate probes\n",
      "--- 13514 probes passed check_dic selection.\n",
      "finish in 1.415s, 269 probes kept.\n",
      "-- check region:1154 17:30735001-30765000_strand_+_gene_1154, 57030 candidate probes\n",
      "--- 1418 probes passed check_dic selection.\n",
      "finish in 0.960s, 54 probes kept.\n",
      "-- check region:1155 17:32235001-32265000_strand_+_gene_1155, 57146 candidate probes\n",
      "--- 15463 probes passed check_dic selection.\n",
      "finish in 1.491s, 254 probes kept.\n",
      "-- check region:1156 17:33735001-33765000_strand_+_gene_1156, 59339 candidate probes\n",
      "--- 19916 probes passed check_dic selection.\n",
      "finish in 1.677s, 327 probes kept.\n",
      "-- check region:1157 17:35235001-35265000_strand_+_gene_1157, 58518 candidate probes\n",
      "--- 16298 probes passed check_dic selection.\n",
      "finish in 1.529s, 276 probes kept.\n",
      "-- check region:1158 17:36735001-36765000_strand_+_gene_1158, 59342 candidate probes\n",
      "--- 690 probes passed check_dic selection.\n",
      "finish in 0.997s, 20 probes kept.\n",
      "-- check region:1159 17:38235001-38265000_strand_+_gene_1159, 54094 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.857s, 0 probes kept.\n",
      "-- check region:1160 17:39735001-39765000_strand_+_gene_1160, 58672 candidate probes\n",
      "--- 17391 probes passed check_dic selection.\n",
      "finish in 1.590s, 317 probes kept.\n",
      "-- check region:1161 17:41235001-41265000_strand_+_gene_1161, 53046 candidate probes\n",
      "--- 24 probes passed check_dic selection.\n",
      "finish in 0.845s, 3 probes kept.\n",
      "-- check region:1162 17:42735001-42765000_strand_+_gene_1162, 57652 candidate probes\n",
      "--- 11949 probes passed check_dic selection.\n",
      "finish in 1.358s, 222 probes kept.\n",
      "-- check region:1163 17:44235001-44265000_strand_+_gene_1163, 56470 candidate probes\n",
      "--- 15429 probes passed check_dic selection.\n",
      "finish in 1.476s, 272 probes kept.\n",
      "-- check region:1164 17:45735001-45765000_strand_+_gene_1164, 59000 candidate probes\n",
      "--- 52 probes passed check_dic selection.\n",
      "finish in 0.967s, 3 probes kept.\n",
      "-- check region:1165 17:47235001-47265000_strand_+_gene_1165, 58640 candidate probes\n",
      "--- 16446 probes passed check_dic selection.\n",
      "finish in 1.520s, 270 probes kept.\n",
      "-- check region:1166 17:48735001-48765000_strand_+_gene_1166, 59218 candidate probes\n",
      "--- 22559 probes passed check_dic selection.\n",
      "finish in 1.772s, 342 probes kept.\n",
      "-- check region:1167 17:50235001-50265000_strand_+_gene_1167, 58532 candidate probes\n",
      "--- 9860 probes passed check_dic selection.\n",
      "finish in 1.285s, 192 probes kept.\n",
      "-- check region:1168 17:51735001-51765000_strand_+_gene_1168, 59490 candidate probes\n",
      "--- 22720 probes passed check_dic selection.\n",
      "finish in 1.771s, 346 probes kept.\n",
      "-- check region:1169 17:53235001-53265000_strand_+_gene_1169, 57240 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 8520 probes passed check_dic selection.\n",
      "finish in 1.192s, 185 probes kept.\n",
      "-- check region:1170 17:54735001-54765000_strand_+_gene_1170, 59522 candidate probes\n",
      "--- 12547 probes passed check_dic selection.\n",
      "finish in 1.388s, 232 probes kept.\n",
      "-- check region:1171 17:56235001-56265000_strand_+_gene_1171, 59358 candidate probes\n",
      "--- 15239 probes passed check_dic selection.\n",
      "finish in 1.480s, 268 probes kept.\n",
      "-- check region:1172 17:57735001-57765000_strand_+_gene_1172, 58822 candidate probes\n",
      "--- 22615 probes passed check_dic selection.\n",
      "finish in 1.765s, 329 probes kept.\n",
      "-- check region:1173 17:59235001-59265000_strand_+_gene_1173, 56443 candidate probes\n",
      "--- 3837 probes passed check_dic selection.\n",
      "finish in 1.030s, 96 probes kept.\n",
      "-- check region:1174 17:60735001-60765000_strand_+_gene_1174, 56646 candidate probes\n",
      "--- 4662 probes passed check_dic selection.\n",
      "finish in 1.054s, 128 probes kept.\n",
      "-- check region:1175 17:62235001-62265000_strand_+_gene_1175, 57584 candidate probes\n",
      "--- 1192 probes passed check_dic selection.\n",
      "finish in 0.962s, 44 probes kept.\n",
      "-- check region:1176 17:63735001-63765000_strand_+_gene_1176, 57806 candidate probes\n",
      "--- 9858 probes passed check_dic selection.\n",
      "finish in 1.278s, 212 probes kept.\n",
      "-- check region:1177 17:65235001-65265000_strand_+_gene_1177, 58900 candidate probes\n",
      "--- 14497 probes passed check_dic selection.\n",
      "finish in 1.467s, 241 probes kept.\n",
      "-- check region:1178 17:66735001-66765000_strand_+_gene_1178, 58416 candidate probes\n",
      "--- 20744 probes passed check_dic selection.\n",
      "finish in 1.691s, 321 probes kept.\n",
      "-- check region:1179 17:68235001-68265000_strand_+_gene_1179, 58958 candidate probes\n",
      "--- 7607 probes passed check_dic selection.\n",
      "finish in 1.213s, 197 probes kept.\n",
      "-- check region:1180 17:69735001-69765000_strand_+_gene_1180, 56278 candidate probes\n",
      "--- 9900 probes passed check_dic selection.\n",
      "finish in 1.241s, 213 probes kept.\n",
      "-- check region:1181 17:71235001-71265000_strand_+_gene_1181, 59236 candidate probes\n",
      "--- 14571 probes passed check_dic selection.\n",
      "finish in 1.444s, 271 probes kept.\n",
      "-- check region:1182 17:72735001-72765000_strand_+_gene_1182, 58224 candidate probes\n",
      "--- 20385 probes passed check_dic selection.\n",
      "finish in 1.668s, 291 probes kept.\n",
      "-- check region:1183 17:74235001-74265000_strand_+_gene_1183, 58840 candidate probes\n",
      "--- 20318 probes passed check_dic selection.\n",
      "finish in 1.708s, 337 probes kept.\n",
      "-- check region:1184 17:75735001-75765000_strand_+_gene_1184, 58250 candidate probes\n",
      "--- 18079 probes passed check_dic selection.\n",
      "finish in 1.602s, 310 probes kept.\n",
      "-- check region:1185 17:77235001-77265000_strand_+_gene_1185, 57304 candidate probes\n",
      "--- 202 probes passed check_dic selection.\n",
      "finish in 0.939s, 4 probes kept.\n",
      "-- check region:1186 17:78735001-78765000_strand_+_gene_1186, 58864 candidate probes\n",
      "--- 15108 probes passed check_dic selection.\n",
      "finish in 1.484s, 265 probes kept.\n",
      "-- check region:1187 17:80235001-80265000_strand_+_gene_1187, 57108 candidate probes\n",
      "--- 18795 probes passed check_dic selection.\n",
      "finish in 1.593s, 285 probes kept.\n",
      "-- check region:1188 17:81735001-81765000_strand_+_gene_1188, 14538 candidate probes\n",
      "--- 6139 probes passed check_dic selection.\n",
      "finish in 0.461s, 77 probes kept.\n",
      "-- check region:1189 18:735001-765000_strand_+_gene_1189, 57644 candidate probes\n",
      "--- 7096 probes passed check_dic selection.\n",
      "finish in 1.164s, 171 probes kept.\n",
      "-- check region:1190 18:2235001-2265000_strand_+_gene_1190, 52140 candidate probes\n",
      "--- 9189 probes passed check_dic selection.\n",
      "finish in 1.155s, 187 probes kept.\n",
      "-- check region:1191 18:3735001-3765000_strand_+_gene_1191, 54522 candidate probes\n",
      "--- 10060 probes passed check_dic selection.\n",
      "finish in 1.221s, 194 probes kept.\n",
      "-- check region:1192 18:5235001-5265000_strand_+_gene_1192, 59758 candidate probes\n",
      "--- 9373 probes passed check_dic selection.\n",
      "finish in 1.291s, 242 probes kept.\n",
      "-- check region:1193 18:6735001-6765000_strand_+_gene_1193, 59512 candidate probes\n",
      "--- 15014 probes passed check_dic selection.\n",
      "finish in 1.470s, 279 probes kept.\n",
      "-- check region:1194 18:8235001-8265000_strand_+_gene_1194, 59074 candidate probes\n",
      "--- 16333 probes passed check_dic selection.\n",
      "finish in 1.521s, 291 probes kept.\n",
      "-- check region:1195 18:9735001-9765000_strand_+_gene_1195, 59004 candidate probes\n",
      "--- 18999 probes passed check_dic selection.\n",
      "finish in 1.621s, 288 probes kept.\n",
      "-- check region:1196 18:11235001-11265000_strand_+_gene_1196, 58964 candidate probes\n",
      "--- 15547 probes passed check_dic selection.\n",
      "finish in 1.481s, 277 probes kept.\n",
      "-- check region:1197 18:12735001-12765000_strand_+_gene_1197, 56960 candidate probes\n",
      "--- 10378 probes passed check_dic selection.\n",
      "finish in 1.291s, 184 probes kept.\n",
      "-- check region:1198 18:14235001-14265000_strand_+_gene_1198, 59672 candidate probes\n",
      "--- 367 probes passed check_dic selection.\n",
      "finish in 0.969s, 30 probes kept.\n",
      "-- check region:1199 18:15735001-15765000_strand_+_gene_1199, 23734 candidate probes\n",
      "--- 10 probes passed check_dic selection.\n",
      "finish in 0.387s, 2 probes kept.\n",
      "-- check region:1200 18:17235001-17265000_strand_+_gene_1200, 8108 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.132s, 0 probes kept.\n",
      "-- check region:1201 18:18735001-18765000_strand_+_gene_1201, 8552 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.138s, 0 probes kept.\n",
      "-- check region:1202 18:20235001-20265000_strand_+_gene_1202, 13110 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.216s, 0 probes kept.\n",
      "-- check region:1203 18:21735001-21765000_strand_+_gene_1203, 58394 candidate probes\n",
      "--- 9079 probes passed check_dic selection.\n",
      "finish in 1.244s, 191 probes kept.\n",
      "-- check region:1204 18:23235001-23265000_strand_+_gene_1204, 58866 candidate probes\n",
      "--- 22842 probes passed check_dic selection.\n",
      "finish in 1.799s, 346 probes kept.\n",
      "-- check region:1205 18:24735001-24765000_strand_+_gene_1205, 59700 candidate probes\n",
      "--- 20588 probes passed check_dic selection.\n",
      "finish in 1.694s, 321 probes kept.\n",
      "-- check region:1206 18:26235001-26265000_strand_+_gene_1206, 52008 candidate probes\n",
      "--- 3965 probes passed check_dic selection.\n",
      "finish in 0.979s, 108 probes kept.\n",
      "-- check region:1207 18:27735001-27765000_strand_+_gene_1207, 51782 candidate probes\n",
      "--- 10618 probes passed check_dic selection.\n",
      "finish in 1.196s, 213 probes kept.\n",
      "-- check region:1208 18:29235001-29265000_strand_+_gene_1208, 59308 candidate probes\n",
      "--- 8807 probes passed check_dic selection.\n",
      "finish in 1.235s, 208 probes kept.\n",
      "-- check region:1209 18:30735001-30765000_strand_+_gene_1209, 59246 candidate probes\n",
      "--- 7695 probes passed check_dic selection.\n",
      "finish in 1.188s, 169 probes kept.\n",
      "-- check region:1210 18:32235001-32265000_strand_+_gene_1210, 59058 candidate probes\n",
      "--- 15145 probes passed check_dic selection.\n",
      "finish in 1.489s, 270 probes kept.\n",
      "-- check region:1211 18:33735001-33765000_strand_+_gene_1211, 59874 candidate probes\n",
      "--- 13724 probes passed check_dic selection.\n",
      "finish in 1.454s, 302 probes kept.\n",
      "-- check region:1212 18:35235001-35265000_strand_+_gene_1212, 59144 candidate probes\n",
      "--- 10851 probes passed check_dic selection.\n",
      "finish in 1.344s, 278 probes kept.\n",
      "-- check region:1213 18:36735001-36765000_strand_+_gene_1213, 58980 candidate probes\n",
      "--- 17961 probes passed check_dic selection.\n",
      "finish in 1.572s, 293 probes kept.\n",
      "-- check region:1214 18:38235001-38265000_strand_+_gene_1214, 59830 candidate probes\n",
      "--- 17620 probes passed check_dic selection.\n",
      "finish in 1.574s, 317 probes kept.\n",
      "-- check region:1215 18:39735001-39765000_strand_+_gene_1215, 59682 candidate probes\n",
      "--- 11122 probes passed check_dic selection.\n",
      "finish in 1.334s, 221 probes kept.\n",
      "-- check region:1216 18:41235001-41265000_strand_+_gene_1216, 59352 candidate probes\n",
      "--- 11677 probes passed check_dic selection.\n",
      "finish in 1.325s, 216 probes kept.\n",
      "-- check region:1217 18:42735001-42765000_strand_+_gene_1217, 59056 candidate probes\n",
      "--- 9338 probes passed check_dic selection.\n",
      "finish in 1.261s, 190 probes kept.\n",
      "-- check region:1218 18:44235001-44265000_strand_+_gene_1218, 59088 candidate probes\n",
      "--- 12128 probes passed check_dic selection.\n",
      "finish in 1.360s, 235 probes kept.\n",
      "-- check region:1219 18:45735001-45765000_strand_+_gene_1219, 58932 candidate probes\n",
      "--- 20142 probes passed check_dic selection.\n",
      "finish in 1.675s, 315 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:1220 18:47235001-47265000_strand_+_gene_1220, 59756 candidate probes\n",
      "--- 22586 probes passed check_dic selection.\n",
      "finish in 1.740s, 358 probes kept.\n",
      "-- check region:1221 18:48735001-48765000_strand_+_gene_1221, 59670 candidate probes\n",
      "--- 27239 probes passed check_dic selection.\n",
      "finish in 1.971s, 411 probes kept.\n",
      "-- check region:1222 18:50235001-50265000_strand_+_gene_1222, 58282 candidate probes\n",
      "--- 8971 probes passed check_dic selection.\n",
      "finish in 1.247s, 179 probes kept.\n",
      "-- check region:1223 18:51735001-51765000_strand_+_gene_1223, 52010 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.818s, 0 probes kept.\n",
      "-- check region:1224 18:53235001-53265000_strand_+_gene_1224, 56100 candidate probes\n",
      "--- 12868 probes passed check_dic selection.\n",
      "finish in 1.345s, 231 probes kept.\n",
      "-- check region:1225 18:54735001-54765000_strand_+_gene_1225, 59210 candidate probes\n",
      "--- 14691 probes passed check_dic selection.\n",
      "finish in 1.461s, 282 probes kept.\n",
      "-- check region:1226 18:56235001-56265000_strand_+_gene_1226, 56232 candidate probes\n",
      "--- 13033 probes passed check_dic selection.\n",
      "finish in 1.361s, 238 probes kept.\n",
      "-- check region:1227 18:57735001-57765000_strand_+_gene_1227, 58110 candidate probes\n",
      "--- 12225 probes passed check_dic selection.\n",
      "finish in 1.362s, 215 probes kept.\n",
      "-- check region:1228 18:59235001-59265000_strand_+_gene_1228, 53920 candidate probes\n",
      "--- 9937 probes passed check_dic selection.\n",
      "finish in 1.215s, 167 probes kept.\n",
      "-- check region:1229 18:60735001-60765000_strand_+_gene_1229, 59842 candidate probes\n",
      "--- 10356 probes passed check_dic selection.\n",
      "finish in 1.296s, 235 probes kept.\n",
      "-- check region:1230 18:62235001-62265000_strand_+_gene_1230, 59788 candidate probes\n",
      "--- 8868 probes passed check_dic selection.\n",
      "finish in 1.241s, 223 probes kept.\n",
      "-- check region:1231 18:63735001-63765000_strand_+_gene_1231, 58527 candidate probes\n",
      "--- 13794 probes passed check_dic selection.\n",
      "finish in 1.414s, 241 probes kept.\n",
      "-- check region:1232 18:65235001-65265000_strand_+_gene_1232, 59726 candidate probes\n",
      "--- 12814 probes passed check_dic selection.\n",
      "finish in 1.393s, 237 probes kept.\n",
      "-- check region:1233 18:66735001-66765000_strand_+_gene_1233, 58182 candidate probes\n",
      "--- 10850 probes passed check_dic selection.\n",
      "finish in 1.296s, 223 probes kept.\n",
      "-- check region:1234 18:68235001-68265000_strand_+_gene_1234, 57708 candidate probes\n",
      "--- 8539 probes passed check_dic selection.\n",
      "finish in 1.202s, 169 probes kept.\n",
      "-- check region:1235 18:69735001-69765000_strand_+_gene_1235, 58716 candidate probes\n",
      "--- 12895 probes passed check_dic selection.\n",
      "finish in 1.391s, 241 probes kept.\n",
      "-- check region:1236 18:71235001-71265000_strand_+_gene_1236, 59658 candidate probes\n",
      "--- 15045 probes passed check_dic selection.\n",
      "finish in 1.463s, 271 probes kept.\n",
      "-- check region:1237 18:72735001-72765000_strand_+_gene_1237, 59676 candidate probes\n",
      "--- 10914 probes passed check_dic selection.\n",
      "finish in 1.310s, 228 probes kept.\n",
      "-- check region:1238 18:74235001-74265000_strand_+_gene_1238, 58844 candidate probes\n",
      "--- 15841 probes passed check_dic selection.\n",
      "finish in 1.507s, 287 probes kept.\n",
      "-- check region:1239 18:75735001-75765000_strand_+_gene_1239, 59660 candidate probes\n",
      "--- 13905 probes passed check_dic selection.\n",
      "finish in 1.436s, 249 probes kept.\n",
      "-- check region:1240 18:77235001-77265000_strand_+_gene_1240, 59276 candidate probes\n",
      "--- 18596 probes passed check_dic selection.\n",
      "finish in 1.613s, 297 probes kept.\n",
      "-- check region:1241 18:78735001-78765000_strand_+_gene_1241, 58946 candidate probes\n",
      "--- 25275 probes passed check_dic selection.\n",
      "finish in 1.858s, 344 probes kept.\n",
      "-- check region:1242 19:485001-515000_strand_+_gene_1242, 56288 candidate probes\n",
      "--- 15473 probes passed check_dic selection.\n",
      "finish in 1.447s, 279 probes kept.\n",
      "-- check region:1243 19:1485001-1515000_strand_+_gene_1243, 56996 candidate probes\n",
      "--- 16468 probes passed check_dic selection.\n",
      "finish in 1.514s, 297 probes kept.\n",
      "-- check region:1244 19:2485001-2515000_strand_+_gene_1244, 56828 candidate probes\n",
      "--- 16670 probes passed check_dic selection.\n",
      "finish in 1.525s, 260 probes kept.\n",
      "-- check region:1245 19:3485001-3515000_strand_+_gene_1245, 57806 candidate probes\n",
      "--- 18034 probes passed check_dic selection.\n",
      "finish in 1.587s, 283 probes kept.\n",
      "-- check region:1246 19:4485001-4515000_strand_+_gene_1246, 57486 candidate probes\n",
      "--- 17480 probes passed check_dic selection.\n",
      "finish in 1.582s, 305 probes kept.\n",
      "-- check region:1247 19:5485001-5515000_strand_+_gene_1247, 56364 candidate probes\n",
      "--- 15411 probes passed check_dic selection.\n",
      "finish in 1.476s, 234 probes kept.\n",
      "-- check region:1248 19:6485001-6515000_strand_+_gene_1248, 56274 candidate probes\n",
      "--- 8231 probes passed check_dic selection.\n",
      "finish in 1.219s, 160 probes kept.\n",
      "-- check region:1249 19:7485001-7515000_strand_+_gene_1249, 57666 candidate probes\n",
      "--- 13787 probes passed check_dic selection.\n",
      "finish in 1.431s, 237 probes kept.\n",
      "-- check region:1250 19:8485001-8515000_strand_+_gene_1250, 58086 candidate probes\n",
      "--- 18194 probes passed check_dic selection.\n",
      "finish in 1.609s, 310 probes kept.\n",
      "-- check region:1251 19:9485001-9515000_strand_+_gene_1251, 58304 candidate probes\n",
      "--- 8894 probes passed check_dic selection.\n",
      "finish in 1.255s, 181 probes kept.\n",
      "-- check region:1252 19:10485001-10515000_strand_+_gene_1252, 57182 candidate probes\n",
      "--- 9158 probes passed check_dic selection.\n",
      "finish in 1.258s, 181 probes kept.\n",
      "-- check region:1253 19:11485001-11515000_strand_+_gene_1253, 57644 candidate probes\n",
      "--- 13446 probes passed check_dic selection.\n",
      "finish in 1.431s, 246 probes kept.\n",
      "-- check region:1254 19:12485001-12515000_strand_+_gene_1254, 55432 candidate probes\n",
      "--- 9454 probes passed check_dic selection.\n",
      "finish in 1.235s, 178 probes kept.\n",
      "-- check region:1255 19:13485001-13515000_strand_+_gene_1255, 58666 candidate probes\n",
      "--- 18750 probes passed check_dic selection.\n",
      "finish in 1.647s, 303 probes kept.\n",
      "-- check region:1256 19:14485001-14515000_strand_+_gene_1256, 56834 candidate probes\n",
      "--- 11434 probes passed check_dic selection.\n",
      "finish in 1.337s, 186 probes kept.\n",
      "-- check region:1257 19:15485001-15515000_strand_+_gene_1257, 52788 candidate probes\n",
      "--- 9011 probes passed check_dic selection.\n",
      "finish in 1.174s, 173 probes kept.\n",
      "-- check region:1258 19:16485001-16515000_strand_+_gene_1258, 56572 candidate probes\n",
      "--- 9106 probes passed check_dic selection.\n",
      "finish in 1.252s, 168 probes kept.\n",
      "-- check region:1259 19:17485001-17515000_strand_+_gene_1259, 57042 candidate probes\n",
      "--- 10537 probes passed check_dic selection.\n",
      "finish in 1.321s, 216 probes kept.\n",
      "-- check region:1260 19:18485001-18515000_strand_+_gene_1260, 58710 candidate probes\n",
      "--- 25864 probes passed check_dic selection.\n",
      "finish in 1.888s, 364 probes kept.\n",
      "-- check region:1261 19:19485001-19515000_strand_+_gene_1261, 59330 candidate probes\n",
      "--- 25052 probes passed check_dic selection.\n",
      "finish in 1.903s, 377 probes kept.\n",
      "-- check region:1262 19:20485001-20515000_strand_+_gene_1262, 56392 candidate probes\n",
      "--- 2198 probes passed check_dic selection.\n",
      "finish in 0.984s, 85 probes kept.\n",
      "-- check region:1263 19:21485001-21515000_strand_+_gene_1263, 56332 candidate probes\n",
      "--- 6287 probes passed check_dic selection.\n",
      "finish in 1.144s, 189 probes kept.\n",
      "-- check region:1264 19:22485001-22515000_strand_+_gene_1264, 59238 candidate probes\n",
      "--- 14505 probes passed check_dic selection.\n",
      "finish in 1.479s, 279 probes kept.\n",
      "-- check region:1265 19:23485001-23515000_strand_+_gene_1265, 59282 candidate probes\n",
      "--- 2747 probes passed check_dic selection.\n",
      "finish in 1.027s, 99 probes kept.\n",
      "-- check region:1266 19:24485001-24515000_strand_+_gene_1266, 20878 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.332s, 0 probes kept.\n",
      "-- check region:1267 19:25485001-25515000_strand_+_gene_1267, 22550 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.364s, 0 probes kept.\n",
      "-- check region:1268 19:26485001-26515000_strand_+_gene_1268, 31012 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.504s, 0 probes kept.\n",
      "-- check region:1269 19:27485001-27515000_strand_+_gene_1269, 57738 candidate probes\n",
      "--- 822 probes passed check_dic selection.\n",
      "finish in 0.936s, 51 probes kept.\n",
      "-- check region:1270 19:28485001-28515000_strand_+_gene_1270, 58418 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 19578 probes passed check_dic selection.\n",
      "finish in 1.645s, 305 probes kept.\n",
      "-- check region:1271 19:29485001-29515000_strand_+_gene_1271, 59160 candidate probes\n",
      "--- 21005 probes passed check_dic selection.\n",
      "finish in 1.716s, 312 probes kept.\n",
      "-- check region:1272 19:30485001-30515000_strand_+_gene_1272, 59460 candidate probes\n",
      "--- 21842 probes passed check_dic selection.\n",
      "finish in 1.730s, 317 probes kept.\n",
      "-- check region:1273 19:31485001-31515000_strand_+_gene_1273, 58640 candidate probes\n",
      "--- 19115 probes passed check_dic selection.\n",
      "finish in 1.628s, 297 probes kept.\n",
      "-- check region:1274 19:32485001-32515000_strand_+_gene_1274, 58402 candidate probes\n",
      "--- 9122 probes passed check_dic selection.\n",
      "finish in 1.250s, 192 probes kept.\n",
      "-- check region:1275 19:33485001-33515000_strand_+_gene_1275, 59294 candidate probes\n",
      "--- 24844 probes passed check_dic selection.\n",
      "finish in 1.858s, 356 probes kept.\n",
      "-- check region:1276 19:34485001-34515000_strand_+_gene_1276, 58174 candidate probes\n",
      "--- 11915 probes passed check_dic selection.\n",
      "finish in 1.394s, 244 probes kept.\n",
      "-- check region:1277 19:35485001-35515000_strand_+_gene_1277, 59040 candidate probes\n",
      "--- 21971 probes passed check_dic selection.\n",
      "finish in 1.768s, 362 probes kept.\n",
      "-- check region:1278 19:36485001-36515000_strand_+_gene_1278, 55924 candidate probes\n",
      "--- 6573 probes passed check_dic selection.\n",
      "finish in 1.138s, 150 probes kept.\n",
      "-- check region:1279 19:37485001-37515000_strand_+_gene_1279, 57780 candidate probes\n",
      "--- 9485 probes passed check_dic selection.\n",
      "finish in 1.264s, 193 probes kept.\n",
      "-- check region:1280 19:38485001-38515000_strand_+_gene_1280, 58442 candidate probes\n",
      "--- 17256 probes passed check_dic selection.\n",
      "finish in 1.581s, 296 probes kept.\n",
      "-- check region:1281 19:39485001-39515000_strand_+_gene_1281, 57530 candidate probes\n",
      "--- 15422 probes passed check_dic selection.\n",
      "finish in 1.488s, 275 probes kept.\n",
      "-- check region:1282 19:40485001-40515000_strand_+_gene_1282, 58330 candidate probes\n",
      "--- 13423 probes passed check_dic selection.\n",
      "finish in 1.438s, 234 probes kept.\n",
      "-- check region:1283 19:41485001-41515000_strand_+_gene_1283, 54888 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.898s, 0 probes kept.\n",
      "-- check region:1284 19:42485001-42515000_strand_+_gene_1284, 58452 candidate probes\n",
      "--- 8743 probes passed check_dic selection.\n",
      "finish in 1.251s, 188 probes kept.\n",
      "-- check region:1285 19:43485001-43515000_strand_+_gene_1285, 57426 candidate probes\n",
      "--- 11009 probes passed check_dic selection.\n",
      "finish in 1.331s, 206 probes kept.\n",
      "-- check region:1286 19:44485001-44515000_strand_+_gene_1286, 58624 candidate probes\n",
      "--- 15915 probes passed check_dic selection.\n",
      "finish in 1.505s, 263 probes kept.\n",
      "-- check region:1287 19:45485001-45515000_strand_+_gene_1287, 57798 candidate probes\n",
      "--- 16734 probes passed check_dic selection.\n",
      "finish in 1.525s, 287 probes kept.\n",
      "-- check region:1288 19:46485001-46515000_strand_+_gene_1288, 57602 candidate probes\n",
      "--- 14123 probes passed check_dic selection.\n",
      "finish in 1.452s, 239 probes kept.\n",
      "-- check region:1289 19:47485001-47515000_strand_+_gene_1289, 59248 candidate probes\n",
      "--- 23648 probes passed check_dic selection.\n",
      "finish in 1.835s, 368 probes kept.\n",
      "-- check region:1290 19:48485001-48515000_strand_+_gene_1290, 57934 candidate probes\n",
      "--- 18310 probes passed check_dic selection.\n",
      "finish in 1.575s, 315 probes kept.\n",
      "-- check region:1291 19:49485001-49515000_strand_+_gene_1291, 57822 candidate probes\n",
      "--- 16539 probes passed check_dic selection.\n",
      "finish in 1.544s, 273 probes kept.\n",
      "-- check region:1292 19:50485001-50515000_strand_+_gene_1292, 57788 candidate probes\n",
      "--- 12440 probes passed check_dic selection.\n",
      "finish in 1.394s, 248 probes kept.\n",
      "-- check region:1293 19:51485001-51515000_strand_+_gene_1293, 59632 candidate probes\n",
      "--- 16600 probes passed check_dic selection.\n",
      "finish in 1.569s, 312 probes kept.\n",
      "-- check region:1294 19:52485001-52515000_strand_+_gene_1294, 58292 candidate probes\n",
      "--- 2392 probes passed check_dic selection.\n",
      "finish in 1.023s, 90 probes kept.\n",
      "-- check region:1295 19:53485001-53515000_strand_+_gene_1295, 58898 candidate probes\n",
      "--- 7360 probes passed check_dic selection.\n",
      "finish in 1.211s, 167 probes kept.\n",
      "-- check region:1296 19:54485001-54515000_strand_+_gene_1296, 57672 candidate probes\n",
      "--- 16 probes passed check_dic selection.\n",
      "finish in 0.937s, 1 probes kept.\n",
      "-- check region:1297 19:55485001-55515000_strand_+_gene_1297, 57892 candidate probes\n",
      "--- 16913 probes passed check_dic selection.\n",
      "finish in 1.559s, 312 probes kept.\n",
      "-- check region:1298 19:56485001-56515000_strand_+_gene_1298, 59204 candidate probes\n",
      "--- 15083 probes passed check_dic selection.\n",
      "finish in 1.502s, 282 probes kept.\n",
      "-- check region:1299 19:57485001-57515000_strand_+_gene_1299, 55476 candidate probes\n",
      "--- 12190 probes passed check_dic selection.\n",
      "finish in 1.336s, 267 probes kept.\n",
      "-- check region:1300 20:485001-515000_strand_+_gene_1300, 58700 candidate probes\n",
      "--- 14106 probes passed check_dic selection.\n",
      "finish in 1.439s, 266 probes kept.\n",
      "-- check region:1301 20:1485001-1515000_strand_+_gene_1301, 59640 candidate probes\n",
      "--- 14457 probes passed check_dic selection.\n",
      "finish in 1.463s, 252 probes kept.\n",
      "-- check region:1302 20:2485001-2515000_strand_+_gene_1302, 55052 candidate probes\n",
      "--- 12281 probes passed check_dic selection.\n",
      "finish in 1.319s, 220 probes kept.\n",
      "-- check region:1303 20:3485001-3515000_strand_+_gene_1303, 51866 candidate probes\n",
      "--- 7947 probes passed check_dic selection.\n",
      "finish in 1.111s, 174 probes kept.\n",
      "-- check region:1304 20:4485001-4515000_strand_+_gene_1304, 58106 candidate probes\n",
      "--- 16252 probes passed check_dic selection.\n",
      "finish in 1.514s, 268 probes kept.\n",
      "-- check region:1305 20:5485001-5515000_strand_+_gene_1305, 59160 candidate probes\n",
      "--- 16302 probes passed check_dic selection.\n",
      "finish in 1.569s, 297 probes kept.\n",
      "-- check region:1306 20:6485001-6515000_strand_+_gene_1306, 59660 candidate probes\n",
      "--- 17777 probes passed check_dic selection.\n",
      "finish in 1.590s, 285 probes kept.\n",
      "-- check region:1307 20:7485001-7515000_strand_+_gene_1307, 59674 candidate probes\n",
      "--- 14377 probes passed check_dic selection.\n",
      "finish in 1.453s, 271 probes kept.\n",
      "-- check region:1308 20:8485001-8515000_strand_+_gene_1308, 58590 candidate probes\n",
      "--- 13312 probes passed check_dic selection.\n",
      "finish in 1.387s, 243 probes kept.\n",
      "-- check region:1309 20:9485001-9515000_strand_+_gene_1309, 59788 candidate probes\n",
      "--- 18555 probes passed check_dic selection.\n",
      "finish in 1.624s, 325 probes kept.\n",
      "-- check region:1310 20:10485001-10515000_strand_+_gene_1310, 59820 candidate probes\n",
      "--- 16428 probes passed check_dic selection.\n",
      "finish in 1.530s, 285 probes kept.\n",
      "-- check region:1311 20:11485001-11515000_strand_+_gene_1311, 59304 candidate probes\n",
      "--- 12063 probes passed check_dic selection.\n",
      "finish in 1.360s, 211 probes kept.\n",
      "-- check region:1312 20:12485001-12515000_strand_+_gene_1312, 55692 candidate probes\n",
      "--- 14455 probes passed check_dic selection.\n",
      "finish in 1.397s, 257 probes kept.\n",
      "-- check region:1313 20:13485001-13515000_strand_+_gene_1313, 58770 candidate probes\n",
      "--- 9311 probes passed check_dic selection.\n",
      "finish in 1.245s, 201 probes kept.\n",
      "-- check region:1314 20:14485001-14515000_strand_+_gene_1314, 59578 candidate probes\n",
      "--- 14057 probes passed check_dic selection.\n",
      "finish in 1.426s, 272 probes kept.\n",
      "-- check region:1315 20:15485001-15515000_strand_+_gene_1315, 59908 candidate probes\n",
      "--- 21051 probes passed check_dic selection.\n",
      "finish in 1.706s, 347 probes kept.\n",
      "-- check region:1316 20:16485001-16515000_strand_+_gene_1316, 59380 candidate probes\n",
      "--- 16276 probes passed check_dic selection.\n",
      "finish in 1.526s, 301 probes kept.\n",
      "-- check region:1317 20:17485001-17515000_strand_+_gene_1317, 59660 candidate probes\n",
      "--- 21960 probes passed check_dic selection.\n",
      "finish in 1.763s, 344 probes kept.\n",
      "-- check region:1318 20:18485001-18515000_strand_+_gene_1318, 57968 candidate probes\n",
      "--- 13162 probes passed check_dic selection.\n",
      "finish in 1.406s, 253 probes kept.\n",
      "-- check region:1319 20:19485001-19515000_strand_+_gene_1319, 59682 candidate probes\n",
      "--- 24951 probes passed check_dic selection.\n",
      "finish in 1.880s, 385 probes kept.\n",
      "-- check region:1320 20:20485001-20515000_strand_+_gene_1320, 59884 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 16153 probes passed check_dic selection.\n",
      "finish in 1.529s, 286 probes kept.\n",
      "-- check region:1321 20:21485001-21515000_strand_+_gene_1321, 59504 candidate probes\n",
      "--- 20139 probes passed check_dic selection.\n",
      "finish in 1.667s, 321 probes kept.\n",
      "-- check region:1322 20:22485001-22515000_strand_+_gene_1322, 59278 candidate probes\n",
      "--- 20471 probes passed check_dic selection.\n",
      "finish in 1.676s, 310 probes kept.\n",
      "-- check region:1323 20:23485001-23515000_strand_+_gene_1323, 59402 candidate probes\n",
      "--- 18457 probes passed check_dic selection.\n",
      "finish in 1.615s, 306 probes kept.\n",
      "-- check region:1324 20:24485001-24515000_strand_+_gene_1324, 58566 candidate probes\n",
      "--- 21023 probes passed check_dic selection.\n",
      "finish in 1.705s, 321 probes kept.\n",
      "-- check region:1325 20:25485001-25515000_strand_+_gene_1325, 59006 candidate probes\n",
      "--- 24951 probes passed check_dic selection.\n",
      "finish in 1.873s, 367 probes kept.\n",
      "-- check region:1326 20:26485001-26515000_strand_+_gene_1326, 14778 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.241s, 0 probes kept.\n",
      "-- check region:1327 20:27485001-27515000_strand_+_gene_1327, 6236 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.102s, 0 probes kept.\n",
      "-- check region:1328 20:28485001-28515000_strand_+_gene_1328, 28964 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.468s, 0 probes kept.\n",
      "-- check region:1329 20:29485001-29515000_strand_+_gene_1329, 59880 candidate probes\n",
      "--- 747 probes passed check_dic selection.\n",
      "finish in 0.955s, 35 probes kept.\n",
      "-- check region:1330 20:30485001-30515000_strand_+_gene_1330, 58496 candidate probes\n",
      "--- 4033 probes passed check_dic selection.\n",
      "finish in 1.095s, 129 probes kept.\n",
      "-- check region:1331 20:31485001-31515000_strand_+_gene_1331, 57724 candidate probes\n",
      "--- 13858 probes passed check_dic selection.\n",
      "finish in 1.432s, 242 probes kept.\n",
      "-- check region:1332 20:32485001-32515000_strand_+_gene_1332, 58594 candidate probes\n",
      "--- 17267 probes passed check_dic selection.\n",
      "finish in 1.566s, 278 probes kept.\n",
      "-- check region:1333 20:33485001-33515000_strand_+_gene_1333, 56884 candidate probes\n",
      "--- 7589 probes passed check_dic selection.\n",
      "finish in 1.189s, 136 probes kept.\n",
      "-- check region:1334 20:34485001-34515000_strand_+_gene_1334, 58802 candidate probes\n",
      "--- 6479 probes passed check_dic selection.\n",
      "finish in 1.160s, 161 probes kept.\n",
      "-- check region:1335 20:35485001-35515000_strand_+_gene_1335, 59078 candidate probes\n",
      "--- 17792 probes passed check_dic selection.\n",
      "finish in 1.612s, 326 probes kept.\n",
      "-- check region:1336 20:36485001-36515000_strand_+_gene_1336, 57734 candidate probes\n",
      "--- 15527 probes passed check_dic selection.\n",
      "finish in 1.507s, 266 probes kept.\n",
      "-- check region:1337 20:37485001-37515000_strand_+_gene_1337, 58956 candidate probes\n",
      "--- 18895 probes passed check_dic selection.\n",
      "finish in 1.635s, 339 probes kept.\n",
      "-- check region:1338 20:38485001-38515000_strand_+_gene_1338, 58212 candidate probes\n",
      "--- 9892 probes passed check_dic selection.\n",
      "finish in 1.272s, 188 probes kept.\n",
      "-- check region:1339 20:39485001-39515000_strand_+_gene_1339, 57442 candidate probes\n",
      "--- 13598 probes passed check_dic selection.\n",
      "finish in 1.394s, 238 probes kept.\n",
      "-- check region:1340 20:40485001-40515000_strand_+_gene_1340, 59726 candidate probes\n",
      "--- 23193 probes passed check_dic selection.\n",
      "finish in 1.796s, 344 probes kept.\n",
      "-- check region:1341 20:41485001-41515000_strand_+_gene_1341, 59860 candidate probes\n",
      "--- 18311 probes passed check_dic selection.\n",
      "finish in 1.610s, 308 probes kept.\n",
      "-- check region:1342 20:42485001-42515000_strand_+_gene_1342, 58744 candidate probes\n",
      "--- 12570 probes passed check_dic selection.\n",
      "finish in 1.371s, 261 probes kept.\n",
      "-- check region:1343 20:43485001-43515000_strand_+_gene_1343, 58452 candidate probes\n",
      "--- 11013 probes passed check_dic selection.\n",
      "finish in 1.338s, 201 probes kept.\n",
      "-- check region:1344 20:44485001-44515000_strand_+_gene_1344, 58736 candidate probes\n",
      "--- 12500 probes passed check_dic selection.\n",
      "finish in 1.408s, 274 probes kept.\n",
      "-- check region:1345 20:45485001-45515000_strand_+_gene_1345, 58952 candidate probes\n",
      "--- 17536 probes passed check_dic selection.\n",
      "finish in 1.583s, 312 probes kept.\n",
      "-- check region:1346 20:46485001-46515000_strand_+_gene_1346, 59012 candidate probes\n",
      "--- 14043 probes passed check_dic selection.\n",
      "finish in 1.445s, 298 probes kept.\n",
      "-- check region:1347 20:47485001-47515000_strand_+_gene_1347, 58054 candidate probes\n",
      "--- 9822 probes passed check_dic selection.\n",
      "finish in 1.278s, 179 probes kept.\n",
      "-- check region:1348 20:48485001-48515000_strand_+_gene_1348, 47686 candidate probes\n",
      "--- 4560 probes passed check_dic selection.\n",
      "finish in 0.944s, 130 probes kept.\n",
      "-- check region:1349 20:49485001-49515000_strand_+_gene_1349, 59304 candidate probes\n",
      "--- 16231 probes passed check_dic selection.\n",
      "finish in 1.546s, 302 probes kept.\n",
      "-- check region:1350 20:50485001-50515000_strand_+_gene_1350, 58630 candidate probes\n",
      "--- 16406 probes passed check_dic selection.\n",
      "finish in 1.526s, 265 probes kept.\n",
      "-- check region:1351 20:51485001-51515000_strand_+_gene_1351, 59144 candidate probes\n",
      "--- 23657 probes passed check_dic selection.\n",
      "finish in 1.822s, 353 probes kept.\n",
      "-- check region:1352 20:52485001-52515000_strand_+_gene_1352, 59268 candidate probes\n",
      "--- 14122 probes passed check_dic selection.\n",
      "finish in 1.436s, 243 probes kept.\n",
      "-- check region:1353 20:53485001-53515000_strand_+_gene_1353, 51788 candidate probes\n",
      "--- 9094 probes passed check_dic selection.\n",
      "finish in 1.162s, 192 probes kept.\n",
      "-- check region:1354 20:54485001-54515000_strand_+_gene_1354, 59166 candidate probes\n",
      "--- 14557 probes passed check_dic selection.\n",
      "finish in 1.452s, 278 probes kept.\n",
      "-- check region:1355 20:55485001-55515000_strand_+_gene_1355, 57148 candidate probes\n",
      "--- 9763 probes passed check_dic selection.\n",
      "finish in 1.253s, 187 probes kept.\n",
      "-- check region:1356 20:56485001-56515000_strand_+_gene_1356, 59004 candidate probes\n",
      "--- 16289 probes passed check_dic selection.\n",
      "finish in 1.530s, 291 probes kept.\n",
      "-- check region:1357 20:57485001-57515000_strand_+_gene_1357, 58458 candidate probes\n",
      "--- 12802 probes passed check_dic selection.\n",
      "finish in 1.403s, 248 probes kept.\n",
      "-- check region:1358 20:58485001-58515000_strand_+_gene_1358, 59608 candidate probes\n",
      "--- 20391 probes passed check_dic selection.\n",
      "finish in 1.681s, 327 probes kept.\n",
      "-- check region:1359 20:59485001-59515000_strand_+_gene_1359, 57672 candidate probes\n",
      "--- 16841 probes passed check_dic selection.\n",
      "finish in 1.538s, 267 probes kept.\n",
      "-- check region:1360 20:60485001-60515000_strand_+_gene_1360, 58954 candidate probes\n",
      "--- 21254 probes passed check_dic selection.\n",
      "finish in 1.711s, 324 probes kept.\n",
      "-- check region:1361 20:61485001-61515000_strand_+_gene_1361, 59334 candidate probes\n",
      "--- 27403 probes passed check_dic selection.\n",
      "finish in 1.945s, 397 probes kept.\n",
      "-- check region:1362 20:62485001-62515000_strand_+_gene_1362, 59254 candidate probes\n",
      "--- 25924 probes passed check_dic selection.\n",
      "finish in 1.928s, 385 probes kept.\n",
      "-- check region:1363 20:63485001-63515000_strand_+_gene_1363, 58414 candidate probes\n",
      "--- 19305 probes passed check_dic selection.\n",
      "finish in 1.614s, 314 probes kept.\n",
      "-- check region:1364 21:485001-515000_strand_+_gene_1364, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.002s, 0 probes kept.\n",
      "-- check region:1365 21:1485001-1515000_strand_+_gene_1365, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1366 21:2485001-2515000_strand_+_gene_1366, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1367 21:3485001-3515000_strand_+_gene_1367, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1368 21:4485001-4515000_strand_+_gene_1368, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1369 21:5485001-5515000_strand_+_gene_1369, 31680 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.510s, 0 probes kept.\n",
      "-- check region:1370 21:6485001-6515000_strand_+_gene_1370, 56744 candidate probes\n",
      "--- 285 probes passed check_dic selection.\n",
      "finish in 0.919s, 10 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:1371 21:7485001-7515000_strand_+_gene_1371, 31554 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.523s, 0 probes kept.\n",
      "-- check region:1372 21:8485001-8515000_strand_+_gene_1372, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1373 21:9485001-9515000_strand_+_gene_1373, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1374 21:10485001-10515000_strand_+_gene_1374, 58988 candidate probes\n",
      "--- 609 probes passed check_dic selection.\n",
      "finish in 0.941s, 20 probes kept.\n",
      "-- check region:1375 21:11485001-11515000_strand_+_gene_1375, 22244 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.358s, 0 probes kept.\n",
      "-- check region:1376 21:12485001-12515000_strand_+_gene_1376, 14540 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.236s, 0 probes kept.\n",
      "-- check region:1377 21:13485001-13515000_strand_+_gene_1377, 58784 candidate probes\n",
      "--- 511 probes passed check_dic selection.\n",
      "finish in 0.931s, 28 probes kept.\n",
      "-- check region:1378 21:14485001-14515000_strand_+_gene_1378, 59228 candidate probes\n",
      "--- 10625 probes passed check_dic selection.\n",
      "finish in 1.307s, 217 probes kept.\n",
      "-- check region:1379 21:15485001-15515000_strand_+_gene_1379, 59880 candidate probes\n",
      "--- 14963 probes passed check_dic selection.\n",
      "finish in 1.483s, 292 probes kept.\n",
      "-- check region:1380 21:16485001-16515000_strand_+_gene_1380, 59458 candidate probes\n",
      "--- 8539 probes passed check_dic selection.\n",
      "finish in 1.228s, 192 probes kept.\n",
      "-- check region:1381 21:17485001-17515000_strand_+_gene_1381, 57600 candidate probes\n",
      "--- 11588 probes passed check_dic selection.\n",
      "finish in 1.327s, 214 probes kept.\n",
      "-- check region:1382 21:18485001-18515000_strand_+_gene_1382, 59346 candidate probes\n",
      "--- 10356 probes passed check_dic selection.\n",
      "finish in 1.295s, 210 probes kept.\n",
      "-- check region:1383 21:19485001-19515000_strand_+_gene_1383, 59682 candidate probes\n",
      "--- 12771 probes passed check_dic selection.\n",
      "finish in 1.394s, 230 probes kept.\n",
      "-- check region:1384 21:20485001-20515000_strand_+_gene_1384, 59630 candidate probes\n",
      "--- 6867 probes passed check_dic selection.\n",
      "finish in 1.180s, 148 probes kept.\n",
      "-- check region:1385 21:21485001-21515000_strand_+_gene_1385, 59420 candidate probes\n",
      "--- 5879 probes passed check_dic selection.\n",
      "finish in 1.113s, 163 probes kept.\n",
      "-- check region:1386 21:22485001-22515000_strand_+_gene_1386, 59050 candidate probes\n",
      "--- 4613 probes passed check_dic selection.\n",
      "finish in 1.075s, 128 probes kept.\n",
      "-- check region:1387 21:23485001-23515000_strand_+_gene_1387, 59658 candidate probes\n",
      "--- 8588 probes passed check_dic selection.\n",
      "finish in 1.228s, 190 probes kept.\n",
      "-- check region:1388 21:24485001-24515000_strand_+_gene_1388, 59020 candidate probes\n",
      "--- 7306 probes passed check_dic selection.\n",
      "finish in 1.177s, 176 probes kept.\n",
      "-- check region:1389 21:25485001-25515000_strand_+_gene_1389, 59446 candidate probes\n",
      "--- 13679 probes passed check_dic selection.\n",
      "finish in 1.423s, 251 probes kept.\n",
      "-- check region:1390 21:26485001-26515000_strand_+_gene_1390, 59440 candidate probes\n",
      "--- 16291 probes passed check_dic selection.\n",
      "finish in 1.514s, 288 probes kept.\n",
      "-- check region:1391 21:27485001-27515000_strand_+_gene_1391, 59702 candidate probes\n",
      "--- 10896 probes passed check_dic selection.\n",
      "finish in 1.316s, 233 probes kept.\n",
      "-- check region:1392 21:28485001-28515000_strand_+_gene_1392, 59326 candidate probes\n",
      "--- 12032 probes passed check_dic selection.\n",
      "finish in 1.366s, 244 probes kept.\n",
      "-- check region:1393 21:29485001-29515000_strand_+_gene_1393, 59066 candidate probes\n",
      "--- 20103 probes passed check_dic selection.\n",
      "finish in 1.662s, 323 probes kept.\n",
      "-- check region:1394 21:30485001-30515000_strand_+_gene_1394, 59386 candidate probes\n",
      "--- 8207 probes passed check_dic selection.\n",
      "finish in 1.221s, 174 probes kept.\n",
      "-- check region:1395 21:31485001-31515000_strand_+_gene_1395, 58500 candidate probes\n",
      "--- 17069 probes passed check_dic selection.\n",
      "finish in 1.553s, 282 probes kept.\n",
      "-- check region:1396 21:32485001-32515000_strand_+_gene_1396, 59026 candidate probes\n",
      "--- 9040 probes passed check_dic selection.\n",
      "finish in 1.283s, 192 probes kept.\n",
      "-- check region:1397 21:33485001-33515000_strand_+_gene_1397, 57250 candidate probes\n",
      "--- 518 probes passed check_dic selection.\n",
      "finish in 0.940s, 12 probes kept.\n",
      "-- check region:1398 21:34485001-34515000_strand_+_gene_1398, 59502 candidate probes\n",
      "--- 10557 probes passed check_dic selection.\n",
      "finish in 1.337s, 184 probes kept.\n",
      "-- check region:1399 21:35485001-35515000_strand_+_gene_1399, 52718 candidate probes\n",
      "--- 16742 probes passed check_dic selection.\n",
      "finish in 1.438s, 268 probes kept.\n",
      "-- check region:1400 21:36485001-36515000_strand_+_gene_1400, 58570 candidate probes\n",
      "--- 19307 probes passed check_dic selection.\n",
      "finish in 1.646s, 308 probes kept.\n",
      "-- check region:1401 21:37485001-37515000_strand_+_gene_1401, 59804 candidate probes\n",
      "--- 12135 probes passed check_dic selection.\n",
      "finish in 1.378s, 268 probes kept.\n",
      "-- check region:1402 21:38485001-38515000_strand_+_gene_1402, 59248 candidate probes\n",
      "--- 19212 probes passed check_dic selection.\n",
      "finish in 1.627s, 308 probes kept.\n",
      "-- check region:1403 21:39485001-39515000_strand_+_gene_1403, 58465 candidate probes\n",
      "--- 12226 probes passed check_dic selection.\n",
      "finish in 1.371s, 223 probes kept.\n",
      "-- check region:1404 21:40485001-40515000_strand_+_gene_1404, 58622 candidate probes\n",
      "--- 13851 probes passed check_dic selection.\n",
      "finish in 1.417s, 235 probes kept.\n",
      "-- check region:1405 21:41485001-41515000_strand_+_gene_1405, 58990 candidate probes\n",
      "--- 21181 probes passed check_dic selection.\n",
      "finish in 1.707s, 336 probes kept.\n",
      "-- check region:1406 21:42485001-42515000_strand_+_gene_1406, 59476 candidate probes\n",
      "--- 18463 probes passed check_dic selection.\n",
      "finish in 1.625s, 328 probes kept.\n",
      "-- check region:1407 21:43485001-43515000_strand_+_gene_1407, 59266 candidate probes\n",
      "--- 594 probes passed check_dic selection.\n",
      "finish in 0.997s, 16 probes kept.\n",
      "-- check region:1408 21:44485001-44515000_strand_+_gene_1408, 57570 candidate probes\n",
      "--- 21544 probes passed check_dic selection.\n",
      "finish in 1.713s, 354 probes kept.\n",
      "-- check region:1409 21:45485001-45515000_strand_+_gene_1409, 57600 candidate probes\n",
      "--- 21632 probes passed check_dic selection.\n",
      "finish in 1.729s, 374 probes kept.\n",
      "-- check region:1410 22:485001-515000_strand_+_gene_1410, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1411 22:1485001-1515000_strand_+_gene_1411, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1412 22:2485001-2515000_strand_+_gene_1412, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1413 22:3485001-3515000_strand_+_gene_1413, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1414 22:4485001-4515000_strand_+_gene_1414, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1415 22:5485001-5515000_strand_+_gene_1415, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1416 22:6485001-6515000_strand_+_gene_1416, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1417 22:7485001-7515000_strand_+_gene_1417, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1418 22:8485001-8515000_strand_+_gene_1418, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1419 22:9485001-9515000_strand_+_gene_1419, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1420 22:10485001-10515000_strand_+_gene_1420, 9918 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.154s, 0 probes kept.\n",
      "-- check region:1421 22:11485001-11515000_strand_+_gene_1421, 24432 candidate probes\n",
      "--- 34 probes passed check_dic selection.\n",
      "finish in 0.390s, 2 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:1422 22:12485001-12515000_strand_+_gene_1422, 49006 candidate probes\n",
      "--- 6 probes passed check_dic selection.\n",
      "finish in 0.767s, 2 probes kept.\n",
      "-- check region:1423 22:13485001-13515000_strand_+_gene_1423, 30824 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.494s, 0 probes kept.\n",
      "-- check region:1424 22:14485001-14515000_strand_+_gene_1424, 24076 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.387s, 0 probes kept.\n",
      "-- check region:1425 22:15485001-15515000_strand_+_gene_1425, 57756 candidate probes\n",
      "--- 70 probes passed check_dic selection.\n",
      "finish in 0.915s, 4 probes kept.\n",
      "-- check region:1426 22:16485001-16515000_strand_+_gene_1426, 57836 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.924s, 0 probes kept.\n",
      "-- check region:1427 22:17485001-17515000_strand_+_gene_1427, 58050 candidate probes\n",
      "--- 13549 probes passed check_dic selection.\n",
      "finish in 1.415s, 253 probes kept.\n",
      "-- check region:1428 22:18485001-18515000_strand_+_gene_1428, 59258 candidate probes\n",
      "--- 12 probes passed check_dic selection.\n",
      "finish in 0.961s, 1 probes kept.\n",
      "-- check region:1429 22:19485001-19515000_strand_+_gene_1429, 58994 candidate probes\n",
      "--- 15548 probes passed check_dic selection.\n",
      "finish in 1.507s, 266 probes kept.\n",
      "-- check region:1430 22:20485001-20515000_strand_+_gene_1430, 57474 candidate probes\n",
      "--- 14857 probes passed check_dic selection.\n",
      "finish in 1.462s, 240 probes kept.\n",
      "-- check region:1431 22:21485001-21515000_strand_+_gene_1431, 58600 candidate probes\n",
      "--- 226 probes passed check_dic selection.\n",
      "finish in 0.975s, 10 probes kept.\n",
      "-- check region:1432 22:22485001-22515000_strand_+_gene_1432, 58804 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.941s, 0 probes kept.\n",
      "-- check region:1433 22:23485001-23515000_strand_+_gene_1433, 58844 candidate probes\n",
      "--- 652 probes passed check_dic selection.\n",
      "finish in 0.980s, 28 probes kept.\n",
      "-- check region:1434 22:24485001-24515000_strand_+_gene_1434, 58880 candidate probes\n",
      "--- 18319 probes passed check_dic selection.\n",
      "finish in 1.604s, 295 probes kept.\n",
      "-- check region:1435 22:25485001-25515000_strand_+_gene_1435, 59130 candidate probes\n",
      "--- 15385 probes passed check_dic selection.\n",
      "finish in 1.502s, 255 probes kept.\n",
      "-- check region:1436 22:26485001-26515000_strand_+_gene_1436, 59232 candidate probes\n",
      "--- 15898 probes passed check_dic selection.\n",
      "finish in 1.549s, 298 probes kept.\n",
      "-- check region:1437 22:27485001-27515000_strand_+_gene_1437, 59372 candidate probes\n",
      "--- 23777 probes passed check_dic selection.\n",
      "finish in 1.830s, 360 probes kept.\n",
      "-- check region:1438 22:28485001-28515000_strand_+_gene_1438, 58654 candidate probes\n",
      "--- 8365 probes passed check_dic selection.\n",
      "finish in 1.218s, 174 probes kept.\n",
      "-- check region:1439 22:29485001-29515000_strand_+_gene_1439, 57212 candidate probes\n",
      "--- 10096 probes passed check_dic selection.\n",
      "finish in 1.290s, 198 probes kept.\n",
      "-- check region:1440 22:30485001-30515000_strand_+_gene_1440, 57876 candidate probes\n",
      "--- 14068 probes passed check_dic selection.\n",
      "finish in 1.456s, 236 probes kept.\n",
      "-- check region:1441 22:31485001-31515000_strand_+_gene_1441, 58342 candidate probes\n",
      "--- 10172 probes passed check_dic selection.\n",
      "finish in 1.302s, 184 probes kept.\n",
      "-- check region:1442 22:32485001-32515000_strand_+_gene_1442, 58900 candidate probes\n",
      "--- 12718 probes passed check_dic selection.\n",
      "finish in 1.406s, 248 probes kept.\n",
      "-- check region:1443 22:33485001-33515000_strand_+_gene_1443, 59184 candidate probes\n",
      "--- 20151 probes passed check_dic selection.\n",
      "finish in 1.684s, 316 probes kept.\n",
      "-- check region:1444 22:34485001-34515000_strand_+_gene_1444, 59824 candidate probes\n",
      "--- 18930 probes passed check_dic selection.\n",
      "finish in 1.630s, 314 probes kept.\n",
      "-- check region:1445 22:35485001-35515000_strand_+_gene_1445, 58838 candidate probes\n",
      "--- 18043 probes passed check_dic selection.\n",
      "finish in 1.636s, 316 probes kept.\n",
      "-- check region:1446 22:36485001-36515000_strand_+_gene_1446, 58398 candidate probes\n",
      "--- 16684 probes passed check_dic selection.\n",
      "finish in 1.561s, 274 probes kept.\n",
      "-- check region:1447 22:37485001-37515000_strand_+_gene_1447, 58966 candidate probes\n",
      "--- 23550 probes passed check_dic selection.\n",
      "finish in 1.851s, 380 probes kept.\n",
      "-- check region:1448 22:38485001-38515000_strand_+_gene_1448, 59078 candidate probes\n",
      "--- 12851 probes passed check_dic selection.\n",
      "finish in 1.421s, 267 probes kept.\n",
      "-- check region:1449 22:39485001-39515000_strand_+_gene_1449, 58714 candidate probes\n",
      "--- 19186 probes passed check_dic selection.\n",
      "finish in 1.662s, 343 probes kept.\n",
      "-- check region:1450 22:40485001-40515000_strand_+_gene_1450, 57068 candidate probes\n",
      "--- 11704 probes passed check_dic selection.\n",
      "finish in 1.314s, 211 probes kept.\n",
      "-- check region:1451 22:41485001-41515000_strand_+_gene_1451, 58680 candidate probes\n",
      "--- 16779 probes passed check_dic selection.\n",
      "finish in 1.557s, 255 probes kept.\n",
      "-- check region:1452 22:42485001-42515000_strand_+_gene_1452, 58410 candidate probes\n",
      "--- 5656 probes passed check_dic selection.\n",
      "finish in 1.162s, 113 probes kept.\n",
      "-- check region:1453 22:43485001-43515000_strand_+_gene_1453, 58166 candidate probes\n",
      "--- 19071 probes passed check_dic selection.\n",
      "finish in 1.640s, 304 probes kept.\n",
      "-- check region:1454 22:44485001-44515000_strand_+_gene_1454, 59152 candidate probes\n",
      "--- 13931 probes passed check_dic selection.\n",
      "finish in 1.465s, 277 probes kept.\n",
      "-- check region:1455 22:45485001-45515000_strand_+_gene_1455, 59142 candidate probes\n",
      "--- 17915 probes passed check_dic selection.\n",
      "finish in 1.587s, 290 probes kept.\n",
      "-- check region:1456 22:46485001-46515000_strand_+_gene_1456, 58534 candidate probes\n",
      "--- 18561 probes passed check_dic selection.\n",
      "finish in 1.615s, 270 probes kept.\n",
      "-- check region:1457 22:47485001-47515000_strand_+_gene_1457, 59836 candidate probes\n",
      "--- 16255 probes passed check_dic selection.\n",
      "finish in 1.532s, 281 probes kept.\n",
      "-- check region:1458 22:48485001-48515000_strand_+_gene_1458, 58512 candidate probes\n",
      "--- 27765 probes passed check_dic selection.\n",
      "finish in 1.949s, 396 probes kept.\n",
      "-- check region:1459 22:49485001-49515000_strand_+_gene_1459, 58628 candidate probes\n",
      "--- 21007 probes passed check_dic selection.\n",
      "finish in 1.699s, 326 probes kept.\n",
      "-- check region:1460 X:1235001-1265000_strand_+_gene_1460, 55362 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.899s, 0 probes kept.\n",
      "-- check region:1461 X:3735001-3765000_strand_+_gene_1461, 58728 candidate probes\n",
      "--- 6190 probes passed check_dic selection.\n",
      "finish in 1.162s, 143 probes kept.\n",
      "-- check region:1462 X:6235001-6265000_strand_+_gene_1462, 59254 candidate probes\n",
      "--- 12499 probes passed check_dic selection.\n",
      "finish in 1.378s, 238 probes kept.\n",
      "-- check region:1463 X:8735001-8765000_strand_+_gene_1463, 59380 candidate probes\n",
      "--- 14874 probes passed check_dic selection.\n",
      "finish in 1.470s, 268 probes kept.\n",
      "-- check region:1464 X:11235001-11265000_strand_+_gene_1464, 59812 candidate probes\n",
      "--- 15527 probes passed check_dic selection.\n",
      "finish in 1.503s, 281 probes kept.\n",
      "-- check region:1465 X:13735001-13765000_strand_+_gene_1465, 59464 candidate probes\n",
      "--- 10659 probes passed check_dic selection.\n",
      "finish in 1.319s, 231 probes kept.\n",
      "-- check region:1466 X:16235001-16265000_strand_+_gene_1466, 58333 candidate probes\n",
      "--- 15096 probes passed check_dic selection.\n",
      "finish in 1.460s, 286 probes kept.\n",
      "-- check region:1467 X:18735001-18765000_strand_+_gene_1467, 57376 candidate probes\n",
      "--- 9946 probes passed check_dic selection.\n",
      "finish in 1.265s, 193 probes kept.\n",
      "-- check region:1468 X:21235001-21265000_strand_+_gene_1468, 58220 candidate probes\n",
      "--- 7266 probes passed check_dic selection.\n",
      "finish in 1.172s, 180 probes kept.\n",
      "-- check region:1469 X:23735001-23765000_strand_+_gene_1469, 58684 candidate probes\n",
      "--- 12063 probes passed check_dic selection.\n",
      "finish in 1.370s, 202 probes kept.\n",
      "-- check region:1470 X:26235001-26265000_strand_+_gene_1470, 59730 candidate probes\n",
      "--- 7810 probes passed check_dic selection.\n",
      "finish in 1.205s, 194 probes kept.\n",
      "-- check region:1471 X:28735001-28765000_strand_+_gene_1471, 59248 candidate probes\n",
      "--- 10801 probes passed check_dic selection.\n",
      "finish in 1.326s, 221 probes kept.\n",
      "-- check region:1472 X:31235001-31265000_strand_+_gene_1472, 59684 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 14614 probes passed check_dic selection.\n",
      "finish in 1.463s, 278 probes kept.\n",
      "-- check region:1473 X:33735001-33765000_strand_+_gene_1473, 59578 candidate probes\n",
      "--- 5273 probes passed check_dic selection.\n",
      "finish in 1.121s, 117 probes kept.\n",
      "-- check region:1474 X:36235001-36265000_strand_+_gene_1474, 59624 candidate probes\n",
      "--- 8020 probes passed check_dic selection.\n",
      "finish in 1.218s, 173 probes kept.\n",
      "-- check region:1475 X:38735001-38765000_strand_+_gene_1475, 59298 candidate probes\n",
      "--- 13004 probes passed check_dic selection.\n",
      "finish in 1.402s, 234 probes kept.\n",
      "-- check region:1476 X:41235001-41265000_strand_+_gene_1476, 58072 candidate probes\n",
      "--- 10764 probes passed check_dic selection.\n",
      "finish in 1.325s, 197 probes kept.\n",
      "-- check region:1477 X:43735001-43765000_strand_+_gene_1477, 59508 candidate probes\n",
      "--- 20954 probes passed check_dic selection.\n",
      "finish in 1.703s, 341 probes kept.\n",
      "-- check region:1478 X:46235001-46265000_strand_+_gene_1478, 59218 candidate probes\n",
      "--- 12451 probes passed check_dic selection.\n",
      "finish in 1.391s, 230 probes kept.\n",
      "-- check region:1479 X:48735001-48765000_strand_+_gene_1479, 59316 candidate probes\n",
      "--- 13858 probes passed check_dic selection.\n",
      "finish in 1.462s, 216 probes kept.\n",
      "-- check region:1480 X:51235001-51265000_strand_+_gene_1480, 52854 candidate probes\n",
      "--- 5648 probes passed check_dic selection.\n",
      "finish in 1.033s, 117 probes kept.\n",
      "-- check region:1481 X:53735001-53765000_strand_+_gene_1481, 58697 candidate probes\n",
      "--- 9625 probes passed check_dic selection.\n",
      "finish in 1.273s, 198 probes kept.\n",
      "-- check region:1482 X:56235001-56265000_strand_+_gene_1482, 59576 candidate probes\n",
      "--- 9227 probes passed check_dic selection.\n",
      "finish in 1.272s, 212 probes kept.\n",
      "-- check region:1483 X:58735001-58765000_strand_+_gene_1483, 9078 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.149s, 0 probes kept.\n",
      "-- check region:1484 X:61235001-61265000_strand_+_gene_1484, 14938 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.245s, 0 probes kept.\n",
      "-- check region:1485 X:63735001-63765000_strand_+_gene_1485, 59766 candidate probes\n",
      "--- 21627 probes passed check_dic selection.\n",
      "finish in 1.762s, 351 probes kept.\n",
      "-- check region:1486 X:66235001-66265000_strand_+_gene_1486, 59378 candidate probes\n",
      "--- 11101 probes passed check_dic selection.\n",
      "finish in 1.331s, 227 probes kept.\n",
      "-- check region:1487 X:68735001-68765000_strand_+_gene_1487, 54946 candidate probes\n",
      "--- 16748 probes passed check_dic selection.\n",
      "finish in 1.496s, 250 probes kept.\n",
      "-- check region:1488 X:71235001-71265000_strand_+_gene_1488, 59148 candidate probes\n",
      "--- 17961 probes passed check_dic selection.\n",
      "finish in 1.637s, 311 probes kept.\n",
      "-- check region:1489 X:73735001-73765000_strand_+_gene_1489, 59226 candidate probes\n",
      "--- 3364 probes passed check_dic selection.\n",
      "finish in 1.043s, 94 probes kept.\n",
      "-- check region:1490 X:76235001-76265000_strand_+_gene_1490, 58148 candidate probes\n",
      "--- 6258 probes passed check_dic selection.\n",
      "finish in 1.142s, 128 probes kept.\n",
      "-- check region:1491 X:78735001-78765000_strand_+_gene_1491, 58668 candidate probes\n",
      "--- 11796 probes passed check_dic selection.\n",
      "finish in 1.349s, 239 probes kept.\n",
      "-- check region:1492 X:81235001-81265000_strand_+_gene_1492, 59818 candidate probes\n",
      "--- 11494 probes passed check_dic selection.\n",
      "finish in 1.349s, 238 probes kept.\n",
      "-- check region:1493 X:83735001-83765000_strand_+_gene_1493, 55734 candidate probes\n",
      "--- 4698 probes passed check_dic selection.\n",
      "finish in 1.040s, 129 probes kept.\n",
      "-- check region:1494 X:86235001-86265000_strand_+_gene_1494, 59918 candidate probes\n",
      "--- 8284 probes passed check_dic selection.\n",
      "finish in 1.227s, 189 probes kept.\n",
      "-- check region:1495 X:88735001-88765000_strand_+_gene_1495, 58958 candidate probes\n",
      "--- 9101 probes passed check_dic selection.\n",
      "finish in 1.247s, 199 probes kept.\n",
      "-- check region:1496 X:91235001-91265000_strand_+_gene_1496, 59392 candidate probes\n",
      "--- 2340 probes passed check_dic selection.\n",
      "finish in 1.004s, 72 probes kept.\n",
      "-- check region:1497 X:93735001-93765000_strand_+_gene_1497, 59808 candidate probes\n",
      "--- 11783 probes passed check_dic selection.\n",
      "finish in 1.357s, 230 probes kept.\n",
      "-- check region:1498 X:96235001-96265000_strand_+_gene_1498, 56348 candidate probes\n",
      "--- 6540 probes passed check_dic selection.\n",
      "finish in 1.124s, 146 probes kept.\n",
      "-- check region:1499 X:98735001-98765000_strand_+_gene_1499, 59510 candidate probes\n",
      "--- 7158 probes passed check_dic selection.\n",
      "finish in 1.181s, 160 probes kept.\n",
      "-- check region:1500 X:101235001-101265000_strand_+_gene_1500, 59685 candidate probes\n",
      "--- 21757 probes passed check_dic selection.\n",
      "finish in 1.762s, 369 probes kept.\n",
      "-- check region:1501 X:103735001-103765000_strand_+_gene_1501, 59456 candidate probes\n",
      "--- 14928 probes passed check_dic selection.\n",
      "finish in 1.497s, 253 probes kept.\n",
      "-- check region:1502 X:106235001-106265000_strand_+_gene_1502, 59546 candidate probes\n",
      "--- 6128 probes passed check_dic selection.\n",
      "finish in 1.151s, 139 probes kept.\n",
      "-- check region:1503 X:108735001-108765000_strand_+_gene_1503, 59272 candidate probes\n",
      "--- 9066 probes passed check_dic selection.\n",
      "finish in 1.253s, 186 probes kept.\n",
      "-- check region:1504 X:111235001-111265000_strand_+_gene_1504, 59732 candidate probes\n",
      "--- 16068 probes passed check_dic selection.\n",
      "finish in 1.532s, 285 probes kept.\n",
      "-- check region:1505 X:113735001-113765000_strand_+_gene_1505, 58880 candidate probes\n",
      "--- 12208 probes passed check_dic selection.\n",
      "finish in 1.372s, 239 probes kept.\n",
      "-- check region:1506 X:116235001-116265000_strand_+_gene_1506, 57626 candidate probes\n",
      "--- 7983 probes passed check_dic selection.\n",
      "finish in 1.210s, 163 probes kept.\n",
      "-- check region:1507 X:118735001-118765000_strand_+_gene_1507, 59792 candidate probes\n",
      "--- 13805 probes passed check_dic selection.\n",
      "finish in 1.448s, 242 probes kept.\n",
      "-- check region:1508 X:121235001-121265000_strand_+_gene_1508, 58342 candidate probes\n",
      "--- 9602 probes passed check_dic selection.\n",
      "finish in 1.258s, 189 probes kept.\n",
      "-- check region:1509 X:123735001-123765000_strand_+_gene_1509, 58824 candidate probes\n",
      "--- 10320 probes passed check_dic selection.\n",
      "finish in 1.327s, 188 probes kept.\n",
      "-- check region:1510 X:126235001-126265000_strand_+_gene_1510, 59190 candidate probes\n",
      "--- 7934 probes passed check_dic selection.\n",
      "finish in 1.206s, 176 probes kept.\n",
      "-- check region:1511 X:128735001-128765000_strand_+_gene_1511, 55998 candidate probes\n",
      "--- 10713 probes passed check_dic selection.\n",
      "finish in 1.265s, 218 probes kept.\n",
      "-- check region:1512 X:131235001-131265000_strand_+_gene_1512, 55748 candidate probes\n",
      "--- 14099 probes passed check_dic selection.\n",
      "finish in 1.391s, 248 probes kept.\n",
      "-- check region:1513 X:133735001-133765000_strand_+_gene_1513, 59614 candidate probes\n",
      "--- 13108 probes passed check_dic selection.\n",
      "finish in 1.412s, 245 probes kept.\n",
      "-- check region:1514 X:136235001-136265000_strand_+_gene_1514, 59634 candidate probes\n",
      "--- 13366 probes passed check_dic selection.\n",
      "finish in 1.425s, 262 probes kept.\n",
      "-- check region:1515 X:138735001-138765000_strand_+_gene_1515, 59892 candidate probes\n",
      "--- 22348 probes passed check_dic selection.\n",
      "finish in 1.756s, 357 probes kept.\n",
      "-- check region:1516 X:141235001-141265000_strand_+_gene_1516, 59238 candidate probes\n",
      "--- 5375 probes passed check_dic selection.\n",
      "finish in 1.136s, 103 probes kept.\n",
      "-- check region:1517 X:143735001-143765000_strand_+_gene_1517, 58711 candidate probes\n",
      "--- 11908 probes passed check_dic selection.\n",
      "finish in 1.340s, 232 probes kept.\n",
      "-- check region:1518 X:146235001-146265000_strand_+_gene_1518, 59350 candidate probes\n",
      "--- 10178 probes passed check_dic selection.\n",
      "finish in 1.302s, 217 probes kept.\n",
      "-- check region:1519 X:148735001-148765000_strand_+_gene_1519, 59806 candidate probes\n",
      "--- 13113 probes passed check_dic selection.\n",
      "finish in 1.401s, 239 probes kept.\n",
      "-- check region:1520 X:151235001-151265000_strand_+_gene_1520, 59636 candidate probes\n",
      "--- 12627 probes passed check_dic selection.\n",
      "finish in 1.402s, 221 probes kept.\n",
      "-- check region:1521 X:153735001-153765000_strand_+_gene_1521, 59770 candidate probes\n",
      "--- 9160 probes passed check_dic selection.\n",
      "finish in 1.300s, 176 probes kept.\n",
      "-- check region:1522 Y:485001-515000_strand_+_gene_1522, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:1523 Y:1485001-1515000_strand_+_gene_1523, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1524 Y:2485001-2515000_strand_+_gene_1524, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1525 Y:3485001-3515000_strand_+_gene_1525, 59838 candidate probes\n",
      "--- 1152 probes passed check_dic selection.\n",
      "finish in 0.976s, 51 probes kept.\n",
      "-- check region:1526 Y:4485001-4515000_strand_+_gene_1526, 59752 candidate probes\n",
      "--- 1756 probes passed check_dic selection.\n",
      "finish in 0.998s, 60 probes kept.\n",
      "-- check region:1527 Y:5485001-5515000_strand_+_gene_1527, 59722 candidate probes\n",
      "--- 1845 probes passed check_dic selection.\n",
      "finish in 0.989s, 72 probes kept.\n",
      "-- check region:1528 Y:6485001-6515000_strand_+_gene_1528, 58818 candidate probes\n",
      "--- 1213 probes passed check_dic selection.\n",
      "finish in 0.984s, 44 probes kept.\n",
      "-- check region:1529 Y:7485001-7515000_strand_+_gene_1529, 59780 candidate probes\n",
      "--- 6251 probes passed check_dic selection.\n",
      "finish in 1.179s, 166 probes kept.\n",
      "-- check region:1530 Y:8485001-8515000_strand_+_gene_1530, 59662 candidate probes\n",
      "--- 8003 probes passed check_dic selection.\n",
      "finish in 1.245s, 218 probes kept.\n",
      "-- check region:1531 Y:9485001-9515000_strand_+_gene_1531, 43183 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.692s, 0 probes kept.\n",
      "-- check region:1532 Y:10485001-10515000_strand_+_gene_1532, 14916 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.235s, 0 probes kept.\n",
      "-- check region:1533 Y:11485001-11515000_strand_+_gene_1533, 59602 candidate probes\n",
      "--- 1038 probes passed check_dic selection.\n",
      "finish in 0.980s, 69 probes kept.\n",
      "-- check region:1534 Y:12485001-12515000_strand_+_gene_1534, 59602 candidate probes\n",
      "--- 5891 probes passed check_dic selection.\n",
      "finish in 1.178s, 149 probes kept.\n",
      "-- check region:1535 Y:13485001-13515000_strand_+_gene_1535, 59790 candidate probes\n",
      "--- 8766 probes passed check_dic selection.\n",
      "finish in 1.261s, 187 probes kept.\n",
      "-- check region:1536 Y:14485001-14515000_strand_+_gene_1536, 59418 candidate probes\n",
      "--- 8322 probes passed check_dic selection.\n",
      "finish in 1.251s, 187 probes kept.\n",
      "-- check region:1537 Y:15485001-15515000_strand_+_gene_1537, 59910 candidate probes\n",
      "--- 7808 probes passed check_dic selection.\n",
      "finish in 1.229s, 170 probes kept.\n",
      "-- check region:1538 Y:16485001-16515000_strand_+_gene_1538, 59362 candidate probes\n",
      "--- 6829 probes passed check_dic selection.\n",
      "finish in 1.165s, 164 probes kept.\n",
      "-- check region:1539 Y:17485001-17515000_strand_+_gene_1539, 59880 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.940s, 0 probes kept.\n",
      "-- check region:1540 Y:18485001-18515000_strand_+_gene_1540, 59529 candidate probes\n",
      "--- 3 probes passed check_dic selection.\n",
      "finish in 0.920s, 1 probes kept.\n",
      "-- check region:1541 Y:19485001-19515000_strand_+_gene_1541, 59676 candidate probes\n",
      "--- 13487 probes passed check_dic selection.\n",
      "finish in 1.427s, 251 probes kept.\n",
      "-- check region:1542 Y:20485001-20515000_strand_+_gene_1542, 58894 candidate probes\n",
      "--- 10008 probes passed check_dic selection.\n",
      "finish in 1.274s, 204 probes kept.\n",
      "-- check region:1543 Y:21485001-21515000_strand_+_gene_1543, 59474 candidate probes\n",
      "--- 670 probes passed check_dic selection.\n",
      "finish in 0.958s, 30 probes kept.\n",
      "-- check region:1544 Y:22485001-22515000_strand_+_gene_1544, 59894 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.966s, 0 probes kept.\n",
      "-- check region:1545 Y:23485001-23515000_strand_+_gene_1545, 59870 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.938s, 0 probes kept.\n",
      "-- check region:1546 Y:24485001-24515000_strand_+_gene_1546, 59200 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.951s, 0 probes kept.\n",
      "-- check region:1547 Y:25485001-25515000_strand_+_gene_1547, 59862 candidate probes\n",
      "--- 45 probes passed check_dic selection.\n",
      "finish in 0.959s, 2 probes kept.\n",
      "-- check region:1548 Y:26485001-26515000_strand_+_gene_1548, 59088 candidate probes\n",
      "--- 3246 probes passed check_dic selection.\n",
      "finish in 1.051s, 106 probes kept.\n",
      "-- check region:1549 Y:27485001-27515000_strand_+_gene_1549, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1550 Y:28485001-28515000_strand_+_gene_1550, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1551 Y:29485001-29515000_strand_+_gene_1551, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1552 Y:30485001-30515000_strand_+_gene_1552, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1553 Y:31485001-31515000_strand_+_gene_1553, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1554 Y:32485001-32515000_strand_+_gene_1554, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1555 Y:33485001-33515000_strand_+_gene_1555, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1556 Y:34485001-34515000_strand_+_gene_1556, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1557 Y:35485001-35515000_strand_+_gene_1557, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1558 Y:36485001-36515000_strand_+_gene_1558, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1559 Y:37485001-37515000_strand_+_gene_1559, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1560 Y:38485001-38515000_strand_+_gene_1560, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1561 Y:39485001-39515000_strand_+_gene_1561, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1562 Y:40485001-40515000_strand_+_gene_1562, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1563 Y:41485001-41515000_strand_+_gene_1563, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1564 Y:42485001-42515000_strand_+_gene_1564, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1565 Y:43485001-43515000_strand_+_gene_1565, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1566 Y:44485001-44515000_strand_+_gene_1566, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1567 Y:45485001-45515000_strand_+_gene_1567, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1568 Y:46485001-46515000_strand_+_gene_1568, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1569 Y:47485001-47515000_strand_+_gene_1569, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1570 Y:48485001-48515000_strand_+_gene_1570, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1571 Y:49485001-49515000_strand_+_gene_1571, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1572 Y:50485001-50515000_strand_+_gene_1572, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1573 Y:51485001-51515000_strand_+_gene_1573, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1574 Y:52485001-52515000_strand_+_gene_1574, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1575 Y:53485001-53515000_strand_+_gene_1575, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:1576 Y:54485001-54515000_strand_+_gene_1576, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.000s, 0 probes kept.\n",
      "-- check region:1577 Y:55485001-55515000_strand_+_gene_1577, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n",
      "-- check region:1578 Y:56485001-56515000_strand_+_gene_1578, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.001s, 0 probes kept.\n"
     ]
    }
   ],
   "source": [
    "pbs, pb_scores = pb_designer.check_probes()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ae777bea",
   "metadata": {},
   "source": [
    "### save probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "id": "4b217c7c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\reports\\merged_probes.pbr\n",
      "-- number of probes kept: 328466\n"
     ]
    }
   ],
   "source": [
    "overwrite_savefile = True \n",
    "if not os.path.exists(pb_designer.save_file) or overwrite_savefile:\n",
    "    #pb_designer.plots()\n",
    "    #pb_designer.save_csv()\n",
    "    pb_designer.save_to_file()\n",
    "print(f\"-- number of probes kept: {len(pb_designer.kept_probes)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "id": "f342e259",
   "metadata": {},
   "outputs": [],
   "source": [
    "del(pb_designer)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e728e379",
   "metadata": {},
   "source": [
    "## Reload from here:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "id": "842303ee",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Loading from savefile: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\reports\\merged_probes.pbr.\n",
      "- loading cand_probes from file\n",
      "- loading kept_probes from file\n",
      "- 328466 filtered probes loaded\n",
      "- loading sequence_dic from file\n",
      "- loading map_dic from file\n",
      "- loading params_dic from file\n",
      "- loading check_dic from file\n",
      "- loading save_file from file\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 61,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "verbose=True\n",
    "cand_prob_nums = []\n",
    "pb_dict = {}\n",
    "# Parse segemented list\n",
    "_pbde = library_tools.design.pb_reports_class()\n",
    "_pbde.load_from_file(probe_savefile)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1e26111e",
   "metadata": {},
   "outputs": [],
   "source": [
    "list(_pbde.kept_probes.values())[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "id": "779ab3ab",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0"
      ]
     },
     "execution_count": 72,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "id": "7a63afbb",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:0 with 344 probes\n",
      "-- appending region:1 with 319 probes\n",
      "-- appending region:2 with 377 probes\n",
      "-- appending region:3 with 242 probes\n",
      "-- appending region:4 with 256 probes\n",
      "-- appending region:5 with 303 probes\n",
      "-- appending region:6 with 300 probes\n",
      "-- appending region:7 with 341 probes\n",
      "-- appending region:8 with 323 probes\n",
      "-- appending region:9 with 264 probes\n",
      "-- appending region:10 with 216 probes\n",
      "-- appending region:11 with 212 probes\n",
      "-- appending region:12 with 240 probes\n",
      "-- appending region:13 with 341 probes\n",
      "-- appending region:14 with 176 probes\n",
      "-- appending region:15 with 343 probes\n",
      "-- appending region:16 with 245 probes\n",
      "-- appending region:17 with 203 probes\n",
      "-- appending region:18 with 204 probes\n",
      "-- appending region:19 with 391 probes\n",
      "-- appending region:20 with 206 probes\n",
      "-- appending region:21 with 255 probes\n",
      "-- appending region:22 with 303 probes\n",
      "-- appending region:23 with 333 probes\n",
      "-- appending region:24 with 279 probes\n",
      "-- appending region:25 with 238 probes\n",
      "-- appending region:26 with 328 probes\n",
      "-- appending region:27 with 149 probes\n",
      "-- appending region:28 with 262 probes\n",
      "-- appending region:29 with 199 probes\n",
      "-- appending region:30 with 263 probes\n",
      "-- appending region:31 with 215 probes\n",
      "-- appending region:32 with 190 probes\n",
      "-- appending region:33 with 338 probes\n",
      "-- appending region:34 with 230 probes\n",
      "-- appending region:35 with 203 probes\n",
      "-- appending region:36 with 154 probes\n",
      "-- appending region:37 with 358 probes\n",
      "-- appending region:38 with 211 probes\n",
      "-- appending region:39 with 237 probes\n",
      "-- appending region:40 with 293 probes\n",
      "-- appending region:41 with 22 probes\n",
      "-- appending region:42 with 201 probes\n",
      "-- appending region:43 with 235 probes\n",
      "-- appending region:44 with 317 probes\n",
      "-- appending region:45 with 187 probes\n",
      "-- appending region:46 with 309 probes\n",
      "-- appending region:47 with 213 probes\n",
      "-- appending region:48 with 3 probes\n",
      "-- appending region:57 with 60 probes\n",
      "-- appending region:60 with 187 probes\n",
      "-- appending region:62 with 330 probes\n",
      "-- appending region:63 with 241 probes\n",
      "-- appending region:64 with 199 probes\n",
      "-- appending region:65 with 114 probes\n",
      "-- appending region:66 with 127 probes\n",
      "-- appending region:67 with 271 probes\n",
      "-- appending region:68 with 226 probes\n",
      "-- appending region:69 with 128 probes\n",
      "-- appending region:70 with 120 probes\n",
      "-- appending region:71 with 137 probes\n",
      "-- appending region:72 with 267 probes\n",
      "-- appending region:73 with 198 probes\n",
      "-- appending region:74 with 173 probes\n",
      "-- appending region:75 with 149 probes\n",
      "-- appending region:76 with 121 probes\n",
      "-- appending region:77 with 218 probes\n",
      "-- appending region:78 with 142 probes\n",
      "-- appending region:79 with 198 probes\n",
      "-- appending region:80 with 247 probes\n",
      "-- appending region:81 with 223 probes\n",
      "-- appending region:82 with 8 probes\n",
      "-- appending region:83 with 259 probes\n",
      "-- appending region:84 with 191 probes\n",
      "-- appending region:85 with 313 probes\n",
      "-- appending region:86 with 237 probes\n",
      "-- appending region:87 with 236 probes\n",
      "-- appending region:88 with 204 probes\n",
      "-- appending region:89 with 335 probes\n",
      "-- appending region:90 with 118 probes\n",
      "-- appending region:91 with 363 probes\n",
      "-- appending region:92 with 271 probes\n",
      "-- appending region:93 with 212 probes\n",
      "-- appending region:94 with 178 probes\n",
      "-- appending region:95 with 214 probes\n",
      "-- appending region:96 with 258 probes\n",
      "-- appending region:98 with 168 probes\n",
      "-- appending region:100 with 344 probes\n",
      "-- appending region:101 with 286 probes\n",
      "-- appending region:102 with 240 probes\n",
      "-- appending region:103 with 205 probes\n",
      "-- appending region:104 with 119 probes\n",
      "-- appending region:105 with 352 probes\n",
      "-- appending region:106 with 244 probes\n",
      "-- appending region:107 with 305 probes\n",
      "-- appending region:108 with 213 probes\n",
      "-- appending region:109 with 225 probes\n",
      "-- appending region:110 with 230 probes\n",
      "-- appending region:111 with 358 probes\n",
      "-- appending region:112 with 167 probes\n",
      "-- appending region:114 with 215 probes\n",
      "-- appending region:115 with 224 probes\n",
      "-- appending region:116 with 228 probes\n",
      "-- appending region:117 with 276 probes\n",
      "-- appending region:118 with 216 probes\n",
      "-- appending region:119 with 221 probes\n",
      "-- appending region:120 with 200 probes\n",
      "-- appending region:121 with 295 probes\n",
      "-- appending region:122 with 270 probes\n",
      "-- appending region:123 with 182 probes\n",
      "-- appending region:124 with 235 probes\n",
      "-- appending region:125 with 251 probes\n",
      "-- appending region:126 with 291 probes\n",
      "-- appending region:127 with 222 probes\n",
      "-- appending region:128 with 275 probes\n",
      "-- appending region:129 with 216 probes\n",
      "-- appending region:130 with 162 probes\n",
      "-- appending region:131 with 190 probes\n",
      "-- appending region:132 with 192 probes\n",
      "-- appending region:133 with 312 probes\n",
      "-- appending region:134 with 237 probes\n",
      "-- appending region:137 with 315 probes\n",
      "-- appending region:138 with 285 probes\n",
      "-- appending region:139 with 174 probes\n",
      "-- appending region:140 with 261 probes\n",
      "-- appending region:141 with 74 probes\n",
      "-- appending region:142 with 39 probes\n",
      "-- appending region:143 with 186 probes\n",
      "-- appending region:144 with 187 probes\n",
      "-- appending region:145 with 178 probes\n",
      "-- appending region:146 with 294 probes\n",
      "-- appending region:147 with 355 probes\n",
      "-- appending region:148 with 193 probes\n",
      "-- appending region:149 with 309 probes\n",
      "-- appending region:150 with 240 probes\n",
      "-- appending region:151 with 3 probes\n",
      "-- appending region:152 with 248 probes\n",
      "-- appending region:153 with 307 probes\n",
      "-- appending region:154 with 283 probes\n",
      "-- appending region:155 with 252 probes\n",
      "-- appending region:156 with 292 probes\n",
      "-- appending region:157 with 255 probes\n",
      "-- appending region:158 with 227 probes\n",
      "-- appending region:159 with 231 probes\n",
      "-- appending region:160 with 136 probes\n",
      "-- appending region:161 with 134 probes\n",
      "-- appending region:162 with 338 probes\n",
      "-- appending region:163 with 234 probes\n",
      "-- appending region:164 with 303 probes\n",
      "-- appending region:165 with 244 probes\n",
      "-- appending region:166 with 304 probes\n",
      "-- appending region:167 with 217 probes\n",
      "-- appending region:168 with 301 probes\n",
      "-- appending region:169 with 238 probes\n",
      "-- appending region:170 with 303 probes\n",
      "-- appending region:171 with 272 probes\n",
      "-- appending region:172 with 200 probes\n",
      "-- appending region:173 with 173 probes\n",
      "-- appending region:174 with 198 probes\n",
      "-- appending region:175 with 317 probes\n",
      "-- appending region:176 with 152 probes\n",
      "-- appending region:177 with 279 probes\n",
      "-- appending region:178 with 280 probes\n",
      "-- appending region:179 with 230 probes\n",
      "-- appending region:180 with 226 probes\n",
      "-- appending region:181 with 248 probes\n",
      "-- appending region:182 with 93 probes\n",
      "-- appending region:183 with 204 probes\n",
      "-- appending region:184 with 173 probes\n",
      "-- appending region:185 with 313 probes\n",
      "-- appending region:186 with 301 probes\n",
      "-- appending region:187 with 239 probes\n",
      "-- appending region:188 with 233 probes\n",
      "-- appending region:189 with 191 probes\n",
      "-- appending region:190 with 150 probes\n",
      "-- appending region:191 with 274 probes\n",
      "-- appending region:192 with 270 probes\n",
      "-- appending region:193 with 357 probes\n",
      "-- appending region:194 with 297 probes\n",
      "-- appending region:195 with 199 probes\n",
      "-- appending region:196 with 317 probes\n",
      "-- appending region:197 with 251 probes\n",
      "-- appending region:198 with 379 probes\n",
      "-- appending region:199 with 275 probes\n",
      "-- appending region:200 with 229 probes\n",
      "-- appending region:201 with 361 probes\n",
      "-- appending region:202 with 331 probes\n",
      "-- appending region:203 with 190 probes\n",
      "-- appending region:204 with 265 probes\n",
      "-- appending region:205 with 241 probes\n",
      "-- appending region:206 with 262 probes\n",
      "-- appending region:207 with 324 probes\n",
      "-- appending region:208 with 174 probes\n",
      "-- appending region:209 with 180 probes\n",
      "-- appending region:210 with 365 probes\n",
      "-- appending region:211 with 349 probes\n",
      "-- appending region:212 with 330 probes\n",
      "-- appending region:213 with 200 probes\n",
      "-- appending region:214 with 219 probes\n",
      "-- appending region:215 with 142 probes\n",
      "-- appending region:216 with 408 probes\n",
      "-- appending region:217 with 278 probes\n",
      "-- appending region:218 with 252 probes\n",
      "-- appending region:219 with 258 probes\n",
      "-- appending region:220 with 214 probes\n",
      "-- appending region:221 with 256 probes\n",
      "-- appending region:222 with 292 probes\n",
      "-- appending region:223 with 318 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:224 with 322 probes\n",
      "-- appending region:225 with 234 probes\n",
      "-- appending region:226 with 220 probes\n",
      "-- appending region:227 with 242 probes\n",
      "-- appending region:228 with 148 probes\n",
      "-- appending region:229 with 197 probes\n",
      "-- appending region:230 with 166 probes\n",
      "-- appending region:232 with 23 probes\n",
      "-- appending region:233 with 196 probes\n",
      "-- appending region:234 with 255 probes\n",
      "-- appending region:235 with 233 probes\n",
      "-- appending region:236 with 182 probes\n",
      "-- appending region:237 with 179 probes\n",
      "-- appending region:238 with 251 probes\n",
      "-- appending region:239 with 231 probes\n",
      "-- appending region:240 with 234 probes\n",
      "-- appending region:241 with 272 probes\n",
      "-- appending region:242 with 288 probes\n",
      "-- appending region:243 with 165 probes\n",
      "-- appending region:244 with 333 probes\n",
      "-- appending region:245 with 238 probes\n",
      "-- appending region:246 with 205 probes\n",
      "-- appending region:247 with 234 probes\n",
      "-- appending region:248 with 277 probes\n",
      "-- appending region:249 with 242 probes\n",
      "-- appending region:250 with 172 probes\n",
      "-- appending region:251 with 357 probes\n",
      "-- appending region:252 with 287 probes\n",
      "-- appending region:253 with 188 probes\n",
      "-- appending region:254 with 282 probes\n",
      "-- appending region:255 with 225 probes\n",
      "-- appending region:256 with 218 probes\n",
      "-- appending region:257 with 288 probes\n",
      "-- appending region:258 with 314 probes\n",
      "-- appending region:259 with 160 probes\n",
      "-- appending region:260 with 162 probes\n",
      "-- appending region:261 with 159 probes\n",
      "-- appending region:262 with 179 probes\n",
      "-- appending region:263 with 324 probes\n",
      "-- appending region:264 with 259 probes\n",
      "-- appending region:265 with 220 probes\n",
      "-- appending region:266 with 251 probes\n",
      "-- appending region:267 with 263 probes\n",
      "-- appending region:268 with 175 probes\n",
      "-- appending region:269 with 351 probes\n",
      "-- appending region:270 with 313 probes\n",
      "-- appending region:271 with 220 probes\n",
      "-- appending region:272 with 232 probes\n",
      "-- appending region:273 with 231 probes\n",
      "-- appending region:274 with 330 probes\n",
      "-- appending region:275 with 390 probes\n",
      "-- appending region:276 with 325 probes\n",
      "-- appending region:277 with 313 probes\n",
      "-- appending region:278 with 224 probes\n",
      "-- appending region:279 with 210 probes\n",
      "-- appending region:280 with 309 probes\n",
      "-- appending region:281 with 239 probes\n",
      "-- appending region:282 with 262 probes\n",
      "-- appending region:283 with 293 probes\n",
      "-- appending region:284 with 215 probes\n",
      "-- appending region:285 with 234 probes\n",
      "-- appending region:286 with 187 probes\n",
      "-- appending region:287 with 194 probes\n",
      "-- appending region:288 with 249 probes\n",
      "-- appending region:289 with 250 probes\n",
      "-- appending region:290 with 253 probes\n",
      "-- appending region:291 with 195 probes\n",
      "-- appending region:292 with 226 probes\n",
      "-- appending region:293 with 157 probes\n",
      "-- appending region:295 with 209 probes\n",
      "-- appending region:296 with 274 probes\n",
      "-- appending region:297 with 194 probes\n",
      "-- appending region:298 with 253 probes\n",
      "-- appending region:299 with 141 probes\n",
      "-- appending region:300 with 151 probes\n",
      "-- appending region:301 with 25 probes\n",
      "-- appending region:302 with 328 probes\n",
      "-- appending region:303 with 213 probes\n",
      "-- appending region:304 with 232 probes\n",
      "-- appending region:305 with 222 probes\n",
      "-- appending region:306 with 242 probes\n",
      "-- appending region:307 with 231 probes\n",
      "-- appending region:308 with 203 probes\n",
      "-- appending region:309 with 343 probes\n",
      "-- appending region:310 with 166 probes\n",
      "-- appending region:311 with 267 probes\n",
      "-- appending region:312 with 241 probes\n",
      "-- appending region:313 with 182 probes\n",
      "-- appending region:314 with 253 probes\n",
      "-- appending region:315 with 228 probes\n",
      "-- appending region:316 with 183 probes\n",
      "-- appending region:317 with 251 probes\n",
      "-- appending region:318 with 178 probes\n",
      "-- appending region:319 with 249 probes\n",
      "-- appending region:320 with 155 probes\n",
      "-- appending region:321 with 279 probes\n",
      "-- appending region:322 with 258 probes\n",
      "-- appending region:323 with 248 probes\n",
      "-- appending region:324 with 171 probes\n",
      "-- appending region:325 with 176 probes\n",
      "-- appending region:326 with 191 probes\n",
      "-- appending region:327 with 106 probes\n",
      "-- appending region:328 with 170 probes\n",
      "-- appending region:329 with 295 probes\n",
      "-- appending region:330 with 218 probes\n",
      "-- appending region:331 with 154 probes\n",
      "-- appending region:332 with 377 probes\n",
      "-- appending region:333 with 214 probes\n",
      "-- appending region:334 with 234 probes\n",
      "-- appending region:335 with 271 probes\n",
      "-- appending region:336 with 244 probes\n",
      "-- appending region:337 with 181 probes\n",
      "-- appending region:338 with 222 probes\n",
      "-- appending region:339 with 257 probes\n",
      "-- appending region:340 with 201 probes\n",
      "-- appending region:341 with 284 probes\n",
      "-- appending region:342 with 157 probes\n",
      "-- appending region:343 with 260 probes\n",
      "-- appending region:344 with 234 probes\n",
      "-- appending region:345 with 225 probes\n",
      "-- appending region:346 with 279 probes\n",
      "-- appending region:347 with 202 probes\n",
      "-- appending region:348 with 223 probes\n",
      "-- appending region:349 with 161 probes\n",
      "-- appending region:350 with 415 probes\n",
      "-- appending region:351 with 319 probes\n",
      "-- appending region:352 with 325 probes\n",
      "-- appending region:353 with 158 probes\n",
      "-- appending region:354 with 293 probes\n",
      "-- appending region:355 with 234 probes\n",
      "-- appending region:356 with 257 probes\n",
      "-- appending region:357 with 170 probes\n",
      "-- appending region:358 with 216 probes\n",
      "-- appending region:359 with 179 probes\n",
      "-- appending region:360 with 190 probes\n",
      "-- appending region:361 with 175 probes\n",
      "-- appending region:362 with 315 probes\n",
      "-- appending region:363 with 350 probes\n",
      "-- appending region:364 with 253 probes\n",
      "-- appending region:365 with 284 probes\n",
      "-- appending region:366 with 214 probes\n",
      "-- appending region:367 with 102 probes\n",
      "-- appending region:368 with 64 probes\n",
      "-- appending region:370 with 158 probes\n",
      "-- appending region:371 with 84 probes\n",
      "-- appending region:372 with 236 probes\n",
      "-- appending region:373 with 270 probes\n",
      "-- appending region:374 with 224 probes\n",
      "-- appending region:375 with 226 probes\n",
      "-- appending region:376 with 222 probes\n",
      "-- appending region:377 with 340 probes\n",
      "-- appending region:379 with 287 probes\n",
      "-- appending region:380 with 224 probes\n",
      "-- appending region:381 with 278 probes\n",
      "-- appending region:382 with 318 probes\n",
      "-- appending region:383 with 176 probes\n",
      "-- appending region:384 with 166 probes\n",
      "-- appending region:385 with 272 probes\n",
      "-- appending region:386 with 215 probes\n",
      "-- appending region:387 with 192 probes\n",
      "-- appending region:388 with 277 probes\n",
      "-- appending region:389 with 170 probes\n",
      "-- appending region:390 with 164 probes\n",
      "-- appending region:391 with 257 probes\n",
      "-- appending region:392 with 202 probes\n",
      "-- appending region:393 with 229 probes\n",
      "-- appending region:394 with 309 probes\n",
      "-- appending region:395 with 256 probes\n",
      "-- appending region:396 with 182 probes\n",
      "-- appending region:397 with 245 probes\n",
      "-- appending region:398 with 162 probes\n",
      "-- appending region:399 with 196 probes\n",
      "-- appending region:400 with 222 probes\n",
      "-- appending region:401 with 237 probes\n",
      "-- appending region:402 with 191 probes\n",
      "-- appending region:403 with 377 probes\n",
      "-- appending region:404 with 331 probes\n",
      "-- appending region:405 with 205 probes\n",
      "-- appending region:406 with 261 probes\n",
      "-- appending region:407 with 306 probes\n",
      "-- appending region:408 with 163 probes\n",
      "-- appending region:409 with 221 probes\n",
      "-- appending region:410 with 271 probes\n",
      "-- appending region:411 with 282 probes\n",
      "-- appending region:412 with 212 probes\n",
      "-- appending region:413 with 339 probes\n",
      "-- appending region:414 with 188 probes\n",
      "-- appending region:415 with 252 probes\n",
      "-- appending region:416 with 252 probes\n",
      "-- appending region:417 with 315 probes\n",
      "-- appending region:418 with 200 probes\n",
      "-- appending region:419 with 355 probes\n",
      "-- appending region:420 with 37 probes\n",
      "-- appending region:421 with 212 probes\n",
      "-- appending region:422 with 298 probes\n",
      "-- appending region:423 with 345 probes\n",
      "-- appending region:424 with 323 probes\n",
      "-- appending region:425 with 221 probes\n",
      "-- appending region:426 with 350 probes\n",
      "-- appending region:427 with 217 probes\n",
      "-- appending region:428 with 243 probes\n",
      "-- appending region:429 with 193 probes\n",
      "-- appending region:430 with 236 probes\n",
      "-- appending region:431 with 237 probes\n",
      "-- appending region:432 with 270 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:434 with 5 probes\n",
      "-- appending region:435 with 357 probes\n",
      "-- appending region:436 with 294 probes\n",
      "-- appending region:437 with 188 probes\n",
      "-- appending region:438 with 206 probes\n",
      "-- appending region:439 with 310 probes\n",
      "-- appending region:440 with 321 probes\n",
      "-- appending region:441 with 183 probes\n",
      "-- appending region:442 with 276 probes\n",
      "-- appending region:443 with 241 probes\n",
      "-- appending region:444 with 307 probes\n",
      "-- appending region:446 with 108 probes\n",
      "-- appending region:447 with 181 probes\n",
      "-- appending region:448 with 127 probes\n",
      "-- appending region:449 with 228 probes\n",
      "-- appending region:450 with 268 probes\n",
      "-- appending region:451 with 258 probes\n",
      "-- appending region:452 with 197 probes\n",
      "-- appending region:453 with 67 probes\n",
      "-- appending region:454 with 145 probes\n",
      "-- appending region:455 with 142 probes\n",
      "-- appending region:456 with 217 probes\n",
      "-- appending region:457 with 207 probes\n",
      "-- appending region:458 with 236 probes\n",
      "-- appending region:459 with 180 probes\n",
      "-- appending region:460 with 202 probes\n",
      "-- appending region:461 with 209 probes\n",
      "-- appending region:462 with 170 probes\n",
      "-- appending region:463 with 143 probes\n",
      "-- appending region:464 with 211 probes\n",
      "-- appending region:465 with 352 probes\n",
      "-- appending region:466 with 189 probes\n",
      "-- appending region:467 with 200 probes\n",
      "-- appending region:468 with 222 probes\n",
      "-- appending region:469 with 260 probes\n",
      "-- appending region:470 with 131 probes\n",
      "-- appending region:471 with 199 probes\n",
      "-- appending region:472 with 203 probes\n",
      "-- appending region:473 with 203 probes\n",
      "-- appending region:474 with 235 probes\n",
      "-- appending region:475 with 263 probes\n",
      "-- appending region:476 with 236 probes\n",
      "-- appending region:477 with 215 probes\n",
      "-- appending region:478 with 189 probes\n",
      "-- appending region:479 with 287 probes\n",
      "-- appending region:480 with 257 probes\n",
      "-- appending region:481 with 269 probes\n",
      "-- appending region:482 with 230 probes\n",
      "-- appending region:483 with 220 probes\n",
      "-- appending region:484 with 265 probes\n",
      "-- appending region:485 with 255 probes\n",
      "-- appending region:486 with 294 probes\n",
      "-- appending region:487 with 342 probes\n",
      "-- appending region:488 with 3 probes\n",
      "-- appending region:489 with 216 probes\n",
      "-- appending region:490 with 384 probes\n",
      "-- appending region:491 with 276 probes\n",
      "-- appending region:492 with 229 probes\n",
      "-- appending region:493 with 239 probes\n",
      "-- appending region:494 with 293 probes\n",
      "-- appending region:495 with 259 probes\n",
      "-- appending region:496 with 167 probes\n",
      "-- appending region:497 with 228 probes\n",
      "-- appending region:498 with 262 probes\n",
      "-- appending region:499 with 162 probes\n",
      "-- appending region:500 with 142 probes\n",
      "-- appending region:501 with 304 probes\n",
      "-- appending region:502 with 269 probes\n",
      "-- appending region:503 with 318 probes\n",
      "-- appending region:504 with 220 probes\n",
      "-- appending region:505 with 308 probes\n",
      "-- appending region:506 with 306 probes\n",
      "-- appending region:507 with 202 probes\n",
      "-- appending region:508 with 221 probes\n",
      "-- appending region:509 with 272 probes\n",
      "-- appending region:510 with 336 probes\n",
      "-- appending region:511 with 184 probes\n",
      "-- appending region:512 with 197 probes\n",
      "-- appending region:514 with 2 probes\n",
      "-- appending region:515 with 20 probes\n",
      "-- appending region:516 with 227 probes\n",
      "-- appending region:517 with 262 probes\n",
      "-- appending region:518 with 264 probes\n",
      "-- appending region:519 with 188 probes\n",
      "-- appending region:520 with 212 probes\n",
      "-- appending region:521 with 265 probes\n",
      "-- appending region:522 with 187 probes\n",
      "-- appending region:523 with 152 probes\n",
      "-- appending region:524 with 196 probes\n",
      "-- appending region:525 with 216 probes\n",
      "-- appending region:526 with 223 probes\n",
      "-- appending region:527 with 255 probes\n",
      "-- appending region:528 with 249 probes\n",
      "-- appending region:529 with 269 probes\n",
      "-- appending region:530 with 223 probes\n",
      "-- appending region:531 with 201 probes\n",
      "-- appending region:532 with 232 probes\n",
      "-- appending region:533 with 201 probes\n",
      "-- appending region:534 with 181 probes\n",
      "-- appending region:535 with 116 probes\n",
      "-- appending region:536 with 259 probes\n",
      "-- appending region:537 with 231 probes\n",
      "-- appending region:538 with 214 probes\n",
      "-- appending region:539 with 242 probes\n",
      "-- appending region:540 with 179 probes\n",
      "-- appending region:541 with 248 probes\n",
      "-- appending region:542 with 227 probes\n",
      "-- appending region:543 with 302 probes\n",
      "-- appending region:544 with 276 probes\n",
      "-- appending region:545 with 267 probes\n",
      "-- appending region:546 with 305 probes\n",
      "-- appending region:547 with 44 probes\n",
      "-- appending region:548 with 191 probes\n",
      "-- appending region:549 with 206 probes\n",
      "-- appending region:550 with 378 probes\n",
      "-- appending region:551 with 90 probes\n",
      "-- appending region:552 with 264 probes\n",
      "-- appending region:553 with 49 probes\n",
      "-- appending region:554 with 306 probes\n",
      "-- appending region:555 with 241 probes\n",
      "-- appending region:556 with 234 probes\n",
      "-- appending region:557 with 273 probes\n",
      "-- appending region:558 with 251 probes\n",
      "-- appending region:559 with 221 probes\n",
      "-- appending region:560 with 261 probes\n",
      "-- appending region:561 with 240 probes\n",
      "-- appending region:562 with 249 probes\n",
      "-- appending region:563 with 193 probes\n",
      "-- appending region:564 with 270 probes\n",
      "-- appending region:565 with 218 probes\n",
      "-- appending region:566 with 169 probes\n",
      "-- appending region:567 with 229 probes\n",
      "-- appending region:568 with 222 probes\n",
      "-- appending region:569 with 292 probes\n",
      "-- appending region:570 with 104 probes\n",
      "-- appending region:571 with 356 probes\n",
      "-- appending region:572 with 280 probes\n",
      "-- appending region:573 with 206 probes\n",
      "-- appending region:574 with 258 probes\n",
      "-- appending region:575 with 204 probes\n",
      "-- appending region:576 with 291 probes\n",
      "-- appending region:577 with 207 probes\n",
      "-- appending region:578 with 246 probes\n",
      "-- appending region:579 with 151 probes\n",
      "-- appending region:580 with 202 probes\n",
      "-- appending region:581 with 250 probes\n",
      "-- appending region:582 with 199 probes\n",
      "-- appending region:583 with 181 probes\n",
      "-- appending region:584 with 139 probes\n",
      "-- appending region:585 with 241 probes\n",
      "-- appending region:586 with 218 probes\n",
      "-- appending region:587 with 156 probes\n",
      "-- appending region:588 with 177 probes\n",
      "-- appending region:589 with 275 probes\n",
      "-- appending region:590 with 225 probes\n",
      "-- appending region:591 with 283 probes\n",
      "-- appending region:592 with 144 probes\n",
      "-- appending region:593 with 306 probes\n",
      "-- appending region:594 with 155 probes\n",
      "-- appending region:595 with 180 probes\n",
      "-- appending region:596 with 254 probes\n",
      "-- appending region:597 with 277 probes\n",
      "-- appending region:598 with 226 probes\n",
      "-- appending region:599 with 304 probes\n",
      "-- appending region:600 with 294 probes\n",
      "-- appending region:601 with 299 probes\n",
      "-- appending region:602 with 261 probes\n",
      "-- appending region:603 with 251 probes\n",
      "-- appending region:604 with 6 probes\n",
      "-- appending region:605 with 187 probes\n",
      "-- appending region:606 with 293 probes\n",
      "-- appending region:607 with 281 probes\n",
      "-- appending region:608 with 355 probes\n",
      "-- appending region:609 with 323 probes\n",
      "-- appending region:611 with 302 probes\n",
      "-- appending region:612 with 311 probes\n",
      "-- appending region:613 with 200 probes\n",
      "-- appending region:614 with 277 probes\n",
      "-- appending region:615 with 258 probes\n",
      "-- appending region:616 with 173 probes\n",
      "-- appending region:617 with 254 probes\n",
      "-- appending region:618 with 192 probes\n",
      "-- appending region:619 with 282 probes\n",
      "-- appending region:620 with 196 probes\n",
      "-- appending region:621 with 248 probes\n",
      "-- appending region:622 with 247 probes\n",
      "-- appending region:623 with 185 probes\n",
      "-- appending region:624 with 132 probes\n",
      "-- appending region:625 with 168 probes\n",
      "-- appending region:626 with 118 probes\n",
      "-- appending region:627 with 253 probes\n",
      "-- appending region:628 with 273 probes\n",
      "-- appending region:629 with 360 probes\n",
      "-- appending region:630 with 37 probes\n",
      "-- appending region:631 with 21 probes\n",
      "-- appending region:632 with 9 probes\n",
      "-- appending region:641 with 5 probes\n",
      "-- appending region:643 with 1 probes\n",
      "-- appending region:645 with 257 probes\n",
      "-- appending region:646 with 241 probes\n",
      "-- appending region:647 with 219 probes\n",
      "-- appending region:648 with 215 probes\n",
      "-- appending region:649 with 262 probes\n",
      "-- appending region:650 with 241 probes\n",
      "-- appending region:651 with 146 probes\n",
      "-- appending region:652 with 306 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:653 with 373 probes\n",
      "-- appending region:654 with 271 probes\n",
      "-- appending region:655 with 401 probes\n",
      "-- appending region:656 with 292 probes\n",
      "-- appending region:657 with 186 probes\n",
      "-- appending region:658 with 311 probes\n",
      "-- appending region:659 with 54 probes\n",
      "-- appending region:660 with 278 probes\n",
      "-- appending region:662 with 184 probes\n",
      "-- appending region:663 with 205 probes\n",
      "-- appending region:664 with 255 probes\n",
      "-- appending region:665 with 266 probes\n",
      "-- appending region:666 with 273 probes\n",
      "-- appending region:667 with 172 probes\n",
      "-- appending region:668 with 328 probes\n",
      "-- appending region:669 with 312 probes\n",
      "-- appending region:670 with 311 probes\n",
      "-- appending region:671 with 199 probes\n",
      "-- appending region:672 with 171 probes\n",
      "-- appending region:673 with 120 probes\n",
      "-- appending region:674 with 258 probes\n",
      "-- appending region:675 with 327 probes\n",
      "-- appending region:676 with 330 probes\n",
      "-- appending region:677 with 308 probes\n",
      "-- appending region:678 with 347 probes\n",
      "-- appending region:679 with 320 probes\n",
      "-- appending region:680 with 294 probes\n",
      "-- appending region:681 with 308 probes\n",
      "-- appending region:682 with 147 probes\n",
      "-- appending region:683 with 285 probes\n",
      "-- appending region:684 with 253 probes\n",
      "-- appending region:685 with 328 probes\n",
      "-- appending region:686 with 266 probes\n",
      "-- appending region:687 with 78 probes\n",
      "-- appending region:688 with 278 probes\n",
      "-- appending region:689 with 160 probes\n",
      "-- appending region:690 with 289 probes\n",
      "-- appending region:691 with 270 probes\n",
      "-- appending region:692 with 189 probes\n",
      "-- appending region:693 with 246 probes\n",
      "-- appending region:694 with 236 probes\n",
      "-- appending region:695 with 246 probes\n",
      "-- appending region:696 with 170 probes\n",
      "-- appending region:697 with 220 probes\n",
      "-- appending region:698 with 227 probes\n",
      "-- appending region:699 with 66 probes\n",
      "-- appending region:701 with 289 probes\n",
      "-- appending region:702 with 291 probes\n",
      "-- appending region:703 with 181 probes\n",
      "-- appending region:704 with 284 probes\n",
      "-- appending region:705 with 292 probes\n",
      "-- appending region:706 with 246 probes\n",
      "-- appending region:707 with 182 probes\n",
      "-- appending region:708 with 213 probes\n",
      "-- appending region:709 with 289 probes\n",
      "-- appending region:710 with 234 probes\n",
      "-- appending region:711 with 290 probes\n",
      "-- appending region:712 with 201 probes\n",
      "-- appending region:713 with 236 probes\n",
      "-- appending region:714 with 196 probes\n",
      "-- appending region:715 with 353 probes\n",
      "-- appending region:716 with 136 probes\n",
      "-- appending region:717 with 282 probes\n",
      "-- appending region:718 with 391 probes\n",
      "-- appending region:719 with 376 probes\n",
      "-- appending region:720 with 218 probes\n",
      "-- appending region:721 with 270 probes\n",
      "-- appending region:722 with 309 probes\n",
      "-- appending region:723 with 73 probes\n",
      "-- appending region:724 with 288 probes\n",
      "-- appending region:725 with 271 probes\n",
      "-- appending region:726 with 209 probes\n",
      "-- appending region:727 with 138 probes\n",
      "-- appending region:728 with 365 probes\n",
      "-- appending region:729 with 200 probes\n",
      "-- appending region:730 with 379 probes\n",
      "-- appending region:731 with 174 probes\n",
      "-- appending region:732 with 296 probes\n",
      "-- appending region:733 with 274 probes\n",
      "-- appending region:734 with 225 probes\n",
      "-- appending region:735 with 206 probes\n",
      "-- appending region:736 with 362 probes\n",
      "-- appending region:737 with 252 probes\n",
      "-- appending region:738 with 222 probes\n",
      "-- appending region:739 with 229 probes\n",
      "-- appending region:740 with 249 probes\n",
      "-- appending region:741 with 266 probes\n",
      "-- appending region:742 with 353 probes\n",
      "-- appending region:743 with 309 probes\n",
      "-- appending region:744 with 308 probes\n",
      "-- appending region:745 with 367 probes\n",
      "-- appending region:747 with 23 probes\n",
      "-- appending region:748 with 254 probes\n",
      "-- appending region:749 with 199 probes\n",
      "-- appending region:750 with 287 probes\n",
      "-- appending region:751 with 293 probes\n",
      "-- appending region:752 with 307 probes\n",
      "-- appending region:753 with 145 probes\n",
      "-- appending region:754 with 344 probes\n",
      "-- appending region:755 with 59 probes\n",
      "-- appending region:756 with 319 probes\n",
      "-- appending region:757 with 192 probes\n",
      "-- appending region:758 with 141 probes\n",
      "-- appending region:759 with 306 probes\n",
      "-- appending region:760 with 220 probes\n",
      "-- appending region:761 with 222 probes\n",
      "-- appending region:762 with 141 probes\n",
      "-- appending region:763 with 270 probes\n",
      "-- appending region:764 with 261 probes\n",
      "-- appending region:765 with 191 probes\n",
      "-- appending region:766 with 254 probes\n",
      "-- appending region:767 with 199 probes\n",
      "-- appending region:768 with 278 probes\n",
      "-- appending region:769 with 160 probes\n",
      "-- appending region:770 with 81 probes\n",
      "-- appending region:773 with 46 probes\n",
      "-- appending region:774 with 202 probes\n",
      "-- appending region:775 with 202 probes\n",
      "-- appending region:776 with 317 probes\n",
      "-- appending region:777 with 373 probes\n",
      "-- appending region:778 with 252 probes\n",
      "-- appending region:779 with 208 probes\n",
      "-- appending region:780 with 349 probes\n",
      "-- appending region:782 with 206 probes\n",
      "-- appending region:783 with 286 probes\n",
      "-- appending region:784 with 196 probes\n",
      "-- appending region:785 with 295 probes\n",
      "-- appending region:786 with 234 probes\n",
      "-- appending region:787 with 293 probes\n",
      "-- appending region:788 with 182 probes\n",
      "-- appending region:789 with 209 probes\n",
      "-- appending region:790 with 91 probes\n",
      "-- appending region:791 with 164 probes\n",
      "-- appending region:792 with 168 probes\n",
      "-- appending region:793 with 231 probes\n",
      "-- appending region:794 with 298 probes\n",
      "-- appending region:795 with 126 probes\n",
      "-- appending region:796 with 289 probes\n",
      "-- appending region:797 with 249 probes\n",
      "-- appending region:798 with 244 probes\n",
      "-- appending region:799 with 289 probes\n",
      "-- appending region:800 with 196 probes\n",
      "-- appending region:801 with 320 probes\n",
      "-- appending region:802 with 257 probes\n",
      "-- appending region:803 with 289 probes\n",
      "-- appending region:804 with 220 probes\n",
      "-- appending region:805 with 168 probes\n",
      "-- appending region:806 with 291 probes\n",
      "-- appending region:807 with 233 probes\n",
      "-- appending region:808 with 197 probes\n",
      "-- appending region:809 with 405 probes\n",
      "-- appending region:810 with 160 probes\n",
      "-- appending region:811 with 234 probes\n",
      "-- appending region:812 with 358 probes\n",
      "-- appending region:813 with 187 probes\n",
      "-- appending region:814 with 275 probes\n",
      "-- appending region:815 with 332 probes\n",
      "-- appending region:816 with 209 probes\n",
      "-- appending region:817 with 204 probes\n",
      "-- appending region:819 with 268 probes\n",
      "-- appending region:820 with 255 probes\n",
      "-- appending region:821 with 142 probes\n",
      "-- appending region:822 with 257 probes\n",
      "-- appending region:823 with 129 probes\n",
      "-- appending region:824 with 288 probes\n",
      "-- appending region:825 with 225 probes\n",
      "-- appending region:826 with 263 probes\n",
      "-- appending region:827 with 283 probes\n",
      "-- appending region:828 with 377 probes\n",
      "-- appending region:829 with 237 probes\n",
      "-- appending region:832 with 168 probes\n",
      "-- appending region:833 with 204 probes\n",
      "-- appending region:834 with 310 probes\n",
      "-- appending region:835 with 236 probes\n",
      "-- appending region:836 with 164 probes\n",
      "-- appending region:837 with 302 probes\n",
      "-- appending region:838 with 226 probes\n",
      "-- appending region:839 with 194 probes\n",
      "-- appending region:840 with 264 probes\n",
      "-- appending region:841 with 373 probes\n",
      "-- appending region:843 with 148 probes\n",
      "-- appending region:844 with 256 probes\n",
      "-- appending region:845 with 300 probes\n",
      "-- appending region:846 with 261 probes\n",
      "-- appending region:847 with 174 probes\n",
      "-- appending region:848 with 294 probes\n",
      "-- appending region:849 with 123 probes\n",
      "-- appending region:850 with 181 probes\n",
      "-- appending region:851 with 241 probes\n",
      "-- appending region:852 with 279 probes\n",
      "-- appending region:853 with 194 probes\n",
      "-- appending region:854 with 215 probes\n",
      "-- appending region:855 with 199 probes\n",
      "-- appending region:856 with 163 probes\n",
      "-- appending region:857 with 300 probes\n",
      "-- appending region:858 with 252 probes\n",
      "-- appending region:859 with 240 probes\n",
      "-- appending region:860 with 144 probes\n",
      "-- appending region:861 with 333 probes\n",
      "-- appending region:862 with 233 probes\n",
      "-- appending region:863 with 282 probes\n",
      "-- appending region:864 with 250 probes\n",
      "-- appending region:865 with 223 probes\n",
      "-- appending region:866 with 216 probes\n",
      "-- appending region:867 with 164 probes\n",
      "-- appending region:868 with 289 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:869 with 301 probes\n",
      "-- appending region:870 with 324 probes\n",
      "-- appending region:871 with 329 probes\n",
      "-- appending region:872 with 360 probes\n",
      "-- appending region:873 with 265 probes\n",
      "-- appending region:874 with 370 probes\n",
      "-- appending region:875 with 212 probes\n",
      "-- appending region:876 with 262 probes\n",
      "-- appending region:877 with 268 probes\n",
      "-- appending region:878 with 347 probes\n",
      "-- appending region:888 with 215 probes\n",
      "-- appending region:889 with 279 probes\n",
      "-- appending region:890 with 205 probes\n",
      "-- appending region:891 with 169 probes\n",
      "-- appending region:892 with 314 probes\n",
      "-- appending region:893 with 252 probes\n",
      "-- appending region:894 with 308 probes\n",
      "-- appending region:895 with 294 probes\n",
      "-- appending region:896 with 158 probes\n",
      "-- appending region:897 with 286 probes\n",
      "-- appending region:898 with 182 probes\n",
      "-- appending region:899 with 211 probes\n",
      "-- appending region:900 with 251 probes\n",
      "-- appending region:901 with 230 probes\n",
      "-- appending region:902 with 271 probes\n",
      "-- appending region:903 with 144 probes\n",
      "-- appending region:904 with 320 probes\n",
      "-- appending region:905 with 311 probes\n",
      "-- appending region:906 with 194 probes\n",
      "-- appending region:907 with 143 probes\n",
      "-- appending region:908 with 216 probes\n",
      "-- appending region:909 with 181 probes\n",
      "-- appending region:910 with 155 probes\n",
      "-- appending region:911 with 187 probes\n",
      "-- appending region:912 with 230 probes\n",
      "-- appending region:913 with 169 probes\n",
      "-- appending region:914 with 176 probes\n",
      "-- appending region:915 with 166 probes\n",
      "-- appending region:916 with 250 probes\n",
      "-- appending region:917 with 250 probes\n",
      "-- appending region:918 with 259 probes\n",
      "-- appending region:919 with 157 probes\n",
      "-- appending region:920 with 187 probes\n",
      "-- appending region:921 with 173 probes\n",
      "-- appending region:922 with 174 probes\n",
      "-- appending region:923 with 141 probes\n",
      "-- appending region:924 with 236 probes\n",
      "-- appending region:925 with 214 probes\n",
      "-- appending region:926 with 293 probes\n",
      "-- appending region:927 with 314 probes\n",
      "-- appending region:928 with 334 probes\n",
      "-- appending region:929 with 261 probes\n",
      "-- appending region:930 with 266 probes\n",
      "-- appending region:931 with 249 probes\n",
      "-- appending region:932 with 297 probes\n",
      "-- appending region:933 with 224 probes\n",
      "-- appending region:934 with 209 probes\n",
      "-- appending region:935 with 426 probes\n",
      "-- appending region:949 with 160 probes\n",
      "-- appending region:950 with 178 probes\n",
      "-- appending region:951 with 188 probes\n",
      "-- appending region:952 with 190 probes\n",
      "-- appending region:953 with 245 probes\n",
      "-- appending region:954 with 90 probes\n",
      "-- appending region:955 with 174 probes\n",
      "-- appending region:956 with 265 probes\n",
      "-- appending region:957 with 258 probes\n",
      "-- appending region:958 with 328 probes\n",
      "-- appending region:959 with 213 probes\n",
      "-- appending region:960 with 279 probes\n",
      "-- appending region:961 with 237 probes\n",
      "-- appending region:962 with 226 probes\n",
      "-- appending region:963 with 195 probes\n",
      "-- appending region:964 with 149 probes\n",
      "-- appending region:965 with 191 probes\n",
      "-- appending region:966 with 161 probes\n",
      "-- appending region:967 with 229 probes\n",
      "-- appending region:968 with 228 probes\n",
      "-- appending region:969 with 156 probes\n",
      "-- appending region:970 with 110 probes\n",
      "-- appending region:971 with 173 probes\n",
      "-- appending region:972 with 392 probes\n",
      "-- appending region:973 with 270 probes\n",
      "-- appending region:974 with 209 probes\n",
      "-- appending region:975 with 315 probes\n",
      "-- appending region:976 with 194 probes\n",
      "-- appending region:977 with 297 probes\n",
      "-- appending region:978 with 130 probes\n",
      "-- appending region:979 with 255 probes\n",
      "-- appending region:980 with 173 probes\n",
      "-- appending region:981 with 302 probes\n",
      "-- appending region:982 with 171 probes\n",
      "-- appending region:983 with 236 probes\n",
      "-- appending region:984 with 269 probes\n",
      "-- appending region:985 with 342 probes\n",
      "-- appending region:986 with 233 probes\n",
      "-- appending region:987 with 374 probes\n",
      "-- appending region:988 with 348 probes\n",
      "-- appending region:989 with 154 probes\n",
      "-- appending region:990 with 281 probes\n",
      "-- appending region:991 with 247 probes\n",
      "-- appending region:992 with 237 probes\n",
      "-- appending region:993 with 153 probes\n",
      "-- appending region:994 with 286 probes\n",
      "-- appending region:995 with 337 probes\n",
      "-- appending region:996 with 255 probes\n",
      "-- appending region:997 with 304 probes\n",
      "-- appending region:999 with 341 probes\n",
      "-- appending region:1000 with 326 probes\n",
      "-- appending region:1001 with 341 probes\n",
      "-- appending region:1002 with 381 probes\n",
      "-- appending region:1003 with 356 probes\n",
      "-- appending region:1004 with 283 probes\n",
      "-- appending region:1005 with 321 probes\n",
      "-- appending region:1020 with 52 probes\n",
      "-- appending region:1021 with 11 probes\n",
      "-- appending region:1023 with 217 probes\n",
      "-- appending region:1024 with 242 probes\n",
      "-- appending region:1025 with 306 probes\n",
      "-- appending region:1026 with 17 probes\n",
      "-- appending region:1027 with 3 probes\n",
      "-- appending region:1028 with 1 probes\n",
      "-- appending region:1029 with 235 probes\n",
      "-- appending region:1030 with 209 probes\n",
      "-- appending region:1031 with 290 probes\n",
      "-- appending region:1032 with 275 probes\n",
      "-- appending region:1033 with 234 probes\n",
      "-- appending region:1034 with 183 probes\n",
      "-- appending region:1035 with 201 probes\n",
      "-- appending region:1036 with 130 probes\n",
      "-- appending region:1037 with 237 probes\n",
      "-- appending region:1038 with 227 probes\n",
      "-- appending region:1039 with 246 probes\n",
      "-- appending region:1040 with 350 probes\n",
      "-- appending region:1041 with 254 probes\n",
      "-- appending region:1042 with 325 probes\n",
      "-- appending region:1043 with 122 probes\n",
      "-- appending region:1044 with 253 probes\n",
      "-- appending region:1045 with 347 probes\n",
      "-- appending region:1046 with 263 probes\n",
      "-- appending region:1047 with 350 probes\n",
      "-- appending region:1048 with 296 probes\n",
      "-- appending region:1049 with 176 probes\n",
      "-- appending region:1050 with 267 probes\n",
      "-- appending region:1051 with 384 probes\n",
      "-- appending region:1052 with 135 probes\n",
      "-- appending region:1053 with 351 probes\n",
      "-- appending region:1054 with 320 probes\n",
      "-- appending region:1055 with 268 probes\n",
      "-- appending region:1056 with 361 probes\n",
      "-- appending region:1057 with 155 probes\n",
      "-- appending region:1058 with 131 probes\n",
      "-- appending region:1059 with 164 probes\n",
      "-- appending region:1060 with 315 probes\n",
      "-- appending region:1061 with 309 probes\n",
      "-- appending region:1062 with 240 probes\n",
      "-- appending region:1063 with 266 probes\n",
      "-- appending region:1064 with 327 probes\n",
      "-- appending region:1065 with 318 probes\n",
      "-- appending region:1066 with 207 probes\n",
      "-- appending region:1067 with 204 probes\n",
      "-- appending region:1068 with 147 probes\n",
      "-- appending region:1069 with 160 probes\n",
      "-- appending region:1070 with 250 probes\n",
      "-- appending region:1071 with 291 probes\n",
      "-- appending region:1072 with 259 probes\n",
      "-- appending region:1073 with 263 probes\n",
      "-- appending region:1074 with 273 probes\n",
      "-- appending region:1075 with 256 probes\n",
      "-- appending region:1076 with 293 probes\n",
      "-- appending region:1077 with 88 probes\n",
      "-- appending region:1078 with 241 probes\n",
      "-- appending region:1079 with 302 probes\n",
      "-- appending region:1080 with 362 probes\n",
      "-- appending region:1081 with 298 probes\n",
      "-- appending region:1082 with 263 probes\n",
      "-- appending region:1083 with 325 probes\n",
      "-- appending region:1085 with 341 probes\n",
      "-- appending region:1086 with 28 probes\n",
      "-- appending region:1087 with 336 probes\n",
      "-- appending region:1088 with 25 probes\n",
      "-- appending region:1089 with 207 probes\n",
      "-- appending region:1090 with 261 probes\n",
      "-- appending region:1091 with 226 probes\n",
      "-- appending region:1092 with 355 probes\n",
      "-- appending region:1093 with 203 probes\n",
      "-- appending region:1094 with 317 probes\n",
      "-- appending region:1095 with 2 probes\n",
      "-- appending region:1096 with 14 probes\n",
      "-- appending region:1097 with 230 probes\n",
      "-- appending region:1105 with 223 probes\n",
      "-- appending region:1106 with 189 probes\n",
      "-- appending region:1107 with 267 probes\n",
      "-- appending region:1108 with 295 probes\n",
      "-- appending region:1109 with 184 probes\n",
      "-- appending region:1110 with 336 probes\n",
      "-- appending region:1111 with 381 probes\n",
      "-- appending region:1112 with 398 probes\n",
      "-- appending region:1113 with 266 probes\n",
      "-- appending region:1114 with 217 probes\n",
      "-- appending region:1115 with 253 probes\n",
      "-- appending region:1116 with 184 probes\n",
      "-- appending region:1117 with 288 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:1118 with 231 probes\n",
      "-- appending region:1119 with 328 probes\n",
      "-- appending region:1120 with 228 probes\n",
      "-- appending region:1121 with 298 probes\n",
      "-- appending region:1122 with 227 probes\n",
      "-- appending region:1123 with 267 probes\n",
      "-- appending region:1124 with 201 probes\n",
      "-- appending region:1125 with 208 probes\n",
      "-- appending region:1126 with 320 probes\n",
      "-- appending region:1127 with 241 probes\n",
      "-- appending region:1128 with 235 probes\n",
      "-- appending region:1129 with 335 probes\n",
      "-- appending region:1130 with 313 probes\n",
      "-- appending region:1131 with 272 probes\n",
      "-- appending region:1132 with 290 probes\n",
      "-- appending region:1133 with 205 probes\n",
      "-- appending region:1134 with 331 probes\n",
      "-- appending region:1135 with 268 probes\n",
      "-- appending region:1136 with 294 probes\n",
      "-- appending region:1137 with 196 probes\n",
      "-- appending region:1138 with 222 probes\n",
      "-- appending region:1139 with 256 probes\n",
      "-- appending region:1140 with 242 probes\n",
      "-- appending region:1141 with 375 probes\n",
      "-- appending region:1142 with 263 probes\n",
      "-- appending region:1143 with 206 probes\n",
      "-- appending region:1144 with 122 probes\n",
      "-- appending region:1145 with 215 probes\n",
      "-- appending region:1146 with 94 probes\n",
      "-- appending region:1147 with 257 probes\n",
      "-- appending region:1148 with 101 probes\n",
      "-- appending region:1152 with 194 probes\n",
      "-- appending region:1153 with 269 probes\n",
      "-- appending region:1154 with 54 probes\n",
      "-- appending region:1155 with 254 probes\n",
      "-- appending region:1156 with 327 probes\n",
      "-- appending region:1157 with 276 probes\n",
      "-- appending region:1158 with 20 probes\n",
      "-- appending region:1160 with 317 probes\n",
      "-- appending region:1161 with 3 probes\n",
      "-- appending region:1162 with 222 probes\n",
      "-- appending region:1163 with 272 probes\n",
      "-- appending region:1164 with 3 probes\n",
      "-- appending region:1165 with 270 probes\n",
      "-- appending region:1166 with 342 probes\n",
      "-- appending region:1167 with 192 probes\n",
      "-- appending region:1168 with 346 probes\n",
      "-- appending region:1169 with 185 probes\n",
      "-- appending region:1170 with 232 probes\n",
      "-- appending region:1171 with 268 probes\n",
      "-- appending region:1172 with 329 probes\n",
      "-- appending region:1173 with 96 probes\n",
      "-- appending region:1174 with 128 probes\n",
      "-- appending region:1175 with 44 probes\n",
      "-- appending region:1176 with 212 probes\n",
      "-- appending region:1177 with 241 probes\n",
      "-- appending region:1178 with 321 probes\n",
      "-- appending region:1179 with 197 probes\n",
      "-- appending region:1180 with 213 probes\n",
      "-- appending region:1181 with 271 probes\n",
      "-- appending region:1182 with 291 probes\n",
      "-- appending region:1183 with 337 probes\n",
      "-- appending region:1184 with 310 probes\n",
      "-- appending region:1185 with 4 probes\n",
      "-- appending region:1186 with 265 probes\n",
      "-- appending region:1187 with 285 probes\n",
      "-- appending region:1188 with 77 probes\n",
      "-- appending region:1189 with 171 probes\n",
      "-- appending region:1190 with 187 probes\n",
      "-- appending region:1191 with 194 probes\n",
      "-- appending region:1192 with 242 probes\n",
      "-- appending region:1193 with 279 probes\n",
      "-- appending region:1194 with 291 probes\n",
      "-- appending region:1195 with 288 probes\n",
      "-- appending region:1196 with 277 probes\n",
      "-- appending region:1197 with 184 probes\n",
      "-- appending region:1198 with 30 probes\n",
      "-- appending region:1199 with 2 probes\n",
      "-- appending region:1203 with 191 probes\n",
      "-- appending region:1204 with 346 probes\n",
      "-- appending region:1205 with 321 probes\n",
      "-- appending region:1206 with 108 probes\n",
      "-- appending region:1207 with 213 probes\n",
      "-- appending region:1208 with 208 probes\n",
      "-- appending region:1209 with 169 probes\n",
      "-- appending region:1210 with 270 probes\n",
      "-- appending region:1211 with 302 probes\n",
      "-- appending region:1212 with 278 probes\n",
      "-- appending region:1213 with 293 probes\n",
      "-- appending region:1214 with 317 probes\n",
      "-- appending region:1215 with 221 probes\n",
      "-- appending region:1216 with 216 probes\n",
      "-- appending region:1217 with 190 probes\n",
      "-- appending region:1218 with 235 probes\n",
      "-- appending region:1219 with 315 probes\n",
      "-- appending region:1220 with 358 probes\n",
      "-- appending region:1221 with 411 probes\n",
      "-- appending region:1222 with 179 probes\n",
      "-- appending region:1224 with 231 probes\n",
      "-- appending region:1225 with 282 probes\n",
      "-- appending region:1226 with 238 probes\n",
      "-- appending region:1227 with 215 probes\n",
      "-- appending region:1228 with 167 probes\n",
      "-- appending region:1229 with 235 probes\n",
      "-- appending region:1230 with 223 probes\n",
      "-- appending region:1231 with 241 probes\n",
      "-- appending region:1232 with 237 probes\n",
      "-- appending region:1233 with 223 probes\n",
      "-- appending region:1234 with 169 probes\n",
      "-- appending region:1235 with 241 probes\n",
      "-- appending region:1236 with 271 probes\n",
      "-- appending region:1237 with 228 probes\n",
      "-- appending region:1238 with 287 probes\n",
      "-- appending region:1239 with 249 probes\n",
      "-- appending region:1240 with 297 probes\n",
      "-- appending region:1241 with 344 probes\n",
      "-- appending region:1242 with 279 probes\n",
      "-- appending region:1243 with 297 probes\n",
      "-- appending region:1244 with 260 probes\n",
      "-- appending region:1245 with 283 probes\n",
      "-- appending region:1246 with 305 probes\n",
      "-- appending region:1247 with 234 probes\n",
      "-- appending region:1248 with 160 probes\n",
      "-- appending region:1249 with 237 probes\n",
      "-- appending region:1250 with 310 probes\n",
      "-- appending region:1251 with 181 probes\n",
      "-- appending region:1252 with 181 probes\n",
      "-- appending region:1253 with 246 probes\n",
      "-- appending region:1254 with 178 probes\n",
      "-- appending region:1255 with 303 probes\n",
      "-- appending region:1256 with 186 probes\n",
      "-- appending region:1257 with 173 probes\n",
      "-- appending region:1258 with 168 probes\n",
      "-- appending region:1259 with 216 probes\n",
      "-- appending region:1260 with 364 probes\n",
      "-- appending region:1261 with 377 probes\n",
      "-- appending region:1262 with 85 probes\n",
      "-- appending region:1263 with 189 probes\n",
      "-- appending region:1264 with 279 probes\n",
      "-- appending region:1265 with 99 probes\n",
      "-- appending region:1269 with 51 probes\n",
      "-- appending region:1270 with 305 probes\n",
      "-- appending region:1271 with 312 probes\n",
      "-- appending region:1272 with 317 probes\n",
      "-- appending region:1273 with 297 probes\n",
      "-- appending region:1274 with 192 probes\n",
      "-- appending region:1275 with 356 probes\n",
      "-- appending region:1276 with 244 probes\n",
      "-- appending region:1277 with 362 probes\n",
      "-- appending region:1278 with 150 probes\n",
      "-- appending region:1279 with 193 probes\n",
      "-- appending region:1280 with 296 probes\n",
      "-- appending region:1281 with 275 probes\n",
      "-- appending region:1282 with 234 probes\n",
      "-- appending region:1284 with 188 probes\n",
      "-- appending region:1285 with 206 probes\n",
      "-- appending region:1286 with 263 probes\n",
      "-- appending region:1287 with 287 probes\n",
      "-- appending region:1288 with 239 probes\n",
      "-- appending region:1289 with 368 probes\n",
      "-- appending region:1290 with 315 probes\n",
      "-- appending region:1291 with 273 probes\n",
      "-- appending region:1292 with 248 probes\n",
      "-- appending region:1293 with 312 probes\n",
      "-- appending region:1294 with 90 probes\n",
      "-- appending region:1295 with 167 probes\n",
      "-- appending region:1296 with 1 probes\n",
      "-- appending region:1297 with 312 probes\n",
      "-- appending region:1298 with 282 probes\n",
      "-- appending region:1299 with 267 probes\n",
      "-- appending region:1300 with 266 probes\n",
      "-- appending region:1301 with 252 probes\n",
      "-- appending region:1302 with 220 probes\n",
      "-- appending region:1303 with 174 probes\n",
      "-- appending region:1304 with 268 probes\n",
      "-- appending region:1305 with 297 probes\n",
      "-- appending region:1306 with 285 probes\n",
      "-- appending region:1307 with 271 probes\n",
      "-- appending region:1308 with 243 probes\n",
      "-- appending region:1309 with 325 probes\n",
      "-- appending region:1310 with 285 probes\n",
      "-- appending region:1311 with 211 probes\n",
      "-- appending region:1312 with 257 probes\n",
      "-- appending region:1313 with 201 probes\n",
      "-- appending region:1314 with 272 probes\n",
      "-- appending region:1315 with 347 probes\n",
      "-- appending region:1316 with 301 probes\n",
      "-- appending region:1317 with 344 probes\n",
      "-- appending region:1318 with 253 probes\n",
      "-- appending region:1319 with 385 probes\n",
      "-- appending region:1320 with 286 probes\n",
      "-- appending region:1321 with 321 probes\n",
      "-- appending region:1322 with 310 probes\n",
      "-- appending region:1323 with 306 probes\n",
      "-- appending region:1324 with 321 probes\n",
      "-- appending region:1325 with 367 probes\n",
      "-- appending region:1329 with 35 probes\n",
      "-- appending region:1330 with 129 probes\n",
      "-- appending region:1331 with 242 probes\n",
      "-- appending region:1332 with 278 probes\n",
      "-- appending region:1333 with 136 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:1334 with 161 probes\n",
      "-- appending region:1335 with 326 probes\n",
      "-- appending region:1336 with 266 probes\n",
      "-- appending region:1337 with 339 probes\n",
      "-- appending region:1338 with 188 probes\n",
      "-- appending region:1339 with 238 probes\n",
      "-- appending region:1340 with 344 probes\n",
      "-- appending region:1341 with 308 probes\n",
      "-- appending region:1342 with 261 probes\n",
      "-- appending region:1343 with 201 probes\n",
      "-- appending region:1344 with 274 probes\n",
      "-- appending region:1345 with 312 probes\n",
      "-- appending region:1346 with 298 probes\n",
      "-- appending region:1347 with 179 probes\n",
      "-- appending region:1348 with 130 probes\n",
      "-- appending region:1349 with 302 probes\n",
      "-- appending region:1350 with 265 probes\n",
      "-- appending region:1351 with 353 probes\n",
      "-- appending region:1352 with 243 probes\n",
      "-- appending region:1353 with 192 probes\n",
      "-- appending region:1354 with 278 probes\n",
      "-- appending region:1355 with 187 probes\n",
      "-- appending region:1356 with 291 probes\n",
      "-- appending region:1357 with 248 probes\n",
      "-- appending region:1358 with 327 probes\n",
      "-- appending region:1359 with 267 probes\n",
      "-- appending region:1360 with 324 probes\n",
      "-- appending region:1361 with 397 probes\n",
      "-- appending region:1362 with 385 probes\n",
      "-- appending region:1363 with 314 probes\n",
      "-- appending region:1370 with 10 probes\n",
      "-- appending region:1374 with 20 probes\n",
      "-- appending region:1377 with 28 probes\n",
      "-- appending region:1378 with 217 probes\n",
      "-- appending region:1379 with 292 probes\n",
      "-- appending region:1380 with 192 probes\n",
      "-- appending region:1381 with 214 probes\n",
      "-- appending region:1382 with 210 probes\n",
      "-- appending region:1383 with 230 probes\n",
      "-- appending region:1384 with 148 probes\n",
      "-- appending region:1385 with 163 probes\n",
      "-- appending region:1386 with 128 probes\n",
      "-- appending region:1387 with 190 probes\n",
      "-- appending region:1388 with 176 probes\n",
      "-- appending region:1389 with 251 probes\n",
      "-- appending region:1390 with 288 probes\n",
      "-- appending region:1391 with 233 probes\n",
      "-- appending region:1392 with 244 probes\n",
      "-- appending region:1393 with 323 probes\n",
      "-- appending region:1394 with 174 probes\n",
      "-- appending region:1395 with 282 probes\n",
      "-- appending region:1396 with 192 probes\n",
      "-- appending region:1397 with 12 probes\n",
      "-- appending region:1398 with 184 probes\n",
      "-- appending region:1399 with 268 probes\n",
      "-- appending region:1400 with 308 probes\n",
      "-- appending region:1401 with 268 probes\n",
      "-- appending region:1402 with 308 probes\n",
      "-- appending region:1403 with 223 probes\n",
      "-- appending region:1404 with 235 probes\n",
      "-- appending region:1405 with 336 probes\n",
      "-- appending region:1406 with 328 probes\n",
      "-- appending region:1407 with 16 probes\n",
      "-- appending region:1408 with 354 probes\n",
      "-- appending region:1409 with 374 probes\n",
      "-- appending region:1421 with 2 probes\n",
      "-- appending region:1422 with 2 probes\n",
      "-- appending region:1425 with 4 probes\n",
      "-- appending region:1427 with 253 probes\n",
      "-- appending region:1428 with 1 probes\n",
      "-- appending region:1429 with 266 probes\n",
      "-- appending region:1430 with 240 probes\n",
      "-- appending region:1431 with 10 probes\n",
      "-- appending region:1433 with 28 probes\n",
      "-- appending region:1434 with 295 probes\n",
      "-- appending region:1435 with 255 probes\n",
      "-- appending region:1436 with 298 probes\n",
      "-- appending region:1437 with 360 probes\n",
      "-- appending region:1438 with 174 probes\n",
      "-- appending region:1439 with 198 probes\n",
      "-- appending region:1440 with 236 probes\n",
      "-- appending region:1441 with 184 probes\n",
      "-- appending region:1442 with 248 probes\n",
      "-- appending region:1443 with 316 probes\n",
      "-- appending region:1444 with 314 probes\n",
      "-- appending region:1445 with 316 probes\n",
      "-- appending region:1446 with 274 probes\n",
      "-- appending region:1447 with 380 probes\n",
      "-- appending region:1448 with 267 probes\n",
      "-- appending region:1449 with 343 probes\n",
      "-- appending region:1450 with 211 probes\n",
      "-- appending region:1451 with 255 probes\n",
      "-- appending region:1452 with 113 probes\n",
      "-- appending region:1453 with 304 probes\n",
      "-- appending region:1454 with 277 probes\n",
      "-- appending region:1455 with 290 probes\n",
      "-- appending region:1456 with 270 probes\n",
      "-- appending region:1457 with 281 probes\n",
      "-- appending region:1458 with 396 probes\n",
      "-- appending region:1459 with 326 probes\n",
      "-- appending region:1461 with 143 probes\n",
      "-- appending region:1462 with 238 probes\n",
      "-- appending region:1463 with 268 probes\n",
      "-- appending region:1464 with 281 probes\n",
      "-- appending region:1465 with 231 probes\n",
      "-- appending region:1466 with 286 probes\n",
      "-- appending region:1467 with 193 probes\n",
      "-- appending region:1468 with 180 probes\n",
      "-- appending region:1469 with 202 probes\n",
      "-- appending region:1470 with 194 probes\n",
      "-- appending region:1471 with 221 probes\n",
      "-- appending region:1472 with 278 probes\n",
      "-- appending region:1473 with 117 probes\n",
      "-- appending region:1474 with 173 probes\n",
      "-- appending region:1475 with 234 probes\n",
      "-- appending region:1476 with 197 probes\n",
      "-- appending region:1477 with 341 probes\n",
      "-- appending region:1478 with 230 probes\n",
      "-- appending region:1479 with 216 probes\n",
      "-- appending region:1480 with 117 probes\n",
      "-- appending region:1481 with 198 probes\n",
      "-- appending region:1482 with 212 probes\n",
      "-- appending region:1485 with 351 probes\n",
      "-- appending region:1486 with 227 probes\n",
      "-- appending region:1487 with 250 probes\n",
      "-- appending region:1488 with 311 probes\n",
      "-- appending region:1489 with 94 probes\n",
      "-- appending region:1490 with 128 probes\n",
      "-- appending region:1491 with 239 probes\n",
      "-- appending region:1492 with 238 probes\n",
      "-- appending region:1493 with 129 probes\n",
      "-- appending region:1494 with 189 probes\n",
      "-- appending region:1495 with 199 probes\n",
      "-- appending region:1496 with 72 probes\n",
      "-- appending region:1497 with 230 probes\n",
      "-- appending region:1498 with 146 probes\n",
      "-- appending region:1499 with 160 probes\n",
      "-- appending region:1500 with 369 probes\n",
      "-- appending region:1501 with 253 probes\n",
      "-- appending region:1502 with 139 probes\n",
      "-- appending region:1503 with 186 probes\n",
      "-- appending region:1504 with 285 probes\n",
      "-- appending region:1505 with 239 probes\n",
      "-- appending region:1506 with 163 probes\n",
      "-- appending region:1507 with 242 probes\n",
      "-- appending region:1508 with 189 probes\n",
      "-- appending region:1509 with 188 probes\n",
      "-- appending region:1510 with 176 probes\n",
      "-- appending region:1511 with 218 probes\n",
      "-- appending region:1512 with 248 probes\n",
      "-- appending region:1513 with 245 probes\n",
      "-- appending region:1514 with 262 probes\n",
      "-- appending region:1515 with 357 probes\n",
      "-- appending region:1516 with 103 probes\n",
      "-- appending region:1517 with 232 probes\n",
      "-- appending region:1518 with 217 probes\n",
      "-- appending region:1519 with 239 probes\n",
      "-- appending region:1520 with 221 probes\n",
      "-- appending region:1521 with 176 probes\n",
      "-- appending region:1525 with 51 probes\n",
      "-- appending region:1526 with 60 probes\n",
      "-- appending region:1527 with 72 probes\n",
      "-- appending region:1528 with 44 probes\n",
      "-- appending region:1529 with 166 probes\n",
      "-- appending region:1530 with 218 probes\n",
      "-- appending region:1533 with 69 probes\n",
      "-- appending region:1534 with 149 probes\n",
      "-- appending region:1535 with 187 probes\n",
      "-- appending region:1536 with 187 probes\n",
      "-- appending region:1537 with 170 probes\n",
      "-- appending region:1538 with 164 probes\n",
      "-- appending region:1540 with 1 probes\n",
      "-- appending region:1541 with 251 probes\n",
      "-- appending region:1542 with 204 probes\n",
      "-- appending region:1543 with 30 probes\n",
      "-- appending region:1547 with 2 probes\n",
      "-- appending region:1548 with 106 probes\n"
     ]
    }
   ],
   "source": [
    "cand_prob_nums = []\n",
    "pb_dict = {}\n",
    "gene_names = np.sort(np.unique([int(_v['reg_name'].split('gene_')[1]) for _s, _v in _pbde.kept_probes.items()]))\n",
    "for _gene_name in gene_names:\n",
    "    _sel_pbs = {_s:_v for _s, _v in _pbde.kept_probes.items() if int(_v['reg_name'].split('gene_')[1])==_gene_name}\n",
    "    \n",
    "    if len(_sel_pbs) > 0:\n",
    "        cand_prob_nums.append(len(_sel_pbs))\n",
    "        if verbose:\n",
    "            print(f\"-- appending region:{_gene_name} with {len(_sel_pbs)} probes\")\n",
    "        _splitted_pbde = ld.pb_reports_class()\n",
    "        _splitted_pbde.kept_probes = _sel_pbs\n",
    "        pb_dict[_gene_name] = _splitted_pbde"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "id": "a78ce7a4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox  6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7snQXYFcX3xw/d3Snd3Y1IYwuCgqiYqJgoSgooIpg/RUJQ/BvYYNIl3d0t3Sld/+fsvbPvuXNn8+7ee9/3PfM8Po+8d3d25juzu589c+acFDdv3rwJXFgBVoAVYAVYAVaAFWAFWIFkokAKBuBkMtLcTVaAFWAFWAFWgBVgBVgBTQEGYJ4IrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYJ4DrAArwAqwAqwAK8AKsALJSgEG4GQ13NxZVoAVYAVYAVaAFWAFWAEGYB/nwKVLl2D9+vWQJ08eSJ06tY9X4qpZAVaAFWAFWAFWgBUIVeDatWtw7NgxqFy5MqRPn57lIQowAPs4HZYvXw516tTx8QpcNSvACrACrAArwAqwAuYKLFu2DGrXrs0yMQBHZw7s2bMHihcvDjjxChQoEJ2L8lVYAVaAFWAFWAFWgBUAgEOHDmmGuN27d0OxYsVYEwbg6MyB/fv3Q5EiRWDfvn1QuHDh6FyUr8IKsAKsACvACrACrAAAMIcYTwN2gfDxFuGJ56O4XDUrwAqwAqwAK8AKmCrAHMIAHJNbhCdeTGTni7ICrAArwAqwAqwAW4BN5wBbgH28RRiAfRSXq2YFWAFWgBVgBVgBtgC7nAMMwC6Fs3MaA7AdlfgYVoAVYAVYAVaAFfBDAeYQY1UZgP2YccE6eeL5KC5XzQqwAqwAK8AKsAJsAXY5BxiAXQpn5zQGYDsq8TGsACvACrACrAAr4IcCzCFsAfZjXlnWyRPPUiI+gBVgBVgBVoAVYAV8UoA5hAHYp6llXi1PvJjIzhdlBVgBVoAVYAVYAY4CYToH2AXCx1uEAdhHcblqVoAVYAVYAVaAFTBVgDmELcAxuUV44sVEdr4oK8AKsAKsACvACrAFmC3AsboLGIBjpTxflxVgBVgBVoAVYAWYQ9gCHJO7gCdeTGTni7ICrAArwAqwAqwAW4DZAhyru4ABOFbK83VZAVaAFWAFWAFWgDmELcAxuQt44sVEdr4oK8AKsAKsACvACrAFmC3AsboLGIBjpTxflxVgBRK7AqcvXIHj/12GUnmzJPaucPtZgZgpwBzCFuCYTD6eeDGRnS/KCrACSUCBEr3/hhs3ASY+2wBqFM2RBHrEXWAFoq8AcwgDcPRnHS89xERzvigrwAokDQWKvfG31pEONQvD+/dXTRqd4l6wAlFWgAGYATjKUy5wOZ54MZGdL8oKsAJJQAEG4CQwiNyFmCvAHMIAHJNJyBMvJrLzRVkBViAJKMAAnAQGkbsQcwWYQxiAYzIJeeLFRHa+KCvACiQBBRiAk8AgchdirgBzCANwTCYhT7yYyM4XZQVYgSSgAANwEhhE7kLMFWAOYQCOySTkiRcT2fmirAArkAQUYABOAoPIXYi5AswhDMAxmYQ88WIiO1+UFWAFkoACDMBJYBC5CzFXgDmEATgmk5AnXkxk54uyAqxAElCAATgJDCJ3IeYKMIcwAMdkEvLEi4nsif6iH8/cBjdvArzcskyi7wt3gBVwqwADsFvl+DxWIEEB5hAG4JjcDzzxYiJ7or7ohgNn4I5PF2h9+KNHQ6hSOHui7g83nhVwq4AA4PtrFob3OBGGWxn5vGSuAHMIA3BMbgGeeDGRPVFfdM7Wo9Bt/HKtD188Ugual8+XqPvDjWcF3CrAAOxWOT6PFWALsJ05kOLmTVxs5eKHAgzAfqiatOucu/UoPMoAnLQHmXtnSwEGYFsyOToIX/c7j52H4rkzQaqUKRydywcnTgWYQ9gCHJOZyxMvJrIn6otSAB73cC1oUYEtwIl6QLnxrhRAUCvee7J2LrtAuJJQedK4+bvg7b83w+1VCsBnnWt4VzHXFLcKMIcwAMdkcvLEi4nsifqi/2w7Bo98uUzrw9iHa0FLBuBEPZ7ceHcKMAC7083qLGFVx+P2vHu71eH8exJQgDmEATgm05gnXkxk1y668eAZ+G31AXiicQnIlzV97Bri8Mrzth2Dh4MA/HnXmtCqYn6HNfDhrEDiV+DGjZtQog9bgL0eSQZgrxWN//qYQxiAYzJLeeLFRHbtouJBX71odpj0bMPYNcThledvPwZdvwhYgMd0rQmtGYAdKsiHJwUFGID9GUUGYH90jedamUMYgGMyP3nixUT2EADGfySmpb4F24/DQ18sZQCO3dRJVldG0EwZh5uhrt+4CSXZAuz5XGQA9lzSuK+QOSQZA/C2bdugRo0aMGLECHj00Uc1JdasWQMvvvgirFixAnLlygUvvPACvPrqq7pKN27cgEGDBsG4cePg1KlT0KhRIxg5ciSUKlXK0WTniedILk8PTqwPegrAox+qCW0qsQuEpxODK9MVeGfyZvhl5X744al6UCZflrhShgHYn+FIrM9Ff9RIHrUyhyRTAL569So0aNBAA93x48drAHzixAkoV64c3HPPPfDKK6/AkiVL4Nlnn9UAt1u3bppSCL/4bzynUKFC0KtXL9i1axds3LgR0qZNa/uu4YlnWyrPD0ysD/qFO45Dl3EBC/Doh2pAm0oFPNeGK2QFUAFxj1QulA3+fL5RXIly7foNKNV3itYmjgLh3dAk1ueidwokv5qYQ5IpAPfp00cD3Dlz5ugAPHToUPjss89gz549kDp1ak0ZPG7ixImwZcsWuHLlCuTOnRuGDx8O3bt3134/ffo0FCxYEL788kt44IEHbN9BPPFsS+X5gYn1Qb9ox3HozADs+XzgCsMVEPdIyTyZYFbPW+NKIgrAHWsVhuEdqsZV+xJrYxLrczGx6h0P7WYOSYYAPG/ePLj33ns1d4eiRYvqANyuXTvInj07TJgwQVdl5syZ0LJlSzhy5IgGxnXr1oWtW7dCmTJl9GPQDaJKlSqaZdhu4YlnVynvj0usD3oKwKO61IC2ldkC7P3scFfj1A2H4PK1G3B3tULuKoizs+IZgK9evwGlgxZgBmDvJk5ifS56p0Dyq4k5JJkBMFpsq1atCmjt7dy5M6RIkUIHYITYtm3bwrBhw3RVNm3aBBUrVoTly5fD3r17oX379nDhwgXIkCGDfkzHjh21v/3111+Gap49exbwP1EOHToEderUgX379kHhwoWT350Xwx4n1gf9op3HofPYgAsEA3AMJ5B06QOnL0LDd2drf538QmOoUDBr/DTOZUsYgF0Kl4hPS6zPxUQsecybzgCczAD4wQcfBAyk/sMPP2g9pwCMG9kQigcPHqyrgv69JUuWhPnz52sW4K5du8L169chZcqU+jEPP/wwHDx4ENBabFQGDhyo+Q/LhQE4+s+AxPqgpwA8sksNaMcW4OhPHsUVl+0+CR3HLNZ++d8D1ZKEFTieAfjKtRtQpl/AB5gtwN7dAon1ueidAsmvJgbgZATA33zzDfTu3RvWr18POXLkCAPgypUrA7pBqCzAq1at0ja7dejQQWkBvnz5Mvz++++GarIFOH4eLon1Qb945wl4cOwSTUhMVYopS7nEXgEG4OiOQVIB4JX/noRXf14HPZqVgvY1Y78KmFifizj70Kh17cZNSJMqwTAV3VmZOK/GAJyMALhZs2awYMECSJcund7r8+fPa/9GK+8tt9yihT5DUBZF5QO8Y8cO7XhR0AcY3SpwA53dwhPPrlLeH5dYH/QUgEd0rg53VCnovThco2MFlu85CfePZguwY+FcnnD52nUo22+qdnZitgDH23Mo3trjZHr0mLAK5m8/Dn893wiK5Mzo5FTbx+LmyxPnrySq7KFWnWMOSUYAfODAAbh48WJIj0uXLq1ZfDt16gTff/89jB49Gnbu3AmpUqXSjkOL8aRJk7QoEGjlzZs3L3z44Yfw+OOPa79zFAirWyz+fk+sD/olu07AA58HLMCfPlgd7qzKABwPs4sBOLqjwADsj96J9bmIaoi2t6mYH0Z3remLQJ3GLIalu0/ChCfrQoOSuX25RrQrZQBORgCs6ir1AT569KgWB/iuu+7S4vsuW7YMnnnmGQ2KH3nkEe30vn37wpgxY7SwZ8WKFYPXXntN8w1GtwqOAxzt29fd9RLrg37prhPQiQHY3aD7eBYDsI/iKqq+dPU6lOvPFmCvVU+sz0UKwI1L54ZvHq/rtTRafUKfcvmzwNSXmvhyjWhXygDMAKxHgUApMNoDZn9bvXo1FChQAHr27Ak9evTQVcINcBgbGBNhoDW5SZMmmusDwrCTwhPPiVreHptYH/TU1/STB6vDXWwB9nZiuKyNAdilcC5PYwB2KZzFaYn1uUjhtE7xnPDT0/V9EUjoUzZfFpj2MgOwLyLHUaUpbqJnORdfFGAA9kVWW5Um1gd9UgQtWwMW5wfRcfm4UzW4p3rijwUcz1Eg7ADwoTMX4b1pW6FDjcLQoFR8LlfH23Mo3trj5LYXba9WJDv89lxDJ6faPpYB2LZUSeJABmAfhzGWAIxL6fhy6NmqLNQvmcvHXsZn1Yn1Qc8AHJ/zacWek9AhuAnOSwBeu+80vPXXJniuWSloVi5vVDuf2AG4/ahFsPLfU5pme969Para2b1YvD2H4q09dnWkFuAKBbLC5BcbOznV9rFCnzL5MsP0l5vaPi+eD4wlh8SzLtg2BmAfRyiWEy8xP+i8GBKv+3/u0lW4b+QiKF8gK6Brgl+FglZSiTfrl1bRrNcvAMZYtxjyKxYQF88AfPHKdSg/wNwH2Ot73I/5FG9tjLf2ONE8GvOVAdjJiCT+YxmAfRxDBmAfxbWo2usH/SeztsOHM7ZpV904qDVkSpfal85h3ND2o5JWuC1fhIpypX4BsNfz1Iks4tql8maGma/El7WLAdjJSNo/NpbzzX4r1UeKthfNmRHm9WoWaXXK8xmAfZE1bitlAPZxaBiAfRQ3ygA8dPJmGDNvl3bVDYNaQ+YoALCXS+2xG4mkcWUG4OiOIwOwP3onBQDOnzU9LOnT3BeBhD6l82aGGXH2Uei2w7HkELdtjtZ5DMA+Kh3LiZeYH3ReDInX/Y8eAJ8C9G3EwgDsxUzwpg4GYG90tFvLhSvXoMKAadrhRokwvL7H7bbNyXHx1sZ4a49dLXGvfvHek7XDc2RMA6sHtLJ7qqPjGIAdyZXoD2YA9nEIGYB9FDfaFuApm2HMP9GwACcA8EedqsK91WOfPjV2oxg/V2YAju5YUADuVKsIDOtQJawBiQHm4q2N8dCerYfPwS8r98FjjYpDgWwZbE2sGzduQok+AQDOlDYVbBzcxtZ5Tg+KZ7cgp30Rx8eSQ9y2OVrnMQD7qHQsJ148POh8lNayaq/7Ty3A6we2gizp01i2wc0Bq/ae0jbbYfmwY1W4r0biA+BhU7fA1Ws3oO/t5QGT0CSFwgAc3VE8f/kaVHwzYAGONgBfvX4D+k5aD6XzZoEnm5SIqONeP4ciagxJ9ID1xCpyhtDESbKJ6zduQskgAKdNlRK2DWkbqRTK8xmAfZE1bitlAPZxaBiAfRTXomqvXzwUgNcNbAVZfQLg1XtPwb1BAP7g/qrQvmbiAuCNB8/A7Z8s0Ebn5+71oXaxnLGbBB5emQHYQzFtVBVLAP5myb/Q/7cNWiu3vNUG0qdJZaPF6kO8fg65bkjwxHhoj5s24EdJ6b5TtF5EA4BL5skEs3reGqnccXF+LDkkLgQwaQQDsI8jFMuJ5+Yh46MUUa/a6/5TAF47oBVky+iPBZgC8Pv3V4UOiQyAF+44Dl3GLdXGe/RDNaBNpQJRH3s/LsgA7IeqxnX+d/kaVPLQAoxhDBFk06RKadkRL/39vX4OWTY+yoYBN+1xo8nla9ehbL9AWLxoAHCJPJlgNgOwm+FNVOcwAPs4XAzAPoob5Qc9fSmuGdASsmdM60vn1uw7Dfd8tlCrOzEC8KIdx6EzA7DtuVGm7xS4cj22cYDjMQyalwB89OwlaDR8DpTInQmmvNjY0i2H3uuRhjx0A3u2J4+LA+OhPW7aQDMDegnACNbbj/wHFQtm1eaFaBvOldmvsgXYxRRLVKcwAPs4XAzAPoobrHr9/jNw8ep1wPzwtLh5yJq1lr4UV/VvCTkz+Q/A73WoAvfXKuK/iB5egQLwqC41oG3lpGEBpvGZvdycWHHAVDh/5bo2AuiTibvdZ285CiXzZIZiuTN5ODLhVcWzvyNabCsPnK41OlIf4HenbIHR/+zU6rIDtO9M3gyfB0Me2jnebJC8fg5FOiHioT1u2kDD4nkJwJ3HLoFFO0/AG23LQfemJRmAI51giex8BmAfB4wB2EdxAeDMhatQdXDgJTnzlSZQKm8W/YJuHrJmraUvxRX9WkDuzOl86Rymxr07aAEe3qEKdExsALzzOHQeG3CB8AqAcQf4d8v2Qqk8mWOW1tsvAK42eDqcvnBVB+A/1h6EF75frf/bl0kWrDS5ADBuyhw1NwDAdjawDvl7E4ydv1s7ftPg1pAxrfukN14/hyKdD/HQHjdtoFFBvARguS3i38VzZ4I5bAGOdLrF/fkMwD4OUTQAeMfR/2D8wt3waINiUDqffwDoo0yuq95w4Azc8Wlgw9Ww9pWhU+2iUQHg5X1bQJ4skQHw0XOX4NNZO+DOqgVDrNfr9p+Gu0YEXCAiBeCx83bBjZs34emmJV1r7PTERT4A8KTV++HlH9dGBQqN+usXANceMhOOnbusXXbnO+3gqa9XwKwtR6PSVyMARpeBT2Zvh3urF4Kat8RmE+PZS1ehikcW4PembYHP5gQAeO2brSBbBnP/fQrAmwe3gQxpeRMczv9Jqw/AC7eVhrxZ0zt9LIQc7waAqUsMA7Az+aPBIc5aFD9HMwD7OBbRmHhiCTVd6pSw9e2E0DBuHjI+SuFL1RSA372vMjxQJzoAvKxP84hfAl2/WArztx8PAx106bhzRADqh7evAh1ru3OBwFibrT+ep9XzR4+GUKVwdl/GQK6UAvDILjWgnQcuEG/8ug5+WL4vKlAYbQBu+O5sOHD6onZZhK0eE1bFHIA7jl4My/acjKneFIAfqF0E3m3vPg7w+9O2wog5O7T+2HFfevuvTTBuQcACzAAcuCPE+6RhqVzw3RP1InqWuHk30fkQDQAulisjzH3Nn3TLEYnn4uRocIiLZsXFKQzAPg5DNCae0cPEzUPGRyl8qZqG3IomAC/t0xzy+WQF8QqA528/Bl2/WKbpPvqhmtCmUn5fxkCudPHOE/Dg2CXan70C4Nd/WQc/rkiaANxk+BzYe/KCphdaJ1/5cU3UAVhO+xoPz44zF69C1UEB9yYjAC7e+2+4eTMwA2lM22vXb0DKFCkgZcpADOoPpm+FT2cHANjOx2ssAPj0hSua61DdEjnhzTsr+navuh1bcR6G9d499PaI2uemDXQ+pE2dErYRY49VYzCEmlH0DyMXCAZgK1WTxu8MwD6OIwOwj+LihhYSczaaALy4921gN4ORkQJGLwEKwLJbhxM1F2w/Dg994a0vrp3rUwD+rHMNuL1K5Jvg4gGAl+85CfePXqxJ4GWCkqbvzYF/TwQBeEAreOWnyAAY/aWPnrsM+bNZL1OLOWgFwLgx79CZS1AgW3rLCAp25oidY+wAcInef8MNCYDRV7Tlh/Mgd5Z0MOmZBhoEfzhjG3wya7t22UVv3AYFs5tnH3vrr03wRZQtwAP/2AhfLdoTBvN2tHJyjBv4xPrFeenTpIQtb0WWhMJNG/ADodrgGVpXnQBwn0nr4Y81B+HXZxpA2fwJLoJCMyMAviVXRviHLcBOplaiPJYB2MdhYwD2UdwYAvDCN26DQhYvUaueG70EqA+wDPVWddLfaTQGr0DUzvWX7DoBD3wesAB7dd14AOBab8+E4/8FfHX9BOCXf1qjRYHA4iZTV/dvVsLUjYdtxWC2C8AjZm+H96dvg+dvKwU9W5W1Mw0iPoZucDWyAKsA+Lul/0LfSYEkFjNebqLti/h45jb4eGYAgOf3agZFcmY0bR8F4Ghtgnv5xzWaj63bcbcruBv4pACcNX1qWDewtd3LKY9z04ZT569A9becA7C4Vs1bcmgQLBcG4IiGMtGfzADs4xAyAPsobgwBeMHrzaBwDvOXqFXP/QZgaon99MHq2ma7aBQKwCM6V4c7qkR+3V6/rIWfVuz3HQ7M9KHj5RcAY3zpV35aGxEAi3ZmSpsKNg5uYzrkdgHYDbBEOtfcAjDN4oYxf8sXyAr/m7kdPpq5TWsS7uzHHf5mZfCfm+DLhdGNApFYABij32AUnEiKm/l04r/LUPPtmdplnViAxbWqFskOvz/X0DYAF82ZEeb1Yh/gSMY5MZzLAOzjKDEA+yiuBMBD76sMD/q4CY7uDLdjRbLquR0AlvtkVSf9nYLoJw9Wh7uiBMBLd52ATkELMAOw9YhRF4jV/VtCz5+9AeDM6VLDhkHmljo/ABj9bzGEWOVC2aBR6dzWAhgcQZe8nViAJyzdC7jsjeWv5xtBpULZ4NNZ2+GDGQEAlsMlqi4/6M+NMH5hwB2BLcABhcRcwZUvXAGLpLgBYIyUghFT3AJw7WI54Ofu9i3ARXJmgPm9IutnJBp5eW40OMTL9kazLgZgH9WOxsRLzpvgNh08C+0+ma+NYDQBeN5rzaBorsgswEYbeGgc4HfurQyd6yZEtnAyVZftPgkdxwR8Vv/3QDW4u1ohJ6e7PpYCsFeW56RsAaab4LwE4CzpUsP6GADwlwt2w+C/Nmnzx40Lh5h4dgC4ZJ/JcD3oBCyu9dPyfdDr13VaNSL6iXDhwL8Jq7DZBKcAHK1EGLj5cWIicIHwIkOaGwDGsJF1hsxyDcD1S+SC758Kj15h5AJROEcGWPC6fQD+bM4OQP/zV1uVjZqfvN2HdDQ4xG5b4u04BmAfRyQaE48BOPoAPPfVWyPO0mUHgIfcWwm61L3F1Qylm7Y+7lQN7qkeHQCm4J1UAfiD+6tC+5qFXY2LfBIFYAzR9apHFuAs6VPDegtfTT8swE9/swKmbTwSMQBTn08jC7AKgH9ZuV/TEMvEZxtAjaI5AOHkvWlbtb8Jq7DZ4NENaZEAMG4eLN57sn4psw+CeAZg2o9y+bPA1JeahMl3/vI1eOa7VVCzaA54sUVp03vDDQAfOXsJ6r7jHoAbl84N3zxeN6xdXgDwzmP/QfMP/tHq/v7JejFL1mMkejQ4xJOHYQwqYQD2UfRoTDwG4OgD8OyeTaFEnswRzRwjAF6z7zTcE8wE9/Y9leCheu4AmCZu8NJn1arTFIC9cr2INwuwXwC8sl8LeO2XdUof4Ge+XQm7j5/XoM4sM5l4HtjZrJTYAbhUn8lwTbIA06Qpv3SvD7WK5YSRc3fA8KkBAP7tuYZQrYh5TGwKwOhGgu4kbgpapxHSRUmsAHzp6nUo13+q1o2qhbPB7z0ahcnx4fSt8Ekw1JyV5Z++s/JlTQe925a3/EA/fOYS1BvMWm8dAAAgAElEQVTqDIApuDcrmwfGd6tjG4CduHqs2HMSOgQjxHiZJt3NnFOdEw0O8aqt0a6HAdhHxaMx8RiAQwH48rXrcOj0Jbj1/bm2Xjx2h5/6AM/q2RRKRgjAqh3s2BavAHjV3lNw38hFWvfev78qdPDIYmmllx8A/NrPa+HnlfGzCc5PAO7+7UpYvueUJrMACbr82+/28vBE4xKGwyCeB5jtDOMKm5V4BuCT569AjeCufyMLsAqAf19zAF78YY3W7R+fqgd1S+TS0iBjOmQsvz5T3zK7nVcAfOXaDSjTb4qt5xC1AO8e2s63ZXQ31lcakq5OsZzwU/f6YdPKSaQW2gY7Hwd4zMHTF6HBu7O1w+1ugsP4v6X7BvRvWSEfjH24li8A7FeWSKtnrd3fo8EhdtsSb8cxAPs4ItGYeAzAAQAW/rL3jVwIq/aeDhlVK4uEnSlAAdjORhqrOu0A8Fv3VIKuLi3AFKTf61AF7q/lLqOcVT/k3/0GYD/hwA4s4jF+ATAmWBHLvBSA6cv/tdZl4blmpSwBOHvGNLBmQPQBGFM5T98UuQuEHQAu3XcyXL0eCAQs7vG/1x2C5yas0v424cm60KBkbhjzz04YOiUAwAKKzcb6zd83wP8t/lc7JBILMLWc0jaqrk0BeNc77fQkHk7vP6vj3QAw/QAzciWg2Rqt7lE3ALz/1AVoNGyO1j27AHzxynUoPyBguW5XOT+M7FIzTB4jFwgnFuBYrbZZjbX4PRocYrct8XYcA7CPIxKNiRdtAJ637RhcvHodWleMTmYxs+Ghm+AEALt5uNqZAhSARXxRO+cZHWMEwKv3noJ7g5bbSACYbqaLJKWy0z76sfmOWoCtXq5O22v3eDqv/AJg9E+949NAGmwKTPTl/3qbcvDMrSUNmy3amSNjGlidiAGYhr0ysgCrAHjqhkPQ/dsAAH/zeB1oXDoPjJ23C4ZM3qz9bcITdaFBKfPoFBSA1w9sBVnSp7E7TUKOowBGx1NVGSZAmbgqEAd45zvtIFUwi52rC5uc5AaA9528AI2HB+Czebm88MWjtcOu0Hvievh+2V7t7zuGtIXUqVJazlF6gJWRgrYhXeqUsNVGJjiaPhmj4KBLlly8AeBT0H5UYLUtmu5mdudGNDjEblvi7TgGYB9HJBoTL5oATC0Bk19oDBUKZvVRPeuqNx86C23/F2oB9guAaXrUaS81UWYVsm5xwhG2APjuitC1fjEn1erH0oxykSTUcHpxuvnOq+gTuKkJNzdhiQcA9tKlpPHw2bDv5EWtbzIAv9G2HOTNkg5qF8upAwj+rXtTawDOmSkt4KY6s+KHC4RXFmC3ADx942F46puVWre/6lYbbi2bF8bN3wVv/x0A4P97rA40LZPHVJcBv2+Ar4MWYCsA/nPtQfj3xHl49tZSYVbb/y5fg0pvTtOvZdcHON4AeO+JC9DkvQAAU1cC1DVr+jTQsXYR6DtpPXy3NADAmKYYrbRGxc0zmrbBLgDTVYT7ahSCDztWC2uS1wDs5cex02ev0fHR4BCv2hrtehiAfVQ8GhMvmgBMrZPRXFY3GiIKwHgMWh5KBX2+6DlW1gU7U4AC8NSXGkO5/JHBvxEAU9/dwXdXhIddAvCGA2d0a2Ik8YTtaEOP8RuA/VwetgOLeEy0AFi0588ejeDOEQHLcO+25eBpGwCcK1NaWBkDAH7y6xUwwwMXCMy6h9n3sDixAM/afAQe/78V2nlfPloLbiuXT0trjNnd6N/MxpoCMC75P9awODQrlzfsFOrioPrYoxZIPNkuAFtZUJ3ek/R4NxZgBPym7wX2VAgApvc5xgUeNXcHfLskAMBb3moD6dOkMmymGwCmbbALwDRyRKdaRWBYhyphbTIC4ILZ0sOi3s1tSb3y3wQLMAOwLcni5iAGYB+HIqkBcKz8Su0CMFo635gYCIJPixcATF0gvLB++w3AGw+egds/CUBTJOHUnN4edEe0HxbgaADwb6sPwNj5u2B4hypQsWA2TQL6ovQSgGkYNAq6VPfvnqgLXcYt1f7Ut115eLKJ9Sa43JnTwop+0bcAuwEs1RyzA8Bl+k6BK9dvaKeLe3zu1qPw6Pjl2t9w05MGbMQF4vOuNaGVhftW/982AGaUs3qGUMBFtxR0T6GFZrOjbVT1l/oAbx/SFtKYuBA4vSfp8W7GZ8/x8/qm4lYV8sHnD9cCGm8Z5+1PK/bpmm0e3AYypPUWgDECSrPgxma7AExdhzCeOrrJoVX+sfHLoXrR7NC7XfmQ+xrnkNCnQLb0sJgBOJKplijOZQD2cZjiFYDPXboKGdKkMvXTUslCARjhoCPZWIUhZ3C3cPaMaX1UNLRq2QLco1kpGDFnR9j1vQBgagG2E0vUSgQjAKbWhEF3VYRHGrhzgaDaROJLbNUP+fekAMDiJUjdCPwCYOoCYQTA6Mva9YtlmtR2o0DYSVkr+oRuFsv6JqS3NbKKWUEcjd1rdazVvKKZvx6sUwSG3hduvVMB8ILtx+GhLwIfC6MfqgFtKhUI2QQ3qksNaFu5gOnlvQJgugRvpQf1AbZyIbDSzux3NwBM4VMFwPg8RAAWbiNWsZPdWIBprF27AEzB/ZH6t8CguyvB+9O26u8ICrxifETb8mdND0v62LMA01U7tgBHMjujfy4DsI+axyMA445ytDqVyZcF/n6hkaNwO2Ybq4SF1KvkB1bDgll3pm08DC//GAh6jwWtMBjySC52ABhDFm07cg4qFMiq3IEdCwAeeGcFeLRhcSsplL9vOXwW2nwc8I+240qBm0zwxZI3a3pX1xMn0R3RfliA/fSPFH1QQUI0ABgzl901YmGY/l8/Vgce/tIZAOfJkg6WE6hVDSrtk/CXxePcAjBdcrYCPqtJZguA+00BvG/ptRbtPA6dxwYA+LPONeD2KgVC4gDbeT71+229vpwv2ql6htDwYCoLMLViW+lBAXjr220gXWpjC6qVdma/ewXAPy7fC6//mpByGn30v1oUSB+9bmArzTfYqLgB4B1H/4MWHwaSTaRPkxK2vNXWUoYdR89Biw/nacc90ag49LujAtBoFX4AMF0dQhcZBHd8p6RIkcKyvX4dEA0O8avtftfLAOyjwtGYeE59gOkOZ6cwsW7/af0FPax9ZehUOyFNr5sHayTSN/9gLuw8dj6kiqeblIAx83aFVWsHgB/7armWgODlFmWUmYwoAIsUq5G0X5XFCuujFuA376wA3TwAYCtL8qEzF6H+0ECMzU2DW5smWrDqMwVgrzLQ9fxpLfy6KrAJTjVnD5y+CBgR5LZyeT3ZPR9vAIxwKpb1B9xRAR5rZPxRZGTVVY0b7SeuCG1+q412mFsApvPICvis5hHdcGtoAbYAYJGIhaZCtpOogG7oEu10A8C0D1Z6UAC28qG10s7sdzfP6V3H/oPbgpnOWlfMB2O61gKa+AItwHh/jl8YAOC1A1pBtozeAvD2I+eg5UcBmLULwDRKEG4cxQ2kcrxiqgdusBWZ+zBBx9I+CasiZprSvTEUgO/8dAGsP3AGorkHQ9XOaHBIJHMylucyAPuofjQmXjQB2CyygNMHK+7WPn/lGtxb3V1KWZUVAb/yxy3Y7QqArdpPAfj35xpCVYtsUlbTyhiAT0L7UYu10yMBYOoCQetBH7gJS/+FZmXzQul8WbTrYPgiDGOEJVL3DgrwfgCwaoOQGDsnrh5oIZ+/7Tigb2AmKdNXNAG40bDZsP9UIAqEkQV4/KO1odtXAb9Wq1WBWAIwfog0DCYrsAI+q/vDNQDvOA6dg/7SAnY/mbUdPpyxTbuknc27kQAwxiHGgpZnmr3MSo94BmDqfiAAmN4j+MzAEG5fLgw8e1f3bwk5Mhm7wrmxAOPqXCuHAEwNNuge92rrskCzSlLgxXbj/oISwcx9bgGYzi/RzyzpUsP6Qa2tprxvv0eDQ3xrvM8VMwD7KHA0Jl4kAOx0tzGNLCCH1rICSCrz0bOXoE4wr7tba6rqIYq7tcVDmF7PjgXYqv0UgCc92wCqF80R0cyxA8BW1j6zBlAApvVQa6rQhS5noh9q5cKBjV9uCgVgO9Y2O9egbTYD4IoFs8LfLzS2U6Vu5exStygMubdyyDmxAmD8sLo7mAabNuiLR2rpkQ2srPlOfBi9tgDTWK1WwGc1SG4BeOGO4/qGQWGN+2jGNvjfrO3aJe3ExO4zaT1MCIb0Eu1UPUNOX7gC1QbP0A559taSWrZFYSnFbJFoVRfZy6z0oABstYnMSjuz362ec6pzKQC3qZgfRnetGbJKgAA8afUBLdoGFkzpnStzOsNmuAFg6tJl1wJMn0UvtSgNL7UoA3JMcWHxxcbi6pJIXS37xZtparQ5XAfg9Klh/UAG4EjmrV/nMgD7pSwAJGUAlpd1nDxY6Ze5HYuMaohUD9FHGxTT/dDoOV4D8K/PNICat/gDwHQTWf87KsDjJsvdZlOXLv/RjVOq9LG4gaXXL+u06iK1bvsBwBQOzAAYfe0mv+gMgLOmTw3rpJdTtACYbp5B7X97riHcowDgYrkywp4TF7TxecsiNrRou51d7LSfGdOmgk2DI3OBoKGqrIDP6rFLP5KNXCDK9psClyUfYLoJTsAuXa6XXbdU7bALwHSTG/oA17olR0gINtxnIbKXWelBP/IidUPyGoCp/60RAGPkFLH6hr7n6INuVOwCMH5QYUi6DjWLQPHcmaDdJ4E9DXYBmCblebVVGehxW2mgMcWpxRfrpWE03QIw3Rwu+ql6xljNfy9/jwaHeNneaNbFAOyj2tGYeJFYgJ2G26EWYJF5beTcHTBp1QHYfvQ/XUkr4KRfzG5DSqkeopg2WA5fZPXiEY02A3jMKT9syhb9Af/rM/Wh5i05I5o5RhZgrwCYhkGjobNU1/15xT54LQjACGHVTNw73pm8GeZvPw7fPl5HaeWhUOeVBZgCsGrOirErXyArTHEIwKrlyWgBsDyHjQCYTjQrNw9Rp504pl4DMI0WYPe+M7qJvABg8ZH+3rQt8NmcwOZYO/6YNKuZaJ/qmUaTdagAuFSeLHoCCSs9KABbRVGI5MHjxFAhrkM3k6kAGDdT/77mIHwe3H+xrE9z0820dgH43pELYXUwrT1eQ4R1tAvAS3edgE6fL9G6IVKIU51lAMZnS+lgHHmzTaT4PqBh6kI2h5PoSAzAkczU6JzLAOyjzvEIwAP/2KhbSZ2G26FQFUnqYQrAbsPGqB6i6M8pL11avXisABiX/9Aqd+7SNX2m/Ny9vpadK5JiB4Cp5RY3GGFoHru7ielY9WlXDp5qEsgeRq+Lm20uXLmubf5DywgWK/cOofuDdYpqMCEXCsBepQW1C8Dl8meBqS81sTUsoh+Z0qaCjUHLp9lcoPPN7aqF3DB5DqP2Ig22USfevqcSPFTvFsM+ijoLZc8AmKDArHgNwNRSaPe+M2ofjShhZAEu138KXLoaGgVi/vZjesg48bHw7pQtMPqfAACL55Z8XYzpi/v0M6dLDbghds7WYyGHqACYRqpAF4haxXLAY18lJOEokTuzHj/XSg8KZhsGtdba4UeJFIDbVsoPox4KdYFAOP1j7UEY809gA/Li3rdBgWwZLOcoPUClb/n+U+Hi1evaYTRLYiQALKecFi4PeA18H5bpN0W7nhEAoyvG/aMWw13VCuquU0bRkYTW2TKkgbVvtvJjOG3VGQ0OsdWQODyIAdjHQYnGxHNqAfYKgEVyBbtf81RmumvWLSSprosvyu+X7QsbUSuLNJ5gpGPnsUtg0c4TIXX+9HR9qFM8MgBWuSLgRWiGJQHAYpPaQ/WKwtv3hEOnagpTaz1Nn0sBGJfJD525pEW++GhmYJOQlXuH0Kld5fwwskvNsEt7MbZypX4CMI1+IK5rZQGOJQAbAZyq7VZRXrwGYLpT3wr4rB67bgF43rZjesg44S89dPJmPTqM6gPi/OVrUHXQdMBIVV3q3mLbjYpaqVUAXCxXJt0n2EoPCsBW6ZettDP73Q0A03FVATAmBkIAFh8Z+OGFH2BGxe47g37g0BjZdgF4ya4T8IBkAaYJR6jPL7YVw8+V7TdVa7ZRHG0R2YGOJ3Xpoz7mDMCRzNTonMsA7KPO8Q7ATuNNUr/SSADYCz9R1UO0Y63C8NOKQLgsWiIB4I6jF8OyPSdD6vvhqXpQr0SuiGaOEwB289KiANyrTVl49tZSWntpAg7RAbR2oDUrAMDm7h1OANitdd8MgFWrFqJNZfNlgWkvO7MAq16msQLgic82gPtGLjKdV/LmU/lg2vbnbysFPVuVtQUiXvgAbz18Dlp/HAhVhQV32dtdsZAbSSMoGFmAqYVQ3OP/bDsGjwRjJovNn3QDq8qFZOamI/DE1wHLrVFRPUNoGxGAcV8ATcNcNGdGPQ4t1mv2HKK+qVZxdN0+eDBZEd30Zee5iNeiERiMAPjPdQf1GOzzezWDIjkz2pp34iBVWygA0w2iqo9W1cVUAPzSD6vhtzUHtcOpzy/+OxSA1ZkUW380D7YeOaedL9psBcDZM6aBNQPYAux23vp5HgOwj+rGOwA7jTdJIwsIS4rdr3kqsxexYlXXxV3YGJBdLnYe9EaQ2WnMYli6OxSAJzxZFxqUzB3RzDECYLpxQ/juRgrAwv8NG6wCYNzwcTQIwL90rw+1TNw7rADYC/cWWVhqtTED4DL5MsP0l5vaGhfRj7SpU2pLn7S4AWD0C8T0ydUKZ4cGpezNDXkO2wFgq01ccp1mc58eS11B5P7bnX/0Axn1xHn3ZOMSgBo7LaEArHa3sQJgcf8M/nOTHh1GtYnQLQDTuMcyAGPoukI5Muihu6wAmFqAcbkcl829Ljdu3NTDfFm1h16bftgYAfBf6w7CyGASonmvNYOiudQAPGn1/pDkRXYBmLoH+QXA+D4s1z9gAc6VKS2s7B+eStwNAOfImAZWMwB7PZ09qY8B2BMZ1ZUkNQCmoWiEJcUNAHuRLld13fuqF4KJqw94CsAPfr4EFu8KdYGY8ERd25BjNL2iCcBiBzS2pXjvv+HmzdBWoW/x4bOXtD9auXc4AWC3GxzdAnDpvJlhxivOADhNqhSwfUi7iAH483k74Z3JW7R67Hxw4XFuANgqjFcsAZiuOghB6ceXk0cthUsnFuC5W4/qSUMEAFO3L1UYObcATOMeqyzABbNn0LMxpkwBsGvo7YYSWAEwWm+HTd0KCFNPNw3485sVjKCAbk2d6xTVP2jxI01s8nIyT+0A8N/rD+obDee+eisUy50prHlylBB6gJUFmH4c2gXgxTtPwINjQzfBvfjDam3DHha66Q3/7QUA0w9UcS8yAFvN1tj9zgDso/ZJGoCD4ZjcADD1c3WbLld13burFdQfblYPV3nYjaxcXcYtgYU7QgH428frQqPS9qx8kQCw2Lxm1wJHr0WTlrzSsgy80Ly09rMKgIUvMP5u5d5hBcB0Q4gfAKxy2xFtKpU3M8x0CMCpUqbQ4n/SYmUBpqGOxHkik6ATsJDnMPpftx9l7gJh5X8cSwCmYy90aVQqN3z7RF3HT1k7AFxhwFRtEyfVfM7Wo9BtfCBpiPCh7//bBj06jCqRiFsApnGPVQCcL2t6PXJB6pQpYIc0z6goIQCsyKRGXTsWvN4MCucwdjHAelt99A9sOxKIzCPg8vK167qPq5N5Sg0fwvefzjOMvDJ5/SH4dPYO7XqzezaFEnkyh405BVL5RysARtcskSAoEgB+/vvV8OfaAADTTW/4b4y/XH5AwAKcM1NaWKWwALf5eB5sORzqAkGftSoANqrL8U3h8oRocIjLpsX8NAZgH4fA74n3+5oD8OIPa/QeiIfI9Rs39YDe8oOOWkOcBlynloDGpXNrvmFHzgZ8R2mxsoDRZX6RrtTpMKgA+I4qBeCvYCYmJ+3BY40gs+sXS7WwX7R883gdaFw6j2WTL165DhnSplIeV7rvZLh6PWCKpXrR0D2925bTrD1uAJj6pdH0zioAxg0raM3C8v2T9aB+SWP/ZicAbAVrlgIGD6AuEGYAXDJPJpjV81Zb1Yp+4Man3ZJlzg0AP/7Vcpi15WjYeJo1JhyAE17yRueh7/13S/Zqy+ufd60Z5mPrFoAx6gBGH1DdC3bnnxzXGOvC58Q3jzsH4IOnL+pJJIwijigBeMtRPWueAGCa2U2VXMYPAEYXCPStv+PTBZqmqpUGOsYUgNcMaAnZM4ZmUqPZGnHTWYWCWU3nuWrMLl29ri/xy88d/DemjMZNxF8+WhvK5g9kicRCXd9ur1wAPutSI+SZhAA8ZcNhwIx7WGa+0gRK5U04X9RjBcD40dNpzBJoUT4fDLizAtA4z+ia1WF0IENmJADcY8Iq/R0hAzDGX64wYJp2DbcATH30xRgYuVPYelB5cJDfHOJBE2NWBQOwj9L7PfHwSxa/aEVBkMKlMswmtW7/mZC/i39QAHYacJ1uhjCTzQqAKeR9+mB1uLNqQcejoAJgfDj/vT6QipQWq/bgsUYv+Ye/XAa4s5yW/3usDjQtYw7A6Iv8+q/rAF+4jzQoFtYmOwAsojfYBRB6EQrALzYvDS+3LKP9rNKNArCVf7MVAM/afETfCOQVAL/84xot0xQWld+6aFOJPJlgtkMAxjrl+eEGgJ/4v+Uwc7P/AIwxmtHPGou8217e5KTqG50jtJ9eADBd2RHX8ROAKw6YCuclC/CU9Yfgme9WaZcXANx74jo9OowquYxbAKZL+iof4FyZ08JdIxZqbUmbKiVsGxLqa07Hgm6CU6US/mrhbhj45ybtFLcAfOHKNR3wzOa97EtvBcBTX2oMU9Yf1rPtTX+5CWASELnQTWmqZzSFU7wnKQCja1bHMZED8HMTVoFIV003vWF7MHxb/aGztaYZuS2ofICNMqSK+yt3ZvWGujCBfPqD3xziU7OjUi0DsI8y+z3x6MMeu4FL83mzpgvZeCE/6OIBgOmDcETn6nBHFW8AGIO0T914OGxEIwHgR8cvg7lSTNDx3WpDs7J5TWeOFbQaATDV5vU25QAD7FvVpWoIXY5+4bZS8EowGoAKgAvnyAD7TwUswFb+zeJ8YQmi1z7+32Wo9fZM/U8qVwE3t5ttAM6dCWa/6swCbAYC9Deqm6pfT369AmZsOqJ1z270A3ksqJXLSCeMdSyWYOXd9vImJ1XfaL1eA7AKcNwCMPWvNbIAqwCY9gkBGDMpNnt/rp5Jj8bWFlq4BWCa+APTcONqzQtBgwRagHH3v4jrnC51StgqbbakY0EBGJfe0QJpNFZuAfi/y9eg0psBC6fZvJezoNHNjSoLMALw1A2H4eOZAQsw/rtc/nALNTV8yPMbn9EYvQNdPUTbKACja5YIaWbXArxo53HoPHapVp/wRX/uu1W6kUQGYHxOfhJ045ABGP2nMc12n4nr4eCZwH4J8V6hAEwTrTAAu3naR/ccBmAf9fYbgKdvPAxPfbPSsgcUACkAO804JMf5NLqwFXDSpbDPOteA26sUsOyDfIAK5FpWyKdDCD3eqj14rBFk0qVtUSe+3JqViy0Ao4UXffxw419K3GEjFQrANByWSrciOTPAvpMBALbybzYD4J+W74NevwZSKmOJNgBjutQ5MQLgp75eAdODAKxK16ya4G4AGNM9bzp0VqtOBuBr129AqWAmK3E9u1EgvLAAL9pxHDqPCwCHKLEGYPQLfzToE4xtolkRRRvdAjAmyWn+wT/KZxc+I7JmSKP7dFvFrqUAvLJfi7Asi3SuuAVgTPZRZeB0vb1GKx/5sqaDpX1a6MfRpDr4rMZnNm3PtJeaaAAsYokbtY+6vsmiYVu6jV+mJyCRLcDomiU2tEUCwM9+txImrw8YSeimN/x3j2alYMScgB+zHLps6JTNeqIP+d6yBuB0sKJfgp7KCePjH/3mEB+b7nvVDMA+Suz3xJu95YieecisG/RBN+jPjTB+4R7tcKcZh2hKTLvXUx1HX5Qju9SAdpW9AeDm5fLqfpj0upEA8BP/twJmbg5Y9kT54pFa0Lx8PtOZY2W1NbIA048DEb/XbEneKNYuDUf2XLOSgFa0/1u0B8bO3x3W7ltyZYR/T1zQ/m7l32wGwNRPEeuyilhg99azawEulisjzH2tma1qzcZH/Eb9g0MswO2rQMfaRUKu8/Q3K2DaxsA8sRteUAZgzDB4f9DP0agTaGnceFANwFeu3dAzWckvaVV9XluAaRIKcb0mZfLA14/VsTUm9KD9py5Ao2FztD9FYgFGVzBM0iAKzYoo/uYWgOXMd7T9CMBZ0qfW/VZpnGWVGBSAEZYwEQMtXgDwmQtXoepg5wBMAc8IgNEY88GMQDIdzNpWqVC2sG6qXGTEQfgukq3TmJUN5zQWXJkSH1eRAPAz367U/JXFfSrCnuG/zQBYZThQWYBpohpxjllaZcc3hosT/OYQF02Km1MYgH0cCr8nHt0ZbNYNcaPS+Lt4vHMA/g9afKi2eNDrWwEnBeBRXWpA2yAAo//oxFUHoO/t5QFDCJkV1QOpWdk8YSlMsQ6r9uAxRkBELXuiPeMergUtKsQHAOPGvxGda4RJRQEY3Shwl7aAXPlgtJzici4WhBWEFqNiBsDfLf0X+k7aoJ8abQBGkP/HQwAWoatk31pVv+iL1e7KijyHqZ+jkf6VCmWFDQfUACxvcrKa+/T6WdKlhvURboKjERhE+/0EYAQmXNan/aR9Qmsv+q/S0Ig0K6Joo1sANlsRQwBG6O0UzERGLeyqsX3t57XwczCG+fK+LbQNdLR4AcAnz1+BGm/N0Ku1awG2A8BoJHhv2lat7j96NIQqhbOHddMMgB+ufwt8vfjfkLZRAP7uibrQJbi6YPUxISqh7xnhAiF/qFIARkPBZ3MCKbPl9MVmAEwt5AzApq/tuPuRAdjHIfEbgNEnSTwUzLohHnTyTew05aaZxYNe3wo4abtHP1QT2lTKr50u2ocZlTAklFlRPZDwZStvWLOCAHENIwDu/s3KML13F58AACAASURBVL9i3H3fqmKgzUbFCwuweGibWYCFRUZuB01J/HTTEmHLd/T4Erkzwa4gAH/VrTbcauLfbAbA3yz5FzDklCgqUESYxOIkQxjN3mS2CQ4zb83r5Z0FWACwHFVF1S+6tGo3kYE8h398qp4OTEbzqnKhbLD+QGCDq+wCgVFHRBgncb5dFwgvAJhugBTXx82iuGnUaaEhxowswFYA/ESj4jBuQeiKh/Crp+1xC8By5jtaJ+4TSJ86lb5sT/VVaUEBeFnf5pA3S/qQw4wAGNMPo/vBmK41AcOumT3PZB99IwDGuOBL+jTX66IALD64ZRcICsC/PdcQcLOmXGj8d/k33MC661jgIxwLtq1M3ylw5XrAAowrU12/WKb9v10Apu8Z8SylBg3ZBYICcNb0qWHdwEBUFCyRALDsU+30Xoj0eL85JNL2xfJ8BmAf1fd74pntqqXdMgJgpyk3zXzeVNczknb+9mP6wwwf3K2DMCkeMlYhg4weSBhzdMGO0JBl4mFqNcxGwErBRtRBod2oXrcArNq4YQrAwbBEcjtoSKqnm5SAMfN2GUpAXz5WG/zMAPjrxXtgwO8b9evIWcsQfh/6YikcOn0J/n6hsWGIOLmhFIBVoftEm7wGYBEjWPatVWVjo5trVJuYVOLLL1W60cdosKoUzqZHeJEB+Pzla1CRbHKymvteW4CnbTwMT0t7Em4tmwe+6uYPAFd+cxqcM7EAqzRUJeZwC8A0OoJ8LbyP0qVKqS/bozvEegJUePzpC1c0SyN+DIYAcJ/mkJfArPy8oz62YgxxA/DorjX1ZtCMj+L5f/TcJagzZJZ+jF0ApnFuVQCMUR8QgIdPDViAMWlFjaI5wuSXVyDpAfQjXMxbCsD4ESVSXNsB4BP/XYaaZEOuGHe6WZXG/cVrUhcIBmCrN2bS+J0B2Mdx9BuAzR4odgDYrqVK1KVKC6ySz8oC7BcANyiZCxbtDE1aYQUBov1GwErD5iQAcA1oU8ncb9kKgOnDnerlFIBFYHozAH6ycXGl7684B+Pn7gxaX6w2+JkBMA3VhHXLoEjDRqlS0hrdinYBuEjODDC/12227miz8ZE/xGTfWhrrU1yMhnBSWfDsADDd6GPUiaqFs8HaYIhDGYDlXf5Wc99rAJaj0uD1nQIwfiRtPnROi5vb8qN5mgxGFmCvAFhluZb1Vz3T5NTP9BwE4DQpU2offFhkoBIhLNE9Ca3SFICX9mkeYs3F840swOLvVYtkh9+fa6g3QfV8OXL2EtR9xxqAMTHO4t4JFmA7ADxr81EYNjWQCRGTVtS8JWfYFF757ynDRC/UDQtPxEgqZftN1S3AuDIlNjPaAeB3p2wBtI6LIgCY7umgcX/xOGoBlj9Y7FqAMU53l7q3aJcV58ibCm09oDw8yG8O8bCpUa+KAdhHyf2eeNTP06wbwmIg38ROAVj1EFBd1wqA6WaZsQ/XAozeQB8YVlmT5BeCaEPd4jlh6e6TYU0yaw+G5smULrUesF6GBpo5SFRsZ+OeawAmO+lFCmMzC3DbSvlh1EMJlh/RRvqywVBQX0hLwVQkTCG8/Wgga5TZBj/qC6sKg/blgt0w+K9ArFIsMijSFQRVPFajOUzTl5pZgDGeMcbGtVPsALCI3Sr71tJQR+JadJ5gPNEC2cx92FVzmG70MeoDjQMsA7C8y1+ey3KdXgDw3hMXYPvRc1pYwMkbDkGPCQlxyfF6TgH415X7oefPa7UNZOcuBfx7vQRgmhZc6OEWgFWpn0WdCMD4HBPL9mY+pfh8ogC8pHdzyJ/NnguEGENcGfijRyN9iMv1nwKXrgbcB8Tzj2bXU80NUVfBbOlhEQFgGlMcY7Zj7HY6d2a83ARmbzkKQ6cEABg3c9YuFg7AqkQposEyAGN2xvL9EwAYP8y7fRXI8GcHgOWoDQkAnBCvWwZgjOU8cm4Amp0AMP0QUgGw7FJi5/nk5TF+c4iXbY12XQzAPiru98QzewDL3ZJvdvx9rSLlppkcXgHw3K1H9a95uqFM1O8WgOsUywnL9tgHYJrik/abAjO1Popj7IRu8wKAe7YsA883L63coGe09CnaSAH4sYbF4cuF4dEfxLFl82WBrUcC6T3NNvjROLMqAB43fxe8/fdmXUoZgKkP+Zt3VoBuDYvbuvsoAKuStwgtogXAdKOL6AAFYBlMjTop3092ALhG0eywam8gEYZ8HXmXvwpyaFvCPoiDzwN57tr5WMBoJKlTpQjJTInXwo2p4x24QNR6ewYc/+9KiGSd6xYF1FwubizANC24qM8tAFPLqNw2GYDNltTxedPrl7Xw04r9WjWqDygrCzD6hv/5fAIAq2Ik09jKqrlhBMA0pKIRAGOs9CGTA/c++rLXLRGeTZLuS5D1wggue4KRaPC37UPaQsUB03QL8JeP1tIjHtkBYMxK92EwKgXWJwCYhrXEzarUZSgEgMmmUDzfzAJMAfjteyrBQ/VCLcCyRd3WQ8/Dg/zmEA+bGvWqGIB9lNzviWfmgyZ3C1+WjYcHwgqJokq5aSaHVwBMd4urAFj4XjptC26eQ/CTi5EFWF6yF+fR42kaXvG7nex1lgBMQvyEuEAQC7B4WZtZgFtXzAdjutYK6zN1j3m0QTH4alEg9J2q0OQK1CIvH0s3g6k2342dt0t/CeK5sqWU7pofeGcFeNRjAJYtV3bnj5EvZNrUKQHTpcoZtKiVR1yDQvo/r90Kt+TKZPlkke8nutPd6GQKwAtebwaFc2TUDz11/gpUJ7v88Qe7m+DwWFyJwfF3A8ANS+WC9jUKwys/rQ1p+m3l8mqpde2WaoOnw+kLV0MONwTggdN0K7HRPgf5ujQtuPjNTjhJlY7UMipfBwE4VYoUgJkksZgBMEZAwE1/c4IJd9wAMIbHQ796UaoMnAZngxZ00Xa6sVA1N4w+JK0AGFMfIwCLj1+jbJJmK5Y0FCO2DTeoYcxisQkO3xNPfL1C655VTGU8Rv4YFwD82FfLNWs1Fq8AmL6HVQDs5Llk9z5xcpzfHOKkLfF2LAOwjyPi98Szm5gCu4gvSxFXU3RZAPC5S1cB41DWL5HLFEo8A+AtR/XlLPyyv61cqAuEWwCmy8N0WI0gAOPivvlHwqYtcQ49vudPa+HXVQHLjCifPFgd7rJI3+wWgOnOZfGyNgPgVhXywecPmwPwI/Vvgf8jIYbkKU8BmG5KlI+zAuAx/+zUl0HxXDMAHnRXRWWKaNXtiNm1RCxXVYgxI8uV2a1tx6opsnfJm8voS05cg9Y3q2dTKJkns+WTRb6fMAmJ8Bk1OrnWLTlgRfAjTwZgeeOPCnJovUZWLbcAfE+1QvDaLwmJUPBaTgGYWnVFW70EYJoWXNTvFoDNgA59VnFzm9i4ZWdTlWjPojduCwsDaWUBLl8gK0x5MQGAMdwZhj2jc4D64KvmhhEA036qLMAyAOOHXMNSucOmsBMARlenKoOmwdXrgagxGHlHJH2yyqqHx09Yuhf6TFqvt0EAMM3sKccephZgOWydmQWYAvBb91SCrpIFmAHY8lEYswMYgH2U3m8A3nXsP7jNIBOR3C2VBVjknH/7r016uCAnFiMj6ax8gOkLh266Eg8ZEX7KLsCI4/wAYOqbJ67zvweqwd3VCimbh24CN27eDMnIpdKDxrikv3sFwDTkkBUA48sTH+JYzCJc0GgIKgvwqLk79Y0wWJfsKrDtyDk9TbcfAOxkqdEOAAtLE34gViYZtOhLDlOkHj5zKWR1BXfFo09jmlQpTZ8u8kuVhnoyOpG6+cgAfOzcZag9JCEVdTQBGDeg4kfhGxMToMMNAKPf58Wr10O6LwAYtaaaqqycVh/pLzQvDbiyQotbADbzaUULMOZnFBu3EIDXDGilPRtSp0qpXFIXbUI/dnTnoUXul0g2If6OH7FTX2qin4LzAOcDnQM0dbMTAKauCzjGaACg7UEA/mfbcXgr6P9vlEzHDIAxgsvek4FkPFgQTqsPnq4DMH6Yiwgjwjff7Ob6fc2BEHccOwCMGxLxGYbFCQBTFwgVADtxzTJ9YLj80W8OcdmsuDiNAdjHYfB74slLWmZdwU0YZy6GLi2KcE00NmI0AJj63NGwW/EIwK//sg5+XLEvRNqPOlWFe6sXDpMb4ffeUYvg5PnLemphIwgxAuAF24/rVsCXWpSGl1qUCVuSppvRxLK13BgKwHKQeflYml539EPGES4QQEoHU+2qAPizOTv0YPh4DRmAadxUJ1Eg7FqAney2tgPAIuOUvLlMtL3vpPXwx5qD8GKL0iG+z+iKs+3wOfjlmQZQNn8Ww9tShhoa6snopDrFcwKmlMUiA/DRs5egDtnlbzT38O9ycg9xPbz/3ViAEYBxNWLgnwmbILFOzM74hQMXCBq9QLQJAfiB2kXgwc+XaBvi+t1RQfvJDQDTtOCifjsAjFEJDp25FGKZNYvCIwMwbqoqkSczoJUeP5AqDJhmOC/EuKLrDUYgyZ4xbRgwi+xiRgBc751ZcPjsJe0a4pkuh7E0cv0pnCMDLHg9YTMpBX0jAJ6//TgMCo49zmOM/ywXsw+GIiQdO56HITprvjVDB2A0Orz4wxqtSjthMjHxz7PfrdKbIAAYLfKYQAoLxsGnH7bdm5bUI0dkSpsKNg5uo59v1weYPteMLOo+Ioeyar85JNr98fJ6DMBeqinV5ffEO3j6IjR4d7brHoic89EGYBp3kyZeEA8MmoLWqHOqB1L1otlhdXCDED3PCOrluLUUAsT/9564Dr5fFgrAH3asCvfVCAdgauG0ur5XANyifD4Y90i4CwTNumQFwDS7mFmECxoOTAXAI2Zvh/enB9KhYpF9ZY2WCq0mMN1gpspeKOaCk4DzdgBYbLaRN5cNvrsiPFy/mKkVD/uEIct+Jzvz5X7Kc5iGejLShEY6kQEYLdH1hiaEuaLwI9fnNQCj+1TtYjngk9k7Qi7lFIBL9pkM6GpDCwIwRo7Zf+qi9mdxP1cdNF3/qLfrA4yxXl9tXTakfjsAjB+jH8/cDjSTnFliBxxL7EW38YHIBbTghyFdnpd/x9U6jAJR6+2ZWv/wOU1j2tLnlJhDZfJlhukvN9WravjubMBNb1h2vdMOUqZMAXIqey8BeOGOE7o7mVEscbMwaLIG6J6HVmzhAkF/t+Mi9/e6Q4AhLEURANz1i6WAsI4FIRv9jEVxC8A0ExwDsNXTPL5+ZwD2cTz8BmCVxcdJdwQA0/SQ0bAAz9h0BDAgORZqLYhHAMYXFfqT0fL+/VWhQ81wALYTVULUYweA76teCD7sVC3MIkejMRgBBloJO45ZrF3OygWCJlcwi3BhBcDyzmsZgI12S1vNWbsALKxiVvXh704AGBMWVBuckELWLgDLy9Jyu2QARnBQARM9zwyA5TBXeJ7R/UznEK3frQUYARgTqnwn3SstyueFcY/Y3wSn+rBFAJ6+8QhgJjPaJzcAjLFeX2tdLmQo7ACwrBH+m95j8thqAHwT9L0O9HdcJqcZE+VzEYDRDaRVMA4yugsZ7VUQepXKmxlmvpIAwE2Gz9FdCjCiArqOyB/oRgCM1lgaT5uC693VCsL/Hgh3gaAArAqliKt+GOVBuElY3aO4Oln3HTUA23GR+2vdwZCQfHYAmLpAiNUf0U4zC7AVAMsWdau+e/273xzidXujWR8DsI9q+z3xVJtenHRnRb8WkDtzOqDpfqMBwNM3HtY3NHz9WB3AFMZWUGIFD2b9NurTN4v3QH+SuUzUgcej1Q8jJ2DSDrHpSPw+vEMV6FirSNgljVKjOvEBpklCxMtehjW6Gc1okxF9OVtZgGlyBbMIF5evXdeC02NRWYA/mrEN/jdru66LDMD0RaGKpGA0hrECYLEMipuJcFORKMJ/2crf1DEAk1inRlqYAbAc5orColyf1wBcr0ROyJkpLUxefzjkUkYrFEb9Mwbgw3p4tEgswCLxBL2+WwA2y8RpBcADft+gAbKqzHutGeC9JhKBYMQU2bVEQKDQC5PZzOp5q15ds/fnAvr8YhGxs+kHugoiRV3hAHwS2o8KfEyrABjjAOPeBdFGOZIMXY2y+35a3rcF1B86C65JqwH0GW1Wl0g0Io4RAPzQuKV6tlCMg48fUaJQC7ATAKbhSFUWYAZgu6Me/eMYgH3U3G8AVsX9dNIdfMig1SzaAIy567t/u1JrKt0wYWaVk/tlBR/0eEMAXvKv0hKDx6tSIIs6VVEA8LdoATDdjGYEwJjgo9PnS7QmWwJwkeyAoY6wmEW4oAkhVAD84fStIUvgsk70RaGKpWs0dykAo99elvRpQg4VcwE/5vCjzk6xYwEWG2HkD00Rws1qDjoFYBrr1KgPZj7Aqj0BRnOffkTJ94obH2AE4BSQAhbvCs3E6CcA05Bp2E8jtw7aP4QcdGOgxS0AL955Ah4cG7jH5IIAjBveHvsqsNJFC1qAB/+5Ubm8j8dhGD30t2/xYSATngqA0Q1gx5C2ULz3ZO0YTCU8+9UEAG7x4T+AcbexiHuGfoCqYq2LcZdTilNfZyMAxhT0wgdYjiTzv5nb4aOZCa5Rdu7PZX2aa+59dgF4/6kL8NGM7dCpdhHAewSjxuDeAVEEAHcZtwTQWo3FDIDlWMNmFmD6XBOrQ1i/0QeFnf57eYzfHOJlW6NdFwOwj4r7PfFUqU+ddCceABhDPzUqHQiZYxeA7bzoqA79bi+vxan8+IFqmsVblG9MANgMbrC+JxqXCJPaiQ9w2X5T4PK10ExNWCHNkof/Vi1J081oRokGKABjWB7sq1GhvtNmES6sAPj9aVthxJwEH1AzAFZlUzNqn30ATgsr+rW0dQvYAeAswWD4cnQFkcTDCoAxwci0lxN25ssNk8/HpePH/y8cmOh5FIDlaAGYla3Je6GxvqMJwNeu3wxbLfEKgKdtOAwnpLBeMgAbQT3V7+kmJaB3u/IhQ+EWgBeRmN3y2CIAY3tU44kAPOTvTXqmNvncua/eCtduJAAwzjcBl+JY3Ai25a22gD7TWDDqyBwCwG0+ngdbDgeS24hwlxTU5EgK9JmKMXn/ea2Z3izq63xPtYLwseQCgZv6cPOuyAIpb6T9dNZ2+IAkpbBzg2Is5MbD5hgCMG5KxDBzorT733zYFIxkg3PeKApE57FLYNHOIAAPaAVVBydYgJ9uWgLG/LNLq5IB2M4oJf5jGIB9HEO/AVhO0eq0K8v6Noe8WdLHwAJ8CLp/G9ig4AaAjZZvrfqPCSEG3lVRP+zbJf9Cv982hJ0mQ6d8gLAmyH83isusgpBIAJj64hqlmqXLs1YATJMrYJ9UsVLx71YAPHzqFj2VKB4vAzDNnDWsfWXoVLuo1ZBpv/eYsAr+WndI+38zC3CuTGlhZX8PATh9alg/sDXIvvYD7qgAjzUqbrkJzikA49Kx8I03EsYMgPccPw+3vj835FTV3Lt45bq2k75UMKIHPcGtDzC6Zly6dkNfSRB1ylFKELTwwy99mlTKLhq5QKgAGMNknQomzcB205URs4klw5NbAKYRW+TrWQHwwD82hm32E3UgyCI8oxUXC843mmJcHJc7c1rdLQQzqc0l0EqBULi60cQdcixd+kyVAZi6MNxbvRB8JO1LQADGjWXCv1feSCtvjrVz0+PHXdPhxgCMqZLRCi6K/EFrBMAYSUSsUmAmVCMAdhIFgn5Y0PCORhZ1O/338hi/OcTLtka7LgZgHxX3e+LZfeAbdRGXmfJmTQ/PfLsSpmwI+O5FwweYhqhxA8B2LD2qPouHt/jNLQC/cFspeKVV6E5yrNMLAMYQPSJ4vhgP+eFOfXEx3BBuJJSLEwBWRc8wAqfyA4x9gN+dskUPI4TtkQGYZpQa3r4KdKwd7ketGjcKwLhzO6uBC0SOjGlg9YBWtu5oOxZgkbzgyNlLUJeEF+t/RwV43AcApsH+DQGYpPuWUyGr4oLL44gZsoZO2QKof8+fQ7O2Gc03O1ohAJ+/cg02HAjEkxZFTtTy3HertExcfz7fEErlDQ8RZwTA6DYlJ3aQAZiujJhNAgRMtJiKMock5rEzeYSm8moNPRcBGC3iInsZ/U3+4JSvObtnU0DXVwHAuOJEU4yr2ii7Ldzx6Xx9LMRznsbhlbOp0XeJDNN0P4EKgKe91ETzqxUAPKJzdbijSkG9mXJ4RDsa49zGjzk5Iog4F90/MJ6yKPIc/W31AXjpx0DYNCzCaNFpzGJYGgwjiJZxurmVWoCdADD9sFcBsPxBYaf/Xh7jN4d42dZo18UA7KPifk88p64AclfFg5H6u0YbgN34ALsFf3n587ul/0LfSWoLcPHefxtuUnmqSQnoIy2jorZymCGht0rTcv2n6Eug9Hc7ACxb/lVWUeqfaGUBtgvANCXwHVUKwIjONUKm1NDJm2HMvMASIhYRFB7BBJcdMTWsyHRmtJFQdTvGCoAxdjb6CcrRFYQLjJULhByaSu6bfD4N9m/0WMJQY8v3BNJ9Vy6UDf58vpF+KPp8CmgymntWbXZrAUbL9NmLV/Vld3F9GYDF9RGYf3y6flg3IwFg+mFo9lhHwMSYvKK4BWD5XqXXxI9SXKmxsuir2omZBLE0DyY56tuufEiKcdU58kYrFQDTOLzyEj9dVXIDwLh5V0C67EblBoDRDxqTPBkBMKYox1TlosgAPGn1fnj5x4QPPFsA3KSE/vxykgiDAjBN8S7aJOvpI3Ioq/abQ6LdHy+vxwDspZpSXdGYeFYvNLPu/fhUPdhx7D/AuLwiD300AJjGaLQC4CnrDwGCSAOSWtOupUfuu+y7awbAJXr/rVlhVAXDig26u1LYTyoAwYP8BmAMY/aHFG920c7j0HnsUq2ND9UrCt8uCQ3lRhuvyqCnarMVAKNf49j5u/WqBQDLf8cD3utQBe5XRNJQ6U0BGIEU5wMt4h7InjGNlm3LqsgfjkbhoER9crxtvwAYs/CJzaFGfaAALM8t1QeYUd+M6o8EgHGz4M5jgcgDorSumA/GdE2IUy3GCmNP//V8QupecbwRAONzgLo74PGyBdiuSxj62BbzwAI8Z+tRw7B1uEqAVlk3AIzhzNC9VQAwTVRjNG5ytjEKwEv7NAfcJPbb6oP6XgDZwkm1k/2J6X4CDM34aMNicNeIhXpTpr7UWPMBFgD8cadqcE/1hEyZcoZIq/sTf8ePFPyYM3oGb327DaRLneBGIwPwxFX74ZWfwgEYQ0OKRDKYCbU6ie6CBhLxAe8EgKlriQqAZT3t9N/LY6LBIV62N5p1MQD7qHY0Jp4cezXS7kQDgGmIGrMwaNRigf5a2TIGwMeupUfWQg69ZQbAqoD8or5OtYrAsA5VwqT2AoDnbj2qp08VOeTlh7vqRS+Pmx8AfP7yNaj4ZiCDlcoCjEugXywgAHx3RehqkDDCKJayav7aBWBhsbW6B2QXGiNIFC4VCA+NhiVsLkOL3JNNSlj6ADu1AOPmIeEb7waAVZswowbAxXLC0XOXtFivdgAYYwbPJmG73AAwhqYTbhHo13vp6g0QLjpmcwDDjBXNlVE/xK0F2Mp32I5Li6qdCMDo3mo3zT3WIZ4Voj4KwOhPi4kxaBEbPMXf0C9caCdHlKDuVAjAE1cfCKkLAXj+tuO6lVpOFDT6n52A7lFOCmrQ6iNjAN7yVpsQP3L5Gfnryv0hLj7CAnz/6EX6CoocHx1X9j4PrmC5BWCxQRb7KtrEAOxk5KN7LAOwj3pHA4Dd7LA167K8QYQea9farIJo3CiAS2EYhujfkxf0EDWqRBh4TawDYUr4leGyYMngsqVdS4/cTzn0Fia4UGVkwmuX6jPZcAey2Akt1x8tAKYvK9GGMAAmO9StLMB2XSBo1BEVAA/6cyOMX7hHl0XExFTNmw/urwrtFclEVHMTMzrhqgEWMwuw8Nm1uqVlFxorAJbDi/VpVw6ealLScwAe1aUGPEPSt6r6UeuWHCGRFmjbVYlYognAGIdYZB8TbW9TMT+M7lpT74qYCwWypYfFvZuHdVE1V7rULQp/rz8Ep8mGNzyRAjBq17RsHtMUw+JicgY9twBMU7qrxsqOS4sagJtAqpQpAWP52i35s6aHJX0S9KQAjM9OYU0W9cn3Cr235Y8TCsB3Vi0IaMCgBQEY/aHfmRyAXPnjdsw/OzW/cycFN9ZhJAsjC7CIbSzqlAH4l5X74VXi4y4AuMOoRWGRSkQdFIDlDwTVvBT3Ft3boAJg+YPCiQ5eHBsNDvGinbGogwHYR9WjMfHc7LA167JImykf48TfWAXA1KWA5nW3C8DCb0/sIi/XP7AZy0mRLcBGANygZC49VI6qfvmlLo7Zeey/sBcN/ubEBYIuqxpZgKkrgrh2JABs1wXi3KWrUDmYOlQFwLizHZOHiGKWMe2jTlXh3urh2fRUeocAMFkJkF9+WYJRG6zmBPV3VI2PeNlhYgfMSCWHF+vdthw83dQagEvnzQwzSHYuuV3ySxV3zz8bAQDTVNNG88LqI9atCwS6Zuw7eREOn70UJj/OFUywgmGrxPWNrPVGAIxRQDAtMB2vmm/N0EOj4d8xTXal4AqF2RxY9MZtUDB7Bv0QtwBMM1qqrucWgDGxBG7wcgLA+bKmg6V9EmJgUwBGmBRZ5UQ7Zf3PXrqqpwWWAZjuJ8BNt+j7TMuUFxtrCYMEAMv+/WPn7bL0YZb1Q6jGSBZGALxxUGvIlC61fpoMwD+v2Aev/bJO/10AcPtRiwAz26kKdYHwFIANVjusnlNe/R4NDvGqrdGuhwHYR8WjMfHcbDAw67K8uxaPRYvj7Z/Oh12Sf59RPSrgow8oCsC4W/rWsnm1quSHmGwBXvXvKc0ijL6lL/6QsMPX7hDKUQm+X7YXek9cb/d0/Tij0GPRAmDqimAEOpiZqcs4ez7AVUkiDKP68O9WAIzpXWm8YTMAlv0EzQbBNgAH4/ZaDajsQmNkJRVh1f49cR6avpdgjcNECriSYQWTTgEY01BjX81KzVtyhLzAdcn6CQAAIABJREFUadtpqmmjcbRqcyQAjJnHjv93Rdn8X59pANh2cX05Dq04yQiA0ep49tI17TDRZxmAcTOo+EAz03BJ7+aQP1t6/RC3ADxt42F4+ptAQh9VsePTrToPgRUTVThxgZDTgFMARkBt+7/5IZeSI6bQpEoyAFN3KtXHMtaPFmBh5cUVEsyUiZCNHz0YecQqioWsw4Qn6mobZo0AGD920E1BNW9wfvy0Yh/0UgDwfSMXwqq9gaQ/cnFrAabRNZQWYAZgq0dyzH5nAPZR+sQIwCJvPJXl68V7YIAiZbCRdDJQyGCI8CNC1IzvVhua2QRgeRnP6dDRNJV47g/L9sIbLgC4Yalc8N0T9cIurwpDRV/Y9ASjKBDUAiyWieUPA1UCFFlzCsC4hPzdUuNNcHYBmFqJVBbgvpPWh1wn2gAs++0ZzQ/ZhUZ2+xF6Y5xVTKyBYEetcZ3rFgV0p7GCSacAjOGjekxIyF6lan88A/D2o//pbgpy20W4Q3kuy8c5AeBab88IAW45s5fR+IvoN+J3twA8dUNCPHM1AFv7dKvOw7BiGONWjuhh9ryTsyBSAP7r+UZwx6cLQk4Xqxvij6fOX9E3hMlplSkA43L+rmCKZXHu5Bcaw7ztx8L8fB+sUwSG3lfFFQBbPdstAXj5Puj1a7gF+N6RC2G1RwAsou9QABYxwrH9Yi7Lelr1zevfo8EhXrc5WvUxAPuodDQmntcWYDm8DMrj9AtehjGaahnrw+VvEaLGLgCjC4QTi4hqWGmaykgAGF0kJjwZDsAyKIk2OHKBIDFJjQCYWmKNrkGD9HsFwLgEXXVQIHOSCoDRmo5WdVFETEwV1JhlnJPHDmPHog8oFpHVih4j6pd3thvd2rIPtRwdJAGAA6mVVR82GKaJWoVV1yqVNzPgZh6jIuuCaahp+lbVeXIMWTq3aKpbo3lhBe2yBfiX7vWhw+jFelOMrOXom4yZx/DjTFVEtBe3AIypbc9JFuAwAJYSGxjpLhIAid/dAjBGpjDz2bazqVHVRlz+RwuwSIVs5xUlJ4GhADzp2QZw78hFIdXIAEzTfeO8xXPwGdKkTB4tuUnn4GoSXkdk5BMVIgCjW8SwqeF+vjhfvlywW5nIw06/jI6xAuAfl++F139NWN0TLhD3fLYQEFhVJcQCLLlTqe4b4Uayeu8pXV8G4EhGNfrnMgD7qHk0AHjk3B0wfOpWz3ohh5fBiiMFYJpoA+vDXcIiRM0rLcvAC81La+2XX470ul4AMA1SjteTH5JORESwwRcFLU4AuHz/qXDx6nXtdAoV9GVsBMDUEktBB7M5fbdsL2AGMvRzFTF30WKJ/s5Gxa4FmC6T4mYY9Ouk5Y1f18EPy/fpfzIDYIS9u6omBMs3094uAMuxTY3qtPKhFvNQLCurNje+3qac8oVPr+kUgG+vUkDf7GfUdjMAphmp6LygdTkFYLkdZgC84eAZw/S+wtffTwCWExsYaegVANNwjqpr2dnUqDoPXQoQgFt+NM/2I0l2abj9k/mw8WAgKYn8EYN/k4EZI3jUGTJLOx5XLtCijBnTWpTPC90aFtfdqbBd1yS/hL9faKQBsOo9hPNl/MLdYamcbXfM4EA59rk8r+TVPQHAd3+2MCxbobjEk42L62Ec5f0ERvcN9o8CsEiSQ99nVs+BSLWwOj8aHGLVhnj9nQHYx5GJxsRzE2PRrMtyeBk81msAxggANAuVeKmaATBmb3KyKUTVRxqjEX+PBIDxfBkGVKloVcfh34wAmIZWMgJgaomloEM3fqDFresXy7SfLQG4cDZYu/9MiGQqqzUFYJUFuNcva+GnFfv1eoTeqpcHwjNCtJ1iF4AzpEkFm99qY1mllQuJDMCqDH9+ALBlwzH2bdHsIUu4dJxiCcDomoGWQhmORJ9EEhrXALzmIJwLWpdFn2ULMG5YxMgQVsUrFwgazlF1TTubGlXnoUUVXSBaf2wfgGWgpQD8/ZP14MGxS0IuJdx7xB9ptkMEYHRnEQXdV8THtKq9VgD81cLdMPDPTVbD4uh3MwDG5458PR2ARywIe9aJC7sFYBquUwXAVq5Qjjru4uBocIiLZsXFKQzAPg5DNCaemxiLZl32A4Bppjm8NiZBoDt021XODx/cXy0khie+5Ch4ewHAdIMCtsNrAJY3S1E4lTW3A8AitJEMDUYA/PKPa2BSMEYnxld++Ev3AIyA+2HHaiHZlk5fuKKnDlUBMIYdwvBDopgBMG74QounnUIBGIPX58iUNuQ0oY+c3tWobisXElGf2Fmviq+LG+GsYpta+f5ZWWPtaEMBmGakMpp7VteUXSDkNhhZgFWh9FTnegvAM+H4f5f1y6zs1wJqvj3TUjYZgK3i+Rpp8PuaA6abcd0CMAIlAnCbj0M3rpl1TPYBpgBMP4ZFHfLxNNuhUwBGH2O0AL83LXwlEueL1R6SNKlSwNXrBlmHDDptBsCqUwQA3zViAayTPvbF8U80Kg7jgnHM5TBxZhZgCsDCnYpGTWIAtrwlY3YAA7CP0idGAN40uDVkTJuwuxblidQCLAPw8PZVQjYo4DVealEaPp65XR8NPwCY+mfhhX6SNko4nQoyDHgNwALAwgD4wlWoOjjgi0tB55Uf1+hB6ikAW/WrqsICjOfIetGNMioXiFd+WgMTVyUEyRcfHKqXB8JBu8r2APjpb1bAtI1HtG7kzZIOJj7bAArnSEhkIOpPlzolbH27bUh38UX05NcrteQpX3Wro4GF7EKCy7o73mmnnyfqEx8gqvi68QLAdIxoRqpoA7DRHJLnqBsAxjjWmMVM+BcnWIBDAXh53xZQe4g1AKPvZo9mpbRkJlis4vnK94/YNGkFwHaieqjuTQTKlClSQLtP7AOwHAWCAvD4R2tDt6+Wh1xKBmCM3yySZcgArAJoWpkVAH+zeA/0N9lEjb77568E3MHslnUDW0HW9AkZIa0+7AQAU99o+VpuARjDqmF4NSwCgGmyHauEOHb77Pa4aHCI27bF+rwkCcBHjx6Fnj17wtSpU+HixYvQtGlTeO+996BChQqa3mvWrIEXX3wRVqxYAbly5YIXXngBXn31VX0sbty4AYMGDYJx48bBqVOnoFGjRjBy5EgoVaqUo/GKxsRzE2TcrBNyfEU8loYjsyOADIYyAA9rXzlkgwLW2b5GYfh1VYL1EMOxYUxZET4H05fe6iAwvKqd8mYnrwFYjhdrBCH4dyMLMH0ZGwEwtcTSa1AARp/LR4IWYKsxM4IXsWwtzrcCYGqBxnO8AGCV/yBuzEHAF0W8/NKmTgm4iZMW6p+H0F2vRC5IlSJFyAeEbDkW9QkXFFV8XYwFbBXcPxoWYOyruN9oQH6juWcGCph9bNfQ202jWxhZgMsXyAqok1lRhViTj1e1zwiAEXaPnUuwAKNvr/BjtZrzVDdMBf/E1yvsnKIdI2KlT1q9X9/MqzrZTlQP1XkIlJgK+fZPQiM3mDVQjgNMAViVkU4GYDnZC72W1cc0thczWL4/fVtYE3HMv13yL/T7bYNh89HfVmxwtDsIbgGY6iJfC1e1MN40FjlOspkFWAXANNkO7smY9nITu13z/LhocIjnjY5ShUkSgOvVqwcpU6aETz75BDJlygT9+/eHRYsWwY4dOzQgLleuHNxzzz3wyiuvwJIlS+DZZ5/VALdbt26a7Ai/+O/x48dDoUKFoFevXrBr1y7YuHEjpE0buvxqNk7RmHifz9upByD3Ys7Iu2uxTqe7eOWXJF3CxvqG3lc5LP6unGITY1FWLZxdX9L3wgUiDIClWJFO9ZP76TUAo7VzWd8WYZsDKYhS0KEWWIyv/Oj4UKuPUf8MAbhpCejdtrx+GqadFT6WKgvwiz+sht/XJGSJQuskWnnrDQ1srqEFd8i3qWRtAVa9eCoUyAqTX2ysV6cDcKqUsG1IKADTLFbihN+eawi4G1wUow0vAoBV8XXtALBRul9xXSurld35KOYhDcdE5wWtx+yaaB3f+U47VwAsWw1VbfcbgJf2aQ513wmfa0Y6Ct2sElrI56NGqJWcblc+zi0A/9kjAMBy6DKz+SCnQqagp3LFkAHY6NmF18SNyh/OCIdb0R4EYNy8+4HiGNTYKOU8vf+iBcCYXGOTxYcatssZAJ+E9qMCkVLEO+bq9RtQuu8U7W/l8meBqS8xANt9nkXzuCQHwCdOnIAePXpAv379oGLFipqWa9euhWrVqsGyZctg5syZ8Nlnn8GePXsgderAUn+fPn1g4sSJsGXLFrhy5Qrkzp0bhg8fDt27d9d+P336NBQsWBC+/PJLeOCBB2yPTzQA2E2WHbMOyL5VeGzEAExS2WJ9mJGt76RQiwCmF/6NwJPcRi8AuG+78vqyJ9YvB0u3PbDBAyMB4AoDpsKF4LIfrYdao8SyprxsTEGUgk7Pn9bqVvQvH60Fj31lz6pVpXA2pV8cJnvApX5RaKgkFQA///3qkDSpCMD4YvtoZvjLE5MEtKmU31Tyw2cuKeEZrY24U14UoQ/6Em4fkuDKgL+rALhtpfwwZcNh/Xw5JJSoT0CFCoAx2L/IfGXUiWgDMLV203lB22cGwMIVxOwYIwtw8dyZtHjJZiUSAJ606oC+TC7aIFuAccVhkIPNVqKe6RsPw1MmCS3kPolkQXK6Xfk43OiJ94TT8kePhpACUsCdI+xbgAtlzwAL37hNvxQF4G4Ni4WkKMeDZAA2imBjp+0I7Bi/XAXJqLFRxk1RtxsLsJwS3epjUrhAYEIQq5UKbFekAEyzTTIA25lFsTkmyQGwLOORI0c094a5c+dqgHv//fdD9uzZYcKECfqhCMUtW7YEPBbBuG7durB161YoU6aMfgy6QVSpUkWzDNst0QBgp/65Vm2Xl5bweKdhbOSXZI8Jq/SlJawPM7LJS2J3VysYYj2U2+mFCwRCy1NNSupVy+kyrbSRf5f7abSMqIqoEAkAUxCloEMBeNzDtWwv61YulA3WHwiNAoH1PnNrScBoB6LghqNawU1GKgCWLf24I/rilWvKpVFME9u6ojkAVx88HU5dCKS/pUV+oYiXn+zLi+eoABgz+c3dmpDOlS5R0s0rAipU8XXjEYDpZhw6L6h2ZqAgPiDcAHDhHBlg/6mLpreQFQAbpVtHFwj0LZc/GGUAdnv/WmV0k+sVyYKsPqDtxHVWtRkBGMtdIxJWKaz6hvoveF0NwKpz5SgQRlksra6Lv1sBsFXCIUw7LCJ82LkeHuMWgNt8PE+LV21VnADwij0n9VjZwgJMs00yAFupHbvfkzQAP/XUUzB27FhIly4d/PHHH9CqVSsNYtu2bQvDhg3TVd+0aZNmLV6+fDns3bsX2rdvDxcuXIAMGRLyxXfs2FH7219//WU4WmfPngX8T5RDhw5BnTp1YN++fVC4cGFfRtlrAF47oBVky5iwuQAb7TUAY0Y2eVME9b9SCeUFAOOy9dNNEwDYyoJjNWB+ADBdjhVWGtkCTEGUgg6NwoCAaZamlfbNCICfvbUk9HIAwHK8ZwRgzLqm2h2OfomtLADYCMRknzpxnFjCp31TAXCd4jlh2e6T+mGPNSwOA+4M7A/AWMol+kzW/l8AsCq8WHIFYEyQUL1oDm1cMS25yDCIGwYPn71kesug5bRUcFkYD5TvH7psTCtCAP515YGwuNl1hsyEo8QH2Op+lX8X15+64TB0/9Y4pbF8nkgWZBVFhqZ8d9K2359rCBgTgbrpWJ1fNGdGmNermX6Yma8rHiQD8I6j5xwl3qDtQWCfs+WYcqUHNbbSCTM4GiVQMeq3/J6yawH2G4DFKiPNNimvWFmNpde/R8MQ53Wbo1VfkgZgBFv0+R01apRm8V2wYAEgyHbu3BkGDx6sa4z+vSVLloT58+drFuCuXbvC9evXNT9iUR5++GE4ePCg5kJhVAYOHKj5D8slMQGwKtOW1wCMCRLe/GNjiEy3Vy6gZ/tS6esFAKM1E62aoiRWAMaNP/Jud3zRvPbzWvg5GIbMSRB+IxeI55qVhNdaJ1iA6XVVFmA54x8CMC4FqjJERQLA8q5q8fITm7jo/FEBsDy/KADT3dvCqqYCYHzRDZm82fQ5jWljZ796q+ExVi9tuy8BAXJ0M4449//ZOw8oq4kujl96k9577yy9994sIEVARUEQARGxwCe9g6KCHVAUlCYWEAUEpHeWpffee+8dvvPP23nMy5skk/Z22c2c853zyc5MJjd5yS937v1frZAF0dwJI2OojdaFOb9YtC9AtQWFGETeev44UJgpMmCB/5/UaxMVKEFn9wFYv6Sx2lasWBCqHqL6oVazCsCIUYc3XF29Te9+yJU2KS3rKQ/Aat1gkdSf7P0HAF6y51zA/cDff0bJxlZUIKwCcIMxK2jvWWMPcKqkCWjLgPp+E+glwW04colaRlZL9ABY9q6JHv1iNAAzE0PVISwsTAltgJe3cePGQg/wpk2blGS3Fi1aCD3Ad+/epdmzZ2teuajwAJtVaDC67UQ6q2aFzNUvNnVsqChWTx2XqV6nTOlZo3Pr1bAgda35RMnDaQA+cfkWVf1kadAyzIRABHqAE1JEv3pBSXBaAMwXojAjwaTlAYZU1IcNCvrPh68WJQLgN3+JIKyfNWwHAihFagl2AFitq8leTkgcOjzy2QD7ywAwYiQHPu/LF+Czt7OnSUIre9WmpweAnyTj8ACCc2r9/TpKlCAurT5wUfNnwlQ0ZABY7e1HERJW2VDrAOpKberfhUjfmgEwfqt37j9SpmbjKoxYRGevPVGBMPr9q//O5jEqaawex7TSjWJbrQIwvOwotsaktWTOS/2xZZTspQZgkdSfzHHRBx5rAPCXi5/IWPL3n5EMGio4svAW2WNaBeD6Y5bTvrNPinxoHc8qALMwO77cujppV/YcnerneYC1LRnjABgSaIsXL1Y8vfHixfOfOWJ/Aaj4N0ifTZ482f83UQwwFCPgFWYNMcAlSpRQEuhkWyhuPLMJakZrRzUlJAXx7ec1R4I8tnrzGAEwkqOGzAmsDNSgaEa/1qtobicAmCVCsPmNsriNbGUnBKLogPlBST04Hp+Qw15S6hAIHkT5Fw0PwGbiD4tlTUE7TgZLWL1TOx99UJ8D4Gt3qHxklj3KGOMYfOv48wZatPuc/58AwI8fk9BTKhMDrAVi6tKiavvwa5IB4HaVc9GgF3wAzG/DMwAWFZjAuTGJPq37JNQeYD4Wkb8vZFUOmBycDACr472Nfiv4u1qlQf37EcW2Yxw8wCIArjhisWHYhcxzat7209R16iaZU1D6MAA2kvf6olVJ6jFji/S8rCN0ruEBZsoCMhOoJfeMAFid+CmKc5c5LvpAZ3jz8Sv0lQCAhzYtRv11JNAw3goAq3cqje5Z9uyvO3o5obS5UVOXlpb1ADMA5nczPAA2snbU/T3GATA0fkuVKqVAcO3avqSA+/fvK9JnL7zwAmXMmJHGjRtHBw8e9ANy7969adasWUqSHLy8GTJkoNGjR1OHDh2U8dFZBcJpAEY1pbTPJAq4I40q+ahvX2wRrj14kcrmSkOI71J7gEXwUK9IxgDvoXrOFT1rUfVPg72rZn460QWA+QxhrJ8HASkA5kCUB53//bGNZkQcV/7JjPdJC4C7185H75sA4DcmbVA8QaxhOzBu3DhKrKi62QFg9cveSQDmk1dypk1Ky3vWIqsADGUEqJdoNaOXtuy9ze4ffiuWjV3buzYhNKLbNGM1AlZK2mhdON7bKmUXmbWu7FWLqo168htWAzBfjpefDwD8e8QJuvsg0APsFADP3XZaOR/ZxgDYyLOJ540o/t3oOH928QFwi8htdaP++Lt6V8QsAIt2OWSOy/q8Xikn/bz2qJkh/r4yuwfqic0C8If1C9DbtfJR7t6++H6jZgaAkU/w0nifDBrLM7l59wEVHegL9ymaJQXN7f5Etcbo2E7/PRSOOKfXHKr5YhwA48GBJDfE8iIBLnXq1DR8+HBasGCBUgAjceLEfhiGvi+k0bp06aJA8euvv67YvW/fvjR+/HhF9ixXrlzUs2dPZb7t27dHOx1gs/G5RjdWRL+6ikQO38wCcNuKOWnyuqNUNmdq+qNLZeo+fTP9vfWJPqwofrJu4QwB3kP1Op0AYDwEu9XO75965qYT9P5vW41Movl3qx7gQX/vVIp8sMbPw2ekMy+NGvBEoIA5PvpzG/26wQfAo18qIX1umgBcJ7+iAcoaf1yRB7jdxPAAdQV4QxLEiyuUppKRQdMCMbVn1S4A4+U9uEkx5TT55BUWVymqsCbjAQ41APMvYv6mlZXjgidu15CGujrAmBcgi7huVjRA9ge06P3qAYlW6t+PVgiR2wA8Z9spqQ8Edp67hzSkJAnjGZb4lbWLut+fXSrRw0fkhyqZedSJoUYArAY80T0uc1zWB7syMp5V0Zyo4Mg+bmSPaQWAscbOU+Q+dLSkEdXrwz0sAmAk9RWLBGA8X+e84wGw7LUNZb8YB8Aw3tWrVwle3b/++kvx3larVk3x6DJdYMQBo/rb5s2bKXPmzErVOGgHs4YEOGgDoxAGkuiqV6+uhD4Ahs20UHx5mY3PNVo/qillSJ44oJtZAOYH4wGhLpAgyqCvXShDgPdQvU4nAPiDegXonTrOAfBvb1UiKAqwJiuDpgY7LQDGvKJtY5E+rhqAP2tZgqAKIdO0APjdOvnpPQ6A+eOKAPi1n8Jpxb4n8mI4NpIOxy47GLQMmUIYWgCsBku7APxapZw0RAeARRXWoiMArz90kVp9vy7I1kaFDNgAWQCGXFqjYpkDPmpl7rO53asGVDdTA/CRCzeF1R4BwL9FnFASKtHYOKc8wP9sPWVKrxfJfIhtR9ENs7GrMnb6ozMA+LHwWmqNV0ttmQVgUREVmbWyPvhYPHLxlpkh/r5WAFidq2K0awHnR8TRywEf6HqLNQPA/O8OuunQT+fLrSPH4p93qlqyjRODQsEhTqwzKuaIkQAcFYYUHTMUN57Z+Fwj24T3qUMZUgQCsNFWn96ceFn1+HVzQJELPCQ+/neP0VIC/u4EAL9XtwC9W9c5AOZfxvj/bgCwukob7Hn66m2qNHJJgH3w771nbqPp4T4P8KjmxanXn9ukbKwFwD3q5qcedZ94gPnjigC47Y/raeX+C1LHtAPAfMa7OntdDVVaUMgvEjsWiFVE45NXGGhbBWB1Zj52pxA3jC1fJBcavbSlDMkBoUy8s96cyMbfKeEBxhxGut2i46BiGF/dTH2t4EFEjKa6aQFwpZGL6fRVfek1o2cT/o7dKexSyTaUix/yzy5/uJHsONl+v3euRA8ePqY2PwR/zGjNoY4zNQvAIg1p2fWiHyQDT17R14HWmg/Jl+zjRvaYVfKlpfaVc1PdIhmVIUa/JQDw5mNXaDEXoqV3LLsAzCd0QmXn724eAMte21D28wDYRWuHAoDteGdFpw6PY0YHARg/fiRC8YUWIEcmksbSuxROALAa6GZtPkHvzZDzkmqtjX+Jy5ZC1vMAG2mS4ninrtymyh+LAHg7QZoJ7eNmYfSRjkQTfz6IUdt5KjgJTv3BYATAr0xYp6sywB8TMm2NwvRLIWu91FhsLuYLG7ggQEQfyZRjXymjxB6jyUAhAGtY0zClP5+8wkItRN4xSLypY5uRRMaUCpSXMidNhdAKFD9Bdj/aovdrCGHPyuOI3YOIuzcDTepjIV4fpdCNYALjXiyV1V+mXHbNRgC8/+x1qjdmRdB0CgBvOEH3EBfAAb9dAD48sjHFiROHZm85Se/+Kp+sViF3GlrP6UjLnr9sP+wsPXj0iF7+Yb3sEGLb7AivgqID1DH0KvOpVQ42Hg1WEJE+OBFlTJHIsiIH5PfYtTVzTP5eMLpnsfu39cRVWrT7iUqN3rHMADD/jGEe4Ku37lOJIQuVQ3gAbPaqhq6/B8Au2vppBOB1vetQppQqD/C6o4aZvGbMCDmyUfP3mhlCTgCwekv/r80nLWVp8wuPCgCGp6WKAID7zNqulB1FE5Wb1jK4FgBj67w7FzLCg7fIA9zm+3W09pC2zBZ//O9eKU2NLQIwL/ovevFNal+OahbMoBxOBoBfqZCDhr/oA2A+do+VMhaVGBYpmaj1TFFKefEHNZV40W+W7A+oiGfF66V1/dg9uObgBVPQpJ4PJWm3D5ID4GalstLMzSdN/YZRMYwv76v2AGtp0QKAZ2w4Tvcf+r4e2LjKIxfTKRse4MwpE1OlvGmpRoH0pgDY1Elb6DyjU0V68OgxvTJBHoABWfAcF+w3X+qIagDWih+XmiyysMaFG/dkuwf0Q0gNu7ZmJ2D3ggwAwwmzkJNpdBOAr9y6RyWH/KccArt4sz0PsNlLG5L+HgC7aOZQALCd8ATRqSNrPHPKJxXw0MdI7sesCa0AsBMyaIA5Pqlr6vqj1HfWDrPLD+jvPADri/LjeDIALKq2ZxaA1THT/HFFANxq/Fppz5gdAGbyZDgf0YsPGsjPFvd5l2VCIF6ukINGRALw9Tv3KWyQz3PD1CZkAVhU0hWeJCSNQSN5zUG5jwOzN6RTAJwicXxa/VFt//nrraNZ6axKeWIzDQUT+PK+agDW0qJFiAp2NgCFTgIwW7tVuTIz526m768A4IeP6dUf5QG4dI5UNLFdeb/X0eh46lK/Mh+KenNiPmz7W2koYc6urdnxuIf46o1a4/Eswy7X/J1npA6h1knWk0Hjd15YsaXLN+9RqaGRAJw9laKVHFUtFBwSVedm97geANu1oM74UNx4UFsw0lk0c4prPqpNWVK5C8BW5IEcAWCVrJeR10DGbvxL/OjFm1Tj02VBw9Qvev0QCGMAFmXL4xh9Z233l6YVVdszC8Bq2TgtAEYZ1TNX7yo6oOFHnpQY1rMfD6la/bSuDytRjHFGACzzYm9TPgeNbObzAPPJK0xvWBQfKfIAAyCv3XkQdDpjWpWg8csP0Z4zxhWoZO45dR8/AB+4QC+b8Bqq58H64ZmW8eRYP8pOAAAgAElEQVQ1L52N/tx0wtRyAQFNvl3tH6P+Xew+fY0afbkyaE63AdiMYoqpE7bYefqb8AA/orY/hkvPAMWdb18prSTmyTQ1AP8ecZx6/iGXMyAzv5k+KGGOpD8rDfcQwulEibb8fHB84ANr3vbQAPClm/eotAfAVi5pSMd4AOyiuUMBwE57Z1f9rxZlS500wCpOeEr5Ca0AsBOlkNWFHZ5WABYl2+FF0O+v7TRlnS8EQgRodgGYB2/mAZbxvoiO+83Lpei54ll0f31RAcB88opZAMa28pVbwV6wUS2K08TVRwiA50ZjILn6wAVT2+bqtZjx4rUsk81fdlv2nNRFEdQAvOvUNWr8lRiAp4Uf80OSUyEQbN1m4uVlz9VOPwAwYmJf/8kcAI9+qaS0Vjo+drYNauBfphPPQqvnjHB9i/yrhMPIrB0AjBAbWek+7OZAAQcqP8WyptQ8Bo7Phx6xaqN8UZeS2VMRyltHVQsFh0TVudk9rgfAdi2oMz4UN57TAIzt2uxp3AVgZOR+tnCfKcujoECtz4K9q2YmebtWXurZoJCjD32nPcBGZVlxPBkAlpHpYoZABvkuAZypS0fzx2UArC7qIXs97AAwYmvX9K6jHMrIAywTAtGmfHYa2ay4Mh+fvMKKC6CQhLosraicN8Id4PlRN3iXJ689KrSxrL30+rF7cNX+C6a2zdVzagG86Nityma3rYKgBmCtYgzwAIsAGHHwVpUH+HMys1vixPUymmPamxUUVYR2EzcYdQ34u5m4/+gEwChhjkRpK80MAENlhNejlz2e3jHUAIw58Q5F3H/ZYYuUQ3gALGvp0PfzANhFm4cCgJ32zoYCgJGMJqobr3cpnADgrjXzUq+G7gGwlo6pUQgEdEU/+G0rlcmZWpET6qJTlhVJing5qqvi4RgIhUFIDJpIa1nLvoUzpxB6J1k8GxvHAzBksL5sXUpZS4F+/5r+FaE4w/MlrHmAkby0VgeAebiWCYFoXS47fdzcB8B87F6BjM/QwvdqkChDftDzRWjQP4EV7tI9k1AYPoB4bMjTiT4yTBtOMIDdXyv3nze1ba6eCsURLgs82KI1wmas6IrVczADwHjOMS8hG+cUAJv5rVg9VzPjpnWsoBSGaD/JHACbOQZLeGRjZLyoZuYPVV8zAHzo/I0AOU7ZNRoCsCr0CEm6qOZXbrgPgEvlSEWzunoeYFl7h7KfB8AuWjsUAIysf2T/O9WgtpAjbaAH2OljIBlNVDde7xyWfFCDan8erBFq5rxRlAFQh4aYs7x95Mpi6h2Df4lbBeButfLRN0sPKIdBclhXHQBGxvqQJkWDYo2xjgGzd9AvkeVIzWgta3mAmaSPHgDzpYPNXIuv2pQieJH1mtZLOVOKxLSuj7YHmAfgzpM3Gia+wJsJ7xliEQGALHaPVdeSBeD0yRPR+et3g04J3uI/Np4QSs2ZsZlWX3YPoggJipFYbVoebNF88JozzWmrx1MDsKjkNOaGB9hNAEaC1Of/mduRsnrOMuOmKgD8kN6YFCHT3VIfVvSEDY4KADZzv+nd+zJrh6Tj0Us3TSdu4rhmARhjUFCq/HBfPLYHwJZu0ZAM8gDYRTOHAoCRHd1bUu9V5lSRbJYzbbKArk4fA7G4Xy/xAZ9sW/yBTzfV6lYZjoMKPYA6NL7krewaRP2cAGCAINuaMwJgeIjxgqypCgdRA7CZOGstDzCra8/Om9c5Zh5gq3a0A8AZkiei8L51lWUZhUDIvBxRhRClYLFV+Unz4lQmcuuSVdeKOHKJWoxbG3D5RdvmWlqoKP3915aTrgPw8n3nTcWNqu9nZL5fFIRwiO57KGcwyT2rvx81AIsKjmBuAPCU9Uf9v32nPcD8B6jVc3Fy3JQOPgDu8LN7AIz18jkRMr8TJ88Rc4UagI9fvqV8iJptRgAsir1HQanykQmJUOiY6XmAzZo9JP09AHbRzE8jACPUABWw+PZr+DHpogoy5kQs7rdLg8vj6o2d8FpZ6jQ5wnKyBOZ+q0Ye6t2osHIYPtlJZs1affiXOITnRXHKRiEQCAVAOVY0qCO8PU27Xn221ElococKQcfBMQbO3kE/R3qAzXi1tAAYW8Odquf1nzqvcmEXgL9sXZKalMyqa3qtlzI8rRscBGB+ERH96vpj9xgAbzhyiVpKADBik0W6tIilnrf9NO046W4S3LK950zHjfLnrhXCIbpI0E6eGqk5bfX3o/5daJXjdQKA1RUV+TW/WS03/bDysNXTcHzc5A7llYIqkM5zu7FrEBsA+OSVW0pJbbPNCgDzJewR2oaQiKhqoeCQqDo3u8f1ANiuBXXGh+LGcxpOEWqQJ/0zrgIwQhGMZGvcuCydquehPo19AHzhxl0/6Ng5liwAI+QCEFQkSwqqowrl4AG4buEMtGj3Oc0lIb7s5zfKCwEYVaAmrTmijMWW35hFctu6AD2RRBc8l29Wz+NfCx/iwQCYLx1sxo52ADjdM4kIoIomenHzIRBmX+z81qVZANYqB4sM9IW7zrgOwEv3nqP2JhOnAgE4kfK7kGn4SNarNCYzhxqARXrLmAcAzGLb8d+ImU+aML5SDEY2CQ4QgmRGUXutUk5/6JDMut3u88sbAOCH1GnyRrcP5S8qYvZ34sTCQukBRhXQM1fvWIpbNwJgUfIp9PRZuXpI1P3hAbATt4zjc3gA7LhJn0wYCgCeseEY/e9P52KAEWqQ1yYAG+k6whMLXdRQN3h6+j5bRDmsqJywlfXIArBesuJzxTNLy/OgvO5P7coFxUNjHTwAm4mz1gJgKEl0rKYPwHzpYDP2swfACSmiXz1XAJj33CA2et671UhUJatxWKYgTVEU6Dh+6XaQGXAtlu45F1AO3IytjPoeGtGYflp9WFEHYTsARmNEf9eKYbYyl8wYNQCL1DZEAIwqdKNblaSqnyyhE5eD7S06drlcqWnDETEAO6FoIXO+sn3wgYsPy85TPAA2splsEhwAGOWhWal4o3n5v+sp6uD4IgCGnj4rV4977/fOngfYjM1D1dcDYBctHQoA/m3Dcer1p3MC5over0HQP+WbWcg2KvP6VvU8NH5F6AG4Y9Xc1O85HwBrFa0wezvwL3FkGYsS9dBHr0ras2GZae7201KHhudtwutlg7zIeImjDPGxS7eUeczEWcsCMB/iwTzAN+8+oKIDF0itne+E6ltNS1kLgeCrNDntAeY9NygRPbd7Nalqcji3nGmT0tGLPvvzDeojK/dfcA2Anapkhtjqc4IkPtMXV3JAMABfouZjA2OtMZXIQ4uxZgC4fK40mkVampbMYkkdQPI0TXdDKW94gDtP0Q6FMj2pxoCoDIEwE3Oudb5mABi7G0wn3Un7idRXUFGRlav3ANgpazs/jwfAztvUP2NIADjiOPVysILPoverU74MyW0BsJFJoyrmrkPV3NQ/EoD3n71O9casMFqq4d9lAfiVCeto9QFxKVyRN1HrwHnSJaPvXyurJATqNUDXd8vk4qy1ABi2gs1Y4wGfAfCNuw+omAUARnW0F0tl0z0HrW1ZyHVtHlBfGes0APMvrmJZU9Ccd+QBWCssAKE3KJe6/eRVw/vJSofXK+W05fllx9RK4rOyJpkxagAWJRvqAXC1UUuEHnfRsSvkTqNZprtRsUz07w65CmEy52W3z8T25RQPsJ4ajN1jsPFRCcChDoGARjdTyXHSfiIARkGpqp8sVQ6Dj6/fOldy6pCm5wkFh5heVDQZ4AGwixciFDfebw4D8H/vVaf8GQMB2GkvM6Dqx1WhTzp5o0puGvC8zwOsJbpv9nZgLxBkbWMLuu7oYKhGH1R1Qpa+qJl5AedJn4y+b1tGeBx+bihejFtuD4BRTe4NDoAPnr/h9zzDa/ZF61J0/c59Chu00KzZCOVnm5W2BsAo2LBFAoC1gEpvsdDBrjbK9+JiACyjJYz+uDaHzt8Mmr59lVwUceSyawDsVAxrVAOwKNTEKQCulCetskMialABWbJHO+7e9M1tc8DEduXo1r2HusmwNg/hHx5bABja80h8ZjkSTtpPJD/IP0dQTe63tzwAdsrmTs7jAbCT1lTNFQoAdrqG+4Ie1algJhUAOwzZAFHELIa6AUQGPl9UOaxWwo3ZNeEFgjAAqD9cuX1fKQyhbujTYdIGWqzxkjULwONfLWPovYbX8XvJMBMtDzD0a9tXeeIB5gGYeYCv3blPxUMMwLCv3oubJcHl6zOPHpissQoZwBqf+ioOhmVNSf+8U1Xx3rb5YZ3hrYHKcfvP3QjqB0CFwsG2E+54gJ0CYOgrn7l2x/A8neqg9gBrVe3TCoGoPmqpP+THaE2V86alNQfFAFwtfzolRCW6tJ/alVUAuNu0za4vKTYBMJ5VKEnuZIP9PAB20qKhncsDYBftHQoAhq7hh79vdews5veoRoUypQiYz2kvc7vKuRz/EpcxAI476AUfAMuUyJWZEw9Ao2p86NPplwhauOuscMoGRTPSgp3iv6kH5E2fjMa+WobqG4RvIN55gqSXXQuAUe2sHQfAKCXKQi8YAPOlg2Xsxfp83rIENS9jzQOMORjk6oVAWMls5wuuFM+Wkv7uJg/AKJyx9+z1IDO0KZ+Ddp666hoAq1USzFwHvi8q7J2+GnUArOVpdwKAq+RLqxmCpAfHVm1pZ9yPr5elm/ceUvfpHgAb2VE2BhgeYIRrOb3ziOOL9LdRUIpV60T4zQzPA2x0KaPk7x4Au2j2UADwnxtP0AcOAvCrFXNQt1r5KVPKxH7LOO1ldipm0eylw3EHNymmDBNl7pqdD/3xAJy89gj1n71Tczj6dJ26MUg1gA3Q2joXTYgERWgFN/hCP34Z3m5Zb4cWAKPYw+uVc/mXceDcdX/oBQuBuHLrHpUc8p9p033WsgS1sAHALAzCaQBGCBCLDUdhjL/erkJrDl6gl39Yb3iOWnZ8qWw22nvmOm11yQOM36wTyT1aOsaGJ26xg9oDrOVpdwKA9by82KJG+EV0aQBgwNq7v25xfUm4BlZ/w3YXF8oYYCixILFQdldM9twOj2xMK/ZfCCpAw+8keQAsa83Q9/MA2EWbP40ADHMgFnB9H5/OKprTAOyUx8rspcOLdEgkANstGsCOjRfIL2uP0AADAO42bZO01JneeUFqq3CmFJreZDbWCS87Si6/VkkfgC/fvEelhoYegBMniEudquWhrwQVBfW8w0b3zJx3qtJzX69SurHs7TUHLtDLE4wBWKukNGS7Dl64Sah0Fp1bVAOwlp21ALjGp0uFqhsiG1cvkF7ZqhY16LRGaGgER8X1QtEfAHCPGaEB4Jd/WKcZHuLm+YcagJGn4bT8JgAYHuB2Kv3tZR/W9FfrrJgnDf3ayYsBdvNesjq3B8BWLScxLhQAPHPTCXr/N+dCIHiwcwuAnagiJWH+oC4A76FNfR7gJXvO0huT7FdakgXgHr9uDqnUkhNxoUObFKW2HADzyhnMA4zM6tIWAPjTFsWpZdns/muEamsTVx8mFI1gKiRWQhgwoR0Antm1MjX7bo2yLiROTe9UkWQBGElzompvKHQCeToPgAN/kmoPsKikLEY4AcA1CqTXTEItlSMVbT4WfT5OfnitrFLVUVRV0MpzT2+MbAiB08fFfKEG4PsPHzlegAka3Cv2BwMwKqqyqqDsOeKGDWXmDAWHyKwjOvbxANjFqxKKG2/W5hP03gznARhftnHixFGs43ScMWIirQiS271U2Coe1jRMmWbRrrPU0YFSo7IAjDhtK3XorZ6zUx8ZfNjIvrPX/bHHDIAv3rhLZYYtMr3MUS2K00scADPYhRZtuE6ZY5kD2QHgGZ0qUqvvfQlv2DZH2WktMFOvBTHDokQ3ADAUQpAI57UnFlADsFZYkhYA1/x0KR0R6C6LbFyrYHpaulfsAda6blF1rcxoeNtdY2wC4IePHtG3S+WUcWTtCgBevv98UAVGPpcAMebT3qwoO6Xj/ULBIY4vOkQTegDsoqFDceP9tfmkK1tl+DqHHE+J7KnI6TjjNuWz0/Tw4y5aXjw1oHD4iz4AXrDzDL3lQKlRvEB+XnOEBv6tHQOMeDCUfv51Q+jO2cmPjM3961HqZAmVOFYWe8wA2GpJaS0AxrWxm5luB4CndKhAr/7oC3eAPBaq7snGiyNmWAS5qPSHimUeAAf+LtUALMqmxwgtAK792TI6dCFYdk70669TKIOmCgsKnuw8dS3kzyOtA4by+RhrALh2PkWlx2kd4IORHmB1CXIUlGIJw0jAnNrRA+Bo8wPjFuIBsItXJRQAPHvLSdeSJVAadUPfuo4DcFSVHgUUjmzmA+B/t5+mLlPtV1qSAWB4EnOkSUpT1x9z8W4LnNpJG6M4RNZUSYQAfP76XSo33IIHuDlCILLRrtPXKH+G5FSg37/+E2BgVLDfv3RXICtnZMSv25QieF2thFBAgoqFxtQvklEpOiILwKVzpKJNgq10APDJK7ej1Ta7kQ1D8Xc1AIuy6XUB+PNlQt1l0drrFs5Ai3aLtX61khdDYQPRMZz87RqdQ2wBYCM7sL9j0/PxY9neRAoA7ztP7SdtCBiEglJME94DYHl7hrqnB8AuWvxpB2AkGu0Z2oicjjNuWSYb/b7xhIuWF08Nz8rIZsWVP87ZdsoRnU28QCatPkyD/tmleT7wDJbIltKRal2yRnPSxohnQ5WzPWeuUcMvVipLYB7gc9fvUPnhi2WX5e8HjeGZm04qxSHU29MMjIoOmK/IQZltdgB43KtlqPOUjcohUaHvu1fKkKjSk2hNWslUzxbPTKc8AA4ymRqAtRJTIbk3e8upgPEYW+fzZXRQUHhEDMAZadFusdRggYzP0L6zwfrNZu87p/pDNeS3iNA8H2M7AEOB58iFm8TkwuPFjUMPTWiHHxjeSNGQVgMwryZTNV86mtKxglO3h+l5QsEhphcVTQZ4AOzihQjFjeemBzhR/Li0d1gjxz3AzUtnoz83heYBz1/e1uWy08fNfQD899ZTjuhsygAwYgzL5kwT0uIfzUpnVQDTica0oXefvkaNvnwCwLDlF4v2S1eck10LA6MiA+YrBQHMNjshEF+1KeW/L14okYXw31pb8+p1oeRp+JFgOa1nwzLT6au3hd5hs+cWk/qrAXjpnnNBIIHzzZU2aVCsL8Ziixna1DIN3nwtHW4zMoQyx7LbB/KAocoXiO0ADFlJAO/hyFCa+HHjmCqeAwBGEpw6oXrhe9X9+RIeANv9Rbg33gNg92xLoQBgp0BOZAYGwE4nwUEWauZmZ+DMzOXD1uInLXwA7FTsNF4gUC8YrOMBhjpA5bzpHNeg1Dt3eGj/UnnNzNiK7zura2UqlSM17Tp1jRp/5QPgF0tlJRRO+G6Zs0klmJuBUeH+8+n2/dACMAp0MF1t3Kefo2Tz2DVS4QuQO1p3KHYDMEJ9oHgh09QArKXMgop82CngG8bWH7Nc2nPbsGgmmr/zjHBZIsCWWb9bfZz8eDVaY2wHYFRvBACzWPIE8eLQ/YfyMRD7FQ9wMACjoirLl2DJtEbXwq2/h4JD3Fq72/N6AOyihUNx4/2z9RS941LFoITx49K+YY0c1wFO90wiQvJUqBu2Fke1KKEc1qnEPhkALpw5hbLN7wYsatkQ3kt8HDnRpr1ZQQF4HoAB2Ejqks3CN7OOqIwB/rhZGH00c7uyXABw/aKZ/CERRuegVVEMHmCUGN4YjbRmjc7F6t/NwKQagBfvPksdfg6WJkQhgfWqQhUY22DMCmHlPdHa9cqNQ1v7+KXbVk/Z8XH4uJwVIgdBbAdgVG+EPBoD4ITx4tK9h8Hl7LUuMgAYO0Tq+/bfd6v5d8s8AHb8J+LYhB4AO2bK4ImeegCOF5f2DXcegF00ue7U2FpEBTI0p4p74AXy06rDNGSOdgwwHrIodywq2uCWLRB3OnfbaUemR2WqOoUzKuV8n/3KVyTCzYZtRcTiFew/n+6FOAkOxT9YURMAcP6MyemT+XukThdbnasOXAjqG5sAGLHibDvZyGgMgB8/fqx44SBT9qZAmlDkWcfYhl+soD1ngktPi46LeO5528UeYCR4IkkxujQnP16Nzim2AzASIAG8hyJjyeH0MfPMgYMIHmA1AP/TrSo9/43vWQkFkh/blTO6FK79PRQc4triXZ7YA2AXDRyKG8+pZC6RGbAdtH94Y0vZ9C6a1fLUiD3GljbajA3H6H9/+jx9dhr0kn9afYSG6gAw4syeL56FxizaZ+dQpsbqvfBNTcQVlthx8qq/SprZOcz0R2hFrrTJKOLoJVPbkewYdpLg+j1bmIbN3a1MBU9c4czJacQ8OQDWKreLa3Hu2t1oVW3MzPUw0zdPumTS0mQMgDv9EqF4x3vUK0D9/9oRdDgtD7AZANb7IETly7PXQr8jpWVXqIbMcejj1ejaxXYAxu7cvQcP/cmUCPszozwDAIYHWK0p/2eXytR8rK+gDpwf49uWNboUrv09FBzi2uJdntgDYBcNHIobD16+t6fZl/OKDQCM2LrRL5VUTnXa+mPUZ5Z9AIYQ+sQ1+gAMKGheJht9umCvi3db4NR46C7YKc56N7sIptkbKgA2uz51fztJcL0aFqRR833XCdD08OFjzdhR9XG1qo1h+x1ycdGp3K5dG2uNz5s+mbQyA+Dr0aPHlKfPPGU6rVLMouRCjEVCJhIzZRpk8RAuJmpRFZKlte5QyrJFJQCnTpqALt+6L3P5NPvYXT80oAG8LJkSykd37suHQOwd1pBW7rsQBMC/dqpIrSML6uCD5puXS9s6TzuDQ8EhdtYXlWM9AHbR+qG48eZtP01dHdCzFZkBGbEHRsQcDzC2tEe38gHw5HVHhd4ms7cDdCAnGQBwzrRJCUU4ZD2JZtcg6l+vSEb6b5czADzixTB6uUIOeloA2I4H+L26BSx76rWqjcUmAEZS0X5JZQbAy827D6jowAXKLawViqAFwI2/XKnoSMs0vbACJ0ryyqwhOvbBDlbu3r4PENZCpYoRHQAYCcoAXgbASRPGM6U8AwBese9CUOjO5A7lqe2P4YpJ+fdOVNwDoeCQqDgvJ47pAbATVtSYIxQ3nlMFHUSngDhMAJ6VggIumtXy1NjSfrtWPiU29tHjx/Tl4v2W52IDEa8KAGbb5qIJs6VOQu0q59LtY3shqgn0Kl+ZPdbwF4vRKxVy0vYTV/1xbWbnCGV/OwD8dq28lsulonLckj3BxRZiEwCb0dQFAPM60lrgVS5Xatpw5HLALYSxz329knaclANgkZYwmzBlkgR09bY9T2Qo728nj4UdLOaBZ/Oa8eLbWYsTHx52PcCQqLx+54E/bj1ZwnimtMe1AJgvqANN9k8jc0/s2Mvq2FBwiNW1RfU4D4BdvAKhuPHcBOC4cYgOjXw2xgCwk9Jg7LZBDNgva/UBGFu77avkpuHzfLGloWha3kgrxx7atBi1rZiTtp24Qi98s9rKFCEdYweAO1XPY1muTqvaGCS4Lt68GwRxITVKiA5mZvse8HL04k2q8ekyZXVaJYlFBUYw9vmvVwXJo2md5of1C9BnC8Ux+M8kik837j4IkYWi12GgYpC/75MqjFidGS++nbOJDh5gFCjaeuKJxJ7Ze2HPUHiAz1Onyb7iOazxBXV49SE79rI6NhQcYnVtUT3OA2AXr0Aobrz5O05T5ynuxADHNAB2I7u6e+18UuoO8ADDUxyqphWPauX4UEZ4rVKuWAHAdq5T3cLiamOIx758876wSIaV6xGdx5gFYF5aT6uUtOjfAcAvfLOKtnHwomUXlLdd0bMWVRu1VNglSYJ4lvSmo/N1kF0bPJgF+80P6G7Giy97HFG/6ADAqNIJOUfWkieKT9dNfAxpATBfUMfzANu5S9wd6wGwi/YNBQC7mQSHF8fhGOQB1kuEcfE2iJKptRQJrCxm8AtF6fXKuWjr8SvU5NuY7QFGrPbU9cesmIm04q5jEwAjq142MQ0Qu/HoJWo+dq1iby0d5VI5UgUVIsHYJt+sCvDeaV20/zUsRM+XyExVPxEDsFnpK0s3RzQdtHtIQyo8IBCAIdu496ycvJyd04I84Nzt9qQa7YZAqD+ukieOr4REyDYA8PJ95+ktlQeYL6ijBcB37j+kxAniyR7Kcr9QcIjlxUXxQA+AXbwAobjxZm0+Qe/N2OrKWcQ0AA6lvJArF8TEpFqatCam8Hcd+HwRJYQDnpKmTwEA925USAH2Qv0DX+wy596mfA6aHm4NgLWUN1CG98rt+xSuKuYgs56nrY9WGIPoPAAvy/aeo3YTNyh/1oqhVnvp0FcB4G9XKx9lRu2jRgDgLFTl4yXCrmbL3xod72n6+87BDfxJiGzdZrz4Vs8VxXU2H7tiWxnHLgCXyZk6oEBNisTx6ZoDAMwX1BEB8BeL9tG3Sw/Q923LUq1CGayaUWpcKDhEaiHRsJMHwC5elFDceE7p2WqZwe4DxkXzmp7ayeIQpg8e4gFa3jQry+j/XBHqUDU3bT52mV78zqdtGd2bGT1a/lxal8tOv244bun0tMrtxiYAFpUt1nu2tP1xPa3c7yseohVCUiJ7qiDQxXMJH2P89rXWcYwA2NLFjiGDtg+qT2GDFgacjRkvvhUzZEieiML71lUA0K40pN33kzrB0mxCJDzo8AB3nhIYAzy0SVHqP3unYh6+ABOzF59Yrq6IaMWmemNCwSFOrzlU83kA7KKlQ3HjOSXnFSsA2IEtNxdvF0enFlXPsnoAFIfoWC3PUwXAVs+1ZsH0tGzveUvDtYqPAIChMqAu52vpINF8ELLqZeJycRp48Xf8OYIW7fbJ9Wkpl6gTldjYZt+tpk3HjD3A2BF4TscDHM1N6urytg6oTyWGBAKwGS++lcVlSpGY1vWpEy0AWC2xlyppArpiQptYC4D5D2kPgK3cJaEZ4wGwi3YOBQD/uOqwbhUyu6f3fdsyQRmudueMqvFOVkeLqnOQPW753Gkc23Lv27gwvWrHFvkAACAASURBVFk9D206dpmaPSUeYFk7OdlPK6YRscHXYgkAi7y1eh/XHX/eQIt2+6TjtJRLRF5lwDMqbaGCnFHr07gQPVc8C1XWCIEwGh+T/765fz0qNfQ//yl2r5Ofluw5Ky0vZ8U2UMVZ0zt6ALC6yqDZxLxdQxooKhB6ieh8BVJmL88DbOXOcX6MB8DO29Q/YygAeNzyg/Txv3KlWl081adiaq0t6qdi8SYXKdJONTmFvzs8aG/VyKvABivvaXWumDwuNoXYaF1HUbyuHgB3mLSBFkdqJ1cvkF6BCfU2NIoVqPV+AcAtxq6Rqq6HDzhcGw+Ag6/Exn51qcywRf4/LHq/Br3/2xZpL76V3zMKnqz+qLZpD7AoVttuCIR6pyxtsoR08eY96dPyAFjaVNGyowfALl6WUADwV4v30+j/xPqWLp7aUzm1k+WBn0oDWFw0sui71AQAP8nYtzhVjB4Wm1RGtC6kSLFBD4DfmLTBXzyEKZfIbEMDfF4at1ZKWg4hPI3DPAAWXYfwvnWo/PDF/j8t+aAGvTdji5S6htUfc/Y0SWhlr9r03bID/rLjMnOJFFrsArA6VyLdMwnpwg15AEYSIT7auuhUY21WOiuNfslXgZQ1zwMsc8Xd7+MBsIs2DgUAf7ZgL32z9ICLZxFzpkYs5kKHygPHHKsYn0mvhgWpa818HgAbmEqv2pixlWNGD3VWvd5ZAV7aTQz3x1xXyZeWVh+4SDLb0AoAj18rFeYDAIYHuNJIsQpEzLC8tbNY36cOVRjxBICXfViT3gUAS6hrWDsiEUrDL+9ZyzQAv1oxB01ZF6jQYheA2T3HziXdM4nowo270qfmAbC0qaJlRw+AXbwsoQDgkfN20/gVh1w8i5gzda60SenIxVsx54RCdCY9GxRUSkhHHLlELcb5NFu9FmwBNyoNPm12NhN6A3h5/adwJYserVKetLT20EWSKZGLsa3Gr5VKLPQAWPsuWtu7dsCHAQqGvDtjc5DuspP3IZ7Dy6IJAKv10tMnT0Tnr8sD8I5ID3BXlQeYL67ieYCdvHucncsDYGftGTBbKAB49MK9UpXIXDxNb+oYboEP6hWgd+rkpw1HLlFLD4A1r/aLpbLSrM0nY/jdoH966qx6vd6A2Nd+Cle2kNFY4qbMNrQZAIaMHxIUK4584umM1ReJO3mUOe//1w7/v6zsVYve/XWzlLqGVRtaBWCUY4fqEd/seoBZ3DmbExJt5xwAYH6NzUplpdGtvBAIq/eLm+M8AHbRuqEA4Ku37tOLY1dTyWypaGYsf/m6eClj9dTv1ytAyA5HIQdsO3tNbAF4emZuit0ArM6qNwJgXgeYeY9ltqHNAPCA54ooMcAeABv/clf9rxb1+HWLVHKh8WziHgyAxy47SJ/Ml0/gdgOA1SXjM6ZIRGevyXuAoaO8Yt8FenvaJk1zeABs9U5xf5wHwC7aOBQAzC+fD6x38bS8qWOZBXrUzU896hbwANjgukPu6M9NJ2LZ3RF4umb0pwGxr05YT6sO+AphsPhhmW1oMzHAqGTYqJgHwDI35pqPaise4A1HjOXlZOYT9WExwNEBgNXSe9AoPnPtjvSpDWlSlNImS6QLwJhsUvtyVLPgk4pvXhKctIld7egBsIvm9QDYReN6U4fMAvD+wgu8/tBFavX9upAd92k7EATv/9gYuwGYxfHKXDtA7CsT1imJb2hMQk3GC2cWgOEB5pO9ZNYXG/us612Huv+6WSq50Kp9nATgQyMaU54+86wuJaj8duaUien0VXkAxoGhMz1inrEnm6/45gGw5Uvm6EAPgB01Z+BkHgC7aFxv6pBZoHvtfEoIRMTRy9TaA2BNu7csk41+j+UArM6q17tJAQQv/7CO1hz0ATCr+CbjhTMjgzYIHmAPgKWeF1CF6D59s1RyoWhCmY8XBsBmNexFIRAHhjeifH3/lTo3Uae6hTP4C7Hg7yjSccokAMuqC3kAbPkyuTbQA2DXTEvkAbCLxvWmDpkFahfKQBsOX6LsaZLSrtPXQnbcp+1ArcpmpxkRx5+2ZTu6XnVWvREAt/l+naL8YLaZAeDBLxSlRsUyUXlO7svs8WJLf+gCvzPNGgAjhvvBo8eGChI50iSlFb1qkRMAvG9YIyrQzw4AZ/SX4sY1RpGOk1dum7rcaojWGuwBsCmzhqSzB8AumtkDYBeN603tWSCaWaBN+ew0PdwD4JX7fTG9Rs1MIpt6LoxtOW6NVKwq4jRRBdIDYKMrQhTRry69PXWTJQ8wVDwePHxkqCBhFYBfq5STflkbqAKxZ2hDKtR/vvGJafRQl+7OljoJnbhsDoDhIFgSWc3Q6IOP/d0LgbB8yRwd6AGwo+YMnMwDYBeN603tWSCaWaBN+Rw0PTxQqD+aLdH15aiz6o2AQLaYhXqewyMbK4okMslaHgDLX/ZN/etRlykbLQEwEiDvPTAGYFYJbvzygzTyX+PYWbZ6EQCjFHGRAQvkT1DVU125EGs7fskcAKsT6bQW43mALV8m1wZ6AOyaab0QCBdN603tWSDaWeDlCjlo2vrYDcCyMICLZyaMQX2xsfWN+GHEpRu1oU2KUgOEQHAlf43GhOLvZiTjQrEeHGNz/3rU2SIAIwHy3sNHtNHgmsDLuup/tcksAL9eKSf9rPIAQ4YsbNBCy+ZRVy6Ed/rYJXPFkmQ/+vDRFidOHGWtngfY8iVzdKAHwI6aM3AyzwPsonG9qT0LRDMLiEq1RrMlur4c2e1gBsCyYQzqhW8dWJ86TNogB8BNi1GDohmjHQCbiZd2/cJFHsAOACMB8s599wBY5AHGfVBisHUAVlcuRILeUZPVQtXFNLSu1f7hjShBvLgeAIfqZpY4jgfAEkay2sUDYKuW88Z5Fnj6LCDKUn/6zsLeiusUykCLJeIhcRSRR0/26OF96lCXqZsMvY2Yb1jTYlTfYQCGI+/xY9nVivvJgpO9o5gbbScEAkB/+95Dw48S5gH+fsVBKfkwdgYiADYjuyeyhNoLjyIdR0wCsKyFEa6RNGF8D4BlDRaCfh4Au2hkD4BdNK43tWeBaGYB0Qs6mi3R9eXULRyYVe/WARPFj0t3HzySmn74i8WofpFMVG74Iqn+Mp3ixY1DDx/ZI+CaBdPTsr2+MtDRpdkBYAD9rbsPXANgOx9MWvZVF27JnS4ZHb5w05XLsXVAfUqZNIEHwK5Y19qkHgBbs5vUKA+ApczkdfIsECMs0K5yLpq05kiMOBerJ1GvSEb6b9dZq8NdGecGAMePG0eR/LLToiMAb+xXl7paVIHA+dy4YwzAkBpb/VFtMusBdgOA1R7gPOmS0SGXAHhD37qEKodoXgywnV+Oc2M9AHbOlkEzeQDsonG9qT0LRDMLeABMSqztgp3RC4BHvBimhECUHeacBzhhvLhKwpedZiZh0M5xzIxFZbU2P6yzpAKB87lx94GhMgcD4B9WHKLh83ZLL+9pB+C1vWtT5pRJPACWvuLud/QA2EUbewDsonG5qfOmT0YHz7uzbRWaM/COEhMs8EaV3PTT6sMx4VQsnwMKTvy744zl8W4MHNksjOCZdhKAzYRgaJ2TmXhpN+yinnNi+3JUq2AGajV+rSUAxvlcu3PfNQB24wMzlB7gFT1rUY60ST0ADsXNLHkMD4AlDWWlmwfA8lazE3uVJ30yOuQBsLyxvZ6uWKBD1dz046rYDcCNwzLRvO3RC4A/bhZGdR0G4CQJ4tHt+w9N30eV86b1l36WrSBm+iAWB0zrWIEq50tnGYAR/33t9n0KP3JJdwUoN7ymdx0y6wF2A4BRvCP88JP1uvkuWfR+DcqX4RkPgC3en24M8wDYDatGzukBsLxx7Tx43Izbkj8Dr+fTYIEXS2WlWZtPurLUjlVz04RYDsDPhmWmudtP27avEyoLbBGfNA8jwFkZB0MgkiaMR7fumQdgXvosusVLT3+zIlXKm9YyANcvkpGu3JIH4AkrD9GwufIhEK4AcK40AcBu5z1kdNP/+241Kpw5hQfARoYK4d89AHbR2B4AyxvXThiDB8Dydo7tPZuVykozXQLgTtXz0PcrDsVqEz9XPDPN2WYfgBPGj6tUFXOiuQHAzySKr8S7mm180QQA48JolDD4a6eKVDGPdQBGuelLt+4FeFRF9mEe4OgAwGodYDvvIaN74Z9uVSksW0oPgI0MFcK/ewDsorE9AJY3rp0Hj53wCfkVej1jggWal85Gf2464cqpvFUjD41f7jwA58/wDO0/d8OVNTs96fMlstA/W0/ZntYqYIoOPKp5cSUEovTQ/2yvi02QPFF8um4BgPnEt+iWMPjbW5UIIQFWY4AR/nLhRvQA4HTPJKILN+4aXm81ACNE4YBLv7U/u1QmVJ5D81QgDC9NSDp4AOyimT0AljeunQePleo98iuLnj1Rw/654llo6Jxd0XOB0XRVLctko983ugPAnWvkpXHLDzp+5lsG1KOSQ5yDN8cXyE3YpGQWmr3FPgCnTpqALt+678hSR7UoroRAOAnAKRLHp2t3zHuA+bjf6JYw+HvnSlQul3UARvgLoHM9F1MruoCZUyamtb3rkJseYHij5+80jkUvmzN1gG6xnfeQ0c06o1NFqpAnrQfARoYK4d89AHbR2B4AyxvXzoPHSv12+ZU96dmrYUEaNX+vlaGOj+laMy+VzJ6KOk3e6Pjc0X3CMa1K0HsztlpaZquy2WlGxHFLY40GdamZl8Yucx6Aj3z8bIDHyGgdUfn3piWz0F8OAHCG5Ino3HVjD57MuX7aojjVcRiAUyVNoMS7mm183G90Sxj8o3MlKmsDgBH+cv569ABg2Y+LUAIwVDbu3n+kxFnz5Zvx+3azhZpD3DwXp+f2ANhpi3LzhfrG47dVXDwtV6a2s80rA8BFs6Sgnaeu2Vo7KvmUGGK97rytg6sGv10rL5XIFr0AeHnPmlTj02VOnqZwrknty1G7iRssHadN+ew0PdwdAMY1+XZp7AZgp2KAUS73xOXblq6xehAAGB7gUg6GQKRJlpAu3bxnen28Z/LZ4plprgPx0qYXoTGAbdFbDYF4oUQWOnf9Dq07pK8CkSlFYlrXx10PsOzHRSgBGOd95todKp4tJW07cdV/FTwAduoONj+PB8DmbSY9wgNgaVOR2wCM2KuNRy/LL0jQc8/QhlSo/3xbczg1GLBVMntqevOXCKemtD1PRL+6jmqtai3IDgC/XCEHTVt/zPa5iiboVisffbP0gONzP00eYKeqmzmZ2PpZyxIEjVonAThtsoR00QIA82Dm1MeCUzfczK6VqXSO1JZjgOH9B+AZAXDGFIlofZ+6roZAyH5cqN8Ldt5DVq+DB8BWLWd/nAfA9m2oOYMHwPLGLZDxGdp31pfog5fVpmOXpWMAs6dJQscv6XuL1F/68it70hNVkvL0mWdlqONjAFsIgehoA4DthJ2ITghlVJ2UmtIy2s9vlKfXfwq3ZNNXK+agKevcAeB3auejr5fEbgBWJxVZukhEVChTctpz5rrV4QHjPm9Zgmo7DMCySVbqE+Ch16mEQRkjycjK/fV2FeWZYtUDDIWV01fv0NpDF3WXxAAYmtlmchjaV8lFE1fLlRqX/bjwAFjm7om5fTwAdvHaegAsb1wegH95ozxdvnWP3v11i9QEKK158oo+ADvxYo5OnjjAFl5WHX627gF2WhJsU/96jiYauQHAbSvmpMnrjkrdV2Y7da+dj76K5QDsxE4L7B6WNSVtP/lkm9jsteD7j37JB8BOJhKmT55IiXc12/gkQYQM/O2AYobMGmQq181+uwqViMYAbEYHWPbjonSOVLTp2BW/CT0PsMzdFHP6eADs4rX0AFjeuAUzJqe9Z30en8kdAMD3qfv0zVITyABweZXgudTEqk4xDYBblMlGfzioiBAqtQJ8IL1m0QP8eqWc9PNadwD43Tr56cvF+63cWrpjotN9Z3RyaqAw6q/1d6dAGvMjaRIlfu0CMO9FhRfz7DXzAMwXYnFKMUPGxjKyckyntvX3aw3DGETHxAc1QiDWHHTHA2wGgGU/LjwAlrl7Ym4fD4BdvLahBuDigxZYkuZx0QTSU/NbngDgq7fvU7dp9gEYUPJKxRz0zrTNhvI8RouNTiACb2PJHKnojUnWPcBOKyKEKknQDgCbeYka3Q/qv79XtwCNWbTP7DDD/tHpvjNaLOT5NnMeNaP+or+PbBZGs7ectARhovkAwLULZrSdwBo3DtGjx74jQMoL2/1mW7PSWWnmJl8lQqcUM2TWIKNaMeedqlQsa0qyCsCA+7MSAAyFj/C+demnVYdpiAkZRzMhELK2Vd+v/E6kjF2d6OPFADthRWtzeABszW5So0INwBuPXqLmY9dKrc3pTnaztnkAntKhggLAb0/bJLVMvZfRxHblqFahDJYf6vwCohOIdK+Tn0plT0XtJ1lTQ8B5OZ0QtnVg/QB5H6mLZ6GTHQB+o0pu+mn1YQtHNR7yfr0CNPq/2A3ACMtJ90xCJeHUqo7v121KKVrNK/adDzK6lQIUX7QqqXiA7Sq48AAss+skumP4QixOhyDp3aEyMctzu1elolnsATBUIFYf0PcAI3xkg8sALFvy3ANg4+daTO7hAbCLVzfUAIxTqTd6eZRUjbKbZKYG4Gt37lPXqXIAzORlRJeSAXCb79cZJmcY3QoxDYCdTgjbPqg+hQ1yXyYOOwRtf7SWBNexam6asModAP6gXgH6PJYDcIlsKQnJVA8ePab8ff81+kkJ//7Ny6Xor80nadHuc0F/t1L05svWJalmAfsAHC9uHHoY6QK2CsB8IRY3qxKqDSejqzyvezUqkiWFZWcBvK7nb9wVAjA0cNtHSheGAoBlPy4AwJDIZImxngfY0k/2qR3kAbCLly4qALjxlytp12l7erdWTGI3Zq9w5hS0O3LdUztWoOt37lPnKQ4AcPtyivfn5R/WGcamGZ03APjPjSfon22nCDF1c6JQwxOhHQiBYC8Vo7WL/u50POyOwQ2o2MAFVpZiaowdAH6zWm76YaU7ANyzQUH6dIHzhVKi04eX0YWCxunf3aoq3azqkn/3Smk6eO6G8GPCijyaAsDwAA+293EWP24cBezRZJRnRLZ6qWw2+i3CV4nQ6Rh8vWuTLGE8unnvoe7lm9+jGhXK5A4A87tDDIAnrj5Mg/+Rr2RpZveGDzXRO2nsWPRuVIhafb9O6cbnohjd60793QuBcMqS5ufxANi8zaRHRAUAN/pypR8kpRfqQEe7yS/BAPyAOk+Rq3Kml5ACzwMA+JUJ6wy35ozMwD+oRszbTd+vOGQ0xLW/96ibX1GBsFoQAgtzOh42VACMEJlXf1xvybZvVc9D4126bh4A+9Qb/nnHHgCPfaU01S6cgQr2C9bctpKl/1WbUlSjQHrbAJwgXhy6/9AHwDLFd4wA2OkYfEs/CG7Qgh7VqWCm5JY9wEjqu3jjHq06cCFoKfzuUCgAWNa7jmdon8aF6aXxvtBBJ+X3ZK+HB8CylnK+nwfAztvUP2NsAmA8SLYcfyInY9asRTKn8Huup3WsQDfuPpAu86u3vccA+NUJ64UPZjPr5B9UH/+7h8Ytd77ql+x6AMClcqS2rIeL43SompugxelUexoAuHONvK5dN7dKZT9NHuBiWVPQnHeqKbeUVQ/wuFdLU8NimanjzxG0aPfZgNvTCqAAgFGgo7jN8Bw+/jhX2qR05OIt0z8dPjbV6d+f3mJkdIAXvledCmS0DsBQXkB1PBEA888GxCOjaI6bHmA+1ETPLpB96/dsYWo5zgNg0zdzDBgQIwH40qVL1KdPH5ozZw5du3aNihcvTh9//DFVrerzTGzZsoXeffddioiIoLRp01L37t3pww8/9F/OR48e0eDBg2nChAl0+fJlZdx3331H+fLlM3XJYxMA40Gy1UEAxnadbJUzPQBG1TBsf7b9cT2t3B/smTBzQXkAHjV/D323LOoAGIoDCIGwWhAC592peh5Hvdg7BzegoiEIgbDjAe5SMy+Ndem6fdSoEOHDyMmGymHfvVLGMkw6uRaZuRBPObe7XQAuQw2LZVJ+///tCgRg/kNZZj3og6S6Gg4AMA/fudMlo8MXbsouwd8PJZkZ1OMj9otFzsvmiRbFxy9rLXrR+9UpXwbrAAwPMABY9Jzlnw0MgCetPkyDXAqB4ENNPAA+QdmzZ6fjx49TtmzZTN+zMXlAjATg+vXr07lz5+ibb76h9OnT07fffqvA7KZNm5T/LlSoEDVt2pTef/99WrduHXXt2lUB3Pbt2yvXGvCL/544cSJlzZqVevXqRYcOHaKdO3dSwoQJpe+H6ADAn7YoTj3/2Ca9ZqsdkfyylatvbnYevDh3nvLFLk97swLdvvdQusiDnii9WwD82YK9rpS9lbUbFAfgdbeqh4vjOO0NDRUAI0b8lQnWQiDcBGDEEo50EICx3T/v3WqUIF5cUwBsdzdG9h4U9QOgYs1oVj3A49uWoQZFxQCMGONtJp8zSKqrXsC+B7h+kYy0MBLI86RPRofOmwfgSnnS+pNxkXSG0CyrahlmrlPCeHHp3sNHukMWvV+DUB3Sqgwa4m5RHEQEwLuGNKAiA3z5AVAJiehXj9wEYNnwEry3+j9XhFp4HmAzt1OM6RvjAPjAgQOUP39+Wr16NVWuXFm5UI8fP6YCBQpQmzZtKEmSJAoQHzlyhOLHj6/8Hd7imTNn0p49e+jevXuULl06GjVqFHXu3Fn5+5UrVyhLliz0008/UevWraUvflQAsDoJbtmHNanmZ8uk12y1o5UXE38sNQDfuf9QWuNWT+IHZXMR/wdP6XKBrJKZ8+U9wKMX7nWl6pfseqA4AA+wVTUEHOftWnnp26XOebGfBgDuWjOva577vo0L0/B5u2UvoWE/xDF+/lIJ0zDphBav4eI0OsBLOr9HddNr5qf7vm0Zql80E3X6JcIPnOzvVuD+25dLU/UC6WwrlPDb6lbLiPMV7sL71qHkiRJQ4QHBsc5W7a81TqYS3JIPalCe9NYBGPZBIQwRAO8e0tB/nmmTJaSN/d0F4NblstOvG44bmhEAPOD5In75UCshNoYHMejgxQDbtaD18TEOgAGra9asoTp16lCiRIn8lkH4QsOGDRVPbqpUqWjatGn+vy1atIjq1atHZ8+eVcC4QoUKtHfvXgWaWUMYBEIp4BmWbVEBwHwSHLa4EeBv1RMje57oh9i/HSetq0/w46e/WZHuPHgorXAgA8DtJobTsr3BuqJmzpF/UI35b58rVb9k1wMARgyw1WQwHAfllL92sHSvHQDmNVaNbIAY8ZcFHmAZAHUTgBFLOGxu1AMwn1BqZEun/+4EAP/wWlmqVySjEICtqM04BcC8V9GqXBYfOrFnaENKnCBeSJ7PSRLEo9v39VUgln5Yk7A+qx5gQCdK0osAGOdaqL8P9BkA/7zmCA38e6f0LWgmZrpN+Rw0PfyY4dw+AC5KzceuUfp6AGxoshjVIcYBsOjq/P777/TSSy/R33//TX379qVGjRrRJ5984u+6a9cuKlq0KG3YsIGOHTtGzZs3p1u3bineYtYwHv+GuGKthnhj/I+106dPU/ny5UMae8MDMAO2pxGA7z54KK1wgC21CzfuCS8L8wC3nxhOSx0E4K8W79cteuB0eIH65D6sjxAIewDsdOlefptTdDHeqpGHxi8XK2ckTRiPbhnINLE5tQAYW5lDDSpLOe315s/TaQCGTNZnLc17gPk401C/rZwA4AmvlaW6GgBcPncaCj98ydRpQVatWn77HuA25bPT9HCfV9GqXFbDoplo/s4zyhyHRzamOHHihASAZUohL+9Zk3KmtQ7AKKxz4vJtYQETHoChF/xF61LkJgDLFvnBzuVADoCj4uPR8wCb+jk72jnGAzBCIeD5hUf4r7/+UhLZXn75ZRoyZIjfkPAK582bl1auXKl4gNu2bUsPHz6kuHHj+vu89tprdOrUKYK3WKsNGjRIiR9Wt1AGn/MhEKEEYD6Ewcodym8N/tqpIt178Eg6vhUehYs3xQCMqmGI/3tj0gZasidYWN/MWvkH1TdL9tNnC7WrfrkNwJDcwnaw1VhYnLfTpXuNABj2g9yQCGBQqhXg9sdGn0aqXtMC4AHPFTEsrcoDsJWqYnrrkjm+0bnxf7cKwEObFKX+s+U9a2bWZNQXYLjgPXshED++XpbqFM5Ib02OoAU7A5Pg+Bhao7Wwv0NWrVqB9LY1qnmosuopRGGeTccuKyWHEeeMFgoHRfLE8en6nQe6JlvRsxblSJvUsgf4tUo5FWUMUQW/vcMaKrHbi3adpa4181HKpAlcBeBXKuSgqeuNPcAA4EEvFKVm3/k8wB4Ay/6qYka/GA3As2fPVmC3YsWKiucWHt2wsDBq3Lix0AOMJDnAcIsWLYQe4Lt37xLm1Gqx2QPMZ2fjhdNFsoobsyUfQwwAfvTosXCbW2R7vZciA+AOkzbQYgcB+NulB3SLHoQCgFEKWRQKIPtoghdZD+Jl52H97AAwvPhjWpWUimlGkuTLPwQnwQ16vohhVjkPwE7Llskc34xNLQNw02LU/68dZg7lWF+EBix8r4YtsPupXVmqXUgMwFXzpTMtZ+gUAPOVE62CEoq4VMufPsDeoQDglEkSKOXl9drKXrUoexrrANy+Si4lMVCUa7FvWCNKGP+JQwnr0PIAa+1MmaniKFvlEo6XIU2K0oseADv2DHiaJoqxAAwFCEidNWvWjKZMmeKPBwb8Qvps8uTJ/uskigFGMh28wqwhBrhEiRJKAp1si+oY4FB6gPkXwqr/1aKqnyyVNZPSj1eRmNGpIkFuvnVkdR6tieoWzkDv1ytIY5cfpH+2nhJ2Yy8cp2XQYgIAO124gU90EV0M3I+lh/6nSCWpG4qZfNm6lOE1xzgtAB78QlHDmMJutfL51TsQsgDNUrux4excZI5v5kdhFYCHNS1G/aIIgKFc8d/79gCYlS/vA3ZGZAAAIABJREFUPHmjP1yA2Q0JrWaTWRECgXF2Jfrg4fxl7VFlKVZ3vNgHOX8fhAKA0yRLKPzd8evAcztbausADEDdf+6G8PrsH95IUTPh2y9rj9AAwU6FljycGQDmr5Xebw4APLRpMWr67Wqlm9UPGzO/a3VfLwTCjvXsjY2RADx27FhF2gz6vmPGjAkIZYAe8Lhx4+jgwYMUL148xXq9e/emWbNmKSoQ8PJmyJCBRo8eTR06dFD+/rSqQIQSgPktwbW9a1OlkUtM3ZlqAIZuJZOm0Zqoe538BDmwd6ZvNgRgJ14y/IMqqgEY3kuEQIg8obKGjwoA1roOWVImpq9fLuXPxtY7ByRJtvnBV7qUbzJb/zwAI2Z45f7zjgEwPEmiF7rs9VD3Q1b9pxZigKMSgPOmT0aLP6ipnIrV3xyTLhQBcO1CGUyHMiEJDoUw7AIwXzqcD9kyc32jCoD18iTY+td8VJuypEpiOQQCVRb3nr0u/D0dHNGY8EyXAWCt0CwzACxb5h3J18OahvkB2IrOtJnrL+rrAbBdC1ofH+MAeN++fVSsWDF67rnnghQbEAIBwIUO8AsvvKDo+4aHh1OXLl0UKH799dcVSyJRbvz48YrsWa5cuahnz55KbPD27dujvQ5wVMUA80kh6/vUoQojFpu6K/lCGr+9VYlQdpRtS2lNhK2y9+oVoG7TNtGcbaeF3ZgHOE/vufTIV8XUcuMfVN8tO0Cj5u/VnMvtEAgAcKnsqYUgKHuC0QmAs6VOQgCVJpGeGCsALAN+vPIFYnZXOAjA8CQ5GXpgFYCHv1iMkPX//m9bZW8Fx/pBH3eJTQBmiasiALaS4OcUAPOlw63qnrNz4w3+a/gxJaE2ddKECkDKttRJExCegTLFJPS00tnx4LjInNI6AENje/fpa0IAPjSiMcWVBGAo3Hz+X3B+hRkAli3zDgAe3jTM/9yx6tmXvWYeANuxlPNjYxwAjxgxQgFYUQPgTpo0SVF7gHd48+bNlDlzZvrggw+oW7du/iFIgIM2MAph3L59m6pXr66EPgCGzbSoCIGIKgDG1ie2v9Cgb1l+uD0Axgv8+W9W6ZobW2U96hagt6dtorkaAIyqYVXzp6N8febRA5sEbMYDrKd4YOYe0ur7v4aFCHqvRmEiesdyGoD5TG+tB72WVzB7miQ07tUy9OxX+tcc82p5gEe8GEZ9Zm3XNS8PwAOfL6Js1zoVAiED4PzieG+0aNGoZjWqhXkVCNgBCVs/rDgUpEsMPVgkX/2tETJk997Mky4ZLfnQngeYeUm7TNlI/+7wKSaw1qBoxqDEOKM1A4BrFUrvL8Rg1F/r729UyU0/rfaVDrda+VIEwJgPWvVD5uyiiauPSC8va6okhOeMzK5DphSJFY1evbaudx3KlDKxZQ8wfls7Tl4Vqu0wxQv++JPXHhEma2o9l9wAYADvyGZh9MI3vhAID4Clb78Y0THGAXB0uipRAcBRJYPGC8OjznvZYdpqGaJrxAvc/965EiVLGJ8af7VSDoCnbqK528UeYAbABfr9qyhL2Gk8ABtVgnMbgFF2FzazA8BOJ4HZAeAcaZISsv/rjVlheImQJAn1i4eqDxq8yHrP1Afg7rXz+QuYIGltmYMADM9r31nyyWduA/C45QeDSjMDgFFmePYWccy8ofENOkBHFnqyaFZDINiujQiAGxXLFATFRmt2CoB5ALNabISFd4jWPOSfXX7ANjon/B0A3LlGHinFj8wpE9Ppq/oAHN6nDmVIkZjafL/OX61OZh2sD7zR205cEQKwaJvfLAC/WS03/bDS9wFi1PiPFb2+HgAbWTJm/90DYBevb2wCYMT+HYwsDbqpfz0l2clMUwMwspbrG8AQiwF+WwKAC/efbygEb7Re/iE+6O+dNGmNtrfG7RAIADBUIFoZJArqnRO8yHgB2CmnzM9vB4Bzpk1Kk9qXp1oSVQsBwNjSrfP58oDT+7hZGH1kAoCRtLZ07znHPMAyHmh+wUYAjMILn7Qobhom8SGAQgCiOHUAcOOwzDRr80mj293S33OlTUrLetYyvWb+YOyjtevUjTRve6AH+NnimTV3e7QWjFLIdQpltF1xDYWFvl/h07EunSMVbTp2Rfn/ceLAgytnLj0Ahob1j6vkAM8sAMt4gLFzlyG5dQ8w8jG2HL8ijNE2A8BaH+ZmAFi2aAaefx83K+7fbbRb0EnuLgjs5cUAW7GaM2M8AHbGjsJZogKAoyoEArF/kMBB2zqgPpUYstCUZXmPyh+dK1GqpAmp7uhAwFFPiC23D+oXJNGLkvWd2rECVcmXjooOmE83JQstaC2cf1D1+mMr/RahrVnrtge4NwA4R2pFV9dqAwAjbs+qp059XDsADHCa3KECVRtlrB4ClZAKedIGrXtU8+LU689tuubgPcBIWlu8+5w/ax1x5/cfSpKM4CgyAM4PMyrKYRWAsY7W5XPQ14v3B8VSAoCfDctMM10CYHzILLcJwOw3K/pdP18ii2bCq9aF/7pNKUVj2m7JYf43XTZnaoo4etk0AE9sX45qFcwgXOqwObtoggkARtw8Es9kNJ+hsnL22l3d3wZ27lBVU6YSnKgkNUIXNh69LA/A644KY+bxcf/xv3uC1moGgGXDJZD0NqpFcXrua1/olQfAVt8mT+c4D4BdvG5RAcBRFQKB2L9DF3wAvG1QfSo+yB4A40Fc08AbCA/ahw0K0q5T1zTDJdjLNGzQAkMheKNbgQdgvbhjzOM2APdp7APgluP0AVgvaQgvGniqowMAY+scsb0VRxrHjmsCcIvi1OsPAwCuk59QxQ8NqhGLOACWgQS9e0QGwEMJwF8u2k9jFgUmE7kNwAhlWdHLngcYhU4q50sn/LBtUjKL6fANJwAYXt63quclhJWglc+VhsKP+CrSmfEA6wHw8Lm7pLf4cVwFgGvkDYBIKC0gnlid7oAdk1RJEvjzNET38cZ+dSmtJACLvLR4nqDIjajgkNADrAHA+LgfaROAZWFZDcBeDLDRWzBm/d0DYBevZ2wCYL7G/Y7BDUxXXeK3FP/sUknZijPyBnatmZd6NSykXMFz1+7Qqz+up31nfYl4rLGXaYnBCw2F4I1uBf4h3m5iuO7WeXQA4ITx4lKjMO14TycAGNvC09cfU0rXYuu9UP/5mmaE/bRgGx9QiP0uIxE7rgXAKBv84e/6ygcIm/EDcNNitHj3Wf91hBTbKYM4SV0AlgBwfjzu3++W+YBK1Kx6gD9pHkatyuVQlAXYubL5FQAunplmbnInBAJQtup/tZXDWf2wgs5z5bzpSBTa9GKprKbDNwDA9Ypk1L03jX77gNwuNZ5crwq509D6yJLMpgC4XTmqVUjsATYLwEgc7VQ9EIAROnbz7oOghF84FFZ/VIumrjumWS1xc/96lDpZQikPsAiAEQO/ZPc5YcEhMwCslUwqC7W4lny4it61he7vp5wH2JNBM/olxKy/ewDs4vWMCgCOqhAIbGGjDCaaUUUwkcnL5EytbJ+h/dmlMmVJldhQSxjb99jGZ+25r1fSjpPXhABcashCunxLvxKS0a3AP8ShNjBNp9Sm2wDct3FhRQVCTytZnfEP7xyqQbGKUPC0wINkFVRgL2y3NyudTanydOf+Q1sAPOvtKoQPFaMGmbzyudMErfvzliXoAwMA5qtM4UW7iANgJBWdvHLb6PCaf8eLtKeBB5ofjPt3rA4Aty6XnT5ubj4GGJ7ol8plJ1GiJu6J54pnoT83GZectmII2HD1R/YAGDsBlfKmdQyAv2pTiuo7AMBv13xSRKVinjS07pAFD7DLAAz7n7t+JyiUBzrAEf3qKeof3advFl7aLQPqKaFnMklwagCGA2N6p4rUdcomaQCesu5oUMEWNk/BfsEf0lqeYdHJyD5/AcCftSzuV5/xCmFY+dU/vWM8AHbx2sUmAEbs39FIADaKBRWZnI+pAwDDu2EkpYaHXO9Ghf3TPf/1Ktp+8qoQgMsM/Y8uCiqQqdfyZeuSihdRFAvKAzCqmaFf2VyphXrAsg9gq7cfqpgBgJuP1Q6BgJRcncIZ/BrJiM3Gtu2bv0Qoh0UYBTxIdgCYeRsx390HD0n04mLnqOsBTp+M5rxTVUqqSguAR79UwlD7lgdgeKwW7jzrjwHGPXf8knUAlgHwUALwJ/P3BAF24gQ+AP5jYyAAO1UW2wkARpJjxTzOATB+05B+09udMPodwsv7Tq0nCiKV86alNQcvKsPMeIBZmWfR8cx6gPFBC08nX/UPyci4h+89DFS8QSU4JCfP3nKS3v11i/B0tw6sT/AgywAwHA+4v/jfNv5/x583KGFF6ibyAIsAWFS8CTsl+DDFs14mRArHlk1CRgGn0S+V9IfQeQBs9EuIWX/3ANjF6xndANjMg9qsWfAwPnbJ5wHeO6yhLggZAfDMrpUpV9pkhkoSePj3aWwAwJHbqZBlu3BDPwkE6/qiVUkljlT9AsHfRA/xa3fuC+OdowMAJ0sYj2oWyuDPmkds9vpDzgIwEkheKptduaR2AXj+u9UJcnVGDaES5XIFe4Bx7XrMEL/c2Zx8mVU1APMfcUZrEP1dBsDNAHCb8tlpZDMLHuDIazJy3m4aH6lawI4LDzASydQAvLJXLcOQIxmbIIxkTe86SlerH1YsxKXjzxGKh55vVkIgnALg7rXz05eR8eNV8qWl1Qd8AIz6DrIS404CMO7XN6sFAjAA7uD5G0GSj6mSJqAtA+rTX5tPav5G8HxIkTj6ATB77sKzbeQUYfeK0e4K6wcAHtOqJCF3Bo2vaCpzvzvRx1OBcMKK1ubwANia3aRGxSYARuzfics+79mB4Y0oX19jkOGNWC5XatpwxBcCAQDOm+4ZQyUJZPr2e66Ifxo9D3D54Yvo3HVjAB7TqgT974/t0gB84+4DYbxzaAA4NTUfu0bzXkyeKD5VL5Der5GM2Ox1By9Sx0gPMMIo3qyexzKo4MDY9m/pBACnQwndGpS79zzD35YWAAN0tLxbIgCGbNmCnWf8HmA+jt1wEYIOuHfemyFffc3oJW0VgNk1EakKAICRSKZWMFn1v1pU9RNjBQ4ju0Bvdq1NAGbX99UJ62nVgQvRBoB71CngTyqslj8drdzvW5sRAPOhNazcsMiOI+bt9susGdkZfxcBMCrU7Tlzne6qNM/h2YWHd+amE5q7JHg+PJMovpQHWK3UwCBO9NGCtYogb+r6o0G62SIPsBUANoqvZ/aFFjgUZTwAlrnjYl4fD4BdvKbRDYCNHtR2TME/5FH2Mk8fY5DRAuBZXStT/ozJDRPpoPXY3wCAmQpEhRGLDGWAsB548aAlKyqaIXqII+Gk6MAFQaZzG4Bx3giBaPadNgCnSBxfqYLHtFQRm73mgLMAjG3/5mWyKecPm+l5cGE/LfF7VkBBxmsImbyyAg8wkp3e0Yhv1ALghbvO+JPgeCk/K78FGQDn5zXaprULwIP/2RlUWQwhEE1KZKUZEccDThFxu1U+XmLltAPGQG92XR97HmB2ffFxx/IC2EGalsxCf5ks4oHrguIfeuE5RieOZyeqTiKxEE0GgPExgB0qfIQeUKpkPqYyOdNoHkoGgPkSv8i7wAcsX3wFjgSEgd25HxgCkTxxfNo+qAH9ufGEZpz8zsENKFk0BuDz1+9SueFyBZaMJAbZRTg4ojHtP3edGn7h8wAXzJjcVDlqo/tG5u+eB1jGSu708QDYHbsqs8ZWAEbZy65TNykVm5B8ceHGPUMr87JCAGBs5RnF7LWvkosGPl/UP7fIA8xE9WU9wAA6JLipPSg4iOhBdfveQ6G+aCgAGAkjL+oAMLY9K+ZOS/N3+ooJ7B7SkFYfuOD3ACOOuGO1QA8wqkGVH2EsRcaMjg8GJMHJAvA/W08JIdUJAEbBg27TxAk+bL3v1X3ixYNqxfwdTzzAfDVDwxtW0MEMADcrlVWpusVktUTHswrAUMNoUSYbnbpymyqroFYLgOGZVPe1YoMMyRNReN+6ylCZjxnRMaACA1DkE3qjAwCj0MNnC30AXKNAev/OgZZjAffTovdrSJtRJgaYB2D8ZjpWyx0AwIhN3nzsSlDRH+wGbR/cgH6POK6ZqIkP5KQJ5TzA6oQ09mxEfsF/uwLDVrSenWY9wGYA2KjIDNYEb/nsblVp75nr1OALXwVKD4Clb9cY0dEDYBcvY3QDYGhEqsvHOnX6vIQUHoaPHj1W1AY+mrmNFuwMfiCqj8sD8F9vV1EqlOU3CKPAy2DQC08AWKQCwcqqwnOAB6hRAzz0NQHAWsoHPAAb2R1yZaKYY721Doj0AKsBmBdyT5ssoRIrywAYyYkA4A4/+5LgRACsl6gmWg/ibpuWyioNwHO2nRJCKqsgJgNNWh5glLyFPrNe4wEYChb4SFu+77wyJH+GZ3R1Uo3uHRkPNOZAjCtULD6Zv9cAgHMo0nJmYZL3yqtj3wHATUtmpV83BHqA1/aubai6YnT++Dv0ZjfYBuDKBFWY2p8t82uL2wFg3KOQA7TrAUbRnU8X7FWWUqtgen/JX63ftxsAjI/+iat9FShFAFyzYHolzv/2/YcBlwv5ADuHNKTfIo5ramXj+ZA4QTypEAgnABgqOnA28E0vBMJpAIYDYWbXKgEAXCDjM0FSmjL3vZ0+ngfYjvXsjfUA2J79dEfHJgDmS23yP2gtj0AQAOdOo4ioowGAi2dNaRhG8XqlnDS4STH/VLU/X+avRsf+8Zc3yitbkLJJcIif7D97R9AWIuYTPai0Er94AE6aMB7d0qhCF5Y1JUHVwGyVqoHPIwQiNTX9dnWAKXkAhvZnmZyp/B8g+4Y1opX7zzsKwPB6NinpA+D7Dx/pfrTAfloAzCqIyQAw8xCq+459pTR1maoPwPDisW1sNQDb9f7IeKARf/tl61KKvVDtSt8DLA/AfIIrD8Dq3wRigJuVzkrTw90BYNxzqChmFtr5mxg5AKVzpKZKIxfTaZUus5UQCAAwyj/LJFhqPdDh5e3Z4InyQZ1CGfxyX04BsEwIBA/A0M7GDg4PkdA7xkeu+nkDRZjdQxvS0j3nqP2kDcLTdAKAO/0SQQslPcBmARhJzHiOyzRUCf16yQHdrnC6/Na5Eu07e53qj/F5gPmP4JfKZqMV+y7QmWt3ZA5puY8HwJZNZ3ugB8C2Tag9gQfAkMUJzuQWWQweMQbAs6EHmz2V4RZq24o5aWjTJwBcdth/QeEWP79RXtmuVP9NK+EJqgYDZ+8M8qBoAbAW9PEADA1S0UsBcwJYIftm1jsFAAYkNNEBYGxHo2QpOzaSE+HtZB5gxBEjjpoHSbMeYGisvlAiizQAz912WuilFQFwo2KZFEF/9ZaqFgCPe7U0dZ4iD8CQcEN8NPMA280Al/FAA+C+iATgkf/upvHLD2k+QF6ukIOQqCcDk/w2PB+WUufzZXQwskQ55vEBcDaaHn4s4LjreteRlpjSe2QyvVmZNWvNgxAofNyJtLutVIJzCoAh/cUqlNUtnMEv9xVKAH6jSm76afVhxXQiAEaRk2V7zgWVfYfnf8/QRkqVuAGzd9LkdUeDzI8PZOh5y8igQUJxxLxgGbSnCYCh5fxrp0q0/+x1qicAYCh2lMqemkoN/c9FShA7Vpw8YFRwiJPrd3MuD4BdtG5U3Hh6hTDix40TVCHIqdPny8jyX7QdJm0QCqOrj8tXVpIF4Fcq5KDhkYCgfL33nRek3zupfTmqWTADqXWAF71fneqO9n318w1FBAb+LQ/ADx4+Eipe8ACsJzGFUI9ZXauY9k4he7l0ztT0wjfaHmB45YtlTemXkkJy4rJ95+iNSb4QCCcAGNv+kNVCk/EAz9t+WokPR+OhjZXQVcM4wlGmqgqO4IMBW+RqD/D4tmXorckbdW/pD+oVoM8jE5nUAGy3CpSMB9otAOYhDGoUL5byxWXXHb08MgHLZxaAEABYXcRlfZ86VMFE7LeWkRF2s7F/PeXPMt580TzYAcKHW7GBCwgqK3yzAsCwB7SPjUKq9G4c3KvQoR0+b7fSDZ5W9mEWZQCcPhl1rBroAYZM3MKdZ4IAGGALwGUNCY/qoi/7hzeiBPECARiOiK3HrwSZRguA35ocIQx5E3k53fQAd6/9RLNZ67pWzZeOpnSsQAfOXfe/C/g8gIntyin3oQfAuo/Up/qPHgC7ePmiAoB/DT+mqBhgS/TwyGcDXkRuAjC8jUxmjH/YtZ8Y7o+V403Ne3zx7zwA/92tChXPZuwB5j1kWi/cie3LUa2CGYK8SUhOARyoGwB40D87hSELooc4Yp1Fihc8AKMARYkh4gpngK7Z3aqYfjkPfqGoAoHPfb0q4BQQUsGKgUCZo3Dm5H5PFZITl+59AsCII35D0gOcIF4cYXEQeD3hdUIzC8D8nE4A8Pdty1AnEwCMaz13+2m/B5gPH7HyWBj3ahnqPEUfwAMAWKDTyx+X/8Azgkn+t60HwPAAI0FO/VFhNvlRyz6s4IIdAGYfwAX7/RuUjGoFgOERx0eaHQAG5CLuddhcHwDzuzpRBcAoeoEQiN4zn8TRonw27mn1hwN+a/uHN9YFYCgi4Fx4DzCSiFFiXt0gocg+BvA39mw0A8DYheDXzs+j/hDG3y7euCtVKh19+ZLnWvcqdgaxQ8gDMGzKdkwAwFDaKTnE8wBbeR4+DWM8AHbxKkUFAAPIoJ2J7VxkmfMvIjcBGMkvLMmMB8XXfwr3A4YeAPOlRWUBmM+S1wTgyNKjJYcspCtcKWQtAIZXcMg/u4I8KKxssPp2wZaiSLv2rep5/EUI9AAYahf/dKtiWjd5SJOiVCVfOqrzeSDE8wAMbWbEtS7e46vMhOuyZM9ZvwfYDADDc6iWVsKc8Ho2CvMBsJY3nNkMx/93+2l/nC4/J6qwrexVOygcw4wHeMJrZf0KF1o/a94DrAZg3nZWHgsyHmi3AJj/mOATE+uNXh6Q2AcAblk2G01ZFxgCEd63jnSRAT3bsIILdgCY/f7z9J4bVGAiFACM54o6RhpgCEmzoXN2KaffsGgmf3KpFgAjnvQ/EyoQosIlalsjZOnHVb4QCBEAv1oxB83efIquqzznePYfGKEPwNghiqsCYGgytxwXXG1SC4A7T97otwu/dpHzIKoBGHHcP7Yrp+yQMGeIGoARZqblvND7HSAp/OHjx1LSm14MsJWnrTNjPAB2xo7CWaICgEULYV/TbgIwL3fG/6Db/rjeLxjPr433+OLfK+VJS2sP+Sor/dOtKoVlS2m4hdq6XHb6uLmvUhYa4kvfmb5J8R5vidy2Y5WXSgxeqKhSsIaiC2p4xN+QGIWX3M3IpDUkeSFMIW/6ZygO3OqCJvLOBQDwwPqE44uaLAAj5GHQP76XLxoAuG7hjEHSVTzEwauKLb0lHADzSTBmAFgrkQ9xtw2LWQNgZKYzOwPWV/1PDoBZkpTa7rjWLLxD62fNl/xFvDfif3HfoEEWaeuJwFLaZh4PTgIwChLgIy1TysCPWK318EoifGJi/THLA7La8dHRskz2oBhQKDfIaqzq2YQVXEAfI6+11jx6v3+rAIw4ddniPD0bPFF7YGsE5EI1ZXDkb7BxWCa/vrbWc9VtAMZvG8WAsOPHGmKEIXWmBmCEcByK3BFE36qfLPEXLmJjsUOEZxzvAcbHiDrMCv1lABi7W4cu3FByMMa3LRt0uZ0EYIA//1HHlzwX3WdICpzbvSrlSf+MUjmPvQt4LXDsHpbObg2AsXuD99khLv5e6373ANjMU9bZvh4AO2vPgNliEwBj6/PSTZ/eL/+D5qs5IeaUeVB4jy/GQL9yzcFAAIYnliV8iC4TsnRHtSgR8CcUpkAyHct0/vH1slSncEYFQHkAXvJBDaqt8p5iou9eKa3IBLEtRGxrQ0RfrwkBuEYef4ITKjBpATA89XO7V6O8BoVD1GV2hzYpqmzrqrfneACGtFiudMn8hR5wXXgARiJd+ypySXA8rPK2APQ1KOqzj4wHeP6O0/5ENRTquHbHF+PpBABjy1Irw52tmQdgKH6gUAg8XPiYwEcT+3Cy8liQCcGQ9QBDszlJwnj+ZRjBJGI8WfEWHoAbjFkRIOyvBcBQbpDNsNezDa7ptkENlC5Ga9aaZ847VZXYddH4qARgfDAiPwANYT/swymUHmAA74RID7AIgBF6hdja65G/K97G/HO52qgldPySr3Ina+zvPAAv/bAm1fpsWdClwscACwfhn/ldpmxUpAXR8HzCjhpgU+Q8YOF6ojWIQiDwfimtkZAGh0OqpAnpk/m+xDw9AEZYAwokQfMY7dD5G/53ARILD124qfy7AsDwAGs4L/R+Bx4AW3mChn6MB8Au2jy6AbBWHKcTJkidNAFdjgwx4B+0r0xYR6sP+MAWXktkIKPpATB7ARq9RBHLCN1edUOca/uJPqkfbIvXLZKRig9a4Ict/LsagPHAwgP2m5dLKw88BsAMoPVsZMcDDACe172aoeRbEAA3LUYty2QLKhbCAzAe5tnTJPWHoNgBYAjpq71KsAmgr34kAENjWg/kcXwegPl7BvHKqEamfvFB4kmdsMVUAtR2R8Jju8jrrnW9eO8eKxmMMBa8oJt9t5o2HQtO+NGai1cCQJ8fXitLkP3TazwA68leqb1CRjCpBcANv1ihlMZlDQD8Utns9MvaQBWAjf3qSsdX6p0fqzhm9NvVmwOeuaJZnANgyMIBnO14gOHlxQdj/8jn13PFM9OcyJ0DpwBYRgaNB2B4mBESwXuAUQDil7VHAp51zNb8PVV91FI6dulWwGUQAbDWx7sMAMPrDpUYreYkAKOqIvJQhkSGqPSom5++WLRfeGim/8v+yAMwrw6E5wkSjYsPEu/e6d3D8EjDoeN5gJ2gC/fm8ADYPdtGSSU40emwl6ebAIzYPxZjyz9oeW/C8BeL+asW8SEPWHOVfGn9oCwLwNAzHf1SSV0ABpQgYzts0IIAr4jas8GvuXD/+X4ZNCajZgdF7/dMAAAgAElEQVSAtw2qr/kQRYzu/B7VhHHE7Jh4KHevk4/em7HVvwzIv71aIUfQOF7JAB6iLKmS0IrIQg8KAHMfB2Y8wAAbkVeJfWBgYXIAfMafKAbNWGh7ojkBwLhWiDnXazwAs4pprH+LsWso4uhl6ScC7Dl22UG/10kmBtktAEZsL6teyEvTiQAYiVI/qwB4U/96mt41aYMQEas4ZgeA8UFYJEsKoQcYUPX31lOaS8JH4e8bTwT8HdcZ6ghGuyxskCgEAgCMojv9/tqhdOPX4VQIhAwAv1ktN/2w0hcDjKINAOD//fkkBALJX5NWH3YMgBEWIcpxiG4A3KVmXkr/zBMA5gveqG8WNQAfvnDT7+V2CoA9D7CZp0bU9fUA2EXbRzcPsJWKY7LmQewfCzHgYfKl8Wv9+r7QNGWi7XzIg2UALpWVRrcKBuBle8/5PYF+AB64IMCDqQfABfr+66/MNq1jBaqcL52uGYw8wHoAjJfYwvdqaG4XAzwX9Kiu2LDHjC3+dQwDAFfMGTQOyWRsaxNzZ0yR2B+DbQeA+evLG4P3kJsFYF46zwoAowgIH7KAqn9tf7QOwGY9wLDn+OUH/dqwWgDMV83iC2Holb416wHmEwp5AG705Uraffqa/5KhX+tyOWjSGl81MdY296/niNwTYpd3DPaFQCDDX603LPM8+ffdakopdNHvygiAoXWt9vSaBeBeDQvSqPm+im+sAXKHNCnmf37xoRhOAbBMEpwRAGPrH9eWD/fCOcAjOaypT1MarcanS+noRWMPsJYuuBYAd5260R8bbeQBnrHhWAC8Y13svheFQFy+eU/zHn27Vl7CBzWL0dYDYKjnQEqRtSMXblLNyDAPhI0dibQLPqgBy2EWPMAKAMMDHBlOoXffezHAMk8Fd/p4AOyOXZVZYxMA8/Gc/A+65bg1tOGIz6umB8DQZIR6BZqsB5j3pvGXEYlNzBPItujVmqLLPqzpf+jxD178/9y959Ljx74ZkQWNcsJ6TfSi7lQ9D32/4v/snQeU1FQXxy/Se++9L22R3nsHy6dYQRDEgogoKChVOooNFZAmRRQFKSJI77DUpS6dZell6b2Kfucm87J33iSTl0wyk1nyzvEc2Xl5Se57SX65ufd/5SIHMQObat5EWaKM1iduTH7rUKswzN91Bj74PQGA0ZvetpovAFNQxfAKfCgwu/IAzMYWKYRBPfzUFhh32yAih/QnLUk45UHzeStYuu+8otWbO2MqpdIXZk1v7N1IKASC6cTiA7HZqHWK/J6WZBM9XurdoxXTsI9emAG/BtCeE9fFKXJQ+DLAiozQvsu711WE9u0CYIyzZOVvqTYz1QXHY0IA/u2t6sCX0N41oIlXPLmWx1/vdonJkvsHN5e63X3wSKr813P2Hr3NvH7HLyIRucwBMEp58Z5eBODnK+TVDTNiB6EFwPjVhcl2oUd53s4z0iZaAMxebkVPXgSA6X0Fvx69UbuQF0Tip38EYKp4g/vnY8opAGMFQ/R8YtgJNvrVDte4WsKcFgBT3XeMAca1qNWsBGCs/IZ5KAyAacVHfv/+ABgL8rAXA6wiiiEQ+Oww2lwANmqx0PR3AdhGuzsOgEmiTKCnjcL5WNBgu+eTMSY74Kc4/vMS/aw84vlyygOEhjzgsVAAZjGAelBCYYKeD5b7ZZ5AlsRWZsASL2mztT3rQ70vE5I7KLRTEGLxpv7spQZO+EkOP49j8wfAGKaA2f5WATAtiYtetCxpkyuhJXiO1DtuBIBpvC61BdNZxr/xAEw9vPg77p8CMELvWU+pWy0AVvMiMgDGMb9cehDGrJbtjIktbSf5apbS47UagCetj1OSgbRUKGjRFbsAmKp0INBg4QdsaioQWBEs5vR1eHp0goY0L9WnlSSqd99gJXdpP6MvFvjFo2Su9KY8wCjlxetyY6x364r5AgZgfOlk4QYYfjV3hwzAWqFldqhAUADGF1xUfeg1J+EFAz2fUzYe8wFg3svI7gP4JW7GW9W9prXNxM1KQjJud+2O/KIZf0MOV8JGE5rZtc3fr1FRYdVH9TWXjJUAjIUvEICZUo4/AK5cMDPMJh7gE5dvK88CVM5hsdGBAjDGAGN4hV5zPcB6FrLvdxeA7bOt8zzAFgIwXrSDFuyDKVHyp1T0qKZKnhSypkshVRNijX5WpgBMgRf71imeTflULwrAWh6GDUcuKeLtTKaLxvXi/kQBmHqjtZaK2gMek4qajVov1Zaf8VY1zRhf1J1c+VF9TQDGghev1ywkxT12+22ncgjoTcdCIAP/2gezok+pFu7Awg4ZUiX3epiZBWCs8HXZo/JB7eAPgA8Oae6VpIdrBqtUsWIVFIDRG7xJxQPce+4eH01WVigBj4MCMNq5zUT/AEy9e7RkMP/wFrkt4PmgJitTNtFSoaCa03YBMFXpoACMWfNMnQXPiZXE5c+XB2CUReswZSvsO5sQPiFiEzo+628UgJd1rwslcqoDMF7zC/zEAKvFrKLc3QsGABhLHjM1AXYO6OUd/nw5SSEGGybgzvbEGgczBEIEgKduPKYkJLPjV4Osc9fvSnGzycj9GvvzAIx/e2/GDkX1QhiAs6WFVR9rA/Csbae84F0LpNmxI4hrFaXA0A8EYKbSQfW++XXLA/DJy3eg7perpW4UgDGkClUgypj0ALsALHLHCG0fF4BttH9i9gDzAIxAWTBrWh9r0jhN1NhlGcsUeHkAZkkwelCCUkRYiYxvFIBZoYaI/ou9Cjms69kAzt+4Bz1n75Ykc7A8rNoDm32O9bdM1B7w+Cn23//+kz6PosKAFgQwL4nW7wyA8aH/PgFgfJl4tWoB6bBQ/qpEv8XS/1OojMyXEdKmSKboKwfiAaY6z9QWrNQ0/o0vCsLHD/IAjHG/rBwrlm3e3Mc3BMIIAOOn/VcnbvZ7RVMAphXT9Naa2qB4PpM3HFMyz40C8NCF+xVJK358ozHAGHvLlEtodT5+XWkCMCfVhwCMGfX+YFPNJrjm/upa2+snfwCslpeAISPFVQAYv26gZ9sMAGNynFoyl9o5aAEwao5//IeciEqT7SwLgVh8QJFO1FrEVF9czQOMnk8MgaAvPRQsRR53qgD86w6pwhxrKAnHFBfo+HSu9TzAVgIwer4xTIsBMJU75M+ZB+BTV+5AnZEyANMcCheARVZLePdxAdjG+XMaANNM8UBPGx/QVKdXC4CfHROl1JK3GoBRjH5s20o+pxIVe0n5FM4AmC+rigBcIGsaVTPQmzgWzMAiGGYAGOWRWNMEYI+XROt3lI5rX6OQFEvZdUaCB5gCMO4DPbuT1h+Dvq1KASY+YSufP5MkNzd+rRyLjHNG46ONhEDQSn/UFlQlQwSAl++PV6TCrADgr5YegtGrY6VDmvl2dXh5gn8ApnBjBQBjxj377IrecCa/R21Ei65QD7CVAExl6lDLuqWnOp8oAPOJmqgLjA09njtOXvX5pM5fD+hlRVmwoc+W9bmu/AKwylcpDBkplsPXA4yXUysdAFZL2sJjMwLAn7aIgM8Xy3qyrKHMHN6/esySARiVNGZGn5L+XysEwnAMsBkArl1Y8UrjsaDnEwGYfq3BF4djpAiG3r2fjwHG/u+ZAGD2dUtrf0YB+Pqdh5pV2RD8EYCZzKY/AK5SKDP80TkhCY4CMOqRn74q6yNjTkHFgpmg9ADjMcDouJgSdUxJqPNnczcEQm9F2ve7C8D22dZxIRBWAzB++mVlObWA8rsVR+DbFYclK7M4Yfx/fx5glgWO/fw9PFuUzQU/vuYLwBtjL0EbTywogwEegNf3aiBp5Ko1uk9/oMy2VTtGVlbUXx/8DZNPUJFCzwOMovv4GZI1fBi/4vEA8+fAxsKHwpt1CksZ7TWKZpUKVlAAZt5lkSQ41Ni8cDMhBpDtE+Pk6pbILv0zEADGeOEtfRr7JMGpeYBZqVzcJwVgrbKt1D4UgGnJYL21prZO8MGFmqvsoaulQ0zjaWlmPL1++PGNeoBp0poZAObj1DGEJ2u6lNJhjVABM0wqm+tJAsM++wY1g7Qp5cICWutR7TeqX8x+x5ARjI3nrwnTANw6El6qkl84yVENgDG2GV86mRILVqH8fVsIAJgU2MEY/461CnkBMIIfhqVRAOarwOk98tQAmKo74PYoocgSzuj4dM5YfoPW/jB0i4WUsD7+VCD8ATCed4bUCQCsJmXH9uEPgOlLOQIwJsyVGrBEz2Rev+PaQJUeLK/MFCX8DeACsCHzWtrZBWBLzek9WGL3AFMPlhYo3v/nEUzfdEK6kRyOv6kkkSA0MX1atBr9tygANyuTU7XE5sajl5RYUBYPiSECrFIW7k8UgDd+2lDS0vXXAgFglN1Z07OBLgAvijknJR2yhi8TL1eRQyD4tvPkVSk5sV2NgpAyWUI1MexnFoD5hDa2T6MAvGJ/PLzpKRZBHzZWAPDszjXghXGb/M6V1QA8fdNxpTiClg5xsAGYffVAQ4h6gFG6jGa76wEwqgBggs+vW05K9rYSgNlXFzUARs82K0ChNtGqHmCDAExl69g+MMYaY4CZEguGHzGJt6CGQBAARs3vDioAjB7gS7fkqpzYjAKwWggErfCGY1IAxvOPHd5S2hedMz0PuJUAjMCLSkSsUIk/AK5aKAvM6lxDsc/pq3eg9hdyCAS9J2FSLcYAGwVgdi2oSc1prVm/N60Af3QKhwR4GrZs7gKwLWaVB3XKwmM3Jas9wCIATM1LP3nxAIz14hHOsNG4W38e4Kalc8KE9r415lF/kcWCKgBMtH1xH04BYEy6WNcrAYDRq40JQN+tlKsYsRCIxTHn4F1BAPa3pPGlo72nWIQRDzCVLKPjY5xcneKyB5h/AKrFAFMApp8btQD40zl7FE8b28eCrrWhXD5Zsol6gOe8WwNa/+gfgKl3j5YM5o9d5LaA5zd98wno7ymOoAXAVHPaLg8wlb9jiZ9q50RjgJmGMQIBhm/QZB8sjIFJRdjU5LkwBvToxVsKAO8f3EwpLcvbzmgMML4wFMmeDj6bv9erYAeGFOH1YRSA2cuiaDKeGgCjh33Yc+WURFTU1f1lswz/wQyBwIpn49bKqicIwOgBpjJzCH7oAWYFZrAf2g1zEkQbK5aDaworwWHzB8AYx314WAuf6x9l2pZ2r6u5W8MAfPehZllijO3HMCAzAIx5CLU+XyUdJw/A6LiJ6G/MA8wk51wAFl1xoevnArCNtncaAFOx/EBPGx/+VMjfH1CyfdEbHgVe/L1+yeyw5pAxAG5cKidMet0XgDfHXYZXPLGgTBO1eN9F8PCRR9wXADZ80gDyZVYPgWCyOAin+DDms6RFHvCYjY7Jb6xpPXwx6WJ9r4QSwBgj+n7D4tLnM2xM7YGWEMa/j/R4tYzOIwVgBtciIRA0uY7u0ygArzwQr2jlUgDGEIutfX1DIIwAsIgH2GoA/mXzCaU6GHrD2csFsxFqAyPMNfAI7VPlEhpDz8+j0RAIqtMsCsAYshJz5rr0woUJmzTWkQKwWoUy9AAiADMItBKA8YUBQ4OwsMquU1eVlxoEOfQA+0uCU/MAGwXgPi0jYPgi7xhghEGUQWNx+LQkcagAuEyeDNChpvUAjGsRi6fkzZxaUpHB1nn6dliy77yyTDF/gMW+o2Pl0FBfAMYkvSUfBgeA8brGEBz2Mqqm5cwOnvcAWw3Ah4Y2l768uQBs9OkU/P4uANto88QOwPTBaBSAKfDyAMx0QPHv/rw2jUvlgEmvV/GZwS1xl5VkKFYVq1ifRfDPv2IAjANiRn3SJEkgdQrvEAK15aJ2jKIAjBC44ZMEAEaPDnrXMJnr0s378He3OtIxLNmbUEJYAuAXIuGlyvkNr16zAEw9I3SnGCdXu3hCpTx/MI1wsupgPLwxNVoagmZcmwXgr5cdgh9WyUlwIgBMvXtWeIBnbDmpVAdTq0SHlQQxhIZVmrILgKlOM9O+Vrt+1GTKsB8WraCferEyXGaPB1gNgBGAjl68LXnAJWAa3FzzWsE8gVErDgN+RTgcf8trzaqpQNCE2n8e/atUdsNP7c2D4AFWA2D0hmNcJwtD6tqgmJJ8qQXAeh5Q/uJVi7Xm+1B9cS0AxhCIiyRen4YoGL5heDZ4Z3o0LN0XrwrAWsoiegD8R/QpnyIpfmOA/XiAcc7SpEimvIwaAeCz1+5CTY8HmL7o47WLhTCMeoCPDGshSYHWHbla0RT2Z3c3Btjsqgx8OxeAA7eh5giJHYDpp1ERAKY3vAYls8Nqj8cXDUj/LQrADSNywOQOvgCMZYOxBDM2BjlYHQo9SqxFfdpQ+txlRRu9ChP9jniNbxSARy45KMmVTe1QFTKmSS4llGE1uic8ShK0gEQgAEyLhBjxAFNvLbUZxsnVIqWizQIwqkyg9Ba/vZoHmGozUwAWSYKjAExLBqvBot7awAcXxoGy6mBqlehQmxjXGSu6QgGY6mjz+zLqAaY6zePbVZISHo0A8L2Hj7we9FgZLlMaOQRCrWQzrp3YC7fg5036AIxjYJGUIX/vV3TD2fkiAGP4BS1gQvMJaHltBE08L6MhECxhVDQEQg2AUQZw4DNlFA8w1ZnVKjFvNwCjzneHmoUVaTa0KYZATNt43Cth1QoAfvvnaFi2PwGAMXyKSY7R6n/Uxpikh/kcWs0oAN+49xAiNcoS8wCsJmXHjoNPgkM95Boj5BAIGuqF1y6GQJTsZywEAsNN8GuFC8B6d9DQ/+4CsI1z4DQApuVSAz1tfEBTj4WVAMyE8PWgBL3IUztW9TmVbcevwIueZCgGwLS8MW4gktxmxEZ7z1yHp35IqKwlCjAIRwjjeo0HYKxu9aIJDzAF4CHPloF2NQr5QKcaKFBvLT1W9JLUNADAqw9egI5Tt0lD0DGxZDNKb/EA/MnsPYrcFNuvFgDPeqeG8uKjZk+UhMOXJvZ52woA/n3rSUXbOpQATHWagwHAuHaOEADGoidYCMdfUwN+FsePsmMstpXeS5wCwPiChpDVfaYsg9a3ZSmlBLZVAExtoGVH6gEulzejVCSHaRPjNuj5nBrlDcD44nBkmHgMsNq+3/o5GlDCkDUKwKhBjUmU/P0aY5QXBQmA8cU2Xapk0HfeXuk4jADw+ev3oPqIldJ2qEeO+vDYUFccAZhprOvdo9nvzPlRZ+QqOHVFllTz11wPsJ6F7PvdBWD7bOu4JDgEYBQnN1rdSc1EPAD7i6ll22PlJHazRhBZdfCCMjT9tygAYyIdxl3yLfr4FUUNgEld8VBnNQDvO3sdWn2vDcD7z96AOTtOS15i/ETJGia1YPyaXqMV1LDvVy+Wl6pRGW1mAbhg1jRw4vIdn93ZDcC9Zu+GWdGnvfZrFoDRwzxv52kFgFl8OBtc1EPI+uM1QMu5qpVixocovuSwSlN2eYDxBYIlPk1oVwmaGvQAo1oL9XRhZTj8EoFNTa8YwwFQ1YV5gEUAWC3mmcUa4xeQsZ7S4RSAaXltUQ8w3mNYlTY8fqaZLTq/FG7ZXCMAo9QWK4WMHnAmf2dVCIQIAHepX1SxkxYAoweYli22AoDfnBYNKw6oAzAmCMYM9AVgDNHAEC6tRp8H9JrC/+dfhPFv/jzAeA/F42AArCZlx/bBxwBTAKZqN2YBmAGtC8BGn07B7+8CsI02d5oHGD9VoeLAzG2n4MulhwI6c7zI6Q3bKAA3isgBKzUAmFWC4m+E/AGjlvD0TtV8zmP7iStK4gwrdsA//Db1bgi5M1oTAoEHgIDb8nu5AAU2rbd69HIxkX2MYR7dpqKu5wzHowUk8N9mAZhWyTPiAUa5NjVNS/xMWLOoeAwwyzDHc/D2AKeA6H5NfB58RgBYrxAGzsmEdUctBWCa2IkvA0x/mq0DfIhi+AirNGUXANNCJRPbV4YmpXNKh+BPBo1eOLSaIP4ds/8x8QubGgBjQtjh8zcVlQYzAEx1YqmaB72XUG1p9LQ2KZPTqyQvf/HjHN++/4/0ksnucUxVIhAAxheMDxoXV5KsMAm11xy5LHIwPcAUgLHq3us1CsFHnup0eCzo+UQAZl5MdnxMpcHsTZ8HYPoCQNUiqI0xRGPh+9YB8M17D6GcRggEnjceR595MdIpqil5sHPnATj+xj2oNlz2AFMA/v1t2QNcvK9cZVO0uQAsaqnQ93MB2MY5cCIA7x/cHO48+MdUdRtqKrzIv1hyEH70eG1EYmrnbD+t3Kx5AKb/pgDc6vv1mh7r2sWywS9vqgEwZo5vlA736xfLQ+tK+XxAIFQAjPZCu2GjZYT1liGVD6Pnpbcd/7tZAC6SLS3EXbrtszsEPCyywZpeDDAPwDnTp4LoE1eVWG1+e6sBeOK6OOXTNZPIUzt2EbviNUDjGPFloM3ELV6b4kMUPcBqADzwr31eXwP464v+Ww/e6IPbagBWK9iB6iSHzt9QAJhlvvuzG+8BpgD8zbJD8L0nmZHeS7wAONkTEthjURitRl880UN958EjeDJ/Jqm7ng3ZmKhxPPTvA167wBCTLvWLKeV/MbSKaQIHE4Dfa1AUxqyWZdDK58soVYmkAIyeTwyBsB6At8GKAwlf7CgAYwLmzgGyXBq1MXqoF7zvXRabGtWoB9gfAGPsc+Y0KUwB8IUb96CqBgBj2eRiJgG49herlKpy/q4LNwRC5G5rTx8XgO2xqzSq0wAYxdz3WQjA9LOlUQBG7ye9oVIAZqVQ0Yb4afbM1bvQ8GtZFow2rUIYWLr1+bH+AXhz70aQK2Mqy2YfZYNYCWIcVOumRgGYFpHQOxAqH4Z9v3mpPDxf0XgIhGkAzp4W4i76AjACXvUi4gCMJZs7TEmIAcaqXxgnhzDEP0DRhmoA/He32lAmj6wDTJPg0HP41s/bJQkn1jCHkOU+4nhWAzB9iIcSgGns4qT2laGxQQ/ww0f/enm6sDQyk8BSA2BMLDt4/qYC8GYAmJbK/Xb5YUX7mg9PYlCFVeOMADB/TQUCwKgC8U7dIjDCUyIZpeY6/yIXptEKgdBTQeCPTyQEQg+A0fOJ3u9z1+U4Vmxot8MemTK9+4zW729O8wZg/HrENHfx5XjVx/V9rl/0UP/VVRuAqUOE7defCoQeAOMcsYRUtURGtg8+CY4CMK14ifc2F4DNrpjw2M4FYBvnyWkAzJISeMkjMybAG5VRAJ674zT0mCUnkaCGL40p0wJgdmz04YVSYcv2xUvVfNSUHLAS2nMeAGahAvzDb0ufRpAzg3UAfPD8DWg+Sj8EYuyaWKk0MTZeQ9ffPFD5MOzHQjuMzl1U7CUl495ICATCCspe8Q3DDqoFAMCogUybiAeYAjD1HCI44QMMwYStLczGZuofuGYnrY9TvHtj2lSEVpG5ld2LAhJ9WNOHOHrDWQEW1gftg3qqrNLUU5G5pbAXbFZ6gGn2OmoPNyplLASCyo3hsWFp5PQeDVi12F3U1j1wLgGAEbAQtPw1fhzMR1j1kQxOtGQ6/3WGzQvqzb5WvaBSfl1tX/68aaLzq+YBRi9np9qF4atlcll3tHGnabKcn5YH2A4ApvJr5fNngvbVC3p5gBGAMQTirMUA3GnqNq+QNbx3XLvzUCqEgioeqPiAjdoYPdTzLQRglKak1Qrp/GN8NoapfDpXDoEwBMA370HVYXIIBA0lwmu3cqEsgApCRhpbg1hcAzWG9ZrrAdazkH2/uwBsn20d4wHGh/Tv207CyBfKSwLzVgHwl0sPKp/jRJLKMAGJZVGjJ4dmFVOPMHoFmUeQTQ8PRvhplBaaoNO469Q1+N+YKOlPTC3h+bFRsOPkNaWb1QB86PxNaDZqnTK+1k2NAjAvIeZvKVL1BOxnCQD/ryy0q15QSAWieI50UtY/34wCMC3FjDHAegDc84/d8Md27yS4Rd3qQOk88gOXAjADp26/7YS/dp+VfvcHwGPbVpQKK6itMZHbAs4xXdPoMWIFWNj2wQJgqtPsD4AxEfbAkOY+p0fVFvBHCsC03DPbEHWoMe6dJXSKADDvSaYA/MPKI/D1chku+a8zFICxWApWJcNiN0TVUPe6ww6BADAWGsFwg+89FRqp5rOTABjBD0MgKADTQhUi61qtzxtTt3klLbOXZ/4+7AXA+TPB/Pdqae7SqAfYHwD3aFJCevllAKyWyMgOhPcAo2ZylWErpJ9pMileu1UKZYEiLgCbXTaO384FYBunyCkeYP4UrQJgmrgSOAAneIRFANjftFEAZgUjqNg5bru1TyPIYaEHGOMNm35rDID5BDK/AHzoAnT0hA5gP6ZuYXT5enmADQBwiZzpfIoY4L5Reqxq4SyqEMlX5cJ/BwOAP/h9J8zfJQMw/TzNe4CtAOA/d56BD2fukvalloSH9kEPMCu1apcHmALw5A6VoWGEMQ8wVVvAc0FZK5S3wobhEShhdvfBv5KSCTZ8sUQ1mYAAmHw6H7M6Vkla0wqBoCD3zfLDCozSa8AKD3D/p0oDwjptGVIlk4rPsLCHOe/WVPIMggnA7zcsphR+wdhmfIGlMcAIwNM2nvDyPNoCwJ57B3//oQCMx/enHwCmXwTZOP5CIHgArlY4C2w5dkXatHvjEpArY0pFpUPNi68FwKieUnmoLwDjtYshEC4AG33KhE9/F4BtnCunAjAvem/GBHijovGXIkllFBZQ/5MKq1OP8MqP6kHR7HJMKGtqsjhax73n9DV4ZrTsAaYlg/HzGd5EsVkNwEfib0ITAQCmD3o+gczfPNDYWezHVzETncONsZcUpYIhBgAYP2+jUgDfsPgEekm05omfN1qJTsQDzEta4X70PMDdZ+6CeTvPSIdE4YQH4B/bVoQWBjzAmGV+/e5D5VxxvPm7zijJUAjAWG0QCwRgkQhs+BBFFQhWaQpDLjD0ApuVIRC0UMmUDlWgQUQOaR+iKhA02Qy32zeomVRaljYaW4+hRSj9NyVKlvQT8QDzahL4NQrLHmOjsfG8ogw7B1pxjMYM02O0E4BRGWPi+jgomDWtNPeFxlQAACAASURBVKdM9lDr2rAjBIICcIUCmeC1at4AjJ5PfCmhn961qv+J3jOwX8cpW70KF7F7Bz8GXW94fPO6aHuAjQIwqnuU+WypsksaDvJh4+JSEQsmU2cegFPApVsPlGsXvcWFe9sbAsEXTTIyLyJ9ncohIsdudx8XgG20sFMXnlUArPb52Z85KQBjAhstrWklAMecvg5Pj5Y1eTFW8eUqBaT/LzNgCdx+8Ej6/619G0GO9NbFAIsmwVEA5sMH/NmOek6xnxUAjFquGFPpT7lB7/LA8sMYJ8eangqEFQCM1aVYzCFdg+zTeY+Zu2AuA2AC7mYBGLWa8XM9yu6xilF4vmoAjPHQdJ3jCwJ6Z9UA+LP5exUVBd7OPMjpfb7Hlwkmuu8PgLVCIHD/dB9qALz9RIK6CiZhxpxJAGCRJDgegFFab03PBtKpU3lAvqgOOy567FhaedSKIz7L0woAxlLkgzkPMNW6xZ1S3W+rAJiq6mhdd6EC4A5TtsIaUrmT3Tv8AXDFAplgro0A/EGj4kriZLdGxSFfptSKNJ0RAL586z5U8niAaUVF9nKvd+1pXbuiMcAuAOs9Zez73QVg+2zrmBhg/hTtAGARVQXqLeMBmHqEmW4nPW4jHmAKwKwMKg/AWBQBEx6sajTxjsGR2tgUgHnvqb9joeAYzgBMC3GY9QDrAfBHs3Yrn+pR6qzrjJ2AMeaTXq8iJVCxz9uYyd+8rH4MMMYX4gOWh0SErQW7z8L7v+2UfmOKGDwAo7eQgTP1AFsJwAWypIGTV+RCJZiU1KCkMQ8wf277BzeDNCm8PcC0wAzGoFfInxnqf7VGuo7wi4pWTD5b1ycu31ZKQuPfMK58eY960s9Un5mWQqbHFVIATpkMYjzVzvCYqO53MAG4W8NiilwcAmZbzgOM4Ice4NNXE5Kv/L30iN7/7ABgGj/PjsNfCATvAcbrEkNhsKFd8mVOYwqAr9x+ABWHLJfGoTHA7OXeLADXHLHSKxZby9Zxw1sqJe9F58NIP6c64oycg119XQC2y7IOkkHjT9EyACZxeEYBuHmZXLBk33nl0CgA42dR/DxKmxEApvG41FNKQyCsBuBNRy97KQBoeaIoAPPeU39LkYIj9uPL+Iou441HLylatUY9wFhR67P5+6BakSyw/sglaZdz3q0BlQqKe4CNAjCFWXaOSz6sAxG5fJPgWGIjDZtA7wpKQqHiBybEmQHgj5qUgPc1AHjhnrMSYGNjIS0UgHGOMQY4mABM9aVFQyB4AD4wuDmkTuFd2piWGGcx6JhAhLHCfF+t9Yj9UT3m75hzUnhI2byynB1V51jzcX0oRK5/dg5YyAd1zLFR1Qi6Lys8wJ89XRoGLfCOAWYSkmxfTgbgaZuOe5XgtQKAX5+8VYrfZw0LoSB8842uNz0PMA/ASZIAHBvRShpS7X7P69ej9i8rdoKe8fxZ0kCv2XJxErU4bnasfBLc1dsPoIIHgNtUKwAoFYlrDavY4T3DbgA+OryltB+7mgvA2pZ1AdiuVfcYADCNwxNRVaAe4BZlc8HivQkATD3CgQIwxjN2/mU74Jv9jLeqQ/KksjwTBeDofo2lt32rmhkPMCbSYKUhkUb1e7G/FQDMHmIiIRDoYcQqghgHjNDH5Oz0ABiBF8uT9mpeEp6KzCM9XF77SS4WgZ7RDZ/4l0EzA8BUO5gHoskbjimft8e9Vgmal82lmF/rQecPgLEow3szZD1YVhaaB2D0TFUfIcss2eUBpqWq7QLgLXGX4eUJm2UAfeVJePbJvCJLV7UPriMqm0bvDVpJcKEEYLpvPCEa8oRJZvdV4uONxgCLhEDglwimRIH3jjZVC3glwaEHmAdg/tjNTFq7n7YoL724vR0AnOyJJBA7vKUwANNyx6iPXChrWujpAWC1MBYRAO5QsxCgDfFrBoNSF4DNrJjw2MYFYBvnyalvXlhcomS/JQGdOYIFjcMzCsAty+WCRTHqAMx7gPBAjXiAtU7MTgDmk4i0PFGjVx1RtETndqkJFQuYA+AfXq0AWFbXaKMeYCMAjF6xjrUKS7ujySs8xOvNE1WhsASAyVcItgY/mb0HZkafko7VHwCPb1cJmpUJDIAXxZyDLr/KAMxk7SgA4wsCAjArtUoBeMD8vfDzphOqUygaA4xygW/WLizF0LJS1dPeqAr1SmT3AQn8g7+EKBqSoVbaGKXHmMyb2RcwrfWKMmw9Zu0CLOjRu2Upr25sTVEvLEIg+/xNO9vlAea9qBSAgxoCQQAYFQpe5QAYPZ+oA8zCYdA2TgVgep3gcVK1ChEPMI3zxRLR+NXQDABfu/MAnhwsh0AgAA98pozq+hO917I1WGPESq+CJFrbxw5rAck8ThrRfRjp51QOMXIOdvV1AdguyzrYAxwqAEZtVtRoxcYDMA2JCAYAb+/XGLJa6AEWhXSqd4oSQaxMq94ypOCIffkyvnrbs9/NAvDAp0tDBw8AU/1OHuKDAcBLP6wLJXOll06JymExZQ8K6FYAMIrsd22oHgO8ZO85RRrrl07VoHbxbF5JcD4AXC43jGkrq0AECsC0klr9L1crAEwrDPLeK3+fw6kqhRoA0zAfs+tPdJ3SfuwcMNQC5dmw0esoGADMS4nRwjfB9ADTxC81AEbPJ3qAT1yW48Gx8eEbZuaA9wBjKWwMF+CbERUIHoApqKvdR3j5Turlfbd+UUk5CMOfsKmFsbBj5UMgrt95COUHL5N+dgHYzOoI321cALZx7pz65mUVANM4PBFZMZow1KpcbikOkDUKwGt71pekhtQegvg3s5Vzyn22FG56ZNCcAMB/da0FkfkyCa1AKl+GG5gFEAoxRjzAg58tIxUCwEbL/87rUhMqEC+2HgDT8zDrAdYDYNS0nb75hJRkVbNYNi/7Tok6psR3TmhXCZoKeID9A/B5KdwGG6vsx3uA82dOA1WHe0IgCAD3/3OvdJxqTcQDTAtJiAKwPw+wHgDTueOr6AktYpOdgg3A+LI3kIsBdiIAI8ihwg2DPjSvXQD82qQtsCFWjvvHZgcApyeJhiIAjPekAfP3ScfzTr0iUDxH+pACMErQoSY6VujDVn34Sjh/I6EktdbyPzKshRKmZ/IS8buZUznEjnM1OqYLwEYtZqC/Uxcext+V6LfYwJn4dsUHNP0MaRiAI3MDxk+yRmOC7QJgGgKxo38TwNrxVjY9+MN9Uc/Vgq61oVw+OQlIr1HPLfY1CyAUgNlDTCQGmFV+wn3/EX1K+dRoNwDjZ/G5O2RNX9aWda8LJXKqeIAFpO3MADAm27zXoJi0e95WS/edh3emywDMQg98ADhLGqXUKr74MQ+wFgDzMmD8fpkdKAA3+GoNHLskl6qmJbaNJMFRAFaTNaNfIfgiInrrN5Df1QCYhhLRsa0IgVADYAxzQK1j1mjlRy0PMEr1oWKJaBOJAUa9Wyb/VrVQFnipSn4vAEbPJ4ZAsHAY3Df1nIseC9+PB2BMiMXwC74F4gHGanu7BjRVvc7wj7wHGJN4+/25V+r/Tt0i0j2BFQVRm0N2rD4e4LsPofygwD3A/NoTBWARDW2z84bbOZVDAjknq7Z1AdgqS6qM49SFZxUAU5gTAWCaMY+xkBSA8XPajC0nJSvyMkhq4GFm2igA7+zfBDKHGIAXvl9byYLXOx9eZcIsgJgGYFL5iQIwH8ah9xJAQV7EA2w1AE+NOqZ49ya2rwyoP82aVrKLPwBetu88vO0BYJZ8JgrAp67cgTojV0PJnOnhUPxN6TC04jXVjo0CcMOv1kCcB4BZKAZ/3eC/RUMg1B7KNIGRl5DTW7+B/M7OnXoIqZoKHdsKAB70TBmpmAltWFHwyDA5QQubUwD45Sr5vZLgEIAxrpy9DOGxWgHAbSdthqjYy8r5iwCwXiU4mviIA2vFrrM55dWLUOKSlT5+u24R6TpiAKw2h+zgMXRk9rs1lXO5ce8hRA50ATiQazRct3UB2MaZcyoAY2nT4n0D9wBTL4xRAMaSsAuJBxh1RbvPlOO3Ai2FrDWlNAQiVABMveZ/d6sNZfKIeYBpAhKen1kApuMY8QDTrO9Z0acUuSHbAZgUtWDzGogH2GoAXrE/Ht78OVo6NKa/ywNwgSxpocowudQq9QDjvzH+ME3KpMr16GQAplrUvIKGjbdRxeuuBcAon/Xff/IRWAXAKKW4ZO95uHxbrgpGFQrw31RqEcNK7j30rZJo1AOM8nBj1xz1a0ovD3DhLPBSZW8PMHo+EYDZyxAORu1mdp7aTNwMG4/aB8CfNI8AjONlTe1FmgdgLMfNkt4wEbR0ngyKOo0RAL557yGUswGAqw1fAfE37uuaXKSIjO4gfjo4lUMCOSertnUB2CpLqozj1IVnBwCLqEBQyShUMMCYYNYwE7bzLzsAH2YYm8mL6ut5FkWmkQLwrgFNIFOa4IdA0LhpWtBB7/ipBBX25cv46m3PfqcAzLw4IiEQNOZv1rZTiuD8/PdqKTFvuA+9eaIeaCEPsAoAL+9eF4qrhECIaDvj52Hm3ZvUvjI0DtADvPJAPHSa5gFgTwliHoAxnr2yp9IUD8D8A98IANMkuIZfr4G4i3IIBFOj4OcD/y3qAVaLS6TVCPn4adH1Z6af4gFOlQxiBspJcNQDjBKq/1oIwCzeHWPJi/SRy+CiJBbqtbJGAdiqEAgRAO7euAR8u0Iu/oDxpqECYFpgiM4pvf6NeIBpyXqt+wgPwFiNkMkxdqpdGMrmzaA4UWh8ML/meA+wC8BmrsrEsY0LwDbOY2IHYPoQMgrAz5TPA6gKwZpeYpseWIlMo9MAmCZz6R3/1mNX4KXxm5RuZj9BmwVg+slz5raT8MmcGOlY+EQ+vXmiAJw/S2pY38u/DjAta8xO3ioA/un1ytCoVEIIBNoX7cw3fyEQqw7GwxtTZQBm4/kDYFQ/Gdu2ks8+mN2MADANgWj09Ro46gFgpkesBsCiSXBqALz60AXoOGWbdOx8+Ije+g3kd2abDKmSwR4VAEbv7D8eArbCA0wTPtm+aZEGPJcj8TehybfrpNMKpgeYAnC1wlngRc4DjJ5PVIFgL0N4fHwZZzNz8eqEzbApLsEDLALAmAyGL8hajYZAjHwhUoJ51kQ8wKhF/cHvu6RN3qhVGMrlMwfAt+7/I2nEYwtEBYJfe1WHrYALN/U9wGqKK2bmSGsbp3KIledodiwXgM1aTmA7py68fx79C8UsCIGgACxSCY5qpoYagHcPaAoZ0yQXmEXxLnrwhyNRDzD9lK+3F1qFC/uaBWDqSTbiAf6idTkp4xzb3jPX4akfNkj/v5VLPNOzAQVwEQDuPnMXzNvpnQS3okddKJbDNwlOpLgJ9QDzAHzh5j0lWY3OBxbx6FJfPQlu9cEL0HGqDIXMo8wDMAr0V/J4gMMagFXOVW/dWvG7GgCPXRMLI5cckoZPkfQJePBIDkGwAoBpwifb9ytV8sPnrSOV0wkVANPyv9WLZIEXKnmHQCAA/7zpuPIyhAdsBwDT+wGdY3r9GwFgDGdAmGdNBIBZmXMGruhx/nCmDMQ4h/09ChH8GuST4GiJZReArbhiw2cMF4BtnKvEDsD0IbSpd0PInTG1X2tSAH72yTwwf1foPMC7P2sKGVMHH4Bp8RAKcnrLMPr4FXhhHPUAe1cx09ue/U4BmHlxREIg+E+UOJforaxfMofXroMNwLQaoQgAIxww6aTJHSpDw4gEDzCeiFqymT8AXnPoAnTgvKI8ABfOlg4qekqtWgnANASi8TdrIfbCLWkuZrxVDWoWleXf+PMJJASCervVbCe6Bo320wPgjrUKwZSo46YAGJMgl++P94JoCsCYTIaxzy9UygdpUyZTDj32wk1o/I1/D3Dp3BlgkQEVCJEQCD0ARu81vuSxrwF4wNRzbtT2rP8rEzbB5riEryNWA/BXL5aXbMza4AX7YXLUMSiRMx0s615P+jMfAoFOAAybUwXg/5UFVFlRa3wIBC2x7AKw2RUSntu5AGzjvDkVgLHqUlFPbJvZ00dPCwVgvnyp2riLY87Bu56qWf97Mg/8+ZgD8MqP6kni7SJt+4kr0PrHBADmq5iJjIF9TAMw94lSa396AEz3b4UHONQArBYXGwoAbvLNWjjiAeDf3qoONYpmNQzALK4VPaqYmMPH4Xsl/HninUXXXSD91AD4xzVHAWXDsKGaCpY9RxmsXBlTae5K7eUGJefwvNOnSq6EGA0hiidag4kAsN1JcBjz2qFmYS8ZNIT3aZtOKC9DVgHwy+M3wRYSHiQEwPkywvyutTXngxZG+vrF8tCaADAqFW2IvQiVCmRRvtTx+vV4D2QShAiuFQpkUkIicA7NAPDrNQrCoGfLeh2zljoMf2JmQyAODG4OqVMkDeQS8butUznEthM2MLALwAaMZbSrUxeeVQBMH0JRnzaEvJn8e4Bp1axQA/CegU0hQ6rge4ApsK36qB4UEQbgq9D6x43KEjQLwDSW2IgHmPfQhBaA6wGWAMZG7SmSBEc9wFNUIE7tYUcz1HnAV1NG4AG4SLZ0UMFmD7AVAIz2RGk2DA1SuzbQU/qWR/GCSb4ZvSea6c9sjl9s8MsNNiz9/PliGYBFk0nV5pap19DSxmIAfAvQ644Nvep3Hz7yOTU7AJgPHcPrkhbCwGP/eeNx5WUID4razYz9cRsegPkvQmxcrxAIAwCMCW3PV0zwAKsdJw/AGMKE6+DAuZuAzgQMr2IxwVQjmB+L9wBTfWErARiVXy4KxAC7AGx2VQa+nQvAgdtQc4TEDsD0IbThkwaQL3Mav9akAPxchbxesZ1BSYIbuBRu3vtHOkYnALBayWctA+44eRWeH5sAwGZlqMwCMO+hMQvAdP/mPcDOAeD1Ry5Cu5+2ylD2WkVoXja3TylkCsBY8OXH17ST4LTK1qrBGw2BaPrtWjgcL4dA/P52daheRN0DHJErPSz5sK6pu55a0Q9TAxnciJ07LZQwfu1RGOEBYNFkUlUA9sSwUwBGeHqtekG/R4nhJgyAtZLg7AiB4NVzggXAfIKo1QCMMpjPVfAPwLx+PYbh1CuRAxCM06RIBjSpzggA09AKKwEYlV8u3dJPgts/uJl0/HY1p3KIXedrZFwXgI1Yy2Bfpy48Ku9j8JSU7gis9CGkVr2KHxt1NVnZ2Ocr5IW5JLkp2AAcM7Cp9NnTyqb3+R/3RT2WahXvxAFYhi2jzSwAi3ho8Fj0bGAUgD/8fadXqAzug4aOGPUA0yQ4pttLbagGSZ+2iIDO9WSNUv78aHEIJk3He4AxzOXJwcul7e0C4GbfrlOKacx8uzpUUwFg9Hxhtr3oVwd/1+/Pb1SFuiWyG11+pvrrATBVBfG3A7sAWMsDbBSAv1x6EMas9q8DzBcQ4gEYwQ+/crCXIbQHfXEwNQEA8OK4jbDt+FVlc161gf1AbRyZLyP85ScEAmUw3/9tp7TpqJefhP9VyOv38NQAmMbw05AK1C3vO08sBjjUALxvUDOv+HKzc6S1nVM5xOrzNDOeC8BmrCa4jVMXnlUAPGHdURi+SP4MaRiAK+b1KnEbbADeO6iZVCHJyqYHf7ivb5YflkpIi9qMHd/Ok1fhOeIBNqsDTNUkjIRAiDygRACY7l/EA6wHwDSpUCQEggKw2md8owC8MfYStJm0RZomVp6aB+Bi2dND+cFypalQArDeNaZ3LdAvOLTanN52gf7O5iRzmuSw01Mql758iyaTJgYAxiTKRTHnFZOqATCucRYPHi4AjJJmzz5pDID5ECYKwKhb3meeLNXINz4EwgXgQK/Q8N3eBWAb584FYG/j0k+oz7sADCJhI8yCu05dg/+NiVIMagUAs0QWERUIkQeUHQD8we87vdRCcB/UAxxqAKa6xj+8WgHwE3UoALj5qHVw8LxcTnnWOzWkIgki82Hk9oeKCA2+WiNtIpL0amRsf33VAJi+fIsmk6oBMHtp2n/2BrT8fr10GEZDIILpAW5eJhcs2acNwOj5nBrlDcD0xcHsnLzw40aIPpHgAeZly9i4Zj3AIvcXvoAT/wWHepSNADCNLW5foyAMtigJTjQEwg5nDJ1np3KI2bVo5XYuAFtpTW4spy68//77Dwr3liscmW3oTZq4Lg6GLTogDSHiAaYA3LpiPpiz47Syez3vlIh3Ve9cypEYYDs+O4kcI/UAiyQOsnPafeoaPJsIAJjKuYlUglMDYJo8GAgAT3ujKtTjPuOrQVLvFhHwjkYIBNU1/v7VCoBJSj4AnCM9lB9krweYAvAfnWtAlULWAzCuRUz6w8ITNYvJMmvBaGxOsqRNATv6N5F2Se89qz+uD4WzpdU9FH8ATIshiIQm0RhgLQAukycD/N2tju5xsQ5UX1tro2ZlcsLSffHKz7wHGAEYPcA0BILaTfhguI6YgLvdYgBeuOcsdJ0hh0Cwa8ff8RkBYNQ47z1XzANMQyusBeDlcOmWXErbX3MBWM9C9v3uArB9toXHCYDX9WwABbL6T4Jbtu88vD19u2Rx1Hycvf0xBOBlh+D7VbGSDUS0k9ny3HP6GjwzOsEDPLZtRWhZzngMMAVQIx5g5t3Uu1z0XgKonFtiAGAa0qHlAcayzZEDEwcA682/Hb+zNZU1bQrYrgLAosmkagBMtaNPXL4tqTlE5MqgexpUBg31sO888FWBMArAuFO1Y6QH07R0Tli2PwGA0RPbc/YepQt6PqduPBZ2ACxyf+EBmA9hokDtBACuNGQ5XL6tD8B25KPQNeNUDtG9yILQwQVgG43s5IWnd6P9oFFxeKVqfilpa1Z0Aqgyc6HHdsaWk0qclcjnfCcBsB2Zt3rwh7b7hgCwSPU8Zu+Y09fh6dFy9TVsLN7U6PKlAMwyuUVCILDq0lOReXR3p2cDowDc7bedXiWz8QCoxy8QD7BaIpfaddGnZQS8XVc9CQ6rKrb6foMETljZL1XypD4eYArA+Al7XDvrVSBafLceUMkA2+zONaCyTR5g3QVgQwc1AJ60Pg6G/i1/fbIKgI0cOgVgqzzAIgDcuFROWHHAGADTFwcj50j7Pj82CnacvKb8SUsWkV4/5fJmhAXva+sAU2AVub/wFUx5AP57zzl4b4ZcGAPzGz4V9ABTsLbSAywKwHYoErkALLbSXQAWs5OpXuEKwPQTMyYSIOhiw09pKDiPDQEYY6dembAZ8mRKLQGZXqM6oi9Wygd/hNADbIf2oh78oX2+XnYIfvB4gJkGqZ7d8Hf+86jIA0NtXLMALArcejbAz6hMz1jEA2w1AE+NOgYDF+yXTGMFAOM4qKuNYUXJkj4hjUtDIBBGS+ZKD+U8HuBgAPCcd2tApYL2hECIrFWr+9gJwNv7NYas6VIaPuRQeYAbReSAlQcveIEo1QFGz+eUKG8PsBUA/NzYKNhpMQBTYBW5v/AAzIcw0fHw69Ync8RCIOi47aoXBNRSpk3PWcT68mF8LgAbvqyCvoELwDaaPFwBmGbTUwDGN+7JUccBJcz0JGvUzEoB+KXK+bw8y8GIAY4cuBRueHSAQwXAXy09BKNXyyEQIqoFzI48AIt8MlSbA+qBNeIBFk26CyUAb/Vouvq7pCkAT+9UFeoU95byolqibBx/HmC1ffEAHJE7A5T9bKnU1S4Abvndetjv8QAnVgDOli4FRPeTY4CpBziQGOBwA+CGETlgFQFgPgQCPZ9YQpjGAFO7mX3c8QCspQtuxANMgVUkpEsPgLE8exdPpVEjAEwLQ4UCgLG4CxYrsas5mUPsOmfRcV0AFrWUiX5OXnj+3mrpJxkKwKJyQ1qmoqVUX66cH2ZGn1K6BhuADw5pLn2utrLpwR/uiwIwjT/UO459Z69Ln9pZswSAPeWNRUIgWJEHvePUswEt6KHmAX529AbYffo6sMx1NQ8w/eT93Yoj8O2Kw9JhGQVgLSkvmqiI4/ZtWQreqltE2ofe+WEfPgmuZC77AbjV9+th31k5BGLOuzWhUsHMwserN6eh/p3ZnILcTxuOwZCFsidftKKi2j0Pk+rwy5bRFioPcP2S2WHNoYvK4fIAjOCHtrEagFGBBpVoWBMBYCzVvPB97SRACsAiL9h8BVP+Cw4FYHy57zUnITaazi8vg6YHwHicGOuP94Cxq2OhUakc8MbUaJ8lwz/DKg5Zrnwx9be+XAA2evVZ198FYOts6TNSuAIwhcO+82LgV08IRKAAvPJAPHSaJt84XAAGKaNd9OFLZZrQfiJZ02pL28sDbAiAK0Hzsrl0rxY9QNQDYCwdOiv6lKSmkD9LGkkoH+WNaLMbgGlxDSsAGJOqytjsAX48ADgl4EsjNgrAgcigGbkG6RoUAWA9AFS7mPQ+t2PxEVTiYE0MgBPspnsBa3SwG4BFXrD1AHhxzDl41+MB9gfA+HKIL4msUV18NQ8wbxKqG0x/4wG4wuBlcPXOQ12T7x7QVCo/bldzMofYdc6i47oALGopE/2cvPD83WiPjWgJSZIkkc7YLgB+pUp++H3b4+0B3jWgCWRKI+Z9oqVacV5EdDPVATghBpdVcxLxAE9oVwmalgkcgGlBDzUPMH/MagBMZaoC8QD/+mY1qKUi58UDcL9WpeDNOuIeYBpGgeEIpXJngNID7A2BeOqH9bD3jOwBntulJlQskPg8wNnTp5TChrBN3nAMBns8wCt6JJTG9nebtssDjOWrb6uoQNgBwHWKZ4P1Ry5pAjCC36QNcZwHOHAARglGlGJkTasyJLWx3vlTj61IaXcegPkQJi8AfiESehF1DLoueACmsqChAGAjzwETGOJYNSoz52L1Ni4AW21RMl64AfA7dYvA23WLeCWFeAOw2INGy6SrDsYrn45erZofftsaOgA+NLQ5pEwW/BAIWu7UyJv/wfM3oPkoWag/FAA8sX1laFI6p+7VoucBftwBGHVcx7er7GNHZjeEqX2Dm2v+Tn8omj0trPyovvSnp3/YADFnrkv/P69LTaiQyAEYE70GeZIZAwHgnf2bQGab/uiH9gAAIABJREFUQiD0AFDtYtLzAJsBYPrioHsBa3RgoUl2AfD4dpWgmc4LNl/BlA9hopUKec+4HgDjvfVQ/E0hPXtaOIOOa9YD7AKw2VUZ+HYuAAduQ80Rwg2A1ar7eMcAuwDsb7nowR9uO3LJQRi75qg0jBEAPnT+JjQbtc5SAGbzLeIBntS+MjS2AIBpRTsRD3DXGTtg4Z5zXma3ygM8481qqgUdqLQa7jhQD3Dp3Bmh1IAl0jm4AGz8hsvWJwU5bwCuC8VypNcdWA0urQDgYHqAaxXLClGxl5Vz5UEPv+pggiCNAbYDgL99uTw8VyGf3xc1PR1k6gEW+cKkD8DnofMvss68EQDG/iiFduf+I6FQBKsB2Owa1F3wng5O5hDRc7CrnwvAdlkWwNGfHtQeBnoALBprp2XS1QcvQMep26SfQ+0BPjy0BaRIJstWWdWCCcCjXn7SlBIHjcE1AsCTO1SGhhGBe4CtAGBadMVoCAQFp8QEwM+M3gB7Tsse4D/fqwVP5s8k/b/ImrRq/ds1DjuHHOlTwlZPCAT93C1allntnmfW+3Yk/iY0+VZ+IQ0mANcokhU2xRkDYGo3s3NE1xeOYQUA0zkU+cLEAzAfwrRkbwIA8xXy6HnzIRBGbSIKwE8OXgbXBGKAXQA2OgPW9XcB2Dpb+ozk5DcvtYeBmrg5DYGwFoALwG9bZX1hbMFQgcBytNfvykkJ4QbAh+NvQlPPA9ffA0hvOZsF4CkdqkCDiBx6w+sCFy3pbNYDbBkAv1UNahb1LelrtQe4TJ6MENFf9gBjJa8J7a0PgaCfqOe/VwvKJ3IARhjq++deQLjr3qSE7rrkXwbYBuEGwNUKZ4Etx64o58t7OvHfEzkPsBUATGPMcedaL+D0uaLnAaYALPKFicbq4jHwALx033l4x1Np1E4ApqWT6cLjn2GiAGw2EVNo0TvcESd6Dnb1cwHYLss6fOGpAbCatI2lAHzoAnScwjzAwQdgqgN8ZFgLSO4pXGDVEhDxtn2x5CD8yEIgDOg/Uo+T3QDc8Os1EHfxtpdZpnSsAg1KBh+AsbITyhDRtr5XA0khApuXB7hPI8iRIZXf6aQe4N/eqg41imb16U/HxB/7P1UaOtUuLPUTmWM+Cc4F4MCuMGbznBlSwpY+chKcmaZ2zzMShkT3KeIB1quEpnYOejHAVQtlga3HZQDGPGVMeqOlkBGAJ60/JsWzsvZWncLQt1VpMyZTtjEDwKVzZ4BFH2jLoFEA/un1ytColP8vTDwA819wKADjs+yjP3arnnOgHmBRAKYOF3/GN1IQycwkOtkRZ+Z8rNzGBWArrcmN5eSFp3ajVcvspQAsqrepZdLVDgLg2GEtlMpdVi0BETiiAGykBCaVXcLj1crC1jsXmoSmFQJx4cY9+DvmnBQi0nfeXmlIvuqS1n70bLDn9DV4ZnSUtLmIB9gFYNnSatcrTYKjWfp/da0FkfkSXwhErgypYHOfRnpLXPN3uwA4XcpkcOv+Pz77tQOAEd6wmiJrvAcYPZ8T18UpANyyXC745qUnA9Y8pzJ7uG8RD7ARABYJsdID4GX7zsPbHg8w3h97zLIHgGnpZDrpvAdYFIBxDL0voKYXvcMdcYGclxXbJnoAHjp0KKxYsQLWrFmj2GvXrl3wwQcfQHR0NGTNmhW6desGH3/8sfL7v//+C4MGDYJJkybB1atXoXbt2jB27FgoVqyYIZuHGwCrxXVZCcBrDl2ADiH0ANMbUqgA+OjFW9Do67XwRBKAI8NaQlL8H4EWe+EWNP5mrdJTS4hebygKwCzkRQtaaVa1WtlgtX2xsdA7dWxEK58uTgLg39+uDtWL6HuABzxVGt4w6QHGUsjl8mWEkv3sDYGgOq0LutaW9smDs50PWb11F8jvbE3ZAsAGvsLQc6AhSVoxwHYAMMZ204IUPADjfWECAWCtrxxG54NWGsRttWQY6b1ED4BpEpyZEKsZXAiTC8Dqs+pkDjG6Dq3un6gBeNSoUdCjRw+oW7euAsCXL1+GiIgI+N///if9tnnzZujSpYsEuB07dpTsi/CL/54yZQrkzZsXevXqBXFxcbBv3z5IkUJMtxXHcfLCU/OG6AGwaMlRrUW69vBFeH3yVunnV6sGPwSCAvDR4eLwKXrR6Xk/2Tinr96B9KmSGyp/ycCZjRFsANaqmsbbRg+AY05fh6dHyxXtrPAAT1h3FIYvOiiNJ1JamoZAaAHw9yuPAFaDYy1QAEZvbIl+i6XhrIwBLpYjHaAEGDYXgPWvUlUPsAUAHEwPcGS+jEqyI54xD8Do+Ry/NsEDHEoARv3rxX5CILwA2ESIFQ/Ay/fHw1s/y4WW8FnWfaY9HmC+JDNbefwLJg2501uddr6cOplD9Oxi9++JEoDPnDkDb775Jqxfvx7y588POXPmVAB4xIgRMGbMGDh+/DgkS5ZMsm+fPn1g7ty5cPDgQXjw4AFky5YNRo4cCZ07d5Z+v3btGuTJkwcmT54Mr7zyivCcOHnhqT0M1D5rUQ+wC8D+p14UgIUXEOkYd/EWNPw6wQOslrAoMq5ZD7BW0QjbAfjXHVI4Bm00BvjOg3/g2dFRUDhbWtXkMv74KADPfLs6VFPxAIcLANMQiOfGRsHOk3KhgoXv14ayeROfBzh3xlSwqbe1IRBGwpDoWqIe4GACMCaWsZLXIgCs9ZIncq+gfVp8tx6wGA9rIh5gPQCmpZCndqwC9Q3mGPBwv2J/PLzpIADmlTP82dwFYKMr0pr+iRKAFyxYAL/99htg+MPgwYMl2GUhEC1btoRMmTLBjBkzFAtiiESTJk0gPj5e6lutWjU4dOgQlCiRkF2MYRCRkZGSZ1i0hRsAq93UKADTErSiNqD9sIRne4d4gOOGt4QnBMMPRM/VTgA+duk2NPgqIYxHTbJO5DipCgTzImsdN01S0ZIMMwrAe89ch6d+MOABVgHgDZ80gHyZ5SQ4bBgbyCoX6tmAVhATBeDPni4NHWuZS4LDEAhUZCje13oPMAXg58dGwQ4XgP1Ov9pLf7gBcESu9HDwfEKCG+8BRs/nuDUJHmCtNa53nfC/Nx+1zmu/VgDwwj1noeuMndKuzOQY8HBPARidOR/O3KV6moEmwfEV6dhOeIg9fuk2fPzHbjh3/R6cuXbXr8ldADa6Iq3pnygBmJqmQ4cOXgCMENuiRQv44osvlG779++HMmXKwLZt2+DkyZPQunVruHPnDqROnVrp89JLL0l/W7hwoablb9y4Afgfa+fOnYOqVavCqVOnIF8+X9Fwa6bQ3ChqD4PHCYBpuWdzFvTdyk4APnn5DtT9crWyU1bG2OixGwFg+omS/9yotV9mA3y3iFOJAbYDgI3YgALwrHdqQNXCWXw2/2HlEfiahEAECsAYt1nMBgCmIRCPAwDnyZgKNjrQA5w+ZTK4qZIEh+EKf3WtbWR5qiY70gFK5kzvpfDgZABGWF/yYV3N86cAbDTHAAflAXjlgXjoNE0OgcBn2Qe/hxaA2YnzqjJqBnEB2NBlYlnnxw6AMZGtTZs2kmeYNYzvLVq0qBQygR7gdu3awaNHj+CJJxIKJbRv3x7Onj0rJdRptYEDB0rxw3xzAVi2yPojF6HdT6GLAaa6jOEGwKeu3IE6IwkAt46El6rkN3wjMAvAorGEVgNwl1+3w6KY817nGfVpQ8ibKeHl1IgRKAD/0bkGVCmkD8ADny4NHQLwAFMAxnLSKPrPN2Y3/Jy+d1Azzd/pDxSAW/+4UVEH+LtbbUDpNWx2vpQZsXsgfdk54Jzj3Jttai/9MQObSvH4RptICIQdAIxzjgmxrOGLcK/Ze5R/o+cTZRaZDJrWS57R8+U9wN+/WgGeKZ/H7zrVA+AFu8/C+7/JHuDpnapCneLZdQ+LziHv3RYF4IoFMsHcLrV096XVgS/IwfppQSwfUqU2rgvApqcjoA0fOwAuV64cYBiEmgd4x44dUrLbCy+8oOoBvn//PsyfP1/T4OHuAVa7qdEQCFqC1syqowDcploBmLEluIUwKADbccOxEzbOXrsLNT9flfDgMwnAKKGEsIRNLwSCxuhZBcD7zl6HVt+Lh0AkBgCuUCAzFO2zSLK5XQD8wo8bIdojj7WoWx0onSeDC8DcTcpKAKalybVigO0A4CLZ0kLcpQSN7mABcLNv13l5nq0A4L92n4VuHgA2mmSLU8sD8KqD8fDGVH0PcLABmP+i5AKwGXqwZ5vHDoARflH6bPr06YpF1WKAY2NjJa8waxgDXL58eSmBTrSFWwzwD69WgKe5t3orAXjDkUvw2k9bJPO5ACy6iuR+8TfuQbXhK5WNvmhdDl6uUsDYIACSl5ABMNMS1gL3QAAY5d1QaYNvTgJgjM+tHAQPcMUCmaGIDQBMY4BdANa/FNQAGL3tCLBGGwXgYIZAFMyaBk5cvqMcLh8CgZ/+x65O8ABrfeUwer48AKs9K3BMamM9DzAFYKNJtrgv3rstCsCBxgDzesTMllpOldGrjsBXyxJUZVwANrr67Ov/2AHw559/DuPGjYOjR49C0qRJJcv27t0b5s2bJ6lAoJc3R44c8M0330CnTp2k3x8XFYjRbSrAU5Hen7UoANMStGaWpAvAZqwmb3Px5n2oMiwh/CYYAExj9ESzydkDUAuA95+9AS2/Xy+dk4gM2ru/bIfFe71DIDZ+2hDyWBACoQXA/ANr0DNl4PWahaRjFvHy00pwuA8KwI1L5YRJr1sTAkEB+MVxG2HbcblAAkpPYQa+6PGaX5XB2dLOEIhwA+AKBTIpah9ofd4DzAOw1ho3OnNNv10Lh+MTQi+sAGB6nRhNslUD4NUHL0DHqXKlUTtjgF0ANrp6nNv/sQPgCxcuSDrAzzzzjKTvu3XrVnj33XclKH799delmerbty+MHz9ekj0rVKgQ9OzZU4oNjomJSdQ6wGoA3O/PGPhlsxyqECgAR8VegraTXA+wmdvB5Vv3odLQBAD+/Ply8EpVMx7gK9D6x03SIeh5gGmMXmIE4Dnv1oBKBX1jgK0GYPQ4Fe4th0BYCcA0BvilcZuUErlLPqwDEbkSHwCLvDD5u7bUPMD7BjWDtGHkAcZqnFQOMVgA3OSbtXCExB5bDsBcUQuteaRzyHu3aaVROwGYf7Fkx6rlAR6zOha+XHrI723fjpA8tkMnf4k28yy0cpvHDoDReKj2gNXfdu7cCblz54aPPvoIunbtqtgVE+BQGxgLYdy9e1cqpIGhDwjDRpqTF57aw2BMm4rQKjK31ylaCcAbYy9BGxeAjSwhpe+1Ow/gycHLlX+PeL6cVEzEaNt+whwAi8op6XmAUUsUNUWxiQCNmgd4U++GkDujuSS4nzYcgyEL90v7FwXgwc+WgfY1zHuAXQA2ukq9+7M1JbJenATA5fNlhPkGVSD4UAP+fBCU6L2bB2CMzR2zKlaJ19Va40ZnBKtQ0uQ7NWcJD4Z6IRB/7jyjSJUZzTHAfZkF4EBjgF0ANrp6nNs/0QNwKE0fbgA8tm1FaFlOG4BpAQIzdrUCgOuWyA4omWOmhXMS3I17DyFy4LKQAbBoNnl4AXBNQDjlG++xCRSAMc6Y2cU2D/D4TbD12BXpVJZ+WBdK5kov/b9IyIaZaymY27BzyJ8lNazvZa0KxP7BzSBNCufEAGOVSFx/v209pWpiPQBGzyxuz7SCrQLgRl+vgaMXE5LvrAZgo1+Y0Dh8eMeaQxegwxT9EAgrAHjO9tOw/9wNwBdq1rS8uGPXxMLIJa4HOJj3DNF9uQAsaikT/cINgH9sWxFa2AnARy9Bm4nmQiCwEhrGgrapWgAypxUvR02nrcLgZXD1zkPpT3Z8crITNm7f/wfKfLZUOZ3hz5WTEgmNNuoBZqWvtY6bJqlYBcAHz9+A5qPEPcCdp2+HJfu8Y4Ct8gDP7VJTis9NDAD88vhNsMUDwMu614USOV0A5udV7auXWQCm69jKJDh2zGrHyu5b9Dc+CQ4BeLSXB1j9Jc/ofYMHYLWvhfwLF2oWL+2urQM8b+dppVyx0S9MuC8e7oMJwGrzpPVMQVm6L5bI5dq1mh3PI7YvJ3OI0XVodX8XgK22KBnPyQtP7QY77rWK0LysjR5gAsBtqxWAXw3IoFkxTeHsAb774BGUGrBEMcOw58pC22oFDZsl+vgVeGGcHAPMSl9rATBNUjEKwMmeSAKxKioQVgDw5t6NIFfGVIbPHTegIRCiADzk2TLQLoAQiGB4gLUAeMTiAzB+bZxm7LEpIwZ5Iyd6gL0AOFUyuHnvHx+rmAmBMArAfAgEemZ/WJkQAqG1xo1OYcOv10Ac8QBbAcBzd5yGHrN2S4di9P6iBsBrD1+E1z2VRjEUhEms8edqhQfYCACPW3sUPl/sArDRNReM/i4A22hlF4C9jbvp6GV4deJm6Y+JEYAL9/4b/vtPPmer3+jv//MISvZLAOCh/ysLr1UPHgCLyikxWNECYCofJRLTqeYBtgqA53WpCajRyzc+BMJaAM4Bk16v4rNPZjezhTAoAC/vXheKezzAWLZ1z+lrUmGMFMkSCvvYeNuzfOh3pkfD0n3xIFqNUOsA1F76DwxuDqlTyGpARpqTARg9wCwEwjIA/mqNl/6wWricBLLRp5TCHHoeYArARu8vMgB7e7cpAPtLgnMB2MhKT9x9XQC2cX7DD4ArQfOyubwsQpPgAo0BDjUA2x0CUaT33/CvTQD8z6N/lXK6OEFD/lcW2gURgEXllKwGYAY/dFFu6dMIcmYw5wGetD4Ohv59QBpOGICJrUXCXHgZNG8PsD0A/MqETbA5To4BXtGjLhTLIYdAJIaGlbeu3nkAWdOlDOh0rATgY5duQ4Ov1kjHkztjKjh3/Z7PsYXCA4ye2R9WHVEAWGuNGzVkQw6AtTzAD/75F0r0WywNrwfAGEf70R+yB9jo/UUNgNcdvgjtPR5gJwHw+LVHYYTrATa65ILS3wVgG80cbgA8vl0laFZGG4A3fNIA8mVOY9pim+MuwysTQucBDmcA5rUnzQLwtuNX4EUTIRBGH1BaHmBaQlbEA2wnAP/5Xi3AMsV88/EAhwEAvzphM2yKu5woAdj0DYfbUA2ADw5pDqmSG/cA49A9Zu6Ci7fuS5rLE9bFJWoARthH6GdNSwaNAnCJnOlgWfd6mtM3e/tp+NgDwKLJenQOee82rTTqJACesO4oDF/khkBYdR1bOY4LwFZakxsr3AB4QrtK0DRIAPxa9QKKvjCazeqQAbVpDWcAxvOhN3/6Wd7IEqYAjA+JZ5/Mq6kUQD2ZRh9QyZMmgSPDfCvBhQMA81nb9GXDqR5gbwCuB6gR7DZvC1gNwGx0THDCRCe+hcIDjKEJ369M8ABrveQZXRv1v1wNx0kFOq1SyA8f/QvF+8oeYGMALJasF44APHFdHAxbJH910mp2Pv+czCFG16HV/V0AttqiZDwnLzy1h4EeAEd92hDymqzAhWbZEncZXk7EHuCifRYBxlzaBfR0zqg0l5ElbASAqU6nVQB8JP4mNPl2nXTIuJZwTflrb/8cDcv2x3t12dqnEeSwIARi/nu1oLyKB5gHYBpv7VQAbjNxM2w8KnuAV35UD4pmdwGYX1ePAwCjks93BIC11riRewb2/WrpIRi9OlbZjL088+PQUC09AP4j+hT0nL1HGkI0VpnOIR/eQSuNOikJjoZduQBsdOXZ298FYBvtG24APLF9ZWhSOqeXRWgMcKAAjDqlL42XFQhCkQRntwc4mABME52MLGE6B3oeYCpTxCecaO2TPaC0PMCxF25C42+cAcB/da0Fkfl8QyDCEYDbTtoMUbEuAPu7FoIOwPkzAQKomSYqg8arQKCSz6gVCR5gqwD43sNH0GPWLlgUI0sSMgUZfwBcPEc6WN5DOwSCJsyJxiqLArCTQiBcADZzBQRnGxeAbbRzuAHwpPaVobEfAN74aUPIE4AHmMJXYgyBKNZnEfwTJA+w2U9mZgHYqIcmRdIn4PCwFj5XlyUA3LcR5EgfeBKcFgDzup1Ucs6pHmAKwFgut4jrAfZZe2pQeWhoc0iZzFwMMNuBZgiEAwBYa42beezRHA5WRp0fB7+AoSMAmxEAFg3VoHPIbxMVewnaeiqNugCcMDNO5hAz69DKbVwAttKa3FhOXnhqD4OfXq8MjUp5e4D7/7kXpm8+IZ1ZoABMP78nRgAu3ncRPHxkfwhEtnQpILpfE1Mr1wgAU5ki6wD4FmBZVWymQyAsAuAFXWtDuXwZfewYjgD82qQtsCH2knQuqz+uD4WzpTW1PhLzRo8HAFeCUSsOKyoQVgIwvXd89WJ5eKFSPp/lgoodRUQBeNsp6DVHDoEQ9VSHIwBT7XGt68usQ0PkenUyh4gcv519XAC20bpOXnhqD4PJHSpDwwjtEIhwB+CKQ5bDldsPpBm344YTLADOkjYF7Oj/eADwWz9Hw3I+BjiMAbhRRA74qYP1OsDtftoC64+4AOzvdv44ArDWS56Zxx4tooOhFy9Vzu8XgDERc4WfEIiZ207CJ3NipDFEQZ3OIQ/NG2MvQRsHeoAnbzgGgxfu1zR5kiQAx0a0MjMlQts4mUOETsDGTi4A22hcJy88MwAcSAlaNDO9gYbCA5xYADhzmuSwc0BTUyvXiAeY6nQajdHTCoE4evEWNPpa3AOsBsDb+jaG7OnNacLSeLyF79eGsnl9PcB85SZadjrQEAg9AMayujGDmvnMrdr1SgEDK2BhIQBsaz6uD4VcD7CQDQ8PbRFwgRAnhUCglOW3yxM8wFpr3MzNY/uJq9D6x43Spl+0LgcvV/EtxU7lGo0AsCio0+uAh+aNpNKovyS4CgUywbwu5mKzebuJ3A+mRB2DQQtcADaz5uzexgVgGy0cbgA8pUMVaBCRw8siNAnOSgDGIg4stMIujyw/tXYDcIm+i+HBo39t8zCzm23G1Mlh92f2AzDV6XQBWH7YizzwsPLaM6OjpP6seAzbzi4A7jhlK6w+JAPw2p71oWBWNwTCH6yw31wAFn8A7jx5FZ4bKwPw58+Xg1eqBgbAv289CZ/OlT3AoqAejgA8NeoYDPQDwE8kAYhzPcDiC9HCni4AW2hMfqiwA+COVaBBSW0ADqQELdpm+4kr0PpHWQXCBWDjC4/d/DOkSgZ7Bvp6CUVGpFJ0TMtTC+ooABtNUtHyAMddvAUNDXiA35wWDSsOeMugBdsDPOL5cvCq52EvAsA4D79uOSElV7E4SbZdw4gcMNlPCIRZD/Cb07bBigMXXAD2cxGoedGPDGsByZMGViJaywOMRVbwujHThFUgWkcqcbTs5eed6duVGGBRsBQ5xt2nrsGzY+QXO/pVROtFo2j2tLDyo/qaQ/+29ST0DgCAea8xrTTqJA+wC8Aiqys0fVwAttHu4QbAUztWgfocANMkOBeA/S8WLAGKlZCw2RFjzB6KWpAkspTNArDRJJUUyZ4A9K7xzUkA/He32lAmj28IBF+61AwAa0GBlQBMs+xpqMi6ng2gQFbzFRtF1lE49knsANyzWUl4r0ExaD5qnS0AvPfMdXjqhw3S1FNtbCsAWOta1Bob/+4PgP2pQAQ7BGLaxuPw2V/7NC8Z1wMcuruJC8A22t4FYG/j0hiyxBgDXLLfYrgfBABOlzIZ7FWJExVZyqEGYCynimVVsYmoQNjpAQ53AKYxlrRkNAu7EFkPj1MfNQCOHdYCkoWzB/iFSOjlKSbBXlIpAIuCpcg62Hf2OrT6XgZgf6XYmZ31PMAztpyEPvPkEAjR46RzyHu3qUybk2TQft50HAbM1wbgpE8kgaPDfatmisyJSB8nc4jI8dvZxwVgG63r5IWn9jCY9kZVqFciu5dFqAd4S59GkNNkBS4clMaQuQBsfOGxOUuTIinsH9zc+AAAQB8SeiEQtFKT0SztlMmegEMqHmDjAJzwaZ+dsFUhEIu61YHSeTL42JH3ANN4R9EQCC3PlZ4HuFDWNLCmZwOfY9JLguvy63alSIELwOqXRqIEYBICoQbAWmvczM3jwLkb0OK79dKm/ipRMjsXyZ4WVvkJgcAwob7z9krjiR6nPwDmX+67/bZT9TSD7QGevuk49HcB2MySs30bF4BtNHG4AfDPb1SFujYCME0OwphKjAFjzY6QAX5q7U6CC5YHOFXyJ+DgEN/wApGlTAH4h1crwNPl82gmdtFKTUaztLUA+Pil21DfkAfYWgCeuC4Ohi064PehO2HdURi+6KBizmAAMKpTYFIoZvFH5PKFcj0Afm/GDvh7zznpmF0AFgdg9LyhBy6Q5pQYYPaSSj3AomApcv6Hzt+EZqPkKo4Dny4NHWoVVt1MFIB/2XwC+v0pA/DiD+pAqdy+617rRRL/znuAQwHAtT5fBWeu3ZUOU+sZhtc1OpK0WrInkkCs6wEWWYKW93EB2HKTJgwYbgA8vVNVqFNc2wO8tU8jyBGAB5jGkL1SJT/8vu1UUAG40pDlcNlGHeCI/ovh3kP7Y4C14FJkKbsAnADAWg9dHoCp5JNdHmC9udMD4Pd/2wkLdp+VhtnwSQPIl9mNAfYHT+y3xA7AomCpt/7w9yPxN6HJtzIAD3iqNLxR2wVgrGw58K/90K5GQWhWJpeqGSnoq3VwAVhk9dnTxwVge+wqjeoCsLdx6Se0lyrng1nRp10ANrD+GARpKSyIDEUzpXU9wKRSk1Ue4BOXb0O9L43EAPt6gLcGUAiDeoBDAcANSmaHKR2rikyVVx81AKZJcB/8vhPm73IB2J9h1WxoJwAH8qndjAqEmgfYSgCmGt79nyoNnQIEYOoZXfJhHdUvH/5eYvi4YeoBDlYSnMiF7AKwiJVC08cFYBvtHm4A/EunalC7eDYvi9AY4EDAAwc9HH8Tmno8CCgPhTJbrAUjBMJuD3Cp/kvg7sNH0inZcT7soRiIx8AsAIvKKbFj1PJSGwXgTlO3wcqDsrwXa4GsQwrAWg9dOz3AVgIwTYLrPnMXzNt5RjJR1KcNpQTHMLlFAAAgAElEQVRDt3lbQA0q44a3hCdsCoFwAgCLgqXIWqHx+/1alYI36xRR3Uw0BILGxi79sC6UzJVe9zDoHPLhHU4FYBrrrHaCyZMmgSPD3CQ43cm3oYMLwDYYlQ0ZbgD865vVoFYx+wA49sItaPyNXAXs+Yp5Ye4O+YFtFzDyU5tYADiQrGEjAExLlRoFYK045ZOX70DdL1dLUyOiAmEnAGs9dCkk43GObB0JL1WRy74GGgJhFwD3mLVLuZ4CLVlu4y0xpEM/jgAsCpYiE0NfXvu0jIC36xb1D8DZ0sKqj7V1gO0E4GDpAIvYjapdYP936haB8evilE1dABaxoj19XAC2x67SqOEGwDPerAY1bQRg6kF4rkJexWPlArDYImQP8EB0IykAj25TAZ6K1E6Cu373IVQZtgJQd3hT70ZCJWPZMToVgKl3NxQA3LxMLhjXrpLYhJNeeiEQPf/YDX94vqi4AKxuXjUbHhvREpIksScJzgkeYCsB+NSVO1BnpPzy+mmLCOhcLzAApvJgy7rXhRI5jXmA+fCOUCTBiVzItOAH9ufhPJCQNpH9O5lDRI7fzj4uANtoXScvPLWHwYy3qkHNotoe4EDkp9DM1Pv37JN5lJjFxALApQcsgTsP7A+BCMReRgAY93Pz3kOpUlaq5EmFrhQ9AKYPUREP8BtTt8EqC0MgKABrPXR9PMAvRMJLlQPzAH/8x25YffCCVBksfxbjCWp6APzJ7D0wM1pOKg20ZLnQRIdhp8cRgEXBUmQ6T1+9A7W/kAG4V/OS0KV+MdXNlBAIHQ8wLRCxvHtdKJ5IAZiWfEaDYe4FJq2y5gKwyOqzp48LwPbYVRrVBWBv49Ib6FORuWGhR7YpEKAzMn12h0AEC4DL5s0AC9+vY+TUlb4bj16CNhO3SP/W8wCb2UFiAGCUJBv6tyyVhm2kBQCM4zz69z/Tklt6ANx77h74basMwIFWbDQz7+GwjZoNrYjV15JBC5UHGLV6MeEYm5UAfPbaXaj5+SppXFZ1Tm3emZ0LZ0sLq/2EQFAAXtGjLhTLYcwDzMc3i3qAAylRbWad01Ay3H5Mm4qAsoWsuQBsxqrWbOMCsDV2VB0l3AD4t7eqQ42iWb3OhSbBRfdrDNnSpTRtsXPX70KNEfINtFW53PB3jKxbis2KB5HegVUeuhwu3Xpg2/7KDFgCt230AK88EC9Jx6EEkRkvIp44BWC8EbeKzG06rtXfw08rBMJJHmAtr5NdAKy3Pv39rgfAfefFwK9bZF1tF4DVLRlsAK5YIBPM7VLL1LRbpQIh6lkVOcj4G/eg2vCVUtePm5aArg2Lq24mCsBTo47BwAX7pTEeJwAe27YidPmVALBG2XiRORHp42QOETl+O/u4AGyjdZ288NRusL+/XR2qF/EG4AHz98LPm05IVgoUgOkNtEXZXLB47/mgArDdHuCyny2FW/f/sQ2wrViqoQZg+hVAJASi45StsPrQRa9TD0QFgoZAiALwly9EwosBhkAEOnd6AGxlxcZAj9Wp2z+OACwKliJzduHmPag6TAbgHk1KQLdGgQHwlKhjMEgB4HqAqiZ6jc4hH98s6gEOxDOvd3xqv88icpL4+49tK8K7LgCbMaXl27gAbLlJEwYMNwCe+XZ1qGYjAF+8eV9KqsLWtHROWLY/3gVgG9ef2tDBAuDUyZPCgSG+5ZqdBMBacMB7gMMBgD+bvxemeV5UAy1YE+QlGbTdPS4ATEMgrATgS7fuQ+Wh8v27e+MS8EFj6wB45Uf1oGj2RArA0aeg1+w9yjof91pF6PxLggc4kMJGIhePkzlE5Pjt7OMCsI3WdfLCU3sY6AHw9n6NIWsAIRBXbj8ALEeMrXGpnLDiQHAB2O4QiLDwAMdegjaT5Bhg/BTXspw9IRBaAIxlQ7F8KLZQe4BX9FD3OvEA/NWL5QF1q7GZlUEL9Dajdr1SHeBBC/bBlKjj0m5cAFa3dmIEYKxS+MmcGOmE1QphaK1xM+vx8q37UMkDwB80Kg7dm5RQHUY0BGLyhmMweKEcArHqo3pQxCAA8/HNTvUA8zHA416rBJ1/2a7YzgVgM6vRmm1cALbGjqqjhBsAz+5cAyoXyuJ1LjQEIlAAvn7nIZQfvEwav1FEDq8CB4khBrjcZ0vhptNDIIIEwGlSJIX9g309wJYAcAAlucevPQojFh+U1mC4AjBKCP695xz80bkGlM+fSTqXIQv3w08bjskAHEClPBtvhyEfOrEDMKvW2HzUOjh4/qZkb1HPqsjkXL39ACp4HBjdGhaDHk1L+gXgQlnTwJqeDTSHxvWK6xYbJsth0pxeo3PIhzCJAnCwk+BoqAee3/h2leCd6S4A6811MH53AdhGK4cbAM95tyZUKphZE4B39G8CWdKmMG0xlNQqN1AGYCwIQGM7XQA2bVZDG24MMQDTTHIRD3CHKVthDR8DbBEAa8EBfTCjcb9+sTy0dpAHGLVrMdkyXcpkytwP+3s/TFwvA3CgcoWGFlQYdU6MAIxFWnrNkT+vMw8wDYGwEoCpA6Nrg2LwcTPrAHjNx/WhUCIF4DGrY+HLpYeUK2VCu0rwNgFgrYRhqy4tJ3OIVedodhwXgM1aTmA7Jy88tYfBvC41oUIB+wD4zoN/oPSApZLl6pbIDusOJyQ3JQoAHrgUbt5zdhJcVOwlaBuEEAgtD7ALwAI3DpUueqEXIxYfgPFr5epSLgCr2zjYAIzOBHQqmGmiKhBqIRAUgEVDC0SO8ca9hxDpcWB0qV8UejWPUN2MHbueB5iGGpkBYD6+2ake4K+WHoLRq2MVW01sXxne+jla+bcLwCKrz54+LgDbY1dp1HADYBTpx89DtNEQiEA9wPcePoKI/kuk4esUzwbrj1xSduUCsI0LkQxNARizkVvYFAOsBcBUCi/UHmAtOHC6B1jtWqFatIGqtQRnJQZ/L8EG4GDIoAUTgFHhBvMcsL1bvyh8ogPABbOmgbV+QiAoAK/tWR8KZjUWAhEuADx4wX6YHCV/ncH20+uVodO0BADWypew6gpxModYdY5mx3EB2KzlBLZz8sJTexiwT2haALyzfxPIHEAIxMNH/0Lxvoul4WsVywpRsZddABZYR1Z2CTcAfn3yVlhLvhSgLQJJ8qIxwFoATJNzcH/fvFQenq8oJ8HVGbkKTl25Cw0jcsDkDlWsnBq/Y+l5gL9cehDGrD4qjRForH7QTirIOwonAC7edxE8fPSfj4Xw5Yeehx4Ai8bWikzF7fv/QBkPAL9Trwj0blFKdTN2fHoATCsuruvZAApk1a+QSM+dj+F3qge499wYwHLIrE3uUBnemOoCsMias7uPC8A2WjjcAHjh+7WhbN6MXhahHuBAAfjff/+DIn0WSeNXL5IFNsddSVQAHDlwKdxweAjEhiOX4LWfZBUIOz3AaVMkhX0qSXDnr9+D6iNkLVERD7CdAKwFB/4AGLVQVx24AC0jc0OGVMltvHt4D60HwN8sOwTfr5I/s7oArD4twQbgQEIgqFwgPRt/AMyS4GgIhGhogchCvvvgEZQaIH/Be6duEejd0lkAvDnuMrwyYbN0fN+/WgG6kXLD9PyCnQT34e874c9dZ5VDmNKhCnScuk35t9bXMpE5EenjZA4ROX47+7gAbKN1nbzw1B4Gf3erDWXyaAPwrgFNIFMa80lwaGq236qFs8DWY8EG4BWAWpbY7Ai5KD9oGVy/+9C28a1Yqi4AJ6hAaMEBD8Dfvlwenqsge4BD1fQA+Nvlh+G7lUekw3NDIMIfgPEMXhy3EbYdv+p1MkY9wFYCMA1hs8IDTIvSrO/VQKi6Jb0O+AQ/pwIwgvhfuwkAd6wCHae4AByqeyndrwvANs5CuAHw4g/qQKncGbwsQj3AVgJwlUKZvW7udgApP7Uo4u4CcGg9wLQaoIgHuP3krV7JkjingYRAjFt7FD73yKBpwQEvWxQOAPzdiiPw7YrD0pIPxD423g5DPnQ4eYDRWC+N2wRbjyc4CdiLu5EQCNHYWtHJaTNxsySxhhJkWprw7PgKZEkD63ppy6DRcCQrANipIRAnLt+Gel+uUUw8xQVg0eVmez8XgG00cbgB8JIP60BELm0A3j2gKWRME9hnX3ZzxM+D208keDcSAwA/OXgZXLsTPh5grEjUvKw9hTC0QiCsAOBAQnEoAGvBQTgC8OhVR+CrZTIAb+7dCHJlTGXjnS08hw43ADbqAVYLgbAagP/77z8pNjlFsic0F4EZAN7wSQPIl9lYDDAfw+9UAEZDvTdjh6TdjW3aG1UBQ7tY07pXWnWVOZlDrDpHs+O4AGzWcgLbOXnhqT0M+Mo6eIrUA2wlAGMc1q5T1xQrugAssKAs6EJDIEIBwBdu3IOqw8VjgKkHuBUqVmRLAz2bqcsviZjHDACPevlJ+F+FvCLD29ZHLwSCao1u/LQh5MmU2rZjCdeBgw3AlQtmhtkmZdDQxkYBWE0HWDS5zMo5FQVgei2aAWA+ht/JAPzRrN0wZ8dpycw/v1EV8L7mArCVq87cWC4Am7Ob0FbhBsBqdePtAuDy+TLC7tPXXQAWWknWdVp/5CK0+0m++YYbAFvxkkQfulpwMDXqGAxcIFeowhYOAPzjmqOAUmjYRD8nW7eqwmOkcANgK0IgQrEWmJ3zZ0kN63s11FwcdM1GfdpQSorVa3QOeQB2agwwnlOv2bthVrQMwNM7VVXuwfhvLGizd1AzvVM3/buTOcT0SVm0oQvAFhlSbRgnLzy1h4Fa1SAvAP6sKWRMbU0IRLm8GSHmTHABuMqwFXDxpn1JcBUGL4OrDg+B8AbgStC8bC4vWaVAIZOtK62bOqooVB0m7gFu99MWRS860GOToJ/EAIsC8HevPAnPPulsD/Ce09fgmdFR0m1o36BmkJZUibPxFhdWQyd2AGYhEC2/Ww/7z90I2cuQGQAW/WpB55CP4XcyAPeeuwd+23pKmpNfOlVTlHhcAA7tLcQFYBvtH24ArCYLZRcAl86dQblJ4xRYATd6U2l3ElzFIcvhyu0H0mEE43z0zlft92ABcPqUySBGxauBLyD4IoJNJAnOTgDW8o7xHuBwAGC0J84tvqBG5vMuZmNmnSTGbVwADs6sigLw2DWxMHKJXCI4sQNw33kx8OsWWQv41zerKdU4XQAOzprU2osLwDbaP9wAWC0rngLwnoFNA9Y+ZTfHiFzppWxi1oIBjC4Ay5CUEAJhnwfYKgDed/Y6tPp+A6BqyB+dzZWVpZc49QBrAfC0jcfhs7/2KZuFCwDbeCtLFEMnFgAeunA/TNpwDEa2joT/4D/4ZE6MND9O8wDny5waNnyiHQJB49Y39W4IuTMaC4HgE/xEPcDl82eC+e/VCuqaPnXlDtQZuRpK5kwPnz1dGtp4ytHjQWjdK606QCdziFXnaHYcF4DNWk5gOycvPLWHgdon4c/m74Vpm05IZ2slAOON4FB8cAHY7hCISkOWw2WHe4DXHb6oJGCMey34AIwydPgiIuoBxn7oVc+QKhkkS6qdeS5wOUpdRGJlXQAWtWZ49UssAIxKDJduPYDs6VPCzG0n/QKwaHKZlTPJ7GwEgEWVS+gc8gAsmgQXCgBG+1678wDSpEgG0SeuQJuJcjEiF4CtXHnGx3IB2LjNhLcINwBW84jZBcAlcqaDw/G3FFsGwwPsAjBIZYWZBM/4dpWgWRl7YoC1vBpmAFj4ghPo6AKwgJESaZdgA3CgXy20kuDo9OgBsGhymZVTHiwA5h021AP8w6sV4H2NSnChAmBm441HL3kDcKpkEDPQTYKzcg2KjuUCsKilTPQLNwBWu1lSAI4Z2BTSB1j+ld0ci2ZPC0cv3nYB2MS6CmSTUAPw5Vv3oZJBD3Ag58tvawaAsazqM+XzWHkYhsfSk0EzPOBjuIELwMGZdGZnvRh/ql29pU8jyJlBX7uazqE/APZXCtkF4OCsg3DYiwvANs5SuAGwWhyWXQD8fsNi8MOq2EQFwJWHLpc+TWILhkfbzNJ1AVhfLuznTcdhwPyEGGAXgM2sNOdtkxgB+PetJ+HTudoxwKLJZVbOlhkAFq1eSOeQ/2IpGgOMGvR/BjkGmNrX9QBbudoCG8sF4MDs53frcANgtbdwCsCoVYjyVoG0g+dvwNZjV+Clyvkhov8SF4ADMaaJbSkAT2hXCZraFQKh8VkP43lRLQObnofIxOnpbiLiAXYBWNeMYdnBBeDgTJsoAP+w8gh8vdxY+W4rADjUHuBNRy/DqxM3K5OB+Q173BCI4CxObi8uANto9nADYLW3cKsBmJo72J917Y4BtltlwoqleuPeQ4gcuEwaipUUtnIe2FjpHQrAVHpJK0Hol80noN+fexVzYzzh024IhBXLL6RjJEYA1osBFlVXsHJiRAH4+5VH4BsGwH0bQY70xkIgzHqAXQC2crbDeywXgG2cv3AD4G19G0uZxbS5ACy+QMIBgPFssBgF/AeQwxNzF0wAvnr7AVQIoQdYBIBnRZ+CXrP3uAAsvvTDomdiBGC9EAhRdQUrJ1AUgCesOwrDF8nVC6P7NYZs6byfPWrHROeQf4ENlyQ43gOM2t27P2tq5RR4jeVkDrHtpAUHdgFY0FBmujl54ak9DLb3awxZuZsQBWCrK0xZCV4i81N12Aq4YGMluHABYN5WVs6Dngc4HAB4/q4z8MHvuxQzjW5TAZ6KdJPgRK4xJ/cJNgBXLZQFZnWuYdokIioQ4QzAt+//A8+M3gBFsqeDie0rC9lJFICdnATnArDQVAelkwvANpo53ACYfRLX8gCHOwDbHQJh9/h2LVU7AFgrrg21MJ8cHLoYYBEP8JK956DzLztcALZrwYVo3McRgEXVFaycElEPMO4TNY2TJEkivHs6h7xqkWgSnNNCIFwPsPD0W97RBWDLTZowYLgB8O4BTSFjmuReFqEe4P2Dm0lC3lY1K8FL5JjsBlS7Pcwi52imj5XzwMYKZwBeffACdJy6zQVgM4vJwdskdgBe+H5tKJs3I7T6fj3sO3tDmgmnA7DR5ZIYAJiCOp5/pjTJYdcANwTC6Fqwor8LwFZYUWOMsAPgz5oCvo1qeYDDHYDtBlS7x7drqQYTgK/feQjlB8tJeKFQgaAeYK0iAVGxl6AtKVU6pk1FaBWZ2y7zC41r5RwJ7TARdkrsAKxWCllUXszK6WZ2zpMxFWzs3cjKoYHOIS/xFi4eYBeALV0SAQ3mAnBA5vO/cbgBsFqhC+oBPjC4OaROkdQyiwX7oW43oFYbvgLib9yX7ONUHWC1ybNyHthYWp/1wgGAo49fgRfGbVJM5QKwZZd8SAd6XACYeoC3CqorWDkxTgBgJ1eC4wHYDYGwcvUZG8sFYGP2MtQ73ABYLcZ34F/7YOrG49J5uwDsf/pdAAbFQ6MJwHcfQvlBofMAj1kdC18uPSRNpJYHeM/pa/DM6Chlsse2rQgty7keYEM3Pwd2tguAv1p6CEavTijqw0492ElwLAQCE8v2nL4uHUZiBmBe4i1cVCB4ANaSjLTqEnIyh1h1jmbHcQHYrOUEtnPywlN7GKgBrgvAAhPt6eICsD4AUx3iUIRAiAAwFmtpPmq9C8DiSz8setoFwFTZhBqiauEsMOudAFQgxm+SigbRxn9ZoioQDICfGxsFO09ekzZTk7a0e7KYnf/f3plAR1FsffwaQgggq2ELW9jXAAKGVUSRsCiCggiy6YfyEBXfU+Q7LI9NJLjhxg4fuKCICooiCgIqsibsYSdAACHsIFsSSOA7t8LEztDJ9GS6umum/3XOO+dJem5V/+p21b9v36oqUySU1ktMgXDfOs2oAG5YoSgtGtRCNoZs7bsL4IIheWjXuPbS2qOyDpF20wYNQwAbBJWby1R2PL3JYO8b7Sk0b9YUB60A1vt7bri4fmPmp3cj7ZCdAtF0wko6eSlFNAUpEPp7W/qDAD589io9+O7vEMBGHio/ukaWAGYEerbtEsDdpq2jTUcuBLwAdg/YGM0B9rVffHX5jYfO0VMz/zkJLjRvEO19o4OvZrP9vco6RNpNGzQMAWwQVG4uU9nx9Abs/eM7UEhwUJZbDSQBLDtC+9KXW2jJjiQIYCKxmFJvc/fLKTco8vZJdKpGgI9fTKYWE1dlPgfTejWkDkiByM0QqNRvZArghyf9QQmnr2S5X1+FVvdcRoC1+wcbPWDCzI6yKgJ8OKZjli3UjArgFlXvoS+ea2rmLXtly10A85zLc6+sorIOkXXPRu1CABsllYvrVHY8vcngwJsdKG8eCOBcdLX4CX8KfWf5PoquXYpa1yiZWzOW/87MSLynRXD+IIDPXkklPtTEVSCALXdJKRXKFMDHzl+jyasS6IEaJWjQFxl7SNslgHvO3EDrD50TbdA73EgKXI1RqwSw+1c2oykQrWuUoE+ejZKNIVv77gI4OOguSpjQUVp7VNYh0m7aoGEIYIOgcnOZyo6nNxkcnNCR8gRl3ZQ8kCLAH688QO/dPnven1IUcuN73vxGhgDObm/LK6lpVHf0MtE8uyPA7tsouZhp0zT436b3bkjt69q7CK79B6tp78nL1L5OaZrep5E33YtrbxOQKYBdkE9cTKbmt78e2CWAe8/eSGsSztougEsXDqUNw+Vtg5ZbAfxwrVI0u5+xk+dkPDyc183RfVfhc0AOxzwioyphU2UdIu2mDRqGADYIKjeXqex4zWJWUtLfGfmqruL+SYn/PZAE8I30m7Rs10m6t0IxIb5QMghAAGf1hNS0dKox8pfMf1RBAPPXhdUHzhBP3gXzmXcYjZOeASsEcNLfydQsJiN9xgoBPD/2KA1bFC/qcy2C6zsnllbvPyP+bct/21LxgiGWdrOLs6oCuGNkaZray76XSHcBzJ0jMyCjsg6x1DF1KoMAltgDKjsef4pOPHuNJv6yh9YmZHwu03sItQJ43/j2lC/Yf/cBltjVfm3aSgF8NTWN6igeAebjWSsNW6qUAPZrB1Ok8VYL4CaVitMCH3aB0ApZF0L3MVpPAD87N5Z+2+dsAfxRz3tp8Pytup73WP1w4r/bVSCA7SJ/Z70QwBL7QmUB7LrtXrM3QABL9AF/MC1DABcrkJe26hzvqRXA5Yvnpz+HPmQpIu02aNmlQHCDqo/8ma6n3RRtm967EbWvW9rSdqIy8wlYLYB9jQAnnr1KrTW7kegFKfS2QXvu0zhasee0ALj1v22pmE0R4FKF89HG4Q+b2pE5jVVGF8E90bAsTerewNR2eWMMAtgbWnKvhQCWyNcfBPDTszbQuoPGIsB6u0T4gs9M4eVLO5z+WzP7wWUrOwF87Xoa1R6VkQNcOawgrRrS2lL8RgVw5OhldDk1DQLY0t6RW5kVAvjk3ynUNGaluBFfBTDbcG+zkQjwgM820fLdp0Qbto1qS0UL2JMCYacAzukkuKcal6e3utWT62w5WIcAtg39HRVDAEvsC38QwD1mrqcNhzI2W/eUAgEBLNFZbDRtlwCuXupuWv6fByy9c6MCuPH4X+nsleuibTP6NKJ2dRABtrSjJFRmtQD2NQUitwL4hXmb6eedJyGAs0mB6N20Ao3vEinBw4yZhAA2xsmKqyCAJVL2BwH81Iz1tPH2aUMQwBKdQWHTVgrg5OvpVGtUxgKzOuGF6afB91tKxqgAbh6zkk7cXiQKAWxpF0mrzCkC+MUvt9BPt/cj3z4qmooUyCuNqZ5hF+eShfJR7Ah7UiA4Ajxvw5HMuU3bzmdbRNDoTnUsZaKtLC7xPD05/Z9dILILPpnVQH/QIWbdq7d2IIC9JebF9f7geNpN0z0JYL19gr3AccelZgovX9rh9N+a2Q+eUiC0Arh++aK0+EVrjyTVCuD1wx6iMkX0dwPhk+D4RDguEMCB8YRYIYBPXUqhJhMyUiDsigDz4q8ftp8QbQhUAczbdfK2ndrivg9wi6phdN+bKyj95q0s1w1oVZmGd6xlm1NDANuG/o6KIYAl9oU/COAnp6+juMSMYzMhgCU6g8KmrRTAKTfSqeZ/MyLAjSsWo29faG4pGaMCuN37q2nfqcuibTP7NKJopEBY2k8yKrNaADevcg99+bxvJ455ygHWWwT3nwXb6LutxzME8OhocSqjlcWKCHChfMEUP7ZdjgK4U/1wmrB0D81cfSjLdYNaV6Gh7WtaiSRLXRDAtqGHALYSvT8IYO258RDAVnqHOnXJEMC89yjvQepetAK4aeXi9NWAZpaCmLzqAL27fL+oM6cIcKeP11D88b8hgC3tHbmVWS2AH6xRgub6eOKYJwGstw3akG+207eb/xIwd4yJpsKh9gjgEoXyUZykFAi99IrNR85T12kZqQWuvbv1BPDgNtXo1bbV5TpbDtYhgG1DDwFsJXp/EMBdp62jzUcQAbbSL1Sryy4BfH+1MPq8fxNLcRgVwNoXw1l9G1Pb2qUsbScqM59Ay7dW0V8XkrMYNvsAgtOXUijqdgqEGaf25UYA/++3O2jBpmMBLYArhRWk39x2kLl58xb1mLmBkm+k06JBzSlvniDdCPCQ6Or00kPVzHcwgxYhgA2CsuAypEBIhOwPAvjxqWtp69GLggIiwBKdQWHTdgngh2qWpDnP3GcpGaMCWLs9IASwpV0krbIDpy7T0IU7Mse77MY8XxqgFcBdGoTTBz18O3AhNwJ42KIdND82QwDHj4mmQgEYAc5uL18+xIbLXXy+MJGuAP6wRwPq3KCsL93s0283JZ6nblgE5xNDs34MAWwWSR07/iCAu0xZS9uOQQBLdAPlTVspgLXHDEfXLkUz+za2lI9RAdz/kzhauTfjMAEIYEu7SHplZvq7e2O1AtiM/WZzI4BHfh9P8zYcFU3bObYd3W3x0dmuNstIgViy4wSt3HOaxnSqY2h3i5ile2iGJgf4noIhIi0jKChDINtRIIDtoK5fJwSwxL7wBwHcefIa2v5XRq6jXgR45Z5T1P/TTeLvhyZ0NHXgkDkRSezWgDNtZj+4bPFEs1knB2H1H/QAABSfSURBVJhPV+NT1rg8ElmGpvRqaClPowJY+xl5dt/G9DBSICztJ5mVmenvdwjgyykU9WbGLhB9m1WkcZ3r+nQruRHAoxfvpE/XHwlIAewtTHcBzNuj8eI4OwsEsJ30s9YNASyxL/xBAD82eQ3tyEEA8yelJTuSqHzxAtSgfFFTacmciExtaIAbM7MfvBHAnRuE04c+fiL2tmu0AnjDsDZUukiorolJy/fRR6sSxN8ggL2lrPb1Zvp7TgLYjO22ciOAx/64i+auTRRN2zW2HRW0KQIcdnc+2jTS3H2AvfUsFQWwdrGe637MzkXXcvIHHeJtv5p1PQRwNiRv3rxJY8eOpdmzZ9OFCxeoZcuWNHXqVKpataph9v7geNrV7jIfQj1oMiciw52EC7Mct+qrD3gSwDfSb1K1ERkR4G6NytG7T9a3tAdOXEym5hNXUf68eUR+ZHCeIN36eRP9kd/vFH/7v36NqU0tLIKztKMkViZz3DmtiQC//FBVei26hk93khsBPH7Jbpq95jAEMBHF/LyHZvzxzzZoKkSAIYB9eiRM/TEEcDY4Wfyy4J07dy6VLVuWhg4dSocOHaJdu3ZRSIixs9X9QQA/+vGftPP4JUHBV/HjrWfKnIi8bYuTr1+x+xQN+mILvfJwNXrxQeMveDm91ITdHUKbRt65DZpWAPeMKk8xT9SzHP2Zy6lUICRPjpGx5btO0oDPN4u2YR9gy7tIaoUyx52Tf6dQ05iMFAgzdhvwJIC/33qc/r1gm6hv2b9bUY3ShUgb9bQzB1iJCDAEMPmDDpH6wOdgHAJYB87169cpLCyM3n77bRo4cKC44uLFixQeHk5z5syhHj16GOovf3C8F7/YQj/FJ0EAG+rRwL2IF6flC87j8w26JuyoiOL09cA79/hNS79JVW9HgPs0rUhvdPEtR9LnBmdjYPuxi9R5ylrx14lPRFKPqAqyqoJdiwnIFMCJZ69S63d/F3c08pFa9Nz9lX26O08CmHPqe87aQMUKhNCsvo3E7gdv/bKXpv1+UNRr50EY2b0E+wTEyx+P+3E3zVmbEQ3nokYE+ALx9qPaIjP45A86xMtuNe1yCGAdlLGxsdSkSRPat28fVa/+z4bZnAZRr149ERk2UvzB8fiT3Wtfb6f2dUtTryYVjdyWadesTThLH686QMM61CI+FhfF/wks2vIXLdpynCZ2jaRyxQrccUN8LGmV4UvFv7P4ZRGsYtFG8v7zcHURHUcJDALsf67jcc0WHtqjkM1InVl/8Bz9zydxdItu0Zx+91HzqmEeO2Hh5r/otW+2i+t2j2tHBUKCPf7GzAtcol0FAax9GVBVAIcXCaV1w9qY2QVZbPmDDpF28x4MQwDrAFq0aBF17dqVrl27Rvnz58+8onv37uLflixZoov10qVLxP9zlaSkJIqKiqJjx45RuXLl7Opj1AsCShFYEHeU9p28QsM71sw2B9fuBmsj1a5TpexuE+o3h4A2L9RsAcwtnLPmMJ27mkpDomtk7kdrTsuNWeEDIfgeI8IKWh7U4Ba6BPDTTSrQhMcjjTVa0lWc8z94/lbadPuwp58Gt6Q64UUk1WbMrGsdAl9dJ7wwTX66IfHBHrIKBHD2ZCGAddjMmzeP+vTpQ+np6RQU9M8imb59+9KJEydoxYoVukTHjBkjFs65FwhgWY827IKAPAKr95+hhNNX6JnmEaZu/yevxbBshADnoc/68xBxmk7jiOJGfoJrvCCw8/jftPrAGerXLMLyHSiya+aP208Qp4t0baRGIOqnHUnixDpeCCy7QABDAHvlYwsXLqRu3brpRoBTU1Np8eLFuvYQAfYKMy4GARAAARAAARCQSAACGALYK/dy5QAnJCRQlSpVMn/LOcD169enKVOmGLIHxzOECReBAAiAAAiAAAhIIAAdAgHslVtxlLdkyZI0adIk6t+/v/htoO4C4RUYXAwCIAACIAACIOA3BCCAIYC9dtYRI0bQjBkzxLZnERER9Prrr1NiYiLFx8cH1D7AXoPBD0AABEAABEAABPyCAAQwBLDXjsoL4IYPHy4OwkhOTqZWrVqJ1AcWw0YLHM8oKVwHAiAAAiAAAiBgNgHoEAhgs33KkD04niFMuAgEQAAEQAAEQEACAegQCGAJbuXZJBzPMyNcAQIgAAIgAAIgIIcAdAgEsBzP8mAVjmcLdlQKAiAAAiAAAiBARNAhEMC2PAhwPFuwo1IQAAEQAAEQAAEI4Bx9ACfBSXxEIIAlwoVpEAABEAABEACBHAlAhyACbMsjAsezBTsqBQEQAAEQAAEQQAQYEWC7ngIIYLvIo14QAAEQAAEQAAHoEESAbXkK4Hi2YEelIAACIAACIAACiAAjAmzXUwABbBd51AsCIAACIAACIAAdggiwLU8BHM8W7KgUBEAABEAABEAAEWBEgO16CiCA7SKPekEABEAABEAABKBDEAG25SlITEykSpUqUWxsLJUpU8aWNqBSEAABEAABEAABZxJISkqiqKgoOnz4MEVERDgTQjZ3jX2AJbpDXFyccDwUEAABEAABEAABELCLAAfi7rvvPruqV7JeCGCJ3ZKSkkLx8fFUokQJCg4OllKT6+0OUeaseMEle3cDG7DxdjCCz8Bn4DPeElDDZ9LS0ujMmTMUGRlJoaGh5t1EAFiCAPbzTkR+j34Hgkv2jg02YOPtsAefgc/AZ7wlAJ8xj5gcSxDAcrhaZhUTEwSwt84Gn8HEBJ/xlgB8xltiGGfgM976jNXXQwBbTdzk+jDIQAB761LwGUxM8BlvCcBnvCWGcQY+463PWH09BLDVxE2u79KlSzRp0iR69dVXqXDhwiZb919z4JJ934EN2Hj7ZMNn4DPwGW8JwGfMIybHEgSwHK6wCgIgAAIgAAIgAAIgoCgBCGBFOwbNAgEQAAEQAAEQAAEQkEMAAlgOV1gFARAAARAAARAAARBQlAAEsKIdg2aBAAiAAAiAAAiAAAjIIQABLIcrrIIACIAACIAACIAACChKAAJY0Y7x1KybN2/S2LFjafbs2XThwgVq2bIlTZ06lapWrerpp3799/Pnz9Pw4cNpyZIlxCvT69WrRxMnThT3z2Xbtm30yiuv0KZNm+iee+6hwYMH05AhQzLv2Snc9u/fTw0bNqTJkyfTM88843g2n332mfCTQ4cOUZUqVWjMmDH05JNPOp7LjRs3BIvPP/9cjCMNGjSgt956i5o3b+5oNuPHj6cVK1bQ77//njl2mDG2eLKh+uCsx+XHH3+kcePG0Z49eygsLEw8V/zf+fPnF7djZMz1dy58n3pstP35/PPP06+//kqJiYlezUeBwEZVv4YAVrVnPLSLxS8L3rlz51LZsmVp6NChYnLftWsXhYSE+OldeW52dHQ0nT59Wgg7PmJ6ypQp4iVgy5Yt4r9r1qxJXbp0EdvCbdiwgQYNGiQ4Pfvss8K4E7ixqGEBwy8B7B8sgM+dO+dYNvPmzRMM3nvvPXrkkUdo/vz5QvStWbOGqlev7lgu/DyMHj2aZs6cSZ9++ilVrlxZiN+vv/5aiJl8+fI5ks0HH3wgxo9WrVplCmAznh8jNjyPgPZdocflzz//pAcffFCIv27dulFCQgINGDCA2rRpI8YeI2Ouv3Phe9Rjo+2p77//nh5//HGqWLFiFgHsaT4KBDb2eaznmiGAPTNS7orr16+LN+23336bBg4cKNp38eJFCg8Ppzlz5lCPHj2Ua7MZDeLBtVq1arR27drMCNWtW7eEiOnZs6eIOLAg5jfs4OBgUSVHixctWkR79+4lp3Dje2bx/9tvv2UK4JiYGEeyYf9gYceT8zvvvJPphu3ataPWrVuL/3ayz3DEl8UKvxxw4a8qRYoUoW+//Zb4K4KT2Bw/fpyee+45YlFXvnx5KlWqVKYANuP58WTDjDFSho2cuPTu3VsEJJYvX55ZNb9wcsDhypUrdNddd3mcq/yVC99wTmxcQJKSkqh+/fpUu3ZtMTe5IsBG5iN/ZiPDF822CQFsNlEL7MXGxlKTJk1o3759Qvy5CqcBcEoARzwDsbDIX7dunZiwOTrlKpz20b59exEBL1q0KH355ZeZf+PPmG3btqVTp06JgSfQua1evVpEGvizWYUKFTIFcMeOHR3Jhl98atWqJb4Q3HvvvXc8Fk7l4gLBzxKf2LVs2TIh+t5//30aOXKk+JL08ssvO8pn+FM+fx3gaCZ/wufxwpUCYYafeLJRsmRJJYftnLhs3bqVgoKChMBzFRbAffr0oTNnzogx2dOY669c+H5zYsN/5xdwftmOjIykQoUK0SeffJIpgI3M4/7MRklndmsUBLA/9JJbGzmi2bVrV7p27VpmnhVf0r17d/FvnB/rlPLNN9+I+/7hhx9oxIgR1KFDB/EZ11V2795NderUobi4ODp69GhAc+MXBJ6IOGrw9NNPi+iLKwWCX4ycyIb9onPnziKq98YbbxBP2JUqVRIir1OnTuKF0YlcXM/Hzp076amnniJ+TvLkySPEDEd/H3vsMUez4ZQZrQA2w0882WjcuLHyw7Y7F/cGc1STBS/70ebNm8XXN09zVSBwYQ56bPiUVh6DOR2Nx2WtAHYSG1UdGwJY1Z7JoV2uN+z09HQx0LhK37596cSJE2LxhhMKp0Jw5JejWJxjxZFgFn4cvXEV16InFkA8oXFkIlC5cRoIRxy++uorcftaAexUNq5nhUUv57vyC8LChQvpzTffFAtS/vWvfznaZ/gF8sMPP6TXX39drCWYNWsWLViwgPhLwhNPPOFYNu5ixoznx5MN10JelcfunARwWloa9erVS4zF7D8shI3MVYHARU8A79ixg+6//37x8s0in9cdaAWwk9io6tMQwKr2TA7t4gmccxr1IsCpqam0ePFiP7wr75rM98hit2nTpiLizfm//JmJPxnpRYD5EziL4UDlxqv4hw0bRvHx8VSsWLE7BLBT2fCCLo5wclrQCy+8kOlkvBiOC38VcKrPHDlyROTUr1y5UkzUrsL/n9cYcM69U9m4Cz0znh9PNvRSdLwbFeVfnZ0Avnz5svgSx+sO+KWKv65wMTJXBQIXdwGckpJCHNHv16+feLnk4i6AncRGvmfmrgYI4Nxxs/VXrtwhnqB4SydX4QgCR7h44UogF94Bgrc64wgVv0W78oF5suatz1gMuopeDnAgcuOV2LyrgTY3+urVq+K/2Ud49bET2XDOeIsWLUQURhth411T+MUpIiLCkVz4+XClDyUnJ1NoaGjmM8MT9tKlSx3rM3rRPDPGFk82VM0B1s4legKYF3lxGtHhw4dF9JfHIlcxMlcFAhd3n/njjz/EItsCBQqIL3FcOD2Eo+T8bzNmzBAvnxwlz2k+ChQ2quoRCGBVeyaHdnGUlwdLzi/q37+/uNIJu0DwfU6bNk1sbcb7+/KCHW0KCO/zOn36dDp48KDIZ+TCUdHvvvtO7AIRyNx4NTILGW3hAZaj4RwB5cU9TmTDTHg1/6hRo7LsB80Tdt68ecVuIk7kwn7CO4U0a9aMNm7cSFFRUZmuw2lFLIj564pT2bgLPTPGFk82/GEqcufCe0fzM8S7h/zyyy/iK5y2GBlzA4GLuwDmcYfHZG356KOPRE40L6zkMYm3K/U0jwcKG1V9GwJY1Z7x0C5e8MVvkbztGUexOGrDOa78CTxQ9wHmbZnq1q1Ljz766B07XXAKBA+2vA8wL+DhCB9HH/izN0/i/CmKi5O4aXOAeasip7LhVf28ZSA/Lyz0OEeaBTF/+uetiZzKhQ8o4CgVr9bnFJFy5coRHxjCi3X4awJvH+dUNu5Cz4znx4gN1acjdy783/xyzeKXd1vRFt6XnQMRnsbcQODiLoD1+tE9BcLIfBQobFT1awhgVXvGQ7t4IRfv98orTPltkzdt59QHFsOBWiZMmCAGU73CApcXGPBuDxwd5tX+ZcqUoddee41eeumlzJ84iZtWADMAJ7PhryWcOsNRGZ6oeQN63h3C6Vw4gsc7YnA6CP9/juDxc/bAAw84mo3ep34znh9PNlQfu7Vc+AWqYMGCxPmueoVTIng+MjLm+juX3Apgp7BR1a8hgFXtGbQLBEAABEAABEAABEBACgEIYClYYRQEQAAEQAAEQAAEQEBVAhDAqvYM2gUCIAACIAACIAACICCFAASwFKwwCgIgAAIgAAIgAAIgoCoBCGBVewbtAgEQAAEQAAEQAAEQkEIAAlgKVhgFARAAARAAARAAARBQlQAEsKo9g3aBAAiAAAiAAAiAAAhIIQABLAUrjIIACIAACIAACIAACKhKAAJY1Z5Bu0AABEAABEAABEAABKQQgACWghVGQQAEQAAEQAAEQAAEVCUAAaxqz6BdIAACIAACIAACIAACUghAAEvBCqMgAAIgAAIgAAIgAAKqEoAAVrVn0C4QAAEQAAEQAAEQAAEpBCCApWCFURAAARAAARAAARAAAVUJQACr2jNoFwiAAAiAAAiAAAiAgBQCEMBSsMIoCIAACIAACIAACICAqgQggFXtGbQLBEAABEAABEAABEBACgEIYClYYRQEQAAEQAAEQAAEQEBVAhDAqvYM2gUCIAACIAACIAACICCFAASwFKwwCgIgAAIgAAIgAAIgoCoBCGBVewbtAgEQAAEQAAEQAAEQkEIAAlgKVhgFARAAARAAARAAARBQlQAEsKo9g3aBAAiAAAiAAAiAAAhIIQABLAUrjIIACIAACIAACIAACKhKAAJY1Z5Bu0AABEAABEAABEAABKQQgACWghVGQQAEQAAEQAAEQAAEVCUAAaxqz6BdIAACIAACIAACIAACUghAAEvBCqMgAAIgAAIgAAIgAAKqEoAAVrVn0C4QAAEQAAEQAAEQAAEpBCCApWCFURAAARAAARAAARAAAVUJQACr2jNoFwiAAAiAAAiAAAiAgBQCEMBSsMIoCIAACIAACIAACICAqgQggFXtGbQLBEAABEAABEAABEBACgEIYClYYRQEQAAEQAAEQAAEQEBVAhDAqvYM2gUCIAACIAACIAACICCFAASwFKwwCgIgAAIgAAIgAAIgoCqB/wcV0/LQz9QajAAAAABJRU5ErkJggg==\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# number of probes histogram\n",
    "fig, ax = plt.subplots()\n",
    "ax.plot(cand_prob_nums)\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "id": "7a4d0b43",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Screen probes by genome counts.\n",
      "-- filtering region:0, screen genome to keep 300 probes\n",
      "-- filtering region:1, screen genome to keep 300 probes\n",
      "-- filtering region:2, screen genome to keep 300 probes\n",
      "-- filtering region:3, directly append 242 probes\n",
      "-- filtering region:4, directly append 256 probes\n",
      "-- filtering region:5, screen genome to keep 300 probes\n",
      "-- filtering region:6, directly append 300 probes\n",
      "-- filtering region:7, screen genome to keep 300 probes\n",
      "-- filtering region:8, screen genome to keep 300 probes\n",
      "-- filtering region:9, directly append 264 probes\n",
      "-- filtering region:10, directly append 216 probes\n",
      "-- filtering region:11, directly append 212 probes\n",
      "-- filtering region:12, directly append 240 probes\n",
      "-- filtering region:13, screen genome to keep 300 probes\n",
      "-- filtering region:14, directly append 176 probes\n",
      "-- filtering region:15, screen genome to keep 300 probes\n",
      "-- filtering region:16, directly append 245 probes\n",
      "-- filtering region:17, directly append 203 probes\n",
      "-- filtering region:18, directly append 204 probes\n",
      "-- filtering region:19, screen genome to keep 300 probes\n",
      "-- filtering region:20, directly append 206 probes\n",
      "-- filtering region:21, directly append 255 probes\n",
      "-- filtering region:22, screen genome to keep 300 probes\n",
      "-- filtering region:23, screen genome to keep 300 probes\n",
      "-- filtering region:24, directly append 279 probes\n",
      "-- filtering region:25, directly append 238 probes\n",
      "-- filtering region:26, screen genome to keep 300 probes\n",
      "-- filtering region:27, directly append 149 probes\n",
      "-- filtering region:28, directly append 262 probes\n",
      "-- filtering region:29, directly append 199 probes\n",
      "-- filtering region:30, directly append 263 probes\n",
      "-- filtering region:31, directly append 215 probes\n",
      "-- filtering region:32, directly append 190 probes\n",
      "-- filtering region:33, screen genome to keep 300 probes\n",
      "-- filtering region:34, directly append 230 probes\n",
      "-- filtering region:35, directly append 203 probes\n",
      "-- filtering region:36, directly append 154 probes\n",
      "-- filtering region:37, screen genome to keep 300 probes\n",
      "-- filtering region:38, directly append 211 probes\n",
      "-- filtering region:39, directly append 237 probes\n",
      "-- filtering region:40, directly append 293 probes\n",
      "-- filtering region:41, directly append 22 probes\n",
      "-- filtering region:42, directly append 201 probes\n",
      "-- filtering region:43, directly append 235 probes\n",
      "-- filtering region:44, screen genome to keep 300 probes\n",
      "-- filtering region:45, directly append 187 probes\n",
      "-- filtering region:46, screen genome to keep 300 probes\n",
      "-- filtering region:47, directly append 213 probes\n",
      "-- filtering region:48, directly append 3 probes\n",
      "-- filtering region:57, directly append 60 probes\n",
      "-- filtering region:60, directly append 187 probes\n",
      "-- filtering region:62, screen genome to keep 300 probes\n",
      "-- filtering region:63, directly append 241 probes\n",
      "-- filtering region:64, directly append 199 probes\n",
      "-- filtering region:65, directly append 114 probes\n",
      "-- filtering region:66, directly append 127 probes\n",
      "-- filtering region:67, directly append 271 probes\n",
      "-- filtering region:68, directly append 226 probes\n",
      "-- filtering region:69, directly append 128 probes\n",
      "-- filtering region:70, directly append 120 probes\n",
      "-- filtering region:71, directly append 137 probes\n",
      "-- filtering region:72, directly append 267 probes\n",
      "-- filtering region:73, directly append 198 probes\n",
      "-- filtering region:74, directly append 173 probes\n",
      "-- filtering region:75, directly append 149 probes\n",
      "-- filtering region:76, directly append 121 probes\n",
      "-- filtering region:77, directly append 218 probes\n",
      "-- filtering region:78, directly append 142 probes\n",
      "-- filtering region:79, directly append 198 probes\n",
      "-- filtering region:80, directly append 247 probes\n",
      "-- filtering region:81, directly append 223 probes\n",
      "-- filtering region:82, directly append 8 probes\n",
      "-- filtering region:83, directly append 259 probes\n",
      "-- filtering region:84, directly append 191 probes\n",
      "-- filtering region:85, screen genome to keep 300 probes\n",
      "-- filtering region:86, directly append 237 probes\n",
      "-- filtering region:87, directly append 236 probes\n",
      "-- filtering region:88, directly append 204 probes\n",
      "-- filtering region:89, screen genome to keep 300 probes\n",
      "-- filtering region:90, directly append 118 probes\n",
      "-- filtering region:91, screen genome to keep 300 probes\n",
      "-- filtering region:92, directly append 271 probes\n",
      "-- filtering region:93, directly append 212 probes\n",
      "-- filtering region:94, directly append 178 probes\n",
      "-- filtering region:95, directly append 214 probes\n",
      "-- filtering region:96, directly append 258 probes\n",
      "-- filtering region:98, directly append 168 probes\n",
      "-- filtering region:100, screen genome to keep 300 probes\n",
      "-- filtering region:101, directly append 286 probes\n",
      "-- filtering region:102, directly append 240 probes\n",
      "-- filtering region:103, directly append 205 probes\n",
      "-- filtering region:104, directly append 119 probes\n",
      "-- filtering region:105, screen genome to keep 300 probes\n",
      "-- filtering region:106, directly append 244 probes\n",
      "-- filtering region:107, screen genome to keep 300 probes\n",
      "-- filtering region:108, directly append 213 probes\n",
      "-- filtering region:109, directly append 225 probes\n",
      "-- filtering region:110, directly append 230 probes\n",
      "-- filtering region:111, screen genome to keep 300 probes\n",
      "-- filtering region:112, directly append 167 probes\n",
      "-- filtering region:114, directly append 215 probes\n",
      "-- filtering region:115, directly append 224 probes\n",
      "-- filtering region:116, directly append 228 probes\n",
      "-- filtering region:117, directly append 276 probes\n",
      "-- filtering region:118, directly append 216 probes\n",
      "-- filtering region:119, directly append 221 probes\n",
      "-- filtering region:120, directly append 200 probes\n",
      "-- filtering region:121, directly append 295 probes\n",
      "-- filtering region:122, directly append 270 probes\n",
      "-- filtering region:123, directly append 182 probes\n",
      "-- filtering region:124, directly append 235 probes\n",
      "-- filtering region:125, directly append 251 probes\n",
      "-- filtering region:126, directly append 291 probes\n",
      "-- filtering region:127, directly append 222 probes\n",
      "-- filtering region:128, directly append 275 probes\n",
      "-- filtering region:129, directly append 216 probes\n",
      "-- filtering region:130, directly append 162 probes\n",
      "-- filtering region:131, directly append 190 probes\n",
      "-- filtering region:132, directly append 192 probes\n",
      "-- filtering region:133, screen genome to keep 300 probes\n",
      "-- filtering region:134, directly append 237 probes\n",
      "-- filtering region:137, screen genome to keep 300 probes\n",
      "-- filtering region:138, directly append 285 probes\n",
      "-- filtering region:139, directly append 174 probes\n",
      "-- filtering region:140, directly append 261 probes\n",
      "-- filtering region:141, directly append 74 probes\n",
      "-- filtering region:142, directly append 39 probes\n",
      "-- filtering region:143, directly append 186 probes\n",
      "-- filtering region:144, directly append 187 probes\n",
      "-- filtering region:145, directly append 178 probes\n",
      "-- filtering region:146, directly append 294 probes\n",
      "-- filtering region:147, screen genome to keep 300 probes\n",
      "-- filtering region:148, directly append 193 probes\n",
      "-- filtering region:149, screen genome to keep 300 probes\n",
      "-- filtering region:150, directly append 240 probes\n",
      "-- filtering region:151, directly append 3 probes\n",
      "-- filtering region:152, directly append 248 probes\n",
      "-- filtering region:153, screen genome to keep 300 probes\n",
      "-- filtering region:154, directly append 283 probes\n",
      "-- filtering region:155, directly append 252 probes\n",
      "-- filtering region:156, directly append 292 probes\n",
      "-- filtering region:157, directly append 255 probes\n",
      "-- filtering region:158, directly append 227 probes\n",
      "-- filtering region:159, directly append 231 probes\n",
      "-- filtering region:160, directly append 136 probes\n",
      "-- filtering region:161, directly append 134 probes\n",
      "-- filtering region:162, screen genome to keep 300 probes\n",
      "-- filtering region:163, directly append 234 probes\n",
      "-- filtering region:164, screen genome to keep 300 probes\n",
      "-- filtering region:165, directly append 244 probes\n",
      "-- filtering region:166, screen genome to keep 300 probes\n",
      "-- filtering region:167, directly append 217 probes\n",
      "-- filtering region:168, screen genome to keep 300 probes\n",
      "-- filtering region:169, directly append 238 probes\n",
      "-- filtering region:170, screen genome to keep 300 probes\n",
      "-- filtering region:171, directly append 272 probes\n",
      "-- filtering region:172, directly append 200 probes\n",
      "-- filtering region:173, directly append 173 probes\n",
      "-- filtering region:174, directly append 198 probes\n",
      "-- filtering region:175, screen genome to keep 300 probes\n",
      "-- filtering region:176, directly append 152 probes\n",
      "-- filtering region:177, directly append 279 probes\n",
      "-- filtering region:178, directly append 280 probes\n",
      "-- filtering region:179, directly append 230 probes\n",
      "-- filtering region:180, directly append 226 probes\n",
      "-- filtering region:181, directly append 248 probes\n",
      "-- filtering region:182, directly append 93 probes\n",
      "-- filtering region:183, directly append 204 probes\n",
      "-- filtering region:184, directly append 173 probes\n",
      "-- filtering region:185, screen genome to keep 300 probes\n",
      "-- filtering region:186, screen genome to keep 300 probes\n",
      "-- filtering region:187, directly append 239 probes\n",
      "-- filtering region:188, directly append 233 probes\n",
      "-- filtering region:189, directly append 191 probes\n",
      "-- filtering region:190, directly append 150 probes\n",
      "-- filtering region:191, directly append 274 probes\n",
      "-- filtering region:192, directly append 270 probes\n",
      "-- filtering region:193, screen genome to keep 300 probes\n",
      "-- filtering region:194, directly append 297 probes\n",
      "-- filtering region:195, directly append 199 probes\n",
      "-- filtering region:196, screen genome to keep 300 probes\n",
      "-- filtering region:197, directly append 251 probes\n",
      "-- filtering region:198, screen genome to keep 300 probes\n",
      "-- filtering region:199, directly append 275 probes\n",
      "-- filtering region:200, directly append 229 probes\n",
      "-- filtering region:201, screen genome to keep 300 probes\n",
      "-- filtering region:202, screen genome to keep 300 probes\n",
      "-- filtering region:203, directly append 190 probes\n",
      "-- filtering region:204, directly append 265 probes\n",
      "-- filtering region:205, directly append 241 probes\n",
      "-- filtering region:206, directly append 262 probes\n",
      "-- filtering region:207, screen genome to keep 300 probes\n",
      "-- filtering region:208, directly append 174 probes\n",
      "-- filtering region:209, directly append 180 probes\n",
      "-- filtering region:210, screen genome to keep 300 probes\n",
      "-- filtering region:211, screen genome to keep 300 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:212, screen genome to keep 300 probes\n",
      "-- filtering region:213, directly append 200 probes\n",
      "-- filtering region:214, directly append 219 probes\n",
      "-- filtering region:215, directly append 142 probes\n",
      "-- filtering region:216, screen genome to keep 300 probes\n",
      "-- filtering region:217, directly append 278 probes\n",
      "-- filtering region:218, directly append 252 probes\n",
      "-- filtering region:219, directly append 258 probes\n",
      "-- filtering region:220, directly append 214 probes\n",
      "-- filtering region:221, directly append 256 probes\n",
      "-- filtering region:222, directly append 292 probes\n",
      "-- filtering region:223, screen genome to keep 300 probes\n",
      "-- filtering region:224, screen genome to keep 300 probes\n",
      "-- filtering region:225, directly append 234 probes\n",
      "-- filtering region:226, directly append 220 probes\n",
      "-- filtering region:227, directly append 242 probes\n",
      "-- filtering region:228, directly append 148 probes\n",
      "-- filtering region:229, directly append 197 probes\n",
      "-- filtering region:230, directly append 166 probes\n",
      "-- filtering region:232, directly append 23 probes\n",
      "-- filtering region:233, directly append 196 probes\n",
      "-- filtering region:234, directly append 255 probes\n",
      "-- filtering region:235, directly append 233 probes\n",
      "-- filtering region:236, directly append 182 probes\n",
      "-- filtering region:237, directly append 179 probes\n",
      "-- filtering region:238, directly append 251 probes\n",
      "-- filtering region:239, directly append 231 probes\n",
      "-- filtering region:240, directly append 234 probes\n",
      "-- filtering region:241, directly append 272 probes\n",
      "-- filtering region:242, directly append 288 probes\n",
      "-- filtering region:243, directly append 165 probes\n",
      "-- filtering region:244, screen genome to keep 300 probes\n",
      "-- filtering region:245, directly append 238 probes\n",
      "-- filtering region:246, directly append 205 probes\n",
      "-- filtering region:247, directly append 234 probes\n",
      "-- filtering region:248, directly append 277 probes\n",
      "-- filtering region:249, directly append 242 probes\n",
      "-- filtering region:250, directly append 172 probes\n",
      "-- filtering region:251, screen genome to keep 300 probes\n",
      "-- filtering region:252, directly append 287 probes\n",
      "-- filtering region:253, directly append 188 probes\n",
      "-- filtering region:254, directly append 282 probes\n",
      "-- filtering region:255, directly append 225 probes\n",
      "-- filtering region:256, directly append 218 probes\n",
      "-- filtering region:257, directly append 288 probes\n",
      "-- filtering region:258, screen genome to keep 300 probes\n",
      "-- filtering region:259, directly append 160 probes\n",
      "-- filtering region:260, directly append 162 probes\n",
      "-- filtering region:261, directly append 159 probes\n",
      "-- filtering region:262, directly append 179 probes\n",
      "-- filtering region:263, screen genome to keep 300 probes\n",
      "-- filtering region:264, directly append 259 probes\n",
      "-- filtering region:265, directly append 220 probes\n",
      "-- filtering region:266, directly append 251 probes\n",
      "-- filtering region:267, directly append 263 probes\n",
      "-- filtering region:268, directly append 175 probes\n",
      "-- filtering region:269, screen genome to keep 300 probes\n",
      "-- filtering region:270, screen genome to keep 300 probes\n",
      "-- filtering region:271, directly append 220 probes\n",
      "-- filtering region:272, directly append 232 probes\n",
      "-- filtering region:273, directly append 231 probes\n",
      "-- filtering region:274, screen genome to keep 300 probes\n",
      "-- filtering region:275, screen genome to keep 300 probes\n",
      "-- filtering region:276, screen genome to keep 300 probes\n",
      "-- filtering region:277, screen genome to keep 300 probes\n",
      "-- filtering region:278, directly append 224 probes\n",
      "-- filtering region:279, directly append 210 probes\n",
      "-- filtering region:280, screen genome to keep 300 probes\n",
      "-- filtering region:281, directly append 239 probes\n",
      "-- filtering region:282, directly append 262 probes\n",
      "-- filtering region:283, directly append 293 probes\n",
      "-- filtering region:284, directly append 215 probes\n",
      "-- filtering region:285, directly append 234 probes\n",
      "-- filtering region:286, directly append 187 probes\n",
      "-- filtering region:287, directly append 194 probes\n",
      "-- filtering region:288, directly append 249 probes\n",
      "-- filtering region:289, directly append 250 probes\n",
      "-- filtering region:290, directly append 253 probes\n",
      "-- filtering region:291, directly append 195 probes\n",
      "-- filtering region:292, directly append 226 probes\n",
      "-- filtering region:293, directly append 157 probes\n",
      "-- filtering region:295, directly append 209 probes\n",
      "-- filtering region:296, directly append 274 probes\n",
      "-- filtering region:297, directly append 194 probes\n",
      "-- filtering region:298, directly append 253 probes\n",
      "-- filtering region:299, directly append 141 probes\n",
      "-- filtering region:300, directly append 151 probes\n",
      "-- filtering region:301, directly append 25 probes\n",
      "-- filtering region:302, screen genome to keep 300 probes\n",
      "-- filtering region:303, directly append 213 probes\n",
      "-- filtering region:304, directly append 232 probes\n",
      "-- filtering region:305, directly append 222 probes\n",
      "-- filtering region:306, directly append 242 probes\n",
      "-- filtering region:307, directly append 231 probes\n",
      "-- filtering region:308, directly append 203 probes\n",
      "-- filtering region:309, screen genome to keep 300 probes\n",
      "-- filtering region:310, directly append 166 probes\n",
      "-- filtering region:311, directly append 267 probes\n",
      "-- filtering region:312, directly append 241 probes\n",
      "-- filtering region:313, directly append 182 probes\n",
      "-- filtering region:314, directly append 253 probes\n",
      "-- filtering region:315, directly append 228 probes\n",
      "-- filtering region:316, directly append 183 probes\n",
      "-- filtering region:317, directly append 251 probes\n",
      "-- filtering region:318, directly append 178 probes\n",
      "-- filtering region:319, directly append 249 probes\n",
      "-- filtering region:320, directly append 155 probes\n",
      "-- filtering region:321, directly append 279 probes\n",
      "-- filtering region:322, directly append 258 probes\n",
      "-- filtering region:323, directly append 248 probes\n",
      "-- filtering region:324, directly append 171 probes\n",
      "-- filtering region:325, directly append 176 probes\n",
      "-- filtering region:326, directly append 191 probes\n",
      "-- filtering region:327, directly append 106 probes\n",
      "-- filtering region:328, directly append 170 probes\n",
      "-- filtering region:329, directly append 295 probes\n",
      "-- filtering region:330, directly append 218 probes\n",
      "-- filtering region:331, directly append 154 probes\n",
      "-- filtering region:332, screen genome to keep 300 probes\n",
      "-- filtering region:333, directly append 214 probes\n",
      "-- filtering region:334, directly append 234 probes\n",
      "-- filtering region:335, directly append 271 probes\n",
      "-- filtering region:336, directly append 244 probes\n",
      "-- filtering region:337, directly append 181 probes\n",
      "-- filtering region:338, directly append 222 probes\n",
      "-- filtering region:339, directly append 257 probes\n",
      "-- filtering region:340, directly append 201 probes\n",
      "-- filtering region:341, directly append 284 probes\n",
      "-- filtering region:342, directly append 157 probes\n",
      "-- filtering region:343, directly append 260 probes\n",
      "-- filtering region:344, directly append 234 probes\n",
      "-- filtering region:345, directly append 225 probes\n",
      "-- filtering region:346, directly append 279 probes\n",
      "-- filtering region:347, directly append 202 probes\n",
      "-- filtering region:348, directly append 223 probes\n",
      "-- filtering region:349, directly append 161 probes\n",
      "-- filtering region:350, screen genome to keep 300 probes\n",
      "-- filtering region:351, screen genome to keep 300 probes\n",
      "-- filtering region:352, screen genome to keep 300 probes\n",
      "-- filtering region:353, directly append 158 probes\n",
      "-- filtering region:354, directly append 293 probes\n",
      "-- filtering region:355, directly append 234 probes\n",
      "-- filtering region:356, directly append 257 probes\n",
      "-- filtering region:357, directly append 170 probes\n",
      "-- filtering region:358, directly append 216 probes\n",
      "-- filtering region:359, directly append 179 probes\n",
      "-- filtering region:360, directly append 190 probes\n",
      "-- filtering region:361, directly append 175 probes\n",
      "-- filtering region:362, screen genome to keep 300 probes\n",
      "-- filtering region:363, screen genome to keep 300 probes\n",
      "-- filtering region:364, directly append 253 probes\n",
      "-- filtering region:365, directly append 284 probes\n",
      "-- filtering region:366, directly append 214 probes\n",
      "-- filtering region:367, directly append 102 probes\n",
      "-- filtering region:368, directly append 64 probes\n",
      "-- filtering region:370, directly append 158 probes\n",
      "-- filtering region:371, directly append 84 probes\n",
      "-- filtering region:372, directly append 236 probes\n",
      "-- filtering region:373, directly append 270 probes\n",
      "-- filtering region:374, directly append 224 probes\n",
      "-- filtering region:375, directly append 226 probes\n",
      "-- filtering region:376, directly append 222 probes\n",
      "-- filtering region:377, screen genome to keep 300 probes\n",
      "-- filtering region:379, directly append 287 probes\n",
      "-- filtering region:380, directly append 224 probes\n",
      "-- filtering region:381, directly append 278 probes\n",
      "-- filtering region:382, screen genome to keep 300 probes\n",
      "-- filtering region:383, directly append 176 probes\n",
      "-- filtering region:384, directly append 166 probes\n",
      "-- filtering region:385, directly append 272 probes\n",
      "-- filtering region:386, directly append 215 probes\n",
      "-- filtering region:387, directly append 192 probes\n",
      "-- filtering region:388, directly append 277 probes\n",
      "-- filtering region:389, directly append 170 probes\n",
      "-- filtering region:390, directly append 164 probes\n",
      "-- filtering region:391, directly append 257 probes\n",
      "-- filtering region:392, directly append 202 probes\n",
      "-- filtering region:393, directly append 229 probes\n",
      "-- filtering region:394, screen genome to keep 300 probes\n",
      "-- filtering region:395, directly append 256 probes\n",
      "-- filtering region:396, directly append 182 probes\n",
      "-- filtering region:397, directly append 245 probes\n",
      "-- filtering region:398, directly append 162 probes\n",
      "-- filtering region:399, directly append 196 probes\n",
      "-- filtering region:400, directly append 222 probes\n",
      "-- filtering region:401, directly append 237 probes\n",
      "-- filtering region:402, directly append 191 probes\n",
      "-- filtering region:403, screen genome to keep 300 probes\n",
      "-- filtering region:404, screen genome to keep 300 probes\n",
      "-- filtering region:405, directly append 205 probes\n",
      "-- filtering region:406, directly append 261 probes\n",
      "-- filtering region:407, screen genome to keep 300 probes\n",
      "-- filtering region:408, directly append 163 probes\n",
      "-- filtering region:409, directly append 221 probes\n",
      "-- filtering region:410, directly append 271 probes\n",
      "-- filtering region:411, directly append 282 probes\n",
      "-- filtering region:412, directly append 212 probes\n",
      "-- filtering region:413, screen genome to keep 300 probes\n",
      "-- filtering region:414, directly append 188 probes\n",
      "-- filtering region:415, directly append 252 probes\n",
      "-- filtering region:416, directly append 252 probes\n",
      "-- filtering region:417, screen genome to keep 300 probes\n",
      "-- filtering region:418, directly append 200 probes\n",
      "-- filtering region:419, screen genome to keep 300 probes\n",
      "-- filtering region:420, directly append 37 probes\n",
      "-- filtering region:421, directly append 212 probes\n",
      "-- filtering region:422, directly append 298 probes\n",
      "-- filtering region:423, screen genome to keep 300 probes\n",
      "-- filtering region:424, screen genome to keep 300 probes\n",
      "-- filtering region:425, directly append 221 probes\n",
      "-- filtering region:426, screen genome to keep 300 probes\n",
      "-- filtering region:427, directly append 217 probes\n",
      "-- filtering region:428, directly append 243 probes\n",
      "-- filtering region:429, directly append 193 probes\n",
      "-- filtering region:430, directly append 236 probes\n",
      "-- filtering region:431, directly append 237 probes\n",
      "-- filtering region:432, directly append 270 probes\n",
      "-- filtering region:434, directly append 5 probes\n",
      "-- filtering region:435, screen genome to keep 300 probes\n",
      "-- filtering region:436, directly append 294 probes\n",
      "-- filtering region:437, directly append 188 probes\n",
      "-- filtering region:438, directly append 206 probes\n",
      "-- filtering region:439, screen genome to keep 300 probes\n",
      "-- filtering region:440, screen genome to keep 300 probes\n",
      "-- filtering region:441, directly append 183 probes\n",
      "-- filtering region:442, directly append 276 probes\n",
      "-- filtering region:443, directly append 241 probes\n",
      "-- filtering region:444, screen genome to keep 300 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:446, directly append 108 probes\n",
      "-- filtering region:447, directly append 181 probes\n",
      "-- filtering region:448, directly append 127 probes\n",
      "-- filtering region:449, directly append 228 probes\n",
      "-- filtering region:450, directly append 268 probes\n",
      "-- filtering region:451, directly append 258 probes\n",
      "-- filtering region:452, directly append 197 probes\n",
      "-- filtering region:453, directly append 67 probes\n",
      "-- filtering region:454, directly append 145 probes\n",
      "-- filtering region:455, directly append 142 probes\n",
      "-- filtering region:456, directly append 217 probes\n",
      "-- filtering region:457, directly append 207 probes\n",
      "-- filtering region:458, directly append 236 probes\n",
      "-- filtering region:459, directly append 180 probes\n",
      "-- filtering region:460, directly append 202 probes\n",
      "-- filtering region:461, directly append 209 probes\n",
      "-- filtering region:462, directly append 170 probes\n",
      "-- filtering region:463, directly append 143 probes\n",
      "-- filtering region:464, directly append 211 probes\n",
      "-- filtering region:465, screen genome to keep 300 probes\n",
      "-- filtering region:466, directly append 189 probes\n",
      "-- filtering region:467, directly append 200 probes\n",
      "-- filtering region:468, directly append 222 probes\n",
      "-- filtering region:469, directly append 260 probes\n",
      "-- filtering region:470, directly append 131 probes\n",
      "-- filtering region:471, directly append 199 probes\n",
      "-- filtering region:472, directly append 203 probes\n",
      "-- filtering region:473, directly append 203 probes\n",
      "-- filtering region:474, directly append 235 probes\n",
      "-- filtering region:475, directly append 263 probes\n",
      "-- filtering region:476, directly append 236 probes\n",
      "-- filtering region:477, directly append 215 probes\n",
      "-- filtering region:478, directly append 189 probes\n",
      "-- filtering region:479, directly append 287 probes\n",
      "-- filtering region:480, directly append 257 probes\n",
      "-- filtering region:481, directly append 269 probes\n",
      "-- filtering region:482, directly append 230 probes\n",
      "-- filtering region:483, directly append 220 probes\n",
      "-- filtering region:484, directly append 265 probes\n",
      "-- filtering region:485, directly append 255 probes\n",
      "-- filtering region:486, directly append 294 probes\n",
      "-- filtering region:487, screen genome to keep 300 probes\n",
      "-- filtering region:488, directly append 3 probes\n",
      "-- filtering region:489, directly append 216 probes\n",
      "-- filtering region:490, screen genome to keep 300 probes\n",
      "-- filtering region:491, directly append 276 probes\n",
      "-- filtering region:492, directly append 229 probes\n",
      "-- filtering region:493, directly append 239 probes\n",
      "-- filtering region:494, directly append 293 probes\n",
      "-- filtering region:495, directly append 259 probes\n",
      "-- filtering region:496, directly append 167 probes\n",
      "-- filtering region:497, directly append 228 probes\n",
      "-- filtering region:498, directly append 262 probes\n",
      "-- filtering region:499, directly append 162 probes\n",
      "-- filtering region:500, directly append 142 probes\n",
      "-- filtering region:501, screen genome to keep 300 probes\n",
      "-- filtering region:502, directly append 269 probes\n",
      "-- filtering region:503, screen genome to keep 300 probes\n",
      "-- filtering region:504, directly append 220 probes\n",
      "-- filtering region:505, screen genome to keep 300 probes\n",
      "-- filtering region:506, screen genome to keep 300 probes\n",
      "-- filtering region:507, directly append 202 probes\n",
      "-- filtering region:508, directly append 221 probes\n",
      "-- filtering region:509, directly append 272 probes\n",
      "-- filtering region:510, screen genome to keep 300 probes\n",
      "-- filtering region:511, directly append 184 probes\n",
      "-- filtering region:512, directly append 197 probes\n",
      "-- filtering region:514, directly append 2 probes\n",
      "-- filtering region:515, directly append 20 probes\n",
      "-- filtering region:516, directly append 227 probes\n",
      "-- filtering region:517, directly append 262 probes\n",
      "-- filtering region:518, directly append 264 probes\n",
      "-- filtering region:519, directly append 188 probes\n",
      "-- filtering region:520, directly append 212 probes\n",
      "-- filtering region:521, directly append 265 probes\n",
      "-- filtering region:522, directly append 187 probes\n",
      "-- filtering region:523, directly append 152 probes\n",
      "-- filtering region:524, directly append 196 probes\n",
      "-- filtering region:525, directly append 216 probes\n",
      "-- filtering region:526, directly append 223 probes\n",
      "-- filtering region:527, directly append 255 probes\n",
      "-- filtering region:528, directly append 249 probes\n",
      "-- filtering region:529, directly append 269 probes\n",
      "-- filtering region:530, directly append 223 probes\n",
      "-- filtering region:531, directly append 201 probes\n",
      "-- filtering region:532, directly append 232 probes\n",
      "-- filtering region:533, directly append 201 probes\n",
      "-- filtering region:534, directly append 181 probes\n",
      "-- filtering region:535, directly append 116 probes\n",
      "-- filtering region:536, directly append 259 probes\n",
      "-- filtering region:537, directly append 231 probes\n",
      "-- filtering region:538, directly append 214 probes\n",
      "-- filtering region:539, directly append 242 probes\n",
      "-- filtering region:540, directly append 179 probes\n",
      "-- filtering region:541, directly append 248 probes\n",
      "-- filtering region:542, directly append 227 probes\n",
      "-- filtering region:543, screen genome to keep 300 probes\n",
      "-- filtering region:544, directly append 276 probes\n",
      "-- filtering region:545, directly append 267 probes\n",
      "-- filtering region:546, screen genome to keep 300 probes\n",
      "-- filtering region:547, directly append 44 probes\n",
      "-- filtering region:548, directly append 191 probes\n",
      "-- filtering region:549, directly append 206 probes\n",
      "-- filtering region:550, screen genome to keep 300 probes\n",
      "-- filtering region:551, directly append 90 probes\n",
      "-- filtering region:552, directly append 264 probes\n",
      "-- filtering region:553, directly append 49 probes\n",
      "-- filtering region:554, screen genome to keep 300 probes\n",
      "-- filtering region:555, directly append 241 probes\n",
      "-- filtering region:556, directly append 234 probes\n",
      "-- filtering region:557, directly append 273 probes\n",
      "-- filtering region:558, directly append 251 probes\n",
      "-- filtering region:559, directly append 221 probes\n",
      "-- filtering region:560, directly append 261 probes\n",
      "-- filtering region:561, directly append 240 probes\n",
      "-- filtering region:562, directly append 249 probes\n",
      "-- filtering region:563, directly append 193 probes\n",
      "-- filtering region:564, directly append 270 probes\n",
      "-- filtering region:565, directly append 218 probes\n",
      "-- filtering region:566, directly append 169 probes\n",
      "-- filtering region:567, directly append 229 probes\n",
      "-- filtering region:568, directly append 222 probes\n",
      "-- filtering region:569, directly append 292 probes\n",
      "-- filtering region:570, directly append 104 probes\n",
      "-- filtering region:571, screen genome to keep 300 probes\n",
      "-- filtering region:572, directly append 280 probes\n",
      "-- filtering region:573, directly append 206 probes\n",
      "-- filtering region:574, directly append 258 probes\n",
      "-- filtering region:575, directly append 204 probes\n",
      "-- filtering region:576, directly append 291 probes\n",
      "-- filtering region:577, directly append 207 probes\n",
      "-- filtering region:578, directly append 246 probes\n",
      "-- filtering region:579, directly append 151 probes\n",
      "-- filtering region:580, directly append 202 probes\n",
      "-- filtering region:581, directly append 250 probes\n",
      "-- filtering region:582, directly append 199 probes\n",
      "-- filtering region:583, directly append 181 probes\n",
      "-- filtering region:584, directly append 139 probes\n",
      "-- filtering region:585, directly append 241 probes\n",
      "-- filtering region:586, directly append 218 probes\n",
      "-- filtering region:587, directly append 156 probes\n",
      "-- filtering region:588, directly append 177 probes\n",
      "-- filtering region:589, directly append 275 probes\n",
      "-- filtering region:590, directly append 225 probes\n",
      "-- filtering region:591, directly append 283 probes\n",
      "-- filtering region:592, directly append 144 probes\n",
      "-- filtering region:593, screen genome to keep 300 probes\n",
      "-- filtering region:594, directly append 155 probes\n",
      "-- filtering region:595, directly append 180 probes\n",
      "-- filtering region:596, directly append 254 probes\n",
      "-- filtering region:597, directly append 277 probes\n",
      "-- filtering region:598, directly append 226 probes\n",
      "-- filtering region:599, screen genome to keep 300 probes\n",
      "-- filtering region:600, directly append 294 probes\n",
      "-- filtering region:601, directly append 299 probes\n",
      "-- filtering region:602, directly append 261 probes\n",
      "-- filtering region:603, directly append 251 probes\n",
      "-- filtering region:604, directly append 6 probes\n",
      "-- filtering region:605, directly append 187 probes\n",
      "-- filtering region:606, directly append 293 probes\n",
      "-- filtering region:607, directly append 281 probes\n",
      "-- filtering region:608, screen genome to keep 300 probes\n",
      "-- filtering region:609, screen genome to keep 300 probes\n",
      "-- filtering region:611, screen genome to keep 300 probes\n",
      "-- filtering region:612, screen genome to keep 300 probes\n",
      "-- filtering region:613, directly append 200 probes\n",
      "-- filtering region:614, directly append 277 probes\n",
      "-- filtering region:615, directly append 258 probes\n",
      "-- filtering region:616, directly append 173 probes\n",
      "-- filtering region:617, directly append 254 probes\n",
      "-- filtering region:618, directly append 192 probes\n",
      "-- filtering region:619, directly append 282 probes\n",
      "-- filtering region:620, directly append 196 probes\n",
      "-- filtering region:621, directly append 248 probes\n",
      "-- filtering region:622, directly append 247 probes\n",
      "-- filtering region:623, directly append 185 probes\n",
      "-- filtering region:624, directly append 132 probes\n",
      "-- filtering region:625, directly append 168 probes\n",
      "-- filtering region:626, directly append 118 probes\n",
      "-- filtering region:627, directly append 253 probes\n",
      "-- filtering region:628, directly append 273 probes\n",
      "-- filtering region:629, screen genome to keep 300 probes\n",
      "-- filtering region:630, directly append 37 probes\n",
      "-- filtering region:631, directly append 21 probes\n",
      "-- filtering region:632, directly append 9 probes\n",
      "-- filtering region:641, directly append 5 probes\n",
      "-- filtering region:643, directly append 1 probes\n",
      "-- filtering region:645, directly append 257 probes\n",
      "-- filtering region:646, directly append 241 probes\n",
      "-- filtering region:647, directly append 219 probes\n",
      "-- filtering region:648, directly append 215 probes\n",
      "-- filtering region:649, directly append 262 probes\n",
      "-- filtering region:650, directly append 241 probes\n",
      "-- filtering region:651, directly append 146 probes\n",
      "-- filtering region:652, screen genome to keep 300 probes\n",
      "-- filtering region:653, screen genome to keep 300 probes\n",
      "-- filtering region:654, directly append 271 probes\n",
      "-- filtering region:655, screen genome to keep 300 probes\n",
      "-- filtering region:656, directly append 292 probes\n",
      "-- filtering region:657, directly append 186 probes\n",
      "-- filtering region:658, screen genome to keep 300 probes\n",
      "-- filtering region:659, directly append 54 probes\n",
      "-- filtering region:660, directly append 278 probes\n",
      "-- filtering region:662, directly append 184 probes\n",
      "-- filtering region:663, directly append 205 probes\n",
      "-- filtering region:664, directly append 255 probes\n",
      "-- filtering region:665, directly append 266 probes\n",
      "-- filtering region:666, directly append 273 probes\n",
      "-- filtering region:667, directly append 172 probes\n",
      "-- filtering region:668, screen genome to keep 300 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:669, screen genome to keep 300 probes\n",
      "-- filtering region:670, screen genome to keep 300 probes\n",
      "-- filtering region:671, directly append 199 probes\n",
      "-- filtering region:672, directly append 171 probes\n",
      "-- filtering region:673, directly append 120 probes\n",
      "-- filtering region:674, directly append 258 probes\n",
      "-- filtering region:675, screen genome to keep 300 probes\n",
      "-- filtering region:676, screen genome to keep 300 probes\n",
      "-- filtering region:677, screen genome to keep 300 probes\n",
      "-- filtering region:678, screen genome to keep 300 probes\n",
      "-- filtering region:679, screen genome to keep 300 probes\n",
      "-- filtering region:680, directly append 294 probes\n",
      "-- filtering region:681, screen genome to keep 300 probes\n",
      "-- filtering region:682, directly append 147 probes\n",
      "-- filtering region:683, directly append 285 probes\n",
      "-- filtering region:684, directly append 253 probes\n",
      "-- filtering region:685, screen genome to keep 300 probes\n",
      "-- filtering region:686, directly append 266 probes\n",
      "-- filtering region:687, directly append 78 probes\n",
      "-- filtering region:688, directly append 278 probes\n",
      "-- filtering region:689, directly append 160 probes\n",
      "-- filtering region:690, directly append 289 probes\n",
      "-- filtering region:691, directly append 270 probes\n",
      "-- filtering region:692, directly append 189 probes\n",
      "-- filtering region:693, directly append 246 probes\n",
      "-- filtering region:694, directly append 236 probes\n",
      "-- filtering region:695, directly append 246 probes\n",
      "-- filtering region:696, directly append 170 probes\n",
      "-- filtering region:697, directly append 220 probes\n",
      "-- filtering region:698, directly append 227 probes\n",
      "-- filtering region:699, directly append 66 probes\n",
      "-- filtering region:701, directly append 289 probes\n",
      "-- filtering region:702, directly append 291 probes\n",
      "-- filtering region:703, directly append 181 probes\n",
      "-- filtering region:704, directly append 284 probes\n",
      "-- filtering region:705, directly append 292 probes\n",
      "-- filtering region:706, directly append 246 probes\n",
      "-- filtering region:707, directly append 182 probes\n",
      "-- filtering region:708, directly append 213 probes\n",
      "-- filtering region:709, directly append 289 probes\n",
      "-- filtering region:710, directly append 234 probes\n",
      "-- filtering region:711, directly append 290 probes\n",
      "-- filtering region:712, directly append 201 probes\n",
      "-- filtering region:713, directly append 236 probes\n",
      "-- filtering region:714, directly append 196 probes\n",
      "-- filtering region:715, screen genome to keep 300 probes\n",
      "-- filtering region:716, directly append 136 probes\n",
      "-- filtering region:717, directly append 282 probes\n",
      "-- filtering region:718, screen genome to keep 300 probes\n",
      "-- filtering region:719, screen genome to keep 300 probes\n",
      "-- filtering region:720, directly append 218 probes\n",
      "-- filtering region:721, directly append 270 probes\n",
      "-- filtering region:722, screen genome to keep 300 probes\n",
      "-- filtering region:723, directly append 73 probes\n",
      "-- filtering region:724, directly append 288 probes\n",
      "-- filtering region:725, directly append 271 probes\n",
      "-- filtering region:726, directly append 209 probes\n",
      "-- filtering region:727, directly append 138 probes\n",
      "-- filtering region:728, screen genome to keep 300 probes\n",
      "-- filtering region:729, directly append 200 probes\n",
      "-- filtering region:730, screen genome to keep 300 probes\n",
      "-- filtering region:731, directly append 174 probes\n",
      "-- filtering region:732, directly append 296 probes\n",
      "-- filtering region:733, directly append 274 probes\n",
      "-- filtering region:734, directly append 225 probes\n",
      "-- filtering region:735, directly append 206 probes\n",
      "-- filtering region:736, screen genome to keep 300 probes\n",
      "-- filtering region:737, directly append 252 probes\n",
      "-- filtering region:738, directly append 222 probes\n",
      "-- filtering region:739, directly append 229 probes\n",
      "-- filtering region:740, directly append 249 probes\n",
      "-- filtering region:741, directly append 266 probes\n",
      "-- filtering region:742, screen genome to keep 300 probes\n",
      "-- filtering region:743, screen genome to keep 300 probes\n",
      "-- filtering region:744, screen genome to keep 300 probes\n",
      "-- filtering region:745, screen genome to keep 300 probes\n",
      "-- filtering region:747, directly append 23 probes\n",
      "-- filtering region:748, directly append 254 probes\n",
      "-- filtering region:749, directly append 199 probes\n",
      "-- filtering region:750, directly append 287 probes\n",
      "-- filtering region:751, directly append 293 probes\n",
      "-- filtering region:752, screen genome to keep 300 probes\n",
      "-- filtering region:753, directly append 145 probes\n",
      "-- filtering region:754, screen genome to keep 300 probes\n",
      "-- filtering region:755, directly append 59 probes\n",
      "-- filtering region:756, screen genome to keep 300 probes\n",
      "-- filtering region:757, directly append 192 probes\n",
      "-- filtering region:758, directly append 141 probes\n",
      "-- filtering region:759, screen genome to keep 300 probes\n",
      "-- filtering region:760, directly append 220 probes\n",
      "-- filtering region:761, directly append 222 probes\n",
      "-- filtering region:762, directly append 141 probes\n",
      "-- filtering region:763, directly append 270 probes\n",
      "-- filtering region:764, directly append 261 probes\n",
      "-- filtering region:765, directly append 191 probes\n",
      "-- filtering region:766, directly append 254 probes\n",
      "-- filtering region:767, directly append 199 probes\n",
      "-- filtering region:768, directly append 278 probes\n",
      "-- filtering region:769, directly append 160 probes\n",
      "-- filtering region:770, directly append 81 probes\n",
      "-- filtering region:773, directly append 46 probes\n",
      "-- filtering region:774, directly append 202 probes\n",
      "-- filtering region:775, directly append 202 probes\n",
      "-- filtering region:776, screen genome to keep 300 probes\n",
      "-- filtering region:777, screen genome to keep 300 probes\n",
      "-- filtering region:778, directly append 252 probes\n",
      "-- filtering region:779, directly append 208 probes\n",
      "-- filtering region:780, screen genome to keep 300 probes\n",
      "-- filtering region:782, directly append 206 probes\n",
      "-- filtering region:783, directly append 286 probes\n",
      "-- filtering region:784, directly append 196 probes\n",
      "-- filtering region:785, directly append 295 probes\n",
      "-- filtering region:786, directly append 234 probes\n",
      "-- filtering region:787, directly append 293 probes\n",
      "-- filtering region:788, directly append 182 probes\n",
      "-- filtering region:789, directly append 209 probes\n",
      "-- filtering region:790, directly append 91 probes\n",
      "-- filtering region:791, directly append 164 probes\n",
      "-- filtering region:792, directly append 168 probes\n",
      "-- filtering region:793, directly append 231 probes\n",
      "-- filtering region:794, directly append 298 probes\n",
      "-- filtering region:795, directly append 126 probes\n",
      "-- filtering region:796, directly append 289 probes\n",
      "-- filtering region:797, directly append 249 probes\n",
      "-- filtering region:798, directly append 244 probes\n",
      "-- filtering region:799, directly append 289 probes\n",
      "-- filtering region:800, directly append 196 probes\n",
      "-- filtering region:801, screen genome to keep 300 probes\n",
      "-- filtering region:802, directly append 257 probes\n",
      "-- filtering region:803, directly append 289 probes\n",
      "-- filtering region:804, directly append 220 probes\n",
      "-- filtering region:805, directly append 168 probes\n",
      "-- filtering region:806, directly append 291 probes\n",
      "-- filtering region:807, directly append 233 probes\n",
      "-- filtering region:808, directly append 197 probes\n",
      "-- filtering region:809, screen genome to keep 300 probes\n",
      "-- filtering region:810, directly append 160 probes\n",
      "-- filtering region:811, directly append 234 probes\n",
      "-- filtering region:812, screen genome to keep 300 probes\n",
      "-- filtering region:813, directly append 187 probes\n",
      "-- filtering region:814, directly append 275 probes\n",
      "-- filtering region:815, screen genome to keep 300 probes\n",
      "-- filtering region:816, directly append 209 probes\n",
      "-- filtering region:817, directly append 204 probes\n",
      "-- filtering region:819, directly append 268 probes\n",
      "-- filtering region:820, directly append 255 probes\n",
      "-- filtering region:821, directly append 142 probes\n",
      "-- filtering region:822, directly append 257 probes\n",
      "-- filtering region:823, directly append 129 probes\n",
      "-- filtering region:824, directly append 288 probes\n",
      "-- filtering region:825, directly append 225 probes\n",
      "-- filtering region:826, directly append 263 probes\n",
      "-- filtering region:827, directly append 283 probes\n",
      "-- filtering region:828, screen genome to keep 300 probes\n",
      "-- filtering region:829, directly append 237 probes\n",
      "-- filtering region:832, directly append 168 probes\n",
      "-- filtering region:833, directly append 204 probes\n",
      "-- filtering region:834, screen genome to keep 300 probes\n",
      "-- filtering region:835, directly append 236 probes\n",
      "-- filtering region:836, directly append 164 probes\n",
      "-- filtering region:837, screen genome to keep 300 probes\n",
      "-- filtering region:838, directly append 226 probes\n",
      "-- filtering region:839, directly append 194 probes\n",
      "-- filtering region:840, directly append 264 probes\n",
      "-- filtering region:841, screen genome to keep 300 probes\n",
      "-- filtering region:843, directly append 148 probes\n",
      "-- filtering region:844, directly append 256 probes\n",
      "-- filtering region:845, directly append 300 probes\n",
      "-- filtering region:846, directly append 261 probes\n",
      "-- filtering region:847, directly append 174 probes\n",
      "-- filtering region:848, directly append 294 probes\n",
      "-- filtering region:849, directly append 123 probes\n",
      "-- filtering region:850, directly append 181 probes\n",
      "-- filtering region:851, directly append 241 probes\n",
      "-- filtering region:852, directly append 279 probes\n",
      "-- filtering region:853, directly append 194 probes\n",
      "-- filtering region:854, directly append 215 probes\n",
      "-- filtering region:855, directly append 199 probes\n",
      "-- filtering region:856, directly append 163 probes\n",
      "-- filtering region:857, directly append 300 probes\n",
      "-- filtering region:858, directly append 252 probes\n",
      "-- filtering region:859, directly append 240 probes\n",
      "-- filtering region:860, directly append 144 probes\n",
      "-- filtering region:861, screen genome to keep 300 probes\n",
      "-- filtering region:862, directly append 233 probes\n",
      "-- filtering region:863, directly append 282 probes\n",
      "-- filtering region:864, directly append 250 probes\n",
      "-- filtering region:865, directly append 223 probes\n",
      "-- filtering region:866, directly append 216 probes\n",
      "-- filtering region:867, directly append 164 probes\n",
      "-- filtering region:868, directly append 289 probes\n",
      "-- filtering region:869, screen genome to keep 300 probes\n",
      "-- filtering region:870, screen genome to keep 300 probes\n",
      "-- filtering region:871, screen genome to keep 300 probes\n",
      "-- filtering region:872, screen genome to keep 300 probes\n",
      "-- filtering region:873, directly append 265 probes\n",
      "-- filtering region:874, screen genome to keep 300 probes\n",
      "-- filtering region:875, directly append 212 probes\n",
      "-- filtering region:876, directly append 262 probes\n",
      "-- filtering region:877, directly append 268 probes\n",
      "-- filtering region:878, screen genome to keep 300 probes\n",
      "-- filtering region:888, directly append 215 probes\n",
      "-- filtering region:889, directly append 279 probes\n",
      "-- filtering region:890, directly append 205 probes\n",
      "-- filtering region:891, directly append 169 probes\n",
      "-- filtering region:892, screen genome to keep 300 probes\n",
      "-- filtering region:893, directly append 252 probes\n",
      "-- filtering region:894, screen genome to keep 300 probes\n",
      "-- filtering region:895, directly append 294 probes\n",
      "-- filtering region:896, directly append 158 probes\n",
      "-- filtering region:897, directly append 286 probes\n",
      "-- filtering region:898, directly append 182 probes\n",
      "-- filtering region:899, directly append 211 probes\n",
      "-- filtering region:900, directly append 251 probes\n",
      "-- filtering region:901, directly append 230 probes\n",
      "-- filtering region:902, directly append 271 probes\n",
      "-- filtering region:903, directly append 144 probes\n",
      "-- filtering region:904, screen genome to keep 300 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:905, screen genome to keep 300 probes\n",
      "-- filtering region:906, directly append 194 probes\n",
      "-- filtering region:907, directly append 143 probes\n",
      "-- filtering region:908, directly append 216 probes\n",
      "-- filtering region:909, directly append 181 probes\n",
      "-- filtering region:910, directly append 155 probes\n",
      "-- filtering region:911, directly append 187 probes\n",
      "-- filtering region:912, directly append 230 probes\n",
      "-- filtering region:913, directly append 169 probes\n",
      "-- filtering region:914, directly append 176 probes\n",
      "-- filtering region:915, directly append 166 probes\n",
      "-- filtering region:916, directly append 250 probes\n",
      "-- filtering region:917, directly append 250 probes\n",
      "-- filtering region:918, directly append 259 probes\n",
      "-- filtering region:919, directly append 157 probes\n",
      "-- filtering region:920, directly append 187 probes\n",
      "-- filtering region:921, directly append 173 probes\n",
      "-- filtering region:922, directly append 174 probes\n",
      "-- filtering region:923, directly append 141 probes\n",
      "-- filtering region:924, directly append 236 probes\n",
      "-- filtering region:925, directly append 214 probes\n",
      "-- filtering region:926, directly append 293 probes\n",
      "-- filtering region:927, screen genome to keep 300 probes\n",
      "-- filtering region:928, screen genome to keep 300 probes\n",
      "-- filtering region:929, directly append 261 probes\n",
      "-- filtering region:930, directly append 266 probes\n",
      "-- filtering region:931, directly append 249 probes\n",
      "-- filtering region:932, directly append 297 probes\n",
      "-- filtering region:933, directly append 224 probes\n",
      "-- filtering region:934, directly append 209 probes\n",
      "-- filtering region:935, screen genome to keep 300 probes\n",
      "-- filtering region:949, directly append 160 probes\n",
      "-- filtering region:950, directly append 178 probes\n",
      "-- filtering region:951, directly append 188 probes\n",
      "-- filtering region:952, directly append 190 probes\n",
      "-- filtering region:953, directly append 245 probes\n",
      "-- filtering region:954, directly append 90 probes\n",
      "-- filtering region:955, directly append 174 probes\n",
      "-- filtering region:956, directly append 265 probes\n",
      "-- filtering region:957, directly append 258 probes\n",
      "-- filtering region:958, screen genome to keep 300 probes\n",
      "-- filtering region:959, directly append 213 probes\n",
      "-- filtering region:960, directly append 279 probes\n",
      "-- filtering region:961, directly append 237 probes\n",
      "-- filtering region:962, directly append 226 probes\n",
      "-- filtering region:963, directly append 195 probes\n",
      "-- filtering region:964, directly append 149 probes\n",
      "-- filtering region:965, directly append 191 probes\n",
      "-- filtering region:966, directly append 161 probes\n",
      "-- filtering region:967, directly append 229 probes\n",
      "-- filtering region:968, directly append 228 probes\n",
      "-- filtering region:969, directly append 156 probes\n",
      "-- filtering region:970, directly append 110 probes\n",
      "-- filtering region:971, directly append 173 probes\n",
      "-- filtering region:972, screen genome to keep 300 probes\n",
      "-- filtering region:973, directly append 270 probes\n",
      "-- filtering region:974, directly append 209 probes\n",
      "-- filtering region:975, screen genome to keep 300 probes\n",
      "-- filtering region:976, directly append 194 probes\n",
      "-- filtering region:977, directly append 297 probes\n",
      "-- filtering region:978, directly append 130 probes\n",
      "-- filtering region:979, directly append 255 probes\n",
      "-- filtering region:980, directly append 173 probes\n",
      "-- filtering region:981, screen genome to keep 300 probes\n",
      "-- filtering region:982, directly append 171 probes\n",
      "-- filtering region:983, directly append 236 probes\n",
      "-- filtering region:984, directly append 269 probes\n",
      "-- filtering region:985, screen genome to keep 300 probes\n",
      "-- filtering region:986, directly append 233 probes\n",
      "-- filtering region:987, screen genome to keep 300 probes\n",
      "-- filtering region:988, screen genome to keep 300 probes\n",
      "-- filtering region:989, directly append 154 probes\n",
      "-- filtering region:990, directly append 281 probes\n",
      "-- filtering region:991, directly append 247 probes\n",
      "-- filtering region:992, directly append 237 probes\n",
      "-- filtering region:993, directly append 153 probes\n",
      "-- filtering region:994, directly append 286 probes\n",
      "-- filtering region:995, screen genome to keep 300 probes\n",
      "-- filtering region:996, directly append 255 probes\n",
      "-- filtering region:997, screen genome to keep 300 probes\n",
      "-- filtering region:999, screen genome to keep 300 probes\n",
      "-- filtering region:1000, screen genome to keep 300 probes\n",
      "-- filtering region:1001, screen genome to keep 300 probes\n",
      "-- filtering region:1002, screen genome to keep 300 probes\n",
      "-- filtering region:1003, screen genome to keep 300 probes\n",
      "-- filtering region:1004, directly append 283 probes\n",
      "-- filtering region:1005, screen genome to keep 300 probes\n",
      "-- filtering region:1020, directly append 52 probes\n",
      "-- filtering region:1021, directly append 11 probes\n",
      "-- filtering region:1023, directly append 217 probes\n",
      "-- filtering region:1024, directly append 242 probes\n",
      "-- filtering region:1025, screen genome to keep 300 probes\n",
      "-- filtering region:1026, directly append 17 probes\n",
      "-- filtering region:1027, directly append 3 probes\n",
      "-- filtering region:1028, directly append 1 probes\n",
      "-- filtering region:1029, directly append 235 probes\n",
      "-- filtering region:1030, directly append 209 probes\n",
      "-- filtering region:1031, directly append 290 probes\n",
      "-- filtering region:1032, directly append 275 probes\n",
      "-- filtering region:1033, directly append 234 probes\n",
      "-- filtering region:1034, directly append 183 probes\n",
      "-- filtering region:1035, directly append 201 probes\n",
      "-- filtering region:1036, directly append 130 probes\n",
      "-- filtering region:1037, directly append 237 probes\n",
      "-- filtering region:1038, directly append 227 probes\n",
      "-- filtering region:1039, directly append 246 probes\n",
      "-- filtering region:1040, screen genome to keep 300 probes\n",
      "-- filtering region:1041, directly append 254 probes\n",
      "-- filtering region:1042, screen genome to keep 300 probes\n",
      "-- filtering region:1043, directly append 122 probes\n",
      "-- filtering region:1044, directly append 253 probes\n",
      "-- filtering region:1045, screen genome to keep 300 probes\n",
      "-- filtering region:1046, directly append 263 probes\n",
      "-- filtering region:1047, screen genome to keep 300 probes\n",
      "-- filtering region:1048, directly append 296 probes\n",
      "-- filtering region:1049, directly append 176 probes\n",
      "-- filtering region:1050, directly append 267 probes\n",
      "-- filtering region:1051, screen genome to keep 300 probes\n",
      "-- filtering region:1052, directly append 135 probes\n",
      "-- filtering region:1053, screen genome to keep 300 probes\n",
      "-- filtering region:1054, screen genome to keep 300 probes\n",
      "-- filtering region:1055, directly append 268 probes\n",
      "-- filtering region:1056, screen genome to keep 300 probes\n",
      "-- filtering region:1057, directly append 155 probes\n",
      "-- filtering region:1058, directly append 131 probes\n",
      "-- filtering region:1059, directly append 164 probes\n",
      "-- filtering region:1060, screen genome to keep 300 probes\n",
      "-- filtering region:1061, screen genome to keep 300 probes\n",
      "-- filtering region:1062, directly append 240 probes\n",
      "-- filtering region:1063, directly append 266 probes\n",
      "-- filtering region:1064, screen genome to keep 300 probes\n",
      "-- filtering region:1065, screen genome to keep 300 probes\n",
      "-- filtering region:1066, directly append 207 probes\n",
      "-- filtering region:1067, directly append 204 probes\n",
      "-- filtering region:1068, directly append 147 probes\n",
      "-- filtering region:1069, directly append 160 probes\n",
      "-- filtering region:1070, directly append 250 probes\n",
      "-- filtering region:1071, directly append 291 probes\n",
      "-- filtering region:1072, directly append 259 probes\n",
      "-- filtering region:1073, directly append 263 probes\n",
      "-- filtering region:1074, directly append 273 probes\n",
      "-- filtering region:1075, directly append 256 probes\n",
      "-- filtering region:1076, directly append 293 probes\n",
      "-- filtering region:1077, directly append 88 probes\n",
      "-- filtering region:1078, directly append 241 probes\n",
      "-- filtering region:1079, screen genome to keep 300 probes\n",
      "-- filtering region:1080, screen genome to keep 300 probes\n",
      "-- filtering region:1081, directly append 298 probes\n",
      "-- filtering region:1082, directly append 263 probes\n",
      "-- filtering region:1083, screen genome to keep 300 probes\n",
      "-- filtering region:1085, screen genome to keep 300 probes\n",
      "-- filtering region:1086, directly append 28 probes\n",
      "-- filtering region:1087, screen genome to keep 300 probes\n",
      "-- filtering region:1088, directly append 25 probes\n",
      "-- filtering region:1089, directly append 207 probes\n",
      "-- filtering region:1090, directly append 261 probes\n",
      "-- filtering region:1091, directly append 226 probes\n",
      "-- filtering region:1092, screen genome to keep 300 probes\n",
      "-- filtering region:1093, directly append 203 probes\n",
      "-- filtering region:1094, screen genome to keep 300 probes\n",
      "-- filtering region:1095, directly append 2 probes\n",
      "-- filtering region:1096, directly append 14 probes\n",
      "-- filtering region:1097, directly append 230 probes\n",
      "-- filtering region:1105, directly append 223 probes\n",
      "-- filtering region:1106, directly append 189 probes\n",
      "-- filtering region:1107, directly append 267 probes\n",
      "-- filtering region:1108, directly append 295 probes\n",
      "-- filtering region:1109, directly append 184 probes\n",
      "-- filtering region:1110, screen genome to keep 300 probes\n",
      "-- filtering region:1111, screen genome to keep 300 probes\n",
      "-- filtering region:1112, screen genome to keep 300 probes\n",
      "-- filtering region:1113, directly append 266 probes\n",
      "-- filtering region:1114, directly append 217 probes\n",
      "-- filtering region:1115, directly append 253 probes\n",
      "-- filtering region:1116, directly append 184 probes\n",
      "-- filtering region:1117, directly append 288 probes\n",
      "-- filtering region:1118, directly append 231 probes\n",
      "-- filtering region:1119, screen genome to keep 300 probes\n",
      "-- filtering region:1120, directly append 228 probes\n",
      "-- filtering region:1121, directly append 298 probes\n",
      "-- filtering region:1122, directly append 227 probes\n",
      "-- filtering region:1123, directly append 267 probes\n",
      "-- filtering region:1124, directly append 201 probes\n",
      "-- filtering region:1125, directly append 208 probes\n",
      "-- filtering region:1126, screen genome to keep 300 probes\n",
      "-- filtering region:1127, directly append 241 probes\n",
      "-- filtering region:1128, directly append 235 probes\n",
      "-- filtering region:1129, screen genome to keep 300 probes\n",
      "-- filtering region:1130, screen genome to keep 300 probes\n",
      "-- filtering region:1131, directly append 272 probes\n",
      "-- filtering region:1132, directly append 290 probes\n",
      "-- filtering region:1133, directly append 205 probes\n",
      "-- filtering region:1134, screen genome to keep 300 probes\n",
      "-- filtering region:1135, directly append 268 probes\n",
      "-- filtering region:1136, directly append 294 probes\n",
      "-- filtering region:1137, directly append 196 probes\n",
      "-- filtering region:1138, directly append 222 probes\n",
      "-- filtering region:1139, directly append 256 probes\n",
      "-- filtering region:1140, directly append 242 probes\n",
      "-- filtering region:1141, screen genome to keep 300 probes\n",
      "-- filtering region:1142, directly append 263 probes\n",
      "-- filtering region:1143, directly append 206 probes\n",
      "-- filtering region:1144, directly append 122 probes\n",
      "-- filtering region:1145, directly append 215 probes\n",
      "-- filtering region:1146, directly append 94 probes\n",
      "-- filtering region:1147, directly append 257 probes\n",
      "-- filtering region:1148, directly append 101 probes\n",
      "-- filtering region:1152, directly append 194 probes\n",
      "-- filtering region:1153, directly append 269 probes\n",
      "-- filtering region:1154, directly append 54 probes\n",
      "-- filtering region:1155, directly append 254 probes\n",
      "-- filtering region:1156, screen genome to keep 300 probes\n",
      "-- filtering region:1157, directly append 276 probes\n",
      "-- filtering region:1158, directly append 20 probes\n",
      "-- filtering region:1160, screen genome to keep 300 probes\n",
      "-- filtering region:1161, directly append 3 probes\n",
      "-- filtering region:1162, directly append 222 probes\n",
      "-- filtering region:1163, directly append 272 probes\n",
      "-- filtering region:1164, directly append 3 probes\n",
      "-- filtering region:1165, directly append 270 probes\n",
      "-- filtering region:1166, screen genome to keep 300 probes\n",
      "-- filtering region:1167, directly append 192 probes\n",
      "-- filtering region:1168, screen genome to keep 300 probes\n",
      "-- filtering region:1169, directly append 185 probes\n",
      "-- filtering region:1170, directly append 232 probes\n",
      "-- filtering region:1171, directly append 268 probes\n",
      "-- filtering region:1172, screen genome to keep 300 probes\n",
      "-- filtering region:1173, directly append 96 probes\n",
      "-- filtering region:1174, directly append 128 probes\n",
      "-- filtering region:1175, directly append 44 probes\n",
      "-- filtering region:1176, directly append 212 probes\n",
      "-- filtering region:1177, directly append 241 probes\n",
      "-- filtering region:1178, screen genome to keep 300 probes\n",
      "-- filtering region:1179, directly append 197 probes\n",
      "-- filtering region:1180, directly append 213 probes\n",
      "-- filtering region:1181, directly append 271 probes\n",
      "-- filtering region:1182, directly append 291 probes\n",
      "-- filtering region:1183, screen genome to keep 300 probes\n",
      "-- filtering region:1184, screen genome to keep 300 probes\n",
      "-- filtering region:1185, directly append 4 probes\n",
      "-- filtering region:1186, directly append 265 probes\n",
      "-- filtering region:1187, directly append 285 probes\n",
      "-- filtering region:1188, directly append 77 probes\n",
      "-- filtering region:1189, directly append 171 probes\n",
      "-- filtering region:1190, directly append 187 probes\n",
      "-- filtering region:1191, directly append 194 probes\n",
      "-- filtering region:1192, directly append 242 probes\n",
      "-- filtering region:1193, directly append 279 probes\n",
      "-- filtering region:1194, directly append 291 probes\n",
      "-- filtering region:1195, directly append 288 probes\n",
      "-- filtering region:1196, directly append 277 probes\n",
      "-- filtering region:1197, directly append 184 probes\n",
      "-- filtering region:1198, directly append 30 probes\n",
      "-- filtering region:1199, directly append 2 probes\n",
      "-- filtering region:1203, directly append 191 probes\n",
      "-- filtering region:1204, screen genome to keep 300 probes\n",
      "-- filtering region:1205, screen genome to keep 300 probes\n",
      "-- filtering region:1206, directly append 108 probes\n",
      "-- filtering region:1207, directly append 213 probes\n",
      "-- filtering region:1208, directly append 208 probes\n",
      "-- filtering region:1209, directly append 169 probes\n",
      "-- filtering region:1210, directly append 270 probes\n",
      "-- filtering region:1211, screen genome to keep 300 probes\n",
      "-- filtering region:1212, directly append 278 probes\n",
      "-- filtering region:1213, directly append 293 probes\n",
      "-- filtering region:1214, screen genome to keep 300 probes\n",
      "-- filtering region:1215, directly append 221 probes\n",
      "-- filtering region:1216, directly append 216 probes\n",
      "-- filtering region:1217, directly append 190 probes\n",
      "-- filtering region:1218, directly append 235 probes\n",
      "-- filtering region:1219, screen genome to keep 300 probes\n",
      "-- filtering region:1220, screen genome to keep 300 probes\n",
      "-- filtering region:1221, screen genome to keep 300 probes\n",
      "-- filtering region:1222, directly append 179 probes\n",
      "-- filtering region:1224, directly append 231 probes\n",
      "-- filtering region:1225, directly append 282 probes\n",
      "-- filtering region:1226, directly append 238 probes\n",
      "-- filtering region:1227, directly append 215 probes\n",
      "-- filtering region:1228, directly append 167 probes\n",
      "-- filtering region:1229, directly append 235 probes\n",
      "-- filtering region:1230, directly append 223 probes\n",
      "-- filtering region:1231, directly append 241 probes\n",
      "-- filtering region:1232, directly append 237 probes\n",
      "-- filtering region:1233, directly append 223 probes\n",
      "-- filtering region:1234, directly append 169 probes\n",
      "-- filtering region:1235, directly append 241 probes\n",
      "-- filtering region:1236, directly append 271 probes\n",
      "-- filtering region:1237, directly append 228 probes\n",
      "-- filtering region:1238, directly append 287 probes\n",
      "-- filtering region:1239, directly append 249 probes\n",
      "-- filtering region:1240, directly append 297 probes\n",
      "-- filtering region:1241, screen genome to keep 300 probes\n",
      "-- filtering region:1242, directly append 279 probes\n",
      "-- filtering region:1243, directly append 297 probes\n",
      "-- filtering region:1244, directly append 260 probes\n",
      "-- filtering region:1245, directly append 283 probes\n",
      "-- filtering region:1246, screen genome to keep 300 probes\n",
      "-- filtering region:1247, directly append 234 probes\n",
      "-- filtering region:1248, directly append 160 probes\n",
      "-- filtering region:1249, directly append 237 probes\n",
      "-- filtering region:1250, screen genome to keep 300 probes\n",
      "-- filtering region:1251, directly append 181 probes\n",
      "-- filtering region:1252, directly append 181 probes\n",
      "-- filtering region:1253, directly append 246 probes\n",
      "-- filtering region:1254, directly append 178 probes\n",
      "-- filtering region:1255, screen genome to keep 300 probes\n",
      "-- filtering region:1256, directly append 186 probes\n",
      "-- filtering region:1257, directly append 173 probes\n",
      "-- filtering region:1258, directly append 168 probes\n",
      "-- filtering region:1259, directly append 216 probes\n",
      "-- filtering region:1260, screen genome to keep 300 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:1261, screen genome to keep 300 probes\n",
      "-- filtering region:1262, directly append 85 probes\n",
      "-- filtering region:1263, directly append 189 probes\n",
      "-- filtering region:1264, directly append 279 probes\n",
      "-- filtering region:1265, directly append 99 probes\n",
      "-- filtering region:1269, directly append 51 probes\n",
      "-- filtering region:1270, screen genome to keep 300 probes\n",
      "-- filtering region:1271, screen genome to keep 300 probes\n",
      "-- filtering region:1272, screen genome to keep 300 probes\n",
      "-- filtering region:1273, directly append 297 probes\n",
      "-- filtering region:1274, directly append 192 probes\n",
      "-- filtering region:1275, screen genome to keep 300 probes\n",
      "-- filtering region:1276, directly append 244 probes\n",
      "-- filtering region:1277, screen genome to keep 300 probes\n",
      "-- filtering region:1278, directly append 150 probes\n",
      "-- filtering region:1279, directly append 193 probes\n",
      "-- filtering region:1280, directly append 296 probes\n",
      "-- filtering region:1281, directly append 275 probes\n",
      "-- filtering region:1282, directly append 234 probes\n",
      "-- filtering region:1284, directly append 188 probes\n",
      "-- filtering region:1285, directly append 206 probes\n",
      "-- filtering region:1286, directly append 263 probes\n",
      "-- filtering region:1287, directly append 287 probes\n",
      "-- filtering region:1288, directly append 239 probes\n",
      "-- filtering region:1289, screen genome to keep 300 probes\n",
      "-- filtering region:1290, screen genome to keep 300 probes\n",
      "-- filtering region:1291, directly append 273 probes\n",
      "-- filtering region:1292, directly append 248 probes\n",
      "-- filtering region:1293, screen genome to keep 300 probes\n",
      "-- filtering region:1294, directly append 90 probes\n",
      "-- filtering region:1295, directly append 167 probes\n",
      "-- filtering region:1296, directly append 1 probes\n",
      "-- filtering region:1297, screen genome to keep 300 probes\n",
      "-- filtering region:1298, directly append 282 probes\n",
      "-- filtering region:1299, directly append 267 probes\n",
      "-- filtering region:1300, directly append 266 probes\n",
      "-- filtering region:1301, directly append 252 probes\n",
      "-- filtering region:1302, directly append 220 probes\n",
      "-- filtering region:1303, directly append 174 probes\n",
      "-- filtering region:1304, directly append 268 probes\n",
      "-- filtering region:1305, directly append 297 probes\n",
      "-- filtering region:1306, directly append 285 probes\n",
      "-- filtering region:1307, directly append 271 probes\n",
      "-- filtering region:1308, directly append 243 probes\n",
      "-- filtering region:1309, screen genome to keep 300 probes\n",
      "-- filtering region:1310, directly append 285 probes\n",
      "-- filtering region:1311, directly append 211 probes\n",
      "-- filtering region:1312, directly append 257 probes\n",
      "-- filtering region:1313, directly append 201 probes\n",
      "-- filtering region:1314, directly append 272 probes\n",
      "-- filtering region:1315, screen genome to keep 300 probes\n",
      "-- filtering region:1316, screen genome to keep 300 probes\n",
      "-- filtering region:1317, screen genome to keep 300 probes\n",
      "-- filtering region:1318, directly append 253 probes\n",
      "-- filtering region:1319, screen genome to keep 300 probes\n",
      "-- filtering region:1320, directly append 286 probes\n",
      "-- filtering region:1321, screen genome to keep 300 probes\n",
      "-- filtering region:1322, screen genome to keep 300 probes\n",
      "-- filtering region:1323, screen genome to keep 300 probes\n",
      "-- filtering region:1324, screen genome to keep 300 probes\n",
      "-- filtering region:1325, screen genome to keep 300 probes\n",
      "-- filtering region:1329, directly append 35 probes\n",
      "-- filtering region:1330, directly append 129 probes\n",
      "-- filtering region:1331, directly append 242 probes\n",
      "-- filtering region:1332, directly append 278 probes\n",
      "-- filtering region:1333, directly append 136 probes\n",
      "-- filtering region:1334, directly append 161 probes\n",
      "-- filtering region:1335, screen genome to keep 300 probes\n",
      "-- filtering region:1336, directly append 266 probes\n",
      "-- filtering region:1337, screen genome to keep 300 probes\n",
      "-- filtering region:1338, directly append 188 probes\n",
      "-- filtering region:1339, directly append 238 probes\n",
      "-- filtering region:1340, screen genome to keep 300 probes\n",
      "-- filtering region:1341, screen genome to keep 300 probes\n",
      "-- filtering region:1342, directly append 261 probes\n",
      "-- filtering region:1343, directly append 201 probes\n",
      "-- filtering region:1344, directly append 274 probes\n",
      "-- filtering region:1345, screen genome to keep 300 probes\n",
      "-- filtering region:1346, directly append 298 probes\n",
      "-- filtering region:1347, directly append 179 probes\n",
      "-- filtering region:1348, directly append 130 probes\n",
      "-- filtering region:1349, screen genome to keep 300 probes\n",
      "-- filtering region:1350, directly append 265 probes\n",
      "-- filtering region:1351, screen genome to keep 300 probes\n",
      "-- filtering region:1352, directly append 243 probes\n",
      "-- filtering region:1353, directly append 192 probes\n",
      "-- filtering region:1354, directly append 278 probes\n",
      "-- filtering region:1355, directly append 187 probes\n",
      "-- filtering region:1356, directly append 291 probes\n",
      "-- filtering region:1357, directly append 248 probes\n",
      "-- filtering region:1358, screen genome to keep 300 probes\n",
      "-- filtering region:1359, directly append 267 probes\n",
      "-- filtering region:1360, screen genome to keep 300 probes\n",
      "-- filtering region:1361, screen genome to keep 300 probes\n",
      "-- filtering region:1362, screen genome to keep 300 probes\n",
      "-- filtering region:1363, screen genome to keep 300 probes\n",
      "-- filtering region:1370, directly append 10 probes\n",
      "-- filtering region:1374, directly append 20 probes\n",
      "-- filtering region:1377, directly append 28 probes\n",
      "-- filtering region:1378, directly append 217 probes\n",
      "-- filtering region:1379, directly append 292 probes\n",
      "-- filtering region:1380, directly append 192 probes\n",
      "-- filtering region:1381, directly append 214 probes\n",
      "-- filtering region:1382, directly append 210 probes\n",
      "-- filtering region:1383, directly append 230 probes\n",
      "-- filtering region:1384, directly append 148 probes\n",
      "-- filtering region:1385, directly append 163 probes\n",
      "-- filtering region:1386, directly append 128 probes\n",
      "-- filtering region:1387, directly append 190 probes\n",
      "-- filtering region:1388, directly append 176 probes\n",
      "-- filtering region:1389, directly append 251 probes\n",
      "-- filtering region:1390, directly append 288 probes\n",
      "-- filtering region:1391, directly append 233 probes\n",
      "-- filtering region:1392, directly append 244 probes\n",
      "-- filtering region:1393, screen genome to keep 300 probes\n",
      "-- filtering region:1394, directly append 174 probes\n",
      "-- filtering region:1395, directly append 282 probes\n",
      "-- filtering region:1396, directly append 192 probes\n",
      "-- filtering region:1397, directly append 12 probes\n",
      "-- filtering region:1398, directly append 184 probes\n",
      "-- filtering region:1399, directly append 268 probes\n",
      "-- filtering region:1400, screen genome to keep 300 probes\n",
      "-- filtering region:1401, directly append 268 probes\n",
      "-- filtering region:1402, screen genome to keep 300 probes\n",
      "-- filtering region:1403, directly append 223 probes\n",
      "-- filtering region:1404, directly append 235 probes\n",
      "-- filtering region:1405, screen genome to keep 300 probes\n",
      "-- filtering region:1406, screen genome to keep 300 probes\n",
      "-- filtering region:1407, directly append 16 probes\n",
      "-- filtering region:1408, screen genome to keep 300 probes\n",
      "-- filtering region:1409, screen genome to keep 300 probes\n",
      "-- filtering region:1421, directly append 2 probes\n",
      "-- filtering region:1422, directly append 2 probes\n",
      "-- filtering region:1425, directly append 4 probes\n",
      "-- filtering region:1427, directly append 253 probes\n",
      "-- filtering region:1428, directly append 1 probes\n",
      "-- filtering region:1429, directly append 266 probes\n",
      "-- filtering region:1430, directly append 240 probes\n",
      "-- filtering region:1431, directly append 10 probes\n",
      "-- filtering region:1433, directly append 28 probes\n",
      "-- filtering region:1434, directly append 295 probes\n",
      "-- filtering region:1435, directly append 255 probes\n",
      "-- filtering region:1436, directly append 298 probes\n",
      "-- filtering region:1437, screen genome to keep 300 probes\n",
      "-- filtering region:1438, directly append 174 probes\n",
      "-- filtering region:1439, directly append 198 probes\n",
      "-- filtering region:1440, directly append 236 probes\n",
      "-- filtering region:1441, directly append 184 probes\n",
      "-- filtering region:1442, directly append 248 probes\n",
      "-- filtering region:1443, screen genome to keep 300 probes\n",
      "-- filtering region:1444, screen genome to keep 300 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- filtering region:1445, screen genome to keep 300 probes\n",
      "-- filtering region:1446, directly append 274 probes\n",
      "-- filtering region:1447, screen genome to keep 300 probes\n",
      "-- filtering region:1448, directly append 267 probes\n",
      "-- filtering region:1449, screen genome to keep 300 probes\n",
      "-- filtering region:1450, directly append 211 probes\n",
      "-- filtering region:1451, directly append 255 probes\n",
      "-- filtering region:1452, directly append 113 probes\n",
      "-- filtering region:1453, screen genome to keep 300 probes\n",
      "-- filtering region:1454, directly append 277 probes\n",
      "-- filtering region:1455, directly append 290 probes\n",
      "-- filtering region:1456, directly append 270 probes\n",
      "-- filtering region:1457, directly append 281 probes\n",
      "-- filtering region:1458, screen genome to keep 300 probes\n",
      "-- filtering region:1459, screen genome to keep 300 probes\n",
      "-- filtering region:1461, directly append 143 probes\n",
      "-- filtering region:1462, directly append 238 probes\n",
      "-- filtering region:1463, directly append 268 probes\n",
      "-- filtering region:1464, directly append 281 probes\n",
      "-- filtering region:1465, directly append 231 probes\n",
      "-- filtering region:1466, directly append 286 probes\n",
      "-- filtering region:1467, directly append 193 probes\n",
      "-- filtering region:1468, directly append 180 probes\n",
      "-- filtering region:1469, directly append 202 probes\n",
      "-- filtering region:1470, directly append 194 probes\n",
      "-- filtering region:1471, directly append 221 probes\n",
      "-- filtering region:1472, directly append 278 probes\n",
      "-- filtering region:1473, directly append 117 probes\n",
      "-- filtering region:1474, directly append 173 probes\n",
      "-- filtering region:1475, directly append 234 probes\n",
      "-- filtering region:1476, directly append 197 probes\n",
      "-- filtering region:1477, screen genome to keep 300 probes\n",
      "-- filtering region:1478, directly append 230 probes\n",
      "-- filtering region:1479, directly append 216 probes\n",
      "-- filtering region:1480, directly append 117 probes\n",
      "-- filtering region:1481, directly append 198 probes\n",
      "-- filtering region:1482, directly append 212 probes\n",
      "-- filtering region:1485, screen genome to keep 300 probes\n",
      "-- filtering region:1486, directly append 227 probes\n",
      "-- filtering region:1487, directly append 250 probes\n",
      "-- filtering region:1488, screen genome to keep 300 probes\n",
      "-- filtering region:1489, directly append 94 probes\n",
      "-- filtering region:1490, directly append 128 probes\n",
      "-- filtering region:1491, directly append 239 probes\n",
      "-- filtering region:1492, directly append 238 probes\n",
      "-- filtering region:1493, directly append 129 probes\n",
      "-- filtering region:1494, directly append 189 probes\n",
      "-- filtering region:1495, directly append 199 probes\n",
      "-- filtering region:1496, directly append 72 probes\n",
      "-- filtering region:1497, directly append 230 probes\n",
      "-- filtering region:1498, directly append 146 probes\n",
      "-- filtering region:1499, directly append 160 probes\n",
      "-- filtering region:1500, screen genome to keep 300 probes\n",
      "-- filtering region:1501, directly append 253 probes\n",
      "-- filtering region:1502, directly append 139 probes\n",
      "-- filtering region:1503, directly append 186 probes\n",
      "-- filtering region:1504, directly append 285 probes\n",
      "-- filtering region:1505, directly append 239 probes\n",
      "-- filtering region:1506, directly append 163 probes\n",
      "-- filtering region:1507, directly append 242 probes\n",
      "-- filtering region:1508, directly append 189 probes\n",
      "-- filtering region:1509, directly append 188 probes\n",
      "-- filtering region:1510, directly append 176 probes\n",
      "-- filtering region:1511, directly append 218 probes\n",
      "-- filtering region:1512, directly append 248 probes\n",
      "-- filtering region:1513, directly append 245 probes\n",
      "-- filtering region:1514, directly append 262 probes\n",
      "-- filtering region:1515, screen genome to keep 300 probes\n",
      "-- filtering region:1516, directly append 103 probes\n",
      "-- filtering region:1517, directly append 232 probes\n",
      "-- filtering region:1518, directly append 217 probes\n",
      "-- filtering region:1519, directly append 239 probes\n",
      "-- filtering region:1520, directly append 221 probes\n",
      "-- filtering region:1521, directly append 176 probes\n",
      "-- filtering region:1525, directly append 51 probes\n",
      "-- filtering region:1526, directly append 60 probes\n",
      "-- filtering region:1527, directly append 72 probes\n",
      "-- filtering region:1528, directly append 44 probes\n",
      "-- filtering region:1529, directly append 166 probes\n",
      "-- filtering region:1530, directly append 218 probes\n",
      "-- filtering region:1533, directly append 69 probes\n",
      "-- filtering region:1534, directly append 149 probes\n",
      "-- filtering region:1535, directly append 187 probes\n",
      "-- filtering region:1536, directly append 187 probes\n",
      "-- filtering region:1537, directly append 170 probes\n",
      "-- filtering region:1538, directly append 164 probes\n",
      "-- filtering region:1540, directly append 1 probes\n",
      "-- filtering region:1541, directly append 251 probes\n",
      "-- filtering region:1542, directly append 204 probes\n",
      "-- filtering region:1543, directly append 30 probes\n",
      "-- filtering region:1547, directly append 2 probes\n",
      "-- filtering region:1548, directly append 106 probes\n",
      "region: 41 removed with 22 probes\n",
      "region: 48 removed with 3 probes\n",
      "region: 82 removed with 8 probes\n",
      "region: 142 removed with 39 probes\n",
      "region: 151 removed with 3 probes\n",
      "region: 232 removed with 23 probes\n",
      "region: 301 removed with 25 probes\n",
      "region: 420 removed with 37 probes\n",
      "region: 434 removed with 5 probes\n",
      "region: 488 removed with 3 probes\n",
      "region: 514 removed with 2 probes\n",
      "region: 515 removed with 20 probes\n",
      "region: 547 removed with 44 probes\n",
      "region: 553 removed with 49 probes\n",
      "region: 604 removed with 6 probes\n",
      "region: 630 removed with 37 probes\n",
      "region: 631 removed with 21 probes\n",
      "region: 632 removed with 9 probes\n",
      "region: 641 removed with 5 probes\n",
      "region: 643 removed with 1 probes\n",
      "region: 747 removed with 23 probes\n",
      "region: 773 removed with 46 probes\n",
      "region: 1021 removed with 11 probes\n",
      "region: 1026 removed with 17 probes\n",
      "region: 1027 removed with 3 probes\n",
      "region: 1028 removed with 1 probes\n",
      "region: 1086 removed with 28 probes\n",
      "region: 1088 removed with 25 probes\n",
      "region: 1095 removed with 2 probes\n",
      "region: 1096 removed with 14 probes\n",
      "region: 1158 removed with 20 probes\n",
      "region: 1161 removed with 3 probes\n",
      "region: 1164 removed with 3 probes\n",
      "region: 1175 removed with 44 probes\n",
      "region: 1185 removed with 4 probes\n",
      "region: 1198 removed with 30 probes\n",
      "region: 1199 removed with 2 probes\n",
      "region: 1296 removed with 1 probes\n",
      "region: 1329 removed with 35 probes\n",
      "region: 1370 removed with 10 probes\n",
      "region: 1374 removed with 20 probes\n",
      "region: 1377 removed with 28 probes\n",
      "region: 1397 removed with 12 probes\n",
      "region: 1407 removed with 16 probes\n",
      "region: 1421 removed with 2 probes\n",
      "region: 1422 removed with 2 probes\n",
      "region: 1425 removed with 4 probes\n",
      "region: 1428 removed with 1 probes\n",
      "region: 1431 removed with 10 probes\n",
      "region: 1433 removed with 28 probes\n",
      "region: 1528 removed with 44 probes\n",
      "region: 1540 removed with 1 probes\n",
      "region: 1543 removed with 30 probes\n",
      "region: 1547 removed with 2 probes\n"
     ]
    }
   ],
   "source": [
    "# filter by genome counts\n",
    "reload(library_tools.quality_check)\n",
    "\n",
    "num_probes_kept = 300 # max num of probes kept for each region\n",
    "filtered_pb_dict = library_tools.quality_check.Screen_probe_by_hit(library_folder, pb_dict, num_probes_kept,\n",
    "                                                                  hit_type='genome')\n",
    "\n",
    "## filter by minimum number of probes in each region\n",
    "min_num_probes = 50\n",
    "removed_regions = []\n",
    "for _rid, _pb_class in filtered_pb_dict.items():\n",
    "    if len(_pb_class.kept_probes) < min_num_probes:\n",
    "        print(f\"region: {_rid} removed with {len(_pb_class.kept_probes)} probes\")\n",
    "        removed_regions.append(_rid)\n",
    "for _rid in removed_regions:\n",
    "    del(filtered_pb_dict[_rid])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 90,
   "id": "5571da09",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1352"
      ]
     },
     "execution_count": 90,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(filtered_pb_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "id": "4b910471",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox  6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAApQAAAFKCAYAAACjE8/+AAAAAXNSR0IArs4c6QAAIABJREFUeF7sXQV0FdcWPQQI7g7BAgRNcIcgwUuBYkWKFX7RQou1SCmuLaUtWijQIi3FKe4uCR5cQnAIJHgIwf4692Ve5k1m3tyxlxd67lp//ZK5uue+mT3nnrNPovfv378HKoQAIUAIEAKEACFACBAChIBOBBIRodSJHDUjBAgBQoAQIAQIAUKAEGAIEKGkjUAIEAKEACFACBAChAAhYAgBIpSG4KPGhAAhQAgQAoQAIUAIEAJEKGkPEAKEACFACBAChAAhQAgYQoAIpSH4qDEhQAgQAoQAIUAIEAKEABFK2gOEACFACBAChAAhQAgQAoYQIEJpCD5qTAgQAoQAIUAIEAKEACFAhJL2ACFACBAChAAhQAgQAoSAIQSIUBqCjxoTAoQAIUAIEAKEACFACBChpD1ACBAChAAhQAgQAoQAIWAIASKUhuCjxoQAIUAIEAKEACFACBACRChpDxAChAAhQAgQAoQAIUAIGEKACKUh+KgxIUAIEAKEACFACBAChAARStoDhAAhQAgQAoQAIUAIEAKGECBCaQg+akwIEAKEACFACBAChAAhQISS9gAhQAgQAoQAIUAIEAKEgCEEiFAago8aEwKEACFACBAChAAhQAgQoaQ9QAgQAoQAIUAIEAKEACFgCAEilIbgo8aEACFACBAChAAhQAgQAkQoaQ8QAoQAIUAIEAKEACFACBhCgAilIfioMSFACBAChAAhQAgQAoQAEUraA4QAIUAIEAKEACFACBAChhAgQmkIPmpMCBAChAAhQAgQAoQAIUCEkvYAIUAIEAKEACFACBAChIAhBIhQGoKPGhMChAAhQAgQAoQAIUAIEKGkPUAIEAKEACFACBAChAAhYAgBIpSG4KPGhAAhQAgQAoQAIUAIEAJEKGkPEAKEACFACBAChAAhQAgYQoAIpSH4qDEhQAgQAoQAIUAIEAKEABFK2gOEACFACBAChAAhQAgQAoYQIEJpCD5qTAgQAoQAIUAIEAKEACFAhJL2ACFACBAChAAhQAgQAoSAIQSIUBqCjxoTAoQAIUAIEAKEACFACBChpD1ACBAChAAhQAgQAoQAIWAIASKUhuCjxoQAIUAIEAKEACFACBACRChpDxAChAAhQAgQAoQAIUAIGEKACKUh+KgxIUAIEAKEACFACBAChAARStoDhAAhQAgQAoQAIUAIEAKGECBCaQi+2MZRUVEQHBwMWbJkgSRJkpjUK3VDCBAChAAhQAgQAu6KwJs3b+DBgwfg6+sLyZMnd9dpumReRChNgjkoKAgqVKhgUm/UDSFACBAChAAhQAgkFAQCAwOhfPnyCWW6lsyTCKVJsIaGhkL+/PkBN1WOHDlM6pW6IQQIAUKAECAECAF3ReDu3bvMmHTt2jXIly+fu07TJfMiQmkSzLdu3YLcuXPDzZs3wcvLy6ReqRtCgBAgBAgBQoAQcFcE6N0fe2eIUJq0S2lTmQQkdUMIEAKEACFACCQQBOjdT4TS9K1Km8p0SKlDQoAQIAQIAULArRGgdz8RStM3KG0q0yGlDgkBQoAQIAQIAbdGgN79RChN36C0qUyHlDokBAgBQoAQIATcGgF69xOhNH2D0qYyHVLqkBAgBAgBQoAQcGsE6N1PhNL0DUqbynRIqUNCgBAgBAgBQsCtEaB3PxFK0zcobSrTIaUOCQFCgBAgBAgBt0aA3v1EKE3foLSpTIeUOiQECAFCgBAgBNwaAXr3E6E0fYPSpjIdUuqQECAECAFCgBBwawTo3U+E0vQNSpvKdEipQ0KAECAECAFCwK0RoHc/EUrTNyhtKtMhpQ4JAUKAECAECAG3RoDe/UQoTd+gtKlMh5Q6JAQIAUKAECAE3BoBevcToTR9g9KmMh1S6pAQIAQIAUKAEHBrBOjdT4TS9A1Km8p0SKlDQoAQIAQIAULArRGgd/8HRCjDwsJgwIABsHnzZnj58iXUqFEDpkyZAsWKFWOrPHnyJPTr1w+OHj0KmTJlgr59+8LAgQPtCLx79w5GjRoF8+bNg0ePHkG1atVg5syZULBgQU2bmDaVJrioMiFACBAChAAhkOARoHf/B0QoK1WqBB4eHvDLL79AqlSp4LvvvoODBw/ClStXGMEsUqQINGvWDPr37w+HDx+GXr16McLYpUsXhgKSSfz3ggULIFeuXDB48GAICQmBs2fPgqenJ/dmp03FDRVVJAQIAUKAECAEPggE6N3/gRDK8PBw6NOnDwwfPhyKFy/OVnXq1CkoVaoUBAYGwvbt22HGjBkQGhoKSZIkYdeHDh0Kq1atggsXLkB0dDRkzpwZJk+eDD169GDXHz9+DDlz5oT58+dDmzZtuDc8bSpuqKgiIUAIEAKEACHwQSBA7/4PhFBKd+P9+/fZcfbu3bsZYWzVqhWkT58eli5daq+KJLNu3bqAdZFoVqxYES5evAg+Pj72Onjs7efnxyyXvIU2FS9SVI8QIAQIAUKAEPgwEKB3/wdIKL/44guYO3cuJEuWDNatWwf16tVjpLBhw4YwadIk+4rPnTvHrJlBQUFw48YNaNGiBURGRkKKFCnsdVq3bs3+tn79eu4db9WmunDvKey//NA+j5SeScA3VzpYfuwmVCmQGe48fgnvASARADQokR0eR76GkIfP4d17gIr5M8LyozchbYqkkC5FUlY/S5pkcdZ09cFzOHP7Cdx69BIypfKEZqVzwfbz9yFfplTw7+k7UDhbGkiS2AMCimSFVMlsll6hRL95B0uPXIc3797DZ5XyQtTrtzBv3zVIkjgRFM+ZDuoUzQq3H7+EmbuvwpPI1+DhkQhalvWCl9FvoFiOdJAnU0rA8Y+EREAj3+xw7s5TOHnrMYQ9fQU50yeHEjnTwdOo13AjIhLSJE8KqZMlgULZUkMqzySw9/IDqFssG2RNk9w+n/fv38OO82GQL3MqSOKRCC7cewY1C2eBbefuQ/7MqSA0/AWUy5sRdl4Ig8ypPVm/796/h7yZUkHdotnY/MQFcXkUGQ0lc6eH3RcfsL7SJk8aB8OIF9Gw/8pD1kcKz8T264jH+tN3IWniRNDINwd4JEoEm87cZfejeqEsrN6LV29gx4UwSJwoEeTOmAL8vNKzv5++9RieR72BKgUzw+X7z2DPpQeQPGlitubAaxHsvh+//oit48GzV9CtujdERr8F3DM4boaUnmwfYP3EknVhnbtPoqBW4axx1vL6Ld7TG6yPTlXysTGvhD2DmbuuQoZUnmxfvX33nvWL+0KpCGt/HBnNquDacY/mTB/7WxPW7l8oM6RPGdfFBPdD2LMo8MqQEkIePGdjHrwaDufvPoUyeTNAmTwZZO897p2Vx27B1QcvIF+mlNDILwdkTOkJm8/eYxj5ZEsDR0Mj4N6TKLYH8X5UK5gZMqWO+/sQBoiMfsPuJe7jRIkASseM/e+pO/B51fwM94JZU7M9FvLgBZtf0LUIOHvnKRTLmRZK504PFb0zse6EfYq/1TwZU0G9YnH33tk7TyD8eTT4+2SBXRfC2O8ES8X8mcDXKx3771M3H7PfbURkNHgmTgQpPJNA2NMoqFYoMxTJnpbVOXY9gu2P129t9wznh8+UkIcv2P3He5ksqQccvPKQ7aFsaZOz58G5u0+hsV8OwGeOtASFRsCtR5HQsEQO2H0xzOG3jGvH330iBClm/BM3HrPnAeKM67gc9hzSJEsCrcvnZvPbe+kheGVIATcjIhlu2Fb4LT1/9QYW7L8GuC9xnfgcOxr6CKLevGV78VnUG8B7g9jXLpLNPtVDV8Ph5M3H7LeXK30KOHXrCZTLmwEqeGeE9afuQqbUnuBfKAtsO38fqhbIBNcevmD18XmZLIkHlMqdHvD5dj08EgJi1oPPsrl7Q6Bcvgxw+tYTKJYjLXtm5M2YEsJfvIJEkIj9buoXz25/lhy/8QiuPXgBVQtmZmMl9UjE9kNQ6COoUiATFM1hu09Cefj8FRy48pA9k/D5hCV/plRw+3Ekmxveywr5M7JnGu5lfAd0qZqfPd+EPfr05WvWDvdo2by23wjup4fPoyFX+uSwKfgeXAp7DvkzpYTMaZKxfbHlzD249/QVZEubDHKkSwHRb9/B1bDnkDdTSnaf/zgUyp5ViDP+2zOJ7bcvfuZeCXvOnk050iVn1xuWyA53nkRB5Ks37Pe99dw99pus5J2RPcsLZEkNp249jvMcx9/YnssP2P3F/RX99j185JsDMqbyZPsA147PZLyvQtl1MYyNK+z7OJvWoj9Y9e63aLqWdpvoPe6GD6AgUUSfyVmzZjGL5P79+wGJYbt27WD06NH2FaJ/ZIECBWDfvn3MQtmhQwd4+/Yt88MUSseOHeHOnTvsyFypPH36FPB/Qrl79y5UqFABbt68CV5eXqYh+lfgDRiyKtiU/vDHdmhIQJy+8n27weFv+NB/EvNAEl9AwjezfVmHulO3XoRfdl5hf+tSNR8jOPjgFsrM9mWg15LjivMPnfgRVBy/He4/fcUe9kevP+Jaa6GsqdlLCV9c8zqVt7fZGHw3znipPBPDi+i3qv1O+7QUI9NCeRn9FoqO2Mz+iQ8/fLFVLZgJlnSrFKev2j/sZi/o5mVywdTWpezXf99/DcasP8f+/XObUuxF1WOxDY89g2qyl0a/v0/A2pN37G1Oj6zH/ttv5Fb2/xv7Vod28w6zjwU9ZWyzEozsCwUJRIGhG9k/l/6vIntBi8uvOy7Dj9susT+1q5gHxn/iC9I9gtdGNC4Gn1fLrzilIatOw1+BNx2uI7G9Or6R/W+9lxyHDcF32UfSv19Wc6iLL45CwzY5/K1I9jTsIwEL9nVyRF32oYFl85m7dmy/rF0Qfo3Zl3gNX0QTm/vCF4uOKc7XO3Mq2DmwpuL1vn+dgHWnYu+Tnnuxe2BN9rGDHwtNph+wdzGlpR+0Kpfb/u9Xb95C4eG2vdcvoBD8vOOyw3BnR9VnL9ZSo7cpTgNxRjJeecJOe532FfPA5jP3IPyFjeRLC353nB/TAIqN2MKIZp9aBWFg/cIO1cKfv4KyY23PRvzYvHjfdj/wtyzsk1/bloaPS+aE+0+joOL4HYpzLJErLZy5HfscFVdsXc4LJrcsCd3+OMo+cHnKhr7VGHHFcStN2AFyb7cK+TJCYGgE6w7vOf5ukfwgeVQqczuWY6RH7ncg12ZJt4qMQD6Leg2+Mb9juXr4PDg3uoHDB1/dqXvYs01rQfynbLkAM3ZddWgaODQA0qVMat9PWvvF+mmSJ2HEXShTW5eE5mVs7zncTz0Wy/+u/LzSMeKtVmoVzgILulSwV2s+8wAcv/HYoZlPttSw9esaMGPXFZiy5aJ9z+F/4AfSZ78fYX8LGd8ojmFAbXwj14lQxqL3wRBKYUkYte3r68uOstEK2ahRI1kL5fHjx1nwTcuWLWUtlK9evYK1a9cq7rORI0eygB5pMZtQrj99B/osPWFkvzu0xYeOtPA+JLGdtL3v91vg2avYB420byShG4PvKc5f/BLSu0jxnHovPQ4bTt/V1VXtIllhfudYcopf/lUmxr6MhU7VMBRfH7/xPPy2N4Q1HdaoKLx9/x4mbrrA/v1P98rM0iDFf+eAGsxqWmfqXlbv+4+Lwah/baRUTymTJz2s6lXV3lRMlL/w94ahjYo6dFtsxGZm+RCvV26PIJnY8rW/4pSU9pUYH3EdKa5omRZItdIg+7+pxayXWMT3vqRXOocPG7w+pmlx+G7tWacQyt1boYGW34nSIL+0LQ1NSuaErWfvOZBbtI4u7lYRroe/ACSuaGn7abuN1MuVfYNrsQ+Mj6fvV6xzYUwDOBwSDp0XBGnaNqdG1IOSo20fM8VzpoUNfas7tJeSYbl9gpapWZ+VZZa29vNsL3o9RevzYUHn8lCrSFZmYW38qzI2WufSoowX/Ni6JDeh/KFVSXYSg1bcapN2OR0O7xOeAhjdZ4hVo5/3McuyuCC5Reul3LNMKw5C/W8aFIGeNQuwf5rxoYX9KD0XxHPEOlUm7GBWT3Gb79eegT8OXWd/uzKuodOTE71rVmpHhDIWmQRNKFEyaMeOHcwSmThx7A8SfSfReoh/Q6mgRYsW2Vcs50OJEeFotRQK+lCWLFmSBfQoFVdZKHH8bn8EwfbzYab8DtTIkNogcQjlyC0OX67S9niUt/WcsnVB6wtDbn7iOX351wnAI0g9RfqVfPfJSwfrjtCnGobOCCUSxQkxhPLvLypBJe9McV5SewfVYsdNdabuYUOiJXB0jJVTz7rwyGtlzyr2pngshhYoLHKEUkqclO4RHtVt6udINsTzcwWhRGKVO2MMoYyxduIcSudJD3jMKi6jmxaHEfFMKNFK3bRULthy9h50F1lLBULZdMYBdoytVg5+WxvuPY2C5jMPKlY9P7oB4NF0x/mBat05XEerr2D5lCOUYmuQuKF4nwinGQevPoR2c11HKOd3LsfIuBLp1QSEqHLz0rlg6qeluAnl5BZ+7Dgfj/CrT3ZOKPE+CW4yaCX1Ge5oleed8+EhAdDtz6A4Ft/+dX1gasyJA29favUGNygMvWrapPWMPHOl+0f4t7NnR9WJO5kbFRbhWfvdmjOw6DARSrX7ZvX1BE0oUWOydOnSjFTWrl2bYfX69WsmFdSkSRPIli0bzJ49G65evWonnEOGDIHVq1ezoB20QmbNmhWmTp0KXbt2Ze3dMcr7f38eZT4jZhQ1MqQ2Bh6fLA28AZ+UzsWOa/1GboGnoqMQaXu0+qG/olIxm1Aa+VqWEkr0r8NjM2mRYoj+jXV/slkTxQ85/O9xG87B3H3X2N+HNirC/n/8RpuFcmm3isw/UvrwPPBtbeaHFfCjjVB+17iY/dhc7f7IXS+fLwMs7yFPKLv7e8MQiYWSl1DiceX6L92HUPZcfAw2nbFZw5FEH5O4T7gToZQ7Jtze3x/a/HaY+bmplSNDA5h/XavZhxSrnhlVH07ceAQdftdGKI8Nr2M/0pYjlHgCgNZgud+FsHfQ321G+zLMn671HOU5qq1T6/Ph907lIKBoNuYL2WxGrEuB2jhq1/F595MGQjmhuS+0rZCHi1CeG13f7qf66EU0lB6j7MagNk85y7xaGz3XB9UvDL1r2Qhln6XHmW+x0cJroaw2aSfzGxY/a4euDmZ+31jIQmn0Tuhvn6AJJbp/YtAN+kJiQE6GDBlg3LhxsGXLFiZonjx5cju5RH1JlBLq2bMnI5mdOnViqA0bNgzmzJnDZILy5csHgwYNYv0FBwe7jQ5l90VHYctZ9yCU+ILBIAN0qA8eVV+VUFYvlBn2iYKKnL2E9G5j8YNI6o+opU8Mfvjz81g/Hl5CKUfAhHHHrj8H8/bbCOW3DYuwQKGxG86zfy/uWpEFT0jbI2lHn89aP+xm9YZ/VNTeRst6hLroM/ZPj8r2puhYX/x7m4XSCKHEl9faPo5+j+L5ucJCidZcDKphaxH9TpBEY9CDuIxqUhy+Xxe/R96ChVLs7ymeIwbFKfk3iusFDasDGADRdu5hxS3RuUo+5ven9cgZXS5qx3zMyBFKJb9uMfnDYJ7p7cqwgJsWs1xHKAVfR6PjSkFtVionTGtTmttCKfgt3wiPBP8pzi2U6A8rBDuK/VP1/NYxkAjJtNVlYD0f6FO7EBtG8IM2OiYvofSfvMserCS0EftrXx7XEJI6CRY0Ok9pezryjkUkQRNKXMaTJ08ArY5r1qxh1sXq1aszi6OgS4l+lJgd58SJE5AjRw6WVQe1K4WCATmoTYnC5hjU4+/vz466kVxqKVZuKrHlRcuc5OoatVCK+8S+So7aKhvAI9TLnjY5O5pTKlotENJ+UnomZk7tQjFCKLEPfJl6Z0nNujODUGJADgbmCAX9KMdttBFKJK9IYuUIJfow1owhlOI2eu4/+mmiv6ZQHAhlDW8Y0tDRh1I6HwzMEb7+xePjsfJqkW+mdG6uIJRCYBOO/cWfR+3uFXKEcuTHxWCkii+q9PeBEeoYoIbWeDN8KIXAL7ngMS33NnBYAFy699weiKDUVi0oTq7dmt5V7dY9jGLeKHFrmL3nqt0PWNz+2oRGkH+ILdgL/UTRXxQtpJ84OZZXW7PW58NvHcqyjzTBpUOtf97rTUvlhJ81EEr8eEGFBPSJrTHF9mGoVNCSjEF/WDCIqsI45SAmtfmiv7Q0mEWtjZ7rn1XKA2Ob+bKmvZYcc+onz9s/L6GsMWUXi7zHIrT5ZsVpWHbUFgB4aWxDewQ679hG6ln57jcyr/hom+AJZXyAJjemlZvKrC9A8Q9QvAa9L0o81kELHE8EtdJ90vrCkPaDMkhorRHKV3+fgDWiiGmt+0PwZcN2ShGqUtLhzEI5+t9zMP9ALKEU+5Qu6FKeybZI2+NxJgbOCIQSj8qFY3Kt68H6KNHx9xexhBKj1UsIFkoOQqk0JkblrxD5ZkrrqRFKlP5p+PM+ezPcTxj8IMiR8ATl7BpYkwUcYBG7hogjeYUBeIKbxPdW7P+3vX8Nu0+rnnsgtPnp05LwSWkvFjgmd2zM2zdasc/efQpdVAJu0CLa7++TvN2yeqt7VbGTQDlCOW37JZi23THqHNvhUWPBmKh8gYAZ9WXU+nzoG1AIdl64rxg5rgkIUWWBIPM+K9HvuUbhLPDXkRv2EwqlscWEUi0qXm3+cr7Dam30XkdfW5QCMsvgwUso8eQGJZ7E77OBy0/BimO32N+IUOq9o8bbEaE0jiHrwUpCaZaPivgHKF4270PSJKgcutH6wpDOAXXI0OdQKF8vOwmrT9zWPVXx8TDq+VWQkTzRQihH/XsWFhwItc8HZY6EACshgAAlfFCiRSjoXI+yMYJlA4/KhchwPQur7J0J/voiVupITNR61CjAjuL17Aep5VM6NzVCOWLtGfgzJjJTaDugrg98GWA7SuMhlGKLsjh4DeeG/nviwhPcJNxb1BUtI/JlQ8kt1O00WgRCaVS9AT86UI4FSbSzglZC9CvWUjCAq8UsW7CPHKHEAI9fJDJGWPfi2AZ2aRrhiNhotPWOATXsvsRa1mB2XZRAQikk3mcluqkg6caPN7USPLKeXfpKKRBQrQ/huqt8KHG8eR3LQZ1i2aDHomNM39Vo4SWUtX/czXResQht+v9zElYdtz33cR8mSxIbpGt0XmrtrXz3q43tbteJUJp0R6zcVEYCTaTLM/PI2wzozCaURmSDcD0opLss5njYCkIpDlIS/L2KfrcZXr6Olek5NKQ2vH7z3u57hRGVkzfbdNf0FKl2Jh7hoqsCFiOEUkpUpXNTI5TiyEyhrViKiIdQIuFAcWQsXRcGMYF4LHKEkie4KU/GlDCvUzkm6NzTiX6qnvuAbQT9PlQiwOhYvQX3CAbqqUWtz2hXRrMldEWPytAyJthHllCKtGfF80f5myLf2bQzhahoFKZv9EusFVrrevEEAoW847sIPqG8hHJIwyJ2NQe1uaPurJAsgUdmyFl/znQ91eah9bpAKFvOOsitH+xsDF5CieoX6D+MRWgjPpkiQqn1TppXnwilSVhaSSiNHuOKl/ihEUrMrrH/G5uFEoO0BB8uvbdVTJLMIJQj152FhQdjLZSYaQcz7mCZ06Esy6hRavRWB9FylIRB0WrBQokRlYKQr551iY/xsT36BQqyMKglh5pyWFCzEDMQOdM2FI8v7Vc6N+8hG1imHmkR9uDwNcGw+LAtMlMoWgklHkVj5g4sXRYEwq4YbPHD4IjEQskb3IT9tSmf21AglNJ9+rFVSWhR1osJpGu1HIr75NHUxPqz2pfRTIyX96hsjx6Xk4b6YctFmL7LlsxAXDC4RAj2EgT+L957BvWnxSog6Nm/7tDmI78cgOScl1Bq+c2e+r4eyyCExezodCuxw4h6zKpklt4nL6EUC78LbcTSRVJdTysxwL6tfPdbPXez+ydCaRKiVm6q/stOwioDx7jiJX5ohBItSnsH12JL5LFoqd1uMUlScpDXcuQtJZQYhLP3ko1Q4su+oW8OqDBuO4SJrDB4hP/6zTu7DyUeAwuZa9TmL3ddSvzE0iQCoZRKH/GMI42Kl7YpNGwjS/cnLQJ+w1YHw5IYqQ+hjnZC6Q8Fs6ZhzTvND2TpKbGg3+jhEMcjb15CiQoG/eoUspRQrj15W7NvI889kdaZ/VlZxSwmSv2JLeKYmWjzV47i9XLZWLAvtLQJQvSCEPgHQyhjZJB4CaWW3ywKyWMmGyy8/evZC2a3QUKJYuLC88xo/92q5WeqBJie1NnpRv2f9jpkZ8JxxSdTRCiN3gn97YlQ6sfOoaWVhHLAP6dg5XGbw7HRIiVDZlj1jMzJ6JE35pndM8hGKI1KbmAfKHO0qGtF1h8etZUfFzf9Jg+hxKAaFCsWZ3AQ+hdklKa3Kw2N/XJCpfE7HCLhMfsLChwL0i1f1/FxmjVFDX/xmqQ4CYRST4pPaWYh6TwKD98Er2TS2Qn4ibXjhLZiQik+mlda47av/aFQNhuhRAFv4eUmRyh5o+WTJ/WAQfWLGNL+VJqvkEFlzYnb8NUybcEyavdZ7jpawcUC6lr7kCOUkzZfgFm7HdP7Yb9iQfRWZb1gSquSLL95g2n6j7y1zteq+oJQuxWETwhuwblb0b9VmKAPOPqHO5OF0zO2s3cCXmswba89/Sqe+KDrEFr7BQ1aIpR6UDenDRFKc3C01Ow9aPkpWB4TwWZ0ulIy9O7de/COyetstG897c0klEYd2nH+NXyywB8xWpR6LZSTWvjC0NVnYGST4oCWP3HgSZUCmeDg1XA7VLj+yhN2OAR8YPYXJGJCphy5XM5asJZaEh8+fwXlYnIx96pZAAY3KALLgm7ANyu15YyvUzQb8zdUKtIUjkI9YQ9ijnoksuIiJpRKsk3i+lu/9gefGELZ4fcj9pebEUKJWqEo9i7kYNeCtVpdIWf36hO34Otlp9SqG76OL1u1wB1ng8gRSgwQQ+kgaTn+XV17IJOQh1sayW94QfHUgZBK0grCd+K7upAhlSfVd91AAAAgAElEQVRbmRX9WwWZOxBKXBs+bzF5hqDVLM48ZNXaxf1aaUxyxfzNHIMIpUloWrmpxBpbRqcrJZQYXYxRxvFVjBLKfJlSwu4YCyWPiLDaOsXZcvT6UIrH6Fg5rwOhlBIdZUL51p7Lu2/tgvDLzrg+a2prEa7jV/zCLrGC7WKiLBDK5UdvwqAVp3m7ZPXqF88GczooE0pfhbScsYTyNPwVaNOOE4qYUH70yz4mou+soJU39OEL+KxSXnbsdeCKjayjL+yhkFjijn/TIr/EExGuCayYypNb+kHrcrlh1fFb0P8f6wklHkt2/cN5JLizdcgRygkbz8OcmPz04rYo3yVY9NEHdWILP/hQCKWw160gfEKaRsTSiv717FOeNkgo5+8Phf1XHvJU566jZqGU5ivHAChMYrD9vC35BxFKbqhNr0iE0iRIrSSU4iwARqcrJZRv3r6za8cZ7VtPe7EYsp724iNvjPwTrHp6+sI2KOszr1N51twMQtmhUl57jlnsUxp9LEcoMftL1Ju3UC8mnWOfWgVlgyB41yhNKSleV+9aBdjxrh6CIxwDKs1DGmwk1BP24LcrT8PfQY6E0idbatj6dQ1WVcvLFa2lmKNcsP5KLcHYn5bIW6sJ5cpjt2DAcusJ5YLO5aHLwiDerRKnnhyhHL/xPPwmQyhRyqhijMxW2wq5YUJzPzAa5a174iY3RP3Y3zqW07QntUxBCMzRsue19G9FXSSUmLRB+Igzaww1Qtn4130OOqMoexZ0LcKu8CBOZWnWnJz1Y+W73xXzN3MMIpQmoWnlppLzNdM7bSmhxGjiQjFixHr7NNIO5UWMBByJg3LMsIagUzgeE2JxJmyOvqeX7j9noto+wzcpQoABMeIveGkGF7wfVSbsgDsijUPM/oIyQoLvGfo5yvms8eIu9XUUr0sglHp8+gQpFaV5lB2zTTaNoLAHB684Bf8cdfQN1ksocQ5i668codSi58kjgs6Lv7ieYI1CEWYUY7a68EglOZuDHKEU56cXt0V1gioTd7I/CVHRZ+88gY9+2c+1zPQpkzqoHXA1clEl4blgFeFDN5fcGVNaRlitgAk/VubuC3Fw4TFjHDVC2WT6fqbBKhRUqThyLdyunkGE0oy7oK8PIpT6cIvTykpCKSevonfaUkKJAtqFh9u04xJiEcsGnbr5GJrOOGBoGeJjXGepFxcdCoXv1p5lqRO1RDmWzZsBjl2PzTEtRyh3D7QRSiGLTPca3jBnT4judYmtrtiJ2NcUrZ8D6xfWJWMjZENRmhgef8ppCFpFKMVSQXKyQVp8Ua0ilOjvhVHpKAwuRKTrvrEuaChHKKXpRIVpYDBZtUmxeauvjm/Ejrx5ZWXcmVCiBXxux7KGZcmUbllCJZRoqZa6lhjdlugWMljB/QafHU2n74dTIkKJEk0oESY8h8W50Y3Ohae9le9+nvHdqQ4RSpPuhpWbSi6jiN5pSwll1Ou3djFivX3GZztxppyg0Ai7fp7eOYmPcZWCfIz4fUpTo2FfVSfuhNuPX9qnjFZbzLCx9ZzNJ+gLf2/ZI0beNUqDZ+48fmm3JAmEUk/mFkG8Wmke0uh1oZ6wB+WCzYxYKMXpFjEt5FERcefFSqhnFaEc90kJGLb6jNbpxFt9sU+rMAlpOlHh7+iq4T8lllBiBhjMufxhEMqsgBJMQmpJs29IgiSUXcrDnD1X48hzmY2NuD98djSbcYDpdQoFCSVq6ArR5uJUllbORejbyne/K+Zv5hhEKE1C08pNJdUyNDLlD41Q5kyXHA4OCWCQHLjyENrPO2IEHvtRHXYiJl7Sh5reo6+SudMDWlLF5EpKKKULQH22eftj84FrXaDg/2V/AD6KtFuSvqxdEAbUKwybgu9qFsBuWdYLUAZHqSitS9iDcnJYRgilmEQazWk88uNiMPLfc1qhVq1vlW+m6sA6K8gRSqXnEeZVxzzLQkFdyusPI7mF8jOkTAqPIl/rnKm1zQKKZIVZn5V16t5iZAYJlVDO3n01TgIBIziotcVnR/OZB+D4jdhnKGp+opVU8J8mQqmGonXXiVCahK2VhFKaD9rIlKWEEvUSi45IuEfe2dMmh8NDbYRy14UwQwEI2IeQsxelddadvAOj18clFYihUhYYtXvj55XOwf9HzkIp7ePzqvlh/gH9hFIajX0zIhKqT7ZZkgRCueXsPc16hZ+Wyw2TWvopLrnmlF0QGh4Z57qwB8X5d4VKAqE8GhphT/+nhqlwXexOYDSnsVWEkldcnXfNVtfTQigxa5E4KA4DTa6Hv4Am0/ncUNyZUKIf8sz2ZSw7zUmIhHJhl/Iwc9dVCAx1TCBg5Z7EZwfmmRe7DfWv6wMHrz60W0rFudGtnIvQt5XvflfM38wxiFCahKaVm0rpiEnP1KWEEiNji43Yoqcrt2iTNU0yCBxWh81l85m70GPxcUPzEvwCpdqQ4k4RQ59hmyD67TvNY/nmSgfBt2MdynkIZecq+RzSN2odtEHx7DC7Q1l7M7G8EkoS9a9XGLafuw/d/tQmL9O2Qh6Y0NxXcTroA4o+dNKytFtFqFIwM8hlgBIIpVQahGfNYqtk8ZxpVSWHnPU5qklx+H7dWZ5hNdX5EAilkgsOiszXjVEmQFAwA0xo+Atuv+aMqTwh4kW0JjxdVRmVEma0L2PZszKhEsoZu64wyR5XFXxetpp90GFMTPyAp1MCsSVC6aq7EXccIpQmYW8loRy7/pyhI0/xEqVZBF68emPPv2sSFC7tJnPqZHB0uI1QmpHOTvALdHakjQ81JdFutcVLiQ4PoexUOS9Lcaa3SOV90Gok5AkXCOXOC/fh84XaCCVKIo1pVkJxWlJLgpSUf73sJKyWpBQVCCWPBqV0YLE7AQaTXLj3TC9kYBWh5M3Wo3viJjcUuyAIXX+35oyDFJbw981fVXfIioOC3dcjIpnPG09xZ0KJCQ+QUJb43pqPb/Q/zZMpYUV5o4UyPghl69mHHKyiX9UpBPsvP7T7TKOrRdrktlSWrihWvvtdMX8zxyBCaRKaVm4qJd03PVP3TOwBf3atAJW8M7HmGPxh1UNSz/y0tsmc2hOODq/LmpkhxSLkIFYjlEqi3WrzL5ojrYPVTiwGrdRWqmWpNob0+kcxeYiFv197+MLu6yYQyl0Xw6DLAm16hWg5xWxAivMWZa6R1jk8JAAws83lsOcOlwQCI9Wa41mz2J0A+0FZJ73FqiNvDJASBJj1zs2V7QplTQ3b+tt0QYUil4Mdr23oW81BIgg/9NC94pOZB7mmnCmVp6zMFFdjiyuhmsOvbUtDyVFbLRkJlR0iIqOhOSdWlkxCY6eYUezXHZcNBb9pHBLwA/zTOYcc/Db7BiChfGD3qyRCqRVV8+oToTQJSysJ5YRN5w3JxkiXmMozMZwd3YD9+VnUa/Adac1D0iRonXaDVg1M+Ybln6CbMHiltmwv0s6FHMRqhLLMmG26juf0WM7aV8wDS444pijUgq1ULzLkwXN7nnB8GKMPEkpuYC5sLUVMKFHP9I+DoVAqd3ooly8j66bbH0c1kyeBUEq15njmJbb+emdJBSEPXvA0k61jFaHUPaF4aihHKJV0cf/tU80hACdwaABTL/gQCGX1QpmhW3Vv6KTxN8J723YMqMH6vvUoVu2Bt2181UNCifJXYn9Gq+eChLLNb4ccIsvxo3jP5Yf2YEdBJN7quQj9W/nud9UazBqHCKVJSFq5qSZtvmBI2Fq6RLRSXhzbABIlSgRPo16DXwImlKhdd3JEPbZEzAuN+aGNFCHQRI1QKmksqo2tx3KGvorSnNdq40ivd/f3ZvmpsYgzCgmEEo+MPvtdW4Q8BguN+LgY6xNzO2OOZyyCn263P4Jg+/kwTVMVCKVUa46nEzFZx5SccgFBPP1gHatkg3jHd5d6coRSLgc7zndt76oO/pKHhtRmOep5rW542vDwuXv6UEoTFJh9f9D/FH9/95++Mrtry/r78/MKMG37JYeIa8sGi+kYny3t5h52EFNH6bO9lx/Ygx3RdzddSjrytvpeyPVPhNIk1K0klFO2XIAZu66aNFNbN/ii+PfLavDqzTvLjnFMnbBCZ2mTJ4HTI+uzq4sPX4fha4xp/AmBJs4IJaaY+2TGAYfsNrxrLZAlFVzVaDnDvMjSFIW844nrbfnKHwpnTwNXwp7F5gmPsVDuu/wAOvyuzULZtVp+wEwsWMTHUAKh/HxhEOy8oJNQzjjgIK/Es14xWUfBeyPWHiKUNsQLZk0NGL0tLnIpM/H6ql5VHMjjgW9rAyYHQF9anuLOhLJqwUympxiU/jbRBSTsWcIilD9tvwQnRBI+PPfZSB18tnw274hD9jHM9rX74gN7EB4RSiMIG2tLhNIYfvbWVhLKH7dehF93XjFpprHdYBq4+sWzQ8nRCffIO03yJBAcQyj/PBQKI9Yai8zF4+Vxn/g6TYGGgTtB1yPgZoT24ynvzKkg5KG2o9jW5bzipCjUsxmE6OpL95/Z84QbOfJGfcxaRbKyaPcZO6/YfakEQtllQSDsuvhA01QFC6VUvJinEzFZR31ScTpLnvbiOglNL1Lr+njryxHKb1achmVHHXOwY3+YsvR/IqUAjFwOe4aE8hDXcBhgh3Jd7ljkUnmaOc+NfaszlxN3Xb/cWtFCOXXbJQeRcTMxkesLny1IvAURc6zTq2YB9pwRFCVOjqgL6VN6Wj0Vl7z7XbYIkwYiQmkSkFYSSvzRoq+K2QUjdD/2ywGlRm8zu2uX9Zc6WRJAIVssCw5cg1EGxaiFyGVnFkoUCsdgEgxu0Vr0HMWigDgGHBkt8zuXg9pFssHFe8+g/rS9rDtMR/h1XR/YfTEMOmsMyhHPBy3FT6PesD8JhBJ9wrSmFxQI5SczD2i2fIixzZY2maHjQyKUtrsrRyjlMhxh3SQeieDNu/f2bYGBJkiQWs7mI5RZ0iSTTdVpdN+b0b6ydybTUwyK57X+y2rQeQESSvc88lcilD9uu6T5JMHI/cBnCxJvcbrbnkgoL4TZVR2IUBpB2FhbIpTG8HPJVwr6qUzbbg2hbOybA0qPSbiEMqVnYjgXE2A0b18IjN1w3tAdRYmeUU1LOLVQYm5sTCknjVDmGThPxpRwIyKu2Lezthh5vvK4cUI5q30ZaOibAy7ce2qXd0FCidaXT387zDN9rjoCoZQ++HkaC4RSmg2Dp23ujCl0WY3l+sajfMxZ/V8vaPXdMaCmAwwDl5/i+sDBQBPUlWz1ARDKSt4ZLU0xuK5PVaayEO6mOpxyv4NFXSvAD1suOuTV5vm9YLpccapZnjZCHXy2SD9Uu9fwhp3nw+zPY5SrypCKLJRacDWrLhFKk5C00kKJ1km0UppdxjQtDh/55QSMWE6oJXlSD7gwpiGb/m97r8L4jbbAEL1FiFx2ZqFEkWMMNtCjc6jHt++T0rni6DXqWd9Pn5aET0p7wbk7T6HRL/tYF6jhZvbHikAopUdTPHMWCKUzDUulfrS+qDwSAYgMag7dEqG0wSFHKOUyHMndk+39/SHixWtoPYfPQolJCtzVh7Bi/oyWphhc3asKdP3jqC7lCJ7flRV1kFBO2XLRIfMXzzh6PqrFhFLqSoMBhyjFJfimE6HkuQvW1CFCaRKuVhLK6Tsvww9bzSeUo5FQ+uaAsmO3m4SC67tJlgQj1m2Ectbuq4AR8UYK+gUOb1zMqYUSJUQeR752yHjDO6ZW0oP9YvaetSfv8A6hWG9ic19oUyEPnL3zxEEv0HDHkg4EQil1nucZRyCU0mwYPG0xDee9p1E8VVmdpIkTweu3sUe04oYJLaMN96I1VpQllMtOwiqJIL1ctxgE9jgymtv67c6E0moL5cqeVQBVEdw1l7nc/V3ctSJM3HweztyOmw3L2TbLnzmVLnch7BOfLV0XBsEOUbDf/6rnh63n7rNTIywoI4dycq4qVr77XbUGs8YhQmkSklZuKsxGgF+CZhcklPjD67P0hNldu6w/lEC6NM5GKM3ASZDXcWahxIjPyOi3mn38cI450iVn1k0tBfOL/3vKOKFEbcXOVfPDmdtPoPGv+7VMQVPdgfV8mL/Zk5evNb9sBEIpzYbBMwGtPnj4MYIqB3LlQyeUzqyzYjwEQhn95h30XHwM8mVOBeHPX8Eajg8cDDRBWbI2nO4UVhDKRIkA3st/M/BsKXsdqwnl8h6VWUATfqgmlIKEEjWSz97RRijRLxely/QUJJRSfVs0AmwMvmsPwjs2vA5kSp1MT/e62lj57tc1oXhsRITSJPCt3FQzd1+ByZvNJ5RWpZczCVKubjAQ4Mr4RqyuGa4B6I8zpGFRpxZKPP7Cl5SQO5ZrojGVMIgIsxNpKR/55YANp+9qaSJb99uGRaBHjQIQfOuJgwC14Y5N7MAIodSaug/9b/HD4L9IKKUBNEq3EAXidw6oCQsPXIORMQFvtYtk5ZKDwkCTZ1FvoO1cPv9cKwglL3FW28J6EhKo9Sm+vuyLSoxQCoFtWtrGV10klJjF7dxdbYSycLY0cPG+vrSoSCgRp23n7tuXjXq4607dsUfIE6GMrx0BQITSJOwTIqH8ELKBJPZIBFdjCOVP2y7Bzwaj4b/w94ahjZwTyvL5MkCyJIkdtNBM2kay3aBbwoZg44RyUP3C0LtWQRaV2ZQzv7KV65LrWyCU0vRqPPNACSkkMLwFM0a9+I8SSrTso9yTWhEI5bgN52DuvmusekCRrA5HjkIf0jz1GGjy4tVbbkJpNDJfbi28xFkNB6uvL/1fRei+6JjD/hWnErV6fD39ow8l+qwLcj28fUjTz/K2w3pIKL/48yg74hZKl6r5YNXx2+xEBAum/EQJKlcVK9/9rlqDWeMkeEIZEREBQ4cOhfXr18PTp0/Bz88PJk6cCNWqVWMYdenSBRYuXOiAV65cuQA3AZZ3797BqFGjYN68efDo0SPWbubMmVCwYEFNGFu5qcw4ypVbzIcg3oxHWvM7lWepCdHihF+qRgr64wz7yLkPZZk86SFjqmSa0wrqnVeD4tlh89l7epvb2wmE8uTNx4A6j+5YhMws0vRqPHPFAK2o1+okSejLmbV4WKOiMG6jMcUAnjnHVx1nx/3iOQmEUpxusWbhLExIWlpQTkv8osdAk5ev30K7uXwZmLT6wPJg58xPlqe9q+qgtQ9dCp6JTi9KeqXTHEHtqvniOJh6EbNjaSWURqy9SCh7LDrm8DzEQMp/jt60nzYQoXTlLnAcK8ETynr16kFYWBhMnz4dsmTJAjNmzGDk8Pjx41CkSBEoW7YsNGjQAL788kv7yhMnTszqYkEyiQRywYIFgERz8ODBEBISAmfPngVPT37HXisJ5c/bLwNmJDC7kNZeXEQx+wtmy6kzdY8i3JlSeUKVgplN8Wvkuad1i2VzOOLhaSNXRyCUx2884k6Hp3csve0EQtn2t8Oadf/QWv1WKWxbZkJpkiVxeIGLq3zohDJF0sSM7KkVFOLfObAmfL3spF1pAIPSxMLSQh91imZz+Mha2bMyvHr9DtrN4yOU6I6BKTzNLJ5JPAD9P929oEg4EkqxxbxU7vQuFQ3XitHCLuUZodSidmHUpxUJZa8lx2BjcOwHNhLKJUeu2wPsgobVAfSndlWx8t3vqjWYNU6CJpRXrlyBQoUKwYEDB6BKlSoMk/fv34OPjw+0bdsWvv/+e0idOjX89ddf0KxZsziYRUdHQ+bMmWHy5MnQo0cPdv3x48eQM2dOmD9/PrRp04YbZys3FWp9Td9lfqYckkaJe3t5fRyNRCpyb6qYikpHjFr7wWCZPrULwbHrj7jT4Wkdw2h94chbmq/XaL9y7Z0dkQ9tVMSwBJUVczarT2fH/eIxcJ/vGljT4ZhRKa+1dJ9ioAmSufachBKjc82WMOO1xJqFq95+FnQpD32WHHcglHgSctyFaQ21zh3nPEkjodT60SedExLK3kuPO/iUd6ycF/48dN1eNXBYAGRNk1zrcnTXt/Ldr3tS8dQwQRNKJH8HDx6EgIAASJYs9osEj6vRKtmnTx8oWrQoXLx4kZFMaQkMDISKFSvGuY7H3nh0jpZL3mLlpkLH59/2hvBOhbseEUpuqOK1otIRo9ZJ9a/rA5hq8WhoBHf2Eq1jGK0vWCjbzztsae5knKc4u4903h86oeT1NxUIpVhTVClrDOqzilNt/v1FJXjz9j189ru6hRKlscY2KwG+I81NA8triTW6b422xyxWqLYhDhIrmzcD+/izqqDP6sd+OWHefptvrNayoLPNQqklwMaoCwISyj5Lj8N6UZAiZjdbdFiZUM7ffw3CX7yCgfUKQyI0kZpcrHz3mzxVy7tL0IRSDp3ly5dD69atYd26dRAVFQWffvopsz5u2rQJPDw8oFGjRjB27FhIly4drFq1Clq0aAGRkZGQIkUKe3fYHv+GfplKBf018X9CuXv3LlSoUAFu3rwJXl5ept64kevOwsKDoab2iZ196NIopgMWTx36+2RxSDWmdxooYv5VHR8ICo3gzl6idywj7TATER7HHrgSbqQb1bbpUiS1O/JLKw9pWAQmbDKmaao6gXis4Gzt4mkJhFIso6Uk8l3DJ4tDqk3MHf/2/Xvo8Hug6kqbIaH8xBdKfL9Fta6WCs4i+bX0Y3VdzIP+5V/HHXyAMfgvKNQ6QnltQiOYfyBUd0YoJMFIKC/d55cAMuqCIEco21fMw3zohRI4NACyprVZKO8+eQmVJ+xk/40EuFaRrKbfSiKUsZB+UIQSj77RMokWyzVr1rAj73HjxsH48eMZkbx8+TIMHDgQcufODTt37oSlS5dChw4d4O3bt4xsCqVjx45w584d2L5dWfB75MiRzP9SWqwglMNWBzv8YMz6RRChNAtJa/tROmLUOipaJ9FKeSQknFtsWusYZtXXGmCjZ9z/MqHMkDIpl4i2HKFUIjpS30qMAkZ5LUzBqVYwG9SYZiVMJ5S8R/tq87P6+uzPykK/v0846KJWyJ8RAq9FWDY0kjO03o3WmWL0907lmJydFgul0d+1HKFsVzEPLFUglBfvPYP60/YyDDEzXIfK+UzHkwjlB0go165dC+3atYNKlSoxyyJaHNGf8smTJ5A+fXr7ig8fPgyVK1cG/H/cCC1btpS1UL569QqwT6XiSgvl4BWn4J+jxnM5S9fyTYMihjPLmP7rpA7jIIC5tg9eNW6t+7J2QRhQrzAcDgnnFpuOr9thln6gs/mnT5lUUUgaNTvR+vKhlsypPeHh82jV5eXLlBJ2D6rloMuqdBSLgv9iqzJGAWPB3MtqpXkMoSxusoWS1ydabX5WX5/Vvgz0W3bSHkCEllXfXOksTfeI5GzBgWswKkZfVOsa53UsxxJuaCGURi3GcoQS3WQui4TSjwwNgGwxFspL959BvZ9shBITeXQkQqn1Nmuq/0FYKDHCu1+/ftC8eXNYvHixgz+lFI0XL16wQJ0VK1YwSyX6UGJwT4ECBexV0YeyZMmSLGKct1j5lSKOsOSdD089K3TfeMalOtoQMMuXqk+tgjCwfmE4ePUht5SLtpmaV9uo8z7PTHitdDx9JbQ6vFmF5Ahl6TzpWZYoqU6i1HKJR4zostZ5QZAqPM3L5IIxTUuA2YTSWSS/6qRcWGF6u9Iskl5IBfpr29LM6oYZp6wqRgnlbx3KwtRtlzRFeRsl+DjnL/864VRhQ0woL99/BnVjCKWQKcxsPK1895s9V6v7S/CEctasWdCrVy/o27cv/PTTTw5H12ixDA8Phy1bYv1ydu/eDbVq1YJz586Bt7c3ZM2aFaZOnQpdu3ZlWLtjlHfvJcdNEba2ejNR/+6NQK+aBWBwgyJw8MpDbimX+FqRKwSptWbWiS8srBiX92NSjlAK+ohqsjZowUqcOBF04SCULcp4wZhmxaHYCHN9KJ0FXlmBq94+f2lbGgb8E0so8d/Lgm5Y6keM5EycAYln7uIj6zkdygImk9AiG8QbDKY0Fx5CeXhIAGRPZ/OhvBL2DOpMtVkoUXe5S9X8PMvUVIcIZSxcCZpQXrp0CUqUKAGNGzeOE5GNR9779u2DJk2awJgxY5gEENbv3bs3O/JesmQJQ2HYsGEwZ84cJhOUL18+GDRoEISGhkJwcLDb6FBKU01p2u1UmRCIQQB1/vAo98CVh9xSLvEFntFoUJ55o55o+Av1Y1+evhJaHd6c8nKEskSutCxHu5rlHC1YSZN4cBHKlmW92JGk2YSSN/govu/ftE9LwYDlp+w6qkgo/wm6aWk2LiRnfxwMhe/XneVevphQot/ntO3aCKXR+yFHKKV+mYeG1IYc6WxBtpgzXNAURt3lz6sRoeS+2ToqJmhCicE2SAjlSqdOnViGnJUrV7LAnAsXLjBfSrRaYpR38uS2LxgMyMFMOyhs/vLlS/D392dH3UgutRQrv1I6LwiUzUyhZX5UlxDo7u8NQxoVhX2XH3BF3sYnYrypAY3MkdeP0MgY7to2V/oUcPvxS9Xp5c2UEvZIfCiF1HkV8mV0ms9+9mdlWIrSLgvVj7ytIpQJxa3hx1YlGaEUys9tSsGKY7dkBeRVbxpnBT2EUqzrifd32vbLmiyURu8Hj4VSTChDHjyH2j/aklSgTB4mrjC7WPnuN3uuVveXoAml1eBo6d/KTYW+NJj6jAohYAQBIa3k3ksPuCJvjYyVENr+lwll7owp4GaEPkIppM6r5J0RDocoRyHPaFcGUnh6wOcLj6puh1bMQlkCio7YrFpXS4WE6taAhHLl8dumyIUp4YXk7M9DoTBiLb+FUkwoMZDo5x3aCKXRUwEeQnnw29qQM73NQnnt4Quo9cNu9t+oatKtureW7cNV18p3P9cE3KgSEUqTboaVmwrTya0+cRvQ/6T7omMmzZi6+a8hgF/n+JW++2IYV6DEh45P5tTJ4OHzVx/6MmXXh5bH6+GRqmuXs1AKUbXSqG5pZ3hsi7I9Xf9QJ5Sty3nBqCbmE0qjBEYVIJAKlM4AACAASURBVIsqIKHEZ75cznSzhkRytuhQKHynk1DObF8Gft15RVMub6O/OfSPxEQf607dUYQB1Uvw4zlJYg8IffgCahKhNGvLqPZDhFIVIr4KVhJKYQa3HkVCtUm7+CbkBrUES4YbTIWmAABdquaD7z8uDrsuhnH5tX3ooPFGOn+IOPCmDs2TMSXsHewoG1QgSyq4+uAFKOX0FvBCUpTKMwl0+1OdUH5aLjeMalocinxnroUyoVqh0acSSdPOC2GWbT+jhBIt0DN2XYFzd2MTfIgni9qi+Kx5HPna/uesaZJB2DP9H3HoL1mvWHanhBIHEySCroe/gBpTbBbKYY2Kwv/8yUJp2YYCACKUJqHrCkJ55/FLqDLRpvqfEMqHnm0kIdwD8Rw7V8kHX9fxgaPXI7isRgltfVrna/TlpnU8d6rvnSUVhDx4oTolPBrfN7i2gw5l9rTJ4d7TKFBLCTq1dUlIkzwpYFChWsEjbxQ2J0JpQ+qnT0vC+lN3YYfVhPLwdfhuzRm122O/Lj7yRmmj2Xuuwtk78oQSZXpQ97H0mG32jFTC3uEeUKai2ocMNkFpq9W9qjpYKPF91L1GrDygkTmI27ri3W/WXK3uhwilSQi7YlPdexIFlSbsMGnG1ndDucKtx1jLCBidiw//1J5J4NmrN1qafpB1eaVzPsTFF8yamkXAqhU5Qim0UUsJOqmFL2RI6QlfcLjpYC7vSS38iFDGgIuEcsPpe7D9/H21WxTnOpLz5cfUE2GghXLx4eswXCehRJeGOU4I5agmxaFTlXzgN3ILPI2yPW/MyK2u5ruL46Ck1ZreVR18KFHhApUuzC6uePebPWer+iNCaRKyrthUYc+ioMK4hEMo8YGiRZLCpFtB3RACXAiYYS3hGsgNKxXOloYrw4kzQqkW8IIWqhzpU3D5fTcpmRMmtyRCKWwVtO5uPnMPtp7TTijRfWDZ0Zuqu84ooUSXhrn7QpiElFwRCKU4D7zqpDgqKOWSFzctmTs9rO1dFa4+eA4BMVHe6FvZsyYRSg6IdVchQqkbOseGriCUGEBQbqxyfnGTlmJaN+M+KQHDVvMfp5g2MHVECHAg8F8mlLz+zUgo9w6qBfmHbIyDqJoEDL7A0Vezx2L1QEIklFNa+UHh4eRDiUCjjNDWc/dgy1nthLJthTzwV+AN1V8AEsolR65rekaLj7zVCOX4T3wB82zHC6H0Sgdr+1Rz0KEc3KAw9KpZUBUXrRVc8e7XOqf4qk+E0iTkXbGpHr2IZv4oCaXgkdc3K0nuKKHcr//aPHnFvT9EXIrlSKsYTCFer1cGG6H0HhqXUDrLhY599K1dEIrlTAs9Fh9XhfDjkjnhB0sIZcKM5P+hVUnYcf4+bDpzTxU7aYX2FfPAkiN8hFKrJJ2YUGLg0PwD1+D0rSeyc0RS3KKsl+mEskL+jBB4TVmuCieDaUHXMUIZmylnUP3C0LsWEUrNG0pDAyKUGsByVtUVhPJJ5GsoOXqrSTO2vhupWK/1I9IIhAA/AjnTJYc7T6L4G3xANYVsN2pLQgH0HjULyAZuqGU9QekWzKbDSyjxeeEzfJPalDRdNypTo2kwEytPaenHJIM2BN/V3GvHynnhz0PXVduhhdIIofTNlQ6ev3rD/BTlCuYnb+yX03RCqbowAMD9vf7L6nDp/jOoF5PLmwglD3LG6hChNIafvbUrCOXTqNfgNzLhEEp0LP96WWz2B5Ogpm4IAVMQ4M0WY8pgbtaJkI/byLTU8mSrEU7x2GihJEIZiwgjlJcewIbT2gklqjksPBiqemuRUOLR+JBV/KdInkk8IPrNO9W+scLcjuWgbrFs8UIoi+dMCxv6VoeL955B/Wm2XN4D6/lAn9qFuOaupZIr3v1a5hOfdYlQmoS+KzbVi1dvoPj3W0yasfXdoFP+yH/PWT8QjUAI6EDgv0woMQr25M3HOlCLbYKJFp7FRO8a6ggAGvvlgJ8+LQWFhpGFErHEAKV9lx/Cv04EvJUw/7xqfnYUrVaQUP4deAO+tYhQ/vl5BUAlALN9KNXWhdfRpWNjv+pw4d5TaDBtH2syoK4PfBlAhJIHP711iFDqRU7SzhWE8mX0W9NTk5m0fNluUKZh4qYLVg5BfRMCuhFA/8Bbj9TTD+oewI0blsmTHo7fMEgok5knP2UVoUyo4vWBQwNg7AbnGWGUthe6GszdF/+EctkXlaCid6Z4IZSYb35Tv+pw7s5TaPSLjVD2r+sDfYlQWvpUchtC+fLlS0icODF4enrC+fPnYcOGDVClShX2v4RQXEEoX715a3oUpJXYYk7VhCTEbiUW1Lf7IfBfJpTl82WAoNBHhm5KGhMJ5Ud+OQCDPMy2UCZEQomEcNhHxeCrv0/AmpPKKQaVbl53f2+YszdE9d6ihXJZ0A1NgZNajrxRBxIt4fFhoUQVg81f+cPZO0/go1/2Myy+qlMIvqrjo4qL1gquePdrnVN81XcLQrl3715o1qwZrFixAooVKwY+Pj7g4eEBz58/h7/++gtatWoVX/hwj+uKTfX67TvTH7jcC9RRMXhkPaj1w57/bL5kHZBRExcigJI4NyP+mxbKCvkyQmCo80hZtVuROlkSFpRhRvnINwegDE1Bk4+8EyKhRFehzlXzQ/9lJ2HVidua4UWtxVm7r6q2Q0L5T9BNGLzytGpdoYIWQokWQrQUxgehRJ3VLV/7w5nbT6DxrzZC2S+gEHxdlwgl983WUdEtCGW1atUYifz5559h/vz5MGXKFLh8+TL773nz5sGJEyd0LM21TVxBKN++ew8FZOQ7XLtS/tHOjKoPtX/YbSh3K/9oVPNDR0AtM4vW9WOe6hsRkVqbfRD1eaRX1BZKhFINIX3XBUHw/v+chFXHtRPK3rUKwIxd8U8odw6oAd5ZUscLofTJlhq2fl0Dgm89gY+n2wglHnfjsbfZxRXvfrPnbFV/bkEoU6ZMCWfOnAFvb29o1KgR5M6dG+bMmQPXr1+HIkWKAB6Hu3txxaZ6//69rMCwu2JzdlR9qEWE0l1vT4Kbl1ruaK0LypspJVwPJ0KpFTehvpoOpZZ+G/lmh1/bljH9gzkhWijHNC0OHSrnA72EEvU/f9l5RRV+ZqE8ehMGrzDfQolHzhv7VgcPj0TxQigLZU0N2/rXgNO3HkOT6QdshLJ2Qehfr7AqLloruOLdr3VO8VXfLQhllixZYN++fZA/f37IlCkTLFiwgB1zHzlyBJo2bQr37mkXd3U1oK7aVPFxfKAXy3OjbYTy/tNXerv4z7RDvb5j1435tH3oYAUUyQo7LoSZtsx8mVJCqAWEcmm3itBu3hHT5mlFR2YceZup8UiEMvYuj21WAj6rlFc3ocSj3Z93XFbdNkgolx+9CYO0EMrEHhD91rlsUPca3jCoXmFIktiDzSE+3lmYq357/xpw6uZjaDrDRii/rF0QBhChVN0XRiq4BaFs3bo1s0JmzJgRVq5cCXfv3oWrV69C9+7dmdUS/SjdvRChjHuHzo9uADV/2EWEUmXz4tEhugfEx4PX1b8r3rRwcvOqUzQrbD/v/oQSX9Tufi95M+U42x9mWv+QUE5vW0Y2I4/WPeqRCODde1srM+conodaHnOtcxbXx5S17SvqJ5R4rDt12yXVKVhFKL+u4wP96sTK88THbwEtlChDhT6+bX47zLDoU6sgDKxPFkrVjWGgglsQygcPHkCPHj0gJCQEvv/+exagM2DAAGahxECd7NmzG1iia5oSoYyL84UxDaDmlN1w76l7ZiNpWCI7RLyIhiMqabys3kHuRih5o0T14ILWF9TW04M5iiRvO6c9t7HSPDHPtFKWDz1rE9okBEJpRkBS5tSe8PB5tBGo7G3xtzijnTmEMolHIngTwyizpklmiQ93plSeEP7CnLVLAZzQ3Bfww0vvkTcKeP+wlY9Qrjh2CwYu508+4clhoZTK88QHoRQwFX9coG/poPpFTNmv4k5c9e43feIWdOgWhFJuXdHR0UxCKKEUV22q+Pxxar0X7k4o8cG9MfguExCOz4LyK8FuZKFc27uq/ZjIbFzQ+oKE8nCI9gjjBsWzw+az5rm/eGdOBSEKaeOMrDshEEotWWyUsDDTSof3dtZnZUzxEReTnoRIKCc294U2Bgjl4AaFYfLmi6pbGPfpymO3YIAGQlkxf0bVj0GpgLi7vLMw+v2bBkQoVTeGgQpuQyjxmHvu3Llw4cIFmDZtGuzZswd8fX1ZUE5CKEQo496li2NtFsq7MfmSzdC+M3MvIKHccf6+qceoeuaHGUeCR7rPkXfgsACoMG6HnqWothn/iS+sO3VbF6HEY9GNwSYSyiypIOSBfB5i1YU4qZAQCGXSxIng9duYc2GdizWDlApDm0kokyXxgFcx6QGtIpRmWmel8E9u4Qety+fWbaHkTSihh1AeHhIAlSY4fzZIc2Y7I5SJEgG8N7YNuXdvjxoFALExu7jq3W/2vK3ozy0I5ZUrV6BixYqQLl06wJuDpPKbb76BTZs2wdatWxOEuLmrNpW7fO3xbMZLYxtCjSm77ITyr/9VgiPXwmHadnWHcZ7+jdZBcrP/ygNTSYqeOWFO5NNuRCjxRbPn0gN4Gf0Geiw+rmdJim0Qc7RQHgoJ19wvil/ryW2sNFCBLKng6n+UUCb2SAQoQ2akZEubzDT/6PrFs8GcDuVM8T1NntQDol7bAkesI5TJLNPXxTzercrpJ5TDGhWFcRvPq95a/J2vOn4L+v/Df+TN87HkroQSg4WGNCyqiovWCq5692udV3zUdwtCiZHcGOmNFsq0adPCqVOnIG/evNC5c2e4ceMGs1a6e3HVpkpIhPLyuIZQY/IuuBNjofz7i0oQeC2Cy2Fc7X6jZEnrcrnhN46MEEp9IbkJvBauKxuF2vy0XHdHQinM3+z9hlbhdSf1EcqPS+bkym2MYwzhyE+MkaBXwp5ruVVcdXleulwdWVhJ7Fumd5jlPSpDq9mH9DZ3aGcmoUzpmRgio9+y/s0kveIJmxnhLgXwh1YloWVZL90Wyu8aF4Mx68+p3hfcp6tP3IKvl5lLKPFYGY+XeZ4hrrRQom/4kEZEKFU3hoEKbkEokUwiacQsOWnSpGGEEqO70VKJlssnT54YWKJrmrqKUNaZuseSl6AVKF1BQjllN9x+bNMRxdyuQaERXA7javO5Or4RrDh2U1PaMGmfSChP3nwE/xy9pTacpdfx6PDU9/UMWWfMDBLAFw3Py0APKEj20EJ58Kp2C2WzUjm5yP+s9mWg5xJ1y+p/mVCa8SJHl5bCwzfr2QZx2phJKMWC61YRSquixxGYqa1LQvMy+gnl9x8Xg1H/xh+hxGNlPF7meYaY8WHDuwG/8PeGoUQoeeHSVc8tCGWGDBmYDmWJEiUcCCVGeTds2BAiIrQ78OtCw0AjVxHKe0+iVH1YDCxDtqneiNCQ8Y2g+uRddkL5T/fKjFBO2aLuMK62Bj1pw6R9YoDI+btPYfHhG2rDWXodra0nRxgjlFULZoIDV7STNLmFWUkoMeBgnU5C2bx0Lq5UdLM/Kws9Fh9TvWcoLXL5P2qhRHCWdKsIp289gUmbL6hiJVcBTyDkcm976/BNrVcsG/zW0Zwjb/RJfhZlSwmZEAnlT5+WhE9K6yeUo5sWhxFrz8a5Zc3L5HLIvIO/8zUnbsNXy05y338e6/uQhkWguxNCWTxnWjh75ykb0wzXC97JCznSeevz1nPVu593PvFZzy0IZZs2bRgGS5YsgfTp08Pp06fZ0Xfz5s0ha9assHz58vjEiGtsV24qs48h1RbolSEF3HqkPVvRtQmNoNokR0J59HoEVwSi2pz0aKjJEco6RbNBwI97ANPwnbtre8i5umRImRROGCCUGCXulztdgiCUk1rYCKUe8ovHgChzolbmdCgL3RepE0pMz3bp/n/zyBsxFD4c9D5P8ARCLvc25lG+eP+Z2m1yuG4moRQHC1lFKK3yzURQMKd501K5dB95j2lWAr5bcyYO/tIPLbz/a0/ehn5/m0so0Yfzf/7e9vGl+2v9l9Xs+bVdSSi7VcsPwxsX07QveSq78t3PM5/4rOMWhPLOnTtQs2ZNCA8PZ8fbGNmNaRdR6Hzv3r3Mn9Ldiys3ld4XgF4Mc6VPYbcyaukDH1hVJ+60t0Wfq6Ohj3RbRMRjY99aNdTkCCUKCEdG26wZxUZs0bI80+qaQSh9vdLpOkaWW4SVFkojhLJ1OS8u94S5HcvB//48qnp/9BAf1U5jiJqrf6M885LWMUoo0e3k5+2X4qT5E1ugeOeFGqN438zADX9PjyJfs6ETIqH8pW1paFIyp25Cia48Q1cHx4Fe+qFlFaEc/lFR6FZdmVBu+9of6v60l83PlYSya7X8gP6lZhdXvvvNnrvZ/bkFocRFRUZGsow4J06cgHfv3rHj788++4xZKp0VPA4fOnQorF+/Hp4+fQp+fn4wceJEqFatGmt28uRJ6NevHxw9epSldezbty8MHDjQ3iWONWrUKJg3bx48evSItZs5cyYULFhQE9au3FRmPHS1LM4sQrmiR2WWXnDCJn1HbFJCqVVDTYlQ4t+j37wDn+GbtMBiWl3U8zv+XV3dL1O0UJbIlU5X5LSrCSVKoqCFcv8V7dqfbcrnhr+DbqriPr9zOfh8oTqhxHzDF+5ps6SpDv4fIpTo0nLlwXOoF0MOBGxK5k7PUt5pKXhSMK+TI6HUkhpTLBUk9ifOnjY5SxWICQzMLFYRVZzjr21LAwag6RU2R7eSb2WC0n7rUBa+EFnu44tQbu/vD3Wm2gilWITezPsj19fnVfPDiI+JUFqJs9sQSlxkVFQUXLp0CZIlS8aCcpImTaq69nr16kFYWBhMnz6dRYrPmDGDkcPjx4+zf6O1EzPv9O/fHw4fPgy9evVihLFLly6sbyST+G/MH54rVy4YPHgwy9hz9uxZTcLq8UkozdCUcwZ0znTJ7ZHaqjdEVAEfWNUm7bQflyOhPHHjMZekhdo4eiQvpH0KOXPx7yihUmDoRrVhLbmOL8Bj/xVC2dKPRXnrIZS8aRsXdCkPXRYEqd4rKwilcNzn6o8+1cXKVDBqoUSXFoySF6xNwhDl8maAoxrz0ssRSi3C82Jrs1gjEgnluj5VocJ4c3VVsV+rMoBhxiCUyNJLKFF2SC4/97yO5aCbyHKP9x8/7vr+dYJ7+/D4UI5oXAw+r5bf3qf0t4A5tjG41NWEsnOVfDCySXHutfJWdOW7n3dO8VXPLQjl27dvmdVw9uzZgBlysKRIkYJpUQ4fPhwSYUiiTEH9ykKFCsGBAwfsWpXv378HHx8faNu2LesDCWZoaCgkSZKE9YDWzFWrVrEIchwrc+bMMHnyZJb6Ecvjx48hZ86cMH/+fBB8O3lujis3lfQHyuszxrMOuTo50iW3a0lq6UNKKFf2tBHKsRvUNdLUxtHjUC7tE32NOlSyuVPgvsk/JH4IJb4Ajw7/j1goW/qxKG892YnaV8wDS46oB1D98XkF6DQ/UG0LgdmEUvzC4iWUjf1ywPrTd1XnakUFo4QS218Je2a3NglzrOSdUbNwPeZpn9epvIOVXksUvjiLkjgCG4nf4aEB0Ojnfdw+0uiW8c3KuEfG4ntgJaGc2b4MNPLVTyjxyFyOJP7eqRx0/SPWcm8VocQo8y5VlQnljgE1mN86FquNIeJ7RoTSiqeIY59uQSiHDRvGCNyYMWOYTBASTIz6HjlyJPTp04dZEeUKkr+DBw9CQEAAs2oKBY+rGzRowCyNGOSzdOlS+7Xt27dD3bp14f79+4xo4ngXL15kJFQoeOyNR+doueQt8Uko0fLX0iQ9OLn16n144gOr+uSdcDPCFtCzsmcVOHnzMZdGmhrueo5rnBFKvMZLAtTmpvW6GYSyeK60ml/iSvO00ocSrSdoFdFDKDtWzgt/HrquCu+irhWgw+/qhLJojrQsyt+sIvYd491LvNqaZs1R3I8RQonf+NcmIKF8brc2CX1XL5RZ8/2VI5RagqbEEczi42iBUE7feZlbrgxx6b30uFMRfb0f2Tz3cfZnZaBBCf2EUknlQOoKguvEj7svTbZQjvy4GHTmJJQ8ucF5MOOpQ4SSByVjddyCUGbLlo0dOTdq1MhhNStXrmSEEtMy8haMCG/dujWsW7cOkKii7NCkSZPszc+dOwfFixeHoKAgJpreokUL5r+J1kyhYHv8G/pl8pb4IpR7B9WCB8+joMUscwSG5dar119IjlCib9VoDtFdNdxNIZRNi0OHyvnsQzkjAcVypOW2cKjNXXodRZKPDq+jm9CiD6WrCKVR3TgUbcbIUimhrOGThWXncVbwhbDwYKgqvCiH037eEdV6Zt9TPYSyZuEssPui83WrLkRnBSOEEvdByISP4OqD53ZrkzCNWoWzwC6Na5IjlFqW1aqsFyyPUQAQkz3870NDApiP9IRN52HBAef7R/i467n4GGw6o5zm01pCWRYalMiu+8hbyYd4Qefy0GVhrCsI3v/1p+9An6XmHnmjbFFHJ8/VnQNqQO0YC6VnEg92b1xRiFBaj7JbEEq0Ih46dAiKFnVUsUfyhxbEZ8/4HOfx6Bstk2ixXLNmDQusadeuHYwePdqOJFotCxQowCygaKHs0KEDs4h6eHjY63Ts2BEw8hytmUoFA4Dwf0JB0luhQgW4efMmeHl5WXrnxMQHHwoY6NJi1kHLxtQrkYFz85+8C25ERLK5repVBU7ffAwjOUR31Raj57hG2ucYDYSS1zqmNm+563hEFzRMP6HEIIgUST1cYqFE/6c2vx2WTTvXsER2py9hXDsSSrRQ7pWQRx5i1aVqPlVCgGMs/V9FaDdXnVDqiUZ2dn/1EEqr3VWczdcIocToXIzyliOUGLG97dx9TT+FgCJZ4ffOjkfeWjoQB2yJgwgFCyX2FfrwBdT8Ybdst5iWb/Gh67CoW0UokycDfPHnUdjqZA16/cp51oTBM/WK6yeUSh9UC7uUh7Bnr2DE2jMwqYUfkyayglCqPVd3DawJtWLuAxFKnh2RcOq4BaHEKGwMrPnjjz8cAmE+//xz5vv422+/qSK6du1aRh4rVarELItocfT19WVWTzkLJQbtILls2bKlrIXy1atXgH0qFTyOlzuKjw9CefzGI2g+My6hNCs9mNgn6dNyuWHZUfVIW/YAn/gRy+V9PdxGKFf3qgJn7jyV1UhTvcGSCnqOa9yVUCJhD9RJKFN5JobNX/nDwOWn4Mg1cxIAODvyPjI0AL5edlJWooiHyP2IFkoZQlm7SFbYeSHM6TZA2Y/f919jdZzlK8YUn0h61UqJXGnhzG1rjrzxAw8/9NSK1K9Nrb6Z140QSozOvTK+EYQ8eG63Nglza+SbHTYGK1v35NZglFCKA7aUCOXNiEiWaEGuIBZv3r6DJIlthoVufwTB9vPK+9FKQonBM3WKZdNtoVRKiYm+xXgSIF7nhtN32fF+nGejgpYlT1CO2Dcd+5We/OweWNNO7IlQmvmLjv++4o1Q1q5d2776N2/ewP79+yFHjhxQvnx5SJw4MZMPQnKGeb5XrFjhFCmM8EZSikLoixcvtvtTIplEqaBFixbZ28v5UGJwD1othYI+lCVLlmQBPUrFnSyUJ248gk9kCKVe30fpmsWEUhyhp7Z95QglyrSo5VnGl0v/ej7w0S/7FYfAvpUehmrzEq6rfUmL+7HSQmmEUCJJL50nA3w655BLCGXg0ADmcyVHXnl8eTGt3NqTd+Icb+M936FCKDF1mpC7HV0Eyo2VP0HAjEyt56i7gPjmSgfBt81L6yq2UIY9i4Jm0w+oqiPwShzx7mkt9YwQSvR9uzSuIVx7+MJubRLGRg1FtEJrKfhBMd+AhfKzSnnsGa/EiRiEI2+cy53HL6HKxJ2KhFJ84fOFQU4/cPRKqfFggh8ZAUX1E8o1vatCsxkH4gyFFsqahbM6/H1j8F3oJZOmFFOkyj2neQglZiBDfV+hSAnlnkE1WUpeLGK5Jx5sjNTpVDkvjGpawkgXsm1d6e5m+uRN7jDeCKUg28OzHvSvVCqzZs1iUkCoL/nTTz85HF2jHiVGjl+9epWRVCxDhgyB1atXsyhvtEJiJp6pU6dC165d2fWEFuWNP3AMdJF7gJjl5yO2dIr9X9TunZRQ4oPu0v1nMHjFaadNMZMC6io682nEvpUehmrzEq6r+fqI+8Fo8EWH1QNCeMcW10Mf1SND9R15I6alcqd3HaEcFgA9Fx+Xtb4pvcjEa1UilOhD58wihH3gseScPSGsu2PD60BZBULJQ2yxDysJJfbPI7wv9WvTs3/0tjFEKJN4wKWxDWWPkaUp/njmh36XC7pU0O1HjGThj5iALTHZExPK+0+joKKCfJDYKo/zHbT8lN0nU27+SoTyy9oF4dedV3iWrFgH90Qt/Kj+56RDqkTeTjf2rQ6NftkXp7rQr/iC0jNUScuSh1CisHq7innswzgjlMmTekDUa9f4UKJRYDQRSt5tpKtevBFKXbOVNELNShRAb9y4cZyIbDzyRsKIOpRNmjRh+pKBgYHQs2dPRjI7derEesPAnTlz5rAo83z58sGgQYOYb2VwcHCC0KF0BaEUCwWLvy7V7iHOreaUXRAac+S9tndV5nPV/59TTptu6FsNiudUJ5Sbgu9CT5mva7V5uRuhFPy8eCODxetDTNGHEi1ygS448kZfT7TeyFn2xCnVlO4B5ilGC6U0EAV1CLefd+5317NmAZi1+6oqoUQ1AR6fYj+vdCyXtVlFmiGEi1B2Kc8izSdvNp7fXus6jBBKtCxdHNsQroe/sFubhPF5MxqJ52uUUIr9a5UI5YNnr6D8OHmrtpRQPnz+CrouDILkSRPLWuPlCCWSqK8CChnWvEQd1VqF9RNKpVMktIbXLpLNYZsoPUOVpJN4CCWS0TYVlAklBpL6T7G5HriSUKJRAI/jzS5koYxF1G0IJd4UPGLGPN4oaI6R2N270dFv+QAAIABJREFUd4c8eWI3pnQjjB8/nhFCuYKEceHChSyaG62XeISOR+oDBgxgkeNCwYAc1KZEK+jLly/B39+fzQPJpZbiyk0lDcrByOmmMkccZlkoMZOLkGli3+Bain5IUrzkCOX1iEhVIV38wi6WM62qhXLzmbvQY3Fc/x/e++YuFkrBiqKHUKJos5+Xawll+3mHZXNg4337ePp+JhKvVKZ9WgrWnLwdh1DyBHL0rlUAZuyyEUpnR948llLso6RXOjilgVCKj9zl1qeHUArHkHL3fnJLP1VrPu9el6tnhFAiEbgwpiHcCI+0kwNhDCRWSzn0Qs0klJineV6Mf60SocRnWJkx22QhkxJKodKRkHD4VMYfV45Q9qhRAD6vlg8qjDMmoi7sCb0WSqVntHCULgZA6RmKGa0Gr4x7ksRDKJGMflpemVCK5+dKQoluEWOb+Rr5yci2deW73/TJm9yhWxDKM2fOMCKXMmVKFimNJA9TJb548YKJliO5dPfiyk0lJZSnbz2GJtPj+syY5UMpzo174NvaLD+3WsG0absH1WL+VehnhQXJz61HL2V9dsT9bepXHVAjUO3Ie/OZe9Bj8TG1qSheH9WkOHSqwicbZOWRtxmEMig0AlqZpEXqLCgHiRwGgAmR+w4vp6+qQ9PpB+CVExkQJJT5MqeK46LBQyj71i5ozxuNllIlaxNabeU+sKQbQWuKQLGF1CxCKQRKJDRCmSJpYjg/poEsodTjb4xR/gsNHHl39/eGOXtt7hBisofBMweHBLC/P46MhlKjtRFKtPrL+ePKaZj2qlkAOlc1TiiFPdF/2UlYdeK25ucbBs7JHe0LwT6OhFL+Gar0MWM2ocR99PL1W81r1NMAEyOM+4QIpR7seNu4BaFErchUqVLBkiVL7AE1mIYRc3nj/2vRg+RduNn14pNQBt96wixD0qJX7kfaj5hQHhpSGypPUCeUF8Y0YMdFYkL5b59qcOfJS+guyicrdx82f1UdimRXJ5Rbz95zyE2r9Z5qIZRip3+t46jVF156RiyUOAYGHfy68zL8FcgXha80L2eEEn0X0T/r/tNXcZpv+9qfBYc9f/VGcck/tynF5Eo+mXmAZU0SSr1i2ZzKtGC9fgGF4Ocdl1kTZ4QS95nc70E6Ka1H3n1qFYTpu5T94/RYKP/8vAL4+2SR/XhyZwslqgucHd0A5CKnefVCxffDKKFE6+DsPTbrtRKhfPLyNZQctVV2bypZKI+GRsgmjdj6tX+cHOZoQe9cJb/ih47ac0C4brdQ6iSUJ76rC6VlLLFzO5YD/HDjIZRK6RsRp7l7Q5ymz1ULdtz/TS2oNsl25O1KQolKABhsZHZx5bvf7Lmb3Z9bEMrUqVOzPNvoDykuePyNlksMlHH34spNJbVQKhFKLbJBzshn+pRJ4XHka3YLlL5+xfcnpWdiODe6AftT7R92Q0iMhRJ97NAxXpz+S+6+bvnKHwpnTwMFh26ENwrHp/hgQ627/4ly02rdI2oZHcT9WUko8QWIll81QimXKhDJk69XOvtUMTLzr0D19ITOsFIjlAFT99j3g7gf9N1qOftgnGt5Mqa0WzQFQtlryTEHaRkeC+VXdQrBtO3qhJLHlxPnrVWHsm9AIfglhtDK4aeHUGJWn+qFEh6hTJ0sCZwZVV+WUIqPn3l/kyhng5Y5td+AUn9idwglQvks6jX4jtRGKI9dj5BNGiFnqcMPDjzxULKcC3NXy9CkxYdyYD2fOBmA8L6U+H5LHKhQ87R+8exchBL1YlGKTFqEZ8OQVacVP1y/aVAE0JovFOk9jT9CmRsmNPfj3ZLc9Vz57ueeVDxVdAtCib6NaIUsW7asAwzo/4hpEolQOu4OKaE8c/sJNP41roUSsz48fG7Lje6sYFSmd+ZUiqnJ0qVICvh1j8WZZUgYQxA9xn9LCWX4i2jVPMv49e+TLQ1z+MfABrmoSXywbT93H7oZIJRqOWfdjVAiAZaKwgsR8cJch60O5sp3Ld0PpfOkZxZDTJu3qGtFxZfB8e/qQrVJOyEyOu4xFQoW47E7BjSIC1pGBOJvFqEMHBag6KumFOUqXbPWXN4D6vrAj9suKf6U9BDKxV0rQrVCmROchRKzMwWPqg+3HkXarU0CMOJofLVnj3AdrbRordVLKMXuEEqEEn17CwzdKDslJQulUtIIOUKJc+hYJZ+inJUwMO7ddSfvwNgN52XnoiXKe1D9wjBli2NA18WxDaDw8M1x+saUjJiBR1yUfChRL3aAE0I5dHWwop8sEuuB9QsrPkPEblOulA1C8fuJLYhQ8v4m9dRzC0KJR9uYmQb1JjNmzMjW8fDhQ5ZCEbPorFq1Ss/aXNrGlV8pzghls1I5Yc1JmwZc4Wxp4OJ99SxDLcp4Qf7MKbkIpTO5FjHgwgO69o+7IeSBzYcSyQ8SU7W0eHh0WihbGtZGye8J+99x/r6qtdPZJtBCKNH/ZonGQAPeDchrocT5jhJlGULZG/QX9MA8eDEFLcANf95nD6LinQO+xDKk8gQkWeiqIBTpCx6P08qN2y4beIPRm5/+dgjuPolyGBZ9twTi/0vb0oA6hb2XHIcNwbEpVXkslKhvN2z1Gda30rEeXkMLd/1pe1WXXihrargc9ly1nlBB7uUtbqyHUGJWk6oFEyChTJ4EgkfWh9uPX8bxqUZfwpkx0fi84BollGLrtRKhxLmgPqhc0IwSoVTS+JUllAGFAOWLlOSsBCzUgqG0EMrBDQo7KASgnidaIgsN2xQHeiFHuPiCHKFElwV0B5FT4xDmPnxNsF33UzqQNMWh9Bly8Nvadj1QVxJKTMoxqSURSt7fpJ56bkEokYxVqVIFIiIiwMfHBxIlSgQXL16EDBkywJ49e8Db21vP2lzaJj4J5dk7T+wi4M1L57I7cn/XuBiM4cibrUYo0yRPAs+ibH5xzl7kYsCFB0/Aj7vhagyhRDkg7Ecti8n2/v5QMKuNUD6JfA0lR8c9psL+d10Ic8hNq3TDxYRGXEdKKJccuc60LQ9cCY/TlZWEEoWY93+jfuQ9onExex50JOeYi1pMJoVJv377TvaF4uwHIfjySetIXwYnR9R1CGwQ+0Bh9Ga7eYfhZsRLh27E2WB+bVsaPtZJKC+Pawj/Z+88oKQotj5+JecgSXLOsuSMIJLRJyiIiKKggIiIioKSgwTBT0QFRERBQTCCSBAk5yUICJJhyRkEQXLwO/+erdk7vR1numd6d6vOeefJTnV11a3qrl/fuuG1GVupYPZ01KdZaV1tFj5IGn1kDpRFcqT3f+xYeVm826wUvf/bHt2qwQDljE7VqVYCBMpMaVLQ9sFNNIOFc22hFbmijtCMB6uh5NpjDpTiY03047///qPCfeJrKfWAUi/GrxZQwsYXDknhBEocL49a6FuTVQpmJWTJgSa2mAZQfvZsJWpWLnfAlKiBUgT/nr31OL35ve/IG2GgEPoNHwoiMPqAX/7SjcuLvOofPFXefx/1nHI7/HBmysE4RreO65fVtWlWL5x7v1lfIv27J4ASDzk8upHRBh7f+DfSJiKVYubMcfZhkRaW0f3DuajUGkoOlK0r51OOiVFwRPr3tduGdl+oh2vglf1/v2sf5+F460qso8WfgxrrGrZbAcrrt+5qGrnzawOAUseQXgHKvWep45RNpstCLxuJGihFQ1WGLY5nKuAmUOa/Py2t7m0OlPwDQZgF6A3e7sYsjl7tAiXA4nLsxwY0D9A+C69+0ZYVoLTilKPe9PXGiPXTcIw5UBbMls6fFtR0ERFR3+alaMQCh4Gyc3WqVdS6hrJa4fsdjzdqd61AVrCr3jawsSZQvtmwBH20RN80QEvWoQIl1x4bASXurTVePaDUi6CB+k3HriJk/hIF425fs6BuaCJRz0xDKeJFWgkbxD9yahbJRjO71FCAUutof3y7SvRoVCBQquNQIr0p3jO/bD1Bb3y/TemyOpUi/oZ84N/EBpJXz+djUblpXLtKukAZ3acB1RjpC62EjEu37oYnsLkadK0881bqhHPvt9KfSNbxBFBWqVJFCSweFeW8Ojpcwg3nolID5a6Tl/2ZEXiubWi0cMQsPGP1ZGEHKHcMbqxr2G4GlLBtu3nnrmaayECgrEfFcmZQ/qTnmYmX8oq9Z6mDBaCEkbsWeEI+L9YpHE8s4QZKOK2s6l3f1H4MGjBhd8WhW2te7UKCOHo1A8o/Bzam6EMXlHBNcL74YfNxv30tNornv9oQL0YlB/px7SrSY1F5lPzBSJ0pipUjb6tAufStetTgw5Wmjz5P0Wda2SWgnNm5BtUsms2yDWX1wvc7nmLT7lqBrBD5YevAxnTqn+vxoj6YmQa4AZQcrMyAEiGldp38h7rXL66Ab89GJQgOV1pFz+ERaxHvJpy2IDA9Ctp5rkYgUGqFG3MSKPs0K0UjY7XmdYplp+mdqusCpXj2+Dj1gHLOthP0+nc+oBzW8kFlXLwM/nUnTV13WFNmcM4T729U4OsLiQ1qFskeEaDESdyHbaSG0sq7Ltg6ngDK7Nmz04YNGwLyaQc7oEhdF0mgxAsNdnMoz1TL7/e+w5cmPBuFZ6yebPDlBm2NnoYSHp0iFMyuoU2ozMD4HoTqtsVLs+GYlXQg1k4N8SXx9azlQMSvBxAUzeEDyss3blOUhmcm2l+575ypgw/agPfoC19tjDd8PaCs/N5igvMQL8EEa7a6FoMBSi4jrft8+PteWyngZsRqytRtqWEDGmrFSevabcqUNoVy/C0ctuBs0OGrTbQrdoMVbXGgFFqS7jO20DyXgBLOQQhXZVbsBv53Q0P5XZcaVKNItoDwWqLfWqFbahS5n6Jj/jYbmqXfzcDGqBEkO4CD1ul/bvjhQNQ3Mw3QaldLQ2nHW5zPDcJwnYy141UfeePed+7eU2IfZkzjW8eZ06XUHaqew6OQ3eiFe/z2ovC4Rg5rHrLn0Mjm1PyTNX7oxI3M5G4nlzcft5DhvXv/UREN5yNhbsIH6zRQ4sMQTni8qBUgPAVmyuT30e27+okQLC1ki5XgfDqmTQWLta1XC+feb71XkanpCaD84IMPaP78+Uraw2LFihHSJvJilC0nMmKLf9dwLiojDSU/moVG6+rNu6bHT7AtAdToASVizl2N9epFfEmA3LoD55U8s+IoXC0RPaBEPQG/enOHfOFFYoFSL9QH2l+17xw9rwGK6nYRmqX9l94FSsD8yl7mGkq+eQCaCmdPb7j87WieBNiYAeX2wY0pU5q4DbjC0N/9YYLgsNVx6qZ46Qx5vupwACVP7WYkIDtxWuFMdPqf644feX/fpQZVL5KNtMBFKw6lV4AS6Vj/GNBICQOmDqLNNelW389aQPnDyzU1g4prtcnvaQaUVvuEetyciF8n3m8fLNrjz94EzWy7agUCgBL1nv58fYBW2Umg5OMWjk16QCkc4vg41Lm8xZH3r3+e9Gc0gzMcQJmXoXN30VdrD8UTpZYWUL1fnb18w5+eMkWy+3RDw9mZJyt14V8w5mkJlFZkFWwdTwBlsmTJ/P2HQ44osKXEv5E5x+vFK0DJs1T0a15aCe9iZs+EY3LY8ekBJeJKijAxCEmROoXPCxjx1pAfV6uIl2ajMSv9nrQIWJ78vvtMHSY4LEEzqhVTDe2v2X+envtyg+nSwHGulmc5NLh4gapLKBpK5BRGBh+1p7NRJ0VWITMA5JofQFOBbOkcA0ps3rDPswuUFYf+ThdjY5TCYavDlI3x0hmGGyitpge1GlYLZh7QZk1addBxoBRy18ozrQWUsJNbHxPfacz0IdCoYAY2Rm1Cdpv7NyIOB6I+dx6z2i9xXMufgZ+61jS1txbt8wgIZkfeVvuEetyciF+nB5T4oOfZeIIBSpHRxooNJQdKERxez/FIhOzi49ADyrl/nqTXZm5Vqo54ohzhhIYXOHt+GZvqkv/dElAyT3sOlDw8nZ05slr3iYp56SMJlFbFFVQ9TwAlPLmNSr169YIaXDgv8gpQ8iwV0GjduH2PxhjEzoOMzICSe/IeGN6MUiT3fQBo2RoKmYsXbuOPVvpt6hDOBTEqcQxuVFa8/bCSng/l6s07VFYjSC/aX3vgfAAoQmuiPqpGG/CkbTc5PnjqASXy/Yrc5aKf/MjbSLOFY8qdJy/r2hdpjRuaRkC0GVByj05AU/77nQNKeIdWLWQOlAKuxDi4rHAc/vyXGwKAEjmBc2RK7bdhDYeG0mp6UJ4Bymg9irXsBlAKuSN2Z5VhSwK6oXXkXatoNlp3MPJAmSNjaiUmrVYYnqEtytLAOTttvX5rF8tG33aqEfAM/PxKLWr12TpL7fB7Oqmh5OZEWkD5f4v2+rMn4fmEyZFVoNSzQxQZbawAJXfUQ8igrzpUVZxatTzZkfa0ZcW8AfLUA0rYN8POWQ8oh83b5c+dzhuEPT6CovOi1lDyjyfsBzCDQoGt+83bd6nrdN99nS4IqTe2bUWnm6Vw7v2Od97hBj0BlHxMCB2UPHnyBOPdLfoezkVldOQNDdmUtT5jaWi0bt+5ZxiMGfUQ8BUOCnoayjQpkylgihIzork/VI0WeKEOtKRDW/iyHgUDlCt7PUwFs/mA8tqtO5o2m9jkcezOQRH2SlovUtgHtvsiNKBE2i6RgQaa1qZjfTar6gKtErQaegbrWtdYBUoecw7QBE2MUTEDVH7tz6/UpMoFzYESWThgUysK1+YCNmFagFArKOLDgHvjC6CE9gNaEFGcdMqxmh6Ue6gbyTEcQHnh35vxws1oaSgBXlphrYLZF0LRUOKjaqMOUPJ4oVb7pQWUs7rVUvLGWylwHOn/iy9GqZNAuff0Fc2YpkJ2HCjxvn2maoGAMGeoh1zgyAkuCncu+2b94XjwbQcouTa4YemcNPmFqspttJ79MW3K05OV8gWIk4MjfhBH3ty2EukK8f7jZcSC3TQpNnc6/7sVoOQfTwihK5KhwTQJ2d3MTKKsrAetOi0q5KGPJVAGKz5L13kGKGFH+fHHH9OpUz7Pz8KFC9M777xDnTt3tjSQSFcKJ1CKh1l8kXI7H27IDgCBPY0eKAqZmQElDz7LX4bcfo7Ln6cD5ECJUDd4gZiFdOHHuQgzVHpg/KwPClAePB8Aivib1otUDyj1wgZpgTIHSsRD1AocDBkEA5TIUrTMgoaSe88CmnJndg4osXlXKpA13mOklqcRUMJhC6YFIke3+DDg3vgTnq1EzcvlVo7T3AJKK+lBMVDubBYpoMSxbpVC9ysacaw7XrSAEkfDaw6cd+R1FwpQCqccraP6Ua3K0Ts/77DVR2heZ3QO1FD+8mptajl+raV2AD1IO+o0UO47cyVezm7cQ8iOO7/B47pttQIBYdXMgHLa+sM0QKXNndS+MjUu+wBxDSXsH3vEHkFzgfDsWQ1L56LJL/gcYrTeg8h+06qyNaCE2Q4iOaC8/2Q5ZVy8jFywmz53ASih+db7WLe0EAwqIf4tHJOcLuHc+53uu9PteQIoR40aRUOHDqUePXpQzZo16d69e7R27VqaMGECffTRR9SlSxenx+14e+FcVDgigBbowbyZFHtGDpQv1y3if9ABIDj+MANKHNNA26VXjwef5UBZbvAif8BzLlCeDrDJR6v82XoQcBomsmZAyY9zb9y+S6UGaAPl+oMX6Jkvov23tgqUcIK5efseLXmrXoC2TTSkDZRx3vM49tcKHKwAZasoxcvZjoYSAbaXvWV+5M2BEiF6HsicxnAd29FQzu5WiypaAMqdQ5pQeh0NJRy22n0RTVuO+jSU4sOAe+OHAyiN0jJygXHNuxWg/HzlQX+IFq366g0b8WC18iHza4Vm+OLVWwHOHKijdeQN55XV+yMPlAKqrB7Vm71wtYASWaAQ4sdK4RDrpIZy/5krmjbfWkAJm/U2VfPHB8qJ62njYW0N5bToI4Qg4bxoAaVecoYhj5elQb/6zAsQy3VSrIe11rMvNJ/8XnoaysW7zvjTpWoBJQL8T1x5UGmKe2ojp3qvJqUCxiP6IkJNcW089oP/Yp28oaHMmTGNpSxX6jUB0xGkfTUq6viYVtaVlTrh3Put9CeSdTwBlPnz56cRI0ZQ+/btA2SB2JQjR46k/fv3R1JGlu4dyUXFgbJrvaL+Bx1hLODUpM71qh4QtG95s6TRB0oWfJYDJZxlRDgh3iYHSh78F0CJ9H5qWzF1f9T2gSN/200HzvxLS/ec9VdFPzbEXKCnJ5kDJWL9cfCEpzLsQlPG2oKq728GlAdHNNfNCYyNbfepK4ZACfBA7DiR87pojvS01AJQYj4F9EMLlyuTc0CJzbt8/izx1rp6Y4IWMl2quCNvbke7b1gzajtpvR8oxTy6DZTq4Mib+zc0XWMYqNWgymLNGwFlk7K5CMf5wr4Y7VsDylpUuWBWzRSjWhpKrwGl1lE9Yg2KLCuWXp5EpAWUOOn437g18ZrQStfH4ZuHg9IKG2S1T6iHkGdaNt9iTYz5fS99suyA0iQcZACUPMwZ6j01cR1tOnwx4N0l/qEFlEid2ESlodSzA+e2o80efIA+e66y0jR/buEohNBen7StGC+z1rztJ6n7DJ/zDYo48uZpbfFOe7pqoIYS2Xk+i02viYxdyBG++chFQjYe/sGJNqHt/HJNjGIGVTp3pgBtPAdKJFfImSm1pkbYbM7wTtcKL8evQ1B3PKNOl0ju/U6PJdT2PAGUGTJkoC1btihpF3kBSJYvX56uXbsW6jhdvz6Si4oDJb4Qxy/3fTki0C6Mns2AEg4n+KrX01DyL1AOlGUGLvR7f+O4ZcnuM8p9kWKxbB5fhiMOlCIYt5lzw5p36lO+rPEdTtS2o7BLgn2SKHoaSmjfnmC2WNCk8XzV6sVhBpSwI9WK84Z28DUPQ/6vdbJIoA6OXYbO2+X3kLcKlJhP4WAFLRy+5o2KlpZCL+4bN1Pgbarb2D20KaVNFZfrGx8HAowB2piPP474Nk8BlDy8k9BQ4vgOoUlECcaGEllMsFkVz5UhAGAQH1F9fKwlJ+4QYCRHK0DJnwvRlhWgFKYGWilGEwJQah3V6x3PGslYZHnh6w0fploxa7WAEvaBIvc0B0qR1jTYDeDguX81g+T7gXLxPn8mMjVQYn3hmTACyunRR/y2n6KPWkAJc6HGGulE32tR1n9k3rzcAzTh2fhAqbU2xb30gJKntdXSUPL4mzgpm/faQ5ZFrKWNx8WuA2W53DT+WQmUlicqiIqeAMoXXniBMmbMSOPGjQsYwpAhQ5RUjD/++GMQQwvvJZEESh7D7rVHivkDWiMVWMoU99HohXsNhREsUJYa8JvfWQcG4Ut2+zSIyIhTJk8mDaD0ZVDQ8gzlHdRzOFED5abDfwccc6iBEjH+YJODkEd8Y8JLHi97vcJD4Yg6PGC8nvMP6sKWa48JUCJwMey9zsaGXIJMkF3C7Iiap7SDhy3GZhco9Y55uVbZDlDy0FGQC4BSaGPEh4FbQCn6qQ5wr843LupBiwXtSbdv7XmRugmUwtRAKyOU1pE3Yg1Cnk6UUGwocX9crwWU+Ggwk7EaChFf87suNQOeAXyYPvqJNQ0lQuKIzC5OAmXMuX/pEY2sS1pACQeZp6rk82cSgwZ83/Bm1PqzdYr2ThQOeN9uOEL9ZgceeWsBJUJy8YDpoi3ujMQ1cOp3pd560QNK/kxphQ3izkhR+TLTr93rWF6Sl67dCvCEFxcivBucvRppgLNZ41YyuHHgNmvPzu+R3Pvt9DMcdT0BlD179lTsJUuXLk0IEZQyZUratGkTrV69mlq0aEFZssQdxeEY3IslkouKA+XrDYr7Uy2+0bA4wf7RDChxJIKXsJ6GkscK4y/DEv1/o1t3fN7fHCiREQdHGyhcQymyu2gZ8rsBlKKv6qDRRl/s6IcZUOppQq0AJYAAcRlrvb+Uzlz2xfAsnjMDLbYAlHxuEUQ8Wwb7QMlDQHGZc62yEVCqtbscKBUHBGYvJoBy9f5z/sDyQkOJ4zIcm4kSjIZSXMvbx9/08s3Dhgyx+vQcqkR7cFL7btOxeABgdOQdrIYSjicV8mfRzAilBZT1SuRQEgs4UUSf4VUr0gfaaRfXa2mbAEQvT/M5dGiVzg8VVoJZi2gUqKMFlHiPaHn8amkoofUXcROdBMrD56/SwxpZl4TsPlq8z/++hZMfvJzLxWb2Qj/3DrMPlGJNcKccmJPgfasu3KOeO51YBUoebxJtiyNv/kxpBTbnzkjl82WmOQ4BZa5MqU1t7LXWlBWgbFr2AZrY3qfBdbJEcu93chxOtOUJoKxfv77lsSxfvtxy3XBWjOSi4sDEj0WRnxYAwTduLZmYASUP7cA3zuL9FvjTZnGgRFidUg/oA6WZhnLdu49QHo2QOHY1lE4CJYcMI6DE1/ye05fpG50jb9GnGiOW0unLN5TpKJErA/3+prmGEvP5yVKfPTGOdeFpa1S0NJ486xG/ln8E8L+r21ADZbXhS/yaVjVQCk2zFlAipzu0tMi2hBIKUKrjkerZUwEoERmhqEZaOj5mdbgaNzWUwnZVKyMUgPLr9YfprxO+XNEoAOIVe50FyhOXrtMnS/bT95vjINrKuxNy0dI2QQP/0tebdZtA9InzV24FZFpBjvLvXw7UUOqF59ICSq4VdRIoj1y4SvU+iJ/GUwso4XHdukp+fyIGAZSIpSnMQBCjEdApyowNR6nvbJ93+uPl8ygnOCKvOAdKPUdAvG/E9TwsTrBAiSgh/R8rExDj972WDxJykvMC0xvxLsIHESDYatEy78C1sBOFDaWZ06bWfRB9QisBBq8LO+fP2wemhbTaZ6N6kdz7nei/k214AiidHFCk2orkouJAyR03ejxSjNKlTkHwyDMqz9UoQA9kitNQfvNiNYKxODz9UIRmDNl0Vvd+xN8UNmYRlLZBqZx+pxkEMC/5QEalHtd+iJSKZkCpFxLHrg2leOmr06eZaSi1wiHxgPFGQImvecSuMwPKmiOX+rMEEmpkAAAgAElEQVTpCKAs3Ge+3+NRa74wn8IBAEdgcHAyKlpAmTF1Cs10mfwjgLeJzQqbnihqoKw+Yolf06oAJYu5Jz4MeEYjoaFEezxPeyhAqQ4fpQeU8JRtUDqnZqxSPmY9oIRXq96zFKyG8tfutSkqXxbFuU29IQIoG8A2edcZ6v3zdqWLaqDEB+M7TUvS4Lm7bL/61H02M7lQ3wDXa8EBAlR3nLJJtz8ASpxScA2lFlDq2Q1qAeXE5yr5A2I7CZTH/r5GD42Or8QQshu7ZB+NXeL7yIODDOI8inmEecme95opwdkFUKpTl/LTGvWzxYESZjqAW/XxOw+XxDPBBAuUQkPJI2hgXM/XLBQwn1wzW6lAFprVzQZQXr8d4AkvGvYBZRrTxBdaCwvRJ7QSYPC63Ave9sNicEEk934nx+FEWxIonZAiUUSj5XOg5MGvu9cvRhnTpDAMdYLh4+sTRw3iyBtZWwBFIg4ZwjkUypZesdnjziwcgKD5WRbrhY2NoEQuH1A2/3i1EkYHRaRUNANKvZA46pck9/JW2yHCe/DQyEeV+6rTpwUDlNAI/u/TNVS1UFYl24Le5gubJsSuMwPKWiOX0sl/fBrKkrky0qI36yrBjyFzdZYesUS5fSzsBLOkCwIo06TQDPXE54w/Euq8wDz1JupxTSvkyvMWiw8DPaDkWrlQgDI65gK1Zd7+esdfIqWdGTipM7SI9eIGUApnKK2MUADKp6rkV6ZD9Ll+yRy0nGko8XyveecRzQ3a7NXmCFBqwAHeFwhwzwv35rUKlHDi09JWaQElQu10iT1mjxRQwkEGQCnABrC/+72mAUCpzjQFGeF9mBrJPNKlDJAZB0rYJyNiR4MPV9DBc1f99eAw825s/M0nK+WlMW18uaqtAiXP2Y3rBFDydytCE71QKxAoP16y35/St0rBrPTTK7XMlpv/d/4hyS9CvGBErmigYbNq1rgVoMQ7BqcUThcJlHESlUDp0OqK5KLiQIngughJg9Lt4aKE/Kji33pDRWYbGEMLoESGk71nrvjtoLBBPFQ8R7zL+UuLb3QID1Q8BKDUC4mjfknyr2jYUMGG6I8jf9NnK2II9qMP5vV5mqvTpwUDlLhG5JZXv7C5YHA8hNh1ZkBZ+/1lhKNGlFIPZKSFb9RV/hv3QMaPb5lWULTPPfhhJ4i5NSpa4KSXL1d44Gu1V27QIr9WE7ZcsMsVhWtaIaNnJkX7c02LDwN+JM01lE4BpdrbX+/4C0ex0PgFC5Ru2FAKZyitjFBaQMk/3DAHAEqESSrZP36sVqO1gQ+jH7sGQoCRXPDRiVMLXjDfWnCglUiAAyVSFAKiuIYSeeSR15z3AY5qWiF7tICSH7PjtEWYk4Tq5X384jWqM0pfQ8nBCh+T0BIKoEyXKjntGtqUnpyw1h9KS50YwGiOOFCKd9YjH66gGAaUiHsrtNc8j3aoQMkdHrUSQOC4W0ScQLgxmG5YLVrmHbgW4d2g2NBygjJrG+HMygxcZFiNB343a8/O75Hc++30Mxx1JVA6JOVILioOlAiuO3zBbmVUrzxclBBMdsQC4yNvNVAiwwk0lOKLH0fgcCZRF25DyYGSwwnXUIpUfGYayo19GyhHH+qifknyo85x7SrSY1F5NGfTLlCWH/K7ErdNvXnyf+ttvtBS7D/7b9BAiXsALhBOBBDPnRK4kwLPqa63hLX6mCVdSrp0LXBsuB6bN7S8mkDJAtgjSxCP36kGSgQ2F7mmzYCSH/OGoqHcfPhvas2CGutpK7wMlFoZobSAkpuWKECZOgXtGNJE8fw+fvG6355Ob00gPSAK7PiQ5s7KmkYdPaDU8k5Xx33F9TxEk1WghImMFlxoASXeUc9/5dOKcqBUm+nYfd2fvHSdar2/LN5lAvA4UMLcBUApwEYLKNWJAYz6YwkoW0dR75985hBPVc5HH8Tm0Q4VKHFEL/KoI184NJe8IGA/wmKJYvaRzq91GihhF76pf0MLQBmXmtLuOjCqH8m938lxONFWxIDy119/pcaNG1OaNMax9JwYZDjaiOSi4kCJWGjD5vuA8uV6RSh7+tR+wNSTwws1CyrH2UJDqQbKr1+sRvAuVRdonjpO3URdHiqiZOsRR3EcTh79ZDXtPOk78hap+M5evkHVRizVnRa9GIvxgJLl8hY5orUatevlHQpQwt4IwZDNNJR1Ri1TAACFayjV/f9yzSF6/7fd9NHTFRRgBugDCs2CmqMdLaCEI4/Wkbqwb9WSH7cptQOUQtPMc64j8HGzcrmV2zgFlHzzQ7t62oqvOsApJ3gNpRtH3sK7XisjlBZQcuc3jFUApZg39ZxDmwwbaBGLVmjyteY5GA2lVtig77rUCDBBwL14pAgA5ZnLNwKC/1crdD/90DVQQ4kPUC0Pay2g5FpRJ4Hy1D/XqeZIfaDkmjo4yAAoRapYAZRPTFjrT0eqTgxgtDdZAUoeCQCOg++3ilKatAqUc7ad8IdbwnXiyHvr0Yv+2L3YUzo9VCSgq91nbKF5231pklHsAKWWvTDawIcIsn/V1/CqN5ITzH+QMU4rRS+/TqQqNmormN8iufcH0183r4kYUCKY+Z49eyhfvnxUpEgRJUxQtmzZ3Byrq21HclHtOP6PP6MEjieGxBrod6lbRDnKFoCpJwA1UCJlHo68O3/j89TUA0r8Bm9dPMwdpmz0e5+K8ED4/bFPV/u9VK0CpV6MRfVLUu8oVT1OdXBis5df1OBFdPnGnYBmrNqbwd4I9xNAyeeDv3gfGr2Mjv3tA0qEWIL2Ua8IGdtdwFqAkC19Krpw9Va8poT2WOsePBuOWjPKbUEho2cnR9PaAxeUZoSmmWuSOVByu8FQNJR888N9EXxda3OxCpQIyM4dMdy0oRQxW60CJbeRxljNgBJ2fHCSETamTgOlFhzg6JonHEA/eUB9aElP/xMIlOIInq9ZvC+0PKy1gBI50YWW2kmgRD9rjIz/8SvWxKdL99OHi/cp6x32jC0qxAElNGc7hzYlDpTqxABGz/Q36w/TwNg83/4j7/9bQTHn42wo4TUu0nsiVu7IJ0MDSuwZfZuXpj+PXfKnvcSpV+e6gUD56rdbaP6O4IBSy15YACXsX7U+IozkBNloafjV1+AUbUrHanZfo6b1I7n3m3YuzBUiBpQ5c+akl19+mRo2bEgIGzR79mzKmjWr5vDr1vXZl3m5RHJRcaDkqbgQ7w2aLDOghAdz9gyp/BpKbKjQhHWKBcqpHavSwyVzGoqfAyXXdnGgFLmdzY68YROmPo7DzdVAyZ09OKhodVRAohm84dpQgRIaSmFvps6qIzaGuqOX09G/fRmgoB0GRDtdtIASchWZbfj9BOxr9YE73qiDwnNbUIztuckbaM0BX65poWnWA0puNxgKUPLND/eFt6xW/nfE/6xfKqepDaXai1/MmRs2lCJcEz4a1HaQXEO5fO9Z2nrkIr36SLGAehlSpyDY5YminnNAzZcd4oASWWwQnkarBKOhRDtqLZdWXmWe5tIqUKrBXvRZCygRtqZlbN5vHHOLj7VQj7z1TlO0gBIpCgGUYu2JuUG/th3z5bc3y9LF5+XO3Xs0bvkBJesYng8UaO8OMaDkGYKQoAJaUq13pd67RT13MJOCBlltl/9yvaIBTXSd9gct3Hna/zezj3R+sZa9MH6HZhsfA8EApdYHmXrMiJAwVQKl09tMQHsRA8pRo0ZR3759/Y4OcEbQKvBsu3v3rqtCcKJxrwAlnEIG/OLLvICYYjhCCBUooeGobwMohTc3+sCBUqTiMwNKvaDdaqDk8Q0nPldZySerV27fvad4oVcqkNU0w0yjMSsVO0herGooEYvu3L83/ekvAZS/7zqjeG/zI6l6HyynIxfiUoraeSFbXa9agAB4RagSdVHnT+e/8+N5jCcZyzKkBsr2X26g1ft9QCk0zXpH3k4BJf+gEpt2sECplbXITQ2lCNeEBAHqwNUcKPl88Hk1A0r8DttRkfMeGWUAPVrlze+30eytvrig6qJnQynqjViwmyatilH+ybWF4nczoBSewnxsdoCSZ9UpkiO933GlwP3paFVv63GO1ePWe1eJNTFu2X7/hzgcZB6vkMcPlEJ73GL8WkXjh6KOkmD1WRb1Hv5gOR1m7w2eNx3h34a1DA0oERnk7SYlFRMmkaUIHwBdVUCJ0ysRVg59s/P+0tMmIrMZ9istrbSRnHBvK0AJsy2ctjldIrn3Oz2WUNuLGFCi4//88w9dunSJChcuTBs3bqQcOeLb6aFewYKBQVX1Bj1s2DBasmQJrVgRF4i2Y8eONHXq1IBL8ubNq4T5Qbl37x4hxePkyZPp4sWLVKdOHSVrT7FixWzJNpKLim+oPHPCi7ULU96saem9ecYx6jrWLkQ4ChU2lHiRI/SNCE5sBShf+GqjP4MHPz5FqJ0dJ/5RZOkHShMbSqsaSp7ST6QrszVpOpXhpQ0A5OE5rAKlyJbxzBfRVD5fFhoeqzGAVhAyxgcSSqSAEl6UIkMPH75e/nTU4VoRJ4GSbyyhaCjVNrK6GsrYDyMjTVy4gVLEbMUHjzqDjxWgNDvyxu/QUIojaCOgxEc9bCKfnbyB9py+EvB0AChhf/txbGB9NUQMn7+Lvlh9SLlGHXYJfwsGKBEYHx8s6qKloeQxK2GTLPofKlDqZfXSAkrMFyJNiI8ZLaBUR0mw+75SA+XYpyvQG99vU5qBc+XQFg8q/x2sDSXsbd9sVCIgMgbMLLo9HLgfvjR1kz/usHotmI3JCChzZ05LdT+wl7zEKlDCsRTOW06XSO79To8l1PYiCpSi8ytXrqTatWtTihQpgh7P2LFjCSkccTzOgbJy5crUtGlTeu211/xtJ0+e3A+vgEkA5JQpUwig2bt3b4qJiaGdO3dSqlTGcf54ZyO5qLYfv0SPj1urdIcHugUo5s+ajoYGAZT7z16hF6f6bCjFUaHR5HCgFEfbqB8MUFp1yuH5Zp0ESjFOo5eyHpQgn++LKo9ILblFCii5fRnvl17+dNThGlsRD09cy7WXeLHD01bkmhYfBnoaSjtAWbtYNvq2Uw3NJaiOMwotkFYYHfFhZASUWmkw3dRQivifON4s1i8wtZ4TQJkpTQqa/II1oBTCbfLRKsWGmhfE7wP06z0Tw+btoslrfECpDgyPv8E5SKRp1Try1tJQIo6plkOMFlDiVEQ4cyAV4J/HfR+xoQLlhX9vUuVhS+KtO7Emxi8/4Hd4gj3jY1G544AyTQraMbgJtRi3xt8ftVOb3Q1P/d7gOcxhujT48bJKk8ECJRJjdH+kuKJQaBybU7tXk5L0av1AoOTPOe5nR0Opp02EhjJYoNQyGVHL9qHi2WnaS9Xtity0fiT3ftPOhbmCJ4ASY543bx4NHz6ctm/fruTyLlOmDPXq1YueeOIJQ5GcOHGCOnXqpOT9zp8/P+XKlcsPlDgqh/PPzJkzqWXLlvHauXXrFmXPnp1Gjx5NXbt2VX6HxjRPnjyEnOFt27a1PB2RXFTchmx06yga/OtOunbrLiEeJI4fzYASmsz706f0ayihrdp/5l/FgxtFODMYCYO/YPjx6ePj1tD22Je70IKZeXlbjUO5Yu9Z6hCbkUNseJYnzEJFo5dyzLl/lewXny474LeFRJN6WX7Ut1NrGuy8kC10Pd6mIq7JkzmNP6A6b8eo3zz0k7qf3LkIv/EPC2G6oAeUfGMx01D+ObBxvMDPov9Iddl07Gr/cPTyHgtbYC8BpYjZioxT6pSQVoDS7MgbEQEmta9iSUMpBKgGyqI50ithpaBdtwKUs7vV8nsIizbVQInwR41igQV1xGbP20fYKS2HGC2g5MfjcPDZdPiicutQgVLLi50DFAfKDwGU5XP7P2YA89tVQGkl3JfR881tr1EPNrE9Zm5VLoECYdD/7AHlL1tP+DWcaENoIw+cveIPKi8gk/eLhwfj8rDybtIDSjhzwSlHKzORUbt471gByjrFstP0ThIorcxRsHU8AZRz5syhJ598UoFHaBgBgqtWraK5c+fSrFmz6PHHH9cdH+oAGHHcPXToUDp8+LAfKOFFXrp0adq7dy+VKFEiXhs4Zq9evXq833HsHRUVpWgurRavACU2IdgSIj5cvqzpaMraQ36vb72xqIES2qp9p684ApT861xowfRyuYr+WU29uHzPWX8fRRYUq/NlpZ6Vr3weaFgEqbbSdqQ0lHmzpPUHVOf91IN41OFzqAbKTl9vpiW7fSk68Rt3zhL5xp0ASiPg5toU9ANaIPXxMf7uRaAUMVvVGYnQXyeAEnFoJz1fhZ6KjdNpdOQt1oMaKJFbumcj3/tT75mAWQ1CXKEgyDXsBnnhXt5IvgAnDwQNR9pCnDT8iKDm2dMHtI8oAVrhxbSAEuu3emwoMmizRaSBUIFSK1c5BygOlLBnbF4uDiiRRAAJCPjzo3Zqs/Ku4HX4Bxz+zoES7/GB/ytjOE/q+6mBsm/zUtSlblElUoXIWIO5F/nFxfVPTVznh3YuDyvj0YO/UIBSywZZ3RejUw4r/darE8m9P5R+u3GtJ4CyUqVKCkwOGDAgYIwARGguAX5WSocOHQKA8scff6Snn35a0T7+9ttvlCxZMmrevLkCn5kzZ1ZgtVWrVnTt2jVKmzat/xZt2rRR/oZ765XLly8T/ifKqVOnqFq1anTs2DElFFI4C9dQ4tgFQYtFmbr2kGmeX8Qew8YjbCgVoDxzxZ+PVwSENhoT11ByezwtoFRehEv3E1LmiSDYvG2941f1ZrZszxn/sbyVPtqdE7tAKQDKyn0iBZTIHCLiX/J+6pkZoA7P9KEGOzgt9J/9Fz0alVtx9ug4ZaM/HqnIN+42UHJtCvoLLZD6+Bh/F+GvvGRDKWK2wn6xcJ8FAUvHCaCE3S7MQURIHdjctayo7ZQjbt507KoAG0orQDl07i76aq0PKJGfXJjgiDZ5HEoBlOI3vQxUWJPVhscP2aMFlNCGi6Npnu+8YLZ0tLJX8E45eh+/4jmYsOIAjV64VxkKZAugFM5VAij5KY3aBtnKu4LXUQMlP/KGE2b/x0IDShFz8vD5q35va2Qde6NhoEKGOxqhf3ZOWPTgD9EBoKHUykxkJCfcW8sGWX1NraLZaEZnbbMZu/PA60ugjJOGJ4ASMLdjx454jjD79++n8uXLK3BnpaiBctCgQcox+ogRIxSQRHtvv/22cjS+bNkymjFjBrVv317RiAI2RXn++efp5MmTioOPXhk8eLDizKMukQBKbkOJY5dWDCi/XneYBv2601B8eBHhaEwA5bpYoBTHyVa0f9y7lwMhB0q0mydLHLjzoMHckF4vtIYa8JbuPuN3HLJyLG9lDfE6doFSAJSV+0QKKHlIFd5PvdifqAOHDqQ3tLJxvDh1kz+nuxZQTnyuEjV90BfYnGsqcOQNh4Ahc3cGaD5EH402LHWc0YQElDxmqxp0rQCliHUo5KRuA+HAEAEhJKB8pBj1bFxSuYXeM8GBEvnJ/zduTcBjYASUes8cbHCraNgvagElTCLKD/1daapxmVxKZAWUkIFSI1c5fw44UALuAJRCO64FlGobZCvvCl6H2yzj73ieuk7folQReeGN5kl9v9lbj9Ob3//p/zOiVHSoXZiOXrjmd44Rjjr8WmTwQgYyK8+n+p56QInoAPDydgsoaxbJRjO7SKC0u+bs1PcEUBYtWpQ+/fRTBfp4mT9/PnXu3FmBOytFDZT48oUneZYsWfyXR0dHU82aNQn/jy+L1q1ba2oob968STiK1yte0lDC/qrZx6voyo07tOythyltquT+blsBSsSrxMtPACWOnBGHMlig5ODIv2TVR9k8aPCj5XLTaw2KEWzCcFSvVdSbGTScImAzjkuQD9jJYgkoWaBhIzs/db8iBZTYYHm4ItEvvVBN+J0HKzfTRHDvT2SwyJIuFXENpRFQwg52ya4z/vinXGZG90VcPp5dA8eKantEtCVSiHpJQ8ljtroBlAgThRitAihxLPtEReMTlHgaSgtAiQ8BkZsbph/475+3xKXm46kX1RpKPs9cBtD4V3pvcbxHWgsodwxuTOUG+4ASjjEii0uoQKmXJlCsx89WHKRRC32pbXH83OzBB/xAiY/0bQMbBzgmmj0/Zu8vNVBOal+ZyuXLrHjnI16lKFbeXairBkqEnYNH/7G/r/ltGXuw+Rfth+LlradNBFDmzpJW07PfSC6QqZZTm/qaGkXup++61DQTse3fpYYyTmSeAEocdU+bNo0+++wzxdsbZc2aNdStWzdq0aIFffzxx5YmWQ2UWhddvXpVcdT56aefFE0lbCgPHDhAgFpRYEMJzej48eMt3ReVIr2o8EDd+8/nTckLz7agNxgtoNx35l/FwQLFisML11BycORBfQ2BMio3IX2iUVG/JPHB8Pp32whhEZGaUITksTxpJhWtvJQfYUC5fXBjypQmpaXbRwooC2VLFxDHTnTWSLvKHW3MNsROX2+iJbvPKs0KwNYDSq6paFg6F01+oQpxMwYuSKP78uM5XKMOJi/a0QJKHrMQ9cIdNsgIKNXmK2IcfF2qNZQwJVl/8AKtj/FlK0KYqAnPVqJWn61X/h0MUL72SDF6y0RDCUfAqesOK/dATMgyuTMpwcVFCBg8o3g/oQg7Pa0HhY8Na7KiRaDk6TYRaUE4IoYKlHppArWAElmIAJTC3AJmRFsdBkoe9xXyA1A2Lhs//q6VdxeuVwMlooQ8U60A8RzmIjYlny91jnOz9wK/Vg/+AJQ4wdLKnW70UrUKlNUL30/fvyyB0tIGFWQlTwDljRs3FI9q5PcWUABYgMbyhx9+oHTptDVW6jGrgbJdu3Z04cIFWrRokb8qQgohM8+uXbuUlI/I2DNmzBh66aWXlDoJ0cvbaO6nrT9MA2LTd+nVQ7oteCQKDSW8K2FDCbtIo5cWb48DJa7H0QUKB0r+d/yGfL7CkB42eHaBMsg1b/kyKy9lHqMRGUugYbVS1N6adl7IVtpHHSNNnLoNoU3UaptDolk/A4ByUGNF881TZHINJddUCKBERpiOsZ77vC9G9+XHc7gGx4rCHpF7j097qRo9VDxHgFxgnjFlzSF/yJtwAyVPAuCEhlLITLSFMFHjn63oB0pkVnmykj0NJQcKvWeCA6VIJ8ljOCL8qshdYRUo+TE2XwtaGkqEilqx95yS4QVHtALq8AG1IgQbSr00gVqhpPD+alI2l//eiNsJLSsPnWb2/Jg922qg1PvYt/Luwr1mbTlOPX+IO/JGlJA2VfIrqTGFh/2r9YtSryal4nXN6j3UF2pFNECdn1+BDWVwQKnXJr+3yBdvJmO7v0damWS3v27W9wRQigHCKxu2lIDJcuXKKR7adooaKOFUAw/x9957TwHWffv20auvvqoceX/77bdK0/369aPPP/9cCRNUqFAhJVQRPMXRj4QSh9JIRkgBKDLn6NV7uW4RyqgCSsShbP+lDyitxHjkQMk9hjlQqj2JefighAqUPPwPtCTpUiVMoIQ3KuBPq/A0a2YbIs+gITS2VoAS4IfNkYeC4nZ3Rvflx3PoP+rib1jD0IIKGzMtoBTtis3RCCj58aZaTlr9Q6SF8kN8x7B6hScBcAMo4eQAuT72qc+mEdpK2PkZFW7igHp2gVKkk9SL4WgZKAc11pSfFlCqw/EIWYaqodQLwq0FlJBtk7IP+M0tBFDybGFmz4/Zfldr5NKAsF9OA6WwwefvZpGOUd23YIFSK6IB2kZA/DxZ0mjGHjWSC2RqBShFvngzGdv9XQJlnMQ8BZR2J1JdX+vI++eff1YccwCrsKWE1hJe3mnS+DRocMhBCkgENr9+/boStghH3YBLO8Wri8oSUNYrQsjqwDWUyEf93JcbLANlQA7nvg0oZyaffJ+YsJa2HvWlHYsHlFdu+L04EwNQ7h7aNMB+1Wj9eE1DCRu0jDrH9d1nbPHbpJltiF2+2ex3iDADSn70JYCSB6sf8nhZmh59RAmo3btpfA2JkO+JS9cD7K54HxftPE0vT/tDqRpuoMQ9zbTEPIe6um4wR95CJqIthIlC1AWA/uUbd2hGp+qUInmgWYx6nULji0xPkCuK0Fqpx8PlzDWUIvuPXsidfs1LU+e6RTQfDy4DbhfJK2sBpdrZxSmg1IuZKMbO87vDVhXHz8J+Fx72fwxoRI9+spp2nvQ5sJg9P2Z7jhoo9RwmrcKeWkMpwkohs5dwiHq5XhHq0yy+csfqPdRj4kCJfefKzTtKFZh/4INcK/aokVwgUz1IxXXC3EIEzzeTsd3fvbr32x2HE/UTFVA6IZBg2/DqosKG3D82t7fe2BCfD0dFekBplicb7QYAZb8GlDOjDyh5yBnElROgid94nlwY0o8zsaFs/dk62nzkIsEjfOEbdYOdKsvXWXlhcg2lnne61g0jCZS/vFqbvll3mM5fveXPamN0XD9n2wnFVtXKhvjytM20aKfPw1ZAqp6GUgso7eRnF3Ll0QLUffQ6UPKsUlY1lH1m7aCZG48qw1fbUAqZiLYQJmrNO49YXvO8IsxlkH50wGNlCE41KHrPxKA5f9HX648odUT2Hz0NrVWg3DmkCZUdFGeuJPqmBZRqUBP9DPXI2w5Q4j0JD/MifX3hn+Bhv7l/IzJKDGB3YpwGyp//OE5v/Rh35D2uXUV6LCqP4uQjHKJwgtWnuXNAyUNkIcB9qdwZKX3qFEp6R24GZVU2mHutsFviehEDtXLBrIoW1Oni1b3f6XFaaU8CpRUpWajj1UX17YYj1G/2X4YjwNc9AvRyoEQoFuT0ReF2b3oNcaDkIWgCgJKBJtrhNlZWgPLi1Vu04K9TyrFS9gypLcxKaFWQgeLXP08SUsV1rRfntMVb5c41dvL0Rgoo21TJR6Nbl1eGwCHA6LgeL+tftp2gojkyUFS+uIgJWtLlQCkgVQ8o+TGVsKHkdaHxaWZyPIs+qDchr2so89+fVnFYQeFZpf8heBkAACAASURBVKwC5bVbd6jMQB9opUuVnHYNbRpvKkRbuNfq3sEBpdb8ikQJAAHkCBeFryWR/UfPocUqUHJHG96XcAKlXogbscYmrTpIIxb4vLxhGtSoNAfK1ITQR04CZc2RS+nUPzf84tCLv2vlYxiNqIFSvOu5dhlOm/0e9cW35MXqPbTWkbhWvY7cAEqRpalSgSw0q5vP6dfJ4tW938kxWm1LAqVVSZnU8+qimrHhKPWdvUO39/h62z+8OY1btj8AKJFasJ0NoOR2VyKHM27a6rN1SopCFHXwbH6sYgUoHZoqy81AgwYNTYlcGXQ9yDlQ2kmrpg5QHOpRmNFLm//G4Ri2tTCJQLGjXTUSILe3hIYJmgc9oOTHVAIooSkX9n4zOlenWkWzm84X13SjsleBEk5p/9F/9PrMbbTxsC+upxFQ6h154zqxIZsBZag2hGrh4+Ni/9l/qXD29JSSHZ0PnPMXfROroRTB2jn48nasAiXWZKkBC+PNfziBUi/EjVhjX6yKoeELdit9hMc1zDOEQxg+evEu5DEbQ33Oa4xYSqcvuweUwmv88o3bFBUbhgmJL6ClVhcngPKRUjnpK/ZhYpaWV90HmHTAsY4/E+o6aVMmp+u371KF/FkIpzNOF6/u/U6P00p7EiitSMlCHa8uKjOgxElWzMhHDYHSiqaIAyWPaciBUh08mxvtexEoLUw7cU2jnbRqkQJKHgPQDaDs9u0ftGDHaUV0Quu5Zv95vz0uN5/gQClsKHEd0vhhAxje8kFLoaD4h4mXgVLARJuJ6/1AybNKWdVQ2gHKUI98rTwDqMOBUgRr1zsuTkhAqRfiRgsoYc/YoHROP1AiBijeeU4CZfURS+jM5Zv+adFL6GAV9n764zi9zY68RXtcu8xTOvL1YPUeWmtIXBsPKJldvdnaq18yh5IZCCcnRkAJs5Crt+5S+fxZlLSgThev7v1Oj9NKe54AykOHDtE777xDf/31FyGEkLrExMRYGUtE63h1UcHWCjZXRgUvx3gayvP/UrsvfEfeVjxD230R7U+jyFMQCrtHtMM1l/g3B8r/lc9DiOOW0AoHSjtZMCIFlHwz7//LDpoe7bPFs3NcbzRHr367hebvOGUJKLndk9BQBjP/am9itzSUE1cepPd/8x1v8mKUw11r0+WZh3hWKTeAEppEhCZyu/CPExFbU++42CpQYk2KNIa8/3Y0lKGOX8/ZQ6yxyatjaNh8n4ZSHD+LeUyIQCnSk3LtcodahWjw42XjLSEngFJ95K0+bdBbt0L7y3/Xc4BDBBMk/SifLzPN6V7H8UfBq3u/4wO10KAngBKBxJELGzm0hfc17ztSKHq9eHVRBQuUyD4CT89ggJIHyeZHoGqg5IbfiQEo7RxnRQooRa5ezGu/2Tvo2w0+oLRzXG8VKIXXe+CRd2Vq+mBcIGaxCYQClOHSUMLJBIHsc2dJQ3+d8Hntwkt1x5AmuiLR2nSf/nw9bYhNZcmzSln18sbNRF0c5+1+T9+GEoHbkT3L7cI/TkQoJL1QLnwNqvvFZaCXQhMmKEi8wIueU06oQKnn7KEFlEK7J8aQM2Nq2tivIfHMQ3beEVpzVm34Ejp7xbqGEvE/D418VHf61RpK8WHJtcsv1CxIQ1o86ApQqjWU3K7eaM3aAUqEQ8OzG5UvM/0qgdLVV4EngDJ9+vS0evVqqlTJOFOKq5IIsXGvAqXZkTeGraWh5ECJgL0I62NUuIaSB8me++dJem3mVuVSdSYZONmITBgJFSg5GNrZLNQp1Oxca3Wpan2xwxYKNlFqoLRzXG8IlDO20PztPg2lsMvUO/LmYBQKUKo3Ibc0lOgvNlrYDorQMMEAZdtJ6yk6xmdDybNHuQGUIq6m1TUTbD0OlCIUkh6MWQVKvRSaOFJ/64c/adsxXzgy8Q7jfReyDBUo+Rrl7WsdeU/pWJXql8zph31kKdrQ112gnNKhKtUvlVMX9oRJk968/rj5GPX6abvyMzLVVCnkS1/LtcvP1yxIQ8MElOqPQ71+Cw96rTlXX4N4oFBelMubmea+JjWUwT7jVq7zBFACJMeOHavEgEyoJbEB5eELV/15skUoCaO5eWZStD/dG89pjU3ls5UHlaDZz1YvGNAE9yRMqEDJwdAOFEYKKJGW7sVYoISzFj44UOwc1xutAx6zMlxAGS6nHK3NK1Sg5NmjggFKM6ec4jkz0OKe9Vx/rXKgNAqFhI5YBUqe8UgL5hp8uEJxmoskUHIvb4Rfe9hloKw6fIkSHUMUM6BEuCeAuRWghMMKHFdQuO3oczUK0LCW5XShVUv+ZgtOz4ZSLxi+uj07QAltJkD1wbyZaN5rD5l1zfbvXt37bQ/EgQs8AZTR0dH0yiuvUI8ePahw4cKULFlg4N2EAJpeXVTBaiiPXLhKT0/yHXnbBkqDrCt8zXKgfLx8HvokAdpQJjSgHPy/MtShtk9DOXLBbvp8lc8+2Q4MG713oI2GVhoF6fBSp0hObmso1UerbmooxdjFhhgMUPKPLx7s3w2gLJkrIy160/0PdW4+YeS5bgcoMY9aWna/c9Pn62ljrOmAW0fe6K9RH3hgc3WueKS9jO7bwNEjbwQbBxz5gTJWK6p+JkWfkW3qQBBAybXL7aoXoBFPJFyghOkBzASQX37B6xIoHeBG3SY8AZRffvmlApR37vgi5vOC3N7IZuP1ktiA8ujf1wjOAyhwloEG0ajwYzz10bbedf9cu03lh/pS00mg1LdzCnbta22EyD7zQi1fFijIH3NcqWBWGvlk/A0jmPuK2J16QKlO4+mEDSXupZfebuFfp6nrdF+mnOkvVac6xbNrBucW/TBKvcjl4RRQ8mD/wQClmQ1luJIAcG23lue6yFYCGRpFjVDbnBrB3OHzV6nj1E3UskJeer1h8YDlKq4rkj09LQvRKcmoD9xRS52JCWkv1/dpQE0+WkV7z1xR+hfqh1uVYYvp/L+3/GMVx+x6QCnCwuk9yz9sPka9Y4+8uYYS9cW49TTKTjjlqG0ouV290ftHZCHSeibV12EeELuzdO5MhLSgThev7v1Oj9NKe54Ayvz589Njjz2maCjTpUsXr98FCwYelVoZWLjreHVRWQlsrmVDeeziNXpqog8ooTkE8BkVDpRGafx4GzyThgTK8ADl0BZl6fma9tKK2nmWXv9uK83Z5tNQCgN/tzWUuNdTE9fRpsO+eKd80/Y8ULJg/8EApZl9m1ubqHpNcKDU8lyHxzMyrpy8dIMGPFZaNxyUGlLgVLjjxD9+0wyrUBYJoFR/sOTJnIbW9WlAjT9a6XcichooxTF7sEC56+Rlav7JauVydZxgIfvB/ytLCA6udw+rc6IFfwj9M6VjNf9P3K7eKaBErEqkEXXr48qre7+d97ZTdT0BlHDK2blzp+382U4JwYl2vLqoIgGURmn8uKx58NyECpS131/mz3lsZ7Pg1wXzQrayZrU0K++1fJDa13DvA+2N77bSL7FAuX94M8WBJRxAyWM7eh0ouQMbj81qByihnVt78Dw9WTGfZv540ZZbx3xGQKnlua7llau1hvW0Xna1YeECygkrDtDohXuVoSBPeq1icRpwN4Cy8nuL6cLVOA2lGVCmSp6M9g1vZvi6QHpS2OI+VDyHldeKv46I4NGrSUl6tX4xW9eK+QkHUCL96PGLEihtTVCQlT0BlK1bt6bmzZvTiy++GOQwIn+ZV4EyWBvK4xevUetYDeXHbStQiwp5DYXMQ6GIDClms8KBskWFPPRx24QXhzKhAeWwlg/Scy4C5Zvfb6PZW08oUy9CEfH83G4defPYjgkJKHkoLTVQ6gWtNnuu8Ltoyy1HBHUfeH5xLUejxAqUPH7v911qUPUi2fyyF1lcGo1ZqWQXcuLDUQ2UIm6kej7E/EOziJMCNwqyCB04+6+i+YNpmp2iB5Tcrt6oPTtOOQXuT0cw4XLLntire7+d+XCqrieActy4cdSnTx9q1qwZlSxZklKmTBkwvoEDBzo1Xtfa8eqiChYoT1y6Rq0+8x15WwFKvqEb5YXmE3Dlxm0qF5veSwJleI68YVwPI3u3Ss/vt9GsWKAUoYjCAZTc5CKsQJkmBe0YbC8OpV5WKQ6UyLk+qlWU7Y1azKtoy61QKfGBcjvN3HhM+bOWo5EWAGitQcc1lA7E4TSyofx4yX76aMk+ZSg/v1KTKhe831WgrPTeYiUEjihmQIkg8HtdAspQ3iFCpg+XzEFT2ZG3VaC0Y0OJbFGHL1xTUuj+/qbzEQ+8uveHMj/BXusJoIRnt17Bl4/MlBPs9BIFe+RtGyhZOjkR0Nqs1zy9lwTK8AAlnG+eqeYiUP6wjWZt8WkoBVDyI2+3NJT8GDkhASXPKiU2WbPYgWbPFX4XbbmVHcQIKLUcjbQAIDEA5ZjF++iTpfuVoczuVosqFsjqlz2OWte88wg5qaGsOPR3unjttl90wrNcLUsx/2lSJqM977mjobSyDvXq6AGlGzaUiEWKuMpuxWSVQBk3y54AylAWpleu9eqiCh4orxPycKOMfboCtaxofOTNbdhE/EGzueFA2bJCHhqbAI+8a41cSif/8aULTQg2lKNalaOnq7oHlAg4/fOW44o8YkY0p2TJ7guwoXQLKLnWz+tA+dzkDbTmwHlFRjyrlNhkzWIHmj1X+B3e9rCNm92tNpXJk8nKJSHVeffn7fTdJp+GUssuFMGlAc9mJaFpKP9v0V4at/yAMqy53etQuXyZ4wFlwzErlaNhu+8ILVlVGPo7XbIBlHpRAMzmwe3f9YDSDS9vZIuKOXeViuZIT0tdyBrl1b3f7TnUal8CpUNS9+qiMgNKcQygzuV98p/r9OQEH1B+9HR5eqJiPkNJBQOUV2/eobKDFintSqAMj4ZydKsoalM1v0OrPn4zb//4JyGdG4oIlh4ODSWHtIQElDyrlJNAiTiC127dpfSpU7g217zhd37aTt9v9gGlll1oqEC55ehFGjjnL3rtkeLUpGxc6k69wfmdclw+8h61cA99tuKg0o35PepQ2TzhBUoRqkgtBzF+vcD3YVkUBjfRA0qrgc3tHHlDMwmgdysNqVf3/kjMsSeAEoHMjYx6ZRzK4JeGHlA+VDw7IUZb57pFKF/WdGQElGPalKcnKxkDJQ/bIgJam/X62q07VGagDyifqJiXPnq6gtklnvu95silSowzlISgoRzdOoraVHEPKHv9+Cf9GAuUQh7hsKFs/+UGWr3fp/UzA8rHx62h7cf/oSZlc9Hn7aso14gNznYcyiBsKHlfeVYp0QezYNSeewiIiAPlH/0bUrYMqQPkmjVdSto6sLFp1+16c4cDKHkCAHE/scb4b4veqEslH8joX0v5709Lq3s/QkYZfUwFoqqg1lCaAWX6VMlp59D4ud7t3tfp+npAaTX1otYHipatK/otcr87EZNUSw4SKOOk4gmgnDp1agBQ3r59m/bv30/4+5gxY+jZZ591ej073p5XF9X06CPU/5e/AsaLzDfInZwmZXL/379cc4jem7dL+TfikWVKk5IqDl1M12/fJb7p6QmOA6WIP2gm5Ou37lLpgQslUNqEUTO5it+1XrAftI6ip1wEyt4//Uk/bPZpKMMJlDxDjxlQwvB/1f7z1KBUTr8Gz0tAaRaM2ur8h7Men3etY/zECpTD5u2iyWsOKaJe0rOeYqcn1hK8i1f1ru8oUJYf8jshfq8oZkCZIXUKQhg3rxUho3olchAci0RRp1HV67cdoIQX+p7TV8iJvO4SKI1XkieAUq+LM2bMoGnTptFvv/3mtechXn8SElBqadIAd60nrqNC2dLT+GcrKeODjePdu/9R5nSBXvdak8GBUsQfNJs0DpRPVsxLY6SG0kxktn6PBFByTVU4gfLM5RuElIb1S+WkAY+V8ctJK7C5lhAjBpQsTak/1IuF2IG2FkIYKnPNtNYxfpZ0KWlbItRQDpm7k6asPaxIePnbDyvQIuaxYLZ0tLKXs0AZNXgRXb4Rl1FOBFNXT7Hog1lq0DAsDc1b6AElT3Zh1Dc7QIng/rtPXSZ4e6/oVd/xIXt173d8oBYa9DRQHjlyhMqUKUNXr161MJTIVvHqotLSUNo5mrUq1dafraPNR3yZSkT8QbNrb9y+S6UG+DSUCRUoa4xYSqcv2z/y5s48GL8bc6IFlP/3VHlqXdnYfMFs3ox+n7f9JHWfsVWpogWUk9pXpsbMBk70ERrzyS/4jp+dLF4HSp6m1A+ULsYOdFK2vC0OlFrH+JnTpqQ/B4X/yNsJR4yRv+2mz1f6ct6LItb2oDl/0dfrjyh/FjnM1UD5yIcrFKcQJ57zcoMX0RU7QGlikuHWejBrVw8ocd3k1TFK3EiY5ny+KoZK587oDx4v2tXSeOsdeZfNk4l2nrxMQmNs1je7v3t177c7Difqexoo33//fRo/fjwdO+Yz9vZy8eqimhZ9hAaojrzdgBcOlCJcjNl8JQagrD5iCZ25fNP2ZpFYgRLOIHDKKZIjA1UumNW30e4/R+2/3Kj8twRKIm5DydOUJhqg1NC6ZkqTgrYbxOsU7wqnbSjdBsr+v+yg6dFHle6LlJNiDEIj9sj/raCY8+4A5bedqlPtYtnjvWpFH6zK3exd7fTvRkCpvtfyPWeVnO282AFKxGJF+k5h0+r0WLy69zs9TivteQIoEYeSO+VgU7p8+TJdunSJhg8fTu+++66VsUS0jlcXlRoo32xYgl5vWNxxWSHE0B+xGkoRLsbsJjfv3KWS/RO2hjKhAeWHT5WnVi5qKLXmXAKlTyriQ+75rzbSqn3nlL/xNKVik/VqMGqj55l792tBckaLmrKEBpQ8Q5AI6C7GIGz26v/fCiUOIl8DZu9Gvd/LDVpEV27GHXmbAaVVzXCw/Qn2OiGj52oUoGEtyxk2s3T3GXrp680BdewceUfly6w44Ym4oMH2We86r+79To/TSnueAMrBgwfH8/JOlSoV1a5dm+rWrWtlHBGv49VFxYHyx641qWqh+12RFQdKES7G7EYBQFkpL41pk/C8vCVQms0yKfAEiEKRGkqiF77aSCtjgZKnKU3IQMnjj2pBckIGyvd/20MTV/pCA4kiPg64zbCIvynmUXgVOwmUDw5apNi2iyLyh6ufQtEHq7ar5k+xszWmrj2kRGXAO9/MRv/3naepy7Q/ggZKBPf/8/g/JFJhOjsSIq/u/U6P00p7ngBKKx31eh2vLioOlLO61aJKBXzHkE4XDpRWj9Rv3blHJfr7HK6eTKBAWW34Ejp7JeEceVsJAeX02ogsUJ6irtO3KEPS0+bgN7EBiyNSM02Z3+khiLBBHaZspBV7fRpKnqZUtOnV7CZG64IDpRYkJ1ag5JpZ4d3uJlCWHbiQrt66axkorXrXO/3MO9newr/inmHRrp0j74oFstDWo5ckUDo5KTpteQYo165dS2vWrKFbt24Rjrx5kbm8g18J09YfpgFzdioNiNRgwbemf+WTE9bSlqOXlApWgfL23XtUvF/SBEoev9KOzOzMnZaRupUg9XbuYaVuZI+8vQ2UPE2pmC+vZjcxmuueLOUmHxPst2duPErfvFSNahWNb+unbtMM5K2sN/6B4ES6PSMN5Zvfb6PZsbnrhYOVGigf/mC5kkvaiec8HlB2rq4pV9EHqwHlrco1EvXmbz9Fr87wfRQGA5Sw5YY5Vu7MaWh9nwaOD8GryiTHB2qhQU8AJZxv+vbtS1mzZqVMmQLThMlc3hZm0aAKB8pfXq1NFfJnCa1BnauDAco7d+9RsQQOlFWHL6FzQWgowwGUTceuUuKv8RIJoJQaSt8MiA+tjlM20vJYDSVPU5pYgJKPCcoBaNQQD9FKSWhA+fp3W2nOtpPK0IRm1g+UsVl6tGKzWpGFVp0yAxcqGZBEmWEClFZzqAfbn3Bc9+ufJ5VUosECZdVCWWnT4Yv0QKY0FN1XAqWbc+YJoMyXLx+9+OKLNHTo0JDGOmzYMFqyZAmtWLHC3862bdvo9ddfp82bN1O2bNmoR48e9Pbbb/t/v3fvHg0ZMoQmT55MFy9epDp16tCECROoWLFitvri1a8UDpRzXq1N5V0CyicmrFWOFfjGaSZADpStKuWjD9uUN7vEc797GShPXLpOE1ccVLwbRyzYo8jOSl52p4UsgdIeUHo1XZ7Ruuj5/TaaFaup40Bpdy15ESh5ekUxHvFx0H3GFpq3/ZTyZzFutfkEYit+tHgfNSidkx4qnsOuSALqq4FyZucaVLNotnhtij5kz5CKNvc3z6EeUqdcvviXrSfoje+3BQ2U1QrdTxsP/025MqWmDX0bOt5br+79jg/UQoOeAMp06dLR9u3bbUMcH9/YsWOpZ8+eihOPAMoLFy5QqVKlqGXLlspv0dHR1K1bNwUYO3bsqFwOmMS/p0yZQnnz5qXevXtTTEwM7dy5k+AYZLV4dVFxoPy1e22KyueOhjIYoLx77z8q2neBIuKECpRLdp2hTt9sprolctA3LOOD2boJh4ZS9GHdwfPU7osNEih1QqxAMGIDDkfqxRenbqJle84q88HTlIo+JESg5Ee/VjNlaT0jCQ0ou337By3YcVoZikjooF5LZu8CO7+XHrBQyV4mijlQplZyqyfk8vMfx+mtH/8MGIKWs5GWiQ8uql74ftpw6G/KmTE1beznvCy8uvdHYs49AZSNGzdWAO+ZZ56xLYMTJ05Qp06daPXq1ZQ/f37KlSuXHyhHjhypxLE8fPgwpUjhO3LB0fqsWbNoz549ir1m9uzZafTo0dS1a1fld4QqypMnD3311VfUtm1by/3x6qL6Zv1hGhhrQzm3ex0qly+z5THZqRgMUN679x8VSeBACRmdvHRdeVmlSJ7Mssh4QHRcZNXu1PINWMX1By/QM19EK3/5uG0FalEhbzDNBH0N11B+8XwValQml78tsQl0rVeU3m1WKuh76F3IDfq94pTDgZLDl5CFV/MvG00OB0qrmbK02nMaKIvnzECLe9YLaV0ZaShfnraZFu08o7QvwqW5CZSlBvxGN27fswyUOTKmJnifJ+Tyw+Zj1Pun7UEDJTJnVcifmVIlT+7K/ufVvT8Sc+4JoMRxMzSDgEpoFFOnTh0gi+eff15XNnPnzqWZM2cSjrtxZA54FBrK5s2bU5YsWQgpHEXBkXijRo3ozJkzSt3q1avT3r17qUSJEv46OPaOiopSNJdWi1cXFQfKea/VoQfzugOULcevpW3H7B15w76qcJ+EraG0uj7U9cIJlNExF6jtJG8C5Y+bj1F0zN/0XsuylC6VNTs7OzL3OlBy+EosQGk1U1ZCAcrRC/fQhBXaYYM6fb2Zluz2AaX4KBTz6ATMqmVUsv9vdPNOHFB+16UG1Siif+TtllbOzjMYat3vNx2ld37eETRQuv0R7dW9P1S5B3O9J4AyWTJ9zQ6ccu7ejVPxGw2yQ4cOAUAJKGzWrBmNGjXKf9muXbuobNmytGnTJjp69Ci1atWKrl27RmnTpvXXadOmjfK3efPmWZapVxdVQgFKpANEWsCkUsJ55L0h5gI97VGgdHu+vQiUL03dREtjj7w5fAkQgQMLYjkmpPLGd1vpl1jnFKuJDRIKUBppKLm2WQ2UJXJloN/fDE07GipQumU3GM61OWPDUeo72xwotx+/RH8eu+SPaiL6+MkzFenx8nlc67JX937XBmzQsCeA0qmBq4ESjjXt2rULcPaBfWTRokWVI3JoKNu3b68AK4daaERPnjypOPjoFWTywf9EOXXqFFWrVk1JEwknI6+UcAFli/FrlYeZf6lbkYHYRJMaUNZ+fxnBaUYUN4+8OVC6/XLVmnME8UYwbxT1kbeVNRJKHa8DJU9T6o9tmToF7UhgQMm9na0mNkgMQMmzHqmBsmSujLToTWcTc6g1lN93qUHVDTSUbnk2h/JM2r1WK32wXsB2bpcvgdKupEOvn6iBsly5coRjby0N5ZYtWxTnm9atW2tqKG/evElz5szRlTCy+8ChR128DJTze9ShsnncOfKWQGnvYawzahkdvxgeoORH3p8+U5H+5+LXugTKQAlo2QR2+noTLdntc8rh2rzEApShfBx50YbS6Mib52UPB1AiEQQSQohiBpRuxV6097YLrTZXioiW9ICS2+WLum6/86SGMm5+EzVQAiYRKmjatGn+EWvZUB44cEDRWooCG8ry5csrDj16JSFqKBf0eIjK5AmM8xnaox53dahA+VTlfPRBEjryfmj0Mjr2d3iAkjvluP1y1VpPRk45Tq0/vXa8qKHkQMm1eVaz77gts2Da5xrKpASU7b6IpnUHLygiUwNlqQcy0sI3nNVQluj3G926GweUP7xck6oVjp9OV6ylxACUU9YeoiFzdwUsSz2g5Hb54oJx7SrSY1HyyDuY59ruNYkaKBEwfeLEiXTw4EFKnjy5Ips+ffrQ7NmzFS9vaCFz5sxJY8aMoZdeekn5PTF7ef/2+kNUOrdLQDlujZIvlb9YrSxG8eJLakBZ74PldCQ2e4ZdmVmRK6/DgdLtl6tW3+SRt08qAji0HDnwe0IGSgSeRgDqUNdyQtNQvjZzK81VjVuMwQ2gLN5vAd2+G5dJzgwo82ROQ+tcyA5j9x0USv3Jq2No2PzdAU1kTpuS/hzUWLNZdfggt995UkMZNw2JGijPnj2reI0//vjjihf5xo0b6ZVXXlEg84UXXlCk0K9fP/r888+VMEGFChWiXr16KbaVO3bsSBRxKPlxwcI3HqJSD0igDOXl5tS19f9vBR06f9XfXChaHbM+8TiUbr9cJVAGSkD7yDu+ZzAHykxpUtD2wQnLKUcLrMzWpdbvTgOlE44xRkfeZy/fUOLQNiydi3o0KK4MKZxA+WPXmlS1kL6GMm+WtLT23UeCmQrPXDNp1UF/YgbRKTtAOb5dJXo0Krdr45FAmUSAEsOENzey42zdupVy585Nb731FnXv3t0vATjkIDYlAptfv35dCYyOo27AC9pXQQAAIABJREFUpZ3i1UX19brDNOhXXy7vRW/UpZIPZLQzLMt1W0gNpWVZoeIjH66gmHPhB0q3X64SKM2BsvM3m2nxrsBQMxIofXLzIlByDb+YXaMPQDEGnAbhVMjJUqzvArpzL05DmRSAEuHoEJaOFztAOeHZStS8nARKJ9ehXluJSkMZDoHp3SMhAOXvb9alErncAcrHx62h7UEceYvwOW7CbiTXhd69G45ZSQfO/uv/2VUN5YHz1G6yL1NOpIFy8vNVqCELbO723HjRhtIMKI02S7flFWz7PAVhKGvZi0AJmSzdfYZe+nqzpec1nED5U9eaVMVAQ5kva1pa807C1lBC6Mv3nqX706Ui2OqjGGnx1Ufenz1biZpJoAz20bZ1nQRKW+LSr5wQgHLxm3WpuMeA8sbtu4Rct7kypXFoJhJGM43GrKT9Eihdn6yECJQPFc9O016q7rpsnLzBqzO20PzYnNaJESjtaE8F0JTJnYkWOKyhRKpahMYRxQwo89+fllb3TvhAKcYrZCuB0smn17m2JFA6JMukDpT/+3QN7Thh3ynHIfEnuGaafLSK9p65YknjEerg1h44T8/GaijdPv7R6uuRC1ep3gcrlJ+WvlWPiubIEOqQLF/PgXJGp+pUq1h2zWvFRlU0R3pa+tbDpkevVh1otDRuXb7ZTL9rHHnjGPyXbSdo4GNlEtwHFs9m4iWgdDIWpFXtqahXNk8mmt/D2SPvIn3mE+NJ+vmVmlS5oL4NpVspTS0/gA5XDAYoJz5XiZo+KI+8HZ4KzeYkUDokZa8C5dS1h2hwbMgFNzWUEijtLaSmY1fRntPhAco1+8/Tc1/6jrzdPv7Rk8KinacJIT3cfLFr3duLQMnzP4cCX/ZWnLu1oTX7btNRxemvcsGsQd/MKrSZ3UC082DeTDTvNWegzmrf3Li3GG/hPvPpvzgFpS5Q7jl9maIPXqC21QpQmpS+CCeJoUig9PYsSqB0aH4SAlAu6VmXiuV0x4ZSAqW9hdT849W061RcpiU3wYIDpdtf6/ak4H5tLwJl12l/0MKdp5XBuznv7kvX+TtYhTazOw/45S+avfUE/fRKTcciW1jtmxeA0kw+CfX34ICyMjV98AHXhuzVvd+1ARs0LIHSIal7dVFxDaWbQPnYp6vprxM+QJKbpPmievST1bTzZHiAcvX+c9T+S1/qQwmUkT/ylkCp/3xYhTbzJ4zo9t17lDJ5MitVLdWx2jdRr1zezDT3tTqW2rZaSe1w8vMrtULSCFu9r1fqBQOUn7evTE3KSqAMxxxKoHRIykkdKDkgSaA0X1Rco+s2hAcCpbtf6+YjD28NL2ooX5n+B/32l9RQaq0Eq9AW3lXku5vVvol6Ufky06/dnQXKMYv30SdL9/uHL4Ey/kpQQ7cEyvA9LRIoHZK1V4GSp61yU0PJj3AlUJovKh63s/+jpanTQ0XMLwqyBk99OPE5CZRGIGPXKccsCLkWhEigDI+GMsjHRfcyLwAltK4IO7Vi7zmln7O61aJKBYK3WXVaRm63F4yGclL7ytRYaijdnhqlfQmUDok5YQBlPSqW0x0PWwmU9hYSAvUiYC/K/uHNHD2aU/eEA6XbX+v2pOB+bbc1lMEAZbdv/6AFO6SGMrFrKMvny0xzHNZQQmY8+1lSBcqMaVLQDp1sUmoN5RfPV6FGLsa+9ere7/7bNf4dJFA6JHWvLqpADaUESoemO+RmnpywlrYc9QHlgeHNKIWDtl7qzvFc2hIonbWhDAYo52w7Qa9/t02ZJqnND1ytVrWAIT+AQTRgtW+iXvn8WWjOq7WDuJPxJdOijxCcjlBmd6tFFZOghtIOULqdTMGre7/jC89CgxIoLQjJShWvLqpwAWWzj1fT7livZblJmq+YVp+toz+OXFQqxoxoTsmS3Wd+UZA1Vuw9Sx2mbFKudvv4J8guunaZFzWUCJ80f8cpKp4zo2upUF0TqMsNW4U2l7uh2bzVvol6FfJnoV9cAMrp0UeovwRKyxpKCZThe1okUDoka68C5VdrDtHQebuUUS7p6Z6GUgKlvYX01MR1tOmwDygPjWxO993nHlAibVnHWKB0+/jHnhTcr20XKB8tl5vGP1vJ1AHDii0XRmcVQtyXRMK4g5flZbVvbgPljA1Hqe/sHcqEAlgBrkmlCNlmTJ2CdgxpYgr+qPDlC1WoQelcronIq3u/awM2aFgCpUNS9+qi4kDpZpYSHqhbaijNF1Wbz9fTxkN/KxXdlhcHSre/1s1HHt4aVoHy1z9P0qK/TtPQFmUpW4bUpiAogdKdebQKbe7c3bhVq30T9Xo1KUmv1i/meFe/23iU3p0lgdIqUH7VoQo9UkoCpeMLUaNBCZQOSVkCZVzmF7cByaEpi2gzbSetp+iYMAHlnrPUcarvyFsCpbYNpXoxmMGDBEp3Hh8zubtzV2utWu3bjuP/0B9H/qZ21QtSqhTOxcEUvfxh0zHq/fN25Z+w0YStZlIpwWgoJVCGb3VIoHRI1gkBKN088tbLT+yQeBNdM+2+iKZ1By+ER0PJgNLt4x+vTdRvO07RK99uUbpllMtbAqU3Zs4qtEWit17p24+bj1Gvn3xAKZ1y4q8EtZf3lA5VqX6pnK4tGa/u/a4N2KBhCZQOSd2riypcNpRnL99QPFebl3uA2tcs5JBUE28zz06OprUHJFC6PcNuA2WLCnno47YVdYfhFQhxW85Ote9leXmlbz//cZze+vFPReRJNbC5HRtKCZROPZ3m7UigNJeRpRoJASjdtKG0JCRZyS+B9l9uoNX7z4dFQ7lszxl6cepm5V5SQ+nMkffWoxdpye4z1KVuUcqcNqUESoeeba9Am9ZwvNK32VuP05vfC6CsSZUL3u+Q9L3fTMWhv9PFa7fp7cYlqPsjxTU7HE9D2bEq1S8pNZThmF0JlA5JWQKlQ4JMIs08/9VGQsBxFLdtTjlQum1P5LXpc0tDaXWcXoEQq/2NdD0vy8srfeNxTH/sWpOqFko6QHn+35u0/fglqls8h27sXjVQTu1YlR6WQBmWR1sCpUNilkDpkCCTSDMdpmz0p09zGyiX7j5DL33t01BKoHRGQ2l1mXoFQqz2N9L1vCwvr/QNEQl6zNyqTNX3XWpQ9SLZIj1tnrq/GigX9HiIyuTJ5Fofvbr3uzZgg4YlUDokda8uqi/XHKL3YuNQyiNvhybbgWY6TtlIy2Pz8boNlEt2naFO3/iA0m17IgdE42gTUkPpqDhdb8wr0KY1UK/0bd72k9R9hg8oZ3auQTWLSqDk88Xnya3QTfx+Xt37XX9YNW4ggdIhqXt1UXGgXPZWPSqSw51c3g6JMck089LUTbR0z1llvG4D5eUbtylq8O/KvbYPbkyZ0ujb/CW2CZBAmbBm1CvQ5mWgXLDjFHULInJBwloJwfc23GvIq3t/8BIM/koJlMHLLuBKry4qCZQOTbDDzXT6erPi1BEOoMQ9Lvx7U8nGc3/6VA6PxNvNBQuU09Yfpo+W7KdPn6lItYtZOyb3MoR4e5biehduGLAjF6/0jQfrn/5SdapTPPj1aWf8CaVuuOfJq3t/JOZLAqVDUvfqopJA6dAEO9zMN+sP08A5O8MGlA53P8E0FyxQYoDIuR1qSsxwb24JZmJ0OupleXmlb4t2nqaXp/2hSPCbF6tR3RI5Evq0O9r/cM+TV/d+R4VqsTEJlBYFZVbNq4tq8uoYGjZ/t9J9eeRtNovh+/3uvf8IWrCyeTMnKS/N8EnYd6dQgNKJvoZ7c3Oiz5Fsw8vy8krfft95mrrEAqXbHsyRXAvB3jvc8+TVvT9Y+YVynQTKUKTHrvXqouJAufzth6lw9vQOjVg2IyXgfQlIoPT+HPEehhsG7EjHK31Lyk52VuYr3PPk1b3fiqycriOB0iGJenVRrdh7ljpM8eVx/qN/Q8qWIbVDI5bNSAl4XwISKL0/RxIo7c1RUk5UYEVSEiitSMmdOhIoHZKrV4ESdmCTVsVQjoyp6clK+RwarWxGSiBhSCDSQDl64R6asOKg8vxt6tcwYQgtgr0MNwzYGapX+rZ871nqGKskmPx8FWpYJpedYST6uuGeJ6/u/ZGYaAmUDkldLiqHBCmbkRJwUAKRBspbd+7Ryn3nqGKBLJRdng6Yzmy4YcC0Q6yCV/qGDFvItIXyefvK1KTsA3aGkejrhnue5N4ft6QSPVAeOXKEChUqFO8h+uKLL6hTp060bds2ev3112nz5s2ULVs26tGjB7399tu2Hzq5qGyLTF4gJeC6BCINlK4PMJHdINwwYEd8Xunb6v3nqP2XPqCc+FwlavpgbjvDSPR1wz1Pcu9PQkA5d+5catOmDcXExASEAMmcOTNdu3aNSpUqRS1btqSePXtSdHQ0devWjSZMmEAdO3a09eDJRWVLXLKylEBYJBAAlJ2rU62iMmZfWAQf5E3CDQN2uumVvq09cJ6enbxB6fqEZytR83ISKPk8inlKmfw+2j+8uZ0pDqqu3PuTEFCOGDGCfvjhB0UTqS4jR46k8ePH0+HDhylFihTKz3379qVZs2bRnj17bC0uuahsiUtWlhIIiwQuXbtFFYYuVu7115AmlCG17zmXxZsS8Aq0aUnnqzWHaOi8XdSkbC76vH2ViAlw3cHz1O4LH1COa1eRHovKE7G+ePHGYg1lTpuS/hzU2PUuyr0/CQFl27ZtKVmyZDRjxox4C6t58+aUJUuWgN+WLFlCjRo1ojNnzlDOnDktL0a5qCyLSlaUEgirBM5duUnJ7iMZ4SCsUg/uZgIGiuXMQEt61guuEZeugoPjwXNXqVC2dJQieTKX7mLebHTMBWo7KVqpiExO/ysvgZJLTayh3JnT0Po+DcwFGmINufcnIaAsW7Ys5c6dm27evEn79u2j4sWL04ABA6hJkyYUFRVFzZo1o1GjRvklsmvXLsI1mzZtoipV9L9CL1++TPifKKdOnaJq1arRsWPHKF8+6U0d4jMqL5cSkBJIghJASJwZG45S/0fLUCEZM1dzBRy/eI3qjFqu/Da/Rx0qmydzElwp+kMWQNmlbhHq27y067KRQJlEgPLWrVuUPn16qlixIn3wwQeUIUMGmj59On388ce0ePFievnll6ldu3Y0dOhQv0Rga1m0aFFavXo11alTR3cxDh48mIYMGRLvdwmUrj+/8gZSAlICUgJJWgILdpyim3fu0hMVpfJCvRC2H79E6w9eoBdqFaI0KZO7vk4kUCYRoMQwr169qthHpk4dF9C7adOmioMOFgKOvbU0lFu2bFFAVK9IDaXrz6m8gZSAlICUgJSAlICnJSCBMgkBpdZK7NWrFy1cuJDy58+vhAqaNm2av5q0ofT0sys7JyUgJSAlICUgJeAZCUigTCJAuX37dqpVq5YCj/z4un79+pQrVy6qUKECTZw4kQ4ePEjJk/tU43369KHZs2dLL2/PPK6yI1ICUgJSAlICUgLelIAEyiQClPfu3VOAEsfen332GWXPnp0mTZqkxJmE0w2gEnEoH3/8cerduzdt3LiRXnnlFQUyX3jhBVurVy4qW+KSlaUEpASkBKQEpAQSvATk3p9EgBLDPHfuHL377ru0YMECunTpElWqVInef/99euihhxQpACyRHWfr1q2KN/hbb71F3bt3t73I5aKyLTJ5gZSAlICUgJSAlECCloDc+5MQUIZrpcpFFS5Jy/tICUgJSAlICUgJeEMCcu+XQOn4SpSLynGRygalBKQEpASkBKQEPC0BufdLoHR8gcpF5bhIZYNSAlICUgJSAlICnpaA3PslUDq+QOWiclykskEpASkBKQEpASkBT0tA7v0SKB1foHJROS5S2aCUgJSAlICUgJSApyUg934JlI4vULmoHBepbFBKQEpASkBKQErA0xKQe78ESscXqFxUjotUNiglICUgJSAlICXgaQnIvV8CpeMLVC4qx0UqG5QSkBKQEpASkBLwtATk3i+B0vEFKheV4yKVDUoJSAlICUgJSAl4WgJy75dA6fgClYvKcZHKBqUEpASkBKQEpAQ8LQG590ugdHyBykXluEhlg1ICUgJSAlICUgKeloDc+yVQOr5A5aJyXKSyQSkBKQEpASkBKQFPS0Du/RIoHV+gclE5LlLZoJSAlICUgJSAlICnJSD3fgmUji9QuagcF6lsUEpASkBKQEpASsDTEpB7vwRKxxeoXFSOi1Q2KCUgJSAlICUgJeBpCci9XwKl4wtULirHRSoblBKQEpASkBKQEvC0BOTeL4HS8QUqF5XjIpUNSglICUgJSAlICXhaAnLvl0Dp+AKVi8pxkcoGpQSkBKQEpASkBDwtAbn3S6B0fIHKReW4SGWDUgJSAlICUgJSAp6WgNz7JVA6vkDlonJcpLJBKQEpASkBKQEpAU9LQO79EigdX6ByUTkuUtmglICUgJSAlICUgKclIPd+CZSOL1C5qBwXqWxQSkBKQEpASkBKwNMSkHu/BErHF6hcVI6LVDYoJSAlICUgJSAl4GkJyL1fAqXjC1QuKsdFKhuUEpASkBKQEpAS8LQE5N4vgdLxBSoXleMilQ1KCUgJSAlICUgJeFoCcu+XQOn4ApWLynGRygalBKQEpASkBKQEPC0BufdLoHR8gcpF5bhIZYNSAlICUgJSAlICnpaA3PslUAYs0Hv37tGQIUNo8uTJdPHiRapTpw5NmDCBihUrZnkhy0VlWVSyopSAlICUgJSAlECikIDc+yVQBixkwCQAcsqUKZQ3b17q3bs3xcTE0M6dOylVqlSWFr1cVJbEJCtJCUgJSAlICUgJJBoJyL1fAqVfArdu3aLs2bPT6NGjqWvXrsrfL126RHny5KGvvvqK2rZta2nhy0VlSUyykpSAlICUgJSAlECikYDc+yVQ+iWwceNGql69Ou3du5dKlCjh/zuOvaOiohTNpZUiF5UVKck6UgJSAlICUgJSAolHAnLvl0Dpl8CsWbOoVatWdO3aNUqbNq3/723atFH+Nm/ePEsrXy4qS2KSlaQEpASkBKQEpAQSjQTk3i+B0i+B6dOnU/v27enu3buULFky/9+ff/55OnnyJC1ZskRz4V++fJnwP1GOHTtGtWrVImg8c+fOnWgeFjkQKQEpASkBKQEpASkBbQmcOnWKqlWrRocOHaJChQolaTHd999///2XlCXw888/U+vWrTU1lDdv3qQ5c+Zoimfw4MGKZ7gsUgJSAlICUgJSAlICSVsCUCZVrVo1SQshyQOlsKE8cOAAFS1a1L8YYENZvnx5Gj9+vCUN5Y0bNwhaysKFC1OKFCkcXVTiC0hqP+PEKmUSuMSkPOI/clImUiZWXsRynch1Eso6uXPnDp07d47KlStHadKksdJUoq2T5IESWsicOXPSmDFj6KWXXlImOhgvbzdXiLTRiC9dKZNAmUh5yDVi5R0k14lcJ3KdWJGAXCfBSCnJAyWE1q9fP/r888+VMEGwgejVqxcdPnyYduzYYTkOZTDCt3qN3ATkw222VuQakWvEbI3gd7lO5DqR68SKBOQ6CUZKEiiJFIecvn37KoHNr1+/TnXr1lWOur1iYCs3Aflwmz3cco3INWK2RiRQaktIPjvy2ZHPjhUJmNeRQGkuo4jXgDc5juR79uxJmTJlinh/vNABKZPAWZDyiL8qpUykTKy8q+Q6ketErhMrEjCvI4HSXEayhpSAlICUgJSAlICUgJSAlICBBCRQyuUhJSAlICUgJSAlICUgJSAlEJIEJFCGJD55sZSAlICUgJSAlICUgJSAlIAESrkGpASkBKQEpASkBKQEpASkBEKSgATKkMTn7sX37t1TsvFMnjyZLl68SAi2PmHCBCpWrJi7N45Q63///bfibY/86TCUj4qKovfff18ZN8q2bdvo9ddfp82bN1O2bNmoR48e9Pbbb/t7m9jltW/fPqpUqRKNGzeOOnTokKRl8s033yhrIyYmRklIgMxVTz31VJKVye3btxUZTJs2TXlXVKhQgUaNGqWkg02Kz86wYcOUtLkrVqzwvx+ceH+YtRGhV6fpbbXkMXfuXBo6dCjt3r2bsmfPrjw/+HfatGmV9qy8TxOqPDA+LZlwQXbu3JkWL16shBAUJbHLxHQhmVSQQBmqBF28HjAJgEQ4o7x581Lv3r2VDXTnzp2eiI/p9NAbN25MZ8+eVYApR44cSugmwPSWLVuUf5cqVYpatmypeLtHR0dTt27dFPl07NhR6UpilheAAXAAmMZ6AFBeuHAhScpk+vTpyvg//PBDevTRR2nmzJkKTK1Zs4ZKlCiRJGUyaNAgmjRpEn399ddUpEgRBSZ/+OEHBRZSp06dpGQyduxY5R2B8G8CKJ14Vqy04fQ70Yn2tOSxevVqql+/vgJVSD2MTHFdunShBg0aKO8XK+/ThCoPjE1LJlzWv/zyCz3xxBNUsGDBAKA022MSskycWGsSKJ2Qogtt3Lp1S/lqHD16NHXt2lW5g9cy+Dg5bLzQihcvTmvXrvVrVZBmHoDwzDPPKF/NAEx8LYrUltBmzpo1i/bs2UOJXV4YKyB6+fLlfqAcOXJkkpMJ1gSACZvgBx984F+CTZo0oYcfflj5d1JcJ9BIAgYA2SjQ8GfOnJl++ukngmY7KcjkxIkT1KlTJwIs5c+fn3LlyuUHSieeFbM2nHwfOtGWkTyee+455eP9999/998KH2r4OP/333/pvvvuM91/Epo8MFAjmQhBIBUn0i6XKVNG2W+EhtLKHpMQZeLEWhNtSKB0UpoOtiVyjO/du1eBKlFw/IujYGjmElMBLK9bt07ZFKFREQXH+02bNlU0s1myZKEZM2b4f8ORVqNGjejMmTPKQ1+9enVKjPJatWqV8rWM46UCBQr4gbJ58+ZJTib4eChdurSita5YsWK8RyApygRCwHODAN2LFi1SYOqjjz6i/v37K6cZr732WpJYJzjChbYaWjcc3eKdIDSUTqwLszaQwtdLxUgeW7dupWTJ/r+9M4utqWvj+BMNkVLzBTE24QYJkpoiTS8EEWpoo6WCIFESCUKFxBCaiktD0GoMl60qao4g4QYxRKhGgtBeSNwIEVNUvPk9X/bJ7ra/9rz22d5zzn6eG9qevc7ev7POWv/1DGt1UuHkGIJyyZIlei41421H42mq8eA522PC31mwsjjlXO6srCw5efJkTFDGMyenIpNE9lkTlImkmcC28LwVFhbKly9fYjktNF9UVKS/I88w3a2urk6f9/z583o85syZMzWU51hTU5OMGjVK7t+/Ly0tLWnJC6HNoM/Kt6SkRD0HTsibhUXUmNAX5s6dq16o8vJyYWLMzs5W8ZSfn6+Lragx4fvQ2NgoxcXFwnciIyNDxQLeyTlz5kSSCSkRbkGZiH7RURs5OTlJOyR7eXhvFO8bApJ+8/DhQ438dDT/pDIPnt+PCQeIML6SWsSY6xaUUWAStAOboAxKMKTrndUix0LyJXds6dKl8vbtW004T2cj9I1nEs8L+Sx4KhFUeB4ccwoyEBdMHqyu040X4X5WzTU1NfrYbkEZRSbO9wIRSd4gYru+vl4qKio0gb60tDSS/YTF1/79+6WsrEzzraurq6W2tlbwbhcUFESOiVcsJOK70lEbTvFgMo7L7QnK1tZWWbx4sY6z9BeEZTzzTyrz8BOUT548kdzcXF2sIpbJy3YLyigwCdp3TVAGJRjS9UyS5In5eSi/f/8uDQ0NIb3zf98sz4Z4nDRpknpiyZ8kBEE4wc9DSfgTcZluvKjY3bp1qzx9+lR69+79m6CMIhMKTfDEkfKxZs2aWGelOAfDUx21ftLc3Kz5xzdu3NAJ0TH+Tx42+clRY+IVUIn4rnTUhl8Kxn8/mv7vDv6foPz06ZNGgcjNZlGClx+LZ/5JZR5eJt++fRM8zMuWLdNFGeYVlFFgErS/mqAMSjCk6518DSYDtkVxjFUwXhmS7NPRqPBmayC8KqwInXxKJkS2CkJkOeaXQ5lOvKjCpHLZnVP6+fNn/Zk+QQVi1JiQZztlyhT1Irg9QuyAwOJj2LBhkWPipIZ8/fpVunbtGvt+MDFevnw5kv3EK6ASMX501Eay5VC65wc/QUnxCekhr1+/Vu8k441j8cw/qczDKyhv3bqlRX2ZmZkaBcJIA8B7y++qqqp00Yb3tr05JtWZBNUUJiiDEgzperyQDFDkdKxcuVLfJZ2rvHm+I0eO6FZA7C9JUYE71M+eg5WVlfLq1SvNEcPw3p09e1arvNORFxWJiAS3MajhpcVLRwFC1JjAg+rdHTt2tNmDlImxc+fOukNA1JhQ/T958mS5d++eTJgwIdZdSBlBYOLpjxoTr4BKxPjRURshTQUJadbLg71K+a6wG8DVq1c1AuS2eMbTVObhFZSMK4y3bjtw4IDmklLYxZjTpUuXDufkVGcStLOZoAxKMMTrKURhZXT8+HH1vOBxIFeQECidO52MrU1Gjx4ts2fP/q2CnZA3Axz7UFJkgDeKFTQhTyZKwhRYFHi5cyjZ9iOKTKjiZTstvhsIKPJLEZiEfNnqI2pM2GwZ7wrVuaQCDBo0SNj4naICPNxssxQ1Jl4BlYjvSjxtJOuY7OXBzyxIEZPsmuA29vxl0d7ReJrKPLyC0u9z84a845ljUp1J0P5rgjIowRCvp8CE/QepOmMFxUa9hLoRl+lme/bs0QHMzxCMJEdTzY33ksreAQMGyMaNG2Xt2rWxS6LAyy0oefCoMsFzT3oEXgUmRDYcpvo7qkzwOFHpTtif/+Nx4juVl5cXSSZ+Id5EfFc6aiNZx2U3DxYg3bp1E/IG/YwQOHNMPONpqvL4U0GZ7kyC9l8TlEEJ2vVGwAgYASNgBIyAEYg4AROUEe8A9vhGwAgYASNgBIyAEQhKwARlUIJ2vREwAkbACBgBI2AEIk7ABGXEO4A9vhEwAkbACBgBI2AEghIwQRmUoF1vBIyAETACRsAIGIGIEzBBGfEOYI9vBIxA/ASofqVili1FwjAt46mYAAAEBklEQVT2vGODaafS1u89wr6HMJ7L2jQCRiD9CZigTP/P2J7QCBiBBBFgr0f2Re3evXuCWmzbDKdzvH//Xpy9AE1QhoLZGjUCRiAEAiYoQ4BqTRoBI2AEwiJgHsqwyFq7RsAIBCFggjIIPbvWCBiBpCTABvBslM/Z75yyxFm92dnZsn37dj0j/uPHj3oy0+7du2X69OmxZ7h27Zoe6dnU1KQnzLB5PkefOiFor5i7dOmSlJeXS2Njo/To0UMWLVokFRUVsTO1uY+jR4/KqVOn9NSaPn366Gb8vIefeUPe3Ceb+Tc0NOjpWBx0sG/fvlDD7kn5gdpNGQEjkPQETFAm/UdkN2gEjMC/JYCQ69evn1y5ckVaW1v1POuSkhJ59uyZCjKOJ7xw4YJs2bJFz4OfNWuWPH78WMaPHy8bNmxQEcnPHO/JyTN+gvLcuXNSWFio+ZRFRUXy4sULfT3HQdbX1+stcx89e/bUU334PQKXoyNv374tubm5vz2WV1DOmDFDWlpaVJRmZWWpwL1586bs3LkztDzOf8vaXm8EjIAR0PHu169fvwyFETACRiCdCCDkEIYc0Yi9fPlSRowYoUdV5uTkxB6VYz2bm5sFIcf/OVP+zp07sb8fPHhQPYR+gnLixIkyePBgOX36dOz1iFTOm8fDyZGQ3Me6detUxDqGl7KsrMzXS+kWlM759devX5epU6fq5e/evZOhQ4eqEA6rMCid+oE9ixEwAn+PgAnKv8fa3skIGIG/RAAhd/jwYfUYYnV1depF5Axjt/348UN69eqlQo3zr6dNmxYTobwOL+W4ceN8BWVmZqael71+/fpYk4Soaa+mpkaKi4tVUFZWVkppaWnsNYTNly9frl5Gr7kF5YMHD2TBggUaniec7tjIkSP1WUxQ/qXOZG9jBIxAXARMUMaFyV5kBIxAKhFAyJ04cUJzDbHa2lpZuHChPHr0SEPHbsvIyND8SoRjXl5eG29ie4KSau+9e/eqB9IxKrT79u2rIe+CggIVlO774HXtFdX4CUpC7ohUx8aMGSPz5883QZlKHdLu1QhEgIAJygh8yPaIRiBqBLxCjtxJinAuXryo+ZKOUbjTqVMnLaxBfD5//lzu3r0b+3tVVZWsXr3a10NJTuSQIUPahLwpnpk3b16bkPefCsqfP3/K8OHD29zzhw8fZODAgRoyNw9l1Hq1Pa8RSG4CJiiT+/OxuzMCRuAPCPh5BvPz8zWEfejQIRWXZ86ckc2bN8uxY8c0BE2l9tixY2XTpk2yYsUKFYWrVq0S9p588+aN5i66vYuE0Qlr79q1S/8l/5IQO22QS4kF8VDyXoS2yfusrq6W/v37a94lhUbbtm0zQfkH/cIuMQJGIDwC/wAQcInAK+WMnAAAAABJRU5ErkJggg==\" width=\"599.9999869953505\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "pb_num_dict = {_k:len(_v.kept_probes) for _k,_v in filtered_pb_dict.items()}\n",
    "\n",
    "pb_nums = np.array(list(pb_num_dict.values()))\n",
    "plt.figure(figsize=(6,3), dpi=100)\n",
    "plt.plot(pb_nums)\n",
    "#plt.plot(np.ones(len(pb_nums))*20)\n",
    "plt.ylabel(f\"num of probes\", labelpad=1)\n",
    "plt.xlabel(f\"region id\", labelpad=1)\n",
    "plt.ylim([0, num_probes_kept*1.1])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "id": "d87753a7",
   "metadata": {},
   "outputs": [],
   "source": [
    "chr_2_region_savefile = os.path.join(library_folder, 'chr_2_final_region_num.pkl')\n",
    "if not os.path.exists(chr_2_region_savefile) or overwrite:\n",
    "    filtered_reg_names = {}\n",
    "    for _reg, _pbde in filtered_pb_dict.items():\n",
    "        if len(_pbde.kept_probes) > 0:\n",
    "            filtered_reg_names[_reg] = list(_pbde.kept_probes.values())[0]['reg_name']\n",
    "\n",
    "    chr_2_region_num = {}\n",
    "    for _reg, _info in filtered_reg_names.items():\n",
    "        _chr = _info.split('_')[0].split(':')[0]\n",
    "        if _chr not in chr_2_region_num:\n",
    "            chr_2_region_num[_chr] = 1\n",
    "        else:\n",
    "            chr_2_region_num[_chr] += 1\n",
    "    pickle.dump(chr_2_region_num, open(chr_2_region_savefile, 'wb'))\n",
    "else:\n",
    "    chr_2_region_num = pickle.dump(open(chr_2_region_savefile, 'rb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "id": "f9b134b0",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'1': 84,\n",
       " '2': 90,\n",
       " '3': 77,\n",
       " '4': 74,\n",
       " '5': 69,\n",
       " '6': 64,\n",
       " '7': 59,\n",
       " '8': 55,\n",
       " '9': 53,\n",
       " '10': 65,\n",
       " '11': 61,\n",
       " '12': 62,\n",
       " '13': 48,\n",
       " '14': 56,\n",
       " '15': 49,\n",
       " '16': 48,\n",
       " '17': 46,\n",
       " '18': 47,\n",
       " '19': 53,\n",
       " '20': 60,\n",
       " '21': 30,\n",
       " '22': 29,\n",
       " 'X': 59,\n",
       " 'Y': 14}"
      ]
     },
     "execution_count": 94,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "chr_2_region_num"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 95,
   "id": "58d8ef19",
   "metadata": {},
   "outputs": [],
   "source": [
    "chr_2_region_names = {}\n",
    "for _reg, _info in filtered_reg_names.items():\n",
    "    _chr = _info.split('_')[0].split(':')[0]\n",
    "    if _chr not in chr_2_region_names:\n",
    "        chr_2_region_names[_chr] = [_reg]\n",
    "    else:\n",
    "        chr_2_region_names[_chr].append(_reg)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 96,
   "id": "5469fa71",
   "metadata": {},
   "outputs": [],
   "source": [
    "pickle.dump(chr_2_region_names, open(os.path.join(library_folder, 'chr_2_region_names.pkl'), 'wb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "id": "42c9910a",
   "metadata": {},
   "outputs": [],
   "source": [
    "# clear memory\n",
    "del(_pbde)\n",
    "del(_pb_class)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2c3d9549",
   "metadata": {},
   "source": [
    "<a id='3'></a>\n",
    "# 3. Load and asssemble probes"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d373567a",
   "metadata": {},
   "source": [
    "<a id='3.1'></a>\n",
    "## 3.1 design readout mapping dict (region_to_readout)\n",
    "\n",
    "load directly from http://localhost:8889/notebooks/Chromatin_Analysis_Scripts/Library_design/CTP-10_Aire/CTP-10_Aire_introns_DNA-MERFISH.ipynb#3.3-assemble-probes"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a4dd2783",
   "metadata": {},
   "source": [
    "<a id='3.2'></a>\n",
    "## 3.2 loading components"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8e899029",
   "metadata": {},
   "source": [
    "### primers\n",
    "pick primer set 2,9"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "id": "a69492f8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Picked primer: ID: W1A03_primer_2\n",
      "Name: W1A03_primer_2\n",
      "Description: W1A03_primer_2\n",
      "Number of features: 0\n",
      "Seq('CCCGCAATGGCTGACAACCG')\n",
      "- Picked primer: ID: W1A10_primer_9\n",
      "Name: W1A10_primer_9\n",
      "Description: W1A10_primer_9\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGATTGCCGCATGGTTTCCG')\n"
     ]
    }
   ],
   "source": [
    "from ImageAnalysis3 import library_tools\n",
    "primer_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Primers'\n",
    "primers = library_tools.assemble.load_primers([2,9], _primer_folder=primer_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3dfa7962",
   "metadata": {},
   "source": [
    "### readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "id": "136b756f",
   "metadata": {},
   "outputs": [],
   "source": [
    "# summarize total readout usage\n",
    "region_2_readout_dict = pickle.load(open(os.path.join(library_folder, 'region_2_readout.pkl'), 'rb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 99,
   "id": "847d66ef",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{100: ['c28', 'c43', 'c97'],\n",
       " 101: ['c5', 'c12', 'c60'],\n",
       " 102: ['c15', 'c53', 'c67'],\n",
       " 103: ['c3', 'c57', 'c80'],\n",
       " 104: ['c4', 'c27', 'c87'],\n",
       " 105: ['c37', 'c47', 'c69'],\n",
       " 106: ['c1', 'c68', 'c85'],\n",
       " 107: ['c21', 'c33', 'c91'],\n",
       " 108: ['c9', 'c70', 'c94'],\n",
       " 109: ['c45', 'c56', 'c65'],\n",
       " 110: ['c13', 'c76', 'c84'],\n",
       " 111: ['c22', 'c25', 'c78'],\n",
       " 112: ['c11', 'c26', 'c49'],\n",
       " 114: ['c46', 'c62', 'c88'],\n",
       " 115: ['c6', 'c24', 'c30'],\n",
       " 116: ['c7', 'c29', 'c32'],\n",
       " 117: ['c16', 'c90', 'c95'],\n",
       " 118: ['c34', 'c44', 'c93'],\n",
       " 119: ['c19', 'c83', 'c85'],\n",
       " 120: ['c8', 'c18', 'c58'],\n",
       " 121: ['c35', 'c39', 'c72'],\n",
       " 122: ['c45', 'c57', 'c81'],\n",
       " 123: ['c41', 'c71', 'c97'],\n",
       " 124: ['c17', 'c43', 'c96'],\n",
       " 125: ['c20', 'c28', 'c36'],\n",
       " 126: ['c23', 'c31', 'c55'],\n",
       " 127: ['c53', 'c73', 'c82'],\n",
       " 128: ['c62', 'c64', 'c79'],\n",
       " 129: ['c74', 'c84', 'c95'],\n",
       " 130: ['c60', 'c61', 'c89'],\n",
       " 131: ['c0', 'c24', 'c25'],\n",
       " 132: ['c5', 'c10', 'c54'],\n",
       " 133: ['c4', 'c40', 'c70'],\n",
       " 134: ['c48', 'c66', 'c85'],\n",
       " 137: ['c16', 'c63', 'c76'],\n",
       " 138: ['c78', 'c86', 'c94'],\n",
       " 139: ['c15', 'c36', 'c56'],\n",
       " 140: ['c13', 'c81', 'c91'],\n",
       " 141: ['c8', 'c35', 'c68'],\n",
       " 143: ['c75', 'c77', 'c83'],\n",
       " 144: ['c2', 'c33', 'c71'],\n",
       " 145: ['c11', 'c17', 'c29'],\n",
       " 146: ['c67', 'c92', 'c98'],\n",
       " 147: ['c7', 'c47', 'c72'],\n",
       " 148: ['c10', 'c50', 'c74'],\n",
       " 149: ['c9', 'c41', 'c79'],\n",
       " 150: ['c49', 'c58', 'c65'],\n",
       " 152: ['c0', 'c51', 'c59'],\n",
       " 153: ['c44', 'c69', 'c80'],\n",
       " 154: ['c22', 'c26', 'c40'],\n",
       " 155: ['c14', 'c18', 'c27'],\n",
       " 156: ['c32', 'c86', 'c88'],\n",
       " 157: ['c3', 'c91', 'c93'],\n",
       " 158: ['c21', 'c30', 'c70'],\n",
       " 159: ['c12', 'c34', 'c42'],\n",
       " 160: ['c31', 'c37', 'c78'],\n",
       " 161: ['c15', 'c23', 'c50'],\n",
       " 162: ['c19', 'c24', 'c84'],\n",
       " 163: ['c38', 'c42', 'c98'],\n",
       " 164: ['c52', 'c55', 'c73'],\n",
       " 165: ['c28', 'c29', 'c63'],\n",
       " 166: ['c2', 'c43', 'c59'],\n",
       " 167: ['c6', 'c67', 'c87'],\n",
       " 168: ['c17', 'c64', 'c90'],\n",
       " 169: ['c14', 'c48', 'c68'],\n",
       " 170: ['c26', 'c39', 'c92'],\n",
       " 171: ['c3', 'c86', 'c89'],\n",
       " 172: ['c4', 'c54', 'c81'],\n",
       " 173: ['c61', 'c74', 'c79'],\n",
       " 174: ['c2', 'c57', 'c82'],\n",
       " 175: ['c38', 'c53', 'c72'],\n",
       " 176: ['c1', 'c31', 'c52'],\n",
       " 177: ['c20', 'c21', 'c46'],\n",
       " 178: ['c51', 'c65', 'c76'],\n",
       " 179: ['c5', 'c66', 'c88'],\n",
       " 180: ['c56', 'c96', 'c97'],\n",
       " 181: ['c9', 'c47', 'c75'],\n",
       " 182: ['c22', 'c71', 'c77'],\n",
       " 183: ['c37', 'c58', 'c63'],\n",
       " 184: ['c40', 'c44', 'c60'],\n",
       " 185: ['c8', 'c80', 'c87'],\n",
       " 186: ['c19', 'c64', 'c93'],\n",
       " 187: ['c12', 'c62', 'c95'],\n",
       " 188: ['c6', 'c45', 'c83'],\n",
       " 189: ['c7', 'c33', 'c35'],\n",
       " 190: ['c11', 'c36', 'c94'],\n",
       " 191: ['c0', 'c30', 'c69'],\n",
       " 192: ['c25', 'c49', 'c90'],\n",
       " 193: ['c16', 'c27', 'c32'],\n",
       " 194: ['c1', 'c13', 'c46'],\n",
       " 0: ['c13', 'c47', 'c55'],\n",
       " 1: ['c10', 'c46', 'c72'],\n",
       " 2: ['c12', 'c58', 'c81'],\n",
       " 3: ['c1', 'c65', 'c78'],\n",
       " 4: ['c7', 'c73', 'c89'],\n",
       " 5: ['c14', 'c52', 'c54'],\n",
       " 6: ['c42', 'c50', 'c66'],\n",
       " 7: ['c23', 'c34', 'c38'],\n",
       " 8: ['c26', 'c28', 'c60'],\n",
       " 9: ['c45', 'c85', 'c86'],\n",
       " 10: ['c22', 'c31', 'c74'],\n",
       " 11: ['c39', 'c77', 'c90'],\n",
       " 12: ['c18', 'c79', 'c96'],\n",
       " 13: ['c25', 'c47', 'c93'],\n",
       " 14: ['c24', 'c46', 'c53'],\n",
       " 15: ['c8', 'c88', 'c97'],\n",
       " 16: ['c15', 'c55', 'c61'],\n",
       " 17: ['c14', 'c89', 'c98'],\n",
       " 18: ['c16', 'c75', 'c92'],\n",
       " 19: ['c17', 'c26', 'c65'],\n",
       " 20: ['c49', 'c83', 'c94'],\n",
       " 21: ['c11', 'c19', 'c40'],\n",
       " 22: ['c41', 'c42', 'c48'],\n",
       " 23: ['c21', 'c62', 'c68'],\n",
       " 24: ['c34', 'c35', 'c57'],\n",
       " 25: ['c51', 'c58', 'c73'],\n",
       " 26: ['c66', 'c67', 'c71'],\n",
       " 27: ['c3', 'c72', 'c87'],\n",
       " 28: ['c0', 'c27', 'c84'],\n",
       " 29: ['c69', 'c81', 'c90'],\n",
       " 30: ['c5', 'c26', 'c70'],\n",
       " 31: ['c7', 'c25', 'c31'],\n",
       " 32: ['c20', 'c77', 'c82'],\n",
       " 33: ['c19', 'c91', 'c98'],\n",
       " 34: ['c10', 'c23', 'c89'],\n",
       " 35: ['c29', 'c44', 'c56'],\n",
       " 36: ['c2', 'c4', 'c32'],\n",
       " 37: ['c6', 'c9', 'c64'],\n",
       " 38: ['c18', 'c41', 'c59'],\n",
       " 39: ['c12', 'c13', 'c28'],\n",
       " 40: ['c21', 'c43', 'c57'],\n",
       " 42: ['c22', 'c36', 'c48'],\n",
       " 43: ['c24', 'c78', 'c83'],\n",
       " 44: ['c3', 'c30', 'c92'],\n",
       " 45: ['c77', 'c86', 'c96'],\n",
       " 46: ['c38', 'c54', 'c97'],\n",
       " 47: ['c0', 'c52', 'c75'],\n",
       " 57: ['c29', 'c76', 'c94'],\n",
       " 60: ['c9', 'c33', 'c37'],\n",
       " 62: ['c25', 'c51', 'c55'],\n",
       " 63: ['c17', 'c46', 'c61'],\n",
       " 64: ['c43', 'c63', 'c80'],\n",
       " 65: ['c2', 'c44', 'c45'],\n",
       " 66: ['c6', 'c49', 'c95'],\n",
       " 67: ['c12', 'c40', 'c85'],\n",
       " 68: ['c37', 'c53', 'c76'],\n",
       " 69: ['c18', 'c66', 'c82'],\n",
       " 70: ['c5', 'c27', 'c33'],\n",
       " 71: ['c4', 'c74', 'c86'],\n",
       " 72: ['c36', 'c81', 'c88'],\n",
       " 73: ['c20', 'c30', 'c32'],\n",
       " 74: ['c11', 'c79', 'c98'],\n",
       " 75: ['c1', 'c16', 'c87'],\n",
       " 76: ['c50', 'c59', 'c92'],\n",
       " 77: ['c24', 'c35', 'c71'],\n",
       " 78: ['c0', 'c80', 'c95'],\n",
       " 79: ['c23', 'c70', 'c84'],\n",
       " 80: ['c10', 'c39', 'c64'],\n",
       " 81: ['c31', 'c56', 'c83'],\n",
       " 83: ['c60', 'c62', 'c85'],\n",
       " 84: ['c41', 'c52', 'c68'],\n",
       " 85: ['c15', 'c42', 'c82'],\n",
       " 86: ['c63', 'c72', 'c78'],\n",
       " 87: ['c47', 'c65', 'c67'],\n",
       " 88: ['c7', 'c91', 'c97'],\n",
       " 89: ['c13', 'c50', 'c93'],\n",
       " 90: ['c8', 'c36', 'c59'],\n",
       " 91: ['c28', 'c69', 'c88'],\n",
       " 92: ['c48', 'c51', 'c96'],\n",
       " 93: ['c17', 'c40', 'c58'],\n",
       " 94: ['c14', 'c38', 'c61'],\n",
       " 95: ['c54', 'c73', 'c75'],\n",
       " 96: ['c20', 'c34', 'c39'],\n",
       " 98: ['c1', 'c22', 'c62'],\n",
       " 195: ['c11', 'c77', 'c93'],\n",
       " 196: ['c8', 'c44', 'c65'],\n",
       " 197: ['c50', 'c87', 'c89'],\n",
       " 198: ['c21', 'c35', 'c41'],\n",
       " 199: ['c52', 'c60', 'c69'],\n",
       " 200: ['c2', 'c54', 'c55'],\n",
       " 201: ['c47', 'c62', 'c91'],\n",
       " 202: ['c3', 'c25', 'c40'],\n",
       " 203: ['c28', 'c39', 'c79'],\n",
       " 204: ['c20', 'c33', 'c59'],\n",
       " 205: ['c37', 'c70', 'c96'],\n",
       " 206: ['c31', 'c36', 'c84'],\n",
       " 207: ['c51', 'c64', 'c75'],\n",
       " 208: ['c27', 'c34', 'c63'],\n",
       " 209: ['c10', 'c44', 'c73'],\n",
       " 210: ['c9', 'c74', 'c92'],\n",
       " 211: ['c32', 'c93', 'c98'],\n",
       " 212: ['c19', 'c21', 'c94'],\n",
       " 213: ['c17', 'c72', 'c83'],\n",
       " 214: ['c1', 'c24', 'c26'],\n",
       " 215: ['c46', 'c76', 'c97'],\n",
       " 216: ['c16', 'c78', 'c91'],\n",
       " 217: ['c30', 'c55', 'c77'],\n",
       " 218: ['c29', 'c45', 'c49'],\n",
       " 219: ['c38', 'c57', 'c68'],\n",
       " 220: ['c6', 'c23', 'c42'],\n",
       " 221: ['c48', 'c67', 'c82'],\n",
       " 222: ['c53', 'c61', 'c85'],\n",
       " 223: ['c5', 'c64', 'c73'],\n",
       " 224: ['c11', 'c15', 'c27'],\n",
       " 225: ['c43', 'c50', 'c94'],\n",
       " 226: ['c18', 'c65', 'c81'],\n",
       " 227: ['c10', 'c56', 'c69'],\n",
       " 228: ['c22', 'c32', 'c72'],\n",
       " 229: ['c2', 'c75', 'c80'],\n",
       " 230: ['c34', 'c66', 'c95'],\n",
       " 233: ['c3', 'c9', 'c19'],\n",
       " 234: ['c14', 'c90', 'c92'],\n",
       " 235: ['c12', 'c87', 'c91'],\n",
       " 236: ['c46', 'c58', 'c78'],\n",
       " 237: ['c7', 'c13', 'c83'],\n",
       " 238: ['c30', 'c35', 'c84'],\n",
       " 239: ['c60', 'c64', 'c71'],\n",
       " 240: ['c0', 'c86', 'c97'],\n",
       " 241: ['c24', 'c28', 'c40'],\n",
       " 242: ['c16', 'c21', 'c77'],\n",
       " 243: ['c8', 'c20', 'c56'],\n",
       " 244: ['c41', 'c43', 'c69'],\n",
       " 245: ['c25', 'c52', 'c88'],\n",
       " 246: ['c4', 'c5', 'c15'],\n",
       " 247: ['c37', 'c42', 'c89'],\n",
       " 248: ['c45', 'c67', 'c79'],\n",
       " 249: ['c36', 'c58', 'c85'],\n",
       " 250: ['c27', 'c54', 'c95'],\n",
       " 251: ['c38', 'c44', 'c70'],\n",
       " 252: ['c23', 'c39', 'c48'],\n",
       " 253: ['c59', 'c76', 'c96'],\n",
       " 254: ['c49', 'c60', 'c66'],\n",
       " 255: ['c6', 'c71', 'c80'],\n",
       " 256: ['c29', 'c68', 'c90'],\n",
       " 257: ['c3', 'c18', 'c33'],\n",
       " 258: ['c20', 'c53', 'c57'],\n",
       " 259: ['c26', 'c41', 'c61'],\n",
       " 260: ['c1', 'c47', 'c94'],\n",
       " 261: ['c10', 'c51', 'c62'],\n",
       " 262: ['c4', 'c63', 'c93'],\n",
       " 263: ['c12', 'c43', 'c98'],\n",
       " 264: ['c7', 'c22', 'c30'],\n",
       " 265: ['c0', 'c2', 'c31'],\n",
       " 266: ['c16', 'c81', 'c86'],\n",
       " 267: ['c9', 'c13', 'c88'],\n",
       " 268: ['c8', 'c82', 'c95'],\n",
       " 269: ['c11', 'c14', 'c73'],\n",
       " 270: ['c17', 'c19', 'c71'],\n",
       " 271: ['c54', 'c56', 'c74'],\n",
       " 272: ['c50', 'c84', 'c90'],\n",
       " 273: ['c39', 'c87', 'c96'],\n",
       " 274: ['c5', 'c6', 'c13'],\n",
       " 275: ['c48', 'c79', 'c80'],\n",
       " 276: ['c59', 'c61', 'c75'],\n",
       " 277: ['c49', 'c55', 'c82'],\n",
       " 278: ['c58', 'c72', 'c90'],\n",
       " 279: ['c2', 'c10', 'c17'],\n",
       " 280: ['c16', 'c51', 'c69'],\n",
       " 281: ['c12', 'c14', 'c31'],\n",
       " 282: ['c38', 'c81', 'c83'],\n",
       " 283: ['c43', 'c52', 'c84'],\n",
       " 284: ['c21', 'c26', 'c63'],\n",
       " 285: ['c23', 'c27', 'c85'],\n",
       " 286: ['c7', 'c39', 'c62'],\n",
       " 287: ['c86', 'c87', 'c93'],\n",
       " 288: ['c8', 'c19', 'c46'],\n",
       " 289: ['c42', 'c59', 'c68'],\n",
       " 290: ['c49', 'c74', 'c80'],\n",
       " 291: ['c5', 'c55', 'c67'],\n",
       " 292: ['c30', 'c47', 'c73'],\n",
       " 293: ['c22', 'c44', 'c64'],\n",
       " 295: ['c6', 'c32', 'c34'],\n",
       " 296: ['c25', 'c36', 'c96'],\n",
       " 297: ['c3', 'c56', 'c94'],\n",
       " 298: ['c9', 'c20', 'c95'],\n",
       " 299: ['c40', 'c65', 'c77'],\n",
       " 300: ['c24', 'c54', 'c60'],\n",
       " 302: ['c28', 'c50', 'c78'],\n",
       " 303: ['c0', 'c72', 'c85'],\n",
       " 304: ['c48', 'c63', 'c92'],\n",
       " 305: ['c18', 'c38', 'c67'],\n",
       " 306: ['c49', 'c68', 'c71'],\n",
       " 307: ['c4', 'c57', 'c97'],\n",
       " 308: ['c15', 'c41', 'c66'],\n",
       " 309: ['c53', 'c89', 'c91'],\n",
       " 310: ['c11', 'c33', 'c45'],\n",
       " 311: ['c14', 'c42', 'c76'],\n",
       " 312: ['c35', 'c70', 'c98'],\n",
       " 313: ['c32', 'c37', 'c82'],\n",
       " 314: ['c34', 'c55', 'c74'],\n",
       " 315: ['c1', 'c29', 'c83'],\n",
       " 316: ['c39', 'c84', 'c88'],\n",
       " 317: ['c41', 'c78', 'c93'],\n",
       " 318: ['c46', 'c92', 'c96'],\n",
       " 319: ['c3', 'c26', 'c90'],\n",
       " 320: ['c16', 'c28', 'c94'],\n",
       " 321: ['c0', 'c12', 'c73'],\n",
       " 322: ['c7', 'c60', 'c65'],\n",
       " 323: ['c9', 'c24', 'c77'],\n",
       " 324: ['c13', 'c21', 'c58'],\n",
       " 325: ['c51', 'c57', 'c67'],\n",
       " 326: ['c23', 'c37', 'c61'],\n",
       " 327: ['c52', 'c59', 'c66'],\n",
       " 328: ['c30', 'c31', 'c62'],\n",
       " 329: ['c1', 'c2', 'c27'],\n",
       " 330: ['c47', 'c50', 'c86'],\n",
       " 331: ['c10', 'c71', 'c91'],\n",
       " 332: ['c17', 'c36', 'c87'],\n",
       " 333: ['c8', 'c25', 'c64'],\n",
       " 334: ['c15', 'c54', 'c69'],\n",
       " 335: ['c75', 'c79', 'c97'],\n",
       " 336: ['c4', 'c33', 'c76'],\n",
       " 337: ['c29', 'c53', 'c92'],\n",
       " 338: ['c70', 'c80', 'c89'],\n",
       " 339: ['c18', 'c23', 'c40'],\n",
       " 340: ['c2', 'c19', 'c22'],\n",
       " 341: ['c11', 'c52', 'c56'],\n",
       " 342: ['c51', 'c81', 'c95'],\n",
       " 343: ['c44', 'c72', 'c88'],\n",
       " 344: ['c37', 'c68', 'c98'],\n",
       " 345: ['c5', 'c35', 'c74'],\n",
       " 346: ['c14', 'c32', 'c45'],\n",
       " 347: ['c34', 'c43', 'c48'],\n",
       " 348: ['c20', 'c42', 'c61'],\n",
       " 349: ['c6', 'c38', 'c63'],\n",
       " 350: ['c13', 'c78', 'c79'],\n",
       " 351: ['c15', 'c76', 'c85'],\n",
       " 352: ['c10', 'c32', 'c36'],\n",
       " 353: ['c41', 'c47', 'c60'],\n",
       " 354: ['c26', 'c51', 'c88'],\n",
       " 355: ['c12', 'c68', 'c96'],\n",
       " 356: ['c4', 'c18', 'c21'],\n",
       " 357: ['c30', 'c49', 'c93'],\n",
       " 358: ['c6', 'c44', 'c59'],\n",
       " 359: ['c37', 'c67', 'c97'],\n",
       " 360: ['c3', 'c28', 'c65'],\n",
       " 361: ['c27', 'c42', 'c69'],\n",
       " 362: ['c9', 'c45', 'c87'],\n",
       " 363: ['c17', 'c57', 'c66'],\n",
       " 364: ['c34', 'c72', 'c82'],\n",
       " 365: ['c16', 'c20', 'c35'],\n",
       " 366: ['c33', 'c50', 'c55'],\n",
       " 367: ['c2', 'c7', 'c86'],\n",
       " 368: ['c43', 'c46', 'c73'],\n",
       " 370: ['c12', 'c19', 'c61'],\n",
       " 371: ['c24', 'c63', 'c91'],\n",
       " 372: ['c13', 'c48', 'c62'],\n",
       " 373: ['c52', 'c77', 'c80'],\n",
       " 374: ['c25', 'c37', 'c84'],\n",
       " 375: ['c4', 'c29', 'c89'],\n",
       " 376: ['c21', 'c23', 'c32'],\n",
       " 377: ['c27', 'c71', 'c94'],\n",
       " 379: ['c6', 'c17', 'c54'],\n",
       " 380: ['c56', 'c92', 'c93'],\n",
       " 381: ['c14', 'c87', 'c95'],\n",
       " 382: ['c15', 'c22', 'c98'],\n",
       " 383: ['c59', 'c83', 'c97'],\n",
       " 384: ['c0', 'c39', 'c40'],\n",
       " 385: ['c5', 'c42', 'c65'],\n",
       " 386: ['c1', 'c49', 'c61'],\n",
       " 387: ['c9', 'c78', 'c90'],\n",
       " 388: ['c8', 'c28', 'c67'],\n",
       " 389: ['c3', 'c38', 'c64'],\n",
       " 390: ['c11', 'c31', 'c69'],\n",
       " 391: ['c44', 'c53', 'c75'],\n",
       " 392: ['c30', 'c33', 'c57'],\n",
       " 393: ['c35', 'c81', 'c82'],\n",
       " 394: ['c29', 'c70', 'c79'],\n",
       " 395: ['c19', 'c45', 'c66'],\n",
       " 396: ['c74', 'c76', 'c89'],\n",
       " 397: ['c47', 'c85', 'c98'],\n",
       " 398: ['c15', 'c18', 'c72'],\n",
       " 399: ['c14', 'c23', 'c58'],\n",
       " 400: ['c39', 'c52', 'c95'],\n",
       " 401: ['c26', 'c34', 'c86'],\n",
       " 402: ['c60', 'c68', 'c73'],\n",
       " 403: ['c50', 'c63', 'c83'],\n",
       " 404: ['c8', 'c51', 'c91'],\n",
       " 405: ['c5', 'c48', 'c94'],\n",
       " 406: ['c54', 'c80', 'c96'],\n",
       " 407: ['c36', 'c43', 'c71'],\n",
       " 408: ['c7', 'c16', 'c88'],\n",
       " 409: ['c0', 'c1', 'c56'],\n",
       " 410: ['c20', 'c62', 'c92'],\n",
       " 411: ['c11', 'c55', 'c64'],\n",
       " 412: ['c2', 'c38', 'c74'],\n",
       " 413: ['c10', 'c53', 'c79'],\n",
       " 414: ['c46', 'c75', 'c98'],\n",
       " 415: ['c18', 'c70', 'c90'],\n",
       " 416: ['c32', 'c77', 'c81'],\n",
       " 417: ['c40', 'c51', 'c84'],\n",
       " 418: ['c12', 'c25', 'c41'],\n",
       " 419: ['c31', 'c47', 'c58'],\n",
       " 421: ['c22', 'c24', 'c29'],\n",
       " 680: ['c53', 'c64', 'c70'],\n",
       " 681: ['c22', 'c57', 'c89'],\n",
       " 682: ['c71', 'c73', 'c74'],\n",
       " 683: ['c1', 'c80', 'c86'],\n",
       " 684: ['c7', 'c43', 'c79'],\n",
       " 685: ['c13', 'c36', 'c41'],\n",
       " 686: ['c8', 'c61', 'c94'],\n",
       " 687: ['c21', 'c42', 'c52'],\n",
       " 688: ['c40', 'c54', 'c62'],\n",
       " 689: ['c3', 'c58', 'c84'],\n",
       " 690: ['c14', 'c16', 'c29'],\n",
       " 691: ['c39', 'c53', 'c78'],\n",
       " 692: ['c23', 'c25', 'c98'],\n",
       " 693: ['c4', 'c35', 'c46'],\n",
       " 694: ['c26', 'c45', 'c75'],\n",
       " 695: ['c28', 'c51', 'c66'],\n",
       " 696: ['c12', 'c67', 'c69'],\n",
       " 697: ['c32', 'c33', 'c96'],\n",
       " 698: ['c9', 'c68', 'c89'],\n",
       " 699: ['c18', 'c49', 'c76'],\n",
       " 701: ['c0', 'c21', 'c48'],\n",
       " 702: ['c11', 'c85', 'c87'],\n",
       " 703: ['c17', 'c20', 'c91'],\n",
       " 704: ['c19', 'c34', 'c54'],\n",
       " 705: ['c56', 'c59', 'c78'],\n",
       " 706: ['c50', 'c88', 'c95'],\n",
       " 707: ['c16', 'c64', 'c84'],\n",
       " 708: ['c15', 'c46', 'c63'],\n",
       " 709: ['c14', 'c25', 'c79'],\n",
       " 710: ['c24', 'c80', 'c92'],\n",
       " 711: ['c38', 'c52', 'c65'],\n",
       " 712: ['c30', 'c41', 'c94'],\n",
       " 713: ['c44', 'c77', 'c98'],\n",
       " 714: ['c10', 'c26', 'c31'],\n",
       " 715: ['c1', 'c5', 'c7'],\n",
       " 716: ['c3', 'c55', 'c68'],\n",
       " 717: ['c28', 'c61', 'c73'],\n",
       " 718: ['c13', 'c23', 'c90'],\n",
       " 719: ['c8', 'c43', 'c74'],\n",
       " 720: ['c27', 'c37', 'c60'],\n",
       " 721: ['c33', 'c83', 'c86'],\n",
       " 722: ['c45', 'c62', 'c76'],\n",
       " 723: ['c4', 'c58', 'c75'],\n",
       " 724: ['c69', 'c70', 'c82'],\n",
       " 725: ['c9', 'c35', 'c66'],\n",
       " 726: ['c47', 'c81', 'c97'],\n",
       " 727: ['c2', 'c39', 'c42'],\n",
       " 728: ['c71', 'c81', 'c93'],\n",
       " 729: ['c6', 'c29', 'c72'],\n",
       " 730: ['c0', 'c11', 'c57'],\n",
       " 731: ['c44', 'c49', 'c92'],\n",
       " 732: ['c37', 'c48', 'c56'],\n",
       " 733: ['c17', 'c22', 'c67'],\n",
       " 734: ['c10', 'c20', 'c90'],\n",
       " 735: ['c30', 'c60', 'c91'],\n",
       " 736: ['c15', 'c24', 'c88'],\n",
       " 737: ['c19', 'c63', 'c95'],\n",
       " 738: ['c31', 'c38', 'c50'],\n",
       " 739: ['c34', 'c65', 'c96'],\n",
       " 740: ['c18', 'c55', 'c83'],\n",
       " 741: ['c12', 'c27', 'c77'],\n",
       " 742: ['c51', 'c87', 'c97'],\n",
       " 743: ['c32', 'c59', 'c85'],\n",
       " 744: ['c36', 'c40', 'c53'],\n",
       " 745: ['c5', 'c82', 'c89'],\n",
       " 422: ['c23', 'c35', 'c64'],\n",
       " 423: ['c53', 'c86', 'c95'],\n",
       " 424: ['c7', 'c34', 'c77'],\n",
       " 425: ['c15', 'c29', 'c78'],\n",
       " 426: ['c13', 'c24', 'c52'],\n",
       " 427: ['c55', 'c89', 'c97'],\n",
       " 428: ['c3', 'c27', 'c59'],\n",
       " 429: ['c18', 'c20', 'c37'],\n",
       " 430: ['c6', 'c36', 'c66'],\n",
       " 431: ['c41', 'c45', 'c82'],\n",
       " 432: ['c4', 'c47', 'c92'],\n",
       " 435: ['c32', 'c65', 'c68'],\n",
       " 436: ['c9', 'c31', 'c80'],\n",
       " 437: ['c1', 'c23', 'c81'],\n",
       " 438: ['c62', 'c94', 'c98'],\n",
       " 439: ['c8', 'c30', 'c83'],\n",
       " 440: ['c21', 'c39', 'c61'],\n",
       " 441: ['c40', 'c59', 'c90'],\n",
       " 442: ['c0', 'c10', 'c49'],\n",
       " 443: ['c71', 'c78', 'c87'],\n",
       " 444: ['c16', 'c46', 'c48'],\n",
       " 446: ['c7', 'c26', 'c67'],\n",
       " 447: ['c31', 'c70', 'c93'],\n",
       " 448: ['c11', 'c43', 'c58'],\n",
       " 449: ['c38', 'c69', 'c96'],\n",
       " 450: ['c3', 'c5', 'c44'],\n",
       " 451: ['c18', 'c53', 'c84'],\n",
       " 452: ['c9', 'c22', 'c28'],\n",
       " 453: ['c14', 'c34', 'c85'],\n",
       " 454: ['c13', 'c25', 'c54'],\n",
       " 455: ['c20', 'c52', 'c63'],\n",
       " 456: ['c2', 'c15', 'c95'],\n",
       " 457: ['c41', 'c55', 'c57'],\n",
       " 458: ['c4', 'c66', 'c73'],\n",
       " 459: ['c36', 'c70', 'c76'],\n",
       " 460: ['c35', 'c45', 'c60'],\n",
       " 461: ['c33', 'c82', 'c93'],\n",
       " 462: ['c29', 'c42', 'c51'],\n",
       " 463: ['c12', 'c74', 'c75'],\n",
       " 464: ['c50', 'c56', 'c77'],\n",
       " 465: ['c19', 'c27', 'c88'],\n",
       " 466: ['c17', 'c24', 'c37'],\n",
       " 467: ['c5', 'c79', 'c91'],\n",
       " 468: ['c6', 'c28', 'c75'],\n",
       " 469: ['c42', 'c72', 'c97'],\n",
       " 470: ['c49', 'c64', 'c89'],\n",
       " 471: ['c10', 'c57', 'c76'],\n",
       " 472: ['c33', 'c62', 'c87'],\n",
       " 473: ['c46', 'c74', 'c81'],\n",
       " 474: ['c2', 'c16', 'c67'],\n",
       " 475: ['c54', 'c90', 'c94'],\n",
       " 476: ['c48', 'c58', 'c83'],\n",
       " 477: ['c71', 'c85', 'c92'],\n",
       " 478: ['c11', 'c30', 'c61'],\n",
       " 479: ['c26', 'c80', 'c84'],\n",
       " 480: ['c21', 'c88', 'c98'],\n",
       " 481: ['c56', 'c63', 'c68'],\n",
       " 482: ['c65', 'c72', 'c73'],\n",
       " 483: ['c8', 'c47', 'c96'],\n",
       " 484: ['c0', 'c43', 'c44'],\n",
       " 485: ['c14', 'c39', 'c69'],\n",
       " 486: ['c25', 'c38', 'c60'],\n",
       " 487: ['c1', 'c32', 'c40'],\n",
       " 489: ['c22', 'c35', 'c86'],\n",
       " 813: ['c25', 'c39', 'c65'],\n",
       " 814: ['c59', 'c64', 'c88'],\n",
       " 815: ['c23', 'c36', 'c63'],\n",
       " 816: ['c26', 'c35', 'c42'],\n",
       " 817: ['c40', 'c73', 'c98'],\n",
       " 819: ['c19', 'c38', 'c80'],\n",
       " 820: ['c7', 'c17', 'c93'],\n",
       " 821: ['c14', 'c33', 'c66'],\n",
       " 822: ['c4', 'c45', 'c90'],\n",
       " 823: ['c50', 'c57', 'c85'],\n",
       " 824: ['c16', 'c43', 'c60'],\n",
       " 825: ['c11', 'c70', 'c86'],\n",
       " 826: ['c27', 'c68', 'c92'],\n",
       " 827: ['c44', 'c55', 'c58'],\n",
       " 828: ['c34', 'c62', 'c81'],\n",
       " 829: ['c3', 'c29', 'c41'],\n",
       " 832: ['c10', 'c18', 'c78'],\n",
       " 833: ['c5', 'c31', 'c40'],\n",
       " 834: ['c30', 'c54', 'c72'],\n",
       " 835: ['c21', 'c22', 'c84'],\n",
       " 836: ['c6', 'c43', 'c82'],\n",
       " 837: ['c24', 'c38', 'c47'],\n",
       " 838: ['c42', 'c56', 'c88'],\n",
       " 839: ['c26', 'c64', 'c94'],\n",
       " 840: ['c69', 'c73', 'c83'],\n",
       " 841: ['c7', 'c37', 'c49'],\n",
       " 843: ['c9', 'c15', 'c51'],\n",
       " 844: ['c2', 'c23', 'c93'],\n",
       " 845: ['c8', 'c33', 'c48'],\n",
       " 846: ['c11', 'c18', 'c95'],\n",
       " 847: ['c1', 'c17', 'c45'],\n",
       " 848: ['c19', 'c82', 'c97'],\n",
       " 849: ['c29', 'c54', 'c77'],\n",
       " 850: ['c66', 'c75', 'c90'],\n",
       " 851: ['c57', 'c70', 'c91'],\n",
       " 852: ['c20', 'c76', 'c80'],\n",
       " 853: ['c36', 'c46', 'c79'],\n",
       " 854: ['c13', 'c63', 'c89'],\n",
       " 855: ['c52', 'c61', 'c96'],\n",
       " 856: ['c25', 'c28', 'c87'],\n",
       " 857: ['c14', 'c53', 'c59'],\n",
       " 858: ['c0', 'c65', 'c98'],\n",
       " 859: ['c16', 'c39', 'c74'],\n",
       " 860: ['c35', 'c67', 'c95'],\n",
       " 861: ['c12', 'c32', 'c71'],\n",
       " 862: ['c4', 'c50', 'c79'],\n",
       " 863: ['c6', 'c76', 'c77'],\n",
       " 864: ['c68', 'c75', 'c91'],\n",
       " 865: ['c44', 'c47', 'c83'],\n",
       " 866: ['c48', 'c61', 'c81'],\n",
       " 867: ['c31', 'c41', 'c72'],\n",
       " 868: ['c2', 'c85', 'c96'],\n",
       " 869: ['c51', 'c60', 'c78'],\n",
       " 870: ['c13', 'c56', 'c86'],\n",
       " 871: ['c15', 'c30', 'c34'],\n",
       " 872: ['c22', 'c58', 'c69'],\n",
       " 873: ['c8', 'c62', 'c84'],\n",
       " 874: ['c1', 'c3', 'c10'],\n",
       " 875: ['c9', 'c21', 'c49'],\n",
       " 876: ['c55', 'c87', 'c94'],\n",
       " 877: ['c5', 'c37', 'c92'],\n",
       " 878: ['c24', 'c27', 'c97'],\n",
       " 748: ['c10', 'c55', 'c59'],\n",
       " 749: ['c40', 'c43', 'c72'],\n",
       " 750: ['c37', 'c54', 'c88'],\n",
       " 751: ['c3', 'c21', 'c78'],\n",
       " 752: ['c14', 'c56', 'c84'],\n",
       " 753: ['c15', 'c32', 'c48'],\n",
       " 754: ['c31', 'c65', 'c97'],\n",
       " 755: ['c34', 'c87', 'c92'],\n",
       " 756: ['c26', 'c44', 'c62'],\n",
       " 757: ['c16', 'c22', 'c23'],\n",
       " 758: ['c5', 'c45', 'c77'],\n",
       " 759: ['c20', 'c50', 'c75'],\n",
       " 760: ['c52', 'c71', 'c79'],\n",
       " 761: ['c0', 'c66', 'c91'],\n",
       " 762: ['c12', 'c33', 'c53'],\n",
       " 763: ['c19', 'c47', 'c57'],\n",
       " 764: ['c42', 'c64', 'c85'],\n",
       " 765: ['c9', 'c10', 'c27'],\n",
       " 766: ['c8', 'c54', 'c98'],\n",
       " 767: ['c61', 'c69', 'c86'],\n",
       " 768: ['c39', 'c73', 'c94'],\n",
       " 769: ['c15', 'c26', 'c97'],\n",
       " 770: ['c5', 'c78', 'c84'],\n",
       " 774: ['c31', 'c49', 'c87'],\n",
       " 775: ['c2', 'c13', 'c29'],\n",
       " 776: ['c16', 'c40', 'c55'],\n",
       " 777: ['c33', 'c67', 'c77'],\n",
       " 778: ['c6', 'c41', 'c70'],\n",
       " 779: ['c3', 'c66', 'c79'],\n",
       " 780: ['c28', 'c74', 'c93'],\n",
       " 782: ['c17', 'c50', 'c82'],\n",
       " 783: ['c19', 'c75', 'c89'],\n",
       " 784: ['c12', 'c46', 'c52'],\n",
       " 785: ['c0', 'c4', 'c20'],\n",
       " 786: ['c1', 'c34', 'c51'],\n",
       " 787: ['c25', 'c35', 'c56'],\n",
       " 788: ['c38', 'c58', 'c88'],\n",
       " 789: ['c36', 'c83', 'c90'],\n",
       " 790: ['c11', 'c48', 'c60'],\n",
       " 791: ['c22', 'c43', 'c81'],\n",
       " 792: ['c21', 'c72', 'c80'],\n",
       " 793: ['c23', 'c24', 'c57'],\n",
       " 794: ['c44', 'c68', 'c76'],\n",
       " 795: ['c53', 'c63', 'c71'],\n",
       " 796: ['c18', 'c45', 'c91'],\n",
       " 797: ['c7', 'c58', 'c96'],\n",
       " 798: ['c30', 'c37', 'c95'],\n",
       " 799: ['c28', 'c32', 'c92'],\n",
       " 800: ['c59', 'c62', 'c89'],\n",
       " 801: ['c14', 'c65', 'c74'],\n",
       " 802: ['c46', 'c67', 'c93'],\n",
       " 803: ['c17', 'c70', 'c85'],\n",
       " 804: ['c47', 'c76', 'c82'],\n",
       " 805: ['c4', 'c6', 'c94'],\n",
       " 806: ['c1', 'c9', 'c73'],\n",
       " 807: ['c39', 'c41', 'c98'],\n",
       " 808: ['c8', 'c60', 'c86'],\n",
       " 809: ['c29', 'c64', 'c69'],\n",
       " 810: ['c36', 'c42', 'c49'],\n",
       " 811: ['c2', 'c61', 'c83'],\n",
       " 812: ['c13', 'c27', 'c35'],\n",
       " 1300: ['c2', 'c35', 'c87'],\n",
       " 1301: ['c7', 'c42', 'c80'],\n",
       " 1302: ['c12', 'c38', 'c51'],\n",
       " 1303: ['c25', 'c33', 'c89'],\n",
       " 1304: ['c17', 'c63', 'c81'],\n",
       " 1305: ['c18', 'c28', 'c57'],\n",
       " 1306: ['c56', 'c62', 'c66'],\n",
       " 1307: ['c23', 'c26', 'c83'],\n",
       " 1308: ['c10', 'c13', 'c98'],\n",
       " 1309: ['c41', 'c49', 'c54'],\n",
       " 1310: ['c29', 'c43', 'c85'],\n",
       " 1311: ['c58', 'c64', 'c77'],\n",
       " 1312: ['c14', 'c55', 'c86'],\n",
       " 1313: ['c22', 'c45', 'c97'],\n",
       " 1314: ['c24', 'c59', 'c70'],\n",
       " 1315: ['c34', 'c80', 'c91'],\n",
       " 1316: ['c46', 'c69', 'c71'],\n",
       " 1317: ['c19', 'c52', 'c67'],\n",
       " 1318: ['c12', 'c18', 'c50'],\n",
       " 1319: ['c75', 'c76', 'c95'],\n",
       " 1320: ['c57', 'c90', 'c96'],\n",
       " 1321: ['c17', 'c68', 'c79'],\n",
       " 1322: ['c4', 'c25', 'c53'],\n",
       " 1323: ['c7', 'c74', 'c82'],\n",
       " 1324: ['c0', 'c38', 'c93'],\n",
       " 1325: ['c10', 'c84', 'c94'],\n",
       " 1330: ['c11', 'c66', 'c81'],\n",
       " 1331: ['c14', 'c15', 'c77'],\n",
       " 1332: ['c21', 'c36', 'c54'],\n",
       " 1333: ['c6', 'c37', 'c40'],\n",
       " 1334: ['c27', 'c46', 'c65'],\n",
       " 1335: ['c5', 'c47', 'c71'],\n",
       " 1336: ['c1', 'c30', 'c39'],\n",
       " 1337: ['c2', 'c73', 'c92'],\n",
       " 1338: ['c35', 'c61', 'c63'],\n",
       " 1339: ['c20', 'c44', 'c87'],\n",
       " 1340: ['c32', 'c42', 'c78'],\n",
       " 1341: ['c3', 'c8', 'c16'],\n",
       " 1342: ['c31', 'c48', 'c88'],\n",
       " 1343: ['c9', 'c60', 'c72'],\n",
       " 1344: ['c28', 'c30', 'c89'],\n",
       " 1345: ['c6', 'c33', 'c51'],\n",
       " 1346: ['c15', 'c79', 'c94'],\n",
       " 1347: ['c68', 'c84', 'c93'],\n",
       " 1348: ['c11', 'c82', 'c90'],\n",
       " 1349: ['c0', 'c67', 'c76'],\n",
       " 1350: ['c24', 'c45', 'c74'],\n",
       " 1351: ['c19', 'c49', 'c70'],\n",
       " 1352: ['c22', 'c75', 'c96'],\n",
       " 1353: ['c59', 'c91', 'c95'],\n",
       " 1354: ['c50', 'c52', 'c97'],\n",
       " 1355: ['c4', 'c13', 'c43'],\n",
       " 1356: ['c20', 'c29', 'c98'],\n",
       " 1357: ['c26', 'c53', 'c69'],\n",
       " 1358: ['c23', 'c41', 'c62'],\n",
       " 1359: ['c16', 'c65', 'c83'],\n",
       " 1360: ['c58', 'c86', 'c92'],\n",
       " 1361: ['c1', 'c36', 'c64'],\n",
       " 1362: ['c37', 'c55', 'c85'],\n",
       " 1363: ['c34', 'c47', 'c56'],\n",
       " 490: ['c60', 'c63', 'c75'],\n",
       " 491: ['c41', 'c77', 'c89'],\n",
       " 492: ['c2', 'c58', 'c97'],\n",
       " 493: ['c4', 'c56', 'c91'],\n",
       " 494: ['c12', 'c57', 'c72'],\n",
       " 495: ['c27', 'c70', 'c73'],\n",
       " 496: ['c9', 'c25', 'c76'],\n",
       " 497: ['c28', 'c44', 'c96'],\n",
       " 498: ['c20', 'c48', 'c74'],\n",
       " 499: ['c7', 'c46', 'c68'],\n",
       " 500: ['c32', 'c51', 'c79'],\n",
       " 501: ['c6', 'c18', 'c19'],\n",
       " 502: ['c3', 'c24', 'c75'],\n",
       " 503: ['c17', 'c60', 'c88'],\n",
       " 504: ['c61', 'c71', 'c72'],\n",
       " 505: ['c30', 'c40', 'c81'],\n",
       " 506: ['c52', 'c90', 'c93'],\n",
       " 507: ['c8', 'c11', 'c50'],\n",
       " 508: ['c15', 'c38', 'c62'],\n",
       " 509: ['c39', 'c67', 'c70'],\n",
       " 510: ['c31', 'c35', 'c85'],\n",
       " 511: ['c42', 'c83', 'c92'],\n",
       " 512: ['c1', 'c37', 'c43'],\n",
       " 516: ['c33', 'c54', 'c78'],\n",
       " 517: ['c23', 'c45', 'c53'],\n",
       " 518: ['c5', 'c61', 'c87'],\n",
       " 519: ['c21', 'c64', 'c82'],\n",
       " 520: ['c0', 'c36', 'c89'],\n",
       " 521: ['c13', 'c66', 'c80'],\n",
       " 522: ['c14', 'c22', 'c49'],\n",
       " 523: ['c65', 'c69', 'c94'],\n",
       " 524: ['c10', 'c29', 'c34'],\n",
       " 525: ['c26', 'c47', 'c95'],\n",
       " 526: ['c59', 'c84', 'c86'],\n",
       " 527: ['c31', 'c63', 'c67'],\n",
       " 528: ['c38', 'c39', 'c71'],\n",
       " 529: ['c16', 'c24', 'c98'],\n",
       " 530: ['c55', 'c88', 'c96'],\n",
       " 531: ['c8', 'c17', 'c78'],\n",
       " 532: ['c27', 'c76', 'c91'],\n",
       " 533: ['c11', 'c68', 'c74'],\n",
       " 534: ['c0', 'c5', 'c32'],\n",
       " 535: ['c30', 'c50', 'c51'],\n",
       " 536: ['c9', 'c44', 'c46'],\n",
       " 537: ['c33', 'c79', 'c90'],\n",
       " 538: ['c7', 'c53', 'c66'],\n",
       " 539: ['c21', 'c25', 'c45'],\n",
       " 540: ['c18', 'c80', 'c93'],\n",
       " 541: ['c3', 'c48', 'c52'],\n",
       " 542: ['c2', 'c20', 'c40'],\n",
       " 543: ['c10', 'c28', 'c82'],\n",
       " 544: ['c19', 'c73', 'c77'],\n",
       " 545: ['c4', 'c12', 'c37'],\n",
       " 546: ['c6', 'c56', 'c98'],\n",
       " 548: ['c42', 'c55', 'c81'],\n",
       " 549: ['c49', 'c57', 'c62'],\n",
       " 550: ['c43', 'c83', 'c95'],\n",
       " 551: ['c35', 'c69', 'c97'],\n",
       " 552: ['c16', 'c41', 'c58'],\n",
       " 1461: ['c24', 'c51', 'c85'],\n",
       " 1462: ['c4', 'c23', 'c28'],\n",
       " 1463: ['c25', 'c29', 'c82'],\n",
       " 1464: ['c36', 'c68', 'c95'],\n",
       " 1465: ['c1', 'c54', 'c92'],\n",
       " 1466: ['c9', 'c91', 'c96'],\n",
       " 1467: ['c11', 'c47', 'c59'],\n",
       " 1468: ['c7', 'c18', 'c87'],\n",
       " 1469: ['c15', 'c52', 'c64'],\n",
       " 1470: ['c32', 'c63', 'c73'],\n",
       " 1471: ['c8', 'c38', 'c89'],\n",
       " 1472: ['c26', 'c48', 'c72'],\n",
       " 1473: ['c12', 'c20', 'c23'],\n",
       " 1474: ['c27', 'c53', 'c93'],\n",
       " 1475: ['c31', 'c66', 'c94'],\n",
       " 1476: ['c13', 'c44', 'c74'],\n",
       " 1477: ['c34', 'c45', 'c84'],\n",
       " 1478: ['c6', 'c39', 'c46'],\n",
       " 1479: ['c57', 'c75', 'c88'],\n",
       " 1480: ['c60', 'c76', 'c90'],\n",
       " 1481: ['c25', 'c80', 'c83'],\n",
       " 1482: ['c43', 'c56', 'c61'],\n",
       " 1485: ['c14', 'c40', 'c41'],\n",
       " 1486: ['c0', 'c58', 'c62'],\n",
       " 1487: ['c7', 'c21', 'c38'],\n",
       " 1488: ['c16', 'c19', 'c33'],\n",
       " 1489: ['c4', 'c42', 'c71'],\n",
       " 1490: ['c44', 'c52', 'c86'],\n",
       " 1491: ['c2', 'c70', 'c81'],\n",
       " 1492: ['c22', 'c50', 'c65'],\n",
       " 1493: ['c5', 'c17', 'c51'],\n",
       " 1494: ['c10', 'c68', 'c97'],\n",
       " 1495: ['c55', 'c90', 'c98'],\n",
       " 1496: ['c20', 'c49', 'c79'],\n",
       " 1497: ['c3', 'c37', 'c77'],\n",
       " 1498: ['c30', 'c67', 'c78'],\n",
       " 1499: ['c33', 'c36', 'c69'],\n",
       " 1500: ['c14', 'c35', 'c47'],\n",
       " 1501: ['c24', 'c61', 'c82'],\n",
       " 1502: ['c21', 'c40', 'c86'],\n",
       " 1503: ['c28', 'c80', 'c85'],\n",
       " 1504: ['c32', 'c84', 'c89'],\n",
       " 1505: ['c0', 'c19', 'c92'],\n",
       " 1506: ['c12', 'c22', 'c88'],\n",
       " 1507: ['c15', 'c31', 'c59'],\n",
       " 1508: ['c34', 'c73', 'c78'],\n",
       " 1509: ['c5', 'c95', 'c96'],\n",
       " 1510: ['c18', 'c48', 'c71'],\n",
       " 1511: ['c8', 'c70', 'c75'],\n",
       " 1512: ['c26', 'c29', 'c74'],\n",
       " 1513: ['c27', 'c64', 'c66'],\n",
       " 1514: ['c3', 'c45', 'c54'],\n",
       " 1515: ['c53', 'c60', 'c94'],\n",
       " 1516: ['c46', 'c57', 'c87'],\n",
       " 1517: ['c9', 'c39', 'c63'],\n",
       " 1518: ['c11', 'c13', 'c72'],\n",
       " 1519: ['c1', 'c6', 'c58'],\n",
       " 1520: ['c43', 'c76', 'c93'],\n",
       " 1521: ['c2', 'c65', 'c91'],\n",
       " 949: ['c14', 'c70', 'c78'],\n",
       " 950: ['c15', 'c73', 'c84'],\n",
       " 951: ['c27', 'c57', 'c83'],\n",
       " 952: ['c23', 'c30', 'c80'],\n",
       " 953: ['c21', 'c81', 'c92'],\n",
       " 954: ['c0', 'c26', 'c82'],\n",
       " 955: ['c13', 'c19', 'c69'],\n",
       " 956: ['c46', 'c54', 'c85'],\n",
       " 957: ['c33', 'c38', 'c49'],\n",
       " 958: ['c5', 'c53', 'c62'],\n",
       " 959: ['c1', 'c25', 'c77'],\n",
       " 960: ['c45', 'c61', 'c68'],\n",
       " 961: ['c21', 'c59', 'c79'],\n",
       " 962: ['c50', 'c58', 'c76'],\n",
       " 963: ['c42', 'c43', 'c67'],\n",
       " 964: ['c40', 'c74', 'c97'],\n",
       " 965: ['c4', 'c24', 'c65'],\n",
       " 966: ['c32', 'c69', 'c87'],\n",
       " 967: ['c2', 'c28', 'c41'],\n",
       " 968: ['c29', 'c33', 'c52'],\n",
       " 969: ['c27', 'c89', 'c90'],\n",
       " 970: ['c66', 'c84', 'c92'],\n",
       " 971: ['c14', 'c60', 'c96'],\n",
       " 972: ['c3', 'c35', 'c49'],\n",
       " 973: ['c44', 'c63', 'c94'],\n",
       " 974: ['c64', 'c78', 'c95'],\n",
       " 975: ['c6', 'c48', 'c93'],\n",
       " 976: ['c47', 'c53', 'c54'],\n",
       " 977: ['c61', 'c77', 'c88'],\n",
       " 978: ['c16', 'c85', 'c97'],\n",
       " 979: ['c12', 'c17', 'c30'],\n",
       " 980: ['c29', 'c36', 'c75'],\n",
       " 981: ['c19', 'c28', 'c37'],\n",
       " 982: ['c60', 'c82', 'c98'],\n",
       " 983: ['c9', 'c11', 'c32'],\n",
       " 984: ['c8', 'c10', 'c15'],\n",
       " 985: ['c39', 'c86', 'c91'],\n",
       " 986: ['c22', 'c34', 'c76'],\n",
       " 987: ['c56', 'c70', 'c72'],\n",
       " 988: ['c7', 'c23', 'c51'],\n",
       " 989: ['c16', 'c18', 'c31'],\n",
       " 990: ['c20', 'c55', 'c71'],\n",
       " 991: ['c3', 'c67', 'c81'],\n",
       " 992: ['c50', 'c80', 'c98'],\n",
       " 993: ['c35', 'c40', 'c83'],\n",
       " 994: ['c37', 'c65', 'c87'],\n",
       " 995: ['c47', 'c79', 'c88'],\n",
       " 996: ['c5', 'c52', 'c57'],\n",
       " 997: ['c17', 'c62', 'c73'],\n",
       " 999: ['c24', 'c68', 'c94'],\n",
       " 1000: ['c0', 'c42', 'c90'],\n",
       " 1001: ['c38', 'c46', 'c95'],\n",
       " 1002: ['c1', 'c41', 'c44'],\n",
       " 1003: ['c6', 'c25', 'c86'],\n",
       " 1004: ['c13', 'c45', 'c59'],\n",
       " 1005: ['c26', 'c55', 'c93'],\n",
       " 554: ['c30', 'c56', 'c64'],\n",
       " 555: ['c23', 'c43', 'c78'],\n",
       " 556: ['c0', 'c35', 'c94'],\n",
       " 557: ['c57', 'c61', 'c92'],\n",
       " 558: ['c21', 'c50', 'c96'],\n",
       " 559: ['c10', 'c22', 'c47'],\n",
       " 560: ['c25', 'c66', 'c72'],\n",
       " 561: ['c18', 'c34', 'c89'],\n",
       " 562: ['c67', 'c74', 'c91'],\n",
       " 563: ['c4', 'c9', 'c48'],\n",
       " 564: ['c39', 'c56', 'c75'],\n",
       " 565: ['c33', 'c84', 'c85'],\n",
       " 566: ['c20', 'c65', 'c86'],\n",
       " 567: ['c28', 'c58', 'c71'],\n",
       " 568: ['c26', 'c37', 'c79'],\n",
       " 569: ['c16', 'c54', 'c68'],\n",
       " 570: ['c45', 'c51', 'c92'],\n",
       " 571: ['c2', 'c60', 'c77'],\n",
       " 572: ['c53', 'c87', 'c88'],\n",
       " 573: ['c24', 'c32', 'c55'],\n",
       " 574: ['c29', 'c46', 'c80'],\n",
       " 575: ['c10', 'c12', 'c66'],\n",
       " 576: ['c8', 'c21', 'c34'],\n",
       " 577: ['c1', 'c15', 'c90'],\n",
       " 578: ['c72', 'c81', 'c89'],\n",
       " 579: ['c6', 'c14', 'c62'],\n",
       " 580: ['c17', 'c31', 'c76'],\n",
       " 581: ['c30', 'c36', 'c38'],\n",
       " 582: ['c18', 'c63', 'c77'],\n",
       " 583: ['c7', 'c40', 'c71'],\n",
       " 584: ['c11', 'c44', 'c51'],\n",
       " 585: ['c4', 'c19', 'c41'],\n",
       " 586: ['c9', 'c83', 'c93'],\n",
       " 587: ['c52', 'c82', 'c85'],\n",
       " 588: ['c5', 'c59', 'c69'],\n",
       " 589: ['c27', 'c78', 'c98'],\n",
       " 590: ['c8', 'c13', 'c49'],\n",
       " 591: ['c3', 'c42', 'c73'],\n",
       " 592: ['c70', 'c95', 'c97'],\n",
       " 593: ['c12', 'c63', 'c79'],\n",
       " 594: ['c20', 'c64', 'c81'],\n",
       " 595: ['c36', 'c45', 'c50'],\n",
       " 596: ['c11', 'c22', 'c39'],\n",
       " 597: ['c7', 'c24', 'c48'],\n",
       " 598: ['c17', 'c74', 'c94'],\n",
       " 599: ['c62', 'c67', 'c75'],\n",
       " 600: ['c31', 'c46', 'c91'],\n",
       " 601: ['c15', 'c35', 'c96'],\n",
       " 602: ['c61', 'c65', 'c80'],\n",
       " 603: ['c6', 'c16', 'c47'],\n",
       " 605: ['c26', 'c57', 'c59'],\n",
       " 606: ['c0', 'c23', 'c87'],\n",
       " 607: ['c43', 'c53', 'c90'],\n",
       " 608: ['c55', 'c60', 'c84'],\n",
       " 609: ['c25', 'c58', 'c68'],\n",
       " 611: ['c3', 'c83', 'c98'],\n",
       " 612: ['c12', 'c39', 'c82'],\n",
       " 613: ['c18', 'c64', 'c86'],\n",
       " 614: ['c36', 'c51', 'c52'],\n",
       " 615: ['c19', 'c42', 'c79'],\n",
       " 616: ['c9', 'c38', 'c56'],\n",
       " 617: ['c17', 'c44', 'c89'],\n",
       " 618: ['c20', 'c22', 'c93'],\n",
       " 619: ['c5', 'c50', 'c81'],\n",
       " 620: ['c14', 'c30', 'c75'],\n",
       " 621: ['c33', 'c34', 'c94'],\n",
       " 622: ['c29', 'c47', 'c87'],\n",
       " 623: ['c2', 'c62', 'c63'],\n",
       " 624: ['c21', 'c28', 'c55'],\n",
       " 625: ['c7', 'c27', 'c61'],\n",
       " 626: ['c13', 'c31', 'c77'],\n",
       " 627: ['c25', 'c69', 'c91'],\n",
       " 628: ['c8', 'c37', 'c72'],\n",
       " 629: ['c3', 'c53', 'c97'],\n",
       " 645: ['c32', 'c41', 'c95'],\n",
       " 646: ['c4', 'c10', 'c11'],\n",
       " 647: ['c1', 'c71', 'c76'],\n",
       " 648: ['c38', 'c43', 'c45'],\n",
       " 649: ['c14', 'c44', 'c67'],\n",
       " 650: ['c5', 'c85', 'c93'],\n",
       " 651: ['c46', 'c49', 'c84'],\n",
       " 652: ['c23', 'c68', 'c88'],\n",
       " 653: ['c52', 'c78', 'c92'],\n",
       " 654: ['c0', 'c60', 'c74'],\n",
       " 655: ['c29', 'c35', 'c48'],\n",
       " 656: ['c54', 'c66', 'c70'],\n",
       " 657: ['c40', 'c49', 'c96'],\n",
       " 658: ['c15', 'c57', 'c65'],\n",
       " 659: ['c58', 'c59', 'c80'],\n",
       " 660: ['c16', 'c26', 'c73'],\n",
       " 662: ['c6', 'c74', 'c90'],\n",
       " 663: ['c2', 'c24', 'c50'],\n",
       " 664: ['c33', 'c63', 'c97'],\n",
       " 665: ['c34', 'c67', 'c88'],\n",
       " 666: ['c10', 'c30', 'c85'],\n",
       " 667: ['c9', 'c53', 'c81'],\n",
       " 668: ['c8', 'c32', 'c39'],\n",
       " 669: ['c20', 'c51', 'c54'],\n",
       " 670: ['c4', 'c82', 'c83'],\n",
       " 671: ['c36', 'c37', 'c86'],\n",
       " 672: ['c69', 'c79', 'c89'],\n",
       " 673: ['c7', 'c12', 'c56'],\n",
       " 674: ['c28', 'c64', 'c91'],\n",
       " 675: ['c1', 'c18', 'c75'],\n",
       " 676: ['c13', 'c22', 'c42'],\n",
       " 677: ['c31', 'c95', 'c98'],\n",
       " 678: ['c19', 'c76', 'c87'],\n",
       " 679: ['c17', 'c21', 'c27'],\n",
       " 1242: ['c31', 'c32', 'c90'],\n",
       " 1243: ['c11', 'c25', 'c42'],\n",
       " 1244: ['c0', 'c28', 'c70'],\n",
       " 1245: ['c38', 'c48', 'c73'],\n",
       " 1246: ['c36', 'c39', 'c80'],\n",
       " 1247: ['c8', 'c40', 'c57'],\n",
       " 1248: ['c35', 'c50', 'c62'],\n",
       " 1249: ['c12', 'c64', 'c83'],\n",
       " 1250: ['c4', 'c7', 'c55'],\n",
       " 1251: ['c6', 'c84', 'c97'],\n",
       " 1252: ['c5', 'c14', 'c46'],\n",
       " 1253: ['c11', 'c37', 'c41'],\n",
       " ...}"
      ]
     },
     "execution_count": 99,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "region_2_readout_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 100,
   "id": "152a93c4",
   "metadata": {},
   "outputs": [],
   "source": [
    "# collect types\n",
    "readout_usage_by_type = {}\n",
    "for _region, _infos in region_2_readout_dict.items():\n",
    "    for _info in _infos:\n",
    "        if _info[0] not in readout_usage_by_type:\n",
    "            readout_usage_by_type[_info[0]] = []\n",
    "        if int(_info[1:]) not in readout_usage_by_type[_info[0]]:\n",
    "            readout_usage_by_type[_info[0]].append(int(_info[1:]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "id": "f3648545",
   "metadata": {},
   "outputs": [],
   "source": [
    "readout_usage_dict = {_type:len(_v) for _type,_v in readout_usage_by_type.items()}\n",
    "readout_dict ={}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 102,
   "id": "54fa06dc",
   "metadata": {},
   "outputs": [],
   "source": [
    "reload(library_tools.assemble)\n",
    "overwrite_readout_usage = True\n",
    "readout_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Readouts'\n",
    "readout_start = 0\n",
    "num_color = 2\n",
    "comb_readouts = library_tools.assemble.load_readouts(readout_usage_dict['c'], \n",
    "                                                       _type='snr_after_630', \n",
    "                                                       _readout_folder=readout_folder, \n",
    "                                                       _num_colors=2, # 750, 647\n",
    "                                                       _start_id=readout_start,\n",
    "                                                       )\n",
    "readout_dict['c'] = comb_readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "id": "3f1ffff7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "save\n"
     ]
    }
   ],
   "source": [
    "# save\n",
    "readout_usage_file = os.path.join(library_folder, 'readout_usage.pkl')\n",
    "if not os.path.isfile(readout_usage_file) or overwrite_readout_usage:\n",
    "    print('save')\n",
    "    pickle.dump(readout_dict, open(readout_usage_file, 'wb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3d1eeb3b",
   "metadata": {},
   "source": [
    "<a id='3.3'></a>\n",
    "## 3.3 assemble probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 108,
   "id": "ec61f5f6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Assemble probes by given target sequences, readouts and primers.\n",
      "-- included readout types: ['c']\n",
      "--- assemblying 300 probes in region: 0\n",
      "--- assemblying 300 probes in region: 1\n",
      "--- assemblying 300 probes in region: 2\n",
      "--- assemblying 242 probes in region: 3\n",
      "--- assemblying 256 probes in region: 4\n",
      "--- assemblying 300 probes in region: 5\n",
      "--- assemblying 300 probes in region: 6\n",
      "--- assemblying 300 probes in region: 7\n",
      "--- assemblying 300 probes in region: 8\n",
      "--- assemblying 264 probes in region: 9\n",
      "--- assemblying 216 probes in region: 10\n",
      "--- assemblying 212 probes in region: 11\n",
      "--- assemblying 240 probes in region: 12\n",
      "--- assemblying 300 probes in region: 13\n",
      "--- assemblying 176 probes in region: 14\n",
      "--- assemblying 300 probes in region: 15\n",
      "--- assemblying 245 probes in region: 16\n",
      "--- assemblying 203 probes in region: 17\n",
      "--- assemblying 204 probes in region: 18\n",
      "--- assemblying 300 probes in region: 19\n",
      "--- assemblying 206 probes in region: 20\n",
      "--- assemblying 255 probes in region: 21\n",
      "--- assemblying 300 probes in region: 22\n",
      "--- assemblying 300 probes in region: 23\n",
      "--- assemblying 279 probes in region: 24\n",
      "--- assemblying 238 probes in region: 25\n",
      "--- assemblying 300 probes in region: 26\n",
      "--- assemblying 149 probes in region: 27\n",
      "--- assemblying 262 probes in region: 28\n",
      "--- assemblying 199 probes in region: 29\n",
      "--- assemblying 263 probes in region: 30\n",
      "--- assemblying 215 probes in region: 31\n",
      "--- assemblying 190 probes in region: 32\n",
      "--- assemblying 300 probes in region: 33\n",
      "--- assemblying 230 probes in region: 34\n",
      "--- assemblying 203 probes in region: 35\n",
      "--- assemblying 154 probes in region: 36\n",
      "--- assemblying 300 probes in region: 37\n",
      "--- assemblying 211 probes in region: 38\n",
      "--- assemblying 237 probes in region: 39\n",
      "--- assemblying 293 probes in region: 40\n",
      "--- assemblying 201 probes in region: 42\n",
      "--- assemblying 235 probes in region: 43\n",
      "--- assemblying 300 probes in region: 44\n",
      "--- assemblying 187 probes in region: 45\n",
      "--- assemblying 300 probes in region: 46\n",
      "--- assemblying 213 probes in region: 47\n",
      "--- assemblying 60 probes in region: 57\n",
      "--- assemblying 187 probes in region: 60\n",
      "--- assemblying 300 probes in region: 62\n",
      "--- assemblying 241 probes in region: 63\n",
      "--- assemblying 199 probes in region: 64\n",
      "--- assemblying 114 probes in region: 65\n",
      "--- assemblying 127 probes in region: 66\n",
      "--- assemblying 271 probes in region: 67\n",
      "--- assemblying 226 probes in region: 68\n",
      "--- assemblying 128 probes in region: 69\n",
      "--- assemblying 120 probes in region: 70\n",
      "--- assemblying 137 probes in region: 71\n",
      "--- assemblying 267 probes in region: 72\n",
      "--- assemblying 198 probes in region: 73\n",
      "--- assemblying 173 probes in region: 74\n",
      "--- assemblying 149 probes in region: 75\n",
      "--- assemblying 121 probes in region: 76\n",
      "--- assemblying 218 probes in region: 77\n",
      "--- assemblying 142 probes in region: 78\n",
      "--- assemblying 198 probes in region: 79\n",
      "--- assemblying 247 probes in region: 80\n",
      "--- assemblying 223 probes in region: 81\n",
      "--- assemblying 259 probes in region: 83\n",
      "--- assemblying 191 probes in region: 84\n",
      "--- assemblying 300 probes in region: 85\n",
      "--- assemblying 237 probes in region: 86\n",
      "--- assemblying 236 probes in region: 87\n",
      "--- assemblying 204 probes in region: 88\n",
      "--- assemblying 300 probes in region: 89\n",
      "--- assemblying 118 probes in region: 90\n",
      "--- assemblying 300 probes in region: 91\n",
      "--- assemblying 271 probes in region: 92\n",
      "--- assemblying 212 probes in region: 93\n",
      "--- assemblying 178 probes in region: 94\n",
      "--- assemblying 214 probes in region: 95\n",
      "--- assemblying 258 probes in region: 96\n",
      "--- assemblying 168 probes in region: 98\n",
      "--- assemblying 300 probes in region: 100\n",
      "--- assemblying 286 probes in region: 101\n",
      "--- assemblying 240 probes in region: 102\n",
      "--- assemblying 205 probes in region: 103\n",
      "--- assemblying 119 probes in region: 104\n",
      "--- assemblying 300 probes in region: 105\n",
      "--- assemblying 244 probes in region: 106\n",
      "--- assemblying 300 probes in region: 107\n",
      "--- assemblying 213 probes in region: 108\n",
      "--- assemblying 225 probes in region: 109\n",
      "--- assemblying 230 probes in region: 110\n",
      "--- assemblying 300 probes in region: 111\n",
      "--- assemblying 167 probes in region: 112\n",
      "--- assemblying 215 probes in region: 114\n",
      "--- assemblying 224 probes in region: 115\n",
      "--- assemblying 228 probes in region: 116\n",
      "--- assemblying 276 probes in region: 117\n",
      "--- assemblying 216 probes in region: 118\n",
      "--- assemblying 221 probes in region: 119\n",
      "--- assemblying 200 probes in region: 120\n",
      "--- assemblying 295 probes in region: 121\n",
      "--- assemblying 270 probes in region: 122\n",
      "--- assemblying 182 probes in region: 123\n",
      "--- assemblying 235 probes in region: 124\n",
      "--- assemblying 251 probes in region: 125\n",
      "--- assemblying 291 probes in region: 126\n",
      "--- assemblying 222 probes in region: 127\n",
      "--- assemblying 275 probes in region: 128\n",
      "--- assemblying 216 probes in region: 129\n",
      "--- assemblying 162 probes in region: 130\n",
      "--- assemblying 190 probes in region: 131\n",
      "--- assemblying 192 probes in region: 132\n",
      "--- assemblying 300 probes in region: 133\n",
      "--- assemblying 237 probes in region: 134\n",
      "--- assemblying 300 probes in region: 137\n",
      "--- assemblying 285 probes in region: 138\n",
      "--- assemblying 174 probes in region: 139\n",
      "--- assemblying 261 probes in region: 140\n",
      "--- assemblying 74 probes in region: 141\n",
      "--- assemblying 186 probes in region: 143\n",
      "--- assemblying 187 probes in region: 144\n",
      "--- assemblying 178 probes in region: 145\n",
      "--- assemblying 294 probes in region: 146\n",
      "--- assemblying 300 probes in region: 147\n",
      "--- assemblying 193 probes in region: 148\n",
      "--- assemblying 300 probes in region: 149\n",
      "--- assemblying 240 probes in region: 150\n",
      "--- assemblying 248 probes in region: 152\n",
      "--- assemblying 300 probes in region: 153\n",
      "--- assemblying 283 probes in region: 154\n",
      "--- assemblying 252 probes in region: 155\n",
      "--- assemblying 292 probes in region: 156\n",
      "--- assemblying 255 probes in region: 157\n",
      "--- assemblying 227 probes in region: 158\n",
      "--- assemblying 231 probes in region: 159\n",
      "--- assemblying 136 probes in region: 160\n",
      "--- assemblying 134 probes in region: 161\n",
      "--- assemblying 300 probes in region: 162\n",
      "--- assemblying 234 probes in region: 163\n",
      "--- assemblying 300 probes in region: 164\n",
      "--- assemblying 244 probes in region: 165\n",
      "--- assemblying 300 probes in region: 166\n",
      "--- assemblying 217 probes in region: 167\n",
      "--- assemblying 300 probes in region: 168\n",
      "--- assemblying 238 probes in region: 169\n",
      "--- assemblying 300 probes in region: 170\n",
      "--- assemblying 272 probes in region: 171\n",
      "--- assemblying 200 probes in region: 172\n",
      "--- assemblying 173 probes in region: 173\n",
      "--- assemblying 198 probes in region: 174\n",
      "--- assemblying 300 probes in region: 175\n",
      "--- assemblying 152 probes in region: 176\n",
      "--- assemblying 279 probes in region: 177\n",
      "--- assemblying 280 probes in region: 178\n",
      "--- assemblying 230 probes in region: 179\n",
      "--- assemblying 226 probes in region: 180\n",
      "--- assemblying 248 probes in region: 181\n",
      "--- assemblying 93 probes in region: 182\n",
      "--- assemblying 204 probes in region: 183\n",
      "--- assemblying 173 probes in region: 184\n",
      "--- assemblying 300 probes in region: 185\n",
      "--- assemblying 300 probes in region: 186\n",
      "--- assemblying 239 probes in region: 187\n",
      "--- assemblying 233 probes in region: 188\n",
      "--- assemblying 191 probes in region: 189\n",
      "--- assemblying 150 probes in region: 190\n",
      "--- assemblying 274 probes in region: 191\n",
      "--- assemblying 270 probes in region: 192\n",
      "--- assemblying 300 probes in region: 193\n",
      "--- assemblying 297 probes in region: 194\n",
      "--- assemblying 199 probes in region: 195\n",
      "--- assemblying 300 probes in region: 196\n",
      "--- assemblying 251 probes in region: 197\n",
      "--- assemblying 300 probes in region: 198\n",
      "--- assemblying 275 probes in region: 199\n",
      "--- assemblying 229 probes in region: 200\n",
      "--- assemblying 300 probes in region: 201\n",
      "--- assemblying 300 probes in region: 202\n",
      "--- assemblying 190 probes in region: 203\n",
      "--- assemblying 265 probes in region: 204\n",
      "--- assemblying 241 probes in region: 205\n",
      "--- assemblying 262 probes in region: 206\n",
      "--- assemblying 300 probes in region: 207\n",
      "--- assemblying 174 probes in region: 208\n",
      "--- assemblying 180 probes in region: 209\n",
      "--- assemblying 300 probes in region: 210\n",
      "--- assemblying 300 probes in region: 211\n",
      "--- assemblying 300 probes in region: 212\n",
      "--- assemblying 200 probes in region: 213\n",
      "--- assemblying 219 probes in region: 214\n",
      "--- assemblying 142 probes in region: 215\n",
      "--- assemblying 300 probes in region: 216\n",
      "--- assemblying 278 probes in region: 217\n",
      "--- assemblying 252 probes in region: 218\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 258 probes in region: 219\n",
      "--- assemblying 214 probes in region: 220\n",
      "--- assemblying 256 probes in region: 221\n",
      "--- assemblying 292 probes in region: 222\n",
      "--- assemblying 300 probes in region: 223\n",
      "--- assemblying 300 probes in region: 224\n",
      "--- assemblying 234 probes in region: 225\n",
      "--- assemblying 220 probes in region: 226\n",
      "--- assemblying 242 probes in region: 227\n",
      "--- assemblying 148 probes in region: 228\n",
      "--- assemblying 197 probes in region: 229\n",
      "--- assemblying 166 probes in region: 230\n",
      "--- assemblying 196 probes in region: 233\n",
      "--- assemblying 255 probes in region: 234\n",
      "--- assemblying 233 probes in region: 235\n",
      "--- assemblying 182 probes in region: 236\n",
      "--- assemblying 179 probes in region: 237\n",
      "--- assemblying 251 probes in region: 238\n",
      "--- assemblying 231 probes in region: 239\n",
      "--- assemblying 234 probes in region: 240\n",
      "--- assemblying 272 probes in region: 241\n",
      "--- assemblying 288 probes in region: 242\n",
      "--- assemblying 165 probes in region: 243\n",
      "--- assemblying 300 probes in region: 244\n",
      "--- assemblying 238 probes in region: 245\n",
      "--- assemblying 205 probes in region: 246\n",
      "--- assemblying 234 probes in region: 247\n",
      "--- assemblying 277 probes in region: 248\n",
      "--- assemblying 242 probes in region: 249\n",
      "--- assemblying 172 probes in region: 250\n",
      "--- assemblying 300 probes in region: 251\n",
      "--- assemblying 287 probes in region: 252\n",
      "--- assemblying 188 probes in region: 253\n",
      "--- assemblying 282 probes in region: 254\n",
      "--- assemblying 225 probes in region: 255\n",
      "--- assemblying 218 probes in region: 256\n",
      "--- assemblying 288 probes in region: 257\n",
      "--- assemblying 300 probes in region: 258\n",
      "--- assemblying 160 probes in region: 259\n",
      "--- assemblying 162 probes in region: 260\n",
      "--- assemblying 159 probes in region: 261\n",
      "--- assemblying 179 probes in region: 262\n",
      "--- assemblying 300 probes in region: 263\n",
      "--- assemblying 259 probes in region: 264\n",
      "--- assemblying 220 probes in region: 265\n",
      "--- assemblying 251 probes in region: 266\n",
      "--- assemblying 263 probes in region: 267\n",
      "--- assemblying 175 probes in region: 268\n",
      "--- assemblying 300 probes in region: 269\n",
      "--- assemblying 300 probes in region: 270\n",
      "--- assemblying 220 probes in region: 271\n",
      "--- assemblying 232 probes in region: 272\n",
      "--- assemblying 231 probes in region: 273\n",
      "--- assemblying 300 probes in region: 274\n",
      "--- assemblying 300 probes in region: 275\n",
      "--- assemblying 300 probes in region: 276\n",
      "--- assemblying 300 probes in region: 277\n",
      "--- assemblying 224 probes in region: 278\n",
      "--- assemblying 210 probes in region: 279\n",
      "--- assemblying 300 probes in region: 280\n",
      "--- assemblying 239 probes in region: 281\n",
      "--- assemblying 262 probes in region: 282\n",
      "--- assemblying 293 probes in region: 283\n",
      "--- assemblying 215 probes in region: 284\n",
      "--- assemblying 234 probes in region: 285\n",
      "--- assemblying 187 probes in region: 286\n",
      "--- assemblying 194 probes in region: 287\n",
      "--- assemblying 249 probes in region: 288\n",
      "--- assemblying 250 probes in region: 289\n",
      "--- assemblying 253 probes in region: 290\n",
      "--- assemblying 195 probes in region: 291\n",
      "--- assemblying 226 probes in region: 292\n",
      "--- assemblying 157 probes in region: 293\n",
      "--- assemblying 209 probes in region: 295\n",
      "--- assemblying 274 probes in region: 296\n",
      "--- assemblying 194 probes in region: 297\n",
      "--- assemblying 253 probes in region: 298\n",
      "--- assemblying 141 probes in region: 299\n",
      "--- assemblying 151 probes in region: 300\n",
      "--- assemblying 300 probes in region: 302\n",
      "--- assemblying 213 probes in region: 303\n",
      "--- assemblying 232 probes in region: 304\n",
      "--- assemblying 222 probes in region: 305\n",
      "--- assemblying 242 probes in region: 306\n",
      "--- assemblying 231 probes in region: 307\n",
      "--- assemblying 203 probes in region: 308\n",
      "--- assemblying 300 probes in region: 309\n",
      "--- assemblying 166 probes in region: 310\n",
      "--- assemblying 267 probes in region: 311\n",
      "--- assemblying 241 probes in region: 312\n",
      "--- assemblying 182 probes in region: 313\n",
      "--- assemblying 253 probes in region: 314\n",
      "--- assemblying 228 probes in region: 315\n",
      "--- assemblying 183 probes in region: 316\n",
      "--- assemblying 251 probes in region: 317\n",
      "--- assemblying 178 probes in region: 318\n",
      "--- assemblying 249 probes in region: 319\n",
      "--- assemblying 155 probes in region: 320\n",
      "--- assemblying 279 probes in region: 321\n",
      "--- assemblying 258 probes in region: 322\n",
      "--- assemblying 248 probes in region: 323\n",
      "--- assemblying 171 probes in region: 324\n",
      "--- assemblying 176 probes in region: 325\n",
      "--- assemblying 191 probes in region: 326\n",
      "--- assemblying 106 probes in region: 327\n",
      "--- assemblying 170 probes in region: 328\n",
      "--- assemblying 295 probes in region: 329\n",
      "--- assemblying 218 probes in region: 330\n",
      "--- assemblying 154 probes in region: 331\n",
      "--- assemblying 300 probes in region: 332\n",
      "--- assemblying 214 probes in region: 333\n",
      "--- assemblying 234 probes in region: 334\n",
      "--- assemblying 271 probes in region: 335\n",
      "--- assemblying 244 probes in region: 336\n",
      "--- assemblying 181 probes in region: 337\n",
      "--- assemblying 222 probes in region: 338\n",
      "--- assemblying 257 probes in region: 339\n",
      "--- assemblying 201 probes in region: 340\n",
      "--- assemblying 284 probes in region: 341\n",
      "--- assemblying 157 probes in region: 342\n",
      "--- assemblying 260 probes in region: 343\n",
      "--- assemblying 234 probes in region: 344\n",
      "--- assemblying 225 probes in region: 345\n",
      "--- assemblying 279 probes in region: 346\n",
      "--- assemblying 202 probes in region: 347\n",
      "--- assemblying 223 probes in region: 348\n",
      "--- assemblying 161 probes in region: 349\n",
      "--- assemblying 300 probes in region: 350\n",
      "--- assemblying 300 probes in region: 351\n",
      "--- assemblying 300 probes in region: 352\n",
      "--- assemblying 158 probes in region: 353\n",
      "--- assemblying 293 probes in region: 354\n",
      "--- assemblying 234 probes in region: 355\n",
      "--- assemblying 257 probes in region: 356\n",
      "--- assemblying 170 probes in region: 357\n",
      "--- assemblying 216 probes in region: 358\n",
      "--- assemblying 179 probes in region: 359\n",
      "--- assemblying 190 probes in region: 360\n",
      "--- assemblying 175 probes in region: 361\n",
      "--- assemblying 300 probes in region: 362\n",
      "--- assemblying 300 probes in region: 363\n",
      "--- assemblying 253 probes in region: 364\n",
      "--- assemblying 284 probes in region: 365\n",
      "--- assemblying 214 probes in region: 366\n",
      "--- assemblying 102 probes in region: 367\n",
      "--- assemblying 64 probes in region: 368\n",
      "--- assemblying 158 probes in region: 370\n",
      "--- assemblying 84 probes in region: 371\n",
      "--- assemblying 236 probes in region: 372\n",
      "--- assemblying 270 probes in region: 373\n",
      "--- assemblying 224 probes in region: 374\n",
      "--- assemblying 226 probes in region: 375\n",
      "--- assemblying 222 probes in region: 376\n",
      "--- assemblying 300 probes in region: 377\n",
      "--- assemblying 287 probes in region: 379\n",
      "--- assemblying 224 probes in region: 380\n",
      "--- assemblying 278 probes in region: 381\n",
      "--- assemblying 300 probes in region: 382\n",
      "--- assemblying 176 probes in region: 383\n",
      "--- assemblying 166 probes in region: 384\n",
      "--- assemblying 272 probes in region: 385\n",
      "--- assemblying 215 probes in region: 386\n",
      "--- assemblying 192 probes in region: 387\n",
      "--- assemblying 277 probes in region: 388\n",
      "--- assemblying 170 probes in region: 389\n",
      "--- assemblying 164 probes in region: 390\n",
      "--- assemblying 257 probes in region: 391\n",
      "--- assemblying 202 probes in region: 392\n",
      "--- assemblying 229 probes in region: 393\n",
      "--- assemblying 300 probes in region: 394\n",
      "--- assemblying 256 probes in region: 395\n",
      "--- assemblying 182 probes in region: 396\n",
      "--- assemblying 245 probes in region: 397\n",
      "--- assemblying 162 probes in region: 398\n",
      "--- assemblying 196 probes in region: 399\n",
      "--- assemblying 222 probes in region: 400\n",
      "--- assemblying 237 probes in region: 401\n",
      "--- assemblying 191 probes in region: 402\n",
      "--- assemblying 300 probes in region: 403\n",
      "--- assemblying 300 probes in region: 404\n",
      "--- assemblying 205 probes in region: 405\n",
      "--- assemblying 261 probes in region: 406\n",
      "--- assemblying 300 probes in region: 407\n",
      "--- assemblying 163 probes in region: 408\n",
      "--- assemblying 221 probes in region: 409\n",
      "--- assemblying 271 probes in region: 410\n",
      "--- assemblying 282 probes in region: 411\n",
      "--- assemblying 212 probes in region: 412\n",
      "--- assemblying 300 probes in region: 413\n",
      "--- assemblying 188 probes in region: 414\n",
      "--- assemblying 252 probes in region: 415\n",
      "--- assemblying 252 probes in region: 416\n",
      "--- assemblying 300 probes in region: 417\n",
      "--- assemblying 200 probes in region: 418\n",
      "--- assemblying 300 probes in region: 419\n",
      "--- assemblying 212 probes in region: 421\n",
      "--- assemblying 298 probes in region: 422\n",
      "--- assemblying 300 probes in region: 423\n",
      "--- assemblying 300 probes in region: 424\n",
      "--- assemblying 221 probes in region: 425\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 300 probes in region: 426\n",
      "--- assemblying 217 probes in region: 427\n",
      "--- assemblying 243 probes in region: 428\n",
      "--- assemblying 193 probes in region: 429\n",
      "--- assemblying 236 probes in region: 430\n",
      "--- assemblying 237 probes in region: 431\n",
      "--- assemblying 270 probes in region: 432\n",
      "--- assemblying 300 probes in region: 435\n",
      "--- assemblying 294 probes in region: 436\n",
      "--- assemblying 188 probes in region: 437\n",
      "--- assemblying 206 probes in region: 438\n",
      "--- assemblying 300 probes in region: 439\n",
      "--- assemblying 300 probes in region: 440\n",
      "--- assemblying 183 probes in region: 441\n",
      "--- assemblying 276 probes in region: 442\n",
      "--- assemblying 241 probes in region: 443\n",
      "--- assemblying 300 probes in region: 444\n",
      "--- assemblying 108 probes in region: 446\n",
      "--- assemblying 181 probes in region: 447\n",
      "--- assemblying 127 probes in region: 448\n",
      "--- assemblying 228 probes in region: 449\n",
      "--- assemblying 268 probes in region: 450\n",
      "--- assemblying 258 probes in region: 451\n",
      "--- assemblying 197 probes in region: 452\n",
      "--- assemblying 67 probes in region: 453\n",
      "--- assemblying 145 probes in region: 454\n",
      "--- assemblying 142 probes in region: 455\n",
      "--- assemblying 217 probes in region: 456\n",
      "--- assemblying 207 probes in region: 457\n",
      "--- assemblying 236 probes in region: 458\n",
      "--- assemblying 180 probes in region: 459\n",
      "--- assemblying 202 probes in region: 460\n",
      "--- assemblying 209 probes in region: 461\n",
      "--- assemblying 170 probes in region: 462\n",
      "--- assemblying 143 probes in region: 463\n",
      "--- assemblying 211 probes in region: 464\n",
      "--- assemblying 300 probes in region: 465\n",
      "--- assemblying 189 probes in region: 466\n",
      "--- assemblying 200 probes in region: 467\n",
      "--- assemblying 222 probes in region: 468\n",
      "--- assemblying 260 probes in region: 469\n",
      "--- assemblying 131 probes in region: 470\n",
      "--- assemblying 199 probes in region: 471\n",
      "--- assemblying 203 probes in region: 472\n",
      "--- assemblying 203 probes in region: 473\n",
      "--- assemblying 235 probes in region: 474\n",
      "--- assemblying 263 probes in region: 475\n",
      "--- assemblying 236 probes in region: 476\n",
      "--- assemblying 215 probes in region: 477\n",
      "--- assemblying 189 probes in region: 478\n",
      "--- assemblying 287 probes in region: 479\n",
      "--- assemblying 257 probes in region: 480\n",
      "--- assemblying 269 probes in region: 481\n",
      "--- assemblying 230 probes in region: 482\n",
      "--- assemblying 220 probes in region: 483\n",
      "--- assemblying 265 probes in region: 484\n",
      "--- assemblying 255 probes in region: 485\n",
      "--- assemblying 294 probes in region: 486\n",
      "--- assemblying 300 probes in region: 487\n",
      "--- assemblying 216 probes in region: 489\n",
      "--- assemblying 300 probes in region: 490\n",
      "--- assemblying 276 probes in region: 491\n",
      "--- assemblying 229 probes in region: 492\n",
      "--- assemblying 239 probes in region: 493\n",
      "--- assemblying 293 probes in region: 494\n",
      "--- assemblying 259 probes in region: 495\n",
      "--- assemblying 167 probes in region: 496\n",
      "--- assemblying 228 probes in region: 497\n",
      "--- assemblying 262 probes in region: 498\n",
      "--- assemblying 162 probes in region: 499\n",
      "--- assemblying 142 probes in region: 500\n",
      "--- assemblying 300 probes in region: 501\n",
      "--- assemblying 269 probes in region: 502\n",
      "--- assemblying 300 probes in region: 503\n",
      "--- assemblying 220 probes in region: 504\n",
      "--- assemblying 300 probes in region: 505\n",
      "--- assemblying 300 probes in region: 506\n",
      "--- assemblying 202 probes in region: 507\n",
      "--- assemblying 221 probes in region: 508\n",
      "--- assemblying 272 probes in region: 509\n",
      "--- assemblying 300 probes in region: 510\n",
      "--- assemblying 184 probes in region: 511\n",
      "--- assemblying 197 probes in region: 512\n",
      "--- assemblying 227 probes in region: 516\n",
      "--- assemblying 262 probes in region: 517\n",
      "--- assemblying 264 probes in region: 518\n",
      "--- assemblying 188 probes in region: 519\n",
      "--- assemblying 212 probes in region: 520\n",
      "--- assemblying 265 probes in region: 521\n",
      "--- assemblying 187 probes in region: 522\n",
      "--- assemblying 152 probes in region: 523\n",
      "--- assemblying 196 probes in region: 524\n",
      "--- assemblying 216 probes in region: 525\n",
      "--- assemblying 223 probes in region: 526\n",
      "--- assemblying 255 probes in region: 527\n",
      "--- assemblying 249 probes in region: 528\n",
      "--- assemblying 269 probes in region: 529\n",
      "--- assemblying 223 probes in region: 530\n",
      "--- assemblying 201 probes in region: 531\n",
      "--- assemblying 232 probes in region: 532\n",
      "--- assemblying 201 probes in region: 533\n",
      "--- assemblying 181 probes in region: 534\n",
      "--- assemblying 116 probes in region: 535\n",
      "--- assemblying 259 probes in region: 536\n",
      "--- assemblying 231 probes in region: 537\n",
      "--- assemblying 214 probes in region: 538\n",
      "--- assemblying 242 probes in region: 539\n",
      "--- assemblying 179 probes in region: 540\n",
      "--- assemblying 248 probes in region: 541\n",
      "--- assemblying 227 probes in region: 542\n",
      "--- assemblying 300 probes in region: 543\n",
      "--- assemblying 276 probes in region: 544\n",
      "--- assemblying 267 probes in region: 545\n",
      "--- assemblying 300 probes in region: 546\n",
      "--- assemblying 191 probes in region: 548\n",
      "--- assemblying 206 probes in region: 549\n",
      "--- assemblying 300 probes in region: 550\n",
      "--- assemblying 90 probes in region: 551\n",
      "--- assemblying 264 probes in region: 552\n",
      "--- assemblying 300 probes in region: 554\n",
      "--- assemblying 241 probes in region: 555\n",
      "--- assemblying 234 probes in region: 556\n",
      "--- assemblying 273 probes in region: 557\n",
      "--- assemblying 251 probes in region: 558\n",
      "--- assemblying 221 probes in region: 559\n",
      "--- assemblying 261 probes in region: 560\n",
      "--- assemblying 240 probes in region: 561\n",
      "--- assemblying 249 probes in region: 562\n",
      "--- assemblying 193 probes in region: 563\n",
      "--- assemblying 270 probes in region: 564\n",
      "--- assemblying 218 probes in region: 565\n",
      "--- assemblying 169 probes in region: 566\n",
      "--- assemblying 229 probes in region: 567\n",
      "--- assemblying 222 probes in region: 568\n",
      "--- assemblying 292 probes in region: 569\n",
      "--- assemblying 104 probes in region: 570\n",
      "--- assemblying 300 probes in region: 571\n",
      "--- assemblying 280 probes in region: 572\n",
      "--- assemblying 206 probes in region: 573\n",
      "--- assemblying 258 probes in region: 574\n",
      "--- assemblying 204 probes in region: 575\n",
      "--- assemblying 291 probes in region: 576\n",
      "--- assemblying 207 probes in region: 577\n",
      "--- assemblying 246 probes in region: 578\n",
      "--- assemblying 151 probes in region: 579\n",
      "--- assemblying 202 probes in region: 580\n",
      "--- assemblying 250 probes in region: 581\n",
      "--- assemblying 199 probes in region: 582\n",
      "--- assemblying 181 probes in region: 583\n",
      "--- assemblying 139 probes in region: 584\n",
      "--- assemblying 241 probes in region: 585\n",
      "--- assemblying 218 probes in region: 586\n",
      "--- assemblying 156 probes in region: 587\n",
      "--- assemblying 177 probes in region: 588\n",
      "--- assemblying 275 probes in region: 589\n",
      "--- assemblying 225 probes in region: 590\n",
      "--- assemblying 283 probes in region: 591\n",
      "--- assemblying 144 probes in region: 592\n",
      "--- assemblying 300 probes in region: 593\n",
      "--- assemblying 155 probes in region: 594\n",
      "--- assemblying 180 probes in region: 595\n",
      "--- assemblying 254 probes in region: 596\n",
      "--- assemblying 277 probes in region: 597\n",
      "--- assemblying 226 probes in region: 598\n",
      "--- assemblying 300 probes in region: 599\n",
      "--- assemblying 294 probes in region: 600\n",
      "--- assemblying 299 probes in region: 601\n",
      "--- assemblying 261 probes in region: 602\n",
      "--- assemblying 251 probes in region: 603\n",
      "--- assemblying 187 probes in region: 605\n",
      "--- assemblying 293 probes in region: 606\n",
      "--- assemblying 281 probes in region: 607\n",
      "--- assemblying 300 probes in region: 608\n",
      "--- assemblying 300 probes in region: 609\n",
      "--- assemblying 300 probes in region: 611\n",
      "--- assemblying 300 probes in region: 612\n",
      "--- assemblying 200 probes in region: 613\n",
      "--- assemblying 277 probes in region: 614\n",
      "--- assemblying 258 probes in region: 615\n",
      "--- assemblying 173 probes in region: 616\n",
      "--- assemblying 254 probes in region: 617\n",
      "--- assemblying 192 probes in region: 618\n",
      "--- assemblying 282 probes in region: 619\n",
      "--- assemblying 196 probes in region: 620\n",
      "--- assemblying 248 probes in region: 621\n",
      "--- assemblying 247 probes in region: 622\n",
      "--- assemblying 185 probes in region: 623\n",
      "--- assemblying 132 probes in region: 624\n",
      "--- assemblying 168 probes in region: 625\n",
      "--- assemblying 118 probes in region: 626\n",
      "--- assemblying 253 probes in region: 627\n",
      "--- assemblying 273 probes in region: 628\n",
      "--- assemblying 300 probes in region: 629\n",
      "--- assemblying 257 probes in region: 645\n",
      "--- assemblying 241 probes in region: 646\n",
      "--- assemblying 219 probes in region: 647\n",
      "--- assemblying 215 probes in region: 648\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 262 probes in region: 649\n",
      "--- assemblying 241 probes in region: 650\n",
      "--- assemblying 146 probes in region: 651\n",
      "--- assemblying 300 probes in region: 652\n",
      "--- assemblying 300 probes in region: 653\n",
      "--- assemblying 271 probes in region: 654\n",
      "--- assemblying 300 probes in region: 655\n",
      "--- assemblying 292 probes in region: 656\n",
      "--- assemblying 186 probes in region: 657\n",
      "--- assemblying 300 probes in region: 658\n",
      "--- assemblying 54 probes in region: 659\n",
      "--- assemblying 278 probes in region: 660\n",
      "--- assemblying 184 probes in region: 662\n",
      "--- assemblying 205 probes in region: 663\n",
      "--- assemblying 255 probes in region: 664\n",
      "--- assemblying 266 probes in region: 665\n",
      "--- assemblying 273 probes in region: 666\n",
      "--- assemblying 172 probes in region: 667\n",
      "--- assemblying 300 probes in region: 668\n",
      "--- assemblying 300 probes in region: 669\n",
      "--- assemblying 300 probes in region: 670\n",
      "--- assemblying 199 probes in region: 671\n",
      "--- assemblying 171 probes in region: 672\n",
      "--- assemblying 120 probes in region: 673\n",
      "--- assemblying 258 probes in region: 674\n",
      "--- assemblying 300 probes in region: 675\n",
      "--- assemblying 300 probes in region: 676\n",
      "--- assemblying 300 probes in region: 677\n",
      "--- assemblying 300 probes in region: 678\n",
      "--- assemblying 300 probes in region: 679\n",
      "--- assemblying 294 probes in region: 680\n",
      "--- assemblying 300 probes in region: 681\n",
      "--- assemblying 147 probes in region: 682\n",
      "--- assemblying 285 probes in region: 683\n",
      "--- assemblying 253 probes in region: 684\n",
      "--- assemblying 300 probes in region: 685\n",
      "--- assemblying 266 probes in region: 686\n",
      "--- assemblying 78 probes in region: 687\n",
      "--- assemblying 278 probes in region: 688\n",
      "--- assemblying 160 probes in region: 689\n",
      "--- assemblying 289 probes in region: 690\n",
      "--- assemblying 270 probes in region: 691\n",
      "--- assemblying 189 probes in region: 692\n",
      "--- assemblying 246 probes in region: 693\n",
      "--- assemblying 236 probes in region: 694\n",
      "--- assemblying 246 probes in region: 695\n",
      "--- assemblying 170 probes in region: 696\n",
      "--- assemblying 220 probes in region: 697\n",
      "--- assemblying 227 probes in region: 698\n",
      "--- assemblying 66 probes in region: 699\n",
      "--- assemblying 289 probes in region: 701\n",
      "--- assemblying 291 probes in region: 702\n",
      "--- assemblying 181 probes in region: 703\n",
      "--- assemblying 284 probes in region: 704\n",
      "--- assemblying 292 probes in region: 705\n",
      "--- assemblying 246 probes in region: 706\n",
      "--- assemblying 182 probes in region: 707\n",
      "--- assemblying 213 probes in region: 708\n",
      "--- assemblying 289 probes in region: 709\n",
      "--- assemblying 234 probes in region: 710\n",
      "--- assemblying 290 probes in region: 711\n",
      "--- assemblying 201 probes in region: 712\n",
      "--- assemblying 236 probes in region: 713\n",
      "--- assemblying 196 probes in region: 714\n",
      "--- assemblying 300 probes in region: 715\n",
      "--- assemblying 136 probes in region: 716\n",
      "--- assemblying 282 probes in region: 717\n",
      "--- assemblying 300 probes in region: 718\n",
      "--- assemblying 300 probes in region: 719\n",
      "--- assemblying 218 probes in region: 720\n",
      "--- assemblying 270 probes in region: 721\n",
      "--- assemblying 300 probes in region: 722\n",
      "--- assemblying 73 probes in region: 723\n",
      "--- assemblying 288 probes in region: 724\n",
      "--- assemblying 271 probes in region: 725\n",
      "--- assemblying 209 probes in region: 726\n",
      "--- assemblying 138 probes in region: 727\n",
      "--- assemblying 300 probes in region: 728\n",
      "--- assemblying 200 probes in region: 729\n",
      "--- assemblying 300 probes in region: 730\n",
      "--- assemblying 174 probes in region: 731\n",
      "--- assemblying 296 probes in region: 732\n",
      "--- assemblying 274 probes in region: 733\n",
      "--- assemblying 225 probes in region: 734\n",
      "--- assemblying 206 probes in region: 735\n",
      "--- assemblying 300 probes in region: 736\n",
      "--- assemblying 252 probes in region: 737\n",
      "--- assemblying 222 probes in region: 738\n",
      "--- assemblying 229 probes in region: 739\n",
      "--- assemblying 249 probes in region: 740\n",
      "--- assemblying 266 probes in region: 741\n",
      "--- assemblying 300 probes in region: 742\n",
      "--- assemblying 300 probes in region: 743\n",
      "--- assemblying 300 probes in region: 744\n",
      "--- assemblying 300 probes in region: 745\n",
      "--- assemblying 254 probes in region: 748\n",
      "--- assemblying 199 probes in region: 749\n",
      "--- assemblying 287 probes in region: 750\n",
      "--- assemblying 293 probes in region: 751\n",
      "--- assemblying 300 probes in region: 752\n",
      "--- assemblying 145 probes in region: 753\n",
      "--- assemblying 300 probes in region: 754\n",
      "--- assemblying 59 probes in region: 755\n",
      "--- assemblying 300 probes in region: 756\n",
      "--- assemblying 192 probes in region: 757\n",
      "--- assemblying 141 probes in region: 758\n",
      "--- assemblying 300 probes in region: 759\n",
      "--- assemblying 220 probes in region: 760\n",
      "--- assemblying 222 probes in region: 761\n",
      "--- assemblying 141 probes in region: 762\n",
      "--- assemblying 270 probes in region: 763\n",
      "--- assemblying 261 probes in region: 764\n",
      "--- assemblying 191 probes in region: 765\n",
      "--- assemblying 254 probes in region: 766\n",
      "--- assemblying 199 probes in region: 767\n",
      "--- assemblying 278 probes in region: 768\n",
      "--- assemblying 160 probes in region: 769\n",
      "--- assemblying 81 probes in region: 770\n",
      "--- assemblying 202 probes in region: 774\n",
      "--- assemblying 202 probes in region: 775\n",
      "--- assemblying 300 probes in region: 776\n",
      "--- assemblying 300 probes in region: 777\n",
      "--- assemblying 252 probes in region: 778\n",
      "--- assemblying 208 probes in region: 779\n",
      "--- assemblying 300 probes in region: 780\n",
      "--- assemblying 206 probes in region: 782\n",
      "--- assemblying 286 probes in region: 783\n",
      "--- assemblying 196 probes in region: 784\n",
      "--- assemblying 295 probes in region: 785\n",
      "--- assemblying 234 probes in region: 786\n",
      "--- assemblying 293 probes in region: 787\n",
      "--- assemblying 182 probes in region: 788\n",
      "--- assemblying 209 probes in region: 789\n",
      "--- assemblying 91 probes in region: 790\n",
      "--- assemblying 164 probes in region: 791\n",
      "--- assemblying 168 probes in region: 792\n",
      "--- assemblying 231 probes in region: 793\n",
      "--- assemblying 298 probes in region: 794\n",
      "--- assemblying 126 probes in region: 795\n",
      "--- assemblying 289 probes in region: 796\n",
      "--- assemblying 249 probes in region: 797\n",
      "--- assemblying 244 probes in region: 798\n",
      "--- assemblying 289 probes in region: 799\n",
      "--- assemblying 196 probes in region: 800\n",
      "--- assemblying 300 probes in region: 801\n",
      "--- assemblying 257 probes in region: 802\n",
      "--- assemblying 289 probes in region: 803\n",
      "--- assemblying 220 probes in region: 804\n",
      "--- assemblying 168 probes in region: 805\n",
      "--- assemblying 291 probes in region: 806\n",
      "--- assemblying 233 probes in region: 807\n",
      "--- assemblying 197 probes in region: 808\n",
      "--- assemblying 300 probes in region: 809\n",
      "--- assemblying 160 probes in region: 810\n",
      "--- assemblying 234 probes in region: 811\n",
      "--- assemblying 300 probes in region: 812\n",
      "--- assemblying 187 probes in region: 813\n",
      "--- assemblying 275 probes in region: 814\n",
      "--- assemblying 300 probes in region: 815\n",
      "--- assemblying 209 probes in region: 816\n",
      "--- assemblying 204 probes in region: 817\n",
      "--- assemblying 268 probes in region: 819\n",
      "--- assemblying 255 probes in region: 820\n",
      "--- assemblying 142 probes in region: 821\n",
      "--- assemblying 257 probes in region: 822\n",
      "--- assemblying 129 probes in region: 823\n",
      "--- assemblying 288 probes in region: 824\n",
      "--- assemblying 225 probes in region: 825\n",
      "--- assemblying 263 probes in region: 826\n",
      "--- assemblying 283 probes in region: 827\n",
      "--- assemblying 300 probes in region: 828\n",
      "--- assemblying 237 probes in region: 829\n",
      "--- assemblying 168 probes in region: 832\n",
      "--- assemblying 204 probes in region: 833\n",
      "--- assemblying 300 probes in region: 834\n",
      "--- assemblying 236 probes in region: 835\n",
      "--- assemblying 164 probes in region: 836\n",
      "--- assemblying 300 probes in region: 837\n",
      "--- assemblying 226 probes in region: 838\n",
      "--- assemblying 194 probes in region: 839\n",
      "--- assemblying 264 probes in region: 840\n",
      "--- assemblying 300 probes in region: 841\n",
      "--- assemblying 148 probes in region: 843\n",
      "--- assemblying 256 probes in region: 844\n",
      "--- assemblying 300 probes in region: 845\n",
      "--- assemblying 261 probes in region: 846\n",
      "--- assemblying 174 probes in region: 847\n",
      "--- assemblying 294 probes in region: 848\n",
      "--- assemblying 123 probes in region: 849\n",
      "--- assemblying 181 probes in region: 850\n",
      "--- assemblying 241 probes in region: 851\n",
      "--- assemblying 279 probes in region: 852\n",
      "--- assemblying 194 probes in region: 853\n",
      "--- assemblying 215 probes in region: 854\n",
      "--- assemblying 199 probes in region: 855\n",
      "--- assemblying 163 probes in region: 856\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 300 probes in region: 857\n",
      "--- assemblying 252 probes in region: 858\n",
      "--- assemblying 240 probes in region: 859\n",
      "--- assemblying 144 probes in region: 860\n",
      "--- assemblying 300 probes in region: 861\n",
      "--- assemblying 233 probes in region: 862\n",
      "--- assemblying 282 probes in region: 863\n",
      "--- assemblying 250 probes in region: 864\n",
      "--- assemblying 223 probes in region: 865\n",
      "--- assemblying 216 probes in region: 866\n",
      "--- assemblying 164 probes in region: 867\n",
      "--- assemblying 289 probes in region: 868\n",
      "--- assemblying 300 probes in region: 869\n",
      "--- assemblying 300 probes in region: 870\n",
      "--- assemblying 300 probes in region: 871\n",
      "--- assemblying 300 probes in region: 872\n",
      "--- assemblying 265 probes in region: 873\n",
      "--- assemblying 300 probes in region: 874\n",
      "--- assemblying 212 probes in region: 875\n",
      "--- assemblying 262 probes in region: 876\n",
      "--- assemblying 268 probes in region: 877\n",
      "--- assemblying 300 probes in region: 878\n",
      "--- assemblying 215 probes in region: 888\n",
      "--- assemblying 279 probes in region: 889\n",
      "--- assemblying 205 probes in region: 890\n",
      "--- assemblying 169 probes in region: 891\n",
      "--- assemblying 300 probes in region: 892\n",
      "--- assemblying 252 probes in region: 893\n",
      "--- assemblying 300 probes in region: 894\n",
      "--- assemblying 294 probes in region: 895\n",
      "--- assemblying 158 probes in region: 896\n",
      "--- assemblying 286 probes in region: 897\n",
      "--- assemblying 182 probes in region: 898\n",
      "--- assemblying 211 probes in region: 899\n",
      "--- assemblying 251 probes in region: 900\n",
      "--- assemblying 230 probes in region: 901\n",
      "--- assemblying 271 probes in region: 902\n",
      "--- assemblying 144 probes in region: 903\n",
      "--- assemblying 300 probes in region: 904\n",
      "--- assemblying 300 probes in region: 905\n",
      "--- assemblying 194 probes in region: 906\n",
      "--- assemblying 143 probes in region: 907\n",
      "--- assemblying 216 probes in region: 908\n",
      "--- assemblying 181 probes in region: 909\n",
      "--- assemblying 155 probes in region: 910\n",
      "--- assemblying 187 probes in region: 911\n",
      "--- assemblying 230 probes in region: 912\n",
      "--- assemblying 169 probes in region: 913\n",
      "--- assemblying 176 probes in region: 914\n",
      "--- assemblying 166 probes in region: 915\n",
      "--- assemblying 250 probes in region: 916\n",
      "--- assemblying 250 probes in region: 917\n",
      "--- assemblying 259 probes in region: 918\n",
      "--- assemblying 157 probes in region: 919\n",
      "--- assemblying 187 probes in region: 920\n",
      "--- assemblying 173 probes in region: 921\n",
      "--- assemblying 174 probes in region: 922\n",
      "--- assemblying 141 probes in region: 923\n",
      "--- assemblying 236 probes in region: 924\n",
      "--- assemblying 214 probes in region: 925\n",
      "--- assemblying 293 probes in region: 926\n",
      "--- assemblying 300 probes in region: 927\n",
      "--- assemblying 300 probes in region: 928\n",
      "--- assemblying 261 probes in region: 929\n",
      "--- assemblying 266 probes in region: 930\n",
      "--- assemblying 249 probes in region: 931\n",
      "--- assemblying 297 probes in region: 932\n",
      "--- assemblying 224 probes in region: 933\n",
      "--- assemblying 209 probes in region: 934\n",
      "--- assemblying 300 probes in region: 935\n",
      "--- assemblying 160 probes in region: 949\n",
      "--- assemblying 178 probes in region: 950\n",
      "--- assemblying 188 probes in region: 951\n",
      "--- assemblying 190 probes in region: 952\n",
      "--- assemblying 245 probes in region: 953\n",
      "--- assemblying 90 probes in region: 954\n",
      "--- assemblying 174 probes in region: 955\n",
      "--- assemblying 265 probes in region: 956\n",
      "--- assemblying 258 probes in region: 957\n",
      "--- assemblying 300 probes in region: 958\n",
      "--- assemblying 213 probes in region: 959\n",
      "--- assemblying 279 probes in region: 960\n",
      "--- assemblying 237 probes in region: 961\n",
      "--- assemblying 226 probes in region: 962\n",
      "--- assemblying 195 probes in region: 963\n",
      "--- assemblying 149 probes in region: 964\n",
      "--- assemblying 191 probes in region: 965\n",
      "--- assemblying 161 probes in region: 966\n",
      "--- assemblying 229 probes in region: 967\n",
      "--- assemblying 228 probes in region: 968\n",
      "--- assemblying 156 probes in region: 969\n",
      "--- assemblying 110 probes in region: 970\n",
      "--- assemblying 173 probes in region: 971\n",
      "--- assemblying 300 probes in region: 972\n",
      "--- assemblying 270 probes in region: 973\n",
      "--- assemblying 209 probes in region: 974\n",
      "--- assemblying 300 probes in region: 975\n",
      "--- assemblying 194 probes in region: 976\n",
      "--- assemblying 297 probes in region: 977\n",
      "--- assemblying 130 probes in region: 978\n",
      "--- assemblying 255 probes in region: 979\n",
      "--- assemblying 173 probes in region: 980\n",
      "--- assemblying 300 probes in region: 981\n",
      "--- assemblying 171 probes in region: 982\n",
      "--- assemblying 236 probes in region: 983\n",
      "--- assemblying 269 probes in region: 984\n",
      "--- assemblying 300 probes in region: 985\n",
      "--- assemblying 233 probes in region: 986\n",
      "--- assemblying 300 probes in region: 987\n",
      "--- assemblying 300 probes in region: 988\n",
      "--- assemblying 154 probes in region: 989\n",
      "--- assemblying 281 probes in region: 990\n",
      "--- assemblying 247 probes in region: 991\n",
      "--- assemblying 237 probes in region: 992\n",
      "--- assemblying 153 probes in region: 993\n",
      "--- assemblying 286 probes in region: 994\n",
      "--- assemblying 300 probes in region: 995\n",
      "--- assemblying 255 probes in region: 996\n",
      "--- assemblying 300 probes in region: 997\n",
      "--- assemblying 300 probes in region: 999\n",
      "--- assemblying 300 probes in region: 1000\n",
      "--- assemblying 300 probes in region: 1001\n",
      "--- assemblying 300 probes in region: 1002\n",
      "--- assemblying 300 probes in region: 1003\n",
      "--- assemblying 283 probes in region: 1004\n",
      "--- assemblying 300 probes in region: 1005\n",
      "--- assemblying 52 probes in region: 1020\n",
      "--- assemblying 217 probes in region: 1023\n",
      "--- assemblying 242 probes in region: 1024\n",
      "--- assemblying 300 probes in region: 1025\n",
      "--- assemblying 235 probes in region: 1029\n",
      "--- assemblying 209 probes in region: 1030\n",
      "--- assemblying 290 probes in region: 1031\n",
      "--- assemblying 275 probes in region: 1032\n",
      "--- assemblying 234 probes in region: 1033\n",
      "--- assemblying 183 probes in region: 1034\n",
      "--- assemblying 201 probes in region: 1035\n",
      "--- assemblying 130 probes in region: 1036\n",
      "--- assemblying 237 probes in region: 1037\n",
      "--- assemblying 227 probes in region: 1038\n",
      "--- assemblying 246 probes in region: 1039\n",
      "--- assemblying 300 probes in region: 1040\n",
      "--- assemblying 254 probes in region: 1041\n",
      "--- assemblying 300 probes in region: 1042\n",
      "--- assemblying 122 probes in region: 1043\n",
      "--- assemblying 253 probes in region: 1044\n",
      "--- assemblying 300 probes in region: 1045\n",
      "--- assemblying 263 probes in region: 1046\n",
      "--- assemblying 300 probes in region: 1047\n",
      "--- assemblying 296 probes in region: 1048\n",
      "--- assemblying 176 probes in region: 1049\n",
      "--- assemblying 267 probes in region: 1050\n",
      "--- assemblying 300 probes in region: 1051\n",
      "--- assemblying 135 probes in region: 1052\n",
      "--- assemblying 300 probes in region: 1053\n",
      "--- assemblying 300 probes in region: 1054\n",
      "--- assemblying 268 probes in region: 1055\n",
      "--- assemblying 300 probes in region: 1056\n",
      "--- assemblying 155 probes in region: 1057\n",
      "--- assemblying 131 probes in region: 1058\n",
      "--- assemblying 164 probes in region: 1059\n",
      "--- assemblying 300 probes in region: 1060\n",
      "--- assemblying 300 probes in region: 1061\n",
      "--- assemblying 240 probes in region: 1062\n",
      "--- assemblying 266 probes in region: 1063\n",
      "--- assemblying 300 probes in region: 1064\n",
      "--- assemblying 300 probes in region: 1065\n",
      "--- assemblying 207 probes in region: 1066\n",
      "--- assemblying 204 probes in region: 1067\n",
      "--- assemblying 147 probes in region: 1068\n",
      "--- assemblying 160 probes in region: 1069\n",
      "--- assemblying 250 probes in region: 1070\n",
      "--- assemblying 291 probes in region: 1071\n",
      "--- assemblying 259 probes in region: 1072\n",
      "--- assemblying 263 probes in region: 1073\n",
      "--- assemblying 273 probes in region: 1074\n",
      "--- assemblying 256 probes in region: 1075\n",
      "--- assemblying 293 probes in region: 1076\n",
      "--- assemblying 88 probes in region: 1077\n",
      "--- assemblying 241 probes in region: 1078\n",
      "--- assemblying 300 probes in region: 1079\n",
      "--- assemblying 300 probes in region: 1080\n",
      "--- assemblying 298 probes in region: 1081\n",
      "--- assemblying 263 probes in region: 1082\n",
      "--- assemblying 300 probes in region: 1083\n",
      "--- assemblying 300 probes in region: 1085\n",
      "--- assemblying 300 probes in region: 1087\n",
      "--- assemblying 207 probes in region: 1089\n",
      "--- assemblying 261 probes in region: 1090\n",
      "--- assemblying 226 probes in region: 1091\n",
      "--- assemblying 300 probes in region: 1092\n",
      "--- assemblying 203 probes in region: 1093\n",
      "--- assemblying 300 probes in region: 1094\n",
      "--- assemblying 230 probes in region: 1097\n",
      "--- assemblying 223 probes in region: 1105\n",
      "--- assemblying 189 probes in region: 1106\n",
      "--- assemblying 267 probes in region: 1107\n",
      "--- assemblying 295 probes in region: 1108\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 184 probes in region: 1109\n",
      "--- assemblying 300 probes in region: 1110\n",
      "--- assemblying 300 probes in region: 1111\n",
      "--- assemblying 300 probes in region: 1112\n",
      "--- assemblying 266 probes in region: 1113\n",
      "--- assemblying 217 probes in region: 1114\n",
      "--- assemblying 253 probes in region: 1115\n",
      "--- assemblying 184 probes in region: 1116\n",
      "--- assemblying 288 probes in region: 1117\n",
      "--- assemblying 231 probes in region: 1118\n",
      "--- assemblying 300 probes in region: 1119\n",
      "--- assemblying 228 probes in region: 1120\n",
      "--- assemblying 298 probes in region: 1121\n",
      "--- assemblying 227 probes in region: 1122\n",
      "--- assemblying 267 probes in region: 1123\n",
      "--- assemblying 201 probes in region: 1124\n",
      "--- assemblying 208 probes in region: 1125\n",
      "--- assemblying 300 probes in region: 1126\n",
      "--- assemblying 241 probes in region: 1127\n",
      "--- assemblying 235 probes in region: 1128\n",
      "--- assemblying 300 probes in region: 1129\n",
      "--- assemblying 300 probes in region: 1130\n",
      "--- assemblying 272 probes in region: 1131\n",
      "--- assemblying 290 probes in region: 1132\n",
      "--- assemblying 205 probes in region: 1133\n",
      "--- assemblying 300 probes in region: 1134\n",
      "--- assemblying 268 probes in region: 1135\n",
      "--- assemblying 294 probes in region: 1136\n",
      "--- assemblying 196 probes in region: 1137\n",
      "--- assemblying 222 probes in region: 1138\n",
      "--- assemblying 256 probes in region: 1139\n",
      "--- assemblying 242 probes in region: 1140\n",
      "--- assemblying 300 probes in region: 1141\n",
      "--- assemblying 263 probes in region: 1142\n",
      "--- assemblying 206 probes in region: 1143\n",
      "--- assemblying 122 probes in region: 1144\n",
      "--- assemblying 215 probes in region: 1145\n",
      "--- assemblying 94 probes in region: 1146\n",
      "--- assemblying 257 probes in region: 1147\n",
      "--- assemblying 101 probes in region: 1148\n",
      "--- assemblying 194 probes in region: 1152\n",
      "--- assemblying 269 probes in region: 1153\n",
      "--- assemblying 54 probes in region: 1154\n",
      "--- assemblying 254 probes in region: 1155\n",
      "--- assemblying 300 probes in region: 1156\n",
      "--- assemblying 276 probes in region: 1157\n",
      "--- assemblying 300 probes in region: 1160\n",
      "--- assemblying 222 probes in region: 1162\n",
      "--- assemblying 272 probes in region: 1163\n",
      "--- assemblying 270 probes in region: 1165\n",
      "--- assemblying 300 probes in region: 1166\n",
      "--- assemblying 192 probes in region: 1167\n",
      "--- assemblying 300 probes in region: 1168\n",
      "--- assemblying 185 probes in region: 1169\n",
      "--- assemblying 232 probes in region: 1170\n",
      "--- assemblying 268 probes in region: 1171\n",
      "--- assemblying 300 probes in region: 1172\n",
      "--- assemblying 96 probes in region: 1173\n",
      "--- assemblying 128 probes in region: 1174\n",
      "--- assemblying 212 probes in region: 1176\n",
      "--- assemblying 241 probes in region: 1177\n",
      "--- assemblying 300 probes in region: 1178\n",
      "--- assemblying 197 probes in region: 1179\n",
      "--- assemblying 213 probes in region: 1180\n",
      "--- assemblying 271 probes in region: 1181\n",
      "--- assemblying 291 probes in region: 1182\n",
      "--- assemblying 300 probes in region: 1183\n",
      "--- assemblying 300 probes in region: 1184\n",
      "--- assemblying 265 probes in region: 1186\n",
      "--- assemblying 285 probes in region: 1187\n",
      "--- assemblying 77 probes in region: 1188\n",
      "--- assemblying 171 probes in region: 1189\n",
      "--- assemblying 187 probes in region: 1190\n",
      "--- assemblying 194 probes in region: 1191\n",
      "--- assemblying 242 probes in region: 1192\n",
      "--- assemblying 279 probes in region: 1193\n",
      "--- assemblying 291 probes in region: 1194\n",
      "--- assemblying 288 probes in region: 1195\n",
      "--- assemblying 277 probes in region: 1196\n",
      "--- assemblying 184 probes in region: 1197\n",
      "--- assemblying 191 probes in region: 1203\n",
      "--- assemblying 300 probes in region: 1204\n",
      "--- assemblying 300 probes in region: 1205\n",
      "--- assemblying 108 probes in region: 1206\n",
      "--- assemblying 213 probes in region: 1207\n",
      "--- assemblying 208 probes in region: 1208\n",
      "--- assemblying 169 probes in region: 1209\n",
      "--- assemblying 270 probes in region: 1210\n",
      "--- assemblying 300 probes in region: 1211\n",
      "--- assemblying 278 probes in region: 1212\n",
      "--- assemblying 293 probes in region: 1213\n",
      "--- assemblying 300 probes in region: 1214\n",
      "--- assemblying 221 probes in region: 1215\n",
      "--- assemblying 216 probes in region: 1216\n",
      "--- assemblying 190 probes in region: 1217\n",
      "--- assemblying 235 probes in region: 1218\n",
      "--- assemblying 300 probes in region: 1219\n",
      "--- assemblying 300 probes in region: 1220\n",
      "--- assemblying 300 probes in region: 1221\n",
      "--- assemblying 179 probes in region: 1222\n",
      "--- assemblying 231 probes in region: 1224\n",
      "--- assemblying 282 probes in region: 1225\n",
      "--- assemblying 238 probes in region: 1226\n",
      "--- assemblying 215 probes in region: 1227\n",
      "--- assemblying 167 probes in region: 1228\n",
      "--- assemblying 235 probes in region: 1229\n",
      "--- assemblying 223 probes in region: 1230\n",
      "--- assemblying 241 probes in region: 1231\n",
      "--- assemblying 237 probes in region: 1232\n",
      "--- assemblying 223 probes in region: 1233\n",
      "--- assemblying 169 probes in region: 1234\n",
      "--- assemblying 241 probes in region: 1235\n",
      "--- assemblying 271 probes in region: 1236\n",
      "--- assemblying 228 probes in region: 1237\n",
      "--- assemblying 287 probes in region: 1238\n",
      "--- assemblying 249 probes in region: 1239\n",
      "--- assemblying 297 probes in region: 1240\n",
      "--- assemblying 300 probes in region: 1241\n",
      "--- assemblying 279 probes in region: 1242\n",
      "--- assemblying 297 probes in region: 1243\n",
      "--- assemblying 260 probes in region: 1244\n",
      "--- assemblying 283 probes in region: 1245\n",
      "--- assemblying 300 probes in region: 1246\n",
      "--- assemblying 234 probes in region: 1247\n",
      "--- assemblying 160 probes in region: 1248\n",
      "--- assemblying 237 probes in region: 1249\n",
      "--- assemblying 300 probes in region: 1250\n",
      "--- assemblying 181 probes in region: 1251\n",
      "--- assemblying 181 probes in region: 1252\n",
      "--- assemblying 246 probes in region: 1253\n",
      "--- assemblying 178 probes in region: 1254\n",
      "--- assemblying 300 probes in region: 1255\n",
      "--- assemblying 186 probes in region: 1256\n",
      "--- assemblying 173 probes in region: 1257\n",
      "--- assemblying 168 probes in region: 1258\n",
      "--- assemblying 216 probes in region: 1259\n",
      "--- assemblying 300 probes in region: 1260\n",
      "--- assemblying 300 probes in region: 1261\n",
      "--- assemblying 85 probes in region: 1262\n",
      "--- assemblying 189 probes in region: 1263\n",
      "--- assemblying 279 probes in region: 1264\n",
      "--- assemblying 99 probes in region: 1265\n",
      "--- assemblying 51 probes in region: 1269\n",
      "--- assemblying 300 probes in region: 1270\n",
      "--- assemblying 300 probes in region: 1271\n",
      "--- assemblying 300 probes in region: 1272\n",
      "--- assemblying 297 probes in region: 1273\n",
      "--- assemblying 192 probes in region: 1274\n",
      "--- assemblying 300 probes in region: 1275\n",
      "--- assemblying 244 probes in region: 1276\n",
      "--- assemblying 300 probes in region: 1277\n",
      "--- assemblying 150 probes in region: 1278\n",
      "--- assemblying 193 probes in region: 1279\n",
      "--- assemblying 296 probes in region: 1280\n",
      "--- assemblying 275 probes in region: 1281\n",
      "--- assemblying 234 probes in region: 1282\n",
      "--- assemblying 188 probes in region: 1284\n",
      "--- assemblying 206 probes in region: 1285\n",
      "--- assemblying 263 probes in region: 1286\n",
      "--- assemblying 287 probes in region: 1287\n",
      "--- assemblying 239 probes in region: 1288\n",
      "--- assemblying 300 probes in region: 1289\n",
      "--- assemblying 300 probes in region: 1290\n",
      "--- assemblying 273 probes in region: 1291\n",
      "--- assemblying 248 probes in region: 1292\n",
      "--- assemblying 300 probes in region: 1293\n",
      "--- assemblying 90 probes in region: 1294\n",
      "--- assemblying 167 probes in region: 1295\n",
      "--- assemblying 300 probes in region: 1297\n",
      "--- assemblying 282 probes in region: 1298\n",
      "--- assemblying 267 probes in region: 1299\n",
      "--- assemblying 266 probes in region: 1300\n",
      "--- assemblying 252 probes in region: 1301\n",
      "--- assemblying 220 probes in region: 1302\n",
      "--- assemblying 174 probes in region: 1303\n",
      "--- assemblying 268 probes in region: 1304\n",
      "--- assemblying 297 probes in region: 1305\n",
      "--- assemblying 285 probes in region: 1306\n",
      "--- assemblying 271 probes in region: 1307\n",
      "--- assemblying 243 probes in region: 1308\n",
      "--- assemblying 300 probes in region: 1309\n",
      "--- assemblying 285 probes in region: 1310\n",
      "--- assemblying 211 probes in region: 1311\n",
      "--- assemblying 257 probes in region: 1312\n",
      "--- assemblying 201 probes in region: 1313\n",
      "--- assemblying 272 probes in region: 1314\n",
      "--- assemblying 300 probes in region: 1315\n",
      "--- assemblying 300 probes in region: 1316\n",
      "--- assemblying 300 probes in region: 1317\n",
      "--- assemblying 253 probes in region: 1318\n",
      "--- assemblying 300 probes in region: 1319\n",
      "--- assemblying 286 probes in region: 1320\n",
      "--- assemblying 300 probes in region: 1321\n",
      "--- assemblying 300 probes in region: 1322\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- assemblying 300 probes in region: 1323\n",
      "--- assemblying 300 probes in region: 1324\n",
      "--- assemblying 300 probes in region: 1325\n",
      "--- assemblying 129 probes in region: 1330\n",
      "--- assemblying 242 probes in region: 1331\n",
      "--- assemblying 278 probes in region: 1332\n",
      "--- assemblying 136 probes in region: 1333\n",
      "--- assemblying 161 probes in region: 1334\n",
      "--- assemblying 300 probes in region: 1335\n",
      "--- assemblying 266 probes in region: 1336\n",
      "--- assemblying 300 probes in region: 1337\n",
      "--- assemblying 188 probes in region: 1338\n",
      "--- assemblying 238 probes in region: 1339\n",
      "--- assemblying 300 probes in region: 1340\n",
      "--- assemblying 300 probes in region: 1341\n",
      "--- assemblying 261 probes in region: 1342\n",
      "--- assemblying 201 probes in region: 1343\n",
      "--- assemblying 274 probes in region: 1344\n",
      "--- assemblying 300 probes in region: 1345\n",
      "--- assemblying 298 probes in region: 1346\n",
      "--- assemblying 179 probes in region: 1347\n",
      "--- assemblying 130 probes in region: 1348\n",
      "--- assemblying 300 probes in region: 1349\n",
      "--- assemblying 265 probes in region: 1350\n",
      "--- assemblying 300 probes in region: 1351\n",
      "--- assemblying 243 probes in region: 1352\n",
      "--- assemblying 192 probes in region: 1353\n",
      "--- assemblying 278 probes in region: 1354\n",
      "--- assemblying 187 probes in region: 1355\n",
      "--- assemblying 291 probes in region: 1356\n",
      "--- assemblying 248 probes in region: 1357\n",
      "--- assemblying 300 probes in region: 1358\n",
      "--- assemblying 267 probes in region: 1359\n",
      "--- assemblying 300 probes in region: 1360\n",
      "--- assemblying 300 probes in region: 1361\n",
      "--- assemblying 300 probes in region: 1362\n",
      "--- assemblying 300 probes in region: 1363\n",
      "--- assemblying 217 probes in region: 1378\n",
      "--- assemblying 292 probes in region: 1379\n",
      "--- assemblying 192 probes in region: 1380\n",
      "--- assemblying 214 probes in region: 1381\n",
      "--- assemblying 210 probes in region: 1382\n",
      "--- assemblying 230 probes in region: 1383\n",
      "--- assemblying 148 probes in region: 1384\n",
      "--- assemblying 163 probes in region: 1385\n",
      "--- assemblying 128 probes in region: 1386\n",
      "--- assemblying 190 probes in region: 1387\n",
      "--- assemblying 176 probes in region: 1388\n",
      "--- assemblying 251 probes in region: 1389\n",
      "--- assemblying 288 probes in region: 1390\n",
      "--- assemblying 233 probes in region: 1391\n",
      "--- assemblying 244 probes in region: 1392\n",
      "--- assemblying 300 probes in region: 1393\n",
      "--- assemblying 174 probes in region: 1394\n",
      "--- assemblying 282 probes in region: 1395\n",
      "--- assemblying 192 probes in region: 1396\n",
      "--- assemblying 184 probes in region: 1398\n",
      "--- assemblying 268 probes in region: 1399\n",
      "--- assemblying 300 probes in region: 1400\n",
      "--- assemblying 268 probes in region: 1401\n",
      "--- assemblying 300 probes in region: 1402\n",
      "--- assemblying 223 probes in region: 1403\n",
      "--- assemblying 235 probes in region: 1404\n",
      "--- assemblying 300 probes in region: 1405\n",
      "--- assemblying 300 probes in region: 1406\n",
      "--- assemblying 300 probes in region: 1408\n",
      "--- assemblying 300 probes in region: 1409\n",
      "--- assemblying 253 probes in region: 1427\n",
      "--- assemblying 266 probes in region: 1429\n",
      "--- assemblying 240 probes in region: 1430\n",
      "--- assemblying 295 probes in region: 1434\n",
      "--- assemblying 255 probes in region: 1435\n",
      "--- assemblying 298 probes in region: 1436\n",
      "--- assemblying 300 probes in region: 1437\n",
      "--- assemblying 174 probes in region: 1438\n",
      "--- assemblying 198 probes in region: 1439\n",
      "--- assemblying 236 probes in region: 1440\n",
      "--- assemblying 184 probes in region: 1441\n",
      "--- assemblying 248 probes in region: 1442\n",
      "--- assemblying 300 probes in region: 1443\n",
      "--- assemblying 300 probes in region: 1444\n",
      "--- assemblying 300 probes in region: 1445\n",
      "--- assemblying 274 probes in region: 1446\n",
      "--- assemblying 300 probes in region: 1447\n",
      "--- assemblying 267 probes in region: 1448\n",
      "--- assemblying 300 probes in region: 1449\n",
      "--- assemblying 211 probes in region: 1450\n",
      "--- assemblying 255 probes in region: 1451\n",
      "--- assemblying 113 probes in region: 1452\n",
      "--- assemblying 300 probes in region: 1453\n",
      "--- assemblying 277 probes in region: 1454\n",
      "--- assemblying 290 probes in region: 1455\n",
      "--- assemblying 270 probes in region: 1456\n",
      "--- assemblying 281 probes in region: 1457\n",
      "--- assemblying 300 probes in region: 1458\n",
      "--- assemblying 300 probes in region: 1459\n",
      "--- assemblying 143 probes in region: 1461\n",
      "--- assemblying 238 probes in region: 1462\n",
      "--- assemblying 268 probes in region: 1463\n",
      "--- assemblying 281 probes in region: 1464\n",
      "--- assemblying 231 probes in region: 1465\n",
      "--- assemblying 286 probes in region: 1466\n",
      "--- assemblying 193 probes in region: 1467\n",
      "--- assemblying 180 probes in region: 1468\n",
      "--- assemblying 202 probes in region: 1469\n",
      "--- assemblying 194 probes in region: 1470\n",
      "--- assemblying 221 probes in region: 1471\n",
      "--- assemblying 278 probes in region: 1472\n",
      "--- assemblying 117 probes in region: 1473\n",
      "--- assemblying 173 probes in region: 1474\n",
      "--- assemblying 234 probes in region: 1475\n",
      "--- assemblying 197 probes in region: 1476\n",
      "--- assemblying 300 probes in region: 1477\n",
      "--- assemblying 230 probes in region: 1478\n",
      "--- assemblying 216 probes in region: 1479\n",
      "--- assemblying 117 probes in region: 1480\n",
      "--- assemblying 198 probes in region: 1481\n",
      "--- assemblying 212 probes in region: 1482\n",
      "--- assemblying 300 probes in region: 1485\n",
      "--- assemblying 227 probes in region: 1486\n",
      "--- assemblying 250 probes in region: 1487\n",
      "--- assemblying 300 probes in region: 1488\n",
      "--- assemblying 94 probes in region: 1489\n",
      "--- assemblying 128 probes in region: 1490\n",
      "--- assemblying 239 probes in region: 1491\n",
      "--- assemblying 238 probes in region: 1492\n",
      "--- assemblying 129 probes in region: 1493\n",
      "--- assemblying 189 probes in region: 1494\n",
      "--- assemblying 199 probes in region: 1495\n",
      "--- assemblying 72 probes in region: 1496\n",
      "--- assemblying 230 probes in region: 1497\n",
      "--- assemblying 146 probes in region: 1498\n",
      "--- assemblying 160 probes in region: 1499\n",
      "--- assemblying 300 probes in region: 1500\n",
      "--- assemblying 253 probes in region: 1501\n",
      "--- assemblying 139 probes in region: 1502\n",
      "--- assemblying 186 probes in region: 1503\n",
      "--- assemblying 285 probes in region: 1504\n",
      "--- assemblying 239 probes in region: 1505\n",
      "--- assemblying 163 probes in region: 1506\n",
      "--- assemblying 242 probes in region: 1507\n",
      "--- assemblying 189 probes in region: 1508\n",
      "--- assemblying 188 probes in region: 1509\n",
      "--- assemblying 176 probes in region: 1510\n",
      "--- assemblying 218 probes in region: 1511\n",
      "--- assemblying 248 probes in region: 1512\n",
      "--- assemblying 245 probes in region: 1513\n",
      "--- assemblying 262 probes in region: 1514\n",
      "--- assemblying 300 probes in region: 1515\n",
      "--- assemblying 103 probes in region: 1516\n",
      "--- assemblying 232 probes in region: 1517\n",
      "--- assemblying 217 probes in region: 1518\n",
      "--- assemblying 239 probes in region: 1519\n",
      "--- assemblying 221 probes in region: 1520\n",
      "--- assemblying 176 probes in region: 1521\n",
      "--- assemblying 51 probes in region: 1525\n",
      "--- assemblying 60 probes in region: 1526\n",
      "--- assemblying 72 probes in region: 1527\n",
      "--- assemblying 166 probes in region: 1529\n",
      "--- assemblying 218 probes in region: 1530\n",
      "--- assemblying 69 probes in region: 1533\n",
      "--- assemblying 149 probes in region: 1534\n",
      "--- assemblying 187 probes in region: 1535\n",
      "--- assemblying 187 probes in region: 1536\n",
      "--- assemblying 170 probes in region: 1537\n",
      "--- assemblying 164 probes in region: 1538\n",
      "--- assemblying 251 probes in region: 1541\n",
      "--- assemblying 204 probes in region: 1542\n",
      "--- assemblying 106 probes in region: 1548\n",
      "-- 317858 probes assembled in total.\n",
      "-- saving 317858 probes into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\candidate_probes.fasta\n",
      "-- saving readout_summary into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\readout_summary.pkl\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.assemble)\n",
    "from ImageAnalysis3.library_tools.assemble import Assemble_probes\n",
    "\n",
    "cand_probes, readout_summary = Assemble_probes(library_folder, filtered_pb_dict, \n",
    "                                               region_2_readout_dict, readout_dict, primers, \n",
    "                                               primer_len=20, \n",
    "                                               num_readout_per_probe=4, \n",
    "                                               unique_readout_per_probe=True,\n",
    "                                               rc_targets=False, overwrite=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9265b92d",
   "metadata": {},
   "source": [
    "# 4.Check quality"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8dca4823",
   "metadata": {},
   "source": [
    "\n",
    "## 4.1 Load probes\n",
    "\n",
    "Checking procedures could restart from here"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 106,
   "id": "a651e7eb",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total probe loaded: 317858\n"
     ]
    }
   ],
   "source": [
    "# folder for this library\n",
    "# candidate full-length probe filename\n",
    "candidate_basename = 'candidate_probes.fasta' # old version\n",
    "# load full probes\n",
    "full_records = []\n",
    "with open(os.path.join(library_folder, candidate_basename), 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        full_records.append(record)\n",
    "\n",
    "print(f\"Total probe loaded: {len(full_records)}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fe532250",
   "metadata": {},
   "source": [
    "### Manually input the following info to double check!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "cf1d4bf8",
   "metadata": {},
   "outputs": [],
   "source": [
    "num_readout_per_probe = 4\n",
    "primer_len=20\n",
    "target_len=42\n",
    "min_num_probes = 40\n",
    "# merfish related\n",
    "num_on_bits = 3"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2da410b1",
   "metadata": {},
   "source": [
    "### 4.1.1 check primer usage"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 110,
   "id": "3cba3f0b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Checking primer usage, total probes: 317858\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "primer_check = check._check_primer_usage(full_records, primers[0], primers[1], primer_len=primer_len)\n",
    "print(primer_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "30ce1025",
   "metadata": {},
   "source": [
    "### 4.1.2 check region probe number"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 111,
   "id": "761720bd",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gene: 0 -> 300\n",
      "gene: 1 -> 300\n",
      "gene: 2 -> 300\n",
      "gene: 3 -> 242\n",
      "gene: 4 -> 256\n",
      "gene: 5 -> 300\n",
      "gene: 6 -> 300\n",
      "gene: 7 -> 300\n",
      "gene: 8 -> 300\n",
      "gene: 9 -> 264\n",
      "gene: 10 -> 216\n",
      "gene: 11 -> 212\n",
      "gene: 12 -> 240\n",
      "gene: 13 -> 300\n",
      "gene: 14 -> 176\n",
      "gene: 15 -> 300\n",
      "gene: 16 -> 245\n",
      "gene: 17 -> 203\n",
      "gene: 18 -> 204\n",
      "gene: 19 -> 300\n",
      "gene: 20 -> 206\n",
      "gene: 21 -> 255\n",
      "gene: 22 -> 300\n",
      "gene: 23 -> 300\n",
      "gene: 24 -> 279\n",
      "gene: 25 -> 238\n",
      "gene: 26 -> 300\n",
      "gene: 27 -> 149\n",
      "gene: 28 -> 262\n",
      "gene: 29 -> 199\n",
      "gene: 30 -> 263\n",
      "gene: 31 -> 215\n",
      "gene: 32 -> 190\n",
      "gene: 33 -> 300\n",
      "gene: 34 -> 230\n",
      "gene: 35 -> 203\n",
      "gene: 36 -> 154\n",
      "gene: 37 -> 300\n",
      "gene: 38 -> 211\n",
      "gene: 39 -> 237\n",
      "gene: 40 -> 293\n",
      "gene: 42 -> 201\n",
      "gene: 43 -> 235\n",
      "gene: 44 -> 300\n",
      "gene: 45 -> 187\n",
      "gene: 46 -> 300\n",
      "gene: 47 -> 213\n",
      "gene: 57 -> 60\n",
      "gene: 60 -> 187\n",
      "gene: 62 -> 300\n",
      "gene: 63 -> 241\n",
      "gene: 64 -> 199\n",
      "gene: 65 -> 114\n",
      "gene: 66 -> 127\n",
      "gene: 67 -> 271\n",
      "gene: 68 -> 226\n",
      "gene: 69 -> 128\n",
      "gene: 70 -> 120\n",
      "gene: 71 -> 137\n",
      "gene: 72 -> 267\n",
      "gene: 73 -> 198\n",
      "gene: 74 -> 173\n",
      "gene: 75 -> 149\n",
      "gene: 76 -> 121\n",
      "gene: 77 -> 218\n",
      "gene: 78 -> 142\n",
      "gene: 79 -> 198\n",
      "gene: 80 -> 247\n",
      "gene: 81 -> 223\n",
      "gene: 83 -> 259\n",
      "gene: 84 -> 191\n",
      "gene: 85 -> 300\n",
      "gene: 86 -> 237\n",
      "gene: 87 -> 236\n",
      "gene: 88 -> 204\n",
      "gene: 89 -> 300\n",
      "gene: 90 -> 118\n",
      "gene: 91 -> 300\n",
      "gene: 92 -> 271\n",
      "gene: 93 -> 212\n",
      "gene: 94 -> 178\n",
      "gene: 95 -> 214\n",
      "gene: 96 -> 258\n",
      "gene: 98 -> 168\n",
      "gene: 100 -> 300\n",
      "gene: 101 -> 286\n",
      "gene: 102 -> 240\n",
      "gene: 103 -> 205\n",
      "gene: 104 -> 119\n",
      "gene: 105 -> 300\n",
      "gene: 106 -> 244\n",
      "gene: 107 -> 300\n",
      "gene: 108 -> 213\n",
      "gene: 109 -> 225\n",
      "gene: 110 -> 230\n",
      "gene: 111 -> 300\n",
      "gene: 112 -> 167\n",
      "gene: 114 -> 215\n",
      "gene: 115 -> 224\n",
      "gene: 116 -> 228\n",
      "gene: 117 -> 276\n",
      "gene: 118 -> 216\n",
      "gene: 119 -> 221\n",
      "gene: 120 -> 200\n",
      "gene: 121 -> 295\n",
      "gene: 122 -> 270\n",
      "gene: 123 -> 182\n",
      "gene: 124 -> 235\n",
      "gene: 125 -> 251\n",
      "gene: 126 -> 291\n",
      "gene: 127 -> 222\n",
      "gene: 128 -> 275\n",
      "gene: 129 -> 216\n",
      "gene: 130 -> 162\n",
      "gene: 131 -> 190\n",
      "gene: 132 -> 192\n",
      "gene: 133 -> 300\n",
      "gene: 134 -> 237\n",
      "gene: 137 -> 300\n",
      "gene: 138 -> 285\n",
      "gene: 139 -> 174\n",
      "gene: 140 -> 261\n",
      "gene: 141 -> 74\n",
      "gene: 143 -> 186\n",
      "gene: 144 -> 187\n",
      "gene: 145 -> 178\n",
      "gene: 146 -> 294\n",
      "gene: 147 -> 300\n",
      "gene: 148 -> 193\n",
      "gene: 149 -> 300\n",
      "gene: 150 -> 240\n",
      "gene: 152 -> 248\n",
      "gene: 153 -> 300\n",
      "gene: 154 -> 283\n",
      "gene: 155 -> 252\n",
      "gene: 156 -> 292\n",
      "gene: 157 -> 255\n",
      "gene: 158 -> 227\n",
      "gene: 159 -> 231\n",
      "gene: 160 -> 136\n",
      "gene: 161 -> 134\n",
      "gene: 162 -> 300\n",
      "gene: 163 -> 234\n",
      "gene: 164 -> 300\n",
      "gene: 165 -> 244\n",
      "gene: 166 -> 300\n",
      "gene: 167 -> 217\n",
      "gene: 168 -> 300\n",
      "gene: 169 -> 238\n",
      "gene: 170 -> 300\n",
      "gene: 171 -> 272\n",
      "gene: 172 -> 200\n",
      "gene: 173 -> 173\n",
      "gene: 174 -> 198\n",
      "gene: 175 -> 300\n",
      "gene: 176 -> 152\n",
      "gene: 177 -> 279\n",
      "gene: 178 -> 280\n",
      "gene: 179 -> 230\n",
      "gene: 180 -> 226\n",
      "gene: 181 -> 248\n",
      "gene: 182 -> 93\n",
      "gene: 183 -> 204\n",
      "gene: 184 -> 173\n",
      "gene: 185 -> 300\n",
      "gene: 186 -> 300\n",
      "gene: 187 -> 239\n",
      "gene: 188 -> 233\n",
      "gene: 189 -> 191\n",
      "gene: 190 -> 150\n",
      "gene: 191 -> 274\n",
      "gene: 192 -> 270\n",
      "gene: 193 -> 300\n",
      "gene: 194 -> 297\n",
      "gene: 195 -> 199\n",
      "gene: 196 -> 300\n",
      "gene: 197 -> 251\n",
      "gene: 198 -> 300\n",
      "gene: 199 -> 275\n",
      "gene: 200 -> 229\n",
      "gene: 201 -> 300\n",
      "gene: 202 -> 300\n",
      "gene: 203 -> 190\n",
      "gene: 204 -> 265\n",
      "gene: 205 -> 241\n",
      "gene: 206 -> 262\n",
      "gene: 207 -> 300\n",
      "gene: 208 -> 174\n",
      "gene: 209 -> 180\n",
      "gene: 210 -> 300\n",
      "gene: 211 -> 300\n",
      "gene: 212 -> 300\n",
      "gene: 213 -> 200\n",
      "gene: 214 -> 219\n",
      "gene: 215 -> 142\n",
      "gene: 216 -> 300\n",
      "gene: 217 -> 278\n",
      "gene: 218 -> 252\n",
      "gene: 219 -> 258\n",
      "gene: 220 -> 214\n",
      "gene: 221 -> 256\n",
      "gene: 222 -> 292\n",
      "gene: 223 -> 300\n",
      "gene: 224 -> 300\n",
      "gene: 225 -> 234\n",
      "gene: 226 -> 220\n",
      "gene: 227 -> 242\n",
      "gene: 228 -> 148\n",
      "gene: 229 -> 197\n",
      "gene: 230 -> 166\n",
      "gene: 233 -> 196\n",
      "gene: 234 -> 255\n",
      "gene: 235 -> 233\n",
      "gene: 236 -> 182\n",
      "gene: 237 -> 179\n",
      "gene: 238 -> 251\n",
      "gene: 239 -> 231\n",
      "gene: 240 -> 234\n",
      "gene: 241 -> 272\n",
      "gene: 242 -> 288\n",
      "gene: 243 -> 165\n",
      "gene: 244 -> 300\n",
      "gene: 245 -> 238\n",
      "gene: 246 -> 205\n",
      "gene: 247 -> 234\n",
      "gene: 248 -> 277\n",
      "gene: 249 -> 242\n",
      "gene: 250 -> 172\n",
      "gene: 251 -> 300\n",
      "gene: 252 -> 287\n",
      "gene: 253 -> 188\n",
      "gene: 254 -> 282\n",
      "gene: 255 -> 225\n",
      "gene: 256 -> 218\n",
      "gene: 257 -> 288\n",
      "gene: 258 -> 300\n",
      "gene: 259 -> 160\n",
      "gene: 260 -> 162\n",
      "gene: 261 -> 159\n",
      "gene: 262 -> 179\n",
      "gene: 263 -> 300\n",
      "gene: 264 -> 259\n",
      "gene: 265 -> 220\n",
      "gene: 266 -> 251\n",
      "gene: 267 -> 263\n",
      "gene: 268 -> 175\n",
      "gene: 269 -> 300\n",
      "gene: 270 -> 300\n",
      "gene: 271 -> 220\n",
      "gene: 272 -> 232\n",
      "gene: 273 -> 231\n",
      "gene: 274 -> 300\n",
      "gene: 275 -> 300\n",
      "gene: 276 -> 300\n",
      "gene: 277 -> 300\n",
      "gene: 278 -> 224\n",
      "gene: 279 -> 210\n",
      "gene: 280 -> 300\n",
      "gene: 281 -> 239\n",
      "gene: 282 -> 262\n",
      "gene: 283 -> 293\n",
      "gene: 284 -> 215\n",
      "gene: 285 -> 234\n",
      "gene: 286 -> 187\n",
      "gene: 287 -> 194\n",
      "gene: 288 -> 249\n",
      "gene: 289 -> 250\n",
      "gene: 290 -> 253\n",
      "gene: 291 -> 195\n",
      "gene: 292 -> 226\n",
      "gene: 293 -> 157\n",
      "gene: 295 -> 209\n",
      "gene: 296 -> 274\n",
      "gene: 297 -> 194\n",
      "gene: 298 -> 253\n",
      "gene: 299 -> 141\n",
      "gene: 300 -> 151\n",
      "gene: 302 -> 300\n",
      "gene: 303 -> 213\n",
      "gene: 304 -> 232\n",
      "gene: 305 -> 222\n",
      "gene: 306 -> 242\n",
      "gene: 307 -> 231\n",
      "gene: 308 -> 203\n",
      "gene: 309 -> 300\n",
      "gene: 310 -> 166\n",
      "gene: 311 -> 267\n",
      "gene: 312 -> 241\n",
      "gene: 313 -> 182\n",
      "gene: 314 -> 253\n",
      "gene: 315 -> 228\n",
      "gene: 316 -> 183\n",
      "gene: 317 -> 251\n",
      "gene: 318 -> 178\n",
      "gene: 319 -> 249\n",
      "gene: 320 -> 155\n",
      "gene: 321 -> 279\n",
      "gene: 322 -> 258\n",
      "gene: 323 -> 248\n",
      "gene: 324 -> 171\n",
      "gene: 325 -> 176\n",
      "gene: 326 -> 191\n",
      "gene: 327 -> 106\n",
      "gene: 328 -> 170\n",
      "gene: 329 -> 295\n",
      "gene: 330 -> 218\n",
      "gene: 331 -> 154\n",
      "gene: 332 -> 300\n",
      "gene: 333 -> 214\n",
      "gene: 334 -> 234\n",
      "gene: 335 -> 271\n",
      "gene: 336 -> 244\n",
      "gene: 337 -> 181\n",
      "gene: 338 -> 222\n",
      "gene: 339 -> 257\n",
      "gene: 340 -> 201\n",
      "gene: 341 -> 284\n",
      "gene: 342 -> 157\n",
      "gene: 343 -> 260\n",
      "gene: 344 -> 234\n",
      "gene: 345 -> 225\n",
      "gene: 346 -> 279\n",
      "gene: 347 -> 202\n",
      "gene: 348 -> 223\n",
      "gene: 349 -> 161\n",
      "gene: 350 -> 300\n",
      "gene: 351 -> 300\n",
      "gene: 352 -> 300\n",
      "gene: 353 -> 158\n",
      "gene: 354 -> 293\n",
      "gene: 355 -> 234\n",
      "gene: 356 -> 257\n",
      "gene: 357 -> 170\n",
      "gene: 358 -> 216\n",
      "gene: 359 -> 179\n",
      "gene: 360 -> 190\n",
      "gene: 361 -> 175\n",
      "gene: 362 -> 300\n",
      "gene: 363 -> 300\n",
      "gene: 364 -> 253\n",
      "gene: 365 -> 284\n",
      "gene: 366 -> 214\n",
      "gene: 367 -> 102\n",
      "gene: 368 -> 64\n",
      "gene: 370 -> 158\n",
      "gene: 371 -> 84\n",
      "gene: 372 -> 236\n",
      "gene: 373 -> 270\n",
      "gene: 374 -> 224\n",
      "gene: 375 -> 226\n",
      "gene: 376 -> 222\n",
      "gene: 377 -> 300\n",
      "gene: 379 -> 287\n",
      "gene: 380 -> 224\n",
      "gene: 381 -> 278\n",
      "gene: 382 -> 300\n",
      "gene: 383 -> 176\n",
      "gene: 384 -> 166\n",
      "gene: 385 -> 272\n",
      "gene: 386 -> 215\n",
      "gene: 387 -> 192\n",
      "gene: 388 -> 277\n",
      "gene: 389 -> 170\n",
      "gene: 390 -> 164\n",
      "gene: 391 -> 257\n",
      "gene: 392 -> 202\n",
      "gene: 393 -> 229\n",
      "gene: 394 -> 300\n",
      "gene: 395 -> 256\n",
      "gene: 396 -> 182\n",
      "gene: 397 -> 245\n",
      "gene: 398 -> 162\n",
      "gene: 399 -> 196\n",
      "gene: 400 -> 222\n",
      "gene: 401 -> 237\n",
      "gene: 402 -> 191\n",
      "gene: 403 -> 300\n",
      "gene: 404 -> 300\n",
      "gene: 405 -> 205\n",
      "gene: 406 -> 261\n",
      "gene: 407 -> 300\n",
      "gene: 408 -> 163\n",
      "gene: 409 -> 221\n",
      "gene: 410 -> 271\n",
      "gene: 411 -> 282\n",
      "gene: 412 -> 212\n",
      "gene: 413 -> 300\n",
      "gene: 414 -> 188\n",
      "gene: 415 -> 252\n",
      "gene: 416 -> 252\n",
      "gene: 417 -> 300\n",
      "gene: 418 -> 200\n",
      "gene: 419 -> 300\n",
      "gene: 421 -> 212\n",
      "gene: 422 -> 298\n",
      "gene: 423 -> 300\n",
      "gene: 424 -> 300\n",
      "gene: 425 -> 221\n",
      "gene: 426 -> 300\n",
      "gene: 427 -> 217\n",
      "gene: 428 -> 243\n",
      "gene: 429 -> 193\n",
      "gene: 430 -> 236\n",
      "gene: 431 -> 237\n",
      "gene: 432 -> 270\n",
      "gene: 435 -> 300\n",
      "gene: 436 -> 294\n",
      "gene: 437 -> 188\n",
      "gene: 438 -> 206\n",
      "gene: 439 -> 300\n",
      "gene: 440 -> 300\n",
      "gene: 441 -> 183\n",
      "gene: 442 -> 276\n",
      "gene: 443 -> 241\n",
      "gene: 444 -> 300\n",
      "gene: 446 -> 108\n",
      "gene: 447 -> 181\n",
      "gene: 448 -> 127\n",
      "gene: 449 -> 228\n",
      "gene: 450 -> 268\n",
      "gene: 451 -> 258\n",
      "gene: 452 -> 197\n",
      "gene: 453 -> 67\n",
      "gene: 454 -> 145\n",
      "gene: 455 -> 142\n",
      "gene: 456 -> 217\n",
      "gene: 457 -> 207\n",
      "gene: 458 -> 236\n",
      "gene: 459 -> 180\n",
      "gene: 460 -> 202\n",
      "gene: 461 -> 209\n",
      "gene: 462 -> 170\n",
      "gene: 463 -> 143\n",
      "gene: 464 -> 211\n",
      "gene: 465 -> 300\n",
      "gene: 466 -> 189\n",
      "gene: 467 -> 200\n",
      "gene: 468 -> 222\n",
      "gene: 469 -> 260\n",
      "gene: 470 -> 131\n",
      "gene: 471 -> 199\n",
      "gene: 472 -> 203\n",
      "gene: 473 -> 203\n",
      "gene: 474 -> 235\n",
      "gene: 475 -> 263\n",
      "gene: 476 -> 236\n",
      "gene: 477 -> 215\n",
      "gene: 478 -> 189\n",
      "gene: 479 -> 287\n",
      "gene: 480 -> 257\n",
      "gene: 481 -> 269\n",
      "gene: 482 -> 230\n",
      "gene: 483 -> 220\n",
      "gene: 484 -> 265\n",
      "gene: 485 -> 255\n",
      "gene: 486 -> 294\n",
      "gene: 487 -> 300\n",
      "gene: 489 -> 216\n",
      "gene: 490 -> 300\n",
      "gene: 491 -> 276\n",
      "gene: 492 -> 229\n",
      "gene: 493 -> 239\n",
      "gene: 494 -> 293\n",
      "gene: 495 -> 259\n",
      "gene: 496 -> 167\n",
      "gene: 497 -> 228\n",
      "gene: 498 -> 262\n",
      "gene: 499 -> 162\n",
      "gene: 500 -> 142\n",
      "gene: 501 -> 300\n",
      "gene: 502 -> 269\n",
      "gene: 503 -> 300\n",
      "gene: 504 -> 220\n",
      "gene: 505 -> 300\n",
      "gene: 506 -> 300\n",
      "gene: 507 -> 202\n",
      "gene: 508 -> 221\n",
      "gene: 509 -> 272\n",
      "gene: 510 -> 300\n",
      "gene: 511 -> 184\n",
      "gene: 512 -> 197\n",
      "gene: 516 -> 227\n",
      "gene: 517 -> 262\n",
      "gene: 518 -> 264\n",
      "gene: 519 -> 188\n",
      "gene: 520 -> 212\n",
      "gene: 521 -> 265\n",
      "gene: 522 -> 187\n",
      "gene: 523 -> 152\n",
      "gene: 524 -> 196\n",
      "gene: 525 -> 216\n",
      "gene: 526 -> 223\n",
      "gene: 527 -> 255\n",
      "gene: 528 -> 249\n",
      "gene: 529 -> 269\n",
      "gene: 530 -> 223\n",
      "gene: 531 -> 201\n",
      "gene: 532 -> 232\n",
      "gene: 533 -> 201\n",
      "gene: 534 -> 181\n",
      "gene: 535 -> 116\n",
      "gene: 536 -> 259\n",
      "gene: 537 -> 231\n",
      "gene: 538 -> 214\n",
      "gene: 539 -> 242\n",
      "gene: 540 -> 179\n",
      "gene: 541 -> 248\n",
      "gene: 542 -> 227\n",
      "gene: 543 -> 300\n",
      "gene: 544 -> 276\n",
      "gene: 545 -> 267\n",
      "gene: 546 -> 300\n",
      "gene: 548 -> 191\n",
      "gene: 549 -> 206\n",
      "gene: 550 -> 300\n",
      "gene: 551 -> 90\n",
      "gene: 552 -> 264\n",
      "gene: 554 -> 300\n",
      "gene: 555 -> 241\n",
      "gene: 556 -> 234\n",
      "gene: 557 -> 273\n",
      "gene: 558 -> 251\n",
      "gene: 559 -> 221\n",
      "gene: 560 -> 261\n",
      "gene: 561 -> 240\n",
      "gene: 562 -> 249\n",
      "gene: 563 -> 193\n",
      "gene: 564 -> 270\n",
      "gene: 565 -> 218\n",
      "gene: 566 -> 169\n",
      "gene: 567 -> 229\n",
      "gene: 568 -> 222\n",
      "gene: 569 -> 292\n",
      "gene: 570 -> 104\n",
      "gene: 571 -> 300\n",
      "gene: 572 -> 280\n",
      "gene: 573 -> 206\n",
      "gene: 574 -> 258\n",
      "gene: 575 -> 204\n",
      "gene: 576 -> 291\n",
      "gene: 577 -> 207\n",
      "gene: 578 -> 246\n",
      "gene: 579 -> 151\n",
      "gene: 580 -> 202\n",
      "gene: 581 -> 250\n",
      "gene: 582 -> 199\n",
      "gene: 583 -> 181\n",
      "gene: 584 -> 139\n",
      "gene: 585 -> 241\n",
      "gene: 586 -> 218\n",
      "gene: 587 -> 156\n",
      "gene: 588 -> 177\n",
      "gene: 589 -> 275\n",
      "gene: 590 -> 225\n",
      "gene: 591 -> 283\n",
      "gene: 592 -> 144\n",
      "gene: 593 -> 300\n",
      "gene: 594 -> 155\n",
      "gene: 595 -> 180\n",
      "gene: 596 -> 254\n",
      "gene: 597 -> 277\n",
      "gene: 598 -> 226\n",
      "gene: 599 -> 300\n",
      "gene: 600 -> 294\n",
      "gene: 601 -> 299\n",
      "gene: 602 -> 261\n",
      "gene: 603 -> 251\n",
      "gene: 605 -> 187\n",
      "gene: 606 -> 293\n",
      "gene: 607 -> 281\n",
      "gene: 608 -> 300\n",
      "gene: 609 -> 300\n",
      "gene: 611 -> 300\n",
      "gene: 612 -> 300\n",
      "gene: 613 -> 200\n",
      "gene: 614 -> 277\n",
      "gene: 615 -> 258\n",
      "gene: 616 -> 173\n",
      "gene: 617 -> 254\n",
      "gene: 618 -> 192\n",
      "gene: 619 -> 282\n",
      "gene: 620 -> 196\n",
      "gene: 621 -> 248\n",
      "gene: 622 -> 247\n",
      "gene: 623 -> 185\n",
      "gene: 624 -> 132\n",
      "gene: 625 -> 168\n",
      "gene: 626 -> 118\n",
      "gene: 627 -> 253\n",
      "gene: 628 -> 273\n",
      "gene: 629 -> 300\n",
      "gene: 645 -> 257\n",
      "gene: 646 -> 241\n",
      "gene: 647 -> 219\n",
      "gene: 648 -> 215\n",
      "gene: 649 -> 262\n",
      "gene: 650 -> 241\n",
      "gene: 651 -> 146\n",
      "gene: 652 -> 300\n",
      "gene: 653 -> 300\n",
      "gene: 654 -> 271\n",
      "gene: 655 -> 300\n",
      "gene: 656 -> 292\n",
      "gene: 657 -> 186\n",
      "gene: 658 -> 300\n",
      "gene: 659 -> 54\n",
      "gene: 660 -> 278\n",
      "gene: 662 -> 184\n",
      "gene: 663 -> 205\n",
      "gene: 664 -> 255\n",
      "gene: 665 -> 266\n",
      "gene: 666 -> 273\n",
      "gene: 667 -> 172\n",
      "gene: 668 -> 300\n",
      "gene: 669 -> 300\n",
      "gene: 670 -> 300\n",
      "gene: 671 -> 199\n",
      "gene: 672 -> 171\n",
      "gene: 673 -> 120\n",
      "gene: 674 -> 258\n",
      "gene: 675 -> 300\n",
      "gene: 676 -> 300\n",
      "gene: 677 -> 300\n",
      "gene: 678 -> 300\n",
      "gene: 679 -> 300\n",
      "gene: 680 -> 294\n",
      "gene: 681 -> 300\n",
      "gene: 682 -> 147\n",
      "gene: 683 -> 285\n",
      "gene: 684 -> 253\n",
      "gene: 685 -> 300\n",
      "gene: 686 -> 266\n",
      "gene: 687 -> 78\n",
      "gene: 688 -> 278\n",
      "gene: 689 -> 160\n",
      "gene: 690 -> 289\n",
      "gene: 691 -> 270\n",
      "gene: 692 -> 189\n",
      "gene: 693 -> 246\n",
      "gene: 694 -> 236\n",
      "gene: 695 -> 246\n",
      "gene: 696 -> 170\n",
      "gene: 697 -> 220\n",
      "gene: 698 -> 227\n",
      "gene: 699 -> 66\n",
      "gene: 701 -> 289\n",
      "gene: 702 -> 291\n",
      "gene: 703 -> 181\n",
      "gene: 704 -> 284\n",
      "gene: 705 -> 292\n",
      "gene: 706 -> 246\n",
      "gene: 707 -> 182\n",
      "gene: 708 -> 213\n",
      "gene: 709 -> 289\n",
      "gene: 710 -> 234\n",
      "gene: 711 -> 290\n",
      "gene: 712 -> 201\n",
      "gene: 713 -> 236\n",
      "gene: 714 -> 196\n",
      "gene: 715 -> 300\n",
      "gene: 716 -> 136\n",
      "gene: 717 -> 282\n",
      "gene: 718 -> 300\n",
      "gene: 719 -> 300\n",
      "gene: 720 -> 218\n",
      "gene: 721 -> 270\n",
      "gene: 722 -> 300\n",
      "gene: 723 -> 73\n",
      "gene: 724 -> 288\n",
      "gene: 725 -> 271\n",
      "gene: 726 -> 209\n",
      "gene: 727 -> 138\n",
      "gene: 728 -> 300\n",
      "gene: 729 -> 200\n",
      "gene: 730 -> 300\n",
      "gene: 731 -> 174\n",
      "gene: 732 -> 296\n",
      "gene: 733 -> 274\n",
      "gene: 734 -> 225\n",
      "gene: 735 -> 206\n",
      "gene: 736 -> 300\n",
      "gene: 737 -> 252\n",
      "gene: 738 -> 222\n",
      "gene: 739 -> 229\n",
      "gene: 740 -> 249\n",
      "gene: 741 -> 266\n",
      "gene: 742 -> 300\n",
      "gene: 743 -> 300\n",
      "gene: 744 -> 300\n",
      "gene: 745 -> 300\n",
      "gene: 748 -> 254\n",
      "gene: 749 -> 199\n",
      "gene: 750 -> 287\n",
      "gene: 751 -> 293\n",
      "gene: 752 -> 300\n",
      "gene: 753 -> 145\n",
      "gene: 754 -> 300\n",
      "gene: 755 -> 59\n",
      "gene: 756 -> 300\n",
      "gene: 757 -> 192\n",
      "gene: 758 -> 141\n",
      "gene: 759 -> 300\n",
      "gene: 760 -> 220\n",
      "gene: 761 -> 222\n",
      "gene: 762 -> 141\n",
      "gene: 763 -> 270\n",
      "gene: 764 -> 261\n",
      "gene: 765 -> 191\n",
      "gene: 766 -> 254\n",
      "gene: 767 -> 199\n",
      "gene: 768 -> 278\n",
      "gene: 769 -> 160\n",
      "gene: 770 -> 81\n",
      "gene: 774 -> 202\n",
      "gene: 775 -> 202\n",
      "gene: 776 -> 300\n",
      "gene: 777 -> 300\n",
      "gene: 778 -> 252\n",
      "gene: 779 -> 208\n",
      "gene: 780 -> 300\n",
      "gene: 782 -> 206\n",
      "gene: 783 -> 286\n",
      "gene: 784 -> 196\n",
      "gene: 785 -> 295\n",
      "gene: 786 -> 234\n",
      "gene: 787 -> 293\n",
      "gene: 788 -> 182\n",
      "gene: 789 -> 209\n",
      "gene: 790 -> 91\n",
      "gene: 791 -> 164\n",
      "gene: 792 -> 168\n",
      "gene: 793 -> 231\n",
      "gene: 794 -> 298\n",
      "gene: 795 -> 126\n",
      "gene: 796 -> 289\n",
      "gene: 797 -> 249\n",
      "gene: 798 -> 244\n",
      "gene: 799 -> 289\n",
      "gene: 800 -> 196\n",
      "gene: 801 -> 300\n",
      "gene: 802 -> 257\n",
      "gene: 803 -> 289\n",
      "gene: 804 -> 220\n",
      "gene: 805 -> 168\n",
      "gene: 806 -> 291\n",
      "gene: 807 -> 233\n",
      "gene: 808 -> 197\n",
      "gene: 809 -> 300\n",
      "gene: 810 -> 160\n",
      "gene: 811 -> 234\n",
      "gene: 812 -> 300\n",
      "gene: 813 -> 187\n",
      "gene: 814 -> 275\n",
      "gene: 815 -> 300\n",
      "gene: 816 -> 209\n",
      "gene: 817 -> 204\n",
      "gene: 819 -> 268\n",
      "gene: 820 -> 255\n",
      "gene: 821 -> 142\n",
      "gene: 822 -> 257\n",
      "gene: 823 -> 129\n",
      "gene: 824 -> 288\n",
      "gene: 825 -> 225\n",
      "gene: 826 -> 263\n",
      "gene: 827 -> 283\n",
      "gene: 828 -> 300\n",
      "gene: 829 -> 237\n",
      "gene: 832 -> 168\n",
      "gene: 833 -> 204\n",
      "gene: 834 -> 300\n",
      "gene: 835 -> 236\n",
      "gene: 836 -> 164\n",
      "gene: 837 -> 300\n",
      "gene: 838 -> 226\n",
      "gene: 839 -> 194\n",
      "gene: 840 -> 264\n",
      "gene: 841 -> 300\n",
      "gene: 843 -> 148\n",
      "gene: 844 -> 256\n",
      "gene: 845 -> 300\n",
      "gene: 846 -> 261\n",
      "gene: 847 -> 174\n",
      "gene: 848 -> 294\n",
      "gene: 849 -> 123\n",
      "gene: 850 -> 181\n",
      "gene: 851 -> 241\n",
      "gene: 852 -> 279\n",
      "gene: 853 -> 194\n",
      "gene: 854 -> 215\n",
      "gene: 855 -> 199\n",
      "gene: 856 -> 163\n",
      "gene: 857 -> 300\n",
      "gene: 858 -> 252\n",
      "gene: 859 -> 240\n",
      "gene: 860 -> 144\n",
      "gene: 861 -> 300\n",
      "gene: 862 -> 233\n",
      "gene: 863 -> 282\n",
      "gene: 864 -> 250\n",
      "gene: 865 -> 223\n",
      "gene: 866 -> 216\n",
      "gene: 867 -> 164\n",
      "gene: 868 -> 289\n",
      "gene: 869 -> 300\n",
      "gene: 870 -> 300\n",
      "gene: 871 -> 300\n",
      "gene: 872 -> 300\n",
      "gene: 873 -> 265\n",
      "gene: 874 -> 300\n",
      "gene: 875 -> 212\n",
      "gene: 876 -> 262\n",
      "gene: 877 -> 268\n",
      "gene: 878 -> 300\n",
      "gene: 888 -> 215\n",
      "gene: 889 -> 279\n",
      "gene: 890 -> 205\n",
      "gene: 891 -> 169\n",
      "gene: 892 -> 300\n",
      "gene: 893 -> 252\n",
      "gene: 894 -> 300\n",
      "gene: 895 -> 294\n",
      "gene: 896 -> 158\n",
      "gene: 897 -> 286\n",
      "gene: 898 -> 182\n",
      "gene: 899 -> 211\n",
      "gene: 900 -> 251\n",
      "gene: 901 -> 230\n",
      "gene: 902 -> 271\n",
      "gene: 903 -> 144\n",
      "gene: 904 -> 300\n",
      "gene: 905 -> 300\n",
      "gene: 906 -> 194\n",
      "gene: 907 -> 143\n",
      "gene: 908 -> 216\n",
      "gene: 909 -> 181\n",
      "gene: 910 -> 155\n",
      "gene: 911 -> 187\n",
      "gene: 912 -> 230\n",
      "gene: 913 -> 169\n",
      "gene: 914 -> 176\n",
      "gene: 915 -> 166\n",
      "gene: 916 -> 250\n",
      "gene: 917 -> 250\n",
      "gene: 918 -> 259\n",
      "gene: 919 -> 157\n",
      "gene: 920 -> 187\n",
      "gene: 921 -> 173\n",
      "gene: 922 -> 174\n",
      "gene: 923 -> 141\n",
      "gene: 924 -> 236\n",
      "gene: 925 -> 214\n",
      "gene: 926 -> 293\n",
      "gene: 927 -> 300\n",
      "gene: 928 -> 300\n",
      "gene: 929 -> 261\n",
      "gene: 930 -> 266\n",
      "gene: 931 -> 249\n",
      "gene: 932 -> 297\n",
      "gene: 933 -> 224\n",
      "gene: 934 -> 209\n",
      "gene: 935 -> 300\n",
      "gene: 949 -> 160\n",
      "gene: 950 -> 178\n",
      "gene: 951 -> 188\n",
      "gene: 952 -> 190\n",
      "gene: 953 -> 245\n",
      "gene: 954 -> 90\n",
      "gene: 955 -> 174\n",
      "gene: 956 -> 265\n",
      "gene: 957 -> 258\n",
      "gene: 958 -> 300\n",
      "gene: 959 -> 213\n",
      "gene: 960 -> 279\n",
      "gene: 961 -> 237\n",
      "gene: 962 -> 226\n",
      "gene: 963 -> 195\n",
      "gene: 964 -> 149\n",
      "gene: 965 -> 191\n",
      "gene: 966 -> 161\n",
      "gene: 967 -> 229\n",
      "gene: 968 -> 228\n",
      "gene: 969 -> 156\n",
      "gene: 970 -> 110\n",
      "gene: 971 -> 173\n",
      "gene: 972 -> 300\n",
      "gene: 973 -> 270\n",
      "gene: 974 -> 209\n",
      "gene: 975 -> 300\n",
      "gene: 976 -> 194\n",
      "gene: 977 -> 297\n",
      "gene: 978 -> 130\n",
      "gene: 979 -> 255\n",
      "gene: 980 -> 173\n",
      "gene: 981 -> 300\n",
      "gene: 982 -> 171\n",
      "gene: 983 -> 236\n",
      "gene: 984 -> 269\n",
      "gene: 985 -> 300\n",
      "gene: 986 -> 233\n",
      "gene: 987 -> 300\n",
      "gene: 988 -> 300\n",
      "gene: 989 -> 154\n",
      "gene: 990 -> 281\n",
      "gene: 991 -> 247\n",
      "gene: 992 -> 237\n",
      "gene: 993 -> 153\n",
      "gene: 994 -> 286\n",
      "gene: 995 -> 300\n",
      "gene: 996 -> 255\n",
      "gene: 997 -> 300\n",
      "gene: 999 -> 300\n",
      "gene: 1000 -> 300\n",
      "gene: 1001 -> 300\n",
      "gene: 1002 -> 300\n",
      "gene: 1003 -> 300\n",
      "gene: 1004 -> 283\n",
      "gene: 1005 -> 300\n",
      "gene: 1020 -> 52\n",
      "gene: 1023 -> 217\n",
      "gene: 1024 -> 242\n",
      "gene: 1025 -> 300\n",
      "gene: 1029 -> 235\n",
      "gene: 1030 -> 209\n",
      "gene: 1031 -> 290\n",
      "gene: 1032 -> 275\n",
      "gene: 1033 -> 234\n",
      "gene: 1034 -> 183\n",
      "gene: 1035 -> 201\n",
      "gene: 1036 -> 130\n",
      "gene: 1037 -> 237\n",
      "gene: 1038 -> 227\n",
      "gene: 1039 -> 246\n",
      "gene: 1040 -> 300\n",
      "gene: 1041 -> 254\n",
      "gene: 1042 -> 300\n",
      "gene: 1043 -> 122\n",
      "gene: 1044 -> 253\n",
      "gene: 1045 -> 300\n",
      "gene: 1046 -> 263\n",
      "gene: 1047 -> 300\n",
      "gene: 1048 -> 296\n",
      "gene: 1049 -> 176\n",
      "gene: 1050 -> 267\n",
      "gene: 1051 -> 300\n",
      "gene: 1052 -> 135\n",
      "gene: 1053 -> 300\n",
      "gene: 1054 -> 300\n",
      "gene: 1055 -> 268\n",
      "gene: 1056 -> 300\n",
      "gene: 1057 -> 155\n",
      "gene: 1058 -> 131\n",
      "gene: 1059 -> 164\n",
      "gene: 1060 -> 300\n",
      "gene: 1061 -> 300\n",
      "gene: 1062 -> 240\n",
      "gene: 1063 -> 266\n",
      "gene: 1064 -> 300\n",
      "gene: 1065 -> 300\n",
      "gene: 1066 -> 207\n",
      "gene: 1067 -> 204\n",
      "gene: 1068 -> 147\n",
      "gene: 1069 -> 160\n",
      "gene: 1070 -> 250\n",
      "gene: 1071 -> 291\n",
      "gene: 1072 -> 259\n",
      "gene: 1073 -> 263\n",
      "gene: 1074 -> 273\n",
      "gene: 1075 -> 256\n",
      "gene: 1076 -> 293\n",
      "gene: 1077 -> 88\n",
      "gene: 1078 -> 241\n",
      "gene: 1079 -> 300\n",
      "gene: 1080 -> 300\n",
      "gene: 1081 -> 298\n",
      "gene: 1082 -> 263\n",
      "gene: 1083 -> 300\n",
      "gene: 1085 -> 300\n",
      "gene: 1087 -> 300\n",
      "gene: 1089 -> 207\n",
      "gene: 1090 -> 261\n",
      "gene: 1091 -> 226\n",
      "gene: 1092 -> 300\n",
      "gene: 1093 -> 203\n",
      "gene: 1094 -> 300\n",
      "gene: 1097 -> 230\n",
      "gene: 1105 -> 223\n",
      "gene: 1106 -> 189\n",
      "gene: 1107 -> 267\n",
      "gene: 1108 -> 295\n",
      "gene: 1109 -> 184\n",
      "gene: 1110 -> 300\n",
      "gene: 1111 -> 300\n",
      "gene: 1112 -> 300\n",
      "gene: 1113 -> 266\n",
      "gene: 1114 -> 217\n",
      "gene: 1115 -> 253\n",
      "gene: 1116 -> 184\n",
      "gene: 1117 -> 288\n",
      "gene: 1118 -> 231\n",
      "gene: 1119 -> 300\n",
      "gene: 1120 -> 228\n",
      "gene: 1121 -> 298\n",
      "gene: 1122 -> 227\n",
      "gene: 1123 -> 267\n",
      "gene: 1124 -> 201\n",
      "gene: 1125 -> 208\n",
      "gene: 1126 -> 300\n",
      "gene: 1127 -> 241\n",
      "gene: 1128 -> 235\n",
      "gene: 1129 -> 300\n",
      "gene: 1130 -> 300\n",
      "gene: 1131 -> 272\n",
      "gene: 1132 -> 290\n",
      "gene: 1133 -> 205\n",
      "gene: 1134 -> 300\n",
      "gene: 1135 -> 268\n",
      "gene: 1136 -> 294\n",
      "gene: 1137 -> 196\n",
      "gene: 1138 -> 222\n",
      "gene: 1139 -> 256\n",
      "gene: 1140 -> 242\n",
      "gene: 1141 -> 300\n",
      "gene: 1142 -> 263\n",
      "gene: 1143 -> 206\n",
      "gene: 1144 -> 122\n",
      "gene: 1145 -> 215\n",
      "gene: 1146 -> 94\n",
      "gene: 1147 -> 257\n",
      "gene: 1148 -> 101\n",
      "gene: 1152 -> 194\n",
      "gene: 1153 -> 269\n",
      "gene: 1154 -> 54\n",
      "gene: 1155 -> 254\n",
      "gene: 1156 -> 300\n",
      "gene: 1157 -> 276\n",
      "gene: 1160 -> 300\n",
      "gene: 1162 -> 222\n",
      "gene: 1163 -> 272\n",
      "gene: 1165 -> 270\n",
      "gene: 1166 -> 300\n",
      "gene: 1167 -> 192\n",
      "gene: 1168 -> 300\n",
      "gene: 1169 -> 185\n",
      "gene: 1170 -> 232\n",
      "gene: 1171 -> 268\n",
      "gene: 1172 -> 300\n",
      "gene: 1173 -> 96\n",
      "gene: 1174 -> 128\n",
      "gene: 1176 -> 212\n",
      "gene: 1177 -> 241\n",
      "gene: 1178 -> 300\n",
      "gene: 1179 -> 197\n",
      "gene: 1180 -> 213\n",
      "gene: 1181 -> 271\n",
      "gene: 1182 -> 291\n",
      "gene: 1183 -> 300\n",
      "gene: 1184 -> 300\n",
      "gene: 1186 -> 265\n",
      "gene: 1187 -> 285\n",
      "gene: 1188 -> 77\n",
      "gene: 1189 -> 171\n",
      "gene: 1190 -> 187\n",
      "gene: 1191 -> 194\n",
      "gene: 1192 -> 242\n",
      "gene: 1193 -> 279\n",
      "gene: 1194 -> 291\n",
      "gene: 1195 -> 288\n",
      "gene: 1196 -> 277\n",
      "gene: 1197 -> 184\n",
      "gene: 1203 -> 191\n",
      "gene: 1204 -> 300\n",
      "gene: 1205 -> 300\n",
      "gene: 1206 -> 108\n",
      "gene: 1207 -> 213\n",
      "gene: 1208 -> 208\n",
      "gene: 1209 -> 169\n",
      "gene: 1210 -> 270\n",
      "gene: 1211 -> 300\n",
      "gene: 1212 -> 278\n",
      "gene: 1213 -> 293\n",
      "gene: 1214 -> 300\n",
      "gene: 1215 -> 221\n",
      "gene: 1216 -> 216\n",
      "gene: 1217 -> 190\n",
      "gene: 1218 -> 235\n",
      "gene: 1219 -> 300\n",
      "gene: 1220 -> 300\n",
      "gene: 1221 -> 300\n",
      "gene: 1222 -> 179\n",
      "gene: 1224 -> 231\n",
      "gene: 1225 -> 282\n",
      "gene: 1226 -> 238\n",
      "gene: 1227 -> 215\n",
      "gene: 1228 -> 167\n",
      "gene: 1229 -> 235\n",
      "gene: 1230 -> 223\n",
      "gene: 1231 -> 241\n",
      "gene: 1232 -> 237\n",
      "gene: 1233 -> 223\n",
      "gene: 1234 -> 169\n",
      "gene: 1235 -> 241\n",
      "gene: 1236 -> 271\n",
      "gene: 1237 -> 228\n",
      "gene: 1238 -> 287\n",
      "gene: 1239 -> 249\n",
      "gene: 1240 -> 297\n",
      "gene: 1241 -> 300\n",
      "gene: 1242 -> 279\n",
      "gene: 1243 -> 297\n",
      "gene: 1244 -> 260\n",
      "gene: 1245 -> 283\n",
      "gene: 1246 -> 300\n",
      "gene: 1247 -> 234\n",
      "gene: 1248 -> 160\n",
      "gene: 1249 -> 237\n",
      "gene: 1250 -> 300\n",
      "gene: 1251 -> 181\n",
      "gene: 1252 -> 181\n",
      "gene: 1253 -> 246\n",
      "gene: 1254 -> 178\n",
      "gene: 1255 -> 300\n",
      "gene: 1256 -> 186\n",
      "gene: 1257 -> 173\n",
      "gene: 1258 -> 168\n",
      "gene: 1259 -> 216\n",
      "gene: 1260 -> 300\n",
      "gene: 1261 -> 300\n",
      "gene: 1262 -> 85\n",
      "gene: 1263 -> 189\n",
      "gene: 1264 -> 279\n",
      "gene: 1265 -> 99\n",
      "gene: 1269 -> 51\n",
      "gene: 1270 -> 300\n",
      "gene: 1271 -> 300\n",
      "gene: 1272 -> 300\n",
      "gene: 1273 -> 297\n",
      "gene: 1274 -> 192\n",
      "gene: 1275 -> 300\n",
      "gene: 1276 -> 244\n",
      "gene: 1277 -> 300\n",
      "gene: 1278 -> 150\n",
      "gene: 1279 -> 193\n",
      "gene: 1280 -> 296\n",
      "gene: 1281 -> 275\n",
      "gene: 1282 -> 234\n",
      "gene: 1284 -> 188\n",
      "gene: 1285 -> 206\n",
      "gene: 1286 -> 263\n",
      "gene: 1287 -> 287\n",
      "gene: 1288 -> 239\n",
      "gene: 1289 -> 300\n",
      "gene: 1290 -> 300\n",
      "gene: 1291 -> 273\n",
      "gene: 1292 -> 248\n",
      "gene: 1293 -> 300\n",
      "gene: 1294 -> 90\n",
      "gene: 1295 -> 167\n",
      "gene: 1297 -> 300\n",
      "gene: 1298 -> 282\n",
      "gene: 1299 -> 267\n",
      "gene: 1300 -> 266\n",
      "gene: 1301 -> 252\n",
      "gene: 1302 -> 220\n",
      "gene: 1303 -> 174\n",
      "gene: 1304 -> 268\n",
      "gene: 1305 -> 297\n",
      "gene: 1306 -> 285\n",
      "gene: 1307 -> 271\n",
      "gene: 1308 -> 243\n",
      "gene: 1309 -> 300\n",
      "gene: 1310 -> 285\n",
      "gene: 1311 -> 211\n",
      "gene: 1312 -> 257\n",
      "gene: 1313 -> 201\n",
      "gene: 1314 -> 272\n",
      "gene: 1315 -> 300\n",
      "gene: 1316 -> 300\n",
      "gene: 1317 -> 300\n",
      "gene: 1318 -> 253\n",
      "gene: 1319 -> 300\n",
      "gene: 1320 -> 286\n",
      "gene: 1321 -> 300\n",
      "gene: 1322 -> 300\n",
      "gene: 1323 -> 300\n",
      "gene: 1324 -> 300\n",
      "gene: 1325 -> 300\n",
      "gene: 1330 -> 129\n",
      "gene: 1331 -> 242\n",
      "gene: 1332 -> 278\n",
      "gene: 1333 -> 136\n",
      "gene: 1334 -> 161\n",
      "gene: 1335 -> 300\n",
      "gene: 1336 -> 266\n",
      "gene: 1337 -> 300\n",
      "gene: 1338 -> 188\n",
      "gene: 1339 -> 238\n",
      "gene: 1340 -> 300\n",
      "gene: 1341 -> 300\n",
      "gene: 1342 -> 261\n",
      "gene: 1343 -> 201\n",
      "gene: 1344 -> 274\n",
      "gene: 1345 -> 300\n",
      "gene: 1346 -> 298\n",
      "gene: 1347 -> 179\n",
      "gene: 1348 -> 130\n",
      "gene: 1349 -> 300\n",
      "gene: 1350 -> 265\n",
      "gene: 1351 -> 300\n",
      "gene: 1352 -> 243\n",
      "gene: 1353 -> 192\n",
      "gene: 1354 -> 278\n",
      "gene: 1355 -> 187\n",
      "gene: 1356 -> 291\n",
      "gene: 1357 -> 248\n",
      "gene: 1358 -> 300\n",
      "gene: 1359 -> 267\n",
      "gene: 1360 -> 300\n",
      "gene: 1361 -> 300\n",
      "gene: 1362 -> 300\n",
      "gene: 1363 -> 300\n",
      "gene: 1378 -> 217\n",
      "gene: 1379 -> 292\n",
      "gene: 1380 -> 192\n",
      "gene: 1381 -> 214\n",
      "gene: 1382 -> 210\n",
      "gene: 1383 -> 230\n",
      "gene: 1384 -> 148\n",
      "gene: 1385 -> 163\n",
      "gene: 1386 -> 128\n",
      "gene: 1387 -> 190\n",
      "gene: 1388 -> 176\n",
      "gene: 1389 -> 251\n",
      "gene: 1390 -> 288\n",
      "gene: 1391 -> 233\n",
      "gene: 1392 -> 244\n",
      "gene: 1393 -> 300\n",
      "gene: 1394 -> 174\n",
      "gene: 1395 -> 282\n",
      "gene: 1396 -> 192\n",
      "gene: 1398 -> 184\n",
      "gene: 1399 -> 268\n",
      "gene: 1400 -> 300\n",
      "gene: 1401 -> 268\n",
      "gene: 1402 -> 300\n",
      "gene: 1403 -> 223\n",
      "gene: 1404 -> 235\n",
      "gene: 1405 -> 300\n",
      "gene: 1406 -> 300\n",
      "gene: 1408 -> 300\n",
      "gene: 1409 -> 300\n",
      "gene: 1427 -> 253\n",
      "gene: 1429 -> 266\n",
      "gene: 1430 -> 240\n",
      "gene: 1434 -> 295\n",
      "gene: 1435 -> 255\n",
      "gene: 1436 -> 298\n",
      "gene: 1437 -> 300\n",
      "gene: 1438 -> 174\n",
      "gene: 1439 -> 198\n",
      "gene: 1440 -> 236\n",
      "gene: 1441 -> 184\n",
      "gene: 1442 -> 248\n",
      "gene: 1443 -> 300\n",
      "gene: 1444 -> 300\n",
      "gene: 1445 -> 300\n",
      "gene: 1446 -> 274\n",
      "gene: 1447 -> 300\n",
      "gene: 1448 -> 267\n",
      "gene: 1449 -> 300\n",
      "gene: 1450 -> 211\n",
      "gene: 1451 -> 255\n",
      "gene: 1452 -> 113\n",
      "gene: 1453 -> 300\n",
      "gene: 1454 -> 277\n",
      "gene: 1455 -> 290\n",
      "gene: 1456 -> 270\n",
      "gene: 1457 -> 281\n",
      "gene: 1458 -> 300\n",
      "gene: 1459 -> 300\n",
      "gene: 1461 -> 143\n",
      "gene: 1462 -> 238\n",
      "gene: 1463 -> 268\n",
      "gene: 1464 -> 281\n",
      "gene: 1465 -> 231\n",
      "gene: 1466 -> 286\n",
      "gene: 1467 -> 193\n",
      "gene: 1468 -> 180\n",
      "gene: 1469 -> 202\n",
      "gene: 1470 -> 194\n",
      "gene: 1471 -> 221\n",
      "gene: 1472 -> 278\n",
      "gene: 1473 -> 117\n",
      "gene: 1474 -> 173\n",
      "gene: 1475 -> 234\n",
      "gene: 1476 -> 197\n",
      "gene: 1477 -> 300\n",
      "gene: 1478 -> 230\n",
      "gene: 1479 -> 216\n",
      "gene: 1480 -> 117\n",
      "gene: 1481 -> 198\n",
      "gene: 1482 -> 212\n",
      "gene: 1485 -> 300\n",
      "gene: 1486 -> 227\n",
      "gene: 1487 -> 250\n",
      "gene: 1488 -> 300\n",
      "gene: 1489 -> 94\n",
      "gene: 1490 -> 128\n",
      "gene: 1491 -> 239\n",
      "gene: 1492 -> 238\n",
      "gene: 1493 -> 129\n",
      "gene: 1494 -> 189\n",
      "gene: 1495 -> 199\n",
      "gene: 1496 -> 72\n",
      "gene: 1497 -> 230\n",
      "gene: 1498 -> 146\n",
      "gene: 1499 -> 160\n",
      "gene: 1500 -> 300\n",
      "gene: 1501 -> 253\n",
      "gene: 1502 -> 139\n",
      "gene: 1503 -> 186\n",
      "gene: 1504 -> 285\n",
      "gene: 1505 -> 239\n",
      "gene: 1506 -> 163\n",
      "gene: 1507 -> 242\n",
      "gene: 1508 -> 189\n",
      "gene: 1509 -> 188\n",
      "gene: 1510 -> 176\n",
      "gene: 1511 -> 218\n",
      "gene: 1512 -> 248\n",
      "gene: 1513 -> 245\n",
      "gene: 1514 -> 262\n",
      "gene: 1515 -> 300\n",
      "gene: 1516 -> 103\n",
      "gene: 1517 -> 232\n",
      "gene: 1518 -> 217\n",
      "gene: 1519 -> 239\n",
      "gene: 1520 -> 221\n",
      "gene: 1521 -> 176\n",
      "gene: 1525 -> 51\n",
      "gene: 1526 -> 60\n",
      "gene: 1527 -> 72\n",
      "gene: 1529 -> 166\n",
      "gene: 1530 -> 218\n",
      "gene: 1533 -> 69\n",
      "gene: 1534 -> 149\n",
      "gene: 1535 -> 187\n",
      "gene: 1536 -> 187\n",
      "gene: 1537 -> 170\n",
      "gene: 1538 -> 164\n",
      "gene: 1541 -> 251\n",
      "gene: 1542 -> 204\n",
      "gene: 1548 -> 106\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "reg_size_dic, len_check = check._check_region_size(full_records, \n",
    "                                                   min_size=min_num_probes)\n",
    "print(len_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "id": "95f60f31",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "/* global mpl */\n",
       "window.mpl = {};\n",
       "\n",
       "mpl.get_websocket_type = function () {\n",
       "    if (typeof WebSocket !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof MozWebSocket !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert(\n",
       "            'Your browser does not have WebSocket support. ' +\n",
       "                'Please try Chrome, Safari or Firefox  6. ' +\n",
       "                'Firefox 4 and 5 are also supported but you ' +\n",
       "                'have to enable WebSockets in about:config.'\n",
       "        );\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure = function (figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = this.ws.binaryType !== undefined;\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById('mpl-warnings');\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent =\n",
       "                'This browser does not support binary websocket messages. ' +\n",
       "                'Performance may be slow.';\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = document.createElement('div');\n",
       "    this.root.setAttribute('style', 'display: inline-block');\n",
       "    this._root_extra_style(this.root);\n",
       "\n",
       "    parent_element.appendChild(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen = function () {\n",
       "        fig.send_message('supports_binary', { value: fig.supports_binary });\n",
       "        fig.send_message('send_image_mode', {});\n",
       "        if (fig.ratio !== 1) {\n",
       "            fig.send_message('set_device_pixel_ratio', {\n",
       "                device_pixel_ratio: fig.ratio,\n",
       "            });\n",
       "        }\n",
       "        fig.send_message('refresh', {});\n",
       "    };\n",
       "\n",
       "    this.imageObj.onload = function () {\n",
       "        if (fig.image_mode === 'full') {\n",
       "            // Full images could contain transparency (where diff images\n",
       "            // almost always do), so we need to clear the canvas so that\n",
       "            // there is no ghosting.\n",
       "            fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "        }\n",
       "        fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "    };\n",
       "\n",
       "    this.imageObj.onunload = function () {\n",
       "        fig.ws.close();\n",
       "    };\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_header = function () {\n",
       "    var titlebar = document.createElement('div');\n",
       "    titlebar.classList =\n",
       "        'ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix';\n",
       "    var titletext = document.createElement('div');\n",
       "    titletext.classList = 'ui-dialog-title';\n",
       "    titletext.setAttribute(\n",
       "        'style',\n",
       "        'width: 100%; text-align: center; padding: 3px;'\n",
       "    );\n",
       "    titlebar.appendChild(titletext);\n",
       "    this.root.appendChild(titlebar);\n",
       "    this.header = titletext;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (_canvas_div) {};\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = (this.canvas_div = document.createElement('div'));\n",
       "    canvas_div.setAttribute(\n",
       "        'style',\n",
       "        'border: 1px solid #ddd;' +\n",
       "            'box-sizing: content-box;' +\n",
       "            'clear: both;' +\n",
       "            'min-height: 1px;' +\n",
       "            'min-width: 1px;' +\n",
       "            'outline: 0;' +\n",
       "            'overflow: hidden;' +\n",
       "            'position: relative;' +\n",
       "            'resize: both;'\n",
       "    );\n",
       "\n",
       "    function on_keyboard_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.key_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    canvas_div.addEventListener(\n",
       "        'keydown',\n",
       "        on_keyboard_event_closure('key_press')\n",
       "    );\n",
       "    canvas_div.addEventListener(\n",
       "        'keyup',\n",
       "        on_keyboard_event_closure('key_release')\n",
       "    );\n",
       "\n",
       "    this._canvas_extra_style(canvas_div);\n",
       "    this.root.appendChild(canvas_div);\n",
       "\n",
       "    var canvas = (this.canvas = document.createElement('canvas'));\n",
       "    canvas.classList.add('mpl-canvas');\n",
       "    canvas.setAttribute('style', 'box-sizing: content-box;');\n",
       "\n",
       "    this.context = canvas.getContext('2d');\n",
       "\n",
       "    var backingStore =\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        this.context.webkitBackingStorePixelRatio ||\n",
       "        this.context.mozBackingStorePixelRatio ||\n",
       "        this.context.msBackingStorePixelRatio ||\n",
       "        this.context.oBackingStorePixelRatio ||\n",
       "        this.context.backingStorePixelRatio ||\n",
       "        1;\n",
       "\n",
       "    this.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband_canvas = (this.rubberband_canvas = document.createElement(\n",
       "        'canvas'\n",
       "    ));\n",
       "    rubberband_canvas.setAttribute(\n",
       "        'style',\n",
       "        'box-sizing: content-box; position: absolute; left: 0; top: 0; z-index: 1;'\n",
       "    );\n",
       "\n",
       "    // Apply a ponyfill if ResizeObserver is not implemented by browser.\n",
       "    if (this.ResizeObserver === undefined) {\n",
       "        if (window.ResizeObserver !== undefined) {\n",
       "            this.ResizeObserver = window.ResizeObserver;\n",
       "        } else {\n",
       "            var obs = _JSXTOOLS_RESIZE_OBSERVER({});\n",
       "            this.ResizeObserver = obs.ResizeObserver;\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.resizeObserverInstance = new this.ResizeObserver(function (entries) {\n",
       "        var nentries = entries.length;\n",
       "        for (var i = 0; i < nentries; i++) {\n",
       "            var entry = entries[i];\n",
       "            var width, height;\n",
       "            if (entry.contentBoxSize) {\n",
       "                if (entry.contentBoxSize instanceof Array) {\n",
       "                    // Chrome 84 implements new version of spec.\n",
       "                    width = entry.contentBoxSize[0].inlineSize;\n",
       "                    height = entry.contentBoxSize[0].blockSize;\n",
       "                } else {\n",
       "                    // Firefox implements old version of spec.\n",
       "                    width = entry.contentBoxSize.inlineSize;\n",
       "                    height = entry.contentBoxSize.blockSize;\n",
       "                }\n",
       "            } else {\n",
       "                // Chrome <84 implements even older version of spec.\n",
       "                width = entry.contentRect.width;\n",
       "                height = entry.contentRect.height;\n",
       "            }\n",
       "\n",
       "            // Keep the size of the canvas and rubber band canvas in sync with\n",
       "            // the canvas container.\n",
       "            if (entry.devicePixelContentBoxSize) {\n",
       "                // Chrome 84 implements new version of spec.\n",
       "                canvas.setAttribute(\n",
       "                    'width',\n",
       "                    entry.devicePixelContentBoxSize[0].inlineSize\n",
       "                );\n",
       "                canvas.setAttribute(\n",
       "                    'height',\n",
       "                    entry.devicePixelContentBoxSize[0].blockSize\n",
       "                );\n",
       "            } else {\n",
       "                canvas.setAttribute('width', width * fig.ratio);\n",
       "                canvas.setAttribute('height', height * fig.ratio);\n",
       "            }\n",
       "            canvas.setAttribute(\n",
       "                'style',\n",
       "                'width: ' + width + 'px; height: ' + height + 'px;'\n",
       "            );\n",
       "\n",
       "            rubberband_canvas.setAttribute('width', width);\n",
       "            rubberband_canvas.setAttribute('height', height);\n",
       "\n",
       "            // And update the size in Python. We ignore the initial 0/0 size\n",
       "            // that occurs as the element is placed into the DOM, which should\n",
       "            // otherwise not happen due to the minimum size styling.\n",
       "            if (fig.ws.readyState == 1 && width != 0 && height != 0) {\n",
       "                fig.request_resize(width, height);\n",
       "            }\n",
       "        }\n",
       "    });\n",
       "    this.resizeObserverInstance.observe(canvas_div);\n",
       "\n",
       "    function on_mouse_event_closure(name) {\n",
       "        return function (event) {\n",
       "            return fig.mouse_event(event, name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousedown',\n",
       "        on_mouse_event_closure('button_press')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseup',\n",
       "        on_mouse_event_closure('button_release')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'dblclick',\n",
       "        on_mouse_event_closure('dblclick')\n",
       "    );\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mousemove',\n",
       "        on_mouse_event_closure('motion_notify')\n",
       "    );\n",
       "\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseenter',\n",
       "        on_mouse_event_closure('figure_enter')\n",
       "    );\n",
       "    rubberband_canvas.addEventListener(\n",
       "        'mouseleave',\n",
       "        on_mouse_event_closure('figure_leave')\n",
       "    );\n",
       "\n",
       "    canvas_div.addEventListener('wheel', function (event) {\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        on_mouse_event_closure('scroll')(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.appendChild(canvas);\n",
       "    canvas_div.appendChild(rubberband_canvas);\n",
       "\n",
       "    this.rubberband_context = rubberband_canvas.getContext('2d');\n",
       "    this.rubberband_context.strokeStyle = '#000000';\n",
       "\n",
       "    this._resize_canvas = function (width, height, forward) {\n",
       "        if (forward) {\n",
       "            canvas_div.style.width = width + 'px';\n",
       "            canvas_div.style.height = height + 'px';\n",
       "        }\n",
       "    };\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    this.rubberband_canvas.addEventListener('contextmenu', function (_e) {\n",
       "        event.preventDefault();\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus() {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'mpl-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'mpl-button-group';\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'mpl-button-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        var button = (fig.buttons[name] = document.createElement('button'));\n",
       "        button.classList = 'mpl-widget';\n",
       "        button.setAttribute('role', 'button');\n",
       "        button.setAttribute('aria-disabled', 'false');\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "\n",
       "        var icon_img = document.createElement('img');\n",
       "        icon_img.src = '_images/' + image + '.png';\n",
       "        icon_img.srcset = '_images/' + image + '_large.png 2x';\n",
       "        icon_img.alt = tooltip;\n",
       "        button.appendChild(icon_img);\n",
       "\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    var fmt_picker = document.createElement('select');\n",
       "    fmt_picker.classList = 'mpl-widget';\n",
       "    toolbar.appendChild(fmt_picker);\n",
       "    this.format_dropdown = fmt_picker;\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = document.createElement('option');\n",
       "        option.selected = fmt === mpl.default_extension;\n",
       "        option.innerHTML = fmt;\n",
       "        fmt_picker.appendChild(option);\n",
       "    }\n",
       "\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.request_resize = function (x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', { width: x_pixels, height: y_pixels });\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_message = function (type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function () {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({ type: 'draw', figure_id: this.id }));\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function (fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] !== fig.canvas.width || size[1] !== fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1], msg['forward']);\n",
       "        fig.send_message('refresh', {});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function (fig, msg) {\n",
       "    var x0 = msg['x0'] / fig.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / fig.ratio;\n",
       "    var x1 = msg['x1'] / fig.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / fig.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0,\n",
       "        0,\n",
       "        fig.canvas.width / fig.ratio,\n",
       "        fig.canvas.height / fig.ratio\n",
       "    );\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function (fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function (fig, msg) {\n",
       "    fig.rubberband_canvas.style.cursor = msg['cursor'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_message = function (fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function (fig, _msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function (fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_history_buttons = function (fig, msg) {\n",
       "    for (var key in msg) {\n",
       "        if (!(key in fig.buttons)) {\n",
       "            continue;\n",
       "        }\n",
       "        fig.buttons[key].disabled = !msg[key];\n",
       "        fig.buttons[key].setAttribute('aria-disabled', !msg[key]);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_navigate_mode = function (fig, msg) {\n",
       "    if (msg['mode'] === 'PAN') {\n",
       "        fig.buttons['Pan'].classList.add('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    } else if (msg['mode'] === 'ZOOM') {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.add('active');\n",
       "    } else {\n",
       "        fig.buttons['Pan'].classList.remove('active');\n",
       "        fig.buttons['Zoom'].classList.remove('active');\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message('ack', {});\n",
       "};\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function (fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            var img = evt.data;\n",
       "            if (img.type !== 'image/png') {\n",
       "                /* FIXME: We get \"Resource interpreted as Image but\n",
       "                 * transferred with MIME type text/plain:\" errors on\n",
       "                 * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "                 * to be part of the websocket stream */\n",
       "                img.type = 'image/png';\n",
       "            }\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src\n",
       "                );\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                img\n",
       "            );\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        } else if (\n",
       "            typeof evt.data === 'string' &&\n",
       "            evt.data.slice(0, 21) === 'data:image/png;base64'\n",
       "        ) {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig['handle_' + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\n",
       "                \"No handler for the '\" + msg_type + \"' message type: \",\n",
       "                msg\n",
       "            );\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\n",
       "                    \"Exception inside the 'handler_\" + msg_type + \"' callback:\",\n",
       "                    e,\n",
       "                    e.stack,\n",
       "                    msg\n",
       "                );\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "};\n",
       "\n",
       "// from https://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function (e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e) {\n",
       "        e = window.event;\n",
       "    }\n",
       "    if (e.target) {\n",
       "        targ = e.target;\n",
       "    } else if (e.srcElement) {\n",
       "        targ = e.srcElement;\n",
       "    }\n",
       "    if (targ.nodeType === 3) {\n",
       "        // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "    }\n",
       "\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    var boundingRect = targ.getBoundingClientRect();\n",
       "    var x = e.pageX - (boundingRect.left + document.body.scrollLeft);\n",
       "    var y = e.pageY - (boundingRect.top + document.body.scrollTop);\n",
       "\n",
       "    return { x: x, y: y };\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * https://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys(original) {\n",
       "    return Object.keys(original).reduce(function (obj, key) {\n",
       "        if (typeof original[key] !== 'object') {\n",
       "            obj[key] = original[key];\n",
       "        }\n",
       "        return obj;\n",
       "    }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function (event, name) {\n",
       "    var canvas_pos = mpl.findpos(event);\n",
       "\n",
       "    if (name === 'button_press') {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * this.ratio;\n",
       "    var y = canvas_pos.y * this.ratio;\n",
       "\n",
       "    this.send_message(name, {\n",
       "        x: x,\n",
       "        y: y,\n",
       "        button: event.button,\n",
       "        step: event.step,\n",
       "        guiEvent: simpleKeys(event),\n",
       "    });\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (_event, _name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.key_event = function (event, name) {\n",
       "    // Prevent repeat events\n",
       "    if (name === 'key_press') {\n",
       "        if (event.key === this._key) {\n",
       "            return;\n",
       "        } else {\n",
       "            this._key = event.key;\n",
       "        }\n",
       "    }\n",
       "    if (name === 'key_release') {\n",
       "        this._key = null;\n",
       "    }\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.key !== 'Control') {\n",
       "        value += 'ctrl+';\n",
       "    }\n",
       "    else if (event.altKey && event.key !== 'Alt') {\n",
       "        value += 'alt+';\n",
       "    }\n",
       "    else if (event.shiftKey && event.key !== 'Shift') {\n",
       "        value += 'shift+';\n",
       "    }\n",
       "\n",
       "    value += 'k' + event.key;\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, { key: value, guiEvent: simpleKeys(event) });\n",
       "    return false;\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function (name) {\n",
       "    if (name === 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message('toolbar_button', { name: name });\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function (tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "\n",
       "///////////////// REMAINING CONTENT GENERATED BY embed_js.py /////////////////\n",
       "// prettier-ignore\n",
       "var _JSXTOOLS_RESIZE_OBSERVER=function(A){var t,i=new WeakMap,n=new WeakMap,a=new WeakMap,r=new WeakMap,o=new Set;function s(e){if(!(this instanceof s))throw new TypeError(\"Constructor requires 'new' operator\");i.set(this,e)}function h(){throw new TypeError(\"Function is not a constructor\")}function c(e,t,i,n){e=0 in arguments?Number(arguments[0]):0,t=1 in arguments?Number(arguments[1]):0,i=2 in arguments?Number(arguments[2]):0,n=3 in arguments?Number(arguments[3]):0,this.right=(this.x=this.left=e)+(this.width=i),this.bottom=(this.y=this.top=t)+(this.height=n),Object.freeze(this)}function d(){t=requestAnimationFrame(d);var s=new WeakMap,p=new Set;o.forEach((function(t){r.get(t).forEach((function(i){var r=t instanceof window.SVGElement,o=a.get(t),d=r?0:parseFloat(o.paddingTop),f=r?0:parseFloat(o.paddingRight),l=r?0:parseFloat(o.paddingBottom),u=r?0:parseFloat(o.paddingLeft),g=r?0:parseFloat(o.borderTopWidth),m=r?0:parseFloat(o.borderRightWidth),w=r?0:parseFloat(o.borderBottomWidth),b=u+f,F=d+l,v=(r?0:parseFloat(o.borderLeftWidth))+m,W=g+w,y=r?0:t.offsetHeight-W-t.clientHeight,E=r?0:t.offsetWidth-v-t.clientWidth,R=b+v,z=F+W,M=r?t.width:parseFloat(o.width)-R-E,O=r?t.height:parseFloat(o.height)-z-y;if(n.has(t)){var k=n.get(t);if(k[0]===M&&k[1]===O)return}n.set(t,[M,O]);var S=Object.create(h.prototype);S.target=t,S.contentRect=new c(u,d,M,O),s.has(i)||(s.set(i,[]),p.add(i)),s.get(i).push(S)}))})),p.forEach((function(e){i.get(e).call(e,s.get(e),e)}))}return s.prototype.observe=function(i){if(i instanceof window.Element){r.has(i)||(r.set(i,new Set),o.add(i),a.set(i,window.getComputedStyle(i)));var n=r.get(i);n.has(this)||n.add(this),cancelAnimationFrame(t),t=requestAnimationFrame(d)}},s.prototype.unobserve=function(i){if(i instanceof window.Element&&r.has(i)){var n=r.get(i);n.has(this)&&(n.delete(this),n.size||(r.delete(i),o.delete(i))),n.size||r.delete(i),o.size||cancelAnimationFrame(t)}},A.DOMRectReadOnly=c,A.ResizeObserver=s,A.ResizeObserverEntry=h,A}; // eslint-disable-line\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Left button pans, Right button zooms\\nx/y fixes axis, CTRL fixes aspect\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\\nx/y fixes axis\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pgf\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";/* global mpl */\n",
       "\n",
       "var comm_websocket_adapter = function (comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.binaryType = comm.kernel.ws.binaryType;\n",
       "    ws.readyState = comm.kernel.ws.readyState;\n",
       "    function updateReadyState(_event) {\n",
       "        if (comm.kernel.ws) {\n",
       "            ws.readyState = comm.kernel.ws.readyState;\n",
       "        } else {\n",
       "            ws.readyState = 3; // Closed state.\n",
       "        }\n",
       "    }\n",
       "    comm.kernel.ws.addEventListener('open', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('close', updateReadyState);\n",
       "    comm.kernel.ws.addEventListener('error', updateReadyState);\n",
       "\n",
       "    ws.close = function () {\n",
       "        comm.close();\n",
       "    };\n",
       "    ws.send = function (m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function (msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        var data = msg['content']['data'];\n",
       "        if (data['blob'] !== undefined) {\n",
       "            data = {\n",
       "                data: new Blob(msg['buffers'], { type: data['blob'] }),\n",
       "            };\n",
       "        }\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(data);\n",
       "    });\n",
       "    return ws;\n",
       "};\n",
       "\n",
       "mpl.mpl_figure_comm = function (comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = document.getElementById(id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm);\n",
       "\n",
       "    function ondownload(figure, _format) {\n",
       "        window.open(figure.canvas.toDataURL());\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy, ondownload, element);\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element;\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error('Failed to find cell for figure', id, fig);\n",
       "        return;\n",
       "    }\n",
       "    fig.cell_info[0].output_area.element.on(\n",
       "        'cleared',\n",
       "        { fig: fig },\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function (fig, msg) {\n",
       "    var width = fig.canvas.width / fig.ratio;\n",
       "    fig.cell_info[0].output_area.element.off(\n",
       "        'cleared',\n",
       "        fig._remove_fig_handler\n",
       "    );\n",
       "    fig.resizeObserverInstance.unobserve(fig.canvas_div);\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable();\n",
       "    fig.parent_element.innerHTML =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "    fig.close_ws(fig, msg);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.close_ws = function (fig, msg) {\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function (_remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width / this.ratio;\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] =\n",
       "        '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function () {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message('ack', {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () {\n",
       "        fig.push_to_output();\n",
       "    }, 1000);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function () {\n",
       "    var fig = this;\n",
       "\n",
       "    var toolbar = document.createElement('div');\n",
       "    toolbar.classList = 'btn-toolbar';\n",
       "    this.root.appendChild(toolbar);\n",
       "\n",
       "    function on_click_closure(name) {\n",
       "        return function (_event) {\n",
       "            return fig.toolbar_button_onclick(name);\n",
       "        };\n",
       "    }\n",
       "\n",
       "    function on_mouseover_closure(tooltip) {\n",
       "        return function (event) {\n",
       "            if (!event.currentTarget.disabled) {\n",
       "                return fig.toolbar_button_onmouseover(tooltip);\n",
       "            }\n",
       "        };\n",
       "    }\n",
       "\n",
       "    fig.buttons = {};\n",
       "    var buttonGroup = document.createElement('div');\n",
       "    buttonGroup.classList = 'btn-group';\n",
       "    var button;\n",
       "    for (var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            /* Instead of a spacer, we start a new button group. */\n",
       "            if (buttonGroup.hasChildNodes()) {\n",
       "                toolbar.appendChild(buttonGroup);\n",
       "            }\n",
       "            buttonGroup = document.createElement('div');\n",
       "            buttonGroup.classList = 'btn-group';\n",
       "            continue;\n",
       "        }\n",
       "\n",
       "        button = fig.buttons[name] = document.createElement('button');\n",
       "        button.classList = 'btn btn-default';\n",
       "        button.href = '#';\n",
       "        button.title = name;\n",
       "        button.innerHTML = '<i class=\"fa ' + image + ' fa-lg\"></i>';\n",
       "        button.addEventListener('click', on_click_closure(method_name));\n",
       "        button.addEventListener('mouseover', on_mouseover_closure(tooltip));\n",
       "        buttonGroup.appendChild(button);\n",
       "    }\n",
       "\n",
       "    if (buttonGroup.hasChildNodes()) {\n",
       "        toolbar.appendChild(buttonGroup);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = document.createElement('span');\n",
       "    status_bar.classList = 'mpl-message pull-right';\n",
       "    toolbar.appendChild(status_bar);\n",
       "    this.message = status_bar;\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = document.createElement('div');\n",
       "    buttongrp.classList = 'btn-group inline pull-right';\n",
       "    button = document.createElement('button');\n",
       "    button.classList = 'btn btn-mini btn-primary';\n",
       "    button.href = '#';\n",
       "    button.title = 'Stop Interaction';\n",
       "    button.innerHTML = '<i class=\"fa fa-power-off icon-remove icon-large\"></i>';\n",
       "    button.addEventListener('click', function (_evt) {\n",
       "        fig.handle_close(fig, {});\n",
       "    });\n",
       "    button.addEventListener(\n",
       "        'mouseover',\n",
       "        on_mouseover_closure('Stop Interaction')\n",
       "    );\n",
       "    buttongrp.appendChild(button);\n",
       "    var titlebar = this.root.querySelector('.ui-dialog-titlebar');\n",
       "    titlebar.insertBefore(buttongrp, titlebar.firstChild);\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._remove_fig_handler = function (event) {\n",
       "    var fig = event.data.fig;\n",
       "    if (event.target !== this) {\n",
       "        // Ignore bubbled events from children.\n",
       "        return;\n",
       "    }\n",
       "    fig.close_ws(fig, {});\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function (el) {\n",
       "    el.style.boxSizing = 'content-box'; // override notebook setting of border-box.\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function (el) {\n",
       "    // this is important to make the div 'focusable\n",
       "    el.setAttribute('tabindex', 0);\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    } else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function (event, _name) {\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which === 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_save = function (fig, _msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "};\n",
       "\n",
       "mpl.find_output_cell = function (html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i = 0; i < ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code') {\n",
       "            for (var j = 0; j < cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] === html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "};\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel !== null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target(\n",
       "        'matplotlib',\n",
       "        mpl.mpl_figure_comm\n",
       "    );\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAsAAAAIQCAYAAACPEdjAAAAAAXNSR0IArs4c6QAAIABJREFUeF7t3QuoXdWdP/BlFB9TGR2MEV+QNip2rLGtbXxgHRzRqiMi1YmpoFaUqVofoCbUOMVHrVULth00mtZRwRfUGuvj3wfNFMFpCVHUaVDUapo2U4OK1QpGk6nJn7Xv5IxJbnLX3ud37z45+3NARLPO2md/9vfc+8266+y71dq1a9cmDwIECBAgQIAAAQIdEdhKAe7IlXaaBAgQIECAAAEClYACLAgECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACLAMECBAgQIAAAQKdElCAO3W5nSwBAgQIECBAgIACPI4Z+OCDD9KSJUvSrrvumrbZZptxPJKpCRAgQIAAAQLrC/z1r39Nb775ZjrwwAPT9ttvj+cjAgrwOMbhqaeeSjNmzBjHI5iaAAECBAgQILB5gcWLF6fPf/7zmBTgicnAsmXL0sc//vGUg7f77rtPzEEdhQABAgQIECCQUlqxYkW1EPf73/8+TZ06lYkCPDEZ+O///u+09957p+XLl6e99tprYg7qKAQIECBAgACBlJIesukY2AIxjm8RwRtHXFMTIECAAAECmxXQQxTgVt4igtcKu4MSIECAAAECVoA3mwErwOP4FlGAxxHX1AQIECBAgIAV4IYZUIAbwpU8TQEuUTKGAAECBAgQGA8BPWTTqgrweCTuf+cUvHHENTUBAgQIECBgBbhhBhTghnAlT1OAS5SMIUCAAAECBMZDQA+xAjweuRpzTsEbk8gAAgQIECBAYJwE9BAFeJyitflpBa8VdgclQIAAAQIE3AVisxmwBWIc3yIK8DjimpoAAQIECBDYrIAeYgW4lbeI4LXC7qAECBAgQICAFWArwG29CxTgtuQdlwABAgQIENBDrAC38i4QvFbYHZQAAQIECBCwAmwFuK13gQLclrzjEiBAgAABAnqIFeBW3gWC1wq7gxIgQIAAAQJWgK0At/UuUIDbkndcAgQIECBAQA+xAtzKu0DwWmF3UAIECBAgQMAKsBXgtt4FCnBb8o5LgAABAgQI6CFWgFt5FwheK+wOSoAAAQJbsMDUr/+/xq9+2Q3/1Pi5w/hEPUQBbiXXgtcKu4MSIECAwBYsoADHXTw9RAGOS1ONmQSvBpahBAgQIEAgpaQAx8VAD1GA49JUYybBq4FlKAECBAgQUIBDM6CHKMChgSqdTPBKpYwjQIAAAQIjAlaA45KghyjAcWmqMZPg1cAylAABAgQIKMChGdBDFODQQJVOJnilUsYRIECAAAErwNEZ0EMU4OhMFc0neEVMBhEgQIAAgZ6ALRBxYdBDFOC4NNWYSfBqYBlKgAABAgRsgQjNgB6iAIcGqnQywSuVMo4AAQIECIwIWAGOS4IeogDHpanGTIJXA8tQAgQIECCgAIdmQA9RgEMDVTqZ4JVKGUeAAAECBKwAR2dAD1GAozNVNJ/gFTEZRIAAAQIEegK2QMSFQQ9RgOPSVGMmwauBZSgBAgQIELAFIjQDeogCHBqo0skEr1TKOAIECBAgMCJgBTguCXqIAhyXphozCV4NLEMJECBAgIACHJoBPUQBDg1U6WSCVyplHAECBAgQsAIcnQE9RAGOzlTRfIJXxGQQAQIECBDoCdgCERcGPUQBjktTjZkErwaWoQQIECBAwBaI0AzoIQpwaKBKJxO8UinjCBAgQIDAiIAV4Lgk6CEKcFyaaswkeDWwDCVAgAABAgpwaAb0EAU4NFClkwleqZRxBAgQIEDACnB0BvQQBTg6U0XzCV4Rk0EECBAgQKAnYAtEXBj0EAU4Lk01ZhK8GliGEiBAgAABWyBCM6CHKMChgSqdTPBKpYwjQIAAAQIjAlaA45KghyjAcWmqMZPg1cAylAABAgQIKMChGdBDFODQQJVOJnilUsYRIECAAAErwNEZ0EMU4OhMFc0neEVMBhEgQIAAgZ6ALRBxYdBDFOC4NNWYSfBqYBlKgAABAgRsgQjNgB6iAIcGqnQywSuVMo4AAQIECIwIWAGOS4IeogDHpanGTIJXA8tQAgQIECCgAIdmQA9RgEMDVTqZ4JVKGUeAAAECBKwAR2dAD1GAozNVNJ/gFTEZRIAAAQIEegK2QMSFQQ/pWAH+85//nObOnZsef/zx9O6776bp06enG264IR1xxBGVxNlnn53uvvvu9VT23HPPlIOSH2vWrEnXXHNNuuOOO9Lbb79dPW/evHlpn332qZVKwavFZTABAgQIELAHODADekjHCvCxxx6b3njjjXTLLbekXXfdNd16661VmX3mmWfS/vvvnw4++OB03HHHpYsuuqgns/XWW1dj8yOX31x477rrrpSL8Zw5c9LSpUvT888/n7bddtviaApeMZWBBAgQIECgErACHBcEPaRDBfiVV15J++67b/r1r3+dDj/88OrM165dm/bbb7/05S9/OV111VVpxx13TA888EA6+eSTN5JZvXp1mjx5crrpppvSeeedV/35O++8k/bYY4905513plmzZhUnU/CKqQwkQIAAAQIKcHAG9JAOFeBcVn/zm9+ko48+Om233Xa9M8/bF/Kq74UXXpg++clPppdeeqkqxRs+Fi9enA455JCN/jxvg8hbKfLKcOlD8EqljCNAgAABAiMCVoDjkqCHdKgAj3aqDz74YJo5c2Z69NFH0wcffJBOO+20anX3Zz/7WZo0aVI64YQT0nXXXZd22mmntGDBgnTKKaeklStXph122KE3XX5+/n95X3HpQ/BKpYwjQIAAAQIKcHQG9JAOF+C8FSKv/OYV4Z/85CfVFohvfetb6frrr6+K7+9+97t0+eWXp7333jv96le/Svfff38644wz0ocffliV43WPM888M7322mtp4cKFm9TMH7jL/6x7rFixIs2YMSMtX7487bXXXtG5Nh8BAgQIEBg6ASvAcZdUAe5oAX7kkUfS6aefng499NBq5Tav6Ob9wH/5y1/Szjvv3FNZtGhROuyww1L+dw7LqaeeOuoK8KpVq1Kec1OPq6++uvoA3YYPBTjuzWwmAgQIEBhuAQU47voqwB0swPkOEJdcckn60pe+lO6999719gNvyPHee+9VH4z78Y9/XK0E5z3A+cN006ZN6w3Ne4APOuig6o4Sm3pYAY5705qJAAECBLopoADHXXcFuGMF+LbbbksXXHBBuvjii9N3v/vd9bYy5BXht956K/3iF7/oqTzxxBPpqKOOSi+88EL6xCc+kaZMmZJuvvnmdM4551Rj3AUi7s1oJgIECBAgsDkBBTguHwpwhwrwyy+/nD71qU+lE088caM7NuQtEE8++WQ66aST0je/+c3qlmZ5/Ne+9rVqC8R9991XSV155ZVp/vz51W3Ppk6dmmbPnp2WLVuWlixZ4j7Ace9LMxEgQIAAgY0EFOC4UCjAHSrA+cNtucCO9jjrrLOq3wD30EMPVR+Ee/HFF6u9wHlVON8FYvvtt6+elj8Al3+TXP5FGO+//3468sgjq60PuQzXeQheHS1jCRAgQICA26BFZkAP6VABjgxOv3MJXr+Cnk+AAAECXROwAhx3xfUQBTguTTVmErwaWIYSIECAAAG/CCM0A3qIAhwaqNLJBK9UyjgCBAgQIDAiYAU4Lgl6iAIcl6YaMwleDSxDCRAgQICAAhyaAT1EAQ4NVOlkglcqZRwBAgQIELACHJ0BPUQBjs5U0XyCV8RkEAECBAgQ6AnYAhEXBj1EAY5LU42ZBK8GlqEECBAgQMAWiNAM6CEKcGigSicTvFIp4wgQIECAwIiAFeC4JOghCnBcmmrMJHg1sAwlQIAAAQIKcGgG9BAFODRQpZMJXqmUcQQIECBAwApwdAb0EAU4OlNF8wleEZNBBAgQIECgJ2ALRFwY9BAFOC5NNWYSvBpYhhIgQIAAAVsgQjOghyjAoYEqnUzwSqWMI0CAAAECIwJWgOOSoIcowHFpqjGT4NXAMpQAAQIECCjAoRnQQxTg0ECVTiZ4pVLGESBAgAABK8DRGdBDFODoTBXNJ3hFTAYRIECAAIGegC0QcWHQQxTguDTVmEnwamAZSoAAAQIEbIEIzYAeogCHBqp0MsErlTKOAAECBAiMCFgBjkuCHqIAx6WpxkyCVwPLUAIECBAgoACHZkAPUYBDA1U6meCVShlHgAABAgSsAEdnQA9RgKMzVTSf4BUxGUSAAAECBHoCtkDEhUEPUYDj0lRjJsGrgWUoAQIECBCwBSI0A3qIAhwaqNLJBK9UyjgCBAgQIDAiYAU4Lgl6iAIcl6YaMwleDSxDCRAgQICAAhyaAT1EAQ4NVOlkglcqZRwBAgQIELACHJ0BPUQBjs5U0XyCV8RkEAECBAgQ6AnYAhEXBj1EAY5LU42ZBK8GlqEECBAgQMAWiNAM6CEKcGigSicTvFIp4wgQIECAwIiAFeC4JOghCnBcmmrMJHg1sAwlQIAAAQIKcGgG9BAFODRQpZMJXqmUcQQIECBAwApwdAb0EAU4OlNF8wleEZNBBAgQIECgJ2ALRFwY9BAFOC5NNWYSvBpYhhIgQIAAAVsgQjOghyjAoYEqnUzwSqWMI0CAAAECIwJWgOOSoIcowHFpqjGT4NXAMpQAAQIECCjAoRnQQxTg0ECVTiZ4pVLGESBAgAABK8DRGdBDFODoTBXNJ3hFTAYRIECAAIGegC0QcWHQQxTguDTVmEnwamAZSoAAAQIEbIEIzYAeogCHBqp0MsErlTKOAAECBAiMCFgBjkuCHqIAx6WpxkyCVwPLUAIECBAgoACHZkAPUYBDA1U6meCVShlHgAABAgSsAEdnQA9RgKMzVTSf4BUxGUSAAAECBHoCtkDEhUEPUYDj0lRjJsGrgWUoAQIECBCwBSI0A3qIAhwaqNLJBK9UyjgCBAgQIDAiYAU4Lgl6iAIcl6YaMwleDSxDCRAgQICAAhyaAT1EAQ4NVOlkglcqZRwBAgQIELACHJ0BPUQBjs5U0XyCV8RkEAECBAgQ6AnYAhEXBj1EAY5LU42ZBK8GlqEECBAgQMAWiNAM6CEKcGigSicTvFIp4wgQIECAwIiAFeC4JOghCnBcmmrMJHg1sAwlQIAAAQIKcGgG9BAFODRQpZMJXqmUcQQIECBAwApwdAb0EAU4OlNF8wleEZNBBAgQIECgJ2ALRFwY9BAFOC5NNWYSvBpYhhIgQIAAAVsgQjOghyjAoYEqnUzwSqWMI0CAAAECIwJWgOOSoIcowHFpqjGT4NXAMpQAAQIECCjAoRnQQxTg0ECVTiZ4pVLGESBAgAABK8DRGdBDFODoTBXNJ3hFTAYRIECAAIGegC0QcWHQQxTguDTVmEnwamAZSoAAAQIEbIEIzYAeogCHBqp0MsErlTKOAAECBAiMCFgBjkuCHqIAx6WpxkyCVwPLUAIECBAgoACHZkAP6VgB/vOf/5zmzp2bHn/88fTuu++m6dOnpxtuuCEdccQRlcRzzz2XLrnkkvT000+nXXbZJV188cXp8ssv7ymtWbMmXXPNNemOO+5Ib7/9dvW8efPmpX322adWMAWvFpfBBAgQIEDACnBgBvSQjhXgY489Nr3xxhvplltuSbvuumu69dZbqzL7zDPPVP+9//77p5NPPjldeumladGiRemCCy6oCu7ZZ59dSeXym//7rrvuSnvuuWeaM2dOWrp0aXr++efTtttuWxxNwSumMpAAAQIECFQCtkDEBUEP6VABfuWVV9K+++6bfv3rX6fDDz+8OvO1a9em/fbbL335y19OO+ywQ1WIly1blrbZZpvqz/Nq8YIFC9KLL76YVq9enSZPnpxuuummdN5551V//s4776Q99tgj3XnnnWnWrFnFyRS8YioDCRAgQICAAhycAT2kQwU4l9Xf/OY36eijj07bbbdd78zz9oXjjjuuWsndeeed0/3339/7s4ULF6Zjjjkmvf7661UxPuSQQ9JLL71UleZ1j7wNIm+lyCvDpQ/BK5UyjgABAgQIjAhYAY5Lgh7SoQI82qk++OCDaebMmenRRx9NV155ZTr++OPTjTfe2Bv6wgsvpAMOOCA99dRT6Y9//GM65ZRT0sqVK6vV4nWP/Pz8//K+4k098n7j/M+6x4oVK9KMGTPS8uXL01577RWXaDMRIECAAIEhFVCA4y6sAtzhApy3QuSV37wi/JOf/KT6INvpp5+err322p5KXhWeNm1aevLJJ6sV4DPOOCN9+OGHadKkSb0xZ555ZnrttddSXi3e1OPqq6+u9g9v+FCA497MZiJAgACB4RZQgOOurwLc0QL8yCOPVGX30EMPrVZu84rugQcemE444YRRV4Dzh+RyGT711FNHXQFetWpVynNaAY57c5qJAAECBAh8VEABjsuDAtzBApzvAJFvdfalL30p3Xvvvb39wLn85luf3XPPPT2V0fYA5w/T5VXhdY+8B/iggw6qPkBX+hC8UinjCBAgQIDAiIACHJcEPaRjBfi2226rbm2W7+/73e9+d72tDPl+wLfffnt69dVX09Zbb13JXHHFFenhhx+u7gKRV3mnTJmSbr755nTOOedUf+4uEHFvRjMRIECAAIHNCSjAcflQgDtUgF9++eX0qU99Kp144okb3bEhb4HIBTffB/ikk06q7u+7ePHidP7551el+Kyzzqqk8gfl5s+fX932bOrUqWn27NnV3uAlS5a4D3Dc+9JMBAgQIEBgIwEFOC4UCnCHCvD1119fFdjRHrng3n333dXdHvLq8LPPPpt23333dNlll6ULL7yw95T8Abh8b+D8izDef//9dOSRR1ZbH3IZrvMQvDpaxhIgQIAAAVsgIjOgh3SoAEcGp9+5BK9fQc8nQIAAga4JWAGOu+J6iAIcl6YaMwleDSxDCRAgQICAD8GFZkAPUYBDA1U6meCVShlHgAABAgRGBKwAxyVBD1GA49JUYybBq4FlKAECBAgQUIBDM6CHKMChgSqdTPBKpYwjQIAAAQJWgKMzoIcowNGZKppP8IqYDCJAgAABAj0BWyDiwqCHKMBxaaoxk+DVwDKUAAECBAjYAhGaAT1EAQ4NVOlkglcqZRwBAgQIEBgRsAIclwQ9RAGOS1ONmQSvBpahBAgQIEBAAQ7NgB6iAIcGqnQywSuVMo4AAQIECFgBjs6AHqIAR2eqaD7BK2IyiAABAgQI9ARsgYgLgx6iAMelqcZMglcDy1ACBAgQIGALRGgG9BAFODRQpZMJXqmUcQQIECBAYETACnBcEvQQBTguTTVmErwaWIYSIECAAAEFODQDeogCHBqo0skEr1TKOAIECBAgYAU4OgN6iAIcnami+QSviMkgAgQIECDQE7AFIi4MeogCHJemGjMJXg0sQwkQIECAgC0QoRnQQxTg0ECVTiZ4pVLGESBAgACBEQErwHFJ0EMU4Lg01ZhJ8GpgGUqAAAECBBTg0AzoIQpwaKBKJxO8UinjCBAgQICAFeDoDOghCnB0pormE7wiJoMIECBAgEBPwBaIuDDoIQpwXJpqzCR4NbAMJUCAAAECtkCEZkAPUYBDA1U6meCVShlHgAABAgRGBKwAxyVBD1GA49JUYybBq4FlKAECBAgQUIBDM6CHKMChgSqdTPBKpYwjQIAAAQJWgKMzoIcowNGZKppP8IqYDCJAgAABAj0BWyDiwqCHKMBxaaoxk+DVwDKUAAECBAjYAhGaAT1EAQ4NVOlkglcqZRwBAgQIEBgRsAIclwQ9RAGOS1ONmQSvBpahBAgQIEBAAQ7NgB6iAIcGqnQywSuVMo4AAQIECFgBjs6AHqIAR2eqaD7BK2IyiAABAgQI9ARsgYgLgx6iAMelqcZMglcDy1ACBAgQIGALRGgG9BAFODRQpZMJXqmUcQQIECBAYETACnBcEvQQBTguTTVmErwaWIYSIECAAAEFODQDeogCHBqo0skEr1TKOAIECBAgYAU4OgN6iAIcnami+QSviMkgAgQIECDQE7AFIi4MeogCHJemGjMJXg0sQwkQIECAgC0QoRnQQxTg0ECVTiZ4pVLGESBAgACBEQErwHFJ0EMU4Lg01ZhJ8GpgGUqAAAECBBTg0AzoIQpwaKBKJxO8UinjCBAgQICAFeDoDOghCnB0pormE7wiJoMIECBAgEBPwBaIuDDoIQpwXJpqzCR4NbAMJUCAAAECtkCEZkAPUYBDA1U6meCVShlHgAABAgRGBKwAxyVBD1GA49JUYybBq4FlKAECBAgQUIBDM6CHKMChgSqdTPBKpYwjQIAAAQJWgKMzoIcowNGZKppP8IqYDCJAgAABAj0BWyDiwqCHKMBxaaoxk+DVwDKUAAECBAjYAhGaAT1EAQ4NVOlkglcqZRwBAgQIEBgRsAIclwQ9RAGOS1ONmQSvBpahBAgQIEBAAQ7NgB6iAIcGqnQywSuVMo4AAQIECFgBjs6AHqIAR2eqaD7BK2IyiAABAgQI9ARsgYgLgx6iAMelqcZMglcDy1ACBAgQIGALRGgG9BAFODRQpZMJXqmUcQQIECBAYETACnBcEvQQBTguTTVmErwaWIYSIECAAAEFODQDeogCHBqo0skEr1TKOAIECBAgYAU4OgN6iAIcnami+QSviMkgAgQIECDQE7AFIi4MeogCHJemGjMJXg0sQwkQIECAgC0QoRnQQxTg0ECVTiZ4pVLGESBAgACBEQErwHFJ0EM6XICvu+66tHDhwvTEE0/0FM4+++x09913r6ey5557phyU/FizZk265ppr0h133JHefvvtdMQRR6R58+alffbZp1YqBa8Wl8EECBAgQEABDsyAHtLRAvy9730vXXrppenII49crwAffPDB6bjjjksXXXRRT2brrbdOu+66a/XfufzmwnvXXXelXIznzJmTli5dmp5//vm07bbbFkdT8IqpDCRAgAABAlaAgzOgh3SsAP/pT39K5557bnryySfT3nvvnXbbbbdeAf7www/TjjvumB544IF08sknbySzevXqNHny5HTTTTel8847r/rzd955J+2xxx7pzjvvTLNmzSqOp+AVUxlIgAABAgQU4OAM6CEdK8CPPfZYVXDz9odrr702LVu2rFeAX3zxxfTJT34yvfTSS2m//fbbSGbx4sXpkEMO2ejP8zaI6dOnVyvDpQ/BK5UyjgABAgQIjAjYAxyXBD2kYwX4o6f7la98Zb0C/OCDD6bTTjutWt392c9+liZNmpROOOGEqizvtNNOacGCBemUU05JK1euTDvssENvqpkzZ1b/7/HHHy9OpuAVUxlIgAABAgQU4OAM6CEKcG8F+Kqrrkrf+ta30vXXX18V39/97nfp8ssvr7ZK/OpXv0r3339/OuOMM1LeKpHL8brHmWeemV577bXqA3Wberz77rsp/7PusWLFijRjxoy0fPnytNdeewXH2nQECBAgQGD4BKwAx11TBVgB7hXgtWvXpr/85S9p55137qksWrQoHXbYYSn/O4fl1FNPHXUFeNWqVemRRx7ZpObVV19dfYBuw4cCHPdmNhMBAgQIDLeAAhx3fRVgBXi9u0BsyPHee+9VH4z78Y9/XK0E5z3Ar7zySpo2bVpvaN4DfNBBB6Vbb73VCnDce9NMBAgQIEBgPQEFOC4QCrAC3CvAp59+enrrrbfSL37xi55KvkfwUUcdlV544YX0iU98Ik2ZMiXdfPPN6ZxzzqnGuAtE3JvRTAQIECBAYHMCCnBcPhRgBbhXgPOH2E466aT0zW9+s7ql2csvv5y+9rWvVVsg7rvvvkrqyiuvTPPnz69uezZ16tQ0e/bs6oN0S5YscR/guPelmQgQIECAwEYCCnBcKBRgBXi9LRAPPfRQ9UG4fEu0vBc4rwrnu0Bsv/32lVT+ANzcuXOrX4Tx/vvvV79II299yGW4zkPCnMGDAAAgAElEQVTw6mgZS4AAAQIE3AYtMgN6SIcLcGSQ6s4leHXFjCdAgACBrgtYAY5LgB6iAMelqcZMglcDy1ACBAgQIOAXYYRmQA9RgEMDVTqZ4JVKGUeAAAECBEYErADHJUEPUYDj0lRjJsGrgWUoAQIECBBQgEMzoIcowKGBKp1M8EqljCNAgAABAlaAozOghyjA0Zkqmk/wipgMIkCAAAECPQFbIOLCoIcowHFpqjGT4NXAMpQAAQIECNgCEZoBPUQBDg1U6WSCVyplHAECBAgQGBGwAhyXBD1EAY5LU42ZBK8GlqEECBAgQEABDs2AHqIAhwaqdDLBK5UyjgABAgQIWAGOzoAeogBHZ6poPsErYjKIAAECBAj0BGyBiAuDHqIAx6WpxkyCVwPLUAIECBAgYAtEaAb0EAU4NFClkwleqZRxBAgQIEBgRMAKcFwS9BAFOC5NNWYSvBpYhhIgQIAAAQU4NAN6iAIcGqjSyQSvVMo4AgQIECBgBTg6A3qIAhydqaL5BK+IySACBAgQINATsAUiLgx6iAIcl6YaMwleDSxDCRAgQICALRChGdBDFODQQJVOJnilUsYRIECAAIERASvAcUnQQxTguDTVmEnwamAZSoAAAQIEFODQDOghCnBooEonE7xSKeMIECBAgIAV4OgM6CEKcHSmiuYTvCImgwgQIECAQE/AFoi4MOghCnBcmmrMJHg1sAwlQIAAAQK2QIRmQA9RgEMDVTqZ4JVKGUeAAAECBEYErADHJUEPUYDj0lRjJsGrgWUoAQIECBBQgEMzoIcowKGBKp1M8EqljCNAgAABAlaAozOghyjA0Zkqmk/wipgMIkCAAAECPQFbIOLCoIcowHFpqjGT4NXAMpQAAQIECNgCEZoBPUQBDg1U6WSCVyplHAECBAgQGBGwAhyXBD1EAY5LU42ZBK8GlqEECBAgQEABDs2AHqIAhwaqdDLBK5UyjgABAgQIWAGOzoAeogBHZ6poPsErYjKIAAECBAj0BGyBiAuDHqIAx6WpxkyCVwPLUAIECBAgYAtEaAb0EAU4NFClkwleqZRxBAgQIEBgRMAKcFwS9BAFOC5NNWYSvBpYhhIgQIAAAQU4NAN6iAIcGqjSyQSvVMo4AgQIECBgBTg6A3qIAhydqaL5BK+IySACBAgQINATsAUiLgx6iAIcl6YaMwleDSxDCRAgQICALRChGdBDFODQQJVOJnilUsYRIECAAIERASvAcUnQQxTguDTVmEnwamAZSoAAAQIEFODQDOghCnBooEonE7xSKeMIECBAgIAV4OgM6CEKcHSmiuYTvCImgwgQIECAQE/AFoi4MOghCnBcmmrMJHg1sAwlQIAAAQK2QIRmQA9RgEMDVTqZ4JVKGUeAAAECBEYErADHJUEPUYDj0lRjJsGrgWUoAQIECBBQgEMzoIcowKGBKp1M8EqljCNAgAABAlaAozOghyjA0Zkqmk/wipgMIkCAAAECPQFbIOLCoIcowHFpqjGT4NXAMpQAAQIECNgCEZoBPUQBDg1U6WSCVyplHAECBAgQGBGwAhyXBD1EAY5LU42ZBK8GlqEECBAgQEABDs2AHqIAhwaqdDLBK5UyjgABAgQIWAGOzoAeogBHZ6poPsErYjKIAAECBAj0BGyBiAuDHqIAx6WpxkyCVwPLUAIECBAgYAtEaAb0EAU4NFClkwleqZRxBAgQIEBgRMAKcFwS9BAFOC5NNWYSvBpYhhIgQIAAAQU4NAN6iAIcGqjSyQSvVMo4AgQIECBgBTg6A3qIAhydqaL5BK+IySACBAgQINATsAUiLgx6iAIcl6YaMwleDSxDCRAgQICALRChGdBDFODQQJVOJnilUsYRIECAAIERASvAcUnQQxTguDTVmEnwamAZSoAAAQIEFODQDOghHS7A1113XVq4cGF64oknegrPPfdcuuSSS9LTTz+ddtlll3TxxRenyy+/vPfna9asSddcc02644470ttvv52OOOKING/evLTPPvvUCqbg1eIymAABAgQIWAEOzIAe0tEC/L3vfS9deuml6cgjj+wV4Lfeeivtv//+6eSTT67+bNGiRemCCy6oCu7ZZ59dSeXym//7rrvuSnvuuWeaM2dOWrp0aXr++efTtttuWxxNwSumMpAAAQIECFQCtkDEBUEP6VgB/tOf/pTOPffc9OSTT6a999477bbbbr0C/O1vfzvdeuutadmyZWmbbbapZObOnZsWLFiQXnzxxbR69eo0efLkdNNNN6Xzzjuv+vN33nkn7bHHHunOO+9Ms2bNKk6m4BVTGUiAAAECBBTg4AzoIR0rwI899lh64IEHUt7+cO2111Zld90WiBNOOCHtvPPO6f777++p5C0SxxxzTHr99dersYccckh66aWX0n777dcbk7dBTJ8+vVoZLn0IXqmUcQQIECBAYETACnBcEvSQjhXgj57uV77ylfUKcC6xxx9/fLrxxht7w1544YV0wAEHpKeeeir98Y9/TKecckpauXJl2mGHHXpjZs6cWf2/xx9/vDiZgldMZSABAgQIEFCAgzOghyjAvRXg/EG2008/vVoZXvfI+3unTZtWbZnIK8BnnHFG+vDDD9OkSZN6Y84888z02muvVR+o29Tj3XffTfmfdY8VK1akGTNmpOXLl6e99torONamI0CAAAECwydgBTjumirACnCvAB944IEpb4MYbQX4mWeeqT7sduqpp466Arxq1ar0yCOPbFLz6quvrj5At+FDAY57M5uJAAECBIZbQAGOu74KsAK83h7gfOuze+65p6cy2h7gV155pVoVXvfIe4APOuig6gN0VoDj3pxmIkCAAAECHxVQgOPyoAArwL0CfMMNN6Tbb789vfrqq2nrrbeuZK644or08MMPV3eByKu8U6ZMSTfffHM655xzqj93F4i4N6OZCBAgQIDA5gQU4Lh8KMAKcK8Av/HGG9V9gE866aTq/r6LFy9O559/flWKzzrrrErqyiuvTPPnz69uezZ16tQ0e/bsam/wkiVL3Ac47n1pJgIECBAgsJGAAhwXCgVYAV7vN8Hluz3k3/727LPPpt133z1ddtll6cILL+wp5Q/A5XsD51+E8f7771e/SCNvfchluM5D8OpoGUuAAAECBNwGLTIDekiHC3BkkOrOJXh1xYwnQIAAga4LWAGOS4AeogDHpanGTIJXA8tQAgQIECDgF2GEZkAPUYBDA1U6meCVShlHgAABAgRGBKwAxyVBD1GA49JUYybBq4FlKAECBAgQUIBDM6CHKMChgSqdTPBKpYwjQIAAAQJWgKMzoIcowNGZKppP8IqYDCJAgAABAj0BWyDiwqCHKMBxaaoxk+DVwDKUAAECBAjYAhGaAT1EAQ4NVOlkglcqZRwBAgQIEBgRsAIclwQ9RAGOS1ONmQSvBpahBAgQIEBAAQ7NgB6iAIcGqnQywSuVMo4AAQITL2ClceLNS47oupQolY3RQxTgsqQEjxK8YFDTESBAIFBA0QrEDJzKdYnD1EMU4Lg01ZhJ8GpgGUqAAIEJFlC0Jhi88HCuSyFUwTA9RAEuiEn8EMGLNzUjAQIEogQUrSjJ2HlclzhPPUQBjktTjZkErwaWoQQIEJhgAUVrgsELD+e6FEIVDNNDFOCCmMQPEbx4UzMSIEAgSkDRipKMncd1ifPUQxTguDTVmEnwamAZSoAAgQkWULQmGLzwcK5LIVTBMD1EAS6ISfwQwYs3NSMBAgSiBBStKMnYeVyXOE89RAGOS1ONmQSvBpahBAgQmGABRWuCwQsP57oUQhUM00MU4IKYxA8RvHhTMxIgQCBKoK2i1fS4y274p6hTH+h5mvrkk+qKUekF1EMU4NKshI4TvFBOkxEgQCBUoK2i1fS4XSl3TX0U4I3fHnqIAhz6RbN0MsErlTKOAAECEy/QVtFqelwFeOyMdMVobImREXqIAlyaldBxghfKaTICBAiECjQtov2uNDY9blfKXVOffq9LaLgGZDI9RAFuJYqC1wq7gxIgQKBIoK2i1fS4CvDYl7UrRmNLWAEey2irtWvXrh1rkD9vJqAAN3PzLAIECEyEQNMi2u9KY9PjdqXcNfXp97pMROYm+hh6iBXgic5cdTzBa4XdQQkQIFAk0FbRanpcBXjsy9oVo7ElrACPZWQFeCyhPv5cAe4Dz1MJECAwzgJNi2i/K41Nj9uVctfUp9/rMs5xa2V6PcQKsOC1IuCgBAgQGFyBtopW0+MqwGNnqStGY0tYAR7LyArwWEJ9/Lm/efWB56kECBAYZ4GmRbTflcamx+1KuWvq0+91Gee4tTK9HmIFWPBaEXBQAgQIjL9AP4Wp6avrp4w2fb39HLPpebbxvKY+CvDGV0sBVoDbeA/7EFwr6g5KgEDXBPopTE2t+imjTV/vlnbMprZNfRRgBbhO5myBqKNVc6y/edUEM5wAAQINBPopTA0OVz1lSyujTY36Oc+mtk1fa7/XpenrHeTn6SFWgFvJp+C1wu6gBAh0TKCfwtSUqp9i2PT1bmnHbGrb1EcBtgJcJ3NWgOto1RyrANcEM5wAAQINBPopTA0O19oKcNPX2s/z+indTY/bz/Vs4/U2Pc+JeJ4eYgV4InK20TEErxV2ByVAoGMC/RSmplT9FK02Xm8b59n0mP349HNdmr7eQX6eHqIAt5JPwWuF3UEJEOiYQD+FqSlVP0Wrjdfbxnk2PWY/Pv1cl6avd5Cfp4cowK3kU/BaYXdQAgQ6JtBPYWpK1U/RauP1tnGeTY/Zj08/16Xp6x3k5+khCnAr+RS8VtgdlACBjgn0U5g6RlX7dNsolP1czzZeb23UCXyCHqIAT2Dc/u9QgtcKu4MSINAxgX4KU8eoap9uG4Wyn+vZxuutjTqBT9BDFOAJjJsC3Aq2gxIgECawpRWQfl5vGNqQTtRGoeznerbxegf50ivACnAr+RS8VtgdlACBPgW2tALSz+vtk2ron95Goeznerbxegc5BHqIAtxKPgWvFXYHJUCgT4EtrYD083r7pBr6p7dRKPu5nm283kEOgR6iALeST8Frhd1BCRDoU6CfAtLnoT19wATaKJT95K+N1ztgl2y9l6OHKMCt5FPwWmF3UAIE+hTop4D0eWhPHzCBNgplP/lr4/UO2CVTgAsviF+FXAjVZJgC3ETNcwgQaFugnwLS9mt3/FiBNgplP/lr4/XGisfOpodYAY5NVOFsglcIZRgBAgMl0E8BGagT8WL6FmijUPaTvzZeb9/I4ziBHqIAj2O8Nj214LXC7qAECKSU+ikRAAmsE2ijUPaT3TZe7yCnRQ9RgFvJp+C1wu6gBAgowDIQJNBGoVSAgy5eSkkPUYDj0lRjJsGrgWUoAQKhAv2UiNAXYrItWkAB3qIvnwK8mcvnQ3DjmG0FeBxxTU2AwGYFFGABiRBQgCMU25tDD7EC3Er6BK8VdgclQMAWCBkIElCAgyBbmkYPUYBbiZ7gtcLuoAQIKMAyECSgAAdBtjSNHqIAtxI9wWuF3UEJEFCAZSBIQAEOgmxpGj1EAW4leoLXCruDEiCgAMtAkIACHATZ0jR6iALcSvQErxV2ByVAQAGWgSCBfgpwGx/E7Of1BpEN1DR6iALcSiAFrxV2ByVAQAGWgSCBfgqlAhx0EfqYRg9RgPuIT/OnCl5zO88kQKA/gTbKR3+v2LMHUUABHsSrUv6a9BAFuDwtgSMFLxDTVAQI1BJQgGtxGbwJAQV4y46GHqIAt5JgwWuF3UEJELAFQgaCBBTgIMiWptFDFOBWoid4rbA7KAECCrAMBAkowEGQLU2jhyjAGwn84Q9/SFOnTt3o///whz9M5557bnruuefSJZdckp5++um0yy67pIsvvjhdfvnltSIseLW4DCZAIFDAFohATFNtMQL9FPYt5iRrvFA9RAHeSOCxxx5LM2fOTEuXLk1bbbVV78932mmntHLlyrT//vunk08+OV166aVp0aJF6YILLkjz5s1LZ599dnH0BK+YykACBIIFFOBgUNNtEQIK8PqXSQ9RgDcSuP7669OPfvSjaqV3w8e3v/3tdOutt6Zly5albbbZpvrjuXPnpgULFqQXX3yx+IuA4BVTGUiAwCYEFFnRIFAuoAArwKVp2Wrt2rVrSwcP07hZs2alSZMmpfvvv3+j0zrhhBPSzjvvvN6fLVy4MB1zzDHp9ddfT1OmTCmiUICLmAwiQGAzAgqweBAoF1CAFeDStHS2AB9wwAFp9913T6tWrUovv/xy2nfffdM3vvGN9MUvfjFNnz49HX/88enGG2/sOb7wwgspP+epp55Kn/vc50b1fffdd1P+Z91jxYoVacaMGWn58uVpr732Kr0mxhEgQKAnoAALA4FyAQVYAS5NSycL8OrVq9PHPvax9JnPfCZ95zvfSTvuuGO699570/e///30y1/+Mn31q19Np59+err22mt7jnmv8LRp09KTTz6ZjjjiiFF9r7766nTNNdds9GcKcGkcjSNAYEMBBVgmCJQLKMAKcGlaOlmAM857771X7e/dbrvtelbHHXdc9YG4vHUhb4MYbQX4mWeeqYrzaA8rwKWxM44AgVIBBbhUyjgCKSnACnDp+6CzBXg0oNmzZ6ef//znae+9965ufXbPPff0htkDXBop4wgQiBRQgCM1zTXsAgqwAlya8U4W4N/+9rfp8MMPr8ruR7czHHXUUWm33XZLn/70p9Ptt9+eXn311bT11ltXlldccUV6+OGH3QWiNFnGESAQIqAAhzCapCMCCrACXBr1ThbgNWvWVAU4b4O47bbb0uTJk9MPfvCD6j6/+UNuuQTn+wCfdNJJac6cOWnx4sXp/PPPr0rxWWedVWpbbaXIq8n2ABeTGUiAwAYCCrBIECgXUIAV4NK0dLIAZ5w333wzff3rX08//elP0zvvvJM++9nPphtuuCF94QtfqOxyEc6//e3ZZ5+t7hZx2WWXpQsvvLDUtRqnANfiMpgAgVEEFGCxIFAuoAArwKVp6WwBLgXqZ5wC3I+e5xIYHgEldniupTMZbAEFWAEuTagCXCrVYJwC3ADNUwgMoYACPIQX1SkNpIACrACXBlMBLpVqME4BboDmKQSGUEABHsKL6pQGUkABVoBLg6kAl0o1GKcAN0DzFAJDKKAAD+FFdUoDKaAAK8ClwVSAS6UajFOAG6B5CoEhFFCAh/CiOqWBFFCAFeDSYCrApVINxinADdA8hcAQCijAQ3hRndJACijACnBpMBXgUqkG4xTgBmieQmAIBRTgIbyoTmkgBRRgBbg0mApwqVSDcQpwAzRPITCEAgrwEF5UpzSQAgqwAlwaTAW4VKrBOAW4AZqnEBhCAQV4CC+qUxpIAQVYAS4NpgJcKtVgnALcAM1TCAyhgAI8hBfVKQ2kgAKsAJcGUwEulWowTgFugOYpBIZQQAEewovqlAZSQAFWgEuDqQCXSjUYpwA3QPMUAkMooAAP4UV1SgMpoAArwKXBVIBLpRqMU4AboHkKgSEUUICH8KI6pYEUUIAV4NJgKsClUg3GKcAN0DyFwBAKKMBDeFGd0kAKKMAKcGkwFeBSqQbjFOAGaJ5CYAgFFOAhvKhOaSAFFGAFuDSYCnCpVINxCnADNE8hMIQCCvAQXlSnNJACCrACXBpMBbhUqsE4BbgBmqcQGEIBBXgIL6pTGkgBBVgBLg2mAlwq1WCcAtwAzVMIDKGAAjyEF9UpDaSAAqwAlwZTAS6VajBOAW6A5ikEhlBAAR7Ci+qUBlJAAVaAS4OpAJdKNRinADdA8xQCQyigAA/hRXVKAymgACvApcFUgEulGoxTgBugeQqBIRRQgIfwojqlgRRQgBXg0mAqwKVSDcYpwA3QPIXAEAoowEN4UZ3SQAoowApwaTAV4FKpBuMU4AZonkJgCAUU4CG8qE5pIAUUYAW4NJgKcKlUg3EKcAM0TyEwhAIK8BBeVKc0kAIKsAJcGkwFuFSqwTgFuAGapxAYQgEFeAgvqlMaSAEFWAEuDaYCXCrVYJwC3ADNUwiMs0A/ZbTpN9d+jjnOHKYnMFQCTd+jQ4XwkZPRQzZ9ZRXgcUy94I0jrqkJNBTop4w2/ebazzEbnqanEeikQNP36LBi6SEKcCvZFrxW2B2UwGYF+imjTb+59nNMl5MAgXKBpu/R8iNsWSP1EAW4lcQKXivsDkpAAZYBAh0VUIDXv/B6iALcypcCwWuF3UEJKMAyQKCjAgqwAlwafXuAS6UajFOAG6B5CoFxFuhnO0LTb679HHOcOUxPYKgEmr5HhwrhIyejh1gBbiXbgtcKu4MSsAIsAwQ6KqAAWwEujb4V4FKpBuMU4AZonkJgnAWsxo4zsOkJtCigACvApfFTgEulGoxTgBugeQqBcRZQgMcZ2PQEWhRQgBXg0vgpwKVSDcYpwA3QPIXAOAsowOMMbHoCLQoowApwafwU4FKpBuMU4AZonkJgnAUU4HEGNj2BFgUUYAW4NH4KcKlUg3EKcAM0TyEwzgIK8DgDm55AiwIKsAJcGj8FuFSqwTgFuAGapxAYZwEFeJyBTU+gRQEFWAEujZ8CXCrVYJwC3ADNUwiMs4ACPM7ApifQooACrACXxk8BLpVqME4BboDmKQTGWUABHmdg0xNoUUABVoBL46cAl0o1GKcAN0DzFALjLKAAjzOw6Qm0KKAAK8Cl8VOAS6UajFOAG6B5CoFxFlCAxxnY9ARaFFCAFeDS+CnApVINxinADdA8pVMCTctoP9/kmh6zUxfGyRLYQgX6+dqwhZ7yZl+2HrJpHgV4HBMveOOIa+qhEFBGh+IyOgkCAyOgAFsBLg2jAlwq1WCcAtwAzVM6JaAAd+pyO1kC4y6gACvApSFTgEulGoxTgBugeUqnBBTgTl1uJ0tg3AUUYAW4NGQKcKlUg3EKcAM0T+mUgALcqcvtZAmMu4ACrACXhkwBLpVqME4BboDmKZ0SUIA7dbmdLIFxF1CAFeDSkCnApVINxinADdA8pVMCCnCnLreTJTDuAgqwAlwaMgW4VKrBOAW4AZqndEpAAe7U5XayBMZdQAFWgEtDpgCXSjUYpwA3QPOUTgkowJ263E6WwLgLKMAKcGnIFOBSqQbjFOAGaJ7Sl0A/hbLpN45+jtnXyXoyAQIENhBo+nVsWCH1kE1fWQV4HFMveOOIa+pRBfopo02/cfRzTJeRAAECkQJNv45FvoZBmksPUYBbyaPgtcLe6YMqo52+/E6eQOcFFOD1I6CHKMCtfFEQvFbYO31QBbjTl9/JE+i8gAKsAJe+CWyBKJVqME4BboDmKX0JKMB98XkyAQJbuIACrACXRlgBLpVqME4BboA2Tk/Z0oph0y/iW9p5jtPlNi0BAh0VaPq1c1i59BBbIFrJtuC1wj7qQbe0Ytj0i/iWdp6DkxCvhACBYRBo+rVzGM59tHPQQxTg2tles2ZNuuaaa9Idd9yR3n777XTEEUekefPmpX322ad4LsErphr3gYrhuBM7AAECBFoXUIDXvwR6iAJc+02Zy28uvHfddVfac88905w5c9LSpUvT888/n7bddtui+QSviGlCBinAE8LsIAQIEGhVQAFWgEsDaA/wKFKrV69OkydPTjfddFM677zzqhHvvPNO2mOPPdKdd96ZZs2aVeSrABcxTcggBXhCmB2EAAECrQoowApwaQAV4FGkFi9enA455JD00ksvpf322683Im+DmD59erUyXPJQgEuUJmaMAjwxzo5CgACBNgUUYAW4NH8K8ChSCxYsSKecckpauXJl2mGHHXojZs6cWf2/xx9/vMhXAS5impBBCvCEMDsIAQIEWhVQgBXg0gAqwKNI3XvvvemMM85IH374YZo0aVJvxJlnnplee+21tHDhwlF933333ZT/WfdYvnx5Ovzww1NeUd59991Lr4lx4yBw6PX/MQ6zmpIAAQIEBklg0dyjB+nltP5aVqxYkWbMmJF+//vfp6lTp7b+egbpBSjAo1yNhx56KJ166qmjrgCvWrUqPfLII6New6uvvrq6c4QHAQIECBAgQGBQBPJC3Oc///lBeTkD8ToU4FEuw7o9wK+88kqaNm1ab0TeA3zQQQelW2+9ddSLt+EK8AcffJDyKvDHP/7xtM0224zLBV/3tzurzJvmZTR29BgxGltg7BFyxGhsgc2PkKGxBesY/fWvf01vvvlmOvDAA9P2228/9uQdGqEAj3Kx8yrvlClT0s0335zOOeecakSTu0BMRI7sMx5bmRGjsQXGHiFHjMYWGHuEHG3eiI8MjS0QM0IB3oTjlVdemebPn1/d9izvm5k9e3ZatmxZWrJkSfF9gGMukS8W/Tr6gjq2ICNGYwuMPUKOGI0t4Hsao34FYp6vAG/CMX8Abu7cudUvwnj//ffTkUceWW19GLRN5L7hjP1GYMRobIGxR8gRo7EFxh4hRwrw2Clh1K9RyfMV4BKlAR6T9x3nrRqXXnpp+tu//dsBfqXtvTRGY9szYjS2wNgj5IjR2AKbHyFDYwsyGtuoZIQCXKJkDAECBAgQIECAwNAIKMBDcymdCAECBAgQIECAQImAAlyiZAwBAgQIECBAgMDQCCjAQ3MpnQgBAgQIECBAgECJgAJcomQMAQIECBAgQIDA0AgowFvopVyzZk31a5fvuOOO9Pbbb6f8W+rmzZuX9tlnny30jPp/2X/4wx9GvU3dD3/4w3Tuueem5557Ll1yySXp6aefTrvssku6+OKL0+WXX97/gbeQGa677rq0cOHC9MQTT/Re8VgmXcvZaEZnn312uvvuu9e7ynvuuWfKt7PKj2E3+vOf/1zdEvLxxx9P+dPn06dPTzfccEP1NSc/ZCilsYy6nqGckzfeeCNddtll6ec//3l1a9F/+Id/SN/5znfS3//938vR/351GctIjmK/2SrAsZ4TNlsuv7nw5vsU52/Gc+bMSUuXLk3PP//8QP2ijgkDSSk99thjaebMmZXDVjkpZowAAAm4SURBVFtt1Tv0TjvtlFauXJn233//dPLJJ1e3jFu0aFG64IILKsP8RWXYH9/73veq8873s15XgN96660xTbqUs9GMci4OPvjgdNxxx6WLLrqoF5Ott9467brrrtV/D7vRscceW5WXW265pTrnfD/0/BfvZ555pvrvsd5Xw+6TM7A5o+zT9Qxlo0MPPTRNmjQp/du//Vv62Mc+lr7xjW+k3/zmN+mVV16pCrEcbd7ob/7mb+Qo+Bu1AhwMOhHTrV69Ok2ePDnddNNN6bzzzqsOOai/qnkiPNYd4/rrr08/+tGPqhWpDR/f/va3q2/c+bf5bbPNNtUf51WtBQsWpBdffHEiX+aEHutPf/pTtfr95JNPpr333jvttttuvQI8lklXcrY5o/wLcXbcccf0wAMPVH952vAx7Ea5nOy7777p17/+dTr88MOr01+7dm3ab7/90pe//OW0ww47bPZ9New+2WMso6uuuqrTGcpG+S/bF154YfrXf/3XdMABB1Q5+q//+q/06U9/Oi1evLj6ydTmvj53IUdjGX32s5/tfI6iv3kqwNGiEzBf/oJxyCGHpJdeeqn6RrTukX8kmX88mVc1u/iYNWtWtcJw//33b3T6J5xwQtp5553X+7P8RfeYY45Jr7/+epoyZcpQkuVV8Vze8o/2r7322uovAOtWgMcyyWO7kLPNGeW/HH3yk5/c6L22LizD/l7Mf7HOq3RHH3102m677XrvkbzVKq+K55+2bO591YUMjWWUi1+XMzTaF9b8NTdvP8tfi/J77J//+Z87n6MNnTY0Wr58uRwFf5dWgINBJ2K6vGp5yimnVD/Wzysw6x75x//5/+W9el185JWF3XffPa1atSq9/PLL1cpV/jHbF7/4xeovBscff3y68cYbezQvvPBCtRrx1FNPpc997nNDT/aVr3xlvQI8lskf//jHzuVsQ6MHH3wwnXbaadVPWn72s59Vf8HKf3HIf6HIW2u6+F7MJvlrzaOPPpquvPLKzb6vupih/IXko0YffPCBDH3kq+u//Mu/pPy5jPwXqpyhvH3E16L1v/2MZuRrUfy3aAU43nTcZ7z33nvTGWeckfKPZ/M35HWPM888M7322mvVj5O69sg/Isv7yj7zmc9UH6zIP7bOTt///vfTL3/5y/TVr341nX766dUq6LpHXr2aNm1atT1g3Qd6htltw3KXV/E2Z5JX77qWsw2N8o+vv/Wtb6W8vSYX39/97nfVylXeTvKrX/2q+olCl4zyVoi88ptXhH/yk59UH7qVofW/amxoJEPr++SFh7zn97bbbqveP//5n/9Z/YVKjv7PaTSjRx55xNei4G/QCnAw6ERM99BDD6VTTz111BXgvPqZ3yhdfLz33nvV/t6P/qg2f7POH4jLn9jPBWa0FeD8YZ5cnIf9sWG5O/DAAzdrkv+C0LWcbWiU97v+5S9/qX48u+6RP0B52GGHVR+kzLnqilH+upJLSv4wU/4pU/7pkwyt/1VjNCMZGv0ra757Ss5P3maVfwq3ua/PXfxalNU+avTv//7vvhYFf5NWgINBJ2K6dfsO84cv8grmukdexTzooIOqDxN4jAjMnj27uu1OXrHLtz675557ejRd2AP80RxsWO7yN5zNmazbv9mlnG1oNNr7KP9FK/+E4cc//nGVq/wNfNiN8h0g8i0Ev/SlL1U/WVn3l0wZ+r+EbMpIhkZugfYf//Ef1UpvvoPKukfe+5tvrZf/X9e/Fo1l9Itf/GKjKHXxa1Fkt1GAIzUnaK68yps/tHXzzTenc845pzpq1+8C8dvf/rb6lHouux/dznDUUUdVdz7Inza+/fbb06uvvtr7AnzFFVekhx9+eKjvArG5Apzv5bo5ky7mbMMCnFc886ezP/rNJ39wJ+cq/5jyE5/4xNC/F/OPqvMtA/N9s7/73e+ut+1KhkbeYZszkqGRe0Xnn7LlEvyP//iPldn//M//VLc+O+mkk6qv0V3/WjSWUf5QXNe/FkVXLAU4WnSC5ssfPpk/f3668847q1/+kFc684rdkiVLOnkf4PyjolyA89+I8zejfJu4H/zgB9UdMfKP1/IX2HVfbPM9k/Mq+vnnn1990T3rrLMm6Kq1e5gNy11ecRjLpGs529Ao/6g/f4P+5je/mfJdRvKHK7/2ta9VWyDuu+++6oIOs1E+30996lPpxBNP3OjuMnkLRP5LUtczNJZR/oxBlzOU3yN5G0j+EHL+HpU/APd3f/d31X7W/BfLXPy23377zudoLKO8yNP1HEV/B1WAo0UnaL78Abh8H9v8izDyBwryLzjIWx9yGe7q480330xf//rX009/+tNqRTzfNzGvUH3hC1+oSHIRzqtYzz77bHW3iPxbifItirryGO3H+2OZdC1noxnlPff5m3W+XVPeC5xX9PJdIPI37fwYZqP84b9c8Ed75L845t+Q1/UMlRh1OUPrspP30uefuuUPT+avz/nrcv4p5rr7Anc9R9lpLCM5iv1urQDHepqNAAECBAgQIEBgwAUU4AG/QF4eAQIECBAgQIBArIACHOtpNgIECBAgQIAAgQEXUIAH/AJ5eQQIECBAgAABArECCnCsp9kIECBAgAABAgQGXEABHvAL5OURIECAAAECBAjECijAsZ5mI0CAAAECBAgQGHABBXjAL5CXR4AAAQIECBAgECugAMd6mo0AAQIECBAgQGDABRTgAb9AXh4BAgQIECBAgECsgAIc62k2AgQIECBAgACBARdQgAf8Anl5BAgQIECAAAECsQIKcKyn2QgQIECAAAECBAZcQAEe8Avk5REgQIAAAQIECMQKKMCxnmYjQIAAAQIECBAYcAEFeMAvkJdHgAABAgQIECAQK6AAx3qajQABAgQIECBAYMAFFOABv0BeHgECBAgQIECAQKyAAhzraTYCBAgQIECAAIEBF1CAB/wCeXkECBAgQIAAAQKxAgpwrKfZCBAgQIAAAQIEBlxAAR7wC+TlESBAgAABAgQIxAoowLGeZiNAgAABAgQIEBhwAQV4wC+Ql0eAAAECBAgQIBAroADHepqNAAECBAgQIEBgwAUU4AG/QF4eAQIECBAgQIBArIACHOtpNgIECBAgQIAAgQEXUIAH/AJ5eQQIECBAgAABArECCnCsp9kIECBAgAABAgQGXEABHvAL5OURIECAAAECBAjECijAsZ5mI0CAAAECBAgQGHABBXjAL5CXR4AAAQIECBAgECugAMd6mo0AAQIECBAgQGDABRTgAb9AXh4BAgQIECBAgECsgAIc62k2AgQIECBAgACBARdQgAf8Anl5BAgQIECAAAECsQL/HxscvKFZEwkZAAAAAElFTkSuQmCC\" width=\"639.9999861283738\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "plt.figure()\n",
    "plt.hist(list(reg_size_dic.values()), bins=np.arange(0,350,10))\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2db97b4d",
   "metadata": {},
   "source": [
    "### 4.1.3 check readout usage for each gene"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 116,
   "id": "8d0a6c3d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0 -> ['NDB_683', 'NDB_758', 'NDB_887']\n",
      "1 -> ['NDB_718', 'NDB_631', 'NDB_1066']\n",
      "2 -> ['NDB_847', 'NDB_832', 'NDB_779']\n",
      "3 -> ['NDB_755', 'NDB_899', 'NDB_889']\n",
      "4 -> ['NDB_710', 'NDB_665', 'NDB_1028']\n",
      "5 -> ['NDB_643', 'NDB_805', 'NDB_730']\n",
      "6 -> ['NDB_883', 'NDB_1060', 'NDB_853']\n",
      "7 -> ['NDB_974', 'NDB_754', 'NDB_634']\n",
      "8 -> ['NDB_661', 'NDB_901', 'NDB_835']\n",
      "9 -> ['NDB_878', 'NDB_836', 'NDB_742']\n",
      "10 -> ['NDB_844', 'NDB_812', 'NDB_637']\n",
      "11 -> ['NDB_737', 'NDB_935', 'NDB_925']\n",
      "12 -> ['NDB_790', 'NDB_788', 'NDB_934']\n",
      "13 -> ['NDB_746', 'NDB_758', 'NDB_818']\n",
      "14 -> ['NDB_961', 'NDB_631', 'NDB_866']\n",
      "15 -> ['NDB_652', 'NDB_958', 'NDB_926']\n",
      "16 -> ['NDB_815', 'NDB_887', 'NDB_989']\n",
      "17 -> ['NDB_643', 'NDB_1028', 'NDB_910']\n",
      "18 -> ['NDB_760', 'NDB_734', 'NDB_712']\n",
      "19 -> ['NDB_965', 'NDB_661', 'NDB_899']\n",
      "20 -> ['NDB_653', 'NDB_977', 'NDB_940']\n",
      "21 -> ['NDB_743', 'NDB_917', 'NDB_715']\n",
      "22 -> ['NDB_698', 'NDB_883', 'NDB_1075']\n",
      "23 -> ['NDB_728', 'NDB_946', 'NDB_838']\n",
      "24 -> ['NDB_754', 'NDB_860', 'NDB_752']\n",
      "25 -> ['NDB_905', 'NDB_832', 'NDB_665']\n",
      "26 -> ['NDB_853', 'NDB_941', 'NDB_881']\n",
      "27 -> ['NDB_713', 'NDB_1066', 'NDB_929']\n",
      "28 -> ['NDB_784', 'NDB_875', 'NDB_751']\n",
      "29 -> ['NDB_647', 'NDB_779', 'NDB_925']\n",
      "30 -> ['NDB_725', 'NDB_661', 'NDB_994']\n",
      "31 -> ['NDB_710', 'NDB_746', 'NDB_812']\n",
      "32 -> ['NDB_778', 'NDB_935', 'NDB_694']\n",
      "33 -> ['NDB_917', 'NDB_824', 'NDB_910']\n",
      "34 -> ['NDB_718', 'NDB_974', 'NDB_1028']\n",
      "35 -> ['NDB_773', 'NDB_1033', 'NDB_880']\n",
      "36 -> ['NDB_826', 'NDB_865', 'NDB_1027']\n",
      "37 -> ['NDB_817', 'NDB_971', 'NDB_721']\n",
      "38 -> ['NDB_790', 'NDB_698', 'NDB_896']\n",
      "39 -> ['NDB_847', 'NDB_683', 'NDB_901']\n",
      "40 -> ['NDB_728', 'NDB_863', 'NDB_752']\n",
      "42 -> ['NDB_844', 'NDB_856', 'NDB_1075']\n",
      "43 -> ['NDB_961', 'NDB_889', 'NDB_977']\n",
      "44 -> ['NDB_713', 'NDB_868', 'NDB_712']\n",
      "45 -> ['NDB_935', 'NDB_742', 'NDB_934']\n",
      "46 -> ['NDB_634', 'NDB_730', 'NDB_926']\n",
      "47 -> ['NDB_784', 'NDB_805', 'NDB_734']\n",
      "57 -> ['NDB_773', 'NDB_706', 'NDB_940']\n",
      "60 -> ['NDB_971', 'NDB_716', 'NDB_680']\n",
      "62 -> ['NDB_746', 'NDB_905', 'NDB_887']\n",
      "63 -> ['NDB_965', 'NDB_631', 'NDB_989']\n",
      "64 -> ['NDB_863', 'NDB_641', 'NDB_862']\n",
      "65 -> ['NDB_826', 'NDB_1033', 'NDB_878']\n",
      "66 -> ['NDB_817', 'NDB_653', 'NDB_872']\n",
      "67 -> ['NDB_847', 'NDB_715', 'NDB_836']\n",
      "68 -> ['NDB_680', 'NDB_866', 'NDB_706']\n",
      "69 -> ['NDB_790', 'NDB_853', 'NDB_694']\n",
      "70 -> ['NDB_725', 'NDB_875', 'NDB_716']\n",
      "71 -> ['NDB_865', 'NDB_637', 'NDB_742']\n",
      "72 -> ['NDB_856', 'NDB_779', 'NDB_958']\n",
      "73 -> ['NDB_778', 'NDB_868', 'NDB_1027']\n",
      "74 -> ['NDB_743', 'NDB_788', 'NDB_910']\n",
      "75 -> ['NDB_755', 'NDB_760', 'NDB_929']\n",
      "76 -> ['NDB_1060', 'NDB_896', 'NDB_712']\n",
      "77 -> ['NDB_961', 'NDB_860', 'NDB_881']\n",
      "78 -> ['NDB_784', 'NDB_862', 'NDB_872']\n",
      "79 -> ['NDB_974', 'NDB_994', 'NDB_751']\n",
      "80 -> ['NDB_718', 'NDB_737', 'NDB_721']\n",
      "81 -> ['NDB_812', 'NDB_880', 'NDB_977']\n",
      "83 -> ['NDB_835', 'NDB_946', 'NDB_836']\n",
      "84 -> ['NDB_698', 'NDB_805', 'NDB_838']\n",
      "85 -> ['NDB_815', 'NDB_883', 'NDB_694']\n",
      "86 -> ['NDB_641', 'NDB_1066', 'NDB_889']\n",
      "87 -> ['NDB_758', 'NDB_899', 'NDB_941']\n",
      "88 -> ['NDB_710', 'NDB_824', 'NDB_926']\n",
      "89 -> ['NDB_683', 'NDB_1060', 'NDB_818']\n",
      "90 -> ['NDB_652', 'NDB_856', 'NDB_896']\n",
      "91 -> ['NDB_901', 'NDB_647', 'NDB_958']\n",
      "92 -> ['NDB_1075', 'NDB_905', 'NDB_934']\n",
      "93 -> ['NDB_965', 'NDB_715', 'NDB_832']\n",
      "94 -> ['NDB_643', 'NDB_634', 'NDB_989']\n",
      "95 -> ['NDB_730', 'NDB_665', 'NDB_734']\n",
      "96 -> ['NDB_778', 'NDB_754', 'NDB_737']\n",
      "98 -> ['NDB_755', 'NDB_844', 'NDB_946']\n",
      "100 -> ['NDB_901', 'NDB_863', 'NDB_926']\n",
      "101 -> ['NDB_725', 'NDB_847', 'NDB_835']\n",
      "102 -> ['NDB_815', 'NDB_866', 'NDB_941']\n",
      "103 -> ['NDB_713', 'NDB_752', 'NDB_862']\n",
      "104 -> ['NDB_865', 'NDB_875', 'NDB_929']\n",
      "105 -> ['NDB_680', 'NDB_758', 'NDB_647']\n",
      "106 -> ['NDB_755', 'NDB_838', 'NDB_836']\n",
      "107 -> ['NDB_728', 'NDB_716', 'NDB_824']\n",
      "108 -> ['NDB_971', 'NDB_994', 'NDB_940']\n",
      "109 -> ['NDB_878', 'NDB_880', 'NDB_899']\n",
      "110 -> ['NDB_683', 'NDB_706', 'NDB_751']\n",
      "111 -> ['NDB_844', 'NDB_746', 'NDB_889']\n",
      "112 -> ['NDB_743', 'NDB_661', 'NDB_653']\n",
      "114 -> ['NDB_631', 'NDB_946', 'NDB_958']\n",
      "115 -> ['NDB_817', 'NDB_961', 'NDB_868']\n",
      "116 -> ['NDB_710', 'NDB_773', 'NDB_1027']\n",
      "117 -> ['NDB_760', 'NDB_925', 'NDB_872']\n",
      "118 -> ['NDB_754', 'NDB_1033', 'NDB_818']\n",
      "119 -> ['NDB_917', 'NDB_977', 'NDB_836']\n",
      "120 -> ['NDB_652', 'NDB_790', 'NDB_832']\n",
      "121 -> ['NDB_860', 'NDB_737', 'NDB_1066']\n",
      "122 -> ['NDB_878', 'NDB_752', 'NDB_779']\n",
      "123 -> ['NDB_698', 'NDB_881', 'NDB_926']\n",
      "124 -> ['NDB_965', 'NDB_863', 'NDB_934']\n",
      "125 -> ['NDB_778', 'NDB_901', 'NDB_856']\n",
      "126 -> ['NDB_974', 'NDB_812', 'NDB_887']\n",
      "127 -> ['NDB_866', 'NDB_665', 'NDB_694']\n",
      "128 -> ['NDB_946', 'NDB_721', 'NDB_788']\n",
      "129 -> ['NDB_637', 'NDB_751', 'NDB_872']\n",
      "130 -> ['NDB_835', 'NDB_989', 'NDB_1028']\n",
      "131 -> ['NDB_784', 'NDB_961', 'NDB_746']\n",
      "132 -> ['NDB_725', 'NDB_718', 'NDB_730']\n",
      "133 -> ['NDB_865', 'NDB_715', 'NDB_994']\n",
      "134 -> ['NDB_1075', 'NDB_853', 'NDB_836']\n",
      "137 -> ['NDB_760', 'NDB_641', 'NDB_706']\n",
      "138 -> ['NDB_889', 'NDB_742', 'NDB_940']\n",
      "139 -> ['NDB_815', 'NDB_856', 'NDB_880']\n",
      "140 -> ['NDB_683', 'NDB_779', 'NDB_824']\n",
      "141 -> ['NDB_652', 'NDB_860', 'NDB_838']\n",
      "143 -> ['NDB_734', 'NDB_935', 'NDB_977']\n",
      "144 -> ['NDB_826', 'NDB_716', 'NDB_881']\n",
      "145 -> ['NDB_743', 'NDB_965', 'NDB_773']\n",
      "146 -> ['NDB_941', 'NDB_712', 'NDB_910']\n",
      "147 -> ['NDB_710', 'NDB_758', 'NDB_1066']\n",
      "148 -> ['NDB_718', 'NDB_1060', 'NDB_637']\n",
      "149 -> ['NDB_971', 'NDB_698', 'NDB_788']\n",
      "150 -> ['NDB_653', 'NDB_832', 'NDB_899']\n",
      "152 -> ['NDB_784', 'NDB_905', 'NDB_896']\n",
      "153 -> ['NDB_1033', 'NDB_647', 'NDB_862']\n",
      "154 -> ['NDB_844', 'NDB_661', 'NDB_715']\n",
      "155 -> ['NDB_643', 'NDB_790', 'NDB_875']\n",
      "156 -> ['NDB_1027', 'NDB_742', 'NDB_958']\n",
      "157 -> ['NDB_713', 'NDB_824', 'NDB_818']\n",
      "158 -> ['NDB_728', 'NDB_868', 'NDB_994']\n",
      "159 -> ['NDB_847', 'NDB_754', 'NDB_883']\n",
      "160 -> ['NDB_812', 'NDB_680', 'NDB_889']\n",
      "161 -> ['NDB_815', 'NDB_974', 'NDB_1060']\n",
      "162 -> ['NDB_917', 'NDB_961', 'NDB_751']\n",
      "163 -> ['NDB_634', 'NDB_883', 'NDB_910']\n",
      "164 -> ['NDB_805', 'NDB_887', 'NDB_665']\n",
      "165 -> ['NDB_901', 'NDB_773', 'NDB_641']\n",
      "166 -> ['NDB_826', 'NDB_863', 'NDB_896']\n",
      "167 -> ['NDB_817', 'NDB_941', 'NDB_929']\n",
      "168 -> ['NDB_965', 'NDB_721', 'NDB_925']\n",
      "169 -> ['NDB_643', 'NDB_1075', 'NDB_838']\n",
      "170 -> ['NDB_661', 'NDB_737', 'NDB_712']\n",
      "171 -> ['NDB_713', 'NDB_742', 'NDB_1028']\n",
      "172 -> ['NDB_865', 'NDB_730', 'NDB_779']\n",
      "173 -> ['NDB_989', 'NDB_637', 'NDB_788']\n",
      "174 -> ['NDB_826', 'NDB_752', 'NDB_694']\n",
      "175 -> ['NDB_634', 'NDB_866', 'NDB_1066']\n",
      "176 -> ['NDB_755', 'NDB_812', 'NDB_805']\n",
      "177 -> ['NDB_778', 'NDB_728', 'NDB_631']\n",
      "178 -> ['NDB_905', 'NDB_899', 'NDB_706']\n",
      "179 -> ['NDB_725', 'NDB_853', 'NDB_958']\n",
      "180 -> ['NDB_880', 'NDB_934', 'NDB_926']\n",
      "181 -> ['NDB_971', 'NDB_758', 'NDB_734']\n",
      "182 -> ['NDB_844', 'NDB_881', 'NDB_935']\n",
      "183 -> ['NDB_680', 'NDB_832', 'NDB_641']\n",
      "184 -> ['NDB_715', 'NDB_1033', 'NDB_835']\n",
      "185 -> ['NDB_652', 'NDB_862', 'NDB_929']\n",
      "186 -> ['NDB_917', 'NDB_721', 'NDB_818']\n",
      "187 -> ['NDB_847', 'NDB_946', 'NDB_872']\n",
      "188 -> ['NDB_817', 'NDB_878', 'NDB_977']\n",
      "189 -> ['NDB_710', 'NDB_716', 'NDB_860']\n",
      "190 -> ['NDB_743', 'NDB_856', 'NDB_940']\n",
      "191 -> ['NDB_784', 'NDB_868', 'NDB_647']\n",
      "192 -> ['NDB_746', 'NDB_653', 'NDB_925']\n",
      "193 -> ['NDB_760', 'NDB_875', 'NDB_1027']\n",
      "194 -> ['NDB_755', 'NDB_683', 'NDB_631']\n",
      "195 -> ['NDB_743', 'NDB_935', 'NDB_818']\n",
      "196 -> ['NDB_652', 'NDB_1033', 'NDB_899']\n",
      "197 -> ['NDB_1060', 'NDB_929', 'NDB_1028']\n",
      "198 -> ['NDB_728', 'NDB_860', 'NDB_698']\n",
      "199 -> ['NDB_805', 'NDB_835', 'NDB_647']\n",
      "200 -> ['NDB_826', 'NDB_730', 'NDB_887']\n",
      "201 -> ['NDB_758', 'NDB_946', 'NDB_824']\n",
      "202 -> ['NDB_713', 'NDB_746', 'NDB_715']\n",
      "203 -> ['NDB_901', 'NDB_737', 'NDB_788']\n",
      "204 -> ['NDB_778', 'NDB_716', 'NDB_896']\n",
      "205 -> ['NDB_680', 'NDB_994', 'NDB_934']\n",
      "206 -> ['NDB_812', 'NDB_856', 'NDB_751']\n",
      "207 -> ['NDB_905', 'NDB_721', 'NDB_734']\n",
      "208 -> ['NDB_875', 'NDB_754', 'NDB_641']\n",
      "209 -> ['NDB_718', 'NDB_1033', 'NDB_665']\n",
      "210 -> ['NDB_971', 'NDB_637', 'NDB_712']\n",
      "211 -> ['NDB_1027', 'NDB_818', 'NDB_910']\n",
      "212 -> ['NDB_917', 'NDB_728', 'NDB_940']\n",
      "213 -> ['NDB_965', 'NDB_1066', 'NDB_977']\n",
      "214 -> ['NDB_755', 'NDB_961', 'NDB_661']\n",
      "215 -> ['NDB_631', 'NDB_706', 'NDB_926']\n",
      "216 -> ['NDB_760', 'NDB_889', 'NDB_824']\n",
      "217 -> ['NDB_868', 'NDB_887', 'NDB_935']\n",
      "218 -> ['NDB_773', 'NDB_878', 'NDB_653']\n",
      "219 -> ['NDB_634', 'NDB_752', 'NDB_838']\n",
      "220 -> ['NDB_817', 'NDB_974', 'NDB_883']\n",
      "221 -> ['NDB_1075', 'NDB_941', 'NDB_694']\n",
      "222 -> ['NDB_866', 'NDB_989', 'NDB_836']\n",
      "223 -> ['NDB_725', 'NDB_721', 'NDB_665']\n",
      "224 -> ['NDB_743', 'NDB_815', 'NDB_875']\n",
      "225 -> ['NDB_863', 'NDB_1060', 'NDB_940']\n",
      "226 -> ['NDB_790', 'NDB_899', 'NDB_779']\n",
      "227 -> ['NDB_718', 'NDB_880', 'NDB_647']\n",
      "228 -> ['NDB_844', 'NDB_1027', 'NDB_1066']\n",
      "229 -> ['NDB_826', 'NDB_734', 'NDB_862']\n",
      "230 -> ['NDB_754', 'NDB_853', 'NDB_872']\n",
      "233 -> ['NDB_713', 'NDB_971', 'NDB_917']\n",
      "234 -> ['NDB_643', 'NDB_925', 'NDB_712']\n",
      "235 -> ['NDB_847', 'NDB_929', 'NDB_824']\n",
      "236 -> ['NDB_631', 'NDB_832', 'NDB_889']\n",
      "237 -> ['NDB_710', 'NDB_683', 'NDB_977']\n",
      "238 -> ['NDB_868', 'NDB_860', 'NDB_751']\n",
      "239 -> ['NDB_835', 'NDB_721', 'NDB_881']\n",
      "240 -> ['NDB_784', 'NDB_742', 'NDB_926']\n",
      "241 -> ['NDB_961', 'NDB_901', 'NDB_715']\n",
      "242 -> ['NDB_760', 'NDB_728', 'NDB_935']\n",
      "243 -> ['NDB_652', 'NDB_778', 'NDB_880']\n",
      "244 -> ['NDB_698', 'NDB_863', 'NDB_647']\n",
      "245 -> ['NDB_746', 'NDB_805', 'NDB_958']\n",
      "246 -> ['NDB_865', 'NDB_725', 'NDB_815']\n",
      "247 -> ['NDB_680', 'NDB_883', 'NDB_1028']\n",
      "248 -> ['NDB_878', 'NDB_941', 'NDB_788']\n",
      "249 -> ['NDB_856', 'NDB_832', 'NDB_836']\n",
      "250 -> ['NDB_875', 'NDB_730', 'NDB_872']\n",
      "251 -> ['NDB_634', 'NDB_1033', 'NDB_994']\n",
      "252 -> ['NDB_974', 'NDB_737', 'NDB_1075']\n",
      "253 -> ['NDB_896', 'NDB_706', 'NDB_934']\n",
      "254 -> ['NDB_653', 'NDB_835', 'NDB_853']\n",
      "255 -> ['NDB_817', 'NDB_881', 'NDB_862']\n",
      "256 -> ['NDB_773', 'NDB_838', 'NDB_925']\n",
      "257 -> ['NDB_713', 'NDB_790', 'NDB_716']\n",
      "258 -> ['NDB_778', 'NDB_866', 'NDB_752']\n",
      "259 -> ['NDB_661', 'NDB_698', 'NDB_989']\n",
      "260 -> ['NDB_755', 'NDB_758', 'NDB_940']\n",
      "261 -> ['NDB_718', 'NDB_905', 'NDB_946']\n",
      "262 -> ['NDB_865', 'NDB_641', 'NDB_818']\n",
      "263 -> ['NDB_847', 'NDB_863', 'NDB_910']\n",
      "264 -> ['NDB_710', 'NDB_844', 'NDB_868']\n",
      "265 -> ['NDB_784', 'NDB_826', 'NDB_812']\n",
      "266 -> ['NDB_760', 'NDB_779', 'NDB_742']\n",
      "267 -> ['NDB_971', 'NDB_683', 'NDB_958']\n",
      "268 -> ['NDB_652', 'NDB_694', 'NDB_872']\n",
      "269 -> ['NDB_743', 'NDB_643', 'NDB_665']\n",
      "270 -> ['NDB_965', 'NDB_917', 'NDB_881']\n",
      "271 -> ['NDB_730', 'NDB_880', 'NDB_637']\n",
      "272 -> ['NDB_1060', 'NDB_751', 'NDB_925']\n",
      "273 -> ['NDB_737', 'NDB_929', 'NDB_934']\n",
      "274 -> ['NDB_725', 'NDB_817', 'NDB_683']\n",
      "275 -> ['NDB_1075', 'NDB_788', 'NDB_862']\n",
      "276 -> ['NDB_896', 'NDB_989', 'NDB_734']\n",
      "277 -> ['NDB_653', 'NDB_887', 'NDB_694']\n",
      "278 -> ['NDB_832', 'NDB_1066', 'NDB_925']\n",
      "279 -> ['NDB_826', 'NDB_718', 'NDB_965']\n",
      "280 -> ['NDB_760', 'NDB_905', 'NDB_647']\n",
      "281 -> ['NDB_847', 'NDB_643', 'NDB_812']\n",
      "282 -> ['NDB_634', 'NDB_779', 'NDB_977']\n",
      "283 -> ['NDB_863', 'NDB_805', 'NDB_751']\n",
      "284 -> ['NDB_728', 'NDB_661', 'NDB_641']\n",
      "285 -> ['NDB_974', 'NDB_875', 'NDB_836']\n",
      "286 -> ['NDB_710', 'NDB_737', 'NDB_946']\n",
      "287 -> ['NDB_742', 'NDB_929', 'NDB_818']\n",
      "288 -> ['NDB_652', 'NDB_917', 'NDB_631']\n",
      "289 -> ['NDB_883', 'NDB_896', 'NDB_838']\n",
      "290 -> ['NDB_653', 'NDB_637', 'NDB_862']\n",
      "291 -> ['NDB_725', 'NDB_887', 'NDB_941']\n",
      "292 -> ['NDB_868', 'NDB_758', 'NDB_665']\n",
      "293 -> ['NDB_844', 'NDB_1033', 'NDB_721']\n",
      "295 -> ['NDB_817', 'NDB_1027', 'NDB_754']\n",
      "296 -> ['NDB_746', 'NDB_856', 'NDB_934']\n",
      "297 -> ['NDB_713', 'NDB_880', 'NDB_940']\n",
      "298 -> ['NDB_971', 'NDB_778', 'NDB_872']\n",
      "299 -> ['NDB_715', 'NDB_899', 'NDB_935']\n",
      "300 -> ['NDB_961', 'NDB_730', 'NDB_835']\n",
      "302 -> ['NDB_901', 'NDB_1060', 'NDB_889']\n",
      "303 -> ['NDB_784', 'NDB_1066', 'NDB_836']\n",
      "304 -> ['NDB_1075', 'NDB_641', 'NDB_712']\n",
      "305 -> ['NDB_790', 'NDB_634', 'NDB_941']\n",
      "306 -> ['NDB_653', 'NDB_838', 'NDB_881']\n",
      "307 -> ['NDB_865', 'NDB_752', 'NDB_926']\n",
      "308 -> ['NDB_815', 'NDB_698', 'NDB_853']\n",
      "309 -> ['NDB_866', 'NDB_1028', 'NDB_824']\n",
      "310 -> ['NDB_743', 'NDB_716', 'NDB_878']\n",
      "311 -> ['NDB_643', 'NDB_883', 'NDB_706']\n",
      "312 -> ['NDB_860', 'NDB_994', 'NDB_910']\n",
      "313 -> ['NDB_1027', 'NDB_680', 'NDB_694']\n",
      "314 -> ['NDB_754', 'NDB_887', 'NDB_637']\n",
      "315 -> ['NDB_755', 'NDB_773', 'NDB_977']\n",
      "316 -> ['NDB_737', 'NDB_751', 'NDB_958']\n",
      "317 -> ['NDB_698', 'NDB_889', 'NDB_818']\n",
      "318 -> ['NDB_631', 'NDB_712', 'NDB_934']\n",
      "319 -> ['NDB_713', 'NDB_661', 'NDB_925']\n",
      "320 -> ['NDB_760', 'NDB_901', 'NDB_940']\n",
      "321 -> ['NDB_784', 'NDB_847', 'NDB_665']\n",
      "322 -> ['NDB_710', 'NDB_835', 'NDB_899']\n",
      "323 -> ['NDB_971', 'NDB_961', 'NDB_935']\n",
      "324 -> ['NDB_683', 'NDB_728', 'NDB_832']\n",
      "325 -> ['NDB_905', 'NDB_752', 'NDB_941']\n",
      "326 -> ['NDB_974', 'NDB_680', 'NDB_989']\n",
      "327 -> ['NDB_805', 'NDB_896', 'NDB_853']\n",
      "328 -> ['NDB_868', 'NDB_812', 'NDB_946']\n",
      "329 -> ['NDB_755', 'NDB_826', 'NDB_875']\n",
      "330 -> ['NDB_758', 'NDB_1060', 'NDB_742']\n",
      "331 -> ['NDB_718', 'NDB_881', 'NDB_824']\n",
      "332 -> ['NDB_965', 'NDB_856', 'NDB_929']\n",
      "333 -> ['NDB_652', 'NDB_746', 'NDB_721']\n",
      "334 -> ['NDB_815', 'NDB_730', 'NDB_647']\n",
      "335 -> ['NDB_734', 'NDB_788', 'NDB_926']\n",
      "336 -> ['NDB_865', 'NDB_716', 'NDB_706']\n",
      "337 -> ['NDB_773', 'NDB_866', 'NDB_712']\n",
      "338 -> ['NDB_994', 'NDB_862', 'NDB_1028']\n",
      "339 -> ['NDB_790', 'NDB_974', 'NDB_715']\n",
      "340 -> ['NDB_826', 'NDB_917', 'NDB_844']\n",
      "341 -> ['NDB_743', 'NDB_805', 'NDB_880']\n",
      "342 -> ['NDB_905', 'NDB_779', 'NDB_872']\n",
      "343 -> ['NDB_1033', 'NDB_1066', 'NDB_958']\n",
      "344 -> ['NDB_680', 'NDB_838', 'NDB_910']\n",
      "345 -> ['NDB_725', 'NDB_860', 'NDB_637']\n",
      "346 -> ['NDB_643', 'NDB_1027', 'NDB_878']\n",
      "347 -> ['NDB_754', 'NDB_863', 'NDB_1075']\n",
      "348 -> ['NDB_778', 'NDB_883', 'NDB_989']\n",
      "349 -> ['NDB_817', 'NDB_634', 'NDB_641']\n",
      "350 -> ['NDB_683', 'NDB_889', 'NDB_788']\n",
      "351 -> ['NDB_815', 'NDB_706', 'NDB_836']\n",
      "352 -> ['NDB_718', 'NDB_1027', 'NDB_856']\n",
      "353 -> ['NDB_698', 'NDB_758', 'NDB_835']\n",
      "354 -> ['NDB_661', 'NDB_905', 'NDB_958']\n",
      "355 -> ['NDB_847', 'NDB_838', 'NDB_934']\n",
      "356 -> ['NDB_865', 'NDB_790', 'NDB_728']\n",
      "357 -> ['NDB_868', 'NDB_653', 'NDB_818']\n",
      "358 -> ['NDB_817', 'NDB_1033', 'NDB_896']\n",
      "359 -> ['NDB_680', 'NDB_941', 'NDB_926']\n",
      "360 -> ['NDB_713', 'NDB_901', 'NDB_899']\n",
      "361 -> ['NDB_875', 'NDB_883', 'NDB_647']\n",
      "362 -> ['NDB_971', 'NDB_878', 'NDB_929']\n",
      "363 -> ['NDB_965', 'NDB_752', 'NDB_853']\n",
      "364 -> ['NDB_754', 'NDB_1066', 'NDB_694']\n",
      "365 -> ['NDB_760', 'NDB_778', 'NDB_860']\n",
      "366 -> ['NDB_716', 'NDB_1060', 'NDB_887']\n",
      "367 -> ['NDB_826', 'NDB_710', 'NDB_742']\n",
      "368 -> ['NDB_863', 'NDB_631', 'NDB_665']\n",
      "370 -> ['NDB_847', 'NDB_917', 'NDB_989']\n",
      "371 -> ['NDB_961', 'NDB_641', 'NDB_824']\n",
      "372 -> ['NDB_683', 'NDB_1075', 'NDB_946']\n",
      "373 -> ['NDB_805', 'NDB_935', 'NDB_862']\n",
      "374 -> ['NDB_746', 'NDB_680', 'NDB_751']\n",
      "375 -> ['NDB_865', 'NDB_773', 'NDB_1028']\n",
      "376 -> ['NDB_728', 'NDB_974', 'NDB_1027']\n",
      "377 -> ['NDB_875', 'NDB_881', 'NDB_940']\n",
      "379 -> ['NDB_817', 'NDB_965', 'NDB_730']\n",
      "380 -> ['NDB_880', 'NDB_712', 'NDB_818']\n",
      "381 -> ['NDB_643', 'NDB_929', 'NDB_872']\n",
      "382 -> ['NDB_815', 'NDB_844', 'NDB_910']\n",
      "383 -> ['NDB_896', 'NDB_977', 'NDB_926']\n",
      "384 -> ['NDB_784', 'NDB_737', 'NDB_715']\n",
      "385 -> ['NDB_725', 'NDB_883', 'NDB_899']\n",
      "386 -> ['NDB_755', 'NDB_653', 'NDB_989']\n",
      "387 -> ['NDB_971', 'NDB_889', 'NDB_925']\n",
      "388 -> ['NDB_652', 'NDB_901', 'NDB_941']\n",
      "389 -> ['NDB_713', 'NDB_634', 'NDB_721']\n",
      "390 -> ['NDB_743', 'NDB_812', 'NDB_647']\n",
      "391 -> ['NDB_1033', 'NDB_866', 'NDB_734']\n",
      "392 -> ['NDB_868', 'NDB_716', 'NDB_752']\n",
      "393 -> ['NDB_860', 'NDB_779', 'NDB_694']\n",
      "394 -> ['NDB_773', 'NDB_994', 'NDB_788']\n",
      "395 -> ['NDB_917', 'NDB_878', 'NDB_853']\n",
      "396 -> ['NDB_637', 'NDB_706', 'NDB_1028']\n",
      "397 -> ['NDB_758', 'NDB_836', 'NDB_910']\n",
      "398 -> ['NDB_815', 'NDB_790', 'NDB_1066']\n",
      "399 -> ['NDB_643', 'NDB_974', 'NDB_832']\n",
      "400 -> ['NDB_737', 'NDB_805', 'NDB_872']\n",
      "401 -> ['NDB_661', 'NDB_754', 'NDB_742']\n",
      "402 -> ['NDB_835', 'NDB_838', 'NDB_665']\n",
      "403 -> ['NDB_1060', 'NDB_641', 'NDB_977']\n",
      "404 -> ['NDB_652', 'NDB_905', 'NDB_824']\n",
      "405 -> ['NDB_725', 'NDB_1075', 'NDB_940']\n",
      "406 -> ['NDB_730', 'NDB_862', 'NDB_934']\n",
      "407 -> ['NDB_856', 'NDB_863', 'NDB_881']\n",
      "408 -> ['NDB_710', 'NDB_760', 'NDB_958']\n",
      "409 -> ['NDB_784', 'NDB_755', 'NDB_880']\n",
      "410 -> ['NDB_778', 'NDB_946', 'NDB_712']\n",
      "411 -> ['NDB_743', 'NDB_887', 'NDB_721']\n",
      "412 -> ['NDB_826', 'NDB_634', 'NDB_637']\n",
      "413 -> ['NDB_718', 'NDB_866', 'NDB_788']\n",
      "414 -> ['NDB_631', 'NDB_734', 'NDB_910']\n",
      "415 -> ['NDB_790', 'NDB_994', 'NDB_925']\n",
      "416 -> ['NDB_1027', 'NDB_935', 'NDB_779']\n",
      "417 -> ['NDB_715', 'NDB_905', 'NDB_751']\n",
      "418 -> ['NDB_847', 'NDB_746', 'NDB_698']\n",
      "419 -> ['NDB_812', 'NDB_758', 'NDB_832']\n",
      "421 -> ['NDB_844', 'NDB_961', 'NDB_773']\n",
      "422 -> ['NDB_974', 'NDB_860', 'NDB_721']\n",
      "423 -> ['NDB_866', 'NDB_742', 'NDB_872']\n",
      "424 -> ['NDB_710', 'NDB_754', 'NDB_935']\n",
      "425 -> ['NDB_815', 'NDB_773', 'NDB_889']\n",
      "426 -> ['NDB_683', 'NDB_961', 'NDB_805']\n",
      "427 -> ['NDB_887', 'NDB_1028', 'NDB_926']\n",
      "428 -> ['NDB_713', 'NDB_875', 'NDB_896']\n",
      "429 -> ['NDB_790', 'NDB_778', 'NDB_680']\n",
      "430 -> ['NDB_817', 'NDB_856', 'NDB_853']\n",
      "431 -> ['NDB_698', 'NDB_878', 'NDB_694']\n",
      "432 -> ['NDB_865', 'NDB_758', 'NDB_712']\n",
      "435 -> ['NDB_1027', 'NDB_899', 'NDB_838']\n",
      "436 -> ['NDB_971', 'NDB_812', 'NDB_862']\n",
      "437 -> ['NDB_755', 'NDB_974', 'NDB_779']\n",
      "438 -> ['NDB_946', 'NDB_940', 'NDB_910']\n",
      "439 -> ['NDB_652', 'NDB_868', 'NDB_977']\n",
      "440 -> ['NDB_728', 'NDB_737', 'NDB_989']\n",
      "441 -> ['NDB_715', 'NDB_896', 'NDB_925']\n",
      "442 -> ['NDB_784', 'NDB_718', 'NDB_653']\n",
      "443 -> ['NDB_881', 'NDB_889', 'NDB_929']\n",
      "444 -> ['NDB_760', 'NDB_631', 'NDB_1075']\n",
      "446 -> ['NDB_710', 'NDB_661', 'NDB_941']\n",
      "447 -> ['NDB_812', 'NDB_994', 'NDB_818']\n",
      "448 -> ['NDB_743', 'NDB_863', 'NDB_832']\n",
      "449 -> ['NDB_634', 'NDB_647', 'NDB_934']\n",
      "450 -> ['NDB_713', 'NDB_725', 'NDB_1033']\n",
      "451 -> ['NDB_790', 'NDB_866', 'NDB_751']\n",
      "452 -> ['NDB_971', 'NDB_844', 'NDB_901']\n",
      "453 -> ['NDB_643', 'NDB_754', 'NDB_836']\n",
      "454 -> ['NDB_683', 'NDB_746', 'NDB_730']\n",
      "455 -> ['NDB_778', 'NDB_805', 'NDB_641']\n",
      "456 -> ['NDB_826', 'NDB_815', 'NDB_872']\n",
      "457 -> ['NDB_698', 'NDB_887', 'NDB_752']\n",
      "458 -> ['NDB_865', 'NDB_853', 'NDB_665']\n",
      "459 -> ['NDB_856', 'NDB_994', 'NDB_706']\n",
      "460 -> ['NDB_860', 'NDB_878', 'NDB_835']\n",
      "461 -> ['NDB_716', 'NDB_694', 'NDB_818']\n",
      "462 -> ['NDB_773', 'NDB_883', 'NDB_905']\n",
      "463 -> ['NDB_847', 'NDB_637', 'NDB_734']\n",
      "464 -> ['NDB_1060', 'NDB_880', 'NDB_935']\n",
      "465 -> ['NDB_917', 'NDB_875', 'NDB_958']\n",
      "466 -> ['NDB_965', 'NDB_961', 'NDB_680']\n",
      "467 -> ['NDB_725', 'NDB_788', 'NDB_824']\n",
      "468 -> ['NDB_817', 'NDB_901', 'NDB_734']\n",
      "469 -> ['NDB_883', 'NDB_1066', 'NDB_926']\n",
      "470 -> ['NDB_653', 'NDB_721', 'NDB_1028']\n",
      "471 -> ['NDB_718', 'NDB_752', 'NDB_706']\n",
      "472 -> ['NDB_716', 'NDB_946', 'NDB_929']\n",
      "473 -> ['NDB_631', 'NDB_637', 'NDB_779']\n",
      "474 -> ['NDB_826', 'NDB_760', 'NDB_941']\n",
      "475 -> ['NDB_730', 'NDB_925', 'NDB_940']\n",
      "476 -> ['NDB_1075', 'NDB_832', 'NDB_977']\n",
      "477 -> ['NDB_881', 'NDB_836', 'NDB_712']\n",
      "478 -> ['NDB_743', 'NDB_868', 'NDB_989']\n",
      "479 -> ['NDB_661', 'NDB_862', 'NDB_751']\n",
      "480 -> ['NDB_728', 'NDB_958', 'NDB_910']\n",
      "481 -> ['NDB_880', 'NDB_641', 'NDB_838']\n",
      "482 -> ['NDB_899', 'NDB_1066', 'NDB_665']\n",
      "483 -> ['NDB_652', 'NDB_758', 'NDB_934']\n",
      "484 -> ['NDB_784', 'NDB_863', 'NDB_1033']\n",
      "485 -> ['NDB_643', 'NDB_737', 'NDB_647']\n",
      "486 -> ['NDB_746', 'NDB_634', 'NDB_835']\n",
      "487 -> ['NDB_755', 'NDB_1027', 'NDB_715']\n",
      "489 -> ['NDB_844', 'NDB_860', 'NDB_742']\n",
      "490 -> ['NDB_835', 'NDB_641', 'NDB_734']\n",
      "491 -> ['NDB_698', 'NDB_935', 'NDB_1028']\n",
      "492 -> ['NDB_826', 'NDB_832', 'NDB_926']\n",
      "493 -> ['NDB_865', 'NDB_880', 'NDB_824']\n",
      "494 -> ['NDB_847', 'NDB_752', 'NDB_1066']\n",
      "495 -> ['NDB_875', 'NDB_994', 'NDB_665']\n",
      "496 -> ['NDB_971', 'NDB_746', 'NDB_706']\n",
      "497 -> ['NDB_901', 'NDB_1033', 'NDB_934']\n",
      "498 -> ['NDB_778', 'NDB_1075', 'NDB_637']\n",
      "499 -> ['NDB_710', 'NDB_631', 'NDB_838']\n",
      "500 -> ['NDB_1027', 'NDB_905', 'NDB_788']\n",
      "501 -> ['NDB_817', 'NDB_790', 'NDB_917']\n",
      "502 -> ['NDB_713', 'NDB_961', 'NDB_734']\n",
      "503 -> ['NDB_965', 'NDB_835', 'NDB_958']\n",
      "504 -> ['NDB_989', 'NDB_881', 'NDB_1066']\n",
      "505 -> ['NDB_868', 'NDB_715', 'NDB_779']\n",
      "506 -> ['NDB_805', 'NDB_925', 'NDB_818']\n",
      "507 -> ['NDB_652', 'NDB_743', 'NDB_1060']\n",
      "508 -> ['NDB_815', 'NDB_634', 'NDB_946']\n",
      "509 -> ['NDB_737', 'NDB_941', 'NDB_994']\n",
      "510 -> ['NDB_812', 'NDB_860', 'NDB_836']\n",
      "511 -> ['NDB_883', 'NDB_977', 'NDB_712']\n",
      "512 -> ['NDB_755', 'NDB_680', 'NDB_863']\n",
      "516 -> ['NDB_716', 'NDB_730', 'NDB_889']\n",
      "517 -> ['NDB_974', 'NDB_878', 'NDB_866']\n",
      "518 -> ['NDB_725', 'NDB_989', 'NDB_929']\n",
      "519 -> ['NDB_728', 'NDB_721', 'NDB_694']\n",
      "520 -> ['NDB_784', 'NDB_856', 'NDB_1028']\n",
      "521 -> ['NDB_683', 'NDB_853', 'NDB_862']\n",
      "522 -> ['NDB_643', 'NDB_844', 'NDB_653']\n",
      "523 -> ['NDB_899', 'NDB_647', 'NDB_940']\n",
      "524 -> ['NDB_718', 'NDB_773', 'NDB_754']\n",
      "525 -> ['NDB_661', 'NDB_758', 'NDB_872']\n",
      "526 -> ['NDB_896', 'NDB_751', 'NDB_742']\n",
      "527 -> ['NDB_812', 'NDB_641', 'NDB_941']\n",
      "528 -> ['NDB_634', 'NDB_737', 'NDB_881']\n",
      "529 -> ['NDB_760', 'NDB_961', 'NDB_910']\n",
      "530 -> ['NDB_887', 'NDB_958', 'NDB_934']\n",
      "531 -> ['NDB_652', 'NDB_965', 'NDB_889']\n",
      "532 -> ['NDB_875', 'NDB_706', 'NDB_824']\n",
      "533 -> ['NDB_743', 'NDB_838', 'NDB_637']\n",
      "534 -> ['NDB_784', 'NDB_725', 'NDB_1027']\n",
      "535 -> ['NDB_868', 'NDB_1060', 'NDB_905']\n",
      "536 -> ['NDB_971', 'NDB_1033', 'NDB_631']\n",
      "537 -> ['NDB_716', 'NDB_788', 'NDB_925']\n",
      "538 -> ['NDB_710', 'NDB_866', 'NDB_853']\n",
      "539 -> ['NDB_728', 'NDB_746', 'NDB_878']\n",
      "540 -> ['NDB_790', 'NDB_862', 'NDB_818']\n",
      "541 -> ['NDB_713', 'NDB_1075', 'NDB_805']\n",
      "542 -> ['NDB_826', 'NDB_778', 'NDB_715']\n",
      "543 -> ['NDB_718', 'NDB_901', 'NDB_694']\n",
      "544 -> ['NDB_917', 'NDB_665', 'NDB_935']\n",
      "545 -> ['NDB_865', 'NDB_847', 'NDB_680']\n",
      "546 -> ['NDB_817', 'NDB_880', 'NDB_910']\n",
      "548 -> ['NDB_883', 'NDB_887', 'NDB_779']\n",
      "549 -> ['NDB_653', 'NDB_752', 'NDB_946']\n",
      "550 -> ['NDB_863', 'NDB_977', 'NDB_872']\n",
      "551 -> ['NDB_860', 'NDB_647', 'NDB_926']\n",
      "552 -> ['NDB_760', 'NDB_698', 'NDB_832']\n",
      "554 -> ['NDB_868', 'NDB_880', 'NDB_721']\n",
      "555 -> ['NDB_974', 'NDB_863', 'NDB_889']\n",
      "556 -> ['NDB_784', 'NDB_860', 'NDB_940']\n",
      "557 -> ['NDB_752', 'NDB_989', 'NDB_712']\n",
      "558 -> ['NDB_728', 'NDB_1060', 'NDB_934']\n",
      "559 -> ['NDB_718', 'NDB_844', 'NDB_758']\n",
      "560 -> ['NDB_746', 'NDB_853', 'NDB_1066']\n",
      "561 -> ['NDB_790', 'NDB_754', 'NDB_1028']\n",
      "562 -> ['NDB_941', 'NDB_637', 'NDB_824']\n",
      "563 -> ['NDB_865', 'NDB_971', 'NDB_1075']\n",
      "564 -> ['NDB_737', 'NDB_880', 'NDB_734']\n",
      "565 -> ['NDB_716', 'NDB_751', 'NDB_836']\n",
      "566 -> ['NDB_778', 'NDB_899', 'NDB_742']\n",
      "567 -> ['NDB_901', 'NDB_832', 'NDB_881']\n",
      "568 -> ['NDB_661', 'NDB_680', 'NDB_788']\n",
      "569 -> ['NDB_760', 'NDB_730', 'NDB_838']\n",
      "570 -> ['NDB_878', 'NDB_905', 'NDB_712']\n",
      "571 -> ['NDB_826', 'NDB_835', 'NDB_935']\n",
      "572 -> ['NDB_866', 'NDB_929', 'NDB_958']\n",
      "573 -> ['NDB_961', 'NDB_1027', 'NDB_887']\n",
      "574 -> ['NDB_773', 'NDB_631', 'NDB_862']\n",
      "575 -> ['NDB_718', 'NDB_847', 'NDB_853']\n",
      "576 -> ['NDB_652', 'NDB_728', 'NDB_754']\n",
      "577 -> ['NDB_755', 'NDB_815', 'NDB_925']\n",
      "578 -> ['NDB_1066', 'NDB_779', 'NDB_1028']\n",
      "579 -> ['NDB_817', 'NDB_643', 'NDB_946']\n",
      "580 -> ['NDB_965', 'NDB_812', 'NDB_706']\n",
      "581 -> ['NDB_868', 'NDB_856', 'NDB_634']\n",
      "582 -> ['NDB_790', 'NDB_641', 'NDB_935']\n",
      "583 -> ['NDB_710', 'NDB_715', 'NDB_881']\n",
      "584 -> ['NDB_743', 'NDB_1033', 'NDB_905']\n",
      "585 -> ['NDB_865', 'NDB_917', 'NDB_698']\n",
      "586 -> ['NDB_971', 'NDB_977', 'NDB_818']\n",
      "587 -> ['NDB_805', 'NDB_694', 'NDB_836']\n",
      "588 -> ['NDB_725', 'NDB_896', 'NDB_647']\n",
      "589 -> ['NDB_875', 'NDB_889', 'NDB_910']\n",
      "590 -> ['NDB_652', 'NDB_683', 'NDB_653']\n",
      "591 -> ['NDB_713', 'NDB_883', 'NDB_665']\n",
      "592 -> ['NDB_994', 'NDB_872', 'NDB_926']\n",
      "593 -> ['NDB_847', 'NDB_641', 'NDB_788']\n",
      "594 -> ['NDB_778', 'NDB_721', 'NDB_779']\n",
      "595 -> ['NDB_856', 'NDB_878', 'NDB_1060']\n",
      "596 -> ['NDB_743', 'NDB_844', 'NDB_737']\n",
      "597 -> ['NDB_710', 'NDB_961', 'NDB_1075']\n",
      "598 -> ['NDB_965', 'NDB_637', 'NDB_940']\n",
      "599 -> ['NDB_946', 'NDB_941', 'NDB_734']\n",
      "600 -> ['NDB_812', 'NDB_631', 'NDB_824']\n",
      "601 -> ['NDB_815', 'NDB_860', 'NDB_934']\n",
      "602 -> ['NDB_989', 'NDB_899', 'NDB_862']\n",
      "603 -> ['NDB_817', 'NDB_760', 'NDB_758']\n",
      "605 -> ['NDB_661', 'NDB_752', 'NDB_896']\n",
      "606 -> ['NDB_784', 'NDB_974', 'NDB_929']\n",
      "607 -> ['NDB_863', 'NDB_866', 'NDB_925']\n",
      "608 -> ['NDB_887', 'NDB_835', 'NDB_751']\n",
      "609 -> ['NDB_746', 'NDB_832', 'NDB_838']\n",
      "611 -> ['NDB_713', 'NDB_977', 'NDB_910']\n",
      "612 -> ['NDB_847', 'NDB_737', 'NDB_694']\n",
      "613 -> ['NDB_790', 'NDB_721', 'NDB_742']\n",
      "614 -> ['NDB_856', 'NDB_905', 'NDB_805']\n",
      "615 -> ['NDB_917', 'NDB_883', 'NDB_788']\n",
      "616 -> ['NDB_971', 'NDB_634', 'NDB_880']\n",
      "617 -> ['NDB_965', 'NDB_1033', 'NDB_1028']\n",
      "618 -> ['NDB_778', 'NDB_844', 'NDB_818']\n",
      "619 -> ['NDB_725', 'NDB_1060', 'NDB_779']\n",
      "620 -> ['NDB_643', 'NDB_868', 'NDB_734']\n",
      "621 -> ['NDB_716', 'NDB_754', 'NDB_940']\n",
      "622 -> ['NDB_773', 'NDB_758', 'NDB_929']\n",
      "623 -> ['NDB_826', 'NDB_946', 'NDB_641']\n",
      "624 -> ['NDB_728', 'NDB_901', 'NDB_887']\n",
      "625 -> ['NDB_710', 'NDB_875', 'NDB_989']\n",
      "626 -> ['NDB_683', 'NDB_812', 'NDB_935']\n",
      "627 -> ['NDB_746', 'NDB_647', 'NDB_824']\n",
      "628 -> ['NDB_652', 'NDB_680', 'NDB_1066']\n",
      "629 -> ['NDB_713', 'NDB_866', 'NDB_926']\n",
      "645 -> ['NDB_1027', 'NDB_698', 'NDB_872']\n",
      "646 -> ['NDB_865', 'NDB_718', 'NDB_743']\n",
      "647 -> ['NDB_755', 'NDB_881', 'NDB_706']\n",
      "648 -> ['NDB_634', 'NDB_863', 'NDB_878']\n",
      "649 -> ['NDB_643', 'NDB_1033', 'NDB_941']\n",
      "650 -> ['NDB_725', 'NDB_836', 'NDB_818']\n",
      "651 -> ['NDB_631', 'NDB_653', 'NDB_751']\n",
      "652 -> ['NDB_974', 'NDB_838', 'NDB_958']\n",
      "653 -> ['NDB_805', 'NDB_889', 'NDB_712']\n",
      "654 -> ['NDB_784', 'NDB_835', 'NDB_637']\n",
      "655 -> ['NDB_773', 'NDB_860', 'NDB_1075']\n",
      "656 -> ['NDB_730', 'NDB_853', 'NDB_994']\n",
      "657 -> ['NDB_715', 'NDB_653', 'NDB_934']\n",
      "658 -> ['NDB_815', 'NDB_752', 'NDB_899']\n",
      "659 -> ['NDB_832', 'NDB_896', 'NDB_862']\n",
      "660 -> ['NDB_760', 'NDB_661', 'NDB_665']\n",
      "662 -> ['NDB_817', 'NDB_637', 'NDB_925']\n",
      "663 -> ['NDB_826', 'NDB_961', 'NDB_1060']\n",
      "664 -> ['NDB_716', 'NDB_641', 'NDB_926']\n",
      "665 -> ['NDB_754', 'NDB_941', 'NDB_958']\n",
      "666 -> ['NDB_718', 'NDB_868', 'NDB_836']\n",
      "667 -> ['NDB_971', 'NDB_866', 'NDB_779']\n",
      "668 -> ['NDB_652', 'NDB_1027', 'NDB_737']\n",
      "669 -> ['NDB_778', 'NDB_905', 'NDB_730']\n",
      "670 -> ['NDB_865', 'NDB_694', 'NDB_977']\n",
      "671 -> ['NDB_856', 'NDB_680', 'NDB_742']\n",
      "672 -> ['NDB_647', 'NDB_788', 'NDB_1028']\n",
      "673 -> ['NDB_710', 'NDB_847', 'NDB_880']\n",
      "674 -> ['NDB_901', 'NDB_721', 'NDB_824']\n",
      "675 -> ['NDB_755', 'NDB_790', 'NDB_734']\n",
      "676 -> ['NDB_683', 'NDB_844', 'NDB_883']\n",
      "677 -> ['NDB_812', 'NDB_872', 'NDB_910']\n",
      "678 -> ['NDB_917', 'NDB_706', 'NDB_929']\n",
      "679 -> ['NDB_965', 'NDB_728', 'NDB_875']\n",
      "680 -> ['NDB_866', 'NDB_721', 'NDB_994']\n",
      "681 -> ['NDB_844', 'NDB_752', 'NDB_1028']\n",
      "682 -> ['NDB_881', 'NDB_665', 'NDB_637']\n",
      "683 -> ['NDB_755', 'NDB_862', 'NDB_742']\n",
      "684 -> ['NDB_710', 'NDB_863', 'NDB_788']\n",
      "685 -> ['NDB_683', 'NDB_856', 'NDB_698']\n",
      "686 -> ['NDB_652', 'NDB_989', 'NDB_940']\n",
      "687 -> ['NDB_728', 'NDB_883', 'NDB_805']\n",
      "688 -> ['NDB_715', 'NDB_730', 'NDB_946']\n",
      "689 -> ['NDB_713', 'NDB_832', 'NDB_751']\n",
      "690 -> ['NDB_643', 'NDB_760', 'NDB_773']\n",
      "691 -> ['NDB_737', 'NDB_866', 'NDB_889']\n",
      "692 -> ['NDB_974', 'NDB_746', 'NDB_910']\n",
      "693 -> ['NDB_865', 'NDB_860', 'NDB_631']\n",
      "694 -> ['NDB_661', 'NDB_878', 'NDB_734']\n",
      "695 -> ['NDB_901', 'NDB_905', 'NDB_853']\n",
      "696 -> ['NDB_847', 'NDB_941', 'NDB_647']\n",
      "697 -> ['NDB_1027', 'NDB_716', 'NDB_934']\n",
      "698 -> ['NDB_971', 'NDB_838', 'NDB_1028']\n",
      "699 -> ['NDB_790', 'NDB_653', 'NDB_706']\n",
      "701 -> ['NDB_784', 'NDB_728', 'NDB_1075']\n",
      "702 -> ['NDB_743', 'NDB_836', 'NDB_929']\n",
      "703 -> ['NDB_965', 'NDB_778', 'NDB_824']\n",
      "704 -> ['NDB_917', 'NDB_754', 'NDB_730']\n",
      "705 -> ['NDB_880', 'NDB_896', 'NDB_889']\n",
      "706 -> ['NDB_1060', 'NDB_958', 'NDB_872']\n",
      "707 -> ['NDB_760', 'NDB_721', 'NDB_751']\n",
      "708 -> ['NDB_815', 'NDB_631', 'NDB_641']\n",
      "709 -> ['NDB_643', 'NDB_746', 'NDB_788']\n",
      "710 -> ['NDB_961', 'NDB_862', 'NDB_712']\n",
      "711 -> ['NDB_634', 'NDB_805', 'NDB_899']\n",
      "712 -> ['NDB_868', 'NDB_698', 'NDB_940']\n",
      "713 -> ['NDB_1033', 'NDB_935', 'NDB_910']\n",
      "714 -> ['NDB_718', 'NDB_661', 'NDB_812']\n",
      "715 -> ['NDB_755', 'NDB_725', 'NDB_710']\n",
      "716 -> ['NDB_713', 'NDB_887', 'NDB_838']\n",
      "717 -> ['NDB_901', 'NDB_989', 'NDB_665']\n",
      "718 -> ['NDB_683', 'NDB_974', 'NDB_925']\n",
      "719 -> ['NDB_652', 'NDB_863', 'NDB_637']\n",
      "720 -> ['NDB_875', 'NDB_680', 'NDB_835']\n",
      "721 -> ['NDB_716', 'NDB_977', 'NDB_742']\n",
      "722 -> ['NDB_878', 'NDB_946', 'NDB_706']\n",
      "723 -> ['NDB_865', 'NDB_832', 'NDB_734']\n",
      "724 -> ['NDB_647', 'NDB_994', 'NDB_694']\n",
      "725 -> ['NDB_971', 'NDB_860', 'NDB_853']\n",
      "726 -> ['NDB_758', 'NDB_779', 'NDB_926']\n",
      "727 -> ['NDB_826', 'NDB_737', 'NDB_883']\n",
      "728 -> ['NDB_881', 'NDB_779', 'NDB_818']\n",
      "729 -> ['NDB_817', 'NDB_773', 'NDB_1066']\n",
      "730 -> ['NDB_784', 'NDB_743', 'NDB_752']\n",
      "731 -> ['NDB_1033', 'NDB_653', 'NDB_712']\n",
      "732 -> ['NDB_680', 'NDB_1075', 'NDB_880']\n",
      "733 -> ['NDB_965', 'NDB_844', 'NDB_941']\n",
      "734 -> ['NDB_718', 'NDB_778', 'NDB_925']\n",
      "735 -> ['NDB_868', 'NDB_835', 'NDB_824']\n",
      "736 -> ['NDB_815', 'NDB_961', 'NDB_958']\n",
      "737 -> ['NDB_917', 'NDB_641', 'NDB_872']\n",
      "738 -> ['NDB_812', 'NDB_634', 'NDB_1060']\n",
      "739 -> ['NDB_754', 'NDB_899', 'NDB_934']\n",
      "740 -> ['NDB_790', 'NDB_887', 'NDB_977']\n",
      "741 -> ['NDB_847', 'NDB_875', 'NDB_935']\n",
      "742 -> ['NDB_905', 'NDB_929', 'NDB_926']\n",
      "743 -> ['NDB_1027', 'NDB_896', 'NDB_836']\n",
      "744 -> ['NDB_856', 'NDB_715', 'NDB_866']\n",
      "745 -> ['NDB_725', 'NDB_694', 'NDB_1028']\n",
      "748 -> ['NDB_718', 'NDB_887', 'NDB_896']\n",
      "749 -> ['NDB_715', 'NDB_863', 'NDB_1066']\n",
      "750 -> ['NDB_680', 'NDB_730', 'NDB_958']\n",
      "751 -> ['NDB_713', 'NDB_728', 'NDB_889']\n",
      "752 -> ['NDB_643', 'NDB_880', 'NDB_751']\n",
      "753 -> ['NDB_815', 'NDB_1027', 'NDB_1075']\n",
      "754 -> ['NDB_812', 'NDB_899', 'NDB_926']\n",
      "755 -> ['NDB_754', 'NDB_929', 'NDB_712']\n",
      "756 -> ['NDB_661', 'NDB_1033', 'NDB_946']\n",
      "757 -> ['NDB_760', 'NDB_844', 'NDB_974']\n",
      "758 -> ['NDB_725', 'NDB_878', 'NDB_935']\n",
      "759 -> ['NDB_778', 'NDB_1060', 'NDB_734']\n",
      "760 -> ['NDB_805', 'NDB_881', 'NDB_788']\n",
      "761 -> ['NDB_784', 'NDB_853', 'NDB_824']\n",
      "762 -> ['NDB_847', 'NDB_716', 'NDB_866']\n",
      "763 -> ['NDB_917', 'NDB_758', 'NDB_752']\n",
      "764 -> ['NDB_883', 'NDB_721', 'NDB_836']\n",
      "765 -> ['NDB_971', 'NDB_718', 'NDB_875']\n",
      "766 -> ['NDB_652', 'NDB_730', 'NDB_910']\n",
      "767 -> ['NDB_989', 'NDB_647', 'NDB_742']\n",
      "768 -> ['NDB_737', 'NDB_665', 'NDB_940']\n",
      "769 -> ['NDB_815', 'NDB_661', 'NDB_926']\n",
      "770 -> ['NDB_725', 'NDB_889', 'NDB_751']\n",
      "774 -> ['NDB_812', 'NDB_653', 'NDB_929']\n",
      "775 -> ['NDB_826', 'NDB_683', 'NDB_773']\n",
      "776 -> ['NDB_760', 'NDB_715', 'NDB_887']\n",
      "777 -> ['NDB_716', 'NDB_941', 'NDB_935']\n",
      "778 -> ['NDB_817', 'NDB_698', 'NDB_994']\n",
      "779 -> ['NDB_713', 'NDB_853', 'NDB_788']\n",
      "780 -> ['NDB_901', 'NDB_637', 'NDB_818']\n",
      "782 -> ['NDB_965', 'NDB_1060', 'NDB_694']\n",
      "783 -> ['NDB_917', 'NDB_734', 'NDB_1028']\n",
      "784 -> ['NDB_847', 'NDB_631', 'NDB_805']\n",
      "785 -> ['NDB_784', 'NDB_865', 'NDB_778']\n",
      "786 -> ['NDB_755', 'NDB_754', 'NDB_905']\n",
      "787 -> ['NDB_746', 'NDB_860', 'NDB_880']\n",
      "788 -> ['NDB_634', 'NDB_832', 'NDB_958']\n",
      "789 -> ['NDB_856', 'NDB_977', 'NDB_925']\n",
      "790 -> ['NDB_743', 'NDB_1075', 'NDB_835']\n",
      "791 -> ['NDB_844', 'NDB_863', 'NDB_779']\n",
      "792 -> ['NDB_728', 'NDB_1066', 'NDB_862']\n",
      "793 -> ['NDB_974', 'NDB_961', 'NDB_752']\n",
      "794 -> ['NDB_1033', 'NDB_838', 'NDB_706']\n",
      "795 -> ['NDB_866', 'NDB_641', 'NDB_881']\n",
      "796 -> ['NDB_790', 'NDB_878', 'NDB_824']\n",
      "797 -> ['NDB_710', 'NDB_832', 'NDB_934']\n",
      "798 -> ['NDB_868', 'NDB_680', 'NDB_872']\n",
      "799 -> ['NDB_901', 'NDB_1027', 'NDB_712']\n",
      "800 -> ['NDB_896', 'NDB_946', 'NDB_1028']\n",
      "801 -> ['NDB_643', 'NDB_899', 'NDB_637']\n",
      "802 -> ['NDB_631', 'NDB_941', 'NDB_818']\n",
      "803 -> ['NDB_965', 'NDB_994', 'NDB_836']\n",
      "804 -> ['NDB_758', 'NDB_706', 'NDB_694']\n",
      "805 -> ['NDB_865', 'NDB_817', 'NDB_940']\n",
      "806 -> ['NDB_755', 'NDB_971', 'NDB_665']\n",
      "807 -> ['NDB_737', 'NDB_698', 'NDB_910']\n",
      "808 -> ['NDB_652', 'NDB_835', 'NDB_742']\n",
      "809 -> ['NDB_773', 'NDB_721', 'NDB_647']\n",
      "810 -> ['NDB_856', 'NDB_883', 'NDB_653']\n",
      "811 -> ['NDB_826', 'NDB_989', 'NDB_977']\n",
      "812 -> ['NDB_683', 'NDB_875', 'NDB_860']\n",
      "813 -> ['NDB_746', 'NDB_737', 'NDB_899']\n",
      "814 -> ['NDB_896', 'NDB_721', 'NDB_958']\n",
      "815 -> ['NDB_974', 'NDB_856', 'NDB_641']\n",
      "816 -> ['NDB_661', 'NDB_860', 'NDB_883']\n",
      "817 -> ['NDB_715', 'NDB_665', 'NDB_910']\n",
      "819 -> ['NDB_917', 'NDB_634', 'NDB_862']\n",
      "820 -> ['NDB_710', 'NDB_965', 'NDB_818']\n",
      "821 -> ['NDB_643', 'NDB_716', 'NDB_853']\n",
      "822 -> ['NDB_865', 'NDB_878', 'NDB_925']\n",
      "823 -> ['NDB_1060', 'NDB_752', 'NDB_836']\n",
      "824 -> ['NDB_760', 'NDB_863', 'NDB_835']\n",
      "825 -> ['NDB_743', 'NDB_994', 'NDB_742']\n",
      "826 -> ['NDB_875', 'NDB_838', 'NDB_712']\n",
      "827 -> ['NDB_1033', 'NDB_887', 'NDB_832']\n",
      "828 -> ['NDB_754', 'NDB_946', 'NDB_779']\n",
      "829 -> ['NDB_713', 'NDB_773', 'NDB_698']\n",
      "832 -> ['NDB_718', 'NDB_790', 'NDB_889']\n",
      "833 -> ['NDB_725', 'NDB_812', 'NDB_715']\n",
      "834 -> ['NDB_868', 'NDB_730', 'NDB_1066']\n",
      "835 -> ['NDB_728', 'NDB_844', 'NDB_751']\n",
      "836 -> ['NDB_817', 'NDB_863', 'NDB_694']\n",
      "837 -> ['NDB_961', 'NDB_634', 'NDB_758']\n",
      "838 -> ['NDB_883', 'NDB_880', 'NDB_958']\n",
      "839 -> ['NDB_661', 'NDB_721', 'NDB_940']\n",
      "840 -> ['NDB_647', 'NDB_665', 'NDB_977']\n",
      "841 -> ['NDB_710', 'NDB_680', 'NDB_653']\n",
      "843 -> ['NDB_971', 'NDB_815', 'NDB_905']\n",
      "844 -> ['NDB_826', 'NDB_974', 'NDB_818']\n",
      "845 -> ['NDB_652', 'NDB_716', 'NDB_1075']\n",
      "846 -> ['NDB_743', 'NDB_790', 'NDB_872']\n",
      "847 -> ['NDB_755', 'NDB_965', 'NDB_878']\n",
      "848 -> ['NDB_917', 'NDB_694', 'NDB_926']\n",
      "849 -> ['NDB_773', 'NDB_730', 'NDB_935']\n",
      "850 -> ['NDB_853', 'NDB_734', 'NDB_925']\n",
      "851 -> ['NDB_752', 'NDB_994', 'NDB_824']\n",
      "852 -> ['NDB_778', 'NDB_706', 'NDB_862']\n",
      "853 -> ['NDB_856', 'NDB_631', 'NDB_788']\n",
      "854 -> ['NDB_683', 'NDB_641', 'NDB_1028']\n",
      "855 -> ['NDB_805', 'NDB_989', 'NDB_934']\n",
      "856 -> ['NDB_746', 'NDB_901', 'NDB_929']\n",
      "857 -> ['NDB_643', 'NDB_866', 'NDB_896']\n",
      "858 -> ['NDB_784', 'NDB_899', 'NDB_910']\n",
      "859 -> ['NDB_760', 'NDB_737', 'NDB_637']\n",
      "860 -> ['NDB_860', 'NDB_941', 'NDB_872']\n",
      "861 -> ['NDB_847', 'NDB_1027', 'NDB_881']\n",
      "862 -> ['NDB_865', 'NDB_1060', 'NDB_788']\n",
      "863 -> ['NDB_817', 'NDB_706', 'NDB_935']\n",
      "864 -> ['NDB_838', 'NDB_734', 'NDB_824']\n",
      "865 -> ['NDB_1033', 'NDB_758', 'NDB_977']\n",
      "866 -> ['NDB_1075', 'NDB_989', 'NDB_779']\n",
      "867 -> ['NDB_812', 'NDB_698', 'NDB_1066']\n",
      "868 -> ['NDB_826', 'NDB_836', 'NDB_934']\n",
      "869 -> ['NDB_905', 'NDB_835', 'NDB_889']\n",
      "870 -> ['NDB_683', 'NDB_880', 'NDB_742']\n",
      "871 -> ['NDB_815', 'NDB_868', 'NDB_754']\n",
      "872 -> ['NDB_844', 'NDB_832', 'NDB_647']\n",
      "873 -> ['NDB_652', 'NDB_946', 'NDB_751']\n",
      "874 -> ['NDB_755', 'NDB_713', 'NDB_718']\n",
      "875 -> ['NDB_971', 'NDB_728', 'NDB_653']\n",
      "876 -> ['NDB_887', 'NDB_929', 'NDB_940']\n",
      "877 -> ['NDB_725', 'NDB_680', 'NDB_712']\n",
      "878 -> ['NDB_961', 'NDB_875', 'NDB_926']\n",
      "888 -> ['NDB_917', 'NDB_1033', 'NDB_1060']\n",
      "889 -> ['NDB_743', 'NDB_881', 'NDB_734']\n",
      "890 -> ['NDB_905', 'NDB_838', 'NDB_977']\n",
      "891 -> ['NDB_631', 'NDB_880', 'NDB_925']\n",
      "892 -> ['NDB_716', 'NDB_989', 'NDB_910']\n",
      "893 -> ['NDB_715', 'NDB_641', 'NDB_721']\n",
      "894 -> ['NDB_643', 'NDB_728', 'NDB_961']\n",
      "895 -> ['NDB_835', 'NDB_941', 'NDB_862']\n",
      "896 -> ['NDB_883', 'NDB_758', 'NDB_751']\n",
      "897 -> ['NDB_917', 'NDB_1075', 'NDB_889']\n",
      "898 -> ['NDB_971', 'NDB_887', 'NDB_946']\n",
      "899 -> ['NDB_773', 'NDB_680', 'NDB_752']\n",
      "900 -> ['NDB_817', 'NDB_958', 'NDB_824']\n",
      "901 -> ['NDB_652', 'NDB_647', 'NDB_935']\n",
      "902 -> ['NDB_683', 'NDB_965', 'NDB_737']\n",
      "903 -> ['NDB_826', 'NDB_853', 'NDB_838']\n",
      "904 -> ['NDB_713', 'NDB_710', 'NDB_872']\n",
      "905 -> ['NDB_634', 'NDB_836', 'NDB_940']\n",
      "906 -> ['NDB_760', 'NDB_866', 'NDB_934']\n",
      "907 -> ['NDB_743', 'NDB_974', 'NDB_926']\n",
      "908 -> ['NDB_865', 'NDB_754', 'NDB_896']\n",
      "909 -> ['NDB_718', 'NDB_641', 'NDB_899']\n",
      "910 -> ['NDB_725', 'NDB_790', 'NDB_868']\n",
      "911 -> ['NDB_728', 'NDB_680', 'NDB_665']\n",
      "912 -> ['NDB_961', 'NDB_856', 'NDB_941']\n",
      "913 -> ['NDB_716', 'NDB_698', 'NDB_712']\n",
      "914 -> ['NDB_815', 'NDB_878', 'NDB_832']\n",
      "915 -> ['NDB_746', 'NDB_661', 'NDB_779']\n",
      "916 -> ['NDB_875', 'NDB_880', 'NDB_694']\n",
      "917 -> ['NDB_755', 'NDB_1060', 'NDB_994']\n",
      "918 -> ['NDB_784', 'NDB_730', 'NDB_788']\n",
      "919 -> ['NDB_860', 'NDB_905', 'NDB_818']\n",
      "920 -> ['NDB_812', 'NDB_863', 'NDB_1028']\n",
      "921 -> ['NDB_901', 'NDB_805', 'NDB_1066']\n",
      "922 -> ['NDB_847', 'NDB_653', 'NDB_742']\n",
      "923 -> ['NDB_896', 'NDB_637', 'NDB_929']\n",
      "924 -> ['NDB_844', 'NDB_665', 'NDB_836']\n",
      "925 -> ['NDB_965', 'NDB_1027', 'NDB_734']\n",
      "926 -> ['NDB_652', 'NDB_853', 'NDB_706']\n",
      "927 -> ['NDB_683', 'NDB_778', 'NDB_661']\n",
      "928 -> ['NDB_718', 'NDB_977', 'NDB_934']\n",
      "929 -> ['NDB_710', 'NDB_856', 'NDB_1033']\n",
      "930 -> ['NDB_899', 'NDB_881', 'NDB_925']\n",
      "931 -> ['NDB_971', 'NDB_790', 'NDB_883']\n",
      "932 -> ['NDB_773', 'NDB_751', 'NDB_824']\n",
      "933 -> ['NDB_715', 'NDB_878', 'NDB_862']\n",
      "934 -> ['NDB_1075', 'NDB_887', 'NDB_647']\n",
      "935 -> ['NDB_784', 'NDB_817', 'NDB_989']\n",
      "949 -> ['NDB_643', 'NDB_994', 'NDB_889']\n",
      "950 -> ['NDB_815', 'NDB_665', 'NDB_751']\n",
      "951 -> ['NDB_875', 'NDB_752', 'NDB_977']\n",
      "952 -> ['NDB_974', 'NDB_868', 'NDB_862']\n",
      "953 -> ['NDB_728', 'NDB_779', 'NDB_712']\n",
      "954 -> ['NDB_784', 'NDB_661', 'NDB_694']\n",
      "955 -> ['NDB_683', 'NDB_917', 'NDB_647']\n",
      "956 -> ['NDB_631', 'NDB_730', 'NDB_836']\n",
      "957 -> ['NDB_716', 'NDB_634', 'NDB_653']\n",
      "958 -> ['NDB_725', 'NDB_866', 'NDB_946']\n",
      "959 -> ['NDB_755', 'NDB_746', 'NDB_935']\n",
      "960 -> ['NDB_878', 'NDB_989', 'NDB_838']\n",
      "961 -> ['NDB_728', 'NDB_896', 'NDB_788']\n",
      "962 -> ['NDB_1060', 'NDB_832', 'NDB_706']\n",
      "963 -> ['NDB_883', 'NDB_863', 'NDB_941']\n",
      "964 -> ['NDB_715', 'NDB_637', 'NDB_926']\n",
      "965 -> ['NDB_865', 'NDB_961', 'NDB_899']\n",
      "966 -> ['NDB_1027', 'NDB_647', 'NDB_929']\n",
      "967 -> ['NDB_826', 'NDB_901', 'NDB_698']\n",
      "968 -> ['NDB_773', 'NDB_716', 'NDB_805']\n",
      "969 -> ['NDB_875', 'NDB_1028', 'NDB_925']\n",
      "970 -> ['NDB_853', 'NDB_751', 'NDB_712']\n",
      "971 -> ['NDB_643', 'NDB_835', 'NDB_934']\n",
      "972 -> ['NDB_713', 'NDB_860', 'NDB_653']\n",
      "973 -> ['NDB_1033', 'NDB_641', 'NDB_940']\n",
      "974 -> ['NDB_721', 'NDB_889', 'NDB_872']\n",
      "975 -> ['NDB_817', 'NDB_1075', 'NDB_818']\n",
      "976 -> ['NDB_758', 'NDB_866', 'NDB_730']\n",
      "977 -> ['NDB_989', 'NDB_935', 'NDB_958']\n",
      "978 -> ['NDB_760', 'NDB_836', 'NDB_926']\n",
      "979 -> ['NDB_847', 'NDB_965', 'NDB_868']\n",
      "980 -> ['NDB_773', 'NDB_856', 'NDB_734']\n",
      "981 -> ['NDB_917', 'NDB_901', 'NDB_680']\n",
      "982 -> ['NDB_835', 'NDB_694', 'NDB_910']\n",
      "983 -> ['NDB_971', 'NDB_743', 'NDB_1027']\n",
      "984 -> ['NDB_652', 'NDB_718', 'NDB_815']\n",
      "985 -> ['NDB_737', 'NDB_742', 'NDB_824']\n",
      "986 -> ['NDB_844', 'NDB_754', 'NDB_706']\n",
      "987 -> ['NDB_880', 'NDB_994', 'NDB_1066']\n",
      "988 -> ['NDB_710', 'NDB_974', 'NDB_905']\n",
      "989 -> ['NDB_760', 'NDB_790', 'NDB_812']\n",
      "990 -> ['NDB_778', 'NDB_887', 'NDB_881']\n",
      "991 -> ['NDB_713', 'NDB_941', 'NDB_779']\n",
      "992 -> ['NDB_1060', 'NDB_862', 'NDB_910']\n",
      "993 -> ['NDB_860', 'NDB_715', 'NDB_977']\n",
      "994 -> ['NDB_680', 'NDB_899', 'NDB_929']\n",
      "995 -> ['NDB_758', 'NDB_788', 'NDB_958']\n",
      "996 -> ['NDB_725', 'NDB_805', 'NDB_752']\n",
      "997 -> ['NDB_965', 'NDB_946', 'NDB_665']\n",
      "999 -> ['NDB_961', 'NDB_838', 'NDB_940']\n",
      "1000 -> ['NDB_784', 'NDB_883', 'NDB_925']\n",
      "1001 -> ['NDB_634', 'NDB_631', 'NDB_872']\n",
      "1002 -> ['NDB_755', 'NDB_698', 'NDB_1033']\n",
      "1003 -> ['NDB_817', 'NDB_746', 'NDB_742']\n",
      "1004 -> ['NDB_683', 'NDB_878', 'NDB_896']\n",
      "1005 -> ['NDB_661', 'NDB_887', 'NDB_818']\n",
      "1020 -> ['NDB_971', 'NDB_754', 'NDB_715']\n",
      "1023 -> ['NDB_868', 'NDB_832', 'NDB_929']\n",
      "1024 -> ['NDB_1033', 'NDB_1075', 'NDB_872']\n",
      "1025 -> ['NDB_743', 'NDB_641', 'NDB_958']\n",
      "1029 -> ['NDB_680', 'NDB_896', 'NDB_818']\n",
      "1030 -> ['NDB_713', 'NDB_815', 'NDB_881']\n",
      "1031 -> ['NDB_730', 'NDB_694', 'NDB_824']\n",
      "1032 -> ['NDB_784', 'NDB_698', 'NDB_631']\n",
      "1033 -> ['NDB_880', 'NDB_752', 'NDB_788']\n",
      "1034 -> ['NDB_875', 'NDB_653', 'NDB_941']\n",
      "1035 -> ['NDB_974', 'NDB_853', 'NDB_934']\n",
      "1036 -> ['NDB_917', 'NDB_838', 'NDB_1066']\n",
      "1037 -> ['NDB_683', 'NDB_716', 'NDB_899']\n",
      "1038 -> ['NDB_835', 'NDB_712', 'NDB_926']\n",
      "1039 -> ['NDB_878', 'NDB_721', 'NDB_910']\n",
      "1040 -> ['NDB_773', 'NDB_634', 'NDB_742']\n",
      "1041 -> ['NDB_737', 'NDB_706', 'NDB_977']\n",
      "1042 -> ['NDB_790', 'NDB_1033', 'NDB_989']\n",
      "1043 -> ['NDB_901', 'NDB_860', 'NDB_935']\n",
      "1044 -> ['NDB_817', 'NDB_710', 'NDB_778']\n",
      "1045 -> ['NDB_994', 'NDB_637', 'NDB_958']\n",
      "1046 -> ['NDB_826', 'NDB_754', 'NDB_647']\n",
      "1047 -> ['NDB_865', 'NDB_643', 'NDB_905']\n",
      "1048 -> ['NDB_965', 'NDB_698', 'NDB_751']\n",
      "1049 -> ['NDB_880', 'NDB_941', 'NDB_665']\n",
      "1050 -> ['NDB_917', 'NDB_779', 'NDB_934']\n",
      "1051 -> ['NDB_1075', 'NDB_899', 'NDB_1028']\n",
      "1052 -> ['NDB_652', 'NDB_661', 'NDB_836']\n",
      "1053 -> ['NDB_844', 'NDB_716', 'NDB_872']\n",
      "1054 -> ['NDB_725', 'NDB_746', 'NDB_863']\n",
      "1055 -> ['NDB_755', 'NDB_743', 'NDB_778']\n",
      "1056 -> ['NDB_758', 'NDB_653', 'NDB_641']\n",
      "1057 -> ['NDB_887', 'NDB_889', 'NDB_862']\n",
      "1058 -> ['NDB_631', 'NDB_905', 'NDB_940']\n",
      "1059 -> ['NDB_812', 'NDB_883', 'NDB_734']\n",
      "1060 -> ['NDB_1027', 'NDB_1060', 'NDB_866']\n",
      "1061 -> ['NDB_760', 'NDB_805', 'NDB_946']\n",
      "1062 -> ['NDB_847', 'NDB_728', 'NDB_925']\n",
      "1063 -> ['NDB_718', 'NDB_994', 'NDB_935']\n",
      "1064 -> ['NDB_790', 'NDB_856', 'NDB_926']\n",
      "1065 -> ['NDB_961', 'NDB_721', 'NDB_929']\n",
      "1066 -> ['NDB_755', 'NDB_1028', 'NDB_818']\n",
      "1067 -> ['NDB_896', 'NDB_665', 'NDB_779']\n",
      "1068 -> ['NDB_868', 'NDB_881', 'NDB_742']\n",
      "1069 -> ['NDB_715', 'NDB_989', 'NDB_706']\n",
      "1070 -> ['NDB_844', 'NDB_824', 'NDB_940']\n",
      "1071 -> ['NDB_971', 'NDB_832', 'NDB_694']\n",
      "1072 -> ['NDB_815', 'NDB_758', 'NDB_637']\n",
      "1073 -> ['NDB_974', 'NDB_1066', 'NDB_788']\n",
      "1074 -> ['NDB_725', 'NDB_838', 'NDB_742']\n",
      "1075 -> ['NDB_1027', 'NDB_863', 'NDB_653']\n",
      "1076 -> ['NDB_643', 'NDB_779', 'NDB_940']\n",
      "1077 -> ['NDB_760', 'NDB_853', 'NDB_818']\n",
      "1078 -> ['NDB_844', 'NDB_634', 'NDB_896']\n",
      "1079 -> ['NDB_784', 'NDB_889', 'NDB_934']\n",
      "1080 -> ['NDB_847', 'NDB_661', 'NDB_1028']\n",
      "1081 -> ['NDB_683', 'NDB_754', 'NDB_866']\n",
      "1082 -> ['NDB_965', 'NDB_901', 'NDB_758']\n",
      "1083 -> ['NDB_718', 'NDB_805', 'NDB_832']\n",
      "1085 -> ['NDB_755', 'NDB_865', 'NDB_872']\n",
      "1087 -> ['NDB_875', 'NDB_698', 'NDB_742']\n",
      "1089 -> ['NDB_778', 'NDB_961', 'NDB_812']\n",
      "1090 -> ['NDB_746', 'NDB_1075', 'NDB_926']\n",
      "1091 -> ['NDB_773', 'NDB_881', 'NDB_958']\n",
      "1092 -> ['NDB_847', 'NDB_860', 'NDB_712']\n",
      "1093 -> ['NDB_887', 'NDB_880', 'NDB_706']\n",
      "1094 -> ['NDB_917', 'NDB_856', 'NDB_835']\n",
      "1097 -> ['NDB_737', 'NDB_838', 'NDB_779']\n",
      "1105 -> ['NDB_743', 'NDB_946', 'NDB_899']\n",
      "1106 -> ['NDB_716', 'NDB_863', 'NDB_994']\n",
      "1107 -> ['NDB_683', 'NDB_760', 'NDB_868']\n",
      "1108 -> ['NDB_815', 'NDB_680', 'NDB_1033']\n",
      "1109 -> ['NDB_698', 'NDB_941', 'NDB_925']\n",
      "1110 -> ['NDB_641', 'NDB_929', 'NDB_910']\n",
      "1111 -> ['NDB_713', 'NDB_1027', 'NDB_758']\n",
      "1112 -> ['NDB_790', 'NDB_860', 'NDB_730']\n",
      "1113 -> ['NDB_832', 'NDB_788', 'NDB_818']\n",
      "1114 -> ['NDB_905', 'NDB_1066', 'NDB_935']\n",
      "1115 -> ['NDB_971', 'NDB_989', 'NDB_751']\n",
      "1116 -> ['NDB_652', 'NDB_883', 'NDB_878']\n",
      "1117 -> ['NDB_817', 'NDB_1060', 'NDB_665']\n",
      "1118 -> ['NDB_710', 'NDB_734', 'NDB_836']\n",
      "1119 -> ['NDB_728', 'NDB_647', 'NDB_637']\n",
      "1120 -> ['NDB_974', 'NDB_631', 'NDB_694']\n",
      "1121 -> ['NDB_815', 'NDB_715', 'NDB_824']\n",
      "1122 -> ['NDB_826', 'NDB_746', 'NDB_868']\n",
      "1123 -> ['NDB_812', 'NDB_737', 'NDB_752']\n",
      "1124 -> ['NDB_875', 'NDB_946', 'NDB_862']\n",
      "1125 -> ['NDB_784', 'NDB_716', 'NDB_721']\n",
      "1126 -> ['NDB_805', 'NDB_866', 'NDB_977']\n",
      "1127 -> ['NDB_713', 'NDB_706', 'NDB_958']\n",
      "1128 -> ['NDB_634', 'NDB_788', 'NDB_712']\n",
      "1129 -> ['NDB_844', 'NDB_835', 'NDB_994']\n",
      "1130 -> ['NDB_725', 'NDB_653', 'NDB_910']\n",
      "1131 -> ['NDB_1028', 'NDB_940', 'NDB_872']\n",
      "1132 -> ['NDB_755', 'NDB_643', 'NDB_901']\n",
      "1133 -> ['NDB_853', 'NDB_935', 'NDB_889']\n",
      "1134 -> ['NDB_721', 'NDB_706', 'NDB_712']\n",
      "1135 -> ['NDB_905', 'NDB_880', 'NDB_881']\n",
      "1136 -> ['NDB_755', 'NDB_652', 'NDB_847']\n",
      "1137 -> ['NDB_728', 'NDB_899', 'NDB_818']\n",
      "1138 -> ['NDB_826', 'NDB_817', 'NDB_805']\n",
      "1139 -> ['NDB_875', 'NDB_868', 'NDB_788']\n",
      "1140 -> ['NDB_773', 'NDB_737', 'NDB_887']\n",
      "1141 -> ['NDB_715', 'NDB_838', 'NDB_694']\n",
      "1142 -> ['NDB_965', 'NDB_883', 'NDB_742']\n",
      "1143 -> ['NDB_698', 'NDB_637', 'NDB_934']\n",
      "1144 -> ['NDB_856', 'NDB_752', 'NDB_910']\n",
      "1145 -> ['NDB_680', 'NDB_946', 'NDB_881']\n",
      "1146 -> ['NDB_901', 'NDB_1075', 'NDB_751']\n",
      "1147 -> ['NDB_812', 'NDB_716', 'NDB_665']\n",
      "1148 -> ['NDB_653', 'NDB_836', 'NDB_958']\n",
      "1152 -> ['NDB_917', 'NDB_778', 'NDB_863']\n",
      "1153 -> ['NDB_865', 'NDB_989', 'NDB_721']\n",
      "1154 -> ['NDB_710', 'NDB_878', 'NDB_889']\n",
      "1155 -> ['NDB_718', 'NDB_961', 'NDB_872']\n",
      "1156 -> ['NDB_844', 'NDB_641', 'NDB_853']\n",
      "1157 -> ['NDB_647', 'NDB_1066', 'NDB_706']\n",
      "1160 -> ['NDB_683', 'NDB_1027', 'NDB_835']\n",
      "1162 -> ['NDB_698', 'NDB_977', 'NDB_824']\n",
      "1163 -> ['NDB_847', 'NDB_887', 'NDB_994']\n",
      "1165 -> ['NDB_868', 'NDB_926', 'NDB_910']\n",
      "1166 -> ['NDB_880', 'NDB_832', 'NDB_1028']\n",
      "1167 -> ['NDB_860', 'NDB_1033', 'NDB_925']\n",
      "1168 -> ['NDB_784', 'NDB_758', 'NDB_935']\n",
      "1169 -> ['NDB_971', 'NDB_896', 'NDB_941']\n",
      "1170 -> ['NDB_790', 'NDB_746', 'NDB_631']\n",
      "1171 -> ['NDB_743', 'NDB_866', 'NDB_862']\n",
      "1172 -> ['NDB_661', 'NDB_1060', 'NDB_730']\n",
      "1173 -> ['NDB_683', 'NDB_694', 'NDB_940']\n",
      "1174 -> ['NDB_725', 'NDB_634', 'NDB_734']\n",
      "1176 -> ['NDB_815', 'NDB_779', 'NDB_929']\n",
      "1177 -> ['NDB_901', 'NDB_754', 'NDB_977']\n",
      "1178 -> ['NDB_713', 'NDB_863', 'NDB_905']\n",
      "1179 -> ['NDB_974', 'NDB_665', 'NDB_824']\n",
      "1180 -> ['NDB_643', 'NDB_965', 'NDB_926']\n",
      "1181 -> ['NDB_760', 'NDB_883', 'NDB_752']\n",
      "1182 -> ['NDB_826', 'NDB_958', 'NDB_940']\n",
      "1183 -> ['NDB_899', 'NDB_994', 'NDB_712']\n",
      "1184 -> ['NDB_755', 'NDB_641', 'NDB_637']\n",
      "1186 -> ['NDB_652', 'NDB_773', 'NDB_818']\n",
      "1187 -> ['NDB_715', 'NDB_788', 'NDB_872']\n",
      "1188 -> ['NDB_680', 'NDB_878', 'NDB_805']\n",
      "1189 -> ['NDB_790', 'NDB_961', 'NDB_737']\n",
      "1190 -> ['NDB_718', 'NDB_728', 'NDB_734']\n",
      "1191 -> ['NDB_752', 'NDB_935', 'NDB_751']\n",
      "1192 -> ['NDB_865', 'NDB_941', 'NDB_836']\n",
      "1193 -> ['NDB_758', 'NDB_1075', 'NDB_721']\n",
      "1194 -> ['NDB_1060', 'NDB_989', 'NDB_824']\n",
      "1195 -> ['NDB_817', 'NDB_844', 'NDB_838']\n",
      "1196 -> ['NDB_847', 'NDB_760', 'NDB_1033']\n",
      "1197 -> ['NDB_778', 'NDB_634', 'NDB_853']\n",
      "1203 -> ['NDB_905', 'NDB_862', 'NDB_694']\n",
      "1204 -> ['NDB_661', 'NDB_716', 'NDB_880']\n",
      "1205 -> ['NDB_653', 'NDB_889', 'NDB_926']\n",
      "1206 -> ['NDB_710', 'NDB_815', 'NDB_994']\n",
      "1207 -> ['NDB_755', 'NDB_917', 'NDB_866']\n",
      "1208 -> ['NDB_812', 'NDB_878', 'NDB_881']\n",
      "1209 -> ['NDB_817', 'NDB_743', 'NDB_860']\n",
      "1210 -> ['NDB_826', 'NDB_725', 'NDB_652']\n",
      "1211 -> ['NDB_683', 'NDB_680', 'NDB_734']\n",
      "1212 -> ['NDB_746', 'NDB_875', 'NDB_1060']\n",
      "1213 -> ['NDB_961', 'NDB_883', 'NDB_946']\n",
      "1214 -> ['NDB_971', 'NDB_868', 'NDB_863']\n",
      "1215 -> ['NDB_901', 'NDB_896', 'NDB_910']\n",
      "1216 -> ['NDB_641', 'NDB_647', 'NDB_751']\n",
      "1217 -> ['NDB_974', 'NDB_706', 'NDB_742']\n",
      "1218 -> ['NDB_860', 'NDB_805', 'NDB_1028']\n",
      "1219 -> ['NDB_812', 'NDB_754', 'NDB_788']\n",
      "1220 -> ['NDB_865', 'NDB_965', 'NDB_653']\n",
      "1221 -> ['NDB_758', 'NDB_862', 'NDB_925']\n",
      "1222 -> ['NDB_844', 'NDB_694', 'NDB_929']\n",
      "1224 -> ['NDB_728', 'NDB_835', 'NDB_977']\n",
      "1225 -> ['NDB_790', 'NDB_773', 'NDB_665']\n",
      "1226 -> ['NDB_698', 'NDB_779', 'NDB_836']\n",
      "1227 -> ['NDB_746', 'NDB_1027', 'NDB_940']\n",
      "1228 -> ['NDB_974', 'NDB_1033', 'NDB_730']\n",
      "1229 -> ['NDB_713', 'NDB_856', 'NDB_637']\n",
      "1230 -> ['NDB_715', 'NDB_958', 'NDB_818']\n",
      "1231 -> ['NDB_887', 'NDB_899', 'NDB_872']\n",
      "1232 -> ['NDB_826', 'NDB_643', 'NDB_1066']\n",
      "1233 -> ['NDB_710', 'NDB_743', 'NDB_712']\n",
      "1234 -> ['NDB_917', 'NDB_737', 'NDB_832']\n",
      "1235 -> ['NDB_661', 'NDB_631', 'NDB_935']\n",
      "1236 -> ['NDB_778', 'NDB_875', 'NDB_934']\n",
      "1237 -> ['NDB_784', 'NDB_971', 'NDB_965']\n",
      "1238 -> ['NDB_1075', 'NDB_730', 'NDB_752']\n",
      "1239 -> ['NDB_718', 'NDB_989', 'NDB_941']\n",
      "1240 -> ['NDB_634', 'NDB_929', 'NDB_925']\n",
      "1241 -> ['NDB_760', 'NDB_896', 'NDB_1066']\n",
      "1242 -> ['NDB_812', 'NDB_1027', 'NDB_925']\n",
      "1243 -> ['NDB_743', 'NDB_746', 'NDB_883']\n",
      "1244 -> ['NDB_784', 'NDB_901', 'NDB_994']\n",
      "1245 -> ['NDB_634', 'NDB_1075', 'NDB_665']\n",
      "1246 -> ['NDB_856', 'NDB_737', 'NDB_862']\n",
      "1247 -> ['NDB_652', 'NDB_715', 'NDB_752']\n",
      "1248 -> ['NDB_860', 'NDB_1060', 'NDB_946']\n",
      "1249 -> ['NDB_847', 'NDB_721', 'NDB_977']\n",
      "1250 -> ['NDB_865', 'NDB_710', 'NDB_887']\n",
      "1251 -> ['NDB_817', 'NDB_751', 'NDB_926']\n",
      "1252 -> ['NDB_725', 'NDB_643', 'NDB_631']\n",
      "1253 -> ['NDB_743', 'NDB_680', 'NDB_698']\n",
      "1254 -> ['NDB_878', 'NDB_665', 'NDB_934']\n",
      "1255 -> ['NDB_826', 'NDB_866', 'NDB_910']\n",
      "1256 -> ['NDB_778', 'NDB_889', 'NDB_836']\n",
      "1257 -> ['NDB_961', 'NDB_863', 'NDB_742']\n",
      "1258 -> ['NDB_875', 'NDB_812', 'NDB_1033']\n",
      "1259 -> ['NDB_784', 'NDB_713', 'NDB_844']\n",
      "1260 -> ['NDB_773', 'NDB_896', 'NDB_835']\n",
      "1261 -> ['NDB_661', 'NDB_634', 'NDB_706']\n",
      "1262 -> ['NDB_974', 'NDB_637', 'NDB_935']\n",
      "1263 -> ['NDB_710', 'NDB_1060', 'NDB_647']\n",
      "1264 -> ['NDB_965', 'NDB_754', 'NDB_805']\n",
      "1265 -> ['NDB_683', 'NDB_994', 'NDB_881']\n",
      "1269 -> ['NDB_865', 'NDB_946', 'NDB_1066']\n",
      "1270 -> ['NDB_728', 'NDB_853', 'NDB_929']\n",
      "1271 -> ['NDB_760', 'NDB_788', 'NDB_694']\n",
      "1272 -> ['NDB_958', 'NDB_1028', 'NDB_712']\n",
      "1273 -> ['NDB_734', 'NDB_818', 'NDB_940']\n",
      "1274 -> ['NDB_815', 'NDB_901', 'NDB_653']\n",
      "1275 -> ['NDB_868', 'NDB_883', 'NDB_641']\n",
      "1276 -> ['NDB_971', 'NDB_856', 'NDB_899']\n",
      "1277 -> ['NDB_905', 'NDB_989', 'NDB_925']\n",
      "1278 -> ['NDB_716', 'NDB_862', 'NDB_779']\n",
      "1279 -> ['NDB_730', 'NDB_832', 'NDB_941']\n",
      "1280 -> ['NDB_790', 'NDB_838', 'NDB_647']\n",
      "1281 -> ['NDB_917', 'NDB_1027', 'NDB_880']\n",
      "1282 -> ['NDB_755', 'NDB_1075', 'NDB_824']\n",
      "1284 -> ['NDB_715', 'NDB_758', 'NDB_853']\n",
      "1285 -> ['NDB_1066', 'NDB_712', 'NDB_872']\n",
      "1286 -> ['NDB_718', 'NDB_760', 'NDB_746']\n",
      "1287 -> ['NDB_661', 'NDB_881', 'NDB_934']\n",
      "1288 -> ['NDB_652', 'NDB_961', 'NDB_698']\n",
      "1289 -> ['NDB_805', 'NDB_637', 'NDB_910']\n",
      "1290 -> ['NDB_826', 'NDB_847', 'NDB_751']\n",
      "1291 -> ['NDB_875', 'NDB_878', 'NDB_887']\n",
      "1292 -> ['NDB_737', 'NDB_863', 'NDB_730']\n",
      "1293 -> ['NDB_754', 'NDB_721', 'NDB_926']\n",
      "1294 -> ['NDB_680', 'NDB_631', 'NDB_977']\n",
      "1295 -> ['NDB_860', 'NDB_734', 'NDB_889']\n",
      "1297 -> ['NDB_725', 'NDB_974', 'NDB_773']\n",
      "1298 -> ['NDB_713', 'NDB_643', 'NDB_778']\n",
      "1299 -> ['NDB_817', 'NDB_752', 'NDB_835']\n",
      "1300 -> ['NDB_826', 'NDB_860', 'NDB_929']\n",
      "1301 -> ['NDB_710', 'NDB_883', 'NDB_862']\n",
      "1302 -> ['NDB_847', 'NDB_634', 'NDB_905']\n",
      "1303 -> ['NDB_746', 'NDB_716', 'NDB_1028']\n",
      "1304 -> ['NDB_965', 'NDB_641', 'NDB_779']\n",
      "1305 -> ['NDB_790', 'NDB_901', 'NDB_752']\n",
      "1306 -> ['NDB_880', 'NDB_946', 'NDB_853']\n",
      "1307 -> ['NDB_974', 'NDB_661', 'NDB_977']\n",
      "1308 -> ['NDB_718', 'NDB_683', 'NDB_910']\n",
      "1309 -> ['NDB_698', 'NDB_653', 'NDB_730']\n",
      "1310 -> ['NDB_773', 'NDB_863', 'NDB_836']\n",
      "1311 -> ['NDB_832', 'NDB_721', 'NDB_935']\n",
      "1312 -> ['NDB_643', 'NDB_887', 'NDB_742']\n",
      "1313 -> ['NDB_844', 'NDB_878', 'NDB_926']\n",
      "1314 -> ['NDB_961', 'NDB_896', 'NDB_994']\n",
      "1315 -> ['NDB_754', 'NDB_862', 'NDB_824']\n",
      "1316 -> ['NDB_631', 'NDB_647', 'NDB_881']\n",
      "1317 -> ['NDB_917', 'NDB_805', 'NDB_941']\n",
      "1318 -> ['NDB_847', 'NDB_790', 'NDB_1060']\n",
      "1319 -> ['NDB_734', 'NDB_706', 'NDB_872']\n",
      "1320 -> ['NDB_752', 'NDB_925', 'NDB_934']\n",
      "1321 -> ['NDB_965', 'NDB_838', 'NDB_788']\n",
      "1322 -> ['NDB_865', 'NDB_746', 'NDB_866']\n",
      "1323 -> ['NDB_710', 'NDB_637', 'NDB_694']\n",
      "1324 -> ['NDB_784', 'NDB_634', 'NDB_818']\n",
      "1325 -> ['NDB_718', 'NDB_751', 'NDB_940']\n",
      "1330 -> ['NDB_743', 'NDB_853', 'NDB_779']\n",
      "1331 -> ['NDB_643', 'NDB_815', 'NDB_935']\n",
      "1332 -> ['NDB_728', 'NDB_856', 'NDB_730']\n",
      "1333 -> ['NDB_817', 'NDB_680', 'NDB_715']\n",
      "1334 -> ['NDB_875', 'NDB_631', 'NDB_899']\n",
      "1335 -> ['NDB_725', 'NDB_758', 'NDB_881']\n",
      "1336 -> ['NDB_755', 'NDB_868', 'NDB_737']\n",
      "1337 -> ['NDB_826', 'NDB_665', 'NDB_712']\n",
      "1338 -> ['NDB_860', 'NDB_989', 'NDB_641']\n",
      "1339 -> ['NDB_778', 'NDB_1033', 'NDB_929']\n",
      "1340 -> ['NDB_1027', 'NDB_883', 'NDB_889']\n",
      "1341 -> ['NDB_713', 'NDB_652', 'NDB_760']\n",
      "1342 -> ['NDB_812', 'NDB_1075', 'NDB_958']\n",
      "1343 -> ['NDB_971', 'NDB_835', 'NDB_1066']\n",
      "1344 -> ['NDB_901', 'NDB_868', 'NDB_1028']\n",
      "1345 -> ['NDB_817', 'NDB_716', 'NDB_905']\n",
      "1346 -> ['NDB_815', 'NDB_788', 'NDB_940']\n",
      "1347 -> ['NDB_838', 'NDB_751', 'NDB_818']\n",
      "1348 -> ['NDB_743', 'NDB_694', 'NDB_925']\n",
      "1349 -> ['NDB_784', 'NDB_941', 'NDB_706']\n",
      "1350 -> ['NDB_961', 'NDB_878', 'NDB_637']\n",
      "1351 -> ['NDB_917', 'NDB_653', 'NDB_994']\n",
      "1352 -> ['NDB_844', 'NDB_734', 'NDB_934']\n",
      "1353 -> ['NDB_896', 'NDB_824', 'NDB_872']\n",
      "1354 -> ['NDB_1060', 'NDB_805', 'NDB_926']\n",
      "1355 -> ['NDB_865', 'NDB_683', 'NDB_863']\n",
      "1356 -> ['NDB_778', 'NDB_773', 'NDB_910']\n",
      "1357 -> ['NDB_661', 'NDB_866', 'NDB_647']\n",
      "1358 -> ['NDB_974', 'NDB_698', 'NDB_946']\n",
      "1359 -> ['NDB_760', 'NDB_899', 'NDB_977']\n",
      "1360 -> ['NDB_832', 'NDB_742', 'NDB_712']\n",
      "1361 -> ['NDB_755', 'NDB_856', 'NDB_721']\n",
      "1362 -> ['NDB_680', 'NDB_887', 'NDB_836']\n",
      "1363 -> ['NDB_754', 'NDB_758', 'NDB_880']\n",
      "1378 -> ['NDB_1066', 'NDB_751', 'NDB_934']\n",
      "1379 -> ['NDB_883', 'NDB_1033', 'NDB_824']\n",
      "1380 -> ['NDB_826', 'NDB_1075', 'NDB_925']\n",
      "1381 -> ['NDB_728', 'NDB_758', 'NDB_905']\n",
      "1382 -> ['NDB_760', 'NDB_878', 'NDB_994']\n",
      "1383 -> ['NDB_917', 'NDB_860', 'NDB_887']\n",
      "1384 -> ['NDB_935', 'NDB_940', 'NDB_926']\n",
      "1385 -> ['NDB_844', 'NDB_788', 'NDB_977']\n",
      "1386 -> ['NDB_746', 'NDB_752', 'NDB_641']\n",
      "1387 -> ['NDB_971', 'NDB_974', 'NDB_881']\n",
      "1388 -> ['NDB_715', 'NDB_880', 'NDB_929']\n",
      "1389 -> ['NDB_847', 'NDB_773', 'NDB_899']\n",
      "1390 -> ['NDB_989', 'NDB_818', 'NDB_872']\n",
      "1391 -> ['NDB_683', 'NDB_721', 'NDB_941']\n",
      "1392 -> ['NDB_875', 'NDB_901', 'NDB_634']\n",
      "1393 -> ['NDB_755', 'NDB_896', 'NDB_694']\n",
      "1394 -> ['NDB_784', 'NDB_790', 'NDB_958']\n",
      "1395 -> ['NDB_961', 'NDB_653', 'NDB_665']\n",
      "1396 -> ['NDB_710', 'NDB_805', 'NDB_779']\n",
      "1398 -> ['NDB_737', 'NDB_836', 'NDB_1028']\n",
      "1399 -> ['NDB_868', 'NDB_866', 'NDB_637']\n",
      "1400 -> ['NDB_1027', 'NDB_730', 'NDB_706']\n",
      "1401 -> ['NDB_718', 'NDB_643', 'NDB_862']\n",
      "1402 -> ['NDB_725', 'NDB_778', 'NDB_832']\n",
      "1403 -> ['NDB_865', 'NDB_661', 'NDB_856']\n",
      "1404 -> ['NDB_815', 'NDB_863', 'NDB_838']\n",
      "1405 -> ['NDB_713', 'NDB_754', 'NDB_835']\n",
      "1406 -> ['NDB_946', 'NDB_647', 'NDB_889']\n",
      "1408 -> ['NDB_631', 'NDB_853', 'NDB_742']\n",
      "1409 -> ['NDB_716', 'NDB_1066', 'NDB_734']\n",
      "1427 -> ['NDB_698', 'NDB_1060', 'NDB_721']\n",
      "1429 -> ['NDB_643', 'NDB_680', 'NDB_824']\n",
      "1430 -> ['NDB_725', 'NDB_754', 'NDB_856']\n",
      "1434 -> ['NDB_773', 'NDB_715', 'NDB_941']\n",
      "1435 -> ['NDB_737', 'NDB_653', 'NDB_896']\n",
      "1436 -> ['NDB_746', 'NDB_1033', 'NDB_836']\n",
      "1437 -> ['NDB_826', 'NDB_971', 'NDB_661']\n",
      "1438 -> ['NDB_905', 'NDB_641', 'NDB_994']\n",
      "1439 -> ['NDB_790', 'NDB_805', 'NDB_940']\n",
      "1440 -> ['NDB_868', 'NDB_878', 'NDB_1075']\n",
      "1441 -> ['NDB_755', 'NDB_634', 'NDB_751']\n",
      "1442 -> ['NDB_683', 'NDB_989', 'NDB_926']\n",
      "1443 -> ['NDB_784', 'NDB_881', 'NDB_977']\n",
      "1444 -> ['NDB_652', 'NDB_665', 'NDB_925']\n",
      "1445 -> ['NDB_961', 'NDB_716', 'NDB_832']\n",
      "1446 -> ['NDB_844', 'NDB_631', 'NDB_887']\n",
      "1447 -> ['NDB_718', 'NDB_863', 'NDB_712']\n",
      "1448 -> ['NDB_743', 'NDB_901', 'NDB_730']\n",
      "1449 -> ['NDB_865', 'NDB_889', 'NDB_958']\n",
      "1450 -> ['NDB_1027', 'NDB_946', 'NDB_637']\n",
      "1451 -> ['NDB_815', 'NDB_917', 'NDB_742']\n",
      "1452 -> ['NDB_713', 'NDB_965', 'NDB_974']\n",
      "1453 -> ['NDB_866', 'NDB_838', 'NDB_935']\n",
      "1454 -> ['NDB_817', 'NDB_1028', 'NDB_934']\n",
      "1455 -> ['NDB_812', 'NDB_835', 'NDB_779']\n",
      "1456 -> ['NDB_853', 'NDB_647', 'NDB_910']\n",
      "1457 -> ['NDB_698', 'NDB_880', 'NDB_862']\n",
      "1458 -> ['NDB_643', 'NDB_752', 'NDB_818']\n",
      "1459 -> ['NDB_875', 'NDB_856', 'NDB_1066']\n",
      "1461 -> ['NDB_961', 'NDB_905', 'NDB_836']\n",
      "1462 -> ['NDB_865', 'NDB_974', 'NDB_901']\n",
      "1463 -> ['NDB_746', 'NDB_773', 'NDB_694']\n",
      "1464 -> ['NDB_856', 'NDB_838', 'NDB_872']\n",
      "1465 -> ['NDB_755', 'NDB_730', 'NDB_712']\n",
      "1466 -> ['NDB_971', 'NDB_824', 'NDB_934']\n",
      "1467 -> ['NDB_743', 'NDB_758', 'NDB_896']\n",
      "1468 -> ['NDB_710', 'NDB_790', 'NDB_929']\n",
      "1469 -> ['NDB_815', 'NDB_805', 'NDB_721']\n",
      "1470 -> ['NDB_1027', 'NDB_641', 'NDB_665']\n",
      "1471 -> ['NDB_652', 'NDB_634', 'NDB_1028']\n",
      "1472 -> ['NDB_661', 'NDB_1075', 'NDB_1066']\n",
      "1473 -> ['NDB_847', 'NDB_778', 'NDB_974']\n",
      "1474 -> ['NDB_875', 'NDB_866', 'NDB_818']\n",
      "1475 -> ['NDB_812', 'NDB_853', 'NDB_940']\n",
      "1476 -> ['NDB_683', 'NDB_1033', 'NDB_637']\n",
      "1477 -> ['NDB_754', 'NDB_878', 'NDB_751']\n",
      "1478 -> ['NDB_817', 'NDB_737', 'NDB_631']\n",
      "1479 -> ['NDB_752', 'NDB_734', 'NDB_958']\n",
      "1480 -> ['NDB_835', 'NDB_706', 'NDB_925']\n",
      "1481 -> ['NDB_746', 'NDB_862', 'NDB_977']\n",
      "1482 -> ['NDB_863', 'NDB_880', 'NDB_989']\n",
      "1485 -> ['NDB_643', 'NDB_715', 'NDB_698']\n",
      "1486 -> ['NDB_784', 'NDB_832', 'NDB_946']\n",
      "1487 -> ['NDB_710', 'NDB_728', 'NDB_634']\n",
      "1488 -> ['NDB_760', 'NDB_917', 'NDB_716']\n",
      "1489 -> ['NDB_865', 'NDB_883', 'NDB_881']\n",
      "1490 -> ['NDB_1033', 'NDB_805', 'NDB_742']\n",
      "1491 -> ['NDB_826', 'NDB_994', 'NDB_779']\n",
      "1492 -> ['NDB_844', 'NDB_1060', 'NDB_899']\n",
      "1493 -> ['NDB_725', 'NDB_965', 'NDB_905']\n",
      "1494 -> ['NDB_718', 'NDB_838', 'NDB_926']\n",
      "1495 -> ['NDB_887', 'NDB_925', 'NDB_910']\n",
      "1496 -> ['NDB_778', 'NDB_653', 'NDB_788']\n",
      "1497 -> ['NDB_713', 'NDB_680', 'NDB_935']\n",
      "1498 -> ['NDB_868', 'NDB_941', 'NDB_889']\n",
      "1499 -> ['NDB_716', 'NDB_856', 'NDB_647']\n",
      "1500 -> ['NDB_643', 'NDB_860', 'NDB_758']\n",
      "1501 -> ['NDB_961', 'NDB_989', 'NDB_694']\n",
      "1502 -> ['NDB_728', 'NDB_715', 'NDB_742']\n",
      "1503 -> ['NDB_901', 'NDB_862', 'NDB_836']\n",
      "1504 -> ['NDB_1027', 'NDB_751', 'NDB_1028']\n",
      "1505 -> ['NDB_784', 'NDB_917', 'NDB_712']\n",
      "1506 -> ['NDB_847', 'NDB_844', 'NDB_958']\n",
      "1507 -> ['NDB_815', 'NDB_812', 'NDB_896']\n",
      "1508 -> ['NDB_754', 'NDB_665', 'NDB_889']\n",
      "1509 -> ['NDB_725', 'NDB_872', 'NDB_934']\n",
      "1510 -> ['NDB_790', 'NDB_1075', 'NDB_881']\n",
      "1511 -> ['NDB_652', 'NDB_994', 'NDB_734']\n",
      "1512 -> ['NDB_661', 'NDB_773', 'NDB_637']\n",
      "1513 -> ['NDB_875', 'NDB_721', 'NDB_853']\n",
      "1514 -> ['NDB_713', 'NDB_878', 'NDB_730']\n",
      "1515 -> ['NDB_866', 'NDB_835', 'NDB_940']\n",
      "1516 -> ['NDB_631', 'NDB_752', 'NDB_929']\n",
      "1517 -> ['NDB_971', 'NDB_737', 'NDB_641']\n",
      "1518 -> ['NDB_743', 'NDB_683', 'NDB_1066']\n",
      "1519 -> ['NDB_755', 'NDB_817', 'NDB_832']\n",
      "1520 -> ['NDB_863', 'NDB_706', 'NDB_818']\n",
      "1521 -> ['NDB_826', 'NDB_899', 'NDB_824']\n",
      "1525 -> ['NDB_661', 'NDB_838', 'NDB_889']\n",
      "1526 -> ['NDB_815', 'NDB_746', 'NDB_734']\n",
      "1527 -> ['NDB_965', 'NDB_860', 'NDB_862']\n",
      "1529 -> ['NDB_1027', 'NDB_989', 'NDB_853']\n",
      "1530 -> ['NDB_760', 'NDB_680', 'NDB_634']\n",
      "1533 -> ['NDB_779', 'NDB_751', 'NDB_910']\n",
      "1534 -> ['NDB_718', 'NDB_883', 'NDB_929']\n",
      "1535 -> ['NDB_725', 'NDB_728', 'NDB_706']\n",
      "1536 -> ['NDB_743', 'NDB_754', 'NDB_631']\n",
      "1537 -> ['NDB_778', 'NDB_758', 'NDB_1028']\n",
      "1538 -> ['NDB_832', 'NDB_835', 'NDB_872']\n",
      "1541 -> ['NDB_713', 'NDB_865', 'NDB_647']\n",
      "1542 -> ['NDB_817', 'NDB_652', 'NDB_866']\n",
      "1548 -> ['NDB_812', 'NDB_730', 'NDB_742']\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "check_readout_dict = pickle.load(open(os.path.join(library_folder, 'readout_usage.pkl'), 'rb'))\n",
    "\n",
    "reg_readout_dic, reg2readout_check = check._check_region_to_readouts(full_records, check_readout_dict, \n",
    "                                                                     total_readout_num=num_on_bits)\n",
    "print(reg2readout_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3e64afd0",
   "metadata": {},
   "source": [
    "### 4.1.4 check readout usage in the library"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "id": "1647e94f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- extracting probes from >region:0, region:1, region:2, region:3, region:4, region:5, region:6, region:7, region:8, region:9, region:10, region:11, region:12, region:13, region:14, region:15, region:16, region:17, region:18, region:19, region:20, region:21, region:22, region:23, region:24, region:25, region:26, region:27, region:28, region:29, region:30, region:31, region:32, region:33, region:34, region:35, region:36, region:37, region:38, region:39, region:40, region:42, region:43, region:44, region:45, region:46, region:47, region:57, region:60, region:62, region:63, region:64, region:65, region:66, region:67, region:68, region:69, region:70, region:71, region:72, region:73, region:74, region:75, region:76, region:77, region:78, region:79, region:80, region:81, region:83, region:84, region:85, region:86, region:87, region:88, region:89, region:90, region:91, region:92, region:93, region:94, region:95, region:96, region:98, region:100, region:101, region:102, region:103, region:104, region:105, region:106, region:107, region:108, region:109, region:110, region:111, region:112, region:114, region:115, region:116, region:117, region:118, region:119, region:120, region:121, region:122, region:123, region:124, region:125, region:126, region:127, region:128, region:129, region:130, region:131, region:132, region:133, region:134, region:137, region:138, region:139, region:140, region:141, region:143, region:144, region:145, region:146, region:147, region:148, region:149, region:150, region:152, region:153, region:154, region:155, region:156, region:157, region:158, region:159, region:160, region:161, region:162, region:163, region:164, region:165, region:166, region:167, region:168, region:169, region:170, region:171, region:172, region:173, region:174, region:175, region:176, region:177, region:178, region:179, region:180, region:181, region:182, region:183, region:184, region:185, region:186, region:187, region:188, region:189, region:190, region:191, region:192, region:193, region:194, region:195, region:196, region:197, region:198, region:199, region:200, region:201, region:202, region:203, region:204, region:205, region:206, region:207, region:208, region:209, region:210, region:211, region:212, region:213, region:214, region:215, region:216, region:217, region:218, region:219, region:220, region:221, region:222, region:223, region:224, region:225, region:226, region:227, region:228, region:229, region:230, region:233, region:234, region:235, region:236, region:237, region:238, region:239, region:240, region:241, region:242, region:243, region:244, region:245, region:246, region:247, region:248, region:249, region:250, region:251, region:252, region:253, region:254, region:255, region:256, region:257, region:258, region:259, region:260, region:261, region:262, region:263, region:264, region:265, region:266, region:267, region:268, region:269, region:270, region:271, region:272, region:273, region:274, region:275, region:276, region:277, region:278, region:279, region:280, region:281, region:282, region:283, region:284, region:285, region:286, region:287, region:288, region:289, region:290, region:291, region:292, region:293, region:295, region:296, region:297, region:298, region:299, region:300, region:302, region:303, region:304, region:305, region:306, region:307, region:308, region:309, region:310, region:311, region:312, region:313, region:314, region:315, region:316, region:317, region:318, region:319, region:320, region:321, region:322, region:323, region:324, region:325, region:326, region:327, region:328, region:329, region:330, region:331, region:332, region:333, region:334, region:335, region:336, region:337, region:338, region:339, region:340, region:341, region:342, region:343, region:344, region:345, region:346, region:347, region:348, region:349, region:350, region:351, region:352, region:353, region:354, region:355, region:356, region:357, region:358, region:359, region:360, region:361, region:362, region:363, region:364, region:365, region:366, region:367, region:368, region:370, region:371, region:372, region:373, region:374, region:375, region:376, region:377, region:379, region:380, region:381, region:382, region:383, region:384, region:385, region:386, region:387, region:388, region:389, region:390, region:391, region:392, region:393, region:394, region:395, region:396, region:397, region:398, region:399, region:400, region:401, region:402, region:403, region:404, region:405, region:406, region:407, region:408, region:409, region:410, region:411, region:412, region:413, region:414, region:415, region:416, region:417, region:418, region:419, region:421, region:422, region:423, region:424, region:425, region:426, region:427, region:428, region:429, region:430, region:431, region:432, region:435, region:436, region:437, region:438, region:439, region:440, region:441, region:442, region:443, region:444, region:446, region:447, region:448, region:449, region:450, region:451, region:452, region:453, region:454, region:455, region:456, region:457, region:458, region:459, region:460, region:461, region:462, region:463, region:464, region:465, region:466, region:467, region:468, region:469, region:470, region:471, region:472, region:473, region:474, region:475, region:476, region:477, region:478, region:479, region:480, region:481, region:482, region:483, region:484, region:485, region:486, region:487, region:489, region:490, region:491, region:492, region:493, region:494, region:495, region:496, region:497, region:498, region:499, region:500, region:501, region:502, region:503, region:504, region:505, region:506, region:507, region:508, region:509, region:510, region:511, region:512, region:516, region:517, region:518, region:519, region:520, region:521, region:522, region:523, region:524, region:525, region:526, region:527, region:528, region:529, region:530, region:531, region:532, region:533, region:534, region:535, region:536, region:537, region:538, region:539, region:540, region:541, region:542, region:543, region:544, region:545, region:546, region:548, region:549, region:550, region:551, region:552, region:554, region:555, region:556, region:557, region:558, region:559, region:560, region:561, region:562, region:563, region:564, region:565, region:566, region:567, region:568, region:569, region:570, region:571, region:572, region:573, region:574, region:575, region:576, region:577, region:578, region:579, region:580, region:581, region:582, region:583, region:584, region:585, region:586, region:587, region:588, region:589, region:590, region:591, region:592, region:593, region:594, region:595, region:596, region:597, region:598, region:599, region:600, region:601, region:602, region:603, region:605, region:606, region:607, region:608, region:609, region:611, region:612, region:613, region:614, region:615, region:616, region:617, region:618, region:619, region:620, region:621, region:622, region:623, region:624, region:625, region:626, region:627, region:628, region:629, region:645, region:646, region:647, region:648, region:649, region:650, region:651, region:652, region:653, region:654, region:655, region:656, region:657, region:658, region:659, region:660, region:662, region:663, region:664, region:665, region:666, region:667, region:668, region:669, region:670, region:671, region:672, region:673, region:674, region:675, region:676, region:677, region:678, region:679, region:680, region:681, region:682, region:683, region:684, region:685, region:686, region:687, region:688, region:689, region:690, region:691, region:692, region:693, region:694, region:695, region:696, region:697, region:698, region:699, region:701, region:702, region:703, region:704, region:705, region:706, region:707, region:708, region:709, region:710, region:711, region:712, region:713, region:714, region:715, region:716, region:717, region:718, region:719, region:720, region:721, region:722, region:723, region:724, region:725, region:726, region:727, region:728, region:729, region:730, region:731, region:732, region:733, region:734, region:735, region:736, region:737, region:738, region:739, region:740, region:741, region:742, region:743, region:744, region:745, region:748"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ", region:749, region:750, region:751, region:752, region:753, region:754, region:755, region:756, region:757, region:758, region:759, region:760, region:761, region:762, region:763, region:764, region:765, region:766, region:767, region:768, region:769, region:770, region:774, region:775, region:776, region:777, region:778, region:779, region:780, region:782, region:783, region:784, region:785, region:786, region:787, region:788, region:789, region:790, region:791, region:792, region:793, region:794, region:795, region:796, region:797, region:798, region:799, region:800, region:801, region:802, region:803, region:804, region:805, region:806, region:807, region:808, region:809, region:810, region:811, region:812, region:813, region:814, region:815, region:816, region:817, region:819, region:820, region:821, region:822, region:823, region:824, region:825, region:826, region:827, region:828, region:829, region:832, region:833, region:834, region:835, region:836, region:837, region:838, region:839, region:840, region:841, region:843, region:844, region:845, region:846, region:847, region:848, region:849, region:850, region:851, region:852, region:853, region:854, region:855, region:856, region:857, region:858, region:859, region:860, region:861, region:862, region:863, region:864, region:865, region:866, region:867, region:868, region:869, region:870, region:871, region:872, region:873, region:874, region:875, region:876, region:877, region:878, region:888, region:889, region:890, region:891, region:892, region:893, region:894, region:895, region:896, region:897, region:898, region:899, region:900, region:901, region:902, region:903, region:904, region:905, region:906, region:907, region:908, region:909, region:910, region:911, region:912, region:913, region:914, region:915, region:916, region:917, region:918, region:919, region:920, region:921, region:922, region:923, region:924, region:925, region:926, region:927, region:928, region:929, region:930, region:931, region:932, region:933, region:934, region:935, region:949, region:950, region:951, region:952, region:953, region:954, region:955, region:956, region:957, region:958, region:959, region:960, region:961, region:962, region:963, region:964, region:965, region:966, region:967, region:968, region:969, region:970, region:971, region:972, region:973, region:974, region:975, region:976, region:977, region:978, region:979, region:980, region:981, region:982, region:983, region:984, region:985, region:986, region:987, region:988, region:989, region:990, region:991, region:992, region:993, region:994, region:995, region:996, region:997, region:999, region:1000, region:1001, region:1002, region:1003, region:1004, region:1005, region:1020, region:1023, region:1024, region:1025, region:1029, region:1030, region:1031, region:1032, region:1033, region:1034, region:1035, region:1036, region:1037, region:1038, region:1039, region:1040, region:1041, region:1042, region:1043, region:1044, region:1045, region:1046, region:1047, region:1048, region:1049, region:1050, region:1051, region:1052, region:1053, region:1054, region:1055, region:1056, region:1057, region:1058, region:1059, region:1060, region:1061, region:1062, region:1063, region:1064, region:1065, region:1066, region:1067, region:1068, region:1069, region:1070, region:1071, region:1072, region:1073, region:1074, region:1075, region:1076, region:1077, region:1078, region:1079, region:1080, region:1081, region:1082, region:1083, region:1085, region:1087, region:1089, region:1090, region:1091, region:1092, region:1093, region:1094, region:1097, region:1105, region:1106, region:1107, region:1108, region:1109, region:1110, region:1111, region:1112, region:1113, region:1114, region:1115, region:1116, region:1117, region:1118, region:1119, region:1120, region:1121, region:1122, region:1123, region:1124, region:1125, region:1126, region:1127, region:1128, region:1129, region:1130, region:1131, region:1132, region:1133, region:1134, region:1135, region:1136, region:1137, region:1138, region:1139, region:1140, region:1141, region:1142, region:1143, region:1144, region:1145, region:1146, region:1147, region:1148, region:1152, region:1153, region:1154, region:1155, region:1156, region:1157, region:1160, region:1162, region:1163, region:1165, region:1166, region:1167, region:1168, region:1169, region:1170, region:1171, region:1172, region:1173, region:1174, region:1176, region:1177, region:1178, region:1179, region:1180, region:1181, region:1182, region:1183, region:1184, region:1186, region:1187, region:1188, region:1189, region:1190, region:1191, region:1192, region:1193, region:1194, region:1195, region:1196, region:1197, region:1203, region:1204, region:1205, region:1206, region:1207, region:1208, region:1209, region:1210, region:1211, region:1212, region:1213, region:1214, region:1215, region:1216, region:1217, region:1218, region:1219, region:1220, region:1221, region:1222, region:1224, region:1225, region:1226, region:1227, region:1228, region:1229, region:1230, region:1231, region:1232, region:1233, region:1234, region:1235, region:1236, region:1237, region:1238, region:1239, region:1240, region:1241, region:1242, region:1243, region:1244, region:1245, region:1246, region:1247, region:1248, region:1249, region:1250, region:1251, region:1252, region:1253, region:1254, region:1255, region:1256, region:1257, region:1258, region:1259, region:1260, region:1261, region:1262, region:1263, region:1264, region:1265, region:1269, region:1270, region:1271, region:1272, region:1273, region:1274, region:1275, region:1276, region:1277, region:1278, region:1279, region:1280, region:1281, region:1282, region:1284, region:1285, region:1286, region:1287, region:1288, region:1289, region:1290, region:1291, region:1292, region:1293, region:1294, region:1295, region:1297, region:1298, region:1299, region:1300, region:1301, region:1302, region:1303, region:1304, region:1305, region:1306, region:1307, region:1308, region:1309, region:1310, region:1311, region:1312, region:1313, region:1314, region:1315, region:1316, region:1317, region:1318, region:1319, region:1320, region:1321, region:1322, region:1323, region:1324, region:1325, region:1330, region:1331, region:1332, region:1333, region:1334, region:1335, region:1336, region:1337, region:1338, region:1339, region:1340, region:1341, region:1342, region:1343, region:1344, region:1345, region:1346, region:1347, region:1348, region:1349, region:1350, region:1351, region:1352, region:1353, region:1354, region:1355, region:1356, region:1357, region:1358, region:1359, region:1360, region:1361, region:1362, region:1363, region:1378, region:1379, region:1380, region:1381, region:1382, region:1383, region:1384, region:1385, region:1386, region:1387, region:1388, region:1389, region:1390, region:1391, region:1392, region:1393, region:1394, region:1395, region:1396, region:1398, region:1399, region:1400, region:1401, region:1402, region:1403, region:1404, region:1405, region:1406, region:1408, region:1409, region:1427, region:1429, region:1430, region:1434, region:1435, region:1436, region:1437, region:1438, region:1439, region:1440, region:1441, region:1442, region:1443, region:1444, region:1445, region:1446, region:1447, region:1448, region:1449, region:1450, region:1451, region:1452, region:1453, region:1454, region:1455, region:1456, region:1457, region:1458, region:1459, region:1461, region:1462, region:1463, region:1464, region:1465, region:1466, region:1467, region:1468, region:1469, region:1470, region:1471, region:1472, region:1473, region:1474, region:1475, region:1476, region:1477, region:1478, region:1479, region:1480, region:1481, region:1482, region:1485, region:1486, region:1487, region:1488, region:1489, region:1490, region:1491, region:1492, region:1493, region:1494, region:1495, region:1496, region:1497, region:1498, region:1499, region:1500, region:1501, region:1502, region:1503, region:1504, region:1505, region:1506, region:1507, region:1508, region:1509, region:1510, region:1511, region:1512, region:1513, region:1514, region:1515, region:1516, region:1517, region:1518, region:1519, region:1520, region:1521, region:1525, region:1526, region:1527, region:1529, region:1530, region:1533, region:1534, region:1535, region:1536, region:1537, region:1538"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ", region:1541, region:1542, region:1548, {'NDB_683': ['0', '0', '0', '39', '39', '39', '89', '89', '89', '110', '110', '110', '140', '140', '140', '194', '194', '194', '237', '237', '237', '267', '267', '267', '274', '274', '274', '324', '324', '324', '350', '350', '350', '372', '372', '372', '426', '426', '426', '454', '454', '454', '521', '521', '521', '590', '590', '590', '626', '626', '626', '676', '676', '676', '685', '685', '685', '718', '718', '718', '775', '775', '775', '812', '812', '812', '854', '854', '854', '870', '870', '870', '902', '902', '902', '927', '927', '927', '955', '955', '955', '1004', '1004', '1004', '1037', '1037', '1037', '1081', '1081', '1081', '1107', '1107', '1107', '1160', '1160', '1160', '1173', '1173', '1173', '1211', '1211', '1211', '1265', '1265', '1265', '1308', '1308', '1308', '1355', '1355', '1355', '1391', '1391', '1391', '1442', '1442', '1442', '1476', '1476', '1476', '1518', '1518', '1518'], 'NDB_758': ['0', '0', '0', '13', '13', '13', '87', '87', '87', '105', '105', '105', '147', '147', '147', '181', '181', '181', '201', '201', '201', '260', '260', '260', '292', '292', '292', '330', '330', '330', '353', '353', '353', '397', '397', '397', '419', '419', '419', '432', '432', '432', '483', '483', '483', '525', '525', '525', '559', '559', '559', '603', '603', '603', '622', '622', '622', '726', '726', '726', '763', '763', '763', '804', '804', '804', '837', '837', '837', '865', '865', '865', '896', '896', '896', '976', '976', '976', '995', '995', '995', '1056', '1056', '1056', '1072', '1072', '1072', '1082', '1082', '1082', '1111', '1111', '1111', '1168', '1168', '1168', '1193', '1193', '1193', '1221', '1221', '1221', '1284', '1284', '1284', '1335', '1335', '1335', '1363', '1363', '1363', '1381', '1381', '1381', '1467', '1467', '1467', '1500', '1500', '1500', '1537', '1537', '1537'], 'NDB_887': ['0', '0', '0', '16', '16', '16', '62', '62', '62', '126', '126', '126', '164', '164', '164', '200', '200', '200', '217', '217', '217', '277', '277', '277', '291', '291', '291', '314', '314', '314', '366', '366', '366', '411', '411', '411', '427', '427', '427', '457', '457', '457', '530', '530', '530', '548', '548', '548', '573', '573', '573', '608', '608', '608', '624', '624', '624', '716', '716', '716', '740', '740', '740', '748', '748', '748', '776', '776', '776', '827', '827', '827', '876', '876', '876', '898', '898', '898', '934', '934', '934', '990', '990', '990', '1005', '1005', '1005', '1057', '1057', '1057', '1093', '1093', '1093', '1140', '1140', '1140', '1163', '1163', '1163', '1231', '1231', '1231', '1250', '1250', '1250', '1291', '1291', '1291', '1312', '1312', '1312', '1362', '1362', '1362', '1383', '1383', '1383', '1446', '1446', '1446', '1495', '1495', '1495'], 'NDB_718': ['1', '1', '1', '34', '34', '34', '80', '80', '80', '132', '132', '132', '148', '148', '148', '209', '209', '209', '227', '227', '227', '261', '261', '261', '279', '279', '279', '331', '331', '331', '352', '352', '352', '413', '413', '413', '442', '442', '442', '471', '471', '471', '524', '524', '524', '543', '543', '543', '559', '559', '559', '575', '575', '575', '646', '646', '646', '666', '666', '666', '714', '714', '714', '734', '734', '734', '748', '748', '748', '765', '765', '765', '832', '832', '832', '874', '874', '874', '909', '909', '909', '928', '928', '928', '984', '984', '984', '1063', '1063', '1063', '1083', '1083', '1083', '1155', '1155', '1155', '1190', '1190', '1190', '1239', '1239', '1239', '1286', '1286', '1286', '1308', '1308', '1308', '1325', '1325', '1325', '1401', '1401', '1401', '1447', '1447', '1447', '1494', '1494', '1494', '1534', '1534', '1534'], 'NDB_631': ['1', '1', '1', '14', '14', '14', '63', '63', '63', '114', '114', '114', '177', '177', '177', '194', '194', '194', '215', '215', '215', '236', '236', '236', '288', '288', '288', '318', '318', '318', '368', '368', '368', '414', '414', '414', '444', '444', '444', '473', '473', '473', '499', '499', '499', '536', '536', '536', '574', '574', '574', '600', '600', '600', '651', '651', '651', '693', '693', '693', '708', '708', '708', '784', '784', '784', '802', '802', '802', '853', '853', '853', '891', '891', '891', '956', '956', '956', '1001', '1001', '1001', '1032', '1032', '1032', '1058', '1058', '1058', '1120', '1120', '1120', '1170', '1170', '1170', '1235', '1235', '1235', '1252', '1252', '1252', '1294', '1294', '1294', '1316', '1316', '1316', '1334', '1334', '1334', '1408', '1408', '1408', '1446', '1446', '1446', '1478', '1478', '1478', '1516', '1516', '1516', '1536', '1536', '1536'], 'NDB_1066': ['1', '1', '1', '27', '27', '27', '86', '86', '86', '121', '121', '121', '147', '147', '147', '175', '175', '175', '213', '213', '213', '228', '228', '228', '278', '278', '278', '303', '303', '303', '343', '343', '343', '364', '364', '364', '398', '398', '398', '469', '469', '469', '482', '482', '482', '494', '494', '494', '504', '504', '504', '560', '560', '560', '578', '578', '578', '628', '628', '628', '729', '729', '729', '749', '749', '749', '792', '792', '792', '834', '834', '834', '867', '867', '867', '921', '921', '921', '987', '987', '987', '1036', '1036', '1036', '1073', '1073', '1073', '1114', '1114', '1114', '1157', '1157', '1157', '1232', '1232', '1232', '1241', '1241', '1241', '1269', '1269', '1269', '1285', '1285', '1285', '1343', '1343', '1343', '1378', '1378', '1378', '1409', '1409', '1409', '1459', '1459', '1459', '1472', '1472', '1472', '1518', '1518', '1518'], 'NDB_847': ['2', '2', '2', '39', '39', '39', '67', '67', '67', '101', '101', '101', '159', '159', '159', '187', '187', '187', '235', '235', '235', '263', '263', '263', '281', '281', '281', '321', '321', '321', '355', '355', '355', '370', '370', '370', '418', '418', '418', '463', '463', '463', '494', '494', '494', '545', '545', '545', '575', '575', '575', '593', '593', '593', '612', '612', '612', '673', '673', '673', '696', '696', '696', '741', '741', '741', '762', '762', '762', '784', '784', '784', '861', '861', '861', '922', '922', '922', '979', '979', '979', '1062', '1062', '1062', '1080', '1080', '1080', '1092', '1092', '1092', '1136', '1136', '1136', '1163', '1163', '1163', '1196', '1196', '1196', '1249', '1249', '1249', '1290', '1290', '1290', '1302', '1302', '1302', '1318', '1318', '1318', '1389', '1389', '1389', '1473', '1473', '1473', '1506', '1506', '1506'], 'NDB_832': ['2', '2', '2', '25', '25', '25', '93', '93', '93', '120', '120', '120', '150', '150', '150', '183', '183', '183', '236', '236', '236', '249', '249', '249', '278', '278', '278', '324', '324', '324', '399', '399', '399', '419', '419', '419', '448', '448', '448', '476', '476', '476', '492', '492', '492', '552', '552', '552', '567', '567', '567', '609', '609', '609', '659', '659', '659', '689', '689', '689', '723', '723', '723', '788', '788', '788', '797', '797', '797', '827', '827', '827', '872', '872', '872', '914', '914', '914', '962', '962', '962', '1023', '1023', '1023', '1071', '1071', '1071', '1083', '1083', '1083', '1113', '1113', '1113', '1166', '1166', '1166', '1234', '1234', '1234', '1279', '1279', '1279', '1311', '1311', '1311', '1360', '1360', '1360', '1402', '1402', '1402', '1445', '1445', '1445', '1486', '1486', '1486', '1519', '1519', '1519', '1538', '1538', '1538'], 'NDB_779': ['2', '2', '2', '29', '29', '29', '72', '72', '72', '122', '122', '122', '140', '140', '140', '172', '172', '172', '226', '226', '226', '266', '266', '266', '282', '282', '282', '342', '342', '342', '393', '393', '393', '416', '416', '416', '437', '437', '437', '473', '473', '473', '505', '505', '505', '548', '548', '548', '578', '578', '578', '594', '594', '594', '619', '619', '619', '667', '667', '667', '726', '726', '726', '728', '728', '728', '791', '791', '791', '828', '828', '828', '866', '866', '866', '915', '915', '915', '953', '953', '953', '991', '991', '991', '1050', '1050', '1050', '1067', '1067', '1067', '1076', '1076', '1076', '1097', '1097', '1097', '1176', '1176', '1176', '1226', '1226', '1226', '1278', '1278', '1278', '1304', '1304', '1304', '1330', '1330', '1330', '1396', '1396', '1396', '1455', '1455', '1455', '1491', '1491', '1491', '1533', '1533', '1533'], 'NDB_755': ['3', '3', '3', '75', '75', '75', '98', '98', '98', '106', '106', '106', '176', '176', '176', '194', '194', '194', '214', '214', '214', '260', '260', '260', '315', '315', '315', '329', '329', '329', '386', '386', '386', '409', '409', '409', '437', '437', '437', '487', '487', '487', '512', '512', '512', '577', '577', '577', '647', '647', '647', '675', '675', '675', '683', '683', '683', '715', '715', '715', '786', '786', '786', '806', '806', '806', '847', '847', '847', '874', '874', '874', '917', '917', '917', '959', '959', '959', '1002', '1002', '1002', '1055', '1055', '1055', '1066', '1066', '1066', '1085', '1085', '1085', '1132', '1132', '1132', '1136', '1136', '1136', '1184', '1184', '1184', '1207', '1207', '1207', '1282', '1282', '1282', '1336', '1336', '1336', '1361', '1361', '1361', '1393', '1393', '1393', '1441', '1441', '1441', '1465', '1465', '1465', '1519', '1519', '1519'], 'NDB_899': ['3', '3', '3', '19', '19', '19', '87', '87', '87', '109', '109', '109', '150', '150', '150', '178', '178', '178', '196', '196', '196', '226', '226', '226', '299', '299', '299', '322', '322', '322', '360', '360', '360', '385', '385', '385', '435', '435', '435', '482', '482', '482', '523', '523', '523', '566', '566', '566', '602', '602', '602', '658', '658', '658', '711', '711', '711', '739', '739', '739', '754', '754', '754', '801', '801', '801', '813', '813', '813', '858', '858', '858', '909', '909', '909', '930', '930', '930', '965', '965', '965', '994', '994', '994', '1037', '1037', '1037', '1051', '1051', '1051', '1105', '1105', '1105', '1137', '1137', '1137', '1183', '1183', '1183', '1231', '1231', '1231', '1276', '1276', '1276', '1334', '1334', '1334', '1359', '1359', '1359', '1389', '1389', '1389', '1492', '1492', '1492', '1521', '1521', '1521'], 'NDB_889': ['3', '3', '3', '43', '43', '43', '86', '86', '86', '111', '111', '111', '138', '138', '138', '160', '160', '160', '216', '216', '216', '236', '236', '236', '302', '302', '302', '317', '317', '317', '350', '350', '350', '387', '387', '387', '425', '425', '425', '443', '443', '443', '516', '516', '516', '531', '531', '531', '555', '555', '555', '589', '589', '589', '653', '653', '653', '691', '691', '691', '705', '705', '705', '751', '751', '751', '770', '770', '770', '832', '832', '832', '869', '869', '869', '897', '897', '897', '949', '949', '949', '974', '974', '974', '1057', '1057', '1057', '1079', '1079', '1079', '1133', '1133', '1133', '1154', '1154', '1154', '1205', '1205', '1205', '1256', '1256', '1256', '1295', '1295', '1295', '1340', '1340', '1340', '1406', '1406', '1406', '1449', '1449', '1449', '1498', '1498', '1498', '1508', '1508', '1508', '1525', '1525', '1525'], 'NDB_710': ['4', '4', '4', '31', '31', '31', '88', '88', '88', '116', '116', '116', '147', '147', '147', '189', '189', '189', '237', '237', '237', '264', '264', '264', '286', '286', '286', '322', '322', '322', '367', '367', '367', '408', '408', '408', '424', '424', '424', '446', '446', '446', '499', '499', '499', '538', '538', '538', '583', '583', '583', '597', '597', '597', '625', '625', '625', '673', '673', '673', '684', '684', '684', '715', '715', '715', '797', '797', '797', '820', '820', '820', '841', '841', '841', '904', '904', '904', '929', '929', '929', '988', '988', '988', '1044', '1044', '1044', '1118', '1118', '1118', '1154', '1154', '1154', '1206', '1206', '1206', '1233', '1233', '1233', '1250', '1250', '1250', '1263', '1263', '1263', '1301', '1301', '1301', '1323', '1323', '1323', '1396', '1396', '1396', '1468', '1468', '1468', '1487', '1487', '1487'], 'NDB_665': ['4', '4', '4', '25', '25', '25', '95', '95', '95', '127', '127', '127', '164', '164', '164', '209', '209', '209', '223', '223', '223', '269', '269', '269', '292', '292', '292', '321', '321', '321', '368', '368', '368', '402', '402', '402', '458', '458', '458', '482', '482', '482', '495', '495', '495', '544', '544', '544', '591', '591', '591', '660', '660', '660', '682', '682', '682', '717', '717', '717', '768', '768', '768', '806', '806', '806', '817', '817', '817', '840', '840', '840', '911', '911', '911', '924', '924', '924', '950', '950', '950', '997', '997', '997', '1049', '1049', '1049', '1067', '1067', '1067', '1117', '1117', '1117', '1147', '1147', '1147', '1179', '1179', '1179', '1225', '1225', '1225', '1245', '1245', '1245', '1254', '1254', '1254', '1337', '1337', '1337', '1395', '1395', '1395', '1444', '1444', '1444', '1470', '1470', '1470', '1508', '1508', '1508'], 'NDB_1028': ['4', '4', '4', '17', '17', '17', '34', '34', '34', '130', '130', '130', '171', '171', '171', '197', '197', '197', '247', '247', '247', '309', '309', '309', '338', '338', '338', '375', '375', '375', '396', '396', '396', '427', '427', '427', '470', '470', '470', '491', '491', '491', '520', '520', '520', '561', '561', '561', '578', '578', '578', '617', '617', '617', '672', '672', '672', '681', '681', '681', '698', '698', '698', '745', '745', '745', '783', '783', '783', '800', '800', '800', '854', '854', '854', '920', '920', '920', '969', '969', '969', '1051', '1051', '1051', '1066', '1066', '1066', '1080', '1080', '1080', '1131', '1131', '1131', '1166', '1166', '1166', '1218', '1218', '1218', '1272', '1272', '1272', '1303', '1303', '1303', '1344', '1344', '1344', '1398', '1398', '1398', '1454', '1454', '1454', '1471', '1471', '1471', '1504', '1504', '1504', '1537', '1537', '1537'], 'NDB_643': ['5', '5', '5', '17', '17', '17', '94', '94', '94', '155', '155', '155', '169', '169', '169', '234', '234', '234', '269', '269', '269', '281', '281', '281', '311', '311', '311', '346', '346', '346', '381', '381', '381', '399', '399', '399', '453', '453', '453', '485', '485', '485', '522', '522', '522', '579', '579', '579', '620', '620', '620', '649', '649', '649', '690', '690', '690', '709', '709', '709', '752', '752', '752', '801', '801', '801', '821', '821', '821', '857', '857', '857', '894', '894', '894', '949', '949', '949', '971', '971', '971', '1047', '1047', '1047', '1076', '1076', '1076', '1132', '1132', '1132', '1180', '1180', '1180', '1232', '1232', '1232', '1252', '1252', '1252', '1298', '1298', '1298', '1312', '1312', '1312', '1331', '1331', '1331', '1401', '1401', '1401', '1429', '1429', '1429', '1458', '1458', '1458', '1485', '1485', '1485', '1500', '1500', '1500'], 'NDB_805': ['5', '5', '5', '47', '47', '47', '84', '84', '84', '164', '164', '164', '176', '176', '176', '199', '199', '199', '245', '245', '245', '283', '283', '283', '327', '327', '327', '341', '341', '341', '373', '373', '373', '400', '400', '400', '426', '426', '426', '455', '455', '455', '506', '506', '506', '541', '541', '541', '587', '587', '587', '614', '614', '614', '653', '653', '653', '687', '687', '687', '711', '711', '711', '760', '760', '760', '784', '784', '784', '855', '855', '855', '921', '921', '921', '968', '968', '968', '996', '996', '996', '1061', '1061', '1061', '1083', '1083', '1083', '1126', '1126', '1126', '1138', '1138', '1138', '1188', '1188', '1188', '1218', '1218', '1218', '1264', '1264', '1264', '1289', '1289', '1289', '1317', '1317', '1317', '1354', '1354', '1354', '1396', '1396', '1396', '1439', '1439', '1439', '1469', '1469', '1469', '1490', '1490', '1490'], 'NDB_730': ['5', '5', '5', '46', '46', '46', '95', '95', '95', '132', '132', '132', '172', '172', '172', '200', '200', '200', '250', '250', '250', '271', '271', '271', '300', '300', '300', '334', '334', '334', '379', '379', '379', '406', '406', '406', '454', '454', '454', '475', '475', '475', '516', '516', '516', '569', '569', '569', '656', '656', '656', '669', '669', '669', '688', '688', '688', '704', '704', '704', '750', '750', '750', '766', '766', '766', '834', '834', '834', '849', '849', '849', '918', '918', '918', '956', '956', '956', '976', '976', '976', '1031', '1031', '1031', '1112', '1112', '1112', '1172', '1172', '1172', '1228', '1228', '1228', '1238', '1238', '1238', '1279', '1279', '1279', '1292', '1292', '1292', '1309', '1309', '1309', '1332', '1332', '1332', '1400', '1400', '1400', '1448', '1448', '1448', '1465', '1465', '1465', '1514', '1514', '1514', '1548', '1548', '1548'], 'NDB_883': ['6', '6', '6', '22', '22', '22', '85', '85', '85', '159', '159', '159', '163', '163', '163', '220', '220', '220', '247', '247', '247', '289', '289', '289', '311', '311', '311', '348', '348', '348', '361', '361', '361', '385', '385', '385', '462', '462', '462', '469', '469', '469', '511', '511', '511', '548', '548', '548', '591', '591', '591', '615', '615', '615', '676', '676', '676', '687', '687', '687', '727', '727', '727', '764', '764', '764', '810', '810', '810', '816', '816', '816', '838', '838', '838', '896', '896', '896', '931', '931', '931', '963', '963', '963', '1000', '1000', '1000', '1059', '1059', '1059', '1116', '1116', '1116', '1142', '1142', '1142', '1181', '1181', '1181', '1213', '1213', '1213', '1243', '1243', '1243', '1275', '1275', '1275', '1301', '1301', '1301', '1340', '1340', '1340', '1379', '1379', '1379', '1489', '1489', '1489', '1534', '1534', '1534'], 'NDB_1060': ['6', '6', '6', '76', '76', '76', '89', '89', '89', '148', '148', '148', '161', '161', '161', '197', '197', '197', '225', '225', '225', '272', '272', '272', '302', '302', '302', '330', '330', '330', '366', '366', '366', '403', '403', '403', '464', '464', '464', '507', '507', '507', '535', '535', '535', '558', '558', '558', '595', '595', '595', '619', '619', '619', '663', '663', '663', '706', '706', '706', '738', '738', '738', '759', '759', '759', '782', '782', '782', '823', '823', '823', '862', '862', '862', '888', '888', '888', '917', '917', '917', '962', '962', '962', '992', '992', '992', '1060', '1060', '1060', '1117', '1117', '1117', '1172', '1172', '1172', '1194', '1194', '1194', '1212', '1212', '1212', '1248', '1248', '1248', '1263', '1263', '1263', '1318', '1318', '1318', '1354', '1354', '1354', '1427', '1427', '1427', '1492', '1492', '1492'], 'NDB_853': ['6', '6', '6', '26', '26', '26', '69', '69', '69', '134', '134', '134', '179', '179', '179', '230', '230', '230', '254', '254', '254', '308', '308', '308', '327', '327', '327', '363', '363', '363', '395', '395', '395', '430', '430', '430', '458', '458', '458', '521', '521', '521', '538', '538', '538', '560', '560', '560', '575', '575', '575', '656', '656', '656', '695', '695', '695', '725', '725', '725', '761', '761', '761', '779', '779', '779', '821', '821', '821', '850', '850', '850', '903', '903', '903', '926', '926', '926', '970', '970', '970', '1035', '1035', '1035', '1077', '1077', '1077', '1133', '1133', '1133', '1156', '1156', '1156', '1197', '1197', '1197', '1270', '1270', '1270', '1284', '1284', '1284', '1306', '1306', '1306', '1330', '1330', '1330', '1408', '1408', '1408', '1456', '1456', '1456', '1475', '1475', '1475', '1513', '1513', '1513', '1529', '1529', '1529'], 'NDB_974': ['7', '7', '7', '34', '34', '34', '79', '79', '79', '126', '126', '126', '161', '161', '161', '220', '220', '220', '252', '252', '252', '285', '285', '285', '326', '326', '326', '339', '339', '339', '376', '376', '376', '399', '399', '399', '422', '422', '422', '437', '437', '437', '517', '517', '517', '555', '555', '555', '606', '606', '606', '652', '652', '652', '692', '692', '692', '718', '718', '718', '757', '757', '757', '793', '793', '793', '815', '815', '815', '844', '844', '844', '907', '907', '907', '952', '952', '952', '988', '988', '988', '1035', '1035', '1035', '1073', '1073', '1073', '1120', '1120', '1120', '1179', '1179', '1179', '1217', '1217', '1217', '1228', '1228', '1228', '1262', '1262', '1262', '1297', '1297', '1297', '1307', '1307', '1307', '1358', '1358', '1358', '1387', '1387', '1387', '1452', '1452', '1452', '1462', '1462', '1462', '1473', '1473', '1473'], 'NDB_754': ['7', '7', '7', '24', '24', '24', '96', '96', '96', '118', '118', '118', '159', '159', '159', '208', '208', '208', '230', '230', '230', '295', '295', '295', '314', '314', '314', '347', '347', '347', '364', '364', '364', '401', '401', '401', '424', '424', '424', '453', '453', '453', '524', '524', '524', '561', '561', '561', '576', '576', '576', '621', '621', '621', '665', '665', '665', '704', '704', '704', '739', '739', '739', '755', '755', '755', '786', '786', '786', '828', '828', '828', '871', '871', '871', '908', '908', '908', '986', '986', '986', '1020', '1020', '1020', '1046', '1046', '1046', '1081', '1081', '1081', '1177', '1177', '1177', '1219', '1219', '1219', '1264', '1264', '1264', '1293', '1293', '1293', '1315', '1315', '1315', '1363', '1363', '1363', '1405', '1405', '1405', '1430', '1430', '1430', '1477', '1477', '1477', '1508', '1508', '1508', '1536', '1536', '1536'], 'NDB_634': ['7', '7', '7', '46', '46', '46', '94', '94', '94', '163', '163', '163', '175', '175', '175', '219', '219', '219', '251', '251', '251', '282', '282', '282', '305', '305', '305', '349', '349', '349', '389', '389', '389', '412', '412', '412', '449', '449', '449', '486', '486', '486', '508', '508', '508', '528', '528', '528', '581', '581', '581', '616', '616', '616', '648', '648', '648', '711', '711', '711', '738', '738', '738', '788', '788', '788', '819', '819', '819', '837', '837', '837', '905', '905', '905', '957', '957', '957', '1001', '1001', '1001', '1040', '1040', '1040', '1078', '1078', '1078', '1128', '1128', '1128', '1174', '1174', '1174', '1197', '1197', '1197', '1240', '1240', '1240', '1245', '1245', '1245', '1261', '1261', '1261', '1302', '1302', '1302', '1324', '1324', '1324', '1392', '1392', '1392', '1441', '1441', '1441', '1471', '1471', '1471', '1487', '1487', '1487', '1530', '1530', '1530'], 'NDB_661': ['8', '8', '8', '19', '19', '19', '30', '30', '30', '112', '112', '112', '154', '154', '154', '170', '170', '170', '214', '214', '214', '259', '259', '259', '284', '284', '284', '319', '319', '319', '354', '354', '354', '401', '401', '401', '446', '446', '446', '479', '479', '479', '525', '525', '525', '568', '568', '568', '605', '605', '605', '660', '660', '660', '694', '694', '694', '714', '714', '714', '756', '756', '756', '769', '769', '769', '816', '816', '816', '839', '839', '839', '915', '915', '915', '927', '927', '927', '954', '954', '954', '1005', '1005', '1005', '1052', '1052', '1052', '1080', '1080', '1080', '1172', '1172', '1172', '1204', '1204', '1204', '1235', '1235', '1235', '1261', '1261', '1261', '1287', '1287', '1287', '1307', '1307', '1307', '1357', '1357', '1357', '1403', '1403', '1403', '1437', '1437', '1437', '1472', '1472', '1472', '1512', '1512', '1512', '1525', '1525', '1525'], 'NDB_901': ['8', '8', '8', '39', '39', '39', '91', '91', '91', '100', '100', '100', '125', '125', '125', '165', '165', '165', '203', '203', '203', '241', '241', '241', '302', '302', '302', '320', '320', '320', '360', '360', '360', '388', '388', '388', '452', '452', '452', '468', '468', '468', '497', '497', '497', '543', '543', '543', '567', '567', '567', '624', '624', '624', '674', '674', '674', '695', '695', '695', '717', '717', '717', '780', '780', '780', '799', '799', '799', '856', '856', '856', '921', '921', '921', '967', '967', '967', '981', '981', '981', '1043', '1043', '1043', '1082', '1082', '1082', '1132', '1132', '1132', '1146', '1146', '1146', '1177', '1177', '1177', '1215', '1215', '1215', '1244', '1244', '1244', '1274', '1274', '1274', '1305', '1305', '1305', '1344', '1344', '1344', '1392', '1392', '1392', '1448', '1448', '1448', '1462', '1462', '1462', '1503', '1503', '1503'], 'NDB_835': ['8', '8', '8', '83', '83', '83', '101', '101', '101', '130', '130', '130', '184', '184', '184', '199', '199', '199', '239', '239', '239', '254', '254', '254', '300', '300', '300', '322', '322', '322', '353', '353', '353', '402', '402', '402', '460', '460', '460', '486', '486', '486', '490', '490', '490', '503', '503', '503', '571', '571', '571', '608', '608', '608', '654', '654', '654', '720', '720', '720', '735', '735', '735', '790', '790', '790', '808', '808', '808', '824', '824', '824', '869', '869', '869', '895', '895', '895', '971', '971', '971', '982', '982', '982', '1038', '1038', '1038', '1094', '1094', '1094', '1129', '1129', '1129', '1160', '1160', '1160', '1224', '1224', '1224', '1260', '1260', '1260', '1299', '1299', '1299', '1343', '1343', '1343', '1405', '1405', '1405', '1455', '1455', '1455', '1480', '1480', '1480', '1515', '1515', '1515', '1538', '1538', '1538'], 'NDB_878': ['9', '9', '9', '65', '65', '65', '109', '109', '109', '122', '122', '122', '188', '188', '188', '218', '218', '218', '248', '248', '248', '310', '310', '310', '346', '346', '346', '362', '362', '362', '395', '395', '395', '431', '431', '431', '460', '460', '460', '517', '517', '517', '539', '539', '539', '570', '570', '570', '595', '595', '595', '648', '648', '648', '694', '694', '694', '722', '722', '722', '758', '758', '758', '796', '796', '796', '822', '822', '822', '847', '847', '847', '914', '914', '914', '933', '933', '933', '960', '960', '960', '1004', '1004', '1004', '1039', '1039', '1039', '1116', '1116', '1116', '1154', '1154', '1154', '1188', '1188', '1188', '1208', '1208', '1208', '1254', '1254', '1254', '1291', '1291', '1291', '1313', '1313', '1313', '1350', '1350', '1350', '1382', '1382', '1382', '1440', '1440', '1440', '1477', '1477', '1477', '1514', '1514', '1514'], 'NDB_836': ['9', '9', '9', '67', '67', '67', '83', '83', '83', '106', '106', '106', '119', '119', '119', '134', '134', '134', '222', '222', '222', '249', '249', '249', '285', '285', '285', '303', '303', '303', '351', '351', '351', '397', '397', '397', '453', '453', '453', '477', '477', '477', '510', '510', '510', '565', '565', '565', '587', '587', '587', '650', '650', '650', '666', '666', '666', '702', '702', '702', '743', '743', '743', '764', '764', '764', '803', '803', '803', '823', '823', '823', '868', '868', '868', '905', '905', '905', '924', '924', '924', '956', '956', '956', '978', '978', '978', '1052', '1052', '1052', '1118', '1118', '1118', '1148', '1148', '1148', '1192', '1192', '1192', '1226', '1226', '1226', '1256', '1256', '1256', '1310', '1310', '1310', '1362', '1362', '1362', '1398', '1398', '1398', '1436', '1436', '1436', '1461', '1461', '1461', '1503', '1503', '1503'], 'NDB_742': ['9', '9', '9', '45', '45', '45', '71', '71', '71', '138', '138', '138', '156', '156', '156', '171', '171', '171', '240', '240', '240', '266', '266', '266', '287', '287', '287', '330', '330', '330', '367', '367', '367', '401', '401', '401', '423', '423', '423', '489', '489', '489', '526', '526', '526', '566', '566', '566', '613', '613', '613', '671', '671', '671', '683', '683', '683', '721', '721', '721', '767', '767', '767', '808', '808', '808', '825', '825', '825', '870', '870', '870', '922', '922', '922', '985', '985', '985', '1003', '1003', '1003', '1040', '1040', '1040', '1068', '1068', '1068', '1074', '1074', '1074', '1087', '1087', '1087', '1142', '1142', '1142', '1217', '1217', '1217', '1257', '1257', '1257', '1312', '1312', '1312', '1360', '1360', '1360', '1408', '1408', '1408', '1451', '1451', '1451', '1490', '1490', '1490', '1502', '1502', '1502', '1548', '1548', '1548'], 'NDB_844': ['10', '10', '10', '42', '42', '42', '98', '98', '98', '111', '111', '111', '154', '154', '154', '182', '182', '182', '228', '228', '228', '264', '264', '264', '293', '293', '293', '340', '340', '340', '382', '382', '382', '421', '421', '421', '452', '452', '452', '489', '489', '489', '522', '522', '522', '559', '559', '559', '596', '596', '596', '618', '618', '618', '676', '676', '676', '681', '681', '681', '733', '733', '733', '757', '757', '757', '791', '791', '791', '835', '835', '835', '872', '872', '872', '924', '924', '924', '986', '986', '986', '1053', '1053', '1053', '1070', '1070', '1070', '1078', '1078', '1078', '1129', '1129', '1129', '1156', '1156', '1156', '1195', '1195', '1195', '1222', '1222', '1222', '1259', '1259', '1259', '1313', '1313', '1313', '1352', '1352', '1352', '1385', '1385', '1385', '1446', '1446', '1446', '1492', '1492', '1492', '1506', '1506', '1506'], 'NDB_812': ['10', '10', '10', '31', '31', '31', '81', '81', '81', '126', '126', '126', '160', '160', '160', '176', '176', '176', '206', '206', '206', '265', '265', '265', '281', '281', '281', '328', '328', '328', '390', '390', '390', '419', '419', '419', '436', '436', '436', '447', '447', '447', '510', '510', '510', '527', '527', '527', '580', '580', '580', '600', '600', '600', '626', '626', '626', '677', '677', '677', '714', '714', '714', '738', '738', '738', '754', '754', '754', '774', '774', '774', '833', '833', '833', '867', '867', '867', '920', '920', '920', '989', '989', '989', '1059', '1059', '1059', '1089', '1089', '1089', '1123', '1123', '1123', '1147', '1147', '1147', '1208', '1208', '1208', '1219', '1219', '1219', '1242', '1242', '1242', '1258', '1258', '1258', '1342', '1342', '1342', '1455', '1455', '1455', '1475', '1475', '1475', '1507', '1507', '1507', '1548', '1548', '1548'], 'NDB_637': ['10', '10', '10', '71', '71', '71', '129', '129', '129', '148', '148', '148', '173', '173', '173', '210', '210', '210', '271', '271', '271', '290', '290', '290', '314', '314', '314', '345', '345', '345', '396', '396', '396', '412', '412', '412', '463', '463', '463', '473', '473', '473', '498', '498', '498', '533', '533', '533', '562', '562', '562', '598', '598', '598', '654', '654', '654', '662', '662', '662', '682', '682', '682', '719', '719', '719', '780', '780', '780', '801', '801', '801', '859', '859', '859', '923', '923', '923', '964', '964', '964', '1045', '1045', '1045', '1072', '1072', '1072', '1119', '1119', '1119', '1143', '1143', '1143', '1184', '1184', '1184', '1229', '1229', '1229', '1262', '1262', '1262', '1289', '1289', '1289', '1323', '1323', '1323', '1350', '1350', '1350', '1399', '1399', '1399', '1450', '1450', '1450', '1476', '1476', '1476', '1512', '1512', '1512'], 'NDB_737': ['11', '11', '11', '80', '80', '80', '96', '96', '96', '121', '121', '121', '170', '170', '170', '203', '203', '203', '252', '252', '252', '273', '273', '273', '286', '286', '286', '316', '316', '316', '384', '384', '384', '400', '400', '400', '440', '440', '440', '485', '485', '485', '509', '509', '509', '528', '528', '528', '564', '564', '564', '596', '596', '596', '612', '612', '612', '668', '668', '668', '691', '691', '691', '727', '727', '727', '768', '768', '768', '807', '807', '807', '813', '813', '813', '859', '859', '859', '902', '902', '902', '985', '985', '985', '1041', '1041', '1041', '1097', '1097', '1097', '1123', '1123', '1123', '1140', '1140', '1140', '1189', '1189', '1189', '1234', '1234', '1234', '1246', '1246', '1246', '1292', '1292', '1292', '1336', '1336', '1336', '1398', '1398', '1398', '1435', '1435', '1435', '1478', '1478', '1478', '1517', '1517', '1517'], 'NDB_935': ['11', '11', '11', '32', '32', '32', '45', '45', '45', '143', '143', '143', '182', '182', '182', '195', '195', '195', '217', '217', '217', '242', '242', '242', '299', '299', '299', '323', '323', '323', '373', '373', '373', '416', '416', '416', '424', '424', '424', '464', '464', '464', '491', '491', '491', '544', '544', '544', '571', '571', '571', '582', '582', '582', '626', '626', '626', '713', '713', '713', '741', '741', '741', '758', '758', '758', '777', '777', '777', '849', '849', '849', '863', '863', '863', '901', '901', '901', '959', '959', '959', '977', '977', '977', '1043', '1043', '1043', '1063', '1063', '1063', '1114', '1114', '1114', '1133', '1133', '1133', '1168', '1168', '1168', '1191', '1191', '1191', '1235', '1235', '1235', '1262', '1262', '1262', '1311', '1311', '1311', '1331', '1331', '1331', '1384', '1384', '1384', '1453', '1453', '1453', '1497', '1497', '1497'], 'NDB_925': ['11', '11', '11', '29', '29', '29', '117', '117', '117', '168', '168', '168', '192', '192', '192', '234', '234', '234', '256', '256', '256', '272', '272', '272', '278', '278', '278', '319', '319', '319', '387', '387', '387', '415', '415', '415', '441', '441', '441', '475', '475', '475', '506', '506', '506', '537', '537', '537', '577', '577', '577', '607', '607', '607', '662', '662', '662', '718', '718', '718', '734', '734', '734', '789', '789', '789', '822', '822', '822', '850', '850', '850', '891', '891', '891', '930', '930', '930', '969', '969', '969', '1000', '1000', '1000', '1062', '1062', '1062', '1109', '1109', '1109', '1167', '1167', '1167', '1221', '1221', '1221', '1240', '1240', '1240', '1242', '1242', '1242', '1277', '1277', '1277', '1320', '1320', '1320', '1348', '1348', '1348', '1380', '1380', '1380', '1444', '1444', '1444', '1480', '1480', '1480', '1495', '1495', '1495'], 'NDB_790': ['12', '12', '12', '38', '38', '38', '69', '69', '69', '120', '120', '120', '155', '155', '155', '226', '226', '226', '257', '257', '257', '305', '305', '305', '339', '339', '339', '356', '356', '356', '398', '398', '398', '415', '415', '415', '429', '429', '429', '451', '451', '451', '501', '501', '501', '540', '540', '540', '561', '561', '561', '582', '582', '582', '613', '613', '613', '675', '675', '675', '699', '699', '699', '740', '740', '740', '796', '796', '796', '832', '832', '832', '846', '846', '846', '910', '910', '910', '931', '931', '931', '989', '989', '989', '1042', '1042', '1042', '1064', '1064', '1064', '1112', '1112', '1112', '1170', '1170', '1170', '1189', '1189', '1189', '1225', '1225', '1225', '1280', '1280', '1280', '1305', '1305', '1305', '1318', '1318', '1318', '1394', '1394', '1394', '1439', '1439', '1439', '1468', '1468', '1468', '1510', '1510', '1510'], 'NDB_788': ['12', '12', '12', '74', '74', '74', '128', '128', '128', '149', '149', '149', '173', '173', '173', '203', '203', '203', '248', '248', '248', '275', '275', '275', '335', '335', '335', '350', '350', '350', '394', '394', '394', '413', '413', '413', '467', '467', '467', '500', '500', '500', '537', '537', '537', '568', '568', '568', '593', '593', '593', '615', '615', '615', '672', '672', '672', '684', '684', '684', '709', '709', '709', '760', '760', '760', '779', '779', '779', '853', '853', '853', '862', '862', '862', '918', '918', '918', '961', '961', '961', '995', '995', '995', '1033', '1033', '1033', '1073', '1073', '1073', '1113', '1113', '1113', '1128', '1128', '1128', '1139', '1139', '1139', '1187', '1187', '1187', '1219', '1219', '1219', '1271', '1271', '1271', '1321', '1321', '1321', '1346', '1346', '1346', '1385', '1385', '1385', '1496', '1496', '1496'], 'NDB_934': ['12', '12', '12', '45', '45', '45', '92', '92', '92', '124', '124', '124', '180', '180', '180', '205', '205', '205', '253', '253', '253', '273', '273', '273', '296', '296', '296', '318', '318', '318', '355', '355', '355', '406', '406', '406', '449', '449', '449', '483', '483', '483', '497', '497', '497', '530', '530', '530', '558', '558', '558', '601', '601', '601', '657', '657', '657', '697', '697', '697', '739', '739', '739', '797', '797', '797', '855', '855', '855', '868', '868', '868', '906', '906', '906', '928', '928', '928', '971', '971', '971', '1035', '1035', '1035', '1050', '1050', '1050', '1079', '1079', '1079', '1143', '1143', '1143', '1236', '1236', '1236', '1254', '1254', '1254', '1287', '1287', '1287', '1320', '1320', '1320', '1352', '1352', '1352', '1378', '1378', '1378', '1454', '1454', '1454', '1466', '1466', '1466', '1509', '1509', '1509'], 'NDB_746': ['13', '13', '13', '31', '31', '31', '62', '62', '62', '111', '111', '111', '131', '131', '131', '192', '192', '192', '202', '202', '202', '245', '245', '245', '296', '296', '296', '333', '333', '333', '374', '374', '374', '418', '418', '418', '454', '454', '454', '486', '486', '486', '496', '496', '496', '539', '539', '539', '560', '560', '560', '609', '609', '609', '627', '627', '627', '692', '692', '692', '709', '709', '709', '787', '787', '787', '813', '813', '813', '856', '856', '856', '915', '915', '915', '959', '959', '959', '1003', '1003', '1003', '1054', '1054', '1054', '1090', '1090', '1090', '1122', '1122', '1122', '1170', '1170', '1170', '1212', '1212', '1212', '1227', '1227', '1227', '1243', '1243', '1243', '1286', '1286', '1286', '1303', '1303', '1303', '1322', '1322', '1322', '1386', '1386', '1386', '1436', '1436', '1436', '1463', '1463', '1463', '1481', '1481', '1481', '1526', '1526', '1526'], 'NDB_818': ['13', '13', '13', '89', '89', '89', '118', '118', '118', '157', '157', '157', '186', '186', '186', '195', '195', '195', '211', '211', '211', '262', '262', '262', '287', '287', '287', '317', '317', '317', '357', '357', '357', '380', '380', '380', '447', '447', '447', '461', '461', '461', '506', '506', '506', '540', '540', '540', '586', '586', '586', '618', '618', '618', '650', '650', '650', '728', '728', '728', '780', '780', '780', '802', '802', '802', '820', '820', '820', '844', '844', '844', '919', '919', '919', '975', '975', '975', '1005', '1005', '1005', '1029', '1029', '1029', '1066', '1066', '1066', '1077', '1077', '1077', '1113', '1113', '1113', '1137', '1137', '1137', '1186', '1186', '1186', '1230', '1230', '1230', '1273', '1273', '1273', '1324', '1324', '1324', '1347', '1347', '1347', '1390', '1390', '1390', '1458', '1458', '1458', '1474', '1474', '1474', '1520', '1520', '1520'], 'NDB_961': ['14', '14', '14', '43', '43', '43', '77', '77', '77', '115', '115', '115', '131', '131', '131', '162', '162', '162', '214', '214', '214', '241', '241', '241', '300', '300', '300', '323', '323', '323', '371', '371', '371', '421', '421', '421', '426', '426', '426', '466', '466', '466', '502', '502', '502', '529', '529', '529', '573', '573', '573', '597', '597', '597', '663', '663', '663', '710', '710', '710', '736', '736', '736', '793', '793', '793', '837', '837', '837', '878', '878', '878', '894', '894', '894', '912', '912', '912', '965', '965', '965', '999', '999', '999', '1065', '1065', '1065', '1089', '1089', '1089', '1155', '1155', '1155', '1189', '1189', '1189', '1213', '1213', '1213', '1257', '1257', '1257', '1288', '1288', '1288', '1314', '1314', '1314', '1350', '1350', '1350', '1395', '1395', '1395', '1445', '1445', '1445', '1461', '1461', '1461', '1501', '1501', '1501'], 'NDB_866': ['14', '14', '14', '68', '68', '68', '102', '102', '102', '127', '127', '127', '175', '175', '175', '222', '222', '222', '258', '258', '258', '309', '309', '309', '337', '337', '337', '391', '391', '391', '413', '413', '413', '423', '423', '423', '451', '451', '451', '517', '517', '517', '538', '538', '538', '572', '572', '572', '607', '607', '607', '629', '629', '629', '667', '667', '667', '680', '680', '680', '691', '691', '691', '744', '744', '744', '762', '762', '762', '795', '795', '795', '857', '857', '857', '906', '906', '906', '958', '958', '958', '976', '976', '976', '1060', '1060', '1060', '1081', '1081', '1081', '1126', '1126', '1126', '1171', '1171', '1171', '1207', '1207', '1207', '1255', '1255', '1255', '1322', '1322', '1322', '1357', '1357', '1357', '1399', '1399', '1399', '1453', '1453', '1453', '1474', '1474', '1474', '1515', '1515', '1515', '1542', '1542', '1542'], 'NDB_652': ['15', '15', '15', '90', '90', '90', '120', '120', '120', '141', '141', '141', '185', '185', '185', '196', '196', '196', '243', '243', '243', '268', '268', '268', '288', '288', '288', '333', '333', '333', '388', '388', '388', '404', '404', '404', '439', '439', '439', '483', '483', '483', '507', '507', '507', '531', '531', '531', '576', '576', '576', '590', '590', '590', '628', '628', '628', '668', '668', '668', '686', '686', '686', '719', '719', '719', '766', '766', '766', '808', '808', '808', '845', '845', '845', '873', '873', '873', '901', '901', '901', '926', '926', '926', '984', '984', '984', '1052', '1052', '1052', '1116', '1116', '1116', '1136', '1136', '1136', '1186', '1186', '1186', '1210', '1210', '1210', '1247', '1247', '1247', '1288', '1288', '1288', '1341', '1341', '1341', '1444', '1444', '1444', '1471', '1471', '1471', '1511', '1511', '1511', '1542', '1542', '1542'], 'NDB_958': ['15', '15', '15', '72', '72', '72', '91', '91', '91', '114', '114', '114', '156', '156', '156', '179', '179', '179', '245', '245', '245', '267', '267', '267', '316', '316', '316', '343', '343', '343', '354', '354', '354', '408', '408', '408', '465', '465', '465', '480', '480', '480', '503', '503', '503', '530', '530', '530', '572', '572', '572', '652', '652', '652', '665', '665', '665', '706', '706', '706', '736', '736', '736', '750', '750', '750', '788', '788', '788', '814', '814', '814', '838', '838', '838', '900', '900', '900', '977', '977', '977', '995', '995', '995', '1025', '1025', '1025', '1045', '1045', '1045', '1091', '1091', '1091', '1127', '1127', '1127', '1148', '1148', '1148', '1182', '1182', '1182', '1230', '1230', '1230', '1272', '1272', '1272', '1342', '1342', '1342', '1394', '1394', '1394', '1449', '1449', '1449', '1479', '1479', '1479', '1506', '1506', '1506'], 'NDB_926': ['15', '15', '15', '46', '46', '46', '88', '88', '88', '100', '100', '100', '123', '123', '123', '180', '180', '180', '215', '215', '215', '240', '240', '240', '307', '307', '307', '335', '335', '335', '359', '359', '359', '383', '383', '383', '427', '427', '427', '469', '469', '469', '492', '492', '492', '551', '551', '551', '592', '592', '592', '629', '629', '629', '664', '664', '664', '726', '726', '726', '742', '742', '742', '754', '754', '754', '769', '769', '769', '848', '848', '848', '878', '878', '878', '907', '907', '907', '964', '964', '964', '978', '978', '978', '1038', '1038', '1038', '1064', '1064', '1064', '1090', '1090', '1090', '1165', '1165', '1165', '1180', '1180', '1180', '1205', '1205', '1205', '1251', '1251', '1251', '1293', '1293', '1293', '1313', '1313', '1313', '1354', '1354', '1354', '1384', '1384', '1384', '1442', '1442', '1442', '1494', '1494', '1494'], 'NDB_815': ['16', '16', '16', '85', '85', '85', '102', '102', '102', '139', '139', '139', '161', '161', '161', '224', '224', '224', '246', '246', '246', '308', '308', '308', '334', '334', '334', '351', '351', '351', '382', '382', '382', '398', '398', '398', '425', '425', '425', '456', '456', '456', '508', '508', '508', '577', '577', '577', '601', '601', '601', '658', '658', '658', '708', '708', '708', '736', '736', '736', '753', '753', '753', '769', '769', '769', '843', '843', '843', '871', '871', '871', '914', '914', '914', '950', '950', '950', '984', '984', '984', '1030', '1030', '1030', '1072', '1072', '1072', '1108', '1108', '1108', '1121', '1121', '1121', '1176', '1176', '1176', '1206', '1206', '1206', '1274', '1274', '1274', '1331', '1331', '1331', '1346', '1346', '1346', '1404', '1404', '1404', '1451', '1451', '1451', '1469', '1469', '1469', '1507', '1507', '1507', '1526', '1526', '1526'], 'NDB_989': ['16', '16', '16', '63', '63', '63', '94', '94', '94', '130', '130', '130', '173', '173', '173', '222', '222', '222', '259', '259', '259', '276', '276', '276', '326', '326', '326', '348', '348', '348', '370', '370', '370', '386', '386', '386', '440', '440', '440', '478', '478', '478', '504', '504', '504', '518', '518', '518', '557', '557', '557', '602', '602', '602', '625', '625', '625', '686', '686', '686', '717', '717', '717', '767', '767', '767', '811', '811', '811', '855', '855', '855', '866', '866', '866', '892', '892', '892', '935', '935', '935', '960', '960', '960', '977', '977', '977', '1042', '1042', '1042', '1069', '1069', '1069', '1115', '1115', '1115', '1153', '1153', '1153', '1194', '1194', '1194', '1239', '1239', '1239', '1277', '1277', '1277', '1338', '1338', '1338', '1390', '1390', '1390', '1442', '1442', '1442', '1482', '1482', '1482', '1501', '1501', '1501', '1529', '1529', '1529'], 'NDB_910': ['17', '17', '17', '33', '33', '33', '74', '74', '74', '146', '146', '146', '163', '163', '163', '211', '211', '211', '263', '263', '263', '312', '312', '312', '344', '344', '344', '382', '382', '382', '397', '397', '397', '414', '414', '414', '438', '438', '438', '480', '480', '480', '529', '529', '529', '546', '546', '546', '589', '589', '589', '611', '611', '611', '677', '677', '677', '692', '692', '692', '713', '713', '713', '766', '766', '766', '807', '807', '807', '817', '817', '817', '858', '858', '858', '892', '892', '892', '982', '982', '982', '992', '992', '992', '1039', '1039', '1039', '1110', '1110', '1110', '1130', '1130', '1130', '1144', '1144', '1144', '1165', '1165', '1165', '1215', '1215', '1215', '1255', '1255', '1255', '1289', '1289', '1289', '1308', '1308', '1308', '1356', '1356', '1356', '1456', '1456', '1456', '1495', '1495', '1495', '1533', '1533', '1533'], 'NDB_760': ['18', '18', '18', '75', '75', '75', '117', '117', '117', '137', '137', '137', '193', '193', '193', '216', '216', '216', '242', '242', '242', '266', '266', '266', '280', '280', '280', '320', '320', '320', '365', '365', '365', '408', '408', '408', '444', '444', '444', '474', '474', '474', '529', '529', '529', '552', '552', '552', '569', '569', '569', '603', '603', '603', '660', '660', '660', '690', '690', '690', '707', '707', '707', '757', '757', '757', '776', '776', '776', '824', '824', '824', '859', '859', '859', '906', '906', '906', '978', '978', '978', '989', '989', '989', '1061', '1061', '1061', '1077', '1077', '1077', '1107', '1107', '1107', '1181', '1181', '1181', '1196', '1196', '1196', '1241', '1241', '1241', '1271', '1271', '1271', '1286', '1286', '1286', '1341', '1341', '1341', '1359', '1359', '1359', '1382', '1382', '1382', '1488', '1488', '1488', '1530', '1530', '1530'], 'NDB_734': ['18', '18', '18', '47', '47', '47', '95', '95', '95', '143', '143', '143', '181', '181', '181', '207', '207', '207', '229', '229', '229', '276', '276', '276', '335', '335', '335', '391', '391', '391', '414', '414', '414', '463', '463', '463', '468', '468', '468', '490', '490', '490', '502', '502', '502', '564', '564', '564', '599', '599', '599', '620', '620', '620', '675', '675', '675', '694', '694', '694', '723', '723', '723', '759', '759', '759', '783', '783', '783', '850', '850', '850', '864', '864', '864', '889', '889', '889', '925', '925', '925', '980', '980', '980', '1059', '1059', '1059', '1118', '1118', '1118', '1174', '1174', '1174', '1190', '1190', '1190', '1211', '1211', '1211', '1273', '1273', '1273', '1295', '1295', '1295', '1319', '1319', '1319', '1352', '1352', '1352', '1409', '1409', '1409', '1479', '1479', '1479', '1511', '1511', '1511', '1526', '1526', '1526'], 'NDB_712': ['18', '18', '18', '44', '44', '44', '76', '76', '76', '146', '146', '146', '170', '170', '170', '210', '210', '210', '234', '234', '234', '304', '304', '304', '318', '318', '318', '337', '337', '337', '380', '380', '380', '410', '410', '410', '432', '432', '432', '477', '477', '477', '511', '511', '511', '557', '557', '557', '570', '570', '570', '653', '653', '653', '710', '710', '710', '731', '731', '731', '755', '755', '755', '799', '799', '799', '826', '826', '826', '877', '877', '877', '913', '913', '913', '953', '953', '953', '970', '970', '970', '1038', '1038', '1038', '1092', '1092', '1092', '1128', '1128', '1128', '1134', '1134', '1134', '1183', '1183', '1183', '1233', '1233', '1233', '1272', '1272', '1272', '1285', '1285', '1285', '1337', '1337', '1337', '1360', '1360', '1360', '1447', '1447', '1447', '1465', '1465', '1465', '1505', '1505', '1505'], 'NDB_965': ['19', '19', '19', '63', '63', '63', '93', '93', '93', '124', '124', '124', '145', '145', '145', '168', '168', '168', '213', '213', '213', '270', '270', '270', '279', '279', '279', '332', '332', '332', '363', '363', '363', '379', '379', '379', '466', '466', '466', '503', '503', '503', '531', '531', '531', '580', '580', '580', '598', '598', '598', '617', '617', '617', '679', '679', '679', '703', '703', '703', '733', '733', '733', '782', '782', '782', '803', '803', '803', '820', '820', '820', '847', '847', '847', '902', '902', '902', '925', '925', '925', '979', '979', '979', '997', '997', '997', '1048', '1048', '1048', '1082', '1082', '1082', '1142', '1142', '1142', '1180', '1180', '1180', '1220', '1220', '1220', '1237', '1237', '1237', '1264', '1264', '1264', '1304', '1304', '1304', '1321', '1321', '1321', '1452', '1452', '1452', '1493', '1493', '1493', '1527', '1527', '1527'], 'NDB_653': ['20', '20', '20', '66', '66', '66', '112', '112', '112', '150', '150', '150', '192', '192', '192', '218', '218', '218', '254', '254', '254', '277', '277', '277', '290', '290', '290', '306', '306', '306', '357', '357', '357', '386', '386', '386', '442', '442', '442', '470', '470', '470', '522', '522', '522', '549', '549', '549', '590', '590', '590', '651', '651', '651', '657', '657', '657', '699', '699', '699', '731', '731', '731', '774', '774', '774', '810', '810', '810', '841', '841', '841', '875', '875', '875', '922', '922', '922', '957', '957', '957', '972', '972', '972', '1034', '1034', '1034', '1056', '1056', '1056', '1075', '1075', '1075', '1130', '1130', '1130', '1148', '1148', '1148', '1205', '1205', '1205', '1220', '1220', '1220', '1274', '1274', '1274', '1309', '1309', '1309', '1351', '1351', '1351', '1395', '1395', '1395', '1435', '1435', '1435', '1496', '1496', '1496'], 'NDB_977': ['20', '20', '20', '43', '43', '43', '81', '81', '81', '119', '119', '119', '143', '143', '143', '188', '188', '188', '213', '213', '213', '237', '237', '237', '282', '282', '282', '315', '315', '315', '383', '383', '383', '403', '403', '403', '439', '439', '439', '476', '476', '476', '511', '511', '511', '550', '550', '550', '586', '586', '586', '611', '611', '611', '670', '670', '670', '721', '721', '721', '740', '740', '740', '789', '789', '789', '811', '811', '811', '840', '840', '840', '865', '865', '865', '890', '890', '890', '928', '928', '928', '951', '951', '951', '993', '993', '993', '1041', '1041', '1041', '1126', '1126', '1126', '1162', '1162', '1162', '1177', '1177', '1177', '1224', '1224', '1224', '1249', '1249', '1249', '1294', '1294', '1294', '1307', '1307', '1307', '1359', '1359', '1359', '1385', '1385', '1385', '1443', '1443', '1443', '1481', '1481', '1481'], 'NDB_940': ['20', '20', '20', '57', '57', '57', '108', '108', '108', '138', '138', '138', '190', '190', '190', '212', '212', '212', '225', '225', '225', '260', '260', '260', '297', '297', '297', '320', '320', '320', '377', '377', '377', '405', '405', '405', '438', '438', '438', '475', '475', '475', '523', '523', '523', '556', '556', '556', '598', '598', '598', '621', '621', '621', '686', '686', '686', '712', '712', '712', '768', '768', '768', '805', '805', '805', '839', '839', '839', '876', '876', '876', '905', '905', '905', '973', '973', '973', '999', '999', '999', '1058', '1058', '1058', '1070', '1070', '1070', '1076', '1076', '1076', '1131', '1131', '1131', '1173', '1173', '1173', '1182', '1182', '1182', '1227', '1227', '1227', '1273', '1273', '1273', '1325', '1325', '1325', '1346', '1346', '1346', '1384', '1384', '1384', '1439', '1439', '1439', '1475', '1475', '1475', '1515', '1515', '1515'], 'NDB_743': ['21', '21', '21', '74', '74', '74', '112', '112', '112', '145', '145', '145', '190', '190', '190', '195', '195', '195', '224', '224', '224', '269', '269', '269', '310', '310', '310', '341', '341', '341', '390', '390', '390', '411', '411', '411', '448', '448', '448', '478', '478', '478', '507', '507', '507', '533', '533', '533', '584', '584', '584', '596', '596', '596', '646', '646', '646', '702', '702', '702', '730', '730', '730', '790', '790', '790', '825', '825', '825', '846', '846', '846', '889', '889', '889', '907', '907', '907', '983', '983', '983', '1025', '1025', '1025', '1055', '1055', '1055', '1105', '1105', '1105', '1171', '1171', '1171', '1209', '1209', '1209', '1233', '1233', '1233', '1243', '1243', '1243', '1253', '1253', '1253', '1330', '1330', '1330', '1348', '1348', '1348', '1448', '1448', '1448', '1467', '1467', '1467', '1518', '1518', '1518', '1536', '1536', '1536'], 'NDB_917': ['21', '21', '21', '33', '33', '33', '119', '119', '119', '162', '162', '162', '186', '186', '186', '212', '212', '212', '233', '233', '233', '270', '270', '270', '288', '288', '288', '340', '340', '340', '370', '370', '370', '395', '395', '395', '465', '465', '465', '501', '501', '501', '544', '544', '544', '585', '585', '585', '615', '615', '615', '678', '678', '678', '704', '704', '704', '737', '737', '737', '763', '763', '763', '783', '783', '783', '819', '819', '819', '848', '848', '848', '888', '888', '888', '897', '897', '897', '955', '955', '955', '981', '981', '981', '1036', '1036', '1036', '1050', '1050', '1050', '1094', '1094', '1094', '1152', '1152', '1152', '1207', '1207', '1207', '1234', '1234', '1234', '1281', '1281', '1281', '1317', '1317', '1317', '1351', '1351', '1351', '1383', '1383', '1383', '1451', '1451', '1451', '1488', '1488', '1488', '1505', '1505', '1505'], 'NDB_715': ['21', '21', '21', '67', '67', '67', '93', '93', '93', '133', '133', '133', '154', '154', '154', '184', '184', '184', '202', '202', '202', '241', '241', '241', '299', '299', '299', '339', '339', '339', '384', '384', '384', '417', '417', '417', '441', '441', '441', '487', '487', '487', '505', '505', '505', '542', '542', '542', '583', '583', '583', '657', '657', '657', '688', '688', '688', '744', '744', '744', '749', '749', '749', '776', '776', '776', '817', '817', '817', '833', '833', '833', '893', '893', '893', '933', '933', '933', '964', '964', '964', '993', '993', '993', '1020', '1020', '1020', '1069', '1069', '1069', '1121', '1121', '1121', '1141', '1141', '1141', '1187', '1187', '1187', '1230', '1230', '1230', '1247', '1247', '1247', '1284', '1284', '1284', '1333', '1333', '1333', '1388', '1388', '1388', '1434', '1434', '1434', '1485', '1485', '1485', '1502', '1502', '1502'], 'NDB_698': ['22', '22', '22', '38', '38', '38', '84', '84', '84', '123', '123', '123', '149', '149', '149', '198', '198', '198', '244', '244', '244', '259', '259', '259', '308', '308', '308', '317', '317', '317', '353', '353', '353', '418', '418', '418', '431', '431', '431', '457', '457', '457', '491', '491', '491', '552', '552', '552', '585', '585', '585', '645', '645', '645', '685', '685', '685', '712', '712', '712', '778', '778', '778', '807', '807', '807', '829', '829', '829', '867', '867', '867', '913', '913', '913', '967', '967', '967', '1002', '1002', '1002', '1032', '1032', '1032', '1048', '1048', '1048', '1087', '1087', '1087', '1109', '1109', '1109', '1143', '1143', '1143', '1162', '1162', '1162', '1226', '1226', '1226', '1253', '1253', '1253', '1288', '1288', '1288', '1309', '1309', '1309', '1358', '1358', '1358', '1427', '1427', '1427', '1457', '1457', '1457', '1485', '1485', '1485'], 'NDB_1075': ['22', '22', '22', '42', '42', '42', '92', '92', '92', '134', '134', '134', '169', '169', '169', '221', '221', '221', '252', '252', '252', '275', '275', '275', '304', '304', '304', '347', '347', '347', '372', '372', '372', '405', '405', '405', '444', '444', '444', '476', '476', '476', '498', '498', '498', '541', '541', '541', '563', '563', '563', '597', '597', '597', '655', '655', '655', '701', '701', '701', '732', '732', '732', '753', '753', '753', '790', '790', '790', '845', '845', '845', '866', '866', '866', '897', '897', '897', '934', '934', '934', '975', '975', '975', '1024', '1024', '1024', '1051', '1051', '1051', '1090', '1090', '1090', '1146', '1146', '1146', '1193', '1193', '1193', '1238', '1238', '1238', '1245', '1245', '1245', '1282', '1282', '1282', '1342', '1342', '1342', '1380', '1380', '1380', '1440', '1440', '1440', '1472', '1472', '1472', '1510', '1510', '1510'], 'NDB_728': ['23', '23', '23', '40', '40', '40', '107', '107', '107', '158', '158', '158', '177', '177', '177', '198', '198', '198', '212', '212', '212', '242', '242', '242', '284', '284', '284', '324', '324', '324', '356', '356', '356', '376', '376', '376', '440', '440', '440', '480', '480', '480', '519', '519', '519', '539', '539', '539', '558', '558', '558', '576', '576', '576', '624', '624', '624', '679', '679', '679', '687', '687', '687', '701', '701', '701', '751', '751', '751', '792', '792', '792', '835', '835', '835', '875', '875', '875', '894', '894', '894', '911', '911', '911', '953', '953', '953', '961', '961', '961', '1062', '1062', '1062', '1119', '1119', '1119', '1137', '1137', '1137', '1190', '1190', '1190', '1224', '1224', '1224', '1270', '1270', '1270', '1332', '1332', '1332', '1381', '1381', '1381', '1487', '1487', '1487', '1502', '1502', '1502', '1535', '1535', '1535'], 'NDB_946': ['23', '23', '23', '83', '83', '83', '98', '98', '98', '114', '114', '114', '128', '128', '128', '187', '187', '187', '201', '201', '201', '261', '261', '261', '286', '286', '286', '328', '328', '328', '372', '372', '372', '410', '410', '410', '438', '438', '438', '472', '472', '472', '508', '508', '508', '549', '549', '549', '579', '579', '579', '599', '599', '599', '623', '623', '623', '688', '688', '688', '722', '722', '722', '756', '756', '756', '800', '800', '800', '828', '828', '828', '873', '873', '873', '898', '898', '898', '958', '958', '958', '997', '997', '997', '1061', '1061', '1061', '1105', '1105', '1105', '1124', '1124', '1124', '1145', '1145', '1145', '1213', '1213', '1213', '1248', '1248', '1248', '1269', '1269', '1269', '1306', '1306', '1306', '1358', '1358', '1358', '1406', '1406', '1406', '1450', '1450', '1450', '1486', '1486', '1486'], 'NDB_838': ['23', '23', '23', '84', '84', '84', '106', '106', '106', '141', '141', '141', '169', '169', '169', '219', '219', '219', '256', '256', '256', '289', '289', '289', '306', '306', '306', '344', '344', '344', '355', '355', '355', '402', '402', '402', '435', '435', '435', '481', '481', '481', '499', '499', '499', '533', '533', '533', '569', '569', '569', '609', '609', '609', '652', '652', '652', '698', '698', '698', '716', '716', '716', '794', '794', '794', '826', '826', '826', '864', '864', '864', '890', '890', '890', '903', '903', '903', '960', '960', '960', '999', '999', '999', '1036', '1036', '1036', '1074', '1074', '1074', '1097', '1097', '1097', '1141', '1141', '1141', '1195', '1195', '1195', '1280', '1280', '1280', '1321', '1321', '1321', '1347', '1347', '1347', '1404', '1404', '1404', '1453', '1453', '1453', '1464', '1464', '1464', '1494', '1494', '1494', '1525', '1525', '1525'], 'NDB_860': ['24', '24', '24', '77', '77', '77', '121', '121', '121', '141', '141', '141', '189', '189', '189', '198', '198', '198', '238', '238', '238', '312', '312', '312', '345', '345', '345', '365', '365', '365', '393', '393', '393', '422', '422', '422', '460', '460', '460', '489', '489', '489', '510', '510', '510', '551', '551', '551', '556', '556', '556', '601', '601', '601', '655', '655', '655', '693', '693', '693', '725', '725', '725', '787', '787', '787', '812', '812', '812', '816', '816', '816', '860', '860', '860', '919', '919', '919', '972', '972', '972', '993', '993', '993', '1043', '1043', '1043', '1092', '1092', '1092', '1112', '1112', '1112', '1167', '1167', '1167', '1209', '1209', '1209', '1218', '1218', '1218', '1248', '1248', '1248', '1295', '1295', '1295', '1300', '1300', '1300', '1338', '1338', '1338', '1383', '1383', '1383', '1500', '1500', '1500', '1527', '1527', '1527'], 'NDB_752': ['24', '24', '24', '40', '40', '40', '103', '103', '103', '122', '122', '122', '174', '174', '174', '219', '219', '219', '258', '258', '258', '307', '307', '307', '325', '325', '325', '363', '363', '363', '392', '392', '392', '457', '457', '457', '471', '471', '471', '494', '494', '494', '549', '549', '549', '557', '557', '557', '605', '605', '605', '658', '658', '658', '681', '681', '681', '730', '730', '730', '763', '763', '763', '793', '793', '793', '823', '823', '823', '851', '851', '851', '899', '899', '899', '951', '951', '951', '996', '996', '996', '1033', '1033', '1033', '1123', '1123', '1123', '1144', '1144', '1144', '1181', '1181', '1181', '1191', '1191', '1191', '1238', '1238', '1238', '1247', '1247', '1247', '1299', '1299', '1299', '1305', '1305', '1305', '1320', '1320', '1320', '1386', '1386', '1386', '1458', '1458', '1458', '1479', '1479', '1479', '1516', '1516', '1516'], 'NDB_905': ['25', '25', '25', '62', '62', '62', '92', '92', '92', '152', '152', '152', '178', '178', '178', '207', '207', '207', '261', '261', '261', '280', '280', '280', '325', '325', '325', '342', '342', '342', '354', '354', '354', '404', '404', '404', '417', '417', '417', '462', '462', '462', '500', '500', '500', '535', '535', '535', '570', '570', '570', '584', '584', '584', '614', '614', '614', '669', '669', '669', '695', '695', '695', '742', '742', '742', '786', '786', '786', '843', '843', '843', '869', '869', '869', '890', '890', '890', '919', '919', '919', '988', '988', '988', '1047', '1047', '1047', '1058', '1058', '1058', '1114', '1114', '1114', '1135', '1135', '1135', '1178', '1178', '1178', '1203', '1203', '1203', '1277', '1277', '1277', '1302', '1302', '1302', '1345', '1345', '1345', '1381', '1381', '1381', '1438', '1438', '1438', '1461', '1461', '1461', '1493', '1493', '1493'], 'NDB_941': ['26', '26', '26', '87', '87', '87', '102', '102', '102', '146', '146', '146', '167', '167', '167', '221', '221', '221', '248', '248', '248', '291', '291', '291', '305', '305', '305', '325', '325', '325', '359', '359', '359', '388', '388', '388', '446', '446', '446', '474', '474', '474', '509', '509', '509', '527', '527', '527', '562', '562', '562', '599', '599', '599', '649', '649', '649', '665', '665', '665', '696', '696', '696', '733', '733', '733', '777', '777', '777', '802', '802', '802', '860', '860', '860', '895', '895', '895', '912', '912', '912', '963', '963', '963', '991', '991', '991', '1034', '1034', '1034', '1049', '1049', '1049', '1109', '1109', '1109', '1169', '1169', '1169', '1192', '1192', '1192', '1239', '1239', '1239', '1279', '1279', '1279', '1317', '1317', '1317', '1349', '1349', '1349', '1391', '1391', '1391', '1434', '1434', '1434', '1498', '1498', '1498'], 'NDB_881': ['26', '26', '26', '77', '77', '77', '123', '123', '123', '144', '144', '144', '182', '182', '182', '239', '239', '239', '255', '255', '255', '270', '270', '270', '306', '306', '306', '331', '331', '331', '377', '377', '377', '407', '407', '407', '443', '443', '443', '477', '477', '477', '504', '504', '504', '528', '528', '528', '567', '567', '567', '583', '583', '583', '647', '647', '647', '682', '682', '682', '728', '728', '728', '760', '760', '760', '795', '795', '795', '861', '861', '861', '889', '889', '889', '930', '930', '930', '990', '990', '990', '1030', '1030', '1030', '1068', '1068', '1068', '1091', '1091', '1091', '1135', '1135', '1135', '1145', '1145', '1145', '1208', '1208', '1208', '1265', '1265', '1265', '1287', '1287', '1287', '1316', '1316', '1316', '1335', '1335', '1335', '1387', '1387', '1387', '1443', '1443', '1443', '1489', '1489', '1489', '1510', '1510', '1510'], 'NDB_713': ['27', '27', '27', '44', '44', '44', '103', '103', '103', '157', '157', '157', '171', '171', '171', '202', '202', '202', '233', '233', '233', '257', '257', '257', '297', '297', '297', '319', '319', '319', '360', '360', '360', '389', '389', '389', '428', '428', '428', '450', '450', '450', '502', '502', '502', '541', '541', '541', '591', '591', '591', '611', '611', '611', '629', '629', '629', '689', '689', '689', '716', '716', '716', '751', '751', '751', '779', '779', '779', '829', '829', '829', '874', '874', '874', '904', '904', '904', '972', '972', '972', '991', '991', '991', '1030', '1030', '1030', '1111', '1111', '1111', '1127', '1127', '1127', '1178', '1178', '1178', '1229', '1229', '1229', '1259', '1259', '1259', '1298', '1298', '1298', '1341', '1341', '1341', '1405', '1405', '1405', '1452', '1452', '1452', '1497', '1497', '1497', '1514', '1514', '1514', '1541', '1541', '1541'], 'NDB_929': ['27', '27', '27', '75', '75', '75', '104', '104', '104', '167', '167', '167', '185', '185', '185', '197', '197', '197', '235', '235', '235', '273', '273', '273', '287', '287', '287', '332', '332', '332', '362', '362', '362', '381', '381', '381', '443', '443', '443', '472', '472', '472', '518', '518', '518', '572', '572', '572', '606', '606', '606', '622', '622', '622', '678', '678', '678', '702', '702', '702', '742', '742', '742', '755', '755', '755', '774', '774', '774', '856', '856', '856', '876', '876', '876', '923', '923', '923', '966', '966', '966', '994', '994', '994', '1023', '1023', '1023', '1065', '1065', '1065', '1110', '1110', '1110', '1176', '1176', '1176', '1222', '1222', '1222', '1240', '1240', '1240', '1270', '1270', '1270', '1300', '1300', '1300', '1339', '1339', '1339', '1388', '1388', '1388', '1468', '1468', '1468', '1516', '1516', '1516', '1534', '1534', '1534'], 'NDB_784': ['28', '28', '28', '47', '47', '47', '78', '78', '78', '131', '131', '131', '152', '152', '152', '191', '191', '191', '240', '240', '240', '265', '265', '265', '303', '303', '303', '321', '321', '321', '384', '384', '384', '409', '409', '409', '442', '442', '442', '484', '484', '484', '520', '520', '520', '534', '534', '534', '556', '556', '556', '606', '606', '606', '654', '654', '654', '701', '701', '701', '730', '730', '730', '761', '761', '761', '785', '785', '785', '858', '858', '858', '918', '918', '918', '935', '935', '935', '954', '954', '954', '1000', '1000', '1000', '1032', '1032', '1032', '1079', '1079', '1079', '1125', '1125', '1125', '1168', '1168', '1168', '1237', '1237', '1237', '1244', '1244', '1244', '1259', '1259', '1259', '1324', '1324', '1324', '1349', '1349', '1349', '1394', '1394', '1394', '1443', '1443', '1443', '1486', '1486', '1486', '1505', '1505', '1505'], 'NDB_875': ['28', '28', '28', '70', '70', '70', '104', '104', '104', '155', '155', '155', '193', '193', '193', '208', '208', '208', '224', '224', '224', '250', '250', '250', '285', '285', '285', '329', '329', '329', '361', '361', '361', '377', '377', '377', '428', '428', '428', '465', '465', '465', '495', '495', '495', '532', '532', '532', '589', '589', '589', '625', '625', '625', '679', '679', '679', '720', '720', '720', '741', '741', '741', '765', '765', '765', '812', '812', '812', '826', '826', '826', '878', '878', '878', '916', '916', '916', '951', '951', '951', '969', '969', '969', '1034', '1034', '1034', '1087', '1087', '1087', '1124', '1124', '1124', '1139', '1139', '1139', '1212', '1212', '1212', '1236', '1236', '1236', '1258', '1258', '1258', '1291', '1291', '1291', '1334', '1334', '1334', '1392', '1392', '1392', '1459', '1459', '1459', '1474', '1474', '1474', '1513', '1513', '1513'], 'NDB_751': ['28', '28', '28', '79', '79', '79', '110', '110', '110', '129', '129', '129', '162', '162', '162', '206', '206', '206', '238', '238', '238', '272', '272', '272', '283', '283', '283', '316', '316', '316', '374', '374', '374', '417', '417', '417', '451', '451', '451', '479', '479', '479', '526', '526', '526', '565', '565', '565', '608', '608', '608', '651', '651', '651', '689', '689', '689', '707', '707', '707', '752', '752', '752', '770', '770', '770', '835', '835', '835', '873', '873', '873', '896', '896', '896', '932', '932', '932', '950', '950', '950', '970', '970', '970', '1048', '1048', '1048', '1115', '1115', '1115', '1146', '1146', '1146', '1191', '1191', '1191', '1216', '1216', '1216', '1251', '1251', '1251', '1290', '1290', '1290', '1325', '1325', '1325', '1347', '1347', '1347', '1378', '1378', '1378', '1441', '1441', '1441', '1477', '1477', '1477', '1504', '1504', '1504', '1533', '1533', '1533'], 'NDB_647': ['29', '29', '29', '91', '91', '91', '105', '105', '105', '153', '153', '153', '191', '191', '191', '199', '199', '199', '227', '227', '227', '244', '244', '244', '280', '280', '280', '334', '334', '334', '361', '361', '361', '390', '390', '390', '449', '449', '449', '485', '485', '485', '523', '523', '523', '551', '551', '551', '588', '588', '588', '627', '627', '627', '672', '672', '672', '696', '696', '696', '724', '724', '724', '767', '767', '767', '809', '809', '809', '840', '840', '840', '872', '872', '872', '901', '901', '901', '934', '934', '934', '955', '955', '955', '966', '966', '966', '1046', '1046', '1046', '1119', '1119', '1119', '1157', '1157', '1157', '1216', '1216', '1216', '1263', '1263', '1263', '1280', '1280', '1280', '1316', '1316', '1316', '1357', '1357', '1357', '1406', '1406', '1406', '1456', '1456', '1456', '1499', '1499', '1499', '1541', '1541', '1541'], 'NDB_725': ['30', '30', '30', '70', '70', '70', '101', '101', '101', '132', '132', '132', '179', '179', '179', '223', '223', '223', '246', '246', '246', '274', '274', '274', '291', '291', '291', '345', '345', '345', '385', '385', '385', '405', '405', '405', '450', '450', '450', '467', '467', '467', '518', '518', '518', '534', '534', '534', '588', '588', '588', '619', '619', '619', '650', '650', '650', '715', '715', '715', '745', '745', '745', '758', '758', '758', '770', '770', '770', '833', '833', '833', '877', '877', '877', '910', '910', '910', '958', '958', '958', '996', '996', '996', '1054', '1054', '1054', '1074', '1074', '1074', '1130', '1130', '1130', '1174', '1174', '1174', '1210', '1210', '1210', '1252', '1252', '1252', '1297', '1297', '1297', '1335', '1335', '1335', '1402', '1402', '1402', '1430', '1430', '1430', '1493', '1493', '1493', '1509', '1509', '1509', '1535', '1535', '1535'], 'NDB_994': ['30', '30', '30', '79', '79', '79', '108', '108', '108', '133', '133', '133', '158', '158', '158', '205', '205', '205', '251', '251', '251', '312', '312', '312', '338', '338', '338', '394', '394', '394', '415', '415', '415', '447', '447', '447', '459', '459', '459', '495', '495', '495', '509', '509', '509', '592', '592', '592', '656', '656', '656', '680', '680', '680', '724', '724', '724', '778', '778', '778', '803', '803', '803', '825', '825', '825', '851', '851', '851', '917', '917', '917', '949', '949', '949', '987', '987', '987', '1045', '1045', '1045', '1063', '1063', '1063', '1106', '1106', '1106', '1129', '1129', '1129', '1163', '1163', '1163', '1183', '1183', '1183', '1206', '1206', '1206', '1244', '1244', '1244', '1265', '1265', '1265', '1314', '1314', '1314', '1351', '1351', '1351', '1382', '1382', '1382', '1438', '1438', '1438', '1491', '1491', '1491', '1511', '1511', '1511'], 'NDB_778': ['32', '32', '32', '73', '73', '73', '96', '96', '96', '125', '125', '125', '177', '177', '177', '204', '204', '204', '243', '243', '243', '258', '258', '258', '298', '298', '298', '348', '348', '348', '365', '365', '365', '410', '410', '410', '429', '429', '429', '455', '455', '455', '498', '498', '498', '542', '542', '542', '566', '566', '566', '594', '594', '594', '618', '618', '618', '669', '669', '669', '703', '703', '703', '734', '734', '734', '759', '759', '759', '785', '785', '785', '852', '852', '852', '927', '927', '927', '990', '990', '990', '1044', '1044', '1044', '1055', '1055', '1055', '1089', '1089', '1089', '1152', '1152', '1152', '1197', '1197', '1197', '1236', '1236', '1236', '1256', '1256', '1256', '1298', '1298', '1298', '1339', '1339', '1339', '1356', '1356', '1356', '1402', '1402', '1402', '1473', '1473', '1473', '1496', '1496', '1496', '1537', '1537', '1537'], 'NDB_694': ['32', '32', '32', '69', '69', '69', '85', '85', '85', '127', '127', '127', '174', '174', '174', '221', '221', '221', '268', '268', '268', '277', '277', '277', '313', '313', '313', '364', '364', '364', '393', '393', '393', '431', '431', '431', '461', '461', '461', '519', '519', '519', '543', '543', '543', '587', '587', '587', '612', '612', '612', '670', '670', '670', '724', '724', '724', '745', '745', '745', '782', '782', '782', '804', '804', '804', '836', '836', '836', '848', '848', '848', '916', '916', '916', '954', '954', '954', '982', '982', '982', '1031', '1031', '1031', '1071', '1071', '1071', '1120', '1120', '1120', '1141', '1141', '1141', '1173', '1173', '1173', '1203', '1203', '1203', '1222', '1222', '1222', '1271', '1271', '1271', '1323', '1323', '1323', '1348', '1348', '1348', '1393', '1393', '1393', '1463', '1463', '1463', '1501', '1501', '1501'], 'NDB_824': ['33', '33', '33', '88', '88', '88', '107', '107', '107', '140', '140', '140', '157', '157', '157', '201', '201', '201', '216', '216', '216', '235', '235', '235', '309', '309', '309', '331', '331', '331', '371', '371', '371', '404', '404', '404', '467', '467', '467', '493', '493', '493', '532', '532', '532', '562', '562', '562', '600', '600', '600', '627', '627', '627', '674', '674', '674', '703', '703', '703', '735', '735', '735', '761', '761', '761', '796', '796', '796', '851', '851', '851', '864', '864', '864', '900', '900', '900', '932', '932', '932', '985', '985', '985', '1031', '1031', '1031', '1070', '1070', '1070', '1121', '1121', '1121', '1162', '1162', '1162', '1179', '1179', '1179', '1194', '1194', '1194', '1282', '1282', '1282', '1315', '1315', '1315', '1353', '1353', '1353', '1379', '1379', '1379', '1429', '1429', '1429', '1466', '1466', '1466', '1521', '1521', '1521'], 'NDB_773': ['35', '35', '35', '57', '57', '57', '116', '116', '116', '145', '145', '145', '165', '165', '165', '218', '218', '218', '256', '256', '256', '315', '315', '315', '337', '337', '337', '375', '375', '375', '394', '394', '394', '421', '421', '421', '425', '425', '425', '462', '462', '462', '524', '524', '524', '574', '574', '574', '622', '622', '622', '655', '655', '655', '690', '690', '690', '729', '729', '729', '775', '775', '775', '809', '809', '809', '829', '829', '829', '849', '849', '849', '899', '899', '899', '932', '932', '932', '968', '968', '968', '980', '980', '980', '1040', '1040', '1040', '1091', '1091', '1091', '1140', '1140', '1140', '1186', '1186', '1186', '1225', '1225', '1225', '1260', '1260', '1260', '1297', '1297', '1297', '1310', '1310', '1310', '1356', '1356', '1356', '1389', '1389', '1389', '1434', '1434', '1434', '1463', '1463', '1463', '1512', '1512', '1512'], 'NDB_1033': ['35', '35', '35', '65', '65', '65', '118', '118', '118', '153', '153', '153', '184', '184', '184', '196', '196', '196', '209', '209', '209', '251', '251', '251', '293', '293', '293', '343', '343', '343', '358', '358', '358', '391', '391', '391', '450', '450', '450', '484', '484', '484', '497', '497', '497', '536', '536', '536', '584', '584', '584', '617', '617', '617', '649', '649', '649', '713', '713', '713', '731', '731', '731', '756', '756', '756', '794', '794', '794', '827', '827', '827', '865', '865', '865', '888', '888', '888', '929', '929', '929', '973', '973', '973', '1002', '1002', '1002', '1024', '1024', '1024', '1042', '1042', '1042', '1108', '1108', '1108', '1167', '1167', '1167', '1196', '1196', '1196', '1228', '1228', '1228', '1258', '1258', '1258', '1339', '1339', '1339', '1379', '1379', '1379', '1436', '1436', '1436', '1476', '1476', '1476', '1490', '1490', '1490'], 'NDB_880': ['35', '35', '35', '81', '81', '81', '109', '109', '109', '139', '139', '139', '180', '180', '180', '227', '227', '227', '243', '243', '243', '271', '271', '271', '297', '297', '297', '341', '341', '341', '380', '380', '380', '409', '409', '409', '464', '464', '464', '481', '481', '481', '493', '493', '493', '546', '546', '546', '554', '554', '554', '564', '564', '564', '616', '616', '616', '673', '673', '673', '705', '705', '705', '732', '732', '732', '752', '752', '752', '787', '787', '787', '838', '838', '838', '870', '870', '870', '891', '891', '891', '916', '916', '916', '987', '987', '987', '1033', '1033', '1033', '1049', '1049', '1049', '1093', '1093', '1093', '1135', '1135', '1135', '1166', '1166', '1166', '1204', '1204', '1204', '1281', '1281', '1281', '1306', '1306', '1306', '1363', '1363', '1363', '1388', '1388', '1388', '1457', '1457', '1457', '1482', '1482', '1482'], 'NDB_826': ['36', '36', '36', '65', '65', '65', '144', '144', '144', '166', '166', '166', '174', '174', '174', '200', '200', '200', '229', '229', '229', '265', '265', '265', '279', '279', '279', '329', '329', '329', '340', '340', '340', '367', '367', '367', '412', '412', '412', '456', '456', '456', '474', '474', '474', '492', '492', '492', '542', '542', '542', '571', '571', '571', '623', '623', '623', '663', '663', '663', '727', '727', '727', '775', '775', '775', '811', '811', '811', '844', '844', '844', '868', '868', '868', '903', '903', '903', '967', '967', '967', '1046', '1046', '1046', '1122', '1122', '1122', '1138', '1138', '1138', '1182', '1182', '1182', '1210', '1210', '1210', '1232', '1232', '1232', '1255', '1255', '1255', '1290', '1290', '1290', '1300', '1300', '1300', '1337', '1337', '1337', '1380', '1380', '1380', '1437', '1437', '1437', '1491', '1491', '1491', '1521', '1521', '1521'], 'NDB_865': ['36', '36', '36', '71', '71', '71', '104', '104', '104', '133', '133', '133', '172', '172', '172', '246', '246', '246', '262', '262', '262', '307', '307', '307', '336', '336', '336', '356', '356', '356', '375', '375', '375', '432', '432', '432', '458', '458', '458', '493', '493', '493', '545', '545', '545', '563', '563', '563', '585', '585', '585', '646', '646', '646', '670', '670', '670', '693', '693', '693', '723', '723', '723', '785', '785', '785', '805', '805', '805', '822', '822', '822', '862', '862', '862', '908', '908', '908', '965', '965', '965', '1047', '1047', '1047', '1085', '1085', '1085', '1153', '1153', '1153', '1192', '1192', '1192', '1220', '1220', '1220', '1250', '1250', '1250', '1269', '1269', '1269', '1322', '1322', '1322', '1355', '1355', '1355', '1403', '1403', '1403', '1449', '1449', '1449', '1462', '1462', '1462', '1489', '1489', '1489', '1541', '1541', '1541'], 'NDB_1027': ['36', '36', '36', '73', '73', '73', '116', '116', '116', '156', '156', '156', '193', '193', '193', '211', '211', '211', '228', '228', '228', '295', '295', '295', '313', '313', '313', '346', '346', '346', '352', '352', '352', '376', '376', '376', '416', '416', '416', '435', '435', '435', '487', '487', '487', '500', '500', '500', '534', '534', '534', '573', '573', '573', '645', '645', '645', '668', '668', '668', '697', '697', '697', '743', '743', '743', '753', '753', '753', '799', '799', '799', '861', '861', '861', '925', '925', '925', '966', '966', '966', '983', '983', '983', '1060', '1060', '1060', '1075', '1075', '1075', '1111', '1111', '1111', '1160', '1160', '1160', '1227', '1227', '1227', '1242', '1242', '1242', '1281', '1281', '1281', '1340', '1340', '1340', '1400', '1400', '1400', '1450', '1450', '1450', '1470', '1470', '1470', '1504', '1504', '1504', '1529', '1529', '1529'], 'NDB_817': ['37', '37', '37', '66', '66', '66', '115', '115', '115', '167', '167', '167', '188', '188', '188', '220', '220', '220', '255', '255', '255', '274', '274', '274', '295', '295', '295', '349', '349', '349', '358', '358', '358', '379', '379', '379', '430', '430', '430', '468', '468', '468', '501', '501', '501', '546', '546', '546', '579', '579', '579', '603', '603', '603', '662', '662', '662', '729', '729', '729', '778', '778', '778', '805', '805', '805', '836', '836', '836', '863', '863', '863', '900', '900', '900', '935', '935', '935', '975', '975', '975', '1003', '1003', '1003', '1044', '1044', '1044', '1117', '1117', '1117', '1138', '1138', '1138', '1195', '1195', '1195', '1209', '1209', '1209', '1251', '1251', '1251', '1299', '1299', '1299', '1333', '1333', '1333', '1345', '1345', '1345', '1454', '1454', '1454', '1478', '1478', '1478', '1519', '1519', '1519', '1542', '1542', '1542'], 'NDB_971': ['37', '37', '37', '60', '60', '60', '108', '108', '108', '149', '149', '149', '181', '181', '181', '210', '210', '210', '233', '233', '233', '267', '267', '267', '298', '298', '298', '323', '323', '323', '362', '362', '362', '387', '387', '387', '436', '436', '436', '452', '452', '452', '496', '496', '496', '536', '536', '536', '563', '563', '563', '586', '586', '586', '616', '616', '616', '667', '667', '667', '698', '698', '698', '725', '725', '725', '765', '765', '765', '806', '806', '806', '843', '843', '843', '875', '875', '875', '898', '898', '898', '931', '931', '931', '983', '983', '983', '1020', '1020', '1020', '1071', '1071', '1071', '1115', '1115', '1115', '1169', '1169', '1169', '1214', '1214', '1214', '1237', '1237', '1237', '1276', '1276', '1276', '1343', '1343', '1343', '1387', '1387', '1387', '1437', '1437', '1437', '1466', '1466', '1466', '1517', '1517', '1517'], 'NDB_721': ['37', '37', '37', '80', '80', '80', '128', '128', '128', '168', '168', '168', '186', '186', '186', '207', '207', '207', '223', '223', '223', '239', '239', '239', '293', '293', '293', '333', '333', '333', '389', '389', '389', '411', '411', '411', '422', '422', '422', '470', '470', '470', '519', '519', '519', '554', '554', '554', '594', '594', '594', '613', '613', '613', '674', '674', '674', '680', '680', '680', '707', '707', '707', '764', '764', '764', '809', '809', '809', '814', '814', '814', '839', '839', '839', '893', '893', '893', '974', '974', '974', '1039', '1039', '1039', '1065', '1065', '1065', '1125', '1125', '1125', '1134', '1134', '1134', '1153', '1153', '1153', '1193', '1193', '1193', '1249', '1249', '1249', '1293', '1293', '1293', '1311', '1311', '1311', '1361', '1361', '1361', '1391', '1391', '1391', '1427', '1427', '1427', '1469', '1469', '1469', '1513', '1513', '1513'], 'NDB_896': ['38', '38', '38', '76', '76', '76', '90', '90', '90', '152', '152', '152', '166', '166', '166', '204', '204', '204', '253', '253', '253', '276', '276', '276', '289', '289', '289', '327', '327', '327', '358', '358', '358', '383', '383', '383', '428', '428', '428', '441', '441', '441', '526', '526', '526', '588', '588', '588', '605', '605', '605', '659', '659', '659', '705', '705', '705', '743', '743', '743', '748', '748', '748', '800', '800', '800', '814', '814', '814', '857', '857', '857', '908', '908', '908', '923', '923', '923', '961', '961', '961', '1004', '1004', '1004', '1029', '1029', '1029', '1067', '1067', '1067', '1078', '1078', '1078', '1169', '1169', '1169', '1215', '1215', '1215', '1241', '1241', '1241', '1260', '1260', '1260', '1314', '1314', '1314', '1353', '1353', '1353', '1393', '1393', '1393', '1435', '1435', '1435', '1467', '1467', '1467', '1507', '1507', '1507'], 'NDB_863': ['40', '40', '40', '64', '64', '64', '100', '100', '100', '124', '124', '124', '166', '166', '166', '225', '225', '225', '244', '244', '244', '263', '263', '263', '283', '283', '283', '347', '347', '347', '368', '368', '368', '407', '407', '407', '448', '448', '448', '484', '484', '484', '512', '512', '512', '550', '550', '550', '555', '555', '555', '607', '607', '607', '648', '648', '648', '684', '684', '684', '719', '719', '719', '749', '749', '749', '791', '791', '791', '824', '824', '824', '836', '836', '836', '920', '920', '920', '963', '963', '963', '1054', '1054', '1054', '1075', '1075', '1075', '1106', '1106', '1106', '1152', '1152', '1152', '1178', '1178', '1178', '1214', '1214', '1214', '1257', '1257', '1257', '1292', '1292', '1292', '1310', '1310', '1310', '1355', '1355', '1355', '1404', '1404', '1404', '1447', '1447', '1447', '1482', '1482', '1482', '1520', '1520', '1520'], 'NDB_856': ['42', '42', '42', '72', '72', '72', '90', '90', '90', '125', '125', '125', '139', '139', '139', '190', '190', '190', '206', '206', '206', '249', '249', '249', '296', '296', '296', '332', '332', '332', '352', '352', '352', '407', '407', '407', '430', '430', '430', '459', '459', '459', '520', '520', '520', '581', '581', '581', '595', '595', '595', '614', '614', '614', '671', '671', '671', '685', '685', '685', '744', '744', '744', '789', '789', '789', '810', '810', '810', '815', '815', '815', '853', '853', '853', '912', '912', '912', '929', '929', '929', '980', '980', '980', '1064', '1064', '1064', '1094', '1094', '1094', '1144', '1144', '1144', '1229', '1229', '1229', '1246', '1246', '1246', '1276', '1276', '1276', '1332', '1332', '1332', '1361', '1361', '1361', '1403', '1403', '1403', '1430', '1430', '1430', '1459', '1459', '1459', '1464', '1464', '1464', '1499', '1499', '1499'], 'NDB_868': ['44', '44', '44', '73', '73', '73', '115', '115', '115', '158', '158', '158', '191', '191', '191', '217', '217', '217', '238', '238', '238', '264', '264', '264', '292', '292', '292', '328', '328', '328', '357', '357', '357', '392', '392', '392', '439', '439', '439', '478', '478', '478', '505', '505', '505', '535', '535', '535', '554', '554', '554', '581', '581', '581', '620', '620', '620', '666', '666', '666', '712', '712', '712', '735', '735', '735', '798', '798', '798', '834', '834', '834', '871', '871', '871', '910', '910', '910', '952', '952', '952', '979', '979', '979', '1023', '1023', '1023', '1068', '1068', '1068', '1107', '1107', '1107', '1122', '1122', '1122', '1139', '1139', '1139', '1165', '1165', '1165', '1214', '1214', '1214', '1275', '1275', '1275', '1336', '1336', '1336', '1344', '1344', '1344', '1399', '1399', '1399', '1440', '1440', '1440', '1498', '1498', '1498'], 'NDB_706': ['57', '57', '57', '68', '68', '68', '110', '110', '110', '137', '137', '137', '178', '178', '178', '215', '215', '215', '253', '253', '253', '311', '311', '311', '336', '336', '336', '351', '351', '351', '396', '396', '396', '459', '459', '459', '471', '471', '471', '496', '496', '496', '532', '532', '532', '580', '580', '580', '647', '647', '647', '678', '678', '678', '699', '699', '699', '722', '722', '722', '794', '794', '794', '804', '804', '804', '852', '852', '852', '863', '863', '863', '926', '926', '926', '962', '962', '962', '986', '986', '986', '1041', '1041', '1041', '1069', '1069', '1069', '1093', '1093', '1093', '1127', '1127', '1127', '1134', '1134', '1134', '1157', '1157', '1157', '1217', '1217', '1217', '1261', '1261', '1261', '1319', '1319', '1319', '1349', '1349', '1349', '1400', '1400', '1400', '1480', '1480', '1480', '1520', '1520', '1520', '1535', '1535', '1535'], 'NDB_716': ['60', '60', '60', '70', '70', '70', '107', '107', '107', '144', '144', '144', '189', '189', '189', '204', '204', '204', '257', '257', '257', '310', '310', '310', '336', '336', '336', '366', '366', '366', '392', '392', '392', '461', '461', '461', '472', '472', '472', '516', '516', '516', '537', '537', '537', '565', '565', '565', '621', '621', '621', '664', '664', '664', '697', '697', '697', '721', '721', '721', '762', '762', '762', '777', '777', '777', '821', '821', '821', '845', '845', '845', '892', '892', '892', '913', '913', '913', '957', '957', '957', '968', '968', '968', '1037', '1037', '1037', '1053', '1053', '1053', '1106', '1106', '1106', '1125', '1125', '1125', '1147', '1147', '1147', '1204', '1204', '1204', '1278', '1278', '1278', '1303', '1303', '1303', '1345', '1345', '1345', '1409', '1409', '1409', '1445', '1445', '1445', '1488', '1488', '1488', '1499', '1499', '1499'], 'NDB_680': ['60', '60', '60', '68', '68', '68', '105', '105', '105', '160', '160', '160', '183', '183', '183', '205', '205', '205', '247', '247', '247', '313', '313', '313', '326', '326', '326', '344', '344', '344', '359', '359', '359', '374', '374', '374', '429', '429', '429', '466', '466', '466', '512', '512', '512', '545', '545', '545', '568', '568', '568', '628', '628', '628', '671', '671', '671', '720', '720', '720', '732', '732', '732', '750', '750', '750', '798', '798', '798', '841', '841', '841', '877', '877', '877', '899', '899', '899', '911', '911', '911', '981', '981', '981', '994', '994', '994', '1029', '1029', '1029', '1108', '1108', '1108', '1145', '1145', '1145', '1188', '1188', '1188', '1211', '1211', '1211', '1253', '1253', '1253', '1294', '1294', '1294', '1333', '1333', '1333', '1362', '1362', '1362', '1429', '1429', '1429', '1497', '1497', '1497', '1530', '1530', '1530'], 'NDB_641': ['64', '64', '64', '86', '86', '86', '137', '137', '137', '165', '165', '165', '183', '183', '183', '208', '208', '208', '262', '262', '262', '284', '284', '284', '304', '304', '304', '349', '349', '349', '371', '371', '371', '403', '403', '403', '455', '455', '455', '481', '481', '481', '490', '490', '490', '527', '527', '527', '582', '582', '582', '593', '593', '593', '623', '623', '623', '664', '664', '664', '708', '708', '708', '737', '737', '737', '795', '795', '795', '815', '815', '815', '854', '854', '854', '893', '893', '893', '909', '909', '909', '973', '973', '973', '1025', '1025', '1025', '1056', '1056', '1056', '1110', '1110', '1110', '1156', '1156', '1156', '1184', '1184', '1184', '1216', '1216', '1216', '1275', '1275', '1275', '1304', '1304', '1304', '1338', '1338', '1338', '1386', '1386', '1386', '1438', '1438', '1438', '1470', '1470', '1470', '1517', '1517', '1517'], 'NDB_862': ['64', '64', '64', '78', '78', '78', '103', '103', '103', '153', '153', '153', '185', '185', '185', '229', '229', '229', '255', '255', '255', '275', '275', '275', '290', '290', '290', '338', '338', '338', '373', '373', '373', '406', '406', '406', '436', '436', '436', '479', '479', '479', '521', '521', '521', '540', '540', '540', '574', '574', '574', '602', '602', '602', '659', '659', '659', '683', '683', '683', '710', '710', '710', '792', '792', '792', '819', '819', '819', '852', '852', '852', '895', '895', '895', '933', '933', '933', '952', '952', '952', '992', '992', '992', '1057', '1057', '1057', '1124', '1124', '1124', '1171', '1171', '1171', '1203', '1203', '1203', '1221', '1221', '1221', '1246', '1246', '1246', '1278', '1278', '1278', '1301', '1301', '1301', '1315', '1315', '1315', '1401', '1401', '1401', '1457', '1457', '1457', '1481', '1481', '1481', '1503', '1503', '1503', '1527', '1527', '1527'], 'NDB_872': ['66', '66', '66', '78', '78', '78', '117', '117', '117', '129', '129', '129', '187', '187', '187', '230', '230', '230', '250', '250', '250', '268', '268', '268', '298', '298', '298', '342', '342', '342', '381', '381', '381', '400', '400', '400', '423', '423', '423', '456', '456', '456', '525', '525', '525', '550', '550', '550', '592', '592', '592', '645', '645', '645', '677', '677', '677', '706', '706', '706', '737', '737', '737', '798', '798', '798', '846', '846', '846', '860', '860', '860', '904', '904', '904', '974', '974', '974', '1001', '1001', '1001', '1024', '1024', '1024', '1053', '1053', '1053', '1085', '1085', '1085', '1131', '1131', '1131', '1155', '1155', '1155', '1187', '1187', '1187', '1231', '1231', '1231', '1285', '1285', '1285', '1319', '1319', '1319', '1353', '1353', '1353', '1390', '1390', '1390', '1464', '1464', '1464', '1509', '1509', '1509', '1538', '1538', '1538']}\n",
      " > Done.\n",
      "-- inverting region_to_readout dict\n",
      "NDB_1027 -> ['1060', '1075', '1111', '116', '1160', '1227', '1242', '1281', '1340', '1400', '1450', '1470', '1504', '1529', '156', '193', '211', '228', '295', '313', '346', '352', '36', '376', '416', '435', '487', '500', '534', '573', '645', '668', '697', '73', '743', '753', '799', '861', '925', '966', '983']\n",
      "NDB_1028 -> ['1051', '1066', '1080', '1131', '1166', '1218', '1272', '130', '1303', '1344', '1398', '1454', '1471', '1504', '1537', '17', '171', '197', '247', '309', '338', '34', '375', '396', '4', '427', '470', '491', '520', '561', '578', '617', '672', '681', '698', '745', '783', '800', '854', '920', '969']\n",
      "NDB_1033 -> ['1002', '1024', '1042', '1108', '1167', '118', '1196', '1228', '1258', '1339', '1379', '1436', '1476', '1490', '153', '184', '196', '209', '251', '293', '343', '35', '358', '391', '450', '484', '497', '536', '584', '617', '649', '65', '713', '731', '756', '794', '827', '865', '888', '929', '973']\n",
      "NDB_1060 -> ['1060', '1117', '1172', '1194', '1212', '1248', '1263', '1318', '1354', '1427', '148', '1492', '161', '197', '225', '272', '302', '330', '366', '403', '464', '507', '535', '558', '595', '6', '619', '663', '706', '738', '759', '76', '782', '823', '862', '888', '89', '917', '962', '992']\n",
      "NDB_1066 -> ['1', '1036', '1073', '1114', '1157', '121', '1232', '1241', '1269', '1285', '1343', '1378', '1409', '1459', '147', '1472', '1518', '175', '213', '228', '27', '278', '303', '343', '364', '398', '469', '482', '494', '504', '560', '578', '628', '729', '749', '792', '834', '86', '867', '921', '987']\n",
      "NDB_1075 -> ['1024', '1051', '1090', '1146', '1193', '1238', '1245', '1282', '134', '1342', '1380', '1440', '1472', '1510', '169', '22', '221', '252', '275', '304', '347', '372', '405', '42', '444', '476', '498', '541', '563', '597', '655', '701', '732', '753', '790', '845', '866', '897', '92', '934', '975']\n",
      "NDB_631 -> ['1', '1001', '1032', '1058', '1120', '114', '1170', '1235', '1252', '1294', '1316', '1334', '14', '1408', '1446', '1478', '1516', '1536', '177', '194', '215', '236', '288', '318', '368', '414', '444', '473', '499', '536', '574', '600', '63', '651', '693', '708', '784', '802', '853', '891', '956']\n",
      "NDB_634 -> ['1001', '1040', '1078', '1128', '1174', '1197', '1240', '1245', '1261', '1302', '1324', '1392', '1441', '1471', '1487', '1530', '163', '175', '219', '251', '282', '305', '349', '389', '412', '449', '46', '486', '508', '528', '581', '616', '648', '7', '711', '738', '788', '819', '837', '905', '94', '957']\n",
      "NDB_637 -> ['10', '1045', '1072', '1119', '1143', '1184', '1229', '1262', '1289', '129', '1323', '1350', '1399', '1450', '1476', '148', '1512', '173', '210', '271', '290', '314', '345', '396', '412', '463', '473', '498', '533', '562', '598', '654', '662', '682', '71', '719', '780', '801', '859', '923', '964']\n",
      "NDB_641 -> ['1025', '1056', '1110', '1156', '1184', '1216', '1275', '1304', '1338', '137', '1386', '1438', '1470', '1517', '165', '183', '208', '262', '284', '304', '349', '371', '403', '455', '481', '490', '527', '582', '593', '623', '64', '664', '708', '737', '795', '815', '854', '86', '893', '909', '973']\n",
      "NDB_643 -> ['1047', '1076', '1132', '1180', '1232', '1252', '1298', '1312', '1331', '1401', '1429', '1458', '1485', '1500', '155', '169', '17', '234', '269', '281', '311', '346', '381', '399', '453', '485', '5', '522', '579', '620', '649', '690', '709', '752', '801', '821', '857', '894', '94', '949', '971']\n",
      "NDB_647 -> ['1046', '105', '1119', '1157', '1216', '1263', '1280', '1316', '1357', '1406', '1456', '1499', '153', '1541', '191', '199', '227', '244', '280', '29', '334', '361', '390', '449', '485', '523', '551', '588', '627', '672', '696', '724', '767', '809', '840', '872', '901', '91', '934', '955', '966']\n",
      "NDB_652 -> ['1052', '1116', '1136', '1186', '120', '1210', '1247', '1288', '1341', '141', '1444', '1471', '15', '1511', '1542', '185', '196', '243', '268', '288', '333', '388', '404', '439', '483', '507', '531', '576', '590', '628', '668', '686', '719', '766', '808', '845', '873', '90', '901', '926', '984']\n",
      "NDB_653 -> ['1034', '1056', '1075', '112', '1130', '1148', '1205', '1220', '1274', '1309', '1351', '1395', '1435', '1496', '150', '192', '20', '218', '254', '277', '290', '306', '357', '386', '442', '470', '522', '549', '590', '651', '657', '66', '699', '731', '774', '810', '841', '875', '922', '957', '972']\n",
      "NDB_661 -> ['1005', '1052', '1080', '112', '1172', '1204', '1235', '1261', '1287', '1307', '1357', '1403', '1437', '1472', '1512', '1525', '154', '170', '19', '214', '259', '284', '30', '319', '354', '401', '446', '479', '525', '568', '605', '660', '694', '714', '756', '769', '8', '816', '839', '915', '927', '954']\n",
      "NDB_665 -> ['1049', '1067', '1117', '1147', '1179', '1225', '1245', '1254', '127', '1337', '1395', '1444', '1470', '1508', '164', '209', '223', '25', '269', '292', '321', '368', '4', '402', '458', '482', '495', '544', '591', '660', '682', '717', '768', '806', '817', '840', '911', '924', '95', '950', '997']\n",
      "NDB_680 -> ['1029', '105', '1108', '1145', '1188', '1211', '1253', '1294', '1333', '1362', '1429', '1497', '1530', '160', '183', '205', '247', '313', '326', '344', '359', '374', '429', '466', '512', '545', '568', '60', '628', '671', '68', '720', '732', '750', '798', '841', '877', '899', '911', '981', '994']\n",
      "NDB_683 -> ['0', '1004', '1037', '1081', '110', '1107', '1160', '1173', '1211', '1265', '1308', '1355', '1391', '140', '1442', '1476', '1518', '194', '237', '267', '274', '324', '350', '372', '39', '426', '454', '521', '590', '626', '676', '685', '718', '775', '812', '854', '870', '89', '902', '927', '955']\n",
      "NDB_694 -> ['1031', '1071', '1120', '1141', '1173', '1203', '1222', '127', '1271', '1323', '1348', '1393', '1463', '1501', '174', '221', '268', '277', '313', '32', '364', '393', '431', '461', '519', '543', '587', '612', '670', '69', '724', '745', '782', '804', '836', '848', '85', '916', '954', '982']\n",
      "NDB_698 -> ['1002', '1032', '1048', '1087', '1109', '1143', '1162', '1226', '123', '1253', '1288', '1309', '1358', '1427', '1457', '1485', '149', '198', '22', '244', '259', '308', '317', '353', '38', '418', '431', '457', '491', '552', '585', '645', '685', '712', '778', '807', '829', '84', '867', '913', '967']\n",
      "NDB_706 -> ['1041', '1069', '1093', '110', '1127', '1134', '1157', '1217', '1261', '1319', '1349', '137', '1400', '1480', '1520', '1535', '178', '215', '253', '311', '336', '351', '396', '459', '471', '496', '532', '57', '580', '647', '678', '68', '699', '722', '794', '804', '852', '863', '926', '962', '986']\n",
      "NDB_710 -> ['1044', '1118', '1154', '116', '1206', '1233', '1250', '1263', '1301', '1323', '1396', '1468', '147', '1487', '189', '237', '264', '286', '31', '322', '367', '4', '408', '424', '446', '499', '538', '583', '597', '625', '673', '684', '715', '797', '820', '841', '88', '904', '929', '988']\n",
      "NDB_712 -> ['1038', '1092', '1128', '1134', '1183', '1233', '1272', '1285', '1337', '1360', '1447', '146', '1465', '1505', '170', '18', '210', '234', '304', '318', '337', '380', '410', '432', '44', '477', '511', '557', '570', '653', '710', '731', '755', '76', '799', '826', '877', '913', '953', '970']\n",
      "NDB_713 -> ['103', '1030', '1111', '1127', '1178', '1229', '1259', '1298', '1341', '1405', '1452', '1497', '1514', '1541', '157', '171', '202', '233', '257', '27', '297', '319', '360', '389', '428', '44', '450', '502', '541', '591', '611', '629', '689', '716', '751', '779', '829', '874', '904', '972', '991']\n",
      "NDB_715 -> ['1020', '1069', '1121', '1141', '1187', '1230', '1247', '1284', '133', '1333', '1388', '1434', '1485', '1502', '154', '184', '202', '21', '241', '299', '339', '384', '417', '441', '487', '505', '542', '583', '657', '67', '688', '744', '749', '776', '817', '833', '893', '93', '933', '964', '993']\n",
      "NDB_716 -> ['1037', '1053', '107', '1106', '1125', '1147', '1204', '1278', '1303', '1345', '1409', '144', '1445', '1488', '1499', '189', '204', '257', '310', '336', '366', '392', '461', '472', '516', '537', '565', '60', '621', '664', '697', '70', '721', '762', '777', '821', '845', '892', '913', '957', '968']\n",
      "NDB_718 -> ['1', '1063', '1083', '1155', '1190', '1239', '1286', '1308', '132', '1325', '1401', '1447', '148', '1494', '1534', '209', '227', '261', '279', '331', '34', '352', '413', '442', '471', '524', '543', '559', '575', '646', '666', '714', '734', '748', '765', '80', '832', '874', '909', '928', '984']\n",
      "NDB_721 -> ['1039', '1065', '1125', '1134', '1153', '1193', '1249', '128', '1293', '1311', '1361', '1391', '1427', '1469', '1513', '168', '186', '207', '223', '239', '293', '333', '37', '389', '411', '422', '470', '519', '554', '594', '613', '674', '680', '707', '764', '80', '809', '814', '839', '893', '974']\n",
      "NDB_725 -> ['101', '1054', '1074', '1130', '1174', '1210', '1252', '1297', '132', '1335', '1402', '1430', '1493', '1509', '1535', '179', '223', '246', '274', '291', '30', '345', '385', '405', '450', '467', '518', '534', '588', '619', '650', '70', '715', '745', '758', '770', '833', '877', '910', '958', '996']\n",
      "NDB_728 -> ['1062', '107', '1119', '1137', '1190', '1224', '1270', '1332', '1381', '1487', '1502', '1535', '158', '177', '198', '212', '23', '242', '284', '324', '356', '376', '40', '440', '480', '519', '539', '558', '576', '624', '679', '687', '701', '751', '792', '835', '875', '894', '911', '953', '961']\n",
      "NDB_730 -> ['1031', '1112', '1172', '1228', '1238', '1279', '1292', '1309', '132', '1332', '1400', '1448', '1465', '1514', '1548', '172', '200', '250', '271', '300', '334', '379', '406', '454', '46', '475', '5', '516', '569', '656', '669', '688', '704', '750', '766', '834', '849', '918', '95', '956', '976']\n",
      "NDB_734 -> ['1059', '1118', '1174', '1190', '1211', '1273', '1295', '1319', '1352', '1409', '143', '1479', '1511', '1526', '18', '181', '207', '229', '276', '335', '391', '414', '463', '468', '47', '490', '502', '564', '599', '620', '675', '694', '723', '759', '783', '850', '864', '889', '925', '95', '980']\n",
      "NDB_737 -> ['1041', '1097', '11', '1123', '1140', '1189', '121', '1234', '1246', '1292', '1336', '1398', '1435', '1478', '1517', '170', '203', '252', '273', '286', '316', '384', '400', '440', '485', '509', '528', '564', '596', '612', '668', '691', '727', '768', '80', '807', '813', '859', '902', '96', '985']\n",
      "NDB_742 -> ['1003', '1040', '1068', '1074', '1087', '1142', '1217', '1257', '1312', '1360', '138', '1408', '1451', '1490', '1502', '1548', '156', '171', '240', '266', '287', '330', '367', '401', '423', '45', '489', '526', '566', '613', '671', '683', '71', '721', '767', '808', '825', '870', '9', '922', '985']\n",
      "NDB_743 -> ['1025', '1055', '1105', '112', '1171', '1209', '1233', '1243', '1253', '1330', '1348', '1448', '145', '1467', '1518', '1536', '190', '195', '21', '224', '269', '310', '341', '390', '411', '448', '478', '507', '533', '584', '596', '646', '702', '730', '74', '790', '825', '846', '889', '907', '983']\n",
      "NDB_746 -> ['1003', '1054', '1090', '111', '1122', '1170', '1212', '1227', '1243', '1286', '13', '1303', '131', '1322', '1386', '1436', '1463', '1481', '1526', '192', '202', '245', '296', '31', '333', '374', '418', '454', '486', '496', '539', '560', '609', '62', '627', '692', '709', '787', '813', '856', '915', '959']\n",
      "NDB_751 -> ['1048', '110', '1115', '1146', '1191', '1216', '1251', '129', '1290', '1325', '1347', '1378', '1441', '1477', '1504', '1533', '162', '206', '238', '272', '28', '283', '316', '374', '417', '451', '479', '526', '565', '608', '651', '689', '707', '752', '770', '79', '835', '873', '896', '932', '950', '970']\n",
      "NDB_752 -> ['103', '1033', '1123', '1144', '1181', '1191', '122', '1238', '1247', '1299', '1305', '1320', '1386', '1458', '1479', '1516', '174', '219', '24', '258', '307', '325', '363', '392', '40', '457', '471', '494', '549', '557', '605', '658', '681', '730', '763', '793', '823', '851', '899', '951', '996']\n",
      "NDB_754 -> ['1020', '1046', '1081', '1177', '118', '1219', '1264', '1293', '1315', '1363', '1405', '1430', '1477', '1508', '1536', '159', '208', '230', '24', '295', '314', '347', '364', '401', '424', '453', '524', '561', '576', '621', '665', '7', '704', '739', '755', '786', '828', '871', '908', '96', '986']\n",
      "NDB_755 -> ['1002', '1055', '106', '1066', '1085', '1132', '1136', '1184', '1207', '1282', '1336', '1361', '1393', '1441', '1465', '1519', '176', '194', '214', '260', '3', '315', '329', '386', '409', '437', '487', '512', '577', '647', '675', '683', '715', '75', '786', '806', '847', '874', '917', '959', '98']\n",
      "NDB_758 -> ['0', '105', '1056', '1072', '1082', '1111', '1168', '1193', '1221', '1284', '13', '1335', '1363', '1381', '1467', '147', '1500', '1537', '181', '201', '260', '292', '330', '353', '397', '419', '432', '483', '525', '559', '603', '622', '726', '763', '804', '837', '865', '87', '896', '976', '995']\n",
      "NDB_760 -> ['1061', '1077', '1107', '117', '1181', '1196', '1241', '1271', '1286', '1341', '1359', '137', '1382', '1488', '1530', '18', '193', '216', '242', '266', '280', '320', '365', '408', '444', '474', '529', '552', '569', '603', '660', '690', '707', '75', '757', '776', '824', '859', '906', '978', '989']\n",
      "NDB_773 -> ['1040', '1091', '1140', '116', '1186', '1225', '1260', '1297', '1310', '1356', '1389', '1434', '145', '1463', '1512', '165', '218', '256', '315', '337', '35', '375', '394', '421', '425', '462', '524', '57', '574', '622', '655', '690', '729', '775', '809', '829', '849', '899', '932', '968', '980']\n",
      "NDB_778 -> ['1044', '1055', '1089', '1152', '1197', '1236', '125', '1256', '1298', '1339', '1356', '1402', '1473', '1496', '1537', '177', '204', '243', '258', '298', '32', '348', '365', '410', '429', '455', '498', '542', '566', '594', '618', '669', '703', '73', '734', '759', '785', '852', '927', '96', '990']\n",
      "NDB_779 -> ['1050', '1067', '1076', '1097', '1176', '122', '1226', '1278', '1304', '1330', '1396', '140', '1455', '1491', '1533', '172', '2', '226', '266', '282', '29', '342', '393', '416', '437', '473', '505', '548', '578', '594', '619', '667', '72', '726', '728', '791', '828', '866', '915', '953', '991']\n",
      "NDB_784 -> ['1000', '1032', '1079', '1125', '1168', '1237', '1244', '1259', '131', '1324', '1349', '1394', '1443', '1486', '1505', '152', '191', '240', '265', '28', '303', '321', '384', '409', '442', '47', '484', '520', '534', '556', '606', '654', '701', '730', '761', '78', '785', '858', '918', '935', '954']\n",
      "NDB_788 -> ['1033', '1073', '1113', '1128', '1139', '1187', '12', '1219', '1271', '128', '1321', '1346', '1385', '149', '1496', '173', '203', '248', '275', '335', '350', '394', '413', '467', '500', '537', '568', '593', '615', '672', '684', '709', '74', '760', '779', '853', '862', '918', '961', '995']\n",
      "NDB_790 -> ['1042', '1064', '1112', '1170', '1189', '12', '120', '1225', '1280', '1305', '1318', '1394', '1439', '1468', '1510', '155', '226', '257', '305', '339', '356', '38', '398', '415', '429', '451', '501', '540', '561', '582', '613', '675', '69', '699', '740', '796', '832', '846', '910', '931', '989']\n",
      "NDB_805 -> ['1061', '1083', '1126', '1138', '1188', '1218', '1264', '1289', '1317', '1354', '1396', '1439', '1469', '1490', '164', '176', '199', '245', '283', '327', '341', '373', '400', '426', '455', '47', '5', '506', '541', '587', '614', '653', '687', '711', '760', '784', '84', '855', '921', '968', '996']\n",
      "NDB_812 -> ['10', '1059', '1089', '1123', '1147', '1208', '1219', '1242', '1258', '126', '1342', '1455', '1475', '1507', '1548', '160', '176', '206', '265', '281', '31', '328', '390', '419', '436', '447', '510', '527', '580', '600', '626', '677', '714', '738', '754', '774', '81', '833', '867', '920', '989']\n",
      "NDB_815 -> ['102', '1030', '1072', '1108', '1121', '1176', '1206', '1274', '1331', '1346', '139', '1404', '1451', '1469', '1507', '1526', '16', '161', '224', '246', '308', '334', '351', '382', '398', '425', '456', '508', '577', '601', '658', '708', '736', '753', '769', '843', '85', '871', '914', '950', '984']\n",
      "NDB_817 -> ['1003', '1044', '1117', '1138', '115', '1195', '1209', '1251', '1299', '1333', '1345', '1454', '1478', '1519', '1542', '167', '188', '220', '255', '274', '295', '349', '358', '37', '379', '430', '468', '501', '546', '579', '603', '66', '662', '729', '778', '805', '836', '863', '900', '935', '975']\n",
      "NDB_818 -> ['1005', '1029', '1066', '1077', '1113', '1137', '118', '1186', '1230', '1273', '13', '1324', '1347', '1390', '1458', '1474', '1520', '157', '186', '195', '211', '262', '287', '317', '357', '380', '447', '461', '506', '540', '586', '618', '650', '728', '780', '802', '820', '844', '89', '919', '975']\n",
      "NDB_824 -> ['1031', '107', '1070', '1121', '1162', '1179', '1194', '1282', '1315', '1353', '1379', '140', '1429', '1466', '1521', '157', '201', '216', '235', '309', '33', '331', '371', '404', '467', '493', '532', '562', '600', '627', '674', '703', '735', '761', '796', '851', '864', '88', '900', '932', '985']\n",
      "NDB_826 -> ['1046', '1122', '1138', '1182', '1210', '1232', '1255', '1290', '1300', '1337', '1380', '1437', '144', '1491', '1521', '166', '174', '200', '229', '265', '279', '329', '340', '36', '367', '412', '456', '474', '492', '542', '571', '623', '65', '663', '727', '775', '811', '844', '868', '903', '967']\n",
      "NDB_832 -> ['1023', '1071', '1083', '1113', '1166', '120', '1234', '1279', '1311', '1360', '1402', '1445', '1486', '150', '1519', '1538', '183', '2', '236', '249', '25', '278', '324', '399', '419', '448', '476', '492', '552', '567', '609', '659', '689', '723', '788', '797', '827', '872', '914', '93', '962']\n",
      "NDB_835 -> ['101', '1038', '1094', '1129', '1160', '1224', '1260', '1299', '130', '1343', '1405', '1455', '1480', '1515', '1538', '184', '199', '239', '254', '300', '322', '353', '402', '460', '486', '490', '503', '571', '608', '654', '720', '735', '790', '8', '808', '824', '83', '869', '895', '971', '982']\n",
      "NDB_836 -> ['1052', '106', '1118', '1148', '119', '1192', '1226', '1256', '1310', '134', '1362', '1398', '1436', '1461', '1503', '222', '249', '285', '303', '351', '397', '453', '477', '510', '565', '587', '650', '666', '67', '702', '743', '764', '803', '823', '83', '868', '9', '905', '924', '956', '978']\n",
      "NDB_838 -> ['1036', '106', '1074', '1097', '1141', '1195', '1280', '1321', '1347', '1404', '141', '1453', '1464', '1494', '1525', '169', '219', '23', '256', '289', '306', '344', '355', '402', '435', '481', '499', '533', '569', '609', '652', '698', '716', '794', '826', '84', '864', '890', '903', '960', '999']\n",
      "NDB_844 -> ['10', '1053', '1070', '1078', '111', '1129', '1156', '1195', '1222', '1259', '1313', '1352', '1385', '1446', '1492', '1506', '154', '182', '228', '264', '293', '340', '382', '42', '421', '452', '489', '522', '559', '596', '618', '676', '681', '733', '757', '791', '835', '872', '924', '98', '986']\n",
      "NDB_847 -> ['101', '1062', '1080', '1092', '1136', '1163', '1196', '1249', '1290', '1302', '1318', '1389', '1473', '1506', '159', '187', '2', '235', '263', '281', '321', '355', '370', '39', '418', '463', '494', '545', '575', '593', '612', '67', '673', '696', '741', '762', '784', '861', '922', '979']\n",
      "NDB_853 -> ['1035', '1077', '1133', '1156', '1197', '1270', '1284', '1306', '1330', '134', '1408', '1456', '1475', '1513', '1529', '179', '230', '254', '26', '308', '327', '363', '395', '430', '458', '521', '538', '560', '575', '6', '656', '69', '695', '725', '761', '779', '821', '850', '903', '926', '970']\n",
      "NDB_856 -> ['1064', '1094', '1144', '1229', '1246', '125', '1276', '1332', '1361', '139', '1403', '1430', '1459', '1464', '1499', '190', '206', '249', '296', '332', '352', '407', '42', '430', '459', '520', '581', '595', '614', '671', '685', '72', '744', '789', '810', '815', '853', '90', '912', '929', '980']\n",
      "NDB_860 -> ['1043', '1092', '1112', '1167', '1209', '121', '1218', '1248', '1295', '1300', '1338', '1383', '141', '1500', '1527', '189', '198', '238', '24', '312', '345', '365', '393', '422', '460', '489', '510', '551', '556', '601', '655', '693', '725', '77', '787', '812', '816', '860', '919', '972', '993']\n",
      "NDB_862 -> ['103', '1057', '1124', '1171', '1203', '1221', '1246', '1278', '1301', '1315', '1401', '1457', '1481', '1503', '1527', '153', '185', '229', '255', '275', '290', '338', '373', '406', '436', '479', '521', '540', '574', '602', '64', '659', '683', '710', '78', '792', '819', '852', '895', '933', '952', '992']\n",
      "NDB_863 -> ['100', '1054', '1075', '1106', '1152', '1178', '1214', '124', '1257', '1292', '1310', '1355', '1404', '1447', '1482', '1520', '166', '225', '244', '263', '283', '347', '368', '40', '407', '448', '484', '512', '550', '555', '607', '64', '648', '684', '719', '749', '791', '824', '836', '920', '963']\n",
      "NDB_865 -> ['104', '1047', '1085', '1153', '1192', '1220', '1250', '1269', '1322', '133', '1355', '1403', '1449', '1462', '1489', '1541', '172', '246', '262', '307', '336', '356', '36', '375', '432', '458', '493', '545', '563', '585', '646', '670', '693', '71', '723', '785', '805', '822', '862', '908', '965']\n",
      "NDB_866 -> ['102', '1060', '1081', '1126', '1171', '1207', '1255', '127', '1322', '1357', '1399', '14', '1453', '1474', '1515', '1542', '175', '222', '258', '309', '337', '391', '413', '423', '451', '517', '538', '572', '607', '629', '667', '68', '680', '691', '744', '762', '795', '857', '906', '958', '976']\n",
      "NDB_868 -> ['1023', '1068', '1107', '1122', '1139', '115', '1165', '1214', '1275', '1336', '1344', '1399', '1440', '1498', '158', '191', '217', '238', '264', '292', '328', '357', '392', '439', '44', '478', '505', '535', '554', '581', '620', '666', '712', '73', '735', '798', '834', '871', '910', '952', '979']\n",
      "NDB_872 -> ['1001', '1024', '1053', '1085', '1131', '1155', '117', '1187', '1231', '1285', '129', '1319', '1353', '1390', '1464', '1509', '1538', '187', '230', '250', '268', '298', '342', '381', '400', '423', '456', '525', '550', '592', '645', '66', '677', '706', '737', '78', '798', '846', '860', '904', '974']\n",
      "NDB_875 -> ['1034', '104', '1087', '1124', '1139', '1212', '1236', '1258', '1291', '1334', '1392', '1459', '1474', '1513', '155', '193', '208', '224', '250', '28', '285', '329', '361', '377', '428', '465', '495', '532', '589', '625', '679', '70', '720', '741', '765', '812', '826', '878', '916', '951', '969']\n",
      "NDB_878 -> ['1004', '1039', '109', '1116', '1154', '1188', '1208', '122', '1254', '1291', '1313', '1350', '1382', '1440', '1477', '1514', '188', '218', '248', '310', '346', '362', '395', '431', '460', '517', '539', '570', '595', '648', '65', '694', '722', '758', '796', '822', '847', '9', '914', '933', '960']\n",
      "NDB_880 -> ['1033', '1049', '109', '1093', '1135', '1166', '1204', '1281', '1306', '1363', '1388', '139', '1457', '1482', '180', '227', '243', '271', '297', '341', '35', '380', '409', '464', '481', '493', '546', '554', '564', '616', '673', '705', '732', '752', '787', '81', '838', '870', '891', '916', '987']\n",
      "NDB_881 -> ['1030', '1068', '1091', '1135', '1145', '1208', '123', '1265', '1287', '1316', '1335', '1387', '144', '1443', '1489', '1510', '182', '239', '255', '26', '270', '306', '331', '377', '407', '443', '477', '504', '528', '567', '583', '647', '682', '728', '760', '77', '795', '861', '889', '930', '990']\n",
      "NDB_883 -> ['1000', '1059', '1116', '1142', '1181', '1213', '1243', '1275', '1301', '1340', '1379', '1489', '1534', '159', '163', '22', '220', '247', '289', '311', '348', '361', '385', '462', '469', '511', '548', '591', '6', '615', '676', '687', '727', '764', '810', '816', '838', '85', '896', '931', '963']\n",
      "NDB_887 -> ['0', '1005', '1057', '1093', '1140', '1163', '1231', '1250', '126', '1291', '1312', '1362', '1383', '1446', '1495', '16', '164', '200', '217', '277', '291', '314', '366', '411', '427', '457', '530', '548', '573', '608', '62', '624', '716', '740', '748', '776', '827', '876', '898', '934', '990']\n",
      "NDB_889 -> ['1057', '1079', '111', '1133', '1154', '1205', '1256', '1295', '1340', '138', '1406', '1449', '1498', '1508', '1525', '160', '216', '236', '3', '302', '317', '350', '387', '425', '43', '443', '516', '531', '555', '589', '653', '691', '705', '751', '770', '832', '86', '869', '897', '949', '974']\n",
      "NDB_896 -> ['1004', '1029', '1067', '1078', '1169', '1215', '1241', '1260', '1314', '1353', '1393', '1435', '1467', '1507', '152', '166', '204', '253', '276', '289', '327', '358', '38', '383', '428', '441', '526', '588', '605', '659', '705', '743', '748', '76', '800', '814', '857', '90', '908', '923', '961']\n",
      "NDB_899 -> ['1037', '1051', '109', '1105', '1137', '1183', '1231', '1276', '1334', '1359', '1389', '1492', '150', '1521', '178', '19', '196', '226', '299', '3', '322', '360', '385', '435', '482', '523', '566', '602', '658', '711', '739', '754', '801', '813', '858', '87', '909', '930', '965', '994']\n",
      "NDB_901 -> ['100', '1043', '1082', '1132', '1146', '1177', '1215', '1244', '125', '1274', '1305', '1344', '1392', '1448', '1462', '1503', '165', '203', '241', '302', '320', '360', '388', '39', '452', '468', '497', '543', '567', '624', '674', '695', '717', '780', '799', '8', '856', '91', '921', '967', '981']\n",
      "NDB_905 -> ['1047', '1058', '1114', '1135', '1178', '1203', '1277', '1302', '1345', '1381', '1438', '1461', '1493', '152', '178', '207', '25', '261', '280', '325', '342', '354', '404', '417', '462', '500', '535', '570', '584', '614', '62', '669', '695', '742', '786', '843', '869', '890', '919', '92', '988']\n",
      "NDB_910 -> ['1039', '1110', '1130', '1144', '1165', '1215', '1255', '1289', '1308', '1356', '1456', '146', '1495', '1533', '163', '17', '211', '263', '312', '33', '344', '382', '397', '414', '438', '480', '529', '546', '589', '611', '677', '692', '713', '74', '766', '807', '817', '858', '892', '982', '992']\n",
      "NDB_917 -> ['1036', '1050', '1094', '1152', '119', '1207', '1234', '1281', '1317', '1351', '1383', '1451', '1488', '1505', '162', '186', '21', '212', '233', '270', '288', '33', '340', '370', '395', '465', '501', '544', '585', '615', '678', '704', '737', '763', '783', '819', '848', '888', '897', '955', '981']\n",
      "NDB_925 -> ['1000', '1062', '11', '1109', '1167', '117', '1221', '1240', '1242', '1277', '1320', '1348', '1380', '1444', '1480', '1495', '168', '192', '234', '256', '272', '278', '29', '319', '387', '415', '441', '475', '506', '537', '577', '607', '662', '718', '734', '789', '822', '850', '891', '930', '969']\n",
      "NDB_926 -> ['100', '1038', '1064', '1090', '1165', '1180', '1205', '123', '1251', '1293', '1313', '1354', '1384', '1442', '1494', '15', '180', '215', '240', '307', '335', '359', '383', '427', '46', '469', '492', '551', '592', '629', '664', '726', '742', '754', '769', '848', '878', '88', '907', '964', '978']\n",
      "NDB_929 -> ['1023', '104', '1065', '1110', '1176', '1222', '1240', '1270', '1300', '1339', '1388', '1468', '1516', '1534', '167', '185', '197', '235', '27', '273', '287', '332', '362', '381', '443', '472', '518', '572', '606', '622', '678', '702', '742', '75', '755', '774', '856', '876', '923', '966', '994']\n",
      "NDB_934 -> ['1035', '1050', '1079', '1143', '12', '1236', '124', '1254', '1287', '1320', '1352', '1378', '1454', '1466', '1509', '180', '205', '253', '273', '296', '318', '355', '406', '449', '45', '483', '497', '530', '558', '601', '657', '697', '739', '797', '855', '868', '906', '92', '928', '971']\n",
      "NDB_935 -> ['1043', '1063', '11', '1114', '1133', '1168', '1191', '1235', '1262', '1311', '1331', '1384', '143', '1453', '1497', '182', '195', '217', '242', '299', '32', '323', '373', '416', '424', '45', '464', '491', '544', '571', '582', '626', '713', '741', '758', '777', '849', '863', '901', '959', '977']\n",
      "NDB_940 -> ['1058', '1070', '1076', '108', '1131', '1173', '1182', '1227', '1273', '1325', '1346', '138', '1384', '1439', '1475', '1515', '190', '20', '212', '225', '260', '297', '320', '377', '405', '438', '475', '523', '556', '57', '598', '621', '686', '712', '768', '805', '839', '876', '905', '973', '999']\n",
      "NDB_941 -> ['102', '1034', '1049', '1109', '1169', '1192', '1239', '1279', '1317', '1349', '1391', '1434', '146', '1498', '167', '221', '248', '26', '291', '305', '325', '359', '388', '446', '474', '509', '527', '562', '599', '649', '665', '696', '733', '777', '802', '860', '87', '895', '912', '963', '991']\n",
      "NDB_946 -> ['1061', '1105', '1124', '114', '1145', '1213', '1248', '1269', '128', '1306', '1358', '1406', '1450', '1486', '187', '201', '23', '261', '286', '328', '372', '410', '438', '472', '508', '549', '579', '599', '623', '688', '722', '756', '800', '828', '83', '873', '898', '958', '98', '997']\n",
      "NDB_958 -> ['1025', '1045', '1091', '1127', '114', '1148', '1182', '1230', '1272', '1342', '1394', '1449', '1479', '15', '1506', '156', '179', '245', '267', '316', '343', '354', '408', '465', '480', '503', '530', '572', '652', '665', '706', '72', '736', '750', '788', '814', '838', '900', '91', '977', '995']\n",
      "NDB_961 -> ['1065', '1089', '115', '1155', '1189', '1213', '1257', '1288', '131', '1314', '1350', '1395', '14', '1445', '1461', '1501', '162', '214', '241', '300', '323', '371', '421', '426', '43', '466', '502', '529', '573', '597', '663', '710', '736', '77', '793', '837', '878', '894', '912', '965', '999']\n",
      "NDB_965 -> ['1048', '1082', '1142', '1180', '1220', '1237', '124', '1264', '1304', '1321', '145', '1452', '1493', '1527', '168', '19', '213', '270', '279', '332', '363', '379', '466', '503', '531', '580', '598', '617', '63', '679', '703', '733', '782', '803', '820', '847', '902', '925', '93', '979', '997']\n",
      "NDB_971 -> ['1020', '1071', '108', '1115', '1169', '1214', '1237', '1276', '1343', '1387', '1437', '1466', '149', '1517', '181', '210', '233', '267', '298', '323', '362', '37', '387', '436', '452', '496', '536', '563', '586', '60', '616', '667', '698', '725', '765', '806', '843', '875', '898', '931', '983']\n",
      "NDB_974 -> ['1035', '1073', '1120', '1179', '1217', '1228', '126', '1262', '1297', '1307', '1358', '1387', '1452', '1462', '1473', '161', '220', '252', '285', '326', '339', '34', '376', '399', '422', '437', '517', '555', '606', '652', '692', '7', '718', '757', '79', '793', '815', '844', '907', '952', '988']\n",
      "NDB_977 -> ['1041', '1126', '1162', '1177', '119', '1224', '1249', '1294', '1307', '1359', '1385', '143', '1443', '1481', '188', '20', '213', '237', '282', '315', '383', '403', '43', '439', '476', '511', '550', '586', '611', '670', '721', '740', '789', '81', '811', '840', '865', '890', '928', '951', '993']\n",
      "NDB_989 -> ['1042', '1069', '1115', '1153', '1194', '1239', '1277', '130', '1338', '1390', '1442', '1482', '1501', '1529', '16', '173', '222', '259', '276', '326', '348', '370', '386', '440', '478', '504', '518', '557', '602', '625', '63', '686', '717', '767', '811', '855', '866', '892', '935', '94', '960', '977']\n",
      "NDB_994 -> ['1045', '1063', '108', '1106', '1129', '1163', '1183', '1206', '1244', '1265', '1314', '133', '1351', '1382', '1438', '1491', '1511', '158', '205', '251', '30', '312', '338', '394', '415', '447', '459', '495', '509', '592', '656', '680', '724', '778', '79', '803', '825', '851', '917', '949', '987']\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "reload(check)\n",
    "readout_reg_dic, readout2reg_check = check._check_readout_to_region(reg_readout_dic, full_records, \n",
    "                                                                    check_readout_dict, primer_len=primer_len, \n",
    "                                                                    target_len=target_len,\n",
    "                                                                    probe_readout_num=num_on_bits)\n",
    "print(readout2reg_check)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7445b10e",
   "metadata": {},
   "source": [
    "### 4.1.5 check bewteen probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "id": "3a6a50cc",
   "metadata": {},
   "outputs": [],
   "source": [
    "# construct internal map\n",
    "reload(check)\n",
    "int_map = check._construct_internal_map(full_records, library_folder, save=False) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 121,
   "id": "11132a68",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "True\n"
     ]
    }
   ],
   "source": [
    "# internal check probes\n",
    "readout_count_dic, readout_count_check = check._check_readout_in_probes(readout_reg_dic, reg_size_dic, int_map, readout_dict)\n",
    "print(readout_count_check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "id": "056864be",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_1:1235001-1265000_gene_0_pb_141_pos_13330_strand_+_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:81, dumped\n",
      "--- Sequence:loc_1:1235001-1265000_gene_0_pb_145_pos_13649_strand_+_readouts_[NDB_758_c,NDB_758_c,NDB_758_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_1:1235001-1265000_gene_0_pb_146_pos_13789_strand_+_readouts_[NDB_887_c,NDB_887_c,NDB_887_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_1:1235001-1265000_gene_0_pb_147_pos_13855_strand_+_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:84, dumped\n",
      "--- Sequence:loc_1:1235001-1265000_gene_0_pb_149_pos_14659_strand_-_readouts_[NDB_887_c,NDB_887_c,NDB_887_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_56_pos_4401_strand_-_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_57_pos_4503_strand_-_readouts_[NDB_718_c,NDB_718_c,NDB_718_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_58_pos_4557_strand_-_readouts_[NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:77, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_59_pos_4601_strand_-_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_60_pos_4656_strand_-_readouts_[NDB_718_c,NDB_718_c,NDB_718_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:101, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_61_pos_4700_strand_-_readouts_[NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_62_pos_4773_strand_-_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:81, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_63_pos_4817_strand_-_readouts_[NDB_718_c,NDB_718_c,NDB_718_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_64_pos_4900_strand_-_readouts_[NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:119, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_65_pos_4947_strand_-_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:92, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_66_pos_5012_strand_-_readouts_[NDB_718_c,NDB_718_c,NDB_718_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_67_pos_5084_strand_-_readouts_[NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_1:3735001-3765000_gene_1_pb_68_pos_5133_strand_-_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:98, dumped\n",
      "--- Sequence:loc_1:168735001-168765000_gene_67_pb_101_pos_9463_strand_+_readouts_[NDB_836_c,NDB_836_c,NDB_836_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_1:168735001-168765000_gene_67_pb_102_pos_9523_strand_+_readouts_[NDB_847_c,NDB_847_c,NDB_847_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_1:201235001-201265000_gene_80_pb_152_pos_21019_strand_-_readouts_[NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_1:201235001-201265000_gene_80_pb_154_pos_21355_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_1:201235001-201265000_gene_80_pb_157_pos_22005_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_1:201235001-201265000_gene_80_pb_158_pos_22153_strand_+_readouts_[NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_1:201235001-201265000_gene_80_pb_159_pos_22288_strand_+_readouts_[NDB_718_c,NDB_718_c,NDB_718_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_1:201235001-201265000_gene_80_pb_164_pos_23479_strand_+_readouts_[NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_1:246235001-246265000_gene_98_pb_98_pos_16513_strand_+_readouts_[NDB_946_c,NDB_946_c,NDB_946_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_1:246235001-246265000_gene_98_pb_100_pos_17135_strand_-_readouts_[NDB_844_c,NDB_844_c,NDB_844_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_2:6235001-6265000_gene_101_pb_36_pos_3462_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_2:6235001-6265000_gene_101_pb_37_pos_3528_strand_-_readouts_[NDB_847_c,NDB_847_c,NDB_847_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_2:6235001-6265000_gene_101_pb_38_pos_3816_strand_-_readouts_[NDB_835_c,NDB_835_c,NDB_835_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:75, dumped\n",
      "--- Sequence:loc_2:6235001-6265000_gene_101_pb_39_pos_3860_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_2:168735001-168765000_gene_166_pb_84_pos_7550_strand_+_readouts_[NDB_826_c,NDB_826_c,NDB_826_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_2:178735001-178765000_gene_170_pb_166_pos_16820_strand_-_readouts_[NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:134, dumped\n",
      "--- Sequence:loc_3:76235001-76265000_gene_225_pb_25_pos_3536_strand_+_readouts_[NDB_1060_c,NDB_1060_c,NDB_1060_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_3:171235001-171265000_gene_263_pb_131_pos_15442_strand_+_readouts_[NDB_910_c,NDB_910_c,NDB_910_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_136_pos_14761_strand_+_readouts_[NDB_929_c,NDB_929_c,NDB_929_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_137_pos_14877_strand_+_readouts_[NDB_934_c,NDB_934_c,NDB_934_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:133, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_138_pos_14942_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_139_pos_14986_strand_+_readouts_[NDB_929_c,NDB_929_c,NDB_929_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_140_pos_15057_strand_+_readouts_[NDB_934_c,NDB_934_c,NDB_934_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:100, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_142_pos_15216_strand_+_readouts_[NDB_929_c,NDB_929_c,NDB_929_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_143_pos_15260_strand_+_readouts_[NDB_934_c,NDB_934_c,NDB_934_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_144_pos_15408_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:161, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_145_pos_15483_strand_+_readouts_[NDB_929_c,NDB_929_c,NDB_929_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:135, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_146_pos_15530_strand_+_readouts_[NDB_934_c,NDB_934_c,NDB_934_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:125, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_147_pos_15585_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:141, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_148_pos_15652_strand_+_readouts_[NDB_929_c,NDB_929_c,NDB_929_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:88, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_149_pos_15711_strand_+_readouts_[NDB_934_c,NDB_934_c,NDB_934_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:134, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_150_pos_15769_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:136, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_151_pos_15828_strand_+_readouts_[NDB_929_c,NDB_929_c,NDB_929_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:135, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_152_pos_15918_strand_+_readouts_[NDB_934_c,NDB_934_c,NDB_934_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:155, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_154_pos_16125_strand_+_readouts_[NDB_929_c,NDB_929_c,NDB_929_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:98, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_155_pos_16235_strand_+_readouts_[NDB_934_c,NDB_934_c,NDB_934_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:116, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_156_pos_16285_strand_+_readouts_[NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:120, dumped\n",
      "--- Sequence:loc_3:196235001-196265000_gene_273_pb_157_pos_16344_strand_+_readouts_[NDB_929_c,NDB_929_c,NDB_929_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:113, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_25_pos_1515_strand_-_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:135, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_26_pos_1589_strand_-_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:167, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_27_pos_1653_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:122, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_28_pos_1726_strand_-_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:107, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_29_pos_1826_strand_-_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:113, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_30_pos_1898_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:130, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_33_pos_2069_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:91, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_34_pos_2164_strand_-_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:88, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_35_pos_2234_strand_-_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_36_pos_2304_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:117, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_38_pos_2511_strand_-_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:95, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_39_pos_2588_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_40_pos_2692_strand_-_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:166, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_41_pos_2759_strand_-_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:153, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_42_pos_2803_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:114, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_44_pos_2938_strand_-_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:152, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_45_pos_3014_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_4:1235001-1265000_gene_274_pb_81_pos_5590_strand_+_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_4:83735001-83765000_gene_307_pb_125_pos_20855_strand_-_readouts_[NDB_926_c,NDB_926_c,NDB_926_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:150, dumped\n",
      "--- Sequence:loc_4:181235001-181265000_gene_346_pb_36_pos_4620_strand_+_readouts_[NDB_643_c,NDB_643_c,NDB_643_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_4:181235001-181265000_gene_346_pb_37_pos_4701_strand_-_readouts_[NDB_1027_c,NDB_1027_c,NDB_1027_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:91, dumped\n",
      "--- Sequence:loc_4:181235001-181265000_gene_346_pb_38_pos_5467_strand_-_readouts_[NDB_878_c,NDB_878_c,NDB_878_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:84, dumped\n",
      "--- Sequence:loc_4:181235001-181265000_gene_346_pb_39_pos_5728_strand_+_readouts_[NDB_643_c,NDB_643_c,NDB_643_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_4:181235001-181265000_gene_346_pb_40_pos_6638_strand_-_readouts_[NDB_1027_c,NDB_1027_c,NDB_1027_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:82, dumped\n",
      "--- Sequence:loc_4:181235001-181265000_gene_346_pb_41_pos_6697_strand_-_readouts_[NDB_878_c,NDB_878_c,NDB_878_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:110, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_222_pos_23951_strand_+_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:74, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_224_pos_24083_strand_+_readouts_[NDB_788_c,NDB_788_c,NDB_788_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_225_pos_24141_strand_+_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:114, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_227_pos_24259_strand_+_readouts_[NDB_788_c,NDB_788_c,NDB_788_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:113, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_228_pos_24316_strand_+_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_231_pos_24475_strand_-_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_233_pos_24604_strand_+_readouts_[NDB_788_c,NDB_788_c,NDB_788_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_234_pos_24666_strand_+_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_235_pos_24726_strand_+_readouts_[NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:99, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_236_pos_24833_strand_-_readouts_[NDB_788_c,NDB_788_c,NDB_788_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_237_pos_24934_strand_+_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_5:1235001-1265000_gene_350_pb_238_pos_24985_strand_-_readouts_[NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_5:3735001-3765000_gene_351_pb_254_pos_26376_strand_-_readouts_[NDB_836_c,NDB_836_c,NDB_836_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_5:3735001-3765000_gene_351_pb_255_pos_26420_strand_+_readouts_[NDB_815_c,NDB_815_c,NDB_815_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_5:3735001-3765000_gene_351_pb_257_pos_26524_strand_-_readouts_[NDB_836_c,NDB_836_c,NDB_836_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_5:3735001-3765000_gene_351_pb_258_pos_26606_strand_-_readouts_[NDB_815_c,NDB_815_c,NDB_815_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_5:3735001-3765000_gene_351_pb_260_pos_26734_strand_-_readouts_[NDB_836_c,NDB_836_c,NDB_836_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_5:3735001-3765000_gene_351_pb_261_pos_26778_strand_-_readouts_[NDB_815_c,NDB_815_c,NDB_815_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_5:26235001-26265000_gene_360_pb_5_pos_876_strand_+_readouts_[NDB_899_c,NDB_899_c,NDB_899_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_5:26235001-26265000_gene_360_pb_7_pos_1193_strand_+_readouts_[NDB_901_c,NDB_901_c,NDB_901_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_5:26235001-26265000_gene_360_pb_8_pos_1297_strand_+_readouts_[NDB_899_c,NDB_899_c,NDB_899_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_5:143735001-143765000_gene_407_pb_164_pos_16397_strand_+_readouts_[NDB_881_c,NDB_881_c,NDB_881_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_5:146235001-146265000_gene_408_pb_105_pos_21980_strand_-_readouts_[NDB_710_c,NDB_710_c,NDB_710_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_6:1235001-1265000_gene_422_pb_244_pos_25980_strand_+_readouts_[NDB_860_c,NDB_860_c,NDB_860_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_6:1235001-1265000_gene_422_pb_245_pos_26025_strand_+_readouts_[NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_6:158735001-158765000_gene_485_pb_158_pos_21795_strand_-_readouts_[NDB_647_c,NDB_647_c,NDB_647_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:225, dumped\n",
      "--- Sequence:loc_7:1235001-1265000_gene_490_pb_253_pos_26194_strand_+_readouts_[NDB_641_c,NDB_641_c,NDB_641_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:163, dumped\n",
      "--- Sequence:loc_7:1235001-1265000_gene_490_pb_254_pos_26259_strand_+_readouts_[NDB_734_c,NDB_734_c,NDB_734_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:138, dumped\n",
      "--- Sequence:loc_7:1235001-1265000_gene_490_pb_255_pos_26354_strand_-_readouts_[NDB_835_c,NDB_835_c,NDB_835_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:157, dumped\n",
      "--- Sequence:loc_7:1235001-1265000_gene_490_pb_256_pos_26424_strand_-_readouts_[NDB_641_c,NDB_641_c,NDB_641_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:117, dumped\n",
      "--- Sequence:loc_7:1235001-1265000_gene_490_pb_258_pos_26524_strand_+_readouts_[NDB_835_c,NDB_835_c,NDB_835_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:169, dumped\n",
      "--- Sequence:loc_7:1235001-1265000_gene_490_pb_259_pos_26586_strand_+_readouts_[NDB_641_c,NDB_641_c,NDB_641_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:146, dumped\n",
      "--- Sequence:loc_7:1235001-1265000_gene_490_pb_260_pos_26632_strand_-_readouts_[NDB_734_c,NDB_734_c,NDB_734_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:111, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_200_pos_20841_strand_+_readouts_[NDB_1028_c,NDB_1028_c,NDB_1028_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_201_pos_20896_strand_+_readouts_[NDB_698_c,NDB_698_c,NDB_698_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_202_pos_20951_strand_+_readouts_[NDB_935_c,NDB_935_c,NDB_935_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:125, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_203_pos_21000_strand_+_readouts_[NDB_1028_c,NDB_1028_c,NDB_1028_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:121, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_204_pos_21062_strand_+_readouts_[NDB_698_c,NDB_698_c,NDB_698_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:126, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_205_pos_21179_strand_+_readouts_[NDB_935_c,NDB_935_c,NDB_935_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:96, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_206_pos_21240_strand_-_readouts_[NDB_1028_c,NDB_1028_c,NDB_1028_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:102, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_207_pos_21302_strand_-_readouts_[NDB_698_c,NDB_698_c,NDB_698_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:124, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_208_pos_21367_strand_-_readouts_[NDB_935_c,NDB_935_c,NDB_935_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_209_pos_21414_strand_+_readouts_[NDB_1028_c,NDB_1028_c,NDB_1028_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:128, dumped\n",
      "--- Sequence:loc_7:3735001-3765000_gene_491_pb_210_pos_21474_strand_+_readouts_[NDB_698_c,NDB_698_c,NDB_698_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_24_pos_2347_strand_-_readouts_[NDB_974_c,NDB_974_c,NDB_974_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_25_pos_2407_strand_-_readouts_[NDB_878_c,NDB_878_c,NDB_878_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_26_pos_2479_strand_-_readouts_[NDB_866_c,NDB_866_c,NDB_866_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_29_pos_2681_strand_+_readouts_[NDB_866_c,NDB_866_c,NDB_866_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_30_pos_2755_strand_+_readouts_[NDB_974_c,NDB_974_c,NDB_974_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_31_pos_2825_strand_+_readouts_[NDB_878_c,NDB_878_c,NDB_878_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:102, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_32_pos_2882_strand_-_readouts_[NDB_866_c,NDB_866_c,NDB_866_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_33_pos_2968_strand_+_readouts_[NDB_974_c,NDB_974_c,NDB_974_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:115, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_36_pos_3235_strand_+_readouts_[NDB_974_c,NDB_974_c,NDB_974_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:90, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_37_pos_3308_strand_+_readouts_[NDB_878_c,NDB_878_c,NDB_878_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:96, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_38_pos_3378_strand_+_readouts_[NDB_866_c,NDB_866_c,NDB_866_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_39_pos_3443_strand_+_readouts_[NDB_974_c,NDB_974_c,NDB_974_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_40_pos_3518_strand_+_readouts_[NDB_878_c,NDB_878_c,NDB_878_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_7:68735001-68765000_gene_517_pb_41_pos_3594_strand_+_readouts_[NDB_866_c,NDB_866_c,NDB_866_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_179_pos_17367_strand_-_readouts_[NDB_838_c,NDB_838_c,NDB_838_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:84, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_180_pos_17440_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_181_pos_17498_strand_-_readouts_[NDB_832_c,NDB_832_c,NDB_832_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:156, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_182_pos_17567_strand_-_readouts_[NDB_838_c,NDB_838_c,NDB_838_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:137, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_183_pos_17612_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:141, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_184_pos_17716_strand_-_readouts_[NDB_832_c,NDB_832_c,NDB_832_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:158, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_185_pos_17792_strand_-_readouts_[NDB_838_c,NDB_838_c,NDB_838_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:104, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_187_pos_17892_strand_-_readouts_[NDB_832_c,NDB_832_c,NDB_832_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_188_pos_17936_strand_-_readouts_[NDB_838_c,NDB_838_c,NDB_838_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:99, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_189_pos_17986_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:111, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_190_pos_18069_strand_-_readouts_[NDB_832_c,NDB_832_c,NDB_832_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:159, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_191_pos_18113_strand_-_readouts_[NDB_838_c,NDB_838_c,NDB_838_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:148, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_192_pos_18163_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:82, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_193_pos_18207_strand_-_readouts_[NDB_832_c,NDB_832_c,NDB_832_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:129, dumped\n",
      "--- Sequence:loc_8:141235001-141265000_gene_609_pb_194_pos_18256_strand_-_readouts_[NDB_838_c,NDB_838_c,NDB_838_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:157, dumped\n",
      "--- Sequence:loc_9:130985001-131015000_gene_676_pb_120_pos_11443_strand_+_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_10:985001-1015000_gene_680_pb_254_pos_26107_strand_-_readouts_[NDB_994_c,NDB_994_c,NDB_994_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_10:2985001-3015000_gene_681_pb_151_pos_17054_strand_-_readouts_[NDB_752_c,NDB_752_c,NDB_752_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_10:2985001-3015000_gene_681_pb_154_pos_17237_strand_-_readouts_[NDB_752_c,NDB_752_c,NDB_752_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_10:88985001-89015000_gene_724_pb_20_pos_1662_strand_+_readouts_[NDB_694_c,NDB_694_c,NDB_694_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_10:116985001-117015000_gene_738_pb_163_pos_19989_strand_-_readouts_[NDB_634_c,NDB_634_c,NDB_634_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:358, dumped\n",
      "--- Sequence:loc_10:126985001-127015000_gene_743_pb_170_pos_18183_strand_+_readouts_[NDB_836_c,NDB_836_c,NDB_836_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_10:126985001-127015000_gene_743_pb_171_pos_18261_strand_+_readouts_[NDB_1027_c,NDB_1027_c,NDB_1027_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_10:126985001-127015000_gene_743_pb_172_pos_18332_strand_+_readouts_[NDB_896_c,NDB_896_c,NDB_896_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_178_pos_16773_strand_+_readouts_[NDB_728_c,NDB_728_c,NDB_728_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_179_pos_16846_strand_-_readouts_[NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_180_pos_16892_strand_+_readouts_[NDB_713_c,NDB_713_c,NDB_713_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_182_pos_17002_strand_-_readouts_[NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_183_pos_17084_strand_+_readouts_[NDB_713_c,NDB_713_c,NDB_713_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_184_pos_17128_strand_-_readouts_[NDB_728_c,NDB_728_c,NDB_728_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_185_pos_17200_strand_+_readouts_[NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_186_pos_17244_strand_+_readouts_[NDB_713_c,NDB_713_c,NDB_713_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_187_pos_17288_strand_+_readouts_[NDB_728_c,NDB_728_c,NDB_728_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:97, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_191_pos_17568_strand_-_readouts_[NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:95, dumped\n",
      "--- Sequence:loc_11:10985001-11015000_gene_751_pb_192_pos_17612_strand_-_readouts_[NDB_713_c,NDB_713_c,NDB_713_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_11:32985001-33015000_gene_762_pb_93_pos_21050_strand_+_readouts_[NDB_847_c,NDB_847_c,NDB_847_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:127, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_11:62985001-63015000_gene_777_pb_48_pos_4612_strand_-_readouts_[NDB_716_c,NDB_716_c,NDB_716_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_11:74985001-75015000_gene_783_pb_197_pos_23135_strand_+_readouts_[NDB_1028_c,NDB_1028_c,NDB_1028_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:125, dumped\n",
      "--- Sequence:loc_12:4985001-5015000_gene_815_pb_258_pos_25626_strand_+_readouts_[NDB_974_c,NDB_974_c,NDB_974_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:91, dumped\n",
      "--- Sequence:loc_12:4985001-5015000_gene_815_pb_260_pos_25784_strand_+_readouts_[NDB_641_c,NDB_641_c,NDB_641_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:95, dumped\n",
      "--- Sequence:loc_12:4985001-5015000_gene_815_pb_261_pos_25887_strand_+_readouts_[NDB_974_c,NDB_974_c,NDB_974_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:90, dumped\n",
      "--- Sequence:loc_12:4985001-5015000_gene_815_pb_262_pos_25939_strand_+_readouts_[NDB_856_c,NDB_856_c,NDB_856_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:99, dumped\n",
      "--- Sequence:loc_12:118985001-119015000_gene_872_pb_232_pos_23481_strand_-_readouts_[NDB_832_c,NDB_832_c,NDB_832_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_12:120985001-121015000_gene_873_pb_79_pos_9825_strand_+_readouts_[NDB_946_c,NDB_946_c,NDB_946_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_12:120985001-121015000_gene_873_pb_80_pos_9884_strand_+_readouts_[NDB_751_c,NDB_751_c,NDB_751_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_12:120985001-121015000_gene_873_pb_83_pos_10046_strand_-_readouts_[NDB_751_c,NDB_751_c,NDB_751_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:82, dumped\n",
      "--- Sequence:loc_12:120985001-121015000_gene_873_pb_84_pos_10105_strand_+_readouts_[NDB_652_c,NDB_652_c,NDB_652_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_164_pos_19738_strand_-_readouts_[NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:139, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_165_pos_19825_strand_+_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:148, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_166_pos_20061_strand_+_readouts_[NDB_680_c,NDB_680_c,NDB_680_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_167_pos_20118_strand_+_readouts_[NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_168_pos_20162_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:129, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_169_pos_20230_strand_+_readouts_[NDB_680_c,NDB_680_c,NDB_680_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_170_pos_20292_strand_-_readouts_[NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:132, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_171_pos_20346_strand_+_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:149, dumped\n",
      "--- Sequence:loc_12:128985001-129015000_gene_877_pb_177_pos_20690_strand_+_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_13:22985001-23015000_gene_890_pb_119_pos_16706_strand_+_readouts_[NDB_977_c,NDB_977_c,NDB_977_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_13:108985001-109015000_gene_933_pb_60_pos_6577_strand_+_readouts_[NDB_715_c,NDB_715_c,NDB_715_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_13:108985001-109015000_gene_933_pb_61_pos_6683_strand_+_readouts_[NDB_878_c,NDB_878_c,NDB_878_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_13:108985001-109015000_gene_933_pb_62_pos_6738_strand_-_readouts_[NDB_862_c,NDB_862_c,NDB_862_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_13:108985001-109015000_gene_933_pb_185_pos_23270_strand_-_readouts_[NDB_862_c,NDB_862_c,NDB_862_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_13:108985001-109015000_gene_933_pb_186_pos_23660_strand_+_readouts_[NDB_715_c,NDB_715_c,NDB_715_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_13:108985001-109015000_gene_933_pb_187_pos_23785_strand_-_readouts_[NDB_878_c,NDB_878_c,NDB_878_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_38_pos_3544_strand_+_readouts_[NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:117, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_40_pos_3721_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:116, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_42_pos_3826_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_44_pos_4004_strand_+_readouts_[NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_45_pos_4056_strand_+_readouts_[NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:145, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_46_pos_4232_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:125, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_47_pos_4304_strand_-_readouts_[NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:91, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_48_pos_4380_strand_+_readouts_[NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:141, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_49_pos_4428_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:127, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_51_pos_4567_strand_+_readouts_[NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:138, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_52_pos_4637_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:75, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_53_pos_4681_strand_+_readouts_[NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:111, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_176_pos_20501_strand_+_readouts_[NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_177_pos_20550_strand_+_readouts_[NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_256_pos_26406_strand_-_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_257_pos_26460_strand_-_readouts_[NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_258_pos_26572_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_259_pos_26643_strand_-_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_260_pos_26687_strand_+_readouts_[NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:93, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_261_pos_26731_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_262_pos_26910_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:124, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_263_pos_27020_strand_-_readouts_[NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:89, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_264_pos_27146_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_266_pos_27360_strand_+_readouts_[NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:106, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_267_pos_27420_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:69, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_268_pos_27464_strand_-_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_270_pos_27591_strand_-_readouts_[NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:119, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_271_pos_27647_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:157, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_272_pos_27727_strand_-_readouts_[NDB_989_c,NDB_989_c,NDB_989_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:120, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_273_pos_27864_strand_+_readouts_[NDB_784_c,NDB_784_c,NDB_784_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:114, dumped\n",
      "--- Sequence:loc_13:112985001-113015000_gene_935_pb_274_pos_27954_strand_-_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:157, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_14:50235001-50265000_gene_969_pb_21_pos_3641_strand_-_readouts_[NDB_875_c,NDB_875_c,NDB_875_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_14:57735001-57765000_gene_974_pb_121_pos_18282_strand_+_readouts_[NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n",
      "--- Sequence:loc_14:59235001-59265000_gene_975_pb_60_pos_5799_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:86, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_7_pos_405_strand_+_readouts_[NDB_994_c,NDB_994_c,NDB_994_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_9_pos_565_strand_+_readouts_[NDB_880_c,NDB_880_c,NDB_880_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_10_pos_669_strand_-_readouts_[NDB_994_c,NDB_994_c,NDB_994_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_12_pos_765_strand_-_readouts_[NDB_880_c,NDB_880_c,NDB_880_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_13_pos_898_strand_-_readouts_[NDB_994_c,NDB_994_c,NDB_994_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_14_pos_985_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_15_pos_1101_strand_+_readouts_[NDB_880_c,NDB_880_c,NDB_880_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_17_pos_1273_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_14:77235001-77265000_gene_987_pb_18_pos_1386_strand_-_readouts_[NDB_880_c,NDB_880_c,NDB_880_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_90_pos_10644_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:102, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_91_pos_10734_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:101, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_92_pos_10851_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:124, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_94_pos_11094_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:108, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_95_pos_11155_strand_-_readouts_[NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:124, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_96_pos_11221_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:112, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_97_pos_11265_strand_+_readouts_[NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_99_pos_11382_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:96, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_100_pos_11443_strand_+_readouts_[NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:124, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_102_pos_11566_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:121, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_103_pos_11625_strand_-_readouts_[NDB_746_c,NDB_746_c,NDB_746_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:91, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_104_pos_11688_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:113, dumped\n",
      "--- Sequence:loc_14:101235001-101265000_gene_1003_pb_105_pos_11744_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:117, dumped\n",
      "--- Sequence:loc_14:104235001-104265000_gene_1005_pb_158_pos_15818_strand_+_readouts_[NDB_818_c,NDB_818_c,NDB_818_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:96, dumped\n",
      "--- Sequence:loc_14:104235001-104265000_gene_1005_pb_159_pos_15865_strand_+_readouts_[NDB_661_c,NDB_661_c,NDB_661_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_14:104235001-104265000_gene_1005_pb_161_pos_16197_strand_-_readouts_[NDB_818_c,NDB_818_c,NDB_818_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:86, dumped\n",
      "--- Sequence:loc_14:104235001-104265000_gene_1005_pb_162_pos_16852_strand_+_readouts_[NDB_661_c,NDB_661_c,NDB_661_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:98, dumped\n",
      "--- Sequence:loc_14:104235001-104265000_gene_1005_pb_163_pos_16897_strand_-_readouts_[NDB_887_c,NDB_887_c,NDB_887_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_14:104235001-104265000_gene_1005_pb_164_pos_16946_strand_+_readouts_[NDB_818_c,NDB_818_c,NDB_818_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:63, dumped\n",
      "--- Sequence:loc_14:104235001-104265000_gene_1005_pb_165_pos_16993_strand_+_readouts_[NDB_661_c,NDB_661_c,NDB_661_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_14:104235001-104265000_gene_1005_pb_166_pos_17039_strand_+_readouts_[NDB_887_c,NDB_887_c,NDB_887_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_15:62235001-62265000_gene_1048_pb_100_pos_10851_strand_-_readouts_[NDB_698_c,NDB_698_c,NDB_698_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_15:62235001-62265000_gene_1048_pb_103_pos_11083_strand_-_readouts_[NDB_698_c,NDB_698_c,NDB_698_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_15:62235001-62265000_gene_1048_pb_107_pos_11397_strand_-_readouts_[NDB_751_c,NDB_751_c,NDB_751_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_15:62235001-62265000_gene_1048_pb_155_pos_16135_strand_-_readouts_[NDB_751_c,NDB_751_c,NDB_751_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_15:68235001-68265000_gene_1052_pb_25_pos_4563_strand_+_readouts_[NDB_661_c,NDB_661_c,NDB_661_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_16:735001-765000_gene_1074_pb_102_pos_11145_strand_+_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n",
      "--- Sequence:loc_16:735001-765000_gene_1074_pb_104_pos_11273_strand_-_readouts_[NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_16:735001-765000_gene_1074_pb_105_pos_11363_strand_+_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_16:735001-765000_gene_1074_pb_106_pos_11450_strand_-_readouts_[NDB_838_c,NDB_838_c,NDB_838_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_16:735001-765000_gene_1074_pb_107_pos_11499_strand_-_readouts_[NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_16:735001-765000_gene_1074_pb_108_pos_11580_strand_-_readouts_[NDB_725_c,NDB_725_c,NDB_725_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_16:735001-765000_gene_1074_pb_109_pos_11629_strand_-_readouts_[NDB_838_c,NDB_838_c,NDB_838_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_16:60735001-60765000_gene_1114_pb_118_pos_16300_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_16:60735001-60765000_gene_1114_pb_132_pos_18301_strand_-_readouts_[NDB_905_c,NDB_905_c,NDB_905_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_16:84735001-84765000_gene_1130_pb_290_pos_28753_strand_+_readouts_[NDB_910_c,NDB_910_c,NDB_910_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_16:84735001-84765000_gene_1130_pb_292_pos_28847_strand_-_readouts_[NDB_653_c,NDB_653_c,NDB_653_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_16:87735001-87765000_gene_1132_pb_38_pos_3587_strand_-_readouts_[NDB_901_c,NDB_901_c,NDB_901_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_16:87735001-87765000_gene_1132_pb_39_pos_4304_strand_-_readouts_[NDB_755_c,NDB_755_c,NDB_755_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_37_pos_5569_strand_+_readouts_[NDB_706_c,NDB_706_c,NDB_706_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_38_pos_5613_strand_+_readouts_[NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_39_pos_5664_strand_+_readouts_[NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_40_pos_5708_strand_-_readouts_[NDB_706_c,NDB_706_c,NDB_706_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_41_pos_5752_strand_+_readouts_[NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_127_pos_13640_strand_-_readouts_[NDB_706_c,NDB_706_c,NDB_706_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:130, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_128_pos_13697_strand_-_readouts_[NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:131, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_129_pos_13743_strand_-_readouts_[NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:145, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_130_pos_13799_strand_-_readouts_[NDB_706_c,NDB_706_c,NDB_706_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_131_pos_13910_strand_-_readouts_[NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:135, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_132_pos_13998_strand_-_readouts_[NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:124, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_133_pos_14089_strand_-_readouts_[NDB_706_c,NDB_706_c,NDB_706_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:155, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_134_pos_14187_strand_-_readouts_[NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:158, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_135_pos_14263_strand_-_readouts_[NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:111, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_136_pos_14320_strand_-_readouts_[NDB_706_c,NDB_706_c,NDB_706_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:145, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_137_pos_14364_strand_-_readouts_[NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:124, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_139_pos_14477_strand_-_readouts_[NDB_706_c,NDB_706_c,NDB_706_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:132, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_141_pos_14590_strand_-_readouts_[NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_142_pos_14634_strand_-_readouts_[NDB_706_c,NDB_706_c,NDB_706_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:90, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_143_pos_14710_strand_-_readouts_[NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:126, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_189_pos_18063_strand_+_readouts_[NDB_721_c,NDB_721_c,NDB_721_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_17:735001-765000_gene_1134_pb_191_pos_18339_strand_+_readouts_[NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_17:3735001-3765000_gene_1136_pb_158_pos_16971_strand_+_readouts_[NDB_847_c,NDB_847_c,NDB_847_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_17:3735001-3765000_gene_1136_pb_159_pos_17016_strand_-_readouts_[NDB_755_c,NDB_755_c,NDB_755_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:62, dumped\n",
      "--- Sequence:loc_17:3735001-3765000_gene_1136_pb_160_pos_17060_strand_+_readouts_[NDB_652_c,NDB_652_c,NDB_652_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_17:3735001-3765000_gene_1136_pb_161_pos_17107_strand_+_readouts_[NDB_847_c,NDB_847_c,NDB_847_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_17:5235001-5265000_gene_1137_pb_46_pos_5724_strand_-_readouts_[NDB_899_c,NDB_899_c,NDB_899_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_17:5235001-5265000_gene_1137_pb_48_pos_6055_strand_-_readouts_[NDB_728_c,NDB_728_c,NDB_728_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_17:5235001-5265000_gene_1137_pb_49_pos_6179_strand_-_readouts_[NDB_899_c,NDB_899_c,NDB_899_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_146_pos_23288_strand_+_readouts_[NDB_805_c,NDB_805_c,NDB_805_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_147_pos_23342_strand_+_readouts_[NDB_826_c,NDB_826_c,NDB_826_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_148_pos_23387_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:81, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_149_pos_23437_strand_+_readouts_[NDB_805_c,NDB_805_c,NDB_805_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_150_pos_23486_strand_+_readouts_[NDB_826_c,NDB_826_c,NDB_826_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:90, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_151_pos_23533_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_152_pos_23582_strand_+_readouts_[NDB_805_c,NDB_805_c,NDB_805_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:78, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_153_pos_23680_strand_+_readouts_[NDB_826_c,NDB_826_c,NDB_826_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_154_pos_23729_strand_+_readouts_[NDB_817_c,NDB_817_c,NDB_817_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_17:6735001-6765000_gene_1138_pb_156_pos_23875_strand_+_readouts_[NDB_826_c,NDB_826_c,NDB_826_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_17:66735001-66765000_gene_1178_pb_140_pos_13995_strand_+_readouts_[NDB_905_c,NDB_905_c,NDB_905_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_17:66735001-66765000_gene_1178_pb_142_pos_14146_strand_+_readouts_[NDB_863_c,NDB_863_c,NDB_863_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_184_pos_19397_strand_+_readouts_[NDB_896_c,NDB_896_c,NDB_896_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_194_pos_20036_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:83, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_196_pos_20157_strand_+_readouts_[NDB_896_c,NDB_896_c,NDB_896_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:83, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_197_pos_20222_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:109, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_200_pos_20393_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_201_pos_20437_strand_-_readouts_[NDB_760_c,NDB_760_c,NDB_760_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_202_pos_20489_strand_+_readouts_[NDB_896_c,NDB_896_c,NDB_896_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_204_pos_20611_strand_+_readouts_[NDB_760_c,NDB_760_c,NDB_760_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_205_pos_20659_strand_+_readouts_[NDB_896_c,NDB_896_c,NDB_896_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_206_pos_20745_strand_-_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_207_pos_20796_strand_+_readouts_[NDB_760_c,NDB_760_c,NDB_760_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:108, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_209_pos_20922_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:101, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_211_pos_21085_strand_+_readouts_[NDB_896_c,NDB_896_c,NDB_896_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_18:78735001-78765000_gene_1241_pb_212_pos_21143_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_253_pos_26196_strand_-_readouts_[NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_259_pos_26520_strand_-_readouts_[NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_261_pos_26668_strand_-_readouts_[NDB_856_c,NDB_856_c,NDB_856_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_262_pos_26721_strand_-_readouts_[NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_263_pos_26805_strand_-_readouts_[NDB_862_c,NDB_862_c,NDB_862_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_264_pos_26859_strand_-_readouts_[NDB_856_c,NDB_856_c,NDB_856_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_270_pos_27156_strand_-_readouts_[NDB_856_c,NDB_856_c,NDB_856_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_275_pos_27512_strand_-_readouts_[NDB_862_c,NDB_862_c,NDB_862_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_19:4485001-4515000_gene_1246_pb_283_pos_28065_strand_-_readouts_[NDB_737_c,NDB_737_c,NDB_737_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_19:5485001-5515000_gene_1247_pb_105_pos_13735_strand_+_readouts_[NDB_652_c,NDB_652_c,NDB_652_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_19:5485001-5515000_gene_1247_pb_106_pos_13837_strand_-_readouts_[NDB_715_c,NDB_715_c,NDB_715_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_19:30485001-30515000_gene_1272_pb_150_pos_15605_strand_-_readouts_[NDB_958_c,NDB_958_c,NDB_958_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_19:31485001-31515000_gene_1273_pb_212_pos_21958_strand_+_readouts_[NDB_940_c,NDB_940_c,NDB_940_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_19:31485001-31515000_gene_1273_pb_214_pos_22419_strand_+_readouts_[NDB_818_c,NDB_818_c,NDB_818_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_19:33485001-33515000_gene_1275_pb_176_pos_20742_strand_-_readouts_[NDB_641_c,NDB_641_c,NDB_641_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_19:33485001-33515000_gene_1275_pb_177_pos_20787_strand_+_readouts_[NDB_868_c,NDB_868_c,NDB_868_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_19:33485001-33515000_gene_1275_pb_178_pos_20900_strand_+_readouts_[NDB_883_c,NDB_883_c,NDB_883_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:74, dumped\n",
      "--- Sequence:loc_19:33485001-33515000_gene_1275_pb_180_pos_21183_strand_-_readouts_[NDB_868_c,NDB_868_c,NDB_868_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_19:33485001-33515000_gene_1275_pb_181_pos_21456_strand_+_readouts_[NDB_883_c,NDB_883_c,NDB_883_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_19:33485001-33515000_gene_1275_pb_182_pos_21579_strand_+_readouts_[NDB_641_c,NDB_641_c,NDB_641_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:64, dumped\n",
      "--- Sequence:loc_19:33485001-33515000_gene_1275_pb_183_pos_21869_strand_+_readouts_[NDB_868_c,NDB_868_c,NDB_868_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:66, dumped\n",
      "--- Sequence:loc_19:47485001-47515000_gene_1289_pb_139_pos_16691_strand_+_readouts_[NDB_637_c,NDB_637_c,NDB_637_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_20:8485001-8515000_gene_1308_pb_72_pos_8628_strand_-_readouts_[NDB_718_c,NDB_718_c,NDB_718_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n",
      "--- Sequence:loc_20:8485001-8515000_gene_1308_pb_73_pos_8794_strand_+_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_20:8485001-8515000_gene_1308_pb_74_pos_8867_strand_+_readouts_[NDB_910_c,NDB_910_c,NDB_910_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_20:8485001-8515000_gene_1308_pb_76_pos_9020_strand_-_readouts_[NDB_683_c,NDB_683_c,NDB_683_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:81, dumped\n",
      "--- Sequence:loc_20:18485001-18515000_gene_1318_pb_125_pos_16718_strand_+_readouts_[NDB_1060_c,NDB_1060_c,NDB_1060_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_20:18485001-18515000_gene_1318_pb_126_pos_16781_strand_-_readouts_[NDB_847_c,NDB_847_c,NDB_847_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_20:18485001-18515000_gene_1318_pb_127_pos_16829_strand_-_readouts_[NDB_790_c,NDB_790_c,NDB_790_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_20:25485001-25515000_gene_1325_pb_155_pos_18146_strand_+_readouts_[NDB_940_c,NDB_940_c,NDB_940_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_20:25485001-25515000_gene_1325_pb_156_pos_18203_strand_-_readouts_[NDB_718_c,NDB_718_c,NDB_718_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:106, dumped\n",
      "--- Sequence:loc_20:25485001-25515000_gene_1325_pb_157_pos_18322_strand_+_readouts_[NDB_751_c,NDB_751_c,NDB_751_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_20:25485001-25515000_gene_1325_pb_158_pos_18437_strand_-_readouts_[NDB_940_c,NDB_940_c,NDB_940_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:107, dumped\n",
      "--- Sequence:loc_20:25485001-25515000_gene_1325_pb_159_pos_18484_strand_+_readouts_[NDB_718_c,NDB_718_c,NDB_718_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:85, dumped\n",
      "--- Sequence:loc_20:25485001-25515000_gene_1325_pb_160_pos_18551_strand_-_readouts_[NDB_751_c,NDB_751_c,NDB_751_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:107, dumped\n",
      "--- Sequence:loc_20:37485001-37515000_gene_1337_pb_188_pos_21169_strand_+_readouts_[NDB_712_c,NDB_712_c,NDB_712_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_20:48485001-48515000_gene_1348_pb_98_pos_20150_strand_+_readouts_[NDB_925_c,NDB_925_c,NDB_925_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:65, dumped\n",
      "--- Sequence:loc_20:48485001-48515000_gene_1348_pb_99_pos_21022_strand_+_readouts_[NDB_743_c,NDB_743_c,NDB_743_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_20:48485001-48515000_gene_1348_pb_102_pos_22426_strand_+_readouts_[NDB_743_c,NDB_743_c,NDB_743_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_20:48485001-48515000_gene_1348_pb_111_pos_25553_strand_+_readouts_[NDB_743_c,NDB_743_c,NDB_743_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_20:48485001-48515000_gene_1348_pb_122_pos_27710_strand_+_readouts_[NDB_925_c,NDB_925_c,NDB_925_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:86, dumped\n",
      "--- Sequence:loc_20:48485001-48515000_gene_1348_pb_127_pos_29552_strand_-_readouts_[NDB_694_c,NDB_694_c,NDB_694_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_20:48485001-48515000_gene_1348_pb_129_pos_29957_strand_+_readouts_[NDB_743_c,NDB_743_c,NDB_743_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:88, dumped\n",
      "--- Sequence:loc_20:57485001-57515000_gene_1357_pb_54_pos_9610_strand_-_readouts_[NDB_661_c,NDB_661_c,NDB_661_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_20:57485001-57515000_gene_1357_pb_83_pos_13447_strand_-_readouts_[NDB_647_c,NDB_647_c,NDB_647_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:53, dumped\n",
      "--- Sequence:loc_20:60485001-60515000_gene_1360_pb_150_pos_15690_strand_-_readouts_[NDB_832_c,NDB_832_c,NDB_832_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_20:62485001-62515000_gene_1362_pb_124_pos_15898_strand_+_readouts_[NDB_887_c,NDB_887_c,NDB_887_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:98, dumped\n",
      "--- Sequence:loc_20:62485001-62515000_gene_1362_pb_125_pos_15942_strand_+_readouts_[NDB_836_c,NDB_836_c,NDB_836_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:94, dumped\n",
      "--- Sequence:loc_20:62485001-62515000_gene_1362_pb_126_pos_16086_strand_-_readouts_[NDB_680_c,NDB_680_c,NDB_680_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:97, dumped\n",
      "--- Sequence:loc_20:62485001-62515000_gene_1362_pb_127_pos_16142_strand_+_readouts_[NDB_887_c,NDB_887_c,NDB_887_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:98, dumped\n",
      "--- Sequence:loc_20:62485001-62515000_gene_1362_pb_130_pos_16331_strand_-_readouts_[NDB_887_c,NDB_887_c,NDB_887_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_20:62485001-62515000_gene_1362_pb_131_pos_16382_strand_-_readouts_[NDB_836_c,NDB_836_c,NDB_836_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_20:63485001-63515000_gene_1363_pb_254_pos_22445_strand_-_readouts_[NDB_880_c,NDB_880_c,NDB_880_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:82, dumped\n",
      "--- Sequence:loc_20:63485001-63515000_gene_1363_pb_255_pos_22519_strand_+_readouts_[NDB_754_c,NDB_754_c,NDB_754_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:55, dumped\n",
      "--- Sequence:loc_20:63485001-63515000_gene_1363_pb_257_pos_22640_strand_-_readouts_[NDB_880_c,NDB_880_c,NDB_880_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:79, dumped\n",
      "--- Sequence:loc_20:63485001-63515000_gene_1363_pb_258_pos_22706_strand_-_readouts_[NDB_754_c,NDB_754_c,NDB_754_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:69, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_133_pos_17824_strand_+_readouts_[NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:105, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_134_pos_17870_strand_-_readouts_[NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:132, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_136_pos_17963_strand_+_readouts_[NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:69, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_137_pos_18007_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:180, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_138_pos_18053_strand_+_readouts_[NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:105, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_139_pos_18097_strand_-_readouts_[NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:71, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_140_pos_18176_strand_-_readouts_[NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:75, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_141_pos_18222_strand_-_readouts_[NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_142_pos_18267_strand_+_readouts_[NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:177, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_143_pos_18314_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:104, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_144_pos_18450_strand_+_readouts_[NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:150, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_145_pos_18510_strand_-_readouts_[NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_146_pos_18567_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:102, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_147_pos_18648_strand_+_readouts_[NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:67, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_148_pos_18692_strand_+_readouts_[NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:124, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_149_pos_18736_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:88, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_150_pos_18785_strand_+_readouts_[NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:173, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_151_pos_18861_strand_-_readouts_[NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:73, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_152_pos_18906_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:119, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_154_pos_18994_strand_+_readouts_[NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:98, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_155_pos_19038_strand_-_readouts_[NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:51, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_156_pos_19151_strand_-_readouts_[NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:80, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_157_pos_19213_strand_-_readouts_[NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:134, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_158_pos_19284_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:84, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_159_pos_19334_strand_+_readouts_[NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:130, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_160_pos_19378_strand_+_readouts_[NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:134, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_161_pos_19454_strand_+_readouts_[NDB_742_c,NDB_742_c,NDB_742_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:99, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_162_pos_19509_strand_+_readouts_[NDB_631_c,NDB_631_c,NDB_631_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:97, dumped\n",
      "--- Sequence:loc_21:44485001-44515000_gene_1408_pb_163_pos_19690_strand_+_readouts_[NDB_853_c,NDB_853_c,NDB_853_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:109, dumped\n",
      "--- Sequence:loc_21:45485001-45515000_gene_1409_pb_14_pos_1380_strand_-_readouts_[NDB_734_c,NDB_734_c,NDB_734_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:54, dumped\n",
      "--- Sequence:loc_21:45485001-45515000_gene_1409_pb_149_pos_16939_strand_-_readouts_[NDB_734_c,NDB_734_c,NDB_734_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:59, dumped\n",
      "--- Sequence:loc_21:45485001-45515000_gene_1409_pb_150_pos_16986_strand_-_readouts_[NDB_716_c,NDB_716_c,NDB_716_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n",
      "--- Sequence:loc_21:45485001-45515000_gene_1409_pb_151_pos_17035_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_21:45485001-45515000_gene_1409_pb_152_pos_17079_strand_-_readouts_[NDB_734_c,NDB_734_c,NDB_734_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:72, dumped\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- Sequence:loc_22:29485001-29515000_gene_1439_pb_152_pos_24719_strand_-_readouts_[NDB_940_c,NDB_940_c,NDB_940_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:60, dumped\n",
      "--- Sequence:loc_22:29485001-29515000_gene_1439_pb_153_pos_24869_strand_+_readouts_[NDB_790_c,NDB_790_c,NDB_790_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "--- Sequence:loc_22:35485001-35515000_gene_1445_pb_73_pos_6569_strand_+_readouts_[NDB_716_c,NDB_716_c,NDB_716_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:52, dumped\n",
      "--- Sequence:loc_22:35485001-35515000_gene_1445_pb_74_pos_6800_strand_+_readouts_[NDB_832_c,NDB_832_c,NDB_832_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_22:35485001-35515000_gene_1445_pb_75_pos_6856_strand_+_readouts_[NDB_961_c,NDB_961_c,NDB_961_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:70, dumped\n",
      "--- Sequence:loc_22:35485001-35515000_gene_1445_pb_76_pos_6937_strand_+_readouts_[NDB_716_c,NDB_716_c,NDB_716_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:58, dumped\n",
      "--- Sequence:loc_22:39485001-39515000_gene_1449_pb_247_pos_25786_strand_+_readouts_[NDB_889_c,NDB_889_c,NDB_889_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:57, dumped\n",
      "--- Sequence:loc_22:39485001-39515000_gene_1449_pb_248_pos_25854_strand_+_readouts_[NDB_958_c,NDB_958_c,NDB_958_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:56, dumped\n",
      "--- Sequence:loc_22:49485001-49515000_gene_1459_pb_49_pos_4444_strand_+_readouts_[NDB_856_c,NDB_856_c,NDB_856_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:81, dumped\n",
      "--- Sequence:loc_22:49485001-49515000_gene_1459_pb_50_pos_4510_strand_+_readouts_[NDB_1066_c,NDB_1066_c,NDB_1066_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:76, dumped\n",
      "--- Sequence:loc_22:49485001-49515000_gene_1459_pb_52_pos_4620_strand_+_readouts_[NDB_856_c,NDB_856_c,NDB_856_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:68, dumped\n",
      "--- Sequence:loc_X:116235001-116265000_gene_1506_pb_77_pos_7640_strand_+_readouts_[NDB_958_c,NDB_958_c,NDB_958_c]_primers_[W1A03_primer_2,W1A10_primer_9] got hits:61, dumped\n",
      "-- total probes removed by internal screening: 441\n"
     ]
    }
   ],
   "source": [
    "# check between probes\n",
    "kept_records, removed_count = check._check_between_probes(full_records, int_map, \n",
    "                                                          primer_len=primer_len, \n",
    "                                                          target_len=target_len)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "id": "495f8a7d",
   "metadata": {},
   "outputs": [],
   "source": [
    "# clear memory\n",
    "del(int_map)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ef0c0c1c",
   "metadata": {},
   "source": [
    "### 4.1.6 finish checking, save"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 128,
   "id": "0b11845a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "317417 317858\n"
     ]
    }
   ],
   "source": [
    "print(len(kept_records), len(cand_probes))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "id": "56768f43",
   "metadata": {},
   "outputs": [],
   "source": [
    "# save kept records\n",
    "with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'w') as output_handle:\n",
    "    SeqIO.write(kept_records, output_handle, \"fasta\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4c40e93f",
   "metadata": {},
   "source": [
    "## 4.2 BLAST screening of probes\n",
    "\n",
    "create the local blast database by:\n",
    "```bash\n",
    "makeblastdb -in C:\\Users\\puzheng\\Documents\\Tools\\lowercase_masked\\Mus_musculus.GRCm38.dna_sm.merged.fa -dbtype nucl -parse_seqids -out GRCm38_ensembl -title \"Mouse chromosomes soft masked\" -blastdb_version 4![image.png](attachment:image.png)\n",
    "```\n",
    "\n",
    "The outputs are:\n",
    "```bash\n",
    "Building a new DB, current time: 03/30/2021 21:49:18\n",
    "New DB name:   C:\\Users\\puzheng\\Documents\\Tools\\GRCm38_ensembl\n",
    "New DB title:  Mouse chromosomes soft masked\n",
    "Sequence type: Nucleotide\n",
    "Keep MBits: T\n",
    "Maximum file size: 1000000000B\n",
    "Adding sequences from FASTA; added 66 sequences in 42.2668 seconds.\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "5045ff8f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<module 'ImageAnalysis3.library_tools.quality_check' from 'E:\\\\Users\\\\puzheng\\\\Documents\\\\ImageAnalysis3\\\\library_tools\\\\quality_check.py'>"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "bef107d6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "317417\n"
     ]
    }
   ],
   "source": [
    "if 'kept_records' not in locals() or len(kept_records) == 0:\n",
    "    kept_records = []\n",
    "    with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'r') as handle:\n",
    "        for record in SeqIO.parse(handle, \"fasta\"):\n",
    "            kept_records.append(record)\n",
    "print(len(kept_records))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "75e15498",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Blast probe against \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\human\\hg38\\blast_database\\hg38\n",
      "-- number of probes loaded: 317417\n",
      "-- processing region: 0 with 295 probes\n",
      "-- processing region: 1 with 287 probes\n",
      "-- processing region: 10 with 216 probes\n",
      "-- processing region: 100 with 300 probes\n",
      "-- processing region: 1000 with 300 probes\n",
      "-- processing region: 1001 with 300 probes\n",
      "-- processing region: 1002 with 300 probes\n",
      "-- processing region: 1003 with 287 probes\n",
      "-- processing region: 1004 with 283 probes\n",
      "-- processing region: 1005 with 292 probes\n",
      "-- processing region: 101 with 282 probes\n",
      "-- processing region: 102 with 240 probes\n",
      "-- processing region: 1020 with 52 probes\n",
      "-- processing region: 1023 with 217 probes\n",
      "-- processing region: 1024 with 242 probes\n",
      "-- processing region: 1025 with 300 probes\n",
      "-- processing region: 1029 with 235 probes\n",
      "-- processing region: 103 with 205 probes\n",
      "-- processing region: 1030 with 209 probes\n",
      "-- processing region: 1031 with 290 probes\n",
      "-- processing region: 1032 with 275 probes\n",
      "-- processing region: 1033 with 234 probes\n",
      "-- processing region: 1034 with 183 probes\n",
      "-- processing region: 1035 with 201 probes\n",
      "-- processing region: 1036 with 130 probes\n",
      "-- processing region: 1037 with 237 probes\n",
      "-- processing region: 1038 with 227 probes\n",
      "-- processing region: 1039 with 246 probes\n",
      "-- processing region: 104 with 119 probes\n",
      "-- processing region: 1040 with 300 probes\n",
      "-- processing region: 1041 with 254 probes\n",
      "-- processing region: 1042 with 300 probes\n",
      "-- processing region: 1043 with 122 probes\n",
      "-- processing region: 1044 with 253 probes\n",
      "-- processing region: 1045 with 300 probes\n",
      "-- processing region: 1046 with 263 probes\n",
      "-- processing region: 1047 with 300 probes\n",
      "-- processing region: 1048 with 292 probes\n",
      "-- processing region: 1049 with 176 probes\n",
      "-- processing region: 105 with 300 probes\n",
      "-- processing region: 1050 with 267 probes\n",
      "-- processing region: 1051 with 300 probes\n",
      "-- processing region: 1052 with 134 probes\n",
      "-- processing region: 1053 with 300 probes\n",
      "-- processing region: 1054 with 300 probes\n",
      "-- processing region: 1055 with 268 probes\n",
      "-- processing region: 1056 with 300 probes\n",
      "-- processing region: 1057 with 155 probes\n",
      "-- processing region: 1058 with 131 probes\n",
      "-- processing region: 1059 with 164 probes\n",
      "-- processing region: 106 with 244 probes\n",
      "-- processing region: 1060 with 300 probes\n",
      "-- processing region: 1061 with 300 probes\n",
      "-- processing region: 1062 with 240 probes\n",
      "-- processing region: 1063 with 266 probes\n",
      "-- processing region: 1064 with 300 probes\n",
      "-- processing region: 1065 with 300 probes\n",
      "-- processing region: 1066 with 207 probes\n",
      "-- processing region: 1067 with 204 probes\n",
      "-- processing region: 1068 with 147 probes\n",
      "-- processing region: 1069 with 160 probes\n",
      "-- processing region: 107 with 300 probes\n",
      "-- processing region: 1070 with 250 probes\n",
      "-- processing region: 1071 with 291 probes\n",
      "-- processing region: 1072 with 259 probes\n",
      "-- processing region: 1073 with 263 probes\n",
      "-- processing region: 1074 with 266 probes\n",
      "-- processing region: 1075 with 256 probes\n",
      "-- processing region: 1076 with 293 probes\n",
      "-- processing region: 1077 with 88 probes\n",
      "-- processing region: 1078 with 241 probes\n",
      "-- processing region: 1079 with 300 probes\n",
      "-- processing region: 108 with 213 probes\n",
      "-- processing region: 1080 with 300 probes\n",
      "-- processing region: 1081 with 298 probes\n",
      "-- processing region: 1082 with 263 probes\n",
      "-- processing region: 1083 with 300 probes\n",
      "-- processing region: 1085 with 300 probes\n",
      "-- processing region: 1087 with 300 probes\n",
      "-- processing region: 1089 with 207 probes\n",
      "-- processing region: 109 with 225 probes\n",
      "-- processing region: 1090 with 261 probes\n",
      "-- processing region: 1091 with 226 probes\n",
      "-- processing region: 1092 with 300 probes\n",
      "-- processing region: 1093 with 203 probes\n",
      "-- processing region: 1094 with 300 probes\n",
      "-- processing region: 1097 with 230 probes\n",
      "-- processing region: 11 with 212 probes\n",
      "-- processing region: 110 with 230 probes\n",
      "-- processing region: 1105 with 223 probes\n",
      "-- processing region: 1106 with 189 probes\n",
      "-- processing region: 1107 with 267 probes\n",
      "-- processing region: 1108 with 295 probes\n",
      "-- processing region: 1109 with 184 probes\n",
      "-- processing region: 111 with 300 probes\n",
      "-- processing region: 1110 with 300 probes\n",
      "-- processing region: 1111 with 300 probes\n",
      "-- processing region: 1112 with 300 probes\n",
      "-- processing region: 1113 with 266 probes\n",
      "-- processing region: 1114 with 215 probes\n",
      "-- processing region: 1115 with 253 probes\n",
      "-- processing region: 1116 with 184 probes\n",
      "-- processing region: 1117 with 288 probes\n",
      "-- processing region: 1118 with 231 probes\n",
      "-- processing region: 1119 with 300 probes\n",
      "-- processing region: 112 with 167 probes\n",
      "-- processing region: 1120 with 228 probes\n",
      "-- processing region: 1121 with 298 probes\n",
      "-- processing region: 1122 with 227 probes\n",
      "-- processing region: 1123 with 267 probes\n",
      "-- processing region: 1124 with 201 probes\n",
      "-- processing region: 1125 with 208 probes\n",
      "-- processing region: 1126 with 300 probes\n",
      "-- processing region: 1127 with 241 probes\n",
      "-- processing region: 1128 with 235 probes\n",
      "-- processing region: 1129 with 300 probes\n",
      "-- processing region: 1130 with 298 probes\n",
      "-- processing region: 1131 with 272 probes\n",
      "-- processing region: 1132 with 288 probes\n",
      "-- processing region: 1133 with 205 probes\n",
      "-- processing region: 1134 with 278 probes\n",
      "-- processing region: 1135 with 268 probes\n",
      "-- processing region: 1136 with 290 probes\n",
      "-- processing region: 1137 with 193 probes\n",
      "-- processing region: 1138 with 212 probes\n",
      "-- processing region: 1139 with 256 probes\n",
      "-- processing region: 114 with 215 probes\n",
      "-- processing region: 1140 with 242 probes\n",
      "-- processing region: 1141 with 300 probes\n",
      "-- processing region: 1142 with 263 probes\n",
      "-- processing region: 1143 with 206 probes\n",
      "-- processing region: 1144 with 122 probes\n",
      "-- processing region: 1145 with 215 probes\n",
      "-- processing region: 1146 with 94 probes\n",
      "-- processing region: 1147 with 257 probes\n",
      "-- processing region: 1148 with 101 probes\n",
      "-- processing region: 115 with 224 probes\n",
      "-- processing region: 1152 with 194 probes\n",
      "-- processing region: 1153 with 269 probes\n",
      "-- processing region: 1154 with 54 probes\n",
      "-- processing region: 1155 with 254 probes\n",
      "-- processing region: 1156 with 300 probes\n",
      "-- processing region: 1157 with 276 probes\n",
      "-- processing region: 116 with 228 probes\n",
      "-- processing region: 1160 with 300 probes\n",
      "-- processing region: 1162 with 222 probes\n",
      "-- processing region: 1163 with 272 probes\n",
      "-- processing region: 1165 with 270 probes\n",
      "-- processing region: 1166 with 300 probes\n",
      "-- processing region: 1167 with 192 probes\n",
      "-- processing region: 1168 with 300 probes\n",
      "-- processing region: 1169 with 185 probes\n",
      "-- processing region: 117 with 276 probes\n",
      "-- processing region: 1170 with 232 probes\n",
      "-- processing region: 1171 with 268 probes\n",
      "-- processing region: 1172 with 300 probes\n",
      "-- processing region: 1173 with 96 probes\n",
      "-- processing region: 1174 with 128 probes\n",
      "-- processing region: 1176 with 212 probes\n",
      "-- processing region: 1177 with 241 probes\n",
      "-- processing region: 1178 with 298 probes\n",
      "-- processing region: 1179 with 197 probes\n",
      "-- processing region: 118 with 216 probes\n",
      "-- processing region: 1180 with 213 probes\n",
      "-- processing region: 1181 with 271 probes\n",
      "-- processing region: 1182 with 291 probes\n",
      "-- processing region: 1183 with 300 probes\n",
      "-- processing region: 1184 with 300 probes\n",
      "-- processing region: 1186 with 265 probes\n",
      "-- processing region: 1187 with 285 probes\n",
      "-- processing region: 1188 with 77 probes\n",
      "-- processing region: 1189 with 171 probes\n",
      "-- processing region: 119 with 221 probes\n",
      "-- processing region: 1190 with 187 probes\n",
      "-- processing region: 1191 with 194 probes\n",
      "-- processing region: 1192 with 242 probes\n",
      "-- processing region: 1193 with 279 probes\n",
      "-- processing region: 1194 with 291 probes\n",
      "-- processing region: 1195 with 288 probes\n",
      "-- processing region: 1196 with 277 probes\n",
      "-- processing region: 1197 with 184 probes\n",
      "-- processing region: 12 with 240 probes\n",
      "-- processing region: 120 with 200 probes\n",
      "-- processing region: 1203 with 191 probes\n",
      "-- processing region: 1204 with 300 probes\n",
      "-- processing region: 1205 with 300 probes\n",
      "-- processing region: 1206 with 108 probes\n",
      "-- processing region: 1207 with 213 probes\n",
      "-- processing region: 1208 with 208 probes\n",
      "-- processing region: 1209 with 169 probes\n",
      "-- processing region: 121 with 295 probes\n",
      "-- processing region: 1210 with 270 probes\n",
      "-- processing region: 1211 with 300 probes\n",
      "-- processing region: 1212 with 278 probes\n",
      "-- processing region: 1213 with 293 probes\n",
      "-- processing region: 1214 with 300 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- processing region: 1215 with 221 probes\n",
      "-- processing region: 1216 with 216 probes\n",
      "-- processing region: 1217 with 190 probes\n",
      "-- processing region: 1218 with 235 probes\n",
      "-- processing region: 1219 with 300 probes\n",
      "-- processing region: 122 with 270 probes\n",
      "-- processing region: 1220 with 300 probes\n",
      "-- processing region: 1221 with 300 probes\n",
      "-- processing region: 1222 with 179 probes\n",
      "-- processing region: 1224 with 231 probes\n",
      "-- processing region: 1225 with 282 probes\n",
      "-- processing region: 1226 with 238 probes\n",
      "-- processing region: 1227 with 215 probes\n",
      "-- processing region: 1228 with 167 probes\n",
      "-- processing region: 1229 with 235 probes\n",
      "-- processing region: 123 with 182 probes\n",
      "-- processing region: 1230 with 223 probes\n",
      "-- processing region: 1231 with 241 probes\n",
      "-- processing region: 1232 with 237 probes\n",
      "-- processing region: 1233 with 223 probes\n",
      "-- processing region: 1234 with 169 probes\n",
      "-- processing region: 1235 with 241 probes\n",
      "-- processing region: 1236 with 271 probes\n",
      "-- processing region: 1237 with 228 probes\n",
      "-- processing region: 1238 with 287 probes\n",
      "-- processing region: 1239 with 249 probes\n",
      "-- processing region: 124 with 235 probes\n",
      "-- processing region: 1240 with 297 probes\n",
      "-- processing region: 1241 with 286 probes\n",
      "-- processing region: 1242 with 279 probes\n",
      "-- processing region: 1243 with 297 probes\n",
      "-- processing region: 1244 with 260 probes\n",
      "-- processing region: 1245 with 283 probes\n",
      "-- processing region: 1246 with 291 probes\n",
      "-- processing region: 1247 with 232 probes\n",
      "-- processing region: 1248 with 160 probes\n",
      "-- processing region: 1249 with 237 probes\n",
      "-- processing region: 125 with 251 probes\n",
      "-- processing region: 1250 with 300 probes\n",
      "-- processing region: 1251 with 181 probes\n",
      "-- processing region: 1252 with 181 probes\n",
      "-- processing region: 1253 with 246 probes\n",
      "-- processing region: 1254 with 178 probes\n",
      "-- processing region: 1255 with 300 probes\n",
      "-- processing region: 1256 with 186 probes\n",
      "-- processing region: 1257 with 173 probes\n",
      "-- processing region: 1258 with 168 probes\n",
      "-- processing region: 1259 with 216 probes\n",
      "-- processing region: 126 with 291 probes\n",
      "-- processing region: 1260 with 300 probes\n",
      "-- processing region: 1261 with 300 probes\n",
      "-- processing region: 1262 with 85 probes\n",
      "-- processing region: 1263 with 189 probes\n",
      "-- processing region: 1264 with 279 probes\n",
      "-- processing region: 1265 with 99 probes\n",
      "-- processing region: 1269 with 51 probes\n",
      "-- processing region: 127 with 222 probes\n",
      "-- processing region: 1270 with 300 probes\n",
      "-- processing region: 1271 with 300 probes\n",
      "-- processing region: 1272 with 299 probes\n",
      "-- processing region: 1273 with 295 probes\n",
      "-- processing region: 1274 with 192 probes\n",
      "-- processing region: 1275 with 293 probes\n",
      "-- processing region: 1276 with 244 probes\n",
      "-- processing region: 1277 with 300 probes\n",
      "-- processing region: 1278 with 150 probes\n",
      "-- processing region: 1279 with 193 probes\n",
      "-- processing region: 128 with 275 probes\n",
      "-- processing region: 1280 with 296 probes\n",
      "-- processing region: 1281 with 275 probes\n",
      "-- processing region: 1282 with 234 probes\n",
      "-- processing region: 1284 with 188 probes\n",
      "-- processing region: 1285 with 206 probes\n",
      "-- processing region: 1286 with 263 probes\n",
      "-- processing region: 1287 with 287 probes\n",
      "-- processing region: 1288 with 239 probes\n",
      "-- processing region: 1289 with 299 probes\n",
      "-- processing region: 129 with 216 probes\n",
      "-- processing region: 1290 with 300 probes\n",
      "-- processing region: 1291 with 273 probes\n",
      "-- processing region: 1292 with 248 probes\n",
      "-- processing region: 1293 with 300 probes\n",
      "-- processing region: 1294 with 90 probes\n",
      "-- processing region: 1295 with 167 probes\n",
      "-- processing region: 1297 with 300 probes\n",
      "-- processing region: 1298 with 282 probes\n",
      "-- processing region: 1299 with 267 probes\n",
      "-- processing region: 13 with 300 probes\n",
      "-- processing region: 130 with 162 probes\n",
      "-- processing region: 1300 with 266 probes\n",
      "-- processing region: 1301 with 252 probes\n",
      "-- processing region: 1302 with 220 probes\n",
      "-- processing region: 1303 with 174 probes\n",
      "-- processing region: 1304 with 268 probes\n",
      "-- processing region: 1305 with 297 probes\n",
      "-- processing region: 1306 with 285 probes\n",
      "-- processing region: 1307 with 271 probes\n",
      "-- processing region: 1308 with 239 probes\n",
      "-- processing region: 1309 with 300 probes\n",
      "-- processing region: 131 with 190 probes\n",
      "-- processing region: 1310 with 285 probes\n",
      "-- processing region: 1311 with 211 probes\n",
      "-- processing region: 1312 with 257 probes\n",
      "-- processing region: 1313 with 201 probes\n",
      "-- processing region: 1314 with 272 probes\n",
      "-- processing region: 1315 with 300 probes\n",
      "-- processing region: 1316 with 300 probes\n",
      "-- processing region: 1317 with 300 probes\n",
      "-- processing region: 1318 with 250 probes\n",
      "-- processing region: 1319 with 300 probes\n",
      "-- processing region: 132 with 192 probes\n",
      "-- processing region: 1320 with 286 probes\n",
      "-- processing region: 1321 with 300 probes\n",
      "-- processing region: 1322 with 300 probes\n",
      "-- processing region: 1323 with 300 probes\n",
      "-- processing region: 1324 with 300 probes\n",
      "-- processing region: 1325 with 294 probes\n",
      "-- processing region: 133 with 300 probes\n",
      "-- processing region: 1330 with 129 probes\n",
      "-- processing region: 1331 with 242 probes\n",
      "-- processing region: 1332 with 278 probes\n",
      "-- processing region: 1333 with 136 probes\n",
      "-- processing region: 1334 with 161 probes\n",
      "-- processing region: 1335 with 300 probes\n",
      "-- processing region: 1336 with 266 probes\n",
      "-- processing region: 1337 with 299 probes\n",
      "-- processing region: 1338 with 188 probes\n",
      "-- processing region: 1339 with 238 probes\n",
      "-- processing region: 134 with 237 probes\n",
      "-- processing region: 1340 with 300 probes\n",
      "-- processing region: 1341 with 300 probes\n",
      "-- processing region: 1342 with 261 probes\n",
      "-- processing region: 1343 with 201 probes\n",
      "-- processing region: 1344 with 274 probes\n",
      "-- processing region: 1345 with 300 probes\n",
      "-- processing region: 1346 with 298 probes\n",
      "-- processing region: 1347 with 179 probes\n",
      "-- processing region: 1348 with 123 probes\n",
      "-- processing region: 1349 with 300 probes\n",
      "-- processing region: 1350 with 265 probes\n",
      "-- processing region: 1351 with 300 probes\n",
      "-- processing region: 1352 with 243 probes\n",
      "-- processing region: 1353 with 192 probes\n",
      "-- processing region: 1354 with 278 probes\n",
      "-- processing region: 1355 with 187 probes\n",
      "-- processing region: 1356 with 291 probes\n",
      "-- processing region: 1357 with 246 probes\n",
      "-- processing region: 1358 with 300 probes\n",
      "-- processing region: 1359 with 267 probes\n",
      "-- processing region: 1360 with 299 probes\n",
      "-- processing region: 1361 with 300 probes\n",
      "-- processing region: 1362 with 294 probes\n",
      "-- processing region: 1363 with 296 probes\n",
      "-- processing region: 137 with 300 probes\n",
      "-- processing region: 1378 with 217 probes\n",
      "-- processing region: 1379 with 292 probes\n",
      "-- processing region: 138 with 285 probes\n",
      "-- processing region: 1380 with 192 probes\n",
      "-- processing region: 1381 with 214 probes\n",
      "-- processing region: 1382 with 210 probes\n",
      "-- processing region: 1383 with 230 probes\n",
      "-- processing region: 1384 with 148 probes\n",
      "-- processing region: 1385 with 163 probes\n",
      "-- processing region: 1386 with 128 probes\n",
      "-- processing region: 1387 with 190 probes\n",
      "-- processing region: 1388 with 176 probes\n",
      "-- processing region: 1389 with 251 probes\n",
      "-- processing region: 139 with 174 probes\n",
      "-- processing region: 1390 with 288 probes\n",
      "-- processing region: 1391 with 233 probes\n",
      "-- processing region: 1392 with 244 probes\n",
      "-- processing region: 1393 with 300 probes\n",
      "-- processing region: 1394 with 174 probes\n",
      "-- processing region: 1395 with 282 probes\n",
      "-- processing region: 1396 with 192 probes\n",
      "-- processing region: 1398 with 184 probes\n",
      "-- processing region: 1399 with 268 probes\n",
      "-- processing region: 14 with 176 probes\n",
      "-- processing region: 140 with 261 probes\n",
      "-- processing region: 1400 with 300 probes\n",
      "-- processing region: 1401 with 268 probes\n",
      "-- processing region: 1402 with 300 probes\n",
      "-- processing region: 1403 with 223 probes\n",
      "-- processing region: 1404 with 235 probes\n",
      "-- processing region: 1405 with 300 probes\n",
      "-- processing region: 1406 with 300 probes\n",
      "-- processing region: 1408 with 271 probes\n",
      "-- processing region: 1409 with 295 probes\n",
      "-- processing region: 141 with 74 probes\n",
      "-- processing region: 1427 with 253 probes\n",
      "-- processing region: 1429 with 266 probes\n",
      "-- processing region: 143 with 186 probes\n",
      "-- processing region: 1430 with 240 probes\n",
      "-- processing region: 1434 with 295 probes\n",
      "-- processing region: 1435 with 255 probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- processing region: 1436 with 298 probes\n",
      "-- processing region: 1437 with 300 probes\n",
      "-- processing region: 1438 with 174 probes\n",
      "-- processing region: 1439 with 196 probes\n",
      "-- processing region: 144 with 187 probes\n",
      "-- processing region: 1440 with 236 probes\n",
      "-- processing region: 1441 with 184 probes\n",
      "-- processing region: 1442 with 248 probes\n",
      "-- processing region: 1443 with 300 probes\n",
      "-- processing region: 1444 with 300 probes\n",
      "-- processing region: 1445 with 296 probes\n",
      "-- processing region: 1446 with 274 probes\n",
      "-- processing region: 1447 with 300 probes\n",
      "-- processing region: 1448 with 267 probes\n",
      "-- processing region: 1449 with 298 probes\n",
      "-- processing region: 145 with 178 probes\n",
      "-- processing region: 1450 with 211 probes\n",
      "-- processing region: 1451 with 255 probes\n",
      "-- processing region: 1452 with 113 probes\n",
      "-- processing region: 1453 with 300 probes\n",
      "-- processing region: 1454 with 277 probes\n",
      "-- processing region: 1455 with 290 probes\n",
      "-- processing region: 1456 with 270 probes\n",
      "-- processing region: 1457 with 281 probes\n",
      "-- processing region: 1458 with 300 probes\n",
      "-- processing region: 1459 with 297 probes\n",
      "-- processing region: 146 with 294 probes\n",
      "-- processing region: 1461 with 143 probes\n",
      "-- processing region: 1462 with 238 probes\n",
      "-- processing region: 1463 with 268 probes\n",
      "-- processing region: 1464 with 281 probes\n",
      "-- processing region: 1465 with 231 probes\n",
      "-- processing region: 1466 with 286 probes\n",
      "-- processing region: 1467 with 193 probes\n",
      "-- processing region: 1468 with 180 probes\n",
      "-- processing region: 1469 with 202 probes\n",
      "-- processing region: 147 with 300 probes\n",
      "-- processing region: 1470 with 194 probes\n",
      "-- processing region: 1471 with 221 probes\n",
      "-- processing region: 1472 with 278 probes\n",
      "-- processing region: 1473 with 117 probes\n",
      "-- processing region: 1474 with 173 probes\n",
      "-- processing region: 1475 with 234 probes\n",
      "-- processing region: 1476 with 197 probes\n",
      "-- processing region: 1477 with 300 probes\n",
      "-- processing region: 1478 with 230 probes\n",
      "-- processing region: 1479 with 216 probes\n",
      "-- processing region: 148 with 193 probes\n",
      "-- processing region: 1480 with 117 probes\n",
      "-- processing region: 1481 with 198 probes\n",
      "-- processing region: 1482 with 212 probes\n",
      "-- processing region: 1485 with 300 probes\n",
      "-- processing region: 1486 with 227 probes\n",
      "-- processing region: 1487 with 250 probes\n",
      "-- processing region: 1488 with 300 probes\n",
      "-- processing region: 1489 with 94 probes\n",
      "-- processing region: 149 with 300 probes\n",
      "-- processing region: 1490 with 128 probes\n",
      "-- processing region: 1491 with 239 probes\n",
      "-- processing region: 1492 with 238 probes\n",
      "-- processing region: 1493 with 129 probes\n",
      "-- processing region: 1494 with 189 probes\n",
      "-- processing region: 1495 with 199 probes\n",
      "-- processing region: 1496 with 72 probes\n",
      "-- processing region: 1497 with 230 probes\n",
      "-- processing region: 1498 with 146 probes\n",
      "-- processing region: 1499 with 160 probes\n",
      "-- processing region: 15 with 300 probes\n",
      "-- processing region: 150 with 240 probes\n",
      "-- processing region: 1500 with 300 probes\n",
      "-- processing region: 1501 with 253 probes\n",
      "-- processing region: 1502 with 139 probes\n",
      "-- processing region: 1503 with 186 probes\n",
      "-- processing region: 1504 with 285 probes\n",
      "-- processing region: 1505 with 239 probes\n",
      "-- processing region: 1506 with 162 probes\n",
      "-- processing region: 1507 with 242 probes\n",
      "-- processing region: 1508 with 189 probes\n",
      "-- processing region: 1509 with 188 probes\n",
      "-- processing region: 1510 with 176 probes\n",
      "-- processing region: 1511 with 218 probes\n",
      "-- processing region: 1512 with 248 probes\n",
      "-- processing region: 1513 with 245 probes\n",
      "-- processing region: 1514 with 262 probes\n",
      "-- processing region: 1515 with 300 probes\n",
      "-- processing region: 1516 with 103 probes\n",
      "-- processing region: 1517 with 232 probes\n",
      "-- processing region: 1518 with 217 probes\n",
      "-- processing region: 1519 with 239 probes\n",
      "-- processing region: 152 with 248 probes\n",
      "-- processing region: 1520 with 221 probes\n",
      "-- processing region: 1521 with 176 probes\n",
      "-- processing region: 1525 with 51 probes\n",
      "-- processing region: 1526 with 60 probes\n",
      "-- processing region: 1527 with 72 probes\n",
      "-- processing region: 1529 with 166 probes\n",
      "-- processing region: 153 with 300 probes\n",
      "-- processing region: 1530 with 218 probes\n",
      "-- processing region: 1533 with 69 probes\n",
      "-- processing region: 1534 with 149 probes\n",
      "-- processing region: 1535 with 187 probes\n",
      "-- processing region: 1536 with 187 probes\n",
      "-- processing region: 1537 with 170 probes\n",
      "-- processing region: 1538 with 164 probes\n",
      "-- processing region: 154 with 283 probes\n",
      "-- processing region: 1541 with 251 probes\n",
      "-- processing region: 1542 with 204 probes\n",
      "-- processing region: 1548 with 106 probes\n",
      "-- processing region: 155 with 252 probes\n",
      "-- processing region: 156 with 292 probes\n",
      "-- processing region: 157 with 255 probes\n",
      "-- processing region: 158 with 227 probes\n",
      "-- processing region: 159 with 231 probes\n",
      "-- processing region: 16 with 245 probes\n",
      "-- processing region: 160 with 136 probes\n",
      "-- processing region: 161 with 134 probes\n",
      "-- processing region: 162 with 300 probes\n",
      "-- processing region: 163 with 234 probes\n",
      "-- processing region: 164 with 300 probes\n",
      "-- processing region: 165 with 244 probes\n",
      "-- processing region: 166 with 299 probes\n",
      "-- processing region: 167 with 217 probes\n",
      "-- processing region: 168 with 300 probes\n",
      "-- processing region: 169 with 238 probes\n",
      "-- processing region: 17 with 203 probes\n",
      "-- processing region: 170 with 299 probes\n",
      "-- processing region: 171 with 272 probes\n",
      "-- processing region: 172 with 200 probes\n",
      "-- processing region: 173 with 173 probes\n",
      "-- processing region: 174 with 198 probes\n",
      "-- processing region: 175 with 300 probes\n",
      "-- processing region: 176 with 152 probes\n",
      "-- processing region: 177 with 279 probes\n",
      "-- processing region: 178 with 280 probes\n",
      "-- processing region: 179 with 230 probes\n",
      "-- processing region: 18 with 204 probes\n",
      "-- processing region: 180 with 226 probes\n",
      "-- processing region: 181 with 248 probes\n",
      "-- processing region: 182 with 93 probes\n",
      "-- processing region: 183 with 204 probes\n",
      "-- processing region: 184 with 173 probes\n",
      "-- processing region: 185 with 300 probes\n",
      "-- processing region: 186 with 300 probes\n",
      "-- processing region: 187 with 239 probes\n",
      "-- processing region: 188 with 233 probes\n",
      "-- processing region: 189 with 191 probes\n",
      "-- processing region: 19 with 300 probes\n",
      "-- processing region: 190 with 150 probes\n",
      "-- processing region: 191 with 274 probes\n",
      "-- processing region: 192 with 270 probes\n",
      "-- processing region: 193 with 300 probes\n",
      "-- processing region: 194 with 297 probes\n",
      "-- processing region: 195 with 199 probes\n",
      "-- processing region: 196 with 300 probes\n",
      "-- processing region: 197 with 251 probes\n",
      "-- processing region: 198 with 300 probes\n",
      "-- processing region: 199 with 275 probes\n",
      "-- processing region: 2 with 300 probes\n",
      "-- processing region: 20 with 206 probes\n",
      "-- processing region: 200 with 229 probes\n",
      "-- processing region: 201 with 300 probes\n",
      "-- processing region: 202 with 300 probes\n",
      "-- processing region: 203 with 190 probes\n",
      "-- processing region: 204 with 265 probes\n",
      "-- processing region: 205 with 241 probes\n",
      "-- processing region: 206 with 262 probes\n",
      "-- processing region: 207 with 300 probes\n",
      "-- processing region: 208 with 174 probes\n",
      "-- processing region: 209 with 180 probes\n",
      "-- processing region: 21 with 255 probes\n",
      "-- processing region: 210 with 300 probes\n",
      "-- processing region: 211 with 300 probes\n",
      "-- processing region: 212 with 300 probes\n",
      "-- processing region: 213 with 200 probes\n",
      "-- processing region: 214 with 219 probes\n",
      "-- processing region: 215 with 142 probes\n",
      "-- processing region: 216 with 300 probes\n",
      "-- processing region: 217 with 278 probes\n",
      "-- processing region: 218 with 252 probes\n",
      "-- processing region: 219 with 258 probes\n",
      "-- processing region: 22 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_22.fasta\n",
      "--- *blasting region: 22\n",
      "--- total time for blast 22: 17.166801691055298\n",
      "-- processing region: 220 with 214 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_220.fasta\n",
      "--- *blasting region: 220\n",
      "--- total time for blast 220: 14.315695762634277\n",
      "-- processing region: 221 with 256 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_221.fasta\n",
      "--- *blasting region: 221\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 221: 12.402216911315918\n",
      "-- processing region: 222 with 292 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_222.fasta\n",
      "--- *blasting region: 222\n",
      "--- total time for blast 222: 17.160659551620483\n",
      "-- processing region: 223 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_223.fasta\n",
      "--- *blasting region: 223\n",
      "--- total time for blast 223: 12.219984292984009\n",
      "-- processing region: 224 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_224.fasta\n",
      "--- *blasting region: 224\n",
      "--- total time for blast 224: 11.832507133483887\n",
      "-- processing region: 225 with 233 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_225.fasta\n",
      "--- *blasting region: 225\n",
      "--- total time for blast 225: 13.05338716506958\n",
      "-- processing region: 226 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_226.fasta\n",
      "--- *blasting region: 226\n",
      "--- total time for blast 226: 12.154352188110352\n",
      "-- processing region: 227 with 242 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_227.fasta\n",
      "--- *blasting region: 227\n",
      "--- total time for blast 227: 13.023585319519043\n",
      "-- processing region: 228 with 148 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_228.fasta\n",
      "--- *blasting region: 228\n",
      "--- total time for blast 228: 8.851688146591187\n",
      "-- processing region: 229 with 197 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_229.fasta\n",
      "--- *blasting region: 229\n",
      "--- total time for blast 229: 9.960451126098633\n",
      "-- processing region: 23 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_23.fasta\n",
      "--- *blasting region: 23\n",
      "--- total time for blast 23: 12.820537567138672\n",
      "-- processing region: 230 with 166 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_230.fasta\n",
      "--- *blasting region: 230\n",
      "--- total time for blast 230: 14.918058633804321\n",
      "-- processing region: 233 with 196 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_233.fasta\n",
      "--- *blasting region: 233\n",
      "--- total time for blast 233: 11.238606214523315\n",
      "-- processing region: 234 with 255 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_234.fasta\n",
      "--- *blasting region: 234\n",
      "--- total time for blast 234: 13.360327005386353\n",
      "-- processing region: 235 with 233 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_235.fasta\n",
      "--- *blasting region: 235\n",
      "--- total time for blast 235: 20.515861749649048\n",
      "-- processing region: 236 with 182 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_236.fasta\n",
      "--- *blasting region: 236\n",
      "--- total time for blast 236: 17.12877082824707\n",
      "-- processing region: 237 with 179 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_237.fasta\n",
      "--- *blasting region: 237\n",
      "--- total time for blast 237: 9.983517408370972\n",
      "-- processing region: 238 with 251 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_238.fasta\n",
      "--- *blasting region: 238\n",
      "--- total time for blast 238: 11.575196743011475\n",
      "-- processing region: 239 with 231 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_239.fasta\n",
      "--- *blasting region: 239\n",
      "--- total time for blast 239: 12.471686124801636\n",
      "-- processing region: 24 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_24.fasta\n",
      "--- *blasting region: 24\n",
      "--- total time for blast 24: 11.334409713745117\n",
      "-- processing region: 240 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_240.fasta\n",
      "--- *blasting region: 240\n",
      "--- total time for blast 240: 10.543800830841064\n",
      "-- processing region: 241 with 272 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_241.fasta\n",
      "--- *blasting region: 241\n",
      "--- total time for blast 241: 25.94383454322815\n",
      "-- processing region: 242 with 288 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_242.fasta\n",
      "--- *blasting region: 242\n",
      "--- total time for blast 242: 12.283382654190063\n",
      "-- processing region: 243 with 165 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_243.fasta\n",
      "--- *blasting region: 243\n",
      "--- total time for blast 243: 12.386510848999023\n",
      "-- processing region: 244 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_244.fasta\n",
      "--- *blasting region: 244\n",
      "--- total time for blast 244: 15.265943050384521\n",
      "-- processing region: 245 with 238 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_245.fasta\n",
      "--- *blasting region: 245\n",
      "--- total time for blast 245: 13.739700078964233\n",
      "-- processing region: 246 with 205 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_246.fasta\n",
      "--- *blasting region: 246\n",
      "--- total time for blast 246: 10.41929030418396\n",
      "-- processing region: 247 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_247.fasta\n",
      "--- *blasting region: 247\n",
      "--- total time for blast 247: 11.678228855133057\n",
      "-- processing region: 248 with 277 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_248.fasta\n",
      "--- *blasting region: 248\n",
      "--- total time for blast 248: 13.290191888809204\n",
      "-- processing region: 249 with 242 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_249.fasta\n",
      "--- *blasting region: 249\n",
      "--- total time for blast 249: 12.743911027908325\n",
      "-- processing region: 25 with 238 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_25.fasta\n",
      "--- *blasting region: 25\n",
      "--- total time for blast 25: 11.41175389289856\n",
      "-- processing region: 250 with 172 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_250.fasta\n",
      "--- *blasting region: 250\n",
      "--- total time for blast 250: 9.298581600189209\n",
      "-- processing region: 251 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_251.fasta\n",
      "--- *blasting region: 251\n",
      "--- total time for blast 251: 14.673905372619629\n",
      "-- processing region: 252 with 287 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_252.fasta\n",
      "--- *blasting region: 252\n",
      "--- total time for blast 252: 15.65293264389038\n",
      "-- processing region: 253 with 188 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_253.fasta\n",
      "--- *blasting region: 253\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 253: 13.196040391921997\n",
      "-- processing region: 254 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_254.fasta\n",
      "--- *blasting region: 254\n",
      "--- total time for blast 254: 11.911815881729126\n",
      "-- processing region: 255 with 225 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_255.fasta\n",
      "--- *blasting region: 255\n",
      "--- total time for blast 255: 11.930468797683716\n",
      "-- processing region: 256 with 218 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_256.fasta\n",
      "--- *blasting region: 256\n",
      "--- total time for blast 256: 9.774390935897827\n",
      "-- processing region: 257 with 288 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_257.fasta\n",
      "--- *blasting region: 257\n",
      "--- total time for blast 257: 14.836668014526367\n",
      "-- processing region: 258 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_258.fasta\n",
      "--- *blasting region: 258\n",
      "--- total time for blast 258: 30.677406787872314\n",
      "-- processing region: 259 with 160 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_259.fasta\n",
      "--- *blasting region: 259\n",
      "--- total time for blast 259: 10.453039646148682\n",
      "-- processing region: 26 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_26.fasta\n",
      "--- *blasting region: 26\n",
      "--- total time for blast 26: 14.538836479187012\n",
      "-- processing region: 260 with 162 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_260.fasta\n",
      "--- *blasting region: 260\n",
      "--- total time for blast 260: 18.763302326202393\n",
      "-- processing region: 261 with 159 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_261.fasta\n",
      "--- *blasting region: 261\n",
      "--- total time for blast 261: 10.293473243713379\n",
      "-- processing region: 262 with 179 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_262.fasta\n",
      "--- *blasting region: 262\n",
      "--- total time for blast 262: 9.289199829101562\n",
      "-- processing region: 263 with 299 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_263.fasta\n",
      "--- *blasting region: 263\n",
      "--- total time for blast 263: 14.769041061401367\n",
      "-- processing region: 264 with 259 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_264.fasta\n",
      "--- *blasting region: 264\n",
      "--- total time for blast 264: 13.192579746246338\n",
      "-- processing region: 265 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_265.fasta\n",
      "--- *blasting region: 265\n",
      "--- total time for blast 265: 13.008704900741577\n",
      "-- processing region: 266 with 251 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_266.fasta\n",
      "--- *blasting region: 266\n",
      "--- total time for blast 266: 12.16141152381897\n",
      "-- processing region: 267 with 263 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_267.fasta\n",
      "--- *blasting region: 267\n",
      "--- total time for blast 267: 12.226928472518921\n",
      "-- processing region: 268 with 175 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_268.fasta\n",
      "--- *blasting region: 268\n",
      "--- total time for blast 268: 32.70736813545227\n",
      "-- processing region: 269 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_269.fasta\n",
      "--- *blasting region: 269\n",
      "--- total time for blast 269: 11.423524618148804\n",
      "-- processing region: 27 with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_27.fasta\n",
      "--- *blasting region: 27\n",
      "--- total time for blast 27: 11.156268835067749\n",
      "-- processing region: 270 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_270.fasta\n",
      "--- *blasting region: 270\n",
      "--- total time for blast 270: 13.84995722770691\n",
      "-- processing region: 271 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_271.fasta\n",
      "--- *blasting region: 271\n",
      "--- total time for blast 271: 13.45970368385315\n",
      "-- processing region: 272 with 232 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_272.fasta\n",
      "--- *blasting region: 272\n",
      "--- total time for blast 272: 12.567035436630249\n",
      "-- processing region: 273 with 211 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_273.fasta\n",
      "--- *blasting region: 273\n",
      "--- total time for blast 273: 10.359297037124634\n",
      "-- processing region: 274 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_274.fasta\n",
      "--- *blasting region: 274\n",
      "--- total time for blast 274: 12.703562259674072\n",
      "-- processing region: 275 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_275.fasta\n",
      "--- *blasting region: 275\n",
      "--- total time for blast 275: 12.49679160118103\n",
      "-- processing region: 276 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_276.fasta\n",
      "--- *blasting region: 276\n",
      "--- total time for blast 276: 21.215625047683716\n",
      "-- processing region: 277 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_277.fasta\n",
      "--- *blasting region: 277\n",
      "--- total time for blast 277: 13.237363815307617\n",
      "-- processing region: 278 with 224 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_278.fasta\n",
      "--- *blasting region: 278\n",
      "--- total time for blast 278: 15.466113567352295\n",
      "-- processing region: 279 with 210 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_279.fasta\n",
      "--- *blasting region: 279\n",
      "--- total time for blast 279: 12.213085412979126\n",
      "-- processing region: 28 with 262 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_28.fasta\n",
      "--- *blasting region: 28\n",
      "--- total time for blast 28: 16.991304636001587\n",
      "-- processing region: 280 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_280.fasta\n",
      "--- *blasting region: 280\n",
      "--- total time for blast 280: 13.183019161224365\n",
      "-- processing region: 281 with 239 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_281.fasta\n",
      "--- *blasting region: 281\n",
      "--- total time for blast 281: 11.50216555595398\n",
      "-- processing region: 282 with 262 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_282.fasta\n",
      "--- *blasting region: 282\n",
      "--- total time for blast 282: 10.755021333694458\n",
      "-- processing region: 283 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_283.fasta\n",
      "--- *blasting region: 283\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 283: 15.906776428222656\n",
      "-- processing region: 284 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_284.fasta\n",
      "--- *blasting region: 284\n",
      "--- total time for blast 284: 11.548819541931152\n",
      "-- processing region: 285 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_285.fasta\n",
      "--- *blasting region: 285\n",
      "--- total time for blast 285: 12.743504047393799\n",
      "-- processing region: 286 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_286.fasta\n",
      "--- *blasting region: 286\n",
      "--- total time for blast 286: 12.945245742797852\n",
      "-- processing region: 287 with 194 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_287.fasta\n",
      "--- *blasting region: 287\n",
      "--- total time for blast 287: 10.447790622711182\n",
      "-- processing region: 288 with 249 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_288.fasta\n",
      "--- *blasting region: 288\n",
      "--- total time for blast 288: 13.254760265350342\n",
      "-- processing region: 289 with 250 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_289.fasta\n",
      "--- *blasting region: 289\n",
      "--- total time for blast 289: 12.907626867294312\n",
      "-- processing region: 29 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_29.fasta\n",
      "--- *blasting region: 29\n",
      "--- total time for blast 29: 11.23324728012085\n",
      "-- processing region: 290 with 253 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_290.fasta\n",
      "--- *blasting region: 290\n",
      "--- total time for blast 290: 14.679978370666504\n",
      "-- processing region: 291 with 195 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_291.fasta\n",
      "--- *blasting region: 291\n",
      "--- total time for blast 291: 10.389766216278076\n",
      "-- processing region: 292 with 226 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_292.fasta\n",
      "--- *blasting region: 292\n",
      "--- total time for blast 292: 10.456711292266846\n",
      "-- processing region: 293 with 157 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_293.fasta\n",
      "--- *blasting region: 293\n",
      "--- total time for blast 293: 11.743475914001465\n",
      "-- processing region: 295 with 209 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_295.fasta\n",
      "--- *blasting region: 295\n",
      "--- total time for blast 295: 12.434353828430176\n",
      "-- processing region: 296 with 274 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_296.fasta\n",
      "--- *blasting region: 296\n",
      "--- total time for blast 296: 15.186445474624634\n",
      "-- processing region: 297 with 194 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_297.fasta\n",
      "--- *blasting region: 297\n",
      "--- total time for blast 297: 10.662309408187866\n",
      "-- processing region: 298 with 253 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_298.fasta\n",
      "--- *blasting region: 298\n",
      "--- total time for blast 298: 13.480329513549805\n",
      "-- processing region: 299 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_299.fasta\n",
      "--- *blasting region: 299\n",
      "--- total time for blast 299: 6.461807012557983\n",
      "-- processing region: 3 with 242 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_3.fasta\n",
      "--- *blasting region: 3\n",
      "--- total time for blast 3: 11.232196807861328\n",
      "-- processing region: 30 with 263 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_30.fasta\n",
      "--- *blasting region: 30\n",
      "--- total time for blast 30: 16.138895273208618\n",
      "-- processing region: 300 with 151 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_300.fasta\n",
      "--- *blasting region: 300\n",
      "--- total time for blast 300: 10.465223550796509\n",
      "-- processing region: 302 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_302.fasta\n",
      "--- *blasting region: 302\n",
      "--- total time for blast 302: 12.340761184692383\n",
      "-- processing region: 303 with 213 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_303.fasta\n",
      "--- *blasting region: 303\n",
      "--- total time for blast 303: 12.595636367797852\n",
      "-- processing region: 304 with 232 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_304.fasta\n",
      "--- *blasting region: 304\n",
      "--- total time for blast 304: 12.474502801895142\n",
      "-- processing region: 305 with 222 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_305.fasta\n",
      "--- *blasting region: 305\n",
      "--- total time for blast 305: 11.850329160690308\n",
      "-- processing region: 306 with 242 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_306.fasta\n",
      "--- *blasting region: 306\n",
      "--- total time for blast 306: 14.209911346435547\n",
      "-- processing region: 307 with 230 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_307.fasta\n",
      "--- *blasting region: 307\n",
      "--- total time for blast 307: 12.316221237182617\n",
      "-- processing region: 308 with 203 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_308.fasta\n",
      "--- *blasting region: 308\n",
      "--- total time for blast 308: 9.99837327003479\n",
      "-- processing region: 309 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_309.fasta\n",
      "--- *blasting region: 309\n",
      "--- total time for blast 309: 12.566691398620605\n",
      "-- processing region: 31 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_31.fasta\n",
      "--- *blasting region: 31\n",
      "--- total time for blast 31: 13.276671171188354\n",
      "-- processing region: 310 with 166 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_310.fasta\n",
      "--- *blasting region: 310\n",
      "--- total time for blast 310: 9.554620742797852\n",
      "-- processing region: 311 with 267 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_311.fasta\n",
      "--- *blasting region: 311\n",
      "--- total time for blast 311: 11.408895015716553\n",
      "-- processing region: 312 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_312.fasta\n",
      "--- *blasting region: 312\n",
      "--- total time for blast 312: 13.130583047866821\n",
      "-- processing region: 313 with 182 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_313.fasta\n",
      "--- *blasting region: 313\n",
      "--- total time for blast 313: 10.913211822509766\n",
      "-- processing region: 314 with 253 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_314.fasta\n",
      "--- *blasting region: 314\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 314: 11.746732950210571\n",
      "-- processing region: 315 with 228 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_315.fasta\n",
      "--- *blasting region: 315\n",
      "--- total time for blast 315: 10.720430850982666\n",
      "-- processing region: 316 with 183 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_316.fasta\n",
      "--- *blasting region: 316\n",
      "--- total time for blast 316: 9.849338054656982\n",
      "-- processing region: 317 with 251 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_317.fasta\n",
      "--- *blasting region: 317\n",
      "--- total time for blast 317: 11.530102252960205\n",
      "-- processing region: 318 with 178 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_318.fasta\n",
      "--- *blasting region: 318\n",
      "--- total time for blast 318: 11.45658564567566\n",
      "-- processing region: 319 with 249 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_319.fasta\n",
      "--- *blasting region: 319\n",
      "--- total time for blast 319: 13.473633527755737\n",
      "-- processing region: 32 with 190 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_32.fasta\n",
      "--- *blasting region: 32\n",
      "--- total time for blast 32: 10.867369890213013\n",
      "-- processing region: 320 with 155 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_320.fasta\n",
      "--- *blasting region: 320\n",
      "--- total time for blast 320: 9.448590755462646\n",
      "-- processing region: 321 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_321.fasta\n",
      "--- *blasting region: 321\n",
      "--- total time for blast 321: 13.556326150894165\n",
      "-- processing region: 322 with 258 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_322.fasta\n",
      "--- *blasting region: 322\n",
      "--- total time for blast 322: 14.377999782562256\n",
      "-- processing region: 323 with 248 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_323.fasta\n",
      "--- *blasting region: 323\n",
      "--- total time for blast 323: 11.505647659301758\n",
      "-- processing region: 324 with 171 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_324.fasta\n",
      "--- *blasting region: 324\n",
      "--- total time for blast 324: 10.979480504989624\n",
      "-- processing region: 325 with 176 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_325.fasta\n",
      "--- *blasting region: 325\n",
      "--- total time for blast 325: 11.030146598815918\n",
      "-- processing region: 326 with 191 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_326.fasta\n",
      "--- *blasting region: 326\n",
      "--- total time for blast 326: 13.731928586959839\n",
      "-- processing region: 327 with 106 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_327.fasta\n",
      "--- *blasting region: 327\n",
      "--- total time for blast 327: 9.644863605499268\n",
      "-- processing region: 328 with 170 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_328.fasta\n",
      "--- *blasting region: 328\n",
      "--- total time for blast 328: 13.136351108551025\n",
      "-- processing region: 329 with 295 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_329.fasta\n",
      "--- *blasting region: 329\n",
      "--- total time for blast 329: 13.470232009887695\n",
      "-- processing region: 33 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_33.fasta\n",
      "--- *blasting region: 33\n",
      "--- total time for blast 33: 13.145335912704468\n",
      "-- processing region: 330 with 218 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_330.fasta\n",
      "--- *blasting region: 330\n",
      "--- total time for blast 330: 11.935566186904907\n",
      "-- processing region: 331 with 154 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_331.fasta\n",
      "--- *blasting region: 331\n",
      "--- total time for blast 331: 10.138142585754395\n",
      "-- processing region: 332 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_332.fasta\n",
      "--- *blasting region: 332\n",
      "--- total time for blast 332: 13.810239315032959\n",
      "-- processing region: 333 with 214 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_333.fasta\n",
      "--- *blasting region: 333\n",
      "--- total time for blast 333: 10.40946888923645\n",
      "-- processing region: 334 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_334.fasta\n",
      "--- *blasting region: 334\n",
      "--- total time for blast 334: 34.54782485961914\n",
      "-- processing region: 335 with 271 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_335.fasta\n",
      "--- *blasting region: 335\n",
      "--- total time for blast 335: 13.803612470626831\n",
      "-- processing region: 336 with 244 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_336.fasta\n",
      "--- *blasting region: 336\n",
      "--- total time for blast 336: 17.950642824172974\n",
      "-- processing region: 337 with 181 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_337.fasta\n",
      "--- *blasting region: 337\n",
      "--- total time for blast 337: 12.629153728485107\n",
      "-- processing region: 338 with 222 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_338.fasta\n",
      "--- *blasting region: 338\n",
      "--- total time for blast 338: 11.896459817886353\n",
      "-- processing region: 339 with 257 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_339.fasta\n",
      "--- *blasting region: 339\n",
      "--- total time for blast 339: 17.75067925453186\n",
      "-- processing region: 34 with 230 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_34.fasta\n",
      "--- *blasting region: 34\n",
      "--- total time for blast 34: 12.530975818634033\n",
      "-- processing region: 340 with 201 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_340.fasta\n",
      "--- *blasting region: 340\n",
      "--- total time for blast 340: 10.017180681228638\n",
      "-- processing region: 341 with 284 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_341.fasta\n",
      "--- *blasting region: 341\n",
      "--- total time for blast 341: 14.191501140594482\n",
      "-- processing region: 342 with 157 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_342.fasta\n",
      "--- *blasting region: 342\n",
      "--- total time for blast 342: 11.329412698745728\n",
      "-- processing region: 343 with 260 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_343.fasta\n",
      "--- *blasting region: 343\n",
      "--- total time for blast 343: 13.679201602935791\n",
      "-- processing region: 344 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_344.fasta\n",
      "--- *blasting region: 344\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 344: 11.510384798049927\n",
      "-- processing region: 345 with 225 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_345.fasta\n",
      "--- *blasting region: 345\n",
      "--- total time for blast 345: 10.707138776779175\n",
      "-- processing region: 346 with 273 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_346.fasta\n",
      "--- *blasting region: 346\n",
      "--- total time for blast 346: 10.291482210159302\n",
      "-- processing region: 347 with 202 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_347.fasta\n",
      "--- *blasting region: 347\n",
      "--- total time for blast 347: 10.538837671279907\n",
      "-- processing region: 348 with 223 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_348.fasta\n",
      "--- *blasting region: 348\n",
      "--- total time for blast 348: 13.493919849395752\n",
      "-- processing region: 349 with 161 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_349.fasta\n",
      "--- *blasting region: 349\n",
      "--- total time for blast 349: 10.110821723937988\n",
      "-- processing region: 35 with 203 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_35.fasta\n",
      "--- *blasting region: 35\n",
      "--- total time for blast 35: 13.56460428237915\n",
      "-- processing region: 350 with 288 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_350.fasta\n",
      "--- *blasting region: 350\n",
      "--- total time for blast 350: 12.808094263076782\n",
      "-- processing region: 351 with 294 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_351.fasta\n",
      "--- *blasting region: 351\n",
      "--- total time for blast 351: 41.57358384132385\n",
      "-- processing region: 352 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_352.fasta\n",
      "--- *blasting region: 352\n",
      "--- total time for blast 352: 14.88390564918518\n",
      "-- processing region: 353 with 158 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_353.fasta\n",
      "--- *blasting region: 353\n",
      "--- total time for blast 353: 14.618166208267212\n",
      "-- processing region: 354 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_354.fasta\n",
      "--- *blasting region: 354\n",
      "--- total time for blast 354: 14.764918565750122\n",
      "-- processing region: 355 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_355.fasta\n",
      "--- *blasting region: 355\n",
      "--- total time for blast 355: 10.968126773834229\n",
      "-- processing region: 356 with 257 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_356.fasta\n",
      "--- *blasting region: 356\n",
      "--- total time for blast 356: 14.433645725250244\n",
      "-- processing region: 357 with 170 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_357.fasta\n",
      "--- *blasting region: 357\n",
      "--- total time for blast 357: 9.342721462249756\n",
      "-- processing region: 358 with 216 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_358.fasta\n",
      "--- *blasting region: 358\n",
      "--- total time for blast 358: 17.458075284957886\n",
      "-- processing region: 359 with 179 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_359.fasta\n",
      "--- *blasting region: 359\n",
      "--- total time for blast 359: 10.772974729537964\n",
      "-- processing region: 36 with 154 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_36.fasta\n",
      "--- *blasting region: 36\n",
      "--- total time for blast 36: 11.732458114624023\n",
      "-- processing region: 360 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_360.fasta\n",
      "--- *blasting region: 360\n",
      "--- total time for blast 360: 11.269886255264282\n",
      "-- processing region: 361 with 175 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_361.fasta\n",
      "--- *blasting region: 361\n",
      "--- total time for blast 361: 8.838965892791748\n",
      "-- processing region: 362 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_362.fasta\n",
      "--- *blasting region: 362\n",
      "--- total time for blast 362: 13.922383069992065\n",
      "-- processing region: 363 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_363.fasta\n",
      "--- *blasting region: 363\n",
      "--- total time for blast 363: 13.436784982681274\n",
      "-- processing region: 364 with 253 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_364.fasta\n",
      "--- *blasting region: 364\n",
      "--- total time for blast 364: 10.990753412246704\n",
      "-- processing region: 365 with 284 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_365.fasta\n",
      "--- *blasting region: 365\n",
      "--- total time for blast 365: 12.661028385162354\n",
      "-- processing region: 366 with 214 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_366.fasta\n",
      "--- *blasting region: 366\n",
      "--- total time for blast 366: 10.750284194946289\n",
      "-- processing region: 367 with 102 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_367.fasta\n",
      "--- *blasting region: 367\n",
      "--- total time for blast 367: 14.864255905151367\n",
      "-- processing region: 368 with 64 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_368.fasta\n",
      "--- *blasting region: 368\n",
      "--- total time for blast 368: 18.9457790851593\n",
      "-- processing region: 37 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_37.fasta\n",
      "--- *blasting region: 37\n",
      "--- total time for blast 37: 14.731627702713013\n",
      "-- processing region: 370 with 158 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_370.fasta\n",
      "--- *blasting region: 370\n",
      "--- total time for blast 370: 8.62662959098816\n",
      "-- processing region: 371 with 84 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_371.fasta\n",
      "--- *blasting region: 371\n",
      "--- total time for blast 371: 11.130022525787354\n",
      "-- processing region: 372 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_372.fasta\n",
      "--- *blasting region: 372\n",
      "--- total time for blast 372: 11.303420782089233\n",
      "-- processing region: 373 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_373.fasta\n",
      "--- *blasting region: 373\n",
      "--- total time for blast 373: 11.015136003494263\n",
      "-- processing region: 374 with 224 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_374.fasta\n",
      "--- *blasting region: 374\n",
      "--- total time for blast 374: 11.223273754119873\n",
      "-- processing region: 375 with 226 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_375.fasta\n",
      "--- *blasting region: 375\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 375: 10.71120810508728\n",
      "-- processing region: 376 with 222 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_376.fasta\n",
      "--- *blasting region: 376\n",
      "--- total time for blast 376: 10.64453411102295\n",
      "-- processing region: 377 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_377.fasta\n",
      "--- *blasting region: 377\n",
      "--- total time for blast 377: 12.023329257965088\n",
      "-- processing region: 379 with 287 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_379.fasta\n",
      "--- *blasting region: 379\n",
      "--- total time for blast 379: 21.092720985412598\n",
      "-- processing region: 38 with 211 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_38.fasta\n",
      "--- *blasting region: 38\n",
      "--- total time for blast 38: 11.811304569244385\n",
      "-- processing region: 380 with 224 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_380.fasta\n",
      "--- *blasting region: 380\n",
      "--- total time for blast 380: 12.705012798309326\n",
      "-- processing region: 381 with 278 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_381.fasta\n",
      "--- *blasting region: 381\n",
      "--- total time for blast 381: 12.993234634399414\n",
      "-- processing region: 382 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_382.fasta\n",
      "--- *blasting region: 382\n",
      "--- total time for blast 382: 11.998948574066162\n",
      "-- processing region: 383 with 176 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_383.fasta\n",
      "--- *blasting region: 383\n",
      "--- total time for blast 383: 10.362824440002441\n",
      "-- processing region: 384 with 166 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_384.fasta\n",
      "--- *blasting region: 384\n",
      "--- total time for blast 384: 9.729597091674805\n",
      "-- processing region: 385 with 272 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_385.fasta\n",
      "--- *blasting region: 385\n",
      "--- total time for blast 385: 12.866636037826538\n",
      "-- processing region: 386 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_386.fasta\n",
      "--- *blasting region: 386\n",
      "--- total time for blast 386: 11.546873807907104\n",
      "-- processing region: 387 with 192 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_387.fasta\n",
      "--- *blasting region: 387\n",
      "--- total time for blast 387: 10.527787923812866\n",
      "-- processing region: 388 with 277 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_388.fasta\n",
      "--- *blasting region: 388\n",
      "--- total time for blast 388: 11.506422996520996\n",
      "-- processing region: 389 with 170 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_389.fasta\n",
      "--- *blasting region: 389\n",
      "--- total time for blast 389: 9.772910833358765\n",
      "-- processing region: 39 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_39.fasta\n",
      "--- *blasting region: 39\n",
      "--- total time for blast 39: 10.930376768112183\n",
      "-- processing region: 390 with 164 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_390.fasta\n",
      "--- *blasting region: 390\n",
      "--- total time for blast 390: 12.373038291931152\n",
      "-- processing region: 391 with 257 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_391.fasta\n",
      "--- *blasting region: 391\n",
      "--- total time for blast 391: 16.628339052200317\n",
      "-- processing region: 392 with 202 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_392.fasta\n",
      "--- *blasting region: 392\n",
      "--- total time for blast 392: 13.792670726776123\n",
      "-- processing region: 393 with 229 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_393.fasta\n",
      "--- *blasting region: 393\n",
      "--- total time for blast 393: 10.805343627929688\n",
      "-- processing region: 394 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_394.fasta\n",
      "--- *blasting region: 394\n",
      "--- total time for blast 394: 14.00006628036499\n",
      "-- processing region: 395 with 256 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_395.fasta\n",
      "--- *blasting region: 395\n",
      "--- total time for blast 395: 13.811997413635254\n",
      "-- processing region: 396 with 182 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_396.fasta\n",
      "--- *blasting region: 396\n",
      "--- total time for blast 396: 10.263386011123657\n",
      "-- processing region: 397 with 245 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_397.fasta\n",
      "--- *blasting region: 397\n",
      "--- total time for blast 397: 13.298110485076904\n",
      "-- processing region: 398 with 162 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_398.fasta\n",
      "--- *blasting region: 398\n",
      "--- total time for blast 398: 12.591817855834961\n",
      "-- processing region: 399 with 196 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_399.fasta\n",
      "--- *blasting region: 399\n",
      "--- total time for blast 399: 11.8880033493042\n",
      "-- processing region: 4 with 256 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_4.fasta\n",
      "--- *blasting region: 4\n",
      "--- total time for blast 4: 12.445030212402344\n",
      "-- processing region: 40 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_40.fasta\n",
      "--- *blasting region: 40\n",
      "--- total time for blast 40: 14.13437032699585\n",
      "-- processing region: 400 with 222 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_400.fasta\n",
      "--- *blasting region: 400\n",
      "--- total time for blast 400: 16.52639603614807\n",
      "-- processing region: 401 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_401.fasta\n",
      "--- *blasting region: 401\n",
      "--- total time for blast 401: 15.726693868637085\n",
      "-- processing region: 402 with 191 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_402.fasta\n",
      "--- *blasting region: 402\n",
      "--- total time for blast 402: 12.148091793060303\n",
      "-- processing region: 403 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_403.fasta\n",
      "--- *blasting region: 403\n",
      "--- total time for blast 403: 12.149739503860474\n",
      "-- processing region: 404 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_404.fasta\n",
      "--- *blasting region: 404\n",
      "--- total time for blast 404: 12.16305685043335\n",
      "-- processing region: 405 with 205 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_405.fasta\n",
      "--- *blasting region: 405\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 405: 10.012699604034424\n",
      "-- processing region: 406 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_406.fasta\n",
      "--- *blasting region: 406\n",
      "--- total time for blast 406: 12.061725616455078\n",
      "-- processing region: 407 with 299 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_407.fasta\n",
      "--- *blasting region: 407\n",
      "--- total time for blast 407: 15.941278219223022\n",
      "-- processing region: 408 with 162 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_408.fasta\n",
      "--- *blasting region: 408\n",
      "--- total time for blast 408: 11.155323266983032\n",
      "-- processing region: 409 with 221 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_409.fasta\n",
      "--- *blasting region: 409\n",
      "--- total time for blast 409: 17.68422532081604\n",
      "-- processing region: 410 with 271 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_410.fasta\n",
      "--- *blasting region: 410\n",
      "--- total time for blast 410: 13.897688388824463\n",
      "-- processing region: 411 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_411.fasta\n",
      "--- *blasting region: 411\n",
      "--- total time for blast 411: 12.75132942199707\n",
      "-- processing region: 412 with 212 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_412.fasta\n",
      "--- *blasting region: 412\n",
      "--- total time for blast 412: 25.673240184783936\n",
      "-- processing region: 413 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_413.fasta\n",
      "--- *blasting region: 413\n",
      "--- total time for blast 413: 14.360691547393799\n",
      "-- processing region: 414 with 188 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_414.fasta\n",
      "--- *blasting region: 414\n",
      "--- total time for blast 414: 11.073258399963379\n",
      "-- processing region: 415 with 252 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_415.fasta\n",
      "--- *blasting region: 415\n",
      "--- total time for blast 415: 16.5457661151886\n",
      "-- processing region: 416 with 252 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_416.fasta\n",
      "--- *blasting region: 416\n",
      "--- total time for blast 416: 11.943660020828247\n",
      "-- processing region: 417 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_417.fasta\n",
      "--- *blasting region: 417\n",
      "--- total time for blast 417: 12.754103899002075\n",
      "-- processing region: 418 with 200 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_418.fasta\n",
      "--- *blasting region: 418\n",
      "--- total time for blast 418: 10.877422571182251\n",
      "-- processing region: 419 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_419.fasta\n",
      "--- *blasting region: 419\n",
      "--- total time for blast 419: 12.92458438873291\n",
      "-- processing region: 42 with 201 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_42.fasta\n",
      "--- *blasting region: 42\n",
      "--- total time for blast 42: 11.685397624969482\n",
      "-- processing region: 421 with 212 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_421.fasta\n",
      "--- *blasting region: 421\n",
      "--- total time for blast 421: 16.852230548858643\n",
      "-- processing region: 422 with 296 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_422.fasta\n",
      "--- *blasting region: 422\n",
      "--- total time for blast 422: 13.610787868499756\n",
      "-- processing region: 423 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_423.fasta\n",
      "--- *blasting region: 423\n",
      "--- total time for blast 423: 15.056446552276611\n",
      "-- processing region: 424 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_424.fasta\n",
      "--- *blasting region: 424\n",
      "--- total time for blast 424: 12.642068147659302\n",
      "-- processing region: 425 with 221 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_425.fasta\n",
      "--- *blasting region: 425\n",
      "--- total time for blast 425: 10.01430058479309\n",
      "-- processing region: 426 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_426.fasta\n",
      "--- *blasting region: 426\n",
      "--- total time for blast 426: 13.317179679870605\n",
      "-- processing region: 427 with 217 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_427.fasta\n",
      "--- *blasting region: 427\n",
      "--- total time for blast 427: 14.892279148101807\n",
      "-- processing region: 428 with 243 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_428.fasta\n",
      "--- *blasting region: 428\n",
      "--- total time for blast 428: 11.480497121810913\n",
      "-- processing region: 429 with 193 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_429.fasta\n",
      "--- *blasting region: 429\n",
      "--- total time for blast 429: 11.741243124008179\n",
      "-- processing region: 43 with 235 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_43.fasta\n",
      "--- *blasting region: 43\n",
      "--- total time for blast 43: 10.599442720413208\n",
      "-- processing region: 430 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_430.fasta\n",
      "--- *blasting region: 430\n",
      "--- total time for blast 430: 12.33490777015686\n",
      "-- processing region: 431 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_431.fasta\n",
      "--- *blasting region: 431\n",
      "--- total time for blast 431: 14.155025720596313\n",
      "-- processing region: 432 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_432.fasta\n",
      "--- *blasting region: 432\n",
      "--- total time for blast 432: 13.511040925979614\n",
      "-- processing region: 435 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_435.fasta\n",
      "--- *blasting region: 435\n",
      "--- total time for blast 435: 11.338163137435913\n",
      "-- processing region: 436 with 294 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_436.fasta\n",
      "--- *blasting region: 436\n",
      "--- total time for blast 436: 12.757833480834961\n",
      "-- processing region: 437 with 188 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_437.fasta\n",
      "--- *blasting region: 437\n",
      "--- total time for blast 437: 20.603737115859985\n",
      "-- processing region: 438 with 206 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_438.fasta\n",
      "--- *blasting region: 438\n",
      "--- total time for blast 438: 10.581600427627563\n",
      "-- processing region: 439 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_439.fasta\n",
      "--- *blasting region: 439\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 439: 13.222516775131226\n",
      "-- processing region: 44 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_44.fasta\n",
      "--- *blasting region: 44\n",
      "--- total time for blast 44: 15.341411113739014\n",
      "-- processing region: 440 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_440.fasta\n",
      "--- *blasting region: 440\n",
      "--- total time for blast 440: 14.095738410949707\n",
      "-- processing region: 441 with 183 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_441.fasta\n",
      "--- *blasting region: 441\n",
      "--- total time for blast 441: 7.746721982955933\n",
      "-- processing region: 442 with 276 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_442.fasta\n",
      "--- *blasting region: 442\n",
      "--- total time for blast 442: 14.568716764450073\n",
      "-- processing region: 443 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_443.fasta\n",
      "--- *blasting region: 443\n",
      "--- total time for blast 443: 11.106611013412476\n",
      "-- processing region: 444 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_444.fasta\n",
      "--- *blasting region: 444\n",
      "--- total time for blast 444: 16.07619285583496\n",
      "-- processing region: 446 with 108 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_446.fasta\n",
      "--- *blasting region: 446\n",
      "--- total time for blast 446: 16.653868436813354\n",
      "-- processing region: 447 with 181 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_447.fasta\n",
      "--- *blasting region: 447\n",
      "--- total time for blast 447: 10.401949882507324\n",
      "-- processing region: 448 with 127 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_448.fasta\n",
      "--- *blasting region: 448\n",
      "--- total time for blast 448: 8.969688892364502\n",
      "-- processing region: 449 with 228 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_449.fasta\n",
      "--- *blasting region: 449\n",
      "--- total time for blast 449: 11.12748408317566\n",
      "-- processing region: 45 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_45.fasta\n",
      "--- *blasting region: 45\n",
      "--- total time for blast 45: 13.585134267807007\n",
      "-- processing region: 450 with 268 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_450.fasta\n",
      "--- *blasting region: 450\n",
      "--- total time for blast 450: 18.6248562335968\n",
      "-- processing region: 451 with 258 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_451.fasta\n",
      "--- *blasting region: 451\n",
      "--- total time for blast 451: 15.163258790969849\n",
      "-- processing region: 452 with 197 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_452.fasta\n",
      "--- *blasting region: 452\n",
      "--- total time for blast 452: 13.702560424804688\n",
      "-- processing region: 453 with 67 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_453.fasta\n",
      "--- *blasting region: 453\n",
      "--- total time for blast 453: 6.901477098464966\n",
      "-- processing region: 454 with 145 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_454.fasta\n",
      "--- *blasting region: 454\n",
      "--- total time for blast 454: 10.345715522766113\n",
      "-- processing region: 455 with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_455.fasta\n",
      "--- *blasting region: 455\n",
      "--- total time for blast 455: 11.999095439910889\n",
      "-- processing region: 456 with 217 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_456.fasta\n",
      "--- *blasting region: 456\n",
      "--- total time for blast 456: 10.914491891860962\n",
      "-- processing region: 457 with 207 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_457.fasta\n",
      "--- *blasting region: 457\n",
      "--- total time for blast 457: 13.02756142616272\n",
      "-- processing region: 458 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_458.fasta\n",
      "--- *blasting region: 458\n",
      "--- total time for blast 458: 12.715970039367676\n",
      "-- processing region: 459 with 180 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_459.fasta\n",
      "--- *blasting region: 459\n",
      "--- total time for blast 459: 9.478864908218384\n",
      "-- processing region: 46 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_46.fasta\n",
      "--- *blasting region: 46\n",
      "--- total time for blast 46: 12.701462030410767\n",
      "-- processing region: 460 with 202 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_460.fasta\n",
      "--- *blasting region: 460\n",
      "--- total time for blast 460: 12.044517040252686\n",
      "-- processing region: 461 with 209 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_461.fasta\n",
      "--- *blasting region: 461\n",
      "--- total time for blast 461: 12.420642375946045\n",
      "-- processing region: 462 with 170 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_462.fasta\n",
      "--- *blasting region: 462\n",
      "--- total time for blast 462: 9.268074989318848\n",
      "-- processing region: 463 with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_463.fasta\n",
      "--- *blasting region: 463\n",
      "--- total time for blast 463: 10.51385760307312\n",
      "-- processing region: 464 with 211 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_464.fasta\n",
      "--- *blasting region: 464\n",
      "--- total time for blast 464: 29.290029287338257\n",
      "-- processing region: 465 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_465.fasta\n",
      "--- *blasting region: 465\n",
      "--- total time for blast 465: 13.536603450775146\n",
      "-- processing region: 466 with 189 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_466.fasta\n",
      "--- *blasting region: 466\n",
      "--- total time for blast 466: 8.053005933761597\n",
      "-- processing region: 467 with 200 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_467.fasta\n",
      "--- *blasting region: 467\n",
      "--- total time for blast 467: 11.227879762649536\n",
      "-- processing region: 468 with 222 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_468.fasta\n",
      "--- *blasting region: 468\n",
      "--- total time for blast 468: 11.988088846206665\n",
      "-- processing region: 469 with 260 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_469.fasta\n",
      "--- *blasting region: 469\n",
      "--- total time for blast 469: 12.669059753417969\n",
      "-- processing region: 47 with 213 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_47.fasta\n",
      "--- *blasting region: 47\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 47: 12.653436183929443\n",
      "-- processing region: 470 with 131 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_470.fasta\n",
      "--- *blasting region: 470\n",
      "--- total time for blast 470: 7.9474122524261475\n",
      "-- processing region: 471 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_471.fasta\n",
      "--- *blasting region: 471\n",
      "--- total time for blast 471: 11.260337352752686\n",
      "-- processing region: 472 with 203 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_472.fasta\n",
      "--- *blasting region: 472\n",
      "--- total time for blast 472: 18.646443605422974\n",
      "-- processing region: 473 with 203 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_473.fasta\n",
      "--- *blasting region: 473\n",
      "--- total time for blast 473: 14.38745641708374\n",
      "-- processing region: 474 with 235 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_474.fasta\n",
      "--- *blasting region: 474\n",
      "--- total time for blast 474: 11.395555019378662\n",
      "-- processing region: 475 with 263 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_475.fasta\n",
      "--- *blasting region: 475\n",
      "--- total time for blast 475: 11.563387632369995\n",
      "-- processing region: 476 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_476.fasta\n",
      "--- *blasting region: 476\n",
      "--- total time for blast 476: 29.329899311065674\n",
      "-- processing region: 477 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_477.fasta\n",
      "--- *blasting region: 477\n",
      "--- total time for blast 477: 12.391119003295898\n",
      "-- processing region: 478 with 189 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_478.fasta\n",
      "--- *blasting region: 478\n",
      "--- total time for blast 478: 10.582887411117554\n",
      "-- processing region: 479 with 287 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_479.fasta\n",
      "--- *blasting region: 479\n",
      "--- total time for blast 479: 14.35974645614624\n",
      "-- processing region: 480 with 257 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_480.fasta\n",
      "--- *blasting region: 480\n",
      "--- total time for blast 480: 13.15907883644104\n",
      "-- processing region: 481 with 269 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_481.fasta\n",
      "--- *blasting region: 481\n",
      "--- total time for blast 481: 13.84635591506958\n",
      "-- processing region: 482 with 230 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_482.fasta\n",
      "--- *blasting region: 482\n",
      "--- total time for blast 482: 11.187995672225952\n",
      "-- processing region: 483 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_483.fasta\n",
      "--- *blasting region: 483\n",
      "--- total time for blast 483: 12.010859727859497\n",
      "-- processing region: 484 with 265 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_484.fasta\n",
      "--- *blasting region: 484\n",
      "--- total time for blast 484: 15.23006272315979\n",
      "-- processing region: 485 with 254 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_485.fasta\n",
      "--- *blasting region: 485\n",
      "--- total time for blast 485: 13.801854133605957\n",
      "-- processing region: 486 with 294 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_486.fasta\n",
      "--- *blasting region: 486\n",
      "--- total time for blast 486: 19.518824338912964\n",
      "-- processing region: 487 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_487.fasta\n",
      "--- *blasting region: 487\n",
      "--- total time for blast 487: 12.578721761703491\n",
      "-- processing region: 489 with 216 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_489.fasta\n",
      "--- *blasting region: 489\n",
      "--- total time for blast 489: 10.637588739395142\n",
      "-- processing region: 490 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_490.fasta\n",
      "--- *blasting region: 490\n",
      "--- total time for blast 490: 10.741729259490967\n",
      "-- processing region: 491 with 265 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_491.fasta\n",
      "--- *blasting region: 491\n",
      "--- total time for blast 491: 12.629451036453247\n",
      "-- processing region: 492 with 229 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_492.fasta\n",
      "--- *blasting region: 492\n",
      "--- total time for blast 492: 11.797960758209229\n",
      "-- processing region: 493 with 239 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_493.fasta\n",
      "--- *blasting region: 493\n",
      "--- total time for blast 493: 12.92764925956726\n",
      "-- processing region: 494 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_494.fasta\n",
      "--- *blasting region: 494\n",
      "--- total time for blast 494: 14.736579656600952\n",
      "-- processing region: 495 with 259 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_495.fasta\n",
      "--- *blasting region: 495\n",
      "--- total time for blast 495: 12.658236503601074\n",
      "-- processing region: 496 with 167 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_496.fasta\n",
      "--- *blasting region: 496\n",
      "--- total time for blast 496: 10.245419979095459\n",
      "-- processing region: 497 with 228 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_497.fasta\n",
      "--- *blasting region: 497\n",
      "--- total time for blast 497: 17.26126503944397\n",
      "-- processing region: 498 with 262 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_498.fasta\n",
      "--- *blasting region: 498\n",
      "--- total time for blast 498: 14.275608777999878\n",
      "-- processing region: 499 with 162 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_499.fasta\n",
      "--- *blasting region: 499\n",
      "--- total time for blast 499: 12.729025602340698\n",
      "-- processing region: 5 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_5.fasta\n",
      "--- *blasting region: 5\n",
      "--- total time for blast 5: 12.617058515548706\n",
      "-- processing region: 500 with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_500.fasta\n",
      "--- *blasting region: 500\n",
      "--- total time for blast 500: 9.486621141433716\n",
      "-- processing region: 501 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_501.fasta\n",
      "--- *blasting region: 501\n",
      "--- total time for blast 501: 15.503056764602661\n",
      "-- processing region: 502 with 269 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_502.fasta\n",
      "--- *blasting region: 502\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 502: 13.75308609008789\n",
      "-- processing region: 503 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_503.fasta\n",
      "--- *blasting region: 503\n",
      "--- total time for blast 503: 14.589881181716919\n",
      "-- processing region: 504 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_504.fasta\n",
      "--- *blasting region: 504\n",
      "--- total time for blast 504: 15.697477340698242\n",
      "-- processing region: 505 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_505.fasta\n",
      "--- *blasting region: 505\n",
      "--- total time for blast 505: 13.156829595565796\n",
      "-- processing region: 506 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_506.fasta\n",
      "--- *blasting region: 506\n",
      "--- total time for blast 506: 13.455999612808228\n",
      "-- processing region: 507 with 202 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_507.fasta\n",
      "--- *blasting region: 507\n",
      "--- total time for blast 507: 12.252530574798584\n",
      "-- processing region: 508 with 221 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_508.fasta\n",
      "--- *blasting region: 508\n",
      "--- total time for blast 508: 10.582678318023682\n",
      "-- processing region: 509 with 272 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_509.fasta\n",
      "--- *blasting region: 509\n",
      "--- total time for blast 509: 16.371089935302734\n",
      "-- processing region: 510 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_510.fasta\n",
      "--- *blasting region: 510\n",
      "--- total time for blast 510: 16.77577567100525\n",
      "-- processing region: 511 with 184 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_511.fasta\n",
      "--- *blasting region: 511\n",
      "--- total time for blast 511: 15.359434127807617\n",
      "-- processing region: 512 with 197 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_512.fasta\n",
      "--- *blasting region: 512\n",
      "--- total time for blast 512: 12.61323881149292\n",
      "-- processing region: 516 with 227 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_516.fasta\n",
      "--- *blasting region: 516\n",
      "--- total time for blast 516: 11.738361120223999\n",
      "-- processing region: 517 with 248 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_517.fasta\n",
      "--- *blasting region: 517\n",
      "--- total time for blast 517: 14.566091060638428\n",
      "-- processing region: 518 with 264 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_518.fasta\n",
      "--- *blasting region: 518\n",
      "--- total time for blast 518: 15.137150287628174\n",
      "-- processing region: 519 with 188 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_519.fasta\n",
      "--- *blasting region: 519\n",
      "--- total time for blast 519: 24.161128759384155\n",
      "-- processing region: 520 with 212 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_520.fasta\n",
      "--- *blasting region: 520\n",
      "--- total time for blast 520: 15.729046821594238\n",
      "-- processing region: 521 with 265 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_521.fasta\n",
      "--- *blasting region: 521\n",
      "--- total time for blast 521: 11.712713956832886\n",
      "-- processing region: 522 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_522.fasta\n",
      "--- *blasting region: 522\n",
      "--- total time for blast 522: 9.686869621276855\n",
      "-- processing region: 523 with 152 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_523.fasta\n",
      "--- *blasting region: 523\n",
      "--- total time for blast 523: 6.945757150650024\n",
      "-- processing region: 524 with 196 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_524.fasta\n",
      "--- *blasting region: 524\n",
      "--- total time for blast 524: 10.065438032150269\n",
      "-- processing region: 525 with 216 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_525.fasta\n",
      "--- *blasting region: 525\n",
      "--- total time for blast 525: 11.564059257507324\n",
      "-- processing region: 526 with 223 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_526.fasta\n",
      "--- *blasting region: 526\n",
      "--- total time for blast 526: 15.427335262298584\n",
      "-- processing region: 527 with 255 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_527.fasta\n",
      "--- *blasting region: 527\n",
      "--- total time for blast 527: 13.355796575546265\n",
      "-- processing region: 528 with 249 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_528.fasta\n",
      "--- *blasting region: 528\n",
      "--- total time for blast 528: 11.159199237823486\n",
      "-- processing region: 529 with 269 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_529.fasta\n",
      "--- *blasting region: 529\n",
      "--- total time for blast 529: 10.88641881942749\n",
      "-- processing region: 530 with 223 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_530.fasta\n",
      "--- *blasting region: 530\n",
      "--- total time for blast 530: 12.911388874053955\n",
      "-- processing region: 531 with 201 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_531.fasta\n",
      "--- *blasting region: 531\n",
      "--- total time for blast 531: 8.635958433151245\n",
      "-- processing region: 532 with 232 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_532.fasta\n",
      "--- *blasting region: 532\n",
      "--- total time for blast 532: 9.743180274963379\n",
      "-- processing region: 533 with 201 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_533.fasta\n",
      "--- *blasting region: 533\n",
      "--- total time for blast 533: 18.582168102264404\n",
      "-- processing region: 534 with 181 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_534.fasta\n",
      "--- *blasting region: 534\n",
      "--- total time for blast 534: 15.38303017616272\n",
      "-- processing region: 535 with 116 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_535.fasta\n",
      "--- *blasting region: 535\n",
      "--- total time for blast 535: 6.590914726257324\n",
      "-- processing region: 536 with 259 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_536.fasta\n",
      "--- *blasting region: 536\n",
      "--- total time for blast 536: 24.000820875167847\n",
      "-- processing region: 537 with 231 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_537.fasta\n",
      "--- *blasting region: 537\n",
      "--- total time for blast 537: 12.832486391067505\n",
      "-- processing region: 538 with 214 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_538.fasta\n",
      "--- *blasting region: 538\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 538: 11.440531015396118\n",
      "-- processing region: 539 with 242 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_539.fasta\n",
      "--- *blasting region: 539\n",
      "--- total time for blast 539: 10.701436042785645\n",
      "-- processing region: 540 with 179 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_540.fasta\n",
      "--- *blasting region: 540\n",
      "--- total time for blast 540: 11.630674362182617\n",
      "-- processing region: 541 with 248 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_541.fasta\n",
      "--- *blasting region: 541\n",
      "--- total time for blast 541: 11.41148829460144\n",
      "-- processing region: 542 with 227 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_542.fasta\n",
      "--- *blasting region: 542\n",
      "--- total time for blast 542: 10.7734215259552\n",
      "-- processing region: 543 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_543.fasta\n",
      "--- *blasting region: 543\n",
      "--- total time for blast 543: 17.77323031425476\n",
      "-- processing region: 544 with 276 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_544.fasta\n",
      "--- *blasting region: 544\n",
      "--- total time for blast 544: 13.48601222038269\n",
      "-- processing region: 545 with 267 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_545.fasta\n",
      "--- *blasting region: 545\n",
      "--- total time for blast 545: 12.273153305053711\n",
      "-- processing region: 546 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_546.fasta\n",
      "--- *blasting region: 546\n",
      "--- total time for blast 546: 15.612211465835571\n",
      "-- processing region: 548 with 191 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_548.fasta\n",
      "--- *blasting region: 548\n",
      "--- total time for blast 548: 11.485734939575195\n",
      "-- processing region: 549 with 206 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_549.fasta\n",
      "--- *blasting region: 549\n",
      "--- total time for blast 549: 15.067681550979614\n",
      "-- processing region: 550 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_550.fasta\n",
      "--- *blasting region: 550\n",
      "--- total time for blast 550: 13.160256385803223\n",
      "-- processing region: 551 with 90 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_551.fasta\n",
      "--- *blasting region: 551\n",
      "--- total time for blast 551: 4.782567024230957\n",
      "-- processing region: 552 with 264 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_552.fasta\n",
      "--- *blasting region: 552\n",
      "--- total time for blast 552: 11.862942934036255\n",
      "-- processing region: 554 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_554.fasta\n",
      "--- *blasting region: 554\n",
      "--- total time for blast 554: 15.440935850143433\n",
      "-- processing region: 555 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_555.fasta\n",
      "--- *blasting region: 555\n",
      "--- total time for blast 555: 16.54758620262146\n",
      "-- processing region: 556 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_556.fasta\n",
      "--- *blasting region: 556\n",
      "--- total time for blast 556: 13.035853624343872\n",
      "-- processing region: 557 with 273 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_557.fasta\n",
      "--- *blasting region: 557\n",
      "--- total time for blast 557: 14.702041625976562\n",
      "-- processing region: 558 with 251 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_558.fasta\n",
      "--- *blasting region: 558\n",
      "--- total time for blast 558: 14.027366876602173\n",
      "-- processing region: 559 with 221 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_559.fasta\n",
      "--- *blasting region: 559\n",
      "--- total time for blast 559: 11.918697118759155\n",
      "-- processing region: 560 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_560.fasta\n",
      "--- *blasting region: 560\n",
      "--- total time for blast 560: 13.432191371917725\n",
      "-- processing region: 561 with 240 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_561.fasta\n",
      "--- *blasting region: 561\n",
      "--- total time for blast 561: 16.132096529006958\n",
      "-- processing region: 562 with 249 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_562.fasta\n",
      "--- *blasting region: 562\n",
      "--- total time for blast 562: 13.70287299156189\n",
      "-- processing region: 563 with 193 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_563.fasta\n",
      "--- *blasting region: 563\n",
      "--- total time for blast 563: 10.230208158493042\n",
      "-- processing region: 564 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_564.fasta\n",
      "--- *blasting region: 564\n",
      "--- total time for blast 564: 12.682912826538086\n",
      "-- processing region: 565 with 218 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_565.fasta\n",
      "--- *blasting region: 565\n",
      "--- total time for blast 565: 13.924404621124268\n",
      "-- processing region: 566 with 169 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_566.fasta\n",
      "--- *blasting region: 566\n",
      "--- total time for blast 566: 10.514867544174194\n",
      "-- processing region: 567 with 229 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_567.fasta\n",
      "--- *blasting region: 567\n",
      "--- total time for blast 567: 11.139307260513306\n",
      "-- processing region: 568 with 222 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_568.fasta\n",
      "--- *blasting region: 568\n",
      "--- total time for blast 568: 12.69591474533081\n",
      "-- processing region: 569 with 292 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_569.fasta\n",
      "--- *blasting region: 569\n",
      "--- total time for blast 569: 13.459124565124512\n",
      "-- processing region: 57 with 60 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_57.fasta\n",
      "--- *blasting region: 57\n",
      "--- total time for blast 57: 4.273186683654785\n",
      "-- processing region: 570 with 104 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_570.fasta\n",
      "--- *blasting region: 570\n",
      "--- total time for blast 570: 13.430460214614868\n",
      "-- processing region: 571 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_571.fasta\n",
      "--- *blasting region: 571\n",
      "--- total time for blast 571: 14.576584100723267\n",
      "-- processing region: 572 with 280 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_572.fasta\n",
      "--- *blasting region: 572\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 572: 15.364760160446167\n",
      "-- processing region: 573 with 206 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_573.fasta\n",
      "--- *blasting region: 573\n",
      "--- total time for blast 573: 13.91887640953064\n",
      "-- processing region: 574 with 258 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_574.fasta\n",
      "--- *blasting region: 574\n",
      "--- total time for blast 574: 14.44568920135498\n",
      "-- processing region: 575 with 204 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_575.fasta\n",
      "--- *blasting region: 575\n",
      "--- total time for blast 575: 10.291867733001709\n",
      "-- processing region: 576 with 291 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_576.fasta\n",
      "--- *blasting region: 576\n",
      "--- total time for blast 576: 12.921540260314941\n",
      "-- processing region: 577 with 207 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_577.fasta\n",
      "--- *blasting region: 577\n",
      "--- total time for blast 577: 10.250953912734985\n",
      "-- processing region: 578 with 246 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_578.fasta\n",
      "--- *blasting region: 578\n",
      "--- total time for blast 578: 15.465978860855103\n",
      "-- processing region: 579 with 151 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_579.fasta\n",
      "--- *blasting region: 579\n",
      "--- total time for blast 579: 13.16679334640503\n",
      "-- processing region: 580 with 202 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_580.fasta\n",
      "--- *blasting region: 580\n",
      "--- total time for blast 580: 9.494128942489624\n",
      "-- processing region: 581 with 250 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_581.fasta\n",
      "--- *blasting region: 581\n",
      "--- total time for blast 581: 19.276670217514038\n",
      "-- processing region: 582 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_582.fasta\n",
      "--- *blasting region: 582\n",
      "--- total time for blast 582: 10.265313863754272\n",
      "-- processing region: 583 with 181 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_583.fasta\n",
      "--- *blasting region: 583\n",
      "--- total time for blast 583: 11.089465618133545\n",
      "-- processing region: 584 with 139 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_584.fasta\n",
      "--- *blasting region: 584\n",
      "--- total time for blast 584: 14.093388795852661\n",
      "-- processing region: 585 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_585.fasta\n",
      "--- *blasting region: 585\n",
      "--- total time for blast 585: 11.560665369033813\n",
      "-- processing region: 586 with 218 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_586.fasta\n",
      "--- *blasting region: 586\n",
      "--- total time for blast 586: 10.54097580909729\n",
      "-- processing region: 587 with 156 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_587.fasta\n",
      "--- *blasting region: 587\n",
      "--- total time for blast 587: 11.948889255523682\n",
      "-- processing region: 588 with 177 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_588.fasta\n",
      "--- *blasting region: 588\n",
      "--- total time for blast 588: 9.834037780761719\n",
      "-- processing region: 589 with 275 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_589.fasta\n",
      "--- *blasting region: 589\n",
      "--- total time for blast 589: 11.846688985824585\n",
      "-- processing region: 590 with 225 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_590.fasta\n",
      "--- *blasting region: 590\n",
      "--- total time for blast 590: 10.613848686218262\n",
      "-- processing region: 591 with 283 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_591.fasta\n",
      "--- *blasting region: 591\n",
      "--- total time for blast 591: 12.586905002593994\n",
      "-- processing region: 592 with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_592.fasta\n",
      "--- *blasting region: 592\n",
      "--- total time for blast 592: 14.457770109176636\n",
      "-- processing region: 593 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_593.fasta\n",
      "--- *blasting region: 593\n",
      "--- total time for blast 593: 25.609933376312256\n",
      "-- processing region: 594 with 155 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_594.fasta\n",
      "--- *blasting region: 594\n",
      "--- total time for blast 594: 9.554550170898438\n",
      "-- processing region: 595 with 180 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_595.fasta\n",
      "--- *blasting region: 595\n",
      "--- total time for blast 595: 14.0115327835083\n",
      "-- processing region: 596 with 254 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_596.fasta\n",
      "--- *blasting region: 596\n",
      "--- total time for blast 596: 11.528194665908813\n",
      "-- processing region: 597 with 277 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_597.fasta\n",
      "--- *blasting region: 597\n",
      "--- total time for blast 597: 18.29120135307312\n",
      "-- processing region: 598 with 226 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_598.fasta\n",
      "--- *blasting region: 598\n",
      "--- total time for blast 598: 13.566559076309204\n",
      "-- processing region: 599 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_599.fasta\n",
      "--- *blasting region: 599\n",
      "--- total time for blast 599: 13.798959016799927\n",
      "-- processing region: 6 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_6.fasta\n",
      "--- *blasting region: 6\n",
      "--- total time for blast 6: 16.890007257461548\n",
      "-- processing region: 60 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_60.fasta\n",
      "--- *blasting region: 60\n",
      "--- total time for blast 60: 9.951392650604248\n",
      "-- processing region: 600 with 294 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_600.fasta\n",
      "--- *blasting region: 600\n",
      "--- total time for blast 600: 12.623144149780273\n",
      "-- processing region: 601 with 299 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_601.fasta\n",
      "--- *blasting region: 601\n",
      "--- total time for blast 601: 14.483458995819092\n",
      "-- processing region: 602 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_602.fasta\n",
      "--- *blasting region: 602\n",
      "--- total time for blast 602: 13.370126724243164\n",
      "-- processing region: 603 with 251 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_603.fasta\n",
      "--- *blasting region: 603\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 603: 12.269195079803467\n",
      "-- processing region: 605 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_605.fasta\n",
      "--- *blasting region: 605\n",
      "--- total time for blast 605: 10.249423027038574\n",
      "-- processing region: 606 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_606.fasta\n",
      "--- *blasting region: 606\n",
      "--- total time for blast 606: 16.251372814178467\n",
      "-- processing region: 607 with 281 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_607.fasta\n",
      "--- *blasting region: 607\n",
      "--- total time for blast 607: 15.5404212474823\n",
      "-- processing region: 608 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_608.fasta\n",
      "--- *blasting region: 608\n",
      "--- total time for blast 608: 14.031540155410767\n",
      "-- processing region: 609 with 285 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_609.fasta\n",
      "--- *blasting region: 609\n",
      "--- total time for blast 609: 11.806691646575928\n",
      "-- processing region: 611 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_611.fasta\n",
      "--- *blasting region: 611\n",
      "--- total time for blast 611: 13.641173839569092\n",
      "-- processing region: 612 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_612.fasta\n",
      "--- *blasting region: 612\n",
      "--- total time for blast 612: 13.321877717971802\n",
      "-- processing region: 613 with 200 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_613.fasta\n",
      "--- *blasting region: 613\n",
      "--- total time for blast 613: 10.329439878463745\n",
      "-- processing region: 614 with 277 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_614.fasta\n",
      "--- *blasting region: 614\n",
      "--- total time for blast 614: 12.64187479019165\n",
      "-- processing region: 615 with 258 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_615.fasta\n",
      "--- *blasting region: 615\n",
      "--- total time for blast 615: 12.18522024154663\n",
      "-- processing region: 616 with 173 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_616.fasta\n",
      "--- *blasting region: 616\n",
      "--- total time for blast 616: 12.593449354171753\n",
      "-- processing region: 617 with 254 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_617.fasta\n",
      "--- *blasting region: 617\n",
      "--- total time for blast 617: 15.238195896148682\n",
      "-- processing region: 618 with 192 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_618.fasta\n",
      "--- *blasting region: 618\n",
      "--- total time for blast 618: 9.403937339782715\n",
      "-- processing region: 619 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_619.fasta\n",
      "--- *blasting region: 619\n",
      "--- total time for blast 619: 12.144018173217773\n",
      "-- processing region: 62 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_62.fasta\n",
      "--- *blasting region: 62\n",
      "--- total time for blast 62: 11.004381656646729\n",
      "-- processing region: 620 with 196 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_620.fasta\n",
      "--- *blasting region: 620\n",
      "--- total time for blast 620: 11.516700983047485\n",
      "-- processing region: 621 with 248 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_621.fasta\n",
      "--- *blasting region: 621\n",
      "--- total time for blast 621: 11.844389200210571\n",
      "-- processing region: 622 with 247 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_622.fasta\n",
      "--- *blasting region: 622\n",
      "--- total time for blast 622: 14.758130550384521\n",
      "-- processing region: 623 with 185 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_623.fasta\n",
      "--- *blasting region: 623\n",
      "--- total time for blast 623: 11.849069833755493\n",
      "-- processing region: 624 with 132 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_624.fasta\n",
      "--- *blasting region: 624\n",
      "--- total time for blast 624: 6.658967018127441\n",
      "-- processing region: 625 with 168 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_625.fasta\n",
      "--- *blasting region: 625\n",
      "--- total time for blast 625: 8.659702777862549\n",
      "-- processing region: 626 with 118 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_626.fasta\n",
      "--- *blasting region: 626\n",
      "--- total time for blast 626: 18.775667667388916\n",
      "-- processing region: 627 with 253 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_627.fasta\n",
      "--- *blasting region: 627\n",
      "--- total time for blast 627: 18.008201599121094\n",
      "-- processing region: 628 with 273 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_628.fasta\n",
      "--- *blasting region: 628\n",
      "--- total time for blast 628: 11.11997127532959\n",
      "-- processing region: 629 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_629.fasta\n",
      "--- *blasting region: 629\n",
      "--- total time for blast 629: 15.206885814666748\n",
      "-- processing region: 63 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_63.fasta\n",
      "--- *blasting region: 63\n",
      "--- total time for blast 63: 12.315100908279419\n",
      "-- processing region: 64 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_64.fasta\n",
      "--- *blasting region: 64\n",
      "--- total time for blast 64: 9.670558214187622\n",
      "-- processing region: 645 with 257 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_645.fasta\n",
      "--- *blasting region: 645\n",
      "--- total time for blast 645: 11.401451110839844\n",
      "-- processing region: 646 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_646.fasta\n",
      "--- *blasting region: 646\n",
      "--- total time for blast 646: 10.987350702285767\n",
      "-- processing region: 647 with 219 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_647.fasta\n",
      "--- *blasting region: 647\n",
      "--- total time for blast 647: 12.146694421768188\n",
      "-- processing region: 648 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_648.fasta\n",
      "--- *blasting region: 648\n",
      "--- total time for blast 648: 10.579355001449585\n",
      "-- processing region: 649 with 262 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_649.fasta\n",
      "--- *blasting region: 649\n",
      "--- total time for blast 649: 13.225941896438599\n",
      "-- processing region: 65 with 114 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_65.fasta\n",
      "--- *blasting region: 65\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 65: 11.213362455368042\n",
      "-- processing region: 650 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_650.fasta\n",
      "--- *blasting region: 650\n",
      "--- total time for blast 650: 12.30185604095459\n",
      "-- processing region: 651 with 146 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_651.fasta\n",
      "--- *blasting region: 651\n",
      "--- total time for blast 651: 9.416022539138794\n",
      "-- processing region: 652 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_652.fasta\n",
      "--- *blasting region: 652\n",
      "--- total time for blast 652: 14.274945735931396\n",
      "-- processing region: 653 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_653.fasta\n",
      "--- *blasting region: 653\n",
      "--- total time for blast 653: 13.796114683151245\n",
      "-- processing region: 654 with 271 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_654.fasta\n",
      "--- *blasting region: 654\n",
      "--- total time for blast 654: 15.114470720291138\n",
      "-- processing region: 655 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_655.fasta\n",
      "--- *blasting region: 655\n",
      "--- total time for blast 655: 10.783444166183472\n",
      "-- processing region: 656 with 292 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_656.fasta\n",
      "--- *blasting region: 656\n",
      "--- total time for blast 656: 13.081571578979492\n",
      "-- processing region: 657 with 186 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_657.fasta\n",
      "--- *blasting region: 657\n",
      "--- total time for blast 657: 9.704237461090088\n",
      "-- processing region: 658 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_658.fasta\n",
      "--- *blasting region: 658\n",
      "--- total time for blast 658: 15.07975721359253\n",
      "-- processing region: 659 with 54 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_659.fasta\n",
      "--- *blasting region: 659\n",
      "--- total time for blast 659: 4.219146966934204\n",
      "-- processing region: 66 with 127 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_66.fasta\n",
      "--- *blasting region: 66\n",
      "--- total time for blast 66: 8.571601390838623\n",
      "-- processing region: 660 with 278 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_660.fasta\n",
      "--- *blasting region: 660\n",
      "--- total time for blast 660: 12.430772066116333\n",
      "-- processing region: 662 with 184 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_662.fasta\n",
      "--- *blasting region: 662\n",
      "--- total time for blast 662: 15.41485857963562\n",
      "-- processing region: 663 with 205 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_663.fasta\n",
      "--- *blasting region: 663\n",
      "--- total time for blast 663: 13.853206872940063\n",
      "-- processing region: 664 with 255 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_664.fasta\n",
      "--- *blasting region: 664\n",
      "--- total time for blast 664: 11.027223587036133\n",
      "-- processing region: 665 with 266 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_665.fasta\n",
      "--- *blasting region: 665\n",
      "--- total time for blast 665: 11.47400689125061\n",
      "-- processing region: 666 with 273 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_666.fasta\n",
      "--- *blasting region: 666\n",
      "--- total time for blast 666: 14.315999746322632\n",
      "-- processing region: 667 with 172 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_667.fasta\n",
      "--- *blasting region: 667\n",
      "--- total time for blast 667: 15.212234020233154\n",
      "-- processing region: 668 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_668.fasta\n",
      "--- *blasting region: 668\n",
      "--- total time for blast 668: 12.823610305786133\n",
      "-- processing region: 669 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_669.fasta\n",
      "--- *blasting region: 669\n",
      "--- total time for blast 669: 15.107369899749756\n",
      "-- processing region: 67 with 269 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_67.fasta\n",
      "--- *blasting region: 67\n",
      "--- total time for blast 67: 12.416603803634644\n",
      "-- processing region: 670 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_670.fasta\n",
      "--- *blasting region: 670\n",
      "--- total time for blast 670: 13.228532791137695\n",
      "-- processing region: 671 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_671.fasta\n",
      "--- *blasting region: 671\n",
      "--- total time for blast 671: 11.395549774169922\n",
      "-- processing region: 672 with 171 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_672.fasta\n",
      "--- *blasting region: 672\n",
      "--- total time for blast 672: 9.661417007446289\n",
      "-- processing region: 673 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_673.fasta\n",
      "--- *blasting region: 673\n",
      "--- total time for blast 673: 9.391346216201782\n",
      "-- processing region: 674 with 258 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_674.fasta\n",
      "--- *blasting region: 674\n",
      "--- total time for blast 674: 10.035605669021606\n",
      "-- processing region: 675 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_675.fasta\n",
      "--- *blasting region: 675\n",
      "--- total time for blast 675: 13.763378143310547\n",
      "-- processing region: 676 with 299 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_676.fasta\n",
      "--- *blasting region: 676\n",
      "--- total time for blast 676: 15.814012289047241\n",
      "-- processing region: 677 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_677.fasta\n",
      "--- *blasting region: 677\n",
      "--- total time for blast 677: 12.371113538742065\n",
      "-- processing region: 678 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_678.fasta\n",
      "--- *blasting region: 678\n",
      "--- total time for blast 678: 13.218759298324585\n",
      "-- processing region: 679 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_679.fasta\n",
      "--- *blasting region: 679\n",
      "--- total time for blast 679: 21.465256214141846\n",
      "-- processing region: 68 with 226 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_68.fasta\n",
      "--- *blasting region: 68\n",
      "--- total time for blast 68: 10.660309314727783\n",
      "-- processing region: 680 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_680.fasta\n",
      "--- *blasting region: 680\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 680: 13.789268493652344\n",
      "-- processing region: 681 with 298 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_681.fasta\n",
      "--- *blasting region: 681\n",
      "--- total time for blast 681: 13.361067295074463\n",
      "-- processing region: 682 with 147 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_682.fasta\n",
      "--- *blasting region: 682\n",
      "--- total time for blast 682: 8.378686428070068\n",
      "-- processing region: 683 with 285 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_683.fasta\n",
      "--- *blasting region: 683\n",
      "--- total time for blast 683: 12.08499526977539\n",
      "-- processing region: 684 with 253 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_684.fasta\n",
      "--- *blasting region: 684\n",
      "--- total time for blast 684: 15.0111403465271\n",
      "-- processing region: 685 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_685.fasta\n",
      "--- *blasting region: 685\n",
      "--- total time for blast 685: 14.189312219619751\n",
      "-- processing region: 686 with 266 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_686.fasta\n",
      "--- *blasting region: 686\n",
      "--- total time for blast 686: 13.680232048034668\n",
      "-- processing region: 687 with 78 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_687.fasta\n",
      "--- *blasting region: 687\n",
      "--- total time for blast 687: 5.156846046447754\n",
      "-- processing region: 688 with 278 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_688.fasta\n",
      "--- *blasting region: 688\n",
      "--- total time for blast 688: 11.567736864089966\n",
      "-- processing region: 689 with 160 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_689.fasta\n",
      "--- *blasting region: 689\n",
      "--- total time for blast 689: 10.52768087387085\n",
      "-- processing region: 69 with 128 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_69.fasta\n",
      "--- *blasting region: 69\n",
      "--- total time for blast 69: 7.818705081939697\n",
      "-- processing region: 690 with 289 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_690.fasta\n",
      "--- *blasting region: 690\n",
      "--- total time for blast 690: 12.183085918426514\n",
      "-- processing region: 691 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_691.fasta\n",
      "--- *blasting region: 691\n",
      "--- total time for blast 691: 11.561654806137085\n",
      "-- processing region: 692 with 189 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_692.fasta\n",
      "--- *blasting region: 692\n",
      "--- total time for blast 692: 16.878424644470215\n",
      "-- processing region: 693 with 246 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_693.fasta\n",
      "--- *blasting region: 693\n",
      "--- total time for blast 693: 11.410962104797363\n",
      "-- processing region: 694 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_694.fasta\n",
      "--- *blasting region: 694\n",
      "--- total time for blast 694: 12.199119091033936\n",
      "-- processing region: 695 with 246 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_695.fasta\n",
      "--- *blasting region: 695\n",
      "--- total time for blast 695: 13.07269549369812\n",
      "-- processing region: 696 with 170 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_696.fasta\n",
      "--- *blasting region: 696\n",
      "--- total time for blast 696: 10.552067041397095\n",
      "-- processing region: 697 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_697.fasta\n",
      "--- *blasting region: 697\n",
      "--- total time for blast 697: 10.965169668197632\n",
      "-- processing region: 698 with 227 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_698.fasta\n",
      "--- *blasting region: 698\n",
      "--- total time for blast 698: 10.924590349197388\n",
      "-- processing region: 699 with 66 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_699.fasta\n",
      "--- *blasting region: 699\n",
      "--- total time for blast 699: 9.971873760223389\n",
      "-- processing region: 7 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_7.fasta\n",
      "--- *blasting region: 7\n",
      "--- total time for blast 7: 13.519081830978394\n",
      "-- processing region: 70 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_70.fasta\n",
      "--- *blasting region: 70\n",
      "--- total time for blast 70: 7.828425645828247\n",
      "-- processing region: 701 with 289 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_701.fasta\n",
      "--- *blasting region: 701\n",
      "--- total time for blast 701: 15.140381336212158\n",
      "-- processing region: 702 with 291 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_702.fasta\n",
      "--- *blasting region: 702\n",
      "--- total time for blast 702: 15.185357809066772\n",
      "-- processing region: 703 with 181 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_703.fasta\n",
      "--- *blasting region: 703\n",
      "--- total time for blast 703: 10.815837860107422\n",
      "-- processing region: 704 with 284 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_704.fasta\n",
      "--- *blasting region: 704\n",
      "--- total time for blast 704: 12.649433135986328\n",
      "-- processing region: 705 with 292 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_705.fasta\n",
      "--- *blasting region: 705\n",
      "--- total time for blast 705: 13.159152269363403\n",
      "-- processing region: 706 with 246 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_706.fasta\n",
      "--- *blasting region: 706\n",
      "--- total time for blast 706: 14.127605676651001\n",
      "-- processing region: 707 with 182 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_707.fasta\n",
      "--- *blasting region: 707\n",
      "--- total time for blast 707: 11.988335609436035\n",
      "-- processing region: 708 with 213 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_708.fasta\n",
      "--- *blasting region: 708\n",
      "--- total time for blast 708: 10.202177286148071\n",
      "-- processing region: 709 with 289 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_709.fasta\n",
      "--- *blasting region: 709\n",
      "--- total time for blast 709: 15.873067378997803\n",
      "-- processing region: 71 with 137 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_71.fasta\n",
      "--- *blasting region: 71\n",
      "--- total time for blast 71: 9.672285795211792\n",
      "-- processing region: 710 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_710.fasta\n",
      "--- *blasting region: 710\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 710: 17.071231603622437\n",
      "-- processing region: 711 with 290 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_711.fasta\n",
      "--- *blasting region: 711\n",
      "--- total time for blast 711: 12.810210227966309\n",
      "-- processing region: 712 with 201 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_712.fasta\n",
      "--- *blasting region: 712\n",
      "--- total time for blast 712: 10.929173946380615\n",
      "-- processing region: 713 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_713.fasta\n",
      "--- *blasting region: 713\n",
      "--- total time for blast 713: 11.233841896057129\n",
      "-- processing region: 714 with 196 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_714.fasta\n",
      "--- *blasting region: 714\n",
      "--- total time for blast 714: 16.44212293624878\n",
      "-- processing region: 715 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_715.fasta\n",
      "--- *blasting region: 715\n",
      "--- total time for blast 715: 14.098965406417847\n",
      "-- processing region: 716 with 136 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_716.fasta\n",
      "--- *blasting region: 716\n",
      "--- total time for blast 716: 8.105021238327026\n",
      "-- processing region: 717 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_717.fasta\n",
      "--- *blasting region: 717\n",
      "--- total time for blast 717: 12.923535585403442\n",
      "-- processing region: 718 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_718.fasta\n",
      "--- *blasting region: 718\n",
      "--- total time for blast 718: 14.602014064788818\n",
      "-- processing region: 719 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_719.fasta\n",
      "--- *blasting region: 719\n",
      "--- total time for blast 719: 14.10887360572815\n",
      "-- processing region: 72 with 267 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_72.fasta\n",
      "--- *blasting region: 72\n",
      "--- total time for blast 72: 16.80471968650818\n",
      "-- processing region: 720 with 218 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_720.fasta\n",
      "--- *blasting region: 720\n",
      "--- total time for blast 720: 10.72913408279419\n",
      "-- processing region: 721 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_721.fasta\n",
      "--- *blasting region: 721\n",
      "--- total time for blast 721: 12.02250075340271\n",
      "-- processing region: 722 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_722.fasta\n",
      "--- *blasting region: 722\n",
      "--- total time for blast 722: 12.872783422470093\n",
      "-- processing region: 723 with 73 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_723.fasta\n",
      "--- *blasting region: 723\n",
      "--- total time for blast 723: 6.162848472595215\n",
      "-- processing region: 724 with 287 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_724.fasta\n",
      "--- *blasting region: 724\n",
      "--- total time for blast 724: 13.961160659790039\n",
      "-- processing region: 725 with 271 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_725.fasta\n",
      "--- *blasting region: 725\n",
      "--- total time for blast 725: 12.52406644821167\n",
      "-- processing region: 726 with 209 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_726.fasta\n",
      "--- *blasting region: 726\n",
      "--- total time for blast 726: 9.659671783447266\n",
      "-- processing region: 727 with 138 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_727.fasta\n",
      "--- *blasting region: 727\n",
      "--- total time for blast 727: 7.274068832397461\n",
      "-- processing region: 728 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_728.fasta\n",
      "--- *blasting region: 728\n",
      "--- total time for blast 728: 13.257670402526855\n",
      "-- processing region: 729 with 200 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_729.fasta\n",
      "--- *blasting region: 729\n",
      "--- total time for blast 729: 10.720995664596558\n",
      "-- processing region: 73 with 198 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_73.fasta\n",
      "--- *blasting region: 73\n",
      "--- total time for blast 73: 11.728778839111328\n",
      "-- processing region: 730 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_730.fasta\n",
      "--- *blasting region: 730\n",
      "--- total time for blast 730: 11.527403593063354\n",
      "-- processing region: 731 with 174 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_731.fasta\n",
      "--- *blasting region: 731\n",
      "--- total time for blast 731: 9.954948663711548\n",
      "-- processing region: 732 with 296 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_732.fasta\n",
      "--- *blasting region: 732\n",
      "--- total time for blast 732: 13.35723876953125\n",
      "-- processing region: 733 with 274 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_733.fasta\n",
      "--- *blasting region: 733\n",
      "--- total time for blast 733: 13.044893264770508\n",
      "-- processing region: 734 with 225 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_734.fasta\n",
      "--- *blasting region: 734\n",
      "--- total time for blast 734: 15.55044674873352\n",
      "-- processing region: 735 with 206 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_735.fasta\n",
      "--- *blasting region: 735\n",
      "--- total time for blast 735: 10.150569200515747\n",
      "-- processing region: 736 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_736.fasta\n",
      "--- *blasting region: 736\n",
      "--- total time for blast 736: 12.095284461975098\n",
      "-- processing region: 737 with 252 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_737.fasta\n",
      "--- *blasting region: 737\n",
      "--- total time for blast 737: 11.925163507461548\n",
      "-- processing region: 738 with 221 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_738.fasta\n",
      "--- *blasting region: 738\n",
      "--- total time for blast 738: 11.590375185012817\n",
      "-- processing region: 739 with 229 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_739.fasta\n",
      "--- *blasting region: 739\n",
      "--- total time for blast 739: 14.421582221984863\n",
      "-- processing region: 74 with 173 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_74.fasta\n",
      "--- *blasting region: 74\n",
      "--- total time for blast 74: 9.415639877319336\n",
      "-- processing region: 740 with 249 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_740.fasta\n",
      "--- *blasting region: 740\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 740: 14.664421796798706\n",
      "-- processing region: 741 with 266 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_741.fasta\n",
      "--- *blasting region: 741\n",
      "--- total time for blast 741: 10.722052097320557\n",
      "-- processing region: 742 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_742.fasta\n",
      "--- *blasting region: 742\n",
      "--- total time for blast 742: 11.295562982559204\n",
      "-- processing region: 743 with 297 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_743.fasta\n",
      "--- *blasting region: 743\n",
      "--- total time for blast 743: 13.253595113754272\n",
      "-- processing region: 744 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_744.fasta\n",
      "--- *blasting region: 744\n",
      "--- total time for blast 744: 15.122435331344604\n",
      "-- processing region: 745 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_745.fasta\n",
      "--- *blasting region: 745\n",
      "--- total time for blast 745: 12.954124927520752\n",
      "-- processing region: 748 with 254 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_748.fasta\n",
      "--- *blasting region: 748\n",
      "--- total time for blast 748: 11.902436971664429\n",
      "-- processing region: 749 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_749.fasta\n",
      "--- *blasting region: 749\n",
      "--- total time for blast 749: 11.492199659347534\n",
      "-- processing region: 75 with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_75.fasta\n",
      "--- *blasting region: 75\n",
      "--- total time for blast 75: 10.008111000061035\n",
      "-- processing region: 750 with 287 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_750.fasta\n",
      "--- *blasting region: 750\n",
      "--- total time for blast 750: 18.221160173416138\n",
      "-- processing region: 751 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_751.fasta\n",
      "--- *blasting region: 751\n",
      "--- total time for blast 751: 14.428097248077393\n",
      "-- processing region: 752 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_752.fasta\n",
      "--- *blasting region: 752\n",
      "--- total time for blast 752: 14.612006664276123\n",
      "-- processing region: 753 with 145 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_753.fasta\n",
      "--- *blasting region: 753\n",
      "--- total time for blast 753: 7.599684953689575\n",
      "-- processing region: 754 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_754.fasta\n",
      "--- *blasting region: 754\n",
      "--- total time for blast 754: 10.621859788894653\n",
      "-- processing region: 755 with 59 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_755.fasta\n",
      "--- *blasting region: 755\n",
      "--- total time for blast 755: 7.3629469871521\n",
      "-- processing region: 756 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_756.fasta\n",
      "--- *blasting region: 756\n",
      "--- total time for blast 756: 16.0713369846344\n",
      "-- processing region: 757 with 192 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_757.fasta\n",
      "--- *blasting region: 757\n",
      "--- total time for blast 757: 13.813313245773315\n",
      "-- processing region: 758 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_758.fasta\n",
      "--- *blasting region: 758\n",
      "--- total time for blast 758: 9.420695066452026\n",
      "-- processing region: 759 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_759.fasta\n",
      "--- *blasting region: 759\n",
      "--- total time for blast 759: 14.259833335876465\n",
      "-- processing region: 76 with 121 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_76.fasta\n",
      "--- *blasting region: 76\n",
      "--- total time for blast 76: 14.793221235275269\n",
      "-- processing region: 760 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_760.fasta\n",
      "--- *blasting region: 760\n",
      "--- total time for blast 760: 13.015400886535645\n",
      "-- processing region: 761 with 222 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_761.fasta\n",
      "--- *blasting region: 761\n",
      "--- total time for blast 761: 12.597407102584839\n",
      "-- processing region: 762 with 140 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_762.fasta\n",
      "--- *blasting region: 762\n",
      "--- total time for blast 762: 10.797797203063965\n",
      "-- processing region: 763 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_763.fasta\n",
      "--- *blasting region: 763\n",
      "--- total time for blast 763: 14.346873760223389\n",
      "-- processing region: 764 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_764.fasta\n",
      "--- *blasting region: 764\n",
      "--- total time for blast 764: 14.625373363494873\n",
      "-- processing region: 765 with 191 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_765.fasta\n",
      "--- *blasting region: 765\n",
      "--- total time for blast 765: 12.500711917877197\n",
      "-- processing region: 766 with 254 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_766.fasta\n",
      "--- *blasting region: 766\n",
      "--- total time for blast 766: 11.869963884353638\n",
      "-- processing region: 767 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_767.fasta\n",
      "--- *blasting region: 767\n",
      "--- total time for blast 767: 13.6296546459198\n",
      "-- processing region: 768 with 278 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_768.fasta\n",
      "--- *blasting region: 768\n",
      "--- total time for blast 768: 11.050233364105225\n",
      "-- processing region: 769 with 160 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_769.fasta\n",
      "--- *blasting region: 769\n",
      "--- total time for blast 769: 8.724685192108154\n",
      "-- processing region: 77 with 218 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_77.fasta\n",
      "--- *blasting region: 77\n",
      "--- total time for blast 77: 11.729358434677124\n",
      "-- processing region: 770 with 81 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_770.fasta\n",
      "--- *blasting region: 770\n",
      "--- total time for blast 770: 7.762789726257324\n",
      "-- processing region: 774 with 202 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_774.fasta\n",
      "--- *blasting region: 774\n",
      "--- total time for blast 774: 10.787036895751953\n",
      "-- processing region: 775 with 202 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_775.fasta\n",
      "--- *blasting region: 775\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 775: 19.122682332992554\n",
      "-- processing region: 776 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_776.fasta\n",
      "--- *blasting region: 776\n",
      "--- total time for blast 776: 10.828905820846558\n",
      "-- processing region: 777 with 299 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_777.fasta\n",
      "--- *blasting region: 777\n",
      "--- total time for blast 777: 12.669139862060547\n",
      "-- processing region: 778 with 252 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_778.fasta\n",
      "--- *blasting region: 778\n",
      "--- total time for blast 778: 12.660850286483765\n",
      "-- processing region: 779 with 208 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_779.fasta\n",
      "--- *blasting region: 779\n",
      "--- total time for blast 779: 11.162277460098267\n",
      "-- processing region: 78 with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_78.fasta\n",
      "--- *blasting region: 78\n",
      "--- total time for blast 78: 11.203376531600952\n",
      "-- processing region: 780 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_780.fasta\n",
      "--- *blasting region: 780\n",
      "--- total time for blast 780: 12.812947750091553\n",
      "-- processing region: 782 with 206 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_782.fasta\n",
      "--- *blasting region: 782\n",
      "--- total time for blast 782: 9.550203084945679\n",
      "-- processing region: 783 with 285 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_783.fasta\n",
      "--- *blasting region: 783\n",
      "--- total time for blast 783: 12.998729228973389\n",
      "-- processing region: 784 with 196 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_784.fasta\n",
      "--- *blasting region: 784\n",
      "--- total time for blast 784: 16.546908855438232\n",
      "-- processing region: 785 with 295 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_785.fasta\n",
      "--- *blasting region: 785\n",
      "--- total time for blast 785: 17.829694032669067\n",
      "-- processing region: 786 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_786.fasta\n",
      "--- *blasting region: 786\n",
      "--- total time for blast 786: 10.069793462753296\n",
      "-- processing region: 787 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_787.fasta\n",
      "--- *blasting region: 787\n",
      "--- total time for blast 787: 13.38442850112915\n",
      "-- processing region: 788 with 182 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_788.fasta\n",
      "--- *blasting region: 788\n",
      "--- total time for blast 788: 13.798232316970825\n",
      "-- processing region: 789 with 209 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_789.fasta\n",
      "--- *blasting region: 789\n",
      "--- total time for blast 789: 18.018069744110107\n",
      "-- processing region: 79 with 198 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_79.fasta\n",
      "--- *blasting region: 79\n",
      "--- total time for blast 79: 13.202585935592651\n",
      "-- processing region: 790 with 91 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_790.fasta\n",
      "--- *blasting region: 790\n",
      "--- total time for blast 790: 4.811157464981079\n",
      "-- processing region: 791 with 164 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_791.fasta\n",
      "--- *blasting region: 791\n",
      "--- total time for blast 791: 10.422845602035522\n",
      "-- processing region: 792 with 168 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_792.fasta\n",
      "--- *blasting region: 792\n",
      "--- total time for blast 792: 13.121583223342896\n",
      "-- processing region: 793 with 231 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_793.fasta\n",
      "--- *blasting region: 793\n",
      "--- total time for blast 793: 14.472434282302856\n",
      "-- processing region: 794 with 298 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_794.fasta\n",
      "--- *blasting region: 794\n",
      "--- total time for blast 794: 14.250967502593994\n",
      "-- processing region: 795 with 126 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_795.fasta\n",
      "--- *blasting region: 795\n",
      "--- total time for blast 795: 11.825475454330444\n",
      "-- processing region: 796 with 289 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_796.fasta\n",
      "--- *blasting region: 796\n",
      "--- total time for blast 796: 14.6143159866333\n",
      "-- processing region: 797 with 249 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_797.fasta\n",
      "--- *blasting region: 797\n",
      "--- total time for blast 797: 14.80964708328247\n",
      "-- processing region: 798 with 244 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_798.fasta\n",
      "--- *blasting region: 798\n",
      "--- total time for blast 798: 13.101338863372803\n",
      "-- processing region: 799 with 289 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_799.fasta\n",
      "--- *blasting region: 799\n",
      "--- total time for blast 799: 11.794811725616455\n",
      "-- processing region: 8 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_8.fasta\n",
      "--- *blasting region: 8\n",
      "--- total time for blast 8: 12.255507469177246\n",
      "-- processing region: 80 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_80.fasta\n",
      "--- *blasting region: 80\n",
      "--- total time for blast 80: 10.274662733078003\n",
      "-- processing region: 800 with 196 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_800.fasta\n",
      "--- *blasting region: 800\n",
      "--- total time for blast 800: 12.470907211303711\n",
      "-- processing region: 801 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_801.fasta\n",
      "--- *blasting region: 801\n",
      "--- total time for blast 801: 14.595777988433838\n",
      "-- processing region: 802 with 257 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_802.fasta\n",
      "--- *blasting region: 802\n",
      "--- total time for blast 802: 12.750405311584473\n",
      "-- processing region: 803 with 289 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_803.fasta\n",
      "--- *blasting region: 803\n",
      "--- total time for blast 803: 14.278452634811401\n",
      "-- processing region: 804 with 220 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_804.fasta\n",
      "--- *blasting region: 804\n",
      "--- total time for blast 804: 10.295775651931763\n",
      "-- processing region: 805 with 168 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_805.fasta\n",
      "--- *blasting region: 805\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 805: 10.070269107818604\n",
      "-- processing region: 806 with 291 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_806.fasta\n",
      "--- *blasting region: 806\n",
      "--- total time for blast 806: 12.350224018096924\n",
      "-- processing region: 807 with 233 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_807.fasta\n",
      "--- *blasting region: 807\n",
      "--- total time for blast 807: 9.862711191177368\n",
      "-- processing region: 808 with 197 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_808.fasta\n",
      "--- *blasting region: 808\n",
      "--- total time for blast 808: 12.026119232177734\n",
      "-- processing region: 809 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_809.fasta\n",
      "--- *blasting region: 809\n",
      "--- total time for blast 809: 13.273778676986694\n",
      "-- processing region: 81 with 223 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_81.fasta\n",
      "--- *blasting region: 81\n",
      "--- total time for blast 81: 11.605437278747559\n",
      "-- processing region: 810 with 160 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_810.fasta\n",
      "--- *blasting region: 810\n",
      "--- total time for blast 810: 11.361473083496094\n",
      "-- processing region: 811 with 234 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_811.fasta\n",
      "--- *blasting region: 811\n",
      "--- total time for blast 811: 12.044564247131348\n",
      "-- processing region: 812 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_812.fasta\n",
      "--- *blasting region: 812\n",
      "--- total time for blast 812: 12.404266834259033\n",
      "-- processing region: 813 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_813.fasta\n",
      "--- *blasting region: 813\n",
      "--- total time for blast 813: 10.230833292007446\n",
      "-- processing region: 814 with 275 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_814.fasta\n",
      "--- *blasting region: 814\n",
      "--- total time for blast 814: 15.30886197090149\n",
      "-- processing region: 815 with 296 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_815.fasta\n",
      "--- *blasting region: 815\n",
      "--- total time for blast 815: 14.351244449615479\n",
      "-- processing region: 816 with 209 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_816.fasta\n",
      "--- *blasting region: 816\n",
      "--- total time for blast 816: 11.21829104423523\n",
      "-- processing region: 817 with 204 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_817.fasta\n",
      "--- *blasting region: 817\n",
      "--- total time for blast 817: 10.078869581222534\n",
      "-- processing region: 819 with 268 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_819.fasta\n",
      "--- *blasting region: 819\n",
      "--- total time for blast 819: 37.04188942909241\n",
      "-- processing region: 820 with 255 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_820.fasta\n",
      "--- *blasting region: 820\n",
      "--- total time for blast 820: 15.536699056625366\n",
      "-- processing region: 821 with 142 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_821.fasta\n",
      "--- *blasting region: 821\n",
      "--- total time for blast 821: 9.193992376327515\n",
      "-- processing region: 822 with 257 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_822.fasta\n",
      "--- *blasting region: 822\n",
      "--- total time for blast 822: 11.452338695526123\n",
      "-- processing region: 823 with 129 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_823.fasta\n",
      "--- *blasting region: 823\n",
      "--- total time for blast 823: 15.350026607513428\n",
      "-- processing region: 824 with 288 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_824.fasta\n",
      "--- *blasting region: 824\n",
      "--- total time for blast 824: 21.27464461326599\n",
      "-- processing region: 825 with 225 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_825.fasta\n",
      "--- *blasting region: 825\n",
      "--- total time for blast 825: 12.898075342178345\n",
      "-- processing region: 826 with 263 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_826.fasta\n",
      "--- *blasting region: 826\n",
      "--- total time for blast 826: 13.709153652191162\n",
      "-- processing region: 827 with 283 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_827.fasta\n",
      "--- *blasting region: 827\n",
      "--- total time for blast 827: 15.329891443252563\n",
      "-- processing region: 828 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_828.fasta\n",
      "--- *blasting region: 828\n",
      "--- total time for blast 828: 12.182750225067139\n",
      "-- processing region: 829 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_829.fasta\n",
      "--- *blasting region: 829\n",
      "--- total time for blast 829: 12.968728303909302\n",
      "-- processing region: 83 with 259 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_83.fasta\n",
      "--- *blasting region: 83\n",
      "--- total time for blast 83: 14.149018287658691\n",
      "-- processing region: 832 with 168 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_832.fasta\n",
      "--- *blasting region: 832\n",
      "--- total time for blast 832: 9.611391067504883\n",
      "-- processing region: 833 with 204 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_833.fasta\n",
      "--- *blasting region: 833\n",
      "--- total time for blast 833: 9.623736143112183\n",
      "-- processing region: 834 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_834.fasta\n",
      "--- *blasting region: 834\n",
      "--- total time for blast 834: 13.93242359161377\n",
      "-- processing region: 835 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_835.fasta\n",
      "--- *blasting region: 835\n",
      "--- total time for blast 835: 12.906678676605225\n",
      "-- processing region: 836 with 164 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_836.fasta\n",
      "--- *blasting region: 836\n",
      "--- total time for blast 836: 15.496958255767822\n",
      "-- processing region: 837 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_837.fasta\n",
      "--- *blasting region: 837\n",
      "--- total time for blast 837: 15.369630575180054\n",
      "-- processing region: 838 with 226 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_838.fasta\n",
      "--- *blasting region: 838\n",
      "--- total time for blast 838: 12.543875455856323\n",
      "-- processing region: 839 with 194 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_839.fasta\n",
      "--- *blasting region: 839\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 839: 38.57307434082031\n",
      "-- processing region: 84 with 191 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_84.fasta\n",
      "--- *blasting region: 84\n",
      "--- total time for blast 84: 10.678701162338257\n",
      "-- processing region: 840 with 264 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_840.fasta\n",
      "--- *blasting region: 840\n",
      "--- total time for blast 840: 14.639617681503296\n",
      "-- processing region: 841 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_841.fasta\n",
      "--- *blasting region: 841\n",
      "--- total time for blast 841: 11.882013082504272\n",
      "-- processing region: 843 with 148 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_843.fasta\n",
      "--- *blasting region: 843\n",
      "--- total time for blast 843: 7.853752374649048\n",
      "-- processing region: 844 with 256 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_844.fasta\n",
      "--- *blasting region: 844\n",
      "--- total time for blast 844: 14.553742408752441\n",
      "-- processing region: 845 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_845.fasta\n",
      "--- *blasting region: 845\n",
      "--- total time for blast 845: 14.135751962661743\n",
      "-- processing region: 846 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_846.fasta\n",
      "--- *blasting region: 846\n",
      "--- total time for blast 846: 14.89052438735962\n",
      "-- processing region: 847 with 174 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_847.fasta\n",
      "--- *blasting region: 847\n",
      "--- total time for blast 847: 9.671679019927979\n",
      "-- processing region: 848 with 294 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_848.fasta\n",
      "--- *blasting region: 848\n",
      "--- total time for blast 848: 14.13826036453247\n",
      "-- processing region: 849 with 123 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_849.fasta\n",
      "--- *blasting region: 849\n",
      "--- total time for blast 849: 8.640957593917847\n",
      "-- processing region: 85 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_85.fasta\n",
      "--- *blasting region: 85\n",
      "--- total time for blast 85: 14.579360008239746\n",
      "-- processing region: 850 with 181 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_850.fasta\n",
      "--- *blasting region: 850\n",
      "--- total time for blast 850: 10.485285997390747\n",
      "-- processing region: 851 with 241 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_851.fasta\n",
      "--- *blasting region: 851\n",
      "--- total time for blast 851: 13.42987871170044\n",
      "-- processing region: 852 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_852.fasta\n",
      "--- *blasting region: 852\n",
      "--- total time for blast 852: 13.33017897605896\n",
      "-- processing region: 853 with 194 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_853.fasta\n",
      "--- *blasting region: 853\n",
      "--- total time for blast 853: 13.603792190551758\n",
      "-- processing region: 854 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_854.fasta\n",
      "--- *blasting region: 854\n",
      "--- total time for blast 854: 16.28662657737732\n",
      "-- processing region: 855 with 199 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_855.fasta\n",
      "--- *blasting region: 855\n",
      "--- total time for blast 855: 16.322468519210815\n",
      "-- processing region: 856 with 163 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_856.fasta\n",
      "--- *blasting region: 856\n",
      "--- total time for blast 856: 13.371919393539429\n",
      "-- processing region: 857 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_857.fasta\n",
      "--- *blasting region: 857\n",
      "--- total time for blast 857: 13.585647106170654\n",
      "-- processing region: 858 with 252 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_858.fasta\n",
      "--- *blasting region: 858\n",
      "--- total time for blast 858: 14.353741884231567\n",
      "-- processing region: 859 with 240 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_859.fasta\n",
      "--- *blasting region: 859\n",
      "--- total time for blast 859: 20.26037573814392\n",
      "-- processing region: 86 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_86.fasta\n",
      "--- *blasting region: 86\n",
      "--- total time for blast 86: 11.865970611572266\n",
      "-- processing region: 860 with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_860.fasta\n",
      "--- *blasting region: 860\n",
      "--- total time for blast 860: 12.753858089447021\n",
      "-- processing region: 861 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_861.fasta\n",
      "--- *blasting region: 861\n",
      "--- total time for blast 861: 15.054278135299683\n",
      "-- processing region: 862 with 233 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_862.fasta\n",
      "--- *blasting region: 862\n",
      "--- total time for blast 862: 11.640234470367432\n",
      "-- processing region: 863 with 282 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_863.fasta\n",
      "--- *blasting region: 863\n",
      "--- total time for blast 863: 12.673021078109741\n",
      "-- processing region: 864 with 250 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_864.fasta\n",
      "--- *blasting region: 864\n",
      "--- total time for blast 864: 13.921842575073242\n",
      "-- processing region: 865 with 223 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_865.fasta\n",
      "--- *blasting region: 865\n",
      "--- total time for blast 865: 16.435852766036987\n",
      "-- processing region: 866 with 216 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_866.fasta\n",
      "--- *blasting region: 866\n",
      "--- total time for blast 866: 11.763036489486694\n",
      "-- processing region: 867 with 164 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_867.fasta\n",
      "--- *blasting region: 867\n",
      "--- total time for blast 867: 8.68976616859436\n",
      "-- processing region: 868 with 289 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_868.fasta\n",
      "--- *blasting region: 868\n",
      "--- total time for blast 868: 27.02759313583374\n",
      "-- processing region: 869 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_869.fasta\n",
      "--- *blasting region: 869\n",
      "--- total time for blast 869: 16.187849521636963\n",
      "-- processing region: 87 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_87.fasta\n",
      "--- *blasting region: 87\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 87: 13.098751068115234\n",
      "-- processing region: 870 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_870.fasta\n",
      "--- *blasting region: 870\n",
      "--- total time for blast 870: 13.680725336074829\n",
      "-- processing region: 871 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_871.fasta\n",
      "--- *blasting region: 871\n",
      "--- total time for blast 871: 12.06839632987976\n",
      "-- processing region: 872 with 299 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_872.fasta\n",
      "--- *blasting region: 872\n",
      "--- total time for blast 872: 14.057522296905518\n",
      "-- processing region: 873 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_873.fasta\n",
      "--- *blasting region: 873\n",
      "--- total time for blast 873: 13.232883930206299\n",
      "-- processing region: 874 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_874.fasta\n",
      "--- *blasting region: 874\n",
      "--- total time for blast 874: 13.526030778884888\n",
      "-- processing region: 875 with 212 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_875.fasta\n",
      "--- *blasting region: 875\n",
      "--- total time for blast 875: 12.851231336593628\n",
      "-- processing region: 876 with 262 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_876.fasta\n",
      "--- *blasting region: 876\n",
      "--- total time for blast 876: 13.19244933128357\n",
      "-- processing region: 877 with 259 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_877.fasta\n",
      "--- *blasting region: 877\n",
      "--- total time for blast 877: 15.623155117034912\n",
      "-- processing region: 878 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_878.fasta\n",
      "--- *blasting region: 878\n",
      "--- total time for blast 878: 10.951655864715576\n",
      "-- processing region: 88 with 204 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_88.fasta\n",
      "--- *blasting region: 88\n",
      "--- total time for blast 88: 9.35932970046997\n",
      "-- processing region: 888 with 215 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_888.fasta\n",
      "--- *blasting region: 888\n",
      "--- total time for blast 888: 25.524353504180908\n",
      "-- processing region: 889 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_889.fasta\n",
      "--- *blasting region: 889\n",
      "--- total time for blast 889: 11.665059566497803\n",
      "-- processing region: 89 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_89.fasta\n",
      "--- *blasting region: 89\n",
      "--- total time for blast 89: 11.811694383621216\n",
      "-- processing region: 890 with 204 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_890.fasta\n",
      "--- *blasting region: 890\n",
      "--- total time for blast 890: 11.735766887664795\n",
      "-- processing region: 891 with 169 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_891.fasta\n",
      "--- *blasting region: 891\n",
      "--- total time for blast 891: 9.667688608169556\n",
      "-- processing region: 892 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_892.fasta\n",
      "--- *blasting region: 892\n",
      "--- total time for blast 892: 14.966028928756714\n",
      "-- processing region: 893 with 252 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_893.fasta\n",
      "--- *blasting region: 893\n",
      "--- total time for blast 893: 11.891309976577759\n",
      "-- processing region: 894 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_894.fasta\n",
      "--- *blasting region: 894\n",
      "--- total time for blast 894: 12.54969835281372\n",
      "-- processing region: 895 with 294 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_895.fasta\n",
      "--- *blasting region: 895\n",
      "--- total time for blast 895: 14.508991003036499\n",
      "-- processing region: 896 with 158 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_896.fasta\n",
      "--- *blasting region: 896\n",
      "--- total time for blast 896: 12.156623363494873\n",
      "-- processing region: 897 with 286 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_897.fasta\n",
      "--- *blasting region: 897\n",
      "--- total time for blast 897: 11.85446572303772\n",
      "-- processing region: 898 with 182 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_898.fasta\n",
      "--- *blasting region: 898\n",
      "--- total time for blast 898: 15.300673961639404\n",
      "-- processing region: 899 with 211 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_899.fasta\n",
      "--- *blasting region: 899\n",
      "--- total time for blast 899: 10.944273948669434\n",
      "-- processing region: 9 with 264 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_9.fasta\n",
      "--- *blasting region: 9\n",
      "--- total time for blast 9: 11.55761981010437\n",
      "-- processing region: 90 with 118 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_90.fasta\n",
      "--- *blasting region: 90\n",
      "--- total time for blast 90: 7.421700477600098\n",
      "-- processing region: 900 with 251 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_900.fasta\n",
      "--- *blasting region: 900\n",
      "--- total time for blast 900: 12.30684518814087\n",
      "-- processing region: 901 with 230 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_901.fasta\n",
      "--- *blasting region: 901\n",
      "--- total time for blast 901: 11.278066635131836\n",
      "-- processing region: 902 with 271 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_902.fasta\n",
      "--- *blasting region: 902\n",
      "--- total time for blast 902: 11.860300302505493\n",
      "-- processing region: 903 with 144 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_903.fasta\n",
      "--- *blasting region: 903\n",
      "--- total time for blast 903: 8.414030313491821\n",
      "-- processing region: 904 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_904.fasta\n",
      "--- *blasting region: 904\n",
      "--- total time for blast 904: 14.398298978805542\n",
      "-- processing region: 905 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_905.fasta\n",
      "--- *blasting region: 905\n",
      "--- total time for blast 905: 13.364300727844238\n",
      "-- processing region: 906 with 194 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_906.fasta\n",
      "--- *blasting region: 906\n",
      "--- total time for blast 906: 15.737520217895508\n",
      "-- processing region: 907 with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_907.fasta\n",
      "--- *blasting region: 907\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 907: 7.740641117095947\n",
      "-- processing region: 908 with 216 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_908.fasta\n",
      "--- *blasting region: 908\n",
      "--- total time for blast 908: 14.478037118911743\n",
      "-- processing region: 909 with 181 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_909.fasta\n",
      "--- *blasting region: 909\n",
      "--- total time for blast 909: 11.29277515411377\n",
      "-- processing region: 91 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_91.fasta\n",
      "--- *blasting region: 91\n",
      "--- total time for blast 91: 11.439051151275635\n",
      "-- processing region: 910 with 155 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_910.fasta\n",
      "--- *blasting region: 910\n",
      "--- total time for blast 910: 15.660606384277344\n",
      "-- processing region: 911 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_911.fasta\n",
      "--- *blasting region: 911\n",
      "--- total time for blast 911: 9.182634115219116\n",
      "-- processing region: 912 with 230 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_912.fasta\n",
      "--- *blasting region: 912\n",
      "--- total time for blast 912: 10.745898485183716\n",
      "-- processing region: 913 with 169 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_913.fasta\n",
      "--- *blasting region: 913\n",
      "--- total time for blast 913: 13.98435091972351\n",
      "-- processing region: 914 with 176 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_914.fasta\n",
      "--- *blasting region: 914\n",
      "--- total time for blast 914: 10.193817138671875\n",
      "-- processing region: 915 with 166 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_915.fasta\n",
      "--- *blasting region: 915\n",
      "--- total time for blast 915: 8.64945101737976\n",
      "-- processing region: 916 with 250 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_916.fasta\n",
      "--- *blasting region: 916\n",
      "--- total time for blast 916: 12.078452110290527\n",
      "-- processing region: 917 with 250 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_917.fasta\n",
      "--- *blasting region: 917\n",
      "--- total time for blast 917: 12.303846597671509\n",
      "-- processing region: 918 with 259 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_918.fasta\n",
      "--- *blasting region: 918\n",
      "--- total time for blast 918: 15.572087287902832\n",
      "-- processing region: 919 with 157 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_919.fasta\n",
      "--- *blasting region: 919\n",
      "--- total time for blast 919: 18.032289743423462\n",
      "-- processing region: 92 with 271 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_92.fasta\n",
      "--- *blasting region: 92\n",
      "--- total time for blast 92: 13.267817258834839\n",
      "-- processing region: 920 with 187 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_920.fasta\n",
      "--- *blasting region: 920\n",
      "--- total time for blast 920: 10.77608585357666\n",
      "-- processing region: 921 with 173 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_921.fasta\n",
      "--- *blasting region: 921\n",
      "--- total time for blast 921: 8.860661745071411\n",
      "-- processing region: 922 with 174 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_922.fasta\n",
      "--- *blasting region: 922\n",
      "--- total time for blast 922: 17.50263214111328\n",
      "-- processing region: 923 with 141 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_923.fasta\n",
      "--- *blasting region: 923\n",
      "--- total time for blast 923: 16.59884023666382\n",
      "-- processing region: 924 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_924.fasta\n",
      "--- *blasting region: 924\n",
      "--- total time for blast 924: 11.814677476882935\n",
      "-- processing region: 925 with 214 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_925.fasta\n",
      "--- *blasting region: 925\n",
      "--- total time for blast 925: 12.332602739334106\n",
      "-- processing region: 926 with 293 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_926.fasta\n",
      "--- *blasting region: 926\n",
      "--- total time for blast 926: 12.588776588439941\n",
      "-- processing region: 927 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_927.fasta\n",
      "--- *blasting region: 927\n",
      "--- total time for blast 927: 16.592374324798584\n",
      "-- processing region: 928 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_928.fasta\n",
      "--- *blasting region: 928\n",
      "--- total time for blast 928: 13.372739553451538\n",
      "-- processing region: 929 with 261 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_929.fasta\n",
      "--- *blasting region: 929\n",
      "--- total time for blast 929: 15.680521726608276\n",
      "-- processing region: 93 with 212 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_93.fasta\n",
      "--- *blasting region: 93\n",
      "--- total time for blast 93: 10.151910066604614\n",
      "-- processing region: 930 with 266 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_930.fasta\n",
      "--- *blasting region: 930\n",
      "--- total time for blast 930: 14.03131365776062\n",
      "-- processing region: 931 with 249 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_931.fasta\n",
      "--- *blasting region: 931\n",
      "--- total time for blast 931: 13.61165738105774\n",
      "-- processing region: 932 with 297 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_932.fasta\n",
      "--- *blasting region: 932\n",
      "--- total time for blast 932: 13.91091275215149\n",
      "-- processing region: 933 with 218 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_933.fasta\n",
      "--- *blasting region: 933\n",
      "--- total time for blast 933: 9.621408462524414\n",
      "-- processing region: 934 with 209 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_934.fasta\n",
      "--- *blasting region: 934\n",
      "--- total time for blast 934: 10.54566240310669\n",
      "-- processing region: 935 with 269 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_935.fasta\n",
      "--- *blasting region: 935\n",
      "--- total time for blast 935: 12.445286750793457\n",
      "-- processing region: 94 with 178 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_94.fasta\n",
      "--- *blasting region: 94\n",
      "--- total time for blast 94: 9.363290071487427\n",
      "-- processing region: 949 with 160 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_949.fasta\n",
      "--- *blasting region: 949\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 949: 9.246926546096802\n",
      "-- processing region: 95 with 214 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_95.fasta\n",
      "--- *blasting region: 95\n",
      "--- total time for blast 95: 11.763420820236206\n",
      "-- processing region: 950 with 178 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_950.fasta\n",
      "--- *blasting region: 950\n",
      "--- total time for blast 950: 10.96482229232788\n",
      "-- processing region: 951 with 188 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_951.fasta\n",
      "--- *blasting region: 951\n",
      "--- total time for blast 951: 9.5006582736969\n",
      "-- processing region: 952 with 190 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_952.fasta\n",
      "--- *blasting region: 952\n",
      "--- total time for blast 952: 12.900669574737549\n",
      "-- processing region: 953 with 245 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_953.fasta\n",
      "--- *blasting region: 953\n",
      "--- total time for blast 953: 14.535314798355103\n",
      "-- processing region: 954 with 90 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_954.fasta\n",
      "--- *blasting region: 954\n",
      "--- total time for blast 954: 7.638408184051514\n",
      "-- processing region: 955 with 174 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_955.fasta\n",
      "--- *blasting region: 955\n",
      "--- total time for blast 955: 14.153279542922974\n",
      "-- processing region: 956 with 265 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_956.fasta\n",
      "--- *blasting region: 956\n",
      "--- total time for blast 956: 13.226128339767456\n",
      "-- processing region: 957 with 258 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_957.fasta\n",
      "--- *blasting region: 957\n",
      "--- total time for blast 957: 10.670824527740479\n",
      "-- processing region: 958 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_958.fasta\n",
      "--- *blasting region: 958\n",
      "--- total time for blast 958: 14.873305797576904\n",
      "-- processing region: 959 with 213 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_959.fasta\n",
      "--- *blasting region: 959\n",
      "--- total time for blast 959: 10.934039831161499\n",
      "-- processing region: 96 with 258 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_96.fasta\n",
      "--- *blasting region: 96\n",
      "--- total time for blast 96: 12.104035377502441\n",
      "-- processing region: 960 with 279 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_960.fasta\n",
      "--- *blasting region: 960\n",
      "--- total time for blast 960: 12.558539152145386\n",
      "-- processing region: 961 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_961.fasta\n",
      "--- *blasting region: 961\n",
      "--- total time for blast 961: 11.127954244613647\n",
      "-- processing region: 962 with 226 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_962.fasta\n",
      "--- *blasting region: 962\n",
      "--- total time for blast 962: 10.730002164840698\n",
      "-- processing region: 963 with 195 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_963.fasta\n",
      "--- *blasting region: 963\n",
      "--- total time for blast 963: 13.647285223007202\n",
      "-- processing region: 964 with 149 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_964.fasta\n",
      "--- *blasting region: 964\n",
      "--- total time for blast 964: 12.783631086349487\n",
      "-- processing region: 965 with 191 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_965.fasta\n",
      "--- *blasting region: 965\n",
      "--- total time for blast 965: 10.694849252700806\n",
      "-- processing region: 966 with 161 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_966.fasta\n",
      "--- *blasting region: 966\n",
      "--- total time for blast 966: 12.220637798309326\n",
      "-- processing region: 967 with 229 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_967.fasta\n",
      "--- *blasting region: 967\n",
      "--- total time for blast 967: 14.797562599182129\n",
      "-- processing region: 968 with 228 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_968.fasta\n",
      "--- *blasting region: 968\n",
      "--- total time for blast 968: 15.059322118759155\n",
      "-- processing region: 969 with 155 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_969.fasta\n",
      "--- *blasting region: 969\n",
      "--- total time for blast 969: 12.63329529762268\n",
      "-- processing region: 970 with 110 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_970.fasta\n",
      "--- *blasting region: 970\n",
      "--- total time for blast 970: 6.484042644500732\n",
      "-- processing region: 971 with 173 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_971.fasta\n",
      "--- *blasting region: 971\n",
      "--- total time for blast 971: 9.518835544586182\n",
      "-- processing region: 972 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_972.fasta\n",
      "--- *blasting region: 972\n",
      "--- total time for blast 972: 12.029902219772339\n",
      "-- processing region: 973 with 270 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_973.fasta\n",
      "--- *blasting region: 973\n",
      "--- total time for blast 973: 13.884195566177368\n",
      "-- processing region: 974 with 208 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_974.fasta\n",
      "--- *blasting region: 974\n",
      "--- total time for blast 974: 10.596196413040161\n",
      "-- processing region: 975 with 299 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_975.fasta\n",
      "--- *blasting region: 975\n",
      "--- total time for blast 975: 13.23330807685852\n",
      "-- processing region: 976 with 194 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_976.fasta\n",
      "--- *blasting region: 976\n",
      "--- total time for blast 976: 12.458486557006836\n",
      "-- processing region: 977 with 297 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_977.fasta\n",
      "--- *blasting region: 977\n",
      "--- total time for blast 977: 15.101963758468628\n",
      "-- processing region: 978 with 130 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_978.fasta\n",
      "--- *blasting region: 978\n",
      "--- total time for blast 978: 17.203653812408447\n",
      "-- processing region: 979 with 255 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_979.fasta\n",
      "--- *blasting region: 979\n",
      "--- total time for blast 979: 12.658254384994507\n",
      "-- processing region: 98 with 166 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_98.fasta\n",
      "--- *blasting region: 98\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- total time for blast 98: 9.363692045211792\n",
      "-- processing region: 980 with 173 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_980.fasta\n",
      "--- *blasting region: 980\n",
      "--- total time for blast 980: 12.124159336090088\n",
      "-- processing region: 981 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_981.fasta\n",
      "--- *blasting region: 981\n",
      "--- total time for blast 981: 12.631026029586792\n",
      "-- processing region: 982 with 171 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_982.fasta\n",
      "--- *blasting region: 982\n",
      "--- total time for blast 982: 9.988569736480713\n",
      "-- processing region: 983 with 236 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_983.fasta\n",
      "--- *blasting region: 983\n",
      "--- total time for blast 983: 10.24655556678772\n",
      "-- processing region: 984 with 269 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_984.fasta\n",
      "--- *blasting region: 984\n",
      "--- total time for blast 984: 11.153067350387573\n",
      "-- processing region: 985 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_985.fasta\n",
      "--- *blasting region: 985\n",
      "--- total time for blast 985: 10.437487363815308\n",
      "-- processing region: 986 with 233 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_986.fasta\n",
      "--- *blasting region: 986\n",
      "--- total time for blast 986: 9.504576921463013\n",
      "-- processing region: 987 with 291 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_987.fasta\n",
      "--- *blasting region: 987\n",
      "--- total time for blast 987: 13.320543050765991\n",
      "-- processing region: 988 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_988.fasta\n",
      "--- *blasting region: 988\n",
      "--- total time for blast 988: 13.382763147354126\n",
      "-- processing region: 989 with 154 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_989.fasta\n",
      "--- *blasting region: 989\n",
      "--- total time for blast 989: 10.575189352035522\n",
      "-- processing region: 990 with 281 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_990.fasta\n",
      "--- *blasting region: 990\n",
      "--- total time for blast 990: 14.25624680519104\n",
      "-- processing region: 991 with 247 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_991.fasta\n",
      "--- *blasting region: 991\n",
      "--- total time for blast 991: 13.6489896774292\n",
      "-- processing region: 992 with 237 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_992.fasta\n",
      "--- *blasting region: 992\n",
      "--- total time for blast 992: 10.074311256408691\n",
      "-- processing region: 993 with 153 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_993.fasta\n",
      "--- *blasting region: 993\n",
      "--- total time for blast 993: 7.354485034942627\n",
      "-- processing region: 994 with 286 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_994.fasta\n",
      "--- *blasting region: 994\n",
      "--- total time for blast 994: 13.01123332977295\n",
      "-- processing region: 995 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_995.fasta\n",
      "--- *blasting region: 995\n",
      "--- total time for blast 995: 13.436149597167969\n",
      "-- processing region: 996 with 255 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_996.fasta\n",
      "--- *blasting region: 996\n",
      "--- total time for blast 996: 12.363887786865234\n",
      "-- processing region: 997 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_997.fasta\n",
      "--- *blasting region: 997\n",
      "--- total time for blast 997: 13.060990333557129\n",
      "-- processing region: 999 with 300 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\blast\\probe_gene_999.fasta\n",
      "--- *blasting region: 999\n",
      "--- total time for blast 999: 13.45621919631958\n"
     ]
    }
   ],
   "source": [
    "blast_database = os.path.join(reference_folder, 'blast_database', 'hg38')\n",
    "\n",
    "check.Blast_probes(kept_records, library_folder, num_threads=44, \n",
    "                   blast_db=blast_database, blast_evalue=100, overwrite=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7c6a9506",
   "metadata": {},
   "source": [
    "### 4.2.1 screen probe by blast results\n",
    "\n",
    "This step has 2 different cutoffs:\n",
    "\n",
    "hard_cutoff: whether a probe has any ~40mer hit somewhere else, if so just dump the probe\n",
    "\n",
    "soft_cutoff: whether a probe has a lot of ~20mer off-target hits, if exceed certain amount then dump the probe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "6f6749b3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Number of probes loaded: 317417\n",
      "- Number of regions in this library: 1352\n",
      "-- checking probes in region: 0\n",
      "--- gene=0, id=137 removed by soft count = 135\n",
      "--- gene=0, id=144 removed by hard count = 12\n",
      "--- gene=0, id=158 removed by soft count = 61\n",
      "--- gene=0, id=159 removed by soft count = 87\n",
      "--- gene=0, id=160 removed by soft count = 230\n",
      "--- number of probes: 295 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1\n",
      "--- gene=1, id=67 removed by soft count = 197\n",
      "--- gene=1, id=68 removed by soft count = 113\n",
      "--- gene=1, id=132 removed by soft count = 106\n",
      "--- gene=1, id=140 removed by soft count = 104\n",
      "--- gene=1, id=143 removed by soft count = 134\n",
      "--- gene=1, id=144 removed by soft count = 168\n",
      "--- gene=1, id=145 removed by soft count = 144\n",
      "--- gene=1, id=146 removed by soft count = 132\n",
      "--- gene=1, id=147 removed by soft count = 307\n",
      "--- number of probes: 287 , kept by blast: 278 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 2\n",
      "--- gene=2, id=185 removed by soft count = 66\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 3\n",
      "--- gene=3, id=6 removed by soft count = 164\n",
      "--- gene=3, id=49 removed by soft count = 242\n",
      "--- gene=3, id=55 removed by soft count = 294\n",
      "--- gene=3, id=70 removed by soft count = 140\n",
      "--- number of probes: 242 , kept by blast: 238 , if remove dups: 238\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 4\n",
      "--- gene=4, id=82 removed by soft count = 480\n",
      "--- gene=4, id=154 removed by soft count = 353\n",
      "--- number of probes: 256 , kept by blast: 254 , if remove dups: 254\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 5\n",
      "--- gene=5, id=67 removed by soft count = 114\n",
      "--- gene=5, id=120 removed by soft count = 72\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 6\n",
      "--- gene=6, id=53 removed by soft count = 123\n",
      "--- gene=6, id=85 removed by soft count = 128\n",
      "--- gene=6, id=264 removed by soft count = 110\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 7\n",
      "--- gene=7, id=122 removed by hard count = 0\n",
      "--- gene=7, id=209 removed by soft count = 57\n",
      "--- gene=7, id=213 removed by soft count = 243\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 8\n",
      "--- gene=8, id=191 removed by soft count = 74\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 9\n",
      "--- gene=9, id=159 removed by soft count = 104\n",
      "--- number of probes: 264 , kept by blast: 263 , if remove dups: 263\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 10\n",
      "--- gene=10, id=17 removed by soft count = 360\n",
      "--- gene=10, id=20 removed by soft count = 126\n",
      "--- gene=10, id=21 removed by soft count = 123\n",
      "--- gene=10, id=22 removed by soft count = 119\n",
      "--- gene=10, id=89 removed by soft count = 77\n",
      "--- gene=10, id=107 removed by soft count = 1509\n",
      "--- gene=10, id=120 removed by soft count = 99\n",
      "--- gene=10, id=208 removed by soft count = 66\n",
      "--- number of probes: 216 , kept by blast: 208 , if remove dups: 208\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 11\n",
      "--- gene=11, id=185 removed by soft count = 525\n",
      "--- gene=11, id=198 removed by soft count = 1270\n",
      "--- gene=11, id=199 removed by soft count = 382\n",
      "--- gene=11, id=200 removed by soft count = 76\n",
      "--- gene=11, id=201 removed by soft count = 197\n",
      "--- number of probes: 212 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 12\n",
      "--- gene=12, id=8 removed by soft count = 84\n",
      "--- gene=12, id=92 removed by soft count = 54\n",
      "--- gene=12, id=171 removed by soft count = 247\n",
      "--- gene=12, id=172 removed by soft count = 1643\n",
      "--- number of probes: 240 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 13\n",
      "--- gene=13, id=157 removed by soft count = 59\n",
      "--- gene=13, id=161 removed by soft count = 79\n",
      "--- gene=13, id=162 removed by soft count = 304\n",
      "--- gene=13, id=165 removed by soft count = 296\n",
      "--- gene=13, id=166 removed by soft count = 53\n",
      "--- gene=13, id=167 removed by soft count = 1069\n",
      "--- gene=13, id=168 removed by soft count = 150\n",
      "--- gene=13, id=170 removed by soft count = 176\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 14\n",
      "--- gene=14, id=35 removed by soft count = 319\n",
      "--- gene=14, id=56 removed by soft count = 179\n",
      "--- gene=14, id=174 removed by soft count = 124\n",
      "--- number of probes: 176 , kept by blast: 173 , if remove dups: 173\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 15\n",
      "--- gene=15, id=253 removed by soft count = 100\n",
      "--- gene=15, id=265 removed by soft count = 63\n",
      "--- gene=15, id=268 removed by soft count = 78\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 16\n",
      "--- gene=16, id=154 removed by soft count = 4544\n",
      "--- gene=16, id=161 removed by soft count = 331\n",
      "--- gene=16, id=162 removed by soft count = 889\n",
      "--- gene=16, id=163 removed by soft count = 87\n",
      "--- gene=16, id=166 removed by soft count = 55\n",
      "--- gene=16, id=167 removed by soft count = 101\n",
      "--- gene=16, id=169 removed by soft count = 90\n",
      "--- gene=16, id=170 removed by soft count = 141\n",
      "--- gene=16, id=172 removed by soft count = 64\n",
      "--- gene=16, id=173 removed by soft count = 57\n",
      "--- gene=16, id=174 removed by soft count = 52\n",
      "--- gene=16, id=176 removed by soft count = 197\n",
      "--- gene=16, id=179 removed by soft count = 63\n",
      "--- gene=16, id=181 removed by soft count = 114\n",
      "--- gene=16, id=183 removed by soft count = 76\n",
      "--- number of probes: 245 , kept by blast: 230 , if remove dups: 230\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 17\n",
      "--- gene=17, id=2 removed by soft count = 143\n",
      "--- gene=17, id=82 removed by soft count = 130\n",
      "--- gene=17, id=86 removed by soft count = 104\n",
      "--- gene=17, id=89 removed by soft count = 62\n",
      "--- gene=17, id=99 removed by soft count = 172\n",
      "--- number of probes: 203 , kept by blast: 198 , if remove dups: 198\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 18\n",
      "--- gene=18, id=4 removed by soft count = 99\n",
      "--- gene=18, id=5 removed by soft count = 411\n",
      "--- gene=18, id=22 removed by soft count = 85\n",
      "--- gene=18, id=82 removed by soft count = 323\n",
      "--- gene=18, id=94 removed by soft count = 4875\n",
      "--- gene=18, id=103 removed by soft count = 237\n",
      "--- gene=18, id=175 removed by soft count = 90\n",
      "--- gene=18, id=176 removed by soft count = 82\n",
      "--- number of probes: 204 , kept by blast: 196 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 19\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 20\n",
      "--- gene=20, id=97 removed by soft count = 1203\n",
      "--- gene=20, id=112 removed by soft count = 139\n",
      "--- gene=20, id=128 removed by soft count = 212\n",
      "--- number of probes: 206 , kept by blast: 203 , if remove dups: 203\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 21\n",
      "--- gene=21, id=108 removed by soft count = 61\n",
      "--- gene=21, id=220 removed by soft count = 390\n",
      "--- number of probes: 255 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 22\n",
      "--- gene=22, id=72 removed by soft count = 145\n",
      "--- gene=22, id=162 removed by soft count = 79\n",
      "--- gene=22, id=163 removed by soft count = 51\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 23\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 24\n",
      "--- gene=24, id=249 removed by soft count = 105\n",
      "--- number of probes: 279 , kept by blast: 278 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 25\n",
      "--- gene=25, id=2 removed by soft count = 78\n",
      "--- gene=25, id=3 removed by soft count = 173\n",
      "--- gene=25, id=4 removed by soft count = 85\n",
      "--- gene=25, id=14 removed by soft count = 70\n",
      "--- gene=25, id=15 removed by soft count = 591\n",
      "--- gene=25, id=16 removed by soft count = 61\n",
      "--- gene=25, id=17 removed by soft count = 432\n",
      "--- gene=25, id=19 removed by soft count = 78\n",
      "--- gene=25, id=147 removed by soft count = 128\n",
      "--- gene=25, id=221 removed by soft count = 57\n",
      "--- number of probes: 238 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 26\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 27\n",
      "--- gene=27, id=17 removed by soft count = 461\n",
      "--- gene=27, id=18 removed by soft count = 324\n",
      "--- gene=27, id=19 removed by soft count = 603\n",
      "--- gene=27, id=20 removed by soft count = 887\n",
      "--- gene=27, id=135 removed by soft count = 68\n",
      "--- gene=27, id=137 removed by soft count = 146\n",
      "--- gene=27, id=138 removed by soft count = 86\n",
      "--- gene=27, id=140 removed by soft count = 144\n",
      "--- gene=27, id=141 removed by soft count = 83\n",
      "--- gene=27, id=142 removed by soft count = 754\n",
      "--- gene=27, id=143 removed by soft count = 277\n",
      "--- gene=27, id=144 removed by soft count = 136\n",
      "--- gene=27, id=145 removed by soft count = 133\n",
      "--- gene=27, id=146 removed by soft count = 64\n",
      "--- gene=27, id=147 removed by soft count = 1939\n",
      "--- gene=27, id=148 removed by soft count = 1515\n",
      "--- number of probes: 149 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 28\n",
      "--- gene=28, id=8 removed by soft count = 1410\n",
      "--- gene=28, id=226 removed by soft count = 359\n",
      "--- gene=28, id=227 removed by soft count = 241\n",
      "--- number of probes: 262 , kept by blast: 259 , if remove dups: 259\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 29\n",
      "--- gene=29, id=26 removed by soft count = 52\n",
      "--- gene=29, id=33 removed by soft count = 257\n",
      "--- gene=29, id=67 removed by soft count = 102\n",
      "--- gene=29, id=76 removed by soft count = 139\n",
      "--- gene=29, id=106 removed by soft count = 100\n",
      "--- gene=29, id=133 removed by soft count = 60\n",
      "--- number of probes: 199 , kept by blast: 193 , if remove dups: 193\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 30\n",
      "--- gene=30, id=37 removed by soft count = 161\n",
      "--- gene=30, id=55 removed by soft count = 74\n",
      "--- gene=30, id=62 removed by soft count = 2849\n",
      "--- gene=30, id=64 removed by soft count = 546\n",
      "--- gene=30, id=65 removed by soft count = 368\n",
      "--- gene=30, id=67 removed by soft count = 101\n",
      "--- gene=30, id=68 removed by soft count = 152\n",
      "--- gene=30, id=69 removed by soft count = 141\n",
      "--- gene=30, id=80 removed by soft count = 76\n",
      "--- gene=30, id=81 removed by soft count = 97\n",
      "--- number of probes: 263 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 31\n",
      "--- gene=31, id=29 removed by soft count = 351\n",
      "--- gene=31, id=131 removed by soft count = 2515\n",
      "--- gene=31, id=132 removed by soft count = 192\n",
      "--- gene=31, id=133 removed by soft count = 1415\n",
      "--- number of probes: 215 , kept by blast: 211 , if remove dups: 211\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 32\n",
      "--- gene=32, id=78 removed by soft count = 766\n",
      "--- gene=32, id=79 removed by soft count = 813\n",
      "--- gene=32, id=80 removed by soft count = 571\n",
      "--- gene=32, id=81 removed by soft count = 603\n",
      "--- gene=32, id=85 removed by soft count = 61\n",
      "--- gene=32, id=135 removed by soft count = 85\n",
      "--- gene=32, id=168 removed by soft count = 209\n",
      "--- gene=32, id=169 removed by soft count = 181\n",
      "--- number of probes: 190 , kept by blast: 182 , if remove dups: 182\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 33\n",
      "--- gene=33, id=34 removed by soft count = 408\n",
      "--- gene=33, id=35 removed by soft count = 371\n",
      "--- gene=33, id=218 removed by soft count = 410\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 34\n",
      "--- gene=34, id=34 removed by soft count = 66\n",
      "--- gene=34, id=109 removed by soft count = 62\n",
      "--- number of probes: 230 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 35\n",
      "--- gene=35, id=6 removed by soft count = 105\n",
      "--- gene=35, id=74 removed by soft count = 245\n",
      "--- gene=35, id=75 removed by soft count = 135\n",
      "--- gene=35, id=77 removed by soft count = 302\n",
      "--- number of probes: 203 , kept by blast: 199 , if remove dups: 199\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 36\n",
      "--- gene=36, id=5 removed by soft count = 75\n",
      "--- gene=36, id=6 removed by soft count = 293\n",
      "--- gene=36, id=108 removed by soft count = 68\n",
      "--- gene=36, id=109 removed by soft count = 126\n",
      "--- gene=36, id=110 removed by soft count = 94\n",
      "--- gene=36, id=115 removed by soft count = 103\n",
      "--- gene=36, id=117 removed by soft count = 58\n",
      "--- gene=36, id=120 removed by soft count = 116\n",
      "--- gene=36, id=121 removed by soft count = 245\n",
      "--- gene=36, id=124 removed by soft count = 240\n",
      "--- gene=36, id=125 removed by soft count = 1218\n",
      "--- gene=36, id=126 removed by soft count = 990\n",
      "--- gene=36, id=127 removed by soft count = 3009\n",
      "--- gene=36, id=128 removed by soft count = 868\n",
      "--- gene=36, id=135 removed by soft count = 63\n",
      "--- number of probes: 154 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 37\n",
      "--- gene=37, id=23 removed by soft count = 384\n",
      "--- gene=37, id=175 removed by soft count = 71\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 38\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=38, id=151 removed by soft count = 972\n",
      "--- gene=38, id=174 removed by soft count = 95\n",
      "--- number of probes: 211 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 39\n",
      "--- gene=39, id=165 removed by soft count = 72\n",
      "--- gene=39, id=178 removed by soft count = 76\n",
      "--- number of probes: 237 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 40\n",
      "--- gene=40, id=131 removed by soft count = 60\n",
      "--- gene=40, id=288 removed by soft count = 529\n",
      "--- number of probes: 293 , kept by blast: 291 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 42\n",
      "--- gene=42, id=38 removed by soft count = 58\n",
      "--- gene=42, id=85 removed by soft count = 87\n",
      "--- gene=42, id=161 removed by soft count = 131\n",
      "--- gene=42, id=162 removed by soft count = 91\n",
      "--- gene=42, id=163 removed by soft count = 52\n",
      "--- gene=42, id=164 removed by soft count = 60\n",
      "--- gene=42, id=166 removed by soft count = 160\n",
      "--- gene=42, id=167 removed by soft count = 82\n",
      "--- gene=42, id=168 removed by soft count = 70\n",
      "--- gene=42, id=169 removed by soft count = 96\n",
      "--- gene=42, id=170 removed by soft count = 106\n",
      "--- gene=42, id=175 removed by soft count = 200\n",
      "--- gene=42, id=178 removed by soft count = 133\n",
      "--- number of probes: 201 , kept by blast: 188 , if remove dups: 188\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 43\n",
      "--- gene=43, id=45 removed by soft count = 108\n",
      "--- gene=43, id=58 removed by soft count = 58\n",
      "--- gene=43, id=163 removed by soft count = 183\n",
      "--- gene=43, id=234 removed by soft count = 187\n",
      "--- number of probes: 235 , kept by blast: 231 , if remove dups: 231\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 44\n",
      "--- gene=44, id=25 removed by soft count = 95\n",
      "--- gene=44, id=26 removed by soft count = 66\n",
      "--- gene=44, id=28 removed by soft count = 234\n",
      "--- gene=44, id=29 removed by soft count = 212\n",
      "--- gene=44, id=34 removed by soft count = 97\n",
      "--- gene=44, id=293 removed by soft count = 705\n",
      "--- gene=44, id=295 removed by soft count = 153\n",
      "--- gene=44, id=296 removed by soft count = 103\n",
      "--- gene=44, id=297 removed by soft count = 194\n",
      "--- number of probes: 300 , kept by blast: 291 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 45\n",
      "--- gene=45, id=27 removed by soft count = 51\n",
      "--- gene=45, id=29 removed by soft count = 58\n",
      "--- gene=45, id=37 removed by soft count = 485\n",
      "--- gene=45, id=40 removed by soft count = 238\n",
      "--- gene=45, id=41 removed by soft count = 62\n",
      "--- gene=45, id=74 removed by soft count = 968\n",
      "--- gene=45, id=85 removed by soft count = 189\n",
      "--- gene=45, id=98 removed by soft count = 923\n",
      "--- number of probes: 187 , kept by blast: 179 , if remove dups: 179\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 46\n",
      "--- gene=46, id=3 removed by soft count = 331\n",
      "--- gene=46, id=34 removed by soft count = 71\n",
      "--- gene=46, id=37 removed by soft count = 233\n",
      "--- gene=46, id=52 removed by soft count = 62\n",
      "--- gene=46, id=133 removed by soft count = 107\n",
      "--- gene=46, id=172 removed by soft count = 186\n",
      "--- gene=46, id=173 removed by soft count = 186\n",
      "--- gene=46, id=174 removed by soft count = 71\n",
      "--- gene=46, id=175 removed by soft count = 58\n",
      "--- gene=46, id=216 removed by soft count = 61\n",
      "--- number of probes: 300 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 47\n",
      "--- gene=47, id=96 removed by soft count = 84\n",
      "--- gene=47, id=97 removed by soft count = 78\n",
      "--- gene=47, id=207 removed by soft count = 134\n",
      "--- gene=47, id=209 removed by soft count = 66\n",
      "--- gene=47, id=211 removed by soft count = 165\n",
      "--- number of probes: 213 , kept by blast: 208 , if remove dups: 208\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 57\n",
      "--- gene=57, id=50 removed by soft count = 311\n",
      "--- number of probes: 60 , kept by blast: 59 , if remove dups: 59\n",
      "-- number of probes kept for this region: 59\n",
      "-- checking probes in region: 60\n",
      "--- gene=60, id=59 removed by soft count = 76\n",
      "--- gene=60, id=68 removed by soft count = 864\n",
      "--- gene=60, id=69 removed by soft count = 265\n",
      "--- gene=60, id=79 removed by soft count = 60\n",
      "--- number of probes: 187 , kept by blast: 183 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 62\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 63\n",
      "--- gene=63, id=29 removed by soft count = 128\n",
      "--- gene=63, id=33 removed by soft count = 271\n",
      "--- gene=63, id=34 removed by soft count = 87\n",
      "--- gene=63, id=120 removed by soft count = 80\n",
      "--- gene=63, id=149 removed by soft count = 63\n",
      "--- gene=63, id=154 removed by soft count = 161\n",
      "--- number of probes: 241 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 64\n",
      "--- gene=64, id=111 removed by soft count = 52\n",
      "--- gene=64, id=112 removed by soft count = 239\n",
      "--- gene=64, id=113 removed by soft count = 274\n",
      "--- gene=64, id=114 removed by soft count = 172\n",
      "--- gene=64, id=117 removed by soft count = 176\n",
      "--- gene=64, id=118 removed by soft count = 185\n",
      "--- gene=64, id=119 removed by soft count = 200\n",
      "--- gene=64, id=120 removed by soft count = 72\n",
      "--- gene=64, id=121 removed by soft count = 198\n",
      "--- gene=64, id=122 removed by soft count = 196\n",
      "--- gene=64, id=123 removed by soft count = 178\n",
      "--- gene=64, id=124 removed by soft count = 237\n",
      "--- gene=64, id=125 removed by soft count = 265\n",
      "--- gene=64, id=126 removed by soft count = 205\n",
      "--- gene=64, id=128 removed by soft count = 118\n",
      "--- number of probes: 199 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 65\n",
      "--- gene=65, id=16 removed by soft count = 395\n",
      "--- gene=65, id=19 removed by soft count = 78\n",
      "--- gene=65, id=23 removed by soft count = 59\n",
      "--- gene=65, id=25 removed by soft count = 172\n",
      "--- gene=65, id=26 removed by soft count = 78\n",
      "--- gene=65, id=27 removed by soft count = 88\n",
      "--- gene=65, id=29 removed by soft count = 137\n",
      "--- gene=65, id=30 removed by soft count = 286\n",
      "--- gene=65, id=102 removed by soft count = 64\n",
      "--- gene=65, id=107 removed by soft count = 187\n",
      "--- gene=65, id=108 removed by soft count = 392\n",
      "--- gene=65, id=109 removed by soft count = 85\n",
      "--- gene=65, id=110 removed by soft count = 2081\n",
      "--- number of probes: 114 , kept by blast: 101 , if remove dups: 101\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 101\n",
      "-- checking probes in region: 66\n",
      "--- gene=66, id=41 removed by soft count = 51\n",
      "--- gene=66, id=42 removed by soft count = 3372\n",
      "--- gene=66, id=44 removed by soft count = 279\n",
      "--- gene=66, id=124 removed by soft count = 85\n",
      "--- number of probes: 127 , kept by blast: 123 , if remove dups: 123\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 67\n",
      "--- gene=67, id=66 removed by soft count = 183\n",
      "--- gene=67, id=113 removed by soft count = 104\n",
      "--- gene=67, id=114 removed by soft count = 710\n",
      "--- number of probes: 269 , kept by blast: 266 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 68\n",
      "--- gene=68, id=11 removed by soft count = 473\n",
      "--- gene=68, id=12 removed by soft count = 416\n",
      "--- gene=68, id=15 removed by soft count = 143\n",
      "--- gene=68, id=18 removed by soft count = 269\n",
      "--- gene=68, id=19 removed by soft count = 108\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=68, id=45 removed by soft count = 82\n",
      "--- gene=68, id=46 removed by soft count = 102\n",
      "--- gene=68, id=83 removed by soft count = 87\n",
      "--- number of probes: 226 , kept by blast: 218 , if remove dups: 218\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 69\n",
      "--- gene=69, id=6 removed by soft count = 128\n",
      "--- gene=69, id=7 removed by soft count = 94\n",
      "--- gene=69, id=9 removed by soft count = 126\n",
      "--- gene=69, id=119 removed by soft count = 356\n",
      "--- gene=69, id=120 removed by soft count = 413\n",
      "--- gene=69, id=121 removed by soft count = 149\n",
      "--- gene=69, id=123 removed by soft count = 461\n",
      "--- gene=69, id=124 removed by soft count = 789\n",
      "--- gene=69, id=125 removed by soft count = 222\n",
      "--- gene=69, id=127 removed by soft count = 230\n",
      "--- number of probes: 128 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 118\n",
      "-- checking probes in region: 70\n",
      "--- gene=70, id=2 removed by soft count = 138\n",
      "--- gene=70, id=6 removed by soft count = 53\n",
      "--- gene=70, id=10 removed by soft count = 54\n",
      "--- gene=70, id=18 removed by soft count = 101\n",
      "--- gene=70, id=22 removed by soft count = 51\n",
      "--- gene=70, id=41 removed by soft count = 101\n",
      "--- gene=70, id=85 removed by soft count = 166\n",
      "--- gene=70, id=89 removed by soft count = 253\n",
      "--- gene=70, id=90 removed by soft count = 432\n",
      "--- number of probes: 120 , kept by blast: 111 , if remove dups: 111\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 111\n",
      "-- checking probes in region: 71\n",
      "--- gene=71, id=116 removed by soft count = 87\n",
      "--- gene=71, id=122 removed by soft count = 236\n",
      "--- gene=71, id=123 removed by soft count = 56\n",
      "--- gene=71, id=124 removed by soft count = 109\n",
      "--- gene=71, id=126 removed by soft count = 314\n",
      "--- gene=71, id=129 removed by soft count = 64\n",
      "--- gene=71, id=130 removed by soft count = 103\n",
      "--- gene=71, id=134 removed by soft count = 123\n",
      "--- gene=71, id=135 removed by soft count = 62\n",
      "--- gene=71, id=136 removed by soft count = 106\n",
      "--- number of probes: 137 , kept by blast: 127 , if remove dups: 127\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 72\n",
      "--- gene=72, id=61 removed by soft count = 2033\n",
      "--- gene=72, id=117 removed by soft count = 56\n",
      "--- gene=72, id=129 removed by soft count = 51\n",
      "--- gene=72, id=136 removed by soft count = 195\n",
      "--- gene=72, id=137 removed by soft count = 59\n",
      "--- gene=72, id=143 removed by soft count = 51\n",
      "--- gene=72, id=144 removed by soft count = 82\n",
      "--- gene=72, id=145 removed by soft count = 488\n",
      "--- gene=72, id=146 removed by soft count = 333\n",
      "--- number of probes: 267 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 73\n",
      "--- gene=73, id=99 removed by soft count = 115\n",
      "--- gene=73, id=100 removed by soft count = 159\n",
      "--- gene=73, id=106 removed by soft count = 92\n",
      "--- gene=73, id=107 removed by soft count = 147\n",
      "--- gene=73, id=108 removed by soft count = 268\n",
      "--- gene=73, id=109 removed by soft count = 620\n",
      "--- gene=73, id=185 removed by soft count = 114\n",
      "--- number of probes: 198 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 74\n",
      "--- gene=74, id=82 removed by soft count = 1181\n",
      "--- gene=74, id=84 removed by soft count = 115\n",
      "--- gene=74, id=86 removed by soft count = 92\n",
      "--- gene=74, id=87 removed by soft count = 2074\n",
      "--- gene=74, id=88 removed by soft count = 310\n",
      "--- gene=74, id=90 removed by soft count = 73\n",
      "--- gene=74, id=97 removed by soft count = 172\n",
      "--- number of probes: 173 , kept by blast: 166 , if remove dups: 166\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 75\n",
      "--- gene=75, id=2 removed by soft count = 244\n",
      "--- gene=75, id=30 removed by soft count = 241\n",
      "--- gene=75, id=81 removed by soft count = 421\n",
      "--- gene=75, id=82 removed by soft count = 142\n",
      "--- gene=75, id=83 removed by soft count = 1686\n",
      "--- gene=75, id=88 removed by soft count = 65\n",
      "--- gene=75, id=89 removed by soft count = 93\n",
      "--- gene=75, id=90 removed by soft count = 69\n",
      "--- gene=75, id=92 removed by soft count = 485\n",
      "--- gene=75, id=139 removed by soft count = 95\n",
      "--- number of probes: 149 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 76\n",
      "--- gene=76, id=1 removed by soft count = 285\n",
      "--- gene=76, id=3 removed by soft count = 406\n",
      "--- gene=76, id=4 removed by soft count = 90\n",
      "--- gene=76, id=5 removed by soft count = 432\n",
      "--- gene=76, id=6 removed by soft count = 279\n",
      "--- gene=76, id=7 removed by soft count = 417\n",
      "--- gene=76, id=8 removed by soft count = 111\n",
      "--- gene=76, id=9 removed by soft count = 293\n",
      "--- gene=76, id=10 removed by soft count = 363\n",
      "--- gene=76, id=11 removed by soft count = 232\n",
      "--- gene=76, id=12 removed by soft count = 488\n",
      "--- gene=76, id=13 removed by soft count = 493\n",
      "--- gene=76, id=14 removed by soft count = 288\n",
      "--- gene=76, id=15 removed by soft count = 389\n",
      "--- gene=76, id=17 removed by soft count = 425\n",
      "--- gene=76, id=18 removed by soft count = 591\n",
      "--- gene=76, id=19 removed by soft count = 350\n",
      "--- gene=76, id=20 removed by soft count = 57\n",
      "--- gene=76, id=35 removed by soft count = 2224\n",
      "--- gene=76, id=36 removed by soft count = 692\n",
      "--- gene=76, id=38 removed by soft count = 5399\n",
      "--- gene=76, id=40 removed by soft count = 476\n",
      "--- gene=76, id=46 removed by soft count = 488\n",
      "--- gene=76, id=53 removed by soft count = 389\n",
      "--- gene=76, id=55 removed by soft count = 148\n",
      "--- gene=76, id=58 removed by soft count = 462\n",
      "--- gene=76, id=60 removed by soft count = 224\n",
      "--- gene=76, id=61 removed by soft count = 231\n",
      "--- gene=76, id=62 removed by soft count = 321\n",
      "--- gene=76, id=66 removed by soft count = 83\n",
      "--- gene=76, id=67 removed by soft count = 65\n",
      "--- gene=76, id=69 removed by soft count = 305\n",
      "--- gene=76, id=70 removed by soft count = 152\n",
      "--- gene=76, id=117 removed by soft count = 72\n",
      "--- gene=76, id=120 removed by soft count = 60\n",
      "--- number of probes: 121 , kept by blast: 86 , if remove dups: 86\n",
      "-- number of probes kept for this region: 86\n",
      "-- checking probes in region: 77\n",
      "--- gene=77, id=51 removed by soft count = 143\n",
      "--- gene=77, id=88 removed by soft count = 285\n",
      "--- gene=77, id=89 removed by soft count = 187\n",
      "--- gene=77, id=130 removed by soft count = 69\n",
      "--- gene=77, id=171 removed by soft count = 53\n",
      "--- gene=77, id=172 removed by soft count = 221\n",
      "--- gene=77, id=181 removed by soft count = 105\n",
      "--- gene=77, id=184 removed by soft count = 129\n",
      "--- number of probes: 218 , kept by blast: 210 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 78\n",
      "--- gene=78, id=30 removed by soft count = 63\n",
      "--- gene=78, id=56 removed by soft count = 131\n",
      "--- gene=78, id=60 removed by soft count = 57\n",
      "--- gene=78, id=61 removed by soft count = 371\n",
      "--- gene=78, id=65 removed by soft count = 80\n",
      "--- gene=78, id=66 removed by soft count = 409\n",
      "--- gene=78, id=67 removed by soft count = 62\n",
      "--- gene=78, id=71 removed by soft count = 136\n",
      "--- gene=78, id=73 removed by soft count = 710\n",
      "--- gene=78, id=74 removed by soft count = 54\n",
      "--- gene=78, id=75 removed by soft count = 478\n",
      "--- gene=78, id=78 removed by soft count = 227\n",
      "--- gene=78, id=79 removed by soft count = 517\n",
      "--- number of probes: 142 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 79\n",
      "--- gene=79, id=6 removed by soft count = 153\n",
      "--- gene=79, id=7 removed by soft count = 226\n",
      "--- gene=79, id=9 removed by soft count = 551\n",
      "--- gene=79, id=11 removed by soft count = 110\n",
      "--- gene=79, id=173 removed by soft count = 135\n",
      "--- gene=79, id=174 removed by soft count = 765\n",
      "--- gene=79, id=175 removed by soft count = 635\n",
      "--- number of probes: 198 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 80\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=80, id=34 removed by soft count = 174\n",
      "--- gene=80, id=72 removed by soft count = 60\n",
      "--- gene=80, id=151 removed by soft count = 122\n",
      "--- gene=80, id=152 removed by soft count = 215\n",
      "--- gene=80, id=153 removed by soft count = 195\n",
      "--- gene=80, id=154 removed by soft count = 125\n",
      "--- gene=80, id=155 removed by soft count = 133\n",
      "--- gene=80, id=156 removed by soft count = 148\n",
      "--- gene=80, id=157 removed by soft count = 191\n",
      "--- gene=80, id=158 removed by soft count = 162\n",
      "--- number of probes: 241 , kept by blast: 231 , if remove dups: 231\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 81\n",
      "--- gene=81, id=181 removed by soft count = 215\n",
      "--- gene=81, id=182 removed by soft count = 1061\n",
      "--- gene=81, id=189 removed by soft count = 814\n",
      "--- gene=81, id=199 removed by soft count = 1580\n",
      "--- number of probes: 223 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 83\n",
      "--- gene=83, id=31 removed by soft count = 313\n",
      "--- gene=83, id=32 removed by soft count = 1844\n",
      "--- gene=83, id=58 removed by soft count = 76\n",
      "--- gene=83, id=60 removed by soft count = 187\n",
      "--- gene=83, id=104 removed by soft count = 198\n",
      "--- number of probes: 259 , kept by blast: 254 , if remove dups: 254\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 84\n",
      "--- gene=84, id=21 removed by soft count = 74\n",
      "--- gene=84, id=67 removed by soft count = 698\n",
      "--- gene=84, id=85 removed by soft count = 93\n",
      "--- gene=84, id=86 removed by soft count = 63\n",
      "--- gene=84, id=87 removed by soft count = 1439\n",
      "--- gene=84, id=188 removed by soft count = 155\n",
      "--- number of probes: 191 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 85\n",
      "--- gene=85, id=45 removed by soft count = 118\n",
      "--- gene=85, id=56 removed by soft count = 534\n",
      "--- gene=85, id=79 removed by soft count = 98\n",
      "--- gene=85, id=204 removed by soft count = 590\n",
      "--- gene=85, id=227 removed by soft count = 108\n",
      "--- gene=85, id=229 removed by soft count = 64\n",
      "--- gene=85, id=230 removed by soft count = 368\n",
      "--- gene=85, id=231 removed by soft count = 111\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 86\n",
      "--- gene=86, id=232 removed by soft count = 328\n",
      "--- gene=86, id=233 removed by soft count = 164\n",
      "--- gene=86, id=235 removed by soft count = 263\n",
      "--- gene=86, id=236 removed by soft count = 190\n",
      "--- number of probes: 237 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 87\n",
      "--- gene=87, id=65 removed by soft count = 430\n",
      "--- gene=87, id=77 removed by soft count = 330\n",
      "--- gene=87, id=79 removed by soft count = 61\n",
      "--- gene=87, id=82 removed by soft count = 74\n",
      "--- gene=87, id=87 removed by soft count = 116\n",
      "--- gene=87, id=88 removed by soft count = 110\n",
      "--- gene=87, id=91 removed by soft count = 163\n",
      "--- gene=87, id=92 removed by soft count = 329\n",
      "--- gene=87, id=93 removed by soft count = 109\n",
      "--- gene=87, id=94 removed by soft count = 167\n",
      "--- gene=87, id=96 removed by soft count = 191\n",
      "--- gene=87, id=103 removed by soft count = 62\n",
      "--- gene=87, id=104 removed by soft count = 68\n",
      "--- gene=87, id=107 removed by soft count = 99\n",
      "--- gene=87, id=108 removed by soft count = 78\n",
      "--- gene=87, id=111 removed by soft count = 72\n",
      "--- gene=87, id=116 removed by soft count = 96\n",
      "--- gene=87, id=117 removed by soft count = 53\n",
      "--- gene=87, id=118 removed by soft count = 81\n",
      "--- gene=87, id=119 removed by soft count = 59\n",
      "--- gene=87, id=129 removed by soft count = 85\n",
      "--- gene=87, id=130 removed by soft count = 95\n",
      "--- gene=87, id=131 removed by soft count = 53\n",
      "--- gene=87, id=140 removed by soft count = 105\n",
      "--- gene=87, id=143 removed by soft count = 77\n",
      "--- gene=87, id=144 removed by soft count = 150\n",
      "--- gene=87, id=150 removed by soft count = 579\n",
      "--- number of probes: 236 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 88\n",
      "--- number of probes: 204 , kept by blast: 204 , if remove dups: 204\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 89\n",
      "--- gene=89, id=232 removed by soft count = 55\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 90\n",
      "--- gene=90, id=18 removed by soft count = 71\n",
      "--- gene=90, id=45 removed by soft count = 744\n",
      "--- gene=90, id=91 removed by soft count = 57\n",
      "--- gene=90, id=97 removed by soft count = 101\n",
      "--- gene=90, id=100 removed by soft count = 147\n",
      "--- gene=90, id=107 removed by soft count = 166\n",
      "--- gene=90, id=111 removed by soft count = 101\n",
      "--- gene=90, id=112 removed by soft count = 86\n",
      "--- number of probes: 118 , kept by blast: 110 , if remove dups: 110\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 110\n",
      "-- checking probes in region: 91\n",
      "--- gene=91, id=294 removed by soft count = 119\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 92\n",
      "--- gene=92, id=2 removed by soft count = 557\n",
      "--- gene=92, id=134 removed by soft count = 85\n",
      "--- gene=92, id=263 removed by soft count = 191\n",
      "--- number of probes: 271 , kept by blast: 268 , if remove dups: 268\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 93\n",
      "--- gene=93, id=42 removed by soft count = 57\n",
      "--- gene=93, id=132 removed by soft count = 166\n",
      "--- number of probes: 212 , kept by blast: 210 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 94\n",
      "--- gene=94, id=163 removed by soft count = 107\n",
      "--- gene=94, id=164 removed by soft count = 159\n",
      "--- number of probes: 178 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 95\n",
      "--- gene=95, id=27 removed by soft count = 138\n",
      "--- gene=95, id=40 removed by soft count = 59\n",
      "--- gene=95, id=48 removed by soft count = 72\n",
      "--- gene=95, id=50 removed by soft count = 64\n",
      "--- gene=95, id=52 removed by soft count = 63\n",
      "--- gene=95, id=54 removed by soft count = 74\n",
      "--- gene=95, id=74 removed by soft count = 54\n",
      "--- gene=95, id=101 removed by soft count = 54\n",
      "--- gene=95, id=119 removed by soft count = 138\n",
      "--- gene=95, id=200 removed by soft count = 534\n",
      "--- gene=95, id=201 removed by soft count = 791\n",
      "--- gene=95, id=202 removed by soft count = 90\n",
      "--- gene=95, id=204 removed by soft count = 52\n",
      "--- gene=95, id=206 removed by soft count = 107\n",
      "--- gene=95, id=207 removed by soft count = 402\n",
      "--- gene=95, id=208 removed by soft count = 76\n",
      "--- gene=95, id=210 removed by soft count = 65\n",
      "--- gene=95, id=212 removed by soft count = 545\n",
      "--- number of probes: 214 , kept by blast: 196 , if remove dups: 195\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 96\n",
      "--- gene=96, id=71 removed by soft count = 348\n",
      "--- gene=96, id=82 removed by soft count = 132\n",
      "--- gene=96, id=105 removed by soft count = 143\n",
      "--- gene=96, id=163 removed by soft count = 268\n",
      "--- gene=96, id=185 removed by soft count = 282\n",
      "--- number of probes: 258 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 98\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=98, id=27 removed by soft count = 69\n",
      "--- gene=98, id=107 removed by soft count = 120\n",
      "--- gene=98, id=114 removed by soft count = 146\n",
      "--- gene=98, id=115 removed by soft count = 155\n",
      "--- gene=98, id=116 removed by soft count = 115\n",
      "--- gene=98, id=118 removed by soft count = 99\n",
      "--- gene=98, id=121 removed by soft count = 107\n",
      "--- gene=98, id=135 removed by soft count = 126\n",
      "--- number of probes: 166 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 100\n",
      "--- gene=100, id=47 removed by soft count = 103\n",
      "--- gene=100, id=48 removed by soft count = 92\n",
      "--- gene=100, id=49 removed by soft count = 63\n",
      "--- gene=100, id=51 removed by soft count = 86\n",
      "--- gene=100, id=52 removed by soft count = 197\n",
      "--- gene=100, id=53 removed by soft count = 466\n",
      "--- gene=100, id=54 removed by soft count = 273\n",
      "--- gene=100, id=271 removed by soft count = 2348\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 101\n",
      "--- gene=101, id=79 removed by soft count = 190\n",
      "--- gene=101, id=104 removed by soft count = 97\n",
      "--- gene=101, id=107 removed by soft count = 78\n",
      "--- gene=101, id=169 removed by soft count = 148\n",
      "--- number of probes: 282 , kept by blast: 278 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 102\n",
      "--- gene=102, id=167 removed by soft count = 1149\n",
      "--- gene=102, id=170 removed by soft count = 89\n",
      "--- gene=102, id=175 removed by soft count = 187\n",
      "--- gene=102, id=189 removed by soft count = 282\n",
      "--- gene=102, id=232 removed by soft count = 588\n",
      "--- number of probes: 240 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 103\n",
      "--- gene=103, id=21 removed by soft count = 75\n",
      "--- gene=103, id=29 removed by soft count = 62\n",
      "--- number of probes: 205 , kept by blast: 203 , if remove dups: 203\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 104\n",
      "--- gene=104, id=2 removed by soft count = 135\n",
      "--- gene=104, id=4 removed by soft count = 167\n",
      "--- gene=104, id=5 removed by soft count = 1193\n",
      "--- gene=104, id=6 removed by soft count = 848\n",
      "--- gene=104, id=7 removed by soft count = 508\n",
      "--- gene=104, id=8 removed by soft count = 136\n",
      "--- gene=104, id=9 removed by soft count = 66\n",
      "--- gene=104, id=11 removed by soft count = 88\n",
      "--- gene=104, id=13 removed by soft count = 95\n",
      "--- gene=104, id=14 removed by soft count = 226\n",
      "--- gene=104, id=15 removed by soft count = 177\n",
      "--- gene=104, id=16 removed by soft count = 203\n",
      "--- gene=104, id=18 removed by soft count = 223\n",
      "--- gene=104, id=20 removed by soft count = 1021\n",
      "--- gene=104, id=21 removed by soft count = 113\n",
      "--- gene=104, id=23 removed by soft count = 52\n",
      "--- gene=104, id=24 removed by soft count = 88\n",
      "--- gene=104, id=25 removed by soft count = 63\n",
      "--- gene=104, id=26 removed by soft count = 85\n",
      "--- gene=104, id=27 removed by soft count = 79\n",
      "--- gene=104, id=28 removed by soft count = 72\n",
      "--- gene=104, id=29 removed by soft count = 590\n",
      "--- gene=104, id=30 removed by soft count = 105\n",
      "--- number of probes: 119 , kept by blast: 96 , if remove dups: 96\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 96\n",
      "-- checking probes in region: 105\n",
      "--- gene=105, id=92 removed by soft count = 73\n",
      "--- gene=105, id=193 removed by soft count = 70\n",
      "--- gene=105, id=196 removed by soft count = 52\n",
      "--- gene=105, id=198 removed by soft count = 85\n",
      "--- gene=105, id=204 removed by soft count = 97\n",
      "--- gene=105, id=205 removed by soft count = 124\n",
      "--- gene=105, id=209 removed by soft count = 94\n",
      "--- gene=105, id=210 removed by soft count = 58\n",
      "--- gene=105, id=212 removed by soft count = 56\n",
      "--- number of probes: 300 , kept by blast: 291 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 106\n",
      "--- gene=106, id=179 removed by soft count = 242\n",
      "--- gene=106, id=183 removed by soft count = 501\n",
      "--- gene=106, id=240 removed by soft count = 273\n",
      "--- number of probes: 244 , kept by blast: 241 , if remove dups: 241\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 107\n",
      "--- gene=107, id=4 removed by soft count = 62\n",
      "--- gene=107, id=41 removed by soft count = 3348\n",
      "--- gene=107, id=42 removed by soft count = 284\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 108\n",
      "--- gene=108, id=180 removed by soft count = 283\n",
      "--- gene=108, id=181 removed by soft count = 192\n",
      "--- number of probes: 213 , kept by blast: 211 , if remove dups: 211\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 109\n",
      "--- gene=109, id=175 removed by soft count = 73\n",
      "--- gene=109, id=176 removed by soft count = 93\n",
      "--- gene=109, id=179 removed by soft count = 91\n",
      "--- gene=109, id=181 removed by soft count = 648\n",
      "--- gene=109, id=186 removed by soft count = 104\n",
      "--- number of probes: 225 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 110\n",
      "--- gene=110, id=43 removed by soft count = 3166\n",
      "--- gene=110, id=176 removed by soft count = 226\n",
      "--- gene=110, id=197 removed by soft count = 385\n",
      "--- gene=110, id=200 removed by soft count = 447\n",
      "--- gene=110, id=228 removed by soft count = 1153\n",
      "--- number of probes: 230 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 111\n",
      "--- gene=111, id=231 removed by soft count = 54\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 112\n",
      "--- gene=112, id=3 removed by soft count = 82\n",
      "--- gene=112, id=7 removed by soft count = 69\n",
      "--- gene=112, id=9 removed by soft count = 55\n",
      "--- gene=112, id=15 removed by soft count = 118\n",
      "--- gene=112, id=16 removed by soft count = 2979\n",
      "--- gene=112, id=17 removed by soft count = 1739\n",
      "--- gene=112, id=19 removed by soft count = 281\n",
      "--- gene=112, id=20 removed by soft count = 66\n",
      "--- gene=112, id=22 removed by soft count = 192\n",
      "--- gene=112, id=24 removed by soft count = 148\n",
      "--- gene=112, id=26 removed by soft count = 420\n",
      "--- gene=112, id=27 removed by soft count = 149\n",
      "--- gene=112, id=29 removed by soft count = 187\n",
      "--- gene=112, id=121 removed by soft count = 87\n",
      "--- gene=112, id=123 removed by soft count = 146\n",
      "--- gene=112, id=138 removed by soft count = 1289\n",
      "--- number of probes: 167 , kept by blast: 151 , if remove dups: 151\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 114\n",
      "--- gene=114, id=9 removed by soft count = 74\n",
      "--- gene=114, id=181 removed by soft count = 331\n",
      "--- number of probes: 215 , kept by blast: 213 , if remove dups: 213\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 115\n",
      "--- gene=115, id=6 removed by soft count = 843\n",
      "--- gene=115, id=11 removed by soft count = 99\n",
      "--- gene=115, id=101 removed by soft count = 123\n",
      "--- gene=115, id=102 removed by soft count = 53\n",
      "--- number of probes: 224 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 116\n",
      "--- gene=116, id=1 removed by soft count = 145\n",
      "--- gene=116, id=103 removed by soft count = 524\n",
      "--- gene=116, id=104 removed by soft count = 1086\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 228 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 117\n",
      "--- gene=117, id=100 removed by soft count = 129\n",
      "--- gene=117, id=106 removed by soft count = 674\n",
      "--- gene=117, id=209 removed by soft count = 73\n",
      "--- gene=117, id=268 removed by soft count = 142\n",
      "--- number of probes: 276 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 118\n",
      "--- number of probes: 216 , kept by blast: 216 , if remove dups: 216\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 119\n",
      "--- gene=119, id=90 removed by soft count = 142\n",
      "--- gene=119, id=142 removed by soft count = 133\n",
      "--- gene=119, id=148 removed by soft count = 75\n",
      "--- gene=119, id=150 removed by soft count = 89\n",
      "--- gene=119, id=154 removed by soft count = 116\n",
      "--- gene=119, id=155 removed by soft count = 73\n",
      "--- gene=119, id=201 removed by soft count = 90\n",
      "--- number of probes: 221 , kept by blast: 214 , if remove dups: 214\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 120\n",
      "--- gene=120, id=3 removed by soft count = 136\n",
      "--- gene=120, id=11 removed by soft count = 1962\n",
      "--- gene=120, id=105 removed by soft count = 70\n",
      "--- gene=120, id=126 removed by soft count = 211\n",
      "--- gene=120, id=127 removed by soft count = 56\n",
      "--- gene=120, id=128 removed by soft count = 73\n",
      "--- gene=120, id=159 removed by soft count = 241\n",
      "--- gene=120, id=160 removed by soft count = 585\n",
      "--- gene=120, id=163 removed by soft count = 76\n",
      "--- gene=120, id=164 removed by soft count = 119\n",
      "--- gene=120, id=166 removed by soft count = 82\n",
      "--- gene=120, id=167 removed by soft count = 63\n",
      "--- gene=120, id=168 removed by soft count = 124\n",
      "--- gene=120, id=169 removed by soft count = 135\n",
      "--- gene=120, id=170 removed by soft count = 113\n",
      "--- gene=120, id=171 removed by soft count = 121\n",
      "--- gene=120, id=173 removed by soft count = 169\n",
      "--- gene=120, id=185 removed by soft count = 76\n",
      "--- number of probes: 200 , kept by blast: 182 , if remove dups: 182\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 121\n",
      "--- gene=121, id=62 removed by soft count = 293\n",
      "--- gene=121, id=63 removed by soft count = 79\n",
      "--- gene=121, id=65 removed by soft count = 138\n",
      "--- gene=121, id=67 removed by soft count = 106\n",
      "--- gene=121, id=70 removed by soft count = 128\n",
      "--- gene=121, id=71 removed by soft count = 165\n",
      "--- gene=121, id=72 removed by soft count = 255\n",
      "--- gene=121, id=76 removed by soft count = 63\n",
      "--- gene=121, id=148 removed by soft count = 97\n",
      "--- number of probes: 295 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 122\n",
      "--- gene=122, id=5 removed by soft count = 130\n",
      "--- gene=122, id=98 removed by soft count = 3966\n",
      "--- gene=122, id=161 removed by soft count = 170\n",
      "--- number of probes: 270 , kept by blast: 267 , if remove dups: 267\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 123\n",
      "--- gene=123, id=23 removed by soft count = 498\n",
      "--- gene=123, id=24 removed by soft count = 171\n",
      "--- gene=123, id=48 removed by soft count = 640\n",
      "--- gene=123, id=168 removed by soft count = 67\n",
      "--- gene=123, id=170 removed by soft count = 151\n",
      "--- gene=123, id=174 removed by soft count = 172\n",
      "--- gene=123, id=176 removed by soft count = 94\n",
      "--- number of probes: 182 , kept by blast: 175 , if remove dups: 174\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 124\n",
      "--- gene=124, id=93 removed by soft count = 142\n",
      "--- gene=124, id=96 removed by soft count = 915\n",
      "--- gene=124, id=113 removed by soft count = 91\n",
      "--- gene=124, id=124 removed by soft count = 61\n",
      "--- gene=124, id=125 removed by soft count = 67\n",
      "--- gene=124, id=127 removed by soft count = 81\n",
      "--- gene=124, id=128 removed by soft count = 79\n",
      "--- number of probes: 235 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 125\n",
      "--- gene=125, id=24 removed by soft count = 4093\n",
      "--- gene=125, id=100 removed by soft count = 798\n",
      "--- gene=125, id=113 removed by soft count = 51\n",
      "--- number of probes: 251 , kept by blast: 248 , if remove dups: 242\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 126\n",
      "--- gene=126, id=249 removed by soft count = 69\n",
      "--- gene=126, id=251 removed by soft count = 51\n",
      "--- gene=126, id=259 removed by soft count = 109\n",
      "--- gene=126, id=263 removed by soft count = 65\n",
      "--- gene=126, id=265 removed by soft count = 56\n",
      "--- number of probes: 291 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 127\n",
      "--- gene=127, id=2 removed by soft count = 52\n",
      "--- gene=127, id=17 removed by soft count = 64\n",
      "--- gene=127, id=116 removed by soft count = 290\n",
      "--- gene=127, id=117 removed by soft count = 162\n",
      "--- gene=127, id=178 removed by soft count = 85\n",
      "--- number of probes: 222 , kept by blast: 217 , if remove dups: 217\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 128\n",
      "--- gene=128, id=24 removed by soft count = 565\n",
      "--- gene=128, id=28 removed by soft count = 126\n",
      "--- gene=128, id=172 removed by soft count = 362\n",
      "--- number of probes: 275 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 129\n",
      "--- gene=129, id=49 removed by soft count = 92\n",
      "--- gene=129, id=55 removed by soft count = 134\n",
      "--- gene=129, id=56 removed by soft count = 181\n",
      "--- number of probes: 216 , kept by blast: 213 , if remove dups: 213\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 130\n",
      "--- gene=130, id=59 removed by soft count = 61\n",
      "--- gene=130, id=70 removed by soft count = 365\n",
      "--- gene=130, id=92 removed by soft count = 74\n",
      "--- gene=130, id=127 removed by soft count = 688\n",
      "--- gene=130, id=145 removed by soft count = 72\n",
      "--- gene=130, id=152 removed by soft count = 124\n",
      "--- gene=130, id=153 removed by soft count = 250\n",
      "--- gene=130, id=154 removed by soft count = 608\n",
      "--- gene=130, id=155 removed by soft count = 271\n",
      "--- gene=130, id=156 removed by soft count = 136\n",
      "--- gene=130, id=157 removed by soft count = 195\n",
      "--- gene=130, id=158 removed by soft count = 214\n",
      "--- gene=130, id=159 removed by soft count = 376\n",
      "--- number of probes: 162 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 131\n",
      "--- gene=131, id=38 removed by soft count = 783\n",
      "--- gene=131, id=131 removed by soft count = 352\n",
      "--- gene=131, id=132 removed by soft count = 267\n",
      "--- gene=131, id=134 removed by soft count = 529\n",
      "--- gene=131, id=138 removed by soft count = 81\n",
      "--- gene=131, id=187 removed by soft count = 76\n",
      "--- gene=131, id=189 removed by soft count = 150\n",
      "--- number of probes: 190 , kept by blast: 183 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 132\n",
      "--- gene=132, id=35 removed by soft count = 76\n",
      "--- gene=132, id=36 removed by soft count = 923\n",
      "--- gene=132, id=37 removed by soft count = 201\n",
      "--- gene=132, id=38 removed by soft count = 56\n",
      "--- gene=132, id=39 removed by soft count = 84\n",
      "--- gene=132, id=105 removed by soft count = 58\n",
      "--- gene=132, id=106 removed by soft count = 69\n",
      "--- gene=132, id=108 removed by soft count = 110\n",
      "--- gene=132, id=110 removed by soft count = 122\n",
      "--- gene=132, id=111 removed by soft count = 116\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=132, id=112 removed by soft count = 318\n",
      "--- gene=132, id=113 removed by soft count = 265\n",
      "--- gene=132, id=114 removed by soft count = 238\n",
      "--- gene=132, id=115 removed by soft count = 63\n",
      "--- gene=132, id=169 removed by soft count = 62\n",
      "--- gene=132, id=174 removed by soft count = 64\n",
      "--- gene=132, id=183 removed by soft count = 67\n",
      "--- number of probes: 192 , kept by blast: 175 , if remove dups: 175\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 133\n",
      "--- gene=133, id=111 removed by soft count = 133\n",
      "--- gene=133, id=239 removed by soft count = 166\n",
      "--- gene=133, id=266 removed by soft count = 91\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 134\n",
      "--- gene=134, id=82 removed by soft count = 1065\n",
      "--- gene=134, id=86 removed by soft count = 138\n",
      "--- gene=134, id=88 removed by soft count = 158\n",
      "--- gene=134, id=213 removed by soft count = 215\n",
      "--- gene=134, id=224 removed by soft count = 735\n",
      "--- number of probes: 237 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 137\n",
      "--- gene=137, id=93 removed by soft count = 2696\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 138\n",
      "--- gene=138, id=143 removed by soft count = 71\n",
      "--- gene=138, id=144 removed by soft count = 101\n",
      "--- gene=138, id=158 removed by soft count = 54\n",
      "--- gene=138, id=166 removed by soft count = 80\n",
      "--- number of probes: 285 , kept by blast: 281 , if remove dups: 281\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 139\n",
      "--- gene=139, id=0 removed by soft count = 205\n",
      "--- gene=139, id=2 removed by soft count = 93\n",
      "--- gene=139, id=4 removed by soft count = 220\n",
      "--- gene=139, id=5 removed by soft count = 144\n",
      "--- gene=139, id=15 removed by soft count = 59\n",
      "--- gene=139, id=30 removed by soft count = 294\n",
      "--- gene=139, id=59 removed by soft count = 87\n",
      "--- gene=139, id=119 removed by soft count = 89\n",
      "--- gene=139, id=145 removed by soft count = 54\n",
      "--- number of probes: 174 , kept by blast: 165 , if remove dups: 165\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 140\n",
      "--- gene=140, id=121 removed by soft count = 124\n",
      "--- gene=140, id=215 removed by soft count = 164\n",
      "--- gene=140, id=234 removed by soft count = 215\n",
      "--- number of probes: 261 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 141\n",
      "--- gene=141, id=0 removed by soft count = 405\n",
      "--- gene=141, id=1 removed by soft count = 181\n",
      "--- gene=141, id=2 removed by soft count = 305\n",
      "--- gene=141, id=5 removed by soft count = 79\n",
      "--- gene=141, id=12 removed by soft count = 79\n",
      "--- gene=141, id=19 removed by soft count = 109\n",
      "--- number of probes: 74 , kept by blast: 68 , if remove dups: 65\n",
      "-- number of probes kept for this region: 68\n",
      "-- checking probes in region: 143\n",
      "--- gene=143, id=151 removed by soft count = 280\n",
      "--- number of probes: 186 , kept by blast: 185 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 144\n",
      "--- gene=144, id=5 removed by soft count = 103\n",
      "--- gene=144, id=29 removed by soft count = 68\n",
      "--- gene=144, id=37 removed by soft count = 347\n",
      "--- gene=144, id=84 removed by soft count = 83\n",
      "--- gene=144, id=184 removed by soft count = 176\n",
      "--- gene=144, id=186 removed by soft count = 78\n",
      "--- number of probes: 187 , kept by blast: 181 , if remove dups: 181\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 145\n",
      "--- gene=145, id=0 removed by soft count = 3822\n",
      "--- gene=145, id=29 removed by soft count = 52\n",
      "--- gene=145, id=50 removed by soft count = 74\n",
      "--- gene=145, id=59 removed by soft count = 83\n",
      "--- gene=145, id=76 removed by soft count = 68\n",
      "--- gene=145, id=120 removed by soft count = 615\n",
      "--- gene=145, id=121 removed by soft count = 71\n",
      "--- gene=145, id=122 removed by soft count = 201\n",
      "--- gene=145, id=123 removed by soft count = 199\n",
      "--- gene=145, id=124 removed by soft count = 101\n",
      "--- gene=145, id=125 removed by soft count = 142\n",
      "--- gene=145, id=126 removed by soft count = 141\n",
      "--- gene=145, id=135 removed by soft count = 244\n",
      "--- gene=145, id=136 removed by soft count = 384\n",
      "--- gene=145, id=138 removed by soft count = 105\n",
      "--- gene=145, id=140 removed by soft count = 74\n",
      "--- gene=145, id=141 removed by soft count = 419\n",
      "--- gene=145, id=142 removed by soft count = 1371\n",
      "--- gene=145, id=153 removed by soft count = 143\n",
      "--- gene=145, id=159 removed by soft count = 188\n",
      "--- gene=145, id=175 removed by soft count = 51\n",
      "--- number of probes: 178 , kept by blast: 157 , if remove dups: 157\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 146\n",
      "--- gene=146, id=0 removed by soft count = 1060\n",
      "--- gene=146, id=33 removed by soft count = 319\n",
      "--- gene=146, id=36 removed by soft count = 102\n",
      "--- gene=146, id=72 removed by soft count = 1061\n",
      "--- gene=146, id=73 removed by soft count = 70\n",
      "--- gene=146, id=74 removed by soft count = 193\n",
      "--- gene=146, id=75 removed by soft count = 2239\n",
      "--- gene=146, id=244 removed by soft count = 365\n",
      "--- gene=146, id=245 removed by soft count = 80\n",
      "--- gene=146, id=247 removed by soft count = 2233\n",
      "--- number of probes: 294 , kept by blast: 284 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 147\n",
      "--- gene=147, id=90 removed by soft count = 108\n",
      "--- gene=147, id=153 removed by soft count = 1150\n",
      "--- gene=147, id=166 removed by soft count = 249\n",
      "--- gene=147, id=234 removed by soft count = 54\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 148\n",
      "--- gene=148, id=190 removed by soft count = 188\n",
      "--- number of probes: 193 , kept by blast: 192 , if remove dups: 192\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 149\n",
      "--- gene=149, id=255 removed by soft count = 51\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 150\n",
      "--- gene=150, id=66 removed by soft count = 163\n",
      "--- gene=150, id=94 removed by soft count = 505\n",
      "--- gene=150, id=228 removed by soft count = 56\n",
      "--- gene=150, id=230 removed by soft count = 191\n",
      "--- gene=150, id=233 removed by soft count = 60\n",
      "--- gene=150, id=236 removed by soft count = 81\n",
      "--- gene=150, id=238 removed by soft count = 8688\n",
      "--- number of probes: 240 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 152\n",
      "--- gene=152, id=60 removed by soft count = 248\n",
      "--- gene=152, id=177 removed by soft count = 62\n",
      "--- gene=152, id=196 removed by soft count = 103\n",
      "--- number of probes: 248 , kept by blast: 245 , if remove dups: 245\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 153\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 154\n",
      "--- gene=154, id=3 removed by soft count = 188\n",
      "--- gene=154, id=160 removed by soft count = 93\n",
      "--- gene=154, id=162 removed by soft count = 99\n",
      "--- gene=154, id=167 removed by soft count = 83\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=154, id=180 removed by soft count = 101\n",
      "--- number of probes: 283 , kept by blast: 278 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 155\n",
      "--- gene=155, id=133 removed by soft count = 60\n",
      "--- gene=155, id=134 removed by soft count = 61\n",
      "--- gene=155, id=136 removed by soft count = 74\n",
      "--- number of probes: 252 , kept by blast: 249 , if remove dups: 249\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 156\n",
      "--- gene=156, id=125 removed by soft count = 368\n",
      "--- gene=156, id=231 removed by soft count = 5316\n",
      "--- number of probes: 292 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 157\n",
      "--- number of probes: 255 , kept by blast: 255 , if remove dups: 255\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 158\n",
      "--- gene=158, id=105 removed by soft count = 72\n",
      "--- number of probes: 227 , kept by blast: 226 , if remove dups: 226\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 159\n",
      "--- gene=159, id=3 removed by soft count = 132\n",
      "--- gene=159, id=57 removed by soft count = 92\n",
      "--- gene=159, id=122 removed by soft count = 97\n",
      "--- number of probes: 231 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 160\n",
      "--- gene=160, id=68 removed by soft count = 112\n",
      "--- gene=160, id=69 removed by soft count = 149\n",
      "--- gene=160, id=73 removed by soft count = 78\n",
      "--- gene=160, id=75 removed by soft count = 398\n",
      "--- gene=160, id=77 removed by soft count = 123\n",
      "--- gene=160, id=86 removed by soft count = 325\n",
      "--- gene=160, id=87 removed by soft count = 100\n",
      "--- gene=160, id=90 removed by soft count = 163\n",
      "--- gene=160, id=92 removed by soft count = 137\n",
      "--- gene=160, id=106 removed by soft count = 54\n",
      "--- gene=160, id=134 removed by soft count = 175\n",
      "--- gene=160, id=135 removed by soft count = 473\n",
      "--- number of probes: 136 , kept by blast: 124 , if remove dups: 124\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 161\n",
      "--- gene=161, id=122 removed by soft count = 56\n",
      "--- number of probes: 134 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 162\n",
      "--- gene=162, id=53 removed by soft count = 154\n",
      "--- gene=162, id=79 removed by soft count = 201\n",
      "--- gene=162, id=80 removed by soft count = 60\n",
      "--- gene=162, id=82 removed by soft count = 57\n",
      "--- gene=162, id=210 removed by soft count = 71\n",
      "--- gene=162, id=228 removed by soft count = 81\n",
      "--- gene=162, id=231 removed by soft count = 439\n",
      "--- number of probes: 300 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 163\n",
      "--- gene=163, id=215 removed by soft count = 132\n",
      "--- gene=163, id=220 removed by soft count = 165\n",
      "--- gene=163, id=230 removed by soft count = 172\n",
      "--- number of probes: 234 , kept by blast: 231 , if remove dups: 231\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 164\n",
      "--- gene=164, id=57 removed by soft count = 727\n",
      "--- gene=164, id=131 removed by soft count = 219\n",
      "--- gene=164, id=299 removed by soft count = 104\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 165\n",
      "--- gene=165, id=230 removed by soft count = 373\n",
      "--- number of probes: 244 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 166\n",
      "--- gene=166, id=10 removed by soft count = 275\n",
      "--- gene=166, id=14 removed by soft count = 8707\n",
      "--- gene=166, id=268 removed by soft count = 1650\n",
      "--- number of probes: 299 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 167\n",
      "--- gene=167, id=1 removed by soft count = 64\n",
      "--- gene=167, id=3 removed by soft count = 66\n",
      "--- gene=167, id=4 removed by soft count = 71\n",
      "--- gene=167, id=39 removed by soft count = 138\n",
      "--- gene=167, id=65 removed by soft count = 94\n",
      "--- gene=167, id=79 removed by soft count = 52\n",
      "--- gene=167, id=81 removed by soft count = 64\n",
      "--- gene=167, id=83 removed by soft count = 137\n",
      "--- gene=167, id=91 removed by soft count = 109\n",
      "--- gene=167, id=93 removed by soft count = 526\n",
      "--- gene=167, id=184 removed by soft count = 168\n",
      "--- gene=167, id=186 removed by soft count = 93\n",
      "--- number of probes: 217 , kept by blast: 205 , if remove dups: 205\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 168\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 169\n",
      "--- gene=169, id=15 removed by soft count = 60\n",
      "--- gene=169, id=50 removed by soft count = 54\n",
      "--- gene=169, id=87 removed by soft count = 174\n",
      "--- gene=169, id=128 removed by soft count = 300\n",
      "--- gene=169, id=129 removed by soft count = 428\n",
      "--- number of probes: 238 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 170\n",
      "--- number of probes: 299 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 171\n",
      "--- number of probes: 272 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 172\n",
      "--- gene=172, id=134 removed by soft count = 516\n",
      "--- gene=172, id=135 removed by soft count = 120\n",
      "--- number of probes: 200 , kept by blast: 198 , if remove dups: 198\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 173\n",
      "--- gene=173, id=0 removed by soft count = 61\n",
      "--- gene=173, id=1 removed by soft count = 292\n",
      "--- gene=173, id=2 removed by soft count = 528\n",
      "--- gene=173, id=3 removed by soft count = 177\n",
      "--- gene=173, id=6 removed by soft count = 1383\n",
      "--- gene=173, id=52 removed by soft count = 216\n",
      "--- gene=173, id=112 removed by soft count = 218\n",
      "--- gene=173, id=159 removed by soft count = 142\n",
      "--- gene=173, id=162 removed by soft count = 162\n",
      "--- gene=173, id=164 removed by soft count = 127\n",
      "--- gene=173, id=166 removed by soft count = 123\n",
      "--- gene=173, id=167 removed by soft count = 118\n",
      "--- gene=173, id=172 removed by soft count = 153\n",
      "--- number of probes: 173 , kept by blast: 160 , if remove dups: 160\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 174\n",
      "--- gene=174, id=136 removed by soft count = 5795\n",
      "--- gene=174, id=138 removed by soft count = 1335\n",
      "--- number of probes: 198 , kept by blast: 196 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 175\n",
      "--- gene=175, id=226 removed by soft count = 132\n",
      "--- gene=175, id=228 removed by soft count = 159\n",
      "--- gene=175, id=229 removed by soft count = 122\n",
      "--- gene=175, id=230 removed by soft count = 249\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 176\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=176, id=17 removed by soft count = 110\n",
      "--- gene=176, id=18 removed by soft count = 269\n",
      "--- gene=176, id=20 removed by soft count = 153\n",
      "--- gene=176, id=23 removed by soft count = 239\n",
      "--- gene=176, id=24 removed by soft count = 110\n",
      "--- gene=176, id=25 removed by soft count = 276\n",
      "--- gene=176, id=26 removed by soft count = 369\n",
      "--- gene=176, id=28 removed by soft count = 53\n",
      "--- gene=176, id=29 removed by soft count = 100\n",
      "--- gene=176, id=39 removed by soft count = 208\n",
      "--- gene=176, id=41 removed by soft count = 241\n",
      "--- gene=176, id=42 removed by soft count = 480\n",
      "--- gene=176, id=44 removed by soft count = 103\n",
      "--- gene=176, id=45 removed by soft count = 4189\n",
      "--- gene=176, id=91 removed by soft count = 462\n",
      "--- gene=176, id=100 removed by soft count = 130\n",
      "--- number of probes: 152 , kept by blast: 136 , if remove dups: 136\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 177\n",
      "--- gene=177, id=8 removed by soft count = 211\n",
      "--- gene=177, id=9 removed by soft count = 81\n",
      "--- number of probes: 279 , kept by blast: 277 , if remove dups: 277\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 178\n",
      "--- gene=178, id=15 removed by soft count = 68\n",
      "--- number of probes: 280 , kept by blast: 279 , if remove dups: 279\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 179\n",
      "--- gene=179, id=7 removed by soft count = 79\n",
      "--- gene=179, id=12 removed by soft count = 208\n",
      "--- gene=179, id=39 removed by soft count = 53\n",
      "--- gene=179, id=43 removed by soft count = 504\n",
      "--- gene=179, id=113 removed by soft count = 59\n",
      "--- gene=179, id=121 removed by soft count = 342\n",
      "--- gene=179, id=150 removed by soft count = 2762\n",
      "--- number of probes: 230 , kept by blast: 223 , if remove dups: 223\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 180\n",
      "--- gene=180, id=107 removed by soft count = 161\n",
      "--- gene=180, id=170 removed by soft count = 263\n",
      "--- gene=180, id=171 removed by soft count = 2219\n",
      "--- gene=180, id=172 removed by soft count = 1278\n",
      "--- gene=180, id=174 removed by soft count = 105\n",
      "--- gene=180, id=175 removed by soft count = 372\n",
      "--- gene=180, id=179 removed by soft count = 305\n",
      "--- gene=180, id=180 removed by soft count = 839\n",
      "--- gene=180, id=182 removed by soft count = 186\n",
      "--- gene=180, id=222 removed by soft count = 71\n",
      "--- gene=180, id=224 removed by soft count = 53\n",
      "--- gene=180, id=225 removed by soft count = 55\n",
      "--- number of probes: 226 , kept by blast: 214 , if remove dups: 214\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 181\n",
      "--- gene=181, id=21 removed by soft count = 61\n",
      "--- gene=181, id=92 removed by soft count = 93\n",
      "--- gene=181, id=187 removed by soft count = 178\n",
      "--- gene=181, id=189 removed by soft count = 59\n",
      "--- gene=181, id=247 removed by soft count = 138\n",
      "--- number of probes: 248 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 182\n",
      "--- gene=182, id=0 removed by soft count = 141\n",
      "--- gene=182, id=2 removed by soft count = 184\n",
      "--- gene=182, id=3 removed by soft count = 70\n",
      "--- gene=182, id=4 removed by soft count = 102\n",
      "--- gene=182, id=5 removed by soft count = 253\n",
      "--- gene=182, id=7 removed by soft count = 67\n",
      "--- gene=182, id=11 removed by soft count = 54\n",
      "--- gene=182, id=12 removed by soft count = 192\n",
      "--- gene=182, id=14 removed by soft count = 128\n",
      "--- gene=182, id=15 removed by soft count = 117\n",
      "--- gene=182, id=62 removed by soft count = 513\n",
      "--- gene=182, id=65 removed by soft count = 162\n",
      "--- number of probes: 93 , kept by blast: 81 , if remove dups: 81\n",
      "-- number of probes kept for this region: 81\n",
      "-- checking probes in region: 183\n",
      "--- gene=183, id=201 removed by soft count = 753\n",
      "--- gene=183, id=202 removed by soft count = 587\n",
      "--- gene=183, id=203 removed by soft count = 1064\n",
      "--- number of probes: 204 , kept by blast: 201 , if remove dups: 201\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 184\n",
      "--- gene=184, id=63 removed by soft count = 189\n",
      "--- gene=184, id=64 removed by soft count = 382\n",
      "--- gene=184, id=149 removed by soft count = 291\n",
      "--- gene=184, id=150 removed by soft count = 111\n",
      "--- gene=184, id=154 removed by soft count = 464\n",
      "--- gene=184, id=163 removed by soft count = 295\n",
      "--- number of probes: 173 , kept by blast: 167 , if remove dups: 167\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 185\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 186\n",
      "--- gene=186, id=160 removed by soft count = 73\n",
      "--- gene=186, id=289 removed by soft count = 92\n",
      "--- gene=186, id=298 removed by soft count = 71\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 187\n",
      "--- gene=187, id=132 removed by soft count = 568\n",
      "--- gene=187, id=147 removed by soft count = 88\n",
      "--- gene=187, id=188 removed by soft count = 77\n",
      "--- gene=187, id=218 removed by soft count = 64\n",
      "--- number of probes: 239 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 188\n",
      "--- gene=188, id=58 removed by soft count = 203\n",
      "--- gene=188, id=60 removed by soft count = 184\n",
      "--- gene=188, id=61 removed by soft count = 460\n",
      "--- gene=188, id=92 removed by soft count = 70\n",
      "--- number of probes: 233 , kept by blast: 229 , if remove dups: 229\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 189\n",
      "--- gene=189, id=27 removed by soft count = 143\n",
      "--- gene=189, id=32 removed by soft count = 2028\n",
      "--- gene=189, id=38 removed by soft count = 546\n",
      "--- gene=189, id=39 removed by soft count = 116\n",
      "--- gene=189, id=106 removed by soft count = 311\n",
      "--- gene=189, id=107 removed by soft count = 101\n",
      "--- gene=189, id=110 removed by soft count = 1312\n",
      "--- number of probes: 191 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 190\n",
      "--- gene=190, id=10 removed by soft count = 121\n",
      "--- gene=190, id=28 removed by soft count = 423\n",
      "--- gene=190, id=29 removed by soft count = 113\n",
      "--- gene=190, id=31 removed by soft count = 856\n",
      "--- gene=190, id=38 removed by soft count = 62\n",
      "--- gene=190, id=126 removed by soft count = 65\n",
      "--- gene=190, id=130 removed by soft count = 95\n",
      "--- gene=190, id=132 removed by soft count = 67\n",
      "--- gene=190, id=135 removed by soft count = 83\n",
      "--- gene=190, id=148 removed by soft count = 53\n",
      "--- number of probes: 150 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 191\n",
      "--- gene=191, id=141 removed by hard count = 5\n",
      "--- gene=191, id=146 removed by soft count = 67\n",
      "--- gene=191, id=167 removed by soft count = 180\n",
      "--- gene=191, id=207 removed by soft count = 51\n",
      "--- number of probes: 274 , kept by blast: 270 , if remove dups: 270\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 192\n",
      "--- gene=192, id=0 removed by soft count = 2798\n",
      "--- gene=192, id=1 removed by soft count = 859\n",
      "--- gene=192, id=3 removed by soft count = 1433\n",
      "--- gene=192, id=8 removed by soft count = 264\n",
      "--- gene=192, id=9 removed by soft count = 4654\n",
      "--- gene=192, id=10 removed by soft count = 359\n",
      "--- gene=192, id=11 removed by soft count = 2463\n",
      "--- gene=192, id=12 removed by soft count = 266\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=192, id=13 removed by soft count = 2188\n",
      "--- gene=192, id=110 removed by soft count = 2555\n",
      "--- gene=192, id=112 removed by soft count = 240\n",
      "--- gene=192, id=197 removed by soft count = 64\n",
      "--- number of probes: 270 , kept by blast: 258 , if remove dups: 247\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 193\n",
      "--- gene=193, id=1 removed by soft count = 136\n",
      "--- gene=193, id=2 removed by soft count = 87\n",
      "--- gene=193, id=181 removed by soft count = 333\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 194\n",
      "--- gene=194, id=52 removed by soft count = 51\n",
      "--- gene=194, id=101 removed by soft count = 296\n",
      "--- gene=194, id=181 removed by soft count = 68\n",
      "--- gene=194, id=187 removed by soft count = 74\n",
      "--- number of probes: 297 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 195\n",
      "--- gene=195, id=7 removed by soft count = 99\n",
      "--- gene=195, id=8 removed by soft count = 67\n",
      "--- gene=195, id=9 removed by soft count = 157\n",
      "--- gene=195, id=10 removed by soft count = 152\n",
      "--- gene=195, id=11 removed by soft count = 383\n",
      "--- gene=195, id=12 removed by soft count = 271\n",
      "--- gene=195, id=13 removed by soft count = 191\n",
      "--- gene=195, id=14 removed by soft count = 125\n",
      "--- gene=195, id=17 removed by soft count = 365\n",
      "--- gene=195, id=18 removed by soft count = 752\n",
      "--- gene=195, id=20 removed by soft count = 704\n",
      "--- gene=195, id=95 removed by soft count = 351\n",
      "--- number of probes: 199 , kept by blast: 187 , if remove dups: 187\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 196\n",
      "--- gene=196, id=32 removed by soft count = 59\n",
      "--- gene=196, id=105 removed by soft count = 123\n",
      "--- gene=196, id=156 removed by soft count = 77\n",
      "--- gene=196, id=161 removed by soft count = 254\n",
      "--- gene=196, id=162 removed by soft count = 902\n",
      "--- gene=196, id=188 removed by soft count = 101\n",
      "--- gene=196, id=261 removed by soft count = 274\n",
      "--- gene=196, id=262 removed by soft count = 178\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 197\n",
      "--- gene=197, id=10 removed by soft count = 76\n",
      "--- gene=197, id=14 removed by soft count = 159\n",
      "--- gene=197, id=166 removed by soft count = 55\n",
      "--- gene=197, id=186 removed by soft count = 53\n",
      "--- gene=197, id=197 removed by soft count = 422\n",
      "--- gene=197, id=198 removed by soft count = 212\n",
      "--- number of probes: 251 , kept by blast: 245 , if remove dups: 245\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 198\n",
      "--- gene=198, id=112 removed by soft count = 104\n",
      "--- gene=198, id=115 removed by soft count = 99\n",
      "--- gene=198, id=116 removed by soft count = 134\n",
      "--- gene=198, id=117 removed by soft count = 162\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 199\n",
      "--- gene=199, id=7 removed by soft count = 121\n",
      "--- gene=199, id=33 removed by soft count = 259\n",
      "--- gene=199, id=54 removed by soft count = 56\n",
      "--- gene=199, id=55 removed by soft count = 1149\n",
      "--- gene=199, id=56 removed by soft count = 205\n",
      "--- gene=199, id=105 removed by soft count = 168\n",
      "--- gene=199, id=110 removed by soft count = 60\n",
      "--- gene=199, id=115 removed by soft count = 82\n",
      "--- gene=199, id=116 removed by soft count = 195\n",
      "--- gene=199, id=117 removed by soft count = 69\n",
      "--- gene=199, id=120 removed by soft count = 100\n",
      "--- gene=199, id=143 removed by soft count = 160\n",
      "--- gene=199, id=144 removed by soft count = 58\n",
      "--- gene=199, id=151 removed by soft count = 52\n",
      "--- gene=199, id=154 removed by soft count = 79\n",
      "--- gene=199, id=161 removed by soft count = 51\n",
      "--- gene=199, id=164 removed by soft count = 58\n",
      "--- gene=199, id=167 removed by soft count = 55\n",
      "--- gene=199, id=171 removed by soft count = 282\n",
      "--- gene=199, id=172 removed by soft count = 1048\n",
      "--- gene=199, id=268 removed by soft count = 72\n",
      "--- gene=199, id=269 removed by soft count = 170\n",
      "--- number of probes: 275 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 200\n",
      "--- gene=200, id=0 removed by soft count = 699\n",
      "--- gene=200, id=1 removed by soft count = 1180\n",
      "--- gene=200, id=3 removed by soft count = 64\n",
      "--- gene=200, id=4 removed by soft count = 119\n",
      "--- gene=200, id=5 removed by soft count = 469\n",
      "--- gene=200, id=6 removed by soft count = 52\n",
      "--- gene=200, id=7 removed by soft count = 90\n",
      "--- gene=200, id=8 removed by soft count = 104\n",
      "--- gene=200, id=172 removed by soft count = 308\n",
      "--- number of probes: 229 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 201\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 202\n",
      "--- gene=202, id=113 removed by soft count = 133\n",
      "--- gene=202, id=251 removed by soft count = 55\n",
      "--- gene=202, id=282 removed by soft count = 58\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 203\n",
      "--- gene=203, id=11 removed by soft count = 112\n",
      "--- gene=203, id=81 removed by soft count = 930\n",
      "--- gene=203, id=84 removed by soft count = 129\n",
      "--- gene=203, id=92 removed by soft count = 671\n",
      "--- gene=203, id=160 removed by soft count = 311\n",
      "--- number of probes: 190 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 204\n",
      "--- gene=204, id=129 removed by soft count = 599\n",
      "--- number of probes: 265 , kept by blast: 264 , if remove dups: 264\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 205\n",
      "--- gene=205, id=127 removed by soft count = 71\n",
      "--- gene=205, id=128 removed by soft count = 56\n",
      "--- gene=205, id=134 removed by soft count = 83\n",
      "--- gene=205, id=136 removed by soft count = 733\n",
      "--- gene=205, id=137 removed by soft count = 310\n",
      "--- number of probes: 241 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 206\n",
      "--- gene=206, id=0 removed by soft count = 180\n",
      "--- gene=206, id=1 removed by soft count = 690\n",
      "--- gene=206, id=2 removed by soft count = 168\n",
      "--- gene=206, id=3 removed by soft count = 143\n",
      "--- gene=206, id=4 removed by soft count = 72\n",
      "--- gene=206, id=5 removed by soft count = 60\n",
      "--- gene=206, id=8 removed by soft count = 64\n",
      "--- gene=206, id=144 removed by soft count = 2871\n",
      "--- gene=206, id=156 removed by soft count = 120\n",
      "--- gene=206, id=258 removed by soft count = 147\n",
      "--- gene=206, id=259 removed by soft count = 245\n",
      "--- gene=206, id=260 removed by soft count = 62\n",
      "--- number of probes: 262 , kept by blast: 250 , if remove dups: 250\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 207\n",
      "--- gene=207, id=130 removed by soft count = 66\n",
      "--- gene=207, id=255 removed by soft count = 577\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 208\n",
      "--- gene=208, id=56 removed by soft count = 78\n",
      "--- gene=208, id=58 removed by soft count = 257\n",
      "--- gene=208, id=59 removed by soft count = 450\n",
      "--- gene=208, id=60 removed by soft count = 65\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=208, id=122 removed by soft count = 494\n",
      "--- gene=208, id=126 removed by soft count = 163\n",
      "--- number of probes: 174 , kept by blast: 168 , if remove dups: 168\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 209\n",
      "--- gene=209, id=0 removed by soft count = 230\n",
      "--- gene=209, id=29 removed by soft count = 77\n",
      "--- gene=209, id=30 removed by soft count = 65\n",
      "--- gene=209, id=91 removed by soft count = 83\n",
      "--- gene=209, id=92 removed by soft count = 59\n",
      "--- gene=209, id=94 removed by soft count = 52\n",
      "--- gene=209, id=95 removed by soft count = 84\n",
      "--- gene=209, id=97 removed by soft count = 84\n",
      "--- gene=209, id=101 removed by soft count = 253\n",
      "--- gene=209, id=102 removed by soft count = 325\n",
      "--- gene=209, id=103 removed by soft count = 158\n",
      "--- gene=209, id=104 removed by soft count = 353\n",
      "--- gene=209, id=105 removed by soft count = 410\n",
      "--- gene=209, id=107 removed by soft count = 167\n",
      "--- gene=209, id=109 removed by soft count = 1218\n",
      "--- gene=209, id=167 removed by soft count = 457\n",
      "--- gene=209, id=168 removed by soft count = 205\n",
      "--- number of probes: 180 , kept by blast: 163 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 210\n",
      "--- gene=210, id=299 removed by soft count = 92\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 211\n",
      "--- gene=211, id=202 removed by soft count = 75\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 212\n",
      "--- gene=212, id=174 removed by soft count = 140\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 213\n",
      "--- gene=213, id=26 removed by soft count = 151\n",
      "--- gene=213, id=81 removed by soft count = 283\n",
      "--- gene=213, id=84 removed by soft count = 579\n",
      "--- gene=213, id=103 removed by soft count = 865\n",
      "--- gene=213, id=104 removed by soft count = 694\n",
      "--- gene=213, id=112 removed by soft count = 56\n",
      "--- gene=213, id=141 removed by soft count = 191\n",
      "--- gene=213, id=142 removed by soft count = 75\n",
      "--- gene=213, id=146 removed by soft count = 210\n",
      "--- number of probes: 200 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 214\n",
      "--- gene=214, id=0 removed by soft count = 85\n",
      "--- gene=214, id=1 removed by soft count = 155\n",
      "--- gene=214, id=2 removed by soft count = 238\n",
      "--- gene=214, id=4 removed by soft count = 146\n",
      "--- gene=214, id=201 removed by soft count = 100\n",
      "--- number of probes: 219 , kept by blast: 214 , if remove dups: 214\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 215\n",
      "--- gene=215, id=31 removed by soft count = 205\n",
      "--- gene=215, id=33 removed by soft count = 67\n",
      "--- gene=215, id=34 removed by soft count = 2263\n",
      "--- gene=215, id=35 removed by soft count = 128\n",
      "--- gene=215, id=48 removed by soft count = 151\n",
      "--- gene=215, id=49 removed by soft count = 140\n",
      "--- gene=215, id=53 removed by soft count = 162\n",
      "--- gene=215, id=89 removed by soft count = 391\n",
      "--- gene=215, id=140 removed by soft count = 920\n",
      "--- gene=215, id=141 removed by soft count = 79\n",
      "--- number of probes: 142 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 216\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 217\n",
      "--- gene=217, id=63 removed by soft count = 159\n",
      "--- gene=217, id=82 removed by soft count = 935\n",
      "--- gene=217, id=84 removed by soft count = 156\n",
      "--- gene=217, id=169 removed by soft count = 64\n",
      "--- number of probes: 278 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 218\n",
      "--- gene=218, id=229 removed by soft count = 72\n",
      "--- number of probes: 252 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 219\n",
      "--- gene=219, id=83 removed by soft count = 286\n",
      "--- gene=219, id=84 removed by soft count = 336\n",
      "--- gene=219, id=85 removed by soft count = 467\n",
      "--- gene=219, id=112 removed by soft count = 259\n",
      "--- gene=219, id=245 removed by soft count = 11800\n",
      "--- gene=219, id=246 removed by soft count = 825\n",
      "--- number of probes: 258 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 220\n",
      "--- number of probes: 214 , kept by blast: 214 , if remove dups: 214\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 221\n",
      "--- gene=221, id=24 removed by soft count = 72\n",
      "--- gene=221, id=117 removed by soft count = 110\n",
      "--- gene=221, id=165 removed by soft count = 216\n",
      "--- gene=221, id=176 removed by soft count = 147\n",
      "--- gene=221, id=223 removed by soft count = 111\n",
      "--- number of probes: 256 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 222\n",
      "--- gene=222, id=62 removed by soft count = 85\n",
      "--- gene=222, id=66 removed by soft count = 76\n",
      "--- gene=222, id=71 removed by soft count = 57\n",
      "--- gene=222, id=102 removed by soft count = 1693\n",
      "--- gene=222, id=183 removed by soft count = 56\n",
      "--- gene=222, id=230 removed by soft count = 122\n",
      "--- gene=222, id=281 removed by soft count = 287\n",
      "--- number of probes: 292 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 223\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 224\n",
      "--- gene=224, id=69 removed by soft count = 113\n",
      "--- gene=224, id=149 removed by soft count = 72\n",
      "--- gene=224, id=247 removed by soft count = 456\n",
      "--- gene=224, id=265 removed by soft count = 264\n",
      "--- gene=224, id=279 removed by soft count = 144\n",
      "--- gene=224, id=281 removed by soft count = 120\n",
      "--- gene=224, id=286 removed by soft count = 76\n",
      "--- number of probes: 300 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 225\n",
      "--- gene=225, id=4 removed by soft count = 62\n",
      "--- gene=225, id=25 removed by soft count = 54\n",
      "--- gene=225, id=26 removed by soft count = 103\n",
      "--- gene=225, id=64 removed by soft count = 1379\n",
      "--- gene=225, id=176 removed by soft count = 352\n",
      "--- number of probes: 233 , kept by blast: 228 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 226\n",
      "--- gene=226, id=186 removed by soft count = 393\n",
      "--- gene=226, id=190 removed by soft count = 488\n",
      "--- number of probes: 220 , kept by blast: 218 , if remove dups: 218\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 227\n",
      "--- gene=227, id=126 removed by soft count = 171\n",
      "--- gene=227, id=128 removed by soft count = 266\n",
      "--- gene=227, id=129 removed by soft count = 160\n",
      "--- gene=227, id=148 removed by soft count = 166\n",
      "--- number of probes: 242 , kept by blast: 238 , if remove dups: 238\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 228\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=228, id=42 removed by soft count = 123\n",
      "--- gene=228, id=67 removed by soft count = 469\n",
      "--- gene=228, id=97 removed by soft count = 224\n",
      "--- gene=228, id=108 removed by soft count = 66\n",
      "--- gene=228, id=109 removed by soft count = 376\n",
      "--- gene=228, id=110 removed by soft count = 346\n",
      "--- gene=228, id=111 removed by soft count = 64\n",
      "--- gene=228, id=112 removed by soft count = 353\n",
      "--- number of probes: 148 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 229\n",
      "--- gene=229, id=40 removed by soft count = 93\n",
      "--- gene=229, id=186 removed by soft count = 76\n",
      "--- number of probes: 197 , kept by blast: 195 , if remove dups: 195\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 230\n",
      "--- gene=230, id=0 removed by soft count = 789\n",
      "--- gene=230, id=1 removed by soft count = 186\n",
      "--- gene=230, id=92 removed by soft count = 1267\n",
      "--- gene=230, id=93 removed by soft count = 122\n",
      "--- gene=230, id=94 removed by soft count = 3630\n",
      "--- gene=230, id=152 removed by soft count = 68\n",
      "--- gene=230, id=157 removed by soft count = 2464\n",
      "--- gene=230, id=162 removed by soft count = 168\n",
      "--- gene=230, id=165 removed by soft count = 3321\n",
      "--- number of probes: 166 , kept by blast: 157 , if remove dups: 157\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 233\n",
      "--- gene=233, id=58 removed by soft count = 216\n",
      "--- gene=233, id=114 removed by soft count = 648\n",
      "--- gene=233, id=157 removed by soft count = 207\n",
      "--- gene=233, id=186 removed by soft count = 95\n",
      "--- gene=233, id=194 removed by soft count = 207\n",
      "--- number of probes: 196 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 234\n",
      "--- gene=234, id=35 removed by soft count = 409\n",
      "--- gene=234, id=79 removed by soft count = 278\n",
      "--- gene=234, id=152 removed by soft count = 215\n",
      "--- gene=234, id=153 removed by soft count = 140\n",
      "--- gene=234, id=154 removed by soft count = 188\n",
      "--- gene=234, id=155 removed by soft count = 324\n",
      "--- gene=234, id=156 removed by soft count = 471\n",
      "--- gene=234, id=158 removed by soft count = 69\n",
      "--- gene=234, id=160 removed by soft count = 141\n",
      "--- number of probes: 255 , kept by blast: 246 , if remove dups: 246\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 235\n",
      "--- gene=235, id=14 removed by soft count = 914\n",
      "--- gene=235, id=15 removed by soft count = 438\n",
      "--- gene=235, id=16 removed by soft count = 634\n",
      "--- gene=235, id=55 removed by soft count = 1526\n",
      "--- gene=235, id=91 removed by soft count = 381\n",
      "--- number of probes: 233 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 236\n",
      "--- gene=236, id=31 removed by soft count = 95\n",
      "--- gene=236, id=59 removed by soft count = 141\n",
      "--- gene=236, id=69 removed by soft count = 100\n",
      "--- gene=236, id=70 removed by soft count = 237\n",
      "--- gene=236, id=75 removed by soft count = 263\n",
      "--- gene=236, id=79 removed by soft count = 97\n",
      "--- gene=236, id=96 removed by soft count = 179\n",
      "--- gene=236, id=97 removed by soft count = 245\n",
      "--- gene=236, id=133 removed by soft count = 57\n",
      "--- gene=236, id=134 removed by soft count = 454\n",
      "--- gene=236, id=142 removed by soft count = 189\n",
      "--- gene=236, id=147 removed by soft count = 14074\n",
      "--- gene=236, id=148 removed by soft count = 764\n",
      "--- number of probes: 182 , kept by blast: 169 , if remove dups: 169\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 237\n",
      "--- gene=237, id=13 removed by soft count = 89\n",
      "--- gene=237, id=16 removed by soft count = 229\n",
      "--- gene=237, id=123 removed by soft count = 64\n",
      "--- gene=237, id=130 removed by soft count = 108\n",
      "--- gene=237, id=131 removed by soft count = 69\n",
      "--- gene=237, id=132 removed by soft count = 58\n",
      "--- gene=237, id=135 removed by soft count = 208\n",
      "--- gene=237, id=137 removed by soft count = 58\n",
      "--- gene=237, id=146 removed by soft count = 82\n",
      "--- gene=237, id=147 removed by soft count = 58\n",
      "--- gene=237, id=148 removed by soft count = 69\n",
      "--- gene=237, id=150 removed by soft count = 183\n",
      "--- gene=237, id=151 removed by soft count = 200\n",
      "--- gene=237, id=155 removed by soft count = 155\n",
      "--- gene=237, id=156 removed by soft count = 365\n",
      "--- number of probes: 179 , kept by blast: 164 , if remove dups: 164\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 238\n",
      "--- gene=238, id=22 removed by soft count = 98\n",
      "--- gene=238, id=27 removed by soft count = 181\n",
      "--- gene=238, id=28 removed by soft count = 57\n",
      "--- gene=238, id=249 removed by soft count = 748\n",
      "--- gene=238, id=250 removed by soft count = 75\n",
      "--- number of probes: 251 , kept by blast: 246 , if remove dups: 246\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 239\n",
      "--- gene=239, id=3 removed by soft count = 60\n",
      "--- gene=239, id=4 removed by soft count = 55\n",
      "--- gene=239, id=5 removed by soft count = 105\n",
      "--- gene=239, id=37 removed by soft count = 580\n",
      "--- gene=239, id=47 removed by soft count = 74\n",
      "--- gene=239, id=50 removed by soft count = 57\n",
      "--- gene=239, id=148 removed by soft count = 74\n",
      "--- gene=239, id=192 removed by soft count = 57\n",
      "--- gene=239, id=197 removed by soft count = 55\n",
      "--- gene=239, id=198 removed by soft count = 81\n",
      "--- gene=239, id=217 removed by soft count = 114\n",
      "--- number of probes: 231 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 240\n",
      "--- gene=240, id=204 removed by soft count = 52\n",
      "--- number of probes: 234 , kept by blast: 233 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 241\n",
      "--- gene=241, id=269 removed by soft count = 20024\n",
      "--- number of probes: 272 , kept by blast: 271 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 242\n",
      "--- gene=242, id=13 removed by soft count = 114\n",
      "--- gene=242, id=15 removed by soft count = 380\n",
      "--- gene=242, id=16 removed by soft count = 79\n",
      "--- gene=242, id=65 removed by soft count = 106\n",
      "--- gene=242, id=168 removed by soft count = 71\n",
      "--- gene=242, id=287 removed by soft count = 272\n",
      "--- number of probes: 288 , kept by blast: 282 , if remove dups: 282\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 243\n",
      "--- gene=243, id=7 removed by soft count = 55\n",
      "--- gene=243, id=11 removed by soft count = 60\n",
      "--- gene=243, id=22 removed by soft count = 108\n",
      "--- gene=243, id=23 removed by soft count = 280\n",
      "--- gene=243, id=42 removed by soft count = 56\n",
      "--- gene=243, id=56 removed by soft count = 77\n",
      "--- gene=243, id=57 removed by soft count = 100\n",
      "--- gene=243, id=58 removed by soft count = 88\n",
      "--- gene=243, id=67 removed by soft count = 61\n",
      "--- gene=243, id=71 removed by soft count = 79\n",
      "--- gene=243, id=76 removed by soft count = 272\n",
      "--- gene=243, id=78 removed by soft count = 2167\n",
      "--- gene=243, id=79 removed by soft count = 86\n",
      "--- gene=243, id=84 removed by soft count = 83\n",
      "--- number of probes: 165 , kept by blast: 151 , if remove dups: 151\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 244\n",
      "--- gene=244, id=274 removed by soft count = 56\n",
      "--- gene=244, id=275 removed by soft count = 118\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 245\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=245, id=11 removed by soft count = 77\n",
      "--- gene=245, id=13 removed by soft count = 753\n",
      "--- gene=245, id=14 removed by soft count = 231\n",
      "--- gene=245, id=33 removed by soft count = 137\n",
      "--- gene=245, id=39 removed by soft count = 105\n",
      "--- gene=245, id=41 removed by soft count = 73\n",
      "--- gene=245, id=48 removed by soft count = 66\n",
      "--- gene=245, id=49 removed by soft count = 61\n",
      "--- gene=245, id=69 removed by soft count = 140\n",
      "--- gene=245, id=70 removed by soft count = 232\n",
      "--- gene=245, id=73 removed by soft count = 266\n",
      "--- gene=245, id=74 removed by soft count = 80\n",
      "--- gene=245, id=90 removed by soft count = 375\n",
      "--- gene=245, id=101 removed by soft count = 221\n",
      "--- gene=245, id=109 removed by soft count = 103\n",
      "--- gene=245, id=114 removed by soft count = 214\n",
      "--- gene=245, id=115 removed by soft count = 155\n",
      "--- gene=245, id=120 removed by soft count = 165\n",
      "--- gene=245, id=122 removed by soft count = 66\n",
      "--- gene=245, id=124 removed by soft count = 99\n",
      "--- gene=245, id=125 removed by soft count = 51\n",
      "--- number of probes: 238 , kept by blast: 217 , if remove dups: 217\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 246\n",
      "--- gene=246, id=48 removed by soft count = 567\n",
      "--- gene=246, id=49 removed by soft count = 105\n",
      "--- gene=246, id=53 removed by soft count = 312\n",
      "--- gene=246, id=99 removed by soft count = 437\n",
      "--- gene=246, id=129 removed by soft count = 88\n",
      "--- number of probes: 205 , kept by blast: 200 , if remove dups: 200\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 247\n",
      "--- gene=247, id=99 removed by soft count = 192\n",
      "--- gene=247, id=101 removed by soft count = 225\n",
      "--- gene=247, id=102 removed by soft count = 117\n",
      "--- gene=247, id=145 removed by soft count = 76\n",
      "--- gene=247, id=189 removed by soft count = 647\n",
      "--- gene=247, id=217 removed by soft count = 253\n",
      "--- gene=247, id=233 removed by soft count = 87\n",
      "--- number of probes: 234 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 248\n",
      "--- gene=248, id=0 removed by soft count = 173\n",
      "--- gene=248, id=1 removed by soft count = 1871\n",
      "--- number of probes: 277 , kept by blast: 275 , if remove dups: 275\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 249\n",
      "--- gene=249, id=9 removed by soft count = 98\n",
      "--- gene=249, id=115 removed by soft count = 59\n",
      "--- number of probes: 242 , kept by blast: 240 , if remove dups: 240\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 250\n",
      "--- gene=250, id=29 removed by soft count = 67\n",
      "--- gene=250, id=55 removed by soft count = 68\n",
      "--- gene=250, id=56 removed by soft count = 80\n",
      "--- gene=250, id=58 removed by soft count = 142\n",
      "--- gene=250, id=60 removed by soft count = 151\n",
      "--- gene=250, id=61 removed by soft count = 149\n",
      "--- gene=250, id=63 removed by soft count = 65\n",
      "--- gene=250, id=65 removed by soft count = 142\n",
      "--- gene=250, id=86 removed by soft count = 171\n",
      "--- number of probes: 172 , kept by blast: 163 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 251\n",
      "--- gene=251, id=50 removed by soft count = 547\n",
      "--- gene=251, id=163 removed by soft count = 64\n",
      "--- gene=251, id=164 removed by soft count = 61\n",
      "--- gene=251, id=299 removed by soft count = 91\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 252\n",
      "--- gene=252, id=34 removed by soft count = 57\n",
      "--- gene=252, id=63 removed by soft count = 217\n",
      "--- gene=252, id=65 removed by soft count = 91\n",
      "--- gene=252, id=74 removed by soft count = 52\n",
      "--- number of probes: 287 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 253\n",
      "--- gene=253, id=1 removed by soft count = 90\n",
      "--- gene=253, id=2 removed by soft count = 146\n",
      "--- gene=253, id=3 removed by soft count = 306\n",
      "--- gene=253, id=4 removed by soft count = 265\n",
      "--- gene=253, id=15 removed by soft count = 106\n",
      "--- gene=253, id=16 removed by soft count = 235\n",
      "--- gene=253, id=20 removed by soft count = 344\n",
      "--- gene=253, id=28 removed by soft count = 110\n",
      "--- gene=253, id=49 removed by soft count = 70\n",
      "--- gene=253, id=51 removed by soft count = 206\n",
      "--- gene=253, id=68 removed by soft count = 129\n",
      "--- gene=253, id=105 removed by soft count = 735\n",
      "--- gene=253, id=131 removed by soft count = 919\n",
      "--- gene=253, id=132 removed by soft count = 84\n",
      "--- gene=253, id=133 removed by soft count = 208\n",
      "--- gene=253, id=135 removed by soft count = 61\n",
      "--- gene=253, id=139 removed by soft count = 150\n",
      "--- gene=253, id=145 removed by soft count = 343\n",
      "--- number of probes: 188 , kept by blast: 170 , if remove dups: 170\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 254\n",
      "--- number of probes: 282 , kept by blast: 282 , if remove dups: 282\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 255\n",
      "--- gene=255, id=75 removed by soft count = 119\n",
      "--- gene=255, id=94 removed by soft count = 285\n",
      "--- number of probes: 225 , kept by blast: 223 , if remove dups: 223\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 256\n",
      "--- gene=256, id=5 removed by soft count = 68\n",
      "--- gene=256, id=175 removed by soft count = 136\n",
      "--- number of probes: 218 , kept by blast: 216 , if remove dups: 216\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 257\n",
      "--- gene=257, id=278 removed by soft count = 94\n",
      "--- gene=257, id=279 removed by soft count = 74\n",
      "--- number of probes: 288 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 258\n",
      "--- gene=258, id=119 removed by soft count = 10444\n",
      "--- gene=258, id=123 removed by soft count = 96\n",
      "--- gene=258, id=190 removed by soft count = 184\n",
      "--- gene=258, id=200 removed by soft count = 102\n",
      "--- gene=258, id=207 removed by soft count = 197\n",
      "--- gene=258, id=211 removed by soft count = 208\n",
      "--- gene=258, id=212 removed by soft count = 427\n",
      "--- number of probes: 300 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 259\n",
      "--- gene=259, id=51 removed by soft count = 155\n",
      "--- gene=259, id=114 removed by soft count = 349\n",
      "--- gene=259, id=126 removed by soft count = 76\n",
      "--- gene=259, id=129 removed by soft count = 147\n",
      "--- gene=259, id=135 removed by soft count = 106\n",
      "--- gene=259, id=139 removed by soft count = 68\n",
      "--- gene=259, id=141 removed by soft count = 771\n",
      "--- number of probes: 160 , kept by blast: 153 , if remove dups: 153\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 260\n",
      "--- gene=260, id=40 removed by soft count = 194\n",
      "--- gene=260, id=41 removed by soft count = 1671\n",
      "--- gene=260, id=45 removed by soft count = 434\n",
      "--- gene=260, id=46 removed by soft count = 260\n",
      "--- gene=260, id=56 removed by soft count = 264\n",
      "--- gene=260, id=59 removed by soft count = 198\n",
      "--- gene=260, id=125 removed by soft count = 4490\n",
      "--- gene=260, id=126 removed by soft count = 806\n",
      "--- gene=260, id=134 removed by soft count = 86\n",
      "--- gene=260, id=138 removed by soft count = 190\n",
      "--- gene=260, id=142 removed by soft count = 51\n",
      "--- gene=260, id=144 removed by soft count = 58\n",
      "--- gene=260, id=145 removed by soft count = 1327\n",
      "--- number of probes: 162 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 261\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=261, id=74 removed by soft count = 851\n",
      "--- gene=261, id=77 removed by soft count = 63\n",
      "--- gene=261, id=78 removed by soft count = 325\n",
      "--- gene=261, id=106 removed by soft count = 271\n",
      "--- gene=261, id=107 removed by soft count = 477\n",
      "--- gene=261, id=118 removed by soft count = 288\n",
      "--- gene=261, id=119 removed by soft count = 359\n",
      "--- gene=261, id=131 removed by soft count = 58\n",
      "--- gene=261, id=142 removed by soft count = 115\n",
      "--- number of probes: 159 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 262\n",
      "--- gene=262, id=79 removed by soft count = 985\n",
      "--- gene=262, id=80 removed by soft count = 94\n",
      "--- gene=262, id=150 removed by soft count = 207\n",
      "--- gene=262, id=151 removed by soft count = 80\n",
      "--- gene=262, id=154 removed by soft count = 71\n",
      "--- gene=262, id=156 removed by soft count = 138\n",
      "--- number of probes: 179 , kept by blast: 173 , if remove dups: 172\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 263\n",
      "--- gene=263, id=87 removed by soft count = 193\n",
      "--- gene=263, id=240 removed by soft count = 51\n",
      "--- gene=263, id=242 removed by soft count = 297\n",
      "--- number of probes: 299 , kept by blast: 296 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 264\n",
      "--- gene=264, id=25 removed by soft count = 832\n",
      "--- gene=264, id=27 removed by soft count = 180\n",
      "--- gene=264, id=194 removed by soft count = 90\n",
      "--- number of probes: 259 , kept by blast: 256 , if remove dups: 256\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 265\n",
      "--- gene=265, id=58 removed by soft count = 185\n",
      "--- gene=265, id=64 removed by soft count = 95\n",
      "--- gene=265, id=189 removed by soft count = 129\n",
      "--- gene=265, id=200 removed by soft count = 84\n",
      "--- gene=265, id=202 removed by soft count = 57\n",
      "--- gene=265, id=209 removed by soft count = 73\n",
      "--- gene=265, id=211 removed by soft count = 146\n",
      "--- gene=265, id=213 removed by soft count = 240\n",
      "--- gene=265, id=214 removed by soft count = 447\n",
      "--- gene=265, id=216 removed by soft count = 249\n",
      "--- gene=265, id=219 removed by soft count = 109\n",
      "--- number of probes: 220 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 266\n",
      "--- gene=266, id=118 removed by soft count = 275\n",
      "--- gene=266, id=119 removed by soft count = 153\n",
      "--- gene=266, id=120 removed by soft count = 834\n",
      "--- gene=266, id=121 removed by soft count = 2334\n",
      "--- number of probes: 251 , kept by blast: 247 , if remove dups: 247\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 267\n",
      "--- gene=267, id=101 removed by soft count = 303\n",
      "--- gene=267, id=202 removed by soft count = 632\n",
      "--- number of probes: 263 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 268\n",
      "--- gene=268, id=24 removed by soft count = 61\n",
      "--- gene=268, id=26 removed by soft count = 498\n",
      "--- gene=268, id=27 removed by soft count = 418\n",
      "--- gene=268, id=28 removed by soft count = 376\n",
      "--- gene=268, id=30 removed by soft count = 3067\n",
      "--- gene=268, id=31 removed by soft count = 78\n",
      "--- gene=268, id=32 removed by soft count = 809\n",
      "--- gene=268, id=70 removed by soft count = 40011\n",
      "--- gene=268, id=71 removed by soft count = 53\n",
      "--- gene=268, id=130 removed by soft count = 3300\n",
      "--- gene=268, id=167 removed by soft count = 477\n",
      "--- number of probes: 175 , kept by blast: 164 , if remove dups: 164\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 269\n",
      "--- gene=269, id=155 removed by soft count = 111\n",
      "--- gene=269, id=277 removed by soft count = 72\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 270\n",
      "--- gene=270, id=67 removed by soft count = 166\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 271\n",
      "--- gene=271, id=16 removed by soft count = 141\n",
      "--- gene=271, id=18 removed by soft count = 67\n",
      "--- gene=271, id=24 removed by soft count = 53\n",
      "--- gene=271, id=26 removed by soft count = 109\n",
      "--- gene=271, id=42 removed by soft count = 95\n",
      "--- gene=271, id=43 removed by soft count = 70\n",
      "--- gene=271, id=68 removed by soft count = 108\n",
      "--- gene=271, id=71 removed by soft count = 101\n",
      "--- gene=271, id=105 removed by soft count = 151\n",
      "--- gene=271, id=144 removed by soft count = 52\n",
      "--- gene=271, id=147 removed by soft count = 52\n",
      "--- gene=271, id=148 removed by soft count = 166\n",
      "--- gene=271, id=155 removed by soft count = 269\n",
      "--- gene=271, id=160 removed by soft count = 459\n",
      "--- gene=271, id=161 removed by soft count = 81\n",
      "--- gene=271, id=193 removed by soft count = 65\n",
      "--- number of probes: 220 , kept by blast: 204 , if remove dups: 204\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 272\n",
      "--- gene=272, id=46 removed by soft count = 257\n",
      "--- gene=272, id=53 removed by soft count = 195\n",
      "--- gene=272, id=111 removed by soft count = 235\n",
      "--- gene=272, id=112 removed by soft count = 95\n",
      "--- gene=272, id=117 removed by soft count = 57\n",
      "--- gene=272, id=136 removed by soft count = 788\n",
      "--- gene=272, id=137 removed by soft count = 567\n",
      "--- gene=272, id=138 removed by soft count = 86\n",
      "--- gene=272, id=208 removed by soft count = 73\n",
      "--- gene=272, id=209 removed by soft count = 88\n",
      "--- number of probes: 232 , kept by blast: 222 , if remove dups: 221\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 273\n",
      "--- gene=273, id=109 removed by hard count = 4\n",
      "--- gene=273, id=136 removed by soft count = 51\n",
      "--- gene=273, id=137 removed by soft count = 59\n",
      "--- gene=273, id=186 removed by soft count = 138\n",
      "--- gene=273, id=188 removed by soft count = 196\n",
      "--- number of probes: 211 , kept by blast: 206 , if remove dups: 206\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 274\n",
      "--- gene=274, id=28 removed by soft count = 51\n",
      "--- gene=274, id=191 removed by soft count = 151\n",
      "--- gene=274, id=192 removed by soft count = 147\n",
      "--- gene=274, id=193 removed by soft count = 247\n",
      "--- gene=274, id=199 removed by soft count = 132\n",
      "--- gene=274, id=241 removed by soft count = 119\n",
      "--- gene=274, id=278 removed by soft count = 63\n",
      "--- gene=274, id=281 removed by soft count = 226\n",
      "--- number of probes: 282 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 275\n",
      "--- gene=275, id=169 removed by soft count = 57\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 276\n",
      "--- gene=276, id=20 removed by soft count = 1382\n",
      "--- gene=276, id=25 removed by soft count = 483\n",
      "--- gene=276, id=26 removed by soft count = 556\n",
      "--- gene=276, id=39 removed by soft count = 1074\n",
      "--- gene=276, id=108 removed by soft count = 454\n",
      "--- gene=276, id=109 removed by soft count = 69\n",
      "--- gene=276, id=110 removed by soft count = 104\n",
      "--- gene=276, id=111 removed by soft count = 187\n",
      "--- gene=276, id=112 removed by soft count = 115\n",
      "--- gene=276, id=212 removed by soft count = 6667\n",
      "--- gene=276, id=243 removed by soft count = 60\n",
      "--- number of probes: 300 , kept by blast: 289 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 277\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=277, id=90 removed by soft count = 460\n",
      "--- gene=277, id=91 removed by soft count = 51\n",
      "--- gene=277, id=192 removed by soft count = 62\n",
      "--- gene=277, id=203 removed by soft count = 125\n",
      "--- gene=277, id=206 removed by soft count = 68\n",
      "--- gene=277, id=208 removed by soft count = 125\n",
      "--- gene=277, id=217 removed by soft count = 157\n",
      "--- gene=277, id=219 removed by soft count = 539\n",
      "--- gene=277, id=220 removed by soft count = 612\n",
      "--- number of probes: 300 , kept by blast: 291 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 278\n",
      "--- gene=278, id=18 removed by soft count = 57\n",
      "--- gene=278, id=20 removed by soft count = 54\n",
      "--- gene=278, id=22 removed by soft count = 82\n",
      "--- gene=278, id=45 removed by soft count = 53\n",
      "--- gene=278, id=128 removed by soft count = 154\n",
      "--- gene=278, id=164 removed by soft count = 715\n",
      "--- gene=278, id=165 removed by soft count = 132\n",
      "--- gene=278, id=166 removed by soft count = 81\n",
      "--- gene=278, id=206 removed by soft count = 94\n",
      "--- number of probes: 224 , kept by blast: 215 , if remove dups: 215\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 279\n",
      "--- gene=279, id=44 removed by soft count = 641\n",
      "--- gene=279, id=47 removed by soft count = 333\n",
      "--- gene=279, id=49 removed by soft count = 163\n",
      "--- gene=279, id=50 removed by soft count = 336\n",
      "--- gene=279, id=147 removed by soft count = 296\n",
      "--- gene=279, id=149 removed by soft count = 118\n",
      "--- gene=279, id=150 removed by soft count = 298\n",
      "--- gene=279, id=151 removed by soft count = 528\n",
      "--- gene=279, id=152 removed by soft count = 191\n",
      "--- gene=279, id=153 removed by soft count = 177\n",
      "--- gene=279, id=155 removed by soft count = 407\n",
      "--- number of probes: 210 , kept by blast: 199 , if remove dups: 199\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 280\n",
      "--- gene=280, id=98 removed by soft count = 108\n",
      "--- gene=280, id=185 removed by soft count = 160\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 281\n",
      "--- gene=281, id=23 removed by soft count = 98\n",
      "--- gene=281, id=25 removed by soft count = 125\n",
      "--- gene=281, id=37 removed by soft count = 110\n",
      "--- gene=281, id=38 removed by soft count = 317\n",
      "--- gene=281, id=126 removed by soft count = 73\n",
      "--- gene=281, id=127 removed by soft count = 59\n",
      "--- gene=281, id=152 removed by soft count = 79\n",
      "--- gene=281, id=183 removed by soft count = 52\n",
      "--- gene=281, id=194 removed by soft count = 1011\n",
      "--- number of probes: 239 , kept by blast: 230 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 282\n",
      "--- gene=282, id=36 removed by soft count = 147\n",
      "--- gene=282, id=39 removed by soft count = 52\n",
      "--- number of probes: 262 , kept by blast: 260 , if remove dups: 260\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 283\n",
      "--- gene=283, id=41 removed by soft count = 288\n",
      "--- gene=283, id=271 removed by soft count = 506\n",
      "--- gene=283, id=272 removed by soft count = 289\n",
      "--- gene=283, id=273 removed by soft count = 553\n",
      "--- gene=283, id=274 removed by soft count = 452\n",
      "--- gene=283, id=275 removed by soft count = 2699\n",
      "--- number of probes: 293 , kept by blast: 287 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 284\n",
      "--- gene=284, id=20 removed by soft count = 61\n",
      "--- gene=284, id=84 removed by soft count = 76\n",
      "--- gene=284, id=92 removed by soft count = 102\n",
      "--- gene=284, id=93 removed by soft count = 273\n",
      "--- gene=284, id=165 removed by soft count = 112\n",
      "--- gene=284, id=167 removed by soft count = 160\n",
      "--- gene=284, id=170 removed by soft count = 295\n",
      "--- number of probes: 215 , kept by blast: 208 , if remove dups: 208\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 285\n",
      "--- gene=285, id=49 removed by soft count = 133\n",
      "--- gene=285, id=198 removed by soft count = 71\n",
      "--- gene=285, id=202 removed by soft count = 131\n",
      "--- gene=285, id=224 removed by soft count = 72\n",
      "--- gene=285, id=228 removed by soft count = 103\n",
      "--- number of probes: 234 , kept by blast: 229 , if remove dups: 229\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 286\n",
      "--- gene=286, id=30 removed by soft count = 109\n",
      "--- gene=286, id=79 removed by soft count = 55\n",
      "--- gene=286, id=90 removed by soft count = 158\n",
      "--- gene=286, id=92 removed by soft count = 108\n",
      "--- gene=286, id=139 removed by soft count = 2463\n",
      "--- gene=286, id=140 removed by soft count = 388\n",
      "--- gene=286, id=144 removed by soft count = 1968\n",
      "--- number of probes: 187 , kept by blast: 180 , if remove dups: 180\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 287\n",
      "--- gene=287, id=0 removed by soft count = 266\n",
      "--- gene=287, id=1 removed by soft count = 75\n",
      "--- gene=287, id=2 removed by soft count = 342\n",
      "--- gene=287, id=3 removed by soft count = 262\n",
      "--- gene=287, id=25 removed by soft count = 127\n",
      "--- gene=287, id=26 removed by soft count = 171\n",
      "--- gene=287, id=30 removed by soft count = 88\n",
      "--- gene=287, id=32 removed by soft count = 59\n",
      "--- gene=287, id=36 removed by soft count = 116\n",
      "--- gene=287, id=37 removed by soft count = 165\n",
      "--- gene=287, id=38 removed by soft count = 215\n",
      "--- gene=287, id=39 removed by soft count = 130\n",
      "--- gene=287, id=52 removed by soft count = 90\n",
      "--- gene=287, id=71 removed by soft count = 341\n",
      "--- gene=287, id=166 removed by soft count = 79\n",
      "--- gene=287, id=184 removed by soft count = 324\n",
      "--- gene=287, id=185 removed by soft count = 248\n",
      "--- gene=287, id=186 removed by soft count = 111\n",
      "--- number of probes: 194 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 288\n",
      "--- gene=288, id=98 removed by soft count = 3639\n",
      "--- gene=288, id=138 removed by soft count = 57\n",
      "--- gene=288, id=139 removed by soft count = 254\n",
      "--- gene=288, id=141 removed by soft count = 78\n",
      "--- gene=288, id=142 removed by soft count = 51\n",
      "--- number of probes: 249 , kept by blast: 244 , if remove dups: 244\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 289\n",
      "--- gene=289, id=90 removed by soft count = 52\n",
      "--- gene=289, id=91 removed by soft count = 71\n",
      "--- gene=289, id=93 removed by soft count = 98\n",
      "--- gene=289, id=94 removed by soft count = 683\n",
      "--- gene=289, id=123 removed by soft count = 133\n",
      "--- gene=289, id=125 removed by soft count = 71\n",
      "--- gene=289, id=126 removed by soft count = 161\n",
      "--- gene=289, id=206 removed by soft count = 568\n",
      "--- number of probes: 250 , kept by blast: 242 , if remove dups: 242\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 290\n",
      "--- gene=290, id=31 removed by soft count = 77\n",
      "--- gene=290, id=87 removed by soft count = 60\n",
      "--- gene=290, id=88 removed by soft count = 271\n",
      "--- gene=290, id=89 removed by soft count = 84\n",
      "--- gene=290, id=91 removed by soft count = 493\n",
      "--- gene=290, id=109 removed by soft count = 380\n",
      "--- gene=290, id=110 removed by soft count = 147\n",
      "--- gene=290, id=111 removed by soft count = 66\n",
      "--- number of probes: 253 , kept by blast: 245 , if remove dups: 245\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 291\n",
      "--- gene=291, id=0 removed by soft count = 111\n",
      "--- gene=291, id=32 removed by soft count = 51\n",
      "--- gene=291, id=47 removed by soft count = 64\n",
      "--- gene=291, id=48 removed by soft count = 71\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=291, id=99 removed by soft count = 139\n",
      "--- gene=291, id=100 removed by soft count = 88\n",
      "--- gene=291, id=113 removed by soft count = 81\n",
      "--- number of probes: 195 , kept by blast: 188 , if remove dups: 188\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 292\n",
      "--- gene=292, id=222 removed by soft count = 124\n",
      "--- number of probes: 226 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 293\n",
      "--- gene=293, id=5 removed by soft count = 120\n",
      "--- gene=293, id=9 removed by soft count = 383\n",
      "--- gene=293, id=13 removed by soft count = 57\n",
      "--- gene=293, id=25 removed by soft count = 611\n",
      "--- gene=293, id=107 removed by soft count = 278\n",
      "--- gene=293, id=109 removed by soft count = 95\n",
      "--- gene=293, id=142 removed by soft count = 297\n",
      "--- gene=293, id=144 removed by soft count = 51\n",
      "--- gene=293, id=154 removed by soft count = 69\n",
      "--- number of probes: 157 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 295\n",
      "--- gene=295, id=20 removed by soft count = 396\n",
      "--- gene=295, id=73 removed by soft count = 79\n",
      "--- gene=295, id=74 removed by soft count = 87\n",
      "--- gene=295, id=75 removed by soft count = 392\n",
      "--- gene=295, id=179 removed by soft count = 1702\n",
      "--- gene=295, id=180 removed by soft count = 57\n",
      "--- number of probes: 209 , kept by blast: 203 , if remove dups: 203\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 296\n",
      "--- gene=296, id=11 removed by soft count = 77\n",
      "--- gene=296, id=13 removed by soft count = 317\n",
      "--- gene=296, id=208 removed by soft count = 226\n",
      "--- gene=296, id=213 removed by soft count = 182\n",
      "--- gene=296, id=216 removed by soft count = 310\n",
      "--- gene=296, id=217 removed by soft count = 129\n",
      "--- gene=296, id=218 removed by soft count = 92\n",
      "--- gene=296, id=220 removed by soft count = 146\n",
      "--- gene=296, id=238 removed by soft count = 116\n",
      "--- number of probes: 274 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 297\n",
      "--- gene=297, id=99 removed by soft count = 53\n",
      "--- gene=297, id=155 removed by soft count = 475\n",
      "--- gene=297, id=181 removed by soft count = 305\n",
      "--- gene=297, id=192 removed by soft count = 85\n",
      "--- number of probes: 194 , kept by blast: 190 , if remove dups: 190\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 298\n",
      "--- gene=298, id=41 removed by soft count = 126\n",
      "--- gene=298, id=216 removed by soft count = 430\n",
      "--- gene=298, id=236 removed by soft count = 78\n",
      "--- number of probes: 253 , kept by blast: 250 , if remove dups: 250\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 299\n",
      "--- gene=299, id=12 removed by soft count = 241\n",
      "--- gene=299, id=16 removed by soft count = 89\n",
      "--- number of probes: 141 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 300\n",
      "--- gene=300, id=71 removed by soft count = 1797\n",
      "--- gene=300, id=98 removed by soft count = 5357\n",
      "--- gene=300, id=99 removed by soft count = 52\n",
      "--- gene=300, id=127 removed by soft count = 56\n",
      "--- number of probes: 151 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 302\n",
      "--- gene=302, id=233 removed by soft count = 66\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 303\n",
      "--- gene=303, id=74 removed by soft count = 115\n",
      "--- gene=303, id=134 removed by soft count = 62\n",
      "--- gene=303, id=200 removed by soft count = 70\n",
      "--- gene=303, id=212 removed by soft count = 164\n",
      "--- number of probes: 213 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 304\n",
      "--- gene=304, id=15 removed by soft count = 116\n",
      "--- gene=304, id=31 removed by soft count = 197\n",
      "--- gene=304, id=53 removed by soft count = 58\n",
      "--- gene=304, id=80 removed by soft count = 144\n",
      "--- gene=304, id=125 removed by soft count = 441\n",
      "--- gene=304, id=126 removed by soft count = 86\n",
      "--- gene=304, id=191 removed by soft count = 54\n",
      "--- gene=304, id=192 removed by soft count = 52\n",
      "--- gene=304, id=193 removed by soft count = 271\n",
      "--- gene=304, id=202 removed by soft count = 726\n",
      "--- gene=304, id=212 removed by soft count = 53\n",
      "--- number of probes: 232 , kept by blast: 221 , if remove dups: 221\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 305\n",
      "--- gene=305, id=24 removed by soft count = 99\n",
      "--- gene=305, id=67 removed by soft count = 69\n",
      "--- gene=305, id=175 removed by soft count = 78\n",
      "--- gene=305, id=220 removed by soft count = 106\n",
      "--- number of probes: 222 , kept by blast: 218 , if remove dups: 218\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 306\n",
      "--- gene=306, id=7 removed by soft count = 530\n",
      "--- gene=306, id=8 removed by soft count = 375\n",
      "--- gene=306, id=10 removed by soft count = 479\n",
      "--- gene=306, id=12 removed by soft count = 52\n",
      "--- gene=306, id=16 removed by soft count = 253\n",
      "--- gene=306, id=107 removed by soft count = 425\n",
      "--- gene=306, id=108 removed by soft count = 919\n",
      "--- gene=306, id=109 removed by soft count = 88\n",
      "--- gene=306, id=110 removed by soft count = 106\n",
      "--- gene=306, id=134 removed by soft count = 411\n",
      "--- number of probes: 242 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 307\n",
      "--- gene=307, id=35 removed by soft count = 82\n",
      "--- gene=307, id=37 removed by soft count = 679\n",
      "--- gene=307, id=54 removed by soft count = 100\n",
      "--- gene=307, id=55 removed by soft count = 132\n",
      "--- gene=307, id=85 removed by soft count = 163\n",
      "--- gene=307, id=86 removed by soft count = 157\n",
      "--- gene=307, id=89 removed by soft count = 290\n",
      "--- gene=307, id=91 removed by soft count = 490\n",
      "--- gene=307, id=94 removed by soft count = 53\n",
      "--- gene=307, id=95 removed by soft count = 92\n",
      "--- gene=307, id=166 removed by soft count = 188\n",
      "--- number of probes: 230 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 308\n",
      "--- gene=308, id=9 removed by soft count = 51\n",
      "--- gene=308, id=17 removed by soft count = 1092\n",
      "--- gene=308, id=18 removed by soft count = 466\n",
      "--- gene=308, id=19 removed by soft count = 108\n",
      "--- gene=308, id=20 removed by soft count = 290\n",
      "--- gene=308, id=21 removed by soft count = 73\n",
      "--- gene=308, id=24 removed by soft count = 112\n",
      "--- number of probes: 203 , kept by blast: 196 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 309\n",
      "--- gene=309, id=77 removed by soft count = 54\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 310\n",
      "--- gene=310, id=42 removed by soft count = 69\n",
      "--- gene=310, id=43 removed by soft count = 98\n",
      "--- gene=310, id=44 removed by soft count = 62\n",
      "--- gene=310, id=45 removed by soft count = 63\n",
      "--- gene=310, id=48 removed by soft count = 132\n",
      "--- gene=310, id=51 removed by soft count = 198\n",
      "--- gene=310, id=56 removed by soft count = 73\n",
      "--- gene=310, id=57 removed by soft count = 595\n",
      "--- gene=310, id=59 removed by soft count = 100\n",
      "--- gene=310, id=67 removed by soft count = 86\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=310, id=71 removed by soft count = 203\n",
      "--- gene=310, id=97 removed by soft count = 154\n",
      "--- gene=310, id=111 removed by soft count = 156\n",
      "--- gene=310, id=112 removed by soft count = 76\n",
      "--- number of probes: 166 , kept by blast: 152 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 311\n",
      "--- gene=311, id=51 removed by soft count = 82\n",
      "--- gene=311, id=70 removed by soft count = 157\n",
      "--- gene=311, id=74 removed by soft count = 435\n",
      "--- gene=311, id=75 removed by soft count = 112\n",
      "--- gene=311, id=158 removed by soft count = 70\n",
      "--- gene=311, id=229 removed by soft count = 230\n",
      "--- number of probes: 267 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 312\n",
      "--- gene=312, id=3 removed by soft count = 133\n",
      "--- gene=312, id=5 removed by soft count = 1439\n",
      "--- gene=312, id=6 removed by soft count = 97\n",
      "--- gene=312, id=9 removed by soft count = 404\n",
      "--- gene=312, id=82 removed by soft count = 141\n",
      "--- gene=312, id=99 removed by soft count = 81\n",
      "--- gene=312, id=119 removed by soft count = 123\n",
      "--- gene=312, id=214 removed by soft count = 51\n",
      "--- number of probes: 241 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 313\n",
      "--- gene=313, id=0 removed by soft count = 99\n",
      "--- gene=313, id=16 removed by soft count = 1808\n",
      "--- gene=313, id=18 removed by soft count = 68\n",
      "--- gene=313, id=46 removed by soft count = 192\n",
      "--- gene=313, id=49 removed by soft count = 497\n",
      "--- gene=313, id=127 removed by soft count = 180\n",
      "--- number of probes: 182 , kept by blast: 176 , if remove dups: 174\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 314\n",
      "--- gene=314, id=71 removed by soft count = 76\n",
      "--- number of probes: 253 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 315\n",
      "--- gene=315, id=60 removed by soft count = 211\n",
      "--- gene=315, id=85 removed by soft count = 59\n",
      "--- gene=315, id=90 removed by soft count = 59\n",
      "--- number of probes: 228 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 316\n",
      "--- gene=316, id=82 removed by soft count = 163\n",
      "--- gene=316, id=125 removed by soft count = 90\n",
      "--- gene=316, id=126 removed by soft count = 219\n",
      "--- gene=316, id=129 removed by soft count = 160\n",
      "--- number of probes: 183 , kept by blast: 179 , if remove dups: 179\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 317\n",
      "--- gene=317, id=6 removed by soft count = 56\n",
      "--- gene=317, id=94 removed by soft count = 209\n",
      "--- gene=317, id=123 removed by soft count = 962\n",
      "--- number of probes: 251 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 318\n",
      "--- gene=318, id=21 removed by soft count = 106\n",
      "--- gene=318, id=64 removed by soft count = 643\n",
      "--- gene=318, id=68 removed by soft count = 214\n",
      "--- gene=318, id=70 removed by soft count = 344\n",
      "--- gene=318, id=77 removed by soft count = 2870\n",
      "--- number of probes: 178 , kept by blast: 173 , if remove dups: 173\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 319\n",
      "--- gene=319, id=13 removed by soft count = 485\n",
      "--- gene=319, id=14 removed by soft count = 62\n",
      "--- gene=319, id=121 removed by soft count = 73\n",
      "--- gene=319, id=122 removed by soft count = 895\n",
      "--- number of probes: 249 , kept by blast: 245 , if remove dups: 245\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 320\n",
      "--- gene=320, id=22 removed by soft count = 421\n",
      "--- gene=320, id=27 removed by soft count = 69\n",
      "--- gene=320, id=34 removed by soft count = 244\n",
      "--- gene=320, id=36 removed by soft count = 284\n",
      "--- gene=320, id=44 removed by soft count = 248\n",
      "--- gene=320, id=118 removed by soft count = 415\n",
      "--- gene=320, id=134 removed by soft count = 62\n",
      "--- gene=320, id=135 removed by soft count = 338\n",
      "--- gene=320, id=136 removed by soft count = 651\n",
      "--- gene=320, id=137 removed by soft count = 122\n",
      "--- gene=320, id=138 removed by soft count = 206\n",
      "--- gene=320, id=149 removed by soft count = 1462\n",
      "--- gene=320, id=151 removed by soft count = 188\n",
      "--- gene=320, id=153 removed by soft count = 412\n",
      "--- gene=320, id=154 removed by soft count = 443\n",
      "--- number of probes: 155 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 321\n",
      "--- gene=321, id=45 removed by soft count = 51\n",
      "--- gene=321, id=235 removed by soft count = 54\n",
      "--- number of probes: 279 , kept by blast: 277 , if remove dups: 277\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 322\n",
      "--- gene=322, id=32 removed by soft count = 158\n",
      "--- gene=322, id=124 removed by soft count = 202\n",
      "--- gene=322, id=166 removed by soft count = 90\n",
      "--- gene=322, id=214 removed by soft count = 6828\n",
      "--- gene=322, id=255 removed by soft count = 67\n",
      "--- number of probes: 258 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 323\n",
      "--- gene=323, id=0 removed by soft count = 299\n",
      "--- gene=323, id=95 removed by soft count = 131\n",
      "--- gene=323, id=238 removed by soft count = 53\n",
      "--- number of probes: 248 , kept by blast: 245 , if remove dups: 245\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 324\n",
      "--- gene=324, id=5 removed by soft count = 111\n",
      "--- gene=324, id=58 removed by soft count = 208\n",
      "--- gene=324, id=66 removed by soft count = 438\n",
      "--- gene=324, id=84 removed by soft count = 74\n",
      "--- gene=324, id=91 removed by soft count = 202\n",
      "--- gene=324, id=92 removed by soft count = 70\n",
      "--- gene=324, id=94 removed by soft count = 78\n",
      "--- gene=324, id=98 removed by soft count = 492\n",
      "--- gene=324, id=99 removed by soft count = 72\n",
      "--- gene=324, id=105 removed by soft count = 714\n",
      "--- number of probes: 171 , kept by blast: 161 , if remove dups: 161\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 325\n",
      "--- gene=325, id=1 removed by soft count = 75\n",
      "--- gene=325, id=3 removed by soft count = 67\n",
      "--- gene=325, id=11 removed by soft count = 60\n",
      "--- gene=325, id=12 removed by soft count = 51\n",
      "--- gene=325, id=14 removed by soft count = 160\n",
      "--- gene=325, id=16 removed by soft count = 95\n",
      "--- gene=325, id=35 removed by soft count = 64\n",
      "--- gene=325, id=40 removed by soft count = 909\n",
      "--- gene=325, id=61 removed by soft count = 355\n",
      "--- gene=325, id=65 removed by soft count = 54\n",
      "--- gene=325, id=76 removed by soft count = 53\n",
      "--- gene=325, id=89 removed by soft count = 63\n",
      "--- gene=325, id=112 removed by soft count = 69\n",
      "--- number of probes: 176 , kept by blast: 163 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 326\n",
      "--- gene=326, id=45 removed by soft count = 1809\n",
      "--- gene=326, id=70 removed by soft count = 80\n",
      "--- gene=326, id=94 removed by soft count = 376\n",
      "--- gene=326, id=95 removed by soft count = 617\n",
      "--- gene=326, id=96 removed by soft count = 281\n",
      "--- number of probes: 191 , kept by blast: 186 , if remove dups: 186\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 327\n",
      "--- gene=327, id=26 removed by soft count = 191\n",
      "--- gene=327, id=48 removed by soft count = 188\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=327, id=52 removed by soft count = 76\n",
      "--- gene=327, id=53 removed by soft count = 86\n",
      "--- gene=327, id=57 removed by soft count = 265\n",
      "--- gene=327, id=63 removed by soft count = 228\n",
      "--- gene=327, id=65 removed by soft count = 676\n",
      "--- gene=327, id=67 removed by soft count = 235\n",
      "--- gene=327, id=69 removed by soft count = 495\n",
      "--- gene=327, id=70 removed by soft count = 1985\n",
      "--- gene=327, id=71 removed by soft count = 128\n",
      "--- gene=327, id=73 removed by soft count = 76\n",
      "--- gene=327, id=74 removed by soft count = 61\n",
      "--- gene=327, id=75 removed by soft count = 74\n",
      "--- gene=327, id=78 removed by soft count = 56\n",
      "--- number of probes: 106 , kept by blast: 91 , if remove dups: 91\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 91\n",
      "-- checking probes in region: 328\n",
      "--- gene=328, id=11 removed by soft count = 1216\n",
      "--- gene=328, id=14 removed by soft count = 233\n",
      "--- gene=328, id=15 removed by soft count = 781\n",
      "--- gene=328, id=16 removed by soft count = 1380\n",
      "--- gene=328, id=23 removed by soft count = 258\n",
      "--- gene=328, id=28 removed by soft count = 139\n",
      "--- gene=328, id=29 removed by soft count = 105\n",
      "--- gene=328, id=30 removed by soft count = 62\n",
      "--- gene=328, id=32 removed by soft count = 170\n",
      "--- gene=328, id=33 removed by soft count = 59\n",
      "--- gene=328, id=34 removed by soft count = 125\n",
      "--- gene=328, id=36 removed by soft count = 77\n",
      "--- gene=328, id=37 removed by soft count = 325\n",
      "--- gene=328, id=46 removed by soft count = 74\n",
      "--- gene=328, id=55 removed by soft count = 68\n",
      "--- gene=328, id=104 removed by soft count = 247\n",
      "--- gene=328, id=105 removed by soft count = 181\n",
      "--- gene=328, id=106 removed by soft count = 60\n",
      "--- gene=328, id=107 removed by soft count = 606\n",
      "--- gene=328, id=114 removed by soft count = 168\n",
      "--- gene=328, id=155 removed by soft count = 220\n",
      "--- gene=328, id=160 removed by soft count = 175\n",
      "--- gene=328, id=161 removed by soft count = 52\n",
      "--- gene=328, id=162 removed by soft count = 151\n",
      "--- gene=328, id=165 removed by soft count = 162\n",
      "--- gene=328, id=166 removed by soft count = 58\n",
      "--- number of probes: 170 , kept by blast: 144 , if remove dups: 144\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 329\n",
      "--- gene=329, id=254 removed by soft count = 420\n",
      "--- number of probes: 295 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 330\n",
      "--- gene=330, id=52 removed by soft count = 323\n",
      "--- gene=330, id=93 removed by soft count = 84\n",
      "--- gene=330, id=112 removed by soft count = 105\n",
      "--- gene=330, id=147 removed by soft count = 183\n",
      "--- gene=330, id=148 removed by soft count = 113\n",
      "--- gene=330, id=149 removed by soft count = 283\n",
      "--- gene=330, id=151 removed by soft count = 74\n",
      "--- gene=330, id=152 removed by soft count = 63\n",
      "--- gene=330, id=157 removed by soft count = 59\n",
      "--- gene=330, id=159 removed by soft count = 59\n",
      "--- gene=330, id=161 removed by soft count = 98\n",
      "--- gene=330, id=203 removed by soft count = 118\n",
      "--- gene=330, id=204 removed by soft count = 56\n",
      "--- gene=330, id=205 removed by soft count = 139\n",
      "--- gene=330, id=209 removed by soft count = 242\n",
      "--- gene=330, id=211 removed by soft count = 179\n",
      "--- gene=330, id=212 removed by soft count = 299\n",
      "--- gene=330, id=214 removed by soft count = 472\n",
      "--- gene=330, id=215 removed by soft count = 185\n",
      "--- gene=330, id=216 removed by soft count = 123\n",
      "--- gene=330, id=217 removed by soft count = 191\n",
      "--- number of probes: 218 , kept by blast: 197 , if remove dups: 197\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 331\n",
      "--- gene=331, id=0 removed by soft count = 114\n",
      "--- gene=331, id=1 removed by soft count = 448\n",
      "--- gene=331, id=30 removed by soft count = 166\n",
      "--- gene=331, id=31 removed by soft count = 167\n",
      "--- gene=331, id=72 removed by soft count = 230\n",
      "--- gene=331, id=73 removed by soft count = 128\n",
      "--- gene=331, id=92 removed by soft count = 64\n",
      "--- gene=331, id=93 removed by soft count = 308\n",
      "--- gene=331, id=94 removed by soft count = 109\n",
      "--- gene=331, id=97 removed by soft count = 134\n",
      "--- gene=331, id=98 removed by soft count = 291\n",
      "--- gene=331, id=99 removed by soft count = 208\n",
      "--- gene=331, id=100 removed by soft count = 864\n",
      "--- gene=331, id=101 removed by soft count = 890\n",
      "--- gene=331, id=125 removed by soft count = 75\n",
      "--- gene=331, id=146 removed by soft count = 330\n",
      "--- number of probes: 154 , kept by blast: 138 , if remove dups: 138\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 332\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 333\n",
      "--- gene=333, id=38 removed by soft count = 458\n",
      "--- gene=333, id=207 removed by soft count = 139\n",
      "--- gene=333, id=209 removed by soft count = 65\n",
      "--- gene=333, id=210 removed by soft count = 106\n",
      "--- gene=333, id=212 removed by soft count = 93\n",
      "--- number of probes: 214 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 334\n",
      "--- gene=334, id=7 removed by soft count = 61\n",
      "--- gene=334, id=43 removed by soft count = 1283\n",
      "--- gene=334, id=46 removed by soft count = 90\n",
      "--- gene=334, id=48 removed by soft count = 32158\n",
      "--- gene=334, id=131 removed by soft count = 217\n",
      "--- gene=334, id=173 removed by soft count = 72\n",
      "--- gene=334, id=229 removed by soft count = 157\n",
      "--- number of probes: 234 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 335\n",
      "--- gene=335, id=2 removed by soft count = 54\n",
      "--- gene=335, id=3 removed by soft count = 51\n",
      "--- gene=335, id=5 removed by soft count = 265\n",
      "--- gene=335, id=63 removed by soft count = 53\n",
      "--- gene=335, id=64 removed by soft count = 104\n",
      "--- gene=335, id=77 removed by soft count = 81\n",
      "--- gene=335, id=81 removed by soft count = 55\n",
      "--- gene=335, id=84 removed by soft count = 177\n",
      "--- gene=335, id=149 removed by soft count = 56\n",
      "--- gene=335, id=189 removed by soft count = 75\n",
      "--- number of probes: 271 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 336\n",
      "--- gene=336, id=4 removed by soft count = 58\n",
      "--- gene=336, id=10 removed by soft count = 56\n",
      "--- gene=336, id=26 removed by soft count = 2547\n",
      "--- gene=336, id=73 removed by soft count = 1613\n",
      "--- gene=336, id=75 removed by soft count = 493\n",
      "--- gene=336, id=76 removed by soft count = 178\n",
      "--- gene=336, id=147 removed by soft count = 92\n",
      "--- gene=336, id=219 removed by soft count = 53\n",
      "--- gene=336, id=222 removed by soft count = 102\n",
      "--- gene=336, id=223 removed by soft count = 73\n",
      "--- number of probes: 244 , kept by blast: 234 , if remove dups: 234\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 337\n",
      "--- gene=337, id=50 removed by soft count = 133\n",
      "--- gene=337, id=60 removed by soft count = 97\n",
      "--- gene=337, id=68 removed by soft count = 79\n",
      "--- gene=337, id=71 removed by soft count = 61\n",
      "--- gene=337, id=105 removed by soft count = 63\n",
      "--- gene=337, id=109 removed by soft count = 329\n",
      "--- gene=337, id=148 removed by soft count = 441\n",
      "--- number of probes: 181 , kept by blast: 174 , if remove dups: 174\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 338\n",
      "--- gene=338, id=52 removed by soft count = 72\n",
      "--- gene=338, id=53 removed by soft count = 369\n",
      "--- gene=338, id=75 removed by soft count = 59\n",
      "--- gene=338, id=77 removed by soft count = 110\n",
      "--- gene=338, id=79 removed by soft count = 63\n",
      "--- gene=338, id=100 removed by soft count = 54\n",
      "--- gene=338, id=107 removed by soft count = 58\n",
      "--- gene=338, id=108 removed by soft count = 68\n",
      "--- gene=338, id=109 removed by soft count = 62\n",
      "--- gene=338, id=111 removed by soft count = 267\n",
      "--- gene=338, id=112 removed by soft count = 88\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=338, id=128 removed by soft count = 209\n",
      "--- gene=338, id=134 removed by soft count = 57\n",
      "--- gene=338, id=139 removed by soft count = 53\n",
      "--- gene=338, id=156 removed by soft count = 68\n",
      "--- gene=338, id=175 removed by soft count = 114\n",
      "--- number of probes: 222 , kept by blast: 206 , if remove dups: 206\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 339\n",
      "--- gene=339, id=7 removed by soft count = 155\n",
      "--- gene=339, id=63 removed by soft count = 2802\n",
      "--- gene=339, id=96 removed by soft count = 207\n",
      "--- gene=339, id=116 removed by soft count = 88\n",
      "--- gene=339, id=117 removed by soft count = 71\n",
      "--- number of probes: 257 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 340\n",
      "--- gene=340, id=7 removed by soft count = 160\n",
      "--- number of probes: 201 , kept by blast: 200 , if remove dups: 199\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 341\n",
      "--- gene=341, id=45 removed by soft count = 1041\n",
      "--- gene=341, id=104 removed by soft count = 294\n",
      "--- number of probes: 284 , kept by blast: 282 , if remove dups: 281\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 342\n",
      "--- gene=342, id=26 removed by soft count = 118\n",
      "--- gene=342, id=63 removed by soft count = 53\n",
      "--- gene=342, id=64 removed by soft count = 93\n",
      "--- gene=342, id=75 removed by soft count = 323\n",
      "--- gene=342, id=77 removed by soft count = 448\n",
      "--- gene=342, id=78 removed by soft count = 54\n",
      "--- gene=342, id=79 removed by soft count = 618\n",
      "--- gene=342, id=85 removed by soft count = 93\n",
      "--- gene=342, id=86 removed by soft count = 582\n",
      "--- gene=342, id=88 removed by soft count = 169\n",
      "--- gene=342, id=139 removed by soft count = 53\n",
      "--- gene=342, id=141 removed by soft count = 53\n",
      "--- gene=342, id=142 removed by soft count = 154\n",
      "--- gene=342, id=143 removed by soft count = 93\n",
      "--- number of probes: 157 , kept by blast: 143 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 343\n",
      "--- gene=343, id=54 removed by soft count = 73\n",
      "--- gene=343, id=56 removed by soft count = 369\n",
      "--- number of probes: 260 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 344\n",
      "--- gene=344, id=82 removed by soft count = 92\n",
      "--- gene=344, id=148 removed by soft count = 97\n",
      "--- gene=344, id=149 removed by soft count = 819\n",
      "--- gene=344, id=151 removed by soft count = 52\n",
      "--- gene=344, id=152 removed by soft count = 81\n",
      "--- number of probes: 234 , kept by blast: 229 , if remove dups: 229\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 345\n",
      "--- gene=345, id=50 removed by soft count = 60\n",
      "--- gene=345, id=53 removed by soft count = 538\n",
      "--- number of probes: 225 , kept by blast: 223 , if remove dups: 223\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 346\n",
      "--- gene=346, id=35 removed by soft count = 54\n",
      "--- gene=346, id=36 removed by soft count = 51\n",
      "--- number of probes: 273 , kept by blast: 271 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 347\n",
      "--- gene=347, id=12 removed by soft count = 255\n",
      "--- gene=347, id=58 removed by soft count = 190\n",
      "--- gene=347, id=89 removed by soft count = 226\n",
      "--- gene=347, id=118 removed by soft count = 241\n",
      "--- gene=347, id=164 removed by soft count = 56\n",
      "--- number of probes: 202 , kept by blast: 197 , if remove dups: 197\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 348\n",
      "--- gene=348, id=61 removed by soft count = 431\n",
      "--- gene=348, id=62 removed by soft count = 179\n",
      "--- gene=348, id=63 removed by soft count = 138\n",
      "--- gene=348, id=64 removed by soft count = 657\n",
      "--- gene=348, id=67 removed by soft count = 92\n",
      "--- gene=348, id=72 removed by soft count = 64\n",
      "--- gene=348, id=75 removed by soft count = 115\n",
      "--- gene=348, id=181 removed by soft count = 1799\n",
      "--- gene=348, id=204 removed by soft count = 446\n",
      "--- gene=348, id=208 removed by soft count = 546\n",
      "--- gene=348, id=209 removed by soft count = 71\n",
      "--- gene=348, id=211 removed by soft count = 57\n",
      "--- number of probes: 223 , kept by blast: 211 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 349\n",
      "--- gene=349, id=0 removed by soft count = 70\n",
      "--- gene=349, id=88 removed by soft count = 782\n",
      "--- gene=349, id=115 removed by soft count = 79\n",
      "--- gene=349, id=146 removed by soft count = 190\n",
      "--- gene=349, id=157 removed by soft count = 352\n",
      "--- number of probes: 161 , kept by blast: 156 , if remove dups: 155\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 350\n",
      "--- gene=350, id=87 removed by hard count = 7\n",
      "--- gene=350, id=102 removed by soft count = 90\n",
      "--- number of probes: 288 , kept by blast: 286 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 351\n",
      "--- gene=351, id=125 removed by soft count = 145\n",
      "--- gene=351, id=126 removed by soft count = 89\n",
      "--- gene=351, id=128 removed by soft count = 295\n",
      "--- gene=351, id=129 removed by soft count = 245\n",
      "--- gene=351, id=130 removed by soft count = 131\n",
      "--- gene=351, id=131 removed by soft count = 758\n",
      "--- gene=351, id=233 removed by soft count = 115522\n",
      "--- number of probes: 294 , kept by blast: 287 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 352\n",
      "--- gene=352, id=40 removed by soft count = 247\n",
      "--- gene=352, id=100 removed by soft count = 149\n",
      "--- gene=352, id=103 removed by soft count = 72\n",
      "--- gene=352, id=104 removed by soft count = 190\n",
      "--- gene=352, id=264 removed by soft count = 70\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 353\n",
      "--- gene=353, id=14 removed by soft count = 200\n",
      "--- gene=353, id=15 removed by soft count = 113\n",
      "--- gene=353, id=16 removed by soft count = 378\n",
      "--- gene=353, id=18 removed by soft count = 608\n",
      "--- gene=353, id=30 removed by soft count = 84\n",
      "--- gene=353, id=31 removed by soft count = 176\n",
      "--- gene=353, id=71 removed by soft count = 472\n",
      "--- gene=353, id=84 removed by soft count = 554\n",
      "--- gene=353, id=87 removed by soft count = 115\n",
      "--- gene=353, id=89 removed by soft count = 160\n",
      "--- gene=353, id=102 removed by soft count = 403\n",
      "--- gene=353, id=143 removed by soft count = 1923\n",
      "--- gene=353, id=144 removed by soft count = 206\n",
      "--- gene=353, id=145 removed by soft count = 3282\n",
      "--- gene=353, id=146 removed by soft count = 293\n",
      "--- gene=353, id=148 removed by soft count = 1283\n",
      "--- gene=353, id=156 removed by soft count = 261\n",
      "--- number of probes: 158 , kept by blast: 141 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 354\n",
      "--- gene=354, id=5 removed by soft count = 94\n",
      "--- gene=354, id=52 removed by soft count = 133\n",
      "--- gene=354, id=94 removed by soft count = 60\n",
      "--- gene=354, id=174 removed by soft count = 755\n",
      "--- gene=354, id=210 removed by soft count = 52\n",
      "--- gene=354, id=218 removed by soft count = 852\n",
      "--- number of probes: 293 , kept by blast: 287 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 355\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=355, id=167 removed by soft count = 165\n",
      "--- number of probes: 234 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 356\n",
      "--- gene=356, id=47 removed by soft count = 82\n",
      "--- gene=356, id=48 removed by soft count = 53\n",
      "--- gene=356, id=67 removed by soft count = 931\n",
      "--- gene=356, id=79 removed by soft count = 195\n",
      "--- gene=356, id=80 removed by soft count = 152\n",
      "--- gene=356, id=155 removed by soft count = 355\n",
      "--- gene=356, id=156 removed by soft count = 295\n",
      "--- gene=356, id=157 removed by soft count = 208\n",
      "--- gene=356, id=158 removed by soft count = 286\n",
      "--- number of probes: 257 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 357\n",
      "--- gene=357, id=49 removed by soft count = 303\n",
      "--- gene=357, id=56 removed by soft count = 95\n",
      "--- gene=357, id=59 removed by soft count = 82\n",
      "--- gene=357, id=60 removed by soft count = 197\n",
      "--- gene=357, id=61 removed by soft count = 64\n",
      "--- gene=357, id=67 removed by soft count = 56\n",
      "--- gene=357, id=70 removed by soft count = 112\n",
      "--- gene=357, id=84 removed by soft count = 69\n",
      "--- gene=357, id=88 removed by soft count = 118\n",
      "--- gene=357, id=92 removed by soft count = 246\n",
      "--- gene=357, id=93 removed by soft count = 187\n",
      "--- gene=357, id=94 removed by soft count = 333\n",
      "--- gene=357, id=95 removed by soft count = 136\n",
      "--- gene=357, id=96 removed by soft count = 295\n",
      "--- gene=357, id=97 removed by soft count = 255\n",
      "--- gene=357, id=98 removed by soft count = 263\n",
      "--- gene=357, id=99 removed by soft count = 332\n",
      "--- gene=357, id=100 removed by soft count = 344\n",
      "--- gene=357, id=101 removed by soft count = 581\n",
      "--- gene=357, id=121 removed by soft count = 108\n",
      "--- gene=357, id=122 removed by soft count = 70\n",
      "--- gene=357, id=123 removed by soft count = 191\n",
      "--- gene=357, id=143 removed by soft count = 178\n",
      "--- number of probes: 170 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 358\n",
      "--- gene=358, id=80 removed by soft count = 369\n",
      "--- gene=358, id=81 removed by soft count = 96\n",
      "--- gene=358, id=84 removed by soft count = 218\n",
      "--- gene=358, id=88 removed by soft count = 227\n",
      "--- gene=358, id=92 removed by soft count = 63\n",
      "--- gene=358, id=107 removed by soft count = 64\n",
      "--- gene=358, id=113 removed by soft count = 406\n",
      "--- gene=358, id=114 removed by soft count = 290\n",
      "--- gene=358, id=115 removed by soft count = 108\n",
      "--- gene=358, id=190 removed by soft count = 199\n",
      "--- gene=358, id=195 removed by soft count = 6111\n",
      "--- number of probes: 216 , kept by blast: 205 , if remove dups: 205\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 359\n",
      "--- gene=359, id=75 removed by soft count = 53\n",
      "--- gene=359, id=77 removed by soft count = 158\n",
      "--- gene=359, id=78 removed by soft count = 65\n",
      "--- gene=359, id=81 removed by soft count = 452\n",
      "--- gene=359, id=84 removed by soft count = 1135\n",
      "--- gene=359, id=85 removed by soft count = 1095\n",
      "--- gene=359, id=164 removed by soft count = 243\n",
      "--- number of probes: 179 , kept by blast: 172 , if remove dups: 172\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 360\n",
      "--- gene=360, id=3 removed by hard count = 0\n",
      "--- gene=360, id=5 removed by hard count = 0\n",
      "--- gene=360, id=131 removed by soft count = 85\n",
      "--- gene=360, id=132 removed by soft count = 501\n",
      "--- gene=360, id=133 removed by soft count = 260\n",
      "--- gene=360, id=168 removed by soft count = 441\n",
      "--- gene=360, id=170 removed by soft count = 140\n",
      "--- gene=360, id=171 removed by soft count = 566\n",
      "--- gene=360, id=174 removed by soft count = 393\n",
      "--- gene=360, id=175 removed by soft count = 562\n",
      "--- gene=360, id=182 removed by soft count = 61\n",
      "--- gene=360, id=184 removed by soft count = 107\n",
      "--- number of probes: 187 , kept by blast: 175 , if remove dups: 174\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 361\n",
      "--- gene=361, id=95 removed by soft count = 287\n",
      "--- gene=361, id=96 removed by soft count = 93\n",
      "--- gene=361, id=97 removed by soft count = 193\n",
      "--- gene=361, id=163 removed by soft count = 65\n",
      "--- gene=361, id=164 removed by soft count = 51\n",
      "--- number of probes: 175 , kept by blast: 170 , if remove dups: 170\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 362\n",
      "--- gene=362, id=205 removed by soft count = 108\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 363\n",
      "--- gene=363, id=89 removed by soft count = 53\n",
      "--- gene=363, id=205 removed by soft count = 70\n",
      "--- gene=363, id=278 removed by soft count = 246\n",
      "--- gene=363, id=281 removed by soft count = 61\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 364\n",
      "--- gene=364, id=120 removed by soft count = 168\n",
      "--- gene=364, id=121 removed by soft count = 442\n",
      "--- number of probes: 253 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 365\n",
      "--- gene=365, id=273 removed by soft count = 116\n",
      "--- number of probes: 284 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 366\n",
      "--- gene=366, id=3 removed by soft count = 60\n",
      "--- gene=366, id=66 removed by soft count = 97\n",
      "--- gene=366, id=106 removed by soft count = 70\n",
      "--- gene=366, id=108 removed by soft count = 83\n",
      "--- gene=366, id=146 removed by soft count = 887\n",
      "--- number of probes: 214 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 367\n",
      "--- gene=367, id=56 removed by soft count = 19588\n",
      "--- gene=367, id=57 removed by soft count = 379\n",
      "--- gene=367, id=65 removed by soft count = 1348\n",
      "--- gene=367, id=66 removed by soft count = 1317\n",
      "--- gene=367, id=67 removed by soft count = 359\n",
      "--- gene=367, id=68 removed by soft count = 444\n",
      "--- gene=367, id=69 removed by soft count = 189\n",
      "--- gene=367, id=71 removed by soft count = 130\n",
      "--- gene=367, id=72 removed by soft count = 152\n",
      "--- gene=367, id=82 removed by soft count = 93\n",
      "--- gene=367, id=84 removed by soft count = 93\n",
      "--- gene=367, id=85 removed by soft count = 154\n",
      "--- gene=367, id=86 removed by soft count = 53\n",
      "--- gene=367, id=87 removed by soft count = 281\n",
      "--- gene=367, id=88 removed by soft count = 79\n",
      "--- gene=367, id=91 removed by soft count = 125\n",
      "--- gene=367, id=93 removed by soft count = 97\n",
      "--- gene=367, id=94 removed by soft count = 62\n",
      "--- gene=367, id=99 removed by soft count = 78\n",
      "--- number of probes: 102 , kept by blast: 83 , if remove dups: 83\n",
      "-- number of probes kept for this region: 83\n",
      "-- checking probes in region: 368\n",
      "--- gene=368, id=1 removed by soft count = 1163\n",
      "--- gene=368, id=2 removed by soft count = 76\n",
      "--- gene=368, id=3 removed by soft count = 612\n",
      "--- gene=368, id=4 removed by soft count = 1258\n",
      "--- gene=368, id=5 removed by soft count = 279\n",
      "--- gene=368, id=6 removed by soft count = 84\n",
      "--- gene=368, id=7 removed by soft count = 473\n",
      "--- gene=368, id=8 removed by soft count = 158\n",
      "--- gene=368, id=9 removed by soft count = 257\n",
      "--- gene=368, id=11 removed by soft count = 888\n",
      "--- gene=368, id=13 removed by soft count = 61\n",
      "--- gene=368, id=14 removed by soft count = 9082\n",
      "--- gene=368, id=16 removed by soft count = 354\n",
      "--- gene=368, id=17 removed by soft count = 75\n",
      "--- gene=368, id=18 removed by soft count = 154\n",
      "--- gene=368, id=22 removed by soft count = 68\n",
      "--- gene=368, id=23 removed by soft count = 57\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=368, id=25 removed by soft count = 65\n",
      "--- gene=368, id=27 removed by soft count = 288\n",
      "--- gene=368, id=29 removed by soft count = 493\n",
      "--- gene=368, id=30 removed by soft count = 157\n",
      "--- gene=368, id=33 removed by soft count = 68\n",
      "--- gene=368, id=35 removed by soft count = 103\n",
      "--- gene=368, id=40 removed by soft count = 896\n",
      "--- gene=368, id=43 removed by soft count = 365\n",
      "--- gene=368, id=44 removed by soft count = 948\n",
      "--- gene=368, id=50 removed by soft count = 66\n",
      "--- gene=368, id=56 removed by soft count = 187\n",
      "--- gene=368, id=58 removed by soft count = 197\n",
      "--- gene=368, id=59 removed by soft count = 396\n",
      "--- gene=368, id=60 removed by soft count = 1065\n",
      "--- gene=368, id=61 removed by soft count = 167\n",
      "--- gene=368, id=62 removed by soft count = 12348\n",
      "--- gene=368, id=63 removed by soft count = 469\n",
      "--- number of probes: 64 , kept by blast: 30 , if remove dups: 30\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: 370\n",
      "--- gene=370, id=152 removed by soft count = 436\n",
      "--- number of probes: 158 , kept by blast: 157 , if remove dups: 157\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 371\n",
      "--- gene=371, id=3 removed by soft count = 5476\n",
      "--- gene=371, id=10 removed by soft count = 56\n",
      "--- gene=371, id=11 removed by soft count = 63\n",
      "--- gene=371, id=12 removed by soft count = 526\n",
      "--- gene=371, id=13 removed by soft count = 724\n",
      "--- gene=371, id=14 removed by soft count = 675\n",
      "--- gene=371, id=27 removed by soft count = 93\n",
      "--- gene=371, id=42 removed by soft count = 386\n",
      "--- gene=371, id=43 removed by soft count = 2481\n",
      "--- gene=371, id=52 removed by soft count = 126\n",
      "--- number of probes: 84 , kept by blast: 74 , if remove dups: 74\n",
      "-- number of probes kept for this region: 74\n",
      "-- checking probes in region: 372\n",
      "--- gene=372, id=4 removed by soft count = 80\n",
      "--- gene=372, id=11 removed by soft count = 107\n",
      "--- gene=372, id=13 removed by soft count = 77\n",
      "--- gene=372, id=14 removed by soft count = 79\n",
      "--- gene=372, id=17 removed by soft count = 87\n",
      "--- gene=372, id=20 removed by soft count = 57\n",
      "--- gene=372, id=29 removed by soft count = 437\n",
      "--- gene=372, id=30 removed by soft count = 326\n",
      "--- gene=372, id=84 removed by soft count = 573\n",
      "--- gene=372, id=181 removed by soft count = 129\n",
      "--- number of probes: 236 , kept by blast: 226 , if remove dups: 226\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 373\n",
      "--- gene=373, id=66 removed by soft count = 69\n",
      "--- number of probes: 270 , kept by blast: 269 , if remove dups: 269\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 374\n",
      "--- gene=374, id=73 removed by soft count = 1101\n",
      "--- gene=374, id=74 removed by soft count = 1226\n",
      "--- gene=374, id=179 removed by soft count = 111\n",
      "--- gene=374, id=200 removed by soft count = 61\n",
      "--- number of probes: 224 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 375\n",
      "--- gene=375, id=37 removed by soft count = 66\n",
      "--- gene=375, id=38 removed by soft count = 70\n",
      "--- gene=375, id=84 removed by soft count = 57\n",
      "--- gene=375, id=88 removed by soft count = 68\n",
      "--- gene=375, id=89 removed by soft count = 115\n",
      "--- gene=375, id=138 removed by soft count = 60\n",
      "--- gene=375, id=139 removed by soft count = 127\n",
      "--- gene=375, id=140 removed by soft count = 84\n",
      "--- number of probes: 226 , kept by blast: 218 , if remove dups: 218\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 376\n",
      "--- gene=376, id=4 removed by soft count = 127\n",
      "--- gene=376, id=138 removed by soft count = 151\n",
      "--- number of probes: 222 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 377\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 379\n",
      "--- gene=379, id=48 removed by soft count = 4068\n",
      "--- gene=379, id=179 removed by soft count = 4154\n",
      "--- gene=379, id=188 removed by soft count = 426\n",
      "--- gene=379, id=231 removed by soft count = 51\n",
      "--- number of probes: 287 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 380\n",
      "--- gene=380, id=112 removed by soft count = 63\n",
      "--- gene=380, id=222 removed by soft count = 1715\n",
      "--- gene=380, id=223 removed by soft count = 316\n",
      "--- number of probes: 224 , kept by blast: 221 , if remove dups: 221\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 381\n",
      "--- gene=381, id=5 removed by soft count = 101\n",
      "--- gene=381, id=266 removed by soft count = 114\n",
      "--- gene=381, id=267 removed by soft count = 66\n",
      "--- gene=381, id=268 removed by soft count = 63\n",
      "--- number of probes: 278 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 382\n",
      "--- gene=382, id=271 removed by soft count = 185\n",
      "--- gene=382, id=273 removed by soft count = 64\n",
      "--- gene=382, id=282 removed by soft count = 54\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 383\n",
      "--- gene=383, id=0 removed by soft count = 308\n",
      "--- gene=383, id=1 removed by soft count = 225\n",
      "--- gene=383, id=2 removed by soft count = 175\n",
      "--- gene=383, id=3 removed by soft count = 70\n",
      "--- gene=383, id=4 removed by soft count = 58\n",
      "--- gene=383, id=83 removed by soft count = 177\n",
      "--- gene=383, id=91 removed by soft count = 258\n",
      "--- gene=383, id=120 removed by soft count = 206\n",
      "--- gene=383, id=121 removed by soft count = 176\n",
      "--- gene=383, id=122 removed by soft count = 718\n",
      "--- number of probes: 176 , kept by blast: 166 , if remove dups: 166\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 384\n",
      "--- gene=384, id=72 removed by soft count = 1291\n",
      "--- gene=384, id=73 removed by soft count = 83\n",
      "--- gene=384, id=76 removed by soft count = 55\n",
      "--- gene=384, id=145 removed by soft count = 56\n",
      "--- gene=384, id=148 removed by soft count = 53\n",
      "--- gene=384, id=151 removed by soft count = 66\n",
      "--- gene=384, id=161 removed by soft count = 68\n",
      "--- gene=384, id=162 removed by soft count = 55\n",
      "--- number of probes: 166 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 385\n",
      "--- number of probes: 272 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 386\n",
      "--- gene=386, id=10 removed by soft count = 52\n",
      "--- gene=386, id=13 removed by soft count = 52\n",
      "--- gene=386, id=16 removed by soft count = 51\n",
      "--- gene=386, id=21 removed by soft count = 160\n",
      "--- gene=386, id=125 removed by soft count = 1076\n",
      "--- gene=386, id=201 removed by soft count = 300\n",
      "--- number of probes: 215 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 387\n",
      "--- gene=387, id=24 removed by soft count = 266\n",
      "--- gene=387, id=186 removed by soft count = 124\n",
      "--- gene=387, id=187 removed by soft count = 230\n",
      "--- gene=387, id=188 removed by soft count = 473\n",
      "--- gene=387, id=189 removed by soft count = 191\n",
      "--- gene=387, id=190 removed by soft count = 358\n",
      "--- gene=387, id=191 removed by soft count = 1884\n",
      "--- number of probes: 192 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 388\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=388, id=20 removed by soft count = 103\n",
      "--- number of probes: 277 , kept by blast: 276 , if remove dups: 276\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 389\n",
      "--- gene=389, id=31 removed by soft count = 460\n",
      "--- gene=389, id=35 removed by soft count = 123\n",
      "--- gene=389, id=38 removed by soft count = 227\n",
      "--- gene=389, id=40 removed by soft count = 219\n",
      "--- gene=389, id=41 removed by soft count = 100\n",
      "--- number of probes: 170 , kept by blast: 165 , if remove dups: 165\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 390\n",
      "--- gene=390, id=1 removed by soft count = 52\n",
      "--- gene=390, id=2 removed by soft count = 174\n",
      "--- gene=390, id=4 removed by soft count = 3050\n",
      "--- gene=390, id=5 removed by soft count = 175\n",
      "--- gene=390, id=6 removed by soft count = 469\n",
      "--- gene=390, id=8 removed by soft count = 95\n",
      "--- gene=390, id=9 removed by soft count = 163\n",
      "--- gene=390, id=13 removed by soft count = 127\n",
      "--- gene=390, id=15 removed by soft count = 100\n",
      "--- gene=390, id=29 removed by soft count = 83\n",
      "--- gene=390, id=70 removed by soft count = 64\n",
      "--- gene=390, id=90 removed by soft count = 114\n",
      "--- gene=390, id=102 removed by soft count = 58\n",
      "--- gene=390, id=112 removed by soft count = 70\n",
      "--- gene=390, id=116 removed by soft count = 76\n",
      "--- gene=390, id=136 removed by soft count = 115\n",
      "--- gene=390, id=138 removed by soft count = 61\n",
      "--- gene=390, id=139 removed by soft count = 2035\n",
      "--- gene=390, id=140 removed by soft count = 883\n",
      "--- gene=390, id=147 removed by soft count = 76\n",
      "--- gene=390, id=150 removed by soft count = 54\n",
      "--- gene=390, id=154 removed by soft count = 68\n",
      "--- gene=390, id=160 removed by soft count = 69\n",
      "--- number of probes: 164 , kept by blast: 141 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 391\n",
      "--- gene=391, id=9 removed by soft count = 52\n",
      "--- gene=391, id=31 removed by soft count = 56\n",
      "--- gene=391, id=69 removed by soft count = 139\n",
      "--- gene=391, id=141 removed by soft count = 108\n",
      "--- gene=391, id=241 removed by soft count = 210\n",
      "--- gene=391, id=246 removed by soft count = 164\n",
      "--- gene=391, id=248 removed by soft count = 70\n",
      "--- gene=391, id=253 removed by soft count = 95\n",
      "--- number of probes: 257 , kept by blast: 249 , if remove dups: 249\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 392\n",
      "--- gene=392, id=82 removed by soft count = 296\n",
      "--- gene=392, id=109 removed by soft count = 533\n",
      "--- gene=392, id=143 removed by soft count = 689\n",
      "--- gene=392, id=144 removed by soft count = 261\n",
      "--- gene=392, id=145 removed by soft count = 389\n",
      "--- gene=392, id=146 removed by soft count = 64\n",
      "--- gene=392, id=148 removed by soft count = 98\n",
      "--- gene=392, id=150 removed by soft count = 157\n",
      "--- gene=392, id=166 removed by soft count = 466\n",
      "--- gene=392, id=167 removed by soft count = 2605\n",
      "--- gene=392, id=183 removed by soft count = 294\n",
      "--- number of probes: 202 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 393\n",
      "--- gene=393, id=12 removed by soft count = 398\n",
      "--- gene=393, id=23 removed by soft count = 240\n",
      "--- gene=393, id=200 removed by soft count = 59\n",
      "--- number of probes: 229 , kept by blast: 226 , if remove dups: 226\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 394\n",
      "--- gene=394, id=44 removed by soft count = 71\n",
      "--- gene=394, id=134 removed by soft count = 96\n",
      "--- gene=394, id=225 removed by soft count = 149\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 395\n",
      "--- gene=395, id=12 removed by soft count = 152\n",
      "--- gene=395, id=13 removed by soft count = 541\n",
      "--- gene=395, id=105 removed by soft count = 69\n",
      "--- gene=395, id=106 removed by soft count = 293\n",
      "--- gene=395, id=107 removed by soft count = 215\n",
      "--- gene=395, id=127 removed by soft count = 125\n",
      "--- gene=395, id=134 removed by soft count = 421\n",
      "--- gene=395, id=136 removed by soft count = 269\n",
      "--- gene=395, id=155 removed by soft count = 238\n",
      "--- gene=395, id=173 removed by soft count = 118\n",
      "--- gene=395, id=175 removed by soft count = 259\n",
      "--- gene=395, id=233 removed by soft count = 65\n",
      "--- gene=395, id=235 removed by soft count = 249\n",
      "--- number of probes: 256 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 396\n",
      "--- gene=396, id=9 removed by soft count = 351\n",
      "--- gene=396, id=15 removed by soft count = 124\n",
      "--- gene=396, id=76 removed by soft count = 78\n",
      "--- gene=396, id=77 removed by soft count = 67\n",
      "--- gene=396, id=78 removed by soft count = 76\n",
      "--- number of probes: 182 , kept by blast: 177 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 397\n",
      "--- gene=397, id=60 removed by soft count = 197\n",
      "--- gene=397, id=177 removed by soft count = 138\n",
      "--- gene=397, id=220 removed by soft count = 487\n",
      "--- gene=397, id=236 removed by soft count = 66\n",
      "--- gene=397, id=244 removed by soft count = 479\n",
      "--- number of probes: 245 , kept by blast: 240 , if remove dups: 240\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 398\n",
      "--- gene=398, id=5 removed by soft count = 113\n",
      "--- gene=398, id=23 removed by soft count = 216\n",
      "--- gene=398, id=27 removed by soft count = 631\n",
      "--- gene=398, id=31 removed by soft count = 238\n",
      "--- gene=398, id=32 removed by soft count = 124\n",
      "--- gene=398, id=33 removed by soft count = 70\n",
      "--- gene=398, id=43 removed by soft count = 61\n",
      "--- gene=398, id=58 removed by soft count = 366\n",
      "--- gene=398, id=126 removed by soft count = 106\n",
      "--- gene=398, id=127 removed by soft count = 63\n",
      "--- gene=398, id=128 removed by soft count = 499\n",
      "--- gene=398, id=130 removed by soft count = 1184\n",
      "--- gene=398, id=131 removed by soft count = 1381\n",
      "--- gene=398, id=133 removed by soft count = 72\n",
      "--- gene=398, id=137 removed by soft count = 206\n",
      "--- gene=398, id=138 removed by soft count = 122\n",
      "--- gene=398, id=154 removed by soft count = 58\n",
      "--- number of probes: 162 , kept by blast: 145 , if remove dups: 145\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 399\n",
      "--- gene=399, id=124 removed by soft count = 1869\n",
      "--- gene=399, id=173 removed by soft count = 61\n",
      "--- gene=399, id=191 removed by soft count = 140\n",
      "--- number of probes: 196 , kept by blast: 193 , if remove dups: 193\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 400\n",
      "--- gene=400, id=138 removed by soft count = 304\n",
      "--- gene=400, id=142 removed by soft count = 120\n",
      "--- gene=400, id=173 removed by soft count = 91\n",
      "--- gene=400, id=174 removed by soft count = 94\n",
      "--- gene=400, id=202 removed by soft count = 54\n",
      "--- gene=400, id=205 removed by soft count = 132\n",
      "--- gene=400, id=206 removed by soft count = 9438\n",
      "--- number of probes: 222 , kept by blast: 215 , if remove dups: 215\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 401\n",
      "--- gene=401, id=25 removed by soft count = 183\n",
      "--- gene=401, id=70 removed by soft count = 94\n",
      "--- gene=401, id=71 removed by soft count = 198\n",
      "--- gene=401, id=109 removed by soft count = 79\n",
      "--- gene=401, id=194 removed by soft count = 4117\n",
      "--- gene=401, id=195 removed by soft count = 138\n",
      "--- gene=401, id=196 removed by soft count = 158\n",
      "--- gene=401, id=197 removed by soft count = 579\n",
      "--- gene=401, id=198 removed by soft count = 663\n",
      "--- gene=401, id=208 removed by soft count = 109\n",
      "--- number of probes: 237 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 402\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=402, id=1 removed by soft count = 320\n",
      "--- gene=402, id=11 removed by soft count = 1086\n",
      "--- gene=402, id=78 removed by soft count = 120\n",
      "--- gene=402, id=83 removed by soft count = 100\n",
      "--- gene=402, id=89 removed by soft count = 154\n",
      "--- gene=402, id=164 removed by soft count = 135\n",
      "--- gene=402, id=166 removed by soft count = 583\n",
      "--- number of probes: 191 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 403\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 404\n",
      "--- gene=404, id=189 removed by soft count = 151\n",
      "--- gene=404, id=192 removed by soft count = 56\n",
      "--- gene=404, id=193 removed by soft count = 100\n",
      "--- gene=404, id=194 removed by soft count = 289\n",
      "--- gene=404, id=195 removed by soft count = 74\n",
      "--- gene=404, id=196 removed by soft count = 64\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 405\n",
      "--- gene=405, id=56 removed by soft count = 359\n",
      "--- gene=405, id=105 removed by hard count = 0\n",
      "--- gene=405, id=152 removed by soft count = 244\n",
      "--- gene=405, id=164 removed by soft count = 108\n",
      "--- gene=405, id=168 removed by soft count = 55\n",
      "--- gene=405, id=184 removed by soft count = 683\n",
      "--- number of probes: 205 , kept by blast: 199 , if remove dups: 199\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 406\n",
      "--- gene=406, id=151 removed by soft count = 134\n",
      "--- gene=406, id=208 removed by soft count = 319\n",
      "--- gene=406, id=209 removed by soft count = 210\n",
      "--- gene=406, id=223 removed by soft count = 254\n",
      "--- number of probes: 261 , kept by blast: 257 , if remove dups: 257\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 407\n",
      "--- gene=407, id=37 removed by soft count = 55\n",
      "--- gene=407, id=42 removed by soft count = 54\n",
      "--- gene=407, id=295 removed by soft count = 93\n",
      "--- number of probes: 299 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 408\n",
      "--- gene=408, id=0 removed by soft count = 1290\n",
      "--- gene=408, id=28 removed by soft count = 54\n",
      "--- gene=408, id=43 removed by soft count = 711\n",
      "--- gene=408, id=161 removed by soft count = 58\n",
      "--- number of probes: 162 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 409\n",
      "--- gene=409, id=89 removed by soft count = 77\n",
      "--- gene=409, id=90 removed by soft count = 3755\n",
      "--- gene=409, id=91 removed by soft count = 233\n",
      "--- gene=409, id=92 removed by soft count = 118\n",
      "--- gene=409, id=93 removed by soft count = 1478\n",
      "--- gene=409, id=100 removed by soft count = 74\n",
      "--- gene=409, id=158 removed by soft count = 162\n",
      "--- gene=409, id=161 removed by soft count = 387\n",
      "--- gene=409, id=162 removed by soft count = 90\n",
      "--- gene=409, id=164 removed by soft count = 182\n",
      "--- gene=409, id=168 removed by soft count = 198\n",
      "--- gene=409, id=169 removed by soft count = 77\n",
      "--- number of probes: 221 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 410\n",
      "--- gene=410, id=86 removed by soft count = 139\n",
      "--- gene=410, id=88 removed by soft count = 58\n",
      "--- gene=410, id=90 removed by soft count = 58\n",
      "--- gene=410, id=91 removed by soft count = 145\n",
      "--- gene=410, id=92 removed by soft count = 81\n",
      "--- gene=410, id=134 removed by soft count = 80\n",
      "--- gene=410, id=165 removed by soft count = 369\n",
      "--- gene=410, id=183 removed by soft count = 130\n",
      "--- number of probes: 271 , kept by blast: 263 , if remove dups: 263\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 411\n",
      "--- gene=411, id=120 removed by soft count = 379\n",
      "--- gene=411, id=121 removed by soft count = 342\n",
      "--- gene=411, id=212 removed by soft count = 90\n",
      "--- gene=411, id=213 removed by soft count = 72\n",
      "--- gene=411, id=239 removed by soft count = 214\n",
      "--- gene=411, id=242 removed by soft count = 217\n",
      "--- gene=411, id=243 removed by soft count = 75\n",
      "--- gene=411, id=244 removed by soft count = 230\n",
      "--- gene=411, id=245 removed by soft count = 124\n",
      "--- gene=411, id=246 removed by soft count = 423\n",
      "--- number of probes: 282 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 412\n",
      "--- gene=412, id=12 removed by soft count = 5174\n",
      "--- gene=412, id=55 removed by soft count = 94\n",
      "--- gene=412, id=166 removed by soft count = 23483\n",
      "--- gene=412, id=177 removed by soft count = 53\n",
      "--- gene=412, id=188 removed by soft count = 54\n",
      "--- gene=412, id=189 removed by soft count = 54\n",
      "--- gene=412, id=198 removed by soft count = 60\n",
      "--- gene=412, id=199 removed by soft count = 54\n",
      "--- gene=412, id=200 removed by soft count = 125\n",
      "--- gene=412, id=201 removed by soft count = 918\n",
      "--- number of probes: 212 , kept by blast: 202 , if remove dups: 202\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 413\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 414\n",
      "--- gene=414, id=46 removed by soft count = 1322\n",
      "--- gene=414, id=47 removed by soft count = 453\n",
      "--- gene=414, id=48 removed by soft count = 110\n",
      "--- gene=414, id=49 removed by soft count = 479\n",
      "--- gene=414, id=50 removed by soft count = 854\n",
      "--- gene=414, id=51 removed by soft count = 492\n",
      "--- gene=414, id=52 removed by soft count = 853\n",
      "--- gene=414, id=53 removed by soft count = 230\n",
      "--- gene=414, id=78 removed by soft count = 217\n",
      "--- gene=414, id=79 removed by soft count = 469\n",
      "--- gene=414, id=80 removed by soft count = 242\n",
      "--- gene=414, id=81 removed by soft count = 79\n",
      "--- number of probes: 188 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 415\n",
      "--- gene=415, id=0 removed by soft count = 126\n",
      "--- gene=415, id=81 removed by soft count = 304\n",
      "--- gene=415, id=82 removed by soft count = 1323\n",
      "--- gene=415, id=84 removed by soft count = 115\n",
      "--- gene=415, id=132 removed by soft count = 90\n",
      "--- gene=415, id=133 removed by soft count = 77\n",
      "--- gene=415, id=169 removed by soft count = 53\n",
      "--- gene=415, id=171 removed by soft count = 254\n",
      "--- gene=415, id=180 removed by soft count = 58\n",
      "--- number of probes: 252 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 416\n",
      "--- gene=416, id=9 removed by soft count = 185\n",
      "--- gene=416, id=10 removed by soft count = 1921\n",
      "--- gene=416, id=12 removed by soft count = 57\n",
      "--- gene=416, id=14 removed by soft count = 193\n",
      "--- number of probes: 252 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 417\n",
      "--- gene=417, id=76 removed by soft count = 204\n",
      "--- gene=417, id=77 removed by soft count = 164\n",
      "--- gene=417, id=80 removed by soft count = 345\n",
      "--- gene=417, id=81 removed by soft count = 794\n",
      "--- gene=417, id=82 removed by soft count = 95\n",
      "--- gene=417, id=84 removed by soft count = 58\n",
      "--- gene=417, id=167 removed by soft count = 186\n",
      "--- number of probes: 300 , kept by blast: 293 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 418\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=418, id=34 removed by soft count = 574\n",
      "--- gene=418, id=120 removed by soft count = 85\n",
      "--- gene=418, id=121 removed by soft count = 171\n",
      "--- gene=418, id=123 removed by soft count = 65\n",
      "--- gene=418, id=124 removed by soft count = 287\n",
      "--- gene=418, id=151 removed by soft count = 550\n",
      "--- gene=418, id=152 removed by soft count = 112\n",
      "--- gene=418, id=154 removed by soft count = 357\n",
      "--- gene=418, id=193 removed by soft count = 249\n",
      "--- number of probes: 200 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 419\n",
      "--- gene=419, id=64 removed by soft count = 88\n",
      "--- gene=419, id=110 removed by soft count = 179\n",
      "--- gene=419, id=240 removed by soft count = 84\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 421\n",
      "--- gene=421, id=0 removed by soft count = 177\n",
      "--- gene=421, id=1 removed by soft count = 15293\n",
      "--- gene=421, id=46 removed by soft count = 68\n",
      "--- gene=421, id=51 removed by soft count = 85\n",
      "--- gene=421, id=54 removed by soft count = 485\n",
      "--- gene=421, id=164 removed by soft count = 553\n",
      "--- gene=421, id=165 removed by soft count = 253\n",
      "--- gene=421, id=166 removed by soft count = 101\n",
      "--- gene=421, id=211 removed by soft count = 177\n",
      "--- number of probes: 212 , kept by blast: 203 , if remove dups: 203\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 422\n",
      "--- gene=422, id=106 removed by hard count = 4\n",
      "--- gene=422, id=147 removed by soft count = 105\n",
      "--- gene=422, id=232 removed by soft count = 93\n",
      "--- gene=422, id=235 removed by soft count = 439\n",
      "--- gene=422, id=236 removed by soft count = 470\n",
      "--- gene=422, id=237 removed by soft count = 263\n",
      "--- gene=422, id=242 removed by soft count = 59\n",
      "--- gene=422, id=245 removed by soft count = 133\n",
      "--- number of probes: 296 , kept by blast: 288 , if remove dups: 288\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 423\n",
      "--- gene=423, id=197 removed by soft count = 129\n",
      "--- gene=423, id=234 removed by soft count = 299\n",
      "--- gene=423, id=235 removed by soft count = 695\n",
      "--- gene=423, id=236 removed by soft count = 930\n",
      "--- gene=423, id=238 removed by soft count = 561\n",
      "--- gene=423, id=260 removed by soft count = 567\n",
      "--- gene=423, id=261 removed by soft count = 318\n",
      "--- gene=423, id=262 removed by soft count = 496\n",
      "--- gene=423, id=264 removed by soft count = 320\n",
      "--- gene=423, id=266 removed by soft count = 217\n",
      "--- gene=423, id=268 removed by soft count = 152\n",
      "--- number of probes: 300 , kept by blast: 289 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 424\n",
      "--- gene=424, id=5 removed by soft count = 73\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 425\n",
      "--- gene=425, id=22 removed by soft count = 63\n",
      "--- gene=425, id=32 removed by soft count = 83\n",
      "--- number of probes: 221 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 426\n",
      "--- gene=426, id=112 removed by soft count = 117\n",
      "--- gene=426, id=207 removed by soft count = 69\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 427\n",
      "--- gene=427, id=41 removed by soft count = 720\n",
      "--- gene=427, id=42 removed by soft count = 307\n",
      "--- gene=427, id=44 removed by soft count = 85\n",
      "--- gene=427, id=48 removed by soft count = 527\n",
      "--- gene=427, id=58 removed by soft count = 362\n",
      "--- gene=427, id=139 removed by soft count = 55\n",
      "--- number of probes: 217 , kept by blast: 211 , if remove dups: 211\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 428\n",
      "--- gene=428, id=4 removed by soft count = 120\n",
      "--- gene=428, id=6 removed by soft count = 56\n",
      "--- gene=428, id=7 removed by soft count = 90\n",
      "--- gene=428, id=75 removed by soft count = 407\n",
      "--- gene=428, id=76 removed by soft count = 91\n",
      "--- gene=428, id=173 removed by soft count = 125\n",
      "--- gene=428, id=215 removed by soft count = 180\n",
      "--- gene=428, id=216 removed by soft count = 160\n",
      "--- gene=428, id=220 removed by soft count = 220\n",
      "--- gene=428, id=239 removed by soft count = 568\n",
      "--- gene=428, id=240 removed by soft count = 384\n",
      "--- number of probes: 243 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 429\n",
      "--- gene=429, id=19 removed by soft count = 447\n",
      "--- gene=429, id=22 removed by soft count = 105\n",
      "--- gene=429, id=154 removed by soft count = 295\n",
      "--- gene=429, id=155 removed by soft count = 800\n",
      "--- gene=429, id=185 removed by soft count = 52\n",
      "--- number of probes: 193 , kept by blast: 188 , if remove dups: 188\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 430\n",
      "--- gene=430, id=26 removed by soft count = 117\n",
      "--- gene=430, id=29 removed by soft count = 118\n",
      "--- gene=430, id=123 removed by soft count = 110\n",
      "--- gene=430, id=125 removed by soft count = 92\n",
      "--- gene=430, id=126 removed by soft count = 189\n",
      "--- gene=430, id=127 removed by soft count = 266\n",
      "--- gene=430, id=128 removed by soft count = 115\n",
      "--- number of probes: 236 , kept by blast: 229 , if remove dups: 229\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 431\n",
      "--- gene=431, id=54 removed by soft count = 76\n",
      "--- gene=431, id=55 removed by soft count = 147\n",
      "--- gene=431, id=98 removed by soft count = 1497\n",
      "--- gene=431, id=129 removed by soft count = 484\n",
      "--- gene=431, id=142 removed by soft count = 242\n",
      "--- gene=431, id=143 removed by soft count = 197\n",
      "--- gene=431, id=154 removed by soft count = 110\n",
      "--- gene=431, id=170 removed by soft count = 258\n",
      "--- gene=431, id=174 removed by soft count = 202\n",
      "--- gene=431, id=176 removed by soft count = 147\n",
      "--- gene=431, id=204 removed by soft count = 121\n",
      "--- gene=431, id=217 removed by soft count = 835\n",
      "--- number of probes: 237 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 432\n",
      "--- gene=432, id=88 removed by soft count = 137\n",
      "--- gene=432, id=209 removed by soft count = 105\n",
      "--- gene=432, id=211 removed by soft count = 89\n",
      "--- gene=432, id=212 removed by soft count = 101\n",
      "--- gene=432, id=215 removed by soft count = 51\n",
      "--- gene=432, id=227 removed by soft count = 81\n",
      "--- gene=432, id=228 removed by soft count = 58\n",
      "--- gene=432, id=269 removed by soft count = 177\n",
      "--- number of probes: 270 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 435\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 436\n",
      "--- gene=436, id=8 removed by soft count = 123\n",
      "--- gene=436, id=11 removed by soft count = 210\n",
      "--- gene=436, id=12 removed by soft count = 97\n",
      "--- gene=436, id=102 removed by soft count = 89\n",
      "--- gene=436, id=244 removed by soft count = 1328\n",
      "--- number of probes: 294 , kept by blast: 289 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 437\n",
      "--- gene=437, id=54 removed by soft count = 130\n",
      "--- gene=437, id=60 removed by soft count = 11376\n",
      "--- gene=437, id=67 removed by soft count = 166\n",
      "--- gene=437, id=72 removed by soft count = 83\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=437, id=156 removed by soft count = 51\n",
      "--- number of probes: 188 , kept by blast: 183 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 438\n",
      "--- gene=438, id=36 removed by soft count = 109\n",
      "--- gene=438, id=183 removed by soft count = 133\n",
      "--- gene=438, id=184 removed by soft count = 272\n",
      "--- gene=438, id=187 removed by soft count = 118\n",
      "--- gene=438, id=188 removed by soft count = 115\n",
      "--- gene=438, id=191 removed by soft count = 83\n",
      "--- gene=438, id=192 removed by soft count = 81\n",
      "--- gene=438, id=194 removed by soft count = 134\n",
      "--- gene=438, id=195 removed by soft count = 71\n",
      "--- gene=438, id=196 removed by soft count = 144\n",
      "--- gene=438, id=199 removed by soft count = 768\n",
      "--- gene=438, id=202 removed by soft count = 167\n",
      "--- gene=438, id=205 removed by soft count = 51\n",
      "--- number of probes: 206 , kept by blast: 193 , if remove dups: 193\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 439\n",
      "--- gene=439, id=158 removed by soft count = 60\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 440\n",
      "--- gene=440, id=39 removed by soft count = 51\n",
      "--- gene=440, id=130 removed by soft count = 55\n",
      "--- gene=440, id=185 removed by soft count = 56\n",
      "--- gene=440, id=240 removed by soft count = 89\n",
      "--- gene=440, id=242 removed by soft count = 155\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 441\n",
      "--- gene=441, id=130 removed by soft count = 117\n",
      "--- number of probes: 183 , kept by blast: 182 , if remove dups: 182\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 442\n",
      "--- gene=442, id=75 removed by soft count = 355\n",
      "--- gene=442, id=129 removed by soft count = 76\n",
      "--- gene=442, id=177 removed by soft count = 305\n",
      "--- gene=442, id=267 removed by soft count = 155\n",
      "--- gene=442, id=269 removed by soft count = 286\n",
      "--- number of probes: 276 , kept by blast: 271 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 443\n",
      "--- gene=443, id=228 removed by soft count = 54\n",
      "--- number of probes: 241 , kept by blast: 240 , if remove dups: 240\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 444\n",
      "--- gene=444, id=55 removed by soft count = 6432\n",
      "--- gene=444, id=145 removed by soft count = 89\n",
      "--- gene=444, id=259 removed by soft count = 126\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 446\n",
      "--- gene=446, id=1 removed by soft count = 82\n",
      "--- gene=446, id=2 removed by soft count = 892\n",
      "--- gene=446, id=3 removed by soft count = 108\n",
      "--- gene=446, id=4 removed by soft count = 1213\n",
      "--- gene=446, id=59 removed by soft count = 7035\n",
      "--- gene=446, id=60 removed by soft count = 391\n",
      "--- gene=446, id=61 removed by soft count = 65\n",
      "--- gene=446, id=62 removed by soft count = 87\n",
      "--- gene=446, id=63 removed by soft count = 8922\n",
      "--- gene=446, id=89 removed by soft count = 114\n",
      "--- gene=446, id=91 removed by soft count = 284\n",
      "--- gene=446, id=92 removed by soft count = 87\n",
      "--- gene=446, id=93 removed by soft count = 166\n",
      "--- gene=446, id=94 removed by soft count = 538\n",
      "--- gene=446, id=96 removed by soft count = 265\n",
      "--- gene=446, id=100 removed by soft count = 1523\n",
      "--- gene=446, id=101 removed by soft count = 8945\n",
      "--- gene=446, id=103 removed by soft count = 72\n",
      "--- number of probes: 108 , kept by blast: 90 , if remove dups: 90\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 90\n",
      "-- checking probes in region: 447\n",
      "--- gene=447, id=49 removed by soft count = 706\n",
      "--- gene=447, id=74 removed by soft count = 105\n",
      "--- gene=447, id=78 removed by soft count = 75\n",
      "--- gene=447, id=114 removed by soft count = 475\n",
      "--- number of probes: 181 , kept by blast: 177 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 448\n",
      "--- gene=448, id=87 removed by soft count = 122\n",
      "--- gene=448, id=91 removed by soft count = 55\n",
      "--- gene=448, id=93 removed by soft count = 468\n",
      "--- gene=448, id=94 removed by soft count = 1719\n",
      "--- gene=448, id=96 removed by soft count = 139\n",
      "--- gene=448, id=97 removed by soft count = 690\n",
      "--- gene=448, id=108 removed by soft count = 272\n",
      "--- gene=448, id=117 removed by soft count = 179\n",
      "--- gene=448, id=118 removed by soft count = 164\n",
      "--- gene=448, id=120 removed by soft count = 162\n",
      "--- gene=448, id=122 removed by soft count = 171\n",
      "--- gene=448, id=123 removed by soft count = 95\n",
      "--- gene=448, id=124 removed by soft count = 238\n",
      "--- gene=448, id=125 removed by soft count = 87\n",
      "--- gene=448, id=126 removed by soft count = 250\n",
      "--- number of probes: 127 , kept by blast: 112 , if remove dups: 112\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 112\n",
      "-- checking probes in region: 449\n",
      "--- gene=449, id=5 removed by soft count = 361\n",
      "--- gene=449, id=84 removed by soft count = 585\n",
      "--- number of probes: 228 , kept by blast: 226 , if remove dups: 226\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 450\n",
      "--- gene=450, id=39 removed by soft count = 85\n",
      "--- gene=450, id=41 removed by soft count = 61\n",
      "--- gene=450, id=42 removed by soft count = 94\n",
      "--- gene=450, id=44 removed by soft count = 589\n",
      "--- gene=450, id=45 removed by soft count = 1810\n",
      "--- gene=450, id=46 removed by soft count = 1287\n",
      "--- gene=450, id=47 removed by soft count = 446\n",
      "--- gene=450, id=63 removed by soft count = 84\n",
      "--- gene=450, id=64 removed by soft count = 57\n",
      "--- gene=450, id=65 removed by soft count = 437\n",
      "--- gene=450, id=67 removed by soft count = 744\n",
      "--- gene=450, id=68 removed by soft count = 204\n",
      "--- gene=450, id=71 removed by soft count = 115\n",
      "--- gene=450, id=72 removed by soft count = 270\n",
      "--- gene=450, id=145 removed by soft count = 59\n",
      "--- number of probes: 268 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 451\n",
      "--- number of probes: 258 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 452\n",
      "--- gene=452, id=2 removed by soft count = 69\n",
      "--- gene=452, id=15 removed by soft count = 122\n",
      "--- gene=452, id=17 removed by soft count = 90\n",
      "--- gene=452, id=19 removed by soft count = 127\n",
      "--- gene=452, id=94 removed by soft count = 108\n",
      "--- gene=452, id=123 removed by soft count = 165\n",
      "--- gene=452, id=146 removed by soft count = 916\n",
      "--- gene=452, id=148 removed by soft count = 186\n",
      "--- gene=452, id=149 removed by soft count = 77\n",
      "--- gene=452, id=151 removed by soft count = 70\n",
      "--- gene=452, id=152 removed by soft count = 95\n",
      "--- gene=452, id=179 removed by soft count = 1125\n",
      "--- gene=452, id=180 removed by soft count = 143\n",
      "--- gene=452, id=181 removed by soft count = 2151\n",
      "--- number of probes: 197 , kept by blast: 183 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 453\n",
      "--- gene=453, id=1 removed by soft count = 64\n",
      "--- gene=453, id=2 removed by soft count = 53\n",
      "--- gene=453, id=7 removed by soft count = 123\n",
      "--- gene=453, id=9 removed by soft count = 66\n",
      "--- gene=453, id=20 removed by soft count = 208\n",
      "--- gene=453, id=22 removed by soft count = 96\n",
      "--- gene=453, id=44 removed by soft count = 74\n",
      "--- gene=453, id=45 removed by soft count = 77\n",
      "--- gene=453, id=46 removed by soft count = 84\n",
      "--- gene=453, id=49 removed by soft count = 126\n",
      "--- gene=453, id=54 removed by soft count = 85\n",
      "--- gene=453, id=57 removed by soft count = 163\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=453, id=58 removed by soft count = 715\n",
      "--- gene=453, id=59 removed by soft count = 72\n",
      "--- gene=453, id=63 removed by soft count = 877\n",
      "--- gene=453, id=64 removed by soft count = 476\n",
      "--- gene=453, id=65 removed by soft count = 254\n",
      "--- gene=453, id=66 removed by soft count = 121\n",
      "--- number of probes: 67 , kept by blast: 49 , if remove dups: 49\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: 454\n",
      "--- gene=454, id=104 removed by soft count = 7466\n",
      "--- gene=454, id=124 removed by soft count = 160\n",
      "--- gene=454, id=129 removed by soft count = 82\n",
      "--- gene=454, id=132 removed by soft count = 132\n",
      "--- number of probes: 145 , kept by blast: 141 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 455\n",
      "--- gene=455, id=14 removed by soft count = 619\n",
      "--- gene=455, id=15 removed by soft count = 1030\n",
      "--- gene=455, id=16 removed by soft count = 352\n",
      "--- gene=455, id=17 removed by soft count = 97\n",
      "--- gene=455, id=19 removed by soft count = 174\n",
      "--- gene=455, id=20 removed by soft count = 125\n",
      "--- gene=455, id=21 removed by soft count = 157\n",
      "--- gene=455, id=26 removed by soft count = 60\n",
      "--- gene=455, id=42 removed by soft count = 195\n",
      "--- gene=455, id=50 removed by soft count = 189\n",
      "--- gene=455, id=61 removed by soft count = 181\n",
      "--- gene=455, id=63 removed by soft count = 492\n",
      "--- gene=455, id=64 removed by soft count = 137\n",
      "--- gene=455, id=69 removed by soft count = 123\n",
      "--- gene=455, id=70 removed by soft count = 236\n",
      "--- gene=455, id=135 removed by soft count = 169\n",
      "--- number of probes: 142 , kept by blast: 126 , if remove dups: 126\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 456\n",
      "--- gene=456, id=12 removed by soft count = 464\n",
      "--- gene=456, id=13 removed by soft count = 51\n",
      "--- gene=456, id=14 removed by soft count = 185\n",
      "--- gene=456, id=16 removed by hard count = 6\n",
      "--- number of probes: 217 , kept by blast: 213 , if remove dups: 213\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 457\n",
      "--- gene=457, id=9 removed by soft count = 63\n",
      "--- gene=457, id=10 removed by soft count = 106\n",
      "--- gene=457, id=12 removed by soft count = 65\n",
      "--- gene=457, id=16 removed by soft count = 60\n",
      "--- gene=457, id=27 removed by soft count = 108\n",
      "--- gene=457, id=28 removed by soft count = 329\n",
      "--- gene=457, id=61 removed by soft count = 1151\n",
      "--- gene=457, id=175 removed by soft count = 84\n",
      "--- gene=457, id=191 removed by soft count = 2121\n",
      "--- number of probes: 207 , kept by blast: 198 , if remove dups: 198\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 458\n",
      "--- gene=458, id=70 removed by soft count = 116\n",
      "--- gene=458, id=171 removed by soft count = 934\n",
      "--- gene=458, id=172 removed by soft count = 297\n",
      "--- gene=458, id=173 removed by soft count = 571\n",
      "--- gene=458, id=174 removed by soft count = 953\n",
      "--- gene=458, id=181 removed by soft count = 62\n",
      "--- gene=458, id=197 removed by soft count = 54\n",
      "--- gene=458, id=198 removed by soft count = 80\n",
      "--- gene=458, id=211 removed by soft count = 78\n",
      "--- number of probes: 236 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 459\n",
      "--- gene=459, id=164 removed by soft count = 175\n",
      "--- number of probes: 180 , kept by blast: 179 , if remove dups: 179\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 460\n",
      "--- gene=460, id=143 removed by soft count = 166\n",
      "--- gene=460, id=144 removed by soft count = 100\n",
      "--- gene=460, id=145 removed by soft count = 143\n",
      "--- gene=460, id=146 removed by soft count = 293\n",
      "--- gene=460, id=147 removed by soft count = 51\n",
      "--- gene=460, id=149 removed by soft count = 306\n",
      "--- gene=460, id=183 removed by soft count = 144\n",
      "--- gene=460, id=187 removed by soft count = 1561\n",
      "--- gene=460, id=188 removed by soft count = 1096\n",
      "--- gene=460, id=189 removed by soft count = 712\n",
      "--- gene=460, id=192 removed by soft count = 83\n",
      "--- gene=460, id=194 removed by soft count = 637\n",
      "--- gene=460, id=196 removed by soft count = 57\n",
      "--- number of probes: 202 , kept by blast: 189 , if remove dups: 189\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 461\n",
      "--- gene=461, id=86 removed by soft count = 1594\n",
      "--- gene=461, id=87 removed by soft count = 578\n",
      "--- gene=461, id=88 removed by soft count = 397\n",
      "--- gene=461, id=146 removed by soft count = 69\n",
      "--- gene=461, id=195 removed by soft count = 608\n",
      "--- number of probes: 209 , kept by blast: 204 , if remove dups: 204\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 462\n",
      "--- gene=462, id=9 removed by soft count = 115\n",
      "--- gene=462, id=23 removed by soft count = 158\n",
      "--- gene=462, id=27 removed by soft count = 236\n",
      "--- gene=462, id=124 removed by soft count = 99\n",
      "--- gene=462, id=125 removed by soft count = 73\n",
      "--- gene=462, id=126 removed by soft count = 79\n",
      "--- gene=462, id=129 removed by soft count = 61\n",
      "--- gene=462, id=132 removed by soft count = 62\n",
      "--- gene=462, id=136 removed by soft count = 85\n",
      "--- gene=462, id=138 removed by soft count = 56\n",
      "--- gene=462, id=139 removed by soft count = 57\n",
      "--- gene=462, id=140 removed by soft count = 79\n",
      "--- number of probes: 170 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 463\n",
      "--- gene=463, id=0 removed by soft count = 430\n",
      "--- gene=463, id=2 removed by soft count = 444\n",
      "--- gene=463, id=67 removed by soft count = 94\n",
      "--- gene=463, id=89 removed by soft count = 186\n",
      "--- gene=463, id=90 removed by soft count = 81\n",
      "--- gene=463, id=111 removed by soft count = 128\n",
      "--- gene=463, id=112 removed by soft count = 234\n",
      "--- gene=463, id=119 removed by soft count = 77\n",
      "--- gene=463, id=123 removed by soft count = 104\n",
      "--- gene=463, id=124 removed by soft count = 604\n",
      "--- gene=463, id=128 removed by soft count = 58\n",
      "--- gene=463, id=130 removed by soft count = 52\n",
      "--- gene=463, id=134 removed by soft count = 136\n",
      "--- number of probes: 143 , kept by blast: 130 , if remove dups: 130\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 464\n",
      "--- gene=464, id=1 removed by soft count = 101\n",
      "--- gene=464, id=36 removed by soft count = 105\n",
      "--- gene=464, id=38 removed by soft count = 240\n",
      "--- gene=464, id=89 removed by soft count = 74\n",
      "--- gene=464, id=91 removed by soft count = 334\n",
      "--- gene=464, id=110 removed by soft count = 123\n",
      "--- gene=464, id=162 removed by soft count = 292\n",
      "--- gene=464, id=174 removed by soft count = 23842\n",
      "--- gene=464, id=203 removed by soft count = 349\n",
      "--- gene=464, id=207 removed by soft count = 663\n",
      "--- gene=464, id=208 removed by soft count = 156\n",
      "--- gene=464, id=209 removed by soft count = 85\n",
      "--- gene=464, id=210 removed by soft count = 63\n",
      "--- number of probes: 211 , kept by blast: 198 , if remove dups: 198\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 465\n",
      "--- gene=465, id=39 removed by soft count = 87\n",
      "--- gene=465, id=41 removed by soft count = 84\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 466\n",
      "--- gene=466, id=24 removed by soft count = 116\n",
      "--- gene=466, id=25 removed by soft count = 142\n",
      "--- number of probes: 189 , kept by blast: 187 , if remove dups: 187\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 467\n",
      "--- gene=467, id=93 removed by soft count = 265\n",
      "--- gene=467, id=115 removed by soft count = 624\n",
      "--- gene=467, id=116 removed by soft count = 71\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=467, id=146 removed by soft count = 253\n",
      "--- number of probes: 200 , kept by blast: 196 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 468\n",
      "--- gene=468, id=48 removed by soft count = 74\n",
      "--- gene=468, id=194 removed by soft count = 84\n",
      "--- gene=468, id=204 removed by soft count = 80\n",
      "--- gene=468, id=213 removed by soft count = 239\n",
      "--- gene=468, id=215 removed by soft count = 273\n",
      "--- gene=468, id=217 removed by soft count = 61\n",
      "--- gene=468, id=219 removed by soft count = 145\n",
      "--- gene=468, id=221 removed by soft count = 56\n",
      "--- number of probes: 222 , kept by blast: 214 , if remove dups: 214\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 469\n",
      "--- gene=469, id=213 removed by soft count = 375\n",
      "--- gene=469, id=242 removed by soft count = 379\n",
      "--- number of probes: 260 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 470\n",
      "--- gene=470, id=0 removed by soft count = 379\n",
      "--- gene=470, id=1 removed by soft count = 211\n",
      "--- gene=470, id=2 removed by soft count = 70\n",
      "--- gene=470, id=3 removed by soft count = 86\n",
      "--- gene=470, id=4 removed by soft count = 156\n",
      "--- gene=470, id=5 removed by soft count = 211\n",
      "--- gene=470, id=64 removed by soft count = 64\n",
      "--- gene=470, id=73 removed by soft count = 81\n",
      "--- gene=470, id=76 removed by soft count = 406\n",
      "--- gene=470, id=77 removed by soft count = 264\n",
      "--- gene=470, id=78 removed by soft count = 279\n",
      "--- gene=470, id=80 removed by soft count = 296\n",
      "--- gene=470, id=81 removed by soft count = 1591\n",
      "--- gene=470, id=122 removed by soft count = 107\n",
      "--- gene=470, id=127 removed by soft count = 91\n",
      "--- gene=470, id=128 removed by soft count = 120\n",
      "--- gene=470, id=130 removed by soft count = 343\n",
      "--- number of probes: 131 , kept by blast: 114 , if remove dups: 114\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 114\n",
      "-- checking probes in region: 471\n",
      "--- gene=471, id=4 removed by soft count = 84\n",
      "--- gene=471, id=7 removed by soft count = 53\n",
      "--- gene=471, id=18 removed by soft count = 57\n",
      "--- gene=471, id=29 removed by soft count = 102\n",
      "--- gene=471, id=134 removed by soft count = 51\n",
      "--- gene=471, id=139 removed by soft count = 110\n",
      "--- gene=471, id=141 removed by soft count = 63\n",
      "--- gene=471, id=144 removed by soft count = 1060\n",
      "--- number of probes: 199 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 472\n",
      "--- gene=472, id=0 removed by soft count = 621\n",
      "--- gene=472, id=95 removed by soft count = 299\n",
      "--- gene=472, id=121 removed by soft count = 117\n",
      "--- gene=472, id=136 removed by soft count = 12970\n",
      "--- gene=472, id=154 removed by soft count = 77\n",
      "--- gene=472, id=156 removed by soft count = 106\n",
      "--- gene=472, id=199 removed by soft count = 54\n",
      "--- number of probes: 203 , kept by blast: 196 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 473\n",
      "--- gene=473, id=17 removed by soft count = 119\n",
      "--- gene=473, id=19 removed by soft count = 56\n",
      "--- gene=473, id=40 removed by soft count = 51\n",
      "--- gene=473, id=64 removed by soft count = 127\n",
      "--- gene=473, id=65 removed by soft count = 73\n",
      "--- gene=473, id=92 removed by soft count = 130\n",
      "--- gene=473, id=182 removed by soft count = 417\n",
      "--- gene=473, id=185 removed by soft count = 78\n",
      "--- gene=473, id=199 removed by soft count = 73\n",
      "--- gene=473, id=202 removed by soft count = 5597\n",
      "--- number of probes: 203 , kept by blast: 193 , if remove dups: 193\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 474\n",
      "--- gene=474, id=193 removed by soft count = 131\n",
      "--- number of probes: 235 , kept by blast: 234 , if remove dups: 234\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 475\n",
      "--- gene=475, id=0 removed by soft count = 100\n",
      "--- gene=475, id=65 removed by soft count = 89\n",
      "--- gene=475, id=66 removed by soft count = 65\n",
      "--- gene=475, id=73 removed by soft count = 103\n",
      "--- gene=475, id=109 removed by soft count = 431\n",
      "--- number of probes: 263 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 476\n",
      "--- gene=476, id=49 removed by soft count = 33507\n",
      "--- gene=476, id=65 removed by soft count = 99\n",
      "--- gene=476, id=66 removed by soft count = 576\n",
      "--- gene=476, id=67 removed by soft count = 433\n",
      "--- gene=476, id=68 removed by soft count = 108\n",
      "--- number of probes: 236 , kept by blast: 231 , if remove dups: 231\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 477\n",
      "--- gene=477, id=6 removed by soft count = 397\n",
      "--- gene=477, id=128 removed by soft count = 137\n",
      "--- gene=477, id=175 removed by soft count = 236\n",
      "--- number of probes: 215 , kept by blast: 212 , if remove dups: 212\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 478\n",
      "--- gene=478, id=63 removed by soft count = 63\n",
      "--- gene=478, id=64 removed by soft count = 132\n",
      "--- gene=478, id=78 removed by soft count = 213\n",
      "--- gene=478, id=82 removed by soft count = 64\n",
      "--- gene=478, id=107 removed by soft count = 113\n",
      "--- gene=478, id=148 removed by soft count = 143\n",
      "--- gene=478, id=152 removed by soft count = 58\n",
      "--- gene=478, id=160 removed by soft count = 113\n",
      "--- gene=478, id=165 removed by soft count = 465\n",
      "--- gene=478, id=166 removed by soft count = 69\n",
      "--- gene=478, id=172 removed by soft count = 152\n",
      "--- number of probes: 189 , kept by blast: 178 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 479\n",
      "--- gene=479, id=30 removed by soft count = 388\n",
      "--- gene=479, id=31 removed by soft count = 1230\n",
      "--- number of probes: 287 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 480\n",
      "--- gene=480, id=92 removed by soft count = 206\n",
      "--- gene=480, id=128 removed by soft count = 57\n",
      "--- gene=480, id=130 removed by soft count = 182\n",
      "--- gene=480, id=131 removed by soft count = 129\n",
      "--- gene=480, id=132 removed by soft count = 919\n",
      "--- gene=480, id=133 removed by soft count = 2071\n",
      "--- gene=480, id=227 removed by soft count = 76\n",
      "--- number of probes: 257 , kept by blast: 250 , if remove dups: 250\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 481\n",
      "--- gene=481, id=190 removed by soft count = 85\n",
      "--- gene=481, id=207 removed by soft count = 173\n",
      "--- gene=481, id=268 removed by soft count = 1520\n",
      "--- number of probes: 269 , kept by blast: 266 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 482\n",
      "--- gene=482, id=33 removed by soft count = 159\n",
      "--- gene=482, id=133 removed by soft count = 464\n",
      "--- gene=482, id=159 removed by soft count = 328\n",
      "--- number of probes: 230 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 483\n",
      "--- gene=483, id=7 removed by soft count = 54\n",
      "--- gene=483, id=29 removed by soft count = 66\n",
      "--- gene=483, id=31 removed by soft count = 681\n",
      "--- gene=483, id=32 removed by soft count = 261\n",
      "--- gene=483, id=126 removed by soft count = 876\n",
      "--- gene=483, id=187 removed by soft count = 80\n",
      "--- gene=483, id=200 removed by soft count = 174\n",
      "--- gene=483, id=202 removed by soft count = 52\n",
      "--- gene=483, id=203 removed by soft count = 80\n",
      "--- gene=483, id=208 removed by soft count = 65\n",
      "--- gene=483, id=209 removed by soft count = 76\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=483, id=214 removed by soft count = 61\n",
      "--- gene=483, id=215 removed by soft count = 84\n",
      "--- number of probes: 220 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 484\n",
      "--- gene=484, id=85 removed by soft count = 128\n",
      "--- gene=484, id=154 removed by soft count = 101\n",
      "--- number of probes: 265 , kept by blast: 263 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 485\n",
      "--- gene=485, id=46 removed by soft count = 102\n",
      "--- gene=485, id=48 removed by soft count = 103\n",
      "--- gene=485, id=49 removed by soft count = 73\n",
      "--- gene=485, id=222 removed by soft count = 1779\n",
      "--- number of probes: 254 , kept by blast: 250 , if remove dups: 250\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 486\n",
      "--- gene=486, id=7 removed by soft count = 102\n",
      "--- gene=486, id=43 removed by soft count = 2391\n",
      "--- gene=486, id=137 removed by soft count = 21035\n",
      "--- gene=486, id=188 removed by soft count = 117\n",
      "--- number of probes: 294 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 487\n",
      "--- gene=487, id=202 removed by soft count = 119\n",
      "--- gene=487, id=221 removed by soft count = 105\n",
      "--- gene=487, id=232 removed by soft count = 876\n",
      "--- gene=487, id=235 removed by soft count = 364\n",
      "--- gene=487, id=236 removed by soft count = 473\n",
      "--- gene=487, id=237 removed by soft count = 196\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 489\n",
      "--- gene=489, id=55 removed by soft count = 402\n",
      "--- gene=489, id=116 removed by soft count = 182\n",
      "--- gene=489, id=118 removed by soft count = 144\n",
      "--- gene=489, id=141 removed by soft count = 67\n",
      "--- gene=489, id=142 removed by soft count = 87\n",
      "--- gene=489, id=143 removed by soft count = 84\n",
      "--- gene=489, id=160 removed by soft count = 69\n",
      "--- gene=489, id=167 removed by soft count = 103\n",
      "--- gene=489, id=198 removed by soft count = 69\n",
      "--- gene=489, id=203 removed by soft count = 92\n",
      "--- gene=489, id=206 removed by soft count = 277\n",
      "--- gene=489, id=208 removed by soft count = 267\n",
      "--- gene=489, id=209 removed by soft count = 253\n",
      "--- gene=489, id=211 removed by soft count = 99\n",
      "--- number of probes: 216 , kept by blast: 202 , if remove dups: 201\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 490\n",
      "--- gene=490, id=130 removed by soft count = 186\n",
      "--- number of probes: 293 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 491\n",
      "--- gene=491, id=6 removed by soft count = 82\n",
      "--- gene=491, id=24 removed by soft count = 61\n",
      "--- gene=491, id=98 removed by soft count = 63\n",
      "--- gene=491, id=102 removed by soft count = 164\n",
      "--- gene=491, id=103 removed by soft count = 94\n",
      "--- gene=491, id=134 removed by soft count = 70\n",
      "--- gene=491, id=136 removed by soft count = 160\n",
      "--- gene=491, id=209 removed by soft count = 118\n",
      "--- gene=491, id=227 removed by soft count = 57\n",
      "--- number of probes: 265 , kept by blast: 256 , if remove dups: 256\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 492\n",
      "--- gene=492, id=76 removed by soft count = 204\n",
      "--- gene=492, id=112 removed by soft count = 67\n",
      "--- number of probes: 229 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 493\n",
      "--- gene=493, id=0 removed by soft count = 60\n",
      "--- gene=493, id=1 removed by soft count = 718\n",
      "--- gene=493, id=2 removed by soft count = 182\n",
      "--- gene=493, id=3 removed by soft count = 107\n",
      "--- gene=493, id=4 removed by soft count = 1124\n",
      "--- gene=493, id=15 removed by soft count = 60\n",
      "--- gene=493, id=43 removed by soft count = 1141\n",
      "--- gene=493, id=53 removed by soft count = 185\n",
      "--- gene=493, id=82 removed by soft count = 115\n",
      "--- number of probes: 239 , kept by blast: 230 , if remove dups: 230\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 494\n",
      "--- gene=494, id=0 removed by soft count = 144\n",
      "--- gene=494, id=1 removed by soft count = 143\n",
      "--- gene=494, id=2 removed by soft count = 66\n",
      "--- gene=494, id=7 removed by soft count = 837\n",
      "--- gene=494, id=268 removed by soft count = 164\n",
      "--- gene=494, id=290 removed by soft count = 714\n",
      "--- number of probes: 293 , kept by blast: 287 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 495\n",
      "--- gene=495, id=220 removed by soft count = 143\n",
      "--- gene=495, id=225 removed by soft count = 56\n",
      "--- gene=495, id=226 removed by soft count = 101\n",
      "--- gene=495, id=227 removed by soft count = 100\n",
      "--- gene=495, id=240 removed by soft count = 664\n",
      "--- gene=495, id=241 removed by soft count = 61\n",
      "--- gene=495, id=257 removed by soft count = 186\n",
      "--- gene=495, id=258 removed by soft count = 87\n",
      "--- number of probes: 259 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 496\n",
      "--- gene=496, id=6 removed by soft count = 79\n",
      "--- gene=496, id=24 removed by soft count = 51\n",
      "--- gene=496, id=59 removed by soft count = 414\n",
      "--- gene=496, id=62 removed by soft count = 102\n",
      "--- gene=496, id=82 removed by soft count = 412\n",
      "--- gene=496, id=85 removed by soft count = 296\n",
      "--- gene=496, id=86 removed by soft count = 87\n",
      "--- gene=496, id=88 removed by soft count = 276\n",
      "--- gene=496, id=97 removed by soft count = 80\n",
      "--- gene=496, id=99 removed by soft count = 205\n",
      "--- gene=496, id=100 removed by soft count = 1136\n",
      "--- number of probes: 167 , kept by blast: 156 , if remove dups: 156\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 497\n",
      "--- gene=497, id=0 removed by soft count = 12319\n",
      "--- gene=497, id=1 removed by soft count = 1303\n",
      "--- gene=497, id=3 removed by soft count = 57\n",
      "--- gene=497, id=7 removed by soft count = 177\n",
      "--- gene=497, id=8 removed by soft count = 65\n",
      "--- gene=497, id=9 removed by soft count = 392\n",
      "--- gene=497, id=10 removed by soft count = 93\n",
      "--- gene=497, id=11 removed by soft count = 651\n",
      "--- gene=497, id=175 removed by soft count = 2015\n",
      "--- number of probes: 228 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 498\n",
      "--- number of probes: 262 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 499\n",
      "--- gene=499, id=55 removed by soft count = 271\n",
      "--- gene=499, id=56 removed by soft count = 55\n",
      "--- gene=499, id=58 removed by soft count = 95\n",
      "--- gene=499, id=83 removed by soft count = 69\n",
      "--- gene=499, id=85 removed by soft count = 101\n",
      "--- gene=499, id=91 removed by soft count = 125\n",
      "--- gene=499, id=94 removed by soft count = 67\n",
      "--- gene=499, id=102 removed by soft count = 312\n",
      "--- gene=499, id=107 removed by soft count = 64\n",
      "--- gene=499, id=126 removed by soft count = 1101\n",
      "--- gene=499, id=128 removed by soft count = 274\n",
      "--- gene=499, id=129 removed by soft count = 5042\n",
      "--- number of probes: 162 , kept by blast: 150 , if remove dups: 150\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 500\n",
      "--- gene=500, id=15 removed by soft count = 59\n",
      "--- gene=500, id=32 removed by soft count = 454\n",
      "--- gene=500, id=80 removed by soft count = 157\n",
      "--- gene=500, id=81 removed by soft count = 94\n",
      "--- gene=500, id=100 removed by soft count = 177\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=500, id=108 removed by soft count = 59\n",
      "--- gene=500, id=125 removed by soft count = 1794\n",
      "--- gene=500, id=140 removed by soft count = 215\n",
      "--- gene=500, id=141 removed by soft count = 83\n",
      "--- number of probes: 142 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 501\n",
      "--- gene=501, id=96 removed by soft count = 93\n",
      "--- gene=501, id=293 removed by soft count = 356\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 502\n",
      "--- gene=502, id=58 removed by soft count = 71\n",
      "--- gene=502, id=199 removed by soft count = 65\n",
      "--- gene=502, id=201 removed by soft count = 77\n",
      "--- gene=502, id=251 removed by soft count = 57\n",
      "--- gene=502, id=262 removed by soft count = 129\n",
      "--- number of probes: 269 , kept by blast: 264 , if remove dups: 264\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 503\n",
      "--- gene=503, id=14 removed by soft count = 103\n",
      "--- gene=503, id=34 removed by soft count = 93\n",
      "--- gene=503, id=58 removed by soft count = 325\n",
      "--- gene=503, id=59 removed by soft count = 70\n",
      "--- gene=503, id=143 removed by soft count = 212\n",
      "--- gene=503, id=144 removed by soft count = 195\n",
      "--- gene=503, id=155 removed by soft count = 220\n",
      "--- number of probes: 300 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 504\n",
      "--- gene=504, id=10 removed by soft count = 161\n",
      "--- gene=504, id=12 removed by soft count = 51\n",
      "--- gene=504, id=14 removed by soft count = 413\n",
      "--- gene=504, id=15 removed by soft count = 78\n",
      "--- gene=504, id=16 removed by soft count = 240\n",
      "--- gene=504, id=159 removed by soft count = 189\n",
      "--- gene=504, id=162 removed by soft count = 435\n",
      "--- gene=504, id=164 removed by soft count = 127\n",
      "--- gene=504, id=190 removed by soft count = 79\n",
      "--- gene=504, id=191 removed by soft count = 77\n",
      "--- gene=504, id=192 removed by soft count = 68\n",
      "--- gene=504, id=193 removed by soft count = 276\n",
      "--- gene=504, id=195 removed by soft count = 5308\n",
      "--- number of probes: 220 , kept by blast: 207 , if remove dups: 204\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 505\n",
      "--- gene=505, id=125 removed by soft count = 136\n",
      "--- gene=505, id=249 removed by soft count = 63\n",
      "--- gene=505, id=250 removed by soft count = 125\n",
      "--- gene=505, id=253 removed by soft count = 126\n",
      "--- gene=505, id=267 removed by soft count = 86\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 506\n",
      "--- gene=506, id=215 removed by soft count = 137\n",
      "--- gene=506, id=254 removed by soft count = 146\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 507\n",
      "--- gene=507, id=0 removed by soft count = 139\n",
      "--- gene=507, id=1 removed by soft count = 165\n",
      "--- gene=507, id=9 removed by soft count = 509\n",
      "--- gene=507, id=28 removed by soft count = 136\n",
      "--- gene=507, id=78 removed by soft count = 55\n",
      "--- gene=507, id=80 removed by soft count = 95\n",
      "--- gene=507, id=82 removed by soft count = 228\n",
      "--- gene=507, id=97 removed by soft count = 93\n",
      "--- gene=507, id=105 removed by soft count = 110\n",
      "--- gene=507, id=106 removed by soft count = 704\n",
      "--- gene=507, id=161 removed by soft count = 146\n",
      "--- number of probes: 202 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 508\n",
      "--- gene=508, id=15 removed by soft count = 83\n",
      "--- gene=508, id=36 removed by soft count = 69\n",
      "--- gene=508, id=41 removed by soft count = 67\n",
      "--- gene=508, id=42 removed by soft count = 66\n",
      "--- gene=508, id=46 removed by soft count = 149\n",
      "--- gene=508, id=90 removed by soft count = 98\n",
      "--- gene=508, id=93 removed by soft count = 51\n",
      "--- gene=508, id=134 removed by soft count = 63\n",
      "--- gene=508, id=142 removed by soft count = 53\n",
      "--- gene=508, id=156 removed by soft count = 56\n",
      "--- gene=508, id=170 removed by soft count = 89\n",
      "--- number of probes: 221 , kept by blast: 210 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 509\n",
      "--- gene=509, id=18 removed by soft count = 56\n",
      "--- gene=509, id=19 removed by soft count = 52\n",
      "--- gene=509, id=80 removed by soft count = 1044\n",
      "--- gene=509, id=81 removed by soft count = 3951\n",
      "--- gene=509, id=82 removed by soft count = 57\n",
      "--- gene=509, id=94 removed by soft count = 75\n",
      "--- gene=509, id=96 removed by soft count = 70\n",
      "--- gene=509, id=184 removed by soft count = 126\n",
      "--- gene=509, id=185 removed by soft count = 507\n",
      "--- gene=509, id=206 removed by soft count = 150\n",
      "--- gene=509, id=208 removed by soft count = 57\n",
      "--- number of probes: 272 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 510\n",
      "--- gene=510, id=47 removed by soft count = 518\n",
      "--- gene=510, id=54 removed by soft count = 1440\n",
      "--- gene=510, id=55 removed by soft count = 506\n",
      "--- gene=510, id=79 removed by soft count = 68\n",
      "--- gene=510, id=163 removed by soft count = 224\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 511\n",
      "--- gene=511, id=0 removed by soft count = 51\n",
      "--- gene=511, id=19 removed by soft count = 369\n",
      "--- gene=511, id=21 removed by soft count = 105\n",
      "--- gene=511, id=23 removed by soft count = 237\n",
      "--- gene=511, id=25 removed by soft count = 63\n",
      "--- gene=511, id=26 removed by soft count = 1045\n",
      "--- gene=511, id=28 removed by soft count = 287\n",
      "--- gene=511, id=40 removed by soft count = 123\n",
      "--- gene=511, id=48 removed by soft count = 1475\n",
      "--- gene=511, id=81 removed by soft count = 119\n",
      "--- gene=511, id=85 removed by soft count = 179\n",
      "--- gene=511, id=86 removed by soft count = 231\n",
      "--- gene=511, id=89 removed by soft count = 76\n",
      "--- gene=511, id=90 removed by soft count = 60\n",
      "--- gene=511, id=93 removed by soft count = 147\n",
      "--- gene=511, id=104 removed by soft count = 56\n",
      "--- gene=511, id=113 removed by soft count = 134\n",
      "--- gene=511, id=121 removed by soft count = 75\n",
      "--- gene=511, id=182 removed by soft count = 427\n",
      "--- number of probes: 184 , kept by blast: 165 , if remove dups: 165\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 512\n",
      "--- gene=512, id=6 removed by soft count = 58\n",
      "--- gene=512, id=7 removed by soft count = 88\n",
      "--- gene=512, id=8 removed by soft count = 164\n",
      "--- gene=512, id=41 removed by soft count = 190\n",
      "--- gene=512, id=52 removed by soft count = 436\n",
      "--- gene=512, id=53 removed by soft count = 57\n",
      "--- gene=512, id=101 removed by soft count = 275\n",
      "--- gene=512, id=136 removed by soft count = 99\n",
      "--- gene=512, id=146 removed by soft count = 53\n",
      "--- gene=512, id=168 removed by soft count = 1020\n",
      "--- gene=512, id=185 removed by soft count = 51\n",
      "--- number of probes: 197 , kept by blast: 186 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 516\n",
      "--- gene=516, id=2 removed by soft count = 353\n",
      "--- gene=516, id=3 removed by soft count = 676\n",
      "--- gene=516, id=21 removed by soft count = 219\n",
      "--- gene=516, id=79 removed by soft count = 138\n",
      "--- gene=516, id=81 removed by soft count = 135\n",
      "--- gene=516, id=107 removed by soft count = 216\n",
      "--- gene=516, id=222 removed by soft count = 56\n",
      "--- gene=516, id=226 removed by soft count = 52\n",
      "--- number of probes: 227 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 517\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=517, id=104 removed by soft count = 232\n",
      "--- gene=517, id=136 removed by soft count = 80\n",
      "--- gene=517, id=157 removed by soft count = 153\n",
      "--- gene=517, id=168 removed by soft count = 233\n",
      "--- gene=517, id=169 removed by soft count = 51\n",
      "--- gene=517, id=170 removed by soft count = 130\n",
      "--- gene=517, id=171 removed by soft count = 56\n",
      "--- number of probes: 248 , kept by blast: 241 , if remove dups: 241\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 518\n",
      "--- gene=518, id=55 removed by soft count = 60\n",
      "--- gene=518, id=65 removed by soft count = 220\n",
      "--- gene=518, id=66 removed by soft count = 115\n",
      "--- gene=518, id=111 removed by soft count = 240\n",
      "--- gene=518, id=112 removed by soft count = 223\n",
      "--- gene=518, id=153 removed by soft count = 199\n",
      "--- gene=518, id=156 removed by soft count = 56\n",
      "--- gene=518, id=162 removed by soft count = 94\n",
      "--- gene=518, id=165 removed by soft count = 133\n",
      "--- gene=518, id=179 removed by soft count = 180\n",
      "--- gene=518, id=182 removed by soft count = 691\n",
      "--- number of probes: 264 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 519\n",
      "--- gene=519, id=67 removed by soft count = 181\n",
      "--- gene=519, id=68 removed by soft count = 22325\n",
      "--- gene=519, id=117 removed by soft count = 126\n",
      "--- gene=519, id=129 removed by soft count = 93\n",
      "--- gene=519, id=130 removed by soft count = 194\n",
      "--- gene=519, id=131 removed by soft count = 153\n",
      "--- gene=519, id=132 removed by soft count = 322\n",
      "--- gene=519, id=134 removed by soft count = 274\n",
      "--- gene=519, id=135 removed by soft count = 178\n",
      "--- gene=519, id=137 removed by soft count = 944\n",
      "--- gene=519, id=138 removed by soft count = 1336\n",
      "--- number of probes: 188 , kept by blast: 177 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 520\n",
      "--- gene=520, id=187 removed by soft count = 5860\n",
      "--- number of probes: 212 , kept by blast: 211 , if remove dups: 211\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 521\n",
      "--- number of probes: 265 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 522\n",
      "--- gene=522, id=0 removed by soft count = 260\n",
      "--- gene=522, id=33 removed by soft count = 118\n",
      "--- gene=522, id=49 removed by soft count = 196\n",
      "--- gene=522, id=136 removed by soft count = 130\n",
      "--- gene=522, id=172 removed by soft count = 72\n",
      "--- gene=522, id=174 removed by soft count = 65\n",
      "--- gene=522, id=178 removed by soft count = 70\n",
      "--- number of probes: 187 , kept by blast: 180 , if remove dups: 180\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 523\n",
      "--- gene=523, id=134 removed by soft count = 85\n",
      "--- number of probes: 152 , kept by blast: 151 , if remove dups: 151\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 524\n",
      "--- gene=524, id=149 removed by soft count = 152\n",
      "--- gene=524, id=152 removed by soft count = 122\n",
      "--- gene=524, id=155 removed by soft count = 78\n",
      "--- gene=524, id=157 removed by soft count = 67\n",
      "--- number of probes: 196 , kept by blast: 192 , if remove dups: 192\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 525\n",
      "--- gene=525, id=54 removed by soft count = 75\n",
      "--- gene=525, id=105 removed by soft count = 73\n",
      "--- gene=525, id=124 removed by soft count = 94\n",
      "--- gene=525, id=126 removed by soft count = 65\n",
      "--- gene=525, id=127 removed by soft count = 329\n",
      "--- number of probes: 216 , kept by blast: 211 , if remove dups: 211\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 526\n",
      "--- gene=526, id=134 removed by soft count = 190\n",
      "--- gene=526, id=150 removed by soft count = 163\n",
      "--- gene=526, id=151 removed by soft count = 9041\n",
      "--- gene=526, id=152 removed by soft count = 3819\n",
      "--- gene=526, id=153 removed by soft count = 524\n",
      "--- gene=526, id=154 removed by soft count = 272\n",
      "--- number of probes: 223 , kept by blast: 217 , if remove dups: 217\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 527\n",
      "--- gene=527, id=54 removed by soft count = 55\n",
      "--- gene=527, id=66 removed by soft count = 69\n",
      "--- gene=527, id=67 removed by soft count = 146\n",
      "--- gene=527, id=93 removed by soft count = 73\n",
      "--- gene=527, id=210 removed by soft count = 64\n",
      "--- gene=527, id=211 removed by soft count = 138\n",
      "--- gene=527, id=252 removed by soft count = 1998\n",
      "--- gene=527, id=254 removed by soft count = 487\n",
      "--- number of probes: 255 , kept by blast: 247 , if remove dups: 247\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 528\n",
      "--- gene=528, id=41 removed by soft count = 111\n",
      "--- gene=528, id=81 removed by soft count = 54\n",
      "--- gene=528, id=217 removed by soft count = 68\n",
      "--- number of probes: 249 , kept by blast: 246 , if remove dups: 246\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 529\n",
      "--- gene=529, id=122 removed by soft count = 80\n",
      "--- gene=529, id=149 removed by soft count = 74\n",
      "--- gene=529, id=150 removed by soft count = 166\n",
      "--- gene=529, id=152 removed by soft count = 91\n",
      "--- number of probes: 269 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 530\n",
      "--- gene=530, id=92 removed by soft count = 276\n",
      "--- gene=530, id=149 removed by soft count = 1775\n",
      "--- gene=530, id=150 removed by soft count = 92\n",
      "--- gene=530, id=200 removed by soft count = 377\n",
      "--- number of probes: 223 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 531\n",
      "--- number of probes: 201 , kept by blast: 201 , if remove dups: 201\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 532\n",
      "--- gene=532, id=1 removed by soft count = 56\n",
      "--- gene=532, id=2 removed by soft count = 63\n",
      "--- gene=532, id=178 removed by soft count = 52\n",
      "--- gene=532, id=195 removed by soft count = 144\n",
      "--- gene=532, id=196 removed by soft count = 301\n",
      "--- gene=532, id=198 removed by soft count = 251\n",
      "--- gene=532, id=201 removed by soft count = 155\n",
      "--- gene=532, id=202 removed by soft count = 105\n",
      "--- gene=532, id=203 removed by soft count = 148\n",
      "--- number of probes: 232 , kept by blast: 223 , if remove dups: 223\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 533\n",
      "--- gene=533, id=6 removed by soft count = 270\n",
      "--- gene=533, id=41 removed by soft count = 138\n",
      "--- gene=533, id=42 removed by soft count = 192\n",
      "--- gene=533, id=45 removed by soft count = 209\n",
      "--- gene=533, id=46 removed by soft count = 75\n",
      "--- gene=533, id=51 removed by soft count = 107\n",
      "--- gene=533, id=53 removed by soft count = 98\n",
      "--- gene=533, id=54 removed by soft count = 54\n",
      "--- gene=533, id=62 removed by soft count = 103\n",
      "--- gene=533, id=64 removed by soft count = 54\n",
      "--- gene=533, id=66 removed by soft count = 59\n",
      "--- gene=533, id=68 removed by soft count = 61\n",
      "--- gene=533, id=127 removed by soft count = 1119\n",
      "--- gene=533, id=169 removed by soft count = 59\n",
      "--- gene=533, id=173 removed by soft count = 121\n",
      "--- gene=533, id=174 removed by soft count = 165\n",
      "--- gene=533, id=175 removed by soft count = 154\n",
      "--- gene=533, id=176 removed by soft count = 58\n",
      "--- gene=533, id=177 removed by soft count = 233\n",
      "--- gene=533, id=178 removed by soft count = 4482\n",
      "--- gene=533, id=179 removed by soft count = 761\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 201 , kept by blast: 180 , if remove dups: 180\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 534\n",
      "--- gene=534, id=73 removed by soft count = 561\n",
      "--- gene=534, id=74 removed by soft count = 443\n",
      "--- gene=534, id=75 removed by soft count = 176\n",
      "--- gene=534, id=76 removed by soft count = 7913\n",
      "--- number of probes: 181 , kept by blast: 177 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 535\n",
      "--- gene=535, id=20 removed by soft count = 89\n",
      "--- gene=535, id=23 removed by soft count = 73\n",
      "--- gene=535, id=24 removed by soft count = 56\n",
      "--- gene=535, id=32 removed by soft count = 54\n",
      "--- number of probes: 116 , kept by blast: 112 , if remove dups: 112\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 112\n",
      "-- checking probes in region: 536\n",
      "--- gene=536, id=22 removed by soft count = 155\n",
      "--- gene=536, id=73 removed by soft count = 312\n",
      "--- gene=536, id=76 removed by soft count = 382\n",
      "--- gene=536, id=97 removed by soft count = 324\n",
      "--- gene=536, id=98 removed by soft count = 348\n",
      "--- gene=536, id=99 removed by soft count = 18666\n",
      "--- gene=536, id=242 removed by soft count = 77\n",
      "--- number of probes: 259 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 537\n",
      "--- gene=537, id=0 removed by soft count = 396\n",
      "--- gene=537, id=1 removed by soft count = 76\n",
      "--- gene=537, id=2 removed by soft count = 77\n",
      "--- gene=537, id=4 removed by soft count = 105\n",
      "--- gene=537, id=94 removed by soft count = 126\n",
      "--- gene=537, id=165 removed by soft count = 66\n",
      "--- number of probes: 231 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 538\n",
      "--- gene=538, id=116 removed by soft count = 82\n",
      "--- gene=538, id=132 removed by soft count = 61\n",
      "--- gene=538, id=133 removed by soft count = 68\n",
      "--- gene=538, id=135 removed by soft count = 93\n",
      "--- gene=538, id=139 removed by soft count = 450\n",
      "--- gene=538, id=140 removed by soft count = 122\n",
      "--- gene=538, id=144 removed by soft count = 135\n",
      "--- gene=538, id=204 removed by soft count = 306\n",
      "--- number of probes: 214 , kept by blast: 206 , if remove dups: 206\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 539\n",
      "--- gene=539, id=18 removed by soft count = 71\n",
      "--- gene=539, id=28 removed by soft count = 104\n",
      "--- gene=539, id=30 removed by soft count = 59\n",
      "--- gene=539, id=229 removed by soft count = 1088\n",
      "--- gene=539, id=230 removed by soft count = 96\n",
      "--- number of probes: 242 , kept by blast: 237 , if remove dups: 237\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 540\n",
      "--- gene=540, id=42 removed by soft count = 464\n",
      "--- gene=540, id=43 removed by soft count = 81\n",
      "--- gene=540, id=45 removed by soft count = 348\n",
      "--- gene=540, id=61 removed by soft count = 80\n",
      "--- gene=540, id=63 removed by soft count = 105\n",
      "--- gene=540, id=64 removed by soft count = 76\n",
      "--- gene=540, id=66 removed by soft count = 94\n",
      "--- gene=540, id=71 removed by soft count = 73\n",
      "--- gene=540, id=109 removed by soft count = 144\n",
      "--- number of probes: 179 , kept by blast: 170 , if remove dups: 170\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 541\n",
      "--- gene=541, id=173 removed by soft count = 193\n",
      "--- gene=541, id=225 removed by soft count = 128\n",
      "--- gene=541, id=226 removed by soft count = 54\n",
      "--- gene=541, id=244 removed by soft count = 65\n",
      "--- number of probes: 248 , kept by blast: 244 , if remove dups: 244\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 542\n",
      "--- gene=542, id=53 removed by soft count = 120\n",
      "--- gene=542, id=67 removed by hard count = 3\n",
      "--- gene=542, id=184 removed by soft count = 123\n",
      "--- gene=542, id=188 removed by soft count = 109\n",
      "--- number of probes: 227 , kept by blast: 223 , if remove dups: 222\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 543\n",
      "--- gene=543, id=5 removed by soft count = 66\n",
      "--- gene=543, id=235 removed by soft count = 7265\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 544\n",
      "--- gene=544, id=3 removed by soft count = 102\n",
      "--- gene=544, id=13 removed by soft count = 109\n",
      "--- gene=544, id=15 removed by soft count = 221\n",
      "--- gene=544, id=36 removed by soft count = 426\n",
      "--- gene=544, id=164 removed by soft count = 697\n",
      "--- gene=544, id=178 removed by soft count = 617\n",
      "--- gene=544, id=183 removed by soft count = 887\n",
      "--- gene=544, id=258 removed by soft count = 58\n",
      "--- gene=544, id=259 removed by soft count = 231\n",
      "--- gene=544, id=272 removed by soft count = 54\n",
      "--- gene=544, id=273 removed by soft count = 86\n",
      "--- number of probes: 276 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 545\n",
      "--- gene=545, id=156 removed by soft count = 451\n",
      "--- gene=545, id=182 removed by soft count = 394\n",
      "--- number of probes: 267 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 546\n",
      "--- gene=546, id=126 removed by soft count = 316\n",
      "--- gene=546, id=127 removed by soft count = 171\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 548\n",
      "--- gene=548, id=46 removed by soft count = 427\n",
      "--- gene=548, id=59 removed by soft count = 969\n",
      "--- gene=548, id=60 removed by soft count = 530\n",
      "--- gene=548, id=61 removed by soft count = 720\n",
      "--- gene=548, id=62 removed by soft count = 565\n",
      "--- gene=548, id=63 removed by soft count = 221\n",
      "--- gene=548, id=64 removed by soft count = 840\n",
      "--- gene=548, id=65 removed by soft count = 1103\n",
      "--- gene=548, id=150 removed by soft count = 58\n",
      "--- gene=548, id=151 removed by soft count = 428\n",
      "--- gene=548, id=152 removed by soft count = 272\n",
      "--- number of probes: 191 , kept by blast: 180 , if remove dups: 180\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 549\n",
      "--- gene=549, id=47 removed by soft count = 113\n",
      "--- gene=549, id=48 removed by soft count = 2229\n",
      "--- gene=549, id=50 removed by soft count = 862\n",
      "--- gene=549, id=53 removed by soft count = 143\n",
      "--- gene=549, id=105 removed by soft count = 123\n",
      "--- gene=549, id=108 removed by soft count = 125\n",
      "--- gene=549, id=147 removed by soft count = 93\n",
      "--- gene=549, id=151 removed by soft count = 974\n",
      "--- gene=549, id=173 removed by soft count = 114\n",
      "--- gene=549, id=175 removed by soft count = 175\n",
      "--- number of probes: 206 , kept by blast: 196 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 550\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 551\n",
      "--- number of probes: 90 , kept by blast: 90 , if remove dups: 88\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 88\n",
      "-- checking probes in region: 552\n",
      "--- gene=552, id=34 removed by soft count = 56\n",
      "--- gene=552, id=37 removed by soft count = 63\n",
      "--- gene=552, id=39 removed by hard count = 0\n",
      "--- gene=552, id=87 removed by soft count = 258\n",
      "--- gene=552, id=89 removed by soft count = 52\n",
      "--- gene=552, id=90 removed by soft count = 1019\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=552, id=94 removed by soft count = 84\n",
      "--- gene=552, id=95 removed by soft count = 152\n",
      "--- gene=552, id=96 removed by soft count = 218\n",
      "--- gene=552, id=169 removed by soft count = 55\n",
      "--- gene=552, id=209 removed by soft count = 61\n",
      "--- number of probes: 264 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 554\n",
      "--- gene=554, id=14 removed by soft count = 418\n",
      "--- gene=554, id=89 removed by soft count = 81\n",
      "--- gene=554, id=91 removed by soft count = 57\n",
      "--- gene=554, id=93 removed by soft count = 128\n",
      "--- gene=554, id=270 removed by soft count = 62\n",
      "--- gene=554, id=273 removed by soft count = 147\n",
      "--- gene=554, id=281 removed by soft count = 423\n",
      "--- number of probes: 300 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 555\n",
      "--- gene=555, id=49 removed by soft count = 190\n",
      "--- gene=555, id=113 removed by soft count = 383\n",
      "--- gene=555, id=136 removed by soft count = 318\n",
      "--- gene=555, id=141 removed by soft count = 212\n",
      "--- gene=555, id=142 removed by soft count = 301\n",
      "--- gene=555, id=171 removed by soft count = 166\n",
      "--- gene=555, id=172 removed by soft count = 84\n",
      "--- gene=555, id=175 removed by soft count = 170\n",
      "--- gene=555, id=203 removed by soft count = 178\n",
      "--- gene=555, id=204 removed by soft count = 99\n",
      "--- gene=555, id=211 removed by soft count = 1090\n",
      "--- gene=555, id=212 removed by soft count = 229\n",
      "--- gene=555, id=213 removed by soft count = 313\n",
      "--- gene=555, id=234 removed by soft count = 253\n",
      "--- gene=555, id=235 removed by soft count = 77\n",
      "--- gene=555, id=236 removed by soft count = 267\n",
      "--- number of probes: 241 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 556\n",
      "--- gene=556, id=34 removed by soft count = 69\n",
      "--- gene=556, id=42 removed by soft count = 56\n",
      "--- gene=556, id=44 removed by soft count = 64\n",
      "--- gene=556, id=87 removed by soft count = 349\n",
      "--- number of probes: 234 , kept by blast: 230 , if remove dups: 230\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 557\n",
      "--- gene=557, id=47 removed by soft count = 200\n",
      "--- gene=557, id=57 removed by soft count = 675\n",
      "--- gene=557, id=58 removed by soft count = 323\n",
      "--- gene=557, id=161 removed by soft count = 433\n",
      "--- gene=557, id=162 removed by soft count = 119\n",
      "--- number of probes: 273 , kept by blast: 268 , if remove dups: 268\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 558\n",
      "--- gene=558, id=12 removed by soft count = 64\n",
      "--- gene=558, id=117 removed by soft count = 807\n",
      "--- gene=558, id=120 removed by soft count = 239\n",
      "--- gene=558, id=121 removed by soft count = 407\n",
      "--- gene=558, id=131 removed by soft count = 224\n",
      "--- gene=558, id=156 removed by soft count = 64\n",
      "--- gene=558, id=213 removed by soft count = 52\n",
      "--- gene=558, id=234 removed by soft count = 81\n",
      "--- gene=558, id=250 removed by soft count = 980\n",
      "--- number of probes: 251 , kept by blast: 242 , if remove dups: 242\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 559\n",
      "--- gene=559, id=80 removed by soft count = 52\n",
      "--- gene=559, id=86 removed by soft count = 96\n",
      "--- gene=559, id=113 removed by soft count = 1746\n",
      "--- gene=559, id=217 removed by soft count = 87\n",
      "--- number of probes: 221 , kept by blast: 217 , if remove dups: 217\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 560\n",
      "--- gene=560, id=104 removed by soft count = 409\n",
      "--- gene=560, id=140 removed by soft count = 65\n",
      "--- gene=560, id=199 removed by soft count = 62\n",
      "--- gene=560, id=216 removed by soft count = 129\n",
      "--- number of probes: 261 , kept by blast: 257 , if remove dups: 250\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 561\n",
      "--- gene=561, id=39 removed by soft count = 1343\n",
      "--- gene=561, id=41 removed by soft count = 8743\n",
      "--- gene=561, id=42 removed by soft count = 304\n",
      "--- gene=561, id=171 removed by soft count = 60\n",
      "--- number of probes: 240 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 562\n",
      "--- gene=562, id=24 removed by soft count = 1444\n",
      "--- gene=562, id=137 removed by soft count = 54\n",
      "--- gene=562, id=139 removed by soft count = 53\n",
      "--- gene=562, id=182 removed by soft count = 102\n",
      "--- gene=562, id=234 removed by soft count = 67\n",
      "--- number of probes: 249 , kept by blast: 244 , if remove dups: 244\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 563\n",
      "--- gene=563, id=101 removed by soft count = 96\n",
      "--- gene=563, id=111 removed by soft count = 352\n",
      "--- gene=563, id=192 removed by soft count = 170\n",
      "--- number of probes: 193 , kept by blast: 190 , if remove dups: 190\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 564\n",
      "--- gene=564, id=225 removed by soft count = 67\n",
      "--- gene=564, id=226 removed by soft count = 52\n",
      "--- number of probes: 270 , kept by blast: 268 , if remove dups: 268\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 565\n",
      "--- gene=565, id=0 removed by soft count = 811\n",
      "--- gene=565, id=7 removed by soft count = 558\n",
      "--- gene=565, id=45 removed by soft count = 103\n",
      "--- gene=565, id=152 removed by soft count = 320\n",
      "--- gene=565, id=153 removed by soft count = 83\n",
      "--- gene=565, id=157 removed by soft count = 95\n",
      "--- gene=565, id=186 removed by soft count = 67\n",
      "--- gene=565, id=197 removed by soft count = 419\n",
      "--- gene=565, id=207 removed by soft count = 789\n",
      "--- gene=565, id=208 removed by soft count = 67\n",
      "--- gene=565, id=211 removed by soft count = 323\n",
      "--- number of probes: 218 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 566\n",
      "--- gene=566, id=2 removed by soft count = 82\n",
      "--- gene=566, id=53 removed by soft count = 283\n",
      "--- gene=566, id=103 removed by soft count = 374\n",
      "--- gene=566, id=104 removed by soft count = 891\n",
      "--- gene=566, id=105 removed by soft count = 196\n",
      "--- gene=566, id=106 removed by soft count = 440\n",
      "--- gene=566, id=107 removed by soft count = 56\n",
      "--- gene=566, id=108 removed by soft count = 76\n",
      "--- gene=566, id=109 removed by soft count = 277\n",
      "--- gene=566, id=135 removed by soft count = 65\n",
      "--- gene=566, id=161 removed by soft count = 218\n",
      "--- gene=566, id=162 removed by soft count = 172\n",
      "--- number of probes: 169 , kept by blast: 157 , if remove dups: 157\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 567\n",
      "--- gene=567, id=87 removed by soft count = 75\n",
      "--- gene=567, id=135 removed by soft count = 71\n",
      "--- gene=567, id=136 removed by soft count = 97\n",
      "--- gene=567, id=182 removed by soft count = 152\n",
      "--- gene=567, id=183 removed by soft count = 99\n",
      "--- gene=567, id=185 removed by soft count = 83\n",
      "--- gene=567, id=186 removed by soft count = 68\n",
      "--- gene=567, id=201 removed by soft count = 66\n",
      "--- gene=567, id=202 removed by soft count = 86\n",
      "--- gene=567, id=205 removed by soft count = 52\n",
      "--- gene=567, id=206 removed by soft count = 128\n",
      "--- gene=567, id=212 removed by soft count = 174\n",
      "--- gene=567, id=215 removed by soft count = 97\n",
      "--- gene=567, id=218 removed by soft count = 90\n",
      "--- number of probes: 229 , kept by blast: 215 , if remove dups: 215\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 568\n",
      "--- gene=568, id=13 removed by soft count = 225\n",
      "--- gene=568, id=34 removed by soft count = 69\n",
      "--- gene=568, id=36 removed by soft count = 255\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=568, id=38 removed by soft count = 145\n",
      "--- gene=568, id=39 removed by soft count = 229\n",
      "--- gene=568, id=43 removed by soft count = 71\n",
      "--- gene=568, id=45 removed by soft count = 73\n",
      "--- gene=568, id=87 removed by soft count = 99\n",
      "--- gene=568, id=183 removed by soft count = 770\n",
      "--- gene=568, id=215 removed by soft count = 87\n",
      "--- gene=568, id=217 removed by soft count = 316\n",
      "--- number of probes: 222 , kept by blast: 211 , if remove dups: 211\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 569\n",
      "--- gene=569, id=38 removed by soft count = 283\n",
      "--- gene=569, id=45 removed by soft count = 54\n",
      "--- gene=569, id=49 removed by soft count = 125\n",
      "--- gene=569, id=51 removed by soft count = 170\n",
      "--- gene=569, id=66 removed by soft count = 54\n",
      "--- gene=569, id=96 removed by soft count = 296\n",
      "--- gene=569, id=199 removed by soft count = 1031\n",
      "--- gene=569, id=221 removed by soft count = 62\n",
      "--- number of probes: 292 , kept by blast: 284 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 570\n",
      "--- gene=570, id=1 removed by soft count = 58\n",
      "--- gene=570, id=2 removed by soft count = 901\n",
      "--- gene=570, id=3 removed by soft count = 256\n",
      "--- gene=570, id=5 removed by soft count = 388\n",
      "--- gene=570, id=6 removed by soft count = 599\n",
      "--- gene=570, id=8 removed by soft count = 140\n",
      "--- gene=570, id=9 removed by soft count = 86\n",
      "--- gene=570, id=10 removed by soft count = 242\n",
      "--- gene=570, id=11 removed by soft count = 63\n",
      "--- gene=570, id=14 removed by soft count = 64\n",
      "--- gene=570, id=16 removed by soft count = 1310\n",
      "--- gene=570, id=18 removed by soft count = 108\n",
      "--- gene=570, id=19 removed by soft count = 329\n",
      "--- gene=570, id=25 removed by soft count = 127\n",
      "--- gene=570, id=26 removed by soft count = 369\n",
      "--- gene=570, id=27 removed by soft count = 112\n",
      "--- gene=570, id=30 removed by soft count = 138\n",
      "--- gene=570, id=32 removed by soft count = 181\n",
      "--- gene=570, id=33 removed by soft count = 88\n",
      "--- gene=570, id=35 removed by soft count = 98\n",
      "--- gene=570, id=38 removed by soft count = 79\n",
      "--- gene=570, id=41 removed by soft count = 517\n",
      "--- gene=570, id=42 removed by soft count = 96\n",
      "--- gene=570, id=43 removed by soft count = 64\n",
      "--- gene=570, id=44 removed by soft count = 128\n",
      "--- gene=570, id=45 removed by soft count = 661\n",
      "--- gene=570, id=48 removed by soft count = 150\n",
      "--- gene=570, id=49 removed by soft count = 56\n",
      "--- gene=570, id=50 removed by soft count = 191\n",
      "--- gene=570, id=51 removed by soft count = 73\n",
      "--- gene=570, id=52 removed by soft count = 101\n",
      "--- gene=570, id=53 removed by soft count = 331\n",
      "--- gene=570, id=54 removed by soft count = 147\n",
      "--- gene=570, id=55 removed by soft count = 1233\n",
      "--- gene=570, id=56 removed by soft count = 67\n",
      "--- gene=570, id=57 removed by soft count = 661\n",
      "--- gene=570, id=58 removed by soft count = 78\n",
      "--- gene=570, id=59 removed by soft count = 59\n",
      "--- gene=570, id=60 removed by soft count = 796\n",
      "--- gene=570, id=61 removed by soft count = 574\n",
      "--- gene=570, id=62 removed by soft count = 57\n",
      "--- gene=570, id=65 removed by soft count = 292\n",
      "--- gene=570, id=68 removed by soft count = 100\n",
      "--- gene=570, id=69 removed by soft count = 287\n",
      "--- gene=570, id=70 removed by soft count = 444\n",
      "--- gene=570, id=71 removed by soft count = 580\n",
      "--- gene=570, id=72 removed by soft count = 253\n",
      "--- gene=570, id=74 removed by soft count = 527\n",
      "--- gene=570, id=76 removed by soft count = 148\n",
      "--- gene=570, id=78 removed by soft count = 110\n",
      "--- gene=570, id=79 removed by soft count = 354\n",
      "--- gene=570, id=81 removed by soft count = 74\n",
      "--- gene=570, id=83 removed by soft count = 180\n",
      "--- gene=570, id=86 removed by soft count = 86\n",
      "--- gene=570, id=90 removed by soft count = 1107\n",
      "--- gene=570, id=93 removed by soft count = 416\n",
      "--- gene=570, id=97 removed by soft count = 79\n",
      "--- gene=570, id=101 removed by soft count = 52\n",
      "--- number of probes: 104 , kept by blast: 46 , if remove dups: 46\n",
      "-- number of probes kept for this region: 46\n",
      "-- checking probes in region: 571\n",
      "--- gene=571, id=6 removed by soft count = 64\n",
      "--- gene=571, id=17 removed by soft count = 114\n",
      "--- gene=571, id=22 removed by soft count = 56\n",
      "--- gene=571, id=28 removed by soft count = 101\n",
      "--- gene=571, id=34 removed by soft count = 54\n",
      "--- gene=571, id=35 removed by soft count = 174\n",
      "--- gene=571, id=45 removed by soft count = 114\n",
      "--- gene=571, id=53 removed by soft count = 56\n",
      "--- gene=571, id=55 removed by soft count = 288\n",
      "--- gene=571, id=58 removed by soft count = 127\n",
      "--- gene=571, id=59 removed by soft count = 115\n",
      "--- gene=571, id=60 removed by soft count = 73\n",
      "--- gene=571, id=61 removed by soft count = 218\n",
      "--- gene=571, id=64 removed by soft count = 95\n",
      "--- gene=571, id=66 removed by soft count = 90\n",
      "--- gene=571, id=68 removed by soft count = 145\n",
      "--- gene=571, id=76 removed by soft count = 184\n",
      "--- gene=571, id=77 removed by soft count = 121\n",
      "--- gene=571, id=87 removed by soft count = 167\n",
      "--- gene=571, id=88 removed by soft count = 92\n",
      "--- gene=571, id=101 removed by soft count = 95\n",
      "--- gene=571, id=113 removed by soft count = 190\n",
      "--- gene=571, id=119 removed by soft count = 229\n",
      "--- gene=571, id=120 removed by soft count = 77\n",
      "--- gene=571, id=123 removed by soft count = 79\n",
      "--- gene=571, id=124 removed by soft count = 103\n",
      "--- gene=571, id=129 removed by soft count = 83\n",
      "--- gene=571, id=132 removed by soft count = 160\n",
      "--- gene=571, id=139 removed by soft count = 62\n",
      "--- gene=571, id=145 removed by soft count = 77\n",
      "--- gene=571, id=148 removed by soft count = 78\n",
      "--- gene=571, id=151 removed by soft count = 122\n",
      "--- gene=571, id=156 removed by soft count = 120\n",
      "--- gene=571, id=160 removed by soft count = 87\n",
      "--- gene=571, id=162 removed by soft count = 58\n",
      "--- gene=571, id=170 removed by soft count = 55\n",
      "--- gene=571, id=178 removed by soft count = 124\n",
      "--- gene=571, id=182 removed by soft count = 126\n",
      "--- gene=571, id=183 removed by soft count = 110\n",
      "--- gene=571, id=184 removed by soft count = 176\n",
      "--- gene=571, id=187 removed by soft count = 94\n",
      "--- gene=571, id=189 removed by soft count = 187\n",
      "--- gene=571, id=199 removed by soft count = 63\n",
      "--- gene=571, id=203 removed by soft count = 87\n",
      "--- gene=571, id=207 removed by soft count = 93\n",
      "--- gene=571, id=211 removed by soft count = 66\n",
      "--- gene=571, id=215 removed by soft count = 117\n",
      "--- gene=571, id=225 removed by soft count = 85\n",
      "--- gene=571, id=229 removed by soft count = 179\n",
      "--- gene=571, id=231 removed by soft count = 149\n",
      "--- gene=571, id=234 removed by soft count = 78\n",
      "--- gene=571, id=240 removed by soft count = 88\n",
      "--- gene=571, id=249 removed by soft count = 146\n",
      "--- gene=571, id=252 removed by soft count = 76\n",
      "--- gene=571, id=255 removed by soft count = 154\n",
      "--- gene=571, id=257 removed by soft count = 73\n",
      "--- gene=571, id=258 removed by soft count = 63\n",
      "--- gene=571, id=264 removed by soft count = 74\n",
      "--- gene=571, id=270 removed by soft count = 73\n",
      "--- gene=571, id=275 removed by soft count = 198\n",
      "--- gene=571, id=277 removed by soft count = 241\n",
      "--- gene=571, id=279 removed by soft count = 105\n",
      "--- gene=571, id=287 removed by soft count = 149\n",
      "--- gene=571, id=289 removed by soft count = 122\n",
      "--- gene=571, id=290 removed by soft count = 70\n",
      "--- gene=571, id=291 removed by soft count = 196\n",
      "--- gene=571, id=292 removed by soft count = 74\n",
      "--- gene=571, id=299 removed by soft count = 345\n",
      "--- number of probes: 300 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 572\n",
      "--- gene=572, id=14 removed by soft count = 208\n",
      "--- gene=572, id=15 removed by soft count = 62\n",
      "--- gene=572, id=107 removed by soft count = 105\n",
      "--- gene=572, id=123 removed by soft count = 77\n",
      "--- gene=572, id=125 removed by soft count = 69\n",
      "--- gene=572, id=126 removed by soft count = 72\n",
      "--- gene=572, id=133 removed by soft count = 53\n",
      "--- gene=572, id=138 removed by soft count = 154\n",
      "--- gene=572, id=147 removed by soft count = 77\n",
      "--- gene=572, id=194 removed by soft count = 52\n",
      "--- gene=572, id=243 removed by soft count = 113\n",
      "--- gene=572, id=252 removed by soft count = 51\n",
      "--- gene=572, id=253 removed by soft count = 55\n",
      "--- gene=572, id=256 removed by soft count = 71\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 280 , kept by blast: 266 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 573\n",
      "--- gene=573, id=86 removed by soft count = 302\n",
      "--- gene=573, id=116 removed by soft count = 1529\n",
      "--- gene=573, id=192 removed by soft count = 110\n",
      "--- gene=573, id=202 removed by soft count = 125\n",
      "--- number of probes: 206 , kept by blast: 202 , if remove dups: 202\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 574\n",
      "--- gene=574, id=1 removed by soft count = 94\n",
      "--- gene=574, id=2 removed by soft count = 79\n",
      "--- gene=574, id=3 removed by soft count = 51\n",
      "--- gene=574, id=9 removed by soft count = 91\n",
      "--- gene=574, id=15 removed by soft count = 62\n",
      "--- gene=574, id=113 removed by soft count = 128\n",
      "--- gene=574, id=168 removed by soft count = 169\n",
      "--- gene=574, id=215 removed by soft count = 161\n",
      "--- gene=574, id=216 removed by soft count = 143\n",
      "--- gene=574, id=231 removed by soft count = 114\n",
      "--- gene=574, id=232 removed by soft count = 88\n",
      "--- gene=574, id=246 removed by soft count = 79\n",
      "--- gene=574, id=250 removed by soft count = 101\n",
      "--- gene=574, id=254 removed by soft count = 77\n",
      "--- gene=574, id=256 removed by soft count = 178\n",
      "--- number of probes: 258 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 575\n",
      "--- gene=575, id=1 removed by soft count = 138\n",
      "--- gene=575, id=2 removed by soft count = 178\n",
      "--- gene=575, id=52 removed by soft count = 63\n",
      "--- gene=575, id=104 removed by soft count = 116\n",
      "--- gene=575, id=143 removed by soft count = 57\n",
      "--- gene=575, id=169 removed by soft count = 255\n",
      "--- gene=575, id=200 removed by soft count = 238\n",
      "--- number of probes: 204 , kept by blast: 197 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 576\n",
      "--- gene=576, id=40 removed by soft count = 64\n",
      "--- gene=576, id=215 removed by soft count = 96\n",
      "--- gene=576, id=257 removed by soft count = 820\n",
      "--- gene=576, id=258 removed by soft count = 354\n",
      "--- number of probes: 291 , kept by blast: 287 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 577\n",
      "--- gene=577, id=41 removed by soft count = 78\n",
      "--- gene=577, id=86 removed by soft count = 283\n",
      "--- gene=577, id=89 removed by soft count = 776\n",
      "--- gene=577, id=105 removed by soft count = 70\n",
      "--- gene=577, id=170 removed by soft count = 209\n",
      "--- gene=577, id=174 removed by soft count = 57\n",
      "--- gene=577, id=201 removed by soft count = 126\n",
      "--- gene=577, id=203 removed by soft count = 53\n",
      "--- number of probes: 207 , kept by blast: 199 , if remove dups: 199\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 578\n",
      "--- gene=578, id=210 removed by soft count = 62\n",
      "--- gene=578, id=211 removed by soft count = 121\n",
      "--- gene=578, id=212 removed by soft count = 89\n",
      "--- gene=578, id=213 removed by soft count = 463\n",
      "--- gene=578, id=214 removed by soft count = 11773\n",
      "--- gene=578, id=215 removed by soft count = 176\n",
      "--- gene=578, id=216 removed by soft count = 364\n",
      "--- gene=578, id=227 removed by soft count = 65\n",
      "--- gene=578, id=233 removed by soft count = 115\n",
      "--- gene=578, id=235 removed by soft count = 99\n",
      "--- gene=578, id=236 removed by soft count = 151\n",
      "--- gene=578, id=241 removed by soft count = 87\n",
      "--- number of probes: 246 , kept by blast: 234 , if remove dups: 234\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 579\n",
      "--- gene=579, id=1 removed by soft count = 264\n",
      "--- gene=579, id=3 removed by soft count = 80\n",
      "--- gene=579, id=5 removed by soft count = 197\n",
      "--- gene=579, id=6 removed by soft count = 88\n",
      "--- gene=579, id=7 removed by soft count = 324\n",
      "--- gene=579, id=8 removed by soft count = 76\n",
      "--- gene=579, id=10 removed by soft count = 78\n",
      "--- gene=579, id=11 removed by soft count = 137\n",
      "--- gene=579, id=16 removed by soft count = 122\n",
      "--- gene=579, id=17 removed by soft count = 270\n",
      "--- gene=579, id=41 removed by soft count = 80\n",
      "--- gene=579, id=62 removed by soft count = 1690\n",
      "--- gene=579, id=63 removed by soft count = 231\n",
      "--- gene=579, id=69 removed by soft count = 136\n",
      "--- gene=579, id=70 removed by soft count = 328\n",
      "--- gene=579, id=72 removed by soft count = 378\n",
      "--- gene=579, id=78 removed by soft count = 225\n",
      "--- gene=579, id=82 removed by soft count = 187\n",
      "--- gene=579, id=90 removed by soft count = 126\n",
      "--- gene=579, id=131 removed by soft count = 666\n",
      "--- gene=579, id=147 removed by soft count = 344\n",
      "--- gene=579, id=148 removed by soft count = 94\n",
      "--- gene=579, id=150 removed by soft count = 510\n",
      "--- number of probes: 151 , kept by blast: 128 , if remove dups: 128\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 580\n",
      "--- gene=580, id=24 removed by soft count = 563\n",
      "--- gene=580, id=25 removed by soft count = 135\n",
      "--- gene=580, id=26 removed by soft count = 81\n",
      "--- gene=580, id=29 removed by soft count = 347\n",
      "--- gene=580, id=30 removed by soft count = 117\n",
      "--- gene=580, id=31 removed by soft count = 99\n",
      "--- gene=580, id=55 removed by soft count = 70\n",
      "--- number of probes: 202 , kept by blast: 195 , if remove dups: 195\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 581\n",
      "--- gene=581, id=10 removed by soft count = 6864\n",
      "--- gene=581, id=69 removed by soft count = 483\n",
      "--- gene=581, id=162 removed by soft count = 795\n",
      "--- number of probes: 250 , kept by blast: 247 , if remove dups: 247\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 582\n",
      "--- gene=582, id=108 removed by soft count = 58\n",
      "--- number of probes: 199 , kept by blast: 198 , if remove dups: 198\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 583\n",
      "--- gene=583, id=44 removed by soft count = 243\n",
      "--- gene=583, id=49 removed by soft count = 62\n",
      "--- gene=583, id=117 removed by soft count = 69\n",
      "--- gene=583, id=118 removed by soft count = 374\n",
      "--- gene=583, id=146 removed by soft count = 51\n",
      "--- gene=583, id=147 removed by soft count = 78\n",
      "--- gene=583, id=148 removed by soft count = 62\n",
      "--- gene=583, id=149 removed by soft count = 124\n",
      "--- gene=583, id=150 removed by soft count = 52\n",
      "--- gene=583, id=153 removed by soft count = 64\n",
      "--- gene=583, id=156 removed by soft count = 520\n",
      "--- gene=583, id=158 removed by soft count = 430\n",
      "--- number of probes: 181 , kept by blast: 169 , if remove dups: 168\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 584\n",
      "--- gene=584, id=56 removed by soft count = 300\n",
      "--- gene=584, id=58 removed by soft count = 194\n",
      "--- gene=584, id=64 removed by soft count = 103\n",
      "--- gene=584, id=104 removed by soft count = 121\n",
      "--- gene=584, id=107 removed by soft count = 183\n",
      "--- gene=584, id=108 removed by soft count = 186\n",
      "--- gene=584, id=109 removed by soft count = 203\n",
      "--- gene=584, id=110 removed by soft count = 12991\n",
      "--- gene=584, id=113 removed by soft count = 677\n",
      "--- gene=584, id=115 removed by soft count = 85\n",
      "--- gene=584, id=118 removed by soft count = 51\n",
      "--- gene=584, id=119 removed by soft count = 57\n",
      "--- gene=584, id=129 removed by soft count = 64\n",
      "--- number of probes: 139 , kept by blast: 126 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 585\n",
      "--- gene=585, id=26 removed by soft count = 143\n",
      "--- gene=585, id=71 removed by soft count = 330\n",
      "--- gene=585, id=103 removed by soft count = 139\n",
      "--- gene=585, id=105 removed by soft count = 101\n",
      "--- gene=585, id=106 removed by soft count = 134\n",
      "--- gene=585, id=108 removed by soft count = 125\n",
      "--- gene=585, id=109 removed by soft count = 116\n",
      "--- gene=585, id=110 removed by soft count = 80\n",
      "--- gene=585, id=111 removed by soft count = 54\n",
      "--- gene=585, id=112 removed by soft count = 84\n",
      "--- gene=585, id=113 removed by soft count = 57\n",
      "--- gene=585, id=115 removed by soft count = 71\n",
      "--- gene=585, id=117 removed by soft count = 58\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=585, id=191 removed by soft count = 105\n",
      "--- number of probes: 241 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 586\n",
      "--- gene=586, id=36 removed by soft count = 388\n",
      "--- gene=586, id=39 removed by soft count = 85\n",
      "--- gene=586, id=91 removed by soft count = 113\n",
      "--- gene=586, id=156 removed by soft count = 107\n",
      "--- gene=586, id=213 removed by soft count = 89\n",
      "--- number of probes: 218 , kept by blast: 213 , if remove dups: 213\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 587\n",
      "--- gene=587, id=2 removed by soft count = 61\n",
      "--- gene=587, id=3 removed by soft count = 142\n",
      "--- gene=587, id=4 removed by soft count = 108\n",
      "--- gene=587, id=6 removed by soft count = 120\n",
      "--- gene=587, id=9 removed by soft count = 86\n",
      "--- gene=587, id=13 removed by soft count = 729\n",
      "--- gene=587, id=29 removed by soft count = 290\n",
      "--- gene=587, id=38 removed by soft count = 81\n",
      "--- gene=587, id=39 removed by soft count = 109\n",
      "--- gene=587, id=44 removed by soft count = 1491\n",
      "--- gene=587, id=46 removed by soft count = 291\n",
      "--- gene=587, id=49 removed by soft count = 382\n",
      "--- gene=587, id=55 removed by soft count = 56\n",
      "--- gene=587, id=56 removed by soft count = 282\n",
      "--- gene=587, id=61 removed by soft count = 69\n",
      "--- gene=587, id=67 removed by soft count = 107\n",
      "--- gene=587, id=143 removed by soft count = 162\n",
      "--- gene=587, id=148 removed by soft count = 55\n",
      "--- gene=587, id=149 removed by soft count = 56\n",
      "--- gene=587, id=153 removed by soft count = 67\n",
      "--- gene=587, id=155 removed by soft count = 133\n",
      "--- number of probes: 156 , kept by blast: 135 , if remove dups: 135\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 588\n",
      "--- gene=588, id=36 removed by soft count = 200\n",
      "--- gene=588, id=37 removed by soft count = 386\n",
      "--- gene=588, id=56 removed by soft count = 528\n",
      "--- gene=588, id=57 removed by soft count = 751\n",
      "--- gene=588, id=58 removed by soft count = 385\n",
      "--- gene=588, id=59 removed by soft count = 290\n",
      "--- gene=588, id=97 removed by soft count = 179\n",
      "--- gene=588, id=100 removed by soft count = 58\n",
      "--- gene=588, id=116 removed by soft count = 108\n",
      "--- gene=588, id=135 removed by soft count = 54\n",
      "--- number of probes: 177 , kept by blast: 167 , if remove dups: 167\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 589\n",
      "--- gene=589, id=224 removed by soft count = 73\n",
      "--- number of probes: 275 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 590\n",
      "--- gene=590, id=88 removed by soft count = 66\n",
      "--- gene=590, id=114 removed by soft count = 86\n",
      "--- gene=590, id=143 removed by soft count = 117\n",
      "--- gene=590, id=144 removed by soft count = 98\n",
      "--- gene=590, id=148 removed by soft count = 912\n",
      "--- gene=590, id=149 removed by soft count = 388\n",
      "--- gene=590, id=164 removed by soft count = 57\n",
      "--- number of probes: 225 , kept by blast: 218 , if remove dups: 218\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 591\n",
      "--- number of probes: 283 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 592\n",
      "--- gene=592, id=26 removed by soft count = 3229\n",
      "--- gene=592, id=33 removed by soft count = 151\n",
      "--- gene=592, id=40 removed by soft count = 841\n",
      "--- gene=592, id=67 removed by soft count = 188\n",
      "--- gene=592, id=68 removed by soft count = 2835\n",
      "--- gene=592, id=105 removed by soft count = 6940\n",
      "--- gene=592, id=106 removed by soft count = 926\n",
      "--- gene=592, id=107 removed by soft count = 433\n",
      "--- gene=592, id=108 removed by soft count = 314\n",
      "--- gene=592, id=111 removed by soft count = 296\n",
      "--- number of probes: 144 , kept by blast: 134 , if remove dups: 134\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 593\n",
      "--- gene=593, id=30 removed by soft count = 86\n",
      "--- gene=593, id=54 removed by soft count = 65\n",
      "--- gene=593, id=104 removed by soft count = 121\n",
      "--- gene=593, id=159 removed by soft count = 2064\n",
      "--- gene=593, id=182 removed by soft count = 63\n",
      "--- gene=593, id=185 removed by soft count = 219\n",
      "--- gene=593, id=193 removed by soft count = 145\n",
      "--- number of probes: 300 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 594\n",
      "--- gene=594, id=83 removed by soft count = 119\n",
      "--- gene=594, id=84 removed by soft count = 240\n",
      "--- gene=594, id=85 removed by soft count = 817\n",
      "--- number of probes: 155 , kept by blast: 152 , if remove dups: 152\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 595\n",
      "--- gene=595, id=61 removed by soft count = 874\n",
      "--- gene=595, id=64 removed by soft count = 87\n",
      "--- gene=595, id=65 removed by soft count = 62\n",
      "--- gene=595, id=66 removed by soft count = 71\n",
      "--- gene=595, id=68 removed by soft count = 58\n",
      "--- gene=595, id=69 removed by soft count = 70\n",
      "--- gene=595, id=70 removed by soft count = 55\n",
      "--- gene=595, id=75 removed by soft count = 53\n",
      "--- gene=595, id=77 removed by soft count = 62\n",
      "--- gene=595, id=80 removed by soft count = 57\n",
      "--- gene=595, id=87 removed by soft count = 90\n",
      "--- gene=595, id=88 removed by soft count = 73\n",
      "--- gene=595, id=91 removed by soft count = 102\n",
      "--- gene=595, id=94 removed by soft count = 500\n",
      "--- gene=595, id=96 removed by soft count = 1815\n",
      "--- gene=595, id=133 removed by soft count = 642\n",
      "--- gene=595, id=134 removed by soft count = 388\n",
      "--- gene=595, id=135 removed by soft count = 381\n",
      "--- gene=595, id=136 removed by soft count = 233\n",
      "--- gene=595, id=137 removed by soft count = 394\n",
      "--- gene=595, id=138 removed by soft count = 175\n",
      "--- gene=595, id=139 removed by soft count = 858\n",
      "--- gene=595, id=141 removed by soft count = 294\n",
      "--- gene=595, id=142 removed by soft count = 135\n",
      "--- gene=595, id=144 removed by soft count = 73\n",
      "--- gene=595, id=146 removed by soft count = 63\n",
      "--- gene=595, id=148 removed by soft count = 90\n",
      "--- gene=595, id=177 removed by soft count = 59\n",
      "--- number of probes: 180 , kept by blast: 152 , if remove dups: 152\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 596\n",
      "--- gene=596, id=46 removed by soft count = 103\n",
      "--- gene=596, id=53 removed by soft count = 136\n",
      "--- gene=596, id=84 removed by soft count = 54\n",
      "--- gene=596, id=93 removed by soft count = 356\n",
      "--- gene=596, id=117 removed by soft count = 147\n",
      "--- gene=596, id=118 removed by soft count = 123\n",
      "--- gene=596, id=120 removed by soft count = 96\n",
      "--- gene=596, id=121 removed by soft count = 73\n",
      "--- gene=596, id=122 removed by soft count = 102\n",
      "--- gene=596, id=163 removed by soft count = 137\n",
      "--- gene=596, id=166 removed by soft count = 60\n",
      "--- number of probes: 254 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 597\n",
      "--- gene=597, id=0 removed by soft count = 121\n",
      "--- gene=597, id=106 removed by soft count = 431\n",
      "--- gene=597, id=108 removed by soft count = 60\n",
      "--- gene=597, id=109 removed by soft count = 112\n",
      "--- gene=597, id=132 removed by soft count = 139\n",
      "--- gene=597, id=147 removed by soft count = 13423\n",
      "--- gene=597, id=150 removed by soft count = 1163\n",
      "--- gene=597, id=212 removed by soft count = 89\n",
      "--- gene=597, id=266 removed by soft count = 70\n",
      "--- number of probes: 277 , kept by blast: 268 , if remove dups: 268\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 598\n",
      "--- gene=598, id=7 removed by soft count = 3075\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=598, id=158 removed by soft count = 174\n",
      "--- number of probes: 226 , kept by blast: 224 , if remove dups: 224\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 599\n",
      "--- gene=599, id=104 removed by soft count = 61\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 600\n",
      "--- gene=600, id=126 removed by soft count = 86\n",
      "--- gene=600, id=157 removed by soft count = 94\n",
      "--- gene=600, id=290 removed by soft count = 77\n",
      "--- number of probes: 294 , kept by blast: 291 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 601\n",
      "--- gene=601, id=7 removed by soft count = 1484\n",
      "--- gene=601, id=86 removed by soft count = 429\n",
      "--- gene=601, id=91 removed by soft count = 179\n",
      "--- gene=601, id=284 removed by soft count = 78\n",
      "--- number of probes: 299 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 602\n",
      "--- gene=602, id=63 removed by soft count = 75\n",
      "--- gene=602, id=141 removed by soft count = 571\n",
      "--- gene=602, id=168 removed by soft count = 58\n",
      "--- gene=602, id=170 removed by soft count = 197\n",
      "--- gene=602, id=174 removed by soft count = 78\n",
      "--- gene=602, id=182 removed by soft count = 159\n",
      "--- number of probes: 261 , kept by blast: 255 , if remove dups: 255\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 603\n",
      "--- gene=603, id=173 removed by soft count = 69\n",
      "--- gene=603, id=181 removed by soft count = 74\n",
      "--- gene=603, id=208 removed by soft count = 106\n",
      "--- number of probes: 251 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 605\n",
      "--- gene=605, id=0 removed by soft count = 3595\n",
      "--- gene=605, id=160 removed by soft count = 113\n",
      "--- number of probes: 187 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 606\n",
      "--- gene=606, id=56 removed by soft count = 81\n",
      "--- gene=606, id=113 removed by soft count = 76\n",
      "--- gene=606, id=149 removed by soft count = 83\n",
      "--- gene=606, id=167 removed by soft count = 71\n",
      "--- gene=606, id=175 removed by soft count = 73\n",
      "--- gene=606, id=184 removed by soft count = 374\n",
      "--- number of probes: 293 , kept by blast: 287 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 607\n",
      "--- gene=607, id=108 removed by soft count = 194\n",
      "--- gene=607, id=109 removed by soft count = 149\n",
      "--- gene=607, id=110 removed by soft count = 97\n",
      "--- gene=607, id=111 removed by soft count = 290\n",
      "--- gene=607, id=118 removed by soft count = 74\n",
      "--- gene=607, id=136 removed by soft count = 64\n",
      "--- number of probes: 281 , kept by blast: 275 , if remove dups: 275\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 608\n",
      "--- gene=608, id=85 removed by soft count = 438\n",
      "--- gene=608, id=86 removed by soft count = 155\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 609\n",
      "--- number of probes: 285 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 611\n",
      "--- gene=611, id=143 removed by hard count = 4\n",
      "--- gene=611, id=144 removed by hard count = 5\n",
      "--- gene=611, id=248 removed by soft count = 229\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 612\n",
      "--- gene=612, id=41 removed by soft count = 245\n",
      "--- gene=612, id=199 removed by soft count = 398\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 613\n",
      "--- gene=613, id=22 removed by soft count = 257\n",
      "--- gene=613, id=148 removed by soft count = 166\n",
      "--- gene=613, id=168 removed by soft count = 211\n",
      "--- number of probes: 200 , kept by blast: 197 , if remove dups: 197\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 614\n",
      "--- gene=614, id=161 removed by soft count = 90\n",
      "--- gene=614, id=182 removed by soft count = 116\n",
      "--- gene=614, id=251 removed by soft count = 78\n",
      "--- number of probes: 277 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 615\n",
      "--- gene=615, id=47 removed by soft count = 85\n",
      "--- gene=615, id=49 removed by soft count = 127\n",
      "--- gene=615, id=100 removed by soft count = 62\n",
      "--- gene=615, id=103 removed by soft count = 326\n",
      "--- gene=615, id=104 removed by soft count = 132\n",
      "--- gene=615, id=105 removed by soft count = 672\n",
      "--- gene=615, id=106 removed by soft count = 326\n",
      "--- number of probes: 258 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 616\n",
      "--- gene=616, id=1 removed by soft count = 2796\n",
      "--- gene=616, id=2 removed by soft count = 319\n",
      "--- gene=616, id=3 removed by soft count = 210\n",
      "--- gene=616, id=4 removed by soft count = 307\n",
      "--- gene=616, id=5 removed by soft count = 1253\n",
      "--- gene=616, id=6 removed by soft count = 191\n",
      "--- gene=616, id=44 removed by soft count = 310\n",
      "--- gene=616, id=123 removed by soft count = 51\n",
      "--- gene=616, id=138 removed by soft count = 125\n",
      "--- gene=616, id=139 removed by soft count = 2447\n",
      "--- number of probes: 173 , kept by blast: 163 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 617\n",
      "--- gene=617, id=115 removed by soft count = 123\n",
      "--- gene=617, id=127 removed by soft count = 994\n",
      "--- gene=617, id=231 removed by soft count = 110\n",
      "--- gene=617, id=232 removed by soft count = 94\n",
      "--- number of probes: 254 , kept by blast: 250 , if remove dups: 250\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 618\n",
      "--- number of probes: 192 , kept by blast: 192 , if remove dups: 190\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 619\n",
      "--- gene=619, id=59 removed by soft count = 180\n",
      "--- gene=619, id=110 removed by soft count = 52\n",
      "--- gene=619, id=273 removed by soft count = 187\n",
      "--- number of probes: 282 , kept by blast: 279 , if remove dups: 279\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 620\n",
      "--- gene=620, id=0 removed by soft count = 124\n",
      "--- gene=620, id=1 removed by soft count = 80\n",
      "--- gene=620, id=61 removed by soft count = 104\n",
      "--- gene=620, id=62 removed by soft count = 500\n",
      "--- gene=620, id=114 removed by soft count = 807\n",
      "--- gene=620, id=115 removed by soft count = 76\n",
      "--- gene=620, id=116 removed by soft count = 201\n",
      "--- gene=620, id=117 removed by soft count = 422\n",
      "--- gene=620, id=118 removed by soft count = 98\n",
      "--- gene=620, id=119 removed by soft count = 131\n",
      "--- gene=620, id=121 removed by soft count = 53\n",
      "--- gene=620, id=123 removed by soft count = 86\n",
      "--- number of probes: 196 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 621\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=621, id=80 removed by soft count = 329\n",
      "--- gene=621, id=215 removed by soft count = 354\n",
      "--- gene=621, id=216 removed by soft count = 605\n",
      "--- gene=621, id=240 removed by soft count = 192\n",
      "--- gene=621, id=241 removed by soft count = 133\n",
      "--- gene=621, id=244 removed by soft count = 116\n",
      "--- number of probes: 248 , kept by blast: 242 , if remove dups: 242\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 622\n",
      "--- gene=622, id=107 removed by soft count = 263\n",
      "--- gene=622, id=108 removed by soft count = 1118\n",
      "--- gene=622, id=109 removed by soft count = 3750\n",
      "--- gene=622, id=221 removed by soft count = 58\n",
      "--- gene=622, id=225 removed by soft count = 153\n",
      "--- number of probes: 247 , kept by blast: 242 , if remove dups: 242\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 623\n",
      "--- gene=623, id=18 removed by soft count = 1960\n",
      "--- gene=623, id=20 removed by soft count = 2208\n",
      "--- gene=623, id=105 removed by soft count = 150\n",
      "--- gene=623, id=106 removed by soft count = 254\n",
      "--- gene=623, id=111 removed by soft count = 83\n",
      "--- gene=623, id=142 removed by soft count = 560\n",
      "--- gene=623, id=146 removed by soft count = 200\n",
      "--- gene=623, id=150 removed by soft count = 59\n",
      "--- number of probes: 185 , kept by blast: 177 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 624\n",
      "--- gene=624, id=0 removed by soft count = 417\n",
      "--- gene=624, id=76 removed by soft count = 121\n",
      "--- number of probes: 132 , kept by blast: 130 , if remove dups: 130\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 625\n",
      "--- gene=625, id=0 removed by soft count = 69\n",
      "--- gene=625, id=1 removed by soft count = 119\n",
      "--- gene=625, id=6 removed by soft count = 56\n",
      "--- gene=625, id=105 removed by soft count = 126\n",
      "--- gene=625, id=106 removed by soft count = 168\n",
      "--- number of probes: 168 , kept by blast: 163 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 626\n",
      "--- gene=626, id=0 removed by soft count = 383\n",
      "--- gene=626, id=1 removed by soft count = 70\n",
      "--- gene=626, id=2 removed by soft count = 13782\n",
      "--- gene=626, id=18 removed by soft count = 1603\n",
      "--- gene=626, id=47 removed by soft count = 617\n",
      "--- gene=626, id=54 removed by soft count = 63\n",
      "--- gene=626, id=56 removed by soft count = 56\n",
      "--- gene=626, id=89 removed by soft count = 106\n",
      "--- gene=626, id=96 removed by soft count = 9062\n",
      "--- gene=626, id=105 removed by soft count = 71\n",
      "--- gene=626, id=107 removed by soft count = 608\n",
      "--- gene=626, id=108 removed by soft count = 52\n",
      "--- gene=626, id=109 removed by soft count = 352\n",
      "--- gene=626, id=113 removed by soft count = 262\n",
      "--- gene=626, id=115 removed by soft count = 393\n",
      "--- number of probes: 118 , kept by blast: 103 , if remove dups: 103\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 103\n",
      "-- checking probes in region: 627\n",
      "--- gene=627, id=96 removed by soft count = 59\n",
      "--- gene=627, id=97 removed by soft count = 83\n",
      "--- gene=627, id=99 removed by soft count = 907\n",
      "--- gene=627, id=100 removed by soft count = 149\n",
      "--- gene=627, id=169 removed by soft count = 1608\n",
      "--- number of probes: 253 , kept by blast: 248 , if remove dups: 247\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 628\n",
      "--- gene=628, id=268 removed by soft count = 581\n",
      "--- gene=628, id=270 removed by soft count = 613\n",
      "--- gene=628, id=272 removed by soft count = 372\n",
      "--- number of probes: 273 , kept by blast: 270 , if remove dups: 270\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 629\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 645\n",
      "--- number of probes: 257 , kept by blast: 257 , if remove dups: 257\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 646\n",
      "--- gene=646, id=37 removed by soft count = 58\n",
      "--- gene=646, id=219 removed by soft count = 668\n",
      "--- gene=646, id=220 removed by soft count = 86\n",
      "--- gene=646, id=223 removed by soft count = 82\n",
      "--- gene=646, id=230 removed by soft count = 53\n",
      "--- number of probes: 241 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 647\n",
      "--- gene=647, id=80 removed by soft count = 118\n",
      "--- gene=647, id=81 removed by soft count = 594\n",
      "--- gene=647, id=82 removed by soft count = 166\n",
      "--- gene=647, id=83 removed by soft count = 537\n",
      "--- gene=647, id=109 removed by soft count = 321\n",
      "--- gene=647, id=112 removed by soft count = 546\n",
      "--- gene=647, id=113 removed by soft count = 116\n",
      "--- gene=647, id=116 removed by soft count = 477\n",
      "--- gene=647, id=118 removed by soft count = 168\n",
      "--- gene=647, id=122 removed by soft count = 59\n",
      "--- gene=647, id=123 removed by soft count = 134\n",
      "--- gene=647, id=131 removed by soft count = 114\n",
      "--- gene=647, id=133 removed by soft count = 78\n",
      "--- gene=647, id=175 removed by soft count = 75\n",
      "--- number of probes: 219 , kept by blast: 205 , if remove dups: 205\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 648\n",
      "--- number of probes: 215 , kept by blast: 215 , if remove dups: 215\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 649\n",
      "--- gene=649, id=51 removed by soft count = 68\n",
      "--- gene=649, id=70 removed by soft count = 260\n",
      "--- gene=649, id=71 removed by soft count = 251\n",
      "--- gene=649, id=137 removed by soft count = 135\n",
      "--- number of probes: 262 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 650\n",
      "--- gene=650, id=6 removed by soft count = 106\n",
      "--- gene=650, id=144 removed by soft count = 119\n",
      "--- gene=650, id=176 removed by soft count = 95\n",
      "--- number of probes: 241 , kept by blast: 238 , if remove dups: 238\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 651\n",
      "--- gene=651, id=17 removed by soft count = 89\n",
      "--- gene=651, id=57 removed by soft count = 104\n",
      "--- gene=651, id=58 removed by soft count = 604\n",
      "--- gene=651, id=59 removed by soft count = 232\n",
      "--- gene=651, id=60 removed by soft count = 153\n",
      "--- gene=651, id=61 removed by soft count = 130\n",
      "--- gene=651, id=62 removed by soft count = 80\n",
      "--- gene=651, id=118 removed by soft count = 84\n",
      "--- gene=651, id=127 removed by soft count = 57\n",
      "--- gene=651, id=136 removed by soft count = 70\n",
      "--- gene=651, id=140 removed by soft count = 142\n",
      "--- gene=651, id=144 removed by soft count = 667\n",
      "--- gene=651, id=145 removed by soft count = 573\n",
      "--- number of probes: 146 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 652\n",
      "--- gene=652, id=38 removed by soft count = 278\n",
      "--- gene=652, id=39 removed by soft count = 310\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 653\n",
      "--- gene=653, id=270 removed by soft count = 65\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 654\n",
      "--- gene=654, id=49 removed by soft count = 219\n",
      "--- gene=654, id=63 removed by soft count = 444\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=654, id=140 removed by soft count = 76\n",
      "--- gene=654, id=151 removed by soft count = 52\n",
      "--- number of probes: 271 , kept by blast: 267 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 655\n",
      "--- gene=655, id=208 removed by soft count = 63\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 656\n",
      "--- gene=656, id=54 removed by soft count = 87\n",
      "--- gene=656, id=75 removed by soft count = 376\n",
      "--- gene=656, id=80 removed by soft count = 60\n",
      "--- number of probes: 292 , kept by blast: 289 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 657\n",
      "--- gene=657, id=13 removed by soft count = 79\n",
      "--- gene=657, id=15 removed by soft count = 82\n",
      "--- gene=657, id=25 removed by soft count = 92\n",
      "--- gene=657, id=26 removed by soft count = 869\n",
      "--- gene=657, id=32 removed by soft count = 72\n",
      "--- gene=657, id=37 removed by soft count = 68\n",
      "--- gene=657, id=38 removed by soft count = 759\n",
      "--- gene=657, id=44 removed by soft count = 51\n",
      "--- gene=657, id=45 removed by soft count = 55\n",
      "--- number of probes: 186 , kept by blast: 177 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 658\n",
      "--- gene=658, id=172 removed by soft count = 2617\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 659\n",
      "--- gene=659, id=2 removed by soft count = 265\n",
      "--- gene=659, id=4 removed by soft count = 104\n",
      "--- gene=659, id=7 removed by soft count = 199\n",
      "--- gene=659, id=8 removed by soft count = 78\n",
      "--- gene=659, id=30 removed by soft count = 116\n",
      "--- gene=659, id=33 removed by soft count = 93\n",
      "--- number of probes: 54 , kept by blast: 48 , if remove dups: 42\n",
      "-- number of probes kept for this region: 48\n",
      "-- checking probes in region: 660\n",
      "--- gene=660, id=76 removed by soft count = 75\n",
      "--- gene=660, id=78 removed by soft count = 1259\n",
      "--- gene=660, id=253 removed by soft count = 523\n",
      "--- gene=660, id=254 removed by soft count = 75\n",
      "--- gene=660, id=258 removed by soft count = 122\n",
      "--- gene=660, id=259 removed by soft count = 92\n",
      "--- gene=660, id=260 removed by soft count = 190\n",
      "--- gene=660, id=262 removed by soft count = 60\n",
      "--- gene=660, id=265 removed by soft count = 75\n",
      "--- gene=660, id=266 removed by soft count = 69\n",
      "--- gene=660, id=276 removed by soft count = 124\n",
      "--- gene=660, id=277 removed by soft count = 164\n",
      "--- number of probes: 278 , kept by blast: 266 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 662\n",
      "--- gene=662, id=14 removed by soft count = 155\n",
      "--- gene=662, id=17 removed by soft count = 147\n",
      "--- gene=662, id=23 removed by soft count = 108\n",
      "--- gene=662, id=27 removed by soft count = 716\n",
      "--- gene=662, id=29 removed by soft count = 107\n",
      "--- gene=662, id=31 removed by soft count = 51\n",
      "--- gene=662, id=32 removed by soft count = 56\n",
      "--- gene=662, id=34 removed by soft count = 146\n",
      "--- gene=662, id=40 removed by soft count = 2404\n",
      "--- gene=662, id=44 removed by soft count = 139\n",
      "--- gene=662, id=46 removed by soft count = 64\n",
      "--- gene=662, id=48 removed by soft count = 76\n",
      "--- gene=662, id=49 removed by soft count = 79\n",
      "--- gene=662, id=163 removed by soft count = 766\n",
      "--- number of probes: 184 , kept by blast: 170 , if remove dups: 170\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 663\n",
      "--- gene=663, id=24 removed by soft count = 144\n",
      "--- gene=663, id=83 removed by soft count = 84\n",
      "--- gene=663, id=85 removed by soft count = 73\n",
      "--- gene=663, id=199 removed by soft count = 484\n",
      "--- gene=663, id=200 removed by soft count = 460\n",
      "--- gene=663, id=202 removed by soft count = 1144\n",
      "--- number of probes: 205 , kept by blast: 199 , if remove dups: 199\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 664\n",
      "--- gene=664, id=18 removed by soft count = 240\n",
      "--- gene=664, id=19 removed by soft count = 157\n",
      "--- gene=664, id=22 removed by soft count = 153\n",
      "--- gene=664, id=146 removed by soft count = 146\n",
      "--- number of probes: 255 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 665\n",
      "--- gene=665, id=105 removed by soft count = 274\n",
      "--- gene=665, id=107 removed by soft count = 204\n",
      "--- gene=665, id=170 removed by soft count = 318\n",
      "--- gene=665, id=171 removed by soft count = 56\n",
      "--- gene=665, id=172 removed by soft count = 103\n",
      "--- number of probes: 266 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 666\n",
      "--- gene=666, id=31 removed by soft count = 65\n",
      "--- gene=666, id=43 removed by soft count = 56\n",
      "--- gene=666, id=51 removed by soft count = 55\n",
      "--- gene=666, id=116 removed by soft count = 638\n",
      "--- gene=666, id=119 removed by soft count = 170\n",
      "--- gene=666, id=176 removed by soft count = 75\n",
      "--- number of probes: 273 , kept by blast: 267 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 667\n",
      "--- gene=667, id=9 removed by soft count = 2334\n",
      "--- gene=667, id=10 removed by soft count = 520\n",
      "--- gene=667, id=12 removed by soft count = 422\n",
      "--- gene=667, id=13 removed by soft count = 80\n",
      "--- gene=667, id=20 removed by soft count = 51\n",
      "--- gene=667, id=47 removed by soft count = 86\n",
      "--- gene=667, id=49 removed by soft count = 153\n",
      "--- gene=667, id=74 removed by soft count = 76\n",
      "--- gene=667, id=75 removed by soft count = 78\n",
      "--- gene=667, id=76 removed by soft count = 298\n",
      "--- gene=667, id=81 removed by soft count = 132\n",
      "--- gene=667, id=86 removed by soft count = 273\n",
      "--- gene=667, id=100 removed by soft count = 57\n",
      "--- gene=667, id=131 removed by soft count = 52\n",
      "--- number of probes: 172 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 668\n",
      "--- gene=668, id=109 removed by soft count = 140\n",
      "--- gene=668, id=111 removed by soft count = 112\n",
      "--- gene=668, id=113 removed by soft count = 259\n",
      "--- gene=668, id=115 removed by soft count = 506\n",
      "--- gene=668, id=116 removed by soft count = 314\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 669\n",
      "--- gene=669, id=32 removed by soft count = 88\n",
      "--- gene=669, id=35 removed by soft count = 135\n",
      "--- gene=669, id=298 removed by soft count = 504\n",
      "--- gene=669, id=299 removed by soft count = 320\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 670\n",
      "--- gene=670, id=52 removed by soft count = 58\n",
      "--- gene=670, id=213 removed by soft count = 134\n",
      "--- gene=670, id=258 removed by soft count = 172\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 671\n",
      "--- gene=671, id=125 removed by soft count = 242\n",
      "--- gene=671, id=131 removed by soft count = 79\n",
      "--- gene=671, id=180 removed by soft count = 305\n",
      "--- gene=671, id=182 removed by soft count = 141\n",
      "--- gene=671, id=183 removed by soft count = 184\n",
      "--- number of probes: 199 , kept by blast: 194 , if remove dups: 194\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 672\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=672, id=120 removed by soft count = 79\n",
      "--- gene=672, id=128 removed by soft count = 75\n",
      "--- number of probes: 171 , kept by blast: 169 , if remove dups: 169\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 673\n",
      "--- gene=673, id=69 removed by soft count = 172\n",
      "--- gene=673, id=116 removed by soft count = 248\n",
      "--- gene=673, id=118 removed by soft count = 218\n",
      "--- gene=673, id=119 removed by soft count = 5650\n",
      "--- number of probes: 120 , kept by blast: 116 , if remove dups: 116\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 116\n",
      "-- checking probes in region: 674\n",
      "--- number of probes: 258 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 675\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 676\n",
      "--- gene=676, id=119 removed by hard count = 0\n",
      "--- gene=676, id=120 removed by hard count = 0\n",
      "--- gene=676, id=121 removed by hard count = 0\n",
      "--- gene=676, id=170 removed by soft count = 1746\n",
      "--- gene=676, id=227 removed by soft count = 73\n",
      "--- number of probes: 299 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 677\n",
      "--- gene=677, id=20 removed by soft count = 85\n",
      "--- gene=677, id=239 removed by soft count = 176\n",
      "--- gene=677, id=291 removed by soft count = 261\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 678\n",
      "--- gene=678, id=45 removed by soft count = 67\n",
      "--- gene=678, id=146 removed by soft count = 80\n",
      "--- gene=678, id=192 removed by soft count = 199\n",
      "--- gene=678, id=226 removed by soft count = 192\n",
      "--- gene=678, id=242 removed by soft count = 57\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 679\n",
      "--- gene=679, id=40 removed by soft count = 116\n",
      "--- gene=679, id=140 removed by soft count = 106\n",
      "--- gene=679, id=143 removed by soft count = 11385\n",
      "--- gene=679, id=156 removed by soft count = 63\n",
      "--- gene=679, id=167 removed by soft count = 209\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 680\n",
      "--- gene=680, id=162 removed by soft count = 137\n",
      "--- gene=680, id=165 removed by soft count = 452\n",
      "--- gene=680, id=169 removed by soft count = 80\n",
      "--- gene=680, id=224 removed by soft count = 208\n",
      "--- gene=680, id=243 removed by hard count = 3\n",
      "--- number of probes: 293 , kept by blast: 288 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 681\n",
      "--- gene=681, id=84 removed by soft count = 58\n",
      "--- gene=681, id=88 removed by soft count = 228\n",
      "--- gene=681, id=89 removed by soft count = 386\n",
      "--- gene=681, id=90 removed by soft count = 295\n",
      "--- gene=681, id=115 removed by soft count = 65\n",
      "--- gene=681, id=116 removed by soft count = 443\n",
      "--- gene=681, id=117 removed by soft count = 138\n",
      "--- gene=681, id=141 removed by soft count = 148\n",
      "--- number of probes: 298 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 682\n",
      "--- gene=682, id=10 removed by soft count = 89\n",
      "--- gene=682, id=11 removed by soft count = 189\n",
      "--- gene=682, id=14 removed by soft count = 150\n",
      "--- gene=682, id=111 removed by soft count = 63\n",
      "--- number of probes: 147 , kept by blast: 143 , if remove dups: 142\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 683\n",
      "--- gene=683, id=64 removed by soft count = 105\n",
      "--- gene=683, id=123 removed by soft count = 60\n",
      "--- gene=683, id=234 removed by soft count = 105\n",
      "--- gene=683, id=235 removed by soft count = 53\n",
      "--- gene=683, id=245 removed by soft count = 117\n",
      "--- number of probes: 285 , kept by blast: 280 , if remove dups: 280\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 684\n",
      "--- gene=684, id=63 removed by soft count = 132\n",
      "--- gene=684, id=76 removed by soft count = 130\n",
      "--- gene=684, id=210 removed by soft count = 923\n",
      "--- number of probes: 253 , kept by blast: 250 , if remove dups: 250\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 685\n",
      "--- gene=685, id=59 removed by soft count = 93\n",
      "--- gene=685, id=79 removed by soft count = 406\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 686\n",
      "--- gene=686, id=50 removed by soft count = 88\n",
      "--- gene=686, id=98 removed by soft count = 178\n",
      "--- gene=686, id=99 removed by soft count = 685\n",
      "--- gene=686, id=187 removed by soft count = 74\n",
      "--- gene=686, id=190 removed by soft count = 295\n",
      "--- gene=686, id=216 removed by soft count = 77\n",
      "--- gene=686, id=253 removed by soft count = 1287\n",
      "--- gene=686, id=255 removed by soft count = 1605\n",
      "--- number of probes: 266 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 687\n",
      "--- gene=687, id=0 removed by soft count = 109\n",
      "--- gene=687, id=3 removed by soft count = 77\n",
      "--- gene=687, id=4 removed by soft count = 55\n",
      "--- gene=687, id=9 removed by soft count = 93\n",
      "--- gene=687, id=10 removed by soft count = 60\n",
      "--- gene=687, id=13 removed by soft count = 67\n",
      "--- gene=687, id=14 removed by soft count = 56\n",
      "--- gene=687, id=15 removed by soft count = 121\n",
      "--- gene=687, id=16 removed by soft count = 115\n",
      "--- gene=687, id=17 removed by soft count = 70\n",
      "--- gene=687, id=20 removed by soft count = 55\n",
      "--- gene=687, id=21 removed by soft count = 118\n",
      "--- gene=687, id=28 removed by soft count = 58\n",
      "--- gene=687, id=33 removed by soft count = 66\n",
      "--- gene=687, id=34 removed by soft count = 51\n",
      "--- gene=687, id=38 removed by soft count = 82\n",
      "--- gene=687, id=40 removed by soft count = 110\n",
      "--- gene=687, id=42 removed by soft count = 76\n",
      "--- gene=687, id=48 removed by soft count = 55\n",
      "--- gene=687, id=49 removed by soft count = 58\n",
      "--- gene=687, id=51 removed by soft count = 55\n",
      "--- gene=687, id=57 removed by soft count = 53\n",
      "--- gene=687, id=58 removed by soft count = 54\n",
      "--- gene=687, id=62 removed by soft count = 307\n",
      "--- gene=687, id=64 removed by soft count = 51\n",
      "--- gene=687, id=65 removed by soft count = 77\n",
      "--- gene=687, id=66 removed by soft count = 57\n",
      "--- gene=687, id=68 removed by soft count = 61\n",
      "--- gene=687, id=69 removed by soft count = 59\n",
      "--- gene=687, id=71 removed by soft count = 86\n",
      "--- gene=687, id=72 removed by soft count = 67\n",
      "--- gene=687, id=74 removed by soft count = 90\n",
      "--- gene=687, id=75 removed by soft count = 134\n",
      "--- gene=687, id=76 removed by soft count = 147\n",
      "--- gene=687, id=77 removed by soft count = 150\n",
      "--- number of probes: 78 , kept by blast: 43 , if remove dups: 43\n",
      "-- number of probes kept for this region: 43\n",
      "-- checking probes in region: 688\n",
      "--- gene=688, id=2 removed by soft count = 61\n",
      "--- gene=688, id=3 removed by soft count = 95\n",
      "--- gene=688, id=161 removed by soft count = 275\n",
      "--- gene=688, id=162 removed by soft count = 931\n",
      "--- gene=688, id=271 removed by soft count = 270\n",
      "--- gene=688, id=272 removed by soft count = 114\n",
      "--- gene=688, id=273 removed by soft count = 309\n",
      "--- gene=688, id=274 removed by soft count = 189\n",
      "--- gene=688, id=275 removed by soft count = 100\n",
      "--- gene=688, id=276 removed by soft count = 186\n",
      "--- gene=688, id=277 removed by soft count = 103\n",
      "--- number of probes: 278 , kept by blast: 267 , if remove dups: 267\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 689\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=689, id=20 removed by soft count = 351\n",
      "--- gene=689, id=21 removed by soft count = 373\n",
      "--- gene=689, id=22 removed by soft count = 2988\n",
      "--- gene=689, id=56 removed by soft count = 66\n",
      "--- gene=689, id=115 removed by soft count = 71\n",
      "--- gene=689, id=122 removed by soft count = 136\n",
      "--- gene=689, id=123 removed by soft count = 168\n",
      "--- gene=689, id=126 removed by soft count = 112\n",
      "--- number of probes: 160 , kept by blast: 152 , if remove dups: 152\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 690\n",
      "--- gene=690, id=57 removed by soft count = 89\n",
      "--- gene=690, id=84 removed by soft count = 58\n",
      "--- gene=690, id=141 removed by hard count = 0\n",
      "--- gene=690, id=162 removed by soft count = 339\n",
      "--- gene=690, id=224 removed by soft count = 67\n",
      "--- number of probes: 289 , kept by blast: 284 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 691\n",
      "--- number of probes: 270 , kept by blast: 270 , if remove dups: 270\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 692\n",
      "--- gene=692, id=23 removed by soft count = 567\n",
      "--- gene=692, id=49 removed by soft count = 81\n",
      "--- gene=692, id=50 removed by soft count = 242\n",
      "--- gene=692, id=51 removed by soft count = 159\n",
      "--- gene=692, id=52 removed by soft count = 158\n",
      "--- gene=692, id=89 removed by soft count = 74\n",
      "--- gene=692, id=138 removed by soft count = 11235\n",
      "--- gene=692, id=172 removed by soft count = 496\n",
      "--- gene=692, id=173 removed by soft count = 308\n",
      "--- number of probes: 189 , kept by blast: 180 , if remove dups: 180\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 693\n",
      "--- gene=693, id=55 removed by soft count = 299\n",
      "--- gene=693, id=57 removed by soft count = 102\n",
      "--- gene=693, id=120 removed by soft count = 194\n",
      "--- gene=693, id=159 removed by soft count = 204\n",
      "--- gene=693, id=168 removed by soft count = 64\n",
      "--- gene=693, id=174 removed by soft count = 71\n",
      "--- number of probes: 246 , kept by blast: 240 , if remove dups: 238\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 694\n",
      "--- gene=694, id=35 removed by soft count = 93\n",
      "--- gene=694, id=114 removed by soft count = 51\n",
      "--- gene=694, id=169 removed by soft count = 213\n",
      "--- gene=694, id=170 removed by soft count = 100\n",
      "--- gene=694, id=209 removed by soft count = 122\n",
      "--- gene=694, id=217 removed by soft count = 124\n",
      "--- gene=694, id=218 removed by soft count = 478\n",
      "--- gene=694, id=219 removed by soft count = 144\n",
      "--- number of probes: 236 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 695\n",
      "--- gene=695, id=9 removed by soft count = 259\n",
      "--- gene=695, id=12 removed by soft count = 78\n",
      "--- gene=695, id=18 removed by soft count = 195\n",
      "--- gene=695, id=62 removed by soft count = 255\n",
      "--- gene=695, id=73 removed by soft count = 75\n",
      "--- gene=695, id=158 removed by soft count = 74\n",
      "--- gene=695, id=168 removed by soft count = 317\n",
      "--- gene=695, id=185 removed by soft count = 128\n",
      "--- gene=695, id=186 removed by soft count = 118\n",
      "--- gene=695, id=188 removed by soft count = 163\n",
      "--- gene=695, id=240 removed by soft count = 64\n",
      "--- gene=695, id=241 removed by soft count = 785\n",
      "--- gene=695, id=242 removed by soft count = 52\n",
      "--- number of probes: 246 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 696\n",
      "--- gene=696, id=48 removed by soft count = 63\n",
      "--- gene=696, id=84 removed by soft count = 151\n",
      "--- gene=696, id=86 removed by soft count = 1362\n",
      "--- gene=696, id=87 removed by soft count = 245\n",
      "--- number of probes: 170 , kept by blast: 166 , if remove dups: 166\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 697\n",
      "--- gene=697, id=36 removed by soft count = 64\n",
      "--- gene=697, id=76 removed by soft count = 390\n",
      "--- gene=697, id=98 removed by soft count = 65\n",
      "--- gene=697, id=105 removed by soft count = 51\n",
      "--- gene=697, id=106 removed by soft count = 74\n",
      "--- gene=697, id=115 removed by soft count = 281\n",
      "--- number of probes: 220 , kept by blast: 214 , if remove dups: 214\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 698\n",
      "--- gene=698, id=2 removed by soft count = 86\n",
      "--- gene=698, id=31 removed by soft count = 136\n",
      "--- gene=698, id=33 removed by soft count = 644\n",
      "--- gene=698, id=36 removed by soft count = 238\n",
      "--- gene=698, id=142 removed by soft count = 92\n",
      "--- gene=698, id=178 removed by soft count = 212\n",
      "--- gene=698, id=179 removed by soft count = 163\n",
      "--- number of probes: 227 , kept by blast: 220 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 699\n",
      "--- gene=699, id=10 removed by soft count = 229\n",
      "--- gene=699, id=14 removed by soft count = 966\n",
      "--- gene=699, id=15 removed by soft count = 1033\n",
      "--- gene=699, id=38 removed by soft count = 704\n",
      "--- gene=699, id=39 removed by soft count = 782\n",
      "--- gene=699, id=42 removed by soft count = 1892\n",
      "--- gene=699, id=46 removed by soft count = 454\n",
      "--- gene=699, id=47 removed by soft count = 243\n",
      "--- gene=699, id=48 removed by soft count = 170\n",
      "--- gene=699, id=51 removed by soft count = 381\n",
      "--- gene=699, id=52 removed by soft count = 1851\n",
      "--- gene=699, id=54 removed by soft count = 153\n",
      "--- gene=699, id=55 removed by soft count = 64\n",
      "--- gene=699, id=58 removed by soft count = 247\n",
      "--- gene=699, id=59 removed by soft count = 108\n",
      "--- gene=699, id=60 removed by soft count = 58\n",
      "--- gene=699, id=61 removed by soft count = 101\n",
      "--- gene=699, id=64 removed by soft count = 151\n",
      "--- gene=699, id=65 removed by soft count = 65\n",
      "--- number of probes: 66 , kept by blast: 47 , if remove dups: 47\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: 701\n",
      "--- gene=701, id=4 removed by soft count = 445\n",
      "--- gene=701, id=118 removed by soft count = 102\n",
      "--- gene=701, id=159 removed by soft count = 175\n",
      "--- gene=701, id=241 removed by soft count = 290\n",
      "--- gene=701, id=244 removed by soft count = 413\n",
      "--- gene=701, id=245 removed by soft count = 129\n",
      "--- gene=701, id=248 removed by soft count = 90\n",
      "--- gene=701, id=249 removed by soft count = 638\n",
      "--- gene=701, id=266 removed by soft count = 221\n",
      "--- gene=701, id=285 removed by soft count = 74\n",
      "--- number of probes: 289 , kept by blast: 279 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 702\n",
      "--- gene=702, id=33 removed by soft count = 544\n",
      "--- gene=702, id=34 removed by soft count = 153\n",
      "--- gene=702, id=169 removed by soft count = 558\n",
      "--- gene=702, id=172 removed by soft count = 284\n",
      "--- gene=702, id=174 removed by soft count = 65\n",
      "--- gene=702, id=289 removed by soft count = 84\n",
      "--- gene=702, id=290 removed by soft count = 5296\n",
      "--- number of probes: 291 , kept by blast: 284 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 703\n",
      "--- gene=703, id=2 removed by soft count = 71\n",
      "--- gene=703, id=14 removed by soft count = 80\n",
      "--- gene=703, id=18 removed by soft count = 51\n",
      "--- gene=703, id=40 removed by soft count = 98\n",
      "--- gene=703, id=132 removed by soft count = 710\n",
      "--- gene=703, id=133 removed by soft count = 185\n",
      "--- gene=703, id=143 removed by soft count = 63\n",
      "--- number of probes: 181 , kept by blast: 174 , if remove dups: 174\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 704\n",
      "--- gene=704, id=71 removed by soft count = 1242\n",
      "--- gene=704, id=72 removed by soft count = 558\n",
      "--- gene=704, id=73 removed by soft count = 606\n",
      "--- gene=704, id=74 removed by soft count = 439\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=704, id=75 removed by soft count = 500\n",
      "--- gene=704, id=76 removed by soft count = 51\n",
      "--- gene=704, id=77 removed by soft count = 321\n",
      "--- gene=704, id=78 removed by soft count = 287\n",
      "--- gene=704, id=79 removed by soft count = 456\n",
      "--- gene=704, id=80 removed by soft count = 405\n",
      "--- gene=704, id=81 removed by soft count = 55\n",
      "--- gene=704, id=82 removed by soft count = 148\n",
      "--- gene=704, id=83 removed by soft count = 395\n",
      "--- gene=704, id=84 removed by soft count = 276\n",
      "--- gene=704, id=85 removed by soft count = 369\n",
      "--- gene=704, id=164 removed by soft count = 62\n",
      "--- gene=704, id=165 removed by soft count = 86\n",
      "--- gene=704, id=166 removed by soft count = 64\n",
      "--- gene=704, id=169 removed by soft count = 55\n",
      "--- gene=704, id=170 removed by soft count = 92\n",
      "--- gene=704, id=175 removed by soft count = 57\n",
      "--- gene=704, id=176 removed by soft count = 147\n",
      "--- number of probes: 284 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 705\n",
      "--- gene=705, id=236 removed by soft count = 233\n",
      "--- gene=705, id=265 removed by soft count = 597\n",
      "--- number of probes: 292 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 706\n",
      "--- gene=706, id=13 removed by soft count = 1658\n",
      "--- gene=706, id=34 removed by soft count = 254\n",
      "--- gene=706, id=104 removed by soft count = 428\n",
      "--- gene=706, id=192 removed by soft count = 72\n",
      "--- number of probes: 246 , kept by blast: 242 , if remove dups: 242\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 707\n",
      "--- gene=707, id=13 removed by soft count = 280\n",
      "--- gene=707, id=14 removed by soft count = 763\n",
      "--- gene=707, id=16 removed by soft count = 2466\n",
      "--- gene=707, id=69 removed by soft count = 97\n",
      "--- gene=707, id=78 removed by soft count = 52\n",
      "--- gene=707, id=82 removed by soft count = 126\n",
      "--- gene=707, id=83 removed by soft count = 469\n",
      "--- gene=707, id=86 removed by soft count = 57\n",
      "--- gene=707, id=101 removed by soft count = 122\n",
      "--- gene=707, id=106 removed by soft count = 54\n",
      "--- gene=707, id=113 removed by soft count = 479\n",
      "--- gene=707, id=123 removed by soft count = 62\n",
      "--- gene=707, id=126 removed by soft count = 58\n",
      "--- number of probes: 182 , kept by blast: 169 , if remove dups: 169\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 708\n",
      "--- gene=708, id=56 removed by soft count = 56\n",
      "--- gene=708, id=68 removed by soft count = 73\n",
      "--- gene=708, id=79 removed by soft count = 172\n",
      "--- gene=708, id=80 removed by soft count = 55\n",
      "--- gene=708, id=87 removed by soft count = 125\n",
      "--- gene=708, id=138 removed by soft count = 52\n",
      "--- number of probes: 213 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 709\n",
      "--- gene=709, id=19 removed by soft count = 90\n",
      "--- gene=709, id=49 removed by soft count = 2653\n",
      "--- gene=709, id=50 removed by soft count = 79\n",
      "--- gene=709, id=51 removed by soft count = 383\n",
      "--- gene=709, id=52 removed by soft count = 1075\n",
      "--- gene=709, id=155 removed by soft count = 56\n",
      "--- gene=709, id=242 removed by soft count = 90\n",
      "--- number of probes: 289 , kept by blast: 282 , if remove dups: 282\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 710\n",
      "--- gene=710, id=178 removed by soft count = 1230\n",
      "--- gene=710, id=180 removed by soft count = 5243\n",
      "--- gene=710, id=181 removed by soft count = 5156\n",
      "--- gene=710, id=182 removed by soft count = 1410\n",
      "--- gene=710, id=183 removed by soft count = 488\n",
      "--- gene=710, id=184 removed by soft count = 141\n",
      "--- gene=710, id=185 removed by soft count = 201\n",
      "--- number of probes: 234 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 711\n",
      "--- gene=711, id=194 removed by soft count = 238\n",
      "--- gene=711, id=214 removed by soft count = 320\n",
      "--- number of probes: 290 , kept by blast: 288 , if remove dups: 288\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 712\n",
      "--- gene=712, id=2 removed by soft count = 68\n",
      "--- gene=712, id=27 removed by soft count = 154\n",
      "--- gene=712, id=43 removed by soft count = 98\n",
      "--- gene=712, id=115 removed by soft count = 95\n",
      "--- gene=712, id=142 removed by soft count = 159\n",
      "--- gene=712, id=149 removed by soft count = 143\n",
      "--- gene=712, id=154 removed by soft count = 91\n",
      "--- number of probes: 201 , kept by blast: 194 , if remove dups: 194\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 713\n",
      "--- number of probes: 236 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 714\n",
      "--- gene=714, id=3 removed by soft count = 84\n",
      "--- gene=714, id=6 removed by soft count = 137\n",
      "--- gene=714, id=18 removed by soft count = 1417\n",
      "--- gene=714, id=54 removed by soft count = 3577\n",
      "--- gene=714, id=112 removed by soft count = 1351\n",
      "--- number of probes: 196 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 715\n",
      "--- gene=715, id=1 removed by soft count = 109\n",
      "--- gene=715, id=4 removed by soft count = 628\n",
      "--- gene=715, id=122 removed by soft count = 505\n",
      "--- gene=715, id=127 removed by soft count = 480\n",
      "--- gene=715, id=141 removed by soft count = 59\n",
      "--- gene=715, id=210 removed by soft count = 79\n",
      "--- gene=715, id=213 removed by soft count = 58\n",
      "--- number of probes: 300 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 716\n",
      "--- gene=716, id=18 removed by soft count = 100\n",
      "--- gene=716, id=36 removed by soft count = 53\n",
      "--- gene=716, id=37 removed by soft count = 361\n",
      "--- number of probes: 136 , kept by blast: 133 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 717\n",
      "--- gene=717, id=82 removed by soft count = 58\n",
      "--- gene=717, id=252 removed by soft count = 157\n",
      "--- number of probes: 282 , kept by blast: 280 , if remove dups: 280\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 718\n",
      "--- gene=718, id=294 removed by soft count = 99\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 719\n",
      "--- gene=719, id=62 removed by soft count = 128\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 720\n",
      "--- gene=720, id=53 removed by soft count = 745\n",
      "--- gene=720, id=54 removed by soft count = 233\n",
      "--- gene=720, id=56 removed by soft count = 79\n",
      "--- gene=720, id=58 removed by soft count = 113\n",
      "--- gene=720, id=61 removed by soft count = 75\n",
      "--- gene=720, id=62 removed by soft count = 148\n",
      "--- gene=720, id=138 removed by soft count = 682\n",
      "--- gene=720, id=197 removed by soft count = 64\n",
      "--- number of probes: 218 , kept by blast: 210 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 721\n",
      "--- gene=721, id=102 removed by soft count = 170\n",
      "--- gene=721, id=228 removed by soft count = 557\n",
      "--- number of probes: 270 , kept by blast: 268 , if remove dups: 268\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 722\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=722, id=6 removed by soft count = 111\n",
      "--- gene=722, id=8 removed by soft count = 260\n",
      "--- gene=722, id=11 removed by soft count = 841\n",
      "--- gene=722, id=28 removed by soft count = 61\n",
      "--- gene=722, id=87 removed by soft count = 76\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 723\n",
      "--- gene=723, id=6 removed by soft count = 785\n",
      "--- gene=723, id=8 removed by soft count = 85\n",
      "--- gene=723, id=14 removed by soft count = 760\n",
      "--- gene=723, id=25 removed by soft count = 76\n",
      "--- gene=723, id=49 removed by soft count = 241\n",
      "--- number of probes: 73 , kept by blast: 68 , if remove dups: 68\n",
      "-- number of probes kept for this region: 68\n",
      "-- checking probes in region: 724\n",
      "--- gene=724, id=20 removed by soft count = 66\n",
      "--- gene=724, id=35 removed by soft count = 68\n",
      "--- gene=724, id=122 removed by soft count = 80\n",
      "--- gene=724, id=140 removed by soft count = 79\n",
      "--- number of probes: 287 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 725\n",
      "--- gene=725, id=163 removed by soft count = 688\n",
      "--- gene=725, id=209 removed by soft count = 144\n",
      "--- number of probes: 271 , kept by blast: 269 , if remove dups: 269\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 726\n",
      "--- gene=726, id=53 removed by soft count = 203\n",
      "--- number of probes: 209 , kept by blast: 208 , if remove dups: 208\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 727\n",
      "--- gene=727, id=22 removed by soft count = 76\n",
      "--- gene=727, id=48 removed by soft count = 73\n",
      "--- gene=727, id=51 removed by soft count = 195\n",
      "--- gene=727, id=67 removed by soft count = 190\n",
      "--- gene=727, id=68 removed by soft count = 118\n",
      "--- gene=727, id=75 removed by soft count = 217\n",
      "--- gene=727, id=82 removed by soft count = 115\n",
      "--- gene=727, id=102 removed by soft count = 115\n",
      "--- gene=727, id=103 removed by soft count = 377\n",
      "--- number of probes: 138 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 728\n",
      "--- gene=728, id=73 removed by soft count = 208\n",
      "--- gene=728, id=234 removed by soft count = 68\n",
      "--- gene=728, id=238 removed by soft count = 414\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 729\n",
      "--- gene=729, id=1 removed by soft count = 74\n",
      "--- gene=729, id=4 removed by soft count = 58\n",
      "--- gene=729, id=5 removed by soft count = 69\n",
      "--- gene=729, id=7 removed by soft count = 63\n",
      "--- gene=729, id=119 removed by soft count = 56\n",
      "--- gene=729, id=122 removed by soft count = 253\n",
      "--- number of probes: 200 , kept by blast: 194 , if remove dups: 194\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 730\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 731\n",
      "--- gene=731, id=119 removed by soft count = 654\n",
      "--- number of probes: 174 , kept by blast: 173 , if remove dups: 172\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 732\n",
      "--- gene=732, id=0 removed by soft count = 89\n",
      "--- gene=732, id=4 removed by soft count = 236\n",
      "--- gene=732, id=6 removed by soft count = 114\n",
      "--- number of probes: 296 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 733\n",
      "--- gene=733, id=174 removed by soft count = 115\n",
      "--- number of probes: 274 , kept by blast: 273 , if remove dups: 273\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 734\n",
      "--- gene=734, id=83 removed by soft count = 663\n",
      "--- gene=734, id=84 removed by soft count = 59\n",
      "--- gene=734, id=131 removed by soft count = 4395\n",
      "--- gene=734, id=132 removed by soft count = 51\n",
      "--- gene=734, id=133 removed by soft count = 248\n",
      "--- number of probes: 225 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 735\n",
      "--- gene=735, id=37 removed by soft count = 307\n",
      "--- gene=735, id=144 removed by soft count = 89\n",
      "--- number of probes: 206 , kept by blast: 204 , if remove dups: 204\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 736\n",
      "--- gene=736, id=146 removed by soft count = 108\n",
      "--- gene=736, id=199 removed by soft count = 135\n",
      "--- gene=736, id=286 removed by soft count = 104\n",
      "--- gene=736, id=287 removed by soft count = 176\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 737\n",
      "--- gene=737, id=0 removed by soft count = 216\n",
      "--- gene=737, id=1 removed by soft count = 288\n",
      "--- gene=737, id=3 removed by soft count = 64\n",
      "--- gene=737, id=223 removed by soft count = 256\n",
      "--- number of probes: 252 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 738\n",
      "--- gene=738, id=19 removed by soft count = 591\n",
      "--- gene=738, id=61 removed by soft count = 55\n",
      "--- gene=738, id=70 removed by soft count = 169\n",
      "--- gene=738, id=91 removed by soft count = 188\n",
      "--- gene=738, id=102 removed by soft count = 222\n",
      "--- gene=738, id=201 removed by soft count = 174\n",
      "--- gene=738, id=202 removed by soft count = 67\n",
      "--- gene=738, id=205 removed by soft count = 232\n",
      "--- gene=738, id=208 removed by soft count = 150\n",
      "--- number of probes: 221 , kept by blast: 212 , if remove dups: 212\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 739\n",
      "--- gene=739, id=19 removed by soft count = 228\n",
      "--- gene=739, id=160 removed by soft count = 4032\n",
      "--- gene=739, id=197 removed by soft count = 68\n",
      "--- gene=739, id=220 removed by soft count = 858\n",
      "--- number of probes: 229 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 740\n",
      "--- gene=740, id=2 removed by soft count = 87\n",
      "--- gene=740, id=13 removed by soft count = 703\n",
      "--- gene=740, id=24 removed by soft count = 67\n",
      "--- gene=740, id=35 removed by soft count = 68\n",
      "--- gene=740, id=51 removed by soft count = 141\n",
      "--- gene=740, id=56 removed by soft count = 165\n",
      "--- gene=740, id=59 removed by soft count = 405\n",
      "--- gene=740, id=60 removed by soft count = 65\n",
      "--- gene=740, id=61 removed by soft count = 87\n",
      "--- gene=740, id=62 removed by soft count = 111\n",
      "--- gene=740, id=63 removed by soft count = 121\n",
      "--- gene=740, id=64 removed by soft count = 101\n",
      "--- gene=740, id=65 removed by soft count = 66\n",
      "--- gene=740, id=66 removed by soft count = 430\n",
      "--- gene=740, id=67 removed by soft count = 147\n",
      "--- gene=740, id=69 removed by soft count = 184\n",
      "--- gene=740, id=71 removed by soft count = 74\n",
      "--- gene=740, id=72 removed by soft count = 141\n",
      "--- gene=740, id=73 removed by soft count = 123\n",
      "--- gene=740, id=74 removed by soft count = 55\n",
      "--- gene=740, id=75 removed by soft count = 118\n",
      "--- gene=740, id=180 removed by soft count = 210\n",
      "--- gene=740, id=183 removed by soft count = 494\n",
      "--- gene=740, id=187 removed by soft count = 175\n",
      "--- gene=740, id=190 removed by soft count = 103\n",
      "--- number of probes: 249 , kept by blast: 224 , if remove dups: 224\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 741\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=741, id=154 removed by soft count = 70\n",
      "--- gene=741, id=176 removed by soft count = 445\n",
      "--- gene=741, id=255 removed by soft count = 62\n",
      "--- gene=741, id=256 removed by soft count = 137\n",
      "--- number of probes: 266 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 742\n",
      "--- gene=742, id=11 removed by soft count = 129\n",
      "--- gene=742, id=214 removed by soft count = 235\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 743\n",
      "--- gene=743, id=104 removed by soft count = 229\n",
      "--- gene=743, id=148 removed by soft count = 51\n",
      "--- gene=743, id=186 removed by soft count = 152\n",
      "--- number of probes: 297 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 744\n",
      "--- gene=744, id=35 removed by soft count = 498\n",
      "--- gene=744, id=36 removed by soft count = 66\n",
      "--- gene=744, id=254 removed by soft count = 130\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 745\n",
      "--- gene=745, id=204 removed by soft count = 95\n",
      "--- gene=745, id=231 removed by soft count = 105\n",
      "--- gene=745, id=232 removed by soft count = 144\n",
      "--- gene=745, id=234 removed by soft count = 227\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 748\n",
      "--- gene=748, id=97 removed by soft count = 53\n",
      "--- gene=748, id=161 removed by soft count = 78\n",
      "--- gene=748, id=162 removed by soft count = 176\n",
      "--- gene=748, id=163 removed by soft count = 63\n",
      "--- gene=748, id=187 removed by soft count = 363\n",
      "--- gene=748, id=188 removed by soft count = 503\n",
      "--- gene=748, id=194 removed by soft count = 74\n",
      "--- gene=748, id=196 removed by soft count = 52\n",
      "--- number of probes: 254 , kept by blast: 246 , if remove dups: 246\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 749\n",
      "--- gene=749, id=0 removed by soft count = 546\n",
      "--- gene=749, id=1 removed by soft count = 63\n",
      "--- gene=749, id=2 removed by soft count = 134\n",
      "--- gene=749, id=3 removed by soft count = 141\n",
      "--- gene=749, id=4 removed by soft count = 426\n",
      "--- gene=749, id=34 removed by soft count = 55\n",
      "--- gene=749, id=35 removed by soft count = 62\n",
      "--- gene=749, id=65 removed by soft count = 54\n",
      "--- gene=749, id=73 removed by soft count = 188\n",
      "--- number of probes: 199 , kept by blast: 190 , if remove dups: 190\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 750\n",
      "--- gene=750, id=261 removed by soft count = 24370\n",
      "--- gene=750, id=265 removed by soft count = 83\n",
      "--- number of probes: 287 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 751\n",
      "--- gene=751, id=29 removed by soft count = 294\n",
      "--- gene=751, id=115 removed by soft count = 77\n",
      "--- gene=751, id=117 removed by soft count = 77\n",
      "--- gene=751, id=118 removed by soft count = 72\n",
      "--- gene=751, id=119 removed by soft count = 181\n",
      "--- gene=751, id=120 removed by soft count = 67\n",
      "--- gene=751, id=124 removed by soft count = 89\n",
      "--- gene=751, id=199 removed by soft count = 444\n",
      "--- gene=751, id=200 removed by soft count = 61\n",
      "--- gene=751, id=201 removed by soft count = 383\n",
      "--- gene=751, id=202 removed by soft count = 170\n",
      "--- gene=751, id=240 removed by soft count = 315\n",
      "--- number of probes: 282 , kept by blast: 270 , if remove dups: 270\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 752\n",
      "--- gene=752, id=17 removed by soft count = 107\n",
      "--- gene=752, id=54 removed by soft count = 155\n",
      "--- gene=752, id=55 removed by soft count = 82\n",
      "--- gene=752, id=96 removed by soft count = 745\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 753\n",
      "--- gene=753, id=5 removed by soft count = 64\n",
      "--- gene=753, id=6 removed by soft count = 57\n",
      "--- gene=753, id=7 removed by soft count = 58\n",
      "--- gene=753, id=8 removed by soft count = 54\n",
      "--- gene=753, id=9 removed by soft count = 51\n",
      "--- gene=753, id=10 removed by soft count = 96\n",
      "--- gene=753, id=11 removed by soft count = 54\n",
      "--- gene=753, id=16 removed by soft count = 253\n",
      "--- gene=753, id=19 removed by soft count = 69\n",
      "--- gene=753, id=20 removed by soft count = 181\n",
      "--- gene=753, id=21 removed by soft count = 374\n",
      "--- gene=753, id=22 removed by soft count = 91\n",
      "--- gene=753, id=28 removed by soft count = 66\n",
      "--- gene=753, id=40 removed by soft count = 92\n",
      "--- gene=753, id=100 removed by soft count = 57\n",
      "--- gene=753, id=124 removed by soft count = 83\n",
      "--- gene=753, id=125 removed by soft count = 188\n",
      "--- gene=753, id=126 removed by soft count = 256\n",
      "--- gene=753, id=139 removed by soft count = 77\n",
      "--- number of probes: 145 , kept by blast: 126 , if remove dups: 126\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 754\n",
      "--- gene=754, id=167 removed by hard count = 3\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 755\n",
      "--- gene=755, id=0 removed by soft count = 359\n",
      "--- gene=755, id=1 removed by soft count = 320\n",
      "--- gene=755, id=3 removed by soft count = 61\n",
      "--- gene=755, id=4 removed by soft count = 56\n",
      "--- gene=755, id=7 removed by soft count = 126\n",
      "--- gene=755, id=16 removed by soft count = 528\n",
      "--- gene=755, id=18 removed by soft count = 734\n",
      "--- gene=755, id=20 removed by soft count = 355\n",
      "--- gene=755, id=22 removed by soft count = 267\n",
      "--- gene=755, id=25 removed by soft count = 84\n",
      "--- gene=755, id=26 removed by soft count = 207\n",
      "--- gene=755, id=29 removed by soft count = 61\n",
      "--- gene=755, id=32 removed by soft count = 74\n",
      "--- gene=755, id=36 removed by soft count = 146\n",
      "--- gene=755, id=39 removed by soft count = 83\n",
      "--- gene=755, id=41 removed by soft count = 92\n",
      "--- gene=755, id=43 removed by soft count = 345\n",
      "--- gene=755, id=44 removed by soft count = 107\n",
      "--- gene=755, id=46 removed by soft count = 74\n",
      "--- gene=755, id=49 removed by soft count = 520\n",
      "--- gene=755, id=51 removed by soft count = 743\n",
      "--- gene=755, id=54 removed by soft count = 259\n",
      "--- gene=755, id=57 removed by soft count = 177\n",
      "--- number of probes: 59 , kept by blast: 36 , if remove dups: 36\n",
      "-- number of probes kept for this region: 36\n",
      "-- checking probes in region: 756\n",
      "--- gene=756, id=21 removed by soft count = 231\n",
      "--- gene=756, id=88 removed by soft count = 95\n",
      "--- gene=756, id=113 removed by soft count = 153\n",
      "--- gene=756, id=281 removed by soft count = 92\n",
      "--- gene=756, id=282 removed by soft count = 87\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 757\n",
      "--- gene=757, id=2 removed by soft count = 232\n",
      "--- gene=757, id=22 removed by soft count = 93\n",
      "--- gene=757, id=47 removed by soft count = 81\n",
      "--- gene=757, id=48 removed by soft count = 115\n",
      "--- gene=757, id=59 removed by soft count = 124\n",
      "--- gene=757, id=92 removed by soft count = 92\n",
      "--- gene=757, id=113 removed by soft count = 57\n",
      "--- gene=757, id=114 removed by soft count = 57\n",
      "--- gene=757, id=142 removed by soft count = 410\n",
      "--- gene=757, id=143 removed by soft count = 398\n",
      "--- gene=757, id=145 removed by soft count = 389\n",
      "--- gene=757, id=146 removed by soft count = 100\n",
      "--- gene=757, id=149 removed by soft count = 1095\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=757, id=187 removed by soft count = 172\n",
      "--- number of probes: 192 , kept by blast: 178 , if remove dups: 178\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 758\n",
      "--- gene=758, id=59 removed by soft count = 70\n",
      "--- gene=758, id=77 removed by soft count = 132\n",
      "--- gene=758, id=78 removed by soft count = 207\n",
      "--- gene=758, id=79 removed by soft count = 112\n",
      "--- gene=758, id=81 removed by soft count = 63\n",
      "--- gene=758, id=82 removed by soft count = 359\n",
      "--- gene=758, id=83 removed by soft count = 272\n",
      "--- gene=758, id=84 removed by soft count = 233\n",
      "--- gene=758, id=85 removed by soft count = 248\n",
      "--- gene=758, id=86 removed by soft count = 89\n",
      "--- gene=758, id=87 removed by soft count = 237\n",
      "--- gene=758, id=88 removed by soft count = 272\n",
      "--- gene=758, id=89 removed by soft count = 66\n",
      "--- gene=758, id=90 removed by soft count = 286\n",
      "--- gene=758, id=91 removed by soft count = 330\n",
      "--- gene=758, id=96 removed by soft count = 67\n",
      "--- gene=758, id=124 removed by soft count = 136\n",
      "--- gene=758, id=134 removed by soft count = 1080\n",
      "--- gene=758, id=135 removed by soft count = 147\n",
      "--- gene=758, id=136 removed by soft count = 75\n",
      "--- gene=758, id=137 removed by soft count = 53\n",
      "--- number of probes: 141 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 759\n",
      "--- gene=759, id=208 removed by soft count = 71\n",
      "--- gene=759, id=212 removed by soft count = 73\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 760\n",
      "--- gene=760, id=198 removed by soft count = 2081\n",
      "--- gene=760, id=206 removed by soft count = 378\n",
      "--- gene=760, id=207 removed by soft count = 64\n",
      "--- number of probes: 220 , kept by blast: 217 , if remove dups: 217\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 761\n",
      "--- gene=761, id=80 removed by soft count = 280\n",
      "--- gene=761, id=83 removed by soft count = 108\n",
      "--- gene=761, id=84 removed by soft count = 65\n",
      "--- gene=761, id=86 removed by soft count = 61\n",
      "--- gene=761, id=192 removed by soft count = 86\n",
      "--- number of probes: 222 , kept by blast: 217 , if remove dups: 216\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 762\n",
      "--- gene=762, id=7 removed by soft count = 74\n",
      "--- gene=762, id=50 removed by soft count = 5734\n",
      "--- gene=762, id=60 removed by soft count = 112\n",
      "--- gene=762, id=62 removed by soft count = 108\n",
      "--- gene=762, id=100 removed by soft count = 415\n",
      "--- number of probes: 140 , kept by blast: 135 , if remove dups: 135\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 763\n",
      "--- gene=763, id=31 removed by soft count = 74\n",
      "--- gene=763, id=62 removed by soft count = 392\n",
      "--- gene=763, id=70 removed by soft count = 335\n",
      "--- gene=763, id=71 removed by soft count = 511\n",
      "--- gene=763, id=72 removed by soft count = 64\n",
      "--- number of probes: 270 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 764\n",
      "--- gene=764, id=18 removed by soft count = 387\n",
      "--- gene=764, id=23 removed by soft count = 62\n",
      "--- gene=764, id=206 removed by soft count = 1363\n",
      "--- gene=764, id=207 removed by soft count = 253\n",
      "--- gene=764, id=223 removed by soft count = 132\n",
      "--- gene=764, id=246 removed by soft count = 898\n",
      "--- gene=764, id=249 removed by soft count = 146\n",
      "--- number of probes: 261 , kept by blast: 254 , if remove dups: 254\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 765\n",
      "--- gene=765, id=3 removed by soft count = 226\n",
      "--- gene=765, id=100 removed by soft count = 93\n",
      "--- gene=765, id=102 removed by soft count = 187\n",
      "--- gene=765, id=103 removed by soft count = 74\n",
      "--- gene=765, id=114 removed by soft count = 1030\n",
      "--- gene=765, id=145 removed by soft count = 534\n",
      "--- gene=765, id=146 removed by soft count = 623\n",
      "--- gene=765, id=149 removed by soft count = 168\n",
      "--- number of probes: 191 , kept by blast: 183 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 766\n",
      "--- gene=766, id=55 removed by soft count = 254\n",
      "--- gene=766, id=57 removed by soft count = 112\n",
      "--- gene=766, id=58 removed by soft count = 247\n",
      "--- gene=766, id=99 removed by soft count = 152\n",
      "--- gene=766, id=100 removed by soft count = 238\n",
      "--- gene=766, id=216 removed by soft count = 66\n",
      "--- gene=766, id=240 removed by soft count = 150\n",
      "--- gene=766, id=242 removed by soft count = 166\n",
      "--- number of probes: 254 , kept by blast: 246 , if remove dups: 246\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 767\n",
      "--- gene=767, id=128 removed by soft count = 3389\n",
      "--- gene=767, id=160 removed by soft count = 371\n",
      "--- gene=767, id=161 removed by soft count = 58\n",
      "--- gene=767, id=163 removed by soft count = 667\n",
      "--- gene=767, id=164 removed by soft count = 171\n",
      "--- number of probes: 199 , kept by blast: 194 , if remove dups: 194\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 768\n",
      "--- gene=768, id=23 removed by soft count = 150\n",
      "--- gene=768, id=24 removed by soft count = 52\n",
      "--- gene=768, id=26 removed by soft count = 69\n",
      "--- gene=768, id=227 removed by soft count = 88\n",
      "--- number of probes: 278 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 769\n",
      "--- gene=769, id=31 removed by soft count = 62\n",
      "--- gene=769, id=40 removed by soft count = 917\n",
      "--- gene=769, id=41 removed by soft count = 217\n",
      "--- number of probes: 160 , kept by blast: 157 , if remove dups: 157\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 770\n",
      "--- gene=770, id=13 removed by soft count = 1666\n",
      "--- gene=770, id=69 removed by soft count = 85\n",
      "--- number of probes: 81 , kept by blast: 79 , if remove dups: 79\n",
      "-- number of probes kept for this region: 79\n",
      "-- checking probes in region: 774\n",
      "--- gene=774, id=0 removed by soft count = 148\n",
      "--- gene=774, id=6 removed by soft count = 83\n",
      "--- gene=774, id=29 removed by soft count = 78\n",
      "--- gene=774, id=31 removed by soft count = 412\n",
      "--- gene=774, id=35 removed by soft count = 112\n",
      "--- gene=774, id=36 removed by soft count = 306\n",
      "--- gene=774, id=38 removed by soft count = 59\n",
      "--- gene=774, id=43 removed by soft count = 115\n",
      "--- gene=774, id=44 removed by soft count = 311\n",
      "--- gene=774, id=45 removed by soft count = 60\n",
      "--- gene=774, id=47 removed by soft count = 53\n",
      "--- gene=774, id=64 removed by soft count = 168\n",
      "--- gene=774, id=67 removed by soft count = 73\n",
      "--- gene=774, id=71 removed by soft count = 51\n",
      "--- gene=774, id=76 removed by soft count = 61\n",
      "--- gene=774, id=86 removed by soft count = 120\n",
      "--- gene=774, id=87 removed by soft count = 74\n",
      "--- gene=774, id=92 removed by soft count = 60\n",
      "--- gene=774, id=112 removed by soft count = 90\n",
      "--- gene=774, id=113 removed by soft count = 240\n",
      "--- gene=774, id=114 removed by soft count = 100\n",
      "--- gene=774, id=133 removed by soft count = 107\n",
      "--- gene=774, id=134 removed by soft count = 96\n",
      "--- gene=774, id=135 removed by soft count = 123\n",
      "--- gene=774, id=137 removed by soft count = 264\n",
      "--- gene=774, id=140 removed by soft count = 51\n",
      "--- gene=774, id=141 removed by soft count = 246\n",
      "--- gene=774, id=143 removed by soft count = 91\n",
      "--- gene=774, id=146 removed by soft count = 79\n",
      "--- gene=774, id=190 removed by soft count = 59\n",
      "--- gene=774, id=196 removed by soft count = 328\n",
      "--- number of probes: 202 , kept by blast: 171 , if remove dups: 171\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 775\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=775, id=0 removed by soft count = 1471\n",
      "--- gene=775, id=1 removed by soft count = 12736\n",
      "--- gene=775, id=2 removed by soft count = 4524\n",
      "--- gene=775, id=3 removed by soft count = 4275\n",
      "--- gene=775, id=4 removed by soft count = 373\n",
      "--- gene=775, id=5 removed by soft count = 78\n",
      "--- gene=775, id=6 removed by soft count = 203\n",
      "--- gene=775, id=10 removed by soft count = 379\n",
      "--- gene=775, id=106 removed by soft count = 63\n",
      "--- gene=775, id=107 removed by soft count = 53\n",
      "--- gene=775, id=111 removed by soft count = 51\n",
      "--- gene=775, id=112 removed by soft count = 74\n",
      "--- gene=775, id=113 removed by soft count = 56\n",
      "--- gene=775, id=119 removed by soft count = 69\n",
      "--- gene=775, id=120 removed by soft count = 62\n",
      "--- gene=775, id=121 removed by soft count = 210\n",
      "--- gene=775, id=122 removed by soft count = 392\n",
      "--- gene=775, id=123 removed by soft count = 311\n",
      "--- gene=775, id=124 removed by soft count = 388\n",
      "--- gene=775, id=125 removed by soft count = 273\n",
      "--- gene=775, id=174 removed by soft count = 70\n",
      "--- gene=775, id=175 removed by soft count = 92\n",
      "--- gene=775, id=191 removed by soft count = 57\n",
      "--- gene=775, id=195 removed by soft count = 54\n",
      "--- gene=775, id=197 removed by soft count = 63\n",
      "--- gene=775, id=200 removed by soft count = 220\n",
      "--- gene=775, id=201 removed by soft count = 162\n",
      "--- number of probes: 202 , kept by blast: 175 , if remove dups: 175\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 776\n",
      "--- gene=776, id=0 removed by soft count = 87\n",
      "--- gene=776, id=17 removed by soft count = 84\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 777\n",
      "--- gene=777, id=68 removed by soft count = 56\n",
      "--- gene=777, id=140 removed by soft count = 60\n",
      "--- number of probes: 299 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 778\n",
      "--- gene=778, id=14 removed by soft count = 99\n",
      "--- gene=778, id=221 removed by soft count = 848\n",
      "--- number of probes: 252 , kept by blast: 250 , if remove dups: 249\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 779\n",
      "--- gene=779, id=87 removed by soft count = 117\n",
      "--- gene=779, id=120 removed by soft count = 62\n",
      "--- gene=779, id=121 removed by soft count = 153\n",
      "--- gene=779, id=124 removed by soft count = 537\n",
      "--- number of probes: 208 , kept by blast: 204 , if remove dups: 204\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 780\n",
      "--- gene=780, id=65 removed by soft count = 63\n",
      "--- gene=780, id=75 removed by soft count = 166\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 782\n",
      "--- gene=782, id=163 removed by soft count = 148\n",
      "--- gene=782, id=164 removed by soft count = 100\n",
      "--- gene=782, id=165 removed by soft count = 55\n",
      "--- number of probes: 206 , kept by blast: 203 , if remove dups: 203\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 783\n",
      "--- gene=783, id=78 removed by soft count = 202\n",
      "--- gene=783, id=79 removed by soft count = 80\n",
      "--- gene=783, id=103 removed by soft count = 498\n",
      "--- gene=783, id=104 removed by soft count = 644\n",
      "--- gene=783, id=106 removed by soft count = 55\n",
      "--- gene=783, id=127 removed by soft count = 65\n",
      "--- number of probes: 285 , kept by blast: 279 , if remove dups: 279\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 784\n",
      "--- gene=784, id=28 removed by soft count = 184\n",
      "--- gene=784, id=114 removed by soft count = 247\n",
      "--- gene=784, id=115 removed by soft count = 199\n",
      "--- gene=784, id=122 removed by soft count = 109\n",
      "--- gene=784, id=137 removed by soft count = 74\n",
      "--- gene=784, id=142 removed by soft count = 884\n",
      "--- gene=784, id=158 removed by soft count = 347\n",
      "--- gene=784, id=185 removed by soft count = 4771\n",
      "--- gene=784, id=187 removed by soft count = 169\n",
      "--- number of probes: 196 , kept by blast: 187 , if remove dups: 187\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 785\n",
      "--- gene=785, id=15 removed by soft count = 93\n",
      "--- gene=785, id=84 removed by soft count = 64\n",
      "--- gene=785, id=135 removed by soft count = 76\n",
      "--- gene=785, id=159 removed by soft count = 136\n",
      "--- gene=785, id=162 removed by soft count = 52\n",
      "--- gene=785, id=173 removed by soft count = 95\n",
      "--- gene=785, id=176 removed by soft count = 170\n",
      "--- gene=785, id=177 removed by soft count = 233\n",
      "--- gene=785, id=178 removed by soft count = 126\n",
      "--- gene=785, id=180 removed by soft count = 66\n",
      "--- gene=785, id=183 removed by soft count = 1307\n",
      "--- gene=785, id=185 removed by soft count = 145\n",
      "--- gene=785, id=195 removed by soft count = 132\n",
      "--- number of probes: 295 , kept by blast: 282 , if remove dups: 282\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 786\n",
      "--- gene=786, id=129 removed by soft count = 118\n",
      "--- gene=786, id=130 removed by soft count = 77\n",
      "--- number of probes: 234 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 787\n",
      "--- gene=787, id=214 removed by soft count = 134\n",
      "--- number of probes: 293 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 788\n",
      "--- gene=788, id=1 removed by soft count = 164\n",
      "--- gene=788, id=2 removed by soft count = 115\n",
      "--- gene=788, id=7 removed by soft count = 236\n",
      "--- gene=788, id=9 removed by soft count = 325\n",
      "--- gene=788, id=102 removed by soft count = 63\n",
      "--- gene=788, id=103 removed by soft count = 108\n",
      "--- gene=788, id=114 removed by soft count = 364\n",
      "--- gene=788, id=120 removed by soft count = 4304\n",
      "--- number of probes: 182 , kept by blast: 174 , if remove dups: 173\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 789\n",
      "--- gene=789, id=30 removed by soft count = 123\n",
      "--- gene=789, id=57 removed by soft count = 2455\n",
      "--- gene=789, id=81 removed by soft count = 9815\n",
      "--- gene=789, id=89 removed by soft count = 51\n",
      "--- gene=789, id=95 removed by soft count = 397\n",
      "--- gene=789, id=173 removed by soft count = 242\n",
      "--- gene=789, id=174 removed by soft count = 111\n",
      "--- number of probes: 209 , kept by blast: 202 , if remove dups: 202\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 790\n",
      "--- gene=790, id=22 removed by soft count = 64\n",
      "--- number of probes: 91 , kept by blast: 90 , if remove dups: 87\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 87\n",
      "-- checking probes in region: 791\n",
      "--- gene=791, id=31 removed by soft count = 60\n",
      "--- gene=791, id=84 removed by soft count = 364\n",
      "--- gene=791, id=149 removed by soft count = 345\n",
      "--- gene=791, id=154 removed by soft count = 60\n",
      "--- number of probes: 164 , kept by blast: 160 , if remove dups: 160\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 792\n",
      "--- gene=792, id=10 removed by soft count = 227\n",
      "--- gene=792, id=11 removed by soft count = 473\n",
      "--- gene=792, id=12 removed by soft count = 483\n",
      "--- gene=792, id=14 removed by soft count = 320\n",
      "--- gene=792, id=15 removed by soft count = 570\n",
      "--- gene=792, id=17 removed by soft count = 52\n",
      "--- gene=792, id=37 removed by hard count = 3\n",
      "--- gene=792, id=38 removed by soft count = 104\n",
      "--- gene=792, id=40 removed by soft count = 3262\n",
      "--- gene=792, id=42 removed by soft count = 52\n",
      "--- gene=792, id=45 removed by soft count = 127\n",
      "--- gene=792, id=47 removed by soft count = 101\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=792, id=48 removed by soft count = 795\n",
      "--- gene=792, id=50 removed by soft count = 194\n",
      "--- gene=792, id=56 removed by soft count = 61\n",
      "--- gene=792, id=58 removed by soft count = 83\n",
      "--- gene=792, id=70 removed by soft count = 64\n",
      "--- gene=792, id=167 removed by soft count = 151\n",
      "--- number of probes: 168 , kept by blast: 150 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 793\n",
      "--- gene=793, id=130 removed by soft count = 1245\n",
      "--- gene=793, id=133 removed by soft count = 53\n",
      "--- gene=793, id=183 removed by soft count = 142\n",
      "--- gene=793, id=184 removed by soft count = 219\n",
      "--- gene=793, id=186 removed by soft count = 98\n",
      "--- number of probes: 231 , kept by blast: 226 , if remove dups: 226\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 794\n",
      "--- gene=794, id=30 removed by soft count = 62\n",
      "--- gene=794, id=53 removed by soft count = 55\n",
      "--- gene=794, id=59 removed by soft count = 104\n",
      "--- gene=794, id=81 removed by soft count = 199\n",
      "--- gene=794, id=88 removed by soft count = 80\n",
      "--- gene=794, id=261 removed by soft count = 109\n",
      "--- number of probes: 298 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 795\n",
      "--- gene=795, id=1 removed by soft count = 68\n",
      "--- gene=795, id=3 removed by soft count = 155\n",
      "--- gene=795, id=4 removed by soft count = 88\n",
      "--- gene=795, id=5 removed by soft count = 114\n",
      "--- gene=795, id=14 removed by soft count = 60\n",
      "--- gene=795, id=16 removed by soft count = 117\n",
      "--- gene=795, id=47 removed by soft count = 4904\n",
      "--- gene=795, id=48 removed by soft count = 4149\n",
      "--- gene=795, id=49 removed by soft count = 465\n",
      "--- gene=795, id=57 removed by soft count = 95\n",
      "--- gene=795, id=110 removed by soft count = 226\n",
      "--- gene=795, id=112 removed by soft count = 52\n",
      "--- number of probes: 126 , kept by blast: 114 , if remove dups: 114\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 114\n",
      "-- checking probes in region: 796\n",
      "--- gene=796, id=36 removed by soft count = 556\n",
      "--- gene=796, id=37 removed by soft count = 62\n",
      "--- gene=796, id=101 removed by soft count = 195\n",
      "--- gene=796, id=102 removed by soft count = 174\n",
      "--- gene=796, id=182 removed by soft count = 140\n",
      "--- gene=796, id=200 removed by soft count = 79\n",
      "--- number of probes: 289 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 797\n",
      "--- gene=797, id=80 removed by soft count = 503\n",
      "--- gene=797, id=147 removed by soft count = 119\n",
      "--- gene=797, id=148 removed by soft count = 84\n",
      "--- gene=797, id=154 removed by soft count = 936\n",
      "--- gene=797, id=192 removed by soft count = 86\n",
      "--- gene=797, id=201 removed by soft count = 128\n",
      "--- gene=797, id=204 removed by soft count = 96\n",
      "--- gene=797, id=205 removed by soft count = 149\n",
      "--- gene=797, id=206 removed by soft count = 237\n",
      "--- gene=797, id=218 removed by soft count = 203\n",
      "--- number of probes: 249 , kept by blast: 239 , if remove dups: 239\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 798\n",
      "--- gene=798, id=5 removed by soft count = 54\n",
      "--- gene=798, id=11 removed by soft count = 92\n",
      "--- gene=798, id=14 removed by soft count = 164\n",
      "--- gene=798, id=19 removed by soft count = 73\n",
      "--- gene=798, id=20 removed by soft count = 249\n",
      "--- gene=798, id=21 removed by soft count = 115\n",
      "--- gene=798, id=22 removed by soft count = 54\n",
      "--- gene=798, id=23 removed by soft count = 67\n",
      "--- gene=798, id=61 removed by soft count = 115\n",
      "--- gene=798, id=137 removed by soft count = 59\n",
      "--- gene=798, id=225 removed by soft count = 1054\n",
      "--- gene=798, id=227 removed by soft count = 61\n",
      "--- number of probes: 244 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 799\n",
      "--- number of probes: 289 , kept by blast: 289 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 800\n",
      "--- gene=800, id=35 removed by soft count = 156\n",
      "--- gene=800, id=80 removed by soft count = 316\n",
      "--- gene=800, id=81 removed by soft count = 1001\n",
      "--- gene=800, id=83 removed by soft count = 133\n",
      "--- gene=800, id=87 removed by soft count = 102\n",
      "--- gene=800, id=93 removed by soft count = 166\n",
      "--- gene=800, id=95 removed by soft count = 121\n",
      "--- gene=800, id=96 removed by soft count = 307\n",
      "--- gene=800, id=131 removed by soft count = 446\n",
      "--- gene=800, id=144 removed by soft count = 95\n",
      "--- gene=800, id=146 removed by soft count = 801\n",
      "--- number of probes: 196 , kept by blast: 185 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 801\n",
      "--- gene=801, id=55 removed by soft count = 180\n",
      "--- gene=801, id=102 removed by soft count = 76\n",
      "--- gene=801, id=105 removed by soft count = 54\n",
      "--- gene=801, id=106 removed by soft count = 63\n",
      "--- gene=801, id=153 removed by soft count = 54\n",
      "--- gene=801, id=155 removed by soft count = 61\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 802\n",
      "--- gene=802, id=3 removed by soft count = 63\n",
      "--- gene=802, id=12 removed by soft count = 64\n",
      "--- number of probes: 257 , kept by blast: 255 , if remove dups: 255\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 803\n",
      "--- gene=803, id=146 removed by soft count = 109\n",
      "--- gene=803, id=184 removed by soft count = 210\n",
      "--- gene=803, id=185 removed by soft count = 350\n",
      "--- gene=803, id=186 removed by soft count = 101\n",
      "--- number of probes: 289 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 804\n",
      "--- gene=804, id=126 removed by soft count = 189\n",
      "--- number of probes: 220 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 805\n",
      "--- gene=805, id=25 removed by soft count = 4171\n",
      "--- gene=805, id=35 removed by soft count = 90\n",
      "--- gene=805, id=47 removed by soft count = 156\n",
      "--- gene=805, id=91 removed by soft count = 56\n",
      "--- gene=805, id=94 removed by soft count = 76\n",
      "--- gene=805, id=95 removed by soft count = 55\n",
      "--- gene=805, id=96 removed by soft count = 78\n",
      "--- gene=805, id=102 removed by soft count = 89\n",
      "--- gene=805, id=115 removed by soft count = 277\n",
      "--- gene=805, id=132 removed by soft count = 92\n",
      "--- number of probes: 168 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 806\n",
      "--- gene=806, id=281 removed by soft count = 169\n",
      "--- gene=806, id=282 removed by soft count = 354\n",
      "--- number of probes: 291 , kept by blast: 289 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 807\n",
      "--- number of probes: 233 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 808\n",
      "--- gene=808, id=26 removed by soft count = 70\n",
      "--- gene=808, id=27 removed by soft count = 503\n",
      "--- gene=808, id=28 removed by soft count = 789\n",
      "--- gene=808, id=30 removed by soft count = 128\n",
      "--- gene=808, id=31 removed by soft count = 1010\n",
      "--- gene=808, id=33 removed by soft count = 53\n",
      "--- gene=808, id=123 removed by soft count = 75\n",
      "--- gene=808, id=129 removed by soft count = 148\n",
      "--- gene=808, id=130 removed by soft count = 87\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=808, id=153 removed by soft count = 766\n",
      "--- gene=808, id=158 removed by soft count = 102\n",
      "--- number of probes: 197 , kept by blast: 186 , if remove dups: 186\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 809\n",
      "--- gene=809, id=211 removed by soft count = 603\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 810\n",
      "--- gene=810, id=71 removed by soft count = 1201\n",
      "--- gene=810, id=95 removed by soft count = 430\n",
      "--- gene=810, id=97 removed by soft count = 653\n",
      "--- gene=810, id=101 removed by soft count = 156\n",
      "--- gene=810, id=107 removed by soft count = 150\n",
      "--- number of probes: 160 , kept by blast: 155 , if remove dups: 155\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 811\n",
      "--- gene=811, id=173 removed by soft count = 183\n",
      "--- gene=811, id=200 removed by soft count = 58\n",
      "--- number of probes: 234 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 812\n",
      "--- gene=812, id=138 removed by soft count = 763\n",
      "--- gene=812, id=198 removed by soft count = 77\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 813\n",
      "--- gene=813, id=6 removed by soft count = 959\n",
      "--- gene=813, id=9 removed by soft count = 100\n",
      "--- gene=813, id=124 removed by soft count = 66\n",
      "--- number of probes: 187 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 814\n",
      "--- gene=814, id=29 removed by soft count = 239\n",
      "--- gene=814, id=92 removed by soft count = 101\n",
      "--- gene=814, id=122 removed by soft count = 380\n",
      "--- gene=814, id=126 removed by soft count = 231\n",
      "--- gene=814, id=136 removed by soft count = 87\n",
      "--- gene=814, id=177 removed by soft count = 101\n",
      "--- gene=814, id=179 removed by soft count = 160\n",
      "--- gene=814, id=258 removed by soft count = 2208\n",
      "--- gene=814, id=259 removed by soft count = 299\n",
      "--- number of probes: 275 , kept by blast: 266 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 815\n",
      "--- number of probes: 296 , kept by blast: 296 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 816\n",
      "--- gene=816, id=42 removed by soft count = 131\n",
      "--- gene=816, id=43 removed by soft count = 77\n",
      "--- gene=816, id=130 removed by soft count = 314\n",
      "--- gene=816, id=131 removed by soft count = 191\n",
      "--- number of probes: 209 , kept by blast: 205 , if remove dups: 205\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 817\n",
      "--- gene=817, id=1 removed by soft count = 70\n",
      "--- gene=817, id=4 removed by soft count = 157\n",
      "--- gene=817, id=115 removed by soft count = 208\n",
      "--- gene=817, id=116 removed by soft count = 99\n",
      "--- gene=817, id=137 removed by soft count = 121\n",
      "--- gene=817, id=159 removed by soft count = 379\n",
      "--- gene=817, id=192 removed by soft count = 129\n",
      "--- gene=817, id=195 removed by soft count = 165\n",
      "--- gene=817, id=200 removed by soft count = 212\n",
      "--- gene=817, id=201 removed by soft count = 53\n",
      "--- gene=817, id=202 removed by soft count = 187\n",
      "--- number of probes: 204 , kept by blast: 193 , if remove dups: 193\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 819\n",
      "--- gene=819, id=51 removed by soft count = 159\n",
      "--- gene=819, id=53 removed by soft count = 332\n",
      "--- gene=819, id=54 removed by soft count = 114\n",
      "--- gene=819, id=144 removed by soft count = 54\n",
      "--- gene=819, id=214 removed by soft count = 20430\n",
      "--- number of probes: 268 , kept by blast: 263 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 820\n",
      "--- gene=820, id=28 removed by soft count = 410\n",
      "--- gene=820, id=29 removed by soft count = 383\n",
      "--- gene=820, id=34 removed by soft count = 8096\n",
      "--- gene=820, id=35 removed by soft count = 319\n",
      "--- gene=820, id=53 removed by soft count = 139\n",
      "--- gene=820, id=56 removed by soft count = 137\n",
      "--- gene=820, id=58 removed by soft count = 188\n",
      "--- gene=820, id=60 removed by soft count = 81\n",
      "--- number of probes: 255 , kept by blast: 247 , if remove dups: 247\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 821\n",
      "--- gene=821, id=107 removed by soft count = 86\n",
      "--- gene=821, id=117 removed by soft count = 1684\n",
      "--- gene=821, id=131 removed by soft count = 95\n",
      "--- gene=821, id=134 removed by soft count = 895\n",
      "--- gene=821, id=135 removed by soft count = 96\n",
      "--- gene=821, id=136 removed by soft count = 910\n",
      "--- gene=821, id=138 removed by soft count = 111\n",
      "--- gene=821, id=141 removed by soft count = 124\n",
      "--- number of probes: 142 , kept by blast: 134 , if remove dups: 134\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 822\n",
      "--- gene=822, id=17 removed by soft count = 433\n",
      "--- gene=822, id=211 removed by soft count = 112\n",
      "--- gene=822, id=217 removed by soft count = 83\n",
      "--- gene=822, id=218 removed by soft count = 120\n",
      "--- gene=822, id=219 removed by soft count = 71\n",
      "--- gene=822, id=220 removed by soft count = 52\n",
      "--- number of probes: 257 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 823\n",
      "--- gene=823, id=0 removed by soft count = 431\n",
      "--- gene=823, id=3 removed by soft count = 73\n",
      "--- gene=823, id=5 removed by soft count = 250\n",
      "--- gene=823, id=8 removed by soft count = 102\n",
      "--- gene=823, id=11 removed by soft count = 57\n",
      "--- gene=823, id=20 removed by soft count = 56\n",
      "--- gene=823, id=25 removed by soft count = 115\n",
      "--- gene=823, id=40 removed by soft count = 65\n",
      "--- gene=823, id=43 removed by soft count = 58\n",
      "--- gene=823, id=44 removed by soft count = 2180\n",
      "--- gene=823, id=46 removed by soft count = 57\n",
      "--- gene=823, id=47 removed by soft count = 62\n",
      "--- gene=823, id=50 removed by soft count = 74\n",
      "--- gene=823, id=52 removed by soft count = 150\n",
      "--- gene=823, id=53 removed by soft count = 56\n",
      "--- gene=823, id=54 removed by soft count = 488\n",
      "--- gene=823, id=69 removed by soft count = 63\n",
      "--- gene=823, id=70 removed by soft count = 85\n",
      "--- gene=823, id=84 removed by soft count = 59\n",
      "--- gene=823, id=92 removed by soft count = 18007\n",
      "--- gene=823, id=114 removed by soft count = 154\n",
      "--- gene=823, id=117 removed by soft count = 630\n",
      "--- number of probes: 129 , kept by blast: 107 , if remove dups: 107\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 107\n",
      "-- checking probes in region: 824\n",
      "--- gene=824, id=137 removed by soft count = 58\n",
      "--- gene=824, id=188 removed by soft count = 185\n",
      "--- gene=824, id=274 removed by soft count = 15799\n",
      "--- gene=824, id=275 removed by soft count = 8462\n",
      "--- number of probes: 288 , kept by blast: 284 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 825\n",
      "--- gene=825, id=147 removed by soft count = 356\n",
      "--- gene=825, id=173 removed by soft count = 103\n",
      "--- gene=825, id=174 removed by soft count = 366\n",
      "--- gene=825, id=175 removed by soft count = 68\n",
      "--- gene=825, id=184 removed by soft count = 417\n",
      "--- gene=825, id=187 removed by soft count = 134\n",
      "--- number of probes: 225 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 826\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=826, id=4 removed by soft count = 201\n",
      "--- gene=826, id=77 removed by soft count = 138\n",
      "--- gene=826, id=78 removed by soft count = 57\n",
      "--- gene=826, id=79 removed by soft count = 386\n",
      "--- gene=826, id=92 removed by soft count = 100\n",
      "--- gene=826, id=137 removed by soft count = 232\n",
      "--- gene=826, id=195 removed by soft count = 63\n",
      "--- gene=826, id=200 removed by soft count = 430\n",
      "--- number of probes: 263 , kept by blast: 255 , if remove dups: 255\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 827\n",
      "--- gene=827, id=205 removed by soft count = 742\n",
      "--- gene=827, id=225 removed by soft count = 66\n",
      "--- gene=827, id=246 removed by soft count = 832\n",
      "--- number of probes: 283 , kept by blast: 280 , if remove dups: 280\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 828\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 829\n",
      "--- gene=829, id=172 removed by soft count = 155\n",
      "--- gene=829, id=217 removed by soft count = 51\n",
      "--- gene=829, id=220 removed by soft count = 94\n",
      "--- gene=829, id=230 removed by soft count = 595\n",
      "--- gene=829, id=231 removed by soft count = 1827\n",
      "--- number of probes: 237 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 832\n",
      "--- gene=832, id=8 removed by soft count = 241\n",
      "--- number of probes: 168 , kept by blast: 167 , if remove dups: 167\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 833\n",
      "--- gene=833, id=122 removed by soft count = 130\n",
      "--- gene=833, id=124 removed by soft count = 271\n",
      "--- gene=833, id=125 removed by soft count = 490\n",
      "--- gene=833, id=126 removed by soft count = 139\n",
      "--- gene=833, id=127 removed by soft count = 100\n",
      "--- gene=833, id=128 removed by soft count = 94\n",
      "--- gene=833, id=129 removed by soft count = 209\n",
      "--- gene=833, id=141 removed by soft count = 202\n",
      "--- gene=833, id=144 removed by soft count = 110\n",
      "--- gene=833, id=145 removed by soft count = 76\n",
      "--- gene=833, id=177 removed by soft count = 70\n",
      "--- gene=833, id=180 removed by soft count = 53\n",
      "--- gene=833, id=189 removed by soft count = 82\n",
      "--- number of probes: 204 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 834\n",
      "--- gene=834, id=170 removed by soft count = 480\n",
      "--- gene=834, id=184 removed by soft count = 1109\n",
      "--- gene=834, id=223 removed by soft count = 54\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 835\n",
      "--- gene=835, id=8 removed by soft count = 961\n",
      "--- gene=835, id=17 removed by soft count = 79\n",
      "--- gene=835, id=20 removed by soft count = 583\n",
      "--- gene=835, id=170 removed by soft count = 485\n",
      "--- number of probes: 236 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 836\n",
      "--- gene=836, id=0 removed by soft count = 81\n",
      "--- gene=836, id=2 removed by soft count = 311\n",
      "--- gene=836, id=3 removed by soft count = 134\n",
      "--- gene=836, id=6 removed by soft count = 98\n",
      "--- gene=836, id=7 removed by soft count = 723\n",
      "--- gene=836, id=11 removed by soft count = 91\n",
      "--- gene=836, id=48 removed by soft count = 1016\n",
      "--- gene=836, id=79 removed by soft count = 88\n",
      "--- number of probes: 164 , kept by blast: 156 , if remove dups: 156\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 837\n",
      "--- gene=837, id=222 removed by soft count = 820\n",
      "--- gene=837, id=223 removed by soft count = 264\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 838\n",
      "--- gene=838, id=45 removed by soft count = 899\n",
      "--- gene=838, id=173 removed by soft count = 221\n",
      "--- gene=838, id=174 removed by soft count = 259\n",
      "--- gene=838, id=175 removed by soft count = 319\n",
      "--- number of probes: 226 , kept by blast: 222 , if remove dups: 222\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 839\n",
      "--- gene=839, id=13 removed by soft count = 6873\n",
      "--- gene=839, id=60 removed by soft count = 73\n",
      "--- gene=839, id=61 removed by soft count = 63\n",
      "--- gene=839, id=65 removed by soft count = 100\n",
      "--- gene=839, id=66 removed by soft count = 37322\n",
      "--- gene=839, id=89 removed by soft count = 393\n",
      "--- gene=839, id=90 removed by soft count = 507\n",
      "--- number of probes: 194 , kept by blast: 187 , if remove dups: 187\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 840\n",
      "--- gene=840, id=36 removed by soft count = 89\n",
      "--- gene=840, id=48 removed by soft count = 92\n",
      "--- gene=840, id=67 removed by soft count = 1103\n",
      "--- gene=840, id=68 removed by soft count = 143\n",
      "--- gene=840, id=69 removed by soft count = 175\n",
      "--- gene=840, id=70 removed by soft count = 277\n",
      "--- gene=840, id=71 removed by soft count = 124\n",
      "--- gene=840, id=81 removed by soft count = 333\n",
      "--- gene=840, id=200 removed by soft count = 142\n",
      "--- gene=840, id=256 removed by soft count = 595\n",
      "--- gene=840, id=257 removed by soft count = 461\n",
      "--- gene=840, id=258 removed by soft count = 224\n",
      "--- number of probes: 264 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 841\n",
      "--- gene=841, id=50 removed by soft count = 228\n",
      "--- gene=841, id=51 removed by soft count = 118\n",
      "--- gene=841, id=69 removed by soft count = 210\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 843\n",
      "--- gene=843, id=1 removed by soft count = 123\n",
      "--- gene=843, id=4 removed by soft count = 645\n",
      "--- gene=843, id=17 removed by soft count = 71\n",
      "--- number of probes: 148 , kept by blast: 145 , if remove dups: 145\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 844\n",
      "--- gene=844, id=36 removed by soft count = 120\n",
      "--- gene=844, id=37 removed by soft count = 57\n",
      "--- gene=844, id=39 removed by soft count = 60\n",
      "--- gene=844, id=43 removed by soft count = 58\n",
      "--- gene=844, id=49 removed by soft count = 79\n",
      "--- gene=844, id=51 removed by soft count = 421\n",
      "--- gene=844, id=115 removed by soft count = 511\n",
      "--- gene=844, id=181 removed by soft count = 57\n",
      "--- number of probes: 256 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 845\n",
      "--- gene=845, id=24 removed by soft count = 80\n",
      "--- gene=845, id=34 removed by soft count = 125\n",
      "--- gene=845, id=50 removed by soft count = 76\n",
      "--- gene=845, id=172 removed by soft count = 257\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 846\n",
      "--- gene=846, id=65 removed by soft count = 115\n",
      "--- gene=846, id=69 removed by soft count = 126\n",
      "--- gene=846, id=74 removed by soft count = 2160\n",
      "--- gene=846, id=230 removed by soft count = 91\n",
      "--- number of probes: 261 , kept by blast: 257 , if remove dups: 257\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 847\n",
      "--- gene=847, id=5 removed by soft count = 129\n",
      "--- gene=847, id=7 removed by soft count = 83\n",
      "--- gene=847, id=9 removed by soft count = 161\n",
      "--- gene=847, id=10 removed by soft count = 172\n",
      "--- gene=847, id=11 removed by soft count = 85\n",
      "--- gene=847, id=12 removed by soft count = 134\n",
      "--- gene=847, id=14 removed by soft count = 85\n",
      "--- gene=847, id=15 removed by soft count = 170\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=847, id=16 removed by soft count = 508\n",
      "--- gene=847, id=68 removed by soft count = 108\n",
      "--- gene=847, id=71 removed by soft count = 122\n",
      "--- gene=847, id=144 removed by soft count = 96\n",
      "--- gene=847, id=154 removed by soft count = 89\n",
      "--- gene=847, id=160 removed by soft count = 459\n",
      "--- gene=847, id=161 removed by soft count = 51\n",
      "--- gene=847, id=162 removed by soft count = 222\n",
      "--- gene=847, id=163 removed by soft count = 86\n",
      "--- gene=847, id=164 removed by soft count = 65\n",
      "--- gene=847, id=167 removed by soft count = 128\n",
      "--- number of probes: 174 , kept by blast: 155 , if remove dups: 155\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 848\n",
      "--- gene=848, id=139 removed by soft count = 56\n",
      "--- gene=848, id=195 removed by soft count = 289\n",
      "--- gene=848, id=198 removed by soft count = 1994\n",
      "--- gene=848, id=203 removed by soft count = 69\n",
      "--- gene=848, id=206 removed by soft count = 80\n",
      "--- number of probes: 294 , kept by blast: 289 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 849\n",
      "--- gene=849, id=92 removed by soft count = 56\n",
      "--- gene=849, id=93 removed by soft count = 843\n",
      "--- gene=849, id=94 removed by soft count = 142\n",
      "--- gene=849, id=95 removed by soft count = 268\n",
      "--- gene=849, id=96 removed by soft count = 90\n",
      "--- gene=849, id=97 removed by soft count = 534\n",
      "--- gene=849, id=98 removed by soft count = 2008\n",
      "--- gene=849, id=99 removed by soft count = 159\n",
      "--- gene=849, id=112 removed by soft count = 229\n",
      "--- gene=849, id=113 removed by soft count = 53\n",
      "--- gene=849, id=115 removed by soft count = 104\n",
      "--- gene=849, id=116 removed by soft count = 226\n",
      "--- gene=849, id=117 removed by soft count = 240\n",
      "--- gene=849, id=121 removed by soft count = 63\n",
      "--- number of probes: 123 , kept by blast: 109 , if remove dups: 109\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 109\n",
      "-- checking probes in region: 850\n",
      "--- gene=850, id=0 removed by soft count = 289\n",
      "--- gene=850, id=3 removed by soft count = 114\n",
      "--- gene=850, id=5 removed by soft count = 140\n",
      "--- gene=850, id=130 removed by soft count = 105\n",
      "--- gene=850, id=132 removed by soft count = 328\n",
      "--- gene=850, id=176 removed by soft count = 450\n",
      "--- gene=850, id=177 removed by soft count = 82\n",
      "--- gene=850, id=178 removed by soft count = 240\n",
      "--- gene=850, id=180 removed by soft count = 288\n",
      "--- number of probes: 181 , kept by blast: 172 , if remove dups: 172\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 851\n",
      "--- gene=851, id=65 removed by soft count = 149\n",
      "--- gene=851, id=67 removed by soft count = 1758\n",
      "--- gene=851, id=68 removed by soft count = 284\n",
      "--- gene=851, id=70 removed by soft count = 68\n",
      "--- gene=851, id=80 removed by soft count = 105\n",
      "--- gene=851, id=116 removed by soft count = 300\n",
      "--- gene=851, id=125 removed by soft count = 61\n",
      "--- gene=851, id=126 removed by soft count = 199\n",
      "--- number of probes: 241 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 852\n",
      "--- gene=852, id=19 removed by soft count = 138\n",
      "--- number of probes: 279 , kept by blast: 278 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 853\n",
      "--- gene=853, id=18 removed by soft count = 58\n",
      "--- gene=853, id=26 removed by soft count = 63\n",
      "--- gene=853, id=28 removed by soft count = 58\n",
      "--- gene=853, id=32 removed by soft count = 60\n",
      "--- gene=853, id=35 removed by soft count = 69\n",
      "--- gene=853, id=88 removed by soft count = 784\n",
      "--- gene=853, id=179 removed by soft count = 695\n",
      "--- gene=853, id=191 removed by soft count = 171\n",
      "--- gene=853, id=192 removed by soft count = 1162\n",
      "--- number of probes: 194 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 854\n",
      "--- gene=854, id=0 removed by soft count = 142\n",
      "--- gene=854, id=84 removed by soft count = 1184\n",
      "--- gene=854, id=100 removed by soft count = 414\n",
      "--- gene=854, id=125 removed by soft count = 103\n",
      "--- gene=854, id=126 removed by soft count = 1007\n",
      "--- gene=854, id=136 removed by soft count = 586\n",
      "--- gene=854, id=139 removed by soft count = 174\n",
      "--- gene=854, id=183 removed by soft count = 178\n",
      "--- gene=854, id=185 removed by soft count = 71\n",
      "--- gene=854, id=188 removed by soft count = 100\n",
      "--- number of probes: 215 , kept by blast: 205 , if remove dups: 205\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 855\n",
      "--- gene=855, id=75 removed by soft count = 51\n",
      "--- gene=855, id=77 removed by soft count = 158\n",
      "--- gene=855, id=81 removed by soft count = 1577\n",
      "--- gene=855, id=86 removed by soft count = 521\n",
      "--- gene=855, id=88 removed by soft count = 74\n",
      "--- gene=855, id=142 removed by soft count = 422\n",
      "--- gene=855, id=144 removed by soft count = 297\n",
      "--- gene=855, id=145 removed by soft count = 1732\n",
      "--- gene=855, id=147 removed by soft count = 108\n",
      "--- number of probes: 199 , kept by blast: 190 , if remove dups: 190\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 856\n",
      "--- gene=856, id=71 removed by soft count = 65\n",
      "--- gene=856, id=72 removed by soft count = 172\n",
      "--- gene=856, id=74 removed by soft count = 61\n",
      "--- gene=856, id=75 removed by soft count = 64\n",
      "--- gene=856, id=77 removed by soft count = 57\n",
      "--- gene=856, id=79 removed by soft count = 77\n",
      "--- gene=856, id=81 removed by soft count = 146\n",
      "--- gene=856, id=82 removed by soft count = 627\n",
      "--- gene=856, id=83 removed by soft count = 199\n",
      "--- gene=856, id=84 removed by soft count = 835\n",
      "--- gene=856, id=86 removed by soft count = 123\n",
      "--- gene=856, id=88 removed by soft count = 82\n",
      "--- gene=856, id=90 removed by soft count = 131\n",
      "--- gene=856, id=91 removed by soft count = 193\n",
      "--- gene=856, id=98 removed by soft count = 346\n",
      "--- gene=856, id=100 removed by soft count = 66\n",
      "--- gene=856, id=101 removed by soft count = 322\n",
      "--- gene=856, id=106 removed by soft count = 62\n",
      "--- gene=856, id=107 removed by soft count = 721\n",
      "--- gene=856, id=109 removed by soft count = 220\n",
      "--- gene=856, id=112 removed by soft count = 72\n",
      "--- gene=856, id=113 removed by soft count = 604\n",
      "--- gene=856, id=133 removed by soft count = 143\n",
      "--- number of probes: 163 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 857\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 858\n",
      "--- gene=858, id=1 removed by soft count = 271\n",
      "--- gene=858, id=2 removed by soft count = 245\n",
      "--- gene=858, id=3 removed by soft count = 167\n",
      "--- gene=858, id=12 removed by soft count = 149\n",
      "--- gene=858, id=13 removed by soft count = 146\n",
      "--- gene=858, id=56 removed by soft count = 582\n",
      "--- gene=858, id=249 removed by soft count = 825\n",
      "--- gene=858, id=250 removed by soft count = 447\n",
      "--- gene=858, id=251 removed by soft count = 146\n",
      "--- number of probes: 252 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 859\n",
      "--- gene=859, id=1 removed by soft count = 153\n",
      "--- gene=859, id=16 removed by soft count = 232\n",
      "--- gene=859, id=69 removed by soft count = 220\n",
      "--- gene=859, id=122 removed by soft count = 55\n",
      "--- gene=859, id=123 removed by soft count = 185\n",
      "--- gene=859, id=124 removed by soft count = 84\n",
      "--- gene=859, id=151 removed by soft count = 155\n",
      "--- gene=859, id=213 removed by soft count = 15840\n",
      "--- gene=859, id=225 removed by soft count = 209\n",
      "--- number of probes: 240 , kept by blast: 231 , if remove dups: 231\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 860\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=860, id=1 removed by soft count = 898\n",
      "--- gene=860, id=2 removed by soft count = 145\n",
      "--- gene=860, id=23 removed by soft count = 140\n",
      "--- gene=860, id=81 removed by soft count = 57\n",
      "--- gene=860, id=82 removed by soft count = 973\n",
      "--- gene=860, id=109 removed by soft count = 284\n",
      "--- gene=860, id=112 removed by soft count = 90\n",
      "--- gene=860, id=114 removed by soft count = 528\n",
      "--- gene=860, id=115 removed by soft count = 267\n",
      "--- gene=860, id=117 removed by soft count = 412\n",
      "--- gene=860, id=120 removed by soft count = 72\n",
      "--- gene=860, id=121 removed by soft count = 90\n",
      "--- gene=860, id=124 removed by soft count = 310\n",
      "--- gene=860, id=125 removed by soft count = 69\n",
      "--- number of probes: 144 , kept by blast: 130 , if remove dups: 130\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 861\n",
      "--- gene=861, id=172 removed by soft count = 1566\n",
      "--- gene=861, id=173 removed by soft count = 116\n",
      "--- gene=861, id=271 removed by soft count = 78\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 862\n",
      "--- gene=862, id=11 removed by soft count = 273\n",
      "--- gene=862, id=51 removed by soft count = 60\n",
      "--- gene=862, id=107 removed by soft count = 172\n",
      "--- gene=862, id=109 removed by soft count = 81\n",
      "--- gene=862, id=112 removed by soft count = 165\n",
      "--- number of probes: 233 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 863\n",
      "--- gene=863, id=14 removed by soft count = 51\n",
      "--- gene=863, id=125 removed by soft count = 923\n",
      "--- number of probes: 282 , kept by blast: 280 , if remove dups: 280\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 864\n",
      "--- gene=864, id=97 removed by soft count = 54\n",
      "--- gene=864, id=109 removed by soft count = 1147\n",
      "--- gene=864, id=110 removed by soft count = 273\n",
      "--- gene=864, id=112 removed by soft count = 86\n",
      "--- gene=864, id=159 removed by soft count = 227\n",
      "--- gene=864, id=160 removed by soft count = 2268\n",
      "--- number of probes: 250 , kept by blast: 244 , if remove dups: 244\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 865\n",
      "--- gene=865, id=0 removed by soft count = 57\n",
      "--- gene=865, id=2 removed by soft count = 199\n",
      "--- gene=865, id=119 removed by soft count = 58\n",
      "--- gene=865, id=120 removed by soft count = 88\n",
      "--- gene=865, id=123 removed by soft count = 129\n",
      "--- gene=865, id=125 removed by soft count = 70\n",
      "--- gene=865, id=126 removed by soft count = 128\n",
      "--- gene=865, id=144 removed by soft count = 107\n",
      "--- gene=865, id=145 removed by soft count = 103\n",
      "--- gene=865, id=146 removed by soft count = 105\n",
      "--- gene=865, id=197 removed by soft count = 344\n",
      "--- gene=865, id=199 removed by soft count = 81\n",
      "--- gene=865, id=200 removed by soft count = 90\n",
      "--- gene=865, id=202 removed by soft count = 126\n",
      "--- gene=865, id=204 removed by soft count = 72\n",
      "--- gene=865, id=209 removed by soft count = 5318\n",
      "--- gene=865, id=210 removed by soft count = 888\n",
      "--- gene=865, id=211 removed by soft count = 322\n",
      "--- gene=865, id=212 removed by soft count = 121\n",
      "--- gene=865, id=214 removed by soft count = 102\n",
      "--- number of probes: 223 , kept by blast: 203 , if remove dups: 202\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 866\n",
      "--- gene=866, id=156 removed by soft count = 55\n",
      "--- gene=866, id=190 removed by soft count = 56\n",
      "--- gene=866, id=192 removed by soft count = 98\n",
      "--- gene=866, id=195 removed by soft count = 65\n",
      "--- gene=866, id=198 removed by soft count = 72\n",
      "--- gene=866, id=199 removed by soft count = 151\n",
      "--- gene=866, id=201 removed by soft count = 528\n",
      "--- gene=866, id=202 removed by soft count = 83\n",
      "--- number of probes: 216 , kept by blast: 208 , if remove dups: 208\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 867\n",
      "--- gene=867, id=83 removed by soft count = 323\n",
      "--- gene=867, id=90 removed by soft count = 189\n",
      "--- gene=867, id=110 removed by soft count = 53\n",
      "--- gene=867, id=152 removed by soft count = 87\n",
      "--- gene=867, id=162 removed by soft count = 311\n",
      "--- gene=867, id=163 removed by soft count = 54\n",
      "--- number of probes: 164 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 868\n",
      "--- gene=868, id=65 removed by soft count = 325\n",
      "--- gene=868, id=66 removed by soft count = 749\n",
      "--- gene=868, id=100 removed by soft count = 17262\n",
      "--- gene=868, id=101 removed by soft count = 56\n",
      "--- gene=868, id=132 removed by soft count = 651\n",
      "--- gene=868, id=136 removed by soft count = 134\n",
      "--- gene=868, id=171 removed by soft count = 189\n",
      "--- gene=868, id=275 removed by soft count = 178\n",
      "--- gene=868, id=276 removed by soft count = 77\n",
      "--- gene=868, id=278 removed by soft count = 732\n",
      "--- number of probes: 289 , kept by blast: 279 , if remove dups: 279\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 869\n",
      "--- gene=869, id=1 removed by soft count = 115\n",
      "--- gene=869, id=2 removed by soft count = 67\n",
      "--- gene=869, id=37 removed by soft count = 663\n",
      "--- gene=869, id=40 removed by soft count = 818\n",
      "--- gene=869, id=41 removed by soft count = 647\n",
      "--- gene=869, id=45 removed by soft count = 268\n",
      "--- gene=869, id=54 removed by soft count = 348\n",
      "--- gene=869, id=73 removed by soft count = 56\n",
      "--- gene=869, id=75 removed by soft count = 541\n",
      "--- gene=869, id=77 removed by soft count = 1403\n",
      "--- gene=869, id=102 removed by soft count = 112\n",
      "--- gene=869, id=104 removed by soft count = 64\n",
      "--- gene=869, id=124 removed by soft count = 120\n",
      "--- gene=869, id=125 removed by soft count = 53\n",
      "--- gene=869, id=126 removed by soft count = 164\n",
      "--- gene=869, id=159 removed by soft count = 356\n",
      "--- gene=869, id=250 removed by soft count = 55\n",
      "--- gene=869, id=257 removed by soft count = 75\n",
      "--- gene=869, id=290 removed by soft count = 95\n",
      "--- number of probes: 300 , kept by blast: 281 , if remove dups: 281\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 870\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 871\n",
      "--- gene=871, id=0 removed by soft count = 89\n",
      "--- gene=871, id=268 removed by soft count = 124\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 872\n",
      "--- number of probes: 299 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 873\n",
      "--- gene=873, id=74 removed by soft count = 107\n",
      "--- gene=873, id=76 removed by soft count = 573\n",
      "--- gene=873, id=82 removed by soft count = 447\n",
      "--- gene=873, id=85 removed by soft count = 155\n",
      "--- gene=873, id=86 removed by soft count = 250\n",
      "--- gene=873, id=144 removed by soft count = 51\n",
      "--- gene=873, id=204 removed by soft count = 66\n",
      "--- gene=873, id=217 removed by soft count = 803\n",
      "--- number of probes: 261 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 874\n",
      "--- gene=874, id=147 removed by soft count = 107\n",
      "--- gene=874, id=241 removed by soft count = 393\n",
      "--- gene=874, id=260 removed by soft count = 110\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 875\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=875, id=92 removed by soft count = 2099\n",
      "--- gene=875, id=93 removed by soft count = 588\n",
      "--- gene=875, id=98 removed by soft count = 103\n",
      "--- gene=875, id=127 removed by soft count = 61\n",
      "--- gene=875, id=156 removed by soft count = 121\n",
      "--- number of probes: 212 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 876\n",
      "--- gene=876, id=58 removed by soft count = 703\n",
      "--- gene=876, id=59 removed by soft count = 318\n",
      "--- gene=876, id=60 removed by soft count = 269\n",
      "--- gene=876, id=70 removed by soft count = 165\n",
      "--- gene=876, id=71 removed by soft count = 269\n",
      "--- gene=876, id=96 removed by soft count = 798\n",
      "--- gene=876, id=97 removed by soft count = 196\n",
      "--- gene=876, id=98 removed by soft count = 135\n",
      "--- gene=876, id=139 removed by soft count = 1698\n",
      "--- gene=876, id=225 removed by soft count = 221\n",
      "--- gene=876, id=247 removed by soft count = 71\n",
      "--- number of probes: 262 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 877\n",
      "--- gene=877, id=74 removed by soft count = 54\n",
      "--- gene=877, id=75 removed by soft count = 231\n",
      "--- gene=877, id=76 removed by soft count = 79\n",
      "--- gene=877, id=77 removed by soft count = 559\n",
      "--- gene=877, id=119 removed by soft count = 122\n",
      "--- gene=877, id=196 removed by soft count = 174\n",
      "--- gene=877, id=198 removed by soft count = 209\n",
      "--- gene=877, id=199 removed by soft count = 1266\n",
      "--- gene=877, id=203 removed by soft count = 1799\n",
      "--- number of probes: 259 , kept by blast: 250 , if remove dups: 249\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 878\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 888\n",
      "--- gene=888, id=9 removed by soft count = 72\n",
      "--- gene=888, id=11 removed by soft count = 65\n",
      "--- gene=888, id=114 removed by soft count = 3389\n",
      "--- gene=888, id=115 removed by soft count = 783\n",
      "--- gene=888, id=116 removed by soft count = 589\n",
      "--- gene=888, id=119 removed by soft count = 8226\n",
      "--- gene=888, id=120 removed by soft count = 13847\n",
      "--- gene=888, id=121 removed by soft count = 9290\n",
      "--- gene=888, id=122 removed by soft count = 5275\n",
      "--- gene=888, id=124 removed by soft count = 4195\n",
      "--- gene=888, id=125 removed by soft count = 1517\n",
      "--- gene=888, id=126 removed by soft count = 1614\n",
      "--- gene=888, id=127 removed by soft count = 1843\n",
      "--- gene=888, id=128 removed by soft count = 263\n",
      "--- gene=888, id=129 removed by soft count = 472\n",
      "--- gene=888, id=130 removed by soft count = 56\n",
      "--- gene=888, id=169 removed by soft count = 175\n",
      "--- gene=888, id=206 removed by soft count = 143\n",
      "--- gene=888, id=207 removed by soft count = 361\n",
      "--- gene=888, id=208 removed by soft count = 64\n",
      "--- number of probes: 215 , kept by blast: 195 , if remove dups: 195\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 889\n",
      "--- gene=889, id=252 removed by soft count = 94\n",
      "--- number of probes: 279 , kept by blast: 278 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 890\n",
      "--- gene=890, id=42 removed by soft count = 287\n",
      "--- gene=890, id=143 removed by soft count = 66\n",
      "--- gene=890, id=148 removed by soft count = 126\n",
      "--- gene=890, id=149 removed by soft count = 260\n",
      "--- gene=890, id=150 removed by soft count = 1555\n",
      "--- gene=890, id=163 removed by soft count = 109\n",
      "--- gene=890, id=164 removed by soft count = 192\n",
      "--- gene=890, id=165 removed by soft count = 63\n",
      "--- gene=890, id=166 removed by soft count = 166\n",
      "--- gene=890, id=168 removed by soft count = 195\n",
      "--- gene=890, id=169 removed by soft count = 142\n",
      "--- gene=890, id=170 removed by soft count = 161\n",
      "--- gene=890, id=202 removed by soft count = 64\n",
      "--- number of probes: 204 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 891\n",
      "--- gene=891, id=10 removed by soft count = 98\n",
      "--- gene=891, id=47 removed by soft count = 79\n",
      "--- gene=891, id=78 removed by soft count = 81\n",
      "--- gene=891, id=86 removed by soft count = 163\n",
      "--- gene=891, id=90 removed by soft count = 73\n",
      "--- gene=891, id=91 removed by soft count = 74\n",
      "--- gene=891, id=92 removed by soft count = 151\n",
      "--- gene=891, id=94 removed by soft count = 94\n",
      "--- gene=891, id=95 removed by soft count = 83\n",
      "--- gene=891, id=96 removed by soft count = 174\n",
      "--- gene=891, id=143 removed by soft count = 126\n",
      "--- gene=891, id=144 removed by soft count = 212\n",
      "--- number of probes: 169 , kept by blast: 157 , if remove dups: 155\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 892\n",
      "--- gene=892, id=108 removed by soft count = 234\n",
      "--- gene=892, id=151 removed by soft count = 128\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 893\n",
      "--- gene=893, id=8 removed by soft count = 65\n",
      "--- gene=893, id=69 removed by soft count = 575\n",
      "--- gene=893, id=90 removed by soft count = 261\n",
      "--- gene=893, id=91 removed by soft count = 2341\n",
      "--- gene=893, id=92 removed by soft count = 274\n",
      "--- number of probes: 252 , kept by blast: 247 , if remove dups: 247\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 894\n",
      "--- gene=894, id=31 removed by soft count = 55\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 895\n",
      "--- gene=895, id=98 removed by soft count = 112\n",
      "--- gene=895, id=105 removed by soft count = 187\n",
      "--- gene=895, id=106 removed by soft count = 53\n",
      "--- gene=895, id=111 removed by soft count = 86\n",
      "--- gene=895, id=126 removed by soft count = 94\n",
      "--- gene=895, id=169 removed by soft count = 177\n",
      "--- gene=895, id=232 removed by soft count = 84\n",
      "--- gene=895, id=272 removed by soft count = 100\n",
      "--- gene=895, id=275 removed by soft count = 238\n",
      "--- number of probes: 294 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 896\n",
      "--- gene=896, id=0 removed by soft count = 5592\n",
      "--- gene=896, id=1 removed by soft count = 1088\n",
      "--- gene=896, id=2 removed by soft count = 779\n",
      "--- gene=896, id=5 removed by soft count = 77\n",
      "--- gene=896, id=71 removed by soft count = 131\n",
      "--- gene=896, id=137 removed by soft count = 118\n",
      "--- number of probes: 158 , kept by blast: 152 , if remove dups: 152\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 897\n",
      "--- gene=897, id=10 removed by soft count = 61\n",
      "--- number of probes: 286 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 898\n",
      "--- gene=898, id=2 removed by soft count = 6630\n",
      "--- gene=898, id=66 removed by soft count = 86\n",
      "--- gene=898, id=67 removed by soft count = 70\n",
      "--- gene=898, id=68 removed by soft count = 72\n",
      "--- gene=898, id=72 removed by soft count = 94\n",
      "--- gene=898, id=79 removed by soft count = 65\n",
      "--- gene=898, id=101 removed by soft count = 56\n",
      "--- gene=898, id=108 removed by soft count = 197\n",
      "--- gene=898, id=109 removed by soft count = 219\n",
      "--- number of probes: 182 , kept by blast: 173 , if remove dups: 173\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 899\n",
      "--- gene=899, id=24 removed by soft count = 200\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 211 , kept by blast: 210 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 900\n",
      "--- gene=900, id=3 removed by soft count = 162\n",
      "--- gene=900, id=32 removed by soft count = 102\n",
      "--- gene=900, id=157 removed by soft count = 244\n",
      "--- gene=900, id=164 removed by soft count = 233\n",
      "--- gene=900, id=224 removed by soft count = 93\n",
      "--- gene=900, id=234 removed by soft count = 83\n",
      "--- gene=900, id=235 removed by soft count = 71\n",
      "--- gene=900, id=240 removed by soft count = 67\n",
      "--- number of probes: 251 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 901\n",
      "--- gene=901, id=4 removed by soft count = 95\n",
      "--- gene=901, id=5 removed by soft count = 71\n",
      "--- gene=901, id=6 removed by soft count = 237\n",
      "--- gene=901, id=8 removed by soft count = 600\n",
      "--- gene=901, id=19 removed by soft count = 491\n",
      "--- gene=901, id=64 removed by soft count = 446\n",
      "--- gene=901, id=83 removed by soft count = 229\n",
      "--- gene=901, id=127 removed by soft count = 128\n",
      "--- number of probes: 230 , kept by blast: 222 , if remove dups: 222\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 902\n",
      "--- gene=902, id=9 removed by soft count = 77\n",
      "--- gene=902, id=82 removed by soft count = 396\n",
      "--- gene=902, id=116 removed by soft count = 217\n",
      "--- gene=902, id=120 removed by soft count = 350\n",
      "--- gene=902, id=197 removed by soft count = 112\n",
      "--- gene=902, id=217 removed by soft count = 107\n",
      "--- gene=902, id=235 removed by soft count = 62\n",
      "--- gene=902, id=259 removed by soft count = 268\n",
      "--- gene=902, id=260 removed by soft count = 161\n",
      "--- number of probes: 271 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 903\n",
      "--- gene=903, id=32 removed by soft count = 56\n",
      "--- gene=903, id=52 removed by soft count = 75\n",
      "--- gene=903, id=63 removed by soft count = 74\n",
      "--- gene=903, id=74 removed by soft count = 1183\n",
      "--- gene=903, id=75 removed by soft count = 679\n",
      "--- gene=903, id=139 removed by soft count = 61\n",
      "--- number of probes: 144 , kept by blast: 138 , if remove dups: 138\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 904\n",
      "--- gene=904, id=65 removed by soft count = 153\n",
      "--- gene=904, id=80 removed by soft count = 66\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 905\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 906\n",
      "--- gene=906, id=20 removed by soft count = 51\n",
      "--- gene=906, id=23 removed by soft count = 171\n",
      "--- gene=906, id=67 removed by soft count = 98\n",
      "--- gene=906, id=68 removed by soft count = 298\n",
      "--- gene=906, id=86 removed by soft count = 197\n",
      "--- gene=906, id=88 removed by soft count = 304\n",
      "--- gene=906, id=121 removed by soft count = 54\n",
      "--- gene=906, id=124 removed by soft count = 236\n",
      "--- gene=906, id=129 removed by soft count = 86\n",
      "--- gene=906, id=144 removed by soft count = 156\n",
      "--- gene=906, id=145 removed by soft count = 840\n",
      "--- gene=906, id=158 removed by soft count = 60\n",
      "--- gene=906, id=164 removed by soft count = 6146\n",
      "--- gene=906, id=165 removed by soft count = 4590\n",
      "--- gene=906, id=166 removed by soft count = 603\n",
      "--- gene=906, id=183 removed by soft count = 71\n",
      "--- number of probes: 194 , kept by blast: 178 , if remove dups: 178\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 907\n",
      "--- gene=907, id=31 removed by soft count = 56\n",
      "--- gene=907, id=33 removed by soft count = 257\n",
      "--- gene=907, id=99 removed by soft count = 401\n",
      "--- number of probes: 143 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 908\n",
      "--- gene=908, id=56 removed by soft count = 176\n",
      "--- gene=908, id=113 removed by soft count = 60\n",
      "--- gene=908, id=118 removed by soft count = 70\n",
      "--- gene=908, id=124 removed by soft count = 8508\n",
      "--- gene=908, id=169 removed by soft count = 1171\n",
      "--- gene=908, id=178 removed by soft count = 2476\n",
      "--- number of probes: 216 , kept by blast: 210 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 909\n",
      "--- gene=909, id=69 removed by soft count = 325\n",
      "--- gene=909, id=78 removed by soft count = 161\n",
      "--- gene=909, id=80 removed by soft count = 280\n",
      "--- gene=909, id=83 removed by soft count = 62\n",
      "--- gene=909, id=85 removed by soft count = 78\n",
      "--- gene=909, id=86 removed by soft count = 176\n",
      "--- gene=909, id=90 removed by soft count = 53\n",
      "--- gene=909, id=92 removed by soft count = 131\n",
      "--- gene=909, id=93 removed by soft count = 95\n",
      "--- gene=909, id=110 removed by soft count = 73\n",
      "--- gene=909, id=112 removed by soft count = 226\n",
      "--- gene=909, id=113 removed by soft count = 119\n",
      "--- gene=909, id=120 removed by soft count = 153\n",
      "--- gene=909, id=121 removed by soft count = 199\n",
      "--- gene=909, id=122 removed by soft count = 468\n",
      "--- gene=909, id=123 removed by soft count = 525\n",
      "--- gene=909, id=129 removed by soft count = 71\n",
      "--- gene=909, id=130 removed by soft count = 61\n",
      "--- number of probes: 181 , kept by blast: 163 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 910\n",
      "--- gene=910, id=49 removed by soft count = 189\n",
      "--- gene=910, id=52 removed by soft count = 806\n",
      "--- gene=910, id=53 removed by soft count = 723\n",
      "--- gene=910, id=57 removed by soft count = 73\n",
      "--- gene=910, id=60 removed by soft count = 74\n",
      "--- gene=910, id=62 removed by soft count = 89\n",
      "--- gene=910, id=63 removed by soft count = 113\n",
      "--- gene=910, id=65 removed by soft count = 76\n",
      "--- gene=910, id=70 removed by soft count = 82\n",
      "--- gene=910, id=79 removed by soft count = 79\n",
      "--- gene=910, id=93 removed by soft count = 113\n",
      "--- gene=910, id=94 removed by soft count = 79\n",
      "--- gene=910, id=98 removed by soft count = 263\n",
      "--- gene=910, id=100 removed by soft count = 54\n",
      "--- gene=910, id=104 removed by soft count = 113\n",
      "--- gene=910, id=105 removed by soft count = 64\n",
      "--- gene=910, id=106 removed by soft count = 246\n",
      "--- gene=910, id=108 removed by soft count = 87\n",
      "--- gene=910, id=109 removed by soft count = 114\n",
      "--- gene=910, id=110 removed by soft count = 272\n",
      "--- gene=910, id=111 removed by soft count = 326\n",
      "--- gene=910, id=112 removed by soft count = 482\n",
      "--- gene=910, id=113 removed by soft count = 133\n",
      "--- gene=910, id=114 removed by soft count = 586\n",
      "--- gene=910, id=115 removed by soft count = 213\n",
      "--- gene=910, id=116 removed by soft count = 612\n",
      "--- gene=910, id=117 removed by soft count = 294\n",
      "--- gene=910, id=118 removed by soft count = 116\n",
      "--- gene=910, id=119 removed by soft count = 265\n",
      "--- gene=910, id=120 removed by soft count = 1217\n",
      "--- gene=910, id=121 removed by soft count = 345\n",
      "--- gene=910, id=122 removed by soft count = 174\n",
      "--- gene=910, id=125 removed by soft count = 70\n",
      "--- gene=910, id=131 removed by soft count = 195\n",
      "--- gene=910, id=134 removed by soft count = 1473\n",
      "--- number of probes: 155 , kept by blast: 120 , if remove dups: 120\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 911\n",
      "--- gene=911, id=39 removed by soft count = 212\n",
      "--- gene=911, id=56 removed by soft count = 70\n",
      "--- number of probes: 187 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 912\n",
      "--- number of probes: 230 , kept by blast: 230 , if remove dups: 230\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 913\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=913, id=26 removed by soft count = 128\n",
      "--- gene=913, id=29 removed by soft count = 1741\n",
      "--- gene=913, id=72 removed by soft count = 151\n",
      "--- gene=913, id=87 removed by soft count = 1662\n",
      "--- gene=913, id=89 removed by soft count = 367\n",
      "--- gene=913, id=90 removed by soft count = 108\n",
      "--- gene=913, id=114 removed by soft count = 113\n",
      "--- number of probes: 169 , kept by blast: 162 , if remove dups: 161\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 914\n",
      "--- gene=914, id=0 removed by soft count = 1597\n",
      "--- gene=914, id=1 removed by soft count = 52\n",
      "--- gene=914, id=2 removed by soft count = 180\n",
      "--- gene=914, id=3 removed by soft count = 989\n",
      "--- gene=914, id=4 removed by soft count = 323\n",
      "--- gene=914, id=7 removed by soft count = 83\n",
      "--- gene=914, id=135 removed by soft count = 85\n",
      "--- number of probes: 176 , kept by blast: 169 , if remove dups: 169\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 915\n",
      "--- gene=915, id=136 removed by soft count = 147\n",
      "--- gene=915, id=158 removed by soft count = 74\n",
      "--- gene=915, id=160 removed by soft count = 65\n",
      "--- number of probes: 166 , kept by blast: 163 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 916\n",
      "--- gene=916, id=104 removed by soft count = 131\n",
      "--- gene=916, id=152 removed by soft count = 56\n",
      "--- gene=916, id=154 removed by soft count = 89\n",
      "--- gene=916, id=177 removed by soft count = 405\n",
      "--- gene=916, id=194 removed by soft count = 74\n",
      "--- gene=916, id=236 removed by soft count = 102\n",
      "--- number of probes: 250 , kept by blast: 244 , if remove dups: 244\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 917\n",
      "--- gene=917, id=57 removed by soft count = 283\n",
      "--- gene=917, id=62 removed by hard count = 3\n",
      "--- gene=917, id=135 removed by soft count = 119\n",
      "--- gene=917, id=136 removed by soft count = 212\n",
      "--- gene=917, id=163 removed by soft count = 1057\n",
      "--- gene=917, id=222 removed by soft count = 79\n",
      "--- gene=917, id=233 removed by soft count = 128\n",
      "--- gene=917, id=234 removed by soft count = 218\n",
      "--- gene=917, id=235 removed by soft count = 78\n",
      "--- number of probes: 250 , kept by blast: 241 , if remove dups: 240\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 918\n",
      "--- gene=918, id=70 removed by soft count = 217\n",
      "--- gene=918, id=71 removed by soft count = 1154\n",
      "--- gene=918, id=233 removed by soft count = 337\n",
      "--- gene=918, id=234 removed by soft count = 537\n",
      "--- gene=918, id=254 removed by soft count = 248\n",
      "--- gene=918, id=255 removed by soft count = 725\n",
      "--- gene=918, id=256 removed by soft count = 181\n",
      "--- gene=918, id=258 removed by soft count = 157\n",
      "--- number of probes: 259 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 919\n",
      "--- gene=919, id=31 removed by soft count = 96\n",
      "--- gene=919, id=34 removed by soft count = 54\n",
      "--- gene=919, id=37 removed by soft count = 90\n",
      "--- gene=919, id=39 removed by soft count = 1457\n",
      "--- gene=919, id=45 removed by soft count = 141\n",
      "--- gene=919, id=46 removed by soft count = 1581\n",
      "--- gene=919, id=68 removed by soft count = 272\n",
      "--- gene=919, id=69 removed by soft count = 105\n",
      "--- gene=919, id=93 removed by soft count = 79\n",
      "--- gene=919, id=94 removed by soft count = 1485\n",
      "--- gene=919, id=109 removed by soft count = 327\n",
      "--- gene=919, id=119 removed by soft count = 728\n",
      "--- gene=919, id=120 removed by soft count = 3860\n",
      "--- gene=919, id=122 removed by soft count = 596\n",
      "--- gene=919, id=123 removed by soft count = 81\n",
      "--- gene=919, id=124 removed by soft count = 189\n",
      "--- gene=919, id=125 removed by soft count = 311\n",
      "--- gene=919, id=126 removed by soft count = 103\n",
      "--- gene=919, id=127 removed by soft count = 69\n",
      "--- gene=919, id=128 removed by soft count = 69\n",
      "--- gene=919, id=131 removed by soft count = 476\n",
      "--- gene=919, id=133 removed by soft count = 75\n",
      "--- gene=919, id=135 removed by soft count = 109\n",
      "--- gene=919, id=144 removed by soft count = 67\n",
      "--- gene=919, id=146 removed by soft count = 115\n",
      "--- gene=919, id=147 removed by soft count = 179\n",
      "--- gene=919, id=149 removed by soft count = 565\n",
      "--- gene=919, id=150 removed by soft count = 149\n",
      "--- gene=919, id=151 removed by soft count = 79\n",
      "--- gene=919, id=152 removed by soft count = 58\n",
      "--- gene=919, id=153 removed by soft count = 500\n",
      "--- number of probes: 157 , kept by blast: 126 , if remove dups: 126\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 920\n",
      "--- gene=920, id=93 removed by soft count = 2707\n",
      "--- gene=920, id=116 removed by soft count = 118\n",
      "--- number of probes: 187 , kept by blast: 185 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 921\n",
      "--- gene=921, id=34 removed by soft count = 165\n",
      "--- gene=921, id=146 removed by soft count = 62\n",
      "--- gene=921, id=165 removed by soft count = 121\n",
      "--- number of probes: 173 , kept by blast: 170 , if remove dups: 170\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 922\n",
      "--- gene=922, id=45 removed by soft count = 134\n",
      "--- gene=922, id=46 removed by soft count = 228\n",
      "--- gene=922, id=49 removed by soft count = 64\n",
      "--- gene=922, id=50 removed by soft count = 5440\n",
      "--- gene=922, id=51 removed by soft count = 1565\n",
      "--- gene=922, id=52 removed by soft count = 9894\n",
      "--- gene=922, id=53 removed by soft count = 14157\n",
      "--- gene=922, id=54 removed by soft count = 163\n",
      "--- gene=922, id=55 removed by soft count = 321\n",
      "--- gene=922, id=57 removed by soft count = 314\n",
      "--- gene=922, id=70 removed by soft count = 221\n",
      "--- gene=922, id=72 removed by soft count = 65\n",
      "--- gene=922, id=74 removed by soft count = 165\n",
      "--- gene=922, id=139 removed by soft count = 193\n",
      "--- number of probes: 174 , kept by blast: 160 , if remove dups: 160\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 923\n",
      "--- gene=923, id=32 removed by soft count = 866\n",
      "--- gene=923, id=33 removed by soft count = 896\n",
      "--- gene=923, id=34 removed by soft count = 1591\n",
      "--- gene=923, id=35 removed by soft count = 331\n",
      "--- gene=923, id=36 removed by soft count = 189\n",
      "--- gene=923, id=80 removed by soft count = 452\n",
      "--- gene=923, id=81 removed by soft count = 1499\n",
      "--- gene=923, id=82 removed by soft count = 2697\n",
      "--- gene=923, id=131 removed by soft count = 100\n",
      "--- number of probes: 141 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 924\n",
      "--- gene=924, id=99 removed by soft count = 123\n",
      "--- gene=924, id=100 removed by soft count = 140\n",
      "--- gene=924, id=115 removed by soft count = 110\n",
      "--- gene=924, id=179 removed by soft count = 81\n",
      "--- number of probes: 236 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 925\n",
      "--- gene=925, id=10 removed by soft count = 164\n",
      "--- gene=925, id=111 removed by soft count = 348\n",
      "--- gene=925, id=113 removed by soft count = 74\n",
      "--- gene=925, id=146 removed by soft count = 309\n",
      "--- gene=925, id=159 removed by soft count = 1266\n",
      "--- gene=925, id=161 removed by soft count = 252\n",
      "--- gene=925, id=162 removed by soft count = 473\n",
      "--- number of probes: 214 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 926\n",
      "--- gene=926, id=62 removed by soft count = 128\n",
      "--- gene=926, id=116 removed by soft count = 490\n",
      "--- gene=926, id=129 removed by soft count = 80\n",
      "--- gene=926, id=133 removed by soft count = 62\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 293 , kept by blast: 289 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 927\n",
      "--- gene=927, id=58 removed by soft count = 53\n",
      "--- gene=927, id=75 removed by soft count = 255\n",
      "--- gene=927, id=76 removed by soft count = 1126\n",
      "--- gene=927, id=108 removed by soft count = 122\n",
      "--- gene=927, id=132 removed by soft count = 846\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 928\n",
      "--- gene=928, id=234 removed by soft count = 134\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 929\n",
      "--- gene=929, id=6 removed by soft count = 60\n",
      "--- gene=929, id=12 removed by soft count = 143\n",
      "--- gene=929, id=60 removed by soft count = 179\n",
      "--- gene=929, id=62 removed by soft count = 166\n",
      "--- gene=929, id=143 removed by soft count = 418\n",
      "--- gene=929, id=184 removed by soft count = 1232\n",
      "--- number of probes: 261 , kept by blast: 255 , if remove dups: 255\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 930\n",
      "--- gene=930, id=13 removed by soft count = 1348\n",
      "--- gene=930, id=14 removed by soft count = 1079\n",
      "--- gene=930, id=15 removed by soft count = 287\n",
      "--- gene=930, id=16 removed by soft count = 83\n",
      "--- gene=930, id=46 removed by soft count = 51\n",
      "--- gene=930, id=50 removed by soft count = 186\n",
      "--- gene=930, id=51 removed by soft count = 501\n",
      "--- gene=930, id=53 removed by soft count = 257\n",
      "--- gene=930, id=88 removed by soft count = 279\n",
      "--- gene=930, id=91 removed by soft count = 53\n",
      "--- number of probes: 266 , kept by blast: 256 , if remove dups: 256\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 931\n",
      "--- gene=931, id=33 removed by soft count = 101\n",
      "--- gene=931, id=196 removed by soft count = 184\n",
      "--- gene=931, id=198 removed by soft count = 195\n",
      "--- number of probes: 249 , kept by blast: 246 , if remove dups: 246\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 932\n",
      "--- gene=932, id=46 removed by soft count = 86\n",
      "--- gene=932, id=286 removed by soft count = 75\n",
      "--- number of probes: 297 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 933\n",
      "--- gene=933, id=214 removed by soft count = 150\n",
      "--- gene=933, id=215 removed by soft count = 133\n",
      "--- number of probes: 218 , kept by blast: 216 , if remove dups: 216\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 934\n",
      "--- gene=934, id=0 removed by soft count = 51\n",
      "--- gene=934, id=4 removed by soft count = 139\n",
      "--- gene=934, id=8 removed by soft count = 62\n",
      "--- gene=934, id=63 removed by soft count = 74\n",
      "--- gene=934, id=67 removed by soft count = 104\n",
      "--- gene=934, id=68 removed by soft count = 69\n",
      "--- gene=934, id=69 removed by soft count = 167\n",
      "--- gene=934, id=71 removed by soft count = 191\n",
      "--- gene=934, id=100 removed by soft count = 266\n",
      "--- gene=934, id=102 removed by soft count = 191\n",
      "--- gene=934, id=141 removed by soft count = 104\n",
      "--- gene=934, id=148 removed by soft count = 201\n",
      "--- gene=934, id=149 removed by soft count = 87\n",
      "--- gene=934, id=150 removed by soft count = 103\n",
      "--- gene=934, id=151 removed by soft count = 80\n",
      "--- gene=934, id=152 removed by soft count = 110\n",
      "--- gene=934, id=154 removed by soft count = 96\n",
      "--- gene=934, id=158 removed by soft count = 151\n",
      "--- gene=934, id=160 removed by soft count = 83\n",
      "--- gene=934, id=161 removed by soft count = 68\n",
      "--- number of probes: 209 , kept by blast: 189 , if remove dups: 189\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 935\n",
      "--- gene=935, id=41 removed by hard count = 6\n",
      "--- gene=935, id=43 removed by soft count = 201\n",
      "--- gene=935, id=44 removed by soft count = 79\n",
      "--- gene=935, id=70 removed by soft count = 202\n",
      "--- gene=935, id=71 removed by soft count = 212\n",
      "--- gene=935, id=123 removed by soft count = 70\n",
      "--- gene=935, id=153 removed by soft count = 98\n",
      "--- gene=935, id=158 removed by soft count = 118\n",
      "--- gene=935, id=242 removed by hard count = 3\n",
      "--- number of probes: 269 , kept by blast: 260 , if remove dups: 259\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 949\n",
      "--- gene=949, id=28 removed by soft count = 364\n",
      "--- gene=949, id=63 removed by soft count = 58\n",
      "--- gene=949, id=85 removed by soft count = 55\n",
      "--- gene=949, id=95 removed by soft count = 847\n",
      "--- gene=949, id=102 removed by soft count = 109\n",
      "--- gene=949, id=106 removed by soft count = 88\n",
      "--- gene=949, id=138 removed by soft count = 89\n",
      "--- number of probes: 160 , kept by blast: 153 , if remove dups: 153\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 950\n",
      "--- gene=950, id=55 removed by soft count = 787\n",
      "--- gene=950, id=96 removed by soft count = 98\n",
      "--- gene=950, id=165 removed by soft count = 398\n",
      "--- gene=950, id=166 removed by soft count = 353\n",
      "--- gene=950, id=167 removed by soft count = 1332\n",
      "--- gene=950, id=168 removed by soft count = 2273\n",
      "--- gene=950, id=174 removed by soft count = 157\n",
      "--- gene=950, id=177 removed by soft count = 285\n",
      "--- number of probes: 178 , kept by blast: 170 , if remove dups: 170\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 951\n",
      "--- gene=951, id=0 removed by soft count = 68\n",
      "--- gene=951, id=79 removed by soft count = 79\n",
      "--- gene=951, id=80 removed by soft count = 153\n",
      "--- gene=951, id=81 removed by soft count = 129\n",
      "--- gene=951, id=121 removed by soft count = 82\n",
      "--- gene=951, id=163 removed by soft count = 304\n",
      "--- gene=951, id=164 removed by soft count = 127\n",
      "--- gene=951, id=179 removed by soft count = 55\n",
      "--- gene=951, id=180 removed by soft count = 54\n",
      "--- number of probes: 188 , kept by blast: 179 , if remove dups: 179\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 952\n",
      "--- gene=952, id=17 removed by soft count = 53\n",
      "--- gene=952, id=20 removed by soft count = 261\n",
      "--- gene=952, id=21 removed by soft count = 135\n",
      "--- gene=952, id=52 removed by soft count = 635\n",
      "--- gene=952, id=99 removed by soft count = 1135\n",
      "--- number of probes: 190 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 953\n",
      "--- gene=953, id=89 removed by soft count = 89\n",
      "--- gene=953, id=90 removed by soft count = 279\n",
      "--- gene=953, id=91 removed by soft count = 4446\n",
      "--- gene=953, id=92 removed by soft count = 2596\n",
      "--- gene=953, id=199 removed by soft count = 189\n",
      "--- gene=953, id=200 removed by soft count = 174\n",
      "--- gene=953, id=201 removed by soft count = 398\n",
      "--- gene=953, id=202 removed by soft count = 251\n",
      "--- number of probes: 245 , kept by blast: 237 , if remove dups: 237\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 954\n",
      "--- gene=954, id=25 removed by soft count = 63\n",
      "--- gene=954, id=26 removed by soft count = 58\n",
      "--- gene=954, id=27 removed by soft count = 363\n",
      "--- gene=954, id=31 removed by soft count = 207\n",
      "--- gene=954, id=52 removed by soft count = 132\n",
      "--- gene=954, id=80 removed by soft count = 96\n",
      "--- gene=954, id=85 removed by soft count = 138\n",
      "--- gene=954, id=86 removed by soft count = 124\n",
      "--- gene=954, id=87 removed by soft count = 159\n",
      "--- number of probes: 90 , kept by blast: 81 , if remove dups: 80\n",
      "-- number of probes kept for this region: 81\n",
      "-- checking probes in region: 955\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=955, id=94 removed by soft count = 75\n",
      "--- gene=955, id=102 removed by soft count = 51\n",
      "--- gene=955, id=105 removed by soft count = 121\n",
      "--- gene=955, id=128 removed by soft count = 157\n",
      "--- gene=955, id=129 removed by soft count = 1040\n",
      "--- gene=955, id=130 removed by soft count = 350\n",
      "--- gene=955, id=131 removed by soft count = 5262\n",
      "--- gene=955, id=141 removed by soft count = 79\n",
      "--- gene=955, id=170 removed by soft count = 162\n",
      "--- number of probes: 174 , kept by blast: 165 , if remove dups: 165\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 956\n",
      "--- gene=956, id=29 removed by soft count = 64\n",
      "--- gene=956, id=52 removed by soft count = 107\n",
      "--- gene=956, id=98 removed by soft count = 106\n",
      "--- gene=956, id=232 removed by soft count = 135\n",
      "--- number of probes: 265 , kept by blast: 261 , if remove dups: 260\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 957\n",
      "--- gene=957, id=0 removed by soft count = 110\n",
      "--- gene=957, id=1 removed by soft count = 447\n",
      "--- gene=957, id=2 removed by soft count = 250\n",
      "--- gene=957, id=8 removed by soft count = 80\n",
      "--- gene=957, id=113 removed by soft count = 171\n",
      "--- gene=957, id=257 removed by soft count = 63\n",
      "--- number of probes: 258 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 958\n",
      "--- gene=958, id=68 removed by soft count = 254\n",
      "--- gene=958, id=88 removed by soft count = 137\n",
      "--- gene=958, id=89 removed by soft count = 83\n",
      "--- gene=958, id=91 removed by soft count = 198\n",
      "--- gene=958, id=187 removed by soft count = 156\n",
      "--- gene=958, id=189 removed by soft count = 457\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 959\n",
      "--- gene=959, id=3 removed by soft count = 54\n",
      "--- gene=959, id=173 removed by soft count = 53\n",
      "--- gene=959, id=174 removed by soft count = 272\n",
      "--- gene=959, id=187 removed by soft count = 62\n",
      "--- gene=959, id=193 removed by soft count = 72\n",
      "--- gene=959, id=196 removed by soft count = 76\n",
      "--- number of probes: 213 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 960\n",
      "--- gene=960, id=94 removed by soft count = 205\n",
      "--- number of probes: 279 , kept by blast: 278 , if remove dups: 277\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 961\n",
      "--- gene=961, id=5 removed by soft count = 206\n",
      "--- gene=961, id=111 removed by soft count = 69\n",
      "--- gene=961, id=112 removed by soft count = 113\n",
      "--- gene=961, id=137 removed by soft count = 83\n",
      "--- gene=961, id=138 removed by soft count = 58\n",
      "--- gene=961, id=152 removed by soft count = 127\n",
      "--- gene=961, id=229 removed by soft count = 125\n",
      "--- gene=961, id=231 removed by soft count = 290\n",
      "--- gene=961, id=233 removed by soft count = 343\n",
      "--- gene=961, id=236 removed by soft count = 156\n",
      "--- number of probes: 237 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 962\n",
      "--- gene=962, id=0 removed by soft count = 258\n",
      "--- gene=962, id=14 removed by soft count = 152\n",
      "--- gene=962, id=27 removed by soft count = 52\n",
      "--- gene=962, id=31 removed by soft count = 66\n",
      "--- gene=962, id=35 removed by soft count = 491\n",
      "--- gene=962, id=37 removed by soft count = 180\n",
      "--- gene=962, id=39 removed by soft count = 123\n",
      "--- gene=962, id=171 removed by soft count = 189\n",
      "--- gene=962, id=172 removed by soft count = 54\n",
      "--- number of probes: 226 , kept by blast: 217 , if remove dups: 217\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 963\n",
      "--- gene=963, id=0 removed by soft count = 335\n",
      "--- gene=963, id=1 removed by soft count = 146\n",
      "--- gene=963, id=2 removed by soft count = 201\n",
      "--- gene=963, id=3 removed by soft count = 473\n",
      "--- gene=963, id=6 removed by soft count = 128\n",
      "--- gene=963, id=7 removed by soft count = 368\n",
      "--- gene=963, id=8 removed by soft count = 419\n",
      "--- gene=963, id=9 removed by soft count = 94\n",
      "--- gene=963, id=10 removed by soft count = 435\n",
      "--- gene=963, id=11 removed by soft count = 370\n",
      "--- gene=963, id=12 removed by soft count = 199\n",
      "--- gene=963, id=13 removed by soft count = 187\n",
      "--- gene=963, id=14 removed by soft count = 350\n",
      "--- gene=963, id=15 removed by soft count = 63\n",
      "--- gene=963, id=16 removed by soft count = 559\n",
      "--- gene=963, id=17 removed by soft count = 381\n",
      "--- gene=963, id=18 removed by soft count = 163\n",
      "--- gene=963, id=19 removed by soft count = 506\n",
      "--- gene=963, id=20 removed by soft count = 57\n",
      "--- gene=963, id=21 removed by soft count = 853\n",
      "--- gene=963, id=40 removed by soft count = 176\n",
      "--- gene=963, id=41 removed by soft count = 69\n",
      "--- gene=963, id=42 removed by soft count = 105\n",
      "--- gene=963, id=43 removed by soft count = 91\n",
      "--- gene=963, id=45 removed by soft count = 98\n",
      "--- gene=963, id=66 removed by soft count = 87\n",
      "--- gene=963, id=101 removed by soft count = 69\n",
      "--- gene=963, id=104 removed by soft count = 339\n",
      "--- gene=963, id=107 removed by soft count = 63\n",
      "--- gene=963, id=175 removed by soft count = 1046\n",
      "--- gene=963, id=178 removed by soft count = 83\n",
      "--- number of probes: 195 , kept by blast: 164 , if remove dups: 164\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 964\n",
      "--- gene=964, id=30 removed by soft count = 457\n",
      "--- gene=964, id=33 removed by soft count = 163\n",
      "--- gene=964, id=45 removed by soft count = 681\n",
      "--- gene=964, id=49 removed by soft count = 52\n",
      "--- gene=964, id=51 removed by soft count = 82\n",
      "--- gene=964, id=53 removed by soft count = 104\n",
      "--- gene=964, id=54 removed by soft count = 115\n",
      "--- gene=964, id=58 removed by soft count = 658\n",
      "--- gene=964, id=59 removed by soft count = 245\n",
      "--- gene=964, id=104 removed by soft count = 2581\n",
      "--- gene=964, id=108 removed by soft count = 69\n",
      "--- gene=964, id=137 removed by soft count = 68\n",
      "--- gene=964, id=139 removed by soft count = 217\n",
      "--- gene=964, id=140 removed by soft count = 152\n",
      "--- gene=964, id=141 removed by soft count = 781\n",
      "--- gene=964, id=145 removed by soft count = 113\n",
      "--- number of probes: 149 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 965\n",
      "--- gene=965, id=30 removed by soft count = 151\n",
      "--- gene=965, id=38 removed by soft count = 78\n",
      "--- gene=965, id=40 removed by soft count = 51\n",
      "--- gene=965, id=45 removed by soft count = 101\n",
      "--- gene=965, id=47 removed by soft count = 72\n",
      "--- gene=965, id=48 removed by soft count = 162\n",
      "--- gene=965, id=49 removed by soft count = 78\n",
      "--- gene=965, id=114 removed by soft count = 82\n",
      "--- gene=965, id=116 removed by soft count = 128\n",
      "--- gene=965, id=128 removed by soft count = 128\n",
      "--- gene=965, id=153 removed by soft count = 57\n",
      "--- gene=965, id=156 removed by soft count = 74\n",
      "--- gene=965, id=159 removed by soft count = 99\n",
      "--- number of probes: 191 , kept by blast: 178 , if remove dups: 178\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 966\n",
      "--- gene=966, id=107 removed by soft count = 3315\n",
      "--- gene=966, id=108 removed by soft count = 6698\n",
      "--- gene=966, id=110 removed by soft count = 276\n",
      "--- number of probes: 161 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 967\n",
      "--- gene=967, id=66 removed by soft count = 1248\n",
      "--- gene=967, id=100 removed by soft count = 438\n",
      "--- gene=967, id=119 removed by soft count = 125\n",
      "--- gene=967, id=135 removed by soft count = 742\n",
      "--- gene=967, id=136 removed by soft count = 1539\n",
      "--- gene=967, id=138 removed by soft count = 1168\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=967, id=139 removed by soft count = 2877\n",
      "--- gene=967, id=140 removed by soft count = 3602\n",
      "--- gene=967, id=141 removed by soft count = 163\n",
      "--- gene=967, id=142 removed by soft count = 51\n",
      "--- number of probes: 229 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 968\n",
      "--- gene=968, id=0 removed by soft count = 83\n",
      "--- gene=968, id=5 removed by soft count = 2873\n",
      "--- gene=968, id=102 removed by soft count = 57\n",
      "--- gene=968, id=137 removed by soft count = 174\n",
      "--- gene=968, id=139 removed by soft count = 724\n",
      "--- gene=968, id=208 removed by soft count = 107\n",
      "--- gene=968, id=209 removed by soft count = 705\n",
      "--- gene=968, id=212 removed by soft count = 141\n",
      "--- number of probes: 228 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 969\n",
      "--- gene=969, id=4 removed by soft count = 126\n",
      "--- gene=969, id=36 removed by soft count = 668\n",
      "--- gene=969, id=67 removed by soft count = 538\n",
      "--- gene=969, id=84 removed by soft count = 59\n",
      "--- gene=969, id=90 removed by soft count = 347\n",
      "--- gene=969, id=93 removed by soft count = 54\n",
      "--- gene=969, id=95 removed by soft count = 53\n",
      "--- gene=969, id=98 removed by soft count = 100\n",
      "--- gene=969, id=99 removed by soft count = 73\n",
      "--- gene=969, id=100 removed by soft count = 112\n",
      "--- gene=969, id=130 removed by soft count = 62\n",
      "--- gene=969, id=133 removed by soft count = 370\n",
      "--- number of probes: 155 , kept by blast: 143 , if remove dups: 141\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 970\n",
      "--- gene=970, id=50 removed by soft count = 89\n",
      "--- gene=970, id=51 removed by soft count = 132\n",
      "--- gene=970, id=53 removed by soft count = 59\n",
      "--- gene=970, id=54 removed by soft count = 83\n",
      "--- gene=970, id=56 removed by soft count = 165\n",
      "--- gene=970, id=57 removed by soft count = 61\n",
      "--- gene=970, id=58 removed by soft count = 116\n",
      "--- gene=970, id=59 removed by soft count = 82\n",
      "--- gene=970, id=60 removed by soft count = 187\n",
      "--- gene=970, id=63 removed by soft count = 81\n",
      "--- gene=970, id=70 removed by soft count = 149\n",
      "--- gene=970, id=71 removed by soft count = 80\n",
      "--- gene=970, id=72 removed by soft count = 73\n",
      "--- gene=970, id=75 removed by soft count = 70\n",
      "--- gene=970, id=76 removed by soft count = 88\n",
      "--- gene=970, id=77 removed by soft count = 56\n",
      "--- gene=970, id=78 removed by soft count = 57\n",
      "--- gene=970, id=79 removed by soft count = 87\n",
      "--- gene=970, id=80 removed by soft count = 92\n",
      "--- gene=970, id=81 removed by soft count = 64\n",
      "--- gene=970, id=82 removed by soft count = 98\n",
      "--- gene=970, id=84 removed by soft count = 57\n",
      "--- gene=970, id=86 removed by soft count = 56\n",
      "--- gene=970, id=88 removed by soft count = 252\n",
      "--- gene=970, id=91 removed by soft count = 150\n",
      "--- gene=970, id=96 removed by soft count = 58\n",
      "--- gene=970, id=97 removed by soft count = 131\n",
      "--- gene=970, id=100 removed by soft count = 183\n",
      "--- gene=970, id=103 removed by soft count = 116\n",
      "--- gene=970, id=106 removed by soft count = 53\n",
      "--- number of probes: 110 , kept by blast: 80 , if remove dups: 80\n",
      "-- number of probes kept for this region: 80\n",
      "-- checking probes in region: 971\n",
      "--- gene=971, id=80 removed by soft count = 160\n",
      "--- gene=971, id=81 removed by soft count = 301\n",
      "--- gene=971, id=87 removed by soft count = 107\n",
      "--- gene=971, id=155 removed by soft count = 217\n",
      "--- gene=971, id=156 removed by soft count = 546\n",
      "--- gene=971, id=157 removed by soft count = 99\n",
      "--- gene=971, id=159 removed by soft count = 105\n",
      "--- gene=971, id=161 removed by soft count = 87\n",
      "--- gene=971, id=164 removed by soft count = 87\n",
      "--- gene=971, id=166 removed by soft count = 80\n",
      "--- gene=971, id=167 removed by soft count = 1198\n",
      "--- number of probes: 173 , kept by blast: 162 , if remove dups: 162\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 972\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 973\n",
      "--- gene=973, id=83 removed by soft count = 1321\n",
      "--- gene=973, id=173 removed by soft count = 993\n",
      "--- gene=973, id=197 removed by soft count = 53\n",
      "--- gene=973, id=222 removed by soft count = 110\n",
      "--- gene=973, id=261 removed by soft count = 55\n",
      "--- number of probes: 270 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 974\n",
      "--- gene=974, id=162 removed by soft count = 74\n",
      "--- gene=974, id=187 removed by soft count = 351\n",
      "--- number of probes: 208 , kept by blast: 206 , if remove dups: 206\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 975\n",
      "--- number of probes: 299 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 976\n",
      "--- gene=976, id=21 removed by soft count = 92\n",
      "--- gene=976, id=69 removed by soft count = 144\n",
      "--- gene=976, id=70 removed by soft count = 210\n",
      "--- gene=976, id=71 removed by soft count = 281\n",
      "--- gene=976, id=73 removed by soft count = 286\n",
      "--- gene=976, id=74 removed by soft count = 267\n",
      "--- gene=976, id=99 removed by soft count = 95\n",
      "--- number of probes: 194 , kept by blast: 187 , if remove dups: 187\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 977\n",
      "--- gene=977, id=96 removed by soft count = 76\n",
      "--- gene=977, id=97 removed by soft count = 295\n",
      "--- gene=977, id=264 removed by soft count = 61\n",
      "--- gene=977, id=268 removed by soft count = 52\n",
      "--- gene=977, id=269 removed by soft count = 102\n",
      "--- number of probes: 297 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 978\n",
      "--- gene=978, id=39 removed by soft count = 126\n",
      "--- gene=978, id=55 removed by soft count = 4621\n",
      "--- gene=978, id=63 removed by soft count = 79\n",
      "--- gene=978, id=64 removed by soft count = 121\n",
      "--- gene=978, id=78 removed by soft count = 70\n",
      "--- gene=978, id=79 removed by soft count = 130\n",
      "--- gene=978, id=84 removed by soft count = 366\n",
      "--- gene=978, id=109 removed by soft count = 144\n",
      "--- gene=978, id=122 removed by soft count = 6323\n",
      "--- number of probes: 130 , kept by blast: 121 , if remove dups: 121\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 979\n",
      "--- gene=979, id=112 removed by soft count = 57\n",
      "--- gene=979, id=115 removed by soft count = 57\n",
      "--- gene=979, id=118 removed by soft count = 95\n",
      "--- gene=979, id=119 removed by soft count = 58\n",
      "--- gene=979, id=120 removed by soft count = 128\n",
      "--- gene=979, id=123 removed by soft count = 452\n",
      "--- gene=979, id=127 removed by soft count = 322\n",
      "--- gene=979, id=129 removed by soft count = 432\n",
      "--- gene=979, id=130 removed by soft count = 256\n",
      "--- gene=979, id=131 removed by soft count = 320\n",
      "--- gene=979, id=139 removed by soft count = 128\n",
      "--- gene=979, id=142 removed by soft count = 89\n",
      "--- gene=979, id=143 removed by soft count = 58\n",
      "--- gene=979, id=144 removed by soft count = 312\n",
      "--- gene=979, id=147 removed by soft count = 98\n",
      "--- gene=979, id=148 removed by soft count = 85\n",
      "--- gene=979, id=149 removed by soft count = 92\n",
      "--- gene=979, id=204 removed by soft count = 72\n",
      "--- gene=979, id=206 removed by soft count = 58\n",
      "--- number of probes: 255 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 980\n",
      "--- gene=980, id=38 removed by soft count = 70\n",
      "--- gene=980, id=73 removed by soft count = 94\n",
      "--- gene=980, id=91 removed by soft count = 121\n",
      "--- gene=980, id=93 removed by soft count = 181\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=980, id=142 removed by soft count = 232\n",
      "--- gene=980, id=143 removed by soft count = 54\n",
      "--- gene=980, id=144 removed by soft count = 101\n",
      "--- gene=980, id=146 removed by soft count = 162\n",
      "--- gene=980, id=147 removed by soft count = 125\n",
      "--- gene=980, id=148 removed by soft count = 433\n",
      "--- gene=980, id=166 removed by soft count = 539\n",
      "--- gene=980, id=167 removed by soft count = 1017\n",
      "--- number of probes: 173 , kept by blast: 161 , if remove dups: 161\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 981\n",
      "--- gene=981, id=5 removed by soft count = 77\n",
      "--- gene=981, id=247 removed by soft count = 65\n",
      "--- gene=981, id=277 removed by soft count = 118\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 982\n",
      "--- gene=982, id=0 removed by soft count = 474\n",
      "--- gene=982, id=36 removed by soft count = 151\n",
      "--- gene=982, id=40 removed by soft count = 167\n",
      "--- gene=982, id=42 removed by soft count = 55\n",
      "--- gene=982, id=43 removed by soft count = 90\n",
      "--- gene=982, id=46 removed by soft count = 172\n",
      "--- gene=982, id=47 removed by soft count = 186\n",
      "--- gene=982, id=131 removed by soft count = 810\n",
      "--- number of probes: 171 , kept by blast: 163 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 983\n",
      "--- gene=983, id=157 removed by soft count = 842\n",
      "--- gene=983, id=159 removed by soft count = 157\n",
      "--- gene=983, id=161 removed by soft count = 58\n",
      "--- gene=983, id=162 removed by soft count = 81\n",
      "--- number of probes: 236 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 984\n",
      "--- gene=984, id=157 removed by soft count = 603\n",
      "--- gene=984, id=168 removed by soft count = 65\n",
      "--- gene=984, id=246 removed by soft count = 55\n",
      "--- gene=984, id=257 removed by soft count = 163\n",
      "--- gene=984, id=258 removed by soft count = 85\n",
      "--- gene=984, id=260 removed by soft count = 74\n",
      "--- gene=984, id=267 removed by soft count = 68\n",
      "--- number of probes: 269 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 985\n",
      "--- gene=985, id=213 removed by hard count = 0\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 986\n",
      "--- gene=986, id=133 removed by soft count = 54\n",
      "--- gene=986, id=135 removed by soft count = 254\n",
      "--- number of probes: 233 , kept by blast: 231 , if remove dups: 231\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 987\n",
      "--- gene=987, id=9 removed by hard count = 0\n",
      "--- number of probes: 291 , kept by blast: 290 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 988\n",
      "--- gene=988, id=212 removed by soft count = 89\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 989\n",
      "--- gene=989, id=6 removed by soft count = 1378\n",
      "--- gene=989, id=7 removed by soft count = 1230\n",
      "--- gene=989, id=8 removed by soft count = 208\n",
      "--- gene=989, id=9 removed by soft count = 183\n",
      "--- gene=989, id=29 removed by soft count = 65\n",
      "--- gene=989, id=30 removed by soft count = 119\n",
      "--- gene=989, id=51 removed by soft count = 115\n",
      "--- gene=989, id=78 removed by soft count = 121\n",
      "--- gene=989, id=79 removed by soft count = 625\n",
      "--- gene=989, id=80 removed by soft count = 921\n",
      "--- gene=989, id=82 removed by soft count = 718\n",
      "--- number of probes: 154 , kept by blast: 143 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 990\n",
      "--- gene=990, id=27 removed by soft count = 65\n",
      "--- gene=990, id=31 removed by soft count = 118\n",
      "--- gene=990, id=74 removed by soft count = 72\n",
      "--- gene=990, id=223 removed by soft count = 107\n",
      "--- gene=990, id=250 removed by soft count = 131\n",
      "--- gene=990, id=251 removed by soft count = 173\n",
      "--- gene=990, id=252 removed by soft count = 54\n",
      "--- number of probes: 281 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 991\n",
      "--- gene=991, id=24 removed by soft count = 118\n",
      "--- gene=991, id=91 removed by soft count = 347\n",
      "--- gene=991, id=94 removed by soft count = 104\n",
      "--- gene=991, id=96 removed by soft count = 361\n",
      "--- gene=991, id=183 removed by soft count = 94\n",
      "--- gene=991, id=211 removed by soft count = 245\n",
      "--- gene=991, id=214 removed by soft count = 78\n",
      "--- gene=991, id=221 removed by soft count = 56\n",
      "--- gene=991, id=222 removed by soft count = 188\n",
      "--- number of probes: 247 , kept by blast: 238 , if remove dups: 238\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 992\n",
      "--- gene=992, id=20 removed by soft count = 62\n",
      "--- number of probes: 237 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 993\n",
      "--- gene=993, id=1 removed by soft count = 204\n",
      "--- gene=993, id=35 removed by soft count = 156\n",
      "--- gene=993, id=142 removed by soft count = 97\n",
      "--- number of probes: 153 , kept by blast: 150 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 994\n",
      "--- gene=994, id=37 removed by soft count = 128\n",
      "--- gene=994, id=61 removed by soft count = 88\n",
      "--- gene=994, id=131 removed by soft count = 70\n",
      "--- gene=994, id=182 removed by soft count = 52\n",
      "--- gene=994, id=268 removed by soft count = 80\n",
      "--- gene=994, id=269 removed by soft count = 78\n",
      "--- gene=994, id=271 removed by soft count = 70\n",
      "--- gene=994, id=272 removed by soft count = 91\n",
      "--- gene=994, id=276 removed by soft count = 66\n",
      "--- number of probes: 286 , kept by blast: 277 , if remove dups: 277\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 995\n",
      "--- gene=995, id=26 removed by soft count = 177\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 996\n",
      "--- gene=996, id=83 removed by soft count = 57\n",
      "--- gene=996, id=95 removed by soft count = 74\n",
      "--- gene=996, id=103 removed by soft count = 53\n",
      "--- gene=996, id=159 removed by soft count = 69\n",
      "--- gene=996, id=252 removed by soft count = 236\n",
      "--- gene=996, id=253 removed by soft count = 176\n",
      "--- number of probes: 255 , kept by blast: 249 , if remove dups: 249\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 997\n",
      "--- gene=997, id=112 removed by soft count = 71\n",
      "--- gene=997, id=113 removed by soft count = 90\n",
      "--- gene=997, id=114 removed by soft count = 78\n",
      "--- gene=997, id=220 removed by soft count = 107\n",
      "--- gene=997, id=289 removed by soft count = 72\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 999\n",
      "--- gene=999, id=67 removed by soft count = 57\n",
      "--- gene=999, id=198 removed by soft count = 54\n",
      "--- gene=999, id=238 removed by soft count = 157\n",
      "--- gene=999, id=257 removed by soft count = 173\n",
      "--- gene=999, id=261 removed by soft count = 62\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1000\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1000, id=158 removed by soft count = 93\n",
      "--- gene=1000, id=166 removed by soft count = 164\n",
      "--- gene=1000, id=179 removed by soft count = 173\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1001\n",
      "--- gene=1001, id=30 removed by soft count = 169\n",
      "--- gene=1001, id=34 removed by soft count = 1070\n",
      "--- gene=1001, id=213 removed by soft count = 59\n",
      "--- gene=1001, id=219 removed by soft count = 61\n",
      "--- gene=1001, id=221 removed by soft count = 92\n",
      "--- gene=1001, id=223 removed by soft count = 192\n",
      "--- gene=1001, id=226 removed by soft count = 165\n",
      "--- gene=1001, id=227 removed by soft count = 57\n",
      "--- gene=1001, id=240 removed by soft count = 168\n",
      "--- gene=1001, id=254 removed by soft count = 320\n",
      "--- gene=1001, id=256 removed by soft count = 52\n",
      "--- gene=1001, id=276 removed by soft count = 539\n",
      "--- gene=1001, id=293 removed by soft count = 102\n",
      "--- gene=1001, id=295 removed by soft count = 249\n",
      "--- number of probes: 300 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1002\n",
      "--- gene=1002, id=277 removed by soft count = 4821\n",
      "--- gene=1002, id=296 removed by soft count = 85\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1003\n",
      "--- gene=1003, id=76 removed by soft count = 113\n",
      "--- gene=1003, id=86 removed by soft count = 141\n",
      "--- gene=1003, id=90 removed by soft count = 73\n",
      "--- gene=1003, id=91 removed by soft count = 80\n",
      "--- gene=1003, id=92 removed by hard count = 6\n",
      "--- gene=1003, id=267 removed by soft count = 75\n",
      "--- gene=1003, id=274 removed by soft count = 106\n",
      "--- number of probes: 287 , kept by blast: 280 , if remove dups: 280\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1004\n",
      "--- number of probes: 283 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1005\n",
      "--- gene=1005, id=209 removed by soft count = 112\n",
      "--- gene=1005, id=210 removed by soft count = 311\n",
      "--- gene=1005, id=231 removed by soft count = 69\n",
      "--- gene=1005, id=237 removed by soft count = 383\n",
      "--- number of probes: 292 , kept by blast: 288 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1020\n",
      "--- gene=1020, id=19 removed by soft count = 261\n",
      "--- gene=1020, id=21 removed by hard count = 3\n",
      "--- gene=1020, id=25 removed by hard count = 3\n",
      "--- number of probes: 52 , kept by blast: 49 , if remove dups: 49\n",
      "-- number of probes kept for this region: 49\n",
      "-- checking probes in region: 1023\n",
      "--- gene=1023, id=20 removed by soft count = 492\n",
      "--- gene=1023, id=74 removed by soft count = 527\n",
      "--- gene=1023, id=91 removed by soft count = 69\n",
      "--- gene=1023, id=97 removed by soft count = 93\n",
      "--- gene=1023, id=98 removed by soft count = 74\n",
      "--- gene=1023, id=99 removed by soft count = 51\n",
      "--- gene=1023, id=100 removed by soft count = 52\n",
      "--- gene=1023, id=101 removed by soft count = 107\n",
      "--- gene=1023, id=109 removed by soft count = 182\n",
      "--- gene=1023, id=112 removed by soft count = 89\n",
      "--- gene=1023, id=113 removed by soft count = 79\n",
      "--- gene=1023, id=164 removed by soft count = 107\n",
      "--- gene=1023, id=165 removed by soft count = 74\n",
      "--- gene=1023, id=167 removed by soft count = 975\n",
      "--- gene=1023, id=191 removed by soft count = 79\n",
      "--- number of probes: 217 , kept by blast: 202 , if remove dups: 202\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1024\n",
      "--- gene=1024, id=17 removed by soft count = 561\n",
      "--- gene=1024, id=46 removed by soft count = 96\n",
      "--- gene=1024, id=47 removed by soft count = 1358\n",
      "--- gene=1024, id=53 removed by soft count = 158\n",
      "--- gene=1024, id=91 removed by soft count = 617\n",
      "--- gene=1024, id=92 removed by soft count = 98\n",
      "--- gene=1024, id=110 removed by soft count = 109\n",
      "--- gene=1024, id=117 removed by soft count = 251\n",
      "--- gene=1024, id=118 removed by soft count = 67\n",
      "--- gene=1024, id=124 removed by soft count = 88\n",
      "--- gene=1024, id=130 removed by soft count = 51\n",
      "--- gene=1024, id=137 removed by soft count = 68\n",
      "--- gene=1024, id=172 removed by soft count = 78\n",
      "--- number of probes: 242 , kept by blast: 229 , if remove dups: 229\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1025\n",
      "--- gene=1025, id=41 removed by soft count = 72\n",
      "--- gene=1025, id=100 removed by soft count = 54\n",
      "--- gene=1025, id=103 removed by soft count = 195\n",
      "--- gene=1025, id=104 removed by soft count = 91\n",
      "--- gene=1025, id=260 removed by soft count = 55\n",
      "--- gene=1025, id=278 removed by soft count = 128\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1029\n",
      "--- gene=1029, id=159 removed by soft count = 57\n",
      "--- gene=1029, id=230 removed by soft count = 69\n",
      "--- gene=1029, id=232 removed by soft count = 361\n",
      "--- number of probes: 235 , kept by blast: 232 , if remove dups: 232\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1030\n",
      "--- gene=1030, id=72 removed by soft count = 98\n",
      "--- number of probes: 209 , kept by blast: 208 , if remove dups: 208\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1031\n",
      "--- gene=1031, id=62 removed by soft count = 433\n",
      "--- gene=1031, id=104 removed by soft count = 64\n",
      "--- gene=1031, id=219 removed by soft count = 150\n",
      "--- gene=1031, id=224 removed by soft count = 113\n",
      "--- number of probes: 290 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1032\n",
      "--- gene=1032, id=259 removed by soft count = 445\n",
      "--- number of probes: 275 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1033\n",
      "--- gene=1033, id=2 removed by soft count = 442\n",
      "--- gene=1033, id=4 removed by soft count = 574\n",
      "--- gene=1033, id=27 removed by soft count = 1796\n",
      "--- gene=1033, id=120 removed by soft count = 71\n",
      "--- gene=1033, id=121 removed by soft count = 365\n",
      "--- gene=1033, id=122 removed by soft count = 191\n",
      "--- gene=1033, id=123 removed by soft count = 1418\n",
      "--- gene=1033, id=124 removed by soft count = 195\n",
      "--- gene=1033, id=125 removed by soft count = 122\n",
      "--- gene=1033, id=129 removed by soft count = 183\n",
      "--- gene=1033, id=196 removed by soft count = 56\n",
      "--- gene=1033, id=200 removed by soft count = 874\n",
      "--- gene=1033, id=202 removed by soft count = 76\n",
      "--- number of probes: 234 , kept by blast: 221 , if remove dups: 221\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1034\n",
      "--- gene=1034, id=9 removed by soft count = 674\n",
      "--- gene=1034, id=17 removed by soft count = 295\n",
      "--- gene=1034, id=72 removed by soft count = 152\n",
      "--- number of probes: 183 , kept by blast: 180 , if remove dups: 180\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1035\n",
      "--- number of probes: 201 , kept by blast: 201 , if remove dups: 201\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1036\n",
      "--- number of probes: 130 , kept by blast: 130 , if remove dups: 130\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1037\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1037, id=68 removed by soft count = 344\n",
      "--- gene=1037, id=142 removed by soft count = 69\n",
      "--- gene=1037, id=145 removed by soft count = 159\n",
      "--- gene=1037, id=230 removed by soft count = 53\n",
      "--- number of probes: 237 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1038\n",
      "--- gene=1038, id=45 removed by soft count = 838\n",
      "--- gene=1038, id=121 removed by soft count = 57\n",
      "--- gene=1038, id=123 removed by soft count = 426\n",
      "--- gene=1038, id=175 removed by soft count = 236\n",
      "--- gene=1038, id=177 removed by soft count = 63\n",
      "--- gene=1038, id=178 removed by soft count = 111\n",
      "--- gene=1038, id=183 removed by soft count = 86\n",
      "--- gene=1038, id=195 removed by soft count = 100\n",
      "--- gene=1038, id=199 removed by soft count = 84\n",
      "--- gene=1038, id=225 removed by soft count = 205\n",
      "--- number of probes: 227 , kept by blast: 217 , if remove dups: 217\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1039\n",
      "--- gene=1039, id=113 removed by soft count = 56\n",
      "--- gene=1039, id=114 removed by soft count = 229\n",
      "--- number of probes: 246 , kept by blast: 244 , if remove dups: 244\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1040\n",
      "--- gene=1040, id=174 removed by soft count = 68\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1041\n",
      "--- gene=1041, id=70 removed by soft count = 1141\n",
      "--- gene=1041, id=73 removed by soft count = 203\n",
      "--- gene=1041, id=233 removed by soft count = 466\n",
      "--- number of probes: 254 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1042\n",
      "--- gene=1042, id=64 removed by soft count = 58\n",
      "--- gene=1042, id=65 removed by soft count = 94\n",
      "--- gene=1042, id=287 removed by soft count = 233\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1043\n",
      "--- gene=1043, id=47 removed by soft count = 83\n",
      "--- gene=1043, id=53 removed by soft count = 259\n",
      "--- gene=1043, id=60 removed by soft count = 439\n",
      "--- gene=1043, id=61 removed by soft count = 71\n",
      "--- gene=1043, id=65 removed by soft count = 899\n",
      "--- gene=1043, id=68 removed by soft count = 60\n",
      "--- gene=1043, id=84 removed by soft count = 15390\n",
      "--- gene=1043, id=98 removed by soft count = 324\n",
      "--- gene=1043, id=99 removed by soft count = 77\n",
      "--- gene=1043, id=100 removed by soft count = 403\n",
      "--- gene=1043, id=102 removed by soft count = 383\n",
      "--- gene=1043, id=106 removed by soft count = 856\n",
      "--- gene=1043, id=110 removed by soft count = 60\n",
      "--- gene=1043, id=112 removed by soft count = 88\n",
      "--- gene=1043, id=113 removed by soft count = 111\n",
      "--- gene=1043, id=118 removed by soft count = 75\n",
      "--- gene=1043, id=120 removed by soft count = 768\n",
      "--- number of probes: 122 , kept by blast: 105 , if remove dups: 105\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 105\n",
      "-- checking probes in region: 1044\n",
      "--- gene=1044, id=35 removed by soft count = 300\n",
      "--- gene=1044, id=74 removed by hard count = 0\n",
      "--- gene=1044, id=246 removed by soft count = 72\n",
      "--- gene=1044, id=248 removed by soft count = 67\n",
      "--- number of probes: 253 , kept by blast: 249 , if remove dups: 249\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1045\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1046\n",
      "--- gene=1046, id=216 removed by soft count = 59\n",
      "--- gene=1046, id=217 removed by soft count = 216\n",
      "--- gene=1046, id=218 removed by soft count = 157\n",
      "--- number of probes: 263 , kept by blast: 260 , if remove dups: 260\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1047\n",
      "--- gene=1047, id=27 removed by soft count = 133\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1048\n",
      "--- gene=1048, id=0 removed by soft count = 369\n",
      "--- gene=1048, id=16 removed by soft count = 1010\n",
      "--- gene=1048, id=53 removed by soft count = 53\n",
      "--- gene=1048, id=71 removed by soft count = 52\n",
      "--- gene=1048, id=72 removed by soft count = 54\n",
      "--- gene=1048, id=80 removed by soft count = 60\n",
      "--- gene=1048, id=81 removed by soft count = 58\n",
      "--- gene=1048, id=84 removed by soft count = 72\n",
      "--- gene=1048, id=88 removed by soft count = 62\n",
      "--- gene=1048, id=91 removed by soft count = 82\n",
      "--- gene=1048, id=93 removed by soft count = 62\n",
      "--- gene=1048, id=108 removed by soft count = 52\n",
      "--- gene=1048, id=119 removed by soft count = 51\n",
      "--- gene=1048, id=134 removed by soft count = 61\n",
      "--- gene=1048, id=135 removed by soft count = 55\n",
      "--- gene=1048, id=136 removed by soft count = 99\n",
      "--- gene=1048, id=137 removed by soft count = 247\n",
      "--- gene=1048, id=140 removed by soft count = 52\n",
      "--- gene=1048, id=142 removed by soft count = 63\n",
      "--- gene=1048, id=143 removed by soft count = 57\n",
      "--- gene=1048, id=147 removed by soft count = 67\n",
      "--- gene=1048, id=148 removed by soft count = 52\n",
      "--- gene=1048, id=149 removed by soft count = 61\n",
      "--- gene=1048, id=151 removed by soft count = 68\n",
      "--- gene=1048, id=153 removed by soft count = 54\n",
      "--- gene=1048, id=157 removed by soft count = 51\n",
      "--- gene=1048, id=158 removed by soft count = 53\n",
      "--- gene=1048, id=163 removed by soft count = 59\n",
      "--- gene=1048, id=164 removed by soft count = 75\n",
      "--- gene=1048, id=185 removed by soft count = 69\n",
      "--- gene=1048, id=187 removed by soft count = 51\n",
      "--- gene=1048, id=191 removed by soft count = 69\n",
      "--- gene=1048, id=193 removed by soft count = 70\n",
      "--- gene=1048, id=204 removed by soft count = 60\n",
      "--- gene=1048, id=208 removed by soft count = 56\n",
      "--- gene=1048, id=212 removed by soft count = 196\n",
      "--- gene=1048, id=227 removed by soft count = 60\n",
      "--- gene=1048, id=232 removed by soft count = 60\n",
      "--- gene=1048, id=233 removed by soft count = 59\n",
      "--- gene=1048, id=235 removed by soft count = 70\n",
      "--- gene=1048, id=238 removed by soft count = 59\n",
      "--- gene=1048, id=239 removed by soft count = 67\n",
      "--- gene=1048, id=249 removed by soft count = 59\n",
      "--- gene=1048, id=251 removed by soft count = 74\n",
      "--- gene=1048, id=255 removed by soft count = 76\n",
      "--- gene=1048, id=256 removed by soft count = 61\n",
      "--- gene=1048, id=258 removed by soft count = 66\n",
      "--- gene=1048, id=263 removed by soft count = 53\n",
      "--- gene=1048, id=270 removed by soft count = 56\n",
      "--- gene=1048, id=271 removed by soft count = 57\n",
      "--- number of probes: 292 , kept by blast: 242 , if remove dups: 240\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1049\n",
      "--- gene=1049, id=31 removed by soft count = 463\n",
      "--- gene=1049, id=39 removed by soft count = 159\n",
      "--- gene=1049, id=41 removed by soft count = 154\n",
      "--- gene=1049, id=45 removed by soft count = 184\n",
      "--- gene=1049, id=50 removed by soft count = 181\n",
      "--- gene=1049, id=85 removed by soft count = 125\n",
      "--- gene=1049, id=175 removed by soft count = 600\n",
      "--- number of probes: 176 , kept by blast: 169 , if remove dups: 169\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1050\n",
      "--- gene=1050, id=24 removed by soft count = 51\n",
      "--- gene=1050, id=28 removed by soft count = 51\n",
      "--- number of probes: 267 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1051\n",
      "--- gene=1051, id=36 removed by soft count = 137\n",
      "--- gene=1051, id=60 removed by soft count = 430\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1051, id=220 removed by soft count = 1910\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1052\n",
      "--- gene=1052, id=11 removed by soft count = 87\n",
      "--- gene=1052, id=13 removed by soft count = 497\n",
      "--- gene=1052, id=17 removed by soft count = 1850\n",
      "--- gene=1052, id=31 removed by soft count = 97\n",
      "--- gene=1052, id=43 removed by soft count = 53\n",
      "--- gene=1052, id=44 removed by soft count = 79\n",
      "--- gene=1052, id=46 removed by soft count = 78\n",
      "--- gene=1052, id=48 removed by soft count = 73\n",
      "--- gene=1052, id=51 removed by soft count = 704\n",
      "--- gene=1052, id=83 removed by soft count = 340\n",
      "--- gene=1052, id=112 removed by soft count = 58\n",
      "--- number of probes: 134 , kept by blast: 123 , if remove dups: 123\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1053\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1054\n",
      "--- gene=1054, id=259 removed by soft count = 329\n",
      "--- gene=1054, id=260 removed by soft count = 118\n",
      "--- gene=1054, id=261 removed by soft count = 52\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1055\n",
      "--- gene=1055, id=73 removed by soft count = 1593\n",
      "--- gene=1055, id=74 removed by soft count = 170\n",
      "--- gene=1055, id=77 removed by soft count = 543\n",
      "--- gene=1055, id=83 removed by soft count = 86\n",
      "--- gene=1055, id=85 removed by soft count = 290\n",
      "--- gene=1055, id=265 removed by soft count = 147\n",
      "--- number of probes: 268 , kept by blast: 262 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1056\n",
      "--- gene=1056, id=91 removed by soft count = 99\n",
      "--- gene=1056, id=92 removed by soft count = 53\n",
      "--- gene=1056, id=105 removed by soft count = 79\n",
      "--- gene=1056, id=106 removed by soft count = 84\n",
      "--- gene=1056, id=107 removed by soft count = 109\n",
      "--- gene=1056, id=108 removed by soft count = 76\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1057\n",
      "--- gene=1057, id=0 removed by soft count = 144\n",
      "--- gene=1057, id=57 removed by soft count = 335\n",
      "--- gene=1057, id=62 removed by soft count = 55\n",
      "--- gene=1057, id=64 removed by soft count = 69\n",
      "--- gene=1057, id=70 removed by soft count = 94\n",
      "--- gene=1057, id=74 removed by soft count = 171\n",
      "--- gene=1057, id=106 removed by soft count = 759\n",
      "--- number of probes: 155 , kept by blast: 148 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1058\n",
      "--- gene=1058, id=1 removed by soft count = 509\n",
      "--- gene=1058, id=2 removed by soft count = 203\n",
      "--- gene=1058, id=36 removed by soft count = 76\n",
      "--- gene=1058, id=62 removed by soft count = 106\n",
      "--- gene=1058, id=69 removed by soft count = 134\n",
      "--- gene=1058, id=86 removed by soft count = 82\n",
      "--- gene=1058, id=87 removed by soft count = 167\n",
      "--- number of probes: 131 , kept by blast: 124 , if remove dups: 124\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1059\n",
      "--- gene=1059, id=2 removed by soft count = 56\n",
      "--- gene=1059, id=3 removed by soft count = 62\n",
      "--- gene=1059, id=4 removed by soft count = 79\n",
      "--- gene=1059, id=5 removed by soft count = 65\n",
      "--- gene=1059, id=16 removed by soft count = 51\n",
      "--- number of probes: 164 , kept by blast: 159 , if remove dups: 154\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1060\n",
      "--- gene=1060, id=21 removed by soft count = 65\n",
      "--- gene=1060, id=24 removed by soft count = 186\n",
      "--- gene=1060, id=27 removed by soft count = 348\n",
      "--- gene=1060, id=28 removed by soft count = 195\n",
      "--- gene=1060, id=299 removed by soft count = 85\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1061\n",
      "--- gene=1061, id=3 removed by soft count = 146\n",
      "--- gene=1061, id=7 removed by soft count = 2973\n",
      "--- gene=1061, id=172 removed by soft count = 54\n",
      "--- gene=1061, id=208 removed by soft count = 580\n",
      "--- gene=1061, id=234 removed by soft count = 530\n",
      "--- gene=1061, id=235 removed by soft count = 145\n",
      "--- gene=1061, id=236 removed by soft count = 64\n",
      "--- gene=1061, id=286 removed by hard count = 4\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1062\n",
      "--- gene=1062, id=22 removed by soft count = 132\n",
      "--- gene=1062, id=24 removed by soft count = 656\n",
      "--- gene=1062, id=26 removed by soft count = 136\n",
      "--- gene=1062, id=43 removed by soft count = 88\n",
      "--- gene=1062, id=44 removed by soft count = 125\n",
      "--- number of probes: 240 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1063\n",
      "--- gene=1063, id=85 removed by soft count = 259\n",
      "--- gene=1063, id=186 removed by soft count = 676\n",
      "--- number of probes: 266 , kept by blast: 264 , if remove dups: 264\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1064\n",
      "--- gene=1064, id=21 removed by soft count = 128\n",
      "--- gene=1064, id=23 removed by soft count = 61\n",
      "--- gene=1064, id=279 removed by soft count = 110\n",
      "--- gene=1064, id=280 removed by soft count = 61\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1065\n",
      "--- gene=1065, id=33 removed by soft count = 189\n",
      "--- gene=1065, id=92 removed by soft count = 116\n",
      "--- gene=1065, id=213 removed by soft count = 147\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1066\n",
      "--- gene=1066, id=0 removed by soft count = 97\n",
      "--- gene=1066, id=2 removed by soft count = 126\n",
      "--- gene=1066, id=4 removed by soft count = 82\n",
      "--- gene=1066, id=5 removed by soft count = 95\n",
      "--- gene=1066, id=6 removed by soft count = 53\n",
      "--- gene=1066, id=174 removed by soft count = 9327\n",
      "--- number of probes: 207 , kept by blast: 201 , if remove dups: 201\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1067\n",
      "--- gene=1067, id=12 removed by soft count = 115\n",
      "--- gene=1067, id=14 removed by soft count = 240\n",
      "--- gene=1067, id=80 removed by soft count = 104\n",
      "--- number of probes: 204 , kept by blast: 201 , if remove dups: 201\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1068\n",
      "--- gene=1068, id=0 removed by soft count = 260\n",
      "--- gene=1068, id=1 removed by soft count = 122\n",
      "--- gene=1068, id=4 removed by soft count = 120\n",
      "--- gene=1068, id=10 removed by soft count = 584\n",
      "--- gene=1068, id=11 removed by soft count = 292\n",
      "--- gene=1068, id=12 removed by soft count = 103\n",
      "--- gene=1068, id=13 removed by soft count = 3098\n",
      "--- gene=1068, id=15 removed by soft count = 99\n",
      "--- gene=1068, id=16 removed by soft count = 66\n",
      "--- gene=1068, id=17 removed by soft count = 288\n",
      "--- gene=1068, id=18 removed by soft count = 124\n",
      "--- gene=1068, id=19 removed by soft count = 107\n",
      "--- gene=1068, id=25 removed by soft count = 65\n",
      "--- gene=1068, id=106 removed by soft count = 906\n",
      "--- gene=1068, id=112 removed by soft count = 538\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1068, id=126 removed by soft count = 667\n",
      "--- gene=1068, id=127 removed by soft count = 896\n",
      "--- gene=1068, id=128 removed by soft count = 59\n",
      "--- number of probes: 147 , kept by blast: 129 , if remove dups: 129\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1069\n",
      "--- gene=1069, id=0 removed by soft count = 514\n",
      "--- gene=1069, id=2 removed by soft count = 90\n",
      "--- gene=1069, id=9 removed by soft count = 59\n",
      "--- gene=1069, id=10 removed by soft count = 57\n",
      "--- gene=1069, id=12 removed by soft count = 140\n",
      "--- gene=1069, id=15 removed by soft count = 61\n",
      "--- gene=1069, id=17 removed by soft count = 125\n",
      "--- gene=1069, id=18 removed by soft count = 55\n",
      "--- gene=1069, id=20 removed by soft count = 160\n",
      "--- gene=1069, id=49 removed by soft count = 183\n",
      "--- gene=1069, id=56 removed by soft count = 82\n",
      "--- gene=1069, id=58 removed by soft count = 105\n",
      "--- gene=1069, id=59 removed by soft count = 306\n",
      "--- gene=1069, id=61 removed by soft count = 522\n",
      "--- gene=1069, id=63 removed by soft count = 5687\n",
      "--- gene=1069, id=64 removed by soft count = 309\n",
      "--- gene=1069, id=65 removed by soft count = 280\n",
      "--- gene=1069, id=66 removed by soft count = 674\n",
      "--- gene=1069, id=71 removed by soft count = 56\n",
      "--- gene=1069, id=139 removed by soft count = 391\n",
      "--- number of probes: 160 , kept by blast: 140 , if remove dups: 140\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1070\n",
      "--- gene=1070, id=62 removed by soft count = 598\n",
      "--- gene=1070, id=63 removed by soft count = 278\n",
      "--- gene=1070, id=64 removed by soft count = 227\n",
      "--- gene=1070, id=74 removed by soft count = 609\n",
      "--- gene=1070, id=75 removed by soft count = 1061\n",
      "--- gene=1070, id=210 removed by soft count = 149\n",
      "--- gene=1070, id=238 removed by soft count = 196\n",
      "--- number of probes: 250 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1071\n",
      "--- gene=1071, id=13 removed by soft count = 62\n",
      "--- gene=1071, id=28 removed by soft count = 51\n",
      "--- gene=1071, id=194 removed by soft count = 51\n",
      "--- gene=1071, id=196 removed by soft count = 359\n",
      "--- gene=1071, id=238 removed by soft count = 128\n",
      "--- number of probes: 291 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1072\n",
      "--- gene=1072, id=74 removed by soft count = 97\n",
      "--- gene=1072, id=76 removed by soft count = 80\n",
      "--- gene=1072, id=82 removed by soft count = 685\n",
      "--- gene=1072, id=84 removed by soft count = 124\n",
      "--- gene=1072, id=86 removed by soft count = 114\n",
      "--- gene=1072, id=87 removed by soft count = 94\n",
      "--- gene=1072, id=256 removed by soft count = 52\n",
      "--- number of probes: 259 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1073\n",
      "--- gene=1073, id=81 removed by soft count = 135\n",
      "--- gene=1073, id=121 removed by soft count = 95\n",
      "--- gene=1073, id=123 removed by soft count = 69\n",
      "--- gene=1073, id=125 removed by soft count = 68\n",
      "--- gene=1073, id=146 removed by soft count = 162\n",
      "--- gene=1073, id=168 removed by soft count = 334\n",
      "--- gene=1073, id=261 removed by soft count = 129\n",
      "--- gene=1073, id=262 removed by soft count = 60\n",
      "--- number of probes: 263 , kept by blast: 255 , if remove dups: 255\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1074\n",
      "--- gene=1074, id=107 removed by soft count = 289\n",
      "--- gene=1074, id=111 removed by soft count = 100\n",
      "--- gene=1074, id=155 removed by soft count = 123\n",
      "--- gene=1074, id=163 removed by soft count = 109\n",
      "--- gene=1074, id=168 removed by soft count = 70\n",
      "--- number of probes: 266 , kept by blast: 261 , if remove dups: 260\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1075\n",
      "--- gene=1075, id=171 removed by soft count = 470\n",
      "--- gene=1075, id=221 removed by soft count = 69\n",
      "--- number of probes: 256 , kept by blast: 254 , if remove dups: 254\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1076\n",
      "--- gene=1076, id=38 removed by soft count = 186\n",
      "--- gene=1076, id=53 removed by soft count = 290\n",
      "--- gene=1076, id=239 removed by soft count = 1888\n",
      "--- number of probes: 293 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1077\n",
      "--- gene=1077, id=24 removed by soft count = 58\n",
      "--- gene=1077, id=76 removed by soft count = 154\n",
      "--- number of probes: 88 , kept by blast: 86 , if remove dups: 85\n",
      "-- number of probes kept for this region: 86\n",
      "-- checking probes in region: 1078\n",
      "--- gene=1078, id=0 removed by soft count = 68\n",
      "--- gene=1078, id=38 removed by soft count = 72\n",
      "--- gene=1078, id=56 removed by soft count = 82\n",
      "--- gene=1078, id=65 removed by soft count = 123\n",
      "--- gene=1078, id=166 removed by soft count = 58\n",
      "--- number of probes: 241 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1079\n",
      "--- gene=1079, id=62 removed by soft count = 95\n",
      "--- gene=1079, id=96 removed by soft count = 429\n",
      "--- gene=1079, id=98 removed by soft count = 717\n",
      "--- gene=1079, id=99 removed by soft count = 93\n",
      "--- gene=1079, id=186 removed by soft count = 159\n",
      "--- gene=1079, id=205 removed by soft count = 477\n",
      "--- gene=1079, id=243 removed by soft count = 67\n",
      "--- gene=1079, id=247 removed by soft count = 129\n",
      "--- gene=1079, id=248 removed by soft count = 74\n",
      "--- gene=1079, id=297 removed by soft count = 421\n",
      "--- number of probes: 300 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1080\n",
      "--- gene=1080, id=86 removed by soft count = 62\n",
      "--- gene=1080, id=148 removed by soft count = 168\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1081\n",
      "--- gene=1081, id=20 removed by soft count = 66\n",
      "--- number of probes: 298 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1082\n",
      "--- gene=1082, id=74 removed by soft count = 68\n",
      "--- gene=1082, id=94 removed by soft count = 296\n",
      "--- gene=1082, id=105 removed by soft count = 2737\n",
      "--- gene=1082, id=107 removed by soft count = 190\n",
      "--- gene=1082, id=121 removed by soft count = 782\n",
      "--- gene=1082, id=123 removed by soft count = 61\n",
      "--- gene=1082, id=250 removed by soft count = 58\n",
      "--- gene=1082, id=251 removed by soft count = 126\n",
      "--- number of probes: 263 , kept by blast: 255 , if remove dups: 255\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1083\n",
      "--- gene=1083, id=224 removed by soft count = 132\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1085\n",
      "--- gene=1085, id=76 removed by soft count = 1264\n",
      "--- gene=1085, id=110 removed by soft count = 412\n",
      "--- gene=1085, id=187 removed by soft count = 53\n",
      "--- gene=1085, id=251 removed by soft count = 195\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1087\n",
      "--- gene=1087, id=218 removed by soft count = 350\n",
      "--- gene=1087, id=258 removed by soft count = 58\n",
      "--- gene=1087, id=290 removed by soft count = 51\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1089\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1089, id=6 removed by soft count = 282\n",
      "--- gene=1089, id=48 removed by soft count = 61\n",
      "--- gene=1089, id=51 removed by soft count = 54\n",
      "--- gene=1089, id=88 removed by soft count = 51\n",
      "--- gene=1089, id=93 removed by soft count = 845\n",
      "--- gene=1089, id=112 removed by soft count = 196\n",
      "--- gene=1089, id=113 removed by soft count = 88\n",
      "--- gene=1089, id=119 removed by soft count = 91\n",
      "--- gene=1089, id=155 removed by soft count = 120\n",
      "--- gene=1089, id=205 removed by soft count = 71\n",
      "--- number of probes: 207 , kept by blast: 197 , if remove dups: 197\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1090\n",
      "--- gene=1090, id=27 removed by soft count = 57\n",
      "--- gene=1090, id=34 removed by soft count = 54\n",
      "--- gene=1090, id=57 removed by soft count = 54\n",
      "--- number of probes: 261 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1091\n",
      "--- gene=1091, id=34 removed by soft count = 107\n",
      "--- gene=1091, id=39 removed by soft count = 127\n",
      "--- gene=1091, id=41 removed by soft count = 396\n",
      "--- gene=1091, id=43 removed by soft count = 203\n",
      "--- gene=1091, id=44 removed by soft count = 640\n",
      "--- gene=1091, id=45 removed by soft count = 385\n",
      "--- gene=1091, id=47 removed by soft count = 158\n",
      "--- gene=1091, id=50 removed by soft count = 162\n",
      "--- gene=1091, id=65 removed by soft count = 543\n",
      "--- gene=1091, id=122 removed by soft count = 142\n",
      "--- gene=1091, id=143 removed by soft count = 78\n",
      "--- gene=1091, id=147 removed by soft count = 606\n",
      "--- gene=1091, id=160 removed by soft count = 82\n",
      "--- gene=1091, id=204 removed by soft count = 355\n",
      "--- gene=1091, id=205 removed by soft count = 816\n",
      "--- number of probes: 226 , kept by blast: 211 , if remove dups: 211\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1092\n",
      "--- gene=1092, id=25 removed by soft count = 4840\n",
      "--- gene=1092, id=231 removed by soft count = 70\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1093\n",
      "--- gene=1093, id=106 removed by soft count = 96\n",
      "--- gene=1093, id=189 removed by soft count = 116\n",
      "--- gene=1093, id=192 removed by soft count = 143\n",
      "--- gene=1093, id=194 removed by soft count = 93\n",
      "--- number of probes: 203 , kept by blast: 199 , if remove dups: 195\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1094\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1097\n",
      "--- gene=1097, id=15 removed by soft count = 74\n",
      "--- gene=1097, id=39 removed by soft count = 82\n",
      "--- gene=1097, id=40 removed by soft count = 95\n",
      "--- gene=1097, id=183 removed by soft count = 565\n",
      "--- gene=1097, id=208 removed by soft count = 256\n",
      "--- gene=1097, id=216 removed by soft count = 188\n",
      "--- gene=1097, id=217 removed by soft count = 122\n",
      "--- gene=1097, id=219 removed by soft count = 149\n",
      "--- number of probes: 230 , kept by blast: 222 , if remove dups: 222\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1105\n",
      "--- gene=1105, id=37 removed by soft count = 82\n",
      "--- gene=1105, id=47 removed by soft count = 74\n",
      "--- gene=1105, id=48 removed by soft count = 238\n",
      "--- gene=1105, id=58 removed by soft count = 95\n",
      "--- gene=1105, id=112 removed by soft count = 142\n",
      "--- gene=1105, id=157 removed by soft count = 51\n",
      "--- number of probes: 223 , kept by blast: 217 , if remove dups: 217\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1106\n",
      "--- gene=1106, id=167 removed by soft count = 79\n",
      "--- gene=1106, id=168 removed by soft count = 420\n",
      "--- gene=1106, id=169 removed by soft count = 77\n",
      "--- gene=1106, id=173 removed by soft count = 440\n",
      "--- gene=1106, id=174 removed by soft count = 55\n",
      "--- gene=1106, id=176 removed by soft count = 138\n",
      "--- gene=1106, id=181 removed by soft count = 174\n",
      "--- gene=1106, id=185 removed by soft count = 4057\n",
      "--- gene=1106, id=186 removed by soft count = 174\n",
      "--- gene=1106, id=187 removed by soft count = 2824\n",
      "--- gene=1106, id=188 removed by soft count = 25737\n",
      "--- number of probes: 189 , kept by blast: 178 , if remove dups: 178\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1107\n",
      "--- number of probes: 267 , kept by blast: 267 , if remove dups: 267\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1108\n",
      "--- gene=1108, id=75 removed by soft count = 112\n",
      "--- gene=1108, id=76 removed by soft count = 73\n",
      "--- gene=1108, id=77 removed by soft count = 144\n",
      "--- number of probes: 295 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1109\n",
      "--- gene=1109, id=157 removed by soft count = 128\n",
      "--- number of probes: 184 , kept by blast: 183 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1110\n",
      "--- gene=1110, id=100 removed by soft count = 356\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1111\n",
      "--- gene=1111, id=94 removed by soft count = 289\n",
      "--- gene=1111, id=95 removed by soft count = 58\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1112\n",
      "--- gene=1112, id=248 removed by soft count = 53\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1113\n",
      "--- gene=1113, id=14 removed by soft count = 73\n",
      "--- gene=1113, id=120 removed by soft count = 149\n",
      "--- gene=1113, id=213 removed by soft count = 114\n",
      "--- gene=1113, id=230 removed by soft count = 206\n",
      "--- gene=1113, id=232 removed by soft count = 664\n",
      "--- number of probes: 266 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1114\n",
      "--- gene=1114, id=8 removed by soft count = 51\n",
      "--- gene=1114, id=9 removed by soft count = 72\n",
      "--- gene=1114, id=79 removed by soft count = 186\n",
      "--- gene=1114, id=106 removed by soft count = 108\n",
      "--- gene=1114, id=188 removed by soft count = 388\n",
      "--- number of probes: 215 , kept by blast: 210 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1115\n",
      "--- gene=1115, id=24 removed by soft count = 92\n",
      "--- gene=1115, id=62 removed by soft count = 120\n",
      "--- gene=1115, id=68 removed by soft count = 240\n",
      "--- gene=1115, id=83 removed by soft count = 127\n",
      "--- gene=1115, id=118 removed by soft count = 96\n",
      "--- gene=1115, id=201 removed by soft count = 130\n",
      "--- gene=1115, id=215 removed by soft count = 1139\n",
      "--- number of probes: 253 , kept by blast: 246 , if remove dups: 245\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1116\n",
      "--- gene=1116, id=36 removed by soft count = 73\n",
      "--- gene=1116, id=47 removed by soft count = 59\n",
      "--- gene=1116, id=56 removed by soft count = 60\n",
      "--- gene=1116, id=75 removed by soft count = 62\n",
      "--- gene=1116, id=77 removed by soft count = 84\n",
      "--- gene=1116, id=78 removed by soft count = 73\n",
      "--- gene=1116, id=79 removed by soft count = 545\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1116, id=93 removed by soft count = 102\n",
      "--- gene=1116, id=116 removed by soft count = 185\n",
      "--- gene=1116, id=117 removed by soft count = 498\n",
      "--- gene=1116, id=123 removed by soft count = 235\n",
      "--- gene=1116, id=125 removed by soft count = 192\n",
      "--- gene=1116, id=145 removed by soft count = 246\n",
      "--- gene=1116, id=146 removed by soft count = 59\n",
      "--- gene=1116, id=148 removed by soft count = 60\n",
      "--- gene=1116, id=149 removed by soft count = 290\n",
      "--- gene=1116, id=150 removed by soft count = 53\n",
      "--- gene=1116, id=157 removed by soft count = 8907\n",
      "--- gene=1116, id=158 removed by soft count = 89\n",
      "--- gene=1116, id=159 removed by soft count = 75\n",
      "--- gene=1116, id=161 removed by soft count = 60\n",
      "--- gene=1116, id=162 removed by soft count = 81\n",
      "--- number of probes: 184 , kept by blast: 162 , if remove dups: 162\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1117\n",
      "--- gene=1117, id=69 removed by soft count = 57\n",
      "--- gene=1117, id=265 removed by soft count = 340\n",
      "--- number of probes: 288 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1118\n",
      "--- gene=1118, id=94 removed by soft count = 82\n",
      "--- gene=1118, id=171 removed by soft count = 138\n",
      "--- gene=1118, id=185 removed by soft count = 89\n",
      "--- number of probes: 231 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1119\n",
      "--- gene=1119, id=80 removed by soft count = 136\n",
      "--- gene=1119, id=199 removed by soft count = 87\n",
      "--- gene=1119, id=202 removed by soft count = 90\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1120\n",
      "--- gene=1120, id=1 removed by soft count = 61\n",
      "--- gene=1120, id=107 removed by soft count = 1154\n",
      "--- gene=1120, id=183 removed by soft count = 1288\n",
      "--- gene=1120, id=190 removed by soft count = 3716\n",
      "--- gene=1120, id=193 removed by soft count = 51\n",
      "--- number of probes: 228 , kept by blast: 223 , if remove dups: 222\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1121\n",
      "--- gene=1121, id=20 removed by soft count = 111\n",
      "--- gene=1121, id=23 removed by soft count = 55\n",
      "--- gene=1121, id=156 removed by soft count = 54\n",
      "--- gene=1121, id=197 removed by soft count = 265\n",
      "--- gene=1121, id=256 removed by soft count = 56\n",
      "--- number of probes: 298 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1122\n",
      "--- gene=1122, id=1 removed by soft count = 89\n",
      "--- gene=1122, id=2 removed by soft count = 157\n",
      "--- gene=1122, id=72 removed by soft count = 56\n",
      "--- gene=1122, id=93 removed by soft count = 168\n",
      "--- gene=1122, id=139 removed by soft count = 666\n",
      "--- gene=1122, id=162 removed by soft count = 218\n",
      "--- number of probes: 227 , kept by blast: 221 , if remove dups: 221\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1123\n",
      "--- gene=1123, id=18 removed by soft count = 80\n",
      "--- gene=1123, id=22 removed by soft count = 67\n",
      "--- gene=1123, id=55 removed by soft count = 99\n",
      "--- gene=1123, id=56 removed by soft count = 93\n",
      "--- gene=1123, id=107 removed by soft count = 93\n",
      "--- gene=1123, id=108 removed by soft count = 121\n",
      "--- gene=1123, id=110 removed by soft count = 86\n",
      "--- gene=1123, id=230 removed by soft count = 159\n",
      "--- gene=1123, id=237 removed by soft count = 60\n",
      "--- gene=1123, id=244 removed by soft count = 227\n",
      "--- gene=1123, id=249 removed by soft count = 116\n",
      "--- number of probes: 267 , kept by blast: 256 , if remove dups: 256\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1124\n",
      "--- gene=1124, id=0 removed by soft count = 906\n",
      "--- gene=1124, id=3 removed by soft count = 141\n",
      "--- gene=1124, id=8 removed by soft count = 68\n",
      "--- gene=1124, id=44 removed by soft count = 197\n",
      "--- number of probes: 201 , kept by blast: 197 , if remove dups: 197\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1125\n",
      "--- gene=1125, id=0 removed by soft count = 62\n",
      "--- gene=1125, id=71 removed by soft count = 217\n",
      "--- gene=1125, id=172 removed by soft count = 117\n",
      "--- gene=1125, id=173 removed by soft count = 55\n",
      "--- gene=1125, id=174 removed by soft count = 238\n",
      "--- gene=1125, id=175 removed by soft count = 2410\n",
      "--- gene=1125, id=176 removed by soft count = 14407\n",
      "--- gene=1125, id=179 removed by soft count = 75\n",
      "--- gene=1125, id=180 removed by soft count = 161\n",
      "--- gene=1125, id=181 removed by soft count = 355\n",
      "--- number of probes: 208 , kept by blast: 198 , if remove dups: 198\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1126\n",
      "--- gene=1126, id=18 removed by soft count = 3690\n",
      "--- gene=1126, id=93 removed by soft count = 242\n",
      "--- gene=1126, id=94 removed by soft count = 112\n",
      "--- gene=1126, id=95 removed by soft count = 69\n",
      "--- gene=1126, id=96 removed by soft count = 150\n",
      "--- gene=1126, id=155 removed by soft count = 388\n",
      "--- gene=1126, id=222 removed by soft count = 173\n",
      "--- number of probes: 300 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1127\n",
      "--- gene=1127, id=32 removed by soft count = 87\n",
      "--- gene=1127, id=130 removed by soft count = 657\n",
      "--- gene=1127, id=151 removed by soft count = 84\n",
      "--- number of probes: 241 , kept by blast: 238 , if remove dups: 238\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1128\n",
      "--- gene=1128, id=14 removed by soft count = 153\n",
      "--- gene=1128, id=17 removed by soft count = 231\n",
      "--- gene=1128, id=72 removed by soft count = 12576\n",
      "--- gene=1128, id=139 removed by soft count = 135\n",
      "--- gene=1128, id=171 removed by soft count = 62\n",
      "--- gene=1128, id=197 removed by soft count = 79\n",
      "--- gene=1128, id=219 removed by soft count = 1481\n",
      "--- number of probes: 235 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1129\n",
      "--- gene=1129, id=0 removed by soft count = 391\n",
      "--- gene=1129, id=66 removed by soft count = 113\n",
      "--- gene=1129, id=67 removed by soft count = 139\n",
      "--- gene=1129, id=70 removed by soft count = 88\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1130\n",
      "--- gene=1130, id=77 removed by soft count = 65\n",
      "--- gene=1130, id=110 removed by soft count = 116\n",
      "--- gene=1130, id=111 removed by soft count = 132\n",
      "--- gene=1130, id=146 removed by soft count = 1076\n",
      "--- number of probes: 298 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1131\n",
      "--- gene=1131, id=170 removed by soft count = 68\n",
      "--- gene=1131, id=171 removed by soft count = 83\n",
      "--- number of probes: 272 , kept by blast: 270 , if remove dups: 269\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1132\n",
      "--- gene=1132, id=38 removed by soft count = 56\n",
      "--- gene=1132, id=39 removed by soft count = 61\n",
      "--- gene=1132, id=40 removed by soft count = 57\n",
      "--- gene=1132, id=94 removed by soft count = 88\n",
      "--- gene=1132, id=178 removed by soft count = 180\n",
      "--- number of probes: 288 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1133\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1133, id=41 removed by soft count = 71\n",
      "--- gene=1133, id=42 removed by soft count = 998\n",
      "--- gene=1133, id=55 removed by soft count = 153\n",
      "--- gene=1133, id=69 removed by soft count = 3373\n",
      "--- gene=1133, id=110 removed by soft count = 76\n",
      "--- gene=1133, id=116 removed by soft count = 175\n",
      "--- gene=1133, id=118 removed by soft count = 179\n",
      "--- gene=1133, id=124 removed by soft count = 61\n",
      "--- gene=1133, id=130 removed by soft count = 2806\n",
      "--- gene=1133, id=157 removed by soft count = 77\n",
      "--- gene=1133, id=174 removed by soft count = 78\n",
      "--- gene=1133, id=189 removed by soft count = 64\n",
      "--- gene=1133, id=190 removed by soft count = 117\n",
      "--- gene=1133, id=198 removed by soft count = 216\n",
      "--- number of probes: 205 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1134\n",
      "--- gene=1134, id=12 removed by soft count = 1513\n",
      "--- gene=1134, id=170 removed by hard count = 3\n",
      "--- gene=1134, id=193 removed by soft count = 332\n",
      "--- gene=1134, id=205 removed by soft count = 83\n",
      "--- gene=1134, id=206 removed by soft count = 120\n",
      "--- gene=1134, id=218 removed by soft count = 2950\n",
      "--- gene=1134, id=269 removed by soft count = 105\n",
      "--- gene=1134, id=275 removed by soft count = 83\n",
      "--- number of probes: 278 , kept by blast: 270 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1135\n",
      "--- gene=1135, id=8 removed by soft count = 321\n",
      "--- gene=1135, id=37 removed by soft count = 69\n",
      "--- gene=1135, id=85 removed by soft count = 42294\n",
      "--- number of probes: 268 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1136\n",
      "--- number of probes: 290 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1137\n",
      "--- gene=1137, id=46 removed by hard count = 4\n",
      "--- gene=1137, id=47 removed by soft count = 62\n",
      "--- gene=1137, id=70 removed by soft count = 65\n",
      "--- gene=1137, id=75 removed by soft count = 56\n",
      "--- gene=1137, id=77 removed by soft count = 79\n",
      "--- gene=1137, id=81 removed by soft count = 340\n",
      "--- gene=1137, id=82 removed by soft count = 56\n",
      "--- gene=1137, id=96 removed by soft count = 293\n",
      "--- gene=1137, id=185 removed by soft count = 99\n",
      "--- gene=1137, id=187 removed by soft count = 4550\n",
      "--- number of probes: 193 , kept by blast: 183 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1138\n",
      "--- gene=1138, id=0 removed by soft count = 140\n",
      "--- gene=1138, id=35 removed by soft count = 698\n",
      "--- gene=1138, id=36 removed by soft count = 298\n",
      "--- gene=1138, id=37 removed by soft count = 97\n",
      "--- gene=1138, id=44 removed by soft count = 452\n",
      "--- gene=1138, id=45 removed by soft count = 696\n",
      "--- gene=1138, id=46 removed by soft count = 229\n",
      "--- gene=1138, id=63 removed by soft count = 84\n",
      "--- gene=1138, id=64 removed by soft count = 75\n",
      "--- gene=1138, id=147 removed by hard count = 3\n",
      "--- number of probes: 212 , kept by blast: 202 , if remove dups: 202\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1139\n",
      "--- gene=1139, id=77 removed by soft count = 238\n",
      "--- gene=1139, id=128 removed by soft count = 75\n",
      "--- gene=1139, id=209 removed by soft count = 74\n",
      "--- number of probes: 256 , kept by blast: 253 , if remove dups: 253\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1140\n",
      "--- number of probes: 242 , kept by blast: 242 , if remove dups: 242\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1141\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1142\n",
      "--- gene=1142, id=54 removed by soft count = 279\n",
      "--- gene=1142, id=55 removed by soft count = 176\n",
      "--- gene=1142, id=106 removed by soft count = 64\n",
      "--- gene=1142, id=218 removed by soft count = 338\n",
      "--- number of probes: 263 , kept by blast: 259 , if remove dups: 259\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1143\n",
      "--- gene=1143, id=16 removed by soft count = 184\n",
      "--- gene=1143, id=88 removed by soft count = 128\n",
      "--- gene=1143, id=115 removed by soft count = 75\n",
      "--- gene=1143, id=166 removed by soft count = 102\n",
      "--- gene=1143, id=176 removed by soft count = 60\n",
      "--- gene=1143, id=194 removed by soft count = 113\n",
      "--- gene=1143, id=196 removed by soft count = 1375\n",
      "--- gene=1143, id=197 removed by soft count = 91\n",
      "--- gene=1143, id=198 removed by soft count = 155\n",
      "--- number of probes: 206 , kept by blast: 197 , if remove dups: 196\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1144\n",
      "--- gene=1144, id=88 removed by soft count = 236\n",
      "--- gene=1144, id=89 removed by soft count = 178\n",
      "--- number of probes: 122 , kept by blast: 120 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 117\n",
      "-- checking probes in region: 1145\n",
      "--- gene=1145, id=135 removed by soft count = 85\n",
      "--- number of probes: 215 , kept by blast: 214 , if remove dups: 213\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1146\n",
      "--- gene=1146, id=39 removed by soft count = 171\n",
      "--- number of probes: 94 , kept by blast: 93 , if remove dups: 92\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 92\n",
      "-- checking probes in region: 1147\n",
      "--- gene=1147, id=144 removed by soft count = 51\n",
      "--- gene=1147, id=147 removed by soft count = 247\n",
      "--- gene=1147, id=212 removed by soft count = 1134\n",
      "--- gene=1147, id=254 removed by soft count = 164\n",
      "--- gene=1147, id=255 removed by soft count = 165\n",
      "--- number of probes: 257 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1148\n",
      "--- gene=1148, id=4 removed by hard count = 3\n",
      "--- number of probes: 101 , kept by blast: 100 , if remove dups: 98\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 98\n",
      "-- checking probes in region: 1152\n",
      "--- gene=1152, id=0 removed by soft count = 112\n",
      "--- gene=1152, id=3 removed by soft count = 304\n",
      "--- gene=1152, id=5 removed by soft count = 353\n",
      "--- gene=1152, id=6 removed by soft count = 490\n",
      "--- gene=1152, id=9 removed by soft count = 507\n",
      "--- number of probes: 194 , kept by blast: 189 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1153\n",
      "--- gene=1153, id=72 removed by soft count = 76\n",
      "--- gene=1153, id=145 removed by soft count = 407\n",
      "--- number of probes: 269 , kept by blast: 267 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1154\n",
      "--- gene=1154, id=30 removed by soft count = 160\n",
      "--- gene=1154, id=31 removed by soft count = 160\n",
      "--- number of probes: 54 , kept by blast: 52 , if remove dups: 51\n",
      "-- number of probes kept for this region: 52\n",
      "-- checking probes in region: 1155\n",
      "--- gene=1155, id=17 removed by soft count = 234\n",
      "--- gene=1155, id=79 removed by soft count = 85\n",
      "--- number of probes: 254 , kept by blast: 252 , if remove dups: 252\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1156\n",
      "--- gene=1156, id=179 removed by soft count = 261\n",
      "--- gene=1156, id=180 removed by soft count = 91\n",
      "--- gene=1156, id=181 removed by soft count = 307\n",
      "--- gene=1156, id=216 removed by soft count = 130\n",
      "--- gene=1156, id=234 removed by soft count = 117\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1156, id=241 removed by soft count = 211\n",
      "--- gene=1156, id=296 removed by soft count = 81\n",
      "--- gene=1156, id=297 removed by soft count = 99\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1157\n",
      "--- gene=1157, id=127 removed by soft count = 73\n",
      "--- gene=1157, id=157 removed by soft count = 112\n",
      "--- gene=1157, id=185 removed by soft count = 217\n",
      "--- gene=1157, id=219 removed by soft count = 2586\n",
      "--- number of probes: 276 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1160\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1162\n",
      "--- gene=1162, id=4 removed by soft count = 108\n",
      "--- gene=1162, id=12 removed by soft count = 281\n",
      "--- gene=1162, id=155 removed by soft count = 99\n",
      "--- number of probes: 222 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1163\n",
      "--- number of probes: 272 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1165\n",
      "--- gene=1165, id=1 removed by soft count = 727\n",
      "--- gene=1165, id=16 removed by soft count = 129\n",
      "--- gene=1165, id=20 removed by soft count = 151\n",
      "--- gene=1165, id=21 removed by soft count = 110\n",
      "--- gene=1165, id=117 removed by soft count = 223\n",
      "--- gene=1165, id=118 removed by soft count = 96\n",
      "--- gene=1165, id=131 removed by soft count = 61\n",
      "--- gene=1165, id=263 removed by soft count = 58\n",
      "--- number of probes: 270 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1166\n",
      "--- gene=1166, id=27 removed by soft count = 823\n",
      "--- gene=1166, id=108 removed by soft count = 91\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1167\n",
      "--- gene=1167, id=0 removed by soft count = 71\n",
      "--- gene=1167, id=1 removed by soft count = 57\n",
      "--- gene=1167, id=12 removed by soft count = 460\n",
      "--- gene=1167, id=13 removed by soft count = 125\n",
      "--- gene=1167, id=14 removed by soft count = 57\n",
      "--- gene=1167, id=15 removed by soft count = 74\n",
      "--- gene=1167, id=17 removed by soft count = 159\n",
      "--- gene=1167, id=23 removed by soft count = 56\n",
      "--- gene=1167, id=31 removed by soft count = 880\n",
      "--- gene=1167, id=40 removed by soft count = 1849\n",
      "--- gene=1167, id=53 removed by soft count = 67\n",
      "--- gene=1167, id=64 removed by soft count = 347\n",
      "--- gene=1167, id=65 removed by soft count = 1248\n",
      "--- gene=1167, id=66 removed by soft count = 193\n",
      "--- gene=1167, id=67 removed by soft count = 220\n",
      "--- gene=1167, id=71 removed by soft count = 181\n",
      "--- gene=1167, id=72 removed by soft count = 66\n",
      "--- number of probes: 192 , kept by blast: 175 , if remove dups: 175\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1168\n",
      "--- gene=1168, id=5 removed by soft count = 466\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1169\n",
      "--- gene=1169, id=5 removed by soft count = 60\n",
      "--- gene=1169, id=68 removed by soft count = 76\n",
      "--- gene=1169, id=123 removed by soft count = 61\n",
      "--- gene=1169, id=140 removed by soft count = 233\n",
      "--- gene=1169, id=142 removed by soft count = 61\n",
      "--- gene=1169, id=145 removed by soft count = 263\n",
      "--- gene=1169, id=148 removed by soft count = 253\n",
      "--- gene=1169, id=150 removed by soft count = 321\n",
      "--- gene=1169, id=152 removed by soft count = 239\n",
      "--- number of probes: 185 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1170\n",
      "--- gene=1170, id=0 removed by soft count = 61\n",
      "--- gene=1170, id=114 removed by soft count = 643\n",
      "--- gene=1170, id=118 removed by soft count = 105\n",
      "--- gene=1170, id=119 removed by soft count = 129\n",
      "--- gene=1170, id=121 removed by soft count = 365\n",
      "--- gene=1170, id=123 removed by soft count = 68\n",
      "--- number of probes: 232 , kept by blast: 226 , if remove dups: 226\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1171\n",
      "--- gene=1171, id=0 removed by soft count = 8357\n",
      "--- gene=1171, id=1 removed by soft count = 7962\n",
      "--- gene=1171, id=2 removed by soft count = 210\n",
      "--- gene=1171, id=3 removed by soft count = 52\n",
      "--- gene=1171, id=4 removed by soft count = 928\n",
      "--- gene=1171, id=5 removed by soft count = 447\n",
      "--- gene=1171, id=28 removed by soft count = 121\n",
      "--- gene=1171, id=30 removed by soft count = 107\n",
      "--- gene=1171, id=32 removed by soft count = 176\n",
      "--- gene=1171, id=129 removed by soft count = 4150\n",
      "--- gene=1171, id=227 removed by soft count = 166\n",
      "--- number of probes: 268 , kept by blast: 257 , if remove dups: 257\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1172\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1173\n",
      "--- gene=1173, id=18 removed by soft count = 278\n",
      "--- gene=1173, id=24 removed by soft count = 156\n",
      "--- gene=1173, id=25 removed by soft count = 64\n",
      "--- gene=1173, id=26 removed by soft count = 167\n",
      "--- gene=1173, id=32 removed by soft count = 334\n",
      "--- gene=1173, id=87 removed by soft count = 139\n",
      "--- number of probes: 96 , kept by blast: 90 , if remove dups: 90\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 90\n",
      "-- checking probes in region: 1174\n",
      "--- gene=1174, id=6 removed by soft count = 89\n",
      "--- gene=1174, id=11 removed by soft count = 120\n",
      "--- gene=1174, id=15 removed by soft count = 75\n",
      "--- gene=1174, id=103 removed by soft count = 4387\n",
      "--- gene=1174, id=115 removed by soft count = 144\n",
      "--- gene=1174, id=117 removed by soft count = 3714\n",
      "--- gene=1174, id=118 removed by soft count = 184\n",
      "--- gene=1174, id=121 removed by soft count = 73\n",
      "--- gene=1174, id=122 removed by soft count = 383\n",
      "--- gene=1174, id=123 removed by soft count = 322\n",
      "--- gene=1174, id=124 removed by soft count = 58\n",
      "--- gene=1174, id=125 removed by soft count = 135\n",
      "--- gene=1174, id=127 removed by soft count = 82\n",
      "--- number of probes: 128 , kept by blast: 115 , if remove dups: 115\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 115\n",
      "-- checking probes in region: 1176\n",
      "--- gene=1176, id=159 removed by soft count = 56\n",
      "--- gene=1176, id=163 removed by soft count = 88\n",
      "--- number of probes: 212 , kept by blast: 210 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1177\n",
      "--- gene=1177, id=224 removed by soft count = 64\n",
      "--- gene=1177, id=225 removed by soft count = 540\n",
      "--- gene=1177, id=226 removed by soft count = 1267\n",
      "--- gene=1177, id=228 removed by soft count = 116\n",
      "--- number of probes: 241 , kept by blast: 237 , if remove dups: 237\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1178\n",
      "--- gene=1178, id=160 removed by soft count = 95\n",
      "--- gene=1178, id=173 removed by soft count = 122\n",
      "--- gene=1178, id=230 removed by soft count = 55\n",
      "--- gene=1178, id=232 removed by soft count = 202\n",
      "--- gene=1178, id=244 removed by soft count = 59\n",
      "--- gene=1178, id=245 removed by soft count = 55\n",
      "--- gene=1178, id=246 removed by soft count = 79\n",
      "--- gene=1178, id=248 removed by soft count = 58\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- number of probes: 298 , kept by blast: 290 , if remove dups: 289\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1179\n",
      "--- gene=1179, id=8 removed by soft count = 3994\n",
      "--- gene=1179, id=40 removed by soft count = 770\n",
      "--- gene=1179, id=128 removed by soft count = 88\n",
      "--- gene=1179, id=130 removed by soft count = 81\n",
      "--- number of probes: 197 , kept by blast: 193 , if remove dups: 192\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1180\n",
      "--- gene=1180, id=6 removed by soft count = 419\n",
      "--- gene=1180, id=7 removed by soft count = 600\n",
      "--- gene=1180, id=8 removed by soft count = 255\n",
      "--- gene=1180, id=9 removed by soft count = 366\n",
      "--- gene=1180, id=10 removed by soft count = 368\n",
      "--- gene=1180, id=11 removed by soft count = 149\n",
      "--- gene=1180, id=62 removed by soft count = 61\n",
      "--- gene=1180, id=79 removed by soft count = 188\n",
      "--- number of probes: 213 , kept by blast: 205 , if remove dups: 205\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1181\n",
      "--- number of probes: 271 , kept by blast: 271 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1182\n",
      "--- gene=1182, id=58 removed by soft count = 55\n",
      "--- gene=1182, id=63 removed by soft count = 112\n",
      "--- gene=1182, id=101 removed by soft count = 86\n",
      "--- gene=1182, id=171 removed by soft count = 98\n",
      "--- gene=1182, id=183 removed by soft count = 403\n",
      "--- gene=1182, id=257 removed by soft count = 597\n",
      "--- number of probes: 291 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1183\n",
      "--- gene=1183, id=84 removed by soft count = 105\n",
      "--- gene=1183, id=124 removed by soft count = 82\n",
      "--- gene=1183, id=134 removed by soft count = 139\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1184\n",
      "--- gene=1184, id=39 removed by soft count = 67\n",
      "--- gene=1184, id=101 removed by soft count = 319\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1186\n",
      "--- gene=1186, id=133 removed by soft count = 55\n",
      "--- gene=1186, id=141 removed by soft count = 142\n",
      "--- gene=1186, id=143 removed by soft count = 116\n",
      "--- gene=1186, id=166 removed by soft count = 129\n",
      "--- gene=1186, id=184 removed by soft count = 39134\n",
      "--- gene=1186, id=258 removed by soft count = 64\n",
      "--- number of probes: 265 , kept by blast: 259 , if remove dups: 259\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1187\n",
      "--- gene=1187, id=35 removed by soft count = 270\n",
      "--- gene=1187, id=44 removed by soft count = 56\n",
      "--- gene=1187, id=48 removed by soft count = 184\n",
      "--- gene=1187, id=208 removed by soft count = 87\n",
      "--- gene=1187, id=217 removed by soft count = 67\n",
      "--- number of probes: 285 , kept by blast: 280 , if remove dups: 279\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1188\n",
      "--- number of probes: 77 , kept by blast: 77 , if remove dups: 77\n",
      "-- number of probes kept for this region: 77\n",
      "-- checking probes in region: 1189\n",
      "--- number of probes: 171 , kept by blast: 171 , if remove dups: 171\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1190\n",
      "--- gene=1190, id=28 removed by soft count = 88\n",
      "--- gene=1190, id=84 removed by soft count = 139\n",
      "--- gene=1190, id=93 removed by soft count = 347\n",
      "--- gene=1190, id=94 removed by soft count = 460\n",
      "--- gene=1190, id=96 removed by soft count = 87\n",
      "--- gene=1190, id=97 removed by soft count = 897\n",
      "--- gene=1190, id=98 removed by soft count = 166\n",
      "--- gene=1190, id=99 removed by soft count = 130\n",
      "--- gene=1190, id=101 removed by soft count = 528\n",
      "--- gene=1190, id=102 removed by soft count = 563\n",
      "--- gene=1190, id=103 removed by soft count = 265\n",
      "--- number of probes: 187 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1191\n",
      "--- gene=1191, id=39 removed by soft count = 267\n",
      "--- gene=1191, id=92 removed by soft count = 633\n",
      "--- gene=1191, id=106 removed by soft count = 85\n",
      "--- gene=1191, id=107 removed by soft count = 90\n",
      "--- gene=1191, id=161 removed by soft count = 130\n",
      "--- gene=1191, id=162 removed by soft count = 61\n",
      "--- gene=1191, id=177 removed by soft count = 181\n",
      "--- gene=1191, id=178 removed by soft count = 1557\n",
      "--- number of probes: 194 , kept by blast: 186 , if remove dups: 186\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1192\n",
      "--- gene=1192, id=68 removed by soft count = 101\n",
      "--- gene=1192, id=85 removed by soft count = 52\n",
      "--- gene=1192, id=115 removed by soft count = 215\n",
      "--- gene=1192, id=116 removed by soft count = 58\n",
      "--- gene=1192, id=131 removed by soft count = 62\n",
      "--- gene=1192, id=162 removed by soft count = 88\n",
      "--- gene=1192, id=164 removed by soft count = 57\n",
      "--- gene=1192, id=183 removed by soft count = 78\n",
      "--- gene=1192, id=186 removed by soft count = 205\n",
      "--- gene=1192, id=212 removed by soft count = 68\n",
      "--- gene=1192, id=215 removed by soft count = 54\n",
      "--- gene=1192, id=217 removed by soft count = 254\n",
      "--- gene=1192, id=218 removed by soft count = 254\n",
      "--- gene=1192, id=219 removed by soft count = 116\n",
      "--- gene=1192, id=220 removed by soft count = 330\n",
      "--- gene=1192, id=221 removed by soft count = 436\n",
      "--- gene=1192, id=225 removed by soft count = 60\n",
      "--- gene=1192, id=227 removed by soft count = 88\n",
      "--- gene=1192, id=237 removed by soft count = 85\n",
      "--- gene=1192, id=238 removed by soft count = 69\n",
      "--- gene=1192, id=239 removed by soft count = 282\n",
      "--- gene=1192, id=240 removed by soft count = 560\n",
      "--- number of probes: 242 , kept by blast: 220 , if remove dups: 220\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1193\n",
      "--- gene=1193, id=85 removed by soft count = 63\n",
      "--- gene=1193, id=196 removed by soft count = 293\n",
      "--- gene=1193, id=270 removed by soft count = 123\n",
      "--- number of probes: 279 , kept by blast: 276 , if remove dups: 276\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1194\n",
      "--- gene=1194, id=0 removed by soft count = 60\n",
      "--- gene=1194, id=1 removed by soft count = 195\n",
      "--- gene=1194, id=2 removed by soft count = 78\n",
      "--- gene=1194, id=4 removed by soft count = 195\n",
      "--- gene=1194, id=8 removed by soft count = 53\n",
      "--- gene=1194, id=11 removed by soft count = 77\n",
      "--- gene=1194, id=14 removed by soft count = 55\n",
      "--- gene=1194, id=27 removed by soft count = 238\n",
      "--- gene=1194, id=238 removed by soft count = 2972\n",
      "--- number of probes: 291 , kept by blast: 282 , if remove dups: 282\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1195\n",
      "--- gene=1195, id=31 removed by soft count = 77\n",
      "--- gene=1195, id=33 removed by soft count = 346\n",
      "--- gene=1195, id=93 removed by soft count = 84\n",
      "--- gene=1195, id=95 removed by soft count = 93\n",
      "--- gene=1195, id=96 removed by soft count = 100\n",
      "--- gene=1195, id=97 removed by soft count = 130\n",
      "--- gene=1195, id=210 removed by soft count = 122\n",
      "--- number of probes: 288 , kept by blast: 281 , if remove dups: 281\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1196\n",
      "--- gene=1196, id=28 removed by soft count = 101\n",
      "--- gene=1196, id=29 removed by soft count = 133\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1196, id=71 removed by soft count = 65\n",
      "--- gene=1196, id=72 removed by soft count = 69\n",
      "--- gene=1196, id=103 removed by hard count = 0\n",
      "--- gene=1196, id=111 removed by soft count = 368\n",
      "--- gene=1196, id=274 removed by soft count = 76\n",
      "--- gene=1196, id=276 removed by soft count = 211\n",
      "--- number of probes: 277 , kept by blast: 269 , if remove dups: 268\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1197\n",
      "--- gene=1197, id=2 removed by soft count = 282\n",
      "--- gene=1197, id=52 removed by soft count = 103\n",
      "--- gene=1197, id=94 removed by soft count = 133\n",
      "--- gene=1197, id=148 removed by soft count = 5143\n",
      "--- gene=1197, id=150 removed by soft count = 507\n",
      "--- gene=1197, id=151 removed by soft count = 413\n",
      "--- gene=1197, id=152 removed by soft count = 72\n",
      "--- gene=1197, id=153 removed by soft count = 527\n",
      "--- gene=1197, id=154 removed by soft count = 416\n",
      "--- gene=1197, id=155 removed by soft count = 93\n",
      "--- gene=1197, id=156 removed by soft count = 374\n",
      "--- gene=1197, id=174 removed by soft count = 210\n",
      "--- gene=1197, id=177 removed by soft count = 251\n",
      "--- number of probes: 184 , kept by blast: 171 , if remove dups: 171\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1203\n",
      "--- gene=1203, id=0 removed by soft count = 58\n",
      "--- gene=1203, id=1 removed by soft count = 127\n",
      "--- gene=1203, id=39 removed by soft count = 69\n",
      "--- number of probes: 191 , kept by blast: 188 , if remove dups: 188\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1204\n",
      "--- gene=1204, id=185 removed by soft count = 266\n",
      "--- gene=1204, id=262 removed by soft count = 53\n",
      "--- gene=1204, id=299 removed by soft count = 120\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1205\n",
      "--- gene=1205, id=9 removed by hard count = 0\n",
      "--- gene=1205, id=125 removed by soft count = 158\n",
      "--- gene=1205, id=190 removed by soft count = 234\n",
      "--- gene=1205, id=191 removed by soft count = 94\n",
      "--- gene=1205, id=253 removed by soft count = 270\n",
      "--- gene=1205, id=256 removed by soft count = 54\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1206\n",
      "--- gene=1206, id=0 removed by soft count = 188\n",
      "--- gene=1206, id=1 removed by soft count = 276\n",
      "--- gene=1206, id=2 removed by soft count = 204\n",
      "--- gene=1206, id=3 removed by soft count = 316\n",
      "--- gene=1206, id=4 removed by soft count = 191\n",
      "--- gene=1206, id=5 removed by soft count = 283\n",
      "--- gene=1206, id=6 removed by soft count = 349\n",
      "--- gene=1206, id=8 removed by soft count = 268\n",
      "--- gene=1206, id=9 removed by soft count = 64\n",
      "--- gene=1206, id=10 removed by soft count = 172\n",
      "--- gene=1206, id=11 removed by soft count = 4481\n",
      "--- gene=1206, id=12 removed by soft count = 281\n",
      "--- gene=1206, id=13 removed by soft count = 306\n",
      "--- gene=1206, id=14 removed by soft count = 196\n",
      "--- gene=1206, id=16 removed by soft count = 275\n",
      "--- gene=1206, id=26 removed by soft count = 103\n",
      "--- gene=1206, id=31 removed by hard count = 3\n",
      "--- gene=1206, id=51 removed by soft count = 268\n",
      "--- gene=1206, id=56 removed by soft count = 121\n",
      "--- gene=1206, id=96 removed by soft count = 1537\n",
      "--- gene=1206, id=100 removed by soft count = 93\n",
      "--- gene=1206, id=102 removed by soft count = 161\n",
      "--- number of probes: 108 , kept by blast: 86 , if remove dups: 86\n",
      "-- number of probes kept for this region: 86\n",
      "-- checking probes in region: 1207\n",
      "--- gene=1207, id=18 removed by soft count = 106\n",
      "--- gene=1207, id=26 removed by soft count = 59\n",
      "--- gene=1207, id=32 removed by soft count = 2179\n",
      "--- gene=1207, id=33 removed by soft count = 1001\n",
      "--- gene=1207, id=177 removed by soft count = 52\n",
      "--- number of probes: 213 , kept by blast: 208 , if remove dups: 208\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1208\n",
      "--- gene=1208, id=5 removed by soft count = 146\n",
      "--- gene=1208, id=9 removed by soft count = 134\n",
      "--- gene=1208, id=26 removed by soft count = 2753\n",
      "--- gene=1208, id=27 removed by soft count = 59\n",
      "--- gene=1208, id=32 removed by soft count = 826\n",
      "--- gene=1208, id=55 removed by soft count = 88\n",
      "--- gene=1208, id=122 removed by soft count = 1488\n",
      "--- gene=1208, id=202 removed by soft count = 847\n",
      "--- number of probes: 208 , kept by blast: 200 , if remove dups: 200\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1209\n",
      "--- gene=1209, id=32 removed by soft count = 330\n",
      "--- gene=1209, id=34 removed by soft count = 60\n",
      "--- gene=1209, id=72 removed by soft count = 385\n",
      "--- gene=1209, id=114 removed by soft count = 243\n",
      "--- gene=1209, id=116 removed by soft count = 2926\n",
      "--- gene=1209, id=118 removed by soft count = 165\n",
      "--- gene=1209, id=121 removed by soft count = 115\n",
      "--- gene=1209, id=161 removed by soft count = 60\n",
      "--- number of probes: 169 , kept by blast: 161 , if remove dups: 161\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1210\n",
      "--- gene=1210, id=34 removed by soft count = 56\n",
      "--- gene=1210, id=115 removed by soft count = 70\n",
      "--- gene=1210, id=153 removed by soft count = 2309\n",
      "--- number of probes: 270 , kept by blast: 267 , if remove dups: 267\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1211\n",
      "--- gene=1211, id=25 removed by soft count = 271\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1212\n",
      "--- gene=1212, id=123 removed by soft count = 297\n",
      "--- gene=1212, id=172 removed by soft count = 82\n",
      "--- number of probes: 278 , kept by blast: 276 , if remove dups: 276\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1213\n",
      "--- gene=1213, id=15 removed by soft count = 107\n",
      "--- gene=1213, id=18 removed by soft count = 347\n",
      "--- gene=1213, id=42 removed by soft count = 220\n",
      "--- gene=1213, id=148 removed by soft count = 113\n",
      "--- gene=1213, id=150 removed by soft count = 254\n",
      "--- gene=1213, id=153 removed by soft count = 114\n",
      "--- gene=1213, id=167 removed by soft count = 130\n",
      "--- number of probes: 293 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1214\n",
      "--- gene=1214, id=135 removed by soft count = 742\n",
      "--- gene=1214, id=137 removed by soft count = 566\n",
      "--- gene=1214, id=269 removed by soft count = 71\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1215\n",
      "--- gene=1215, id=16 removed by soft count = 1095\n",
      "--- gene=1215, id=118 removed by soft count = 79\n",
      "--- gene=1215, id=119 removed by soft count = 237\n",
      "--- gene=1215, id=128 removed by soft count = 107\n",
      "--- gene=1215, id=131 removed by soft count = 159\n",
      "--- gene=1215, id=161 removed by soft count = 141\n",
      "--- gene=1215, id=173 removed by soft count = 577\n",
      "--- gene=1215, id=174 removed by soft count = 131\n",
      "--- gene=1215, id=175 removed by soft count = 399\n",
      "--- gene=1215, id=176 removed by soft count = 1636\n",
      "--- gene=1215, id=177 removed by soft count = 396\n",
      "--- gene=1215, id=178 removed by soft count = 1550\n",
      "--- gene=1215, id=182 removed by soft count = 86\n",
      "--- gene=1215, id=218 removed by soft count = 218\n",
      "--- number of probes: 221 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1216\n",
      "--- gene=1216, id=18 removed by soft count = 327\n",
      "--- gene=1216, id=23 removed by soft count = 184\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1216, id=89 removed by soft count = 64\n",
      "--- number of probes: 216 , kept by blast: 213 , if remove dups: 213\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1217\n",
      "--- gene=1217, id=21 removed by soft count = 204\n",
      "--- gene=1217, id=22 removed by soft count = 408\n",
      "--- gene=1217, id=23 removed by soft count = 805\n",
      "--- gene=1217, id=166 removed by soft count = 159\n",
      "--- gene=1217, id=178 removed by soft count = 68\n",
      "--- number of probes: 190 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1218\n",
      "--- gene=1218, id=4 removed by soft count = 73\n",
      "--- gene=1218, id=5 removed by soft count = 51\n",
      "--- gene=1218, id=17 removed by soft count = 78\n",
      "--- gene=1218, id=144 removed by soft count = 1430\n",
      "--- gene=1218, id=145 removed by soft count = 225\n",
      "--- gene=1218, id=232 removed by soft count = 84\n",
      "--- number of probes: 235 , kept by blast: 229 , if remove dups: 229\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1219\n",
      "--- gene=1219, id=66 removed by soft count = 149\n",
      "--- gene=1219, id=232 removed by soft count = 223\n",
      "--- gene=1219, id=233 removed by soft count = 57\n",
      "--- gene=1219, id=234 removed by soft count = 116\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1220\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1221\n",
      "--- gene=1221, id=185 removed by soft count = 107\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1222\n",
      "--- gene=1222, id=7 removed by soft count = 62\n",
      "--- gene=1222, id=33 removed by soft count = 256\n",
      "--- gene=1222, id=34 removed by soft count = 474\n",
      "--- gene=1222, id=35 removed by soft count = 213\n",
      "--- gene=1222, id=36 removed by soft count = 52\n",
      "--- gene=1222, id=63 removed by soft count = 197\n",
      "--- gene=1222, id=68 removed by soft count = 124\n",
      "--- gene=1222, id=74 removed by soft count = 71\n",
      "--- gene=1222, id=105 removed by soft count = 168\n",
      "--- gene=1222, id=107 removed by soft count = 126\n",
      "--- gene=1222, id=110 removed by soft count = 71\n",
      "--- number of probes: 179 , kept by blast: 168 , if remove dups: 168\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1224\n",
      "--- gene=1224, id=205 removed by soft count = 452\n",
      "--- number of probes: 231 , kept by blast: 230 , if remove dups: 230\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1225\n",
      "--- gene=1225, id=83 removed by soft count = 57\n",
      "--- gene=1225, id=91 removed by soft count = 62\n",
      "--- gene=1225, id=177 removed by soft count = 852\n",
      "--- gene=1225, id=222 removed by soft count = 99\n",
      "--- gene=1225, id=247 removed by soft count = 160\n",
      "--- gene=1225, id=250 removed by soft count = 121\n",
      "--- number of probes: 282 , kept by blast: 276 , if remove dups: 275\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1226\n",
      "--- gene=1226, id=229 removed by soft count = 135\n",
      "--- gene=1226, id=231 removed by soft count = 93\n",
      "--- number of probes: 238 , kept by blast: 236 , if remove dups: 236\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1227\n",
      "--- gene=1227, id=2 removed by soft count = 100\n",
      "--- gene=1227, id=5 removed by soft count = 408\n",
      "--- gene=1227, id=6 removed by soft count = 207\n",
      "--- gene=1227, id=40 removed by soft count = 261\n",
      "--- gene=1227, id=167 removed by soft count = 143\n",
      "--- gene=1227, id=172 removed by soft count = 209\n",
      "--- number of probes: 215 , kept by blast: 209 , if remove dups: 209\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1228\n",
      "--- gene=1228, id=58 removed by soft count = 169\n",
      "--- gene=1228, id=64 removed by soft count = 60\n",
      "--- number of probes: 167 , kept by blast: 165 , if remove dups: 165\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1229\n",
      "--- gene=1229, id=29 removed by soft count = 134\n",
      "--- gene=1229, id=30 removed by soft count = 129\n",
      "--- gene=1229, id=34 removed by soft count = 189\n",
      "--- gene=1229, id=37 removed by soft count = 54\n",
      "--- gene=1229, id=40 removed by soft count = 91\n",
      "--- gene=1229, id=43 removed by soft count = 58\n",
      "--- gene=1229, id=78 removed by soft count = 63\n",
      "--- gene=1229, id=114 removed by soft count = 107\n",
      "--- gene=1229, id=146 removed by soft count = 371\n",
      "--- gene=1229, id=165 removed by soft count = 309\n",
      "--- gene=1229, id=208 removed by soft count = 1210\n",
      "--- number of probes: 235 , kept by blast: 224 , if remove dups: 224\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1230\n",
      "--- gene=1230, id=184 removed by soft count = 271\n",
      "--- gene=1230, id=185 removed by soft count = 1295\n",
      "--- gene=1230, id=187 removed by soft count = 279\n",
      "--- gene=1230, id=204 removed by soft count = 548\n",
      "--- number of probes: 223 , kept by blast: 219 , if remove dups: 219\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1231\n",
      "--- gene=1231, id=0 removed by soft count = 92\n",
      "--- gene=1231, id=32 removed by soft count = 222\n",
      "--- number of probes: 241 , kept by blast: 239 , if remove dups: 239\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1232\n",
      "--- gene=1232, id=127 removed by soft count = 597\n",
      "--- gene=1232, id=172 removed by soft count = 86\n",
      "--- gene=1232, id=189 removed by soft count = 81\n",
      "--- number of probes: 237 , kept by blast: 234 , if remove dups: 234\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1233\n",
      "--- gene=1233, id=88 removed by soft count = 153\n",
      "--- gene=1233, id=93 removed by soft count = 1344\n",
      "--- gene=1233, id=157 removed by soft count = 141\n",
      "--- gene=1233, id=197 removed by soft count = 58\n",
      "--- gene=1233, id=198 removed by soft count = 268\n",
      "--- number of probes: 223 , kept by blast: 218 , if remove dups: 218\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1234\n",
      "--- gene=1234, id=24 removed by soft count = 343\n",
      "--- gene=1234, id=46 removed by soft count = 100\n",
      "--- gene=1234, id=85 removed by soft count = 4133\n",
      "--- number of probes: 169 , kept by blast: 166 , if remove dups: 165\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1235\n",
      "--- gene=1235, id=118 removed by soft count = 253\n",
      "--- gene=1235, id=145 removed by soft count = 54\n",
      "--- gene=1235, id=206 removed by hard count = 3\n",
      "--- gene=1235, id=207 removed by soft count = 108\n",
      "--- gene=1235, id=214 removed by soft count = 305\n",
      "--- gene=1235, id=237 removed by soft count = 73\n",
      "--- number of probes: 241 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1236\n",
      "--- gene=1236, id=11 removed by soft count = 63\n",
      "--- gene=1236, id=24 removed by soft count = 58\n",
      "--- gene=1236, id=30 removed by soft count = 106\n",
      "--- gene=1236, id=245 removed by soft count = 146\n",
      "--- number of probes: 271 , kept by blast: 267 , if remove dups: 267\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1237\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1237, id=26 removed by soft count = 94\n",
      "--- number of probes: 228 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1238\n",
      "--- gene=1238, id=96 removed by soft count = 189\n",
      "--- gene=1238, id=98 removed by soft count = 248\n",
      "--- gene=1238, id=275 removed by soft count = 102\n",
      "--- number of probes: 287 , kept by blast: 284 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1239\n",
      "--- gene=1239, id=6 removed by soft count = 175\n",
      "--- gene=1239, id=12 removed by soft count = 99\n",
      "--- gene=1239, id=43 removed by soft count = 81\n",
      "--- gene=1239, id=142 removed by soft count = 92\n",
      "--- gene=1239, id=145 removed by soft count = 123\n",
      "--- number of probes: 249 , kept by blast: 244 , if remove dups: 244\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1240\n",
      "--- gene=1240, id=37 removed by soft count = 194\n",
      "--- gene=1240, id=38 removed by soft count = 343\n",
      "--- gene=1240, id=120 removed by soft count = 231\n",
      "--- gene=1240, id=255 removed by soft count = 88\n",
      "--- gene=1240, id=257 removed by soft count = 85\n",
      "--- gene=1240, id=266 removed by soft count = 53\n",
      "--- gene=1240, id=269 removed by soft count = 150\n",
      "--- number of probes: 297 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1241\n",
      "--- gene=1241, id=89 removed by soft count = 54\n",
      "--- gene=1241, id=241 removed by soft count = 84\n",
      "--- gene=1241, id=259 removed by soft count = 828\n",
      "--- gene=1241, id=281 removed by soft count = 80\n",
      "--- number of probes: 286 , kept by blast: 282 , if remove dups: 282\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1242\n",
      "--- gene=1242, id=29 removed by soft count = 620\n",
      "--- gene=1242, id=106 removed by hard count = 0\n",
      "--- gene=1242, id=155 removed by soft count = 73\n",
      "--- gene=1242, id=156 removed by soft count = 67\n",
      "--- gene=1242, id=169 removed by soft count = 121\n",
      "--- gene=1242, id=172 removed by soft count = 215\n",
      "--- gene=1242, id=173 removed by soft count = 212\n",
      "--- gene=1242, id=217 removed by soft count = 108\n",
      "--- number of probes: 279 , kept by blast: 271 , if remove dups: 269\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1243\n",
      "--- gene=1243, id=161 removed by soft count = 514\n",
      "--- number of probes: 297 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1244\n",
      "--- gene=1244, id=196 removed by soft count = 72\n",
      "--- number of probes: 260 , kept by blast: 259 , if remove dups: 259\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1245\n",
      "--- gene=1245, id=11 removed by soft count = 197\n",
      "--- gene=1245, id=17 removed by soft count = 77\n",
      "--- gene=1245, id=112 removed by soft count = 91\n",
      "--- gene=1245, id=150 removed by soft count = 52\n",
      "--- gene=1245, id=152 removed by soft count = 207\n",
      "--- gene=1245, id=155 removed by soft count = 172\n",
      "--- gene=1245, id=213 removed by soft count = 273\n",
      "--- gene=1245, id=277 removed by soft count = 145\n",
      "--- number of probes: 283 , kept by blast: 275 , if remove dups: 275\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1246\n",
      "--- gene=1246, id=11 removed by soft count = 328\n",
      "--- gene=1246, id=36 removed by soft count = 78\n",
      "--- gene=1246, id=52 removed by soft count = 52\n",
      "--- gene=1246, id=223 removed by soft count = 185\n",
      "--- gene=1246, id=275 removed by hard count = 3\n",
      "--- number of probes: 291 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1247\n",
      "--- gene=1247, id=145 removed by soft count = 55\n",
      "--- gene=1247, id=163 removed by soft count = 85\n",
      "--- gene=1247, id=192 removed by soft count = 152\n",
      "--- number of probes: 232 , kept by blast: 229 , if remove dups: 229\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1248\n",
      "--- gene=1248, id=0 removed by soft count = 223\n",
      "--- gene=1248, id=56 removed by soft count = 77\n",
      "--- gene=1248, id=120 removed by soft count = 1561\n",
      "--- gene=1248, id=125 removed by soft count = 83\n",
      "--- number of probes: 160 , kept by blast: 156 , if remove dups: 156\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1249\n",
      "--- gene=1249, id=50 removed by soft count = 51\n",
      "--- gene=1249, id=51 removed by soft count = 139\n",
      "--- gene=1249, id=52 removed by soft count = 125\n",
      "--- gene=1249, id=53 removed by soft count = 171\n",
      "--- gene=1249, id=114 removed by soft count = 110\n",
      "--- gene=1249, id=116 removed by soft count = 199\n",
      "--- number of probes: 237 , kept by blast: 231 , if remove dups: 231\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1250\n",
      "--- gene=1250, id=121 removed by soft count = 735\n",
      "--- gene=1250, id=148 removed by hard count = 3\n",
      "--- gene=1250, id=157 removed by hard count = 0\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1251\n",
      "--- gene=1251, id=21 removed by soft count = 200\n",
      "--- gene=1251, id=30 removed by soft count = 135\n",
      "--- gene=1251, id=31 removed by soft count = 639\n",
      "--- gene=1251, id=32 removed by soft count = 2208\n",
      "--- gene=1251, id=33 removed by soft count = 1622\n",
      "--- gene=1251, id=106 removed by soft count = 433\n",
      "--- gene=1251, id=111 removed by soft count = 292\n",
      "--- gene=1251, id=115 removed by soft count = 75\n",
      "--- gene=1251, id=116 removed by soft count = 2066\n",
      "--- number of probes: 181 , kept by blast: 172 , if remove dups: 172\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1252\n",
      "--- gene=1252, id=0 removed by soft count = 451\n",
      "--- gene=1252, id=55 removed by soft count = 122\n",
      "--- gene=1252, id=132 removed by soft count = 3285\n",
      "--- gene=1252, id=157 removed by soft count = 79\n",
      "--- number of probes: 181 , kept by blast: 177 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1253\n",
      "--- gene=1253, id=43 removed by soft count = 60\n",
      "--- gene=1253, id=126 removed by soft count = 1513\n",
      "--- number of probes: 246 , kept by blast: 244 , if remove dups: 244\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1254\n",
      "--- gene=1254, id=26 removed by soft count = 1118\n",
      "--- gene=1254, id=116 removed by soft count = 70\n",
      "--- gene=1254, id=117 removed by soft count = 82\n",
      "--- number of probes: 178 , kept by blast: 175 , if remove dups: 175\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1255\n",
      "--- gene=1255, id=48 removed by soft count = 90\n",
      "--- gene=1255, id=157 removed by soft count = 132\n",
      "--- gene=1255, id=160 removed by soft count = 673\n",
      "--- gene=1255, id=298 removed by soft count = 79\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1256\n",
      "--- gene=1256, id=12 removed by soft count = 21710\n",
      "--- gene=1256, id=175 removed by soft count = 265\n",
      "--- number of probes: 186 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1257\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1257, id=0 removed by soft count = 62\n",
      "--- gene=1257, id=6 removed by soft count = 53\n",
      "--- gene=1257, id=34 removed by soft count = 167\n",
      "--- number of probes: 173 , kept by blast: 170 , if remove dups: 170\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1258\n",
      "--- gene=1258, id=11 removed by soft count = 8477\n",
      "--- gene=1258, id=16 removed by soft count = 7234\n",
      "--- gene=1258, id=17 removed by soft count = 125\n",
      "--- gene=1258, id=64 removed by soft count = 73\n",
      "--- gene=1258, id=116 removed by soft count = 460\n",
      "--- gene=1258, id=136 removed by soft count = 54\n",
      "--- number of probes: 168 , kept by blast: 162 , if remove dups: 162\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1259\n",
      "--- gene=1259, id=60 removed by soft count = 76\n",
      "--- gene=1259, id=69 removed by soft count = 136\n",
      "--- gene=1259, id=214 removed by soft count = 274\n",
      "--- gene=1259, id=215 removed by soft count = 1289\n",
      "--- number of probes: 216 , kept by blast: 212 , if remove dups: 212\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1260\n",
      "--- gene=1260, id=102 removed by soft count = 146\n",
      "--- gene=1260, id=107 removed by soft count = 189\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1261\n",
      "--- gene=1261, id=37 removed by hard count = 0\n",
      "--- gene=1261, id=280 removed by soft count = 188\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1262\n",
      "--- gene=1262, id=2 removed by soft count = 950\n",
      "--- gene=1262, id=5 removed by soft count = 52\n",
      "--- gene=1262, id=15 removed by soft count = 310\n",
      "--- gene=1262, id=18 removed by soft count = 61\n",
      "--- gene=1262, id=23 removed by soft count = 57\n",
      "--- gene=1262, id=45 removed by soft count = 52\n",
      "--- gene=1262, id=48 removed by soft count = 61\n",
      "--- gene=1262, id=60 removed by soft count = 618\n",
      "--- gene=1262, id=61 removed by soft count = 356\n",
      "--- gene=1262, id=63 removed by soft count = 75\n",
      "--- gene=1262, id=64 removed by soft count = 129\n",
      "--- gene=1262, id=65 removed by soft count = 82\n",
      "--- gene=1262, id=66 removed by soft count = 95\n",
      "--- gene=1262, id=68 removed by soft count = 127\n",
      "--- gene=1262, id=70 removed by soft count = 88\n",
      "--- gene=1262, id=71 removed by soft count = 66\n",
      "--- gene=1262, id=72 removed by soft count = 91\n",
      "--- gene=1262, id=73 removed by soft count = 58\n",
      "--- gene=1262, id=74 removed by soft count = 61\n",
      "--- gene=1262, id=77 removed by soft count = 58\n",
      "--- gene=1262, id=80 removed by soft count = 68\n",
      "--- gene=1262, id=82 removed by soft count = 51\n",
      "--- number of probes: 85 , kept by blast: 63 , if remove dups: 63\n",
      "-- number of probes kept for this region: 63\n",
      "-- checking probes in region: 1263\n",
      "--- gene=1263, id=0 removed by soft count = 109\n",
      "--- gene=1263, id=8 removed by soft count = 82\n",
      "--- gene=1263, id=65 removed by soft count = 58\n",
      "--- gene=1263, id=101 removed by soft count = 584\n",
      "--- gene=1263, id=114 removed by soft count = 643\n",
      "--- gene=1263, id=117 removed by soft count = 478\n",
      "--- gene=1263, id=126 removed by soft count = 107\n",
      "--- gene=1263, id=131 removed by soft count = 64\n",
      "--- gene=1263, id=132 removed by soft count = 61\n",
      "--- gene=1263, id=144 removed by soft count = 61\n",
      "--- gene=1263, id=148 removed by soft count = 73\n",
      "--- gene=1263, id=171 removed by soft count = 58\n",
      "--- gene=1263, id=176 removed by soft count = 493\n",
      "--- number of probes: 189 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1264\n",
      "--- gene=1264, id=3 removed by soft count = 77\n",
      "--- gene=1264, id=27 removed by soft count = 59\n",
      "--- gene=1264, id=63 removed by soft count = 129\n",
      "--- gene=1264, id=83 removed by soft count = 73\n",
      "--- gene=1264, id=85 removed by soft count = 92\n",
      "--- gene=1264, id=93 removed by soft count = 73\n",
      "--- gene=1264, id=121 removed by soft count = 242\n",
      "--- gene=1264, id=150 removed by soft count = 72\n",
      "--- number of probes: 279 , kept by blast: 271 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1265\n",
      "--- gene=1265, id=0 removed by soft count = 406\n",
      "--- gene=1265, id=1 removed by soft count = 123\n",
      "--- gene=1265, id=2 removed by soft count = 729\n",
      "--- gene=1265, id=5 removed by soft count = 477\n",
      "--- gene=1265, id=6 removed by soft count = 351\n",
      "--- gene=1265, id=7 removed by soft count = 560\n",
      "--- gene=1265, id=13 removed by soft count = 893\n",
      "--- gene=1265, id=16 removed by soft count = 107\n",
      "--- gene=1265, id=17 removed by soft count = 95\n",
      "--- gene=1265, id=18 removed by soft count = 65\n",
      "--- gene=1265, id=28 removed by soft count = 92\n",
      "--- gene=1265, id=54 removed by soft count = 53\n",
      "--- gene=1265, id=76 removed by soft count = 85\n",
      "--- gene=1265, id=80 removed by soft count = 55\n",
      "--- gene=1265, id=84 removed by soft count = 10866\n",
      "--- gene=1265, id=86 removed by soft count = 91\n",
      "--- gene=1265, id=87 removed by soft count = 188\n",
      "--- gene=1265, id=88 removed by soft count = 104\n",
      "--- number of probes: 99 , kept by blast: 81 , if remove dups: 81\n",
      "-- number of probes kept for this region: 81\n",
      "-- checking probes in region: 1269\n",
      "--- gene=1269, id=0 removed by soft count = 1746\n",
      "--- gene=1269, id=1 removed by soft count = 1408\n",
      "--- gene=1269, id=5 removed by soft count = 1241\n",
      "--- gene=1269, id=6 removed by soft count = 188\n",
      "--- gene=1269, id=7 removed by soft count = 463\n",
      "--- gene=1269, id=8 removed by soft count = 662\n",
      "--- gene=1269, id=10 removed by soft count = 317\n",
      "--- gene=1269, id=11 removed by soft count = 939\n",
      "--- gene=1269, id=12 removed by soft count = 57\n",
      "--- gene=1269, id=13 removed by soft count = 307\n",
      "--- gene=1269, id=14 removed by soft count = 106\n",
      "--- gene=1269, id=15 removed by soft count = 488\n",
      "--- gene=1269, id=17 removed by soft count = 308\n",
      "--- gene=1269, id=19 removed by soft count = 171\n",
      "--- gene=1269, id=21 removed by soft count = 91\n",
      "--- gene=1269, id=22 removed by soft count = 167\n",
      "--- gene=1269, id=23 removed by soft count = 465\n",
      "--- gene=1269, id=25 removed by soft count = 196\n",
      "--- gene=1269, id=27 removed by soft count = 224\n",
      "--- gene=1269, id=30 removed by soft count = 1968\n",
      "--- gene=1269, id=33 removed by soft count = 99\n",
      "--- gene=1269, id=35 removed by soft count = 583\n",
      "--- gene=1269, id=36 removed by soft count = 167\n",
      "--- gene=1269, id=37 removed by soft count = 533\n",
      "--- gene=1269, id=38 removed by soft count = 153\n",
      "--- gene=1269, id=43 removed by soft count = 640\n",
      "--- gene=1269, id=44 removed by soft count = 108\n",
      "--- gene=1269, id=45 removed by soft count = 3061\n",
      "--- gene=1269, id=49 removed by soft count = 63\n",
      "--- gene=1269, id=50 removed by soft count = 738\n",
      "--- number of probes: 51 , kept by blast: 21 , if remove dups: 21\n",
      "-- number of probes kept for this region: 21\n",
      "-- checking probes in region: 1270\n",
      "--- gene=1270, id=106 removed by soft count = 222\n",
      "--- gene=1270, id=108 removed by soft count = 69\n",
      "--- gene=1270, id=226 removed by soft count = 691\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1271\n",
      "--- gene=1271, id=297 removed by hard count = 0\n",
      "--- gene=1271, id=298 removed by soft count = 58\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1272\n",
      "--- gene=1272, id=232 removed by soft count = 54\n",
      "--- number of probes: 299 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1273\n",
      "--- gene=1273, id=0 removed by soft count = 381\n",
      "--- gene=1273, id=4 removed by soft count = 676\n",
      "--- gene=1273, id=34 removed by soft count = 155\n",
      "--- gene=1273, id=53 removed by soft count = 145\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1273, id=64 removed by soft count = 513\n",
      "--- gene=1273, id=84 removed by soft count = 64\n",
      "--- gene=1273, id=192 removed by soft count = 84\n",
      "--- gene=1273, id=212 removed by hard count = 6\n",
      "--- gene=1273, id=268 removed by soft count = 77\n",
      "--- gene=1273, id=276 removed by soft count = 99\n",
      "--- number of probes: 295 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1274\n",
      "--- gene=1274, id=63 removed by soft count = 377\n",
      "--- gene=1274, id=65 removed by soft count = 147\n",
      "--- gene=1274, id=85 removed by soft count = 66\n",
      "--- gene=1274, id=99 removed by soft count = 354\n",
      "--- gene=1274, id=100 removed by soft count = 75\n",
      "--- gene=1274, id=108 removed by soft count = 100\n",
      "--- gene=1274, id=109 removed by soft count = 66\n",
      "--- gene=1274, id=149 removed by soft count = 143\n",
      "--- gene=1274, id=158 removed by soft count = 701\n",
      "--- gene=1274, id=180 removed by soft count = 335\n",
      "--- gene=1274, id=186 removed by soft count = 82\n",
      "--- gene=1274, id=187 removed by soft count = 94\n",
      "--- gene=1274, id=188 removed by soft count = 57\n",
      "--- number of probes: 192 , kept by blast: 179 , if remove dups: 179\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1275\n",
      "--- gene=1275, id=72 removed by soft count = 301\n",
      "--- gene=1275, id=229 removed by soft count = 132\n",
      "--- gene=1275, id=230 removed by soft count = 55\n",
      "--- number of probes: 293 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1276\n",
      "--- gene=1276, id=109 removed by soft count = 475\n",
      "--- gene=1276, id=242 removed by soft count = 603\n",
      "--- gene=1276, id=243 removed by soft count = 637\n",
      "--- number of probes: 244 , kept by blast: 241 , if remove dups: 241\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1277\n",
      "--- gene=1277, id=13 removed by soft count = 113\n",
      "--- gene=1277, id=260 removed by soft count = 59\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1278\n",
      "--- gene=1278, id=39 removed by soft count = 411\n",
      "--- gene=1278, id=40 removed by soft count = 230\n",
      "--- gene=1278, id=84 removed by soft count = 96\n",
      "--- gene=1278, id=88 removed by soft count = 90\n",
      "--- gene=1278, id=110 removed by soft count = 200\n",
      "--- gene=1278, id=113 removed by soft count = 111\n",
      "--- gene=1278, id=120 removed by soft count = 66\n",
      "--- number of probes: 150 , kept by blast: 143 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1279\n",
      "--- gene=1279, id=31 removed by soft count = 107\n",
      "--- gene=1279, id=32 removed by soft count = 98\n",
      "--- gene=1279, id=33 removed by soft count = 57\n",
      "--- gene=1279, id=34 removed by soft count = 813\n",
      "--- gene=1279, id=66 removed by soft count = 104\n",
      "--- gene=1279, id=69 removed by soft count = 53\n",
      "--- number of probes: 193 , kept by blast: 187 , if remove dups: 187\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1280\n",
      "--- number of probes: 296 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1281\n",
      "--- gene=1281, id=260 removed by soft count = 173\n",
      "--- gene=1281, id=262 removed by soft count = 119\n",
      "--- number of probes: 275 , kept by blast: 273 , if remove dups: 273\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1282\n",
      "--- number of probes: 234 , kept by blast: 234 , if remove dups: 234\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1284\n",
      "--- gene=1284, id=1 removed by soft count = 81\n",
      "--- gene=1284, id=3 removed by soft count = 158\n",
      "--- gene=1284, id=4 removed by soft count = 145\n",
      "--- gene=1284, id=7 removed by soft count = 173\n",
      "--- gene=1284, id=9 removed by soft count = 116\n",
      "--- gene=1284, id=11 removed by soft count = 56\n",
      "--- gene=1284, id=12 removed by soft count = 54\n",
      "--- gene=1284, id=18 removed by soft count = 63\n",
      "--- gene=1284, id=21 removed by soft count = 72\n",
      "--- gene=1284, id=22 removed by soft count = 127\n",
      "--- gene=1284, id=23 removed by soft count = 115\n",
      "--- gene=1284, id=26 removed by soft count = 61\n",
      "--- gene=1284, id=28 removed by soft count = 57\n",
      "--- gene=1284, id=30 removed by soft count = 131\n",
      "--- gene=1284, id=33 removed by soft count = 79\n",
      "--- gene=1284, id=35 removed by soft count = 67\n",
      "--- gene=1284, id=36 removed by soft count = 77\n",
      "--- gene=1284, id=37 removed by soft count = 1292\n",
      "--- gene=1284, id=38 removed by soft count = 318\n",
      "--- gene=1284, id=89 removed by soft count = 135\n",
      "--- number of probes: 188 , kept by blast: 168 , if remove dups: 168\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1285\n",
      "--- gene=1285, id=22 removed by soft count = 142\n",
      "--- gene=1285, id=50 removed by soft count = 335\n",
      "--- gene=1285, id=99 removed by soft count = 388\n",
      "--- number of probes: 206 , kept by blast: 203 , if remove dups: 203\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1286\n",
      "--- gene=1286, id=61 removed by soft count = 1000\n",
      "--- gene=1286, id=258 removed by soft count = 100\n",
      "--- number of probes: 263 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1287\n",
      "--- gene=1287, id=189 removed by soft count = 69\n",
      "--- gene=1287, id=195 removed by soft count = 56790\n",
      "--- number of probes: 287 , kept by blast: 285 , if remove dups: 285\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1288\n",
      "--- gene=1288, id=155 removed by soft count = 129\n",
      "--- gene=1288, id=195 removed by soft count = 285\n",
      "--- number of probes: 239 , kept by blast: 237 , if remove dups: 237\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1289\n",
      "--- gene=1289, id=59 removed by hard count = 0\n",
      "--- gene=1289, id=60 removed by hard count = 0\n",
      "--- number of probes: 299 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1290\n",
      "--- gene=1290, id=9 removed by hard count = 0\n",
      "--- gene=1290, id=201 removed by soft count = 131\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1291\n",
      "--- gene=1291, id=17 removed by soft count = 91\n",
      "--- gene=1291, id=19 removed by soft count = 68\n",
      "--- gene=1291, id=49 removed by soft count = 1428\n",
      "--- gene=1291, id=99 removed by soft count = 59\n",
      "--- gene=1291, id=105 removed by soft count = 171\n",
      "--- gene=1291, id=143 removed by soft count = 104\n",
      "--- gene=1291, id=145 removed by soft count = 76\n",
      "--- gene=1291, id=254 removed by soft count = 476\n",
      "--- number of probes: 273 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1292\n",
      "--- number of probes: 248 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1293\n",
      "--- gene=1293, id=22 removed by soft count = 1523\n",
      "--- gene=1293, id=67 removed by soft count = 76\n",
      "--- gene=1293, id=81 removed by soft count = 109\n",
      "--- gene=1293, id=83 removed by soft count = 65\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1293, id=186 removed by soft count = 97\n",
      "--- gene=1293, id=208 removed by soft count = 87\n",
      "--- gene=1293, id=209 removed by soft count = 69\n",
      "--- gene=1293, id=210 removed by soft count = 901\n",
      "--- gene=1293, id=211 removed by soft count = 53\n",
      "--- gene=1293, id=266 removed by soft count = 58\n",
      "--- number of probes: 300 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1294\n",
      "--- gene=1294, id=1 removed by soft count = 203\n",
      "--- gene=1294, id=3 removed by soft count = 2809\n",
      "--- gene=1294, id=31 removed by soft count = 74\n",
      "--- gene=1294, id=33 removed by soft count = 119\n",
      "--- gene=1294, id=34 removed by soft count = 236\n",
      "--- gene=1294, id=35 removed by soft count = 209\n",
      "--- gene=1294, id=46 removed by soft count = 453\n",
      "--- gene=1294, id=47 removed by soft count = 5567\n",
      "--- gene=1294, id=66 removed by soft count = 72\n",
      "--- gene=1294, id=71 removed by soft count = 317\n",
      "--- gene=1294, id=76 removed by soft count = 53\n",
      "--- gene=1294, id=83 removed by soft count = 107\n",
      "--- number of probes: 90 , kept by blast: 78 , if remove dups: 78\n",
      "-- number of probes kept for this region: 78\n",
      "-- checking probes in region: 1295\n",
      "--- gene=1295, id=25 removed by soft count = 51\n",
      "--- gene=1295, id=28 removed by soft count = 171\n",
      "--- gene=1295, id=49 removed by soft count = 76\n",
      "--- number of probes: 167 , kept by blast: 164 , if remove dups: 163\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1297\n",
      "--- gene=1297, id=130 removed by soft count = 78\n",
      "--- gene=1297, id=253 removed by hard count = 4\n",
      "--- gene=1297, id=292 removed by soft count = 90\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1298\n",
      "--- gene=1298, id=56 removed by soft count = 1252\n",
      "--- gene=1298, id=130 removed by soft count = 607\n",
      "--- gene=1298, id=134 removed by soft count = 172\n",
      "--- gene=1298, id=190 removed by soft count = 62\n",
      "--- gene=1298, id=198 removed by soft count = 53\n",
      "--- gene=1298, id=235 removed by soft count = 56\n",
      "--- gene=1298, id=278 removed by soft count = 53\n",
      "--- number of probes: 282 , kept by blast: 275 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1299\n",
      "--- gene=1299, id=232 removed by soft count = 98\n",
      "--- gene=1299, id=248 removed by soft count = 52\n",
      "--- number of probes: 267 , kept by blast: 265 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1300\n",
      "--- gene=1300, id=39 removed by soft count = 673\n",
      "--- gene=1300, id=66 removed by soft count = 9304\n",
      "--- gene=1300, id=87 removed by soft count = 120\n",
      "--- gene=1300, id=108 removed by soft count = 95\n",
      "--- gene=1300, id=173 removed by soft count = 80\n",
      "--- gene=1300, id=174 removed by soft count = 249\n",
      "--- gene=1300, id=216 removed by soft count = 73\n",
      "--- number of probes: 266 , kept by blast: 259 , if remove dups: 259\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1301\n",
      "--- gene=1301, id=2 removed by soft count = 390\n",
      "--- gene=1301, id=7 removed by soft count = 57\n",
      "--- gene=1301, id=15 removed by soft count = 108\n",
      "--- gene=1301, id=23 removed by soft count = 65\n",
      "--- gene=1301, id=49 removed by soft count = 144\n",
      "--- gene=1301, id=94 removed by soft count = 841\n",
      "--- gene=1301, id=95 removed by soft count = 335\n",
      "--- gene=1301, id=99 removed by soft count = 134\n",
      "--- gene=1301, id=101 removed by soft count = 62\n",
      "--- gene=1301, id=102 removed by soft count = 54\n",
      "--- gene=1301, id=106 removed by soft count = 105\n",
      "--- gene=1301, id=109 removed by soft count = 134\n",
      "--- gene=1301, id=167 removed by soft count = 64\n",
      "--- gene=1301, id=226 removed by soft count = 121\n",
      "--- gene=1301, id=228 removed by soft count = 152\n",
      "--- number of probes: 252 , kept by blast: 237 , if remove dups: 237\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1302\n",
      "--- gene=1302, id=68 removed by soft count = 87\n",
      "--- gene=1302, id=154 removed by soft count = 3767\n",
      "--- gene=1302, id=205 removed by soft count = 254\n",
      "--- gene=1302, id=207 removed by soft count = 113\n",
      "--- gene=1302, id=211 removed by soft count = 157\n",
      "--- gene=1302, id=212 removed by soft count = 51\n",
      "--- number of probes: 220 , kept by blast: 214 , if remove dups: 214\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1303\n",
      "--- gene=1303, id=123 removed by soft count = 137\n",
      "--- gene=1303, id=125 removed by soft count = 185\n",
      "--- gene=1303, id=126 removed by soft count = 105\n",
      "--- gene=1303, id=133 removed by soft count = 86\n",
      "--- gene=1303, id=155 removed by soft count = 82\n",
      "--- number of probes: 174 , kept by blast: 169 , if remove dups: 169\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1304\n",
      "--- gene=1304, id=127 removed by soft count = 18531\n",
      "--- gene=1304, id=128 removed by soft count = 1216\n",
      "--- gene=1304, id=168 removed by soft count = 1422\n",
      "--- number of probes: 268 , kept by blast: 265 , if remove dups: 265\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1305\n",
      "--- gene=1305, id=133 removed by soft count = 147\n",
      "--- number of probes: 297 , kept by blast: 296 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1306\n",
      "--- gene=1306, id=272 removed by soft count = 289\n",
      "--- number of probes: 285 , kept by blast: 284 , if remove dups: 284\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1307\n",
      "--- gene=1307, id=13 removed by soft count = 51\n",
      "--- gene=1307, id=14 removed by soft count = 88\n",
      "--- gene=1307, id=16 removed by soft count = 57\n",
      "--- gene=1307, id=18 removed by soft count = 59\n",
      "--- gene=1307, id=61 removed by soft count = 76\n",
      "--- gene=1307, id=63 removed by soft count = 3097\n",
      "--- gene=1307, id=144 removed by soft count = 4085\n",
      "--- gene=1307, id=168 removed by soft count = 104\n",
      "--- gene=1307, id=169 removed by soft count = 51\n",
      "--- number of probes: 271 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1308\n",
      "--- gene=1308, id=191 removed by soft count = 80\n",
      "--- gene=1308, id=229 removed by soft count = 240\n",
      "--- number of probes: 239 , kept by blast: 237 , if remove dups: 237\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1309\n",
      "--- gene=1309, id=1 removed by soft count = 118\n",
      "--- gene=1309, id=103 removed by soft count = 134\n",
      "--- gene=1309, id=104 removed by soft count = 100\n",
      "--- gene=1309, id=176 removed by soft count = 101\n",
      "--- gene=1309, id=177 removed by soft count = 55\n",
      "--- gene=1309, id=192 removed by soft count = 55\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1310\n",
      "--- gene=1310, id=86 removed by soft count = 68\n",
      "--- gene=1310, id=196 removed by soft count = 160\n",
      "--- gene=1310, id=257 removed by soft count = 105\n",
      "--- number of probes: 285 , kept by blast: 282 , if remove dups: 282\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1311\n",
      "--- gene=1311, id=2 removed by soft count = 92\n",
      "--- gene=1311, id=3 removed by soft count = 90\n",
      "--- gene=1311, id=187 removed by soft count = 70\n",
      "--- gene=1311, id=188 removed by soft count = 52\n",
      "--- gene=1311, id=189 removed by soft count = 125\n",
      "--- number of probes: 211 , kept by blast: 206 , if remove dups: 206\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1312\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1312, id=21 removed by soft count = 184\n",
      "--- gene=1312, id=227 removed by soft count = 149\n",
      "--- gene=1312, id=231 removed by soft count = 174\n",
      "--- number of probes: 257 , kept by blast: 254 , if remove dups: 254\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1313\n",
      "--- gene=1313, id=7 removed by soft count = 102\n",
      "--- gene=1313, id=79 removed by soft count = 378\n",
      "--- gene=1313, id=119 removed by soft count = 419\n",
      "--- gene=1313, id=124 removed by soft count = 308\n",
      "--- gene=1313, id=153 removed by soft count = 83\n",
      "--- gene=1313, id=154 removed by soft count = 152\n",
      "--- gene=1313, id=163 removed by soft count = 65\n",
      "--- gene=1313, id=165 removed by soft count = 122\n",
      "--- gene=1313, id=166 removed by soft count = 220\n",
      "--- gene=1313, id=167 removed by soft count = 544\n",
      "--- number of probes: 201 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1314\n",
      "--- gene=1314, id=193 removed by soft count = 1573\n",
      "--- number of probes: 272 , kept by blast: 271 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1315\n",
      "--- gene=1315, id=68 removed by soft count = 91\n",
      "--- gene=1315, id=117 removed by soft count = 73\n",
      "--- gene=1315, id=294 removed by soft count = 61\n",
      "--- gene=1315, id=295 removed by soft count = 1263\n",
      "--- gene=1315, id=296 removed by soft count = 126\n",
      "--- gene=1315, id=297 removed by soft count = 346\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1316\n",
      "--- gene=1316, id=238 removed by soft count = 783\n",
      "--- gene=1316, id=239 removed by soft count = 844\n",
      "--- gene=1316, id=240 removed by soft count = 58\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1317\n",
      "--- gene=1317, id=62 removed by soft count = 53\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1318\n",
      "--- gene=1318, id=2 removed by soft count = 82\n",
      "--- gene=1318, id=3 removed by soft count = 110\n",
      "--- gene=1318, id=112 removed by soft count = 102\n",
      "--- gene=1318, id=113 removed by soft count = 74\n",
      "--- gene=1318, id=114 removed by soft count = 69\n",
      "--- gene=1318, id=115 removed by soft count = 128\n",
      "--- gene=1318, id=116 removed by soft count = 172\n",
      "--- gene=1318, id=118 removed by soft count = 100\n",
      "--- gene=1318, id=120 removed by soft count = 344\n",
      "--- number of probes: 250 , kept by blast: 241 , if remove dups: 240\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1319\n",
      "--- gene=1319, id=55 removed by soft count = 212\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1320\n",
      "--- gene=1320, id=2 removed by soft count = 53\n",
      "--- gene=1320, id=10 removed by soft count = 136\n",
      "--- gene=1320, id=20 removed by soft count = 56\n",
      "--- gene=1320, id=148 removed by soft count = 863\n",
      "--- gene=1320, id=150 removed by soft count = 72\n",
      "--- gene=1320, id=151 removed by soft count = 441\n",
      "--- gene=1320, id=152 removed by soft count = 359\n",
      "--- gene=1320, id=202 removed by soft count = 47301\n",
      "--- gene=1320, id=272 removed by soft count = 81\n",
      "--- number of probes: 286 , kept by blast: 277 , if remove dups: 277\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1321\n",
      "--- gene=1321, id=0 removed by soft count = 864\n",
      "--- gene=1321, id=1 removed by soft count = 68\n",
      "--- gene=1321, id=38 removed by soft count = 51\n",
      "--- gene=1321, id=81 removed by soft count = 60\n",
      "--- gene=1321, id=82 removed by soft count = 167\n",
      "--- gene=1321, id=156 removed by soft count = 321\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1322\n",
      "--- gene=1322, id=122 removed by soft count = 868\n",
      "--- gene=1322, id=123 removed by soft count = 349\n",
      "--- gene=1322, id=124 removed by soft count = 108\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1323\n",
      "--- gene=1323, id=0 removed by soft count = 214\n",
      "--- gene=1323, id=3 removed by soft count = 117\n",
      "--- gene=1323, id=7 removed by soft count = 802\n",
      "--- gene=1323, id=8 removed by soft count = 310\n",
      "--- gene=1323, id=11 removed by soft count = 89\n",
      "--- gene=1323, id=12 removed by soft count = 229\n",
      "--- gene=1323, id=35 removed by soft count = 2992\n",
      "--- gene=1323, id=97 removed by soft count = 199\n",
      "--- gene=1323, id=159 removed by soft count = 55\n",
      "--- gene=1323, id=180 removed by soft count = 77\n",
      "--- gene=1323, id=216 removed by soft count = 158\n",
      "--- gene=1323, id=286 removed by soft count = 321\n",
      "--- gene=1323, id=297 removed by soft count = 90\n",
      "--- number of probes: 300 , kept by blast: 287 , if remove dups: 287\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1324\n",
      "--- gene=1324, id=141 removed by soft count = 111\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1325\n",
      "--- gene=1325, id=194 removed by soft count = 65\n",
      "--- gene=1325, id=293 removed by soft count = 1869\n",
      "--- number of probes: 294 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1330\n",
      "--- gene=1330, id=73 removed by soft count = 62\n",
      "--- gene=1330, id=128 removed by soft count = 86\n",
      "--- number of probes: 129 , kept by blast: 127 , if remove dups: 125\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1331\n",
      "--- number of probes: 242 , kept by blast: 242 , if remove dups: 242\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1332\n",
      "--- gene=1332, id=152 removed by soft count = 269\n",
      "--- gene=1332, id=153 removed by soft count = 338\n",
      "--- number of probes: 278 , kept by blast: 276 , if remove dups: 276\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1333\n",
      "--- gene=1333, id=38 removed by soft count = 2539\n",
      "--- gene=1333, id=91 removed by soft count = 104\n",
      "--- gene=1333, id=127 removed by soft count = 91\n",
      "--- number of probes: 136 , kept by blast: 133 , if remove dups: 133\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1334\n",
      "--- gene=1334, id=3 removed by soft count = 373\n",
      "--- gene=1334, id=21 removed by soft count = 143\n",
      "--- gene=1334, id=50 removed by soft count = 365\n",
      "--- gene=1334, id=56 removed by soft count = 481\n",
      "--- gene=1334, id=57 removed by soft count = 158\n",
      "--- gene=1334, id=58 removed by soft count = 80\n",
      "--- gene=1334, id=59 removed by soft count = 147\n",
      "--- gene=1334, id=84 removed by soft count = 2515\n",
      "--- gene=1334, id=85 removed by soft count = 2223\n",
      "--- gene=1334, id=125 removed by soft count = 56\n",
      "--- gene=1334, id=127 removed by soft count = 70\n",
      "--- gene=1334, id=159 removed by soft count = 64\n",
      "--- number of probes: 161 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1335\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1335, id=117 removed by soft count = 482\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1336\n",
      "--- gene=1336, id=38 removed by soft count = 99\n",
      "--- gene=1336, id=201 removed by soft count = 418\n",
      "--- gene=1336, id=205 removed by soft count = 679\n",
      "--- gene=1336, id=208 removed by soft count = 79\n",
      "--- gene=1336, id=212 removed by soft count = 176\n",
      "--- gene=1336, id=213 removed by soft count = 127\n",
      "--- gene=1336, id=228 removed by soft count = 67\n",
      "--- gene=1336, id=246 removed by soft count = 57\n",
      "--- number of probes: 266 , kept by blast: 258 , if remove dups: 258\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1337\n",
      "--- gene=1337, id=133 removed by soft count = 1326\n",
      "--- number of probes: 299 , kept by blast: 298 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1338\n",
      "--- gene=1338, id=130 removed by soft count = 331\n",
      "--- gene=1338, id=133 removed by soft count = 217\n",
      "--- gene=1338, id=140 removed by soft count = 96\n",
      "--- gene=1338, id=142 removed by soft count = 364\n",
      "--- gene=1338, id=153 removed by soft count = 82\n",
      "--- number of probes: 188 , kept by blast: 183 , if remove dups: 183\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1339\n",
      "--- gene=1339, id=7 removed by soft count = 108\n",
      "--- gene=1339, id=9 removed by soft count = 86\n",
      "--- gene=1339, id=10 removed by soft count = 90\n",
      "--- gene=1339, id=65 removed by soft count = 59\n",
      "--- gene=1339, id=203 removed by soft count = 71\n",
      "--- gene=1339, id=231 removed by soft count = 117\n",
      "--- gene=1339, id=236 removed by soft count = 1651\n",
      "--- gene=1339, id=237 removed by soft count = 324\n",
      "--- number of probes: 238 , kept by blast: 230 , if remove dups: 230\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1340\n",
      "--- gene=1340, id=59 removed by soft count = 71\n",
      "--- gene=1340, id=225 removed by soft count = 81\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1341\n",
      "--- gene=1341, id=60 removed by soft count = 454\n",
      "--- gene=1341, id=61 removed by soft count = 401\n",
      "--- gene=1341, id=62 removed by soft count = 537\n",
      "--- gene=1341, id=101 removed by soft count = 69\n",
      "--- gene=1341, id=140 removed by soft count = 179\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1342\n",
      "--- gene=1342, id=59 removed by soft count = 51\n",
      "--- gene=1342, id=67 removed by soft count = 149\n",
      "--- gene=1342, id=105 removed by soft count = 69\n",
      "--- gene=1342, id=106 removed by soft count = 176\n",
      "--- gene=1342, id=108 removed by soft count = 81\n",
      "--- gene=1342, id=110 removed by soft count = 66\n",
      "--- gene=1342, id=117 removed by soft count = 113\n",
      "--- gene=1342, id=123 removed by soft count = 123\n",
      "--- gene=1342, id=128 removed by soft count = 106\n",
      "--- gene=1342, id=229 removed by soft count = 154\n",
      "--- number of probes: 261 , kept by blast: 251 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1343\n",
      "--- gene=1343, id=16 removed by soft count = 249\n",
      "--- gene=1343, id=45 removed by soft count = 452\n",
      "--- gene=1343, id=50 removed by soft count = 239\n",
      "--- gene=1343, id=62 removed by soft count = 124\n",
      "--- gene=1343, id=79 removed by soft count = 765\n",
      "--- gene=1343, id=86 removed by soft count = 337\n",
      "--- gene=1343, id=95 removed by soft count = 64\n",
      "--- gene=1343, id=98 removed by soft count = 128\n",
      "--- gene=1343, id=99 removed by soft count = 313\n",
      "--- gene=1343, id=159 removed by soft count = 221\n",
      "--- gene=1343, id=165 removed by soft count = 114\n",
      "--- number of probes: 201 , kept by blast: 190 , if remove dups: 190\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1344\n",
      "--- gene=1344, id=190 removed by soft count = 2146\n",
      "--- gene=1344, id=221 removed by hard count = 0\n",
      "--- number of probes: 274 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1345\n",
      "--- gene=1345, id=1 removed by soft count = 87\n",
      "--- gene=1345, id=2 removed by soft count = 71\n",
      "--- gene=1345, id=5 removed by soft count = 77\n",
      "--- gene=1345, id=11 removed by soft count = 162\n",
      "--- gene=1345, id=24 removed by soft count = 101\n",
      "--- gene=1345, id=27 removed by soft count = 1051\n",
      "--- gene=1345, id=28 removed by soft count = 224\n",
      "--- gene=1345, id=139 removed by soft count = 78\n",
      "--- gene=1345, id=176 removed by soft count = 73\n",
      "--- gene=1345, id=208 removed by soft count = 87\n",
      "--- number of probes: 300 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1346\n",
      "--- gene=1346, id=61 removed by soft count = 116\n",
      "--- gene=1346, id=167 removed by soft count = 70\n",
      "--- gene=1346, id=290 removed by soft count = 82\n",
      "--- gene=1346, id=291 removed by soft count = 72\n",
      "--- number of probes: 298 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1347\n",
      "--- gene=1347, id=23 removed by soft count = 94\n",
      "--- gene=1347, id=31 removed by soft count = 4777\n",
      "--- gene=1347, id=167 removed by soft count = 2013\n",
      "--- number of probes: 179 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1348\n",
      "--- gene=1348, id=76 removed by soft count = 105\n",
      "--- gene=1348, id=86 removed by soft count = 83\n",
      "--- gene=1348, id=88 removed by soft count = 103\n",
      "--- gene=1348, id=89 removed by soft count = 96\n",
      "--- gene=1348, id=91 removed by soft count = 130\n",
      "--- gene=1348, id=92 removed by soft count = 52\n",
      "--- gene=1348, id=93 removed by soft count = 208\n",
      "--- gene=1348, id=94 removed by soft count = 88\n",
      "--- gene=1348, id=95 removed by soft count = 64\n",
      "--- gene=1348, id=96 removed by soft count = 113\n",
      "--- gene=1348, id=97 removed by soft count = 128\n",
      "--- gene=1348, id=98 removed by soft count = 140\n",
      "--- gene=1348, id=99 removed by soft count = 141\n",
      "--- gene=1348, id=100 removed by soft count = 119\n",
      "--- gene=1348, id=101 removed by soft count = 120\n",
      "--- gene=1348, id=102 removed by soft count = 136\n",
      "--- gene=1348, id=103 removed by soft count = 118\n",
      "--- gene=1348, id=104 removed by soft count = 125\n",
      "--- gene=1348, id=105 removed by soft count = 113\n",
      "--- gene=1348, id=106 removed by soft count = 326\n",
      "--- gene=1348, id=107 removed by soft count = 128\n",
      "--- gene=1348, id=108 removed by soft count = 203\n",
      "--- gene=1348, id=109 removed by soft count = 56\n",
      "--- gene=1348, id=110 removed by soft count = 296\n",
      "--- gene=1348, id=112 removed by soft count = 148\n",
      "--- gene=1348, id=113 removed by hard count = 4\n",
      "--- gene=1348, id=114 removed by soft count = 145\n",
      "--- gene=1348, id=115 removed by hard count = 4\n",
      "--- gene=1348, id=116 removed by soft count = 258\n",
      "--- gene=1348, id=117 removed by soft count = 124\n",
      "--- gene=1348, id=118 removed by soft count = 51\n",
      "--- gene=1348, id=119 removed by soft count = 144\n",
      "--- gene=1348, id=120 removed by soft count = 129\n",
      "--- gene=1348, id=121 removed by soft count = 157\n",
      "--- gene=1348, id=122 removed by soft count = 105\n",
      "--- number of probes: 123 , kept by blast: 88 , if remove dups: 88\n",
      "-- number of probes kept for this region: 88\n",
      "-- checking probes in region: 1349\n",
      "--- gene=1349, id=38 removed by soft count = 146\n",
      "--- gene=1349, id=39 removed by soft count = 872\n",
      "--- gene=1349, id=103 removed by soft count = 53\n",
      "--- gene=1349, id=131 removed by soft count = 69\n",
      "--- gene=1349, id=143 removed by soft count = 184\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1349, id=159 removed by soft count = 185\n",
      "--- gene=1349, id=160 removed by soft count = 96\n",
      "--- gene=1349, id=174 removed by soft count = 1165\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1350\n",
      "--- gene=1350, id=0 removed by soft count = 67\n",
      "--- gene=1350, id=20 removed by soft count = 81\n",
      "--- gene=1350, id=35 removed by soft count = 66\n",
      "--- number of probes: 265 , kept by blast: 262 , if remove dups: 262\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1351\n",
      "--- gene=1351, id=152 removed by soft count = 145\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1352\n",
      "--- gene=1352, id=13 removed by soft count = 182\n",
      "--- gene=1352, id=184 removed by soft count = 88\n",
      "--- gene=1352, id=186 removed by soft count = 307\n",
      "--- gene=1352, id=187 removed by soft count = 889\n",
      "--- gene=1352, id=189 removed by soft count = 323\n",
      "--- gene=1352, id=190 removed by soft count = 672\n",
      "--- gene=1352, id=192 removed by soft count = 324\n",
      "--- gene=1352, id=193 removed by soft count = 63\n",
      "--- number of probes: 243 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1353\n",
      "--- number of probes: 192 , kept by blast: 192 , if remove dups: 192\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1354\n",
      "--- number of probes: 278 , kept by blast: 278 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1355\n",
      "--- gene=1355, id=65 removed by soft count = 1365\n",
      "--- gene=1355, id=66 removed by soft count = 2750\n",
      "--- gene=1355, id=67 removed by soft count = 94\n",
      "--- gene=1355, id=68 removed by soft count = 113\n",
      "--- gene=1355, id=69 removed by soft count = 97\n",
      "--- gene=1355, id=70 removed by soft count = 486\n",
      "--- gene=1355, id=71 removed by soft count = 96\n",
      "--- gene=1355, id=73 removed by soft count = 92\n",
      "--- gene=1355, id=163 removed by soft count = 104\n",
      "--- gene=1355, id=164 removed by soft count = 98\n",
      "--- gene=1355, id=168 removed by soft count = 148\n",
      "--- gene=1355, id=171 removed by soft count = 86\n",
      "--- gene=1355, id=173 removed by soft count = 220\n",
      "--- number of probes: 187 , kept by blast: 174 , if remove dups: 174\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1356\n",
      "--- gene=1356, id=227 removed by soft count = 66\n",
      "--- gene=1356, id=228 removed by soft count = 193\n",
      "--- gene=1356, id=245 removed by soft count = 128\n",
      "--- number of probes: 291 , kept by blast: 288 , if remove dups: 288\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1357\n",
      "--- gene=1357, id=3 removed by soft count = 728\n",
      "--- gene=1357, id=22 removed by soft count = 94\n",
      "--- gene=1357, id=94 removed by soft count = 619\n",
      "--- gene=1357, id=124 removed by soft count = 159\n",
      "--- gene=1357, id=131 removed by soft count = 113\n",
      "--- gene=1357, id=163 removed by soft count = 604\n",
      "--- gene=1357, id=165 removed by soft count = 75\n",
      "--- number of probes: 246 , kept by blast: 239 , if remove dups: 239\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1358\n",
      "--- gene=1358, id=24 removed by soft count = 126\n",
      "--- gene=1358, id=65 removed by soft count = 561\n",
      "--- gene=1358, id=263 removed by soft count = 79\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1359\n",
      "--- gene=1359, id=38 removed by soft count = 76\n",
      "--- gene=1359, id=134 removed by soft count = 268\n",
      "--- gene=1359, id=202 removed by soft count = 64\n",
      "--- gene=1359, id=212 removed by soft count = 85\n",
      "--- gene=1359, id=239 removed by soft count = 838\n",
      "--- gene=1359, id=266 removed by soft count = 69\n",
      "--- number of probes: 267 , kept by blast: 261 , if remove dups: 259\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1360\n",
      "--- gene=1360, id=16 removed by soft count = 63\n",
      "--- gene=1360, id=43 removed by soft count = 141\n",
      "--- number of probes: 299 , kept by blast: 297 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1361\n",
      "--- gene=1361, id=243 removed by soft count = 79\n",
      "--- gene=1361, id=293 removed by soft count = 55\n",
      "--- gene=1361, id=295 removed by soft count = 70\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1362\n",
      "--- gene=1362, id=121 removed by hard count = 0\n",
      "--- gene=1362, id=134 removed by soft count = 139\n",
      "--- number of probes: 294 , kept by blast: 292 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1363\n",
      "--- gene=1363, id=272 removed by soft count = 122\n",
      "--- gene=1363, id=282 removed by soft count = 184\n",
      "--- gene=1363, id=286 removed by hard count = 3\n",
      "--- gene=1363, id=290 removed by soft count = 88\n",
      "--- number of probes: 296 , kept by blast: 292 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1378\n",
      "--- gene=1378, id=155 removed by soft count = 164\n",
      "--- gene=1378, id=157 removed by soft count = 350\n",
      "--- gene=1378, id=158 removed by soft count = 1095\n",
      "--- gene=1378, id=159 removed by soft count = 2260\n",
      "--- number of probes: 217 , kept by blast: 213 , if remove dups: 213\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1379\n",
      "--- gene=1379, id=175 removed by soft count = 60\n",
      "--- number of probes: 292 , kept by blast: 291 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1380\n",
      "--- gene=1380, id=4 removed by soft count = 92\n",
      "--- gene=1380, id=44 removed by soft count = 193\n",
      "--- gene=1380, id=190 removed by soft count = 182\n",
      "--- gene=1380, id=191 removed by soft count = 104\n",
      "--- number of probes: 192 , kept by blast: 188 , if remove dups: 188\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1381\n",
      "--- gene=1381, id=117 removed by soft count = 55\n",
      "--- gene=1381, id=141 removed by soft count = 130\n",
      "--- gene=1381, id=149 removed by soft count = 236\n",
      "--- gene=1381, id=151 removed by soft count = 887\n",
      "--- number of probes: 214 , kept by blast: 210 , if remove dups: 210\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1382\n",
      "--- gene=1382, id=28 removed by soft count = 554\n",
      "--- gene=1382, id=104 removed by soft count = 324\n",
      "--- gene=1382, id=148 removed by soft count = 52\n",
      "--- gene=1382, id=149 removed by soft count = 305\n",
      "--- gene=1382, id=175 removed by soft count = 1963\n",
      "--- gene=1382, id=178 removed by soft count = 190\n",
      "--- number of probes: 210 , kept by blast: 204 , if remove dups: 204\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1383\n",
      "--- gene=1383, id=20 removed by soft count = 53\n",
      "--- gene=1383, id=73 removed by soft count = 1164\n",
      "--- gene=1383, id=74 removed by soft count = 238\n",
      "--- gene=1383, id=110 removed by soft count = 125\n",
      "--- gene=1383, id=111 removed by soft count = 94\n",
      "--- gene=1383, id=112 removed by soft count = 107\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1383, id=139 removed by soft count = 643\n",
      "--- gene=1383, id=140 removed by soft count = 59\n",
      "--- gene=1383, id=141 removed by soft count = 65\n",
      "--- gene=1383, id=146 removed by soft count = 69\n",
      "--- gene=1383, id=147 removed by soft count = 281\n",
      "--- gene=1383, id=148 removed by soft count = 93\n",
      "--- gene=1383, id=152 removed by soft count = 124\n",
      "--- gene=1383, id=187 removed by soft count = 182\n",
      "--- gene=1383, id=192 removed by soft count = 93\n",
      "--- gene=1383, id=228 removed by soft count = 301\n",
      "--- gene=1383, id=229 removed by soft count = 59\n",
      "--- number of probes: 230 , kept by blast: 213 , if remove dups: 213\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1384\n",
      "--- gene=1384, id=33 removed by soft count = 479\n",
      "--- gene=1384, id=39 removed by soft count = 92\n",
      "--- gene=1384, id=58 removed by soft count = 189\n",
      "--- gene=1384, id=72 removed by soft count = 171\n",
      "--- gene=1384, id=73 removed by soft count = 129\n",
      "--- gene=1384, id=75 removed by soft count = 4197\n",
      "--- gene=1384, id=76 removed by soft count = 272\n",
      "--- gene=1384, id=77 removed by soft count = 968\n",
      "--- gene=1384, id=78 removed by soft count = 420\n",
      "--- gene=1384, id=79 removed by soft count = 597\n",
      "--- gene=1384, id=83 removed by soft count = 232\n",
      "--- gene=1384, id=84 removed by soft count = 256\n",
      "--- gene=1384, id=87 removed by soft count = 63\n",
      "--- gene=1384, id=88 removed by soft count = 302\n",
      "--- gene=1384, id=123 removed by soft count = 66\n",
      "--- gene=1384, id=131 removed by soft count = 414\n",
      "--- number of probes: 148 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1385\n",
      "--- gene=1385, id=19 removed by soft count = 71\n",
      "--- gene=1385, id=41 removed by soft count = 73\n",
      "--- gene=1385, id=43 removed by soft count = 55\n",
      "--- gene=1385, id=56 removed by soft count = 416\n",
      "--- gene=1385, id=57 removed by soft count = 307\n",
      "--- gene=1385, id=105 removed by soft count = 109\n",
      "--- gene=1385, id=148 removed by soft count = 164\n",
      "--- gene=1385, id=149 removed by soft count = 55\n",
      "--- gene=1385, id=152 removed by soft count = 314\n",
      "--- gene=1385, id=153 removed by soft count = 624\n",
      "--- number of probes: 163 , kept by blast: 153 , if remove dups: 153\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1386\n",
      "--- gene=1386, id=8 removed by soft count = 102\n",
      "--- gene=1386, id=36 removed by soft count = 130\n",
      "--- gene=1386, id=37 removed by soft count = 176\n",
      "--- gene=1386, id=48 removed by soft count = 1000\n",
      "--- gene=1386, id=49 removed by soft count = 357\n",
      "--- gene=1386, id=50 removed by soft count = 274\n",
      "--- gene=1386, id=104 removed by soft count = 721\n",
      "--- gene=1386, id=105 removed by soft count = 871\n",
      "--- gene=1386, id=107 removed by soft count = 56\n",
      "--- gene=1386, id=108 removed by soft count = 93\n",
      "--- gene=1386, id=113 removed by soft count = 168\n",
      "--- number of probes: 128 , kept by blast: 117 , if remove dups: 117\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 117\n",
      "-- checking probes in region: 1387\n",
      "--- gene=1387, id=16 removed by soft count = 73\n",
      "--- gene=1387, id=91 removed by soft count = 912\n",
      "--- gene=1387, id=92 removed by soft count = 106\n",
      "--- gene=1387, id=93 removed by soft count = 185\n",
      "--- gene=1387, id=94 removed by soft count = 384\n",
      "--- gene=1387, id=100 removed by soft count = 1345\n",
      "--- number of probes: 190 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1388\n",
      "--- gene=1388, id=21 removed by soft count = 665\n",
      "--- gene=1388, id=96 removed by soft count = 69\n",
      "--- gene=1388, id=103 removed by soft count = 121\n",
      "--- gene=1388, id=104 removed by soft count = 761\n",
      "--- gene=1388, id=121 removed by soft count = 98\n",
      "--- number of probes: 176 , kept by blast: 171 , if remove dups: 171\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1389\n",
      "--- gene=1389, id=68 removed by soft count = 833\n",
      "--- gene=1389, id=157 removed by soft count = 53\n",
      "--- gene=1389, id=159 removed by soft count = 52\n",
      "--- gene=1389, id=232 removed by soft count = 73\n",
      "--- gene=1389, id=235 removed by soft count = 126\n",
      "--- gene=1389, id=236 removed by soft count = 171\n",
      "--- number of probes: 251 , kept by blast: 245 , if remove dups: 245\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1390\n",
      "--- gene=1390, id=119 removed by soft count = 381\n",
      "--- gene=1390, id=130 removed by soft count = 104\n",
      "--- number of probes: 288 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1391\n",
      "--- gene=1391, id=83 removed by soft count = 71\n",
      "--- gene=1391, id=160 removed by soft count = 307\n",
      "--- gene=1391, id=187 removed by soft count = 2665\n",
      "--- gene=1391, id=195 removed by soft count = 62\n",
      "--- gene=1391, id=206 removed by soft count = 387\n",
      "--- number of probes: 233 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1392\n",
      "--- gene=1392, id=103 removed by soft count = 136\n",
      "--- gene=1392, id=117 removed by soft count = 3754\n",
      "--- gene=1392, id=159 removed by soft count = 491\n",
      "--- gene=1392, id=161 removed by soft count = 357\n",
      "--- gene=1392, id=165 removed by soft count = 242\n",
      "--- gene=1392, id=166 removed by soft count = 431\n",
      "--- gene=1392, id=167 removed by soft count = 105\n",
      "--- number of probes: 244 , kept by blast: 237 , if remove dups: 237\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1393\n",
      "--- gene=1393, id=41 removed by soft count = 188\n",
      "--- gene=1393, id=42 removed by soft count = 75\n",
      "--- gene=1393, id=43 removed by soft count = 102\n",
      "--- gene=1393, id=44 removed by soft count = 188\n",
      "--- gene=1393, id=218 removed by soft count = 62\n",
      "--- number of probes: 300 , kept by blast: 295 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1394\n",
      "--- gene=1394, id=2 removed by soft count = 120\n",
      "--- gene=1394, id=4 removed by soft count = 145\n",
      "--- gene=1394, id=5 removed by soft count = 86\n",
      "--- gene=1394, id=6 removed by soft count = 130\n",
      "--- gene=1394, id=7 removed by soft count = 185\n",
      "--- gene=1394, id=81 removed by soft count = 72\n",
      "--- gene=1394, id=99 removed by soft count = 1279\n",
      "--- gene=1394, id=100 removed by soft count = 286\n",
      "--- gene=1394, id=138 removed by soft count = 797\n",
      "--- number of probes: 174 , kept by blast: 165 , if remove dups: 165\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1395\n",
      "--- gene=1395, id=41 removed by soft count = 429\n",
      "--- gene=1395, id=73 removed by soft count = 262\n",
      "--- gene=1395, id=75 removed by soft count = 212\n",
      "--- gene=1395, id=98 removed by soft count = 63\n",
      "--- gene=1395, id=105 removed by soft count = 64\n",
      "--- gene=1395, id=106 removed by soft count = 433\n",
      "--- gene=1395, id=109 removed by soft count = 74\n",
      "--- gene=1395, id=206 removed by soft count = 38962\n",
      "--- gene=1395, id=241 removed by soft count = 56\n",
      "--- gene=1395, id=243 removed by soft count = 67\n",
      "--- gene=1395, id=246 removed by soft count = 88\n",
      "--- number of probes: 282 , kept by blast: 271 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1396\n",
      "--- gene=1396, id=25 removed by soft count = 147\n",
      "--- gene=1396, id=26 removed by soft count = 95\n",
      "--- gene=1396, id=40 removed by soft count = 357\n",
      "--- gene=1396, id=139 removed by soft count = 215\n",
      "--- gene=1396, id=146 removed by soft count = 256\n",
      "--- gene=1396, id=161 removed by soft count = 78\n",
      "--- gene=1396, id=165 removed by soft count = 82\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1396, id=183 removed by soft count = 268\n",
      "--- number of probes: 192 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1398\n",
      "--- gene=1398, id=0 removed by soft count = 113\n",
      "--- gene=1398, id=2 removed by soft count = 619\n",
      "--- gene=1398, id=3 removed by soft count = 281\n",
      "--- gene=1398, id=4 removed by soft count = 157\n",
      "--- gene=1398, id=5 removed by soft count = 86\n",
      "--- gene=1398, id=6 removed by soft count = 120\n",
      "--- gene=1398, id=7 removed by soft count = 57\n",
      "--- gene=1398, id=8 removed by soft count = 139\n",
      "--- gene=1398, id=14 removed by soft count = 112\n",
      "--- gene=1398, id=15 removed by soft count = 265\n",
      "--- gene=1398, id=16 removed by soft count = 91\n",
      "--- gene=1398, id=73 removed by soft count = 81\n",
      "--- gene=1398, id=74 removed by soft count = 482\n",
      "--- gene=1398, id=122 removed by soft count = 58\n",
      "--- number of probes: 184 , kept by blast: 170 , if remove dups: 170\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1399\n",
      "--- gene=1399, id=58 removed by soft count = 58\n",
      "--- gene=1399, id=119 removed by soft count = 97\n",
      "--- number of probes: 268 , kept by blast: 266 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1400\n",
      "--- gene=1400, id=1 removed by soft count = 56\n",
      "--- gene=1400, id=65 removed by soft count = 66\n",
      "--- gene=1400, id=172 removed by soft count = 116\n",
      "--- gene=1400, id=203 removed by soft count = 167\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1401\n",
      "--- gene=1401, id=91 removed by soft count = 73\n",
      "--- gene=1401, id=113 removed by soft count = 175\n",
      "--- number of probes: 268 , kept by blast: 266 , if remove dups: 266\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1402\n",
      "--- gene=1402, id=0 removed by soft count = 332\n",
      "--- gene=1402, id=1 removed by soft count = 262\n",
      "--- gene=1402, id=4 removed by soft count = 63\n",
      "--- gene=1402, id=7 removed by soft count = 76\n",
      "--- number of probes: 300 , kept by blast: 296 , if remove dups: 296\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1403\n",
      "--- gene=1403, id=38 removed by soft count = 79\n",
      "--- gene=1403, id=184 removed by hard count = 0\n",
      "--- number of probes: 223 , kept by blast: 221 , if remove dups: 221\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1404\n",
      "--- gene=1404, id=73 removed by soft count = 153\n",
      "--- gene=1404, id=79 removed by soft count = 74\n",
      "--- gene=1404, id=81 removed by soft count = 248\n",
      "--- gene=1404, id=82 removed by soft count = 60\n",
      "--- gene=1404, id=83 removed by soft count = 82\n",
      "--- gene=1404, id=119 removed by soft count = 52\n",
      "--- gene=1404, id=218 removed by soft count = 712\n",
      "--- gene=1404, id=219 removed by soft count = 95\n",
      "--- gene=1404, id=220 removed by soft count = 293\n",
      "--- number of probes: 235 , kept by blast: 226 , if remove dups: 226\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1405\n",
      "--- gene=1405, id=17 removed by soft count = 67\n",
      "--- gene=1405, id=18 removed by soft count = 537\n",
      "--- gene=1405, id=47 removed by soft count = 52\n",
      "--- gene=1405, id=119 removed by soft count = 92\n",
      "--- gene=1405, id=121 removed by soft count = 59\n",
      "--- gene=1405, id=252 removed by soft count = 77\n",
      "--- gene=1405, id=266 removed by soft count = 64\n",
      "--- gene=1405, id=271 removed by soft count = 110\n",
      "--- gene=1405, id=275 removed by soft count = 521\n",
      "--- gene=1405, id=276 removed by soft count = 557\n",
      "--- gene=1405, id=278 removed by soft count = 73\n",
      "--- gene=1405, id=280 removed by soft count = 246\n",
      "--- gene=1405, id=282 removed by soft count = 211\n",
      "--- gene=1405, id=284 removed by soft count = 80\n",
      "--- gene=1405, id=286 removed by soft count = 127\n",
      "--- gene=1405, id=288 removed by soft count = 142\n",
      "--- gene=1405, id=289 removed by soft count = 346\n",
      "--- gene=1405, id=290 removed by soft count = 377\n",
      "--- gene=1405, id=291 removed by soft count = 67\n",
      "--- gene=1405, id=292 removed by soft count = 158\n",
      "--- gene=1405, id=295 removed by soft count = 67\n",
      "--- gene=1405, id=297 removed by soft count = 386\n",
      "--- number of probes: 300 , kept by blast: 278 , if remove dups: 278\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1406\n",
      "--- gene=1406, id=50 removed by soft count = 95\n",
      "--- gene=1406, id=51 removed by soft count = 175\n",
      "--- gene=1406, id=73 removed by soft count = 51\n",
      "--- gene=1406, id=74 removed by soft count = 92\n",
      "--- gene=1406, id=133 removed by soft count = 70\n",
      "--- gene=1406, id=138 removed by soft count = 62\n",
      "--- gene=1406, id=232 removed by soft count = 215\n",
      "--- gene=1406, id=235 removed by soft count = 350\n",
      "--- gene=1406, id=261 removed by soft count = 107\n",
      "--- number of probes: 300 , kept by blast: 291 , if remove dups: 291\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1408\n",
      "--- gene=1408, id=6 removed by soft count = 64\n",
      "--- gene=1408, id=20 removed by soft count = 63\n",
      "--- gene=1408, id=22 removed by soft count = 69\n",
      "--- gene=1408, id=25 removed by soft count = 80\n",
      "--- gene=1408, id=28 removed by soft count = 54\n",
      "--- gene=1408, id=29 removed by soft count = 74\n",
      "--- gene=1408, id=31 removed by soft count = 151\n",
      "--- gene=1408, id=32 removed by soft count = 120\n",
      "--- gene=1408, id=36 removed by soft count = 51\n",
      "--- gene=1408, id=37 removed by soft count = 641\n",
      "--- gene=1408, id=38 removed by soft count = 189\n",
      "--- gene=1408, id=54 removed by soft count = 105\n",
      "--- gene=1408, id=55 removed by soft count = 278\n",
      "--- gene=1408, id=64 removed by soft count = 58\n",
      "--- gene=1408, id=141 removed by soft count = 54\n",
      "--- number of probes: 271 , kept by blast: 256 , if remove dups: 255\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1409\n",
      "--- gene=1409, id=137 removed by soft count = 121\n",
      "--- gene=1409, id=138 removed by hard count = 0\n",
      "--- gene=1409, id=220 removed by soft count = 54\n",
      "--- number of probes: 295 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1427\n",
      "--- gene=1427, id=9 removed by soft count = 315\n",
      "--- gene=1427, id=45 removed by soft count = 243\n",
      "--- gene=1427, id=185 removed by soft count = 550\n",
      "--- gene=1427, id=228 removed by soft count = 329\n",
      "--- number of probes: 253 , kept by blast: 249 , if remove dups: 249\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1429\n",
      "--- gene=1429, id=26 removed by soft count = 119\n",
      "--- gene=1429, id=134 removed by soft count = 53\n",
      "--- number of probes: 266 , kept by blast: 264 , if remove dups: 263\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1430\n",
      "--- gene=1430, id=48 removed by soft count = 629\n",
      "--- gene=1430, id=49 removed by soft count = 54\n",
      "--- gene=1430, id=50 removed by soft count = 112\n",
      "--- gene=1430, id=52 removed by soft count = 196\n",
      "--- gene=1430, id=53 removed by soft count = 52\n",
      "--- gene=1430, id=154 removed by soft count = 58\n",
      "--- gene=1430, id=156 removed by soft count = 62\n",
      "--- number of probes: 240 , kept by blast: 233 , if remove dups: 233\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1434\n",
      "--- gene=1434, id=92 removed by soft count = 191\n",
      "--- gene=1434, id=191 removed by soft count = 58\n",
      "--- number of probes: 295 , kept by blast: 293 , if remove dups: 293\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1435\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1435, id=28 removed by soft count = 93\n",
      "--- gene=1435, id=29 removed by soft count = 119\n",
      "--- gene=1435, id=30 removed by soft count = 108\n",
      "--- number of probes: 255 , kept by blast: 252 , if remove dups: 251\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1436\n",
      "--- gene=1436, id=127 removed by soft count = 632\n",
      "--- number of probes: 298 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1437\n",
      "--- gene=1437, id=0 removed by soft count = 123\n",
      "--- gene=1437, id=69 removed by soft count = 84\n",
      "--- gene=1437, id=85 removed by soft count = 129\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1438\n",
      "--- gene=1438, id=67 removed by soft count = 108\n",
      "--- gene=1438, id=154 removed by soft count = 82\n",
      "--- gene=1438, id=155 removed by soft count = 93\n",
      "--- gene=1438, id=156 removed by soft count = 94\n",
      "--- gene=1438, id=159 removed by soft count = 148\n",
      "--- gene=1438, id=160 removed by soft count = 885\n",
      "--- gene=1438, id=161 removed by soft count = 11701\n",
      "--- gene=1438, id=162 removed by soft count = 1195\n",
      "--- number of probes: 174 , kept by blast: 166 , if remove dups: 166\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1439\n",
      "--- gene=1439, id=93 removed by soft count = 2103\n",
      "--- gene=1439, id=126 removed by soft count = 51\n",
      "--- gene=1439, id=194 removed by soft count = 53\n",
      "--- number of probes: 196 , kept by blast: 193 , if remove dups: 193\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1440\n",
      "--- gene=1440, id=188 removed by soft count = 388\n",
      "--- gene=1440, id=199 removed by soft count = 213\n",
      "--- gene=1440, id=207 removed by soft count = 2561\n",
      "--- gene=1440, id=213 removed by soft count = 98\n",
      "--- gene=1440, id=215 removed by soft count = 178\n",
      "--- number of probes: 236 , kept by blast: 231 , if remove dups: 231\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1441\n",
      "--- gene=1441, id=117 removed by soft count = 597\n",
      "--- gene=1441, id=120 removed by soft count = 276\n",
      "--- gene=1441, id=140 removed by soft count = 87\n",
      "--- gene=1441, id=149 removed by soft count = 108\n",
      "--- gene=1441, id=152 removed by soft count = 117\n",
      "--- gene=1441, id=164 removed by soft count = 113\n",
      "--- gene=1441, id=167 removed by soft count = 156\n",
      "--- number of probes: 184 , kept by blast: 177 , if remove dups: 177\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1442\n",
      "--- gene=1442, id=107 removed by soft count = 426\n",
      "--- gene=1442, id=109 removed by soft count = 249\n",
      "--- gene=1442, id=165 removed by soft count = 837\n",
      "--- gene=1442, id=166 removed by soft count = 212\n",
      "--- gene=1442, id=168 removed by soft count = 145\n",
      "--- number of probes: 248 , kept by blast: 243 , if remove dups: 243\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1443\n",
      "--- gene=1443, id=109 removed by soft count = 70\n",
      "--- gene=1443, id=293 removed by soft count = 61\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1444\n",
      "--- gene=1444, id=6 removed by soft count = 134\n",
      "--- gene=1444, id=119 removed by soft count = 261\n",
      "--- gene=1444, id=120 removed by soft count = 146\n",
      "--- gene=1444, id=124 removed by soft count = 887\n",
      "--- gene=1444, id=125 removed by soft count = 59\n",
      "--- gene=1444, id=126 removed by soft count = 196\n",
      "--- gene=1444, id=226 removed by soft count = 472\n",
      "--- gene=1444, id=227 removed by soft count = 1490\n",
      "--- number of probes: 300 , kept by blast: 292 , if remove dups: 292\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1445\n",
      "--- gene=1445, id=83 removed by soft count = 95\n",
      "--- gene=1445, id=131 removed by soft count = 47085\n",
      "--- gene=1445, id=153 removed by soft count = 693\n",
      "--- gene=1445, id=177 removed by soft count = 900\n",
      "--- gene=1445, id=203 removed by soft count = 250\n",
      "--- gene=1445, id=204 removed by soft count = 320\n",
      "--- number of probes: 296 , kept by blast: 290 , if remove dups: 290\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1446\n",
      "--- gene=1446, id=12 removed by soft count = 134\n",
      "--- gene=1446, id=19 removed by soft count = 239\n",
      "--- gene=1446, id=53 removed by soft count = 332\n",
      "--- number of probes: 274 , kept by blast: 271 , if remove dups: 271\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1447\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1448\n",
      "--- gene=1448, id=111 removed by soft count = 59\n",
      "--- gene=1448, id=228 removed by soft count = 729\n",
      "--- gene=1448, id=229 removed by soft count = 53\n",
      "--- gene=1448, id=231 removed by soft count = 52\n",
      "--- gene=1448, id=243 removed by soft count = 2794\n",
      "--- gene=1448, id=265 removed by soft count = 191\n",
      "--- gene=1448, id=266 removed by soft count = 850\n",
      "--- number of probes: 267 , kept by blast: 260 , if remove dups: 260\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1449\n",
      "--- gene=1449, id=65 removed by soft count = 53\n",
      "--- gene=1449, id=189 removed by soft count = 9384\n",
      "--- gene=1449, id=209 removed by soft count = 79\n",
      "--- number of probes: 298 , kept by blast: 295 , if remove dups: 295\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1450\n",
      "--- gene=1450, id=88 removed by soft count = 91\n",
      "--- gene=1450, id=116 removed by soft count = 388\n",
      "--- number of probes: 211 , kept by blast: 209 , if remove dups: 208\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1451\n",
      "--- gene=1451, id=61 removed by soft count = 250\n",
      "--- gene=1451, id=62 removed by soft count = 79\n",
      "--- gene=1451, id=63 removed by soft count = 153\n",
      "--- gene=1451, id=64 removed by soft count = 102\n",
      "--- gene=1451, id=120 removed by soft count = 78\n",
      "--- number of probes: 255 , kept by blast: 250 , if remove dups: 250\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1452\n",
      "--- gene=1452, id=49 removed by soft count = 227\n",
      "--- gene=1452, id=52 removed by soft count = 2181\n",
      "--- number of probes: 113 , kept by blast: 111 , if remove dups: 109\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 109\n",
      "-- checking probes in region: 1453\n",
      "--- gene=1453, id=77 removed by soft count = 300\n",
      "--- gene=1453, id=106 removed by soft count = 54\n",
      "--- gene=1453, id=107 removed by soft count = 122\n",
      "--- gene=1453, id=190 removed by soft count = 341\n",
      "--- gene=1453, id=237 removed by soft count = 52\n",
      "--- gene=1453, id=290 removed by soft count = 64\n",
      "--- number of probes: 300 , kept by blast: 294 , if remove dups: 294\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1454\n",
      "--- gene=1454, id=28 removed by soft count = 256\n",
      "--- gene=1454, id=82 removed by soft count = 51\n",
      "--- gene=1454, id=184 removed by soft count = 81\n",
      "--- gene=1454, id=185 removed by soft count = 151\n",
      "--- gene=1454, id=186 removed by soft count = 59\n",
      "--- gene=1454, id=213 removed by soft count = 52\n",
      "--- gene=1454, id=214 removed by soft count = 106\n",
      "--- gene=1454, id=219 removed by soft count = 166\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1454, id=231 removed by soft count = 59\n",
      "--- number of probes: 277 , kept by blast: 268 , if remove dups: 268\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1455\n",
      "--- gene=1455, id=2 removed by soft count = 59\n",
      "--- gene=1455, id=20 removed by soft count = 79\n",
      "--- number of probes: 290 , kept by blast: 288 , if remove dups: 288\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1456\n",
      "--- gene=1456, id=78 removed by soft count = 178\n",
      "--- gene=1456, id=83 removed by soft count = 98\n",
      "--- number of probes: 270 , kept by blast: 268 , if remove dups: 268\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1457\n",
      "--- gene=1457, id=29 removed by soft count = 409\n",
      "--- gene=1457, id=65 removed by soft count = 239\n",
      "--- gene=1457, id=162 removed by soft count = 116\n",
      "--- gene=1457, id=186 removed by soft count = 83\n",
      "--- gene=1457, id=187 removed by soft count = 115\n",
      "--- gene=1457, id=201 removed by soft count = 213\n",
      "--- gene=1457, id=202 removed by soft count = 359\n",
      "--- gene=1457, id=203 removed by soft count = 90\n",
      "--- gene=1457, id=245 removed by soft count = 111\n",
      "--- number of probes: 281 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1458\n",
      "--- gene=1458, id=235 removed by soft count = 269\n",
      "--- gene=1458, id=240 removed by soft count = 192\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1459\n",
      "--- gene=1459, id=9 removed by soft count = 254\n",
      "--- gene=1459, id=116 removed by soft count = 117\n",
      "--- gene=1459, id=203 removed by soft count = 72\n",
      "--- gene=1459, id=231 removed by soft count = 76\n",
      "--- gene=1459, id=241 removed by soft count = 88\n",
      "--- gene=1459, id=252 removed by soft count = 238\n",
      "--- gene=1459, id=255 removed by soft count = 79\n",
      "--- gene=1459, id=273 removed by soft count = 90\n",
      "--- gene=1459, id=274 removed by soft count = 139\n",
      "--- gene=1459, id=276 removed by soft count = 66\n",
      "--- gene=1459, id=277 removed by soft count = 268\n",
      "--- number of probes: 297 , kept by blast: 286 , if remove dups: 286\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1461\n",
      "--- gene=1461, id=21 removed by soft count = 453\n",
      "--- gene=1461, id=24 removed by soft count = 247\n",
      "--- gene=1461, id=38 removed by soft count = 269\n",
      "--- gene=1461, id=39 removed by soft count = 272\n",
      "--- gene=1461, id=65 removed by soft count = 267\n",
      "--- gene=1461, id=70 removed by soft count = 215\n",
      "--- gene=1461, id=78 removed by soft count = 131\n",
      "--- gene=1461, id=79 removed by soft count = 53\n",
      "--- gene=1461, id=83 removed by soft count = 72\n",
      "--- gene=1461, id=106 removed by soft count = 374\n",
      "--- gene=1461, id=114 removed by soft count = 63\n",
      "--- number of probes: 143 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1462\n",
      "--- gene=1462, id=25 removed by soft count = 84\n",
      "--- gene=1462, id=53 removed by soft count = 93\n",
      "--- gene=1462, id=188 removed by soft count = 235\n",
      "--- number of probes: 238 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1463\n",
      "--- gene=1463, id=8 removed by soft count = 101\n",
      "--- gene=1463, id=164 removed by soft count = 489\n",
      "--- gene=1463, id=210 removed by soft count = 134\n",
      "--- gene=1463, id=242 removed by soft count = 150\n",
      "--- number of probes: 268 , kept by blast: 264 , if remove dups: 263\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1464\n",
      "--- gene=1464, id=0 removed by soft count = 55\n",
      "--- gene=1464, id=22 removed by soft count = 176\n",
      "--- gene=1464, id=23 removed by soft count = 127\n",
      "--- gene=1464, id=37 removed by soft count = 62\n",
      "--- gene=1464, id=83 removed by soft count = 128\n",
      "--- gene=1464, id=119 removed by soft count = 51\n",
      "--- gene=1464, id=130 removed by soft count = 815\n",
      "--- gene=1464, id=136 removed by soft count = 71\n",
      "--- gene=1464, id=233 removed by soft count = 212\n",
      "--- number of probes: 281 , kept by blast: 272 , if remove dups: 272\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1465\n",
      "--- gene=1465, id=19 removed by soft count = 158\n",
      "--- gene=1465, id=40 removed by soft count = 141\n",
      "--- gene=1465, id=42 removed by soft count = 93\n",
      "--- number of probes: 231 , kept by blast: 228 , if remove dups: 228\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1466\n",
      "--- gene=1466, id=2 removed by soft count = 68\n",
      "--- gene=1466, id=51 removed by soft count = 60\n",
      "--- gene=1466, id=179 removed by soft count = 90\n",
      "--- number of probes: 286 , kept by blast: 283 , if remove dups: 283\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1467\n",
      "--- gene=1467, id=0 removed by soft count = 430\n",
      "--- gene=1467, id=14 removed by soft count = 135\n",
      "--- gene=1467, id=39 removed by soft count = 60\n",
      "--- gene=1467, id=95 removed by soft count = 100\n",
      "--- gene=1467, id=96 removed by soft count = 139\n",
      "--- gene=1467, id=97 removed by soft count = 68\n",
      "--- gene=1467, id=98 removed by soft count = 95\n",
      "--- gene=1467, id=100 removed by soft count = 131\n",
      "--- gene=1467, id=172 removed by soft count = 336\n",
      "--- number of probes: 193 , kept by blast: 184 , if remove dups: 184\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1468\n",
      "--- gene=1468, id=6 removed by soft count = 598\n",
      "--- gene=1468, id=7 removed by soft count = 81\n",
      "--- gene=1468, id=9 removed by soft count = 99\n",
      "--- gene=1468, id=10 removed by soft count = 114\n",
      "--- gene=1468, id=12 removed by soft count = 342\n",
      "--- gene=1468, id=15 removed by soft count = 224\n",
      "--- gene=1468, id=19 removed by soft count = 56\n",
      "--- gene=1468, id=20 removed by soft count = 324\n",
      "--- gene=1468, id=21 removed by soft count = 90\n",
      "--- gene=1468, id=22 removed by soft count = 126\n",
      "--- gene=1468, id=25 removed by soft count = 63\n",
      "--- gene=1468, id=28 removed by soft count = 149\n",
      "--- gene=1468, id=29 removed by soft count = 72\n",
      "--- gene=1468, id=32 removed by soft count = 146\n",
      "--- gene=1468, id=34 removed by soft count = 57\n",
      "--- gene=1468, id=41 removed by soft count = 81\n",
      "--- gene=1468, id=42 removed by soft count = 84\n",
      "--- gene=1468, id=47 removed by soft count = 53\n",
      "--- gene=1468, id=54 removed by soft count = 61\n",
      "--- number of probes: 180 , kept by blast: 161 , if remove dups: 161\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1469\n",
      "--- gene=1469, id=96 removed by soft count = 344\n",
      "--- number of probes: 202 , kept by blast: 201 , if remove dups: 201\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1470\n",
      "--- gene=1470, id=18 removed by soft count = 701\n",
      "--- gene=1470, id=20 removed by soft count = 1099\n",
      "--- gene=1470, id=88 removed by soft count = 386\n",
      "--- gene=1470, id=89 removed by soft count = 414\n",
      "--- gene=1470, id=90 removed by soft count = 368\n",
      "--- gene=1470, id=91 removed by soft count = 223\n",
      "--- gene=1470, id=167 removed by soft count = 86\n",
      "--- gene=1470, id=183 removed by soft count = 406\n",
      "--- number of probes: 194 , kept by blast: 186 , if remove dups: 186\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1471\n",
      "--- gene=1471, id=99 removed by soft count = 11759\n",
      "--- gene=1471, id=149 removed by soft count = 79\n",
      "--- gene=1471, id=191 removed by soft count = 77\n",
      "--- gene=1471, id=192 removed by soft count = 66\n",
      "--- number of probes: 221 , kept by blast: 217 , if remove dups: 216\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1472\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1472, id=104 removed by soft count = 72\n",
      "--- gene=1472, id=107 removed by soft count = 180\n",
      "--- gene=1472, id=108 removed by soft count = 452\n",
      "--- gene=1472, id=109 removed by soft count = 309\n",
      "--- number of probes: 278 , kept by blast: 274 , if remove dups: 274\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1473\n",
      "--- gene=1473, id=42 removed by soft count = 189\n",
      "--- gene=1473, id=44 removed by soft count = 183\n",
      "--- gene=1473, id=45 removed by soft count = 256\n",
      "--- gene=1473, id=64 removed by soft count = 195\n",
      "--- gene=1473, id=67 removed by soft count = 353\n",
      "--- gene=1473, id=70 removed by soft count = 283\n",
      "--- gene=1473, id=73 removed by soft count = 1114\n",
      "--- gene=1473, id=74 removed by soft count = 75\n",
      "--- gene=1473, id=75 removed by soft count = 234\n",
      "--- gene=1473, id=76 removed by soft count = 200\n",
      "--- gene=1473, id=77 removed by soft count = 1264\n",
      "--- gene=1473, id=79 removed by soft count = 626\n",
      "--- gene=1473, id=83 removed by soft count = 274\n",
      "--- gene=1473, id=87 removed by soft count = 474\n",
      "--- gene=1473, id=88 removed by soft count = 70\n",
      "--- gene=1473, id=89 removed by soft count = 68\n",
      "--- gene=1473, id=96 removed by soft count = 59\n",
      "--- number of probes: 117 , kept by blast: 100 , if remove dups: 100\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: 1474\n",
      "--- gene=1474, id=32 removed by soft count = 66\n",
      "--- gene=1474, id=45 removed by soft count = 60\n",
      "--- gene=1474, id=51 removed by soft count = 270\n",
      "--- gene=1474, id=53 removed by soft count = 180\n",
      "--- gene=1474, id=54 removed by soft count = 66\n",
      "--- gene=1474, id=61 removed by soft count = 117\n",
      "--- gene=1474, id=64 removed by soft count = 301\n",
      "--- gene=1474, id=158 removed by soft count = 709\n",
      "--- gene=1474, id=159 removed by soft count = 211\n",
      "--- gene=1474, id=160 removed by soft count = 127\n",
      "--- gene=1474, id=167 removed by soft count = 226\n",
      "--- gene=1474, id=170 removed by soft count = 151\n",
      "--- gene=1474, id=172 removed by soft count = 91\n",
      "--- number of probes: 173 , kept by blast: 160 , if remove dups: 160\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1475\n",
      "--- gene=1475, id=40 removed by soft count = 383\n",
      "--- gene=1475, id=41 removed by soft count = 1627\n",
      "--- gene=1475, id=43 removed by soft count = 1222\n",
      "--- gene=1475, id=44 removed by soft count = 84\n",
      "--- gene=1475, id=48 removed by soft count = 142\n",
      "--- gene=1475, id=50 removed by soft count = 329\n",
      "--- gene=1475, id=51 removed by soft count = 639\n",
      "--- gene=1475, id=52 removed by soft count = 301\n",
      "--- gene=1475, id=110 removed by soft count = 439\n",
      "--- number of probes: 234 , kept by blast: 225 , if remove dups: 225\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1476\n",
      "--- gene=1476, id=155 removed by soft count = 54\n",
      "--- gene=1476, id=184 removed by soft count = 3859\n",
      "--- gene=1476, id=186 removed by soft count = 58\n",
      "--- gene=1476, id=191 removed by soft count = 283\n",
      "--- gene=1476, id=196 removed by soft count = 105\n",
      "--- number of probes: 197 , kept by blast: 192 , if remove dups: 192\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1477\n",
      "--- gene=1477, id=144 removed by soft count = 123\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1478\n",
      "--- gene=1478, id=17 removed by soft count = 343\n",
      "--- gene=1478, id=18 removed by soft count = 51\n",
      "--- gene=1478, id=19 removed by soft count = 129\n",
      "--- gene=1478, id=26 removed by soft count = 104\n",
      "--- gene=1478, id=172 removed by soft count = 79\n",
      "--- gene=1478, id=174 removed by soft count = 472\n",
      "--- gene=1478, id=175 removed by soft count = 154\n",
      "--- gene=1478, id=177 removed by soft count = 115\n",
      "--- gene=1478, id=178 removed by soft count = 56\n",
      "--- gene=1478, id=179 removed by soft count = 58\n",
      "--- gene=1478, id=204 removed by soft count = 227\n",
      "--- gene=1478, id=211 removed by soft count = 140\n",
      "--- number of probes: 230 , kept by blast: 218 , if remove dups: 218\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1479\n",
      "--- gene=1479, id=44 removed by soft count = 57\n",
      "--- gene=1479, id=77 removed by soft count = 175\n",
      "--- gene=1479, id=98 removed by soft count = 54\n",
      "--- gene=1479, id=99 removed by soft count = 232\n",
      "--- gene=1479, id=163 removed by soft count = 260\n",
      "--- gene=1479, id=164 removed by soft count = 185\n",
      "--- gene=1479, id=165 removed by soft count = 157\n",
      "--- gene=1479, id=166 removed by soft count = 60\n",
      "--- gene=1479, id=171 removed by soft count = 340\n",
      "--- gene=1479, id=173 removed by soft count = 158\n",
      "--- gene=1479, id=183 removed by soft count = 352\n",
      "--- number of probes: 216 , kept by blast: 205 , if remove dups: 205\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1480\n",
      "--- gene=1480, id=12 removed by soft count = 64\n",
      "--- gene=1480, id=24 removed by soft count = 77\n",
      "--- gene=1480, id=28 removed by soft count = 64\n",
      "--- gene=1480, id=31 removed by soft count = 162\n",
      "--- gene=1480, id=32 removed by soft count = 348\n",
      "--- gene=1480, id=33 removed by soft count = 51\n",
      "--- gene=1480, id=36 removed by soft count = 256\n",
      "--- gene=1480, id=37 removed by soft count = 85\n",
      "--- gene=1480, id=38 removed by soft count = 96\n",
      "--- gene=1480, id=40 removed by soft count = 54\n",
      "--- gene=1480, id=56 removed by soft count = 207\n",
      "--- gene=1480, id=62 removed by soft count = 82\n",
      "--- gene=1480, id=68 removed by soft count = 75\n",
      "--- number of probes: 117 , kept by blast: 104 , if remove dups: 104\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 104\n",
      "-- checking probes in region: 1481\n",
      "--- gene=1481, id=59 removed by soft count = 83\n",
      "--- gene=1481, id=82 removed by soft count = 500\n",
      "--- gene=1481, id=84 removed by soft count = 92\n",
      "--- gene=1481, id=92 removed by soft count = 123\n",
      "--- gene=1481, id=93 removed by soft count = 70\n",
      "--- number of probes: 198 , kept by blast: 193 , if remove dups: 192\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1482\n",
      "--- gene=1482, id=66 removed by soft count = 58\n",
      "--- gene=1482, id=67 removed by soft count = 72\n",
      "--- gene=1482, id=68 removed by soft count = 66\n",
      "--- gene=1482, id=69 removed by soft count = 61\n",
      "--- gene=1482, id=109 removed by soft count = 287\n",
      "--- gene=1482, id=110 removed by soft count = 244\n",
      "--- gene=1482, id=136 removed by soft count = 344\n",
      "--- gene=1482, id=139 removed by soft count = 173\n",
      "--- gene=1482, id=146 removed by soft count = 533\n",
      "--- gene=1482, id=147 removed by soft count = 164\n",
      "--- gene=1482, id=148 removed by soft count = 174\n",
      "--- gene=1482, id=149 removed by soft count = 89\n",
      "--- number of probes: 212 , kept by blast: 200 , if remove dups: 200\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1485\n",
      "--- gene=1485, id=293 removed by soft count = 309\n",
      "--- gene=1485, id=294 removed by soft count = 253\n",
      "--- number of probes: 300 , kept by blast: 298 , if remove dups: 298\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1486\n",
      "--- gene=1486, id=1 removed by soft count = 259\n",
      "--- gene=1486, id=2 removed by soft count = 1141\n",
      "--- gene=1486, id=3 removed by soft count = 332\n",
      "--- gene=1486, id=5 removed by soft count = 74\n",
      "--- gene=1486, id=6 removed by soft count = 99\n",
      "--- gene=1486, id=7 removed by soft count = 335\n",
      "--- gene=1486, id=9 removed by soft count = 126\n",
      "--- gene=1486, id=21 removed by soft count = 169\n",
      "--- gene=1486, id=24 removed by soft count = 129\n",
      "--- gene=1486, id=25 removed by soft count = 75\n",
      "--- gene=1486, id=26 removed by soft count = 230\n",
      "--- gene=1486, id=27 removed by soft count = 88\n",
      "--- gene=1486, id=31 removed by soft count = 1933\n",
      "--- gene=1486, id=32 removed by soft count = 672\n",
      "--- gene=1486, id=33 removed by soft count = 72\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1486, id=34 removed by soft count = 276\n",
      "--- gene=1486, id=35 removed by soft count = 129\n",
      "--- gene=1486, id=37 removed by soft count = 52\n",
      "--- gene=1486, id=51 removed by soft count = 82\n",
      "--- gene=1486, id=62 removed by soft count = 128\n",
      "--- gene=1486, id=87 removed by soft count = 199\n",
      "--- number of probes: 227 , kept by blast: 206 , if remove dups: 206\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1487\n",
      "--- gene=1487, id=2 removed by soft count = 469\n",
      "--- gene=1487, id=101 removed by soft count = 216\n",
      "--- gene=1487, id=249 removed by soft count = 260\n",
      "--- number of probes: 250 , kept by blast: 247 , if remove dups: 247\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1488\n",
      "--- gene=1488, id=28 removed by soft count = 656\n",
      "--- gene=1488, id=279 removed by soft count = 8283\n",
      "--- gene=1488, id=280 removed by soft count = 189\n",
      "--- number of probes: 300 , kept by blast: 297 , if remove dups: 297\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1489\n",
      "--- gene=1489, id=0 removed by soft count = 229\n",
      "--- gene=1489, id=39 removed by soft count = 171\n",
      "--- gene=1489, id=40 removed by soft count = 158\n",
      "--- gene=1489, id=42 removed by soft count = 276\n",
      "--- gene=1489, id=44 removed by soft count = 56\n",
      "--- gene=1489, id=47 removed by soft count = 781\n",
      "--- gene=1489, id=49 removed by soft count = 149\n",
      "--- gene=1489, id=50 removed by soft count = 185\n",
      "--- gene=1489, id=52 removed by soft count = 112\n",
      "--- gene=1489, id=53 removed by soft count = 120\n",
      "--- gene=1489, id=54 removed by soft count = 77\n",
      "--- gene=1489, id=56 removed by soft count = 217\n",
      "--- gene=1489, id=57 removed by soft count = 267\n",
      "--- gene=1489, id=69 removed by soft count = 224\n",
      "--- gene=1489, id=71 removed by soft count = 91\n",
      "--- gene=1489, id=77 removed by soft count = 56\n",
      "--- gene=1489, id=83 removed by soft count = 201\n",
      "--- gene=1489, id=85 removed by soft count = 331\n",
      "--- gene=1489, id=86 removed by soft count = 13652\n",
      "--- gene=1489, id=88 removed by soft count = 92\n",
      "--- gene=1489, id=91 removed by soft count = 56\n",
      "--- number of probes: 94 , kept by blast: 73 , if remove dups: 73\n",
      "-- number of probes kept for this region: 73\n",
      "-- checking probes in region: 1490\n",
      "--- gene=1490, id=1 removed by soft count = 106\n",
      "--- gene=1490, id=2 removed by soft count = 14436\n",
      "--- gene=1490, id=6 removed by soft count = 587\n",
      "--- gene=1490, id=7 removed by soft count = 135\n",
      "--- gene=1490, id=10 removed by soft count = 257\n",
      "--- gene=1490, id=12 removed by soft count = 113\n",
      "--- gene=1490, id=13 removed by soft count = 373\n",
      "--- gene=1490, id=19 removed by soft count = 55\n",
      "--- gene=1490, id=41 removed by soft count = 72\n",
      "--- gene=1490, id=93 removed by soft count = 100\n",
      "--- gene=1490, id=121 removed by soft count = 115\n",
      "--- gene=1490, id=123 removed by soft count = 120\n",
      "--- gene=1490, id=125 removed by soft count = 194\n",
      "--- gene=1490, id=126 removed by soft count = 217\n",
      "--- gene=1490, id=127 removed by soft count = 88\n",
      "--- number of probes: 128 , kept by blast: 113 , if remove dups: 113\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 113\n",
      "-- checking probes in region: 1491\n",
      "--- gene=1491, id=8 removed by soft count = 89\n",
      "--- gene=1491, id=11 removed by soft count = 80\n",
      "--- gene=1491, id=12 removed by soft count = 60\n",
      "--- gene=1491, id=17 removed by soft count = 62\n",
      "--- gene=1491, id=20 removed by soft count = 612\n",
      "--- gene=1491, id=23 removed by soft count = 51\n",
      "--- gene=1491, id=24 removed by soft count = 66\n",
      "--- gene=1491, id=28 removed by soft count = 104\n",
      "--- gene=1491, id=31 removed by soft count = 165\n",
      "--- gene=1491, id=32 removed by soft count = 124\n",
      "--- gene=1491, id=226 removed by soft count = 52\n",
      "--- gene=1491, id=228 removed by soft count = 52\n",
      "--- number of probes: 239 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1492\n",
      "--- gene=1492, id=26 removed by soft count = 139\n",
      "--- gene=1492, id=28 removed by soft count = 62\n",
      "--- gene=1492, id=30 removed by soft count = 54\n",
      "--- gene=1492, id=32 removed by soft count = 117\n",
      "--- gene=1492, id=34 removed by soft count = 104\n",
      "--- gene=1492, id=35 removed by soft count = 257\n",
      "--- gene=1492, id=37 removed by soft count = 53\n",
      "--- gene=1492, id=39 removed by soft count = 1073\n",
      "--- gene=1492, id=41 removed by soft count = 440\n",
      "--- gene=1492, id=46 removed by soft count = 132\n",
      "--- gene=1492, id=47 removed by soft count = 257\n",
      "--- gene=1492, id=48 removed by soft count = 66\n",
      "--- gene=1492, id=49 removed by soft count = 117\n",
      "--- gene=1492, id=52 removed by soft count = 75\n",
      "--- gene=1492, id=57 removed by soft count = 1588\n",
      "--- gene=1492, id=59 removed by soft count = 138\n",
      "--- gene=1492, id=61 removed by soft count = 57\n",
      "--- gene=1492, id=79 removed by soft count = 66\n",
      "--- gene=1492, id=80 removed by soft count = 69\n",
      "--- gene=1492, id=82 removed by soft count = 72\n",
      "--- gene=1492, id=83 removed by soft count = 201\n",
      "--- gene=1492, id=93 removed by soft count = 79\n",
      "--- gene=1492, id=152 removed by soft count = 91\n",
      "--- number of probes: 238 , kept by blast: 215 , if remove dups: 215\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1493\n",
      "--- gene=1493, id=0 removed by soft count = 546\n",
      "--- gene=1493, id=1 removed by soft count = 278\n",
      "--- gene=1493, id=10 removed by soft count = 58\n",
      "--- gene=1493, id=13 removed by soft count = 248\n",
      "--- gene=1493, id=14 removed by soft count = 189\n",
      "--- gene=1493, id=17 removed by soft count = 223\n",
      "--- gene=1493, id=24 removed by soft count = 133\n",
      "--- gene=1493, id=25 removed by soft count = 83\n",
      "--- gene=1493, id=26 removed by soft count = 652\n",
      "--- gene=1493, id=28 removed by soft count = 109\n",
      "--- gene=1493, id=29 removed by soft count = 86\n",
      "--- gene=1493, id=30 removed by soft count = 93\n",
      "--- gene=1493, id=32 removed by soft count = 119\n",
      "--- gene=1493, id=37 removed by soft count = 81\n",
      "--- gene=1493, id=55 removed by soft count = 185\n",
      "--- number of probes: 129 , kept by blast: 114 , if remove dups: 114\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 114\n",
      "-- checking probes in region: 1494\n",
      "--- gene=1494, id=10 removed by soft count = 1026\n",
      "--- gene=1494, id=12 removed by soft count = 64\n",
      "--- gene=1494, id=13 removed by soft count = 191\n",
      "--- gene=1494, id=17 removed by soft count = 109\n",
      "--- gene=1494, id=37 removed by soft count = 313\n",
      "--- gene=1494, id=38 removed by soft count = 2134\n",
      "--- gene=1494, id=39 removed by soft count = 430\n",
      "--- gene=1494, id=40 removed by soft count = 468\n",
      "--- gene=1494, id=60 removed by soft count = 99\n",
      "--- gene=1494, id=62 removed by soft count = 148\n",
      "--- gene=1494, id=63 removed by soft count = 56\n",
      "--- gene=1494, id=64 removed by soft count = 208\n",
      "--- gene=1494, id=65 removed by soft count = 102\n",
      "--- gene=1494, id=67 removed by soft count = 620\n",
      "--- gene=1494, id=68 removed by soft count = 1947\n",
      "--- gene=1494, id=70 removed by soft count = 116\n",
      "--- gene=1494, id=71 removed by soft count = 123\n",
      "--- gene=1494, id=81 removed by soft count = 2077\n",
      "--- number of probes: 189 , kept by blast: 171 , if remove dups: 171\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1495\n",
      "--- gene=1495, id=0 removed by soft count = 86\n",
      "--- gene=1495, id=13 removed by soft count = 59\n",
      "--- gene=1495, id=35 removed by soft count = 129\n",
      "--- gene=1495, id=36 removed by soft count = 138\n",
      "--- gene=1495, id=40 removed by soft count = 53\n",
      "--- gene=1495, id=181 removed by soft count = 183\n",
      "--- gene=1495, id=183 removed by soft count = 207\n",
      "--- gene=1495, id=187 removed by soft count = 69\n",
      "--- number of probes: 199 , kept by blast: 191 , if remove dups: 191\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1496\n",
      "--- gene=1496, id=7 removed by soft count = 141\n",
      "--- gene=1496, id=8 removed by soft count = 367\n",
      "--- gene=1496, id=9 removed by soft count = 63\n",
      "--- gene=1496, id=10 removed by soft count = 96\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1496, id=19 removed by soft count = 273\n",
      "--- gene=1496, id=36 removed by soft count = 67\n",
      "--- gene=1496, id=61 removed by soft count = 69\n",
      "--- gene=1496, id=65 removed by soft count = 63\n",
      "--- gene=1496, id=68 removed by soft count = 109\n",
      "--- number of probes: 72 , kept by blast: 63 , if remove dups: 62\n",
      "-- number of probes kept for this region: 63\n",
      "-- checking probes in region: 1497\n",
      "--- gene=1497, id=0 removed by soft count = 916\n",
      "--- gene=1497, id=1 removed by soft count = 111\n",
      "--- gene=1497, id=2 removed by soft count = 148\n",
      "--- gene=1497, id=3 removed by soft count = 353\n",
      "--- gene=1497, id=4 removed by soft count = 107\n",
      "--- gene=1497, id=6 removed by soft count = 328\n",
      "--- gene=1497, id=7 removed by soft count = 255\n",
      "--- gene=1497, id=10 removed by soft count = 54\n",
      "--- gene=1497, id=35 removed by soft count = 477\n",
      "--- gene=1497, id=41 removed by soft count = 58\n",
      "--- gene=1497, id=47 removed by soft count = 52\n",
      "--- gene=1497, id=185 removed by soft count = 152\n",
      "--- gene=1497, id=208 removed by soft count = 110\n",
      "--- gene=1497, id=211 removed by soft count = 77\n",
      "--- gene=1497, id=215 removed by soft count = 52\n",
      "--- number of probes: 230 , kept by blast: 215 , if remove dups: 215\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1498\n",
      "--- gene=1498, id=15 removed by soft count = 754\n",
      "--- gene=1498, id=42 removed by soft count = 117\n",
      "--- gene=1498, id=51 removed by soft count = 63\n",
      "--- gene=1498, id=116 removed by soft count = 131\n",
      "--- gene=1498, id=125 removed by soft count = 17446\n",
      "--- gene=1498, id=134 removed by soft count = 356\n",
      "--- gene=1498, id=135 removed by soft count = 78\n",
      "--- number of probes: 146 , kept by blast: 139 , if remove dups: 139\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1499\n",
      "--- gene=1499, id=34 removed by soft count = 137\n",
      "--- gene=1499, id=74 removed by soft count = 241\n",
      "--- gene=1499, id=75 removed by soft count = 346\n",
      "--- gene=1499, id=77 removed by soft count = 93\n",
      "--- gene=1499, id=78 removed by soft count = 197\n",
      "--- gene=1499, id=80 removed by soft count = 180\n",
      "--- gene=1499, id=81 removed by soft count = 110\n",
      "--- gene=1499, id=83 removed by soft count = 74\n",
      "--- gene=1499, id=89 removed by soft count = 1209\n",
      "--- gene=1499, id=90 removed by soft count = 68\n",
      "--- gene=1499, id=91 removed by soft count = 56\n",
      "--- gene=1499, id=93 removed by soft count = 131\n",
      "--- gene=1499, id=95 removed by soft count = 255\n",
      "--- gene=1499, id=97 removed by soft count = 80\n",
      "--- gene=1499, id=101 removed by soft count = 102\n",
      "--- gene=1499, id=102 removed by soft count = 188\n",
      "--- gene=1499, id=103 removed by soft count = 123\n",
      "--- gene=1499, id=105 removed by soft count = 105\n",
      "--- gene=1499, id=144 removed by soft count = 217\n",
      "--- gene=1499, id=146 removed by soft count = 365\n",
      "--- gene=1499, id=153 removed by soft count = 6628\n",
      "--- gene=1499, id=154 removed by soft count = 383\n",
      "--- gene=1499, id=155 removed by soft count = 199\n",
      "--- number of probes: 160 , kept by blast: 137 , if remove dups: 137\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1500\n",
      "--- gene=1500, id=199 removed by soft count = 110\n",
      "--- number of probes: 300 , kept by blast: 299 , if remove dups: 299\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1501\n",
      "--- gene=1501, id=70 removed by soft count = 166\n",
      "--- gene=1501, id=71 removed by soft count = 71\n",
      "--- gene=1501, id=72 removed by soft count = 63\n",
      "--- gene=1501, id=134 removed by soft count = 904\n",
      "--- gene=1501, id=141 removed by soft count = 81\n",
      "--- gene=1501, id=218 removed by soft count = 107\n",
      "--- gene=1501, id=224 removed by soft count = 323\n",
      "--- gene=1501, id=225 removed by soft count = 99\n",
      "--- gene=1501, id=226 removed by soft count = 263\n",
      "--- gene=1501, id=232 removed by soft count = 325\n",
      "--- gene=1501, id=249 removed by soft count = 180\n",
      "--- gene=1501, id=250 removed by soft count = 839\n",
      "--- gene=1501, id=251 removed by soft count = 1001\n",
      "--- gene=1501, id=252 removed by soft count = 239\n",
      "--- number of probes: 253 , kept by blast: 239 , if remove dups: 239\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1502\n",
      "--- gene=1502, id=0 removed by soft count = 276\n",
      "--- gene=1502, id=2 removed by soft count = 320\n",
      "--- gene=1502, id=80 removed by soft count = 122\n",
      "--- gene=1502, id=81 removed by soft count = 109\n",
      "--- gene=1502, id=88 removed by soft count = 303\n",
      "--- gene=1502, id=90 removed by soft count = 185\n",
      "--- gene=1502, id=94 removed by soft count = 106\n",
      "--- gene=1502, id=99 removed by soft count = 641\n",
      "--- gene=1502, id=100 removed by soft count = 254\n",
      "--- gene=1502, id=101 removed by soft count = 58\n",
      "--- gene=1502, id=105 removed by soft count = 106\n",
      "--- gene=1502, id=106 removed by soft count = 119\n",
      "--- gene=1502, id=107 removed by soft count = 83\n",
      "--- gene=1502, id=108 removed by soft count = 98\n",
      "--- gene=1502, id=109 removed by soft count = 59\n",
      "--- gene=1502, id=112 removed by soft count = 81\n",
      "--- gene=1502, id=113 removed by soft count = 153\n",
      "--- gene=1502, id=114 removed by soft count = 200\n",
      "--- gene=1502, id=115 removed by soft count = 284\n",
      "--- gene=1502, id=116 removed by soft count = 122\n",
      "--- gene=1502, id=120 removed by soft count = 60\n",
      "--- gene=1502, id=122 removed by soft count = 118\n",
      "--- gene=1502, id=128 removed by soft count = 108\n",
      "--- gene=1502, id=130 removed by soft count = 59\n",
      "--- gene=1502, id=131 removed by soft count = 60\n",
      "--- gene=1502, id=132 removed by soft count = 68\n",
      "--- gene=1502, id=136 removed by soft count = 134\n",
      "--- gene=1502, id=138 removed by soft count = 268\n",
      "--- number of probes: 139 , kept by blast: 111 , if remove dups: 110\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 110\n",
      "-- checking probes in region: 1503\n",
      "--- gene=1503, id=121 removed by soft count = 87\n",
      "--- gene=1503, id=122 removed by soft count = 60\n",
      "--- gene=1503, id=146 removed by soft count = 105\n",
      "--- gene=1503, id=147 removed by soft count = 313\n",
      "--- gene=1503, id=148 removed by soft count = 199\n",
      "--- gene=1503, id=150 removed by soft count = 81\n",
      "--- gene=1503, id=151 removed by soft count = 262\n",
      "--- gene=1503, id=169 removed by soft count = 180\n",
      "--- gene=1503, id=171 removed by soft count = 103\n",
      "--- gene=1503, id=177 removed by soft count = 57\n",
      "--- number of probes: 186 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1504\n",
      "--- gene=1504, id=5 removed by soft count = 218\n",
      "--- gene=1504, id=11 removed by soft count = 266\n",
      "--- gene=1504, id=13 removed by soft count = 139\n",
      "--- gene=1504, id=14 removed by soft count = 190\n",
      "--- gene=1504, id=15 removed by soft count = 158\n",
      "--- number of probes: 285 , kept by blast: 280 , if remove dups: 280\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1505\n",
      "--- gene=1505, id=0 removed by soft count = 270\n",
      "--- gene=1505, id=1 removed by soft count = 510\n",
      "--- gene=1505, id=2 removed by soft count = 662\n",
      "--- gene=1505, id=3 removed by soft count = 462\n",
      "--- gene=1505, id=156 removed by soft count = 61\n",
      "--- gene=1505, id=229 removed by soft count = 143\n",
      "--- gene=1505, id=230 removed by soft count = 149\n",
      "--- gene=1505, id=231 removed by soft count = 402\n",
      "--- gene=1505, id=232 removed by soft count = 401\n",
      "--- gene=1505, id=233 removed by soft count = 239\n",
      "--- gene=1505, id=234 removed by soft count = 396\n",
      "--- gene=1505, id=235 removed by soft count = 180\n",
      "--- gene=1505, id=236 removed by soft count = 227\n",
      "--- gene=1505, id=237 removed by soft count = 132\n",
      "--- gene=1505, id=238 removed by soft count = 67\n",
      "--- number of probes: 239 , kept by blast: 224 , if remove dups: 224\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1506\n",
      "--- gene=1506, id=0 removed by soft count = 77\n",
      "--- gene=1506, id=78 removed by soft count = 57\n",
      "--- gene=1506, id=80 removed by soft count = 352\n",
      "--- gene=1506, id=81 removed by soft count = 177\n",
      "--- gene=1506, id=82 removed by soft count = 62\n",
      "--- gene=1506, id=83 removed by soft count = 77\n",
      "--- gene=1506, id=84 removed by soft count = 80\n",
      "--- gene=1506, id=85 removed by soft count = 54\n",
      "--- gene=1506, id=86 removed by soft count = 63\n",
      "--- gene=1506, id=87 removed by soft count = 75\n",
      "--- gene=1506, id=88 removed by soft count = 81\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1506, id=89 removed by soft count = 96\n",
      "--- gene=1506, id=100 removed by soft count = 5297\n",
      "--- gene=1506, id=152 removed by soft count = 78\n",
      "--- gene=1506, id=154 removed by soft count = 302\n",
      "--- number of probes: 162 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1507\n",
      "--- gene=1507, id=0 removed by soft count = 125\n",
      "--- gene=1507, id=1 removed by soft count = 61\n",
      "--- gene=1507, id=18 removed by soft count = 88\n",
      "--- gene=1507, id=24 removed by soft count = 83\n",
      "--- gene=1507, id=153 removed by soft count = 140\n",
      "--- number of probes: 242 , kept by blast: 237 , if remove dups: 237\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1508\n",
      "--- gene=1508, id=0 removed by soft count = 261\n",
      "--- gene=1508, id=1 removed by soft count = 65\n",
      "--- gene=1508, id=39 removed by soft count = 199\n",
      "--- gene=1508, id=89 removed by soft count = 354\n",
      "--- gene=1508, id=90 removed by soft count = 219\n",
      "--- gene=1508, id=91 removed by soft count = 133\n",
      "--- gene=1508, id=93 removed by soft count = 1037\n",
      "--- gene=1508, id=99 removed by soft count = 189\n",
      "--- gene=1508, id=102 removed by soft count = 176\n",
      "--- gene=1508, id=103 removed by soft count = 181\n",
      "--- gene=1508, id=105 removed by soft count = 150\n",
      "--- gene=1508, id=106 removed by soft count = 134\n",
      "--- gene=1508, id=188 removed by soft count = 68\n",
      "--- number of probes: 189 , kept by blast: 176 , if remove dups: 176\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1509\n",
      "--- gene=1509, id=49 removed by soft count = 84\n",
      "--- gene=1509, id=164 removed by soft count = 401\n",
      "--- gene=1509, id=165 removed by soft count = 1161\n",
      "--- gene=1509, id=166 removed by soft count = 89\n",
      "--- gene=1509, id=167 removed by soft count = 159\n",
      "--- gene=1509, id=181 removed by soft count = 51\n",
      "--- gene=1509, id=182 removed by soft count = 95\n",
      "--- number of probes: 188 , kept by blast: 181 , if remove dups: 181\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1510\n",
      "--- gene=1510, id=22 removed by soft count = 53\n",
      "--- gene=1510, id=67 removed by soft count = 306\n",
      "--- gene=1510, id=109 removed by soft count = 68\n",
      "--- gene=1510, id=112 removed by soft count = 202\n",
      "--- gene=1510, id=113 removed by soft count = 8952\n",
      "--- gene=1510, id=114 removed by soft count = 1347\n",
      "--- gene=1510, id=116 removed by soft count = 57\n",
      "--- gene=1510, id=119 removed by soft count = 122\n",
      "--- gene=1510, id=135 removed by soft count = 72\n",
      "--- gene=1510, id=136 removed by soft count = 51\n",
      "--- gene=1510, id=167 removed by soft count = 414\n",
      "--- gene=1510, id=169 removed by soft count = 393\n",
      "--- gene=1510, id=170 removed by soft count = 69\n",
      "--- gene=1510, id=171 removed by soft count = 124\n",
      "--- gene=1510, id=172 removed by soft count = 112\n",
      "--- gene=1510, id=173 removed by soft count = 785\n",
      "--- gene=1510, id=174 removed by soft count = 89\n",
      "--- gene=1510, id=175 removed by soft count = 95\n",
      "--- number of probes: 176 , kept by blast: 158 , if remove dups: 158\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1511\n",
      "--- gene=1511, id=62 removed by soft count = 60\n",
      "--- gene=1511, id=63 removed by soft count = 109\n",
      "--- gene=1511, id=64 removed by soft count = 129\n",
      "--- gene=1511, id=112 removed by soft count = 121\n",
      "--- gene=1511, id=115 removed by soft count = 202\n",
      "--- gene=1511, id=157 removed by soft count = 302\n",
      "--- gene=1511, id=177 removed by soft count = 59\n",
      "--- number of probes: 218 , kept by blast: 211 , if remove dups: 211\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1512\n",
      "--- number of probes: 248 , kept by blast: 248 , if remove dups: 248\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1513\n",
      "--- gene=1513, id=0 removed by soft count = 285\n",
      "--- gene=1513, id=2 removed by soft count = 218\n",
      "--- gene=1513, id=3 removed by soft count = 56\n",
      "--- gene=1513, id=209 removed by soft count = 52\n",
      "--- gene=1513, id=232 removed by soft count = 56\n",
      "--- gene=1513, id=236 removed by soft count = 98\n",
      "--- number of probes: 245 , kept by blast: 239 , if remove dups: 239\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1514\n",
      "--- gene=1514, id=11 removed by soft count = 65\n",
      "--- number of probes: 262 , kept by blast: 261 , if remove dups: 261\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1515\n",
      "--- number of probes: 300 , kept by blast: 300 , if remove dups: 300\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1516\n",
      "--- gene=1516, id=3 removed by soft count = 79\n",
      "--- gene=1516, id=4 removed by soft count = 59\n",
      "--- gene=1516, id=6 removed by soft count = 300\n",
      "--- number of probes: 103 , kept by blast: 100 , if remove dups: 100\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: 1517\n",
      "--- gene=1517, id=31 removed by soft count = 81\n",
      "--- gene=1517, id=35 removed by soft count = 153\n",
      "--- gene=1517, id=50 removed by soft count = 107\n",
      "--- gene=1517, id=189 removed by soft count = 57\n",
      "--- gene=1517, id=194 removed by soft count = 235\n",
      "--- number of probes: 232 , kept by blast: 227 , if remove dups: 227\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1518\n",
      "--- gene=1518, id=67 removed by soft count = 961\n",
      "--- gene=1518, id=68 removed by soft count = 118\n",
      "--- gene=1518, id=108 removed by soft count = 84\n",
      "--- gene=1518, id=109 removed by soft count = 59\n",
      "--- gene=1518, id=111 removed by soft count = 315\n",
      "--- gene=1518, id=113 removed by soft count = 58\n",
      "--- gene=1518, id=114 removed by soft count = 85\n",
      "--- gene=1518, id=115 removed by soft count = 156\n",
      "--- gene=1518, id=116 removed by soft count = 65\n",
      "--- gene=1518, id=120 removed by soft count = 76\n",
      "--- gene=1518, id=122 removed by soft count = 296\n",
      "--- gene=1518, id=123 removed by soft count = 59\n",
      "--- gene=1518, id=124 removed by soft count = 236\n",
      "--- gene=1518, id=126 removed by soft count = 189\n",
      "--- gene=1518, id=129 removed by soft count = 141\n",
      "--- gene=1518, id=130 removed by soft count = 131\n",
      "--- gene=1518, id=131 removed by soft count = 80\n",
      "--- gene=1518, id=133 removed by soft count = 226\n",
      "--- gene=1518, id=135 removed by soft count = 59\n",
      "--- gene=1518, id=138 removed by soft count = 685\n",
      "--- gene=1518, id=140 removed by soft count = 342\n",
      "--- gene=1518, id=145 removed by soft count = 844\n",
      "--- gene=1518, id=146 removed by soft count = 150\n",
      "--- gene=1518, id=147 removed by soft count = 470\n",
      "--- gene=1518, id=148 removed by soft count = 64\n",
      "--- gene=1518, id=150 removed by soft count = 57\n",
      "--- gene=1518, id=151 removed by soft count = 777\n",
      "--- gene=1518, id=153 removed by soft count = 1064\n",
      "--- number of probes: 217 , kept by blast: 189 , if remove dups: 189\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1519\n",
      "--- gene=1519, id=6 removed by soft count = 336\n",
      "--- gene=1519, id=8 removed by soft count = 324\n",
      "--- gene=1519, id=9 removed by soft count = 78\n",
      "--- gene=1519, id=10 removed by soft count = 127\n",
      "--- number of probes: 239 , kept by blast: 235 , if remove dups: 235\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1520\n",
      "--- gene=1520, id=77 removed by soft count = 461\n",
      "--- gene=1520, id=163 removed by soft count = 108\n",
      "--- gene=1520, id=164 removed by soft count = 656\n",
      "--- gene=1520, id=165 removed by soft count = 251\n",
      "--- gene=1520, id=166 removed by soft count = 498\n",
      "--- gene=1520, id=167 removed by soft count = 185\n",
      "--- number of probes: 221 , kept by blast: 215 , if remove dups: 215\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1521\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1521, id=6 removed by soft count = 83\n",
      "--- gene=1521, id=48 removed by soft count = 68\n",
      "--- gene=1521, id=53 removed by soft count = 72\n",
      "--- gene=1521, id=116 removed by soft count = 291\n",
      "--- gene=1521, id=117 removed by soft count = 263\n",
      "--- gene=1521, id=122 removed by soft count = 192\n",
      "--- gene=1521, id=126 removed by soft count = 71\n",
      "--- number of probes: 176 , kept by blast: 169 , if remove dups: 169\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1525\n",
      "--- gene=1525, id=9 removed by soft count = 52\n",
      "--- gene=1525, id=12 removed by soft count = 411\n",
      "--- gene=1525, id=13 removed by soft count = 60\n",
      "--- gene=1525, id=45 removed by soft count = 148\n",
      "--- number of probes: 51 , kept by blast: 47 , if remove dups: 45\n",
      "-- number of probes kept for this region: 47\n",
      "-- checking probes in region: 1526\n",
      "--- gene=1526, id=31 removed by soft count = 80\n",
      "--- gene=1526, id=32 removed by soft count = 573\n",
      "--- gene=1526, id=45 removed by soft count = 56\n",
      "--- gene=1526, id=52 removed by soft count = 1147\n",
      "--- gene=1526, id=56 removed by soft count = 78\n",
      "--- number of probes: 60 , kept by blast: 55 , if remove dups: 54\n",
      "-- number of probes kept for this region: 55\n",
      "-- checking probes in region: 1527\n",
      "--- gene=1527, id=6 removed by soft count = 135\n",
      "--- gene=1527, id=11 removed by soft count = 101\n",
      "--- gene=1527, id=12 removed by soft count = 131\n",
      "--- gene=1527, id=32 removed by soft count = 459\n",
      "--- number of probes: 72 , kept by blast: 68 , if remove dups: 63\n",
      "-- number of probes kept for this region: 68\n",
      "-- checking probes in region: 1529\n",
      "--- gene=1529, id=13 removed by soft count = 105\n",
      "--- gene=1529, id=14 removed by soft count = 183\n",
      "--- gene=1529, id=40 removed by soft count = 727\n",
      "--- gene=1529, id=41 removed by soft count = 67\n",
      "--- gene=1529, id=89 removed by soft count = 5672\n",
      "--- gene=1529, id=90 removed by soft count = 409\n",
      "--- gene=1529, id=97 removed by soft count = 398\n",
      "--- gene=1529, id=101 removed by soft count = 52\n",
      "--- gene=1529, id=103 removed by soft count = 83\n",
      "--- gene=1529, id=104 removed by soft count = 56\n",
      "--- gene=1529, id=107 removed by soft count = 1831\n",
      "--- gene=1529, id=108 removed by soft count = 204\n",
      "--- gene=1529, id=110 removed by soft count = 90\n",
      "--- gene=1529, id=111 removed by soft count = 214\n",
      "--- gene=1529, id=112 removed by soft count = 422\n",
      "--- gene=1529, id=113 removed by soft count = 84\n",
      "--- gene=1529, id=114 removed by soft count = 51\n",
      "--- gene=1529, id=115 removed by soft count = 58\n",
      "--- gene=1529, id=116 removed by soft count = 159\n",
      "--- gene=1529, id=118 removed by soft count = 251\n",
      "--- gene=1529, id=120 removed by soft count = 781\n",
      "--- gene=1529, id=121 removed by soft count = 159\n",
      "--- gene=1529, id=122 removed by soft count = 201\n",
      "--- gene=1529, id=123 removed by soft count = 941\n",
      "--- gene=1529, id=124 removed by soft count = 247\n",
      "--- gene=1529, id=125 removed by soft count = 77\n",
      "--- gene=1529, id=126 removed by soft count = 154\n",
      "--- gene=1529, id=127 removed by soft count = 607\n",
      "--- gene=1529, id=128 removed by soft count = 238\n",
      "--- gene=1529, id=129 removed by soft count = 91\n",
      "--- gene=1529, id=130 removed by soft count = 165\n",
      "--- gene=1529, id=151 removed by soft count = 12629\n",
      "--- gene=1529, id=152 removed by soft count = 112\n",
      "--- gene=1529, id=154 removed by soft count = 744\n",
      "--- number of probes: 166 , kept by blast: 132 , if remove dups: 132\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1530\n",
      "--- gene=1530, id=1 removed by soft count = 487\n",
      "--- gene=1530, id=14 removed by soft count = 67\n",
      "--- gene=1530, id=36 removed by soft count = 57\n",
      "--- gene=1530, id=57 removed by soft count = 55\n",
      "--- gene=1530, id=62 removed by soft count = 70\n",
      "--- gene=1530, id=100 removed by soft count = 70\n",
      "--- gene=1530, id=103 removed by soft count = 74\n",
      "--- gene=1530, id=134 removed by soft count = 97\n",
      "--- gene=1530, id=152 removed by soft count = 15792\n",
      "--- gene=1530, id=187 removed by soft count = 51\n",
      "--- gene=1530, id=188 removed by soft count = 443\n",
      "--- number of probes: 218 , kept by blast: 207 , if remove dups: 207\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1533\n",
      "--- gene=1533, id=0 removed by soft count = 52\n",
      "--- gene=1533, id=1 removed by soft count = 2704\n",
      "--- gene=1533, id=2 removed by soft count = 655\n",
      "--- gene=1533, id=3 removed by soft count = 191\n",
      "--- gene=1533, id=4 removed by soft count = 658\n",
      "--- gene=1533, id=5 removed by soft count = 135\n",
      "--- gene=1533, id=8 removed by soft count = 498\n",
      "--- gene=1533, id=9 removed by soft count = 121\n",
      "--- gene=1533, id=10 removed by soft count = 722\n",
      "--- gene=1533, id=12 removed by soft count = 330\n",
      "--- gene=1533, id=16 removed by soft count = 83\n",
      "--- gene=1533, id=17 removed by soft count = 512\n",
      "--- gene=1533, id=18 removed by soft count = 82\n",
      "--- gene=1533, id=21 removed by soft count = 240\n",
      "--- gene=1533, id=23 removed by soft count = 51\n",
      "--- gene=1533, id=24 removed by soft count = 142\n",
      "--- gene=1533, id=26 removed by soft count = 161\n",
      "--- gene=1533, id=30 removed by soft count = 219\n",
      "--- gene=1533, id=31 removed by soft count = 120\n",
      "--- gene=1533, id=33 removed by soft count = 175\n",
      "--- gene=1533, id=35 removed by soft count = 86\n",
      "--- gene=1533, id=38 removed by soft count = 96\n",
      "--- gene=1533, id=39 removed by soft count = 376\n",
      "--- gene=1533, id=41 removed by soft count = 396\n",
      "--- gene=1533, id=43 removed by soft count = 2544\n",
      "--- gene=1533, id=45 removed by soft count = 55\n",
      "--- gene=1533, id=50 removed by soft count = 312\n",
      "--- gene=1533, id=51 removed by soft count = 72\n",
      "--- gene=1533, id=54 removed by soft count = 168\n",
      "--- gene=1533, id=55 removed by soft count = 77\n",
      "--- gene=1533, id=56 removed by soft count = 148\n",
      "--- gene=1533, id=57 removed by soft count = 1146\n",
      "--- gene=1533, id=59 removed by soft count = 266\n",
      "--- gene=1533, id=60 removed by soft count = 165\n",
      "--- gene=1533, id=61 removed by soft count = 188\n",
      "--- gene=1533, id=64 removed by soft count = 176\n",
      "--- gene=1533, id=66 removed by soft count = 528\n",
      "--- gene=1533, id=67 removed by soft count = 322\n",
      "--- gene=1533, id=68 removed by soft count = 157\n",
      "--- number of probes: 69 , kept by blast: 30 , if remove dups: 30\n",
      "-- number of probes kept for this region: 30\n",
      "-- checking probes in region: 1534\n",
      "--- gene=1534, id=55 removed by soft count = 106\n",
      "--- gene=1534, id=57 removed by soft count = 59\n",
      "--- gene=1534, id=60 removed by soft count = 91\n",
      "--- gene=1534, id=70 removed by soft count = 327\n",
      "--- gene=1534, id=71 removed by soft count = 1673\n",
      "--- gene=1534, id=76 removed by soft count = 62\n",
      "--- gene=1534, id=94 removed by soft count = 151\n",
      "--- gene=1534, id=115 removed by hard count = 3\n",
      "--- gene=1534, id=126 removed by soft count = 12176\n",
      "--- gene=1534, id=132 removed by soft count = 51\n",
      "--- gene=1534, id=142 removed by soft count = 77\n",
      "--- gene=1534, id=144 removed by soft count = 654\n",
      "--- number of probes: 149 , kept by blast: 137 , if remove dups: 137\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1535\n",
      "--- gene=1535, id=1 removed by soft count = 189\n",
      "--- gene=1535, id=2 removed by soft count = 6620\n",
      "--- gene=1535, id=3 removed by soft count = 9875\n",
      "--- gene=1535, id=4 removed by soft count = 415\n",
      "--- gene=1535, id=5 removed by soft count = 2112\n",
      "--- gene=1535, id=9 removed by soft count = 52\n",
      "--- gene=1535, id=25 removed by soft count = 52\n",
      "--- gene=1535, id=52 removed by soft count = 61\n",
      "--- gene=1535, id=56 removed by soft count = 464\n",
      "--- gene=1535, id=57 removed by soft count = 1195\n",
      "--- gene=1535, id=58 removed by soft count = 455\n",
      "--- gene=1535, id=69 removed by soft count = 136\n",
      "--- gene=1535, id=76 removed by soft count = 164\n",
      "--- gene=1535, id=103 removed by hard count = 0\n",
      "--- gene=1535, id=127 removed by soft count = 62\n",
      "--- gene=1535, id=135 removed by soft count = 57\n",
      "--- gene=1535, id=137 removed by soft count = 242\n",
      "--- gene=1535, id=138 removed by soft count = 112\n",
      "--- gene=1535, id=141 removed by soft count = 154\n",
      "--- gene=1535, id=143 removed by soft count = 1492\n",
      "--- gene=1535, id=144 removed by soft count = 343\n",
      "--- gene=1535, id=145 removed by soft count = 1519\n",
      "--- number of probes: 187 , kept by blast: 165 , if remove dups: 165\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1536\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- gene=1536, id=15 removed by soft count = 73\n",
      "--- gene=1536, id=45 removed by soft count = 6555\n",
      "--- gene=1536, id=48 removed by soft count = 497\n",
      "--- gene=1536, id=64 removed by soft count = 52\n",
      "--- gene=1536, id=71 removed by soft count = 74\n",
      "--- gene=1536, id=72 removed by soft count = 1531\n",
      "--- gene=1536, id=73 removed by soft count = 77\n",
      "--- gene=1536, id=104 removed by soft count = 68\n",
      "--- gene=1536, id=118 removed by soft count = 213\n",
      "--- gene=1536, id=128 removed by soft count = 258\n",
      "--- gene=1536, id=132 removed by soft count = 150\n",
      "--- gene=1536, id=154 removed by soft count = 104\n",
      "--- gene=1536, id=159 removed by soft count = 853\n",
      "--- gene=1536, id=160 removed by soft count = 184\n",
      "--- gene=1536, id=161 removed by soft count = 211\n",
      "--- gene=1536, id=162 removed by soft count = 265\n",
      "--- gene=1536, id=163 removed by soft count = 151\n",
      "--- gene=1536, id=164 removed by soft count = 187\n",
      "--- gene=1536, id=167 removed by soft count = 283\n",
      "--- gene=1536, id=168 removed by soft count = 132\n",
      "--- gene=1536, id=169 removed by soft count = 70\n",
      "--- gene=1536, id=170 removed by soft count = 203\n",
      "--- gene=1536, id=172 removed by soft count = 287\n",
      "--- gene=1536, id=173 removed by soft count = 133\n",
      "--- gene=1536, id=174 removed by soft count = 883\n",
      "--- number of probes: 187 , kept by blast: 162 , if remove dups: 162\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1537\n",
      "--- gene=1537, id=25 removed by soft count = 64\n",
      "--- gene=1537, id=47 removed by soft count = 87\n",
      "--- gene=1537, id=112 removed by soft count = 1086\n",
      "--- gene=1537, id=115 removed by soft count = 58\n",
      "--- gene=1537, id=118 removed by soft count = 66\n",
      "--- gene=1537, id=119 removed by soft count = 58\n",
      "--- gene=1537, id=136 removed by soft count = 2390\n",
      "--- gene=1537, id=137 removed by soft count = 137\n",
      "--- gene=1537, id=143 removed by soft count = 837\n",
      "--- gene=1537, id=144 removed by soft count = 757\n",
      "--- gene=1537, id=145 removed by soft count = 81\n",
      "--- gene=1537, id=146 removed by soft count = 73\n",
      "--- gene=1537, id=147 removed by soft count = 160\n",
      "--- gene=1537, id=148 removed by soft count = 653\n",
      "--- gene=1537, id=151 removed by soft count = 69\n",
      "--- gene=1537, id=164 removed by soft count = 725\n",
      "--- gene=1537, id=165 removed by soft count = 748\n",
      "--- gene=1537, id=166 removed by soft count = 798\n",
      "--- gene=1537, id=167 removed by soft count = 596\n",
      "--- gene=1537, id=168 removed by soft count = 233\n",
      "--- gene=1537, id=169 removed by soft count = 935\n",
      "--- number of probes: 170 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1538\n",
      "--- gene=1538, id=30 removed by soft count = 785\n",
      "--- gene=1538, id=31 removed by soft count = 474\n",
      "--- gene=1538, id=33 removed by soft count = 11068\n",
      "--- gene=1538, id=42 removed by soft count = 312\n",
      "--- gene=1538, id=43 removed by soft count = 505\n",
      "--- gene=1538, id=52 removed by soft count = 86\n",
      "--- gene=1538, id=53 removed by soft count = 61\n",
      "--- gene=1538, id=56 removed by soft count = 231\n",
      "--- gene=1538, id=59 removed by soft count = 373\n",
      "--- gene=1538, id=92 removed by soft count = 2448\n",
      "--- gene=1538, id=93 removed by soft count = 260\n",
      "--- gene=1538, id=96 removed by soft count = 176\n",
      "--- gene=1538, id=138 removed by soft count = 2964\n",
      "--- gene=1538, id=159 removed by soft count = 1681\n",
      "--- gene=1538, id=161 removed by soft count = 7822\n",
      "--- gene=1538, id=163 removed by soft count = 2989\n",
      "--- number of probes: 164 , kept by blast: 148 , if remove dups: 148\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1541\n",
      "--- gene=1541, id=9 removed by soft count = 1192\n",
      "--- gene=1541, id=41 removed by soft count = 366\n",
      "--- gene=1541, id=42 removed by soft count = 209\n",
      "--- gene=1541, id=54 removed by soft count = 2572\n",
      "--- gene=1541, id=63 removed by soft count = 114\n",
      "--- gene=1541, id=209 removed by soft count = 6814\n",
      "--- gene=1541, id=210 removed by soft count = 2004\n",
      "--- gene=1541, id=211 removed by soft count = 825\n",
      "--- gene=1541, id=212 removed by soft count = 1125\n",
      "--- gene=1541, id=215 removed by soft count = 1273\n",
      "--- gene=1541, id=217 removed by soft count = 76\n",
      "--- gene=1541, id=219 removed by soft count = 229\n",
      "--- gene=1541, id=247 removed by soft count = 594\n",
      "--- gene=1541, id=250 removed by soft count = 190\n",
      "--- number of probes: 251 , kept by blast: 237 , if remove dups: 237\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1542\n",
      "--- gene=1542, id=21 removed by soft count = 804\n",
      "--- gene=1542, id=28 removed by soft count = 8954\n",
      "--- gene=1542, id=37 removed by soft count = 137\n",
      "--- gene=1542, id=39 removed by soft count = 80\n",
      "--- gene=1542, id=51 removed by soft count = 81\n",
      "--- gene=1542, id=52 removed by soft count = 401\n",
      "--- gene=1542, id=70 removed by soft count = 84\n",
      "--- gene=1542, id=72 removed by soft count = 570\n",
      "--- gene=1542, id=77 removed by soft count = 68\n",
      "--- gene=1542, id=81 removed by soft count = 68\n",
      "--- gene=1542, id=83 removed by soft count = 151\n",
      "--- gene=1542, id=84 removed by soft count = 419\n",
      "--- gene=1542, id=85 removed by soft count = 156\n",
      "--- gene=1542, id=86 removed by soft count = 266\n",
      "--- gene=1542, id=90 removed by soft count = 53\n",
      "--- gene=1542, id=91 removed by soft count = 116\n",
      "--- gene=1542, id=104 removed by soft count = 59\n",
      "--- gene=1542, id=108 removed by soft count = 61\n",
      "--- gene=1542, id=188 removed by soft count = 64\n",
      "--- number of probes: 204 , kept by blast: 185 , if remove dups: 185\n",
      "--- remove duplicated probes\n",
      "--- keep probes in the center\n",
      "-- number of probes kept for this region: 120\n",
      "-- checking probes in region: 1548\n",
      "--- gene=1548, id=11 removed by soft count = 60\n",
      "--- gene=1548, id=40 removed by hard count = 4\n",
      "--- number of probes: 106 , kept by blast: 104 , if remove dups: 102\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 102\n",
      "- summarize\n",
      "-- region: 0 , number of probes: 120\n",
      "-- region: 1 , number of probes: 120\n",
      "-- region: 2 , number of probes: 120\n",
      "-- region: 3 , number of probes: 120\n",
      "-- region: 4 , number of probes: 120\n",
      "-- region: 5 , number of probes: 120\n",
      "-- region: 6 , number of probes: 120\n",
      "-- region: 7 , number of probes: 120\n",
      "-- region: 8 , number of probes: 120\n",
      "-- region: 9 , number of probes: 120\n",
      "-- region: 10 , number of probes: 120\n",
      "-- region: 11 , number of probes: 120\n",
      "-- region: 12 , number of probes: 120\n",
      "-- region: 13 , number of probes: 120\n",
      "-- region: 14 , number of probes: 120\n",
      "-- region: 15 , number of probes: 120\n",
      "-- region: 16 , number of probes: 120\n",
      "-- region: 17 , number of probes: 120\n",
      "-- region: 18 , number of probes: 120\n",
      "-- region: 19 , number of probes: 120\n",
      "-- region: 20 , number of probes: 120\n",
      "-- region: 21 , number of probes: 120\n",
      "-- region: 22 , number of probes: 120\n",
      "-- region: 23 , number of probes: 120\n",
      "-- region: 24 , number of probes: 120\n",
      "-- region: 25 , number of probes: 120\n",
      "-- region: 26 , number of probes: 120\n",
      "-- region: 27 , number of probes: 120\n",
      "-- region: 28 , number of probes: 120\n",
      "-- region: 29 , number of probes: 120\n",
      "-- region: 30 , number of probes: 120\n",
      "-- region: 31 , number of probes: 120\n",
      "-- region: 32 , number of probes: 120\n",
      "-- region: 33 , number of probes: 120\n",
      "-- region: 34 , number of probes: 120\n",
      "-- region: 35 , number of probes: 120\n",
      "-- region: 36 , number of probes: 120\n",
      "-- region: 37 , number of probes: 120\n",
      "-- region: 38 , number of probes: 120\n",
      "-- region: 39 , number of probes: 120\n",
      "-- region: 40 , number of probes: 120\n",
      "-- region: 42 , number of probes: 120\n",
      "-- region: 43 , number of probes: 120\n",
      "-- region: 44 , number of probes: 120\n",
      "-- region: 45 , number of probes: 120\n",
      "-- region: 46 , number of probes: 120\n",
      "-- region: 47 , number of probes: 120\n",
      "-- region: 57 , number of probes: 59\n",
      "-- region: 60 , number of probes: 120\n",
      "-- region: 62 , number of probes: 120\n",
      "-- region: 63 , number of probes: 120\n",
      "-- region: 64 , number of probes: 120\n",
      "-- region: 65 , number of probes: 101\n",
      "-- region: 66 , number of probes: 120\n",
      "-- region: 67 , number of probes: 120\n",
      "-- region: 68 , number of probes: 120\n",
      "-- region: 69 , number of probes: 118\n",
      "-- region: 70 , number of probes: 111\n",
      "-- region: 71 , number of probes: 120\n",
      "-- region: 72 , number of probes: 120\n",
      "-- region: 73 , number of probes: 120\n",
      "-- region: 74 , number of probes: 120\n",
      "-- region: 75 , number of probes: 120\n",
      "-- region: 76 , number of probes: 86\n",
      "-- region: 77 , number of probes: 120\n",
      "-- region: 78 , number of probes: 120\n",
      "-- region: 79 , number of probes: 120\n",
      "-- region: 80 , number of probes: 120\n",
      "-- region: 81 , number of probes: 120\n",
      "-- region: 83 , number of probes: 120\n",
      "-- region: 84 , number of probes: 120\n",
      "-- region: 85 , number of probes: 120\n",
      "-- region: 86 , number of probes: 120\n",
      "-- region: 87 , number of probes: 120\n",
      "-- region: 88 , number of probes: 120\n",
      "-- region: 89 , number of probes: 120\n",
      "-- region: 90 , number of probes: 110\n",
      "-- region: 91 , number of probes: 120\n",
      "-- region: 92 , number of probes: 120\n",
      "-- region: 93 , number of probes: 120\n",
      "-- region: 94 , number of probes: 120\n",
      "-- region: 95 , number of probes: 120\n",
      "-- region: 96 , number of probes: 120\n",
      "-- region: 98 , number of probes: 120\n",
      "-- region: 100 , number of probes: 120\n",
      "-- region: 101 , number of probes: 120\n",
      "-- region: 102 , number of probes: 120\n",
      "-- region: 103 , number of probes: 120\n",
      "-- region: 104 , number of probes: 96\n",
      "-- region: 105 , number of probes: 120\n",
      "-- region: 106 , number of probes: 120\n",
      "-- region: 107 , number of probes: 120\n",
      "-- region: 108 , number of probes: 120\n",
      "-- region: 109 , number of probes: 120\n",
      "-- region: 110 , number of probes: 120\n",
      "-- region: 111 , number of probes: 120\n",
      "-- region: 112 , number of probes: 120\n",
      "-- region: 114 , number of probes: 120\n",
      "-- region: 115 , number of probes: 120\n",
      "-- region: 116 , number of probes: 120\n",
      "-- region: 117 , number of probes: 120\n",
      "-- region: 118 , number of probes: 120\n",
      "-- region: 119 , number of probes: 120\n",
      "-- region: 120 , number of probes: 120\n",
      "-- region: 121 , number of probes: 120\n",
      "-- region: 122 , number of probes: 120\n",
      "-- region: 123 , number of probes: 120\n",
      "-- region: 124 , number of probes: 120\n",
      "-- region: 125 , number of probes: 120\n",
      "-- region: 126 , number of probes: 120\n",
      "-- region: 127 , number of probes: 120\n",
      "-- region: 128 , number of probes: 120\n",
      "-- region: 129 , number of probes: 120\n",
      "-- region: 130 , number of probes: 120\n",
      "-- region: 131 , number of probes: 120\n",
      "-- region: 132 , number of probes: 120\n",
      "-- region: 133 , number of probes: 120\n",
      "-- region: 134 , number of probes: 120\n",
      "-- region: 137 , number of probes: 120\n",
      "-- region: 138 , number of probes: 120\n",
      "-- region: 139 , number of probes: 120\n",
      "-- region: 140 , number of probes: 120\n",
      "-- region: 141 , number of probes: 68\n",
      "-- region: 143 , number of probes: 120\n",
      "-- region: 144 , number of probes: 120\n",
      "-- region: 145 , number of probes: 120\n",
      "-- region: 146 , number of probes: 120\n",
      "-- region: 147 , number of probes: 120\n",
      "-- region: 148 , number of probes: 120\n",
      "-- region: 149 , number of probes: 120\n",
      "-- region: 150 , number of probes: 120\n",
      "-- region: 152 , number of probes: 120\n",
      "-- region: 153 , number of probes: 120\n",
      "-- region: 154 , number of probes: 120\n",
      "-- region: 155 , number of probes: 120\n",
      "-- region: 156 , number of probes: 120\n",
      "-- region: 157 , number of probes: 120\n",
      "-- region: 158 , number of probes: 120\n",
      "-- region: 159 , number of probes: 120\n",
      "-- region: 160 , number of probes: 120\n",
      "-- region: 161 , number of probes: 120\n",
      "-- region: 162 , number of probes: 120\n",
      "-- region: 163 , number of probes: 120\n",
      "-- region: 164 , number of probes: 120\n",
      "-- region: 165 , number of probes: 120\n",
      "-- region: 166 , number of probes: 120\n",
      "-- region: 167 , number of probes: 120\n",
      "-- region: 168 , number of probes: 120\n",
      "-- region: 169 , number of probes: 120\n",
      "-- region: 170 , number of probes: 120\n",
      "-- region: 171 , number of probes: 120\n",
      "-- region: 172 , number of probes: 120\n",
      "-- region: 173 , number of probes: 120\n",
      "-- region: 174 , number of probes: 120\n",
      "-- region: 175 , number of probes: 120\n",
      "-- region: 176 , number of probes: 120\n",
      "-- region: 177 , number of probes: 120\n",
      "-- region: 178 , number of probes: 120\n",
      "-- region: 179 , number of probes: 120\n",
      "-- region: 180 , number of probes: 120\n",
      "-- region: 181 , number of probes: 120\n",
      "-- region: 182 , number of probes: 81\n",
      "-- region: 183 , number of probes: 120\n",
      "-- region: 184 , number of probes: 120\n",
      "-- region: 185 , number of probes: 120\n",
      "-- region: 186 , number of probes: 120\n",
      "-- region: 187 , number of probes: 120\n",
      "-- region: 188 , number of probes: 120\n",
      "-- region: 189 , number of probes: 120\n",
      "-- region: 190 , number of probes: 120\n",
      "-- region: 191 , number of probes: 120\n",
      "-- region: 192 , number of probes: 120\n",
      "-- region: 193 , number of probes: 120\n",
      "-- region: 194 , number of probes: 120\n",
      "-- region: 195 , number of probes: 120\n",
      "-- region: 196 , number of probes: 120\n",
      "-- region: 197 , number of probes: 120\n",
      "-- region: 198 , number of probes: 120\n",
      "-- region: 199 , number of probes: 120\n",
      "-- region: 200 , number of probes: 120\n",
      "-- region: 201 , number of probes: 120\n",
      "-- region: 202 , number of probes: 120\n",
      "-- region: 203 , number of probes: 120\n",
      "-- region: 204 , number of probes: 120\n",
      "-- region: 205 , number of probes: 120\n",
      "-- region: 206 , number of probes: 120\n",
      "-- region: 207 , number of probes: 120\n",
      "-- region: 208 , number of probes: 120\n",
      "-- region: 209 , number of probes: 120\n",
      "-- region: 210 , number of probes: 120\n",
      "-- region: 211 , number of probes: 120\n",
      "-- region: 212 , number of probes: 120\n",
      "-- region: 213 , number of probes: 120\n",
      "-- region: 214 , number of probes: 120\n",
      "-- region: 215 , number of probes: 120\n",
      "-- region: 216 , number of probes: 120\n",
      "-- region: 217 , number of probes: 120\n",
      "-- region: 218 , number of probes: 120\n",
      "-- region: 219 , number of probes: 120\n",
      "-- region: 220 , number of probes: 120\n",
      "-- region: 221 , number of probes: 120\n",
      "-- region: 222 , number of probes: 120\n",
      "-- region: 223 , number of probes: 120\n",
      "-- region: 224 , number of probes: 120\n",
      "-- region: 225 , number of probes: 120\n",
      "-- region: 226 , number of probes: 120\n",
      "-- region: 227 , number of probes: 120\n",
      "-- region: 228 , number of probes: 120\n",
      "-- region: 229 , number of probes: 120\n",
      "-- region: 230 , number of probes: 120\n",
      "-- region: 233 , number of probes: 120\n",
      "-- region: 234 , number of probes: 120\n",
      "-- region: 235 , number of probes: 120\n",
      "-- region: 236 , number of probes: 120\n",
      "-- region: 237 , number of probes: 120\n",
      "-- region: 238 , number of probes: 120\n",
      "-- region: 239 , number of probes: 120\n",
      "-- region: 240 , number of probes: 120\n",
      "-- region: 241 , number of probes: 120\n",
      "-- region: 242 , number of probes: 120\n",
      "-- region: 243 , number of probes: 120\n",
      "-- region: 244 , number of probes: 120\n",
      "-- region: 245 , number of probes: 120\n",
      "-- region: 246 , number of probes: 120\n",
      "-- region: 247 , number of probes: 120\n",
      "-- region: 248 , number of probes: 120\n",
      "-- region: 249 , number of probes: 120\n",
      "-- region: 250 , number of probes: 120\n",
      "-- region: 251 , number of probes: 120\n",
      "-- region: 252 , number of probes: 120\n",
      "-- region: 253 , number of probes: 120\n",
      "-- region: 254 , number of probes: 120\n",
      "-- region: 255 , number of probes: 120\n",
      "-- region: 256 , number of probes: 120\n",
      "-- region: 257 , number of probes: 120\n",
      "-- region: 258 , number of probes: 120\n",
      "-- region: 259 , number of probes: 120\n",
      "-- region: 260 , number of probes: 120\n",
      "-- region: 261 , number of probes: 120\n",
      "-- region: 262 , number of probes: 120\n",
      "-- region: 263 , number of probes: 120\n",
      "-- region: 264 , number of probes: 120\n",
      "-- region: 265 , number of probes: 120\n",
      "-- region: 266 , number of probes: 120\n",
      "-- region: 267 , number of probes: 120\n",
      "-- region: 268 , number of probes: 120\n",
      "-- region: 269 , number of probes: 120\n",
      "-- region: 270 , number of probes: 120\n",
      "-- region: 271 , number of probes: 120\n",
      "-- region: 272 , number of probes: 120\n",
      "-- region: 273 , number of probes: 120\n",
      "-- region: 274 , number of probes: 120\n",
      "-- region: 275 , number of probes: 120\n",
      "-- region: 276 , number of probes: 120\n",
      "-- region: 277 , number of probes: 120\n",
      "-- region: 278 , number of probes: 120\n",
      "-- region: 279 , number of probes: 120\n",
      "-- region: 280 , number of probes: 120\n",
      "-- region: 281 , number of probes: 120\n",
      "-- region: 282 , number of probes: 120\n",
      "-- region: 283 , number of probes: 120\n",
      "-- region: 284 , number of probes: 120\n",
      "-- region: 285 , number of probes: 120\n",
      "-- region: 286 , number of probes: 120\n",
      "-- region: 287 , number of probes: 120\n",
      "-- region: 288 , number of probes: 120\n",
      "-- region: 289 , number of probes: 120\n",
      "-- region: 290 , number of probes: 120\n",
      "-- region: 291 , number of probes: 120\n",
      "-- region: 292 , number of probes: 120\n",
      "-- region: 293 , number of probes: 120\n",
      "-- region: 295 , number of probes: 120\n",
      "-- region: 296 , number of probes: 120\n",
      "-- region: 297 , number of probes: 120\n",
      "-- region: 298 , number of probes: 120\n",
      "-- region: 299 , number of probes: 120\n",
      "-- region: 300 , number of probes: 120\n",
      "-- region: 302 , number of probes: 120\n",
      "-- region: 303 , number of probes: 120\n",
      "-- region: 304 , number of probes: 120\n",
      "-- region: 305 , number of probes: 120\n",
      "-- region: 306 , number of probes: 120\n",
      "-- region: 307 , number of probes: 120\n",
      "-- region: 308 , number of probes: 120\n",
      "-- region: 309 , number of probes: 120\n",
      "-- region: 310 , number of probes: 120\n",
      "-- region: 311 , number of probes: 120\n",
      "-- region: 312 , number of probes: 120\n",
      "-- region: 313 , number of probes: 120\n",
      "-- region: 314 , number of probes: 120\n",
      "-- region: 315 , number of probes: 120\n",
      "-- region: 316 , number of probes: 120\n",
      "-- region: 317 , number of probes: 120\n",
      "-- region: 318 , number of probes: 120\n",
      "-- region: 319 , number of probes: 120\n",
      "-- region: 320 , number of probes: 120\n",
      "-- region: 321 , number of probes: 120\n",
      "-- region: 322 , number of probes: 120\n",
      "-- region: 323 , number of probes: 120\n",
      "-- region: 324 , number of probes: 120\n",
      "-- region: 325 , number of probes: 120\n",
      "-- region: 326 , number of probes: 120\n",
      "-- region: 327 , number of probes: 91\n",
      "-- region: 328 , number of probes: 120\n",
      "-- region: 329 , number of probes: 120\n",
      "-- region: 330 , number of probes: 120\n",
      "-- region: 331 , number of probes: 120\n",
      "-- region: 332 , number of probes: 120\n",
      "-- region: 333 , number of probes: 120\n",
      "-- region: 334 , number of probes: 120\n",
      "-- region: 335 , number of probes: 120\n",
      "-- region: 336 , number of probes: 120\n",
      "-- region: 337 , number of probes: 120\n",
      "-- region: 338 , number of probes: 120\n",
      "-- region: 339 , number of probes: 120\n",
      "-- region: 340 , number of probes: 120\n",
      "-- region: 341 , number of probes: 120\n",
      "-- region: 342 , number of probes: 120\n",
      "-- region: 343 , number of probes: 120\n",
      "-- region: 344 , number of probes: 120\n",
      "-- region: 345 , number of probes: 120\n",
      "-- region: 346 , number of probes: 120\n",
      "-- region: 347 , number of probes: 120\n",
      "-- region: 348 , number of probes: 120\n",
      "-- region: 349 , number of probes: 120\n",
      "-- region: 350 , number of probes: 120\n",
      "-- region: 351 , number of probes: 120\n",
      "-- region: 352 , number of probes: 120\n",
      "-- region: 353 , number of probes: 120\n",
      "-- region: 354 , number of probes: 120\n",
      "-- region: 355 , number of probes: 120\n",
      "-- region: 356 , number of probes: 120\n",
      "-- region: 357 , number of probes: 120\n",
      "-- region: 358 , number of probes: 120\n",
      "-- region: 359 , number of probes: 120\n",
      "-- region: 360 , number of probes: 120\n",
      "-- region: 361 , number of probes: 120\n",
      "-- region: 362 , number of probes: 120\n",
      "-- region: 363 , number of probes: 120\n",
      "-- region: 364 , number of probes: 120\n",
      "-- region: 365 , number of probes: 120\n",
      "-- region: 366 , number of probes: 120\n",
      "-- region: 367 , number of probes: 83\n",
      "-- region: 368 , number of probes: 30\n",
      "-- region: 370 , number of probes: 120\n",
      "-- region: 371 , number of probes: 74\n",
      "-- region: 372 , number of probes: 120\n",
      "-- region: 373 , number of probes: 120\n",
      "-- region: 374 , number of probes: 120\n",
      "-- region: 375 , number of probes: 120\n",
      "-- region: 376 , number of probes: 120\n",
      "-- region: 377 , number of probes: 120\n",
      "-- region: 379 , number of probes: 120\n",
      "-- region: 380 , number of probes: 120\n",
      "-- region: 381 , number of probes: 120\n",
      "-- region: 382 , number of probes: 120\n",
      "-- region: 383 , number of probes: 120\n",
      "-- region: 384 , number of probes: 120\n",
      "-- region: 385 , number of probes: 120\n",
      "-- region: 386 , number of probes: 120\n",
      "-- region: 387 , number of probes: 120\n",
      "-- region: 388 , number of probes: 120\n",
      "-- region: 389 , number of probes: 120\n",
      "-- region: 390 , number of probes: 120\n",
      "-- region: 391 , number of probes: 120\n",
      "-- region: 392 , number of probes: 120\n",
      "-- region: 393 , number of probes: 120\n",
      "-- region: 394 , number of probes: 120\n",
      "-- region: 395 , number of probes: 120\n",
      "-- region: 396 , number of probes: 120\n",
      "-- region: 397 , number of probes: 120\n",
      "-- region: 398 , number of probes: 120\n",
      "-- region: 399 , number of probes: 120\n",
      "-- region: 400 , number of probes: 120\n",
      "-- region: 401 , number of probes: 120\n",
      "-- region: 402 , number of probes: 120\n",
      "-- region: 403 , number of probes: 120\n",
      "-- region: 404 , number of probes: 120\n",
      "-- region: 405 , number of probes: 120\n",
      "-- region: 406 , number of probes: 120\n",
      "-- region: 407 , number of probes: 120\n",
      "-- region: 408 , number of probes: 120\n",
      "-- region: 409 , number of probes: 120\n",
      "-- region: 410 , number of probes: 120\n",
      "-- region: 411 , number of probes: 120\n",
      "-- region: 412 , number of probes: 120\n",
      "-- region: 413 , number of probes: 120\n",
      "-- region: 414 , number of probes: 120\n",
      "-- region: 415 , number of probes: 120\n",
      "-- region: 416 , number of probes: 120\n",
      "-- region: 417 , number of probes: 120\n",
      "-- region: 418 , number of probes: 120\n",
      "-- region: 419 , number of probes: 120\n",
      "-- region: 421 , number of probes: 120\n",
      "-- region: 422 , number of probes: 120\n",
      "-- region: 423 , number of probes: 120\n",
      "-- region: 424 , number of probes: 120\n",
      "-- region: 425 , number of probes: 120\n",
      "-- region: 426 , number of probes: 120\n",
      "-- region: 427 , number of probes: 120\n",
      "-- region: 428 , number of probes: 120\n",
      "-- region: 429 , number of probes: 120\n",
      "-- region: 430 , number of probes: 120\n",
      "-- region: 431 , number of probes: 120\n",
      "-- region: 432 , number of probes: 120\n",
      "-- region: 435 , number of probes: 120\n",
      "-- region: 436 , number of probes: 120\n",
      "-- region: 437 , number of probes: 120\n",
      "-- region: 438 , number of probes: 120\n",
      "-- region: 439 , number of probes: 120\n",
      "-- region: 440 , number of probes: 120\n",
      "-- region: 441 , number of probes: 120\n",
      "-- region: 442 , number of probes: 120\n",
      "-- region: 443 , number of probes: 120\n",
      "-- region: 444 , number of probes: 120\n",
      "-- region: 446 , number of probes: 90\n",
      "-- region: 447 , number of probes: 120\n",
      "-- region: 448 , number of probes: 112\n",
      "-- region: 449 , number of probes: 120\n",
      "-- region: 450 , number of probes: 120\n",
      "-- region: 451 , number of probes: 120\n",
      "-- region: 452 , number of probes: 120\n",
      "-- region: 453 , number of probes: 49\n",
      "-- region: 454 , number of probes: 120\n",
      "-- region: 455 , number of probes: 120\n",
      "-- region: 456 , number of probes: 120\n",
      "-- region: 457 , number of probes: 120\n",
      "-- region: 458 , number of probes: 120\n",
      "-- region: 459 , number of probes: 120\n",
      "-- region: 460 , number of probes: 120\n",
      "-- region: 461 , number of probes: 120\n",
      "-- region: 462 , number of probes: 120\n",
      "-- region: 463 , number of probes: 120\n",
      "-- region: 464 , number of probes: 120\n",
      "-- region: 465 , number of probes: 120\n",
      "-- region: 466 , number of probes: 120\n",
      "-- region: 467 , number of probes: 120\n",
      "-- region: 468 , number of probes: 120\n",
      "-- region: 469 , number of probes: 120\n",
      "-- region: 470 , number of probes: 114\n",
      "-- region: 471 , number of probes: 120\n",
      "-- region: 472 , number of probes: 120\n",
      "-- region: 473 , number of probes: 120\n",
      "-- region: 474 , number of probes: 120\n",
      "-- region: 475 , number of probes: 120\n",
      "-- region: 476 , number of probes: 120\n",
      "-- region: 477 , number of probes: 120\n",
      "-- region: 478 , number of probes: 120\n",
      "-- region: 479 , number of probes: 120\n",
      "-- region: 480 , number of probes: 120\n",
      "-- region: 481 , number of probes: 120\n",
      "-- region: 482 , number of probes: 120\n",
      "-- region: 483 , number of probes: 120\n",
      "-- region: 484 , number of probes: 120\n",
      "-- region: 485 , number of probes: 120\n",
      "-- region: 486 , number of probes: 120\n",
      "-- region: 487 , number of probes: 120\n",
      "-- region: 489 , number of probes: 120\n",
      "-- region: 490 , number of probes: 120\n",
      "-- region: 491 , number of probes: 120\n",
      "-- region: 492 , number of probes: 120\n",
      "-- region: 493 , number of probes: 120\n",
      "-- region: 494 , number of probes: 120\n",
      "-- region: 495 , number of probes: 120\n",
      "-- region: 496 , number of probes: 120\n",
      "-- region: 497 , number of probes: 120\n",
      "-- region: 498 , number of probes: 120\n",
      "-- region: 499 , number of probes: 120\n",
      "-- region: 500 , number of probes: 120\n",
      "-- region: 501 , number of probes: 120\n",
      "-- region: 502 , number of probes: 120\n",
      "-- region: 503 , number of probes: 120\n",
      "-- region: 504 , number of probes: 120\n",
      "-- region: 505 , number of probes: 120\n",
      "-- region: 506 , number of probes: 120\n",
      "-- region: 507 , number of probes: 120\n",
      "-- region: 508 , number of probes: 120\n",
      "-- region: 509 , number of probes: 120\n",
      "-- region: 510 , number of probes: 120\n",
      "-- region: 511 , number of probes: 120\n",
      "-- region: 512 , number of probes: 120\n",
      "-- region: 516 , number of probes: 120\n",
      "-- region: 517 , number of probes: 120\n",
      "-- region: 518 , number of probes: 120\n",
      "-- region: 519 , number of probes: 120\n",
      "-- region: 520 , number of probes: 120\n",
      "-- region: 521 , number of probes: 120\n",
      "-- region: 522 , number of probes: 120\n",
      "-- region: 523 , number of probes: 120\n",
      "-- region: 524 , number of probes: 120\n",
      "-- region: 525 , number of probes: 120\n",
      "-- region: 526 , number of probes: 120\n",
      "-- region: 527 , number of probes: 120\n",
      "-- region: 528 , number of probes: 120\n",
      "-- region: 529 , number of probes: 120\n",
      "-- region: 530 , number of probes: 120\n",
      "-- region: 531 , number of probes: 120\n",
      "-- region: 532 , number of probes: 120\n",
      "-- region: 533 , number of probes: 120\n",
      "-- region: 534 , number of probes: 120\n",
      "-- region: 535 , number of probes: 112\n",
      "-- region: 536 , number of probes: 120\n",
      "-- region: 537 , number of probes: 120\n",
      "-- region: 538 , number of probes: 120\n",
      "-- region: 539 , number of probes: 120\n",
      "-- region: 540 , number of probes: 120\n",
      "-- region: 541 , number of probes: 120\n",
      "-- region: 542 , number of probes: 120\n",
      "-- region: 543 , number of probes: 120\n",
      "-- region: 544 , number of probes: 120\n",
      "-- region: 545 , number of probes: 120\n",
      "-- region: 546 , number of probes: 120\n",
      "-- region: 548 , number of probes: 120\n",
      "-- region: 549 , number of probes: 120\n",
      "-- region: 550 , number of probes: 120\n",
      "-- region: 551 , number of probes: 88\n",
      "-- region: 552 , number of probes: 120\n",
      "-- region: 554 , number of probes: 120\n",
      "-- region: 555 , number of probes: 120\n",
      "-- region: 556 , number of probes: 120\n",
      "-- region: 557 , number of probes: 120\n",
      "-- region: 558 , number of probes: 120\n",
      "-- region: 559 , number of probes: 120\n",
      "-- region: 560 , number of probes: 120\n",
      "-- region: 561 , number of probes: 120\n",
      "-- region: 562 , number of probes: 120\n",
      "-- region: 563 , number of probes: 120\n",
      "-- region: 564 , number of probes: 120\n",
      "-- region: 565 , number of probes: 120\n",
      "-- region: 566 , number of probes: 120\n",
      "-- region: 567 , number of probes: 120\n",
      "-- region: 568 , number of probes: 120\n",
      "-- region: 569 , number of probes: 120\n",
      "-- region: 570 , number of probes: 46\n",
      "-- region: 571 , number of probes: 120\n",
      "-- region: 572 , number of probes: 120\n",
      "-- region: 573 , number of probes: 120\n",
      "-- region: 574 , number of probes: 120\n",
      "-- region: 575 , number of probes: 120\n",
      "-- region: 576 , number of probes: 120\n",
      "-- region: 577 , number of probes: 120\n",
      "-- region: 578 , number of probes: 120\n",
      "-- region: 579 , number of probes: 120\n",
      "-- region: 580 , number of probes: 120\n",
      "-- region: 581 , number of probes: 120\n",
      "-- region: 582 , number of probes: 120\n",
      "-- region: 583 , number of probes: 120\n",
      "-- region: 584 , number of probes: 120\n",
      "-- region: 585 , number of probes: 120\n",
      "-- region: 586 , number of probes: 120\n",
      "-- region: 587 , number of probes: 120\n",
      "-- region: 588 , number of probes: 120\n",
      "-- region: 589 , number of probes: 120\n",
      "-- region: 590 , number of probes: 120\n",
      "-- region: 591 , number of probes: 120\n",
      "-- region: 592 , number of probes: 120\n",
      "-- region: 593 , number of probes: 120\n",
      "-- region: 594 , number of probes: 120\n",
      "-- region: 595 , number of probes: 120\n",
      "-- region: 596 , number of probes: 120\n",
      "-- region: 597 , number of probes: 120\n",
      "-- region: 598 , number of probes: 120\n",
      "-- region: 599 , number of probes: 120\n",
      "-- region: 600 , number of probes: 120\n",
      "-- region: 601 , number of probes: 120\n",
      "-- region: 602 , number of probes: 120\n",
      "-- region: 603 , number of probes: 120\n",
      "-- region: 605 , number of probes: 120\n",
      "-- region: 606 , number of probes: 120\n",
      "-- region: 607 , number of probes: 120\n",
      "-- region: 608 , number of probes: 120\n",
      "-- region: 609 , number of probes: 120\n",
      "-- region: 611 , number of probes: 120\n",
      "-- region: 612 , number of probes: 120\n",
      "-- region: 613 , number of probes: 120\n",
      "-- region: 614 , number of probes: 120\n",
      "-- region: 615 , number of probes: 120\n",
      "-- region: 616 , number of probes: 120\n",
      "-- region: 617 , number of probes: 120\n",
      "-- region: 618 , number of probes: 120\n",
      "-- region: 619 , number of probes: 120\n",
      "-- region: 620 , number of probes: 120\n",
      "-- region: 621 , number of probes: 120\n",
      "-- region: 622 , number of probes: 120\n",
      "-- region: 623 , number of probes: 120\n",
      "-- region: 624 , number of probes: 120\n",
      "-- region: 625 , number of probes: 120\n",
      "-- region: 626 , number of probes: 103\n",
      "-- region: 627 , number of probes: 120\n",
      "-- region: 628 , number of probes: 120\n",
      "-- region: 629 , number of probes: 120\n",
      "-- region: 645 , number of probes: 120\n",
      "-- region: 646 , number of probes: 120\n",
      "-- region: 647 , number of probes: 120\n",
      "-- region: 648 , number of probes: 120\n",
      "-- region: 649 , number of probes: 120\n",
      "-- region: 650 , number of probes: 120\n",
      "-- region: 651 , number of probes: 120\n",
      "-- region: 652 , number of probes: 120\n",
      "-- region: 653 , number of probes: 120\n",
      "-- region: 654 , number of probes: 120\n",
      "-- region: 655 , number of probes: 120\n",
      "-- region: 656 , number of probes: 120\n",
      "-- region: 657 , number of probes: 120\n",
      "-- region: 658 , number of probes: 120\n",
      "-- region: 659 , number of probes: 48\n",
      "-- region: 660 , number of probes: 120\n",
      "-- region: 662 , number of probes: 120\n",
      "-- region: 663 , number of probes: 120\n",
      "-- region: 664 , number of probes: 120\n",
      "-- region: 665 , number of probes: 120\n",
      "-- region: 666 , number of probes: 120\n",
      "-- region: 667 , number of probes: 120\n",
      "-- region: 668 , number of probes: 120\n",
      "-- region: 669 , number of probes: 120\n",
      "-- region: 670 , number of probes: 120\n",
      "-- region: 671 , number of probes: 120\n",
      "-- region: 672 , number of probes: 120\n",
      "-- region: 673 , number of probes: 116\n",
      "-- region: 674 , number of probes: 120\n",
      "-- region: 675 , number of probes: 120\n",
      "-- region: 676 , number of probes: 120\n",
      "-- region: 677 , number of probes: 120\n",
      "-- region: 678 , number of probes: 120\n",
      "-- region: 679 , number of probes: 120\n",
      "-- region: 680 , number of probes: 120\n",
      "-- region: 681 , number of probes: 120\n",
      "-- region: 682 , number of probes: 120\n",
      "-- region: 683 , number of probes: 120\n",
      "-- region: 684 , number of probes: 120\n",
      "-- region: 685 , number of probes: 120\n",
      "-- region: 686 , number of probes: 120\n",
      "-- region: 687 , number of probes: 43\n",
      "-- region: 688 , number of probes: 120\n",
      "-- region: 689 , number of probes: 120\n",
      "-- region: 690 , number of probes: 120\n",
      "-- region: 691 , number of probes: 120\n",
      "-- region: 692 , number of probes: 120\n",
      "-- region: 693 , number of probes: 120\n",
      "-- region: 694 , number of probes: 120\n",
      "-- region: 695 , number of probes: 120\n",
      "-- region: 696 , number of probes: 120\n",
      "-- region: 697 , number of probes: 120\n",
      "-- region: 698 , number of probes: 120\n",
      "-- region: 699 , number of probes: 47\n",
      "-- region: 701 , number of probes: 120\n",
      "-- region: 702 , number of probes: 120\n",
      "-- region: 703 , number of probes: 120\n",
      "-- region: 704 , number of probes: 120\n",
      "-- region: 705 , number of probes: 120\n",
      "-- region: 706 , number of probes: 120\n",
      "-- region: 707 , number of probes: 120\n",
      "-- region: 708 , number of probes: 120\n",
      "-- region: 709 , number of probes: 120\n",
      "-- region: 710 , number of probes: 120\n",
      "-- region: 711 , number of probes: 120\n",
      "-- region: 712 , number of probes: 120\n",
      "-- region: 713 , number of probes: 120\n",
      "-- region: 714 , number of probes: 120\n",
      "-- region: 715 , number of probes: 120\n",
      "-- region: 716 , number of probes: 120\n",
      "-- region: 717 , number of probes: 120\n",
      "-- region: 718 , number of probes: 120\n",
      "-- region: 719 , number of probes: 120\n",
      "-- region: 720 , number of probes: 120\n",
      "-- region: 721 , number of probes: 120\n",
      "-- region: 722 , number of probes: 120\n",
      "-- region: 723 , number of probes: 68\n",
      "-- region: 724 , number of probes: 120\n",
      "-- region: 725 , number of probes: 120\n",
      "-- region: 726 , number of probes: 120\n",
      "-- region: 727 , number of probes: 120\n",
      "-- region: 728 , number of probes: 120\n",
      "-- region: 729 , number of probes: 120\n",
      "-- region: 730 , number of probes: 120\n",
      "-- region: 731 , number of probes: 120\n",
      "-- region: 732 , number of probes: 120\n",
      "-- region: 733 , number of probes: 120\n",
      "-- region: 734 , number of probes: 120\n",
      "-- region: 735 , number of probes: 120\n",
      "-- region: 736 , number of probes: 120\n",
      "-- region: 737 , number of probes: 120\n",
      "-- region: 738 , number of probes: 120\n",
      "-- region: 739 , number of probes: 120\n",
      "-- region: 740 , number of probes: 120\n",
      "-- region: 741 , number of probes: 120\n",
      "-- region: 742 , number of probes: 120\n",
      "-- region: 743 , number of probes: 120\n",
      "-- region: 744 , number of probes: 120\n",
      "-- region: 745 , number of probes: 120\n",
      "-- region: 748 , number of probes: 120\n",
      "-- region: 749 , number of probes: 120\n",
      "-- region: 750 , number of probes: 120\n",
      "-- region: 751 , number of probes: 120\n",
      "-- region: 752 , number of probes: 120\n",
      "-- region: 753 , number of probes: 120\n",
      "-- region: 754 , number of probes: 120\n",
      "-- region: 755 , number of probes: 36\n",
      "-- region: 756 , number of probes: 120\n",
      "-- region: 757 , number of probes: 120\n",
      "-- region: 758 , number of probes: 120\n",
      "-- region: 759 , number of probes: 120\n",
      "-- region: 760 , number of probes: 120\n",
      "-- region: 761 , number of probes: 120\n",
      "-- region: 762 , number of probes: 120\n",
      "-- region: 763 , number of probes: 120\n",
      "-- region: 764 , number of probes: 120\n",
      "-- region: 765 , number of probes: 120\n",
      "-- region: 766 , number of probes: 120\n",
      "-- region: 767 , number of probes: 120\n",
      "-- region: 768 , number of probes: 120\n",
      "-- region: 769 , number of probes: 120\n",
      "-- region: 770 , number of probes: 79\n",
      "-- region: 774 , number of probes: 120\n",
      "-- region: 775 , number of probes: 120\n",
      "-- region: 776 , number of probes: 120\n",
      "-- region: 777 , number of probes: 120\n",
      "-- region: 778 , number of probes: 120\n",
      "-- region: 779 , number of probes: 120\n",
      "-- region: 780 , number of probes: 120\n",
      "-- region: 782 , number of probes: 120\n",
      "-- region: 783 , number of probes: 120\n",
      "-- region: 784 , number of probes: 120\n",
      "-- region: 785 , number of probes: 120\n",
      "-- region: 786 , number of probes: 120\n",
      "-- region: 787 , number of probes: 120\n",
      "-- region: 788 , number of probes: 120\n",
      "-- region: 789 , number of probes: 120\n",
      "-- region: 790 , number of probes: 87\n",
      "-- region: 791 , number of probes: 120\n",
      "-- region: 792 , number of probes: 120\n",
      "-- region: 793 , number of probes: 120\n",
      "-- region: 794 , number of probes: 120\n",
      "-- region: 795 , number of probes: 114\n",
      "-- region: 796 , number of probes: 120\n",
      "-- region: 797 , number of probes: 120\n",
      "-- region: 798 , number of probes: 120\n",
      "-- region: 799 , number of probes: 120\n",
      "-- region: 800 , number of probes: 120\n",
      "-- region: 801 , number of probes: 120\n",
      "-- region: 802 , number of probes: 120\n",
      "-- region: 803 , number of probes: 120\n",
      "-- region: 804 , number of probes: 120\n",
      "-- region: 805 , number of probes: 120\n",
      "-- region: 806 , number of probes: 120\n",
      "-- region: 807 , number of probes: 120\n",
      "-- region: 808 , number of probes: 120\n",
      "-- region: 809 , number of probes: 120\n",
      "-- region: 810 , number of probes: 120\n",
      "-- region: 811 , number of probes: 120\n",
      "-- region: 812 , number of probes: 120\n",
      "-- region: 813 , number of probes: 120\n",
      "-- region: 814 , number of probes: 120\n",
      "-- region: 815 , number of probes: 120\n",
      "-- region: 816 , number of probes: 120\n",
      "-- region: 817 , number of probes: 120\n",
      "-- region: 819 , number of probes: 120\n",
      "-- region: 820 , number of probes: 120\n",
      "-- region: 821 , number of probes: 120\n",
      "-- region: 822 , number of probes: 120\n",
      "-- region: 823 , number of probes: 107\n",
      "-- region: 824 , number of probes: 120\n",
      "-- region: 825 , number of probes: 120\n",
      "-- region: 826 , number of probes: 120\n",
      "-- region: 827 , number of probes: 120\n",
      "-- region: 828 , number of probes: 120\n",
      "-- region: 829 , number of probes: 120\n",
      "-- region: 832 , number of probes: 120\n",
      "-- region: 833 , number of probes: 120\n",
      "-- region: 834 , number of probes: 120\n",
      "-- region: 835 , number of probes: 120\n",
      "-- region: 836 , number of probes: 120\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- region: 837 , number of probes: 120\n",
      "-- region: 838 , number of probes: 120\n",
      "-- region: 839 , number of probes: 120\n",
      "-- region: 840 , number of probes: 120\n",
      "-- region: 841 , number of probes: 120\n",
      "-- region: 843 , number of probes: 120\n",
      "-- region: 844 , number of probes: 120\n",
      "-- region: 845 , number of probes: 120\n",
      "-- region: 846 , number of probes: 120\n",
      "-- region: 847 , number of probes: 120\n",
      "-- region: 848 , number of probes: 120\n",
      "-- region: 849 , number of probes: 109\n",
      "-- region: 850 , number of probes: 120\n",
      "-- region: 851 , number of probes: 120\n",
      "-- region: 852 , number of probes: 120\n",
      "-- region: 853 , number of probes: 120\n",
      "-- region: 854 , number of probes: 120\n",
      "-- region: 855 , number of probes: 120\n",
      "-- region: 856 , number of probes: 120\n",
      "-- region: 857 , number of probes: 120\n",
      "-- region: 858 , number of probes: 120\n",
      "-- region: 859 , number of probes: 120\n",
      "-- region: 860 , number of probes: 120\n",
      "-- region: 861 , number of probes: 120\n",
      "-- region: 862 , number of probes: 120\n",
      "-- region: 863 , number of probes: 120\n",
      "-- region: 864 , number of probes: 120\n",
      "-- region: 865 , number of probes: 120\n",
      "-- region: 866 , number of probes: 120\n",
      "-- region: 867 , number of probes: 120\n",
      "-- region: 868 , number of probes: 120\n",
      "-- region: 869 , number of probes: 120\n",
      "-- region: 870 , number of probes: 120\n",
      "-- region: 871 , number of probes: 120\n",
      "-- region: 872 , number of probes: 120\n",
      "-- region: 873 , number of probes: 120\n",
      "-- region: 874 , number of probes: 120\n",
      "-- region: 875 , number of probes: 120\n",
      "-- region: 876 , number of probes: 120\n",
      "-- region: 877 , number of probes: 120\n",
      "-- region: 878 , number of probes: 120\n",
      "-- region: 888 , number of probes: 120\n",
      "-- region: 889 , number of probes: 120\n",
      "-- region: 890 , number of probes: 120\n",
      "-- region: 891 , number of probes: 120\n",
      "-- region: 892 , number of probes: 120\n",
      "-- region: 893 , number of probes: 120\n",
      "-- region: 894 , number of probes: 120\n",
      "-- region: 895 , number of probes: 120\n",
      "-- region: 896 , number of probes: 120\n",
      "-- region: 897 , number of probes: 120\n",
      "-- region: 898 , number of probes: 120\n",
      "-- region: 899 , number of probes: 120\n",
      "-- region: 900 , number of probes: 120\n",
      "-- region: 901 , number of probes: 120\n",
      "-- region: 902 , number of probes: 120\n",
      "-- region: 903 , number of probes: 120\n",
      "-- region: 904 , number of probes: 120\n",
      "-- region: 905 , number of probes: 120\n",
      "-- region: 906 , number of probes: 120\n",
      "-- region: 907 , number of probes: 120\n",
      "-- region: 908 , number of probes: 120\n",
      "-- region: 909 , number of probes: 120\n",
      "-- region: 910 , number of probes: 120\n",
      "-- region: 911 , number of probes: 120\n",
      "-- region: 912 , number of probes: 120\n",
      "-- region: 913 , number of probes: 120\n",
      "-- region: 914 , number of probes: 120\n",
      "-- region: 915 , number of probes: 120\n",
      "-- region: 916 , number of probes: 120\n",
      "-- region: 917 , number of probes: 120\n",
      "-- region: 918 , number of probes: 120\n",
      "-- region: 919 , number of probes: 120\n",
      "-- region: 920 , number of probes: 120\n",
      "-- region: 921 , number of probes: 120\n",
      "-- region: 922 , number of probes: 120\n",
      "-- region: 923 , number of probes: 120\n",
      "-- region: 924 , number of probes: 120\n",
      "-- region: 925 , number of probes: 120\n",
      "-- region: 926 , number of probes: 120\n",
      "-- region: 927 , number of probes: 120\n",
      "-- region: 928 , number of probes: 120\n",
      "-- region: 929 , number of probes: 120\n",
      "-- region: 930 , number of probes: 120\n",
      "-- region: 931 , number of probes: 120\n",
      "-- region: 932 , number of probes: 120\n",
      "-- region: 933 , number of probes: 120\n",
      "-- region: 934 , number of probes: 120\n",
      "-- region: 935 , number of probes: 120\n",
      "-- region: 949 , number of probes: 120\n",
      "-- region: 950 , number of probes: 120\n",
      "-- region: 951 , number of probes: 120\n",
      "-- region: 952 , number of probes: 120\n",
      "-- region: 953 , number of probes: 120\n",
      "-- region: 954 , number of probes: 81\n",
      "-- region: 955 , number of probes: 120\n",
      "-- region: 956 , number of probes: 120\n",
      "-- region: 957 , number of probes: 120\n",
      "-- region: 958 , number of probes: 120\n",
      "-- region: 959 , number of probes: 120\n",
      "-- region: 960 , number of probes: 120\n",
      "-- region: 961 , number of probes: 120\n",
      "-- region: 962 , number of probes: 120\n",
      "-- region: 963 , number of probes: 120\n",
      "-- region: 964 , number of probes: 120\n",
      "-- region: 965 , number of probes: 120\n",
      "-- region: 966 , number of probes: 120\n",
      "-- region: 967 , number of probes: 120\n",
      "-- region: 968 , number of probes: 120\n",
      "-- region: 969 , number of probes: 120\n",
      "-- region: 970 , number of probes: 80\n",
      "-- region: 971 , number of probes: 120\n",
      "-- region: 972 , number of probes: 120\n",
      "-- region: 973 , number of probes: 120\n",
      "-- region: 974 , number of probes: 120\n",
      "-- region: 975 , number of probes: 120\n",
      "-- region: 976 , number of probes: 120\n",
      "-- region: 977 , number of probes: 120\n",
      "-- region: 978 , number of probes: 120\n",
      "-- region: 979 , number of probes: 120\n",
      "-- region: 980 , number of probes: 120\n",
      "-- region: 981 , number of probes: 120\n",
      "-- region: 982 , number of probes: 120\n",
      "-- region: 983 , number of probes: 120\n",
      "-- region: 984 , number of probes: 120\n",
      "-- region: 985 , number of probes: 120\n",
      "-- region: 986 , number of probes: 120\n",
      "-- region: 987 , number of probes: 120\n",
      "-- region: 988 , number of probes: 120\n",
      "-- region: 989 , number of probes: 120\n",
      "-- region: 990 , number of probes: 120\n",
      "-- region: 991 , number of probes: 120\n",
      "-- region: 992 , number of probes: 120\n",
      "-- region: 993 , number of probes: 120\n",
      "-- region: 994 , number of probes: 120\n",
      "-- region: 995 , number of probes: 120\n",
      "-- region: 996 , number of probes: 120\n",
      "-- region: 997 , number of probes: 120\n",
      "-- region: 999 , number of probes: 120\n",
      "-- region: 1000 , number of probes: 120\n",
      "-- region: 1001 , number of probes: 120\n",
      "-- region: 1002 , number of probes: 120\n",
      "-- region: 1003 , number of probes: 120\n",
      "-- region: 1004 , number of probes: 120\n",
      "-- region: 1005 , number of probes: 120\n",
      "-- region: 1020 , number of probes: 49\n",
      "-- region: 1023 , number of probes: 120\n",
      "-- region: 1024 , number of probes: 120\n",
      "-- region: 1025 , number of probes: 120\n",
      "-- region: 1029 , number of probes: 120\n",
      "-- region: 1030 , number of probes: 120\n",
      "-- region: 1031 , number of probes: 120\n",
      "-- region: 1032 , number of probes: 120\n",
      "-- region: 1033 , number of probes: 120\n",
      "-- region: 1034 , number of probes: 120\n",
      "-- region: 1035 , number of probes: 120\n",
      "-- region: 1036 , number of probes: 120\n",
      "-- region: 1037 , number of probes: 120\n",
      "-- region: 1038 , number of probes: 120\n",
      "-- region: 1039 , number of probes: 120\n",
      "-- region: 1040 , number of probes: 120\n",
      "-- region: 1041 , number of probes: 120\n",
      "-- region: 1042 , number of probes: 120\n",
      "-- region: 1043 , number of probes: 105\n",
      "-- region: 1044 , number of probes: 120\n",
      "-- region: 1045 , number of probes: 120\n",
      "-- region: 1046 , number of probes: 120\n",
      "-- region: 1047 , number of probes: 120\n",
      "-- region: 1048 , number of probes: 120\n",
      "-- region: 1049 , number of probes: 120\n",
      "-- region: 1050 , number of probes: 120\n",
      "-- region: 1051 , number of probes: 120\n",
      "-- region: 1052 , number of probes: 120\n",
      "-- region: 1053 , number of probes: 120\n",
      "-- region: 1054 , number of probes: 120\n",
      "-- region: 1055 , number of probes: 120\n",
      "-- region: 1056 , number of probes: 120\n",
      "-- region: 1057 , number of probes: 120\n",
      "-- region: 1058 , number of probes: 120\n",
      "-- region: 1059 , number of probes: 120\n",
      "-- region: 1060 , number of probes: 120\n",
      "-- region: 1061 , number of probes: 120\n",
      "-- region: 1062 , number of probes: 120\n",
      "-- region: 1063 , number of probes: 120\n",
      "-- region: 1064 , number of probes: 120\n",
      "-- region: 1065 , number of probes: 120\n",
      "-- region: 1066 , number of probes: 120\n",
      "-- region: 1067 , number of probes: 120\n",
      "-- region: 1068 , number of probes: 120\n",
      "-- region: 1069 , number of probes: 120\n",
      "-- region: 1070 , number of probes: 120\n",
      "-- region: 1071 , number of probes: 120\n",
      "-- region: 1072 , number of probes: 120\n",
      "-- region: 1073 , number of probes: 120\n",
      "-- region: 1074 , number of probes: 120\n",
      "-- region: 1075 , number of probes: 120\n",
      "-- region: 1076 , number of probes: 120\n",
      "-- region: 1077 , number of probes: 86\n",
      "-- region: 1078 , number of probes: 120\n",
      "-- region: 1079 , number of probes: 120\n",
      "-- region: 1080 , number of probes: 120\n",
      "-- region: 1081 , number of probes: 120\n",
      "-- region: 1082 , number of probes: 120\n",
      "-- region: 1083 , number of probes: 120\n",
      "-- region: 1085 , number of probes: 120\n",
      "-- region: 1087 , number of probes: 120\n",
      "-- region: 1089 , number of probes: 120\n",
      "-- region: 1090 , number of probes: 120\n",
      "-- region: 1091 , number of probes: 120\n",
      "-- region: 1092 , number of probes: 120\n",
      "-- region: 1093 , number of probes: 120\n",
      "-- region: 1094 , number of probes: 120\n",
      "-- region: 1097 , number of probes: 120\n",
      "-- region: 1105 , number of probes: 120\n",
      "-- region: 1106 , number of probes: 120\n",
      "-- region: 1107 , number of probes: 120\n",
      "-- region: 1108 , number of probes: 120\n",
      "-- region: 1109 , number of probes: 120\n",
      "-- region: 1110 , number of probes: 120\n",
      "-- region: 1111 , number of probes: 120\n",
      "-- region: 1112 , number of probes: 120\n",
      "-- region: 1113 , number of probes: 120\n",
      "-- region: 1114 , number of probes: 120\n",
      "-- region: 1115 , number of probes: 120\n",
      "-- region: 1116 , number of probes: 120\n",
      "-- region: 1117 , number of probes: 120\n",
      "-- region: 1118 , number of probes: 120\n",
      "-- region: 1119 , number of probes: 120\n",
      "-- region: 1120 , number of probes: 120\n",
      "-- region: 1121 , number of probes: 120\n",
      "-- region: 1122 , number of probes: 120\n",
      "-- region: 1123 , number of probes: 120\n",
      "-- region: 1124 , number of probes: 120\n",
      "-- region: 1125 , number of probes: 120\n",
      "-- region: 1126 , number of probes: 120\n",
      "-- region: 1127 , number of probes: 120\n",
      "-- region: 1128 , number of probes: 120\n",
      "-- region: 1129 , number of probes: 120\n",
      "-- region: 1130 , number of probes: 120\n",
      "-- region: 1131 , number of probes: 120\n",
      "-- region: 1132 , number of probes: 120\n",
      "-- region: 1133 , number of probes: 120\n",
      "-- region: 1134 , number of probes: 120\n",
      "-- region: 1135 , number of probes: 120\n",
      "-- region: 1136 , number of probes: 120\n",
      "-- region: 1137 , number of probes: 120\n",
      "-- region: 1138 , number of probes: 120\n",
      "-- region: 1139 , number of probes: 120\n",
      "-- region: 1140 , number of probes: 120\n",
      "-- region: 1141 , number of probes: 120\n",
      "-- region: 1142 , number of probes: 120\n",
      "-- region: 1143 , number of probes: 120\n",
      "-- region: 1144 , number of probes: 117\n",
      "-- region: 1145 , number of probes: 120\n",
      "-- region: 1146 , number of probes: 92\n",
      "-- region: 1147 , number of probes: 120\n",
      "-- region: 1148 , number of probes: 98\n",
      "-- region: 1152 , number of probes: 120\n",
      "-- region: 1153 , number of probes: 120\n",
      "-- region: 1154 , number of probes: 52\n",
      "-- region: 1155 , number of probes: 120\n",
      "-- region: 1156 , number of probes: 120\n",
      "-- region: 1157 , number of probes: 120\n",
      "-- region: 1160 , number of probes: 120\n",
      "-- region: 1162 , number of probes: 120\n",
      "-- region: 1163 , number of probes: 120\n",
      "-- region: 1165 , number of probes: 120\n",
      "-- region: 1166 , number of probes: 120\n",
      "-- region: 1167 , number of probes: 120\n",
      "-- region: 1168 , number of probes: 120\n",
      "-- region: 1169 , number of probes: 120\n",
      "-- region: 1170 , number of probes: 120\n",
      "-- region: 1171 , number of probes: 120\n",
      "-- region: 1172 , number of probes: 120\n",
      "-- region: 1173 , number of probes: 90\n",
      "-- region: 1174 , number of probes: 115\n",
      "-- region: 1176 , number of probes: 120\n",
      "-- region: 1177 , number of probes: 120\n",
      "-- region: 1178 , number of probes: 120\n",
      "-- region: 1179 , number of probes: 120\n",
      "-- region: 1180 , number of probes: 120\n",
      "-- region: 1181 , number of probes: 120\n",
      "-- region: 1182 , number of probes: 120\n",
      "-- region: 1183 , number of probes: 120\n",
      "-- region: 1184 , number of probes: 120\n",
      "-- region: 1186 , number of probes: 120\n",
      "-- region: 1187 , number of probes: 120\n",
      "-- region: 1188 , number of probes: 77\n",
      "-- region: 1189 , number of probes: 120\n",
      "-- region: 1190 , number of probes: 120\n",
      "-- region: 1191 , number of probes: 120\n",
      "-- region: 1192 , number of probes: 120\n",
      "-- region: 1193 , number of probes: 120\n",
      "-- region: 1194 , number of probes: 120\n",
      "-- region: 1195 , number of probes: 120\n",
      "-- region: 1196 , number of probes: 120\n",
      "-- region: 1197 , number of probes: 120\n",
      "-- region: 1203 , number of probes: 120\n",
      "-- region: 1204 , number of probes: 120\n",
      "-- region: 1205 , number of probes: 120\n",
      "-- region: 1206 , number of probes: 86\n",
      "-- region: 1207 , number of probes: 120\n",
      "-- region: 1208 , number of probes: 120\n",
      "-- region: 1209 , number of probes: 120\n",
      "-- region: 1210 , number of probes: 120\n",
      "-- region: 1211 , number of probes: 120\n",
      "-- region: 1212 , number of probes: 120\n",
      "-- region: 1213 , number of probes: 120\n",
      "-- region: 1214 , number of probes: 120\n",
      "-- region: 1215 , number of probes: 120\n",
      "-- region: 1216 , number of probes: 120\n",
      "-- region: 1217 , number of probes: 120\n",
      "-- region: 1218 , number of probes: 120\n",
      "-- region: 1219 , number of probes: 120\n",
      "-- region: 1220 , number of probes: 120\n",
      "-- region: 1221 , number of probes: 120\n",
      "-- region: 1222 , number of probes: 120\n",
      "-- region: 1224 , number of probes: 120\n",
      "-- region: 1225 , number of probes: 120\n",
      "-- region: 1226 , number of probes: 120\n",
      "-- region: 1227 , number of probes: 120\n",
      "-- region: 1228 , number of probes: 120\n",
      "-- region: 1229 , number of probes: 120\n",
      "-- region: 1230 , number of probes: 120\n",
      "-- region: 1231 , number of probes: 120\n",
      "-- region: 1232 , number of probes: 120\n",
      "-- region: 1233 , number of probes: 120\n",
      "-- region: 1234 , number of probes: 120\n",
      "-- region: 1235 , number of probes: 120\n",
      "-- region: 1236 , number of probes: 120\n",
      "-- region: 1237 , number of probes: 120\n",
      "-- region: 1238 , number of probes: 120\n",
      "-- region: 1239 , number of probes: 120\n",
      "-- region: 1240 , number of probes: 120\n",
      "-- region: 1241 , number of probes: 120\n",
      "-- region: 1242 , number of probes: 120\n",
      "-- region: 1243 , number of probes: 120\n",
      "-- region: 1244 , number of probes: 120\n",
      "-- region: 1245 , number of probes: 120\n",
      "-- region: 1246 , number of probes: 120\n",
      "-- region: 1247 , number of probes: 120\n",
      "-- region: 1248 , number of probes: 120\n",
      "-- region: 1249 , number of probes: 120\n",
      "-- region: 1250 , number of probes: 120\n",
      "-- region: 1251 , number of probes: 120\n",
      "-- region: 1252 , number of probes: 120\n",
      "-- region: 1253 , number of probes: 120\n",
      "-- region: 1254 , number of probes: 120\n",
      "-- region: 1255 , number of probes: 120\n",
      "-- region: 1256 , number of probes: 120\n",
      "-- region: 1257 , number of probes: 120\n",
      "-- region: 1258 , number of probes: 120\n",
      "-- region: 1259 , number of probes: 120\n",
      "-- region: 1260 , number of probes: 120\n",
      "-- region: 1261 , number of probes: 120\n",
      "-- region: 1262 , number of probes: 63\n",
      "-- region: 1263 , number of probes: 120\n",
      "-- region: 1264 , number of probes: 120\n",
      "-- region: 1265 , number of probes: 81\n",
      "-- region: 1269 , number of probes: 21\n",
      "-- region: 1270 , number of probes: 120\n",
      "-- region: 1271 , number of probes: 120\n",
      "-- region: 1272 , number of probes: 120\n",
      "-- region: 1273 , number of probes: 120\n",
      "-- region: 1274 , number of probes: 120\n",
      "-- region: 1275 , number of probes: 120\n",
      "-- region: 1276 , number of probes: 120\n",
      "-- region: 1277 , number of probes: 120\n",
      "-- region: 1278 , number of probes: 120\n",
      "-- region: 1279 , number of probes: 120\n",
      "-- region: 1280 , number of probes: 120\n",
      "-- region: 1281 , number of probes: 120\n",
      "-- region: 1282 , number of probes: 120\n",
      "-- region: 1284 , number of probes: 120\n",
      "-- region: 1285 , number of probes: 120\n",
      "-- region: 1286 , number of probes: 120\n",
      "-- region: 1287 , number of probes: 120\n",
      "-- region: 1288 , number of probes: 120\n",
      "-- region: 1289 , number of probes: 120\n",
      "-- region: 1290 , number of probes: 120\n",
      "-- region: 1291 , number of probes: 120\n",
      "-- region: 1292 , number of probes: 120\n",
      "-- region: 1293 , number of probes: 120\n",
      "-- region: 1294 , number of probes: 78\n",
      "-- region: 1295 , number of probes: 120\n",
      "-- region: 1297 , number of probes: 120\n",
      "-- region: 1298 , number of probes: 120\n",
      "-- region: 1299 , number of probes: 120\n",
      "-- region: 1300 , number of probes: 120\n",
      "-- region: 1301 , number of probes: 120\n",
      "-- region: 1302 , number of probes: 120\n",
      "-- region: 1303 , number of probes: 120\n",
      "-- region: 1304 , number of probes: 120\n",
      "-- region: 1305 , number of probes: 120\n",
      "-- region: 1306 , number of probes: 120\n",
      "-- region: 1307 , number of probes: 120\n",
      "-- region: 1308 , number of probes: 120\n",
      "-- region: 1309 , number of probes: 120\n",
      "-- region: 1310 , number of probes: 120\n",
      "-- region: 1311 , number of probes: 120\n",
      "-- region: 1312 , number of probes: 120\n",
      "-- region: 1313 , number of probes: 120\n",
      "-- region: 1314 , number of probes: 120\n",
      "-- region: 1315 , number of probes: 120\n",
      "-- region: 1316 , number of probes: 120\n",
      "-- region: 1317 , number of probes: 120\n",
      "-- region: 1318 , number of probes: 120\n",
      "-- region: 1319 , number of probes: 120\n",
      "-- region: 1320 , number of probes: 120\n",
      "-- region: 1321 , number of probes: 120\n",
      "-- region: 1322 , number of probes: 120\n",
      "-- region: 1323 , number of probes: 120\n",
      "-- region: 1324 , number of probes: 120\n",
      "-- region: 1325 , number of probes: 120\n",
      "-- region: 1330 , number of probes: 120\n",
      "-- region: 1331 , number of probes: 120\n",
      "-- region: 1332 , number of probes: 120\n",
      "-- region: 1333 , number of probes: 120\n",
      "-- region: 1334 , number of probes: 120\n",
      "-- region: 1335 , number of probes: 120\n",
      "-- region: 1336 , number of probes: 120\n",
      "-- region: 1337 , number of probes: 120\n",
      "-- region: 1338 , number of probes: 120\n",
      "-- region: 1339 , number of probes: 120\n",
      "-- region: 1340 , number of probes: 120\n",
      "-- region: 1341 , number of probes: 120\n",
      "-- region: 1342 , number of probes: 120\n",
      "-- region: 1343 , number of probes: 120\n",
      "-- region: 1344 , number of probes: 120\n",
      "-- region: 1345 , number of probes: 120\n",
      "-- region: 1346 , number of probes: 120\n",
      "-- region: 1347 , number of probes: 120\n",
      "-- region: 1348 , number of probes: 88\n",
      "-- region: 1349 , number of probes: 120\n",
      "-- region: 1350 , number of probes: 120\n",
      "-- region: 1351 , number of probes: 120\n",
      "-- region: 1352 , number of probes: 120\n",
      "-- region: 1353 , number of probes: 120\n",
      "-- region: 1354 , number of probes: 120\n",
      "-- region: 1355 , number of probes: 120\n",
      "-- region: 1356 , number of probes: 120\n",
      "-- region: 1357 , number of probes: 120\n",
      "-- region: 1358 , number of probes: 120\n",
      "-- region: 1359 , number of probes: 120\n",
      "-- region: 1360 , number of probes: 120\n",
      "-- region: 1361 , number of probes: 120\n",
      "-- region: 1362 , number of probes: 120\n",
      "-- region: 1363 , number of probes: 120\n",
      "-- region: 1378 , number of probes: 120\n",
      "-- region: 1379 , number of probes: 120\n",
      "-- region: 1380 , number of probes: 120\n",
      "-- region: 1381 , number of probes: 120\n",
      "-- region: 1382 , number of probes: 120\n",
      "-- region: 1383 , number of probes: 120\n",
      "-- region: 1384 , number of probes: 120\n",
      "-- region: 1385 , number of probes: 120\n",
      "-- region: 1386 , number of probes: 117\n",
      "-- region: 1387 , number of probes: 120\n",
      "-- region: 1388 , number of probes: 120\n",
      "-- region: 1389 , number of probes: 120\n",
      "-- region: 1390 , number of probes: 120\n",
      "-- region: 1391 , number of probes: 120\n",
      "-- region: 1392 , number of probes: 120\n",
      "-- region: 1393 , number of probes: 120\n",
      "-- region: 1394 , number of probes: 120\n",
      "-- region: 1395 , number of probes: 120\n",
      "-- region: 1396 , number of probes: 120\n",
      "-- region: 1398 , number of probes: 120\n",
      "-- region: 1399 , number of probes: 120\n",
      "-- region: 1400 , number of probes: 120\n",
      "-- region: 1401 , number of probes: 120\n",
      "-- region: 1402 , number of probes: 120\n",
      "-- region: 1403 , number of probes: 120\n",
      "-- region: 1404 , number of probes: 120\n",
      "-- region: 1405 , number of probes: 120\n",
      "-- region: 1406 , number of probes: 120\n",
      "-- region: 1408 , number of probes: 120\n",
      "-- region: 1409 , number of probes: 120\n",
      "-- region: 1427 , number of probes: 120\n",
      "-- region: 1429 , number of probes: 120\n",
      "-- region: 1430 , number of probes: 120\n",
      "-- region: 1434 , number of probes: 120\n",
      "-- region: 1435 , number of probes: 120\n",
      "-- region: 1436 , number of probes: 120\n",
      "-- region: 1437 , number of probes: 120\n",
      "-- region: 1438 , number of probes: 120\n",
      "-- region: 1439 , number of probes: 120\n",
      "-- region: 1440 , number of probes: 120\n",
      "-- region: 1441 , number of probes: 120\n",
      "-- region: 1442 , number of probes: 120\n",
      "-- region: 1443 , number of probes: 120\n",
      "-- region: 1444 , number of probes: 120\n",
      "-- region: 1445 , number of probes: 120\n",
      "-- region: 1446 , number of probes: 120\n",
      "-- region: 1447 , number of probes: 120\n",
      "-- region: 1448 , number of probes: 120\n",
      "-- region: 1449 , number of probes: 120\n",
      "-- region: 1450 , number of probes: 120\n",
      "-- region: 1451 , number of probes: 120\n",
      "-- region: 1452 , number of probes: 109\n",
      "-- region: 1453 , number of probes: 120\n",
      "-- region: 1454 , number of probes: 120\n",
      "-- region: 1455 , number of probes: 120\n",
      "-- region: 1456 , number of probes: 120\n",
      "-- region: 1457 , number of probes: 120\n",
      "-- region: 1458 , number of probes: 120\n",
      "-- region: 1459 , number of probes: 120\n",
      "-- region: 1461 , number of probes: 120\n",
      "-- region: 1462 , number of probes: 120\n",
      "-- region: 1463 , number of probes: 120\n",
      "-- region: 1464 , number of probes: 120\n",
      "-- region: 1465 , number of probes: 120\n",
      "-- region: 1466 , number of probes: 120\n",
      "-- region: 1467 , number of probes: 120\n",
      "-- region: 1468 , number of probes: 120\n",
      "-- region: 1469 , number of probes: 120\n",
      "-- region: 1470 , number of probes: 120\n",
      "-- region: 1471 , number of probes: 120\n",
      "-- region: 1472 , number of probes: 120\n",
      "-- region: 1473 , number of probes: 100\n",
      "-- region: 1474 , number of probes: 120\n",
      "-- region: 1475 , number of probes: 120\n",
      "-- region: 1476 , number of probes: 120\n",
      "-- region: 1477 , number of probes: 120\n",
      "-- region: 1478 , number of probes: 120\n",
      "-- region: 1479 , number of probes: 120\n",
      "-- region: 1480 , number of probes: 104\n",
      "-- region: 1481 , number of probes: 120\n",
      "-- region: 1482 , number of probes: 120\n",
      "-- region: 1485 , number of probes: 120\n",
      "-- region: 1486 , number of probes: 120\n",
      "-- region: 1487 , number of probes: 120\n",
      "-- region: 1488 , number of probes: 120\n",
      "-- region: 1489 , number of probes: 73\n",
      "-- region: 1490 , number of probes: 113\n",
      "-- region: 1491 , number of probes: 120\n",
      "-- region: 1492 , number of probes: 120\n",
      "-- region: 1493 , number of probes: 114\n",
      "-- region: 1494 , number of probes: 120\n",
      "-- region: 1495 , number of probes: 120\n",
      "-- region: 1496 , number of probes: 63\n",
      "-- region: 1497 , number of probes: 120\n",
      "-- region: 1498 , number of probes: 120\n",
      "-- region: 1499 , number of probes: 120\n",
      "-- region: 1500 , number of probes: 120\n",
      "-- region: 1501 , number of probes: 120\n",
      "-- region: 1502 , number of probes: 110\n",
      "-- region: 1503 , number of probes: 120\n",
      "-- region: 1504 , number of probes: 120\n",
      "-- region: 1505 , number of probes: 120\n",
      "-- region: 1506 , number of probes: 120\n",
      "-- region: 1507 , number of probes: 120\n",
      "-- region: 1508 , number of probes: 120\n",
      "-- region: 1509 , number of probes: 120\n",
      "-- region: 1510 , number of probes: 120\n",
      "-- region: 1511 , number of probes: 120\n",
      "-- region: 1512 , number of probes: 120\n",
      "-- region: 1513 , number of probes: 120\n",
      "-- region: 1514 , number of probes: 120\n",
      "-- region: 1515 , number of probes: 120\n",
      "-- region: 1516 , number of probes: 100\n",
      "-- region: 1517 , number of probes: 120\n",
      "-- region: 1518 , number of probes: 120\n",
      "-- region: 1519 , number of probes: 120\n",
      "-- region: 1520 , number of probes: 120\n",
      "-- region: 1521 , number of probes: 120\n",
      "-- region: 1525 , number of probes: 47\n",
      "-- region: 1526 , number of probes: 55\n",
      "-- region: 1527 , number of probes: 68\n",
      "-- region: 1529 , number of probes: 120\n",
      "-- region: 1530 , number of probes: 120\n",
      "-- region: 1533 , number of probes: 30\n",
      "-- region: 1534 , number of probes: 120\n",
      "-- region: 1535 , number of probes: 120\n",
      "-- region: 1536 , number of probes: 120\n",
      "-- region: 1537 , number of probes: 120\n",
      "-- region: 1538 , number of probes: 120\n",
      "-- region: 1541 , number of probes: 120\n",
      "-- region: 1542 , number of probes: 120\n",
      "-- region: 1548 , number of probes: 102\n",
      "- Number of probes kept: 159880\n",
      "- Saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-14_human_brain\\human_genome_library\\.\\blast_full_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "# screen\n",
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "num_probes_per_region = 120\n",
    "kept_pbs, blast_keep_dic, hard_count_list, soft_count_list = check.Screening_Probes_by_Blast(library_folder, \n",
    "                                                                                             num_probes_per_region,\n",
    "                                                                                             hard_thres=target_len,\n",
    "                                                                                             soft_count_th=50,\n",
    "                                                                                             )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "402daac7",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "30612c84",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "postanalysis",
   "language": "python",
   "name": "postanalysis"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
