{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Table of contents\n",
    "\n",
    "\n",
    "> 0. [Minimum required packages and settings](#0)\n",
    ">>\n",
    ">> 0.1: [import required packages](#0.1)\n",
    ">\n",
    "> 1. [Extract region sequences](#1)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<a id='0.1'></a>\n",
    "## 0.1 load required packages"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2022-01-16 16:18:16,783 [INFO] WRITING LOG OUTPUT TO C:\\Users\\shiwei\\.cellpose\\run.log\n",
      "34428\n"
     ]
    }
   ],
   "source": [
    "%run \"C:\\Users\\shiwei\\Documents\\ImageAnalysis3\\required_files\\Startup_py3.py\"\n",
    "sys.path.append(r\"C:\\Users\\shiwei\\Documents\")\n",
    "\n",
    "import ImageAnalysis3 as ia\n",
    "%matplotlib notebook\n",
    "\n",
    "from ImageAnalysis3 import *\n",
    "print(os.getpid())\n",
    "\n",
    "# library design specific tools\n",
    "from ImageAnalysis3.library_tools import LibraryDesigner as ld\n",
    "from ImageAnalysis3.library_tools import LibraryTools as lt\n",
    "# biopython imports\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "from Bio.Blast.Applications import NcbiblastnCommandline\n",
    "from Bio.Blast import NCBIXML"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<a id='1'></a>\n",
    "# 1 Extract region sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "## Some folders\n",
    "# human genome\n",
    "reference_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl'\n",
    "genome_folder = os.path.join(reference_folder, 'Genome')\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- library_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\n",
      "-- sequence_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\n",
      "-- report_folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\reports\n"
     ]
    }
   ],
   "source": [
    "resolution = 0\n",
    "flanking = 10000\n",
    "# folder for sub-pool\n",
    "library_folder = os.path.join(pool_folder, f'mop_SE_1000')\n",
    "if not os.path.exists(library_folder):\n",
    "    print(f\"create library folder: {library_folder}\")\n",
    "    os.makedirs(library_folder)\n",
    "# folder for fasta sequences\n",
    "sequence_folder = os.path.join(library_folder, 'sequences')\n",
    "if not os.path.exists(sequence_folder):\n",
    "    print(f\"create sequence folder: {sequence_folder}\")\n",
    "    os.makedirs(sequence_folder)\n",
    "# folder to save result probes\n",
    "report_folder = os.path.join(library_folder, 'reports')\n",
    "if not os.path.exists(report_folder):\n",
    "    print(f\"create report folder: {report_folder}\")\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "print(f\"-- library_folder: {library_folder}\")\n",
    "print(f\"-- sequence_folder: {sequence_folder}\")\n",
    "print(f\"-- report_folder: {report_folder}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1.1 select regions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm.notebook import tqdm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "\n",
    "sel_SE_df = pd.read_excel(library_folder+os.sep+'merged_selected_SE_all_cell_types.xlsx', index_col=None)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {},
   "outputs": [],
   "source": [
    "sel_SE_df.head()\n",
    "\n",
    "sel_SE_df_chrX = sel_SE_df[984:]\n",
    "sel_SE_df_chrN = sel_SE_df[:984]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-71-438bec6918df>:1: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  sel_SE_df_chrN['sort_CHROM'] = [int(i.split('chr')[-1]) for i in sel_SE_df_chrN['CHROM']]\n",
      "<ipython-input-71-438bec6918df>:2: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  sel_SE_df_chrX['sort_CHROM'] = 20\n"
     ]
    }
   ],
   "source": [
    "sel_SE_df_chrN['sort_CHROM'] = [int(i.split('chr')[-1]) for i in sel_SE_df_chrN['CHROM']]\n",
    "sel_SE_df_chrX['sort_CHROM'] = 20\n",
    "\n",
    "sel_SE_df_sort_chrN = sel_SE_df_chrN.sort_values(by =['sort_CHROM', 'merged_START'])\n",
    "sel_SE_df_sort_chrX = sel_SE_df_chrX.sort_values(by =['sort_CHROM', 'merged_START'])\n",
    "#sel_SE_df_sort[100:200]\n",
    "\n",
    "sel_SE_df_sort_all = pd.concat([sel_SE_df_sort_chrN, sel_SE_df_sort_chrX])\n",
    "sel_SE_df_new = sel_SE_df_sort_all.reset_index()\n",
    "\n",
    "sel_SE_df_new['region_name'] = sel_SE_df_new.index +1\n",
    "#sel_SE_df_new[:100]\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {},
   "outputs": [],
   "source": [
    "sel_SE_df_new.to_excel(library_folder+os.sep+'selected_merged_SE_list.xlsx', index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 78,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index</th>\n",
       "      <th>CHROM</th>\n",
       "      <th>merged_START</th>\n",
       "      <th>merged_STOP</th>\n",
       "      <th>merged_CELL_TYPE</th>\n",
       "      <th>merged_initial_index</th>\n",
       "      <th>score</th>\n",
       "      <th>ave_score</th>\n",
       "      <th>merged_SIZE</th>\n",
       "      <th>sort_CHROM</th>\n",
       "      <th>region_name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>70</td>\n",
       "      <td>chr1</td>\n",
       "      <td>187645279</td>\n",
       "      <td>187694850</td>\n",
       "      <td>['Pvalb']</td>\n",
       "      <td>[17244]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>49571</td>\n",
       "      <td>1</td>\n",
       "      <td>71</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>71</td>\n",
       "      <td>chr1</td>\n",
       "      <td>189017529</td>\n",
       "      <td>189066052</td>\n",
       "      <td>['L4_5_IT', 'L6_CT', 'L5_6_NP']</td>\n",
       "      <td>[4357, 11167, 5708]</td>\n",
       "      <td>[0.7846  0.98129 0.90823]</td>\n",
       "      <td>0.891373</td>\n",
       "      <td>48523</td>\n",
       "      <td>1</td>\n",
       "      <td>72</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>72</td>\n",
       "      <td>chr1</td>\n",
       "      <td>189185060</td>\n",
       "      <td>189229230</td>\n",
       "      <td>['Astro']</td>\n",
       "      <td>[601]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>44170</td>\n",
       "      <td>1</td>\n",
       "      <td>73</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>73</td>\n",
       "      <td>chr1</td>\n",
       "      <td>189490922</td>\n",
       "      <td>189533419</td>\n",
       "      <td>['L5_IT']</td>\n",
       "      <td>[7163]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>42497</td>\n",
       "      <td>1</td>\n",
       "      <td>74</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>74</td>\n",
       "      <td>chr1</td>\n",
       "      <td>189876129</td>\n",
       "      <td>189928237</td>\n",
       "      <td>['Pvalb']</td>\n",
       "      <td>[17149]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>52108</td>\n",
       "      <td>1</td>\n",
       "      <td>75</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75</th>\n",
       "      <td>75</td>\n",
       "      <td>chr1</td>\n",
       "      <td>193085619</td>\n",
       "      <td>193123645</td>\n",
       "      <td>['L4_5_IT', 'L5_IT']</td>\n",
       "      <td>[3426, 6692]</td>\n",
       "      <td>[1.      0.97005]</td>\n",
       "      <td>0.985025</td>\n",
       "      <td>38026</td>\n",
       "      <td>1</td>\n",
       "      <td>76</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76</th>\n",
       "      <td>515</td>\n",
       "      <td>chr2</td>\n",
       "      <td>3139373</td>\n",
       "      <td>3173216</td>\n",
       "      <td>['L5_PT']</td>\n",
       "      <td>[8452]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>33843</td>\n",
       "      <td>2</td>\n",
       "      <td>77</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>77</th>\n",
       "      <td>516</td>\n",
       "      <td>chr2</td>\n",
       "      <td>4036085</td>\n",
       "      <td>4073325</td>\n",
       "      <td>['Astro']</td>\n",
       "      <td>[282]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>37240</td>\n",
       "      <td>2</td>\n",
       "      <td>78</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78</th>\n",
       "      <td>517</td>\n",
       "      <td>chr2</td>\n",
       "      <td>5161601</td>\n",
       "      <td>5200127</td>\n",
       "      <td>['L6_IT', 'L4_5_IT']</td>\n",
       "      <td>[12594, 4307]</td>\n",
       "      <td>[0.93911 0.99842]</td>\n",
       "      <td>0.968765</td>\n",
       "      <td>38526</td>\n",
       "      <td>2</td>\n",
       "      <td>79</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>79</th>\n",
       "      <td>518</td>\n",
       "      <td>chr2</td>\n",
       "      <td>11319886</td>\n",
       "      <td>11375319</td>\n",
       "      <td>['Oligo']</td>\n",
       "      <td>[15952]</td>\n",
       "      <td>[1.]</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>55433</td>\n",
       "      <td>2</td>\n",
       "      <td>80</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    index CHROM  merged_START  merged_STOP                 merged_CELL_TYPE  \\\n",
       "70     70  chr1     187645279    187694850                        ['Pvalb']   \n",
       "71     71  chr1     189017529    189066052  ['L4_5_IT', 'L6_CT', 'L5_6_NP']   \n",
       "72     72  chr1     189185060    189229230                        ['Astro']   \n",
       "73     73  chr1     189490922    189533419                        ['L5_IT']   \n",
       "74     74  chr1     189876129    189928237                        ['Pvalb']   \n",
       "75     75  chr1     193085619    193123645             ['L4_5_IT', 'L5_IT']   \n",
       "76    515  chr2       3139373      3173216                        ['L5_PT']   \n",
       "77    516  chr2       4036085      4073325                        ['Astro']   \n",
       "78    517  chr2       5161601      5200127             ['L6_IT', 'L4_5_IT']   \n",
       "79    518  chr2      11319886     11375319                        ['Oligo']   \n",
       "\n",
       "   merged_initial_index                      score  ave_score  merged_SIZE  \\\n",
       "70              [17244]                       [1.]   1.000000        49571   \n",
       "71  [4357, 11167, 5708]  [0.7846  0.98129 0.90823]   0.891373        48523   \n",
       "72                [601]                       [1.]   1.000000        44170   \n",
       "73               [7163]                       [1.]   1.000000        42497   \n",
       "74              [17149]                       [1.]   1.000000        52108   \n",
       "75         [3426, 6692]          [1.      0.97005]   0.985025        38026   \n",
       "76               [8452]                       [1.]   1.000000        33843   \n",
       "77                [282]                       [1.]   1.000000        37240   \n",
       "78        [12594, 4307]          [0.93911 0.99842]   0.968765        38526   \n",
       "79              [15952]                       [1.]   1.000000        55433   \n",
       "\n",
       "    sort_CHROM  region_name  \n",
       "70           1           71  \n",
       "71           1           72  \n",
       "72           1           73  \n",
       "73           1           74  \n",
       "74           1           75  \n",
       "75           1           76  \n",
       "76           2           77  \n",
       "77           2           78  \n",
       "78           2           79  \n",
       "79           2           80  "
      ]
     },
     "execution_count": 78,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sel_SE_df_new.head()\n",
    "\n",
    "sel_SE_df_new[70:80]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>CHROM</th>\n",
       "      <th>merged_START</th>\n",
       "      <th>merged_STOP</th>\n",
       "      <th>region_name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>chr1</td>\n",
       "      <td>9610471</td>\n",
       "      <td>9646458</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>chr1</td>\n",
       "      <td>9759317</td>\n",
       "      <td>9835402</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>chr1</td>\n",
       "      <td>20898605</td>\n",
       "      <td>20943251</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>chr1</td>\n",
       "      <td>21485446</td>\n",
       "      <td>21554433</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>chr1</td>\n",
       "      <td>21684557</td>\n",
       "      <td>21783163</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>986</th>\n",
       "      <td>chrX</td>\n",
       "      <td>101920335</td>\n",
       "      <td>102005609</td>\n",
       "      <td>987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>987</th>\n",
       "      <td>chrX</td>\n",
       "      <td>136782307</td>\n",
       "      <td>136841006</td>\n",
       "      <td>988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>988</th>\n",
       "      <td>chrX</td>\n",
       "      <td>140491839</td>\n",
       "      <td>140537985</td>\n",
       "      <td>989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>989</th>\n",
       "      <td>chrX</td>\n",
       "      <td>167132110</td>\n",
       "      <td>167181320</td>\n",
       "      <td>990</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>990</th>\n",
       "      <td>chrX</td>\n",
       "      <td>169972881</td>\n",
       "      <td>170022477</td>\n",
       "      <td>991</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>991 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    CHROM  merged_START  merged_STOP  region_name\n",
       "0    chr1       9610471      9646458            1\n",
       "1    chr1       9759317      9835402            2\n",
       "2    chr1      20898605     20943251            3\n",
       "3    chr1      21485446     21554433            4\n",
       "4    chr1      21684557     21783163            5\n",
       "..    ...           ...          ...          ...\n",
       "986  chrX     101920335    102005609          987\n",
       "987  chrX     136782307    136841006          988\n",
       "988  chrX     140491839    140537985          989\n",
       "989  chrX     167132110    167181320          990\n",
       "990  chrX     169972881    170022477          991\n",
       "\n",
       "[991 rows x 4 columns]"
      ]
     },
     "execution_count": 113,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sel_SE_df_for_bed =sel_SE_df_new[['CHROM','merged_START','merged_STOP','region_name']]\n",
    "\n",
    "#sel_SE_df_for_bed['merged_START_str']=[str(i) for i in sel_SE_df_for_bed['merged_START']]\n",
    "#sel_SE_df_for_bed['merged_STOP_str']=[str(i) for i in sel_SE_df_for_bed['merged_STOP']]\n",
    "#sel_SE_df_for_bed['region_name_str']=[str(i) for i in sel_SE_df_for_bed['region_name']]\n",
    "\n",
    "#sel_SE_df_for_bed = sel_SE_df_for_bed[['CHROM','merged_START_str','merged_STOP_str','region_name_str']]\n",
    "sel_SE_df_for_bed"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "metadata": {},
   "outputs": [],
   "source": [
    "# generate a 'bed' file from the processed dataframe\n",
    "\n",
    "region_file = os.path.join(library_folder, 'Seleted_merged_super_enhancer_list.bed')\n",
    "import csv\n",
    "with open(region_file, 'w') as _reg_file:\n",
    "    \n",
    "    #tsv_writer = csv.writer(_reg_file, delimiter='\\t')\n",
    "    #new_line_writer = csv.writer(_reg_file, delimiter='\\n')\n",
    "    df_columns = list(sel_SE_df_for_bed.columns)\n",
    "\n",
    "    for _peak in sel_SE_df_for_bed.iloc():\n",
    "        for  _col in df_columns:\n",
    "            _reg_file.write(str(_peak[_col]))\n",
    "            _reg_file.write('\\t')\n",
    "        _reg_file.write('\\n')\n",
    "         "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Input region file is: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\Seleted_merged_super_enhancer_list.bed\n",
      "- 991 regions loaded from file: Seleted_merged_super_enhancer_list.bed\n",
      "-- searching among 23 references\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.1.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.2.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.3.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.4.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.5.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.6.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.7.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.8.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.9.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.10.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.11.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.12.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.13.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.14.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.15.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.16.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.17.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.18.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.19.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.X.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.X.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.X.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.X.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.X.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.X.fa.\n",
      "-- a match found in file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Genome\\Mus_musculus.GRCm38.dna.chromosome.X.fa.\n",
      "-- saving sequences into folder: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\1_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\2_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\3_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\4_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\5_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\6_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\7_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\8_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\9_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\10_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\11_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\12_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\13_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\14_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\15_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\16_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\17_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\18_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\19_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\20_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\21_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\22_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\23_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\24_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\25_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\26_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\27_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\28_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\29_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\30_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\31_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\32_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\33_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\34_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\35_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\36_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\37_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\38_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\39_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\40_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\41_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\42_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\43_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\44_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\45_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\46_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\47_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\48_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\49_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\50_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\51_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\52_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\53_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\54_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\55_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\56_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\57_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\58_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\59_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\60_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\61_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\62_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\63_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\64_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\65_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\66_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\67_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\68_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\69_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\70_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\71_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\72_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\73_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\74_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\75_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\76_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\77_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\78_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\79_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\80_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\81_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\82_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\83_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\84_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\85_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\86_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\87_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\88_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\89_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\90_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\91_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\92_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\93_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\94_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\95_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\96_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\97_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\98_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\99_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\100_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\101_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\102_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\103_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\104_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\105_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\106_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\107_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\108_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\109_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\110_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\111_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\112_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\113_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\114_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\115_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\116_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\117_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\118_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\119_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\120_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\121_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\122_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\123_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\124_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\125_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\126_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\127_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\128_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\129_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\130_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\131_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\132_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\133_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\134_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\135_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\136_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\137_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\138_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\139_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\140_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\141_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\142_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\143_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\144_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\145_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\146_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\147_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\148_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\149_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\150_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\151_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\152_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\153_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\154_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\155_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\156_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\157_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\158_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\159_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\160_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\161_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\162_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\163_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\164_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\165_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\166_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\167_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\168_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\169_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\170_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\171_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\172_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\173_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\174_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\175_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\176_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\177_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\178_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\179_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\180_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\181_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\182_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\183_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\184_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\185_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\186_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\187_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\188_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\189_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\190_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\191_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\192_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\193_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\194_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\195_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\196_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\197_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\198_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\199_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\200_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\201_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\202_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\203_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\204_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\205_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\206_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\207_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\208_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\209_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\210_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\211_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\212_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\213_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\214_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\215_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\216_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\217_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\218_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\219_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\220_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\221_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\222_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\223_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\224_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\225_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\226_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\227_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\228_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\229_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\230_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\231_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\232_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\233_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\234_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\235_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\236_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\237_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\238_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\239_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\240_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\241_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\242_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\243_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\244_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\245_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\246_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\247_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\248_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\249_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\250_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\251_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\252_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\253_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\254_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\255_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\256_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\257_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\258_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\259_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\260_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\261_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\262_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\263_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\264_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\265_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\266_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\267_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\268_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\269_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\270_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\271_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\272_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\273_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\274_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\275_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\276_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\277_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\278_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\279_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\280_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\281_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\282_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\283_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\284_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\285_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\286_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\287_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\288_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\289_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\290_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\291_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\292_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\293_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\294_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\295_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\296_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\297_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\298_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\299_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\300_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\301_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\302_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\303_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\304_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\305_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\306_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\307_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\308_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\309_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\310_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\311_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\312_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\313_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\314_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\315_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\316_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\317_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\318_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\319_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\320_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\321_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\322_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\323_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\324_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\325_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\326_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\327_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\328_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\329_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\330_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\331_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\332_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\333_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\334_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\335_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\336_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\337_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\338_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\339_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\340_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\341_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\342_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\343_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\344_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\345_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\346_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\347_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\348_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\349_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\350_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\351_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\352_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\353_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\354_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\355_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\356_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\357_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\358_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\359_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\360_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\361_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\362_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\363_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\364_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\365_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\366_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\367_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\368_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\369_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\370_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\371_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\372_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\373_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\374_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\375_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\376_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\377_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\378_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\379_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\380_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\381_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\382_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\383_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\384_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\385_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\386_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\387_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\388_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\389_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\390_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\391_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\392_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\393_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\394_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\395_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\396_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\397_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\398_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\399_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\400_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\401_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\402_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\403_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\404_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\405_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\406_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\407_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\408_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\409_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\410_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\411_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\412_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\413_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\414_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\415_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\416_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\417_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\418_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\419_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\420_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\421_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\422_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\423_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\424_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\425_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\426_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\427_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\428_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\429_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\430_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\431_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\432_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\433_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\434_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\435_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\436_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\437_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\438_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\439_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\440_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\441_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\442_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\443_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\444_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\445_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\446_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\447_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\448_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\449_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\450_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\451_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\452_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\453_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\454_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\455_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\456_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\457_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\458_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\459_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\460_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\461_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\462_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\463_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\464_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\465_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\466_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\467_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\468_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\469_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\470_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\471_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\472_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\473_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\474_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\475_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\476_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\477_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\478_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\479_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\480_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\481_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\482_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\483_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\484_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\485_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\486_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\487_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\488_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\489_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\490_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\491_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\492_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\493_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\494_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\495_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\496_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\497_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\498_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\499_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\500_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\501_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\502_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\503_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\504_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\505_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\506_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\507_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\508_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\509_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\510_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\511_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\512_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\513_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\514_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\515_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\516_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\517_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\518_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\519_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\520_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\521_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\522_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\523_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\524_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\525_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\526_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\527_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\528_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\529_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\530_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\531_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\532_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\533_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\534_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\535_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\536_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\537_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\538_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\539_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\540_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\541_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\542_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\543_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\544_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\545_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\546_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\547_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\548_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\549_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\550_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\551_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\552_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\553_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\554_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\555_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\556_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\557_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\558_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\559_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\560_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\561_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\562_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\563_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\564_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\565_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\566_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\567_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\568_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\569_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\570_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\571_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\572_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\573_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\574_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\575_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\576_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\577_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\578_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\579_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\580_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\581_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\582_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\583_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\584_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\585_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\586_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\587_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\588_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\589_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\590_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\591_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\592_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\593_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\594_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\595_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\596_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\597_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\598_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\599_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\600_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\601_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\602_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\603_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\604_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\605_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\606_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\607_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\608_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\609_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\610_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\611_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\612_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\613_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\614_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\615_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\616_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\617_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\618_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\619_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\620_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\621_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\622_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\623_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\624_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\625_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\626_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\627_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\628_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\629_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\630_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\631_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\632_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\633_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\634_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\635_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\636_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\637_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\638_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\639_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\640_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\641_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\642_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\643_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\644_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\645_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\646_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\647_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\648_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\649_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\650_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\651_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\652_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\653_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\654_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\655_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\656_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\657_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\658_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\659_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\660_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\661_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\662_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\663_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\664_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\665_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\666_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\667_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\668_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\669_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\670_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\671_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\672_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\673_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\674_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\675_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\676_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\677_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\678_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\679_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\680_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\681_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\682_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\683_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\684_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\685_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\686_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\687_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\688_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\689_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\690_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\691_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\692_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\693_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\694_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\695_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\696_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\697_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\698_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\699_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\700_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\701_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\702_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\703_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\704_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\705_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\706_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\707_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\708_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\709_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\710_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\711_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\712_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\713_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\714_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\715_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\716_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\717_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\718_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\719_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\720_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\721_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\722_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\723_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\724_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\725_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\726_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\727_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\728_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\729_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\730_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\731_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\732_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\733_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\734_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\735_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\736_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\737_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\738_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\739_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\740_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\741_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\742_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\743_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\744_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\745_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\746_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\747_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\748_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\749_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\750_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\751_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\752_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\753_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\754_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\755_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\756_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\757_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\758_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\759_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\760_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\761_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\762_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\763_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\764_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\765_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\766_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\767_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\768_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\769_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\770_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\771_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\772_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\773_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\774_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\775_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\776_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\777_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\778_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\779_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\780_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\781_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\782_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\783_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\784_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\785_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\786_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\787_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\788_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\789_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\790_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\791_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\792_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\793_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\794_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\795_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\796_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\797_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\798_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\799_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\800_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\801_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\802_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\803_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\804_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\805_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\806_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\807_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\808_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\809_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\810_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\811_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\812_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\813_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\814_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\815_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\816_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\817_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\818_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\819_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\820_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\821_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\822_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\823_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\824_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\825_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\826_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\827_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\828_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\829_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\830_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\831_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\832_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\833_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\834_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\835_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\836_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\837_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\838_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\839_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\840_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\841_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\842_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\843_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\844_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\845_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\846_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\847_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\848_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\849_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\850_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\851_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\852_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\853_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\854_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\855_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\856_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\857_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\858_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\859_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\860_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\861_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\862_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\863_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\864_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\865_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\866_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\867_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\868_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\869_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\870_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\871_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\872_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\873_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\874_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\875_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\876_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\877_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\878_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\879_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\880_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\881_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\882_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\883_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\884_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\885_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\886_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\887_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\888_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\889_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\890_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\891_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\892_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\893_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\894_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\895_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\896_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\897_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\898_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\899_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\900_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\901_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\902_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\903_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\904_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\905_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\906_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\907_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\908_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\909_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\910_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\911_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\912_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\913_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\914_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\915_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\916_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\917_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\918_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\919_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\920_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\921_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\922_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\923_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\924_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\925_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\926_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\927_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\928_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\929_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\930_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\931_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\932_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\933_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\934_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\935_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\936_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\937_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\938_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\939_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\940_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\941_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\942_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\943_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\944_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\945_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\946_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\947_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\948_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\949_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\950_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\951_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\952_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\953_reg_1.fasta\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\954_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\955_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\956_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\957_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\958_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\959_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\960_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\961_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\962_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\963_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\964_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\965_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\966_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\967_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\968_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\969_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\970_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\971_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\972_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\973_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\974_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\975_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\976_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\977_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\978_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\979_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\980_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\981_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\982_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\983_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\984_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\985_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\986_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\987_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\988_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\989_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\990_reg_1.fasta\n",
      "-- save to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\991_reg_1.fasta\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.sequences)\n",
    "region_file = os.path.join(library_folder, 'Seleted_merged_super_enhancer_list.bed')\n",
    "region_dict = library_tools.sequences.read_region_file(region_file)\n",
    "# add strand information\n",
    "for _i in range(len(region_dict)):\n",
    "    region_dict[_i]['Strand'] = '+'\n",
    "records = library_tools.sequences.extract_sequence(region_dict, genome_folder, \n",
    "                                                   resolution=resolution,\n",
    "                                                   flanking=flanking,\n",
    "                                                   save_folder=sequence_folder)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "metadata": {},
   "outputs": [],
   "source": [
    "# calculate num of regions in each cluster\n",
    "chr_2_region_num = {}\n",
    "for _reg_info in region_dict:\n",
    "    _chr_name = _reg_info['Chr']\n",
    "    if 'chr' in _chr_name:\n",
    "        _chr = _chr_name.split('chr')[1]\n",
    "    else:\n",
    "        _chr = _chr_name\n",
    "    if _chr not in chr_2_region_num:\n",
    "        chr_2_region_num[_chr] = 0\n",
    "    chr_2_region_num[_chr] += 1\n",
    "# save\n",
    "pickle.dump(chr_2_region_num, open(os.path.join(library_folder,'chr_2_region_num.pkl'), 'wb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'1': 76,\n",
       " '2': 75,\n",
       " '3': 60,\n",
       " '4': 63,\n",
       " '5': 60,\n",
       " '6': 57,\n",
       " '7': 45,\n",
       " '8': 53,\n",
       " '9': 56,\n",
       " '10': 65,\n",
       " '11': 57,\n",
       " '12': 36,\n",
       " '13': 49,\n",
       " '14': 57,\n",
       " '15': 35,\n",
       " '16': 38,\n",
       " '17': 34,\n",
       " '18': 33,\n",
       " '19': 35,\n",
       " 'X': 7}"
      ]
     },
     "execution_count": 127,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "chr_2_region_num"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 128,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'Chr': 'chr1',\n",
       "  'Start': '9610471',\n",
       "  'End': '9646458',\n",
       "  'Name': '1',\n",
       "  'Score': '',\n",
       "  'Gene': '1',\n",
       "  'Region': '1:9610471-9646458',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '9759317',\n",
       "  'End': '9835402',\n",
       "  'Name': '2',\n",
       "  'Score': '',\n",
       "  'Gene': '2',\n",
       "  'Region': '1:9759317-9835402',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '20898605',\n",
       "  'End': '20943251',\n",
       "  'Name': '3',\n",
       "  'Score': '',\n",
       "  'Gene': '3',\n",
       "  'Region': '1:20898605-20943251',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '21485446',\n",
       "  'End': '21554433',\n",
       "  'Name': '4',\n",
       "  'Score': '',\n",
       "  'Gene': '4',\n",
       "  'Region': '1:21485446-21554433',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '21684557',\n",
       "  'End': '21783163',\n",
       "  'Name': '5',\n",
       "  'Score': '',\n",
       "  'Gene': '5',\n",
       "  'Region': '1:21684557-21783163',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '23255924',\n",
       "  'End': '23287215',\n",
       "  'Name': '6',\n",
       "  'Score': '',\n",
       "  'Gene': '6',\n",
       "  'Region': '1:23255924-23287215',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '23928055',\n",
       "  'End': '23989346',\n",
       "  'Name': '7',\n",
       "  'Score': '',\n",
       "  'Gene': '7',\n",
       "  'Region': '1:23928055-23989346',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '24218993',\n",
       "  'End': '24265055',\n",
       "  'Name': '8',\n",
       "  'Score': '',\n",
       "  'Gene': '8',\n",
       "  'Region': '1:24218993-24265055',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '35495994',\n",
       "  'End': '35586498',\n",
       "  'Name': '9',\n",
       "  'Score': '',\n",
       "  'Gene': '9',\n",
       "  'Region': '1:35495994-35586498',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '36052487',\n",
       "  'End': '36100247',\n",
       "  'Name': '10',\n",
       "  'Score': '',\n",
       "  'Gene': '10',\n",
       "  'Region': '1:36052487-36100247',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '37053543',\n",
       "  'End': '37122697',\n",
       "  'Name': '11',\n",
       "  'Score': '',\n",
       "  'Gene': '11',\n",
       "  'Region': '1:37053543-37122697',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '39063266',\n",
       "  'End': '39106581',\n",
       "  'Name': '12',\n",
       "  'Score': '',\n",
       "  'Gene': '12',\n",
       "  'Region': '1:39063266-39106581',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '39730514',\n",
       "  'End': '39774704',\n",
       "  'Name': '13',\n",
       "  'Score': '',\n",
       "  'Gene': '13',\n",
       "  'Region': '1:39730514-39774704',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '42229499',\n",
       "  'End': '42263271',\n",
       "  'Name': '14',\n",
       "  'Score': '',\n",
       "  'Gene': '14',\n",
       "  'Region': '1:42229499-42263271',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '45857775',\n",
       "  'End': '45917301',\n",
       "  'Name': '15',\n",
       "  'Score': '',\n",
       "  'Gene': '15',\n",
       "  'Region': '1:45857775-45917301',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '51091069',\n",
       "  'End': '51165247',\n",
       "  'Name': '16',\n",
       "  'Score': '',\n",
       "  'Gene': '16',\n",
       "  'Region': '1:51091069-51165247',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '51580703',\n",
       "  'End': '51650839',\n",
       "  'Name': '17',\n",
       "  'Score': '',\n",
       "  'Gene': '17',\n",
       "  'Region': '1:51580703-51650839',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '52640368',\n",
       "  'End': '52690586',\n",
       "  'Name': '18',\n",
       "  'Score': '',\n",
       "  'Gene': '18',\n",
       "  'Region': '1:52640368-52690586',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '54789256',\n",
       "  'End': '54914777',\n",
       "  'Name': '19',\n",
       "  'Score': '',\n",
       "  'Gene': '19',\n",
       "  'Region': '1:54789256-54914777',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '55193046',\n",
       "  'End': '55221757',\n",
       "  'Name': '20',\n",
       "  'Score': '',\n",
       "  'Gene': '20',\n",
       "  'Region': '1:55193046-55221757',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '55429953',\n",
       "  'End': '55474150',\n",
       "  'Name': '21',\n",
       "  'Score': '',\n",
       "  'Gene': '21',\n",
       "  'Region': '1:55429953-55474150',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '55609660',\n",
       "  'End': '55761983',\n",
       "  'Name': '22',\n",
       "  'Score': '',\n",
       "  'Gene': '22',\n",
       "  'Region': '1:55609660-55761983',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '57205401',\n",
       "  'End': '57225355',\n",
       "  'Name': '23',\n",
       "  'Score': '',\n",
       "  'Gene': '23',\n",
       "  'Region': '1:57205401-57225355',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '57784408',\n",
       "  'End': '57840178',\n",
       "  'Name': '24',\n",
       "  'Score': '',\n",
       "  'Gene': '24',\n",
       "  'Region': '1:57784408-57840178',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '58720248',\n",
       "  'End': '58745177',\n",
       "  'Name': '25',\n",
       "  'Score': '',\n",
       "  'Gene': '25',\n",
       "  'Region': '1:58720248-58745177',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '60416810',\n",
       "  'End': '60500189',\n",
       "  'Name': '26',\n",
       "  'Score': '',\n",
       "  'Gene': '26',\n",
       "  'Region': '1:60416810-60500189',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '60838959',\n",
       "  'End': '60868677',\n",
       "  'Name': '27',\n",
       "  'Score': '',\n",
       "  'Gene': '27',\n",
       "  'Region': '1:60838959-60868677',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '62900963',\n",
       "  'End': '62948089',\n",
       "  'Name': '28',\n",
       "  'Score': '',\n",
       "  'Gene': '28',\n",
       "  'Region': '1:62900963-62948089',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '68038967',\n",
       "  'End': '68161164',\n",
       "  'Name': '29',\n",
       "  'Score': '',\n",
       "  'Gene': '29',\n",
       "  'Region': '1:68038967-68161164',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '68692949',\n",
       "  'End': '68790249',\n",
       "  'Name': '30',\n",
       "  'Score': '',\n",
       "  'Gene': '30',\n",
       "  'Region': '1:68692949-68790249',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '72414007',\n",
       "  'End': '72465858',\n",
       "  'Name': '31',\n",
       "  'Score': '',\n",
       "  'Gene': '31',\n",
       "  'Region': '1:72414007-72465858',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '72593027',\n",
       "  'End': '72620300',\n",
       "  'Name': '32',\n",
       "  'Score': '',\n",
       "  'Gene': '32',\n",
       "  'Region': '1:72593027-72620300',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '73532522',\n",
       "  'End': '73593287',\n",
       "  'Name': '33',\n",
       "  'Score': '',\n",
       "  'Gene': '33',\n",
       "  'Region': '1:73532522-73593287',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '75433170',\n",
       "  'End': '75468047',\n",
       "  'Name': '34',\n",
       "  'Score': '',\n",
       "  'Gene': '34',\n",
       "  'Region': '1:75433170-75468047',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '75864682',\n",
       "  'End': '75910607',\n",
       "  'Name': '35',\n",
       "  'Score': '',\n",
       "  'Gene': '35',\n",
       "  'Region': '1:75864682-75910607',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '79823369',\n",
       "  'End': '79866804',\n",
       "  'Name': '36',\n",
       "  'Score': '',\n",
       "  'Gene': '36',\n",
       "  'Region': '1:79823369-79866804',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '81232704',\n",
       "  'End': '81355836',\n",
       "  'Name': '37',\n",
       "  'Score': '',\n",
       "  'Gene': '37',\n",
       "  'Region': '1:81232704-81355836',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '82432928',\n",
       "  'End': '82483138',\n",
       "  'Name': '38',\n",
       "  'Score': '',\n",
       "  'Gene': '38',\n",
       "  'Region': '1:82432928-82483138',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '83323920',\n",
       "  'End': '83390588',\n",
       "  'Name': '39',\n",
       "  'Score': '',\n",
       "  'Gene': '39',\n",
       "  'Region': '1:83323920-83390588',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '84178846',\n",
       "  'End': '84251608',\n",
       "  'Name': '40',\n",
       "  'Score': '',\n",
       "  'Gene': '40',\n",
       "  'Region': '1:84178846-84251608',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '86969177',\n",
       "  'End': '87052202',\n",
       "  'Name': '41',\n",
       "  'Score': '',\n",
       "  'Gene': '41',\n",
       "  'Region': '1:86969177-87052202',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '89131460',\n",
       "  'End': '89196528',\n",
       "  'Name': '42',\n",
       "  'Score': '',\n",
       "  'Gene': '42',\n",
       "  'Region': '1:89131460-89196528',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '91422081',\n",
       "  'End': '91456772',\n",
       "  'Name': '43',\n",
       "  'Score': '',\n",
       "  'Gene': '43',\n",
       "  'Region': '1:91422081-91456772',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '93199428',\n",
       "  'End': '93228116',\n",
       "  'Name': '44',\n",
       "  'Score': '',\n",
       "  'Gene': '44',\n",
       "  'Region': '1:93199428-93228116',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '97614468',\n",
       "  'End': '97652595',\n",
       "  'Name': '45',\n",
       "  'Score': '',\n",
       "  'Gene': '45',\n",
       "  'Region': '1:97614468-97652595',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '105971567',\n",
       "  'End': '105999318',\n",
       "  'Name': '46',\n",
       "  'Score': '',\n",
       "  'Gene': '46',\n",
       "  'Region': '1:105971567-105999318',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '115737054',\n",
       "  'End': '115794837',\n",
       "  'Name': '47',\n",
       "  'Score': '',\n",
       "  'Gene': '47',\n",
       "  'Region': '1:115737054-115794837',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '119447659',\n",
       "  'End': '119495173',\n",
       "  'Name': '48',\n",
       "  'Score': '',\n",
       "  'Gene': '48',\n",
       "  'Region': '1:119447659-119495173',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '120171583',\n",
       "  'End': '120208334',\n",
       "  'Name': '49',\n",
       "  'Score': '',\n",
       "  'Gene': '49',\n",
       "  'Region': '1:120171583-120208334',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '121440601',\n",
       "  'End': '121479275',\n",
       "  'Name': '50',\n",
       "  'Score': '',\n",
       "  'Gene': '50',\n",
       "  'Region': '1:121440601-121479275',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '124684853',\n",
       "  'End': '124791140',\n",
       "  'Name': '51',\n",
       "  'Score': '',\n",
       "  'Gene': '51',\n",
       "  'Region': '1:124684853-124791140',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '137898029',\n",
       "  'End': '137955735',\n",
       "  'Name': '52',\n",
       "  'Score': '',\n",
       "  'Gene': '52',\n",
       "  'Region': '1:137898029-137955735',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '140364959',\n",
       "  'End': '140531768',\n",
       "  'Name': '53',\n",
       "  'Score': '',\n",
       "  'Gene': '53',\n",
       "  'Region': '1:140364959-140531768',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '143871617',\n",
       "  'End': '143924455',\n",
       "  'Name': '54',\n",
       "  'Score': '',\n",
       "  'Gene': '54',\n",
       "  'Region': '1:143871617-143924455',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '149856612',\n",
       "  'End': '149898891',\n",
       "  'Name': '55',\n",
       "  'Score': '',\n",
       "  'Gene': '55',\n",
       "  'Region': '1:149856612-149898891',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '152407309',\n",
       "  'End': '152476632',\n",
       "  'Name': '56',\n",
       "  'Score': '',\n",
       "  'Gene': '56',\n",
       "  'Region': '1:152407309-152476632',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '155675230',\n",
       "  'End': '155738254',\n",
       "  'Name': '57',\n",
       "  'Score': '',\n",
       "  'Gene': '57',\n",
       "  'Region': '1:155675230-155738254',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '161224625',\n",
       "  'End': '161287711',\n",
       "  'Name': '58',\n",
       "  'Score': '',\n",
       "  'Gene': '58',\n",
       "  'Region': '1:161224625-161287711',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '162022959',\n",
       "  'End': '162094685',\n",
       "  'Name': '59',\n",
       "  'Score': '',\n",
       "  'Gene': '59',\n",
       "  'Region': '1:162022959-162094685',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '162665058',\n",
       "  'End': '162766885',\n",
       "  'Name': '60',\n",
       "  'Score': '',\n",
       "  'Gene': '60',\n",
       "  'Region': '1:162665058-162766885',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '167044296',\n",
       "  'End': '167090658',\n",
       "  'Name': '61',\n",
       "  'Score': '',\n",
       "  'Gene': '61',\n",
       "  'Region': '1:167044296-167090658',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '167207268',\n",
       "  'End': '167274083',\n",
       "  'Name': '62',\n",
       "  'Score': '',\n",
       "  'Gene': '62',\n",
       "  'Region': '1:167207268-167274083',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '169802098',\n",
       "  'End': '169918411',\n",
       "  'Name': '63',\n",
       "  'Score': '',\n",
       "  'Gene': '63',\n",
       "  'Region': '1:169802098-169918411',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '171439944',\n",
       "  'End': '171489765',\n",
       "  'Name': '64',\n",
       "  'Score': '',\n",
       "  'Gene': '64',\n",
       "  'Region': '1:171439944-171489765',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '171815726',\n",
       "  'End': '171867501',\n",
       "  'Name': '65',\n",
       "  'Score': '',\n",
       "  'Gene': '65',\n",
       "  'Region': '1:171815726-171867501',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '173317246',\n",
       "  'End': '173364219',\n",
       "  'Name': '66',\n",
       "  'Score': '',\n",
       "  'Gene': '66',\n",
       "  'Region': '1:173317246-173364219',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '178351436',\n",
       "  'End': '178400984',\n",
       "  'Name': '67',\n",
       "  'Score': '',\n",
       "  'Gene': '67',\n",
       "  'Region': '1:178351436-178400984',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '179041899',\n",
       "  'End': '179115551',\n",
       "  'Name': '68',\n",
       "  'Score': '',\n",
       "  'Gene': '68',\n",
       "  'Region': '1:179041899-179115551',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '182439520',\n",
       "  'End': '182509678',\n",
       "  'Name': '69',\n",
       "  'Score': '',\n",
       "  'Gene': '69',\n",
       "  'Region': '1:182439520-182509678',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '186627359',\n",
       "  'End': '186696395',\n",
       "  'Name': '70',\n",
       "  'Score': '',\n",
       "  'Gene': '70',\n",
       "  'Region': '1:186627359-186696395',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '187645279',\n",
       "  'End': '187694850',\n",
       "  'Name': '71',\n",
       "  'Score': '',\n",
       "  'Gene': '71',\n",
       "  'Region': '1:187645279-187694850',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '189017529',\n",
       "  'End': '189066052',\n",
       "  'Name': '72',\n",
       "  'Score': '',\n",
       "  'Gene': '72',\n",
       "  'Region': '1:189017529-189066052',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '189185060',\n",
       "  'End': '189229230',\n",
       "  'Name': '73',\n",
       "  'Score': '',\n",
       "  'Gene': '73',\n",
       "  'Region': '1:189185060-189229230',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '189490922',\n",
       "  'End': '189533419',\n",
       "  'Name': '74',\n",
       "  'Score': '',\n",
       "  'Gene': '74',\n",
       "  'Region': '1:189490922-189533419',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '189876129',\n",
       "  'End': '189928237',\n",
       "  'Name': '75',\n",
       "  'Score': '',\n",
       "  'Gene': '75',\n",
       "  'Region': '1:189876129-189928237',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr1',\n",
       "  'Start': '193085619',\n",
       "  'End': '193123645',\n",
       "  'Name': '76',\n",
       "  'Score': '',\n",
       "  'Gene': '76',\n",
       "  'Region': '1:193085619-193123645',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '3139373',\n",
       "  'End': '3173216',\n",
       "  'Name': '77',\n",
       "  'Score': '',\n",
       "  'Gene': '77',\n",
       "  'Region': '2:3139373-3173216',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '4036085',\n",
       "  'End': '4073325',\n",
       "  'Name': '78',\n",
       "  'Score': '',\n",
       "  'Gene': '78',\n",
       "  'Region': '2:4036085-4073325',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '5161601',\n",
       "  'End': '5200127',\n",
       "  'Name': '79',\n",
       "  'Score': '',\n",
       "  'Gene': '79',\n",
       "  'Region': '2:5161601-5200127',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '11319886',\n",
       "  'End': '11375319',\n",
       "  'Name': '80',\n",
       "  'Score': '',\n",
       "  'Gene': '80',\n",
       "  'Region': '2:11319886-11375319',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '14907328',\n",
       "  'End': '14958889',\n",
       "  'Name': '81',\n",
       "  'Score': '',\n",
       "  'Gene': '81',\n",
       "  'Region': '2:14907328-14958889',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '16523406',\n",
       "  'End': '16650259',\n",
       "  'Name': '82',\n",
       "  'Score': '',\n",
       "  'Gene': '82',\n",
       "  'Region': '2:16523406-16650259',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '17638682',\n",
       "  'End': '17724416',\n",
       "  'Name': '83',\n",
       "  'Score': '',\n",
       "  'Gene': '83',\n",
       "  'Region': '2:17638682-17724416',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '19898794',\n",
       "  'End': '19989195',\n",
       "  'Name': '84',\n",
       "  'Score': '',\n",
       "  'Gene': '84',\n",
       "  'Region': '2:19898794-19989195',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '21261409',\n",
       "  'End': '21288244',\n",
       "  'Name': '85',\n",
       "  'Score': '',\n",
       "  'Gene': '85',\n",
       "  'Region': '2:21261409-21288244',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '21632583',\n",
       "  'End': '21693613',\n",
       "  'Name': '86',\n",
       "  'Score': '',\n",
       "  'Gene': '86',\n",
       "  'Region': '2:21632583-21693613',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '25430535',\n",
       "  'End': '25451500',\n",
       "  'Name': '87',\n",
       "  'Score': '',\n",
       "  'Gene': '87',\n",
       "  'Region': '2:25430535-25451500',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '27081925',\n",
       "  'End': '27121679',\n",
       "  'Name': '88',\n",
       "  'Score': '',\n",
       "  'Gene': '88',\n",
       "  'Region': '2:27081925-27121679',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '30246602',\n",
       "  'End': '30270512',\n",
       "  'Name': '89',\n",
       "  'Score': '',\n",
       "  'Gene': '89',\n",
       "  'Region': '2:30246602-30270512',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '32103351',\n",
       "  'End': '32138238',\n",
       "  'Name': '90',\n",
       "  'Score': '',\n",
       "  'Gene': '90',\n",
       "  'Region': '2:32103351-32138238',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '37963869',\n",
       "  'End': '38024097',\n",
       "  'Name': '91',\n",
       "  'Score': '',\n",
       "  'Gene': '91',\n",
       "  'Region': '2:37963869-38024097',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '41938242',\n",
       "  'End': '42013962',\n",
       "  'Name': '92',\n",
       "  'Score': '',\n",
       "  'Gene': '92',\n",
       "  'Region': '2:41938242-42013962',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '43763833',\n",
       "  'End': '43789050',\n",
       "  'Name': '93',\n",
       "  'Score': '',\n",
       "  'Gene': '93',\n",
       "  'Region': '2:43763833-43789050',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '44094305',\n",
       "  'End': '44143466',\n",
       "  'Name': '94',\n",
       "  'Score': '',\n",
       "  'Gene': '94',\n",
       "  'Region': '2:44094305-44143466',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '44334153',\n",
       "  'End': '44422010',\n",
       "  'Name': '95',\n",
       "  'Score': '',\n",
       "  'Gene': '95',\n",
       "  'Region': '2:44334153-44422010',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '49908807',\n",
       "  'End': '49960037',\n",
       "  'Name': '96',\n",
       "  'Score': '',\n",
       "  'Gene': '96',\n",
       "  'Region': '2:49908807-49960037',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '58015861',\n",
       "  'End': '58055893',\n",
       "  'Name': '97',\n",
       "  'Score': '',\n",
       "  'Gene': '97',\n",
       "  'Region': '2:58015861-58055893',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '59644429',\n",
       "  'End': '59675853',\n",
       "  'Name': '98',\n",
       "  'Score': '',\n",
       "  'Gene': '98',\n",
       "  'Region': '2:59644429-59675853',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '62477453',\n",
       "  'End': '62533544',\n",
       "  'Name': '99',\n",
       "  'Score': '',\n",
       "  'Gene': '99',\n",
       "  'Region': '2:62477453-62533544',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '62680661',\n",
       "  'End': '62728745',\n",
       "  'Name': '100',\n",
       "  'Score': '',\n",
       "  'Gene': '100',\n",
       "  'Region': '2:62680661-62728745',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '70551303',\n",
       "  'End': '70593453',\n",
       "  'Name': '101',\n",
       "  'Score': '',\n",
       "  'Gene': '101',\n",
       "  'Region': '2:70551303-70593453',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '73263634',\n",
       "  'End': '73301570',\n",
       "  'Name': '102',\n",
       "  'Score': '',\n",
       "  'Gene': '102',\n",
       "  'Region': '2:73263634-73301570',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '74112486',\n",
       "  'End': '74161621',\n",
       "  'Name': '103',\n",
       "  'Score': '',\n",
       "  'Gene': '103',\n",
       "  'Region': '2:74112486-74161621',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '75757159',\n",
       "  'End': '75810879',\n",
       "  'Name': '104',\n",
       "  'Score': '',\n",
       "  'Gene': '104',\n",
       "  'Region': '2:75757159-75810879',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '77023944',\n",
       "  'End': '77081549',\n",
       "  'Name': '105',\n",
       "  'Score': '',\n",
       "  'Gene': '105',\n",
       "  'Region': '2:77023944-77081549',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '77287948',\n",
       "  'End': '77357263',\n",
       "  'Name': '106',\n",
       "  'Score': '',\n",
       "  'Gene': '106',\n",
       "  'Region': '2:77287948-77357263',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '83714396',\n",
       "  'End': '83771664',\n",
       "  'Name': '107',\n",
       "  'Score': '',\n",
       "  'Gene': '107',\n",
       "  'Region': '2:83714396-83771664',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '84525475',\n",
       "  'End': '84557580',\n",
       "  'Name': '108',\n",
       "  'Score': '',\n",
       "  'Gene': '108',\n",
       "  'Region': '2:84525475-84557580',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '84855394',\n",
       "  'End': '84890174',\n",
       "  'Name': '109',\n",
       "  'Score': '',\n",
       "  'Gene': '109',\n",
       "  'Region': '2:84855394-84890174',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '85047645',\n",
       "  'End': '85069130',\n",
       "  'Name': '110',\n",
       "  'Score': '',\n",
       "  'Gene': '110',\n",
       "  'Region': '2:85047645-85069130',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '85047645',\n",
       "  'End': '85069130',\n",
       "  'Name': '111',\n",
       "  'Score': '',\n",
       "  'Gene': '111',\n",
       "  'Region': '2:85047645-85069130',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '85208480',\n",
       "  'End': '85285653',\n",
       "  'Name': '112',\n",
       "  'Score': '',\n",
       "  'Gene': '112',\n",
       "  'Region': '2:85208480-85285653',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '91312426',\n",
       "  'End': '91330781',\n",
       "  'Name': '113',\n",
       "  'Score': '',\n",
       "  'Gene': '113',\n",
       "  'Region': '2:91312426-91330781',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '97574926',\n",
       "  'End': '97640442',\n",
       "  'Name': '114',\n",
       "  'Score': '',\n",
       "  'Gene': '114',\n",
       "  'Region': '2:97574926-97640442',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '110172327',\n",
       "  'End': '110211668',\n",
       "  'Name': '115',\n",
       "  'Score': '',\n",
       "  'Gene': '115',\n",
       "  'Region': '2:110172327-110211668',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '118262753',\n",
       "  'End': '118336305',\n",
       "  'Name': '116',\n",
       "  'Score': '',\n",
       "  'Gene': '116',\n",
       "  'Region': '2:118262753-118336305',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '120628697',\n",
       "  'End': '120664306',\n",
       "  'Name': '117',\n",
       "  'Score': '',\n",
       "  'Gene': '117',\n",
       "  'Region': '2:120628697-120664306',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '122337422',\n",
       "  'End': '122358386',\n",
       "  'Name': '118',\n",
       "  'Score': '',\n",
       "  'Gene': '118',\n",
       "  'Region': '2:122337422-122358386',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '122558219',\n",
       "  'End': '122626386',\n",
       "  'Name': '119',\n",
       "  'Score': '',\n",
       "  'Gene': '119',\n",
       "  'Region': '2:122558219-122626386',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '123310821',\n",
       "  'End': '123364509',\n",
       "  'Name': '120',\n",
       "  'Score': '',\n",
       "  'Gene': '120',\n",
       "  'Region': '2:123310821-123364509',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '124118515',\n",
       "  'End': '124215589',\n",
       "  'Name': '121',\n",
       "  'Score': '',\n",
       "  'Gene': '121',\n",
       "  'Region': '2:124118515-124215589',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '127079492',\n",
       "  'End': '127137676',\n",
       "  'Name': '122',\n",
       "  'Score': '',\n",
       "  'Gene': '122',\n",
       "  'Region': '2:127079492-127137676',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '128299201',\n",
       "  'End': '128380141',\n",
       "  'Name': '123',\n",
       "  'Score': '',\n",
       "  'Gene': '123',\n",
       "  'Region': '2:128299201-128380141',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '128704272',\n",
       "  'End': '128807233',\n",
       "  'Name': '124',\n",
       "  'Score': '',\n",
       "  'Gene': '124',\n",
       "  'Region': '2:128704272-128807233',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '129864913',\n",
       "  'End': '129926172',\n",
       "  'Name': '125',\n",
       "  'Score': '',\n",
       "  'Gene': '125',\n",
       "  'Region': '2:129864913-129926172',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '130454939',\n",
       "  'End': '130497012',\n",
       "  'Name': '126',\n",
       "  'Score': '',\n",
       "  'Gene': '126',\n",
       "  'Region': '2:130454939-130497012',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '131373155',\n",
       "  'End': '131431109',\n",
       "  'Name': '127',\n",
       "  'Score': '',\n",
       "  'Gene': '127',\n",
       "  'Region': '2:131373155-131431109',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '132270572',\n",
       "  'End': '132316514',\n",
       "  'Name': '128',\n",
       "  'Score': '',\n",
       "  'Gene': '128',\n",
       "  'Region': '2:132270572-132316514',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '132270572',\n",
       "  'End': '132316514',\n",
       "  'Name': '129',\n",
       "  'Score': '',\n",
       "  'Gene': '129',\n",
       "  'Region': '2:132270572-132316514',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '135278536',\n",
       "  'End': '135330636',\n",
       "  'Name': '130',\n",
       "  'Score': '',\n",
       "  'Gene': '130',\n",
       "  'Region': '2:135278536-135330636',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '135658540',\n",
       "  'End': '135684141',\n",
       "  'Name': '131',\n",
       "  'Score': '',\n",
       "  'Gene': '131',\n",
       "  'Region': '2:135658540-135684141',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '136189077',\n",
       "  'End': '136252638',\n",
       "  'Name': '132',\n",
       "  'Score': '',\n",
       "  'Gene': '132',\n",
       "  'Region': '2:136189077-136252638',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '139266790',\n",
       "  'End': '139336099',\n",
       "  'Name': '133',\n",
       "  'Score': '',\n",
       "  'Gene': '133',\n",
       "  'Region': '2:139266790-139336099',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '143182015',\n",
       "  'End': '143244256',\n",
       "  'Name': '134',\n",
       "  'Score': '',\n",
       "  'Gene': '134',\n",
       "  'Region': '2:143182015-143244256',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '143975229',\n",
       "  'End': '144005150',\n",
       "  'Name': '135',\n",
       "  'Score': '',\n",
       "  'Gene': '135',\n",
       "  'Region': '2:143975229-144005150',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '147178730',\n",
       "  'End': '147205713',\n",
       "  'Name': '136',\n",
       "  'Score': '',\n",
       "  'Gene': '136',\n",
       "  'Region': '2:147178730-147205713',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '148138049',\n",
       "  'End': '148211409',\n",
       "  'Name': '137',\n",
       "  'Score': '',\n",
       "  'Gene': '137',\n",
       "  'Region': '2:148138049-148211409',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '148677886',\n",
       "  'End': '148721885',\n",
       "  'Name': '138',\n",
       "  'Score': '',\n",
       "  'Gene': '138',\n",
       "  'Region': '2:148677886-148721885',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '152678254',\n",
       "  'End': '152708287',\n",
       "  'Name': '139',\n",
       "  'Score': '',\n",
       "  'Gene': '139',\n",
       "  'Region': '2:152678254-152708287',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '153913330',\n",
       "  'End': '153963662',\n",
       "  'Name': '140',\n",
       "  'Score': '',\n",
       "  'Gene': '140',\n",
       "  'Region': '2:153913330-153963662',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '160207345',\n",
       "  'End': '160250687',\n",
       "  'Name': '141',\n",
       "  'Score': '',\n",
       "  'Gene': '141',\n",
       "  'Region': '2:160207345-160250687',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '161128134',\n",
       "  'End': '161185413',\n",
       "  'Name': '142',\n",
       "  'Score': '',\n",
       "  'Gene': '142',\n",
       "  'Region': '2:161128134-161185413',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '161767252',\n",
       "  'End': '161809816',\n",
       "  'Name': '143',\n",
       "  'Score': '',\n",
       "  'Gene': '143',\n",
       "  'Region': '2:161767252-161809816',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '162756567',\n",
       "  'End': '162825217',\n",
       "  'Name': '144',\n",
       "  'Score': '',\n",
       "  'Gene': '144',\n",
       "  'Region': '2:162756567-162825217',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '172127465',\n",
       "  'End': '172160675',\n",
       "  'Name': '145',\n",
       "  'Score': '',\n",
       "  'Gene': '145',\n",
       "  'Region': '2:172127465-172160675',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '172400054',\n",
       "  'End': '172447136',\n",
       "  'Name': '146',\n",
       "  'Score': '',\n",
       "  'Gene': '146',\n",
       "  'Region': '2:172400054-172447136',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '172807494',\n",
       "  'End': '172876579',\n",
       "  'Name': '147',\n",
       "  'Score': '',\n",
       "  'Gene': '147',\n",
       "  'Region': '2:172807494-172876579',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '178128671',\n",
       "  'End': '178167048',\n",
       "  'Name': '148',\n",
       "  'Score': '',\n",
       "  'Gene': '148',\n",
       "  'Region': '2:178128671-178167048',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '178494185',\n",
       "  'End': '178523313',\n",
       "  'Name': '149',\n",
       "  'Score': '',\n",
       "  'Gene': '149',\n",
       "  'Region': '2:178494185-178523313',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '178664771',\n",
       "  'End': '178729514',\n",
       "  'Name': '150',\n",
       "  'Score': '',\n",
       "  'Gene': '150',\n",
       "  'Region': '2:178664771-178729514',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr2',\n",
       "  'Start': '180420644',\n",
       "  'End': '180456893',\n",
       "  'Name': '151',\n",
       "  'Score': '',\n",
       "  'Gene': '151',\n",
       "  'Region': '2:180420644-180456893',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '10126470',\n",
       "  'End': '10151129',\n",
       "  'Name': '152',\n",
       "  'Score': '',\n",
       "  'Gene': '152',\n",
       "  'Region': '3:10126470-10151129',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '15090099',\n",
       "  'End': '15121257',\n",
       "  'Name': '153',\n",
       "  'Score': '',\n",
       "  'Gene': '153',\n",
       "  'Region': '3:15090099-15121257',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '19294209',\n",
       "  'End': '19328497',\n",
       "  'Name': '154',\n",
       "  'Score': '',\n",
       "  'Gene': '154',\n",
       "  'Region': '3:19294209-19328497',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '20002968',\n",
       "  'End': '20029109',\n",
       "  'Name': '155',\n",
       "  'Score': '',\n",
       "  'Gene': '155',\n",
       "  'Region': '3:20002968-20029109',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '20985804',\n",
       "  'End': '21046576',\n",
       "  'Name': '156',\n",
       "  'Score': '',\n",
       "  'Gene': '156',\n",
       "  'Region': '3:20985804-21046576',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '26814177',\n",
       "  'End': '26854088',\n",
       "  'Name': '157',\n",
       "  'Score': '',\n",
       "  'Gene': '157',\n",
       "  'Region': '3:26814177-26854088',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '27193615',\n",
       "  'End': '27237800',\n",
       "  'Name': '158',\n",
       "  'Score': '',\n",
       "  'Gene': '158',\n",
       "  'Region': '3:27193615-27237800',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '27440699',\n",
       "  'End': '27517522',\n",
       "  'Name': '159',\n",
       "  'Score': '',\n",
       "  'Gene': '159',\n",
       "  'Region': '3:27440699-27517522',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '27649981',\n",
       "  'End': '27704878',\n",
       "  'Name': '160',\n",
       "  'Score': '',\n",
       "  'Gene': '160',\n",
       "  'Region': '3:27649981-27704878',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '27995084',\n",
       "  'End': '28060214',\n",
       "  'Name': '161',\n",
       "  'Score': '',\n",
       "  'Gene': '161',\n",
       "  'Region': '3:27995084-28060214',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '33350421',\n",
       "  'End': '33417551',\n",
       "  'Name': '162',\n",
       "  'Score': '',\n",
       "  'Gene': '162',\n",
       "  'Region': '3:33350421-33417551',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '35084184',\n",
       "  'End': '35103818',\n",
       "  'Name': '163',\n",
       "  'Score': '',\n",
       "  'Gene': '163',\n",
       "  'Region': '3:35084184-35103818',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '35323296',\n",
       "  'End': '35354803',\n",
       "  'Name': '164',\n",
       "  'Score': '',\n",
       "  'Gene': '164',\n",
       "  'Region': '3:35323296-35354803',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '37648345',\n",
       "  'End': '37712903',\n",
       "  'Name': '165',\n",
       "  'Score': '',\n",
       "  'Gene': '165',\n",
       "  'Region': '3:37648345-37712903',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '50421173',\n",
       "  'End': '50438666',\n",
       "  'Name': '166',\n",
       "  'Score': '',\n",
       "  'Gene': '166',\n",
       "  'Region': '3:50421173-50438666',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '50902858',\n",
       "  'End': '50940176',\n",
       "  'Name': '167',\n",
       "  'Score': '',\n",
       "  'Gene': '167',\n",
       "  'Region': '3:50902858-50940176',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '50902858',\n",
       "  'End': '50940176',\n",
       "  'Name': '168',\n",
       "  'Score': '',\n",
       "  'Gene': '168',\n",
       "  'Region': '3:50902858-50940176',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '53729899',\n",
       "  'End': '53795193',\n",
       "  'Name': '169',\n",
       "  'Score': '',\n",
       "  'Gene': '169',\n",
       "  'Region': '3:53729899-53795193',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '54226201',\n",
       "  'End': '54292564',\n",
       "  'Name': '170',\n",
       "  'Score': '',\n",
       "  'Gene': '170',\n",
       "  'Region': '3:54226201-54292564',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '55051185',\n",
       "  'End': '55099341',\n",
       "  'Name': '171',\n",
       "  'Score': '',\n",
       "  'Gene': '171',\n",
       "  'Region': '3:55051185-55099341',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '57746286',\n",
       "  'End': '57837155',\n",
       "  'Name': '172',\n",
       "  'Score': '',\n",
       "  'Gene': '172',\n",
       "  'Region': '3:57746286-57837155',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '58089664',\n",
       "  'End': '58123121',\n",
       "  'Name': '173',\n",
       "  'Score': '',\n",
       "  'Gene': '173',\n",
       "  'Region': '3:58089664-58123121',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '59201631',\n",
       "  'End': '59287167',\n",
       "  'Name': '174',\n",
       "  'Score': '',\n",
       "  'Gene': '174',\n",
       "  'Region': '3:59201631-59287167',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '62395566',\n",
       "  'End': '62465321',\n",
       "  'Name': '175',\n",
       "  'Score': '',\n",
       "  'Gene': '175',\n",
       "  'Region': '3:62395566-62465321',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '78948383',\n",
       "  'End': '79035067',\n",
       "  'Name': '176',\n",
       "  'Score': '',\n",
       "  'Gene': '176',\n",
       "  'Region': '3:78948383-79035067',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '82023253',\n",
       "  'End': '82064011',\n",
       "  'Name': '177',\n",
       "  'Score': '',\n",
       "  'Gene': '177',\n",
       "  'Region': '3:82023253-82064011',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '83936234',\n",
       "  'End': '83993443',\n",
       "  'Name': '178',\n",
       "  'Score': '',\n",
       "  'Gene': '178',\n",
       "  'Region': '3:83936234-83993443',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '85970586',\n",
       "  'End': '86078957',\n",
       "  'Name': '179',\n",
       "  'Score': '',\n",
       "  'Gene': '179',\n",
       "  'Region': '3:85970586-86078957',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '87089865',\n",
       "  'End': '87154939',\n",
       "  'Name': '180',\n",
       "  'Score': '',\n",
       "  'Gene': '180',\n",
       "  'Region': '3:87089865-87154939',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '87272909',\n",
       "  'End': '87324252',\n",
       "  'Name': '181',\n",
       "  'Score': '',\n",
       "  'Gene': '181',\n",
       "  'Region': '3:87272909-87324252',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '88467083',\n",
       "  'End': '88490649',\n",
       "  'Name': '182',\n",
       "  'Score': '',\n",
       "  'Gene': '182',\n",
       "  'Region': '3:88467083-88490649',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '90083915',\n",
       "  'End': '90128447',\n",
       "  'Name': '183',\n",
       "  'Score': '',\n",
       "  'Gene': '183',\n",
       "  'Region': '3:90083915-90128447',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '90272479',\n",
       "  'End': '90337057',\n",
       "  'Name': '184',\n",
       "  'Score': '',\n",
       "  'Gene': '184',\n",
       "  'Region': '3:90272479-90337057',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '92227911',\n",
       "  'End': '92280171',\n",
       "  'Name': '185',\n",
       "  'Score': '',\n",
       "  'Gene': '185',\n",
       "  'Region': '3:92227911-92280171',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '95508508',\n",
       "  'End': '95541908',\n",
       "  'Name': '186',\n",
       "  'Score': '',\n",
       "  'Gene': '186',\n",
       "  'Region': '3:95508508-95541908',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '97281448',\n",
       "  'End': '97339071',\n",
       "  'Name': '187',\n",
       "  'Score': '',\n",
       "  'Gene': '187',\n",
       "  'Region': '3:97281448-97339071',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '97669551',\n",
       "  'End': '97719267',\n",
       "  'Name': '188',\n",
       "  'Score': '',\n",
       "  'Gene': '188',\n",
       "  'Region': '3:97669551-97719267',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '100393410',\n",
       "  'End': '100478704',\n",
       "  'Name': '189',\n",
       "  'Score': '',\n",
       "  'Gene': '189',\n",
       "  'Region': '3:100393410-100478704',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '102145537',\n",
       "  'End': '102194731',\n",
       "  'Name': '190',\n",
       "  'Score': '',\n",
       "  'Gene': '190',\n",
       "  'Region': '3:102145537-102194731',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '102691535',\n",
       "  'End': '102715439',\n",
       "  'Name': '191',\n",
       "  'Score': '',\n",
       "  'Gene': '191',\n",
       "  'Region': '3:102691535-102715439',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '103102173',\n",
       "  'End': '103140905',\n",
       "  'Name': '192',\n",
       "  'Score': '',\n",
       "  'Gene': '192',\n",
       "  'Region': '3:103102173-103140905',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '103373744',\n",
       "  'End': '103446515',\n",
       "  'Name': '193',\n",
       "  'Score': '',\n",
       "  'Gene': '193',\n",
       "  'Region': '3:103373744-103446515',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '104643899',\n",
       "  'End': '104683362',\n",
       "  'Name': '194',\n",
       "  'Score': '',\n",
       "  'Gene': '194',\n",
       "  'Region': '3:104643899-104683362',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '105891717',\n",
       "  'End': '105931481',\n",
       "  'Name': '195',\n",
       "  'Score': '',\n",
       "  'Gene': '195',\n",
       "  'Region': '3:105891717-105931481',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '109012413',\n",
       "  'End': '109063430',\n",
       "  'Name': '196',\n",
       "  'Score': '',\n",
       "  'Gene': '196',\n",
       "  'Region': '3:109012413-109063430',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '117217508',\n",
       "  'End': '117280078',\n",
       "  'Name': '197',\n",
       "  'Score': '',\n",
       "  'Gene': '197',\n",
       "  'Region': '3:117217508-117280078',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '117788923',\n",
       "  'End': '117854154',\n",
       "  'Name': '198',\n",
       "  'Score': '',\n",
       "  'Gene': '198',\n",
       "  'Region': '3:117788923-117854154',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '118425335',\n",
       "  'End': '118480496',\n",
       "  'Name': '199',\n",
       "  'Score': '',\n",
       "  'Gene': '199',\n",
       "  'Region': '3:118425335-118480496',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '125843531',\n",
       "  'End': '125925817',\n",
       "  'Name': '200',\n",
       "  'Score': '',\n",
       "  'Gene': '200',\n",
       "  'Region': '3:125843531-125925817',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '126609799',\n",
       "  'End': '126701558',\n",
       "  'Name': '201',\n",
       "  'Score': '',\n",
       "  'Gene': '201',\n",
       "  'Region': '3:126609799-126701558',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '127694895',\n",
       "  'End': '127729338',\n",
       "  'Name': '202',\n",
       "  'Score': '',\n",
       "  'Gene': '202',\n",
       "  'Region': '3:127694895-127729338',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '129552117',\n",
       "  'End': '129658198',\n",
       "  'Name': '203',\n",
       "  'Score': '',\n",
       "  'Gene': '203',\n",
       "  'Region': '3:129552117-129658198',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '130242712',\n",
       "  'End': '130305166',\n",
       "  'Name': '204',\n",
       "  'Score': '',\n",
       "  'Gene': '204',\n",
       "  'Region': '3:130242712-130305166',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '133461837',\n",
       "  'End': '133510394',\n",
       "  'Name': '205',\n",
       "  'Score': '',\n",
       "  'Gene': '205',\n",
       "  'Region': '3:133461837-133510394',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '135579252',\n",
       "  'End': '135663396',\n",
       "  'Name': '206',\n",
       "  'Score': '',\n",
       "  'Gene': '206',\n",
       "  'Region': '3:135579252-135663396',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '137608529',\n",
       "  'End': '137648281',\n",
       "  'Name': '207',\n",
       "  'Score': '',\n",
       "  'Gene': '207',\n",
       "  'Region': '3:137608529-137648281',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '145805719',\n",
       "  'End': '145906046',\n",
       "  'Name': '208',\n",
       "  'Score': '',\n",
       "  'Gene': '208',\n",
       "  'Region': '3:145805719-145906046',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '146310985',\n",
       "  'End': '146354892',\n",
       "  'Name': '209',\n",
       "  'Score': '',\n",
       "  'Gene': '209',\n",
       "  'Region': '3:146310985-146354892',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '152285792',\n",
       "  'End': '152333485',\n",
       "  'Name': '210',\n",
       "  'Score': '',\n",
       "  'Gene': '210',\n",
       "  'Region': '3:152285792-152333485',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr3',\n",
       "  'Start': '157149588',\n",
       "  'End': '157216143',\n",
       "  'Name': '211',\n",
       "  'Score': '',\n",
       "  'Gene': '211',\n",
       "  'Region': '3:157149588-157216143',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '3050005',\n",
       "  'End': '3090987',\n",
       "  'Name': '212',\n",
       "  'Score': '',\n",
       "  'Gene': '212',\n",
       "  'Region': '4:3050005-3090987',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '3481494',\n",
       "  'End': '3530953',\n",
       "  'Name': '213',\n",
       "  'Score': '',\n",
       "  'Gene': '213',\n",
       "  'Region': '4:3481494-3530953',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '6356853',\n",
       "  'End': '6390453',\n",
       "  'Name': '214',\n",
       "  'Score': '',\n",
       "  'Gene': '214',\n",
       "  'Region': '4:6356853-6390453',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '8556598',\n",
       "  'End': '8611298',\n",
       "  'Name': '215',\n",
       "  'Score': '',\n",
       "  'Gene': '215',\n",
       "  'Region': '4:8556598-8611298',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '9631515',\n",
       "  'End': '9658472',\n",
       "  'Name': '216',\n",
       "  'Score': '',\n",
       "  'Gene': '216',\n",
       "  'Region': '4:9631515-9658472',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '10972894',\n",
       "  'End': '10999200',\n",
       "  'Name': '217',\n",
       "  'Score': '',\n",
       "  'Gene': '217',\n",
       "  'Region': '4:10972894-10999200',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '11972627',\n",
       "  'End': '12041893',\n",
       "  'Name': '218',\n",
       "  'Score': '',\n",
       "  'Gene': '218',\n",
       "  'Region': '4:11972627-12041893',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '22477081',\n",
       "  'End': '22499068',\n",
       "  'Name': '219',\n",
       "  'Score': '',\n",
       "  'Gene': '219',\n",
       "  'Region': '4:22477081-22499068',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '24755226',\n",
       "  'End': '24825054',\n",
       "  'Name': '220',\n",
       "  'Score': '',\n",
       "  'Gene': '220',\n",
       "  'Region': '4:24755226-24825054',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '24965708',\n",
       "  'End': '25029170',\n",
       "  'Name': '221',\n",
       "  'Score': '',\n",
       "  'Gene': '221',\n",
       "  'Region': '4:24965708-25029170',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '25667294',\n",
       "  'End': '25754933',\n",
       "  'Name': '222',\n",
       "  'Score': '',\n",
       "  'Gene': '222',\n",
       "  'Region': '4:25667294-25754933',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '28814761',\n",
       "  'End': '28871872',\n",
       "  'Name': '223',\n",
       "  'Score': '',\n",
       "  'Gene': '223',\n",
       "  'Region': '4:28814761-28871872',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '32855518',\n",
       "  'End': '32890800',\n",
       "  'Name': '224',\n",
       "  'Score': '',\n",
       "  'Gene': '224',\n",
       "  'Region': '4:32855518-32890800',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '33931425',\n",
       "  'End': '33952072',\n",
       "  'Name': '225',\n",
       "  'Score': '',\n",
       "  'Gene': '225',\n",
       "  'Region': '4:33931425-33952072',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '39185238',\n",
       "  'End': '39224393',\n",
       "  'Name': '226',\n",
       "  'Score': '',\n",
       "  'Gene': '226',\n",
       "  'Region': '4:39185238-39224393',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '40478049',\n",
       "  'End': '40522554',\n",
       "  'Name': '227',\n",
       "  'Score': '',\n",
       "  'Gene': '227',\n",
       "  'Region': '4:40478049-40522554',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '40478049',\n",
       "  'End': '40519504',\n",
       "  'Name': '228',\n",
       "  'Score': '',\n",
       "  'Gene': '228',\n",
       "  'Region': '4:40478049-40519504',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '40772947',\n",
       "  'End': '40844743',\n",
       "  'Name': '229',\n",
       "  'Score': '',\n",
       "  'Gene': '229',\n",
       "  'Region': '4:40772947-40844743',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '41147781',\n",
       "  'End': '41211678',\n",
       "  'Name': '230',\n",
       "  'Score': '',\n",
       "  'Gene': '230',\n",
       "  'Region': '4:41147781-41211678',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '43205060',\n",
       "  'End': '43261596',\n",
       "  'Name': '231',\n",
       "  'Score': '',\n",
       "  'Gene': '231',\n",
       "  'Region': '4:43205060-43261596',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '43546822',\n",
       "  'End': '43571834',\n",
       "  'Name': '232',\n",
       "  'Score': '',\n",
       "  'Gene': '232',\n",
       "  'Region': '4:43546822-43571834',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '47566560',\n",
       "  'End': '47621977',\n",
       "  'Name': '233',\n",
       "  'Score': '',\n",
       "  'Gene': '233',\n",
       "  'Region': '4:47566560-47621977',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '49721396',\n",
       "  'End': '49813368',\n",
       "  'Name': '234',\n",
       "  'Score': '',\n",
       "  'Gene': '234',\n",
       "  'Region': '4:49721396-49813368',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '56820348',\n",
       "  'End': '56856569',\n",
       "  'Name': '235',\n",
       "  'Score': '',\n",
       "  'Gene': '235',\n",
       "  'Region': '4:56820348-56856569',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '56963595',\n",
       "  'End': '57049748',\n",
       "  'Name': '236',\n",
       "  'Score': '',\n",
       "  'Gene': '236',\n",
       "  'Region': '4:56963595-57049748',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '57412930',\n",
       "  'End': '57473013',\n",
       "  'Name': '237',\n",
       "  'Score': '',\n",
       "  'Gene': '237',\n",
       "  'Region': '4:57412930-57473013',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '58498835',\n",
       "  'End': '58547154',\n",
       "  'Name': '238',\n",
       "  'Score': '',\n",
       "  'Gene': '238',\n",
       "  'Region': '4:58498835-58547154',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '62298487',\n",
       "  'End': '62369491',\n",
       "  'Name': '239',\n",
       "  'Score': '',\n",
       "  'Gene': '239',\n",
       "  'Region': '4:62298487-62369491',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '65716234',\n",
       "  'End': '65786043',\n",
       "  'Name': '240',\n",
       "  'Score': '',\n",
       "  'Gene': '240',\n",
       "  'Region': '4:65716234-65786043',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '66361813',\n",
       "  'End': '66415032',\n",
       "  'Name': '241',\n",
       "  'Score': '',\n",
       "  'Gene': '241',\n",
       "  'Region': '4:66361813-66415032',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '70442043',\n",
       "  'End': '70529439',\n",
       "  'Name': '242',\n",
       "  'Score': '',\n",
       "  'Gene': '242',\n",
       "  'Region': '4:70442043-70529439',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '86158471',\n",
       "  'End': '86210141',\n",
       "  'Name': '243',\n",
       "  'Score': '',\n",
       "  'Gene': '243',\n",
       "  'Region': '4:86158471-86210141',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '87001120',\n",
       "  'End': '87063412',\n",
       "  'Name': '244',\n",
       "  'Score': '',\n",
       "  'Gene': '244',\n",
       "  'Region': '4:87001120-87063412',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '88152498',\n",
       "  'End': '88265144',\n",
       "  'Name': '245',\n",
       "  'Score': '',\n",
       "  'Gene': '245',\n",
       "  'Region': '4:88152498-88265144',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '98082752',\n",
       "  'End': '98119663',\n",
       "  'Name': '246',\n",
       "  'Score': '',\n",
       "  'Gene': '246',\n",
       "  'Region': '4:98082752-98119663',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '98541272',\n",
       "  'End': '98628773',\n",
       "  'Name': '247',\n",
       "  'Score': '',\n",
       "  'Gene': '247',\n",
       "  'Region': '4:98541272-98628773',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '99340446',\n",
       "  'End': '99385864',\n",
       "  'Name': '248',\n",
       "  'Score': '',\n",
       "  'Gene': '248',\n",
       "  'Region': '4:99340446-99385864',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '100973562',\n",
       "  'End': '101008943',\n",
       "  'Name': '249',\n",
       "  'Score': '',\n",
       "  'Gene': '249',\n",
       "  'Region': '4:100973562-101008943',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '104993954',\n",
       "  'End': '105050480',\n",
       "  'Name': '250',\n",
       "  'Score': '',\n",
       "  'Gene': '250',\n",
       "  'Region': '4:104993954-105050480',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '105909094',\n",
       "  'End': '105942080',\n",
       "  'Name': '251',\n",
       "  'Score': '',\n",
       "  'Gene': '251',\n",
       "  'Region': '4:105909094-105942080',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '114599240',\n",
       "  'End': '114666242',\n",
       "  'Name': '252',\n",
       "  'Score': '',\n",
       "  'Gene': '252',\n",
       "  'Region': '4:114599240-114666242',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '115047700',\n",
       "  'End': '115102192',\n",
       "  'Name': '253',\n",
       "  'Score': '',\n",
       "  'Gene': '253',\n",
       "  'Region': '4:115047700-115102192',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '115847369',\n",
       "  'End': '115903327',\n",
       "  'Name': '254',\n",
       "  'Score': '',\n",
       "  'Gene': '254',\n",
       "  'Region': '4:115847369-115903327',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '115847369',\n",
       "  'End': '115903327',\n",
       "  'Name': '255',\n",
       "  'Score': '',\n",
       "  'Gene': '255',\n",
       "  'Region': '4:115847369-115903327',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '116638906',\n",
       "  'End': '116673403',\n",
       "  'Name': '256',\n",
       "  'Score': '',\n",
       "  'Gene': '256',\n",
       "  'Region': '4:116638906-116673403',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '117628081',\n",
       "  'End': '117677368',\n",
       "  'Name': '257',\n",
       "  'Score': '',\n",
       "  'Gene': '257',\n",
       "  'Region': '4:117628081-117677368',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '123209825',\n",
       "  'End': '123254995',\n",
       "  'Name': '258',\n",
       "  'Score': '',\n",
       "  'Gene': '258',\n",
       "  'Region': '4:123209825-123254995',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '123971238',\n",
       "  'End': '124053077',\n",
       "  'Name': '259',\n",
       "  'Score': '',\n",
       "  'Gene': '259',\n",
       "  'Region': '4:123971238-124053077',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '124834301',\n",
       "  'End': '124856678',\n",
       "  'Name': '260',\n",
       "  'Score': '',\n",
       "  'Gene': '260',\n",
       "  'Region': '4:124834301-124856678',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '126217621',\n",
       "  'End': '126253017',\n",
       "  'Name': '261',\n",
       "  'Score': '',\n",
       "  'Gene': '261',\n",
       "  'Region': '4:126217621-126253017',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '127005025',\n",
       "  'End': '127039608',\n",
       "  'Name': '262',\n",
       "  'Score': '',\n",
       "  'Gene': '262',\n",
       "  'Region': '4:127005025-127039608',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '127503306',\n",
       "  'End': '127553000',\n",
       "  'Name': '263',\n",
       "  'Score': '',\n",
       "  'Gene': '263',\n",
       "  'Region': '4:127503306-127553000',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '127676510',\n",
       "  'End': '127708932',\n",
       "  'Name': '264',\n",
       "  'Score': '',\n",
       "  'Gene': '264',\n",
       "  'Region': '4:127676510-127708932',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '127810701',\n",
       "  'End': '127862488',\n",
       "  'Name': '265',\n",
       "  'Score': '',\n",
       "  'Gene': '265',\n",
       "  'Region': '4:127810701-127862488',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '129495757',\n",
       "  'End': '129522804',\n",
       "  'Name': '266',\n",
       "  'Score': '',\n",
       "  'Gene': '266',\n",
       "  'Region': '4:129495757-129522804',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '130246140',\n",
       "  'End': '130297752',\n",
       "  'Name': '267',\n",
       "  'Score': '',\n",
       "  'Gene': '267',\n",
       "  'Region': '4:130246140-130297752',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '130519810',\n",
       "  'End': '130569025',\n",
       "  'Name': '268',\n",
       "  'Score': '',\n",
       "  'Gene': '268',\n",
       "  'Region': '4:130519810-130569025',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '131743908',\n",
       "  'End': '131834723',\n",
       "  'Name': '269',\n",
       "  'Score': '',\n",
       "  'Gene': '269',\n",
       "  'Region': '4:131743908-131834723',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '132802079',\n",
       "  'End': '132824284',\n",
       "  'Name': '270',\n",
       "  'Score': '',\n",
       "  'Gene': '270',\n",
       "  'Region': '4:132802079-132824284',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '135817186',\n",
       "  'End': '135863467',\n",
       "  'Name': '271',\n",
       "  'Score': '',\n",
       "  'Gene': '271',\n",
       "  'Region': '4:135817186-135863467',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '137287244',\n",
       "  'End': '137314456',\n",
       "  'Name': '272',\n",
       "  'Score': '',\n",
       "  'Gene': '272',\n",
       "  'Region': '4:137287244-137314456',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '148845951',\n",
       "  'End': '148923937',\n",
       "  'Name': '273',\n",
       "  'Score': '',\n",
       "  'Gene': '273',\n",
       "  'Region': '4:148845951-148923937',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr4',\n",
       "  'Start': '155769707',\n",
       "  'End': '155792098',\n",
       "  'Name': '274',\n",
       "  'Score': '',\n",
       "  'Gene': '274',\n",
       "  'Region': '4:155769707-155792098',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '4004952',\n",
       "  'End': '4080244',\n",
       "  'Name': '275',\n",
       "  'Score': '',\n",
       "  'Gene': '275',\n",
       "  'Region': '5:4004952-4080244',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '4947946',\n",
       "  'End': '5020393',\n",
       "  'Name': '276',\n",
       "  'Score': '',\n",
       "  'Gene': '276',\n",
       "  'Region': '5:4947946-5020393',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '4947946',\n",
       "  'End': '5020393',\n",
       "  'Name': '277',\n",
       "  'Score': '',\n",
       "  'Gene': '277',\n",
       "  'Region': '5:4947946-5020393',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '5226972',\n",
       "  'End': '5303712',\n",
       "  'Name': '278',\n",
       "  'Score': '',\n",
       "  'Gene': '278',\n",
       "  'Region': '5:5226972-5303712',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '15957115',\n",
       "  'End': '16016399',\n",
       "  'Name': '279',\n",
       "  'Score': '',\n",
       "  'Gene': '279',\n",
       "  'Region': '5:15957115-16016399',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '19904500',\n",
       "  'End': '19951425',\n",
       "  'Name': '280',\n",
       "  'Score': '',\n",
       "  'Gene': '280',\n",
       "  'Region': '5:19904500-19951425',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '21467453',\n",
       "  'End': '21518145',\n",
       "  'Name': '281',\n",
       "  'Score': '',\n",
       "  'Gene': '281',\n",
       "  'Region': '5:21467453-21518145',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '24780084',\n",
       "  'End': '24821106',\n",
       "  'Name': '282',\n",
       "  'Score': '',\n",
       "  'Gene': '282',\n",
       "  'Region': '5:24780084-24821106',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '25604458',\n",
       "  'End': '25624378',\n",
       "  'Name': '283',\n",
       "  'Score': '',\n",
       "  'Gene': '283',\n",
       "  'Region': '5:25604458-25624378',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '27059822',\n",
       "  'End': '27099916',\n",
       "  'Name': '284',\n",
       "  'Score': '',\n",
       "  'Gene': '284',\n",
       "  'Region': '5:27059822-27099916',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '30391241',\n",
       "  'End': '30442747',\n",
       "  'Name': '285',\n",
       "  'Score': '',\n",
       "  'Gene': '285',\n",
       "  'Region': '5:30391241-30442747',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '33203371',\n",
       "  'End': '33246343',\n",
       "  'Name': '286',\n",
       "  'Score': '',\n",
       "  'Gene': '286',\n",
       "  'Region': '5:33203371-33246343',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '38352152',\n",
       "  'End': '38404629',\n",
       "  'Name': '287',\n",
       "  'Score': '',\n",
       "  'Gene': '287',\n",
       "  'Region': '5:38352152-38404629',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '38516393',\n",
       "  'End': '38554776',\n",
       "  'Name': '288',\n",
       "  'Score': '',\n",
       "  'Gene': '288',\n",
       "  'Region': '5:38516393-38554776',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '38860470',\n",
       "  'End': '38900029',\n",
       "  'Name': '289',\n",
       "  'Score': '',\n",
       "  'Gene': '289',\n",
       "  'Region': '5:38860470-38900029',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '39116286',\n",
       "  'End': '39158083',\n",
       "  'Name': '290',\n",
       "  'Score': '',\n",
       "  'Gene': '290',\n",
       "  'Region': '5:39116286-39158083',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '43328583',\n",
       "  'End': '43360939',\n",
       "  'Name': '291',\n",
       "  'Score': '',\n",
       "  'Gene': '291',\n",
       "  'Region': '5:43328583-43360939',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '44199271',\n",
       "  'End': '44268791',\n",
       "  'Name': '292',\n",
       "  'Score': '',\n",
       "  'Gene': '292',\n",
       "  'Region': '5:44199271-44268791',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '44646554',\n",
       "  'End': '44742497',\n",
       "  'Name': '293',\n",
       "  'Score': '',\n",
       "  'Gene': '293',\n",
       "  'Region': '5:44646554-44742497',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '44863523',\n",
       "  'End': '44909113',\n",
       "  'Name': '294',\n",
       "  'Score': '',\n",
       "  'Gene': '294',\n",
       "  'Region': '5:44863523-44909113',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '44863523',\n",
       "  'End': '44909113',\n",
       "  'Name': '295',\n",
       "  'Score': '',\n",
       "  'Gene': '295',\n",
       "  'Region': '5:44863523-44909113',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '49961907',\n",
       "  'End': '50049873',\n",
       "  'Name': '296',\n",
       "  'Score': '',\n",
       "  'Gene': '296',\n",
       "  'Region': '5:49961907-50049873',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '52789669',\n",
       "  'End': '52823675',\n",
       "  'Name': '297',\n",
       "  'Score': '',\n",
       "  'Gene': '297',\n",
       "  'Region': '5:52789669-52823675',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '58023151',\n",
       "  'End': '58128957',\n",
       "  'Name': '298',\n",
       "  'Score': '',\n",
       "  'Gene': '298',\n",
       "  'Region': '5:58023151-58128957',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '62638998',\n",
       "  'End': '62690814',\n",
       "  'Name': '299',\n",
       "  'Score': '',\n",
       "  'Gene': '299',\n",
       "  'Region': '5:62638998-62690814',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '63815226',\n",
       "  'End': '63906930',\n",
       "  'Name': '300',\n",
       "  'Score': '',\n",
       "  'Gene': '300',\n",
       "  'Region': '5:63815226-63906930',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '64750899',\n",
       "  'End': '64813898',\n",
       "  'Name': '301',\n",
       "  'Score': '',\n",
       "  'Gene': '301',\n",
       "  'Region': '5:64750899-64813898',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '66062841',\n",
       "  'End': '66088951',\n",
       "  'Name': '302',\n",
       "  'Score': '',\n",
       "  'Gene': '302',\n",
       "  'Region': '5:66062841-66088951',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '67340974',\n",
       "  'End': '67401928',\n",
       "  'Name': '303',\n",
       "  'Score': '',\n",
       "  'Gene': '303',\n",
       "  'Region': '5:67340974-67401928',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '71602818',\n",
       "  'End': '71651574',\n",
       "  'Name': '304',\n",
       "  'Score': '',\n",
       "  'Gene': '304',\n",
       "  'Region': '5:71602818-71651574',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '72817519',\n",
       "  'End': '72891992',\n",
       "  'Name': '305',\n",
       "  'Score': '',\n",
       "  'Gene': '305',\n",
       "  'Region': '5:72817519-72891992',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '74611244',\n",
       "  'End': '74648537',\n",
       "  'Name': '306',\n",
       "  'Score': '',\n",
       "  'Gene': '306',\n",
       "  'Region': '5:74611244-74648537',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '75635002',\n",
       "  'End': '75684903',\n",
       "  'Name': '307',\n",
       "  'Score': '',\n",
       "  'Gene': '307',\n",
       "  'Region': '5:75635002-75684903',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '77354237',\n",
       "  'End': '77394169',\n",
       "  'Name': '308',\n",
       "  'Score': '',\n",
       "  'Gene': '308',\n",
       "  'Region': '5:77354237-77394169',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '88564694',\n",
       "  'End': '88601033',\n",
       "  'Name': '309',\n",
       "  'Score': '',\n",
       "  'Gene': '309',\n",
       "  'Region': '5:88564694-88601033',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '91548047',\n",
       "  'End': '91638373',\n",
       "  'Name': '310',\n",
       "  'Score': '',\n",
       "  'Gene': '310',\n",
       "  'Region': '5:91548047-91638373',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '92341917',\n",
       "  'End': '92398829',\n",
       "  'Name': '311',\n",
       "  'Score': '',\n",
       "  'Gene': '311',\n",
       "  'Region': '5:92341917-92398829',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '104301330',\n",
       "  'End': '104345424',\n",
       "  'Name': '312',\n",
       "  'Score': '',\n",
       "  'Gene': '312',\n",
       "  'Region': '5:104301330-104345424',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '106773128',\n",
       "  'End': '106826267',\n",
       "  'Name': '313',\n",
       "  'Score': '',\n",
       "  'Gene': '313',\n",
       "  'Region': '5:106773128-106826267',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '108677189',\n",
       "  'End': '108735317',\n",
       "  'Name': '314',\n",
       "  'Score': '',\n",
       "  'Gene': '314',\n",
       "  'Region': '5:108677189-108735317',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '116985881',\n",
       "  'End': '117037536',\n",
       "  'Name': '315',\n",
       "  'Score': '',\n",
       "  'Gene': '315',\n",
       "  'Region': '5:116985881-117037536',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '119047555',\n",
       "  'End': '119100105',\n",
       "  'Name': '316',\n",
       "  'Score': '',\n",
       "  'Gene': '316',\n",
       "  'Region': '5:119047555-119100105',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '126238032',\n",
       "  'End': '126282172',\n",
       "  'Name': '317',\n",
       "  'Score': '',\n",
       "  'Gene': '317',\n",
       "  'Region': '5:126238032-126282172',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '128191775',\n",
       "  'End': '128244686',\n",
       "  'Name': '318',\n",
       "  'Score': '',\n",
       "  'Gene': '318',\n",
       "  'Region': '5:128191775-128244686',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '129751425',\n",
       "  'End': '129779547',\n",
       "  'Name': '319',\n",
       "  'Score': '',\n",
       "  'Gene': '319',\n",
       "  'Region': '5:129751425-129779547',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '129935623',\n",
       "  'End': '129967156',\n",
       "  'Name': '320',\n",
       "  'Score': '',\n",
       "  'Gene': '320',\n",
       "  'Region': '5:129935623-129967156',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '132247594',\n",
       "  'End': '132314711',\n",
       "  'Name': '321',\n",
       "  'Score': '',\n",
       "  'Gene': '321',\n",
       "  'Region': '5:132247594-132314711',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '133963020',\n",
       "  'End': '134020324',\n",
       "  'Name': '322',\n",
       "  'Score': '',\n",
       "  'Gene': '322',\n",
       "  'Region': '5:133963020-134020324',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '135015402',\n",
       "  'End': '135057436',\n",
       "  'Name': '323',\n",
       "  'Score': '',\n",
       "  'Gene': '323',\n",
       "  'Region': '5:135015402-135057436',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '135015402',\n",
       "  'End': '135054026',\n",
       "  'Name': '324',\n",
       "  'Score': '',\n",
       "  'Gene': '324',\n",
       "  'Region': '5:135015402-135054026',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '138767956',\n",
       "  'End': '138797859',\n",
       "  'Name': '325',\n",
       "  'Score': '',\n",
       "  'Gene': '325',\n",
       "  'Region': '5:138767956-138797859',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '141038058',\n",
       "  'End': '141094622',\n",
       "  'Name': '326',\n",
       "  'Score': '',\n",
       "  'Gene': '326',\n",
       "  'Region': '5:141038058-141094622',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '141745482',\n",
       "  'End': '141794869',\n",
       "  'Name': '327',\n",
       "  'Score': '',\n",
       "  'Gene': '327',\n",
       "  'Region': '5:141745482-141794869',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '142023394',\n",
       "  'End': '142083792',\n",
       "  'Name': '328',\n",
       "  'Score': '',\n",
       "  'Gene': '328',\n",
       "  'Region': '5:142023394-142083792',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '143470529',\n",
       "  'End': '143521173',\n",
       "  'Name': '329',\n",
       "  'Score': '',\n",
       "  'Gene': '329',\n",
       "  'Region': '5:143470529-143521173',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '144542912',\n",
       "  'End': '144604769',\n",
       "  'Name': '330',\n",
       "  'Score': '',\n",
       "  'Gene': '330',\n",
       "  'Region': '5:144542912-144604769',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '147020488',\n",
       "  'End': '147071113',\n",
       "  'Name': '331',\n",
       "  'Score': '',\n",
       "  'Gene': '331',\n",
       "  'Region': '5:147020488-147071113',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '147020488',\n",
       "  'End': '147071113',\n",
       "  'Name': '332',\n",
       "  'Score': '',\n",
       "  'Gene': '332',\n",
       "  'Region': '5:147020488-147071113',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '148486071',\n",
       "  'End': '148542936',\n",
       "  'Name': '333',\n",
       "  'Score': '',\n",
       "  'Gene': '333',\n",
       "  'Region': '5:148486071-148542936',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr5',\n",
       "  'Start': '150097399',\n",
       "  'End': '150169014',\n",
       "  'Name': '334',\n",
       "  'Score': '',\n",
       "  'Gene': '334',\n",
       "  'Region': '5:150097399-150169014',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '4923611',\n",
       "  'End': '5052849',\n",
       "  'Name': '335',\n",
       "  'Score': '',\n",
       "  'Gene': '335',\n",
       "  'Region': '6:4923611-5052849',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '5826895',\n",
       "  'End': '5935849',\n",
       "  'Name': '336',\n",
       "  'Score': '',\n",
       "  'Gene': '336',\n",
       "  'Region': '6:5826895-5935849',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '6152617',\n",
       "  'End': '6208463',\n",
       "  'Name': '337',\n",
       "  'Score': '',\n",
       "  'Gene': '337',\n",
       "  'Region': '6:6152617-6208463',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '17102402',\n",
       "  'End': '17171916',\n",
       "  'Name': '338',\n",
       "  'Score': '',\n",
       "  'Gene': '338',\n",
       "  'Region': '6:17102402-17171916',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '17584012',\n",
       "  'End': '17630341',\n",
       "  'Name': '339',\n",
       "  'Score': '',\n",
       "  'Gene': '339',\n",
       "  'Region': '6:17584012-17630341',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '18408404',\n",
       "  'End': '18451976',\n",
       "  'Name': '340',\n",
       "  'Score': '',\n",
       "  'Gene': '340',\n",
       "  'Region': '6:18408404-18451976',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '23446147',\n",
       "  'End': '23511005',\n",
       "  'Name': '341',\n",
       "  'Score': '',\n",
       "  'Gene': '341',\n",
       "  'Region': '6:23446147-23511005',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '24520973',\n",
       "  'End': '24571391',\n",
       "  'Name': '342',\n",
       "  'Score': '',\n",
       "  'Gene': '342',\n",
       "  'Region': '6:24520973-24571391',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '27885819',\n",
       "  'End': '27985930',\n",
       "  'Name': '343',\n",
       "  'Score': '',\n",
       "  'Gene': '343',\n",
       "  'Region': '6:27885819-27985930',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '29198389',\n",
       "  'End': '29216427',\n",
       "  'Name': '344',\n",
       "  'Score': '',\n",
       "  'Gene': '344',\n",
       "  'Region': '6:29198389-29216427',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '30527033',\n",
       "  'End': '30580432',\n",
       "  'Name': '345',\n",
       "  'Score': '',\n",
       "  'Gene': '345',\n",
       "  'Region': '6:30527033-30580432',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '34481419',\n",
       "  'End': '34515684',\n",
       "  'Name': '346',\n",
       "  'Score': '',\n",
       "  'Gene': '346',\n",
       "  'Region': '6:34481419-34515684',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '34481419',\n",
       "  'End': '34515893',\n",
       "  'Name': '347',\n",
       "  'Score': '',\n",
       "  'Gene': '347',\n",
       "  'Region': '6:34481419-34515893',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '34866828',\n",
       "  'End': '34902427',\n",
       "  'Name': '348',\n",
       "  'Score': '',\n",
       "  'Gene': '348',\n",
       "  'Region': '6:34866828-34902427',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '41077178',\n",
       "  'End': '41137415',\n",
       "  'Name': '349',\n",
       "  'Score': '',\n",
       "  'Gene': '349',\n",
       "  'Region': '6:41077178-41137415',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '41504636',\n",
       "  'End': '41560746',\n",
       "  'Name': '350',\n",
       "  'Score': '',\n",
       "  'Gene': '350',\n",
       "  'Region': '6:41504636-41560746',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '42339795',\n",
       "  'End': '42363704',\n",
       "  'Name': '351',\n",
       "  'Score': '',\n",
       "  'Gene': '351',\n",
       "  'Region': '6:42339795-42363704',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '48389712',\n",
       "  'End': '48434906',\n",
       "  'Name': '352',\n",
       "  'Score': '',\n",
       "  'Gene': '352',\n",
       "  'Region': '6:48389712-48434906',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '49817253',\n",
       "  'End': '49837041',\n",
       "  'Name': '353',\n",
       "  'Score': '',\n",
       "  'Gene': '353',\n",
       "  'Region': '6:49817253-49837041',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '51168260',\n",
       "  'End': '51185983',\n",
       "  'Name': '354',\n",
       "  'Score': '',\n",
       "  'Gene': '354',\n",
       "  'Region': '6:51168260-51185983',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '53580205',\n",
       "  'End': '53685638',\n",
       "  'Name': '355',\n",
       "  'Score': '',\n",
       "  'Gene': '355',\n",
       "  'Region': '6:53580205-53685638',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '54698853',\n",
       "  'End': '54736443',\n",
       "  'Name': '356',\n",
       "  'Score': '',\n",
       "  'Gene': '356',\n",
       "  'Region': '6:54698853-54736443',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '55791236',\n",
       "  'End': '55815137',\n",
       "  'Name': '357',\n",
       "  'Score': '',\n",
       "  'Gene': '357',\n",
       "  'Region': '6:55791236-55815137',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '57743581',\n",
       "  'End': '57816248',\n",
       "  'Name': '358',\n",
       "  'Score': '',\n",
       "  'Gene': '358',\n",
       "  'Region': '6:57743581-57816248',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '72824718',\n",
       "  'End': '72889657',\n",
       "  'Name': '359',\n",
       "  'Score': '',\n",
       "  'Gene': '359',\n",
       "  'Region': '6:72824718-72889657',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '77413708',\n",
       "  'End': '77476471',\n",
       "  'Name': '360',\n",
       "  'Score': '',\n",
       "  'Gene': '360',\n",
       "  'Region': '6:77413708-77476471',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '81782875',\n",
       "  'End': '81823647',\n",
       "  'Name': '361',\n",
       "  'Score': '',\n",
       "  'Gene': '361',\n",
       "  'Region': '6:81782875-81823647',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '81975780',\n",
       "  'End': '81991014',\n",
       "  'Name': '362',\n",
       "  'Score': '',\n",
       "  'Gene': '362',\n",
       "  'Region': '6:81975780-81991014',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '82996732',\n",
       "  'End': '83024917',\n",
       "  'Name': '363',\n",
       "  'Score': '',\n",
       "  'Gene': '363',\n",
       "  'Region': '6:82996732-83024917',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '83184995',\n",
       "  'End': '83216237',\n",
       "  'Name': '364',\n",
       "  'Score': '',\n",
       "  'Gene': '364',\n",
       "  'Region': '6:83184995-83216237',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '83857140',\n",
       "  'End': '83900764',\n",
       "  'Name': '365',\n",
       "  'Score': '',\n",
       "  'Gene': '365',\n",
       "  'Region': '6:83857140-83900764',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '84119647',\n",
       "  'End': '84135755',\n",
       "  'Name': '366',\n",
       "  'Score': '',\n",
       "  'Gene': '366',\n",
       "  'Region': '6:84119647-84135755',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '84562937',\n",
       "  'End': '84590580',\n",
       "  'Name': '367',\n",
       "  'Score': '',\n",
       "  'Gene': '367',\n",
       "  'Region': '6:84562937-84590580',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '90403398',\n",
       "  'End': '90435394',\n",
       "  'Name': '368',\n",
       "  'Score': '',\n",
       "  'Gene': '368',\n",
       "  'Region': '6:90403398-90435394',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '97837648',\n",
       "  'End': '97903932',\n",
       "  'Name': '369',\n",
       "  'Score': '',\n",
       "  'Gene': '369',\n",
       "  'Region': '6:97837648-97903932',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '98208965',\n",
       "  'End': '98288774',\n",
       "  'Name': '370',\n",
       "  'Score': '',\n",
       "  'Gene': '370',\n",
       "  'Region': '6:98208965-98288774',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '98930069',\n",
       "  'End': '99020726',\n",
       "  'Name': '371',\n",
       "  'Score': '',\n",
       "  'Gene': '371',\n",
       "  'Region': '6:98930069-99020726',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '99329695',\n",
       "  'End': '99383737',\n",
       "  'Name': '372',\n",
       "  'Score': '',\n",
       "  'Gene': '372',\n",
       "  'Region': '6:99329695-99383737',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '100911891',\n",
       "  'End': '100939915',\n",
       "  'Name': '373',\n",
       "  'Score': '',\n",
       "  'Gene': '373',\n",
       "  'Region': '6:100911891-100939915',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '106380677',\n",
       "  'End': '106441259',\n",
       "  'Name': '374',\n",
       "  'Score': '',\n",
       "  'Gene': '374',\n",
       "  'Region': '6:106380677-106441259',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '111481365',\n",
       "  'End': '111536119',\n",
       "  'Name': '375',\n",
       "  'Score': '',\n",
       "  'Gene': '375',\n",
       "  'Region': '6:111481365-111536119',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '113261068',\n",
       "  'End': '113287839',\n",
       "  'Name': '376',\n",
       "  'Score': '',\n",
       "  'Gene': '376',\n",
       "  'Region': '6:113261068-113287839',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '115462622',\n",
       "  'End': '115523851',\n",
       "  'Name': '377',\n",
       "  'Score': '',\n",
       "  'Gene': '377',\n",
       "  'Region': '6:115462622-115523851',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '116346040',\n",
       "  'End': '116424204',\n",
       "  'Name': '378',\n",
       "  'Score': '',\n",
       "  'Gene': '378',\n",
       "  'Region': '6:116346040-116424204',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '118593580',\n",
       "  'End': '118649770',\n",
       "  'Name': '379',\n",
       "  'Score': '',\n",
       "  'Gene': '379',\n",
       "  'Region': '6:118593580-118649770',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '121366679',\n",
       "  'End': '121468570',\n",
       "  'Name': '380',\n",
       "  'Score': '',\n",
       "  'Gene': '380',\n",
       "  'Region': '6:121366679-121468570',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '127079741',\n",
       "  'End': '127142836',\n",
       "  'Name': '381',\n",
       "  'Score': '',\n",
       "  'Gene': '381',\n",
       "  'Region': '6:127079741-127142836',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '128401401',\n",
       "  'End': '128433169',\n",
       "  'Name': '382',\n",
       "  'Score': '',\n",
       "  'Gene': '382',\n",
       "  'Region': '6:128401401-128433169',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '131366549',\n",
       "  'End': '131434281',\n",
       "  'Name': '383',\n",
       "  'Score': '',\n",
       "  'Gene': '383',\n",
       "  'Region': '6:131366549-131434281',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '134696900',\n",
       "  'End': '134763941',\n",
       "  'Name': '384',\n",
       "  'Score': '',\n",
       "  'Gene': '384',\n",
       "  'Region': '6:134696900-134763941',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '137088345',\n",
       "  'End': '137163906',\n",
       "  'Name': '385',\n",
       "  'Score': '',\n",
       "  'Gene': '385',\n",
       "  'Region': '6:137088345-137163906',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '142487481',\n",
       "  'End': '142525279',\n",
       "  'Name': '386',\n",
       "  'Score': '',\n",
       "  'Gene': '386',\n",
       "  'Region': '6:142487481-142525279',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '144595170',\n",
       "  'End': '144642901',\n",
       "  'Name': '387',\n",
       "  'Score': '',\n",
       "  'Gene': '387',\n",
       "  'Region': '6:144595170-144642901',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '144990272',\n",
       "  'End': '145037109',\n",
       "  'Name': '388',\n",
       "  'Score': '',\n",
       "  'Gene': '388',\n",
       "  'Region': '6:144990272-145037109',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '145275832',\n",
       "  'End': '145307520',\n",
       "  'Name': '389',\n",
       "  'Score': '',\n",
       "  'Gene': '389',\n",
       "  'Region': '6:145275832-145307520',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '145733149',\n",
       "  'End': '145803383',\n",
       "  'Name': '390',\n",
       "  'Score': '',\n",
       "  'Gene': '390',\n",
       "  'Region': '6:145733149-145803383',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr6',\n",
       "  'Start': '146330927',\n",
       "  'End': '146358669',\n",
       "  'Name': '391',\n",
       "  'Score': '',\n",
       "  'Gene': '391',\n",
       "  'Region': '6:146330927-146358669',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '4478044',\n",
       "  'End': '4513679',\n",
       "  'Name': '392',\n",
       "  'Score': '',\n",
       "  'Gene': '392',\n",
       "  'Region': '7:4478044-4513679',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '25624559',\n",
       "  'End': '25651968',\n",
       "  'Name': '393',\n",
       "  'Score': '',\n",
       "  'Gene': '393',\n",
       "  'Region': '7:25624559-25651968',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '28175434',\n",
       "  'End': '28214615',\n",
       "  'Name': '394',\n",
       "  'Score': '',\n",
       "  'Gene': '394',\n",
       "  'Region': '7:28175434-28214615',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '28559913',\n",
       "  'End': '28601972',\n",
       "  'Name': '395',\n",
       "  'Score': '',\n",
       "  'Gene': '395',\n",
       "  'Region': '7:28559913-28601972',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '30256410',\n",
       "  'End': '30301465',\n",
       "  'Name': '396',\n",
       "  'Score': '',\n",
       "  'Gene': '396',\n",
       "  'Region': '7:30256410-30301465',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '30895416',\n",
       "  'End': '30928012',\n",
       "  'Name': '397',\n",
       "  'Score': '',\n",
       "  'Gene': '397',\n",
       "  'Region': '7:30895416-30928012',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '35686730',\n",
       "  'End': '35776268',\n",
       "  'Name': '398',\n",
       "  'Score': '',\n",
       "  'Gene': '398',\n",
       "  'Region': '7:35686730-35776268',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '36045635',\n",
       "  'End': '36085581',\n",
       "  'Name': '399',\n",
       "  'Score': '',\n",
       "  'Gene': '399',\n",
       "  'Region': '7:36045635-36085581',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '36913879',\n",
       "  'End': '36954287',\n",
       "  'Name': '400',\n",
       "  'Score': '',\n",
       "  'Gene': '400',\n",
       "  'Region': '7:36913879-36954287',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '43543744',\n",
       "  'End': '43572104',\n",
       "  'Name': '401',\n",
       "  'Score': '',\n",
       "  'Gene': '401',\n",
       "  'Region': '7:43543744-43572104',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '47056107',\n",
       "  'End': '47117036',\n",
       "  'Name': '402',\n",
       "  'Score': '',\n",
       "  'Gene': '402',\n",
       "  'Region': '7:47056107-47117036',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '47056107',\n",
       "  'End': '47111005',\n",
       "  'Name': '403',\n",
       "  'Score': '',\n",
       "  'Gene': '403',\n",
       "  'Region': '7:47056107-47111005',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '50503534',\n",
       "  'End': '50585678',\n",
       "  'Name': '404',\n",
       "  'Score': '',\n",
       "  'Gene': '404',\n",
       "  'Region': '7:50503534-50585678',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '64295872',\n",
       "  'End': '64361201',\n",
       "  'Name': '405',\n",
       "  'Score': '',\n",
       "  'Gene': '405',\n",
       "  'Region': '7:64295872-64361201',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '65870622',\n",
       "  'End': '65935471',\n",
       "  'Name': '406',\n",
       "  'Score': '',\n",
       "  'Gene': '406',\n",
       "  'Region': '7:65870622-65935471',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '67823905',\n",
       "  'End': '67883031',\n",
       "  'Name': '407',\n",
       "  'Score': '',\n",
       "  'Gene': '407',\n",
       "  'Region': '7:67823905-67883031',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '70326208',\n",
       "  'End': '70356368',\n",
       "  'Name': '408',\n",
       "  'Score': '',\n",
       "  'Gene': '408',\n",
       "  'Region': '7:70326208-70356368',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '81261500',\n",
       "  'End': '81322036',\n",
       "  'Name': '409',\n",
       "  'Score': '',\n",
       "  'Gene': '409',\n",
       "  'Region': '7:81261500-81322036',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '82801317',\n",
       "  'End': '82853443',\n",
       "  'Name': '410',\n",
       "  'Score': '',\n",
       "  'Gene': '410',\n",
       "  'Region': '7:82801317-82853443',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '83044578',\n",
       "  'End': '83095834',\n",
       "  'Name': '411',\n",
       "  'Score': '',\n",
       "  'Gene': '411',\n",
       "  'Region': '7:83044578-83095834',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '83639439',\n",
       "  'End': '83676053',\n",
       "  'Name': '412',\n",
       "  'Score': '',\n",
       "  'Gene': '412',\n",
       "  'Region': '7:83639439-83676053',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '83800534',\n",
       "  'End': '83839178',\n",
       "  'Name': '413',\n",
       "  'Score': '',\n",
       "  'Gene': '413',\n",
       "  'Region': '7:83800534-83839178',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '84245734',\n",
       "  'End': '84301961',\n",
       "  'Name': '414',\n",
       "  'Score': '',\n",
       "  'Gene': '414',\n",
       "  'Region': '7:84245734-84301961',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '89269710',\n",
       "  'End': '89332531',\n",
       "  'Name': '415',\n",
       "  'Score': '',\n",
       "  'Gene': '415',\n",
       "  'Region': '7:89269710-89332531',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '97745161',\n",
       "  'End': '97805491',\n",
       "  'Name': '416',\n",
       "  'Score': '',\n",
       "  'Gene': '416',\n",
       "  'Region': '7:97745161-97805491',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '98135903',\n",
       "  'End': '98171642',\n",
       "  'Name': '417',\n",
       "  'Score': '',\n",
       "  'Gene': '417',\n",
       "  'Region': '7:98135903-98171642',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '101900453',\n",
       "  'End': '101918530',\n",
       "  'Name': '418',\n",
       "  'Score': '',\n",
       "  'Gene': '418',\n",
       "  'Region': '7:101900453-101918530',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '107661264',\n",
       "  'End': '107709048',\n",
       "  'Name': '419',\n",
       "  'Score': '',\n",
       "  'Gene': '419',\n",
       "  'Region': '7:107661264-107709048',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '109924369',\n",
       "  'End': '109953604',\n",
       "  'Name': '420',\n",
       "  'Score': '',\n",
       "  'Gene': '420',\n",
       "  'Region': '7:109924369-109953604',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '112762685',\n",
       "  'End': '112884620',\n",
       "  'Name': '421',\n",
       "  'Score': '',\n",
       "  'Gene': '421',\n",
       "  'Region': '7:112762685-112884620',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '114291439',\n",
       "  'End': '114323757',\n",
       "  'Name': '422',\n",
       "  'Score': '',\n",
       "  'Gene': '422',\n",
       "  'Region': '7:114291439-114323757',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '115487765',\n",
       "  'End': '115538476',\n",
       "  'Name': '423',\n",
       "  'Score': '',\n",
       "  'Gene': '423',\n",
       "  'Region': '7:115487765-115538476',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '115646108',\n",
       "  'End': '115707089',\n",
       "  'Name': '424',\n",
       "  'Score': '',\n",
       "  'Gene': '424',\n",
       "  'Region': '7:115646108-115707089',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '118123230',\n",
       "  'End': '118160731',\n",
       "  'Name': '425',\n",
       "  'Score': '',\n",
       "  'Gene': '425',\n",
       "  'Region': '7:118123230-118160731',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '119726960',\n",
       "  'End': '119745414',\n",
       "  'Name': '426',\n",
       "  'Score': '',\n",
       "  'Gene': '426',\n",
       "  'Region': '7:119726960-119745414',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '121213744',\n",
       "  'End': '121302825',\n",
       "  'Name': '427',\n",
       "  'Score': '',\n",
       "  'Gene': '427',\n",
       "  'Region': '7:121213744-121302825',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '126248527',\n",
       "  'End': '126280938',\n",
       "  'Name': '428',\n",
       "  'Score': '',\n",
       "  'Gene': '428',\n",
       "  'Region': '7:126248527-126280938',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '127081384',\n",
       "  'End': '127130312',\n",
       "  'Name': '429',\n",
       "  'Score': '',\n",
       "  'Gene': '429',\n",
       "  'Region': '7:127081384-127130312',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '127636855',\n",
       "  'End': '127700283',\n",
       "  'Name': '430',\n",
       "  'Score': '',\n",
       "  'Gene': '430',\n",
       "  'Region': '7:127636855-127700283',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '130215243',\n",
       "  'End': '130259178',\n",
       "  'Name': '431',\n",
       "  'Score': '',\n",
       "  'Gene': '431',\n",
       "  'Region': '7:130215243-130259178',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '132226605',\n",
       "  'End': '132285072',\n",
       "  'Name': '432',\n",
       "  'Score': '',\n",
       "  'Gene': '432',\n",
       "  'Region': '7:132226605-132285072',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '134663339',\n",
       "  'End': '134680281',\n",
       "  'Name': '433',\n",
       "  'Score': '',\n",
       "  'Gene': '433',\n",
       "  'Region': '7:134663339-134680281',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '136046278',\n",
       "  'End': '136095849',\n",
       "  'Name': '434',\n",
       "  'Score': '',\n",
       "  'Gene': '434',\n",
       "  'Region': '7:136046278-136095849',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '138176035',\n",
       "  'End': '138249068',\n",
       "  'Name': '435',\n",
       "  'Score': '',\n",
       "  'Gene': '435',\n",
       "  'Region': '7:138176035-138249068',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr7',\n",
       "  'Start': '142333852',\n",
       "  'End': '142428691',\n",
       "  'Name': '436',\n",
       "  'Score': '',\n",
       "  'Gene': '436',\n",
       "  'Region': '7:142333852-142428691',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '8600019',\n",
       "  'End': '8650327',\n",
       "  'Name': '437',\n",
       "  'Score': '',\n",
       "  'Gene': '437',\n",
       "  'Region': '8:8600019-8650327',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '13919259',\n",
       "  'End': '13966263',\n",
       "  'Name': '438',\n",
       "  'Score': '',\n",
       "  'Gene': '438',\n",
       "  'Region': '8:13919259-13966263',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '15549842',\n",
       "  'End': '15609200',\n",
       "  'Name': '439',\n",
       "  'Score': '',\n",
       "  'Gene': '439',\n",
       "  'Region': '8:15549842-15609200',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '15994403',\n",
       "  'End': '16033579',\n",
       "  'Name': '440',\n",
       "  'Score': '',\n",
       "  'Gene': '440',\n",
       "  'Region': '8:15994403-16033579',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '17454670',\n",
       "  'End': '17519064',\n",
       "  'Name': '441',\n",
       "  'Score': '',\n",
       "  'Gene': '441',\n",
       "  'Region': '8:17454670-17519064',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '18916539',\n",
       "  'End': '18954569',\n",
       "  'Name': '442',\n",
       "  'Score': '',\n",
       "  'Gene': '442',\n",
       "  'Region': '8:18916539-18954569',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '20115193',\n",
       "  'End': '20155338',\n",
       "  'Name': '443',\n",
       "  'Score': '',\n",
       "  'Gene': '443',\n",
       "  'Region': '8:20115193-20155338',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '25348122',\n",
       "  'End': '25369634',\n",
       "  'Name': '444',\n",
       "  'Score': '',\n",
       "  'Gene': '444',\n",
       "  'Region': '8:25348122-25369634',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '26686886',\n",
       "  'End': '26714279',\n",
       "  'Name': '445',\n",
       "  'Score': '',\n",
       "  'Gene': '445',\n",
       "  'Region': '8:26686886-26714279',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '34815877',\n",
       "  'End': '34893394',\n",
       "  'Name': '446',\n",
       "  'Score': '',\n",
       "  'Gene': '446',\n",
       "  'Region': '8:34815877-34893394',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '35159287',\n",
       "  'End': '35244368',\n",
       "  'Name': '447',\n",
       "  'Score': '',\n",
       "  'Gene': '447',\n",
       "  'Region': '8:35159287-35244368',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '35368926',\n",
       "  'End': '35434058',\n",
       "  'Name': '448',\n",
       "  'Score': '',\n",
       "  'Gene': '448',\n",
       "  'Region': '8:35368926-35434058',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '36410955',\n",
       "  'End': '36459588',\n",
       "  'Name': '449',\n",
       "  'Score': '',\n",
       "  'Gene': '449',\n",
       "  'Region': '8:36410955-36459588',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '36629739',\n",
       "  'End': '36715384',\n",
       "  'Name': '450',\n",
       "  'Score': '',\n",
       "  'Gene': '450',\n",
       "  'Region': '8:36629739-36715384',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '40572713',\n",
       "  'End': '40628441',\n",
       "  'Name': '451',\n",
       "  'Score': '',\n",
       "  'Gene': '451',\n",
       "  'Region': '8:40572713-40628441',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '41338066',\n",
       "  'End': '41384762',\n",
       "  'Name': '452',\n",
       "  'Score': '',\n",
       "  'Gene': '452',\n",
       "  'Region': '8:41338066-41384762',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '47680301',\n",
       "  'End': '47707341',\n",
       "  'Name': '453',\n",
       "  'Score': '',\n",
       "  'Gene': '453',\n",
       "  'Region': '8:47680301-47707341',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '47831006',\n",
       "  'End': '47903730',\n",
       "  'Name': '454',\n",
       "  'Score': '',\n",
       "  'Gene': '454',\n",
       "  'Region': '8:47831006-47903730',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '48186246',\n",
       "  'End': '48222764',\n",
       "  'Name': '455',\n",
       "  'Score': '',\n",
       "  'Gene': '455',\n",
       "  'Region': '8:48186246-48222764',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '48480073',\n",
       "  'End': '48568850',\n",
       "  'Name': '456',\n",
       "  'Score': '',\n",
       "  'Gene': '456',\n",
       "  'Region': '8:48480073-48568850',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '61249539',\n",
       "  'End': '61334852',\n",
       "  'Name': '457',\n",
       "  'Score': '',\n",
       "  'Gene': '457',\n",
       "  'Region': '8:61249539-61334852',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '67938931',\n",
       "  'End': '67995568',\n",
       "  'Name': '458',\n",
       "  'Score': '',\n",
       "  'Gene': '458',\n",
       "  'Region': '8:67938931-67995568',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '68171685',\n",
       "  'End': '68214740',\n",
       "  'Name': '459',\n",
       "  'Score': '',\n",
       "  'Gene': '459',\n",
       "  'Region': '8:68171685-68214740',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '68514342',\n",
       "  'End': '68559036',\n",
       "  'Name': '460',\n",
       "  'Score': '',\n",
       "  'Gene': '460',\n",
       "  'Region': '8:68514342-68559036',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '71296208',\n",
       "  'End': '71359935',\n",
       "  'Name': '461',\n",
       "  'Score': '',\n",
       "  'Gene': '461',\n",
       "  'Region': '8:71296208-71359935',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '72642943',\n",
       "  'End': '72698838',\n",
       "  'Name': '462',\n",
       "  'Score': '',\n",
       "  'Gene': '462',\n",
       "  'Region': '8:72642943-72698838',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '75043814',\n",
       "  'End': '75089992',\n",
       "  'Name': '463',\n",
       "  'Score': '',\n",
       "  'Gene': '463',\n",
       "  'Region': '8:75043814-75089992',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '77000149',\n",
       "  'End': '77094743',\n",
       "  'Name': '464',\n",
       "  'Score': '',\n",
       "  'Gene': '464',\n",
       "  'Region': '8:77000149-77094743',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '78018249',\n",
       "  'End': '78064137',\n",
       "  'Name': '465',\n",
       "  'Score': '',\n",
       "  'Gene': '465',\n",
       "  'Region': '8:78018249-78064137',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '79331964',\n",
       "  'End': '79392981',\n",
       "  'Name': '466',\n",
       "  'Score': '',\n",
       "  'Gene': '466',\n",
       "  'Region': '8:79331964-79392981',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '81341631',\n",
       "  'End': '81400269',\n",
       "  'Name': '467',\n",
       "  'Score': '',\n",
       "  'Gene': '467',\n",
       "  'Region': '8:81341631-81400269',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '83402102',\n",
       "  'End': '83451711',\n",
       "  'Name': '468',\n",
       "  'Score': '',\n",
       "  'Gene': '468',\n",
       "  'Region': '8:83402102-83451711',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '84183157',\n",
       "  'End': '84204478',\n",
       "  'Name': '469',\n",
       "  'Score': '',\n",
       "  'Gene': '469',\n",
       "  'Region': '8:84183157-84204478',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '88279593',\n",
       "  'End': '88332980',\n",
       "  'Name': '470',\n",
       "  'Score': '',\n",
       "  'Gene': '470',\n",
       "  'Region': '8:88279593-88332980',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '90273317',\n",
       "  'End': '90324986',\n",
       "  'Name': '471',\n",
       "  'Score': '',\n",
       "  'Gene': '471',\n",
       "  'Region': '8:90273317-90324986',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '90491007',\n",
       "  'End': '90543012',\n",
       "  'Name': '472',\n",
       "  'Score': '',\n",
       "  'Gene': '472',\n",
       "  'Region': '8:90491007-90543012',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '90837081',\n",
       "  'End': '90894757',\n",
       "  'Name': '473',\n",
       "  'Score': '',\n",
       "  'Gene': '473',\n",
       "  'Region': '8:90837081-90894757',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '92119780',\n",
       "  'End': '92134791',\n",
       "  'Name': '474',\n",
       "  'Score': '',\n",
       "  'Gene': '474',\n",
       "  'Region': '8:92119780-92134791',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '95156834',\n",
       "  'End': '95176362',\n",
       "  'Name': '475',\n",
       "  'Score': '',\n",
       "  'Gene': '475',\n",
       "  'Region': '8:95156834-95176362',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '103832726',\n",
       "  'End': '103882126',\n",
       "  'Name': '476',\n",
       "  'Score': '',\n",
       "  'Gene': '476',\n",
       "  'Region': '8:103832726-103882126',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '104403397',\n",
       "  'End': '104437891',\n",
       "  'Name': '477',\n",
       "  'Score': '',\n",
       "  'Gene': '477',\n",
       "  'Region': '8:104403397-104437891',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '105432888',\n",
       "  'End': '105484815',\n",
       "  'Name': '478',\n",
       "  'Score': '',\n",
       "  'Gene': '478',\n",
       "  'Region': '8:105432888-105484815',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '107444145',\n",
       "  'End': '107523024',\n",
       "  'Name': '479',\n",
       "  'Score': '',\n",
       "  'Gene': '479',\n",
       "  'Region': '8:107444145-107523024',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '109903221',\n",
       "  'End': '109984503',\n",
       "  'Name': '480',\n",
       "  'Score': '',\n",
       "  'Gene': '480',\n",
       "  'Region': '8:109903221-109984503',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '110925709',\n",
       "  'End': '110986339',\n",
       "  'Name': '481',\n",
       "  'Score': '',\n",
       "  'Gene': '481',\n",
       "  'Region': '8:110925709-110986339',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '112616029',\n",
       "  'End': '112682110',\n",
       "  'Name': '482',\n",
       "  'Score': '',\n",
       "  'Gene': '482',\n",
       "  'Region': '8:112616029-112682110',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '114450631',\n",
       "  'End': '114506127',\n",
       "  'Name': '483',\n",
       "  'Score': '',\n",
       "  'Gene': '483',\n",
       "  'Region': '8:114450631-114506127',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '114898227',\n",
       "  'End': '115009038',\n",
       "  'Name': '484',\n",
       "  'Score': '',\n",
       "  'Gene': '484',\n",
       "  'Region': '8:114898227-115009038',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '115131077',\n",
       "  'End': '115180969',\n",
       "  'Name': '485',\n",
       "  'Score': '',\n",
       "  'Gene': '485',\n",
       "  'Region': '8:115131077-115180969',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '116582552',\n",
       "  'End': '116622019',\n",
       "  'Name': '486',\n",
       "  'Score': '',\n",
       "  'Gene': '486',\n",
       "  'Region': '8:116582552-116622019',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '118095022',\n",
       "  'End': '118162697',\n",
       "  'Name': '487',\n",
       "  'Score': '',\n",
       "  'Gene': '487',\n",
       "  'Region': '8:118095022-118162697',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '119120924',\n",
       "  'End': '119161282',\n",
       "  'Name': '488',\n",
       "  'Score': '',\n",
       "  'Gene': '488',\n",
       "  'Region': '8:119120924-119161282',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr8',\n",
       "  'Start': '119785828',\n",
       "  'End': '119830110',\n",
       "  'Name': '489',\n",
       "  'Score': '',\n",
       "  'Gene': '489',\n",
       "  'Region': '8:119785828-119830110',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '3000250',\n",
       "  'End': '3036292',\n",
       "  'Name': '490',\n",
       "  'Score': '',\n",
       "  'Gene': '490',\n",
       "  'Region': '9:3000250-3036292',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '8899764',\n",
       "  'End': '8928793',\n",
       "  'Name': '491',\n",
       "  'Score': '',\n",
       "  'Gene': '491',\n",
       "  'Region': '9:8899764-8928793',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '9093125',\n",
       "  'End': '9145126',\n",
       "  'Name': '492',\n",
       "  'Score': '',\n",
       "  'Gene': '492',\n",
       "  'Region': '9:9093125-9145126',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '14512860',\n",
       "  'End': '14585101',\n",
       "  'Name': '493',\n",
       "  'Score': '',\n",
       "  'Gene': '493',\n",
       "  'Region': '9:14512860-14585101',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '15493272',\n",
       "  'End': '15553906',\n",
       "  'Name': '494',\n",
       "  'Score': '',\n",
       "  'Gene': '494',\n",
       "  'Region': '9:15493272-15553906',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '22557286',\n",
       "  'End': '22655336',\n",
       "  'Name': '495',\n",
       "  'Score': '',\n",
       "  'Gene': '495',\n",
       "  'Region': '9:22557286-22655336',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '22793499',\n",
       "  'End': '22854592',\n",
       "  'Name': '496',\n",
       "  'Score': '',\n",
       "  'Gene': '496',\n",
       "  'Region': '9:22793499-22854592',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '24408454',\n",
       "  'End': '24483239',\n",
       "  'Name': '497',\n",
       "  'Score': '',\n",
       "  'Gene': '497',\n",
       "  'Region': '9:24408454-24483239',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '32489082',\n",
       "  'End': '32531627',\n",
       "  'Name': '498',\n",
       "  'Score': '',\n",
       "  'Gene': '498',\n",
       "  'Region': '9:32489082-32531627',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '32701049',\n",
       "  'End': '32759587',\n",
       "  'Name': '499',\n",
       "  'Score': '',\n",
       "  'Gene': '499',\n",
       "  'Region': '9:32701049-32759587',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '37406529',\n",
       "  'End': '37438756',\n",
       "  'Name': '500',\n",
       "  'Score': '',\n",
       "  'Gene': '500',\n",
       "  'Region': '9:37406529-37438756',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '46301898',\n",
       "  'End': '46354450',\n",
       "  'Name': '501',\n",
       "  'Score': '',\n",
       "  'Gene': '501',\n",
       "  'Region': '9:46301898-46354450',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '48017145',\n",
       "  'End': '48073481',\n",
       "  'Name': '502',\n",
       "  'Score': '',\n",
       "  'Gene': '502',\n",
       "  'Region': '9:48017145-48073481',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '48186289',\n",
       "  'End': '48267700',\n",
       "  'Name': '503',\n",
       "  'Score': '',\n",
       "  'Gene': '503',\n",
       "  'Region': '9:48186289-48267700',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '50064571',\n",
       "  'End': '50150541',\n",
       "  'Name': '504',\n",
       "  'Score': '',\n",
       "  'Gene': '504',\n",
       "  'Region': '9:50064571-50150541',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '52367436',\n",
       "  'End': '52436749',\n",
       "  'Name': '505',\n",
       "  'Score': '',\n",
       "  'Gene': '505',\n",
       "  'Region': '9:52367436-52436749',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '53903431',\n",
       "  'End': '53948595',\n",
       "  'Name': '506',\n",
       "  'Score': '',\n",
       "  'Gene': '506',\n",
       "  'Region': '9:53903431-53948595',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '54308886',\n",
       "  'End': '54344997',\n",
       "  'Name': '507',\n",
       "  'Score': '',\n",
       "  'Gene': '507',\n",
       "  'Region': '9:54308886-54344997',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '54518565',\n",
       "  'End': '54555386',\n",
       "  'Name': '508',\n",
       "  'Score': '',\n",
       "  'Gene': '508',\n",
       "  'Region': '9:54518565-54555386',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '54655501',\n",
       "  'End': '54692064',\n",
       "  'Name': '509',\n",
       "  'Score': '',\n",
       "  'Gene': '509',\n",
       "  'Region': '9:54655501-54692064',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '57250601',\n",
       "  'End': '57289305',\n",
       "  'Name': '510',\n",
       "  'Score': '',\n",
       "  'Gene': '510',\n",
       "  'Region': '9:57250601-57289305',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '63610882',\n",
       "  'End': '63656718',\n",
       "  'Name': '511',\n",
       "  'Score': '',\n",
       "  'Gene': '511',\n",
       "  'Region': '9:63610882-63656718',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '64184332',\n",
       "  'End': '64242851',\n",
       "  'Name': '512',\n",
       "  'Score': '',\n",
       "  'Gene': '512',\n",
       "  'Region': '9:64184332-64242851',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '67395710',\n",
       "  'End': '67441674',\n",
       "  'Name': '513',\n",
       "  'Score': '',\n",
       "  'Gene': '513',\n",
       "  'Region': '9:67395710-67441674',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '67683420',\n",
       "  'End': '67753632',\n",
       "  'Name': '514',\n",
       "  'Score': '',\n",
       "  'Gene': '514',\n",
       "  'Region': '9:67683420-67753632',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '69084838',\n",
       "  'End': '69135422',\n",
       "  'Name': '515',\n",
       "  'Score': '',\n",
       "  'Gene': '515',\n",
       "  'Region': '9:69084838-69135422',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '71534978',\n",
       "  'End': '71582718',\n",
       "  'Name': '516',\n",
       "  'Score': '',\n",
       "  'Gene': '516',\n",
       "  'Region': '9:71534978-71582718',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '71911483',\n",
       "  'End': '71960383',\n",
       "  'Name': '517',\n",
       "  'Score': '',\n",
       "  'Gene': '517',\n",
       "  'Region': '9:71911483-71960383',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '73154502',\n",
       "  'End': '73192707',\n",
       "  'Name': '518',\n",
       "  'Score': '',\n",
       "  'Gene': '518',\n",
       "  'Region': '9:73154502-73192707',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '77708777',\n",
       "  'End': '77751284',\n",
       "  'Name': '519',\n",
       "  'Score': '',\n",
       "  'Gene': '519',\n",
       "  'Region': '9:77708777-77751284',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '78121826',\n",
       "  'End': '78169455',\n",
       "  'Name': '520',\n",
       "  'Score': '',\n",
       "  'Gene': '520',\n",
       "  'Region': '9:78121826-78169455',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '80169682',\n",
       "  'End': '80256480',\n",
       "  'Name': '521',\n",
       "  'Score': '',\n",
       "  'Gene': '521',\n",
       "  'Region': '9:80169682-80256480',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '83304219',\n",
       "  'End': '83359953',\n",
       "  'Name': '522',\n",
       "  'Score': '',\n",
       "  'Gene': '522',\n",
       "  'Region': '9:83304219-83359953',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '83304219',\n",
       "  'End': '83359953',\n",
       "  'Name': '523',\n",
       "  'Score': '',\n",
       "  'Gene': '523',\n",
       "  'Region': '9:83304219-83359953',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '86583601',\n",
       "  'End': '86637179',\n",
       "  'Name': '524',\n",
       "  'Score': '',\n",
       "  'Gene': '524',\n",
       "  'Region': '9:86583601-86637179',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '89256295',\n",
       "  'End': '89293992',\n",
       "  'Name': '525',\n",
       "  'Score': '',\n",
       "  'Gene': '525',\n",
       "  'Region': '9:89256295-89293992',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '89841577',\n",
       "  'End': '89901977',\n",
       "  'Name': '526',\n",
       "  'Score': '',\n",
       "  'Gene': '526',\n",
       "  'Region': '9:89841577-89901977',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '90014552',\n",
       "  'End': '90077294',\n",
       "  'Name': '527',\n",
       "  'Score': '',\n",
       "  'Gene': '527',\n",
       "  'Region': '9:90014552-90077294',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '90185735',\n",
       "  'End': '90267232',\n",
       "  'Name': '528',\n",
       "  'Score': '',\n",
       "  'Gene': '528',\n",
       "  'Region': '9:90185735-90267232',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '96424547',\n",
       "  'End': '96472062',\n",
       "  'Name': '529',\n",
       "  'Score': '',\n",
       "  'Gene': '529',\n",
       "  'Region': '9:96424547-96472062',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '97899864',\n",
       "  'End': '97975879',\n",
       "  'Name': '530',\n",
       "  'Score': '',\n",
       "  'Gene': '530',\n",
       "  'Region': '9:97899864-97975879',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '99056362',\n",
       "  'End': '99121260',\n",
       "  'Name': '531',\n",
       "  'Score': '',\n",
       "  'Gene': '531',\n",
       "  'Region': '9:99056362-99121260',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '99763284',\n",
       "  'End': '99808353',\n",
       "  'Name': '532',\n",
       "  'Score': '',\n",
       "  'Gene': '532',\n",
       "  'Region': '9:99763284-99808353',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '102492333',\n",
       "  'End': '102513822',\n",
       "  'Name': '533',\n",
       "  'Score': '',\n",
       "  'Gene': '533',\n",
       "  'Region': '9:102492333-102513822',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '104898780',\n",
       "  'End': '104978138',\n",
       "  'Name': '534',\n",
       "  'Score': '',\n",
       "  'Gene': '534',\n",
       "  'Region': '9:104898780-104978138',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '105275008',\n",
       "  'End': '105319057',\n",
       "  'Name': '535',\n",
       "  'Score': '',\n",
       "  'Gene': '535',\n",
       "  'Region': '9:105275008-105319057',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '105844542',\n",
       "  'End': '105910743',\n",
       "  'Name': '536',\n",
       "  'Score': '',\n",
       "  'Gene': '536',\n",
       "  'Region': '9:105844542-105910743',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '106924617',\n",
       "  'End': '106975928',\n",
       "  'Name': '537',\n",
       "  'Score': '',\n",
       "  'Gene': '537',\n",
       "  'Region': '9:106924617-106975928',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '108400917',\n",
       "  'End': '108453922',\n",
       "  'Name': '538',\n",
       "  'Score': '',\n",
       "  'Gene': '538',\n",
       "  'Region': '9:108400917-108453922',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '109527022',\n",
       "  'End': '109563625',\n",
       "  'Name': '539',\n",
       "  'Score': '',\n",
       "  'Gene': '539',\n",
       "  'Region': '9:109527022-109563625',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '110413831',\n",
       "  'End': '110449627',\n",
       "  'Name': '540',\n",
       "  'Score': '',\n",
       "  'Gene': '540',\n",
       "  'Region': '9:110413831-110449627',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '110889347',\n",
       "  'End': '110934746',\n",
       "  'Name': '541',\n",
       "  'Score': '',\n",
       "  'Gene': '541',\n",
       "  'Region': '9:110889347-110934746',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '112431698',\n",
       "  'End': '112477152',\n",
       "  'Name': '542',\n",
       "  'Score': '',\n",
       "  'Gene': '542',\n",
       "  'Region': '9:112431698-112477152',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '113770660',\n",
       "  'End': '113803213',\n",
       "  'Name': '543',\n",
       "  'Score': '',\n",
       "  'Gene': '543',\n",
       "  'Region': '9:113770660-113803213',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '119409249',\n",
       "  'End': '119439068',\n",
       "  'Name': '544',\n",
       "  'Score': '',\n",
       "  'Gene': '544',\n",
       "  'Region': '9:119409249-119439068',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr9',\n",
       "  'Start': '123431255',\n",
       "  'End': '123481622',\n",
       "  'Name': '545',\n",
       "  'Score': '',\n",
       "  'Gene': '545',\n",
       "  'Region': '9:123431255-123481622',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '8176017',\n",
       "  'End': '8240797',\n",
       "  'Name': '546',\n",
       "  'Score': '',\n",
       "  'Gene': '546',\n",
       "  'Region': '10:8176017-8240797',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '9801406',\n",
       "  'End': '9893185',\n",
       "  'Name': '547',\n",
       "  'Score': '',\n",
       "  'Gene': '547',\n",
       "  'Region': '10:9801406-9893185',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '12840738',\n",
       "  'End': '12875969',\n",
       "  'Name': '548',\n",
       "  'Score': '',\n",
       "  'Gene': '548',\n",
       "  'Region': '10:12840738-12875969',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '13564012',\n",
       "  'End': '13601581',\n",
       "  'Name': '549',\n",
       "  'Score': '',\n",
       "  'Gene': '549',\n",
       "  'Region': '10:13564012-13601581',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '19546426',\n",
       "  'End': '19597009',\n",
       "  'Name': '550',\n",
       "  'Score': '',\n",
       "  'Gene': '550',\n",
       "  'Region': '10:19546426-19597009',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '21891312',\n",
       "  'End': '21981778',\n",
       "  'Name': '551',\n",
       "  'Score': '',\n",
       "  'Gene': '551',\n",
       "  'Region': '10:21891312-21981778',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '23191367',\n",
       "  'End': '23267580',\n",
       "  'Name': '552',\n",
       "  'Score': '',\n",
       "  'Gene': '552',\n",
       "  'Region': '10:23191367-23267580',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '24961345',\n",
       "  'End': '25001868',\n",
       "  'Name': '553',\n",
       "  'Score': '',\n",
       "  'Gene': '553',\n",
       "  'Region': '10:24961345-25001868',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '25410587',\n",
       "  'End': '25466543',\n",
       "  'Name': '554',\n",
       "  'Score': '',\n",
       "  'Gene': '554',\n",
       "  'Region': '10:25410587-25466543',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '26267092',\n",
       "  'End': '26315268',\n",
       "  'Name': '555',\n",
       "  'Score': '',\n",
       "  'Gene': '555',\n",
       "  'Region': '10:26267092-26315268',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '28083265',\n",
       "  'End': '28189418',\n",
       "  'Name': '556',\n",
       "  'Score': '',\n",
       "  'Gene': '556',\n",
       "  'Region': '10:28083265-28189418',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '34131442',\n",
       "  'End': '34198522',\n",
       "  'Name': '557',\n",
       "  'Score': '',\n",
       "  'Gene': '557',\n",
       "  'Region': '10:34131442-34198522',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '38972790',\n",
       "  'End': '39030119',\n",
       "  'Name': '558',\n",
       "  'Score': '',\n",
       "  'Gene': '558',\n",
       "  'Region': '10:38972790-39030119',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '40068995',\n",
       "  'End': '40129830',\n",
       "  'Name': '559',\n",
       "  'Score': '',\n",
       "  'Gene': '559',\n",
       "  'Region': '10:40068995-40129830',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '41186005',\n",
       "  'End': '41249634',\n",
       "  'Name': '560',\n",
       "  'Score': '',\n",
       "  'Gene': '560',\n",
       "  'Region': '10:41186005-41249634',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '42557711',\n",
       "  'End': '42592574',\n",
       "  'Name': '561',\n",
       "  'Score': '',\n",
       "  'Gene': '561',\n",
       "  'Region': '10:42557711-42592574',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '43276076',\n",
       "  'End': '43350048',\n",
       "  'Name': '562',\n",
       "  'Score': '',\n",
       "  'Gene': '562',\n",
       "  'Region': '10:43276076-43350048',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '44292271',\n",
       "  'End': '44355306',\n",
       "  'Name': '563',\n",
       "  'Score': '',\n",
       "  'Gene': '563',\n",
       "  'Region': '10:44292271-44355306',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '44625388',\n",
       "  'End': '44703060',\n",
       "  'Name': '564',\n",
       "  'Score': '',\n",
       "  'Gene': '564',\n",
       "  'Region': '10:44625388-44703060',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '49760898',\n",
       "  'End': '49807819',\n",
       "  'Name': '565',\n",
       "  'Score': '',\n",
       "  'Gene': '565',\n",
       "  'Region': '10:49760898-49807819',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '52702507',\n",
       "  'End': '52761120',\n",
       "  'Name': '566',\n",
       "  'Score': '',\n",
       "  'Gene': '566',\n",
       "  'Region': '10:52702507-52761120',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '52901887',\n",
       "  'End': '52953599',\n",
       "  'Name': '567',\n",
       "  'Score': '',\n",
       "  'Gene': '567',\n",
       "  'Region': '10:52901887-52953599',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '58600997',\n",
       "  'End': '58673875',\n",
       "  'Name': '568',\n",
       "  'Score': '',\n",
       "  'Gene': '568',\n",
       "  'Region': '10:58600997-58673875',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '61317332',\n",
       "  'End': '61375454',\n",
       "  'Name': '569',\n",
       "  'Score': '',\n",
       "  'Gene': '569',\n",
       "  'Region': '10:61317332-61375454',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '61534277',\n",
       "  'End': '61606208',\n",
       "  'Name': '570',\n",
       "  'Score': '',\n",
       "  'Gene': '570',\n",
       "  'Region': '10:61534277-61606208',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '62846832',\n",
       "  'End': '62887538',\n",
       "  'Name': '571',\n",
       "  'Score': '',\n",
       "  'Gene': '571',\n",
       "  'Region': '10:62846832-62887538',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '63143174',\n",
       "  'End': '63190019',\n",
       "  'Name': '572',\n",
       "  'Score': '',\n",
       "  'Gene': '572',\n",
       "  'Region': '10:63143174-63190019',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '64039707',\n",
       "  'End': '64088108',\n",
       "  'Name': '573',\n",
       "  'Score': '',\n",
       "  'Gene': '573',\n",
       "  'Region': '10:64039707-64088108',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '66879118',\n",
       "  'End': '66931818',\n",
       "  'Name': '574',\n",
       "  'Score': '',\n",
       "  'Gene': '574',\n",
       "  'Region': '10:66879118-66931818',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '67851028',\n",
       "  'End': '67904443',\n",
       "  'Name': '575',\n",
       "  'Score': '',\n",
       "  'Gene': '575',\n",
       "  'Region': '10:67851028-67904443',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '69239620',\n",
       "  'End': '69260356',\n",
       "  'Name': '576',\n",
       "  'Score': '',\n",
       "  'Gene': '576',\n",
       "  'Region': '10:69239620-69260356',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '71291190',\n",
       "  'End': '71339781',\n",
       "  'Name': '577',\n",
       "  'Score': '',\n",
       "  'Gene': '577',\n",
       "  'Region': '10:71291190-71339781',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '75872641',\n",
       "  'End': '75903051',\n",
       "  'Name': '578',\n",
       "  'Score': '',\n",
       "  'Gene': '578',\n",
       "  'Region': '10:75872641-75903051',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '75872641',\n",
       "  'End': '75903051',\n",
       "  'Name': '579',\n",
       "  'Score': '',\n",
       "  'Gene': '579',\n",
       "  'Region': '10:75872641-75903051',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '79712586',\n",
       "  'End': '79738477',\n",
       "  'Name': '580',\n",
       "  'Score': '',\n",
       "  'Gene': '580',\n",
       "  'Region': '10:79712586-79738477',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '82638910',\n",
       "  'End': '82682199',\n",
       "  'Name': '581',\n",
       "  'Score': '',\n",
       "  'Gene': '581',\n",
       "  'Region': '10:82638910-82682199',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '83617495',\n",
       "  'End': '83660925',\n",
       "  'Name': '582',\n",
       "  'Score': '',\n",
       "  'Gene': '582',\n",
       "  'Region': '10:83617495-83660925',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '83847002',\n",
       "  'End': '83943108',\n",
       "  'Name': '583',\n",
       "  'Score': '',\n",
       "  'Gene': '583',\n",
       "  'Region': '10:83847002-83943108',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '84087311',\n",
       "  'End': '84165022',\n",
       "  'Name': '584',\n",
       "  'Score': '',\n",
       "  'Gene': '584',\n",
       "  'Region': '10:84087311-84165022',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '84931289',\n",
       "  'End': '85023749',\n",
       "  'Name': '585',\n",
       "  'Score': '',\n",
       "  'Gene': '585',\n",
       "  'Region': '10:84931289-85023749',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '86032255',\n",
       "  'End': '86076646',\n",
       "  'Name': '586',\n",
       "  'Score': '',\n",
       "  'Gene': '586',\n",
       "  'Region': '10:86032255-86076646',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '87485669',\n",
       "  'End': '87504242',\n",
       "  'Name': '587',\n",
       "  'Score': '',\n",
       "  'Gene': '587',\n",
       "  'Region': '10:87485669-87504242',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '87872034',\n",
       "  'End': '87941886',\n",
       "  'Name': '588',\n",
       "  'Score': '',\n",
       "  'Gene': '588',\n",
       "  'Region': '10:87872034-87941886',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '91181367',\n",
       "  'End': '91207531',\n",
       "  'Name': '589',\n",
       "  'Score': '',\n",
       "  'Gene': '589',\n",
       "  'Region': '10:91181367-91207531',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '92225234',\n",
       "  'End': '92249181',\n",
       "  'Name': '590',\n",
       "  'Score': '',\n",
       "  'Gene': '590',\n",
       "  'Region': '10:92225234-92249181',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '93648004',\n",
       "  'End': '93704413',\n",
       "  'Name': '591',\n",
       "  'Score': '',\n",
       "  'Gene': '591',\n",
       "  'Region': '10:93648004-93704413',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '94046937',\n",
       "  'End': '94098212',\n",
       "  'Name': '592',\n",
       "  'Score': '',\n",
       "  'Gene': '592',\n",
       "  'Region': '10:94046937-94098212',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '94230759',\n",
       "  'End': '94266806',\n",
       "  'Name': '593',\n",
       "  'Score': '',\n",
       "  'Gene': '593',\n",
       "  'Region': '10:94230759-94266806',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '96125549',\n",
       "  'End': '96214510',\n",
       "  'Name': '594',\n",
       "  'Score': '',\n",
       "  'Gene': '594',\n",
       "  'Region': '10:96125549-96214510',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '98562540',\n",
       "  'End': '98662759',\n",
       "  'Name': '595',\n",
       "  'Score': '',\n",
       "  'Gene': '595',\n",
       "  'Region': '10:98562540-98662759',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '98921845',\n",
       "  'End': '98959798',\n",
       "  'Name': '596',\n",
       "  'Score': '',\n",
       "  'Gene': '596',\n",
       "  'Region': '10:98921845-98959798',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '99151730',\n",
       "  'End': '99205126',\n",
       "  'Name': '597',\n",
       "  'Score': '',\n",
       "  'Gene': '597',\n",
       "  'Region': '10:99151730-99205126',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '105484481',\n",
       "  'End': '105540782',\n",
       "  'Name': '598',\n",
       "  'Score': '',\n",
       "  'Gene': '598',\n",
       "  'Region': '10:105484481-105540782',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '106585266',\n",
       "  'End': '106732470',\n",
       "  'Name': '599',\n",
       "  'Score': '',\n",
       "  'Gene': '599',\n",
       "  'Region': '10:106585266-106732470',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '110927563',\n",
       "  'End': '110999521',\n",
       "  'Name': '600',\n",
       "  'Score': '',\n",
       "  'Gene': '600',\n",
       "  'Region': '10:110927563-110999521',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '111535577',\n",
       "  'End': '111623055',\n",
       "  'Name': '601',\n",
       "  'Score': '',\n",
       "  'Gene': '601',\n",
       "  'Region': '10:111535577-111623055',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '114488817',\n",
       "  'End': '114618882',\n",
       "  'Name': '602',\n",
       "  'Score': '',\n",
       "  'Gene': '602',\n",
       "  'Region': '10:114488817-114618882',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '115370947',\n",
       "  'End': '115393646',\n",
       "  'Name': '603',\n",
       "  'Score': '',\n",
       "  'Gene': '603',\n",
       "  'Region': '10:115370947-115393646',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '117000815',\n",
       "  'End': '117067224',\n",
       "  'Name': '604',\n",
       "  'Score': '',\n",
       "  'Gene': '604',\n",
       "  'Region': '10:117000815-117067224',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '117669609',\n",
       "  'End': '117695189',\n",
       "  'Name': '605',\n",
       "  'Score': '',\n",
       "  'Gene': '605',\n",
       "  'Region': '10:117669609-117695189',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '118743724',\n",
       "  'End': '118805013',\n",
       "  'Name': '606',\n",
       "  'Score': '',\n",
       "  'Gene': '606',\n",
       "  'Region': '10:118743724-118805013',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '123335185',\n",
       "  'End': '123378689',\n",
       "  'Name': '607',\n",
       "  'Score': '',\n",
       "  'Gene': '607',\n",
       "  'Region': '10:123335185-123378689',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '125258103',\n",
       "  'End': '125389970',\n",
       "  'Name': '608',\n",
       "  'Score': '',\n",
       "  'Gene': '608',\n",
       "  'Region': '10:125258103-125389970',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '126457909',\n",
       "  'End': '126541128',\n",
       "  'Name': '609',\n",
       "  'Score': '',\n",
       "  'Gene': '609',\n",
       "  'Region': '10:126457909-126541128',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr10',\n",
       "  'Start': '128417759',\n",
       "  'End': '128466427',\n",
       "  'Name': '610',\n",
       "  'Score': '',\n",
       "  'Gene': '610',\n",
       "  'Region': '10:128417759-128466427',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '4954021',\n",
       "  'End': '4996201',\n",
       "  'Name': '611',\n",
       "  'Score': '',\n",
       "  'Gene': '611',\n",
       "  'Region': '11:4954021-4996201',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '5346596',\n",
       "  'End': '5437936',\n",
       "  'Name': '612',\n",
       "  'Score': '',\n",
       "  'Gene': '612',\n",
       "  'Region': '11:5346596-5437936',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '6568052',\n",
       "  'End': '6600060',\n",
       "  'Name': '613',\n",
       "  'Score': '',\n",
       "  'Gene': '613',\n",
       "  'Region': '11:6568052-6600060',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '11170943',\n",
       "  'End': '11243518',\n",
       "  'Name': '614',\n",
       "  'Score': '',\n",
       "  'Gene': '614',\n",
       "  'Region': '11:11170943-11243518',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '20435884',\n",
       "  'End': '20500944',\n",
       "  'Name': '615',\n",
       "  'Score': '',\n",
       "  'Gene': '615',\n",
       "  'Region': '11:20435884-20500944',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '21993178',\n",
       "  'End': '22031941',\n",
       "  'Name': '616',\n",
       "  'Score': '',\n",
       "  'Gene': '616',\n",
       "  'Region': '11:21993178-22031941',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '22631290',\n",
       "  'End': '22677776',\n",
       "  'Name': '617',\n",
       "  'Score': '',\n",
       "  'Gene': '617',\n",
       "  'Region': '11:22631290-22677776',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '25848906',\n",
       "  'End': '25903888',\n",
       "  'Name': '618',\n",
       "  'Score': '',\n",
       "  'Gene': '618',\n",
       "  'Region': '11:25848906-25903888',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '26060799',\n",
       "  'End': '26129369',\n",
       "  'Name': '619',\n",
       "  'Score': '',\n",
       "  'Gene': '619',\n",
       "  'Region': '11:26060799-26129369',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '26956039',\n",
       "  'End': '26997404',\n",
       "  'Name': '620',\n",
       "  'Score': '',\n",
       "  'Gene': '620',\n",
       "  'Region': '11:26956039-26997404',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '28655032',\n",
       "  'End': '28701785',\n",
       "  'Name': '621',\n",
       "  'Score': '',\n",
       "  'Gene': '621',\n",
       "  'Region': '11:28655032-28701785',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '34054208',\n",
       "  'End': '34099026',\n",
       "  'Name': '622',\n",
       "  'Score': '',\n",
       "  'Gene': '622',\n",
       "  'Region': '11:34054208-34099026',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '35838462',\n",
       "  'End': '35927732',\n",
       "  'Name': '623',\n",
       "  'Score': '',\n",
       "  'Gene': '623',\n",
       "  'Region': '11:35838462-35927732',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '36140151',\n",
       "  'End': '36204786',\n",
       "  'Name': '624',\n",
       "  'Score': '',\n",
       "  'Gene': '624',\n",
       "  'Region': '11:36140151-36204786',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '38059705',\n",
       "  'End': '38092695',\n",
       "  'Name': '625',\n",
       "  'Score': '',\n",
       "  'Gene': '625',\n",
       "  'Region': '11:38059705-38092695',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '40764297',\n",
       "  'End': '40787367',\n",
       "  'Name': '626',\n",
       "  'Score': '',\n",
       "  'Gene': '626',\n",
       "  'Region': '11:40764297-40787367',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '43348561',\n",
       "  'End': '43413182',\n",
       "  'Name': '627',\n",
       "  'Score': '',\n",
       "  'Gene': '627',\n",
       "  'Region': '11:43348561-43413182',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '43766737',\n",
       "  'End': '43828613',\n",
       "  'Name': '628',\n",
       "  'Score': '',\n",
       "  'Gene': '628',\n",
       "  'Region': '11:43766737-43828613',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '44503963',\n",
       "  'End': '44586487',\n",
       "  'Name': '629',\n",
       "  'Score': '',\n",
       "  'Gene': '629',\n",
       "  'Region': '11:44503963-44586487',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '46430972',\n",
       "  'End': '46462278',\n",
       "  'Name': '630',\n",
       "  'Score': '',\n",
       "  'Gene': '630',\n",
       "  'Region': '11:46430972-46462278',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '50669370',\n",
       "  'End': '50707733',\n",
       "  'Name': '631',\n",
       "  'Score': '',\n",
       "  'Gene': '631',\n",
       "  'Region': '11:50669370-50707733',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '52634034',\n",
       "  'End': '52697755',\n",
       "  'Name': '632',\n",
       "  'Score': '',\n",
       "  'Gene': '632',\n",
       "  'Region': '11:52634034-52697755',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '54311823',\n",
       "  'End': '54360331',\n",
       "  'Name': '633',\n",
       "  'Score': '',\n",
       "  'Gene': '633',\n",
       "  'Region': '11:54311823-54360331',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '55140329',\n",
       "  'End': '55195754',\n",
       "  'Name': '634',\n",
       "  'Score': '',\n",
       "  'Gene': '634',\n",
       "  'Region': '11:55140329-55195754',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '55386303',\n",
       "  'End': '55453852',\n",
       "  'Name': '635',\n",
       "  'Score': '',\n",
       "  'Gene': '635',\n",
       "  'Region': '11:55386303-55453852',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '57667256',\n",
       "  'End': '57760935',\n",
       "  'Name': '636',\n",
       "  'Score': '',\n",
       "  'Gene': '636',\n",
       "  'Region': '11:57667256-57760935',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '58017673',\n",
       "  'End': '58071281',\n",
       "  'Name': '637',\n",
       "  'Score': '',\n",
       "  'Gene': '637',\n",
       "  'Region': '11:58017673-58071281',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '58301211',\n",
       "  'End': '58317455',\n",
       "  'Name': '638',\n",
       "  'Score': '',\n",
       "  'Gene': '638',\n",
       "  'Region': '11:58301211-58317455',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '62294006',\n",
       "  'End': '62342698',\n",
       "  'Name': '639',\n",
       "  'Score': '',\n",
       "  'Gene': '639',\n",
       "  'Region': '11:62294006-62342698',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '63891715',\n",
       "  'End': '63925692',\n",
       "  'Name': '640',\n",
       "  'Score': '',\n",
       "  'Gene': '640',\n",
       "  'Region': '11:63891715-63925692',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '67990926',\n",
       "  'End': '68044725',\n",
       "  'Name': '641',\n",
       "  'Score': '',\n",
       "  'Gene': '641',\n",
       "  'Region': '11:67990926-68044725',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '69591649',\n",
       "  'End': '69614548',\n",
       "  'Name': '642',\n",
       "  'Score': '',\n",
       "  'Gene': '642',\n",
       "  'Region': '11:69591649-69614548',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '70020422',\n",
       "  'End': '70050516',\n",
       "  'Name': '643',\n",
       "  'Score': '',\n",
       "  'Gene': '643',\n",
       "  'Region': '11:70020422-70050516',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '70566947',\n",
       "  'End': '70608824',\n",
       "  'Name': '644',\n",
       "  'Score': '',\n",
       "  'Gene': '644',\n",
       "  'Region': '11:70566947-70608824',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '70870197',\n",
       "  'End': '70945859',\n",
       "  'Name': '645',\n",
       "  'Score': '',\n",
       "  'Gene': '645',\n",
       "  'Region': '11:70870197-70945859',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '71758129',\n",
       "  'End': '71792987',\n",
       "  'Name': '646',\n",
       "  'Score': '',\n",
       "  'Gene': '646',\n",
       "  'Region': '11:71758129-71792987',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '72446391',\n",
       "  'End': '72483071',\n",
       "  'Name': '647',\n",
       "  'Score': '',\n",
       "  'Gene': '647',\n",
       "  'Region': '11:72446391-72483071',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '73281570',\n",
       "  'End': '73315274',\n",
       "  'Name': '648',\n",
       "  'Score': '',\n",
       "  'Gene': '648',\n",
       "  'Region': '11:73281570-73315274',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '78108903',\n",
       "  'End': '78170841',\n",
       "  'Name': '649',\n",
       "  'Score': '',\n",
       "  'Gene': '649',\n",
       "  'Region': '11:78108903-78170841',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '78271740',\n",
       "  'End': '78314241',\n",
       "  'Name': '650',\n",
       "  'Score': '',\n",
       "  'Gene': '650',\n",
       "  'Region': '11:78271740-78314241',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '84032044',\n",
       "  'End': '84074532',\n",
       "  'Name': '651',\n",
       "  'Score': '',\n",
       "  'Gene': '651',\n",
       "  'Region': '11:84032044-84074532',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '88820839',\n",
       "  'End': '88858107',\n",
       "  'Name': '652',\n",
       "  'Score': '',\n",
       "  'Gene': '652',\n",
       "  'Region': '11:88820839-88858107',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '89596560',\n",
       "  'End': '89645737',\n",
       "  'Name': '653',\n",
       "  'Score': '',\n",
       "  'Gene': '653',\n",
       "  'Region': '11:89596560-89645737',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '95675026',\n",
       "  'End': '95705325',\n",
       "  'Name': '654',\n",
       "  'Score': '',\n",
       "  'Gene': '654',\n",
       "  'Region': '11:95675026-95705325',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '97119256',\n",
       "  'End': '97161020',\n",
       "  'Name': '655',\n",
       "  'Score': '',\n",
       "  'Gene': '655',\n",
       "  'Region': '11:97119256-97161020',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '100719943',\n",
       "  'End': '100754691',\n",
       "  'Name': '656',\n",
       "  'Score': '',\n",
       "  'Gene': '656',\n",
       "  'Region': '11:100719943-100754691',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '102009867',\n",
       "  'End': '102031606',\n",
       "  'Name': '657',\n",
       "  'Score': '',\n",
       "  'Gene': '657',\n",
       "  'Region': '11:102009867-102031606',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '106795701',\n",
       "  'End': '106832613',\n",
       "  'Name': '658',\n",
       "  'Score': '',\n",
       "  'Gene': '658',\n",
       "  'Region': '11:106795701-106832613',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '107025657',\n",
       "  'End': '107047289',\n",
       "  'Name': '659',\n",
       "  'Score': '',\n",
       "  'Gene': '659',\n",
       "  'Region': '11:107025657-107047289',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '109467924',\n",
       "  'End': '109521070',\n",
       "  'Name': '660',\n",
       "  'Score': '',\n",
       "  'Gene': '660',\n",
       "  'Region': '11:109467924-109521070',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '109686821',\n",
       "  'End': '109728725',\n",
       "  'Name': '661',\n",
       "  'Score': '',\n",
       "  'Gene': '661',\n",
       "  'Region': '11:109686821-109728725',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '111511725',\n",
       "  'End': '111538093',\n",
       "  'Name': '662',\n",
       "  'Score': '',\n",
       "  'Gene': '662',\n",
       "  'Region': '11:111511725-111538093',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '112798498',\n",
       "  'End': '112840927',\n",
       "  'Name': '663',\n",
       "  'Score': '',\n",
       "  'Gene': '663',\n",
       "  'Region': '11:112798498-112840927',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '113661313',\n",
       "  'End': '113704386',\n",
       "  'Name': '664',\n",
       "  'Score': '',\n",
       "  'Gene': '664',\n",
       "  'Region': '11:113661313-113704386',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '115391470',\n",
       "  'End': '115414234',\n",
       "  'Name': '665',\n",
       "  'Score': '',\n",
       "  'Gene': '665',\n",
       "  'Region': '11:115391470-115414234',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '120979342',\n",
       "  'End': '121024517',\n",
       "  'Name': '666',\n",
       "  'Score': '',\n",
       "  'Gene': '666',\n",
       "  'Region': '11:120979342-121024517',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr11',\n",
       "  'Start': '121475660',\n",
       "  'End': '121550309',\n",
       "  'Name': '667',\n",
       "  'Score': '',\n",
       "  'Gene': '667',\n",
       "  'Region': '11:121475660-121550309',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '4679994',\n",
       "  'End': '4757519',\n",
       "  'Name': '668',\n",
       "  'Score': '',\n",
       "  'Gene': '668',\n",
       "  'Region': '12:4679994-4757519',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '8659279',\n",
       "  'End': '8699465',\n",
       "  'Name': '669',\n",
       "  'Score': '',\n",
       "  'Gene': '669',\n",
       "  'Region': '12:8659279-8699465',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '8916293',\n",
       "  'End': '8936352',\n",
       "  'Name': '670',\n",
       "  'Score': '',\n",
       "  'Gene': '670',\n",
       "  'Region': '12:8916293-8936352',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '11322932',\n",
       "  'End': '11404073',\n",
       "  'Name': '671',\n",
       "  'Score': '',\n",
       "  'Gene': '671',\n",
       "  'Region': '12:11322932-11404073',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '12765678',\n",
       "  'End': '12828753',\n",
       "  'Name': '672',\n",
       "  'Score': '',\n",
       "  'Gene': '672',\n",
       "  'Region': '12:12765678-12828753',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '13079761',\n",
       "  'End': '13145339',\n",
       "  'Name': '673',\n",
       "  'Score': '',\n",
       "  'Gene': '673',\n",
       "  'Region': '12:13079761-13145339',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '21424404',\n",
       "  'End': '21457592',\n",
       "  'Name': '674',\n",
       "  'Score': '',\n",
       "  'Gene': '674',\n",
       "  'Region': '12:21424404-21457592',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '29537911',\n",
       "  'End': '29572105',\n",
       "  'Name': '675',\n",
       "  'Score': '',\n",
       "  'Gene': '675',\n",
       "  'Region': '12:29537911-29572105',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '30227718',\n",
       "  'End': '30270914',\n",
       "  'Name': '676',\n",
       "  'Score': '',\n",
       "  'Gene': '676',\n",
       "  'Region': '12:30227718-30270914',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '34430416',\n",
       "  'End': '34517401',\n",
       "  'Name': '677',\n",
       "  'Score': '',\n",
       "  'Gene': '677',\n",
       "  'Region': '12:34430416-34517401',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '35003090',\n",
       "  'End': '35040003',\n",
       "  'Name': '678',\n",
       "  'Score': '',\n",
       "  'Gene': '678',\n",
       "  'Region': '12:35003090-35040003',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '45014756',\n",
       "  'End': '45083297',\n",
       "  'Name': '679',\n",
       "  'Score': '',\n",
       "  'Gene': '679',\n",
       "  'Region': '12:45014756-45083297',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '49369077',\n",
       "  'End': '49411839',\n",
       "  'Name': '680',\n",
       "  'Score': '',\n",
       "  'Gene': '680',\n",
       "  'Region': '12:49369077-49411839',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '51699811',\n",
       "  'End': '51751995',\n",
       "  'Name': '681',\n",
       "  'Score': '',\n",
       "  'Gene': '681',\n",
       "  'Region': '12:51699811-51751995',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '52741783',\n",
       "  'End': '52796084',\n",
       "  'Name': '682',\n",
       "  'Score': '',\n",
       "  'Gene': '682',\n",
       "  'Region': '12:52741783-52796084',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '67094035',\n",
       "  'End': '67198980',\n",
       "  'Name': '683',\n",
       "  'Score': '',\n",
       "  'Gene': '683',\n",
       "  'Region': '12:67094035-67198980',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '69799098',\n",
       "  'End': '69881374',\n",
       "  'Name': '684',\n",
       "  'Score': '',\n",
       "  'Gene': '684',\n",
       "  'Region': '12:69799098-69881374',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '70867906',\n",
       "  'End': '70914672',\n",
       "  'Name': '685',\n",
       "  'Score': '',\n",
       "  'Gene': '685',\n",
       "  'Region': '12:70867906-70914672',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '72179666',\n",
       "  'End': '72239828',\n",
       "  'Name': '686',\n",
       "  'Score': '',\n",
       "  'Gene': '686',\n",
       "  'Region': '12:72179666-72239828',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '72179666',\n",
       "  'End': '72239828',\n",
       "  'Name': '687',\n",
       "  'Score': '',\n",
       "  'Gene': '687',\n",
       "  'Region': '12:72179666-72239828',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '73593311',\n",
       "  'End': '73634095',\n",
       "  'Name': '688',\n",
       "  'Score': '',\n",
       "  'Gene': '688',\n",
       "  'Region': '12:73593311-73634095',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '75497290',\n",
       "  'End': '75572550',\n",
       "  'Name': '689',\n",
       "  'Score': '',\n",
       "  'Gene': '689',\n",
       "  'Region': '12:75497290-75572550',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '83534044',\n",
       "  'End': '83579479',\n",
       "  'Name': '690',\n",
       "  'Score': '',\n",
       "  'Gene': '690',\n",
       "  'Region': '12:83534044-83579479',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '84161919',\n",
       "  'End': '84188789',\n",
       "  'Name': '691',\n",
       "  'Score': '',\n",
       "  'Gene': '691',\n",
       "  'Region': '12:84161919-84188789',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '84511230',\n",
       "  'End': '84546432',\n",
       "  'Name': '692',\n",
       "  'Score': '',\n",
       "  'Gene': '692',\n",
       "  'Region': '12:84511230-84546432',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '85157576',\n",
       "  'End': '85208677',\n",
       "  'Name': '693',\n",
       "  'Score': '',\n",
       "  'Gene': '693',\n",
       "  'Region': '12:85157576-85208677',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '86682397',\n",
       "  'End': '86715863',\n",
       "  'Name': '694',\n",
       "  'Score': '',\n",
       "  'Gene': '694',\n",
       "  'Region': '12:86682397-86715863',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '88408166',\n",
       "  'End': '88443232',\n",
       "  'Name': '695',\n",
       "  'Score': '',\n",
       "  'Gene': '695',\n",
       "  'Region': '12:88408166-88443232',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '89845982',\n",
       "  'End': '89888938',\n",
       "  'Name': '696',\n",
       "  'Score': '',\n",
       "  'Gene': '696',\n",
       "  'Region': '12:89845982-89888938',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '90074487',\n",
       "  'End': '90174183',\n",
       "  'Name': '697',\n",
       "  'Score': '',\n",
       "  'Gene': '697',\n",
       "  'Region': '12:90074487-90174183',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '103962927',\n",
       "  'End': '104000067',\n",
       "  'Name': '698',\n",
       "  'Score': '',\n",
       "  'Gene': '698',\n",
       "  'Region': '12:103962927-104000067',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '110203166',\n",
       "  'End': '110271004',\n",
       "  'Name': '699',\n",
       "  'Score': '',\n",
       "  'Gene': '699',\n",
       "  'Region': '12:110203166-110271004',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '110498383',\n",
       "  'End': '110586477',\n",
       "  'Name': '700',\n",
       "  'Score': '',\n",
       "  'Gene': '700',\n",
       "  'Region': '12:110498383-110586477',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '112283397',\n",
       "  'End': '112361328',\n",
       "  'Name': '701',\n",
       "  'Score': '',\n",
       "  'Gene': '701',\n",
       "  'Region': '12:112283397-112361328',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '113275834',\n",
       "  'End': '113322430',\n",
       "  'Name': '702',\n",
       "  'Score': '',\n",
       "  'Gene': '702',\n",
       "  'Region': '12:113275834-113322430',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr12',\n",
       "  'Start': '118836228',\n",
       "  'End': '118856588',\n",
       "  'Name': '703',\n",
       "  'Score': '',\n",
       "  'Gene': '703',\n",
       "  'Region': '12:118836228-118856588',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '6155921',\n",
       "  'End': '6192393',\n",
       "  'Name': '704',\n",
       "  'Score': '',\n",
       "  'Gene': '704',\n",
       "  'Region': '13:6155921-6192393',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '9653963',\n",
       "  'End': '9708881',\n",
       "  'Name': '705',\n",
       "  'Score': '',\n",
       "  'Gene': '705',\n",
       "  'Region': '13:9653963-9708881',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '13794267',\n",
       "  'End': '13827202',\n",
       "  'Name': '706',\n",
       "  'Score': '',\n",
       "  'Gene': '706',\n",
       "  'Region': '13:13794267-13827202',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '15521949',\n",
       "  'End': '15603228',\n",
       "  'Name': '707',\n",
       "  'Score': '',\n",
       "  'Gene': '707',\n",
       "  'Region': '13:15521949-15603228',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '15979446',\n",
       "  'End': '16024031',\n",
       "  'Name': '708',\n",
       "  'Score': '',\n",
       "  'Gene': '708',\n",
       "  'Region': '13:15979446-16024031',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '18306121',\n",
       "  'End': '18372753',\n",
       "  'Name': '709',\n",
       "  'Score': '',\n",
       "  'Gene': '709',\n",
       "  'Region': '13:18306121-18372753',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '18800540',\n",
       "  'End': '18876160',\n",
       "  'Name': '710',\n",
       "  'Score': '',\n",
       "  'Gene': '710',\n",
       "  'Region': '13:18800540-18876160',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '23700967',\n",
       "  'End': '23729842',\n",
       "  'Name': '711',\n",
       "  'Score': '',\n",
       "  'Gene': '711',\n",
       "  'Region': '13:23700967-23729842',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '28592966',\n",
       "  'End': '28630622',\n",
       "  'Name': '712',\n",
       "  'Score': '',\n",
       "  'Gene': '712',\n",
       "  'Region': '13:28592966-28630622',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '32788496',\n",
       "  'End': '32834925',\n",
       "  'Name': '713',\n",
       "  'Score': '',\n",
       "  'Gene': '713',\n",
       "  'Region': '13:32788496-32834925',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '36257139',\n",
       "  'End': '36315532',\n",
       "  'Name': '714',\n",
       "  'Score': '',\n",
       "  'Gene': '714',\n",
       "  'Region': '13:36257139-36315532',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '37820544',\n",
       "  'End': '37972052',\n",
       "  'Name': '715',\n",
       "  'Score': '',\n",
       "  'Gene': '715',\n",
       "  'Region': '13:37820544-37972052',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '40921149',\n",
       "  'End': '40979375',\n",
       "  'Name': '716',\n",
       "  'Score': '',\n",
       "  'Gene': '716',\n",
       "  'Region': '13:40921149-40979375',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '41302633',\n",
       "  'End': '41352801',\n",
       "  'Name': '717',\n",
       "  'Score': '',\n",
       "  'Gene': '717',\n",
       "  'Region': '13:41302633-41352801',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '43739520',\n",
       "  'End': '43806764',\n",
       "  'Name': '718',\n",
       "  'Score': '',\n",
       "  'Gene': '718',\n",
       "  'Region': '13:43739520-43806764',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '44018093',\n",
       "  'End': '44101293',\n",
       "  'Name': '719',\n",
       "  'Score': '',\n",
       "  'Gene': '719',\n",
       "  'Region': '13:44018093-44101293',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '44208135',\n",
       "  'End': '44238787',\n",
       "  'Name': '720',\n",
       "  'Score': '',\n",
       "  'Gene': '720',\n",
       "  'Region': '13:44208135-44238787',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '44469166',\n",
       "  'End': '44538478',\n",
       "  'Name': '721',\n",
       "  'Score': '',\n",
       "  'Gene': '721',\n",
       "  'Region': '13:44469166-44538478',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '45074772',\n",
       "  'End': '45159265',\n",
       "  'Name': '722',\n",
       "  'Score': '',\n",
       "  'Gene': '722',\n",
       "  'Region': '13:45074772-45159265',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '46811717',\n",
       "  'End': '46896339',\n",
       "  'Name': '723',\n",
       "  'Score': '',\n",
       "  'Gene': '723',\n",
       "  'Region': '13:46811717-46896339',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '52014806',\n",
       "  'End': '52055454',\n",
       "  'Name': '724',\n",
       "  'Score': '',\n",
       "  'Gene': '724',\n",
       "  'Region': '13:52014806-52055454',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '57246124',\n",
       "  'End': '57292330',\n",
       "  'Name': '725',\n",
       "  'Score': '',\n",
       "  'Gene': '725',\n",
       "  'Region': '13:57246124-57292330',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '59798227',\n",
       "  'End': '59843221',\n",
       "  'Name': '726',\n",
       "  'Score': '',\n",
       "  'Gene': '726',\n",
       "  'Region': '13:59798227-59843221',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '60044633',\n",
       "  'End': '60133990',\n",
       "  'Name': '727',\n",
       "  'Score': '',\n",
       "  'Gene': '727',\n",
       "  'Region': '13:60044633-60133990',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '60444888',\n",
       "  'End': '60498636',\n",
       "  'Name': '728',\n",
       "  'Score': '',\n",
       "  'Gene': '728',\n",
       "  'Region': '13:60444888-60498636',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '62897747',\n",
       "  'End': '62927194',\n",
       "  'Name': '729',\n",
       "  'Score': '',\n",
       "  'Gene': '729',\n",
       "  'Region': '13:62897747-62927194',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '64437959',\n",
       "  'End': '64486783',\n",
       "  'Name': '730',\n",
       "  'Score': '',\n",
       "  'Gene': '730',\n",
       "  'Region': '13:64437959-64486783',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '69322990',\n",
       "  'End': '69358200',\n",
       "  'Name': '731',\n",
       "  'Score': '',\n",
       "  'Gene': '731',\n",
       "  'Region': '13:69322990-69358200',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '70862179',\n",
       "  'End': '70904310',\n",
       "  'Name': '732',\n",
       "  'Score': '',\n",
       "  'Gene': '732',\n",
       "  'Region': '13:70862179-70904310',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '75723653',\n",
       "  'End': '75810309',\n",
       "  'Name': '733',\n",
       "  'Score': '',\n",
       "  'Gene': '733',\n",
       "  'Region': '13:75723653-75810309',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '81133389',\n",
       "  'End': '81160001',\n",
       "  'Name': '734',\n",
       "  'Score': '',\n",
       "  'Gene': '734',\n",
       "  'Region': '13:81133389-81160001',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '83099701',\n",
       "  'End': '83136173',\n",
       "  'Name': '735',\n",
       "  'Score': '',\n",
       "  'Gene': '735',\n",
       "  'Region': '13:83099701-83136173',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '84021657',\n",
       "  'End': '84065757',\n",
       "  'Name': '736',\n",
       "  'Score': '',\n",
       "  'Gene': '736',\n",
       "  'Region': '13:84021657-84065757',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '88899893',\n",
       "  'End': '89015343',\n",
       "  'Name': '737',\n",
       "  'Score': '',\n",
       "  'Gene': '737',\n",
       "  'Region': '13:88899893-89015343',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '89583484',\n",
       "  'End': '89608121',\n",
       "  'Name': '738',\n",
       "  'Score': '',\n",
       "  'Gene': '738',\n",
       "  'Region': '13:89583484-89608121',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '97161046',\n",
       "  'End': '97220203',\n",
       "  'Name': '739',\n",
       "  'Score': '',\n",
       "  'Gene': '739',\n",
       "  'Region': '13:97161046-97220203',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '99352904',\n",
       "  'End': '99397764',\n",
       "  'Name': '740',\n",
       "  'Score': '',\n",
       "  'Gene': '740',\n",
       "  'Region': '13:99352904-99397764',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '101208612',\n",
       "  'End': '101239406',\n",
       "  'Name': '741',\n",
       "  'Score': '',\n",
       "  'Gene': '741',\n",
       "  'Region': '13:101208612-101239406',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '101450340',\n",
       "  'End': '101513506',\n",
       "  'Name': '742',\n",
       "  'Score': '',\n",
       "  'Gene': '742',\n",
       "  'Region': '13:101450340-101513506',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '103262792',\n",
       "  'End': '103327542',\n",
       "  'Name': '743',\n",
       "  'Score': '',\n",
       "  'Gene': '743',\n",
       "  'Region': '13:103262792-103327542',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '106688768',\n",
       "  'End': '106747437',\n",
       "  'Name': '744',\n",
       "  'Score': '',\n",
       "  'Gene': '744',\n",
       "  'Region': '13:106688768-106747437',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '108056902',\n",
       "  'End': '108136316',\n",
       "  'Name': '745',\n",
       "  'Score': '',\n",
       "  'Gene': '745',\n",
       "  'Region': '13:108056902-108136316',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '108264869',\n",
       "  'End': '108302090',\n",
       "  'Name': '746',\n",
       "  'Score': '',\n",
       "  'Gene': '746',\n",
       "  'Region': '13:108264869-108302090',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '111383959',\n",
       "  'End': '111422442',\n",
       "  'Name': '747',\n",
       "  'Score': '',\n",
       "  'Gene': '747',\n",
       "  'Region': '13:111383959-111422442',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '111741847',\n",
       "  'End': '111797966',\n",
       "  'Name': '748',\n",
       "  'Score': '',\n",
       "  'Gene': '748',\n",
       "  'Region': '13:111741847-111797966',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '111929531',\n",
       "  'End': '111980923',\n",
       "  'Name': '749',\n",
       "  'Score': '',\n",
       "  'Gene': '749',\n",
       "  'Region': '13:111929531-111980923',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '114075922',\n",
       "  'End': '114139764',\n",
       "  'Name': '750',\n",
       "  'Score': '',\n",
       "  'Gene': '750',\n",
       "  'Region': '13:114075922-114139764',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '114075922',\n",
       "  'End': '114139495',\n",
       "  'Name': '751',\n",
       "  'Score': '',\n",
       "  'Gene': '751',\n",
       "  'Region': '13:114075922-114139495',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr13',\n",
       "  'Start': '119596445',\n",
       "  'End': '119614951',\n",
       "  'Name': '752',\n",
       "  'Score': '',\n",
       "  'Gene': '752',\n",
       "  'Region': '13:119596445-119614951',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '3308996',\n",
       "  'End': '3358904',\n",
       "  'Name': '753',\n",
       "  'Score': '',\n",
       "  'Gene': '753',\n",
       "  'Region': '14:3308996-3358904',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '3538167',\n",
       "  'End': '3573135',\n",
       "  'Name': '754',\n",
       "  'Score': '',\n",
       "  'Gene': '754',\n",
       "  'Region': '14:3538167-3573135',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '5459405',\n",
       "  'End': '5502217',\n",
       "  'Name': '755',\n",
       "  'Score': '',\n",
       "  'Gene': '755',\n",
       "  'Region': '14:5459405-5502217',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '5960658',\n",
       "  'End': '6023064',\n",
       "  'Name': '756',\n",
       "  'Score': '',\n",
       "  'Gene': '756',\n",
       "  'Region': '14:5960658-6023064',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '7294561',\n",
       "  'End': '7325660',\n",
       "  'Name': '757',\n",
       "  'Score': '',\n",
       "  'Gene': '757',\n",
       "  'Region': '14:7294561-7325660',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '13333083',\n",
       "  'End': '13368913',\n",
       "  'Name': '758',\n",
       "  'Score': '',\n",
       "  'Gene': '758',\n",
       "  'Region': '14:13333083-13368913',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '14053804',\n",
       "  'End': '14098999',\n",
       "  'Name': '759',\n",
       "  'Score': '',\n",
       "  'Gene': '759',\n",
       "  'Region': '14:14053804-14098999',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '15355670',\n",
       "  'End': '15426164',\n",
       "  'Name': '760',\n",
       "  'Score': '',\n",
       "  'Gene': '760',\n",
       "  'Region': '14:15355670-15426164',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '21343204',\n",
       "  'End': '21402833',\n",
       "  'Name': '761',\n",
       "  'Score': '',\n",
       "  'Gene': '761',\n",
       "  'Region': '14:21343204-21402833',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '21969106',\n",
       "  'End': '21994415',\n",
       "  'Name': '762',\n",
       "  'Score': '',\n",
       "  'Gene': '762',\n",
       "  'Region': '14:21969106-21994415',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '26429598',\n",
       "  'End': '26526458',\n",
       "  'Name': '763',\n",
       "  'Score': '',\n",
       "  'Gene': '763',\n",
       "  'Region': '14:26429598-26526458',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '29028427',\n",
       "  'End': '29114966',\n",
       "  'Name': '764',\n",
       "  'Score': '',\n",
       "  'Gene': '764',\n",
       "  'Region': '14:29028427-29114966',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '29323773',\n",
       "  'End': '29389124',\n",
       "  'Name': '765',\n",
       "  'Score': '',\n",
       "  'Gene': '765',\n",
       "  'Region': '14:29323773-29389124',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '31137141',\n",
       "  'End': '31196382',\n",
       "  'Name': '766',\n",
       "  'Score': '',\n",
       "  'Gene': '766',\n",
       "  'Region': '14:31137141-31196382',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '32002648',\n",
       "  'End': '32036083',\n",
       "  'Name': '767',\n",
       "  'Score': '',\n",
       "  'Gene': '767',\n",
       "  'Region': '14:32002648-32036083',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '32527175',\n",
       "  'End': '32610238',\n",
       "  'Name': '768',\n",
       "  'Score': '',\n",
       "  'Gene': '768',\n",
       "  'Region': '14:32527175-32610238',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '34188487',\n",
       "  'End': '34235598',\n",
       "  'Name': '769',\n",
       "  'Score': '',\n",
       "  'Gene': '769',\n",
       "  'Region': '14:34188487-34235598',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '40901564',\n",
       "  'End': '40959316',\n",
       "  'Name': '770',\n",
       "  'Score': '',\n",
       "  'Gene': '770',\n",
       "  'Region': '14:40901564-40959316',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '43240240',\n",
       "  'End': '43351198',\n",
       "  'Name': '771',\n",
       "  'Score': '',\n",
       "  'Gene': '771',\n",
       "  'Region': '14:43240240-43351198',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '45539453',\n",
       "  'End': '45625426',\n",
       "  'Name': '772',\n",
       "  'Score': '',\n",
       "  'Gene': '772',\n",
       "  'Region': '14:45539453-45625426',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '46901206',\n",
       "  'End': '46990060',\n",
       "  'Name': '773',\n",
       "  'Score': '',\n",
       "  'Gene': '773',\n",
       "  'Region': '14:46901206-46990060',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '48155216',\n",
       "  'End': '48224948',\n",
       "  'Name': '774',\n",
       "  'Score': '',\n",
       "  'Gene': '774',\n",
       "  'Region': '14:48155216-48224948',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '55497236',\n",
       "  'End': '55528412',\n",
       "  'Name': '775',\n",
       "  'Score': '',\n",
       "  'Gene': '775',\n",
       "  'Region': '14:55497236-55528412',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '57508975',\n",
       "  'End': '57565644',\n",
       "  'Name': '776',\n",
       "  'Score': '',\n",
       "  'Gene': '776',\n",
       "  'Region': '14:57508975-57565644',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '57508975',\n",
       "  'End': '57559853',\n",
       "  'Name': '777',\n",
       "  'Score': '',\n",
       "  'Gene': '777',\n",
       "  'Region': '14:57508975-57559853',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '57679680',\n",
       "  'End': '57737633',\n",
       "  'Name': '778',\n",
       "  'Score': '',\n",
       "  'Gene': '778',\n",
       "  'Region': '14:57679680-57737633',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '57901541',\n",
       "  'End': '57922428',\n",
       "  'Name': '779',\n",
       "  'Score': '',\n",
       "  'Gene': '779',\n",
       "  'Region': '14:57901541-57922428',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '58074999',\n",
       "  'End': '58113602',\n",
       "  'Name': '780',\n",
       "  'Score': '',\n",
       "  'Gene': '780',\n",
       "  'Region': '14:58074999-58113602',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '59206105',\n",
       "  'End': '59251023',\n",
       "  'Name': '781',\n",
       "  'Score': '',\n",
       "  'Gene': '781',\n",
       "  'Region': '14:59206105-59251023',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '59701594',\n",
       "  'End': '59738067',\n",
       "  'Name': '782',\n",
       "  'Score': '',\n",
       "  'Gene': '782',\n",
       "  'Region': '14:59701594-59738067',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '61638986',\n",
       "  'End': '61673707',\n",
       "  'Name': '783',\n",
       "  'Score': '',\n",
       "  'Gene': '783',\n",
       "  'Region': '14:61638986-61673707',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '61803570',\n",
       "  'End': '61836789',\n",
       "  'Name': '784',\n",
       "  'Score': '',\n",
       "  'Gene': '784',\n",
       "  'Region': '14:61803570-61836789',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '62173770',\n",
       "  'End': '62236386',\n",
       "  'Name': '785',\n",
       "  'Score': '',\n",
       "  'Gene': '785',\n",
       "  'Region': '14:62173770-62236386',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '62844246',\n",
       "  'End': '62940869',\n",
       "  'Name': '786',\n",
       "  'Score': '',\n",
       "  'Gene': '786',\n",
       "  'Region': '14:62844246-62940869',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '63128357',\n",
       "  'End': '63165661',\n",
       "  'Name': '787',\n",
       "  'Score': '',\n",
       "  'Gene': '787',\n",
       "  'Region': '14:63128357-63165661',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '65514499',\n",
       "  'End': '65584282',\n",
       "  'Name': '788',\n",
       "  'Score': '',\n",
       "  'Gene': '788',\n",
       "  'Region': '14:65514499-65584282',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '65970189',\n",
       "  'End': '65988537',\n",
       "  'Name': '789',\n",
       "  'Score': '',\n",
       "  'Gene': '789',\n",
       "  'Region': '14:65970189-65988537',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '67888001',\n",
       "  'End': '67922629',\n",
       "  'Name': '790',\n",
       "  'Score': '',\n",
       "  'Gene': '790',\n",
       "  'Region': '14:67888001-67922629',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '69801321',\n",
       "  'End': '69861987',\n",
       "  'Name': '791',\n",
       "  'Score': '',\n",
       "  'Gene': '791',\n",
       "  'Region': '14:69801321-69861987',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '70302991',\n",
       "  'End': '70345449',\n",
       "  'Name': '792',\n",
       "  'Score': '',\n",
       "  'Gene': '792',\n",
       "  'Region': '14:70302991-70345449',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '75473070',\n",
       "  'End': '75543422',\n",
       "  'Name': '793',\n",
       "  'Score': '',\n",
       "  'Gene': '793',\n",
       "  'Region': '14:75473070-75543422',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '75870410',\n",
       "  'End': '75915995',\n",
       "  'Name': '794',\n",
       "  'Score': '',\n",
       "  'Gene': '794',\n",
       "  'Region': '14:75870410-75915995',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '76425881',\n",
       "  'End': '76480537',\n",
       "  'Name': '795',\n",
       "  'Score': '',\n",
       "  'Gene': '795',\n",
       "  'Region': '14:76425881-76480537',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '79488685',\n",
       "  'End': '79563260',\n",
       "  'Name': '796',\n",
       "  'Score': '',\n",
       "  'Gene': '796',\n",
       "  'Region': '14:79488685-79563260',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '79909686',\n",
       "  'End': '79974133',\n",
       "  'Name': '797',\n",
       "  'Score': '',\n",
       "  'Gene': '797',\n",
       "  'Region': '14:79909686-79974133',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '80237149',\n",
       "  'End': '80278631',\n",
       "  'Name': '798',\n",
       "  'Score': '',\n",
       "  'Gene': '798',\n",
       "  'Region': '14:80237149-80278631',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '99536543',\n",
       "  'End': '99592180',\n",
       "  'Name': '799',\n",
       "  'Score': '',\n",
       "  'Gene': '799',\n",
       "  'Region': '14:99536543-99592180',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '102784665',\n",
       "  'End': '102844185',\n",
       "  'Name': '800',\n",
       "  'Score': '',\n",
       "  'Gene': '800',\n",
       "  'Region': '14:102784665-102844185',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '103109997',\n",
       "  'End': '103163301',\n",
       "  'Name': '801',\n",
       "  'Score': '',\n",
       "  'Gene': '801',\n",
       "  'Region': '14:103109997-103163301',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '103424208',\n",
       "  'End': '103472720',\n",
       "  'Name': '802',\n",
       "  'Score': '',\n",
       "  'Gene': '802',\n",
       "  'Region': '14:103424208-103472720',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '105274551',\n",
       "  'End': '105334049',\n",
       "  'Name': '803',\n",
       "  'Score': '',\n",
       "  'Gene': '803',\n",
       "  'Region': '14:105274551-105334049',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '108791216',\n",
       "  'End': '108844003',\n",
       "  'Name': '804',\n",
       "  'Score': '',\n",
       "  'Gene': '804',\n",
       "  'Region': '14:108791216-108844003',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '114854433',\n",
       "  'End': '114913616',\n",
       "  'Name': '805',\n",
       "  'Score': '',\n",
       "  'Gene': '805',\n",
       "  'Region': '14:114854433-114913616',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '119203071',\n",
       "  'End': '119278918',\n",
       "  'Name': '806',\n",
       "  'Score': '',\n",
       "  'Gene': '806',\n",
       "  'Region': '14:119203071-119278918',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '119631024',\n",
       "  'End': '119687288',\n",
       "  'Name': '807',\n",
       "  'Score': '',\n",
       "  'Gene': '807',\n",
       "  'Region': '14:119631024-119687288',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '120789255',\n",
       "  'End': '120898010',\n",
       "  'Name': '808',\n",
       "  'Score': '',\n",
       "  'Gene': '808',\n",
       "  'Region': '14:120789255-120898010',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr14',\n",
       "  'Start': '121928265',\n",
       "  'End': '122080024',\n",
       "  'Name': '809',\n",
       "  'Score': '',\n",
       "  'Gene': '809',\n",
       "  'Region': '14:121928265-122080024',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '6519514',\n",
       "  'End': '6571190',\n",
       "  'Name': '810',\n",
       "  'Score': '',\n",
       "  'Gene': '810',\n",
       "  'Region': '15:6519514-6571190',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '7180479',\n",
       "  'End': '7226680',\n",
       "  'Name': '811',\n",
       "  'Score': '',\n",
       "  'Gene': '811',\n",
       "  'Region': '15:7180479-7226680',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '8385489',\n",
       "  'End': '8451753',\n",
       "  'Name': '812',\n",
       "  'Score': '',\n",
       "  'Gene': '812',\n",
       "  'Region': '15:8385489-8451753',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '12427846',\n",
       "  'End': '12497837',\n",
       "  'Name': '813',\n",
       "  'Score': '',\n",
       "  'Gene': '813',\n",
       "  'Region': '15:12427846-12497837',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '13081050',\n",
       "  'End': '13165931',\n",
       "  'Name': '814',\n",
       "  'Score': '',\n",
       "  'Gene': '814',\n",
       "  'Region': '15:13081050-13165931',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '16752412',\n",
       "  'End': '16823900',\n",
       "  'Name': '815',\n",
       "  'Score': '',\n",
       "  'Gene': '815',\n",
       "  'Region': '15:16752412-16823900',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '18857236',\n",
       "  'End': '18982658',\n",
       "  'Name': '816',\n",
       "  'Score': '',\n",
       "  'Gene': '816',\n",
       "  'Region': '15:18857236-18982658',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '24968652',\n",
       "  'End': '25036793',\n",
       "  'Name': '817',\n",
       "  'Score': '',\n",
       "  'Gene': '817',\n",
       "  'Region': '15:24968652-25036793',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '28181581',\n",
       "  'End': '28249520',\n",
       "  'Name': '818',\n",
       "  'Score': '',\n",
       "  'Gene': '818',\n",
       "  'Region': '15:28181581-28249520',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '30259955',\n",
       "  'End': '30307077',\n",
       "  'Name': '819',\n",
       "  'Score': '',\n",
       "  'Gene': '819',\n",
       "  'Region': '15:30259955-30307077',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '36488751',\n",
       "  'End': '36518020',\n",
       "  'Name': '820',\n",
       "  'Score': '',\n",
       "  'Gene': '820',\n",
       "  'Region': '15:36488751-36518020',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '40824516',\n",
       "  'End': '40979464',\n",
       "  'Name': '821',\n",
       "  'Score': '',\n",
       "  'Gene': '821',\n",
       "  'Region': '15:40824516-40979464',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '44721034',\n",
       "  'End': '44769894',\n",
       "  'Name': '822',\n",
       "  'Score': '',\n",
       "  'Gene': '822',\n",
       "  'Region': '15:44721034-44769894',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '45085447',\n",
       "  'End': '45132386',\n",
       "  'Name': '823',\n",
       "  'Score': '',\n",
       "  'Gene': '823',\n",
       "  'Region': '15:45085447-45132386',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '50664938',\n",
       "  'End': '50748251',\n",
       "  'Name': '824',\n",
       "  'Score': '',\n",
       "  'Gene': '824',\n",
       "  'Region': '15:50664938-50748251',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '52740443',\n",
       "  'End': '52810439',\n",
       "  'Name': '825',\n",
       "  'Score': '',\n",
       "  'Gene': '825',\n",
       "  'Region': '15:52740443-52810439',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '58168195',\n",
       "  'End': '58220642',\n",
       "  'Name': '826',\n",
       "  'Score': '',\n",
       "  'Gene': '826',\n",
       "  'Region': '15:58168195-58220642',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '67101131',\n",
       "  'End': '67162296',\n",
       "  'Name': '827',\n",
       "  'Score': '',\n",
       "  'Gene': '827',\n",
       "  'Region': '15:67101131-67162296',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '68694491',\n",
       "  'End': '68739615',\n",
       "  'Name': '828',\n",
       "  'Score': '',\n",
       "  'Gene': '828',\n",
       "  'Region': '15:68694491-68739615',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '68933567',\n",
       "  'End': '68992893',\n",
       "  'Name': '829',\n",
       "  'Score': '',\n",
       "  'Gene': '829',\n",
       "  'Region': '15:68933567-68992893',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '71889586',\n",
       "  'End': '71949467',\n",
       "  'Name': '830',\n",
       "  'Score': '',\n",
       "  'Gene': '830',\n",
       "  'Region': '15:71889586-71949467',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '76283099',\n",
       "  'End': '76313683',\n",
       "  'Name': '831',\n",
       "  'Score': '',\n",
       "  'Gene': '831',\n",
       "  'Region': '15:76283099-76313683',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '76444091',\n",
       "  'End': '76468590',\n",
       "  'Name': '832',\n",
       "  'Score': '',\n",
       "  'Gene': '832',\n",
       "  'Region': '15:76444091-76468590',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '80694138',\n",
       "  'End': '80722788',\n",
       "  'Name': '833',\n",
       "  'Score': '',\n",
       "  'Gene': '833',\n",
       "  'Region': '15:80694138-80722788',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '81514063',\n",
       "  'End': '81569541',\n",
       "  'Name': '834',\n",
       "  'Score': '',\n",
       "  'Gene': '834',\n",
       "  'Region': '15:81514063-81569541',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '81682506',\n",
       "  'End': '81719092',\n",
       "  'Name': '835',\n",
       "  'Score': '',\n",
       "  'Gene': '835',\n",
       "  'Region': '15:81682506-81719092',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '83374966',\n",
       "  'End': '83454933',\n",
       "  'Name': '836',\n",
       "  'Score': '',\n",
       "  'Gene': '836',\n",
       "  'Region': '15:83374966-83454933',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '88489865',\n",
       "  'End': '88537648',\n",
       "  'Name': '837',\n",
       "  'Score': '',\n",
       "  'Gene': '837',\n",
       "  'Region': '15:88489865-88537648',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '88828050',\n",
       "  'End': '88854719',\n",
       "  'Name': '838',\n",
       "  'Score': '',\n",
       "  'Gene': '838',\n",
       "  'Region': '15:88828050-88854719',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '92190253',\n",
       "  'End': '92277287',\n",
       "  'Name': '839',\n",
       "  'Score': '',\n",
       "  'Gene': '839',\n",
       "  'Region': '15:92190253-92277287',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '93492810',\n",
       "  'End': '93519982',\n",
       "  'Name': '840',\n",
       "  'Score': '',\n",
       "  'Gene': '840',\n",
       "  'Region': '15:93492810-93519982',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '95044078',\n",
       "  'End': '95086462',\n",
       "  'Name': '841',\n",
       "  'Score': '',\n",
       "  'Gene': '841',\n",
       "  'Region': '15:95044078-95086462',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '96284908',\n",
       "  'End': '96309877',\n",
       "  'Name': '842',\n",
       "  'Score': '',\n",
       "  'Gene': '842',\n",
       "  'Region': '15:96284908-96309877',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '101825997',\n",
       "  'End': '101874700',\n",
       "  'Name': '843',\n",
       "  'Score': '',\n",
       "  'Gene': '843',\n",
       "  'Region': '15:101825997-101874700',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr15',\n",
       "  'Start': '103511670',\n",
       "  'End': '103552538',\n",
       "  'Name': '844',\n",
       "  'Score': '',\n",
       "  'Gene': '844',\n",
       "  'Region': '15:103511670-103552538',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '5150368',\n",
       "  'End': '5195722',\n",
       "  'Name': '845',\n",
       "  'Score': '',\n",
       "  'Gene': '845',\n",
       "  'Region': '16:5150368-5195722',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '10341430',\n",
       "  'End': '10385351',\n",
       "  'Name': '846',\n",
       "  'Score': '',\n",
       "  'Gene': '846',\n",
       "  'Region': '16:10341430-10385351',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '21157080',\n",
       "  'End': '21192863',\n",
       "  'Name': '847',\n",
       "  'Score': '',\n",
       "  'Gene': '847',\n",
       "  'Region': '16:21157080-21192863',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '22766585',\n",
       "  'End': '22844730',\n",
       "  'Name': '848',\n",
       "  'Score': '',\n",
       "  'Gene': '848',\n",
       "  'Region': '16:22766585-22844730',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '23652588',\n",
       "  'End': '23694832',\n",
       "  'Name': '849',\n",
       "  'Score': '',\n",
       "  'Gene': '849',\n",
       "  'Region': '16:23652588-23694832',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '23952714',\n",
       "  'End': '23991471',\n",
       "  'Name': '850',\n",
       "  'Score': '',\n",
       "  'Gene': '850',\n",
       "  'Region': '16:23952714-23991471',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '25983196',\n",
       "  'End': '26078540',\n",
       "  'Name': '851',\n",
       "  'Score': '',\n",
       "  'Gene': '851',\n",
       "  'Region': '16:25983196-26078540',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '28593146',\n",
       "  'End': '28632529',\n",
       "  'Name': '852',\n",
       "  'Score': '',\n",
       "  'Gene': '852',\n",
       "  'Region': '16:28593146-28632529',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '28823123',\n",
       "  'End': '28909744',\n",
       "  'Name': '853',\n",
       "  'Score': '',\n",
       "  'Gene': '853',\n",
       "  'Region': '16:28823123-28909744',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '29128866',\n",
       "  'End': '29179467',\n",
       "  'Name': '854',\n",
       "  'Score': '',\n",
       "  'Gene': '854',\n",
       "  'Region': '16:29128866-29179467',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '32441344',\n",
       "  'End': '32543252',\n",
       "  'Name': '855',\n",
       "  'Score': '',\n",
       "  'Gene': '855',\n",
       "  'Region': '16:32441344-32543252',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '32653912',\n",
       "  'End': '32716340',\n",
       "  'Name': '856',\n",
       "  'Score': '',\n",
       "  'Gene': '856',\n",
       "  'Region': '16:32653912-32716340',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '34967154',\n",
       "  'End': '35007777',\n",
       "  'Name': '857',\n",
       "  'Score': '',\n",
       "  'Gene': '857',\n",
       "  'Region': '16:34967154-35007777',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '35733966',\n",
       "  'End': '35784035',\n",
       "  'Name': '858',\n",
       "  'Score': '',\n",
       "  'Gene': '858',\n",
       "  'Region': '16:35733966-35784035',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '36620671',\n",
       "  'End': '36670286',\n",
       "  'Name': '859',\n",
       "  'Score': '',\n",
       "  'Gene': '859',\n",
       "  'Region': '16:36620671-36670286',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '38160332',\n",
       "  'End': '38249251',\n",
       "  'Name': '860',\n",
       "  'Score': '',\n",
       "  'Gene': '860',\n",
       "  'Region': '16:38160332-38249251',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '41524064',\n",
       "  'End': '41552563',\n",
       "  'Name': '861',\n",
       "  'Score': '',\n",
       "  'Gene': '861',\n",
       "  'Region': '16:41524064-41552563',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '42255627',\n",
       "  'End': '42331328',\n",
       "  'Name': '862',\n",
       "  'Score': '',\n",
       "  'Gene': '862',\n",
       "  'Region': '16:42255627-42331328',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '42885154',\n",
       "  'End': '42959540',\n",
       "  'Name': '863',\n",
       "  'Score': '',\n",
       "  'Gene': '863',\n",
       "  'Region': '16:42885154-42959540',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '43922104',\n",
       "  'End': '43973953',\n",
       "  'Name': '864',\n",
       "  'Score': '',\n",
       "  'Gene': '864',\n",
       "  'Region': '16:43922104-43973953',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '44210258',\n",
       "  'End': '44291077',\n",
       "  'Name': '865',\n",
       "  'Score': '',\n",
       "  'Gene': '865',\n",
       "  'Region': '16:44210258-44291077',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '44210258',\n",
       "  'End': '44291077',\n",
       "  'Name': '866',\n",
       "  'Score': '',\n",
       "  'Gene': '866',\n",
       "  'Region': '16:44210258-44291077',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '50689262',\n",
       "  'End': '50737219',\n",
       "  'Name': '867',\n",
       "  'Score': '',\n",
       "  'Gene': '867',\n",
       "  'Region': '16:50689262-50737219',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '52078029',\n",
       "  'End': '52147877',\n",
       "  'Name': '868',\n",
       "  'Score': '',\n",
       "  'Gene': '868',\n",
       "  'Region': '16:52078029-52147877',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '58444511',\n",
       "  'End': '58519914',\n",
       "  'Name': '869',\n",
       "  'Score': '',\n",
       "  'Gene': '869',\n",
       "  'Region': '16:58444511-58519914',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '67569729',\n",
       "  'End': '67618201',\n",
       "  'Name': '870',\n",
       "  'Score': '',\n",
       "  'Gene': '870',\n",
       "  'Region': '16:67569729-67618201',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '74089872',\n",
       "  'End': '74191979',\n",
       "  'Name': '871',\n",
       "  'Score': '',\n",
       "  'Gene': '871',\n",
       "  'Region': '16:74089872-74191979',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '77384249',\n",
       "  'End': '77449818',\n",
       "  'Name': '872',\n",
       "  'Score': '',\n",
       "  'Gene': '872',\n",
       "  'Region': '16:77384249-77449818',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '77630781',\n",
       "  'End': '77667713',\n",
       "  'Name': '873',\n",
       "  'Score': '',\n",
       "  'Gene': '873',\n",
       "  'Region': '16:77630781-77667713',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '78563141',\n",
       "  'End': '78599787',\n",
       "  'Name': '874',\n",
       "  'Score': '',\n",
       "  'Gene': '874',\n",
       "  'Region': '16:78563141-78599787',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '88127436',\n",
       "  'End': '88188446',\n",
       "  'Name': '875',\n",
       "  'Score': '',\n",
       "  'Gene': '875',\n",
       "  'Region': '16:88127436-88188446',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '89517846',\n",
       "  'End': '89598464',\n",
       "  'Name': '876',\n",
       "  'Score': '',\n",
       "  'Gene': '876',\n",
       "  'Region': '16:89517846-89598464',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '90925327',\n",
       "  'End': '91000712',\n",
       "  'Name': '877',\n",
       "  'Score': '',\n",
       "  'Gene': '877',\n",
       "  'Region': '16:90925327-91000712',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '93977016',\n",
       "  'End': '94021875',\n",
       "  'Name': '878',\n",
       "  'Score': '',\n",
       "  'Gene': '878',\n",
       "  'Region': '16:93977016-94021875',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '94289006',\n",
       "  'End': '94314592',\n",
       "  'Name': '879',\n",
       "  'Score': '',\n",
       "  'Gene': '879',\n",
       "  'Region': '16:94289006-94314592',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '94429141',\n",
       "  'End': '94473370',\n",
       "  'Name': '880',\n",
       "  'Score': '',\n",
       "  'Gene': '880',\n",
       "  'Region': '16:94429141-94473370',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '95371862',\n",
       "  'End': '95450042',\n",
       "  'Name': '881',\n",
       "  'Score': '',\n",
       "  'Gene': '881',\n",
       "  'Region': '16:95371862-95450042',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr16',\n",
       "  'Start': '97697705',\n",
       "  'End': '97752534',\n",
       "  'Name': '882',\n",
       "  'Score': '',\n",
       "  'Gene': '882',\n",
       "  'Region': '16:97697705-97752534',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '6768025',\n",
       "  'End': '6812752',\n",
       "  'Name': '883',\n",
       "  'Score': '',\n",
       "  'Gene': '883',\n",
       "  'Region': '17:6768025-6812752',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '11664178',\n",
       "  'End': '11787180',\n",
       "  'Name': '884',\n",
       "  'Score': '',\n",
       "  'Gene': '884',\n",
       "  'Region': '17:11664178-11787180',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '13818723',\n",
       "  'End': '13919026',\n",
       "  'Name': '885',\n",
       "  'Score': '',\n",
       "  'Gene': '885',\n",
       "  'Region': '17:13818723-13919026',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '15371839',\n",
       "  'End': '15387147',\n",
       "  'Name': '886',\n",
       "  'Score': '',\n",
       "  'Gene': '886',\n",
       "  'Region': '17:15371839-15387147',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '23526967',\n",
       "  'End': '23551137',\n",
       "  'Name': '887',\n",
       "  'Score': '',\n",
       "  'Gene': '887',\n",
       "  'Region': '17:23526967-23551137',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '24048288',\n",
       "  'End': '24091203',\n",
       "  'Name': '888',\n",
       "  'Score': '',\n",
       "  'Gene': '888',\n",
       "  'Region': '17:24048288-24091203',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '31809132',\n",
       "  'End': '31843772',\n",
       "  'Name': '889',\n",
       "  'Score': '',\n",
       "  'Gene': '889',\n",
       "  'Region': '17:31809132-31843772',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '34100951',\n",
       "  'End': '34135951',\n",
       "  'Name': '890',\n",
       "  'Score': '',\n",
       "  'Gene': '890',\n",
       "  'Region': '17:34100951-34135951',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '35526761',\n",
       "  'End': '35550061',\n",
       "  'Name': '891',\n",
       "  'Score': '',\n",
       "  'Gene': '891',\n",
       "  'Region': '17:35526761-35550061',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '43021066',\n",
       "  'End': '43093508',\n",
       "  'Name': '892',\n",
       "  'Score': '',\n",
       "  'Gene': '892',\n",
       "  'Region': '17:43021066-43093508',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '44198990',\n",
       "  'End': '44262216',\n",
       "  'Name': '893',\n",
       "  'Score': '',\n",
       "  'Gene': '893',\n",
       "  'Region': '17:44198990-44262216',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '46838396',\n",
       "  'End': '46867205',\n",
       "  'Name': '894',\n",
       "  'Score': '',\n",
       "  'Gene': '894',\n",
       "  'Region': '17:46838396-46867205',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '48363042',\n",
       "  'End': '48400987',\n",
       "  'Name': '895',\n",
       "  'Score': '',\n",
       "  'Gene': '895',\n",
       "  'Region': '17:48363042-48400987',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '48542940',\n",
       "  'End': '48602895',\n",
       "  'Name': '896',\n",
       "  'Score': '',\n",
       "  'Gene': '896',\n",
       "  'Region': '17:48542940-48602895',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '48771285',\n",
       "  'End': '48825829',\n",
       "  'Name': '897',\n",
       "  'Score': '',\n",
       "  'Gene': '897',\n",
       "  'Region': '17:48771285-48825829',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '49468027',\n",
       "  'End': '49552568',\n",
       "  'Name': '898',\n",
       "  'Score': '',\n",
       "  'Gene': '898',\n",
       "  'Region': '17:49468027-49552568',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '50125429',\n",
       "  'End': '50194110',\n",
       "  'Name': '899',\n",
       "  'Score': '',\n",
       "  'Gene': '899',\n",
       "  'Region': '17:50125429-50194110',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '51737923',\n",
       "  'End': '51806643',\n",
       "  'Name': '900',\n",
       "  'Score': '',\n",
       "  'Gene': '900',\n",
       "  'Region': '17:51737923-51806643',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '53577112',\n",
       "  'End': '53628475',\n",
       "  'Name': '901',\n",
       "  'Score': '',\n",
       "  'Gene': '901',\n",
       "  'Region': '17:53577112-53628475',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '55276212',\n",
       "  'End': '55309734',\n",
       "  'Name': '902',\n",
       "  'Score': '',\n",
       "  'Gene': '902',\n",
       "  'Region': '17:55276212-55309734',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '56882991',\n",
       "  'End': '56929799',\n",
       "  'Name': '903',\n",
       "  'Score': '',\n",
       "  'Gene': '903',\n",
       "  'Region': '17:56882991-56929799',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '57264102',\n",
       "  'End': '57347459',\n",
       "  'Name': '904',\n",
       "  'Score': '',\n",
       "  'Gene': '904',\n",
       "  'Region': '17:57264102-57347459',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '62640369',\n",
       "  'End': '62730980',\n",
       "  'Name': '905',\n",
       "  'Score': '',\n",
       "  'Gene': '905',\n",
       "  'Region': '17:62640369-62730980',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '67440764',\n",
       "  'End': '67473458',\n",
       "  'Name': '906',\n",
       "  'Score': '',\n",
       "  'Gene': '906',\n",
       "  'Region': '17:67440764-67473458',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '73219957',\n",
       "  'End': '73272539',\n",
       "  'Name': '907',\n",
       "  'Score': '',\n",
       "  'Gene': '907',\n",
       "  'Region': '17:73219957-73272539',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '73881483',\n",
       "  'End': '73942612',\n",
       "  'Name': '908',\n",
       "  'Score': '',\n",
       "  'Gene': '908',\n",
       "  'Region': '17:73881483-73942612',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '75442566',\n",
       "  'End': '75537174',\n",
       "  'Name': '909',\n",
       "  'Score': '',\n",
       "  'Gene': '909',\n",
       "  'Region': '17:75442566-75537174',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '80147911',\n",
       "  'End': '80199379',\n",
       "  'Name': '910',\n",
       "  'Score': '',\n",
       "  'Gene': '910',\n",
       "  'Region': '17:80147911-80199379',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '83540997',\n",
       "  'End': '83596275',\n",
       "  'Name': '911',\n",
       "  'Score': '',\n",
       "  'Gene': '911',\n",
       "  'Region': '17:83540997-83596275',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '84474454',\n",
       "  'End': '84520370',\n",
       "  'Name': '912',\n",
       "  'Score': '',\n",
       "  'Gene': '912',\n",
       "  'Region': '17:84474454-84520370',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '84860274',\n",
       "  'End': '84943300',\n",
       "  'Name': '913',\n",
       "  'Score': '',\n",
       "  'Gene': '913',\n",
       "  'Region': '17:84860274-84943300',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '89332309',\n",
       "  'End': '89367903',\n",
       "  'Name': '914',\n",
       "  'Score': '',\n",
       "  'Gene': '914',\n",
       "  'Region': '17:89332309-89367903',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '90054561',\n",
       "  'End': '90116527',\n",
       "  'Name': '915',\n",
       "  'Score': '',\n",
       "  'Gene': '915',\n",
       "  'Region': '17:90054561-90116527',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr17',\n",
       "  'Start': '90286015',\n",
       "  'End': '90411156',\n",
       "  'Name': '916',\n",
       "  'Score': '',\n",
       "  'Gene': '916',\n",
       "  'Region': '17:90286015-90411156',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '6939796',\n",
       "  'End': '7001984',\n",
       "  'Name': '917',\n",
       "  'Score': '',\n",
       "  'Gene': '917',\n",
       "  'Region': '18:6939796-7001984',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '15401109',\n",
       "  'End': '15442687',\n",
       "  'Name': '918',\n",
       "  'Score': '',\n",
       "  'Gene': '918',\n",
       "  'Region': '18:15401109-15442687',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '21161931',\n",
       "  'End': '21193166',\n",
       "  'Name': '919',\n",
       "  'Score': '',\n",
       "  'Gene': '919',\n",
       "  'Region': '18:21161931-21193166',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '24099649',\n",
       "  'End': '24135055',\n",
       "  'Name': '920',\n",
       "  'Score': '',\n",
       "  'Gene': '920',\n",
       "  'Region': '18:24099649-24135055',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '24418559',\n",
       "  'End': '24490784',\n",
       "  'Name': '921',\n",
       "  'Score': '',\n",
       "  'Gene': '921',\n",
       "  'Region': '18:24418559-24490784',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '24722942',\n",
       "  'End': '24878570',\n",
       "  'Name': '922',\n",
       "  'Score': '',\n",
       "  'Gene': '922',\n",
       "  'Region': '18:24722942-24878570',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '31939025',\n",
       "  'End': '31985900',\n",
       "  'Name': '923',\n",
       "  'Score': '',\n",
       "  'Gene': '923',\n",
       "  'Region': '18:31939025-31985900',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '32946362',\n",
       "  'End': '33006449',\n",
       "  'Name': '924',\n",
       "  'Score': '',\n",
       "  'Gene': '924',\n",
       "  'Region': '18:32946362-33006449',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '32946362',\n",
       "  'End': '33006449',\n",
       "  'Name': '925',\n",
       "  'Score': '',\n",
       "  'Gene': '925',\n",
       "  'Region': '18:32946362-33006449',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '34079958',\n",
       "  'End': '34129601',\n",
       "  'Name': '926',\n",
       "  'Score': '',\n",
       "  'Gene': '926',\n",
       "  'Region': '18:34079958-34129601',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '35482165',\n",
       "  'End': '35540427',\n",
       "  'Name': '927',\n",
       "  'Score': '',\n",
       "  'Gene': '927',\n",
       "  'Region': '18:35482165-35540427',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '36691158',\n",
       "  'End': '36720587',\n",
       "  'Name': '928',\n",
       "  'Score': '',\n",
       "  'Gene': '928',\n",
       "  'Region': '18:36691158-36720587',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '37149186',\n",
       "  'End': '37191875',\n",
       "  'Name': '929',\n",
       "  'Score': '',\n",
       "  'Gene': '929',\n",
       "  'Region': '18:37149186-37191875',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '39896744',\n",
       "  'End': '39942821',\n",
       "  'Name': '930',\n",
       "  'Score': '',\n",
       "  'Gene': '930',\n",
       "  'Region': '18:39896744-39942821',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '41746971',\n",
       "  'End': '41818086',\n",
       "  'Name': '931',\n",
       "  'Score': '',\n",
       "  'Gene': '931',\n",
       "  'Region': '18:41746971-41818086',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '42065872',\n",
       "  'End': '42130040',\n",
       "  'Name': '932',\n",
       "  'Score': '',\n",
       "  'Gene': '932',\n",
       "  'Region': '18:42065872-42130040',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '45578608',\n",
       "  'End': '45652084',\n",
       "  'Name': '933',\n",
       "  'Score': '',\n",
       "  'Gene': '933',\n",
       "  'Region': '18:45578608-45652084',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '46179765',\n",
       "  'End': '46204913',\n",
       "  'Name': '934',\n",
       "  'Score': '',\n",
       "  'Gene': '934',\n",
       "  'Region': '18:46179765-46204913',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '49592614',\n",
       "  'End': '49658484',\n",
       "  'Name': '935',\n",
       "  'Score': '',\n",
       "  'Gene': '935',\n",
       "  'Region': '18:49592614-49658484',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '50011602',\n",
       "  'End': '50069289',\n",
       "  'Name': '936',\n",
       "  'Score': '',\n",
       "  'Gene': '936',\n",
       "  'Region': '18:50011602-50069289',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '53229139',\n",
       "  'End': '53325927',\n",
       "  'Name': '937',\n",
       "  'Score': '',\n",
       "  'Gene': '937',\n",
       "  'Region': '18:53229139-53325927',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '56003891',\n",
       "  'End': '56072519',\n",
       "  'Name': '938',\n",
       "  'Score': '',\n",
       "  'Gene': '938',\n",
       "  'Region': '18:56003891-56072519',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '57249664',\n",
       "  'End': '57297027',\n",
       "  'Name': '939',\n",
       "  'Score': '',\n",
       "  'Gene': '939',\n",
       "  'Region': '18:57249664-57297027',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '59190021',\n",
       "  'End': '59340310',\n",
       "  'Name': '940',\n",
       "  'Score': '',\n",
       "  'Gene': '940',\n",
       "  'Region': '18:59190021-59340310',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '63729174',\n",
       "  'End': '63777284',\n",
       "  'Name': '941',\n",
       "  'Score': '',\n",
       "  'Gene': '941',\n",
       "  'Region': '18:63729174-63777284',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '64585603',\n",
       "  'End': '64638451',\n",
       "  'Name': '942',\n",
       "  'Score': '',\n",
       "  'Gene': '942',\n",
       "  'Region': '18:64585603-64638451',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '65853454',\n",
       "  'End': '65935591',\n",
       "  'Name': '943',\n",
       "  'Score': '',\n",
       "  'Gene': '943',\n",
       "  'Region': '18:65853454-65935591',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '68943328',\n",
       "  'End': '68990228',\n",
       "  'Name': '944',\n",
       "  'Score': '',\n",
       "  'Gene': '944',\n",
       "  'Region': '18:68943328-68990228',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '74471676',\n",
       "  'End': '74520383',\n",
       "  'Name': '945',\n",
       "  'Score': '',\n",
       "  'Gene': '945',\n",
       "  'Region': '18:74471676-74520383',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '74679677',\n",
       "  'End': '74741625',\n",
       "  'Name': '946',\n",
       "  'Score': '',\n",
       "  'Gene': '946',\n",
       "  'Region': '18:74679677-74741625',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '77968807',\n",
       "  'End': '78032826',\n",
       "  'Name': '947',\n",
       "  'Score': '',\n",
       "  'Gene': '947',\n",
       "  'Region': '18:77968807-78032826',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '80960121',\n",
       "  'End': '80992541',\n",
       "  'Name': '948',\n",
       "  'Score': '',\n",
       "  'Gene': '948',\n",
       "  'Region': '18:80960121-80992541',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr18',\n",
       "  'Start': '85288941',\n",
       "  'End': '85369839',\n",
       "  'Name': '949',\n",
       "  'Score': '',\n",
       "  'Gene': '949',\n",
       "  'Region': '18:85288941-85369839',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '3635409',\n",
       "  'End': '3699978',\n",
       "  'Name': '950',\n",
       "  'Score': '',\n",
       "  'Gene': '950',\n",
       "  'Region': '19:3635409-3699978',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '3832930',\n",
       "  'End': '3896324',\n",
       "  'Name': '951',\n",
       "  'Score': '',\n",
       "  'Gene': '951',\n",
       "  'Region': '19:3832930-3896324',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '5648442',\n",
       "  'End': '5675296',\n",
       "  'Name': '952',\n",
       "  'Score': '',\n",
       "  'Gene': '952',\n",
       "  'Region': '19:5648442-5675296',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '6286522',\n",
       "  'End': '6305238',\n",
       "  'Name': '953',\n",
       "  'Score': '',\n",
       "  'Gene': '953',\n",
       "  'Region': '19:6286522-6305238',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '7064548',\n",
       "  'End': '7197140',\n",
       "  'Name': '954',\n",
       "  'Score': '',\n",
       "  'Gene': '954',\n",
       "  'Region': '19:7064548-7197140',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '8554284',\n",
       "  'End': '8587137',\n",
       "  'Name': '955',\n",
       "  'Score': '',\n",
       "  'Gene': '955',\n",
       "  'Region': '19:8554284-8587137',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '10695617',\n",
       "  'End': '10730864',\n",
       "  'Name': '956',\n",
       "  'Score': '',\n",
       "  'Gene': '956',\n",
       "  'Region': '19:10695617-10730864',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '10940147',\n",
       "  'End': '10960163',\n",
       "  'Name': '957',\n",
       "  'Score': '',\n",
       "  'Gene': '957',\n",
       "  'Region': '19:10940147-10960163',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '12434826',\n",
       "  'End': '12494327',\n",
       "  'Name': '958',\n",
       "  'Score': '',\n",
       "  'Gene': '958',\n",
       "  'Region': '19:12434826-12494327',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '15790510',\n",
       "  'End': '15815819',\n",
       "  'Name': '959',\n",
       "  'Score': '',\n",
       "  'Gene': '959',\n",
       "  'Region': '19:15790510-15815819',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '17306782',\n",
       "  'End': '17345629',\n",
       "  'Name': '960',\n",
       "  'Score': '',\n",
       "  'Gene': '960',\n",
       "  'Region': '19:17306782-17345629',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '20528919',\n",
       "  'End': '20560378',\n",
       "  'Name': '961',\n",
       "  'Score': '',\n",
       "  'Gene': '961',\n",
       "  'Region': '19:20528919-20560378',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '21723365',\n",
       "  'End': '21747827',\n",
       "  'Name': '962',\n",
       "  'Score': '',\n",
       "  'Gene': '962',\n",
       "  'Region': '19:21723365-21747827',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '23563461',\n",
       "  'End': '23610199',\n",
       "  'Name': '963',\n",
       "  'Score': '',\n",
       "  'Gene': '963',\n",
       "  'Region': '19:23563461-23610199',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '24610840',\n",
       "  'End': '24677705',\n",
       "  'Name': '964',\n",
       "  'Score': '',\n",
       "  'Gene': '964',\n",
       "  'Region': '19:24610840-24677705',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '24954141',\n",
       "  'End': '25003094',\n",
       "  'Name': '965',\n",
       "  'Score': '',\n",
       "  'Gene': '965',\n",
       "  'Region': '19:24954141-25003094',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '27534144',\n",
       "  'End': '27581990',\n",
       "  'Name': '966',\n",
       "  'Score': '',\n",
       "  'Gene': '966',\n",
       "  'Region': '19:27534144-27581990',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '28365825',\n",
       "  'End': '28399866',\n",
       "  'Name': '967',\n",
       "  'Score': '',\n",
       "  'Gene': '967',\n",
       "  'Region': '19:28365825-28399866',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '28573435',\n",
       "  'End': '28636484',\n",
       "  'Name': '968',\n",
       "  'Score': '',\n",
       "  'Gene': '968',\n",
       "  'Region': '19:28573435-28636484',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '29866927',\n",
       "  'End': '29918435',\n",
       "  'Name': '969',\n",
       "  'Score': '',\n",
       "  'Gene': '969',\n",
       "  'Region': '19:29866927-29918435',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '32905563',\n",
       "  'End': '32961276',\n",
       "  'Name': '970',\n",
       "  'Score': '',\n",
       "  'Gene': '970',\n",
       "  'Region': '19:32905563-32961276',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '33223942',\n",
       "  'End': '33285336',\n",
       "  'Name': '971',\n",
       "  'Score': '',\n",
       "  'Gene': '971',\n",
       "  'Region': '19:33223942-33285336',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '36182931',\n",
       "  'End': '36265082',\n",
       "  'Name': '972',\n",
       "  'Score': '',\n",
       "  'Gene': '972',\n",
       "  'Region': '19:36182931-36265082',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '36728674',\n",
       "  'End': '36749528',\n",
       "  'Name': '973',\n",
       "  'Score': '',\n",
       "  'Gene': '973',\n",
       "  'Region': '19:36728674-36749528',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '37482495',\n",
       "  'End': '37510209',\n",
       "  'Name': '974',\n",
       "  'Score': '',\n",
       "  'Gene': '974',\n",
       "  'Region': '19:37482495-37510209',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '37975494',\n",
       "  'End': '38031498',\n",
       "  'Name': '975',\n",
       "  'Score': '',\n",
       "  'Gene': '975',\n",
       "  'Region': '19:37975494-38031498',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '41060775',\n",
       "  'End': '41087097',\n",
       "  'Name': '976',\n",
       "  'Score': '',\n",
       "  'Gene': '976',\n",
       "  'Region': '19:41060775-41087097',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '41291381',\n",
       "  'End': '41345899',\n",
       "  'Name': '977',\n",
       "  'Score': '',\n",
       "  'Gene': '977',\n",
       "  'Region': '19:41291381-41345899',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '41973511',\n",
       "  'End': '42026554',\n",
       "  'Name': '978',\n",
       "  'Score': '',\n",
       "  'Gene': '978',\n",
       "  'Region': '19:41973511-42026554',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '45871417',\n",
       "  'End': '45957203',\n",
       "  'Name': '979',\n",
       "  'Score': '',\n",
       "  'Gene': '979',\n",
       "  'Region': '19:45871417-45957203',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '47644250',\n",
       "  'End': '47659784',\n",
       "  'Name': '980',\n",
       "  'Score': '',\n",
       "  'Gene': '980',\n",
       "  'Region': '19:47644250-47659784',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '48649897',\n",
       "  'End': '48737838',\n",
       "  'Name': '981',\n",
       "  'Score': '',\n",
       "  'Gene': '981',\n",
       "  'Region': '19:48649897-48737838',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '58804802',\n",
       "  'End': '58856457',\n",
       "  'Name': '982',\n",
       "  'Score': '',\n",
       "  'Gene': '982',\n",
       "  'Region': '19:58804802-58856457',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '59025717',\n",
       "  'End': '59053288',\n",
       "  'Name': '983',\n",
       "  'Score': '',\n",
       "  'Gene': '983',\n",
       "  'Region': '19:59025717-59053288',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chr19',\n",
       "  'Start': '59379976',\n",
       "  'End': '59435429',\n",
       "  'Name': '984',\n",
       "  'Score': '',\n",
       "  'Gene': '984',\n",
       "  'Region': '19:59379976-59435429',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chrX',\n",
       "  'Start': '93281165',\n",
       "  'End': '93299819',\n",
       "  'Name': '985',\n",
       "  'Score': '',\n",
       "  'Gene': '985',\n",
       "  'Region': 'X:93281165-93299819',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chrX',\n",
       "  'Start': '98989231',\n",
       "  'End': '99078826',\n",
       "  'Name': '986',\n",
       "  'Score': '',\n",
       "  'Gene': '986',\n",
       "  'Region': 'X:98989231-99078826',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chrX',\n",
       "  'Start': '101920335',\n",
       "  'End': '102005609',\n",
       "  'Name': '987',\n",
       "  'Score': '',\n",
       "  'Gene': '987',\n",
       "  'Region': 'X:101920335-102005609',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chrX',\n",
       "  'Start': '136782307',\n",
       "  'End': '136841006',\n",
       "  'Name': '988',\n",
       "  'Score': '',\n",
       "  'Gene': '988',\n",
       "  'Region': 'X:136782307-136841006',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chrX',\n",
       "  'Start': '140491839',\n",
       "  'End': '140537985',\n",
       "  'Name': '989',\n",
       "  'Score': '',\n",
       "  'Gene': '989',\n",
       "  'Region': 'X:140491839-140537985',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chrX',\n",
       "  'Start': '167132110',\n",
       "  'End': '167181320',\n",
       "  'Name': '990',\n",
       "  'Score': '',\n",
       "  'Gene': '990',\n",
       "  'Region': 'X:167132110-167181320',\n",
       "  'Strand': '+'},\n",
       " {'Chr': 'chrX',\n",
       "  'Start': '169972881',\n",
       "  'End': '170022477',\n",
       "  'Name': '991',\n",
       "  'Score': '',\n",
       "  'Gene': '991',\n",
       "  'Region': 'X:169972881-170022477',\n",
       "  'Strand': '+'}]"
      ]
     },
     "execution_count": 128,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "region_dict"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<a id='2'></a>\n",
    "# 2. Design probe targeting sequences by probe_designer\n",
    "\n",
    "run probe_designer, remeber to clear the memory usage because each table should take ~32GB"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<a id='2.1'></a>\n",
    "## 2.1 Construct count table with all the 17-mers in the genome\n",
    "\n",
    "Only do this if you don't have pre-built 17-mer\n",
    "\n",
    "However you can do almost the same thing for your own library during quality check\n",
    "\n",
    "This library requires mm10 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 130,
   "metadata": {},
   "outputs": [],
   "source": [
    "overwrite_table = False"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### construct map for whole genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 131,
   "metadata": {},
   "outputs": [],
   "source": [
    "from ImageAnalysis3 import library_tools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 132,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\GRCm38_genome_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.design)\n",
    " \n",
    "genome_table_file = os.path.join(reference_folder, 'GRCm38_genome_17w.npy')\n",
    "\n",
    "if not os.path.exists(genome_table_file) or overwrite_table:\n",
    "    # genome\n",
    "    _genome_filenames = [os.path.join(genome_folder, _fl) \n",
    "         for _fl in os.listdir(genome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_genome_filenames))\n",
    "\n",
    "    ct = library_tools.design.countTable(word=17,save_file=genome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_genome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "\n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {genome_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### construct map for transcriptome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 134,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\GRCm38_transcriptome_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "from tqdm import tqdm\n",
    "# transcriptome\n",
    "transcriptome_folder = os.path.join(reference_folder, 'Transcriptome')\n",
    "\n",
    "transcriptome_table_file = os.path.join(reference_folder, 'GRCm38_transcriptome_17w.npy')\n",
    "\n",
    "if not os.path.exists(transcriptome_table_file) or overwrite_table:\n",
    "    # transcriptome\n",
    "    _transcriptome_filenames = [os.path.join(transcriptome_folder, _fl) \n",
    "         for _fl in os.listdir(transcriptome_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_transcriptome_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=transcriptome_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_transcriptome_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {transcriptome_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### construct map for repeats from RepBase"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 135,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "reference table: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\Genomes\\mouse\\GRCm38_ensembl\\Repbase_v2603_repeat_17w.npy already exist, skip.\n"
     ]
    }
   ],
   "source": [
    "from tqdm import tqdm\n",
    "# repeat\n",
    "repeat_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\Repbase'\n",
    "\n",
    "repeat_table_file = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "\n",
    "if not os.path.exists(repeat_table_file) or overwrite_table:\n",
    "    # repeat\n",
    "    _repeat_filenames = [os.path.join(repeat_folder, _fl) \n",
    "         for _fl in os.listdir(repeat_folder) \n",
    "         if _fl.split(os.extsep)[-1]=='fasta' or _fl.split(os.extsep)[-1]=='fa']\n",
    "    print(len(_repeat_filenames))\n",
    "    \n",
    "    ct = library_tools.design.countTable(word=17,save_file=repeat_table_file, \n",
    "                       sparse=False)\n",
    "    ct.verbose=True\n",
    "\n",
    "    ct.read(_repeat_filenames) # read sequences from fasta files\n",
    "\n",
    "    ct.consume_loaded(num_threads=24) # convert sequences into integers\n",
    "\n",
    "    ct.complete(verbose=True)\n",
    "\n",
    "    ct.save()\n",
    "    \n",
    "    # clear RAM if contructed countable \n",
    "    del(ct)\n",
    "else:\n",
    "    print(f\"reference table: {repeat_table_file} already exist, skip.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<a id='2.2'></a>\n",
    "## 2.2 Design probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 137,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "991 regions loaded to design probes.\n",
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\reports\\merged_probes.pbr\n"
     ]
    }
   ],
   "source": [
    "# requires pre_defined genome_folder and library_folder\n",
    "# Indices\n",
    "genome_index = os.path.join(reference_folder, 'GRCm38_genome_17w.npy')\n",
    "transcriptome_index = os.path.join(reference_folder, 'GRCm38_transcriptome_17w.npy') \n",
    "repeat_index = os.path.join(reference_folder, 'Repbase_v2603_repeat_17w.npy')\n",
    "#ref_merfish_index = os.path.join(reference_folder, 'M1_meng_MERFISH_17w.npy') # merfish designed by Meng\n",
    "# get input files \n",
    "input_files = glob.glob(os.path.join(sequence_folder, '*.fasta'))\n",
    "\n",
    "print(f\"{len(input_files)} regions loaded to design probes.\")\n",
    "\n",
    "if not os.path.exists(report_folder):\n",
    "    os.makedirs(report_folder)\n",
    "    \n",
    "# filename to save probe reports\n",
    "save_file = os.path.join(report_folder, f'merged_probes.pbr')\n",
    "print(save_file)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### create pb_designer class"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'save_file' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-5-85410a926f94>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[1;32m----> 1\u001b[1;33m \u001b[0msave_file\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[1;31mNameError\u001b[0m: name 'save_file' is not defined"
     ]
    }
   ],
   "source": [
    "save_file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 144,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Probe designer derived from Bogdan Bintu:\n",
      "https://github.com/BogdanBintu/ChromatinImaging/blob/master/LibraryDesign/LibraryDesigner.py\n",
      "by Pu Zheng, 2020.11\n",
      "\n",
      "Major changes:\n",
      "    1. allow design of two strands\n",
      "    2. separate reverse_complement (rev_com) and from two strands (two_stranded) as \n",
      "    two different inputs for map_dic and sequence_dic\n",
      "    3. replace 'local_genome' with 'self_sequences' to be more explicit, and only \n",
      "    exclude the counts for the corresponding self_sequence within each input. \n",
      "\n",
      "Key information:\n",
      "    - number of input_sequence(s): 991\n",
      "    - save_file location: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\reports\\merged_probes.pbr\n",
      "\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "pb_designer = library_tools.design.pb_reports_class(\n",
    "    sequence_dic={'file':input_files,\n",
    "                  'rev_com':True, # design two strands\n",
    "                  'two_stranded':True},\n",
    "    map_dic={'genome':{'file':genome_index,'rev_com':False,'two_stranded':True},\n",
    "             'transcriptome':{'file':transcriptome_index,'rev_com':True,'two_stranded':False},\n",
    "             'rep_genome':{'file':repeat_index,'rev_com':False,'two_stranded':True},\n",
    "             'self_sequences':{'file':input_files,'force_list':True,'rev_com':False,'two_stranded':True},\n",
    "             #'ref_merfish':{'file':ref_merfish_index,'rev_com':False,'two_stranded':True},\n",
    "             },\n",
    "        save_file=save_file,\n",
    "    params_dic={'word_size':17,'pb_len':42,'buffer_len':2,'max_count':2**16-1,\n",
    "                'check_on_go': False, # whether automatically check probes\n",
    "                'auto': False, # whether automatically convert reference maps\n",
    "               },\n",
    "    check_dic={('genome','self_sequences'): 25,\n",
    "               'rep_genome': 0,\n",
    "               'transcriptome': 14,\n",
    "               #'ref_merfish': 14,\n",
    "               'gc':[0.25,0.85],'tm': 37+0.62*50+5,\n",
    "               }\n",
    "    )\n",
    "print(pb_designer)\n",
    "#pb_designer.load_from_file(load_probes_only=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 140,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Fail to load from savefile: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\reports\\merged_probes.pbr, file doesn't exist.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "False"
      ]
     },
     "execution_count": 140,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pb_designer.load_from_file(load_probes_only=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### calculate probe reports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 145,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- setting attribute: map_genome\n",
      "--- finish map_genome in 97.122s.\n",
      "-- setting attribute: map_transcriptome\n",
      "--- finish map_transcriptome in 128.480s.\n",
      "-- setting attribute: map_rep_genome\n",
      "--- finish map_rep_genome in 131.638s.\n",
      "Time(s): 357.40871381759644\n",
      "- Designing targeting sequence for 991 regions\n",
      "-- designing region: 1:9600471-9656458_:+_gene_1 region: 0, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\1_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.566s.\n",
      "in 33.926s.\n",
      "-- designing region: 1:9749317-9845402_:+_gene_2 region: 1, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\2_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.364s.\n",
      "in 57.170s.\n",
      "-- designing region: 1:20888605-20953251_:+_gene_3 region: 2, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\3_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.239s.\n",
      "in 38.684s.\n",
      "-- designing region: 1:21475446-21564433_:+_gene_4 region: 3, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\4_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "in 52.874s.\n",
      "-- designing region: 1:21674557-21793163_:+_gene_5 region: 4, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\5_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.334s.\n",
      "in 70.284s.\n",
      "-- designing region: 1:23245924-23297215_:+_gene_6 region: 5, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\6_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.201s.\n",
      "in 30.806s.\n",
      "-- designing region: 1:23918055-23999346_:+_gene_7 region: 6, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\7_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.256s.\n",
      "in 48.419s.\n",
      "-- designing region: 1:24208993-24275055_:+_gene_8 region: 7, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\8_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.237s.\n",
      "in 39.484s.\n",
      "-- designing region: 1:35485994-35596498_:+_gene_9 region: 8, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\9_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.317s.\n",
      "in 66.419s.\n",
      "-- designing region: 1:36042487-36110247_:+_gene_10 region: 9, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\10_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.241s.\n",
      "in 40.994s.\n",
      "-- designing region: 1:37043543-37132697_:+_gene_11 region: 10, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\11_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.881s.\n",
      "in 54.466s.\n",
      "-- designing region: 1:39053266-39116581_:+_gene_12 region: 11, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\12_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.045s.\n",
      "in 41.795s.\n",
      "-- designing region: 1:39720514-39784704_:+_gene_13 region: 12, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\13_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.942s.\n",
      "in 39.887s.\n",
      "-- designing region: 1:42219499-42273271_:+_gene_14 region: 13, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\14_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.231s.\n",
      "in 34.659s.\n",
      "-- designing region: 1:45847775-45927301_:+_gene_15 region: 14, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\15_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.323s.\n",
      "in 52.995s.\n",
      "-- designing region: 1:51081069-51175247_:+_gene_16 region: 15, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\16_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.326s.\n",
      "in 59.239s.\n",
      "-- designing region: 1:51570703-51660839_:+_gene_17 region: 16, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\17_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "in 59.267s.\n",
      "-- designing region: 1:52630368-52700586_:+_gene_18 region: 17, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\18_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.242s.\n",
      "in 42.172s.\n",
      "-- designing region: 1:54779256-54924777_:+_gene_19 region: 18, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\19_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.395s.\n",
      "in 86.598s.\n",
      "-- designing region: 1:55183046-55231757_:+_gene_20 region: 19, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\20_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.203s.\n",
      "in 31.470s.\n",
      "-- designing region: 1:55419953-55484150_:+_gene_21 region: 20, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\21_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.231s.\n",
      "in 38.874s.\n",
      "-- designing region: 1:55599660-55771983_:+_gene_22 region: 21, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\22_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.447s.\n",
      "in 107.911s.\n",
      "-- designing region: 1:57195401-57235355_:+_gene_23 region: 22, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\23_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.189s.\n",
      "in 24.308s.\n",
      "-- designing region: 1:57774408-57850178_:+_gene_24 region: 23, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\24_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.258s.\n",
      "in 49.171s.\n",
      "-- designing region: 1:58710248-58755177_:+_gene_25 region: 24, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\25_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.502s.\n",
      "in 28.430s.\n",
      "-- designing region: 1:60406810-60510189_:+_gene_26 region: 25, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\26_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 1.155s.\n",
      "in 70.442s.\n",
      "-- designing region: 1:60828959-60878677_:+_gene_27 region: 26, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\27_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "in 34.064s.\n",
      "-- designing region: 1:62890963-62958089_:+_gene_28 region: 27, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\28_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.615s.\n",
      "in 47.481s.\n",
      "-- designing region: 1:68028967-68171164_:+_gene_29 region: 28, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\29_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.662s.\n",
      "in 95.093s.\n",
      "-- designing region: 1:68682949-68800249_:+_gene_30 region: 29, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\30_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.422s.\n",
      "in 331.435s.\n",
      "-- designing region: 1:72404007-72475858_:+_gene_31 region: 30, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\31_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.253s.\n",
      "in 43.125s.\n",
      "-- designing region: 1:72583027-72630300_:+_gene_32 region: 31, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\32_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.201s.\n",
      "in 28.574s.\n",
      "-- designing region: 1:73522522-73603287_:+_gene_33 region: 32, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\33_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.250s.\n",
      "in 48.340s.\n",
      "-- designing region: 1:75423170-75478047_:+_gene_34 region: 33, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\34_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.225s.\n",
      "in 33.261s.\n",
      "-- designing region: 1:75854682-75920607_:+_gene_35 region: 34, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\35_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.239s.\n",
      "in 40.017s.\n",
      "-- designing region: 1:79813369-79876804_:+_gene_36 region: 35, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\36_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "in 39.036s.\n",
      "-- designing region: 1:81222704-81365836_:+_gene_37 region: 36, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\37_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.383s.\n",
      "in 84.852s.\n",
      "-- designing region: 1:82422928-82493138_:+_gene_38 region: 37, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\38_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.243s.\n",
      "in 46.763s.\n",
      "-- designing region: 1:83313920-83400588_:+_gene_39 region: 38, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\39_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "in 51.743s.\n",
      "-- designing region: 1:84168846-84261608_:+_gene_40 region: 39, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\40_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "in 55.438s.\n",
      "-- designing region: 1:86959177-87062202_:+_gene_41 region: 40, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\41_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.310s.\n",
      "in 61.595s.\n",
      "-- designing region: 1:89121460-89206528_:+_gene_42 region: 41, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\42_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "in 51.063s.\n",
      "-- designing region: 1:91412081-91466772_:+_gene_43 region: 42, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\43_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.229s.\n",
      "in 33.317s.\n",
      "-- designing region: 1:93189428-93238116_:+_gene_44 region: 43, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\44_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.211s.\n",
      "in 29.613s.\n",
      "-- designing region: 1:97604468-97662595_:+_gene_45 region: 44, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\45_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "in 35.630s.\n",
      "-- designing region: 1:105961567-106009318_:+_gene_46 region: 45, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\46_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.200s.\n",
      "in 29.070s.\n",
      "-- designing region: 1:115727054-115804837_:+_gene_47 region: 46, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\47_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.257s.\n",
      "in 46.501s.\n",
      "-- designing region: 1:119437659-119505173_:+_gene_48 region: 47, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\48_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.241s.\n",
      "in 40.537s.\n",
      "-- designing region: 1:120161583-120218334_:+_gene_49 region: 48, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\49_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.234s.\n",
      "in 39.948s.\n",
      "-- designing region: 1:121430601-121489275_:+_gene_50 region: 49, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\50_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.225s.\n",
      "in 35.819s.\n",
      "-- designing region: 1:124674853-124801140_:+_gene_51 region: 50, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\51_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.407s.\n",
      "in 77.089s.\n",
      "-- designing region: 1:137888029-137965735_:+_gene_52 region: 51, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\52_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 46.661s.\n",
      "-- designing region: 1:140354959-140541768_:+_gene_53 region: 52, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\53_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.465s.\n",
      "in 110.989s.\n",
      "-- designing region: 1:143861617-143934455_:+_gene_54 region: 53, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\54_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.247s.\n",
      "in 43.824s.\n",
      "-- designing region: 1:149846612-149908891_:+_gene_55 region: 54, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\55_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "in 37.511s.\n",
      "-- designing region: 1:152397309-152486632_:+_gene_56 region: 55, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\56_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "in 53.550s.\n",
      "-- designing region: 1:155665230-155748254_:+_gene_57 region: 56, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\57_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.314s.\n",
      "in 50.195s.\n",
      "-- designing region: 1:161214625-161297711_:+_gene_58 region: 57, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\58_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "in 49.726s.\n",
      "-- designing region: 1:162012959-162104685_:+_gene_59 region: 58, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\59_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "in 54.663s.\n",
      "-- designing region: 1:162655058-162776885_:+_gene_60 region: 59, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\60_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.342s.\n",
      "in 79.635s.\n",
      "-- designing region: 1:167034296-167100658_:+_gene_61 region: 60, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\61_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.358s.\n",
      "in 39.954s.\n",
      "-- designing region: 1:167197268-167284083_:+_gene_62 region: 61, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\62_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "in 51.878s.\n",
      "-- designing region: 1:169792098-169928411_:+_gene_63 region: 62, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\63_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.372s.\n",
      "in 81.057s.\n",
      "-- designing region: 1:171429944-171499765_:+_gene_64 region: 63, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\64_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.241s.\n",
      "in 41.884s.\n",
      "-- designing region: 1:171805726-171877501_:+_gene_65 region: 64, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\65_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.248s.\n",
      "in 43.008s.\n",
      "-- designing region: 1:173307246-173374219_:+_gene_66 region: 65, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\66_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.246s.\n",
      "in 40.319s.\n",
      "-- designing region: 1:178341436-178410984_:+_gene_67 region: 66, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\67_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.242s.\n",
      "in 41.835s.\n",
      "-- designing region: 1:179031899-179125551_:+_gene_68 region: 67, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\68_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "in 57.194s.\n",
      "-- designing region: 1:182429520-182519678_:+_gene_69 region: 68, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\69_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.302s.\n",
      "in 54.125s.\n",
      "-- designing region: 1:186617359-186706395_:+_gene_70 region: 69, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\70_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.306s.\n",
      "in 54.309s.\n",
      "-- designing region: 1:187635279-187704850_:+_gene_71 region: 70, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\71_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.258s.\n",
      "in 41.825s.\n",
      "-- designing region: 1:189007529-189076052_:+_gene_72 region: 71, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\72_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.250s.\n",
      "in 41.100s.\n",
      "-- designing region: 1:189175060-189239230_:+_gene_73 region: 72, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\73_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.234s.\n",
      "in 38.534s.\n",
      "-- designing region: 1:189480922-189543419_:+_gene_74 region: 73, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\74_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.229s.\n",
      "in 37.485s.\n",
      "-- designing region: 1:189866129-189938237_:+_gene_75 region: 74, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\75_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.241s.\n",
      "in 52.204s.\n",
      "-- designing region: 1:193075619-193133645_:+_gene_76 region: 75, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\76_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.225s.\n",
      "in 34.900s.\n",
      "-- designing region: 2:3129373-3183216_:+_gene_77 region: 76, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\77_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.215s.\n",
      "in 32.406s.\n",
      "-- designing region: 2:4026085-4083325_:+_gene_78 region: 77, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\78_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.216s.\n",
      "in 34.398s.\n",
      "-- designing region: 2:5151601-5210127_:+_gene_79 region: 78, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\79_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.269s.\n",
      "in 35.401s.\n",
      "-- designing region: 2:11309886-11385319_:+_gene_80 region: 79, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\80_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.258s.\n",
      "in 45.421s.\n",
      "-- designing region: 2:14897328-14968889_:+_gene_81 region: 80, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\81_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "in 43.399s.\n",
      "-- designing region: 2:16513406-16660259_:+_gene_82 region: 81, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\82_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.415s.\n",
      "in 87.189s.\n",
      "-- designing region: 2:17628682-17734416_:+_gene_83 region: 82, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\83_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.321s.\n",
      "in 63.122s.\n",
      "-- designing region: 2:19888794-19999195_:+_gene_84 region: 83, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\84_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.340s.\n",
      "in 68.541s.\n",
      "-- designing region: 2:21251409-21298244_:+_gene_85 region: 84, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\85_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.229s.\n",
      "in 28.403s.\n",
      "-- designing region: 2:21622583-21703613_:+_gene_86 region: 85, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\86_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.266s.\n",
      "in 48.609s.\n",
      "-- designing region: 2:25420535-25461500_:+_gene_87 region: 86, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\87_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.190s.\n",
      "in 27.473s.\n",
      "-- designing region: 2:27071925-27131679_:+_gene_88 region: 87, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\88_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.254s.\n",
      "in 40.410s.\n",
      "-- designing region: 2:30236602-30280512_:+_gene_89 region: 88, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\89_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "in 27.528s.\n",
      "-- designing region: 2:32093351-32148238_:+_gene_90 region: 89, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\90_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.239s.\n",
      "in 33.215s.\n",
      "-- designing region: 2:37953869-38034097_:+_gene_91 region: 90, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\91_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "in 48.294s.\n",
      "-- designing region: 2:41928242-42023962_:+_gene_92 region: 91, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\92_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.309s.\n",
      "in 61.731s.\n",
      "-- designing region: 2:43753833-43799050_:+_gene_93 region: 92, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\93_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.212s.\n",
      "in 27.573s.\n",
      "-- designing region: 2:44084305-44153466_:+_gene_94 region: 93, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\94_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.265s.\n",
      "in 41.495s.\n",
      "-- designing region: 2:44324153-44432010_:+_gene_95 region: 94, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\95_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.321s.\n",
      "in 75.797s.\n",
      "-- designing region: 2:49898807-49970037_:+_gene_96 region: 95, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\96_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "in 42.938s.\n",
      "-- designing region: 2:58005861-58065893_:+_gene_97 region: 96, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\97_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.257s.\n",
      "in 36.206s.\n",
      "-- designing region: 2:59634429-59685853_:+_gene_98 region: 97, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\98_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.223s.\n",
      "in 31.120s.\n",
      "-- designing region: 2:62467453-62543544_:+_gene_99 region: 98, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\99_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.260s.\n",
      "in 45.510s.\n",
      "-- designing region: 2:62670661-62738745_:+_gene_100 region: 99, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\100_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.264s.\n",
      "in 43.707s.\n",
      "-- designing region: 2:70541303-70603453_:+_gene_101 region: 100, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\101_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "in 37.612s.\n",
      "-- designing region: 2:73253634-73311570_:+_gene_102 region: 101, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\102_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.231s.\n",
      "in 35.068s.\n",
      "-- designing region: 2:74102486-74171621_:+_gene_103 region: 102, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\103_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.329s.\n",
      "in 42.428s.\n",
      "-- designing region: 2:75747159-75820879_:+_gene_104 region: 103, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\104_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "in 47.448s.\n",
      "-- designing region: 2:77013944-77091549_:+_gene_105 region: 104, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\105_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "in 49.118s.\n",
      "-- designing region: 2:77277948-77367263_:+_gene_106 region: 105, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\106_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.307s.\n",
      "in 53.479s.\n",
      "-- designing region: 2:83704396-83781664_:+_gene_107 region: 106, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\107_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "in 50.454s.\n",
      "-- designing region: 2:84515475-84567580_:+_gene_108 region: 107, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\108_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.216s.\n",
      "in 31.584s.\n",
      "-- designing region: 2:84845394-84900174_:+_gene_109 region: 108, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\109_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.213s.\n",
      "in 35.590s.\n",
      "-- designing region: 2:85037645-85079130_:+_gene_110 region: 109, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\110_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "in 28.054s.\n",
      "-- designing region: 2:85037645-85079130_:+_gene_111 region: 110, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\111_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.202s.\n",
      "in 25.423s.\n",
      "-- designing region: 2:85198480-85295653_:+_gene_112 region: 111, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\112_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.322s.\n",
      "in 60.853s.\n",
      "-- designing region: 2:91302426-91340781_:+_gene_113 region: 112, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\113_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.213s.\n",
      "in 26.108s.\n",
      "-- designing region: 2:97564926-97650442_:+_gene_114 region: 113, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\114_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.313s.\n",
      "in 57.707s.\n",
      "-- designing region: 2:110162327-110221668_:+_gene_115 region: 114, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\115_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.307s.\n",
      "in 38.879s.\n",
      "-- designing region: 2:118252753-118346305_:+_gene_116 region: 115, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\116_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.311s.\n",
      "in 58.630s.\n",
      "-- designing region: 2:120618697-120674306_:+_gene_117 region: 116, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\117_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "in 36.526s.\n",
      "-- designing region: 2:122327422-122368386_:+_gene_118 region: 117, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\118_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.187s.\n",
      "in 24.989s.\n",
      "-- designing region: 2:122548219-122636386_:+_gene_119 region: 118, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\119_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.324s.\n",
      "in 52.947s.\n",
      "-- designing region: 2:123300821-123374509_:+_gene_120 region: 119, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\120_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "in 47.846s.\n",
      "-- designing region: 2:124108515-124225589_:+_gene_121 region: 120, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\121_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.354s.\n",
      "in 73.305s.\n",
      "-- designing region: 2:127069492-127147676_:+_gene_122 region: 121, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\122_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "in 47.119s.\n",
      "-- designing region: 2:128289201-128390141_:+_gene_123 region: 122, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\123_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.340s.\n",
      "in 74.888s.\n",
      "-- designing region: 2:128694272-128817233_:+_gene_124 region: 123, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\124_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.352s.\n",
      "in 74.528s.\n",
      "-- designing region: 2:129854913-129936172_:+_gene_125 region: 124, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\125_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "in 49.068s.\n",
      "-- designing region: 2:130444939-130507012_:+_gene_126 region: 125, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\126_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.253s.\n",
      "in 37.488s.\n",
      "-- designing region: 2:131363155-131441109_:+_gene_127 region: 126, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\127_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "in 47.660s.\n",
      "-- designing region: 2:132260572-132326514_:+_gene_128 region: 127, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\128_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "in 39.907s.\n",
      "-- designing region: 2:132260572-132326514_:+_gene_129 region: 128, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\129_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.264s.\n",
      "in 39.885s.\n",
      "-- designing region: 2:135268536-135340636_:+_gene_130 region: 129, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\130_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "in 43.498s.\n",
      "-- designing region: 2:135648540-135694141_:+_gene_131 region: 130, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\131_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.207s.\n",
      "in 27.648s.\n",
      "-- designing region: 2:136179077-136262638_:+_gene_132 region: 131, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\132_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "in 50.076s.\n",
      "-- designing region: 2:139256790-139346099_:+_gene_133 region: 132, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\133_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.307s.\n",
      "in 53.615s.\n",
      "-- designing region: 2:143172015-143254256_:+_gene_134 region: 133, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\134_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.266s.\n",
      "in 51.852s.\n",
      "-- designing region: 2:143965229-144015150_:+_gene_135 region: 134, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\135_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.217s.\n",
      "in 30.460s.\n",
      "-- designing region: 2:147168730-147215713_:+_gene_136 region: 135, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\136_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.214s.\n",
      "in 28.672s.\n",
      "-- designing region: 2:148128049-148221409_:+_gene_137 region: 136, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\137_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "in 56.840s.\n",
      "-- designing region: 2:148667886-148731885_:+_gene_138 region: 137, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\138_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.239s.\n",
      "in 38.832s.\n",
      "-- designing region: 2:152668254-152718287_:+_gene_139 region: 138, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\139_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.232s.\n",
      "in 30.318s.\n",
      "-- designing region: 2:153903330-153973662_:+_gene_140 region: 139, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\140_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.256s.\n",
      "in 42.437s.\n",
      "-- designing region: 2:160197345-160260687_:+_gene_141 region: 140, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\141_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.250s.\n",
      "in 38.361s.\n",
      "-- designing region: 2:161118134-161195413_:+_gene_142 region: 141, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\142_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "in 49.919s.\n",
      "-- designing region: 2:161757252-161819816_:+_gene_143 region: 142, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\143_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.257s.\n",
      "in 37.783s.\n",
      "-- designing region: 2:162746567-162835217_:+_gene_144 region: 143, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\144_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.321s.\n",
      "in 56.305s.\n",
      "-- designing region: 2:172117465-172170675_:+_gene_145 region: 144, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\145_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.228s.\n",
      "in 34.805s.\n",
      "-- designing region: 2:172390054-172457136_:+_gene_146 region: 145, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\146_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.252s.\n",
      "in 40.522s.\n",
      "-- designing region: 2:172797494-172886579_:+_gene_147 region: 146, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\147_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "in 56.578s.\n",
      "-- designing region: 2:178118671-178177048_:+_gene_148 region: 147, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\148_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.260s.\n",
      "in 35.548s.\n",
      "-- designing region: 2:178484185-178533313_:+_gene_149 region: 148, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\149_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.216s.\n",
      "in 31.379s.\n",
      "-- designing region: 2:178654771-178739514_:+_gene_150 region: 149, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\150_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.423s.\n",
      "in 54.421s.\n",
      "-- designing region: 2:180410644-180466893_:+_gene_151 region: 150, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\151_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "in 36.889s.\n",
      "-- designing region: 3:10116470-10161129_:+_gene_152 region: 151, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\152_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.198s.\n",
      "in 27.178s.\n",
      "-- designing region: 3:15080099-15131257_:+_gene_153 region: 152, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\153_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.214s.\n",
      "in 33.706s.\n",
      "-- designing region: 3:19284209-19338497_:+_gene_154 region: 153, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\154_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.230s.\n",
      "in 34.636s.\n",
      "-- designing region: 3:19992968-20039109_:+_gene_155 region: 154, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\155_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.199s.\n",
      "in 28.023s.\n",
      "-- designing region: 3:20975804-21056576_:+_gene_156 region: 155, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\156_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "in 54.528s.\n",
      "-- designing region: 3:26804177-26864088_:+_gene_157 region: 156, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\157_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.259s.\n",
      "in 56.952s.\n",
      "-- designing region: 3:27183615-27247800_:+_gene_158 region: 157, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\158_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "in 38.727s.\n",
      "-- designing region: 3:27430699-27527522_:+_gene_159 region: 158, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\159_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.309s.\n",
      "in 58.180s.\n",
      "-- designing region: 3:27639981-27714878_:+_gene_160 region: 159, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\160_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "in 45.477s.\n",
      "-- designing region: 3:27985084-28070214_:+_gene_161 region: 160, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\161_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.322s.\n",
      "in 52.025s.\n",
      "-- designing region: 3:33340421-33427551_:+_gene_162 region: 161, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\162_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.371s.\n",
      "in 52.717s.\n",
      "-- designing region: 3:35074184-35113818_:+_gene_163 region: 162, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\163_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.204s.\n",
      "in 24.260s.\n",
      "-- designing region: 3:35313296-35364803_:+_gene_164 region: 163, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\164_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.222s.\n",
      "in 31.245s.\n",
      "-- designing region: 3:37638345-37722903_:+_gene_165 region: 164, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\165_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "in 50.968s.\n",
      "-- designing region: 3:50411173-50448666_:+_gene_166 region: 165, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\166_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.202s.\n",
      "in 22.966s.\n",
      "-- designing region: 3:50892858-50950176_:+_gene_167 region: 166, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\167_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.265s.\n",
      "in 34.644s.\n",
      "-- designing region: 3:50892858-50950176_:+_gene_168 region: 167, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\168_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.252s.\n",
      "in 34.837s.\n",
      "-- designing region: 3:53719899-53805193_:+_gene_169 region: 168, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\169_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "in 51.381s.\n",
      "-- designing region: 3:54216201-54302564_:+_gene_170 region: 169, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\170_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.329s.\n",
      "in 51.887s.\n",
      "-- designing region: 3:55041185-55109341_:+_gene_171 region: 170, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\171_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "in 41.967s.\n",
      "-- designing region: 3:57736286-57847155_:+_gene_172 region: 171, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\172_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.432s.\n",
      "in 68.237s.\n",
      "-- designing region: 3:58079664-58133121_:+_gene_173 region: 172, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\173_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.477s.\n",
      "in 33.215s.\n",
      "-- designing region: 3:59191631-59297167_:+_gene_174 region: 173, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\174_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.351s.\n",
      "in 63.252s.\n",
      "-- designing region: 3:62385566-62475321_:+_gene_175 region: 174, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\175_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "in 54.135s.\n",
      "-- designing region: 3:78938383-79045067_:+_gene_176 region: 175, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\176_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.363s.\n",
      "in 66.883s.\n",
      "-- designing region: 3:82013253-82074011_:+_gene_177 region: 176, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\177_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.261s.\n",
      "in 36.833s.\n",
      "-- designing region: 3:83926234-84003443_:+_gene_178 region: 177, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\178_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "in 50.063s.\n",
      "-- designing region: 3:85960586-86088957_:+_gene_179 region: 178, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\179_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.390s.\n",
      "in 78.884s.\n",
      "-- designing region: 3:87079865-87164939_:+_gene_180 region: 179, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\180_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.394s.\n",
      "in 52.455s.\n",
      "-- designing region: 3:87262909-87334252_:+_gene_181 region: 180, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\181_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "in 42.934s.\n",
      "-- designing region: 3:88457083-88500649_:+_gene_182 region: 181, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\182_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.246s.\n",
      "in 28.781s.\n",
      "-- designing region: 3:90073915-90138447_:+_gene_183 region: 182, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\183_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.242s.\n",
      "in 40.892s.\n",
      "-- designing region: 3:90262479-90347057_:+_gene_184 region: 183, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\184_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "in 55.674s.\n",
      "-- designing region: 3:92217911-92290171_:+_gene_185 region: 184, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\185_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.223s.\n",
      "in 45.320s.\n",
      "-- designing region: 3:95498508-95551908_:+_gene_186 region: 185, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\186_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "in 32.342s.\n",
      "-- designing region: 3:97271448-97349071_:+_gene_187 region: 186, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\187_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "in 48.842s.\n",
      "-- designing region: 3:97659551-97729267_:+_gene_188 region: 187, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\188_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "in 44.488s.\n",
      "-- designing region: 3:100383410-100488704_:+_gene_189 region: 188, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\189_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.348s.\n",
      "in 65.293s.\n",
      "-- designing region: 3:102135537-102204731_:+_gene_190 region: 189, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\190_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "in 45.762s.\n",
      "-- designing region: 3:102681535-102725439_:+_gene_191 region: 190, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\191_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.216s.\n",
      "in 26.699s.\n",
      "-- designing region: 3:103092173-103150905_:+_gene_192 region: 191, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\192_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.231s.\n",
      "in 37.860s.\n",
      "-- designing region: 3:103363744-103456515_:+_gene_193 region: 192, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\193_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.326s.\n",
      "in 55.556s.\n",
      "-- designing region: 3:104633899-104693362_:+_gene_194 region: 193, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\194_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.224s.\n",
      "in 35.839s.\n",
      "-- designing region: 3:105881717-105941481_:+_gene_195 region: 194, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\195_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.234s.\n",
      "in 39.106s.\n",
      "-- designing region: 3:109002413-109073430_:+_gene_196 region: 195, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\196_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.402s.\n",
      "in 64.810s.\n",
      "-- designing region: 3:117207508-117290078_:+_gene_197 region: 196, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\197_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.304s.\n",
      "in 50.103s.\n",
      "-- designing region: 3:117778923-117864154_:+_gene_198 region: 197, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\198_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.343s.\n",
      "in 52.833s.\n",
      "-- designing region: 3:118415335-118490496_:+_gene_199 region: 198, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\199_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.253s.\n",
      "in 47.957s.\n",
      "-- designing region: 3:125833531-125935817_:+_gene_200 region: 199, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\200_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.337s.\n",
      "in 63.682s.\n",
      "-- designing region: 3:126599799-126711558_:+_gene_201 region: 200, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\201_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.463s.\n",
      "in 71.255s.\n",
      "-- designing region: 3:127684895-127739338_:+_gene_202 region: 201, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\202_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.221s.\n",
      "in 32.957s.\n",
      "-- designing region: 3:129542117-129668198_:+_gene_203 region: 202, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\203_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.405s.\n",
      "in 80.785s.\n",
      "-- designing region: 3:130232712-130315166_:+_gene_204 region: 203, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\204_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "in 51.475s.\n",
      "-- designing region: 3:133451837-133520394_:+_gene_205 region: 204, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\205_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "in 43.389s.\n",
      "-- designing region: 3:135569252-135673396_:+_gene_206 region: 205, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\206_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.368s.\n",
      "in 64.552s.\n",
      "-- designing region: 3:137598529-137658281_:+_gene_207 region: 206, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\207_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.253s.\n",
      "in 36.792s.\n",
      "-- designing region: 3:145795719-145916046_:+_gene_208 region: 207, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\208_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.456s.\n",
      "in 74.043s.\n",
      "-- designing region: 3:146300985-146364892_:+_gene_209 region: 208, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\209_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.262s.\n",
      "in 40.770s.\n",
      "-- designing region: 3:152275792-152343485_:+_gene_210 region: 209, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\210_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "in 40.825s.\n",
      "-- designing region: 3:157139588-157226143_:+_gene_211 region: 210, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\211_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.305s.\n",
      "in 56.531s.\n",
      "-- designing region: 4:3040005-3100987_:+_gene_212 region: 211, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\212_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.239s.\n",
      "in 30.943s.\n",
      "-- designing region: 4:3471494-3540953_:+_gene_213 region: 212, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\213_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.248s.\n",
      "in 46.415s.\n",
      "-- designing region: 4:6346853-6400453_:+_gene_214 region: 213, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\214_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.245s.\n",
      "in 34.674s.\n",
      "-- designing region: 4:8546598-8621298_:+_gene_215 region: 214, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\215_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.259s.\n",
      "in 47.347s.\n",
      "-- designing region: 4:9621515-9668472_:+_gene_216 region: 215, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\216_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "in 29.887s.\n",
      "-- designing region: 4:10962894-11009200_:+_gene_217 region: 216, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\217_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.198s.\n",
      "in 30.224s.\n",
      "-- designing region: 4:11962627-12051893_:+_gene_218 region: 217, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\218_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "in 55.732s.\n",
      "-- designing region: 4:22467081-22509068_:+_gene_219 region: 218, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\219_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.224s.\n",
      "in 27.478s.\n",
      "-- designing region: 4:24745226-24835054_:+_gene_220 region: 219, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\220_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "in 56.378s.\n",
      "-- designing region: 4:24955708-25039170_:+_gene_221 region: 220, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\221_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.301s.\n",
      "in 52.528s.\n",
      "-- designing region: 4:25657294-25764933_:+_gene_222 region: 221, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\222_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.342s.\n",
      "in 70.176s.\n",
      "-- designing region: 4:28804761-28881872_:+_gene_223 region: 222, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\223_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.333s.\n",
      "in 48.535s.\n",
      "-- designing region: 4:32845518-32900800_:+_gene_224 region: 223, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\224_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.246s.\n",
      "in 37.484s.\n",
      "-- designing region: 4:33921425-33962072_:+_gene_225 region: 224, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\225_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.194s.\n",
      "in 26.747s.\n",
      "-- designing region: 4:39175238-39234393_:+_gene_226 region: 225, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\226_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.259s.\n",
      "in 37.574s.\n",
      "-- designing region: 4:40468049-40532554_:+_gene_227 region: 226, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\227_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "in 38.901s.\n",
      "-- designing region: 4:40468049-40529504_:+_gene_228 region: 227, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\228_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.238s.\n",
      "in 40.454s.\n",
      "-- designing region: 4:40762947-40854743_:+_gene_229 region: 228, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\229_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.301s.\n",
      "in 60.054s.\n",
      "-- designing region: 4:41137781-41221678_:+_gene_230 region: 229, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\230_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.305s.\n",
      "in 56.630s.\n",
      "-- designing region: 4:43195060-43271596_:+_gene_231 region: 230, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\231_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "in 48.007s.\n",
      "-- designing region: 4:43536822-43581834_:+_gene_232 region: 231, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\232_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.219s.\n",
      "in 29.509s.\n",
      "-- designing region: 4:47556560-47631977_:+_gene_233 region: 232, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\233_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "in 45.459s.\n",
      "-- designing region: 4:49711396-49823368_:+_gene_234 region: 233, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\234_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.363s.\n",
      "in 70.560s.\n",
      "-- designing region: 4:56810348-56866569_:+_gene_235 region: 234, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\235_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.231s.\n",
      "in 36.282s.\n",
      "-- designing region: 4:56953595-57059748_:+_gene_236 region: 235, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\236_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.322s.\n",
      "in 68.340s.\n",
      "-- designing region: 4:57402930-57483013_:+_gene_237 region: 236, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\237_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "in 50.726s.\n",
      "-- designing region: 4:58488835-58557154_:+_gene_238 region: 237, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\238_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.262s.\n",
      "in 43.517s.\n",
      "-- designing region: 4:62288487-62379491_:+_gene_239 region: 238, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\239_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "in 54.606s.\n",
      "-- designing region: 4:65706234-65796043_:+_gene_240 region: 239, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\240_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.318s.\n",
      "in 57.002s.\n",
      "-- designing region: 4:66351813-66425032_:+_gene_241 region: 240, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\241_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "in 44.222s.\n",
      "-- designing region: 4:70432043-70539439_:+_gene_242 region: 241, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\242_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.352s.\n",
      "in 91.697s.\n",
      "-- designing region: 4:86148471-86220141_:+_gene_243 region: 242, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\243_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "in 43.184s.\n",
      "-- designing region: 4:86991120-87073412_:+_gene_244 region: 243, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\244_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "in 49.487s.\n",
      "-- designing region: 4:88142498-88275144_:+_gene_245 region: 244, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\245_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.375s.\n",
      "in 79.274s.\n",
      "-- designing region: 4:98072752-98129663_:+_gene_246 region: 245, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\246_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.227s.\n",
      "in 38.129s.\n",
      "-- designing region: 4:98531272-98638773_:+_gene_247 region: 246, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\247_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.331s.\n",
      "in 64.279s.\n",
      "-- designing region: 4:99330446-99395864_:+_gene_248 region: 247, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\248_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.248s.\n",
      "in 39.494s.\n",
      "-- designing region: 4:100963562-101018943_:+_gene_249 region: 248, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\249_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.245s.\n",
      "in 33.527s.\n",
      "-- designing region: 4:104983954-105060480_:+_gene_250 region: 249, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\250_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "in 49.282s.\n",
      "-- designing region: 4:105899094-105952080_:+_gene_251 region: 250, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\251_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.254s.\n",
      "in 32.099s.\n",
      "-- designing region: 4:114589240-114676242_:+_gene_252 region: 251, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\252_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.331s.\n",
      "in 52.383s.\n",
      "-- designing region: 4:115037700-115112192_:+_gene_253 region: 252, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\253_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.327s.\n",
      "in 47.777s.\n",
      "-- designing region: 4:115837369-115913327_:+_gene_254 region: 253, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\254_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.301s.\n",
      "in 45.919s.\n",
      "-- designing region: 4:115837369-115913327_:+_gene_255 region: 254, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\255_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.304s.\n",
      "in 48.720s.\n",
      "-- designing region: 4:116628906-116683403_:+_gene_256 region: 255, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\256_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.245s.\n",
      "in 33.153s.\n",
      "-- designing region: 4:117618081-117687368_:+_gene_257 region: 256, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\257_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.303s.\n",
      "in 43.790s.\n",
      "-- designing region: 4:123199825-123264995_:+_gene_258 region: 257, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\258_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "in 40.490s.\n",
      "-- designing region: 4:123961238-124063077_:+_gene_259 region: 258, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\259_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.479s.\n",
      "in 62.116s.\n",
      "-- designing region: 4:124824301-124866678_:+_gene_260 region: 259, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\260_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.222s.\n",
      "in 25.874s.\n",
      "-- designing region: 4:126207621-126263017_:+_gene_261 region: 260, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\261_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.224s.\n",
      "in 36.462s.\n",
      "-- designing region: 4:126995025-127049608_:+_gene_262 region: 261, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\262_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.243s.\n",
      "in 32.957s.\n",
      "-- designing region: 4:127493306-127563000_:+_gene_263 region: 262, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\263_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "in 42.003s.\n",
      "-- designing region: 4:127666510-127718932_:+_gene_264 region: 263, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\264_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.237s.\n",
      "in 33.975s.\n",
      "-- designing region: 4:127800701-127872488_:+_gene_265 region: 264, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\265_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "in 43.276s.\n",
      "-- designing region: 4:129485757-129532804_:+_gene_266 region: 265, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\266_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "in 28.638s.\n",
      "-- designing region: 4:130236140-130307752_:+_gene_267 region: 266, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\267_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.265s.\n",
      "in 46.146s.\n",
      "-- designing region: 4:130509810-130579025_:+_gene_268 region: 267, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\268_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "in 37.767s.\n",
      "-- designing region: 4:131733908-131844723_:+_gene_269 region: 268, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\269_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.381s.\n",
      "in 73.207s.\n",
      "-- designing region: 4:132792079-132834284_:+_gene_270 region: 269, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\270_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.222s.\n",
      "in 27.771s.\n",
      "-- designing region: 4:135807186-135873467_:+_gene_271 region: 270, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\271_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.247s.\n",
      "in 40.021s.\n",
      "-- designing region: 4:137277244-137324456_:+_gene_272 region: 271, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\272_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.245s.\n",
      "in 32.540s.\n",
      "-- designing region: 4:148835951-148933937_:+_gene_273 region: 272, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\273_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.458s.\n",
      "in 61.442s.\n",
      "-- designing region: 4:155759707-155802098_:+_gene_274 region: 273, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\274_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.196s.\n",
      "in 26.002s.\n",
      "-- designing region: 5:3994952-4090244_:+_gene_275 region: 274, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\275_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.354s.\n",
      "in 61.069s.\n",
      "-- designing region: 5:4937946-5030393_:+_gene_276 region: 275, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\276_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.376s.\n",
      "in 59.539s.\n",
      "-- designing region: 5:4937946-5030393_:+_gene_277 region: 276, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\277_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.359s.\n",
      "in 55.450s.\n",
      "-- designing region: 5:5216972-5313712_:+_gene_278 region: 277, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\278_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.339s.\n",
      "in 61.335s.\n",
      "-- designing region: 5:15947115-16026399_:+_gene_279 region: 278, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\279_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "in 50.771s.\n",
      "-- designing region: 5:19894500-19961425_:+_gene_280 region: 279, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\280_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "in 43.521s.\n",
      "-- designing region: 5:21457453-21528145_:+_gene_281 region: 280, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\281_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "in 42.674s.\n",
      "-- designing region: 5:24770084-24831106_:+_gene_282 region: 281, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\282_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.237s.\n",
      "in 40.042s.\n",
      "-- designing region: 5:25594458-25634378_:+_gene_283 region: 282, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\283_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.197s.\n",
      "in 24.389s.\n",
      "-- designing region: 5:27049822-27109916_:+_gene_284 region: 283, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\284_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.234s.\n",
      "in 36.364s.\n",
      "-- designing region: 5:30381241-30452747_:+_gene_285 region: 284, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\285_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.260s.\n",
      "in 43.326s.\n",
      "-- designing region: 5:33193371-33256343_:+_gene_286 region: 285, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\286_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "in 38.932s.\n",
      "-- designing region: 5:38342152-38414629_:+_gene_287 region: 286, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\287_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.545s.\n",
      "in 47.010s.\n",
      "-- designing region: 5:38506393-38564776_:+_gene_288 region: 287, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\288_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.324s.\n",
      "in 39.365s.\n",
      "-- designing region: 5:38850470-38910029_:+_gene_289 region: 288, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\289_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.232s.\n",
      "in 35.938s.\n",
      "-- designing region: 5:39106286-39168083_:+_gene_290 region: 289, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\290_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.247s.\n",
      "in 39.530s.\n",
      "-- designing region: 5:43318583-43370939_:+_gene_291 region: 290, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\291_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.255s.\n",
      "in 31.786s.\n",
      "-- designing region: 5:44189271-44278791_:+_gene_292 region: 291, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\292_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.326s.\n",
      "in 53.670s.\n",
      "-- designing region: 5:44636554-44752497_:+_gene_293 region: 292, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\293_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.346s.\n",
      "in 72.716s.\n",
      "-- designing region: 5:44853523-44919113_:+_gene_294 region: 293, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\294_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.245s.\n",
      "in 39.527s.\n",
      "-- designing region: 5:44853523-44919113_:+_gene_295 region: 294, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\295_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.250s.\n",
      "in 39.759s.\n",
      "-- designing region: 5:49951907-50059873_:+_gene_296 region: 295, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\296_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.375s.\n",
      "in 69.326s.\n",
      "-- designing region: 5:52779669-52833675_:+_gene_297 region: 296, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\297_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.304s.\n",
      "in 34.057s.\n",
      "-- designing region: 5:58013151-58138957_:+_gene_298 region: 297, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\298_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.480s.\n",
      "in 79.172s.\n",
      "-- designing region: 5:62628998-62700814_:+_gene_299 region: 298, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\299_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "in 43.325s.\n",
      "-- designing region: 5:63805226-63916930_:+_gene_300 region: 299, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\300_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.362s.\n",
      "in 67.196s.\n",
      "-- designing region: 5:64740899-64823898_:+_gene_301 region: 300, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\301_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "in 53.600s.\n",
      "-- designing region: 5:66052841-66098951_:+_gene_302 region: 301, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\302_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.259s.\n",
      "in 60.954s.\n",
      "-- designing region: 5:67330974-67411928_:+_gene_303 region: 302, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\303_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "in 48.820s.\n",
      "-- designing region: 5:71592818-71661574_:+_gene_304 region: 303, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\304_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.265s.\n",
      "in 41.645s.\n",
      "-- designing region: 5:72807519-72901992_:+_gene_305 region: 304, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\305_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.344s.\n",
      "in 57.121s.\n",
      "-- designing region: 5:74601244-74658537_:+_gene_306 region: 305, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\306_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.239s.\n",
      "in 35.820s.\n",
      "-- designing region: 5:75625002-75694903_:+_gene_307 region: 306, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\307_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.529s.\n",
      "in 42.848s.\n",
      "-- designing region: 5:77344237-77404169_:+_gene_308 region: 307, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\308_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.230s.\n",
      "in 36.297s.\n",
      "-- designing region: 5:88554694-88611033_:+_gene_309 region: 308, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\309_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.234s.\n",
      "in 34.048s.\n",
      "-- designing region: 5:91538047-91648373_:+_gene_310 region: 309, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\310_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.350s.\n",
      "in 69.022s.\n",
      "-- designing region: 5:92331917-92408829_:+_gene_311 region: 310, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\311_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "in 46.365s.\n",
      "-- designing region: 5:104291330-104355424_:+_gene_312 region: 311, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\312_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.239s.\n",
      "in 38.772s.\n",
      "-- designing region: 5:106763128-106836267_:+_gene_313 region: 312, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\313_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.277s.\n",
      "in 43.991s.\n",
      "-- designing region: 5:108667189-108745317_:+_gene_314 region: 313, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\314_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "in 50.212s.\n",
      "-- designing region: 5:116975881-117047536_:+_gene_315 region: 314, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\315_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.262s.\n",
      "in 49.892s.\n",
      "-- designing region: 5:119037555-119110105_:+_gene_316 region: 315, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\316_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.306s.\n",
      "in 47.743s.\n",
      "-- designing region: 5:126228032-126292172_:+_gene_317 region: 316, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\317_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.315s.\n",
      "in 39.112s.\n",
      "-- designing region: 5:128181775-128254686_:+_gene_318 region: 317, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\318_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "in 47.497s.\n",
      "-- designing region: 5:129741425-129789547_:+_gene_319 region: 318, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\319_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.212s.\n",
      "in 29.290s.\n",
      "-- designing region: 5:129925623-129977156_:+_gene_320 region: 319, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\320_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.216s.\n",
      "in 32.907s.\n",
      "-- designing region: 5:132237594-132324711_:+_gene_321 region: 320, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\321_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.388s.\n",
      "in 55.731s.\n",
      "-- designing region: 5:133953020-134030324_:+_gene_322 region: 321, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\322_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.259s.\n",
      "in 46.415s.\n",
      "-- designing region: 5:135005402-135067436_:+_gene_323 region: 322, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\323_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.241s.\n",
      "in 39.151s.\n",
      "-- designing region: 5:135005402-135064026_:+_gene_324 region: 323, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\324_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.307s.\n",
      "in 37.042s.\n",
      "-- designing region: 5:138757956-138807859_:+_gene_325 region: 324, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\325_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.213s.\n",
      "in 33.051s.\n",
      "-- designing region: 5:141028058-141104622_:+_gene_326 region: 325, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\326_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "in 46.020s.\n",
      "-- designing region: 5:141735482-141804869_:+_gene_327 region: 326, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\327_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.260s.\n",
      "in 45.185s.\n",
      "-- designing region: 5:142013394-142093792_:+_gene_328 region: 327, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\328_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.337s.\n",
      "in 49.828s.\n",
      "-- designing region: 5:143460529-143531173_:+_gene_329 region: 328, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\329_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.554s.\n",
      "in 46.526s.\n",
      "-- designing region: 5:144532912-144614769_:+_gene_330 region: 329, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\330_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "in 52.229s.\n",
      "-- designing region: 5:147010488-147081113_:+_gene_331 region: 330, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\331_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "in 45.111s.\n",
      "-- designing region: 5:147010488-147081113_:+_gene_332 region: 331, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\332_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.261s.\n",
      "in 45.257s.\n",
      "-- designing region: 5:148476071-148552936_:+_gene_333 region: 332, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\333_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.267s.\n",
      "in 48.799s.\n",
      "-- designing region: 5:150087399-150179014_:+_gene_334 region: 333, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\334_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "in 57.609s.\n",
      "-- designing region: 6:4913611-5062849_:+_gene_335 region: 334, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\335_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.435s.\n",
      "in 92.243s.\n",
      "-- designing region: 6:5816895-5945849_:+_gene_336 region: 335, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\336_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.417s.\n",
      "in 81.163s.\n",
      "-- designing region: 6:6142617-6218463_:+_gene_337 region: 336, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\337_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "in 47.790s.\n",
      "-- designing region: 6:17092402-17181916_:+_gene_338 region: 337, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\338_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.543s.\n",
      "in 57.596s.\n",
      "-- designing region: 6:17574012-17640341_:+_gene_339 region: 338, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\339_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.269s.\n",
      "in 39.963s.\n",
      "-- designing region: 6:18398404-18461976_:+_gene_340 region: 339, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\340_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.234s.\n",
      "in 38.274s.\n",
      "-- designing region: 6:23436147-23521005_:+_gene_341 region: 340, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\341_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "in 53.606s.\n",
      "-- designing region: 6:24510973-24581391_:+_gene_342 region: 341, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\342_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.251s.\n",
      "in 42.295s.\n",
      "-- designing region: 6:27875819-27995930_:+_gene_343 region: 342, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\343_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.358s.\n",
      "in 77.146s.\n",
      "-- designing region: 6:29188389-29226427_:+_gene_344 region: 343, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\344_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.192s.\n",
      "in 23.238s.\n",
      "-- designing region: 6:30517033-30590432_:+_gene_345 region: 344, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\345_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "in 45.786s.\n",
      "-- designing region: 6:34471419-34525684_:+_gene_346 region: 345, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\346_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.262s.\n",
      "in 34.214s.\n",
      "-- designing region: 6:34471419-34525893_:+_gene_347 region: 346, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\347_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.227s.\n",
      "in 32.965s.\n",
      "-- designing region: 6:34856828-34912427_:+_gene_348 region: 347, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\348_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.240s.\n",
      "in 33.853s.\n",
      "-- designing region: 6:41067178-41147415_:+_gene_349 region: 348, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\349_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.332s.\n",
      "in 52.742s.\n",
      "-- designing region: 6:41494636-41570746_:+_gene_350 region: 349, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\350_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.568s.\n",
      "in 49.506s.\n",
      "-- designing region: 6:42329795-42373704_:+_gene_351 region: 350, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\351_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.204s.\n",
      "in 26.694s.\n",
      "-- designing region: 6:48379712-48444906_:+_gene_352 region: 351, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\352_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "in 42.606s.\n",
      "-- designing region: 6:49807253-49847041_:+_gene_353 region: 352, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\353_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.194s.\n",
      "in 24.233s.\n",
      "-- designing region: 6:51158260-51195983_:+_gene_354 region: 353, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\354_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.200s.\n",
      "in 22.906s.\n",
      "-- designing region: 6:53570205-53695638_:+_gene_355 region: 354, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\355_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.389s.\n",
      "in 78.549s.\n",
      "-- designing region: 6:54688853-54746443_:+_gene_356 region: 355, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\356_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.250s.\n",
      "in 38.056s.\n",
      "-- designing region: 6:55781236-55825137_:+_gene_357 region: 356, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\357_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.242s.\n",
      "in 28.155s.\n",
      "-- designing region: 6:57733581-57826248_:+_gene_358 region: 357, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\358_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.300s.\n",
      "in 59.091s.\n",
      "-- designing region: 6:72814718-72899657_:+_gene_359 region: 358, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\359_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "in 51.695s.\n",
      "-- designing region: 6:77403708-77486471_:+_gene_360 region: 359, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\360_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.396s.\n",
      "in 55.015s.\n",
      "-- designing region: 6:81772875-81833647_:+_gene_361 region: 360, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\361_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.243s.\n",
      "in 40.463s.\n",
      "-- designing region: 6:81965780-82001014_:+_gene_362 region: 361, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\362_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.362s.\n",
      "in 25.488s.\n",
      "-- designing region: 6:82986732-83034917_:+_gene_363 region: 362, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\363_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.279s.\n",
      "in 32.599s.\n",
      "-- designing region: 6:83174995-83226237_:+_gene_364 region: 363, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\364_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.211s.\n",
      "in 33.908s.\n",
      "-- designing region: 6:83847140-83910764_:+_gene_365 region: 364, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\365_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.261s.\n",
      "in 41.142s.\n",
      "-- designing region: 6:84109647-84145755_:+_gene_366 region: 365, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\366_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "in 22.095s.\n",
      "-- designing region: 6:84552937-84600580_:+_gene_367 region: 366, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\367_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "in 33.424s.\n",
      "-- designing region: 6:90393398-90445394_:+_gene_368 region: 367, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\368_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "in 31.900s.\n",
      "-- designing region: 6:97827648-97913932_:+_gene_369 region: 368, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\369_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.331s.\n",
      "in 54.795s.\n",
      "-- designing region: 6:98198965-98298774_:+_gene_370 region: 369, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\370_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.336s.\n",
      "in 68.355s.\n",
      "-- designing region: 6:98920069-99030726_:+_gene_371 region: 370, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\371_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.444s.\n",
      "in 66.457s.\n",
      "-- designing region: 6:99319695-99393737_:+_gene_372 region: 371, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\372_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "in 47.763s.\n",
      "-- designing region: 6:100901891-100949915_:+_gene_373 region: 372, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\373_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.251s.\n",
      "in 29.995s.\n",
      "-- designing region: 6:106370677-106451259_:+_gene_374 region: 373, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\374_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.589s.\n",
      "in 52.359s.\n",
      "-- designing region: 6:111471365-111546119_:+_gene_375 region: 374, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\375_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.306s.\n",
      "in 48.321s.\n",
      "-- designing region: 6:113251068-113297839_:+_gene_376 region: 375, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\376_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.228s.\n",
      "in 28.594s.\n",
      "-- designing region: 6:115452622-115533851_:+_gene_377 region: 376, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\377_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "in 53.531s.\n",
      "-- designing region: 6:116336040-116434204_:+_gene_378 region: 377, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\378_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.389s.\n",
      "in 101.059s.\n",
      "-- designing region: 6:118583580-118659770_:+_gene_379 region: 378, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\379_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "in 48.928s.\n",
      "-- designing region: 6:121356679-121478570_:+_gene_380 region: 379, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\380_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.377s.\n",
      "in 76.409s.\n",
      "-- designing region: 6:127069741-127152836_:+_gene_381 region: 380, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\381_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.304s.\n",
      "in 53.693s.\n",
      "-- designing region: 6:128391401-128443169_:+_gene_382 region: 381, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\382_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.227s.\n",
      "in 32.155s.\n",
      "-- designing region: 6:131356549-131444281_:+_gene_383 region: 382, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\383_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.503s.\n",
      "in 57.229s.\n",
      "-- designing region: 6:134686900-134773941_:+_gene_384 region: 383, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\384_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "in 55.787s.\n",
      "-- designing region: 6:137078345-137173906_:+_gene_385 region: 384, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\385_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.308s.\n",
      "in 57.437s.\n",
      "-- designing region: 6:142477481-142535279_:+_gene_386 region: 385, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\386_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.252s.\n",
      "in 34.984s.\n",
      "-- designing region: 6:144585170-144652901_:+_gene_387 region: 386, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\387_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.244s.\n",
      "in 40.779s.\n",
      "-- designing region: 6:144980272-145047109_:+_gene_388 region: 387, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\388_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "in 40.341s.\n",
      "-- designing region: 6:145265832-145317520_:+_gene_389 region: 388, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\389_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.229s.\n",
      "in 31.375s.\n",
      "-- designing region: 6:145723149-145813383_:+_gene_390 region: 389, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\390_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.301s.\n",
      "in 54.961s.\n",
      "-- designing region: 6:146320927-146368669_:+_gene_391 region: 390, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\391_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.236s.\n",
      "in 29.192s.\n",
      "-- designing region: 7:4468044-4523679_:+_gene_392 region: 391, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\392_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.236s.\n",
      "in 34.990s.\n",
      "-- designing region: 7:25614559-25661968_:+_gene_393 region: 392, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\393_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.349s.\n",
      "in 29.577s.\n",
      "-- designing region: 7:28165434-28224615_:+_gene_394 region: 393, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\394_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.258s.\n",
      "in 36.034s.\n",
      "-- designing region: 7:28549913-28611972_:+_gene_395 region: 394, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\395_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.248s.\n",
      "in 37.522s.\n",
      "-- designing region: 7:30246410-30311465_:+_gene_396 region: 395, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\396_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.239s.\n",
      "in 39.266s.\n",
      "-- designing region: 7:30885416-30938012_:+_gene_397 region: 396, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\397_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.255s.\n",
      "in 31.894s.\n",
      "-- designing region: 7:35676730-35786268_:+_gene_398 region: 397, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\398_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.333s.\n",
      "in 65.600s.\n",
      "-- designing region: 7:36035635-36095581_:+_gene_399 region: 398, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\399_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.260s.\n",
      "in 36.256s.\n",
      "-- designing region: 7:36903879-36964287_:+_gene_400 region: 399, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\400_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.236s.\n",
      "in 36.482s.\n",
      "-- designing region: 7:43533744-43582104_:+_gene_401 region: 400, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\401_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.486s.\n",
      "in 29.683s.\n",
      "-- designing region: 7:47046107-47127036_:+_gene_402 region: 401, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\402_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "in 48.667s.\n",
      "-- designing region: 7:47046107-47121005_:+_gene_403 region: 402, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\403_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "in 45.134s.\n",
      "-- designing region: 7:50493534-50595678_:+_gene_404 region: 403, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\404_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.374s.\n",
      "in 61.682s.\n",
      "-- designing region: 7:64285872-64371201_:+_gene_405 region: 404, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\405_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.352s.\n",
      "in 53.489s.\n",
      "-- designing region: 7:65860622-65945471_:+_gene_406 region: 405, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\406_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.350s.\n",
      "in 51.236s.\n",
      "-- designing region: 7:67813905-67893031_:+_gene_407 region: 406, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\407_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.322s.\n",
      "in 47.600s.\n",
      "-- designing region: 7:70316208-70366368_:+_gene_408 region: 407, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\408_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.244s.\n",
      "in 30.417s.\n",
      "-- designing region: 7:81251500-81332036_:+_gene_409 region: 408, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\409_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "in 48.299s.\n",
      "-- designing region: 7:82791317-82863443_:+_gene_410 region: 409, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\410_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "in 43.448s.\n",
      "-- designing region: 7:83034578-83105834_:+_gene_411 region: 410, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\411_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "in 43.057s.\n",
      "-- designing region: 7:83629439-83686053_:+_gene_412 region: 411, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\412_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.237s.\n",
      "in 34.929s.\n",
      "-- designing region: 7:83790534-83849178_:+_gene_413 region: 412, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\413_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.233s.\n",
      "in 35.468s.\n",
      "-- designing region: 7:84235734-84311961_:+_gene_414 region: 413, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\414_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.316s.\n",
      "in 46.052s.\n",
      "-- designing region: 7:89259710-89342531_:+_gene_415 region: 414, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\415_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.313s.\n",
      "in 51.078s.\n",
      "-- designing region: 7:97735161-97815491_:+_gene_416 region: 415, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\416_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.601s.\n",
      "in 49.296s.\n",
      "-- designing region: 7:98125903-98181642_:+_gene_417 region: 416, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\417_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.256s.\n",
      "in 33.862s.\n",
      "-- designing region: 7:101890453-101928530_:+_gene_418 region: 417, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\418_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.188s.\n",
      "in 23.248s.\n",
      "-- designing region: 7:107651264-107719048_:+_gene_419 region: 418, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\419_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.247s.\n",
      "in 40.846s.\n",
      "-- designing region: 7:109914369-109963604_:+_gene_420 region: 419, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\420_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.214s.\n",
      "in 29.810s.\n",
      "-- designing region: 7:112752685-112894620_:+_gene_421 region: 420, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\421_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.401s.\n",
      "in 84.866s.\n",
      "-- designing region: 7:114281439-114333757_:+_gene_422 region: 421, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\422_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.214s.\n",
      "in 31.672s.\n",
      "-- designing region: 7:115477765-115548476_:+_gene_423 region: 422, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\423_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.254s.\n",
      "in 42.575s.\n",
      "-- designing region: 7:115636108-115717089_:+_gene_424 region: 423, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\424_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "in 48.929s.\n",
      "-- designing region: 7:118113230-118170731_:+_gene_425 region: 424, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\425_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.256s.\n",
      "in 36.124s.\n",
      "-- designing region: 7:119716960-119755414_:+_gene_426 region: 425, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\426_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.396s.\n",
      "in 23.829s.\n",
      "-- designing region: 7:121203744-121312825_:+_gene_427 region: 426, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\427_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.394s.\n",
      "in 65.400s.\n",
      "-- designing region: 7:126238527-126290938_:+_gene_428 region: 427, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\428_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.247s.\n",
      "in 31.846s.\n",
      "-- designing region: 7:127071384-127140312_:+_gene_429 region: 428, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\429_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.307s.\n",
      "in 41.547s.\n",
      "-- designing region: 7:127626855-127710283_:+_gene_430 region: 429, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\430_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "in 50.024s.\n",
      "-- designing region: 7:130205243-130269178_:+_gene_431 region: 430, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\431_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.245s.\n",
      "in 38.669s.\n",
      "-- designing region: 7:132216605-132295072_:+_gene_432 region: 431, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\432_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "in 47.376s.\n",
      "-- designing region: 7:134653339-134690281_:+_gene_433 region: 432, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\433_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.220s.\n",
      "in 22.614s.\n",
      "-- designing region: 7:136036278-136105849_:+_gene_434 region: 433, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\434_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "in 42.200s.\n",
      "-- designing region: 7:138166035-138259068_:+_gene_435 region: 434, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\435_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.313s.\n",
      "in 58.347s.\n",
      "-- designing region: 7:142323852-142438691_:+_gene_436 region: 435, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\436_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.380s.\n",
      "in 68.934s.\n",
      "-- designing region: 8:8590019-8660327_:+_gene_437 region: 436, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\437_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.265s.\n",
      "in 42.362s.\n",
      "-- designing region: 8:13909259-13976263_:+_gene_438 region: 437, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\438_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.245s.\n",
      "in 40.305s.\n",
      "-- designing region: 8:15539842-15619200_:+_gene_439 region: 438, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\439_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "in 48.068s.\n",
      "-- designing region: 8:15984403-16043579_:+_gene_440 region: 439, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\440_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.262s.\n",
      "in 36.299s.\n",
      "-- designing region: 8:17444670-17529064_:+_gene_441 region: 440, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\441_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.326s.\n",
      "in 50.727s.\n",
      "-- designing region: 8:18906539-18964569_:+_gene_442 region: 441, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\442_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.234s.\n",
      "in 35.136s.\n",
      "-- designing region: 8:20105193-20165338_:+_gene_443 region: 442, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\443_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.252s.\n",
      "in 31.858s.\n",
      "-- designing region: 8:25338122-25379634_:+_gene_444 region: 443, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\444_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.203s.\n",
      "in 25.352s.\n",
      "-- designing region: 8:26676886-26724279_:+_gene_445 region: 444, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\445_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.240s.\n",
      "in 28.835s.\n",
      "-- designing region: 8:34805877-34903394_:+_gene_446 region: 445, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\446_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.336s.\n",
      "in 60.804s.\n",
      "-- designing region: 8:35149287-35254368_:+_gene_447 region: 446, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\447_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.351s.\n",
      "in 63.151s.\n",
      "-- designing region: 8:35358926-35444058_:+_gene_448 region: 447, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\448_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.350s.\n",
      "in 51.197s.\n",
      "-- designing region: 8:36400955-36469588_:+_gene_449 region: 448, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\449_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "in 41.336s.\n",
      "-- designing region: 8:36619739-36725384_:+_gene_450 region: 449, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\450_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.353s.\n",
      "in 63.295s.\n",
      "-- designing region: 8:40562713-40638441_:+_gene_451 region: 450, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\451_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.300s.\n",
      "in 45.612s.\n",
      "-- designing region: 8:41328066-41394762_:+_gene_452 region: 451, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\452_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "in 40.190s.\n",
      "-- designing region: 8:47670301-47717341_:+_gene_453 region: 452, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\453_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.208s.\n",
      "in 28.512s.\n",
      "-- designing region: 8:47821006-47913730_:+_gene_454 region: 453, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\454_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.317s.\n",
      "in 56.238s.\n",
      "-- designing region: 8:48176246-48232764_:+_gene_455 region: 454, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\455_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.478s.\n",
      "in 34.934s.\n",
      "-- designing region: 8:48470073-48578850_:+_gene_456 region: 455, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\456_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.347s.\n",
      "in 65.241s.\n",
      "-- designing region: 8:61239539-61344852_:+_gene_457 region: 456, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\457_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.349s.\n",
      "in 63.136s.\n",
      "-- designing region: 8:67928931-68005568_:+_gene_458 region: 457, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\458_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "in 46.146s.\n",
      "-- designing region: 8:68161685-68224740_:+_gene_459 region: 458, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\459_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.242s.\n",
      "in 38.013s.\n",
      "-- designing region: 8:68504342-68569036_:+_gene_460 region: 459, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\460_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "in 40.956s.\n",
      "-- designing region: 8:71286208-71369935_:+_gene_461 region: 460, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\461_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.319s.\n",
      "in 50.543s.\n",
      "-- designing region: 8:72632943-72708838_:+_gene_462 region: 461, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\462_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.254s.\n",
      "in 40.071s.\n",
      "-- designing region: 8:75033814-75099992_:+_gene_463 region: 462, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\463_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.251s.\n",
      "in 39.992s.\n",
      "-- designing region: 8:76990149-77104743_:+_gene_464 region: 463, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\464_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.344s.\n",
      "in 68.735s.\n",
      "-- designing region: 8:78008249-78074137_:+_gene_465 region: 464, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\465_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.264s.\n",
      "in 40.687s.\n",
      "-- designing region: 8:79321964-79402981_:+_gene_466 region: 465, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\466_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.632s.\n",
      "in 49.929s.\n",
      "-- designing region: 8:81331631-81410269_:+_gene_467 region: 466, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\467_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.324s.\n",
      "in 47.508s.\n",
      "-- designing region: 8:83392102-83461711_:+_gene_468 region: 467, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\468_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "in 42.079s.\n",
      "-- designing region: 8:84173157-84214478_:+_gene_469 region: 468, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\469_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.193s.\n",
      "in 25.190s.\n",
      "-- designing region: 8:88269593-88342980_:+_gene_470 region: 469, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\470_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "in 94.329s.\n",
      "-- designing region: 8:90263317-90334986_:+_gene_471 region: 470, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\471_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "in 43.105s.\n",
      "-- designing region: 8:90481007-90553012_:+_gene_472 region: 471, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\472_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "in 43.358s.\n",
      "-- designing region: 8:90827081-90904757_:+_gene_473 region: 472, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\473_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.291s.\n",
      "in 46.733s.\n",
      "-- designing region: 8:92109780-92144791_:+_gene_474 region: 473, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\474_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.209s.\n",
      "in 21.444s.\n",
      "-- designing region: 8:95146834-95186362_:+_gene_475 region: 474, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\475_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.186s.\n",
      "in 24.061s.\n",
      "-- designing region: 8:103822726-103892126_:+_gene_476 region: 475, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\476_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "in 41.893s.\n",
      "-- designing region: 8:104393397-104447891_:+_gene_477 region: 476, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\477_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "in 33.151s.\n",
      "-- designing region: 8:105422888-105494815_:+_gene_478 region: 477, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\478_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "in 43.790s.\n",
      "-- designing region: 8:107434145-107533024_:+_gene_479 region: 478, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\479_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.674s.\n",
      "in 60.366s.\n",
      "-- designing region: 8:109893221-109994503_:+_gene_480 region: 479, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\480_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.358s.\n",
      "in 60.743s.\n",
      "-- designing region: 8:110915709-110996339_:+_gene_481 region: 480, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\481_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "in 49.831s.\n",
      "-- designing region: 8:112606029-112692110_:+_gene_482 region: 481, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\482_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.302s.\n",
      "in 51.711s.\n",
      "-- designing region: 8:114440631-114516127_:+_gene_483 region: 482, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\483_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "in 45.575s.\n",
      "-- designing region: 8:114888227-115019038_:+_gene_484 region: 483, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\484_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.366s.\n",
      "in 78.239s.\n",
      "-- designing region: 8:115121077-115190969_:+_gene_485 region: 484, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\485_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "in 42.266s.\n",
      "-- designing region: 8:116572552-116632019_:+_gene_486 region: 485, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\486_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.261s.\n",
      "in 35.992s.\n",
      "-- designing region: 8:118085022-118172697_:+_gene_487 region: 486, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\487_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.301s.\n",
      "in 52.765s.\n",
      "-- designing region: 8:119110924-119171282_:+_gene_488 region: 487, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\488_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "in 36.675s.\n",
      "-- designing region: 8:119775828-119840110_:+_gene_489 region: 488, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\489_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.253s.\n",
      "in 39.145s.\n",
      "-- designing region: 9:2990250-3046292_:+_gene_490 region: 489, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\490_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "in 28.544s.\n",
      "-- designing region: 9:8889764-8938793_:+_gene_491 region: 490, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\491_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.304s.\n",
      "in 30.036s.\n",
      "-- designing region: 9:9083125-9155126_:+_gene_492 region: 491, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\492_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "in 43.458s.\n",
      "-- designing region: 9:14502860-14595101_:+_gene_493 region: 492, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\493_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.339s.\n",
      "in 55.514s.\n",
      "-- designing region: 9:15483272-15563906_:+_gene_494 region: 493, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\494_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.303s.\n",
      "in 48.601s.\n",
      "-- designing region: 9:22547286-22665336_:+_gene_495 region: 494, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\495_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.352s.\n",
      "in 70.483s.\n",
      "-- designing region: 9:22783499-22864592_:+_gene_496 region: 495, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\496_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.278s.\n",
      "in 48.865s.\n",
      "-- designing region: 9:24398454-24493239_:+_gene_497 region: 496, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\497_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.310s.\n",
      "in 56.839s.\n",
      "-- designing region: 9:32479082-32541627_:+_gene_498 region: 497, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\498_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "in 37.824s.\n",
      "-- designing region: 9:32691049-32769587_:+_gene_499 region: 498, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\499_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.301s.\n",
      "in 47.443s.\n",
      "-- designing region: 9:37396529-37448756_:+_gene_500 region: 499, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\500_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.253s.\n",
      "in 32.522s.\n",
      "-- designing region: 9:46291898-46364450_:+_gene_501 region: 500, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\501_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "in 47.586s.\n",
      "-- designing region: 9:48007145-48083481_:+_gene_502 region: 501, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\502_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "in 46.220s.\n",
      "-- designing region: 9:48176289-48277700_:+_gene_503 region: 502, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\503_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.349s.\n",
      "in 60.913s.\n",
      "-- designing region: 9:50054571-50160541_:+_gene_504 region: 503, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\504_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.336s.\n",
      "in 63.646s.\n",
      "-- designing region: 9:52357436-52446749_:+_gene_505 region: 504, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\505_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "in 53.695s.\n",
      "-- designing region: 9:53893431-53958595_:+_gene_506 region: 505, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\506_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.251s.\n",
      "in 39.356s.\n",
      "-- designing region: 9:54298886-54354997_:+_gene_507 region: 506, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\507_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.267s.\n",
      "in 33.972s.\n",
      "-- designing region: 9:54508565-54565386_:+_gene_508 region: 507, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\508_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.235s.\n",
      "in 34.452s.\n",
      "-- designing region: 9:54645501-54702064_:+_gene_509 region: 508, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\509_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.234s.\n",
      "in 34.245s.\n",
      "-- designing region: 9:57240601-57299305_:+_gene_510 region: 509, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\510_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.248s.\n",
      "in 35.761s.\n",
      "-- designing region: 9:63600882-63666718_:+_gene_511 region: 510, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\511_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "in 39.978s.\n",
      "-- designing region: 9:64174332-64252851_:+_gene_512 region: 511, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\512_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.327s.\n",
      "in 48.891s.\n",
      "-- designing region: 9:67385710-67451674_:+_gene_513 region: 512, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\513_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.532s.\n",
      "in 40.536s.\n",
      "-- designing region: 9:67673420-67763632_:+_gene_514 region: 513, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\514_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.338s.\n",
      "in 54.273s.\n",
      "-- designing region: 9:69074838-69145422_:+_gene_515 region: 514, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\515_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "in 42.666s.\n",
      "-- designing region: 9:71524978-71592718_:+_gene_516 region: 515, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\516_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "in 40.978s.\n",
      "-- designing region: 9:71901483-71970383_:+_gene_517 region: 516, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\517_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "in 41.666s.\n",
      "-- designing region: 9:73144502-73202707_:+_gene_518 region: 517, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\518_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.253s.\n",
      "in 35.931s.\n",
      "-- designing region: 9:77698777-77761284_:+_gene_519 region: 518, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\519_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.462s.\n",
      "in 38.276s.\n",
      "-- designing region: 9:78111826-78179455_:+_gene_520 region: 519, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\520_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "in 41.015s.\n",
      "-- designing region: 9:80159682-80266480_:+_gene_521 region: 520, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\521_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.360s.\n",
      "in 63.906s.\n",
      "-- designing region: 9:83294219-83369953_:+_gene_522 region: 521, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\522_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "in 45.601s.\n",
      "-- designing region: 9:83294219-83369953_:+_gene_523 region: 522, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\523_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.273s.\n",
      "in 45.577s.\n",
      "-- designing region: 9:86573601-86647179_:+_gene_524 region: 523, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\524_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.308s.\n",
      "in 44.609s.\n",
      "-- designing region: 9:89246295-89303992_:+_gene_525 region: 524, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\525_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "in 36.099s.\n",
      "-- designing region: 9:89831577-89911977_:+_gene_526 region: 525, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\526_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.572s.\n",
      "in 48.839s.\n",
      "-- designing region: 9:90004552-90087294_:+_gene_527 region: 526, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\527_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.339s.\n",
      "in 49.725s.\n",
      "-- designing region: 9:90175735-90277232_:+_gene_528 region: 527, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\528_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.349s.\n",
      "in 61.538s.\n",
      "-- designing region: 9:96414547-96482062_:+_gene_529 region: 528, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\529_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.266s.\n",
      "in 40.752s.\n",
      "-- designing region: 9:97889864-97985879_:+_gene_530 region: 529, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\530_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.313s.\n",
      "in 57.561s.\n",
      "-- designing region: 9:99046362-99131260_:+_gene_531 region: 530, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\531_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.307s.\n",
      "in 51.161s.\n",
      "-- designing region: 9:99753284-99818353_:+_gene_532 region: 531, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\532_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "in 40.031s.\n",
      "-- designing region: 9:102482333-102523822_:+_gene_533 region: 532, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\533_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.228s.\n",
      "in 25.422s.\n",
      "-- designing region: 9:104888780-104988138_:+_gene_534 region: 533, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\534_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.347s.\n",
      "in 59.432s.\n",
      "-- designing region: 9:105265008-105329057_:+_gene_535 region: 534, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\535_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.256s.\n",
      "in 38.578s.\n",
      "-- designing region: 9:105834542-105920743_:+_gene_536 region: 535, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\536_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "in 51.754s.\n",
      "-- designing region: 9:106914617-106985928_:+_gene_537 region: 536, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\537_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "in 42.868s.\n",
      "-- designing region: 9:108390917-108463922_:+_gene_538 region: 537, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\538_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "in 44.090s.\n",
      "-- designing region: 9:109517022-109573625_:+_gene_539 region: 538, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\539_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.227s.\n",
      "in 34.245s.\n",
      "-- designing region: 9:110403831-110459627_:+_gene_540 region: 539, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\540_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "in 33.875s.\n",
      "-- designing region: 9:110879347-110944746_:+_gene_541 region: 540, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\541_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.256s.\n",
      "in 41.404s.\n",
      "-- designing region: 9:112421698-112487152_:+_gene_542 region: 541, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\542_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "in 39.654s.\n",
      "-- designing region: 9:113760660-113813213_:+_gene_543 region: 542, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\543_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.226s.\n",
      "in 31.716s.\n",
      "-- designing region: 9:119399249-119449068_:+_gene_544 region: 543, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\544_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.210s.\n",
      "in 30.079s.\n",
      "-- designing region: 9:123421255-123491622_:+_gene_545 region: 544, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\545_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.254s.\n",
      "in 42.304s.\n",
      "-- designing region: 10:8166017-8250797_:+_gene_546 region: 545, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\546_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.293s.\n",
      "in 50.821s.\n",
      "-- designing region: 10:9791406-9903185_:+_gene_547 region: 546, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\547_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.377s.\n",
      "in 66.759s.\n",
      "-- designing region: 10:12830738-12885969_:+_gene_548 region: 547, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\548_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "in 33.682s.\n",
      "-- designing region: 10:13554012-13611581_:+_gene_549 region: 548, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\549_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.285s.\n",
      "in 35.952s.\n",
      "-- designing region: 10:19536426-19607009_:+_gene_550 region: 549, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\550_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.582s.\n",
      "in 44.249s.\n",
      "-- designing region: 10:21881312-21991778_:+_gene_551 region: 550, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\551_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.364s.\n",
      "in 66.305s.\n",
      "-- designing region: 10:23181367-23277580_:+_gene_552 region: 551, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\552_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.318s.\n",
      "in 57.948s.\n",
      "-- designing region: 10:24951345-25011868_:+_gene_553 region: 552, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\553_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "in 38.323s.\n",
      "-- designing region: 10:25400587-25476543_:+_gene_554 region: 553, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\554_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.570s.\n",
      "in 46.116s.\n",
      "-- designing region: 10:26257092-26325268_:+_gene_555 region: 554, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\555_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "in 41.155s.\n",
      "-- designing region: 10:28073265-28199418_:+_gene_556 region: 555, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\556_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.417s.\n",
      "in 75.701s.\n",
      "-- designing region: 10:34121442-34208522_:+_gene_557 region: 556, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\557_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.318s.\n",
      "in 53.961s.\n",
      "-- designing region: 10:38962790-39040119_:+_gene_558 region: 557, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\558_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.305s.\n",
      "in 46.731s.\n",
      "-- designing region: 10:40058995-40139830_:+_gene_559 region: 558, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\559_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.315s.\n",
      "in 48.605s.\n",
      "-- designing region: 10:41176005-41259634_:+_gene_560 region: 559, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\560_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "in 50.407s.\n",
      "-- designing region: 10:42547711-42602574_:+_gene_561 region: 560, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\561_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "in 35.281s.\n",
      "-- designing region: 10:43266076-43360048_:+_gene_562 region: 561, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\562_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.681s.\n",
      "in 57.094s.\n",
      "-- designing region: 10:44282271-44365306_:+_gene_563 region: 562, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\563_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.306s.\n",
      "in 49.950s.\n",
      "-- designing region: 10:44615388-44713060_:+_gene_564 region: 563, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\564_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.347s.\n",
      "in 58.756s.\n",
      "-- designing region: 10:49750898-49817819_:+_gene_565 region: 564, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\565_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "in 42.018s.\n",
      "-- designing region: 10:52692507-52771120_:+_gene_566 region: 565, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\566_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.441s.\n",
      "in 47.438s.\n",
      "-- designing region: 10:52891887-52963599_:+_gene_567 region: 566, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\567_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "in 43.117s.\n",
      "-- designing region: 10:58590997-58683875_:+_gene_568 region: 567, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\568_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.337s.\n",
      "in 55.985s.\n",
      "-- designing region: 10:61307332-61385454_:+_gene_569 region: 568, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\569_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.303s.\n",
      "in 48.724s.\n",
      "-- designing region: 10:61524277-61616208_:+_gene_570 region: 569, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\570_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.348s.\n",
      "in 55.351s.\n",
      "-- designing region: 10:62836832-62897538_:+_gene_571 region: 570, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\571_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.258s.\n",
      "in 36.782s.\n",
      "-- designing region: 10:63133174-63200019_:+_gene_572 region: 571, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\572_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.258s.\n",
      "in 40.450s.\n",
      "-- designing region: 10:64029707-64098108_:+_gene_573 region: 572, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\573_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.287s.\n",
      "in 43.499s.\n",
      "-- designing region: 10:66869118-66941818_:+_gene_574 region: 573, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\574_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.545s.\n",
      "in 44.389s.\n",
      "-- designing region: 10:67841028-67914443_:+_gene_575 region: 574, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\575_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.282s.\n",
      "in 44.193s.\n",
      "-- designing region: 10:69229620-69270356_:+_gene_576 region: 575, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\576_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.213s.\n",
      "in 24.849s.\n",
      "-- designing region: 10:71281190-71349781_:+_gene_577 region: 576, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\577_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "in 41.227s.\n",
      "-- designing region: 10:75862641-75913051_:+_gene_578 region: 577, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\578_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.220s.\n",
      "in 30.501s.\n",
      "-- designing region: 10:75862641-75913051_:+_gene_579 region: 578, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\579_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.235s.\n",
      "in 30.451s.\n",
      "-- designing region: 10:79702586-79748477_:+_gene_580 region: 579, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\580_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.227s.\n",
      "in 28.002s.\n",
      "-- designing region: 10:82628910-82692199_:+_gene_581 region: 580, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\581_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.248s.\n",
      "in 38.089s.\n",
      "-- designing region: 10:83607495-83670925_:+_gene_582 region: 581, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\582_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.244s.\n",
      "in 38.495s.\n",
      "-- designing region: 10:83837002-83953108_:+_gene_583 region: 582, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\583_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.354s.\n",
      "in 137.326s.\n",
      "-- designing region: 10:84077311-84175022_:+_gene_584 region: 583, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\584_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.333s.\n",
      "in 58.686s.\n",
      "-- designing region: 10:84921289-85033749_:+_gene_585 region: 584, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\585_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.390s.\n",
      "in 67.478s.\n",
      "-- designing region: 10:86022255-86086646_:+_gene_586 region: 585, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\586_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.362s.\n",
      "in 40.028s.\n",
      "-- designing region: 10:87475669-87514242_:+_gene_587 region: 586, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\587_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.386s.\n",
      "in 23.984s.\n",
      "-- designing region: 10:87862034-87951886_:+_gene_588 region: 587, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\588_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.387s.\n",
      "in 54.055s.\n",
      "-- designing region: 10:91171367-91217531_:+_gene_589 region: 588, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\589_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.221s.\n",
      "in 28.615s.\n",
      "-- designing region: 10:92215234-92259181_:+_gene_590 region: 589, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\590_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.224s.\n",
      "in 27.448s.\n",
      "-- designing region: 10:93638004-93714413_:+_gene_591 region: 590, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\591_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "in 45.960s.\n",
      "-- designing region: 10:94036937-94108212_:+_gene_592 region: 591, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\592_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.265s.\n",
      "in 103.809s.\n",
      "-- designing region: 10:94220759-94276806_:+_gene_593 region: 592, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\593_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.237s.\n",
      "in 33.938s.\n",
      "-- designing region: 10:96115549-96224510_:+_gene_594 region: 593, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\594_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.344s.\n",
      "in 65.491s.\n",
      "-- designing region: 10:98552540-98672759_:+_gene_595 region: 594, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\595_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.384s.\n",
      "in 74.331s.\n",
      "-- designing region: 10:98911845-98969798_:+_gene_596 region: 595, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\596_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "in 35.097s.\n",
      "-- designing region: 10:99141730-99215126_:+_gene_597 region: 596, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\597_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "in 44.103s.\n",
      "-- designing region: 10:105474481-105550782_:+_gene_598 region: 597, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\598_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "in 45.841s.\n",
      "-- designing region: 10:106575266-106742470_:+_gene_599 region: 598, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\599_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.477s.\n",
      "in 99.357s.\n",
      "-- designing region: 10:110917563-111009521_:+_gene_600 region: 599, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\600_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.332s.\n",
      "in 55.265s.\n",
      "-- designing region: 10:111525577-111633055_:+_gene_601 region: 600, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\601_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.368s.\n",
      "in 64.598s.\n",
      "-- designing region: 10:114478817-114628882_:+_gene_602 region: 601, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\602_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.443s.\n",
      "in 91.813s.\n",
      "-- designing region: 10:115360947-115403646_:+_gene_603 region: 602, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\603_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.264s.\n",
      "in 26.177s.\n",
      "-- designing region: 10:116990815-117077224_:+_gene_604 region: 603, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\604_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.321s.\n",
      "in 51.858s.\n",
      "-- designing region: 10:117659609-117705189_:+_gene_605 region: 604, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\605_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.212s.\n",
      "in 27.676s.\n",
      "-- designing region: 10:118733724-118815013_:+_gene_606 region: 605, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\606_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "in 48.707s.\n",
      "-- designing region: 10:123325185-123388689_:+_gene_607 region: 606, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\607_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.264s.\n",
      "in 38.309s.\n",
      "-- designing region: 10:125248103-125399970_:+_gene_608 region: 607, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\608_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.438s.\n",
      "in 91.249s.\n",
      "-- designing region: 10:126447909-126551128_:+_gene_609 region: 608, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\609_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.343s.\n",
      "in 61.874s.\n",
      "-- designing region: 10:128407759-128476427_:+_gene_610 region: 609, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\610_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "in 41.451s.\n",
      "-- designing region: 11:4944021-5006201_:+_gene_611 region: 610, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\611_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.272s.\n",
      "in 37.567s.\n",
      "-- designing region: 11:5336596-5447936_:+_gene_612 region: 611, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\612_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.361s.\n",
      "in 67.029s.\n",
      "-- designing region: 11:6558052-6610060_:+_gene_613 region: 612, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\613_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.241s.\n",
      "in 32.246s.\n",
      "-- designing region: 11:11160943-11253518_:+_gene_614 region: 613, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\614_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.657s.\n",
      "in 56.260s.\n",
      "-- designing region: 11:20425884-20510944_:+_gene_615 region: 614, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\615_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.308s.\n",
      "in 51.105s.\n",
      "-- designing region: 11:21983178-22041941_:+_gene_616 region: 615, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\616_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.264s.\n",
      "in 35.457s.\n",
      "-- designing region: 11:22621290-22687776_:+_gene_617 region: 616, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\617_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.253s.\n",
      "in 40.004s.\n",
      "-- designing region: 11:25838906-25913888_:+_gene_618 region: 617, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\618_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "in 44.975s.\n",
      "-- designing region: 11:26050799-26139369_:+_gene_619 region: 618, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\619_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.307s.\n",
      "in 53.243s.\n",
      "-- designing region: 11:26946039-27007404_:+_gene_620 region: 619, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\620_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "in 36.943s.\n",
      "-- designing region: 11:28645032-28711785_:+_gene_621 region: 620, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\621_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.252s.\n",
      "in 40.118s.\n",
      "-- designing region: 11:34044208-34109026_:+_gene_622 region: 621, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\622_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "in 39.009s.\n",
      "-- designing region: 11:35828462-35937732_:+_gene_623 region: 622, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\623_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.368s.\n",
      "in 65.315s.\n",
      "-- designing region: 11:36130151-36214786_:+_gene_624 region: 623, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\624_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.313s.\n",
      "in 50.936s.\n",
      "-- designing region: 11:38049705-38102695_:+_gene_625 region: 624, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\625_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.223s.\n",
      "in 32.151s.\n",
      "-- designing region: 11:40754297-40797367_:+_gene_626 region: 625, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\626_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.215s.\n",
      "in 26.871s.\n",
      "-- designing region: 11:43338561-43423182_:+_gene_627 region: 626, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\627_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.641s.\n",
      "in 51.477s.\n",
      "-- designing region: 11:43756737-43838613_:+_gene_628 region: 627, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\628_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.332s.\n",
      "in 50.503s.\n",
      "-- designing region: 11:44493963-44596487_:+_gene_629 region: 628, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\629_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.320s.\n",
      "in 61.340s.\n",
      "-- designing region: 11:46420972-46472278_:+_gene_630 region: 629, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\630_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.256s.\n",
      "in 31.036s.\n",
      "-- designing region: 11:50659370-50717733_:+_gene_631 region: 630, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\631_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.245s.\n",
      "in 35.252s.\n",
      "-- designing region: 11:52624034-52707755_:+_gene_632 region: 631, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\632_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "in 50.195s.\n",
      "-- designing region: 11:54301823-54370331_:+_gene_633 region: 632, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\633_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.290s.\n",
      "in 48.632s.\n",
      "-- designing region: 11:55130329-55205754_:+_gene_634 region: 633, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\634_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.298s.\n",
      "in 45.168s.\n",
      "-- designing region: 11:55376303-55463852_:+_gene_635 region: 634, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\635_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.319s.\n",
      "in 52.396s.\n",
      "-- designing region: 11:57657256-57770935_:+_gene_636 region: 635, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\636_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.384s.\n",
      "in 67.970s.\n",
      "-- designing region: 11:58007673-58081281_:+_gene_637 region: 636, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\637_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "in 44.206s.\n",
      "-- designing region: 11:58291211-58327455_:+_gene_638 region: 637, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\638_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.223s.\n",
      "in 22.100s.\n",
      "-- designing region: 11:62284006-62352698_:+_gene_639 region: 638, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\639_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.257s.\n",
      "in 41.169s.\n",
      "-- designing region: 11:63881715-63935692_:+_gene_640 region: 639, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\640_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.245s.\n",
      "in 32.574s.\n",
      "-- designing region: 11:67980926-68054725_:+_gene_641 region: 640, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\641_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.395s.\n",
      "in 44.599s.\n",
      "-- designing region: 11:69581649-69624548_:+_gene_642 region: 641, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\642_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.228s.\n",
      "in 26.318s.\n",
      "-- designing region: 11:70010422-70060516_:+_gene_643 region: 642, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\643_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.589s.\n",
      "in 31.175s.\n",
      "-- designing region: 11:70556947-70618824_:+_gene_644 region: 643, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\644_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.615s.\n",
      "in 38.273s.\n",
      "-- designing region: 11:70860197-70955859_:+_gene_645 region: 644, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\645_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.723s.\n",
      "in 57.542s.\n",
      "-- designing region: 11:71748129-71802987_:+_gene_646 region: 645, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\646_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.260s.\n",
      "in 33.156s.\n",
      "-- designing region: 11:72436391-72493071_:+_gene_647 region: 646, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\647_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.514s.\n",
      "in 35.552s.\n",
      "-- designing region: 11:73271570-73325274_:+_gene_648 region: 647, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\648_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.663s.\n",
      "in 33.529s.\n",
      "-- designing region: 11:78098903-78180841_:+_gene_649 region: 648, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\649_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.729s.\n",
      "in 50.829s.\n",
      "-- designing region: 11:78261740-78324241_:+_gene_650 region: 649, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\650_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.328s.\n",
      "in 39.699s.\n",
      "-- designing region: 11:84022044-84084532_:+_gene_651 region: 650, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\651_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.264s.\n",
      "in 39.715s.\n",
      "-- designing region: 11:88810839-88868107_:+_gene_652 region: 651, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\652_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.248s.\n",
      "in 34.661s.\n",
      "-- designing region: 11:89586560-89655737_:+_gene_653 region: 652, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\653_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "in 41.605s.\n",
      "-- designing region: 11:95665026-95715325_:+_gene_654 region: 653, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\654_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.251s.\n",
      "in 30.554s.\n",
      "-- designing region: 11:97109256-97171020_:+_gene_655 region: 654, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\655_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.247s.\n",
      "in 37.273s.\n",
      "-- designing region: 11:100709943-100764691_:+_gene_656 region: 655, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\656_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.248s.\n",
      "in 33.116s.\n",
      "-- designing region: 11:101999867-102041606_:+_gene_657 region: 656, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\657_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.202s.\n",
      "in 25.395s.\n",
      "-- designing region: 11:106785701-106842613_:+_gene_658 region: 657, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\658_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.250s.\n",
      "in 34.366s.\n",
      "-- designing region: 11:107015657-107057289_:+_gene_659 region: 658, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\659_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.205s.\n",
      "in 25.255s.\n",
      "-- designing region: 11:109457924-109531070_:+_gene_660 region: 659, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\660_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "in 43.923s.\n",
      "-- designing region: 11:109676821-109738725_:+_gene_661 region: 660, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\661_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "in 37.253s.\n",
      "-- designing region: 11:111501725-111548093_:+_gene_662 region: 661, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\662_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.215s.\n",
      "in 28.043s.\n",
      "-- designing region: 11:112788498-112850927_:+_gene_663 region: 662, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\663_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.240s.\n",
      "in 37.520s.\n",
      "-- designing region: 11:113651313-113714386_:+_gene_664 region: 663, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\664_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.257s.\n",
      "in 37.995s.\n",
      "-- designing region: 11:115381470-115424234_:+_gene_665 region: 664, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\665_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.208s.\n",
      "in 25.926s.\n",
      "-- designing region: 11:120969342-121034517_:+_gene_666 region: 665, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\666_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "in 39.200s.\n",
      "-- designing region: 11:121465660-121560309_:+_gene_667 region: 666, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\667_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.332s.\n",
      "in 56.675s.\n",
      "-- designing region: 12:4669994-4767519_:+_gene_668 region: 667, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\668_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.344s.\n",
      "in 58.276s.\n",
      "-- designing region: 12:8649279-8709465_:+_gene_669 region: 668, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\669_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.258s.\n",
      "in 36.218s.\n",
      "-- designing region: 12:8906293-8946352_:+_gene_670 region: 669, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\670_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.206s.\n",
      "in 24.328s.\n",
      "-- designing region: 12:11312932-11414073_:+_gene_671 region: 670, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\671_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.367s.\n",
      "in 60.420s.\n",
      "-- designing region: 12:12755678-12838753_:+_gene_672 region: 671, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\672_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "in 49.735s.\n",
      "-- designing region: 12:13069761-13155339_:+_gene_673 region: 672, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\673_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.332s.\n",
      "in 51.751s.\n",
      "-- designing region: 12:21414404-21467592_:+_gene_674 region: 673, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\674_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.252s.\n",
      "in 32.491s.\n",
      "-- designing region: 12:29527911-29582105_:+_gene_675 region: 674, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\675_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.516s.\n",
      "in 34.201s.\n",
      "-- designing region: 12:30217718-30280914_:+_gene_676 region: 675, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\676_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.251s.\n",
      "in 40.319s.\n",
      "-- designing region: 12:34420416-34527401_:+_gene_677 region: 676, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\677_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.386s.\n",
      "in 64.821s.\n",
      "-- designing region: 12:34993090-35050003_:+_gene_678 region: 677, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\678_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.274s.\n",
      "in 34.405s.\n",
      "-- designing region: 12:45004756-45093297_:+_gene_679 region: 678, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\679_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.312s.\n",
      "in 54.643s.\n",
      "-- designing region: 12:49359077-49421839_:+_gene_680 region: 679, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\680_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.899s.\n",
      "in 38.805s.\n",
      "-- designing region: 12:51689811-51761995_:+_gene_681 region: 680, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\681_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.704s.\n",
      "in 44.245s.\n",
      "-- designing region: 12:52731783-52806084_:+_gene_682 region: 681, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\682_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.304s.\n",
      "in 45.187s.\n",
      "-- designing region: 12:67084035-67208980_:+_gene_683 region: 682, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\683_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.107s.\n",
      "in 76.384s.\n",
      "-- designing region: 12:69789098-69891374_:+_gene_684 region: 683, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\684_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.345s.\n",
      "in 61.075s.\n",
      "-- designing region: 12:70857906-70924672_:+_gene_685 region: 684, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\685_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "in 40.299s.\n",
      "-- designing region: 12:72169666-72249828_:+_gene_686 region: 685, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\686_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.319s.\n",
      "in 50.295s.\n",
      "-- designing region: 12:72169666-72249828_:+_gene_687 region: 686, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\687_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.392s.\n",
      "in 48.404s.\n",
      "-- designing region: 12:73583311-73644095_:+_gene_688 region: 687, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\688_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.253s.\n",
      "in 36.981s.\n",
      "-- designing region: 12:75487290-75582550_:+_gene_689 region: 688, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\689_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.320s.\n",
      "in 57.061s.\n",
      "-- designing region: 12:83524044-83589479_:+_gene_690 region: 689, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\690_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.268s.\n",
      "in 39.421s.\n",
      "-- designing region: 12:84151919-84198789_:+_gene_691 region: 690, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\691_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.232s.\n",
      "in 28.448s.\n",
      "-- designing region: 12:84501230-84556432_:+_gene_692 region: 691, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\692_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.243s.\n",
      "in 33.290s.\n",
      "-- designing region: 12:85147576-85218677_:+_gene_693 region: 692, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\693_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.266s.\n",
      "in 42.755s.\n",
      "-- designing region: 12:86672397-86725863_:+_gene_694 region: 693, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\694_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.232s.\n",
      "in 32.288s.\n",
      "-- designing region: 12:88398166-88453232_:+_gene_695 region: 694, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\695_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.230s.\n",
      "in 33.290s.\n",
      "-- designing region: 12:89835982-89898938_:+_gene_696 region: 695, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\696_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.248s.\n",
      "in 37.843s.\n",
      "-- designing region: 12:90064487-90184183_:+_gene_697 region: 696, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\697_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.356s.\n",
      "in 72.654s.\n",
      "-- designing region: 12:103952927-104010067_:+_gene_698 region: 697, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\698_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.244s.\n",
      "in 34.587s.\n",
      "-- designing region: 12:110193166-110281004_:+_gene_699 region: 698, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\699_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.322s.\n",
      "in 52.826s.\n",
      "-- designing region: 12:110488383-110596477_:+_gene_700 region: 699, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\700_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.352s.\n",
      "in 64.772s.\n",
      "-- designing region: 12:112273397-112371328_:+_gene_701 region: 700, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\701_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.348s.\n",
      "in 58.785s.\n",
      "-- designing region: 12:113265834-113332430_:+_gene_702 region: 701, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\702_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.247s.\n",
      "in 40.175s.\n",
      "-- designing region: 12:118826228-118866588_:+_gene_703 region: 702, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\703_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.227s.\n",
      "in 24.546s.\n",
      "-- designing region: 13:6145921-6202393_:+_gene_704 region: 703, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\704_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.236s.\n",
      "in 34.334s.\n",
      "-- designing region: 13:9643963-9718881_:+_gene_705 region: 704, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\705_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "in 45.187s.\n",
      "-- designing region: 13:13784267-13837202_:+_gene_706 region: 705, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\706_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.261s.\n",
      "in 32.056s.\n",
      "-- designing region: 13:15511949-15613228_:+_gene_707 region: 706, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\707_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.359s.\n",
      "in 61.775s.\n",
      "-- designing region: 13:15969446-16034031_:+_gene_708 region: 707, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\708_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.598s.\n",
      "in 39.656s.\n",
      "-- designing region: 13:18296121-18382753_:+_gene_709 region: 708, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\709_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.748s.\n",
      "in 52.601s.\n",
      "-- designing region: 13:18790540-18886160_:+_gene_710 region: 709, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\710_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.880s.\n",
      "in 59.081s.\n",
      "-- designing region: 13:23690967-23739842_:+_gene_711 region: 710, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\711_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.508s.\n",
      "in 30.277s.\n",
      "-- designing region: 13:28582966-28640622_:+_gene_712 region: 711, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\712_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.540s.\n",
      "in 35.306s.\n",
      "-- designing region: 13:32778496-32844925_:+_gene_713 region: 712, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\713_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.566s.\n",
      "in 41.396s.\n",
      "-- designing region: 13:36247139-36325532_:+_gene_714 region: 713, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\714_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 65.004s.\n",
      "in 112.186s.\n",
      "-- designing region: 13:37810544-37982052_:+_gene_715 region: 714, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\715_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.519s.\n",
      "in 102.521s.\n",
      "-- designing region: 13:40911149-40989375_:+_gene_716 region: 715, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\716_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "in 48.123s.\n",
      "-- designing region: 13:41292633-41362801_:+_gene_717 region: 716, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\717_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.302s.\n",
      "in 42.336s.\n",
      "-- designing region: 13:43729520-43816764_:+_gene_718 region: 717, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\718_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.304s.\n",
      "in 52.394s.\n",
      "-- designing region: 13:44008093-44111293_:+_gene_719 region: 718, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\719_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.383s.\n",
      "in 61.991s.\n",
      "-- designing region: 13:44198135-44248787_:+_gene_720 region: 719, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\720_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.241s.\n",
      "in 30.761s.\n",
      "-- designing region: 13:44459166-44548478_:+_gene_721 region: 720, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\721_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.307s.\n",
      "in 164.773s.\n",
      "-- designing region: 13:45064772-45169265_:+_gene_722 region: 721, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\722_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.360s.\n",
      "in 63.284s.\n",
      "-- designing region: 13:46801717-46906339_:+_gene_723 region: 722, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\723_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.882s.\n",
      "in 64.580s.\n",
      "-- designing region: 13:52004806-52065454_:+_gene_724 region: 723, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\724_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.259s.\n",
      "in 36.861s.\n",
      "-- designing region: 13:57236124-57302330_:+_gene_725 region: 724, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\725_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.625s.\n",
      "in 41.070s.\n",
      "-- designing region: 13:59788227-59853221_:+_gene_726 region: 725, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\726_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.640s.\n",
      "in 41.286s.\n",
      "-- designing region: 13:60034633-60143990_:+_gene_727 region: 726, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\727_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.391s.\n",
      "in 69.207s.\n",
      "-- designing region: 13:60434888-60508636_:+_gene_728 region: 727, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\728_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.342s.\n",
      "in 45.586s.\n",
      "-- designing region: 13:62887747-62937194_:+_gene_729 region: 728, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\729_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.261s.\n",
      "in 30.467s.\n",
      "-- designing region: 13:64427959-64496783_:+_gene_730 region: 729, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\730_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "in 42.183s.\n",
      "-- designing region: 13:69312990-69368200_:+_gene_731 region: 730, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\731_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.239s.\n",
      "in 33.864s.\n",
      "-- designing region: 13:70852179-70914310_:+_gene_732 region: 731, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\732_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.361s.\n",
      "in 38.088s.\n",
      "-- designing region: 13:75713653-75820309_:+_gene_733 region: 732, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\733_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.346s.\n",
      "in 66.259s.\n",
      "-- designing region: 13:81123389-81170001_:+_gene_734 region: 733, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\734_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.241s.\n",
      "in 28.717s.\n",
      "-- designing region: 13:83089701-83146173_:+_gene_735 region: 734, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\735_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "in 34.459s.\n",
      "-- designing region: 13:84011657-84075757_:+_gene_736 region: 735, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\736_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.280s.\n",
      "in 39.114s.\n",
      "-- designing region: 13:88889893-89025343_:+_gene_737 region: 736, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\737_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.403s.\n",
      "in 81.479s.\n",
      "-- designing region: 13:89573484-89618121_:+_gene_738 region: 737, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\738_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.240s.\n",
      "in 27.518s.\n",
      "-- designing region: 13:97151046-97230203_:+_gene_739 region: 738, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\739_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.361s.\n",
      "in 48.633s.\n",
      "-- designing region: 13:99342904-99407764_:+_gene_740 region: 739, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\740_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "in 40.876s.\n",
      "-- designing region: 13:101198612-101249406_:+_gene_741 region: 740, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\741_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.515s.\n",
      "in 32.348s.\n",
      "-- designing region: 13:101440340-101523506_:+_gene_742 region: 741, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\742_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.773s.\n",
      "in 53.911s.\n",
      "-- designing region: 13:103252792-103337542_:+_gene_743 region: 742, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\743_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.808s.\n",
      "in 55.910s.\n",
      "-- designing region: 13:106678768-106757437_:+_gene_744 region: 743, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\744_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.471s.\n",
      "in 49.621s.\n",
      "-- designing region: 13:108046902-108146316_:+_gene_745 region: 744, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\745_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.338s.\n",
      "in 66.348s.\n",
      "-- designing region: 13:108254869-108312090_:+_gene_746 region: 745, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\746_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.956s.\n",
      "in 37.790s.\n",
      "-- designing region: 13:111373959-111432442_:+_gene_747 region: 746, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\747_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.540s.\n",
      "in 37.107s.\n",
      "-- designing region: 13:111731847-111807966_:+_gene_748 region: 747, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\748_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.698s.\n",
      "in 55.065s.\n",
      "-- designing region: 13:111919531-111990923_:+_gene_749 region: 748, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\749_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.318s.\n",
      "in 45.953s.\n",
      "-- designing region: 13:114065922-114149764_:+_gene_750 region: 749, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\750_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.360s.\n",
      "in 54.021s.\n",
      "-- designing region: 13:114065922-114149495_:+_gene_751 region: 750, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\751_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.350s.\n",
      "in 50.186s.\n",
      "-- designing region: 13:119586445-119624951_:+_gene_752 region: 751, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\752_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.215s.\n",
      "in 19.259s.\n",
      "-- designing region: 14:3298996-3368904_:+_gene_753 region: 752, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\753_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "in 43.807s.\n",
      "-- designing region: 14:3528167-3583135_:+_gene_754 region: 753, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\754_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.243s.\n",
      "in 34.035s.\n",
      "-- designing region: 14:5449405-5512217_:+_gene_755 region: 754, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\755_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.250s.\n",
      "in 38.192s.\n",
      "-- designing region: 14:5950658-6033064_:+_gene_756 region: 755, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\756_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.303s.\n",
      "in 50.449s.\n",
      "-- designing region: 14:7284561-7335660_:+_gene_757 region: 756, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\757_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.243s.\n",
      "in 31.274s.\n",
      "-- designing region: 14:13323083-13378913_:+_gene_758 region: 757, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\758_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.233s.\n",
      "in 35.173s.\n",
      "-- designing region: 14:14043804-14108999_:+_gene_759 region: 758, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\759_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.254s.\n",
      "in 41.426s.\n",
      "-- designing region: 14:15345670-15436164_:+_gene_760 region: 759, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\760_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.326s.\n",
      "in 57.232s.\n",
      "-- designing region: 14:21333204-21412833_:+_gene_761 region: 760, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\761_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.900s.\n",
      "in 52.510s.\n",
      "-- designing region: 14:21959106-22004415_:+_gene_762 region: 761, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\762_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.553s.\n",
      "in 31.653s.\n",
      "-- designing region: 14:26419598-26536458_:+_gene_763 region: 762, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\763_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.981s.\n",
      "in 75.841s.\n",
      "-- designing region: 14:29018427-29124966_:+_gene_764 region: 763, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\764_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.740s.\n",
      "in 71.871s.\n",
      "-- designing region: 14:29313773-29399124_:+_gene_765 region: 764, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\765_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.848s.\n",
      "in 55.909s.\n",
      "-- designing region: 14:31127141-31206382_:+_gene_766 region: 765, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\766_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "in 54.316s.\n",
      "-- designing region: 14:31992648-32046083_:+_gene_767 region: 766, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\767_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.539s.\n",
      "in 35.516s.\n",
      "-- designing region: 14:32517175-32620238_:+_gene_768 region: 767, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\768_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.333s.\n",
      "in 66.887s.\n",
      "-- designing region: 14:34178487-34245598_:+_gene_769 region: 768, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\769_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.301s.\n",
      "in 45.074s.\n",
      "-- designing region: 14:40891564-40969316_:+_gene_770 region: 769, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\770_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.333s.\n",
      "in 51.782s.\n",
      "-- designing region: 14:43230240-43361198_:+_gene_771 region: 770, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\771_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.394s.\n",
      "in 81.241s.\n",
      "-- designing region: 14:45529453-45635426_:+_gene_772 region: 771, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\772_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.376s.\n",
      "in 69.280s.\n",
      "-- designing region: 14:46891206-47000060_:+_gene_773 region: 772, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\773_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.372s.\n",
      "in 69.760s.\n",
      "-- designing region: 14:48145216-48234948_:+_gene_774 region: 773, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\774_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.347s.\n",
      "in 57.998s.\n",
      "-- designing region: 14:55487236-55538412_:+_gene_775 region: 774, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\775_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.254s.\n",
      "in 33.665s.\n",
      "-- designing region: 14:57498975-57575644_:+_gene_776 region: 775, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\776_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.284s.\n",
      "in 49.114s.\n",
      "-- designing region: 14:57498975-57569853_:+_gene_777 region: 776, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\777_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.303s.\n",
      "in 42.503s.\n",
      "-- designing region: 14:57669680-57747633_:+_gene_778 region: 777, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\778_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.301s.\n",
      "in 50.002s.\n",
      "-- designing region: 14:57891541-57932428_:+_gene_779 region: 778, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\779_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.232s.\n",
      "in 26.182s.\n",
      "-- designing region: 14:58064999-58123602_:+_gene_780 region: 779, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\780_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "in 37.821s.\n",
      "-- designing region: 14:59196105-59261023_:+_gene_781 region: 780, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\781_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.288s.\n",
      "in 42.369s.\n",
      "-- designing region: 14:59691594-59748067_:+_gene_782 region: 781, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\782_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.658s.\n",
      "in 37.408s.\n",
      "-- designing region: 14:61628986-61683707_:+_gene_783 region: 782, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\783_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.562s.\n",
      "in 36.957s.\n",
      "-- designing region: 14:61793570-61846789_:+_gene_784 region: 783, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\784_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.555s.\n",
      "in 35.490s.\n",
      "-- designing region: 14:62163770-62246386_:+_gene_785 region: 784, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\785_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.840s.\n",
      "in 54.696s.\n",
      "-- designing region: 14:62834246-62950869_:+_gene_786 region: 785, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\786_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.396s.\n",
      "in 76.804s.\n",
      "-- designing region: 14:63118357-63175661_:+_gene_787 region: 786, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\787_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.827s.\n",
      "in 39.117s.\n",
      "-- designing region: 14:65504499-65594282_:+_gene_788 region: 787, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\788_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.958s.\n",
      "in 61.170s.\n",
      "-- designing region: 14:65960189-65998537_:+_gene_789 region: 788, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\789_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.765s.\n",
      "in 27.743s.\n",
      "-- designing region: 14:67878001-67932629_:+_gene_790 region: 789, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\790_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.253s.\n",
      "in 36.038s.\n",
      "-- designing region: 14:69791321-69871987_:+_gene_791 region: 790, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\791_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.305s.\n",
      "in 53.614s.\n",
      "-- designing region: 14:70292991-70355449_:+_gene_792 region: 791, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\792_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.248s.\n",
      "in 41.107s.\n",
      "-- designing region: 14:75463070-75553422_:+_gene_793 region: 792, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\793_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.316s.\n",
      "in 59.825s.\n",
      "-- designing region: 14:75860410-75925995_:+_gene_794 region: 793, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\794_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.257s.\n",
      "in 42.856s.\n",
      "-- designing region: 14:76415881-76490537_:+_gene_795 region: 794, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\795_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.311s.\n",
      "in 49.546s.\n",
      "-- designing region: 14:79478685-79573260_:+_gene_796 region: 795, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\796_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.318s.\n",
      "in 61.406s.\n",
      "-- designing region: 14:79899686-79984133_:+_gene_797 region: 796, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\797_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.302s.\n",
      "in 54.065s.\n",
      "-- designing region: 14:80227149-80288631_:+_gene_798 region: 797, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\798_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "in 39.729s.\n",
      "-- designing region: 14:99526543-99602180_:+_gene_799 region: 798, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\799_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.397s.\n",
      "in 49.659s.\n",
      "-- designing region: 14:102774665-102854185_:+_gene_800 region: 799, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\800_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.818s.\n",
      "in 52.946s.\n",
      "-- designing region: 14:103099997-103173301_:+_gene_801 region: 800, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\801_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.723s.\n",
      "in 49.653s.\n",
      "-- designing region: 14:103414208-103482720_:+_gene_802 region: 801, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\802_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.722s.\n",
      "in 45.813s.\n",
      "-- designing region: 14:105264551-105344049_:+_gene_803 region: 802, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\803_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.295s.\n",
      "in 53.455s.\n",
      "-- designing region: 14:108781216-108854003_:+_gene_804 region: 803, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\804_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.029s.\n",
      "in 48.244s.\n",
      "-- designing region: 14:114844433-114923616_:+_gene_805 region: 804, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\805_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.855s.\n",
      "in 52.917s.\n",
      "-- designing region: 14:119193071-119288918_:+_gene_806 region: 805, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\806_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.788s.\n",
      "in 67.473s.\n",
      "-- designing region: 14:119621024-119697288_:+_gene_807 region: 806, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\807_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.296s.\n",
      "in 49.623s.\n",
      "-- designing region: 14:120779255-120908010_:+_gene_808 region: 807, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\808_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.432s.\n",
      "in 84.226s.\n",
      "-- designing region: 14:121918265-122090024_:+_gene_809 region: 808, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\809_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.508s.\n",
      "in 112.757s.\n",
      "-- designing region: 15:6509514-6581190_:+_gene_810 region: 809, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\810_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "in 46.368s.\n",
      "-- designing region: 15:7170479-7236680_:+_gene_811 region: 810, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\811_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "in 44.238s.\n",
      "-- designing region: 15:8375489-8461753_:+_gene_812 region: 811, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\812_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.325s.\n",
      "in 55.500s.\n",
      "-- designing region: 15:12417846-12507837_:+_gene_813 region: 812, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\813_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.318s.\n",
      "in 60.366s.\n",
      "-- designing region: 15:13071050-13175931_:+_gene_814 region: 813, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\814_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.084s.\n",
      "in 72.143s.\n",
      "-- designing region: 15:16742412-16833900_:+_gene_815 region: 814, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\815_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.961s.\n",
      "in 59.915s.\n",
      "-- designing region: 15:18847236-18992658_:+_gene_816 region: 815, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\816_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.424s.\n",
      "in 96.160s.\n",
      "-- designing region: 15:24958652-25046793_:+_gene_817 region: 816, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\817_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.325s.\n",
      "in 58.894s.\n",
      "-- designing region: 15:28171581-28259520_:+_gene_818 region: 817, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\818_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.340s.\n",
      "in 62.203s.\n",
      "-- designing region: 15:30249955-30317077_:+_gene_819 region: 818, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\819_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.325s.\n",
      "in 43.665s.\n",
      "-- designing region: 15:36478751-36528020_:+_gene_820 region: 819, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\820_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.254s.\n",
      "in 33.085s.\n",
      "-- designing region: 15:40814516-40989464_:+_gene_821 region: 820, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\821_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.499s.\n",
      "in 110.933s.\n",
      "-- designing region: 15:44711034-44779894_:+_gene_822 region: 821, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\822_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.267s.\n",
      "in 44.182s.\n",
      "-- designing region: 15:45075447-45142386_:+_gene_823 region: 822, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\823_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.283s.\n",
      "in 42.593s.\n",
      "-- designing region: 15:50654938-50758251_:+_gene_824 region: 823, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\824_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.360s.\n",
      "in 66.863s.\n",
      "-- designing region: 15:52730443-52820439_:+_gene_825 region: 824, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\825_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.084s.\n",
      "in 59.670s.\n",
      "-- designing region: 15:58158195-58230642_:+_gene_826 region: 825, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\826_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.287s.\n",
      "in 49.276s.\n",
      "-- designing region: 15:67091131-67172296_:+_gene_827 region: 826, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\827_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.850s.\n",
      "in 54.871s.\n",
      "-- designing region: 15:68684491-68749615_:+_gene_828 region: 827, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\828_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "in 42.727s.\n",
      "-- designing region: 15:68923567-69002893_:+_gene_829 region: 828, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\829_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.776s.\n",
      "in 57.185s.\n",
      "-- designing region: 15:71879586-71959467_:+_gene_830 region: 829, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\830_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.839s.\n",
      "in 53.390s.\n",
      "-- designing region: 15:76273099-76323683_:+_gene_831 region: 830, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\831_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.257s.\n",
      "in 35.904s.\n",
      "-- designing region: 15:76434091-76478590_:+_gene_832 region: 831, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\832_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.456s.\n",
      "in 32.796s.\n",
      "-- designing region: 15:80684138-80732788_:+_gene_833 region: 832, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\833_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.328s.\n",
      "in 32.899s.\n",
      "-- designing region: 15:81504063-81579541_:+_gene_834 region: 833, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\834_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "in 49.260s.\n",
      "-- designing region: 15:81672506-81729092_:+_gene_835 region: 834, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\835_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.242s.\n",
      "in 38.766s.\n",
      "-- designing region: 15:83364966-83464933_:+_gene_836 region: 835, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\836_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.017s.\n",
      "in 68.207s.\n",
      "-- designing region: 15:88479865-88547648_:+_gene_837 region: 836, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\837_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.320s.\n",
      "in 44.590s.\n",
      "-- designing region: 15:88818050-88864719_:+_gene_838 region: 837, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\838_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.265s.\n",
      "in 33.077s.\n",
      "-- designing region: 15:92180253-92287287_:+_gene_839 region: 838, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\839_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.380s.\n",
      "in 72.181s.\n",
      "-- designing region: 15:93482810-93529982_:+_gene_840 region: 839, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\840_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.342s.\n",
      "in 33.265s.\n",
      "-- designing region: 15:95034078-95096462_:+_gene_841 region: 840, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\841_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "in 41.457s.\n",
      "-- designing region: 15:96274908-96319877_:+_gene_842 region: 841, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\842_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.243s.\n",
      "in 31.703s.\n",
      "-- designing region: 15:101815997-101884700_:+_gene_843 region: 842, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\843_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.276s.\n",
      "in 45.727s.\n",
      "-- designing region: 15:103501670-103562538_:+_gene_844 region: 843, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\844_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "in 39.708s.\n",
      "-- designing region: 16:5140368-5205722_:+_gene_845 region: 844, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\845_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.269s.\n",
      "in 43.755s.\n",
      "-- designing region: 16:10331430-10395351_:+_gene_846 region: 845, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\846_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.304s.\n",
      "in 43.267s.\n",
      "-- designing region: 16:21147080-21202863_:+_gene_847 region: 846, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\847_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.246s.\n",
      "in 35.986s.\n",
      "-- designing region: 16:22756585-22854730_:+_gene_848 region: 847, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\848_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.325s.\n",
      "in 64.359s.\n",
      "-- designing region: 16:23642588-23704832_:+_gene_849 region: 848, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\849_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.260s.\n",
      "in 40.217s.\n",
      "-- designing region: 16:23942714-24001471_:+_gene_850 region: 849, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\850_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.270s.\n",
      "in 39.964s.\n",
      "-- designing region: 16:25973196-26088540_:+_gene_851 region: 850, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\851_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.371s.\n",
      "in 73.117s.\n",
      "-- designing region: 16:28583146-28642529_:+_gene_852 region: 851, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\852_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.281s.\n",
      "in 38.950s.\n",
      "-- designing region: 16:28813123-28919744_:+_gene_853 region: 852, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\853_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.144s.\n",
      "in 73.009s.\n",
      "-- designing region: 16:29118866-29189467_:+_gene_854 region: 853, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\854_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.726s.\n",
      "in 47.279s.\n",
      "-- designing region: 16:32431344-32553252_:+_gene_855 region: 854, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\855_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.231s.\n",
      "in 81.490s.\n",
      "-- designing region: 16:32643912-32726340_:+_gene_856 region: 855, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\856_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.889s.\n",
      "in 60.326s.\n",
      "-- designing region: 16:34957154-35017777_:+_gene_857 region: 856, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\857_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.819s.\n",
      "in 47.536s.\n",
      "-- designing region: 16:35723966-35794035_:+_gene_858 region: 857, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\858_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.431s.\n",
      "in 49.792s.\n",
      "-- designing region: 16:36610671-36680286_:+_gene_859 region: 858, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\859_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.291s.\n",
      "in 45.599s.\n",
      "-- designing region: 16:38150332-38259251_:+_gene_860 region: 859, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\860_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.386s.\n",
      "in 70.648s.\n",
      "-- designing region: 16:41514064-41562563_:+_gene_861 region: 860, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\861_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.251s.\n",
      "in 31.498s.\n",
      "-- designing region: 16:42245627-42341328_:+_gene_862 region: 861, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\862_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.333s.\n",
      "in 61.511s.\n",
      "-- designing region: 16:42875154-42969540_:+_gene_863 region: 862, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\863_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.343s.\n",
      "in 60.459s.\n",
      "-- designing region: 16:43912104-43983953_:+_gene_864 region: 863, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\864_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.292s.\n",
      "in 47.915s.\n",
      "-- designing region: 16:44200258-44301077_:+_gene_865 region: 864, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\865_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.359s.\n",
      "in 65.552s.\n",
      "-- designing region: 16:44200258-44301077_:+_gene_866 region: 865, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\866_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.361s.\n",
      "in 60.659s.\n",
      "-- designing region: 16:50679262-50747219_:+_gene_867 region: 866, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\867_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.899s.\n",
      "in 46.956s.\n",
      "-- designing region: 16:52068029-52157877_:+_gene_868 region: 867, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\868_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.905s.\n",
      "in 60.529s.\n",
      "-- designing region: 16:58434511-58529914_:+_gene_869 region: 868, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\869_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.045s.\n",
      "in 63.399s.\n",
      "-- designing region: 16:67559729-67628201_:+_gene_870 region: 869, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\870_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.758s.\n",
      "in 47.173s.\n",
      "-- designing region: 16:74079872-74201979_:+_gene_871 region: 870, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\871_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.691s.\n",
      "in 80.374s.\n",
      "-- designing region: 16:77374249-77459818_:+_gene_872 region: 871, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\872_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.442s.\n",
      "in 81.919s.\n",
      "-- designing region: 16:77620781-77677713_:+_gene_873 region: 872, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\873_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.368s.\n",
      "in 37.008s.\n",
      "-- designing region: 16:78553141-78609787_:+_gene_874 region: 873, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\874_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.255s.\n",
      "in 37.691s.\n",
      "-- designing region: 16:88117436-88198446_:+_gene_875 region: 874, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\875_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.313s.\n",
      "in 51.971s.\n",
      "-- designing region: 16:89507846-89608464_:+_gene_876 region: 875, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\876_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.394s.\n",
      "in 65.603s.\n",
      "-- designing region: 16:90915327-91010712_:+_gene_877 region: 876, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\877_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.355s.\n",
      "in 63.167s.\n",
      "-- designing region: 16:93967016-94031875_:+_gene_878 region: 877, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\878_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "in 43.057s.\n",
      "-- designing region: 16:94279006-94324592_:+_gene_879 region: 878, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\879_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.225s.\n",
      "in 63.164s.\n",
      "-- designing region: 16:94419141-94483370_:+_gene_880 region: 879, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\880_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 6.262s.\n",
      "in 50.046s.\n",
      "-- designing region: 16:95361862-95460042_:+_gene_881 region: 880, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\881_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.386s.\n",
      "in 66.762s.\n",
      "-- designing region: 16:97687705-97762534_:+_gene_882 region: 881, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\882_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.880s.\n",
      "in 49.682s.\n",
      "-- designing region: 17:6758025-6822752_:+_gene_883 region: 882, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\883_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.739s.\n",
      "in 43.722s.\n",
      "-- designing region: 17:11654178-11797180_:+_gene_884 region: 883, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\884_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.423s.\n",
      "in 95.668s.\n",
      "-- designing region: 17:13808723-13929026_:+_gene_885 region: 884, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\885_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 126.568s.\n",
      "in 610.312s.\n",
      "-- designing region: 17:15361839-15397147_:+_gene_886 region: 885, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\886_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.426s.\n",
      "in 25.305s.\n",
      "-- designing region: 17:23516967-23561137_:+_gene_887 region: 886, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\887_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.522s.\n",
      "in 43.893s.\n",
      "-- designing region: 17:24038288-24101203_:+_gene_888 region: 887, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\888_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.925s.\n",
      "in 45.434s.\n",
      "-- designing region: 17:31799132-31853772_:+_gene_889 region: 888, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\889_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.313s.\n",
      "in 38.144s.\n",
      "-- designing region: 17:34090951-34145951_:+_gene_890 region: 889, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\890_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.249s.\n",
      "in 39.115s.\n",
      "-- designing region: 17:35516761-35560061_:+_gene_891 region: 890, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\891_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.217s.\n",
      "in 29.755s.\n",
      "-- designing region: 17:43011066-43103508_:+_gene_892 region: 891, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\892_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.347s.\n",
      "in 61.780s.\n",
      "-- designing region: 17:44188990-44272216_:+_gene_893 region: 892, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\893_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.337s.\n",
      "in 54.577s.\n",
      "-- designing region: 17:46828396-46877205_:+_gene_894 region: 893, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\894_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.227s.\n",
      "in 33.293s.\n",
      "-- designing region: 17:48353042-48410987_:+_gene_895 region: 894, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\895_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.256s.\n",
      "in 38.181s.\n",
      "-- designing region: 17:48532940-48612895_:+_gene_896 region: 895, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\896_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.350s.\n",
      "in 53.589s.\n",
      "-- designing region: 17:48761285-48835829_:+_gene_897 region: 896, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\897_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.348s.\n",
      "in 52.129s.\n",
      "-- designing region: 17:49458027-49562568_:+_gene_898 region: 897, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\898_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.875s.\n",
      "in 71.962s.\n",
      "-- designing region: 17:50115429-50204110_:+_gene_899 region: 898, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\899_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.330s.\n",
      "in 59.835s.\n",
      "-- designing region: 17:51727923-51816643_:+_gene_900 region: 899, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\900_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.316s.\n",
      "in 59.620s.\n",
      "-- designing region: 17:53567112-53638475_:+_gene_901 region: 900, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\901_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.266s.\n",
      "in 46.629s.\n",
      "-- designing region: 17:55266212-55319734_:+_gene_902 region: 901, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\902_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "in 34.123s.\n",
      "-- designing region: 17:56872991-56939799_:+_gene_903 region: 902, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\903_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.286s.\n",
      "in 44.715s.\n",
      "-- designing region: 17:57254102-57357459_:+_gene_904 region: 903, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\904_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.379s.\n",
      "in 70.037s.\n",
      "-- designing region: 17:62630369-62740980_:+_gene_905 region: 904, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\905_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.055s.\n",
      "in 76.422s.\n",
      "-- designing region: 17:67430764-67483458_:+_gene_906 region: 905, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\906_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.610s.\n",
      "in 37.297s.\n",
      "-- designing region: 17:73209957-73282539_:+_gene_907 region: 906, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\907_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.289s.\n",
      "in 48.568s.\n",
      "-- designing region: 17:73871483-73952612_:+_gene_908 region: 907, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\908_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.329s.\n",
      "in 54.590s.\n",
      "-- designing region: 17:75432566-75547174_:+_gene_909 region: 908, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\909_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.368s.\n",
      "in 75.580s.\n",
      "-- designing region: 17:80137911-80209379_:+_gene_910 region: 909, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\910_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.671s.\n",
      "in 49.586s.\n",
      "-- designing region: 17:83530997-83606275_:+_gene_911 region: 910, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\911_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.320s.\n",
      "in 50.556s.\n",
      "-- designing region: 17:84464454-84530370_:+_gene_912 region: 911, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\912_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.288s.\n",
      "in 43.881s.\n",
      "-- designing region: 17:84850274-84953300_:+_gene_913 region: 912, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\913_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.861s.\n",
      "in 68.187s.\n",
      "-- designing region: 17:89322309-89377903_:+_gene_914 region: 913, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\914_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.275s.\n",
      "in 35.447s.\n",
      "-- designing region: 17:90044561-90126527_:+_gene_915 region: 914, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\915_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.314s.\n",
      "in 52.293s.\n",
      "-- designing region: 17:90276015-90421156_:+_gene_916 region: 915, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\916_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.433s.\n",
      "in 91.227s.\n",
      "-- designing region: 18:6929796-7011984_:+_gene_917 region: 916, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\917_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.319s.\n",
      "in 53.722s.\n",
      "-- designing region: 18:15391109-15452687_:+_gene_918 region: 917, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\918_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.271s.\n",
      "in 39.504s.\n",
      "-- designing region: 18:21151931-21203166_:+_gene_919 region: 918, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\919_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.242s.\n",
      "in 33.568s.\n",
      "-- designing region: 18:24089649-24145055_:+_gene_920 region: 919, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\920_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.635s.\n",
      "in 37.448s.\n",
      "-- designing region: 18:24408559-24500784_:+_gene_921 region: 920, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\921_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.821s.\n",
      "in 60.818s.\n",
      "-- designing region: 18:24712942-24888570_:+_gene_922 region: 921, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\922_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.506s.\n",
      "in 114.066s.\n",
      "-- designing region: 18:31929025-31995900_:+_gene_923 region: 922, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\923_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.317s.\n",
      "in 46.970s.\n",
      "-- designing region: 18:32936362-33016449_:+_gene_924 region: 923, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\924_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.376s.\n",
      "in 52.384s.\n",
      "-- designing region: 18:32936362-33016449_:+_gene_925 region: 924, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\925_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.329s.\n",
      "in 47.998s.\n",
      "-- designing region: 18:34069958-34139601_:+_gene_926 region: 925, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\926_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.310s.\n",
      "in 45.833s.\n",
      "-- designing region: 18:35472165-35550427_:+_gene_927 region: 926, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\927_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "in 51.110s.\n",
      "-- designing region: 18:36681158-36730587_:+_gene_928 region: 927, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\928_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.296s.\n",
      "in 32.854s.\n",
      "-- designing region: 18:37139186-37201875_:+_gene_929 region: 928, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\929_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.832s.\n",
      "in 41.889s.\n",
      "-- designing region: 18:39886744-39952821_:+_gene_930 region: 929, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\930_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.481s.\n",
      "in 43.141s.\n",
      "-- designing region: 18:41736971-41828086_:+_gene_931 region: 930, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\931_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.012s.\n",
      "in 59.036s.\n",
      "-- designing region: 18:42055872-42140040_:+_gene_932 region: 931, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\932_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.327s.\n",
      "in 54.444s.\n",
      "-- designing region: 18:45568608-45662084_:+_gene_933 region: 932, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\933_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.349s.\n",
      "in 59.944s.\n",
      "-- designing region: 18:46169765-46214913_:+_gene_934 region: 933, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\934_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.242s.\n",
      "in 30.297s.\n",
      "-- designing region: 18:49582614-49668484_:+_gene_935 region: 934, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\935_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.330s.\n",
      "in 54.867s.\n",
      "-- designing region: 18:50001602-50079289_:+_gene_936 region: 935, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\936_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.337s.\n",
      "in 51.290s.\n",
      "-- designing region: 18:53219139-53335927_:+_gene_937 region: 936, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\937_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.422s.\n",
      "in 75.989s.\n",
      "-- designing region: 18:55993891-56082519_:+_gene_938 region: 937, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\938_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.351s.\n",
      "in 57.205s.\n",
      "-- designing region: 18:57239664-57307027_:+_gene_939 region: 938, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\939_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.337s.\n",
      "in 44.406s.\n",
      "-- designing region: 18:59180021-59350310_:+_gene_940 region: 939, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\940_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.237s.\n",
      "in 109.575s.\n",
      "-- designing region: 18:63719174-63787284_:+_gene_941 region: 940, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\941_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.717s.\n",
      "in 46.442s.\n",
      "-- designing region: 18:64575603-64648451_:+_gene_942 region: 941, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\942_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.319s.\n",
      "in 47.667s.\n",
      "-- designing region: 18:65843454-65945591_:+_gene_943 region: 942, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\943_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.368s.\n",
      "in 68.480s.\n",
      "-- designing region: 18:68933328-69000228_:+_gene_944 region: 943, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\944_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "in 44.799s.\n",
      "-- designing region: 18:74461676-74530383_:+_gene_945 region: 944, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\945_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.043s.\n",
      "in 46.771s.\n",
      "-- designing region: 18:74669677-74751625_:+_gene_946 region: 945, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\946_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.851s.\n",
      "in 55.395s.\n",
      "-- designing region: 18:77958807-78042826_:+_gene_947 region: 946, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\947_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.318s.\n",
      "in 55.322s.\n",
      "-- designing region: 18:80950121-81002541_:+_gene_948 region: 947, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\948_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.266s.\n",
      "in 36.743s.\n",
      "-- designing region: 18:85278941-85379839_:+_gene_949 region: 948, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\949_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.342s.\n",
      "in 63.503s.\n",
      "-- designing region: 19:3625409-3709978_:+_gene_950 region: 949, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\950_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.331s.\n",
      "in 55.473s.\n",
      "-- designing region: 19:3822930-3906324_:+_gene_951 region: 950, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\951_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.301s.\n",
      "in 53.921s.\n",
      "-- designing region: 19:5638442-5685296_:+_gene_952 region: 951, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\952_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.234s.\n",
      "in 32.255s.\n",
      "-- designing region: 19:6276522-6315238_:+_gene_953 region: 952, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\953_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.229s.\n",
      "in 26.461s.\n",
      "-- designing region: 19:7054548-7207140_:+_gene_954 region: 953, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\954_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.457s.\n",
      "in 101.753s.\n",
      "-- designing region: 19:8544284-8597137_:+_gene_955 region: 954, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\955_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.246s.\n",
      "in 34.171s.\n",
      "-- designing region: 19:10685617-10740864_:+_gene_956 region: 955, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\956_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.492s.\n",
      "in 37.744s.\n",
      "-- designing region: 19:10930147-10970163_:+_gene_957 region: 956, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\957_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.767s.\n",
      "in 27.449s.\n",
      "-- designing region: 19:12424826-12504327_:+_gene_958 region: 957, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\958_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.869s.\n",
      "in 52.487s.\n",
      "-- designing region: 19:15780510-15825819_:+_gene_959 region: 958, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\959_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.449s.\n",
      "in 31.167s.\n",
      "-- designing region: 19:17296782-17355629_:+_gene_960 region: 959, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\960_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.318s.\n",
      "in 40.586s.\n",
      "-- designing region: 19:20518919-20570378_:+_gene_961 region: 960, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\961_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.399s.\n",
      "in 35.167s.\n",
      "-- designing region: 19:21713365-21757827_:+_gene_962 region: 961, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\962_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.231s.\n",
      "in 29.215s.\n",
      "-- designing region: 19:23553461-23620199_:+_gene_963 region: 962, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\963_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.315s.\n",
      "in 42.700s.\n",
      "-- designing region: 19:24600840-24687705_:+_gene_964 region: 963, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\964_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.329s.\n",
      "in 56.232s.\n",
      "-- designing region: 19:24944141-25013094_:+_gene_965 region: 964, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\965_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.297s.\n",
      "in 44.570s.\n",
      "-- designing region: 19:27524144-27591990_:+_gene_966 region: 965, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\966_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.299s.\n",
      "in 43.434s.\n",
      "-- designing region: 19:28355825-28409866_:+_gene_967 region: 966, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\967_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.277s.\n",
      "in 35.041s.\n",
      "-- designing region: 19:28563435-28646484_:+_gene_968 region: 967, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\968_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.335s.\n",
      "in 54.136s.\n",
      "-- designing region: 19:29856927-29928435_:+_gene_969 region: 968, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\969_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.308s.\n",
      "in 45.762s.\n",
      "-- designing region: 19:32895563-32971276_:+_gene_970 region: 969, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\970_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.320s.\n",
      "in 48.896s.\n",
      "-- designing region: 19:33213942-33295336_:+_gene_971 region: 970, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\971_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.317s.\n",
      "in 52.396s.\n",
      "-- designing region: 19:36172931-36275082_:+_gene_972 region: 971, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\972_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.165s.\n",
      "in 67.166s.\n",
      "-- designing region: 19:36718674-36759528_:+_gene_973 region: 972, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\973_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.606s.\n",
      "in 27.355s.\n",
      "-- designing region: 19:37472495-37520209_:+_gene_974 region: 973, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\974_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.574s.\n",
      "in 32.079s.\n",
      "-- designing region: 19:37965494-38041498_:+_gene_975 region: 974, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\975_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.771s.\n",
      "in 50.489s.\n",
      "-- designing region: 19:41050775-41097097_:+_gene_976 region: 975, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\976_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.680s.\n",
      "in 32.245s.\n",
      "-- designing region: 19:41281381-41355899_:+_gene_977 region: 976, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\977_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.112s.\n",
      "in 56.110s.\n",
      "-- designing region: 19:41963511-42036554_:+_gene_978 region: 977, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\978_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.393s.\n",
      "in 48.347s.\n",
      "-- designing region: 19:45861417-45967203_:+_gene_979 region: 978, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\979_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.391s.\n",
      "in 67.159s.\n",
      "-- designing region: 19:47634250-47669784_:+_gene_980 region: 979, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\980_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.209s.\n",
      "in 23.553s.\n",
      "-- designing region: 19:48639897-48747838_:+_gene_981 region: 980, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\981_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.407s.\n",
      "in 68.094s.\n",
      "-- designing region: 19:58794802-58866457_:+_gene_982 region: 981, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\982_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "in 47.694s.\n",
      "-- designing region: 19:59015717-59063288_:+_gene_983 region: 982, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\983_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.226s.\n",
      "in 30.771s.\n",
      "-- designing region: 19:59369976-59445429_:+_gene_984 region: 983, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\984_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.294s.\n",
      "in 48.469s.\n",
      "-- designing region: X:93271165-93309819_:+_gene_985 region: 984, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\985_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.240s.\n",
      "in 27.278s.\n",
      "-- designing region: X:98979231-99088826_:+_gene_986 region: 985, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\986_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.067s.\n",
      "in 71.822s.\n",
      "-- designing region: X:101910335-102015609_:+_gene_987 region: 986, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\987_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 1.103s.\n",
      "in 71.553s.\n",
      "-- designing region: X:136772307-136851006_:+_gene_988 region: 987, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\988_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.898s.\n",
      "in 51.622s.\n",
      "-- designing region: X:140481839-140547985_:+_gene_989 region: 988, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\989_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- finish map_self_sequences in 0.267s.\n",
      "in 43.546s.\n",
      "-- designing region: X:167122110-167191320_:+_gene_990 region: 989, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\990_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.798s.\n",
      "in 44.749s.\n",
      "-- designing region: X:169962881-170032477_:+_gene_991 region: 990, input file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\sequences\\991_reg_1.fasta\n",
      "-- setting attribute: map_self_sequences\n",
      "- Mapping no. of seqs: 1\n",
      "--- finish map_self_sequences in 0.263s.\n",
      "in 40.320s.\n",
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\reports\\merged_probes.pbr\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "pb_designer.computeOTmaps() # load the tables \n",
    "pb_designer.compute_pb_report() # design candidate probes"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### check probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 146,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:0 1:9600471-9656458_:+_gene_1, 109986 candidate probes\n",
      "--- 41731 probes passed check_dic selection.\n",
      "finish in 4.106s, 660 probes kept.\n",
      "-- check region:1 1:9749317-9845402_:+_gene_2, 181558 candidate probes\n",
      "--- 48238 probes passed check_dic selection.\n",
      "finish in 5.606s, 865 probes kept.\n",
      "-- check region:2 1:20888605-20953251_:+_gene_3, 126162 candidate probes\n",
      "--- 50944 probes passed check_dic selection.\n",
      "finish in 4.881s, 814 probes kept.\n",
      "-- check region:3 1:21475446-21564433_:+_gene_4, 171916 candidate probes\n",
      "--- 46142 probes passed check_dic selection.\n",
      "finish in 5.386s, 809 probes kept.\n",
      "-- check region:4 1:21674557-21793163_:+_gene_5, 226476 candidate probes\n",
      "--- 59478 probes passed check_dic selection.\n",
      "finish in 6.990s, 1057 probes kept.\n",
      "-- check region:5 1:23245924-23297215_:+_gene_6, 98478 candidate probes\n",
      "--- 30178 probes passed check_dic selection.\n",
      "finish in 3.264s, 525 probes kept.\n",
      "-- check region:6 1:23918055-23999346_:+_gene_7, 155324 candidate probes\n",
      "--- 47834 probes passed check_dic selection.\n",
      "finish in 5.169s, 799 probes kept.\n",
      "-- check region:7 1:24208993-24275055_:+_gene_8, 129970 candidate probes\n",
      "--- 48323 probes passed check_dic selection.\n",
      "finish in 4.746s, 788 probes kept.\n",
      "-- check region:8 1:35485994-35596498_:+_gene_9, 208534 candidate probes\n",
      "--- 68500 probes passed check_dic selection.\n",
      "finish in 7.118s, 1078 probes kept.\n",
      "-- check region:9 1:36042487-36110247_:+_gene_10, 132650 candidate probes\n",
      "--- 61261 probes passed check_dic selection.\n",
      "finish in 5.401s, 873 probes kept.\n",
      "-- check region:10 1:37043543-37132697_:+_gene_11, 175040 candidate probes\n",
      "--- 74574 probes passed check_dic selection.\n",
      "finish in 6.723s, 1078 probes kept.\n",
      "-- check region:11 1:39053266-39116581_:+_gene_12, 118582 candidate probes\n",
      "--- 48405 probes passed check_dic selection.\n",
      "finish in 4.409s, 691 probes kept.\n",
      "-- check region:12 1:39720514-39784704_:+_gene_13, 115744 candidate probes\n",
      "--- 50761 probes passed check_dic selection.\n",
      "finish in 4.579s, 720 probes kept.\n",
      "-- check region:13 1:42219499-42273271_:+_gene_14, 105206 candidate probes\n",
      "--- 44565 probes passed check_dic selection.\n",
      "finish in 4.028s, 665 probes kept.\n",
      "-- check region:14 1:45847775-45927301_:+_gene_15, 156610 candidate probes\n",
      "--- 44949 probes passed check_dic selection.\n",
      "finish in 5.075s, 817 probes kept.\n",
      "-- check region:15 1:51081069-51175247_:+_gene_16, 182041 candidate probes\n",
      "--- 61273 probes passed check_dic selection.\n",
      "finish in 6.275s, 1010 probes kept.\n",
      "-- check region:16 1:51570703-51660839_:+_gene_17, 173706 candidate probes\n",
      "--- 69469 probes passed check_dic selection.\n",
      "finish in 6.397s, 1054 probes kept.\n",
      "-- check region:17 1:52630368-52700586_:+_gene_18, 133074 candidate probes\n",
      "--- 44259 probes passed check_dic selection.\n",
      "finish in 4.585s, 743 probes kept.\n",
      "-- check region:18 1:54779256-54924777_:+_gene_19, 267050 candidate probes\n",
      "--- 114580 probes passed check_dic selection.\n",
      "finish in 10.244s, 1651 probes kept.\n",
      "-- check region:19 1:55183046-55231757_:+_gene_20, 93644 candidate probes\n",
      "--- 32399 probes passed check_dic selection.\n",
      "finish in 3.309s, 501 probes kept.\n",
      "-- check region:20 1:55419953-55484150_:+_gene_21, 105662 candidate probes\n",
      "--- 33699 probes passed check_dic selection.\n",
      "finish in 3.639s, 549 probes kept.\n",
      "-- check region:21 1:55599660-55771983_:+_gene_22, 336590 candidate probes\n",
      "--- 121084 probes passed check_dic selection.\n",
      "finish in 11.748s, 1921 probes kept.\n",
      "-- check region:22 1:57195401-57235355_:+_gene_23, 76302 candidate probes\n",
      "--- 30406 probes passed check_dic selection.\n",
      "finish in 2.885s, 473 probes kept.\n",
      "-- check region:23 1:57774408-57850178_:+_gene_24, 150730 candidate probes\n",
      "--- 68953 probes passed check_dic selection.\n",
      "finish in 6.060s, 983 probes kept.\n",
      "-- check region:24 1:58710248-58755177_:+_gene_25, 84392 candidate probes\n",
      "--- 32709 probes passed check_dic selection.\n",
      "finish in 3.214s, 516 probes kept.\n",
      "-- check region:25 1:60406810-60510189_:+_gene_26, 201366 candidate probes\n",
      "--- 47804 probes passed check_dic selection.\n",
      "finish in 5.982s, 977 probes kept.\n",
      "-- check region:26 1:60828959-60878677_:+_gene_27, 97898 candidate probes\n",
      "--- 33084 probes passed check_dic selection.\n",
      "finish in 3.427s, 526 probes kept.\n",
      "-- check region:27 1:62890963-62958089_:+_gene_28, 132832 candidate probes\n",
      "--- 57107 probes passed check_dic selection.\n",
      "finish in 5.060s, 808 probes kept.\n",
      "-- check region:28 1:68028967-68171164_:+_gene_29, 268855 candidate probes\n",
      "--- 70690 probes passed check_dic selection.\n",
      "finish in 8.112s, 1345 probes kept.\n",
      "-- check region:29 1:68682949-68800249_:+_gene_30, 229212 candidate probes\n",
      "--- 54133 probes passed check_dic selection.\n",
      "finish in 7.547s, 1023 probes kept.\n",
      "-- check region:30 1:72404007-72475858_:+_gene_31, 141568 candidate probes\n",
      "--- 66838 probes passed check_dic selection.\n",
      "finish in 6.551s, 930 probes kept.\n",
      "-- check region:31 1:72583027-72630300_:+_gene_32, 92368 candidate probes\n",
      "--- 40474 probes passed check_dic selection.\n",
      "finish in 4.175s, 598 probes kept.\n",
      "-- check region:32 1:73522522-73603287_:+_gene_33, 148276 candidate probes\n",
      "--- 57778 probes passed check_dic selection.\n",
      "finish in 5.387s, 861 probes kept.\n",
      "-- check region:33 1:75423170-75478047_:+_gene_34, 106952 candidate probes\n",
      "--- 52450 probes passed check_dic selection.\n",
      "finish in 4.568s, 748 probes kept.\n",
      "-- check region:34 1:75854682-75920607_:+_gene_35, 129008 candidate probes\n",
      "--- 45737 probes passed check_dic selection.\n",
      "finish in 4.593s, 690 probes kept.\n",
      "-- check region:35 1:79813369-79876804_:+_gene_36, 120388 candidate probes\n",
      "--- 53258 probes passed check_dic selection.\n",
      "finish in 4.767s, 796 probes kept.\n",
      "-- check region:36 1:81222704-81365836_:+_gene_37, 279708 candidate probes\n",
      "--- 82776 probes passed check_dic selection.\n",
      "finish in 9.049s, 1437 probes kept.\n",
      "-- check region:37 1:82422928-82493138_:+_gene_38, 129794 candidate probes\n",
      "--- 58462 probes passed check_dic selection.\n",
      "finish in 5.258s, 861 probes kept.\n",
      "-- check region:38 1:83313920-83400588_:+_gene_39, 158546 candidate probes\n",
      "--- 49727 probes passed check_dic selection.\n",
      "finish in 5.274s, 831 probes kept.\n",
      "-- check region:39 1:84168846-84261608_:+_gene_40, 182545 candidate probes\n",
      "--- 68335 probes passed check_dic selection.\n",
      "finish in 6.508s, 1048 probes kept.\n",
      "-- check region:40 1:86959177-87062202_:+_gene_41, 198258 candidate probes\n",
      "--- 90435 probes passed check_dic selection.\n",
      "finish in 7.812s, 1285 probes kept.\n",
      "-- check region:41 1:89121460-89206528_:+_gene_42, 160184 candidate probes\n",
      "--- 75602 probes passed check_dic selection.\n",
      "finish in 6.415s, 1038 probes kept.\n",
      "-- check region:42 1:91412081-91466772_:+_gene_43, 107148 candidate probes\n",
      "--- 58735 probes passed check_dic selection.\n",
      "finish in 4.698s, 775 probes kept.\n",
      "-- check region:43 1:93189428-93238116_:+_gene_44, 96596 candidate probes\n",
      "--- 53448 probes passed check_dic selection.\n",
      "finish in 4.294s, 700 probes kept.\n",
      "-- check region:44 1:97604468-97662595_:+_gene_45, 114698 candidate probes\n",
      "--- 28687 probes passed check_dic selection.\n",
      "finish in 3.617s, 682 probes kept.\n",
      "-- check region:45 1:105961567-106009318_:+_gene_46, 95138 candidate probes\n",
      "--- 37175 probes passed check_dic selection.\n",
      "finish in 3.438s, 589 probes kept.\n",
      "-- check region:46 1:115727054-115804837_:+_gene_47, 139164 candidate probes\n",
      "--- 34880 probes passed check_dic selection.\n",
      "finish in 4.197s, 612 probes kept.\n",
      "-- check region:47 1:119437659-119505173_:+_gene_48, 132562 candidate probes\n",
      "--- 49100 probes passed check_dic selection.\n",
      "finish in 4.808s, 763 probes kept.\n",
      "-- check region:48 1:120161583-120218334_:+_gene_49, 110974 candidate probes\n",
      "--- 34933 probes passed check_dic selection.\n",
      "finish in 3.738s, 587 probes kept.\n",
      "-- check region:49 1:121430601-121489275_:+_gene_50, 113394 candidate probes\n",
      "--- 47907 probes passed check_dic selection.\n",
      "finish in 4.353s, 717 probes kept.\n",
      "-- check region:50 1:124674853-124801140_:+_gene_51, 245178 candidate probes\n",
      "--- 74102 probes passed check_dic selection.\n",
      "finish in 7.912s, 1284 probes kept.\n",
      "-- check region:51 1:137888029-137965735_:+_gene_52, 154354 candidate probes\n",
      "--- 50388 probes passed check_dic selection.\n",
      "finish in 5.122s, 826 probes kept.\n",
      "-- check region:52 1:140354959-140541768_:+_gene_53, 358720 candidate probes\n",
      "--- 75635 probes passed check_dic selection.\n",
      "finish in 9.896s, 1501 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:53 1:143861617-143934455_:+_gene_54, 118086 candidate probes\n",
      "--- 39151 probes passed check_dic selection.\n",
      "finish in 3.989s, 638 probes kept.\n",
      "-- check region:54 1:149846612-149908891_:+_gene_55, 123292 candidate probes\n",
      "--- 43249 probes passed check_dic selection.\n",
      "finish in 4.288s, 704 probes kept.\n",
      "-- check region:55 1:152397309-152486632_:+_gene_56, 161488 candidate probes\n",
      "--- 63119 probes passed check_dic selection.\n",
      "finish in 5.974s, 933 probes kept.\n",
      "-- check region:56 1:155665230-155748254_:+_gene_57, 163478 candidate probes\n",
      "--- 74249 probes passed check_dic selection.\n",
      "finish in 6.444s, 1060 probes kept.\n",
      "-- check region:57 1:161214625-161297711_:+_gene_58, 162006 candidate probes\n",
      "--- 61242 probes passed check_dic selection.\n",
      "finish in 5.789s, 960 probes kept.\n",
      "-- check region:58 1:162012959-162104685_:+_gene_59, 165712 candidate probes\n",
      "--- 63630 probes passed check_dic selection.\n",
      "finish in 6.001s, 993 probes kept.\n",
      "-- check region:59 1:162655058-162776885_:+_gene_60, 232123 candidate probes\n",
      "--- 63213 probes passed check_dic selection.\n",
      "finish in 7.251s, 1160 probes kept.\n",
      "-- check region:60 1:167034296-167100658_:+_gene_61, 130802 candidate probes\n",
      "--- 66329 probes passed check_dic selection.\n",
      "finish in 5.563s, 924 probes kept.\n",
      "-- check region:61 1:167197268-167284083_:+_gene_62, 159298 candidate probes\n",
      "--- 71154 probes passed check_dic selection.\n",
      "finish in 6.424s, 1025 probes kept.\n",
      "-- check region:62 1:169792098-169928411_:+_gene_63, 263464 candidate probes\n",
      "--- 99553 probes passed check_dic selection.\n",
      "finish in 9.502s, 1495 probes kept.\n",
      "-- check region:63 1:171429944-171499765_:+_gene_64, 130028 candidate probes\n",
      "--- 39563 probes passed check_dic selection.\n",
      "finish in 4.418s, 622 probes kept.\n",
      "-- check region:64 1:171805726-171877501_:+_gene_65, 121480 candidate probes\n",
      "--- 30089 probes passed check_dic selection.\n",
      "finish in 3.744s, 537 probes kept.\n",
      "-- check region:65 1:173307246-173374219_:+_gene_66, 132006 candidate probes\n",
      "--- 55958 probes passed check_dic selection.\n",
      "finish in 5.771s, 859 probes kept.\n",
      "-- check region:66 1:178341436-178410984_:+_gene_67, 133764 candidate probes\n",
      "--- 36940 probes passed check_dic selection.\n",
      "finish in 5.164s, 648 probes kept.\n",
      "-- check region:67 1:179031899-179125551_:+_gene_68, 185830 candidate probes\n",
      "--- 91327 probes passed check_dic selection.\n",
      "finish in 8.627s, 1244 probes kept.\n",
      "-- check region:68 1:182429520-182519678_:+_gene_69, 176026 candidate probes\n",
      "--- 76625 probes passed check_dic selection.\n",
      "finish in 7.036s, 1140 probes kept.\n",
      "-- check region:69 1:186617359-186706395_:+_gene_70, 174714 candidate probes\n",
      "--- 82764 probes passed check_dic selection.\n",
      "finish in 8.013s, 1182 probes kept.\n",
      "-- check region:70 1:187635279-187704850_:+_gene_71, 138096 candidate probes\n",
      "--- 68012 probes passed check_dic selection.\n",
      "finish in 6.576s, 938 probes kept.\n",
      "-- check region:71 1:189007529-189076052_:+_gene_72, 134014 candidate probes\n",
      "--- 51186 probes passed check_dic selection.\n",
      "finish in 5.829s, 773 probes kept.\n",
      "-- check region:72 1:189175060-189239230_:+_gene_73, 126434 candidate probes\n",
      "--- 54885 probes passed check_dic selection.\n",
      "finish in 5.770s, 803 probes kept.\n",
      "-- check region:73 1:189480922-189543419_:+_gene_74, 116604 candidate probes\n",
      "--- 44599 probes passed check_dic selection.\n",
      "finish in 4.786s, 658 probes kept.\n",
      "-- check region:74 1:189866129-189938237_:+_gene_75, 142142 candidate probes\n",
      "--- 63560 probes passed check_dic selection.\n",
      "finish in 6.364s, 946 probes kept.\n",
      "-- check region:75 1:193075619-193133645_:+_gene_76, 111136 candidate probes\n",
      "--- 45741 probes passed check_dic selection.\n",
      "finish in 4.871s, 707 probes kept.\n",
      "-- check region:76 2:3129373-3183216_:+_gene_77, 105642 candidate probes\n",
      "--- 45938 probes passed check_dic selection.\n",
      "finish in 4.693s, 661 probes kept.\n",
      "-- check region:77 2:4026085-4083325_:+_gene_78, 113334 candidate probes\n",
      "--- 46419 probes passed check_dic selection.\n",
      "finish in 4.965s, 719 probes kept.\n",
      "-- check region:78 2:5151601-5210127_:+_gene_79, 114510 candidate probes\n",
      "--- 43400 probes passed check_dic selection.\n",
      "finish in 4.433s, 663 probes kept.\n",
      "-- check region:79 2:11309886-11385319_:+_gene_80, 143604 candidate probes\n",
      "--- 47365 probes passed check_dic selection.\n",
      "finish in 4.976s, 751 probes kept.\n",
      "-- check region:80 2:14897328-14968889_:+_gene_81, 130488 candidate probes\n",
      "--- 42253 probes passed check_dic selection.\n",
      "finish in 4.432s, 690 probes kept.\n",
      "-- check region:81 2:16513406-16660259_:+_gene_82, 268220 candidate probes\n",
      "--- 68667 probes passed check_dic selection.\n",
      "finish in 8.045s, 1263 probes kept.\n",
      "-- check region:82 2:17628682-17734416_:+_gene_83, 201900 candidate probes\n",
      "--- 79125 probes passed check_dic selection.\n",
      "finish in 7.471s, 1224 probes kept.\n",
      "-- check region:83 2:19888794-19999195_:+_gene_84, 198994 candidate probes\n",
      "--- 54631 probes passed check_dic selection.\n",
      "finish in 6.301s, 926 probes kept.\n",
      "-- check region:84 2:21251409-21298244_:+_gene_85, 88568 candidate probes\n",
      "--- 37726 probes passed check_dic selection.\n",
      "finish in 3.378s, 544 probes kept.\n",
      "-- check region:85 2:21622583-21703613_:+_gene_86, 147344 candidate probes\n",
      "--- 34377 probes passed check_dic selection.\n",
      "finish in 4.870s, 589 probes kept.\n",
      "-- check region:86 2:25420535-25461500_:+_gene_87, 81014 candidate probes\n",
      "--- 40589 probes passed check_dic selection.\n",
      "finish in 3.946s, 597 probes kept.\n",
      "-- check region:87 2:27071925-27131679_:+_gene_88, 116524 candidate probes\n",
      "--- 62396 probes passed check_dic selection.\n",
      "finish in 5.904s, 854 probes kept.\n",
      "-- check region:88 2:30236602-30280512_:+_gene_89, 85708 candidate probes\n",
      "--- 38065 probes passed check_dic selection.\n",
      "finish in 4.145s, 555 probes kept.\n",
      "-- check region:89 2:32093351-32148238_:+_gene_90, 100136 candidate probes\n",
      "--- 41021 probes passed check_dic selection.\n",
      "finish in 4.649s, 584 probes kept.\n",
      "-- check region:90 2:37953869-38034097_:+_gene_91, 154870 candidate probes\n",
      "--- 59519 probes passed check_dic selection.\n",
      "finish in 6.631s, 899 probes kept.\n",
      "-- check region:91 2:41928242-42023962_:+_gene_92, 185462 candidate probes\n",
      "--- 40841 probes passed check_dic selection.\n",
      "finish in 6.143s, 797 probes kept.\n",
      "-- check region:92 2:43753833-43799050_:+_gene_93, 88478 candidate probes\n",
      "--- 32643 probes passed check_dic selection.\n",
      "finish in 3.945s, 501 probes kept.\n",
      "-- check region:93 2:44084305-44153466_:+_gene_94, 136706 candidate probes\n",
      "--- 47370 probes passed check_dic selection.\n",
      "finish in 5.847s, 772 probes kept.\n",
      "-- check region:94 2:44324153-44432010_:+_gene_95, 200980 candidate probes\n",
      "--- 83099 probes passed check_dic selection.\n",
      "finish in 9.030s, 1227 probes kept.\n",
      "-- check region:95 2:49898807-49970037_:+_gene_96, 141324 candidate probes\n",
      "--- 61433 probes passed check_dic selection.\n",
      "finish in 6.596s, 921 probes kept.\n",
      "-- check region:96 2:58005861-58065893_:+_gene_97, 114782 candidate probes\n",
      "--- 39053 probes passed check_dic selection.\n",
      "finish in 4.559s, 663 probes kept.\n",
      "-- check region:97 2:59634429-59685853_:+_gene_98, 100708 candidate probes\n",
      "--- 46497 probes passed check_dic selection.\n",
      "finish in 4.661s, 641 probes kept.\n",
      "-- check region:98 2:62467453-62543544_:+_gene_99, 149004 candidate probes\n",
      "--- 50432 probes passed check_dic selection.\n",
      "finish in 6.199s, 843 probes kept.\n",
      "-- check region:99 2:62670661-62738745_:+_gene_100, 132664 candidate probes\n",
      "--- 32997 probes passed check_dic selection.\n",
      "finish in 4.958s, 654 probes kept.\n",
      "-- check region:100 2:70541303-70603453_:+_gene_101, 122700 candidate probes\n",
      "--- 52989 probes passed check_dic selection.\n",
      "finish in 5.677s, 793 probes kept.\n",
      "-- check region:101 2:73253634-73311570_:+_gene_102, 111876 candidate probes\n",
      "--- 39179 probes passed check_dic selection.\n",
      "finish in 4.711s, 616 probes kept.\n",
      "-- check region:102 2:74102486-74171621_:+_gene_103, 136722 candidate probes\n",
      "--- 57278 probes passed check_dic selection.\n",
      "finish in 5.987s, 828 probes kept.\n",
      "-- check region:103 2:75747159-75820879_:+_gene_104, 142370 candidate probes\n",
      "--- 53804 probes passed check_dic selection.\n",
      "finish in 6.167s, 804 probes kept.\n",
      "-- check region:104 2:77013944-77091549_:+_gene_105, 152470 candidate probes\n",
      "--- 60491 probes passed check_dic selection.\n",
      "finish in 6.607s, 921 probes kept.\n",
      "-- check region:105 2:77277948-77367263_:+_gene_106, 174923 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 63588 probes passed check_dic selection.\n",
      "finish in 7.443s, 988 probes kept.\n",
      "-- check region:106 2:83704396-83781664_:+_gene_107, 148424 candidate probes\n",
      "--- 42261 probes passed check_dic selection.\n",
      "finish in 5.998s, 747 probes kept.\n",
      "-- check region:107 2:84515475-84567580_:+_gene_108, 94862 candidate probes\n",
      "--- 33504 probes passed check_dic selection.\n",
      "finish in 4.010s, 526 probes kept.\n",
      "-- check region:108 2:84845394-84900174_:+_gene_109, 108422 candidate probes\n",
      "--- 52903 probes passed check_dic selection.\n",
      "finish in 5.292s, 736 probes kept.\n",
      "-- check region:109 2:85037645-85079130_:+_gene_110, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.016s, 0 probes kept.\n",
      "-- check region:110 2:85037645-85079130_:+_gene_111, 81532 candidate probes\n",
      "--- 38862 probes passed check_dic selection.\n",
      "finish in 3.977s, 586 probes kept.\n",
      "-- check region:111 2:85198480-85295653_:+_gene_112, 187440 candidate probes\n",
      "--- 62384 probes passed check_dic selection.\n",
      "finish in 7.673s, 1002 probes kept.\n",
      "-- check region:112 2:91302426-91340781_:+_gene_113, 73912 candidate probes\n",
      "--- 28819 probes passed check_dic selection.\n",
      "finish in 3.079s, 426 probes kept.\n",
      "-- check region:113 2:97564926-97650442_:+_gene_114, 167940 candidate probes\n",
      "--- 39617 probes passed check_dic selection.\n",
      "finish in 6.065s, 757 probes kept.\n",
      "-- check region:114 2:110162327-110221668_:+_gene_115, 114156 candidate probes\n",
      "--- 45510 probes passed check_dic selection.\n",
      "finish in 5.021s, 687 probes kept.\n",
      "-- check region:115 2:118252753-118346305_:+_gene_116, 182540 candidate probes\n",
      "--- 73835 probes passed check_dic selection.\n",
      "finish in 7.603s, 1115 probes kept.\n",
      "-- check region:116 2:120618697-120674306_:+_gene_117, 99844 candidate probes\n",
      "--- 30204 probes passed check_dic selection.\n",
      "finish in 3.957s, 515 probes kept.\n",
      "-- check region:117 2:122327422-122368386_:+_gene_118, 81400 candidate probes\n",
      "--- 42489 probes passed check_dic selection.\n",
      "finish in 4.004s, 603 probes kept.\n",
      "-- check region:118 2:122548219-122636386_:+_gene_119, 166748 candidate probes\n",
      "--- 57257 probes passed check_dic selection.\n",
      "finish in 7.187s, 881 probes kept.\n",
      "-- check region:119 2:123300821-123374509_:+_gene_120, 145378 candidate probes\n",
      "--- 40849 probes passed check_dic selection.\n",
      "finish in 5.652s, 710 probes kept.\n",
      "-- check region:120 2:124108515-124225589_:+_gene_121, 230994 candidate probes\n",
      "--- 87334 probes passed check_dic selection.\n",
      "finish in 9.923s, 1373 probes kept.\n",
      "-- check region:121 2:127069492-127147676_:+_gene_122, 149248 candidate probes\n",
      "--- 52859 probes passed check_dic selection.\n",
      "finish in 6.125s, 838 probes kept.\n",
      "-- check region:122 2:128289201-128390141_:+_gene_123, 198262 candidate probes\n",
      "--- 85829 probes passed check_dic selection.\n",
      "finish in 8.678s, 1283 probes kept.\n",
      "-- check region:123 2:128694272-128817233_:+_gene_124, 236656 candidate probes\n",
      "--- 90645 probes passed check_dic selection.\n",
      "finish in 10.198s, 1379 probes kept.\n",
      "-- check region:124 2:129854913-129936172_:+_gene_125, 156780 candidate probes\n",
      "--- 56615 probes passed check_dic selection.\n",
      "finish in 6.554s, 889 probes kept.\n",
      "-- check region:125 2:130444939-130507012_:+_gene_126, 122660 candidate probes\n",
      "--- 36093 probes passed check_dic selection.\n",
      "finish in 4.849s, 625 probes kept.\n",
      "-- check region:126 2:131363155-131441109_:+_gene_127, 143802 candidate probes\n",
      "--- 55305 probes passed check_dic selection.\n",
      "finish in 5.802s, 815 probes kept.\n",
      "-- check region:127 2:132260572-132326514_:+_gene_128, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.014s, 0 probes kept.\n",
      "-- check region:128 2:132260572-132326514_:+_gene_129, 125348 candidate probes\n",
      "--- 43728 probes passed check_dic selection.\n",
      "finish in 5.138s, 716 probes kept.\n",
      "-- check region:129 2:135268536-135340636_:+_gene_130, 139300 candidate probes\n",
      "--- 50976 probes passed check_dic selection.\n",
      "finish in 5.743s, 832 probes kept.\n",
      "-- check region:130 2:135648540-135694141_:+_gene_131, 87708 candidate probes\n",
      "--- 36063 probes passed check_dic selection.\n",
      "finish in 3.710s, 540 probes kept.\n",
      "-- check region:131 2:136179077-136262638_:+_gene_132, 149658 candidate probes\n",
      "--- 36616 probes passed check_dic selection.\n",
      "finish in 5.922s, 688 probes kept.\n",
      "-- check region:132 2:139256790-139346099_:+_gene_133, 167240 candidate probes\n",
      "--- 43924 probes passed check_dic selection.\n",
      "finish in 6.585s, 800 probes kept.\n",
      "-- check region:133 2:143172015-143254256_:+_gene_134, 156022 candidate probes\n",
      "--- 49294 probes passed check_dic selection.\n",
      "finish in 6.415s, 815 probes kept.\n",
      "-- check region:134 2:143965229-144015150_:+_gene_135, 97064 candidate probes\n",
      "--- 49829 probes passed check_dic selection.\n",
      "finish in 4.860s, 683 probes kept.\n",
      "-- check region:135 2:147168730-147215713_:+_gene_136, 90174 candidate probes\n",
      "--- 43505 probes passed check_dic selection.\n",
      "finish in 4.301s, 592 probes kept.\n",
      "-- check region:136 2:148128049-148221409_:+_gene_137, 180010 candidate probes\n",
      "--- 65007 probes passed check_dic selection.\n",
      "finish in 7.217s, 980 probes kept.\n",
      "-- check region:137 2:148667886-148731885_:+_gene_138, 126566 candidate probes\n",
      "--- 48072 probes passed check_dic selection.\n",
      "finish in 5.110s, 782 probes kept.\n",
      "-- check region:138 2:152668254-152718287_:+_gene_139, 99364 candidate probes\n",
      "--- 43406 probes passed check_dic selection.\n",
      "finish in 4.401s, 672 probes kept.\n",
      "-- check region:139 2:153903330-153973662_:+_gene_140, 140028 candidate probes\n",
      "--- 75595 probes passed check_dic selection.\n",
      "finish in 7.038s, 1010 probes kept.\n",
      "-- check region:140 2:160197345-160260687_:+_gene_141, 124664 candidate probes\n",
      "--- 61191 probes passed check_dic selection.\n",
      "finish in 5.934s, 819 probes kept.\n",
      "-- check region:141 2:161118134-161195413_:+_gene_142, 150528 candidate probes\n",
      "--- 54626 probes passed check_dic selection.\n",
      "finish in 6.109s, 860 probes kept.\n",
      "-- check region:142 2:161757252-161819816_:+_gene_143, 122312 candidate probes\n",
      "--- 48554 probes passed check_dic selection.\n",
      "finish in 4.948s, 739 probes kept.\n",
      "-- check region:143 2:162746567-162835217_:+_gene_144, 174440 candidate probes\n",
      "--- 67366 probes passed check_dic selection.\n",
      "finish in 6.833s, 1029 probes kept.\n",
      "-- check region:144 2:172117465-172170675_:+_gene_145, 102454 candidate probes\n",
      "--- 39229 probes passed check_dic selection.\n",
      "finish in 4.291s, 589 probes kept.\n",
      "-- check region:145 2:172390054-172457136_:+_gene_146, 130862 candidate probes\n",
      "--- 54491 probes passed check_dic selection.\n",
      "finish in 5.553s, 822 probes kept.\n",
      "-- check region:146 2:172797494-172886579_:+_gene_147, 176636 candidate probes\n",
      "--- 88000 probes passed check_dic selection.\n",
      "finish in 8.283s, 1171 probes kept.\n",
      "-- check region:147 2:178118671-178177048_:+_gene_148, 116076 candidate probes\n",
      "--- 58734 probes passed check_dic selection.\n",
      "finish in 5.126s, 795 probes kept.\n",
      "-- check region:148 2:178484185-178533313_:+_gene_149, 96310 candidate probes\n",
      "--- 45084 probes passed check_dic selection.\n",
      "finish in 4.150s, 632 probes kept.\n",
      "-- check region:149 2:178654771-178739514_:+_gene_150, 167692 candidate probes\n",
      "--- 83770 probes passed check_dic selection.\n",
      "finish in 7.533s, 1120 probes kept.\n",
      "-- check region:150 2:180410644-180466893_:+_gene_151, 112032 candidate probes\n",
      "--- 60802 probes passed check_dic selection.\n",
      "finish in 5.394s, 799 probes kept.\n",
      "-- check region:151 3:10116470-10161129_:+_gene_152, 86990 candidate probes\n",
      "--- 30378 probes passed check_dic selection.\n",
      "finish in 3.530s, 469 probes kept.\n",
      "-- check region:152 3:15080099-15131257_:+_gene_153, 87086 candidate probes\n",
      "--- 26623 probes passed check_dic selection.\n",
      "finish in 3.293s, 440 probes kept.\n",
      "-- check region:153 3:19284209-19338497_:+_gene_154, 106018 candidate probes\n",
      "--- 34421 probes passed check_dic selection.\n",
      "finish in 3.914s, 573 probes kept.\n",
      "-- check region:154 3:19992968-20039109_:+_gene_155, 88630 candidate probes\n",
      "--- 25005 probes passed check_dic selection.\n",
      "finish in 3.017s, 455 probes kept.\n",
      "-- check region:155 3:20975804-21056576_:+_gene_156, 144312 candidate probes\n",
      "--- 31388 probes passed check_dic selection.\n",
      "finish in 4.315s, 591 probes kept.\n",
      "-- check region:156 3:26804177-26864088_:+_gene_157, 108688 candidate probes\n",
      "--- 42633 probes passed check_dic selection.\n",
      "finish in 4.252s, 624 probes kept.\n",
      "-- check region:157 3:27183615-27247800_:+_gene_158, 127234 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 48403 probes passed check_dic selection.\n",
      "finish in 4.949s, 775 probes kept.\n",
      "-- check region:158 3:27430699-27527522_:+_gene_159, 190666 candidate probes\n",
      "--- 92764 probes passed check_dic selection.\n",
      "finish in 8.111s, 1299 probes kept.\n",
      "-- check region:159 3:27639981-27714878_:+_gene_160, 148402 candidate probes\n",
      "--- 70414 probes passed check_dic selection.\n",
      "finish in 6.265s, 977 probes kept.\n",
      "-- check region:160 3:27985084-28070214_:+_gene_161, 152970 candidate probes\n",
      "--- 51581 probes passed check_dic selection.\n",
      "finish in 5.554s, 832 probes kept.\n",
      "-- check region:161 3:33340421-33427551_:+_gene_162, 168186 candidate probes\n",
      "--- 54703 probes passed check_dic selection.\n",
      "finish in 6.024s, 875 probes kept.\n",
      "-- check region:162 3:35074184-35113818_:+_gene_163, 78584 candidate probes\n",
      "--- 38901 probes passed check_dic selection.\n",
      "finish in 3.388s, 532 probes kept.\n",
      "-- check region:163 3:35313296-35364803_:+_gene_164, 101910 candidate probes\n",
      "--- 49457 probes passed check_dic selection.\n",
      "finish in 4.529s, 659 probes kept.\n",
      "-- check region:164 3:37638345-37722903_:+_gene_165, 166554 candidate probes\n",
      "--- 71864 probes passed check_dic selection.\n",
      "finish in 6.987s, 1060 probes kept.\n",
      "-- check region:165 3:50411173-50448666_:+_gene_166, 74408 candidate probes\n",
      "--- 25763 probes passed check_dic selection.\n",
      "finish in 2.800s, 425 probes kept.\n",
      "-- check region:166 3:50892858-50950176_:+_gene_167, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.012s, 0 probes kept.\n",
      "-- check region:167 3:50892858-50950176_:+_gene_168, 112436 candidate probes\n",
      "--- 52958 probes passed check_dic selection.\n",
      "finish in 4.784s, 741 probes kept.\n",
      "-- check region:168 3:53719899-53805193_:+_gene_169, 157310 candidate probes\n",
      "--- 51297 probes passed check_dic selection.\n",
      "finish in 5.657s, 838 probes kept.\n",
      "-- check region:169 3:54216201-54302564_:+_gene_170, 170246 candidate probes\n",
      "--- 58008 probes passed check_dic selection.\n",
      "finish in 6.174s, 940 probes kept.\n",
      "-- check region:170 3:55041185-55109341_:+_gene_171, 132720 candidate probes\n",
      "--- 46925 probes passed check_dic selection.\n",
      "finish in 4.838s, 739 probes kept.\n",
      "-- check region:171 3:57736286-57847155_:+_gene_172, 212296 candidate probes\n",
      "--- 49193 probes passed check_dic selection.\n",
      "finish in 6.655s, 942 probes kept.\n",
      "-- check region:172 3:58079664-58133121_:+_gene_173, 101234 candidate probes\n",
      "--- 31492 probes passed check_dic selection.\n",
      "finish in 3.974s, 502 probes kept.\n",
      "-- check region:173 3:59191631-59297167_:+_gene_174, 193980 candidate probes\n",
      "--- 69277 probes passed check_dic selection.\n",
      "finish in 8.233s, 1129 probes kept.\n",
      "-- check region:174 3:62385566-62475321_:+_gene_175, 177872 candidate probes\n",
      "--- 65563 probes passed check_dic selection.\n",
      "finish in 7.482s, 1041 probes kept.\n",
      "-- check region:175 3:78938383-79045067_:+_gene_176, 199734 candidate probes\n",
      "--- 62741 probes passed check_dic selection.\n",
      "finish in 8.053s, 1021 probes kept.\n",
      "-- check region:176 3:82013253-82074011_:+_gene_177, 120566 candidate probes\n",
      "--- 41943 probes passed check_dic selection.\n",
      "finish in 5.177s, 698 probes kept.\n",
      "-- check region:177 3:83926234-84003443_:+_gene_178, 152156 candidate probes\n",
      "--- 68482 probes passed check_dic selection.\n",
      "finish in 6.943s, 986 probes kept.\n",
      "-- check region:178 3:85960586-86088957_:+_gene_179, 251502 candidate probes\n",
      "--- 92166 probes passed check_dic selection.\n",
      "finish in 10.075s, 1431 probes kept.\n",
      "-- check region:179 3:87079865-87164939_:+_gene_180, 168064 candidate probes\n",
      "--- 81832 probes passed check_dic selection.\n",
      "finish in 7.761s, 1162 probes kept.\n",
      "-- check region:180 3:87262909-87334252_:+_gene_181, 126060 candidate probes\n",
      "--- 30321 probes passed check_dic selection.\n",
      "finish in 4.330s, 556 probes kept.\n",
      "-- check region:181 3:88457083-88500649_:+_gene_182, 83684 candidate probes\n",
      "--- 28647 probes passed check_dic selection.\n",
      "finish in 3.247s, 463 probes kept.\n",
      "-- check region:182 3:90073915-90138447_:+_gene_183, 123528 candidate probes\n",
      "--- 36027 probes passed check_dic selection.\n",
      "finish in 4.788s, 622 probes kept.\n",
      "-- check region:183 3:90262479-90347057_:+_gene_184, 162858 candidate probes\n",
      "--- 41504 probes passed check_dic selection.\n",
      "finish in 5.611s, 788 probes kept.\n",
      "-- check region:184 3:92217911-92290171_:+_gene_185, 65866 candidate probes\n",
      "--- 2843 probes passed check_dic selection.\n",
      "finish in 1.714s, 115 probes kept.\n",
      "-- check region:185 3:95498508-95551908_:+_gene_186, 103894 candidate probes\n",
      "--- 28752 probes passed check_dic selection.\n",
      "finish in 3.817s, 491 probes kept.\n",
      "-- check region:186 3:97271448-97349071_:+_gene_187, 152532 candidate probes\n",
      "--- 68216 probes passed check_dic selection.\n",
      "finish in 6.539s, 962 probes kept.\n",
      "-- check region:187 3:97659551-97729267_:+_gene_188, 137038 candidate probes\n",
      "--- 61000 probes passed check_dic selection.\n",
      "finish in 5.882s, 900 probes kept.\n",
      "-- check region:188 3:100383410-100488704_:+_gene_189, 202902 candidate probes\n",
      "--- 93021 probes passed check_dic selection.\n",
      "finish in 8.783s, 1330 probes kept.\n",
      "-- check region:189 3:102135537-102204731_:+_gene_190, 137350 candidate probes\n",
      "--- 65921 probes passed check_dic selection.\n",
      "finish in 6.088s, 954 probes kept.\n",
      "-- check region:190 3:102681535-102725439_:+_gene_191, 86572 candidate probes\n",
      "--- 38445 probes passed check_dic selection.\n",
      "finish in 3.651s, 544 probes kept.\n",
      "-- check region:191 3:103092173-103150905_:+_gene_192, 111404 candidate probes\n",
      "--- 38026 probes passed check_dic selection.\n",
      "finish in 4.319s, 619 probes kept.\n",
      "-- check region:192 3:103363744-103456515_:+_gene_193, 173576 candidate probes\n",
      "--- 62969 probes passed check_dic selection.\n",
      "finish in 6.744s, 950 probes kept.\n",
      "-- check region:193 3:104633899-104693362_:+_gene_194, 115538 candidate probes\n",
      "--- 39235 probes passed check_dic selection.\n",
      "finish in 4.214s, 648 probes kept.\n",
      "-- check region:194 3:105881717-105941481_:+_gene_195, 115694 candidate probes\n",
      "--- 40854 probes passed check_dic selection.\n",
      "finish in 4.469s, 634 probes kept.\n",
      "-- check region:195 3:109002413-109073430_:+_gene_196, 134884 candidate probes\n",
      "--- 45664 probes passed check_dic selection.\n",
      "finish in 5.040s, 705 probes kept.\n",
      "-- check region:196 3:117207508-117290078_:+_gene_197, 144356 candidate probes\n",
      "--- 37376 probes passed check_dic selection.\n",
      "finish in 4.818s, 653 probes kept.\n",
      "-- check region:197 3:117778923-117864154_:+_gene_198, 166576 candidate probes\n",
      "--- 53757 probes passed check_dic selection.\n",
      "finish in 6.139s, 918 probes kept.\n",
      "-- check region:198 3:118415335-118490496_:+_gene_199, 148612 candidate probes\n",
      "--- 46952 probes passed check_dic selection.\n",
      "finish in 5.320s, 797 probes kept.\n",
      "-- check region:199 3:125833531-125935817_:+_gene_200, 185994 candidate probes\n",
      "--- 52890 probes passed check_dic selection.\n",
      "finish in 6.436s, 960 probes kept.\n",
      "-- check region:200 3:126599799-126711558_:+_gene_201, 218392 candidate probes\n",
      "--- 81493 probes passed check_dic selection.\n",
      "finish in 8.563s, 1248 probes kept.\n",
      "-- check region:201 3:127684895-127739338_:+_gene_202, 105556 candidate probes\n",
      "--- 42149 probes passed check_dic selection.\n",
      "finish in 4.353s, 632 probes kept.\n",
      "-- check region:202 3:129542117-129668198_:+_gene_203, 247958 candidate probes\n",
      "--- 107812 probes passed check_dic selection.\n",
      "finish in 10.193s, 1552 probes kept.\n",
      "-- check region:203 3:130232712-130315166_:+_gene_204, 161688 candidate probes\n",
      "--- 70048 probes passed check_dic selection.\n",
      "finish in 6.718s, 1004 probes kept.\n",
      "-- check region:204 3:133451837-133520394_:+_gene_205, 130758 candidate probes\n",
      "--- 35585 probes passed check_dic selection.\n",
      "finish in 4.481s, 683 probes kept.\n",
      "-- check region:205 3:135569252-135673396_:+_gene_206, 206686 candidate probes\n",
      "--- 91188 probes passed check_dic selection.\n",
      "finish in 8.803s, 1360 probes kept.\n",
      "-- check region:206 3:137598529-137658281_:+_gene_207, 103688 candidate probes\n",
      "--- 37668 probes passed check_dic selection.\n",
      "finish in 4.070s, 611 probes kept.\n",
      "-- check region:207 3:145795719-145916046_:+_gene_208, 223372 candidate probes\n",
      "--- 100280 probes passed check_dic selection.\n",
      "finish in 9.502s, 1433 probes kept.\n",
      "-- check region:208 3:146300985-146364892_:+_gene_209, 116346 candidate probes\n",
      "--- 56203 probes passed check_dic selection.\n",
      "finish in 5.104s, 756 probes kept.\n",
      "-- check region:209 3:152275792-152343485_:+_gene_210, 127724 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 38512 probes passed check_dic selection.\n",
      "finish in 4.621s, 656 probes kept.\n",
      "-- check region:210 3:157139588-157226143_:+_gene_211, 171974 candidate probes\n",
      "--- 66129 probes passed check_dic selection.\n",
      "finish in 6.789s, 1025 probes kept.\n",
      "-- check region:211 4:3040005-3100987_:+_gene_212, 87872 candidate probes\n",
      "--- 7145 probes passed check_dic selection.\n",
      "finish in 2.501s, 209 probes kept.\n",
      "-- check region:212 4:3471494-3540953_:+_gene_213, 131520 candidate probes\n",
      "--- 10270 probes passed check_dic selection.\n",
      "finish in 3.508s, 283 probes kept.\n",
      "-- check region:213 4:6346853-6400453_:+_gene_214, 104806 candidate probes\n",
      "--- 35117 probes passed check_dic selection.\n",
      "finish in 3.840s, 593 probes kept.\n",
      "-- check region:214 4:8546598-8621298_:+_gene_215, 146210 candidate probes\n",
      "--- 42983 probes passed check_dic selection.\n",
      "finish in 5.022s, 717 probes kept.\n",
      "-- check region:215 4:9621515-9668472_:+_gene_216, 93616 candidate probes\n",
      "--- 35426 probes passed check_dic selection.\n",
      "finish in 3.572s, 560 probes kept.\n",
      "-- check region:216 4:10962894-11009200_:+_gene_217, 90222 candidate probes\n",
      "--- 31455 probes passed check_dic selection.\n",
      "finish in 3.449s, 500 probes kept.\n",
      "-- check region:217 4:11962627-12051893_:+_gene_218, 171792 candidate probes\n",
      "--- 60759 probes passed check_dic selection.\n",
      "finish in 6.304s, 953 probes kept.\n",
      "-- check region:218 4:22467081-22509068_:+_gene_219, 79599 candidate probes\n",
      "--- 30918 probes passed check_dic selection.\n",
      "finish in 3.086s, 469 probes kept.\n",
      "-- check region:219 4:24745226-24835054_:+_gene_220, 167594 candidate probes\n",
      "--- 44822 probes passed check_dic selection.\n",
      "finish in 5.579s, 786 probes kept.\n",
      "-- check region:220 4:24955708-25039170_:+_gene_221, 162782 candidate probes\n",
      "--- 45094 probes passed check_dic selection.\n",
      "finish in 5.373s, 813 probes kept.\n",
      "-- check region:221 4:25657294-25764933_:+_gene_222, 192886 candidate probes\n",
      "--- 45746 probes passed check_dic selection.\n",
      "finish in 5.941s, 868 probes kept.\n",
      "-- check region:222 4:28804761-28881872_:+_gene_223, 153144 candidate probes\n",
      "--- 46792 probes passed check_dic selection.\n",
      "finish in 5.178s, 804 probes kept.\n",
      "-- check region:223 4:32845518-32900800_:+_gene_224, 103740 candidate probes\n",
      "--- 48161 probes passed check_dic selection.\n",
      "finish in 4.664s, 675 probes kept.\n",
      "-- check region:224 4:33921425-33962072_:+_gene_225, 78034 candidate probes\n",
      "--- 32305 probes passed check_dic selection.\n",
      "finish in 3.515s, 513 probes kept.\n",
      "-- check region:225 4:39175238-39234393_:+_gene_226, 106600 candidate probes\n",
      "--- 22916 probes passed check_dic selection.\n",
      "finish in 3.803s, 456 probes kept.\n",
      "-- check region:226 4:40468049-40532554_:+_gene_227, 5980 candidate probes\n",
      "--- 94 probes passed check_dic selection.\n",
      "finish in 0.207s, 6 probes kept.\n",
      "-- check region:227 4:40468049-40529504_:+_gene_228, 118704 candidate probes\n",
      "--- 44087 probes passed check_dic selection.\n",
      "finish in 5.245s, 685 probes kept.\n",
      "-- check region:228 4:40762947-40854743_:+_gene_229, 178430 candidate probes\n",
      "--- 72129 probes passed check_dic selection.\n",
      "finish in 7.386s, 1045 probes kept.\n",
      "-- check region:229 4:41137781-41221678_:+_gene_230, 162742 candidate probes\n",
      "--- 46489 probes passed check_dic selection.\n",
      "finish in 5.922s, 820 probes kept.\n",
      "-- check region:230 4:43195060-43271596_:+_gene_231, 149970 candidate probes\n",
      "--- 56810 probes passed check_dic selection.\n",
      "finish in 6.157s, 925 probes kept.\n",
      "-- check region:231 4:43536822-43581834_:+_gene_232, 89006 candidate probes\n",
      "--- 43699 probes passed check_dic selection.\n",
      "finish in 4.300s, 653 probes kept.\n",
      "-- check region:232 4:47556560-47631977_:+_gene_233, 146398 candidate probes\n",
      "--- 70232 probes passed check_dic selection.\n",
      "finish in 6.816s, 966 probes kept.\n",
      "-- check region:233 4:49711396-49823368_:+_gene_234, 203578 candidate probes\n",
      "--- 54069 probes passed check_dic selection.\n",
      "finish in 7.136s, 973 probes kept.\n",
      "-- check region:234 4:56810348-56866569_:+_gene_235, 108960 candidate probes\n",
      "--- 38357 probes passed check_dic selection.\n",
      "finish in 4.307s, 638 probes kept.\n",
      "-- check region:235 4:56953595-57059748_:+_gene_236, 207246 candidate probes\n",
      "--- 83572 probes passed check_dic selection.\n",
      "finish in 9.343s, 1290 probes kept.\n",
      "-- check region:236 4:57402930-57483013_:+_gene_237, 151350 candidate probes\n",
      "--- 46479 probes passed check_dic selection.\n",
      "finish in 5.125s, 767 probes kept.\n",
      "-- check region:237 4:58488835-58557154_:+_gene_238, 134414 candidate probes\n",
      "--- 57897 probes passed check_dic selection.\n",
      "finish in 5.136s, 839 probes kept.\n",
      "-- check region:238 4:62288487-62379491_:+_gene_239, 176452 candidate probes\n",
      "--- 60500 probes passed check_dic selection.\n",
      "finish in 6.076s, 1037 probes kept.\n",
      "-- check region:239 4:65706234-65796043_:+_gene_240, 165938 candidate probes\n",
      "--- 57957 probes passed check_dic selection.\n",
      "finish in 5.725s, 950 probes kept.\n",
      "-- check region:240 4:66351813-66425032_:+_gene_241, 127894 candidate probes\n",
      "--- 49947 probes passed check_dic selection.\n",
      "finish in 4.789s, 771 probes kept.\n",
      "-- check region:241 4:70432043-70539439_:+_gene_242, 197402 candidate probes\n",
      "--- 51632 probes passed check_dic selection.\n",
      "finish in 6.008s, 960 probes kept.\n",
      "-- check region:242 4:86148471-86220141_:+_gene_243, 139084 candidate probes\n",
      "--- 45689 probes passed check_dic selection.\n",
      "finish in 4.741s, 757 probes kept.\n",
      "-- check region:243 4:86991120-87073412_:+_gene_244, 161720 candidate probes\n",
      "--- 74388 probes passed check_dic selection.\n",
      "finish in 6.399s, 1057 probes kept.\n",
      "-- check region:244 4:88142498-88275144_:+_gene_245, 256150 candidate probes\n",
      "--- 85920 probes passed check_dic selection.\n",
      "finish in 8.580s, 1423 probes kept.\n",
      "-- check region:245 4:98072752-98129663_:+_gene_246, 112704 candidate probes\n",
      "--- 47315 probes passed check_dic selection.\n",
      "finish in 4.279s, 724 probes kept.\n",
      "-- check region:246 4:98531272-98638773_:+_gene_247, 210868 candidate probes\n",
      "--- 81275 probes passed check_dic selection.\n",
      "finish in 7.603s, 1234 probes kept.\n",
      "-- check region:247 4:99330446-99395864_:+_gene_248, 127638 candidate probes\n",
      "--- 61100 probes passed check_dic selection.\n",
      "finish in 5.241s, 828 probes kept.\n",
      "-- check region:248 4:100963562-101018943_:+_gene_249, 107920 candidate probes\n",
      "--- 42060 probes passed check_dic selection.\n",
      "finish in 4.042s, 607 probes kept.\n",
      "-- check region:249 4:104983954-105060480_:+_gene_250, 149192 candidate probes\n",
      "--- 50546 probes passed check_dic selection.\n",
      "finish in 5.234s, 848 probes kept.\n",
      "-- check region:250 4:105899094-105952080_:+_gene_251, 102546 candidate probes\n",
      "--- 49093 probes passed check_dic selection.\n",
      "finish in 4.192s, 679 probes kept.\n",
      "-- check region:251 4:114589240-114676242_:+_gene_252, 167018 candidate probes\n",
      "--- 82309 probes passed check_dic selection.\n",
      "finish in 6.918s, 1129 probes kept.\n",
      "-- check region:252 4:115037700-115112192_:+_gene_253, 145166 candidate probes\n",
      "--- 64512 probes passed check_dic selection.\n",
      "finish in 5.689s, 936 probes kept.\n",
      "-- check region:253 4:115837369-115913327_:+_gene_254, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.013s, 0 probes kept.\n",
      "-- check region:254 4:115837369-115913327_:+_gene_255, 150696 candidate probes\n",
      "--- 75038 probes passed check_dic selection.\n",
      "finish in 6.287s, 1055 probes kept.\n",
      "-- check region:255 4:116628906-116683403_:+_gene_256, 102454 candidate probes\n",
      "--- 26548 probes passed check_dic selection.\n",
      "finish in 3.196s, 469 probes kept.\n",
      "-- check region:256 4:117618081-117687368_:+_gene_257, 136918 candidate probes\n",
      "--- 65608 probes passed check_dic selection.\n",
      "finish in 5.615s, 935 probes kept.\n",
      "-- check region:257 4:123199825-123264995_:+_gene_258, 128402 candidate probes\n",
      "--- 56744 probes passed check_dic selection.\n",
      "finish in 5.000s, 804 probes kept.\n",
      "-- check region:258 4:123961238-124063077_:+_gene_259, 200244 candidate probes\n",
      "--- 81722 probes passed check_dic selection.\n",
      "finish in 7.587s, 1201 probes kept.\n",
      "-- check region:259 4:124824301-124866678_:+_gene_260, 82998 candidate probes\n",
      "--- 31220 probes passed check_dic selection.\n",
      "finish in 3.186s, 521 probes kept.\n",
      "-- check region:260 4:126207621-126263017_:+_gene_261, 106790 candidate probes\n",
      "--- 44638 probes passed check_dic selection.\n",
      "finish in 4.130s, 633 probes kept.\n",
      "-- check region:261 4:126995025-127049608_:+_gene_262, 93988 candidate probes\n",
      "--- 25844 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 3.037s, 467 probes kept.\n",
      "-- check region:262 4:127493306-127563000_:+_gene_263, 132824 candidate probes\n",
      "--- 60619 probes passed check_dic selection.\n",
      "finish in 5.293s, 859 probes kept.\n",
      "-- check region:263 4:127666510-127718932_:+_gene_264, 104122 candidate probes\n",
      "--- 53235 probes passed check_dic selection.\n",
      "finish in 4.364s, 705 probes kept.\n",
      "-- check region:264 4:127800701-127872488_:+_gene_265, 140016 candidate probes\n",
      "--- 56014 probes passed check_dic selection.\n",
      "finish in 5.177s, 853 probes kept.\n",
      "-- check region:265 4:129485757-129532804_:+_gene_266, 90586 candidate probes\n",
      "--- 38539 probes passed check_dic selection.\n",
      "finish in 3.548s, 554 probes kept.\n",
      "-- check region:266 4:130236140-130307752_:+_gene_267, 138854 candidate probes\n",
      "--- 51721 probes passed check_dic selection.\n",
      "finish in 5.082s, 785 probes kept.\n",
      "-- check region:267 4:130509810-130579025_:+_gene_268, 120426 candidate probes\n",
      "--- 48166 probes passed check_dic selection.\n",
      "finish in 4.537s, 713 probes kept.\n",
      "-- check region:268 4:131733908-131844723_:+_gene_269, 217996 candidate probes\n",
      "--- 103826 probes passed check_dic selection.\n",
      "finish in 8.882s, 1458 probes kept.\n",
      "-- check region:269 4:132792079-132834284_:+_gene_270, 80362 candidate probes\n",
      "--- 27966 probes passed check_dic selection.\n",
      "finish in 2.895s, 444 probes kept.\n",
      "-- check region:270 4:135807186-135873467_:+_gene_271, 128878 candidate probes\n",
      "--- 39546 probes passed check_dic selection.\n",
      "finish in 4.277s, 671 probes kept.\n",
      "-- check region:271 4:137277244-137324456_:+_gene_272, 92876 candidate probes\n",
      "--- 48151 probes passed check_dic selection.\n",
      "finish in 3.984s, 652 probes kept.\n",
      "-- check region:272 4:148835951-148933937_:+_gene_273, 193758 candidate probes\n",
      "--- 106021 probes passed check_dic selection.\n",
      "finish in 8.438s, 1372 probes kept.\n",
      "-- check region:273 4:155759707-155802098_:+_gene_274, 80186 candidate probes\n",
      "--- 28215 probes passed check_dic selection.\n",
      "finish in 2.865s, 447 probes kept.\n",
      "-- check region:274 5:3994952-4090244_:+_gene_275, 183512 candidate probes\n",
      "--- 51491 probes passed check_dic selection.\n",
      "finish in 5.801s, 933 probes kept.\n",
      "-- check region:275 5:4937946-5030393_:+_gene_276, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.012s, 0 probes kept.\n",
      "-- check region:276 5:4937946-5030393_:+_gene_277, 173066 candidate probes\n",
      "--- 64515 probes passed check_dic selection.\n",
      "finish in 6.205s, 1010 probes kept.\n",
      "-- check region:277 5:5216972-5313712_:+_gene_278, 186955 candidate probes\n",
      "--- 67227 probes passed check_dic selection.\n",
      "finish in 6.469s, 1079 probes kept.\n",
      "-- check region:278 5:15947115-16026399_:+_gene_279, 153300 candidate probes\n",
      "--- 48431 probes passed check_dic selection.\n",
      "finish in 5.232s, 811 probes kept.\n",
      "-- check region:279 5:19894500-19961425_:+_gene_280, 123632 candidate probes\n",
      "--- 41074 probes passed check_dic selection.\n",
      "finish in 4.256s, 681 probes kept.\n",
      "-- check region:280 5:21457453-21528145_:+_gene_281, 139714 candidate probes\n",
      "--- 51346 probes passed check_dic selection.\n",
      "finish in 4.973s, 788 probes kept.\n",
      "-- check region:281 5:24770084-24831106_:+_gene_282, 114772 candidate probes\n",
      "--- 34890 probes passed check_dic selection.\n",
      "finish in 3.834s, 567 probes kept.\n",
      "-- check region:282 5:25594458-25634378_:+_gene_283, 77254 candidate probes\n",
      "--- 24992 probes passed check_dic selection.\n",
      "finish in 2.721s, 410 probes kept.\n",
      "-- check region:283 5:27049822-27109916_:+_gene_284, 118310 candidate probes\n",
      "--- 49707 probes passed check_dic selection.\n",
      "finish in 4.517s, 746 probes kept.\n",
      "-- check region:284 5:30381241-30452747_:+_gene_285, 141376 candidate probes\n",
      "--- 70118 probes passed check_dic selection.\n",
      "finish in 5.896s, 971 probes kept.\n",
      "-- check region:285 5:33193371-33256343_:+_gene_286, 124260 candidate probes\n",
      "--- 54325 probes passed check_dic selection.\n",
      "finish in 4.881s, 793 probes kept.\n",
      "-- check region:286 5:38342152-38414629_:+_gene_287, 130374 candidate probes\n",
      "--- 63193 probes passed check_dic selection.\n",
      "finish in 5.438s, 885 probes kept.\n",
      "-- check region:287 5:38506393-38564776_:+_gene_288, 115346 candidate probes\n",
      "--- 55518 probes passed check_dic selection.\n",
      "finish in 4.810s, 776 probes kept.\n",
      "-- check region:288 5:38850470-38910029_:+_gene_289, 117882 candidate probes\n",
      "--- 49553 probes passed check_dic selection.\n",
      "finish in 4.477s, 734 probes kept.\n",
      "-- check region:289 5:39106286-39168083_:+_gene_290, 122224 candidate probes\n",
      "--- 49991 probes passed check_dic selection.\n",
      "finish in 4.587s, 754 probes kept.\n",
      "-- check region:290 5:43318583-43370939_:+_gene_291, 94180 candidate probes\n",
      "--- 34747 probes passed check_dic selection.\n",
      "finish in 3.374s, 524 probes kept.\n",
      "-- check region:291 5:44189271-44278791_:+_gene_292, 175374 candidate probes\n",
      "--- 64044 probes passed check_dic selection.\n",
      "finish in 6.199s, 1013 probes kept.\n",
      "-- check region:292 5:44636554-44752497_:+_gene_293, 227438 candidate probes\n",
      "--- 96494 probes passed check_dic selection.\n",
      "finish in 8.647s, 1455 probes kept.\n",
      "-- check region:293 5:44853523-44919113_:+_gene_294, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.014s, 0 probes kept.\n",
      "-- check region:294 5:44853523-44919113_:+_gene_295, 126978 candidate probes\n",
      "--- 43362 probes passed check_dic selection.\n",
      "finish in 4.340s, 698 probes kept.\n",
      "-- check region:295 5:49951907-50059873_:+_gene_296, 210416 candidate probes\n",
      "--- 95107 probes passed check_dic selection.\n",
      "finish in 8.233s, 1352 probes kept.\n",
      "-- check region:296 5:52779669-52833675_:+_gene_297, 105000 candidate probes\n",
      "--- 36529 probes passed check_dic selection.\n",
      "finish in 3.672s, 619 probes kept.\n",
      "-- check region:297 5:58013151-58138957_:+_gene_298, 248744 candidate probes\n",
      "--- 62386 probes passed check_dic selection.\n",
      "finish in 7.363s, 1183 probes kept.\n",
      "-- check region:298 5:62628998-62700814_:+_gene_299, 140468 candidate probes\n",
      "--- 43122 probes passed check_dic selection.\n",
      "finish in 4.781s, 757 probes kept.\n",
      "-- check region:299 5:63805226-63916930_:+_gene_300, 209810 candidate probes\n",
      "--- 91434 probes passed check_dic selection.\n",
      "finish in 8.042s, 1341 probes kept.\n",
      "-- check region:300 5:64740899-64823898_:+_gene_301, 164424 candidate probes\n",
      "--- 56623 probes passed check_dic selection.\n",
      "finish in 5.796s, 953 probes kept.\n",
      "-- check region:301 5:66052841-66098951_:+_gene_302, 89562 candidate probes\n",
      "--- 35281 probes passed check_dic selection.\n",
      "finish in 3.287s, 537 probes kept.\n",
      "-- check region:302 5:67330974-67411928_:+_gene_303, 143766 candidate probes\n",
      "--- 49331 probes passed check_dic selection.\n",
      "finish in 5.097s, 831 probes kept.\n",
      "-- check region:303 5:71592818-71661574_:+_gene_304, 133728 candidate probes\n",
      "--- 38031 probes passed check_dic selection.\n",
      "finish in 4.203s, 674 probes kept.\n",
      "-- check region:304 5:72807519-72901992_:+_gene_305, 184614 candidate probes\n",
      "--- 72417 probes passed check_dic selection.\n",
      "finish in 6.759s, 1073 probes kept.\n",
      "-- check region:305 5:74601244-74658537_:+_gene_306, 113768 candidate probes\n",
      "--- 55842 probes passed check_dic selection.\n",
      "finish in 4.716s, 766 probes kept.\n",
      "-- check region:306 5:75625002-75694903_:+_gene_307, 133458 candidate probes\n",
      "--- 57393 probes passed check_dic selection.\n",
      "finish in 5.201s, 864 probes kept.\n",
      "-- check region:307 5:77344237-77404169_:+_gene_308, 117912 candidate probes\n",
      "--- 57408 probes passed check_dic selection.\n",
      "finish in 4.928s, 789 probes kept.\n",
      "-- check region:308 5:88554694-88611033_:+_gene_309, 99110 candidate probes\n",
      "--- 33688 probes passed check_dic selection.\n",
      "finish in 3.392s, 551 probes kept.\n",
      "-- check region:309 5:91538047-91648373_:+_gene_310, 202785 candidate probes\n",
      "--- 87090 probes passed check_dic selection.\n",
      "finish in 7.691s, 1281 probes kept.\n",
      "-- check region:310 5:92331917-92408829_:+_gene_311, 149852 candidate probes\n",
      "--- 53792 probes passed check_dic selection.\n",
      "finish in 5.256s, 852 probes kept.\n",
      "-- check region:311 5:104291330-104355424_:+_gene_312, 121265 candidate probes\n",
      "--- 47266 probes passed check_dic selection.\n",
      "finish in 4.435s, 695 probes kept.\n",
      "-- check region:312 5:106763128-106836267_:+_gene_313, 142790 candidate probes\n",
      "--- 58766 probes passed check_dic selection.\n",
      "finish in 5.354s, 841 probes kept.\n",
      "-- check region:313 5:108667189-108745317_:+_gene_314, 150818 candidate probes\n",
      "--- 71508 probes passed check_dic selection.\n",
      "finish in 6.126s, 1019 probes kept.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- check region:314 5:116975881-117047536_:+_gene_315, 137930 candidate probes\n",
      "--- 52586 probes passed check_dic selection.\n",
      "finish in 5.156s, 761 probes kept.\n",
      "-- check region:315 5:119037555-119110105_:+_gene_316, 142368 candidate probes\n",
      "--- 67066 probes passed check_dic selection.\n",
      "finish in 6.186s, 900 probes kept.\n",
      "-- check region:316 5:126228032-126292172_:+_gene_317, 124802 candidate probes\n",
      "--- 47276 probes passed check_dic selection.\n",
      "finish in 4.933s, 725 probes kept.\n",
      "-- check region:317 5:128181775-128254686_:+_gene_318, 141854 candidate probes\n",
      "--- 62298 probes passed check_dic selection.\n",
      "finish in 5.952s, 882 probes kept.\n",
      "-- check region:318 5:129741425-129789547_:+_gene_319, 92298 candidate probes\n",
      "--- 28824 probes passed check_dic selection.\n",
      "finish in 3.405s, 477 probes kept.\n",
      "-- check region:319 5:129925623-129977156_:+_gene_320, 98744 candidate probes\n",
      "--- 25536 probes passed check_dic selection.\n",
      "finish in 3.296s, 465 probes kept.\n",
      "-- check region:320 5:132237594-132324711_:+_gene_321, 172968 candidate probes\n",
      "--- 70799 probes passed check_dic selection.\n",
      "finish in 6.739s, 1079 probes kept.\n",
      "-- check region:321 5:133953020-134030324_:+_gene_322, 144678 candidate probes\n",
      "--- 48705 probes passed check_dic selection.\n",
      "finish in 5.246s, 788 probes kept.\n",
      "-- check region:322 5:135005402-135067436_:+_gene_323, 6820 candidate probes\n",
      "--- 2175 probes passed check_dic selection.\n",
      "finish in 0.311s, 38 probes kept.\n",
      "-- check region:323 5:135005402-135064026_:+_gene_324, 112748 candidate probes\n",
      "--- 41803 probes passed check_dic selection.\n",
      "finish in 4.302s, 679 probes kept.\n",
      "-- check region:324 5:138757956-138807859_:+_gene_325, 97782 candidate probes\n",
      "--- 49077 probes passed check_dic selection.\n",
      "finish in 4.343s, 652 probes kept.\n",
      "-- check region:325 5:141028058-141104622_:+_gene_326, 146336 candidate probes\n",
      "--- 54078 probes passed check_dic selection.\n",
      "finish in 5.608s, 856 probes kept.\n",
      "-- check region:326 5:141735482-141804869_:+_gene_327, 136748 candidate probes\n",
      "--- 54002 probes passed check_dic selection.\n",
      "finish in 5.215s, 817 probes kept.\n",
      "-- check region:327 5:142013394-142093792_:+_gene_328, 154726 candidate probes\n",
      "--- 63182 probes passed check_dic selection.\n",
      "finish in 6.064s, 948 probes kept.\n",
      "-- check region:328 5:143460529-143531173_:+_gene_329, 138342 candidate probes\n",
      "--- 47390 probes passed check_dic selection.\n",
      "finish in 5.167s, 774 probes kept.\n",
      "-- check region:329 5:144532912-144614769_:+_gene_330, 161475 candidate probes\n",
      "--- 55511 probes passed check_dic selection.\n",
      "finish in 5.645s, 870 probes kept.\n",
      "-- check region:330 5:147010488-147081113_:+_gene_331, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.017s, 0 probes kept.\n",
      "-- check region:331 5:147010488-147081113_:+_gene_332, 139452 candidate probes\n",
      "--- 66089 probes passed check_dic selection.\n",
      "finish in 5.660s, 914 probes kept.\n",
      "-- check region:332 5:148476071-148552936_:+_gene_333, 151272 candidate probes\n",
      "--- 60371 probes passed check_dic selection.\n",
      "finish in 5.579s, 915 probes kept.\n",
      "-- check region:333 5:150087399-150179014_:+_gene_334, 175510 candidate probes\n",
      "--- 69577 probes passed check_dic selection.\n",
      "finish in 6.436s, 1019 probes kept.\n",
      "-- check region:334 6:4913611-5062849_:+_gene_335, 291895 candidate probes\n",
      "--- 107724 probes passed check_dic selection.\n",
      "finish in 10.230s, 1629 probes kept.\n",
      "-- check region:335 6:5816895-5945849_:+_gene_336, 248225 candidate probes\n",
      "--- 102982 probes passed check_dic selection.\n",
      "finish in 9.279s, 1497 probes kept.\n",
      "-- check region:336 6:6142617-6218463_:+_gene_337, 149918 candidate probes\n",
      "--- 67856 probes passed check_dic selection.\n",
      "finish in 5.970s, 967 probes kept.\n",
      "-- check region:337 6:17092402-17181916_:+_gene_338, 174560 candidate probes\n",
      "--- 61314 probes passed check_dic selection.\n",
      "finish in 6.013s, 998 probes kept.\n",
      "-- check region:338 6:17574012-17640341_:+_gene_339, 128523 candidate probes\n",
      "--- 43631 probes passed check_dic selection.\n",
      "finish in 4.433s, 691 probes kept.\n",
      "-- check region:339 6:18398404-18461976_:+_gene_340, 124932 candidate probes\n",
      "--- 42912 probes passed check_dic selection.\n",
      "finish in 4.345s, 741 probes kept.\n",
      "-- check region:340 6:23436147-23521005_:+_gene_341, 136894 candidate probes\n",
      "--- 43796 probes passed check_dic selection.\n",
      "finish in 4.620s, 734 probes kept.\n",
      "-- check region:341 6:24510973-24581391_:+_gene_342, 135532 candidate probes\n",
      "--- 44387 probes passed check_dic selection.\n",
      "finish in 4.722s, 738 probes kept.\n",
      "-- check region:342 6:27875819-27995930_:+_gene_343, 236676 candidate probes\n",
      "--- 80382 probes passed check_dic selection.\n",
      "finish in 7.942s, 1305 probes kept.\n",
      "-- check region:343 6:29188389-29226427_:+_gene_344, 73194 candidate probes\n",
      "--- 30144 probes passed check_dic selection.\n",
      "finish in 2.890s, 445 probes kept.\n",
      "-- check region:344 6:30517033-30590432_:+_gene_345, 144366 candidate probes\n",
      "--- 60656 probes passed check_dic selection.\n",
      "finish in 5.512s, 916 probes kept.\n",
      "-- check region:345 6:34471419-34525684_:+_gene_346, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.014s, 0 probes kept.\n",
      "-- check region:346 6:34471419-34525893_:+_gene_347, 107986 candidate probes\n",
      "--- 48481 probes passed check_dic selection.\n",
      "finish in 4.324s, 728 probes kept.\n",
      "-- check region:347 6:34856828-34912427_:+_gene_348, 109740 candidate probes\n",
      "--- 45861 probes passed check_dic selection.\n",
      "finish in 4.255s, 709 probes kept.\n",
      "-- check region:348 6:41067178-41147415_:+_gene_349, 156326 candidate probes\n",
      "--- 47786 probes passed check_dic selection.\n",
      "finish in 5.069s, 834 probes kept.\n",
      "-- check region:349 6:41494636-41570746_:+_gene_350, 147524 candidate probes\n",
      "--- 52136 probes passed check_dic selection.\n",
      "finish in 5.151s, 833 probes kept.\n",
      "-- check region:350 6:42329795-42373704_:+_gene_351, 84828 candidate probes\n",
      "--- 36773 probes passed check_dic selection.\n",
      "finish in 3.288s, 579 probes kept.\n",
      "-- check region:351 6:48379712-48444906_:+_gene_352, 129178 candidate probes\n",
      "--- 48677 probes passed check_dic selection.\n",
      "finish in 4.723s, 839 probes kept.\n",
      "-- check region:352 6:49807253-49847041_:+_gene_353, 78940 candidate probes\n",
      "--- 32243 probes passed check_dic selection.\n",
      "finish in 2.932s, 479 probes kept.\n",
      "-- check region:353 6:51158260-51195983_:+_gene_354, 73970 candidate probes\n",
      "--- 35857 probes passed check_dic selection.\n",
      "finish in 3.101s, 483 probes kept.\n",
      "-- check region:354 6:53570205-53695638_:+_gene_355, 248960 candidate probes\n",
      "--- 115725 probes passed check_dic selection.\n",
      "finish in 9.914s, 1677 probes kept.\n",
      "-- check region:355 6:54688853-54746443_:+_gene_356, 112950 candidate probes\n",
      "--- 49680 probes passed check_dic selection.\n",
      "finish in 4.465s, 735 probes kept.\n",
      "-- check region:356 6:55781236-55825137_:+_gene_357, 86024 candidate probes\n",
      "--- 37070 probes passed check_dic selection.\n",
      "finish in 3.307s, 530 probes kept.\n",
      "-- check region:357 6:57733581-57826248_:+_gene_358, 180364 candidate probes\n",
      "--- 56276 probes passed check_dic selection.\n",
      "finish in 5.967s, 950 probes kept.\n",
      "-- check region:358 6:72814718-72899657_:+_gene_359, 164584 candidate probes\n",
      "--- 56079 probes passed check_dic selection.\n",
      "finish in 5.687s, 936 probes kept.\n",
      "-- check region:359 6:77403708-77486471_:+_gene_360, 163292 candidate probes\n",
      "--- 54477 probes passed check_dic selection.\n",
      "finish in 5.494s, 923 probes kept.\n",
      "-- check region:360 6:81772875-81833647_:+_gene_361, 113138 candidate probes\n",
      "--- 38206 probes passed check_dic selection.\n",
      "finish in 3.861s, 597 probes kept.\n",
      "-- check region:361 6:81965780-82001014_:+_gene_362, 60604 candidate probes\n",
      "--- 19665 probes passed check_dic selection.\n",
      "finish in 2.096s, 320 probes kept.\n",
      "-- check region:362 6:82986732-83034917_:+_gene_363, 94750 candidate probes\n",
      "--- 33067 probes passed check_dic selection.\n",
      "finish in 3.342s, 529 probes kept.\n",
      "-- check region:363 6:83174995-83226237_:+_gene_364, 96882 candidate probes\n",
      "--- 41633 probes passed check_dic selection.\n",
      "finish in 3.822s, 646 probes kept.\n",
      "-- check region:364 6:83847140-83910764_:+_gene_365, 122872 candidate probes\n",
      "--- 33049 probes passed check_dic selection.\n",
      "finish in 3.883s, 581 probes kept.\n",
      "-- check region:365 6:84109647-84145755_:+_gene_366, 71460 candidate probes\n",
      "--- 40559 probes passed check_dic selection.\n",
      "finish in 3.255s, 520 probes kept.\n",
      "-- check region:366 6:84552937-84600580_:+_gene_367, 94478 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 48583 probes passed check_dic selection.\n",
      "finish in 4.139s, 656 probes kept.\n",
      "-- check region:367 6:90393398-90445394_:+_gene_368, 89758 candidate probes\n",
      "--- 43100 probes passed check_dic selection.\n",
      "finish in 3.720s, 621 probes kept.\n",
      "-- check region:368 6:97827648-97913932_:+_gene_369, 163572 candidate probes\n",
      "--- 72202 probes passed check_dic selection.\n",
      "finish in 6.312s, 1019 probes kept.\n",
      "-- check region:369 6:98198965-98298774_:+_gene_370, 194065 candidate probes\n",
      "--- 76218 probes passed check_dic selection.\n",
      "finish in 7.008s, 1182 probes kept.\n",
      "-- check region:370 6:98920069-99030726_:+_gene_371, 220732 candidate probes\n",
      "--- 110095 probes passed check_dic selection.\n",
      "finish in 9.044s, 1516 probes kept.\n",
      "-- check region:371 6:99319695-99393737_:+_gene_372, 146822 candidate probes\n",
      "--- 67640 probes passed check_dic selection.\n",
      "finish in 5.890s, 970 probes kept.\n",
      "-- check region:372 6:100901891-100949915_:+_gene_373, 92092 candidate probes\n",
      "--- 37796 probes passed check_dic selection.\n",
      "finish in 3.596s, 547 probes kept.\n",
      "-- check region:373 6:106370677-106451259_:+_gene_374, 146168 candidate probes\n",
      "--- 42835 probes passed check_dic selection.\n",
      "finish in 4.767s, 737 probes kept.\n",
      "-- check region:374 6:111471365-111546119_:+_gene_375, 146400 candidate probes\n",
      "--- 44586 probes passed check_dic selection.\n",
      "finish in 4.694s, 757 probes kept.\n",
      "-- check region:375 6:113251068-113297839_:+_gene_376, 92062 candidate probes\n",
      "--- 41745 probes passed check_dic selection.\n",
      "finish in 3.711s, 626 probes kept.\n",
      "-- check region:376 6:115452622-115533851_:+_gene_377, 159096 candidate probes\n",
      "--- 67551 probes passed check_dic selection.\n",
      "finish in 6.003s, 1004 probes kept.\n",
      "-- check region:377 6:116336040-116434204_:+_gene_378, 191495 candidate probes\n",
      "--- 65359 probes passed check_dic selection.\n",
      "finish in 6.622s, 1116 probes kept.\n",
      "-- check region:378 6:118583580-118659770_:+_gene_379, 151926 candidate probes\n",
      "--- 75126 probes passed check_dic selection.\n",
      "finish in 6.301s, 1102 probes kept.\n",
      "-- check region:379 6:121356679-121478570_:+_gene_380, 239720 candidate probes\n",
      "--- 116187 probes passed check_dic selection.\n",
      "finish in 9.845s, 1650 probes kept.\n",
      "-- check region:380 6:127069741-127152836_:+_gene_381, 154070 candidate probes\n",
      "--- 64646 probes passed check_dic selection.\n",
      "finish in 5.979s, 984 probes kept.\n",
      "-- check region:381 6:128391401-128443169_:+_gene_382, 100074 candidate probes\n",
      "--- 35933 probes passed check_dic selection.\n",
      "finish in 3.607s, 624 probes kept.\n",
      "-- check region:382 6:131356549-131444281_:+_gene_383, 172650 candidate probes\n",
      "--- 51380 probes passed check_dic selection.\n",
      "finish in 5.686s, 900 probes kept.\n",
      "-- check region:383 6:134686900-134773941_:+_gene_384, 166488 candidate probes\n",
      "--- 55889 probes passed check_dic selection.\n",
      "finish in 5.686s, 926 probes kept.\n",
      "-- check region:384 6:137078345-137173906_:+_gene_385, 185636 candidate probes\n",
      "--- 79045 probes passed check_dic selection.\n",
      "finish in 7.045s, 1123 probes kept.\n",
      "-- check region:385 6:142477481-142535279_:+_gene_386, 113564 candidate probes\n",
      "--- 43485 probes passed check_dic selection.\n",
      "finish in 4.307s, 675 probes kept.\n",
      "-- check region:386 6:144585170-144652901_:+_gene_387, 132870 candidate probes\n",
      "--- 58804 probes passed check_dic selection.\n",
      "finish in 5.188s, 847 probes kept.\n",
      "-- check region:387 6:144980272-145047109_:+_gene_388, 128544 candidate probes\n",
      "--- 48965 probes passed check_dic selection.\n",
      "finish in 4.773s, 763 probes kept.\n",
      "-- check region:388 6:145265832-145317520_:+_gene_389, 101580 candidate probes\n",
      "--- 38226 probes passed check_dic selection.\n",
      "finish in 3.622s, 604 probes kept.\n",
      "-- check region:389 6:145723149-145813383_:+_gene_390, 172900 candidate probes\n",
      "--- 75858 probes passed check_dic selection.\n",
      "finish in 6.804s, 1111 probes kept.\n",
      "-- check region:390 6:146320927-146368669_:+_gene_391, 94636 candidate probes\n",
      "--- 40968 probes passed check_dic selection.\n",
      "finish in 3.729s, 592 probes kept.\n",
      "-- check region:391 7:4468044-4523679_:+_gene_392, 109374 candidate probes\n",
      "--- 46738 probes passed check_dic selection.\n",
      "finish in 4.422s, 731 probes kept.\n",
      "-- check region:392 7:25614559-25661968_:+_gene_393, 93058 candidate probes\n",
      "--- 36883 probes passed check_dic selection.\n",
      "finish in 3.537s, 593 probes kept.\n",
      "-- check region:393 7:28165434-28224615_:+_gene_394, 114044 candidate probes\n",
      "--- 39837 probes passed check_dic selection.\n",
      "finish in 4.139s, 622 probes kept.\n",
      "-- check region:394 7:28549913-28611972_:+_gene_395, 121762 candidate probes\n",
      "--- 60610 probes passed check_dic selection.\n",
      "finish in 5.122s, 819 probes kept.\n",
      "-- check region:395 7:30246410-30311465_:+_gene_396, 126756 candidate probes\n",
      "--- 52089 probes passed check_dic selection.\n",
      "finish in 4.829s, 813 probes kept.\n",
      "-- check region:396 7:30885416-30938012_:+_gene_397, 90142 candidate probes\n",
      "--- 30186 probes passed check_dic selection.\n",
      "finish in 3.337s, 475 probes kept.\n",
      "-- check region:397 7:35676730-35786268_:+_gene_398, 215538 candidate probes\n",
      "--- 78307 probes passed check_dic selection.\n",
      "finish in 7.710s, 1242 probes kept.\n",
      "-- check region:398 7:36035635-36095581_:+_gene_399, 119016 candidate probes\n",
      "--- 59472 probes passed check_dic selection.\n",
      "finish in 4.944s, 811 probes kept.\n",
      "-- check region:399 7:36903879-36964287_:+_gene_400, 119410 candidate probes\n",
      "--- 57347 probes passed check_dic selection.\n",
      "finish in 4.871s, 782 probes kept.\n",
      "-- check region:400 7:43533744-43582104_:+_gene_401, 94632 candidate probes\n",
      "--- 33447 probes passed check_dic selection.\n",
      "finish in 3.292s, 549 probes kept.\n",
      "-- check region:401 7:47046107-47127036_:+_gene_402, 11894 candidate probes\n",
      "--- 3592 probes passed check_dic selection.\n",
      "finish in 0.495s, 73 probes kept.\n",
      "-- check region:402 7:47046107-47121005_:+_gene_403, 148720 candidate probes\n",
      "--- 75548 probes passed check_dic selection.\n",
      "finish in 6.386s, 1043 probes kept.\n",
      "-- check region:403 7:50493534-50595678_:+_gene_404, 180580 candidate probes\n",
      "--- 62253 probes passed check_dic selection.\n",
      "finish in 6.191s, 1014 probes kept.\n",
      "-- check region:404 7:64285872-64371201_:+_gene_405, 168020 candidate probes\n",
      "--- 66696 probes passed check_dic selection.\n",
      "finish in 6.170s, 1040 probes kept.\n",
      "-- check region:405 7:65860622-65945471_:+_gene_406, 167746 candidate probes\n",
      "--- 81609 probes passed check_dic selection.\n",
      "finish in 6.898s, 1114 probes kept.\n",
      "-- check region:406 7:67813905-67893031_:+_gene_407, 156514 candidate probes\n",
      "--- 60753 probes passed check_dic selection.\n",
      "finish in 5.742s, 915 probes kept.\n",
      "-- check region:407 7:70316208-70366368_:+_gene_408, 99068 candidate probes\n",
      "--- 46252 probes passed check_dic selection.\n",
      "finish in 3.960s, 694 probes kept.\n",
      "-- check region:408 7:81251500-81332036_:+_gene_409, 158830 candidate probes\n",
      "--- 61036 probes passed check_dic selection.\n",
      "finish in 5.742s, 907 probes kept.\n",
      "-- check region:409 7:82791317-82863443_:+_gene_410, 140956 candidate probes\n",
      "--- 51183 probes passed check_dic selection.\n",
      "finish in 5.063s, 786 probes kept.\n",
      "-- check region:410 7:83034578-83105834_:+_gene_411, 140016 candidate probes\n",
      "--- 53147 probes passed check_dic selection.\n",
      "finish in 5.111s, 841 probes kept.\n",
      "-- check region:411 7:83629439-83686053_:+_gene_412, 112242 candidate probes\n",
      "--- 49421 probes passed check_dic selection.\n",
      "finish in 4.374s, 764 probes kept.\n",
      "-- check region:412 7:83790534-83849178_:+_gene_413, 114586 candidate probes\n",
      "--- 47034 probes passed check_dic selection.\n",
      "finish in 4.371s, 725 probes kept.\n",
      "-- check region:413 7:84235734-84311961_:+_gene_414, 150982 candidate probes\n",
      "--- 67515 probes passed check_dic selection.\n",
      "finish in 5.872s, 985 probes kept.\n",
      "-- check region:414 7:89259710-89342531_:+_gene_415, 156788 candidate probes\n",
      "--- 35928 probes passed check_dic selection.\n",
      "finish in 4.639s, 636 probes kept.\n",
      "-- check region:415 7:97735161-97815491_:+_gene_416, 155920 candidate probes\n",
      "--- 47754 probes passed check_dic selection.\n",
      "finish in 5.139s, 804 probes kept.\n",
      "-- check region:416 7:98125903-98181642_:+_gene_417, 108008 candidate probes\n",
      "--- 53000 probes passed check_dic selection.\n",
      "finish in 4.525s, 766 probes kept.\n",
      "-- check region:417 7:101890453-101928530_:+_gene_418, 73680 candidate probes\n",
      "--- 22882 probes passed check_dic selection.\n",
      "finish in 2.558s, 378 probes kept.\n",
      "-- check region:418 7:107651264-107719048_:+_gene_419, 134834 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 61238 probes passed check_dic selection.\n",
      "finish in 5.456s, 901 probes kept.\n",
      "-- check region:419 7:109914369-109963604_:+_gene_420, 94782 candidate probes\n",
      "--- 29226 probes passed check_dic selection.\n",
      "finish in 3.145s, 512 probes kept.\n",
      "-- check region:420 7:112752685-112894620_:+_gene_421, 280034 candidate probes\n",
      "--- 132423 probes passed check_dic selection.\n",
      "finish in 11.290s, 1854 probes kept.\n",
      "-- check region:421 7:114281439-114333757_:+_gene_422, 102918 candidate probes\n",
      "--- 32240 probes passed check_dic selection.\n",
      "finish in 3.443s, 552 probes kept.\n",
      "-- check region:422 7:115477765-115548476_:+_gene_423, 137390 candidate probes\n",
      "--- 56889 probes passed check_dic selection.\n",
      "finish in 5.145s, 874 probes kept.\n",
      "-- check region:423 7:115636108-115717089_:+_gene_424, 158454 candidate probes\n",
      "--- 56001 probes passed check_dic selection.\n",
      "finish in 5.520s, 895 probes kept.\n",
      "-- check region:424 7:118113230-118170731_:+_gene_425, 112175 candidate probes\n",
      "--- 31778 probes passed check_dic selection.\n",
      "finish in 3.594s, 637 probes kept.\n",
      "-- check region:425 7:119716960-119755414_:+_gene_426, 73534 candidate probes\n",
      "--- 15914 probes passed check_dic selection.\n",
      "finish in 2.200s, 313 probes kept.\n",
      "-- check region:426 7:121203744-121312825_:+_gene_427, 214906 candidate probes\n",
      "--- 85998 probes passed check_dic selection.\n",
      "finish in 7.918s, 1289 probes kept.\n",
      "-- check region:427 7:126238527-126290938_:+_gene_428, 101864 candidate probes\n",
      "--- 48766 probes passed check_dic selection.\n",
      "finish in 4.131s, 661 probes kept.\n",
      "-- check region:428 7:127071384-127140312_:+_gene_429, 134670 candidate probes\n",
      "--- 46856 probes passed check_dic selection.\n",
      "finish in 4.714s, 757 probes kept.\n",
      "-- check region:429 7:127626855-127710283_:+_gene_430, 160400 candidate probes\n",
      "--- 50061 probes passed check_dic selection.\n",
      "finish in 5.308s, 825 probes kept.\n",
      "-- check region:430 7:130205243-130269178_:+_gene_431, 127024 candidate probes\n",
      "--- 63967 probes passed check_dic selection.\n",
      "finish in 5.408s, 886 probes kept.\n",
      "-- check region:431 7:132216605-132295072_:+_gene_432, 153918 candidate probes\n",
      "--- 78390 probes passed check_dic selection.\n",
      "finish in 6.511s, 1068 probes kept.\n",
      "-- check region:432 7:134653339-134690281_:+_gene_433, 72924 candidate probes\n",
      "--- 29177 probes passed check_dic selection.\n",
      "finish in 2.867s, 445 probes kept.\n",
      "-- check region:433 7:136036278-136105849_:+_gene_434, 137556 candidate probes\n",
      "--- 64434 probes passed check_dic selection.\n",
      "finish in 5.455s, 916 probes kept.\n",
      "-- check region:434 7:138166035-138259068_:+_gene_435, 183722 candidate probes\n",
      "--- 93293 probes passed check_dic selection.\n",
      "finish in 7.566s, 1263 probes kept.\n",
      "-- check region:435 7:142323852-142438691_:+_gene_436, 227286 candidate probes\n",
      "--- 104781 probes passed check_dic selection.\n",
      "finish in 9.011s, 1502 probes kept.\n",
      "-- check region:436 8:8590019-8660327_:+_gene_437, 139106 candidate probes\n",
      "--- 65493 probes passed check_dic selection.\n",
      "finish in 5.610s, 957 probes kept.\n",
      "-- check region:437 8:13909259-13976263_:+_gene_438, 127674 candidate probes\n",
      "--- 60369 probes passed check_dic selection.\n",
      "finish in 5.219s, 854 probes kept.\n",
      "-- check region:438 8:15539842-15619200_:+_gene_439, 141268 candidate probes\n",
      "--- 53121 probes passed check_dic selection.\n",
      "finish in 5.088s, 779 probes kept.\n",
      "-- check region:439 8:15984403-16043579_:+_gene_440, 114178 candidate probes\n",
      "--- 47219 probes passed check_dic selection.\n",
      "finish in 4.365s, 709 probes kept.\n",
      "-- check region:440 8:17444670-17529064_:+_gene_441, 154660 candidate probes\n",
      "--- 56830 probes passed check_dic selection.\n",
      "finish in 5.411s, 916 probes kept.\n",
      "-- check region:441 8:18906539-18964569_:+_gene_442, 114570 candidate probes\n",
      "--- 56416 probes passed check_dic selection.\n",
      "finish in 4.757s, 772 probes kept.\n",
      "-- check region:442 8:20105193-20165338_:+_gene_443, 99986 candidate probes\n",
      "--- 22 probes passed check_dic selection.\n",
      "finish in 2.043s, 3 probes kept.\n",
      "-- check region:443 8:25338122-25379634_:+_gene_444, 80714 candidate probes\n",
      "--- 32358 probes passed check_dic selection.\n",
      "finish in 2.990s, 466 probes kept.\n",
      "-- check region:444 8:26676886-26724279_:+_gene_445, 93146 candidate probes\n",
      "--- 41171 probes passed check_dic selection.\n",
      "finish in 3.687s, 594 probes kept.\n",
      "-- check region:445 8:34805877-34903394_:+_gene_446, 192524 candidate probes\n",
      "--- 75141 probes passed check_dic selection.\n",
      "finish in 6.983s, 1186 probes kept.\n",
      "-- check region:446 8:35149287-35254368_:+_gene_447, 202396 candidate probes\n",
      "--- 86693 probes passed check_dic selection.\n",
      "finish in 7.623s, 1238 probes kept.\n",
      "-- check region:447 8:35358926-35444058_:+_gene_448, 167900 candidate probes\n",
      "--- 69321 probes passed check_dic selection.\n",
      "finish in 6.226s, 1055 probes kept.\n",
      "-- check region:448 8:36400955-36469588_:+_gene_449, 134168 candidate probes\n",
      "--- 47484 probes passed check_dic selection.\n",
      "finish in 4.645s, 759 probes kept.\n",
      "-- check region:449 8:36619739-36725384_:+_gene_450, 205694 candidate probes\n",
      "--- 81031 probes passed check_dic selection.\n",
      "finish in 7.493s, 1217 probes kept.\n",
      "-- check region:450 8:40562713-40638441_:+_gene_451, 149284 candidate probes\n",
      "--- 64311 probes passed check_dic selection.\n",
      "finish in 5.816s, 913 probes kept.\n",
      "-- check region:451 8:41328066-41394762_:+_gene_452, 130236 candidate probes\n",
      "--- 43095 probes passed check_dic selection.\n",
      "finish in 4.399s, 698 probes kept.\n",
      "-- check region:452 8:47670301-47717341_:+_gene_453, 91782 candidate probes\n",
      "--- 28294 probes passed check_dic selection.\n",
      "finish in 3.049s, 504 probes kept.\n",
      "-- check region:453 8:47821006-47913730_:+_gene_454, 176360 candidate probes\n",
      "--- 80348 probes passed check_dic selection.\n",
      "finish in 6.934s, 1148 probes kept.\n",
      "-- check region:454 8:48176246-48232764_:+_gene_455, 98336 candidate probes\n",
      "--- 47306 probes passed check_dic selection.\n",
      "finish in 4.018s, 638 probes kept.\n",
      "-- check region:455 8:48470073-48578850_:+_gene_456, 214098 candidate probes\n",
      "--- 93246 probes passed check_dic selection.\n",
      "finish in 8.145s, 1329 probes kept.\n",
      "-- check region:456 8:61239539-61344852_:+_gene_457, 207047 candidate probes\n",
      "--- 89640 probes passed check_dic selection.\n",
      "finish in 7.847s, 1317 probes kept.\n",
      "-- check region:457 8:67928931-68005568_:+_gene_458, 150286 candidate probes\n",
      "--- 55087 probes passed check_dic selection.\n",
      "finish in 5.276s, 887 probes kept.\n",
      "-- check region:458 8:68161685-68224740_:+_gene_459, 113832 candidate probes\n",
      "--- 37768 probes passed check_dic selection.\n",
      "finish in 3.883s, 623 probes kept.\n",
      "-- check region:459 8:68504342-68569036_:+_gene_460, 123018 candidate probes\n",
      "--- 43600 probes passed check_dic selection.\n",
      "finish in 4.284s, 693 probes kept.\n",
      "-- check region:460 8:71286208-71369935_:+_gene_461, 164926 candidate probes\n",
      "--- 63690 probes passed check_dic selection.\n",
      "finish in 6.050s, 1004 probes kept.\n",
      "-- check region:461 8:72632943-72708838_:+_gene_462, 130216 candidate probes\n",
      "--- 40333 probes passed check_dic selection.\n",
      "finish in 4.383s, 686 probes kept.\n",
      "-- check region:462 8:75033814-75099992_:+_gene_463, 129722 candidate probes\n",
      "--- 60554 probes passed check_dic selection.\n",
      "finish in 5.207s, 828 probes kept.\n",
      "-- check region:463 8:76990149-77104743_:+_gene_464, 223838 candidate probes\n",
      "--- 79314 probes passed check_dic selection.\n",
      "finish in 7.707s, 1220 probes kept.\n",
      "-- check region:464 8:78008249-78074137_:+_gene_465, 127724 candidate probes\n",
      "--- 42512 probes passed check_dic selection.\n",
      "finish in 4.364s, 637 probes kept.\n",
      "-- check region:465 8:79321964-79402981_:+_gene_466, 160354 candidate probes\n",
      "--- 72879 probes passed check_dic selection.\n",
      "finish in 6.309s, 1060 probes kept.\n",
      "-- check region:466 8:81331631-81410269_:+_gene_467, 154652 candidate probes\n",
      "--- 51110 probes passed check_dic selection.\n",
      "finish in 5.162s, 849 probes kept.\n",
      "-- check region:467 8:83392102-83461711_:+_gene_468, 132388 candidate probes\n",
      "--- 40515 probes passed check_dic selection.\n",
      "finish in 4.567s, 663 probes kept.\n",
      "-- check region:468 8:84173157-84214478_:+_gene_469, 80476 candidate probes\n",
      "--- 35649 probes passed check_dic selection.\n",
      "finish in 3.137s, 517 probes kept.\n",
      "-- check region:469 8:88269593-88342980_:+_gene_470, 144282 candidate probes\n",
      "--- 65464 probes passed check_dic selection.\n",
      "finish in 5.787s, 942 probes kept.\n",
      "-- check region:470 8:90263317-90334986_:+_gene_471, 142030 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 65375 probes passed check_dic selection.\n",
      "finish in 5.551s, 938 probes kept.\n",
      "-- check region:471 8:90481007-90553012_:+_gene_472, 141918 candidate probes\n",
      "--- 63354 probes passed check_dic selection.\n",
      "finish in 5.498s, 934 probes kept.\n",
      "-- check region:472 8:90827081-90904757_:+_gene_473, 149760 candidate probes\n",
      "--- 49264 probes passed check_dic selection.\n",
      "finish in 5.090s, 766 probes kept.\n",
      "-- check region:473 8:92109780-92144791_:+_gene_474, 63534 candidate probes\n",
      "--- 29917 probes passed check_dic selection.\n",
      "finish in 2.806s, 412 probes kept.\n",
      "-- check region:474 8:95146834-95186362_:+_gene_475, 76770 candidate probes\n",
      "--- 31140 probes passed check_dic selection.\n",
      "finish in 2.955s, 451 probes kept.\n",
      "-- check region:475 8:103822726-103892126_:+_gene_476, 136664 candidate probes\n",
      "--- 59176 probes passed check_dic selection.\n",
      "finish in 5.308s, 862 probes kept.\n",
      "-- check region:476 8:104393397-104447891_:+_gene_477, 105878 candidate probes\n",
      "--- 40828 probes passed check_dic selection.\n",
      "finish in 3.843s, 628 probes kept.\n",
      "-- check region:477 8:105422888-105494815_:+_gene_478, 140656 candidate probes\n",
      "--- 55740 probes passed check_dic selection.\n",
      "finish in 5.164s, 866 probes kept.\n",
      "-- check region:478 8:107434145-107533024_:+_gene_479, 191708 candidate probes\n",
      "--- 71131 probes passed check_dic selection.\n",
      "finish in 6.876s, 1072 probes kept.\n",
      "-- check region:479 8:109893221-109994503_:+_gene_480, 197644 candidate probes\n",
      "--- 67972 probes passed check_dic selection.\n",
      "finish in 6.807s, 1115 probes kept.\n",
      "-- check region:480 8:110915709-110996339_:+_gene_481, 156644 candidate probes\n",
      "--- 57585 probes passed check_dic selection.\n",
      "finish in 5.659s, 897 probes kept.\n",
      "-- check region:481 8:112606029-112692110_:+_gene_482, 157970 candidate probes\n",
      "--- 52733 probes passed check_dic selection.\n",
      "finish in 5.328s, 854 probes kept.\n",
      "-- check region:482 8:114440631-114516127_:+_gene_483, 146966 candidate probes\n",
      "--- 72607 probes passed check_dic selection.\n",
      "finish in 6.025s, 984 probes kept.\n",
      "-- check region:483 8:114888227-115019038_:+_gene_484, 256894 candidate probes\n",
      "--- 125153 probes passed check_dic selection.\n",
      "finish in 10.311s, 1698 probes kept.\n",
      "-- check region:484 8:115121077-115190969_:+_gene_485, 134922 candidate probes\n",
      "--- 68955 probes passed check_dic selection.\n",
      "finish in 5.610s, 936 probes kept.\n",
      "-- check region:485 8:116572552-116632019_:+_gene_486, 117480 candidate probes\n",
      "--- 57937 probes passed check_dic selection.\n",
      "finish in 4.919s, 768 probes kept.\n",
      "-- check region:486 8:118085022-118172697_:+_gene_487, 172832 candidate probes\n",
      "--- 82408 probes passed check_dic selection.\n",
      "finish in 6.922s, 1125 probes kept.\n",
      "-- check region:487 8:119110924-119171282_:+_gene_488, 118406 candidate probes\n",
      "--- 56095 probes passed check_dic selection.\n",
      "finish in 4.747s, 776 probes kept.\n",
      "-- check region:488 8:119775828-119840110_:+_gene_489, 126684 candidate probes\n",
      "--- 59716 probes passed check_dic selection.\n",
      "finish in 5.146s, 829 probes kept.\n",
      "-- check region:489 9:2990250-3046292_:+_gene_490, 29992 candidate probes\n",
      "--- 234 probes passed check_dic selection.\n",
      "finish in 0.688s, 19 probes kept.\n",
      "-- check region:490 9:8889764-8938793_:+_gene_491, 96690 candidate probes\n",
      "--- 28612 probes passed check_dic selection.\n",
      "finish in 3.201s, 465 probes kept.\n",
      "-- check region:491 9:9083125-9155126_:+_gene_492, 142046 candidate probes\n",
      "--- 47160 probes passed check_dic selection.\n",
      "finish in 4.776s, 762 probes kept.\n",
      "-- check region:492 9:14502860-14595101_:+_gene_493, 178900 candidate probes\n",
      "--- 76098 probes passed check_dic selection.\n",
      "finish in 6.888s, 1115 probes kept.\n",
      "-- check region:493 9:15483272-15563906_:+_gene_494, 159118 candidate probes\n",
      "--- 79029 probes passed check_dic selection.\n",
      "finish in 6.525s, 1085 probes kept.\n",
      "-- check region:494 9:22547286-22665336_:+_gene_495, 227646 candidate probes\n",
      "--- 74852 probes passed check_dic selection.\n",
      "finish in 7.628s, 1180 probes kept.\n",
      "-- check region:495 9:22783499-22864592_:+_gene_496, 153454 candidate probes\n",
      "--- 56211 probes passed check_dic selection.\n",
      "finish in 5.440s, 831 probes kept.\n",
      "-- check region:496 9:24398454-24493239_:+_gene_497, 186442 candidate probes\n",
      "--- 67435 probes passed check_dic selection.\n",
      "finish in 6.515s, 1063 probes kept.\n",
      "-- check region:497 9:32479082-32541627_:+_gene_498, 123308 candidate probes\n",
      "--- 63904 probes passed check_dic selection.\n",
      "finish in 5.191s, 862 probes kept.\n",
      "-- check region:498 9:32691049-32769587_:+_gene_499, 156594 candidate probes\n",
      "--- 83022 probes passed check_dic selection.\n",
      "finish in 6.644s, 1128 probes kept.\n",
      "-- check region:499 9:37396529-37448756_:+_gene_500, 93102 candidate probes\n",
      "--- 38644 probes passed check_dic selection.\n",
      "finish in 3.658s, 617 probes kept.\n",
      "-- check region:500 9:46291898-46364450_:+_gene_501, 139434 candidate probes\n",
      "--- 50219 probes passed check_dic selection.\n",
      "finish in 4.958s, 796 probes kept.\n",
      "-- check region:501 9:48007145-48083481_:+_gene_502, 149932 candidate probes\n",
      "--- 51430 probes passed check_dic selection.\n",
      "finish in 5.254s, 847 probes kept.\n",
      "-- check region:502 9:48176289-48277700_:+_gene_503, 198702 candidate probes\n",
      "--- 90512 probes passed check_dic selection.\n",
      "finish in 7.790s, 1295 probes kept.\n",
      "-- check region:503 9:50054571-50160541_:+_gene_504, 192042 candidate probes\n",
      "--- 79456 probes passed check_dic selection.\n",
      "finish in 7.153s, 1198 probes kept.\n",
      "-- check region:504 9:52357436-52446749_:+_gene_505, 157630 candidate probes\n",
      "--- 56242 probes passed check_dic selection.\n",
      "finish in 5.480s, 922 probes kept.\n",
      "-- check region:505 9:53893431-53958595_:+_gene_506, 127674 candidate probes\n",
      "--- 45614 probes passed check_dic selection.\n",
      "finish in 4.466s, 741 probes kept.\n",
      "-- check region:506 9:54298886-54354997_:+_gene_507, 110540 candidate probes\n",
      "--- 42475 probes passed check_dic selection.\n",
      "finish in 4.134s, 661 probes kept.\n",
      "-- check region:507 9:54508565-54565386_:+_gene_508, 110422 candidate probes\n",
      "--- 46400 probes passed check_dic selection.\n",
      "finish in 4.282s, 674 probes kept.\n",
      "-- check region:508 9:54645501-54702064_:+_gene_509, 111246 candidate probes\n",
      "--- 45776 probes passed check_dic selection.\n",
      "finish in 4.326s, 685 probes kept.\n",
      "-- check region:509 9:57240601-57299305_:+_gene_510, 116182 candidate probes\n",
      "--- 54904 probes passed check_dic selection.\n",
      "finish in 4.770s, 781 probes kept.\n",
      "-- check region:510 9:63600882-63666718_:+_gene_511, 130130 candidate probes\n",
      "--- 63833 probes passed check_dic selection.\n",
      "finish in 5.433s, 914 probes kept.\n",
      "-- check region:511 9:64174332-64252851_:+_gene_512, 150050 candidate probes\n",
      "--- 53066 probes passed check_dic selection.\n",
      "finish in 5.297s, 857 probes kept.\n",
      "-- check region:512 9:67385710-67451674_:+_gene_513, 129638 candidate probes\n",
      "--- 52796 probes passed check_dic selection.\n",
      "finish in 4.891s, 804 probes kept.\n",
      "-- check region:513 9:67673420-67763632_:+_gene_514, 177824 candidate probes\n",
      "--- 68603 probes passed check_dic selection.\n",
      "finish in 6.496s, 1073 probes kept.\n",
      "-- check region:514 9:69074838-69145422_:+_gene_515, 136766 candidate probes\n",
      "--- 64504 probes passed check_dic selection.\n",
      "finish in 5.431s, 900 probes kept.\n",
      "-- check region:515 9:71524978-71592718_:+_gene_516, 134560 candidate probes\n",
      "--- 55101 probes passed check_dic selection.\n",
      "finish in 4.995s, 819 probes kept.\n",
      "-- check region:516 9:71901483-71970383_:+_gene_517, 125450 candidate probes\n",
      "--- 33385 probes passed check_dic selection.\n",
      "finish in 3.847s, 618 probes kept.\n",
      "-- check region:517 9:73144502-73202707_:+_gene_518, 112762 candidate probes\n",
      "--- 33588 probes passed check_dic selection.\n",
      "finish in 3.640s, 582 probes kept.\n",
      "-- check region:518 9:77698777-77761284_:+_gene_519, 122776 candidate probes\n",
      "--- 45248 probes passed check_dic selection.\n",
      "finish in 4.399s, 710 probes kept.\n",
      "-- check region:519 9:78111826-78179455_:+_gene_520, 133444 candidate probes\n",
      "--- 51867 probes passed check_dic selection.\n",
      "finish in 4.876s, 815 probes kept.\n",
      "-- check region:520 9:80159682-80266480_:+_gene_521, 207423 candidate probes\n",
      "--- 72388 probes passed check_dic selection.\n",
      "finish in 7.113s, 1160 probes kept.\n",
      "-- check region:521 9:83294219-83369953_:+_gene_522, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.013s, 0 probes kept.\n",
      "-- check region:522 9:83294219-83369953_:+_gene_523, 149186 candidate probes\n",
      "--- 49517 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 5.039s, 816 probes kept.\n",
      "-- check region:523 9:86573601-86647179_:+_gene_524, 143680 candidate probes\n",
      "--- 55599 probes passed check_dic selection.\n",
      "finish in 5.244s, 825 probes kept.\n",
      "-- check region:524 9:89246295-89303992_:+_gene_525, 112838 candidate probes\n",
      "--- 41491 probes passed check_dic selection.\n",
      "finish in 4.007s, 620 probes kept.\n",
      "-- check region:525 9:89831577-89911977_:+_gene_526, 154144 candidate probes\n",
      "--- 67464 probes passed check_dic selection.\n",
      "finish in 5.936s, 982 probes kept.\n",
      "-- check region:526 9:90004552-90087294_:+_gene_527, 157408 candidate probes\n",
      "--- 57844 probes passed check_dic selection.\n",
      "finish in 5.656s, 886 probes kept.\n",
      "-- check region:527 9:90175735-90277232_:+_gene_528, 202226 candidate probes\n",
      "--- 97687 probes passed check_dic selection.\n",
      "finish in 8.222s, 1384 probes kept.\n",
      "-- check region:528 9:96414547-96482062_:+_gene_529, 131516 candidate probes\n",
      "--- 44893 probes passed check_dic selection.\n",
      "finish in 4.589s, 720 probes kept.\n",
      "-- check region:529 9:97889864-97985879_:+_gene_530, 188662 candidate probes\n",
      "--- 85708 probes passed check_dic selection.\n",
      "finish in 7.420s, 1215 probes kept.\n",
      "-- check region:530 9:99046362-99131260_:+_gene_531, 166736 candidate probes\n",
      "--- 53936 probes passed check_dic selection.\n",
      "finish in 5.649s, 914 probes kept.\n",
      "-- check region:531 9:99753284-99818353_:+_gene_532, 117616 candidate probes\n",
      "--- 49378 probes passed check_dic selection.\n",
      "finish in 4.462s, 700 probes kept.\n",
      "-- check region:532 9:102482333-102523822_:+_gene_533, 82574 candidate probes\n",
      "--- 39599 probes passed check_dic selection.\n",
      "finish in 3.384s, 521 probes kept.\n",
      "-- check region:533 9:104888780-104988138_:+_gene_534, 186636 candidate probes\n",
      "--- 73149 probes passed check_dic selection.\n",
      "finish in 6.860s, 1113 probes kept.\n",
      "-- check region:534 9:105265008-105329057_:+_gene_535, 126080 candidate probes\n",
      "--- 51485 probes passed check_dic selection.\n",
      "finish in 4.718s, 769 probes kept.\n",
      "-- check region:535 9:105834542-105920743_:+_gene_536, 170376 candidate probes\n",
      "--- 68983 probes passed check_dic selection.\n",
      "finish in 6.363s, 1046 probes kept.\n",
      "-- check region:536 9:106914617-106985928_:+_gene_537, 128722 candidate probes\n",
      "--- 35429 probes passed check_dic selection.\n",
      "finish in 4.027s, 633 probes kept.\n",
      "-- check region:537 9:108390917-108463922_:+_gene_538, 142142 candidate probes\n",
      "--- 47492 probes passed check_dic selection.\n",
      "finish in 4.928s, 832 probes kept.\n",
      "-- check region:538 9:109517022-109573625_:+_gene_539, 109634 candidate probes\n",
      "--- 7410 probes passed check_dic selection.\n",
      "finish in 2.538s, 268 probes kept.\n",
      "-- check region:539 9:110403831-110459627_:+_gene_540, 105230 candidate probes\n",
      "--- 37813 probes passed check_dic selection.\n",
      "finish in 3.773s, 623 probes kept.\n",
      "-- check region:540 9:110879347-110944746_:+_gene_541, 127772 candidate probes\n",
      "--- 51030 probes passed check_dic selection.\n",
      "finish in 4.812s, 761 probes kept.\n",
      "-- check region:541 9:112421698-112487152_:+_gene_542, 123370 candidate probes\n",
      "--- 43739 probes passed check_dic selection.\n",
      "finish in 4.370s, 692 probes kept.\n",
      "-- check region:542 9:113760660-113813213_:+_gene_543, 103164 candidate probes\n",
      "--- 28942 probes passed check_dic selection.\n",
      "finish in 3.375s, 504 probes kept.\n",
      "-- check region:543 9:119399249-119449068_:+_gene_544, 98792 candidate probes\n",
      "--- 45249 probes passed check_dic selection.\n",
      "finish in 3.939s, 638 probes kept.\n",
      "-- check region:544 9:123421255-123491622_:+_gene_545, 139034 candidate probes\n",
      "--- 54118 probes passed check_dic selection.\n",
      "finish in 5.086s, 826 probes kept.\n",
      "-- check region:545 10:8166017-8250797_:+_gene_546, 167536 candidate probes\n",
      "--- 67765 probes passed check_dic selection.\n",
      "finish in 6.342s, 1013 probes kept.\n",
      "-- check region:546 10:9791406-9903185_:+_gene_547, 212261 candidate probes\n",
      "--- 72844 probes passed check_dic selection.\n",
      "finish in 7.242s, 1168 probes kept.\n",
      "-- check region:547 10:12830738-12885969_:+_gene_548, 107128 candidate probes\n",
      "--- 40891 probes passed check_dic selection.\n",
      "finish in 3.884s, 629 probes kept.\n",
      "-- check region:548 10:13554012-13611581_:+_gene_549, 105058 candidate probes\n",
      "--- 41446 probes passed check_dic selection.\n",
      "finish in 3.813s, 603 probes kept.\n",
      "-- check region:549 10:19536426-19607009_:+_gene_550, 124502 candidate probes\n",
      "--- 42769 probes passed check_dic selection.\n",
      "finish in 4.309s, 692 probes kept.\n",
      "-- check region:550 10:21881312-21991778_:+_gene_551, 216040 candidate probes\n",
      "--- 80421 probes passed check_dic selection.\n",
      "finish in 7.699s, 1268 probes kept.\n",
      "-- check region:551 10:23181367-23277580_:+_gene_552, 189214 candidate probes\n",
      "--- 66378 probes passed check_dic selection.\n",
      "finish in 6.536s, 1076 probes kept.\n",
      "-- check region:552 10:24951345-25011868_:+_gene_553, 113034 candidate probes\n",
      "--- 37288 probes passed check_dic selection.\n",
      "finish in 4.020s, 606 probes kept.\n",
      "-- check region:553 10:25400587-25476543_:+_gene_554, 131486 candidate probes\n",
      "--- 52439 probes passed check_dic selection.\n",
      "finish in 4.926s, 804 probes kept.\n",
      "-- check region:554 10:26257092-26325268_:+_gene_555, 134044 candidate probes\n",
      "--- 46009 probes passed check_dic selection.\n",
      "finish in 4.652s, 749 probes kept.\n",
      "-- check region:555 10:28073265-28199418_:+_gene_556, 247186 candidate probes\n",
      "--- 88263 probes passed check_dic selection.\n",
      "finish in 8.477s, 1393 probes kept.\n",
      "-- check region:556 10:34121442-34208522_:+_gene_557, 171968 candidate probes\n",
      "--- 53150 probes passed check_dic selection.\n",
      "finish in 5.702s, 914 probes kept.\n",
      "-- check region:557 10:38962790-39040119_:+_gene_558, 150518 candidate probes\n",
      "--- 55240 probes passed check_dic selection.\n",
      "finish in 5.412s, 893 probes kept.\n",
      "-- check region:558 10:40058995-40139830_:+_gene_559, 159066 candidate probes\n",
      "--- 53250 probes passed check_dic selection.\n",
      "finish in 5.400s, 876 probes kept.\n",
      "-- check region:559 10:41176005-41259634_:+_gene_560, 157480 candidate probes\n",
      "--- 68004 probes passed check_dic selection.\n",
      "finish in 6.034s, 996 probes kept.\n",
      "-- check region:560 10:42547711-42602574_:+_gene_561, 107640 candidate probes\n",
      "--- 44614 probes passed check_dic selection.\n",
      "finish in 4.055s, 642 probes kept.\n",
      "-- check region:561 10:43266076-43360048_:+_gene_562, 180588 candidate probes\n",
      "--- 58956 probes passed check_dic selection.\n",
      "finish in 6.028s, 954 probes kept.\n",
      "-- check region:562 10:44282271-44365306_:+_gene_563, 157626 candidate probes\n",
      "--- 52593 probes passed check_dic selection.\n",
      "finish in 5.325s, 863 probes kept.\n",
      "-- check region:563 10:44615388-44713060_:+_gene_564, 189820 candidate probes\n",
      "--- 73077 probes passed check_dic selection.\n",
      "finish in 6.859s, 1120 probes kept.\n",
      "-- check region:564 10:49750898-49817819_:+_gene_565, 128672 candidate probes\n",
      "--- 35125 probes passed check_dic selection.\n",
      "finish in 4.018s, 616 probes kept.\n",
      "-- check region:565 10:52692507-52771120_:+_gene_566, 143466 candidate probes\n",
      "--- 48454 probes passed check_dic selection.\n",
      "finish in 4.854s, 777 probes kept.\n",
      "-- check region:566 10:52891887-52963599_:+_gene_567, 127660 candidate probes\n",
      "--- 37594 probes passed check_dic selection.\n",
      "finish in 4.152s, 661 probes kept.\n",
      "-- check region:567 10:58590997-58683875_:+_gene_568, 183800 candidate probes\n",
      "--- 95537 probes passed check_dic selection.\n",
      "finish in 7.821s, 1261 probes kept.\n",
      "-- check region:568 10:61307332-61385454_:+_gene_569, 153664 candidate probes\n",
      "--- 69650 probes passed check_dic selection.\n",
      "finish in 6.093s, 980 probes kept.\n",
      "-- check region:569 10:61524277-61616208_:+_gene_570, 179358 candidate probes\n",
      "--- 68263 probes passed check_dic selection.\n",
      "finish in 6.593s, 1034 probes kept.\n",
      "-- check region:570 10:62836832-62897538_:+_gene_571, 117682 candidate probes\n",
      "--- 40284 probes passed check_dic selection.\n",
      "finish in 4.068s, 646 probes kept.\n",
      "-- check region:571 10:63133174-63200019_:+_gene_572, 130562 candidate probes\n",
      "--- 45712 probes passed check_dic selection.\n",
      "finish in 4.611s, 711 probes kept.\n",
      "-- check region:572 10:64029707-64098108_:+_gene_573, 134978 candidate probes\n",
      "--- 54897 probes passed check_dic selection.\n",
      "finish in 4.992s, 851 probes kept.\n",
      "-- check region:573 10:66869118-66941818_:+_gene_574, 142982 candidate probes\n",
      "--- 50017 probes passed check_dic selection.\n",
      "finish in 4.991s, 786 probes kept.\n",
      "-- check region:574 10:67841028-67914443_:+_gene_575, 144974 candidate probes\n",
      "--- 59387 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 5.467s, 897 probes kept.\n",
      "-- check region:575 10:69229620-69270356_:+_gene_576, 68240 candidate probes\n",
      "--- 31993 probes passed check_dic selection.\n",
      "finish in 2.792s, 431 probes kept.\n",
      "-- check region:576 10:71281190-71349781_:+_gene_577, 133232 candidate probes\n",
      "--- 48348 probes passed check_dic selection.\n",
      "finish in 4.793s, 747 probes kept.\n",
      "-- check region:577 10:75862641-75913051_:+_gene_578, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.013s, 0 probes kept.\n",
      "-- check region:578 10:75862641-75913051_:+_gene_579, 98018 candidate probes\n",
      "--- 36956 probes passed check_dic selection.\n",
      "finish in 3.585s, 561 probes kept.\n",
      "-- check region:579 10:79702586-79748477_:+_gene_580, 90164 candidate probes\n",
      "--- 44449 probes passed check_dic selection.\n",
      "finish in 3.820s, 639 probes kept.\n",
      "-- check region:580 10:82628910-82692199_:+_gene_581, 123536 candidate probes\n",
      "--- 38317 probes passed check_dic selection.\n",
      "finish in 4.177s, 628 probes kept.\n",
      "-- check region:581 10:83607495-83670925_:+_gene_582, 125078 candidate probes\n",
      "--- 55813 probes passed check_dic selection.\n",
      "finish in 5.037s, 821 probes kept.\n",
      "-- check region:582 10:83837002-83953108_:+_gene_583, 227781 candidate probes\n",
      "--- 103817 probes passed check_dic selection.\n",
      "finish in 8.971s, 1473 probes kept.\n",
      "-- check region:583 10:84077311-84175022_:+_gene_584, 180442 candidate probes\n",
      "--- 75045 probes passed check_dic selection.\n",
      "finish in 6.761s, 1118 probes kept.\n",
      "-- check region:584 10:84921289-85033749_:+_gene_585, 222614 candidate probes\n",
      "--- 91967 probes passed check_dic selection.\n",
      "finish in 8.293s, 1389 probes kept.\n",
      "-- check region:585 10:86022255-86086646_:+_gene_586, 127968 candidate probes\n",
      "--- 46995 probes passed check_dic selection.\n",
      "finish in 4.741s, 768 probes kept.\n",
      "-- check region:586 10:87475669-87514242_:+_gene_587, 73900 candidate probes\n",
      "--- 29878 probes passed check_dic selection.\n",
      "finish in 2.876s, 450 probes kept.\n",
      "-- check region:587 10:87862034-87951886_:+_gene_588, 177424 candidate probes\n",
      "--- 69866 probes passed check_dic selection.\n",
      "finish in 6.485s, 1095 probes kept.\n",
      "-- check region:588 10:91171367-91217531_:+_gene_589, 90442 candidate probes\n",
      "--- 27954 probes passed check_dic selection.\n",
      "finish in 3.011s, 461 probes kept.\n",
      "-- check region:589 10:92215234-92259181_:+_gene_590, 86496 candidate probes\n",
      "--- 34594 probes passed check_dic selection.\n",
      "finish in 3.218s, 528 probes kept.\n",
      "-- check region:590 10:93638004-93714413_:+_gene_591, 149158 candidate probes\n",
      "--- 58793 probes passed check_dic selection.\n",
      "finish in 5.538s, 868 probes kept.\n",
      "-- check region:591 10:94036937-94108212_:+_gene_592, 140662 candidate probes\n",
      "--- 54072 probes passed check_dic selection.\n",
      "finish in 5.194s, 869 probes kept.\n",
      "-- check region:592 10:94220759-94276806_:+_gene_593, 110010 candidate probes\n",
      "--- 42551 probes passed check_dic selection.\n",
      "finish in 3.987s, 635 probes kept.\n",
      "-- check region:593 10:96115549-96224510_:+_gene_594, 196886 candidate probes\n",
      "--- 79626 probes passed check_dic selection.\n",
      "finish in 7.285s, 1174 probes kept.\n",
      "-- check region:594 10:98552540-98672759_:+_gene_595, 226180 candidate probes\n",
      "--- 76682 probes passed check_dic selection.\n",
      "finish in 7.739s, 1243 probes kept.\n",
      "-- check region:595 10:98911845-98969798_:+_gene_596, 112334 candidate probes\n",
      "--- 46497 probes passed check_dic selection.\n",
      "finish in 4.286s, 694 probes kept.\n",
      "-- check region:596 10:99141730-99215126_:+_gene_597, 142824 candidate probes\n",
      "--- 55739 probes passed check_dic selection.\n",
      "finish in 5.272s, 838 probes kept.\n",
      "-- check region:597 10:105474481-105550782_:+_gene_598, 150680 candidate probes\n",
      "--- 60122 probes passed check_dic selection.\n",
      "finish in 5.539s, 917 probes kept.\n",
      "-- check region:598 10:106575266-106742470_:+_gene_599, 328768 candidate probes\n",
      "--- 82904 probes passed check_dic selection.\n",
      "finish in 9.829s, 1533 probes kept.\n",
      "-- check region:599 10:110917563-111009521_:+_gene_600, 178404 candidate probes\n",
      "--- 64633 probes passed check_dic selection.\n",
      "finish in 6.282s, 1028 probes kept.\n",
      "-- check region:600 10:111525577-111633055_:+_gene_601, 209960 candidate probes\n",
      "--- 84046 probes passed check_dic selection.\n",
      "finish in 7.720s, 1238 probes kept.\n",
      "-- check region:601 10:114478817-114628882_:+_gene_602, 276124 candidate probes\n",
      "--- 67496 probes passed check_dic selection.\n",
      "finish in 8.062s, 1263 probes kept.\n",
      "-- check region:602 10:115360947-115403646_:+_gene_603, 84066 candidate probes\n",
      "--- 24065 probes passed check_dic selection.\n",
      "finish in 2.691s, 460 probes kept.\n",
      "-- check region:603 10:116990815-117077224_:+_gene_604, 168614 candidate probes\n",
      "--- 71055 probes passed check_dic selection.\n",
      "finish in 6.503s, 1045 probes kept.\n",
      "-- check region:604 10:117659609-117705189_:+_gene_605, 85928 candidate probes\n",
      "--- 28172 probes passed check_dic selection.\n",
      "finish in 3.058s, 469 probes kept.\n",
      "-- check region:605 10:118733724-118815013_:+_gene_606, 160022 candidate probes\n",
      "--- 68887 probes passed check_dic selection.\n",
      "finish in 6.233s, 992 probes kept.\n",
      "-- check region:606 10:123325185-123388689_:+_gene_607, 123954 candidate probes\n",
      "--- 43912 probes passed check_dic selection.\n",
      "finish in 4.313s, 672 probes kept.\n",
      "-- check region:607 10:125248103-125399970_:+_gene_608, 295798 candidate probes\n",
      "--- 118511 probes passed check_dic selection.\n",
      "finish in 10.824s, 1761 probes kept.\n",
      "-- check region:608 10:126447909-126551128_:+_gene_609, 199902 candidate probes\n",
      "--- 76245 probes passed check_dic selection.\n",
      "finish in 7.247s, 1142 probes kept.\n",
      "-- check region:609 10:128407759-128476427_:+_gene_610, 133392 candidate probes\n",
      "--- 41461 probes passed check_dic selection.\n",
      "finish in 4.542s, 718 probes kept.\n",
      "-- check region:610 11:4944021-5006201_:+_gene_611, 121632 candidate probes\n",
      "--- 49916 probes passed check_dic selection.\n",
      "finish in 4.888s, 740 probes kept.\n",
      "-- check region:611 11:5336596-5447936_:+_gene_612, 212308 candidate probes\n",
      "--- 70519 probes passed check_dic selection.\n",
      "finish in 7.520s, 1136 probes kept.\n",
      "-- check region:612 11:6558052-6610060_:+_gene_613, 101070 candidate probes\n",
      "--- 43798 probes passed check_dic selection.\n",
      "finish in 3.970s, 639 probes kept.\n",
      "-- check region:613 11:11160943-11253518_:+_gene_614, 182604 candidate probes\n",
      "--- 54127 probes passed check_dic selection.\n",
      "finish in 5.853s, 887 probes kept.\n",
      "-- check region:614 11:20425884-20510944_:+_gene_615, 166100 candidate probes\n",
      "--- 59730 probes passed check_dic selection.\n",
      "finish in 5.924s, 927 probes kept.\n",
      "-- check region:615 11:21983178-22041941_:+_gene_616, 116272 candidate probes\n",
      "--- 40936 probes passed check_dic selection.\n",
      "finish in 4.043s, 650 probes kept.\n",
      "-- check region:616 11:22621290-22687776_:+_gene_617, 129130 candidate probes\n",
      "--- 41157 probes passed check_dic selection.\n",
      "finish in 4.369s, 624 probes kept.\n",
      "-- check region:617 11:25838906-25913888_:+_gene_618, 146224 candidate probes\n",
      "--- 46844 probes passed check_dic selection.\n",
      "finish in 4.979s, 770 probes kept.\n",
      "-- check region:618 11:26050799-26139369_:+_gene_619, 175568 candidate probes\n",
      "--- 62897 probes passed check_dic selection.\n",
      "finish in 6.071s, 986 probes kept.\n",
      "-- check region:619 11:26946039-27007404_:+_gene_620, 120878 candidate probes\n",
      "--- 36846 probes passed check_dic selection.\n",
      "finish in 4.020s, 605 probes kept.\n",
      "-- check region:620 11:28645032-28711785_:+_gene_621, 129894 candidate probes\n",
      "--- 39385 probes passed check_dic selection.\n",
      "finish in 4.263s, 667 probes kept.\n",
      "-- check region:621 11:34044208-34109026_:+_gene_622, 128400 candidate probes\n",
      "--- 58323 probes passed check_dic selection.\n",
      "finish in 5.106s, 877 probes kept.\n",
      "-- check region:622 11:35828462-35937732_:+_gene_623, 216236 candidate probes\n",
      "--- 99979 probes passed check_dic selection.\n",
      "finish in 8.533s, 1414 probes kept.\n",
      "-- check region:623 11:36130151-36214786_:+_gene_624, 167341 candidate probes\n",
      "--- 73446 probes passed check_dic selection.\n",
      "finish in 6.465s, 1073 probes kept.\n",
      "-- check region:624 11:38049705-38102695_:+_gene_625, 104132 candidate probes\n",
      "--- 22944 probes passed check_dic selection.\n",
      "finish in 3.115s, 415 probes kept.\n",
      "-- check region:625 11:40754297-40797367_:+_gene_626, 82576 candidate probes\n",
      "--- 23157 probes passed check_dic selection.\n",
      "finish in 2.663s, 415 probes kept.\n",
      "-- check region:626 11:43338561-43423182_:+_gene_627, 165960 candidate probes\n",
      "--- 68922 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 6.272s, 1011 probes kept.\n",
      "-- check region:627 11:43756737-43838613_:+_gene_628, 160888 candidate probes\n",
      "--- 64093 probes passed check_dic selection.\n",
      "finish in 5.955s, 968 probes kept.\n",
      "-- check region:628 11:44493963-44596487_:+_gene_629, 199368 candidate probes\n",
      "--- 78154 probes passed check_dic selection.\n",
      "finish in 7.392s, 1192 probes kept.\n",
      "-- check region:629 11:46420972-46472278_:+_gene_630, 99532 candidate probes\n",
      "--- 30856 probes passed check_dic selection.\n",
      "finish in 3.356s, 529 probes kept.\n",
      "-- check region:630 11:50659370-50717733_:+_gene_631, 115808 candidate probes\n",
      "--- 58380 probes passed check_dic selection.\n",
      "finish in 4.954s, 789 probes kept.\n",
      "-- check region:631 11:52624034-52707755_:+_gene_632, 162090 candidate probes\n",
      "--- 75508 probes passed check_dic selection.\n",
      "finish in 6.598s, 1047 probes kept.\n",
      "-- check region:632 11:54301823-54370331_:+_gene_633, 134926 candidate probes\n",
      "--- 61660 probes passed check_dic selection.\n",
      "finish in 5.383s, 908 probes kept.\n",
      "-- check region:633 11:55130329-55205754_:+_gene_634, 147614 candidate probes\n",
      "--- 53822 probes passed check_dic selection.\n",
      "finish in 5.279s, 879 probes kept.\n",
      "-- check region:634 11:55376303-55463852_:+_gene_635, 170586 candidate probes\n",
      "--- 67869 probes passed check_dic selection.\n",
      "finish in 6.432s, 1030 probes kept.\n",
      "-- check region:635 11:57657256-57770935_:+_gene_636, 223902 candidate probes\n",
      "--- 107372 probes passed check_dic selection.\n",
      "finish in 9.117s, 1464 probes kept.\n",
      "-- check region:636 11:58007673-58081281_:+_gene_637, 144372 candidate probes\n",
      "--- 58397 probes passed check_dic selection.\n",
      "finish in 5.404s, 873 probes kept.\n",
      "-- check region:637 11:58291211-58327455_:+_gene_638, 71028 candidate probes\n",
      "--- 30329 probes passed check_dic selection.\n",
      "finish in 2.801s, 450 probes kept.\n",
      "-- check region:638 11:62284006-62352698_:+_gene_639, 133790 candidate probes\n",
      "--- 41142 probes passed check_dic selection.\n",
      "finish in 4.458s, 733 probes kept.\n",
      "-- check region:639 11:63881715-63935692_:+_gene_640, 107102 candidate probes\n",
      "--- 54823 probes passed check_dic selection.\n",
      "finish in 4.547s, 741 probes kept.\n",
      "-- check region:640 11:67980926-68054725_:+_gene_641, 144836 candidate probes\n",
      "--- 64987 probes passed check_dic selection.\n",
      "finish in 5.656s, 932 probes kept.\n",
      "-- check region:641 11:69581649-69624548_:+_gene_642, 83938 candidate probes\n",
      "--- 30473 probes passed check_dic selection.\n",
      "finish in 3.089s, 505 probes kept.\n",
      "-- check region:642 11:70010422-70060516_:+_gene_643, 99228 candidate probes\n",
      "--- 42107 probes passed check_dic selection.\n",
      "finish in 4.017s, 635 probes kept.\n",
      "-- check region:643 11:70556947-70618824_:+_gene_644, 122284 candidate probes\n",
      "--- 51357 probes passed check_dic selection.\n",
      "finish in 4.722s, 775 probes kept.\n",
      "-- check region:644 11:70860197-70955859_:+_gene_645, 186440 candidate probes\n",
      "--- 46787 probes passed check_dic selection.\n",
      "finish in 5.695s, 910 probes kept.\n",
      "-- check region:645 11:71748129-71802987_:+_gene_646, 109186 candidate probes\n",
      "--- 59100 probes passed check_dic selection.\n",
      "finish in 4.743s, 789 probes kept.\n",
      "-- check region:646 11:72436391-72493071_:+_gene_647, 112202 candidate probes\n",
      "--- 55701 probes passed check_dic selection.\n",
      "finish in 4.745s, 798 probes kept.\n",
      "-- check region:647 11:73271570-73325274_:+_gene_648, 105838 candidate probes\n",
      "--- 39111 probes passed check_dic selection.\n",
      "finish in 3.876s, 622 probes kept.\n",
      "-- check region:648 11:78098903-78180841_:+_gene_649, 160478 candidate probes\n",
      "--- 57064 probes passed check_dic selection.\n",
      "finish in 5.657s, 956 probes kept.\n",
      "-- check region:649 11:78261740-78324241_:+_gene_650, 122148 candidate probes\n",
      "--- 40089 probes passed check_dic selection.\n",
      "finish in 4.326s, 727 probes kept.\n",
      "-- check region:650 11:84022044-84084532_:+_gene_651, 122230 candidate probes\n",
      "--- 47445 probes passed check_dic selection.\n",
      "finish in 4.504s, 724 probes kept.\n",
      "-- check region:651 11:88810839-88868107_:+_gene_652, 110820 candidate probes\n",
      "--- 52653 probes passed check_dic selection.\n",
      "finish in 4.467s, 728 probes kept.\n",
      "-- check region:652 11:89586560-89655737_:+_gene_653, 135276 candidate probes\n",
      "--- 58069 probes passed check_dic selection.\n",
      "finish in 5.195s, 850 probes kept.\n",
      "-- check region:653 11:95665026-95715325_:+_gene_654, 97692 candidate probes\n",
      "--- 41088 probes passed check_dic selection.\n",
      "finish in 3.747s, 597 probes kept.\n",
      "-- check region:654 11:97109256-97171020_:+_gene_655, 122286 candidate probes\n",
      "--- 54559 probes passed check_dic selection.\n",
      "finish in 5.006s, 807 probes kept.\n",
      "-- check region:655 11:100709943-100764691_:+_gene_656, 107598 candidate probes\n",
      "--- 52233 probes passed check_dic selection.\n",
      "finish in 4.474s, 759 probes kept.\n",
      "-- check region:656 11:101999867-102041606_:+_gene_657, 82542 candidate probes\n",
      "--- 31951 probes passed check_dic selection.\n",
      "finish in 3.075s, 521 probes kept.\n",
      "-- check region:657 11:106785701-106842613_:+_gene_658, 111908 candidate probes\n",
      "--- 36771 probes passed check_dic selection.\n",
      "finish in 3.881s, 636 probes kept.\n",
      "-- check region:658 11:107015657-107057289_:+_gene_659, 77294 candidate probes\n",
      "--- 28958 probes passed check_dic selection.\n",
      "finish in 2.944s, 451 probes kept.\n",
      "-- check region:659 11:109457924-109531070_:+_gene_660, 143354 candidate probes\n",
      "--- 61823 probes passed check_dic selection.\n",
      "finish in 5.442s, 895 probes kept.\n",
      "-- check region:660 11:109676821-109738725_:+_gene_661, 122210 candidate probes\n",
      "--- 57237 probes passed check_dic selection.\n",
      "finish in 4.940s, 811 probes kept.\n",
      "-- check region:661 11:111501725-111548093_:+_gene_662, 91440 candidate probes\n",
      "--- 31420 probes passed check_dic selection.\n",
      "finish in 3.205s, 505 probes kept.\n",
      "-- check region:662 11:112788498-112850927_:+_gene_663, 122284 candidate probes\n",
      "--- 56890 probes passed check_dic selection.\n",
      "finish in 4.914s, 810 probes kept.\n",
      "-- check region:663 11:113651313-113714386_:+_gene_664, 122888 candidate probes\n",
      "--- 47168 probes passed check_dic selection.\n",
      "finish in 4.564s, 748 probes kept.\n",
      "-- check region:664 11:115381470-115424234_:+_gene_665, 84236 candidate probes\n",
      "--- 35324 probes passed check_dic selection.\n",
      "finish in 3.277s, 549 probes kept.\n",
      "-- check region:665 11:120969342-121034517_:+_gene_666, 128752 candidate probes\n",
      "--- 55199 probes passed check_dic selection.\n",
      "finish in 4.867s, 777 probes kept.\n",
      "-- check region:666 11:121465660-121560309_:+_gene_667, 186514 candidate probes\n",
      "--- 87304 probes passed check_dic selection.\n",
      "finish in 7.457s, 1224 probes kept.\n",
      "-- check region:667 12:4669994-4767519_:+_gene_668, 183474 candidate probes\n",
      "--- 52069 probes passed check_dic selection.\n",
      "finish in 5.836s, 940 probes kept.\n",
      "-- check region:668 12:8649279-8709465_:+_gene_669, 117025 candidate probes\n",
      "--- 30146 probes passed check_dic selection.\n",
      "finish in 3.708s, 539 probes kept.\n",
      "-- check region:669 12:8906293-8946352_:+_gene_670, 75702 candidate probes\n",
      "--- 28348 probes passed check_dic selection.\n",
      "finish in 2.753s, 481 probes kept.\n",
      "-- check region:670 12:11312932-11414073_:+_gene_671, 198054 candidate probes\n",
      "--- 81217 probes passed check_dic selection.\n",
      "finish in 7.714s, 1210 probes kept.\n",
      "-- check region:671 12:12755678-12838753_:+_gene_672, 161738 candidate probes\n",
      "--- 69108 probes passed check_dic selection.\n",
      "finish in 6.214s, 995 probes kept.\n",
      "-- check region:672 12:13069761-13155339_:+_gene_673, 168332 candidate probes\n",
      "--- 77846 probes passed check_dic selection.\n",
      "finish in 6.640s, 1096 probes kept.\n",
      "-- check region:673 12:21414404-21467592_:+_gene_674, 102002 candidate probes\n",
      "--- 37072 probes passed check_dic selection.\n",
      "finish in 3.682s, 588 probes kept.\n",
      "-- check region:674 12:29527911-29582105_:+_gene_675, 107480 candidate probes\n",
      "--- 48882 probes passed check_dic selection.\n",
      "finish in 4.391s, 729 probes kept.\n",
      "-- check region:675 12:30217718-30280914_:+_gene_676, 124670 candidate probes\n",
      "--- 57196 probes passed check_dic selection.\n",
      "finish in 4.921s, 795 probes kept.\n",
      "-- check region:676 12:34420416-34527401_:+_gene_677, 211273 candidate probes\n",
      "--- 64125 probes passed check_dic selection.\n",
      "finish in 6.806s, 1089 probes kept.\n",
      "-- check region:677 12:34993090-35050003_:+_gene_678, 106972 candidate probes\n",
      "--- 35724 probes passed check_dic selection.\n",
      "finish in 3.609s, 573 probes kept.\n",
      "-- check region:678 12:45004756-45093297_:+_gene_679, 175006 candidate probes\n",
      "--- 67074 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 6.401s, 1047 probes kept.\n",
      "-- check region:679 12:49359077-49421839_:+_gene_680, 124156 candidate probes\n",
      "--- 36857 probes passed check_dic selection.\n",
      "finish in 3.993s, 641 probes kept.\n",
      "-- check region:680 12:51689811-51761995_:+_gene_681, 141178 candidate probes\n",
      "--- 40163 probes passed check_dic selection.\n",
      "finish in 4.504s, 704 probes kept.\n",
      "-- check region:681 12:52731783-52806084_:+_gene_682, 142576 candidate probes\n",
      "--- 50038 probes passed check_dic selection.\n",
      "finish in 5.012s, 795 probes kept.\n",
      "-- check region:682 12:67084035-67208980_:+_gene_683, 245516 candidate probes\n",
      "--- 59600 probes passed check_dic selection.\n",
      "finish in 7.238s, 1121 probes kept.\n",
      "-- check region:683 12:69789098-69891374_:+_gene_684, 202228 candidate probes\n",
      "--- 70352 probes passed check_dic selection.\n",
      "finish in 6.966s, 1140 probes kept.\n",
      "-- check region:684 12:70857906-70924672_:+_gene_685, 120780 candidate probes\n",
      "--- 53248 probes passed check_dic selection.\n",
      "finish in 4.748s, 769 probes kept.\n",
      "-- check region:685 12:72169666-72249828_:+_gene_686, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.013s, 0 probes kept.\n",
      "-- check region:686 12:72169666-72249828_:+_gene_687, 141116 candidate probes\n",
      "--- 52335 probes passed check_dic selection.\n",
      "finish in 4.987s, 791 probes kept.\n",
      "-- check region:687 12:73583311-73644095_:+_gene_688, 120480 candidate probes\n",
      "--- 59478 probes passed check_dic selection.\n",
      "finish in 5.054s, 800 probes kept.\n",
      "-- check region:688 12:75487290-75582550_:+_gene_689, 188340 candidate probes\n",
      "--- 63684 probes passed check_dic selection.\n",
      "finish in 6.427s, 1030 probes kept.\n",
      "-- check region:689 12:83524044-83589479_:+_gene_690, 127936 candidate probes\n",
      "--- 53612 probes passed check_dic selection.\n",
      "finish in 4.915s, 780 probes kept.\n",
      "-- check region:690 12:84151919-84198789_:+_gene_691, 92892 candidate probes\n",
      "--- 45829 probes passed check_dic selection.\n",
      "finish in 3.916s, 622 probes kept.\n",
      "-- check region:691 12:84501230-84556432_:+_gene_692, 107822 candidate probes\n",
      "--- 40304 probes passed check_dic selection.\n",
      "finish in 3.884s, 607 probes kept.\n",
      "-- check region:692 12:85147576-85218677_:+_gene_693, 138812 candidate probes\n",
      "--- 60702 probes passed check_dic selection.\n",
      "finish in 5.549s, 885 probes kept.\n",
      "-- check region:693 12:86672397-86725863_:+_gene_694, 106112 candidate probes\n",
      "--- 44794 probes passed check_dic selection.\n",
      "finish in 4.139s, 684 probes kept.\n",
      "-- check region:694 12:88398166-88453232_:+_gene_695, 109056 candidate probes\n",
      "--- 54952 probes passed check_dic selection.\n",
      "finish in 4.684s, 741 probes kept.\n",
      "-- check region:695 12:89835982-89898938_:+_gene_696, 124692 candidate probes\n",
      "--- 45826 probes passed check_dic selection.\n",
      "finish in 4.371s, 746 probes kept.\n",
      "-- check region:696 12:90064487-90184183_:+_gene_697, 222750 candidate probes\n",
      "--- 73072 probes passed check_dic selection.\n",
      "finish in 7.385s, 1205 probes kept.\n",
      "-- check region:697 12:103952927-104010067_:+_gene_698, 111870 candidate probes\n",
      "--- 45087 probes passed check_dic selection.\n",
      "finish in 4.292s, 678 probes kept.\n",
      "-- check region:698 12:110193166-110281004_:+_gene_699, 173288 candidate probes\n",
      "--- 96327 probes passed check_dic selection.\n",
      "finish in 7.648s, 1266 probes kept.\n",
      "-- check region:699 12:110488383-110596477_:+_gene_700, 213326 candidate probes\n",
      "--- 99540 probes passed check_dic selection.\n",
      "finish in 8.574s, 1373 probes kept.\n",
      "-- check region:700 12:112273397-112371328_:+_gene_701, 192894 candidate probes\n",
      "--- 101124 probes passed check_dic selection.\n",
      "finish in 8.274s, 1315 probes kept.\n",
      "-- check region:701 12:113265834-113332430_:+_gene_702, 124272 candidate probes\n",
      "--- 69374 probes passed check_dic selection.\n",
      "finish in 5.531s, 946 probes kept.\n",
      "-- check region:702 12:118826228-118866588_:+_gene_703, 79966 candidate probes\n",
      "--- 29740 probes passed check_dic selection.\n",
      "finish in 2.855s, 477 probes kept.\n",
      "-- check region:703 13:6145921-6202393_:+_gene_704, 107798 candidate probes\n",
      "--- 36293 probes passed check_dic selection.\n",
      "finish in 3.750s, 598 probes kept.\n",
      "-- check region:704 13:9643963-9718881_:+_gene_705, 146788 candidate probes\n",
      "--- 55497 probes passed check_dic selection.\n",
      "finish in 5.259s, 889 probes kept.\n",
      "-- check region:705 13:13784267-13837202_:+_gene_706, 104222 candidate probes\n",
      "--- 47358 probes passed check_dic selection.\n",
      "finish in 4.174s, 682 probes kept.\n",
      "-- check region:706 13:15511949-15613228_:+_gene_707, 200212 candidate probes\n",
      "--- 87251 probes passed check_dic selection.\n",
      "finish in 7.632s, 1261 probes kept.\n",
      "-- check region:707 13:15969446-16034031_:+_gene_708, 126166 candidate probes\n",
      "--- 44278 probes passed check_dic selection.\n",
      "finish in 4.362s, 718 probes kept.\n",
      "-- check region:708 13:18296121-18382753_:+_gene_709, 171598 candidate probes\n",
      "--- 60992 probes passed check_dic selection.\n",
      "finish in 5.966s, 956 probes kept.\n",
      "-- check region:709 13:18790540-18886160_:+_gene_710, 188842 candidate probes\n",
      "--- 69531 probes passed check_dic selection.\n",
      "finish in 6.724s, 1052 probes kept.\n",
      "-- check region:710 13:23690967-23739842_:+_gene_711, 96970 candidate probes\n",
      "--- 32521 probes passed check_dic selection.\n",
      "finish in 3.397s, 542 probes kept.\n",
      "-- check region:711 13:28582966-28640622_:+_gene_712, 108948 candidate probes\n",
      "--- 42246 probes passed check_dic selection.\n",
      "finish in 3.934s, 644 probes kept.\n",
      "-- check region:712 13:32778496-32844925_:+_gene_713, 130750 candidate probes\n",
      "--- 46450 probes passed check_dic selection.\n",
      "finish in 4.592s, 755 probes kept.\n",
      "-- check region:713 13:36247139-36325532_:+_gene_714, 151168 candidate probes\n",
      "--- 60376 probes passed check_dic selection.\n",
      "finish in 5.537s, 921 probes kept.\n",
      "-- check region:714 13:37810544-37982052_:+_gene_715, 340340 candidate probes\n",
      "--- 147630 probes passed check_dic selection.\n",
      "finish in 13.106s, 2223 probes kept.\n",
      "-- check region:715 13:40911149-40989375_:+_gene_716, 147184 candidate probes\n",
      "--- 54167 probes passed check_dic selection.\n",
      "finish in 5.305s, 833 probes kept.\n",
      "-- check region:716 13:41292633-41362801_:+_gene_717, 139474 candidate probes\n",
      "--- 64010 probes passed check_dic selection.\n",
      "finish in 5.496s, 916 probes kept.\n",
      "-- check region:717 13:43729520-43816764_:+_gene_718, 172894 candidate probes\n",
      "--- 75465 probes passed check_dic selection.\n",
      "finish in 6.593s, 1091 probes kept.\n",
      "-- check region:718 13:44008093-44111293_:+_gene_719, 204974 candidate probes\n",
      "--- 99338 probes passed check_dic selection.\n",
      "finish in 8.360s, 1376 probes kept.\n",
      "-- check region:719 13:44198135-44248787_:+_gene_720, 100650 candidate probes\n",
      "--- 47518 probes passed check_dic selection.\n",
      "finish in 4.067s, 646 probes kept.\n",
      "-- check region:720 13:44459166-44548478_:+_gene_721, 171438 candidate probes\n",
      "--- 74694 probes passed check_dic selection.\n",
      "finish in 6.632s, 1066 probes kept.\n",
      "-- check region:721 13:45064772-45169265_:+_gene_722, 202336 candidate probes\n",
      "--- 75172 probes passed check_dic selection.\n",
      "finish in 7.259s, 1148 probes kept.\n",
      "-- check region:722 13:46801717-46906339_:+_gene_723, 206490 candidate probes\n",
      "--- 95082 probes passed check_dic selection.\n",
      "finish in 8.151s, 1329 probes kept.\n",
      "-- check region:723 13:52004806-52065454_:+_gene_724, 114832 candidate probes\n",
      "--- 52888 probes passed check_dic selection.\n",
      "finish in 4.739s, 716 probes kept.\n",
      "-- check region:724 13:57236124-57302330_:+_gene_725, 130704 candidate probes\n",
      "--- 51668 probes passed check_dic selection.\n",
      "finish in 4.848s, 787 probes kept.\n",
      "-- check region:725 13:59788227-59853221_:+_gene_726, 128492 candidate probes\n",
      "--- 40080 probes passed check_dic selection.\n",
      "finish in 4.321s, 710 probes kept.\n",
      "-- check region:726 13:60034633-60143990_:+_gene_727, 213998 candidate probes\n",
      "--- 95645 probes passed check_dic selection.\n",
      "finish in 8.433s, 1355 probes kept.\n",
      "-- check region:727 13:60434888-60508636_:+_gene_728, 139876 candidate probes\n",
      "--- 59394 probes passed check_dic selection.\n",
      "finish in 5.379s, 864 probes kept.\n",
      "-- check region:728 13:62887747-62937194_:+_gene_729, 97006 candidate probes\n",
      "--- 37926 probes passed check_dic selection.\n",
      "finish in 3.615s, 587 probes kept.\n",
      "-- check region:729 13:64427959-64496783_:+_gene_730, 133730 candidate probes\n",
      "--- 45408 probes passed check_dic selection.\n",
      "finish in 4.639s, 772 probes kept.\n",
      "-- check region:730 13:69312990-69368200_:+_gene_731, 108860 candidate probes\n",
      "--- 42956 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 4.079s, 634 probes kept.\n",
      "-- check region:731 13:70852179-70914310_:+_gene_732, 122108 candidate probes\n",
      "--- 50458 probes passed check_dic selection.\n",
      "finish in 4.631s, 741 probes kept.\n",
      "-- check region:732 13:75713653-75820309_:+_gene_733, 208256 candidate probes\n",
      "--- 69492 probes passed check_dic selection.\n",
      "finish in 7.046s, 1142 probes kept.\n",
      "-- check region:733 13:81123389-81170001_:+_gene_734, 92594 candidate probes\n",
      "--- 33241 probes passed check_dic selection.\n",
      "finish in 3.392s, 543 probes kept.\n",
      "-- check region:734 13:83089701-83146173_:+_gene_735, 107014 candidate probes\n",
      "--- 28521 probes passed check_dic selection.\n",
      "finish in 3.285s, 528 probes kept.\n",
      "-- check region:735 13:84011657-84075757_:+_gene_736, 119878 candidate probes\n",
      "--- 35722 probes passed check_dic selection.\n",
      "finish in 3.826s, 617 probes kept.\n",
      "-- check region:736 13:88889893-89025343_:+_gene_737, 255307 candidate probes\n",
      "--- 48451 probes passed check_dic selection.\n",
      "finish in 6.927s, 997 probes kept.\n",
      "-- check region:737 13:89573484-89618121_:+_gene_738, 87725 candidate probes\n",
      "--- 22447 probes passed check_dic selection.\n",
      "finish in 2.713s, 433 probes kept.\n",
      "-- check region:738 13:97151046-97230203_:+_gene_739, 152806 candidate probes\n",
      "--- 57097 probes passed check_dic selection.\n",
      "finish in 5.443s, 875 probes kept.\n",
      "-- check region:739 13:99342904-99407764_:+_gene_740, 126252 candidate probes\n",
      "--- 55878 probes passed check_dic selection.\n",
      "finish in 4.896s, 803 probes kept.\n",
      "-- check region:740 13:101198612-101249406_:+_gene_741, 100976 candidate probes\n",
      "--- 45887 probes passed check_dic selection.\n",
      "finish in 3.984s, 671 probes kept.\n",
      "-- check region:741 13:101440340-101523506_:+_gene_742, 163038 candidate probes\n",
      "--- 63311 probes passed check_dic selection.\n",
      "finish in 6.052s, 954 probes kept.\n",
      "-- check region:742 13:103252792-103337542_:+_gene_743, 165736 candidate probes\n",
      "--- 76296 probes passed check_dic selection.\n",
      "finish in 6.491s, 1083 probes kept.\n",
      "-- check region:743 13:106678768-106757437_:+_gene_744, 142118 candidate probes\n",
      "--- 46453 probes passed check_dic selection.\n",
      "finish in 4.838s, 732 probes kept.\n",
      "-- check region:744 13:108046902-108146316_:+_gene_745, 195456 candidate probes\n",
      "--- 76289 probes passed check_dic selection.\n",
      "finish in 7.105s, 1153 probes kept.\n",
      "-- check region:745 13:108254869-108312090_:+_gene_746, 112911 candidate probes\n",
      "--- 36808 probes passed check_dic selection.\n",
      "finish in 3.864s, 639 probes kept.\n",
      "-- check region:746 13:111373959-111432442_:+_gene_747, 110892 candidate probes\n",
      "--- 35909 probes passed check_dic selection.\n",
      "finish in 3.759s, 597 probes kept.\n",
      "-- check region:747 13:111731847-111807966_:+_gene_748, 149758 candidate probes\n",
      "--- 64183 probes passed check_dic selection.\n",
      "finish in 5.720s, 914 probes kept.\n",
      "-- check region:748 13:111919531-111990923_:+_gene_749, 140992 candidate probes\n",
      "--- 64875 probes passed check_dic selection.\n",
      "finish in 5.589s, 915 probes kept.\n",
      "-- check region:749 13:114065922-114149764_:+_gene_750, 538 candidate probes\n",
      "--- 124 probes passed check_dic selection.\n",
      "finish in 0.037s, 5 probes kept.\n",
      "-- check region:750 13:114065922-114149495_:+_gene_751, 165016 candidate probes\n",
      "--- 75728 probes passed check_dic selection.\n",
      "finish in 6.509s, 1082 probes kept.\n",
      "-- check region:751 13:119586445-119624951_:+_gene_752, 49610 candidate probes\n",
      "--- 10792 probes passed check_dic selection.\n",
      "finish in 1.916s, 176 probes kept.\n",
      "-- check region:752 14:3298996-3368904_:+_gene_753, 29622 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.652s, 0 probes kept.\n",
      "-- check region:753 14:3528167-3583135_:+_gene_754, 55234 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 1.166s, 0 probes kept.\n",
      "-- check region:754 14:5449405-5512217_:+_gene_755, 35116 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.818s, 0 probes kept.\n",
      "-- check region:755 14:5950658-6033064_:+_gene_756, 121030 candidate probes\n",
      "--- 114 probes passed check_dic selection.\n",
      "finish in 2.398s, 8 probes kept.\n",
      "-- check region:756 14:7284561-7335660_:+_gene_757, 99018 candidate probes\n",
      "--- 98 probes passed check_dic selection.\n",
      "finish in 2.077s, 8 probes kept.\n",
      "-- check region:757 14:13323083-13378913_:+_gene_758, 107162 candidate probes\n",
      "--- 32469 probes passed check_dic selection.\n",
      "finish in 3.559s, 511 probes kept.\n",
      "-- check region:758 14:14043804-14108999_:+_gene_759, 128152 candidate probes\n",
      "--- 51282 probes passed check_dic selection.\n",
      "finish in 4.751s, 769 probes kept.\n",
      "-- check region:759 14:15345670-15436164_:+_gene_760, 178306 candidate probes\n",
      "--- 75578 probes passed check_dic selection.\n",
      "finish in 6.687s, 1094 probes kept.\n",
      "-- check region:760 14:21333204-21412833_:+_gene_761, 148672 candidate probes\n",
      "--- 47351 probes passed check_dic selection.\n",
      "finish in 5.005s, 789 probes kept.\n",
      "-- check region:761 14:21959106-22004415_:+_gene_762, 88198 candidate probes\n",
      "--- 40131 probes passed check_dic selection.\n",
      "finish in 3.579s, 558 probes kept.\n",
      "-- check region:762 14:26419598-26536458_:+_gene_763, 230778 candidate probes\n",
      "--- 64465 probes passed check_dic selection.\n",
      "finish in 7.210s, 1161 probes kept.\n",
      "-- check region:763 14:29018427-29124966_:+_gene_764, 209896 candidate probes\n",
      "--- 95182 probes passed check_dic selection.\n",
      "finish in 8.385s, 1417 probes kept.\n",
      "-- check region:764 14:29313773-29399124_:+_gene_765, 167146 candidate probes\n",
      "--- 72117 probes passed check_dic selection.\n",
      "finish in 6.402s, 1052 probes kept.\n",
      "-- check region:765 14:31127141-31206382_:+_gene_766, 156948 candidate probes\n",
      "--- 74051 probes passed check_dic selection.\n",
      "finish in 6.372s, 1119 probes kept.\n",
      "-- check region:766 14:31992648-32046083_:+_gene_767, 104716 candidate probes\n",
      "--- 47411 probes passed check_dic selection.\n",
      "finish in 4.262s, 682 probes kept.\n",
      "-- check region:767 14:32517175-32620238_:+_gene_768, 205600 candidate probes\n",
      "--- 95588 probes passed check_dic selection.\n",
      "finish in 8.125s, 1375 probes kept.\n",
      "-- check region:768 14:34178487-34245598_:+_gene_769, 132742 candidate probes\n",
      "--- 55237 probes passed check_dic selection.\n",
      "finish in 5.133s, 874 probes kept.\n",
      "-- check region:769 14:40891564-40969316_:+_gene_770, 151836 candidate probes\n",
      "--- 76519 probes passed check_dic selection.\n",
      "finish in 6.316s, 1032 probes kept.\n",
      "-- check region:770 14:43230240-43361198_:+_gene_771, 239288 candidate probes\n",
      "--- 492 probes passed check_dic selection.\n",
      "finish in 4.739s, 36 probes kept.\n",
      "-- check region:771 14:45529453-45635426_:+_gene_772, 209036 candidate probes\n",
      "--- 72492 probes passed check_dic selection.\n",
      "finish in 7.323s, 1193 probes kept.\n",
      "-- check region:772 14:46891206-47000060_:+_gene_773, 213146 candidate probes\n",
      "--- 100791 probes passed check_dic selection.\n",
      "finish in 8.633s, 1405 probes kept.\n",
      "-- check region:773 14:48145216-48234948_:+_gene_774, 175780 candidate probes\n",
      "--- 81189 probes passed check_dic selection.\n",
      "finish in 6.965s, 1141 probes kept.\n",
      "-- check region:774 14:55487236-55538412_:+_gene_775, 100248 candidate probes\n",
      "--- 40238 probes passed check_dic selection.\n",
      "finish in 3.865s, 679 probes kept.\n",
      "-- check region:775 14:57498975-57575644_:+_gene_776, 11286 candidate probes\n",
      "--- 3459 probes passed check_dic selection.\n",
      "finish in 0.419s, 62 probes kept.\n",
      "-- check region:776 14:57498975-57569853_:+_gene_777, 136478 candidate probes\n",
      "--- 55231 probes passed check_dic selection.\n",
      "finish in 5.139s, 806 probes kept.\n",
      "-- check region:777 14:57669680-57747633_:+_gene_778, 148022 candidate probes\n",
      "--- 57235 probes passed check_dic selection.\n",
      "finish in 5.348s, 869 probes kept.\n",
      "-- check region:778 14:57891541-57932428_:+_gene_779, 78836 candidate probes\n",
      "--- 21950 probes passed check_dic selection.\n",
      "finish in 2.652s, 393 probes kept.\n",
      "-- check region:779 14:58064999-58123602_:+_gene_780, 116376 candidate probes\n",
      "--- 48694 probes passed check_dic selection.\n",
      "finish in 4.409s, 727 probes kept.\n",
      "-- check region:780 14:59196105-59261023_:+_gene_781, 126414 candidate probes\n",
      "--- 31359 probes passed check_dic selection.\n",
      "finish in 3.846s, 556 probes kept.\n",
      "-- check region:781 14:59691594-59748067_:+_gene_782, 111648 candidate probes\n",
      "--- 50289 probes passed check_dic selection.\n",
      "finish in 4.420s, 739 probes kept.\n",
      "-- check region:782 14:61628986-61683707_:+_gene_783, 108486 candidate probes\n",
      "--- 29766 probes passed check_dic selection.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "finish in 3.355s, 527 probes kept.\n",
      "-- check region:783 14:61793570-61846789_:+_gene_784, 103368 candidate probes\n",
      "--- 38091 probes passed check_dic selection.\n",
      "finish in 3.763s, 581 probes kept.\n",
      "-- check region:784 14:62163770-62246386_:+_gene_785, 161832 candidate probes\n",
      "--- 66173 probes passed check_dic selection.\n",
      "finish in 6.020s, 970 probes kept.\n",
      "-- check region:785 14:62834246-62950869_:+_gene_786, 230488 candidate probes\n",
      "--- 86381 probes passed check_dic selection.\n",
      "finish in 8.192s, 1369 probes kept.\n",
      "-- check region:786 14:63118357-63175661_:+_gene_787, 113290 candidate probes\n",
      "--- 47036 probes passed check_dic selection.\n",
      "finish in 4.420s, 712 probes kept.\n",
      "-- check region:787 14:65504499-65594282_:+_gene_788, 177746 candidate probes\n",
      "--- 81892 probes passed check_dic selection.\n",
      "finish in 7.028s, 1117 probes kept.\n",
      "-- check region:788 14:65960189-65998537_:+_gene_789, 75410 candidate probes\n",
      "--- 34060 probes passed check_dic selection.\n",
      "finish in 3.041s, 503 probes kept.\n",
      "-- check region:789 14:67878001-67932629_:+_gene_790, 108054 candidate probes\n",
      "--- 46872 probes passed check_dic selection.\n",
      "finish in 4.164s, 672 probes kept.\n",
      "-- check region:790 14:69791321-69871987_:+_gene_791, 158132 candidate probes\n",
      "--- 70322 probes passed check_dic selection.\n",
      "finish in 6.188s, 1007 probes kept.\n",
      "-- check region:791 14:70292991-70355449_:+_gene_792, 123434 candidate probes\n",
      "--- 46758 probes passed check_dic selection.\n",
      "finish in 4.621s, 763 probes kept.\n",
      "-- check region:792 14:75463070-75553422_:+_gene_793, 177882 candidate probes\n",
      "--- 79117 probes passed check_dic selection.\n",
      "finish in 6.874s, 1148 probes kept.\n",
      "-- check region:793 14:75860410-75925995_:+_gene_794, 127978 candidate probes\n",
      "--- 45067 probes passed check_dic selection.\n",
      "finish in 4.576s, 710 probes kept.\n",
      "-- check region:794 14:76415881-76490537_:+_gene_795, 147714 candidate probes\n",
      "--- 64474 probes passed check_dic selection.\n",
      "finish in 5.724s, 950 probes kept.\n",
      "-- check region:795 14:79478685-79573260_:+_gene_796, 185462 candidate probes\n",
      "--- 59089 probes passed check_dic selection.\n",
      "finish in 6.205s, 1048 probes kept.\n",
      "-- check region:796 14:79899686-79984133_:+_gene_797, 150674 candidate probes\n",
      "--- 54928 probes passed check_dic selection.\n",
      "finish in 5.404s, 869 probes kept.\n",
      "-- check region:797 14:80227149-80288631_:+_gene_798, 120220 candidate probes\n",
      "--- 49664 probes passed check_dic selection.\n",
      "finish in 4.534s, 743 probes kept.\n",
      "-- check region:798 14:99526543-99602180_:+_gene_799, 148356 candidate probes\n",
      "--- 53230 probes passed check_dic selection.\n",
      "finish in 5.193s, 823 probes kept.\n",
      "-- check region:799 14:102774665-102854185_:+_gene_800, 141308 candidate probes\n",
      "--- 46447 probes passed check_dic selection.\n",
      "finish in 4.769s, 746 probes kept.\n",
      "-- check region:800 14:103099997-103173301_:+_gene_801, 142025 candidate probes\n",
      "--- 39654 probes passed check_dic selection.\n",
      "finish in 4.504s, 735 probes kept.\n",
      "-- check region:801 14:103414208-103482720_:+_gene_802, 132720 candidate probes\n",
      "--- 46463 probes passed check_dic selection.\n",
      "finish in 4.639s, 733 probes kept.\n",
      "-- check region:802 14:105264551-105344049_:+_gene_803, 154392 candidate probes\n",
      "--- 51864 probes passed check_dic selection.\n",
      "finish in 5.332s, 886 probes kept.\n",
      "-- check region:803 14:108781216-108854003_:+_gene_804, 143200 candidate probes\n",
      "--- 27198 probes passed check_dic selection.\n",
      "finish in 3.901s, 564 probes kept.\n",
      "-- check region:804 14:114844433-114923616_:+_gene_805, 143914 candidate probes\n",
      "--- 50189 probes passed check_dic selection.\n",
      "finish in 5.000s, 815 probes kept.\n",
      "-- check region:805 14:119193071-119288918_:+_gene_806, 185422 candidate probes\n",
      "--- 64047 probes passed check_dic selection.\n",
      "finish in 6.338s, 971 probes kept.\n",
      "-- check region:806 14:119621024-119697288_:+_gene_807, 138314 candidate probes\n",
      "--- 46079 probes passed check_dic selection.\n",
      "finish in 4.778s, 734 probes kept.\n",
      "-- check region:807 14:120779255-120908010_:+_gene_808, 234166 candidate probes\n",
      "--- 74111 probes passed check_dic selection.\n",
      "finish in 7.812s, 1163 probes kept.\n",
      "-- check region:808 14:121918265-122090024_:+_gene_809, 339288 candidate probes\n",
      "--- 136789 probes passed check_dic selection.\n",
      "finish in 12.608s, 2018 probes kept.\n",
      "-- check region:809 15:6509514-6581190_:+_gene_810, 127988 candidate probes\n",
      "--- 46438 probes passed check_dic selection.\n",
      "finish in 4.530s, 724 probes kept.\n",
      "-- check region:810 15:7170479-7236680_:+_gene_811, 130090 candidate probes\n",
      "--- 57747 probes passed check_dic selection.\n",
      "finish in 5.152s, 887 probes kept.\n",
      "-- check region:811 15:8375489-8461753_:+_gene_812, 168824 candidate probes\n",
      "--- 38967 probes passed check_dic selection.\n",
      "finish in 4.931s, 767 probes kept.\n",
      "-- check region:812 15:12417846-12507837_:+_gene_813, 178606 candidate probes\n",
      "--- 84646 probes passed check_dic selection.\n",
      "finish in 7.155s, 1191 probes kept.\n",
      "-- check region:813 15:13071050-13175931_:+_gene_814, 198410 candidate probes\n",
      "--- 82098 probes passed check_dic selection.\n",
      "finish in 7.301s, 1207 probes kept.\n",
      "-- check region:814 15:16742412-16833900_:+_gene_815, 178694 candidate probes\n",
      "--- 32764 probes passed check_dic selection.\n",
      "finish in 4.911s, 674 probes kept.\n",
      "-- check region:815 15:18847236-18992658_:+_gene_816, 284288 candidate probes\n",
      "--- 64555 probes passed check_dic selection.\n",
      "finish in 8.049s, 1242 probes kept.\n",
      "-- check region:816 15:24958652-25046793_:+_gene_817, 171996 candidate probes\n",
      "--- 41151 probes passed check_dic selection.\n",
      "finish in 5.118s, 755 probes kept.\n",
      "-- check region:817 15:28171581-28259520_:+_gene_818, 160824 candidate probes\n",
      "--- 61146 probes passed check_dic selection.\n",
      "finish in 5.822s, 959 probes kept.\n",
      "-- check region:818 15:30249955-30317077_:+_gene_819, 123546 candidate probes\n",
      "--- 42464 probes passed check_dic selection.\n",
      "finish in 4.309s, 670 probes kept.\n",
      "-- check region:819 15:36478751-36528020_:+_gene_820, 96526 candidate probes\n",
      "--- 37029 probes passed check_dic selection.\n",
      "finish in 3.551s, 599 probes kept.\n",
      "-- check region:820 15:40814516-40989464_:+_gene_821, 327161 candidate probes\n",
      "--- 100045 probes passed check_dic selection.\n",
      "finish in 10.400s, 1713 probes kept.\n",
      "-- check region:821 15:44711034-44779894_:+_gene_822, 126988 candidate probes\n",
      "--- 37072 probes passed check_dic selection.\n",
      "finish in 4.147s, 644 probes kept.\n",
      "-- check region:822 15:45075447-45142386_:+_gene_823, 132012 candidate probes\n",
      "--- 22676 probes passed check_dic selection.\n",
      "finish in 3.562s, 492 probes kept.\n",
      "-- check region:823 15:50654938-50758251_:+_gene_824, 204720 candidate probes\n",
      "--- 63424 probes passed check_dic selection.\n",
      "finish in 6.629s, 1105 probes kept.\n",
      "-- check region:824 15:52730443-52820439_:+_gene_825, 152788 candidate probes\n",
      "--- 47649 probes passed check_dic selection.\n",
      "finish in 5.124s, 744 probes kept.\n",
      "-- check region:825 15:58158195-58230642_:+_gene_826, 142738 candidate probes\n",
      "--- 56727 probes passed check_dic selection.\n",
      "finish in 5.325s, 875 probes kept.\n",
      "-- check region:826 15:67091131-67172296_:+_gene_827, 158486 candidate probes\n",
      "--- 79367 probes passed check_dic selection.\n",
      "finish in 6.580s, 1099 probes kept.\n",
      "-- check region:827 15:68684491-68749615_:+_gene_828, 124686 candidate probes\n",
      "--- 48506 probes passed check_dic selection.\n",
      "finish in 4.558s, 724 probes kept.\n",
      "-- check region:828 15:68923567-69002893_:+_gene_829, 142846 candidate probes\n",
      "--- 67178 probes passed check_dic selection.\n",
      "finish in 5.683s, 951 probes kept.\n",
      "-- check region:829 15:71879586-71959467_:+_gene_830, 157948 candidate probes\n",
      "--- 73877 probes passed check_dic selection.\n",
      "finish in 6.381s, 1044 probes kept.\n",
      "-- check region:830 15:76273099-76323683_:+_gene_831, 99084 candidate probes\n",
      "--- 39054 probes passed check_dic selection.\n",
      "finish in 3.724s, 617 probes kept.\n",
      "-- check region:831 15:76434091-76478590_:+_gene_832, 88356 candidate probes\n",
      "--- 43596 probes passed check_dic selection.\n",
      "finish in 3.770s, 652 probes kept.\n",
      "-- check region:832 15:80684138-80732788_:+_gene_833, 95732 candidate probes\n",
      "--- 36484 probes passed check_dic selection.\n",
      "finish in 3.591s, 567 probes kept.\n",
      "-- check region:833 15:81504063-81579541_:+_gene_834, 145788 candidate probes\n",
      "--- 38227 probes passed check_dic selection.\n",
      "finish in 4.575s, 652 probes kept.\n",
      "-- check region:834 15:81672506-81729092_:+_gene_835, 110126 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 42538 probes passed check_dic selection.\n",
      "finish in 4.053s, 673 probes kept.\n",
      "-- check region:835 15:83364966-83464933_:+_gene_836, 196066 candidate probes\n",
      "--- 86624 probes passed check_dic selection.\n",
      "finish in 7.584s, 1230 probes kept.\n",
      "-- check region:836 15:88479865-88547648_:+_gene_837, 133094 candidate probes\n",
      "--- 57831 probes passed check_dic selection.\n",
      "finish in 5.179s, 814 probes kept.\n",
      "-- check region:837 15:88818050-88864719_:+_gene_838, 91738 candidate probes\n",
      "--- 51225 probes passed check_dic selection.\n",
      "finish in 4.095s, 648 probes kept.\n",
      "-- check region:838 15:92180253-92287287_:+_gene_839, 209906 candidate probes\n",
      "--- 70945 probes passed check_dic selection.\n",
      "finish in 7.040s, 1154 probes kept.\n",
      "-- check region:839 15:93482810-93529982_:+_gene_840, 91854 candidate probes\n",
      "--- 39949 probes passed check_dic selection.\n",
      "finish in 3.624s, 572 probes kept.\n",
      "-- check region:840 15:95034078-95096462_:+_gene_841, 118548 candidate probes\n",
      "--- 46727 probes passed check_dic selection.\n",
      "finish in 4.340s, 716 probes kept.\n",
      "-- check region:841 15:96274908-96319877_:+_gene_842, 88570 candidate probes\n",
      "--- 31479 probes passed check_dic selection.\n",
      "finish in 3.297s, 495 probes kept.\n",
      "-- check region:842 15:101815997-101884700_:+_gene_843, 135970 candidate probes\n",
      "--- 64262 probes passed check_dic selection.\n",
      "finish in 5.570s, 924 probes kept.\n",
      "-- check region:843 15:103501670-103562538_:+_gene_844, 120550 candidate probes\n",
      "--- 50702 probes passed check_dic selection.\n",
      "finish in 4.679s, 780 probes kept.\n",
      "-- check region:844 16:5140368-5205722_:+_gene_845, 128514 candidate probes\n",
      "--- 47170 probes passed check_dic selection.\n",
      "finish in 4.628s, 760 probes kept.\n",
      "-- check region:845 16:10331430-10395351_:+_gene_846, 126098 candidate probes\n",
      "--- 54385 probes passed check_dic selection.\n",
      "finish in 4.903s, 784 probes kept.\n",
      "-- check region:846 16:21147080-21202863_:+_gene_847, 110194 candidate probes\n",
      "--- 25985 probes passed check_dic selection.\n",
      "finish in 3.346s, 456 probes kept.\n",
      "-- check region:847 16:22756585-22854730_:+_gene_848, 192870 candidate probes\n",
      "--- 81725 probes passed check_dic selection.\n",
      "finish in 7.364s, 1187 probes kept.\n",
      "-- check region:848 16:23642588-23704832_:+_gene_849, 122534 candidate probes\n",
      "--- 47082 probes passed check_dic selection.\n",
      "finish in 4.408s, 715 probes kept.\n",
      "-- check region:849 16:23942714-24001471_:+_gene_850, 116106 candidate probes\n",
      "--- 48465 probes passed check_dic selection.\n",
      "finish in 4.422s, 730 probes kept.\n",
      "-- check region:850 16:25973196-26088540_:+_gene_851, 226416 candidate probes\n",
      "--- 75484 probes passed check_dic selection.\n",
      "finish in 7.567s, 1231 probes kept.\n",
      "-- check region:851 16:28583146-28642529_:+_gene_852, 117174 candidate probes\n",
      "--- 44537 probes passed check_dic selection.\n",
      "finish in 4.191s, 655 probes kept.\n",
      "-- check region:852 16:28813123-28919744_:+_gene_853, 208134 candidate probes\n",
      "--- 101637 probes passed check_dic selection.\n",
      "finish in 8.495s, 1370 probes kept.\n",
      "-- check region:853 16:29118866-29189467_:+_gene_854, 138580 candidate probes\n",
      "--- 46314 probes passed check_dic selection.\n",
      "finish in 4.741s, 724 probes kept.\n",
      "-- check region:854 16:32431344-32553252_:+_gene_855, 237958 candidate probes\n",
      "--- 79120 probes passed check_dic selection.\n",
      "finish in 8.152s, 1295 probes kept.\n",
      "-- check region:855 16:32643912-32726340_:+_gene_856, 161623 candidate probes\n",
      "--- 68754 probes passed check_dic selection.\n",
      "finish in 6.286s, 1008 probes kept.\n",
      "-- check region:856 16:34957154-35017777_:+_gene_857, 120240 candidate probes\n",
      "--- 54370 probes passed check_dic selection.\n",
      "finish in 4.794s, 771 probes kept.\n",
      "-- check region:857 16:35723966-35794035_:+_gene_858, 137756 candidate probes\n",
      "--- 48755 probes passed check_dic selection.\n",
      "finish in 4.890s, 782 probes kept.\n",
      "-- check region:858 16:36610671-36680286_:+_gene_859, 137440 candidate probes\n",
      "--- 52929 probes passed check_dic selection.\n",
      "finish in 5.002s, 821 probes kept.\n",
      "-- check region:859 16:38150332-38259251_:+_gene_860, 214936 candidate probes\n",
      "--- 69735 probes passed check_dic selection.\n",
      "finish in 7.180s, 1199 probes kept.\n",
      "-- check region:860 16:41514064-41562563_:+_gene_861, 96158 candidate probes\n",
      "--- 33992 probes passed check_dic selection.\n",
      "finish in 3.443s, 542 probes kept.\n",
      "-- check region:861 16:42245627-42341328_:+_gene_862, 185395 candidate probes\n",
      "--- 71433 probes passed check_dic selection.\n",
      "finish in 6.692s, 1123 probes kept.\n",
      "-- check region:862 16:42875154-42969540_:+_gene_863, 184858 candidate probes\n",
      "--- 45270 probes passed check_dic selection.\n",
      "finish in 5.464s, 812 probes kept.\n",
      "-- check region:863 16:43912104-43983953_:+_gene_864, 138832 candidate probes\n",
      "--- 52079 probes passed check_dic selection.\n",
      "finish in 5.033s, 859 probes kept.\n",
      "-- check region:864 16:44200258-44301077_:+_gene_865, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.012s, 0 probes kept.\n",
      "-- check region:865 16:44200258-44301077_:+_gene_866, 198300 candidate probes\n",
      "--- 72412 probes passed check_dic selection.\n",
      "finish in 6.992s, 1189 probes kept.\n",
      "-- check region:866 16:50679262-50747219_:+_gene_867, 131978 candidate probes\n",
      "--- 48305 probes passed check_dic selection.\n",
      "finish in 4.733s, 754 probes kept.\n",
      "-- check region:867 16:52068029-52157877_:+_gene_868, 175038 candidate probes\n",
      "--- 67949 probes passed check_dic selection.\n",
      "finish in 6.244s, 1040 probes kept.\n",
      "-- check region:868 16:58434511-58529914_:+_gene_869, 188860 candidate probes\n",
      "--- 67564 probes passed check_dic selection.\n",
      "finish in 6.573s, 1095 probes kept.\n",
      "-- check region:869 16:67559729-67628201_:+_gene_870, 128082 candidate probes\n",
      "--- 33001 probes passed check_dic selection.\n",
      "finish in 3.921s, 574 probes kept.\n",
      "-- check region:870 16:74079872-74201979_:+_gene_871, 235434 candidate probes\n",
      "--- 72475 probes passed check_dic selection.\n",
      "finish in 7.638s, 1237 probes kept.\n",
      "-- check region:871 16:77374249-77459818_:+_gene_872, 168928 candidate probes\n",
      "--- 47047 probes passed check_dic selection.\n",
      "finish in 5.231s, 832 probes kept.\n",
      "-- check region:872 16:77620781-77677713_:+_gene_873, 112566 candidate probes\n",
      "--- 32808 probes passed check_dic selection.\n",
      "finish in 3.568s, 570 probes kept.\n",
      "-- check region:873 16:78553141-78609787_:+_gene_874, 102246 candidate probes\n",
      "--- 34660 probes passed check_dic selection.\n",
      "finish in 3.584s, 562 probes kept.\n",
      "-- check region:874 16:88117436-88198446_:+_gene_875, 160832 candidate probes\n",
      "--- 58667 probes passed check_dic selection.\n",
      "finish in 5.587s, 921 probes kept.\n",
      "-- check region:875 16:89507846-89608464_:+_gene_876, 195386 candidate probes\n",
      "--- 75723 probes passed check_dic selection.\n",
      "finish in 7.218s, 1125 probes kept.\n",
      "-- check region:876 16:90915327-91010712_:+_gene_877, 187792 candidate probes\n",
      "--- 72295 probes passed check_dic selection.\n",
      "finish in 6.791s, 1126 probes kept.\n",
      "-- check region:877 16:93967016-94031875_:+_gene_878, 124710 candidate probes\n",
      "--- 55871 probes passed check_dic selection.\n",
      "finish in 4.873s, 775 probes kept.\n",
      "-- check region:878 16:94279006-94324592_:+_gene_879, 84578 candidate probes\n",
      "--- 29256 probes passed check_dic selection.\n",
      "finish in 2.965s, 454 probes kept.\n",
      "-- check region:879 16:94419141-94483370_:+_gene_880, 125258 candidate probes\n",
      "--- 43345 probes passed check_dic selection.\n",
      "finish in 4.352s, 730 probes kept.\n",
      "-- check region:880 16:95361862-95460042_:+_gene_881, 194460 candidate probes\n",
      "--- 92253 probes passed check_dic selection.\n",
      "finish in 7.757s, 1256 probes kept.\n",
      "-- check region:881 16:97687705-97762534_:+_gene_882, 147356 candidate probes\n",
      "--- 72448 probes passed check_dic selection.\n",
      "finish in 6.035s, 1000 probes kept.\n",
      "-- check region:882 17:6758025-6822752_:+_gene_883, 128272 candidate probes\n",
      "--- 58393 probes passed check_dic selection.\n",
      "finish in 5.127s, 817 probes kept.\n",
      "-- check region:883 17:11654178-11797180_:+_gene_884, 275389 candidate probes\n",
      "--- 121493 probes passed check_dic selection.\n",
      "finish in 10.642s, 1710 probes kept.\n",
      "-- check region:884 17:13808723-13929026_:+_gene_885, 225386 candidate probes\n",
      "--- 72749 probes passed check_dic selection.\n",
      "finish in 7.561s, 1200 probes kept.\n",
      "-- check region:885 17:15361839-15397147_:+_gene_886, 69886 candidate probes\n",
      "--- 29920 probes passed check_dic selection.\n",
      "finish in 2.747s, 423 probes kept.\n",
      "-- check region:886 17:23516967-23561137_:+_gene_887, 82768 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 25787 probes passed check_dic selection.\n",
      "finish in 2.904s, 400 probes kept.\n",
      "-- check region:887 17:24038288-24101203_:+_gene_888, 122354 candidate probes\n",
      "--- 53165 probes passed check_dic selection.\n",
      "finish in 4.796s, 786 probes kept.\n",
      "-- check region:888 17:31799132-31853772_:+_gene_889, 108230 candidate probes\n",
      "--- 57354 probes passed check_dic selection.\n",
      "finish in 4.680s, 784 probes kept.\n",
      "-- check region:889 17:34090951-34145951_:+_gene_890, 108042 candidate probes\n",
      "--- 45909 probes passed check_dic selection.\n",
      "finish in 4.286s, 729 probes kept.\n",
      "-- check region:890 17:35516761-35560061_:+_gene_891, 84700 candidate probes\n",
      "--- 33055 probes passed check_dic selection.\n",
      "finish in 3.153s, 502 probes kept.\n",
      "-- check region:891 17:43011066-43103508_:+_gene_892, 181472 candidate probes\n",
      "--- 75593 probes passed check_dic selection.\n",
      "finish in 6.782s, 1126 probes kept.\n",
      "-- check region:892 17:44188990-44272216_:+_gene_893, 164168 candidate probes\n",
      "--- 75739 probes passed check_dic selection.\n",
      "finish in 6.500s, 1083 probes kept.\n",
      "-- check region:893 17:46828396-46877205_:+_gene_894, 94920 candidate probes\n",
      "--- 34353 probes passed check_dic selection.\n",
      "finish in 3.472s, 522 probes kept.\n",
      "-- check region:894 17:48353042-48410987_:+_gene_895, 113432 candidate probes\n",
      "--- 38202 probes passed check_dic selection.\n",
      "finish in 3.975s, 662 probes kept.\n",
      "-- check region:895 17:48532940-48612895_:+_gene_896, 158478 candidate probes\n",
      "--- 65785 probes passed check_dic selection.\n",
      "finish in 6.071s, 977 probes kept.\n",
      "-- check region:896 17:48761285-48835829_:+_gene_897, 147676 candidate probes\n",
      "--- 66998 probes passed check_dic selection.\n",
      "finish in 5.900s, 928 probes kept.\n",
      "-- check region:897 17:49458027-49562568_:+_gene_898, 207220 candidate probes\n",
      "--- 95039 probes passed check_dic selection.\n",
      "finish in 8.241s, 1375 probes kept.\n",
      "-- check region:898 17:50115429-50204110_:+_gene_899, 174624 candidate probes\n",
      "--- 83885 probes passed check_dic selection.\n",
      "finish in 7.110s, 1184 probes kept.\n",
      "-- check region:899 17:51727923-51816643_:+_gene_900, 176536 candidate probes\n",
      "--- 72216 probes passed check_dic selection.\n",
      "finish in 6.608s, 1145 probes kept.\n",
      "-- check region:900 17:53567112-53638475_:+_gene_901, 127226 candidate probes\n",
      "--- 41456 probes passed check_dic selection.\n",
      "finish in 4.330s, 666 probes kept.\n",
      "-- check region:901 17:55266212-55319734_:+_gene_902, 95544 candidate probes\n",
      "--- 21904 probes passed check_dic selection.\n",
      "finish in 2.799s, 381 probes kept.\n",
      "-- check region:902 17:56872991-56939799_:+_gene_903, 132182 candidate probes\n",
      "--- 59829 probes passed check_dic selection.\n",
      "finish in 5.225s, 855 probes kept.\n",
      "-- check region:903 17:57254102-57357459_:+_gene_904, 203192 candidate probes\n",
      "--- 75617 probes passed check_dic selection.\n",
      "finish in 7.459s, 1167 probes kept.\n",
      "-- check region:904 17:62630369-62740980_:+_gene_905, 219050 candidate probes\n",
      "--- 98354 probes passed check_dic selection.\n",
      "finish in 8.555s, 1396 probes kept.\n",
      "-- check region:905 17:67430764-67483458_:+_gene_906, 104476 candidate probes\n",
      "--- 50902 probes passed check_dic selection.\n",
      "finish in 4.275s, 696 probes kept.\n",
      "-- check region:906 17:73209957-73282539_:+_gene_907, 140756 candidate probes\n",
      "--- 60388 probes passed check_dic selection.\n",
      "finish in 5.358s, 888 probes kept.\n",
      "-- check region:907 17:73871483-73952612_:+_gene_908, 157348 candidate probes\n",
      "--- 71897 probes passed check_dic selection.\n",
      "finish in 6.291s, 1039 probes kept.\n",
      "-- check region:908 17:75432566-75547174_:+_gene_909, 225966 candidate probes\n",
      "--- 85252 probes passed check_dic selection.\n",
      "finish in 8.066s, 1331 probes kept.\n",
      "-- check region:909 17:80137911-80209379_:+_gene_910, 139668 candidate probes\n",
      "--- 47501 probes passed check_dic selection.\n",
      "finish in 4.942s, 748 probes kept.\n",
      "-- check region:910 17:83530997-83606275_:+_gene_911, 147590 candidate probes\n",
      "--- 56773 probes passed check_dic selection.\n",
      "finish in 5.412s, 841 probes kept.\n",
      "-- check region:911 17:84464454-84530370_:+_gene_912, 128478 candidate probes\n",
      "--- 40638 probes passed check_dic selection.\n",
      "finish in 4.359s, 642 probes kept.\n",
      "-- check region:912 17:84850274-84953300_:+_gene_913, 193946 candidate probes\n",
      "--- 69327 probes passed check_dic selection.\n",
      "finish in 6.899s, 1054 probes kept.\n",
      "-- check region:913 17:89322309-89377903_:+_gene_914, 110378 candidate probes\n",
      "--- 33984 probes passed check_dic selection.\n",
      "finish in 3.764s, 577 probes kept.\n",
      "-- check region:914 17:90044561-90126527_:+_gene_915, 156222 candidate probes\n",
      "--- 46596 probes passed check_dic selection.\n",
      "finish in 5.039s, 786 probes kept.\n",
      "-- check region:915 17:90276015-90421156_:+_gene_916, 287021 candidate probes\n",
      "--- 73573 probes passed check_dic selection.\n",
      "finish in 8.506s, 1351 probes kept.\n",
      "-- check region:916 18:6929796-7011984_:+_gene_917, 162792 candidate probes\n",
      "--- 60630 probes passed check_dic selection.\n",
      "finish in 5.807s, 949 probes kept.\n",
      "-- check region:917 18:15391109-15452687_:+_gene_918, 121380 candidate probes\n",
      "--- 41555 probes passed check_dic selection.\n",
      "finish in 4.152s, 705 probes kept.\n",
      "-- check region:918 18:21151931-21203166_:+_gene_919, 100548 candidate probes\n",
      "--- 41742 probes passed check_dic selection.\n",
      "finish in 3.841s, 627 probes kept.\n",
      "-- check region:919 18:24089649-24145055_:+_gene_920, 109366 candidate probes\n",
      "--- 41587 probes passed check_dic selection.\n",
      "finish in 4.045s, 648 probes kept.\n",
      "-- check region:920 18:24408559-24500784_:+_gene_921, 180828 candidate probes\n",
      "--- 66267 probes passed check_dic selection.\n",
      "finish in 6.408s, 999 probes kept.\n",
      "-- check region:921 18:24712942-24888570_:+_gene_922, 346324 candidate probes\n",
      "--- 153150 probes passed check_dic selection.\n",
      "finish in 13.377s, 2217 probes kept.\n",
      "-- check region:922 18:31929025-31995900_:+_gene_923, 133202 candidate probes\n",
      "--- 60058 probes passed check_dic selection.\n",
      "finish in 5.333s, 886 probes kept.\n",
      "-- check region:923 18:32936362-33016449_:+_gene_924, 0 candidate probes\n",
      "--- 0 probes passed check_dic selection.\n",
      "finish in 0.013s, 0 probes kept.\n",
      "-- check region:924 18:32936362-33016449_:+_gene_925, 146198 candidate probes\n",
      "--- 48386 probes passed check_dic selection.\n",
      "finish in 4.960s, 786 probes kept.\n",
      "-- check region:925 18:34069958-34139601_:+_gene_926, 133400 candidate probes\n",
      "--- 46514 probes passed check_dic selection.\n",
      "finish in 4.857s, 703 probes kept.\n",
      "-- check region:926 18:35472165-35550427_:+_gene_927, 153422 candidate probes\n",
      "--- 48045 probes passed check_dic selection.\n",
      "finish in 5.109s, 765 probes kept.\n",
      "-- check region:927 18:36681158-36730587_:+_gene_928, 96388 candidate probes\n",
      "--- 26876 probes passed check_dic selection.\n",
      "finish in 3.206s, 481 probes kept.\n",
      "-- check region:928 18:37139186-37201875_:+_gene_929, 124588 candidate probes\n",
      "--- 41144 probes passed check_dic selection.\n",
      "finish in 4.219s, 724 probes kept.\n",
      "-- check region:929 18:39886744-39952821_:+_gene_930, 125732 candidate probes\n",
      "--- 44794 probes passed check_dic selection.\n",
      "finish in 4.415s, 699 probes kept.\n",
      "-- check region:930 18:41736971-41828086_:+_gene_931, 172826 candidate probes\n",
      "--- 49317 probes passed check_dic selection.\n",
      "finish in 5.488s, 801 probes kept.\n",
      "-- check region:931 18:42055872-42140040_:+_gene_932, 165654 candidate probes\n",
      "--- 67942 probes passed check_dic selection.\n",
      "finish in 6.195s, 973 probes kept.\n",
      "-- check region:932 18:45568608-45662084_:+_gene_933, 186014 candidate probes\n",
      "--- 70346 probes passed check_dic selection.\n",
      "finish in 6.662s, 1106 probes kept.\n",
      "-- check region:933 18:46169765-46214913_:+_gene_934, 88332 candidate probes\n",
      "--- 33604 probes passed check_dic selection.\n",
      "finish in 3.181s, 525 probes kept.\n",
      "-- check region:934 18:49582614-49668484_:+_gene_935, 167234 candidate probes\n",
      "--- 57990 probes passed check_dic selection.\n",
      "finish in 5.681s, 903 probes kept.\n",
      "-- check region:935 18:50001602-50079289_:+_gene_936, 153934 candidate probes\n",
      "--- 68785 probes passed check_dic selection.\n",
      "finish in 6.086s, 1000 probes kept.\n",
      "-- check region:936 18:53219139-53335927_:+_gene_937, 227208 candidate probes\n",
      "--- 86645 probes passed check_dic selection.\n",
      "finish in 8.151s, 1295 probes kept.\n",
      "-- check region:937 18:55993891-56082519_:+_gene_938, 173202 candidate probes\n",
      "--- 57726 probes passed check_dic selection.\n",
      "finish in 5.787s, 935 probes kept.\n",
      "-- check region:938 18:57239664-57307027_:+_gene_939, 131504 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 52697 probes passed check_dic selection.\n",
      "finish in 4.850s, 796 probes kept.\n",
      "-- check region:939 18:59180021-59350310_:+_gene_940, 331026 candidate probes\n",
      "--- 97531 probes passed check_dic selection.\n",
      "finish in 10.439s, 1657 probes kept.\n",
      "-- check region:940 18:63719174-63787284_:+_gene_941, 135330 candidate probes\n",
      "--- 39824 probes passed check_dic selection.\n",
      "finish in 4.365s, 688 probes kept.\n",
      "-- check region:941 18:64575603-64648451_:+_gene_942, 143296 candidate probes\n",
      "--- 56496 probes passed check_dic selection.\n",
      "finish in 5.337s, 810 probes kept.\n",
      "-- check region:942 18:65843454-65945591_:+_gene_943, 199556 candidate probes\n",
      "--- 80100 probes passed check_dic selection.\n",
      "finish in 7.414s, 1206 probes kept.\n",
      "-- check region:943 18:68933328-69000228_:+_gene_944, 128380 candidate probes\n",
      "--- 51010 probes passed check_dic selection.\n",
      "finish in 4.817s, 782 probes kept.\n",
      "-- check region:944 18:74461676-74530383_:+_gene_945, 136324 candidate probes\n",
      "--- 69174 probes passed check_dic selection.\n",
      "finish in 5.646s, 922 probes kept.\n",
      "-- check region:945 18:74669677-74751625_:+_gene_946, 159078 candidate probes\n",
      "--- 80576 probes passed check_dic selection.\n",
      "finish in 6.702s, 1055 probes kept.\n",
      "-- check region:946 18:77958807-78042826_:+_gene_947, 161284 candidate probes\n",
      "--- 64689 probes passed check_dic selection.\n",
      "finish in 6.000s, 996 probes kept.\n",
      "-- check region:947 18:80950121-81002541_:+_gene_948, 103958 candidate probes\n",
      "--- 48418 probes passed check_dic selection.\n",
      "finish in 4.251s, 704 probes kept.\n",
      "-- check region:948 18:85278941-85379839_:+_gene_949, 186437 candidate probes\n",
      "--- 60194 probes passed check_dic selection.\n",
      "finish in 6.237s, 952 probes kept.\n",
      "-- check region:949 19:3625409-3709978_:+_gene_950, 166186 candidate probes\n",
      "--- 72599 probes passed check_dic selection.\n",
      "finish in 6.422s, 994 probes kept.\n",
      "-- check region:950 19:3822930-3906324_:+_gene_951, 161652 candidate probes\n",
      "--- 59290 probes passed check_dic selection.\n",
      "finish in 5.770s, 945 probes kept.\n",
      "-- check region:951 19:5638442-5685296_:+_gene_952, 91892 candidate probes\n",
      "--- 43937 probes passed check_dic selection.\n",
      "finish in 3.845s, 662 probes kept.\n",
      "-- check region:952 19:6276522-6315238_:+_gene_953, 76590 candidate probes\n",
      "--- 40200 probes passed check_dic selection.\n",
      "finish in 3.494s, 550 probes kept.\n",
      "-- check region:953 19:7054548-7207140_:+_gene_954, 302382 candidate probes\n",
      "--- 159830 probes passed check_dic selection.\n",
      "finish in 12.919s, 2153 probes kept.\n",
      "-- check region:954 19:8544284-8597137_:+_gene_955, 105176 candidate probes\n",
      "--- 30683 probes passed check_dic selection.\n",
      "finish in 3.438s, 501 probes kept.\n",
      "-- check region:955 19:10685617-10740864_:+_gene_956, 109142 candidate probes\n",
      "--- 53734 probes passed check_dic selection.\n",
      "finish in 4.598s, 766 probes kept.\n",
      "-- check region:956 19:10930147-10970163_:+_gene_957, 77164 candidate probes\n",
      "--- 33979 probes passed check_dic selection.\n",
      "finish in 3.199s, 489 probes kept.\n",
      "-- check region:957 19:12424826-12504327_:+_gene_958, 155798 candidate probes\n",
      "--- 56276 probes passed check_dic selection.\n",
      "finish in 5.550s, 905 probes kept.\n",
      "-- check region:958 19:15780510-15825819_:+_gene_959, 88668 candidate probes\n",
      "--- 32963 probes passed check_dic selection.\n",
      "finish in 3.270s, 519 probes kept.\n",
      "-- check region:959 19:17296782-17355629_:+_gene_960, 112328 candidate probes\n",
      "--- 36758 probes passed check_dic selection.\n",
      "finish in 3.863s, 590 probes kept.\n",
      "-- check region:960 19:20518919-20570378_:+_gene_961, 93020 candidate probes\n",
      "--- 21335 probes passed check_dic selection.\n",
      "finish in 2.763s, 404 probes kept.\n",
      "-- check region:961 19:21713365-21757827_:+_gene_962, 87522 candidate probes\n",
      "--- 27386 probes passed check_dic selection.\n",
      "finish in 2.971s, 447 probes kept.\n",
      "-- check region:962 19:23553461-23620199_:+_gene_963, 131520 candidate probes\n",
      "--- 43540 probes passed check_dic selection.\n",
      "finish in 4.453s, 698 probes kept.\n",
      "-- check region:963 19:24600840-24687705_:+_gene_964, 170872 candidate probes\n",
      "--- 74472 probes passed check_dic selection.\n",
      "finish in 6.568s, 1086 probes kept.\n",
      "-- check region:964 19:24944141-25013094_:+_gene_965, 137442 candidate probes\n",
      "--- 38361 probes passed check_dic selection.\n",
      "finish in 4.446s, 686 probes kept.\n",
      "-- check region:965 19:27524144-27591990_:+_gene_966, 134150 candidate probes\n",
      "--- 52866 probes passed check_dic selection.\n",
      "finish in 4.893s, 786 probes kept.\n",
      "-- check region:966 19:28355825-28409866_:+_gene_967, 107908 candidate probes\n",
      "--- 51524 probes passed check_dic selection.\n",
      "finish in 4.433s, 744 probes kept.\n",
      "-- check region:967 19:28563435-28646484_:+_gene_968, 164778 candidate probes\n",
      "--- 73317 probes passed check_dic selection.\n",
      "finish in 6.384s, 1046 probes kept.\n",
      "-- check region:968 19:29856927-29928435_:+_gene_969, 142054 candidate probes\n",
      "--- 48238 probes passed check_dic selection.\n",
      "finish in 4.891s, 758 probes kept.\n",
      "-- check region:969 19:32895563-32971276_:+_gene_970, 149922 candidate probes\n",
      "--- 63212 probes passed check_dic selection.\n",
      "finish in 5.641s, 922 probes kept.\n",
      "-- check region:970 19:33213942-33295336_:+_gene_971, 161202 candidate probes\n",
      "--- 51576 probes passed check_dic selection.\n",
      "finish in 5.428s, 812 probes kept.\n",
      "-- check region:971 19:36172931-36275082_:+_gene_972, 202098 candidate probes\n",
      "--- 77573 probes passed check_dic selection.\n",
      "finish in 7.290s, 1197 probes kept.\n",
      "-- check region:972 19:36718674-36759528_:+_gene_973, 80686 candidate probes\n",
      "--- 24163 probes passed check_dic selection.\n",
      "finish in 2.698s, 408 probes kept.\n",
      "-- check region:973 19:37472495-37520209_:+_gene_974, 93710 candidate probes\n",
      "--- 32364 probes passed check_dic selection.\n",
      "finish in 3.345s, 510 probes kept.\n",
      "-- check region:974 19:37965494-38041498_:+_gene_975, 150412 candidate probes\n",
      "--- 63017 probes passed check_dic selection.\n",
      "finish in 5.667s, 931 probes kept.\n",
      "-- check region:975 19:41050775-41097097_:+_gene_976, 91600 candidate probes\n",
      "--- 43482 probes passed check_dic selection.\n",
      "finish in 3.788s, 609 probes kept.\n",
      "-- check region:976 19:41281381-41355899_:+_gene_977, 147874 candidate probes\n",
      "--- 72286 probes passed check_dic selection.\n",
      "finish in 5.995s, 969 probes kept.\n",
      "-- check region:977 19:41963511-42036554_:+_gene_978, 144910 candidate probes\n",
      "--- 66220 probes passed check_dic selection.\n",
      "finish in 5.762s, 907 probes kept.\n",
      "-- check region:978 19:45861417-45967203_:+_gene_979, 208864 candidate probes\n",
      "--- 69804 probes passed check_dic selection.\n",
      "finish in 7.081s, 1129 probes kept.\n",
      "-- check region:979 19:47634250-47669784_:+_gene_980, 70112 candidate probes\n",
      "--- 32485 probes passed check_dic selection.\n",
      "finish in 2.875s, 485 probes kept.\n",
      "-- check region:980 19:48639897-48747838_:+_gene_981, 214012 candidate probes\n",
      "--- 87733 probes passed check_dic selection.\n",
      "finish in 7.901s, 1329 probes kept.\n",
      "-- check region:981 19:58794802-58866457_:+_gene_982, 142650 candidate probes\n",
      "--- 71778 probes passed check_dic selection.\n",
      "finish in 5.993s, 987 probes kept.\n",
      "-- check region:982 19:59015717-59063288_:+_gene_983, 94604 candidate probes\n",
      "--- 34643 probes passed check_dic selection.\n",
      "finish in 3.420s, 521 probes kept.\n",
      "-- check region:983 19:59369976-59445429_:+_gene_984, 149608 candidate probes\n",
      "--- 66530 probes passed check_dic selection.\n",
      "finish in 5.784s, 970 probes kept.\n",
      "-- check region:984 X:93271165-93309819_:+_gene_985, 76912 candidate probes\n",
      "--- 33745 probes passed check_dic selection.\n",
      "finish in 2.987s, 517 probes kept.\n",
      "-- check region:985 X:98979231-99088826_:+_gene_986, 217776 candidate probes\n",
      "--- 100514 probes passed check_dic selection.\n",
      "finish in 8.553s, 1406 probes kept.\n",
      "-- check region:986 X:101910335-102015609_:+_gene_987, 209108 candidate probes\n",
      "--- 87774 probes passed check_dic selection.\n",
      "finish in 7.853s, 1296 probes kept.\n",
      "-- check region:987 X:136772307-136851006_:+_gene_988, 156918 candidate probes\n",
      "--- 66111 probes passed check_dic selection.\n",
      "finish in 5.996s, 983 probes kept.\n",
      "-- check region:988 X:140481839-140547985_:+_gene_989, 131729 candidate probes\n",
      "--- 51350 probes passed check_dic selection.\n",
      "finish in 4.780s, 785 probes kept.\n",
      "-- check region:989 X:167122110-167191320_:+_gene_990, 138103 candidate probes\n",
      "--- 46077 probes passed check_dic selection.\n",
      "finish in 4.749s, 799 probes kept.\n",
      "-- check region:990 X:169962881-170032477_:+_gene_991, 96216 candidate probes\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--- 5815 probes passed check_dic selection.\n",
      "finish in 2.202s, 127 probes kept.\n"
     ]
    }
   ],
   "source": [
    "pbs, pb_scores = pb_designer.check_probes()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### save probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 147,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Save reports into file: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\reports\\merged_probes.pbr\n",
      "-- number of probes kept: 821825\n"
     ]
    }
   ],
   "source": [
    "overwrite_savefile = True \n",
    "if not os.path.exists(pb_designer.save_file) or overwrite_savefile:\n",
    "    #pb_designer.plots()\n",
    "    #pb_designer.save_csv()\n",
    "    pb_designer.save_to_file()\n",
    "print(f\"-- number of probes kept: {len(pb_designer.kept_probes)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 151,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Time to release OTmaps: 4.199s. \n"
     ]
    }
   ],
   "source": [
    "# after finish designing, remove all loaded OT maps to freeup memory usage\n",
    "pb_designer.release_OTmaps()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 152,
   "metadata": {},
   "outputs": [],
   "source": [
    "pb_designer.cand_probes.items()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 150,
   "metadata": {},
   "outputs": [],
   "source": [
    "for _pb, _info in pb_designer.cand_probes.items():\n",
    "    if _info['reg_name'][-1] == '_':\n",
    "        old_name = str(_info['reg_name'])\n",
    "        new_name = _info['reg_name'].replace(':+', 'strand_+') + str(_info['reg_index'])\n",
    "        _info['name'] = _info['name'].replace(old_name, new_name)\n",
    "        _info['reg_name'] = new_name"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2.3 summarize into a pb_dict\n",
    "\n",
    "start over from here is possible"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 154,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\reports\\merged_probes.pbr\n"
     ]
    }
   ],
   "source": [
    "# filename to save probe reports\n",
    "save_file = os.path.join(report_folder, f'merged_probes.pbr')\n",
    "print(save_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "_pbde_from_pickel = pickle.load(open(save_file,'rb'))\n",
    "\n",
    "#_pbde_from_pickel = pickle.load(save_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Loading from savefile: \\\\10.245.74.212\\Chromatin_NAS_2\\Chromatin_Libraries\\CTP-13_brain-super-enhancers\\mop_SE_1000\\reports\\merged_probes.pbr.\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools)\n",
    "reload(library_tools.design)\n",
    "\n",
    "verbose=True\n",
    "cand_prob_nums = []\n",
    "pb_dict = {}\n",
    "# Parse segemented list\n",
    "#_pbde = library_tools.design.pb_reports_class()\n",
    "#_pbde.load_from_file(save_file)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "_region_gid = list(_sel_pbs.values())[0]['reg_name'].split('_')[-1]\n",
    "cand_prob_nums.append(len(_sel_pbs))\n",
    "if verbose:\n",
    "    print(f\"-- appending gene:{_gene_name} with {len(_sel_pbs)} probes\")\n",
    "_splitted_pbde = ld.pb_reports_class()\n",
    "_splitted_pbde.kept_probes = _sel_pbs\n",
    "pb_dict[_gene_name] = _splitted_pbde"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
