{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Library design for MOP mm10"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-09-04T21:51:40.331599Z",
     "start_time": "2019-09-04T21:51:40.314586Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "38332\n"
     ]
    }
   ],
   "source": [
    "%run \"E:\\Users\\puzheng\\Documents\\Startup_py3.py\"\n",
    "sys.path.append(r\"E:\\Users\\puzheng\\Documents\")\n",
    "\n",
    "import ImageAnalysis3\n",
    "from ImageAnalysis3 import get_img_info, visual_tools, corrections, library_tools\n",
    "\n",
    "from ImageAnalysis3.library_tools import LibraryDesigner as ld\n",
    "from ImageAnalysis3.library_tools import LibraryTools as lt\n",
    "\n",
    "%matplotlib notebook\n",
    "print(os.getpid())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 1 Extract region sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-09-04T21:51:41.526486Z",
     "start_time": "2019-09-04T21:51:41.520482Z"
    }
   },
   "outputs": [],
   "source": [
    "## Some folders\n",
    "# human genome\n",
    "genome_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\mm10'\n",
    "# Library directories\n",
    "pool_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "resolution = 10000\n",
    "flanking = 200000\n",
    "# folder for sub-pool\n",
    "library_folder = os.path.join(pool_folder, f'Slc32a1_{int(resolution/1000)}kb')\n",
    "if not os.path.exists(library_folder):\n",
    "    print(f\"create library folder: {library_folder}\")\n",
    "    os.makedirs(library_folder)\n",
    "# folder for fasta sequences\n",
    "sequence_folder = os.path.join(library_folder, 'sequences')\n",
    "if not os.path.exists(sequence_folder):\n",
    "    print(f\"create sequence folder: {sequence_folder}\")\n",
    "    os.makedirs(sequence_folder)\n",
    "# folder to save result probes\n",
    "report_folder = os.path.join(library_folder, 'reports')\n",
    "if not os.path.exists(report_folder):\n",
    "    print(f\"create report folder: {report_folder}\")\n",
    "    os.makedirs(report_folder)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Input region file is: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc32a1_10kb\\Regions.txt\n",
      "['mm10.fa', 'mrna.fa', 'RepBase_Repeat.fasta']\n",
      "mm10.fa\n",
      "-writing region number:41\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.sequences)\n",
    "seq_dic_list, reg_list = library_tools.sequences.Batch_Extract_Sequences(library_folder, genome_folder, \n",
    "                                                                         merge=False, save=True,\n",
    "                                                                         resolution=resolution, flanking=flanking)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 2. Design probe targeting sequences by probe_designer\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2.1 Construct count table with all the 17-mers in the genome\n",
    "\n",
    "Only do this if you don't have pre-built 17-mer\n",
    "\n",
    "However you can do almost the same thing for your own library during quality check\n",
    "\n",
    "This library requires mm9 genome"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### construct map for whole genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "from tqdm import tqdm\n",
    "# genome\n",
    "reload(ld)\n",
    "names,seqs = lt.fastaread(os.path.join(genome_folder, 'mm10.fa'))\n",
    "ct = ld.countTable(word=17,save_file=os.path.join(genome_folder, 'mm10_genome_17w.npy'), sparse=False)\n",
    "print(ct.save_file)\n",
    "for iseq in tqdm(range(len(seqs))):\n",
    "    print(names[iseq], end=' ')\n",
    "    ct.consume(seqs[iseq],verbose=True)\n",
    "ct.complete(verbose=True)\n",
    "ct.save()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### construct map for transcriptome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm import tqdm\n",
    "# transcriptome\n",
    "reload(ld)\n",
    "names,seqs = lt.fastaread(os.path.join(genome_folder, 'mrna.fa'))\n",
    "ct = ld.countTable(word=17,save_file=os.path.join(genome_folder, 'mm10_transcriptome_17w.npy'), sparse=False)\n",
    "print(ct.save_file)\n",
    "for iseq in tqdm(range(len(seqs))):\n",
    "    print(names[iseq], end=' ')\n",
    "    ct.consume(seqs[iseq],verbose=True)\n",
    "ct.complete(verbose=True)\n",
    "ct.save()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm import tqdm\n",
    "reload(lt)\n",
    "\n",
    "# repeats\n",
    "names,seqs = lt.fastaread(os.path.join(genome_folder, 'mm10.fa'))\n",
    "rep_names, rep_seqs = [], []\n",
    "for _name, _seq in zip(names, seqs):\n",
    "    _sub_names, _sub_seqs = lt.extract_lower_case(_name, _seq)\n",
    "    rep_names += _sub_names\n",
    "    rep_seqs += _sub_seqs\n",
    "print(len(rep_names), len(rep_seqs))\n",
    "\n",
    "ct = ld.countTable(word=17,save_file=os.path.join(genome_folder, 'mm10_genome_repeats_17w.npy'), sparse=False)\n",
    "for iseq in tqdm(range(len(rep_seqs))):\n",
    "    print(rep_names[iseq], end=' ')\n",
    "    ct.consume(rep_seqs[iseq],verbose=True)\n",
    "ct.complete(verbose=True)\n",
    "ct.save()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### construct map for meng's MOP MERFISH library"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ref_lib_filename = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Existing_MERFISH\\M1_meng.fasta'\n",
    "names,seqs = lt.fastaread(ref_lib_filename)\n",
    "\n",
    "ct = ld.countTable(word=17,save_file=os.path.join(pool_folder, 'M1_meng_MERFISH_17w.npy'), sparse=False)\n",
    "for iseq in tqdm(range(len(seqs))):\n",
    "    print(names[iseq], end=' ')\n",
    "    ct.consume(seqs[iseq],verbose=True)\n",
    "ct.complete(verbose=True)\n",
    "ct.save()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2.2 Run probe designer"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-09-04T21:51:44.146433Z",
     "start_time": "2019-09-04T21:51:43.956291Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "41 regions loaded to design probes.\n"
     ]
    }
   ],
   "source": [
    "# requires pre_defined genome_folder and library_folder\n",
    "# Indices\n",
    "genome_index = os.path.join(genome_folder, r'mm10_genome_17w.npy')\n",
    "repeat_index = os.path.join(genome_folder, r'repbase_genome_repeats_17w.npy')\n",
    "transcriptome_index = os.path.join(genome_folder, r'mm10_transcriptome_17w.npy') # NA for this design\n",
    "ref_merfish_index = os.path.join(pool_folder, 'M1_meng_MERFISH_17w.npy') # merfish designed by Meng\n",
    "# get input files \n",
    "input_files = glob.glob(os.path.join(sequence_folder, 'Slc32a1', '*.fasta'))\n",
    "# folder to save result probes\n",
    "report_folder = os.path.join(library_folder, 'reports')\n",
    "\n",
    "print(f\"{len(input_files)} regions loaded to design probes.\")\n",
    "overwrite = True\n",
    "\n",
    "if not os.path.exists(report_folder):\n",
    "    os.makedirs(report_folder)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-09-04T09:50:29.990193Z",
     "start_time": "2019-09-04T04:06:16.989980Z"
    },
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Setting attribute: map_genome\n",
      "Setting attribute: map_transcriptome\n",
      "Setting attribute: map_rep_genome\n",
      "Setting attribute: map_local_genome\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Mapping no. of seqs: 1\n",
      "Setting attribute: map_ref_merfish\n",
      "Time(s): 401.58351492881775\n",
      "Dealing with sequence: 1 out of 41\n",
      "Time(s): 137.12395000457764\n",
      "Dealing with sequence: 2 out of 41\n",
      "Time(s): 162.7679886817932\n",
      "Dealing with sequence: 3 out of 41\n",
      "Time(s): 155.38050174713135\n",
      "Dealing with sequence: 4 out of 41\n",
      "Time(s): 137.029860496521\n",
      "Dealing with sequence: 5 out of 41\n",
      "Time(s): 137.63230848312378\n",
      "Dealing with sequence: 6 out of 41\n",
      "Time(s): 143.26749682426453\n",
      "Dealing with sequence: 7 out of 41\n",
      "Time(s): 140.40336799621582\n",
      "Dealing with sequence: 8 out of 41\n",
      "Time(s): 138.59302234649658\n",
      "Dealing with sequence: 9 out of 41\n",
      "Time(s): 139.11240887641907\n",
      "Dealing with sequence: 10 out of 41\n",
      "Time(s): 140.13516879081726\n",
      "Dealing with sequence: 11 out of 41\n",
      "Time(s): 142.38784337043762\n",
      "Dealing with sequence: 12 out of 41\n",
      "Time(s): 140.8336877822876\n",
      "Dealing with sequence: 13 out of 41\n",
      "Time(s): 142.76713514328003\n",
      "Dealing with sequence: 14 out of 41\n",
      "Time(s): 140.63853406906128\n",
      "Dealing with sequence: 15 out of 41\n",
      "Time(s): 140.942768573761\n",
      "Dealing with sequence: 16 out of 41\n",
      "Time(s): 141.14491963386536\n",
      "Dealing with sequence: 17 out of 41\n",
      "Time(s): 143.1774308681488\n",
      "Dealing with sequence: 18 out of 41\n",
      "Time(s): 142.5940182209015\n",
      "Dealing with sequence: 19 out of 41\n",
      "Time(s): 142.32777619361877\n",
      "Dealing with sequence: 20 out of 41\n",
      "Time(s): 142.58098769187927\n",
      "Dealing with sequence: 21 out of 41\n",
      "Time(s): 141.34406733512878\n",
      "Dealing with sequence: 22 out of 41\n",
      "Time(s): 142.0175678730011\n",
      "Dealing with sequence: 23 out of 41\n",
      "Time(s): 141.71434259414673\n",
      "Dealing with sequence: 24 out of 41\n",
      "Time(s): 145.81038689613342\n",
      "Dealing with sequence: 25 out of 41\n",
      "Time(s): 142.28977060317993\n",
      "Dealing with sequence: 26 out of 41\n",
      "Time(s): 143.27550339698792\n",
      "Dealing with sequence: 27 out of 41\n",
      "Time(s): 142.81916451454163\n",
      "Dealing with sequence: 28 out of 41\n",
      "Time(s): 143.45263481140137\n",
      "Dealing with sequence: 29 out of 41\n",
      "Time(s): 142.44892501831055\n",
      "Dealing with sequence: 30 out of 41\n",
      "Time(s): 145.09986019134521\n",
      "Dealing with sequence: 31 out of 41\n",
      "Time(s): 144.02205753326416\n",
      "Dealing with sequence: 32 out of 41\n",
      "Time(s): 146.1846742630005\n",
      "Dealing with sequence: 33 out of 41\n",
      "Time(s): 144.6195023059845\n",
      "Dealing with sequence: 34 out of 41\n",
      "Time(s): 145.1589035987854\n",
      "Dealing with sequence: 35 out of 41\n",
      "Time(s): 148.5704562664032\n",
      "Dealing with sequence: 36 out of 41\n",
      "Time(s): 146.5749547481537\n",
      "Dealing with sequence: 37 out of 41\n",
      "Time(s): 145.55819940567017\n",
      "Dealing with sequence: 38 out of 41\n",
      "Time(s): 145.71131467819214\n",
      "Dealing with sequence: 39 out of 41\n",
      "Time(s): 148.490389585495\n",
      "Dealing with sequence: 40 out of 41\n",
      "Time(s): 146.27272200584412\n",
      "Dealing with sequence: 41 out of 41\n",
      "Time(s): 149.27596378326416\n",
      "Picking non-overlaping sequences.\n",
      "Time(s): 15.875800609588623\n",
      "-- number of probes kept: 5097\n"
     ]
    }
   ],
   "source": [
    "%matplotlib inline\n",
    "save_file = os.path.join(report_folder, f'merged_probes_supp_{int(resolution/1000)}kb.pbr')#'/pb_reports_reg_'+str(-i-1)+'.pbr'\n",
    "if not os.path.exists(save_file) or overwrite: # if probe report file doesnt exist\n",
    "\n",
    "    #local_genome_fl = in_file\n",
    "    #print(in_file)\n",
    "    pb_designer = ld.pb_reports_class(\n",
    "        sequence_dic={'file':input_files,\n",
    "                      'use_revc':True, # design two strands\n",
    "                      'use_kmer':True},\n",
    "        map_dic={'genome':{'file':genome_index,'use_revc':True,'use_kmer':True},\n",
    "                 'transcriptome':{'file':transcriptome_index,'use_revc':False,'use_kmer':True},\n",
    "                 'rep_genome':{'file':repeat_index,'use_revc':True,'use_kmer':True},\n",
    "                 'local_genome':{'file':input_files,'force_list':True,'use_revc':True,'use_kmer':True},\n",
    "                 'ref_merfish':{'file':ref_merfish_index,'use_revc':False,'use_kmer':True},\n",
    "                 },\n",
    "        save_file=save_file,\n",
    "        params_dic={'word_size':17,'pb_len':42,'buffer_len':0,'max_count':2**16-1,'check_on_go':False,'auto':False},\n",
    "        dic_check={#'genome':25+75, \n",
    "                   ('genome','local_genome'):25,\n",
    "                   'rep_genome':0,\n",
    "                   'transcriptome':25, # only one hit of 30 mer allowed\n",
    "                   #'local_genome':40, \n",
    "                   'ref_merfish': 25,\n",
    "                   'gc':[0.25,0.85],'tm':70})\n",
    "\n",
    "    pb_designer.computeOTmaps()\n",
    "    pb_designer.compute_pb_report()\n",
    "    pb_designer.perform_check_end()\n",
    "    pb_designer.plots()\n",
    "    pb_designer.save_csv()\n",
    "    print(f\"-- number of probes kept: {len(pb_designer.pb_reports_keep)}\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "pbde = ld.pb_reports_class()\n",
    "pbde.load_pbr(save_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "list(pbde.pb_reports_keep.values())[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "pb_designer.sequence_dic[3]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2.3 summarize into a pb_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- appending region:1 with 59 probes\n",
      "-- appending region:2 with 103 probes\n",
      "-- appending region:3 with 95 probes\n",
      "-- appending region:4 with 135 probes\n",
      "-- appending region:5 with 115 probes\n",
      "-- appending region:6 with 90 probes\n",
      "-- appending region:7 with 139 probes\n",
      "-- appending region:8 with 84 probes\n",
      "-- appending region:9 with 89 probes\n",
      "-- appending region:10 with 169 probes\n",
      "-- appending region:11 with 156 probes\n",
      "-- appending region:12 with 176 probes\n",
      "-- appending region:13 with 162 probes\n",
      "-- appending region:14 with 127 probes\n",
      "-- appending region:15 with 161 probes\n",
      "-- appending region:16 with 137 probes\n",
      "-- appending region:17 with 153 probes\n",
      "-- appending region:18 with 136 probes\n",
      "-- appending region:19 with 155 probes\n",
      "-- appending region:20 with 161 probes\n",
      "-- appending region:21 with 135 probes\n",
      "-- appending region:22 with 57 probes\n",
      "-- appending region:23 with 27 probes\n",
      "-- appending region:24 with 152 probes\n",
      "-- appending region:25 with 148 probes\n",
      "-- appending region:26 with 138 probes\n",
      "-- appending region:27 with 176 probes\n",
      "-- appending region:28 with 168 probes\n",
      "-- appending region:29 with 162 probes\n",
      "-- appending region:30 with 115 probes\n",
      "-- appending region:31 with 183 probes\n",
      "-- appending region:32 with 148 probes\n",
      "-- appending region:33 with 142 probes\n",
      "-- appending region:34 with 166 probes\n",
      "-- appending region:35 with 121 probes\n",
      "-- appending region:36 with 88 probes\n",
      "-- appending region:37 with 195 probes\n",
      "-- appending region:38 with 112 probes\n",
      "-- appending region:39 with 62 probes\n"
     ]
    }
   ],
   "source": [
    "verbose=True\n",
    "cand_prob_nums = []\n",
    "pb_dic = {}\n",
    "# Parse segemented list\n",
    "_save_file = os.path.join(report_folder, f'merged_probes_supp_{int(resolution/1000)}kb.pbr')\n",
    "_pbde = ld.pb_reports_class()\n",
    "_pbde.load_pbr(_save_file)\n",
    "\n",
    "reg_inds = np.unique([_v['reg_index'] for _s, _v in _pbde.pb_reports_keep.items()])\n",
    "for _rid in reg_inds:\n",
    "    _sel_pbs = {_s:_v for _s, _v in _pbde.pb_reports_keep.items() if _v['reg_index']==_rid}\n",
    "\n",
    "    if len(_sel_pbs) > 0:\n",
    "        _region_gid = list(_sel_pbs.values())[0]['reg_name'].split('_')[-1]\n",
    "        cand_prob_nums.append(len(_sel_pbs))\n",
    "        if verbose:\n",
    "            print(f\"-- appending region:{_region_gid} with {len(_sel_pbs)} probes\")\n",
    "        _splitted_pbde = ld.pb_reports_class()\n",
    "        _splitted_pbde.pb_reports_keep = _sel_pbs\n",
    "        pb_dic[int(_region_gid)] = _splitted_pbde"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Screen probes by genome counts.\n",
      "-- filtering region:1, directly append 113 probes\n",
      "-- filtering region:2, directly append 106 probes\n",
      "-- filtering region:4, directly append 2 probes\n",
      "-- filtering region:5, directly append 104 probes\n",
      "-- filtering region:6, directly append 104 probes\n",
      "-- filtering region:7, directly append 146 probes\n",
      "-- filtering region:8, screen genome to keep 150 probes\n",
      "-- filtering region:9, screen genome to keep 150 probes\n",
      "-- filtering region:10, directly append 68 probes\n",
      "-- filtering region:12, directly append 21 probes\n",
      "-- filtering region:13, screen genome to keep 150 probes\n",
      "-- filtering region:14, screen genome to keep 150 probes\n",
      "-- filtering region:15, directly append 143 probes\n",
      "-- filtering region:16, directly append 118 probes\n",
      "-- filtering region:17, directly append 120 probes\n",
      "-- filtering region:18, directly append 68 probes\n",
      "-- filtering region:19, directly append 44 probes\n",
      "-- filtering region:20, screen genome to keep 150 probes\n",
      "-- filtering region:21, screen genome to keep 150 probes\n",
      "-- filtering region:22, directly append 148 probes\n"
     ]
    }
   ],
   "source": [
    "# filter by genome counts\n",
    "num_probes_kept = 150 # max num of probes kept for each region\n",
    "filtered_pb_dic = library_tools.quality_check.Screen_probe_by_hit(library_folder, pb_dic, num_probes_kept,\n",
    "                                                                  hit_type='genome')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "region: 4 removed with 2 probes\n",
      "region: 12 removed with 21 probes\n"
     ]
    }
   ],
   "source": [
    "## filter by minimum number of probes in each region\n",
    "min_num_probes = 25 \n",
    "removed_regions = []\n",
    "for _rid, _pb_class in filtered_pb_dic.items():\n",
    "    if len(_pb_class.pb_reports_keep) < min_num_probes:\n",
    "        print(f\"region: {_rid} removed with {len(_pb_class.pb_reports_keep)} probes\")\n",
    "        removed_regions.append(_rid)\n",
    "for _rid in removed_regions:\n",
    "    del(filtered_pb_dic[_rid])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2.4 check candidate probe number per region"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2019-09-04T21:20:54.477228Z",
     "start_time": "2019-09-04T21:20:54.469189Z"
    }
   },
   "outputs": [],
   "source": [
    "pb_num_dict = {_k:len(_v.pb_reports_keep) for _k,_v in filtered_pb_dic.items()}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "/* Put everything inside the global mpl namespace */\n",
       "window.mpl = {};\n",
       "\n",
       "\n",
       "mpl.get_websocket_type = function() {\n",
       "    if (typeof(WebSocket) !== 'undefined') {\n",
       "        return WebSocket;\n",
       "    } else if (typeof(MozWebSocket) !== 'undefined') {\n",
       "        return MozWebSocket;\n",
       "    } else {\n",
       "        alert('Your browser does not have WebSocket support. ' +\n",
       "              'Please try Chrome, Safari or Firefox â‰¥ 6. ' +\n",
       "              'Firefox 4 and 5 are also supported but you ' +\n",
       "              'have to enable WebSockets in about:config.');\n",
       "    };\n",
       "}\n",
       "\n",
       "mpl.figure = function(figure_id, websocket, ondownload, parent_element) {\n",
       "    this.id = figure_id;\n",
       "\n",
       "    this.ws = websocket;\n",
       "\n",
       "    this.supports_binary = (this.ws.binaryType != undefined);\n",
       "\n",
       "    if (!this.supports_binary) {\n",
       "        var warnings = document.getElementById(\"mpl-warnings\");\n",
       "        if (warnings) {\n",
       "            warnings.style.display = 'block';\n",
       "            warnings.textContent = (\n",
       "                \"This browser does not support binary websocket messages. \" +\n",
       "                    \"Performance may be slow.\");\n",
       "        }\n",
       "    }\n",
       "\n",
       "    this.imageObj = new Image();\n",
       "\n",
       "    this.context = undefined;\n",
       "    this.message = undefined;\n",
       "    this.canvas = undefined;\n",
       "    this.rubberband_canvas = undefined;\n",
       "    this.rubberband_context = undefined;\n",
       "    this.format_dropdown = undefined;\n",
       "\n",
       "    this.image_mode = 'full';\n",
       "\n",
       "    this.root = $('<div/>');\n",
       "    this._root_extra_style(this.root)\n",
       "    this.root.attr('style', 'display: inline-block');\n",
       "\n",
       "    $(parent_element).append(this.root);\n",
       "\n",
       "    this._init_header(this);\n",
       "    this._init_canvas(this);\n",
       "    this._init_toolbar(this);\n",
       "\n",
       "    var fig = this;\n",
       "\n",
       "    this.waiting = false;\n",
       "\n",
       "    this.ws.onopen =  function () {\n",
       "            fig.send_message(\"supports_binary\", {value: fig.supports_binary});\n",
       "            fig.send_message(\"send_image_mode\", {});\n",
       "            if (mpl.ratio != 1) {\n",
       "                fig.send_message(\"set_dpi_ratio\", {'dpi_ratio': mpl.ratio});\n",
       "            }\n",
       "            fig.send_message(\"refresh\", {});\n",
       "        }\n",
       "\n",
       "    this.imageObj.onload = function() {\n",
       "            if (fig.image_mode == 'full') {\n",
       "                // Full images could contain transparency (where diff images\n",
       "                // almost always do), so we need to clear the canvas so that\n",
       "                // there is no ghosting.\n",
       "                fig.context.clearRect(0, 0, fig.canvas.width, fig.canvas.height);\n",
       "            }\n",
       "            fig.context.drawImage(fig.imageObj, 0, 0);\n",
       "        };\n",
       "\n",
       "    this.imageObj.onunload = function() {\n",
       "        fig.ws.close();\n",
       "    }\n",
       "\n",
       "    this.ws.onmessage = this._make_on_message_function(this);\n",
       "\n",
       "    this.ondownload = ondownload;\n",
       "}\n",
       "\n",
       "mpl.figure.prototype._init_header = function() {\n",
       "    var titlebar = $(\n",
       "        '<div class=\"ui-dialog-titlebar ui-widget-header ui-corner-all ' +\n",
       "        'ui-helper-clearfix\"/>');\n",
       "    var titletext = $(\n",
       "        '<div class=\"ui-dialog-title\" style=\"width: 100%; ' +\n",
       "        'text-align: center; padding: 3px;\"/>');\n",
       "    titlebar.append(titletext)\n",
       "    this.root.append(titlebar);\n",
       "    this.header = titletext[0];\n",
       "}\n",
       "\n",
       "\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function(canvas_div) {\n",
       "\n",
       "}\n",
       "\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function(canvas_div) {\n",
       "\n",
       "}\n",
       "\n",
       "mpl.figure.prototype._init_canvas = function() {\n",
       "    var fig = this;\n",
       "\n",
       "    var canvas_div = $('<div/>');\n",
       "\n",
       "    canvas_div.attr('style', 'position: relative; clear: both; outline: 0');\n",
       "\n",
       "    function canvas_keyboard_event(event) {\n",
       "        return fig.key_event(event, event['data']);\n",
       "    }\n",
       "\n",
       "    canvas_div.keydown('key_press', canvas_keyboard_event);\n",
       "    canvas_div.keyup('key_release', canvas_keyboard_event);\n",
       "    this.canvas_div = canvas_div\n",
       "    this._canvas_extra_style(canvas_div)\n",
       "    this.root.append(canvas_div);\n",
       "\n",
       "    var canvas = $('<canvas/>');\n",
       "    canvas.addClass('mpl-canvas');\n",
       "    canvas.attr('style', \"left: 0; top: 0; z-index: 0; outline: 0\")\n",
       "\n",
       "    this.canvas = canvas[0];\n",
       "    this.context = canvas[0].getContext(\"2d\");\n",
       "\n",
       "    var backingStore = this.context.backingStorePixelRatio ||\n",
       "\tthis.context.webkitBackingStorePixelRatio ||\n",
       "\tthis.context.mozBackingStorePixelRatio ||\n",
       "\tthis.context.msBackingStorePixelRatio ||\n",
       "\tthis.context.oBackingStorePixelRatio ||\n",
       "\tthis.context.backingStorePixelRatio || 1;\n",
       "\n",
       "    mpl.ratio = (window.devicePixelRatio || 1) / backingStore;\n",
       "\n",
       "    var rubberband = $('<canvas/>');\n",
       "    rubberband.attr('style', \"position: absolute; left: 0; top: 0; z-index: 1;\")\n",
       "\n",
       "    var pass_mouse_events = true;\n",
       "\n",
       "    canvas_div.resizable({\n",
       "        start: function(event, ui) {\n",
       "            pass_mouse_events = false;\n",
       "        },\n",
       "        resize: function(event, ui) {\n",
       "            fig.request_resize(ui.size.width, ui.size.height);\n",
       "        },\n",
       "        stop: function(event, ui) {\n",
       "            pass_mouse_events = true;\n",
       "            fig.request_resize(ui.size.width, ui.size.height);\n",
       "        },\n",
       "    });\n",
       "\n",
       "    function mouse_event_fn(event) {\n",
       "        if (pass_mouse_events)\n",
       "            return fig.mouse_event(event, event['data']);\n",
       "    }\n",
       "\n",
       "    rubberband.mousedown('button_press', mouse_event_fn);\n",
       "    rubberband.mouseup('button_release', mouse_event_fn);\n",
       "    // Throttle sequential mouse events to 1 every 20ms.\n",
       "    rubberband.mousemove('motion_notify', mouse_event_fn);\n",
       "\n",
       "    rubberband.mouseenter('figure_enter', mouse_event_fn);\n",
       "    rubberband.mouseleave('figure_leave', mouse_event_fn);\n",
       "\n",
       "    canvas_div.on(\"wheel\", function (event) {\n",
       "        event = event.originalEvent;\n",
       "        event['data'] = 'scroll'\n",
       "        if (event.deltaY < 0) {\n",
       "            event.step = 1;\n",
       "        } else {\n",
       "            event.step = -1;\n",
       "        }\n",
       "        mouse_event_fn(event);\n",
       "    });\n",
       "\n",
       "    canvas_div.append(canvas);\n",
       "    canvas_div.append(rubberband);\n",
       "\n",
       "    this.rubberband = rubberband;\n",
       "    this.rubberband_canvas = rubberband[0];\n",
       "    this.rubberband_context = rubberband[0].getContext(\"2d\");\n",
       "    this.rubberband_context.strokeStyle = \"#000000\";\n",
       "\n",
       "    this._resize_canvas = function(width, height) {\n",
       "        // Keep the size of the canvas, canvas container, and rubber band\n",
       "        // canvas in synch.\n",
       "        canvas_div.css('width', width)\n",
       "        canvas_div.css('height', height)\n",
       "\n",
       "        canvas.attr('width', width * mpl.ratio);\n",
       "        canvas.attr('height', height * mpl.ratio);\n",
       "        canvas.attr('style', 'width: ' + width + 'px; height: ' + height + 'px;');\n",
       "\n",
       "        rubberband.attr('width', width);\n",
       "        rubberband.attr('height', height);\n",
       "    }\n",
       "\n",
       "    // Set the figure to an initial 600x600px, this will subsequently be updated\n",
       "    // upon first draw.\n",
       "    this._resize_canvas(600, 600);\n",
       "\n",
       "    // Disable right mouse context menu.\n",
       "    $(this.rubberband_canvas).bind(\"contextmenu\",function(e){\n",
       "        return false;\n",
       "    });\n",
       "\n",
       "    function set_focus () {\n",
       "        canvas.focus();\n",
       "        canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    window.setTimeout(set_focus, 100);\n",
       "}\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function() {\n",
       "    var fig = this;\n",
       "\n",
       "    var nav_element = $('<div/>');\n",
       "    nav_element.attr('style', 'width: 100%');\n",
       "    this.root.append(nav_element);\n",
       "\n",
       "    // Define a callback function for later on.\n",
       "    function toolbar_event(event) {\n",
       "        return fig.toolbar_button_onclick(event['data']);\n",
       "    }\n",
       "    function toolbar_mouse_event(event) {\n",
       "        return fig.toolbar_button_onmouseover(event['data']);\n",
       "    }\n",
       "\n",
       "    for(var toolbar_ind in mpl.toolbar_items) {\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) {\n",
       "            // put a spacer in here.\n",
       "            continue;\n",
       "        }\n",
       "        var button = $('<button/>');\n",
       "        button.addClass('ui-button ui-widget ui-state-default ui-corner-all ' +\n",
       "                        'ui-button-icon-only');\n",
       "        button.attr('role', 'button');\n",
       "        button.attr('aria-disabled', 'false');\n",
       "        button.click(method_name, toolbar_event);\n",
       "        button.mouseover(tooltip, toolbar_mouse_event);\n",
       "\n",
       "        var icon_img = $('<span/>');\n",
       "        icon_img.addClass('ui-button-icon-primary ui-icon');\n",
       "        icon_img.addClass(image);\n",
       "        icon_img.addClass('ui-corner-all');\n",
       "\n",
       "        var tooltip_span = $('<span/>');\n",
       "        tooltip_span.addClass('ui-button-text');\n",
       "        tooltip_span.html(tooltip);\n",
       "\n",
       "        button.append(icon_img);\n",
       "        button.append(tooltip_span);\n",
       "\n",
       "        nav_element.append(button);\n",
       "    }\n",
       "\n",
       "    var fmt_picker_span = $('<span/>');\n",
       "\n",
       "    var fmt_picker = $('<select/>');\n",
       "    fmt_picker.addClass('mpl-toolbar-option ui-widget ui-widget-content');\n",
       "    fmt_picker_span.append(fmt_picker);\n",
       "    nav_element.append(fmt_picker_span);\n",
       "    this.format_dropdown = fmt_picker[0];\n",
       "\n",
       "    for (var ind in mpl.extensions) {\n",
       "        var fmt = mpl.extensions[ind];\n",
       "        var option = $(\n",
       "            '<option/>', {selected: fmt === mpl.default_extension}).html(fmt);\n",
       "        fmt_picker.append(option);\n",
       "    }\n",
       "\n",
       "    // Add hover states to the ui-buttons\n",
       "    $( \".ui-button\" ).hover(\n",
       "        function() { $(this).addClass(\"ui-state-hover\");},\n",
       "        function() { $(this).removeClass(\"ui-state-hover\");}\n",
       "    );\n",
       "\n",
       "    var status_bar = $('<span class=\"mpl-message\"/>');\n",
       "    nav_element.append(status_bar);\n",
       "    this.message = status_bar[0];\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.request_resize = function(x_pixels, y_pixels) {\n",
       "    // Request matplotlib to resize the figure. Matplotlib will then trigger a resize in the client,\n",
       "    // which will in turn request a refresh of the image.\n",
       "    this.send_message('resize', {'width': x_pixels, 'height': y_pixels});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.send_message = function(type, properties) {\n",
       "    properties['type'] = type;\n",
       "    properties['figure_id'] = this.id;\n",
       "    this.ws.send(JSON.stringify(properties));\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.send_draw_message = function() {\n",
       "    if (!this.waiting) {\n",
       "        this.waiting = true;\n",
       "        this.ws.send(JSON.stringify({type: \"draw\", figure_id: this.id}));\n",
       "    }\n",
       "}\n",
       "\n",
       "\n",
       "mpl.figure.prototype.handle_save = function(fig, msg) {\n",
       "    var format_dropdown = fig.format_dropdown;\n",
       "    var format = format_dropdown.options[format_dropdown.selectedIndex].value;\n",
       "    fig.ondownload(fig, format);\n",
       "}\n",
       "\n",
       "\n",
       "mpl.figure.prototype.handle_resize = function(fig, msg) {\n",
       "    var size = msg['size'];\n",
       "    if (size[0] != fig.canvas.width || size[1] != fig.canvas.height) {\n",
       "        fig._resize_canvas(size[0], size[1]);\n",
       "        fig.send_message(\"refresh\", {});\n",
       "    };\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.handle_rubberband = function(fig, msg) {\n",
       "    var x0 = msg['x0'] / mpl.ratio;\n",
       "    var y0 = (fig.canvas.height - msg['y0']) / mpl.ratio;\n",
       "    var x1 = msg['x1'] / mpl.ratio;\n",
       "    var y1 = (fig.canvas.height - msg['y1']) / mpl.ratio;\n",
       "    x0 = Math.floor(x0) + 0.5;\n",
       "    y0 = Math.floor(y0) + 0.5;\n",
       "    x1 = Math.floor(x1) + 0.5;\n",
       "    y1 = Math.floor(y1) + 0.5;\n",
       "    var min_x = Math.min(x0, x1);\n",
       "    var min_y = Math.min(y0, y1);\n",
       "    var width = Math.abs(x1 - x0);\n",
       "    var height = Math.abs(y1 - y0);\n",
       "\n",
       "    fig.rubberband_context.clearRect(\n",
       "        0, 0, fig.canvas.width / mpl.ratio, fig.canvas.height / mpl.ratio);\n",
       "\n",
       "    fig.rubberband_context.strokeRect(min_x, min_y, width, height);\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.handle_figure_label = function(fig, msg) {\n",
       "    // Updates the figure title.\n",
       "    fig.header.textContent = msg['label'];\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.handle_cursor = function(fig, msg) {\n",
       "    var cursor = msg['cursor'];\n",
       "    switch(cursor)\n",
       "    {\n",
       "    case 0:\n",
       "        cursor = 'pointer';\n",
       "        break;\n",
       "    case 1:\n",
       "        cursor = 'default';\n",
       "        break;\n",
       "    case 2:\n",
       "        cursor = 'crosshair';\n",
       "        break;\n",
       "    case 3:\n",
       "        cursor = 'move';\n",
       "        break;\n",
       "    }\n",
       "    fig.rubberband_canvas.style.cursor = cursor;\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.handle_message = function(fig, msg) {\n",
       "    fig.message.textContent = msg['message'];\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.handle_draw = function(fig, msg) {\n",
       "    // Request the server to send over a new figure.\n",
       "    fig.send_draw_message();\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.handle_image_mode = function(fig, msg) {\n",
       "    fig.image_mode = msg['mode'];\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function() {\n",
       "    // Called whenever the canvas gets updated.\n",
       "    this.send_message(\"ack\", {});\n",
       "}\n",
       "\n",
       "// A function to construct a web socket function for onmessage handling.\n",
       "// Called in the figure constructor.\n",
       "mpl.figure.prototype._make_on_message_function = function(fig) {\n",
       "    return function socket_on_message(evt) {\n",
       "        if (evt.data instanceof Blob) {\n",
       "            /* FIXME: We get \"Resource interpreted as Image but\n",
       "             * transferred with MIME type text/plain:\" errors on\n",
       "             * Chrome.  But how to set the MIME type?  It doesn't seem\n",
       "             * to be part of the websocket stream */\n",
       "            evt.data.type = \"image/png\";\n",
       "\n",
       "            /* Free the memory for the previous frames */\n",
       "            if (fig.imageObj.src) {\n",
       "                (window.URL || window.webkitURL).revokeObjectURL(\n",
       "                    fig.imageObj.src);\n",
       "            }\n",
       "\n",
       "            fig.imageObj.src = (window.URL || window.webkitURL).createObjectURL(\n",
       "                evt.data);\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "        else if (typeof evt.data === 'string' && evt.data.slice(0, 21) == \"data:image/png;base64\") {\n",
       "            fig.imageObj.src = evt.data;\n",
       "            fig.updated_canvas_event();\n",
       "            fig.waiting = false;\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        var msg = JSON.parse(evt.data);\n",
       "        var msg_type = msg['type'];\n",
       "\n",
       "        // Call the  \"handle_{type}\" callback, which takes\n",
       "        // the figure and JSON message as its only arguments.\n",
       "        try {\n",
       "            var callback = fig[\"handle_\" + msg_type];\n",
       "        } catch (e) {\n",
       "            console.log(\"No handler for the '\" + msg_type + \"' message type: \", msg);\n",
       "            return;\n",
       "        }\n",
       "\n",
       "        if (callback) {\n",
       "            try {\n",
       "                // console.log(\"Handling '\" + msg_type + \"' message: \", msg);\n",
       "                callback(fig, msg);\n",
       "            } catch (e) {\n",
       "                console.log(\"Exception inside the 'handler_\" + msg_type + \"' callback:\", e, e.stack, msg);\n",
       "            }\n",
       "        }\n",
       "    };\n",
       "}\n",
       "\n",
       "// from http://stackoverflow.com/questions/1114465/getting-mouse-location-in-canvas\n",
       "mpl.findpos = function(e) {\n",
       "    //this section is from http://www.quirksmode.org/js/events_properties.html\n",
       "    var targ;\n",
       "    if (!e)\n",
       "        e = window.event;\n",
       "    if (e.target)\n",
       "        targ = e.target;\n",
       "    else if (e.srcElement)\n",
       "        targ = e.srcElement;\n",
       "    if (targ.nodeType == 3) // defeat Safari bug\n",
       "        targ = targ.parentNode;\n",
       "\n",
       "    // jQuery normalizes the pageX and pageY\n",
       "    // pageX,Y are the mouse positions relative to the document\n",
       "    // offset() returns the position of the element relative to the document\n",
       "    var x = e.pageX - $(targ).offset().left;\n",
       "    var y = e.pageY - $(targ).offset().top;\n",
       "\n",
       "    return {\"x\": x, \"y\": y};\n",
       "};\n",
       "\n",
       "/*\n",
       " * return a copy of an object with only non-object keys\n",
       " * we need this to avoid circular references\n",
       " * http://stackoverflow.com/a/24161582/3208463\n",
       " */\n",
       "function simpleKeys (original) {\n",
       "  return Object.keys(original).reduce(function (obj, key) {\n",
       "    if (typeof original[key] !== 'object')\n",
       "        obj[key] = original[key]\n",
       "    return obj;\n",
       "  }, {});\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.mouse_event = function(event, name) {\n",
       "    var canvas_pos = mpl.findpos(event)\n",
       "\n",
       "    if (name === 'button_press')\n",
       "    {\n",
       "        this.canvas.focus();\n",
       "        this.canvas_div.focus();\n",
       "    }\n",
       "\n",
       "    var x = canvas_pos.x * mpl.ratio;\n",
       "    var y = canvas_pos.y * mpl.ratio;\n",
       "\n",
       "    this.send_message(name, {x: x, y: y, button: event.button,\n",
       "                             step: event.step,\n",
       "                             guiEvent: simpleKeys(event)});\n",
       "\n",
       "    /* This prevents the web browser from automatically changing to\n",
       "     * the text insertion cursor when the button is pressed.  We want\n",
       "     * to control all of the cursor setting manually through the\n",
       "     * 'cursor' event from matplotlib */\n",
       "    event.preventDefault();\n",
       "    return false;\n",
       "}\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function(event, name) {\n",
       "    // Handle any extra behaviour associated with a key event\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.key_event = function(event, name) {\n",
       "\n",
       "    // Prevent repeat events\n",
       "    if (name == 'key_press')\n",
       "    {\n",
       "        if (event.which === this._key)\n",
       "            return;\n",
       "        else\n",
       "            this._key = event.which;\n",
       "    }\n",
       "    if (name == 'key_release')\n",
       "        this._key = null;\n",
       "\n",
       "    var value = '';\n",
       "    if (event.ctrlKey && event.which != 17)\n",
       "        value += \"ctrl+\";\n",
       "    if (event.altKey && event.which != 18)\n",
       "        value += \"alt+\";\n",
       "    if (event.shiftKey && event.which != 16)\n",
       "        value += \"shift+\";\n",
       "\n",
       "    value += 'k';\n",
       "    value += event.which.toString();\n",
       "\n",
       "    this._key_event_extra(event, name);\n",
       "\n",
       "    this.send_message(name, {key: value,\n",
       "                             guiEvent: simpleKeys(event)});\n",
       "    return false;\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onclick = function(name) {\n",
       "    if (name == 'download') {\n",
       "        this.handle_save(this, null);\n",
       "    } else {\n",
       "        this.send_message(\"toolbar_button\", {name: name});\n",
       "    }\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.toolbar_button_onmouseover = function(tooltip) {\n",
       "    this.message.textContent = tooltip;\n",
       "};\n",
       "mpl.toolbar_items = [[\"Home\", \"Reset original view\", \"fa fa-home icon-home\", \"home\"], [\"Back\", \"Back to previous view\", \"fa fa-arrow-left icon-arrow-left\", \"back\"], [\"Forward\", \"Forward to next view\", \"fa fa-arrow-right icon-arrow-right\", \"forward\"], [\"\", \"\", \"\", \"\"], [\"Pan\", \"Pan axes with left mouse, zoom with right\", \"fa fa-arrows icon-move\", \"pan\"], [\"Zoom\", \"Zoom to rectangle\", \"fa fa-square-o icon-check-empty\", \"zoom\"], [\"\", \"\", \"\", \"\"], [\"Download\", \"Download plot\", \"fa fa-floppy-o icon-save\", \"download\"]];\n",
       "\n",
       "mpl.extensions = [\"eps\", \"jpeg\", \"pdf\", \"png\", \"ps\", \"raw\", \"svg\", \"tif\"];\n",
       "\n",
       "mpl.default_extension = \"png\";var comm_websocket_adapter = function(comm) {\n",
       "    // Create a \"websocket\"-like object which calls the given IPython comm\n",
       "    // object with the appropriate methods. Currently this is a non binary\n",
       "    // socket, so there is still some room for performance tuning.\n",
       "    var ws = {};\n",
       "\n",
       "    ws.close = function() {\n",
       "        comm.close()\n",
       "    };\n",
       "    ws.send = function(m) {\n",
       "        //console.log('sending', m);\n",
       "        comm.send(m);\n",
       "    };\n",
       "    // Register the callback with on_msg.\n",
       "    comm.on_msg(function(msg) {\n",
       "        //console.log('receiving', msg['content']['data'], msg);\n",
       "        // Pass the mpl event to the overridden (by mpl) onmessage function.\n",
       "        ws.onmessage(msg['content']['data'])\n",
       "    });\n",
       "    return ws;\n",
       "}\n",
       "\n",
       "mpl.mpl_figure_comm = function(comm, msg) {\n",
       "    // This is the function which gets called when the mpl process\n",
       "    // starts-up an IPython Comm through the \"matplotlib\" channel.\n",
       "\n",
       "    var id = msg.content.data.id;\n",
       "    // Get hold of the div created by the display call when the Comm\n",
       "    // socket was opened in Python.\n",
       "    var element = $(\"#\" + id);\n",
       "    var ws_proxy = comm_websocket_adapter(comm)\n",
       "\n",
       "    function ondownload(figure, format) {\n",
       "        window.open(figure.imageObj.src);\n",
       "    }\n",
       "\n",
       "    var fig = new mpl.figure(id, ws_proxy,\n",
       "                           ondownload,\n",
       "                           element.get(0));\n",
       "\n",
       "    // Call onopen now - mpl needs it, as it is assuming we've passed it a real\n",
       "    // web socket which is closed, not our websocket->open comm proxy.\n",
       "    ws_proxy.onopen();\n",
       "\n",
       "    fig.parent_element = element.get(0);\n",
       "    fig.cell_info = mpl.find_output_cell(\"<div id='\" + id + \"'></div>\");\n",
       "    if (!fig.cell_info) {\n",
       "        console.error(\"Failed to find cell for figure\", id, fig);\n",
       "        return;\n",
       "    }\n",
       "\n",
       "    var output_index = fig.cell_info[2]\n",
       "    var cell = fig.cell_info[0];\n",
       "\n",
       "};\n",
       "\n",
       "mpl.figure.prototype.handle_close = function(fig, msg) {\n",
       "    var width = fig.canvas.width/mpl.ratio\n",
       "    fig.root.unbind('remove')\n",
       "\n",
       "    // Update the output cell to use the data from the current canvas.\n",
       "    fig.push_to_output();\n",
       "    var dataURL = fig.canvas.toDataURL();\n",
       "    // Re-enable the keyboard manager in IPython - without this line, in FF,\n",
       "    // the notebook keyboard shortcuts fail.\n",
       "    IPython.keyboard_manager.enable()\n",
       "    $(fig.parent_element).html('<img src=\"' + dataURL + '\" width=\"' + width + '\">');\n",
       "    fig.close_ws(fig, msg);\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.close_ws = function(fig, msg){\n",
       "    fig.send_message('closing', msg);\n",
       "    // fig.ws.close()\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.push_to_output = function(remove_interactive) {\n",
       "    // Turn the data on the canvas into data in the output cell.\n",
       "    var width = this.canvas.width/mpl.ratio\n",
       "    var dataURL = this.canvas.toDataURL();\n",
       "    this.cell_info[1]['text/html'] = '<img src=\"' + dataURL + '\" width=\"' + width + '\">';\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.updated_canvas_event = function() {\n",
       "    // Tell IPython that the notebook contents must change.\n",
       "    IPython.notebook.set_dirty(true);\n",
       "    this.send_message(\"ack\", {});\n",
       "    var fig = this;\n",
       "    // Wait a second, then push the new image to the DOM so\n",
       "    // that it is saved nicely (might be nice to debounce this).\n",
       "    setTimeout(function () { fig.push_to_output() }, 1000);\n",
       "}\n",
       "\n",
       "mpl.figure.prototype._init_toolbar = function() {\n",
       "    var fig = this;\n",
       "\n",
       "    var nav_element = $('<div/>');\n",
       "    nav_element.attr('style', 'width: 100%');\n",
       "    this.root.append(nav_element);\n",
       "\n",
       "    // Define a callback function for later on.\n",
       "    function toolbar_event(event) {\n",
       "        return fig.toolbar_button_onclick(event['data']);\n",
       "    }\n",
       "    function toolbar_mouse_event(event) {\n",
       "        return fig.toolbar_button_onmouseover(event['data']);\n",
       "    }\n",
       "\n",
       "    for(var toolbar_ind in mpl.toolbar_items){\n",
       "        var name = mpl.toolbar_items[toolbar_ind][0];\n",
       "        var tooltip = mpl.toolbar_items[toolbar_ind][1];\n",
       "        var image = mpl.toolbar_items[toolbar_ind][2];\n",
       "        var method_name = mpl.toolbar_items[toolbar_ind][3];\n",
       "\n",
       "        if (!name) { continue; };\n",
       "\n",
       "        var button = $('<button class=\"btn btn-default\" href=\"#\" title=\"' + name + '\"><i class=\"fa ' + image + ' fa-lg\"></i></button>');\n",
       "        button.click(method_name, toolbar_event);\n",
       "        button.mouseover(tooltip, toolbar_mouse_event);\n",
       "        nav_element.append(button);\n",
       "    }\n",
       "\n",
       "    // Add the status bar.\n",
       "    var status_bar = $('<span class=\"mpl-message\" style=\"text-align:right; float: right;\"/>');\n",
       "    nav_element.append(status_bar);\n",
       "    this.message = status_bar[0];\n",
       "\n",
       "    // Add the close button to the window.\n",
       "    var buttongrp = $('<div class=\"btn-group inline pull-right\"></div>');\n",
       "    var button = $('<button class=\"btn btn-mini btn-primary\" href=\"#\" title=\"Stop Interaction\"><i class=\"fa fa-power-off icon-remove icon-large\"></i></button>');\n",
       "    button.click(function (evt) { fig.handle_close(fig, {}); } );\n",
       "    button.mouseover('Stop Interaction', toolbar_mouse_event);\n",
       "    buttongrp.append(button);\n",
       "    var titlebar = this.root.find($('.ui-dialog-titlebar'));\n",
       "    titlebar.prepend(buttongrp);\n",
       "}\n",
       "\n",
       "mpl.figure.prototype._root_extra_style = function(el){\n",
       "    var fig = this\n",
       "    el.on(\"remove\", function(){\n",
       "\tfig.close_ws(fig, {});\n",
       "    });\n",
       "}\n",
       "\n",
       "mpl.figure.prototype._canvas_extra_style = function(el){\n",
       "    // this is important to make the div 'focusable\n",
       "    el.attr('tabindex', 0)\n",
       "    // reach out to IPython and tell the keyboard manager to turn it's self\n",
       "    // off when our div gets focus\n",
       "\n",
       "    // location in version 3\n",
       "    if (IPython.notebook.keyboard_manager) {\n",
       "        IPython.notebook.keyboard_manager.register_events(el);\n",
       "    }\n",
       "    else {\n",
       "        // location in version 2\n",
       "        IPython.keyboard_manager.register_events(el);\n",
       "    }\n",
       "\n",
       "}\n",
       "\n",
       "mpl.figure.prototype._key_event_extra = function(event, name) {\n",
       "    var manager = IPython.notebook.keyboard_manager;\n",
       "    if (!manager)\n",
       "        manager = IPython.keyboard_manager;\n",
       "\n",
       "    // Check for shift+enter\n",
       "    if (event.shiftKey && event.which == 13) {\n",
       "        this.canvas_div.blur();\n",
       "        // select the cell after this one\n",
       "        var index = IPython.notebook.find_cell_index(this.cell_info[0]);\n",
       "        IPython.notebook.select(index + 1);\n",
       "    }\n",
       "}\n",
       "\n",
       "mpl.figure.prototype.handle_save = function(fig, msg) {\n",
       "    fig.ondownload(fig, null);\n",
       "}\n",
       "\n",
       "\n",
       "mpl.find_output_cell = function(html_output) {\n",
       "    // Return the cell and output element which can be found *uniquely* in the notebook.\n",
       "    // Note - this is a bit hacky, but it is done because the \"notebook_saving.Notebook\"\n",
       "    // IPython event is triggered only after the cells have been serialised, which for\n",
       "    // our purposes (turning an active figure into a static one), is too late.\n",
       "    var cells = IPython.notebook.get_cells();\n",
       "    var ncells = cells.length;\n",
       "    for (var i=0; i<ncells; i++) {\n",
       "        var cell = cells[i];\n",
       "        if (cell.cell_type === 'code'){\n",
       "            for (var j=0; j<cell.output_area.outputs.length; j++) {\n",
       "                var data = cell.output_area.outputs[j];\n",
       "                if (data.data) {\n",
       "                    // IPython >= 3 moved mimebundle to data attribute of output\n",
       "                    data = data.data;\n",
       "                }\n",
       "                if (data['text/html'] == html_output) {\n",
       "                    return [cell, data, j];\n",
       "                }\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "}\n",
       "\n",
       "// Register the function which deals with the matplotlib target/channel.\n",
       "// The kernel may be null if the page has been refreshed.\n",
       "if (IPython.notebook.kernel != null) {\n",
       "    IPython.notebook.kernel.comm_manager.register_target('matplotlib', mpl.mpl_figure_comm);\n",
       "}\n"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAlgAAAEsCAYAAAAfPc2WAAAgAElEQVR4XuydB1RVx/PHvwqComIDO6iIFRUrih1FJSa2aKKJvcdeY2IHLCT2GhN7TfGnRoyxYkNQURAUEQtYUFFRRARRUOB/9prH34KyPF659+7sOZ4Y39zd2c/OfTNvy2yu9PT0dFAhAkSACBABIkAEiAAR0BmBXBRg6YwlVUQEiAARIAJEgAgQAYkABVhkCESACBABIkAEiAAR0DEBCrB0DJSqIwJEgAgQASJABIgABVhkA0SACBABIkAEiAAR0DEBCrB0DJSqIwJEgAgQASJABIgABVhkA0SACBABIkAEiAAR0DEBCrB0DJSqIwJEgAgQASJABIgABVhkA0SACBABIkAEiAAR0DEBCrB0DJSqIwJEgAgQASJABIgABVhkA0SACBABIkAEiAAR0DEBCrB0DJSqIwJEgAgQASJABIgABVhkA0SACBABIkAEiAAR0DEBCrB0DJSqIwJEgAgQASJABIgABVhkA0SACBABIkAEiAAR0DEBCrB0DJSqIwJEgAgQASJABIgABVhkA0SACBABIkAEiAAR0DEBCrB0DJSqIwJEgAgQASJABIgABVhkA0SACBABIkAEiAAR0DEBCrB0DJSqIwJEgAgQASJABIgABVhkA0SACBABIkAEiAAR0DEBCrB0DJSqIwJEgAgQASJABIgABVhkA0SACBABIkAEiAAR0DEBCrB0DJSqIwJEgAgQASJABIgABVhkA0SACBABIkAEiAAR0DEBCrB0DJSqIwJEgAgQASJABIgABVhkA0SACBABIkAEiAAR0DEBCrB0DJSqIwJEgAgQASJABIgABVhkA0SACBABIkAEiAAR0DEBCrB0DJSqIwJEgAgQASJABIgABVhkA0SACBABIkAEiAAR0DEBCrB0DJSqIwJEgAgQASJABIgABVhkA0SACBABIkAEiAAR0DEBCrB0DJSqIwJEgAgQASJABIgABVhkA0SACBABIkAEiAAR0DEBCrA4gaalpSE6OhoFCxZErly5OJ8iMSJABIgAESACRCA9PR0JCQkoXbo0cufOLQQQCrA4h/nu3buwsbHhlCYxIkAEiAARIAJE4H0Cd+7cQdmyZYUAQwEW5zDHx8ejcOHCYMZhaWnJ+RSJEQEiQASIABEgAs+ePZMmKZ4+fYpChQoJAYQCLM5hZsbBjIIFWhRgcUIjMSJABIgAESACAET0oRRgcZq+iMbBiYbEiAARIAJEgAh8koCIPpQCLM6XQkTj4ERDYkSACBABIkAEKMB6jwAFWJwvBQVYnKBIjAgQASJABIjAewRE9KEUYHG+BiIaBycaEiMCRIAIEAEiQDNYNIOl3VtAAZZ23OgpIkAEiAARIAIi+lCaweK0exGNgxMNiREBIkAEiAARoBksmsHS7i2gAEs7bvTUGwJxz1Nw+kYsQu/FIzUtXa9YiliYYUDT8jA3NdFrO/qs3PfaI/hFPNZnE1LdlYoXQAfH0sibR7ms9A6JGuAmEHAjFkeuxHDLaytYwNwUtkUtYFM0H2yKWMC6oLnsbxgR0YfSDBanhYtoHJxoSCwTAs+TX+PszSc4FfkY/hGxCH/wDOn6jave0WJet1r4ur4ybx5IeZ0GR49DePEq1SC2ZVXAHP0al0OvRuVQ2MLMIG1SI+ok4DTHBzEJyQbvnLlpbtiwgKtIvv/+y4Kv/wKwohawzJvH4Dq936CIPpQCLE6zE9E4ONGQGIDk16k4f/spTrOAKjIWF+48xev3ZqoqlyiABuWLIr+5qd6YBd2OA/vTu1E5zOpcQ2/t6LPiy9HP0H7ZSeQ3M0HPRuX01tSr1DQcuPQA9+NfSm3ky2OC7g1sMLBpBck5USEC2SHwNCkFtT0PS48MblZBbzNK7E6/p0mvcCcuCXeevMD9+BfIalK8UL4878x4lS1q8eb/i+RDmSL5DDLbLaIPpQCL8w0S0Tg40Qgpxpb52HIfm6E6FRGLc7eeIPl12jss2PR9k4pWcK5YTPpTvGBevbPaHXwPY/8KQb1yRbBzWGO9t6ePBnYE3cXE/11AwwpF8ddQZ300kVEnC7L2XozGat+bCL//TPr33LmAz2qWwpBmdnC0KazX9qly9RAIjopDl19OoaRlXpyZ0tpgHWM2HP30hRRssaAr6gkLvJJwJ+4F7j5JQuzzlE/qkisXUKJgXingKvvfkqNmNsypQlGdBYoi+lAKsDhfAxGNgxONEGLsV+O1h4kZS34BN2OR8PL1O31nS01N7IuhcUX2x8oosyDXHyagzWJfWJiZ4JJ7O+Rm0YLCiuc/l7He/yYGNKmAGR2qG0R7Nr5sKfc330icvP7/e79YkDekuR1cqhRXJEuDwKNGJAJ/B9/FuL8uoJFdUfw5RL8/DLKDnG1X0Mx2scCLBWB3/5v9Yv+elJL5Ujzb5xXq3pYCrOzAfk+WAixOeBRgcYJSkVhUbNKbgCoyVlr6e5z47i/BgnlN4Wz3JqBqYm8F++IFdPZlpC3G16lpcJh5UJpNOzaxJSpY5de2KqM912P1aZy58QQLvnJEt3plDa4Hm8la43sDey5EZyzzsrFlyz6dapehDfEGHxFlNLjo8DUsO3Id3zjZwOvLWopQmv2wYDNcmhkv6b/S7FcS8pqaYF2/Bjrrh4g+lAIsTvMR0Tg40ahG7FFCcsaSn3/kY9yNe/FO3/LmyS3toWKzU2ymyqF0IZjIcIao0wo/XLgbj5Xf1sXntUopanzYFz7b4P7s5WvsG90M1UtbGk1/trdlo/8t/B4QhYTkN7OVbJayf5Py6NnQljbEG21k5Nnw6D+CpaB88mdVMbRFRXkqaUStRPShFGBxGpyIxsGJRhVix6/GYOCmwHdSKJjmzoU6toXhzAKqisVQ27awQTaD5hTo5F0X8cfZOxjhUhHft6ua0+oM+jxbumj68zHkMcmFMA83mJnmNmj7mTX27OUr/HX2jrRsqdkQz5Zg2SlN2hBv9OGRjQIdV/jh4t14/Na7Hto5lJSNXnJRREQfSgEWp/WJaBycaFQh1n/DWRy7+gh21vnhWq2EtOyn7xN/+gK35fQtTPcOg0sVa2zo76SvZvRS76GwBxiyJQjVS1li35hmemlD20o/tiG+PdsQ39wOtcrShnht2Sr9OTbzWsv9kDTTeXhcc1QqUVDpXdK5/iL6UAqwOM1IROPgRKN4sZiEl3D2OirNXh2d0AJ21gUU3aeg20/QddVplLA0R8AUV0X1ZYnPNSzxuS7tvWJ7sORYmDNlSVBX+974YEP80BZ2aFmZNsTLcdz0qdPjxGTUn+0DdiIv3NON9ullAltEH0oBFudbJ6JxcKJRvNjakzcw+99waTnw7+FNFN+fxOTXqOl+UEpsGjjNVdo3pJQyZHMgDl1+iBlfVMeAphVkrzbL2cXs5/0N8SzFQ6c6pRWxpCx7yApQMPDWE3T79TTKFM4H/x9bKUBjw6soog+lAIvTzkQ0Dk40ihf7fNlJhEU/kxJzsgSdaiguC47j5uPn2DLQCc0qWSumS01/PiodLvhzSCM0siumGL3ZhvgN/22IZwEuK+z6kn6Ny6O3czlZZNJWDEwFKvq/wDv4fsdFNLW3wtZBDRXYA/2rLKIPpQCL065ENA5ONIoWu/LgGdyWnJQ2VZ+b6qqak2Ejtp3Hv6H3FXWiKf7FK+kEISsXZrYFyz6ttMI2xP95Ngrr/W7hwbM3GeJrlLGUZkbzmBh/w77SeCpF3/kHr2DlsUj0amSL2Z1rKkVtg+opog9VbID16NEjODs7Y+3atWjZsuU7hnL//n3Url0bP//8M/r165fx2aZNmzBr1iywz6tVq4bly5dLdfAUEY2Dh4vSZbz2heM33xto51ACv/Wur/TuZOi/8lgE5h+8ik61S2NpjzqK6NeZG7HosfoMyhbJB78flL3Mwu5T/Dc0Gu57LoMFjlPbV8Pg5naKGAdSMvsEhm8Lwr7QB5j2eTUMakbjnBlBEX2oIgMsf39/9O3bF5GRkTh27Ng7AVZaWhpcXV1x4sQJrFu3LiPAOn78ODp27Ij9+/fDyckJK1aswNy5c3H79m1YWGR975iIxpH9rxllPcE2tTt7HZEuZ1Xb0epjV2LQf+M5VCpeAIfHt1DEwKzzu4lZey+jbfUSWN1HHcHuX+ei8MPOUCmzvs/4FihdOJ8ixoKUzB6Bz5aelK5aWt+vPlpVLZG9hwWRFtGHKi7AYrNQM2bMwLx589CjR48PAix3d3fcuHEDvr6+YH/XzGD16tVLCqRWr16dYc5sFmvSpEno379/liYuonFkCUXhAr7XHqHP+rMobJEHZ6e4yiLnkq6QPnz2Eg3nHpHu1buskFNNE7ZfwM7zdzHWtRLGulbWFQqj1pOWlo6vfzuNwNtxqpslNSpYGTXOTpWy2xPYlTNqOIWsL7Qi+lDFBVgPHjyAlZUVTE1NpWtJ3p7BYn8fNmwYAgMDUaNGjXcCrDp16mDAgAEYNWpUhv107doVNjY2WLJkSZY2JaJxZAlF4QJj/wzG7pBo9HEuB89ONRTem3fVZ1/67Ng4uwZj94gmqK2AS4s1swCre9dDWxUlamT7/D5f5ielAVnXtz5aV6MZDjW9bJofM+xWhyuz3Giv3UcGV0QfqrgA6+2xezvAiomJkZb+du7ciXr16qF8+fLvBFj29vaYMmWKFGRpSu/evWFubi7t43q/JCcng/3RFGYcLBiLj4+HpaXxru9Q0xeTMfvCTno1mO2DF69SFROAZJdX73UBUp6muV1q4tuGttl93KDybM+Sw8wDeJWaDr8fXFC2SNbL9gZVMIeNafb6sWP8bKkwn5lJDmukx+VCQLN3sFwxC5z43kUuaslODwqwZDckn1ZIE2C1aNECbm5uaNeuHcaPHy899H6A5ejoiMGDB2PkyJHvzGDZ2tpi8eLFHzTElhc9PDw++HcKsBRmJB9Rd0fQXUz83wXYWeXHkQktjH5Jsz6oapy6Ek42hUXHS7M8lnlNpROE7N1WU0lKeY02i3xx7+kLDGtZET+4KesKIzWNha77wk6N/rgrFC0qW2PTAGXdnKBrFp+qjwIsQ9LWQVuaAMvOzg6VK1dG3rx535lxYv/fqlUr7N27Fz179pRmnlatWpUhw/ZgTZw4EQMHDvxAG5rB0sEAybiKb9ecwanIWExsWxkjW1WSsabaq+Ydcg9j/gxBXdvC2CXzBKqaPEKN7IrizyF8J3u1J2OcJzXXALE7Ltk1QJXpOhXjDISOW9X8kGE5z9w7Oui4dvVURwGWwsby/T1Yb6v//gzWkSNH0KVLF3h7e6Np06ZYuXIlPD09ERERgaJFi2bZcxGNI0soChVgswgsoSXLdK7G5SjNsFx/mIA2i32lE2yh7u3A9ojItXj8EyYl6hzQpAJmdKguVzVzrNegTYHwCX8Ip/JF8dfQRqqbqcsxIAVWoLl9wKOjA/o2Lq/AHhhGZRF9qGr2YL1vIu8HWOzzrVu3Yvbs2bh79y4cHBywbNkyNGzIl3VXROMwzGtn+FY0OaLUPFvCqLJN1Wxf08tXabI/3dT9t9MIuPkEC79yRNd6ZQ1vFAZq8W5ckrRUyPb+ze9WC1/VtzFQy9SMvgi0XXwC1x4mSsuDbJmQSuYERPShig6wDGnIIhqHIfkaqi12us510QlEPnqOed1q4WuVO7hOK/1x4c5TrPi2Dr6oVdpQmLPVDhuTWh6HkPDyNfaPaYZqpdR9iOTXE5H4af8VFM1vhiPjW6BIfrNs8SJh+RBgaTiqzTiA5Ndp8P3eBbbF1HU4Q5ekRfShFGBxWpCIxsGJRlFiF+8+RccV/jA3zS1dhFwwr/KuY8kO8Mm7QvHH2SgMb1kRk2S6sfrOkyQ0m3cMZia5ccmjnarykWU2Vq9S08Duv2SzHj0a2OCnrrWyM6QkKyMCbLtBk5+OSldthXu6wZSuQ/ro6IjoQynA4nxZRTQOTjSKEnPfE4aNp26ho2NpLPtGGVfI5ATwljO3MX33JbSsYo2N/eV5wulg2AMM3RIEh9KW+Hd0s5x0VzHPnrv1BF/9elrSd8d3zqhfPut9oIrpnECK+kc8Rs+1AbCzzo+jE969sk0gDFxdFdGHUoDFZRqAiMbBiUYxYizXUiOvI3jyPAUb+zdAyyrFFaO7tooG3Y5D11WnULygOc5OddW2Gr0+t/jwNSw9ch1f1SuL+V856rUtOVU+accFbA+8i6olC+KfUU0pQaWcBodTF80PmNZVi2NdvwacT4kpJqIPpQCL09ZFNA5ONIoRO3z5IQZvDoR1QXOc/rGVENP5z5Nfo4b7QenE5LmprlLf5VbYmLCxmdmhOvo3qSA39fSmDwv0Wy88jrikV5jSviqGNK+ot7aoYv0QYHdnsjs0BzWtgGlfqPf0qy7oiehDKcDitBwRjYMTjWLEhm0Nwv5LDzC4WQVM/VycL8NWC47jxuPn2DzACc1leMqJ7WFhe1n+GtIIDe2KKcaedKHo9nN3MGnnRSmVBruUm2V6p6IcAgM3nsORKzGY3bkGejUqpxzFjaCpiD6UAixOQxPRODjRKEIsPukVGszxQUpqmhAn1d4elBG/n8e/F+/jx8+q4rsW8polYePi6HlIUveie1tYqvzQwfsvCzuF1n31aZy7FYe21UtgdZ/6inifSMk3BFotPI4bj55j26CGaGJvRVg+QUBEH0oBFucrIaJxcKJRhNi2gNuY+vclab/LgbHNFaGzrpTU5P2S48b+05Gx+GbNGdgUzYeTk1rpqsuKqufqgwTpVOHrtHSs7VMfrtXpMmglDODr1DQpRQO7P9P/x1Y0+5jFoInoQynA4nyTRTQOTjSKEGMbvdmG76ntq2FwcztF6KwrJY9djUH/DedgX7yAdNGwnArbv8L2sbRzKIHfeos7e+O1Pxy/nbghOenD45vDwsxUTsNEumRC4Hbsc7SYf1xK+cJSNOSW8U0JchhAEX0oBViclieicXCikb3YrcfP0XLBcbDvvzOTW6O45f/fWSl75XWgYMyzl3Cae0Tqf5iHG/KZmeigVt1UMX57CHadv4dxrpUxxlWdd0LykHr7Mmi2jMuWc6nIm8DxqzHot+EcqpQoiIPjxJoV12ZkRPShFGBxWoqIxsGJRvZiiw5fw7Ij14W97Z5lSmf7zx4npmD3iCaobVNYNmPmtsQXVx4kYE2f+mgj+NKYz+WHGLQ5EOwyaJYPrErJgrIZJ1LkQwIb/W/C/R+afeW1DRF9KAVYnNYhonFwopG1GAsums8/hjtPXmBpj9roVLuMrPXVl3K91wXg5PXHmNulJr5taKuvZrJVb/LrVDjMOCjtPaI9LG/QaVJWNChfBH8NcaZlp2xZlGGFNUmLh7aww+TPqhm2cQW2JqIPpQCL01BFNA5ONLIW02TMLmBuKuWBktPymCHBafb49Gpki9mdaxqy6Y+2delePL5Y7odC+fIgZEYb5MqVSxZ6GVMJlq7CdeEJ6TJoEe7KNCbrnLbdd/1ZnLj2CD99WRM9nOTxoyWnfdLn8yL6UAqwOC1KROPgRCNrscm7LuKPs3eEyxL+/qB4h9zDmD9DUNe2MHYNbyKLMdseeAeTdlyEs10x/DGkkSx0koMSv52IhNf+KyhikUe6foUug5bDqHyoQ4v5x3A7Ngl/DmmERoLlb9NmRET0oRRgcVqKiMbBiUa2Yi9fpUp7jxJevsYfgxvBuaJYSSzfHpiImAS4LvJFvjwm0oXKJjI48aRZYhnYtAKmUxbsjOFil0F/scwPVx8moHt9G/zcjS6DltuXDLt2q+r0/UhLBwKmtEYJwQ7OaDMeIvpQCrA4LUVE4+BEI1uxvRejMfL3YOno+8lJLkLvZ0lNS4fDzAN4+SoNRya0QEXrAkYft69/O42zN59g4VeO6FqvrNH1kZMCgbeeoBtdBi2nIXlHl8hHiWi98ISUgT/Mox0tb3OMlIg+lAIsDsNgIiIaByca2YoN2HgOR6/EYKSLPSa2qyJbPQ2lWOeV/gi58xTLv6mDDo6lDdVspu2wwwe13A8hIfk1DoxthqolLY2qjxwb/2HHRfwVeEdKA7B3NF0GLacxOhL+EAM3BaJ6KUvsG9NMTqrJVhcRfSgFWJzmKKJxcKKRpdijhGQ08joCNnMjlxkbY4Oa8ncofg+IwrCWFfGDm3HzLN15koRm847BzCQ3wjzbIY9JbmPjkV37cc9TpKtY2GXQkz+riqEyu+ZIdsAMqNDakzcw+99wfF6zFFb2rGvAlpXblIg+lAIsTnsV0Tg40chSbL3fTXjuvQxHm8LwHiGPTd3GBrX1zG1M231JFvnADlx6gO+2BqFGGUvsHUUzAB+zjf8F3sH3Oy5Ke+d8JtBl0MZ+hzTtT9sdiq1nojDCpSK+b2fcHytyYZKVHiL6UAqwsrKK/z4X0Tg40chS7IvlJ3Hp3jN4dnJAH+fystTR0Eqdj4rDl7+cgnVBcyllhTGLJvnr1/XLYl43R2OqIuu22VJq99/O4OytJ1IiVpaQlYrxCfRaGwC/iMeY360WvqpvY3yFFKCBiD6UAixOwxTRODjRyE6MXZ7bbokv8pjkQsAUVxTNbyY7HY2hELuOxWHmQaSnA2entkbxgsa7MmjQpkD4hD+Ee4fq6NekgjFwKKbNaw8T0H7pm8ugKeO9PIatyU9HwXKW7fjOGfXLF5WHUjLXQkQfSgEWp1GKaBycaGQnpkmq2bZ6CaymX/zvjA/b03Pj0XNsGuAkLRUaq2gc1PahznCqQA4qq3H4af8V/Hoiki6DzgqUAT5n6V+qzTgg/VAJmuaKYgXMDdCq8psQ0YdSgMVptyIaBycaWYmxTe2NfzqCh8+S8WuvenCrUVJW+hlbmZG/n8fei/elTe5ss7sxytOkFNT2PCw1fdG9LSzz5jGGGopq8+3LoOlqFuMOHZtRbLvYFwXzmuLizLaUooFzOET0oYoNsB49egRnZ2esXbsWLVu2lIZ4586dmDVrFm7cuIGiRYuif//+mD59OnLnfnNCadOmTdLn9+/fR7Vq1bB8+XKpDp4ionHwcJGbzMnrj9B73Vnp+hW2DGZuaiI3FY2qzy/HIzDvwFUpTQNL12CMciryMb5dEwDbohbwneRiDBUU2aYmNQBdBm3c4TsY9gBDtwShVtlC2DOyqXGVUVDrIvpQRQZY/v7+6Nu3LyIjI3Hs2DEpwAoKCkKzZs2wfft2tG/fHlevXpX+O3LkSEyYMAHHjx9Hx44dsX//fjg5OWHFihWYO3cubt++DQsLiyzNVETjyBKKDAXG/xWCXcH3IKc79+SE6fjVGPTbcA4VrfPjyIQ3P0wMXTRH3N0cSuLX3vUM3byi2xuyORCHLj8EXQZtvGHUXGXU0bE0lhnpR4rxeq99yyL6UMUFWGwWasaMGZg3bx569OiREWCx2SsWeC1atCjDAsaPHy8FYd7e3ujVq5cUSK1evTrjczaLNWnSJGmmK6sionFkxURunz9Pfo36s32ki3J3DW+MurZF5Kai0fWJSXgJpzlHwO5VZhmoLcxMDa7T+O0h2HX+Hsa3qYzRrSsZvH0lN8g2VrdZdAJJKamY17UWvm5AJ9gMPZ4/7ryIP8/dkWyX2TAVPgIi+lDFBVgPHjyAlZUVTE1NpbVvzQzW+0P84sUL1K5dGz179pQCsjp16mDAgAEYNWpUhmjXrl1hY2ODJUuWZGkhIhpHllBkJrAz6C4m/O8CKljlx9EJLWhvxEfGhwWhjxOT8ffwxqhjhCDUbYkvrjxIwNo+9eFavYTMrEj+6qzxvYE5+8Kly6DZLKS+T8myVBHJr9OQNw8ttzPr6P7baQTcfILF3R3RpQ5d8cT7xojoQxUXYL09mB8LsBISEtCtWzc8fPgQJ0+eRMGCBWFvb48pU6ZIQZam9O7dG+bm5tI+rvdLcnIy2B9NYcbBgrH4+HhYWtK1HrwvlSHleq49A/+IWExoUxmjaGbko+j7rD8L32uPMKdLDfRsWM6QQwR2AqvGzINSyoFTP7ZC6cL5DNq+Ghpjl0F3WO4nBam6yiPGZn/vxCUhKjYJd+JegGXal/7Esf++wMvXqfiZzZhRzic0nOsjHaLZPaIJatsUVoNJGaQPFGAZBLPuGskswGJ7r9jMVIkSJbBjxw4UKfJmmcjR0RGDBw+W9mRpCpOztbXF4sWLP1DK3d0dHh4eH/w7BVi6Gz9d1hT99AWa/HxUOjrNLna2KZr1vjpdtq+kujRH/ns2tMWcLjUNqvqle/H4YrkfClvkQfD0NjTLqCX9oNtP0HXVaenp/33njAZZ5GJiQRl7R6KkwOnFf4GTJoh6gSfPU7LUpI5tYfw9XOxbEdhpzuozDkqsLsxoi0IWdAI2S8P5T4ACLF5SMpF7P8Dat28fvvnmGymQ+umnn6RlRE1hS4Vs5mnVqlUZ/8b2YE2cOBEDBw78oEc0gyWTQeZUQ3M6juVUYrmVqHycwJ4L0Rj9RzCM4TC3n7uDSTsvonHFYvh9cCMaphwQ0OwFqlyigHTdEEt/Ic1CaYKoJ2/+fjfuBe7Hv0Ba+qcbY0GvTREL6XRn2aL5pL+zHyr5zUzQ7dfT0r49dgOAlcB5ny5HP0P7ZSel5dngGW1zMHriPUoBlsLG/O0A68yZM2jRooUUQL29DKjp0pEjR9ClSxdpw3vTpk2xcuVKeHp6IiIiQkrpkFUR0TiyYiKXz9keEZaX5npMIn7uWhPdG9jKRTVZ6hERkwjXRSek++0uebSDSe5cBtPTfU8YNp66hUFNK2DaF9UN1q4aG2KXQbdedEKafWKpG9iy66eKuWluKWBiAZRNkXzS38v+F1DZFM2Hgp/IR/b5spMIi36GBV85ols9cfcd7Qu9j+Hbzhvlx4nSbVhEH6qaPVgsBcPevXs/SLnAUjew1AysbN26FbNnz8bdu3fh4OCAZcuWoWHDhlx2K6JxcIGRgVDo3Xh0WOEH5kDOTXOlxJVZjAlLxuow8wBevkrDkQktUNG6gMFG8etfT0v36i362hFf1v+wJwAAACAASURBVBXXUesK+O7gexj7V4hUHYuTSxVigVO+/4KoNzNQ7P/Zf60LmGu9JLvo0FUsOxqBz2uWwsqedXWlvuLqWXksAvMPXsWXdcpgUffaitPfmAqL6EMVHWAZ0lhENA5D8s1JW5pZEWMmz8yJ/sZ4tvNKf4TceSolG2XcDFHS0tJRy+MQEpNf48DYZqhakg6L6IJ7REwC8pjklg4MsP/qowRHxaHLL6dQ0NwUQdPbwMxUP+3oQ3dd1jnxfxewg51WpoM02cYqog+lAIvTTEQ0Dk40RhVjm3cbzj0iLZNs6NcALlWLG1UfpTQ+9e9QbAuIwnctKuLHz6oaRG12Qq35/GOSc2Y5uPQVDBikM4I1woLjBnN8EPs8Bb8PaojG9laCEXjT3W6rTiHwdpxBf5ioBbSIPpQCLE7rFdE4ONEYVczn8kMM2hwIqwJmODO5NUz19AveqJ3UQ+PbAm5j6t+X0LyyNTYPcNJDCx9WeeDSfXy39TxqlimEf0bRFSMGga7DRjSzNwObVsB0QffP1Z99GI8TU7B3VFPUKFNIh3TVX5WIPpQCLE67FtE4ONEYVWz4tiDsC30Akb/0tRkAzZIPOxEWOM1Vmyqy/YxmH0/3+jb4uVutbD9PDxiXwP7Q+xi27TzsWCLfica5ZsmYBBJevkJN90OSCuxwSAFzw9+CYMz+57RtEX0oBVicViOicXCiMZpYfNIrNJjrg5TXafh3dFM4lKZflLyD8SIlVdrozg6esUuxixfMy/uo1nKDNp2DT3gMPDo6oG/j8lrXQw8ahwALMOrOOoxXqenSTQl2BjwcYZwev9uq5jCNIX+UyKHfutJBRB9KARan9ejDONj+IdqHwjkAmYj9HhCFKX+HomrJgtg/ppnWJ6S010DZT7ZeeByRj55jY/8GaFlF/3vXGnsdQXT8S67EmMomq17tNbclTPu8GgY1s1NvRzPpmXfIPYz5M0S6aPt/3zUWqu+66Kw+fKgu9NJnHRRgcdLVh3GM/P28dKJqSDM7OFcsRgEC51hoxDQbTqe0r4ohzStm82kSH/VHMP65EI1JblUwvKW9XoGwnE11Zh2W2gh1b/vJnEt6VYQqzxGBdX43MWvvZSETxS71uY7FPtd0dj1RjgZCgQ/rw4fKHQMFWJwjpGvjYKfe2J1WbLqdlRplLDG4mZ2UZ4Y2amc9KLdjn6PF/ONS7p/Tk1ujhKX+l7iy1kpZEquOR+LnA1fwRa1SWPGtfnMbnYp4jG/XBqBcMQuc+N5FWaBI2wwCtx4/R8sFx6XEpudntBEq59y4v0Lwd/A9g/wgUaPJ6dqHKoERBVico6QP42BBAvtFuD3wjpT0kZUyhfNhQNMK6N7AhjZRfmJsFh++hqVHrqNZJStsGciXLJZzqIURO3HtEfquPws76/w4OkG/m5bXnryB2f+G47MaJbGqVz1hGKuxo60WHseNR8/xS8+6aF+zlBq7mGmfNLnjVvWsi88E6reuBlgfPlRXuumrHgqwOMnq0zjY8smWM7ex6dQtKc8MK5Z5TdGzUTn0a1yeZmfeGyN2NQ6bvWL3rC3pXhud65ThHEUSe5vAo4RkKbcRu2OO5aWyMNPfqajxf4VgV/A9StCoAhOcvfcy1vrdRNe6ZbHwa0cV9IivC7U9D+Fp0itpv2e1UpQkl4/a/0vp04dmVxdDyVOAxUnaEMbx8lUqdp2/B/Zr/8bj55JmeUxyoXPtMhjc3A6VSxTk1FbdYoG3nkiXz7JLaNnVOPoMDNRNElKAxQKtXcMbo65tEb11t91iX1x9mIB1feujdbUSemuHKtY/gVORj/HtmgAUy2+Gs1NdDXqXpf57l3kLb+8hDPd0Qz4zE2Opoth2DeFD5QaHAizOETGkcbCsyUeuxGC1byTO3YrL0LBlFWsMaW4HZzuxN8RP3hWKP85GSZfOsstnqWhPgC0RsqXC2Z1roFejctpX9Ikn2Q8Hh5kHwe5APD25Fdh9eVSUS4Cdfq7reRgJya/1HpjLhdL5qDh8+csplLTMizNTWstFLUXpYUgfKhcwFGBxjoSxjIO92Gt8b+BA2AOkv9kPL/SGeOas2axLwsvXQl/ZwWm2WYqxTe5ss/u3DW0xt0vNLOW1EdDkDypikQfnp7eh07LaQJTZMyN+P49/L97HqFb2mNC2isy00706u87fxfjtF6Qft38MaaT7BgSo0Vg+1JhoKcDipG9s46AN8W8Gin2psy/30oXywu+HVsjNjhFS0ZoAS9PA0jXUtimM3SOaaF3Ppx7861wUftgZiib2xbBtEDknvUA2cKU72YXH/7uA6qUssW9MMwO3bvjmNLcQfONkC68v9fNDxPC9MmyLxvahhu3tm9YowOKkLhfjYOkdtgq8IX7gxnPS8unwlhUxyc0wlxRzmogixSIfJaL1whPIm4ddwOyml/00M70vYdPp2xjcrAKmfl5dkZxI6XcJxCYmo/4cH2lWXYRlX03OOMq5p/2bIBcfqn0Psv8kBViczORmHCJuiH+cmIyGc49Ie3l8xjeHfXHa9M9pvh8VYyxrzDyIF69S4TO+BeyLF8hplR88/9Wvp6S9hIu7O6JLnbI6r58qNA6BL3/xx/mop5jTpQZ6NtTP/j3j9OzDVjss90PovXis7l0PbR1KykUtRekhNx9qCHgUYHFSlqtxsA3xPuEPsebkDdVviN/gfxMe/1yGY9lC8B7ZlHPkSCwrAl1+8Udw1FMs+6YOOjqWzko8W58z+6zlcUi6seDg2OaoUpKC4mwBlLHwymMRmH/wKlpXLY51/RrIWNOcqcbSwrBLnpkNHx7XHJXoNLdWQOXqQ7XqDOdDFGBxglKCcXxsQ3y3umVRzio/bItaSIlM8+ZR5hFjza9I9w7V0a9JBc6RI7GsCEz9OxTbAqLwXYuK+PEz3S67ajLum5myJch2dPdmVoOhoM/D7z/DZ0tPSsvLITPaKvZ7JSvkb+eLYykalPr9mVU/9f25EnyorhlQgMVJVEnGwa6zYBni/xf0/xni3+5mCUtz2BSxkAKuskUtYFMkn/R3m6IWUlJTExluHL/2MAFtF/tKV3QETGmNYgXMOUeOxLIioLk0u3lla2we4JSVeLY+3x96H8O2nUetsoWwh2Yds8VO7sJsZqfxT0dxP/4lNvRrAJeq+r8w3BhMzt16gq9+PY2yRfJJB2uoaEdAST5Uux5++BQFWJwklWgcbEM8yxfFln/uxiVJmc+TUlI/2WOW2JTNcrFgS/rzXyBmUzSf9PfCFnmMcsz+p/1X8OuJSLhWK4G1fetzjhqJ8RAIufMU7BoQqwLmCJzmyvMIt8zCQ1ex/GgEejSwwU9da3E/R4LKIKCZ/ezVyBazO6vzdB27ymzSjot0LVcOTVKJPjSHXaZThLwA1WAc7BcnC7ruxL3AnSdvAi4WeN158gJ34pJwL+4FXqf9l2zrI2AKmJv+F3j9F4Sx2a9iFiiYNw8vSq3kRv0ejAfPXoLuAdMK3ycfepHCEoEeABv6s1Nao7gOL87WnPr07OSAPs7lda881WhUAkevPMSAjYFS2hT/H1sZ5ceXvgHMO3AFvxyPRO9G5TCrcw19N6fa+tXgQ7M7ODSDxUlMBON4nZomBTGagIsFYdKf/wKymIRkTlr6EWP3M7KrccxNlbmHTD9UdFOr66ITiIhJxIb+DeBSRXdLPc5eR6QlpB3fOaN++aK6UZZqkQ0BdpqZ3dHHLqs/MLYZqpZU3x19w7cFYV/oA0z/ojoGNqW9n9oanwg+9H02FGBxWouIxvE+GvZl+vaMV1QsC77ezICxz/RZ2IXEA5pWUP1xcH0y/FTdo/8Ixp4L0fi+XRWMcLHXiRpstrTurMNSXZc82oHNflJRH4EBG8/h6JUYndqOnCi5LfHFlQcJWN+vPlpVpXs0tR0bEX0oBVic1iKicXCiITEVEGD729g+t89rlcLKb+vqpEf+EY/Rc20AyhezwPHvXXRSJ1UiPwIs8fG03ZdQr1wR7BzWWH4K5kAjtq2i+ow3eeKOTWyJClb5c1Cb2I+K6EMVG2A9evQIzs7OWLt2LVq2bClZbkBAAEaPHo2wsDBYW1tj2rRpGDhwYIZVb9q0CbNmzcL9+/dRrVo1LF++XKqDp4hoHDxcSEYdBHyvPUKf9WdhZ5UfRye+eZ9yWtgdmnP2haN9zZL4pWe9nFZHz8uUQPTTF9JpQjbLHDStDYrmN5OpptlX60H8SzTyOiKdrL4yy43SjGQfYcYTIvpQRQZY/v7+6Nu3LyIjI3Hs2DEpwIqLi4O9vT08PT0xdOhQ+Pr6onPnzvDx8YGTkxOOHz+Ojh07Yv/+/dL/r1ixAnPnzsXt27dhYWGRpdmIaBxZQiEB1RB4O9fPJfd2yK+D5bxxf4Xg7+B7mNi2Mka2qqQaVtSRDwmwfFgsL9airx3xZV31ZOs/HRmLb9acoVlYHRi9iD5UcQEWm4WaMWMG5s2bhx49emQEWGwmi/3btWvXMkxh2LBhSEpKAnumV69eUiC1evXqjM/ZLNakSZPQv3//LM1HROPIEgoJqIqA0xwfsIMMbJmHLffktLRb7IurD2nvSk45KuH5BQevYsWxCHxRqxRW6GiJWQ79ZmluJu8KRcsq1tjYX7c54uTQP0PqIKIPVVyA9eDBA1hZWcHU1FQ6EqyZwRo3bhyioqKwc+fODJthS4Dr1q1DSEgI6tSpgwEDBmDUqFEZn3ft2hU2NjZYsmTJB3aWnJwM9kdTmHEw2fj4eFhaqu+kjCFfNGpLngT6bTiL41cfSUfR2ZH0nBR26MFh5kHp3sgzk1ujZKG8OamOnpU5gaDbcei66hQK5jXF+eltVLOUNndfOFb73kC/xuXh3tFB5qMgb/UowJL3+Hyg3dsB1qBBg5CSkoLNmzdnyLHgysvLCxEREdLy4ZQpU6QgS1N69+4Nc3NzaR/X+8Xd3R0eHh4f/DsFWAozElKXm4Am3883Trbw+jJnSSMv3n2Kjiv8pf04QdNcVZkfiRusAIIskG4wx0fKs/fH4EZwrlhMFb0evDkQhy8/hEdHB/RtTHnccjKoFGDlhJ4Rnn07wBozZgzu3buHHTt2vDODtX79egQHB8PR0RGDBw/GyJEj35nBsrW1xeLFi2kGywjjR03Ki8Dei9EY+XswHG0Kw3tEkxwp9+fZKPy4KxRN7a2wdVDDHNVFDyuDwPjtIdh1/h4GN6uAqZ9XV4bSWWjZZtEJXI9JlK6QYldJUdGeAAVY2rMzypNvB1hr1qzBokWLEB4enqEL24OVmJiILVu2oGfPntLS3qpVqzI+Z3uwJk6c+M5Jw491RETjMMqgUqNGI3DjUSJaLTwB8/8uZjY1ya21LjO8L2Hz6dsY0twOU9pX07oeelA5BP69eB8jfj8PO+v8ODpBNydRjdn7tLR0VJ1xACmv03Bykot0gwUV7QmI6EMVtwfr7eF9O8CKjY2VlgFnzpyJESNGwM/PD506dYK3tzdcXFxw5MgRdOnSRfr/pk2bYuXKldKJQ7Z8WLRo1hmmRTQO7V8lelKJBNgyT033g9J9lT7jm8O+eEGtu9Ft1SkE3o7Dku610blOGa3roQeVQ+DZy1eo63lYum7r+MSWKK/wnFEsqXLTn4/BzCQ3wme5SakaqGhPQEQfqpoAiw17YGAg2FJhaGiolAdr+vTp6NevX4ZFbN26FbNnz8bdu3fh4OCAZcuWoWFDvuULEY1D+1eJnlQqgS9/8cf5qKdY2qM2OtXWLjBiv/xZoPY8JRWHxjVH5RLaB2pK5Siq3t+sPoPTN2Ix44vq0s0LSi5+1x+j17oAVLTOjyMqmJEz9liI6EMVHWAZ0mBENA5D8qW25EFg2u5QbD0ThaEt7DD5M+2W9m49fo6WC47DzDQ3Lnu0Q06WGuVBhbTgJbD25A3M/jdcFXvvtpy+heneYXCtVhxr+zbgRUByHyEgog+lAIvzdRDRODjRkJiKCPweEIUpf4eiWSUrbBnIN7v7fvf3hd7H8G3nUatsIewZ2VRFdKgrWRHQ7OPLY5ILwTPaKvr+Sc9/LmO9/00MaloB075Qx6b9rMZPn5+L6EMpwOK0KBGNgxMNiamIwIU7T9FppT+K5TdDoJbpFTRJJ79xsoHXl7VURIe6wkPAZcFx3Hz8HL/2qgu3GqV4HpGljOYS6zldatAl8zoYIRF9KAVYnIYjonFwoiExFRFgCUKrzziAtHTg7JTWKG6Z/QShGsc0q5MDejtT7iAVmQdXVzQzP1/VK4v5XzlyPSNHoVYLjuPG4+f4fVBDNLa3kqOKitJJRB9KARaniYpoHJxoSExlBDS5fzb0bwCXKsWz3btGc4/gwbOX2DnMGfXKZX1CN9sN0AOyJuAf8Rg91wbAqoAZzk5xRW4Fnr57nZqGqtMPSCciT/3YCqUL55M1cyUoJ6IPpQCL0zJFNA5ONCSmMgJj/gyGd0g0vm9XBSNc7LPVu9jEZNSb7YNcuYBQ93aK3oOTrY6TcAYBljeq7qzDSEx+jd0jmqC2TWHF0bkd+xwt5h+XcsKFe7opMkiUG3QRfSgFWJxWKKJxcKIhMZUR+O1EJLz2X8HnNUthZc+62eqd5mh7Bav8ODZR+ckms9V5Es4gMHxbEPaFPsDoVvYY37aK4sgcuxqD/hvOoUqJgjg4rrni9JejwiL6UAqwOC1RROPgRENiKiNw8voj9F53FtoESat9IzF3n3bBmcowCt2dHUF3MfF/F1CjjCX2jmqmOBYb/G/C45/LaOdQAr/1rq84/eWosIg+lAIsTksU0Tg40ZCYygg8TkxGfS2X+cb+GYzdWi4vqgyj0N1hNsQuf05PB85Mbo2ShbJ/WMKYAGd6X8Km07fxXYuK+PGzqsZURTVti+hDKcDiNF8RjYMTDYmpkEDDuT54+Cw52xvV2y4+gWsPE7GhXwO4VM3+BnkVohS2S51X+iPkzlN4fVkT3zjZKopDn/Vn4XvtEX7uWhPdGyhLd7mCFtGHUoDFaY0iGgcnGhJTIYH+G87i2NVHyE6qBZbiwWHmQbA7DQOmtEYJLVI8qBClsF1afuQ6Fh6+BtdqJbC2r7KW2ZrPO4aoJ0n4a0gjNLQrJuwY6rLjIvpQCrA4LUhE4+BEQ2IqJDD/4BWsPBaJ7CQL1UWSUhWiFLZLYdHx+HyZH/LlMUHwjDbIm8dEESzYKciq0/fnKBecIjpqYCVF9KEUYHEamYjGwYmGxFRI4N+L9zHi9/NwLFsI3pzX3fxxNgqTd+Xsmh0VohS2S+np6XD2OirlRNvYvwFaapFTzRjwImIS4broBPKbmeCSRzvkYjlHqOSYgIg+lAIsTrMR0Tg40ZCYCgmwq07YlScsD1AY54XN03dfwpYztzG0uR0mt9fuomgVohS6SyzgZoF3H+dy8OxUQxEsfC4/xKDNgXAobYl/RyvvBKRcIYvoQynA4rRGEY2DEw2JqZBAWlo6argfRFJKKg6Pa45KJQpm2cuuq04h6HYclvaojU61y2QpTwLqJ6AJVsoUzge/H1wUMRu09uQNzP43HJ/XKoWV32YvD5z6R1T7HoroQynA4rQXEY2DEw2JqZRAdgImbQIylWKjbr1F4EVKKmp7HkLy6zQcHNscVUpmHagbG+DUv0OxLSAKI13sMbGd8pKkGpvfx9oX0YdSgMVpjSIaBycaElMpgews+WmzpKhSbNSt9whoTqT+4FYVw1pWlD2fnmvPwD8iFgu+ckS3emVlr69SFBTRh1KAxWmdIhoHJxoSUymB7Gxa12ZTvEqxUbfeI7Dl9C1M9w5Dg/JF8L/vGsueT2OvI4iOp8vKdT1QIvpQCrA4rUhE4+BEQ2IqJZCdtAvapHVQKTbq1nsE7sYloenPx5A7FxA0rQ2K5DeTLSOWy63q9AOSfkHTXFGsgLlsdVWaYiL6UAqwOK1UROPgRENiKiWQncShGYlJO9dA70blVEqEuqUtAbclvrjyIAFLutdG5zryPQBx9UEC2i3xhWVeU1yY2VYRm/K1HRNDPyeiD6UAi9PKRDQOTjQkpmICvFff/P/VOo1Rr1wRFROhrmlDYN6BK/jleCQ6OJbG8m/qaFOFQZ45cOkBvtsalK38bwZRTAWNiOhDKcDiNFwRjYMTDYmpmADP5c1vXw59yb0d8pubqpgIdU0bAoG3nqDbr6elmaHz09vA1CS3NtXo/ZlfT0Tip/1X0Kl2aSztId9AUO8g9NCAiD6UAixOQxLRODjRkJiKCaz2jcTcfVfwec1SWNkz85xAJ68/Qu91Z2FnlR9HJ7ZUMQ3qmrYE2P2U9WYfxtOkV7K+3+/HnRfx57k7GNO6Esa1qaxtd+m5TAiI6ENVFWCdP38eY8eOxcWLF5EvXz58/fXXmDdvHszNzREQEIDRo0cjLCwM1tbWmDZtGgYOHMj9IohoHNxwSFC1BPyuP0avdQGoYJUfxz4SPP12IhJe+69QYkbVWoFuOjburxD8HXxP1pn+v/7tNM7efCL7vWK6GRHD1iKiD1VNgJWWloayZcvixx9/xMiRIxEdHQ1XV1f07NlT+n97e3t4enpi6NCh8PX1RefOneHj4wMnJycuKxPROLjAkJCqCcQmJqPebB+w69hC3duhQCbLf2P+DIZ3SDS+b1cFI1zsVc2DOqc9gX8uRGPUH8GwL14APuNbaF+RHp90muODmIRk7B7RBLVtCuuxJfGqFtGHqibAio2NhZWVFRYvXoxRo0bh/v37aNu2rTRLVahQIWkm69q1axlWPWzYMCQlJWHTpk1cli6icXCBISHVE2g094h0Ye+O75xRv3zRD/rbZtEJXI9JxIb+DeCikAt9VT9oMuxg/ItXqDvrMNhyoe/3LrAtZiErLZ8nv4bDzIOSThdmtEUhizyy0k/pyojoQ1UTYDHjGz9+PJYuXSodrU1NTUWnTp2wa9cuTJgwAVFRUdi5c2eGjS5fvhzr1q1DSEgIl92KaBxcYEhI9QQGbDyHo1di4NnJAX2cy7/TX5bKofqMA0hLB85OaY3ilnlVz4M6qD2B7r+dRsDNJ3DvUB39mlTQviI9PBkWHY/Pl/mhaH4zaSM+Fd0SENGHqibAYkuE06dPl5YJBwwYgIiICHTp0gXdu3eXZrNSUlKwefPmDIthwZWXl5ckl1lJTk4G+6MpzDhsbGwQHx8PS0tL3Voe1UYEZExgwcGrWHEsAj0a2OCnrrXe0TTkzlN0XukPqwJmODfVlfIGyXgc5aCa5tBEs0pW2DKwoRxUytBBcxtBXdvC2DW8iax0U4MyFGApeBTZ7NTUqVNx5cqVjF5s27ZN2tjeq1cv3Lt3Dzt27HhnBmv9+vUIDg7OtNfu7u7w8PD44DMKsBRsJKS6VgT2hd7H8G3nUatsIewZ2fSdOn4PiMKUv0MhR4epVWfpIb0SiIhJhOuiEzAzyY3zM9pkuqdPrwp8ovKVxyIw/+BVfFm3DBZ9XdtYaqi2XQqwFDy0bO/VsmXLcPPmzYxebN++HWPGjJE2ty9atAjh4eEZn7E9WImJidiyZUumvaYZLAUbA6muUwK3Hj9HywXHYWaaG5c92r2Tw2ja7lBsPROFoS3sMPmzajptlypTH4H09HTJlm7HJuHXXvXgVqOkbDo5YfsF7Dx/FxPaVMao1pVko5daFKEAS8EjefnyZdStWxczZszADz/8gNu3b0snBdu1ayedLGSnCGfOnIkRI0bAz89P2p/l7e0NFxcXrl6LaBxcYEhI9QTS0tJR0/0gnqek4tC45qhcomBGn7/8xR/no55iaY/a6FRbvlegqH6QFNRB9z1h2HjqFrrXt8HP3d5dcjZmN7quOoWg23FSpnmWcZ6KbgmI6ENVsweLmQJLu8DyW7FlQnZykC0NsqDKzMwMgYGB0mxWaGiolAeL7dfq168ftwWJaBzccEhQ9QS6rTqFwNtx7+QHYqfBWOCVlJIKn/HNYV/8/wMv1QOhDmpNQJOY1rqgOQImt0Zudgu0DEq9WYcR+zwFe0c1RY0yhWSgkbpUENGHqirA0qc5imgc+uRJdSuLwAzvS9h8+jaGNLfDlPZvlgJvPEpEq4UnkDdPboR5uMFEJo5SWWTF0zb5dSrqeh6WZkT3jGyCWmWNn2/q2ctXqOV+SBqMSx6Z53sTb6R022MRfSgFWJw2JKJxcKIhMQEI/Hk2Cj/uCkVTeytsHfTm9Nfei9EY+XswHG0Kw3sEnboSwAx01sXvtgThQNgD2VxJc/HuU3Rc4Q82q8ZOw1LRPQERfSgFWJx2JKJxcKIhMQEIaBwQyxEUNO1NOoZ5B67gl+OR+MbJFl5f1hSAAnVRVwS2B97BpB0XMz2Zqqs2slOPd8g9jPkzBE7li2L7d87ZeZRkOQmI6EMpwCLj4CRAYiITYAlFWZZrtu/qzOTWKFkoL/ptOIvjVx9hVuca6N2onMh4qO/ZJBCT8BJOc45IT8khQe0Sn2tY4nMdX9cvi3ndHLPZGxLnIUABFg8lQWVENA5Bh5q6/REC7Rb74urDBKzvVx+tqpaA5t62XcMbo65tEeJGBLJFoNMKP1y4G4+fu9ZE9wa22XpW18Jj/wzG7pBoTHKrguEt6T5NXfNl9YnoQ2kGi9OSRDQOTjQkJgiBcX+F4O/ge5jYtrLkEBvMeXMJdJhHO1iYmQpCgbqpKwJLfa5jsc81tK1eAqv71NdVtVrV02mlPy7ceYpfe9WFW41SWtVBD32agIg+lAIszrdCROPgRENighBY43sDc/aFo33NkujRwBZ91p+FnXV+HJ3QUhAC1E1dErh0Lx5fLPeDhZkJgme0gbmpiS6rz1Zdjh6HwC6jPjC2GaqWpKvQsgWPU1hEH0oBFhkHJwESE52Af8Rj9FwbgPLFLNDDyRY/7b+CL2qVwopv64qOhvqvBQGW1b2R1xE8fJaMzQOc0LyytRa15PyRuOcpqDPrsFRRuKcb8pkZL9DLeW/kWwMFWPIdG6NrJqJxGB06KSArAk+ep6Duf46oddXiMg+lMwAAIABJREFUOHIlhvasyGqElKfMjzsv4s9zd9CvcXm4d3QwSgdY9naWxb1Uobw4Pbm1UXQQoVERfSjNYHFatojGwYmGxAQi4Ox1BPfjX0pJRdmJwo39G6BlleICEaCu6pLAobAHGLIlCDZF88H3excp/Yehy86gu5jwvwtwtiuGP4Y0MnTzwrQnog+lAIvTvEU0Dk40JCYQgYEbz0kzV5pydmprFC+YVyAC1FVdEnie/Fpankt5nYbD45qj0lv3XOqynU/VtfDQVSw/GoFvG9pibhfK56Yv7iL6UAqwOK1JROPgRENiAhHQOCPWZasC5gicRlmvBRp+vXS17/qzOHHtESZ/VhVDW1TUSxufqnTk7+ex9+J9TG1fDYOb2xm8fVEaFNGHUoDFad0iGgcnGhITiMD+0PsYtu281GO2KZltTqZCBHJCYNOpW5i5JwxOFYpi+1DDZ1H/YvlJXLr3DGv61Eeb6iVy0hV69hMERPShFGBxvhIiGgcnGhITiMDt2OdoMf+41OPvWlTEj59VFaj31FV9ELjzJAnN5h2T9vWxa5gKW5jpo5lM62QnGWu6H0Ji8mv4jG8O++IFDda2aA2J6EMpwOK0chGNgxMNiQlEIC0tHbU83jikZd/UQUfH0gL1nrqqLwJtF5/AtYeJ+OnLmlIKEEMVzZU9uXMB4bPcjJqLy1B9NlY7IvpQCrA4rU1E4+BEQ2KCEWD5r45fjcGfQxoZdLZBMMxCdXfV8Uj8fOAKiuU3g8/4FiiS3zCzWGdvPsHXv51G2SL54PdDK6GYG7qzIvpQCrA4rUxE4+BEQ2JEgAgQgRwRYKcI2V4oNovVrV5ZLPjKMBcubz93B5N2XkSzSlbYMrBhjvpAD3+agIg+lAIszrdCROPgRENiRIAIEIEcE2AJP7v9egrp6cDWgQ3RtJJVjuvMqgI2a8Zmz/o4l4NnpxpZidPnOSAgog+lAIvTYEQ0Dk40JEYEiAAR0AkB9z1h2HjqlpR49ODY5nq/RHzY1iDsv/QAM76ojgFNK+ikD1RJ5gRE9KEUYHG+DSIaBycaEiMCRIAI6IQAOzzRdtEJRMe/xOBmFTD18+o6qfdjlbgt8cWVBwnY0K8BXKrSjQT6hC2iD6UAi9OiRDQOTjQkRgSIABHQGYFjV2LQf+M5sJN9u0c0Qa2yhXVW99sVsROx1WcewMtXaTg2sSUqWOXXSztU6RsCIvpQCrA4rV9E4+BEQ2JEgAgQAZ0SGPNnMLxDolGtlCX2jGyCPCa5dVo/q+x+/As4ex2V8m9dmeWmlzZ0rrSCKxTRh1KAxWmwIhoHJxoSIwJEgAjolEBsYjJcF51AXNIrTHKrguEt7XVaP6vsVORjfLsmQJq5YjNYVPRLQEQfqqoA68mTJxg7diz+/fdfpKWloUWLFli1ahVKlSqFgIAAjB49GmFhYbC2tsa0adMwcOBAbosS0Ti44ZAgESACREDHBHadv4vx2y/AzDQ3DoxpBjvrAjpt4feAKEz5OxQuVayxoT9d+aRTuJlUJqIPVVWA5eLigiJFimD9+vUwMTFBv379kJycjC1btsDe3h6enp4YOnQofH190blzZ/j4+MDJie/FEtE49P3CUf1EgAgQgY8RYNfY9Fl/FievP0bDCkXxx+BGyM02ZumozN0XjtW+N9C/SXnM7OCgo1qpmo8RENGHqibACgoKQtOmTfHw4UNYWlpKY8xmtO7fv4/Tp09j3rx5uHbtWsbYDxs2DElJSdi0aRPXGyGicXCBISEiQASIgJ4IsHsK2y72xYtXqfD6sia+0eE1OoM2BcIn/CE8Ozmgj3N5PfWAqtUQENGHqibAYkuBa9euxbfffistCz5//hxubm5YuHAhZs2ahaioKOzcuTPD2pcvX45169YhJCSE6w0Q0Ti4wJAQESACRECPBNaevIHZ/4ajYF5T6RqdEpZ5ddIa2+MVEZOIzQOc0LyytU7qpEo+TkBEH6qaAGvOnDlwd3eX9lXNnz9fmp3q3bs3zMzMULJkSaSkpGDz5s0Zo8+CKy8vL0RERGRqEWxpkf15O/q2sbFBfHx8xgwZvUxEgAgQASKgXwKpaen48hd/XLgbDzeHkvi1d70cN8jqrDb9AFJS03BykgtsilrkuE6q4NMEKMBSsIUsWLAAkydPRkJCAvLmffML59y5c2jYsCH69+8vBUY7dux4ZwaL7dUKDg7OtNcsWPPw8PjgMwqwFGwkpDoRIAKKJBB+/xk6LPfD67R0/NqrLtxqlMpRP9jSY7N5x2Bmkhvhs9ykVA1U9EuAAiz98tVr7fv27UOHDh0QFxeXMcN05swZNG7cGEuXLsUvv/yC8PDwDB3YHqzExERpA3xmhWaw9DpcVDkRIAJEIFsEFhy8ihXHImBd0FxaKiyUL0+2nn9b+OT1R+i97izsixeQ6qKifwIUYOmfsd5aePXqFapXrw5HR0ds3LgRL168QI8ePVCoUCGsWbNGOkU4c+ZMjBgxAn5+fujUqRO8vb3BTh7yFBGNg4cLyRABIkAEDEHg5atUtF96EjceP8c3Tjbw+rKW1s1uOX0L073D4FqtBNb2ra91PfQgPwERfahq9mCxYY6Ojsb48eNx4sQJvHz5Eh07dpRmrwoXLozAwECMGTMGoaGhUh6s6dOnS2kceIuIxsHLhuSIABEgAoYgEHAjFt1Xn5GaYmkbnCsW06pZj3/CsMH/lkHuO9RKQRU+JKIPVVWApU+bFNE49MmT6iYCRIAIaENg6t+h2BYQJWVg3z+mGfLmMcl2Nf03nMWxq48wp0sN9GxYLtvP0wPZJyCiD6UAi9NORDQOTjQkRgSIABEwGIFnL1+hzaITePgsGcNaVsQPblWz3bbLguO4+fg5fh/cEI0rWmX7eXog+wRE9KEUYHHaiYjGwYmGxIgAESACBiVwKOwBhmwJkk7/scugHUoX4m7/dWoaqk4/IJ1IPD25FUoVysf9LAlqT0BEH0oBFqe9iGgcnGhIjAgQASJgcALDtwVhX+gD1CxTCH8PbwxTk9xcOtx6/BwtFxxH3jy5cdnDTafX73ApIKiQiD6UAixOYxfRODjRkBgRIAJEwOAEYhJewnXhCTx7+RpT21fD4OZ2XDocuxKD/hvPoWrJgjgwtjnXMySUcwIi+lAKsDjtRkTj4ERDYkSACBABoxD461wUftgZKs1GHRrbArbFss7Ivt7vJjz3XtZZVnijdFyBjYroQynA4jRUEY2DEw2JEQEiQASMQiA9PR091wbgVGQsmtpbYctAJ+TK9ems7DO8L2Hz6dv4rkVF/PhZ9jfIG6WjKmhURB9KARan4YpoHJxoSIwIEAEiYDQCbE9VuyW+SH6dhvndauGr+jaf1KX3ugCcvP4Y87rWwtcNPi1rtE6psGERfSgFWJyGLKJxcKIhMSJABIiAUQn8eiISP+2/Il2fw66+YdfpfKw0m3cUd568wPahznCqUNSoeovUuIg+lAIsTgsX0Tg40ZAYESACRMCoBFjqhU4r/REW/Qxf1CqFFd/WzVSf5NepqDb9ANLSgbNTW6N4wbxG1VukxkX0oRRgcVq4iMbBiYbEiAARIAJGJ3DpXrwUZKWmpWNtn/pwrV7iA50iYhLgusgX+c1McMmjXZb7tYzeKRUpIKIPpQCL04BFNA5ONCRGBIgAEZAFAa994fjN9wZKFcqLQ+Oao2DePO/odfjyQwzeHAiH0pb4d3QzWegsihIi+lAKsDitW0Tj4ERDYkSACBABWRB4kZIKt6W+uB2bhN6NymFW5xrv6LXG9wbm7AvH57VKYeVHlhFl0REVKiGiD6UAi9OQRTQOTjQkRgSIABGQDYFTEY/x7doASZ8d3zmjfvn/38g+5e9Q/B4QhVGt7DGhbRXZ6CyCIiL6UAqwOC1bROPgRENiRIAIEAFZEZi04wK2B95FRev82DemGcxNTST9vl1zRsqZtfArR3StV1ZWOqtdGRF9KAVYnFYtonFwoiExIkAEiICsCDxNSpE2sz9OTMbo1pUwvk1lSb/GXkcQHf8SO4c1Rr1yRWSls9qVEdGHUoDFadUiGgcnGhIjAkSACMiOwL8X72PE7+eRxyQX9o5qBtuiFqg244Ck5/npbVA0v5nsdFazQiL6UAqwOC1aROPgRENiRIAIEAHZEWDX6AzZEgR2crC2TWHM7VIT7ZedhGVeU1yY2ZZSNBh4xET0oRRgcRqZiMbBiYbEiAARIAKyJPAg/iXaLDqBhOTXaFyxmLT/yrFsIXiPbCpLfdWslIg+lAIsTosW0Tg40ZAYESACREC2BLaeuY1puy9l6Ne5dmks6VFHtvqqVTERfSgFWJzWLKJxcKIhMSJABIiAbAmkpaWj++rTOHcrTtJxrGsljHV9s+mdiuEIiOhDKcDitC8RjYMTDYkRASJABGRNICImEe2XnkRKahqW9qiNTrXLyFpfNSonog+lAIvTkkU0Dk40JEYEiAARkD2BvRejpQ3vXl/WhIWZqez1VZuCIvpQCrA4rVhE4+BEQ2JEgAgQASJABD5JQEQfqsoAKzU1Fa1bt0b58uWxceNGadADAgIwevRohIWFwdraGtOmTcPAgQO5XwkRjYMbDgkSASJABIgAEfgEARF9qCoDrBkzZmDOnDno3bu3FGDFxcXB3t4enp6eGDp0KHx9fdG5c2f4+PjAycmJ66UQ0Ti4wJAQESACRIAIEIEsCIjoQ1UXYB09ehSjRo2Cg4MDLCwspABr7dq1mDdvHq5du5ZhAsOGDUNSUhI2bdrE9WKIaBxcYEiICBABIkAEiAAFWB8QUFWAFRMTg4YNG2L37t1YvHix1FkWYI0bNw5RUVHYuXNnBoDly5dj3bp1CAkJydQskpOTwf5oCguwbGxsEB8fD0tLS3qZiAARIAJEgAgQAU4CIk5SqCbASktLg5ubGzp06CDNYPXr1y8jwBo0aBBSUlKwefPmDFNgwZWXlxciIiIyNQ93d3d4eHh88BkFWJxvE4kRASJABIgAEfiPAAVYCjYFtueKbWTfs2eP1Iu3A6wxY8bg3r172LFjxzszWOvXr0dwcDDNYCl43El1IkAEiAARkD8BCrDkP0Yf1bBq1aqIjo5G7ty5JRm2v4oVtg9r/vz5WLRoEcLDwzOeZ3uwEhMTsWXLFq5ei2gcXGBIiAgQASJABIhAFgRE9KGqWSJ8f2zfnsGKjY2VThHOnDkTI0aMgJ+fHzp16gRvb2+4uLhwvRgiGgcXGBIiAkSACBABIkAB1gcEhAiwWK8DAwPBlgpDQ0OlPFjTp0/PWEbkeTMowOKhRDJEgAgQASJABD4kIKIPVW2ApWsDF9E4dM2Q6iMCRIAIEAExCYjoQynA4rR1EY2DEw2JEQEiQASIABH4JAERfSgFWJwvhYjGwYmGxIgAESACRIAIUID1HgEKsDhfCgqwOEGRGBEgAkSACBCB9wiI6EMpwOJ8DUQ0Dk40JEYEiAARIAJEgGawaAZLu7eAAiztuNFTRIAIEAEiQARE9KE0g8Vp9yIaBycaEiMCRIAIEAEiQDNYNIOl3VtAAZZ23OgpIkAEiAARIAIi+lCaweK0exGNgxMNiREBIkAEiAARoBksmsHS7i2gAEs7bvQUESACRIAIEAERfSjNYHHavYjGwYmGxIgAESACRIAI0AwWzWBp9xZQgKUdN3qKCBABIkAEiICIPpRmsDjtXkTj4ERDYkSACBABIkAEaAaLZrC0ewsowNKOGz1FBIgAESACREBEH0ozWJx2L6JxcKIhMSJABIgAESACNINFM1javQUUYGnHjZ4iAkSACBABIiCiD6UZLE67F9E4ONGQGBEgAkSACBABmsGiGSzt3gIKsLTjRk8RASJABIgAERDRh9IMFqfdi2gcnGhIjAgQASJABIgAzWDRDJZ2bwEFWNpxo6eIABEgAkSACIjoQ2kGi9PuRTQOTjQkRgSIABEgAkSAZrBoBku7t4ACLO240VNEgAgQASJABET0oTSDxWn3IhoHJxoSIwJEgAgQASJAM1hqnsG6cOECJk6ciKCgIJiZmaFt27ZYtGgRrKysEBAQgNGjRyMsLAzW1taYNm0aBg4cyP1KUIDFjYoEiQARIAJEgAi8Q0BEH6qaGawXL16gYsWKGDx4MKZOnYqEhAT06dMHuXPnxubNm2Fvbw9PT08MHToUvr6+6Ny5M3x8fODk5MT1GohoHFxgSIgIEAEiQASIQBYERPShqgmwrl69irFjx2Lv3r0wMTGRhnrPnj3o3bs3Fi5ciHnz5uHatWsZJjBs2DAkJSVh06ZNXC+GiMbBBYaEiAARIAJEgAhQgPUBAdUEWJmNLZvBunv3LhwdHREVFYWdO3dmiC1fvhzr1q1DSEgI14uh8wArPR14lcTVNgkRASJABIgAETA4gTwWQK5cOmlW5z5UJ1rptxJVBljp6emYPn06Vq5cKS0HLl26FCkpKdJSoaaw4MrLywsRERGZEk5OTgb7oynMOGxsbBAfHw9LS8ucj0rKc2Bu6ZzXQzUQASJABIgAEdAHgSnRgFl+ndRMAZZOMBq3EjaI/fv3lza6//PPP6hZsybGjBmDe/fuYceOHe/MYK1fvx7BwcGZKuzu7g4PD48PPqMAy7jjS60TASJABIiAgQhQgJUj0KqawYqMjET79u1ha2uLP/74Qzo9yMqaNWuk04Th4eEZsNgerMTERGzZssU4M1i0RJgjw6WHiQARIAJEQM8EaIkwR4BVE2DFxcWhdu3aaNWqlbS3ip0e1JTY2FjpFOHMmTMxYsQI+Pn5oVOnTvD29oaLiwsXQBGnN7nAkBARIAJEgAgQgSwIiOhDVRNgsRmqCRMmwMLCArne25THZqoCAwOlpcLQ0FApDxbbo9WvXz/ul0JE4+CGQ4JEgAgQASJABD5BQEQfqpoAS9+WLaJx6Jsp1U8EiAARIAJiEBDRh1KAxWnbIhoHJxoSIwJEgAgQASLwSQIi+lAKsDhfChGNgxMNiREBIkAEiAARoADrPQIUYHG+FBRgcYIiMSJABIgAESAC7xEQ0YdSgMX5GohoHJxoSIwIEAEiQASIAM1g0QyWdm8BSzBauHBh3LlzRzeZ3LVTg54iAkSACBABIqA4AprbUJ4+fYpChQopTn9tFKYZLE5q7E5DdlUOFSJABIgAESACREA7AmySomzZsto9rLCnKMDiHLC0tDRER0ejYMGCH+TZ4qziAzFNRC/arBj1W7xZUBpzscacxpvG+32Hx+4ITkhIQOnSpd9JBK6t/1TCcxRgGXGURN3XRf3W0YXhRrTd7DZNYy7WmNN403hn9ztCjfIUYBlxVOlLiL6EjGh+Bm2abJ1s3aAGZ6TGyM7FsvOszIwCrKwI6fFzehnFehlFHW/2Conad+o3veN6dCGyqVpUO89qACjAyoqQHj9PTk6Gl5cXJk+eDHNzcz22JK+qqd9ijTezPhpzscacxpvGW15exzjaUIBlHO7UKhEgAkSACBABIqBiAhRgqXhwqWtEgAgQASJABIiAcQhQgGUc7tQqESACRIAIEAEioGICFGCpeHCpa0SACBABIkAEiIBxCFCApWfuMTExGDJkCI4fPw5TU1P06tULCxYskP7+ftm3bx9++OEH3LhxA7a2tpg/fz6++OILPWuon+ovXLiAiRMnIigoCGZmZmjbti0WLVoEKyurDxr87LPPcOzYsXeY7NixA25ubvpRTo+1/vXXX+jZsyfy5s2b0UqXLl2wZcsWVY/3tm3bMHTo0Hf6mJKSIiXlZRue3y9qGPNHjx7B2dkZa9euRcuWLaUuBgQEYPTo0QgLC4O1tTWmTZuGgQMHftTi5s2bh2XLliEuLg4NGjTAb7/9hipVqujRQnNedWb93rlzJ2bNmiV9dxUtWhT9+/fH9OnTM00oyZI2W1pagiWeZPahKQ8fPkT+/PlzrqCeasis38OGDcP69euRJ0+ejFbZ9xz7zs+sqGG8v/vuO2zduvWd7r148QKurq44ePDgB91W6njrwowowNIFxU/U4eLigjJlymD16tV48OABOnbsiL59++L7779/56nr16+jVq1a+OOPP6SgateuXejXrx/Yv7PnlVTYy1axYkUMHjwYU6dOlbL39unTR/qy/eeffz7oCnNELKBq0aKFkrqZqa4sqIyNjcWGDRs+2Rc1jXdmHb13754UMDCHwn5UvF+UPub+/v7SexwZGSn9OGABFguS7O3t4enpKQWbvr6+6Ny5M3x8fODk5PQBg02bNknvB3NK7H3R/D00NFRnt0Xo+oXKrN/sR1SzZs2wfft2tG/fHlevXpX+O3LkSEyYMOEDFS5duoR69epJ3wvsx5cSSmb9ZnrXr18fo0aNkmwhq6KW8X6/n4cOHcI333wj2buDg4MqxjurseT9nAIsXlJayEVERKBSpUpgzoZdD8AKm+GYNGkSbt++/U6N7Jfu2bNnwYxVU9ivfPbF7OHhoUXrxnuEfcGOHTsWe/fuhYmJiaTInj170Lt3b7BLs98uN2/elJwSuwCUXUOk9MKCxK+//hojRoz4ZFfUNN7vd5TNTLRu3VoKGtasWfMBB6WPOXOUM2bMkILHHj16ZARYbCaL/du1a9cy+sxmOJKSksCeeb80bdpUCkSmTJkiffTq1SsUK1YM3t7eYD/M5FY+1m82e8UCEDZzoynjx4+Xgk/Wl/cL+/Hxyy+/4Ny5c3LrYqb6fKzfbGaWzcSdP38+08BCreP9dr8eP36MatWqYcmSJdLMfWZFaeOtS6OkAEuXNN+ri325DBgwQJrR0BT265TNVLFfu4ULF874d7aMZGdnh4ULF2b8G/v1x76kdu/erUctDVM1m8FiF2YfPXr0nQbZr142nd64cWPpC7dEiRJgX86Mm9IKmwpnt8SzX/NsiSg1NVVyoD///DOKFCnyTnfUPN5sOZTN5LEfGJkFzUofczYTzZa62TI/W+LSzGCNGzcOUVFRYAGHpixfvhzr1q1DSEjIB+bMbGLz5s3o0KFDxmdsZof9EGE/UORWPtbv9/VkM9i1a9eWHC4LRN8vw4cPl2Y78uXLh1u3bkkO+qeffpK+A+RYPtZv9oO4UaNGYO+yn5+f9O6z5WC2OsFm698vahxv1l92R+/+/fv/r73zV6keCOJoavEBLASxERFFrCzsbCwF/+AL2PsIWqitjaJPYGNhJQhiIRZaCIKF+hRW2stZiGyW3HuRpMiGM9XHpzc4Zzab2Znf5A4MXW7xbnMNmmC1STO5Fn1qTqdsuqWRMFGxSb9RnP41GwzthdLQMDw9PYUWQ65GNQM/zs7Owqa6sLBQcYWH8eXlZXF8fFzMz8+Hh9Xm5mbQNWxvb2flNhoS/mZau1Q2ON3ROhgbGytubm4qvvQ13iSZs7OzoT2EFqnO+hTzOMHa3d0t0J2RNJVGcsXLhEk2UyNBo2K9urr69yOS87W1taDd6rLFfsd/J22/ra2tgnvh8fGxNsHm4EjFmr0OvRZ7A1X6t7e3Ynp6ustuVxLqu7u7ENv9/f2wd7++voZki+Q4lYDgVN/iTSUavSCJJgn1IMs53k0XowlWU4JDPn99fR10SDxoSysrWGwwnHhKW19fD+1EBPClsTARjXKdHI2vT0DsikYD7VWaXA3yifYaGzS6rNyNqtzy8nJojcbVnD7Gm1jd398H3RGn/v8IlnONeZxo7O3tBTlAvG6pYHFY4OGbGvc/wwHxIAsVLKq9XKvLVpdgIQ3gcEQVGgZp1XaYP2h3aKeSmHfZBiWW5d/MYBIykJeXl97Hm4MzlTsOxf+1XOL9X7/S3zfBakpwyOcRMs/MzISHDZsOxs1H+4QKVmwIXOnlx6VWNFiIKJnOyc2o1NEeYxoS4X7d9CA+8fAh8YirVVQC0DfUTd51mQMncKpxnGrL6Sg2IATQPz8/la9D6lu8y7jQDvj+/q5UcdKY9Snm8QMXvRk6pI+Pjz+XSRrgUbeWV1ZWwtALk8NYqcHiQIWGrcuWJhpMQCN05kBJu69uSrr0h7VPlWtpaenPRar6fGXYsInLLvCI/Ua6wUEwnp49PDwMQwtU71LrU7zxbW5uLjzLRsk5co530zVngtWU4IjPU/KfnJwMU4RUstBbsLkcHBxUPvn5+Rk2HASVGxsbYYqQ9hKvOyBJy8nQl1EypvVBi6ROj1D6c3JyEjbk29vbYnFxMSSYJFtsUrDLydCY0R5Dd4KODG3Czs5OEMAigI6tT/GO/UJfSGuQJHmQ9Snm8QMXrSWJAi0jKnIk11QqB4nWuTf4XdrHtFp4EDEM8v7+Xhn77+I9EPv9/PwcJoDPz89HPmzxBSbsEWjxqHKhUTw9PS24J2gZdtliv0mE0ZlRnWevgwMJM+u7bnK2L/EmPqx1Ds0UEVjzwyzneDddiyZYTQmO+DwnHMrelFFJNCj/s6EwXTc+Ph7ee1NOX5BUcJql+jM1NRUmkqgC5Wac4mlvoj2K33ODH5zmY7/RaB0dHYUEhHeGIfQn+SQJzdEeHh7CSZxRdN6FhRaLOPLvvsY7jhM+Xl1dFVRf0/8v13qfYp5WcmgN0d5DCsCrKGijoMnDqGrAhQSKyi4cuFfQIPGOJV5rcXFxkcWBKvabpIKJYe732DggcWBK/f76+gr7A1UvKrtMSjOFRnLedUvjzZomhhyuJiYmgl9Ucfscb3xjnbNemZBlWCG2PsW76Xo0wWpK0M9LQAISkIAEJCCBhIAJlktCAhKQgAQkIAEJtEzABKtloF5OAhKQgAQkIAEJmGC5BiQgAQlIQAISkEDLBEywWgbq5SQgAQlIQAISkIAJlmtAAhKQgAQkIAEJtEzABKtloF5OAhKQgAQkIAEJmGC5BiQgAQlIQAISkEDLBEywWgbq5SQgAQlIQAISkIAJlmtAAhKQgAQkIAEJtEzABKtloF5OAhKQgAQkIAEJmGC5BiQgAQlIQAISkEDLBEywWgbq5SQgAQlIQAISkIAJlmtAAhKQgAQkIAEJtEzgFx19cseLAAAAA0lEQVTYND4AG8R2AAAAAElFTkSuQmCC\" width=\"600\">"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "[<matplotlib.lines.Line2D at 0x1d7c8c26588>]"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pb_nums = np.array(list(pb_num_dict.values()))\n",
    "plt.figure(figsize=(6,3), dpi=100)\n",
    "plt.plot(pb_nums)\n",
    "plt.plot(np.ones(len(pb_nums))*20)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 3. Load and asssemble probes"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3.1 design readout mapping dict (region_to_readout)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "36\n"
     ]
    }
   ],
   "source": [
    "num_readout_per_probe = 2\n",
    "total_readout_usage = 0\n",
    "region_2_readout_dict = {}\n",
    "for _i, (_k,_v) in enumerate(sorted(filtered_pb_dic.items())):\n",
    "    region_2_readout_dict[_k] = []\n",
    "    for _j in range(num_readout_per_probe):\n",
    "        region_2_readout_dict[_k].append(f\"u{int(_i+_j*len(filtered_pb_dic))}\")\n",
    "        total_readout_usage += 1\n",
    "print(total_readout_usage)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3.2 loading components"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### primers\n",
    "pick primer set 2, 9 for 5kb (consistent with CTP-06)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Picked primer: ID: W1A03_primer_2\n",
      "Name: W1A03_primer_2\n",
      "Description: W1A03_primer_2\n",
      "Number of features: 0\n",
      "Seq('CCCGCAATGGCTGACAACCG', SingleLetterAlphabet())\n",
      "- Picked primer: ID: W1A10_primer_9\n",
      "Name: W1A10_primer_9\n",
      "Description: W1A10_primer_9\n",
      "Number of features: 0\n",
      "Seq('TAATACGACTCACTATAGGGATTGCCGCATGGTTTCCG', SingleLetterAlphabet())\n"
     ]
    }
   ],
   "source": [
    "primer_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Primers'\n",
    "primers = library_tools.probes.load_primers([2,9], _primer_folder=primer_folder)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "readout_folder = r'\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Readouts'\n",
    "readout_start = 0\n",
    "num_color = 2\n",
    "readouts = library_tools.probes.load_readouts(total_readout_usage, \n",
    "                                              _type='designed_readouts', \n",
    "                                              _readout_folder=readout_folder, )\n",
    "\n",
    "readout_dict = {'u': readouts}"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3.3 assemble probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Assemble probes by given target sequences, readouts and primers.\n",
      "-- included readout types: ['u']\n",
      "[] ID: Stv_3\n",
      "Name: Stv_3\n",
      "Description: Stv_3 old_barcode\n",
      "Number of features: 0\n",
      "Seq('ACAAATCCGACCAGATCGGACGATCATGGG', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('ACAAATCCGACCAGATCGGACGATCATGGG', SingleLetterAlphabet()), id='Stv_3', name='Stv_3', description='Stv_3 old_barcode', dbxrefs=[])] ID: Stv_9\n",
      "Name: Stv_9\n",
      "Description: Stv_9 old_barcode\n",
      "Number of features: 0\n",
      "Seq('GCATGAGTTGCCTGGCGTTGCGACGACTAA', SingleLetterAlphabet())\n",
      "--- assemblying 113 probes in region: 1\n",
      "[] ID: Stv_32\n",
      "Name: Stv_32\n",
      "Description: Stv_32 old_barcode\n",
      "Number of features: 0\n",
      "Seq('AGACGACGCACGTTCGTACCGCGTACTTCG', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('AGACGACGCACGTTCGTACCGCGTACTTCG', SingleLetterAlphabet()), id='Stv_32', name='Stv_32', description='Stv_32 old_barcode', dbxrefs=[])] ID: Stv_40\n",
      "Name: Stv_40\n",
      "Description: Stv_40 old_barcode\n",
      "Number of features: 0\n",
      "Seq('AATCCGGCGTACGTAAGGCAGCTTGCGTTA', SingleLetterAlphabet())\n",
      "--- assemblying 106 probes in region: 2\n",
      "[] ID: Stv_91\n",
      "Name: Stv_91\n",
      "Description: Stv_91 old_barcode\n",
      "Number of features: 0\n",
      "Seq('TTCCCGCCAATGACGTCGGTTTGGACGAGA', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('TTCCCGCCAATGACGTCGGTTTGGACGAGA', SingleLetterAlphabet()), id='Stv_91', name='Stv_91', description='Stv_91 old_barcode', dbxrefs=[])] ID: Stv_101\n",
      "Name: Stv_101\n",
      "Description: Stv_101 old_barcode\n",
      "Number of features: 0\n",
      "Seq('ATGGCCACCCGACTGCGACGAACTCGATGC', SingleLetterAlphabet())\n",
      "--- assemblying 104 probes in region: 5\n",
      "[] ID: Stv_4\n",
      "Name: Stv_4\n",
      "Description: Stv_4 old_barcode\n",
      "Number of features: 0\n",
      "Seq('CAAGTATGCAGCGCGATTGACCGTCTCGTT', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('CAAGTATGCAGCGCGATTGACCGTCTCGTT', SingleLetterAlphabet()), id='Stv_4', name='Stv_4', description='Stv_4 old_barcode', dbxrefs=[])] ID: Stv_10\n",
      "Name: Stv_10\n",
      "Description: Stv_10 old_barcode\n",
      "Number of features: 0\n",
      "Seq('CCGTCGTCTCCGGTCCACCGTTGCGCTTAC', SingleLetterAlphabet())\n",
      "--- assemblying 104 probes in region: 6\n",
      "[] ID: Stv_33\n",
      "Name: Stv_33\n",
      "Description: Stv_33 old_barcode\n",
      "Number of features: 0\n",
      "Seq('TTTGCTCGCAAGTGCGCACGAGTTGAACTG', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('TTTGCTCGCAAGTGCGCACGAGTTGAACTG', SingleLetterAlphabet()), id='Stv_33', name='Stv_33', description='Stv_33 old_barcode', dbxrefs=[])] ID: Stv_42\n",
      "Name: Stv_42\n",
      "Description: Stv_42 old_barcode\n",
      "Number of features: 0\n",
      "Seq('GCGGGATTCGTACGAGCCCTCTTGGACGGG', SingleLetterAlphabet())\n",
      "--- assemblying 146 probes in region: 7\n",
      "[] ID: Stv_92\n",
      "Name: Stv_92\n",
      "Description: Stv_92 old_barcode\n",
      "Number of features: 0\n",
      "Seq('GCGTTGATGTCCCTTGTGAGCGCCCGACAT', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('GCGTTGATGTCCCTTGTGAGCGCCCGACAT', SingleLetterAlphabet()), id='Stv_92', name='Stv_92', description='Stv_92 old_barcode', dbxrefs=[])] ID: Stv_104\n",
      "Name: Stv_104\n",
      "Description: Stv_104 old_barcode\n",
      "Number of features: 0\n",
      "Seq('CTGGCGTTGCGCGATTGCCGCTTCAATCAA', SingleLetterAlphabet())\n",
      "--- assemblying 150 probes in region: 8\n",
      "[] ID: Stv_5\n",
      "Name: Stv_5\n",
      "Description: Stv_5 old_barcode\n",
      "Number of features: 0\n",
      "Seq('AAGTCGTACGCCGATGCGCAGCAATTCACT', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('AAGTCGTACGCCGATGCGCAGCAATTCACT', SingleLetterAlphabet()), id='Stv_5', name='Stv_5', description='Stv_5 old_barcode', dbxrefs=[])] ID: Stv_11\n",
      "Name: Stv_11\n",
      "Description: Stv_11 old_barcode\n",
      "Number of features: 0\n",
      "Seq('GGCCAATGGCCCAGGTCCGTCACGCAATTT', SingleLetterAlphabet())\n",
      "--- assemblying 150 probes in region: 9\n",
      "[] ID: Stv_35\n",
      "Name: Stv_35\n",
      "Description: Stv_35 old_barcode\n",
      "Number of features: 0\n",
      "Seq('AGTTTCCATACGCGGCGTGAATATCGCGGC', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('AGTTTCCATACGCGGCGTGAATATCGCGGC', SingleLetterAlphabet()), id='Stv_35', name='Stv_35', description='Stv_35 old_barcode', dbxrefs=[])] ID: Stv_44\n",
      "Name: Stv_44\n",
      "Description: Stv_44 old_barcode\n",
      "Number of features: 0\n",
      "Seq('CTTTGTTCGGCTCGGAGCGTTACTGCGGGC', SingleLetterAlphabet())\n",
      "--- assemblying 68 probes in region: 10\n",
      "[] ID: Stv_94\n",
      "Name: Stv_94\n",
      "Description: Stv_94 old_barcode\n",
      "Number of features: 0\n",
      "Seq('CGCTTATCGATGTCAGGTCCGCATGGGTCG', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('CGCTTATCGATGTCAGGTCCGCATGGGTCG', SingleLetterAlphabet()), id='Stv_94', name='Stv_94', description='Stv_94 old_barcode', dbxrefs=[])] ID: Stv_105\n",
      "Name: Stv_105\n",
      "Description: Stv_105 old_barcode\n",
      "Number of features: 0\n",
      "Seq('GGCACGCTTCAACAAACGGCACACTCGGAT', SingleLetterAlphabet())\n",
      "--- assemblying 150 probes in region: 13\n",
      "[] ID: Stv_6\n",
      "Name: Stv_6\n",
      "Description: Stv_6 old_barcode\n",
      "Number of features: 0\n",
      "Seq('CGAAACATCGGCCACGGTCCCGTTGAACTT', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('CGAAACATCGGCCACGGTCCCGTTGAACTT', SingleLetterAlphabet()), id='Stv_6', name='Stv_6', description='Stv_6 old_barcode', dbxrefs=[])] ID: Stv_12\n",
      "Name: Stv_12\n",
      "Description: Stv_12 old_barcode\n",
      "Number of features: 0\n",
      "Seq('TTGATCGAATCGGAGCGTAGCGGAATCTGC', SingleLetterAlphabet())\n",
      "--- assemblying 150 probes in region: 14\n",
      "[] ID: Stv_36\n",
      "Name: Stv_36\n",
      "Description: Stv_36 old_barcode\n",
      "Number of features: 0\n",
      "Seq('GCTACAGCGTTACGGGCCCAGACGTTTCAT', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('GCTACAGCGTTACGGGCCCAGACGTTTCAT', SingleLetterAlphabet()), id='Stv_36', name='Stv_36', description='Stv_36 old_barcode', dbxrefs=[])] ID: Stv_45\n",
      "Name: Stv_45\n",
      "Description: Stv_45 old_barcode\n",
      "Number of features: 0\n",
      "Seq('TAATTCCGGTATCGGGCCCTTTTGTCTGAC', SingleLetterAlphabet())\n",
      "--- assemblying 143 probes in region: 15\n",
      "[] ID: Stv_95\n",
      "Name: Stv_95\n",
      "Description: Stv_95 old_barcode\n",
      "Number of features: 0\n",
      "Seq('AACGTCATCGGTGGATCCCAGAGTGCCAAA', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('AACGTCATCGGTGGATCCCAGAGTGCCAAA', SingleLetterAlphabet()), id='Stv_95', name='Stv_95', description='Stv_95 old_barcode', dbxrefs=[])] ID: Stv_106\n",
      "Name: Stv_106\n",
      "Description: Stv_106 old_barcode\n",
      "Number of features: 0\n",
      "Seq('TGACTCCCGACCTGCGCACTTGCCCGTGTT', SingleLetterAlphabet())\n",
      "--- assemblying 118 probes in region: 16\n",
      "[] ID: Stv_7\n",
      "Name: Stv_7\n",
      "Description: Stv_7 old_barcode\n",
      "Number of features: 0\n",
      "Seq('ACGAATCCACCGTCCAGCGCGTCAAACAGA', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('ACGAATCCACCGTCCAGCGCGTCAAACAGA', SingleLetterAlphabet()), id='Stv_7', name='Stv_7', description='Stv_7 old_barcode', dbxrefs=[])] ID: Stv_13\n",
      "Name: Stv_13\n",
      "Description: Stv_13 old_barcode\n",
      "Number of features: 0\n",
      "Seq('CGCGCGGATCCGCTTGTCGGGAACGGATAC', SingleLetterAlphabet())\n",
      "--- assemblying 120 probes in region: 17\n",
      "[] ID: Stv_37\n",
      "Name: Stv_37\n",
      "Description: Stv_37 old_barcode\n",
      "Number of features: 0\n",
      "Seq('TTTATCGTGCGTGTCCGCCAGTACCGTGAG', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('TTTATCGTGCGTGTCCGCCAGTACCGTGAG', SingleLetterAlphabet()), id='Stv_37', name='Stv_37', description='Stv_37 old_barcode', dbxrefs=[])] ID: Stv_46\n",
      "Name: Stv_46\n",
      "Description: Stv_46 old_barcode\n",
      "Number of features: 0\n",
      "Seq('CGTGAGAATGCGTCGCGATACTGGTGTAAG', SingleLetterAlphabet())\n",
      "--- assemblying 68 probes in region: 18\n",
      "[] ID: Stv_99\n",
      "Name: Stv_99\n",
      "Description: Stv_99 old_barcode\n",
      "Number of features: 0\n",
      "Seq('GTCCGATGAAACGTCCCGTGTGCTGTCGCG', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('GTCCGATGAAACGTCCCGTGTGCTGTCGCG', SingleLetterAlphabet()), id='Stv_99', name='Stv_99', description='Stv_99 old_barcode', dbxrefs=[])] ID: Stv_107\n",
      "Name: Stv_107\n",
      "Description: Stv_107 old_barcode\n",
      "Number of features: 0\n",
      "Seq('TTCCCACACGCCAAGCGCAGACGAACGGCA', SingleLetterAlphabet())\n",
      "--- assemblying 44 probes in region: 19\n",
      "[] ID: Stv_8\n",
      "Name: Stv_8\n",
      "Description: Stv_8 old_barcode\n",
      "Number of features: 0\n",
      "Seq('CGCGAAATCCCCGTAACGAGCGTCCCTTGC', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('CGCGAAATCCCCGTAACGAGCGTCCCTTGC', SingleLetterAlphabet()), id='Stv_8', name='Stv_8', description='Stv_8 old_barcode', dbxrefs=[])] ID: Stv_14\n",
      "Name: Stv_14\n",
      "Description: Stv_14 old_barcode\n",
      "Number of features: 0\n",
      "Seq('GCCTCGATTACGACGGATGTAATTCGGCCG', SingleLetterAlphabet())\n",
      "--- assemblying 150 probes in region: 20\n",
      "[] ID: Stv_39\n",
      "Name: Stv_39\n",
      "Description: Stv_39 old_barcode\n",
      "Number of features: 0\n",
      "Seq('CGTGCGGGAACCGTATCCCTGGCGCGGACT', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('CGTGCGGGAACCGTATCCCTGGCGCGGACT', SingleLetterAlphabet()), id='Stv_39', name='Stv_39', description='Stv_39 old_barcode', dbxrefs=[])] ID: Stv_48\n",
      "Name: Stv_48\n",
      "Description: Stv_48 old_barcode\n",
      "Number of features: 0\n",
      "Seq('TCTCATGGTCGCCGGACCGTGATGCCGTGT', SingleLetterAlphabet())\n",
      "--- assemblying 150 probes in region: 21\n",
      "[] ID: Stv_100\n",
      "Name: Stv_100\n",
      "Description: Stv_100 old_barcode\n",
      "Number of features: 0\n",
      "Seq('GACACGACGTCCAAAGTTGGTCTCGCGCAA', SingleLetterAlphabet())\n",
      "[SeqRecord(seq=Seq('GACACGACGTCCAAAGTTGGTCTCGCGCAA', SingleLetterAlphabet()), id='Stv_100', name='Stv_100', description='Stv_100 old_barcode', dbxrefs=[])] ID: Stv_109\n",
      "Name: Stv_109\n",
      "Description: Stv_109 old_barcode\n",
      "Number of features: 0\n",
      "Seq('GCATCGCCCGAAATTCCCTGGTGCGGTAAC', SingleLetterAlphabet())\n",
      "--- assemblying 148 probes in region: 22\n",
      "-- 2182 probes assembled in total.\n",
      "-- saving 2182 probes into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\candidate_probes.fasta\n",
      "-- saving readout_summary into file:\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\readout_summary.pkl\n"
     ]
    }
   ],
   "source": [
    "reload(library_tools.probes)\n",
    "from ImageAnalysis3.library_tools.probes import Assemble_probes\n",
    "\n",
    "cand_probes, readout_summary = Assemble_probes(library_folder, filtered_pb_dic, \n",
    "                                               region_2_readout_dict, readout_dict, primers, \n",
    "                                               primer_len=19,\n",
    "                                               rc_targets=False, overwrite=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 4.Check quality"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "# biopython imports\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.Alphabet import IUPAC\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "from Bio.Blast.Applications import NcbiblastnCommandline\n",
    "from Bio.Blast import NCBIXML"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4.1 Load probes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total probe loaded: 2182\n"
     ]
    }
   ],
   "source": [
    "# folder for this library\n",
    "# candidate full-length probe filename\n",
    "candidate_full_name = 'candidate_probes.fasta'\n",
    "# load full probes\n",
    "full_records = []\n",
    "with open(os.path.join(library_folder, candidate_full_name), 'r') as handle:\n",
    "    for record in SeqIO.parse(handle, \"fasta\"):\n",
    "        full_records.append(record)\n",
    "\n",
    "print(f\"Total probe loaded: {len(full_records)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "num_readout_per_probe = 2\n",
    "primer_len=19\n",
    "min_num_probes = 25 "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Checking primer usage, total probes: 2182\n",
      "--- Forward primer incorrect!\n",
      "gene: 1 -> 113\n",
      "gene: 2 -> 106\n",
      "gene: 5 -> 104\n",
      "gene: 6 -> 104\n",
      "gene: 7 -> 146\n",
      "gene: 8 -> 150\n",
      "gene: 9 -> 150\n",
      "gene: 10 -> 68\n",
      "gene: 13 -> 150\n",
      "gene: 14 -> 150\n",
      "gene: 15 -> 143\n",
      "gene: 16 -> 118\n",
      "gene: 17 -> 120\n",
      "gene: 18 -> 68\n",
      "gene: 19 -> 44\n",
      "gene: 20 -> 150\n",
      "gene: 21 -> 150\n",
      "gene: 22 -> 148\n",
      "1 -> ['Stv_3', 'Stv_9']\n",
      "2 -> ['Stv_32', 'Stv_40']\n",
      "5 -> ['Stv_91', 'Stv_101']\n",
      "6 -> ['Stv_4', 'Stv_10']\n",
      "7 -> ['Stv_33', 'Stv_42']\n",
      "8 -> ['Stv_92', 'Stv_104']\n",
      "9 -> ['Stv_5', 'Stv_11']\n",
      "10 -> ['Stv_35', 'Stv_44']\n",
      "13 -> ['Stv_94', 'Stv_105']\n",
      "14 -> ['Stv_6', 'Stv_12']\n",
      "15 -> ['Stv_36', 'Stv_45']\n",
      "16 -> ['Stv_95', 'Stv_106']\n",
      "17 -> ['Stv_7', 'Stv_13']\n",
      "18 -> ['Stv_37', 'Stv_46']\n",
      "19 -> ['Stv_99', 'Stv_107']\n",
      "20 -> ['Stv_8', 'Stv_14']\n",
      "21 -> ['Stv_39', 'Stv_48']\n",
      "22 -> ['Stv_100', 'Stv_109']\n",
      "-- extracting probes from >region:1, region:2, region:5, region:6, region:7, region:8, region:9, region:10, region:13, region:14, region:15, region:16, region:17, region:18, region:19, region:20, region:21, region:22,  > Done.\n",
      "-- inverting region_to_readout dict\n",
      "Stv_10 -> ['6']\n",
      "Stv_100 -> ['22']\n",
      "Stv_101 -> ['5']\n",
      "Stv_104 -> ['8']\n",
      "Stv_105 -> ['13']\n",
      "Stv_106 -> ['16']\n",
      "Stv_107 -> ['19']\n",
      "Stv_109 -> ['22']\n",
      "Stv_11 -> ['9']\n",
      "Stv_12 -> ['14']\n",
      "Stv_13 -> ['17']\n",
      "Stv_14 -> ['20']\n",
      "Stv_3 -> ['1']\n",
      "Stv_32 -> ['2']\n",
      "Stv_33 -> ['7']\n",
      "Stv_35 -> ['10']\n",
      "Stv_36 -> ['15']\n",
      "Stv_37 -> ['18']\n",
      "Stv_39 -> ['21']\n",
      "Stv_4 -> ['6']\n",
      "Stv_40 -> ['2']\n",
      "Stv_42 -> ['7']\n",
      "Stv_44 -> ['10']\n",
      "Stv_45 -> ['15']\n",
      "Stv_46 -> ['18']\n",
      "Stv_48 -> ['21']\n",
      "Stv_5 -> ['9']\n",
      "Stv_6 -> ['14']\n",
      "Stv_7 -> ['17']\n",
      "Stv_8 -> ['20']\n",
      "Stv_9 -> ['1']\n",
      "Stv_91 -> ['5']\n",
      "Stv_92 -> ['8']\n",
      "Stv_94 -> ['13']\n",
      "Stv_95 -> ['16']\n",
      "Stv_99 -> ['19']\n"
     ]
    }
   ],
   "source": [
    "import ImageAnalysis3.library_tools.quality_check as check\n",
    "reload(check)\n",
    "primer_check = check._check_primer_usage(full_records, primers[0], primers[1], primer_len=primer_len)\n",
    "\n",
    "reg_size_dic, len_check = check._check_region_size(full_records, \n",
    "                                                   min_size=min_num_probes)\n",
    "\n",
    "reg_readout_dic, reg2readout_check = check._check_region_to_readouts(full_records, readout_dict, \n",
    "                                                                     total_readout_num=num_readout_per_probe)\n",
    "\n",
    "readout_reg_dic, readout2reg_check = check._check_readout_to_region(reg_readout_dic, full_records, \n",
    "                                                                    readout_dict, primer_len=primer_len, \n",
    "                                                                    probe_readout_num=num_readout_per_probe)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- saving internal 17-mer map to file:\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\probe_table_17.npz\n"
     ]
    }
   ],
   "source": [
    "# construct internal map\n",
    "int_map = check._construct_internal_map(full_records, library_folder) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- total probes removed by internal screening: 0\n"
     ]
    }
   ],
   "source": [
    "# internal check probes\n",
    "readout_count_dic, readout_count_check = check._check_readout_in_probes(readout_reg_dic, reg_size_dic, int_map, readout_dict)\n",
    "\n",
    "kept_records, removed_count = check._check_between_probes(full_records, int_map)\n",
    "# save kept records\n",
    "with open(os.path.join(library_folder, 'filtered_full_probes.fasta'), 'w') as output_handle:\n",
    "    SeqIO.write(kept_records, output_handle, \"fasta\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4.2 BLAST screening of probes\n",
    "\n",
    "create the local blast database by:\n",
    "```bash\n",
    "makeblastdb -in mm10.fa -parse_seqids -dbtype nucl -out mm10\n",
    "```\n",
    "\n",
    "The outputs are:\n",
    "```bash\n",
    "Building a new DB, current time: 11/02/2020 15:16:50\n",
    "New DB name:   Z:\\Libraries\\Genomes\\mouse\\mm10\\mm10\n",
    "New DB title:  mm10.fa\n",
    "Sequence type: Nucleotide\n",
    "Keep MBits: T\n",
    "Maximum file size: 1000000000B\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Blast probe against \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\Genomes\\mouse\\mm10\\mm10\n",
      "-- number of probes loaded: 2182\n",
      "--- processing region: 1 with 113 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_1.fasta\n",
      "-- *blasting region: 1\n",
      "--- total time for blast 1: 14.386693954467773\n",
      "--- processing region: 10 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_10.fasta\n",
      "-- *blasting region: 10\n",
      "--- total time for blast 10: 9.10476803779602\n",
      "--- processing region: 13 with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_13.fasta\n",
      "-- *blasting region: 13\n",
      "--- total time for blast 13: 12.779499530792236\n",
      "--- processing region: 14 with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_14.fasta\n",
      "-- *blasting region: 14\n",
      "--- total time for blast 14: 13.039693117141724\n",
      "--- processing region: 15 with 143 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_15.fasta\n",
      "-- *blasting region: 15\n",
      "--- total time for blast 15: 14.773982286453247\n",
      "--- processing region: 16 with 118 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_16.fasta\n",
      "-- *blasting region: 16\n",
      "--- total time for blast 16: 14.26560378074646\n",
      "--- processing region: 17 with 120 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_17.fasta\n",
      "-- *blasting region: 17\n",
      "--- total time for blast 17: 10.933127164840698\n",
      "--- processing region: 18 with 68 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_18.fasta\n",
      "-- *blasting region: 18\n",
      "--- total time for blast 18: 8.110029458999634\n",
      "--- processing region: 19 with 44 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_19.fasta\n",
      "-- *blasting region: 19\n",
      "--- total time for blast 19: 15.14625859260559\n",
      "--- processing region: 2 with 106 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_2.fasta\n",
      "-- *blasting region: 2\n",
      "--- total time for blast 2: 11.282386302947998\n",
      "--- processing region: 20 with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_20.fasta\n",
      "-- *blasting region: 20\n",
      "--- total time for blast 20: 11.648664236068726\n",
      "--- processing region: 21 with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_21.fasta\n",
      "-- *blasting region: 21\n",
      "--- total time for blast 21: 12.562338590621948\n",
      "--- processing region: 22 with 148 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_22.fasta\n",
      "-- *blasting region: 22\n",
      "--- total time for blast 22: 13.373941421508789\n",
      "--- processing region: 5 with 104 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_5.fasta\n",
      "-- *blasting region: 5\n",
      "--- total time for blast 5: 10.870117425918579\n",
      "--- processing region: 6 with 104 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_6.fasta\n",
      "-- *blasting region: 6\n",
      "--- total time for blast 6: 11.72567892074585\n",
      "--- processing region: 7 with 146 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_7.fasta\n",
      "-- *blasting region: 7\n",
      "--- total time for blast 7: 11.224343061447144\n",
      "--- processing region: 8 with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_8.fasta\n",
      "-- *blasting region: 8\n",
      "--- total time for blast 8: 14.392699718475342\n",
      "--- processing region: 9 with 150 probes\n",
      "--- *writing file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\blast\\probe_gene_9.fasta\n",
      "-- *blasting region: 9\n",
      "--- total time for blast 9: 12.826534509658813\n"
     ]
    }
   ],
   "source": [
    "check.Blast_probes(kept_records, library_folder, num_threads=32, blast_db=os.path.join(genome_folder, 'mm10'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.2.1 screen probe by blast results\n",
    "\n",
    "This step has 2 different cutoffs:\n",
    "\n",
    "hard_cutoff: whether a probe has any ~40mer hit somewhere else, if so just dump the probe\n",
    "\n",
    "soft_cutoff: whether a probe has a lot of ~20mer off-target hits, if exceed certain amount then dump the probe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "- Number of probes loaded: 2182\n",
      "- Number of regions in this library: 18\n",
      "-- checking probes in region: 1\n",
      "--- gene=1, id=16 removed by soft count = 95\n",
      "--- gene=1, id=26 removed by soft count = 138\n",
      "--- gene=1, id=27 removed by soft count = 158\n",
      "--- gene=1, id=33 removed by soft count = 32\n",
      "--- gene=1, id=34 removed by soft count = 34\n",
      "--- gene=1, id=66 removed by soft count = 2021\n",
      "--- gene=1, id=72 removed by soft count = 58\n",
      "--- gene=1, id=88 removed by soft count = 136\n",
      "--- number of probes: 113 , kept by blast: 105 , if remove dups: 105\n",
      "-- number of probes kept for this region: 105\n",
      "-- checking probes in region: 2\n",
      "--- gene=2, id=30 removed by soft count = 46\n",
      "--- gene=2, id=83 removed by soft count = 368\n",
      "--- gene=2, id=88 removed by soft count = 78\n",
      "--- gene=2, id=89 removed by soft count = 278\n",
      "--- gene=2, id=94 removed by soft count = 167\n",
      "--- number of probes: 106 , kept by blast: 101 , if remove dups: 101\n",
      "-- number of probes kept for this region: 101\n",
      "-- checking probes in region: 5\n",
      "--- gene=5, id=15 removed by soft count = 62\n",
      "--- gene=5, id=59 removed by soft count = 345\n",
      "--- gene=5, id=60 removed by soft count = 40\n",
      "--- gene=5, id=61 removed by hard count = 0\n",
      "--- gene=5, id=64 removed by soft count = 88\n",
      "--- gene=5, id=74 removed by soft count = 247\n",
      "--- number of probes: 104 , kept by blast: 98 , if remove dups: 98\n",
      "-- number of probes kept for this region: 98\n",
      "-- checking probes in region: 6\n",
      "--- gene=6, id=16 removed by soft count = 31\n",
      "--- gene=6, id=28 removed by soft count = 31\n",
      "--- gene=6, id=30 removed by soft count = 159\n",
      "--- gene=6, id=56 removed by soft count = 46\n",
      "--- number of probes: 104 , kept by blast: 100 , if remove dups: 100\n",
      "-- number of probes kept for this region: 100\n",
      "-- checking probes in region: 7\n",
      "--- gene=7, id=6 removed by soft count = 62\n",
      "--- gene=7, id=9 removed by soft count = 42\n",
      "--- gene=7, id=21 removed by hard count = 4\n",
      "--- gene=7, id=106 removed by soft count = 36\n",
      "--- number of probes: 146 , kept by blast: 142 , if remove dups: 142\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 142\n",
      "-- checking probes in region: 8\n",
      "--- gene=8, id=37 removed by hard count = 0\n",
      "--- gene=8, id=83 removed by soft count = 35\n",
      "--- gene=8, id=103 removed by soft count = 39\n",
      "--- number of probes: 150 , kept by blast: 147 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 146\n",
      "-- checking probes in region: 9\n",
      "--- gene=9, id=102 removed by soft count = 371\n",
      "--- gene=9, id=108 removed by soft count = 68\n",
      "--- gene=9, id=126 removed by soft count = 47\n",
      "--- number of probes: 150 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 147\n",
      "-- checking probes in region: 10\n",
      "--- gene=10, id=28 removed by soft count = 4578\n",
      "--- number of probes: 68 , kept by blast: 67 , if remove dups: 67\n",
      "-- number of probes kept for this region: 67\n",
      "-- checking probes in region: 13\n",
      "--- gene=13, id=113 removed by soft count = 85\n",
      "--- number of probes: 150 , kept by blast: 149 , if remove dups: 149\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 149\n",
      "-- checking probes in region: 14\n",
      "--- gene=14, id=43 removed by soft count = 33\n",
      "--- gene=14, id=56 removed by soft count = 86\n",
      "--- gene=14, id=132 removed by soft count = 31\n",
      "--- number of probes: 150 , kept by blast: 147 , if remove dups: 147\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 147\n",
      "-- checking probes in region: 15\n",
      "--- gene=15, id=32 removed by soft count = 89\n",
      "--- gene=15, id=34 removed by soft count = 76\n",
      "--- gene=15, id=39 removed by soft count = 82\n",
      "--- gene=15, id=69 removed by soft count = 41\n",
      "--- gene=15, id=95 removed by soft count = 269\n",
      "--- gene=15, id=140 removed by soft count = 695\n",
      "--- number of probes: 143 , kept by blast: 137 , if remove dups: 137\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 137\n",
      "-- checking probes in region: 16\n",
      "--- gene=16, id=0 removed by soft count = 276\n",
      "--- gene=16, id=104 removed by soft count = 53\n",
      "--- gene=16, id=106 removed by soft count = 35\n",
      "--- number of probes: 118 , kept by blast: 115 , if remove dups: 115\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 115\n",
      "-- checking probes in region: 17\n",
      "--- gene=17, id=2 removed by soft count = 33\n",
      "--- gene=17, id=75 removed by soft count = 67\n",
      "--- number of probes: 120 , kept by blast: 118 , if remove dups: 118\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 118\n",
      "-- checking probes in region: 18\n",
      "--- gene=18, id=2 removed by soft count = 55\n",
      "--- gene=18, id=20 removed by soft count = 50\n",
      "--- gene=18, id=55 removed by soft count = 55\n",
      "--- gene=18, id=58 removed by soft count = 157\n",
      "--- gene=18, id=61 removed by soft count = 315\n",
      "--- gene=18, id=65 removed by soft count = 42\n",
      "--- gene=18, id=66 removed by soft count = 39\n",
      "--- gene=18, id=67 removed by soft count = 48\n",
      "--- number of probes: 68 , kept by blast: 60 , if remove dups: 60\n",
      "-- number of probes kept for this region: 60\n",
      "-- checking probes in region: 19\n",
      "--- gene=19, id=1 removed by soft count = 152\n",
      "--- gene=19, id=2 removed by soft count = 44\n",
      "--- gene=19, id=3 removed by soft count = 65\n",
      "--- gene=19, id=4 removed by soft count = 36\n",
      "--- gene=19, id=5 removed by soft count = 167\n",
      "--- gene=19, id=6 removed by soft count = 104\n",
      "--- gene=19, id=8 removed by soft count = 76\n",
      "--- gene=19, id=12 removed by soft count = 11102\n",
      "--- gene=19, id=14 removed by soft count = 203\n",
      "--- gene=19, id=16 removed by soft count = 32\n",
      "--- gene=19, id=22 removed by soft count = 111\n",
      "--- gene=19, id=24 removed by soft count = 64\n",
      "--- gene=19, id=25 removed by soft count = 48\n",
      "--- gene=19, id=26 removed by soft count = 2283\n",
      "--- gene=19, id=28 removed by soft count = 78\n",
      "--- gene=19, id=31 removed by soft count = 80\n",
      "--- gene=19, id=32 removed by soft count = 461\n",
      "--- gene=19, id=36 removed by soft count = 41\n",
      "--- number of probes: 44 , kept by blast: 26 , if remove dups: 26\n",
      "-- number of probes kept for this region: 26\n",
      "-- checking probes in region: 20\n",
      "--- gene=20, id=6 removed by soft count = 158\n",
      "--- gene=20, id=7 removed by soft count = 379\n",
      "--- gene=20, id=52 removed by hard count = 0\n",
      "--- gene=20, id=53 removed by soft count = 39\n",
      "--- gene=20, id=119 removed by soft count = 39\n",
      "--- gene=20, id=120 removed by soft count = 194\n",
      "--- number of probes: 150 , kept by blast: 144 , if remove dups: 144\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 144\n",
      "-- checking probes in region: 21\n",
      "--- gene=21, id=15 removed by soft count = 389\n",
      "--- gene=21, id=23 removed by soft count = 63\n",
      "--- gene=21, id=107 removed by soft count = 41\n",
      "--- gene=21, id=145 removed by soft count = 572\n",
      "--- number of probes: 150 , kept by blast: 146 , if remove dups: 146\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 146\n",
      "-- checking probes in region: 22\n",
      "--- gene=22, id=24 removed by soft count = 35\n",
      "--- gene=22, id=57 removed by soft count = 57\n",
      "--- gene=22, id=80 removed by soft count = 133\n",
      "--- gene=22, id=83 removed by soft count = 99\n",
      "--- number of probes: 148 , kept by blast: 144 , if remove dups: 143\n",
      "--- remove duplicated probes\n",
      "-- number of probes kept for this region: 143\n",
      "- summarize\n",
      "-- region: 1 , number of probes: 105\n",
      "-- region: 2 , number of probes: 101\n",
      "-- region: 5 , number of probes: 98\n",
      "-- region: 6 , number of probes: 100\n",
      "-- region: 7 , number of probes: 142\n",
      "-- region: 8 , number of probes: 146\n",
      "-- region: 9 , number of probes: 147\n",
      "-- region: 10 , number of probes: 67\n",
      "-- region: 13 , number of probes: 149\n",
      "-- region: 14 , number of probes: 147\n",
      "-- region: 15 , number of probes: 137\n",
      "-- region: 16 , number of probes: 115\n",
      "-- region: 17 , number of probes: 118\n",
      "-- region: 18 , number of probes: 60\n",
      "-- region: 19 , number of probes: 26\n",
      "-- region: 20 , number of probes: 144\n",
      "-- region: 21 , number of probes: 146\n",
      "-- region: 22 , number of probes: 143\n",
      "- Number of probes kept: 2091\n",
      "- Saving to file: \\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\.\\blast_full_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "# screen\n",
    "num_probes_per_region = 150\n",
    "kept_pbs, blast_keep_dic, hard_count_list, soft_count_list = check.Screening_Probes_by_Blast(library_folder, \n",
    "                                                                                             num_probes_per_region,\n",
    "                                                                                             )"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4.3 an extra round of check (could be skipped)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "num_readout_per_probe = 2\n",
    "primer_len=19\n",
    "min_num_probes = 25 "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "-- Checking primer usage, total probes: 2182\n",
      "--- Forward primer incorrect!\n",
      "False\n",
      "gene: 1 -> 105\n",
      "gene: 2 -> 101\n",
      "gene: 5 -> 98\n",
      "gene: 6 -> 100\n",
      "gene: 7 -> 142\n",
      "gene: 8 -> 146\n",
      "gene: 9 -> 147\n",
      "gene: 10 -> 67\n",
      "gene: 13 -> 149\n",
      "gene: 14 -> 147\n",
      "gene: 15 -> 137\n",
      "gene: 16 -> 115\n",
      "gene: 17 -> 118\n",
      "gene: 18 -> 60\n",
      "gene: 19 -> 26\n",
      "gene: 20 -> 144\n",
      "gene: 21 -> 146\n",
      "gene: 22 -> 143\n",
      "True\n",
      "1 -> ['Stv_3', 'Stv_9']\n",
      "2 -> ['Stv_32', 'Stv_40']\n",
      "5 -> ['Stv_91', 'Stv_101']\n",
      "6 -> ['Stv_4', 'Stv_10']\n",
      "7 -> ['Stv_33', 'Stv_42']\n",
      "8 -> ['Stv_92', 'Stv_104']\n",
      "9 -> ['Stv_5', 'Stv_11']\n",
      "10 -> ['Stv_35', 'Stv_44']\n",
      "13 -> ['Stv_94', 'Stv_105']\n",
      "14 -> ['Stv_6', 'Stv_12']\n",
      "15 -> ['Stv_36', 'Stv_45']\n",
      "16 -> ['Stv_95', 'Stv_106']\n",
      "17 -> ['Stv_7', 'Stv_13']\n",
      "18 -> ['Stv_37', 'Stv_46']\n",
      "19 -> ['Stv_99', 'Stv_107']\n",
      "20 -> ['Stv_8', 'Stv_14']\n",
      "21 -> ['Stv_39', 'Stv_48']\n",
      "22 -> ['Stv_100', 'Stv_109']\n",
      "True\n",
      "-- extracting probes from >region:1, region:2, region:5, region:6, region:7, region:8, region:9, region:10, region:13, region:14, region:15, region:16, region:17, region:18, region:19, region:20, region:21, region:22,  > Done.\n",
      "-- inverting region_to_readout dict\n",
      "Stv_10 -> ['6']\n",
      "Stv_100 -> ['22']\n",
      "Stv_101 -> ['5']\n",
      "Stv_104 -> ['8']\n",
      "Stv_105 -> ['13']\n",
      "Stv_106 -> ['16']\n",
      "Stv_107 -> ['19']\n",
      "Stv_109 -> ['22']\n",
      "Stv_11 -> ['9']\n",
      "Stv_12 -> ['14']\n",
      "Stv_13 -> ['17']\n",
      "Stv_14 -> ['20']\n",
      "Stv_3 -> ['1']\n",
      "Stv_32 -> ['2']\n",
      "Stv_33 -> ['7']\n",
      "Stv_35 -> ['10']\n",
      "Stv_36 -> ['15']\n",
      "Stv_37 -> ['18']\n",
      "Stv_39 -> ['21']\n",
      "Stv_4 -> ['6']\n",
      "Stv_40 -> ['2']\n",
      "Stv_42 -> ['7']\n",
      "Stv_44 -> ['10']\n",
      "Stv_45 -> ['15']\n",
      "Stv_46 -> ['18']\n",
      "Stv_48 -> ['21']\n",
      "Stv_5 -> ['9']\n",
      "Stv_6 -> ['14']\n",
      "Stv_7 -> ['17']\n",
      "Stv_8 -> ['20']\n",
      "Stv_9 -> ['1']\n",
      "Stv_91 -> ['5']\n",
      "Stv_92 -> ['8']\n",
      "Stv_94 -> ['13']\n",
      "Stv_95 -> ['16']\n",
      "Stv_99 -> ['19']\n",
      "True\n",
      "-- saving internal 17-mer map to file:\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\probe_table_17.npz\n",
      "True\n",
      "-- total probes removed by internal screening: 0\n"
     ]
    }
   ],
   "source": [
    "# check primers\n",
    "primer_check = check._check_primer_usage(full_records, primers[0], primers[1], \n",
    "                                         primer_len=primer_len)\n",
    "print(primer_check)\n",
    "# check region probe number (size)\n",
    "reg_size_dic, len_check = check._check_region_size(kept_pbs,\n",
    "                                                   min_size=min_num_probes)\n",
    "print(len_check)\n",
    "# check whether readouts in each region are correct\n",
    "reg_readout_dic, reg2readout_check = check._check_region_to_readouts(kept_pbs, readout_dict, \n",
    "                                                                     total_readout_num=num_readout_per_probe)\n",
    "print(reg2readout_check)\n",
    "# check readouts vs regions\n",
    "readout_reg_dic, readout2reg_check = check._check_readout_to_region(reg_readout_dic, kept_pbs, \n",
    "                                                                    readout_dict, \n",
    "                                                                    primer_len=primer_len, \n",
    "                                                                    probe_readout_num=num_readout_per_probe)\n",
    "print(readout2reg_check)\n",
    "# build internal check map\n",
    "int_map = check._construct_internal_map(kept_pbs, library_folder)\n",
    "# in each probe check readouts\n",
    "readout_count_dic, readout_count_check = check._check_readout_in_probes(readout_reg_dic, reg_size_dic, \n",
    "                                                                        int_map, readout_dict)\n",
    "print(readout_count_check)\n",
    "# check between probes\n",
    "kept_records, removed_count = check._check_between_probes(kept_pbs, int_map)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 5. Save"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 5.1 save fasta"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\\\\10.245.74.212\\Chromatin_NAS_2\\Libraries\\CTP-09_MOP\\Slc17a7_10kb\\final_probes\\extra_filtered_full_probes.fasta\n"
     ]
    }
   ],
   "source": [
    "final_probe_folder = os.path.join(library_folder, 'final_probes')\n",
    "if not os.path.exists(final_probe_folder):\n",
    "    os.makedirs(final_probe_folder)\n",
    "    \n",
    "# save kept records\n",
    "final_savefile = os.path.join(library_folder, 'final_probes', 'extra_filtered_full_probes.fasta')\n",
    "with open(final_savefile, 'w') as output_handle:\n",
    "    print(final_savefile)\n",
    "    SeqIO.write(kept_records, output_handle, \"fasta\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 5.2 save csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [],
   "source": [
    "import csv\n",
    "\n",
    "with open(os.path.join(final_probe_folder, 'extra_filtered_full_probes.csv'), 'w', newline='') as _file:\n",
    "    _writer = csv.writer(_file, dialect='excel')\n",
    "    # write csv\n",
    "    for _r in kept_records:\n",
    "        _writer.writerow([_r.id, str(_r.seq)])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## This section is to parse readouts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# biopython\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.Alphabet import IUPAC\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "from Bio.Blast.Applications import NcbiblastnCommandline\n",
    "from Bio.Blast import NCBIXML"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "all_ndbs = []\n",
    "with open(os.path.join(readout_folder, 'updated_NDBs.fasta'), 'r') as _rd_handle:\n",
    "    for _readout in SeqIO.parse(_rd_handle, \"fasta\"):\n",
    "        all_ndbs.append(_readout)\n",
    "all_ndbs = [_ndb for _ndb in sorted(all_ndbs, key=lambda v:int(v.name.split('_')[1]))]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "all_stvs = []\n",
    "with open(os.path.join(readout_folder, 'updated_Stvs.fasta'), 'r') as _rd_handle:\n",
    "    for _readout in SeqIO.parse(_rd_handle, \"fasta\"):\n",
    "        all_stvs.append(_readout)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "len(all_stvs) + len(all_ndbs)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# split by channel\n",
    "readout_by_channel = {'750':all_stvs[:25], '647':all_stvs[25:50], '561':all_stvs[50:]}\n",
    "for _i, _ndb in enumerate(all_ndbs):\n",
    "    _ndb_id = int(_ndb.name.split('NDB_')[1])\n",
    "    if _ndb_id % 3 == 1:\n",
    "        readout_by_channel['750'].append(_ndb)\n",
    "    elif _ndb_id % 3 == 2:\n",
    "        readout_by_channel['647'].append(_ndb)\n",
    "    elif _ndb_id % 3 == 0:\n",
    "        readout_by_channel['561'].append(_ndb)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "readout_by_channel['561']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "readout_by_channel['647']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for _k, _v in readout_by_channel.items():\n",
    "    _save_filename = os.path.join(readout_folder, f'designed_readouts_{_k}.fasta')\n",
    "    print(_save_filename, len(_v))\n",
    "    with open(_save_filename, 'w') as _output_handle:\n",
    "        SeqIO.write(_v, _output_handle, \"fasta\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": true,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": false,
   "toc_position": {},
   "toc_section_display": true,
   "toc_window_display": false
  },
  "varInspector": {
   "cols": {
    "lenName": 16,
    "lenType": 16,
    "lenVar": 40
   },
   "kernels_config": {
    "python": {
     "delete_cmd_postfix": "",
     "delete_cmd_prefix": "del ",
     "library": "var_list.py",
     "varRefreshCmd": "print(var_dic_list())"
    },
    "r": {
     "delete_cmd_postfix": ") ",
     "delete_cmd_prefix": "rm(",
     "library": "var_list.r",
     "varRefreshCmd": "cat(var_dic_list()) "
    }
   },
   "types_to_exclude": [
    "module",
    "function",
    "builtin_function_or_method",
    "instance",
    "_Feature"
   ],
   "window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
